<html lang="en" class="js"><head><link rel="preload" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" as="font" type="font/woff2" crossorigin=""><script async="" src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script type="text/javascript" async="" src="https://pagead2.googlesyndication.com/tag/js/gpt.js"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-MRVXSHQ"></script><script type="text/javascript" src="https://www.nature.com/vxgW9a7jI4F_rvjgMlIyRiJXeO1cnkRgvpM_SusVafTcncuM5uvB5JN-RJgBtMOK8ZEZvBuSAtqXXyEaTXp68w=="></script><script src="https://cmp.nature.com/production_live/en/consent-bundle-8-85.js" onload="initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')"></script>
    <title>Detection of early seeding of Richter transformation in chronic lymphocytic leukemia | Nature Medicine</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nm.rss">


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin="">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480">

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"b-cell-lymphoma;cancer-epigenetics;cancer-genomics;chronic-lymphocytic-leukaemia;tumour-heterogeneity","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Medicine","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41591-022-01927-8"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Ferran Nadeu","Romina Royo","Ramon Massoni-Badosa","Heribert Playa-Albinyana","Beatriz Garcia-Torre","Martí Duran-Ferrer","Kevin J. Dawson","Marta Kulis","Ander Diaz-Navarro","Neus Villamor","Juan L. Melero","Vicente Chapaprieta","Ana Dueso-Barroso","Julio Delgado","Riccardo Moia","Sara Ruiz-Gil","Domenica Marchese","Ariadna Giró","Núria Verdaguer-Dot","Mónica Romo","Guillem Clot","Maria Rozman","Gerard Frigola","Alfredo Rivas-Delgado","Tycho Baumann","Miguel Alcoceba","Marcos González","Fina Climent","Pau Abrisqueta","Josep Castellví","Francesc Bosch","Marta Aymerich","Anna Enjuanes","Sílvia Ruiz-Gaspà","Armando López-Guillermo","Pedro Jares","Sílvia Beà","Salvador Capella-Gutierrez","Josep Ll. Gelpí","Núria López-Bigas","David Torrents","Peter J. Campbell","Ivo Gut","Davide Rossi","Gianluca Gaidano","Xose S. Puente","Pablo M. Garcia-Roves","Dolors Colomer","Holger Heyn","Francesco Maura","José I. Martín-Subero","Elías Campo"],"publishedAt":1660176000,"publishedAtString":"2022-08-11","title":"Detection of early seeding of Richter transformation in chronic lymphocytic leukemia","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"B-cell lymphoma,Cancer epigenetics,Cancer genomics,Chronic lymphocytic leukaemia,Tumour heterogeneity"},"journal":{"pcode":"nm","title":"nature medicine","volume":"28","issue":"8","id":41591,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"BR","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{font-weight:700;-webkit-font-smoothing:antialiased;font-size:1.5rem;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.app-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{margin:4px 4px 0;fill:#888;height:10px;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-c9121fecab.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png">
<link rel="icon" type="image/png" sizes="48x48" href="/static/images/favicons/nature/favicon-48x48-b52890008c.png">
<link rel="icon" type="image/png" sizes="32x32" href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="/static/images/favicons/nature/favicon-16x16-951651ab72.png">
<link rel="manifest" href="/static/manifest.json" crossorigin="use-credentials">
<link rel="mask-icon" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="/static/images/favicons/nature/favicon.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-e9d2061c95.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-1cbf3a8b56.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script><script id="js-position1" data-test="global-article-js" defer="" src="/static/js/global-article-es6-bundle-c8a573ca90.js"></script><script id="js-position2" data-test="shared-js" defer="" src="/static/js/shared-es6-bundle-e9d2061c95.js"></script><script id="js-position3" data-test="header-150-js" defer="" src="/static/js/header-150-es6-bundle-5bb959eaa1.js"></script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Detection of early seeding of Richter transformation in chronic lymphocytic leukemia","description":"Richter transformation (RT) is a paradigmatic evolution of chronic lymphocytic leukemia (CLL) into a very aggressive large B cell lymphoma conferring a dismal prognosis. The mechanisms driving RT remain largely unknown. We characterized the whole genome, epigenome and transcriptome, combined with single-cell DNA/RNA-sequencing analyses and functional experiments, of 19 cases of CLL developing RT. Studying 54 longitudinal samples covering up to 19 years of disease course, we uncovered minute subclones carrying genomic, immunogenetic and transcriptomic features of RT cells already at CLL diagnosis, which were dormant for up to 19 years before transformation. We also identified new driver alterations, discovered a new mutational signature (SBS-RT), recognized an oxidative phosphorylation (OXPHOS)high–B cell receptor (BCR)low-signaling transcriptional axis in RT and showed that OXPHOS inhibition reduces the proliferation of RT cells. These findings demonstrate the early seeding of subclones driving advanced stages of cancer evolution and uncover potential therapeutic targets for RT. Single-cell genomic and transcriptomic analyses of longitudinal samples of patients with Richter syndrome reveal the presence and dynamics of clones driving transformation from chronic lymphocytic leukemia years before clinical manifestation","datePublished":"2022-08-11T00:00:00Z","dateModified":"2022-08-11T00:00:00Z","pageStart":"1662","pageEnd":"1671","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41591-022-01927-8","keywords":["B-cell lymphoma","Cancer epigenetics","Cancer genomics","Chronic lymphocytic leukaemia","Tumour heterogeneity","Biomedicine","general","Cancer Research","Metabolic Diseases","Infectious Diseases","Molecular Medicine","Neurosciences"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig5_HTML.png"],"isPartOf":{"name":"Nature Medicine","issn":["1546-170X","1078-8956"],"volumeNumber":"28","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Ferran Nadeu","url":"http://orcid.org/0000-0003-2910-9440","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"}],"email":"nadeu@recerca.clinic.cat","@type":"Person"},{"name":"Romina Royo","url":"http://orcid.org/0000-0003-3493-0874","affiliation":[{"name":"Barcelona Supercomputing Center (BSC)","address":{"name":"Barcelona Supercomputing Center (BSC), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ramon Massoni-Badosa","affiliation":[{"name":"CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST)","address":{"name":"CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Heribert Playa-Albinyana","url":"http://orcid.org/0000-0002-3447-3848","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Beatriz Garcia-Torre","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Martí Duran-Ferrer","url":"http://orcid.org/0000-0003-1666-5819","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Kevin J. Dawson","affiliation":[{"name":"Wellcome Sanger Institute","address":{"name":"Wellcome Sanger Institute, Hinxton, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marta Kulis","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ander Diaz-Navarro","affiliation":[{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Instituto Universitario de Oncología, Universidad de Oviedo","address":{"name":"Departamento de Bioquímica y Biología Molecular, Instituto Universitario de Oncología, Universidad de Oviedo, Oviedo, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Neus Villamor","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Juan L. Melero","url":"http://orcid.org/0000-0003-1387-6231","affiliation":[{"name":"Omniscope","address":{"name":"Omniscope, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Vicente Chapaprieta","url":"http://orcid.org/0000-0002-4910-5394","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ana Dueso-Barroso","url":"http://orcid.org/0000-0003-2000-9718","affiliation":[{"name":"Barcelona Supercomputing Center (BSC)","address":{"name":"Barcelona Supercomputing Center (BSC), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Julio Delgado","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat de Barcelona","address":{"name":"Universitat de Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Riccardo Moia","url":"http://orcid.org/0000-0001-7393-1138","affiliation":[{"name":"University of Eastern Piedmont","address":{"name":"Division of Hematology, Department of Translational Medicine, University of Eastern Piedmont, Novara, Italy","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sara Ruiz-Gil","affiliation":[{"name":"CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST)","address":{"name":"CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Domenica Marchese","affiliation":[{"name":"CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST)","address":{"name":"CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ariadna Giró","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Núria Verdaguer-Dot","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Mónica Romo","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Guillem Clot","url":"http://orcid.org/0000-0003-2588-7413","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Maria Rozman","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Gerard Frigola","url":"http://orcid.org/0000-0001-6794-6456","affiliation":[{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Alfredo Rivas-Delgado","url":"http://orcid.org/0000-0003-0385-3415","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Tycho Baumann","affiliation":[{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Universitario 12 de Octubre","address":{"name":"Hospital Universitario 12 de Octubre, Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Miguel Alcoceba","url":"http://orcid.org/0000-0002-3819-4846","affiliation":[{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"IBSAL-Hospital Universitario, Centro de Investigación del Cáncer-IBMCC (USAL-CSIC)","address":{"name":"Biología Molecular e Histocompatibilidad, IBSAL-Hospital Universitario, Centro de Investigación del Cáncer-IBMCC (USAL-CSIC), Salamanca, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marcos González","affiliation":[{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"IBSAL-Hospital Universitario, Centro de Investigación del Cáncer-IBMCC (USAL-CSIC)","address":{"name":"Biología Molecular e Histocompatibilidad, IBSAL-Hospital Universitario, Centro de Investigación del Cáncer-IBMCC (USAL-CSIC), Salamanca, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Fina Climent","affiliation":[{"name":"Hospital Universitari de Bellvitge-Institut d’Investigació Biomédica de Bellvitge (IDIBELL), L’Hospitalet de Llobregat","address":{"name":"Hospital Universitari de Bellvitge-Institut d’Investigació Biomédica de Bellvitge (IDIBELL), L’Hospitalet de Llobregat, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Pau Abrisqueta","affiliation":[{"name":"Vall d’Hebron University Hospital","address":{"name":"Department of Hematology, Vall d’Hebron Institute of Oncology, Vall d’Hebron University Hospital, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Josep Castellví","url":"http://orcid.org/0000-0001-5745-9996","affiliation":[{"name":"Vall d’Hebron University Hospital","address":{"name":"Department of Hematology, Vall d’Hebron Institute of Oncology, Vall d’Hebron University Hospital, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Francesc Bosch","affiliation":[{"name":"Vall d’Hebron University Hospital","address":{"name":"Department of Hematology, Vall d’Hebron Institute of Oncology, Vall d’Hebron University Hospital, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marta Aymerich","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Anna Enjuanes","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sílvia Ruiz-Gaspà","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Armando López-Guillermo","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat de Barcelona","address":{"name":"Universitat de Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Pedro Jares","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat de Barcelona","address":{"name":"Universitat de Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sílvia Beà","url":"http://orcid.org/0000-0001-7192-2385","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat de Barcelona","address":{"name":"Universitat de Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Salvador Capella-Gutierrez","url":"http://orcid.org/0000-0002-0309-604X","affiliation":[{"name":"Barcelona Supercomputing Center (BSC)","address":{"name":"Barcelona Supercomputing Center (BSC), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Josep Ll. Gelpí","affiliation":[{"name":"Barcelona Supercomputing Center (BSC)","address":{"name":"Barcelona Supercomputing Center (BSC), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat de Barcelona","address":{"name":"Universitat de Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Núria López-Bigas","url":"http://orcid.org/0000-0003-4925-8988","affiliation":[{"name":"The Barcelona Institute of Science and Technology","address":{"name":"Institute for Research in Biomedicine (IRB Barcelona), The Barcelona Institute of Science and Technology, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat Pompeu Fabra (UPF)","address":{"name":"Universitat Pompeu Fabra (UPF), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Institució Catalana de Recerca i Estudis Avançats (ICREA)","address":{"name":"Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"David Torrents","affiliation":[{"name":"Barcelona Supercomputing Center (BSC)","address":{"name":"Barcelona Supercomputing Center (BSC), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Institució Catalana de Recerca i Estudis Avançats (ICREA)","address":{"name":"Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Peter J. Campbell","url":"http://orcid.org/0000-0002-3921-0510","affiliation":[{"name":"Wellcome Sanger Institute","address":{"name":"Wellcome Sanger Institute, Hinxton, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ivo Gut","url":"http://orcid.org/0000-0001-7219-632X","affiliation":[{"name":"CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST)","address":{"name":"CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat Pompeu Fabra (UPF)","address":{"name":"Universitat Pompeu Fabra (UPF), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Davide Rossi","affiliation":[{"name":"Oncology Institute of Southern Switzerland","address":{"name":"Oncology Institute of Southern Switzerland, Bellinzona, Switzerland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Gianluca Gaidano","url":"http://orcid.org/0000-0002-4681-0151","affiliation":[{"name":"University of Eastern Piedmont","address":{"name":"Division of Hematology, Department of Translational Medicine, University of Eastern Piedmont, Novara, Italy","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Xose S. Puente","url":"http://orcid.org/0000-0001-9525-1483","affiliation":[{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Instituto Universitario de Oncología, Universidad de Oviedo","address":{"name":"Departamento de Bioquímica y Biología Molecular, Instituto Universitario de Oncología, Universidad de Oviedo, Oviedo, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Pablo M. Garcia-Roves","url":"http://orcid.org/0000-0002-8371-2067","affiliation":[{"name":"Universitat de Barcelona","address":{"name":"Universitat de Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Institut d’Investigació Biomèdica de Bellvitge (IDIBELL), L’Hospitalet de Llobregat","address":{"name":"Institut d’Investigació Biomèdica de Bellvitge (IDIBELL), L’Hospitalet de Llobregat, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Dolors Colomer","url":"http://orcid.org/0000-0001-7486-8484","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat de Barcelona","address":{"name":"Universitat de Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Holger Heyn","url":"http://orcid.org/0000-0002-3276-1889","affiliation":[{"name":"CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST)","address":{"name":"CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat Pompeu Fabra (UPF)","address":{"name":"Universitat Pompeu Fabra (UPF), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Francesco Maura","url":"http://orcid.org/0000-0002-5017-1620","affiliation":[{"name":"Myeloma Service, Sylvester Comprehensive Cancer Center, University of Miami","address":{"name":"Myeloma Service, Sylvester Comprehensive Cancer Center, University of Miami, Miami, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"José I. Martín-Subero","url":"http://orcid.org/0000-0001-8809-5195","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat de Barcelona","address":{"name":"Universitat de Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Institució Catalana de Recerca i Estudis Avançats (ICREA)","address":{"name":"Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Elías Campo","url":"http://orcid.org/0000-0001-9850-9793","affiliation":[{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS)","address":{"name":"Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC)","address":{"name":"Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Hospital Clínic of Barcelona","address":{"name":"Hospital Clínic of Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"},{"name":"Universitat de Barcelona","address":{"name":"Universitat de Barcelona, Barcelona, Spain","@type":"PostalAddress"},"@type":"Organization"}],"email":"ecampo@clinic.cat","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41591-022-01927-8">
    
    
    <meta name="journal_id" content="41591">
    <meta name="dc.title" content="Detection of early seeding of Richter transformation in chronic lymphocytic leukemia">
    <meta name="dc.source" content="Nature Medicine 2022 28:8">
    <meta name="dc.format" content="text/html">
    <meta name="dc.publisher" content="Nature Publishing Group">
    <meta name="dc.date" content="2022-08-11">
    <meta name="dc.type" content="OriginalPaper">
    <meta name="dc.language" content="En">
    <meta name="dc.copyright" content="2022 The Author(s)">
    <meta name="dc.rights" content="2022 The Author(s)">
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="dc.description" content="Richter transformation (RT) is a paradigmatic evolution of chronic lymphocytic leukemia (CLL) into a very aggressive large B cell lymphoma conferring a dismal prognosis. The mechanisms driving RT remain largely unknown. We characterized the whole genome, epigenome and transcriptome, combined with single-cell DNA/RNA-sequencing analyses and functional experiments, of 19 cases of CLL developing RT. Studying 54 longitudinal samples covering up to 19 years of disease course, we uncovered minute subclones carrying genomic, immunogenetic and transcriptomic features of RT cells already at CLL diagnosis, which were dormant for up to 19 years before transformation. We also identified new driver alterations, discovered a new mutational signature (SBS-RT), recognized an oxidative phosphorylation (OXPHOS)high–B cell receptor (BCR)low-signaling transcriptional axis in RT and showed that OXPHOS inhibition reduces the proliferation of RT cells. These findings demonstrate the early seeding of subclones driving advanced stages of cancer evolution and uncover potential therapeutic targets for RT. Single-cell genomic and transcriptomic analyses of longitudinal samples of patients with Richter syndrome reveal the presence and dynamics of clones driving transformation from chronic lymphocytic leukemia years before clinical manifestation">
    <meta name="prism.issn" content="1546-170X">
    <meta name="prism.publicationName" content="Nature Medicine">
    <meta name="prism.publicationDate" content="2022-08-11">
    <meta name="prism.volume" content="28">
    <meta name="prism.number" content="8">
    <meta name="prism.section" content="OriginalPaper">
    <meta name="prism.startingPage" content="1662">
    <meta name="prism.endingPage" content="1671">
    <meta name="prism.copyright" content="2022 The Author(s)">
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="prism.url" content="https://www.nature.com/articles/s41591-022-01927-8">
    <meta name="prism.doi" content="doi:10.1038/s41591-022-01927-8">
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41591-022-01927-8.pdf">
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41591-022-01927-8">
    <meta name="citation_journal_title" content="Nature Medicine">
    <meta name="citation_journal_abbrev" content="Nat Med">
    <meta name="citation_publisher" content="Nature Publishing Group">
    <meta name="citation_issn" content="1546-170X">
    <meta name="citation_title" content="Detection of early seeding of Richter transformation in chronic lymphocytic leukemia">
    <meta name="citation_volume" content="28">
    <meta name="citation_issue" content="8">
    <meta name="citation_publication_date" content="2022/08">
    <meta name="citation_online_date" content="2022/08/11">
    <meta name="citation_firstpage" content="1662">
    <meta name="citation_lastpage" content="1671">
    <meta name="citation_article_type" content="Article">
    <meta name="citation_fulltext_world_readable" content="">
    <meta name="citation_language" content="en">
    <meta name="dc.identifier" content="doi:10.1038/s41591-022-01927-8">
    <meta name="DOI" content="10.1038/s41591-022-01927-8">
    <meta name="size" content="378244">
    <meta name="citation_doi" content="10.1038/s41591-022-01927-8">
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41591-022-01927-8&amp;api_key=">
    <meta name="description" content="Richter transformation (RT) is a paradigmatic evolution of chronic lymphocytic leukemia (CLL) into a very aggressive large B cell lymphoma conferring a dismal prognosis. The mechanisms driving RT remain largely unknown. We characterized the whole genome, epigenome and transcriptome, combined with single-cell DNA/RNA-sequencing analyses and functional experiments, of 19 cases of CLL developing RT. Studying 54 longitudinal samples covering up to 19 years of disease course, we uncovered minute subclones carrying genomic, immunogenetic and transcriptomic features of RT cells already at CLL diagnosis, which were dormant for up to 19 years before transformation. We also identified new driver alterations, discovered a new mutational signature (SBS-RT), recognized an oxidative phosphorylation (OXPHOS)high–B cell receptor (BCR)low-signaling transcriptional axis in RT and showed that OXPHOS inhibition reduces the proliferation of RT cells. These findings demonstrate the early seeding of subclones driving advanced stages of cancer evolution and uncover potential therapeutic targets for RT. Single-cell genomic and transcriptomic analyses of longitudinal samples of patients with Richter syndrome reveal the presence and dynamics of clones driving transformation from chronic lymphocytic leukemia years before clinical manifestation">
    <meta name="dc.creator" content="Nadeu, Ferran">
    <meta name="dc.creator" content="Royo, Romina">
    <meta name="dc.creator" content="Massoni-Badosa, Ramon">
    <meta name="dc.creator" content="Playa-Albinyana, Heribert">
    <meta name="dc.creator" content="Garcia-Torre, Beatriz">
    <meta name="dc.creator" content="Duran-Ferrer, Martí">
    <meta name="dc.creator" content="Dawson, Kevin J.">
    <meta name="dc.creator" content="Kulis, Marta">
    <meta name="dc.creator" content="Diaz-Navarro, Ander">
    <meta name="dc.creator" content="Villamor, Neus">
    <meta name="dc.creator" content="Melero, Juan L.">
    <meta name="dc.creator" content="Chapaprieta, Vicente">
    <meta name="dc.creator" content="Dueso-Barroso, Ana">
    <meta name="dc.creator" content="Delgado, Julio">
    <meta name="dc.creator" content="Moia, Riccardo">
    <meta name="dc.creator" content="Ruiz-Gil, Sara">
    <meta name="dc.creator" content="Marchese, Domenica">
    <meta name="dc.creator" content="Giró, Ariadna">
    <meta name="dc.creator" content="Verdaguer-Dot, Núria">
    <meta name="dc.creator" content="Romo, Mónica">
    <meta name="dc.creator" content="Clot, Guillem">
    <meta name="dc.creator" content="Rozman, Maria">
    <meta name="dc.creator" content="Frigola, Gerard">
    <meta name="dc.creator" content="Rivas-Delgado, Alfredo">
    <meta name="dc.creator" content="Baumann, Tycho">
    <meta name="dc.creator" content="Alcoceba, Miguel">
    <meta name="dc.creator" content="González, Marcos">
    <meta name="dc.creator" content="Climent, Fina">
    <meta name="dc.creator" content="Abrisqueta, Pau">
    <meta name="dc.creator" content="Castellví, Josep">
    <meta name="dc.creator" content="Bosch, Francesc">
    <meta name="dc.creator" content="Aymerich, Marta">
    <meta name="dc.creator" content="Enjuanes, Anna">
    <meta name="dc.creator" content="Ruiz-Gaspà, Sílvia">
    <meta name="dc.creator" content="López-Guillermo, Armando">
    <meta name="dc.creator" content="Jares, Pedro">
    <meta name="dc.creator" content="Beà, Sílvia">
    <meta name="dc.creator" content="Capella-Gutierrez, Salvador">
    <meta name="dc.creator" content="Gelpí, Josep Ll.">
    <meta name="dc.creator" content="López-Bigas, Núria">
    <meta name="dc.creator" content="Torrents, David">
    <meta name="dc.creator" content="Campbell, Peter J.">
    <meta name="dc.creator" content="Gut, Ivo">
    <meta name="dc.creator" content="Rossi, Davide">
    <meta name="dc.creator" content="Gaidano, Gianluca">
    <meta name="dc.creator" content="Puente, Xose S.">
    <meta name="dc.creator" content="Garcia-Roves, Pablo M.">
    <meta name="dc.creator" content="Colomer, Dolors">
    <meta name="dc.creator" content="Heyn, Holger">
    <meta name="dc.creator" content="Maura, Francesco">
    <meta name="dc.creator" content="Martín-Subero, José I.">
    <meta name="dc.creator" content="Campo, Elías">
    <meta name="dc.subject" content="B-cell lymphoma">
    <meta name="dc.subject" content="Cancer epigenetics">
    <meta name="dc.subject" content="Cancer genomics">
    <meta name="dc.subject" content="Chronic lymphocytic leukaemia">
    <meta name="dc.subject" content="Tumour heterogeneity">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Mutation selection and the natural history of cancer; citation_author=J Cairns; citation_volume=255; citation_publication_date=1975; citation_pages=197-200; citation_doi=10.1038/255197a0; citation_id=CR1">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=The clonal evolution of tumor cell populations; citation_author=PC Nowell; citation_volume=194; citation_publication_date=1976; citation_pages=23-28; citation_doi=10.1126/science.959840; citation_id=CR2">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes; citation_author=SC Dentro; citation_volume=184; citation_publication_date=2021; citation_pages=2239-2254; citation_doi=10.1016/j.cell.2021.03.009; citation_id=CR3">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The evolutionary history of 2,658 cancers; citation_author=M Gerstung; citation_volume=578; citation_publication_date=2020; citation_pages=122-128; citation_doi=10.1038/s41586-019-1907-7; citation_id=CR4">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=Clonal evolution in leukemia; citation_author=AA Ferrando, C López-Otín; citation_volume=23; citation_publication_date=2017; citation_pages=1135-1145; citation_doi=10.1038/nm.4410; citation_id=CR5">
    <meta name="citation_reference" content="citation_journal_title=Hematology; citation_title=Richter transformation in the era of novel agents; citation_author=W Ding; citation_volume=2018; citation_publication_date=2018; citation_pages=256-263; citation_doi=10.1182/asheducation-2018.1.256; citation_id=CR6">
    <meta name="citation_reference" content="citation_journal_title=JAMA Oncol.; citation_title=Etiology of ibrutinib therapy discontinuation and outcomes in patients with chronic lymphocytic leukemia; citation_author=KJ Maddocks; citation_volume=1; citation_publication_date=2015; citation_pages=80; citation_doi=10.1001/jamaoncol.2014.218; citation_id=CR7">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Clonal evolution leading to ibrutinib resistance in chronic lymphocytic leukemia; citation_author=IE Ahn; citation_volume=129; citation_publication_date=2017; citation_pages=1469-1479; citation_doi=10.1182/blood-2016-06-719294; citation_id=CR8">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Outcomes of patients with chronic lymphocytic leukemia after discontinuing ibrutinib; citation_author=P Jain; citation_volume=125; citation_publication_date=2015; citation_pages=2062-2067; citation_doi=10.1182/blood-2014-09-603670; citation_id=CR9">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=The evolutionary landscape of chronic lymphocytic leukemia treated with ibrutinib targeted therapy; citation_author=DA Landau; citation_volume=8; citation_publication_date=2017; citation_doi=10.1038/s41467-017-02329-y; citation_id=CR10">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Pathol.; citation_title=Genetic imbalances in progressed B-cell chronic lymphocytic leukemia and transformed large-cell lymphoma (Richter’s syndrome); citation_author=S Beà; citation_volume=161; citation_publication_date=2002; citation_pages=957-968; citation_doi=10.1016/S0002-9440(10)64256-3; citation_id=CR11">
    <meta name="citation_reference" content="citation_journal_title=Hematol. Oncol.; citation_title=Genomic profiling of Richter’s syndrome: recurrent lesions and differences with de novo diffuse large B-cell lymphomas; citation_author=M Scandurra; citation_volume=28; citation_publication_date=2010; citation_pages=62-67; citation_doi=10.1002/hon.932; citation_id=CR12">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=The genetics of Richter syndrome reveals disease heterogeneity and predicts survival after transformation; citation_author=D Rossi; citation_volume=117; citation_publication_date=2011; citation_pages=3391-3401; citation_doi=10.1182/blood-2010-09-302174; citation_id=CR13">
    <meta name="citation_reference" content="citation_journal_title=J. Exp. Med.; citation_title=Genetic lesions associated with chronic lymphocytic leukemia transformation to Richter syndrome; citation_author=G Fabbri; citation_volume=210; citation_publication_date=2013; citation_pages=2273-2288; citation_doi=10.1084/jem.20131448; citation_id=CR14">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Two main genetic pathways lead to the transformation of chronic lymphocytic leukemia to Richter syndrome; citation_author=E Chigrinova; citation_volume=122; citation_publication_date=2013; citation_pages=2673-2682; citation_doi=10.1182/blood-2013-03-489518; citation_id=CR15">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Genomic and transcriptomic correlates of Richter transformation in chronic lymphocytic leukemia; citation_author=J Klintman; citation_volume=137; citation_publication_date=2021; citation_pages=2800-2816; citation_doi=10.1182/blood.2020005650; citation_id=CR16">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=B-cell receptor signaling and genetic lesions in TP53 and CDKN2A/CDKN2B cooperate in Richter transformation; citation_author=S Chakraborty; citation_volume=138; citation_publication_date=2021; citation_pages=1053-1066; citation_doi=10.1182/blood.2020008276; citation_id=CR17">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Clinicopathological features and outcomes of progression of CLL on the BCL2 inhibitor venetoclax; citation_author=MA Anderson; citation_volume=129; citation_publication_date=2017; citation_pages=3362-3370; citation_doi=10.1182/blood-2017-01-763003; citation_id=CR18">
    <meta name="citation_reference" content="citation_journal_title=Blood Adv.; citation_title=Near-tetraploidy is associated with Richter transformation in chronic lymphocytic leukemia patients receiving ibrutinib; citation_author=CR Miller; citation_volume=1; citation_publication_date=2017; citation_pages=1584-1588; citation_doi=10.1182/bloodadvances.2017007302; citation_id=CR19">
    <meta name="citation_reference" content="citation_journal_title=Blood Adv.; citation_title=Clonal evolution underlying leukemia progression and Richter transformation in patients with ibrutinib-relapsed CLL; citation_author=S Kadri; citation_volume=1; citation_publication_date=2017; citation_pages=715-727; citation_doi=10.1182/bloodadvances.2016003632; citation_id=CR20">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Clonal dynamics towards the development of venetoclax resistance in chronic lymphocytic leukemia; citation_author=CD Herling; citation_volume=9; citation_publication_date=2018; citation_doi=10.1038/s41467-018-03170-7; citation_id=CR21">
    <meta name="citation_reference" content="citation_journal_title=Leukemia; citation_title=NOTCH1 mutations identify a genetic subgroup of chronic lymphocytic leukemia patients with high risk of transformation and poor outcome; citation_author=N Villamor; citation_volume=27; citation_publication_date=2013; citation_pages=1100-1106; citation_doi=10.1038/leu.2012.357; citation_id=CR22">
    <meta name="citation_reference" content="citation_journal_title=Leuk. Lymphoma; citation_title=MGA, a suppressor of MYC, is recurrently inactivated in high risk chronic lymphocytic leukemia; citation_author=L Paoli; citation_volume=54; citation_publication_date=2013; citation_pages=1087-1090; citation_doi=10.3109/10428194.2012.723706; citation_id=CR23">
    <meta name="citation_reference" content="citation_journal_title=Br. J. Haematol.; citation_title=Different impact of NOTCH1 and SF3B1 mutations on the risk of chronic lymphocytic leukemia transformation to Richter syndrome; citation_author=D Rossi; citation_volume=158; citation_publication_date=2012; citation_pages=426-429; citation_doi=10.1111/j.1365-2141.2012.09155.x; citation_id=CR24">
    <meta name="citation_reference" content="citation_journal_title=Leuk. Lymphoma; citation_title=Descriptive analysis of genetic aberrations and cell of origin in Richter transformation; citation_author=A Chitalia; citation_volume=60; citation_publication_date=2019; citation_pages=971-979; citation_doi=10.1080/10428194.2018.1516878; citation_id=CR25">
    <meta name="citation_reference" content="citation_journal_title=Hematol. Oncol.; citation_title=IRF4 L116R mutation promotes proliferation of chronic lymphocytic leukemia B cells inducing MYC; citation_author=S Benatti; citation_volume=39; citation_publication_date=2021; citation_pages=707-711; citation_doi=10.1002/hon.2915; citation_id=CR26">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Distinct homotypic B-cell receptor interactions shape the outcome of chronic lymphocytic leukaemia; citation_author=C Minici; citation_volume=8; citation_publication_date=2017; citation_doi=10.1038/ncomms15746; citation_id=CR27">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Non-coding recurrent mutations in chronic lymphocytic leukaemia; citation_author=XS Puente; citation_volume=526; citation_publication_date=2015; citation_pages=519-524; citation_doi=10.1038/nature14666; citation_id=CR28">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution; citation_author=S Kasar; citation_volume=6; citation_publication_date=2015; citation_doi=10.1038/ncomms9866; citation_id=CR29">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=A practical guide for mutational signature analysis in hematological malignancies; citation_author=F Maura; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-11037-8; citation_id=CR30">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma; citation_author=SE Arthur; citation_volume=9; citation_publication_date=2018; citation_doi=10.1038/s41467-018-06354-3; citation_id=CR31">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The repertoire of mutational signatures in human cancer; citation_author=LB Alexandrov; citation_volume=578; citation_publication_date=2020; citation_pages=94-101; citation_doi=10.1038/s41586-020-1943-3; citation_id=CR32">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=A compendium of mutational signatures of environmental agents; citation_author=JE Kucab; citation_volume=177; citation_publication_date=2019; citation_pages=821-836; citation_doi=10.1016/j.cell.2019.03.001; citation_id=CR33">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Timing the initiation of multiple myeloma; citation_author=EH Rustad; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-15740-9; citation_id=CR34">
    <meta name="citation_reference" content="citation_journal_title=Cell Stem Cell; citation_title=Antiviral treatment causes a unique mutational signature in cancers of transplantation recipients; citation_author=JK Kanter; citation_volume=28; citation_publication_date=2021; citation_pages=1726-1739; citation_doi=10.1016/j.stem.2021.07.012; citation_id=CR35">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=The mutational footprints of cancer therapies; citation_author=O Pich; citation_volume=51; citation_publication_date=2019; citation_pages=1732-1740; citation_doi=10.1038/s41588-019-0525-5; citation_id=CR36">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Epigenetic evolution and lineage histories of chronic lymphocytic leukaemia; citation_author=F Gaiti; citation_volume=569; citation_publication_date=2019; citation_pages=576-580; citation_doi=10.1038/s41586-019-1198-z; citation_id=CR37">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Higher-order immunoglobulin repertoire restrictions in CLL: the illustrative case of stereotyped subsets 2 and 169; citation_author=K Gemenetzi; citation_volume=137; citation_publication_date=2021; citation_pages=1895-1904; citation_doi=10.1182/blood.2020005216; citation_id=CR38">
    <meta name="citation_reference" content="citation_journal_title=Front. Oncol.; citation_title=Post-transformation IGHV-IGHD-IGHJ mutations in chronic lymphocytic leukemia B cells: implications for mutational mechanisms and impact on clinical course; citation_author=D Bagnara; citation_volume=11; citation_publication_date=2021; citation_pages=1769; citation_doi=10.3389/fonc.2021.640731; citation_id=CR39">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=In vivo intraclonal and interclonal kinetic heterogeneity in B-cell chronic lymphocytic leukemia; citation_author=C Calissano; citation_volume=114; citation_publication_date=2009; citation_pages=4832-4842; citation_doi=10.1182/blood-2009-05-219634; citation_id=CR40">
    <meta name="citation_reference" content="citation_journal_title=Mol. Med.; citation_title=Intraclonal complexity in chronic lymphocytic leukemia: fractions enriched in recently born/divided and older/quiescent cells; citation_author=C Calissano; citation_volume=17; citation_publication_date=2011; citation_pages=1374-1382; citation_doi=10.2119/molmed.2011.00360; citation_id=CR41">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=MicroRNA-155 influences B-cell receptor signaling and associates with aggressive disease in chronic lymphocytic leukemia; citation_author=B Cui; citation_volume=124; citation_publication_date=2014; citation_pages=546-554; citation_doi=10.1182/blood-2014-03-559690; citation_id=CR42">
    <meta name="citation_reference" content="citation_journal_title=Cancer Discov.; citation_title=Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history; citation_author=L Penter; citation_volume=11; citation_publication_date=2021; citation_pages=3048-3063; citation_doi=10.1158/2159-8290.CD-21-0276; citation_id=CR43">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=The reference epigenome and regulatory chromatin landscape of chronic lymphocytic leukemia; citation_author=R Beekman; citation_volume=24; citation_publication_date=2018; citation_pages=868-880; citation_doi=10.1038/s41591-018-0028-4; citation_id=CR44">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Arginine is an epigenetic regulator targeting TEAD4 to modulate OXPHOS in prostate cancer cells; citation_author=C-L Chen; citation_volume=12; citation_publication_date=2021; citation_doi=10.1038/s41467-021-22652-9; citation_id=CR45">
    <meta name="citation_reference" content="citation_journal_title=Cancer Discov.; citation_title=Cross-cohort analysis identifies a TEAD4–MYCN positive feedback loop as the core regulatory element of high-risk neuroblastoma; citation_author=P Rajbhandari; citation_volume=8; citation_publication_date=2018; citation_pages=582-599; citation_doi=10.1158/2159-8290.CD-16-0861; citation_id=CR46">
    <meta name="citation_reference" content="citation_journal_title=Leukemia; citation_title=IRF4 modulates the response to BCR activation in chronic lymphocytic leukemia regulating IKAROS and SYK; citation_author=R Maffei; citation_volume=35; citation_publication_date=2021; citation_pages=1330-1343; citation_doi=10.1038/s41375-021-01178-5; citation_id=CR47">
    <meta name="citation_reference" content="citation_journal_title=J. Immunol.; citation_title=An IRF4–MYC–mTORC1 integrated pathway controls cell growth and the proliferative capacity of activated B cells during B cell differentiation in vivo; citation_author=DG Patterson; citation_volume=207; citation_publication_date=2021; citation_pages=1798-1811; citation_doi=10.4049/jimmunol.2100440; citation_id=CR48">
    <meta name="citation_reference" content="citation_journal_title=Nat. Cancer; citation_title=The proliferative history shapes the DNA methylome of B-cell tumors and predicts clinical outcome; citation_author=M Duran-Ferrer; citation_volume=1; citation_publication_date=2020; citation_pages=1066-1081; citation_doi=10.1038/s43018-020-00131-2; citation_id=CR49">
    <meta name="citation_reference" content="citation_journal_title=Leukemia; citation_title=Wnt5a enhances proliferation of chronic lymphocytic leukemia and ERK1/2 phosphorylation via a ROR1/DOCK2-dependent mechanism; citation_author=MK Hasan, EM Ghia, LZ Rassenti, GF Widhopf, TJ Kipps; citation_volume=35; citation_publication_date=2021; citation_pages=1621-1630; citation_doi=10.1038/s41375-020-01055-7; citation_id=CR50">
    <meta name="citation_reference" content="citation_journal_title=Semin. Cancer Biol.; citation_title=Toll-like receptors signaling: a complex network for NF-κB activation in B-cell lymphoid malignancies; citation_author=S Ntoufa, MG Vilia, K Stamatopoulos, P Ghia, M Muzio; citation_volume=39; citation_publication_date=2016; citation_pages=15-25; citation_doi=10.1016/j.semcancer.2016.07.001; citation_id=CR51">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Molecular profiling of diffuse large B-cell lymphoma identifies robust subtypes including one characterized by host inflammatory response; citation_author=S Monti; citation_volume=105; citation_publication_date=2005; citation_pages=1851-1861; citation_doi=10.1182/blood-2004-07-2947; citation_id=CR52">
    <meta name="citation_reference" content="citation_journal_title=Cancer Cell; citation_title=Metabolic signatures uncover distinct targets in molecular subsets of diffuse large B cell lymphoma; citation_author=P Caro; citation_volume=22; citation_publication_date=2012; citation_pages=547-560; citation_doi=10.1016/j.ccr.2012.08.014; citation_id=CR53">
    <meta name="citation_reference" content="citation_journal_title=Cell Death Differ.; citation_title=Differential contribution of the mitochondrial translation pathway to the survival of diffuse large B-cell lymphoma subsets; citation_author=E Norberg; citation_volume=24; citation_publication_date=2017; citation_pages=251-262; citation_doi=10.1038/cdd.2016.116; citation_id=CR54">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=An inhibitor of oxidative phosphorylation exploits cancer vulnerability; citation_author=JR Molina; citation_volume=24; citation_publication_date=2018; citation_pages=1036-1046; citation_doi=10.1038/s41591-018-0052-4; citation_id=CR55">
    <meta name="citation_reference" content="citation_journal_title=Oncotarget; citation_title=Biological and metabolic effects of IACS-010759, an OxPhos inhibitor, on chronic lymphocytic leukemia cells; citation_author=HV Vangapandu; citation_volume=9; citation_publication_date=2018; citation_pages=24980-24991; citation_doi=10.18632/oncotarget.25166; citation_id=CR56">
    <meta name="citation_reference" content="citation_journal_title=Sci. Transl. Med.; citation_title=Metabolic reprogramming toward oxidative phosphorylation identifies a therapeutic target for mantle cell lymphoma; citation_author=L Zhang; citation_volume=11; citation_publication_date=2019; citation_pages=eaau1167; citation_doi=10.1126/scitranslmed.aau1167; citation_id=CR57">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The B-cell receptor controls fitness of MYC-driven lymphoma cells via GSK3β inhibition; citation_author=G Varano; citation_volume=546; citation_publication_date=2017; citation_pages=302-306; citation_doi=10.1038/nature22353; citation_id=CR58">
    <meta name="citation_reference" content="citation_journal_title=Cancer Res.; citation_title=TLR signaling is activated in lymph node–resident CLL cells and is only partially inhibited by ibrutinib; citation_author=EL Dadashian; citation_volume=79; citation_publication_date=2019; citation_pages=360-371; citation_doi=10.1158/0008-5472.CAN-18-0781; citation_id=CR59">
    <meta name="citation_reference" content="citation_journal_title=Br. J. Haematol.; citation_title=Plasmablastic Richter transformation as a resistance mechanism for chronic lymphocytic leukaemia treated with BCR signalling inhibitors; citation_author=K-L Chan; citation_volume=177; citation_publication_date=2017; citation_pages=324-328; citation_doi=10.1111/bjh.14062; citation_id=CR60">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform; citation_author=H Li, R Durbin; citation_volume=25; citation_publication_date=2009; citation_pages=1754-1760; citation_doi=10.1093/bioinformatics/btp324; citation_id=CR61">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=IgCaller for reconstructing immunoglobulin gene rearrangements and oncogenic translocations from whole-genome sequencing in lymphoid neoplasms; citation_author=F Nadeu; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-17095-7; citation_id=CR62">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Integrative genomics viewer; citation_author=JT Robinson; citation_volume=29; citation_publication_date=2011; citation_pages=24-26; citation_doi=10.1038/nbt.1754; citation_id=CR63">
    <meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinforma.; citation_title=cgpCaVEManWrapper: simple execution of CaVEMan in order to detect somatic single nucleotide variants in NGS data; citation_author=D Jones; citation_volume=56; citation_publication_date=2016; citation_pages=15.10.1-15.10.18; citation_doi=10.1002/cpbi.20; citation_id=CR64">
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data; citation_author=A McKenna; citation_volume=20; citation_publication_date=2010; citation_pages=1297-1303; citation_doi=10.1101/gr.107524.110; citation_id=CR65">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=MuSE: accounting for tumor heterogeneity using a sample-specific error model improves sensitivity and specificity in mutation calling from sequencing data; citation_author=Y Fan; citation_volume=17; citation_publication_date=2016; citation_doi=10.1186/s13059-016-1029-6; citation_id=CR66">
    <meta name="citation_reference" content="Danecek, P. et al. Twelve years of SAMtools and BCFtools. Gigascience 
                  https://doi.org/10.1093/gigascience/giab008
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Comprehensive characterization of complex structural variations in cancer by directly comparing genome sequence reads; citation_author=V Moncunill; citation_volume=32; citation_publication_date=2014; citation_pages=1106-1112; citation_doi=10.1038/nbt.3027; citation_id=CR68">
    <meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinforma.; citation_title=cgpPindel: identifying somatically acquired insertion and deletion events from paired end sequencing; citation_author=KM Raine; citation_volume=52; citation_publication_date=2015; citation_pages=15.7.1-12; citation_doi=10.1002/0471250953.bi1507s52; citation_id=CR69">
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=SvABA: genome-wide detection of structural variants and indels by local assembly; citation_author=JA Wala; citation_volume=28; citation_publication_date=2018; citation_pages=581-591; citation_doi=10.1101/gr.221028.117; citation_id=CR70">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Integrating mapping-, assembly- and haplotype-based approaches for calling variants in clinical sequencing applications; citation_author=A Rimmer; citation_volume=46; citation_publication_date=2014; citation_pages=912-918; citation_doi=10.1038/ng.3036; citation_id=CR71">
    <meta name="citation_reference" content="citation_journal_title=Fly. (Austin).; citation_title=A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3; citation_author=P Cingolani; citation_volume=6; citation_publication_date=2012; citation_pages=80-92; citation_doi=10.4161/fly.19695; citation_id=CR72">
    <meta name="citation_reference" content="citation_journal_title=Leukemia; citation_title=Clinical impact of the subclonal architecture and mutational complexity in chronic lymphocytic leukemia; citation_author=F Nadeu; citation_volume=32; citation_publication_date=2018; citation_pages=645-653; citation_doi=10.1038/leu.2017.291; citation_id=CR73">
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing; citation_author=DC Koboldt; citation_volume=22; citation_publication_date=2012; citation_pages=568-576; citation_doi=10.1101/gr.129684.111; citation_id=CR74">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=VarDict: a novel and versatile variant caller for next-generation sequencing in cancer research; citation_author=Z Lai; citation_volume=44; citation_publication_date=2016; citation_pages=e108; citation_doi=10.1093/nar/gkw227; citation_id=CR75">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets; citation_author=A Wilm; citation_volume=40; citation_publication_date=2012; citation_pages=11189-11201; citation_doi=10.1093/nar/gks918; citation_id=CR76">
    <meta name="citation_reference" content="citation_journal_title=Oncotarget; citation_title=OutLyzer: software for extracting low-allele-frequency tumor mutations from sequencing background noise in clinical practice; citation_author=E Muller; citation_volume=7; citation_publication_date=2016; citation_pages=79485-79493; citation_doi=10.18632/oncotarget.13103; citation_id=CR77">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=The life history of 21 breast cancers; citation_author=S Nik-Zainal; citation_volume=149; citation_publication_date=2012; citation_pages=994-1007; citation_doi=10.1016/j.cell.2012.04.023; citation_id=CR78">
    <meta name="citation_reference" content="citation_journal_title=Curr. Protoc. Bioinforma.; citation_title=ascatNgs: identifying somatically acquired copy-number alterations from whole-genome sequencing data; citation_author=KM Raine; citation_volume=56; citation_publication_date=2016; citation_pages=15.9.1-15.9.17; citation_doi=10.1002/cpbi.17; citation_id=CR79">
    <meta name="citation_reference" content="citation_journal_title=PLoS Comput. Biol.; citation_title=CNVkit: genome-wide copy number detection and visualization from targeted DNA sequencing; citation_author=E Talevich, AH Shain, T Botton, BC Bastian; citation_volume=12; citation_publication_date=2016; citation_pages=e1004873; citation_doi=10.1371/journal.pcbi.1004873; citation_id=CR80">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Landscape of somatic mutations in 560 breast cancer whole-genome sequences; citation_author=S Nik-Zainal; citation_volume=534; citation_publication_date=2016; citation_pages=47-54; citation_doi=10.1038/nature17676; citation_id=CR81">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=DELLY: structural variant discovery by integrated paired-end and split-read analysis; citation_author=T Rausch; citation_volume=28; citation_publication_date=2012; citation_pages=i333-i339; citation_doi=10.1093/bioinformatics/bts378; citation_id=CR82">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Massive genomic rearrangement acquired in a single catastrophic event during cancer development; citation_author=PJ Stephens; citation_volume=144; citation_publication_date=2011; citation_pages=27-40; citation_doi=10.1016/j.cell.2010.11.055; citation_id=CR83">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Criteria for inference of chromothripsis in cancer genomes; citation_author=JO Korbel, PJ Campbell; citation_volume=152; citation_publication_date=2013; citation_pages=1226-1236; citation_doi=10.1016/j.cell.2013.02.023; citation_id=CR84">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Punctuated evolution of prostate cancer genomes; citation_author=SC Baca; citation_volume=153; citation_publication_date=2013; citation_pages=666-677; citation_doi=10.1016/j.cell.2013.03.021; citation_id=CR85">
    <meta name="citation_reference" content="citation_journal_title=Cancer Cell; citation_title=Chromoplexy: a new category of complex rearrangements in the cancer genome; citation_author=MM Shen; citation_volume=23; citation_publication_date=2013; citation_pages=567-569; citation_doi=10.1016/j.ccr.2013.04.025; citation_id=CR86">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Patterns of somatic structural variation in human cancer genomes; citation_author=Y Li; citation_volume=578; citation_publication_date=2020; citation_pages=112-121; citation_doi=10.1038/s41586-019-1913-9; citation_id=CR87">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Genomic landscape and chronological reconstruction of driver events in multiple myeloma; citation_author=F Maura; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-11680-1; citation_id=CR88">
    <meta name="citation_reference" content="citation_journal_title=Cold Spring Harb. Perspect. Med.; citation_title=Principles of reconstructing the subclonal architecture of cancers; citation_author=SC Dentro, DC Wedge, P Loo; citation_volume=7; citation_publication_date=2017; citation_pages=a026625; citation_doi=10.1101/cshperspect.a026625; citation_id=CR89">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors; citation_author=J Kim; citation_volume=48; citation_publication_date=2016; citation_pages=600-606; citation_doi=10.1038/ng.3557; citation_id=CR90">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The landscape of somatic mutation in normal colorectal epithelial cells; citation_author=H Lee-Six; citation_volume=574; citation_publication_date=2019; citation_pages=532-537; citation_doi=10.1038/s41586-019-1672-7; citation_id=CR91">
    <meta name="citation_reference" content="citation_journal_title=Nat. Cancer; citation_title=Chemotherapy and mismatch repair deficiency cooperate to fuel TP53 mutagenesis and ALL relapse; citation_author=F Yang; citation_volume=2; citation_publication_date=2021; citation_pages=819-834; citation_doi=10.1038/s43018-021-00230-8; citation_id=CR92">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.; citation_title=Differential relationship of DNA replication timing to different forms of human mutation and variation; citation_author=A Koren; citation_volume=91; citation_publication_date=2012; citation_pages=1033-1040; citation_doi=10.1016/j.ajhg.2012.10.018; citation_id=CR93">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Mutational strand asymmetries in cancer genomes reveal mechanisms of DNA damage and repair; citation_author=NJ Haradhvala; citation_volume=164; citation_publication_date=2016; citation_pages=538-549; citation_doi=10.1016/j.cell.2015.12.050; citation_id=CR94">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data; citation_author=AM Bolger, M Lohse, B Usadel; citation_volume=30; citation_publication_date=2014; citation_pages=2114-2120; citation_doi=10.1093/bioinformatics/btu170; citation_id=CR95">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Minfi: a flexible and comprehensive Bioconductor package for the analysis of Infinium DNA methylation microarrays; citation_author=MJ Aryee; citation_volume=30; citation_publication_date=2014; citation_pages=1363-1369; citation_doi=10.1093/bioinformatics/btu049; citation_id=CR96">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=Model-based analysis of ChIP-Seq (MACS); citation_author=Y Zhang; citation_volume=9; citation_publication_date=2008; citation_doi=10.1186/gb-2008-9-9-r137; citation_id=CR97">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2; citation_author=MI Love, W Huber, S Anders; citation_volume=15; citation_publication_date=2014; citation_doi=10.1186/s13059-014-0550-8; citation_id=CR98">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data; citation_author=E Kopylova, L Noé, H Touzet; citation_volume=28; citation_publication_date=2012; citation_pages=3211-3217; citation_doi=10.1093/bioinformatics/bts611; citation_id=CR99">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Near-optimal probabilistic RNA-seq quantification; citation_author=NL Bray, H Pimentel, P Melsted, L Pachter; citation_volume=34; citation_publication_date=2016; citation_pages=525-527; citation_doi=10.1038/nbt.3519; citation_id=CR100">
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=Single-cell sequencing data reveal widespread recurrence and loss of mutational hits in the life histories of tumors; citation_author=J Kuipers, K Jahn, BJ Raphael, N Beerenwinkel; citation_volume=27; citation_publication_date=2017; citation_pages=1885-1894; citation_doi=10.1101/gr.220707.117; citation_id=CR101">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Clonal evolution of acute myeloid leukemia revealed by high-throughput single-cell genomics; citation_author=K Morita; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-19119-8; citation_id=CR102">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Smart-seq2 for sensitive full-length transcriptome profiling in single cells; citation_author=S Picelli; citation_volume=10; citation_publication_date=2013; citation_pages=1096-1098; citation_doi=10.1038/nmeth.2639; citation_id=CR103">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics; citation_author=M Stoeckius; citation_volume=19; citation_publication_date=2018; citation_doi=10.1186/s13059-018-1603-1; citation_id=CR104">
    <meta name="citation_reference" content="Parekh, S., Ziegenhain, C., Vieth, B., Enard, W. &amp; Hellmann, I. zUMIs - A fast and flexible pipeline to process RNA sequencing data with UMIs. Gigascience 
                  https://doi.org/10.1093/gigascience/giy059
                  
                 (2018).">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Integrated analysis of multimodal single-cell data; citation_author=Y Hao; citation_volume=184; citation_publication_date=2021; citation_pages=3573-3587; citation_doi=10.1016/j.cell.2021.04.048; citation_id=CR106">
    <meta name="citation_reference" content="citation_journal_title=Cell Syst.; citation_title=Scrublet: computational identification of cell doublets in single-cell transcriptomic data; citation_author=SL Wolock, R Lopez, AM Klein; citation_volume=8; citation_publication_date=2019; citation_pages=281-291; citation_doi=10.1016/j.cels.2018.11.005; citation_id=CR107">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Fast, sensitive and accurate integration of single-cell data with Harmony; citation_author=I Korsunsky; citation_volume=16; citation_publication_date=2019; citation_pages=1289-1296; citation_doi=10.1038/s41592-019-0619-0; citation_id=CR108">
    <meta name="citation_reference" content="citation_journal_title=J. Bioenerg. Biomembr.; citation_title=Control of mitochondrial and cellular respiration by oxygen; citation_author=E Gnaiger, R Steinlechner-Maran, G Méndez, T Eberl, R Margreiter; citation_volume=27; citation_publication_date=1995; citation_pages=583-596; citation_doi=10.1007/BF02111656; citation_id=CR109">
    <meta name="citation_reference" content="citation_journal_title=J. Immunol.; citation_title=TLR-9 and IL-15 synergy promotes the in vitro clonal expansion of chronic lymphocytic leukemia B cells; citation_author=PKA Mongini; citation_volume=195; citation_publication_date=2015; citation_pages=901-923; citation_doi=10.4049/jimmunol.1403189; citation_id=CR110">
    <meta name="citation_author" content="Nadeu, Ferran">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author" content="Royo, Romina">
    <meta name="citation_author_institution" content="Barcelona Supercomputing Center (BSC), Barcelona, Spain">
    <meta name="citation_author" content="Massoni-Badosa, Ramon">
    <meta name="citation_author_institution" content="CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain">
    <meta name="citation_author" content="Playa-Albinyana, Heribert">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author" content="Garcia-Torre, Beatriz">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author" content="Duran-Ferrer, Martí">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author" content="Dawson, Kevin J.">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Hinxton, UK">
    <meta name="citation_author" content="Kulis, Marta">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author" content="Diaz-Navarro, Ander">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Departamento de Bioquímica y Biología Molecular, Instituto Universitario de Oncología, Universidad de Oviedo, Oviedo, Spain">
    <meta name="citation_author" content="Villamor, Neus">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="Melero, Juan L.">
    <meta name="citation_author_institution" content="Omniscope, Barcelona, Spain">
    <meta name="citation_author" content="Chapaprieta, Vicente">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author" content="Dueso-Barroso, Ana">
    <meta name="citation_author_institution" content="Barcelona Supercomputing Center (BSC), Barcelona, Spain">
    <meta name="citation_author" content="Delgado, Julio">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author_institution" content="Universitat de Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="Moia, Riccardo">
    <meta name="citation_author_institution" content="Division of Hematology, Department of Translational Medicine, University of Eastern Piedmont, Novara, Italy">
    <meta name="citation_author" content="Ruiz-Gil, Sara">
    <meta name="citation_author_institution" content="CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain">
    <meta name="citation_author" content="Marchese, Domenica">
    <meta name="citation_author_institution" content="CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain">
    <meta name="citation_author" content="Giró, Ariadna">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author" content="Verdaguer-Dot, Núria">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author" content="Romo, Mónica">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author" content="Clot, Guillem">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author" content="Rozman, Maria">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="Frigola, Gerard">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="Rivas-Delgado, Alfredo">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="Baumann, Tycho">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author_institution" content="Hospital Universitario 12 de Octubre, Madrid, Spain">
    <meta name="citation_author" content="Alcoceba, Miguel">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Biología Molecular e Histocompatibilidad, IBSAL-Hospital Universitario, Centro de Investigación del Cáncer-IBMCC (USAL-CSIC), Salamanca, Spain">
    <meta name="citation_author" content="González, Marcos">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Biología Molecular e Histocompatibilidad, IBSAL-Hospital Universitario, Centro de Investigación del Cáncer-IBMCC (USAL-CSIC), Salamanca, Spain">
    <meta name="citation_author" content="Climent, Fina">
    <meta name="citation_author_institution" content="Hospital Universitari de Bellvitge-Institut d’Investigació Biomédica de Bellvitge (IDIBELL), L’Hospitalet de Llobregat, Barcelona, Spain">
    <meta name="citation_author" content="Abrisqueta, Pau">
    <meta name="citation_author_institution" content="Department of Hematology, Vall d’Hebron Institute of Oncology, Vall d’Hebron University Hospital, Barcelona, Spain">
    <meta name="citation_author" content="Castellví, Josep">
    <meta name="citation_author_institution" content="Department of Hematology, Vall d’Hebron Institute of Oncology, Vall d’Hebron University Hospital, Barcelona, Spain">
    <meta name="citation_author" content="Bosch, Francesc">
    <meta name="citation_author_institution" content="Department of Hematology, Vall d’Hebron Institute of Oncology, Vall d’Hebron University Hospital, Barcelona, Spain">
    <meta name="citation_author" content="Aymerich, Marta">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="Enjuanes, Anna">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author" content="Ruiz-Gaspà, Sílvia">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author" content="López-Guillermo, Armando">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author_institution" content="Universitat de Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="Jares, Pedro">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author_institution" content="Universitat de Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="Beà, Sílvia">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author_institution" content="Universitat de Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="Capella-Gutierrez, Salvador">
    <meta name="citation_author_institution" content="Barcelona Supercomputing Center (BSC), Barcelona, Spain">
    <meta name="citation_author" content="Gelpí, Josep Ll.">
    <meta name="citation_author_institution" content="Barcelona Supercomputing Center (BSC), Barcelona, Spain">
    <meta name="citation_author_institution" content="Universitat de Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="López-Bigas, Núria">
    <meta name="citation_author_institution" content="Institute for Research in Biomedicine (IRB Barcelona), The Barcelona Institute of Science and Technology, Barcelona, Spain">
    <meta name="citation_author_institution" content="Universitat Pompeu Fabra (UPF), Barcelona, Spain">
    <meta name="citation_author_institution" content="Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain">
    <meta name="citation_author" content="Torrents, David">
    <meta name="citation_author_institution" content="Barcelona Supercomputing Center (BSC), Barcelona, Spain">
    <meta name="citation_author_institution" content="Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain">
    <meta name="citation_author" content="Campbell, Peter J.">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Hinxton, UK">
    <meta name="citation_author" content="Gut, Ivo">
    <meta name="citation_author_institution" content="CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain">
    <meta name="citation_author_institution" content="Universitat Pompeu Fabra (UPF), Barcelona, Spain">
    <meta name="citation_author" content="Rossi, Davide">
    <meta name="citation_author_institution" content="Oncology Institute of Southern Switzerland, Bellinzona, Switzerland">
    <meta name="citation_author" content="Gaidano, Gianluca">
    <meta name="citation_author_institution" content="Division of Hematology, Department of Translational Medicine, University of Eastern Piedmont, Novara, Italy">
    <meta name="citation_author" content="Puente, Xose S.">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Departamento de Bioquímica y Biología Molecular, Instituto Universitario de Oncología, Universidad de Oviedo, Oviedo, Spain">
    <meta name="citation_author" content="Garcia-Roves, Pablo M.">
    <meta name="citation_author_institution" content="Universitat de Barcelona, Barcelona, Spain">
    <meta name="citation_author_institution" content="Institut d’Investigació Biomèdica de Bellvitge (IDIBELL), L’Hospitalet de Llobregat, Barcelona, Spain">
    <meta name="citation_author" content="Colomer, Dolors">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author_institution" content="Universitat de Barcelona, Barcelona, Spain">
    <meta name="citation_author" content="Heyn, Holger">
    <meta name="citation_author_institution" content="CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain">
    <meta name="citation_author_institution" content="Universitat Pompeu Fabra (UPF), Barcelona, Spain">
    <meta name="citation_author" content="Maura, Francesco">
    <meta name="citation_author_institution" content="Myeloma Service, Sylvester Comprehensive Cancer Center, University of Miami, Miami, USA">
    <meta name="citation_author" content="Martín-Subero, José I.">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Universitat de Barcelona, Barcelona, Spain">
    <meta name="citation_author_institution" content="Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain">
    <meta name="citation_author" content="Campo, Elías">
    <meta name="citation_author_institution" content="Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain">
    <meta name="citation_author_institution" content="Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain">
    <meta name="citation_author_institution" content="Hospital Clínic of Barcelona, Barcelona, Spain">
    <meta name="citation_author_institution" content="Universitat de Barcelona, Barcelona, Spain">
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">
    <meta name="twitter:site" content="@naturemedicine">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image:alt" content="Content cover image">
    <meta name="twitter:title" content="Detection of early seeding of Richter transformation in chronic lymphocytic leukemia">
    <meta name="twitter:description" content="Nature Medicine - Single-cell genomic and transcriptomic analyses of longitudinal samples of patients with Richter syndrome reveal the presence and dynamics of clones driving transformation from...">
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig1_HTML.png">
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41591-022-01927-8">
    <meta property="og:type" content="article">
    <meta property="og:site_name" content="Nature">
    <meta property="og:title" content="Detection of early seeding of Richter transformation in chronic lymphocytic leukemia - Nature Medicine">
    <meta property="og:description" content="Single-cell genomic and transcriptomic analyses of longitudinal samples of patients with Richter syndrome reveal the presence and dynamics of clones driving transformation from chronic lymphocytic leukemia years before clinical manifestation">
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig1_HTML.png">
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
<style type="text/css">.cc-banner{background-color:#01324b;border:none!important;bottom:0;box-sizing:border-box;color:#fff!important;left:0;line-height:1.3;margin:auto 0 0;max-width:100%;outline:0;overflow:visible;padding:0;position:fixed;right:0;width:100%;z-index:99999}.cc-banner::backdrop{background-color:#0000004d}.cc-banner *{color:inherit!important}.cc-banner:focus{box-shadow:none;outline:0}.cc-banner a{color:#fff!important;text-decoration:underline}.cc-banner a:active,.cc-banner a:focus,.cc-banner a:hover{color:inherit;text-decoration:none}.cc-banner a:focus{outline:3px solid #08c!important}.cc-banner h2,.cc-banner h3,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-banner .cc-h2,.cc-banner h2{font-size:18px}.cc-banner .cc-h3,.cc-banner h3{font-size:16px}.cc-banner .cc-h4,.cc-banner .cc-h5,.cc-banner .cc-h6,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-size:14px}.cc-banner .cc-button{font-size:16px}.cc-banner__content{background-color:#01324b;display:flex;flex-direction:column;margin:0 auto;max-height:90vh;max-width:100%;padding:16px;position:relative}.cc-banner__content:focus{outline:0}@media (min-width:680px){.cc-banner__content{padding:12px}}@media (min-width:980px){.cc-banner__content{max-height:60vh;padding-bottom:20px;padding-top:20px}}@media (min-width:1320px){.cc-banner__content{padding-bottom:40px;padding-top:40px}}.cc-banner__container{display:flex;flex-direction:column;margin:auto;max-width:1320px;overflow:auto}.cc-banner__title{background:none!important;border:0;flex-shrink:0;font-size:18px!important;font-size:22px!important;font-weight:700!important;letter-spacing:normal;margin:0 0 12px!important;padding:0}@media (min-width:680px){.cc-banner__title{font-size:24px!important;margin:0 0 16px!important}}@media (min-width:1320px){.cc-banner__title{font-size:26px!important;margin:0 0 24px!important}}.cc-banner__body{display:flex;flex-direction:column;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif!important;overflow-x:hidden;overflow-y:auto;padding:3px 3px 16px}@media (min-width:980px){.cc-banner__body{flex-direction:row}}.cc-banner__policy p{font-size:16px!important;margin:0;max-width:none;padding:0}.cc-banner__policy p:not(:last-child){margin:0 0 16px}@media (min-width:980px){.cc-banner__policy p:not(:last-child){margin:0 0 24px}}.cc-banner__policy p a{font-size:16px!important;font-weight:700}.cc-banner__footer{box-shadow:none;flex-shrink:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif !important!important;margin:0;padding:12px 0 3px 3px;position:relative}@media (min-width:380px){.cc-banner__footer{align-items:stretch;display:flex;flex-wrap:wrap}}@media (min-width:680px){.cc-banner__footer{box-shadow:none;flex-wrap:nowrap;width:40%}}.cc-banner__footer .cc-banner__button{flex:1 1 auto;font-weight:700;overflow:hidden;padding:.5em 1em;width:100%}@media (min-width:680px){.cc-banner__footer .cc-banner__button{max-width:275px}}.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:12px}@media (min-width:680px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:0;margin-right:16px}}@media (min-width:980px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-right:24px}}@media (min-width:680px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:16px!important;margin-right:0!important}}@media (min-width:980px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:24px!important;margin-right:0!important}}.cc-banner__button-preferences{padding-left:0;padding-right:0}@media (min-width:380px){.cc-banner__button-preferences{flex:0 0 auto;margin:auto}}@media (min-width:680px){.cc-banner__button-preferences{margin:0}}@media (min-width:380px) and (max-width:680px){.cc-banner__button-break{display:block}}@media (min-width:680px){.cc-banner--is-tcf .cc-banner__footer{width:auto}}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__footer{padding-right:48px;width:66%}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px!important}.cc-banner--is-tcf .cc-banner__stacks{margin:16px 0 0}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__stacks{display:flex;flex:0 0 33%;flex-direction:column;margin:0 0 0 48px;overflow:auto}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px;margin:0 0 12px}.cc-banner--is-tcf .cc-banner__stacks-details{padding:0 3px}.cc-banner--is-tcf .cc-banner .cc-details{border-color:#fff3!important}.cc-box{border-radius:10px;padding:12px}.cc-box--info{background-color:#eff6fb}.cc-box--light{background-color:#faf9f6}.cc-button{border:2px solid #0000;border-radius:32px!important;color:inherit;cursor:pointer;font-size:14px;font-weight:700!important;left:auto;line-height:1.2;margin:0;padding:.5em 1em;right:auto;text-transform:none!important;transition:all .2s}@media (min-width:680px){.cc-button{font-size:14px;padding:.75em 1em}}.cc-button--sm{font-size:12px;padding:2px 8px!important}.cc-button--primary{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--primary:focus{outline:3px solid #08c}.cc-button--primary:focus,.cc-button--primary:hover,.cc-button--secondary{background-color:#fff!important;border-color:#025e8d!important;color:#025e8d!important}.cc-button--secondary:focus{background-color:#025e8d!important;border-color:#fff!important;color:#fff!important;outline:3px solid #08c!important}.cc-button--secondary:hover{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--secondary:active{border:2px solid #01324b!important}.cc-button--secondary.cc-button--contrast{border-color:#fff!important}.cc-button--secondary.cc-button--contrast:hover{background-color:initial!important;color:#fff!important}.cc-button--tertiary{background-color:#f0f7fc!important;background-image:none;border:2px solid #f0f7fc;color:#025e8d!important;text-decoration:underline!important;text-underline-offset:.25em}.cc-button--tertiary:focus{outline:3px solid #08c!important}.cc-button--tertiary:hover{text-decoration-thickness:.25em}.cc-button--tertiary:active{color:#013c5b}.cc-button--link{background-color:initial!important;border-color:#0000!important;border-radius:0!important;color:inherit!important;padding:0!important;text-decoration:underline!important;width:inherit!important}.cc-button--link:focus{outline:3px solid #08c}.cc-button--link:hover{background-color:initial!important;box-shadow:none;text-decoration:none}.cc-button--text{border-radius:0;padding:0}.cc-button--details{padding-right:24px!important;position:relative;width:auto}.cc-button--details:after,.cc-button--details:before{background-color:currentColor;content:"";position:absolute;transition:transform .25s ease-out}.cc-button--details:before{height:10px;margin-top:-5px;right:12px;top:50%;width:2px}.cc-button--details:after{height:2px;margin-top:-1px;right:8px;top:50%;width:10px}.cc-button--details.cc-active:before{transform:rotate(90deg);transform-origin:center}.cc-button--details.cc-active:after{display:none}.cc-details{border-bottom:1px solid #0000001a;padding:12px 0;position:relative;width:100%}.cc-details__summary{align-items:center;display:flex;font-size:14px;font-weight:700;list-style-type:none}.cc-details__summary:focus,.cc-details__summary:focus-visible{outline:3px solid #08c!important;will-change:auto}.cc-details__title{align-items:baseline;display:flex}.cc-details__title h2,.cc-details__title h3,.cc-details__title h4,.cc-details__title h5{line-height:1.4;margin:0!important;padding:0!important}.cc-details__title svg{flex-shrink:0;height:auto;margin-right:8px;position:relative;top:-1px;transition:all .2s;width:auto}.cc-details[open] .cc-details__title svg{top:2px;transform:rotate(90deg)}.cc-details__switch{margin-left:auto}.cc-details__section{padding:16px 0 0 18px}.cc-details__section p{margin:0}.cc-details__section p:not(:last-child){margin-bottom:12px}details summary::-webkit-details-marker{display:none}.cc-radio{align-items:center;display:flex;position:relative}.cc-radio *{cursor:pointer}.cc-radio__input{height:22px;left:0;position:absolute;top:0;width:22px}.cc-radio__input:focus{outline:none}.cc-radio__label{color:inherit;font-size:14px;font-weight:700;line-height:23px;margin:0;padding-left:28px}.cc-radio__label:after,.cc-radio__label:before{background-color:#fff;content:"";display:block;position:absolute;transition:transform .25s ease-out}.cc-radio__label:before{border:1px solid #777;border-radius:50%;height:22px;left:0;top:0;width:22px}.cc-radio__label:after{border:7px solid #025e8d;border-radius:50%;height:0;left:4px;opacity:0;top:4px;width:0}.cc-radio__input:focus+.cc-radio__label:before{box-shadow:0 0 0 2px #08c;outline:none}.cc-radio__label--hidden{display:none}.cc-radio__input:checked+.cc-radio__label:after{opacity:1}.cc-radio__input:disabled{cursor:default}.cc-radio__input:disabled+.cc-radio__label{cursor:default;opacity:.5}.cc-switch *{cursor:pointer}.cc-switch{align-items:center;display:flex;line-height:1}.cc-switch__label{cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;-webkit-user-select:none;user-select:none}.cc-switch__input[type=checkbox]{-webkit-appearance:none;appearance:none;background-color:#fff!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='rgba(0, 0, 0, 0.25)'/%3E%3C/svg%3E")!important;background-position:0!important;background-repeat:no-repeat!important;border:1px solid #777!important;border-radius:16px!important;height:18px!important;margin:0 8px 0 0!important;transition:background-position .2s ease-in-out!important;width:32px!important}.cc-switch__input[type=checkbox]:focus{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%2380b3cc'/%3E%3C/svg%3E")!important}.cc-switch__input[type=checkbox]:checked{background-color:#025e8d!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%23fff'/%3E%3C/svg%3E")!important;background-position:100%!important}.cc-switch__input[type=checkbox]:checked~.cc-switch__label>.cc-switch__text-off,.cc-switch__input[type=checkbox]:not(:checked)~.cc-switch__label>.cc-switch__text-on{display:none}.cc-switch__input[type=checkbox]:hover{cursor:pointer;outline:none}.cc-switch__input[type=checkbox]:focus-visible{cursor:pointer;outline:3px solid #fc0}.cc-switch__input[type=checkbox]:checked:before{background-color:none!important;color:#0000!important}html[dir=rtl] .cc-switch__input[type=checkbox]{margin-left:8px!important;margin-right:0!important}.cc-list>ul,ul.cc-list{list-style-type:disc;margin-left:0;padding-left:0}.cc-list>ul>li,ul.cc-list>li{margin-left:1em}.cc-list>ul>li:not(:last-child),ul.cc-list>li:not(:last-child){margin-bottom:4px}.cc-list__title{margin-bottom:8px!important}.cc-list--inline>ul,ul.cc-list--inline{display:flex;flex-wrap:wrap;list-style-type:none}.cc-list--inline>ul>li,ul.cc-list--inline>li{margin:0 .5em .5em 0}.cc-overlay{background-color:#0000004d;z-index:99998}.cc-overlay,.cc-preferences{bottom:0;left:0;position:fixed;right:0;top:0}.cc-preferences{background-color:#050a14f2!important;border:0;box-sizing:border-box;color:#111;font-family:sans-serif!important;line-height:1.4;margin:auto;max-height:100vh;overflow:auto;padding:0;z-index:100000}.cc-preferences:focus{outline:none}.cc-preferences *,.cc-preferences :after,.cc-preferences :before{box-sizing:inherit!important}.cc-preferences h2,.cc-preferences h3,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-preferences .cc-h2,.cc-preferences h2{font-size:18px}.cc-preferences .cc-h3,.cc-preferences h3{font-size:16px}.cc-preferences .cc-h4,.cc-preferences .cc-h5,.cc-preferences .cc-h6,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-size:14px}.cc-preferences a{color:#025e8d;text-decoration:underline}.cc-preferences a:hover{color:inherit;text-decoration:none}.cc-preferences h3{background:none;color:#111;padding:0;text-transform:none}dialog.cc-preferences{background-color:initial}dialog.cc-preferences::backdrop{background-color:#000000e6}.cc-preferences__dialog{display:flex;flex-direction:column;margin:auto;max-height:100vh;max-width:860px;padding:12px;position:relative}.cc-preferences__dialog>:last-child{border-bottom-left-radius:10px;border-bottom-right-radius:10px}@media (min-width:980px){.cc-preferences__dialog{padding:16px}}.cc-preferences__close{background:#0000!important;border:1px solid #ececec;border-radius:50%;color:#111!important;cursor:pointer;font-family:Times New Roman,serif;font-size:40px;height:40px;left:auto;line-height:1;margin-top:-20px;padding:0!important;position:absolute;right:20px;top:50%;width:40px}.cc-preferences__close:focus{outline:3px solid #08c}.cc-preferences__close-label{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__header{background:#fff!important;border-bottom:1px solid #0000001a;border-top-left-radius:10px;border-top-right-radius:10px;padding:16px;position:relative;text-align:center}.cc-preferences__title{background:none!important;color:#111!important;font-family:sans-serif!important;font-size:18px!important;font-weight:700!important;margin:0!important;padding:0 16px 0 0!important}@media (min-width:480px){.cc-preferences__title{padding-right:0!important}}@media (min-width:980px){.cc-preferences__title{font-size:22px!important}}.cc-preferences__body{background:#fff!important;flex:1 1 auto;min-height:200px;overflow-x:hidden;overflow-y:auto;padding:16px}.cc-preferences__footer{background:#fff!important;border-top:1px solid #d0d0d0;box-shadow:0 0 5px 0 #0003;margin-bottom:0;padding:12px;position:relative}@media (min-width:480px){.cc-preferences__footer{align-items:stretch;display:flex}}.cc-preferences__footer>.cc-button{display:block;width:100%}@media (min-width:480px){.cc-preferences__footer>.cc-button{flex:1 1 auto}}@media (min-width:980px){.cc-preferences__footer>.cc-button{flex-basis:auto}}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:first-child){margin-left:12px}}.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:8px}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:0}}.cc-preferences__categories{list-style:none;margin:0;padding:0}.cc-preferences__category:not(:last-child){border-bottom:1px solid #0000001a;margin-bottom:12px;padding-bottom:12px}.cc-preferences__category-description{font-size:14px;margin:0 0 8px;padding:0}.cc-preferences__category-footer{align-items:center;display:flex;justify-content:space-between}.cc-preferences__status{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__controls{display:flex;margin:0}.cc-preferences__controls>:not(:last-child){margin-right:12px}.cc-preferences__always-on{font-size:14px;font-weight:700;padding-left:26px;position:relative}.cc-preferences__always-on:before{background:#0000!important;border:solid;border-top-color:#0000;border-width:0 0 4px 4px;content:"";display:block;height:10px;left:0;position:absolute;top:2px;transform:rotate(-45deg);width:18px}.cc-preferences__details{background-color:#eff6fb!important;border-radius:10px;font-size:14px;margin:12px 0 0;padding:12px}.cc-preferences__cookie-list,.cc-preferences__provider-list{list-style:none;margin:0;padding:0}.cc-preferences__provider-list{columns:170px}.cc-preferences__cookie-title{font-size:1em;font-style:normal;font-weight:400;margin:0}.cc-preferences__cookie-description{font-size:1em;margin:0 0 8px}.cc-preferences__cookie-domain,.cc-preferences__cookie-lifespan{border-left:1px solid #999;color:#666;margin-left:8px;padding-left:4px}body.cc-has-preferences-open{overflow:hidden;position:relative}.cc-table{border-collapse:collapse;width:100%}.cc-table tbody tr{border-top:1px solid #0000001a}.cc-table td,.cc-table th{font-size:14px;padding:4px 8px;vertical-align:top}.cc-table+.cc-table{margin-top:8px}.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:70%}@media (min-width:680px){.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:80%}}.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{padding-left:8px;width:40%}@media (min-width:680px){.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{width:30%}}.cc-tabs,.cc-tabs>ul{background-color:#faf9f6;display:flex;list-style:none;margin:0;padding:0}.cc-tabs__panel{border-top:1px solid #ececec;margin-top:-1px}.cc-tabs__button{background-color:initial;border:0;border-left:1px solid #0000;border-right:1px solid #0000;color:#111;font-size:16px;font-weight:700;margin:0;padding:15px 16px 12px;position:relative}.cc-tabs__button:before{background-color:initial;content:"";display:block;height:3px;left:0;position:absolute;top:0;width:100%}.cc-tabs__button.cc-active{background-color:#fff;border-color:#025e8d #ececec #ececec;color:#111}.cc-tabs__button.cc-active:before{background-color:#025e8d}.cc-tabs__content{background-color:#fff;display:none;font-size:14px;padding:16px}.cc-tabs__section:not(:first-child){padding-top:24px}.cc-tcf{font-size:14px;margin-top:16px}.cc-tcf__list{list-style:none;margin:0;padding:0}.cc-vendor-count{color:#666;font-size:16px;margin:0 0 8px}.cc-hide{display:none!important}.cc-show{display:block!important}.cc-external-link{background-color:#ececec;background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTcgMGExIDEgMCAxIDEgMCAySDIuNmMtLjM3MSAwLS42LjIwOS0uNi41djE1YzAgLjI5MS4yMjkuNS42LjVoMTQuOGMuMzcxIDAgLjYtLjIwOS42LS41VjEzYTEgMSAwIDAgMSAyIDB2NC41YzAgMS40MzgtMS4xNjIgMi41LTIuNiAyLjVIMi42QzEuMTYyIDIwIDAgMTguOTM4IDAgMTcuNXYtMTVDMCAxLjA2MiAxLjE2MiAwIDIuNiAwem02IDBoNmwuMDc1LjAwMy4xMjYuMDE3LjExMS4wMy4xMTEuMDQ0LjA5OC4wNTIuMDk2LjA2Ny4wOS4wOGExIDEgMCAwIDEgLjA5Ny4xMTJsLjA3MS4xMS4wNTQuMTE0LjAzNS4xMDUuMDMuMTQ4TDIwIDF2NmExIDEgMCAwIDEtMiAwVjMuNDE0bC02LjY5MyA2LjY5M2ExIDEgMCAwIDEtMS40MTQtMS40MTRMMTYuNTg0IDJIMTNhMSAxIDAgMCAxLS45OTMtLjg4M0wxMiAxYTEgMSAwIDAgMSAxLTEiLz48L3N2Zz4=");background-position:right 8px center;background-repeat:no-repeat;background-size:10px auto;border-radius:3px;box-shadow:0 1px 0 #0003;color:#111!important;display:inline-block;font-size:12px;padding:4px 26px 4px 8px;text-decoration:none!important}.cc-external-link:hover{text-decoration:underline!important}.cc-m-0{margin:0!important}.cc-grey{color:#666}</style><script async="" type="text/javascript" src="https://injections.readcube.com/nature/inject.fa744f3f.js"></script></head>
<body class="article-page"><dialog class="cc-banner" data-cc-banner="" data-nosnippet="" aria-labelledby="cc-banner-label" open="">
		<div class="cc-banner__content" autofocus="" tabindex="-1">
			<div class="cc-banner__container">
				<div class="cc-banner__header">
					<h2 class="cc-banner__title" id="cc-banner-label">Your privacy, your choice</h2>
				</div>
				<div class="cc-banner__body">
					<div class="cc-banner__policy">
						<p>We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.</p><p>By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.</p><p>See our <a href="https://www.nature.com/info/privacy" data-cc-action="privacy">privacy policy</a> for more information on the use of your personal data.</p><p><button type="button" data-cc-action="preferences" class="cc-button cc-button--link cc-button--text">Manage preferences</button> for further information and to change your choices.</p>
					</div>
					
				</div>
				<div class="cc-banner__footer">
					
			<button data-cc-action="accept" class="cc-button cc-button--secondary cc-button--contrast cc-banner__button cc-banner__button-accept">Accept all cookies</button>
			
		
				</div>
			</div>
		</div>
	</dialog>

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90" style="display: none !important;">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-test="top-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41591-022-01927-8;doi=10.1038/s41591-022-01927-8;subjmeta=1621,1895,1915,1990,2329,2486,283,291,631,67,68,69;kwrd=B-cell+lymphoma,Cancer+epigenetics,Cancer+genomics,Chronic+lymphocytic+leukaemia,Tumour+heterogeneity">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=-1555341097&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01927-8%26doi%3D10.1038/s41591-022-01927-8%26subjmeta%3D1621,1895,1915,1990,2329,2486,283,291,631,67,68,69%26kwrd%3DB-cell+lymphoma,Cancer+epigenetics,Cancer+genomics,Chronic+lymphocytic+leukaemia,Tumour+heterogeneity">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=-1555341097&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01927-8%26doi%3D10.1038/s41591-022-01927-8%26subjmeta%3D1621,1895,1915,1990,2329,2486,283,291,631,67,68,69%26kwrd%3DB-cell+lymphoma,Cancer+epigenetics,Cancer+genomics,Chronic+lymphocytic+leukaemia,Tumour+heterogeneity"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header="" data-track-component="nature-150-split-header" style="border-color:#e40428">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nm" data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" height="32" alt="Nature Medicine">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
                            </a><div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected="">All journals</option>
                                    <option value="nm">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41591-022-01927-8"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header" hidden="">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-articles" data-track="click" data-track-action="research articles" data-track-label="link" data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-and-analysis" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link" data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/podcast" data-track="click" data-track-action="podcasts" data-track-label="link" data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/current-issue" data-track="click" data-track-action="current issue" data-track-label="link" data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/collections" data-track="click" data-track-action="collections" data-track-label="link" data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://www.facebook.com/Nature-Medicine-193691346949/" data-track="click" data-track-action="facebook" data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://twitter.com/naturemedicine" data-track="click" data-track-action="twitter" data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=5" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/aims" data-track="click" data-track-action="aims &amp; scope" data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-information" data-track="click" data-track-action="journal information" data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-impact" data-track="click" data-track-action="journal metrics" data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editors" data-track="click" data-track-action="about the editors" data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-cross-journal-editorial-team" data-track="click" data-track-action="research cross-journal editorial team" data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-cross-journal-editorial-team" data-track="click" data-track-action="reviews cross-journal editorial team" data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/statistics-advisory-panel" data-track="click" data-track-action="statistical advisory panel" data-track-label="link">
                                    Statistical Advisory Panel
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/our-publishing-models" data-track="click" data-track-action="our publishing models" data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-values-statement" data-track="click" data-track-action="editorial values statement" data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-policies" data-track="click" data-track-action="editorial policies" data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/content" data-track="click" data-track-action="content types" data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="javascript:;" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                        </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/submission-guidelines" data-track="click" data-track-action="submission guidelines" data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-context="header publish with us dropdown menu" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/open-access-funding" data-test="funding-eligibility-link" data-track="click_explore_funding" data-track-context="header publish with us" data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link" href="https://mts-nmed.nature.com/cgi-bin/main.plex" data-track="click_submit_manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-action="submit manuscript" data-track-label="link (publish with us dropdown menu)" data-track-external="" data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link" href="https://journal-alerts.springernature.com/subscribe?journal_id=41591" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-label="link (desktop site header)" data-track-external="">
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature medicine"><span itemprop="name">nature medicine</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm/articles?type=article" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" id="js-enable-context-bar" data-test="context-bar" data-context-bar="" aria-hidden="true">
                    <div class="c-context-bar__container u-container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            Detection of early seeding of Richter transformation in chronic lymphocytic leukemia
                        </div>
                        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01927-8.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01927-8.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2022-08-11">11 August 2022</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Detection of early seeding of Richter transformation in chronic lymphocytic leukemia</h1>
                    <ul class="c-article-author-list c-article-author-list--long js-no-scroll" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ferran-Nadeu-Aff1-Aff2" data-author-popup="auth-Ferran-Nadeu-Aff1-Aff2" data-author-search="Nadeu, Ferran" data-corresp-id="c1" data-track-context="researcher popup with no profile" data-track-index="1_52">Ferran Nadeu<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-2910-9440"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2910-9440</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Romina-Royo-Aff3" data-author-popup="auth-Romina-Royo-Aff3" data-author-search="Royo, Romina" data-track-context="researcher popup with no profile" data-track-index="2_52">Romina Royo</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-3493-0874"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-3493-0874</a></span><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ramon-Massoni_Badosa-Aff4" data-author-popup="auth-Ramon-Massoni_Badosa-Aff4" data-author-search="Massoni-Badosa, Ramon" data-track-context="researcher popup with no profile" data-track-index="3_52">Ramon Massoni-Badosa</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Heribert-Playa_Albinyana-Aff1-Aff2" data-author-popup="auth-Heribert-Playa_Albinyana-Aff1-Aff2" data-author-search="Playa-Albinyana, Heribert" data-track-context="researcher popup with no profile" data-track-index="4_52">Heribert Playa-Albinyana</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3447-3848"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3447-3848</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Beatriz-Garcia_Torre-Aff1" data-author-popup="auth-Beatriz-Garcia_Torre-Aff1" data-author-search="Garcia-Torre, Beatriz" data-track-context="researcher popup with no profile" data-track-index="5_52">Beatriz Garcia-Torre</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Mart_-Duran_Ferrer-Aff1-Aff2" data-author-popup="auth-Mart_-Duran_Ferrer-Aff1-Aff2" data-author-search="Duran-Ferrer, Martí" data-track-context="researcher popup with no profile" data-track-index="6_52">Martí Duran-Ferrer</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-1666-5819"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1666-5819</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kevin_J_-Dawson-Aff5" data-author-popup="auth-Kevin_J_-Dawson-Aff5" data-author-search="Dawson, Kevin J." data-track-context="researcher popup with no profile" data-track-index="7_52">Kevin J. Dawson</a><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Marta-Kulis-Aff1" data-author-popup="auth-Marta-Kulis-Aff1" data-author-search="Kulis, Marta" data-track-context="researcher popup with no profile" data-track-index="8_52">Marta Kulis</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ander-Diaz_Navarro-Aff2-Aff6" data-author-popup="auth-Ander-Diaz_Navarro-Aff2-Aff6" data-author-search="Diaz-Navarro, Ander" data-track-context="researcher popup with no profile" data-track-index="9_52">Ander Diaz-Navarro</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Neus-Villamor-Aff1-Aff2-Aff7" data-author-popup="auth-Neus-Villamor-Aff1-Aff2-Aff7" data-author-search="Villamor, Neus" data-track-context="researcher popup with no profile" data-track-index="10_52">Neus Villamor</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Juan_L_-Melero-Aff8" data-author-popup="auth-Juan_L_-Melero-Aff8" data-author-search="Melero, Juan L." data-track-context="researcher popup with no profile" data-track-index="11_52">Juan L. Melero</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-1387-6231"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1387-6231</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Vicente-Chapaprieta-Aff1" data-author-popup="auth-Vicente-Chapaprieta-Aff1" data-author-search="Chapaprieta, Vicente" data-track-context="researcher popup with no profile" data-track-index="12_52">Vicente Chapaprieta</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-4910-5394"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-4910-5394</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ana-Dueso_Barroso-Aff3" data-author-popup="auth-Ana-Dueso_Barroso-Aff3" data-author-search="Dueso-Barroso, Ana" data-track-context="researcher popup with no profile" data-track-index="13_52">Ana Dueso-Barroso</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-2000-9718"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2000-9718</a></span><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Julio-Delgado-Aff1-Aff2-Aff7-Aff9" data-author-popup="auth-Julio-Delgado-Aff1-Aff2-Aff7-Aff9" data-author-search="Delgado, Julio" data-track-context="researcher popup with no profile" data-track-index="14_52">Julio Delgado</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff9" tabindex="-1">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Riccardo-Moia-Aff10" data-author-popup="auth-Riccardo-Moia-Aff10" data-author-search="Moia, Riccardo" data-track-context="researcher popup with no profile" data-track-index="15_52">Riccardo Moia</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-7393-1138"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7393-1138</a></span><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sara-Ruiz_Gil-Aff4" data-author-popup="auth-Sara-Ruiz_Gil-Aff4" data-author-search="Ruiz-Gil, Sara" data-track-context="researcher popup with no profile" data-track-index="16_52">Sara Ruiz-Gil</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Domenica-Marchese-Aff4" data-author-popup="auth-Domenica-Marchese-Aff4" data-author-search="Marchese, Domenica" data-track-context="researcher popup with no profile" data-track-index="17_52">Domenica Marchese</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ariadna-Gir_-Aff1-Aff2" data-author-popup="auth-Ariadna-Gir_-Aff1-Aff2" data-author-search="Giró, Ariadna" data-track-context="researcher popup with no profile" data-track-index="18_52">Ariadna Giró</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-N_ria-Verdaguer_Dot-Aff1" data-author-popup="auth-N_ria-Verdaguer_Dot-Aff1" data-author-search="Verdaguer-Dot, Núria" data-track-context="researcher popup with no profile" data-track-index="19_52">Núria Verdaguer-Dot</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-M_nica-Romo-Aff1" data-author-popup="auth-M_nica-Romo-Aff1" data-author-search="Romo, Mónica" data-track-context="researcher popup with no profile" data-track-index="20_52">Mónica Romo</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Guillem-Clot-Aff1-Aff2" data-author-popup="auth-Guillem-Clot-Aff1-Aff2" data-author-search="Clot, Guillem" data-track-context="researcher popup with no profile" data-track-index="21_52">Guillem Clot</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-2588-7413"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2588-7413</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Maria-Rozman-Aff1-Aff7" data-author-popup="auth-Maria-Rozman-Aff1-Aff7" data-author-search="Rozman, Maria" data-track-context="researcher popup with no profile" data-track-index="22_52">Maria Rozman</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Gerard-Frigola-Aff7" data-author-popup="auth-Gerard-Frigola-Aff7" data-author-search="Frigola, Gerard" data-track-context="researcher popup with no profile" data-track-index="23_52">Gerard Frigola</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6794-6456"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6794-6456</a></span><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Alfredo-Rivas_Delgado-Aff1-Aff7" data-author-popup="auth-Alfredo-Rivas_Delgado-Aff1-Aff7" data-author-search="Rivas-Delgado, Alfredo" data-track-context="researcher popup with no profile" data-track-index="24_52">Alfredo Rivas-Delgado</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-0385-3415"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0385-3415</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tycho-Baumann-Aff2-Aff7-Aff20" data-author-popup="auth-Tycho-Baumann-Aff2-Aff7-Aff20" data-author-search="Baumann, Tycho" data-track-context="researcher popup with no profile" data-track-index="25_52">Tycho Baumann</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff7" tabindex="-1">7</a></sup><sup class="u-js-hide">&nbsp;<a href="#nAff20" tabindex="-1">nAff20</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Miguel-Alcoceba-Aff2-Aff11" data-author-popup="auth-Miguel-Alcoceba-Aff2-Aff11" data-author-search="Alcoceba, Miguel" data-track-context="researcher popup with no profile" data-track-index="26_52">Miguel Alcoceba</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3819-4846"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3819-4846</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff11" tabindex="-1">11</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Marcos-Gonz_lez-Aff2-Aff11" data-author-popup="auth-Marcos-Gonz_lez-Aff2-Aff11" data-author-search="González, Marcos" data-track-context="researcher popup with no profile" data-track-index="27_52">Marcos González</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff11" tabindex="-1">11</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Fina-Climent-Aff12" data-author-popup="auth-Fina-Climent-Aff12" data-author-search="Climent, Fina" data-track-context="researcher popup with no profile" data-track-index="28_52">Fina Climent</a><sup class="u-js-hide"><a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Pau-Abrisqueta-Aff13" data-author-popup="auth-Pau-Abrisqueta-Aff13" data-author-search="Abrisqueta, Pau" data-track-context="researcher popup with no profile" data-track-index="29_52">Pau Abrisqueta</a><sup class="u-js-hide"><a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Josep-Castellv_-Aff13" data-author-popup="auth-Josep-Castellv_-Aff13" data-author-search="Castellví, Josep" data-track-context="researcher popup with no profile" data-track-index="30_52">Josep Castellví</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-5745-9996"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-5745-9996</a></span><sup class="u-js-hide"><a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Francesc-Bosch-Aff13" data-author-popup="auth-Francesc-Bosch-Aff13" data-author-search="Bosch, Francesc" data-track-context="researcher popup with no profile" data-track-index="31_52">Francesc Bosch</a><sup class="u-js-hide"><a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Marta-Aymerich-Aff1-Aff2-Aff7" data-author-popup="auth-Marta-Aymerich-Aff1-Aff2-Aff7" data-author-search="Aymerich, Marta" data-track-context="researcher popup with no profile" data-track-index="32_52">Marta Aymerich</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Anna-Enjuanes-Aff1" data-author-popup="auth-Anna-Enjuanes-Aff1" data-author-search="Enjuanes, Anna" data-track-context="researcher popup with no profile" data-track-index="33_52">Anna Enjuanes</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-S_lvia-Ruiz_Gasp_-Aff1" data-author-popup="auth-S_lvia-Ruiz_Gasp_-Aff1" data-author-search="Ruiz-Gaspà, Sílvia" data-track-context="researcher popup with no profile" data-track-index="34_52">Sílvia Ruiz-Gaspà</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Armando-L_pez_Guillermo-Aff1-Aff2-Aff7-Aff9" data-author-popup="auth-Armando-L_pez_Guillermo-Aff1-Aff2-Aff7-Aff9" data-author-search="López-Guillermo, Armando" data-track-context="researcher popup with no profile" data-track-index="35_52">Armando López-Guillermo</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff9" tabindex="-1">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Pedro-Jares-Aff1-Aff2-Aff7-Aff9" data-author-popup="auth-Pedro-Jares-Aff1-Aff2-Aff7-Aff9" data-author-search="Jares, Pedro" data-track-context="researcher popup with no profile" data-track-index="36_52">Pedro Jares</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff9" tabindex="-1">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-S_lvia-Be_-Aff1-Aff2-Aff7-Aff9" data-author-popup="auth-S_lvia-Be_-Aff1-Aff2-Aff7-Aff9" data-author-search="Beà, Sílvia" data-track-context="researcher popup with no profile" data-track-index="37_52">Sílvia Beà</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-7192-2385"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7192-2385</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff9" tabindex="-1">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Salvador-Capella_Gutierrez-Aff3" data-author-popup="auth-Salvador-Capella_Gutierrez-Aff3" data-author-search="Capella-Gutierrez, Salvador" data-track-context="researcher popup with no profile" data-track-index="38_52">Salvador Capella-Gutierrez</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-0309-604X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0309-604X</a></span><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Josep_Ll_-Gelp_-Aff3-Aff9" data-author-popup="auth-Josep_Ll_-Gelp_-Aff3-Aff9" data-author-search="Gelpí, Josep Ll." data-track-context="researcher popup with no profile" data-track-index="39_52">Josep Ll. Gelpí</a><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff9" tabindex="-1">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-N_ria-L_pez_Bigas-Aff14-Aff15-Aff16" data-author-popup="auth-N_ria-L_pez_Bigas-Aff14-Aff15-Aff16" data-author-search="López-Bigas, Núria" data-track-context="researcher popup with no profile" data-track-index="40_52">Núria López-Bigas</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-4925-8988"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-4925-8988</a></span><sup class="u-js-hide"><a href="#Aff14" tabindex="-1">14</a>,<a href="#Aff15" tabindex="-1">15</a>,<a href="#Aff16" tabindex="-1">16</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-David-Torrents-Aff3-Aff16" data-author-popup="auth-David-Torrents-Aff3-Aff16" data-author-search="Torrents, David" data-track-context="researcher popup with no profile" data-track-index="41_52">David Torrents</a><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff16" tabindex="-1">16</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Peter_J_-Campbell-Aff5" data-author-popup="auth-Peter_J_-Campbell-Aff5" data-author-search="Campbell, Peter J." data-track-context="researcher popup with no profile" data-track-index="42_52">Peter J. Campbell</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3921-0510"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3921-0510</a></span><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ivo-Gut-Aff4-Aff15" data-author-popup="auth-Ivo-Gut-Aff4-Aff15" data-author-search="Gut, Ivo" data-track-context="researcher popup with no profile" data-track-index="43_52">Ivo Gut</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-7219-632X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7219-632X</a></span><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff15" tabindex="-1">15</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Davide-Rossi-Aff17" data-author-popup="auth-Davide-Rossi-Aff17" data-author-search="Rossi, Davide" data-track-context="researcher popup with no profile" data-track-index="44_52">Davide Rossi</a><sup class="u-js-hide"><a href="#Aff17" tabindex="-1">17</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Gianluca-Gaidano-Aff10" data-author-popup="auth-Gianluca-Gaidano-Aff10" data-author-search="Gaidano, Gianluca" data-track-context="researcher popup with no profile" data-track-index="45_52">Gianluca Gaidano</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-4681-0151"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-4681-0151</a></span><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Xose_S_-Puente-Aff2-Aff6" data-author-popup="auth-Xose_S_-Puente-Aff2-Aff6" data-author-search="Puente, Xose S." data-track-context="researcher popup with no profile" data-track-index="46_52">Xose S. Puente</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-9525-1483"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9525-1483</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Pablo_M_-Garcia_Roves-Aff9-Aff18" data-author-popup="auth-Pablo_M_-Garcia_Roves-Aff9-Aff18" data-author-search="Garcia-Roves, Pablo M." data-track-context="researcher popup with no profile" data-track-index="47_52">Pablo M. Garcia-Roves</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-8371-2067"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8371-2067</a></span><sup class="u-js-hide"><a href="#Aff9" tabindex="-1">9</a>,<a href="#Aff18" tabindex="-1">18</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Dolors-Colomer-Aff1-Aff2-Aff7-Aff9" data-author-popup="auth-Dolors-Colomer-Aff1-Aff2-Aff7-Aff9" data-author-search="Colomer, Dolors" data-track-context="researcher popup with no profile" data-track-index="48_52">Dolors Colomer</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-7486-8484"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7486-8484</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff9" tabindex="-1">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Holger-Heyn-Aff4-Aff15" data-author-popup="auth-Holger-Heyn-Aff4-Aff15" data-author-search="Heyn, Holger" data-track-context="researcher popup with no profile" data-track-index="49_52">Holger Heyn</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3276-1889"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3276-1889</a></span><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff15" tabindex="-1">15</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Francesco-Maura-Aff19" data-author-popup="auth-Francesco-Maura-Aff19" data-author-search="Maura, Francesco" data-track-context="researcher popup with no profile" data-track-index="50_52">Francesco Maura</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-5017-1620"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-5017-1620</a></span><sup class="u-js-hide"><a href="#Aff19" tabindex="-1">19</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jos__I_-Mart_n_Subero-Aff1-Aff2-Aff9-Aff16" data-author-popup="auth-Jos__I_-Mart_n_Subero-Aff1-Aff2-Aff9-Aff16" data-author-search="Martín-Subero, José I." data-track-context="researcher popup with no profile" data-track-index="51_52">José I. Martín-Subero</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-8809-5195"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-8809-5195</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff9" tabindex="-1">9</a>,<a href="#Aff16" tabindex="-1">16</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 52 authors for this article" title="Show all 52 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-El_as-Campo-Aff1-Aff2-Aff7-Aff9" data-author-popup="auth-El_as-Campo-Aff1-Aff2-Aff7-Aff9" data-author-search="Campo, Elías" data-corresp-id="c2" data-track-context="researcher popup with no profile" data-track-index="52_52">Elías Campo<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-9850-9793"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9850-9793</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff9" tabindex="-1">9</a></sup>&nbsp;</li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nm" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Medicine</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;28</b>,&nbsp;<span class="u-visually-hidden">pages </span>1662–1671 (<span data-test="article-publication-year">2022</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item" data-test="access-count">
                        <p class="c-article-metrics-bar__count">29k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item" data-test="citation-count">
                        <p class="c-article-metrics-bar__count">83 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item" data-test="altmetric-score">
                            <p class="c-article-metrics-bar__count">430 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41591-022-01927-8/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Richter transformation (RT) is a paradigmatic evolution of chronic lymphocytic leukemia (CLL) into a very aggressive large B cell lymphoma conferring a dismal prognosis. The mechanisms driving RT remain largely unknown. We characterized the whole genome, epigenome and transcriptome, combined with single-cell DNA/RNA-sequencing analyses and functional experiments, of 19 cases of CLL developing RT. Studying 54 longitudinal samples covering up to 19 years of disease course, we uncovered minute subclones carrying genomic, immunogenetic and transcriptomic features of RT cells already at CLL diagnosis, which were dormant for up to 19 years before transformation. We also identified new driver alterations, discovered a new mutational signature (SBS-RT), recognized an oxidative phosphorylation (OXPHOS)<sup>high</sup>–B cell receptor (BCR)<sup>low</sup>-signaling transcriptional axis in RT and showed that OXPHOS inhibition reduces the proliferation of RT cells. These findings demonstrate the early seeding of subclones driving advanced stages of cancer evolution and uncover potential therapeutic targets for RT.</p></div></div></section>

            
                
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41591-022-02113-6/MediaObjects/41591_2022_2113_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41591-022-02113-6?fromPaywallRec=false" data-track="select_recommendations_1" data-track-context="inline recommendations" data-track-action="click recommendations inline - 1" data-track-label="10.1038/s41591-022-02113-6">Evolutionary history of transformation from chronic lymphocytic leukemia to Richter syndrome
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">09 January 2023</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-022-34642-6/MediaObjects/41467_2022_34642_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41467-022-34642-6?fromPaywallRec=false" data-track="select_recommendations_2" data-track-context="inline recommendations" data-track-action="click recommendations inline - 2" data-track-label="10.1038/s41467-022-34642-6">Molecular characterization of Richter syndrome identifies de novo diffuse large B-cell lymphomas with poor prognosis
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">19 January 2023</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-022-01140-w/MediaObjects/41588_2022_1140_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41588-022-01140-w?fromPaywallRec=false" data-track="select_recommendations_3" data-track-context="inline recommendations" data-track-action="click recommendations inline - 3" data-track-label="10.1038/s41588-022-01140-w">Molecular map of chronic lymphocytic leukemia and its impact on outcome
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">04 August 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1749524856,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Clonal evolution<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Cairns, J. Mutation selection and the natural history of cancer. Nature 255, 197–200 (1975)." href="/articles/s41591-022-01927-8#ref-CR1" id="ref-link-section-d73061417e1346">1</a></sup> drives cancer initiation, progression and relapse due to the stepwise acquisition and/or selection of fitter subclones<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Nowell, P. C. The clonal evolution of tumor cell populations. Science 194, 23–28 (1976)." href="/articles/s41591-022-01927-8#ref-CR2" id="ref-link-section-d73061417e1350">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Dentro, S. C. et al. Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes. Cell 184, 2239–2254 (2021)." href="/articles/s41591-022-01927-8#ref-CR3" id="ref-link-section-d73061417e1353">3</a></sup>. The understanding of tumor evolution is hampered by the analysis of bulk tumor cell populations at low resolution and at single or limited time points of the disease course in most studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gerstung, M. et al. The evolutionary history of 2,658 cancers. Nature 578, 122–128 (2020)." href="/articles/s41591-022-01927-8#ref-CR4" id="ref-link-section-d73061417e1357">4</a></sup>. A better knowledge of this process might translate into anticipation-based treatment strategies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Ferrando, A. A. &amp; López-Otín, C. Clonal evolution in leukemia. Nat. Med. 23, 1135–1145 (2017)." href="/articles/s41591-022-01927-8#ref-CR5" id="ref-link-section-d73061417e1361">5</a></sup>. RT in CLL represents a paradigmatic model of cancer evolution occurring rarely in treatment-naive patients with CLL but found in 4–20% of patients after chemoimmunotherapy (CIT) and targeted therapies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Ding, W. Richter transformation in the era of novel agents. Hematology 2018, 256–263 (2018)." href="/articles/s41591-022-01927-8#ref-CR6" id="ref-link-section-d73061417e1365">6</a></sup>. RT sometimes occurs within the first months after treatment initiation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Maddocks, K. J. et al. Etiology of ibrutinib therapy discontinuation and outcomes in patients with chronic lymphocytic leukemia. JAMA Oncol. 1, 80 (2015)." href="#ref-CR7" id="ref-link-section-d73061417e1370">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ahn, I. E. et al. Clonal evolution leading to ibrutinib resistance in chronic lymphocytic leukemia. Blood 129, 1469–1479 (2017)." href="#ref-CR8" id="ref-link-section-d73061417e1370_1">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Jain, P. et al. Outcomes of patients with chronic lymphocytic leukemia after discontinuing ibrutinib. Blood 125, 2062–2067 (2015)." href="/articles/s41591-022-01927-8#ref-CR9" id="ref-link-section-d73061417e1373">9</a></sup>, suggesting selection of pre-existing subclones<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Landau, D. A. et al. The evolutionary landscape of chronic lymphocytic leukemia treated with ibrutinib targeted therapy. Nat. Commun. 8, 2185 (2017)." href="/articles/s41591-022-01927-8#ref-CR10" id="ref-link-section-d73061417e1377">10</a></sup>. Nonetheless, the genomic/epigenomic mechanisms driving RT after CIT<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Beà, S. et al. Genetic imbalances in progressed B-cell chronic lymphocytic leukemia and transformed large-cell lymphoma (Richter’s syndrome). Am. J. Pathol. 161, 957–968 (2002)." href="#ref-CR11" id="ref-link-section-d73061417e1381">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Scandurra, M. et al. Genomic profiling of Richter’s syndrome: recurrent lesions and differences with de novo diffuse large B-cell lymphomas. Hematol. Oncol. 28, 62–67 (2010)." href="#ref-CR12" id="ref-link-section-d73061417e1381_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rossi, D. et al. The genetics of Richter syndrome reveals disease heterogeneity and predicts survival after transformation. Blood 117, 3391–3401 (2011)." href="#ref-CR13" id="ref-link-section-d73061417e1381_2">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fabbri, G. et al. Genetic lesions associated with chronic lymphocytic leukemia transformation to Richter syndrome. J. Exp. Med. 210, 2273–2288 (2013)." href="#ref-CR14" id="ref-link-section-d73061417e1381_3">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chigrinova, E. et al. Two main genetic pathways lead to the transformation of chronic lymphocytic leukemia to Richter syndrome. Blood 122, 2673–2682 (2013)." href="#ref-CR15" id="ref-link-section-d73061417e1381_4">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Klintman, J. et al. Genomic and transcriptomic correlates of Richter transformation in chronic lymphocytic leukemia. Blood 137, 2800–2816 (2021)." href="#ref-CR16" id="ref-link-section-d73061417e1381_5">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Chakraborty, S. et al. B-cell receptor signaling and genetic lesions in TP53 and CDKN2A/CDKN2B cooperate in Richter transformation. Blood 138, 1053–1066 (2021)." href="/articles/s41591-022-01927-8#ref-CR17" id="ref-link-section-d73061417e1384">17</a></sup> or targeted agents<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Anderson, M. A. et al. Clinicopathological features and outcomes of progression of CLL on the BCL2 inhibitor venetoclax. Blood 129, 3362–3370 (2017)." href="#ref-CR18" id="ref-link-section-d73061417e1388">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Miller, C. R. et al. Near-tetraploidy is associated with Richter transformation in chronic lymphocytic leukemia patients receiving ibrutinib. Blood Adv. 1, 1584–1588 (2017)." href="#ref-CR19" id="ref-link-section-d73061417e1388_1">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kadri, S. et al. Clonal evolution underlying leukemia progression and Richter transformation in patients with ibrutinib-relapsed CLL. Blood Adv. 1, 715–727 (2017)." href="#ref-CR20" id="ref-link-section-d73061417e1388_2">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Herling, C. D. et al. Clonal dynamics towards the development of venetoclax resistance in chronic lymphocytic leukemia. Nat. Commun. 9, 727 (2018)." href="/articles/s41591-022-01927-8#ref-CR21" id="ref-link-section-d73061417e1391">21</a></sup> are not well known. The aims of the present study were to reconstruct the evolutionary history of RT and to reveal the molecular processes underlying this transformation.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Genomic characterization of RT</h3><p>We sequenced 53 whole genomes and 1 whole exome of synchronous or longitudinal samples of 19 patients (up to six time points per patient) in whom CLL transformed into diffuse large B cell lymphoma (RT-DLBCL; <i>n</i> = 17), plasmablastic lymphoma (RT-PBL; <i>n</i> = 1) or prolymphocytic leukemia (RT-PLL; <i>n</i> = 1). Nontumor samples were available in 12 patients. RT occurred simultaneously with CLL at diagnosis (<i>n</i> = 3) or after up to 19 years following different lines of treatment with CIT (<i>n</i> = 6) and targeted therapies (<i>n</i> = 10; BCR inhibitors, ibrutinib <i>n</i> = 6; duvelisib <i>n</i> = 2; idelalisib <i>n</i> = 1; and BCL2 inhibitor, venetoclax <i>n</i> = 1). All instances of RT were clonally related to CLL, 15 tumors had unmutated IGHV (U-CLL) and 4 had mutated IGHV (M-CLL). Whole-genome sequencing (WGS) data were integrated with bulk epigenetic and transcriptomic analyses as well as single-cell DNA and RNA sequencing (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1a</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig6">1</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">2</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="The genomic landscape of RT."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: The genomic landscape of RT.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01927-8/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="737"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Summary of the study. mut., mutation. <b>b</b>, Increase in genomic alterations and epigenetic changes compared to healthy naive and memory B cells over the disease course. Center line indicates median; box limits indicate upper and lower quartiles; whiskers indicate 1.5 × interquartile range; and points indicate individual samples. <b>c</b>, Driver alterations of CLL and RT. New drivers in RT are labeled in blue. Each column represents a sample and genes are represented in rows. The transparency of the color of mutations and CNAs indicates the cancer cell fraction (CCF). The number of tumors harboring an alteration at the time of transformation is indicated for each biological group of drivers (left). Complex structural alterations are shown below, together with the total number of SVs. LOH, loss of heterozygosity. <b>d</b>, Schema of the <i>CCND3</i> insertion next to the constant region IGLC1 in the RT sample of patient 835. <b>e</b>, Reciprocal translocation between <i>MYCN</i> and class-switch recombination (CSR) region of IGHG3 in the RT sample of patient 816 (top). <i>MYCN</i> expression based on bulk RNA-seq (bottom). <b>f</b>, Chromoplexy disrupting <i>SMARCA4</i> in the RT sample of patient 4,675. <b>g</b>, The circos plot (left) displays the SVs (links) and CNAs (inner circle) found in the RT sample of patient 1,669. CNAs are colored by type and SVs are colored according to their occurrence within specific complex events. Target driver genes are annotated. Chromosome-specific plots (right) illustrate selected complex rearrangements affecting one or multiple driver genes with CNAs and SVs colored by type.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01927-8/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>The WGS and epigenome of CLL and RT revealed a concordant increased complexity from CLL diagnosis to relapse and RT (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1b</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig7">2a</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">3</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">8</a>). The RT genomes carried a median of 1.8 mutations per megabase, 18 copy number alterations (CNAs) and 37 structural variants (SVs) that contrasted with 1.1 mutations per megabase, 4 CNAs and 5 SVs observed at CLL diagnosis. No major differences were seen among RT occurring after different therapies (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig7">2b</a>). We discovered new driver genes and mechanisms in RT, expanding previous observations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Scandurra, M. et al. Genomic profiling of Richter’s syndrome: recurrent lesions and differences with de novo diffuse large B-cell lymphomas. Hematol. Oncol. 28, 62–67 (2010)." href="#ref-CR12" id="ref-link-section-d73061417e1526">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rossi, D. et al. The genetics of Richter syndrome reveals disease heterogeneity and predicts survival after transformation. Blood 117, 3391–3401 (2011)." href="#ref-CR13" id="ref-link-section-d73061417e1526_1">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Fabbri, G. et al. Genetic lesions associated with chronic lymphocytic leukemia transformation to Richter syndrome. J. Exp. Med. 210, 2273–2288 (2013)." href="#ref-CR14" id="ref-link-section-d73061417e1526_2">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chigrinova, E. et al. Two main genetic pathways lead to the transformation of chronic lymphocytic leukemia to Richter syndrome. Blood 122, 2673–2682 (2013)." href="#ref-CR15" id="ref-link-section-d73061417e1526_3">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Klintman, J. et al. Genomic and transcriptomic correlates of Richter transformation in chronic lymphocytic leukemia. Blood 137, 2800–2816 (2021)." href="#ref-CR16" id="ref-link-section-d73061417e1526_4">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chakraborty, S. et al. B-cell receptor signaling and genetic lesions in TP53 and CDKN2A/CDKN2B cooperate in Richter transformation. Blood 138, 1053–1066 (2021)." href="#ref-CR17" id="ref-link-section-d73061417e1526_5">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Anderson, M. A. et al. Clinicopathological features and outcomes of progression of CLL on the BCL2 inhibitor venetoclax. Blood 129, 3362–3370 (2017)." href="/articles/s41591-022-01927-8#ref-CR18" id="ref-link-section-d73061417e1529">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Herling, C. D. et al. Clonal dynamics towards the development of venetoclax resistance in chronic lymphocytic leukemia. Nat. Commun. 9, 727 (2018)." href="#ref-CR21" id="ref-link-section-d73061417e1532">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Villamor, N. et al. NOTCH1 mutations identify a genetic subgroup of chronic lymphocytic leukemia patients with high risk of transformation and poor outcome. Leukemia 27, 1100–1106 (2013)." href="#ref-CR22" id="ref-link-section-d73061417e1532_1">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="De Paoli, L. et al. MGA, a suppressor of MYC, is recurrently inactivated in high risk chronic lymphocytic leukemia. Leuk. Lymphoma 54, 1087–1090 (2013)." href="#ref-CR23" id="ref-link-section-d73061417e1532_2">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Rossi, D. et al. Different impact of NOTCH1 and SF3B1 mutations on the risk of chronic lymphocytic leukemia transformation to Richter syndrome. Br. J. Haematol. 158, 426–429 (2012)." href="/articles/s41591-022-01927-8#ref-CR24" id="ref-link-section-d73061417e1535">24</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1c</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig7">2c–e</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM1">1</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01927-8#Sec63">9</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">10</a>). The main alterations involved cell-cycle regulators (17 of 19, 89%), chromatin modifiers (79%), MYC (74%), nuclear factor (NF)-κB (74%) and NOTCH (32%) pathways. These aberrations were simultaneously present in most cases but alterations in MYC and NOTCH pathways only co-occurred in 2 of 19 cases (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1c</a>). Aberrations in genes such as <i>TP53</i>, <i>NOTCH1</i>, <i>BIRC3</i>, <i>EGR2</i> and <i>NFKBIE</i> were usually present and clonally dominant after the first CLL sample, whereas others were only detected at RT or during the disease course (for example <i>CDKN2A/B</i>, <i>CDKN1A/B</i>, <i>ARID1A</i>, <i>CREBBP</i>, <i>TRAF3</i> and <i>TNFAIP3</i>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1c</a>). New alterations included deletions of <i>CDKN1A</i> and <i>CDKN1B</i> in five cases of RT associated with downregulation of their expression, one immunoglobulin (IG)-<i>CDK6</i> translocation and one <i>CCND2</i> mutation already present at CLL diagnosis, and <i>CCND3</i>-IG and <i>MYCN</i>-IG translocations acquired at RT in two different cases (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1d,e</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig8">3a,b</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">11</a>). Most chromatin remodelers were affected by deletions with reduced gene expression. New alterations in this group were deletions of <i>ARID4B</i> and truncations of <i>CREBBP</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Chitalia, A. et al. Descriptive analysis of genetic aberrations and cell of origin in Richter transformation. Leuk. Lymphoma 60, 971–979 (2019)." href="/articles/s41591-022-01927-8#ref-CR25" id="ref-link-section-d73061417e1630">25</a></sup> and <i>SMARCA4</i> (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Klintman, J. et al. Genomic and transcriptomic correlates of Richter transformation in chronic lymphocytic leukemia. Blood 137, 2800–2816 (2021)." href="/articles/s41591-022-01927-8#ref-CR16" id="ref-link-section-d73061417e1637">16</a></sup>) by translocations and chromoplexy (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig8">3c–e</a>). We also identified recurrent <i>IRF4</i> alterations in RT, which have been linked to increased MYC levels in CLL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Benatti, S. et al. IRF4 L116R mutation promotes proliferation of chronic lymphocytic leukemia B cells inducing MYC. Hematol. Oncol. 39, 707–711 (2021)." href="/articles/s41591-022-01927-8#ref-CR26" id="ref-link-section-d73061417e1651">26</a></sup>. <i>BTK</i>/<i>PLCG2</i> or <i>BCL2</i> mutations were not detected in any RT after treatment with BCR or BCL2 inhibitors, respectively. Notably, the two cases of M-CLL developing RT after targeted therapies carried the IGLV3–21<sup>R110</sup> mutation, which triggers cell-autonomous BCR signaling<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Minici, C. et al. Distinct homotypic B-cell receptor interactions shape the outcome of chronic lymphocytic leukaemia. Nat. Commun. 8, 15746 (2017)." href="/articles/s41591-022-01927-8#ref-CR27" id="ref-link-section-d73061417e1667">27</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1c</a>).</p><p>In addition to the high frequency of CNAs previously identified in RT<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Beà, S. et al. Genetic imbalances in progressed B-cell chronic lymphocytic leukemia and transformed large-cell lymphoma (Richter’s syndrome). Am. J. Pathol. 161, 957–968 (2002)." href="/articles/s41591-022-01927-8#ref-CR11" id="ref-link-section-d73061417e1677">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Fabbri, G. et al. Genetic lesions associated with chronic lymphocytic leukemia transformation to Richter syndrome. J. Exp. Med. 210, 2273–2288 (2013)." href="/articles/s41591-022-01927-8#ref-CR14" id="ref-link-section-d73061417e1680">14</a></sup>, we observed a high number of complex structural alterations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1c</a>). Chromothripsis was found in eight RT tumors targeting <i>CDKN2A/B</i> and the new <i>CDKN1B</i> in five and one cases, respectively, and <i>MYC</i>, <i>MGA</i>, <i>SPEN</i>, <i>TNFAIP3</i> and chromatin remodeling genes in additional cases (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig8">3f–j</a>).</p><p>Altogether, our analyses expand the catalog of driver genes, pathways and mechanisms involved in RT and recognize a similar distribution of these alterations in RT after different therapies, suggesting that treatment-specific pressure is not a major determinant of the driver genomic landscape of these tumors.</p><h3 class="c-article__sub-heading" id="Sec4">New mutational processes in RT</h3><p>To understand the increased mutational burden of RT, we explored the mutational processes re-shaping the genome of CLL and RT. An unsupervised analysis showed that the mutational profile of RT was notably different from M-CLL and U-CLL before therapy (ICGC-CLL cohort, <i>n</i> = 147)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Puente, X. S. et al. Non-coding recurrent mutations in chronic lymphocytic leukaemia. Nature 526, 519–524 (2015)." href="/articles/s41591-022-01927-8#ref-CR28" id="ref-link-section-d73061417e1726">28</a></sup> or at post-treatment relapse (independent cohort of 27 CLL post-treatment samples) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2a</a>). We identified 11 mutational signatures distributed genome-wide and 2 in clustered mutations (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig9">4</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01927-8#Sec63">12</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">14</a>). Among the former, we extracted a new signature characterized by (T&gt;A)A and, to a lesser extent, (T&gt;C/G)A mutations not recognized previously in any cancer type, including CLL and DLBCL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Puente, X. S. et al. Non-coding recurrent mutations in chronic lymphocytic leukaemia. Nature 526, 519–524 (2015)." href="#ref-CR28" id="ref-link-section-d73061417e1743">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kasar, S. et al. Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution. Nat. Commun. 6, 8866 (2015)." href="#ref-CR29" id="ref-link-section-d73061417e1743_1">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Maura, F. et al. A practical guide for mutational signature analysis in hematological malignancies. Nat. Commun. 10, 2969 (2019)." href="#ref-CR30" id="ref-link-section-d73061417e1743_2">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Arthur, S. E. et al. Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma. Nat. Commun. 9, 4001 (2018)." href="#ref-CR31" id="ref-link-section-d73061417e1743_3">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)." href="#ref-CR32" id="ref-link-section-d73061417e1743_4">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Kucab, J. E. et al. A compendium of mutational signatures of environmental agents. Cell 177, 821–836 (2019)." href="/articles/s41591-022-01927-8#ref-CR33" id="ref-link-section-d73061417e1746">33</a></sup>. We named this single-base substitution signature, SBS-RT (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2b</a>). SBS-RT was present in the RT sample of 7 of 18 patients, 1 of 6 after CIT and 6 of 10 after multiple therapies, including targeted agents and detected in all subtypes of transformation (RT-DLBCL, RT-PBL and RT-PLL) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2c</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">15</a>). It was also present in CLL samples before RT in patients 12 and 3,299 but was not identified in the reanalysis of our ICGC-CLL or post-treatment CLL cohorts. None of the patients in these two additional cohorts had evidence of RT (median follow-up 9.8 years, range 0.2–30.4) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2c</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig10">5a</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">15</a>). Further characterization of this new signature showed (1) a modest correlation between SBS-RT and total number of mutations (<i>R</i> = 0.79, <i>P</i> = 0.11); (2) SBS-RT mutations present in all different chromatin states and early/late replicating regions although with a moderate enrichment in heterochromatin/late replication; and (3) lack of replication and transcriptional strand bias (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig10">5b–f</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">16</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Mutational processes in RT."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Mutational processes in RT.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01927-8/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="644"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Principal component analysis (PCA) of the 96-mutational profile of CLL and RT. <b>b</b>, Signatures identified de novo in CLL/RT not reported in COSMIC. The main peaks of each signature are labeled in black. <b>c</b>, Contribution of mutational processes in CLL/RT. RT time points are marked in a rose color. B, peripheral blood; L, lymph node; M, bone marrow; (M), M-CLL. <b>d</b>, Therapies received before RT and presence/absence of SBS-melphalan, SBS-ganciclovir and SBS-RT at time of RT for each patient. mAB, monoclonal antibody; TBI, total body irradiation; Inh., inhibitor; Sig., signatures. <b>e</b>, Phylogenetic relationship of subclones and contribution of each mutational signature to their mutational profile. <b>f</b>, Relative contribution of mutational processes in CLL (no. 1) and RT subclones (top). Number of mutations (muts) in RT subclones (bottom). w/, with. <b>g</b>, Detection (top) and variant allele frequency (VAF) (bottom) of mutations assigned to the RT subclone during the disease course in patients 12 and 63 by high-coverage UMI-based NGS. Mutations are grouped according to the main peaks of SBS-RT. <i>P</i> values were obtained by Fisher’s test. LC, low confidence; HC, high confidence; NA, not available. <b>h</b>, Distribution of the CCF of the single-nucleotide variants (SNVs) assigned to the RT subclone based on WGS and stratified according to the main peaks of the SBS-RT. <b>i</b>, Relative contribution of mutational processes in regions of kataegis in CLL and RT (left). Two cases acquiring mutations in the immunoglobulin genes at time of RT (right). <b>j</b>, Clonal evolution along the disease course in patient 12 inferred from WGS. Abbreviations for treatment regimens are detailed in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig6">1a</a>. Each subclone is depicted by a different color and number and its CCF is proportional to its height in each time point (vertical line). The phylogeny of the subclones with the main driver alterations is shown (top). Flow cytometry analysis for time points (T) 4, 5 and 6 (bottom). The size of the cells (forward scatter (FSC) versus side scatter (SSC), first row) and the expression levels of CD20 and CD38 (second row) differentiated CLL cells (yellowish) and the two larger size tumor populations (pale and dark rose color, respectively). Numbers along axes are divided by 1,000.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01927-8/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Among the remaining ten genome-wide signatures, five were previously identified in CLL and DLBCL (SBS1 and SBS5 (clock-like), SBS8 (unknown etiology), SBS9 (attributed to polymerase eta) and SBS18 (possibly damage by reactive oxygen species)); three had been only found in DLBCL (SBS2 and SBS13 (APOBEC enzymes) and SBS17b (unknown)); and two have been recently described related to treatments with melphalan<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Rustad, E. H. et al. Timing the initiation of multiple myeloma. Nat. Commun. 11, 1917 (2020)." href="/articles/s41591-022-01927-8#ref-CR34" id="ref-link-section-d73061417e1841">34</a></sup> or ganciclovir<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="de Kanter, J. K. et al. Antiviral treatment causes a unique mutational signature in cancers of transplantation recipients. Cell Stem Cell 28, 1726–1739 (2021)." href="/articles/s41591-022-01927-8#ref-CR35" id="ref-link-section-d73061417e1845">35</a></sup>, which were named here as SBS-melphalan and SBS-ganciclovir, respectively (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2b,c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig9">4</a>). SBS-melphalan was found in three RT cases, two had received melphalan as a conditioning of their allogenic stem-cell transplant 1.9 and 4.2 years before RT, respectively. SBS-ganciclovir was found in the RT sample of one patient that had received valganciclovir (prodrug of ganciclovir) due to cytomegalovirus reactivation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2c,d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig6">1a</a>). Notably, all cases with the new SBS-RT at time of RT had been treated with the alkylating agents bendamustine (<i>n</i> = 5) or chlorambucil (<i>n</i> = 2) during their CLL history at a median of 2.9 years (range 0.7 to 6.8) before RT. Contrarily, RT cases lacking the SBS-RT had never received these drugs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2c,d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig6">1a</a>).</p><p>To time the activity of each mutational process, we reconstructed the phylogenetic tree for the 11 patients with multiple synchronous (<i>n</i> = 2) or longitudinal (<i>n</i> = 9) samples and germline available and measured the contribution of each signature to the mutational profile of each subclone. The major subclone at time of transformation was named ‘RT subclone’ (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">17</a>). As expected, clock-like mutational signatures were present all along the phylogeny (constantly acquired), whereas SBS9 was found only in the trunk of the two M-CLL tumors (patients 365 and 19; early events). DLBCL-related signatures, SBS-ganciclovir, SBS-melphalan and SBS-RT were found in single RT subclones in six cases while two cases carried two simultaneous subclones with SBS-RT (patients 12 and 19) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2e</a>). SBS-RT represented 28.6% of the mutations acquired in RT (mean 679, range 499–1,167) and it was occasionally associated with coding mutations in driver genes (<i>EP300</i> and <i>CIITA</i>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2f</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig10">5g</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">16</a>). By applying a high-coverage, unique molecular identifier (UMI)-based next-generation sequencing (NGS) approach in longitudinal samples of patients 12, 19 and 63 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">18</a>), we observed that mutations of the RT subclones found in the main peaks of the SBS-RT were mainly identified in samples collected after bendamustine or chlorambucil therapy, whereas mutations not associated with SBS-RT were detected earlier during the disease course (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig10">5h</a>). These results suggest a causal link between the exposure to these drugs and SBS-RT. The finding of SBS-melphalan, SBS-ganciclovir and SBS-RT in RT argues in favor of a single-cell expansion model for RT; a single cell that can carry the footprints of cancer therapies (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2h</a>). Contrarily, the lack of SBS-RT in the 27 post-treatment CLL samples (7 patients treated with bendamustine or chlorambucil) suggests that CLL relapse might be driven by the simultaneous expansion of different subclones, hindering the detection of SBS-RT through bulk sequencing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Rustad, E. H. et al. Timing the initiation of multiple myeloma. Nat. Commun. 11, 1917 (2020)." href="/articles/s41591-022-01927-8#ref-CR34" id="ref-link-section-d73061417e1918">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Pich, O. et al. The mutational footprints of cancer therapies. Nat. Genet. 51, 1732–1740 (2019)." href="/articles/s41591-022-01927-8#ref-CR36" id="ref-link-section-d73061417e1921">36</a></sup>.</p><p>RT subclones also acquired kataegis, mainly within the immunoglobulin loci, attributed to activation-induced cytidine deaminase (AID) activity (SBS84 and SBS85)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Kasar, S. et al. Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution. Nat. Commun. 6, 8866 (2015)." href="/articles/s41591-022-01927-8#ref-CR29" id="ref-link-section-d73061417e1928">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)." href="/articles/s41591-022-01927-8#ref-CR32" id="ref-link-section-d73061417e1931">32</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2i</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig9">4</a>). These kataegis led to the acquisition of mutations in the rearranged V(D)J gene in five RT cases (one after CIT and four targeted therapies) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2i</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig10">5i,j</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">19</a>). This canonical AID activity in RT is concordant with the acquisition of SBS9 mutations in two RT samples (4,686 (CIT) and 3,495 (targeted therapies)) and SVs mediated by aberrant class-switch recombination or somatic hypermutation in six RT (one before therapy, two CIT and three new agents), which targeted <i>MYC</i>, <i>MYCN</i>, <i>TRAF3</i> and <i>CCND3</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1c</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">2</a>).</p><p>SBS-RT mutations were found in CLL samples before the transformation in patient 3,299 although it was only present in the RT subclone (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2c,e</a>). SBS-RT was also found in two different subclones in case 12 and 19. We speculated that these secondary subclones with SBS-RT (named ‘RT-like’ subclones) could correspond to the single-cell expansion of a ‘transformed’ cell that could have been missed by the routine analysis (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2e</a>). The reanalysis of flow cytometry data available for case 12 detected two cell populations at time point (T) 4 differing in size and surface markers (likely CLL and RT-like subclones), whereas at T5 we detected an additional population of large cells (RT subclone, 0.2% cells) that expanded at T6, substituting the previous large cell population (RT-like subclone) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2j</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig10">5k–m</a>). WGS analysis showed that the RT-like and RT subclones diverged from a cell carrying a deletion of <i>CDKN2A/B</i> and truncation of <i>CREBBP</i>, each acquiring more than 2,100 specific mutations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2e,j</a>).</p><p>Altogether, these findings show that RT may arise simultaneously from different subclones and that such subclones can be detectable time before their final expansion and clinical manifestation. The identification of mutations in RT associated with early-in-time CLL therapies demonstrates that RT emerges from the clonal expansion of a single cell previously exposed to these therapies.</p><h3 class="c-article__sub-heading" id="Sec5">Dormant seeds of RT at CLL diagnosis</h3><p>The WGS-based subclonal phylogeny of the nine patients with fully characterized longitudinal samples predicted that the RT subclone was present at low cancer cell fraction (CCF) in the preceding CLL samples in five (56%) patients and only detected at time of transformation in the remaining four (44%) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3a</a>). Indeed, the RT subclone was detected at time of CLL diagnosis in three of five patients, remained stable at a minute size (&lt;1%) for 6–19 years of natural and treatment-influenced CLL course and expanded at the moment of clinical manifestations (patients 12, 19 and 63) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3a</a>). In the other two patients, the RT subclone was also detected in the first CLL sample analyzed but rapidly expanded driving the RT 0.6 and 3.5 years later in patients 3,034 and 3,299 (RT-PLL), respectively (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3a</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig11">6</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Early seeding of RT."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Early seeding of RT.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01927-8/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="506"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Evolution of the RT subclone along the disease course based on WGS. Time lapse between the first and last sample analyzed (bottom). RT time points are marked in a rose color. Summary of the three patterns observed (right). <b>b</b>, Fish plot showing the clonal evolution along the course of the disease in patient 19 inferred from WGS analysis. Each subclone is depicted by a different color and number and its CCF is proportional to its height at each time point (vertical lines). Phylogeny of the subclones and main driver events (right). <b>c</b>, Mutation tree reconstructed by scDNA-seq for case 19 together with the fraction of cells carrying each specific combination of mutations in each time point. The total number of cells per sample is shown at the bottom. The number of cells assigned to each subclone is shown in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">20</a>. <b>d</b>, Schematic representation of the clinical course and samples analyzed for patient 3,495 together with the size of the IGH subclones identified using high-coverage NGS analyses. Abbreviations for treatment regimens are detailed in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig6">1a</a>. <b>e</b>, Clinical course and IGH subclones identified by DNA- and RNA-based NGS in patient 12. <b>f</b>, Uniform Manifold and Projection (UMAP) plot for case 12 based on the scRNA-seq data of all time points colored by annotation. <b>g</b>, Expression of key marker genes in each cluster identified in case 12. <b>h</b>, Distribution of cell-cycle phase scores for each cluster based on scRNA-seq in case 12. <b>i</b>, UMAP visualization split by time point in case 12 with the fraction of RT cells annotated. ‘<i>n</i>’, number of cells. <b>j</b>, Chromosomal alterations detected by WGS in chromosomes 1, 11 and 14 in CLL and RT samples of patient 12 (top). Copy number profile of RT cells detected at the different time points according to scRNA-seq. Only a subset of RT cells from time point 6 (time of diagnosis of RT) was included for illustrative purposes (bottom).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01927-8/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We next performed single-cell DNA sequencing (scDNA-seq) of 32 genes in 16 longitudinal samples of 4 patients (12, 19, 365 and 3,299) to validate these evolutionary histories of RT (202,210 cells passing filters, mean of 12,638 cells per sample; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1a</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM1">2</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">20</a>). Focusing on patient 19 with a time lapse of 14.4 years from diagnosis to RT (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3b</a>), the RT subclone (subclone 5) at transformation (T6) carried <i>CDKN2A/B</i> and <i>TP53</i> (p.G245D) alterations, whereas the main CLL subclones driving the relapse after therapy at T4 and T5 harbored a different <i>TP53</i> mutation (p.I195T; subclones 3 and 4). The WGS predicted the presence of all these subclones at CLL diagnosis (T1). Using scDNA-seq we identified two small populations accounting for 0.1% of cells carrying the <i>TP53</i> p.I195T and p.G245D mutations, respectively, at T1, which were also detected at relapse 7.2 years later (T3). The subclone carrying <i>TP53</i> p.I195T expanded to dominate the second relapse after 3.7 years at T4 and T5 but was substituted by the subclone carrying <i>TP53</i> p.G245D at T6 in the RT 14.4 years after diagnosis. All these subclones carried the <i>SF3B1</i> and <i>NOTCH1</i> mutations of the initial CLL subclone (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3c</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">20</a>). The scDNA-seq of the three additional cases also corroborated the phylogenies and most of the dynamics inferred from WGS (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig11">6a</a>). These results suggest that CLL evolution to RT is characterized by an early driver diversification probably generated before diagnosis, consistent with the early immunogenetic and DNA methylation diversification previously reported in CLL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gaiti, F. et al. Epigenetic evolution and lineage histories of chronic lymphocytic leukaemia. Nature 569, 576–580 (2019)." href="#ref-CR37" id="ref-link-section-d73061417e2129">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gemenetzi, K. et al. Higher-order immunoglobulin repertoire restrictions in CLL: the illustrative case of stereotyped subsets 2 and 169. Blood 137, 1895–1904 (2021)." href="#ref-CR38" id="ref-link-section-d73061417e2129_1">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bagnara, D. et al. Post-transformation IGHV-IGHD-IGHJ mutations in chronic lymphocytic leukemia B cells: implications for mutational mechanisms and impact on clinical course. Front. Oncol. 11, 1769 (2021)." href="/articles/s41591-022-01927-8#ref-CR39" id="ref-link-section-d73061417e2132">39</a></sup> and that RT may emerge by a selection of pre-existing subclones carrying potent driver mutations rather than a de novo acquisition of leading clones.</p><p>As we identified five cases of RT carrying specific mutations in the immunoglobulin genes by WGS (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig2">2i</a>), we analyzed whether these immunoglobulin-based RT subclones were already present at CLL diagnosis using high-coverage NGS in patients 12 and 3,495 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">21</a>). Focusing on patient 3,495, for which the lack of germline material precluded our phylogenetic analyses, the RT occurring after treatment with ibrutinib harbored two new V(D)J mutations generating an unproductive IGH gene. NGS identified 0.002% sequences carrying the same two mutations at CLL diagnosis 1.72 years before (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3d</a>). We also observed the expansion of additional unproductive subclones accounting for 11.8% of all sequences at time of RT, suggesting that BCR-independent subclones may have a proliferative advantage under therapy with BCR inhibitors (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3d</a>). Similar results were found in patient 12 in which the V(D)J sequence of RT carrying a new mutation was already identified at CLL diagnosis 19.5 years before at DNA and RNA level (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3e</a>). As the immunogenetic features represent a faithful imprint of the B cell of origin, the early identification of the same immunogenetic subclone provides further evidence for an early seeding of RT.</p><p>We finally tracked RT subclones during the disease course using single-cell RNA sequencing (scRNA-seq) of 19 longitudinal samples of five patients (24,800 tumor cells passing filters, mean of 1,305 cells per sample; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1a</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">22</a>). As expected, RT and CLL cells had remarkably different gene expression profiles (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig12">7a–d</a>). The transcriptome of CLL cells was dominated by three main clusters identified across patients and characterized by different expression of <i>CXCR4</i>, <i>CD27</i> and <i>MIR155HG</i>, respectively, which may represent the recirculation of CLL cells between peripheral blood and lymph nodes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Calissano, C. et al. In vivo intraclonal and interclonal kinetic heterogeneity in B-cell chronic lymphocytic leukemia. Blood 114, 4832–4842 (2009)." href="#ref-CR40" id="ref-link-section-d73061417e2179">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Calissano, C. et al. Intraclonal complexity in chronic lymphocytic leukemia: fractions enriched in recently born/divided and older/quiescent cells. Mol. Med. 17, 1374–1382 (2011)." href="#ref-CR41" id="ref-link-section-d73061417e2179_1">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Cui, B. et al. MicroRNA-155 influences B-cell receptor signaling and associates with aggressive disease in chronic lymphocytic leukemia. Blood 124, 546–554 (2014)." href="/articles/s41591-022-01927-8#ref-CR42" id="ref-link-section-d73061417e2182">42</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3f,g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig12">7a–d</a>). Contrarily, RT intraclonal heterogeneity was mainly related to distinct proliferative capacities with a cluster of cells showing high <i>MKI67</i> and <i>PCNA</i> expression as well as high S and G2M cell-cycle phase scores. The remaining RT clusters were characterized by the expression of different marker genes among patients, including <i>CCND2</i>, <i>MIR155HG</i> and <i>TP53INP1</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3f–h</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig12">7a–d</a>). When considering each time point separately, we detected RT cells in all CLL samples before transformation in patient 12, 19, 63 and 3,299 but not in patient 365 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3i</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig12">7a–i</a>). The presence and dynamics of these RT subclones according to their transcriptomic profile recapitulated the findings obtained by WGS, scDNA-seq and immunoglobulin analyses in all five patients, suggesting that they captured the same cells. Indeed, using scRNA-seq we could identify the CNAs involved in simple and complex structural alterations found at time of RT by WGS already in the dormant RT cells at CLL diagnosis and subsequent time points before their final expansion (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig3">3j</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig13">8</a>). These findings suggest an early acquisition of SVs, including chromothripsis and transcriptomic identity in RT.</p><p>To validate our observations, we reanalyzed the longitudinal scRNA-seq dataset from Penter et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Penter, L. et al. Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history. Cancer Discov. 11, 3048–3063 (2021)." href="/articles/s41591-022-01927-8#ref-CR43" id="ref-link-section-d73061417e2231">43</a></sup> consisting of nine patients with CLL, one of which developed RT. In this case, we identified RT cells in the CLL sample collected 1.6 years before the RT (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig12">7j</a>). Overall, our integrative analyses uncovered a widespread early seeding of RT cells up to 19 years before their expansion and clinical manifestation.</p><h3 class="c-article__sub-heading" id="Sec6">OXPHOS<sup>high</sup>–BCR<sup>low</sup> transcriptional axis of RT</h3><p>To understand the transcriptomic evolution from CLL to RT and its epigenomic regulation, we integrated genome-wide profiles of DNA methylation, chromatin activation (H3K27ac) and chromatin accessibility (ATAC-seq) with bulk RNA-seq and scRNA-seq of multiple longitudinal samples of six patients treated with BCR inhibitors (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig1">1a</a>). The DNA methylome of RT mainly reflected the naive and memory-like B cell derivation of their CLL counterpart, whereas chromatin activation and accessibility were remarkably different upon transformation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4a</a>). We identified 150 regions with increased H3K27ac and 426 regions that gained accessibility in RT (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4b</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig14">9a</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01927-8#Sec63">7</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">8</a>). These de novo active regions were enriched in transcription factor (TF) families different from those known to modulate the epigenome of CLL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Beekman, R. et al. The reference epigenome and regulatory chromatin landscape of chronic lymphocytic leukemia. Nat. Med. 24, 868–880 (2018)." href="/articles/s41591-022-01927-8#ref-CR44" id="ref-link-section-d73061417e2270">44</a></sup>. Among them, 24 were enriched and upregulated in RT (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">7</a>). The top TF was TEAD4, which activates genes involved in oxidative phosphorylation (OXPHOS) through the mTOR pathway<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Chen, C.-L. et al. Arginine is an epigenetic regulator targeting TEAD4 to modulate OXPHOS in prostate cancer cells. Nat. Commun. 12, 2398 (2021)." href="/articles/s41591-022-01927-8#ref-CR45" id="ref-link-section-d73061417e2277">45</a></sup> and co-operates with MYCN<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Rajbhandari, P. et al. Cross-cohort analysis identifies a TEAD4–MYCN positive feedback loop as the core regulatory element of high-risk neuroblastoma. Cancer Discov. 8, 582–599 (2018)." href="/articles/s41591-022-01927-8#ref-CR46" id="ref-link-section-d73061417e2281">46</a></sup>. Additional TFs were related to MYC (MAZ), proliferation/cell cycle (E2F family) or IRF family, among others (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4c</a>). Notably, high IRF4 levels seem to attenuate BCR signaling in CLL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Maffei, R. et al. IRF4 modulates the response to BCR activation in chronic lymphocytic leukemia regulating IKAROS and SYK. Leukemia 35, 1330–1343 (2021)." href="/articles/s41591-022-01927-8#ref-CR47" id="ref-link-section-d73061417e2289">47</a></sup>, whereas they are necessary to induce MYC target genes, OXPHOS and glycolysis in activated healthy B cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Patterson, D. G. et al. An IRF4–MYC–mTORC1 integrated pathway controls cell growth and the proliferative capacity of activated B cells during B cell differentiation in vivo. J. Immunol. 207, 1798–1811 (2021)." href="/articles/s41591-022-01927-8#ref-CR48" id="ref-link-section-d73061417e2293">48</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Proliferation, OXPHOS and BCR pathways dominate the epigenome and transcriptome of RT."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Proliferation, OXPHOS and BCR pathways dominate the epigenome and transcriptome of RT.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01927-8/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="734"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, PCA of the bulk epigenetic and transcriptomic layers analyzed. <b>b</b>, Heat map showing 150 regions with increased H3K27ac levels in RT. <b>c</b>, TF enriched within the ATAC peaks identified in the regions of increase H3K27ac in RT. The motif, percentage of RT-specific active regions and regions with increased H3K27ac in CLL that contained the motif and TF expression (bulk RNA-seq) in CLL and RT are shown. Center line indicates median; box limits indicate upper and lower quartiles; whiskers indicate 1.5 × interquartile range; points indicate individual samples. <i>P</i> values were derived using a one-tailed Wilcoxon rank-sum test. <b>d</b>, Heat map showing the DEGs between CLL and RT identified by bulk RNA-seq. Samples used in the differential expression analysis (DEA) are indicated. The overlap of DEGs with DNA methylation changes, H3K27ac and ATAC peaks is shown on the right. Selected genes are annotated. <b>e</b>, Intersection of upregulated genes in RT compared to CLL in scRNA-seq analyses. <b>f</b>, epiCMIT evolution from CLL to RT. <i>P</i> values were derived by paired Wilcoxon signed-rank test. <b>g</b>, Summary of the main gene sets modulated in RT based on bulk RNA-seq. NES, normalized enrichment score; ROS, reactive oxygen species. <b>h</b>, Gene set enrichment plot for OXPHOS and BCR signaling (bulk RNA-seq). <b>i</b>, OXPHOS and BCR signaling scores depicted at single-cell level for case 12 (all time points together). RT and CLL cells are highlighted (left). Ridge plots show the OXPHOS and BCR score across clusters (right). <b>j</b>, OXPHOS and BCR signaling scores of CLL and RT cells of patient 12 across time points by scRNA-seq. <b>k</b>, Distribution of OXPHOS and BCR signaling scores at a single-cell level across different time points of nine cases included in the study of Penter et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Penter, L. et al. Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history. Cancer Discov. 11, 3048–3063 (2021)." href="/articles/s41591-022-01927-8#ref-CR43" id="ref-link-section-d73061417e2349">43</a></sup>. Center line indicates median; box limits indicate upper and lower quartiles; whiskers indicate 1.5 × interquartile range; points indicate outliers. B, peripheral blood; M, bone marrow. *Sample collected under treatment with ibrutinib.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01927-8/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>The RNA-seq analysis, excluding cases 19 and 3,299 (RT-PLL) due to their intermediate transcriptomic profile, identified 2,248 differentially expressed genes (DEGs) between RT and CLL (1,439 upregulated and 809 downregulated) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4a,d,e</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig15">10a</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01927-8#Sec63">11</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">23</a>). A remarkable fraction of upregulated/downregulated genes overlapped with regions with the respective increase/decrease of H3K27ac (20%) and chromatin accessibility (16%) at RT (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig14">9b</a>). Contrarily, only 4% of the DEGs overlapped with any of the 2,341 differentially methylated CpGs (DMCs) between RT and CLL, emphasizing the limited effect of DNA methylation on gene regulation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Duran-Ferrer, M. et al. The proliferative history shapes the DNA methylome of B-cell tumors and predicts clinical outcome. Nat. Cancer 1, 1066–1081 (2020)." href="/articles/s41591-022-01927-8#ref-CR49" id="ref-link-section-d73061417e2383">49</a></sup>. Most DMCs were hypomethylated at RT (2,112 of 2,341; 90%), found in open sea and intergenic regions and correlated with the proliferative history of the cells measured by the epiCMIT score<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Duran-Ferrer, M. et al. The proliferative history shapes the DNA methylome of B-cell tumors and predicts clinical outcome. Nat. Cancer 1, 1066–1081 (2020)." href="/articles/s41591-022-01927-8#ref-CR49" id="ref-link-section-d73061417e2387">49</a></sup> (1,681; 72%), which increased during CLL evolution and at RT (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4d,f</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig14">9c–g</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">6</a>).</p><p>Genes upregulated in RT involved pathways that seem independent of BCR signaling such as Wnt (<i>WNT5A</i> and others)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Hasan, M. K., Ghia, E. M., Rassenti, L. Z., Widhopf, G. F. &amp; Kipps, T. J. Wnt5a enhances proliferation of chronic lymphocytic leukemia and ERK1/2 phosphorylation via a ROR1/DOCK2-dependent mechanism. Leukemia 35, 1621–1630 (2021)." href="/articles/s41591-022-01927-8#ref-CR50" id="ref-link-section-d73061417e2406">50</a></sup>, Toll-like receptors (<i>TLR9</i> among others)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Ntoufa, S., Vilia, M. G., Stamatopoulos, K., Ghia, P. &amp; Muzio, M. Toll-like receptors signaling: a complex network for NF-κB activation in B-cell lymphoid malignancies. Semin. Cancer Biol. 39, 15–25 (2016)." href="/articles/s41591-022-01927-8#ref-CR51" id="ref-link-section-d73061417e2413">51</a></sup> and a number of cyclin-dependent kinases. Downregulated genes included, among others, <i>CXCR4</i>, <i>HLA-A/B</i> and chromatin remodelers also targeted by genetic alterations in some cases (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig15">10b,c</a>). Gene sets modulated by gene expression in RT were in harmony with the identified chromatin-based changes and included upregulation of E2F targets, G2M checkpoints, MYC targets, MTORC1 signaling, OXPHOS, mitochondrial translation, glycolysis, reactive oxygen species and DNA repair pathways, among others. In addition, RT showed downmodulation of BCR signaling (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4g,h</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig15">10d</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">11</a>). The OXPHOS<sup>high</sup>–BCR<sup>low</sup> pattern observed by bulk RNA-seq in RT was further refined using scRNA-seq: two of five tumors had OXPHOS<sup>high</sup>–BCR<sup>low</sup> (12 and 63, although the latter showed some intercluster variability), the two M-CLL carrying IGLV3–21<sup>R110</sup> had RT with BCR expression similar to CLL and were OXPHOS<sup>high</sup>–BCR<sup>normal</sup> (365) or OXPHOS<sup>normal</sup>–BCR<sup>normal</sup> (19) and the RT-PLL (3,299) was OXPHOS<sup>low</sup>–BCR<sup>low</sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4i</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig15">10e–j</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">23</a>). In addition, the scRNA-seq analysis showed that the OXPHOS/BCR profiles of RT were already identified in the early dormant RT cells, suggesting that they might represent an intrinsic characteristic of RT cells rather than being modulated by BCR inhibitors (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4j</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig15">10g–j</a>). To expand these observations, we measured the expression of OXPHOS and BCR pathways in the scRNA-seq dataset from Penter et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Penter, L. et al. Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history. Cancer Discov. 11, 3048–3063 (2021)." href="/articles/s41591-022-01927-8#ref-CR43" id="ref-link-section-d73061417e2479">43</a></sup>. Case CLL9, which developed RT in the absence of any therapy, showed a remarkably higher OXPHOS and slightly lower BCR expression at time of RT compared to CLL (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig4">4k</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig15">10k,l</a>).</p><p>Overall, the epigenome and transcriptome of RT converge to an OXPHOS<sup>high</sup>–BCR<sup>low</sup> axis reminiscent of that observed in the de novo DLBCL subtype characterized by high OXPHOS (DLBCL-OXPHOS) and insensitive to BCR inhibition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Monti, S. Molecular profiling of diffuse large B-cell lymphoma identifies robust subtypes including one characterized by host inflammatory response. Blood 105, 1851–1861 (2005)." href="#ref-CR52" id="ref-link-section-d73061417e2497">52</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Caro, P. et al. Metabolic signatures uncover distinct targets in molecular subsets of diffuse large B cell lymphoma. Cancer Cell 22, 547–560 (2012)." href="#ref-CR53" id="ref-link-section-d73061417e2497_1">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Norberg, E. et al. Differential contribution of the mitochondrial translation pathway to the survival of diffuse large B-cell lymphoma subsets. Cell Death Differ. 24, 251–262 (2017)." href="/articles/s41591-022-01927-8#ref-CR54" id="ref-link-section-d73061417e2500">54</a></sup>. This axis might explain the selection and rapid expansion of small RT subclones under therapy with BCR inhibitors.</p><h3 class="c-article__sub-heading" id="Sec7">OXPHOS and BCR activity in RT</h3><p>We next validated experimentally the OXPHOS and BCR activity of RT in samples of patients 12, 19 and 63. Respirometry assays confirmed that OXPHOS<sup>high</sup> RT cells (patients 12 and 63) had a 3.5-fold higher oxygen consumption at routine respiration and fivefold higher electron transfer system capacity (ETC) compared to CLL. In addition, OXPHOS<sup>normal</sup> RT (patient 19) showed a routine oxygen consumption similar to CLL, although also had a relatively higher ETC than its CLL counterpart (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig5">5a</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM1">3a–d</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">24</a>). BCR signaling measured by Ca<sup>2+</sup> mobilization upon BCR stimulation with IgM showed that BCR<sup>low</sup> RT cells (patients 12 and 63) had a lower Ca<sup>2+</sup> flux compared to CLL, which contrasted with the higher flux observed in the BCR<sup>normal</sup> RT cells of patient 19, concordant with its IGLV3–21<sup>R110</sup> mutation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Minici, C. et al. Distinct homotypic B-cell receptor interactions shape the outcome of chronic lymphocytic leukaemia. Nat. Commun. 8, 15746 (2017)." href="/articles/s41591-022-01927-8#ref-CR27" id="ref-link-section-d73061417e2537">27</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig5">5b</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM1">4a,b</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">25</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Cellular respiration, BCR signaling and OXPHOS inhibition in RT cells."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Cellular respiration, BCR signaling and OXPHOS inhibition in RT cells.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01927-8/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="408"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Oxygen consumption of intact CLL and RT cells of three patients at routine respiration (routine), oligomycin-inhibited leak respiration (uncoupled) and uncoupler-stimulated ETC. Each dot represents a technical replicate. The mean of the replicates is shown using a horizontal line (left). Summary of the routine respiration of CLL and RT cells of the three patients collapsed (right). <b>b</b>, Calcium kinetics of tumoral cells (CD19<sup>+</sup>, CD5<sup>+</sup>) upon stimulation with 4-hydroxytamoxifen (4-OHT) and anti-BCR (black arrow). Basal calcium was adjusted at 5 × 10<sup>9</sup> Indo-1 ratio for 60 s before cell stimulation with F(ab′)2 anti-human IgM + H<sub>2</sub>O<sub>2</sub> at 37 °C. Then, Ca<sup>2+</sup> flux was recorded up to 500 s (left). Summary of the calcium release after BCR stimulation of CLL and RT cells. Average mean fluorescence after stimulation is represented (right). <b>c</b>, Cell proliferation after 72-h incubation with or without IACS-010759 (IACS) at 100 nM. Percentage of proliferating cells was determined by carboxyfluorescein succinimidyl ester (CFSE) cell tracer. Two technical replicates of each sample were performed (left). Summary of the proliferation for each CLL and RT cells with or without IACS treatment after 72 h. The normalized percentage of growth inhibition is indicated (right).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01927-8/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>To determine the biological effect of OXPHOS<sup>high</sup> in RT, we performed in vitro proliferation assays using IACS-010759 (100 nM), an OXPHOS inhibitor that targets mitochondrial complex I (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM1">3e</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM1">4c</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">25</a>). OXPHOS<sup>high</sup> RT (patients 12 and 63) had a higher proliferation at 72 h compared to OXPHOS<sup>normal</sup> RT (patients 19) and all of them were higher than their respective CLL. OXPHOS inhibition resulted in a marked decrease in proliferation in OXPHOS<sup>high</sup> RT (mean 49.1%), which contrasted with that observed in OXPHOS<sup>normal</sup> RT (2.2% decrease) and CLL (23.2% decrease) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig5">5c</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM1">4d</a>). Overall, these results confirm the role of OXPHOS<sup>high</sup> phenotype in high proliferation of RT and suggest its potential therapeutic value in RT as proposed for other neoplasms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Caro, P. et al. Metabolic signatures uncover distinct targets in molecular subsets of diffuse large B cell lymphoma. Cancer Cell 22, 547–560 (2012)." href="#ref-CR53" id="ref-link-section-d73061417e2624">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Norberg, E. et al. Differential contribution of the mitochondrial translation pathway to the survival of diffuse large B-cell lymphoma subsets. Cell Death Differ. 24, 251–262 (2017)." href="#ref-CR54" id="ref-link-section-d73061417e2624_1">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Molina, J. R. et al. An inhibitor of oxidative phosphorylation exploits cancer vulnerability. Nat. Med. 24, 1036–1046 (2018)." href="#ref-CR55" id="ref-link-section-d73061417e2624_2">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vangapandu, H. V. et al. Biological and metabolic effects of IACS-010759, an OxPhos inhibitor, on chronic lymphocytic leukemia cells. Oncotarget 9, 24980–24991 (2018)." href="#ref-CR56" id="ref-link-section-d73061417e2624_3">56</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Zhang, L. et al. Metabolic reprogramming toward oxidative phosphorylation identifies a therapeutic target for mantle cell lymphoma. Sci. Transl. Med. 11, eaau1167 (2019)." href="/articles/s41591-022-01927-8#ref-CR57" id="ref-link-section-d73061417e2627">57</a></sup>.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec8-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec8">Discussion</h2><div class="c-article-section__content" id="Sec8-content"><p>The genome of RT is characterized by a compendium of driver alterations in cell cycle, MYC, NOTCH and NF-κB pathways, frequently targeted in single catastrophic events and by the footprints of early-in-time, treatment-related, mutational processes, including the new SBS-RT potentially associated with bendamustine and chlorambucil exposure. A very early diversification of CLL leads to emergence of RT cells with fully assembled genomic, immunogenetic and transcriptomic profiles already at CLL diagnosis up to 19 years before the clonal explosion associated with the clinical transformation. RT cells have a notable shift in chromatin configuration and transcriptional program that converges into activation of the OXPHOS pathway and downregulation of BCR signaling, the latter potentially compensated by activating Toll-like, MYC and MAPK pathways<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Chakraborty, S. et al. B-cell receptor signaling and genetic lesions in TP53 and CDKN2A/CDKN2B cooperate in Richter transformation. Blood 138, 1053–1066 (2021)." href="/articles/s41591-022-01927-8#ref-CR17" id="ref-link-section-d73061417e2640">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Ntoufa, S., Vilia, M. G., Stamatopoulos, K., Ghia, P. &amp; Muzio, M. Toll-like receptors signaling: a complex network for NF-κB activation in B-cell lymphoid malignancies. Semin. Cancer Biol. 39, 15–25 (2016)." href="/articles/s41591-022-01927-8#ref-CR51" id="ref-link-section-d73061417e2643">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Varano, G. et al. The B-cell receptor controls fitness of MYC-driven lymphoma cells via GSK3β inhibition. Nature 546, 302–306 (2017)." href="/articles/s41591-022-01927-8#ref-CR58" id="ref-link-section-d73061417e2646">58</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Dadashian, E. L. et al. TLR signaling is activated in lymph node–resident CLL cells and is only partially inhibited by ibrutinib. Cancer Res. 79, 360–371 (2019)." href="/articles/s41591-022-01927-8#ref-CR59" id="ref-link-section-d73061417e2649">59</a></sup>. The rapid expansion of RT subclones under treatment with BCR inhibitors is consistent with its low BCR signaling, except when carrying the IGLV3–21<sup>R110</sup> and further supported by the increased number of subclones carrying unproductive immunoglobulin genes and the development of RT with plasmablastic differentiation, a cell type independent of BCR signaling<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Chan, K.-L. et al. Plasmablastic Richter transformation as a resistance mechanism for chronic lymphocytic leukaemia treated with BCR signalling inhibitors. Br. J. Haematol. 177, 324–328 (2017)." href="/articles/s41591-022-01927-8#ref-CR60" id="ref-link-section-d73061417e2655">60</a></sup>. Finally, we also uncovered that OXPHOS inhibition reduced the proliferation of RT cells in vitro, a finding worth exploring in future therapeutic strategies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Molina, J. R. et al. An inhibitor of oxidative phosphorylation exploits cancer vulnerability. Nat. Med. 24, 1036–1046 (2018)." href="/articles/s41591-022-01927-8#ref-CR55" id="ref-link-section-d73061417e2659">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Zhang, L. et al. Metabolic reprogramming toward oxidative phosphorylation identifies a therapeutic target for mantle cell lymphoma. Sci. Transl. Med. 11, eaau1167 (2019)." href="/articles/s41591-022-01927-8#ref-CR57" id="ref-link-section-d73061417e2662">57</a></sup>.</p><p>In conclusion, our comprehensive characterization of CLL evolution toward RT has revealed new genomic drivers and epigenomic reconfiguration with very early emergence of subclones driving late stages of cancer evolution, which may set the basis for developing single-cell-based predictive strategies. Furthermore, this study also identifies new RT-specific therapeutic targets and suggests that early intervention to eradicate dormant RT subclones may prevent the future development of this lethal complication of CLL.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec9-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">Methods</h2><div class="c-article-section__content" id="Sec9-content"><h3 class="c-article__sub-heading" id="Sec10">Consent and sample processing</h3><p>Written informed consent was obtained from all patients. The study was approved by the Hospital Clinic of Barcelona Ethics Committee. Tumor DNA was extracted from tumor cells purified from fresh/cryopreserved mononuclear cells, frozen lymph nodes or formalin-fixed paraffin-embedded (FFPE) tissue (<i>n</i> = 1, CLL sample of patient 1,669). Germline DNA was obtained from the non-tumoral purified cell fraction in 12 cases. In two patients (1,523 and 4,675) who had received allogeneic stem-cell transplant before RT, germline DNA of the donor was also collected. All extractions were performed using appropriate QIAGEN kits (QIAamp DNA Blood Maxi kit, cat. no. 51194; QIAamp DNA Mini kit, cat. no. 51304; and AllPrep DNA/RNA FFPE kit, cat. no. 80234). Tumor RNA was obtained from tumor cells purified from fresh/cryopreserved mononuclear cells with TRIzol reagent (Invitrogen, cat. no. 15596026).</p><p>A specific flow cytometry analysis was conducted on peripheral blood samples of patient 12, which were stained with the Lymphocyte Screening Tube according to EuroFlow protocols (<a href="https://www.euroflow.org/protocols">https://www.euroflow.org/protocols</a>). At least 100,000 cells were acquired in a FACSCanto II instrument. Analysis was conducted using the Infinicyt 2.0 software. The sequential gating analysis was as follows: singlet identification in a FSC-W versus FSC-H plot; leukocyte identification in SSC-A versus CD45 (V500-C) plot and FSC-A versus SSC-A; lymphocytes identified as SSC-A low and CD45 high and back-gated in FSC-A versus SSC-A to exclude monocytes; in the lymphocyte gate, T cells were identified as CD3<sup>+</sup> cells in SSC-A versus CD3 (APC) followed by sequentially distinguishing TCRγδ<sup>+</sup> T cells, CD4 T cells and CD8 T cells; after excluding T cells, B cells were selected in a SSC-A versus CD19 (PE-Cy7), followed by inspection of CD19 (PECy7) versus CD20 (PacB), CD5 (PerCPCy5.5) versus CD20 (PacB) and CD20 (PacB) versus CD38 (APC-H7) plots to evaluate the expression of these B cell markers and the assignation of κ and λ expression in a plot of IgK (PE) versus IgL (FITC); after excluding B cells, natural killer cells were identified in a SSC-A versus CD56 (PE) plot followed by SSC-A versus CD38 (APC-H7) plot.</p><h3 class="c-article__sub-heading" id="Sec11">WGS and WES</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec12">Library preparation and sequencing</h4><p>All samples available were subjected to WGS except the FFPE CLL, which was analyzed by whole-exome sequencing (WES). WGS libraries were performed using the Kapa Library Preparation kit (Roche, cat. no. 07961901001), TruSeq DNA PCR-Free kit (Illumina, cat. no. 20015963) or TruSeq DNA Nano protocol (Illumina, cat. no. 20015965) and sequenced on a HiSeq 2000/4000/X Ten (2 × 126 bp or 2 × 151 bp) or NovaSeq 6000 (2 × 151 bp) instrument (Illumina). WES was performed using the SureSelect Human All Exon V5 (Agilent Technologies, cat. no. 5190-6209 and G9611B) coupled with a KAPA Hyper Prep kit (Roche, cat. no. 07962363001) for the DNA pre-capture library. Sequencing was performed on a HiSeq 2000 (2 × 101 bp). We also included WGS of three published CLL/germline pairs (patients 12, 19 and 63)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Puente, X. S. et al. Non-coding recurrent mutations in chronic lymphocytic leukaemia. Nature 526, 519–524 (2015)." href="/articles/s41591-022-01927-8#ref-CR28" id="ref-link-section-d73061417e2710">28</a></sup> (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">1</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec13">General considerations</h4><p>Overall, 12 patients had a complete dataset (germline, CLL and RT samples), 6 patients lacked germline DNA and 1 patient had only the RT sample (case 4,676). We conducted tumor versus normal analyses in cases with a complete dataset. For the six patients lacking the germline sample, we used the CLL samples as ‘normal’ to identify SNV acquired at RT for mutational signature analyses. In addition, tumor-only analyses were conducted in these CLL and RT samples, as well as in the patient with only a RT sample available, to identify driver gene mutations and genome-wide CNAs (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">1</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec14">Read mapping and quality control</h4><p>Reads were mapped to the human reference genome (GRCh37) using the BWA-MEM algorithm (v.0.7.15)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009)." href="/articles/s41591-022-01927-8#ref-CR61" id="ref-link-section-d73061417e2736">61</a></sup>. BAM files were generated and optical/PCR duplicates flagged using biobambam2 (v.2.0.65, <a href="https://gitlab.com/german.tischler/biobambam2">https://gitlab.com/german.tischler/biobambam2</a>). FastQC (v.0.11.5, <a href="http://www.bioinformatics.babraham.ac.uk/projects/fastqc">www.bioinformatics.babraham.ac.uk/projects/fastqc</a>) and Picard (v.2.10.2, <a href="https://broadinstitute.github.io/picard">https://broadinstitute.github.io/picard</a>) were used to extract quality control metrics. Mean coverage was 33× and 119× for WGS and WES, respectively (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">1</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec15">Immunoglobulin gene characterization</h4><p>Immunoglobulin gene rearrangements were characterized using IgCaller (v.1.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Nadeu, F. et al. IgCaller for reconstructing immunoglobulin gene rearrangements and oncogenic translocations from whole-genome sequencing in lymphoid neoplasms. Nat. Commun. 11, 3390 (2020)." href="/articles/s41591-022-01927-8#ref-CR62" id="ref-link-section-d73061417e2772">62</a></sup>. The rearranged sequences obtained were reviewed on the Integrative Genomics Viewer (IGV; v.2.9.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Robinson, J. T. et al. Integrative genomics viewer. Nat. Biotechnol. 29, 24–26 (2011)." href="/articles/s41591-022-01927-8#ref-CR63" id="ref-link-section-d73061417e2776">63</a></sup> and annotated using IMGT/V-QUEST (<a href="https://www.imgt.org/IMGT_vquest">https://www.imgt.org/IMGT_vquest</a>) and ARResT/AssignSubsets (<a href="http://bat.infspire.org/arrest/assignsubsets">http://bat.infspire.org/arrest/assignsubsets</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec16">Tumor versus normal SNVs and indel calling</h4><p>SNVs were called using Sidrón<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Puente, X. S. et al. Non-coding recurrent mutations in chronic lymphocytic leukaemia. Nature 526, 519–524 (2015)." href="/articles/s41591-022-01927-8#ref-CR28" id="ref-link-section-d73061417e2803">28</a></sup>, CaVEMan (cgpCaVEManWrapper, v.1.12.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Jones, D. et al. cgpCaVEManWrapper: simple execution of CaVEMan in order to detect somatic single nucleotide variants in NGS data. Curr. Protoc. Bioinforma. 56, 15.10.1–15.10.18 (2016)." href="/articles/s41591-022-01927-8#ref-CR64" id="ref-link-section-d73061417e2807">64</a></sup>, Mutect2 (Genome Analysis Toolkit (GATK) v.4.0.2.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="McKenna, A. et al. The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 20, 1297–1303 (2010)." href="/articles/s41591-022-01927-8#ref-CR65" id="ref-link-section-d73061417e2811">65</a></sup> and MuSE (v.1.0 rc)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Fan, Y. et al. MuSE: accounting for tumor heterogeneity using a sample-specific error model improves sensitivity and specificity in mutation calling from sequencing data. Genome Biol. 17, 178 (2016)." href="/articles/s41591-022-01927-8#ref-CR66" id="ref-link-section-d73061417e2815">66</a></sup> and normalized using bcftools (v.1.8)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Danecek, P. et al. Twelve years of SAMtools and BCFtools. Gigascience 
                  https://doi.org/10.1093/gigascience/giab008
                  
                 (2021)." href="/articles/s41591-022-01927-8#ref-CR67" id="ref-link-section-d73061417e2819">67</a></sup>. Variants detected by CaVEMan with more than half of the mutant reads clipped (CLPM &gt; 0) and with supporting reads with a median alignment score (ASMD) &lt;90, &lt;120 or &lt;140 for sequencing read lengths of 100, 125 or 150 bp, respectively, were excluded. Variants called by Mutect2 with MMQ &lt; 60 were eliminated. Mutations detected by at least two algorithms were considered. Short insertions/deletions (indels) were called by SMuFin (v.0.9.4)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Moncunill, V. et al. Comprehensive characterization of complex structural variations in cancer by directly comparing genome sequence reads. Nat. Biotechnol. 32, 1106–1112 (2014)." href="/articles/s41591-022-01927-8#ref-CR68" id="ref-link-section-d73061417e2824">68</a></sup>, Pindel (cgpPindel, v.2.2.3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Raine, K. M. et al. cgpPindel: identifying somatically acquired insertion and deletion events from paired end sequencing. Curr. Protoc. Bioinforma. 52, 15.7.1–12 (2015)." href="/articles/s41591-022-01927-8#ref-CR69" id="ref-link-section-d73061417e2828">69</a></sup>, SvABA (v.7.0.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Wala, J. A. et al. SvABA: genome-wide detection of structural variants and indels by local assembly. Genome Res. 28, 581–591 (2018)." href="/articles/s41591-022-01927-8#ref-CR70" id="ref-link-section-d73061417e2832">70</a></sup>, Mutect2 (GATK v.4.0.2.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="McKenna, A. et al. The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 20, 1297–1303 (2010)." href="/articles/s41591-022-01927-8#ref-CR65" id="ref-link-section-d73061417e2836">65</a></sup> and Platypus (v.0.8.1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Rimmer, A. et al. Integrating mapping-, assembly- and haplotype-based approaches for calling variants in clinical sequencing applications. Nat. Genet. 46, 912–918 (2014)." href="/articles/s41591-022-01927-8#ref-CR71" id="ref-link-section-d73061417e2840">71</a></sup>. The somaticMutationDetector.py script (<a href="https://github.com/andyrimmer/Platypus/blob/master/extensions/Cancer/somaticMutationDetector.py">https://github.com/andyrimmer/Platypus/blob/master/extensions/Cancer/somaticMutationDetector.py</a>) was used to identify somatic indels called by Platypus. Indels were left-aligned and normalized using bcftools<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Danecek, P. et al. Twelve years of SAMtools and BCFtools. Gigascience 
                  https://doi.org/10.1093/gigascience/giab008
                  
                 (2021)." href="/articles/s41591-022-01927-8#ref-CR67" id="ref-link-section-d73061417e2852">67</a></sup>. Indels with MMQ &lt; 60, MQ &lt; 60 and MAPQ &lt; 60 for Mutect2, Platypus and SvABA, respectively, were removed. Only indels identified by at least two algorithms were retained. Annotation of mutations was performed using snpEff/snpSift (v.4.3t)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. Fly. (Austin). 6, 80–92 (2012)." href="/articles/s41591-022-01927-8#ref-CR72" id="ref-link-section-d73061417e2856">72</a></sup> and GRCh37.p13.RefSeq as a reference. This approach showed a 93% specificity and 88% sensitivity when benchmarked against the mutations found at a VAF &gt;10% in our previous high-coverage NGS study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Nadeu, F. et al. Clinical impact of the subclonal architecture and mutational complexity in chronic lymphocytic leukemia. Leukemia 32, 645–653 (2018)." href="/articles/s41591-022-01927-8#ref-CR73" id="ref-link-section-d73061417e2860">73</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec17">Tumor-only SNVs and indel calling</h4><p>Tumor-only variant calling was restricted to coding regions of 243 genes described as drivers in CLL and other B cell lymphomas (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">10</a>). Mini-BAM files were obtained using Picard tools and variant calling was performed using Mutect2 (GATK v.4.0.4.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="McKenna, A. et al. The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. Genome Res. 20, 1297–1303 (2010)." href="/articles/s41591-022-01927-8#ref-CR65" id="ref-link-section-d73061417e2875">65</a></sup>, VarScan2 (v.2.4.3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Koboldt, D. C. et al. VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing. Genome Res. 22, 568–576 (2012)." href="/articles/s41591-022-01927-8#ref-CR74" id="ref-link-section-d73061417e2879">74</a></sup>, VarDictJava (v.1.4)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Lai, Z. et al. VarDict: a novel and versatile variant caller for next-generation sequencing in cancer research. Nucleic Acids Res. 44, e108 (2016)." href="/articles/s41591-022-01927-8#ref-CR75" id="ref-link-section-d73061417e2883">75</a></sup>, LoFreq (v.2.1.3.1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Wilm, A. et al. LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets. Nucleic Acids Res. 40, 11189–11201 (2012)." href="/articles/s41591-022-01927-8#ref-CR76" id="ref-link-section-d73061417e2887">76</a></sup>, outLyzer (v.1.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Muller, E. et al. OutLyzer: software for extracting low-allele-frequency tumor mutations from sequencing background noise in clinical practice. Oncotarget 7, 79485–79493 (2016)." href="/articles/s41591-022-01927-8#ref-CR77" id="ref-link-section-d73061417e2892">77</a></sup> and freebayes (v.1.1.0, <a href="https://github.com/freebayes/freebayes">https://github.com/freebayes/freebayes</a>). Variants were normalized using bcftools (v.1.9)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Danecek, P. et al. Twelve years of SAMtools and BCFtools. Gigascience 
                  https://doi.org/10.1093/gigascience/giab008
                  
                 (2021)." href="/articles/s41591-022-01927-8#ref-CR67" id="ref-link-section-d73061417e2903">67</a></sup> and annotated using snpEff/snpSift (v.4.3t)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of Drosophila melanogaster strain w1118; iso-2; iso-3. Fly. (Austin). 6, 80–92 (2012)." href="/articles/s41591-022-01927-8#ref-CR72" id="ref-link-section-d73061417e2907">72</a></sup>. Only non-synonymous variants that were identified as PASS by ≥2 algorithms were considered. Variants reported in 1000 Genomes Project, ExAC or gnomAD with a population frequency &gt;1% or reported as germline in our ICGC database of 506 WES/WGS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Puente, X. S. et al. Non-coding recurrent mutations in chronic lymphocytic leukaemia. Nature 526, 519–524 (2015)." href="/articles/s41591-022-01927-8#ref-CR28" id="ref-link-section-d73061417e2911">28</a></sup> were considered as polymorphisms.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec18">Tumor versus normal CNA calling</h4><p>CNAs were called using Battenberg (cgpBattenberg, v.3.2.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Nik-Zainal, S. et al. The life history of 21 breast cancers. Cell 149, 994–1007 (2012)." href="/articles/s41591-022-01927-8#ref-CR78" id="ref-link-section-d73061417e2923">78</a></sup> and ASCAT (ascatNgs, v.4.1.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Raine, K. M. et al. ascatNgs: identifying somatically acquired copy-number alterations from whole-genome sequencing data. Curr. Protoc. Bioinforma. 56, 15.9.1–15.9.17 (2016)." href="/articles/s41591-022-01927-8#ref-CR79" id="ref-link-section-d73061417e2927">79</a></sup>. CNAs within any of the immunoglobulin loci were not considered. We used the tumor purities obtained by Battenberg in downstream analyses. The median tumor cell content was 91.5% (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">1</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec19">Tumor-only CNA calling</h4><p>CNAs were extracted using CNVkit (v.0.9.3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Talevich, E., Shain, A. H., Botton, T. &amp; Bastian, B. C. CNVkit: genome-wide copy number detection and visualization from targeted DNA sequencing. PLoS Comput. Biol. 12, e1004873 (2016)." href="/articles/s41591-022-01927-8#ref-CR80" id="ref-link-section-d73061417e2942">80</a></sup>. CNAs &lt;500 kb, with an absolute log<sub>2</sub> copy ratio (log<sub>2</sub>CR) &lt; 0.3 or located within any of the immunoglobulin loci were removed. CNAs were classified as gains if log<sub>2</sub>CR &gt; 0.3, deletions if log<sub>2</sub>CR &lt; −0.3, high-copy gains if log<sub>2</sub>CR &gt; 1.1 and homozygous deletions if log<sub>2</sub>CR &lt; −1.1. The log<sub>2</sub>CR cutoff was set to 0.15 for two samples with low tumor cell content (102-01-01TD and 4690-03-01BD). To avoid a high segmentation of the CNA profile, CNAs belonging to the same class were merged if they were separated by &lt;1 Mb and had an absolute log<sub>2</sub>CR difference &lt;0.25.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec20">Array-based CNA calling in FFPE</h4><p>CNAs were examined in the FFPE CLL sample using the Oncoscan CNV FFPE Assay kit (Thermo Fisher Scientific, cat. no. 902695) and analyzed using Nexus 9.0 software (Biodiscovery).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">Tumor versus normal SV calling</h4><p>SVs were extracted using SMuFin (v.0.9.4)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Moncunill, V. et al. Comprehensive characterization of complex structural variations in cancer by directly comparing genome sequence reads. Nat. Biotechnol. 32, 1106–1112 (2014)." href="/articles/s41591-022-01927-8#ref-CR68" id="ref-link-section-d73061417e2979">68</a></sup>, BRASS (v.6.0.5)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome sequences. Nature 534, 47–54 (2016)." href="/articles/s41591-022-01927-8#ref-CR81" id="ref-link-section-d73061417e2983">81</a></sup>, SvABA (v.7.0.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Wala, J. A. et al. SvABA: genome-wide detection of structural variants and indels by local assembly. Genome Res. 28, 581–591 (2018)." href="/articles/s41591-022-01927-8#ref-CR70" id="ref-link-section-d73061417e2987">70</a></sup> and DELLY2 (v.0.8.1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Rausch, T. et al. DELLY: structural variant discovery by integrated paired-end and split-read analysis. Bioinformatics 28, i333–i339 (2012)." href="/articles/s41591-022-01927-8#ref-CR82" id="ref-link-section-d73061417e2991">82</a></sup>. SVs identified were intersected considering a window of 300 bp around break points. We kept for downstream analyses the SVs identified by at least two programs if at least one of the algorithms called the alteration with high quality (MAPQ ≥ 90 for BRASS, MAPQ = 60 for SvABA and DELLY2). In addition, IgCaller (v.1.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Nadeu, F. et al. IgCaller for reconstructing immunoglobulin gene rearrangements and oncogenic translocations from whole-genome sequencing in lymphoid neoplasms. Nat. Commun. 11, 3390 (2020)." href="/articles/s41591-022-01927-8#ref-CR62" id="ref-link-section-d73061417e2995">62</a></sup> was used to call SVs within any of the immunoglobulin loci. All SVs were visually inspected using IGV<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Robinson, J. T. et al. Integrative genomics viewer. Nat. Biotechnol. 29, 24–26 (2011)." href="/articles/s41591-022-01927-8#ref-CR63" id="ref-link-section-d73061417e3000">63</a></sup>. SVs were categorized into simple or complex events. Chromothripsis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Stephens, P. J. et al. Massive genomic rearrangement acquired in a single catastrophic event during cancer development. Cell 144, 27–40 (2011)." href="/articles/s41591-022-01927-8#ref-CR83" id="ref-link-section-d73061417e3004">83</a></sup> was defined as ≥7 oscillating changes between two or three copy number states or the presence of &gt;7 SV break points occurring in a single chromosome and supported by additional criteria<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Stephens, P. J. et al. Massive genomic rearrangement acquired in a single catastrophic event during cancer development. Cell 144, 27–40 (2011)." href="/articles/s41591-022-01927-8#ref-CR83" id="ref-link-section-d73061417e3008">83</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Korbel, J. O. &amp; Campbell, P. J. Criteria for inference of chromothripsis in cancer genomes. Cell 152, 1226–1236 (2013)." href="/articles/s41591-022-01927-8#ref-CR84" id="ref-link-section-d73061417e3011">84</a></sup>. Chromoplexy was determined by the presence of ≥3 chained chromosomal rearrangements, where chains were identified using a window of 50 kb<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Baca, S. C. et al. Punctuated evolution of prostate cancer genomes. Cell 153, 666–677 (2013)." href="/articles/s41591-022-01927-8#ref-CR85" id="ref-link-section-d73061417e3015">85</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Shen, M. M. Chromoplexy: a new category of complex rearrangements in the cancer genome. Cancer Cell 23, 567–569 (2013)." href="/articles/s41591-022-01927-8#ref-CR86" id="ref-link-section-d73061417e3018">86</a></sup>. Cycles of templated insertions were defined as copy number gains in ≥3 chromosomes linked by SVs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Li, Y. et al. Patterns of somatic structural variation in human cancer genomes. Nature 578, 112–121 (2020)." href="/articles/s41591-022-01927-8#ref-CR87" id="ref-link-section-d73061417e3022">87</a></sup>. Breakage-fusion bridge cycles were defined as patterns of focal copy number increases and fold-back inversions, together with telomeric deletions. Chains of rearrangements having &gt;2 SVs and not fulfilling any of the previous criteria were classified as ‘other complex events’. Chromothripsis and ‘other complex events’ were subcategorized according to the number of chromosomes involved. The longitudinal nature of our dataset allowed us to refine the obtained classification based on the presence of the involved alterations in each time point analyzed.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec22">Patients who underwent allogenic stem-cell transplant</h4><p>In these patients, we conducted tumor versus patient’s germline and tumor versus donor’s germline variant calling in parallel. Only the intersection of variants identified was considered.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec23">Rescue of alterations based on longitudinal information</h4><p>SNVs called in one sample were automatically added to the samples of additional time point(s) if at least one high-quality read with the mutation was found in the BAM file (alleleCounter v.4.0.0, parameters: min_map_qual = 35; and min_base_qual = 20). Similarly, indels and SVs detected in one sample were added in the additional time point(s) if any of the algorithms detected the alteration, regardless of its filters.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec24">WGS-based subclonal reconstruction</h4><p>A Markov chain Monte Carlo sampler for a Dirichlet process mixture model was used to infer putative subclones, to assign mutations to subclones and to estimate the subclone frequencies in each sample from the SNV read counts, copy number states and tumor purities (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">17</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Nik-Zainal, S. et al. The life history of 21 breast cancers. Cell 149, 994–1007 (2012)." href="/articles/s41591-022-01927-8#ref-CR78" id="ref-link-section-d73061417e3054">78</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Maura, F. et al. Genomic landscape and chronological reconstruction of driver events in multiple myeloma. Nat. Commun. 10, 3835 (2019)." href="/articles/s41591-022-01927-8#ref-CR88" id="ref-link-section-d73061417e3057">88</a></sup>. Clusters with &lt;100 mutations were excluded. The phylogenetic relationships between subclones were identified following the ‘pigeonhole principle’, which was relaxed using a case-specific ‘tolerated error’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Maura, F. et al. Genomic landscape and chronological reconstruction of driver events in multiple myeloma. Nat. Commun. 10, 3835 (2019)." href="/articles/s41591-022-01927-8#ref-CR88" id="ref-link-section-d73061417e3061">88</a></sup>. Clusters not assigned to the reconstructed phylogenetic tree were excluded. Fish plots were generated using the TimeScape R package (v.1.6.0). The CCF of indels was calculated integrating read counts, CNAs and tumor purity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Dentro, S. C., Wedge, D. C. &amp; Van Loo, P. Principles of reconstructing the subclonal architecture of cancers. Cold Spring Harb. Perspect. Med. 7, a026625 (2017)." href="/articles/s41591-022-01927-8#ref-CR89" id="ref-link-section-d73061417e3065">89</a></sup>. Driver indels subjected to validation by scDNA-seq and/or relevant to the tumor phylogeny were manually assigned to subclones. Similarly, driver CNAs relevant to the phylogeny were manually assigned. Seven SNVs found in <i>TP53</i>/<i>ATM</i> overlapping with CNAs were manually assigned to the most likely subclone as they were not automatically assigned by the Dirichlet process and were subjected to scDNA-seq (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">9</a>).</p><h3 class="c-article__sub-heading" id="Sec25">Mutational signatures</h3><p>We studied mutational signatures acting genome-wide and in localized regions (inter-mutation distance ≤1Kb)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Kasar, S. et al. Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution. Nat. Commun. 6, 8866 (2015)." href="/articles/s41591-022-01927-8#ref-CR29" id="ref-link-section-d73061417e3088">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)." href="/articles/s41591-022-01927-8#ref-CR32" id="ref-link-section-d73061417e3091">32</a></sup>. We integrated the mutations identified in this CLL/RT cohort together with those of 147 CLL treatment-naive samples (ICGC-CLL)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Puente, X. S. et al. Non-coding recurrent mutations in chronic lymphocytic leukaemia. Nature 526, 519–524 (2015)." href="/articles/s41591-022-01927-8#ref-CR28" id="ref-link-section-d73061417e3095">28</a></sup> and 27 new CLL collected at relapse post-treatment (mean coverage 31.5×; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">15</a>). The WGS of these two additional cohorts was (re-)analyzed using our current bioinformatic pipeline (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">12</a>). Mutational signatures were analyzed for SNVs or single-base substitutions (SBSs) according to their 5′ and 3′ flanking bases following three steps<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Maura, F. et al. A practical guide for mutational signature analysis in hematological malignancies. Nat. Commun. 10, 2969 (2019)." href="/articles/s41591-022-01927-8#ref-CR30" id="ref-link-section-d73061417e3105">30</a></sup>:</p><ol class="u-list-style-none">
                  <li>
                    <span class="u-custom-list-number">1.</span>
                    
                      <p>Extraction: de novo signature extraction was performed using a hierarchical Dirichlet process (HDP, v.0.1.5; <a href="https://github.com/nicolaroberts/hdp">https://github.com/nicolaroberts/hdp</a>), SignatureAnalyzer (v.0.0.7)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Kim, J. et al. Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors. Nat. Genet. 48, 600–606 (2016)." href="/articles/s41591-022-01927-8#ref-CR90" id="ref-link-section-d73061417e3127">90</a></sup>, SigProfiler (SigProfilerExtractor, v.1.0.8)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)." href="/articles/s41591-022-01927-8#ref-CR32" id="ref-link-section-d73061417e3131">32</a></sup> and sigfit (v.2.0.0; <a href="https://github.com/kgori/sigfit">https://github.com/kgori/sigfit</a>). HDP was run with four independent posterior sampling chains, followed by 20,000 burn-in iterations and the collection of 200 posterior samples off each chain with 200 iterations between each. SigProfiler was run with 1,000 iterations and a maximum of ten extracted signatures. Similarly, sigfit was run to extract five signatures with 10,000 burn-in iterations and 20,000 sampling iterations.</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">2.</span>
                    
                      <p>Assignment: each extracted signature was assigned to a given COSMIC signature (v.3.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)." href="/articles/s41591-022-01927-8#ref-CR32" id="ref-link-section-d73061417e3153">32</a></sup> if their cosine similarity was &gt;0.85. Otherwise, the extracted signature was decomposed into ‘<i>n</i>’ COSMIC signatures using an expectation maximization (EM) algorithm<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. Nature 574, 532–537 (2019)." href="/articles/s41591-022-01927-8#ref-CR91" id="ref-link-section-d73061417e3160">91</a></sup>. The EM algorithm was first run using the COSMIC signatures identified in the previous step. If their cosine similarity was &lt;0.85, we ran the EM algorithm, including all signatures reported in COSMIC and by Kucab et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Kucab, J. E. et al. A compendium of mutational signatures of environmental agents. Cell 177, 821–836 (2019)." href="/articles/s41591-022-01927-8#ref-CR33" id="ref-link-section-d73061417e3164">33</a></sup> (55 mutational signatures related to environmental agents). Three exceptions were made: (1) we combined two HDP signatures that together constituted COSMIC signature SBS5 to avoid splitting of signatures (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig9">4a</a>); (2) APOBEC signatures (SBS2 and SBS13) were favored to be assigned to one of the signatures extracted by HDP and SignatureAnalyzer although it was not the best EM solution probably because they were only found in one sample, which impaired a clean extraction of the signatures (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig9">4f</a>); and (3) one signature extracted by HDP and SignatureAnalyzer was directly assigned to the mutational signature associated with ganciclovir treatment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="de Kanter, J. K. et al. Antiviral treatment causes a unique mutational signature in cancers of transplantation recipients. Cell Stem Cell 28, 1726–1739 (2021)." href="/articles/s41591-022-01927-8#ref-CR35" id="ref-link-section-d73061417e3175">35</a></sup> (cosine similarity 0.987 and 0.993, respectively) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig9">4</a>). The new SBS-RT extracted by HDP was considered for downstream analyses as it had less background noise than the one extracted by SignatureAnalyzer, favoring a higher specificity during the fitting step. Similarly, the SBS-ganciclovir extracted by HDP was used in downstream analyses (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01927-8#Fig9">4</a>). We also performed a detailed review to remove signatures susceptible of being originated due to sequencing artifacts (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">13</a>).</p>
                    
                  </li>
                  <li>
                    <span class="u-custom-list-number">3.</span>
                    
                      <p>Fitting: we used a fitting approach (MutationalPatterns, v.3.0.1) to measure the contribution of each mutational signature in each sample. Based on (1) the de novo identification of the therapy-related SBS-ganciclovir and (2) that two patients received melphalan before RT, the mutational signature associated with melphalan therapy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Rustad, E. H. et al. Timing the initiation of multiple myeloma. Nat. Commun. 11, 1917 (2020)." href="/articles/s41591-022-01927-8#ref-CR34" id="ref-link-section-d73061417e3199">34</a></sup> was also included in this step. To avoid the so-called inter-sample bleeding effect<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Maura, F. et al. A practical guide for mutational signature analysis in hematological malignancies. Nat. Commun. 10, 2969 (2019)." href="/articles/s41591-022-01927-8#ref-CR30" id="ref-link-section-d73061417e3203">30</a></sup>, we iteratively removed the less-contributing signature if its removal decreased the cosine similarity between the original and reconstructed 96-profile &lt;0.01 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)." href="/articles/s41591-022-01927-8#ref-CR32" id="ref-link-section-d73061417e3207">32</a></sup>). SBS1 and SBS5 were added if addition improved the cosine similarity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. Nature 578, 94–101 (2020)." href="/articles/s41591-022-01927-8#ref-CR32" id="ref-link-section-d73061417e3211">32</a></sup>. Similarly, SBS9 was added in CLL/RT samples classified as M-CLL if addition improved the cosine similarity. We also ran mSigAct (v.2.1.1; <a href="https://github.com/steverozen/mSigAct">https://github.com/steverozen/mSigAct</a>) to confirm the presence/absence of SBS-melphalan (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">15</a>). To assess the contribution of each signature to each subclone we followed the same fitting strategy but (1) considered only the signatures that were present in the corresponding sample and (2) removed the final step of adding SBS9 in M-CLL to avoid its addition in multiple subclones with low evidence.</p>
                    
                  </li>
                </ol><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec26">Genomic locations and strand bias</h4><p>We assessed the contribution of SBS-RT to coding SNVs in RT subclones (also including cases in which the CLL sample was used as a ‘germline’) by calculating the probability that a given mutation was caused by SBS-RT. To perform this calculation, we considered the signatures present in the subclone/sample and their signature profile<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Yang, F. et al. Chemotherapy and mismatch repair deficiency cooperate to fuel TP53 mutagenesis and ALL relapse. Nat. Cancer 2, 819–834 (2021)." href="/articles/s41591-022-01927-8#ref-CR92" id="ref-link-section-d73061417e3236">92</a></sup>. The reference epigenomes of CLL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Beekman, R. et al. The reference epigenome and regulatory chromatin landscape of chronic lymphocytic leukemia. Nat. Med. 24, 868–880 (2018)." href="/articles/s41591-022-01927-8#ref-CR44" id="ref-link-section-d73061417e3240">44</a></sup> were used to explore the contribution of the mutational processes in different regulatory regions. We simplified the described chromatin states in four categories: heterochromatin (H3K9me3_Repressed, Heterochromatin Low_Signal), polycomb (Posied_Promoter, H3K27me3_Repressed), enhancer/promoter (Active_Promoter, Strong_Enhancer1, Weak_Promoter, Weak_Enhancer, Strong_Enhancer) and transcription (Transcription_Transition, Weak_Transcription, Transcription_Elongation). We also mapped the activity of mutational processes in early/late replication regions of the genome considering peaks/valleys of early/late replication as those regions of ≥1 kb with absolute replication timing &gt;0.5 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="Koren, A. et al. Differential relationship of DNA replication timing to different forms of human mutation and variation. Am. J. Hum. Genet. 91, 1033–1040 (2012)." href="/articles/s41591-022-01927-8#ref-CR93" id="ref-link-section-d73061417e3244">93</a></sup>). All SNVs of the CLL and RT subclones were classified in any of the four chromatin states and early/late replication regions before fitting mutational signatures. A cutoff of 0.005 was used to remove the less-contributing signature during the fitting step. We also generated replication and transcriptional strand bias profiles of the RT-specific mutations using the MutationalPatterns R package<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Rustad, E. H. et al. Timing the initiation of multiple myeloma. Nat. Commun. 11, 1917 (2020)." href="/articles/s41591-022-01927-8#ref-CR34" id="ref-link-section-d73061417e3248">34</a></sup>. The replication strand was annotated based on the left/right replication direction of the timing transition regions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 94" title="Haradhvala, N. J. et al. Mutational strand asymmetries in cancer genomes reveal mechanisms of DNA damage and repair. Cell 164, 538–549 (2016)." href="/articles/s41591-022-01927-8#ref-CR94" id="ref-link-section-d73061417e3252">94</a></sup>. The transcriptional strand was annotated using the TxDb.Hsapiens.UCSC.hg19.knownGene R package (v.3.2.2). Finally, kataegis was defined as a genomic region having six or more mutations with an average inter-mutation distance ≤1 kb.</p><h3 class="c-article__sub-heading" id="Sec27">High-coverage, UMI-based gene mutation analysis</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec28">Data generation</h4><p>A high-coverage, UMI-based NGS was performed to track 77 mutations identified by WGS (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">18</a>). Molecular-barcoded and target-enriched libraries were prepared using a Custom CleanPlex UMI NGS Panel (Paragon Genomics) and CleanPlex Unique Dual-Indexed PCR Primers for Illumina (Paragon Genomics, cat. no. 716011 and 716013). Libraries were sequenced on a MiSeq and/or NextSeq 2000 instrument (2 × 150 bp, Illumina).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec29">Data analysis</h4><p>Raw reads were trimmed using cutadapt (<a href="https://cutadapt.readthedocs.io">https://cutadapt.readthedocs.io</a>; v.1.15 with parameters: -g CCTACACGACGCTCTTCCGATCT -a AGATCGGAAGAGCACACGTCTGAA -A AGATCGGAAGAGCGTCGTGTAGG -G TTCAGACGTGTGCTCTTCCGATCT -e 0.1 -O 9 -m 20 -n 2). Trimmed FASTQ reads were converted to unmapped BAM using Picard’s FastqToSam tool (v.2.10.2). UMI information was extracted and stored as a tag using fgbio ExtractUmisFromBam (<a href="http://fulcrumgenomics.github.io/fgbio/">http://fulcrumgenomics.github.io/fgbio/</a>; v.1.3.0 with parameters: –read structure = 16M+T 16M+T, –single-tag = RX, –molecular-index-tags = ZA ZB). Template read was converted to FASTQ with Picard’s SamToFastq. Template reads were mapped against the human reference genome (GRCh37) and reads were merged with the UMI information using Picard’s MergeBamAlignment. Finally, reads were grouped by UMI and a consensus was called using fgbio GroupReadsByUmi (parameters were –strategy = adjacency, –edits = 1, –min-map = 10) and CallMolecularConsensusReads (parameters were –min-reads = 3), respectively. A minimum of three reads was required to create a UMI-based final read. Final reads were converted back to FASTQ using Picard’s SamToFastq and mapped against the reference genome using BWA-MEM (v.0.7.15)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009)." href="/articles/s41591-022-01927-8#ref-CR61" id="ref-link-section-d73061417e3294">61</a></sup>. Mean coverage was determined using Picard’s CollectTargetedPcrMetrics (parameters: CLIP_OVERLAPPING_READS = true, MINIMUM_MAPPING_QUALITY = 15 MINIMUM_BASE_QUALITY = 15). Read counts were collected at all targeted genomic positions for all samples using bcftools mpileup (v.1.8, parameters: -B -Q 13 -q 10 -d 100,000 -a FORMAT/DP,FORMAT/AD,FORMAT/ADF,FORMAT/ADR -O v)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Danecek, P. et al. Twelve years of SAMtools and BCFtools. Gigascience 
                  https://doi.org/10.1093/gigascience/giab008
                  
                 (2021)." href="/articles/s41591-022-01927-8#ref-CR67" id="ref-link-section-d73061417e3298">67</a></sup>. Allele positions lacking mutations by WGS were used to model the background sequencing noise, which was unified according to the trinucleotide context of each possible mutation. Mutations of interest were annotated as high confidence when their frequency was above the background noise with a probability of 95%.</p><h3 class="c-article__sub-heading" id="Sec30">High-coverage immunoglobulin gene characterization</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec31">DNA-based</h4><p>The LymphoTrack IGHV Leader Somatic Hypermutation Assay Panel, MiSeq (Invivoscribe Technologies, cat. no. 71210069) was performed in samples of two patients (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">21</a>). Libraries were sequenced on a MiSeq instrument (2 × 301 bp, Illumina). Clonotypes were defined as IGHV-IGHD-IGHJ gene rearrangements with the same IGHV gene and IGH CDR3 amino acid sequence within a sample. Clonotypes with different nucleotide substitutions within the FR1-CDR1-FR2-CDR2-FR3 sequence of the rearranged IGHV gene were defined as subclones. Raw FASTQ files were trimmed using Trimmomatic (v.0.36)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120 (2014)." href="/articles/s41591-022-01927-8#ref-CR95" id="ref-link-section-d73061417e3319">95</a></sup> to keep only high-quality reads and bases (parameters were LEADING:30 TRAILING:30 SLIDINGWINDOW:4:30 MINLEN:100). Trimmed, paired-end FASTQ files were analyzed using the LymphoTrack Software, MiSeq (v.2.3.1, Invivoscribe Technologies, cat. no. 75000009), which combines forward and reverse reads to generate full-length sequences. Identical full-length sequences were grouped and reported together with their cumulative frequency. The reported full-length sequences were annotated using IMGT/HighV-QUEST (v.1.8.3; <a href="https://www.imgt.org/HighV-QUEST">https://www.imgt.org/HighV-QUEST</a>). Finally, we (1) selected the sequences that belonged to the dominant productive clonotype; (2) kept only sequences with complete V-region (missing bases and indels within the V-region were not allowed); and (3) merged sequences that shared the exact V-region nucleotide sequence.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec32">RNA-based</h4><p>For patient 12, cryopreserved samples collected at four different time points were thawed and malignant cells were enriched using the The EasySep Human B Cell Enrichment kit II without CD43 depletion (Stemcell Technologies, cat. no. 17923). Next, 1–2 million tumor cells were used to perform the Omniscope BCR VDJ sequencing assay (<a href="https://www.omniscope.ai">https://www.omniscope.ai</a>). Cells were lysed and the RNA was reverse transcribed to complementary DNA with UMIs before amplification of the V(D)J region using BCR-specific multiplex PCR. Following sequencing, reads were aligned using STARsolo (v.2.7.9a; <a href="https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md">https://github.com/alexdobin/STAR/blob/master/docs/STARsolo.md</a>) to the hg38 human genome. IGV<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Robinson, J. T. et al. Integrative genomics viewer. Nat. Biotechnol. 29, 24–26 (2011)." href="/articles/s41591-022-01927-8#ref-CR63" id="ref-link-section-d73061417e3352">63</a></sup> was used to review and quantify the mutation of interest (chr14:106714886C&gt;T).</p><h3 class="c-article__sub-heading" id="Sec33">DNA methylation</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec34">Data generation and processing</h4><p>DNA methylation data of 39 samples was generated using EPIC BeadChips (Illumina). These samples included different healthy B cell subpopulations (naive B cells (NBCs), <i>n</i> = 2; germinal center B cells (GCs), <i>n</i> = 1; memory B cells (MBCs), <i>n</i> = 3; tonsillar plasma cells (tPCs), <i>n</i> = 1); CLL samples without evidence of RT (<i>n</i> = 12) and longitudinal CLL/RT samples (<i>n</i> = 20) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">6</a>). R and core Bioconductor packages, including minfi (v.1.34.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Aryee, M. J. et al. Minfi: a flexible and comprehensive Bioconductor package for the analysis of Infinium DNA methylation microarrays. Bioinformatics 30, 1363–1369 (2014)." href="/articles/s41591-022-01927-8#ref-CR96" id="ref-link-section-d73061417e3391">96</a></sup>, were used to integrate and normalize DNA methylation data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Duran-Ferrer, M. et al. The proliferative history shapes the DNA methylome of B-cell tumors and predicts clinical outcome. Nat. Cancer 1, 1066–1081 (2020)." href="/articles/s41591-022-01927-8#ref-CR49" id="ref-link-section-d73061417e3395">49</a></sup>. We removed non-CpG probes, CpGs representing single nucleotide polymorphisms, CpGs with individual-specific methylation previously reported in B cells, CpGs in sex chromosomes and CpGs with a detection <i>P</i> value &gt;0.01 in &gt;10% of the samples. The data were normalized using the SWAN algorithm and CpGs were annotated using the IlluminaHumanMethylationEPICanno.ilm10b4.hg19 package (v.0.6). Tumor cell content of each sample was inferred from DNA methylation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Duran-Ferrer, M. et al. The proliferative history shapes the DNA methylome of B-cell tumors and predicts clinical outcome. Nat. Cancer 1, 1066–1081 (2020)." href="/articles/s41591-022-01927-8#ref-CR49" id="ref-link-section-d73061417e3402">49</a></sup> and samples with a tumor cell content &lt;60% were excluded. After all filtering criteria, we retained 33 samples (NBCs, <i>n</i> = 2; GCs, <i>n</i> = 1; MBCs, <i>n</i> = 3; tPCs, <i>n</i> = 1; CLL controls, <i>n</i> = 12; CLL/RT samples, <i>n</i> = 14 (six patients); Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">6</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec35">Differential analyses, CLL epitypes and epiCMIT</h4><p>We compared the DNA methylation status of each CpG to the mean of such CpGs in NBCs to calculate the number of hyper- and hypomethylation changes per CLL/RT sample. Changes in each sample were defined based on a minimum difference of 0.25 methylation. To perform a differential analysis between CLL and RT, we compared the DNA methylation of each CpG in each CLL sample (first available time point used) versus their respective RT sample. Differentially methylated CpGs were considered as those showing a minimum difference of 0.25 in at least four of the five longitudinal cases of RT versus CLL analyzed (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">6</a>). The epigenetic subtypes (epitypes) and epiCMIT score for each CLL and RT sample were calculated<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Duran-Ferrer, M. et al. The proliferative history shapes the DNA methylome of B-cell tumors and predicts clinical outcome. Nat. Cancer 1, 1066–1081 (2020)." href="/articles/s41591-022-01927-8#ref-CR49" id="ref-link-section-d73061417e3440">49</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec36">ChIP-seq of H3K27ac and ATAC-seq</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec37">Data generation</h4><p>ChIP-seq of H3K27ac and ATAC-seq data were generated as described in <a href="http://www.blueprint-epigenome.eu/index.cfm?p=7BF8A4B6-F4FE-861A-2AD57A08D63D0B58">http://www.blueprint-epigenome.eu/index.cfm?p=7BF8A4B6-F4FE-861A-2AD57A08D63D0B58</a> (antibody anti H3K27ac, Diagenode, cat. no. C15410196/pAb-196-050, lot A1723-0041D; Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01927-8#Sec63">7</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">8</a>). Libraries were sequenced on Illumina machines aiming at 60 million reads/sample (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01927-8#Sec63">7</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">8</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec38">Read mapping and initial data processing</h4><p>FASTQ files were aligned to the reference genome (GRCh38) using BWA-ALN (v.0.7.7, parameter: -q 5)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009)." href="/articles/s41591-022-01927-8#ref-CR61" id="ref-link-section-d73061417e3484">61</a></sup>, duplicated reads were marked using Picard tools (v.2.8.1) and low-quality and duplicated reads were removed using SAMtools (v.1.3.1, parameters: -b -F 4 -q 5 -b -F 1,024)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Danecek, P. et al. Twelve years of SAMtools and BCFtools. Gigascience 
                  https://doi.org/10.1093/gigascience/giab008
                  
                 (2021)." href="/articles/s41591-022-01927-8#ref-CR67" id="ref-link-section-d73061417e3488">67</a></sup>. PhantomPeakQualTools (v.1.1.0) were used to generate wiggle plots and for extracting the predominant insert-size. Peaks were called using MACS2 (v.2.1.1.20160309, parameters for H3K27ac: -g hs -q 0.05 -keep-dup all -nomodel -extsize insert-size; parameters for ATAC-seq: -g hs -q 0.05–keep-dup all -f BAM –nomodel –shift −96 –extsize 200; no input control)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). Genome Biol. 9, R137 (2008)." href="/articles/s41591-022-01927-8#ref-CR97" id="ref-link-section-d73061417e3492">97</a></sup>. Peaks with <i>q</i> values &lt;1 × 10<sup>−3</sup> were included for downstream analyses. For each mark separately, a set of consensus peaks, including regions within chromosomes 1–22 and present in published healthy B cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Beekman, R. et al. The reference epigenome and regulatory chromatin landscape of chronic lymphocytic leukemia. Nat. Med. 24, 868–880 (2018)." href="/articles/s41591-022-01927-8#ref-CR44" id="ref-link-section-d73061417e3502">44</a></sup> and CLL samples was generated by merging the locations of the separate peaks per individual sample. For ChIP-seq, the numbers of reads per sample per consensus peak were calculated using the genomecov function (bedtools, v.2.25.0). For ATAC-seq, the number of Tn5 transposase insertions per sample per consensus peak was calculated by first determining the estimated insertion sites (shifting the start of the first mate 4 bp downstream) before using the genomecov function. Variance stabilizing transformation (VST) values were calculated for all consensus peaks using DESeq2 (v.1.28.1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)." href="/articles/s41591-022-01927-8#ref-CR98" id="ref-link-section-d73061417e3506">98</a></sup>, which were then corrected for the consensus SPOT score (the percentage of reads that fall within the consensus peaks) using the ComBat function (sva R package, v.3.36.0). To that purpose, the cell condition (tumor and different healthy B cell subtypes) was assigned to each sample and samples were clustered in 20 bins of 5% according to their consensus SPOT score. The bins on the extremes, which contained fewer than five samples, were joined with their neighboring bins to ensure that each bin contained five samples or more. PCA was generated using the corrected VST values of peaks that were present in more than one sample.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec39">Detection of differential epigenetic regions and RT-specific changes</h4><p>We first determined the regions with stable epigenetic profiles in the healthy B cell counterparts (NBCs and MBCs) by applying a threshold of s.d. &lt; 0.8 with respect to the mean value. For all these NBC/MBC stable regions, we then calculated the log<sub>2</sub>FC between the mean of VST-corrected healthy B cell values and each of the tumor samples. Due to the data distribution variability, we applied slightly different thresholds of log<sub>2</sub>FC for each case (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01927-8#Sec63">7</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">8</a>). To identify regions changing in RT for each case individually, we selected the regions that presented substantial epigenetic changes as compared to the normal counterpart and to the previous CLL (absolute log<sub>2</sub>FC &gt; 1). The ATAC-seq RT-specific signature encompassed differential regions common in two or more cases of RT, whereas the H3K27ac RT-specific signature included differential regions common in three or more cases. Potential protein-coding target genes were assigned to each of the RT-specific regions using two strategies. To identify close target genes, we took the overlap with the regions of genes of interest adding 2 kb upstream of their transcription start site. To identify distant target genes, we used Hi-C data from the GM12878 cell line and selected all genes located within the same topologically associated domain as the region of interest. We only considered DEGs identified by bulk RNA-seq (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01927-8#Sec63">7</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">8</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec40">Transcription factor analysis</h4><p>Enrichment for TF-binding sites was analyzed in chromatin accessible regions within the RT-specific active chromatin regions. Accessible peaks were determined as regions with presence of ATAC peaks in two or more RT cases. Enrichment analysis of known TF-binding motifs was performed using the AME tool (MEME suite) considering the non-redundant <i>Homo sapiens</i> 2020 Jaspar database and applying one-tailed Wilcoxon rank-sum tests with the maximum score of the sequence, a 0.01 FDR cutoff and a background formed by reference GRCh38 sequences extracted from the consensus ATAC-seq peaks (91,671 regions). We then established the occupancy of these motifs in RT and CLL by calculating the percentage of the target RT-specific active regions and of the regions with increased H3K27ac in CLL, respectively, which contained these motifs. Finally, we selected TFs presenting an occupancy difference between RT and CLL ≥ 10% and overexpressed in RT (bulk RNA-seq, log<sub>2</sub>FC &gt; 0, adjusted <i>P</i> value &lt;0.01).</p><h3 class="c-article__sub-heading" id="Sec41">Bulk RNA-seq</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec42">Data generation</h4><p>Bulk RNA-seq data of six patients with paired CLL and RT samples were analyzed. Libraries were prepared using the TruSeq Stranded mRNA Library Prep kit (Illumina, cat. no. 20020595) or the Stranded mRNA Library Prep, Ligation kit (Illumina, cat. no. 20040534) and sequenced on a HiSeq 4000 (2 × 76 bp, Illumina) or NextSeq 2000 (2 × 100 bp, Illumina). All samples had a tumor purity ≥92% as assessed by flow cytometry (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">11</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec43">Data analysis</h4><p>Ribosomal RNA reads were filter out using SortMeRNA (v.4.3.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Kopylova, E., Noé, L. &amp; Touzet, H. SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data. Bioinformatics 28, 3211–3217 (2012)." href="/articles/s41591-022-01927-8#ref-CR99" id="ref-link-section-d73061417e3577">99</a></sup>. Non-ribosomal reads were trimmed using Trimmomatic (v.0.38)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120 (2014)." href="/articles/s41591-022-01927-8#ref-CR95" id="ref-link-section-d73061417e3581">95</a></sup>. Gene-level counts (GRCh38.p13, Ensembl release 100) were calculated using kallisto (v.0.46.1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 100" title="Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. Nat. Biotechnol. 34, 525–527 (2016)." href="/articles/s41591-022-01927-8#ref-CR100" id="ref-link-section-d73061417e3585">100</a></sup> and tximport (v.1.14.2). A paired DEA was conducted using DESeq2 (v.1.26.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. Genome Biol. 15, 550 (2014)." href="/articles/s41591-022-01927-8#ref-CR98" id="ref-link-section-d73061417e3589">98</a></sup>. Adjusted <i>P</i> value &lt;0.01 and absolute log<sub>2</sub>(fold change) &gt; 1 were used to identify DEGs. Gene set enrichment analysis (GSEA) was conducted using a pre-ranked gene list ordered by −log<sub>10</sub>(<i>P</i>) × (sign of fold change) using the ‘GSEA’ function (clusterProfiler R package, v.3.14.3). We focused on C2 (curated) and Hallmark gene sets from the Molecular Signatures Database (v.7.4) with a minimal size of 10 and maximal size of 250. Gene ontology (GO) GSEA was conducted using the pre-ranked gene list as input of the ‘gseGO’ function (clusterProfiler) focusing on biological processes. Redundancy in the output list of GO terms was removed using the ‘simplify’ function (cutoff of 0.35).</p><h3 class="c-article__sub-heading" id="Sec44">Single-cell DNA-seq</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec45">Data generation</h4><p>scDNA-seq was performed for 16 samples of 4 patients using the Tapestri Platform (Mission Bio, cat. no. 191335) and a commercial 32-gene panel (Tapestri single-cell DNA CLL panel, Mission Bio, cat. no. MB53-0011_J01). Cryopreserved cells were thawed on 5 ml of fetal bovine serum (FBS; Fisher Scientific, cat. no. 10082147) and incubated at 37 °C for 5 min. Then, cells were washed twice with 1 ml phosphate buffered saline (PBS; Thermo Fisher, cat. no. 20012-019) with 4% bovine serum albumin (BSA; Miltenyi Biotec, cat. no. 130-091-376) and centrifuged at 400<i>g</i> for 4 min. Cell concentration and viability were verified by counting with a TC20T Automated Cell Counter (Bio-Rad Laboratories, cat. no. 1450102). After a final centrifugation step, supernatant was removed and cells were resuspended in an appropriate volume of Mission Bio cell buffer to obtain a final cell density of 3,000–4,000 cells μl<sup>−1</sup>. Encapsulation, lysis and barcoding of cells were performed following the exact manufacturer’s instructions. Afterwards, PCR products were digested and cleaned up with AMPure XP Reagent (Beckman Coulter, cat. no. 100-265-900), followed by quantification of PCR products using a High-Sensitivity dsDNA 1× Qubit kit (Qubit, Invitrogen, cat. no. Q32851). Final library preparation consisted of a Target Library PCR with the V2 Index Primer for ten cycles and a library cleanup with AMPure XP Reagent (Beckman Coulter). Quality control and final quantification were performed on an Agilent Bioanalyzer High Sensitivity chip (Agilent Technologies, cat. no. 5067-4626). Libraries were sequenced on a NovaSeq 6000 instrument (Illumina) aiming for 1,300 reads per cell (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">20</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec46">Data analysis</h4><p>FASTQ files were analyzed through the Tapestri Pipeline (v.1, Mission Bio), which trims adaptor sequences, aligns reads to the human genome (hg19) using BWA aligner, performs barcode correction, assigns sequence reads to cell barcodes and performs genotype calling using GATK (v.3.7). Loom files generated were analyzed using the Tapestri Insights (v.2.2, Mission Bio). For each patient (considering all time points together), genotypes with quality &lt;30, read depth &lt;10 or allele frequency &lt;20% were marked as missing. Similarly, for each patient, variants genotyped in &lt;50% of the cells or mutated in &lt;1% of the cells were removed. Cells with &lt;50% of genotypes present were removed. Mutations identified in bulk WGS analysis were used as a whitelist. A list of variants not identified in COSMIC and present at low frequency (1–10% of cells) in all samples analyzed by scDNA-seq was used to remove potential artifacts. The analysis was restricted to coding and splice-site mutations. Genotypes of the selected mutations were exported from Tapestri Insights and used as input of ∞SCITE (<a href="https://github.com/cbg-ethz/infSCITE">https://github.com/cbg-ethz/infSCITE</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 101" title="Kuipers, J., Jahn, K., Raphael, B. J. &amp; Beerenwinkel, N. Single-cell sequencing data reveal widespread recurrence and loss of mutational hits in the life histories of tumors. Genome Res. 27, 1885–1894 (2017)." href="/articles/s41591-022-01927-8#ref-CR101" id="ref-link-section-d73061417e3640">101</a></sup>. Genotypes were encoded as zero for wild-type, one for heterozygous mutation, two for homozygous mutation and three for missing data. ∞SCITE was used to find the mutation tree that best fitted the genotypes observed and to assign cells into subclones. ∞SCITE was run using a global sequencing error rate (false-positive rate) of 1%<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 102" title="Morita, K. et al. Clonal evolution of acute myeloid leukemia revealed by high-throughput single-cell genomics. Nat. Commun. 11, 5327 (2020)." href="/articles/s41591-022-01927-8#ref-CR102" id="ref-link-section-d73061417e3644">102</a></sup>, an estimated rate of non-mutated sites called as homozygous mutations of 0% and a patient-specific estimated rate of the allele dropout rate (false-negative rate). For each patient, the estimated rate of missed heterozygous mutations (dropout of the mutated allele) and the estimated rate of heterozygous mutations called as homozygous mutations (dropout of the normal allele) were calculated from germline single-nucleotide polymorphisms reported in gnomAD with a population frequency &gt;1% and called as mutated in at least 75% of cells with a VAF per read count between 47% and 53% according to Tapestri Insights. Patient-specific allele dropout rates were calculated for all patients except for patient 365, which did not have any heterozygous polymorphisms fulfilling the previous criteria. In this case, we used an allele dropout rate of 0.07, which is within the range measured in the other cases. We ran ∞SCITE with and without considering <i>NOTCH1</i> mutations and manually curated the result of patient 3,299 carrying an <i>RPS15</i> mutation due to the high allele dropout rate observed in these genes (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM1">2</a>). We ran ∞SCITE for each patient combining all time points and obtained time-point-specific subclone sizes by counting the cells assigned to each subclone in each sample<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 102" title="Morita, K. et al. Clonal evolution of acute myeloid leukemia revealed by high-throughput single-cell genomics. Nat. Commun. 11, 5327 (2020)." href="/articles/s41591-022-01927-8#ref-CR102" id="ref-link-section-d73061417e3658">102</a></sup>. Only cells uniquely assigned to one subclone were considered. Cells genotyped as wild-type for all selected mutations were considered as non-tumoral cells and were removed.</p><h3 class="c-article__sub-heading" id="Sec47">Single-cell RNA-seq</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec48">Data generation</h4><p>scRNA-seq was performed on longitudinal samples of five patients using three different approaches:</p><ol class="u-list-style-none">
                    <li>
                      <span class="u-custom-list-number">1.</span>
                      
                        <p>Smart-seq2: full-length scRNA-seq libraries were prepared for samples of patient 63 using the Smart-seq2 protocol<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 103" title="Picelli, S. et al. Smart-seq2 for sensitive full-length transcriptome profiling in single cells. Nat. Methods 10, 1096–1098 (2013)." href="/articles/s41591-022-01927-8#ref-CR103" id="ref-link-section-d73061417e3685">103</a></sup> with minor modifications. Single cells were sorted into 96-well plates containing the lysis buffer (0.2% Triton-100, 1 U μl<sup>−1</sup> RNase inhibitor; Applied Biosystems, cat. no. N8080119). Reverse transcription was performed using SuperScript II (Thermo Fisher Scientific, cat. no. 18064014) in the presence of 1 μM oligo-dT30VN (IDT, cat. no. 22859789), 1 μM template-switching oligonucleotides (QIAGEN, cat. no. PER-YCO0075516) and 1 M betaine (Merck, cat. no. W422312-5KG-K). cDNA was amplified using the KAPA Hifi Hotstart ReadyMix (Kapa Biosystems, cat. no. 7958935001) and IS PCR primer (IDT, cat. no. 228597989), with 25 cycles of amplification. Following purification with Agencourt Ampure XP beads (Beckmann Coulter), product size distribution and quantity were assessed on a Bioanalyzer using a High Sensitivity DNA kit (Agilent Technologies). A total of 140 pg of the amplified cDNA was fragmented using Nextera XT (Illumina, cat. no. FC-131-1096) and amplified with Nextera XT indexes (Illumina, cat. no. 20027215). Products of each well of the 96-well plate were pooled and purified twice with Agencourt Ampure XP beads (Beckmann Coulter). Pooled sequencing was performed on a HiSeq 4000 (2x75bp, Illumina) to an average depth of 0.5 million reads per cell.</p>
                      
                    </li>
                    <li>
                      <span class="u-custom-list-number">2.</span>
                      
                        <p>Cell hashing experiment and 10x Genomics: For each patient (12, 19, 365 and 3,299, experiment BCLLATLAS_10), samples obtained at different time points of the disease were labeled following a cell hashing protocol<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 104" title="Stoeckius, M. et al. Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics. Genome Biol. 19, 224 (2018)." href="/articles/s41591-022-01927-8#ref-CR104" id="ref-link-section-d73061417e3702">104</a></sup>. For each sample, 1–2 million cells were resuspended in 100 μl of cell staining buffer (BioLegend, cat. no. 420201) and incubated for 10 min at 4 °C with 5 μl of Human TruStain FcX Fc Blocking reagent (BioLegend, cat. no. 422302). Next, a specific TotalSeq-A antibody-oligo conjugate (BioLegend, TotalSeq-A anti-human Hashtag 1–8, cat. no. 394601, 394603, 394605, 394607, 394609, 394611, 394613 and 394615) was added and incubated on ice for 30 min. Cells were then washed three times with cold PBS-0.05% BSA and centrifuged for 5 min at 500<i>g</i> at 4 °C. Finally, cells were resuspended in an appropriate volume of 1× PBS-0.05% BSA to obtain a final cell concentration of 500–1,000 cells μl<sup>−1</sup>, suitable for 10x Genomics scRNA-seq. An equal volume of hashed cell suspension from each of the conditions was mixed and filtered with a 40-µm strainer (pluriSelect, cat. no. 43-10040-70). Cell concentration was verified by counting with a TC20 Automated Cell Counter (Bio-Rad Laboratories, cat. no. 1450102). Cells were partitioned into Gel Bead In Emulsions with a Target Cell Recovery of 10,000 total cells. Sequencing libraries were prepared using the Chromium Next GEM Single Cell 3′ GEM, Library &amp; Gel Bead kit v.3.1 (10x Genomics, cat. no. 1000121) with some adaptations for cell hashing, as indicated in TotalSeq-A Antibodies and Cell Hashing with 10x Single Cell 3′ Reagent kit v.3.1 Protocol by BioLegend. Briefly, 1 µl of 0.2 µM HTO primer (IDT, Hashtag Oligonucleotides; GTGACTGGAGTTCAGACGTGTGC*T*C; *phosphorothioate bond) was added to the cDNA amplification reaction to amplify the hashtag oligonucleotides together with the full-length cDNAs. An SPRI selection cleanup was performed to separate messenger RNA-derived cDNA (&gt;300 bp) from antibody-oligonucleotide-derived cDNA (&lt;180 bp), as described in the above-mentioned protocol. 10x cDNA sequencing libraries were prepared following 10x Genomics Single Cell 3′ v.3.1 mRNA kit protocol, whereas HTO cDNAs were indexed by PCR as follows: 5 µl of purified hashtag oligonucleotide cDNA were mixed with 2.5 µl of 10 µM Illumina TruSeq D70X_s primer (IDT) carrying a different i7 index for each sample, 2.5 µl of SI primer (10x Genomics, cat. no. 2000095), 50 µl of 2× KAPA Hifi Hotstart ReadyMix (Kapa Biosystems, cat. no. 7958935001) and 40 µl of nuclease-free water. HTO libraries were purified with 1.2× SPRI bead selection. Size distribution and concentrations of cDNA and HTO libraries were verified on an Agilent Bioanalyzer High Sensitivity chip (Agilent Technologies, cat. no. 5067-4626). Finally, HTO and cDNA libraries were sequenced on a NovaSeq 6000 (Illumina) to obtain approximately 25,000 reads per cell.</p>
                      
                    </li>
                    <li>
                      <span class="u-custom-list-number">3.</span>
                      
                        <p>Non-cell hashing experiment and 10x Genomics. Samples with a low number of cells in the previous experiment (samples of patient 365 and a subset of samples of patients 12 and 19) were analyzed using a non-cell hashing experiment (BCLLATLAS_29). Frozen samples were thawed and 1 ml of 37 °C pre-warmed Hibernate-E (Thermo Fisher Scientific, cat. no. A1247601) supplemented with 10% FBS (Thermo Fisher Scientific, cat. no 10082147) was added drop-wise with gently swirling of the sample. After 1 min of incubation at room temperature, 2,000 µl of pre-warmed medium was added as mentioned before. Samples were again kept at room temperature for 1 min and 5,000 µl pre-warmed medium was gently added. This step was conducted twice. Afterwards, samples were centrifuged at 500<i>g</i> for 5 min. Supernatant was removed and pellets were resuspended in 500 µl 1× PBS supplemented with 0.05% BSA and stained with 4,6-diamidino-2-phenylindole (DAPI) (Thermo Fisher Scientific, cat. no. D1306) at 1 µM final concentration. DAPI-negative live individual cells were sorted with a BD FACSAria Fusion Flow cytometer (BD Biosciences) in 1× PBS supplemented with 0.05% BSA. After FACS, cells were partitioned into Gel Bead In Emulsions by using the Chromium Controller system (10x Genomics, cat. no. 1000204) aiming at a Target Cell Recovery of 5,000 total cells. Sequencing libraries were prepared using the v.3.1 single-cell 3′ mRNA kit (10x Genomics). After GEM-RT cleanup, cDNAs were amplified during 14 cycles. cDNA quality control and quantification were performed on an Agilent Bioanalyzer High Sensitivity chip (Agilent Technologies). Libraries were indexed by PCR using the Chromiumi7 Sample Index Plate (10x Genomics, cat. no. 220103). Size distribution and concentration were verified on an Agilent Bioanalyzer High Sensitivity chip (Agilent Technologies, cat. no. 5067-4626). Finally, libraries were sequenced on a NovaSeq 6000 sequencer aiming for 40,000 reads per cell.</p>
                      
                    </li>
                  </ol><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec49">Read alignment</h4><p>Raw reads were aligned to the GRCh38 human genome with Cell Ranger (v.4.0.0), with the ‘chemistry’ parameter set to ‘SC3Pv3’ and the ‘expect-cells’ parameter set to 20,000 and 5,000 for cell-hashed and non-hashed libraries, respectively. The remaining parameters for cell-hashed libraries were specified as described in the ‘Feature Barcode Analysis’ pipeline of Cell Ranger. For Smart-seq2 libraries, alignment and quantification was performed using zUMIs (v.9.4e)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 105" title="Parekh, S., Ziegenhain, C., Vieth, B., Enard, W. &amp; Hellmann, I. zUMIs - A fast and flexible pipeline to process RNA sequencing data with UMIs. Gigascience 
                  https://doi.org/10.1093/gigascience/giy059
                  
                 (2018)." href="/articles/s41591-022-01927-8#ref-CR105" id="ref-link-section-d73061417e3736">105</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec50">Demultiplexing of hashtag oligonucleotides</h4><p>Expression matrices were imported into R (v.4.0.4) with the ‘Read10X’ function from Seurat (v.4.0.3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 106" title="Hao, Y. et al. Integrated analysis of multimodal single-cell data. Cell 184, 3573–3587 (2021)." href="/articles/s41591-022-01927-8#ref-CR106" id="ref-link-section-d73061417e3748">106</a></sup>. HTO counts were normalized with a centered log-ratio transformation applied across features. Each cell barcode was assigned to a specific time point of the disease with the function ‘HTODemux’ (positive.quantile = 0.99) of Seurat. Barcodes that were positive for two or more time points were labeled as doublets and discarded. Likewise, cell barcodes negative for all time points were excluded. Finally, Scrublet (v.0.2.1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 107" title="Wolock, S. L., Lopez, R. &amp; Klein, A. M. Scrublet: computational identification of cell doublets in single-cell transcriptomic data. Cell Syst. 8, 281–291 (2019)." href="/articles/s41591-022-01927-8#ref-CR107" id="ref-link-section-d73061417e3752">107</a></sup> was run to aid in the detection of doublets.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec51">Quality control, normalization and dimensionality reduction</h4><p>Cells that possessed &lt;900 UMIs, &lt;250 expressed genes or a mitochondrial expression &gt;22.5% were considered as poor quality and removed. Similarly, genes expressed in three or fewer cells were filtered out. Following data normalization and correction (Seurat and NormalizeData), we performed PCA (Seurat, RunPCA) using the scaled expression (Seurat and ScaleData) of the top 2,000 highly variable genes (Seurat: FindVariableFeatures, selection.method = VST). For Smart-seq2 data, we filtered out cells with &lt;150,000 counts, &lt;550 expressed genes or mitochondrial expression &gt;18%. Cells with more than 700,000 counts or 3,750 detected genes were excluded. Similarly, genes expressed in three or fewer cells were filtered out. To separate neoplastic cells from the microenvironment, we corrected the top 30 principal components (PCs) for sample-specific variation using Harmony (v.1.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 108" title="Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Nat. Methods 16, 1289–1296 (2019)." href="/articles/s41591-022-01927-8#ref-CR108" id="ref-link-section-d73061417e3764">108</a></sup>, as implemented in the RunHarmony (group.by.vars = sample) function (SeuratWrappers package, v.0.3.0). Subsequently, these 30 corrected PCs were used to embed cells in a UMAP (Seurat, RunUMAP) and in a 20-nearest neighbors graph (Seurat, FindNeighbors) for visualization and clustering, respectively. Following Louvain clustering (Seurat, FindClusters, resolution = 0.1), we focused our downstream analyses only on tumor B cells (CD79A) due to the low number of microenvironment cells.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec52">Dealing with confounders</h4><p>We observed batch effects between 10x Genomics experiments. To avoid batch effects within samples of the same patient, we focused on the BCLLATLAS_10 experiment for patients 12, 19 and 3,299. Conversely, as we did not obtain a clear signal-to-noise separation in the HTO demultiplexing of case 365, we analyzed the cells obtained with BCLLATLAS_29. We also found some cell neighborhoods that harbored a high percentage of mitochondrial expression and a low number of detected genes. In such cases, we were more stringent with the thresholds or fetched and eliminated these clusters with FindClusters. We also excluded some clusters of doublets that expressed markers of microenvironment cells (erythroblasts, T cells or natural killer cells). Finally, for patient 3,299 in which one sample was obtained from peripheral blood (PB), whereas the others were obtained from bone marrow (BM), we focused solely on the BM samples to avoid misinterpretations. For patient 365, the CLL and RT time points were sampled from PB and lymph nodes, respectively. As the same RT sample profiled with bulk RNA-seq clustered with other RT samples from PB, we analyzed them jointly. After all the filtering, we recomputed the highly variable genes and PCAs. To avoid overcorrection, we used the top 20 PCs as input to RunUMAP and FindNeighbors, without rerunning Harmony.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec53">Clustering and annotation</h4><p>Louvain clustering was performed with the FindClusters function, adjusting the resolution parameter for each patient independently. To annotate each cluster, we ran a ‘one-versus-all’ DEA for each cluster (Seurat, FindAllMarkers, Wilcoxon rank-sum test), keeping only upregulated genes with a log<sub>2</sub>FC &gt; 0.3 and a Bonferroni-adjusted <i>P</i> value &lt;0.001. If markers were specific to a subset of the cluster, we further stratified it with the FindSubCluster function. On the contrary, if two clusters possessed similar markers, we merged them. The CellCycleScoring function was used to identify clusters of cycling cells.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec54">DEA and GSEA</h4><p>We conducted a DEA between RT and CLL clusters of each patient independently, merging cells from all time points (Seurat, FindMarkers, logfc.threshold = 0, only.pos = FALSE, Wilcoxon rank-sum test). To find finer-grained gene expression changes, only nonproliferative clusters were considered. Genes with a Bonferroni-adjusted <i>P</i> value &lt;0.05 were considered as significant. The resulting list of genes (sorted by decreasing log<sub>2</sub>FC) was used as input to the ‘gseGO’ function of clusterProfiler (v.3.18.1, parameters: ont = ‘BP’, OrgDB = org.Hs.eg.db, keyType = ‘SYMBOL’, minGSSize = 10, maxGSSize = 250, seed = TRUE). We then removed redundancy in the output list of GO terms with the ‘simplify’ function (cutoff of 0.75) and filtered out GO terms with an adjusted <i>P</i> value &lt;0.05. To convert the expression of specific GO terms of interest into a cell-specific score, we utilized the AddModuleScore function from Seurat.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec55">CNA inference from scRNA-seq data</h4><p>For each patient separately, we ran inferCNV (v.1.11.1) integrating all samples together. We used CLL cells as reference because (1) we aimed to identify CNAs acquired at RT and (2) CLL had flat copy number profiles in virtually all chromosomes according to WGS. CLL cells were downsampled to the number of RT cells. We initialized an ‘infercnv’ object (CreateInfercnvObject) using the raw expression counts and the gene-ordering file <a href="https://data.broadinstitute.org/Trinity/CTAT/cnv/gencode_v21_gen_pos.complete.txt">https://data.broadinstitute.org/Trinity/CTAT/cnv/gencode_v21_gen_pos.complete.txt</a>. CNAs were predicted (infercnv, run, HMM = FALSE, denoise = FALSE) setting the cutoff parameter to 1 and 0.1 for Smart-seq2 and 10x data, respectively. We customized the plotting with the plot_cnv function.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec56">Analysis of an external scRNA-seq dataset</h4><p>We downloaded the expression matrices and metadata of the dataset from Penter et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Penter, L. et al. Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history. Cancer Discov. 11, 3048–3063 (2021)." href="/articles/s41591-022-01927-8#ref-CR43" id="ref-link-section-d73061417e3829">43</a></sup> with the GEOquery (v.2.62.2) (Gene Expression Omnibus identifier <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE165087">GSE165087</a>), created a single Seurat object with all cells from all samples and filtered poor-quality cells as specified in the original publication<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Penter, L. et al. Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history. Cancer Discov. 11, 3048–3063 (2021)." href="/articles/s41591-022-01927-8#ref-CR43" id="ref-link-section-d73061417e3840">43</a></sup>. Dimensionality reduction, DEA, GSEA and gene signature scoring were performed as described above.</p><h3 class="c-article__sub-heading" id="Sec57">Cellular respiration</h3><p>Cryopreserved cells were resuspended on RPMI-1640 (Gibco, cat. no. 21875034) with 10% FBS (Gibco, cat. no. 10270-106) and 1% Glutamax (Gibco, cat. no. 35050-061) at a concentration of 3 million cells ml<sup>−1</sup>. After 1 h of incubation at 37 °C, cellular respiration was performed using O<sub>2</sub>k-respirometers (Oroboros Instruments). Two milliliters of cell suspension were added in each respirometer chamber. Cellular respiration was performed at 37 °C at a stirrer speed of 750 r.p.m. Respiratory control was studied by sequential determination of routine respiration (oxygen consumption in living cells resuspended on RPMI-1640 with 10% FBS and 1% Glutamax), oligomycin-inhibited leak respiration (2 µl ml<sup>−1</sup>, Sigma-Aldrich, cat. no. O4876, CAS, 1404-19-9), uncoupler-stimulated ETC measured by the sequential titration of the ionophore carbonyl cyanide m-chlorophenyl hydrazone (Sigma-Aldrich, cat. no. C2759, CAS, 555-60-2) and residual oxygen consumption after inhibition of the electron transfer system by the addition into the chamber of rotenone (0.5 µM, Sigma-Aldrich, cat. no. R8875, CAS, 83-79-4) and antimycin A (2.5 µM, Sigma-Aldrich, cat. no. A8674, CAS, 1397-94-0). Data acquisition and real-time analysis were performed using the software DatLab 7.4 (Oroboros Instruments). Automatic instrumental background corrections were applied for oxygen consumption by the polarographic oxygen sensor and oxygen diffusion into the chamber<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 109" title="Gnaiger, E., Steinlechner-Maran, R., Méndez, G., Eberl, T. &amp; Margreiter, R. Control of mitochondrial and cellular respiration by oxygen. J. Bioenerg. Biomembr. 27, 583–596 (1995)." href="/articles/s41591-022-01927-8#ref-CR109" id="ref-link-section-d73061417e3860">109</a></sup>. The same experimental workflow was used to study cellular respiration in CLL and RT cells after 1 h of treatment with IACS-010759 (Selleckchem, cat. no. S8731, CAS, 1570496-34-2) at 100 nM.</p><h3 class="c-article__sub-heading" id="Sec58">Calcium flux analysis</h3><p>Cryopreserved cells were resuspended on RPMI-1640 medium with 10% FBS, 1% Glutamax and 5% penicillin (10,000 IU ml<sup>−1)</sup>/streptomycin (10 mg ml<sup>−1</sup>) (Thermo Fisher, cat. no. S8731) at 10<sup>6</sup> cells ml<sup>−1</sup>. After 6 h of incubation at 37 °C and 5% CO<sub>2</sub>, cells were centrifuged and resuspended on RPMI-1640 with 4 µM Indo-1 AM (Thermo Fisher, cat. no. I1223) and 0.08% Pluronic F-127 (Thermo Fisher, cat. no. P3000MP) for 30 min at 37 °C and 5% CO<sub>2</sub>. Cells were subsequently labeled for 20 min at room temperature with surface marker antibodies CD19 (Super Bright 600; Invitrogen, cat. no. 63-0198-42) and CD5 (PE-Cy5; BD Biosciences, cat. no. 555354) for the identification of tumoral cells (CD19<sup>+</sup>CD5<sup>+</sup>). Next, cells were resuspended on RPMI-1640 before flow cytometry acquisition. Basal calcium was measured during 1 min before stimulation, then cells were incubated during 2 min at 37 °C with or without 10 µg ml<sup>−1</sup> anti-human F(ab′)2 IgM (Southern Biotech, cat. no. 2022-01) and 3.3 mM H<sub>2</sub>O<sub>2</sub> (Sigma-Aldrich, cat. no. H1009). Finally, 2 µM 4-hydroxytamoxifen (4-OHT) (Sigma-Aldrich, cat. no. H6278) was added to all conditions before continue recording for up to 8 min. Intracellular Ca<sup>2+</sup> release was measured on LSRFortessa (BD Biosciences) using BD FACSDiva software (v.8) by exciting with ultraviolet laser (355 nm) and appropriate filters: Indo-1 violet (450/50 nm) and Indo-1 blue (530/30 nm). Bound (Indo-1 violet) and unbound (Indo-1 blue) ratiometric was calculated with FlowJo software (v.10). Gating analysis was as follows: cell identification in FSC-A versus SSC-A plot, singlet identification in FSC-A versus FCS-H plot, tumoral cells (CD19<sup>+</sup>CD5<sup>+</sup>) in CD19 (Super Bright 600) versus CD5 (PE-Cy5) plot and Ca<sup>2+</sup> release in time versus Indo-1 violet/Indo-1 blue plot using a kinetics tool. Optimized dilutions for the antibodies were 1:3 for CD19 and 1:10 for CD5.</p><h3 class="c-article__sub-heading" id="Sec59">Cell growth assays</h3><p>Cryopreserved cells were resuspended on PBS at a concentration of 10<sup>7</sup> cells ml<sup>−1</sup> and labeled with 0.5 µM CFSE Cell Tracer (Thermo Fisher, cat. no. C34554) for 10 min. Cells were centrifuged and resuspended on enriched RPMI-1640 medium with 1% Glutamax, 15% FBS, 1× insulin-transferrin-selenium (Merk, cat. no. I3146), 10 mM HEPES (Fisher Scientific, cat. no. BP299), 50 µM 2-mercaptoethanol (Gibco, cat. no. 21985-023), 1× Non-Essential Amino Acids (Gibco, cat. no. 11140-050), 1 mM sodium pyruvate (Gibco, cat. no. 11360-070) and 50 µg ml<sup>−1</sup> gentamicin (Gibco, cat. no. 15710-064) at a concentration of 10<sup>6</sup> cells ml<sup>−1</sup> supplemented with 0.2 µM CpG DNA TLR9 ligand (ODN2006-TL9; InvivoGen, cat. no. TLRL-2006) and 15 ng ml<sup>−1</sup> recombinant human IL-15 (R&amp;D Systems, cat. no. 247-ILB-025)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 110" title="Mongini, P. K. A. et al. TLR-9 and IL-15 synergy promotes the in vitro clonal expansion of chronic lymphocytic leukemia B cells. J. Immunol. 195, 901–923 (2015)." href="/articles/s41591-022-01927-8#ref-CR110" id="ref-link-section-d73061417e3925">110</a></sup>. When indicated, cells were treated for 72 h with 100 nM IACS-010759. Cells were labeled for 20 min at room temperature with surface marker antibodies CD19 (Super Bright 600), CD5 (PE-Cy5) and annexin V (Life Technologies, cat. no. A35122) before acquisition in a LSRFortessa (BD Biosciences) using the BD FACSDiva software (v.8) and analyzed using FlowJo (v.10). Gating analysis for divided cells was as follows: cell identification in FSC-A versus SSC-A plot, singlet identification in FSC-A versus FCS-H plot, alive cells in annexin V (PacB) versus SSC-A plot, tumoral cells (CD19<sup>+</sup>CD5<sup>+</sup>) in CD19 (Super Bright 600) versus CD5 (PE-Cy5) plot and proliferating cells in the CFSE histogram. Optimized dilutions for the antibodies were 1:3 for CD19, 1:10 for CD5 and 1:3 for annexin V.</p><h3 class="c-article__sub-heading" id="Sec60">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>Sequencing data are available from the European Genome–phenome Archive (<a href="http://www.ebi.ac.uk/ega/">http://www.ebi.ac.uk/ega/</a>) under accession no. <a href="https://ega-archive.org/studies/EGAS00001006327">EGAS00001006327</a>. scRNA-seq expression matrices, Seurat objects and corresponding metadata are available at Zenodo (<a href="https://doi.org/10.5281/zenodo.6631966">https://doi.org/10.5281/zenodo.6631966</a>).</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>R markdown notebooks used for mutational signature, bulk RNA-seq, H3K27ac and ATAC-seq analyses can be found at <a href="https://github.com/ferrannadeu/RichterTransformation">https://github.com/ferrannadeu/RichterTransformation</a>. R markdown notebooks to reproduce the scRNA-seq analyses can be accessed at <a href="https://github.com/massonix/richter_transformation">https://github.com/massonix/richter_transformation</a>. Code to normalize DNA methylation data can be found at <a href="https://github.com/Duran-FerrerM/DNAmeth_arrays">https://github.com/Duran-FerrerM/DNAmeth_arrays</a>. Code to calculate the tumor cell content, CLL epitypes and epiCMIT from DNA methylation data can be found at <a href="https://github.com/Duran-FerrerM/Pan-B-cell-methylome">https://github.com/Duran-FerrerM/Pan-B-cell-methylome</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Cairns, J. Mutation selection and the natural history of cancer. <i>Nature</i> <b>255</b>, 197–200 (1975).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/255197a0" data-track-item_id="10.1038/255197a0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2F255197a0" aria-label="Article reference 1" data-doi="10.1038/255197a0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaE2MXkvFCmsb0%3D" aria-label="CAS reference 1">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1143315" aria-label="PubMed reference 1">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Mutation%20selection%20and%20the%20natural%20history%20of%20cancer&amp;journal=Nature&amp;doi=10.1038%2F255197a0&amp;volume=255&amp;pages=197-200&amp;publication_year=1975&amp;author=Cairns%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Nowell, P. C. The clonal evolution of tumor cell populations. <i>Science</i> <b>194</b>, 23–28 (1976).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.959840" data-track-item_id="10.1126/science.959840" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.959840" aria-label="Article reference 2" data-doi="10.1126/science.959840">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaE283ntFWmsw%3D%3D" aria-label="CAS reference 2">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=959840" aria-label="PubMed reference 2">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20clonal%20evolution%20of%20tumor%20cell%20populations&amp;journal=Science&amp;doi=10.1126%2Fscience.959840&amp;volume=194&amp;pages=23-28&amp;publication_year=1976&amp;author=Nowell%2CPC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Dentro, S. C. et al. Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes. <i>Cell</i> <b>184</b>, 2239–2254 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2021.03.009" data-track-item_id="10.1016/j.cell.2021.03.009" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2021.03.009" aria-label="Article reference 3" data-doi="10.1016/j.cell.2021.03.009">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXoslGkur8%3D" aria-label="CAS reference 3">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33831375" aria-label="PubMed reference 3">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8054914" aria-label="PubMed Central reference 3">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Characterizing%20genetic%20intra-tumor%20heterogeneity%20across%202%2C658%20human%20cancer%20genomes&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2021.03.009&amp;volume=184&amp;pages=2239-2254&amp;publication_year=2021&amp;author=Dentro%2CSC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Gerstung, M. et al. The evolutionary history of 2,658 cancers. <i>Nature</i> <b>578</b>, 122–128 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-019-1907-7" data-track-item_id="10.1038/s41586-019-1907-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-019-1907-7" aria-label="Article reference 4" data-doi="10.1038/s41586-019-1907-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsLw%3D" aria-label="CAS reference 4">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32025013" aria-label="PubMed reference 4">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054212" aria-label="PubMed Central reference 4">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20evolutionary%20history%20of%202%2C658%20cancers&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1907-7&amp;volume=578&amp;pages=122-128&amp;publication_year=2020&amp;author=Gerstung%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Ferrando, A. A. &amp; López-Otín, C. Clonal evolution in leukemia. <i>Nat. Med.</i> <b>23</b>, 1135–1145 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nm.4410" data-track-item_id="10.1038/nm.4410" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnm.4410" aria-label="Article reference 5" data-doi="10.1038/nm.4410">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1Wns7nL" aria-label="CAS reference 5">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28985206" aria-label="PubMed reference 5">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20in%20leukemia&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fnm.4410&amp;volume=23&amp;pages=1135-1145&amp;publication_year=2017&amp;author=Ferrando%2CAA&amp;author=L%C3%B3pez-Ot%C3%ADn%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Ding, W. Richter transformation in the era of novel agents. <i>Hematology</i> <b>2018</b>, 256–263 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/asheducation-2018.1.256" data-track-item_id="10.1182/asheducation-2018.1.256" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fasheducation-2018.1.256" aria-label="Article reference 6" data-doi="10.1182/asheducation-2018.1.256">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30504319" aria-label="PubMed reference 6">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6245983" aria-label="PubMed Central reference 6">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Richter%20transformation%20in%20the%20era%20of%20novel%20agents&amp;journal=Hematology&amp;doi=10.1182%2Fasheducation-2018.1.256&amp;volume=2018&amp;pages=256-263&amp;publication_year=2018&amp;author=Ding%2CW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Maddocks, K. J. et al. Etiology of ibrutinib therapy discontinuation and outcomes in patients with chronic lymphocytic leukemia. <i>JAMA Oncol.</i> <b>1</b>, 80 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1001/jamaoncol.2014.218" data-track-item_id="10.1001/jamaoncol.2014.218" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1001%2Fjamaoncol.2014.218" aria-label="Article reference 7" data-doi="10.1001/jamaoncol.2014.218">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26182309" aria-label="PubMed reference 7">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4520535" aria-label="PubMed Central reference 7">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Etiology%20of%20ibrutinib%20therapy%20discontinuation%20and%20outcomes%20in%20patients%20with%20chronic%20lymphocytic%20leukemia&amp;journal=JAMA%20Oncol.&amp;doi=10.1001%2Fjamaoncol.2014.218&amp;volume=1&amp;publication_year=2015&amp;author=Maddocks%2CKJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Ahn, I. E. et al. Clonal evolution leading to ibrutinib resistance in chronic lymphocytic leukemia. <i>Blood</i> <b>129</b>, 1469–1479 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood-2016-06-719294" data-track-item_id="10.1182/blood-2016-06-719294" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood-2016-06-719294" aria-label="Article reference 8" data-doi="10.1182/blood-2016-06-719294">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVejs7%2FP" aria-label="CAS reference 8">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28049639" aria-label="PubMed reference 8">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5356450" aria-label="PubMed Central reference 8">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20leading%20to%20ibrutinib%20resistance%20in%20chronic%20lymphocytic%20leukemia&amp;journal=Blood&amp;doi=10.1182%2Fblood-2016-06-719294&amp;volume=129&amp;pages=1469-1479&amp;publication_year=2017&amp;author=Ahn%2CIE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Jain, P. et al. Outcomes of patients with chronic lymphocytic leukemia after discontinuing ibrutinib. <i>Blood</i> <b>125</b>, 2062–2067 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood-2014-09-603670" data-track-item_id="10.1182/blood-2014-09-603670" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood-2014-09-603670" aria-label="Article reference 9" data-doi="10.1182/blood-2014-09-603670">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXmtV2mtb8%3D" aria-label="CAS reference 9">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25573991" aria-label="PubMed reference 9">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4467871" aria-label="PubMed Central reference 9">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Outcomes%20of%20patients%20with%20chronic%20lymphocytic%20leukemia%20after%20discontinuing%20ibrutinib&amp;journal=Blood&amp;doi=10.1182%2Fblood-2014-09-603670&amp;volume=125&amp;pages=2062-2067&amp;publication_year=2015&amp;author=Jain%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Landau, D. A. et al. The evolutionary landscape of chronic lymphocytic leukemia treated with ibrutinib targeted therapy. <i>Nat. Commun.</i> <b>8</b>, 2185 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-017-02329-y" data-track-item_id="10.1038/s41467-017-02329-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-017-02329-y" aria-label="Article reference 10" data-doi="10.1038/s41467-017-02329-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29259203" aria-label="PubMed reference 10">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5736707" aria-label="PubMed Central reference 10">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ntrjJ" aria-label="CAS reference 10">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20evolutionary%20landscape%20of%20chronic%20lymphocytic%20leukemia%20treated%20with%20ibrutinib%20targeted%20therapy&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-02329-y&amp;volume=8&amp;publication_year=2017&amp;author=Landau%2CDA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Beà, S. et al. Genetic imbalances in progressed B-cell chronic lymphocytic leukemia and transformed large-cell lymphoma (Richter’s syndrome). <i>Am. J. Pathol.</i> <b>161</b>, 957–968 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0002-9440(10)64256-3" data-track-item_id="10.1016/S0002-9440(10)64256-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0002-9440%2810%2964256-3" aria-label="Article reference 11" data-doi="10.1016/S0002-9440(10)64256-3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12213724" aria-label="PubMed reference 11">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1867253" aria-label="PubMed Central reference 11">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20imbalances%20in%20progressed%20B-cell%20chronic%20lymphocytic%20leukemia%20and%20transformed%20large-cell%20lymphoma%20%28Richter%E2%80%99s%20syndrome%29&amp;journal=Am.%20J.%20Pathol.&amp;doi=10.1016%2FS0002-9440%2810%2964256-3&amp;volume=161&amp;pages=957-968&amp;publication_year=2002&amp;author=Be%C3%A0%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Scandurra, M. et al. Genomic profiling of Richter’s syndrome: recurrent lesions and differences with de novo diffuse large B-cell lymphomas. <i>Hematol. Oncol.</i> <b>28</b>, 62–67 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/hon.932" data-track-item_id="10.1002/hon.932" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2Fhon.932" aria-label="Article reference 12" data-doi="10.1002/hon.932">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXnt12jtrw%3D" aria-label="CAS reference 12">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20014148" aria-label="PubMed reference 12">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20profiling%20of%20Richter%E2%80%99s%20syndrome%3A%20recurrent%20lesions%20and%20differences%20with%20de%20novo%20diffuse%20large%20B-cell%20lymphomas&amp;journal=Hematol.%20Oncol.&amp;doi=10.1002%2Fhon.932&amp;volume=28&amp;pages=62-67&amp;publication_year=2010&amp;author=Scandurra%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Rossi, D. et al. The genetics of Richter syndrome reveals disease heterogeneity and predicts survival after transformation. <i>Blood</i> <b>117</b>, 3391–3401 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood-2010-09-302174" data-track-item_id="10.1182/blood-2010-09-302174" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood-2010-09-302174" aria-label="Article reference 13" data-doi="10.1182/blood-2010-09-302174">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXkslamsLo%3D" aria-label="CAS reference 13">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21266718" aria-label="PubMed reference 13">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20genetics%20of%20Richter%20syndrome%20reveals%20disease%20heterogeneity%20and%20predicts%20survival%20after%20transformation&amp;journal=Blood&amp;doi=10.1182%2Fblood-2010-09-302174&amp;volume=117&amp;pages=3391-3401&amp;publication_year=2011&amp;author=Rossi%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Fabbri, G. et al. Genetic lesions associated with chronic lymphocytic leukemia transformation to Richter syndrome. <i>J. Exp. Med.</i> <b>210</b>, 2273–2288 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1084/jem.20131448" data-track-item_id="10.1084/jem.20131448" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1084%2Fjem.20131448" aria-label="Article reference 14" data-doi="10.1084/jem.20131448">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1ygsb%2FN" aria-label="CAS reference 14">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24127483" aria-label="PubMed reference 14">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3804949" aria-label="PubMed Central reference 14">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20lesions%20associated%20with%20chronic%20lymphocytic%20leukemia%20transformation%20to%20Richter%20syndrome&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20131448&amp;volume=210&amp;pages=2273-2288&amp;publication_year=2013&amp;author=Fabbri%2CG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Chigrinova, E. et al. Two main genetic pathways lead to the transformation of chronic lymphocytic leukemia to Richter syndrome. <i>Blood</i> <b>122</b>, 2673–2682 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood-2013-03-489518" data-track-item_id="10.1182/blood-2013-03-489518" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood-2013-03-489518" aria-label="Article reference 15" data-doi="10.1182/blood-2013-03-489518">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1Cgs73P" aria-label="CAS reference 15">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24004666" aria-label="PubMed reference 15">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Two%20main%20genetic%20pathways%20lead%20to%20the%20transformation%20of%20chronic%20lymphocytic%20leukemia%20to%20Richter%20syndrome&amp;journal=Blood&amp;doi=10.1182%2Fblood-2013-03-489518&amp;volume=122&amp;pages=2673-2682&amp;publication_year=2013&amp;author=Chigrinova%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Klintman, J. et al. Genomic and transcriptomic correlates of Richter transformation in chronic lymphocytic leukemia. <i>Blood</i> <b>137</b>, 2800–2816 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood.2020005650" data-track-item_id="10.1182/blood.2020005650" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood.2020005650" aria-label="Article reference 16" data-doi="10.1182/blood.2020005650">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFaiu7rP" aria-label="CAS reference 16">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33206936" aria-label="PubMed reference 16">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8163497" aria-label="PubMed Central reference 16">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20and%20transcriptomic%20correlates%20of%20Richter%20transformation%20in%20chronic%20lymphocytic%20leukemia&amp;journal=Blood&amp;doi=10.1182%2Fblood.2020005650&amp;volume=137&amp;pages=2800-2816&amp;publication_year=2021&amp;author=Klintman%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Chakraborty, S. et al. B-cell receptor signaling and genetic lesions in TP53 and CDKN2A/CDKN2B cooperate in Richter transformation. <i>Blood</i> <b>138</b>, 1053–1066 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood.2020008276" data-track-item_id="10.1182/blood.2020008276" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood.2020008276" aria-label="Article reference 17" data-doi="10.1182/blood.2020008276">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFWgs7zP" aria-label="CAS reference 17">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33900379" aria-label="PubMed reference 17">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=B-cell%20receptor%20signaling%20and%20genetic%20lesions%20in%20TP53%20and%20CDKN2A%2FCDKN2B%20cooperate%20in%20Richter%20transformation&amp;journal=Blood&amp;doi=10.1182%2Fblood.2020008276&amp;volume=138&amp;pages=1053-1066&amp;publication_year=2021&amp;author=Chakraborty%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Anderson, M. A. et al. Clinicopathological features and outcomes of progression of CLL on the BCL2 inhibitor venetoclax. <i>Blood</i> <b>129</b>, 3362–3370 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood-2017-01-763003" data-track-item_id="10.1182/blood-2017-01-763003" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood-2017-01-763003" aria-label="Article reference 18" data-doi="10.1182/blood-2017-01-763003">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVOmsrzO" aria-label="CAS reference 18">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28473407" aria-label="PubMed reference 18">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Clinicopathological%20features%20and%20outcomes%20of%20progression%20of%20CLL%20on%20the%20BCL2%20inhibitor%20venetoclax&amp;journal=Blood&amp;doi=10.1182%2Fblood-2017-01-763003&amp;volume=129&amp;pages=3362-3370&amp;publication_year=2017&amp;author=Anderson%2CMA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Miller, C. R. et al. Near-tetraploidy is associated with Richter transformation in chronic lymphocytic leukemia patients receiving ibrutinib. <i>Blood Adv.</i> <b>1</b>, 1584–1588 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/bloodadvances.2017007302" data-track-item_id="10.1182/bloodadvances.2017007302" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fbloodadvances.2017007302" aria-label="Article reference 19" data-doi="10.1182/bloodadvances.2017007302">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXls1Ohtb4%3D" aria-label="CAS reference 19">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29296799" aria-label="PubMed reference 19">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728464" aria-label="PubMed Central reference 19">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Near-tetraploidy%20is%20associated%20with%20Richter%20transformation%20in%20chronic%20lymphocytic%20leukemia%20patients%20receiving%20ibrutinib&amp;journal=Blood%20Adv.&amp;doi=10.1182%2Fbloodadvances.2017007302&amp;volume=1&amp;pages=1584-1588&amp;publication_year=2017&amp;author=Miller%2CCR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Kadri, S. et al. Clonal evolution underlying leukemia progression and Richter transformation in patients with ibrutinib-relapsed CLL. <i>Blood Adv.</i> <b>1</b>, 715–727 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/bloodadvances.2016003632" data-track-item_id="10.1182/bloodadvances.2016003632" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fbloodadvances.2016003632" aria-label="Article reference 20" data-doi="10.1182/bloodadvances.2016003632">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXntFCjt7o%3D" aria-label="CAS reference 20">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29296715" aria-label="PubMed reference 20">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728051" aria-label="PubMed Central reference 20">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20underlying%20leukemia%20progression%20and%20Richter%20transformation%20in%20patients%20with%20ibrutinib-relapsed%20CLL&amp;journal=Blood%20Adv.&amp;doi=10.1182%2Fbloodadvances.2016003632&amp;volume=1&amp;pages=715-727&amp;publication_year=2017&amp;author=Kadri%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Herling, C. D. et al. Clonal dynamics towards the development of venetoclax resistance in chronic lymphocytic leukemia. <i>Nat. Commun.</i> <b>9</b>, 727 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-018-03170-7" data-track-item_id="10.1038/s41467-018-03170-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-018-03170-7" aria-label="Article reference 21" data-doi="10.1038/s41467-018-03170-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29463802" aria-label="PubMed reference 21">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820258" aria-label="PubMed Central reference 21">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFWqt7vI" aria-label="CAS reference 21">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20dynamics%20towards%20the%20development%20of%20venetoclax%20resistance%20in%20chronic%20lymphocytic%20leukemia&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-03170-7&amp;volume=9&amp;publication_year=2018&amp;author=Herling%2CCD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Villamor, N. et al. NOTCH1 mutations identify a genetic subgroup of chronic lymphocytic leukemia patients with high risk of transformation and poor outcome. <i>Leukemia</i> <b>27</b>, 1100–1106 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/leu.2012.357" data-track-item_id="10.1038/leu.2012.357" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fleu.2012.357" aria-label="Article reference 22" data-doi="10.1038/leu.2012.357">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXnsVOjurg%3D" aria-label="CAS reference 22">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23295735" aria-label="PubMed reference 22">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=NOTCH1%20mutations%20identify%20a%20genetic%20subgroup%20of%20chronic%20lymphocytic%20leukemia%20patients%20with%20high%20risk%20of%20transformation%20and%20poor%20outcome&amp;journal=Leukemia&amp;doi=10.1038%2Fleu.2012.357&amp;volume=27&amp;pages=1100-1106&amp;publication_year=2013&amp;author=Villamor%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">De Paoli, L. et al. MGA, a suppressor of MYC, is recurrently inactivated in high risk chronic lymphocytic leukemia. <i>Leuk. Lymphoma</i> <b>54</b>, 1087–1090 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3109/10428194.2012.723706" data-track-item_id="10.3109/10428194.2012.723706" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3109%2F10428194.2012.723706" aria-label="Article reference 23" data-doi="10.3109/10428194.2012.723706">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23039309" aria-label="PubMed reference 23">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXmtF2isrY%3D" aria-label="CAS reference 23">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=MGA%2C%20a%20suppressor%20of%20MYC%2C%20is%20recurrently%20inactivated%20in%20high%20risk%20chronic%20lymphocytic%20leukemia&amp;journal=Leuk.%20Lymphoma&amp;doi=10.3109%2F10428194.2012.723706&amp;volume=54&amp;pages=1087-1090&amp;publication_year=2013&amp;author=Paoli%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Rossi, D. et al. Different impact of NOTCH1 and SF3B1 mutations on the risk of chronic lymphocytic leukemia transformation to Richter syndrome. <i>Br. J. Haematol.</i> <b>158</b>, 426–429 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/j.1365-2141.2012.09155.x" data-track-item_id="10.1111/j.1365-2141.2012.09155.x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1365-2141.2012.09155.x" aria-label="Article reference 24" data-doi="10.1111/j.1365-2141.2012.09155.x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFKls7nI" aria-label="CAS reference 24">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22571487" aria-label="PubMed reference 24">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Different%20impact%20of%20NOTCH1%20and%20SF3B1%20mutations%20on%20the%20risk%20of%20chronic%20lymphocytic%20leukemia%20transformation%20to%20Richter%20syndrome&amp;journal=Br.%20J.%20Haematol.&amp;doi=10.1111%2Fj.1365-2141.2012.09155.x&amp;volume=158&amp;pages=426-429&amp;publication_year=2012&amp;author=Rossi%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Chitalia, A. et al. Descriptive analysis of genetic aberrations and cell of origin in Richter transformation. <i>Leuk. Lymphoma</i> <b>60</b>, 971–979 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1080/10428194.2018.1516878" data-track-item_id="10.1080/10428194.2018.1516878" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1080%2F10428194.2018.1516878" aria-label="Article reference 25" data-doi="10.1080/10428194.2018.1516878">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1ejtLk%3D" aria-label="CAS reference 25">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30632835" aria-label="PubMed reference 25">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Descriptive%20analysis%20of%20genetic%20aberrations%20and%20cell%20of%20origin%20in%20Richter%20transformation&amp;journal=Leuk.%20Lymphoma&amp;doi=10.1080%2F10428194.2018.1516878&amp;volume=60&amp;pages=971-979&amp;publication_year=2019&amp;author=Chitalia%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Benatti, S. et al. IRF4 L116R mutation promotes proliferation of chronic lymphocytic leukemia B cells inducing MYC. <i>Hematol. Oncol.</i> <b>39</b>, 707–711 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/hon.2915" data-track-item_id="10.1002/hon.2915" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2Fhon.2915" aria-label="Article reference 26" data-doi="10.1002/hon.2915">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38Xmt12gug%3D%3D" aria-label="CAS reference 26">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34431535" aria-label="PubMed reference 26">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=IRF4%20L116R%20mutation%20promotes%20proliferation%20of%20chronic%20lymphocytic%20leukemia%20B%20cells%20inducing%20MYC&amp;journal=Hematol.%20Oncol.&amp;doi=10.1002%2Fhon.2915&amp;volume=39&amp;pages=707-711&amp;publication_year=2021&amp;author=Benatti%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Minici, C. et al. Distinct homotypic B-cell receptor interactions shape the outcome of chronic lymphocytic leukaemia. <i>Nat. Commun.</i> <b>8</b>, 15746 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ncomms15746" data-track-item_id="10.1038/ncomms15746" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms15746" aria-label="Article reference 27" data-doi="10.1038/ncomms15746">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28598442" aria-label="PubMed reference 27">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5472768" aria-label="PubMed Central reference 27">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Distinct%20homotypic%20B-cell%20receptor%20interactions%20shape%20the%20outcome%20of%20chronic%20lymphocytic%20leukaemia&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms15746&amp;volume=8&amp;publication_year=2017&amp;author=Minici%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Puente, X. S. et al. Non-coding recurrent mutations in chronic lymphocytic leukaemia. <i>Nature</i> <b>526</b>, 519–524 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature14666" data-track-item_id="10.1038/nature14666" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature14666" aria-label="Article reference 28" data-doi="10.1038/nature14666">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1WksrvJ" aria-label="CAS reference 28">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26200345" aria-label="PubMed reference 28">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Non-coding%20recurrent%20mutations%20in%20chronic%20lymphocytic%20leukaemia&amp;journal=Nature&amp;doi=10.1038%2Fnature14666&amp;volume=526&amp;pages=519-524&amp;publication_year=2015&amp;author=Puente%2CXS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Kasar, S. et al. Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution. <i>Nat. Commun.</i> <b>6</b>, 8866 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ncomms9866" data-track-item_id="10.1038/ncomms9866" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms9866" aria-label="Article reference 29" data-doi="10.1038/ncomms9866">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFyks7nI" aria-label="CAS reference 29">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26638776" aria-label="PubMed reference 29">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-genome%20sequencing%20reveals%20activation-induced%20cytidine%20deaminase%20signatures%20during%20indolent%20chronic%20lymphocytic%20leukaemia%20evolution&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms9866&amp;volume=6&amp;publication_year=2015&amp;author=Kasar%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Maura, F. et al. A practical guide for mutational signature analysis in hematological malignancies. <i>Nat. Commun.</i> <b>10</b>, 2969 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-019-11037-8" data-track-item_id="10.1038/s41467-019-11037-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-11037-8" aria-label="Article reference 30" data-doi="10.1038/s41467-019-11037-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31278357" aria-label="PubMed reference 30">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611883" aria-label="PubMed Central reference 30">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlalsbzJ" aria-label="CAS reference 30">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20practical%20guide%20for%20mutational%20signature%20analysis%20in%20hematological%20malignancies&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-11037-8&amp;volume=10&amp;publication_year=2019&amp;author=Maura%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Arthur, S. E. et al. Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma. <i>Nat. Commun.</i> <b>9</b>, 4001 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-018-06354-3" data-track-item_id="10.1038/s41467-018-06354-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-018-06354-3" aria-label="Article reference 31" data-doi="10.1038/s41467-018-06354-3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30275490" aria-label="PubMed reference 31">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6167379" aria-label="PubMed Central reference 31">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVegtL3I" aria-label="CAS reference 31">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20discovery%20of%20somatic%20regulatory%20variants%20in%20diffuse%20large%20B-cell%20lymphoma&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-06354-3&amp;volume=9&amp;publication_year=2018&amp;author=Arthur%2CSE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. <i>Nature</i> <b>578</b>, 94–101 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-020-1943-3" data-track-item_id="10.1038/s41586-020-1943-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-1943-3" aria-label="Article reference 32" data-doi="10.1038/s41586-020-1943-3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsL0%3D" aria-label="CAS reference 32">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32025018" aria-label="PubMed reference 32">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054213" aria-label="PubMed Central reference 32">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20repertoire%20of%20mutational%20signatures%20in%20human%20cancer&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-1943-3&amp;volume=578&amp;pages=94-101&amp;publication_year=2020&amp;author=Alexandrov%2CLB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Kucab, J. E. et al. A compendium of mutational signatures of environmental agents. <i>Cell</i> <b>177</b>, 821–836 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2019.03.001" data-track-item_id="10.1016/j.cell.2019.03.001" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.03.001" aria-label="Article reference 33" data-doi="10.1016/j.cell.2019.03.001">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXnt1Cltb0%3D" aria-label="CAS reference 33">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30982602" aria-label="PubMed reference 33">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6506336" aria-label="PubMed Central reference 33">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20compendium%20of%20mutational%20signatures%20of%20environmental%20agents&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.03.001&amp;volume=177&amp;pages=821-836&amp;publication_year=2019&amp;author=Kucab%2CJE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Rustad, E. H. et al. Timing the initiation of multiple myeloma. <i>Nat. Commun.</i> <b>11</b>, 1917 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-15740-9" data-track-item_id="10.1038/s41467-020-15740-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-15740-9" aria-label="Article reference 34" data-doi="10.1038/s41467-020-15740-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXnslSktbo%3D" aria-label="CAS reference 34">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32317634" aria-label="PubMed reference 34">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7174344" aria-label="PubMed Central reference 34">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Timing%20the%20initiation%20of%20multiple%20myeloma&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-15740-9&amp;volume=11&amp;publication_year=2020&amp;author=Rustad%2CEH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">de Kanter, J. K. et al. Antiviral treatment causes a unique mutational signature in cancers of transplantation recipients. <i>Cell Stem Cell</i> <b>28</b>, 1726–1739 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.stem.2021.07.012" data-track-item_id="10.1016/j.stem.2021.07.012" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.stem.2021.07.012" aria-label="Article reference 35" data-doi="10.1016/j.stem.2021.07.012">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34496298" aria-label="PubMed reference 35">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8516432" aria-label="PubMed Central reference 35">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvF2jtbfL" aria-label="CAS reference 35">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Antiviral%20treatment%20causes%20a%20unique%20mutational%20signature%20in%20cancers%20of%20transplantation%20recipients&amp;journal=Cell%20Stem%20Cell&amp;doi=10.1016%2Fj.stem.2021.07.012&amp;volume=28&amp;pages=1726-1739&amp;publication_year=2021&amp;author=Kanter%2CJK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Pich, O. et al. The mutational footprints of cancer therapies. <i>Nat. Genet.</i> <b>51</b>, 1732–1740 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-019-0525-5" data-track-item_id="10.1038/s41588-019-0525-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-019-0525-5" aria-label="Article reference 36" data-doi="10.1038/s41588-019-0525-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOru7nJ" aria-label="CAS reference 36">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31740835" aria-label="PubMed reference 36">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6887544" aria-label="PubMed Central reference 36">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20mutational%20footprints%20of%20cancer%20therapies&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0525-5&amp;volume=51&amp;pages=1732-1740&amp;publication_year=2019&amp;author=Pich%2CO">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Gaiti, F. et al. Epigenetic evolution and lineage histories of chronic lymphocytic leukaemia. <i>Nature</i> <b>569</b>, 576–580 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-019-1198-z" data-track-item_id="10.1038/s41586-019-1198-z" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-019-1198-z" aria-label="Article reference 37" data-doi="10.1038/s41586-019-1198-z">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXpslGmurs%3D" aria-label="CAS reference 37">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31092926" aria-label="PubMed reference 37">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6533116" aria-label="PubMed Central reference 37">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Epigenetic%20evolution%20and%20lineage%20histories%20of%20chronic%20lymphocytic%20leukaemia&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1198-z&amp;volume=569&amp;pages=576-580&amp;publication_year=2019&amp;author=Gaiti%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Gemenetzi, K. et al. Higher-order immunoglobulin repertoire restrictions in CLL: the illustrative case of stereotyped subsets 2 and 169. <i>Blood</i> <b>137</b>, 1895–1904 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood.2020005216" data-track-item_id="10.1182/blood.2020005216" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood.2020005216" aria-label="Article reference 38" data-doi="10.1182/blood.2020005216">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXovVOjsbk%3D" aria-label="CAS reference 38">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33036024" aria-label="PubMed reference 38">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Higher-order%20immunoglobulin%20repertoire%20restrictions%20in%20CLL%3A%20the%20illustrative%20case%20of%20stereotyped%20subsets%202%20and%20169&amp;journal=Blood&amp;doi=10.1182%2Fblood.2020005216&amp;volume=137&amp;pages=1895-1904&amp;publication_year=2021&amp;author=Gemenetzi%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Bagnara, D. et al. Post-transformation IGHV-IGHD-IGHJ mutations in chronic lymphocytic leukemia B cells: implications for mutational mechanisms and impact on clinical course. <i>Front. Oncol.</i> <b>11</b>, 1769 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3389/fonc.2021.640731" data-track-item_id="10.3389/fonc.2021.640731" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3389%2Ffonc.2021.640731" aria-label="Article reference 39" data-doi="10.3389/fonc.2021.640731">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Post-transformation%20IGHV-IGHD-IGHJ%20mutations%20in%20chronic%20lymphocytic%20leukemia%20B%20cells%3A%20implications%20for%20mutational%20mechanisms%20and%20impact%20on%20clinical%20course&amp;journal=Front.%20Oncol.&amp;doi=10.3389%2Ffonc.2021.640731&amp;volume=11&amp;publication_year=2021&amp;author=Bagnara%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Calissano, C. et al. In vivo intraclonal and interclonal kinetic heterogeneity in B-cell chronic lymphocytic leukemia. <i>Blood</i> <b>114</b>, 4832–4842 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood-2009-05-219634" data-track-item_id="10.1182/blood-2009-05-219634" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood-2009-05-219634" aria-label="Article reference 40" data-doi="10.1182/blood-2009-05-219634">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFGqsbzK" aria-label="CAS reference 40">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19789386" aria-label="PubMed reference 40">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2925477" aria-label="PubMed Central reference 40">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vivo%20intraclonal%20and%20interclonal%20kinetic%20heterogeneity%20in%20B-cell%20chronic%20lymphocytic%20leukemia&amp;journal=Blood&amp;doi=10.1182%2Fblood-2009-05-219634&amp;volume=114&amp;pages=4832-4842&amp;publication_year=2009&amp;author=Calissano%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Calissano, C. et al. Intraclonal complexity in chronic lymphocytic leukemia: fractions enriched in recently born/divided and older/quiescent cells. <i>Mol. Med.</i> <b>17</b>, 1374–1382 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.2119/molmed.2011.00360" data-track-item_id="10.2119/molmed.2011.00360" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.2119%2Fmolmed.2011.00360" aria-label="Article reference 41" data-doi="10.2119/molmed.2011.00360">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1ekurzN" aria-label="CAS reference 41">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21968788" aria-label="PubMed reference 41">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3321822" aria-label="PubMed Central reference 41">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Intraclonal%20complexity%20in%20chronic%20lymphocytic%20leukemia%3A%20fractions%20enriched%20in%20recently%20born%2Fdivided%20and%20older%2Fquiescent%20cells&amp;journal=Mol.%20Med.&amp;doi=10.2119%2Fmolmed.2011.00360&amp;volume=17&amp;pages=1374-1382&amp;publication_year=2011&amp;author=Calissano%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Cui, B. et al. MicroRNA-155 influences B-cell receptor signaling and associates with aggressive disease in chronic lymphocytic leukemia. <i>Blood</i> <b>124</b>, 546–554 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood-2014-03-559690" data-track-item_id="10.1182/blood-2014-03-559690" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood-2014-03-559690" aria-label="Article reference 42" data-doi="10.1182/blood-2014-03-559690">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Ggu7%2FL" aria-label="CAS reference 42">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24914134" aria-label="PubMed reference 42">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4110661" aria-label="PubMed Central reference 42">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=MicroRNA-155%20influences%20B-cell%20receptor%20signaling%20and%20associates%20with%20aggressive%20disease%20in%20chronic%20lymphocytic%20leukemia&amp;journal=Blood&amp;doi=10.1182%2Fblood-2014-03-559690&amp;volume=124&amp;pages=546-554&amp;publication_year=2014&amp;author=Cui%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Penter, L. et al. Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history. <i>Cancer Discov.</i> <b>11</b>, 3048–3063 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1158/2159-8290.CD-21-0276" data-track-item_id="10.1158/2159-8290.CD-21-0276" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1158%2F2159-8290.CD-21-0276" aria-label="Article reference 43" data-doi="10.1158/2159-8290.CD-21-0276">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XjtlyhtL0%3D" aria-label="CAS reference 43">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34112698" aria-label="PubMed reference 43">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8660953" aria-label="PubMed Central reference 43">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Longitudinal%20single-cell%20dynamics%20of%20chromatin%20accessibility%20and%20mitochondrial%20mutations%20in%20chronic%20lymphocytic%20leukemia%20mirror%20disease%20history&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-21-0276&amp;volume=11&amp;pages=3048-3063&amp;publication_year=2021&amp;author=Penter%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Beekman, R. et al. The reference epigenome and regulatory chromatin landscape of chronic lymphocytic leukemia. <i>Nat. Med.</i> <b>24</b>, 868–880 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-018-0028-4" data-track-item_id="10.1038/s41591-018-0028-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-018-0028-4" aria-label="Article reference 44" data-doi="10.1038/s41591-018-0028-4">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVShu7rL" aria-label="CAS reference 44">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29785028" aria-label="PubMed reference 44">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6363101" aria-label="PubMed Central reference 44">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20reference%20epigenome%20and%20regulatory%20chromatin%20landscape%20of%20chronic%20lymphocytic%20leukemia&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-018-0028-4&amp;volume=24&amp;pages=868-880&amp;publication_year=2018&amp;author=Beekman%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Chen, C.-L. et al. Arginine is an epigenetic regulator targeting TEAD4 to modulate OXPHOS in prostate cancer cells. <i>Nat. Commun.</i> <b>12</b>, 2398 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-021-22652-9" data-track-item_id="10.1038/s41467-021-22652-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-021-22652-9" aria-label="Article reference 45" data-doi="10.1038/s41467-021-22652-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXpsl2ktrY%3D" aria-label="CAS reference 45">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33893278" aria-label="PubMed reference 45">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8065123" aria-label="PubMed Central reference 45">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Arginine%20is%20an%20epigenetic%20regulator%20targeting%20TEAD4%20to%20modulate%20OXPHOS%20in%20prostate%20cancer%20cells&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-22652-9&amp;volume=12&amp;publication_year=2021&amp;author=Chen%2CC-L">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Rajbhandari, P. et al. Cross-cohort analysis identifies a TEAD4–MYCN positive feedback loop as the core regulatory element of high-risk neuroblastoma. <i>Cancer Discov.</i> <b>8</b>, 582–599 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1158/2159-8290.CD-16-0861" data-track-item_id="10.1158/2159-8290.CD-16-0861" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1158%2F2159-8290.CD-16-0861" aria-label="Article reference 46" data-doi="10.1158/2159-8290.CD-16-0861">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXos1Sjt70%3D" aria-label="CAS reference 46">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29510988" aria-label="PubMed reference 46">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5967627" aria-label="PubMed Central reference 46">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Cross-cohort%20analysis%20identifies%20a%20TEAD4%E2%80%93MYCN%20positive%20feedback%20loop%20as%20the%20core%20regulatory%20element%20of%20high-risk%20neuroblastoma&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-16-0861&amp;volume=8&amp;pages=582-599&amp;publication_year=2018&amp;author=Rajbhandari%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Maffei, R. et al. IRF4 modulates the response to BCR activation in chronic lymphocytic leukemia regulating IKAROS and SYK. <i>Leukemia</i> <b>35</b>, 1330–1343 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41375-021-01178-5" data-track-item_id="10.1038/s41375-021-01178-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41375-021-01178-5" aria-label="Article reference 47" data-doi="10.1038/s41375-021-01178-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXmvVKqt7o%3D" aria-label="CAS reference 47">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33623139" aria-label="PubMed reference 47">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=IRF4%20modulates%20the%20response%20to%20BCR%20activation%20in%20chronic%20lymphocytic%20leukemia%20regulating%20IKAROS%20and%20SYK&amp;journal=Leukemia&amp;doi=10.1038%2Fs41375-021-01178-5&amp;volume=35&amp;pages=1330-1343&amp;publication_year=2021&amp;author=Maffei%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Patterson, D. G. et al. An IRF4–MYC–mTORC1 integrated pathway controls cell growth and the proliferative capacity of activated B cells during B cell differentiation in vivo. <i>J. Immunol.</i> <b>207</b>, 1798–1811 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.4049/jimmunol.2100440" data-track-item_id="10.4049/jimmunol.2100440" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.4049%2Fjimmunol.2100440" aria-label="Article reference 48" data-doi="10.4049/jimmunol.2100440">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlGrsb7K" aria-label="CAS reference 48">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34470852" aria-label="PubMed reference 48">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20IRF4%E2%80%93MYC%E2%80%93mTORC1%20integrated%20pathway%20controls%20cell%20growth%20and%20the%20proliferative%20capacity%20of%20activated%20B%20cells%20during%20B%20cell%20differentiation%20in%20vivo&amp;journal=J.%20Immunol.&amp;doi=10.4049%2Fjimmunol.2100440&amp;volume=207&amp;pages=1798-1811&amp;publication_year=2021&amp;author=Patterson%2CDG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Duran-Ferrer, M. et al. The proliferative history shapes the DNA methylome of B-cell tumors and predicts clinical outcome. <i>Nat. Cancer</i> <b>1</b>, 1066–1081 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s43018-020-00131-2" data-track-item_id="10.1038/s43018-020-00131-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs43018-020-00131-2" aria-label="Article reference 49" data-doi="10.1038/s43018-020-00131-2">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XjsFegtro%3D" aria-label="CAS reference 49">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34079956" aria-label="PubMed reference 49">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8168619" aria-label="PubMed Central reference 49">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20proliferative%20history%20shapes%20the%20DNA%20methylome%20of%20B-cell%20tumors%20and%20predicts%20clinical%20outcome&amp;journal=Nat.%20Cancer&amp;doi=10.1038%2Fs43018-020-00131-2&amp;volume=1&amp;pages=1066-1081&amp;publication_year=2020&amp;author=Duran-Ferrer%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Hasan, M. K., Ghia, E. M., Rassenti, L. Z., Widhopf, G. F. &amp; Kipps, T. J. Wnt5a enhances proliferation of chronic lymphocytic leukemia and ERK1/2 phosphorylation via a ROR1/DOCK2-dependent mechanism. <i>Leukemia</i> <b>35</b>, 1621–1630 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41375-020-01055-7" data-track-item_id="10.1038/s41375-020-01055-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41375-020-01055-7" aria-label="Article reference 50" data-doi="10.1038/s41375-020-01055-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1WmsbrO" aria-label="CAS reference 50">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33097837" aria-label="PubMed reference 50">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Wnt5a%20enhances%20proliferation%20of%20chronic%20lymphocytic%20leukemia%20and%20ERK1%2F2%20phosphorylation%20via%20a%20ROR1%2FDOCK2-dependent%20mechanism&amp;journal=Leukemia&amp;doi=10.1038%2Fs41375-020-01055-7&amp;volume=35&amp;pages=1621-1630&amp;publication_year=2021&amp;author=Hasan%2CMK&amp;author=Ghia%2CEM&amp;author=Rassenti%2CLZ&amp;author=Widhopf%2CGF&amp;author=Kipps%2CTJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Ntoufa, S., Vilia, M. G., Stamatopoulos, K., Ghia, P. &amp; Muzio, M. Toll-like receptors signaling: a complex network for NF-κB activation in B-cell lymphoid malignancies. <i>Semin. Cancer Biol.</i> <b>39</b>, 15–25 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.semcancer.2016.07.001" data-track-item_id="10.1016/j.semcancer.2016.07.001" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.semcancer.2016.07.001" aria-label="Article reference 51" data-doi="10.1016/j.semcancer.2016.07.001">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1Slu7vP" aria-label="CAS reference 51">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27402288" aria-label="PubMed reference 51">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Toll-like%20receptors%20signaling%3A%20a%20complex%20network%20for%20NF-%CE%BAB%20activation%20in%20B-cell%20lymphoid%20malignancies&amp;journal=Semin.%20Cancer%20Biol.&amp;doi=10.1016%2Fj.semcancer.2016.07.001&amp;volume=39&amp;pages=15-25&amp;publication_year=2016&amp;author=Ntoufa%2CS&amp;author=Vilia%2CMG&amp;author=Stamatopoulos%2CK&amp;author=Ghia%2CP&amp;author=Muzio%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Monti, S. Molecular profiling of diffuse large B-cell lymphoma identifies robust subtypes including one characterized by host inflammatory response. <i>Blood</i> <b>105</b>, 1851–1861 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood-2004-07-2947" data-track-item_id="10.1182/blood-2004-07-2947" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood-2004-07-2947" aria-label="Article reference 52" data-doi="10.1182/blood-2004-07-2947">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXitVOnsrY%3D" aria-label="CAS reference 52">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15550490" aria-label="PubMed reference 52">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20profiling%20of%20diffuse%20large%20B-cell%20lymphoma%20identifies%20robust%20subtypes%20including%20one%20characterized%20by%20host%20inflammatory%20response&amp;journal=Blood&amp;doi=10.1182%2Fblood-2004-07-2947&amp;volume=105&amp;pages=1851-1861&amp;publication_year=2005&amp;author=Monti%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Caro, P. et al. Metabolic signatures uncover distinct targets in molecular subsets of diffuse large B cell lymphoma. <i>Cancer Cell</i> <b>22</b>, 547–560 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ccr.2012.08.014" data-track-item_id="10.1016/j.ccr.2012.08.014" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ccr.2012.08.014" aria-label="Article reference 53" data-doi="10.1016/j.ccr.2012.08.014">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFWitbbO" aria-label="CAS reference 53">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23079663" aria-label="PubMed reference 53">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3479446" aria-label="PubMed Central reference 53">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Metabolic%20signatures%20uncover%20distinct%20targets%20in%20molecular%20subsets%20of%20diffuse%20large%20B%20cell%20lymphoma&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccr.2012.08.014&amp;volume=22&amp;pages=547-560&amp;publication_year=2012&amp;author=Caro%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Norberg, E. et al. Differential contribution of the mitochondrial translation pathway to the survival of diffuse large B-cell lymphoma subsets. <i>Cell Death Differ.</i> <b>24</b>, 251–262 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/cdd.2016.116" data-track-item_id="10.1038/cdd.2016.116" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fcdd.2016.116" aria-label="Article reference 54" data-doi="10.1038/cdd.2016.116">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhslGgs7bP" aria-label="CAS reference 54">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27768122" aria-label="PubMed reference 54">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20contribution%20of%20the%20mitochondrial%20translation%20pathway%20to%20the%20survival%20of%20diffuse%20large%20B-cell%20lymphoma%20subsets&amp;journal=Cell%20Death%20Differ.&amp;doi=10.1038%2Fcdd.2016.116&amp;volume=24&amp;pages=251-262&amp;publication_year=2017&amp;author=Norberg%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Molina, J. R. et al. An inhibitor of oxidative phosphorylation exploits cancer vulnerability. <i>Nat. Med.</i> <b>24</b>, 1036–1046 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-018-0052-4" data-track-item_id="10.1038/s41591-018-0052-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-018-0052-4" aria-label="Article reference 55" data-doi="10.1038/s41591-018-0052-4">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFWru7bN" aria-label="CAS reference 55">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29892070" aria-label="PubMed reference 55">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20inhibitor%20of%20oxidative%20phosphorylation%20exploits%20cancer%20vulnerability&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-018-0052-4&amp;volume=24&amp;pages=1036-1046&amp;publication_year=2018&amp;author=Molina%2CJR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Vangapandu, H. V. et al. Biological and metabolic effects of IACS-010759, an OxPhos inhibitor, on chronic lymphocytic leukemia cells. <i>Oncotarget</i> <b>9</b>, 24980–24991 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.18632/oncotarget.25166" data-track-item_id="10.18632/oncotarget.25166" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.18632%2Foncotarget.25166" aria-label="Article reference 56" data-doi="10.18632/oncotarget.25166">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29861847" aria-label="PubMed reference 56">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5982765" aria-label="PubMed Central reference 56">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Biological%20and%20metabolic%20effects%20of%20IACS-010759%2C%20an%20OxPhos%20inhibitor%2C%20on%20chronic%20lymphocytic%20leukemia%20cells&amp;journal=Oncotarget&amp;doi=10.18632%2Foncotarget.25166&amp;volume=9&amp;pages=24980-24991&amp;publication_year=2018&amp;author=Vangapandu%2CHV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Zhang, L. et al. Metabolic reprogramming toward oxidative phosphorylation identifies a therapeutic target for mantle cell lymphoma. <i>Sci. Transl. Med.</i> <b>11</b>, eaau1167 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/scitranslmed.aau1167" data-track-item_id="10.1126/scitranslmed.aau1167" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscitranslmed.aau1167" aria-label="Article reference 57" data-doi="10.1126/scitranslmed.aau1167">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1CgsrrL" aria-label="CAS reference 57">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31068440" aria-label="PubMed reference 57">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Metabolic%20reprogramming%20toward%20oxidative%20phosphorylation%20identifies%20a%20therapeutic%20target%20for%20mantle%20cell%20lymphoma&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.aau1167&amp;volume=11&amp;publication_year=2019&amp;author=Zhang%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Varano, G. et al. The B-cell receptor controls fitness of MYC-driven lymphoma cells via GSK3β inhibition. <i>Nature</i> <b>546</b>, 302–306 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature22353" data-track-item_id="10.1038/nature22353" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature22353" aria-label="Article reference 58" data-doi="10.1038/nature22353">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXovFCrsrw%3D" aria-label="CAS reference 58">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28562582" aria-label="PubMed reference 58">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20B-cell%20receptor%20controls%20fitness%20of%20MYC-driven%20lymphoma%20cells%20via%20GSK3%CE%B2%20inhibition&amp;journal=Nature&amp;doi=10.1038%2Fnature22353&amp;volume=546&amp;pages=302-306&amp;publication_year=2017&amp;author=Varano%2CG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Dadashian, E. L. et al. TLR signaling is activated in lymph node–resident CLL cells and is only partially inhibited by ibrutinib. <i>Cancer Res.</i> <b>79</b>, 360–371 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1158/0008-5472.CAN-18-0781" data-track-item_id="10.1158/0008-5472.CAN-18-0781" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1158%2F0008-5472.CAN-18-0781" aria-label="Article reference 59" data-doi="10.1158/0008-5472.CAN-18-0781">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXos1aiu7c%3D" aria-label="CAS reference 59">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30498085" aria-label="PubMed reference 59">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=TLR%20signaling%20is%20activated%20in%20lymph%20node%E2%80%93resident%20CLL%20cells%20and%20is%20only%20partially%20inhibited%20by%20ibrutinib&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-18-0781&amp;volume=79&amp;pages=360-371&amp;publication_year=2019&amp;author=Dadashian%2CEL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Chan, K.-L. et al. Plasmablastic Richter transformation as a resistance mechanism for chronic lymphocytic leukaemia treated with BCR signalling inhibitors. <i>Br. J. Haematol.</i> <b>177</b>, 324–328 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/bjh.14062" data-track-item_id="10.1111/bjh.14062" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fbjh.14062" aria-label="Article reference 60" data-doi="10.1111/bjh.14062">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27196127" aria-label="PubMed reference 60">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Plasmablastic%20Richter%20transformation%20as%20a%20resistance%20mechanism%20for%20chronic%20lymphocytic%20leukaemia%20treated%20with%20BCR%20signalling%20inhibitors&amp;journal=Br.%20J.%20Haematol.&amp;doi=10.1111%2Fbjh.14062&amp;volume=177&amp;pages=324-328&amp;publication_year=2017&amp;author=Chan%2CK-L">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. <i>Bioinformatics</i> <b>25</b>, 1754–1760 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btp324" data-track-item_id="10.1093/bioinformatics/btp324" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp324" aria-label="Article reference 61" data-doi="10.1093/bioinformatics/btp324">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D" aria-label="CAS reference 61">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19451168" aria-label="PubMed reference 61">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234" aria-label="PubMed Central reference 61">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp324&amp;volume=25&amp;pages=1754-1760&amp;publication_year=2009&amp;author=Li%2CH&amp;author=Durbin%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Nadeu, F. et al. IgCaller for reconstructing immunoglobulin gene rearrangements and oncogenic translocations from whole-genome sequencing in lymphoid neoplasms. <i>Nat. Commun.</i> <b>11</b>, 3390 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-17095-7" data-track-item_id="10.1038/s41467-020-17095-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-17095-7" aria-label="Article reference 62" data-doi="10.1038/s41467-020-17095-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlKgur3L" aria-label="CAS reference 62">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32636395" aria-label="PubMed reference 62">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7341758" aria-label="PubMed Central reference 62">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=IgCaller%20for%20reconstructing%20immunoglobulin%20gene%20rearrangements%20and%20oncogenic%20translocations%20from%20whole-genome%20sequencing%20in%20lymphoid%20neoplasms&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-17095-7&amp;volume=11&amp;publication_year=2020&amp;author=Nadeu%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Robinson, J. T. et al. Integrative genomics viewer. <i>Nat. Biotechnol.</i> <b>29</b>, 24–26 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nbt.1754" data-track-item_id="10.1038/nbt.1754" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.1754" aria-label="Article reference 63" data-doi="10.1038/nbt.1754">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXjsFWrtg%3D%3D" aria-label="CAS reference 63">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21221095" aria-label="PubMed reference 63">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3346182" aria-label="PubMed Central reference 63">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrative%20genomics%20viewer&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.1754&amp;volume=29&amp;pages=24-26&amp;publication_year=2011&amp;author=Robinson%2CJT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Jones, D. et al. cgpCaVEManWrapper: simple execution of CaVEMan in order to detect somatic single nucleotide variants in NGS data. <i>Curr. Protoc. Bioinforma.</i> <b>56</b>, 15.10.1–15.10.18 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/cpbi.20" data-track-item_id="10.1002/cpbi.20" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2Fcpbi.20" aria-label="Article reference 64" data-doi="10.1002/cpbi.20">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=cgpCaVEManWrapper%3A%20simple%20execution%20of%20CaVEMan%20in%20order%20to%20detect%20somatic%20single%20nucleotide%20variants%20in%20NGS%20data&amp;journal=Curr.%20Protoc.%20Bioinforma.&amp;doi=10.1002%2Fcpbi.20&amp;volume=56&amp;pages=15.10.1-15.10.18&amp;publication_year=2016&amp;author=Jones%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">McKenna, A. et al. The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. <i>Genome Res.</i> <b>20</b>, 1297–1303 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/gr.107524.110" data-track-item_id="10.1101/gr.107524.110" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.107524.110" aria-label="Article reference 65" data-doi="10.1101/gr.107524.110">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFeru7jM" aria-label="CAS reference 65">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20644199" aria-label="PubMed reference 65">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928508" aria-label="PubMed Central reference 65">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20genome%20analysis%20toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.107524.110&amp;volume=20&amp;pages=1297-1303&amp;publication_year=2010&amp;author=McKenna%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Fan, Y. et al. MuSE: accounting for tumor heterogeneity using a sample-specific error model improves sensitivity and specificity in mutation calling from sequencing data. <i>Genome Biol.</i> <b>17</b>, 178 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-016-1029-6" data-track-item_id="10.1186/s13059-016-1029-6" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-016-1029-6" aria-label="Article reference 66" data-doi="10.1186/s13059-016-1029-6">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27557938" aria-label="PubMed reference 66">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4995747" aria-label="PubMed Central reference 66">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=MuSE%3A%20accounting%20for%20tumor%20heterogeneity%20using%20a%20sample-specific%20error%20model%20improves%20sensitivity%20and%20specificity%20in%20mutation%20calling%20from%20sequencing%20data&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-016-1029-6&amp;volume=17&amp;publication_year=2016&amp;author=Fan%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Danecek, P. et al. Twelve years of SAMtools and BCFtools. <i>Gigascience</i> <a href="https://doi.org/10.1093/gigascience/giab008" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1093/gigascience/giab008">https://doi.org/10.1093/gigascience/giab008</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Moncunill, V. et al. Comprehensive characterization of complex structural variations in cancer by directly comparing genome sequence reads. <i>Nat. Biotechnol.</i> <b>32</b>, 1106–1112 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nbt.3027" data-track-item_id="10.1038/nbt.3027" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.3027" aria-label="Article reference 68" data-doi="10.1038/nbt.3027">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVSqtrzN" aria-label="CAS reference 68">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25344728" aria-label="PubMed reference 68">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20characterization%20of%20complex%20structural%20variations%20in%20cancer%20by%20directly%20comparing%20genome%20sequence%20reads&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3027&amp;volume=32&amp;pages=1106-1112&amp;publication_year=2014&amp;author=Moncunill%2CV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Raine, K. M. et al. cgpPindel: identifying somatically acquired insertion and deletion events from paired end sequencing. <i>Curr. Protoc. Bioinforma.</i> <b>52</b>, 15.7.1–12 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/0471250953.bi1507s52" data-track-item_id="10.1002/0471250953.bi1507s52" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2F0471250953.bi1507s52" aria-label="Article reference 69" data-doi="10.1002/0471250953.bi1507s52">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=cgpPindel%3A%20identifying%20somatically%20acquired%20insertion%20and%20deletion%20events%20from%20paired%20end%20sequencing&amp;journal=Curr.%20Protoc.%20Bioinforma.&amp;doi=10.1002%2F0471250953.bi1507s52&amp;volume=52&amp;pages=15.7.1-12&amp;publication_year=2015&amp;author=Raine%2CKM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Wala, J. A. et al. SvABA: genome-wide detection of structural variants and indels by local assembly. <i>Genome Res.</i> <b>28</b>, 581–591 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/gr.221028.117" data-track-item_id="10.1101/gr.221028.117" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.221028.117" aria-label="Article reference 70" data-doi="10.1101/gr.221028.117">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXns1Cltbk%3D" aria-label="CAS reference 70">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29535149" aria-label="PubMed reference 70">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5880247" aria-label="PubMed Central reference 70">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=SvABA%3A%20genome-wide%20detection%20of%20structural%20variants%20and%20indels%20by%20local%20assembly&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.221028.117&amp;volume=28&amp;pages=581-591&amp;publication_year=2018&amp;author=Wala%2CJA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Rimmer, A. et al. Integrating mapping-, assembly- and haplotype-based approaches for calling variants in clinical sequencing applications. <i>Nat. Genet.</i> <b>46</b>, 912–918 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ng.3036" data-track-item_id="10.1038/ng.3036" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3036" aria-label="Article reference 71" data-doi="10.1038/ng.3036">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFGqsbjP" aria-label="CAS reference 71">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25017105" aria-label="PubMed reference 71">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4753679" aria-label="PubMed Central reference 71">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrating%20mapping-%2C%20assembly-%20and%20haplotype-based%20approaches%20for%20calling%20variants%20in%20clinical%20sequencing%20applications&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3036&amp;volume=46&amp;pages=912-918&amp;publication_year=2014&amp;author=Rimmer%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <i>Drosophila melanogaster</i> strain w1118; iso-2; iso-3. <i>Fly. (Austin).</i> <b>6</b>, 80–92 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.4161/fly.19695" data-track-item_id="10.4161/fly.19695" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.4161%2Ffly.19695" aria-label="Article reference 72" data-doi="10.4161/fly.19695">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1GmtL3E" aria-label="CAS reference 72">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22728672" aria-label="PubMed reference 72">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3679285" aria-label="PubMed Central reference 72">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;journal=Fly.%20%28Austin%29.&amp;doi=10.4161%2Ffly.19695&amp;volume=6&amp;pages=80-92&amp;publication_year=2012&amp;author=Cingolani%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Nadeu, F. et al. Clinical impact of the subclonal architecture and mutational complexity in chronic lymphocytic leukemia. <i>Leukemia</i> <b>32</b>, 645–653 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/leu.2017.291" data-track-item_id="10.1038/leu.2017.291" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fleu.2017.291" aria-label="Article reference 73" data-doi="10.1038/leu.2017.291">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1Cmt7vN" aria-label="CAS reference 73">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28924241" aria-label="PubMed reference 73">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20impact%20of%20the%20subclonal%20architecture%20and%20mutational%20complexity%20in%20chronic%20lymphocytic%20leukemia&amp;journal=Leukemia&amp;doi=10.1038%2Fleu.2017.291&amp;volume=32&amp;pages=645-653&amp;publication_year=2018&amp;author=Nadeu%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Koboldt, D. C. et al. VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing. <i>Genome Res.</i> <b>22</b>, 568–576 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/gr.129684.111" data-track-item_id="10.1101/gr.129684.111" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.129684.111" aria-label="Article reference 74" data-doi="10.1101/gr.129684.111">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XjsV2ltr4%3D" aria-label="CAS reference 74">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22300766" aria-label="PubMed reference 74">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3290792" aria-label="PubMed Central reference 74">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=VarScan%202%3A%20somatic%20mutation%20and%20copy%20number%20alteration%20discovery%20in%20cancer%20by%20exome%20sequencing&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.129684.111&amp;volume=22&amp;pages=568-576&amp;publication_year=2012&amp;author=Koboldt%2CDC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Lai, Z. et al. VarDict: a novel and versatile variant caller for next-generation sequencing in cancer research. <i>Nucleic Acids Res.</i> <b>44</b>, e108 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkw227" data-track-item_id="10.1093/nar/gkw227" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkw227" aria-label="Article reference 75" data-doi="10.1093/nar/gkw227">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27060149" aria-label="PubMed reference 75">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4914105" aria-label="PubMed Central reference 75">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2jtrnL" aria-label="CAS reference 75">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=VarDict%3A%20a%20novel%20and%20versatile%20variant%20caller%20for%20next-generation%20sequencing%20in%20cancer%20research&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw227&amp;volume=44&amp;publication_year=2016&amp;author=Lai%2CZ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Wilm, A. et al. LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets. <i>Nucleic Acids Res.</i> <b>40</b>, 11189–11201 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gks918" data-track-item_id="10.1093/nar/gks918" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgks918" aria-label="Article reference 76" data-doi="10.1093/nar/gks918">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVCitLjJ" aria-label="CAS reference 76">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23066108" aria-label="PubMed reference 76">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3526318" aria-label="PubMed Central reference 76">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=LoFreq%3A%20a%20sequence-quality%20aware%2C%20ultra-sensitive%20variant%20caller%20for%20uncovering%20cell-population%20heterogeneity%20from%20high-throughput%20sequencing%20datasets&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgks918&amp;volume=40&amp;pages=11189-11201&amp;publication_year=2012&amp;author=Wilm%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Muller, E. et al. OutLyzer: software for extracting low-allele-frequency tumor mutations from sequencing background noise in clinical practice. <i>Oncotarget</i> <b>7</b>, 79485–79493 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.18632/oncotarget.13103" data-track-item_id="10.18632/oncotarget.13103" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.18632%2Foncotarget.13103" aria-label="Article reference 77" data-doi="10.18632/oncotarget.13103">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27825131" aria-label="PubMed reference 77">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5346729" aria-label="PubMed Central reference 77">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=OutLyzer%3A%20software%20for%20extracting%20low-allele-frequency%20tumor%20mutations%20from%20sequencing%20background%20noise%20in%20clinical%20practice&amp;journal=Oncotarget&amp;doi=10.18632%2Foncotarget.13103&amp;volume=7&amp;pages=79485-79493&amp;publication_year=2016&amp;author=Muller%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Nik-Zainal, S. et al. The life history of 21 breast cancers. <i>Cell</i> <b>149</b>, 994–1007 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2012.04.023" data-track-item_id="10.1016/j.cell.2012.04.023" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2012.04.023" aria-label="Article reference 78" data-doi="10.1016/j.cell.2012.04.023">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XntlymtLY%3D" aria-label="CAS reference 78">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22608083" aria-label="PubMed reference 78">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3428864" aria-label="PubMed Central reference 78">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20life%20history%20of%2021%20breast%20cancers&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2012.04.023&amp;volume=149&amp;pages=994-1007&amp;publication_year=2012&amp;author=Nik-Zainal%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Raine, K. M. et al. ascatNgs: identifying somatically acquired copy-number alterations from whole-genome sequencing data. <i>Curr. Protoc. Bioinforma.</i> <b>56</b>, 15.9.1–15.9.17 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/cpbi.17" data-track-item_id="10.1002/cpbi.17" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2Fcpbi.17" aria-label="Article reference 79" data-doi="10.1002/cpbi.17">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=ascatNgs%3A%20identifying%20somatically%20acquired%20copy-number%20alterations%20from%20whole-genome%20sequencing%20data&amp;journal=Curr.%20Protoc.%20Bioinforma.&amp;doi=10.1002%2Fcpbi.17&amp;volume=56&amp;pages=15.9.1-15.9.17&amp;publication_year=2016&amp;author=Raine%2CKM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Talevich, E., Shain, A. H., Botton, T. &amp; Bastian, B. C. CNVkit: genome-wide copy number detection and visualization from targeted DNA sequencing. <i>PLoS Comput. Biol.</i> <b>12</b>, e1004873 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pcbi.1004873" data-track-item_id="10.1371/journal.pcbi.1004873" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pcbi.1004873" aria-label="Article reference 80" data-doi="10.1371/journal.pcbi.1004873">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27100738" aria-label="PubMed reference 80">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4839673" aria-label="PubMed Central reference 80">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFWgtb%2FJ" aria-label="CAS reference 80">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=CNVkit%3A%20genome-wide%20copy%20number%20detection%20and%20visualization%20from%20targeted%20DNA%20sequencing&amp;journal=PLoS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1004873&amp;volume=12&amp;publication_year=2016&amp;author=Talevich%2CE&amp;author=Shain%2CAH&amp;author=Botton%2CT&amp;author=Bastian%2CBC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome sequences. <i>Nature</i> <b>534</b>, 47–54 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature17676" data-track-item_id="10.1038/nature17676" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature17676" aria-label="Article reference 81" data-doi="10.1038/nature17676">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XntVCksrc%3D" aria-label="CAS reference 81">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27135926" aria-label="PubMed reference 81">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4910866" aria-label="PubMed Central reference 81">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=Landscape%20of%20somatic%20mutations%20in%20560%20breast%20cancer%20whole-genome%20sequences&amp;journal=Nature&amp;doi=10.1038%2Fnature17676&amp;volume=534&amp;pages=47-54&amp;publication_year=2016&amp;author=Nik-Zainal%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">Rausch, T. et al. DELLY: structural variant discovery by integrated paired-end and split-read analysis. <i>Bioinformatics</i> <b>28</b>, i333–i339 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/bts378" data-track-item_id="10.1093/bioinformatics/bts378" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts378" aria-label="Article reference 82" data-doi="10.1093/bioinformatics/bts378">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhtlantLjF" aria-label="CAS reference 82">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22962449" aria-label="PubMed reference 82">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3436805" aria-label="PubMed Central reference 82">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=DELLY%3A%20structural%20variant%20discovery%20by%20integrated%20paired-end%20and%20split-read%20analysis&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbts378&amp;volume=28&amp;pages=i333-i339&amp;publication_year=2012&amp;author=Rausch%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">Stephens, P. J. et al. Massive genomic rearrangement acquired in a single catastrophic event during cancer development. <i>Cell</i> <b>144</b>, 27–40 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2010.11.055" data-track-item_id="10.1016/j.cell.2010.11.055" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2010.11.055" aria-label="Article reference 83" data-doi="10.1016/j.cell.2010.11.055">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXksFGjug%3D%3D" aria-label="CAS reference 83">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21215367" aria-label="PubMed reference 83">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3065307" aria-label="PubMed Central reference 83">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=Massive%20genomic%20rearrangement%20acquired%20in%20a%20single%20catastrophic%20event%20during%20cancer%20development&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2010.11.055&amp;volume=144&amp;pages=27-40&amp;publication_year=2011&amp;author=Stephens%2CPJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84."><p class="c-article-references__text" id="ref-CR84">Korbel, J. O. &amp; Campbell, P. J. Criteria for inference of chromothripsis in cancer genomes. <i>Cell</i> <b>152</b>, 1226–1236 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2013.02.023" data-track-item_id="10.1016/j.cell.2013.02.023" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2013.02.023" aria-label="Article reference 84" data-doi="10.1016/j.cell.2013.02.023">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXktFWiurs%3D" aria-label="CAS reference 84">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23498933" aria-label="PubMed reference 84">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=Criteria%20for%20inference%20of%20chromothripsis%20in%20cancer%20genomes&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.02.023&amp;volume=152&amp;pages=1226-1236&amp;publication_year=2013&amp;author=Korbel%2CJO&amp;author=Campbell%2CPJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85."><p class="c-article-references__text" id="ref-CR85">Baca, S. C. et al. Punctuated evolution of prostate cancer genomes. <i>Cell</i> <b>153</b>, 666–677 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2013.03.021" data-track-item_id="10.1016/j.cell.2013.03.021" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2013.03.021" aria-label="Article reference 85" data-doi="10.1016/j.cell.2013.03.021">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXmslelsLg%3D" aria-label="CAS reference 85">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23622249" aria-label="PubMed reference 85">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3690918" aria-label="PubMed Central reference 85">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=Punctuated%20evolution%20of%20prostate%20cancer%20genomes&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.03.021&amp;volume=153&amp;pages=666-677&amp;publication_year=2013&amp;author=Baca%2CSC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86."><p class="c-article-references__text" id="ref-CR86">Shen, M. M. Chromoplexy: a new category of complex rearrangements in the cancer genome. <i>Cancer Cell</i> <b>23</b>, 567–569 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ccr.2013.04.025" data-track-item_id="10.1016/j.ccr.2013.04.025" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ccr.2013.04.025" aria-label="Article reference 86" data-doi="10.1016/j.ccr.2013.04.025">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXnslajtbs%3D" aria-label="CAS reference 86">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23680143" aria-label="PubMed reference 86">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3673705" aria-label="PubMed Central reference 86">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 86" href="http://scholar.google.com/scholar_lookup?&amp;title=Chromoplexy%3A%20a%20new%20category%20of%20complex%20rearrangements%20in%20the%20cancer%20genome&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccr.2013.04.025&amp;volume=23&amp;pages=567-569&amp;publication_year=2013&amp;author=Shen%2CMM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87."><p class="c-article-references__text" id="ref-CR87">Li, Y. et al. Patterns of somatic structural variation in human cancer genomes. <i>Nature</i> <b>578</b>, 112–121 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-019-1913-9" data-track-item_id="10.1038/s41586-019-1913-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-019-1913-9" aria-label="Article reference 87" data-doi="10.1038/s41586-019-1913-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsL8%3D" aria-label="CAS reference 87">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32025012" aria-label="PubMed reference 87">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7025897" aria-label="PubMed Central reference 87">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 87" href="http://scholar.google.com/scholar_lookup?&amp;title=Patterns%20of%20somatic%20structural%20variation%20in%20human%20cancer%20genomes&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1913-9&amp;volume=578&amp;pages=112-121&amp;publication_year=2020&amp;author=Li%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="88."><p class="c-article-references__text" id="ref-CR88">Maura, F. et al. Genomic landscape and chronological reconstruction of driver events in multiple myeloma. <i>Nat. Commun.</i> <b>10</b>, 3835 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-019-11680-1" data-track-item_id="10.1038/s41467-019-11680-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-11680-1" aria-label="Article reference 88" data-doi="10.1038/s41467-019-11680-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31444325" aria-label="PubMed reference 88">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6707220" aria-label="PubMed Central reference 88">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1KktrnP" aria-label="CAS reference 88">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20landscape%20and%20chronological%20reconstruction%20of%20driver%20events%20in%20multiple%20myeloma&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-11680-1&amp;volume=10&amp;publication_year=2019&amp;author=Maura%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="89."><p class="c-article-references__text" id="ref-CR89">Dentro, S. C., Wedge, D. C. &amp; Van Loo, P. Principles of reconstructing the subclonal architecture of cancers. <i>Cold Spring Harb. Perspect. Med.</i> <b>7</b>, a026625 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/cshperspect.a026625" data-track-item_id="10.1101/cshperspect.a026625" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fcshperspect.a026625" aria-label="Article reference 89" data-doi="10.1101/cshperspect.a026625">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28270531" aria-label="PubMed reference 89">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5538405" aria-label="PubMed Central reference 89">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXisFKqurg%3D" aria-label="CAS reference 89">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 89" href="http://scholar.google.com/scholar_lookup?&amp;title=Principles%20of%20reconstructing%20the%20subclonal%20architecture%20of%20cancers&amp;journal=Cold%20Spring%20Harb.%20Perspect.%20Med.&amp;doi=10.1101%2Fcshperspect.a026625&amp;volume=7&amp;publication_year=2017&amp;author=Dentro%2CSC&amp;author=Wedge%2CDC&amp;author=Loo%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="90."><p class="c-article-references__text" id="ref-CR90">Kim, J. et al. Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors. <i>Nat. Genet.</i> <b>48</b>, 600–606 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ng.3557" data-track-item_id="10.1038/ng.3557" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3557" aria-label="Article reference 90" data-doi="10.1038/ng.3557">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xms1Ontr0%3D" aria-label="CAS reference 90">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27111033" aria-label="PubMed reference 90">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4936490" aria-label="PubMed Central reference 90">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 90" href="http://scholar.google.com/scholar_lookup?&amp;title=Somatic%20ERCC2%20mutations%20are%20associated%20with%20a%20distinct%20genomic%20signature%20in%20urothelial%20tumors&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3557&amp;volume=48&amp;pages=600-606&amp;publication_year=2016&amp;author=Kim%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="91."><p class="c-article-references__text" id="ref-CR91">Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. <i>Nature</i> <b>574</b>, 532–537 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-019-1672-7" data-track-item_id="10.1038/s41586-019-1672-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-019-1672-7" aria-label="Article reference 91" data-doi="10.1038/s41586-019-1672-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVagsb3J" aria-label="CAS reference 91">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31645730" aria-label="PubMed reference 91">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 91" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20landscape%20of%20somatic%20mutation%20in%20normal%20colorectal%20epithelial%20cells&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1672-7&amp;volume=574&amp;pages=532-537&amp;publication_year=2019&amp;author=Lee-Six%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="92."><p class="c-article-references__text" id="ref-CR92">Yang, F. et al. Chemotherapy and mismatch repair deficiency cooperate to fuel TP53 mutagenesis and ALL relapse. <i>Nat. Cancer</i> <b>2</b>, 819–834 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s43018-021-00230-8" data-track-item_id="10.1038/s43018-021-00230-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs43018-021-00230-8" aria-label="Article reference 92" data-doi="10.1038/s43018-021-00230-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XjsVOms7g%3D" aria-label="CAS reference 92">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35122027" aria-label="PubMed reference 92">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 92" href="http://scholar.google.com/scholar_lookup?&amp;title=Chemotherapy%20and%20mismatch%20repair%20deficiency%20cooperate%20to%20fuel%20TP53%20mutagenesis%20and%20ALL%20relapse&amp;journal=Nat.%20Cancer&amp;doi=10.1038%2Fs43018-021-00230-8&amp;volume=2&amp;pages=819-834&amp;publication_year=2021&amp;author=Yang%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="93."><p class="c-article-references__text" id="ref-CR93">Koren, A. et al. Differential relationship of DNA replication timing to different forms of human mutation and variation. <i>Am. J. Hum. Genet.</i> <b>91</b>, 1033–1040 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2012.10.018" data-track-item_id="10.1016/j.ajhg.2012.10.018" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2012.10.018" aria-label="Article reference 93" data-doi="10.1016/j.ajhg.2012.10.018">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhsleisLrO" aria-label="CAS reference 93">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23176822" aria-label="PubMed reference 93">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3516607" aria-label="PubMed Central reference 93">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 93" href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20relationship%20of%20DNA%20replication%20timing%20to%20different%20forms%20of%20human%20mutation%20and%20variation&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2012.10.018&amp;volume=91&amp;pages=1033-1040&amp;publication_year=2012&amp;author=Koren%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="94."><p class="c-article-references__text" id="ref-CR94">Haradhvala, N. J. et al. Mutational strand asymmetries in cancer genomes reveal mechanisms of DNA damage and repair. <i>Cell</i> <b>164</b>, 538–549 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2015.12.050" data-track-item_id="10.1016/j.cell.2015.12.050" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2015.12.050" aria-label="Article reference 94" data-doi="10.1016/j.cell.2015.12.050">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVKitbg%3D" aria-label="CAS reference 94">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26806129" aria-label="PubMed reference 94">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4753048" aria-label="PubMed Central reference 94">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 94" href="http://scholar.google.com/scholar_lookup?&amp;title=Mutational%20strand%20asymmetries%20in%20cancer%20genomes%20reveal%20mechanisms%20of%20DNA%20damage%20and%20repair&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2015.12.050&amp;volume=164&amp;pages=538-549&amp;publication_year=2016&amp;author=Haradhvala%2CNJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="95."><p class="c-article-references__text" id="ref-CR95">Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. <i>Bioinformatics</i> <b>30</b>, 2114–2120 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btu170" data-track-item_id="10.1093/bioinformatics/btu170" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu170" aria-label="Article reference 95" data-doi="10.1093/bioinformatics/btu170">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Sqt7nP" aria-label="CAS reference 95">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24695404" aria-label="PubMed reference 95">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590" aria-label="PubMed Central reference 95">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 95" href="http://scholar.google.com/scholar_lookup?&amp;title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu170&amp;volume=30&amp;pages=2114-2120&amp;publication_year=2014&amp;author=Bolger%2CAM&amp;author=Lohse%2CM&amp;author=Usadel%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="96."><p class="c-article-references__text" id="ref-CR96">Aryee, M. J. et al. Minfi: a flexible and comprehensive Bioconductor package for the analysis of Infinium DNA methylation microarrays. <i>Bioinformatics</i> <b>30</b>, 1363–1369 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btu049" data-track-item_id="10.1093/bioinformatics/btu049" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu049" aria-label="Article reference 96" data-doi="10.1093/bioinformatics/btu049">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXnslOit78%3D" aria-label="CAS reference 96">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24478339" aria-label="PubMed reference 96">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4016708" aria-label="PubMed Central reference 96">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 96" href="http://scholar.google.com/scholar_lookup?&amp;title=Minfi%3A%20a%20flexible%20and%20comprehensive%20Bioconductor%20package%20for%20the%20analysis%20of%20Infinium%20DNA%20methylation%20microarrays&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu049&amp;volume=30&amp;pages=1363-1369&amp;publication_year=2014&amp;author=Aryee%2CMJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="97."><p class="c-article-references__text" id="ref-CR97">Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). <i>Genome Biol.</i> <b>9</b>, R137 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/gb-2008-9-9-r137" data-track-item_id="10.1186/gb-2008-9-9-r137" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/gb-2008-9-9-r137" aria-label="Article reference 97" data-doi="10.1186/gb-2008-9-9-r137">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18798982" aria-label="PubMed reference 97">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715" aria-label="PubMed Central reference 97">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtl2ksrjE" aria-label="CAS reference 97">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 97" href="http://scholar.google.com/scholar_lookup?&amp;title=Model-based%20analysis%20of%20ChIP-Seq%20%28MACS%29&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fgb-2008-9-9-r137&amp;volume=9&amp;publication_year=2008&amp;author=Zhang%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="98."><p class="c-article-references__text" id="ref-CR98">Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <i>Genome Biol.</i> <b>15</b>, 550 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-014-0550-8" data-track-item_id="10.1186/s13059-014-0550-8" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-014-0550-8" aria-label="Article reference 98" data-doi="10.1186/s13059-014-0550-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25516281" aria-label="PubMed reference 98">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049" aria-label="PubMed Central reference 98">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXjtVCrsL8%3D" aria-label="CAS reference 98">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 98" href="http://scholar.google.com/scholar_lookup?&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-014-0550-8&amp;volume=15&amp;publication_year=2014&amp;author=Love%2CMI&amp;author=Huber%2CW&amp;author=Anders%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="99."><p class="c-article-references__text" id="ref-CR99">Kopylova, E., Noé, L. &amp; Touzet, H. SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data. <i>Bioinformatics</i> <b>28</b>, 3211–3217 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/bts611" data-track-item_id="10.1093/bioinformatics/bts611" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbts611" aria-label="Article reference 99" data-doi="10.1093/bioinformatics/bts611">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVSmsrvE" aria-label="CAS reference 99">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23071270" aria-label="PubMed reference 99">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 99" href="http://scholar.google.com/scholar_lookup?&amp;title=SortMeRNA%3A%20fast%20and%20accurate%20filtering%20of%20ribosomal%20RNAs%20in%20metatranscriptomic%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbts611&amp;volume=28&amp;pages=3211-3217&amp;publication_year=2012&amp;author=Kopylova%2CE&amp;author=No%C3%A9%2CL&amp;author=Touzet%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="100."><p class="c-article-references__text" id="ref-CR100">Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. <i>Nat. Biotechnol.</i> <b>34</b>, 525–527 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nbt.3519" data-track-item_id="10.1038/nbt.3519" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.3519" aria-label="Article reference 100" data-doi="10.1038/nbt.3519">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XlsVansL8%3D" aria-label="CAS reference 100">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27043002" aria-label="PubMed reference 100">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 100" href="http://scholar.google.com/scholar_lookup?&amp;title=Near-optimal%20probabilistic%20RNA-seq%20quantification&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3519&amp;volume=34&amp;pages=525-527&amp;publication_year=2016&amp;author=Bray%2CNL&amp;author=Pimentel%2CH&amp;author=Melsted%2CP&amp;author=Pachter%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="101."><p class="c-article-references__text" id="ref-CR101">Kuipers, J., Jahn, K., Raphael, B. J. &amp; Beerenwinkel, N. Single-cell sequencing data reveal widespread recurrence and loss of mutational hits in the life histories of tumors. <i>Genome Res.</i> <b>27</b>, 1885–1894 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/gr.220707.117" data-track-item_id="10.1101/gr.220707.117" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.220707.117" aria-label="Article reference 101" data-doi="10.1101/gr.220707.117">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslyitL7E" aria-label="CAS reference 101">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29030470" aria-label="PubMed reference 101">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5668945" aria-label="PubMed Central reference 101">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 101" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20sequencing%20data%20reveal%20widespread%20recurrence%20and%20loss%20of%20mutational%20hits%20in%20the%20life%20histories%20of%20tumors&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.220707.117&amp;volume=27&amp;pages=1885-1894&amp;publication_year=2017&amp;author=Kuipers%2CJ&amp;author=Jahn%2CK&amp;author=Raphael%2CBJ&amp;author=Beerenwinkel%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="102."><p class="c-article-references__text" id="ref-CR102">Morita, K. et al. Clonal evolution of acute myeloid leukemia revealed by high-throughput single-cell genomics. <i>Nat. Commun.</i> <b>11</b>, 5327 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-19119-8" data-track-item_id="10.1038/s41467-020-19119-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-19119-8" aria-label="Article reference 102" data-doi="10.1038/s41467-020-19119-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitF2js7jE" aria-label="CAS reference 102">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33087716" aria-label="PubMed reference 102">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7577981" aria-label="PubMed Central reference 102">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 102" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20of%20acute%20myeloid%20leukemia%20revealed%20by%20high-throughput%20single-cell%20genomics&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19119-8&amp;volume=11&amp;publication_year=2020&amp;author=Morita%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="103."><p class="c-article-references__text" id="ref-CR103">Picelli, S. et al. Smart-seq2 for sensitive full-length transcriptome profiling in single cells. <i>Nat. Methods</i> <b>10</b>, 1096–1098 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmeth.2639" data-track-item_id="10.1038/nmeth.2639" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.2639" aria-label="Article reference 103" data-doi="10.1038/nmeth.2639">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVyqtb%2FM" aria-label="CAS reference 103">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24056875" aria-label="PubMed reference 103">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 103" href="http://scholar.google.com/scholar_lookup?&amp;title=Smart-seq2%20for%20sensitive%20full-length%20transcriptome%20profiling%20in%20single%20cells&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.2639&amp;volume=10&amp;pages=1096-1098&amp;publication_year=2013&amp;author=Picelli%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="104."><p class="c-article-references__text" id="ref-CR104">Stoeckius, M. et al. Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics. <i>Genome Biol.</i> <b>19</b>, 224 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-018-1603-1" data-track-item_id="10.1186/s13059-018-1603-1" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-018-1603-1" aria-label="Article reference 104" data-doi="10.1186/s13059-018-1603-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVKksbfJ" aria-label="CAS reference 104">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30567574" aria-label="PubMed reference 104">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6300015" aria-label="PubMed Central reference 104">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 104" href="http://scholar.google.com/scholar_lookup?&amp;title=Cell%20Hashing%20with%20barcoded%20antibodies%20enables%20multiplexing%20and%20doublet%20detection%20for%20single%20cell%20genomics&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-018-1603-1&amp;volume=19&amp;publication_year=2018&amp;author=Stoeckius%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="105."><p class="c-article-references__text" id="ref-CR105">Parekh, S., Ziegenhain, C., Vieth, B., Enard, W. &amp; Hellmann, I. zUMIs - A fast and flexible pipeline to process RNA sequencing data with UMIs. <i>Gigascience</i> <a href="https://doi.org/10.1093/gigascience/giy059" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1093/gigascience/giy059">https://doi.org/10.1093/gigascience/giy059</a> (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="106."><p class="c-article-references__text" id="ref-CR106">Hao, Y. et al. Integrated analysis of multimodal single-cell data. <i>Cell</i> <b>184</b>, 3573–3587 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2021.04.048" data-track-item_id="10.1016/j.cell.2021.04.048" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2021.04.048" aria-label="Article reference 106" data-doi="10.1016/j.cell.2021.04.048">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlSrtrvE" aria-label="CAS reference 106">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34062119" aria-label="PubMed reference 106">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8238499" aria-label="PubMed Central reference 106">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 106" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrated%20analysis%20of%20multimodal%20single-cell%20data&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2021.04.048&amp;volume=184&amp;pages=3573-3587&amp;publication_year=2021&amp;author=Hao%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="107."><p class="c-article-references__text" id="ref-CR107">Wolock, S. L., Lopez, R. &amp; Klein, A. M. Scrublet: computational identification of cell doublets in single-cell transcriptomic data. <i>Cell Syst.</i> <b>8</b>, 281–291 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cels.2018.11.005" data-track-item_id="10.1016/j.cels.2018.11.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cels.2018.11.005" aria-label="Article reference 107" data-doi="10.1016/j.cels.2018.11.005">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyhtbw%3D" aria-label="CAS reference 107">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30954476" aria-label="PubMed reference 107">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6625319" aria-label="PubMed Central reference 107">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 107" href="http://scholar.google.com/scholar_lookup?&amp;title=Scrublet%3A%20computational%20identification%20of%20cell%20doublets%20in%20single-cell%20transcriptomic%20data&amp;journal=Cell%20Syst.&amp;doi=10.1016%2Fj.cels.2018.11.005&amp;volume=8&amp;pages=281-291&amp;publication_year=2019&amp;author=Wolock%2CSL&amp;author=Lopez%2CR&amp;author=Klein%2CAM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="108."><p class="c-article-references__text" id="ref-CR108">Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. <i>Nat. Methods</i> <b>16</b>, 1289–1296 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41592-019-0619-0" data-track-item_id="10.1038/s41592-019-0619-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-019-0619-0" aria-label="Article reference 108" data-doi="10.1038/s41592-019-0619-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOqsr7N" aria-label="CAS reference 108">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31740819" aria-label="PubMed reference 108">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6884693" aria-label="PubMed Central reference 108">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 108" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%2C%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0619-0&amp;volume=16&amp;pages=1289-1296&amp;publication_year=2019&amp;author=Korsunsky%2CI">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="109."><p class="c-article-references__text" id="ref-CR109">Gnaiger, E., Steinlechner-Maran, R., Méndez, G., Eberl, T. &amp; Margreiter, R. Control of mitochondrial and cellular respiration by oxygen. <i>J. Bioenerg. Biomembr.</i> <b>27</b>, 583–596 (1995).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1007/BF02111656" data-track-item_id="10.1007/BF02111656" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1007/BF02111656" aria-label="Article reference 109" data-doi="10.1007/BF02111656">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK28XhtF2gtw%3D%3D" aria-label="CAS reference 109">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8746845" aria-label="PubMed reference 109">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 109" href="http://scholar.google.com/scholar_lookup?&amp;title=Control%20of%20mitochondrial%20and%20cellular%20respiration%20by%20oxygen&amp;journal=J.%20Bioenerg.%20Biomembr.&amp;doi=10.1007%2FBF02111656&amp;volume=27&amp;pages=583-596&amp;publication_year=1995&amp;author=Gnaiger%2CE&amp;author=Steinlechner-Maran%2CR&amp;author=M%C3%A9ndez%2CG&amp;author=Eberl%2CT&amp;author=Margreiter%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="110."><p class="c-article-references__text" id="ref-CR110">Mongini, P. K. A. et al. TLR-9 and IL-15 synergy promotes the in vitro clonal expansion of chronic lymphocytic leukemia B cells. <i>J. Immunol.</i> <b>195</b>, 901–923 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.4049/jimmunol.1403189" data-track-item_id="10.4049/jimmunol.1403189" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.4049%2Fjimmunol.1403189" aria-label="Article reference 110" data-doi="10.4049/jimmunol.1403189">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFyjt7zJ" aria-label="CAS reference 110">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26136429" aria-label="PubMed reference 110">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4505957" aria-label="PubMed Central reference 110">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 110" href="http://scholar.google.com/scholar_lookup?&amp;title=TLR-9%20and%20IL-15%20synergy%20promotes%20the%20in%20vitro%20clonal%20expansion%20of%20chronic%20lymphocytic%20leukemia%20B%20cells&amp;journal=J.%20Immunol.&amp;doi=10.4049%2Fjimmunol.1403189&amp;volume=195&amp;pages=901-923&amp;publication_year=2015&amp;author=Mongini%2CPKA">
                    Google Scholar</a>&nbsp;
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01927-8?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>The authors thank the Hematopathology Collection registered at the Biobank of Hospital Clínic, Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS) and the Biobank HUB-ICO-IDIBELL (PT20/00171) for sample procurement, S. Martín, F. Arenas, the Genomics Core Facility of the IDIBAPS, CNAG Sequencing Unit, Mission Bio, Omniscope and Barcelona Supercomputing Center for the technical support and the computer resources at MareNostrum4 (RES activity, BCV-2018-3-0001). This study was supported by the la Caixa Foundation (CLLEvolution-LCF/PR/HR17/52150017, Health Research 2017 Program HR17-00221, to E.C.), the European Research Council under the European Union’s Horizon 2020 Research and Innovation Program (810287, BCLLatlas, to E.C., J.I.M.-S., H.H. and I.G.), the Instituto de Salud Carlos III and the European Regional Development Fund Una Manera de Hacer Europa (PMP15/00007 to E.C. and RTI2018-094584-B-I00 to D.C.), the American Association for Cancer Research (2021 AACR-Amgen Fellowship in Clinical/Translational Cancer Research, 21-40-11-NADE to F.N.), the European Hematology Association (EHA Junior Research Grant 2021, RG-202012-00245 to F.N.), the Lady Tata Memorial Trust (International Award for Research in Leukaemia 2021-2022, LADY_TATA_21_3223 to F.N.), the Generalitat de Catalunya Suport Grups de Recerca AGAUR (2017-SGR-1142 to E.C., 2017-SGR-736 to J.I.M.-S. and 2017-SGR-1009 to D.C.), the Accelerator award CRUK/AIRC/AECC joint funder partnership (AECC_AA17_SUBERO to J.I.M.-S.), the Fundació La Marató de TV3 (201924-30 to J.I.M.-S.), the Centro de Investigación Biomédica en Red Cáncer (CIBERONC; CB16/12/00225, CB16/12/00334, CB16/12/00236), the Ministerio de Ciencia e Innovación (PID2020-117185RB-I00 to X.S.P.), the Fundación Asociación Española Contra el Cáncer (FUNCAR-PRYGN211258SUÁR to X.S.P.), the Associazione Italiana per la Ricerca sul Cancro Foundation (AIRC 5 × 1,000 no. 21198 to G.G.) and the CERCA Programme/Generalitat de Catalunya. H.P.-A. is a recipient of a predoctoral fellowship from the Spanish Ministry of Science, Innovation and Universities (FPU19/03110). A.D.-N. is supported by the Department of Education of the Basque Government (PRE_2017_1_0100). E.C. is an Academia Researcher of the Institució Catalana de Recerca i Estudis Avançats of the Generalitat de Catalunya. This work was partially developed at the Center Esther Koplowitz (Barcelona, Spain).</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="nAff20"><p class="c-article-author-information__authors-list">Tycho Baumann</p><p class="js-present-address">Present address: Hospital Universitario 12 de Octubre, Madrid, Spain</p></li><li class="c-article-author-information__item" id="na1"><p>These authors contributed equally: Ferran Nadeu, Romina Royo, Ramon Massoni-Badosa, Heribert Playa-Albinyana, Beatriz Garcia-Torre.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Institut d’Investigacions Biomèdiques August Pi i Sunyer (IDIBAPS), Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Ferran Nadeu,&nbsp;Heribert Playa-Albinyana,&nbsp;Beatriz Garcia-Torre,&nbsp;Martí Duran-Ferrer,&nbsp;Marta Kulis,&nbsp;Neus Villamor,&nbsp;Vicente Chapaprieta,&nbsp;Julio Delgado,&nbsp;Ariadna Giró,&nbsp;Núria Verdaguer-Dot,&nbsp;Mónica Romo,&nbsp;Guillem Clot,&nbsp;Maria Rozman,&nbsp;Alfredo Rivas-Delgado,&nbsp;Marta Aymerich,&nbsp;Anna Enjuanes,&nbsp;Sílvia Ruiz-Gaspà,&nbsp;Armando López-Guillermo,&nbsp;Pedro Jares,&nbsp;Sílvia Beà,&nbsp;Dolors Colomer,&nbsp;José I. Martín-Subero&nbsp;&amp;&nbsp;Elías Campo</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Centro de Investigación Biomédica en Red de Cáncer (CIBERONC), Madrid, Spain</p><p class="c-article-author-affiliation__authors-list">Ferran Nadeu,&nbsp;Heribert Playa-Albinyana,&nbsp;Martí Duran-Ferrer,&nbsp;Ander Diaz-Navarro,&nbsp;Neus Villamor,&nbsp;Julio Delgado,&nbsp;Ariadna Giró,&nbsp;Guillem Clot,&nbsp;Tycho Baumann,&nbsp;Miguel Alcoceba,&nbsp;Marcos González,&nbsp;Marta Aymerich,&nbsp;Armando López-Guillermo,&nbsp;Pedro Jares,&nbsp;Sílvia Beà,&nbsp;Xose S. Puente,&nbsp;Dolors Colomer,&nbsp;José I. Martín-Subero&nbsp;&amp;&nbsp;Elías Campo</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Barcelona Supercomputing Center (BSC), Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Romina Royo,&nbsp;Ana Dueso-Barroso,&nbsp;Salvador Capella-Gutierrez,&nbsp;Josep Ll. Gelpí&nbsp;&amp;&nbsp;David Torrents</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">CNAG-CRG, Centre for Genomic Regulation (CRG), Barcelona Institute of Science and Technology (BIST), Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Ramon Massoni-Badosa,&nbsp;Sara Ruiz-Gil,&nbsp;Domenica Marchese,&nbsp;Ivo Gut&nbsp;&amp;&nbsp;Holger Heyn</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Wellcome Sanger Institute, Hinxton, UK</p><p class="c-article-author-affiliation__authors-list">Kevin J. Dawson&nbsp;&amp;&nbsp;Peter J. Campbell</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Departamento de Bioquímica y Biología Molecular, Instituto Universitario de Oncología, Universidad de Oviedo, Oviedo, Spain</p><p class="c-article-author-affiliation__authors-list">Ander Diaz-Navarro&nbsp;&amp;&nbsp;Xose S. Puente</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Hospital Clínic of Barcelona, Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Neus Villamor,&nbsp;Julio Delgado,&nbsp;Maria Rozman,&nbsp;Gerard Frigola,&nbsp;Alfredo Rivas-Delgado,&nbsp;Tycho Baumann,&nbsp;Marta Aymerich,&nbsp;Armando López-Guillermo,&nbsp;Pedro Jares,&nbsp;Sílvia Beà,&nbsp;Dolors Colomer&nbsp;&amp;&nbsp;Elías Campo</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Omniscope, Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Juan L. Melero</p></li><li id="Aff9"><p class="c-article-author-affiliation__address">Universitat de Barcelona, Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Julio Delgado,&nbsp;Armando López-Guillermo,&nbsp;Pedro Jares,&nbsp;Sílvia Beà,&nbsp;Josep Ll. Gelpí,&nbsp;Pablo M. Garcia-Roves,&nbsp;Dolors Colomer,&nbsp;José I. Martín-Subero&nbsp;&amp;&nbsp;Elías Campo</p></li><li id="Aff10"><p class="c-article-author-affiliation__address">Division of Hematology, Department of Translational Medicine, University of Eastern Piedmont, Novara, Italy</p><p class="c-article-author-affiliation__authors-list">Riccardo Moia&nbsp;&amp;&nbsp;Gianluca Gaidano</p></li><li id="Aff11"><p class="c-article-author-affiliation__address">Biología Molecular e Histocompatibilidad, IBSAL-Hospital Universitario, Centro de Investigación del Cáncer-IBMCC (USAL-CSIC), Salamanca, Spain</p><p class="c-article-author-affiliation__authors-list">Miguel Alcoceba&nbsp;&amp;&nbsp;Marcos González</p></li><li id="Aff12"><p class="c-article-author-affiliation__address">Hospital Universitari de Bellvitge-Institut d’Investigació Biomédica de Bellvitge (IDIBELL), L’Hospitalet de Llobregat, Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Fina Climent</p></li><li id="Aff13"><p class="c-article-author-affiliation__address">Department of Hematology, Vall d’Hebron Institute of Oncology, Vall d’Hebron University Hospital, Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Pau Abrisqueta,&nbsp;Josep Castellví&nbsp;&amp;&nbsp;Francesc Bosch</p></li><li id="Aff14"><p class="c-article-author-affiliation__address">Institute for Research in Biomedicine (IRB Barcelona), The Barcelona Institute of Science and Technology, Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Núria López-Bigas</p></li><li id="Aff15"><p class="c-article-author-affiliation__address">Universitat Pompeu Fabra (UPF), Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Núria López-Bigas,&nbsp;Ivo Gut&nbsp;&amp;&nbsp;Holger Heyn</p></li><li id="Aff16"><p class="c-article-author-affiliation__address">Institució Catalana de Recerca i Estudis Avançats (ICREA), Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Núria López-Bigas,&nbsp;David Torrents&nbsp;&amp;&nbsp;José I. Martín-Subero</p></li><li id="Aff17"><p class="c-article-author-affiliation__address">Oncology Institute of Southern Switzerland, Bellinzona, Switzerland</p><p class="c-article-author-affiliation__authors-list">Davide Rossi</p></li><li id="Aff18"><p class="c-article-author-affiliation__address">Institut d’Investigació Biomèdica de Bellvitge (IDIBELL), L’Hospitalet de Llobregat, Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Pablo M. Garcia-Roves</p></li><li id="Aff19"><p class="c-article-author-affiliation__address">Myeloma Service, Sylvester Comprehensive Cancer Center, University of Miami, Miami, FL, USA</p><p class="c-article-author-affiliation__authors-list">Francesco Maura</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Ferran-Nadeu-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Ferran Nadeu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ferran%20Nadeu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ferran%20Nadeu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ferran%20Nadeu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Romina-Royo-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Romina Royo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Romina%20Royo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Romina%20Royo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Romina%20Royo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ramon-Massoni_Badosa-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Ramon Massoni-Badosa</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ramon%20Massoni-Badosa" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ramon%20Massoni-Badosa" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ramon%20Massoni-Badosa%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Heribert-Playa_Albinyana-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Heribert Playa-Albinyana</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Heribert%20Playa-Albinyana" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Heribert%20Playa-Albinyana" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Heribert%20Playa-Albinyana%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Beatriz-Garcia_Torre-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Beatriz Garcia-Torre</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Beatriz%20Garcia-Torre" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Beatriz%20Garcia-Torre" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Beatriz%20Garcia-Torre%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Mart_-Duran_Ferrer-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Martí Duran-Ferrer</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Mart%C3%AD%20Duran-Ferrer" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Mart%C3%AD%20Duran-Ferrer" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Mart%C3%AD%20Duran-Ferrer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Kevin_J_-Dawson-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Kevin J. Dawson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Kevin%20J.%20Dawson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kevin%20J.%20Dawson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kevin%20J.%20Dawson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marta-Kulis-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Marta Kulis</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Marta%20Kulis" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marta%20Kulis" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marta%20Kulis%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ander-Diaz_Navarro-Aff2-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Ander Diaz-Navarro</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ander%20Diaz-Navarro" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ander%20Diaz-Navarro" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ander%20Diaz-Navarro%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Neus-Villamor-Aff1-Aff2-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Neus Villamor</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Neus%20Villamor" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Neus%20Villamor" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Neus%20Villamor%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Juan_L_-Melero-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Juan L. Melero</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Juan%20L.%20Melero" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Juan%20L.%20Melero" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Juan%20L.%20Melero%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Vicente-Chapaprieta-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Vicente Chapaprieta</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Vicente%20Chapaprieta" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Vicente%20Chapaprieta" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Vicente%20Chapaprieta%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ana-Dueso_Barroso-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Ana Dueso-Barroso</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ana%20Dueso-Barroso" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ana%20Dueso-Barroso" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ana%20Dueso-Barroso%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Julio-Delgado-Aff1-Aff2-Aff7-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Julio Delgado</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Julio%20Delgado" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Julio%20Delgado" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Julio%20Delgado%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Riccardo-Moia-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Riccardo Moia</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Riccardo%20Moia" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Riccardo%20Moia" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Riccardo%20Moia%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sara-Ruiz_Gil-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Sara Ruiz-Gil</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sara%20Ruiz-Gil" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sara%20Ruiz-Gil" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sara%20Ruiz-Gil%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Domenica-Marchese-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Domenica Marchese</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Domenica%20Marchese" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Domenica%20Marchese" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Domenica%20Marchese%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ariadna-Gir_-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Ariadna Giró</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ariadna%20Gir%C3%B3" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ariadna%20Gir%C3%B3" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ariadna%20Gir%C3%B3%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-N_ria-Verdaguer_Dot-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Núria Verdaguer-Dot</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=N%C3%BAria%20Verdaguer-Dot" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=N%C3%BAria%20Verdaguer-Dot" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22N%C3%BAria%20Verdaguer-Dot%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-M_nica-Romo-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Mónica Romo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=M%C3%B3nica%20Romo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=M%C3%B3nica%20Romo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22M%C3%B3nica%20Romo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Guillem-Clot-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Guillem Clot</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Guillem%20Clot" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Guillem%20Clot" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Guillem%20Clot%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Maria-Rozman-Aff1-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Maria Rozman</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Maria%20Rozman" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Maria%20Rozman" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Maria%20Rozman%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Gerard-Frigola-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Gerard Frigola</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Gerard%20Frigola" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Gerard%20Frigola" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Gerard%20Frigola%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Alfredo-Rivas_Delgado-Aff1-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Alfredo Rivas-Delgado</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Alfredo%20Rivas-Delgado" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Alfredo%20Rivas-Delgado" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alfredo%20Rivas-Delgado%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Tycho-Baumann-Aff2-Aff7-Aff20"><span class="c-article-authors-search__title u-h3 js-search-name">Tycho Baumann</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Tycho%20Baumann" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tycho%20Baumann" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tycho%20Baumann%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Miguel-Alcoceba-Aff2-Aff11"><span class="c-article-authors-search__title u-h3 js-search-name">Miguel Alcoceba</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Miguel%20Alcoceba" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Miguel%20Alcoceba" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Miguel%20Alcoceba%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marcos-Gonz_lez-Aff2-Aff11"><span class="c-article-authors-search__title u-h3 js-search-name">Marcos González</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Marcos%20Gonz%C3%A1lez" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marcos%20Gonz%C3%A1lez" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marcos%20Gonz%C3%A1lez%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Fina-Climent-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Fina Climent</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Fina%20Climent" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Fina%20Climent" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Fina%20Climent%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Pau-Abrisqueta-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Pau Abrisqueta</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Pau%20Abrisqueta" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Pau%20Abrisqueta" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Pau%20Abrisqueta%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Josep-Castellv_-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Josep Castellví</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Josep%20Castellv%C3%AD" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Josep%20Castellv%C3%AD" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Josep%20Castellv%C3%AD%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Francesc-Bosch-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Francesc Bosch</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Francesc%20Bosch" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Francesc%20Bosch" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Francesc%20Bosch%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marta-Aymerich-Aff1-Aff2-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Marta Aymerich</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Marta%20Aymerich" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marta%20Aymerich" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marta%20Aymerich%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Anna-Enjuanes-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Anna Enjuanes</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Anna%20Enjuanes" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anna%20Enjuanes" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anna%20Enjuanes%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-S_lvia-Ruiz_Gasp_-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Sílvia Ruiz-Gaspà</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=S%C3%ADlvia%20Ruiz-Gasp%C3%A0" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=S%C3%ADlvia%20Ruiz-Gasp%C3%A0" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22S%C3%ADlvia%20Ruiz-Gasp%C3%A0%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Armando-L_pez_Guillermo-Aff1-Aff2-Aff7-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Armando López-Guillermo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Armando%20L%C3%B3pez-Guillermo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Armando%20L%C3%B3pez-Guillermo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Armando%20L%C3%B3pez-Guillermo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Pedro-Jares-Aff1-Aff2-Aff7-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Pedro Jares</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Pedro%20Jares" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Pedro%20Jares" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Pedro%20Jares%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-S_lvia-Be_-Aff1-Aff2-Aff7-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Sílvia Beà</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=S%C3%ADlvia%20Be%C3%A0" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=S%C3%ADlvia%20Be%C3%A0" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22S%C3%ADlvia%20Be%C3%A0%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Salvador-Capella_Gutierrez-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Salvador Capella-Gutierrez</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Salvador%20Capella-Gutierrez" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Salvador%20Capella-Gutierrez" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Salvador%20Capella-Gutierrez%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Josep_Ll_-Gelp_-Aff3-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Josep Ll. Gelpí</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Josep%20Ll.%20Gelp%C3%AD" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Josep%20Ll.%20Gelp%C3%AD" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Josep%20Ll.%20Gelp%C3%AD%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-N_ria-L_pez_Bigas-Aff14-Aff15-Aff16"><span class="c-article-authors-search__title u-h3 js-search-name">Núria López-Bigas</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=N%C3%BAria%20L%C3%B3pez-Bigas" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=N%C3%BAria%20L%C3%B3pez-Bigas" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22N%C3%BAria%20L%C3%B3pez-Bigas%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-David-Torrents-Aff3-Aff16"><span class="c-article-authors-search__title u-h3 js-search-name">David Torrents</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=David%20Torrents" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=David%20Torrents" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22David%20Torrents%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Peter_J_-Campbell-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Peter J. Campbell</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Peter%20J.%20Campbell" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Peter%20J.%20Campbell" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Peter%20J.%20Campbell%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ivo-Gut-Aff4-Aff15"><span class="c-article-authors-search__title u-h3 js-search-name">Ivo Gut</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ivo%20Gut" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ivo%20Gut" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ivo%20Gut%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Davide-Rossi-Aff17"><span class="c-article-authors-search__title u-h3 js-search-name">Davide Rossi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Davide%20Rossi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Davide%20Rossi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Davide%20Rossi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Gianluca-Gaidano-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Gianluca Gaidano</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Gianluca%20Gaidano" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Gianluca%20Gaidano" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Gianluca%20Gaidano%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Xose_S_-Puente-Aff2-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Xose S. Puente</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Xose%20S.%20Puente" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Xose%20S.%20Puente" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Xose%20S.%20Puente%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Pablo_M_-Garcia_Roves-Aff9-Aff18"><span class="c-article-authors-search__title u-h3 js-search-name">Pablo M. Garcia-Roves</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Pablo%20M.%20Garcia-Roves" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Pablo%20M.%20Garcia-Roves" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Pablo%20M.%20Garcia-Roves%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Dolors-Colomer-Aff1-Aff2-Aff7-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Dolors Colomer</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Dolors%20Colomer" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Dolors%20Colomer" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Dolors%20Colomer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Holger-Heyn-Aff4-Aff15"><span class="c-article-authors-search__title u-h3 js-search-name">Holger Heyn</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Holger%20Heyn" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Holger%20Heyn" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Holger%20Heyn%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Francesco-Maura-Aff19"><span class="c-article-authors-search__title u-h3 js-search-name">Francesco Maura</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Francesco%20Maura" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Francesco%20Maura" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Francesco%20Maura%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jos__I_-Mart_n_Subero-Aff1-Aff2-Aff9-Aff16"><span class="c-article-authors-search__title u-h3 js-search-name">José I. Martín-Subero</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jos%C3%A9%20I.%20Mart%C3%ADn-Subero" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jos%C3%A9%20I.%20Mart%C3%ADn-Subero" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jos%C3%A9%20I.%20Mart%C3%ADn-Subero%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-El_as-Campo-Aff1-Aff2-Aff7-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Elías Campo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=El%C3%ADas%20Campo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=El%C3%ADas%20Campo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22El%C3%ADas%20Campo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>F.N. designed the study, collected samples and data, analyzed genomic, immunogenetic and transcriptomic data, interpreted data, designed the figures and wrote the manuscript. R.R. centralized data collection and analyzed and interpreted WGS and bulk RNA-seq data. R.M.-B. analyzed and interpreted scRNA-seq data. H.P.-A. performed and interpreted calcium flux and cell growth experiments and contributed to respiration experiments. B.G.-T. analyzed and interpreted H3K27ac and ATAC-seq data. M.D.-F. analyzed and interpreted DNA methylation data. K.J.D. provided code for the WGS-based subclonal reconstruction and interpreted the results. M.K., A.D.-N., J.L.M., V.C., A.D.-B., S.R.-G., A.G., D.M., N.V.-D., M. Romo, G.C., M. Rozman, G.F. and A.E. performed experiments, analyzed data and/or interpreted data. N.V. conducted flow cytometry analyses. S.R.-G. provided logistical assistance. J.D., R.M., A.R.-D., T.B., M.A., M.G., F.C., P.A., J.C., F.B., M.A., D.R. and G.G. contributed samples and/or clinical data. A.L.G., P.J., S.B., S.C.-G., J.L.G., N.L.-B., D.T., P.J.C., I.G. and X.S.P. interpreted data. P.M.G.-R. designed, conducted and interpreted respiration experiments. D.C. supervised calcium flux and cell growth experiments and interpreted data. H.H. supervised single-cell experiments and analyses and interpreted data. F.M. contributed to the design and interpretation of WGS analyses. J.I.M.-S. supervised epigenomic experiments and analyses and interpreted data. E.C. designed the study, reviewed pathology, interpreted data, supervised the research and wrote the manuscript. All authors read, commented on and approved the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:nadeu@recerca.clinic.cat">Ferran Nadeu</a> or <a id="corresp-c2" href="mailto:ecampo@clinic.cat">Elías Campo</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar4">Competing interests</h3>
                <p>F.N. has received honoraria from Janssen and AbbVie for speaking at educational activities. J.L.M. is an employee of Omniscope. X.S.P. is cofounder of and holds an equity stake in DREAMgenics. H.H. is cofounder of Omniscope and consultant to MiRXES. E.C. has been a consultant for Takeda, NanoString, AbbVie and Illumina; has received honoraria from Janssen, EUSPharma and Roche for speaking at educational activities; and is an inventor on a Lymphoma and Leukemia Molecular Profiling Project patent ‘Method for subtyping lymphoma subtypes by means of expression profiling’ (PCT/US2014/64161) not related to this project. The remaining authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
              
              
                <h3 class="c-article__sub-heading" id="FPar3">Peer review information</h3>
                <p><i>Nature Medicine</i> thanks Daniel Hodson and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling editor: Anna Maria Ranzoni, in collaboration with the <i>Nature Medicine</i> team.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec62-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec62">Extended data</h2><div class="c-article-section__content" id="Sec62-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 cohort studied and types of r" href="/articles/s41591-022-01927-8/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig6_ESM.jpg">Extended Data Fig. 1 Cohort studied and types of Richter transformation.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>. Representation of the disease course of the patients included in the study. Each sample analyzed, treatment and date of RT are depicted. Patients labeled in gray lacked germline DNA. Patient 4676 also lacked DNA from the previous CLL sample. Patients are grouped based on the last line of therapy received before RT in three groups: patients developing RT before any treatment, after chemo(immuno)therapy, and after targeted therapy. The type of transformation (RT-DLBCL, diffuse large B cell lymphoma type; RT-PLL, prolymphocytic transformation; RT-PBL, plasmablastic transformation) and IGHV mutational status are also shown. Additional molecular studies conducted in each case are also depicted. Abbreviations: Ale: alemtuzumab; AlloSCT: allogenic stem-cell transplantation; AutoSCT: autologous stem-cell transplantation; B: bendamustine; Burkimab: rituximab, methotrexate, dexametasone, ifosfamide, vincristine, etoposide, cytarabine, doxorubicin and vindesine; C: cyclophosphamide; CHOP: cyclophosphamide, doxorubicin, vincristine and prednisone; CLB: chlorambucil; CLB-R: chlorambucil and rituximab; CP: cyclophosphamide and prednisone; F: fludarabine; FCM: fludarabine, cyclophosphamide and mitoxantrone; G-GemOx: rituximab, gemcitabine, and oxaliplatin; LR-ESHAP: lenalidomide, rituximab, etoposide, methyl-prednisolone, cytarabine and cisplatin; M: mitoxantrone; Prd: prednisone; R: rituximab; R-B: rituximab and bendamustine; R-CHOP: rituximab, cyclophosphamide, doxorubicin, vincristine and prednisone; R-CVP: rituximab, cyclophosphamide, vincristine and prednisone; R-DHAP: rituximab, dexamethasone, cytarabine and cisplatin; R-ESHAP: rituximab, etoposide, methyl-prednisolone, cytarabine and cisplatin; RFC: fludarabine, cyclophosphamide and rituximab; RFCM: rituximab, fludarabine, cyclophosphamide and mitoxantrone; R-ICE: rituximab, ifosfamide, carboplatin and etoposide; TBI: total body irradiation. <b>b</b>. Morphology of the RT-DLBCL of patient 63 (hematoxylin-eosin, H&amp;E, staining). <b>c</b>. Morphology of the RT-DLBCL of patient 365 and Ki67 staining showing high proliferative index. <b>d</b>. Morphology of the RT-DLBCL of patient 816. <b>e</b>. Morphology of the RT-PLL of patient 3299. <b>f</b>. Morphology of the RT-PBL of patient 1669 (H&amp;E staining), which was negative for CD20 and PAX5, while positive for MUM1/IRF4. Each experiment for <b>b-f</b> was repeated twice. The scale bars in <b>b-f</b> represents 20 μm.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 genetic and epigenetic change" href="/articles/s41591-022-01927-8/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig7_ESM.jpg">Extended Data Fig. 2 Genetic and epigenetic changes from CLL to RT, CNA profiles, and landscape of driver alterations.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>. Number of somatic genetic alterations and epigenetic changes compared to normal counterparts along the course of the disease. Cases/time points with no grid lines correspond to unavailable data. <b>b</b>. Mutational burden, number of CNAs and number of SVs found in RT stratified according to the last therapy prior transformation. Targeted, targeted therapies. center line, median; box limits, upper/lower quartiles; whiskers, 1.5×interquartile range; points, individual samples. <b>c</b>. Copy number landscape of the studied cohort grouped by patient. The diagnosis, IGHV mutational status, last therapy prior RT, and total number of CNAs are indicated for each time point. <b>d</b>. Aggregated copy number profile of RT vs CLL. The first CLL samples (time point 1, T1) were considered. The plot shows the percentage of samples with gains (up) and losses (down). Among recurrent alterations found either in CLL or RT samples (n ≥ 5), deletions of 9p (<i>PTPRD</i> and <i>CDKN2A/B</i>) and deletions of 15q (<i>MGA</i>) were enriched in RT whereas deletions of <i>ATM</i> (11q), <i>TP53</i> (17p), and 13q14 were found at similar frequencies in CLL and RT. <b>e</b>. Oncoprint of putative driver alterations. Samples, grouped by patient (patient id at the top), are represented by columns while genes in rows. Novel drivers in RT are labeled in blue. Genes are grouped according to their biological function or if they were previously described as potential driver genes in CLL and/or mature B cell lymphomas. Metadata including the type of therapy before RT, number of treatment lines before each sample, the spatial/longitudinal nature of the CLL/RT samples analyzed, IGHV mutational status, and diagnosis is detailed in the upper rows. In the main plot, mutations (SNVs and indels) are depicted with horizontal rectangles, CNAs using the background color of each cell, and SVs with vertical rectangles. The transparency of the color of mutations and CNAs indicates the cancer cell fraction (CCF). For patients lacking the germline sample (patient id indicated in gray), the CCF of the alterations could not be inferred and a CCF of 100% was used for illustrative purposes.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 complex genomic rearrangement" href="/articles/s41591-022-01927-8/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig8_ESM.jpg">Extended Data Fig. 3 Complex genomic rearrangements affecting driver genes.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>. Deletions in chr12 identified in four cases with the minimal deleted region affecting <i>CDKN1B</i>, which expression in CLL and RT sample pairs is shown on the right. The case carrying the deletion at time of RT is labeled in the boxplot. <b>b</b>. Reciprocal translocation juxtaposing <i>CDK6</i> next to IGKJ5 in patient 4687. <b>c</b>. Deletion in chr1 affecting two cases with the minimal deleted region targeting <i>ARID4B</i>. Its expression in CLL and RT sample pairs is shown in the boxplot on the right. <b>d</b>. Reciprocal translocations truncating <i>CREBBP</i> and <i>CIITA</i> in the RT sample of patient 12. <b>e</b>. Expression levels of known and novel RT-driver genes in CLL and RT paired samples. Cases carrying deletions/mutations at time of RT are labeled. <b>f-j</b>. Complex genomic rearrangements affecting driver genes in five selected RT samples. The circos plots show the SVs (inner links) and CNAs (middle circle) found in each sample. SVs are colored based on whether they are part of a complex event, while CNAs are painted according to their type. Chromosome-specific plots on the right show the main chromosomes affected by complex events targeting driver genes (annotated at the bottom). In these chromosome-specific plots, the color of both CNAs and SVs indicates their type. For patient 12 (f), the expression levels of three genes affected by simple (<i>TRAF3</i>) and complex (<i>SPEN</i> and <i>TNFRS14</i>) chromosomal alterations are shown. For patient 4675 (j), the partner of the translocations found in chr3 and chr8 are not specified for simplicity due to the high number of clustered structural events. All boxplots: center line, median; box limits, upper/lower quartiles; whiskers, 1.5×interquartile range; points, individual samples. All p values are from two-sided T tests.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 extraction and assignment of " href="/articles/s41591-022-01927-8/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig9_ESM.jpg">Extended Data Fig. 4 Extraction and assignment of mutational signatures.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a–d</b>. Signatures extracted by the Hierarchical Dirichlet Process (HDP) (a), SignatureAnalyzer (b), SigProfiler (c), and sigfit (d). COSMIC signatures needed to reconstruct the extracted signatures are shown together with their contribution (in percentage). The cosine similarities between the extracted and reconstructed signatures are shown in brackets. <b>e</b>. Workflow of the mutational signature analysis. <b>f</b>. The 96-mutation profile of the RT sample of patient 839 (time point 2), which had marked evidence of APOBEC activity (SBS2 and SBS13). <b>g</b>. Comparison of the SBS-ganciclovir extracted by HDP and SignatureAnalyzer. Based on the high cosine similarity (0.996), we considered that both signatures represented the same mutational process and selected the one extracted by HDP for downstream analyses. <b>h</b>. Comparison of the SBS-ganciclovir extracted by HDP and the ganciclovir signature reported by de Kanter et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="de Kanter, J. K. et al. Antiviral treatment causes a unique mutational signature in cancers of transplantation recipients. Cell Stem Cell 28, 1726–1739 (2021)." href="/articles/s41591-022-01927-8#ref-CR35" id="ref-link-section-d73061417e4313">35</a></sup>. <b>i</b>. Comparison of the SBS-RT extracted by HDP and SignatureAnalyzer. Based on the high cosine similarity (0.941), we considered that both signatures represented the same mutational process and selected the one extracted by HDP for downstream analyses. <b>j</b>. Pairwise comparisons of the SBS-RT with known signatures from COSMIC and Kucab et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Kucab, J. E. et al. A compendium of mutational signatures of environmental agents. Cell 177, 821–836 (2019)." href="/articles/s41591-022-01927-8#ref-CR33" id="ref-link-section-d73061417e4324">33</a></sup>. <b>k</b>. Decomposition of the SBS-RT in “n” known signatures using an expectation maximization approach. The low cosine similarity (&lt;0.85) between SBS-RT and the best reconstituted signature obtained using any combination of known signatures suggests that SBS-RT represents a novel mutational signature.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 fitting of mutational signatu" href="/articles/s41591-022-01927-8/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig10_ESM.jpg">Extended Data Fig. 5 Fitting of mutational signatures, characterization of SBS-RT, and co-occurrence of RT subclones.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>. Mutational processes in ICGC-CLL (left) and post-treatment CLL (right) cohorts. <b>b</b>. Correlation of SBS-RT with the total number of SNVs and other mutational processes in RT subclones. Gray area, 95% confidence interval. <b>c</b>. Activity of the mutational processes identified in regulatory regions of the genome: heterochromatin (Het), polycomb (Pol), enhancer/promoter (EP), and transcription (Tra). The heat map (right) shows the log2-fold change of the observed vs expected number of SBS-RT mutations/region. <b>d</b>. Contribution of the mutational processes in early/late replication regions. <b>e-f</b>. Replication (e) and transcriptional (f) strand bias of the mutational profile of RT subclones with SBS-RT. The main peaks of the SBS-RT are indicated with their context on the x-axis. Significant asymmetries are indicated with asterisks (exact <i>p</i> values are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">16</a>). <b>g</b>. Number of CNAs and SVs in RT samples. <b>h</b>. Detection (top) and variant allele frequency (VAF) (bottom) of mutations assigned to the RT subclone during the disease course in patient 19 based on UMI-based NGS. Mutations are grouped according to the main peaks of SBS-RT. <i>P</i> values by Fisher’s test. L.C., low confidence; H.C., high confidence. Density plot showing the distribution of the cancer cell fraction (CCF) of the SNVs assigned to the RT subclone by WGS (bottom right). <b>i</b>. Mutational profiles of kataegis in ICGC-CLL samples (row 1–2), CLL subclones from the present CLL/RT cohort (row 3–4), and RT subclones (all U-CLL) (row 5). Mutational processes identified are indicated together with its contribution and cosine similarity to the reconstructed profile. <b>j</b>. Immunoglobulin genes of two cases harboring RT-specific SNVs at time of RT (time points, T, highlighted in rose). PB, peripheral blood. BM, bone marrow. <b>k</b>. Complete flow cytometry analysis in case 12. Numbers along axes are divided by 1000. <b>l</b>. Density plot showing the comparison of the CCF of the SNVs of synchronous BM and PB samples analyzed in patient 12. <b>m</b>. Circos plots of the BM samples of patient 12 for comparison with the rearrangements observed at PB (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM1">1</a>).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 clonal dynamics from cll to r" href="/articles/s41591-022-01927-8/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig11_ESM.jpg">Extended Data Fig. 6 Clonal dynamics from CLL to RT.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>. Subclonal reconstruction and clonal evolution of three cases (3299, 12 and 365) with WGS and scDNA-seq data available. The upper fish plot shows the clonal evolution along the course of the disease inferred from WGS analyses. Each color represents a different subclone and their height is proportional to their cancer cell fraction (CCF) in each time point (vertical lines). The treatments that the patient received and the elapsed time (in years) between samples are indicated at the top. The tissue is indicated for samples of patient 3299 in which different tissues were analyzed by WGS and scDNA-seq in the same time point. The phylogeny of the subclones is depicted together with the main driver alterations (top right). The lower bar plots show the dynamics of the different subclones according to the scDNA-seq analyses. The total number of cells per sample is shown at the bottom. The number of cells assigned to each subclone can be found in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01927-8#MOESM3">20</a>. The mutation tree inferred from scDNA-seq data is shown at the bottom-right part. <b>b-c</b>. Subclonal architecture and dynamics of six cases with longitudinal samples (b) and two cases with spatial samples (c) analyzed by WGS.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 scrna-seq characterization of" href="/articles/s41591-022-01927-8/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig12_ESM.jpg">Extended Data Fig. 7 scRNA-seq characterization of CLL and RT.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a–d</b>. UMAP visualization of tumor cells from all time points colored by annotation and tissue of origin. hi, high; lo, low; PB, peripheral blood; LN, lymph node; BM, bone marrow (left). Dot plot with the expression of key markers in each cluster. Color and size represent scaled mean expression and proportion of cells expressing each marker gene, respectively (middle-left). Violin plots showing the cell-cycle phase scores (S and G-to-M) for each cluster of cells (middle-right). UMAP visualization split by time point (right). ‘n’ refers to the total number of cells in that time point, and the percentage refers to the proportion of cells within RT clusters. <b>e-i</b>. Time point-specific UMAP visualizations for each case. RT seed cells are depicted in rose and with an increased size. <b>j</b>. UMAP visualization of case CLL9 from Penter et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Penter, L. et al. Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history. Cancer Discov. 11, 3048–3063 (2021)." href="/articles/s41591-022-01927-8#ref-CR43" id="ref-link-section-d73061417e4463">43</a></sup> split by time point. PB, peripheral blood; BM, bone marrow.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 cna profile of rt cells by sc" href="/articles/s41591-022-01927-8/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig13_ESM.jpg">Extended Data Fig. 8 CNA profile of RT cells by scRNA-seq.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>For each patient, the CNA profile of CLL and RT samples according to WGS is shown (top) together with the CNA profile of each individual RT cell based on scRNA-seq (bottom). For scRNA-seq, each row represents a RT cell and the horizontal dashed line separates the RT cells identified in the time points previous to the diagnosis of RT (that is, seed RT cells) from those present in the sample collected at time of diagnosis of RT. Note that CLL cells were used as reference for CNA analyses using scRNA-seq data.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 epigenomic characterization o" href="/articles/s41591-022-01927-8/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig14_ESM.jpg">Extended Data Fig. 9 Epigenomic characterization of RT.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>. Heatmaps showing the regions with decreased H3K27ac, increased ATAC, and decreased ATAC levels, respectively, in RT. <b>b</b>. Overlap of differentially expressed genes by bulk RNA-seq with regions with increased or decreased H3K27ac and ATAC levels, respectively. <b>c</b>. Heat map showing differentially methylated CpGs (DMC) between CLL and RT. Normal B cells, CLL, CLL at relapse, and RT samples are shown separately with different biological information on top. The correlation of each CpG with the epiCMIT is depicted on the right. To note, the epiCMIT is associated with the gain and loss of methylation upon cell division, but its transformation to 0-1 scale (for interpretability purposes) makes it anticorrelated with hypomethylation, as the epiCMIT=max{epiCMIT-hyper, epiCMIT-hypo}, being the epiCMIT-hyper=hypermethylation, and the epiCMIT-hypo=1-hypomethyaltion at relevant CpGs, as originally reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Duran-Ferrer, M. et al. The proliferative history shapes the DNA methylome of B-cell tumors and predicts clinical outcome. Nat. Cancer 1, 1066–1081 (2020)." href="/articles/s41591-022-01927-8#ref-CR49" id="ref-link-section-d73061417e4518">49</a></sup>. <b>d</b>. Genomic enrichment over the background for hyper- and hypomethylated CpGs in CLL vs RT. <b>e</b>. DMC distribution based on their genetic annotation and their intersection with differentially expressed genes by bulk RNA-seq analyses. <b>f</b>. DMC distribution based on the correlation of each CpG with the epiCMIT and their p values. CpGs were piled up in color-coded bins based on the number of CpGs in each bin to avoid overplotting. <b>g</b>. epiCMIT evolution in longitudinal CLL and RT samples, with the epiCMIT-hyper and epiCMIT-hypo scores depicted separately (RT samples being the last time point labeled in rose). The epiCMIT score used to compare among samples is the greater of the two (hyper and hypo).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig15"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 transcriptomic characterizat" href="/articles/s41591-022-01927-8/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig15_ESM.jpg">Extended Data Fig. 10 Transcriptomic characterization of RT.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>. Volcano plot of the differential expression analysis (RT vs CLL, bulk RNA-seq). <b>b</b>. Expression levels of selected genes in CLL and RT according to bulk RNA-seq. center line, median; box limits, upper/lower quartiles; whiskers, 1.5×interquartile range; points, individual samples. <b>c</b>. Differentially expressed genes (RT vs CLL) for each case by scRNA-seq. <b>d</b>. GSEA plots of selected hallmark gene sets according to bulk RNA-seq analyses. NES, normalized enrichment score. <b>e</b>. UpSet plots highlighting the intersections of the case-specific upregulated (top) and downregulated (bottom) GO terms in RT by scRNA-seq. <b>f</b>. GSEA plots for the terms oxidative phosphorylation (OXPHOS), mitochondrial translation, and BCR signaling pathway for cases 12, 63, and 365 based on scRNA-seq. <b>g-j</b>. scRNA-seq-derived UMAP visualization of tumor cells from all time points colored by OXPHOS and BCR signaling score (left). Ridge plots showing the same scores across clusters (middle). Violin plots displaying the same scores across time points, stratified by CLL and RT clusters (right). <b>k</b>. Violin plots displaying the OXPHOS and BCR signaling scores across time points, stratified by CLL and RT clusters, in case CLL9 from Penter et al<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Penter, L. et al. Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history. Cancer Discov. 11, 3048–3063 (2021)." href="/articles/s41591-022-01927-8#ref-CR43" id="ref-link-section-d73061417e4581">43</a></sup>. <b>l</b>. GSEA between RT and CLL cells of patient CLL9 from Penter et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Penter, L. et al. Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history. Cancer Discov. 11, 3048–3063 (2021)." href="/articles/s41591-022-01927-8#ref-CR43" id="ref-link-section-d73061417e4588">43</a></sup>.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec63-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec63">Supplementary information</h2><div class="c-article-section__content" id="Sec63-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>List of Supplementary Tables and Supplementary Figures.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_MOESM2_ESM.pdf" data-supp-info-image="">Reporting summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary tables" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_MOESM3_ESM.xlsx" data-supp-info-image="">Supplementary Tables</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Tables 1–25.</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Detection%20of%20early%20seeding%20of%20Richter%20transformation%20in%20chronic%20lymphocytic%20leukemia&amp;author=Ferran%20Nadeu%20et%20al&amp;contentID=10.1038%2Fs41591-022-01927-8&amp;copyright=The%20Author%28s%29&amp;publication=1078-8956&amp;publicationDate=2022-08-11&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41591-022-01927-8" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41591-022-01927-8" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Nadeu, F., Royo, R., Massoni-Badosa, R. <i>et al.</i> Detection of early seeding of Richter transformation in chronic lymphocytic leukemia.
                    <i>Nat Med</i> <b>28</b>, 1662–1671 (2022). https://doi.org/10.1038/s41591-022-01927-8</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01927-8?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-11-10">10 November 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-07-01">01 July 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-08-11">11 August 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-08">August 2022</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41591-022-01927-8</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-block"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/b-cell-lymphoma" data-track="click" data-track-action="view subject" data-track-label="link">B-cell lymphoma</a></li><li class="c-article-subject-list__subject"><a href="/subjects/cancer-epigenetics" data-track="click" data-track-action="view subject" data-track-label="link">Cancer epigenetics</a></li><li class="c-article-subject-list__subject"><a href="/subjects/cancer-genomics" data-track="click" data-track-action="view subject" data-track-label="link">Cancer genomics</a></li><li class="c-article-subject-list__subject"><a href="/subjects/chronic-lymphocytic-leukaemia" data-track="click" data-track-action="view subject" data-track-label="link">Chronic lymphocytic leukaemia</a></li><li class="c-article-subject-list__subject"><a href="/subjects/tumour-heterogeneity" data-track="click" data-track-action="view subject" data-track-label="link">Tumour heterogeneity</a></li>
        </ul>
    </div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section" data-test="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Towards understanding cancer dormancy over strategic hitching up mechanisms to technologies" href="https://doi.org/10.1186/s12943-025-02250-9">
                                        Towards understanding cancer dormancy over strategic hitching up mechanisms to technologies
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Sumin Yang</li><li>Jieun Seo</li><li>Jae-Yeol Joo</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Molecular Cancer</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Bcor loss promotes Richter transformation of chronic lymphocytic leukemia associated with Notch1 activation in mice" href="https://doi.org/10.1038/s41375-025-02557-y">
                                        Bcor loss promotes Richter transformation of chronic lymphocytic leukemia associated with Notch1 activation in mice
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Chiara Rompietti</li><li>Francesco Maria Adamo</li><li>Paolo Sportoletti</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Leukemia</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Complementary approaches define the metabolic features that accompany Richter syndrome transformation" href="https://doi.org/10.1007/s00018-025-05670-4">
                                        Complementary approaches define the metabolic features that accompany Richter syndrome transformation
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Giulia Omezzolli</li><li>Andrea Iannello</li><li>Tiziana Vaisitti</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Cellular and Molecular Life Sciences</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Incidence of Richter transformation of chronic lymphocytic leukemia/small lymphocytic lymphoma in the targeted therapy era" href="https://doi.org/10.1038/s41375-024-02492-4">
                                        Incidence of Richter transformation of chronic lymphocytic leukemia/small lymphocytic lymphoma in the targeted therapy era
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Paul J. Hampel</li><li>Kari G. Rabe</li><li>Sameer A. Parikh</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Leukemia</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Single nucleus DNA sequencing of flow sorted archived frozen and formalin fixed paraffin embedded solid tumors" href="https://doi.org/10.1186/s12864-024-10850-w">
                                        Single nucleus DNA sequencing of flow sorted archived frozen and formalin fixed paraffin embedded solid tumors
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Yumei Zhou</li><li>Xianfeng Chen</li><li>Michael T. Barrett</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>BMC Genomics</i> (2024)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01927-8.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                
            
        
    </div>

    
        
    

    
    
        <div class="c-article-associated-content__container">
            <section>
                <h2 class="c-article-associated-content__title u-mb-24">Associated content</h2>
                
                    
                    
                        <div class="u-full-height u-mb-24">
                            
    <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
            <div class="c-card__body">
                <h3 class="c-card__title">
                    <a href="https://www.nature.com/articles/s41591-022-01928-7" class="c-card__link u-link-inherit" data-track="click" data-track-action="view article" data-track-category="associated content" data-track-label="news_and_views">The clonal evolution of Richter transformation cells uncovers therapeutic vulnerabilities</a>
                </h3>
                
                
    <div class="c-card__section c-meta">
        
            <span class="c-meta__item">Nature Medicine</span>
        
        <span class="c-meta__item" data-test="article.type"><span class="c-meta__type">Research Briefing</span></span>
        
        
            <time class="c-meta__item" datetime="2022-08-11">11 Aug 2022</time>
        
    </div>

            </div>
        </div>
    </article>


                        </div>
                    
                
            </section>
        </div>
        <script>
            window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "news_and_views";
            
        </script>
    

    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
            <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" role="tabpanel"><div class="c-reading-companion__scroll-pane" style="max-height: none;"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Main">Main</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec8" class="c-reading-companion__section-item"><a href="#Sec8" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec9" class="c-reading-companion__section-item"><a href="#Sec9" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-data-availability" class="c-reading-companion__section-item"><a href="#data-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Data availability">Data availability</a></li><li id="rc-sec-code-availability" class="c-reading-companion__section-item"><a href="#code-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Code availability">Code availability</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-peer-review" class="c-reading-companion__section-item"><a href="#peer-review" data-track="click" data-track-action="section anchor" data-track-label="link:Peer review">Peer review</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec62" class="c-reading-companion__section-item"><a href="#Sec62" data-track="click" data-track-action="section anchor" data-track-label="link:Extended data">Extended data</a></li><li id="rc-sec-Sec63" class="c-reading-companion__section-item"><a href="#Sec63" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by">This article is cited by</a></li></ul></div>
                <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="" style="display: none !important;">
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-test="right-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41591-022-01927-8;doi=10.1038/s41591-022-01927-8;subjmeta=1621,1895,1915,1990,2329,2486,283,291,631,67,68,69;kwrd=B-cell+lymphoma,Cancer+epigenetics,Cancer+genomics,Chronic+lymphocytic+leukaemia,Tumour+heterogeneity">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=1736642272&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01927-8%26doi%3D10.1038/s41591-022-01927-8%26subjmeta%3D1621,1895,1915,1990,2329,2486,283,291,631,67,68,69%26kwrd%3DB-cell+lymphoma,Cancer+epigenetics,Cancer+genomics,Chronic+lymphocytic+leukaemia,Tumour+heterogeneity">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=1736642272&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01927-8%26doi%3D10.1038/s41591-022-01927-8%26subjmeta%3D1621,1895,1915,1990,2329,2486,283,291,631,67,68,69%26kwrd%3DB-cell+lymphoma,Cancer+epigenetics,Cancer+genomics,Chronic+lymphocytic+leukaemia,Tumour+heterogeneity"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Fig. 1: The genomic landscape of RT.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig1_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig1_HTML.png" alt="figure 1" aria-describedby="rc-Fig1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Fig. 2: Mutational processes in RT.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig2_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig2_HTML.png" alt="figure 2" aria-describedby="rc-Fig2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Fig. 3: Early seeding of RT.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig3_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig3_HTML.png" alt="figure 3" aria-describedby="rc-Fig3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Fig. 4: Proliferation, OXPHOS and BCR pathways dominate the epigenome and transcriptome of RT.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig4_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig4_HTML.png" alt="figure 4" aria-describedby="rc-Fig4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Fig. 5: Cellular respiration, BCR signaling and OXPHOS inhibition in RT cells.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig5_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig5_HTML.png" alt="figure 5" aria-describedby="rc-Fig5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig6">Extended Data Fig. 1 Cohort studied and types of Richter transformation.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig6_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig6_ESM.jpg" alt="extended data figure 6" aria-describedby="rc-Fig6"></picture><p class="c-reading-companion__figure-links"><a href="#Fig6" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/6" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig7">Extended Data Fig. 2 Genetic and epigenetic changes from CLL to RT, CNA profiles, and landscape of driver alterations.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig7_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig7_ESM.jpg" alt="extended data figure 7" aria-describedby="rc-Fig7"></picture><p class="c-reading-companion__figure-links"><a href="#Fig7" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/7" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig8">Extended Data Fig. 3 Complex genomic rearrangements affecting driver genes.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig8_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig8_ESM.jpg" alt="extended data figure 8" aria-describedby="rc-Fig8"></picture><p class="c-reading-companion__figure-links"><a href="#Fig8" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/8" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig9">Extended Data Fig. 4 Extraction and assignment of mutational signatures.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig9_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig9_ESM.jpg" alt="extended data figure 9" aria-describedby="rc-Fig9"></picture><p class="c-reading-companion__figure-links"><a href="#Fig9" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/9" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig10">Extended Data Fig. 5 Fitting of mutational signatures, characterization of SBS-RT, and co-occurrence of RT subclones.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig10_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig10_ESM.jpg" alt="extended data figure 10" aria-describedby="rc-Fig10"></picture><p class="c-reading-companion__figure-links"><a href="#Fig10" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/10" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig11">Extended Data Fig. 6 Clonal dynamics from CLL to RT.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig11_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig11_ESM.jpg" alt="extended data figure 11" aria-describedby="rc-Fig11"></picture><p class="c-reading-companion__figure-links"><a href="#Fig11" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/11" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig12">Extended Data Fig. 7 scRNA-seq characterization of CLL and RT.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig12_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig12_ESM.jpg" alt="extended data figure 12" aria-describedby="rc-Fig12"></picture><p class="c-reading-companion__figure-links"><a href="#Fig12" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/12" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig13">Extended Data Fig. 8 CNA profile of RT cells by scRNA-seq.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig13_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig13_ESM.jpg" alt="extended data figure 13" aria-describedby="rc-Fig13"></picture><p class="c-reading-companion__figure-links"><a href="#Fig13" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/13" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig14">Extended Data Fig. 9 Epigenomic characterization of RT.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig14_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig14_ESM.jpg" alt="extended data figure 14" aria-describedby="rc-Fig14"></picture><p class="c-reading-companion__figure-links"><a href="#Fig14" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/14" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig15">Extended Data Fig. 10 Transcriptomic characterization of RT.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig15_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01927-8/MediaObjects/41591_2022_1927_Fig15_ESM.jpg" alt="extended data figure 15" aria-describedby="rc-Fig15"></picture><p class="c-reading-companion__figure-links"><a href="#Fig15" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01927-8/figures/15" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric" data-track-context="article references sidebar"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">Cairns, J. Mutation selection and the natural history of cancer. <i>Nature</i> <b>255</b>, 197–200 (1975).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2F255197a0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/255197a0" data-track-item_id="10.1038/255197a0">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaE2MXkvFCmsb0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=1143315" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Mutation%20selection%20and%20the%20natural%20history%20of%20cancer&amp;journal=Nature&amp;doi=10.1038%2F255197a0&amp;volume=255&amp;pages=197-200&amp;publication_year=1975&amp;author=Cairns%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">Nowell, P. C. The clonal evolution of tumor cell populations. <i>Science</i> <b>194</b>, 23–28 (1976).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.959840" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.959840" data-track-item_id="10.1126/science.959840">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaE283ntFWmsw%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=959840" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20clonal%20evolution%20of%20tumor%20cell%20populations&amp;journal=Science&amp;doi=10.1126%2Fscience.959840&amp;volume=194&amp;pages=23-28&amp;publication_year=1976&amp;author=Nowell%2CPC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Dentro, S. C. et al. Characterizing genetic intra-tumor heterogeneity across 2,658 human cancer genomes. <i>Cell</i> <b>184</b>, 2239–2254 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2021.03.009" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2021.03.009" data-track-item_id="10.1016/j.cell.2021.03.009">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXoslGkur8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33831375" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8054914" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Characterizing%20genetic%20intra-tumor%20heterogeneity%20across%202%2C658%20human%20cancer%20genomes&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2021.03.009&amp;volume=184&amp;pages=2239-2254&amp;publication_year=2021&amp;author=Dentro%2CSC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">Gerstung, M. et al. The evolutionary history of 2,658 cancers. <i>Nature</i> <b>578</b>, 122–128 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-019-1907-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-019-1907-7" data-track-item_id="10.1038/s41586-019-1907-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsLw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32025013" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054212" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20evolutionary%20history%20of%202%2C658%20cancers&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1907-7&amp;volume=578&amp;pages=122-128&amp;publication_year=2020&amp;author=Gerstung%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">Ferrando, A. A. &amp; López-Otín, C. Clonal evolution in leukemia. <i>Nat. Med.</i> <b>23</b>, 1135–1145 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnm.4410" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nm.4410" data-track-item_id="10.1038/nm.4410">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1Wns7nL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28985206" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20in%20leukemia&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fnm.4410&amp;volume=23&amp;pages=1135-1145&amp;publication_year=2017&amp;author=Ferrando%2CAA&amp;author=L%C3%B3pez-Ot%C3%ADn%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Ding, W. Richter transformation in the era of novel agents. <i>Hematology</i> <b>2018</b>, 256–263 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fasheducation-2018.1.256" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/asheducation-2018.1.256" data-track-item_id="10.1182/asheducation-2018.1.256">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30504319" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6245983" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Richter%20transformation%20in%20the%20era%20of%20novel%20agents&amp;journal=Hematology&amp;doi=10.1182%2Fasheducation-2018.1.256&amp;volume=2018&amp;pages=256-263&amp;publication_year=2018&amp;author=Ding%2CW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Maddocks, K. J. et al. Etiology of ibrutinib therapy discontinuation and outcomes in patients with chronic lymphocytic leukemia. <i>JAMA Oncol.</i> <b>1</b>, 80 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1001%2Fjamaoncol.2014.218" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1001/jamaoncol.2014.218" data-track-item_id="10.1001/jamaoncol.2014.218">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26182309" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4520535" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Etiology%20of%20ibrutinib%20therapy%20discontinuation%20and%20outcomes%20in%20patients%20with%20chronic%20lymphocytic%20leukemia&amp;journal=JAMA%20Oncol.&amp;doi=10.1001%2Fjamaoncol.2014.218&amp;volume=1&amp;publication_year=2015&amp;author=Maddocks%2CKJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Ahn, I. E. et al. Clonal evolution leading to ibrutinib resistance in chronic lymphocytic leukemia. <i>Blood</i> <b>129</b>, 1469–1479 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood-2016-06-719294" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood-2016-06-719294" data-track-item_id="10.1182/blood-2016-06-719294">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVejs7%2FP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28049639" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5356450" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20leading%20to%20ibrutinib%20resistance%20in%20chronic%20lymphocytic%20leukemia&amp;journal=Blood&amp;doi=10.1182%2Fblood-2016-06-719294&amp;volume=129&amp;pages=1469-1479&amp;publication_year=2017&amp;author=Ahn%2CIE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Jain, P. et al. Outcomes of patients with chronic lymphocytic leukemia after discontinuing ibrutinib. <i>Blood</i> <b>125</b>, 2062–2067 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood-2014-09-603670" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood-2014-09-603670" data-track-item_id="10.1182/blood-2014-09-603670">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXmtV2mtb8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25573991" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4467871" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Outcomes%20of%20patients%20with%20chronic%20lymphocytic%20leukemia%20after%20discontinuing%20ibrutinib&amp;journal=Blood&amp;doi=10.1182%2Fblood-2014-09-603670&amp;volume=125&amp;pages=2062-2067&amp;publication_year=2015&amp;author=Jain%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Landau, D. A. et al. The evolutionary landscape of chronic lymphocytic leukemia treated with ibrutinib targeted therapy. <i>Nat. Commun.</i> <b>8</b>, 2185 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-017-02329-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-017-02329-y" data-track-item_id="10.1038/s41467-017-02329-y">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29259203" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5736707" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtV2ntrjJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20evolutionary%20landscape%20of%20chronic%20lymphocytic%20leukemia%20treated%20with%20ibrutinib%20targeted%20therapy&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-02329-y&amp;volume=8&amp;publication_year=2017&amp;author=Landau%2CDA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Beà, S. et al. Genetic imbalances in progressed B-cell chronic lymphocytic leukemia and transformed large-cell lymphoma (Richter’s syndrome). <i>Am. J. Pathol.</i> <b>161</b>, 957–968 (2002).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0002-9440%2810%2964256-3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0002-9440(10)64256-3" data-track-item_id="10.1016/S0002-9440(10)64256-3">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12213724" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1867253" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20imbalances%20in%20progressed%20B-cell%20chronic%20lymphocytic%20leukemia%20and%20transformed%20large-cell%20lymphoma%20%28Richter%E2%80%99s%20syndrome%29&amp;journal=Am.%20J.%20Pathol.&amp;doi=10.1016%2FS0002-9440%2810%2964256-3&amp;volume=161&amp;pages=957-968&amp;publication_year=2002&amp;author=Be%C3%A0%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Scandurra, M. et al. Genomic profiling of Richter’s syndrome: recurrent lesions and differences with de novo diffuse large B-cell lymphomas. <i>Hematol. Oncol.</i> <b>28</b>, 62–67 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1002%2Fhon.932" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1002/hon.932" data-track-item_id="10.1002/hon.932">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXnt12jtrw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20014148" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20profiling%20of%20Richter%E2%80%99s%20syndrome%3A%20recurrent%20lesions%20and%20differences%20with%20de%20novo%20diffuse%20large%20B-cell%20lymphomas&amp;journal=Hematol.%20Oncol.&amp;doi=10.1002%2Fhon.932&amp;volume=28&amp;pages=62-67&amp;publication_year=2010&amp;author=Scandurra%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Rossi, D. et al. The genetics of Richter syndrome reveals disease heterogeneity and predicts survival after transformation. <i>Blood</i> <b>117</b>, 3391–3401 (2011).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood-2010-09-302174" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood-2010-09-302174" data-track-item_id="10.1182/blood-2010-09-302174">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkslamsLo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21266718" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20genetics%20of%20Richter%20syndrome%20reveals%20disease%20heterogeneity%20and%20predicts%20survival%20after%20transformation&amp;journal=Blood&amp;doi=10.1182%2Fblood-2010-09-302174&amp;volume=117&amp;pages=3391-3401&amp;publication_year=2011&amp;author=Rossi%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Fabbri, G. et al. Genetic lesions associated with chronic lymphocytic leukemia transformation to Richter syndrome. <i>J. Exp. Med.</i> <b>210</b>, 2273–2288 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1084%2Fjem.20131448" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1084/jem.20131448" data-track-item_id="10.1084/jem.20131448">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1ygsb%2FN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24127483" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3804949" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genetic%20lesions%20associated%20with%20chronic%20lymphocytic%20leukemia%20transformation%20to%20Richter%20syndrome&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20131448&amp;volume=210&amp;pages=2273-2288&amp;publication_year=2013&amp;author=Fabbri%2CG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">Chigrinova, E. et al. Two main genetic pathways lead to the transformation of chronic lymphocytic leukemia to Richter syndrome. <i>Blood</i> <b>122</b>, 2673–2682 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood-2013-03-489518" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood-2013-03-489518" data-track-item_id="10.1182/blood-2013-03-489518">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhs1Cgs73P" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24004666" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Two%20main%20genetic%20pathways%20lead%20to%20the%20transformation%20of%20chronic%20lymphocytic%20leukemia%20to%20Richter%20syndrome&amp;journal=Blood&amp;doi=10.1182%2Fblood-2013-03-489518&amp;volume=122&amp;pages=2673-2682&amp;publication_year=2013&amp;author=Chigrinova%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">Klintman, J. et al. Genomic and transcriptomic correlates of Richter transformation in chronic lymphocytic leukemia. <i>Blood</i> <b>137</b>, 2800–2816 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood.2020005650" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood.2020005650" data-track-item_id="10.1182/blood.2020005650">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFaiu7rP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33206936" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8163497" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20and%20transcriptomic%20correlates%20of%20Richter%20transformation%20in%20chronic%20lymphocytic%20leukemia&amp;journal=Blood&amp;doi=10.1182%2Fblood.2020005650&amp;volume=137&amp;pages=2800-2816&amp;publication_year=2021&amp;author=Klintman%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Chakraborty, S. et al. B-cell receptor signaling and genetic lesions in TP53 and CDKN2A/CDKN2B cooperate in Richter transformation. <i>Blood</i> <b>138</b>, 1053–1066 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood.2020008276" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood.2020008276" data-track-item_id="10.1182/blood.2020008276">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFWgs7zP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33900379" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=B-cell%20receptor%20signaling%20and%20genetic%20lesions%20in%20TP53%20and%20CDKN2A%2FCDKN2B%20cooperate%20in%20Richter%20transformation&amp;journal=Blood&amp;doi=10.1182%2Fblood.2020008276&amp;volume=138&amp;pages=1053-1066&amp;publication_year=2021&amp;author=Chakraborty%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">Anderson, M. A. et al. Clinicopathological features and outcomes of progression of CLL on the BCL2 inhibitor venetoclax. <i>Blood</i> <b>129</b>, 3362–3370 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood-2017-01-763003" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood-2017-01-763003" data-track-item_id="10.1182/blood-2017-01-763003">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVOmsrzO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28473407" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clinicopathological%20features%20and%20outcomes%20of%20progression%20of%20CLL%20on%20the%20BCL2%20inhibitor%20venetoclax&amp;journal=Blood&amp;doi=10.1182%2Fblood-2017-01-763003&amp;volume=129&amp;pages=3362-3370&amp;publication_year=2017&amp;author=Anderson%2CMA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">Miller, C. R. et al. Near-tetraploidy is associated with Richter transformation in chronic lymphocytic leukemia patients receiving ibrutinib. <i>Blood Adv.</i> <b>1</b>, 1584–1588 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fbloodadvances.2017007302" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/bloodadvances.2017007302" data-track-item_id="10.1182/bloodadvances.2017007302">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXls1Ohtb4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29296799" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728464" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Near-tetraploidy%20is%20associated%20with%20Richter%20transformation%20in%20chronic%20lymphocytic%20leukemia%20patients%20receiving%20ibrutinib&amp;journal=Blood%20Adv.&amp;doi=10.1182%2Fbloodadvances.2017007302&amp;volume=1&amp;pages=1584-1588&amp;publication_year=2017&amp;author=Miller%2CCR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Kadri, S. et al. Clonal evolution underlying leukemia progression and Richter transformation in patients with ibrutinib-relapsed CLL. <i>Blood Adv.</i> <b>1</b>, 715–727 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fbloodadvances.2016003632" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/bloodadvances.2016003632" data-track-item_id="10.1182/bloodadvances.2016003632">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXntFCjt7o%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29296715" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5728051" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20underlying%20leukemia%20progression%20and%20Richter%20transformation%20in%20patients%20with%20ibrutinib-relapsed%20CLL&amp;journal=Blood%20Adv.&amp;doi=10.1182%2Fbloodadvances.2016003632&amp;volume=1&amp;pages=715-727&amp;publication_year=2017&amp;author=Kadri%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Herling, C. D. et al. Clonal dynamics towards the development of venetoclax resistance in chronic lymphocytic leukemia. <i>Nat. Commun.</i> <b>9</b>, 727 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-018-03170-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-018-03170-7" data-track-item_id="10.1038/s41467-018-03170-7">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29463802" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5820258" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFWqt7vI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20dynamics%20towards%20the%20development%20of%20venetoclax%20resistance%20in%20chronic%20lymphocytic%20leukemia&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-03170-7&amp;volume=9&amp;publication_year=2018&amp;author=Herling%2CCD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Villamor, N. et al. NOTCH1 mutations identify a genetic subgroup of chronic lymphocytic leukemia patients with high risk of transformation and poor outcome. <i>Leukemia</i> <b>27</b>, 1100–1106 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fleu.2012.357" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/leu.2012.357" data-track-item_id="10.1038/leu.2012.357">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXnsVOjurg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23295735" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=NOTCH1%20mutations%20identify%20a%20genetic%20subgroup%20of%20chronic%20lymphocytic%20leukemia%20patients%20with%20high%20risk%20of%20transformation%20and%20poor%20outcome&amp;journal=Leukemia&amp;doi=10.1038%2Fleu.2012.357&amp;volume=27&amp;pages=1100-1106&amp;publication_year=2013&amp;author=Villamor%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">De Paoli, L. et al. MGA, a suppressor of MYC, is recurrently inactivated in high risk chronic lymphocytic leukemia. <i>Leuk. Lymphoma</i> <b>54</b>, 1087–1090 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3109%2F10428194.2012.723706" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3109/10428194.2012.723706" data-track-item_id="10.3109/10428194.2012.723706">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23039309" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXmtF2isrY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MGA%2C%20a%20suppressor%20of%20MYC%2C%20is%20recurrently%20inactivated%20in%20high%20risk%20chronic%20lymphocytic%20leukemia&amp;journal=Leuk.%20Lymphoma&amp;doi=10.3109%2F10428194.2012.723706&amp;volume=54&amp;pages=1087-1090&amp;publication_year=2013&amp;author=Paoli%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Rossi, D. et al. Different impact of NOTCH1 and SF3B1 mutations on the risk of chronic lymphocytic leukemia transformation to Richter syndrome. <i>Br. J. Haematol.</i> <b>158</b>, 426–429 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2Fj.1365-2141.2012.09155.x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/j.1365-2141.2012.09155.x" data-track-item_id="10.1111/j.1365-2141.2012.09155.x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtFKls7nI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22571487" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Different%20impact%20of%20NOTCH1%20and%20SF3B1%20mutations%20on%20the%20risk%20of%20chronic%20lymphocytic%20leukemia%20transformation%20to%20Richter%20syndrome&amp;journal=Br.%20J.%20Haematol.&amp;doi=10.1111%2Fj.1365-2141.2012.09155.x&amp;volume=158&amp;pages=426-429&amp;publication_year=2012&amp;author=Rossi%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">Chitalia, A. et al. Descriptive analysis of genetic aberrations and cell of origin in Richter transformation. <i>Leuk. Lymphoma</i> <b>60</b>, 971–979 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1080%2F10428194.2018.1516878" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1080/10428194.2018.1516878" data-track-item_id="10.1080/10428194.2018.1516878">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1ejtLk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30632835" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Descriptive%20analysis%20of%20genetic%20aberrations%20and%20cell%20of%20origin%20in%20Richter%20transformation&amp;journal=Leuk.%20Lymphoma&amp;doi=10.1080%2F10428194.2018.1516878&amp;volume=60&amp;pages=971-979&amp;publication_year=2019&amp;author=Chitalia%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Benatti, S. et al. IRF4 L116R mutation promotes proliferation of chronic lymphocytic leukemia B cells inducing MYC. <i>Hematol. Oncol.</i> <b>39</b>, 707–711 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1002%2Fhon.2915" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1002/hon.2915" data-track-item_id="10.1002/hon.2915">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xmt12gug%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34431535" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=IRF4%20L116R%20mutation%20promotes%20proliferation%20of%20chronic%20lymphocytic%20leukemia%20B%20cells%20inducing%20MYC&amp;journal=Hematol.%20Oncol.&amp;doi=10.1002%2Fhon.2915&amp;volume=39&amp;pages=707-711&amp;publication_year=2021&amp;author=Benatti%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">Minici, C. et al. Distinct homotypic B-cell receptor interactions shape the outcome of chronic lymphocytic leukaemia. <i>Nat. Commun.</i> <b>8</b>, 15746 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fncomms15746" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ncomms15746" data-track-item_id="10.1038/ncomms15746">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28598442" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5472768" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Distinct%20homotypic%20B-cell%20receptor%20interactions%20shape%20the%20outcome%20of%20chronic%20lymphocytic%20leukaemia&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms15746&amp;volume=8&amp;publication_year=2017&amp;author=Minici%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Puente, X. S. et al. Non-coding recurrent mutations in chronic lymphocytic leukaemia. <i>Nature</i> <b>526</b>, 519–524 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnature14666" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nature14666" data-track-item_id="10.1038/nature14666">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1WksrvJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26200345" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Non-coding%20recurrent%20mutations%20in%20chronic%20lymphocytic%20leukaemia&amp;journal=Nature&amp;doi=10.1038%2Fnature14666&amp;volume=526&amp;pages=519-524&amp;publication_year=2015&amp;author=Puente%2CXS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">Kasar, S. et al. Whole-genome sequencing reveals activation-induced cytidine deaminase signatures during indolent chronic lymphocytic leukaemia evolution. <i>Nat. Commun.</i> <b>6</b>, 8866 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fncomms9866" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ncomms9866" data-track-item_id="10.1038/ncomms9866">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFyks7nI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26638776" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-genome%20sequencing%20reveals%20activation-induced%20cytidine%20deaminase%20signatures%20during%20indolent%20chronic%20lymphocytic%20leukaemia%20evolution&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms9866&amp;volume=6&amp;publication_year=2015&amp;author=Kasar%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">Maura, F. et al. A practical guide for mutational signature analysis in hematological malignancies. <i>Nat. Commun.</i> <b>10</b>, 2969 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-019-11037-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-019-11037-8" data-track-item_id="10.1038/s41467-019-11037-8">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31278357" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6611883" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlalsbzJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20practical%20guide%20for%20mutational%20signature%20analysis%20in%20hematological%20malignancies&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-11037-8&amp;volume=10&amp;publication_year=2019&amp;author=Maura%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Arthur, S. E. et al. Genome-wide discovery of somatic regulatory variants in diffuse large B-cell lymphoma. <i>Nat. Commun.</i> <b>9</b>, 4001 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-018-06354-3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-018-06354-3" data-track-item_id="10.1038/s41467-018-06354-3">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30275490" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6167379" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvVegtL3I" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20discovery%20of%20somatic%20regulatory%20variants%20in%20diffuse%20large%20B-cell%20lymphoma&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-06354-3&amp;volume=9&amp;publication_year=2018&amp;author=Arthur%2CSE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Alexandrov, L. B. et al. The repertoire of mutational signatures in human cancer. <i>Nature</i> <b>578</b>, 94–101 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-020-1943-3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-020-1943-3" data-track-item_id="10.1038/s41586-020-1943-3">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsL0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32025018" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7054213" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20repertoire%20of%20mutational%20signatures%20in%20human%20cancer&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-1943-3&amp;volume=578&amp;pages=94-101&amp;publication_year=2020&amp;author=Alexandrov%2CLB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">Kucab, J. E. et al. A compendium of mutational signatures of environmental agents. <i>Cell</i> <b>177</b>, 821–836 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2019.03.001" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2019.03.001" data-track-item_id="10.1016/j.cell.2019.03.001">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXnt1Cltb0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30982602" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6506336" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20compendium%20of%20mutational%20signatures%20of%20environmental%20agents&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.03.001&amp;volume=177&amp;pages=821-836&amp;publication_year=2019&amp;author=Kucab%2CJE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">Rustad, E. H. et al. Timing the initiation of multiple myeloma. <i>Nat. Commun.</i> <b>11</b>, 1917 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-15740-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-15740-9" data-track-item_id="10.1038/s41467-020-15740-9">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnslSktbo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32317634" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7174344" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Timing%20the%20initiation%20of%20multiple%20myeloma&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-15740-9&amp;volume=11&amp;publication_year=2020&amp;author=Rustad%2CEH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">de Kanter, J. K. et al. Antiviral treatment causes a unique mutational signature in cancers of transplantation recipients. <i>Cell Stem Cell</i> <b>28</b>, 1726–1739 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.stem.2021.07.012" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.stem.2021.07.012" data-track-item_id="10.1016/j.stem.2021.07.012">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34496298" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8516432" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvF2jtbfL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Antiviral%20treatment%20causes%20a%20unique%20mutational%20signature%20in%20cancers%20of%20transplantation%20recipients&amp;journal=Cell%20Stem%20Cell&amp;doi=10.1016%2Fj.stem.2021.07.012&amp;volume=28&amp;pages=1726-1739&amp;publication_year=2021&amp;author=Kanter%2CJK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Pich, O. et al. The mutational footprints of cancer therapies. <i>Nat. Genet.</i> <b>51</b>, 1732–1740 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-019-0525-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-019-0525-5" data-track-item_id="10.1038/s41588-019-0525-5">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOru7nJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31740835" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6887544" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20mutational%20footprints%20of%20cancer%20therapies&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0525-5&amp;volume=51&amp;pages=1732-1740&amp;publication_year=2019&amp;author=Pich%2CO" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Gaiti, F. et al. Epigenetic evolution and lineage histories of chronic lymphocytic leukaemia. <i>Nature</i> <b>569</b>, 576–580 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-019-1198-z" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-019-1198-z" data-track-item_id="10.1038/s41586-019-1198-z">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXpslGmurs%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31092926" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6533116" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Epigenetic%20evolution%20and%20lineage%20histories%20of%20chronic%20lymphocytic%20leukaemia&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1198-z&amp;volume=569&amp;pages=576-580&amp;publication_year=2019&amp;author=Gaiti%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Gemenetzi, K. et al. Higher-order immunoglobulin repertoire restrictions in CLL: the illustrative case of stereotyped subsets 2 and 169. <i>Blood</i> <b>137</b>, 1895–1904 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood.2020005216" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood.2020005216" data-track-item_id="10.1182/blood.2020005216">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXovVOjsbk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33036024" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Higher-order%20immunoglobulin%20repertoire%20restrictions%20in%20CLL%3A%20the%20illustrative%20case%20of%20stereotyped%20subsets%202%20and%20169&amp;journal=Blood&amp;doi=10.1182%2Fblood.2020005216&amp;volume=137&amp;pages=1895-1904&amp;publication_year=2021&amp;author=Gemenetzi%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">Bagnara, D. et al. Post-transformation IGHV-IGHD-IGHJ mutations in chronic lymphocytic leukemia B cells: implications for mutational mechanisms and impact on clinical course. <i>Front. Oncol.</i> <b>11</b>, 1769 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3389%2Ffonc.2021.640731" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3389/fonc.2021.640731" data-track-item_id="10.3389/fonc.2021.640731">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Post-transformation%20IGHV-IGHD-IGHJ%20mutations%20in%20chronic%20lymphocytic%20leukemia%20B%20cells%3A%20implications%20for%20mutational%20mechanisms%20and%20impact%20on%20clinical%20course&amp;journal=Front.%20Oncol.&amp;doi=10.3389%2Ffonc.2021.640731&amp;volume=11&amp;publication_year=2021&amp;author=Bagnara%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">Calissano, C. et al. In vivo intraclonal and interclonal kinetic heterogeneity in B-cell chronic lymphocytic leukemia. <i>Blood</i> <b>114</b>, 4832–4842 (2009).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood-2009-05-219634" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood-2009-05-219634" data-track-item_id="10.1182/blood-2009-05-219634">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhsFGqsbzK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19789386" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2925477" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vivo%20intraclonal%20and%20interclonal%20kinetic%20heterogeneity%20in%20B-cell%20chronic%20lymphocytic%20leukemia&amp;journal=Blood&amp;doi=10.1182%2Fblood-2009-05-219634&amp;volume=114&amp;pages=4832-4842&amp;publication_year=2009&amp;author=Calissano%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">Calissano, C. et al. Intraclonal complexity in chronic lymphocytic leukemia: fractions enriched in recently born/divided and older/quiescent cells. <i>Mol. Med.</i> <b>17</b>, 1374–1382 (2011).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.2119%2Fmolmed.2011.00360" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.2119/molmed.2011.00360" data-track-item_id="10.2119/molmed.2011.00360">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1ekurzN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21968788" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3321822" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Intraclonal%20complexity%20in%20chronic%20lymphocytic%20leukemia%3A%20fractions%20enriched%20in%20recently%20born%2Fdivided%20and%20older%2Fquiescent%20cells&amp;journal=Mol.%20Med.&amp;doi=10.2119%2Fmolmed.2011.00360&amp;volume=17&amp;pages=1374-1382&amp;publication_year=2011&amp;author=Calissano%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">Cui, B. et al. MicroRNA-155 influences B-cell receptor signaling and associates with aggressive disease in chronic lymphocytic leukemia. <i>Blood</i> <b>124</b>, 546–554 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood-2014-03-559690" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood-2014-03-559690" data-track-item_id="10.1182/blood-2014-03-559690">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Ggu7%2FL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24914134" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4110661" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MicroRNA-155%20influences%20B-cell%20receptor%20signaling%20and%20associates%20with%20aggressive%20disease%20in%20chronic%20lymphocytic%20leukemia&amp;journal=Blood&amp;doi=10.1182%2Fblood-2014-03-559690&amp;volume=124&amp;pages=546-554&amp;publication_year=2014&amp;author=Cui%2CB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">Penter, L. et al. Longitudinal single-cell dynamics of chromatin accessibility and mitochondrial mutations in chronic lymphocytic leukemia mirror disease history. <i>Cancer Discov.</i> <b>11</b>, 3048–3063 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1158%2F2159-8290.CD-21-0276" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1158/2159-8290.CD-21-0276" data-track-item_id="10.1158/2159-8290.CD-21-0276">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjtlyhtL0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34112698" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8660953" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Longitudinal%20single-cell%20dynamics%20of%20chromatin%20accessibility%20and%20mitochondrial%20mutations%20in%20chronic%20lymphocytic%20leukemia%20mirror%20disease%20history&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-21-0276&amp;volume=11&amp;pages=3048-3063&amp;publication_year=2021&amp;author=Penter%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">Beekman, R. et al. The reference epigenome and regulatory chromatin landscape of chronic lymphocytic leukemia. <i>Nat. Med.</i> <b>24</b>, 868–880 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41591-018-0028-4" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41591-018-0028-4" data-track-item_id="10.1038/s41591-018-0028-4">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVShu7rL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29785028" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6363101" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20reference%20epigenome%20and%20regulatory%20chromatin%20landscape%20of%20chronic%20lymphocytic%20leukemia&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-018-0028-4&amp;volume=24&amp;pages=868-880&amp;publication_year=2018&amp;author=Beekman%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">Chen, C.-L. et al. Arginine is an epigenetic regulator targeting TEAD4 to modulate OXPHOS in prostate cancer cells. <i>Nat. Commun.</i> <b>12</b>, 2398 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-021-22652-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-021-22652-9" data-track-item_id="10.1038/s41467-021-22652-9">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXpsl2ktrY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33893278" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8065123" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Arginine%20is%20an%20epigenetic%20regulator%20targeting%20TEAD4%20to%20modulate%20OXPHOS%20in%20prostate%20cancer%20cells&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-22652-9&amp;volume=12&amp;publication_year=2021&amp;author=Chen%2CC-L" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR46">Rajbhandari, P. et al. Cross-cohort analysis identifies a TEAD4–MYCN positive feedback loop as the core regulatory element of high-risk neuroblastoma. <i>Cancer Discov.</i> <b>8</b>, 582–599 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1158%2F2159-8290.CD-16-0861" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1158/2159-8290.CD-16-0861" data-track-item_id="10.1158/2159-8290.CD-16-0861">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXos1Sjt70%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29510988" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5967627" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Cross-cohort%20analysis%20identifies%20a%20TEAD4%E2%80%93MYCN%20positive%20feedback%20loop%20as%20the%20core%20regulatory%20element%20of%20high-risk%20neuroblastoma&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-16-0861&amp;volume=8&amp;pages=582-599&amp;publication_year=2018&amp;author=Rajbhandari%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR47">Maffei, R. et al. IRF4 modulates the response to BCR activation in chronic lymphocytic leukemia regulating IKAROS and SYK. <i>Leukemia</i> <b>35</b>, 1330–1343 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41375-021-01178-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41375-021-01178-5" data-track-item_id="10.1038/s41375-021-01178-5">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmvVKqt7o%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33623139" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=IRF4%20modulates%20the%20response%20to%20BCR%20activation%20in%20chronic%20lymphocytic%20leukemia%20regulating%20IKAROS%20and%20SYK&amp;journal=Leukemia&amp;doi=10.1038%2Fs41375-021-01178-5&amp;volume=35&amp;pages=1330-1343&amp;publication_year=2021&amp;author=Maffei%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR48">Patterson, D. G. et al. An IRF4–MYC–mTORC1 integrated pathway controls cell growth and the proliferative capacity of activated B cells during B cell differentiation in vivo. <i>J. Immunol.</i> <b>207</b>, 1798–1811 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.4049%2Fjimmunol.2100440" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.4049/jimmunol.2100440" data-track-item_id="10.4049/jimmunol.2100440">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitlGrsb7K" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34470852" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20IRF4%E2%80%93MYC%E2%80%93mTORC1%20integrated%20pathway%20controls%20cell%20growth%20and%20the%20proliferative%20capacity%20of%20activated%20B%20cells%20during%20B%20cell%20differentiation%20in%20vivo&amp;journal=J.%20Immunol.&amp;doi=10.4049%2Fjimmunol.2100440&amp;volume=207&amp;pages=1798-1811&amp;publication_year=2021&amp;author=Patterson%2CDG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR49">Duran-Ferrer, M. et al. The proliferative history shapes the DNA methylome of B-cell tumors and predicts clinical outcome. <i>Nat. Cancer</i> <b>1</b>, 1066–1081 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs43018-020-00131-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s43018-020-00131-2" data-track-item_id="10.1038/s43018-020-00131-2">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsFegtro%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34079956" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8168619" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20proliferative%20history%20shapes%20the%20DNA%20methylome%20of%20B-cell%20tumors%20and%20predicts%20clinical%20outcome&amp;journal=Nat.%20Cancer&amp;doi=10.1038%2Fs43018-020-00131-2&amp;volume=1&amp;pages=1066-1081&amp;publication_year=2020&amp;author=Duran-Ferrer%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR50">Hasan, M. K., Ghia, E. M., Rassenti, L. Z., Widhopf, G. F. &amp; Kipps, T. J. Wnt5a enhances proliferation of chronic lymphocytic leukemia and ERK1/2 phosphorylation via a ROR1/DOCK2-dependent mechanism. <i>Leukemia</i> <b>35</b>, 1621–1630 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41375-020-01055-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41375-020-01055-7" data-track-item_id="10.1038/s41375-020-01055-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXit1WmsbrO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33097837" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Wnt5a%20enhances%20proliferation%20of%20chronic%20lymphocytic%20leukemia%20and%20ERK1%2F2%20phosphorylation%20via%20a%20ROR1%2FDOCK2-dependent%20mechanism&amp;journal=Leukemia&amp;doi=10.1038%2Fs41375-020-01055-7&amp;volume=35&amp;pages=1621-1630&amp;publication_year=2021&amp;author=Hasan%2CMK&amp;author=Ghia%2CEM&amp;author=Rassenti%2CLZ&amp;author=Widhopf%2CGF&amp;author=Kipps%2CTJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR51">Ntoufa, S., Vilia, M. G., Stamatopoulos, K., Ghia, P. &amp; Muzio, M. Toll-like receptors signaling: a complex network for NF-κB activation in B-cell lymphoid malignancies. <i>Semin. Cancer Biol.</i> <b>39</b>, 15–25 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.semcancer.2016.07.001" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.semcancer.2016.07.001" data-track-item_id="10.1016/j.semcancer.2016.07.001">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xht1Slu7vP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27402288" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Toll-like%20receptors%20signaling%3A%20a%20complex%20network%20for%20NF-%CE%BAB%20activation%20in%20B-cell%20lymphoid%20malignancies&amp;journal=Semin.%20Cancer%20Biol.&amp;doi=10.1016%2Fj.semcancer.2016.07.001&amp;volume=39&amp;pages=15-25&amp;publication_year=2016&amp;author=Ntoufa%2CS&amp;author=Vilia%2CMG&amp;author=Stamatopoulos%2CK&amp;author=Ghia%2CP&amp;author=Muzio%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR52">Monti, S. Molecular profiling of diffuse large B-cell lymphoma identifies robust subtypes including one characterized by host inflammatory response. <i>Blood</i> <b>105</b>, 1851–1861 (2005).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood-2004-07-2947" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood-2004-07-2947" data-track-item_id="10.1182/blood-2004-07-2947">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXitVOnsrY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15550490" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Molecular%20profiling%20of%20diffuse%20large%20B-cell%20lymphoma%20identifies%20robust%20subtypes%20including%20one%20characterized%20by%20host%20inflammatory%20response&amp;journal=Blood&amp;doi=10.1182%2Fblood-2004-07-2947&amp;volume=105&amp;pages=1851-1861&amp;publication_year=2005&amp;author=Monti%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR53">Caro, P. et al. Metabolic signatures uncover distinct targets in molecular subsets of diffuse large B cell lymphoma. <i>Cancer Cell</i> <b>22</b>, 547–560 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ccr.2012.08.014" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ccr.2012.08.014" data-track-item_id="10.1016/j.ccr.2012.08.014">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFWitbbO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23079663" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3479446" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Metabolic%20signatures%20uncover%20distinct%20targets%20in%20molecular%20subsets%20of%20diffuse%20large%20B%20cell%20lymphoma&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccr.2012.08.014&amp;volume=22&amp;pages=547-560&amp;publication_year=2012&amp;author=Caro%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR54">Norberg, E. et al. Differential contribution of the mitochondrial translation pathway to the survival of diffuse large B-cell lymphoma subsets. <i>Cell Death Differ.</i> <b>24</b>, 251–262 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fcdd.2016.116" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/cdd.2016.116" data-track-item_id="10.1038/cdd.2016.116">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhslGgs7bP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27768122" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20contribution%20of%20the%20mitochondrial%20translation%20pathway%20to%20the%20survival%20of%20diffuse%20large%20B-cell%20lymphoma%20subsets&amp;journal=Cell%20Death%20Differ.&amp;doi=10.1038%2Fcdd.2016.116&amp;volume=24&amp;pages=251-262&amp;publication_year=2017&amp;author=Norberg%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR55">Molina, J. R. et al. An inhibitor of oxidative phosphorylation exploits cancer vulnerability. <i>Nat. Med.</i> <b>24</b>, 1036–1046 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41591-018-0052-4" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41591-018-0052-4" data-track-item_id="10.1038/s41591-018-0052-4">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFWru7bN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29892070" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20inhibitor%20of%20oxidative%20phosphorylation%20exploits%20cancer%20vulnerability&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-018-0052-4&amp;volume=24&amp;pages=1036-1046&amp;publication_year=2018&amp;author=Molina%2CJR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR56">Vangapandu, H. V. et al. Biological and metabolic effects of IACS-010759, an OxPhos inhibitor, on chronic lymphocytic leukemia cells. <i>Oncotarget</i> <b>9</b>, 24980–24991 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.18632%2Foncotarget.25166" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.18632/oncotarget.25166" data-track-item_id="10.18632/oncotarget.25166">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29861847" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5982765" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Biological%20and%20metabolic%20effects%20of%20IACS-010759%2C%20an%20OxPhos%20inhibitor%2C%20on%20chronic%20lymphocytic%20leukemia%20cells&amp;journal=Oncotarget&amp;doi=10.18632%2Foncotarget.25166&amp;volume=9&amp;pages=24980-24991&amp;publication_year=2018&amp;author=Vangapandu%2CHV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR57">Zhang, L. et al. Metabolic reprogramming toward oxidative phosphorylation identifies a therapeutic target for mantle cell lymphoma. <i>Sci. Transl. Med.</i> <b>11</b>, eaau1167 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscitranslmed.aau1167" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/scitranslmed.aau1167" data-track-item_id="10.1126/scitranslmed.aau1167">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1CgsrrL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31068440" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Metabolic%20reprogramming%20toward%20oxidative%20phosphorylation%20identifies%20a%20therapeutic%20target%20for%20mantle%20cell%20lymphoma&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.aau1167&amp;volume=11&amp;publication_year=2019&amp;author=Zhang%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR58">Varano, G. et al. The B-cell receptor controls fitness of MYC-driven lymphoma cells via GSK3β inhibition. <i>Nature</i> <b>546</b>, 302–306 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnature22353" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nature22353" data-track-item_id="10.1038/nature22353">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXovFCrsrw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28562582" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20B-cell%20receptor%20controls%20fitness%20of%20MYC-driven%20lymphoma%20cells%20via%20GSK3%CE%B2%20inhibition&amp;journal=Nature&amp;doi=10.1038%2Fnature22353&amp;volume=546&amp;pages=302-306&amp;publication_year=2017&amp;author=Varano%2CG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR59">Dadashian, E. L. et al. TLR signaling is activated in lymph node–resident CLL cells and is only partially inhibited by ibrutinib. <i>Cancer Res.</i> <b>79</b>, 360–371 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1158%2F0008-5472.CAN-18-0781" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1158/0008-5472.CAN-18-0781" data-track-item_id="10.1158/0008-5472.CAN-18-0781">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXos1aiu7c%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30498085" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=TLR%20signaling%20is%20activated%20in%20lymph%20node%E2%80%93resident%20CLL%20cells%20and%20is%20only%20partially%20inhibited%20by%20ibrutinib&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-18-0781&amp;volume=79&amp;pages=360-371&amp;publication_year=2019&amp;author=Dadashian%2CEL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR60">Chan, K.-L. et al. Plasmablastic Richter transformation as a resistance mechanism for chronic lymphocytic leukaemia treated with BCR signalling inhibitors. <i>Br. J. Haematol.</i> <b>177</b>, 324–328 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2Fbjh.14062" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/bjh.14062" data-track-item_id="10.1111/bjh.14062">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27196127" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Plasmablastic%20Richter%20transformation%20as%20a%20resistance%20mechanism%20for%20chronic%20lymphocytic%20leukaemia%20treated%20with%20BCR%20signalling%20inhibitors&amp;journal=Br.%20J.%20Haematol.&amp;doi=10.1111%2Fbjh.14062&amp;volume=177&amp;pages=324-328&amp;publication_year=2017&amp;author=Chan%2CK-L" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR61">Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. <i>Bioinformatics</i> <b>25</b>, 1754–1760 (2009).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp324" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btp324" data-track-item_id="10.1093/bioinformatics/btp324">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19451168" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp324&amp;volume=25&amp;pages=1754-1760&amp;publication_year=2009&amp;author=Li%2CH&amp;author=Durbin%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR62">Nadeu, F. et al. IgCaller for reconstructing immunoglobulin gene rearrangements and oncogenic translocations from whole-genome sequencing in lymphoid neoplasms. <i>Nat. Commun.</i> <b>11</b>, 3390 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-17095-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-17095-7" data-track-item_id="10.1038/s41467-020-17095-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlKgur3L" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32636395" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7341758" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=IgCaller%20for%20reconstructing%20immunoglobulin%20gene%20rearrangements%20and%20oncogenic%20translocations%20from%20whole-genome%20sequencing%20in%20lymphoid%20neoplasms&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-17095-7&amp;volume=11&amp;publication_year=2020&amp;author=Nadeu%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR63">Robinson, J. T. et al. Integrative genomics viewer. <i>Nat. Biotechnol.</i> <b>29</b>, 24–26 (2011).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnbt.1754" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nbt.1754" data-track-item_id="10.1038/nbt.1754">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXjsFWrtg%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21221095" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3346182" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Integrative%20genomics%20viewer&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.1754&amp;volume=29&amp;pages=24-26&amp;publication_year=2011&amp;author=Robinson%2CJT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR64">Jones, D. et al. cgpCaVEManWrapper: simple execution of CaVEMan in order to detect somatic single nucleotide variants in NGS data. <i>Curr. Protoc. Bioinforma.</i> <b>56</b>, 15.10.1–15.10.18 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1002%2Fcpbi.20" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1002/cpbi.20" data-track-item_id="10.1002/cpbi.20">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=cgpCaVEManWrapper%3A%20simple%20execution%20of%20CaVEMan%20in%20order%20to%20detect%20somatic%20single%20nucleotide%20variants%20in%20NGS%20data&amp;journal=Curr.%20Protoc.%20Bioinforma.&amp;doi=10.1002%2Fcpbi.20&amp;volume=56&amp;pages=15.10.1-15.10.18&amp;publication_year=2016&amp;author=Jones%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR65">McKenna, A. et al. The genome analysis toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data. <i>Genome Res.</i> <b>20</b>, 1297–1303 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1101%2Fgr.107524.110" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1101/gr.107524.110" data-track-item_id="10.1101/gr.107524.110">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhtFeru7jM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20644199" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2928508" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20genome%20analysis%20toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.107524.110&amp;volume=20&amp;pages=1297-1303&amp;publication_year=2010&amp;author=McKenna%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR66">Fan, Y. et al. MuSE: accounting for tumor heterogeneity using a sample-specific error model improves sensitivity and specificity in mutation calling from sequencing data. <i>Genome Biol.</i> <b>17</b>, 178 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-016-1029-6" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-016-1029-6" data-track-item_id="10.1186/s13059-016-1029-6">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27557938" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4995747" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MuSE%3A%20accounting%20for%20tumor%20heterogeneity%20using%20a%20sample-specific%20error%20model%20improves%20sensitivity%20and%20specificity%20in%20mutation%20calling%20from%20sequencing%20data&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-016-1029-6&amp;volume=17&amp;publication_year=2016&amp;author=Fan%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR67">Danecek, P. et al. Twelve years of SAMtools and BCFtools. <i>Gigascience</i> <a href="https://doi.org/10.1093/gigascience/giab008" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1093/gigascience/giab008">https://doi.org/10.1093/gigascience/giab008</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR68">Moncunill, V. et al. Comprehensive characterization of complex structural variations in cancer by directly comparing genome sequence reads. <i>Nat. Biotechnol.</i> <b>32</b>, 1106–1112 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnbt.3027" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nbt.3027" data-track-item_id="10.1038/nbt.3027">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvVSqtrzN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25344728" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20characterization%20of%20complex%20structural%20variations%20in%20cancer%20by%20directly%20comparing%20genome%20sequence%20reads&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3027&amp;volume=32&amp;pages=1106-1112&amp;publication_year=2014&amp;author=Moncunill%2CV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR69">Raine, K. M. et al. cgpPindel: identifying somatically acquired insertion and deletion events from paired end sequencing. <i>Curr. Protoc. Bioinforma.</i> <b>52</b>, 15.7.1–12 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1002%2F0471250953.bi1507s52" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1002/0471250953.bi1507s52" data-track-item_id="10.1002/0471250953.bi1507s52">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=cgpPindel%3A%20identifying%20somatically%20acquired%20insertion%20and%20deletion%20events%20from%20paired%20end%20sequencing&amp;journal=Curr.%20Protoc.%20Bioinforma.&amp;doi=10.1002%2F0471250953.bi1507s52&amp;volume=52&amp;pages=15.7.1-12&amp;publication_year=2015&amp;author=Raine%2CKM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR70">Wala, J. A. et al. SvABA: genome-wide detection of structural variants and indels by local assembly. <i>Genome Res.</i> <b>28</b>, 581–591 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1101%2Fgr.221028.117" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1101/gr.221028.117" data-track-item_id="10.1101/gr.221028.117">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXns1Cltbk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29535149" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5880247" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SvABA%3A%20genome-wide%20detection%20of%20structural%20variants%20and%20indels%20by%20local%20assembly&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.221028.117&amp;volume=28&amp;pages=581-591&amp;publication_year=2018&amp;author=Wala%2CJA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR71">Rimmer, A. et al. Integrating mapping-, assembly- and haplotype-based approaches for calling variants in clinical sequencing applications. <i>Nat. Genet.</i> <b>46</b>, 912–918 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fng.3036" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ng.3036" data-track-item_id="10.1038/ng.3036">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFGqsbjP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25017105" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4753679" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Integrating%20mapping-%2C%20assembly-%20and%20haplotype-based%20approaches%20for%20calling%20variants%20in%20clinical%20sequencing%20applications&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3036&amp;volume=46&amp;pages=912-918&amp;publication_year=2014&amp;author=Rimmer%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR72">Cingolani, P. et al. A program for annotating and predicting the effects of single nucleotide polymorphisms, SnpEff: SNPs in the genome of <i>Drosophila melanogaster</i> strain w1118; iso-2; iso-3. <i>Fly. (Austin).</i> <b>6</b>, 80–92 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.4161%2Ffly.19695" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.4161/fly.19695" data-track-item_id="10.4161/fly.19695">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1GmtL3E" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22728672" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3679285" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20program%20for%20annotating%20and%20predicting%20the%20effects%20of%20single%20nucleotide%20polymorphisms%2C%20SnpEff%3A%20SNPs%20in%20the%20genome%20of%20Drosophila%20melanogaster%20strain%20w1118%3B%20iso-2%3B%20iso-3&amp;journal=Fly.%20%28Austin%29.&amp;doi=10.4161%2Ffly.19695&amp;volume=6&amp;pages=80-92&amp;publication_year=2012&amp;author=Cingolani%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR73">Nadeu, F. et al. Clinical impact of the subclonal architecture and mutational complexity in chronic lymphocytic leukemia. <i>Leukemia</i> <b>32</b>, 645–653 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fleu.2017.291" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/leu.2017.291" data-track-item_id="10.1038/leu.2017.291">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1Cmt7vN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28924241" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20impact%20of%20the%20subclonal%20architecture%20and%20mutational%20complexity%20in%20chronic%20lymphocytic%20leukemia&amp;journal=Leukemia&amp;doi=10.1038%2Fleu.2017.291&amp;volume=32&amp;pages=645-653&amp;publication_year=2018&amp;author=Nadeu%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR74">Koboldt, D. C. et al. VarScan 2: somatic mutation and copy number alteration discovery in cancer by exome sequencing. <i>Genome Res.</i> <b>22</b>, 568–576 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1101%2Fgr.129684.111" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1101/gr.129684.111" data-track-item_id="10.1101/gr.129684.111">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XjsV2ltr4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22300766" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3290792" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=VarScan%202%3A%20somatic%20mutation%20and%20copy%20number%20alteration%20discovery%20in%20cancer%20by%20exome%20sequencing&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.129684.111&amp;volume=22&amp;pages=568-576&amp;publication_year=2012&amp;author=Koboldt%2CDC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR75">Lai, Z. et al. VarDict: a novel and versatile variant caller for next-generation sequencing in cancer research. <i>Nucleic Acids Res.</i> <b>44</b>, e108 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgkw227" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gkw227" data-track-item_id="10.1093/nar/gkw227">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27060149" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4914105" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsF2jtrnL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=VarDict%3A%20a%20novel%20and%20versatile%20variant%20caller%20for%20next-generation%20sequencing%20in%20cancer%20research&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw227&amp;volume=44&amp;publication_year=2016&amp;author=Lai%2CZ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR76">Wilm, A. et al. LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets. <i>Nucleic Acids Res.</i> <b>40</b>, 11189–11201 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgks918" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gks918" data-track-item_id="10.1093/nar/gks918">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVCitLjJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23066108" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3526318" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=LoFreq%3A%20a%20sequence-quality%20aware%2C%20ultra-sensitive%20variant%20caller%20for%20uncovering%20cell-population%20heterogeneity%20from%20high-throughput%20sequencing%20datasets&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgks918&amp;volume=40&amp;pages=11189-11201&amp;publication_year=2012&amp;author=Wilm%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR77">Muller, E. et al. OutLyzer: software for extracting low-allele-frequency tumor mutations from sequencing background noise in clinical practice. <i>Oncotarget</i> <b>7</b>, 79485–79493 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.18632%2Foncotarget.13103" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.18632/oncotarget.13103" data-track-item_id="10.18632/oncotarget.13103">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27825131" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5346729" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=OutLyzer%3A%20software%20for%20extracting%20low-allele-frequency%20tumor%20mutations%20from%20sequencing%20background%20noise%20in%20clinical%20practice&amp;journal=Oncotarget&amp;doi=10.18632%2Foncotarget.13103&amp;volume=7&amp;pages=79485-79493&amp;publication_year=2016&amp;author=Muller%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR78">Nik-Zainal, S. et al. The life history of 21 breast cancers. <i>Cell</i> <b>149</b>, 994–1007 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2012.04.023" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2012.04.023" data-track-item_id="10.1016/j.cell.2012.04.023">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XntlymtLY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22608083" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3428864" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20life%20history%20of%2021%20breast%20cancers&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2012.04.023&amp;volume=149&amp;pages=994-1007&amp;publication_year=2012&amp;author=Nik-Zainal%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR79">Raine, K. M. et al. ascatNgs: identifying somatically acquired copy-number alterations from whole-genome sequencing data. <i>Curr. Protoc. Bioinforma.</i> <b>56</b>, 15.9.1–15.9.17 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1002%2Fcpbi.17" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1002/cpbi.17" data-track-item_id="10.1002/cpbi.17">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ascatNgs%3A%20identifying%20somatically%20acquired%20copy-number%20alterations%20from%20whole-genome%20sequencing%20data&amp;journal=Curr.%20Protoc.%20Bioinforma.&amp;doi=10.1002%2Fcpbi.17&amp;volume=56&amp;pages=15.9.1-15.9.17&amp;publication_year=2016&amp;author=Raine%2CKM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR80">Talevich, E., Shain, A. H., Botton, T. &amp; Bastian, B. C. CNVkit: genome-wide copy number detection and visualization from targeted DNA sequencing. <i>PLoS Comput. Biol.</i> <b>12</b>, e1004873 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pcbi.1004873" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pcbi.1004873" data-track-item_id="10.1371/journal.pcbi.1004873">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27100738" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4839673" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFWgtb%2FJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CNVkit%3A%20genome-wide%20copy%20number%20detection%20and%20visualization%20from%20targeted%20DNA%20sequencing&amp;journal=PLoS%20Comput.%20Biol.&amp;doi=10.1371%2Fjournal.pcbi.1004873&amp;volume=12&amp;publication_year=2016&amp;author=Talevich%2CE&amp;author=Shain%2CAH&amp;author=Botton%2CT&amp;author=Bastian%2CBC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR81">Nik-Zainal, S. et al. Landscape of somatic mutations in 560 breast cancer whole-genome sequences. <i>Nature</i> <b>534</b>, 47–54 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnature17676" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nature17676" data-track-item_id="10.1038/nature17676">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XntVCksrc%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27135926" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4910866" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Landscape%20of%20somatic%20mutations%20in%20560%20breast%20cancer%20whole-genome%20sequences&amp;journal=Nature&amp;doi=10.1038%2Fnature17676&amp;volume=534&amp;pages=47-54&amp;publication_year=2016&amp;author=Nik-Zainal%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR82">Rausch, T. et al. DELLY: structural variant discovery by integrated paired-end and split-read analysis. <i>Bioinformatics</i> <b>28</b>, i333–i339 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbts378" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/bts378" data-track-item_id="10.1093/bioinformatics/bts378">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhtlantLjF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22962449" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3436805" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=DELLY%3A%20structural%20variant%20discovery%20by%20integrated%20paired-end%20and%20split-read%20analysis&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbts378&amp;volume=28&amp;pages=i333-i339&amp;publication_year=2012&amp;author=Rausch%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR83">Stephens, P. J. et al. Massive genomic rearrangement acquired in a single catastrophic event during cancer development. <i>Cell</i> <b>144</b>, 27–40 (2011).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2010.11.055" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2010.11.055" data-track-item_id="10.1016/j.cell.2010.11.055">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXksFGjug%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21215367" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3065307" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Massive%20genomic%20rearrangement%20acquired%20in%20a%20single%20catastrophic%20event%20during%20cancer%20development&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2010.11.055&amp;volume=144&amp;pages=27-40&amp;publication_year=2011&amp;author=Stephens%2CPJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR84">Korbel, J. O. &amp; Campbell, P. J. Criteria for inference of chromothripsis in cancer genomes. <i>Cell</i> <b>152</b>, 1226–1236 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2013.02.023" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2013.02.023" data-track-item_id="10.1016/j.cell.2013.02.023">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXktFWiurs%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23498933" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Criteria%20for%20inference%20of%20chromothripsis%20in%20cancer%20genomes&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.02.023&amp;volume=152&amp;pages=1226-1236&amp;publication_year=2013&amp;author=Korbel%2CJO&amp;author=Campbell%2CPJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR85">Baca, S. C. et al. Punctuated evolution of prostate cancer genomes. <i>Cell</i> <b>153</b>, 666–677 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2013.03.021" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2013.03.021" data-track-item_id="10.1016/j.cell.2013.03.021">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXmslelsLg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23622249" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3690918" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Punctuated%20evolution%20of%20prostate%20cancer%20genomes&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2013.03.021&amp;volume=153&amp;pages=666-677&amp;publication_year=2013&amp;author=Baca%2CSC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR86">Shen, M. M. Chromoplexy: a new category of complex rearrangements in the cancer genome. <i>Cancer Cell</i> <b>23</b>, 567–569 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ccr.2013.04.025" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ccr.2013.04.025" data-track-item_id="10.1016/j.ccr.2013.04.025">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXnslajtbs%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23680143" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3673705" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Chromoplexy%3A%20a%20new%20category%20of%20complex%20rearrangements%20in%20the%20cancer%20genome&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccr.2013.04.025&amp;volume=23&amp;pages=567-569&amp;publication_year=2013&amp;author=Shen%2CMM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR87">Li, Y. et al. Patterns of somatic structural variation in human cancer genomes. <i>Nature</i> <b>578</b>, 112–121 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-019-1913-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-019-1913-9" data-track-item_id="10.1038/s41586-019-1913-9">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlClsL8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32025012" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7025897" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Patterns%20of%20somatic%20structural%20variation%20in%20human%20cancer%20genomes&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1913-9&amp;volume=578&amp;pages=112-121&amp;publication_year=2020&amp;author=Li%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR88">Maura, F. et al. Genomic landscape and chronological reconstruction of driver events in multiple myeloma. <i>Nat. Commun.</i> <b>10</b>, 3835 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-019-11680-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-019-11680-1" data-track-item_id="10.1038/s41467-019-11680-1">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31444325" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6707220" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1KktrnP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20landscape%20and%20chronological%20reconstruction%20of%20driver%20events%20in%20multiple%20myeloma&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-11680-1&amp;volume=10&amp;publication_year=2019&amp;author=Maura%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR89">Dentro, S. C., Wedge, D. C. &amp; Van Loo, P. Principles of reconstructing the subclonal architecture of cancers. <i>Cold Spring Harb. Perspect. Med.</i> <b>7</b>, a026625 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1101%2Fcshperspect.a026625" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1101/cshperspect.a026625" data-track-item_id="10.1101/cshperspect.a026625">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28270531" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5538405" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisFKqurg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Principles%20of%20reconstructing%20the%20subclonal%20architecture%20of%20cancers&amp;journal=Cold%20Spring%20Harb.%20Perspect.%20Med.&amp;doi=10.1101%2Fcshperspect.a026625&amp;volume=7&amp;publication_year=2017&amp;author=Dentro%2CSC&amp;author=Wedge%2CDC&amp;author=Loo%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR90">Kim, J. et al. Somatic ERCC2 mutations are associated with a distinct genomic signature in urothelial tumors. <i>Nat. Genet.</i> <b>48</b>, 600–606 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fng.3557" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ng.3557" data-track-item_id="10.1038/ng.3557">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xms1Ontr0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27111033" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4936490" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Somatic%20ERCC2%20mutations%20are%20associated%20with%20a%20distinct%20genomic%20signature%20in%20urothelial%20tumors&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3557&amp;volume=48&amp;pages=600-606&amp;publication_year=2016&amp;author=Kim%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR91">Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. <i>Nature</i> <b>574</b>, 532–537 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-019-1672-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-019-1672-7" data-track-item_id="10.1038/s41586-019-1672-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVagsb3J" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31645730" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20landscape%20of%20somatic%20mutation%20in%20normal%20colorectal%20epithelial%20cells&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1672-7&amp;volume=574&amp;pages=532-537&amp;publication_year=2019&amp;author=Lee-Six%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR92">Yang, F. et al. Chemotherapy and mismatch repair deficiency cooperate to fuel TP53 mutagenesis and ALL relapse. <i>Nat. Cancer</i> <b>2</b>, 819–834 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs43018-021-00230-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s43018-021-00230-8" data-track-item_id="10.1038/s43018-021-00230-8">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsVOms7g%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35122027" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Chemotherapy%20and%20mismatch%20repair%20deficiency%20cooperate%20to%20fuel%20TP53%20mutagenesis%20and%20ALL%20relapse&amp;journal=Nat.%20Cancer&amp;doi=10.1038%2Fs43018-021-00230-8&amp;volume=2&amp;pages=819-834&amp;publication_year=2021&amp;author=Yang%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR93">Koren, A. et al. Differential relationship of DNA replication timing to different forms of human mutation and variation. <i>Am. J. Hum. Genet.</i> <b>91</b>, 1033–1040 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ajhg.2012.10.018" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ajhg.2012.10.018" data-track-item_id="10.1016/j.ajhg.2012.10.018">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsleisLrO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23176822" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3516607" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20relationship%20of%20DNA%20replication%20timing%20to%20different%20forms%20of%20human%20mutation%20and%20variation&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2012.10.018&amp;volume=91&amp;pages=1033-1040&amp;publication_year=2012&amp;author=Koren%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR94">Haradhvala, N. J. et al. Mutational strand asymmetries in cancer genomes reveal mechanisms of DNA damage and repair. <i>Cell</i> <b>164</b>, 538–549 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2015.12.050" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2015.12.050" data-track-item_id="10.1016/j.cell.2015.12.050">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVKitbg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26806129" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4753048" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Mutational%20strand%20asymmetries%20in%20cancer%20genomes%20reveal%20mechanisms%20of%20DNA%20damage%20and%20repair&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2015.12.050&amp;volume=164&amp;pages=538-549&amp;publication_year=2016&amp;author=Haradhvala%2CNJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR95">Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. <i>Bioinformatics</i> <b>30</b>, 2114–2120 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu170" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btu170" data-track-item_id="10.1093/bioinformatics/btu170">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Sqt7nP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24695404" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu170&amp;volume=30&amp;pages=2114-2120&amp;publication_year=2014&amp;author=Bolger%2CAM&amp;author=Lohse%2CM&amp;author=Usadel%2CB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR96">Aryee, M. J. et al. Minfi: a flexible and comprehensive Bioconductor package for the analysis of Infinium DNA methylation microarrays. <i>Bioinformatics</i> <b>30</b>, 1363–1369 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu049" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btu049" data-track-item_id="10.1093/bioinformatics/btu049">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXnslOit78%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24478339" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4016708" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Minfi%3A%20a%20flexible%20and%20comprehensive%20Bioconductor%20package%20for%20the%20analysis%20of%20Infinium%20DNA%20methylation%20microarrays&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu049&amp;volume=30&amp;pages=1363-1369&amp;publication_year=2014&amp;author=Aryee%2CMJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR97">Zhang, Y. et al. Model-based analysis of ChIP-Seq (MACS). <i>Genome Biol.</i> <b>9</b>, R137 (2008).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/gb-2008-9-9-r137" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/gb-2008-9-9-r137" data-track-item_id="10.1186/gb-2008-9-9-r137">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18798982" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2592715" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtl2ksrjE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Model-based%20analysis%20of%20ChIP-Seq%20%28MACS%29&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fgb-2008-9-9-r137&amp;volume=9&amp;publication_year=2008&amp;author=Zhang%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR98">Love, M. I., Huber, W. &amp; Anders, S. Moderated estimation of fold change and dispersion for RNA-seq data with DESeq2. <i>Genome Biol.</i> <b>15</b>, 550 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-014-0550-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-014-0550-8" data-track-item_id="10.1186/s13059-014-0550-8">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25516281" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4302049" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjtVCrsL8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Moderated%20estimation%20of%20fold%20change%20and%20dispersion%20for%20RNA-seq%20data%20with%20DESeq2&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-014-0550-8&amp;volume=15&amp;publication_year=2014&amp;author=Love%2CMI&amp;author=Huber%2CW&amp;author=Anders%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR99">Kopylova, E., Noé, L. &amp; Touzet, H. SortMeRNA: fast and accurate filtering of ribosomal RNAs in metatranscriptomic data. <i>Bioinformatics</i> <b>28</b>, 3211–3217 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbts611" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/bts611" data-track-item_id="10.1093/bioinformatics/bts611">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVSmsrvE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23071270" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SortMeRNA%3A%20fast%20and%20accurate%20filtering%20of%20ribosomal%20RNAs%20in%20metatranscriptomic%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbts611&amp;volume=28&amp;pages=3211-3217&amp;publication_year=2012&amp;author=Kopylova%2CE&amp;author=No%C3%A9%2CL&amp;author=Touzet%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR100">Bray, N. L., Pimentel, H., Melsted, P. &amp; Pachter, L. Near-optimal probabilistic RNA-seq quantification. <i>Nat. Biotechnol.</i> <b>34</b>, 525–527 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnbt.3519" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nbt.3519" data-track-item_id="10.1038/nbt.3519">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XlsVansL8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27043002" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Near-optimal%20probabilistic%20RNA-seq%20quantification&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3519&amp;volume=34&amp;pages=525-527&amp;publication_year=2016&amp;author=Bray%2CNL&amp;author=Pimentel%2CH&amp;author=Melsted%2CP&amp;author=Pachter%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR101">Kuipers, J., Jahn, K., Raphael, B. J. &amp; Beerenwinkel, N. Single-cell sequencing data reveal widespread recurrence and loss of mutational hits in the life histories of tumors. <i>Genome Res.</i> <b>27</b>, 1885–1894 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1101%2Fgr.220707.117" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1101/gr.220707.117" data-track-item_id="10.1101/gr.220707.117">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslyitL7E" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29030470" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5668945" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20sequencing%20data%20reveal%20widespread%20recurrence%20and%20loss%20of%20mutational%20hits%20in%20the%20life%20histories%20of%20tumors&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.220707.117&amp;volume=27&amp;pages=1885-1894&amp;publication_year=2017&amp;author=Kuipers%2CJ&amp;author=Jahn%2CK&amp;author=Raphael%2CBJ&amp;author=Beerenwinkel%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR102">Morita, K. et al. Clonal evolution of acute myeloid leukemia revealed by high-throughput single-cell genomics. <i>Nat. Commun.</i> <b>11</b>, 5327 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-19119-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-19119-8" data-track-item_id="10.1038/s41467-020-19119-8">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitF2js7jE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33087716" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7577981" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20evolution%20of%20acute%20myeloid%20leukemia%20revealed%20by%20high-throughput%20single-cell%20genomics&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19119-8&amp;volume=11&amp;publication_year=2020&amp;author=Morita%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR103">Picelli, S. et al. Smart-seq2 for sensitive full-length transcriptome profiling in single cells. <i>Nat. Methods</i> <b>10</b>, 1096–1098 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnmeth.2639" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nmeth.2639" data-track-item_id="10.1038/nmeth.2639">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsVyqtb%2FM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24056875" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Smart-seq2%20for%20sensitive%20full-length%20transcriptome%20profiling%20in%20single%20cells&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.2639&amp;volume=10&amp;pages=1096-1098&amp;publication_year=2013&amp;author=Picelli%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR104">Stoeckius, M. et al. Cell Hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics. <i>Genome Biol.</i> <b>19</b>, 224 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-018-1603-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-018-1603-1" data-track-item_id="10.1186/s13059-018-1603-1">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVKksbfJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30567574" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6300015" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Cell%20Hashing%20with%20barcoded%20antibodies%20enables%20multiplexing%20and%20doublet%20detection%20for%20single%20cell%20genomics&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-018-1603-1&amp;volume=19&amp;publication_year=2018&amp;author=Stoeckius%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR105">Parekh, S., Ziegenhain, C., Vieth, B., Enard, W. &amp; Hellmann, I. zUMIs - A fast and flexible pipeline to process RNA sequencing data with UMIs. <i>Gigascience</i> <a href="https://doi.org/10.1093/gigascience/giy059" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1093/gigascience/giy059">https://doi.org/10.1093/gigascience/giy059</a> (2018).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR106">Hao, Y. et al. Integrated analysis of multimodal single-cell data. <i>Cell</i> <b>184</b>, 3573–3587 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2021.04.048" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2021.04.048" data-track-item_id="10.1016/j.cell.2021.04.048">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlSrtrvE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34062119" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8238499" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Integrated%20analysis%20of%20multimodal%20single-cell%20data&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2021.04.048&amp;volume=184&amp;pages=3573-3587&amp;publication_year=2021&amp;author=Hao%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR107">Wolock, S. L., Lopez, R. &amp; Klein, A. M. Scrublet: computational identification of cell doublets in single-cell transcriptomic data. <i>Cell Syst.</i> <b>8</b>, 281–291 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cels.2018.11.005" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cels.2018.11.005" data-track-item_id="10.1016/j.cels.2018.11.005">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyhtbw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30954476" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6625319" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Scrublet%3A%20computational%20identification%20of%20cell%20doublets%20in%20single-cell%20transcriptomic%20data&amp;journal=Cell%20Syst.&amp;doi=10.1016%2Fj.cels.2018.11.005&amp;volume=8&amp;pages=281-291&amp;publication_year=2019&amp;author=Wolock%2CSL&amp;author=Lopez%2CR&amp;author=Klein%2CAM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR108">Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. <i>Nat. Methods</i> <b>16</b>, 1289–1296 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41592-019-0619-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41592-019-0619-0" data-track-item_id="10.1038/s41592-019-0619-0">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOqsr7N" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31740819" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6884693" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%2C%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0619-0&amp;volume=16&amp;pages=1289-1296&amp;publication_year=2019&amp;author=Korsunsky%2CI" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR109">Gnaiger, E., Steinlechner-Maran, R., Méndez, G., Eberl, T. &amp; Margreiter, R. Control of mitochondrial and cellular respiration by oxygen. <i>J. Bioenerg. Biomembr.</i> <b>27</b>, 583–596 (1995).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1007/BF02111656" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1007/BF02111656" data-track-item_id="10.1007/BF02111656">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28XhtF2gtw%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=8746845" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Control%20of%20mitochondrial%20and%20cellular%20respiration%20by%20oxygen&amp;journal=J.%20Bioenerg.%20Biomembr.&amp;doi=10.1007%2FBF02111656&amp;volume=27&amp;pages=583-596&amp;publication_year=1995&amp;author=Gnaiger%2CE&amp;author=Steinlechner-Maran%2CR&amp;author=M%C3%A9ndez%2CG&amp;author=Eberl%2CT&amp;author=Margreiter%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR110">Mongini, P. K. A. et al. TLR-9 and IL-15 synergy promotes the in vitro clonal expansion of chronic lymphocytic leukemia B cells. <i>J. Immunol.</i> <b>195</b>, 901–923 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.4049%2Fjimmunol.1403189" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.4049/jimmunol.1403189" data-track-item_id="10.4049/jimmunol.1403189">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhtFyjt7zJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26136429" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4505957" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=TLR-9%20and%20IL-15%20synergy%20promotes%20the%20in%20vitro%20clonal%20expansion%20of%20chronic%20lymphocytic%20leukemia%20B%20cells&amp;journal=J.%20Immunol.&amp;doi=10.4049%2Fjimmunol.1403189&amp;volume=195&amp;pages=901-923&amp;publication_year=2015&amp;author=Mongini%2CPKA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li></ol></div></div>
        </div>
    </div>
</aside>
</div>


    
        
    
    
        
            
        

        
            
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>




<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Medicine
                    </span>
                    (<i itemprop="alternateName">Nat Med</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-170X</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1078-8956</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/" data-track="click" data-track-action="protocols.io" data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/authors/editorial_policies" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nature-research/open-access" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/language-editing/" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/" data-track="click" data-track-action="research solutions" data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library" data-track="click" data-track-action="Recommend to library" data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica" data-track="click" data-track-action="nature africa" data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly" data-track="click" data-track-action="nature Italy" data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies" data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice" data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/accessibility-statement" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20">
    </a>
    <p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"></path></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"></path><clipPath id="b"><use xlink:href="#a" style="overflow:visible"></use></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"></path><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"></path><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"></path><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"></path></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"></path></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif c-site-messages--nature-briefing-cancer" data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Cancer">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Cancer">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing: Cancer" src="/static/images/logos/nature-briefing-logo-cancer-white-bc263e7d57.svg" width="373" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing: Cancer</em> newsletter — what matters in cancer research, free to your inbox weekly.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/cancer" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Cancer">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="CancerBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="CancerBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required="" data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:cancer" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required="">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Cancer">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get what matters in cancer research, free to your inbox weekly.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="/briefing/cancer/?brieferEntryPoint=CancerBriefingBanner">Sign up for Nature Briefing: Cancer
            </a>
        </div>

    </div>

</div>

    




<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41591-022-01927-8&amp;format=js&amp;last_modified=2022-08-11" async=""></script>

<hwt-cxlzpfmqqtqj></hwt-cxlzpfmqqtqj><div data-cc-ghost="" style="height: 352px;"></div><script type="text/javascript" id="fetch-contextual-ads-campaign-data" charset="">(function(){var g=function(a){a=new CustomEvent("campaignDataLoaded",{detail:a});document.dispatchEvent(a)},b=google_tag_manager["rm"]["50443292"](18);if(b){var k=1500;b=google_tag_manager["rm"]["50443292"](19);b="link"===b?google_tag_manager["rm"]["50443292"](20):google_tag_manager["rm"]["50443292"](21);var m=function(a,c){var h=!1,l=setTimeout(function(){h=!0;c(null)},k),d=new XMLHttpRequest;d.onload=function(){var e=null,f=[];if(200===d.status){try{e=JSON.parse(d.responseText)}catch(p){e={}}f=e.campaigns||[]}h||(clearTimeout(l),c(f.length?f.join(","):null))};d.open("GET",
"/platform/contextual?doi\x3d"+a);d.send()},n=function(a,c){a&&-1===a.indexOf("not set")?m(a,c):setTimeout(c,1)};n(b,function(a){window.campaignsForContextualAds=a;g(a);window.dataLayer.push({content:{article:{campaignID:a}}})})}else window.campaignsForContextualAds="",g(null)})();</script>
<script type="text/javascript" id="gtm-setup-accessdetailsloaded-handler" charset="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail||{};var b={event:"update-access-details"};a.institutional_business_partner_ids&&a.resolved_by||console.log("BPID data could not be retrieved from /exposed-details");a.logged_in_to_sn_profile||console.log("logged_in_to_sn_profile could not be retrieved from /exposed-details");var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?
a.resolved_by.join(";"):"",e=a.personal_business_partner_id_found,f=a.logged_in_to_sn_profile,g=a.snid;b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null,personal:e||null,logged_in_to_sn_profile:f,snid:g||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy={};window.dataLayer.push(b);window.idpUserDataLoaded=!0},!1);window.dataLayer.push({event:"accessdetailsloaded-handler-added"});
window.accessDetailsLoadedHandlerAdded=!0;</script><script type="text/javascript" id="gtm-mediatrust" charset="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="crossmark-script" text="" charset="" type="text/javascript" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="polyfill-matches" charset="">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="fetch-idp-user-data" charset="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},site=google_tag_manager["rm"]["50443292"](59),idpUrl;
switch(site){case "nature":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?"https://staging-idp.nature.com/exposed-details":"https://idp.nature.com/exposed-details";break;case "link":idpUrl=-1<window.location.hostname.indexOf("link-qa")?"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details";break;case "springer":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?
"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details"}if(void 0!==idpUrl){var transport=new XMLHttpRequest;transport.open("GET",idpUrl,!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script src="https://verify.nature.com/verify/nature.min.js"></script><script type="text/javascript" id="create-ad-slots" charset="">function createAndLoadAds(){function h(b){"nature"===a&&google_tag_manager["rm"]["50443292"](87)(b);"bmc"!==a&&"springeropen"!==a||google_tag_manager["rm"]["50443292"](99)(b);("link"===a&&"oscar"===google_tag_manager["rm"]["50443292"](100)||"springer"===a||"link"===a&&!0===google_tag_manager["rm"]["50443292"](101))&&google_tag_manager["rm"]["50443292"](126)(b);"link"===a&&"bunsen"===google_tag_manager["rm"]["50443292"](127)&&google_tag_manager["rm"]["50443292"](152)(b);"link"===a&&"Core"===google_tag_manager["rm"]["50443292"](153)&&google_tag_manager["rm"]["50443292"](165)(b)}window.googletag=window.googletag||{cmd:[]};var a=google_tag_manager["rm"]["50443292"](166),e=google_tag_manager["rm"]["50443292"](168);
if(-1===window.location.search.indexOf("hide_ads\x3dtrue"))if(window.adSlots&&0!==Object.keys(window.adSlots).length)console.log("Ads previously loaded. Will not update ad slots until next page load.");else{window.adSlots||(window.adSlots={});window.getAd=function(b,f){for(var c in window.adSlots)if(-1<c.indexOf(b)){if("object"===f)return adSlots[c];if("slot"===f)return adSlots[c].slot}};googletag.cmd.push(function(){googletag.pubads().setPrivacySettings({limitedAds:e});console.log("limitedAds is ",
e);googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["rm"]["50443292"](170));googletag.pubads().enableSingleRequest();googletag.pubads().disableInitialLoad()});for(var g=document.querySelectorAll("[data-gpt]"),d=0;g[d];++d)h(g[d]);googletag.cmd.push(function(){googletag.enableServices()})}}createAndLoadAds();</script>
</body></html>