<html lang="en" class="js"><head><link rel="preload" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" as="font" type="font/woff2" crossorigin=""><script async="" src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script type="text/javascript" async="" src="https://pagead2.googlesyndication.com/tag/js/gpt.js"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-MRVXSHQ"></script><script type="text/javascript" src="https://www.nature.com/ipmz8AlPGcIW7vFw4ucZaoBfIyESwyglsGVd_QLGO52eFdxD6S35Iu2HUZp-xJtPvnwduC-BK_7GVHHGzZ5j5g=="></script><script src="https://cmp.nature.com/production_live/en/consent-bundle-8-85.js" onload="initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')"></script>
    <title>Single-cell multi-omics analysis of the immune response in COVID-19 | Nature Medicine</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nm.rss">


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin="">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480">

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"haematopoiesis;infection;rna-sequencing;sars-cov-2;viral-infection","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Medicine","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41591-021-01329-2"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Emily Stephenson","Gary Reynolds","Rachel A. Botting","Fernando J. Calero-Nieto","Michael D. Morgan","Zewen Kelvin Tuong","Karsten Bach","Waradon Sungnak","Kaylee B. Worlock","Masahiro Yoshida","Natsuhiko Kumasaka","Katarzyna Kania","Justin Engelbert","Bayanne Olabi","Jarmila Stremenova Spegarova","Nicola K. Wilson","Nicole Mende","Laura Jardine","Louis C. S. Gardner","Issac Goh","Dave Horsfall","Jim McGrath","Simone Webb","Michael W. Mather","Rik G. H. Lindeboom","Emma Dann","Ni Huang","Krzysztof Polanski","Elena Prigmore","Florian Gothe","Jonathan Scott","Rebecca P. Payne","Kenneth F. Baker","Aidan T. Hanrath","Ina C. D. Schim van der Loeff","Andrew S. Barr","Amada Sanchez-Gonzalez","Laura Bergamaschi","Federica Mescia","Josephine L. Barnes","Eliz Kilich","Angus de Wilton","Anita Saigal","Aarash Saleh","Sam M. Janes","Claire M. Smith","Nusayhah Gopee","Caroline Wilson","Paul Coupland","Jonathan M. Coxhead","Vladimir Yu Kiselev","Stijn van Dongen","Jaume Bacardit","Hamish W. King","Stephen Baker","John R. Bradley","Gordon Dougan","Ian G. Goodfellow","Ravindra K. Gupta","Christoph Hess","Nathalie Kingston","Paul J. Lehner","Nicholas J. Matheson","Willem H. Owehand","Caroline Saunders","Kenneth G. C. Smith","Charlotte Summers","James E. D. Thaventhiran","Mark Toshner","Michael P. Weekes","Ashlea Bucke","Jo Calder","Laura Canna","Jason Domingo","Anne Elmer","Stewart Fuller","Julie Harris","Sarah Hewitt","Jane Kennet","Sherly Jose","Jenny Kourampa","Anne Meadows","Criona O’Brien","Jane Price","Cherry Publico","Rebecca Rastall","Carla Ribeiro","Jane Rowlands","Valentina Ruffolo","Hugo Tordesillas","Ben Bullman","Benjamin J. Dunmore","Stuart Fawke","Stefan Gräf","Josh Hodgson","Christopher Huang","Kelvin Hunter","Emma Jones","Ekaterina Legchenko","Cecilia Matara","Jennifer Martin","Ciara O’Donnell","Linda Pointon","Nicole Pond","Joy Shih","Rachel Sutcliffe","Tobias Tilly","Carmen Treacy","Zhen Tong","Jennifer Wood","Marta Wylot","Ariana Betancourt","Georgie Bower","Aloka De Sa","Madeline Epping","Oisin Huhn","Sarah Jackson","Isobel Jarvis","Jimmy Marsden","Francesca Nice","Georgina Okecha","Ommar Omarjee","Marianne Perera","Nathan Richoz","Rahul Sharma","Lori Turner","Eckart M. D. D. De Bie","Katherine Bunclark","Masa Josipovic","Michael Mackay","Alice Michael","Sabrina Rossi","Mayurun Selvan","Sarah Spencer","Cissy Yong","Ali Ansaripour","Lucy Mwaura","Caroline Patterson","Gary Polwarth","Petra Polgarova","Giovanni di Stefano","John Allison","Helen Butcher","Daniela Caputo","Debbie Clapham-Riley","Eleanor Dewhurst","Anita Furlong","Barbara Graves","Jennifer Gray","Tasmin Ivers","Mary Kasanicki","Emma Le Gresley","Rachel Linger","Sarah Meloy","Francesca Muldoon","Nigel Ovington","Sofia Papadia","Isabel Phelan","Hannah Stark","Kathleen E. Stirrups","Paul Townsend","Neil Walker","Jennifer Webster","Anthony J. Rostron","A. John Simpson","Sophie Hambleton","Elisa Laurenti","Paul A. Lyons","Kerstin B. Meyer","Marko Z. Nikolić","Christopher J. A. Duncan","Kenneth G. C. Smith","Sarah A. Teichmann","Menna R. Clatworthy","John C. Marioni","Berthold Göttgens","Muzlifah Haniffa"],"publishedAt":1618876800,"publishedAtString":"2021-04-20","title":"Single-cell multi-omics analysis of the immune response in COVID-19","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Haematopoiesis,Infection,RNA sequencing,SARS-CoV-2,Viral infection"},"journal":{"pcode":"nm","title":"nature medicine","volume":"27","issue":"5","id":41591,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"BR","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{font-weight:700;-webkit-font-smoothing:antialiased;font-size:1.5rem;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.app-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{margin:4px 4px 0;fill:#888;height:10px;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-c9121fecab.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png">
<link rel="icon" type="image/png" sizes="48x48" href="/static/images/favicons/nature/favicon-48x48-b52890008c.png">
<link rel="icon" type="image/png" sizes="32x32" href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="/static/images/favicons/nature/favicon-16x16-951651ab72.png">
<link rel="manifest" href="/static/manifest.json" crossorigin="use-credentials">
<link rel="mask-icon" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="/static/images/favicons/nature/favicon.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-e9d2061c95.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-1cbf3a8b56.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script><script id="js-position1" data-test="global-article-js" defer="" src="/static/js/global-article-es6-bundle-c8a573ca90.js"></script><script id="js-position2" data-test="shared-js" defer="" src="/static/js/shared-es6-bundle-e9d2061c95.js"></script><script id="js-position3" data-test="header-150-js" defer="" src="/static/js/header-150-es6-bundle-5bb959eaa1.js"></script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Single-cell multi-omics analysis of the immune response in COVID-19","description":"Analysis of human blood immune cells provides insights into the coordinated response to viral infections such as severe acute respiratory syndrome coronavirus 2, which causes coronavirus disease 2019 (COVID-19). We performed single-cell transcriptome, surface proteome and T and B lymphocyte antigen receptor analyses of over 780,000 peripheral blood mononuclear cells from a cross-sectional cohort of 130 patients with varying severities of COVID-19. We identified expansion of nonclassical monocytes expressing complement transcripts (CD16+C1QA/B/C+) that sequester platelets and were predicted to replenish the alveolar macrophage pool in COVID-19. Early, uncommitted CD34+ hematopoietic stem/progenitor cells were primed toward megakaryopoiesis, accompanied by expanded megakaryocyte-committed progenitors and increased platelet activation. Clonally expanded CD8+ T cells and an increased ratio of CD8+ effector T cells to effector memory T cells characterized severe disease, while circulating follicular helper T cells accompanied mild disease. We observed a relative loss of IgA2 in symptomatic disease despite an overall expansion of plasmablasts and plasma cells. Our study highlights the coordinated immune response that contributes to COVID-19 pathogenesis and reveals discrete cellular components that can be targeted for therapy. Transcriptomic and proteomic profiling of blood samples from individuals with COVID-19 reveals immune cell and hematopoietic progenitor cell alterations that are differentially associated with disease severity.","datePublished":"2021-04-20T00:00:00Z","dateModified":"2021-04-20T00:00:00Z","pageStart":"904","pageEnd":"916","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41591-021-01329-2","keywords":["Haematopoiesis","Infection","RNA sequencing","SARS-CoV-2","Viral infection","Biomedicine","general","Cancer Research","Metabolic Diseases","Infectious Diseases","Molecular Medicine","Neurosciences"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig5_HTML.png"],"isPartOf":{"name":"Nature Medicine","issn":["1546-170X","1078-8956"],"volumeNumber":"27","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Emily Stephenson","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Gary Reynolds","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Rachel A. Botting","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Fernando J. Calero-Nieto","url":"http://orcid.org/0000-0003-3358-8253","affiliation":[{"name":"University of Cambridge","address":{"name":"Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michael D. Morgan","url":"http://orcid.org/0000-0003-0757-0711","affiliation":[{"name":"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus","address":{"name":"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Zewen Kelvin Tuong","url":"http://orcid.org/0000-0002-6735-6808","affiliation":[{"name":"University of Cambridge","address":{"name":"Molecular Immunity Unit, Department of Medicine, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Karsten Bach","affiliation":[{"name":"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus","address":{"name":"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Waradon Sungnak","url":"http://orcid.org/0000-0002-0136-4960","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Kaylee B. Worlock","affiliation":[{"name":"University College London","address":{"name":"UCL Respiratory, Division of Medicine, University College London, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Masahiro Yoshida","url":"http://orcid.org/0000-0002-3521-5322","affiliation":[{"name":"University College London","address":{"name":"UCL Respiratory, Division of Medicine, University College London, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Natsuhiko Kumasaka","url":"http://orcid.org/0000-0002-3557-0375","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Katarzyna Kania","affiliation":[{"name":"University of Cambridge","address":{"name":"Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Justin Engelbert","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Bayanne Olabi","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jarmila Stremenova Spegarova","url":"http://orcid.org/0000-0002-0710-9266","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Nicola K. Wilson","url":"http://orcid.org/0000-0003-0865-7333","affiliation":[{"name":"University of Cambridge","address":{"name":"Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Nicole Mende","url":"http://orcid.org/0000-0002-5078-2333","affiliation":[{"name":"University of Cambridge","address":{"name":"Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Laura Jardine","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Louis C. S. Gardner","url":"http://orcid.org/0000-0002-7200-978X","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Issac Goh","url":"http://orcid.org/0000-0002-6397-3518","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Dave Horsfall","url":"http://orcid.org/0000-0002-8086-812X","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jim McGrath","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Simone Webb","url":"http://orcid.org/0000-0003-3058-8952","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michael W. Mather","url":"http://orcid.org/0000-0001-7972-7111","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Rik G. H. Lindeboom","url":"http://orcid.org/0000-0002-3660-504X","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Emma Dann","url":"http://orcid.org/0000-0002-7400-7438","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ni Huang","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Krzysztof Polanski","url":"http://orcid.org/0000-0002-2586-9576","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Elena Prigmore","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Florian Gothe","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University Hospital, Ludwig-Maximilians-Universität Munich","address":{"name":"Department of Pediatrics, Dr. von Hauner Children’s Hospital, University Hospital, Ludwig-Maximilians-Universität Munich, Munich, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jonathan Scott","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Rebecca P. Payne","url":"http://orcid.org/0000-0002-9037-7367","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Kenneth F. Baker","url":"http://orcid.org/0000-0002-6735-2911","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"NIHR Newcastle Biomedical Research Centre, Newcastle Hospitals NHS Foundation Trust","address":{"name":"NIHR Newcastle Biomedical Research Centre, Newcastle Hospitals NHS Foundation Trust, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Aidan T. Hanrath","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Newcastle upon Tyne Hospitals NHS Foundation","address":{"name":"Department of Infection and Tropical Medicine, Newcastle upon Tyne Hospitals NHS Foundation, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ina C. D. Schim van der Loeff","url":"http://orcid.org/0000-0003-1196-6196","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Andrew S. Barr","url":"http://orcid.org/0000-0002-8084-7644","affiliation":[{"name":"Newcastle upon Tyne Hospitals NHS Foundation","address":{"name":"Department of Infection and Tropical Medicine, Newcastle upon Tyne Hospitals NHS Foundation, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Amada Sanchez-Gonzalez","affiliation":[{"name":"Newcastle upon Tyne Hospitals NHS Foundation","address":{"name":"Department of Infection and Tropical Medicine, Newcastle upon Tyne Hospitals NHS Foundation, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Laura Bergamaschi","affiliation":[{"name":"Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus","address":{"name":"Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge, Cambridge Biomedical Campus","address":{"name":"Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Federica Mescia","url":"http://orcid.org/0000-0002-2759-4027","affiliation":[{"name":"Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus","address":{"name":"Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge, Cambridge Biomedical Campus","address":{"name":"Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Josephine L. Barnes","url":"http://orcid.org/0000-0001-9938-3176","affiliation":[{"name":"University College London","address":{"name":"UCL Respiratory, Division of Medicine, University College London, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Eliz Kilich","affiliation":[{"name":"University College London Hospitals NHS Foundation Trust","address":{"name":"University College London Hospitals NHS Foundation Trust, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Angus de Wilton","affiliation":[{"name":"University College London Hospitals NHS Foundation Trust","address":{"name":"University College London Hospitals NHS Foundation Trust, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Anita Saigal","affiliation":[{"name":"Royal Free Hospital NHS Foundation Trust","address":{"name":"Royal Free Hospital NHS Foundation Trust, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Aarash Saleh","affiliation":[{"name":"Royal Free Hospital NHS Foundation Trust","address":{"name":"Royal Free Hospital NHS Foundation Trust, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sam M. Janes","url":"http://orcid.org/0000-0002-6634-5939","affiliation":[{"name":"University College London","address":{"name":"UCL Respiratory, Division of Medicine, University College London, London, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University College London Hospitals NHS Foundation Trust","address":{"name":"University College London Hospitals NHS Foundation Trust, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Claire M. Smith","url":"http://orcid.org/0000-0002-8913-0009","affiliation":[{"name":"UCL Great Ormond Street Institute of Child Health","address":{"name":"UCL Great Ormond Street Institute of Child Health, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Nusayhah Gopee","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Newcastle Hospitals NHS Foundation Trust","address":{"name":"Department of Dermatology, Newcastle Hospitals NHS Foundation Trust, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Caroline Wilson","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"The Innovation Lab Integrated COVID Hub North East","address":{"name":"The Innovation Lab Integrated COVID Hub North East, Newcastle Upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Paul Coupland","url":"http://orcid.org/0000-0002-2871-3374","affiliation":[{"name":"University of Cambridge","address":{"name":"Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jonathan M. Coxhead","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Vladimir Yu Kiselev","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Stijn van Dongen","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jaume Bacardit","url":"http://orcid.org/0000-0002-2692-7205","affiliation":[{"name":"Newcastle University","address":{"name":"School of Computing, Newcastle University, Newcastle Upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Hamish W. King","url":"http://orcid.org/0000-0001-5972-8926","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Queen Mary University of London","address":{"name":"Centre for Immunobiology, Blizard Institute, Queen Mary University of London, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Anthony J. Rostron","url":"http://orcid.org/0000-0002-9336-1723","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Integrated Critical Care Unit, Sunderland Royal Hospital, South Tyneside and Sunderland NHS Foundation Trust","address":{"name":"Integrated Critical Care Unit, Sunderland Royal Hospital, South Tyneside and Sunderland NHS Foundation Trust, Sunderland, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"A. John Simpson","url":"http://orcid.org/0000-0003-4731-7294","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sophie Hambleton","url":"http://orcid.org/0000-0001-7954-3267","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Elisa Laurenti","url":"http://orcid.org/0000-0002-9917-9092","affiliation":[{"name":"University of Cambridge","address":{"name":"Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Paul A. Lyons","url":"http://orcid.org/0000-0001-7035-8997","affiliation":[{"name":"Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus","address":{"name":"Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge, Cambridge Biomedical Campus","address":{"name":"Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Kerstin B. Meyer","url":"http://orcid.org/0000-0001-5906-1498","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marko Z. Nikolić","url":"http://orcid.org/0000-0001-6304-6848","affiliation":[{"name":"University College London","address":{"name":"UCL Respiratory, Division of Medicine, University College London, London, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University College London Hospitals NHS Foundation Trust","address":{"name":"University College London Hospitals NHS Foundation Trust, London, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Christopher J. A. Duncan","affiliation":[{"name":"Newcastle University","address":{"name":"Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Newcastle upon Tyne Hospitals NHS Foundation","address":{"name":"Department of Infection and Tropical Medicine, Newcastle upon Tyne Hospitals NHS Foundation, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Kenneth G. C. Smith","url":"http://orcid.org/0000-0003-3829-4326","affiliation":[{"name":"Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus","address":{"name":"Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge, Cambridge Biomedical Campus","address":{"name":"Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sarah A. Teichmann","url":"http://orcid.org/0000-0002-6294-6366","affiliation":[{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"Theory of Condensed Matter Group, Cavendish Laboratory/Department of Physics, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"st9@sanger.ac.uk","@type":"Person"},{"name":"Menna R. Clatworthy","url":"http://orcid.org/0000-0002-3340-9828","affiliation":[{"name":"University of Cambridge","address":{"name":"Molecular Immunity Unit, Department of Medicine, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge, Cambridge Biomedical Campus","address":{"name":"Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Cambridge Institute for Therapeutic Immunology and Infectious Disease, Cambridge Biomedical Campus","address":{"name":"Cambridge Institute for Therapeutic Immunology and Infectious Disease, Cambridge Biomedical Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"NIHR Cambridge Biomedical Research Centre","address":{"name":"NIHR Cambridge Biomedical Research Centre, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"mrc38@cam.ac.uk","@type":"Person"},{"name":"John C. Marioni","url":"http://orcid.org/0000-0001-9092-0852","affiliation":[{"name":"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus","address":{"name":"European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Cambridge","address":{"name":"Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"marioni@ebi.ac.uk","@type":"Person"},{"name":"Berthold Göttgens","url":"http://orcid.org/0000-0001-6302-5705","affiliation":[{"name":"University of Cambridge","address":{"name":"Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"bg200@cam.ac.uk","@type":"Person"},{"name":"Muzlifah Haniffa","url":"http://orcid.org/0000-0002-3927-2084","affiliation":[{"name":"Newcastle University","address":{"name":"Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"NIHR Newcastle Biomedical Research Centre, Newcastle Hospitals NHS Foundation Trust","address":{"name":"NIHR Newcastle Biomedical Research Centre, Newcastle Hospitals NHS Foundation Trust, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Newcastle Hospitals NHS Foundation Trust","address":{"name":"Department of Dermatology, Newcastle Hospitals NHS Foundation Trust, Newcastle upon Tyne, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"m.a.haniffa@ncl.ac.uk","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41591-021-01329-2">
    
    
    <meta name="journal_id" content="41591">
    <meta name="dc.title" content="Single-cell multi-omics analysis of the immune response in COVID-19">
    <meta name="dc.source" content="Nature Medicine 2021 27:5">
    <meta name="dc.format" content="text/html">
    <meta name="dc.publisher" content="Nature Publishing Group">
    <meta name="dc.date" content="2021-04-20">
    <meta name="dc.type" content="OriginalPaper">
    <meta name="dc.language" content="En">
    <meta name="dc.copyright" content="2021 The Author(s)">
    <meta name="dc.rights" content="2021 The Author(s)">
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="dc.description" content="Analysis of human blood immune cells provides insights into the coordinated response to viral infections such as severe acute respiratory syndrome coronavirus 2, which causes coronavirus disease 2019 (COVID-19). We performed single-cell transcriptome, surface proteome and T and B lymphocyte antigen receptor analyses of over 780,000 peripheral blood mononuclear cells from a cross-sectional cohort of 130 patients with varying severities of COVID-19. We identified expansion of nonclassical monocytes expressing complement transcripts (CD16+C1QA/B/C+) that sequester platelets and were predicted to replenish the alveolar macrophage pool in COVID-19. Early, uncommitted CD34+ hematopoietic stem/progenitor cells were primed toward megakaryopoiesis, accompanied by expanded megakaryocyte-committed progenitors and increased platelet activation. Clonally expanded CD8+ T cells and an increased ratio of CD8+ effector T cells to effector memory T cells characterized severe disease, while circulating follicular helper T cells accompanied mild disease. We observed a relative loss of IgA2 in symptomatic disease despite an overall expansion of plasmablasts and plasma cells. Our study highlights the coordinated immune response that contributes to COVID-19 pathogenesis and reveals discrete cellular components that can be targeted for therapy. Transcriptomic and proteomic profiling of blood samples from individuals with COVID-19 reveals immune cell and hematopoietic progenitor cell alterations that are differentially associated with disease severity.">
    <meta name="prism.issn" content="1546-170X">
    <meta name="prism.publicationName" content="Nature Medicine">
    <meta name="prism.publicationDate" content="2021-04-20">
    <meta name="prism.volume" content="27">
    <meta name="prism.number" content="5">
    <meta name="prism.section" content="OriginalPaper">
    <meta name="prism.startingPage" content="904">
    <meta name="prism.endingPage" content="916">
    <meta name="prism.copyright" content="2021 The Author(s)">
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="prism.url" content="https://www.nature.com/articles/s41591-021-01329-2">
    <meta name="prism.doi" content="doi:10.1038/s41591-021-01329-2">
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41591-021-01329-2.pdf">
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41591-021-01329-2">
    <meta name="citation_journal_title" content="Nature Medicine">
    <meta name="citation_journal_abbrev" content="Nat Med">
    <meta name="citation_publisher" content="Nature Publishing Group">
    <meta name="citation_issn" content="1546-170X">
    <meta name="citation_title" content="Single-cell multi-omics analysis of the immune response in COVID-19">
    <meta name="citation_volume" content="27">
    <meta name="citation_issue" content="5">
    <meta name="citation_publication_date" content="2021/05">
    <meta name="citation_online_date" content="2021/04/20">
    <meta name="citation_firstpage" content="904">
    <meta name="citation_lastpage" content="916">
    <meta name="citation_article_type" content="Article">
    <meta name="citation_fulltext_world_readable" content="">
    <meta name="citation_language" content="en">
    <meta name="dc.identifier" content="doi:10.1038/s41591-021-01329-2">
    <meta name="DOI" content="10.1038/s41591-021-01329-2">
    <meta name="size" content="332137">
    <meta name="citation_doi" content="10.1038/s41591-021-01329-2">
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41591-021-01329-2&amp;api_key=">
    <meta name="description" content="Analysis of human blood immune cells provides insights into the coordinated response to viral infections such as severe acute respiratory syndrome coronavirus 2, which causes coronavirus disease 2019 (COVID-19). We performed single-cell transcriptome, surface proteome and T and B lymphocyte antigen receptor analyses of over 780,000 peripheral blood mononuclear cells from a cross-sectional cohort of 130 patients with varying severities of COVID-19. We identified expansion of nonclassical monocytes expressing complement transcripts (CD16+C1QA/B/C+) that sequester platelets and were predicted to replenish the alveolar macrophage pool in COVID-19. Early, uncommitted CD34+ hematopoietic stem/progenitor cells were primed toward megakaryopoiesis, accompanied by expanded megakaryocyte-committed progenitors and increased platelet activation. Clonally expanded CD8+ T cells and an increased ratio of CD8+ effector T cells to effector memory T cells characterized severe disease, while circulating follicular helper T cells accompanied mild disease. We observed a relative loss of IgA2 in symptomatic disease despite an overall expansion of plasmablasts and plasma cells. Our study highlights the coordinated immune response that contributes to COVID-19 pathogenesis and reveals discrete cellular components that can be targeted for therapy. Transcriptomic and proteomic profiling of blood samples from individuals with COVID-19 reveals immune cell and hematopoietic progenitor cell alterations that are differentially associated with disease severity.">
    <meta name="dc.creator" content="Stephenson, Emily">
    <meta name="dc.creator" content="Reynolds, Gary">
    <meta name="dc.creator" content="Botting, Rachel A.">
    <meta name="dc.creator" content="Calero-Nieto, Fernando J.">
    <meta name="dc.creator" content="Morgan, Michael D.">
    <meta name="dc.creator" content="Tuong, Zewen Kelvin">
    <meta name="dc.creator" content="Bach, Karsten">
    <meta name="dc.creator" content="Sungnak, Waradon">
    <meta name="dc.creator" content="Worlock, Kaylee B.">
    <meta name="dc.creator" content="Yoshida, Masahiro">
    <meta name="dc.creator" content="Kumasaka, Natsuhiko">
    <meta name="dc.creator" content="Kania, Katarzyna">
    <meta name="dc.creator" content="Engelbert, Justin">
    <meta name="dc.creator" content="Olabi, Bayanne">
    <meta name="dc.creator" content="Spegarova, Jarmila Stremenova">
    <meta name="dc.creator" content="Wilson, Nicola K.">
    <meta name="dc.creator" content="Mende, Nicole">
    <meta name="dc.creator" content="Jardine, Laura">
    <meta name="dc.creator" content="Gardner, Louis C. S.">
    <meta name="dc.creator" content="Goh, Issac">
    <meta name="dc.creator" content="Horsfall, Dave">
    <meta name="dc.creator" content="McGrath, Jim">
    <meta name="dc.creator" content="Webb, Simone">
    <meta name="dc.creator" content="Mather, Michael W.">
    <meta name="dc.creator" content="Lindeboom, Rik G. H.">
    <meta name="dc.creator" content="Dann, Emma">
    <meta name="dc.creator" content="Huang, Ni">
    <meta name="dc.creator" content="Polanski, Krzysztof">
    <meta name="dc.creator" content="Prigmore, Elena">
    <meta name="dc.creator" content="Gothe, Florian">
    <meta name="dc.creator" content="Scott, Jonathan">
    <meta name="dc.creator" content="Payne, Rebecca P.">
    <meta name="dc.creator" content="Baker, Kenneth F.">
    <meta name="dc.creator" content="Hanrath, Aidan T.">
    <meta name="dc.creator" content="Schim van der Loeff, Ina C. D.">
    <meta name="dc.creator" content="Barr, Andrew S.">
    <meta name="dc.creator" content="Sanchez-Gonzalez, Amada">
    <meta name="dc.creator" content="Bergamaschi, Laura">
    <meta name="dc.creator" content="Mescia, Federica">
    <meta name="dc.creator" content="Barnes, Josephine L.">
    <meta name="dc.creator" content="Kilich, Eliz">
    <meta name="dc.creator" content="de Wilton, Angus">
    <meta name="dc.creator" content="Saigal, Anita">
    <meta name="dc.creator" content="Saleh, Aarash">
    <meta name="dc.creator" content="Janes, Sam M.">
    <meta name="dc.creator" content="Smith, Claire M.">
    <meta name="dc.creator" content="Gopee, Nusayhah">
    <meta name="dc.creator" content="Wilson, Caroline">
    <meta name="dc.creator" content="Coupland, Paul">
    <meta name="dc.creator" content="Coxhead, Jonathan M.">
    <meta name="dc.creator" content="Kiselev, Vladimir Yu">
    <meta name="dc.creator" content="van Dongen, Stijn">
    <meta name="dc.creator" content="Bacardit, Jaume">
    <meta name="dc.creator" content="King, Hamish W.">
    <meta name="dc.creator" content="Rostron, Anthony J.">
    <meta name="dc.creator" content="Simpson, A. John">
    <meta name="dc.creator" content="Hambleton, Sophie">
    <meta name="dc.creator" content="Laurenti, Elisa">
    <meta name="dc.creator" content="Lyons, Paul A.">
    <meta name="dc.creator" content="Meyer, Kerstin B.">
    <meta name="dc.creator" content="Nikolić, Marko Z.">
    <meta name="dc.creator" content="Duncan, Christopher J. A.">
    <meta name="dc.creator" content="Smith, Kenneth G. C.">
    <meta name="dc.creator" content="Teichmann, Sarah A.">
    <meta name="dc.creator" content="Clatworthy, Menna R.">
    <meta name="dc.creator" content="Marioni, John C.">
    <meta name="dc.creator" content="Göttgens, Berthold">
    <meta name="dc.creator" content="Haniffa, Muzlifah">
    <meta name="dc.subject" content="Haematopoiesis">
    <meta name="dc.subject" content="Infection">
    <meta name="dc.subject" content="RNA sequencing">
    <meta name="dc.subject" content="SARS-CoV-2">
    <meta name="dc.subject" content="Viral infection">
    <meta name="citation_reference" content="citation_journal_title=Lancet; citation_title=COVID-19: towards controlling of a pandemic; citation_author=J Bedford; citation_volume=395; citation_publication_date=2020; citation_pages=1015-1018; citation_doi=10.1016/S0140-6736(20)30673-5; citation_id=CR1">
    <meta name="citation_reference" content="John Hopkins University. COVID-19 map. Johns Hopkins Coronavirus Resource Center. 
                  https://coronavirus.jhu.edu/map.html
                  
                 (2020).">
    <meta name="citation_reference" content="citation_journal_title=Lancet; citation_title=Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study; citation_author=N Chen; citation_volume=395; citation_publication_date=2020; citation_pages=507-513; citation_doi=10.1016/S0140-6736(20)30211-7; citation_id=CR3">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor; citation_author=M Hoffmann; citation_volume=181; citation_publication_date=2020; citation_pages=271-280; citation_id=CR4">
    <meta name="citation_reference" content="citation_journal_title=Front. Immunol.; citation_title=Reduction and functional exhaustion of T cells in patients with coronavirus disease 2019 (COVID-19); citation_author=B Diao; citation_volume=11; citation_publication_date=2020; citation_pages=827; citation_id=CR5">
    <meta name="citation_reference" content="Lee, J. S. et al. Immunophenotyping of COVID-19 and influenza highlights the role of type I interferons in development of severe COVID-19. Sci. Immunol. 5, eabd1554 (2020).">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Systems biological assessment of immunity to mild versus severe COVID-19 infection in humans; citation_author=PS Arunachalam; citation_volume=369; citation_publication_date=2020; citation_pages=1210-1220; citation_id=CR7">
    <meta name="citation_reference" content="citation_journal_title=Nat. Immunol.; citation_title=Extrafollicular B cell responses correlate with neutralizing antibodies and morbidity in COVID-19; citation_author=MC Woodruff; citation_volume=21; citation_publication_date=2020; citation_pages=1506-1516; citation_id=CR8">
    <meta name="citation_reference" content="citation_journal_title=J. Clin. Invest.; citation_title=Clinical and immunological features of severe and moderate coronavirus disease 2019; citation_author=G Chen; citation_volume=130; citation_publication_date=2020; citation_pages=2620-2629; citation_id=CR9">
    <meta name="citation_reference" content="citation_journal_title=Nat. Immunol.; citation_title=Single-cell landscape of immunological responses in patients with COVID-19; citation_author=J-Y Zhang; citation_volume=21; citation_publication_date=2020; citation_pages=1107-1118; citation_id=CR10">
    <meta name="citation_reference" content="Liu, C. et al. Time-resolved systems immunology reveals a late juncture linked to fatal COVID-19. Cell 
                  https://doi.org/10.1016/j.cell.2021.02.018
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Impaired type I interferon activity and inflammatory responses in severe COVID-19 patients; citation_author=J Hadjadj; citation_volume=369; citation_publication_date=2020; citation_pages=718-724; citation_id=CR12">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Severe COVID-19 is marked by a dysregulated myeloid cell compartment; citation_author=J Schulte-Schrepping; citation_volume=182; citation_publication_date=2020; citation_pages=1419-1440; citation_id=CR13">
    <meta name="citation_reference" content="Mann, E. R. et al. Longitudinal immune profiling reveals key myeloid signatures associated with COVID-19. Sci. Immunol. 5, eabd6197 (2020).">
    <meta name="citation_reference" content="citation_journal_title=Immunity; citation_title=Longitudinal multi-omics analyses identify responses of megakaryocytes, erythroid cells and plasmablasts as hallmarks of severe COVID-19; citation_author=JP Bernardes; citation_volume=53; citation_publication_date=2020; citation_pages=1296-1314; citation_id=CR15">
    <meta name="citation_reference" content="World Health Organization. Clinical management of COVID-19. 
                  https://www.who.int/publications/i/item/clinical-management-of-covid-19
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Fast, sensitive and accurate integration of single-cell data with Harmony; citation_author=I Korsunsky; citation_volume=16; citation_publication_date=2019; citation_pages=1289-1296; citation_id=CR17">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Genome-wide association study of severe COVID-19 with respiratory failure; citation_author=; citation_volume=383; citation_publication_date=2020; citation_pages=1522-1534; citation_id=CR18">
    <meta name="citation_reference" content="Pairo-Castineira, E. et al. Genetic mechanisms of critical illness in COVID-19. Nature 
                  https://doi.org/10.1038/s41586-020-03065-y
                  
                 (2020).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Testing for differential abundance in mass cytometry data; citation_author=ATL Lun, AC Richard, JC Marioni; citation_volume=14; citation_publication_date=2017; citation_pages=707-709; citation_id=CR20">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Lipopolysaccharide inhalation recruits monocytes and dendritic cell subsets to the alveolar airspace; citation_author=L Jardine; citation_volume=10; citation_publication_date=2019; citation_id=CR21">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=Single-cell landscape of bronchoalveolar immune cells in patients with COVID-19; citation_author=M Liao; citation_volume=26; citation_publication_date=2020; citation_pages=842-844; citation_id=CR22">
    <meta name="citation_reference" content="Evren, E. et al. Distinct developmental pathways from blood monocytes generate human lung macrophage diversity. Immunity 
                  https://doi.org/10.1016/j.immuni.2020.12.003
                  
                 (2020).">
    <meta name="citation_reference" content="Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. Cell 
                  https://doi.org/10.1016/j.cell.2021.01.053
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Lancet Haematol.; citation_title=Coagulation abnormalities and thrombosis in patients with COVID-19; citation_author=M Levi, J Thachil, T Iba, JH Levy; citation_volume=7; citation_publication_date=2020; citation_pages=e438-e440; citation_id=CR25">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Platelet activation and platelet-monocyte aggregate formation trigger tissue factor expression in patients with severe COVID-19; citation_author=ED Hottz; citation_volume=136; citation_publication_date=2020; citation_pages=1330-1341; citation_id=CR26">
    <meta name="citation_reference" content="citation_journal_title=Nat. Cell Biol.; citation_title=Human haematopoietic stem cell lineage commitment is a continuous process; citation_author=L Velten; citation_volume=19; citation_publication_date=2017; citation_pages=271-281; citation_id=CR27">
    <meta name="citation_reference" content="Mende, N. et al. Quantitative and molecular differences distinguish adult human medullary and extramedullary haematopoietic stem and progenitor cell landscapes. Preprint at BioRxiv 
                  https://doi.org/10.1101/2020.01.26.919753
                  
                 (2020).">
    <meta name="citation_reference" content="citation_journal_title=J. Virol.; citation_title=Interleukin-22 reduces lung inflammation during influenza A virus infection and protects against secondary bacterial infection; citation_author=S Ivanov; citation_volume=87; citation_publication_date=2013; citation_pages=6911-6924; citation_id=CR29">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Temporal and spatial heterogeneity of host response to SARS-CoV-2 pulmonary infection; citation_author=N Desai; citation_volume=11; citation_publication_date=2020; citation_id=CR30">
    <meta name="citation_reference" content="citation_journal_title=Immunity; citation_title=Inflammation directs memory precursor and short-lived effector CD8+ T cell fates via the graded expression of T-bet transcription factor; citation_author=NS Joshi; citation_volume=27; citation_publication_date=2007; citation_pages=281-295; citation_id=CR31">
    <meta name="citation_reference" content="citation_journal_title=Cell Rep.; citation_title=RNA-seq signatures normalized by mRNA abundance allow absolute deconvolution of human immune cell types; citation_author=G Monaco; citation_volume=26; citation_publication_date=2019; citation_pages=1627-1640; citation_id=CR32">
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Expansion of activated peripheral blood memory B cells in rheumatoid arthritis, impact of B cell depletion therapy, and biomarkers of response; citation_author=DG Adlowitz; citation_volume=10; citation_publication_date=2015; citation_pages=e0128269; citation_id=CR33">
    <meta name="citation_reference" content="citation_journal_title=Immunity; citation_title=T follicular helper cell biology: a decade of discovery and diseases; citation_author=ST Crotty; citation_volume=50; citation_publication_date=2019; citation_pages=1132-1148; citation_id=CR34">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Hypoxia-inducible factor-1α is a critical transcription factor for IL-10-producing B cells in autoimmune disease; citation_author=X Meng; citation_volume=9; citation_publication_date=2018; citation_id=CR35">
    <meta name="citation_reference" content="citation_journal_title=Immunology; citation_title=B cell inhibitory receptors and autoimmunity; citation_author=NR Pritchard, KGC Smith; citation_volume=108; citation_publication_date=2003; citation_pages=263-273; citation_id=CR36">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Antigen receptor-mediated changes in glucose metabolism in B lymphocytes: role of phosphatidylinositol 3-kinase signaling in the glycolytic control of growth; citation_author=CA Doughty; citation_volume=107; citation_publication_date=2006; citation_pages=4458-4465; citation_id=CR37">
    <meta name="citation_reference" content="citation_journal_title=Immunity; citation_title=Metabolic regulation of the immune humoral response; citation_author=M Boothby, RC Rickert; citation_volume=46; citation_publication_date=2017; citation_pages=743-755; citation_id=CR38">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Clinical characteristics of coronavirus disease 2019 in China; citation_author=W-J Guan; citation_volume=382; citation_publication_date=2020; citation_pages=1708-1720; citation_id=CR39">
    <meta name="citation_reference" content="citation_journal_title=J. Am. Med. Assoc.; citation_title=Case-fatality rate and characteristics of patients dying in relation to COVID-19 in Italy; citation_author=G Onder, G Rezza, S Brusaferro; citation_volume=323; citation_publication_date=2020; citation_pages=1775-1776; citation_id=CR40">
    <meta name="citation_reference" content="Parrot, T. et al. MAIT cell activation and dynamics associated with COVID-19 disease severity. Sci. Immunol. 5, eabe1670 (2020).">
    <meta name="citation_reference" content="Le Bert, N. et al. Highly functional virus-specific cellular immune response in asymptomatic SARS-CoV-2 infection. J. Exp. Med. 218, e20202617 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Antigen-specific adaptive immunity to SARS-CoV-2 in acute COVID-19 and associations with age and disease severity; citation_author=C Rydyznski Moderbacher; citation_volume=183; citation_publication_date=2020; citation_pages=996-1012; citation_id=CR43">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Loss of Bcl-6-expressing T follicular helper cells and germinal centers in COVID-19; citation_author=N Kaneko; citation_volume=183; citation_publication_date=2020; citation_pages=143-157; citation_id=CR44">
    <meta name="citation_reference" content="citation_journal_title=J. Exp. Med.; citation_title=CD4+ T cell subsets during virus infection. Protective capacity depends on effector cytokine secretion and on migratory capability; citation_author=KJ Maloy; citation_volume=191; citation_publication_date=2000; citation_pages=2159-2170; citation_id=CR45">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=COVID-19 vaccine BNT162b1 elicits human antibody and TH1 T cell responses; citation_author=U Sahin; citation_volume=586; citation_publication_date=2020; citation_pages=594-599; citation_id=CR46">
    <meta name="citation_reference" content="Bastard, P. et al. Autoantibodies against type I IFNs in patients with life-threatening COVID-19. Science 370, eabd4585 (2020).">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Sex differences in immune responses that underlie COVID-19 disease outcomes; citation_author=T Takahashi; citation_volume=588; citation_publication_date=2020; citation_pages=315-320; citation_id=CR48">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=STAR: ultrafast universal RNA-seq aligner; citation_author=A Dobin; citation_volume=29; citation_publication_date=2013; citation_pages=15-21; citation_id=CR49">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Souporcell: robust clustering of single-cell RNA-seq data by genotype without reference genotypes; citation_author=H Heaton; citation_volume=17; citation_publication_date=2020; citation_pages=615-620; citation_id=CR50">
    <meta name="citation_reference" content="citation_journal_title=Cell Syst.; citation_title=DoubletFinder: doublet detection in single-cell RNA-sequencing data using artificial nearest neighbors; citation_author=CS McGinnis, LM Murrow, ZJ Gartner; citation_volume=8; citation_publication_date=2019; citation_pages=329-337; citation_id=CR51">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=A test metric for assessing single-cell RNA-seq batch correction; citation_author=M Büttner, Z Miao, FA Wolf, SA Teichmann, FJ Theis; citation_volume=16; citation_publication_date=2019; citation_pages=43-49; citation_id=CR52">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Single-cell reconstruction of the early maternal-fetal interface in humans; citation_author=R Vento-Tormo; citation_volume=563; citation_publication_date=2018; citation_pages=347-353; citation_id=CR53">
    <meta name="citation_reference" content="citation_journal_title=Cell Syst.; citation_title=The Molecular Signatures Database (MSigDB) hallmark gene set collection; citation_author=A Liberzon; citation_volume=1; citation_publication_date=2015; citation_pages=417-425; citation_id=CR54">
    <meta name="citation_reference" content="Korotkevich, G., Sukhov, V. &amp; Sergushichev, A. Fast gene-set enrichment analysis. Preprint at BioRxiv 
                  https://doi.org/10.1101/060012
                  
                 (2019).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors; citation_author=L Haghverdi, ATL Lun, MD Morgan, JC Marioni; citation_volume=36; citation_publication_date=2018; citation_pages=421-427; citation_id=CR56">
    <meta name="citation_reference" content="citation_journal_title=J. Stat. Mech.; citation_title=Fast unfolding of communities in large networks; citation_author=VD Blondel, J-L Guillaume, R Lambiotte, E Lefebvre; citation_volume=2008; citation_publication_date=2008; citation_pages=P10008; citation_id=CR57">
    <meta name="citation_reference" content="McInnes, L., Healy, J. &amp; Melville, J. UMAP: uniform manifold approximation and projection for dimension reduction. Preprint at
                   https://arxiv.org/abs/1802.03426
                  
                 (2018).">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=edgeR: a Bioconductor package for differential expression analysis of digital gene expression data; citation_author=MD Robinson, DJ McCarthy, GK Smyth; citation_volume=26; citation_publication_date=2010; citation_pages=139-140; citation_id=CR59">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=Differential expression analysis of multifactor RNA-seq experiments with respect to biological variation; citation_author=DJ McCarthy, Y Chen, GK Smyth; citation_volume=40; citation_publication_date=2012; citation_pages=4288-4297; citation_id=CR60">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=Enrichr: a comprehensive gene-set enrichment analysis web server 2016 update; citation_author=MV Kuleshov; citation_volume=44; citation_publication_date=2016; citation_pages=W90-W97; citation_id=CR61">
    <meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Ser. B Stat. Methodol.; citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; citation_author=Y Benjamini, Y Hochberg; citation_volume=57; citation_publication_date=1995; citation_pages=289-300; citation_id=CR62">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=DCARS: differential correlation across ranked samples; citation_author=S Ghazanfar, D Strbenac, JT Ormerod, JYH Yang, E Patrick; citation_volume=35; citation_publication_date=2019; citation_pages=823-829; citation_id=CR63">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Change-O: a toolkit for analyzing large-scale B cell immunoglobulin repertoire sequencing data; citation_author=NT Gupta; citation_volume=31; citation_publication_date=2015; citation_pages=3356-3358; citation_id=CR64">
    <meta name="citation_reference" content="Lefranc, M.-P. IMGT, the International ImMunoGeneTics Information System. Novartis Found. Symp. 254, 126–142 
                  http://imgt.cines.fr
                  
                 (2008).">
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Automated analysis of high-throughput B cell sequencing data reveals a high frequency of novel immunoglobulin V gene segment alleles; citation_author=D Gadala-Maria, G Yaari, M Uduman, SH Kleinstein; citation_volume=112; citation_publication_date=2015; citation_pages=E862-E870; citation_id=CR66">
    <meta name="citation_reference" content="citation_journal_title=Genome Res; citation_title=Network properties derived from deep sequencing of human B cell receptor repertoires delineate B cell populations; citation_author=RJM Bashford-Rogers; citation_volume=23; citation_publication_date=2013; citation_pages=1874-1884; citation_id=CR67">
    <meta name="citation_reference" content="citation_journal_title=Exp. Hematol.; citation_title=The human cell atlas bone marrow single-cell interactive web portal; citation_author=SB Hay, K Ferchen, K Chetal, HL Grimes, N Salomonis; citation_volume=68; citation_publication_date=2018; citation_pages=51-61; citation_id=CR68">
    <meta name="citation_author" content="Stephenson, Emily">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Reynolds, Gary">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Botting, Rachel A.">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Calero-Nieto, Fernando J.">
    <meta name="citation_author_institution" content="Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK">
    <meta name="citation_author" content="Morgan, Michael D.">
    <meta name="citation_author_institution" content="European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK">
    <meta name="citation_author" content="Tuong, Zewen Kelvin">
    <meta name="citation_author_institution" content="Molecular Immunity Unit, Department of Medicine, University of Cambridge, Cambridge, UK">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Bach, Karsten">
    <meta name="citation_author_institution" content="European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK">
    <meta name="citation_author" content="Sungnak, Waradon">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Worlock, Kaylee B.">
    <meta name="citation_author_institution" content="UCL Respiratory, Division of Medicine, University College London, London, UK">
    <meta name="citation_author" content="Yoshida, Masahiro">
    <meta name="citation_author_institution" content="UCL Respiratory, Division of Medicine, University College London, London, UK">
    <meta name="citation_author" content="Kumasaka, Natsuhiko">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Kania, Katarzyna">
    <meta name="citation_author_institution" content="Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK">
    <meta name="citation_author" content="Engelbert, Justin">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Olabi, Bayanne">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Spegarova, Jarmila Stremenova">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Wilson, Nicola K.">
    <meta name="citation_author_institution" content="Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK">
    <meta name="citation_author" content="Mende, Nicole">
    <meta name="citation_author_institution" content="Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK">
    <meta name="citation_author" content="Jardine, Laura">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Gardner, Louis C. S.">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Goh, Issac">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Horsfall, Dave">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="McGrath, Jim">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Webb, Simone">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Mather, Michael W.">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Lindeboom, Rik G. H.">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Dann, Emma">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Huang, Ni">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Polanski, Krzysztof">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Prigmore, Elena">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Gothe, Florian">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author_institution" content="Department of Pediatrics, Dr. von Hauner Children’s Hospital, University Hospital, Ludwig-Maximilians-Universität Munich, Munich, Germany">
    <meta name="citation_author" content="Scott, Jonathan">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Payne, Rebecca P.">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Baker, Kenneth F.">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author_institution" content="NIHR Newcastle Biomedical Research Centre, Newcastle Hospitals NHS Foundation Trust, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Hanrath, Aidan T.">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author_institution" content="Department of Infection and Tropical Medicine, Newcastle upon Tyne Hospitals NHS Foundation, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Schim van der Loeff, Ina C. D.">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Barr, Andrew S.">
    <meta name="citation_author_institution" content="Department of Infection and Tropical Medicine, Newcastle upon Tyne Hospitals NHS Foundation, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Sanchez-Gonzalez, Amada">
    <meta name="citation_author_institution" content="Department of Infection and Tropical Medicine, Newcastle upon Tyne Hospitals NHS Foundation, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Bergamaschi, Laura">
    <meta name="citation_author_institution" content="Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK">
    <meta name="citation_author" content="Mescia, Federica">
    <meta name="citation_author_institution" content="Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK">
    <meta name="citation_author" content="Barnes, Josephine L.">
    <meta name="citation_author_institution" content="UCL Respiratory, Division of Medicine, University College London, London, UK">
    <meta name="citation_author" content="Kilich, Eliz">
    <meta name="citation_author_institution" content="University College London Hospitals NHS Foundation Trust, London, UK">
    <meta name="citation_author" content="de Wilton, Angus">
    <meta name="citation_author_institution" content="University College London Hospitals NHS Foundation Trust, London, UK">
    <meta name="citation_author" content="Saigal, Anita">
    <meta name="citation_author_institution" content="Royal Free Hospital NHS Foundation Trust, London, UK">
    <meta name="citation_author" content="Saleh, Aarash">
    <meta name="citation_author_institution" content="Royal Free Hospital NHS Foundation Trust, London, UK">
    <meta name="citation_author" content="Janes, Sam M.">
    <meta name="citation_author_institution" content="UCL Respiratory, Division of Medicine, University College London, London, UK">
    <meta name="citation_author_institution" content="University College London Hospitals NHS Foundation Trust, London, UK">
    <meta name="citation_author" content="Smith, Claire M.">
    <meta name="citation_author_institution" content="UCL Great Ormond Street Institute of Child Health, London, UK">
    <meta name="citation_author" content="Gopee, Nusayhah">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author_institution" content="Department of Dermatology, Newcastle Hospitals NHS Foundation Trust, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Wilson, Caroline">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author_institution" content="The Innovation Lab Integrated COVID Hub North East, Newcastle Upon Tyne, UK">
    <meta name="citation_author" content="Coupland, Paul">
    <meta name="citation_author_institution" content="Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK">
    <meta name="citation_author" content="Coxhead, Jonathan M.">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Kiselev, Vladimir Yu">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="van Dongen, Stijn">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Bacardit, Jaume">
    <meta name="citation_author_institution" content="School of Computing, Newcastle University, Newcastle Upon Tyne, UK">
    <meta name="citation_author" content="King, Hamish W.">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Centre for Immunobiology, Blizard Institute, Queen Mary University of London, London, UK">
    <meta name="citation_author" content="Rostron, Anthony J.">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author_institution" content="Integrated Critical Care Unit, Sunderland Royal Hospital, South Tyneside and Sunderland NHS Foundation Trust, Sunderland, UK">
    <meta name="citation_author" content="Simpson, A. John">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Hambleton, Sophie">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Laurenti, Elisa">
    <meta name="citation_author_institution" content="Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK">
    <meta name="citation_author" content="Lyons, Paul A.">
    <meta name="citation_author_institution" content="Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK">
    <meta name="citation_author" content="Meyer, Kerstin B.">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Nikolić, Marko Z.">
    <meta name="citation_author_institution" content="UCL Respiratory, Division of Medicine, University College London, London, UK">
    <meta name="citation_author_institution" content="University College London Hospitals NHS Foundation Trust, London, UK">
    <meta name="citation_author" content="Duncan, Christopher J. A.">
    <meta name="citation_author_institution" content="Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author_institution" content="Department of Infection and Tropical Medicine, Newcastle upon Tyne Hospitals NHS Foundation, Newcastle upon Tyne, UK">
    <meta name="citation_author" content="Smith, Kenneth G. C.">
    <meta name="citation_author_institution" content="Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK">
    <meta name="citation_author" content="Teichmann, Sarah A.">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Theory of Condensed Matter Group, Cavendish Laboratory/Department of Physics, University of Cambridge, Cambridge, UK">
    <meta name="citation_author" content="Clatworthy, Menna R.">
    <meta name="citation_author_institution" content="Molecular Immunity Unit, Department of Medicine, University of Cambridge, Cambridge, UK">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Cambridge Institute for Therapeutic Immunology and Infectious Disease, Cambridge Biomedical Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="NIHR Cambridge Biomedical Research Centre, Cambridge, UK">
    <meta name="citation_author" content="Marioni, John C.">
    <meta name="citation_author_institution" content="European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author" content="Göttgens, Berthold">
    <meta name="citation_author_institution" content="Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK">
    <meta name="citation_author" content="Haniffa, Muzlifah">
    <meta name="citation_author_institution" content="Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK">
    <meta name="citation_author_institution" content="NIHR Newcastle Biomedical Research Centre, Newcastle Hospitals NHS Foundation Trust, Newcastle upon Tyne, UK">
    <meta name="citation_author_institution" content="Department of Dermatology, Newcastle Hospitals NHS Foundation Trust, Newcastle upon Tyne, UK">
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">
    <meta name="twitter:site" content="@naturemedicine">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image:alt" content="Content cover image">
    <meta name="twitter:title" content="Single-cell multi-omics analysis of the immune response in COVID-19">
    <meta name="twitter:description" content="Nature Medicine - Transcriptomic and proteomic profiling of blood samples from individuals with COVID-19 reveals immune cell and hematopoietic progenitor cell alterations that are differentially...">
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig1_HTML.png">
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41591-021-01329-2">
    <meta property="og:type" content="article">
    <meta property="og:site_name" content="Nature">
    <meta property="og:title" content="Single-cell multi-omics analysis of the immune response in COVID-19 - Nature Medicine">
    <meta property="og:description" content="Transcriptomic and proteomic profiling of blood samples from individuals with COVID-19 reveals immune cell and hematopoietic progenitor cell alterations that are differentially associated with disease severity.">
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig1_HTML.png">
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
<style type="text/css">.cc-banner{background-color:#01324b;border:none!important;bottom:0;box-sizing:border-box;color:#fff!important;left:0;line-height:1.3;margin:auto 0 0;max-width:100%;outline:0;overflow:visible;padding:0;position:fixed;right:0;width:100%;z-index:99999}.cc-banner::backdrop{background-color:#0000004d}.cc-banner *{color:inherit!important}.cc-banner:focus{box-shadow:none;outline:0}.cc-banner a{color:#fff!important;text-decoration:underline}.cc-banner a:active,.cc-banner a:focus,.cc-banner a:hover{color:inherit;text-decoration:none}.cc-banner a:focus{outline:3px solid #08c!important}.cc-banner h2,.cc-banner h3,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-banner .cc-h2,.cc-banner h2{font-size:18px}.cc-banner .cc-h3,.cc-banner h3{font-size:16px}.cc-banner .cc-h4,.cc-banner .cc-h5,.cc-banner .cc-h6,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-size:14px}.cc-banner .cc-button{font-size:16px}.cc-banner__content{background-color:#01324b;display:flex;flex-direction:column;margin:0 auto;max-height:90vh;max-width:100%;padding:16px;position:relative}.cc-banner__content:focus{outline:0}@media (min-width:680px){.cc-banner__content{padding:12px}}@media (min-width:980px){.cc-banner__content{max-height:60vh;padding-bottom:20px;padding-top:20px}}@media (min-width:1320px){.cc-banner__content{padding-bottom:40px;padding-top:40px}}.cc-banner__container{display:flex;flex-direction:column;margin:auto;max-width:1320px;overflow:auto}.cc-banner__title{background:none!important;border:0;flex-shrink:0;font-size:18px!important;font-size:22px!important;font-weight:700!important;letter-spacing:normal;margin:0 0 12px!important;padding:0}@media (min-width:680px){.cc-banner__title{font-size:24px!important;margin:0 0 16px!important}}@media (min-width:1320px){.cc-banner__title{font-size:26px!important;margin:0 0 24px!important}}.cc-banner__body{display:flex;flex-direction:column;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif!important;overflow-x:hidden;overflow-y:auto;padding:3px 3px 16px}@media (min-width:980px){.cc-banner__body{flex-direction:row}}.cc-banner__policy p{font-size:16px!important;margin:0;max-width:none;padding:0}.cc-banner__policy p:not(:last-child){margin:0 0 16px}@media (min-width:980px){.cc-banner__policy p:not(:last-child){margin:0 0 24px}}.cc-banner__policy p a{font-size:16px!important;font-weight:700}.cc-banner__footer{box-shadow:none;flex-shrink:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif !important!important;margin:0;padding:12px 0 3px 3px;position:relative}@media (min-width:380px){.cc-banner__footer{align-items:stretch;display:flex;flex-wrap:wrap}}@media (min-width:680px){.cc-banner__footer{box-shadow:none;flex-wrap:nowrap;width:40%}}.cc-banner__footer .cc-banner__button{flex:1 1 auto;font-weight:700;overflow:hidden;padding:.5em 1em;width:100%}@media (min-width:680px){.cc-banner__footer .cc-banner__button{max-width:275px}}.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:12px}@media (min-width:680px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:0;margin-right:16px}}@media (min-width:980px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-right:24px}}@media (min-width:680px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:16px!important;margin-right:0!important}}@media (min-width:980px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:24px!important;margin-right:0!important}}.cc-banner__button-preferences{padding-left:0;padding-right:0}@media (min-width:380px){.cc-banner__button-preferences{flex:0 0 auto;margin:auto}}@media (min-width:680px){.cc-banner__button-preferences{margin:0}}@media (min-width:380px) and (max-width:680px){.cc-banner__button-break{display:block}}@media (min-width:680px){.cc-banner--is-tcf .cc-banner__footer{width:auto}}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__footer{padding-right:48px;width:66%}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px!important}.cc-banner--is-tcf .cc-banner__stacks{margin:16px 0 0}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__stacks{display:flex;flex:0 0 33%;flex-direction:column;margin:0 0 0 48px;overflow:auto}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px;margin:0 0 12px}.cc-banner--is-tcf .cc-banner__stacks-details{padding:0 3px}.cc-banner--is-tcf .cc-banner .cc-details{border-color:#fff3!important}.cc-box{border-radius:10px;padding:12px}.cc-box--info{background-color:#eff6fb}.cc-box--light{background-color:#faf9f6}.cc-button{border:2px solid #0000;border-radius:32px!important;color:inherit;cursor:pointer;font-size:14px;font-weight:700!important;left:auto;line-height:1.2;margin:0;padding:.5em 1em;right:auto;text-transform:none!important;transition:all .2s}@media (min-width:680px){.cc-button{font-size:14px;padding:.75em 1em}}.cc-button--sm{font-size:12px;padding:2px 8px!important}.cc-button--primary{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--primary:focus{outline:3px solid #08c}.cc-button--primary:focus,.cc-button--primary:hover,.cc-button--secondary{background-color:#fff!important;border-color:#025e8d!important;color:#025e8d!important}.cc-button--secondary:focus{background-color:#025e8d!important;border-color:#fff!important;color:#fff!important;outline:3px solid #08c!important}.cc-button--secondary:hover{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--secondary:active{border:2px solid #01324b!important}.cc-button--secondary.cc-button--contrast{border-color:#fff!important}.cc-button--secondary.cc-button--contrast:hover{background-color:initial!important;color:#fff!important}.cc-button--tertiary{background-color:#f0f7fc!important;background-image:none;border:2px solid #f0f7fc;color:#025e8d!important;text-decoration:underline!important;text-underline-offset:.25em}.cc-button--tertiary:focus{outline:3px solid #08c!important}.cc-button--tertiary:hover{text-decoration-thickness:.25em}.cc-button--tertiary:active{color:#013c5b}.cc-button--link{background-color:initial!important;border-color:#0000!important;border-radius:0!important;color:inherit!important;padding:0!important;text-decoration:underline!important;width:inherit!important}.cc-button--link:focus{outline:3px solid #08c}.cc-button--link:hover{background-color:initial!important;box-shadow:none;text-decoration:none}.cc-button--text{border-radius:0;padding:0}.cc-button--details{padding-right:24px!important;position:relative;width:auto}.cc-button--details:after,.cc-button--details:before{background-color:currentColor;content:"";position:absolute;transition:transform .25s ease-out}.cc-button--details:before{height:10px;margin-top:-5px;right:12px;top:50%;width:2px}.cc-button--details:after{height:2px;margin-top:-1px;right:8px;top:50%;width:10px}.cc-button--details.cc-active:before{transform:rotate(90deg);transform-origin:center}.cc-button--details.cc-active:after{display:none}.cc-details{border-bottom:1px solid #0000001a;padding:12px 0;position:relative;width:100%}.cc-details__summary{align-items:center;display:flex;font-size:14px;font-weight:700;list-style-type:none}.cc-details__summary:focus,.cc-details__summary:focus-visible{outline:3px solid #08c!important;will-change:auto}.cc-details__title{align-items:baseline;display:flex}.cc-details__title h2,.cc-details__title h3,.cc-details__title h4,.cc-details__title h5{line-height:1.4;margin:0!important;padding:0!important}.cc-details__title svg{flex-shrink:0;height:auto;margin-right:8px;position:relative;top:-1px;transition:all .2s;width:auto}.cc-details[open] .cc-details__title svg{top:2px;transform:rotate(90deg)}.cc-details__switch{margin-left:auto}.cc-details__section{padding:16px 0 0 18px}.cc-details__section p{margin:0}.cc-details__section p:not(:last-child){margin-bottom:12px}details summary::-webkit-details-marker{display:none}.cc-radio{align-items:center;display:flex;position:relative}.cc-radio *{cursor:pointer}.cc-radio__input{height:22px;left:0;position:absolute;top:0;width:22px}.cc-radio__input:focus{outline:none}.cc-radio__label{color:inherit;font-size:14px;font-weight:700;line-height:23px;margin:0;padding-left:28px}.cc-radio__label:after,.cc-radio__label:before{background-color:#fff;content:"";display:block;position:absolute;transition:transform .25s ease-out}.cc-radio__label:before{border:1px solid #777;border-radius:50%;height:22px;left:0;top:0;width:22px}.cc-radio__label:after{border:7px solid #025e8d;border-radius:50%;height:0;left:4px;opacity:0;top:4px;width:0}.cc-radio__input:focus+.cc-radio__label:before{box-shadow:0 0 0 2px #08c;outline:none}.cc-radio__label--hidden{display:none}.cc-radio__input:checked+.cc-radio__label:after{opacity:1}.cc-radio__input:disabled{cursor:default}.cc-radio__input:disabled+.cc-radio__label{cursor:default;opacity:.5}.cc-switch *{cursor:pointer}.cc-switch{align-items:center;display:flex;line-height:1}.cc-switch__label{cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;-webkit-user-select:none;user-select:none}.cc-switch__input[type=checkbox]{-webkit-appearance:none;appearance:none;background-color:#fff!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='rgba(0, 0, 0, 0.25)'/%3E%3C/svg%3E")!important;background-position:0!important;background-repeat:no-repeat!important;border:1px solid #777!important;border-radius:16px!important;height:18px!important;margin:0 8px 0 0!important;transition:background-position .2s ease-in-out!important;width:32px!important}.cc-switch__input[type=checkbox]:focus{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%2380b3cc'/%3E%3C/svg%3E")!important}.cc-switch__input[type=checkbox]:checked{background-color:#025e8d!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%23fff'/%3E%3C/svg%3E")!important;background-position:100%!important}.cc-switch__input[type=checkbox]:checked~.cc-switch__label>.cc-switch__text-off,.cc-switch__input[type=checkbox]:not(:checked)~.cc-switch__label>.cc-switch__text-on{display:none}.cc-switch__input[type=checkbox]:hover{cursor:pointer;outline:none}.cc-switch__input[type=checkbox]:focus-visible{cursor:pointer;outline:3px solid #fc0}.cc-switch__input[type=checkbox]:checked:before{background-color:none!important;color:#0000!important}html[dir=rtl] .cc-switch__input[type=checkbox]{margin-left:8px!important;margin-right:0!important}.cc-list>ul,ul.cc-list{list-style-type:disc;margin-left:0;padding-left:0}.cc-list>ul>li,ul.cc-list>li{margin-left:1em}.cc-list>ul>li:not(:last-child),ul.cc-list>li:not(:last-child){margin-bottom:4px}.cc-list__title{margin-bottom:8px!important}.cc-list--inline>ul,ul.cc-list--inline{display:flex;flex-wrap:wrap;list-style-type:none}.cc-list--inline>ul>li,ul.cc-list--inline>li{margin:0 .5em .5em 0}.cc-overlay{background-color:#0000004d;z-index:99998}.cc-overlay,.cc-preferences{bottom:0;left:0;position:fixed;right:0;top:0}.cc-preferences{background-color:#050a14f2!important;border:0;box-sizing:border-box;color:#111;font-family:sans-serif!important;line-height:1.4;margin:auto;max-height:100vh;overflow:auto;padding:0;z-index:100000}.cc-preferences:focus{outline:none}.cc-preferences *,.cc-preferences :after,.cc-preferences :before{box-sizing:inherit!important}.cc-preferences h2,.cc-preferences h3,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-preferences .cc-h2,.cc-preferences h2{font-size:18px}.cc-preferences .cc-h3,.cc-preferences h3{font-size:16px}.cc-preferences .cc-h4,.cc-preferences .cc-h5,.cc-preferences .cc-h6,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-size:14px}.cc-preferences a{color:#025e8d;text-decoration:underline}.cc-preferences a:hover{color:inherit;text-decoration:none}.cc-preferences h3{background:none;color:#111;padding:0;text-transform:none}dialog.cc-preferences{background-color:initial}dialog.cc-preferences::backdrop{background-color:#000000e6}.cc-preferences__dialog{display:flex;flex-direction:column;margin:auto;max-height:100vh;max-width:860px;padding:12px;position:relative}.cc-preferences__dialog>:last-child{border-bottom-left-radius:10px;border-bottom-right-radius:10px}@media (min-width:980px){.cc-preferences__dialog{padding:16px}}.cc-preferences__close{background:#0000!important;border:1px solid #ececec;border-radius:50%;color:#111!important;cursor:pointer;font-family:Times New Roman,serif;font-size:40px;height:40px;left:auto;line-height:1;margin-top:-20px;padding:0!important;position:absolute;right:20px;top:50%;width:40px}.cc-preferences__close:focus{outline:3px solid #08c}.cc-preferences__close-label{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__header{background:#fff!important;border-bottom:1px solid #0000001a;border-top-left-radius:10px;border-top-right-radius:10px;padding:16px;position:relative;text-align:center}.cc-preferences__title{background:none!important;color:#111!important;font-family:sans-serif!important;font-size:18px!important;font-weight:700!important;margin:0!important;padding:0 16px 0 0!important}@media (min-width:480px){.cc-preferences__title{padding-right:0!important}}@media (min-width:980px){.cc-preferences__title{font-size:22px!important}}.cc-preferences__body{background:#fff!important;flex:1 1 auto;min-height:200px;overflow-x:hidden;overflow-y:auto;padding:16px}.cc-preferences__footer{background:#fff!important;border-top:1px solid #d0d0d0;box-shadow:0 0 5px 0 #0003;margin-bottom:0;padding:12px;position:relative}@media (min-width:480px){.cc-preferences__footer{align-items:stretch;display:flex}}.cc-preferences__footer>.cc-button{display:block;width:100%}@media (min-width:480px){.cc-preferences__footer>.cc-button{flex:1 1 auto}}@media (min-width:980px){.cc-preferences__footer>.cc-button{flex-basis:auto}}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:first-child){margin-left:12px}}.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:8px}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:0}}.cc-preferences__categories{list-style:none;margin:0;padding:0}.cc-preferences__category:not(:last-child){border-bottom:1px solid #0000001a;margin-bottom:12px;padding-bottom:12px}.cc-preferences__category-description{font-size:14px;margin:0 0 8px;padding:0}.cc-preferences__category-footer{align-items:center;display:flex;justify-content:space-between}.cc-preferences__status{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__controls{display:flex;margin:0}.cc-preferences__controls>:not(:last-child){margin-right:12px}.cc-preferences__always-on{font-size:14px;font-weight:700;padding-left:26px;position:relative}.cc-preferences__always-on:before{background:#0000!important;border:solid;border-top-color:#0000;border-width:0 0 4px 4px;content:"";display:block;height:10px;left:0;position:absolute;top:2px;transform:rotate(-45deg);width:18px}.cc-preferences__details{background-color:#eff6fb!important;border-radius:10px;font-size:14px;margin:12px 0 0;padding:12px}.cc-preferences__cookie-list,.cc-preferences__provider-list{list-style:none;margin:0;padding:0}.cc-preferences__provider-list{columns:170px}.cc-preferences__cookie-title{font-size:1em;font-style:normal;font-weight:400;margin:0}.cc-preferences__cookie-description{font-size:1em;margin:0 0 8px}.cc-preferences__cookie-domain,.cc-preferences__cookie-lifespan{border-left:1px solid #999;color:#666;margin-left:8px;padding-left:4px}body.cc-has-preferences-open{overflow:hidden;position:relative}.cc-table{border-collapse:collapse;width:100%}.cc-table tbody tr{border-top:1px solid #0000001a}.cc-table td,.cc-table th{font-size:14px;padding:4px 8px;vertical-align:top}.cc-table+.cc-table{margin-top:8px}.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:70%}@media (min-width:680px){.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:80%}}.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{padding-left:8px;width:40%}@media (min-width:680px){.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{width:30%}}.cc-tabs,.cc-tabs>ul{background-color:#faf9f6;display:flex;list-style:none;margin:0;padding:0}.cc-tabs__panel{border-top:1px solid #ececec;margin-top:-1px}.cc-tabs__button{background-color:initial;border:0;border-left:1px solid #0000;border-right:1px solid #0000;color:#111;font-size:16px;font-weight:700;margin:0;padding:15px 16px 12px;position:relative}.cc-tabs__button:before{background-color:initial;content:"";display:block;height:3px;left:0;position:absolute;top:0;width:100%}.cc-tabs__button.cc-active{background-color:#fff;border-color:#025e8d #ececec #ececec;color:#111}.cc-tabs__button.cc-active:before{background-color:#025e8d}.cc-tabs__content{background-color:#fff;display:none;font-size:14px;padding:16px}.cc-tabs__section:not(:first-child){padding-top:24px}.cc-tcf{font-size:14px;margin-top:16px}.cc-tcf__list{list-style:none;margin:0;padding:0}.cc-vendor-count{color:#666;font-size:16px;margin:0 0 8px}.cc-hide{display:none!important}.cc-show{display:block!important}.cc-external-link{background-color:#ececec;background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTcgMGExIDEgMCAxIDEgMCAySDIuNmMtLjM3MSAwLS42LjIwOS0uNi41djE1YzAgLjI5MS4yMjkuNS42LjVoMTQuOGMuMzcxIDAgLjYtLjIwOS42LS41VjEzYTEgMSAwIDAgMSAyIDB2NC41YzAgMS40MzgtMS4xNjIgMi41LTIuNiAyLjVIMi42QzEuMTYyIDIwIDAgMTguOTM4IDAgMTcuNXYtMTVDMCAxLjA2MiAxLjE2MiAwIDIuNiAwem02IDBoNmwuMDc1LjAwMy4xMjYuMDE3LjExMS4wMy4xMTEuMDQ0LjA5OC4wNTIuMDk2LjA2Ny4wOS4wOGExIDEgMCAwIDEgLjA5Ny4xMTJsLjA3MS4xMS4wNTQuMTE0LjAzNS4xMDUuMDMuMTQ4TDIwIDF2NmExIDEgMCAwIDEtMiAwVjMuNDE0bC02LjY5MyA2LjY5M2ExIDEgMCAwIDEtMS40MTQtMS40MTRMMTYuNTg0IDJIMTNhMSAxIDAgMCAxLS45OTMtLjg4M0wxMiAxYTEgMSAwIDAgMSAxLTEiLz48L3N2Zz4=");background-position:right 8px center;background-repeat:no-repeat;background-size:10px auto;border-radius:3px;box-shadow:0 1px 0 #0003;color:#111!important;display:inline-block;font-size:12px;padding:4px 26px 4px 8px;text-decoration:none!important}.cc-external-link:hover{text-decoration:underline!important}.cc-m-0{margin:0!important}.cc-grey{color:#666}</style><script async="" type="text/javascript" src="https://injections.readcube.com/nature/inject.fa744f3f.js"></script></head>
<body class="article-page"><dialog class="cc-banner" data-cc-banner="" data-nosnippet="" aria-labelledby="cc-banner-label" open="">
		<div class="cc-banner__content" autofocus="" tabindex="-1">
			<div class="cc-banner__container">
				<div class="cc-banner__header">
					<h2 class="cc-banner__title" id="cc-banner-label">Your privacy, your choice</h2>
				</div>
				<div class="cc-banner__body">
					<div class="cc-banner__policy">
						<p>We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.</p><p>By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.</p><p>See our <a href="https://www.nature.com/info/privacy" data-cc-action="privacy">privacy policy</a> for more information on the use of your personal data.</p><p><button type="button" data-cc-action="preferences" class="cc-button cc-button--link cc-button--text">Manage preferences</button> for further information and to change your choices.</p>
					</div>
					
				</div>
				<div class="cc-banner__footer">
					
			<button data-cc-action="accept" class="cc-button cc-button--secondary cc-button--contrast cc-banner__button cc-banner__button-accept">Accept all cookies</button>
			
		
				</div>
			</div>
		</div>
	</dialog>

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90" style="display: none !important;">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-test="top-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41591-021-01329-2;doi=10.1038/s41591-021-01329-2;subjmeta=1949,208,232,250,2514,254,255,326,4130,514,596,631,692,699;kwrd=Haematopoiesis,Infection,RNA+sequencing,SARS-CoV-2,Viral+infection">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=690709882&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-021-01329-2%26doi%3D10.1038/s41591-021-01329-2%26subjmeta%3D1949,208,232,250,2514,254,255,326,4130,514,596,631,692,699%26kwrd%3DHaematopoiesis,Infection,RNA+sequencing,SARS-CoV-2,Viral+infection">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=690709882&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-021-01329-2%26doi%3D10.1038/s41591-021-01329-2%26subjmeta%3D1949,208,232,250,2514,254,255,326,4130,514,596,631,692,699%26kwrd%3DHaematopoiesis,Infection,RNA+sequencing,SARS-CoV-2,Viral+infection"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header="" data-track-component="nature-150-split-header" style="border-color:#e40428">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nm" data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" height="32" alt="Nature Medicine">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
                            </a><div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected="">All journals</option>
                                    <option value="nm">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41591-021-01329-2"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header" hidden="">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-articles" data-track="click" data-track-action="research articles" data-track-label="link" data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-and-analysis" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link" data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/podcast" data-track="click" data-track-action="podcasts" data-track-label="link" data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/current-issue" data-track="click" data-track-action="current issue" data-track-label="link" data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/collections" data-track="click" data-track-action="collections" data-track-label="link" data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://www.facebook.com/Nature-Medicine-193691346949/" data-track="click" data-track-action="facebook" data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://twitter.com/naturemedicine" data-track="click" data-track-action="twitter" data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=5" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/aims" data-track="click" data-track-action="aims &amp; scope" data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-information" data-track="click" data-track-action="journal information" data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-impact" data-track="click" data-track-action="journal metrics" data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editors" data-track="click" data-track-action="about the editors" data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-cross-journal-editorial-team" data-track="click" data-track-action="research cross-journal editorial team" data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-cross-journal-editorial-team" data-track="click" data-track-action="reviews cross-journal editorial team" data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/statistics-advisory-panel" data-track="click" data-track-action="statistical advisory panel" data-track-label="link">
                                    Statistical Advisory Panel
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/our-publishing-models" data-track="click" data-track-action="our publishing models" data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-values-statement" data-track="click" data-track-action="editorial values statement" data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-policies" data-track="click" data-track-action="editorial policies" data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/content" data-track="click" data-track-action="content types" data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="javascript:;" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                        </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/submission-guidelines" data-track="click" data-track-action="submission guidelines" data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-context="header publish with us dropdown menu" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/open-access-funding" data-test="funding-eligibility-link" data-track="click_explore_funding" data-track-context="header publish with us" data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link" href="https://mts-nmed.nature.com/cgi-bin/main.plex" data-track="click_submit_manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-action="submit manuscript" data-track-label="link (publish with us dropdown menu)" data-track-external="" data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link" href="https://journal-alerts.springernature.com/subscribe?journal_id=41591" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-label="link (desktop site header)" data-track-external="">
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature medicine"><span itemprop="name">nature medicine</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm/articles?type=article" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" id="js-enable-context-bar" data-test="context-bar" data-context-bar="" aria-hidden="true">
                    <div class="c-context-bar__container u-container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            Single-cell multi-omics analysis of the immune response in COVID-19
                        </div>
                        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-021-01329-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-021-01329-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2021-04-20">20 April 2021</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Single-cell multi-omics analysis of the immune response in COVID-19</h1>
                    <ul class="c-article-author-list c-article-author-list--long js-no-scroll" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Emily-Stephenson-Aff1" data-author-popup="auth-Emily-Stephenson-Aff1" data-author-search="Stephenson, Emily" data-track-context="researcher popup with no profile" data-track-index="1_69">Emily Stephenson</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Gary-Reynolds-Aff1" data-author-popup="auth-Gary-Reynolds-Aff1" data-author-search="Reynolds, Gary" data-track-context="researcher popup with no profile" data-track-index="2_69">Gary Reynolds</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Rachel_A_-Botting-Aff1" data-author-popup="auth-Rachel_A_-Botting-Aff1" data-author-search="Botting, Rachel A." data-track-context="researcher popup with no profile" data-track-index="3_69">Rachel A. Botting</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Fernando_J_-Calero_Nieto-Aff2" data-author-popup="auth-Fernando_J_-Calero_Nieto-Aff2" data-author-search="Calero-Nieto, Fernando J." data-track-context="researcher popup with no profile" data-track-index="4_69">Fernando J. Calero-Nieto</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-3358-8253"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-3358-8253</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael_D_-Morgan-Aff3-Aff4" data-author-popup="auth-Michael_D_-Morgan-Aff3-Aff4" data-author-search="Morgan, Michael D." data-track-context="researcher popup with no profile" data-track-index="5_69">Michael D. Morgan</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-0757-0711"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0757-0711</a></span><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Zewen_Kelvin-Tuong-Aff5-Aff6" data-author-popup="auth-Zewen_Kelvin-Tuong-Aff5-Aff6" data-author-search="Tuong, Zewen Kelvin" data-track-context="researcher popup with no profile" data-track-index="6_69">Zewen Kelvin Tuong</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-6735-6808"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6735-6808</a></span><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a>,<a href="#Aff6" tabindex="-1">6</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Karsten-Bach-Aff3-Aff4" data-author-popup="auth-Karsten-Bach-Aff3-Aff4" data-author-search="Bach, Karsten" data-track-context="researcher popup with no profile" data-track-index="7_69">Karsten Bach</a><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Waradon-Sungnak-Aff6" data-author-popup="auth-Waradon-Sungnak-Aff6" data-author-search="Sungnak, Waradon" data-track-context="researcher popup with no profile" data-track-index="8_69">Waradon Sungnak</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-0136-4960"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0136-4960</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kaylee_B_-Worlock-Aff7" data-author-popup="auth-Kaylee_B_-Worlock-Aff7" data-author-search="Worlock, Kaylee B." data-track-context="researcher popup with no profile" data-track-index="9_69">Kaylee B. Worlock</a><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Masahiro-Yoshida-Aff7" data-author-popup="auth-Masahiro-Yoshida-Aff7" data-author-search="Yoshida, Masahiro" data-track-context="researcher popup with no profile" data-track-index="10_69">Masahiro Yoshida</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3521-5322"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3521-5322</a></span><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Natsuhiko-Kumasaka-Aff6" data-author-popup="auth-Natsuhiko-Kumasaka-Aff6" data-author-search="Kumasaka, Natsuhiko" data-track-context="researcher popup with no profile" data-track-index="11_69">Natsuhiko Kumasaka</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3557-0375"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3557-0375</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Katarzyna-Kania-Aff4" data-author-popup="auth-Katarzyna-Kania-Aff4" data-author-search="Kania, Katarzyna" data-track-context="researcher popup with no profile" data-track-index="12_69">Katarzyna Kania</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Justin-Engelbert-Aff1" data-author-popup="auth-Justin-Engelbert-Aff1" data-author-search="Engelbert, Justin" data-track-context="researcher popup with no profile" data-track-index="13_69">Justin Engelbert</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Bayanne-Olabi-Aff1" data-author-popup="auth-Bayanne-Olabi-Aff1" data-author-search="Olabi, Bayanne" data-track-context="researcher popup with no profile" data-track-index="14_69">Bayanne Olabi</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jarmila_Stremenova-Spegarova-Aff8" data-author-popup="auth-Jarmila_Stremenova-Spegarova-Aff8" data-author-search="Spegarova, Jarmila Stremenova" data-track-context="researcher popup with no profile" data-track-index="15_69">Jarmila Stremenova Spegarova</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-0710-9266"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0710-9266</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Nicola_K_-Wilson-Aff2" data-author-popup="auth-Nicola_K_-Wilson-Aff2" data-author-search="Wilson, Nicola K." data-track-context="researcher popup with no profile" data-track-index="16_69">Nicola K. Wilson</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-0865-7333"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0865-7333</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Nicole-Mende-Aff2" data-author-popup="auth-Nicole-Mende-Aff2" data-author-search="Mende, Nicole" data-track-context="researcher popup with no profile" data-track-index="17_69">Nicole Mende</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-5078-2333"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-5078-2333</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Laura-Jardine-Aff1" data-author-popup="auth-Laura-Jardine-Aff1" data-author-search="Jardine, Laura" data-track-context="researcher popup with no profile" data-track-index="18_69">Laura Jardine</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Louis_C__S_-Gardner-Aff1" data-author-popup="auth-Louis_C__S_-Gardner-Aff1" data-author-search="Gardner, Louis C. S." data-track-context="researcher popup with no profile" data-track-index="19_69">Louis C. S. Gardner</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-7200-978X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-7200-978X</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Issac-Goh-Aff1" data-author-popup="auth-Issac-Goh-Aff1" data-author-search="Goh, Issac" data-track-context="researcher popup with no profile" data-track-index="20_69">Issac Goh</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-6397-3518"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6397-3518</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Dave-Horsfall-Aff1" data-author-popup="auth-Dave-Horsfall-Aff1" data-author-search="Horsfall, Dave" data-track-context="researcher popup with no profile" data-track-index="21_69">Dave Horsfall</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-8086-812X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8086-812X</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jim-McGrath-Aff1" data-author-popup="auth-Jim-McGrath-Aff1" data-author-search="McGrath, Jim" data-track-context="researcher popup with no profile" data-track-index="22_69">Jim McGrath</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Simone-Webb-Aff1" data-author-popup="auth-Simone-Webb-Aff1" data-author-search="Webb, Simone" data-track-context="researcher popup with no profile" data-track-index="23_69">Simone Webb</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-3058-8952"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-3058-8952</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael_W_-Mather-Aff1" data-author-popup="auth-Michael_W_-Mather-Aff1" data-author-search="Mather, Michael W." data-track-context="researcher popup with no profile" data-track-index="24_69">Michael W. Mather</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-7972-7111"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7972-7111</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Rik_G__H_-Lindeboom-Aff6" data-author-popup="auth-Rik_G__H_-Lindeboom-Aff6" data-author-search="Lindeboom, Rik G. H." data-track-context="researcher popup with no profile" data-track-index="25_69">Rik G. H. Lindeboom</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3660-504X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3660-504X</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Emma-Dann-Aff6" data-author-popup="auth-Emma-Dann-Aff6" data-author-search="Dann, Emma" data-track-context="researcher popup with no profile" data-track-index="26_69">Emma Dann</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-7400-7438"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-7400-7438</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ni-Huang-Aff6" data-author-popup="auth-Ni-Huang-Aff6" data-author-search="Huang, Ni" data-track-context="researcher popup with no profile" data-track-index="27_69">Ni Huang</a><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Krzysztof-Polanski-Aff6" data-author-popup="auth-Krzysztof-Polanski-Aff6" data-author-search="Polanski, Krzysztof" data-track-context="researcher popup with no profile" data-track-index="28_69">Krzysztof Polanski</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-2586-9576"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2586-9576</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Elena-Prigmore-Aff6" data-author-popup="auth-Elena-Prigmore-Aff6" data-author-search="Prigmore, Elena" data-track-context="researcher popup with no profile" data-track-index="29_69">Elena Prigmore</a><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Florian-Gothe-Aff8-Aff24" data-author-popup="auth-Florian-Gothe-Aff8-Aff24" data-author-search="Gothe, Florian" data-track-context="researcher popup with no profile" data-track-index="30_69">Florian Gothe</a><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup><sup class="u-js-hide">&nbsp;<a href="#nAff24" tabindex="-1">nAff24</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jonathan-Scott-Aff8" data-author-popup="auth-Jonathan-Scott-Aff8" data-author-search="Scott, Jonathan" data-track-context="researcher popup with no profile" data-track-index="31_69">Jonathan Scott</a><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Rebecca_P_-Payne-Aff8" data-author-popup="auth-Rebecca_P_-Payne-Aff8" data-author-search="Payne, Rebecca P." data-track-context="researcher popup with no profile" data-track-index="32_69">Rebecca P. Payne</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-9037-7367"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-9037-7367</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kenneth_F_-Baker-Aff8-Aff9" data-author-popup="auth-Kenneth_F_-Baker-Aff8-Aff9" data-author-search="Baker, Kenneth F." data-track-context="researcher popup with no profile" data-track-index="33_69">Kenneth F. Baker</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-6735-2911"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6735-2911</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff9" tabindex="-1">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Aidan_T_-Hanrath-Aff8-Aff10" data-author-popup="auth-Aidan_T_-Hanrath-Aff8-Aff10" data-author-search="Hanrath, Aidan T." data-track-context="researcher popup with no profile" data-track-index="34_69">Aidan T. Hanrath</a><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ina_C__D_-Schim_van_der_Loeff-Aff8" data-author-popup="auth-Ina_C__D_-Schim_van_der_Loeff-Aff8" data-author-search="Schim van der Loeff, Ina C. D." data-track-context="researcher popup with no profile" data-track-index="35_69">Ina C. D. Schim van der Loeff</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-1196-6196"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1196-6196</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Andrew_S_-Barr-Aff10" data-author-popup="auth-Andrew_S_-Barr-Aff10" data-author-search="Barr, Andrew S." data-track-context="researcher popup with no profile" data-track-index="36_69">Andrew S. Barr</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-8084-7644"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8084-7644</a></span><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Amada-Sanchez_Gonzalez-Aff10" data-author-popup="auth-Amada-Sanchez_Gonzalez-Aff10" data-author-search="Sanchez-Gonzalez, Amada" data-track-context="researcher popup with no profile" data-track-index="37_69">Amada Sanchez-Gonzalez</a><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Laura-Bergamaschi-Aff11-Aff12" data-author-popup="auth-Laura-Bergamaschi-Aff11-Aff12" data-author-search="Bergamaschi, Laura" data-track-context="researcher popup with no profile" data-track-index="38_69">Laura Bergamaschi</a><sup class="u-js-hide"><a href="#Aff11" tabindex="-1">11</a>,<a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Federica-Mescia-Aff11-Aff12" data-author-popup="auth-Federica-Mescia-Aff11-Aff12" data-author-search="Mescia, Federica" data-track-context="researcher popup with no profile" data-track-index="39_69">Federica Mescia</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-2759-4027"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2759-4027</a></span><sup class="u-js-hide"><a href="#Aff11" tabindex="-1">11</a>,<a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Josephine_L_-Barnes-Aff7" data-author-popup="auth-Josephine_L_-Barnes-Aff7" data-author-search="Barnes, Josephine L." data-track-context="researcher popup with no profile" data-track-index="40_69">Josephine L. Barnes</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-9938-3176"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9938-3176</a></span><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Eliz-Kilich-Aff13" data-author-popup="auth-Eliz-Kilich-Aff13" data-author-search="Kilich, Eliz" data-track-context="researcher popup with no profile" data-track-index="41_69">Eliz Kilich</a><sup class="u-js-hide"><a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Angus-Wilton-Aff13" data-author-popup="auth-Angus-Wilton-Aff13" data-author-search="de Wilton, Angus" data-track-context="researcher popup with no profile" data-track-index="42_69">Angus de Wilton</a><sup class="u-js-hide"><a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Anita-Saigal-Aff14" data-author-popup="auth-Anita-Saigal-Aff14" data-author-search="Saigal, Anita" data-track-context="researcher popup with no profile" data-track-index="43_69">Anita Saigal</a><sup class="u-js-hide"><a href="#Aff14" tabindex="-1">14</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Aarash-Saleh-Aff14" data-author-popup="auth-Aarash-Saleh-Aff14" data-author-search="Saleh, Aarash" data-track-context="researcher popup with no profile" data-track-index="44_69">Aarash Saleh</a><sup class="u-js-hide"><a href="#Aff14" tabindex="-1">14</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sam_M_-Janes-Aff7-Aff13" data-author-popup="auth-Sam_M_-Janes-Aff7-Aff13" data-author-search="Janes, Sam M." data-track-context="researcher popup with no profile" data-track-index="45_69">Sam M. Janes</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-6634-5939"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6634-5939</a></span><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Claire_M_-Smith-Aff15" data-author-popup="auth-Claire_M_-Smith-Aff15" data-author-search="Smith, Claire M." data-track-context="researcher popup with no profile" data-track-index="46_69">Claire M. Smith</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-8913-0009"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8913-0009</a></span><sup class="u-js-hide"><a href="#Aff15" tabindex="-1">15</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Nusayhah-Gopee-Aff1-Aff16" data-author-popup="auth-Nusayhah-Gopee-Aff1-Aff16" data-author-search="Gopee, Nusayhah" data-track-context="researcher popup with no profile" data-track-index="47_69">Nusayhah Gopee</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff16" tabindex="-1">16</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Caroline-Wilson-Aff1-Aff17" data-author-popup="auth-Caroline-Wilson-Aff1-Aff17" data-author-search="Wilson, Caroline" data-track-context="researcher popup with no profile" data-track-index="48_69">Caroline Wilson</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff17" tabindex="-1">17</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Paul-Coupland-Aff4" data-author-popup="auth-Paul-Coupland-Aff4" data-author-search="Coupland, Paul" data-track-context="researcher popup with no profile" data-track-index="49_69">Paul Coupland</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-2871-3374"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2871-3374</a></span><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jonathan_M_-Coxhead-Aff1" data-author-popup="auth-Jonathan_M_-Coxhead-Aff1" data-author-search="Coxhead, Jonathan M." data-track-context="researcher popup with no profile" data-track-index="50_69">Jonathan M. Coxhead</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Vladimir_Yu-Kiselev-Aff6" data-author-popup="auth-Vladimir_Yu-Kiselev-Aff6" data-author-search="Kiselev, Vladimir Yu" data-track-context="researcher popup with no profile" data-track-index="51_69">Vladimir Yu Kiselev</a><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stijn-Dongen-Aff6" data-author-popup="auth-Stijn-Dongen-Aff6" data-author-search="van Dongen, Stijn" data-track-context="researcher popup with no profile" data-track-index="52_69">Stijn van Dongen</a><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jaume-Bacardit-Aff18" data-author-popup="auth-Jaume-Bacardit-Aff18" data-author-search="Bacardit, Jaume" data-track-context="researcher popup with no profile" data-track-index="53_69">Jaume Bacardit</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-2692-7205"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2692-7205</a></span><sup class="u-js-hide"><a href="#Aff18" tabindex="-1">18</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Hamish_W_-King-Aff6-Aff19" data-author-popup="auth-Hamish_W_-King-Aff6-Aff19" data-author-search="King, Hamish W." data-track-context="researcher popup with no profile" data-track-index="54_69">Hamish W. King</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-5972-8926"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-5972-8926</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a>,<a href="#Aff19" tabindex="-1">19</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-author-popup="group-1" href="#group-1" data-track-context="researcher popup with no profile" data-track-index="55_69">Cambridge Institute of Therapeutic Immunology and Infectious Disease-National Institute of Health Research (CITIID-NIHR) COVID-19 BioResource Collaboration</a>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Anthony_J_-Rostron-Aff8-Aff20" data-author-popup="auth-Anthony_J_-Rostron-Aff8-Aff20" data-author-search="Rostron, Anthony J." data-track-context="researcher popup with no profile" data-track-index="56_69">Anthony J. Rostron</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-9336-1723"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-9336-1723</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff20" tabindex="-1">20</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-A__John-Simpson-Aff8" data-author-popup="auth-A__John-Simpson-Aff8" data-author-search="Simpson, A. John" data-track-context="researcher popup with no profile" data-track-index="57_69">A. John Simpson</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-4731-7294"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-4731-7294</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sophie-Hambleton-Aff8" data-author-popup="auth-Sophie-Hambleton-Aff8" data-author-search="Hambleton, Sophie" data-track-context="researcher popup with no profile" data-track-index="58_69">Sophie Hambleton</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-7954-3267"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7954-3267</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Elisa-Laurenti-Aff2" data-author-popup="auth-Elisa-Laurenti-Aff2" data-author-search="Laurenti, Elisa" data-track-context="researcher popup with no profile" data-track-index="59_69">Elisa Laurenti</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-9917-9092"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-9917-9092</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Paul_A_-Lyons-Aff11-Aff12" data-author-popup="auth-Paul_A_-Lyons-Aff11-Aff12" data-author-search="Lyons, Paul A." data-track-context="researcher popup with no profile" data-track-index="60_69">Paul A. Lyons</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-7035-8997"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7035-8997</a></span><sup class="u-js-hide"><a href="#Aff11" tabindex="-1">11</a>,<a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kerstin_B_-Meyer-Aff6" data-author-popup="auth-Kerstin_B_-Meyer-Aff6" data-author-search="Meyer, Kerstin B." data-track-context="researcher popup with no profile" data-track-index="61_69">Kerstin B. Meyer</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-5906-1498"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-5906-1498</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Marko_Z_-Nikoli_-Aff7-Aff13" data-author-popup="auth-Marko_Z_-Nikoli_-Aff7-Aff13" data-author-search="Nikolić, Marko Z." data-track-context="researcher popup with no profile" data-track-index="62_69">Marko Z. Nikolić</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6304-6848"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6304-6848</a></span><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff13" tabindex="-1">13</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Christopher_J__A_-Duncan-Aff8-Aff10" data-author-popup="auth-Christopher_J__A_-Duncan-Aff8-Aff10" data-author-search="Duncan, Christopher J. A." data-track-context="researcher popup with no profile" data-track-index="63_69">Christopher J. A. Duncan</a><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff10" tabindex="-1">10</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kenneth_G__C_-Smith-Aff11-Aff12" data-author-popup="auth-Kenneth_G__C_-Smith-Aff11-Aff12" data-author-search="Smith, Kenneth G. C." data-track-context="researcher popup with no profile" data-track-index="64_69">Kenneth G. C. Smith</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-3829-4326"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-3829-4326</a></span><sup class="u-js-hide"><a href="#Aff11" tabindex="-1">11</a>,<a href="#Aff12" tabindex="-1">12</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sarah_A_-Teichmann-Aff6-Aff21" data-author-popup="auth-Sarah_A_-Teichmann-Aff6-Aff21" data-author-search="Teichmann, Sarah A." data-corresp-id="c1" data-track-context="researcher popup with no profile" data-track-index="65_69">Sarah A. Teichmann<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-6294-6366"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6294-6366</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a>,<a href="#Aff21" tabindex="-1">21</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Menna_R_-Clatworthy-Aff5-Aff6-Aff12-Aff22-Aff23" data-author-popup="auth-Menna_R_-Clatworthy-Aff5-Aff6-Aff12-Aff22-Aff23" data-author-search="Clatworthy, Menna R." data-corresp-id="c2" data-track-context="researcher popup with no profile" data-track-index="66_69">Menna R. Clatworthy<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3340-9828"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3340-9828</a></span><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a>,<a href="#Aff6" tabindex="-1">6</a>,<a href="#Aff12" tabindex="-1">12</a>,<a href="#Aff22" tabindex="-1">22</a>,<a href="#Aff23" tabindex="-1">23</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-John_C_-Marioni-Aff3-Aff4-Aff6" data-author-popup="auth-John_C_-Marioni-Aff3-Aff4-Aff6" data-author-search="Marioni, John C." data-corresp-id="c3" data-track-context="researcher popup with no profile" data-track-index="67_69">John C. Marioni<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-9092-0852"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9092-0852</a></span><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff6" tabindex="-1">6</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Berthold-G_ttgens-Aff2" data-author-popup="auth-Berthold-G_ttgens-Aff2" data-author-search="Göttgens, Berthold" data-corresp-id="c4" data-track-context="researcher popup with no profile" data-track-index="68_69">Berthold Göttgens<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6302-5705"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6302-5705</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 69 authors for this article" title="Show all 69 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Muzlifah-Haniffa-Aff1-Aff6-Aff9-Aff16" data-author-popup="auth-Muzlifah-Haniffa-Aff1-Aff6-Aff9-Aff16" data-author-search="Haniffa, Muzlifah" data-corresp-id="c5" data-track-context="researcher popup with no profile" data-track-index="69_69">Muzlifah Haniffa<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3927-2084"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3927-2084</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff6" tabindex="-1">6</a>,<a href="#Aff9" tabindex="-1">9</a>,<a href="#Aff16" tabindex="-1">16</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup>&nbsp;</li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nm" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Medicine</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;27</b>,&nbsp;<span class="u-visually-hidden">pages </span>904–916 (<span data-test="article-publication-year">2021</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item" data-test="access-count">
                        <p class="c-article-metrics-bar__count">139k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item" data-test="citation-count">
                        <p class="c-article-metrics-bar__count">403 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item" data-test="altmetric-score">
                            <p class="c-article-metrics-bar__count">492 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41591-021-01329-2/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Analysis of human blood immune cells provides insights into the coordinated response to viral infections such as severe acute respiratory syndrome coronavirus 2, which causes coronavirus disease 2019 (COVID-19). We performed single-cell transcriptome, surface proteome and T and B lymphocyte antigen receptor analyses of over 780,000 peripheral blood mononuclear cells from a cross-sectional cohort of 130 patients with varying severities of COVID-19. We identified expansion of nonclassical monocytes expressing complement transcripts (<i>CD16</i><sup><i>+</i></sup><i>C1QA/B/C</i><sup><i>+</i></sup>) that sequester platelets and were predicted to replenish the alveolar macrophage pool in COVID-19. Early, uncommitted CD34<sup>+</sup> hematopoietic stem/progenitor cells were primed toward megakaryopoiesis, accompanied by expanded megakaryocyte-committed progenitors and increased platelet activation. Clonally expanded CD8<sup>+</sup> T cells and an increased ratio of CD8<sup>+</sup> effector T cells to effector memory T cells characterized severe disease, while circulating follicular helper T cells accompanied mild disease. We observed a relative loss of IgA2 in symptomatic disease despite an overall expansion of plasmablasts and plasma cells. Our study highlights the coordinated immune response that contributes to COVID-19 pathogenesis and reveals discrete cellular components that can be targeted for therapy.</p></div></div></section>

            
                
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41590-020-0762-x/MediaObjects/41590_2020_762_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41590-020-0762-x?fromPaywallRec=false" data-track="select_recommendations_1" data-track-context="inline recommendations" data-track-action="click recommendations inline - 1" data-track-label="10.1038/s41590-020-0762-x">Single-cell landscape of immunological responses in patients with COVID-19
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">12 August 2020</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41421-020-00225-2/MediaObjects/41421_2020_225_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41421-020-00225-2?fromPaywallRec=false" data-track="select_recommendations_2" data-track-context="inline recommendations" data-track-action="click recommendations inline - 2" data-track-label="10.1038/s41421-020-00225-2">The differential immune responses to COVID-19 in peripheral and lung revealed by single-cell RNA sequencing
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">20 October 2020</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41392-021-00753-7/MediaObjects/41392_2021_753_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41392-021-00753-7?fromPaywallRec=false" data-track="select_recommendations_3" data-track-context="inline recommendations" data-track-action="click recommendations inline - 3" data-track-label="10.1038/s41392-021-00753-7">Single-cell immune profiling reveals distinct immune response in asymptomatic COVID-19 patients
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">16 September 2021</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1749525514,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>The COVID-19 global pandemic has caused &gt;120 million infections and 2.6 million deaths (as of 17 March 2021)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Bedford, J. et al. COVID-19: towards controlling of a pandemic. Lancet 395, 1015–1018 (2020)." href="/articles/s41591-021-01329-2#ref-CR1" id="ref-link-section-d73692734e3237">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="John Hopkins University. COVID-19 map. Johns Hopkins Coronavirus Resource Center. 
                  https://coronavirus.jhu.edu/map.html
                  
                 (2020)." href="/articles/s41591-021-01329-2#ref-CR2" id="ref-link-section-d73692734e3240">2</a></sup>. Symptoms vary in severity and include acute respiratory distress syndrome, thrombosis and organ failure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Chen, N. et al. Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. Lancet 395, 507–513 (2020)." href="/articles/s41591-021-01329-2#ref-CR3" id="ref-link-section-d73692734e3244">3</a></sup>. COVID-19 is caused by severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2), a single-stranded RNA betacoronavirus that enters host cells through receptors such as angiotensin-converting enzyme 2 (ACE2) and neuropilin (NRP1), which are expressed widely, including in nasal epithelium<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hoffmann, M. et al. SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor. Cell 181, 271–280 (2020)." href="#ref-CR4" id="ref-link-section-d73692734e3248">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Diao, B. et al. Reduction and functional exhaustion of T cells in patients with coronavirus disease 2019 (COVID-19). Front. Immunol. 11, 827 (2020)." href="#ref-CR5" id="ref-link-section-d73692734e3248_1">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Lee, J. S. et al. Immunophenotyping of COVID-19 and influenza highlights the role of type I interferons in development of severe COVID-19. Sci. Immunol. 5, eabd1554 (2020)." href="/articles/s41591-021-01329-2#ref-CR6" id="ref-link-section-d73692734e3251">6</a></sup>.</p><p>Several studies have highlighted a complex network of peripheral blood immune responses in COVID-19 infection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Arunachalam, P. S. et al. Systems biological assessment of immunity to mild versus severe COVID-19 infection in humans. Science 369, 1210–1220 (2020)." href="/articles/s41591-021-01329-2#ref-CR7" id="ref-link-section-d73692734e3258">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Woodruff, M. C. et al. Extrafollicular B cell responses correlate with neutralizing antibodies and morbidity in COVID-19. Nat. Immunol. 21, 1506–1516 (2020)." href="/articles/s41591-021-01329-2#ref-CR8" id="ref-link-section-d73692734e3261">8</a></sup>. A reduction in T cells with disease severity and reduced interferon (IFN)-γ production by lymphocytes have been reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Chen, G. et al. Clinical and immunological features of severe and moderate coronavirus disease 2019. J. Clin. Invest. 130, 2620–2629 (2020)." href="/articles/s41591-021-01329-2#ref-CR9" id="ref-link-section-d73692734e3265">9</a></sup>. However, an expansion of highly cytotoxic effector T cell subsets in moderate to severe disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Zhang, J.-Y. et al. Single-cell landscape of immunological responses in patients with COVID-19. Nat. Immunol. 21, 1107–1118 (2020)." href="/articles/s41591-021-01329-2#ref-CR10" id="ref-link-section-d73692734e3269">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Liu, C. et al. Time-resolved systems immunology reveals a late juncture linked to fatal COVID-19. Cell 
                  https://doi.org/10.1016/j.cell.2021.02.018
                  
                 (2021)." href="/articles/s41591-021-01329-2#ref-CR11" id="ref-link-section-d73692734e3272">11</a></sup> and higher expression of exhaustion markers programmed cell death protein 1 and Tim-3 on CD8<sup>+</sup> T cells have been described in patients receiving intensive care therapy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Diao, B. et al. Reduction and functional exhaustion of T cells in patients with coronavirus disease 2019 (COVID-19). Front. Immunol. 11, 827 (2020)." href="/articles/s41591-021-01329-2#ref-CR5" id="ref-link-section-d73692734e3278">5</a></sup>. In severe cases, classical monocytes have been shown to display a type 1 IFN inflammatory signature<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Lee, J. S. et al. Immunophenotyping of COVID-19 and influenza highlights the role of type I interferons in development of severe COVID-19. Sci. Immunol. 5, eabd1554 (2020)." href="/articles/s41591-021-01329-2#ref-CR6" id="ref-link-section-d73692734e3283">6</a></sup>; however, low levels of IFNα coupled with a reduction in plasmacytoid dendritic cells (DCs) have been reported in patients with critical disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Hadjadj, J. et al. Impaired type I interferon activity and inflammatory responses in severe COVID-19 patients. Science 369, 718–724 (2020)." href="/articles/s41591-021-01329-2#ref-CR12" id="ref-link-section-d73692734e3287">12</a></sup>. Emergency and dysregulated myelopoiesis, and expanded activated megakaryocytes have also been reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Schulte-Schrepping, J. et al. Severe COVID-19 is marked by a dysregulated myeloid cell compartment. Cell 182, 1419–1440 (2020)." href="#ref-CR13" id="ref-link-section-d73692734e3291">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Mann, E. R. et al. Longitudinal immune profiling reveals key myeloid signatures associated with COVID-19. Sci. Immunol. 5, eabd6197 (2020)." href="#ref-CR14" id="ref-link-section-d73692734e3291_1">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Bernardes, J. P. et al. Longitudinal multi-omics analyses identify responses of megakaryocytes, erythroid cells and plasmablasts as hallmarks of severe COVID-19. Immunity 53, 1296–1314 (2020)." href="/articles/s41591-021-01329-2#ref-CR15" id="ref-link-section-d73692734e3294">15</a></sup>. Proliferating plasmablasts and extrafollicular B cell activation are present in critically ill patients, but despite high levels of SARS-CoV-2-specific antibodies and antibody-secreting cells, many of these patients do not recover<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Woodruff, M. C. et al. Extrafollicular B cell responses correlate with neutralizing antibodies and morbidity in COVID-19. Nat. Immunol. 21, 1506–1516 (2020)." href="/articles/s41591-021-01329-2#ref-CR8" id="ref-link-section-d73692734e3298">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Bernardes, J. P. et al. Longitudinal multi-omics analyses identify responses of megakaryocytes, erythroid cells and plasmablasts as hallmarks of severe COVID-19. Immunity 53, 1296–1314 (2020)." href="/articles/s41591-021-01329-2#ref-CR15" id="ref-link-section-d73692734e3301">15</a></sup>. To better understand the coordinated systemic immune response in individuals with asymptomatic and symptomatic COVID-19, we performed combined single-cell transcriptome, cell-surface protein and lymphocyte antigen receptor repertoire analysis of peripheral blood of a cross-sectional patient cohort and integrated results across three UK medical centers.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Altered cellular profiles across COVID-19 severities</h3><p>We generated single-cell combined transcriptome and surface proteome data from peripheral blood mononuclear cells (PBMCs) from individuals with asymptomatic, mild, moderate, severe and critical<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="World Health Organization. Clinical management of COVID-19. 
                  https://www.who.int/publications/i/item/clinical-management-of-covid-19
                  
                 (2021)." href="/articles/s41591-021-01329-2#ref-CR16" id="ref-link-section-d73692734e3317">16</a></sup> COVID-19 across three UK centers in Newcastle, Cambridge and London (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1a</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig6">1a</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">1</a>). Controls included healthy volunteers, individuals with non-COVID-19 severe respiratory illness and healthy volunteers administered with intravenous lipopolysaccharide (IV-LPS) as a surrogate for an acute systemic inflammatory response (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1a</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">2</a>). We sequenced 1,141,860 cells from 143 samples. Following computational doublet removal, 781,123 cells passed quality control (QC; minimum of 200 genes and &lt;10% mitochondrial reads per cell; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig6">1b</a>). Data were integrated using Harmony<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. Nat. Methods 16, 1289–1296 (2019)." href="/articles/s41591-021-01329-2#ref-CR17" id="ref-link-section-d73692734e3340">17</a></sup> with good mixing of cells by the kBET statistic calculated for each cluster across samples (rejection rate improved from 0.62 to 0.36 following integration, <i>P</i> &lt; 2.1 × 10<sup>−8</sup> by Wilcoxon paired signed-rank test; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig6">1c,d</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Single-cell multi-omics analysis of PBMCs from individuals with COVID-19 and controls."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Single-cell multi-omics analysis of PBMCs from individuals with COVID-19 and controls.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-021-01329-2/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="759"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Overview of the participants included and the samples and data collected. Figure was created using <a href="http://BioRender.com">BioRender.com</a>. <b>b</b>, UMAP visualization of all 781,123 cells after QC. Leiden clusters based on 5′ gene expression shown and colored by cell type. Lymph, lymphocyte; mono, monocyte; prolif, proliferating. <b>c</b>, Bar plot of the proportion of cell types shown in <b>b</b>, separated by condition and COVID-19 severity status. Hypothesis testing was performed using quasi-likelihood <i>F</i>-test comparing healthy controls to individuals with COVID-19 for linear trends across disease severity groups (healthy &gt; asymptomatic &gt; mild &gt; moderate &gt; severe &gt; critical). Differentially abundant cell types were determined using a 10% FDR and are marked with an asterisk. <b>d</b>, Enrichment of interferon response of each cell state separated by severity. IFN response was calculated using a published gene list (<a href="http://amigo.geneontology.org/amigo/term/GO:0034340">GO:0034340</a>) <b>e</b>, UMAP computed using batch-corrected mean staining intensities of 188 antibodies for 4,241 hyperspheres. Each hypersphere represents an area in the 188-dimensional space and is colored by significant (spatial FDR &lt; 0.05) severity-associated changes in abundance of cells within that space.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-021-01329-2/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Following Leiden clustering, cells were manually annotated based on the RNA expression of known marker genes supported by surface protein expression of markers employed in flow cytometry to discriminate subpopulations (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig6">1e</a>). We defined 18 cell subsets (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1b</a>), with an additional 27 cell states identified following subclustering (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1b</a>, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2a</a>, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3a,b</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4a,b</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">3</a>). Our annotation was further validated using Azimuth, whereby more than 50% of cells were mapped and matched to a unique cluster in 32/33 of the clusters defined in the Azimuth PBMC dataset (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-021-01329-2#Sec8">Methods</a>; proliferating CD8<sup>+</sup> T cells mapped across two clusters). Clusters unique to our data included proliferating monocytes, innate lymphoid cell subpopulations and isotype-specific plasma cells (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig6">1f</a>).</p><p>We observed a relative expansion of proliferating lymphocytes, proliferating monocytes, platelets and mobilized hematopoietic stem and progenitor cells (HSPCs) with worsening disease. Plasmablasts and B cells were expanded in severe and critical disease (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2a</a>). These changes matched trends in clinical blood cell counts (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2b</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">4</a>). To assess the broader impacts of patient characteristics and clinical metadata on the altered proportion of cell types/states, we used a Poisson linear mixed model (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-021-01329-2#Sec8">Methods</a>), which predicted the COVID-19 swab result (Bonferroni-corrected logistic regression (BF-corrected LR), <i>P</i> = 1.1 × 10<sup>−3</sup>; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-021-01329-2#Sec8">Methods</a>), disease severity at blood sampling (BF-corrected LR, <i>P</i> = 8.9 × 10<sup>−8</sup>) and center (contributed by increased red blood cells (RBCs) and reduced monocytes in the Cambridge patient cohort; (BF-corrected LR, <i>P</i> = 2.0 × 10<sup>−142</sup>) as the main contributing factors among seven different clinical/technical factors (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2c</a>). PBMC composition varied depending on symptom duration, with increased relative frequency of plasmacytoid dendritic cells (pDCs), natural killer (NK) cells, CD14<sup>+</sup> and CD16<sup>+</sup> monocytes (false discovery rate (FDR), 10%) and decreased relative frequencies of B cells, regulatory T (T<sub>reg</sub>) cells, RBCs, platelets and CD4<sup>+</sup> T cells, with a longer symptomatic interval before hospital admission (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2d</a>). These changes may be due to a subset of individuals in the critically ill category who reported a longer time since symptom onset, consistent with a protracted course of infection in critical disease (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2e,f</a>). However, concordant changes in immune cell composition were observed when excluding patients with either the longest symptom durations (&gt;24 d) or critical disease (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2g</a>), indicating that disease severity changes were not driven by symptom duration. Cell abundance results were also in agreement when performing a leave-one-out analysis (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2h</a>).</p><p>We observed expression of type I/III interferon response genes in monocytes, DCs and HSPCs across the spectrum of COVID-19 severity, but not in individuals challenged with IV-LPS, in keeping with the importance of type I and III interferons in innate immune responses to viral infection (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1d</a>). Type I/III interferon response-related genes were recently implicated in genome-wide association studies (GWAS) for COVID-19 susceptibility<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Severe COVID-19 GWAS Groupet al. Genome-wide association study of severe COVID-19 with respiratory failure. N. Engl. J. Med. 383, 1522–1534 (2020)." href="/articles/s41591-021-01329-2#ref-CR18" id="ref-link-section-d73692734e3509">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Pairo-Castineira, E. et al. Genetic mechanisms of critical illness in COVID-19. Nature 
                  https://doi.org/10.1038/s41586-020-03065-y
                  
                 (2020)." href="/articles/s41591-021-01329-2#ref-CR19" id="ref-link-section-d73692734e3512">19</a></sup>. <i>IFNAR2</i> was both upregulated in individuals with COVID-19 compared to healthy controls in most circulating cell types and highly expressed by plasmablasts, monocytes and DCs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2i</a>).</p><p>Multiplexed analysis of 45 proteins in serum showed two contrasting profiles between mild/moderate and severe/critical patients. CCL4, CXCL10, interleukin (IL)-7 and IL-1α were associated with severe and critical disease, suggesting an augmented drive for monocyte and NK lymphocyte recruitment, as well as support for T cell activity/pathology (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2j</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">5</a>).</p><p>We used Cydar<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Lun, A. T. L., Richard, A. C. &amp; Marioni, J. C. Testing for differential abundance in mass cytometry data. Nat. Methods 14, 707–709 (2017)." href="/articles/s41591-021-01329-2#ref-CR20" id="ref-link-section-d73692734e3535">20</a></sup> to characterize the immune landscape changes with disease severity based on surface protein expression by dividing cells into phenotypic hyperspheres. We quantified the number of cells from each severity group within the hyperspheres, identifying 608 hyperspheres that differed significantly in abundance with increasing severity (spatial FDR &lt; 0.05; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1e</a>). Differentially abundant hyperspheres were present in all major immune compartments. Notably, we found an increase in B cells (CD19<sup>+</sup>/CD20<sup>+</sup>), plasma cells (CD38<sup>+</sup>) and HSPCs (CD34<sup>+</sup>), as well as remodeling of the myeloid compartment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Schulte-Schrepping, J. et al. Severe COVID-19 is marked by a dysregulated myeloid cell compartment. Cell 182, 1419–1440 (2020)." href="/articles/s41591-021-01329-2#ref-CR13" id="ref-link-section-d73692734e3551">13</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1e</a>).</p><h3 class="c-article__sub-heading" id="Sec4">Mononuclear phagocytes and HSPC changes</h3><p>Transcriptome and surface proteome analysis of blood mononuclear phagocytes (MPs) identified known DC subsets (pDC, ASDC (<i>AXL</i><sup>+</sup><i>SIGLEC6</i><sup>+</sup> DC), DC1, DC2 and DC3) and several monocyte states (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2a,b</a>). Three CD14<sup>+</sup> monocyte states were present (proliferating, classical CD14<sup>+</sup> and activated CD83<sup>+</sup>) in addition to two CD16<sup>+</sup> monocyte states (nonclassical CD16<sup>+</sup> and C1QA/B/C<sup>+</sup>CD16<sup>+</sup>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2a,b</a>). Proliferating monocytes and DCs expressing <i>MKI67</i> and <i>TOP2A</i> were increased with disease severity (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2a,b</a>). In contrast, the frequencies of DC2 and DC3 were reduced. Proliferating monocytes, previously reported by flow cytometry analysis of blood from patients with COVID-19<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Mann, E. R. et al. Longitudinal immune profiling reveals key myeloid signatures associated with COVID-19. Sci. Immunol. 5, eabd6197 (2020)." href="/articles/s41591-021-01329-2#ref-CR14" id="ref-link-section-d73692734e3604">14</a></sup>, transcriptionally resembled CD14<sup>+</sup> monocytes and was the only population to change significantly with symptom duration. (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2a,b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig8">3a</a>). Proliferating DCs resembled the DC2 subset (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2a,b</a>). Rare <i>C1QA/B/C</i>-expressing CD16<sup>+</sup> monocytes were the only source of C1 complement components (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig8">3b</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Expansion of complement-expressing nonclassical monocytes and megakaryocyte-primed progenitor cells and increased platelet activation with COVID-19 disease severity."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Expansion of complement-expressing nonclassical monocytes and megakaryocyte-primed progenitor cells and increased platelet activation with COVID-19 disease severity.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-021-01329-2/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="914"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Dot plots of gene (left) and surface protein (right) expression for myeloid populations. <b>b</b>, Bar plot of the proportion of myeloid populations from the Newcastle and London sites. Hypothesis testing was performed using a quasi-likelihood <i>F</i>-test comparing healthy controls to individuals with COVID-19. Differentially abundant cell types were determined using a 10% FDR and are marked with an asterisk. <b>c</b>, PAGA graph representing connectivity between clusters defined in <b>a</b> for healthy (top left) and COVID-19 (bottom left) monocytes and BAL macrophages (mac). Expression of <i>IL6</i> (top right) and <i>TNF</i> (bottom right) in each cluster along the predicted path for COVID-19 monocytes. <b>d</b>, Expression of differentially expressed cytokines between CD83<sup>+</sup>CD14<sup>+</sup> monocytes and BAL macrophages shown by cells ordered by pseudotime calculated for cells from <b>c</b>. <b>e</b>, Dot plot of gene expression of DC-derived T cell polarizing cytokines in peripheral blood DC2 cells and mature BAL DCs. <b>f</b>, Heat map displaying gene-set enrichment scores for type 1/3 IFN response, TNF response and JAK–STAT signatures in the myeloid populations. Asterisks indicate significance compared to healthy controls. Absolute values and other comparisons are provided in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">7</a>. <b>g</b>, Heat map of predicted ligand–receptor interactions between platelets and monocyte subsets, using RNA data. <b>h</b>, Dot plot of significant differentially expressed genes between samples from healthy donors and individuals with COVID-19 filtered for platelet activation markers. <b>i</b>, UMAP representation of HSPCs (top) and dot plot of gene expression markers used to annotate clusters (bottom). MK, megakaryocyte; prog, progenitor. <b>j</b>, Bar chart depicting the proportion of progenitors. <b>k</b>, Box plots displaying the enrichment of a megakaryocyte progenitor signature in CD34<sup>+</sup>CD38<sup>+</sup> HSPCs (right) and CD34<sup>+</sup>CD38<sup>−</sup> (left), averaged per donor scores. Comparisons were made by an analysis of variance (ANOVA) with pairwise comparisons using Tukey’s test. Asterisks above bars indicate significance and are colored by the severity for which they were compared to. Absolute values are provided in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">8</a>. Boxes denote the interquartile range (IQR), and the median is shown as horizontal bars. Whiskers extend to 1.5 times the IQR, and outliers are shown as individual points (<i>P</i> values: CD38-negative cells in healthy versus LPS group (90 min), 0.3 × 10<sup>−3</sup>; CD38-positive cells in healthy versus moderate group, 0.7 × 10<sup>−3</sup>).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-021-01329-2/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We previously demonstrated egress of blood DCs and monocytes to the alveolar space with rapid acquisition of a lung molecular profile following human inhalational LPS challenge<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Jardine, L. et al. Lipopolysaccharide inhalation recruits monocytes and dendritic cell subsets to the alveolar airspace. Nat. Commun. 10, 1999 (2019)." href="/articles/s41591-021-01329-2#ref-CR21" id="ref-link-section-d73692734e3731">21</a></sup>. To investigate the relationship between circulating and lung alveolar MPs in COVID-19, we compared the transcriptome profile of blood DCs and monocytes with their bronchoalveolar lavage (BAL) counterparts using recently published data (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE145926">GSE145926</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Liao, M. et al. Single-cell landscape of bronchoalveolar immune cells in patients with COVID-19. Nat. Med. 26, 842–844 (2020)." href="/articles/s41591-021-01329-2#ref-CR22" id="ref-link-section-d73692734e3742">22</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig8">3d</a>). Partition-based graph abstraction (PAGA) suggested transcriptional similarity between circulating CD14<sup>+</sup> monocytes and BAL macrophages in health, aligning with recent data demonstrating that BAL macrophages can arise from circulating CD14<sup>+</sup> monocytes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2c</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Evren, E. et al. Distinct developmental pathways from blood monocytes generate human lung macrophage diversity. Immunity 
                  https://doi.org/10.1016/j.immuni.2020.12.003
                  
                 (2020)." href="/articles/s41591-021-01329-2#ref-CR23" id="ref-link-section-d73692734e3757">23</a></sup>. In COVID-19, there was greater transcriptional similarity between BAL macrophages and circulating C1QA/B/C<sup>+</sup>CD16<sup>+</sup> monocytes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2c</a>), suggesting a differential origin of alveolar macrophages in healthy donors and individuals with COVID-19. Both BAL macrophages and C1QA/B/C<sup>+</sup>CD16<sup>+</sup> monocytes express <i>FCGR3A</i> and <i>C1QA/B/C</i> and type I interferon response genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2a</a>). Myeloid hyperinflammation causing lung and peripheral tissue damage via secretion of inflammatory cytokines such as IL-6 and tumor necrosis factor (TNF) in COVID-19 has been reported and in our analysis were primarily expressed by tissue rather than blood MPs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2c</a>). Genes differentially expressed by blood monocytes identified <i>S100A8</i>, previously reported in COVID-19 as contributing to the cytokine storm in severe infection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. Cell 
                  https://doi.org/10.1016/j.cell.2021.01.053
                  
                 (2021)." href="/articles/s41591-021-01329-2#ref-CR24" id="ref-link-section-d73692734e3789">24</a></sup>. BAL macrophages expressed leukocyte-recruiting chemokines including <i>CCL2</i>, <i>CCL4</i>, <i>CCL7</i> and <i>CCL8</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2d</a>).</p><p>Tissue DCs respond to local inflammation and pathogen challenge by migrating to the draining lymph node to activate naïve T cells. BAL macrophages contain a population of mature, migratory DCs that express <i>CCR7</i> and <i>LAMP3</i> but downregulate DC-specific markers, such as <i>CD1C</i> and <i>CLEC9A</i> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig8">3c</a>). These migratory DCs express <i>IL10</i> in health, but <i>TNF</i> and the common IL-12 and IL-23 subunit <i>IL12B</i> in COVID-19, suggesting altered capacity for T cell polarization (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2e</a>). In peripheral blood, C1QA/B/C<sup>+</sup>CD16<sup>+</sup> monocytes expressed the highest amount of type 1 IFN response genes compared to all myeloid cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2f</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">6</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">7</a>). We detected minimal TNF-mediated or IL-6-mediated JAK–STAT signaling activation in circulating monocytes and DCs, but this was upregulated by COVID-19 BAL MPs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2f</a>, Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">6</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">7</a>).</p><p>Coagulation abnormalities and monocyte–platelet aggregates have been previously reported in COVID-19 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Levi, M., Thachil, J., Iba, T. &amp; Levy, J. H. Coagulation abnormalities and thrombosis in patients with COVID-19. Lancet Haematol. 7, e438–e440 (2020)." href="/articles/s41591-021-01329-2#ref-CR25" id="ref-link-section-d73692734e3866">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Hottz, E. D. et al. Platelet activation and platelet-monocyte aggregate formation trigger tissue factor expression in patients with severe COVID-19. Blood 136, 1330–1341 (2020)." href="/articles/s41591-021-01329-2#ref-CR26" id="ref-link-section-d73692734e3869">26</a></sup>), leading us to investigate predicted receptor–ligand interactions between monocytes and platelets using the CellPhoneDB repository. The expression levels of <i>SIRPA:CD47, FPR1:ANXA1, FPR2:APP</i> between monocytes:platelets were highest in the C1QA/B/C<sup>+</sup>CD16<sup>+</sup> monocytes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2g</a>). Using protein data, we identified ICAM1 interactions on platelets with CD11a/b/c/CD18 primarily on C1QA/B/C<sup>+</sup>CD16<sup>+</sup> monocytes and CD16<sup>+</sup> monocytes (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig8">3d</a>), accompanied by increased expression of surface proteins indicative of platelet activation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2h</a>).</p><p>Our large dataset of 781,123 PBMCs allowed us to interrogate 3,297 CD34<sup>+</sup> HSPCs. Leiden clustering and uniform manifold and projection (UMAP) visualization showed a cloud-like representation, consistent with a stem/progenitor cell landscape previously described for bone marrow HSPCs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Velten, L. et al. Human haematopoietic stem cell lineage commitment is a continuous process. Nat. Cell Biol. 19, 271–281 (2017)." href="/articles/s41591-021-01329-2#ref-CR27" id="ref-link-section-d73692734e3902">27</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2i</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig8">3e</a>). Absence of CD38 mRNA and protein expression marks the most immature cells within the CD34<sup>+</sup> compartment, while expression of markers such as <i>GATA1</i>, <i>MPO</i> and <i>PF4</i> characterizes distinct erythroid, myeloid and megakaryocytic progenitor populations, respectively (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2i</a>). Accordingly, we were able to annotate six transcriptional clusters as CD34<sup>+</sup>CD38<sup>−</sup> HSPCs, CD34<sup>+</sup>CD38<sup>+</sup> early progenitor HSPCs and CD34<sup>+</sup>CD38<sup>+</sup> erythroid, megakaryocytic and myeloid progenitors, as well as a small population distinguished by the expression of cell cycle (S phase) genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2i</a>). Megakaryocyte progenitors were absent in healthy and asymptomatic individuals but made up ~5% of CD34<sup>+</sup> cells in all symptomatic individuals (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2j</a>). As peripheral blood is not a site for hematopoiesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Mende, N. et al. Quantitative and molecular differences distinguish adult human medullary and extramedullary haematopoietic stem and progenitor cell landscapes. Preprint at BioRxiv 
                  https://doi.org/10.1101/2020.01.26.919753
                  
                 (2020)." href="/articles/s41591-021-01329-2#ref-CR28" id="ref-link-section-d73692734e3949">28</a></sup>, this finding likely reflects COVID-19-mediated perturbation of normal homeostatic functioning of the bone marrow HSPC compartment.</p><p>In light of our earlier observations of platelet activation and enhanced C1QA/B/C<sup>+</sup>CD16<sup>+</sup> monocyte–platelet interactions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2g,h</a>), the appearance of CD34<sup>+</sup> megakaryocyte progenitors was of particular interest, as it suggested a rebalancing of the stem/progenitor cell compartment. To further explore this hypothesis, we generated differential gene expression signatures between the megakaryocyte, myeloid and erythroid progenitor clusters to interrogate potential early activation or priming in the most immature HSPC clusters (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig8">3f</a>). We observed enrichment of the megakaryocyte progenitor signature in the CD38<sup>+</sup> HSPC populations in moderate COVID-19 compared to the healthy condition (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2k</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">8</a>), but no enrichment of erythroid or myeloid signatures in either CD38<sup>−</sup> or CD38<sup>+</sup> HSPCs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig8">3g</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">8</a>). Our earlier observation of increased platelet activation within the context of normal platelet counts (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2h</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2b</a>) is thus consistent with a model whereby a rebalancing of the HSPC compartment toward megakaryopoiesis may be compensating for peripheral platelet consumption in COVID-19.</p><h3 class="c-article__sub-heading" id="Sec5">T lymphocytes and T cell receptor changes</h3><p>Fine-resolution clustering of mRNA profiles revealed 11 initial clusters of CD4<sup>+</sup> T cells, CD8<sup>+</sup> T cells and innate-like T cells including γδ T cells, NK T cells and mucosal-associated invariant T (MAIT) cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3a</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig9">4</a>). Annotations were refined further using RNA expression of effector cytokines and surface protein expression (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3a–c</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Compositional and clonal analyses of T lymphocytes illustrate the expansion of effector subsets."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Compositional and clonal analyses of T lymphocytes illustrate the expansion of effector subsets.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-021-01329-2/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="797"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, UMAP visualization of 309,617 T cells based on gene expression shown and colored by cell type. Insets show the two-dimensional kernel density estimates of select T cell types in UMAP space. <b>b</b>, Dot plots of gene (top) and surface protein (bottom) expression for populations shown in <b>a. c</b>, Dot plots of gene expression of cytokine genes for populations shown in <b>a. d</b>, Box plots of cell type proportions that are differentially abundant between healthy donors and individuals with COVID-19. Boxes denote the IQR, and the median is shown as horizontal bars. Whiskers extend to 1.5 times the IQR and outliers are shown as individual points (<i>n</i> = 24 healthy, n = 86 COVID-19 biologically independent samples). <b>e</b>, Box plots of the proportion of cell types shown in <b>a</b>. Only cell types showing trends of changes by severity status are shown. Boxes denote IQR with median shown as horizontal bars. Whiskers extend to 1.5 times the IQR, and outliers are shown as individual points (<i>n</i> = 9 asymptomatic, <i>n</i> = 23 mild, <i>n</i> = 30 moderate, <i>n</i> = 13 severe, <i>n</i> = 10 critical biologically independent samples). <b>f</b>, Bar plots show the frequency of clonal T cells. Expanded clones denote clonotypes observed more than once. Asterisks indicate significance after multiple-testing correction (logistic regression using two-sided <i>t</i>-test with Benjamini–Hochberg FDR correction; CD4<sup>+</sup> T<sub>CM</sub> adjusted <i>P</i> = 0.119, CD4<sup>+</sup> T<sub>EM</sub> adjusted <i>P</i> = 0.472, CD4<sup>+</sup>IL-22<sup>+</sup> adjusted <i>P</i> = 0.01, CD4<sup>+</sup> prolif. adjusted <i>P</i> = 0.993, CD4<sup>+</sup> T<sub>H</sub>1 adjusted <i>P</i> = 0.993, CD4<sup>+</sup> T<sub>FH</sub> adjusted <i>P</i> = 0.109, T<sub>reg</sub> adjusted <i>P</i> = 0.993, CD8<sup>+</sup> prolif. adjusted <i>P</i> = 0.016, CD8<sup>+</sup> T<sub>TE</sub> adjusted <i>P</i> = 2.49 × 10<sup>−15</sup>, CD8<sup>+</sup> T<sub>EM</sub> adjusted <i>P</i> = 0.259). <b>g</b>, Box plots of the proportion of clonally expanded CD8<sup>+</sup> T<sub>EM</sub> cells (left), effector CD8<sup>+</sup> T cells (middle) and the ratio of effector CD8<sup>+</sup> T cells to CD8<sup>+</sup> T<sub>EM</sub> cells (right). Boxes denote the IQR, and the median is shown as horizontal bars. Whiskers extend to 1.5 times the IQR, and outliers are shown as individual points. Legend is as in <b>e</b>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-021-01329-2/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Cellular composition of the T cell compartment varied between healthy and infected groups (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3d</a>). Based on their relative proportions and differential abundance testing (FDR 10%), we found activated CD4<sup>+</sup> T cells expressing <i>IL</i><i>22</i>, circulating follicular helper T (T<sub>FH</sub>) cells, type 1 helper T (T<sub>H</sub>1) cells, CD8<sup>+</sup> effector memory T (T<sub>EM</sub>) cells and MAIT cells relatively enriched in individuals with asymptomatic and mild infection, with NKT, proliferating CD8<sup>+</sup> and CD4<sup>+</sup>, and CD8<sup>+</sup> terminal effector T (T<sub>TE</sub>) cells enriched in individuals with more severe infection (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig10">5a,b</a>). Treating disease severity as an ordinal variable (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-021-01329-2#Sec8">Methods</a>), multiple cell populations displayed nonlinear differences across disease severity (proliferating CD4<sup>+</sup> and CD8<sup>+</sup> T cells, CD8<sup>+</sup> T<sub>TE</sub>, CD4<sup>+</sup> T<sub>H</sub>1, CD4<sup>+</sup> T<sub>H</sub>17, CD4<sup>+</sup> central memory T (T<sub>CM</sub>) and IL-22<sup>+</sup>CD4<sup>+</sup> T cells), illustrating the complex compositional changes to peripheral T cells that occur with COVID-19 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig10">5b</a>). IL-22-expressing CD4<sup>+</sup> T cells seen in asymptomatic and mild disease could be associated with tissue-protective responses that may restrict immunopathology (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3e</a>) as previously shown for IL-22 in influenza A virus infection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Ivanov, S. et al. Interleukin-22 reduces lung inflammation during influenza A virus infection and protects against secondary bacterial infection. J. Virol. 87, 6911–6924 (2013)." href="/articles/s41591-021-01329-2#ref-CR29" id="ref-link-section-d73692734e4247">29</a></sup> and lower viral load in COVID-19 patients’ lungs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Desai, N. et al. Temporal and spatial heterogeneity of host response to SARS-CoV-2 pulmonary infection. Nat. Commun. 11, 6319 (2020)." href="/articles/s41591-021-01329-2#ref-CR30" id="ref-link-section-d73692734e4251">30</a></sup>. Proliferating CD4<sup>+</sup> and CD8<sup>+</sup> T cells coexpressed exhaustion marker genes <i>LAG3</i> and <i>TOX</i> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig10">5c</a>), in keeping with previous studies of patients with severe COVID-19 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Diao, B. et al. Reduction and functional exhaustion of T cells in patients with coronavirus disease 2019 (COVID-19). Front. Immunol. 11, 827 (2020)." href="/articles/s41591-021-01329-2#ref-CR5" id="ref-link-section-d73692734e4269">5</a></sup>). In contrast to disease severity, CD4<sup>+</sup> T<sub>H</sub>1, CD4<sup>+</sup> T<sub>H</sub>2, CD4<sup>+</sup>IL-22<sup>+</sup> and CD4<sup>+</sup> T<sub>CM</sub> cells were enriched among individuals with longer symptom duration, while effector populations with a cytotoxic phenotype (CD8<sup>+</sup> T<sub>TE</sub>, CD8<sup>+</sup> T<sub>EM</sub>, MAIT and NK T cells) were enriched in individuals with shorter symptom duration (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig10">5d</a>).</p><p>Differential gene expression analysis across disease severity (FDR 1%) and gene-set enrichment analysis (GSEA) identified pathways associated with inflammation and T cell activation across multiple subsets, including IL-2–STAT5 signaling, mTORC1 signaling, inflammatory response, IFNγ response, and IL-6–JAK–STAT3 signaling (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig10">5e</a>). The increased activation and cytotoxic phenotype in T cells from individuals with COVID-19 was functionally validated by flow cytometry analysis of PBMCs stimulated ex vivo with SARS-CoV-2 peptide showing upregulation of CD137 and CD107α (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig10">5f</a>).</p><p>T cell receptor (TCR) clonality analysis showed that effector CD8<sup>+</sup> T cells were the most clonally expanded (odds ratio (OR) (95% confidence interval (CI)) 1.81 (1.58–2.10)), <i>P</i> = 2.49 × 10<sup>−15</sup>) and their relative proportion increased with disease severity (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3f,g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig10">5g,h</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">6</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">7</a>). Conversely, the relative proportion of clonally expanded effector memory CD8<sup>+</sup> T<sub>EM</sub> cells decreased in individuals with more severe disease (OR (95% CI) 0.87 (0.72–1.04), <i>P</i> = 0.26; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3f,g</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">9</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">10</a>). These clonal alterations were primarily driven by severity rather than differences in symptom duration for more severely ill patients, as CD8<sup>+</sup> T<sub>EM</sub> clones expanded in individuals who had longer symptom duration, in line with a more developed infection trajectory in these individuals (per day, OR (95% CI) 1.02 (1.01–1.03), <i>P</i> = 2.66 × 10<sup>−10</sup>). The ratio of effector CD8<sup>+</sup> T cells to CD8<sup>+</sup> T<sub>EM</sub> cells (TE:EM ratio) correlated with disease severity (linear model <i>β</i> 2.97, <i>P</i> = 2.92 × 10<sup>−18</sup>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3g</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">9</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">10</a>), suggesting that CD8<sup>+</sup> T cell differentiation outcome may contribute to both antiviral protection and immunopathology, as previously reported in animal models<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Joshi, N. S. et al. Inflammation directs memory precursor and short-lived effector CD8+ T cell fates via the graded expression of T-bet transcription factor. Immunity 27, 281–295 (2007)." href="/articles/s41591-021-01329-2#ref-CR31" id="ref-link-section-d73692734e4387">31</a></sup>, although bystander T cell activation cannot be excluded.</p><h3 class="c-article__sub-heading" id="Sec6">B lymphocytes and B cell receptor changes</h3><p>Re-clustering of B cells and plasma cells identified nine clusters that were annotated according to canonical marker expression (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4a,b</a>), and previously published transcriptional signatures (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6a</a>). This included immature, naïve, switched and non-switched memory B cells, and a cluster of cells that enriched for markers previously described in exhausted memory B cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Monaco, G. et al. RNA-seq signatures normalized by mRNA abundance allow absolute deconvolution of human immune cell types. Cell Rep. 26, 1627–1640 (2019)." href="/articles/s41591-021-01329-2#ref-CR32" id="ref-link-section-d73692734e4405">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Adlowitz, D. G. et al. Expansion of activated peripheral blood memory B cells in rheumatoid arthritis, impact of B cell depletion therapy, and biomarkers of response. PLoS ONE 10, e0128269 (2015)." href="/articles/s41591-021-01329-2#ref-CR33" id="ref-link-section-d73692734e4408">33</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4a,b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6a</a>). We also found a large population of plasmablasts with negative expression of CD19 and CD20, with high expression of the proliferation marker <i>MKI67</i>, consistent with previous reports on severe SARS-CoV-2 infection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Bernardes, J. P. et al. Longitudinal multi-omics analyses identify responses of megakaryocytes, erythroid cells and plasmablasts as hallmarks of severe COVID-19. Immunity 53, 1296–1314 (2020)." href="/articles/s41591-021-01329-2#ref-CR15" id="ref-link-section-d73692734e4422">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. Cell 
                  https://doi.org/10.1016/j.cell.2021.01.053
                  
                 (2021)." href="/articles/s41591-021-01329-2#ref-CR24" id="ref-link-section-d73692734e4425">24</a></sup>, as well as IgM<sup>+</sup>, IgG<sup>+</sup> and IgA<sup>+</sup> plasma cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4a,b</a>). In individuals with symptomatic COVID-19, there was a significant expansion of plasmablasts and plasma cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6b</a>). The magnitude of this expansion increased from mild to moderate disease but was attenuated in severe to critical disease. This observation persisted even after accounting for days from symptom onset (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6b</a>). IgA<sup>+</sup> cells were decreased in individuals with symptomatic COVID-19 due to a significant decrease of the IgA2 subclass (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6b,c</a>), suggestive of the maintenance of an effective mucosal humoral response in asymptomatic individuals. In parallel, we observed the greatest expansion of circulating follicular helper T (cT<sub>FH</sub>) cells in asymptomatic individuals and a strong positive correlation between cT<sub>FH</sub> cells and plasma cells in individuals with asymptomatic/mild disease that was lost with increasing disease severity (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3e</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig10">5a,b</a>). Together, this suggests the presence of coordinated T cell and B cell responses in asymptomatic and mild disease, generating effective antiviral humoral immunity that becomes uncoupled in severe and critical disease. This is consistent with previous findings relating to the requirement of T<sub>FH</sub> cells for optimal antibody responses and high-quality neutralizing antibodies in viral infection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Crotty, S. T. T follicular helper cell biology: a decade of discovery and diseases. Immunity 50, 1132–1148 (2019)." href="/articles/s41591-021-01329-2#ref-CR34" id="ref-link-section-d73692734e4473">34</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Single-cell analysis of B lymphocytes and BCR repertoire reveal plasmablast expansion and clonality differences between genders."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Single-cell analysis of B lymphocytes and BCR repertoire reveal plasmablast expansion and clonality differences between genders.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-021-01329-2/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="999"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, UMAP visualization of 74,019 cells in the B cell lineage identified from gene expression data. <b>b</b>, Dot plots of gene (top) and surface protein (bottom) expression for populations shown in <b>a. c</b>, Bar plot of the mean proportion of cell types shown in <b>a</b>. <b>d</b>, Proportion of total IgA and IgA2 in plasmablast and plasma cells based on BCR data. Kruskal–Wallis test with Benjamini–Hochberg correction. <b>e</b>, Coordinated changes between T<sub>FH</sub> and B cells assessed by differential correlation analysis (empirical <i>P</i> ≤ 0.1). Shown is the Pearson correlation (± bootstrap s.e.m.) between T<sub>FH</sub> proportions and plasmablast or plasma cell (combined); only significant trends are shown. <b>f</b>, GSEA of MSigDB hallmark signatures in naive B cells, switched memory B cells and plasmablasts for asymptomatic/symptomatic COVID-19 versus healthy groups. Size of circles indicate (absolute) normalized enrichment score (NES). GSEA (permutation) nominal <i>P</i> &lt; 0.05 and FDR &lt; 0.25 denoted by non-gray colored dots. EMT, epithelial–mesenchymal transition; UV, ultraviolet. <b>g</b>, Dot plots of genes related to TNF signaling and BCR signaling in naive B cells, switched memory B cells and plasmablasts. Size of circles indicates the percentage of cells expressing the gene, and color gradient corresponds to increasing mean expression value. <b>h</b>, Scatterplot of clonotype size by node closeness centrality gini indices with marginal histograms indicating the distribution. Each dot represents an individual. <b>i</b>, BCR overlap incidence plot. Nodes correspond to individual donors colored by (inner ring) severity and (outer ring) site from which samples were collected. Edges indicate if at least one cell from each individual displayed an identical combination of heavy and light-chain V and J gene usage with CDR3 similarity allowance (≥85%). <b>j</b>, Clonotype size (left) and node closeness centrality gini indices (right) separated by gender. Mann–Whitney <i>U</i> test with Benjamini–Hochberg correction between the gender groups within each severity status. Color of adjusted <i>P</i> values indicates the gender group with the higher mean value. The box portion of the box plots extends from the 25th to 75th percentiles, whiskers extend from the smallest to largest values, and the middle line corresponds to the median. NS, not significant.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-021-01329-2/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>GSEA analysis identified interferon alpha response and interferon gamma response pathway genes enriched in all B cell subsets in individuals with COVID-19, and this was more marked in those with asymptomatic or mild disease, and attenuated in severe and critical disease (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6d</a>). The magnitude of type 1 interferon transcriptional response in B cells mirrored serum IFNα levels, which were highest in individuals with mild disease (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2j</a>), suggesting that the low expression of IFN response genes in B cells in severe or critical disease does not reflect an inability of B cells to respond to IFNα, but rather attenuation of IFNα. This may be because the initial antiviral response has waned in patients with severe or critical disease or because these individuals fail to sustain adequate IFNα production by myeloid cells and pDCs following symptom onset as previously reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Arunachalam, P. S. et al. Systems biological assessment of immunity to mild versus severe COVID-19 infection in humans. Science 369, 1210–1220 (2020)." href="/articles/s41591-021-01329-2#ref-CR7" id="ref-link-section-d73692734e4559">7</a></sup>. Longitudinal sampling would be required to distinguish these two possibilities.</p><p>In asymptomatic individuals, TNF signaling via nuclear factor kappa B (NF-κB) pathway genes was enriched in immature, naïve and switched memory B cells, but decreased in immature B cells and plasma cells in critical and severe disease (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6d</a>). Assessment of the leading-edge genes in this pathway demonstrated their markedly higher expression in all B cell and plasmablast/cell subsets in asymptomatic individuals with COVID-19 compared with those with symptomatic disease (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6e</a>). TNF was barely detectable in COVID-19 serum samples and highest in individuals with moderate disease (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig7">2j</a>), suggesting that another cytokine, for example IL-6, or stimulus may be responsible for NF-κB activation in asymptomatic individuals with COVID-19.</p><p>Hypoxia pathway genes were enriched in immature and naïve B cells only in asymptomatic individuals (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6d</a>). Since these individuals are unlikely to be hypoxic (given their lack of symptoms), we postulated that this signature may reflect another hypoxia inducible factor-activating stimulus, which includes B cell receptor (BCR) cross-linking<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Meng, X. et al. Hypoxia-inducible factor-1α is a critical transcription factor for IL-10-producing B cells in autoimmune disease. Nat. Commun. 9, 251 (2018)." href="/articles/s41591-021-01329-2#ref-CR35" id="ref-link-section-d73692734e4590">35</a></sup>. We assessed the expression of genes associated with BCR activation, such as <i>CD79A</i> and <i>CD79B</i>, and downstream kinases such as <i>BTK</i> in B cell subsets. Overall, BCR activation-associated genes were most highly expressed in B cells in healthy controls, followed by asymptomatic individuals with COVID-19, with lower expression observed in all symptomatic COVID-19 groups (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6e</a>). BCR activation threshold is also modulated by immune tyrosine inhibitory motif-containing receptors that recruit phosphatases, increasing the activation threshold of B cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Pritchard, N. R. &amp; Smith, K. G. C. B cell inhibitory receptors and autoimmunity. Immunology 108, 263–273 (2003)." href="/articles/s41591-021-01329-2#ref-CR36" id="ref-link-section-d73692734e4610">36</a></sup>. BCR inhibitory gene expression was limited, but <i>CD22</i> was detectable across B cell subsets in asymptomatic COVID-19, while <i>FCGR2B</i>, <i>CD72</i> and <i>PTPN6</i> expression was evident in B cells in severe COVID-19 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6e</a>). Together, this analysis suggests that B cells in asymptomatic individuals with COVID-19 and those with mild disease have a more pronounced response to interferons, increased NF-κB activation and a higher expression of genes associated with BCR activation signaling, suggesting a potential for greater BCR activation. Longitudinal analysis of patient samples will be required to establish if this is due to avid responses early in disease that prevent progression to a more severe phenotype or the immune response in early disease.</p><p>Following activation, B cells differentiate into antibody-producing plasma cells, accompanied by a progressive increase in oxidative metabolism<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Doughty, C. A. et al. Antigen receptor-mediated changes in glucose metabolism in B lymphocytes: role of phosphatidylinositol 3-kinase signaling in the glycolytic control of growth. Blood 107, 4458–4465 (2006)." href="/articles/s41591-021-01329-2#ref-CR37" id="ref-link-section-d73692734e4637">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Boothby, M. &amp; Rickert, R. C. Metabolic regulation of the immune humoral response. Immunity 46, 743–755 (2017)." href="/articles/s41591-021-01329-2#ref-CR38" id="ref-link-section-d73692734e4640">38</a></sup>. We observed differences in metabolic gene pathway expression in plasmablasts and plasma cells between disease severity categories, with enrichment of oxidative phosphorylation pathway genes in all disease groups, and a relative increase in glycolysis pathway genes in asymptomatic patient plasmablasts when compared to symptomatic disease groups (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig11">6e</a>).</p><p>We next assessed BCR clonality using <i>‘</i>dandelion’, a single-cell BCR-sequencing analysis package (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-021-01329-2#Sec8">Methods</a>), and found substantially more clonal expansion in symptomatic individuals with COVID-19 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4h</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig12">7</a>). Expanded clonotypes were found across all major cell types with larger clonotypes present primarily in plasmablast/plasma cell clusters (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig13">8a,b</a>). Within the expanded clonotypes, there was some evidence of class switching within symptomatic COVID-19 groups but not in the asymptomatic/healthy individuals (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig13">8c</a>). Unlike other large-scale single-cell RNA-sequencing (scRNA-seq) studies with BCR profiling<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Bernardes, J. P. et al. Longitudinal multi-omics analyses identify responses of megakaryocytes, erythroid cells and plasmablasts as hallmarks of severe COVID-19. Immunity 53, 1296–1314 (2020)." href="/articles/s41591-021-01329-2#ref-CR15" id="ref-link-section-d73692734e4672">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. Cell 
                  https://doi.org/10.1016/j.cell.2021.01.053
                  
                 (2021)." href="/articles/s41591-021-01329-2#ref-CR24" id="ref-link-section-d73692734e4675">24</a></sup>, there was no obvious bias of immunoglobulin heavy-chain variable (<i>IGHV</i>) gene usage in general (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig14">9a</a>). Disaggregating the <i>IGHV</i> gene usage data to individual gender groups showed that only <i>IGHV1-46</i> was significantly increased in women with critical COVID-19 relative to healthy controls (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig14">9a</a>). Some related BCRs were present in different individuals, with more incidence of V and J gene usage and related amino acid sequences of heavy-chain and light-chain CDR3s observed in individuals with severe or critical disease, and in individuals from one center (Newcastle; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4i</a>), which could arise due to local variants of the virus driving expansion of specific B cell clones. We note that none of these related BCRs were found to be expanded in the individuals, which was expected as only a relatively small number of B cells per individual were sampled. It would have been unlikely to find exactly matching heavy-chain and light-chain sequences across different individuals (even when allowing for somatic hypermutation variation), given the expected low coverage that arises from a small number of single cells (relative to bulk BCR sequencing). Finally, we observed disproportionate distribution in clonotype size, whether considering expanded or all clonotypes, and increased BCR mutations between men and women with COVID-19, with greater levels of both in women compared with men (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig4">4j</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig14">9b</a>). These differences in clonal expansion of B cells are consistent with a role in previous reports of worse outcomes in COVID-19 in men<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Guan, W.-J. et al. Clinical characteristics of coronavirus disease 2019 in China. N. Engl. J. Med. 382, 1708–1720 (2020)." href="/articles/s41591-021-01329-2#ref-CR39" id="ref-link-section-d73692734e4704">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Onder, G., Rezza, G. &amp; Brusaferro, S. Case-fatality rate and characteristics of patients dying in relation to COVID-19 in Italy. J. Am. Med. Assoc. 323, 1775–1776 (2020)." href="/articles/s41591-021-01329-2#ref-CR40" id="ref-link-section-d73692734e4707">40</a></sup>.</p><p>We summarize the immunological cellular and molecular profiles observed in our study distinguishing features between asymptomatic/mild disease from severe/critical disease (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig5">5</a>). Future longitudinal studies may enable us to distinguish if the distinct responses in asymptomatic and milder disease prevent progression to severe phenotypes.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Integrated framework of the peripheral immune response in COVID-19."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Integrated framework of the peripheral immune response in COVID-19.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-021-01329-2/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="757"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p>Schematic illustration of study highlights. Created with <a href="http://BioRender.com">BioRender.com</a>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-021-01329-2/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec7-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec7">Discussion</h2><div class="c-article-section__content" id="Sec7-content"><p>Our cross-sectional multi-omics PBMC survey revealed several new insights into COVID-19 pathogenesis. Firstly, peripheral blood monocytes and DCs exhibit an interferon response to infection. We identified CD1QA/B/C<sup>+</sup>CD16<sup>+</sup> monocytes, coexpressing receptors and ligands for interactions with platelets, that are predicted to replenish alveolar macrophages in COVID-19. Secondly, altered hematopoiesis is evident in the peripheral circulation with megakaryocyte-primed gene expression in the earliest CD34<sup>+</sup>CD38<sup>+</sup> HSPCs, and expanded megakaryocyte progenitors in the response to COVID-19. We reveal a balance in protective versus immunopathogenic adaptive immune responses in COVID-19 patients. Previous studies have reported expanded proliferative CD4<sup>+</sup> and CD8<sup>+</sup> T cells with disease severity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. Cell 
                  https://doi.org/10.1016/j.cell.2021.01.053
                  
                 (2021)." href="/articles/s41591-021-01329-2#ref-CR24" id="ref-link-section-d73692734e4766">24</a></sup>, but a reduction in ɣδ T cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. Cell 
                  https://doi.org/10.1016/j.cell.2021.01.053
                  
                 (2021)." href="/articles/s41591-021-01329-2#ref-CR24" id="ref-link-section-d73692734e4770">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Parrot, T. et al. MAIT cell activation and dynamics associated with COVID-19 disease severity. Sci. Immunol. 5, eabe1670 (2020)." href="/articles/s41591-021-01329-2#ref-CR41" id="ref-link-section-d73692734e4773">41</a></sup>, consistent with our study. In addition, we observed enrichment of T<sub>H</sub>1 cells in asymptomatic donors, consistent with previously reported IFNγ and IL-2 antigen-specific T cells in asymptomatic individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Le Bert, N. et al. Highly functional virus-specific cellular immune response in asymptomatic SARS-CoV-2 infection. J. Exp. Med. 218, e20202617 (2021)." href="/articles/s41591-021-01329-2#ref-CR42" id="ref-link-section-d73692734e4779">42</a></sup>. We report expansion of CD8<sup>+</sup> effector T cells, which likely include antigen-specific short-lived effector cells that could lead to uncontrolled inflammation and immunopathology, expanding on previous reports<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rydyznski Moderbacher, C. et al. Antigen-specific adaptive immunity to SARS-CoV-2 in acute COVID-19 and associations with age and disease severity. Cell 183, 996–1012 (2020)." href="#ref-CR43" id="ref-link-section-d73692734e4786">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kaneko, N. et al. Loss of Bcl-6-expressing T follicular helper cells and germinal centers in COVID-19. Cell 183, 143–157 (2020)." href="#ref-CR44" id="ref-link-section-d73692734e4786_1">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Maloy, K. J. et al. CD4+ T cell subsets during virus infection. Protective capacity depends on effector cytokine secretion and on migratory capability. J. Exp. Med. 191, 2159–2170 (2000)." href="#ref-CR45" id="ref-link-section-d73692734e4786_2">45</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Sahin, U. et al. COVID-19 vaccine BNT162b1 elicits human antibody and TH1 T cell responses. Nature 586, 594–599 (2020)." href="/articles/s41591-021-01329-2#ref-CR46" id="ref-link-section-d73692734e4789">46</a></sup>.</p><p>The expansion of plasmablasts and plasma cells is less evident in critical and severe disease than in moderate and mild disease, in contrast to previous studies that reported the diminished plasmablast expansion in convalescent stages and not within active disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Bernardes, J. P. et al. Longitudinal multi-omics analyses identify responses of megakaryocytes, erythroid cells and plasmablasts as hallmarks of severe COVID-19. Immunity 53, 1296–1314 (2020)." href="/articles/s41591-021-01329-2#ref-CR15" id="ref-link-section-d73692734e4796">15</a></sup>. This response is paralleled by the T<sub>FH</sub> profile in individuals with COVID-19 and is consistent with postmortem observations showing a lack of germinal centers in lymph nodes and spleen in individuals with fatal COVID-19 and a decrease in T<sub>FH</sub> cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Kaneko, N. et al. Loss of Bcl-6-expressing T follicular helper cells and germinal centers in COVID-19. Cell 183, 143–157 (2020)." href="/articles/s41591-021-01329-2#ref-CR44" id="ref-link-section-d73692734e4804">44</a></sup>. Our data revealed a significant decrease in IgA2 in symptomatic COVID-19 compared to asymptomatic donors, suggesting that maintenance of a robust mucosal humoral immune response may influence the fate of individuals infected with SARS-CoV-2. We observed a diminished IFNα response in the B cell compartments of individuals with critical and severe disease, further emphasizing a crucial role of these responses in outcomes, as previously reported in patients with COVID-19 who had type I IFN antibodies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Bastard, P. et al. Autoantibodies against type I IFNs in patients with life-threatening COVID-19. Science 370, eabd4585 (2020)." href="/articles/s41591-021-01329-2#ref-CR47" id="ref-link-section-d73692734e4808">47</a></sup>. Our data also suggest differential BCR clonality and mutation frequencies between gender groups, which may contribute to the differing clinical outcomes observed between men and women with COVID-19<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Takahashi, T. et al. Sex differences in immune responses that underlie COVID-19 disease outcomes. Nature 588, 315–320 (2020)." href="/articles/s41591-021-01329-2#ref-CR48" id="ref-link-section-d73692734e4813">48</a></sup>. Our study provides a valuable resource, exploitable for translational studies, and a template for future integrative meta-analysis of single-cell multi-omics datasets from individuals with COVID-19 worldwide.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec8-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec8">Methods</h2><div class="c-article-section__content" id="Sec8-content"><h3 class="c-article__sub-heading" id="Sec9">Ethics and sample collection</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec10">Newcastle</h4><p>Participants were recruited and consented under the Newcastle Biobank (Research Ethics Committee (REC) no. 17/NE/0361; Integrated Research Application System (IRAS) no. 233551) study and ethical governance. For the COVID-19-positive samples and healthy controls, peripheral blood was collected in EDTA tubes and serum separator tubes and processed within 4 h of collection.</p><p>For the IV-LPS control samples, ethical approval was granted by a REC (17/YH/0021). Healthy volunteers provided informed, written consent and passed the inclusion criteria as set out in the study protocol (Supplementary File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM1">1</a>.) LPS was obtained from Clinical Center Reference Endotoxin (94332B1, National Institutes of Health) and injected intravenously as a bolus dose of 2 ng kg<sup>−1</sup>. Blood samples were taken before IV-LPS administration (baseline), at 90 min, and at 10 h after challenge. Venous blood was drawn from an 18-gauge venous cannula and was collected into EDTA and serum separator tubes. Only samples from 90 min and 10 h were analyzed in this study.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec11">Cambridge</h4><p>Study participants were recruited between 31 March 2020 and 20 July 2020 from patients attending Addenbrooke’s Hospital with suspected COVID-19 or a confirmed diagnosis of COVID-19 by nucleic acid amplification test (including point-of-care testing), patients admitted to Royal Papworth Hospital NHS Foundation Trust or Cambridge and Peterborough Foundation Trust with a confirmed diagnosis of COVID-19, together with health-care workers identified through staff screening with a positive PCR test for SARS-CoV-2. Controls were recruited among hospital staff attending Addenbrooke’s serology screening program, and selected to cover the whole age spectrum of COVID-19-positive study participants, across gender groups. Only controls with negative serology results (45 of 47) were subsequently included in the study. Recruitment of inpatients at Addenbrooke’s Hospital and health-care workers was undertaken by the National Institute for Health Research (NIHR) Cambridge Clinical Research Facility outreach team and the NIHR BioResource research nurse team. Ethical approval was obtained from the East of England Cambridge Central Research Ethics Committee (NIHR BioResource, REC no. 17/EE/0025; ‘Genetic variation and altered leukocyte function in health and disease (GANDALF)’, REC no. 08/H0308/176). Informed consent was obtained from all participants. Each participant provided 27 ml of peripheral venous blood collected into a 9-ml sodium citrate tube.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec12">London</h4><p>Participants aged 18 years and older were recruited from two large hospital sites in London, United Kingdom, namely University College London (UCL) Hospitals NHS Foundation Trust and Royal Free London NHS Foundation Trust during the height of the pandemic in the United Kingdom (April to July 2020). All participants provided informed consent. Ethical approval was obtained through the Living Airway Biobank, administered through UCL Great Ormond Street Institute of Child Health (REC no. 19/NW/0171, IRAS project no. 261511), as well as by the local R&amp;D departments at both hospitals. At daily virtual COVID-19 coordination meetings, suitable participants were chosen from a list of newly diagnosed and admitted patients within the preceding 24 h (based on a positive nasopharyngeal swab for SARS-CoV-2). Participants with typical clinical and radiological COVID-19 features but with a negative screening test for SARS-CoV-2 were excluded. Other exclusion criteria included active hematological malignancy or cancer, known immunodeficiencies, sepsis from any cause and a blood transfusion within 4 weeks. Maximal severity of COVID-19 was determined retrospectively by identifying the presence of symptoms, the need for oxygen supplementation and the level of respiratory support. Peripheral blood sampling was performed before inclusion to any pharmacological interventional trials.</p><p>Samples were collected and transferred to a category level 3 facility at UCL and processed within 2 h of sample collection. Peripheral blood was centrifuged after adding Ficoll-Paque Plus, and PBMCs, serum and neutrophils were separated, collected and frozen for later processing.</p><h3 class="c-article__sub-heading" id="Sec13">Clinical status assignment</h3><p>Clinical metadata were collected at the point-of-care sample collection, including current oxygen requirements and location. This was used to assign disease severity status. Participants based on a ward and not requiring oxygen were defined as having ‘mild’ disease. Participants outside of an intensive care unit (ICU) environment requiring oxygen were defined as having ‘moderate’ disease. All patients in ICU and/or requiring noninvasive ventilation were defined as having ‘severe’ disease. Participants requiring intubation and ventilation were defined as having ‘critical’ disease. There were no patients in ICU that did not require supplemental oxygen.</p><h3 class="c-article__sub-heading" id="Sec14">PBMC isolation and dead cell removal</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec15">Newcastle</h4><p>PBMCs were isolated from blood samples using Lymphoprep (StemCell Technologies) density gradient centrifugation according to the manufacturer’s instructions. Single-cell suspensions were then washed with Dulbecco’s PBS (Sigma) and frozen in aliquots containing 5–10 million cells in 90% (vol/vol) heat-inactivated FCS (Gibco) and 10% (vol/vol) DMSO (Sigma-Aldrich). On the day of the experiment, the cells were thawed for 1 min, transferred to wash buffer (PBS supplemented with 2% (vol/vol) FCS and 2 mM EDTA) and centrifuged at 500<i>g</i> for 5 min. Resuspended cells were passed through a 30-μm filter and counted before live-cell magnetic-activated cell sorting (MACS) enrichment with the dead cell removal kit (Miltenyi Biotech), per the manufacturer’s instructions. Cell pellets were resuspended in microbeads and incubated at room temperature for 15 min. Each stained sample was passed through an LS column and rinsed with binding buffer (all from Miltenyi Biotec) before centrifugation. Cell pellets were resuspended in wash buffer and counted for antibody staining by cellular indexing of transcriptomes and epitopes by sequencing (CITE-seq).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec16">Cambridge</h4><p>PBMCs were isolated using Leucosep tubes (Greiner Bio-One) with Histopaque 1077 (Sigma) by centrifugation at 800<i>g</i> for 15 min at room temperature. PBMCs at the interface were collected, rinsed twice with autoMACS running buffer (Miltenyi Biotech) and cryopreserved in FBS with 10% DMSO. All samples were processed within 4 h of collection. Purified PBMCs were thawed at 37 °C, transferred to a 50-ml tube, and ten volumes of prewarmed thawing medium (IMDM; Gibco, 12440-053) with 50% (vol/vol) FCS (not heat inactivated; PAN-Biotech, P40-37500) and 0.1 mg ml<sup>−1</sup> DNase I (Worthington, LS002139)) were added slowly and dropwise, followed by centrifugation at 500<i>g</i> for 5 min. The pellet was resuspended in 1 ml of FACS buffer (PBS; Sigma, D8537) with 3% (vol/vol) heat-inactivated FCS, and the viability of each sample was assessed by counting in an improved Neubauer chamber using Trypan blue. Pools of four samples were generated by combining 0.5 million live cells per individual (2 million live cells in total). The pools were washed twice in FACS buffer (10 ml and 2 ml, respectively) followed by centrifugation for 5 min at 500<i>g</i>. The pellet was then resuspended in 35 μl of FACS buffer and the viability of each pool was assessed.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec17">London</h4><p>Peripheral whole blood was collected in EDTA tubes and processed fresh via Ficoll-Paque Plus separation (GE Healthcare, 17144002). The blood was first diluted with 5 ml 2 mM EDTA-PBS (Invitrogen, 1555785-038), before 10–20 ml of diluted blood was carefully layered onto 15 ml of Ficoll in a 50-ml falcon tube. If the sample volume was less than 5 ml, blood was diluted with an equal volume of EDTA-PBS and layered onto 3 ml Ficoll. The sample was centrifuged at 800<i>g</i> for 20 min at room temperature. The plasma layer was carefully removed and the PBMC layer collected using a sterile Pasteur pipette. The PBMC layer was washed with three volumes of EDTA-PBS by centrifugation at 500<i>g</i> for 10 min. The pellet was suspended in EDTA-PBS and centrifuged again at 300<i>g</i> for 5 min. The PBMC pellet was collected and the cell number and viability assessed using Trypan blue. Cell freezing medium (90% FBS and 10% DMSO) was added dropwise to PBMCs slowly on ice and the mixture cryopreserved at −80 °C until further full-sample processing.</p><h3 class="c-article__sub-heading" id="Sec18">TotalSeq-C antibody staining and 10x Chromium loading</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec19">Newcastle</h4><p>Approximately 200,000 cells from each donor were stained with Human TruStain FcX Fc Blocking Reagent (BioLegend, 422302) for 10 min at room temperature. The cells were then stained with the custom panel TotalSeq-C (BioLegend, 99813; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">1</a>) for 30 min at 4 °C. Cells were then washed twice with PBS supplemented with 2% (vol/vol) FCS and 2 mM EDTA (Sigma) before resuspending in PBS and counting. Approximately 20,000–30,000 cells per sample were loaded onto the 10x Chromium controller using Chromium NextGEM Single Cell V(D)J Reagent kits v1.1 with Feature Barcoding technology for Cell-Surface Protein (10x Genomics) according to the manufacturer’s protocol.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec20">Cambridge</h4><p>Half a million viable cells were resuspended in 25 μl of FACS buffer and incubated with 2.5 μl of Human TruStain FcX Fc blocking reagent (BioLegend, 422302) for 10 min at 4 °C. The TotalSeq-C antibody cocktail (BioLegend 99813; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">1</a>) was centrifuged at 14,000<i>g</i> at 4 °C for 1 min, resuspended in 52 μl of FACS buffer, incubated at room temperature for 5 min and centrifuged at 14,000<i>g</i> at 4 °C for 10 min. Around 25 μl of solution was subsequently added to each sample pool and incubated for 30 min at 4 °C in the dark. Pools were washed three times with 27 volumes (1.4 ml) of FACS buffer, followed by centrifugation at 500<i>g</i> for 5 min. The pellet was resuspended in 62.5 µl of 1× PBS + 0.04% BSA (Ambion, AM2616), filtered through a 40-μm cell strainer (Flowmi, H13680-0040), and viable cells of each sample pool were counted in an improved Neubauer chamber using Trypan blue. Around 50,000 live cells (up to a maximum of 60,000 total cells) for each pool were processed using Single Cell V(D)J 5′ version 1.1 (1000020) together with Single Cell 5′ Feature Barcode library kit (1000080), Single Cell V(D)J Enrichment Kit, human B cells (1000016) and Single Cell V(D)J Enrichment Kit, human T cells (10x Genomics, 1000005) according to the manufacturer’s protocols.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">London</h4><p>Frozen PBMC samples were thawed quickly in a water bath at 37 °C. Warm RPMI 1640 medium (20–30 ml) containing 10% FBS was added slowly to the cells before centrifuging at 300<i>g</i> for 5 min. The pellet was then washed with 5 ml RPMI 1640-FBS and centrifuged again (300<i>g</i> for 5 min). The PBMC pellet was collected and cell number and viability determined using Trypan blue. PBMCs from four different donors were then pooled together at equal numbers (1.25 × 10<sup>5</sup> PBMCs from each donor) to make up 5.0 × 10<sup>5</sup> cells in total. The remaining cells were used for DNA extraction (Qiagen, 69504). The pooled PBMCs were stained with TotalSeq-C antibodies (BioLegend, 99814) according to manufacturer’s instructions. After incubating with half a vial of TotalSeq-C for 30 min at 4 °C, PBMCs were washed three times by centrifugation at 500<i>g</i> for 5 min at 4 °C. PBMCs were counted again and processed immediately for 10x 5′ single-cell capture (Chromium Next GEM Single Cell V(D)J Reagent Kit v1.1 with Feature Barcoding technology for Cell-Surface Protein-Rev D protocol). Two lanes of 25,000 cells were loaded per pool on a 10x chip.</p><h3 class="c-article__sub-heading" id="Sec22">Library preparation and sequencing</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec23">Newcastle and London</h4><p>Gene expression, TCR-enriched and BCR-enriched libraries were prepared for each sample according to the manufacturer’s protocol (10x Genomics). Cell-surface protein libraries were subjected to double the manufacturer’s recommended primer concentration and seven to eight amplification cycles during the sample index PCR to reduce the likelihood of daisy chains forming. Libraries were pooled per participant using a ratio of 6:2:1:1 for gene expression, cell-surface protein, TCR-enriched and BCR-enriched libraries, respectively. All libraries were sequenced using a NovaSeq 6000 (Illumina) to achieve a minimum of 50,000 paired-end reads per cell for gene expression and 20,000 paired-end reads per cell for cell-surface protein, TCR-enriched and BCR-enriched libraries.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec24">Cambridge</h4><p>The samples were subjected to 12 cycles of cDNA amplification and 8 cycles for the cell-surface protein library construction. Following this, the libraries were processed according to the manufacturer’s protocol. Libraries were pooled per sample using a ratio of 9:2.4:1:0.6 for gene expression, cell-surface protein expression, and TCR and BCR enrichment libraries, respectively. Samples were sequenced using a NovaSeq 6000 (Illumina), using S1 flow cells.</p><h3 class="c-article__sub-heading" id="Sec25">Alignment and quantification</h3><p>Droplet libraries were processed using Cell Ranger v4.0. Reads were aligned to the GRCh38 human genome concatenated to the SARS-CoV-2 genome (NCBI SARS-CoV-2 isolate Wuhan-Hu-1) using STAR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. Bioinformatics 29, 15–21 (2013)." href="/articles/s41591-021-01329-2#ref-CR49" id="ref-link-section-d73692734e5008">49</a></sup> (v2.5.1b) and unique molecular identifiers (UMIs) deduplicated. CITE-seq UMIs were counted for GEX and ADT libraries simultaneously to generate feature-X droplet UMI count matrices.</p><h3 class="c-article__sub-heading" id="Sec26">Doublet identification</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec27">Newcastle</h4><p>Scrublet (v0.2.1) was applied to each sample to generate a doublet score. These formed a bimodal distribution so the tool’s automatic threshold was applied.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec28">Cambridge</h4><p>Non-empty droplets were called within each multiplexed pool of donors using the emptyDrops function implemented in the Bioconductor package DropletUtils (v1.10.3), using a UMI threshold of 100 and FDR of 1%. The probability of being a doublet was estimated for each cell per sample (that is, one 10x lane) using the ‘doubletCells’ function in ‘scran’ based on highly variable genes (HVGs). Next, we used ‘cluster_walktrap’ on the shared nearest neighbor graph that was computed on HVGs to form highly resolved clusters per sample. Per-sample clusters with either a median doublet score greater than the median + 2.5× median absolute deviation (MAD) or clusters containing more than the median + 2.5× MAD genotype doublets were tagged as doublets. This was followed by a second round of highly resolved clustering across the whole dataset, in which again cells belonging to clusters with a high proportion (&gt;60%) of cells previously labeled as doublets were also defined as doublets.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec29">London</h4><p>For pooled donor CITE-seq samples, the donor ID of each cell was determined by genotype-based demultiplexing using souporcell (v2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Heaton, H. et al. Souporcell: robust clustering of single-cell RNA-seq data by genotype without reference genotypes. Nat. Methods 17, 615–620 (2020)." href="/articles/s41591-021-01329-2#ref-CR50" id="ref-link-section-d73692734e5040">50</a></sup>. Souporcell analyses were performed with ‘skip_remap’ enabled and a set of known donor genotypes given under the ‘common_variants’ parameter. The donor ID of each souporcell genotype cluster was annotated by comparing each souporcell genotype to the set of known genotypes. Droplets that contained more than one genotype according to souporcell were flagged as ‘ground-truth’ doublets for heterotypic doublet identification. Ground-truth doublets were used by DoubletFinder (v2.0.3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="McGinnis, C. S., Murrow, L. M. &amp; Gartner, Z. J. DoubletFinder: doublet detection in single-cell RNA-sequencing data using artificial nearest neighbors. Cell Syst. 8, 329–337 (2019)." href="/articles/s41591-021-01329-2#ref-CR51" id="ref-link-section-d73692734e5044">51</a></sup> to empirically determine an optimal ‘p<i>K</i>’ value for doublet detection. DoubletFinder analysis was performed on each sample separately using ten principal components (PCs), a ‘p<i>N</i>’ value of 0.25, and the ‘nExp’ parameter estimated from the fraction of ground-truth doublets and the number of pooled donors.</p><h3 class="c-article__sub-heading" id="Sec30">CITE-seq background signal removal</h3><p>Background antibody-specific and nonspecific staining was subtracted from ADT counts in each data set from the 3 data acquisition sites separately. Antibody-derived tag (ADT) counts for each protein were first normalized using counts per million and log transformed, with a pseudocount of +1. To estimate the background signal for each protein, a two-component Gaussian mixture model, implemented in the R package function ‘mclust’ (v5.4.7), was fit across the droplets with a total UMI count of &gt;10 and &lt;100 from each experimental sample separately. The mean of the first Gaussian mixture model component for each protein was then subtracted from the log counts per million from the QC-passed droplets in the respective experimental sample.</p><h3 class="c-article__sub-heading" id="Sec31">Quality control, normalization, embedding and clustering</h3><p>Combined raw data from the three centers was filtered to remove cells that expressed fewer than 200 genes and &gt;10% mitochondrial reads. Data were normalized (scanpy: normalize_total), log + 1 corrected (scanpy: log1p) and HVGs identified using the Seurat vst algorithm (scanpy: highly_variable_genes). Harmony was used to adjust PCs by sample ID and used to generate the neighborhood graph and embedded using UMAP. Clustering was performed using the Leiden algorithm with an initial resolution of 3. For initial clustering, differentially expressed genes were calculated using the Wilcoxon rank-sum test.</p><h3 class="c-article__sub-heading" id="Sec32">Cluster differential abundance testing</h3><p>Numbers of cells of each cell subtype were quantified in each participant and control sample (donors) to compute a matrix of cell type × donor counts. Cell type abundance counts were modeled as a function of either disease severity (as an ordinal variable: healthy &lt; asymptomatic &lt; mild &lt; moderate &lt; severe &lt; critical) or days from symptom onset, adjusting for age, gender, batch and days from onset, in a negative binomial generalized linear model (NB GLM), implemented in the Bioconductor package edgeR. Counts were normalized in the model using the (log) of the total numbers of all cells captured for each donor. Hypothesis testing was performed using a quasi-likelihood <i>F</i>-test for either a linear or a quadratic trend across disease severity groups (asymptomatic &gt; mild &gt; moderate &gt; severe &gt; critical), or comparing healthy controls to SARS-CoV-2-infected donors (healthy versus all asymptomatic, mild, moderate, severe and critical groups). Differentially abundant cell types were determined using a 10% FDR. Due to compositional differences across sites, when analyzing differential abundance of myeloid populations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2</a>), only samples from Newcastle and London were included.</p><h3 class="c-article__sub-heading" id="Sec33">Relative importance of metadata on cell type composition</h3><p>The number of cells for each sample (<i>N</i> = 110 samples in total with complete metadata) and cell type (18 different cell types in total) combination was modeled with a generalized linear mixed model with a Poisson outcome. The five clinical factors (COVID-19 swab result, age, gender, disease severity at day 0 and days from onset) and the two technical factors (patient and sequencing center) were fitted as random effects to overcome the collinearity among the factors. The effect of each clinical/technical factor on cell type composition was estimated by the interaction term with the cell type. The likelihood ratio test was performed to assess the statistical significance of each factor on cell type abundance by removing one interaction term from the full model at a time. The number of factors was used to adjust multiple testing with the Bonferroni approach. The ‘glmer’ function in the lme4 package implemented on R was used to fit the model. The standard error of variance parameter for each factor was estimated using the ‘numDeriv’ package.</p><h3 class="c-article__sub-heading" id="Sec34">Cydar analysis</h3><p>We utilized cydar to identify changes in cell composition across the different severity groups based on the protein data alone. First, the background-corrected protein counts from the three different sites were integrated using the ‘fastMNN’ method (k = 20, d = 50, cos.norm = TRUE) in batchelor (v1.6.2). The batch-corrected counts for 188 proteins (four rat/mouse antibody isotypes were removed) were then used to construct hyperspheres using the ‘countCells’ function (downsample = 7) with the tolerance parameter chosen so that each hypersphere had at least 20 cells, estimated using the ‘neighborDistances’ function. To assess whether the abundance of cells in each hypersphere were associated with disease status, hypersphere counts were analyzed using the quasi-likelihood method in edgeR (v3.32.1). After filtering out hyperspheres with an average count per sample below five, we fitted a mean-dependent trend to the NB dispersion estimates. The trended dispersion for each hypersphere was used to fit an NB GLM using the log-transformed total number of cells as the offset for each sample and blocking for gender, age and batch. The quasi-likelihood <i>F</i>-test was used to compute <i>P</i> values for each hypersphere, which were corrected for multiple testing using the spatial FDR method in cydar.</p><h3 class="c-article__sub-heading" id="Sec35">Comparisons of PBMC annotation using the Azimuth tool</h3><p>The final annotation of PBMCs was compared to a published PBMC annotation using the Azimuth tool (<a href="http://azimuth.satijalab.org/app/azimuth">http://azimuth.satijalab.org/app/azimuth/</a>). Because of size restrictions of 100,000 cells, our data were subsampled to 10% of the total cells. After running the algorithm, results with a prediction score &lt; 0.5 were removed (5.8% of total removed). For each cluster in the COVID-19 PBMC data, the percentage of cells mapped to each cluster in the Azimuth annotation was calculated.</p><h3 class="c-article__sub-heading" id="Sec36">Interferon, TNF and JAK–STAT response scoring</h3><p>A list of genes related to response to type I interferons was obtained from the GSEA Molecular Signatures Database (MSigDB; <a href="http://amigo.geneontology.org/amigo/term/GO:0034340">GO:0034340</a>). Enrichment of the interferon score was measured using the ‘tl.score_genes’ tool in ‘scanpy’, which subtracts the average expression of all genes in the dataset from the average expression of the genes in this list. The scores were averaged across clusters and clinical status and expressed as a fold change over the interferon score in the equivalent healthy cluster.</p><h3 class="c-article__sub-heading" id="Sec37">kBET analysis</h3><p>The kBET<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Büttner, M., Miao, Z., Wolf, F. A., Teichmann, S. A. &amp; Theis, F. J. A test metric for assessing single-cell RNA-seq batch correction. Nat. Methods 16, 43–49 (2019)." href="/articles/s41591-021-01329-2#ref-CR52" id="ref-link-section-d73692734e5149">52</a></sup> algorithm (<a href="https://github.com/theislab/kBET">https://github.com/theislab/kBET/</a>) was run for each cluster (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1</a>) using the UMAP coordinates generated from Harmony-adjusted PCs and the sample number as the batch factor. The same procedure was then performed with the same annotation but using the UMAP coordinates generated from non-Harmony-adjusted PCs. The resultant rejection rates were averaged across clusters and compared using a Wilcoxon paired signed-rank test.</p><h3 class="c-article__sub-heading" id="Sec38">Bronchoalveolar lavage data analysis</h3><p>scRNA-seq data from BAL was obtained from the Gene Expression Omnibus (accession no. <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE145926">GSE145926</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Liao, M. et al. Single-cell landscape of bronchoalveolar immune cells in patients with COVID-19. Nat. Med. 26, 842–844 (2020)." href="/articles/s41591-021-01329-2#ref-CR22" id="ref-link-section-d73692734e5178">22</a></sup>. Raw data were analyzed using the same pipeline as for PBMC data, specifically using the same QC cutoffs (minimum of 200 genes and &lt;10% mitochondrial reads per cell), and data were batch corrected using Harmony by donor ID. To gain greater resolution of MPs, the DCs and macrophages were analyzed with further rounds of subclustering to identify DC1, DC2 and mature DC subsets.</p><h3 class="c-article__sub-heading" id="Sec39">PAGA analysis of blood monocytes and BAL macrophages</h3><p>Annotated raw expression datasets of BAL macrophages and COVID-19 PBMCs were merged and data log normalized and scaled as for the original datasets. The top 3,000 HVGs were chosen using the Seurat ‘vst’ method and used for downstream analysis. PCs were batch corrected by donor and used to build a neighborhood graph. The PAGA tool in scanpy (tl.paga) was used to generate the abstracted graph between clusters.</p><h3 class="c-article__sub-heading" id="Sec40">CellphoneDB</h3><p>CellphoneDB<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Vento-Tormo, R. et al. Single-cell reconstruction of the early maternal-fetal interface in humans. Nature 563, 347–353 (2018)." href="/articles/s41591-021-01329-2#ref-CR53" id="ref-link-section-d73692734e5199">53</a></sup> was used to assess putative interactions between monocytes (CD14_mono, CD83_CD14_mono, C1_CD16_mono, CD16_mono and Prolif_mono) and platelets. The tool was run for 100 iterations, and an expression threshold of 0.25 (limiting the analysis to genes expressed by 25% of cells). For downstream analysis, we focused on interactions between platelets and any monocyte subset.</p><h3 class="c-article__sub-heading" id="Sec41">Platelet activation</h3><p>Differentially expressed genes in platelets between healthy control and COVID-19 samples were filtered for those predicted to be involved in platelet activation (<a href="https://www.gsea-msigdb.org/gsea/msigdb/cards/REACTOME_PLATELET_ACTIVATION_SIGNALING_AND_AGGREGATION">https://www.gsea-msigdb.org/gsea/msigdb/cards/REACTOME_PLATELET_ACTIVATION_SIGNALING_AND_AGGREGATION</a>).</p><h3 class="c-article__sub-heading" id="Sec42">HSPC commitment scoring</h3><p>HSPCs were subsetted from the data and Leiden clusters generated using the same pipeline and parameters as for the whole PBMC dataset. Differentially expressed genes between the HSPC clusters that showed evidence of lineage commitment (megakaryocyte, erythroid and myeloid) were calculated using the FindAllMarkers tool in Seurat (with thresholds of genes expressed by 25% of cells and with a log fold change of 0.25), and genes with an adjusted <i>P</i>-value cutoff of 0.05 were used to generate gene signatures for each. Enrichment scores of these signatures in the CD38-negative and CD38-positive HSPC clusters were calculated using the tl.score_genes tool in scanpy (v1.6.0). The average expression of these enrichment scores in the CD38-negative and CD38-positive HSPC clusters was averaged by donor then compared across clinical states. Differences between groups were assessed using ANOVA with pairwise comparisons using Tukey’s test.</p><h3 class="c-article__sub-heading" id="Sec43">Multiplex cytokine analysis</h3><p>Serum was obtained from peripheral blood in red-topped serum Vacutainers (BD, 367815) and allowed to clot for at least 30 min before centrifugation (800<i>g</i> for 10 min) to separate the serum. After collection, serum was frozen at −80 °C and thawed on ice on the day of experiment. The assay was carried out using the Cytokine/Chemokine/Growth Factor 45-Plex Human ProcartaPlex Panel 1 kit (Invitrogen, EPX450-12171-901), utilizing the Luminex xMAP technology and according to the manufacturer’s protocol. Each sample was run in duplicate. The values of each analyte were detected using the MAGPIX system and analyzed using the ProcartaPlex Analyst v1.0 Software (Thermo Fisher Scientific).</p><h3 class="c-article__sub-heading" id="Sec44">Restimulation of PBMCs with SARS-Cov-2 peptide S</h3><p>Purified PMBCs were thawed at 37 °C, transferred into a 15-ml tube with 10 ml prewarmed complete culture media RPMI 1640 medium (Sigma-Aldrich, R0883) supplemented with 10% (vol/vol) FCS (Gibco, 10270-106) and 1% (vol/vol) penicillin–streptomycin (100 U ml<sup>−1</sup> and 100 μg ml<sup>−1</sup>, respectively; Sigma-Aldrich, P0781) and 1% (vol/vol) <span class="u-small-caps">l</span>-glutamine (2 mM; Sigma-Aldrich, G7513), referred to as RPMI 10, followed by centrifugation at 500<i>g</i> for 5 min. Cell pellet was resuspended in 500 μl RPMI 10 with added DNase (1 μg ml<sup>−1</sup>; Merck, 10104159001), divided into five wells of a round-bottom 96-well plate and left to rest at 37 °C for 1 h. Cells were stimulated with SARS-CoV-2 PepTivator peptide S for pan-HLA (2 μg ml<sup>−1</sup>; Miltenyi Biotec, 136-126-700) and PMA/ionomycin as a control (2 μl ml<sup>−1</sup> Cell Activation cocktail; BioLegend, 423301), and incubated at 37 °C for 2 h. Negative controls were left untreated. Brefeldin A (2 μg ml<sup>−1</sup>; GolgiPlug, BD Bioscience, 555029) and anti-CD107a-BB700 (1:50 dilution; clone H4A3; BD Bioscience, 566558) was added for an additional 4 h into all conditions. Cells were stained for detection of activation-induced markers and intracellular cytokines 6 h after stimulation and subjected to flow cytometry.</p><h3 class="c-article__sub-heading" id="Sec45">Flow cytometry of stimulated cells</h3><p>PBMCs stimulated for 6 h with the SARS-CoV-2 peptide were washed with PBS, and cell-surface stained for 1 h at room temperature: anti-CD14-FITC (1:50 dilution; clone M5E2; BD Biosciences, 555397), anti-CD19-FITC (1:50 dilution; clone 4G7; BD Biosciences, 345776), anti-CD137-PE-Dazzle594 (1:50 dilution; clone 4B4-1; BioLegend, 309826), anti-CCR7-PE-Cy7 (1:50 dilution; clone G043H7; BioLegend, 353226), anti-CD45RO-APC-H7 (1:50 dilution; clone UCHL1; BD Biosciences, 561137), anti-CD28-BV480 (1:50 dilution; clone CD28.2; BD Biosciences, 566110), anti-CD4-BV785 (1:100 dilution; clone SK3; BioLegend, 344642), anti-CD3-BUV395 (1:50 dilution; clone UCHT1; BD Biosciences, 563546), anti-CD8-BUV496 (1:100 dilution; clone RPA-T8; BD Biosciences, 564804), anti-CD25-BUV737 (1:100 dilution; clone 2A3; BD Biosciences, 612806) and viability dye Zombie Yellow (1:200 dilution; BioLegend, 423104). Cells were washed with PBS 2% (vol/vol) FCS, fixed with 4% (wt/vol) paraformaldehyde (Thermo Fisher Scientific, 28908) and kept at 4 °C overnight. Subsequently, cells were washed with PBS, permeabilized with Perm/Wash buffer (BD Biosciences, 554723) according to the manufacturer’s instructions, and stained with intracellular antibodies for 1 h on ice: anti-IL10-PE (1:10 dilution; clone JES3-19F1; BD Biosciences, 559330), anti-IFN-APC (1:25 dilution; Miltenyi Biotec, 130-090-762), anti-TNF-AF700 (1:50 dilution; clone MAb11; BioLegend, 502928), anti-IL-2-BV421 (1:100 dilution; clone 5344.111; BD Biosciences, 562914) and anti-CD154-BV605 (1:50 dilution; clone 24-31; BioLegend, 310826). Cells were washed, transferred to flow cytometry 5-ml tubes, and acquired on a Symphony A5 flow cytometer (BD Biosciences). Data were analyzed by FlowJo v10 (BD Biosciences).</p><h3 class="c-article__sub-heading" id="Sec46">GSEA analysis</h3><p>Preranked gene-set analysis on MSigDB v7.2 Hallmark gene sets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Liberzon, A. et al. The Molecular Signatures Database (MSigDB) hallmark gene set collection. Cell Syst. 1, 417–425 (2015)." href="/articles/s41591-021-01329-2#ref-CR54" id="ref-link-section-d73692734e5284">54</a></sup> was performed using preranked gene lists with the fgsea<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Korotkevich, G., Sukhov, V. &amp; Sergushichev, A. Fast gene-set enrichment analysis. Preprint at BioRxiv 
                  https://doi.org/10.1101/060012
                  
                 (2019)." href="/articles/s41591-021-01329-2#ref-CR55" id="ref-link-section-d73692734e5288">55</a></sup> package in R. Genes were preranked according to signed −log<sub>10</sub> <i>P</i> values for all preranked gene-set analysis procedures. For B cells, generation of the rank gene list was performed using a Wilcoxon rank-sum test (via ‘tl.rank_genes_groups’ in scanpy) with each day 0 COVID-19 statuses (asymptomatic to symptomatic critical) as the ‘tests’ versus day 0 healthy samples as the ‘reference/controls’.</p><h3 class="c-article__sub-heading" id="Sec47">T cell clustering, annotation and visualization</h3><p>Droplets labeled as T cells (CD4, CD8, T<sub>reg</sub>, MAIT and γδ T) were subset from those in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1b</a> and reclustered using a set of HVGs calculated within each batch, the union of which was used to estimate the first 50 PCs across cells using the ‘irbla’ R package (v2.3.3). Batch effects were removed across the first 30 PCs using the fastMNN<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Haghverdi, L., Lun, A. T. L., Morgan, M. D. &amp; Marioni, J. C. Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors. Nat. Biotechnol. 36, 421–427 (2018)." href="/articles/s41591-021-01329-2#ref-CR56" id="ref-link-section-d73692734e5310">56</a></sup> implementation in the Bioconductor package batchelor (v1.6.2; <i>k</i> = 50). A <i>k</i>-nearest-neighbor graph (<i>k</i> = 20) was computed across these 30 batch-integrated PCs using the ‘buildKNNGraph’ function implemented in the Bioconductor package scran (v1.18.3), which was then used to group cells into connected communities using Louvain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Blondel, V. D., Guillaume, J.-L., Lambiotte, R. &amp; Lefebvre, E. Fast unfolding of communities in large networks. J. Stat. Mech. 2008, P10008 (2008)." href="/articles/s41591-021-01329-2#ref-CR57" id="ref-link-section-d73692734e5324">57</a></sup> clustering implemented in the R package ‘igraph’ (v1.2.6). Clusters that displayed mixed profiles of T and other lymphoid lineages, that is, CD19, CD20 and immunoglobulin genes, were classed as doublets and removed from downstream analyses. Clusters indicative of NK cells (CD3<sup>−</sup>CD56<sup>+</sup>) were subsequently annotated as such and removed from T cell analyses. Remaining clusters were annotated using a combination of canonical protein and mRNA markers for major αβ T cells (CD4, CD8, CCR7, CD45RA, CD45RO, CD62L, CD27, CD38, CD44, CXCR5, CD40LG, <i>CCR7</i>, <i>FOXP3</i> and <i>IKZF2</i>), γδ T cells (Vγ9, Vγ2, <i>TRGV9</i> and<i>TRDV2</i>) and invariant T cells, MAIT (Vα24-Jα18 and <i>TRAV1.2</i>) and NK T (CD3, CD16, CD56, <i>NCAM1</i>, <i>NCR1</i> and <i>FCGR3A</i>) cells. Polarized CD4<sup>+</sup> T cell annotations were refined using the combination of transcription factor genes and expressed cytokines for the respective T<sub>H</sub> cell types: T<sub>H</sub>1 (<i>IFNG</i>, <i>TBX21</i> and <i>TNFA</i>), T<sub>H</sub>2 (<i>GATA3</i>, <i>IL4</i> and <i>IL5</i>) and T<sub>H</sub>17 (<i>RORC</i>, <i>IL17A</i>, <i>IL17F</i> and <i>IL21</i>). Where clusters appeared heterogeneous in their expression of T cell lineage markers, single-cell annotations were refined based on the coexpression of specific marker gene and protein pairs. Dot plots to visualize marker protein and mRNA expression across clusters were generated using the R package ‘ggplot2’ (v3.3.3). UMAP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="McInnes, L., Healy, J. &amp; Melville, J. UMAP: uniform manifold approximation and projection for dimension reduction. Preprint at
                   https://arxiv.org/abs/1802.03426
                  
                 (2018)." href="/articles/s41591-021-01329-2#ref-CR58" id="ref-link-section-d73692734e5403">58</a></sup> was used to project all single T cells into a two-dimensional space (<i>k</i> = 31) using the first 30 batch-integrated PCs as input with the R package ‘umap’. R v4.0.3 and Bioconductor v3.12 were used for all analyses.</p><h3 class="c-article__sub-heading" id="Sec48">T cell differential gene expression analysis</h3><p>Differential gene expression analysis was performed across COVID-19 disease severity groups, ordered from healthy &gt; asymptomatic &gt; mild &gt; moderate &gt; severe &gt; critical. Donor pseudo-bulk samples were first created by aggregating gene counts for each annotated T cell type, within each donor, where there were at least 20 cells of that type. Genes with fewer than three counts in any given pseudo-bulk sample, or fewer than five counts in total across donor pseudo-bulk samples, were removed before analysis. Differential gene expression testing was performed using an NB GLM implemented in the Bioconductor package edgeR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. Bioinformatics 26, 139–140 (2010)." href="/articles/s41591-021-01329-2#ref-CR59" id="ref-link-section-d73692734e5418">59</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="McCarthy, D. J., Chen, Y. &amp; Smyth, G. K. Differential expression analysis of multifactor RNA-seq experiments with respect to biological variation. Nucleic Acids Res. 40, 4288–4297 (2012)." href="/articles/s41591-021-01329-2#ref-CR60" id="ref-link-section-d73692734e5421">60</a></sup> (v3.32.1). Statistically significant differentially expressed genes were defined with FDR &lt; 0.1. Functional annotation enrichment was performed using the Bioconductor package enrichR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Kuleshov, M. V. et al. Enrichr: a comprehensive gene-set enrichment analysis web server 2016 update. Nucleic Acids Res. 44, W90–W97 (2016)." href="/articles/s41591-021-01329-2#ref-CR61" id="ref-link-section-d73692734e5425">61</a></sup>(v3.0). Upregulated and downregulated differentially expressed genes in each T cell type were used as input, testing separately against the MSigDB Hallmark 2020 and Transcription Factor Protein–Protein Interactions gene sets. Significant enrichments were defined with 1% FDR.</p><h3 class="c-article__sub-heading" id="Sec49">T cell receptor analysis</h3><p>Single-cell TCRs were computed from the TCR-sequencing data using Cell Ranger v4.0.0. The unfiltered outputs of reconstructed TCR contigs across all three sites (Newcastle, Cambridge and London) were combined before filtering using: (1) full-length CDR3, (2) each cell droplet barcode matched a TCR droplet barcode and (3) productive CDR3 spanning V + J genes. Chain-specific TCR clones were defined for each observed α-chain and β-chain by first concatenating the V, J and identical CDR3 nucleotide sequences. For each single T cell, these chains were then combined to form a single clonotype, removing cells that contained: (1) greater than two β-chains and greater than two α-chains and (2) a single α-chain or a single β-chain only. T cells with exactly two β-chains and one α-chain, or those with exactly two α-chains and one β-chain were retained. TCR clonotypes were counted within each donor sample, and expanded clones were defined where more than one cell was assigned to the TCR clonotype.</p><p>The proportion of expanded clones as a function of a linear trend across disease severity groups was modeled using logistic regression, adjusted for age, gender and batch. A separate model was run for each T cell subtype that contained at least five cells assigned to the expanded TCR clonotypes. Linear trend <i>P</i> values were corrected for multiple testing using the Benjamini–Hochberg procedure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. Ser. B Stat. Methodol. 57, 289–300 (1995)." href="/articles/s41591-021-01329-2#ref-CR62" id="ref-link-section-d73692734e5443">62</a></sup>.</p><p>The TE:EM ratio was calculated within each donor, using the number of observed expanded clonotypes. The TE:EM ratio change across COVID-19 severity was tested using a robust linear model implemented in the R package ‘robustbase’ (v0.93-7), regressing the TE:EM ratio on either disease severity as an ordered linear variable (asymptomatic &gt; mild &gt; moderate &gt; severe &gt; critical) or symptom duration, adjusted for age, gender and batch. Statistical significance was defined based on the linear trend across disease severity (<i>P</i> ≤ 0.01). An equivalent analysis was performed, restricted to participants with a shorter symptom duration (≤24 d).</p><h3 class="c-article__sub-heading" id="Sec50">Differential correlation analysis</h3><p>Changes in the correlations between PBMC cell types were computed using a differential correlation analysis, implemented in the R package <i>DCARS</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Ghazanfar, S., Strbenac, D., Ormerod, J. T., Yang, J. Y. H. &amp; Patrick, E. DCARS: differential correlation across ranked samples. Bioinformatics 35, 823–829 (2019)." href="/articles/s41591-021-01329-2#ref-CR63" id="ref-link-section-d73692734e5463">63</a></sup> (v0.3.5). Cell type proportions were computed by normalizing the counts of each cell type within each donor by the total number of cells captured for that donor sample. Donor samples were ranked according to their disease severity (healthy &gt; asymptomatic &gt; mild &gt; moderate &gt; severe &gt; critical). Differential correlation analysis was then performed between CD4.T<sub>FH</sub> versus all B cell types. Statistically significant differentially correlated cell types were defined with empirical <i>P</i> value ≤ 0.1, estimated from 10,000 permutations.</p><h3 class="c-article__sub-heading" id="Sec51">BCR V(D)J analysis</h3><p>Single-cell V(D)J data from the 5′ Chromium 10x kit were initially processed with Cell Ranger vdj pipeline (4.0.0). BCR contigs contained in ‘filtered_contigs.fasta’ and ‘filtered_contig_annotations.csv’ from all three sites were then preprocessed using ‘immcantation’ inspired preprocessing pipeline<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Gupta, N. T. et al. Change-O: a toolkit for analyzing large-scale B cell immunoglobulin repertoire sequencing data. Bioinformatics 31, 3356–3358 (2015)." href="/articles/s41591-021-01329-2#ref-CR64" id="ref-link-section-d73692734e5481">64</a></sup> implemented in the dandelion Python package; dandelion is a new single-cell BCR-seq analysis package for 10x Chromium 5′ data. All steps outlined below were performed using dandelion v0.0.27.post2 and are available at <a href="https://github.com/clatworthylab/dandelion">https://github.com/clatworthylab/dandelion/</a>.</p><h3 class="c-article__sub-heading" id="Sec52">BCR preprocessing</h3><p>Individual BCR contigs were reannotated with ‘igblastn’ v1.1.15 using the IMGT reference database (downloaded on 30 June 2020)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Lefranc, M.-P. IMGT, the International ImMunoGeneTics Information System. Novartis Found. Symp. 254, 126–142 
                  http://imgt.cines.fr
                  
                 (2008)." href="/articles/s41591-021-01329-2#ref-CR65" id="ref-link-section-d73692734e5500">65</a></sup> by calling changeo’s ‘AssignGenes.py’ script, and reannotated contigs in ‘blast’ format were parsed into the Adaptive Immune Receptor Repertoire standards 1.3 format with changeo’s ‘MakeDB.py’ script. Amino acid sequence alignment information not present in the output from blast format was retrieved from reannotation with igblastn in AIRR format. Heavy-chain V-gene alleles were corrected for individual genotypes with TIgGER<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Gadala-Maria, D., Yaari, G., Uduman, M. &amp; Kleinstein, S. H. Automated analysis of high-throughput B cell sequencing data reveals a high frequency of novel immunoglobulin V gene segment alleles. Proc. Natl Acad. Sci. USA 112, E862–E870 (2015)." href="/articles/s41591-021-01329-2#ref-CR66" id="ref-link-section-d73692734e5504">66</a></sup> (v1.0.0) using a modified ‘tigger-genotype.R’ script from ‘immcantation’ suite. Germline sequences were reconstructed based on the genotype corrected V-gene assignments using changeo’s (v1.0.1) ‘CreateGermines.py’ script; contigs which failed germline sequence reconstruction were removed from further analysis. Constant genes were reannotated using blastn (v2.10.0+) with CH1 regions of constant gene sequences from IMGT followed by pairwise alignment against curated sequences to correct assignment errors due to insufficient length of constant regions.</p><h3 class="c-article__sub-heading" id="Sec53">BCR filtering</h3><p>Contigs assigned to cells that passed QC on the transcriptome data were retained for further QC assessment, which included the following checks: (1) contigs with mismatched locus and V, J and constant gene assignments were removed from the analysis; (2) cell barcodes with multiple heavy-chain contigs were flagged for filtering (exceptions to this were (a) when the multiple heavy-chain contigs were assessed to have identical V(D)J sequences but assigned as different contigs belonging to the same cell by Cell Ranger vdj pipeline, (b) when there was a clear dominance (assessed by difference in UMI count) by a particular contig, and (c) if and when there was presence of one IgM and one IgD contig assigned to a single cell barcode; in the first two cases, the contig with the highest UMI count was retained); (3) cell barcodes with multiple light-chain contigs were flagged for filtering; and (4) in situations where cell barcodes were matched with only light-chain contigs, the contigs were dropped from the V(D)J data but the transcriptome barcode was retained.</p><h3 class="c-article__sub-heading" id="Sec54">B cell clone/clonotype definition</h3><p>BCRs were grouped into clones/clonotypes based on the following sequential criteria that apply to both heavy-chain and light-chain contigs: (1) identical V and J gene usage, (2) identical junctional CDR3 amino acid length, and (3) at least 85% amino acid sequence similarity at the CDR3 junction (based on hamming distance). Light-chain pairing was performed using the same criteria within each heavy-chain clone. Only samples collected at day 0 of the study were analyzed from this step onwards and clones/clonotypes were called across the entire dataset; the sample from one of the donors who was subsequently found to have a B cell malignancy was separated from the analysis and processed independently.</p><h3 class="c-article__sub-heading" id="Sec55">B cell clone/clonotype network</h3><p>Single-cell BCR networks were constructed using adjacency matrices computed from pairwise Levenshtein distance of the full amino acid sequence alignment for BCR(s) contained in every pair of cells within each disease severity cohort. Construction of the Levenshtein distance matrices were performed separately for heavy-chain and light-chain contigs, and the sum of the total edit distance across all layers/matrices was used as the final adjacency matrix. To construct the BCR neighborhood graph, a minimum-spanning tree was constructed on the adjacency matrix for each clone/clonotype, creating a simple graph with edges indicating the shortest edit distance between a B cell and its nearest neighbor. Cells with identical BCRs, that is, cells with a total pairwise edit distance of zero, were then connected to the graph to recover edges trimmed off during the minimum-spanning-tree construction step. Fruchterman–Reingold graph layout was generated using a modified method to prevent singletons from flying out to infinity in ‘networkx’ (v2.5). Visualization of the resulting single-cell BCR network was achieved via transfer of the graph to relevant ‘anndata’ slots, allowing for access to plotting tools in scanpy.</p><p>The use of the BCR network properties for computing gini indices was inspired from bulk BCR-sequencing network analysis methods where distributions of clone sizes and vertex sizes (sum of identical BCR reads) in BCR clone networks were used to infer the relationships between BCR clonality, somatic hypermutation and diversity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Bashford-Rogers, R. J. M. et al. Network properties derived from deep sequencing of human B cell receptor repertoires delineate B cell populations. Genome Res 23, 1874–1884 (2013)." href="/articles/s41591-021-01329-2#ref-CR67" id="ref-link-section-d73692734e5535">67</a></sup>. However, there are challenges with native implementation of this approach for single-cell data. Firstly, to enable calculation of network-based clone/cluster and vertex/node size distribution, BCR networks needed to be reduced such that nodes/cells with identical BCRs had to be merged and counted; this required the reconstruction of BCR networks per sample and discarding single-cell-level information. Furthermore, the process of node contraction and counting of merging events requires substantial computation time and resource. Secondly, this approach is dependent on sufficient coverage of the BCR repertoire, as the BCRs from the number of cells sampled (after QC) may not necessarily recapitulate the entire repertoire, which may underrepresent or overrepresent merged counts for gini index calculation. We propose the use of node closeness centrality computed on each expanded clone (clone size &gt; 1) as an alternative metric to emulate the statistics to adapt to the single-cell nature of the data; closeness centrality defines how close and central each node is with respect to other nodes in the graph; therefore, cells with identical BCRs will have high closeness centrality scores, due to the way the BCR network is constructed in dandelion. Thus, we can quickly calculate if cells across clones, and/or samples overall, in the entire graph display proportionately/disproportionately high or low closeness centrality scores. One caveat to the current implementation is that it is only meaningful if there are clonotypes with at least two cells, as scores will only be computed for non-singleton components of the graph. Gini indices are computed using ‘skbio.diversity.alpha.gini_index’ (scikit-bio v0.5.6) with the ‘trapezoids’ method after clone definition and network generation. Summary visualization was performed using plotting tools in ‘seaborn’ (v0.11.0).</p><h3 class="c-article__sub-heading" id="Sec56">Definition of BCR convergence across participants</h3><p>BCR overlap was determined by collapsing sharing incidence of V and J gene usage and CDR3 amino acid sequences, in both heavy and light chains, between individuals into a binarized format (1 or 0). The information is turned into an adjacency matrix where an edge is created between two individuals if there is at least one clonotype (at least one cell from each individual displays an identical combination of heavy-chain and light-chain V and J gene usage with allowance for somatic hypermutation at the CDR3 junctional region) that is similar between the two individuals. Visualization was achieved using the ‘CircosPlot’ function from ‘nxviz’ package (v0.6.2).</p><h3 class="c-article__sub-heading" id="Sec57">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>The dataset from our study can be explored interactively through the web portal <a href="https://covid19cellatlas.org">https://covid19cellatlas.org/</a>. The data object, as a h5ad file, can also be downloaded from <a href="https://covid19cellatlas.org">https://covid19cellatlas.org/</a>. The processed data are available to download from Array Express under accession number <a href="http://www.ebi.ac.uk/arrayexpress/experiments/E-MTAB-10026/">E-MTAB-10026</a>. <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-021-01329-2#Sec61">Source data</a> are provided with this paper.</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>All data analysis scripts are available on <a href="https://github.com/scCOVID-19/COVIDPBMC/">https://github.com/scCOVID-19/COVIDPBMC/</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Bedford, J. et al. COVID-19: towards controlling of a pandemic. <i>Lancet</i> <b>395</b>, 1015–1018 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(20)30673-5" data-track-item_id="10.1016/S0140-6736(20)30673-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2820%2930673-5" aria-label="Article reference 1" data-doi="10.1016/S0140-6736(20)30673-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXltFChsbc%3D" aria-label="CAS reference 1">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32197103" aria-label="PubMed reference 1">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7270596" aria-label="PubMed Central reference 1">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=COVID-19%3A%20towards%20controlling%20of%20a%20pandemic&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2820%2930673-5&amp;volume=395&amp;pages=1015-1018&amp;publication_year=2020&amp;author=Bedford%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">John Hopkins University. COVID-19 map. Johns Hopkins Coronavirus Resource Center. <a href="https://coronavirus.jhu.edu/map.html" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://coronavirus.jhu.edu/map.html">https://coronavirus.jhu.edu/map.html</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Chen, N. et al. Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. <i>Lancet</i> <b>395</b>, 507–513 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(20)30211-7" data-track-item_id="10.1016/S0140-6736(20)30211-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2820%2930211-7" aria-label="Article reference 3" data-doi="10.1016/S0140-6736(20)30211-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFOmsb8%3D" aria-label="CAS reference 3">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32007143" aria-label="PubMed reference 3">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7135076" aria-label="PubMed Central reference 3">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Epidemiological%20and%20clinical%20characteristics%20of%2099%20cases%20of%202019%20novel%20coronavirus%20pneumonia%20in%20Wuhan%2C%20China%3A%20a%20descriptive%20study&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2820%2930211-7&amp;volume=395&amp;pages=507-513&amp;publication_year=2020&amp;author=Chen%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Hoffmann, M. et al. SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor. <i>Cell</i> <b>181</b>, 271–280 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXktl2qtb8%3D" aria-label="CAS reference 4">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32142651" aria-label="PubMed reference 4">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102627" aria-label="PubMed Central reference 4">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20cell%20entry%20depends%20on%20ACE2%20and%20TMPRSS2%20and%20is%20blocked%20by%20a%20clinically%20proven%20protease%20inhibitor&amp;journal=Cell&amp;volume=181&amp;pages=271-280&amp;publication_year=2020&amp;author=Hoffmann%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Diao, B. et al. Reduction and functional exhaustion of T cells in patients with coronavirus disease 2019 (COVID-19). <i>Front. Immunol.</i> <b>11</b>, 827 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVWnsb7L" aria-label="CAS reference 5">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32425950" aria-label="PubMed reference 5">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7205903" aria-label="PubMed Central reference 5">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Reduction%20and%20functional%20exhaustion%20of%20T%20cells%20in%20patients%20with%20coronavirus%20disease%202019%20%28COVID-19%29&amp;journal=Front.%20Immunol.&amp;volume=11&amp;publication_year=2020&amp;author=Diao%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Lee, J. S. et al. Immunophenotyping of COVID-19 and influenza highlights the role of type I interferons in development of severe COVID-19. <i>Sci. Immunol.</i> <b>5</b>, eabd1554 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Arunachalam, P. S. et al. Systems biological assessment of immunity to mild versus severe COVID-19 infection in humans. <i>Science</i> <b>369</b>, 1210–1220 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslCltrnJ" aria-label="CAS reference 7">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32788292" aria-label="PubMed reference 7">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7665312" aria-label="PubMed Central reference 7">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Systems%20biological%20assessment%20of%20immunity%20to%20mild%20versus%20severe%20COVID-19%20infection%20in%20humans&amp;journal=Science&amp;volume=369&amp;pages=1210-1220&amp;publication_year=2020&amp;author=Arunachalam%2CPS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Woodruff, M. C. et al. Extrafollicular B cell responses correlate with neutralizing antibodies and morbidity in COVID-19. <i>Nat. Immunol.</i> <b>21</b>, 1506–1516 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33028979" aria-label="PubMed reference 8">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7739702" aria-label="PubMed Central reference 8">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Extrafollicular%20B%20cell%20responses%20correlate%20with%20neutralizing%20antibodies%20and%20morbidity%20in%20COVID-19&amp;journal=Nat.%20Immunol.&amp;volume=21&amp;pages=1506-1516&amp;publication_year=2020&amp;author=Woodruff%2CMC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Chen, G. et al. Clinical and immunological features of severe and moderate coronavirus disease 2019. <i>J. Clin. Invest.</i> <b>130</b>, 2620–2629 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXpt12ks78%3D" aria-label="CAS reference 9">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32217835" aria-label="PubMed reference 9">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7190990" aria-label="PubMed Central reference 9">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20and%20immunological%20features%20of%20severe%20and%20moderate%20coronavirus%20disease%202019&amp;journal=J.%20Clin.%20Invest.&amp;volume=130&amp;pages=2620-2629&amp;publication_year=2020&amp;author=Chen%2CG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Zhang, J.-Y. et al. Single-cell landscape of immunological responses in patients with COVID-19. <i>Nat. Immunol.</i> <b>21</b>, 1107–1118 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsF2itb%2FE" aria-label="CAS reference 10">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32788748" aria-label="PubMed reference 10">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20landscape%20of%20immunological%20responses%20in%20patients%20with%20COVID-19&amp;journal=Nat.%20Immunol.&amp;volume=21&amp;pages=1107-1118&amp;publication_year=2020&amp;author=Zhang%2CJ-Y">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Liu, C. et al. Time-resolved systems immunology reveals a late juncture linked to fatal COVID-19. <i>Cell</i> <a href="https://doi.org/10.1016/j.cell.2021.02.018" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.cell.2021.02.018">https://doi.org/10.1016/j.cell.2021.02.018</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Hadjadj, J. et al. Impaired type I interferon activity and inflammatory responses in severe COVID-19 patients. <i>Science</i> <b>369</b>, 718–724 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFGms7rP" aria-label="CAS reference 12">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32661059" aria-label="PubMed reference 12">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7402632" aria-label="PubMed Central reference 12">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Impaired%20type%20I%20interferon%20activity%20and%20inflammatory%20responses%20in%20severe%20COVID-19%20patients&amp;journal=Science&amp;volume=369&amp;pages=718-724&amp;publication_year=2020&amp;author=Hadjadj%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Schulte-Schrepping, J. et al. Severe COVID-19 is marked by a dysregulated myeloid cell compartment. <i>Cell</i> <b>182</b>, 1419–1440 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1WqsL7P" aria-label="CAS reference 13">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32810438" aria-label="PubMed reference 13">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7405822" aria-label="PubMed Central reference 13">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Severe%20COVID-19%20is%20marked%20by%20a%20dysregulated%20myeloid%20cell%20compartment&amp;journal=Cell&amp;volume=182&amp;pages=1419-1440&amp;publication_year=2020&amp;author=Schulte-Schrepping%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Mann, E. R. et al. Longitudinal immune profiling reveals key myeloid signatures associated with COVID-19. <i>Sci. Immunol.</i> <b>5</b>, eabd6197 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Bernardes, J. P. et al. Longitudinal multi-omics analyses identify responses of megakaryocytes, erythroid cells and plasmablasts as hallmarks of severe COVID-19. <i>Immunity</i> <b>53</b>, 1296–1314 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFeltL3P" aria-label="CAS reference 15">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33296687" aria-label="PubMed reference 15">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7689306" aria-label="PubMed Central reference 15">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Longitudinal%20multi-omics%20analyses%20identify%20responses%20of%20megakaryocytes%2C%20erythroid%20cells%20and%20plasmablasts%20as%20hallmarks%20of%20severe%20COVID-19&amp;journal=Immunity&amp;volume=53&amp;pages=1296-1314&amp;publication_year=2020&amp;author=Bernardes%2CJP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">World Health Organization. Clinical management of COVID-19. <a href="https://www.who.int/publications/i/item/clinical-management-of-covid-19" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.who.int/publications/i/item/clinical-management-of-covid-19">https://www.who.int/publications/i/item/clinical-management-of-covid-19</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. <i>Nat. Methods</i> <b>16</b>, 1289–1296 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOqsr7N" aria-label="CAS reference 17">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31740819" aria-label="PubMed reference 17">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6884693" aria-label="PubMed Central reference 17">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%2C%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony&amp;journal=Nat.%20Methods&amp;volume=16&amp;pages=1289-1296&amp;publication_year=2019&amp;author=Korsunsky%2CI">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Severe COVID-19 GWAS Groupet al. Genome-wide association study of severe COVID-19 with respiratory failure. <i>N. Engl. J. Med.</i> <b>383</b>, 1522–1534 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20association%20study%20of%20severe%20COVID-19%20with%20respiratory%20failure&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=383&amp;pages=1522-1534&amp;publication_year=2020">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Pairo-Castineira, E. et al. Genetic mechanisms of critical illness in COVID-19. <i>Nature</i> <a href="https://doi.org/10.1038/s41586-020-03065-y" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41586-020-03065-y">https://doi.org/10.1038/s41586-020-03065-y</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Lun, A. T. L., Richard, A. C. &amp; Marioni, J. C. Testing for differential abundance in mass cytometry data. <i>Nat. Methods</i> <b>14</b>, 707–709 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXnslKkt70%3D" aria-label="CAS reference 20">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28504682" aria-label="PubMed reference 20">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6155493" aria-label="PubMed Central reference 20">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Testing%20for%20differential%20abundance%20in%20mass%20cytometry%20data&amp;journal=Nat.%20Methods&amp;volume=14&amp;pages=707-709&amp;publication_year=2017&amp;author=Lun%2CATL&amp;author=Richard%2CAC&amp;author=Marioni%2CJC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Jardine, L. et al. Lipopolysaccharide inhalation recruits monocytes and dendritic cell subsets to the alveolar airspace. <i>Nat. Commun.</i> <b>10</b>, 1999 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31040289" aria-label="PubMed reference 21">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6491485" aria-label="PubMed Central reference 21">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Lipopolysaccharide%20inhalation%20recruits%20monocytes%20and%20dendritic%20cell%20subsets%20to%20the%20alveolar%20airspace&amp;journal=Nat.%20Commun.&amp;volume=10&amp;publication_year=2019&amp;author=Jardine%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Liao, M. et al. Single-cell landscape of bronchoalveolar immune cells in patients with COVID-19. <i>Nat. Med.</i> <b>26</b>, 842–844 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXptFyqtr4%3D" aria-label="CAS reference 22">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32398875" aria-label="PubMed reference 22">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20landscape%20of%20bronchoalveolar%20immune%20cells%20in%20patients%20with%20COVID-19&amp;journal=Nat.%20Med.&amp;volume=26&amp;pages=842-844&amp;publication_year=2020&amp;author=Liao%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Evren, E. et al. Distinct developmental pathways from blood monocytes generate human lung macrophage diversity. <i>Immunity</i> <a href="https://doi.org/10.1016/j.immuni.2020.12.003" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.immuni.2020.12.003">https://doi.org/10.1016/j.immuni.2020.12.003</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. <i>Cell</i> <a href="https://doi.org/10.1016/j.cell.2021.01.053" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.cell.2021.01.053">https://doi.org/10.1016/j.cell.2021.01.053</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Levi, M., Thachil, J., Iba, T. &amp; Levy, J. H. Coagulation abnormalities and thrombosis in patients with COVID-19. <i>Lancet Haematol.</i> <b>7</b>, e438–e440 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32407672" aria-label="PubMed reference 25">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7213964" aria-label="PubMed Central reference 25">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Coagulation%20abnormalities%20and%20thrombosis%20in%20patients%20with%20COVID-19&amp;journal=Lancet%20Haematol.&amp;volume=7&amp;pages=e438-e440&amp;publication_year=2020&amp;author=Levi%2CM&amp;author=Thachil%2CJ&amp;author=Iba%2CT&amp;author=Levy%2CJH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Hottz, E. D. et al. Platelet activation and platelet-monocyte aggregate formation trigger tissue factor expression in patients with severe COVID-19. <i>Blood</i> <b>136</b>, 1330–1341 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFygtrrF" aria-label="CAS reference 26">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32678428" aria-label="PubMed reference 26">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Platelet%20activation%20and%20platelet-monocyte%20aggregate%20formation%20trigger%20tissue%20factor%20expression%20in%20patients%20with%20severe%20COVID-19&amp;journal=Blood&amp;volume=136&amp;pages=1330-1341&amp;publication_year=2020&amp;author=Hottz%2CED">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Velten, L. et al. Human haematopoietic stem cell lineage commitment is a continuous process. <i>Nat. Cell Biol.</i> <b>19</b>, 271–281 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXltVWnsr0%3D" aria-label="CAS reference 27">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28319093" aria-label="PubMed reference 27">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5496982" aria-label="PubMed Central reference 27">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20haematopoietic%20stem%20cell%20lineage%20commitment%20is%20a%20continuous%20process&amp;journal=Nat.%20Cell%20Biol.&amp;volume=19&amp;pages=271-281&amp;publication_year=2017&amp;author=Velten%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Mende, N. et al. Quantitative and molecular differences distinguish adult human medullary and extramedullary haematopoietic stem and progenitor cell landscapes. Preprint at <i>BioRxiv</i> <a href="https://doi.org/10.1101/2020.01.26.919753" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2020.01.26.919753">https://doi.org/10.1101/2020.01.26.919753</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Ivanov, S. et al. Interleukin-22 reduces lung inflammation during influenza A virus infection and protects against secondary bacterial infection. <i>J. Virol.</i> <b>87</b>, 6911–6924 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXptlKku7c%3D" aria-label="CAS reference 29">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23596287" aria-label="PubMed reference 29">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3676141" aria-label="PubMed Central reference 29">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-22%20reduces%20lung%20inflammation%20during%20influenza%20A%20virus%20infection%20and%20protects%20against%20secondary%20bacterial%20infection&amp;journal=J.%20Virol.&amp;volume=87&amp;pages=6911-6924&amp;publication_year=2013&amp;author=Ivanov%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Desai, N. et al. Temporal and spatial heterogeneity of host response to SARS-CoV-2 pulmonary infection. <i>Nat. Commun.</i> <b>11</b>, 6319 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFelt7nF" aria-label="CAS reference 30">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33298930" aria-label="PubMed reference 30">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7725958" aria-label="PubMed Central reference 30">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Temporal%20and%20spatial%20heterogeneity%20of%20host%20response%20to%20SARS-CoV-2%20pulmonary%20infection&amp;journal=Nat.%20Commun.&amp;volume=11&amp;publication_year=2020&amp;author=Desai%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Joshi, N. S. et al. Inflammation directs memory precursor and short-lived effector CD8<sup>+</sup> T cell fates via the graded expression of T-bet transcription factor. <i>Immunity</i> <b>27</b>, 281–295 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSgtb3J" aria-label="CAS reference 31">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17723218" aria-label="PubMed reference 31">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2034442" aria-label="PubMed Central reference 31">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Inflammation%20directs%20memory%20precursor%20and%20short-lived%20effector%20CD8%2B%20T%20cell%20fates%20via%20the%20graded%20expression%20of%20T-bet%20transcription%20factor&amp;journal=Immunity&amp;volume=27&amp;pages=281-295&amp;publication_year=2007&amp;author=Joshi%2CNS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Monaco, G. et al. RNA-seq signatures normalized by mRNA abundance allow absolute deconvolution of human immune cell types. <i>Cell Rep.</i> <b>26</b>, 1627–1640 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXisFGhurY%3D" aria-label="CAS reference 32">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30726743" aria-label="PubMed reference 32">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6367568" aria-label="PubMed Central reference 32">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=RNA-seq%20signatures%20normalized%20by%20mRNA%20abundance%20allow%20absolute%20deconvolution%20of%20human%20immune%20cell%20types&amp;journal=Cell%20Rep.&amp;volume=26&amp;pages=1627-1640&amp;publication_year=2019&amp;author=Monaco%2CG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Adlowitz, D. G. et al. Expansion of activated peripheral blood memory B cells in rheumatoid arthritis, impact of B cell depletion therapy, and biomarkers of response. <i>PLoS ONE</i> <b>10</b>, e0128269 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26047509" aria-label="PubMed reference 33">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4457888" aria-label="PubMed Central reference 33">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Expansion%20of%20activated%20peripheral%20blood%20memory%20B%20cells%20in%20rheumatoid%20arthritis%2C%20impact%20of%20B%20cell%20depletion%20therapy%2C%20and%20biomarkers%20of%20response&amp;journal=PLoS%20ONE&amp;volume=10&amp;publication_year=2015&amp;author=Adlowitz%2CDG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Crotty, S. T. T follicular helper cell biology: a decade of discovery and diseases. <i>Immunity</i> <b>50</b>, 1132–1148 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVWjsrrE" aria-label="CAS reference 34">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31117010" aria-label="PubMed reference 34">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6532429" aria-label="PubMed Central reference 34">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=T%20follicular%20helper%20cell%20biology%3A%20a%20decade%20of%20discovery%20and%20diseases&amp;journal=Immunity&amp;volume=50&amp;pages=1132-1148&amp;publication_year=2019&amp;author=Crotty%2CST">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Meng, X. et al. Hypoxia-inducible factor-1α is a critical transcription factor for IL-10-producing B cells in autoimmune disease. <i>Nat. Commun.</i> <b>9</b>, 251 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29343683" aria-label="PubMed reference 35">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5772476" aria-label="PubMed Central reference 35">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Hypoxia-inducible%20factor-1%CE%B1%20is%20a%20critical%20transcription%20factor%20for%20IL-10-producing%20B%20cells%20in%20autoimmune%20disease&amp;journal=Nat.%20Commun.&amp;volume=9&amp;publication_year=2018&amp;author=Meng%2CX">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Pritchard, N. R. &amp; Smith, K. G. C. B cell inhibitory receptors and autoimmunity. <i>Immunology</i> <b>108</b>, 263–273 (2003).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3sXitlSks7Y%3D" aria-label="CAS reference 36">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12603592" aria-label="PubMed reference 36">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1782894" aria-label="PubMed Central reference 36">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=B%20cell%20inhibitory%20receptors%20and%20autoimmunity&amp;journal=Immunology&amp;volume=108&amp;pages=263-273&amp;publication_year=2003&amp;author=Pritchard%2CNR&amp;author=Smith%2CKGC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Doughty, C. A. et al. Antigen receptor-mediated changes in glucose metabolism in B lymphocytes: role of phosphatidylinositol 3-kinase signaling in the glycolytic control of growth. <i>Blood</i> <b>107</b>, 4458–4465 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28Xlt1Cgsbc%3D" aria-label="CAS reference 37">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16449529" aria-label="PubMed reference 37">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1895797" aria-label="PubMed Central reference 37">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Antigen%20receptor-mediated%20changes%20in%20glucose%20metabolism%20in%20B%20lymphocytes%3A%20role%20of%20phosphatidylinositol%203-kinase%20signaling%20in%20the%20glycolytic%20control%20of%20growth&amp;journal=Blood&amp;volume=107&amp;pages=4458-4465&amp;publication_year=2006&amp;author=Doughty%2CCA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Boothby, M. &amp; Rickert, R. C. Metabolic regulation of the immune humoral response. <i>Immunity</i> <b>46</b>, 743–755 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXnvFKjtrs%3D" aria-label="CAS reference 38">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28514675" aria-label="PubMed reference 38">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640164" aria-label="PubMed Central reference 38">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Metabolic%20regulation%20of%20the%20immune%20humoral%20response&amp;journal=Immunity&amp;volume=46&amp;pages=743-755&amp;publication_year=2017&amp;author=Boothby%2CM&amp;author=Rickert%2CRC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Guan, W.-J. et al. Clinical characteristics of coronavirus disease 2019 in China. <i>N. Engl. J. Med.</i> <b>382</b>, 1708–1720 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXovVSjsb8%3D" aria-label="CAS reference 39">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32109013" aria-label="PubMed reference 39">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20characteristics%20of%20coronavirus%20disease%202019%20in%20China&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=382&amp;pages=1708-1720&amp;publication_year=2020&amp;author=Guan%2CW-J">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Onder, G., Rezza, G. &amp; Brusaferro, S. Case-fatality rate and characteristics of patients dying in relation to COVID-19 in Italy. <i>J. Am. Med. Assoc.</i> <b>323</b>, 1775–1776 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXps1Srurw%3D" aria-label="CAS reference 40">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Case-fatality%20rate%20and%20characteristics%20of%20patients%20dying%20in%20relation%20to%20COVID-19%20in%20Italy&amp;journal=J.%20Am.%20Med.%20Assoc.&amp;volume=323&amp;pages=1775-1776&amp;publication_year=2020&amp;author=Onder%2CG&amp;author=Rezza%2CG&amp;author=Brusaferro%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Parrot, T. et al. MAIT cell activation and dynamics associated with COVID-19 disease severity. <i>Sci. Immunol.</i> <b>5</b>, eabe1670 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Le Bert, N. et al. Highly functional virus-specific cellular immune response in asymptomatic SARS-CoV-2 infection. <i>J. Exp. Med</i>. <b>218</b>, e20202617 (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Rydyznski Moderbacher, C. et al. Antigen-specific adaptive immunity to SARS-CoV-2 in acute COVID-19 and associations with age and disease severity. <i>Cell</i> <b>183</b>, 996–1012 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvF2jsLnL" aria-label="CAS reference 43">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33010815" aria-label="PubMed reference 43">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7494270" aria-label="PubMed Central reference 43">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Antigen-specific%20adaptive%20immunity%20to%20SARS-CoV-2%20in%20acute%20COVID-19%20and%20associations%20with%20age%20and%20disease%20severity&amp;journal=Cell&amp;volume=183&amp;pages=996-1012&amp;publication_year=2020&amp;author=Rydyznski%20Moderbacher%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Kaneko, N. et al. Loss of Bcl-6-expressing T follicular helper cells and germinal centers in COVID-19. <i>Cell</i> <b>183</b>, 143–157 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslygsLvO" aria-label="CAS reference 44">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32877699" aria-label="PubMed reference 44">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7437499" aria-label="PubMed Central reference 44">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Loss%20of%20Bcl-6-expressing%20T%20follicular%20helper%20cells%20and%20germinal%20centers%20in%20COVID-19&amp;journal=Cell&amp;volume=183&amp;pages=143-157&amp;publication_year=2020&amp;author=Kaneko%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Maloy, K. J. et al. CD4<sup>+</sup> T cell subsets during virus infection. Protective capacity depends on effector cytokine secretion and on migratory capability. <i>J. Exp. Med.</i> <b>191</b>, 2159–2170 (2000).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3cXkt1Wrurk%3D" aria-label="CAS reference 45">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10859340" aria-label="PubMed reference 45">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2193195" aria-label="PubMed Central reference 45">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=CD4%2B%20T%20cell%20subsets%20during%20virus%20infection.%20Protective%20capacity%20depends%20on%20effector%20cytokine%20secretion%20and%20on%20migratory%20capability&amp;journal=J.%20Exp.%20Med.&amp;volume=191&amp;pages=2159-2170&amp;publication_year=2000&amp;author=Maloy%2CKJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Sahin, U. et al. COVID-19 vaccine BNT162b1 elicits human antibody and T<sub>H</sub>1 T cell responses. <i>Nature</i> <b>586</b>, 594–599 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitV2qs7fI" aria-label="CAS reference 46">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32998157" aria-label="PubMed reference 46">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=COVID-19%20vaccine%20BNT162b1%20elicits%20human%20antibody%20and%20TH1%20T%20cell%20responses&amp;journal=Nature&amp;volume=586&amp;pages=594-599&amp;publication_year=2020&amp;author=Sahin%2CU">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Bastard, P. et al. Autoantibodies against type I IFNs in patients with life-threatening COVID-19. <i>Science</i> <b>370</b>, eabd4585 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Takahashi, T. et al. Sex differences in immune responses that underlie COVID-19 disease outcomes. <i>Nature</i> <b>588</b>, 315–320 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVWjsrzL" aria-label="CAS reference 48">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32846427" aria-label="PubMed reference 48">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7725931" aria-label="PubMed Central reference 48">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Sex%20differences%20in%20immune%20responses%20that%20underlie%20COVID-19%20disease%20outcomes&amp;journal=Nature&amp;volume=588&amp;pages=315-320&amp;publication_year=2020&amp;author=Takahashi%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. <i>Bioinformatics</i> <b>29</b>, 15–21 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhvV2gsbnF" aria-label="CAS reference 49">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23104886" aria-label="PubMed reference 49">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&amp;journal=Bioinformatics&amp;volume=29&amp;pages=15-21&amp;publication_year=2013&amp;author=Dobin%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Heaton, H. et al. Souporcell: robust clustering of single-cell RNA-seq data by genotype without reference genotypes. <i>Nat. Methods</i> <b>17</b>, 615–620 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXosV2jsLw%3D" aria-label="CAS reference 50">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32366989" aria-label="PubMed reference 50">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Souporcell%3A%20robust%20clustering%20of%20single-cell%20RNA-seq%20data%20by%20genotype%20without%20reference%20genotypes&amp;journal=Nat.%20Methods&amp;volume=17&amp;pages=615-620&amp;publication_year=2020&amp;author=Heaton%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">McGinnis, C. S., Murrow, L. M. &amp; Gartner, Z. J. DoubletFinder: doublet detection in single-cell RNA-sequencing data using artificial nearest neighbors. <i>Cell Syst.</i> <b>8</b>, 329–337 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyhtbk%3D" aria-label="CAS reference 51">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30954475" aria-label="PubMed reference 51">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6853612" aria-label="PubMed Central reference 51">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=DoubletFinder%3A%20doublet%20detection%20in%20single-cell%20RNA-sequencing%20data%20using%20artificial%20nearest%20neighbors&amp;journal=Cell%20Syst.&amp;volume=8&amp;pages=329-337&amp;publication_year=2019&amp;author=McGinnis%2CCS&amp;author=Murrow%2CLM&amp;author=Gartner%2CZJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Büttner, M., Miao, Z., Wolf, F. A., Teichmann, S. A. &amp; Theis, F. J. A test metric for assessing single-cell RNA-seq batch correction. <i>Nat. Methods</i> <b>16</b>, 43–49 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30573817" aria-label="PubMed reference 52">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20test%20metric%20for%20assessing%20single-cell%20RNA-seq%20batch%20correction&amp;journal=Nat.%20Methods&amp;volume=16&amp;pages=43-49&amp;publication_year=2019&amp;author=B%C3%BCttner%2CM&amp;author=Miao%2CZ&amp;author=Wolf%2CFA&amp;author=Teichmann%2CSA&amp;author=Theis%2CFJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Vento-Tormo, R. et al. Single-cell reconstruction of the early maternal-fetal interface in humans. <i>Nature</i> <b>563</b>, 347–353 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1SnsbzI" aria-label="CAS reference 53">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30429548" aria-label="PubMed reference 53">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20reconstruction%20of%20the%20early%20maternal-fetal%20interface%20in%20humans&amp;journal=Nature&amp;volume=563&amp;pages=347-353&amp;publication_year=2018&amp;author=Vento-Tormo%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Liberzon, A. et al. The Molecular Signatures Database (MSigDB) hallmark gene set collection. <i>Cell Syst.</i> <b>1</b>, 417–425 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFaltLc%3D" aria-label="CAS reference 54">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26771021" aria-label="PubMed reference 54">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4707969" aria-label="PubMed Central reference 54">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Molecular%20Signatures%20Database%20%28MSigDB%29%20hallmark%20gene%20set%20collection&amp;journal=Cell%20Syst.&amp;volume=1&amp;pages=417-425&amp;publication_year=2015&amp;author=Liberzon%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Korotkevich, G., Sukhov, V. &amp; Sergushichev, A. Fast gene-set enrichment analysis. Preprint at <i>BioRxiv</i> <a href="https://doi.org/10.1101/060012" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/060012">https://doi.org/10.1101/060012</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Haghverdi, L., Lun, A. T. L., Morgan, M. D. &amp; Marioni, J. C. Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors. <i>Nat. Biotechnol.</i> <b>36</b>, 421–427 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXmslKrtLo%3D" aria-label="CAS reference 56">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29608177" aria-label="PubMed reference 56">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6152897" aria-label="PubMed Central reference 56">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Batch%20effects%20in%20single-cell%20RNA-sequencing%20data%20are%20corrected%20by%20matching%20mutual%20nearest%20neighbors&amp;journal=Nat.%20Biotechnol.&amp;volume=36&amp;pages=421-427&amp;publication_year=2018&amp;author=Haghverdi%2CL&amp;author=Lun%2CATL&amp;author=Morgan%2CMD&amp;author=Marioni%2CJC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Blondel, V. D., Guillaume, J.-L., Lambiotte, R. &amp; Lefebvre, E. Fast unfolding of communities in large networks. <i>J. Stat. Mech.</i> <b>2008</b>, P10008 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20unfolding%20of%20communities%20in%20large%20networks&amp;journal=J.%20Stat.%20Mech.&amp;volume=2008&amp;publication_year=2008&amp;author=Blondel%2CVD&amp;author=Guillaume%2CJ-L&amp;author=Lambiotte%2CR&amp;author=Lefebvre%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">McInnes, L., Healy, J. &amp; Melville, J. UMAP: uniform manifold approximation and projection for dimension reduction. Preprint at<a href="https://arxiv.org/abs/1802.03426" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://arxiv.org/abs/1802.03426"> https://arxiv.org/abs/1802.03426</a> (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. <i>Bioinformatics</i> <b>26</b>, 139–140 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1WlurvO" aria-label="CAS reference 59">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19910308" aria-label="PubMed reference 59">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&amp;journal=Bioinformatics&amp;volume=26&amp;pages=139-140&amp;publication_year=2010&amp;author=Robinson%2CMD&amp;author=McCarthy%2CDJ&amp;author=Smyth%2CGK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">McCarthy, D. J., Chen, Y. &amp; Smyth, G. K. Differential expression analysis of multifactor RNA-seq experiments with respect to biological variation. <i>Nucleic Acids Res.</i> <b>40</b>, 4288–4297 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XnsF2ks74%3D" aria-label="CAS reference 60">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22287627" aria-label="PubMed reference 60">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3378882" aria-label="PubMed Central reference 60">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20expression%20analysis%20of%20multifactor%20RNA-seq%20experiments%20with%20respect%20to%20biological%20variation&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=4288-4297&amp;publication_year=2012&amp;author=McCarthy%2CDJ&amp;author=Chen%2CY&amp;author=Smyth%2CGK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Kuleshov, M. V. et al. Enrichr: a comprehensive gene-set enrichment analysis web server 2016 update. <i>Nucleic Acids Res.</i> <b>44</b>, W90–W97 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtV2itrfF" aria-label="CAS reference 61">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27141961" aria-label="PubMed reference 61">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987924" aria-label="PubMed Central reference 61">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Enrichr%3A%20a%20comprehensive%20gene-set%20enrichment%20analysis%20web%20server%202016%20update&amp;journal=Nucleic%20Acids%20Res.&amp;volume=44&amp;pages=W90-W97&amp;publication_year=2016&amp;author=Kuleshov%2CMV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. <i>J. R. Stat. Soc. Ser. B Stat. Methodol.</i> <b>57</b>, 289–300 (1995).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Controlling%20the%20false%20discovery%20rate%3A%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&amp;journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B%20Stat.%20Methodol.&amp;volume=57&amp;pages=289-300&amp;publication_year=1995&amp;author=Benjamini%2CY&amp;author=Hochberg%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Ghazanfar, S., Strbenac, D., Ormerod, J. T., Yang, J. Y. H. &amp; Patrick, E. DCARS: differential correlation across ranked samples. <i>Bioinformatics</i> <b>35</b>, 823–829 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVWgsrbF" aria-label="CAS reference 63">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30102408" aria-label="PubMed reference 63">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=DCARS%3A%20differential%20correlation%20across%20ranked%20samples&amp;journal=Bioinformatics&amp;volume=35&amp;pages=823-829&amp;publication_year=2019&amp;author=Ghazanfar%2CS&amp;author=Strbenac%2CD&amp;author=Ormerod%2CJT&amp;author=Yang%2CJYH&amp;author=Patrick%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Gupta, N. T. et al. Change-O: a toolkit for analyzing large-scale B cell immunoglobulin repertoire sequencing data. <i>Bioinformatics</i> <b>31</b>, 3356–3358 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1Git7%2FE" aria-label="CAS reference 64">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26069265" aria-label="PubMed reference 64">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4793929" aria-label="PubMed Central reference 64">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Change-O%3A%20a%20toolkit%20for%20analyzing%20large-scale%20B%20cell%20immunoglobulin%20repertoire%20sequencing%20data&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3356-3358&amp;publication_year=2015&amp;author=Gupta%2CNT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Lefranc, M.-P. IMGT, the International ImMunoGeneTics Information System. <i>Novartis Found. Symp.</i> <b>254</b>, 126–142 <a href="http://imgt.cines.fr" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="http://imgt.cines.fr">http://imgt.cines.fr</a> (2008).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Gadala-Maria, D., Yaari, G., Uduman, M. &amp; Kleinstein, S. H. Automated analysis of high-throughput B cell sequencing data reveals a high frequency of novel immunoglobulin V gene segment alleles. <i>Proc. Natl Acad. Sci. USA</i> <b>112</b>, E862–E870 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXitlagtLw%3D" aria-label="CAS reference 66">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25675496" aria-label="PubMed reference 66">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4345584" aria-label="PubMed Central reference 66">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20analysis%20of%20high-throughput%20B%20cell%20sequencing%20data%20reveals%20a%20high%20frequency%20of%20novel%20immunoglobulin%20V%20gene%20segment%20alleles&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=E862-E870&amp;publication_year=2015&amp;author=Gadala-Maria%2CD&amp;author=Yaari%2CG&amp;author=Uduman%2CM&amp;author=Kleinstein%2CSH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Bashford-Rogers, R. J. M. et al. Network properties derived from deep sequencing of human B cell receptor repertoires delineate B cell populations. <i>Genome Res</i> <b>23</b>, 1874–1884 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslKktr7M" aria-label="CAS reference 67">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23742949" aria-label="PubMed reference 67">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3814887" aria-label="PubMed Central reference 67">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Network%20properties%20derived%20from%20deep%20sequencing%20of%20human%20B%20cell%20receptor%20repertoires%20delineate%20B%20cell%20populations&amp;journal=Genome%20Res&amp;volume=23&amp;pages=1874-1884&amp;publication_year=2013&amp;author=Bashford-Rogers%2CRJM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Hay, S. B., Ferchen, K., Chetal, K., Grimes, H. L. &amp; Salomonis, N. The human cell atlas bone marrow single-cell interactive web portal. <i>Exp. Hematol.</i> <b>68</b>, 51–61 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30243574" aria-label="PubMed reference 68">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6296228" aria-label="PubMed Central reference 68">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20human%20cell%20atlas%20bone%20marrow%20single-cell%20interactive%20web%20portal&amp;journal=Exp.%20Hematol.&amp;volume=68&amp;pages=51-61&amp;publication_year=2018&amp;author=Hay%2CSB&amp;author=Ferchen%2CK&amp;author=Chetal%2CK&amp;author=Grimes%2CHL&amp;author=Salomonis%2CN">
                    Google Scholar</a>&nbsp;
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-021-01329-2?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>This publication is part of the Human Cell Atlas (<a href="https://www.humancellatlas.org/publications/">https://www.humancellatlas.org/publications/</a>). We acknowledge assistance from P. Vegh, J. Fletcher and D. Dixon to the IV-LPS study and funding from the Wellcome Human Cell Atlas Strategic Science Support (WT211276/Z/18/Z). We acknowledge B. Yeung and K. ‘Kit’ Nazor from BioLegend for helpful discussions in the optimization of the CITE-seq protocol. We acknowledge B. Stewart for his helpful contributions toward curation of the final dataset. We acknowledge R. Queen and R. Hussain from Newcastle University Genomics Core Facility for technical assistance. We acknowledge M. Glanville and the Newcastle University Infectious Diseases Facility for biosafety level 3 laboratory access and assistance. We acknowledge M. Prete for assistance with online data hosting and interactivity. We thank NIHR BioResource volunteers for their participation, and gratefully acknowledge NIHR BioResource centers, NHS Trusts and staff for their contribution. We thank the NIHR, NHS Blood and Transplant, and Health Data Research UK as part of the Digital Innovation Hub Programme. A.J.S. is an NIHR senior investigator; the views expressed in this article are those of the author(s) and not necessarily those of the NHS, the NIHR or the Department of Health and Social Care. M.H. is funded by Wellcome (WT107931/Z/15/Z), The Lister Institute for Preventive Medicine and Newcastle NIHR Biomedical Research Centre. B.G. is funded by Wellcome (206328/Z/17/Z), MRC (MR/S036113/1), and the Aging Biology Foundation. S.A.T. is funded by Wellcome (WT206194), ERC Consolidator and EU MRG-Grammar awards. Z.K.T. and M.R.C. are supported by a Medical Research Council Human Cell Atlas Research Grant (MR/S035842/1). J.C.M. is supported by core funding from Cancer Research UK (C9545/A29580) and from the European Molecular Biology Laboratory. M.R.C. is supported by a Versus Arthritis Cure Challenge Research Grant (21777), and an NIHR Research Professorship (RP-2017-08-ST2-002). S.H. is funded by Wellcome (207556_Z_17_Z). H.W.K. is supported by a Sir Henry Wellcome Postdoctoral Fellowship (213555/Z/18/Z). K.B.M. acknowledges funding from the Chan Zuckerberg Initiative (2017-174169), Wellcome (WT211276/Z/18/Z) and Sanger Core (WT206194). M.Z.N. acknowledges funding from a UKRI Innovation/Rutherford Fund Fellowship allocated by the MRC and the UK Regenerative Medicine Platform (MR/5005579/1). M.Z.N. and K.B.M. have been funded by the Rosetrees Trust (M944). K.B.W. is funded by University College London, Birkbeck MRC Doctoral Training Programme. J.L.B. acknowledges funding from the MRC and the UK Regenerative Medicine Platform (MR/5005579/1). M.Y. is funded by The Jikei University School of Medicine. A.T.H. is funded by the NIHR (ACF-2018-01-004) and the BMA Foundation. K.F.B is funded by an NIHR Clinical Lectureship (CL-2017-01-004). M.W.M. is funded by Action Medical Research (GN2779). N.M. is supported by a DFG Research Fellowship (ME 5209/1-1). C.J.A.D. is funded by Wellcome (211153/Z/18/Z) and acknowledges support from the Barbour Foundation. S.W. is funded by a Barbour Foundation PhD studentship. E.L. is supported by a Sir Henry Dale fellowship from Wellcome/Royal Society (107630/Z/15/Z), BBSRC (BB/P002293/1), and core support grants from Wellcome and MRC to the Wellcome-MRC Cambridge Stem Cell Institute (203151/Z/16/Z) C.M.S is currently supported by grants from BBSRC (BB/V006738/1), GOSH Children’s charity (COVID_CSmith_017), Wellcome Trust (212516/Z/18/Z) and NIHR GOSH Biomedical research centre. This work was partly funded by UKRI/NIHR through the UK Coronavirus Immunology Consortium.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="nAff24"><p class="c-article-author-information__authors-list">Florian Gothe</p><p class="js-present-address">Present address: Department of Pediatrics, Dr. von Hauner Children’s Hospital, University Hospital, Ludwig-Maximilians-Universität Munich, Munich, Germany</p></li><li class="c-article-author-information__item" id="na2"><p>These authors jointly supervised this work: Kerstin B. Meyer, Marko Z. Nikolic, Christopher J. A. Duncan, Ken Smith, Sarah A. Teichmann, Menna R. Clatworthy, John C. Marioni, Berthold Göttgens, Muzlifah Haniffa.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Biosciences Institute, Newcastle University, Newcastle upon Tyne, UK</p><p class="c-article-author-affiliation__authors-list">Emily Stephenson,&nbsp;Gary Reynolds,&nbsp;Rachel A. Botting,&nbsp;Justin Engelbert,&nbsp;Bayanne Olabi,&nbsp;Laura Jardine,&nbsp;Louis C. S. Gardner,&nbsp;Issac Goh,&nbsp;Dave Horsfall,&nbsp;Jim McGrath,&nbsp;Simone Webb,&nbsp;Michael W. Mather,&nbsp;Nusayhah Gopee,&nbsp;Caroline Wilson,&nbsp;Jonathan M. Coxhead&nbsp;&amp;&nbsp;Muzlifah Haniffa</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Wellcome - MRC Cambridge Stem Cell Institute, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Fernando J. Calero-Nieto,&nbsp;Nicola K. Wilson,&nbsp;Nicole Mende,&nbsp;Elisa Laurenti&nbsp;&amp;&nbsp;Berthold Göttgens</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">European Molecular Biology Laboratory, European Bioinformatics Institute (EMBL-EBI), Wellcome Genome Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Michael D. Morgan,&nbsp;Karsten Bach&nbsp;&amp;&nbsp;John C. Marioni</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Cancer Research UK Cambridge Institute, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Michael D. Morgan,&nbsp;Karsten Bach,&nbsp;Katarzyna Kania,&nbsp;Paul Coupland&nbsp;&amp;&nbsp;John C. Marioni</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Molecular Immunity Unit, Department of Medicine, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Zewen Kelvin Tuong&nbsp;&amp;&nbsp;Menna R. Clatworthy</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Wellcome Sanger Institute, Wellcome Genome Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Zewen Kelvin Tuong,&nbsp;Waradon Sungnak,&nbsp;Natsuhiko Kumasaka,&nbsp;Rik G. H. Lindeboom,&nbsp;Emma Dann,&nbsp;Ni Huang,&nbsp;Krzysztof Polanski,&nbsp;Elena Prigmore,&nbsp;Vladimir Yu Kiselev,&nbsp;Stijn van Dongen,&nbsp;Hamish W. King,&nbsp;Kerstin B. Meyer,&nbsp;Sarah A. Teichmann,&nbsp;Menna R. Clatworthy,&nbsp;John C. Marioni&nbsp;&amp;&nbsp;Muzlifah Haniffa</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">UCL Respiratory, Division of Medicine, University College London, London, UK</p><p class="c-article-author-affiliation__authors-list">Kaylee B. Worlock,&nbsp;Masahiro Yoshida,&nbsp;Josephine L. Barnes,&nbsp;Sam M. Janes&nbsp;&amp;&nbsp;Marko Z. Nikolić</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Translational and Clinical Research Institute, Newcastle University, Newcastle upon Tyne, UK</p><p class="c-article-author-affiliation__authors-list">Jarmila Stremenova Spegarova,&nbsp;Florian Gothe,&nbsp;Jonathan Scott,&nbsp;Rebecca P. Payne,&nbsp;Kenneth F. Baker,&nbsp;Aidan T. Hanrath,&nbsp;Ina C. D. Schim van der Loeff,&nbsp;Anthony J. Rostron,&nbsp;A. John Simpson,&nbsp;Sophie Hambleton&nbsp;&amp;&nbsp;Christopher J. A. Duncan</p></li><li id="Aff9"><p class="c-article-author-affiliation__address">NIHR Newcastle Biomedical Research Centre, Newcastle Hospitals NHS Foundation Trust, Newcastle upon Tyne, UK</p><p class="c-article-author-affiliation__authors-list">Kenneth F. Baker&nbsp;&amp;&nbsp;Muzlifah Haniffa</p></li><li id="Aff10"><p class="c-article-author-affiliation__address">Department of Infection and Tropical Medicine, Newcastle upon Tyne Hospitals NHS Foundation, Newcastle upon Tyne, UK</p><p class="c-article-author-affiliation__authors-list">Aidan T. Hanrath,&nbsp;Andrew S. Barr,&nbsp;Amada Sanchez-Gonzalez&nbsp;&amp;&nbsp;Christopher J. A. Duncan</p></li><li id="Aff11"><p class="c-article-author-affiliation__address">Cambridge Institute of Therapeutic Immunology and Infectious Disease, Jeffrey Cheah Biomedical Centre, Cambridge Biomedical Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Laura Bergamaschi,&nbsp;Federica Mescia,&nbsp;Stephen Baker,&nbsp;Gordon Dougan,&nbsp;Ravindra K. Gupta,&nbsp;Christoph Hess,&nbsp;Paul J. Lehner,&nbsp;Nicholas J. Matheson,&nbsp;Kenneth G. C. Smith,&nbsp;James E. D. Thaventhiran,&nbsp;Michael P. Weekes,&nbsp;Ben Bullman,&nbsp;Kelvin Hunter,&nbsp;Nicole Pond,&nbsp;Ariana Betancourt,&nbsp;Georgie Bower,&nbsp;Madeline Epping,&nbsp;Lori Turner,&nbsp;Paul A. Lyons&nbsp;&amp;&nbsp;Kenneth G. C. Smith</p></li><li id="Aff12"><p class="c-article-author-affiliation__address">Department of Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Laura Bergamaschi,&nbsp;Federica Mescia,&nbsp;Stephen Baker,&nbsp;John R. Bradley,&nbsp;Gordon Dougan,&nbsp;Ravindra K. Gupta,&nbsp;Christoph Hess,&nbsp;Paul J. Lehner,&nbsp;Nicholas J. Matheson,&nbsp;Kenneth G. C. Smith,&nbsp;Charlotte Summers,&nbsp;James E. D. Thaventhiran,&nbsp;Mark Toshner,&nbsp;Benjamin J. Dunmore,&nbsp;Stefan Gräf,&nbsp;Josh Hodgson,&nbsp;Christopher Huang,&nbsp;Kelvin Hunter,&nbsp;Ekaterina Legchenko,&nbsp;Cecilia Matara,&nbsp;Jennifer Martin,&nbsp;Ciara O’Donnell,&nbsp;Linda Pointon,&nbsp;Nicole Pond,&nbsp;Joy Shih,&nbsp;Rachel Sutcliffe,&nbsp;Tobias Tilly,&nbsp;Carmen Treacy,&nbsp;Zhen Tong,&nbsp;Jennifer Wood,&nbsp;Ariana Betancourt,&nbsp;Georgie Bower,&nbsp;Aloka De Sa,&nbsp;Madeline Epping,&nbsp;Sarah Jackson,&nbsp;Isobel Jarvis,&nbsp;Jimmy Marsden,&nbsp;Georgina Okecha,&nbsp;Ommar Omarjee,&nbsp;Marianne Perera,&nbsp;Nathan Richoz,&nbsp;Rahul Sharma,&nbsp;Lori Turner,&nbsp;Eckart M. D. D. De Bie,&nbsp;Katherine Bunclark,&nbsp;Michael Mackay,&nbsp;Mayurun Selvan,&nbsp;Paul A. Lyons,&nbsp;Kenneth G. C. Smith&nbsp;&amp;&nbsp;Menna R. Clatworthy</p></li><li id="Aff13"><p class="c-article-author-affiliation__address">University College London Hospitals NHS Foundation Trust, London, UK</p><p class="c-article-author-affiliation__authors-list">Eliz Kilich,&nbsp;Angus de Wilton,&nbsp;Sam M. Janes&nbsp;&amp;&nbsp;Marko Z. Nikolić</p></li><li id="Aff14"><p class="c-article-author-affiliation__address">Royal Free Hospital NHS Foundation Trust, London, UK</p><p class="c-article-author-affiliation__authors-list">Anita Saigal&nbsp;&amp;&nbsp;Aarash Saleh</p></li><li id="Aff15"><p class="c-article-author-affiliation__address">UCL Great Ormond Street Institute of Child Health, London, UK</p><p class="c-article-author-affiliation__authors-list">Claire M. Smith</p></li><li id="Aff16"><p class="c-article-author-affiliation__address">Department of Dermatology, Newcastle Hospitals NHS Foundation Trust, Newcastle upon Tyne, UK</p><p class="c-article-author-affiliation__authors-list">Nusayhah Gopee&nbsp;&amp;&nbsp;Muzlifah Haniffa</p></li><li id="Aff17"><p class="c-article-author-affiliation__address">The Innovation Lab Integrated COVID Hub North East, Newcastle Upon Tyne, UK</p><p class="c-article-author-affiliation__authors-list">Caroline Wilson</p></li><li id="Aff18"><p class="c-article-author-affiliation__address">School of Computing, Newcastle University, Newcastle Upon Tyne, UK</p><p class="c-article-author-affiliation__authors-list">Jaume Bacardit</p></li><li id="Aff19"><p class="c-article-author-affiliation__address">Centre for Immunobiology, Blizard Institute, Queen Mary University of London, London, UK</p><p class="c-article-author-affiliation__authors-list">Hamish W. King</p></li><li id="Aff20"><p class="c-article-author-affiliation__address">Integrated Critical Care Unit, Sunderland Royal Hospital, South Tyneside and Sunderland NHS Foundation Trust, Sunderland, UK</p><p class="c-article-author-affiliation__authors-list">Anthony J. Rostron</p></li><li id="Aff21"><p class="c-article-author-affiliation__address">Theory of Condensed Matter Group, Cavendish Laboratory/Department of Physics, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Sarah A. Teichmann</p></li><li id="Aff22"><p class="c-article-author-affiliation__address">Cambridge Institute for Therapeutic Immunology and Infectious Disease, Cambridge Biomedical Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Menna R. Clatworthy</p></li><li id="Aff23"><p class="c-article-author-affiliation__address">NIHR Cambridge Biomedical Research Centre, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Menna R. Clatworthy</p></li><li id="Aff25"><p class="c-article-author-affiliation__address">NIHR BioResource, Cambridge University Hospitals NHS Foundation, Cambridge Biomedical Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">John R. Bradley,&nbsp;Nathalie Kingston,&nbsp;Stefan Gräf,&nbsp;John Allison,&nbsp;Helen Butcher,&nbsp;Daniela Caputo,&nbsp;Debbie Clapham-Riley,&nbsp;Eleanor Dewhurst,&nbsp;Anita Furlong,&nbsp;Barbara Graves,&nbsp;Jennifer Gray,&nbsp;Tasmin Ivers,&nbsp;Mary Kasanicki,&nbsp;Emma Le Gresley,&nbsp;Rachel Linger,&nbsp;Sarah Meloy,&nbsp;Francesca Muldoon,&nbsp;Nigel Ovington,&nbsp;Sofia Papadia,&nbsp;Isabel Phelan,&nbsp;Hannah Stark,&nbsp;Kathleen E. Stirrups,&nbsp;Paul Townsend,&nbsp;Neil Walker&nbsp;&amp;&nbsp;Jennifer Webster</p></li><li id="Aff26"><p class="c-article-author-affiliation__address">Division of Virology, Department of Pathology, University of Cambridge, Addenbrooke’s Hospital, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Ian G. Goodfellow</p></li><li id="Aff27"><p class="c-article-author-affiliation__address">Department of Biomedicine, University and University Hospital Basel, Basel, Switzerland</p><p class="c-article-author-affiliation__authors-list">Christoph Hess</p></li><li id="Aff28"><p class="c-article-author-affiliation__address">Botnar Research Centre for Child Health (BRCCH), University Basel &amp; ETH Zurich, Basel, Switzerland</p><p class="c-article-author-affiliation__authors-list">Christoph Hess</p></li><li id="Aff29"><p class="c-article-author-affiliation__address">Department of Haematology, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Nathalie Kingston,&nbsp;Willem H. Owehand,&nbsp;Stefan Gräf,&nbsp;John Allison,&nbsp;Nigel Ovington,&nbsp;Kathleen E. Stirrups,&nbsp;Paul Townsend&nbsp;&amp;&nbsp;Neil Walker</p></li><li id="Aff30"><p class="c-article-author-affiliation__address">Cambridge Clinical Research Centre, NIHR Clinical Research Facility, Cambridge University Hospitals NHS Foundation Trust, Addenbrooke’s Hospital, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Caroline Saunders,&nbsp;Ashlea Bucke,&nbsp;Jo Calder,&nbsp;Laura Canna,&nbsp;Jason Domingo,&nbsp;Anne Elmer,&nbsp;Stewart Fuller,&nbsp;Sarah Hewitt,&nbsp;Jane Kennet,&nbsp;Sherly Jose,&nbsp;Jenny Kourampa,&nbsp;Anne Meadows,&nbsp;Jane Price,&nbsp;Cherry Publico,&nbsp;Rebecca Rastall,&nbsp;Carla Ribeiro,&nbsp;Jane Rowlands,&nbsp;Valentina Ruffolo&nbsp;&amp;&nbsp;Hugo Tordesillas</p></li><li id="Aff31"><p class="c-article-author-affiliation__address">Metabolic Research Laboratories, Wellcome Trust-Medical Research Council Institute of Metabolic Science, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Charlotte Summers,&nbsp;Mark Toshner&nbsp;&amp;&nbsp;Masa Josipovic</p></li><li id="Aff32"><p class="c-article-author-affiliation__address">Royal Papworth Hospital NHS Foundation Trust, Cambridge Biomedical Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Charlotte Summers,&nbsp;Mark Toshner,&nbsp;Alice Michael,&nbsp;Ali Ansaripour,&nbsp;Lucy Mwaura,&nbsp;Caroline Patterson&nbsp;&amp;&nbsp;Gary Polwarth</p></li><li id="Aff33"><p class="c-article-author-affiliation__address">Addenbrooke’s Hospital, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Charlotte Summers,&nbsp;Petra Polgarova,&nbsp;Giovanni di Stefano&nbsp;&amp;&nbsp;Mary Kasanicki</p></li><li id="Aff34"><p class="c-article-author-affiliation__address">Department of Obstetrics &amp; Gynaecology, The Rosie Maternity Hospital, Robinson Way, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">James E. D. Thaventhiran&nbsp;&amp;&nbsp;Oisin Huhn</p></li><li id="Aff35"><p class="c-article-author-affiliation__address">Department of Paediatrics, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Julie Harris&nbsp;&amp;&nbsp;Criona O’Brien</p></li><li id="Aff36"><p class="c-article-author-affiliation__address">Cambridge Institute for Medical Research, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Stuart Fawke</p></li><li id="Aff37"><p class="c-article-author-affiliation__address">Department of Veterinary Medicine, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Emma Jones</p></li><li id="Aff38"><p class="c-article-author-affiliation__address">Department of Biochemistry, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Marta Wylot</p></li><li id="Aff39"><p class="c-article-author-affiliation__address">Cancer Molecular Diagnostics Laboratory, Department of Oncology, University of Cambridge, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Francesca Nice</p></li><li id="Aff40"><p class="c-article-author-affiliation__address">Department of Surgery, Addenbrooke’s Hospital, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Sabrina Rossi&nbsp;&amp;&nbsp;Cissy Yong</p></li><li id="Aff41"><p class="c-article-author-affiliation__address">Department of Clinical Biochemistry and Immunology, Addenbrooke’s Hospital, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Sarah Spencer</p></li><li id="Aff42"><p class="c-article-author-affiliation__address">Department of Public Health and Primary Care, School of Clinical Medicine, University of Cambridge, Cambridge Biomedical Campus, Cambridge, UK</p><p class="c-article-author-affiliation__authors-list">Helen Butcher,&nbsp;Daniela Caputo,&nbsp;Debbie Clapham-Riley,&nbsp;Eleanor Dewhurst,&nbsp;Anita Furlong,&nbsp;Barbara Graves,&nbsp;Jennifer Gray,&nbsp;Tasmin Ivers,&nbsp;Emma Le Gresley,&nbsp;Rachel Linger,&nbsp;Sarah Meloy,&nbsp;Francesca Muldoon,&nbsp;Sofia Papadia,&nbsp;Isabel Phelan,&nbsp;Hannah Stark&nbsp;&amp;&nbsp;Jennifer Webster</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Emily-Stephenson-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Emily Stephenson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Emily%20Stephenson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Emily%20Stephenson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Emily%20Stephenson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Gary-Reynolds-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Gary Reynolds</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Gary%20Reynolds" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Gary%20Reynolds" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Gary%20Reynolds%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Rachel_A_-Botting-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Rachel A. Botting</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Rachel%20A.%20Botting" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Rachel%20A.%20Botting" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Rachel%20A.%20Botting%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Fernando_J_-Calero_Nieto-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Fernando J. Calero-Nieto</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Fernando%20J.%20Calero-Nieto" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Fernando%20J.%20Calero-Nieto" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Fernando%20J.%20Calero-Nieto%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Michael_D_-Morgan-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Michael D. Morgan</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michael%20D.%20Morgan" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michael%20D.%20Morgan" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20D.%20Morgan%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Zewen_Kelvin-Tuong-Aff5-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Zewen Kelvin Tuong</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Zewen%20Kelvin%20Tuong" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Zewen%20Kelvin%20Tuong" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Zewen%20Kelvin%20Tuong%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Karsten-Bach-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Karsten Bach</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Karsten%20Bach" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Karsten%20Bach" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Karsten%20Bach%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Waradon-Sungnak-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Waradon Sungnak</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Waradon%20Sungnak" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Waradon%20Sungnak" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Waradon%20Sungnak%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Kaylee_B_-Worlock-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Kaylee B. Worlock</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Kaylee%20B.%20Worlock" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kaylee%20B.%20Worlock" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kaylee%20B.%20Worlock%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Masahiro-Yoshida-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Masahiro Yoshida</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Masahiro%20Yoshida" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Masahiro%20Yoshida" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Masahiro%20Yoshida%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Natsuhiko-Kumasaka-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Natsuhiko Kumasaka</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Natsuhiko%20Kumasaka" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Natsuhiko%20Kumasaka" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Natsuhiko%20Kumasaka%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Katarzyna-Kania-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Katarzyna Kania</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Katarzyna%20Kania" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Katarzyna%20Kania" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Katarzyna%20Kania%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Justin-Engelbert-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Justin Engelbert</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Justin%20Engelbert" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Justin%20Engelbert" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Justin%20Engelbert%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Bayanne-Olabi-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Bayanne Olabi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Bayanne%20Olabi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Bayanne%20Olabi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Bayanne%20Olabi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jarmila_Stremenova-Spegarova-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Jarmila Stremenova Spegarova</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jarmila%20Stremenova%20Spegarova" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jarmila%20Stremenova%20Spegarova" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jarmila%20Stremenova%20Spegarova%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Nicola_K_-Wilson-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Nicola K. Wilson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Nicola%20K.%20Wilson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nicola%20K.%20Wilson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nicola%20K.%20Wilson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Nicole-Mende-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Nicole Mende</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Nicole%20Mende" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nicole%20Mende" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nicole%20Mende%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Laura-Jardine-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Laura Jardine</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Laura%20Jardine" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Laura%20Jardine" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Laura%20Jardine%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Louis_C__S_-Gardner-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Louis C. S. Gardner</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Louis%20C.%20S.%20Gardner" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Louis%20C.%20S.%20Gardner" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Louis%20C.%20S.%20Gardner%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Issac-Goh-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Issac Goh</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Issac%20Goh" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Issac%20Goh" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Issac%20Goh%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Dave-Horsfall-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Dave Horsfall</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Dave%20Horsfall" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Dave%20Horsfall" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Dave%20Horsfall%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jim-McGrath-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Jim McGrath</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jim%20McGrath" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jim%20McGrath" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jim%20McGrath%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Simone-Webb-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Simone Webb</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Simone%20Webb" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Simone%20Webb" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Simone%20Webb%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Michael_W_-Mather-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Michael W. Mather</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michael%20W.%20Mather" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michael%20W.%20Mather" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20W.%20Mather%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Rik_G__H_-Lindeboom-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Rik G. H. Lindeboom</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Rik%20G.%20H.%20Lindeboom" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Rik%20G.%20H.%20Lindeboom" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Rik%20G.%20H.%20Lindeboom%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Emma-Dann-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Emma Dann</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Emma%20Dann" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Emma%20Dann" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Emma%20Dann%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ni-Huang-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Ni Huang</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ni%20Huang" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ni%20Huang" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ni%20Huang%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Krzysztof-Polanski-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Krzysztof Polanski</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Krzysztof%20Polanski" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Krzysztof%20Polanski" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Krzysztof%20Polanski%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Elena-Prigmore-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Elena Prigmore</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Elena%20Prigmore" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Elena%20Prigmore" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Elena%20Prigmore%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Florian-Gothe-Aff8-Aff24"><span class="c-article-authors-search__title u-h3 js-search-name">Florian Gothe</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Florian%20Gothe" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Florian%20Gothe" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Florian%20Gothe%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jonathan-Scott-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Jonathan Scott</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jonathan%20Scott" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jonathan%20Scott" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jonathan%20Scott%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Rebecca_P_-Payne-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Rebecca P. Payne</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Rebecca%20P.%20Payne" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Rebecca%20P.%20Payne" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Rebecca%20P.%20Payne%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Kenneth_F_-Baker-Aff8-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Kenneth F. Baker</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Kenneth%20F.%20Baker" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kenneth%20F.%20Baker" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kenneth%20F.%20Baker%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Aidan_T_-Hanrath-Aff8-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Aidan T. Hanrath</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Aidan%20T.%20Hanrath" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Aidan%20T.%20Hanrath" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Aidan%20T.%20Hanrath%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ina_C__D_-Schim_van_der_Loeff-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Ina C. D. Schim van der Loeff</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ina%20C.%20D.%20Schim%20van%20der%20Loeff" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ina%20C.%20D.%20Schim%20van%20der%20Loeff" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ina%20C.%20D.%20Schim%20van%20der%20Loeff%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Andrew_S_-Barr-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Andrew S. Barr</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Andrew%20S.%20Barr" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Andrew%20S.%20Barr" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Andrew%20S.%20Barr%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Amada-Sanchez_Gonzalez-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Amada Sanchez-Gonzalez</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Amada%20Sanchez-Gonzalez" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Amada%20Sanchez-Gonzalez" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Amada%20Sanchez-Gonzalez%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Laura-Bergamaschi-Aff11-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Laura Bergamaschi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Laura%20Bergamaschi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Laura%20Bergamaschi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Laura%20Bergamaschi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Federica-Mescia-Aff11-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Federica Mescia</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Federica%20Mescia" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Federica%20Mescia" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Federica%20Mescia%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Josephine_L_-Barnes-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Josephine L. Barnes</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Josephine%20L.%20Barnes" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Josephine%20L.%20Barnes" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Josephine%20L.%20Barnes%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Eliz-Kilich-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Eliz Kilich</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Eliz%20Kilich" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Eliz%20Kilich" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Eliz%20Kilich%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Angus-Wilton-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Angus de Wilton</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Angus%20de%20Wilton" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Angus%20de%20Wilton" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Angus%20de%20Wilton%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Anita-Saigal-Aff14"><span class="c-article-authors-search__title u-h3 js-search-name">Anita Saigal</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Anita%20Saigal" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anita%20Saigal" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anita%20Saigal%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Aarash-Saleh-Aff14"><span class="c-article-authors-search__title u-h3 js-search-name">Aarash Saleh</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Aarash%20Saleh" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Aarash%20Saleh" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Aarash%20Saleh%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sam_M_-Janes-Aff7-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Sam M. Janes</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sam%20M.%20Janes" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sam%20M.%20Janes" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sam%20M.%20Janes%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Claire_M_-Smith-Aff15"><span class="c-article-authors-search__title u-h3 js-search-name">Claire M. Smith</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Claire%20M.%20Smith" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Claire%20M.%20Smith" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Claire%20M.%20Smith%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Nusayhah-Gopee-Aff1-Aff16"><span class="c-article-authors-search__title u-h3 js-search-name">Nusayhah Gopee</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Nusayhah%20Gopee" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nusayhah%20Gopee" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nusayhah%20Gopee%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Caroline-Wilson-Aff1-Aff17"><span class="c-article-authors-search__title u-h3 js-search-name">Caroline Wilson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Caroline%20Wilson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Caroline%20Wilson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Caroline%20Wilson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Paul-Coupland-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Paul Coupland</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Paul%20Coupland" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Paul%20Coupland" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Paul%20Coupland%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jonathan_M_-Coxhead-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Jonathan M. Coxhead</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jonathan%20M.%20Coxhead" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jonathan%20M.%20Coxhead" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jonathan%20M.%20Coxhead%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Vladimir_Yu-Kiselev-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Vladimir Yu Kiselev</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Vladimir%20Yu%20Kiselev" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Vladimir%20Yu%20Kiselev" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Vladimir%20Yu%20Kiselev%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Stijn-Dongen-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Stijn van Dongen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Stijn%20van%20Dongen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Stijn%20van%20Dongen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stijn%20van%20Dongen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jaume-Bacardit-Aff18"><span class="c-article-authors-search__title u-h3 js-search-name">Jaume Bacardit</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jaume%20Bacardit" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jaume%20Bacardit" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jaume%20Bacardit%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Hamish_W_-King-Aff6-Aff19"><span class="c-article-authors-search__title u-h3 js-search-name">Hamish W. King</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Hamish%20W.%20King" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Hamish%20W.%20King" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Hamish%20W.%20King%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Anthony_J_-Rostron-Aff8-Aff20"><span class="c-article-authors-search__title u-h3 js-search-name">Anthony J. Rostron</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Anthony%20J.%20Rostron" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anthony%20J.%20Rostron" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anthony%20J.%20Rostron%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-A__John-Simpson-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">A. John Simpson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=A.%20John%20Simpson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=A.%20John%20Simpson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22A.%20John%20Simpson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sophie-Hambleton-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Sophie Hambleton</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sophie%20Hambleton" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sophie%20Hambleton" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sophie%20Hambleton%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Elisa-Laurenti-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Elisa Laurenti</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Elisa%20Laurenti" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Elisa%20Laurenti" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Elisa%20Laurenti%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Paul_A_-Lyons-Aff11-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Paul A. Lyons</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Paul%20A.%20Lyons" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Paul%20A.%20Lyons" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Paul%20A.%20Lyons%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Kerstin_B_-Meyer-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Kerstin B. Meyer</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Kerstin%20B.%20Meyer" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kerstin%20B.%20Meyer" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kerstin%20B.%20Meyer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marko_Z_-Nikoli_-Aff7-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Marko Z. Nikolić</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Marko%20Z.%20Nikoli%C4%87" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marko%20Z.%20Nikoli%C4%87" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marko%20Z.%20Nikoli%C4%87%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Christopher_J__A_-Duncan-Aff8-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Christopher J. A. Duncan</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Christopher%20J.%20A.%20Duncan" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christopher%20J.%20A.%20Duncan" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christopher%20J.%20A.%20Duncan%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Kenneth_G__C_-Smith-Aff11-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Kenneth G. C. Smith</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Kenneth%20G.%20C.%20Smith" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kenneth%20G.%20C.%20Smith" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kenneth%20G.%20C.%20Smith%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sarah_A_-Teichmann-Aff6-Aff21"><span class="c-article-authors-search__title u-h3 js-search-name">Sarah A. Teichmann</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sarah%20A.%20Teichmann" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sarah%20A.%20Teichmann" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sarah%20A.%20Teichmann%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Menna_R_-Clatworthy-Aff5-Aff6-Aff12-Aff22-Aff23"><span class="c-article-authors-search__title u-h3 js-search-name">Menna R. Clatworthy</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Menna%20R.%20Clatworthy" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Menna%20R.%20Clatworthy" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Menna%20R.%20Clatworthy%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-John_C_-Marioni-Aff3-Aff4-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">John C. Marioni</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=John%20C.%20Marioni" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=John%20C.%20Marioni" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22John%20C.%20Marioni%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Berthold-G_ttgens-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Berthold Göttgens</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Berthold%20G%C3%B6ttgens" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Berthold%20G%C3%B6ttgens" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Berthold%20G%C3%B6ttgens%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Muzlifah-Haniffa-Aff1-Aff6-Aff9-Aff16"><span class="c-article-authors-search__title u-h3 js-search-name">Muzlifah Haniffa</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Muzlifah%20Haniffa" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Muzlifah%20Haniffa" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Muzlifah%20Haniffa%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="groups">Consortia</h3><div class="c-article-author-institutional-author" id="group-1"><h3 class="c-article-author-institutional-author__name u-h3">Cambridge Institute of Therapeutic Immunology and Infectious Disease-National Institute of Health Research (CITIID-NIHR) COVID-19 BioResource Collaboration</h3><ul class="c-article-author-institutional-author__author-list"><li class="c-article-author-institutional-author__author-name">Stephen Baker</li><li class="c-article-author-institutional-author__author-name">,&nbsp;John R. Bradley</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Gordon Dougan</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ian G. Goodfellow</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ravindra K. Gupta</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Christoph Hess</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Nathalie Kingston</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Paul J. Lehner</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Paul A. Lyons</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Nicholas J. Matheson</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Willem H. Owehand</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Caroline Saunders</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Kenneth G. C. Smith</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Charlotte Summers</li><li class="c-article-author-institutional-author__author-name">,&nbsp;James E. D. Thaventhiran</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Mark Toshner</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Michael P. Weekes</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ashlea Bucke</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jo Calder</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Laura Canna</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jason Domingo</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Anne Elmer</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Stewart Fuller</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Julie Harris</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Sarah Hewitt</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jane Kennet</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Sherly Jose</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jenny Kourampa</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Anne Meadows</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Criona O’Brien</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jane Price</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Cherry Publico</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Rebecca Rastall</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Carla Ribeiro</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jane Rowlands</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Valentina Ruffolo</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Hugo Tordesillas</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ben Bullman</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Benjamin J. Dunmore</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Stuart Fawke</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Stefan Gräf</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Josh Hodgson</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Christopher Huang</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Kelvin Hunter</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Emma Jones</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ekaterina Legchenko</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Cecilia Matara</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jennifer Martin</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ciara O’Donnell</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Linda Pointon</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Nicole Pond</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Joy Shih</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Rachel Sutcliffe</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Tobias Tilly</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Carmen Treacy</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Zhen Tong</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jennifer Wood</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Marta Wylot</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Laura Bergamaschi</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ariana Betancourt</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Georgie Bower</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Aloka De Sa</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Madeline Epping</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Oisin Huhn</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Sarah Jackson</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Isobel Jarvis</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jimmy Marsden</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Francesca Nice</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Georgina Okecha</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ommar Omarjee</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Marianne Perera</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Nathan Richoz</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Rahul Sharma</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Lori Turner</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Eckart M. D. D. De Bie</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Katherine Bunclark</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Masa Josipovic</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Michael Mackay</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Federica Mescia</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Alice Michael</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Sabrina Rossi</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Mayurun Selvan</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Sarah Spencer</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Cissy Yong</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ali Ansaripour</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Lucy Mwaura</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Caroline Patterson</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Gary Polwarth</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Petra Polgarova</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Giovanni di Stefano</li><li class="c-article-author-institutional-author__author-name">,&nbsp;John Allison</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Helen Butcher</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Daniela Caputo</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Debbie Clapham-Riley</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Eleanor Dewhurst</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Anita Furlong</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Barbara Graves</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jennifer Gray</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Tasmin Ivers</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Mary Kasanicki</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Emma Le Gresley</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Rachel Linger</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Sarah Meloy</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Francesca Muldoon</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Nigel Ovington</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Sofia Papadia</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Isabel Phelan</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Hannah Stark</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Kathleen E. Stirrups</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Paul Townsend</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Neil Walker</li><li class="c-article-author-institutional-author__author-name">&nbsp;&amp;&nbsp;Jennifer Webster</li></ul></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>M.H., B.G., S.T., J.M. and M.R.C. conceived and directed this study. A.J.S. and A.J.R. conceived the IV-LPS study. M.Z.N. and K.M. conceived and led the UCL study. C.J.A.D., K.S., P.L., M.Z.N., F.M., E.K., A.d.W., A. Saigal, A. Saleh, S.M.J., A.T.H., K.F.B., I.C.D.S.v.d.L., L.C.S.G., A.S.B., A.S.-G. and L.B. organized sample acquisition, recruited participants, collected samples and clinical metadata. E.S., R.A.B., F.G., J.S., R.P.P., K.B.W., M.Y. and J.L.B. isolated PBMCs. R.A.B., E.S., K.B.W., M.Y., J.L.B., N.M., F.J.C.-N. performed 10x and CITE-seq analysis. E.S., J.E. and K.K. prepared sequencing libraries. E.P., J.C. and P.C. conducted the sequencing. G.R., Z.K.T., K.B., M.D.M., W.S., N.K., E.S., S.v.D., V.K., N.H., R.L., K.P. and E.D. analyzed the data. J.M.G., D.H., V.Y.K and S.v.D. facilitated online data hosting. M.H., B.G., S.T., J.M., M.R.C., M.Z.N., K.M., G.R., Z.K.T., K.B., M.D.M., W.S., N.K., R.A.B., E.S., L.J., S.W. and J.S.S. interpreted the data. J.S.S. performed flow cytometry. E.S. performed multiplex cytokine analysis. M.H., B.G., S.T., M.R.C., J.M., E.S., G.R., L.J., B.O., M.D.M., K.B., N.K., Z.K.T. and W.S. wrote the manuscript. N.M., L.C.S.G., S.W., K.F.B., C.W., J.M.C., H.W.K., S.H., E.L., M.Z.N., K.M, K.B.M. and K.S. edited the manuscript. Z.K.T. and H.W.K. developed software (dandelion).</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:st9@sanger.ac.uk">Sarah A. Teichmann</a>, <a id="corresp-c2" href="mailto:mrc38@cam.ac.uk">Menna R. Clatworthy</a>, <a id="corresp-c3" href="mailto:marioni@ebi.ac.uk">John C. Marioni</a>, <a id="corresp-c4" href="mailto:bg200@cam.ac.uk">Berthold Göttgens</a> or <a id="corresp-c5" href="mailto:m.a.haniffa@ncl.ac.uk">Muzlifah Haniffa</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar3">Competing interests</h3>
                <p>S.A.T. has received remunerations for consulting and Scientific Advisory Board work from Genentech, Biogen, Roche and GlaxoSmithKline, as well as Foresite Labs over the past 3 years. All other authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Peer review information</b> <i>Nature Medicine</i> thanks the anonymous reviewer(s) for their contribution to the peer review of this work. Saheli Sadanand was the primary editor on this article and managed its editorial process and peer review in collaboration with the rest of the editorial team.</p><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec59-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec59">Extended data</h2><div class="c-article-section__content" id="Sec59-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 single-cell analysis quality " href="/articles/s41591-021-01329-2/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig6_ESM.jpg">Extended Data Fig. 1 Single-cell analysis quality control and cell type definition for COVID PBMC single cell analysis.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Bar chart showing the composition of sample severities across the three sites (n = 23 mild, n = 30 moderate, n = 11, severe, n = 10 critical biologically independent samples). <b>b</b>, Scatter plot displaying the total number of gene counts per sample from each site. <b>c</b>, UMAP visualisations from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1b</a> coloured by site. <b>d</b>, Boxplot of kBET results calculated both before and after batch correction with Harmony for each cluster in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1b</a> kBET statistic calculation using patient ID as the batch factor (n = 130 biologically independent samples, n = 627,172 cells in 1 experiment). Hinges indicate to 25th and 75th percentile and whiskers to lowest and highest value in 1.5*interquartile range. <b>e</b>, Dot plots of 5’ gene expression (top; blue) and surface protein (bottom; red) expression for populations shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1a</a> where the colour is scaled by mean expression and the dot size is proportional to the percent of the population expressing the gene/protein, respectively. <b>f</b>, Tile plot showing percentage concordance between COVID-19 PBMC annotation (x-axis) and Azimuth annotation (y-axis) (<a href="https://satijalab.org/azimuth/">https://satijalab.org/azimuth/</a>).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 differential abundance analys" href="/articles/s41591-021-01329-2/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig7_ESM.jpg">Extended Data Fig. 2 Differential abundance analysis and expression of GWAS hits related to cytokines, chemokines and growth factors.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Volcano plots showing differential abundance testing performed using quasi-likelihood F-test comparing healthy controls to cases for linear trends across disease severity groups. Differentially abundant cell types were determined using a 10% false discovery rate (FDR). <b>b</b>, Scatter plots showing blood counts for Newcastle data grouped by severity status (n = 6 healthy and LPS samples; n = 11 mild, n = 17 moderate, n = 7 severe, and n = 7 critical biologically independent samples). Dotted lines mark the normal ranges. Significance determined using Kruskal-Wallis with Dunn’s post hoc corrected for multiple comparison. WBC, whole blood count. <b>c</b>, Forest plot showing the standard deviation of each clinical/technical factor estimated by the Poisson generalised linear mixed model. Error bars show the standard error estimated from the Fisher information matrix (n = 130 biologically independent samples) SD, standard deviation. <b>d</b>, Volcano plots showing differential abundance testing according to time since symptom onset. Differentially abundant (FDR 10%) points are shown in red. <b>e</b>, Box plots displaying the duration of COVID-19 symptoms from the onset by severity (n = 23 mild, n = 30 moderate, n = 11, severe, n = 10 critical biologically independent samples). Boxes denote IQR with median shown as horizontal bars. Whiskers extend to 1.5x the IQR; outliers are shown as individual points. <b>f</b>, Box plots displaying the duration of COVID-19 symptoms from the onset split by severity and sex (n = 23 mild, n = 30 moderate, n = 11, severe, n = 10 critical biologically independent samples). Boxes denote IQR with median shown as horizontal bars. Whiskers extend to 1.5x the IQR; outliers are shown as individual points. <b>g</b>, Correlated log fold-changes of cell type abundance changes as a function of symptom duration with (x-axis) and without critically ill patients (y-axis). <b>h</b>, Differential abundance testing with a leave-one-out analysis for the T cells (top) of B cells (bottom) (FDR10%). <b>i</b>, Heat map displaying fold change over healthy (left) and dot plot of gene expression where the colour is scaled by mean expression and the dot size is proportional to the percent of the population expressing the gene (right) for genes associated with COVID-19 identified in recent GWAS studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Severe COVID-19 GWAS Groupet al. Genome-wide association study of severe COVID-19 with respiratory failure. N. Engl. J. Med. 383, 1522–1534 (2020)." href="/articles/s41591-021-01329-2#ref-CR18" id="ref-link-section-d73692734e5817">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Pairo-Castineira, E. et al. Genetic mechanisms of critical illness in COVID-19. Nature 
                  https://doi.org/10.1038/s41586-020-03065-y
                  
                 (2020)." href="/articles/s41591-021-01329-2#ref-CR19" id="ref-link-section-d73692734e5820">19</a></sup> for the cell populations in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1b</a>. <b>j</b>, Heat map displaying normalised values of cytokine, chemokine and growth factors in serum of patients with COVID-19.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 myeloid comparisons with bron" href="/articles/s41591-021-01329-2/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig8_ESM.jpg">Extended Data Fig. 3 Myeloid comparisons with bronchial alveolar lavage dataset and receptor-ligand interaction analysis between megakaryocyte, myeloid and progenitor cells.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Volcano plot showing differential abundance testing according to time since symptom onset for the myeloid populations. Differentially abundant (FDR 10%) points are shown in red. <b>b</b>, Dot plots of gene expression of C1 complement components for cells in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig1">1b</a> where the colour is scaled by mean expression and the dot size is proportional to the percent of the population expressing the gene. <b>c</b>, Dot plots of gene expression of a recently published BAL dataset (accession number GSE145926<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Liao, M. et al. Single-cell landscape of bronchoalveolar immune cells in patients with COVID-19. Nat. Med. 26, 842–844 (2020)." href="/articles/s41591-021-01329-2#ref-CR22" id="ref-link-section-d73692734e5863">22</a></sup>) for genes in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2a</a> where the colour is scaled by mean expression and the dot size is proportional to the percent of the population expressing the gene. <b>d</b>, Heat map of CellPhoneDB predicted ligand:receptor interactions between platelets and monocyte subsets, based on the protein data. <b>e</b>, Dot plots of expression of protein markers used to annotate clusters in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig2">2i</a>. MK, Megakaryocyte <b>f</b>. Heatmap of differentially expressed genes between megakaryocyte, myeloid and erythroid progenitor clusters. MK, megakaryocyte; My, myeloid. <b>g</b>, Box plots displaying enrichment of an erythroid signature (top) and a myeloid signature (bottom) found in CD34<sup>−</sup>CD38<sup>−</sup> (left) and CD34<sup>+</sup>CD38<sup>+</sup> HSPCs (right), separated by severity. Asterisks above bars indicate significance and are coloured by the severity for which they were compared to. Absolute values are provided in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">8</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM3">9</a>. (n = 120 biologically independent samples, n = 3297 cells in 1 experiment). Boxes denote IQR with median shown as horizontal bars. Whiskers extend to 1.5x the IQR; outliers are shown as individual points (n = 24 healthy, n = 86 COVID-19 biologically independent samples) (p-values: Myeloid signature in CD38 negative HSPCs, Healthy vs. Mild: 0.8 x 10<sup>-3</sup>, Healthy vs Moderate 0.02).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 umap embedding of t lymphoid " href="/articles/s41591-021-01329-2/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig9_ESM.jpg">Extended Data Fig. 4 UMAP embedding of T lymphoid clusters showing cell type estimation and covariates.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, UMAP visualisation of 309,617 T cells separated by sources of donors. <b>b</b>, UMAP visualisation showing 2-dimensional kernel density estimates of each T cell type in UMAP space. <b>c</b>.-<b>e</b>. UMAP visualisation of T cells coloured by gender (<b>c</b>), disease severity status (<b>d</b>) and age (<b>e</b>).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 differential abundance testin" href="/articles/s41591-021-01329-2/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig10_ESM.jpg">Extended Data Fig. 5 Differential abundance testing, gene set enrichment analysis and clonal diversity analysis of T lymphoid compartment.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Box plots showing the proportion of cell types shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-021-01329-2#Fig3">3a</a>, n = 108 biologically independent samples. Boxes denote IQR with median shown as horizontal bars. Whiskers extend to 1.5x the IQR; outliers are shown as individual points. <b>b</b>, Volcano plots showing results of differential abundance testing. Abundance counts were modelled either comparing healthy vs. COVID-19, or as a function of severity. Hypothesis testing was performed using quasi-likelihood F-test comparing healthy controls to cases, or for either a linear or quadratic trend across disease severity. Differentially abundant cell types were determined using a 10% false discovery rate (FDR). <b>c</b>, Heatmaps showing mean expression levels across T cell subsets for suppressive (<i>FOXP3</i>), proliferating (<i>MKI67</i>) and exhaustion markers (<i>PDCD1, HAVCR2, LAG3, TIGIT, TOX</i>). Columns denote the mean log-normalised expression within each severity category and healthy controls. <b>d</b>, Volcano plot showing differential abundance testing according to time since symptom onset for the T cell populations. Differentially abundant (FDR 10%) points are shown in red. <b>e</b>, Gene set enrichment (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-021-01329-2#Sec8">Methods</a>) in each T cell type based on differential gene expression (DGE) analysis was performed across COVID-19 disease severity groups, ordered from healthy &gt; asymptomatic &gt; mild &gt; moderate &gt; severe &gt; critical. Statistically significant DE genes were defined with FDR &lt; 1%. Significant enrichments were defined with 10% FDR. <b>f</b>, Bar plots showing percent (mean + /- SEM) of CD3<sup>+</sup>CD4<sup>+</sup> (blue) and CD3<sup>+</sup>CD8<sup>+</sup> (green) T cells expressing CD107a (left) and CD137 (right) in response to SARS-CoV-2 S peptide stimulation (n = 12 healthy, n = 10 mild, n = 17 moderate, n = 9 severe, and n = 5 critical biologically independent samples). Significance determined using Kruskal-Wallis with Dunn’s post-hoc corrected for multiple comparison. <b>g</b>, Box plots showing clone size distribution for each T cell subset (n = 9 asymptomatic, n = 19 mild, n = 29 moderate, n = 12 severe, n = 10 critical biologically independent samples). Boxes denote IQR with median shown as horizontal bars. Whiskers extend to 1.5x the IQR; outliers are shown as individual points. <b>h</b>, Box plots slowing clonal diversity for each T cell subset (n = 22 healthy, n = 12 asymptomatic, n = 22 mild, n = 31 moderate, n = 14 severe, n = 13 critical biologically independent samples). Boxes denote IQR with median shown as horizontal bars. Whiskers extend to 1.5x the IQR; outliers are shown as individual points.</p><p><a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-021-01329-2#MOESM4">Source data</a></p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 single-cell analysis of b lym" href="/articles/s41591-021-01329-2/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig11_ESM.jpg">Extended Data Fig. 6 Single-cell analysis of B lymphocytes proportion and gene set differences across COVID-19 severity.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Heatmap of mean gene set enrichment scores of (top) adult peripheral blood B cell signatures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Monaco, G. et al. RNA-seq signatures normalized by mRNA abundance allow absolute deconvolution of human immune cell types. Cell Rep. 26, 1627–1640 (2019)." href="/articles/s41591-021-01329-2#ref-CR32" id="ref-link-section-d73692734e6045">32</a></sup> and (bottom) bone marrow B cell signatures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Hay, S. B., Ferchen, K., Chetal, K., Grimes, H. L. &amp; Salomonis, N. The human cell atlas bone marrow single-cell interactive web portal. Exp. Hematol. 68, 51–61 (2018)." href="/articles/s41591-021-01329-2#ref-CR68" id="ref-link-section-d73692734e6049">68</a></sup>. Row enrichment value is scaled from 0-1 and colour gradient corresponds to increasing mean enrichment score. <b>b</b>, (Top) Kruskal-Wallis test results with Benjamini-Hochberg false discovery correction for cell type proportion differences in plasmablast/plasma cells between severities. Significance is denoted by Benjamini-Hochberg corrected <i>P</i> &lt; 0.05 (red text). (Bottom) Cell type abundance counts were modelled as a function of severity. Hypothesis testing was performed using quasi-likelihood F-test comparing asymptomatic to symptomatic covid, for either a linear or quadratic trend across severities. Differentially abundant cell types were determined using a 10% false discovery rate (FDR). <b>c</b>, (Left) Bar plot showing the mean proportion of plasmablast/plasma cells expressing IgA, IgD, IgE, IgG or IgM, based on V(D)J information. (Right) Proportion of IgM, IgA1, total IgG, IgG1, and IgG2 subclass in plasmablast and plasma cells. Kruskal-Wallis test results with Benjamini-Hochberg false discovery correction for cell type proportion differences in plasmablast/plasma cells between severities showed no significant differences. n = 10, 6, 21, 24, 11, 11 (IgM), n = 18, 9, 21, 27, 12, 9 (IgA1), n = 16, 8, 23, 27, 13, 8 (IgG (all)), n = 11, 6, 22, 25, 11, 8 (IgG1) and n = 8, 4, 16, 23, 9, 7 (IgG2) biologically independent patients/samples for healthy, asymptomatic, mild, moderate, severe and critical respectively. <b>d</b>, GSEA of pathways in B cell subsets for asymptomatic/symptomatic COVID versus healthy. Size of circles indicate (absolute) normalised enrichment score (NES). Pathways were considered statistically significant if GSEA (permutation) nominal <i>P</i> &lt; 0.05 and FDR &lt; 0.25 (denoted by non-grey coloured dots; boundary lines in the middle marks FDR = 0.25). EMT, Epithelial-mesenchymal transition. <b>e</b>, Dot plots of TNF signalling molecules, activating and inhibitory BCR signaling molecules (5’ gene expression data) in immature B cells, non-switched memory B cells, ‘exhausted’ B cells and plasma cells. Size of circles indicate percent of cells expressing the gene and colour gradient corresponds to increasing mean expression value.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 single-cell bcr networks acro" href="/articles/s41591-021-01329-2/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig12_ESM.jpg">Extended Data Fig. 7 Single-cell BCR networks across COVID-19 severity.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Single-cell BCR network plots for each severity status coloured by heavy chain isotype class (IgM, IgD, IgA, IgE, or IgG). Each circle/node corresponds to a single B cell with a corresponding set of BCR(s). Each clonotype is presented as a minimally connected graph with edge widths scaled to 1/<i>d</i> + 1 for edge weight <i>d</i> where <i>d</i> corresponds to the total (Levenshtein) edit distance of BCRs between two cells. Size of nodes is scaled according to increasing node closeness centrality scores that is nodes that are highly central to a clonotype network will be larger.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 single-cell bcr clonotype exp" href="/articles/s41591-021-01329-2/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig13_ESM.jpg">Extended Data Fig. 8 Single-cell BCR clonotype expansion analysis across COVID-19 severity.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, UMAP visualisation of B cell lineage and coloured by clonotype size in the V(D)J data. Only expanded clonotypes are coloured (clonotype size &gt; 2). <b>b</b>, Single-cell BCR network plots for each severity status coloured by assigned cell type. <b>c</b>, Single-cell BCR network plots for each severity status coloured by heavy chain isotype subclass (IgM, IgD, IgA1, IgA2, IgE, IgG1, IgG2, IgG3 or IgG4). Each circle/node corresponds to a single B cell with a corresponding set of BCR(s). Each clonotype is presented as a minimally connected graph with edge widths scaled to 1/<i>d</i> + 1 for edge weight <i>d</i> where <i>d</i> corresponds to the total (Levenshtein) edit distance of BCRs between two cells. Size of nodes is scaled according to increasing node closeness centrality scores that is nodes that are highly central to a clonotype network will be larger.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 single-cell bcr analysis betw" href="/articles/s41591-021-01329-2/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig14_ESM.jpg">Extended Data Fig. 9 Single-cell BCR analysis between genders across COVID-19 severity.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Summary plot for BCR IGHV gene usage split by gender (top: female; bottom: male). Each data point is presented as the mean gene usage proportion + /- standard error of the mean for samples within each sample group. Kruskal-Wallis test with Benjamini-Hochberg *<i>P</i> &lt; 0.05 (adjusted p = 0.035) n = 12, 5, 8, 20, 6 and 6 biologically independent male patients/samples and n = 12, 4, 15, 10, 7 and 5 biologically independent female patients/samples for healthy, asymptomatic, mild, moderate, severe and critical respectively. <b>b</b>, (Top) Scatter plot of clonotype/cluster size by vertex size gini indices computed from contracted BCR networks (identical nodes are merged and counted). Each dot represents the gini indices of an individual coloured by severity status. Marginal histograms indicate the distribution of samples in a given severity status along the axes. (Bottom, left) Cluster/clonotype size (contracted network) gini indices separated by gender. (Bottom, right) Vertex size (contracted network) gini indices separated by gender. The boxes extend from the 25th to 75th percentiles. The whiskers go down to the smallest value and up to the largest. The line in the middle of the boxes is plotted at the median. Statistical tests were performed with non-parametric Mann-Whitney U test between the gender groups within each severity status and were considered statistically significant if Benjamini-Hochberg corrected <i>P</i> &lt; 0.05 (denoted by *; n.s. denotes not significant). The Benjamini-Hochberg adjusted p-values are as follows: cluster/clonotype size (contracted network) gini indices comparisons: 3.42x10<sup>-1</sup>, 1.91x10<sup>-25</sup>, 7.71x10<sup>-272</sup>, 1.31x10<sup>-169</sup>, 3.84x10<sup>-113</sup> and 2.98x10<sup>-1</sup>; vertex size (contracted network) gini indices comparisons: 1.12x10<sup>-57</sup>, 2.17x10<sup>-1</sup>, &lt;1x10<sup>-308</sup>, 3.64x10<sup>-49</sup>, 2.69x10<sup>-53</sup> and 2.37x10<sup>-04</sup> - for healthy, asymptomatic, mild, moderate, severe and critical respectively. Colour of asterisks indicates which gender group displays a higher mean gini index (yellow: female; grey: male). n = 12, 5, 8, 20, 6 and 6 biologically independent male patients/samples and n = 12, 4, 15, 10, 7 and 5 biologically independent female patients/samples for healthy, asymptomatic, mild, moderate, severe and critical respectively. n = 1 for malignant (male).</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec60-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec60">Supplementary information</h2><div class="c-article-section__content" id="Sec60-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary File 1 and 2 and Supplementary Fig. 1</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary tables 1–10" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_MOESM3_ESM.xlsx" data-supp-info-image="">Supplementary Tables 1–10</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Table 1: CITE-seq panel. List of TotalSeq-C antibodies, including clone and barcode. Supplementary Table 2: patient metadata. Status summary is based on the WHO COVID-19 classification (WHO/2019-nCoV/clinical/2020.5; <a href="https://www.who.int/publications/i/item/clinical-management-of-covid-19">https://www.who.int/publications/i/item/clinical-management-of-covid-19/</a>). NA, not applicable. Not-known listed where information was unavailable. O<sub>2</sub>, supplemental oxygen via nasal cannulae, face mask or non-rebreather mask. NIV, noninvasive ventilation under continuous (CPAP) or bilevel (BiPAP) positive airway pressure. Supplementary Table 3: number of cells and donors contributing to each cell cluster. Supplementary Table 4: clinical whole-blood counts for Newcastle samples. Number of cells × 10<sup>9</sup> per liter of blood. WBC, white blood cells. Supplementary Table 5: concentration in pg ml<sup>−1</sup> of 45 analytes measured in serum. ≤0, below the limit of detection; asterisks indicate anti-inflammatory cytokines. Supplementary Table 6: type I/III interferon, JAK–STAT and TNF response enrichment scores for MP populations shown in Fig. 2f. Supplementary Table 7: statistical analysis values for Fig. 2f. Supplementary Table 8: statistical analysis values for Fig. 2k and Extended Data 3f. Supplementary Table 9: T cell clonality logistic regression model results across disease severity groups. <i>P</i> values calculated from a two-sided <i>t</i>-test, with Benjamini–Hochberg FDR correction Supplementary Table 10: T cell clonality logistic regression model results across symptom duration. <i>P</i> values calculated from a two-sided <i>t</i>-test, with Benjamini–Hochberg FDR correction.</p></div></div></div></div></div></section><section data-title="Source data"><div class="c-article-section" id="Sec61-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec61">Source data</h2><div class="c-article-section__content" id="Sec61-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="source data extended data fig. 5" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_MOESM4_ESM.xlsx" data-supp-info-image="">Source Data Extended Data Fig. 5</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Cell percentages for flow cytometry.</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Single-cell%20multi-omics%20analysis%20of%20the%20immune%20response%20in%20COVID-19&amp;author=Emily%20Stephenson%20Stephen%20Baker%20et%20al&amp;contentID=10.1038%2Fs41591-021-01329-2&amp;copyright=The%20Author%28s%29&amp;publication=1078-8956&amp;publicationDate=2021-04-20&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41591-021-01329-2" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41591-021-01329-2" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Stephenson, E., Reynolds, G., Botting, R.A. <i>et al.</i> Single-cell multi-omics analysis of the immune response in COVID-19.
                    <i>Nat Med</i> <b>27</b>, 904–916 (2021). https://doi.org/10.1038/s41591-021-01329-2</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-021-01329-2?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-01-15">15 January 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-03-23">23 March 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-04-20">20 April 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-05">May 2021</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41591-021-01329-2</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-block"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/haematopoiesis" data-track="click" data-track-action="view subject" data-track-label="link">Haematopoiesis</a></li><li class="c-article-subject-list__subject"><a href="/subjects/infection" data-track="click" data-track-action="view subject" data-track-label="link">Infection</a></li><li class="c-article-subject-list__subject"><a href="/subjects/rna-sequencing" data-track="click" data-track-action="view subject" data-track-label="link">RNA sequencing</a></li><li class="c-article-subject-list__subject"><a href="/subjects/sars-cov-2" data-track="click" data-track-action="view subject" data-track-label="link">SARS-CoV-2</a></li><li class="c-article-subject-list__subject"><a href="/subjects/viral-infection" data-track="click" data-track-action="view subject" data-track-label="link">Viral infection</a></li>
        </ul>
    </div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section" data-test="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:A single-cell RNA sequencing dataset of peripheral blood cells in long COVID patients on herbal therapy" href="https://doi.org/10.1038/s41597-025-04510-1">
                                        A single-cell RNA sequencing dataset of peripheral blood cells in long COVID patients on herbal therapy
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Karolina Hanna Prazanowska</li><li>Tae-Hun Kim</li><li>Su Bin Lim</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Scientific Data</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Temporal profiling of human lymphoid tissues reveals coordinated defense against viral challenge" href="https://doi.org/10.1038/s41590-024-02064-9">
                                        Temporal profiling of human lymphoid tissues reveals coordinated defense against viral challenge
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Matthew L. Coates</li><li>Nathan Richoz</li><li>Menna R. Clatworthy</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Immunology</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:CD55 upregulation in T cells of COVID-19 patients suppresses type-I interferon responses" href="https://doi.org/10.1038/s42003-025-08066-z">
                                        CD55 upregulation in T cells of COVID-19 patients suppresses type-I interferon responses
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Maria G. Detsika</li><li>Maria Sakkou</li><li>Anastasia Kotanidou</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Communications Biology</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:scCobra allows contrastive cell embedding learning with domain adaptation for single cell data integration and harmonization" href="https://doi.org/10.1038/s42003-025-07692-x">
                                        scCobra allows contrastive cell embedding learning with domain adaptation for single cell data integration and harmonization
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Bowen Zhao</li><li>Kailu Song</li><li>Jun Ding</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Communications Biology</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Single-cell immune aging clocks reveal inter-individual heterogeneity during infection and vaccination" href="https://doi.org/10.1038/s43587-025-00819-z">
                                        Single-cell immune aging clocks reveal inter-individual heterogeneity during infection and vaccination
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Wenchao Li</li><li>Zhenhua Zhang</li><li>Yang Li</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Aging</i> (2025)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-021-01329-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                
            
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
            <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" role="tabpanel"><div class="c-reading-companion__scroll-pane" style="max-height: none;"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Main">Main</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec7" class="c-reading-companion__section-item"><a href="#Sec7" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec8" class="c-reading-companion__section-item"><a href="#Sec8" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-data-availability" class="c-reading-companion__section-item"><a href="#data-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Data availability">Data availability</a></li><li id="rc-sec-code-availability" class="c-reading-companion__section-item"><a href="#code-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Code availability">Code availability</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec59" class="c-reading-companion__section-item"><a href="#Sec59" data-track="click" data-track-action="section anchor" data-track-label="link:Extended data">Extended data</a></li><li id="rc-sec-Sec60" class="c-reading-companion__section-item"><a href="#Sec60" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-Sec61" class="c-reading-companion__section-item"><a href="#Sec61" data-track="click" data-track-action="section anchor" data-track-label="link:Source data">Source data</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by">This article is cited by</a></li></ul></div>
                <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="" style="display: none !important;">
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-test="right-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41591-021-01329-2;doi=10.1038/s41591-021-01329-2;subjmeta=1949,208,232,250,2514,254,255,326,4130,514,596,631,692,699;kwrd=Haematopoiesis,Infection,RNA+sequencing,SARS-CoV-2,Viral+infection">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=-524963500&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-021-01329-2%26doi%3D10.1038/s41591-021-01329-2%26subjmeta%3D1949,208,232,250,2514,254,255,326,4130,514,596,631,692,699%26kwrd%3DHaematopoiesis,Infection,RNA+sequencing,SARS-CoV-2,Viral+infection">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=-524963500&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-021-01329-2%26doi%3D10.1038/s41591-021-01329-2%26subjmeta%3D1949,208,232,250,2514,254,255,326,4130,514,596,631,692,699%26kwrd%3DHaematopoiesis,Infection,RNA+sequencing,SARS-CoV-2,Viral+infection"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Fig. 1: Single-cell multi-omics analysis of PBMCs from individuals with COVID-19 and controls.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig1_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig1_HTML.png" alt="figure 1" aria-describedby="rc-Fig1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Fig. 2: Expansion of complement-expressing nonclassical monocytes and megakaryocyte-primed progenitor cells and increased platelet activation with COVID-19 disease severity.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig2_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig2_HTML.png" alt="figure 2" aria-describedby="rc-Fig2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Fig. 3: Compositional and clonal analyses of T lymphocytes illustrate the expansion of effector subsets.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig3_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig3_HTML.png" alt="figure 3" aria-describedby="rc-Fig3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Fig. 4: Single-cell analysis of B lymphocytes and BCR repertoire reveal plasmablast expansion and clonality differences between genders.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig4_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig4_HTML.png" alt="figure 4" aria-describedby="rc-Fig4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Fig. 5: Integrated framework of the peripheral immune response in COVID-19.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig5_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig5_HTML.png" alt="figure 5" aria-describedby="rc-Fig5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig6">Extended Data Fig. 1 Single-cell analysis quality control and cell type definition for COVID PBMC single cell analysis.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig6_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig6_ESM.jpg" alt="extended data figure 6" aria-describedby="rc-Fig6"></picture><p class="c-reading-companion__figure-links"><a href="#Fig6" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/6" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig7">Extended Data Fig. 2 Differential abundance analysis and expression of GWAS hits related to cytokines, chemokines and growth factors.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig7_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig7_ESM.jpg" alt="extended data figure 7" aria-describedby="rc-Fig7"></picture><p class="c-reading-companion__figure-links"><a href="#Fig7" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/7" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig8">Extended Data Fig. 3 Myeloid comparisons with bronchial alveolar lavage dataset and receptor-ligand interaction analysis between megakaryocyte, myeloid and progenitor cells.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig8_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig8_ESM.jpg" alt="extended data figure 8" aria-describedby="rc-Fig8"></picture><p class="c-reading-companion__figure-links"><a href="#Fig8" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/8" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig9">Extended Data Fig. 4 UMAP embedding of T lymphoid clusters showing cell type estimation and covariates.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig9_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig9_ESM.jpg" alt="extended data figure 9" aria-describedby="rc-Fig9"></picture><p class="c-reading-companion__figure-links"><a href="#Fig9" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/9" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig10">Extended Data Fig. 5 Differential abundance testing, gene set enrichment analysis and clonal diversity analysis of T lymphoid compartment.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig10_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig10_ESM.jpg" alt="extended data figure 10" aria-describedby="rc-Fig10"></picture><p class="c-reading-companion__figure-links"><a href="#Fig10" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/10" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig11">Extended Data Fig. 6 Single-cell analysis of B lymphocytes proportion and gene set differences across COVID-19 severity.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig11_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig11_ESM.jpg" alt="extended data figure 11" aria-describedby="rc-Fig11"></picture><p class="c-reading-companion__figure-links"><a href="#Fig11" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/11" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig12">Extended Data Fig. 7 Single-cell BCR networks across COVID-19 severity.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig12_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig12_ESM.jpg" alt="extended data figure 12" aria-describedby="rc-Fig12"></picture><p class="c-reading-companion__figure-links"><a href="#Fig12" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/12" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig13">Extended Data Fig. 8 Single-cell BCR clonotype expansion analysis across COVID-19 severity.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig13_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig13_ESM.jpg" alt="extended data figure 13" aria-describedby="rc-Fig13"></picture><p class="c-reading-companion__figure-links"><a href="#Fig13" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/13" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig14">Extended Data Fig. 9 Single-cell BCR analysis between genders across COVID-19 severity.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig14_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-021-01329-2/MediaObjects/41591_2021_1329_Fig14_ESM.jpg" alt="extended data figure 14" aria-describedby="rc-Fig14"></picture><p class="c-reading-companion__figure-links"><a href="#Fig14" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-021-01329-2/figures/14" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric" data-track-context="article references sidebar"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">Bedford, J. et al. COVID-19: towards controlling of a pandemic. <i>Lancet</i> <b>395</b>, 1015–1018 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0140-6736%2820%2930673-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0140-6736(20)30673-5" data-track-item_id="10.1016/S0140-6736(20)30673-5">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXltFChsbc%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32197103" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7270596" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=COVID-19%3A%20towards%20controlling%20of%20a%20pandemic&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2820%2930673-5&amp;volume=395&amp;pages=1015-1018&amp;publication_year=2020&amp;author=Bedford%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">John Hopkins University. COVID-19 map. Johns Hopkins Coronavirus Resource Center. <a href="https://coronavirus.jhu.edu/map.html" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://coronavirus.jhu.edu/map.html">https://coronavirus.jhu.edu/map.html</a> (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Chen, N. et al. Epidemiological and clinical characteristics of 99 cases of 2019 novel coronavirus pneumonia in Wuhan, China: a descriptive study. <i>Lancet</i> <b>395</b>, 507–513 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0140-6736%2820%2930211-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0140-6736(20)30211-7" data-track-item_id="10.1016/S0140-6736(20)30211-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFOmsb8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32007143" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7135076" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Epidemiological%20and%20clinical%20characteristics%20of%2099%20cases%20of%202019%20novel%20coronavirus%20pneumonia%20in%20Wuhan%2C%20China%3A%20a%20descriptive%20study&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2820%2930211-7&amp;volume=395&amp;pages=507-513&amp;publication_year=2020&amp;author=Chen%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">Hoffmann, M. et al. SARS-CoV-2 cell entry depends on ACE2 and TMPRSS2 and is blocked by a clinically proven protease inhibitor. <i>Cell</i> <b>181</b>, 271–280 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktl2qtb8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32142651" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7102627" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20cell%20entry%20depends%20on%20ACE2%20and%20TMPRSS2%20and%20is%20blocked%20by%20a%20clinically%20proven%20protease%20inhibitor&amp;journal=Cell&amp;volume=181&amp;pages=271-280&amp;publication_year=2020&amp;author=Hoffmann%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">Diao, B. et al. Reduction and functional exhaustion of T cells in patients with coronavirus disease 2019 (COVID-19). <i>Front. Immunol.</i> <b>11</b>, 827 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVWnsb7L" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32425950" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7205903" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Reduction%20and%20functional%20exhaustion%20of%20T%20cells%20in%20patients%20with%20coronavirus%20disease%202019%20%28COVID-19%29&amp;journal=Front.%20Immunol.&amp;volume=11&amp;publication_year=2020&amp;author=Diao%2CB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Lee, J. S. et al. Immunophenotyping of COVID-19 and influenza highlights the role of type I interferons in development of severe COVID-19. <i>Sci. Immunol.</i> <b>5</b>, eabd1554 (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Arunachalam, P. S. et al. Systems biological assessment of immunity to mild versus severe COVID-19 infection in humans. <i>Science</i> <b>369</b>, 1210–1220 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslCltrnJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32788292" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7665312" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Systems%20biological%20assessment%20of%20immunity%20to%20mild%20versus%20severe%20COVID-19%20infection%20in%20humans&amp;journal=Science&amp;volume=369&amp;pages=1210-1220&amp;publication_year=2020&amp;author=Arunachalam%2CPS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Woodruff, M. C. et al. Extrafollicular B cell responses correlate with neutralizing antibodies and morbidity in COVID-19. <i>Nat. Immunol.</i> <b>21</b>, 1506–1516 (2020).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33028979" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7739702" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Extrafollicular%20B%20cell%20responses%20correlate%20with%20neutralizing%20antibodies%20and%20morbidity%20in%20COVID-19&amp;journal=Nat.%20Immunol.&amp;volume=21&amp;pages=1506-1516&amp;publication_year=2020&amp;author=Woodruff%2CMC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Chen, G. et al. Clinical and immunological features of severe and moderate coronavirus disease 2019. <i>J. Clin. Invest.</i> <b>130</b>, 2620–2629 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXpt12ks78%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32217835" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7190990" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20and%20immunological%20features%20of%20severe%20and%20moderate%20coronavirus%20disease%202019&amp;journal=J.%20Clin.%20Invest.&amp;volume=130&amp;pages=2620-2629&amp;publication_year=2020&amp;author=Chen%2CG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Zhang, J.-Y. et al. Single-cell landscape of immunological responses in patients with COVID-19. <i>Nat. Immunol.</i> <b>21</b>, 1107–1118 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsF2itb%2FE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32788748" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20landscape%20of%20immunological%20responses%20in%20patients%20with%20COVID-19&amp;journal=Nat.%20Immunol.&amp;volume=21&amp;pages=1107-1118&amp;publication_year=2020&amp;author=Zhang%2CJ-Y" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Liu, C. et al. Time-resolved systems immunology reveals a late juncture linked to fatal COVID-19. <i>Cell</i> <a href="https://doi.org/10.1016/j.cell.2021.02.018" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.cell.2021.02.018">https://doi.org/10.1016/j.cell.2021.02.018</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Hadjadj, J. et al. Impaired type I interferon activity and inflammatory responses in severe COVID-19 patients. <i>Science</i> <b>369</b>, 718–724 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFGms7rP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32661059" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7402632" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Impaired%20type%20I%20interferon%20activity%20and%20inflammatory%20responses%20in%20severe%20COVID-19%20patients&amp;journal=Science&amp;volume=369&amp;pages=718-724&amp;publication_year=2020&amp;author=Hadjadj%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Schulte-Schrepping, J. et al. Severe COVID-19 is marked by a dysregulated myeloid cell compartment. <i>Cell</i> <b>182</b>, 1419–1440 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1WqsL7P" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32810438" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7405822" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Severe%20COVID-19%20is%20marked%20by%20a%20dysregulated%20myeloid%20cell%20compartment&amp;journal=Cell&amp;volume=182&amp;pages=1419-1440&amp;publication_year=2020&amp;author=Schulte-Schrepping%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Mann, E. R. et al. Longitudinal immune profiling reveals key myeloid signatures associated with COVID-19. <i>Sci. Immunol.</i> <b>5</b>, eabd6197 (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">Bernardes, J. P. et al. Longitudinal multi-omics analyses identify responses of megakaryocytes, erythroid cells and plasmablasts as hallmarks of severe COVID-19. <i>Immunity</i> <b>53</b>, 1296–1314 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFeltL3P" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33296687" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7689306" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Longitudinal%20multi-omics%20analyses%20identify%20responses%20of%20megakaryocytes%2C%20erythroid%20cells%20and%20plasmablasts%20as%20hallmarks%20of%20severe%20COVID-19&amp;journal=Immunity&amp;volume=53&amp;pages=1296-1314&amp;publication_year=2020&amp;author=Bernardes%2CJP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">World Health Organization. Clinical management of COVID-19. <a href="https://www.who.int/publications/i/item/clinical-management-of-covid-19" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.who.int/publications/i/item/clinical-management-of-covid-19">https://www.who.int/publications/i/item/clinical-management-of-covid-19</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Korsunsky, I. et al. Fast, sensitive and accurate integration of single-cell data with Harmony. <i>Nat. Methods</i> <b>16</b>, 1289–1296 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFOqsr7N" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31740819" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6884693" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%2C%20sensitive%20and%20accurate%20integration%20of%20single-cell%20data%20with%20Harmony&amp;journal=Nat.%20Methods&amp;volume=16&amp;pages=1289-1296&amp;publication_year=2019&amp;author=Korsunsky%2CI" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">Severe COVID-19 GWAS Groupet al. Genome-wide association study of severe COVID-19 with respiratory failure. <i>N. Engl. J. Med.</i> <b>383</b>, 1522–1534 (2020).</p><p class="c-reading-companion__reference-links"><a href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20association%20study%20of%20severe%20COVID-19%20with%20respiratory%20failure&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=383&amp;pages=1522-1534&amp;publication_year=2020" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">Pairo-Castineira, E. et al. Genetic mechanisms of critical illness in COVID-19. <i>Nature</i> <a href="https://doi.org/10.1038/s41586-020-03065-y" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41586-020-03065-y">https://doi.org/10.1038/s41586-020-03065-y</a> (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Lun, A. T. L., Richard, A. C. &amp; Marioni, J. C. Testing for differential abundance in mass cytometry data. <i>Nat. Methods</i> <b>14</b>, 707–709 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXnslKkt70%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28504682" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6155493" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Testing%20for%20differential%20abundance%20in%20mass%20cytometry%20data&amp;journal=Nat.%20Methods&amp;volume=14&amp;pages=707-709&amp;publication_year=2017&amp;author=Lun%2CATL&amp;author=Richard%2CAC&amp;author=Marioni%2CJC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Jardine, L. et al. Lipopolysaccharide inhalation recruits monocytes and dendritic cell subsets to the alveolar airspace. <i>Nat. Commun.</i> <b>10</b>, 1999 (2019).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31040289" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6491485" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Lipopolysaccharide%20inhalation%20recruits%20monocytes%20and%20dendritic%20cell%20subsets%20to%20the%20alveolar%20airspace&amp;journal=Nat.%20Commun.&amp;volume=10&amp;publication_year=2019&amp;author=Jardine%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Liao, M. et al. Single-cell landscape of bronchoalveolar immune cells in patients with COVID-19. <i>Nat. Med.</i> <b>26</b>, 842–844 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXptFyqtr4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32398875" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20landscape%20of%20bronchoalveolar%20immune%20cells%20in%20patients%20with%20COVID-19&amp;journal=Nat.%20Med.&amp;volume=26&amp;pages=842-844&amp;publication_year=2020&amp;author=Liao%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">Evren, E. et al. Distinct developmental pathways from blood monocytes generate human lung macrophage diversity. <i>Immunity</i> <a href="https://doi.org/10.1016/j.immuni.2020.12.003" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.immuni.2020.12.003">https://doi.org/10.1016/j.immuni.2020.12.003</a> (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Ren, X. et al. COVID-19 immune features revealed by a large-scale single-cell transcriptome atlas. <i>Cell</i> <a href="https://doi.org/10.1016/j.cell.2021.01.053" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/j.cell.2021.01.053">https://doi.org/10.1016/j.cell.2021.01.053</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">Levi, M., Thachil, J., Iba, T. &amp; Levy, J. H. Coagulation abnormalities and thrombosis in patients with COVID-19. <i>Lancet Haematol.</i> <b>7</b>, e438–e440 (2020).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32407672" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7213964" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Coagulation%20abnormalities%20and%20thrombosis%20in%20patients%20with%20COVID-19&amp;journal=Lancet%20Haematol.&amp;volume=7&amp;pages=e438-e440&amp;publication_year=2020&amp;author=Levi%2CM&amp;author=Thachil%2CJ&amp;author=Iba%2CT&amp;author=Levy%2CJH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Hottz, E. D. et al. Platelet activation and platelet-monocyte aggregate formation trigger tissue factor expression in patients with severe COVID-19. <i>Blood</i> <b>136</b>, 1330–1341 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFygtrrF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32678428" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Platelet%20activation%20and%20platelet-monocyte%20aggregate%20formation%20trigger%20tissue%20factor%20expression%20in%20patients%20with%20severe%20COVID-19&amp;journal=Blood&amp;volume=136&amp;pages=1330-1341&amp;publication_year=2020&amp;author=Hottz%2CED" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">Velten, L. et al. Human haematopoietic stem cell lineage commitment is a continuous process. <i>Nat. Cell Biol.</i> <b>19</b>, 271–281 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXltVWnsr0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28319093" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5496982" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Human%20haematopoietic%20stem%20cell%20lineage%20commitment%20is%20a%20continuous%20process&amp;journal=Nat.%20Cell%20Biol.&amp;volume=19&amp;pages=271-281&amp;publication_year=2017&amp;author=Velten%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Mende, N. et al. Quantitative and molecular differences distinguish adult human medullary and extramedullary haematopoietic stem and progenitor cell landscapes. Preprint at <i>BioRxiv</i> <a href="https://doi.org/10.1101/2020.01.26.919753" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2020.01.26.919753">https://doi.org/10.1101/2020.01.26.919753</a> (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">Ivanov, S. et al. Interleukin-22 reduces lung inflammation during influenza A virus infection and protects against secondary bacterial infection. <i>J. Virol.</i> <b>87</b>, 6911–6924 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXptlKku7c%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23596287" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3676141" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Interleukin-22%20reduces%20lung%20inflammation%20during%20influenza%20A%20virus%20infection%20and%20protects%20against%20secondary%20bacterial%20infection&amp;journal=J.%20Virol.&amp;volume=87&amp;pages=6911-6924&amp;publication_year=2013&amp;author=Ivanov%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">Desai, N. et al. Temporal and spatial heterogeneity of host response to SARS-CoV-2 pulmonary infection. <i>Nat. Commun.</i> <b>11</b>, 6319 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFelt7nF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33298930" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7725958" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Temporal%20and%20spatial%20heterogeneity%20of%20host%20response%20to%20SARS-CoV-2%20pulmonary%20infection&amp;journal=Nat.%20Commun.&amp;volume=11&amp;publication_year=2020&amp;author=Desai%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Joshi, N. S. et al. Inflammation directs memory precursor and short-lived effector CD8<sup>+</sup> T cell fates via the graded expression of T-bet transcription factor. <i>Immunity</i> <b>27</b>, 281–295 (2007).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSgtb3J" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17723218" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2034442" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Inflammation%20directs%20memory%20precursor%20and%20short-lived%20effector%20CD8%2B%20T%20cell%20fates%20via%20the%20graded%20expression%20of%20T-bet%20transcription%20factor&amp;journal=Immunity&amp;volume=27&amp;pages=281-295&amp;publication_year=2007&amp;author=Joshi%2CNS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Monaco, G. et al. RNA-seq signatures normalized by mRNA abundance allow absolute deconvolution of human immune cell types. <i>Cell Rep.</i> <b>26</b>, 1627–1640 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisFGhurY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30726743" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6367568" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=RNA-seq%20signatures%20normalized%20by%20mRNA%20abundance%20allow%20absolute%20deconvolution%20of%20human%20immune%20cell%20types&amp;journal=Cell%20Rep.&amp;volume=26&amp;pages=1627-1640&amp;publication_year=2019&amp;author=Monaco%2CG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">Adlowitz, D. G. et al. Expansion of activated peripheral blood memory B cells in rheumatoid arthritis, impact of B cell depletion therapy, and biomarkers of response. <i>PLoS ONE</i> <b>10</b>, e0128269 (2015).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26047509" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4457888" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Expansion%20of%20activated%20peripheral%20blood%20memory%20B%20cells%20in%20rheumatoid%20arthritis%2C%20impact%20of%20B%20cell%20depletion%20therapy%2C%20and%20biomarkers%20of%20response&amp;journal=PLoS%20ONE&amp;volume=10&amp;publication_year=2015&amp;author=Adlowitz%2CDG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">Crotty, S. T. T follicular helper cell biology: a decade of discovery and diseases. <i>Immunity</i> <b>50</b>, 1132–1148 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVWjsrrE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31117010" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6532429" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=T%20follicular%20helper%20cell%20biology%3A%20a%20decade%20of%20discovery%20and%20diseases&amp;journal=Immunity&amp;volume=50&amp;pages=1132-1148&amp;publication_year=2019&amp;author=Crotty%2CST" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">Meng, X. et al. Hypoxia-inducible factor-1α is a critical transcription factor for IL-10-producing B cells in autoimmune disease. <i>Nat. Commun.</i> <b>9</b>, 251 (2018).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29343683" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5772476" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Hypoxia-inducible%20factor-1%CE%B1%20is%20a%20critical%20transcription%20factor%20for%20IL-10-producing%20B%20cells%20in%20autoimmune%20disease&amp;journal=Nat.%20Commun.&amp;volume=9&amp;publication_year=2018&amp;author=Meng%2CX" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Pritchard, N. R. &amp; Smith, K. G. C. B cell inhibitory receptors and autoimmunity. <i>Immunology</i> <b>108</b>, 263–273 (2003).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXitlSks7Y%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12603592" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1782894" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=B%20cell%20inhibitory%20receptors%20and%20autoimmunity&amp;journal=Immunology&amp;volume=108&amp;pages=263-273&amp;publication_year=2003&amp;author=Pritchard%2CNR&amp;author=Smith%2CKGC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Doughty, C. A. et al. Antigen receptor-mediated changes in glucose metabolism in B lymphocytes: role of phosphatidylinositol 3-kinase signaling in the glycolytic control of growth. <i>Blood</i> <b>107</b>, 4458–4465 (2006).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xlt1Cgsbc%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16449529" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1895797" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Antigen%20receptor-mediated%20changes%20in%20glucose%20metabolism%20in%20B%20lymphocytes%3A%20role%20of%20phosphatidylinositol%203-kinase%20signaling%20in%20the%20glycolytic%20control%20of%20growth&amp;journal=Blood&amp;volume=107&amp;pages=4458-4465&amp;publication_year=2006&amp;author=Doughty%2CCA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Boothby, M. &amp; Rickert, R. C. Metabolic regulation of the immune humoral response. <i>Immunity</i> <b>46</b>, 743–755 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXnvFKjtrs%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28514675" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5640164" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Metabolic%20regulation%20of%20the%20immune%20humoral%20response&amp;journal=Immunity&amp;volume=46&amp;pages=743-755&amp;publication_year=2017&amp;author=Boothby%2CM&amp;author=Rickert%2CRC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">Guan, W.-J. et al. Clinical characteristics of coronavirus disease 2019 in China. <i>N. Engl. J. Med.</i> <b>382</b>, 1708–1720 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXovVSjsb8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32109013" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20characteristics%20of%20coronavirus%20disease%202019%20in%20China&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=382&amp;pages=1708-1720&amp;publication_year=2020&amp;author=Guan%2CW-J" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">Onder, G., Rezza, G. &amp; Brusaferro, S. Case-fatality rate and characteristics of patients dying in relation to COVID-19 in Italy. <i>J. Am. Med. Assoc.</i> <b>323</b>, 1775–1776 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXps1Srurw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Case-fatality%20rate%20and%20characteristics%20of%20patients%20dying%20in%20relation%20to%20COVID-19%20in%20Italy&amp;journal=J.%20Am.%20Med.%20Assoc.&amp;volume=323&amp;pages=1775-1776&amp;publication_year=2020&amp;author=Onder%2CG&amp;author=Rezza%2CG&amp;author=Brusaferro%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">Parrot, T. et al. MAIT cell activation and dynamics associated with COVID-19 disease severity. <i>Sci. Immunol.</i> <b>5</b>, eabe1670 (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">Le Bert, N. et al. Highly functional virus-specific cellular immune response in asymptomatic SARS-CoV-2 infection. <i>J. Exp. Med</i>. <b>218</b>, e20202617 (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">Rydyznski Moderbacher, C. et al. Antigen-specific adaptive immunity to SARS-CoV-2 in acute COVID-19 and associations with age and disease severity. <i>Cell</i> <b>183</b>, 996–1012 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvF2jsLnL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33010815" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7494270" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Antigen-specific%20adaptive%20immunity%20to%20SARS-CoV-2%20in%20acute%20COVID-19%20and%20associations%20with%20age%20and%20disease%20severity&amp;journal=Cell&amp;volume=183&amp;pages=996-1012&amp;publication_year=2020&amp;author=Rydyznski%20Moderbacher%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">Kaneko, N. et al. Loss of Bcl-6-expressing T follicular helper cells and germinal centers in COVID-19. <i>Cell</i> <b>183</b>, 143–157 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslygsLvO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32877699" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7437499" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Loss%20of%20Bcl-6-expressing%20T%20follicular%20helper%20cells%20and%20germinal%20centers%20in%20COVID-19&amp;journal=Cell&amp;volume=183&amp;pages=143-157&amp;publication_year=2020&amp;author=Kaneko%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">Maloy, K. J. et al. CD4<sup>+</sup> T cell subsets during virus infection. Protective capacity depends on effector cytokine secretion and on migratory capability. <i>J. Exp. Med.</i> <b>191</b>, 2159–2170 (2000).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3cXkt1Wrurk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=10859340" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2193195" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CD4%2B%20T%20cell%20subsets%20during%20virus%20infection.%20Protective%20capacity%20depends%20on%20effector%20cytokine%20secretion%20and%20on%20migratory%20capability&amp;journal=J.%20Exp.%20Med.&amp;volume=191&amp;pages=2159-2170&amp;publication_year=2000&amp;author=Maloy%2CKJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR46">Sahin, U. et al. COVID-19 vaccine BNT162b1 elicits human antibody and T<sub>H</sub>1 T cell responses. <i>Nature</i> <b>586</b>, 594–599 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitV2qs7fI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32998157" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=COVID-19%20vaccine%20BNT162b1%20elicits%20human%20antibody%20and%20TH1%20T%20cell%20responses&amp;journal=Nature&amp;volume=586&amp;pages=594-599&amp;publication_year=2020&amp;author=Sahin%2CU" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR47">Bastard, P. et al. Autoantibodies against type I IFNs in patients with life-threatening COVID-19. <i>Science</i> <b>370</b>, eabd4585 (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR48">Takahashi, T. et al. Sex differences in immune responses that underlie COVID-19 disease outcomes. <i>Nature</i> <b>588</b>, 315–320 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVWjsrzL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32846427" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7725931" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Sex%20differences%20in%20immune%20responses%20that%20underlie%20COVID-19%20disease%20outcomes&amp;journal=Nature&amp;volume=588&amp;pages=315-320&amp;publication_year=2020&amp;author=Takahashi%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR49">Dobin, A. et al. STAR: ultrafast universal RNA-seq aligner. <i>Bioinformatics</i> <b>29</b>, 15–21 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvV2gsbnF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23104886" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&amp;journal=Bioinformatics&amp;volume=29&amp;pages=15-21&amp;publication_year=2013&amp;author=Dobin%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR50">Heaton, H. et al. Souporcell: robust clustering of single-cell RNA-seq data by genotype without reference genotypes. <i>Nat. Methods</i> <b>17</b>, 615–620 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXosV2jsLw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32366989" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Souporcell%3A%20robust%20clustering%20of%20single-cell%20RNA-seq%20data%20by%20genotype%20without%20reference%20genotypes&amp;journal=Nat.%20Methods&amp;volume=17&amp;pages=615-620&amp;publication_year=2020&amp;author=Heaton%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR51">McGinnis, C. S., Murrow, L. M. &amp; Gartner, Z. J. DoubletFinder: doublet detection in single-cell RNA-sequencing data using artificial nearest neighbors. <i>Cell Syst.</i> <b>8</b>, 329–337 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyhtbk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30954475" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6853612" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=DoubletFinder%3A%20doublet%20detection%20in%20single-cell%20RNA-sequencing%20data%20using%20artificial%20nearest%20neighbors&amp;journal=Cell%20Syst.&amp;volume=8&amp;pages=329-337&amp;publication_year=2019&amp;author=McGinnis%2CCS&amp;author=Murrow%2CLM&amp;author=Gartner%2CZJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR52">Büttner, M., Miao, Z., Wolf, F. A., Teichmann, S. A. &amp; Theis, F. J. A test metric for assessing single-cell RNA-seq batch correction. <i>Nat. Methods</i> <b>16</b>, 43–49 (2019).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30573817" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20test%20metric%20for%20assessing%20single-cell%20RNA-seq%20batch%20correction&amp;journal=Nat.%20Methods&amp;volume=16&amp;pages=43-49&amp;publication_year=2019&amp;author=B%C3%BCttner%2CM&amp;author=Miao%2CZ&amp;author=Wolf%2CFA&amp;author=Teichmann%2CSA&amp;author=Theis%2CFJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR53">Vento-Tormo, R. et al. Single-cell reconstruction of the early maternal-fetal interface in humans. <i>Nature</i> <b>563</b>, 347–353 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1SnsbzI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30429548" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20reconstruction%20of%20the%20early%20maternal-fetal%20interface%20in%20humans&amp;journal=Nature&amp;volume=563&amp;pages=347-353&amp;publication_year=2018&amp;author=Vento-Tormo%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR54">Liberzon, A. et al. The Molecular Signatures Database (MSigDB) hallmark gene set collection. <i>Cell Syst.</i> <b>1</b>, 417–425 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFaltLc%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26771021" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4707969" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Molecular%20Signatures%20Database%20%28MSigDB%29%20hallmark%20gene%20set%20collection&amp;journal=Cell%20Syst.&amp;volume=1&amp;pages=417-425&amp;publication_year=2015&amp;author=Liberzon%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR55">Korotkevich, G., Sukhov, V. &amp; Sergushichev, A. Fast gene-set enrichment analysis. Preprint at <i>BioRxiv</i> <a href="https://doi.org/10.1101/060012" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/060012">https://doi.org/10.1101/060012</a> (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR56">Haghverdi, L., Lun, A. T. L., Morgan, M. D. &amp; Marioni, J. C. Batch effects in single-cell RNA-sequencing data are corrected by matching mutual nearest neighbors. <i>Nat. Biotechnol.</i> <b>36</b>, 421–427 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXmslKrtLo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29608177" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6152897" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Batch%20effects%20in%20single-cell%20RNA-sequencing%20data%20are%20corrected%20by%20matching%20mutual%20nearest%20neighbors&amp;journal=Nat.%20Biotechnol.&amp;volume=36&amp;pages=421-427&amp;publication_year=2018&amp;author=Haghverdi%2CL&amp;author=Lun%2CATL&amp;author=Morgan%2CMD&amp;author=Marioni%2CJC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR57">Blondel, V. D., Guillaume, J.-L., Lambiotte, R. &amp; Lefebvre, E. Fast unfolding of communities in large networks. <i>J. Stat. Mech.</i> <b>2008</b>, P10008 (2008).</p><p class="c-reading-companion__reference-links"><a href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20unfolding%20of%20communities%20in%20large%20networks&amp;journal=J.%20Stat.%20Mech.&amp;volume=2008&amp;publication_year=2008&amp;author=Blondel%2CVD&amp;author=Guillaume%2CJ-L&amp;author=Lambiotte%2CR&amp;author=Lefebvre%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR58">McInnes, L., Healy, J. &amp; Melville, J. UMAP: uniform manifold approximation and projection for dimension reduction. Preprint at<a href="https://arxiv.org/abs/1802.03426" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://arxiv.org/abs/1802.03426"> https://arxiv.org/abs/1802.03426</a> (2018).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR59">Robinson, M. D., McCarthy, D. J. &amp; Smyth, G. K. edgeR: a Bioconductor package for differential expression analysis of digital gene expression data. <i>Bioinformatics</i> <b>26</b>, 139–140 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXhs1WlurvO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19910308" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=edgeR%3A%20a%20Bioconductor%20package%20for%20differential%20expression%20analysis%20of%20digital%20gene%20expression%20data&amp;journal=Bioinformatics&amp;volume=26&amp;pages=139-140&amp;publication_year=2010&amp;author=Robinson%2CMD&amp;author=McCarthy%2CDJ&amp;author=Smyth%2CGK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR60">McCarthy, D. J., Chen, Y. &amp; Smyth, G. K. Differential expression analysis of multifactor RNA-seq experiments with respect to biological variation. <i>Nucleic Acids Res.</i> <b>40</b>, 4288–4297 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XnsF2ks74%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22287627" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3378882" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20expression%20analysis%20of%20multifactor%20RNA-seq%20experiments%20with%20respect%20to%20biological%20variation&amp;journal=Nucleic%20Acids%20Res.&amp;volume=40&amp;pages=4288-4297&amp;publication_year=2012&amp;author=McCarthy%2CDJ&amp;author=Chen%2CY&amp;author=Smyth%2CGK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR61">Kuleshov, M. V. et al. Enrichr: a comprehensive gene-set enrichment analysis web server 2016 update. <i>Nucleic Acids Res.</i> <b>44</b>, W90–W97 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtV2itrfF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27141961" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4987924" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Enrichr%3A%20a%20comprehensive%20gene-set%20enrichment%20analysis%20web%20server%202016%20update&amp;journal=Nucleic%20Acids%20Res.&amp;volume=44&amp;pages=W90-W97&amp;publication_year=2016&amp;author=Kuleshov%2CMV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR62">Benjamini, Y. &amp; Hochberg, Y. Controlling the false discovery rate: a practical and powerful approach to multiple testing. <i>J. R. Stat. Soc. Ser. B Stat. Methodol.</i> <b>57</b>, 289–300 (1995).</p><p class="c-reading-companion__reference-links"><a href="http://scholar.google.com/scholar_lookup?&amp;title=Controlling%20the%20false%20discovery%20rate%3A%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&amp;journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B%20Stat.%20Methodol.&amp;volume=57&amp;pages=289-300&amp;publication_year=1995&amp;author=Benjamini%2CY&amp;author=Hochberg%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR63">Ghazanfar, S., Strbenac, D., Ormerod, J. T., Yang, J. Y. H. &amp; Patrick, E. DCARS: differential correlation across ranked samples. <i>Bioinformatics</i> <b>35</b>, 823–829 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVWgsrbF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30102408" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=DCARS%3A%20differential%20correlation%20across%20ranked%20samples&amp;journal=Bioinformatics&amp;volume=35&amp;pages=823-829&amp;publication_year=2019&amp;author=Ghazanfar%2CS&amp;author=Strbenac%2CD&amp;author=Ormerod%2CJT&amp;author=Yang%2CJYH&amp;author=Patrick%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR64">Gupta, N. T. et al. Change-O: a toolkit for analyzing large-scale B cell immunoglobulin repertoire sequencing data. <i>Bioinformatics</i> <b>31</b>, 3356–3358 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1Git7%2FE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26069265" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4793929" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Change-O%3A%20a%20toolkit%20for%20analyzing%20large-scale%20B%20cell%20immunoglobulin%20repertoire%20sequencing%20data&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3356-3358&amp;publication_year=2015&amp;author=Gupta%2CNT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR65">Lefranc, M.-P. IMGT, the International ImMunoGeneTics Information System. <i>Novartis Found. Symp.</i> <b>254</b>, 126–142 <a href="http://imgt.cines.fr" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="http://imgt.cines.fr">http://imgt.cines.fr</a> (2008).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR66">Gadala-Maria, D., Yaari, G., Uduman, M. &amp; Kleinstein, S. H. Automated analysis of high-throughput B cell sequencing data reveals a high frequency of novel immunoglobulin V gene segment alleles. <i>Proc. Natl Acad. Sci. USA</i> <b>112</b>, E862–E870 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXitlagtLw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25675496" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4345584" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20analysis%20of%20high-throughput%20B%20cell%20sequencing%20data%20reveals%20a%20high%20frequency%20of%20novel%20immunoglobulin%20V%20gene%20segment%20alleles&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=E862-E870&amp;publication_year=2015&amp;author=Gadala-Maria%2CD&amp;author=Yaari%2CG&amp;author=Uduman%2CM&amp;author=Kleinstein%2CSH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR67">Bashford-Rogers, R. J. M. et al. Network properties derived from deep sequencing of human B cell receptor repertoires delineate B cell populations. <i>Genome Res</i> <b>23</b>, 1874–1884 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhslKktr7M" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23742949" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3814887" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Network%20properties%20derived%20from%20deep%20sequencing%20of%20human%20B%20cell%20receptor%20repertoires%20delineate%20B%20cell%20populations&amp;journal=Genome%20Res&amp;volume=23&amp;pages=1874-1884&amp;publication_year=2013&amp;author=Bashford-Rogers%2CRJM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR68">Hay, S. B., Ferchen, K., Chetal, K., Grimes, H. L. &amp; Salomonis, N. The human cell atlas bone marrow single-cell interactive web portal. <i>Exp. Hematol.</i> <b>68</b>, 51–61 (2018).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30243574" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6296228" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20human%20cell%20atlas%20bone%20marrow%20single-cell%20interactive%20web%20portal&amp;journal=Exp.%20Hematol.&amp;volume=68&amp;pages=51-61&amp;publication_year=2018&amp;author=Hay%2CSB&amp;author=Ferchen%2CK&amp;author=Chetal%2CK&amp;author=Grimes%2CHL&amp;author=Salomonis%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li></ol></div></div>
        </div>
    </div>
</aside>
</div>


    
        
    
    
        
            
        

        
            
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>




<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Medicine
                    </span>
                    (<i itemprop="alternateName">Nat Med</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-170X</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1078-8956</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/" data-track="click" data-track-action="protocols.io" data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/authors/editorial_policies" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nature-research/open-access" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/language-editing/" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/" data-track="click" data-track-action="research solutions" data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library" data-track="click" data-track-action="Recommend to library" data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica" data-track="click" data-track-action="nature africa" data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly" data-track="click" data-track-action="nature Italy" data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies" data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice" data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/accessibility-statement" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20">
    </a>
    <p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"></path></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"></path><clipPath id="b"><use xlink:href="#a" style="overflow:visible"></use></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"></path><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"></path><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"></path><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"></path></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"></path></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif " data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required="" data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required="">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41591-021-01329-2&amp;format=js&amp;last_modified=2021-04-20" async=""></script>

<ypb-rxowkoxqdnvo></ypb-rxowkoxqdnvo><div data-cc-ghost="" style="height: 352px;"></div><script type="text/javascript" id="fetch-contextual-ads-campaign-data" charset="">(function(){var g=function(a){a=new CustomEvent("campaignDataLoaded",{detail:a});document.dispatchEvent(a)},b=google_tag_manager["rm"]["50443292"](18);if(b){var k=1500;b=google_tag_manager["rm"]["50443292"](19);b="link"===b?google_tag_manager["rm"]["50443292"](20):google_tag_manager["rm"]["50443292"](21);var m=function(a,c){var h=!1,l=setTimeout(function(){h=!0;c(null)},k),d=new XMLHttpRequest;d.onload=function(){var e=null,f=[];if(200===d.status){try{e=JSON.parse(d.responseText)}catch(p){e={}}f=e.campaigns||[]}h||(clearTimeout(l),c(f.length?f.join(","):null))};d.open("GET",
"/platform/contextual?doi\x3d"+a);d.send()},n=function(a,c){a&&-1===a.indexOf("not set")?m(a,c):setTimeout(c,1)};n(b,function(a){window.campaignsForContextualAds=a;g(a);window.dataLayer.push({content:{article:{campaignID:a}}})})}else window.campaignsForContextualAds="",g(null)})();</script>
<script type="text/javascript" id="gtm-setup-accessdetailsloaded-handler" charset="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail||{};var b={event:"update-access-details"};a.institutional_business_partner_ids&&a.resolved_by||console.log("BPID data could not be retrieved from /exposed-details");a.logged_in_to_sn_profile||console.log("logged_in_to_sn_profile could not be retrieved from /exposed-details");var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?
a.resolved_by.join(";"):"",e=a.personal_business_partner_id_found,f=a.logged_in_to_sn_profile,g=a.snid;b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null,personal:e||null,logged_in_to_sn_profile:f,snid:g||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy={};window.dataLayer.push(b);window.idpUserDataLoaded=!0},!1);window.dataLayer.push({event:"accessdetailsloaded-handler-added"});
window.accessDetailsLoadedHandlerAdded=!0;</script><script type="text/javascript" id="gtm-mediatrust" charset="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="crossmark-script" text="" charset="" type="text/javascript" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="polyfill-matches" charset="">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="fetch-idp-user-data" charset="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},site=google_tag_manager["rm"]["50443292"](59),idpUrl;
switch(site){case "nature":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?"https://staging-idp.nature.com/exposed-details":"https://idp.nature.com/exposed-details";break;case "link":idpUrl=-1<window.location.hostname.indexOf("link-qa")?"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details";break;case "springer":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?
"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details"}if(void 0!==idpUrl){var transport=new XMLHttpRequest;transport.open("GET",idpUrl,!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script src="https://verify.nature.com/verify/nature.min.js"></script><script type="text/javascript" id="create-ad-slots" charset="">function createAndLoadAds(){function h(b){"nature"===a&&google_tag_manager["rm"]["50443292"](87)(b);"bmc"!==a&&"springeropen"!==a||google_tag_manager["rm"]["50443292"](99)(b);("link"===a&&"oscar"===google_tag_manager["rm"]["50443292"](100)||"springer"===a||"link"===a&&!0===google_tag_manager["rm"]["50443292"](101))&&google_tag_manager["rm"]["50443292"](126)(b);"link"===a&&"bunsen"===google_tag_manager["rm"]["50443292"](127)&&google_tag_manager["rm"]["50443292"](152)(b);"link"===a&&"Core"===google_tag_manager["rm"]["50443292"](153)&&google_tag_manager["rm"]["50443292"](165)(b)}window.googletag=window.googletag||{cmd:[]};var a=google_tag_manager["rm"]["50443292"](166),e=google_tag_manager["rm"]["50443292"](168);
if(-1===window.location.search.indexOf("hide_ads\x3dtrue"))if(window.adSlots&&0!==Object.keys(window.adSlots).length)console.log("Ads previously loaded. Will not update ad slots until next page load.");else{window.adSlots||(window.adSlots={});window.getAd=function(b,f){for(var c in window.adSlots)if(-1<c.indexOf(b)){if("object"===f)return adSlots[c];if("slot"===f)return adSlots[c].slot}};googletag.cmd.push(function(){googletag.pubads().setPrivacySettings({limitedAds:e});console.log("limitedAds is ",
e);googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["rm"]["50443292"](170));googletag.pubads().enableSingleRequest();googletag.pubads().disableInitialLoad()});for(var g=document.querySelectorAll("[data-gpt]"),d=0;g[d];++d)h(g[d]);googletag.cmd.push(function(){googletag.enableServices()})}}createAndLoadAds();</script>
</body></html>