<html lang="en" class="js"><head><link rel="preload" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" as="font" type="font/woff2" crossorigin=""><script async="" src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script type="text/javascript" async="" src="https://pagead2.googlesyndication.com/tag/js/gpt.js"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-MRVXSHQ"></script><script type="text/javascript" src="https://www.nature.com/iXwy6YZ_UJ8zmjlhnhuvgk54MaE2JIk_GYdMrdiKbqQ7wj14GVFtrkFAj6jU7qd4Mkg4hZyVmssHI99fmfNwXQ=="></script><script src="https://cmp.nature.com/production_live/en/consent-bundle-8-85.js" onload="initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')"></script>
    <title>Immunogenicity and therapeutic targeting of a public neoantigen derived from mutated PIK3CA | Nature Medicine</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nm.rss">


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin="">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480">

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"cancer-immunotherapy;sequencing;t-cells;tumour-immunology","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Medicine","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41591-022-01786-3"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Smita S. Chandran","Jiaqi Ma","Martin G. Klatt","Friederike Dündar","Chaitanya Bandlamudi","Pedram Razavi","Hannah Y. Wen","Britta Weigelt","Paul Zumbo","Si Ning Fu","Lauren B. Banks","Fei Yi","Enric Vercher","Inaki Etxeberria","Watchain D. Bestman","Arnaud Da Cruz Paula","Ilinca S. Aricescu","Alexander Drilon","Doron Betel","David A. Scheinberg","Brian M. Baker","Christopher A. Klebanoff"],"publishedAt":1651104000,"publishedAtString":"2022-04-28","title":"Immunogenicity and therapeutic targeting of a public neoantigen derived from mutated PIK3CA","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Cancer immunotherapy,Sequencing,T cells,Tumour immunology"},"journal":{"pcode":"nm","title":"nature medicine","volume":"28","issue":"5","id":41591,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"BR","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{font-weight:700;-webkit-font-smoothing:antialiased;font-size:1.5rem;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.app-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{margin:4px 4px 0;fill:#888;height:10px;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-c9121fecab.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png">
<link rel="icon" type="image/png" sizes="48x48" href="/static/images/favicons/nature/favicon-48x48-b52890008c.png">
<link rel="icon" type="image/png" sizes="32x32" href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="/static/images/favicons/nature/favicon-16x16-951651ab72.png">
<link rel="manifest" href="/static/manifest.json" crossorigin="use-credentials">
<link rel="mask-icon" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="/static/images/favicons/nature/favicon.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-e9d2061c95.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-1cbf3a8b56.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script><script id="js-position1" data-test="global-article-js" defer="" src="/static/js/global-article-es6-bundle-c8a573ca90.js"></script><script id="js-position2" data-test="shared-js" defer="" src="/static/js/shared-es6-bundle-e9d2061c95.js"></script><script id="js-position3" data-test="header-150-js" defer="" src="/static/js/header-150-es6-bundle-5bb959eaa1.js"></script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Immunogenicity and therapeutic targeting of a public neoantigen derived from mutated PIK3CA","description":"Public neoantigens (NeoAgs) represent an elite class of shared cancer-specific epitopes derived from recurrently mutated driver genes. Here we describe a high-throughput platform combining single-cell transcriptomic and T cell receptor (TCR) sequencing to establish whether mutant PIK3CA, among the most frequently genomically altered driver oncogenes, generates an immunogenic public NeoAg. Using this strategy, we developed a panel of TCRs that recognize an endogenously processed neopeptide encompassing a common PIK3CA hotspot mutation restricted by the prevalent human leukocyte antigen (HLA)-A*03:01 allele. Mechanistically, immunogenicity to this public NeoAg arises from enhanced neopeptide/HLA complex stability caused by a preferred HLA anchor substitution. Structural studies indicated that the HLA-bound neopeptide presents a comparatively ‘featureless’ surface dominated by the peptide’s backbone. To bind this epitope with high specificity and affinity, we discovered that a lead TCR clinical candidate engages the neopeptide through an extended interface facilitated by an unusually long CDR3β loop. In patients with diverse malignancies, we observed NeoAg clonal conservation and spontaneous immunogenicity to the neoepitope. Finally, adoptive transfer of TCR-engineered T cells led to tumor regression in vivo in mice bearing PIK3CA-mutant tumors but not wild-type PIK3CA tumors. Together, these findings establish the immunogenicity and therapeutic potential of a mutant PIK3CA-derived public NeoAg. A new high-throughput platform to find rare T cells that can recognize shared cancer neoantigens identifies T cell receptors specific for a conserved, immunogenic and therapeutically actionable epitope in mutant PI3Kα, one of the most common driver oncogenes","datePublished":"2022-04-28T00:00:00Z","dateModified":"2022-04-28T00:00:00Z","pageStart":"946","pageEnd":"957","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41591-022-01786-3","keywords":["Cancer immunotherapy","Sequencing","T cells","Tumour immunology","Biomedicine","general","Cancer Research","Metabolic Diseases","Infectious Diseases","Molecular Medicine","Neurosciences"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig6_HTML.png"],"isPartOf":{"name":"Nature Medicine","issn":["1546-170X","1078-8956"],"volumeNumber":"28","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Smita S. Chandran","url":"http://orcid.org/0000-0001-9266-6878","affiliation":[{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cell Engineering, Memorial Sloan Kettering Cancer Center","address":{"name":"Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Parker Institute for Cancer Immunotherapy","address":{"name":"Parker Institute for Cancer Immunotherapy, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"chandrs1@mskcc.org","@type":"Person"},{"name":"Jiaqi Ma","url":"http://orcid.org/0000-0002-6298-0193","affiliation":[{"name":"University of Notre Dame","address":{"name":"Department of Chemistry and Biochemistry, University of Notre Dame, South Bend, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Notre Dame","address":{"name":"Harper Cancer Research Institute, University of Notre Dame, South Bend, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Martin G. Klatt","url":"http://orcid.org/0000-0001-8703-7305","affiliation":[{"name":"Molecular Pharmacology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center","address":{"name":"Molecular Pharmacology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Friederike Dündar","url":"http://orcid.org/0000-0002-2301-112X","affiliation":[{"name":"Weill Cornell Medicine","address":{"name":"Department of Physiology and Biophysics, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Weill Cornell Medicine","address":{"name":"Applied Bioinformatics Core, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Chaitanya Bandlamudi","affiliation":[{"name":"Marie-Josée and Henry R. Kravis Center for Molecular Oncology, Memorial Sloan Kettering Cancer Center","address":{"name":"Marie-Josée and Henry R. Kravis Center for Molecular Oncology, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Pedram Razavi","url":"http://orcid.org/0000-0003-4236-0576","affiliation":[{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Memorial Sloan Kettering Cancer Center","address":{"name":"Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Division of Hematology and Medical Oncology, Weill Cornell Medicine","address":{"name":"Department of Medicine, Division of Hematology and Medical Oncology, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Hannah Y. Wen","affiliation":[{"name":"Memorial Sloan Kettering Cancer Center","address":{"name":"Department of Pathology, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Britta Weigelt","url":"http://orcid.org/0000-0001-9927-1270","affiliation":[{"name":"Memorial Sloan Kettering Cancer Center","address":{"name":"Department of Pathology, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Paul Zumbo","affiliation":[{"name":"Weill Cornell Medicine","address":{"name":"Department of Physiology and Biophysics, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Weill Cornell Medicine","address":{"name":"Applied Bioinformatics Core, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Si Ning Fu","affiliation":[{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cell Engineering, Memorial Sloan Kettering Cancer Center","address":{"name":"Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Lauren B. Banks","affiliation":[{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cell Engineering, Memorial Sloan Kettering Cancer Center","address":{"name":"Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Memorial Sloan Kettering Cancer Center","address":{"name":"Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Fei Yi","affiliation":[{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cell Engineering, Memorial Sloan Kettering Cancer Center","address":{"name":"Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Enric Vercher","affiliation":[{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Inaki Etxeberria","url":"http://orcid.org/0000-0003-2713-0836","affiliation":[{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Watchain D. Bestman","affiliation":[{"name":"Memorial Sloan Kettering Cancer Center","address":{"name":"Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Memorial Sloan Kettering Cancer Center","address":{"name":"Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Arnaud Da Cruz Paula","affiliation":[{"name":"Memorial Sloan Kettering Cancer Center","address":{"name":"Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ilinca S. Aricescu","affiliation":[{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Alexander Drilon","url":"http://orcid.org/0000-0001-6806-9061","affiliation":[{"name":"Memorial Sloan Kettering Cancer Center","address":{"name":"Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Division of Hematology and Medical Oncology, Weill Cornell Medicine","address":{"name":"Department of Medicine, Division of Hematology and Medical Oncology, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Early Drug Development Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Early Drug Development Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Doron Betel","url":"http://orcid.org/0000-0002-8006-7752","affiliation":[{"name":"Weill Cornell Medicine","address":{"name":"Applied Bioinformatics Core, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Division of Hematology and Medical Oncology, Weill Cornell Medicine","address":{"name":"Department of Medicine, Division of Hematology and Medical Oncology, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Institute for Computational Biomedicine, Weill Cornell Medicine","address":{"name":"Institute for Computational Biomedicine, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"David A. Scheinberg","affiliation":[{"name":"Molecular Pharmacology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center","address":{"name":"Molecular Pharmacology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Memorial Sloan Kettering Cancer Center","address":{"name":"Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Brian M. Baker","affiliation":[{"name":"University of Notre Dame","address":{"name":"Department of Chemistry and Biochemistry, University of Notre Dame, South Bend, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Notre Dame","address":{"name":"Harper Cancer Research Institute, University of Notre Dame, South Bend, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Christopher A. Klebanoff","url":"http://orcid.org/0000-0001-9645-3896","affiliation":[{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cell Engineering, Memorial Sloan Kettering Cancer Center","address":{"name":"Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Parker Institute for Cancer Immunotherapy","address":{"name":"Parker Institute for Cancer Immunotherapy, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Memorial Sloan Kettering Cancer Center","address":{"name":"Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Division of Hematology and Medical Oncology, Weill Cornell Medicine","address":{"name":"Department of Medicine, Division of Hematology and Medical Oncology, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Early Drug Development Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Early Drug Development Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Cell Therapy Service, Memorial Sloan Kettering Cancer Center","address":{"name":"Cell Therapy Service, Memorial Sloan Kettering Cancer Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"klebanoc@mskcc.org","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41591-022-01786-3">
    
    
    <meta name="journal_id" content="41591">
    <meta name="dc.title" content="Immunogenicity and therapeutic targeting of a public neoantigen derived from mutated PIK3CA">
    <meta name="dc.source" content="Nature Medicine 2022 28:5">
    <meta name="dc.format" content="text/html">
    <meta name="dc.publisher" content="Nature Publishing Group">
    <meta name="dc.date" content="2022-04-28">
    <meta name="dc.type" content="OriginalPaper">
    <meta name="dc.language" content="En">
    <meta name="dc.copyright" content="2022 The Author(s)">
    <meta name="dc.rights" content="2022 The Author(s)">
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="dc.description" content="Public neoantigens (NeoAgs) represent an elite class of shared cancer-specific epitopes derived from recurrently mutated driver genes. Here we describe a high-throughput platform combining single-cell transcriptomic and T cell receptor (TCR) sequencing to establish whether mutant PIK3CA, among the most frequently genomically altered driver oncogenes, generates an immunogenic public NeoAg. Using this strategy, we developed a panel of TCRs that recognize an endogenously processed neopeptide encompassing a common PIK3CA hotspot mutation restricted by the prevalent human leukocyte antigen (HLA)-A*03:01 allele. Mechanistically, immunogenicity to this public NeoAg arises from enhanced neopeptide/HLA complex stability caused by a preferred HLA anchor substitution. Structural studies indicated that the HLA-bound neopeptide presents a comparatively ‘featureless’ surface dominated by the peptide’s backbone. To bind this epitope with high specificity and affinity, we discovered that a lead TCR clinical candidate engages the neopeptide through an extended interface facilitated by an unusually long CDR3β loop. In patients with diverse malignancies, we observed NeoAg clonal conservation and spontaneous immunogenicity to the neoepitope. Finally, adoptive transfer of TCR-engineered T cells led to tumor regression in vivo in mice bearing PIK3CA-mutant tumors but not wild-type PIK3CA tumors. Together, these findings establish the immunogenicity and therapeutic potential of a mutant PIK3CA-derived public NeoAg. A new high-throughput platform to find rare T cells that can recognize shared cancer neoantigens identifies T cell receptors specific for a conserved, immunogenic and therapeutically actionable epitope in mutant PI3Kα, one of the most common driver oncogenes">
    <meta name="prism.issn" content="1546-170X">
    <meta name="prism.publicationName" content="Nature Medicine">
    <meta name="prism.publicationDate" content="2022-04-28">
    <meta name="prism.volume" content="28">
    <meta name="prism.number" content="5">
    <meta name="prism.section" content="OriginalPaper">
    <meta name="prism.startingPage" content="946">
    <meta name="prism.endingPage" content="957">
    <meta name="prism.copyright" content="2022 The Author(s)">
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="prism.url" content="https://www.nature.com/articles/s41591-022-01786-3">
    <meta name="prism.doi" content="doi:10.1038/s41591-022-01786-3">
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41591-022-01786-3.pdf">
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41591-022-01786-3">
    <meta name="citation_journal_title" content="Nature Medicine">
    <meta name="citation_journal_abbrev" content="Nat Med">
    <meta name="citation_publisher" content="Nature Publishing Group">
    <meta name="citation_issn" content="1546-170X">
    <meta name="citation_title" content="Immunogenicity and therapeutic targeting of a public neoantigen derived from mutated PIK3CA">
    <meta name="citation_volume" content="28">
    <meta name="citation_issue" content="5">
    <meta name="citation_publication_date" content="2022/05">
    <meta name="citation_online_date" content="2022/04/28">
    <meta name="citation_firstpage" content="946">
    <meta name="citation_lastpage" content="957">
    <meta name="citation_article_type" content="Article">
    <meta name="citation_fulltext_world_readable" content="">
    <meta name="citation_language" content="en">
    <meta name="dc.identifier" content="doi:10.1038/s41591-022-01786-3">
    <meta name="DOI" content="10.1038/s41591-022-01786-3">
    <meta name="size" content="287167">
    <meta name="citation_doi" content="10.1038/s41591-022-01786-3">
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41591-022-01786-3&amp;api_key=">
    <meta name="description" content="Public neoantigens (NeoAgs) represent an elite class of shared cancer-specific epitopes derived from recurrently mutated driver genes. Here we describe a high-throughput platform combining single-cell transcriptomic and T cell receptor (TCR) sequencing to establish whether mutant PIK3CA, among the most frequently genomically altered driver oncogenes, generates an immunogenic public NeoAg. Using this strategy, we developed a panel of TCRs that recognize an endogenously processed neopeptide encompassing a common PIK3CA hotspot mutation restricted by the prevalent human leukocyte antigen (HLA)-A*03:01 allele. Mechanistically, immunogenicity to this public NeoAg arises from enhanced neopeptide/HLA complex stability caused by a preferred HLA anchor substitution. Structural studies indicated that the HLA-bound neopeptide presents a comparatively ‘featureless’ surface dominated by the peptide’s backbone. To bind this epitope with high specificity and affinity, we discovered that a lead TCR clinical candidate engages the neopeptide through an extended interface facilitated by an unusually long CDR3β loop. In patients with diverse malignancies, we observed NeoAg clonal conservation and spontaneous immunogenicity to the neoepitope. Finally, adoptive transfer of TCR-engineered T cells led to tumor regression in vivo in mice bearing PIK3CA-mutant tumors but not wild-type PIK3CA tumors. Together, these findings establish the immunogenicity and therapeutic potential of a mutant PIK3CA-derived public NeoAg. A new high-throughput platform to find rare T cells that can recognize shared cancer neoantigens identifies T cell receptors specific for a conserved, immunogenic and therapeutically actionable epitope in mutant PI3Kα, one of the most common driver oncogenes">
    <meta name="dc.creator" content="Chandran, Smita S.">
    <meta name="dc.creator" content="Ma, Jiaqi">
    <meta name="dc.creator" content="Klatt, Martin G.">
    <meta name="dc.creator" content="Dündar, Friederike">
    <meta name="dc.creator" content="Bandlamudi, Chaitanya">
    <meta name="dc.creator" content="Razavi, Pedram">
    <meta name="dc.creator" content="Wen, Hannah Y.">
    <meta name="dc.creator" content="Weigelt, Britta">
    <meta name="dc.creator" content="Zumbo, Paul">
    <meta name="dc.creator" content="Fu, Si Ning">
    <meta name="dc.creator" content="Banks, Lauren B.">
    <meta name="dc.creator" content="Yi, Fei">
    <meta name="dc.creator" content="Vercher, Enric">
    <meta name="dc.creator" content="Etxeberria, Inaki">
    <meta name="dc.creator" content="Bestman, Watchain D.">
    <meta name="dc.creator" content="Da Cruz Paula, Arnaud">
    <meta name="dc.creator" content="Aricescu, Ilinca S.">
    <meta name="dc.creator" content="Drilon, Alexander">
    <meta name="dc.creator" content="Betel, Doron">
    <meta name="dc.creator" content="Scheinberg, David A.">
    <meta name="dc.creator" content="Baker, Brian M.">
    <meta name="dc.creator" content="Klebanoff, Christopher A.">
    <meta name="dc.subject" content="Cancer immunotherapy">
    <meta name="dc.subject" content="Sequencing">
    <meta name="dc.subject" content="T cells">
    <meta name="dc.subject" content="Tumour immunology">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Neoantigens in cancer immunotherapy; citation_author=TN Schumacher, RD Schreiber; citation_volume=348; citation_publication_date=2015; citation_pages=69-74; citation_doi=10.1126/science.aaa4971; citation_id=CR1">
    <meta name="citation_reference" content="citation_journal_title=Cancer Discov.; citation_title=Unique neoantigens arise from somatic mutations in patients with gastrointestinal cancers; citation_author=MR Parkhurst; citation_volume=9; citation_publication_date=2019; citation_pages=1022-1035; citation_doi=10.1158/2159-8290.CD-18-1494; citation_id=CR2">
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Immunol.; citation_title=Towards personalized, tumour-specific, therapeutic vaccines for cancer; citation_author=Z Hu, PA Ott, CJ Wu; citation_volume=18; citation_publication_date=2018; citation_pages=168-182; citation_doi=10.1038/nri.2017.131; citation_id=CR3">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Clonal neoantigens elicit T cell immunoreactivity and sensitivity to immune checkpoint blockade; citation_author=N McGranahan; citation_volume=351; citation_publication_date=2016; citation_pages=1463-1469; citation_doi=10.1126/science.aaf1490; citation_id=CR4">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Genomic correlates of response to immune checkpoint blockade in microsatellite-stable solid tumors; citation_author=D Miao; citation_volume=50; citation_publication_date=2018; citation_pages=1271-1281; citation_doi=10.1038/s41588-018-0200-2; citation_id=CR5">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=UVB-induced tumor heterogeneity diminishes immune response in melanoma; citation_author=Y Wolf; citation_volume=179; citation_publication_date=2019; citation_pages=219-235 e221; citation_doi=10.1016/j.cell.2019.08.032; citation_id=CR6">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Minimal functional driver gene heterogeneity among untreated metastases; citation_author=JG Reiter; citation_volume=361; citation_publication_date=2018; citation_pages=1033-1037; citation_doi=10.1126/science.aat7171; citation_id=CR7">
    <meta name="citation_reference" content="citation_journal_title=J. Exp. Med.; citation_title=Shared cancer neoantigens: making private matters public; citation_author=CA Klebanoff, JD Wolchok; citation_volume=215; citation_publication_date=2018; citation_pages=5-7; citation_doi=10.1084/jem.20172188; citation_id=CR8">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=A p16INK4a-insensitive CDK4 mutant targeted by cytolytic T lymphocytes in a human melanoma; citation_author=T Wolfel; citation_volume=269; citation_publication_date=1995; citation_pages=1281-1284; citation_doi=10.1126/science.7652577; citation_id=CR9">
    <meta name="citation_reference" content="citation_journal_title=J. Immunol.; citation_title=A ras-mutated peptide targeted by CTL infiltrating a human melanoma lesion; citation_author=B Linard; citation_volume=168; citation_publication_date=2002; citation_pages=4802-4808; citation_doi=10.4049/jimmunol.168.9.4802; citation_id=CR10">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A vaccine targeting mutant IDH1 induces antitumour immunity; citation_author=T Schumacher; citation_volume=512; citation_publication_date=2014; citation_pages=324-327; citation_doi=10.1038/nature13387; citation_id=CR11">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Immunogenicity of somatic mutations in human gastrointestinal cancers; citation_author=E Tran; citation_volume=350; citation_publication_date=2015; citation_pages=1387-1390; citation_doi=10.1126/science.aad1253; citation_id=CR12">
    <meta name="citation_reference" content="citation_journal_title=J. Exp. Med.; citation_title=Novel and shared neoantigen derived from histone 3 variant H3.3K27M mutation for glioma T cell therapy; citation_author=ZS Chheda; citation_volume=215; citation_publication_date=2018; citation_pages=141-157; citation_doi=10.1084/jem.20171046; citation_id=CR13">
    <meta name="citation_reference" content="citation_journal_title=J. Clin. Invest.; citation_title=Tumor-infiltrating BRAFV600E-specific CD4+ T cells correlated with complete clinical response in melanoma; citation_author=JR Veatch; citation_volume=128; citation_publication_date=2018; citation_pages=1563-1568; citation_doi=10.1172/JCI98689; citation_id=CR14">
    <meta name="citation_reference" content="citation_journal_title=J. Clin. Invest.; citation_title=Neoantigen screening identifies broad TP53 mutant immunogenicity in patients with epithelial cancers; citation_author=P Malekzadeh; citation_volume=129; citation_publication_date=2019; citation_pages=1109-1114; citation_doi=10.1172/JCI123791; citation_id=CR15">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Targeting a neoantigen derived from a common TP53 mutation; citation_author=EH Hsiue; citation_volume=371; citation_publication_date=2021; citation_pages=eabc8697; citation_doi=10.1126/science.abc8697; citation_id=CR16">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=High frequency of mutations of the PIK3CA gene in human cancers; citation_author=Y Samuels; citation_volume=304; citation_publication_date=2004; citation_pages=554; citation_doi=10.1126/science.1096502; citation_id=CR17">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Comprehensive characterization of cancer driver genes and mutations; citation_author=MH Bailey; citation_volume=173; citation_publication_date=2018; citation_pages=371-385; citation_doi=10.1016/j.cell.2018.02.060; citation_id=CR18">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Pan-cancer whole-genome analyses of metastatic solid tumours; citation_author=P Priestley; citation_volume=575; citation_publication_date=2019; citation_pages=210-216; citation_doi=10.1038/s41586-019-1689-y; citation_id=CR19">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Alpelisib for PIK3CA-mutated, hormone receptor-positive advanced breast cancer; citation_author=F Andre; citation_volume=380; citation_publication_date=2019; citation_pages=1929-1940; citation_doi=10.1056/NEJMoa1813904; citation_id=CR20">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=COSMIC: the Catalogue of Somatic Mutations in Cancer; citation_author=JG Tate; citation_volume=47; citation_publication_date=2019; citation_pages=D941-D947; citation_doi=10.1093/nar/gky1015; citation_id=CR21">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=Allele frequency net database (AFND) 2020 update: gold-standard data classification, open access genotype data and new query tools; citation_author=FF Gonzalez-Galarza; citation_volume=48; citation_publication_date=2020; citation_pages=D783-D788; citation_id=CR22">
    <meta name="citation_reference" content="citation_journal_title=BMC Immunol.; citation_title=HLA class I supertypes: a revised and updated classification; citation_author=J Sidney, B Peters, N Frahm, C Brander, A Sette; citation_volume=9; citation_publication_date=2008; citation_pages=1; citation_doi=10.1186/1471-2172-9-1; citation_id=CR23">
    <meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.; citation_title=Different T cell receptor affinity thresholds and CD8 coreceptor dependence govern cytotoxic T lymphocyte activation and tetramer binding properties; citation_author=B Laugel; citation_volume=282; citation_publication_date=2007; citation_pages=23799-23810; citation_doi=10.1074/jbc.M700976200; citation_id=CR24">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Gene therapy with human and mouse T-cell receptors mediates cancer regression and targets normal tissues expressing cognate antigen; citation_author=LA Johnson; citation_volume=114; citation_publication_date=2009; citation_pages=535-546; citation_doi=10.1182/blood-2009-03-211714; citation_id=CR25">
    <meta name="citation_reference" content="Lu, Y. C. et al. Treatment of patients with metastatic cancer using a major histocompatibility complex class II-restricted T-cell receptor targeting the cancer germline antigen MAGE-A3. J. Clin. Oncol. 35, 3322–3329 (2017).">
    <meta name="citation_reference" content="citation_journal_title=Cancer Discov.; citation_title=Antitumor activity associated with prolonged persistence of adoptively transferred NY-ESO-1 (c259)T cells in synovial sarcoma; citation_author=SP D’Angelo; citation_volume=8; citation_publication_date=2018; citation_pages=944-957; citation_doi=10.1158/2159-8290.CD-17-1417; citation_id=CR27">
    <meta name="citation_reference" content="Nagarsheth, N. B. et al. TCR-engineered T cells targeting E7 for patients with metastatic HPV-associated epithelial cancers. Nat. Med. 27, 419–425 (2021).">
    <meta name="citation_reference" content="Reynisson, B., Alvarez, B., Paul, S., Peters, B. &amp; Nielsen, M. NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. Nucleic Acids Res. 48, W449–W454 (2020).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=A large peptidome dataset improves HLA class I epitope prediction across most of the human population; citation_author=S Sarkizova; citation_volume=38; citation_publication_date=2020; citation_pages=199-209; citation_doi=10.1038/s41587-019-0322-9; citation_id=CR30">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=How the T cell receptor sees antigen—a structural view; citation_author=KC Garcia, EJ Adams; citation_volume=122; citation_publication_date=2005; citation_pages=333-336; citation_doi=10.1016/j.cell.2005.07.015; citation_id=CR31">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=TCR3d: the T cell receptor structural repertoire database; citation_author=R Gowthaman, BG Pierce; citation_volume=35; citation_publication_date=2019; citation_pages=5323-5325; citation_doi=10.1093/bioinformatics/btz517; citation_id=CR32">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=VDJdb in 2019: database extension, new analysis infrastructure and a T-cell receptor motif compendium; citation_author=DV Bagaev; citation_volume=48; citation_publication_date=2020; citation_pages=D1057-D1062; citation_doi=10.1093/nar/gkz874; citation_id=CR33">
    <meta name="citation_reference" content="citation_journal_title=Eur. J. Immunol.; citation_title=Different affinity windows for virus and cancer-specific T-cell receptors: implications for therapeutic strategies; citation_author=M Aleksic; citation_volume=42; citation_publication_date=2012; citation_pages=3174-3179; citation_doi=10.1002/eji.201242606; citation_id=CR34">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Structural basis for oligoclonal T cell recognition of a shared p53 cancer neoantigen; citation_author=D Wu, DT Gallagher, R Gowthaman, BG Pierce, RA Mariuzza; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-16755-y; citation_id=CR35">
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=High-affinity oligoclonal TCRs define effective adoptive T cell therapy targeting mutant KRAS-G12D; citation_author=MJW Sim; citation_volume=117; citation_publication_date=2020; citation_pages=12826-12835; citation_doi=10.1073/pnas.1921964117; citation_id=CR36">
    <meta name="citation_reference" content="citation_journal_title=Nat. Chem. Biol.; citation_title=Structural dissimilarity from self drives neoepitope escape from immune tolerance; citation_author=JR Devlin; citation_volume=16; citation_publication_date=2020; citation_pages=1269-1276; citation_doi=10.1038/s41589-020-0610-1; citation_id=CR37">
    <meta name="citation_reference" content="citation_journal_title=Proteins; citation_title=Geometrical characterization of T cell receptor binding modes reveals class-specific binding to maximize access to antigen; citation_author=NK Singh; citation_volume=88; citation_publication_date=2020; citation_pages=503-513; citation_doi=10.1002/prot.25829; citation_id=CR38">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=ScanProsite: detection of PROSITE signature matches and ProRule-associated functional and structural residues in proteins; citation_author=E Castro; citation_volume=34; citation_publication_date=2006; citation_pages=W362-W365; citation_doi=10.1093/nar/gkl124; citation_id=CR39">
    <meta name="citation_reference" content="citation_journal_title=J. Exp. Med.; citation_title=Removal of homeostatic cytokine sinks by lymphodepletion enhances the efficacy of adoptively transferred tumor-specific CD8+ T cells; citation_author=L Gattinoni; citation_volume=202; citation_publication_date=2005; citation_pages=907-912; citation_doi=10.1084/jem.20050732; citation_id=CR40">
    <meta name="citation_reference" content="citation_journal_title=Cancer Cell; citation_title=Mutant PIK3CA promotes cell growth and invasion of human cancer cells; citation_author=Y Samuels; citation_volume=7; citation_publication_date=2005; citation_pages=561-573; citation_doi=10.1016/j.ccr.2005.05.014; citation_id=CR41">
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Phosphatidylinositol 3-kinase mutations identified in human cancer are oncogenic; citation_author=S Kang, AG Bader, PK Vogt; citation_volume=102; citation_publication_date=2005; citation_pages=802-807; citation_doi=10.1073/pnas.0408864102; citation_id=CR42">
    <meta name="citation_reference" content="citation_journal_title=J. Mol. Diagn.; citation_title=Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (MSK-IMPACT): a hybridization capture-based next-generation sequencing clinical assay for solid tumor molecular oncology; citation_author=DT Cheng; citation_volume=17; citation_publication_date=2015; citation_pages=251-264; citation_doi=10.1016/j.jmoldx.2014.12.006; citation_id=CR43">
    <meta name="citation_reference" content="citation_journal_title=Clin. Cancer Res.; citation_title=HLA genotyping in synovial sarcoma: identifying HLA-A*02 and its association with clinical outcome; citation_author=E Rosenbaum; citation_volume=26; citation_publication_date=2020; citation_pages=5448-5455; citation_doi=10.1158/1078-0432.CCR-20-0832; citation_id=CR44">
    <meta name="citation_reference" content="citation_journal_title=J. Am. Med. Inform. Assoc.; citation_title=Automated eligibility screening and monitoring for genotype-driven precision oncology trials; citation_author=MH Eubank; citation_volume=23; citation_publication_date=2016; citation_pages=777-781; citation_doi=10.1093/jamia/ocw020; citation_id=CR45">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Neoantigen-directed immune escape in lung cancer evolution; citation_author=R Rosenthal; citation_volume=567; citation_publication_date=2019; citation_pages=479-485; citation_doi=10.1038/s41586-019-1032-7; citation_id=CR46">
    <meta name="citation_reference" content="citation_journal_title=Immunol. Rev.; citation_title=T cell receptor-based cancer immunotherapy: emerging efficacy and pathways of resistance; citation_author=SS Chandran, CA Klebanoff; citation_volume=290; citation_publication_date=2019; citation_pages=127-147; citation_doi=10.1111/imr.12772; citation_id=CR47">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Allele-specific HLA loss and immune escape in lung cancer evolution; citation_author=N McGranahan; citation_volume=171; citation_publication_date=2017; citation_pages=1259-1271; citation_doi=10.1016/j.cell.2017.10.001; citation_id=CR48">
    <meta name="citation_reference" content="citation_journal_title=Cancer Discov.; citation_title=Somatic HLA class I loss is a widespread mechanism of immune evasion which refines the use of tumor mutational burden as a biomarker of checkpoint inhibitor response; citation_author=M Montesion; citation_volume=11; citation_publication_date=2021; citation_pages=282-292; citation_doi=10.1158/2159-8290.CD-20-0672; citation_id=CR49">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Mutations associated with acquired resistance to PD-1 blockade in melanoma; citation_author=JM Zaretsky; citation_volume=375; citation_publication_date=2016; citation_pages=819-829; citation_doi=10.1056/NEJMoa1604958; citation_id=CR50">
    <meta name="citation_reference" content="citation_journal_title=Cancer Discov.; citation_title=Primary resistance to PD-1 blockade mediated by JAK1/2 mutations; citation_author=DS Shin; citation_volume=7; citation_publication_date=2017; citation_pages=188-201; citation_doi=10.1158/2159-8290.CD-16-1223; citation_id=CR51">
    <meta name="citation_reference" content="citation_journal_title=Nat. Protoc.; citation_title=Antigen-specific activation and cytokine-facilitated expansion of naive, human CD8+ T cells; citation_author=M Wolfl, PD Greenberg; citation_volume=9; citation_publication_date=2014; citation_pages=950-966; citation_doi=10.1038/nprot.2014.064; citation_id=CR52">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Antigen identification for orphan T cell receptors expressed on tumor-infiltrating lymphocytes; citation_author=MH Gee; citation_volume=172; citation_publication_date=2018; citation_pages=549-563; citation_doi=10.1016/j.cell.2017.11.043; citation_id=CR53">
    <meta name="citation_reference" content="citation_journal_title=Nat. Protoc.; citation_title=Induction of neoantigen-reactive T cells from healthy donors; citation_author=M Ali; citation_volume=14; citation_publication_date=2019; citation_pages=1926-1943; citation_doi=10.1038/s41596-019-0170-6; citation_id=CR54">
    <meta name="citation_reference" content="citation_journal_title=Cell Rep.; citation_title=Sensitive detection and analysis of neoantigen-specific T cell populations from tumors and blood; citation_author=S Peng; citation_volume=28; citation_publication_date=2019; citation_pages=2728-2738; citation_doi=10.1016/j.celrep.2019.07.106; citation_id=CR55">
    <meta name="citation_reference" content="Arnaud, M. et al. Sensitive identification of neoantigens and cognate TCRs in human solid tumors. Nat. Biotechnol. 
                  https://doi.org/10.1038/s41587-021-01072-6
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Cell Rep. Med.; citation_title=Prediction of neo-epitope immunogenicity reveals TCR recognition determinants and provides insight into immunoediting; citation_author=J Schmidt; citation_volume=2; citation_publication_date=2021; citation_pages=100194; citation_doi=10.1016/j.xcrm.2021.100194; citation_id=CR57">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Acquired cancer resistance to combination immunotherapy from transcriptional loss of class I HLA; citation_author=KG Paulson; citation_volume=9; citation_publication_date=2018; citation_doi=10.1038/s41467-018-06300-3; citation_id=CR58">
    <meta name="citation_reference" content="citation_journal_title=Cancer Discov.; citation_title=Safety and antitumor activity of the multitargeted pan-TRK, ROS1, and ALK inhibitor entrectinib: combined results from two phase I trials (ALKA-372-001 and STARTRK-1); citation_author=A Drilon; citation_volume=7; citation_publication_date=2017; citation_pages=400-409; citation_doi=10.1158/2159-8290.CD-16-1237; citation_id=CR59">
    <meta name="citation_reference" content="citation_journal_title=J. Clin. Oncol.; citation_title=Efficacy of pembrolizumab in patients with noncolorectal high microsatellite instability/mismatch repair-deficient cancer: results from the phase II KEYNOTE-158 study; citation_author=A Marabelle; citation_volume=38; citation_publication_date=2020; citation_pages=1-10; citation_doi=10.1200/JCO.19.02105; citation_id=CR60">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Landscape of immunogenic tumor antigens in successful immunotherapy of virally induced epithelial cancer; citation_author=S Stevanovic; citation_volume=356; citation_publication_date=2017; citation_pages=200-205; citation_doi=10.1126/science.aak9510; citation_id=CR61">
    <meta name="citation_reference" content="citation_journal_title=Cancer Res.; citation_title=Enhanced antitumor activity of murine-human hybrid T-cell receptor (TCR) in human lymphocytes is associated with improved pairing and TCR/CD3 stability; citation_author=CJ Cohen, Y Zhao, Z Zheng, SA Rosenberg, RA Morgan; citation_volume=66; citation_publication_date=2006; citation_pages=8878-8886; citation_doi=10.1158/0008-5472.CAN-06-1450; citation_id=CR62">
    <meta name="citation_reference" content="citation_journal_title=JCI Insight; citation_title=Solving an MHC allele-specific bias in the reported immunopeptidome; citation_author=MG Klatt; citation_volume=5; citation_publication_date=2020; citation_pages=e141264; citation_doi=10.1172/jci.insight.141264; citation_id=CR63">
    <meta name="citation_reference" content="citation_journal_title=Sci. Transl. Med.; citation_title=Identification of a Titin-derived HLA-A1-presented peptide as a cross-reactive target for engineered MAGE A3-directed T cells; citation_author=BJ Cameron; citation_volume=5; citation_publication_date=2013; citation_pages=197ra103; citation_doi=10.1126/scitranslmed.3006034; citation_id=CR64">
    <meta name="citation_reference" content="citation_journal_title=J. Clin. Invest.; citation_title=Influenza-specific lung-resident memory T cells are proliferative and polyfunctional and maintain diverse TCR profiles; citation_author=A Pizzolla; citation_volume=128; citation_publication_date=2018; citation_pages=721-733; citation_doi=10.1172/JCI96957; citation_id=CR65">
    <meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.; citation_title=Two different T cell receptors use different thermodynamic strategies to recognize the same peptide/MHC ligand; citation_author=RL Davis-Harrison, KM Armstrong, BM Baker; citation_volume=346; citation_publication_date=2005; citation_pages=533-550; citation_doi=10.1016/j.jmb.2004.11.063; citation_id=CR66">
    <meta name="citation_reference" content="citation_journal_title=Biochemistry; citation_title=An engineered T cell receptor variant realizes the limits of functional binding modes; citation_author=NK Singh; citation_volume=59; citation_publication_date=2020; citation_pages=4163-4175; citation_doi=10.1021/acs.biochem.0c00689; citation_id=CR67">
    <meta name="citation_reference" content="citation_journal_title=J. Immunol. Methods; citation_title=Differential scanning fluorimetry based assessments of the thermal and kinetic stability of peptide-MHC complexes; citation_author=LM Hellman; citation_volume=432; citation_publication_date=2016; citation_pages=95-101; citation_doi=10.1016/j.jim.2016.02.016; citation_id=CR68">
    <meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.; citation_title=Strategic mutations in the class I major histocompatibility complex HLA-A2 independently affect both peptide binding and T cell receptor recognition; citation_author=TK Baxter; citation_volume=279; citation_publication_date=2004; citation_pages=29175-29184; citation_doi=10.1074/jbc.M403372200; citation_id=CR69">
    <meta name="citation_reference" content="citation_journal_title=Acta Crystallogr. D. Biol. Crystallogr.; citation_title=Structure of HLA-A*0301 in complex with a peptide of proteolipid protein: insights into the role of HLA-A alleles in susceptibility to multiple sclerosis; citation_author=RM McMahon; citation_volume=67; citation_publication_date=2011; citation_pages=447-454; citation_doi=10.1107/S0907444911007888; citation_id=CR70">
    <meta name="citation_reference" content="citation_journal_title=J. Biol. Chem.; citation_title=A molecular basis for the interplay between T cells, viral mutants, and human leukocyte antigen micropolymorphism; citation_author=YC Liu; citation_volume=289; citation_publication_date=2014; citation_pages=16688-16698; citation_doi=10.1074/jbc.M114.563502; citation_id=CR71">
    <meta name="citation_reference" content="citation_journal_title=J. Mol. Biol.; citation_title=Disparate degrees of hypervariable loop flexibility control T-cell receptor cross-reactivity, specificity, and binding mechanism; citation_author=DR Scott, OY Borbulevych, KH Piepenbrink, SA Corcelli, BM Baker; citation_volume=414; citation_publication_date=2011; citation_pages=385-400; citation_doi=10.1016/j.jmb.2011.10.006; citation_id=CR72">
    <meta name="citation_reference" content="citation_journal_title=Sci. Transl. Med.; citation_title=Clonal status of actionable driver events and the timing of mutational processes in cancer evolution; citation_author=N McGranahan; citation_volume=7; citation_publication_date=2015; citation_pages=283ra254; citation_doi=10.1126/scitranslmed.aaa1408; citation_id=CR73">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing; citation_author=R Shen, VE Seshan; citation_volume=44; citation_publication_date=2016; citation_pages=e131; citation_doi=10.1093/nar/gkw520; citation_id=CR74">
    <meta name="citation_reference" content="citation_journal_title=Cancer Cell; citation_title=Widespread selection for oncogenic mutant allele imbalance in cancer; citation_author=CM Bielski; citation_volume=34; citation_publication_date=2018; citation_pages=852-862; citation_doi=10.1016/j.ccell.2018.10.003; citation_id=CR75">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Comprehensive analysis of cancer-associated somatic mutations in class I HLA genes; citation_author=SA Shukla; citation_volume=33; citation_publication_date=2015; citation_pages=1152-1158; citation_doi=10.1038/nbt.3344; citation_id=CR76">
    <meta name="citation_author" content="Chandran, Smita S.">
    <meta name="citation_author_institution" content="Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Parker Institute for Cancer Immunotherapy, New York, USA">
    <meta name="citation_author" content="Ma, Jiaqi">
    <meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Notre Dame, South Bend, USA">
    <meta name="citation_author_institution" content="Harper Cancer Research Institute, University of Notre Dame, South Bend, USA">
    <meta name="citation_author" content="Klatt, Martin G.">
    <meta name="citation_author_institution" content="Molecular Pharmacology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Dündar, Friederike">
    <meta name="citation_author_institution" content="Department of Physiology and Biophysics, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author_institution" content="Applied Bioinformatics Core, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author" content="Bandlamudi, Chaitanya">
    <meta name="citation_author_institution" content="Marie-Josée and Henry R. Kravis Center for Molecular Oncology, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Razavi, Pedram">
    <meta name="citation_author_institution" content="Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Division of Hematology and Medical Oncology, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author" content="Wen, Hannah Y.">
    <meta name="citation_author_institution" content="Department of Pathology, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Weigelt, Britta">
    <meta name="citation_author_institution" content="Department of Pathology, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Zumbo, Paul">
    <meta name="citation_author_institution" content="Department of Physiology and Biophysics, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author_institution" content="Applied Bioinformatics Core, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author" content="Fu, Si Ning">
    <meta name="citation_author_institution" content="Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Banks, Lauren B.">
    <meta name="citation_author_institution" content="Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Yi, Fei">
    <meta name="citation_author_institution" content="Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Vercher, Enric">
    <meta name="citation_author_institution" content="Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Etxeberria, Inaki">
    <meta name="citation_author_institution" content="Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Bestman, Watchain D.">
    <meta name="citation_author_institution" content="Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Da Cruz Paula, Arnaud">
    <meta name="citation_author_institution" content="Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Aricescu, Ilinca S.">
    <meta name="citation_author_institution" content="Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Drilon, Alexander">
    <meta name="citation_author_institution" content="Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Division of Hematology and Medical Oncology, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author_institution" content="Early Drug Development Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Betel, Doron">
    <meta name="citation_author_institution" content="Applied Bioinformatics Core, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Division of Hematology and Medical Oncology, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author_institution" content="Institute for Computational Biomedicine, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author" content="Scheinberg, David A.">
    <meta name="citation_author_institution" content="Molecular Pharmacology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author" content="Baker, Brian M.">
    <meta name="citation_author_institution" content="Department of Chemistry and Biochemistry, University of Notre Dame, South Bend, USA">
    <meta name="citation_author_institution" content="Harper Cancer Research Institute, University of Notre Dame, South Bend, USA">
    <meta name="citation_author" content="Klebanoff, Christopher A.">
    <meta name="citation_author_institution" content="Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Parker Institute for Cancer Immunotherapy, New York, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Division of Hematology and Medical Oncology, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author_institution" content="Early Drug Development Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="citation_author_institution" content="Cell Therapy Service, Memorial Sloan Kettering Cancer Center, New York, USA">
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">
    <meta name="twitter:site" content="@naturemedicine">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image:alt" content="Content cover image">
    <meta name="twitter:title" content="Immunogenicity and therapeutic targeting of a public neoantigen derived from mutated PIK3CA">
    <meta name="twitter:description" content="Nature Medicine - A new high-throughput platform to find rare T cells that can recognize shared cancer neoantigens identifies T cell receptors specific for a conserved, immunogenic and...">
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig1_HTML.png">
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41591-022-01786-3">
    <meta property="og:type" content="article">
    <meta property="og:site_name" content="Nature">
    <meta property="og:title" content="Immunogenicity and therapeutic targeting of a public neoantigen derived from mutated PIK3CA - Nature Medicine">
    <meta property="og:description" content="A new high-throughput platform to find rare T cells that can recognize shared cancer neoantigens identifies T cell receptors specific for a conserved, immunogenic and therapeutically actionable epitope in mutant PI3Kα, one of the most common driver oncogenes">
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig1_HTML.png">
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
<style type="text/css">.cc-banner{background-color:#01324b;border:none!important;bottom:0;box-sizing:border-box;color:#fff!important;left:0;line-height:1.3;margin:auto 0 0;max-width:100%;outline:0;overflow:visible;padding:0;position:fixed;right:0;width:100%;z-index:99999}.cc-banner::backdrop{background-color:#0000004d}.cc-banner *{color:inherit!important}.cc-banner:focus{box-shadow:none;outline:0}.cc-banner a{color:#fff!important;text-decoration:underline}.cc-banner a:active,.cc-banner a:focus,.cc-banner a:hover{color:inherit;text-decoration:none}.cc-banner a:focus{outline:3px solid #08c!important}.cc-banner h2,.cc-banner h3,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-banner .cc-h2,.cc-banner h2{font-size:18px}.cc-banner .cc-h3,.cc-banner h3{font-size:16px}.cc-banner .cc-h4,.cc-banner .cc-h5,.cc-banner .cc-h6,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-size:14px}.cc-banner .cc-button{font-size:16px}.cc-banner__content{background-color:#01324b;display:flex;flex-direction:column;margin:0 auto;max-height:90vh;max-width:100%;padding:16px;position:relative}.cc-banner__content:focus{outline:0}@media (min-width:680px){.cc-banner__content{padding:12px}}@media (min-width:980px){.cc-banner__content{max-height:60vh;padding-bottom:20px;padding-top:20px}}@media (min-width:1320px){.cc-banner__content{padding-bottom:40px;padding-top:40px}}.cc-banner__container{display:flex;flex-direction:column;margin:auto;max-width:1320px;overflow:auto}.cc-banner__title{background:none!important;border:0;flex-shrink:0;font-size:18px!important;font-size:22px!important;font-weight:700!important;letter-spacing:normal;margin:0 0 12px!important;padding:0}@media (min-width:680px){.cc-banner__title{font-size:24px!important;margin:0 0 16px!important}}@media (min-width:1320px){.cc-banner__title{font-size:26px!important;margin:0 0 24px!important}}.cc-banner__body{display:flex;flex-direction:column;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif!important;overflow-x:hidden;overflow-y:auto;padding:3px 3px 16px}@media (min-width:980px){.cc-banner__body{flex-direction:row}}.cc-banner__policy p{font-size:16px!important;margin:0;max-width:none;padding:0}.cc-banner__policy p:not(:last-child){margin:0 0 16px}@media (min-width:980px){.cc-banner__policy p:not(:last-child){margin:0 0 24px}}.cc-banner__policy p a{font-size:16px!important;font-weight:700}.cc-banner__footer{box-shadow:none;flex-shrink:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif !important!important;margin:0;padding:12px 0 3px 3px;position:relative}@media (min-width:380px){.cc-banner__footer{align-items:stretch;display:flex;flex-wrap:wrap}}@media (min-width:680px){.cc-banner__footer{box-shadow:none;flex-wrap:nowrap;width:40%}}.cc-banner__footer .cc-banner__button{flex:1 1 auto;font-weight:700;overflow:hidden;padding:.5em 1em;width:100%}@media (min-width:680px){.cc-banner__footer .cc-banner__button{max-width:275px}}.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:12px}@media (min-width:680px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:0;margin-right:16px}}@media (min-width:980px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-right:24px}}@media (min-width:680px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:16px!important;margin-right:0!important}}@media (min-width:980px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:24px!important;margin-right:0!important}}.cc-banner__button-preferences{padding-left:0;padding-right:0}@media (min-width:380px){.cc-banner__button-preferences{flex:0 0 auto;margin:auto}}@media (min-width:680px){.cc-banner__button-preferences{margin:0}}@media (min-width:380px) and (max-width:680px){.cc-banner__button-break{display:block}}@media (min-width:680px){.cc-banner--is-tcf .cc-banner__footer{width:auto}}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__footer{padding-right:48px;width:66%}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px!important}.cc-banner--is-tcf .cc-banner__stacks{margin:16px 0 0}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__stacks{display:flex;flex:0 0 33%;flex-direction:column;margin:0 0 0 48px;overflow:auto}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px;margin:0 0 12px}.cc-banner--is-tcf .cc-banner__stacks-details{padding:0 3px}.cc-banner--is-tcf .cc-banner .cc-details{border-color:#fff3!important}.cc-box{border-radius:10px;padding:12px}.cc-box--info{background-color:#eff6fb}.cc-box--light{background-color:#faf9f6}.cc-button{border:2px solid #0000;border-radius:32px!important;color:inherit;cursor:pointer;font-size:14px;font-weight:700!important;left:auto;line-height:1.2;margin:0;padding:.5em 1em;right:auto;text-transform:none!important;transition:all .2s}@media (min-width:680px){.cc-button{font-size:14px;padding:.75em 1em}}.cc-button--sm{font-size:12px;padding:2px 8px!important}.cc-button--primary{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--primary:focus{outline:3px solid #08c}.cc-button--primary:focus,.cc-button--primary:hover,.cc-button--secondary{background-color:#fff!important;border-color:#025e8d!important;color:#025e8d!important}.cc-button--secondary:focus{background-color:#025e8d!important;border-color:#fff!important;color:#fff!important;outline:3px solid #08c!important}.cc-button--secondary:hover{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--secondary:active{border:2px solid #01324b!important}.cc-button--secondary.cc-button--contrast{border-color:#fff!important}.cc-button--secondary.cc-button--contrast:hover{background-color:initial!important;color:#fff!important}.cc-button--tertiary{background-color:#f0f7fc!important;background-image:none;border:2px solid #f0f7fc;color:#025e8d!important;text-decoration:underline!important;text-underline-offset:.25em}.cc-button--tertiary:focus{outline:3px solid #08c!important}.cc-button--tertiary:hover{text-decoration-thickness:.25em}.cc-button--tertiary:active{color:#013c5b}.cc-button--link{background-color:initial!important;border-color:#0000!important;border-radius:0!important;color:inherit!important;padding:0!important;text-decoration:underline!important;width:inherit!important}.cc-button--link:focus{outline:3px solid #08c}.cc-button--link:hover{background-color:initial!important;box-shadow:none;text-decoration:none}.cc-button--text{border-radius:0;padding:0}.cc-button--details{padding-right:24px!important;position:relative;width:auto}.cc-button--details:after,.cc-button--details:before{background-color:currentColor;content:"";position:absolute;transition:transform .25s ease-out}.cc-button--details:before{height:10px;margin-top:-5px;right:12px;top:50%;width:2px}.cc-button--details:after{height:2px;margin-top:-1px;right:8px;top:50%;width:10px}.cc-button--details.cc-active:before{transform:rotate(90deg);transform-origin:center}.cc-button--details.cc-active:after{display:none}.cc-details{border-bottom:1px solid #0000001a;padding:12px 0;position:relative;width:100%}.cc-details__summary{align-items:center;display:flex;font-size:14px;font-weight:700;list-style-type:none}.cc-details__summary:focus,.cc-details__summary:focus-visible{outline:3px solid #08c!important;will-change:auto}.cc-details__title{align-items:baseline;display:flex}.cc-details__title h2,.cc-details__title h3,.cc-details__title h4,.cc-details__title h5{line-height:1.4;margin:0!important;padding:0!important}.cc-details__title svg{flex-shrink:0;height:auto;margin-right:8px;position:relative;top:-1px;transition:all .2s;width:auto}.cc-details[open] .cc-details__title svg{top:2px;transform:rotate(90deg)}.cc-details__switch{margin-left:auto}.cc-details__section{padding:16px 0 0 18px}.cc-details__section p{margin:0}.cc-details__section p:not(:last-child){margin-bottom:12px}details summary::-webkit-details-marker{display:none}.cc-radio{align-items:center;display:flex;position:relative}.cc-radio *{cursor:pointer}.cc-radio__input{height:22px;left:0;position:absolute;top:0;width:22px}.cc-radio__input:focus{outline:none}.cc-radio__label{color:inherit;font-size:14px;font-weight:700;line-height:23px;margin:0;padding-left:28px}.cc-radio__label:after,.cc-radio__label:before{background-color:#fff;content:"";display:block;position:absolute;transition:transform .25s ease-out}.cc-radio__label:before{border:1px solid #777;border-radius:50%;height:22px;left:0;top:0;width:22px}.cc-radio__label:after{border:7px solid #025e8d;border-radius:50%;height:0;left:4px;opacity:0;top:4px;width:0}.cc-radio__input:focus+.cc-radio__label:before{box-shadow:0 0 0 2px #08c;outline:none}.cc-radio__label--hidden{display:none}.cc-radio__input:checked+.cc-radio__label:after{opacity:1}.cc-radio__input:disabled{cursor:default}.cc-radio__input:disabled+.cc-radio__label{cursor:default;opacity:.5}.cc-switch *{cursor:pointer}.cc-switch{align-items:center;display:flex;line-height:1}.cc-switch__label{cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;-webkit-user-select:none;user-select:none}.cc-switch__input[type=checkbox]{-webkit-appearance:none;appearance:none;background-color:#fff!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='rgba(0, 0, 0, 0.25)'/%3E%3C/svg%3E")!important;background-position:0!important;background-repeat:no-repeat!important;border:1px solid #777!important;border-radius:16px!important;height:18px!important;margin:0 8px 0 0!important;transition:background-position .2s ease-in-out!important;width:32px!important}.cc-switch__input[type=checkbox]:focus{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%2380b3cc'/%3E%3C/svg%3E")!important}.cc-switch__input[type=checkbox]:checked{background-color:#025e8d!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%23fff'/%3E%3C/svg%3E")!important;background-position:100%!important}.cc-switch__input[type=checkbox]:checked~.cc-switch__label>.cc-switch__text-off,.cc-switch__input[type=checkbox]:not(:checked)~.cc-switch__label>.cc-switch__text-on{display:none}.cc-switch__input[type=checkbox]:hover{cursor:pointer;outline:none}.cc-switch__input[type=checkbox]:focus-visible{cursor:pointer;outline:3px solid #fc0}.cc-switch__input[type=checkbox]:checked:before{background-color:none!important;color:#0000!important}html[dir=rtl] .cc-switch__input[type=checkbox]{margin-left:8px!important;margin-right:0!important}.cc-list>ul,ul.cc-list{list-style-type:disc;margin-left:0;padding-left:0}.cc-list>ul>li,ul.cc-list>li{margin-left:1em}.cc-list>ul>li:not(:last-child),ul.cc-list>li:not(:last-child){margin-bottom:4px}.cc-list__title{margin-bottom:8px!important}.cc-list--inline>ul,ul.cc-list--inline{display:flex;flex-wrap:wrap;list-style-type:none}.cc-list--inline>ul>li,ul.cc-list--inline>li{margin:0 .5em .5em 0}.cc-overlay{background-color:#0000004d;z-index:99998}.cc-overlay,.cc-preferences{bottom:0;left:0;position:fixed;right:0;top:0}.cc-preferences{background-color:#050a14f2!important;border:0;box-sizing:border-box;color:#111;font-family:sans-serif!important;line-height:1.4;margin:auto;max-height:100vh;overflow:auto;padding:0;z-index:100000}.cc-preferences:focus{outline:none}.cc-preferences *,.cc-preferences :after,.cc-preferences :before{box-sizing:inherit!important}.cc-preferences h2,.cc-preferences h3,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-preferences .cc-h2,.cc-preferences h2{font-size:18px}.cc-preferences .cc-h3,.cc-preferences h3{font-size:16px}.cc-preferences .cc-h4,.cc-preferences .cc-h5,.cc-preferences .cc-h6,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-size:14px}.cc-preferences a{color:#025e8d;text-decoration:underline}.cc-preferences a:hover{color:inherit;text-decoration:none}.cc-preferences h3{background:none;color:#111;padding:0;text-transform:none}dialog.cc-preferences{background-color:initial}dialog.cc-preferences::backdrop{background-color:#000000e6}.cc-preferences__dialog{display:flex;flex-direction:column;margin:auto;max-height:100vh;max-width:860px;padding:12px;position:relative}.cc-preferences__dialog>:last-child{border-bottom-left-radius:10px;border-bottom-right-radius:10px}@media (min-width:980px){.cc-preferences__dialog{padding:16px}}.cc-preferences__close{background:#0000!important;border:1px solid #ececec;border-radius:50%;color:#111!important;cursor:pointer;font-family:Times New Roman,serif;font-size:40px;height:40px;left:auto;line-height:1;margin-top:-20px;padding:0!important;position:absolute;right:20px;top:50%;width:40px}.cc-preferences__close:focus{outline:3px solid #08c}.cc-preferences__close-label{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__header{background:#fff!important;border-bottom:1px solid #0000001a;border-top-left-radius:10px;border-top-right-radius:10px;padding:16px;position:relative;text-align:center}.cc-preferences__title{background:none!important;color:#111!important;font-family:sans-serif!important;font-size:18px!important;font-weight:700!important;margin:0!important;padding:0 16px 0 0!important}@media (min-width:480px){.cc-preferences__title{padding-right:0!important}}@media (min-width:980px){.cc-preferences__title{font-size:22px!important}}.cc-preferences__body{background:#fff!important;flex:1 1 auto;min-height:200px;overflow-x:hidden;overflow-y:auto;padding:16px}.cc-preferences__footer{background:#fff!important;border-top:1px solid #d0d0d0;box-shadow:0 0 5px 0 #0003;margin-bottom:0;padding:12px;position:relative}@media (min-width:480px){.cc-preferences__footer{align-items:stretch;display:flex}}.cc-preferences__footer>.cc-button{display:block;width:100%}@media (min-width:480px){.cc-preferences__footer>.cc-button{flex:1 1 auto}}@media (min-width:980px){.cc-preferences__footer>.cc-button{flex-basis:auto}}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:first-child){margin-left:12px}}.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:8px}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:0}}.cc-preferences__categories{list-style:none;margin:0;padding:0}.cc-preferences__category:not(:last-child){border-bottom:1px solid #0000001a;margin-bottom:12px;padding-bottom:12px}.cc-preferences__category-description{font-size:14px;margin:0 0 8px;padding:0}.cc-preferences__category-footer{align-items:center;display:flex;justify-content:space-between}.cc-preferences__status{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__controls{display:flex;margin:0}.cc-preferences__controls>:not(:last-child){margin-right:12px}.cc-preferences__always-on{font-size:14px;font-weight:700;padding-left:26px;position:relative}.cc-preferences__always-on:before{background:#0000!important;border:solid;border-top-color:#0000;border-width:0 0 4px 4px;content:"";display:block;height:10px;left:0;position:absolute;top:2px;transform:rotate(-45deg);width:18px}.cc-preferences__details{background-color:#eff6fb!important;border-radius:10px;font-size:14px;margin:12px 0 0;padding:12px}.cc-preferences__cookie-list,.cc-preferences__provider-list{list-style:none;margin:0;padding:0}.cc-preferences__provider-list{columns:170px}.cc-preferences__cookie-title{font-size:1em;font-style:normal;font-weight:400;margin:0}.cc-preferences__cookie-description{font-size:1em;margin:0 0 8px}.cc-preferences__cookie-domain,.cc-preferences__cookie-lifespan{border-left:1px solid #999;color:#666;margin-left:8px;padding-left:4px}body.cc-has-preferences-open{overflow:hidden;position:relative}.cc-table{border-collapse:collapse;width:100%}.cc-table tbody tr{border-top:1px solid #0000001a}.cc-table td,.cc-table th{font-size:14px;padding:4px 8px;vertical-align:top}.cc-table+.cc-table{margin-top:8px}.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:70%}@media (min-width:680px){.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:80%}}.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{padding-left:8px;width:40%}@media (min-width:680px){.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{width:30%}}.cc-tabs,.cc-tabs>ul{background-color:#faf9f6;display:flex;list-style:none;margin:0;padding:0}.cc-tabs__panel{border-top:1px solid #ececec;margin-top:-1px}.cc-tabs__button{background-color:initial;border:0;border-left:1px solid #0000;border-right:1px solid #0000;color:#111;font-size:16px;font-weight:700;margin:0;padding:15px 16px 12px;position:relative}.cc-tabs__button:before{background-color:initial;content:"";display:block;height:3px;left:0;position:absolute;top:0;width:100%}.cc-tabs__button.cc-active{background-color:#fff;border-color:#025e8d #ececec #ececec;color:#111}.cc-tabs__button.cc-active:before{background-color:#025e8d}.cc-tabs__content{background-color:#fff;display:none;font-size:14px;padding:16px}.cc-tabs__section:not(:first-child){padding-top:24px}.cc-tcf{font-size:14px;margin-top:16px}.cc-tcf__list{list-style:none;margin:0;padding:0}.cc-vendor-count{color:#666;font-size:16px;margin:0 0 8px}.cc-hide{display:none!important}.cc-show{display:block!important}.cc-external-link{background-color:#ececec;background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTcgMGExIDEgMCAxIDEgMCAySDIuNmMtLjM3MSAwLS42LjIwOS0uNi41djE1YzAgLjI5MS4yMjkuNS42LjVoMTQuOGMuMzcxIDAgLjYtLjIwOS42LS41VjEzYTEgMSAwIDAgMSAyIDB2NC41YzAgMS40MzgtMS4xNjIgMi41LTIuNiAyLjVIMi42QzEuMTYyIDIwIDAgMTguOTM4IDAgMTcuNXYtMTVDMCAxLjA2MiAxLjE2MiAwIDIuNiAwem02IDBoNmwuMDc1LjAwMy4xMjYuMDE3LjExMS4wMy4xMTEuMDQ0LjA5OC4wNTIuMDk2LjA2Ny4wOS4wOGExIDEgMCAwIDEgLjA5Ny4xMTJsLjA3MS4xMS4wNTQuMTE0LjAzNS4xMDUuMDMuMTQ4TDIwIDF2NmExIDEgMCAwIDEtMiAwVjMuNDE0bC02LjY5MyA2LjY5M2ExIDEgMCAwIDEtMS40MTQtMS40MTRMMTYuNTg0IDJIMTNhMSAxIDAgMCAxLS45OTMtLjg4M0wxMiAxYTEgMSAwIDAgMSAxLTEiLz48L3N2Zz4=");background-position:right 8px center;background-repeat:no-repeat;background-size:10px auto;border-radius:3px;box-shadow:0 1px 0 #0003;color:#111!important;display:inline-block;font-size:12px;padding:4px 26px 4px 8px;text-decoration:none!important}.cc-external-link:hover{text-decoration:underline!important}.cc-m-0{margin:0!important}.cc-grey{color:#666}</style><script async="" type="text/javascript" src="https://injections.readcube.com/nature/inject.fa744f3f.js"></script></head>
<body class="article-page"><dialog class="cc-banner" data-cc-banner="" data-nosnippet="" aria-labelledby="cc-banner-label" open="">
		<div class="cc-banner__content" autofocus="" tabindex="-1">
			<div class="cc-banner__container">
				<div class="cc-banner__header">
					<h2 class="cc-banner__title" id="cc-banner-label">Your privacy, your choice</h2>
				</div>
				<div class="cc-banner__body">
					<div class="cc-banner__policy">
						<p>We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.</p><p>By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.</p><p>See our <a href="https://www.nature.com/info/privacy" data-cc-action="privacy">privacy policy</a> for more information on the use of your personal data.</p><p><button type="button" data-cc-action="preferences" class="cc-button cc-button--link cc-button--text">Manage preferences</button> for further information and to change your choices.</p>
					</div>
					
				</div>
				<div class="cc-banner__footer">
					
			<button data-cc-action="accept" class="cc-button cc-button--secondary cc-button--contrast cc-banner__button cc-banner__button-accept">Accept all cookies</button>
			
		
				</div>
			</div>
		</div>
	</dialog>

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90" style="display: none !important;">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-test="top-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41591-022-01786-3;doi=10.1038/s41591-022-01786-3;subjmeta=1059,1619,1647,2325,250,514,554,580,631,67,692,699;kwrd=Cancer+immunotherapy,Sequencing,T+cells,Tumour+immunology">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=1757826152&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01786-3%26doi%3D10.1038/s41591-022-01786-3%26subjmeta%3D1059,1619,1647,2325,250,514,554,580,631,67,692,699%26kwrd%3DCancer+immunotherapy,Sequencing,T+cells,Tumour+immunology">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=1757826152&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01786-3%26doi%3D10.1038/s41591-022-01786-3%26subjmeta%3D1059,1619,1647,2325,250,514,554,580,631,67,692,699%26kwrd%3DCancer+immunotherapy,Sequencing,T+cells,Tumour+immunology"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header="" data-track-component="nature-150-split-header" style="border-color:#e40428">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nm" data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" height="32" alt="Nature Medicine">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
                            </a><div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected="">All journals</option>
                                    <option value="nm">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41591-022-01786-3"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header" hidden="">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-articles" data-track="click" data-track-action="research articles" data-track-label="link" data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-and-analysis" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link" data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/podcast" data-track="click" data-track-action="podcasts" data-track-label="link" data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/current-issue" data-track="click" data-track-action="current issue" data-track-label="link" data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/collections" data-track="click" data-track-action="collections" data-track-label="link" data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://www.facebook.com/Nature-Medicine-193691346949/" data-track="click" data-track-action="facebook" data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://twitter.com/naturemedicine" data-track="click" data-track-action="twitter" data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=5" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/aims" data-track="click" data-track-action="aims &amp; scope" data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-information" data-track="click" data-track-action="journal information" data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-impact" data-track="click" data-track-action="journal metrics" data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editors" data-track="click" data-track-action="about the editors" data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-cross-journal-editorial-team" data-track="click" data-track-action="research cross-journal editorial team" data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-cross-journal-editorial-team" data-track="click" data-track-action="reviews cross-journal editorial team" data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/statistics-advisory-panel" data-track="click" data-track-action="statistical advisory panel" data-track-label="link">
                                    Statistical Advisory Panel
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/our-publishing-models" data-track="click" data-track-action="our publishing models" data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-values-statement" data-track="click" data-track-action="editorial values statement" data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-policies" data-track="click" data-track-action="editorial policies" data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/content" data-track="click" data-track-action="content types" data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="javascript:;" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                        </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/submission-guidelines" data-track="click" data-track-action="submission guidelines" data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-context="header publish with us dropdown menu" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/open-access-funding" data-test="funding-eligibility-link" data-track="click_explore_funding" data-track-context="header publish with us" data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link" href="https://mts-nmed.nature.com/cgi-bin/main.plex" data-track="click_submit_manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-action="submit manuscript" data-track-label="link (publish with us dropdown menu)" data-track-external="" data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link" href="https://journal-alerts.springernature.com/subscribe?journal_id=41591" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-label="link (desktop site header)" data-track-external="">
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature medicine"><span itemprop="name">nature medicine</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm/articles?type=article" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" id="js-enable-context-bar" data-test="context-bar" data-context-bar="" aria-hidden="true">
                    <div class="c-context-bar__container u-container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            Immunogenicity and therapeutic targeting of a public neoantigen derived from mutated <i>PIK3CA</i>
                        </div>
                        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01786-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01786-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2022-04-28">28 April 2022</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Immunogenicity and therapeutic targeting of a public neoantigen derived from mutated <i>PIK3CA</i></h1>
                    <ul class="c-article-author-list c-article-author-list--short js-no-scroll" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Smita_S_-Chandran-Aff1-Aff2-Aff3" data-author-popup="auth-Smita_S_-Chandran-Aff1-Aff2-Aff3" data-author-search="Chandran, Smita S." data-corresp-id="c1" data-track-context="researcher popup with no profile" data-track-index="1_22">Smita S. Chandran<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-9266-6878"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9266-6878</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jiaqi-Ma-Aff4-Aff5" data-author-popup="auth-Jiaqi-Ma-Aff4-Aff5" data-author-search="Ma, Jiaqi" data-track-context="researcher popup with no profile" data-track-index="2_22">Jiaqi Ma</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-6298-0193"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6298-0193</a></span><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff5" tabindex="-1">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Martin_G_-Klatt-Aff6" data-author-popup="auth-Martin_G_-Klatt-Aff6" data-author-search="Klatt, Martin G." data-track-context="researcher popup with no profile" data-track-index="3_22">Martin G. Klatt</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-8703-7305"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-8703-7305</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Friederike-D_ndar-Aff7-Aff8" data-author-popup="auth-Friederike-D_ndar-Aff7-Aff8" data-author-search="Dündar, Friederike" data-track-context="researcher popup with no profile" data-track-index="4_22">Friederike Dündar</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-2301-112X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2301-112X</a></span><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Chaitanya-Bandlamudi-Aff9" data-author-popup="auth-Chaitanya-Bandlamudi-Aff9" data-author-search="Bandlamudi, Chaitanya" data-track-context="researcher popup with no profile" data-track-index="5_22">Chaitanya Bandlamudi</a><sup class="u-js-hide"><a href="#Aff9" tabindex="-1">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Pedram-Razavi-Aff1-Aff10-Aff11" data-author-popup="auth-Pedram-Razavi-Aff1-Aff10-Aff11" data-author-search="Razavi, Pedram" data-track-context="researcher popup with no profile" data-track-index="6_22">Pedram Razavi</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-4236-0576"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-4236-0576</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff10" tabindex="-1">10</a>,<a href="#Aff11" tabindex="-1">11</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Hannah_Y_-Wen-Aff12" data-author-popup="auth-Hannah_Y_-Wen-Aff12" data-author-search="Wen, Hannah Y." data-track-context="researcher popup with no profile" data-track-index="7_22">Hannah Y. Wen</a><sup class="u-js-hide"><a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Britta-Weigelt-Aff12" data-author-popup="auth-Britta-Weigelt-Aff12" data-author-search="Weigelt, Britta" data-track-context="researcher popup with no profile" data-track-index="8_22">Britta Weigelt</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-9927-1270"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9927-1270</a></span><sup class="u-js-hide"><a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Paul-Zumbo-Aff7-Aff8" data-author-popup="auth-Paul-Zumbo-Aff7-Aff8" data-author-search="Zumbo, Paul" data-track-context="researcher popup with no profile" data-track-index="9_22">Paul Zumbo</a><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Si_Ning-Fu-Aff1-Aff2" data-author-popup="auth-Si_Ning-Fu-Aff1-Aff2" data-author-search="Fu, Si Ning" data-track-context="researcher popup with no profile" data-track-index="10_22">Si Ning Fu</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Lauren_B_-Banks-Aff1-Aff2-Aff10" data-author-popup="auth-Lauren_B_-Banks-Aff1-Aff2-Aff10" data-author-search="Banks, Lauren B." data-track-context="researcher popup with no profile" data-track-index="11_22">Lauren B. Banks</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Fei-Yi-Aff1-Aff2" data-author-popup="auth-Fei-Yi-Aff1-Aff2" data-author-search="Yi, Fei" data-track-context="researcher popup with no profile" data-track-index="12_22">Fei Yi</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Enric-Vercher-Aff1" data-author-popup="auth-Enric-Vercher-Aff1" data-author-search="Vercher, Enric" data-track-context="researcher popup with no profile" data-track-index="13_22">Enric Vercher</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Inaki-Etxeberria-Aff1" data-author-popup="auth-Inaki-Etxeberria-Aff1" data-author-search="Etxeberria, Inaki" data-track-context="researcher popup with no profile" data-track-index="14_22">Inaki Etxeberria</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-2713-0836"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2713-0836</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Watchain_D_-Bestman-Aff10-Aff13" data-author-popup="auth-Watchain_D_-Bestman-Aff10-Aff13" data-author-search="Bestman, Watchain D." data-track-context="researcher popup with no profile" data-track-index="15_22">Watchain D. Bestman</a><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a>,<a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Arnaud-Da_Cruz_Paula-Aff13" data-author-popup="auth-Arnaud-Da_Cruz_Paula-Aff13" data-author-search="Da Cruz Paula, Arnaud" data-track-context="researcher popup with no profile" data-track-index="16_22">Arnaud Da Cruz Paula</a><sup class="u-js-hide"><a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ilinca_S_-Aricescu-Aff1" data-author-popup="auth-Ilinca_S_-Aricescu-Aff1" data-author-search="Aricescu, Ilinca S." data-track-context="researcher popup with no profile" data-track-index="17_22">Ilinca S. Aricescu</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Alexander-Drilon-Aff10-Aff11-Aff14" data-author-popup="auth-Alexander-Drilon-Aff10-Aff11-Aff14" data-author-search="Drilon, Alexander" data-track-context="researcher popup with no profile" data-track-index="18_22">Alexander Drilon</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6806-9061"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6806-9061</a></span><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a>,<a href="#Aff11" tabindex="-1">11</a>,<a href="#Aff14" tabindex="-1">14</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Doron-Betel-Aff8-Aff11-Aff15" data-author-popup="auth-Doron-Betel-Aff8-Aff11-Aff15" data-author-search="Betel, Doron" data-track-context="researcher popup with no profile" data-track-index="19_22">Doron Betel</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-8006-7752"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8006-7752</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff11" tabindex="-1">11</a>,<a href="#Aff15" tabindex="-1">15</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-David_A_-Scheinberg-Aff6-Aff10" data-author-popup="auth-David_A_-Scheinberg-Aff6-Aff10" data-author-search="Scheinberg, David A." data-track-context="researcher popup with no profile" data-track-index="20_22">David A. Scheinberg</a><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a>,<a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Brian_M_-Baker-Aff4-Aff5" data-author-popup="auth-Brian_M_-Baker-Aff4-Aff5" data-author-search="Baker, Brian M." data-track-context="researcher popup with no profile" data-track-index="21_22">Brian M. Baker</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff5" tabindex="-1">5</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 22 authors for this article" title="Show all 22 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Christopher_A_-Klebanoff-Aff1-Aff2-Aff3-Aff10-Aff11-Aff14-Aff16" data-author-popup="auth-Christopher_A_-Klebanoff-Aff1-Aff2-Aff3-Aff10-Aff11-Aff14-Aff16" data-author-search="Klebanoff, Christopher A." data-corresp-id="c2" data-track-context="researcher popup with no profile" data-track-index="22_22">Christopher A. Klebanoff<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-9645-3896"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9645-3896</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff10" tabindex="-1">10</a>,<a href="#Aff11" tabindex="-1">11</a>,<a href="#Aff14" tabindex="-1">14</a>,<a href="#Aff16" tabindex="-1">16</a></sup>&nbsp;</li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nm" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Medicine</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;28</b>,&nbsp;<span class="u-visually-hidden">pages </span>946–957 (<span data-test="article-publication-year">2022</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item" data-test="access-count">
                        <p class="c-article-metrics-bar__count">38k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item" data-test="citation-count">
                        <p class="c-article-metrics-bar__count">71 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item" data-test="altmetric-score">
                            <p class="c-article-metrics-bar__count">101 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41591-022-01786-3/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Public neoantigens (NeoAgs) represent an elite class of shared cancer-specific epitopes derived from recurrently mutated driver genes. Here we describe a high-throughput platform combining single-cell transcriptomic and T cell receptor (TCR) sequencing to establish whether mutant <i>PIK3CA</i>, among the most frequently genomically altered driver oncogenes, generates an immunogenic public NeoAg. Using this strategy, we developed a panel of TCRs that recognize an endogenously processed neopeptide encompassing a common <i>PIK3CA</i> hotspot mutation restricted by the prevalent human leukocyte antigen (HLA)-A*03:01 allele. Mechanistically, immunogenicity to this public NeoAg arises from enhanced neopeptide/HLA complex stability caused by a preferred HLA anchor substitution. Structural studies indicated that the HLA-bound neopeptide presents a comparatively ‘featureless’ surface dominated by the peptide’s backbone. To bind this epitope with high specificity and affinity, we discovered that a lead TCR clinical candidate engages the neopeptide through an extended interface facilitated by an unusually long CDR3β loop. In patients with diverse malignancies, we observed NeoAg clonal conservation and spontaneous immunogenicity to the neoepitope. Finally, adoptive transfer of TCR-engineered T cells led to tumor regression in vivo in mice bearing <i>PIK3CA</i>-mutant tumors but not wild-type <i>PIK3CA</i> tumors. Together, these findings establish the immunogenicity and therapeutic potential of a mutant <i>PIK3CA</i>-derived public NeoAg.</p></div></div></section>

            
                
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs10038-021-00950-w/MediaObjects/10038_2021_950_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s10038-021-00950-w?fromPaywallRec=false" data-track="select_recommendations_1" data-track-context="inline recommendations" data-track-action="click recommendations inline - 1" data-track-label="10.1038/s10038-021-00950-w">Immunogenomics in personalized cancer treatments
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">30 June 2021</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41587-022-01547-0/MediaObjects/41587_2022_1547_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41587-022-01547-0?fromPaywallRec=false" data-track="select_recommendations_2" data-track-context="inline recommendations" data-track-action="click recommendations inline - 2" data-track-label="10.1038/s41587-022-01547-0">Identification of patient-specific CD4<sup>+</sup> and CD8<sup>+</sup> T cell neoantigens through HLA-unbiased genetic screens
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">02 January 2023</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-021-26936-y/MediaObjects/41467_2021_26936_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41467-021-26936-y?fromPaywallRec=false" data-track="select_recommendations_3" data-track-context="inline recommendations" data-track-action="click recommendations inline - 3" data-track-label="10.1038/s41467-021-26936-y">Proteogenomic discovery of neoantigens facilitates personalized multi-antigen targeted T cell immunotherapy for brain tumors
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">18 November 2021</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1749514231,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Neoantigens (NeoAgs)—HLA-bound peptides resulting from non-synonymous somatic mutations (NSSMs)—are a major class of cancer regression antigens<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Schumacher, T. N. &amp; Schreiber, R. D. Neoantigens in cancer immunotherapy. Science 348, 69–74 (2015)." href="/articles/s41591-022-01786-3#ref-CR1" id="ref-link-section-d73616779e977">1</a></sup>. Most NeoAgs result from random passenger mutations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Parkhurst, M. R. et al. Unique neoantigens arise from somatic mutations in patients with gastrointestinal cancers. Cancer Discov. 9, 1022–1035 (2019)." href="/articles/s41591-022-01786-3#ref-CR2" id="ref-link-section-d73616779e981">2</a></sup>. Patient-specific, or ‘private’, NeoAgs pose a significant challenge to therapeutically target because they require customization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Hu, Z., Ott, P. A. &amp; Wu, C. J. Towards personalized, tumour-specific, therapeutic vaccines for cancer. Nat. Rev. Immunol. 18, 168–182 (2018)." href="/articles/s41591-022-01786-3#ref-CR3" id="ref-link-section-d73616779e985">3</a></sup>. Furthermore, private NeoAgs are subject to clonal heterogeneity, a major mechanism of immunotherapy resistance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="McGranahan, N. et al. Clonal neoantigens elicit T cell immunoreactivity and sensitivity to immune checkpoint blockade. Science 351, 1463–1469 (2016)." href="#ref-CR4" id="ref-link-section-d73616779e989">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Miao, D. et al. Genomic correlates of response to immune checkpoint blockade in microsatellite-stable solid tumors. Nat. Genet. 50, 1271–1281 (2018)." href="#ref-CR5" id="ref-link-section-d73616779e989_1">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Wolf, Y. et al. UVB-induced tumor heterogeneity diminishes immune response in melanoma. Cell 179, 219–235 e221 (2019)." href="/articles/s41591-022-01786-3#ref-CR6" id="ref-link-section-d73616779e992">6</a></sup>. In contrast, NeoAgs resulting from gain-of-function mutations in driver genes would generally be clonally conserved because the source proteins directly contribute to cancer cell fitness<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Reiter, J. G. et al. Minimal functional driver gene heterogeneity among untreated metastases. Science 361, 1033–1037 (2018)." href="/articles/s41591-022-01786-3#ref-CR7" id="ref-link-section-d73616779e996">7</a></sup>. If a NeoAg resulting from a driver hotspot mutation is restricted by a prevalent HLA allele, this elite subset of cancer-specific epitopes would be shared among patients with cancer, creating a ‘public’ NeoAg<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Klebanoff, C. A. &amp; Wolchok, J. D. Shared cancer neoantigens: making private matters public. J. Exp. Med. 215, 5–7 (2018)." href="/articles/s41591-022-01786-3#ref-CR8" id="ref-link-section-d73616779e1001">8</a></sup>.</p><p>Only a limited number of public NeoAgs have been identified to date<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wolfel, T. et al. A p16INK4a-insensitive CDK4 mutant targeted by cytolytic T lymphocytes in a human melanoma. Science 269, 1281–1284 (1995)." href="#ref-CR9" id="ref-link-section-d73616779e1008">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Linard, B. et al. A ras-mutated peptide targeted by CTL infiltrating a human melanoma lesion. J. Immunol. 168, 4802–4808 (2002)." href="#ref-CR10" id="ref-link-section-d73616779e1008_1">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Schumacher, T. et al. A vaccine targeting mutant IDH1 induces antitumour immunity. Nature 512, 324–327 (2014)." href="#ref-CR11" id="ref-link-section-d73616779e1008_2">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Tran, E. et al. Immunogenicity of somatic mutations in human gastrointestinal cancers. Science 350, 1387–1390 (2015)." href="#ref-CR12" id="ref-link-section-d73616779e1008_3">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chheda, Z. S. et al. Novel and shared neoantigen derived from histone 3 variant H3.3K27M mutation for glioma T cell therapy. J. Exp. Med. 215, 141–157 (2018)." href="#ref-CR13" id="ref-link-section-d73616779e1008_4">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Veatch, J. R. et al. Tumor-infiltrating BRAFV600E-specific CD4+ T cells correlated with complete clinical response in melanoma. J. Clin. Invest. 128, 1563–1568 (2018)." href="#ref-CR14" id="ref-link-section-d73616779e1008_5">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Malekzadeh, P. et al. Neoantigen screening identifies broad TP53 mutant immunogenicity in patients with epithelial cancers. J. Clin. Invest. 129, 1109–1114 (2019)." href="#ref-CR15" id="ref-link-section-d73616779e1008_6">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Hsiue, E. H. et al. Targeting a neoantigen derived from a common TP53 mutation. Science 371, eabc8697 (2021)." href="/articles/s41591-022-01786-3#ref-CR16" id="ref-link-section-d73616779e1011">16</a></sup>. Critically, none has been reported arising from mutant (Mut) <i>PIK3CA</i> (the gene encoding phosphoinositide 3-kinase alpha, PI3Kα), which is among the most common genetically altered driver oncogenes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Samuels, Y. et al. High frequency of mutations of the PIK3CA gene in human cancers. Science 304, 554 (2004)." href="#ref-CR17" id="ref-link-section-d73616779e1018">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. Cell 173, 371–385 (2018)." href="#ref-CR18" id="ref-link-section-d73616779e1018_1">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Priestley, P. et al. Pan-cancer whole-genome analyses of metastatic solid tumours. Nature 575, 210–216 (2019)." href="/articles/s41591-022-01786-3#ref-CR19" id="ref-link-section-d73616779e1021">19</a></sup>. Small-molecule PI3K inhibitors cause cancer regression, validating PI3Kα as a therapeutic target<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Andre, F. et al. Alpelisib for PIK3CA-mutated, hormone receptor-positive advanced breast cancer. N. Engl. J. Med. 380, 1929–1940 (2019)." href="/articles/s41591-022-01786-3#ref-CR20" id="ref-link-section-d73616779e1025">20</a></sup>. However, these inhibitors are not curative and can be associated with significant on-target/off-tumor toxicities. Novel immunotherapies that target Mut PI3Kα while sparing healthy tissues could have broad utility for many patients with cancer. Here we report on a high-throughput platform combining single-cell transcriptomic and TCR sequencing to test whether Mut <i>PIK3CA</i> generates an immunogenic and therapeutically actionable public NeoAg.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Public NeoAg TCR discovery platform</h3><p>We developed a high-throughput method to test whether Mut PI3Kα is immunogenic and to retrieve TCR gene sequences that confer specificity to this NeoAg (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig1">1a</a>). The platform proceeds in four steps. First, healthy donor (HD) naive T cells (T<sub>N</sub>) undergo in vitro sensitization (IVS) using autologous monocyte-derived dendritic cells (moDCs) electroporated with mRNA encoding Mut <i>PIK3CA</i>. Second, matched aliquots from sensitized wells are re-stimulated with moDCs expressing Mut or wild-type (WT) <i>PIK3CA</i> to identify ‘hit’ wells containing T cells with preferential activation to the Mut antigen using a qPCR screen. Third, T cells from ‘hit’ wells are again re-stimulated to identify reactive clonotypes and retrieve their TCR gene sequences. This step, which we have termed stimulation-induced functional TCR sequencing (SIFT-seq), uses combined single-cell TCR V(D)J and RNA sequencing. Finally, TCR clonotypes that express activation-associated transcripts in response to Mut but not WT <i>PIK3CA</i> are reconstructed for functional validation.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Development and validation of the SIFT-seq discovery platform for Mut PIK3CA-specific TCRs."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Development and validation of the SIFT-seq discovery platform for Mut <i>PIK3CA</i>-specific TCRs.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="896"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Schematic overview of the SIFT-seq TCR discovery platform. <b>b</b>, log<sub>2</sub> fold change (FC) ratio of <i>IFNG</i> transcripts from <i>n</i> = 64 unique TCR clonotypes identified using single-cell RNA and V(D)J TCR sequencing from screen-positive ‘hit’ well MSK 21LT2. Matched aliquots of sensitized T cells from HD1 were stimulated with <i>PIK3CA</i> (H1047L) (Mut) or WT <i>PIK3CA</i> before single-cell sequencing. The mean <i>IFNG</i> ratio for all evaluable TCR clonotypes is shown. Statistical analyses were performed for all clonotypes with a minimal ratio of ≥2 (dashed line). The <i>x</i> axis indicates the relative frequencies of individual TCR clonotypes. ****<i>P</i> = 1.37 × 10<sup>−28</sup> using two-sided Welch’s <i>t</i>-test. <b>c</b>, Volcano plot displaying global transcriptomic changes for MSK 21LT2 clonotype 20 after stimulation with Mut versus WT <i>PIK3CA</i>. Vertical and horizontal dashed lines indicate thresholds for log<sub>2</sub> gene expression FC and statistical significance, respectively. Orange and blue dots represent significantly upregulated and downregulated genes after Mut <i>PIK3CA</i> stimulation, respectively. <b>d</b>, Violin plots depicting transcript levels for the lineage markers <i>CD3E</i>, <i>CD4</i> and <i>CD8A</i> from MSK 21LT2 clonotype 20. ****<i>P</i> &lt; 0.0001 using two-sided Student’s <i>t</i>-test. <b>e</b>, log<sub>2</sub> FC ratio of <i>IFNG</i> transcripts from <i>n</i> = 398 unique TCR clonotypes identified using single-cell sequencing from screen-positive ‘hit’ well MSK 06006T derived from HD2. *<i>P</i> = 1.41 × 10<sup>−5</sup> using two-sided Welch’s <i>t</i>-test. NS, not significant. <b>f</b>, Violin plots depicting lineage marker transcript expression for MSK 0606T clonotype 367. ****<i>P</i> &lt; 0.0001 using two-sided Student’s <i>t</i>-test. Representative FACS plots (<b>g</b>) and summary bar graph (<b>h</b>) (<i>n</i> = 3 biologically independent replicates per condition) displaying the frequency of intracellular IL-2 production in polyclonal T cells following retroviral transduction with SIFT-seq-retrieved TCR candidates. The reconstructed TCR expresses a murine constant chain (mTCR), enabling detection with an anti-mTCR-specific antibody. Transduced T cells (live<sup>+</sup>mTCR<sup>+</sup>CD3<sup>+</sup>) were co-cultured with autologous moDCs electroporated with mRNA encoding Mut or WT <i>PIK3CA</i> in the absence or presence of pan-HLA class I or class II blocking antibodies. ***<i>P</i> &lt; 0.001, **<i>P</i> = 0.0036 and NS using two-sided Student’s <i>t</i>-test with Bonferroni correction. <b>b</b>,<b>e</b>,<b>h</b>, Symbols and bar graphs are displayed as mean ± s.e.m. <b>d</b>,<b>f</b>, Violin distributions are centered around the median (red horizontal line) with quartile ranges displayed above and below (dashed horizontal lines). The maxima and minima are represented by the top and bottom of each plot.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Most <i>PIK3CA</i> mutations occur at three hotspots, with H1047 being the most common<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Tate, J. G. et al. COSMIC: the Catalogue of Somatic Mutations in Cancer. Nucleic Acids Res. 47, D941–D947 (2019)." href="/articles/s41591-022-01786-3#ref-CR21" id="ref-link-section-d73616779e1228">21</a></sup>. Healthy donor-derived T<sub>N</sub> underwent IVS by moDCs transfected with genes encoding the four most frequent <i>PIK3CA</i> hotspot mutations (H1047L/R, E542K and E545K). The qPCR screen identified a single well (21LT2) that preferentially upregulated <i>IFNG</i> in response to <i>PIK3CA</i> (H1047L) (henceforth Mut <i>PIK3CA</i>), which was subsequently subjected to SIFT-seq. To identify candidate TCR sequences, we plotted the ratio of <i>IFNG</i> produced under Mut and WT stimulation conditions for each clonotype (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig1">1b</a>). Among <i>n</i> = 64 clonotypes, only clonotype 20 significantly upregulated <i>IFNG</i> to Mut <i>PIK3CA</i>. Global transcriptomic analysis of clonotype 20 revealed additional Mut-specific gene expression changes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig1">1c</a>), whereas all other clonotypes retained a quiescent profile (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">1a</a>). Lineage marker assessment revealed expression of <i>CD3E</i> and <i>CD8A</i> but minimal to no <i>CD4</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig1">1d</a>), suggesting that clonotype 20 was human leukocyte antigen (HLA) class I (HLA-I) restricted. We performed an additional run using peripheral blood mononuclear cells (PBMCs) from a second HD (HD2). A single qPCR screen-positive well (0606T) was identified and subjected to SIFT-seq. Among <i>n</i> = 398 clonotypes, only clonotype 367 selectively upregulated <i>IFNG</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig1">1e</a>) and other TCR activation-induced genes (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">1b</a>) in response to Mut <i>PIK3CA</i>. Similar to clonotype 20 from HD1, clonotype 367 expressed <i>CD8A</i> but minimal <i>CD4</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig1">1f</a>), suggesting that this TCR was also HLA-I restricted.</p><p>Polyclonal T cells retrovirally (RV) transduced with SIFT-seq-retrieved TCR sequences were co-cultured with autologous moDCs transfected with Mut or WT <i>PIK3CA</i>. Expression of TCRs from 21LT2 clonotype 20 and 0606T clonotype 367 resulted in polyfunctional cytokine production exclusively in response to Mut <i>PIK3CA</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig1">1g</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">2</a>), validating the SIFT-seq approach. As controls, we synthesized TCRs from the dominant clonotypes in each screen-positive well in addition to clonotypes that expressed <i>IFNG</i> under both Mut and WT stimulation conditions and confirmed that these do not confer Mut <i>PIK3CA</i> reactivity (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">1</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">3</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">4</a>). To determine whether the Mut <i>PIK3CA</i>-specific TCRs were HLA-I or HLA-II restricted, we tested their function in the presence of HLA class-specific blocking antibodies. Cytokine release by both TCRs was attenuated exclusively by an HLA-I blocking antibody (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig1">1g,h</a>), demonstrating that SIFT-seq correctly predicted the HLA class restriction of Mut-specific clonotypes. We conclude that the SIFT-seq platform can reproducibly identify TCR sequences that confer specificity to a <i>PIK3CA</i> hotspot mutation.</p><h3 class="c-article__sub-heading" id="Sec4">HLA restriction and functionality of <i>PIK3CA</i> public NeoAg TCRs</h3><p>To resolve which HLA-I allele is required for 21LT2 TCR clonotype 20 recognition, we generated mono-allelic cells that display individual HLA-I molecules from HD1 together with Mut or WT <i>PIK3CA</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig2">2a</a>). Co-culture of these targets with T cells transduced with 21LT2 TCR clonotype 20 revealed cytokine production exclusively in the context of Mut <i>PIK3CA</i> and HLA-A*03:01. This HLA-I allele is among the most common in North America and Europe, occurring in 20–28% of individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Gonzalez-Galarza, F. F. et al. Allele frequency net database (AFND) 2020 update: gold-standard data classification, open access genotype data and new query tools. Nucleic Acids Res. 48, D783–D788 (2020)." href="/articles/s41591-022-01786-3#ref-CR22" id="ref-link-section-d73616779e1369">22</a></sup>. Using a similar approach, we resolved that 0606T TCR clonotype 367 is also restricted by HLA-A*03:01. Based on these data, we performed SIFT-seq on two additional HLA-A*03:01<sup>+</sup> HDs (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">1</a>) and retrieved two additional TCRs that recognize the same Mut <i>PIK3CA</i>/HLA-I combination. TCRs reactive against alternative <i>PIK3CA</i> mutations included in each screen were not detected. The resulting Mut <i>PIK3CA</i> TCR panel (TCRs 1–4) uses distinct variable segments, unique complementarity-determining region 3 (CDR3) sequences and diverse CDR3 lengths (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig7">1a</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="HLA restriction and functional characterization of PIK3CA public NeoAg-specific TCRs."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: HLA restriction and functional characterization of <i>PIK3CA</i> public NeoAg-specific TCRs.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="896"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Deconvolution of the HLA-I restriction element for the MSK 21LT2 clonotype 20 TCR. The frequency of individual HLA-I alleles expressed by HD1 in North American and European populations is displayed as a heat map. FACS plots show the frequency of CD8<sup>+</sup> TCR-transduced T cells that secrete TNFα after co-culture with HLA-I mono-allelic cell lines expressing WT or Mut <i>PIK3CA</i>. <b>b</b>, Cartoon illustrating the experimental design to assess the co-receptor dependence and functionality of SIFT-seq-retrieved <i>PIK3CA</i> public NeoAg TCR panel members. TCRs 1–4 were individually RV transduced into enriched CD8<sup>+</sup> or CD4<sup>+</sup> T cells and co-cultured with target cells co-expressing HLA-A*03:01 and either WT or Mut <i>PIK3CA</i>. <b>c</b>, Representative FACS plots of CD4<sup>+</sup> (black) and CD8<sup>+</sup> (red) T cells expressing individual PIK3CA public NeoAg TCR panel members after co-culture with HLA-A*03:01<sup>+</sup> target cells that express either WT or Mut <i>PIK3CA</i>. Numbers within each plot indicate the frequency of TNFα-producing TCR-transduced CD4<sup>+</sup> (upper left quadrant) or CD8<sup>+</sup> (upper right quadrant) T cells. The FACS plots shown in <b>a</b> and <b>c</b> are pre-gated on live<sup>+</sup>mTCR<sup>+</sup> T cells. <b>d</b>, The functional avidity of CD8<sup>+</sup> (left) or CD4<sup>+</sup> (right) T cells individually transduced with TCRs 1–4. Transduced T cells were co-cultured with an HLA-I mono-allelic cell line expressing HLA-A*03:01 and electroporated with indicated concentrations of WT or Mut <i>PIK3CA</i> mRNA. <b>e</b>, Kinetic impedance-based lytic assay measuring the % specific cytolysis of HLA-A*03:01<sup>+</sup> target cells expressing WT or Mut <i>PIK3CA</i>. <b>f</b>, Adjusted cytolytic AUC values indicating the cumulative cytolytic capacity of TCRs 1–4 against A*03:01<sup>+</sup>/Mut <i>PIK3CA</i><sup>+</sup> target cells. ***<i>P</i> = 0.001, **<i>P</i> = 0.01 and *<i>P</i> = 0.02; two-sided Student’s <i>t</i>-test was used for statistical analysis. Data shown in <b>d</b>, <b>e</b> and <b>f</b> are representative of two independent experiments using <i>n</i> = 3 biologically independent replicates per condition per independent experiment. Symbols and bar graphs are displayed as mean ± s.e.m. AUC, area under the curve.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Polyclonal CD8<sup>+</sup> and CD4<sup>+</sup> T cells were transduced with individual TCR panel members to characterize their specificity, functional avidity and cytolytic capacity (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig2">2b</a>). In the context of HLA-A*03:01, all members exhibited absolute specificity for Mut <i>PIK3CA</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig2">2c</a>). HLA-A*03:01 is a member of the HLA-A3 supertype, a group of homologous HLA-I molecules characterized by overlapping peptide-binding repertoires<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Sidney, J., Peters, B., Frahm, N., Brander, C. &amp; Sette, A. HLA class I supertypes: a revised and updated classification. BMC Immunol. 9, 1 (2008)." href="/articles/s41591-022-01786-3#ref-CR23" id="ref-link-section-d73616779e1535">23</a></sup>. Within this supertype, HLA-A*03:01 shares the highest homology with HLA-A*03:02 (99.5%) and HLA-A*11:01 (98.1%). To determine if these related HLA-I alleles trigger Mut-specific function, we co-cultured target cells expressing HLA-A*03:01, HLA-A*03:02 or HLA-A*11:01 with T cells transduced with TCRs 1–4. All TCR panel members solely recognized Mut <i>PIK3CA</i> in the context of HLA-A*03:01, highlighting each receptor’s selectivity for a single HLA-I allele (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig7">1b,c</a>).</p><p>We next compared each TCR’s functionality when transduced into CD8<sup>+</sup> or CD4<sup>+</sup> T cells. When expressed in CD8<sup>+</sup> T cells, TCRs 3 and 4 triggered significantly greater Mut-specific cytokine production compared to TCRs 1 and 2 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig2">2c</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">5a</a>). When transduced into CD4<sup>+</sup> T cells, only TCRs 3 and 4 triggered T cell function, indicating co-receptor independence (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig2">2c</a>). The capacity of a TCR to function in a co-receptor-independent manner correlates with a receptor’s intrinsic affinity to the peptide/HLA complex<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Laugel, B. et al. Different T cell receptor affinity thresholds and CD8 coreceptor dependence govern cytotoxic T lymphocyte activation and tetramer binding properties. J. Biol. Chem. 282, 23799–23810 (2007)." href="/articles/s41591-022-01786-3#ref-CR24" id="ref-link-section-d73616779e1566">24</a></sup> and clinical anti-tumor efficacy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Johnson, L. A. et al. Gene therapy with human and mouse T-cell receptors mediates cancer regression and targets normal tissues expressing cognate antigen. Blood 114, 535–546 (2009)." href="#ref-CR25" id="ref-link-section-d73616779e1570">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lu, Y. C. et al. Treatment of patients with metastatic cancer using a major histocompatibility complex class II-restricted T-cell receptor targeting the cancer germline antigen MAGE-A3. J. Clin. Oncol. 35, 3322–3329 (2017)." href="#ref-CR26" id="ref-link-section-d73616779e1570_1">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="D’Angelo, S. P. et al. Antitumor activity associated with prolonged persistence of adoptively transferred NY-ESO-1 (c259)T cells in synovial sarcoma. Cancer Discov. 8, 944–957 (2018)." href="#ref-CR27" id="ref-link-section-d73616779e1570_2">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Nagarsheth, N. B. et al. TCR-engineered T cells targeting E7 for patients with metastatic HPV-associated epithelial cancers. Nat. Med. 27, 419–425 (2021)." href="/articles/s41591-022-01786-3#ref-CR28" id="ref-link-section-d73616779e1573">28</a></sup>. Next, we quantified each TCR’s relative antigen sensitivity by determining their functional avidity. In CD8<sup>+</sup> T cells, TCRs 3 and 4 had lower EC<sub>50</sub> values to Mut <i>PIK3CA</i> compared to TCRs 1 and 2 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig2">2d</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">5b</a>); in CD4<sup>+</sup> T cells, only TCRs 3 and 4 were functional.</p><p>Finally, we measured the capacity of CD8<sup>+</sup> T cells transduced with TCRs 1–4 to mediate Mut-specific cell killing. All TCRs mediated cytolysis of Mut <i>PIK3CA</i><sup>+</sup>/HLA-A*03:01<sup>+</sup> target cells but spared HLA-A*03:01<sup>+</sup> cells expressing WT <i>PIK3CA</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig2">2e</a>). In repeated experiments, TCR4 exhibited superior lytic efficiency relative to other panel members (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig2">2f</a>). Together, these data establish the existence of an immunogenic <i>PIK3CA</i> public NeoAg that is targetable using TCRs with diverse functional attributes.</p><h3 class="c-article__sub-heading" id="Sec5">Mechanism of immunogenicity for a <i>PIK3CA</i> public NeoAg</h3><p>We next determined the sequences of endogenously processed PI3Kα peptides presented by HLA-A*03:01. We performed HLA-immune precipitation liquid chromatography–tandem mass spectrometry (HLA-IP LC–MS/MS) on cells that express individual HLA-I alleles and either WT <i>PIK3CA</i> or one of the four <i>PIK3CA</i> mutations we tested by SIFT-seq. Skyline and chromatogram analyses revealed PI3Kα-derived amino acid (AA) sequences encompassing a hotspot Mut position exclusively from cells that co-express HLA-A*03:01 and <i>PIK3CA</i> (H1047L) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig3">3a,b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig8">2</a>). No peptides were detected from cells that express HLA-A*03:01 and WT <i>PIK3CA</i> or <i>PIK3CA</i> (H1047L) and a mismatched HLA-A allele. Although NetMHCpan 4.1 predicts two high-affinity HLA-A*03:01-restricted peptides surrounding PI3Kα E542K and E545K<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Reynisson, B., Alvarez, B., Paul, S., Peters, B. &amp; Nielsen, M. NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. Nucleic Acids Res. 48, W449–W454 (2020)." href="/articles/s41591-022-01786-3#ref-CR29" id="ref-link-section-d73616779e1653">29</a></sup>, neither epitope was detected (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig8">2b</a>). Similarly, although NetMHCpan 4.1 also predicts 15 high-affinity WT PI3Kα-derived peptides that bind HLA-A*03:01, we detected only one (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">6</a>). To exclude that a high-affinity Mut peptide limited the detection of weaker-binding WT peptides, we repeated this experiment after subdividing WT <i>PIK3CA</i> into overlapping minigenes and uncovered only two additional WT <i>PIK3CA</i> peptides (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">6</a>). To establish whether the specificity of our TCR panel members for HLA-A*03:01 was attributable to differential antigen presentation by closely related HLA-A3 superfamily members, we performed HLA-IP LC–MS/MS on cells that co-express <i>PIK3CA</i> (H1047L) and either HLA-A*03:02 or HLA-A*11:01. We discovered that HLA-A*03:02, but not HLA-A*11:01, was capable of presenting a Mut peptide (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig8">2c</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Mechanism of immunogenicity for a PIK3CA public NeoAg."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Mechanism of immunogenicity for a <i>PIK3CA</i> public NeoAg.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="792"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Skyline analysis of HLA-I-bound peptides resulting from an LC–MS/MS-based immune-peptidomic screen. Relative abundance of precursor ions derived from peptides encompassing the PI3Kα protein’s 1047 position eluted from the indicated HLA-A mono-allelic cell lines. Cells expressed either WT <i>PIK3CA</i> or Mut <i>PIK3CA</i> (H1047L). ND indicates conditions in which no PI3Kα-derived AA sequences encompassing the 1047 hotspot position were experimentally detected. <b>b</b>, Corresponding chromatographic retention times of precursor ions derived from a Mut PI3Kα peptide eluted from HLA-A*03:01<sup>+</sup>/Mut <i>PIK3CA</i><sup>+</sup> cells. <b>c</b>, Mirror plot displaying the MS2 spectra of the Mut PI3Kα-derived public neoepitope (A<u>L</u>HGGWTTK; pMut; top) eluted from HLA-A*03:01<sup>+</sup>/Mut <i>PIK3CA</i><sup>+</sup> cells and a synthetically generated peptide (bottom). Peaks represent <i>b</i> ions in blue and <i>y</i> ions in red. Representative intracellular FACS analysis (<b>d</b>) and summary bar graph (<b>e</b>) for TNFα production in T cells transduced with TCR4 and co-cultured with HLA-A*03:01<sup>+</sup> target cells pulsed with 1 μM of pMut versus pWT. Results shown after gating on live<sup>+</sup>mTCR<sup>+</sup>CD8<sup>+</sup> lymphocytes. Bar graph is displayed as mean ± s.e.m. using <i>n</i> = 3 biologically independent replicates per condition. ***<i>P</i> = 0.001 using two-sided Student’s <i>t</i>-test. <b>f</b>, Structural superimposition of the pMut and pWT peptides bound to HLA-A*03:01. The conformations of pMut and pWT peptides are nearly identical with all α carbon atoms superimposing with a root mean square deviation of 0.73 Å. Representative thermal melt curves (<b>g</b>) and summary scatter plot (<b>h</b>) displaying the melting temperatures (T<sub>m</sub>) of the pMut and pWT/HLA-A*03:01 complexes using differential scanning fluorimetry. Symbols are displayed as mean ± s.e.m. using <i>n</i> = 5 independently performed experiments. ****<i>P</i> &lt; 0.0001 using two-sided Student’s <i>t</i>-test. <b>i</b>, Dissociation of a fluorescently labeled pMut or pWT from soluble HLA-A*03:01 complexes at 37 °C using fluorescence anisotropy. Solid lines show fits to exponential decay functions. Half-lives (t<sub>1/2</sub>) are shown for each peptide ± s.e.m. Data are representative of <i>n</i> = 3 technical replicates per condition per time point. mA, millianisotropy.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Deconvolution of the MS/MS spectrum from HLA-A*03:01<sup>+</sup>/<i>PIK3CA</i> (H1047L)<sup>+</sup> cells divulged a single Mut-containing peptide with the His→Leu substitution occurring at the second position (P2) (A<u>L</u>HGGWTTK; pMut). The sequence of the eluted neopeptide was validated by matching its MS/MS spectra with that of a synthetic peptide (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig3">3c</a>). Co-culture of Mut <i>PIK3CA</i>-specific T cells with HLA-A*03:01<sup>+</sup> target cells pulsed with pMut confirmed that this sequence, but not its WT counterpart (A<u>H</u>HGGWTTK; pWT), triggered T cell function (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig3">3d,e</a>).</p><p>We next sought to establish the physical basis for <i>PIK3CA</i> public NeoAg immunogenicity. We crystallized and determined the X-ray structures of the pMut and pWT peptides bound to HLA-A*03:01 at resolutions of 2.0 Å and 2.1 Å, respectively (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig3">3f</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">7</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">2</a>). The pMut and pWT adopt nearly identical conformations when bound by HLA-A*03:01 (root mean square deviation = 0.72 Å). In both structures, the P6 Trp packs between the peptide backbone and the HLA-I α1 helix, whereas the P7 Thr packs against the α2 helix. Together with the P4/P5 Gly residues, this results in a largely ‘featureless’ central peptide region dominated by the peptide’s backbone. The overall similarities of the two complexes suggest that structural differences alone likely cannot explain the immunogenic potential of the <i>PIK3CA</i> public NeoAg.</p><p>Because the <i>PIK3CA</i> public NeoAg introduces a preferred P2 anchor residue for HLA-A*03:01 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Sarkizova, S. et al. A large peptidome dataset improves HLA class I epitope prediction across most of the human population. Nat. Biotechnol. 38, 199–209 (2020)." href="/articles/s41591-022-01786-3#ref-CR30" id="ref-link-section-d73616779e1846">30</a></sup>), we next quantified the stability of the pMut and pWT peptides bound to HLA-A*03:01 using differential scanning fluorimetry. Consistent with the introduction of an optimal P2 anchor, we discovered that the melting temperature (T<sub>m</sub>) of the pMut/HLA-I complex was significantly higher than the pWT/HLA-I complex (<i>P</i> &lt; 0.0001; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig3">3g,h</a>). Similar results were obtained with HLA-A*03:02 (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">8</a>), suggesting that the specificity for the TCR panel members for HLA-A*03:01 over HLA-A*03:02 might result from engagement of a key residue in HLA-A*03:01 and/or subtle conformational differences in the HLA-bound pMut.</p><p>Improvements in peptide-binding affinity are often associated with longer p/HLA half-lives (t<sub>1/2</sub>). Therefore, we next resolved the kinetic stability of the two p/HLA-A*03:01 complexes at physiologic temperature (37 °C) using fluorescence anisotropy. At 37 °C, the pMut/HLA-I complex had a significantly longer t<sub>1/2</sub> compared to pWT (t<sub>1/2</sub> = 5.497 hours ± 0.130 versus 0.078 hours ± 0.001; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig3">3I</a>). We conclude that the formation of the <i>PIK3CA</i> public NeoAg is driven by the creation of a peptide sequence containing a preferred HLA-I anchor residue, resulting in a stable, high-affinity p/HLA-I complex with a prolonged t<sub>1/2</sub>.</p><h3 class="c-article__sub-heading" id="Sec6">Structural correlates of <i>PIK3CA</i> public NeoAg TCR specificity</h3><p>TCR specificity is determined by six CDR loops, especially the hypervariable CDR3 loops<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Garcia, K. C. &amp; Adams, E. J. How the T cell receptor sees antigen—a structural view. Cell 122, 333–336 (2005)." href="/articles/s41591-022-01786-3#ref-CR31" id="ref-link-section-d73616779e1893">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Gowthaman, R. &amp; Pierce, B. G. TCR3d: the T cell receptor structural repertoire database. Bioinformatics 35, 5323–5325 (2019)." href="/articles/s41591-022-01786-3#ref-CR32" id="ref-link-section-d73616779e1896">32</a></sup>. We noted that TCR4 has a significantly longer (19 AAs) CDR3β loop than TCRs 1–3 (range, 14–16 AAs) and a panel of more than 17,000 HLA-A*03:01-restricted and HLA-A*11:01-restricted TCRs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig4">4a</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Bagaev, D. V. et al. VDJdb in 2019: database extension, new analysis infrastructure and a T-cell receptor motif compendium. Nucleic Acids Res. 48, D1057–D1062 (2020)." href="/articles/s41591-022-01786-3#ref-CR33" id="ref-link-section-d73616779e1903">33</a></sup>. Because of TCR4’s unique CDR3β length and superior cytolytic potency, we sought to characterize this receptor’s binding affinity, three-dimensional structure and fine specificity. For comparison, we used TCR3 as this receptor has a similar EC<sub>50</sub> to TCR4 yet possesses more common CDR3 loop lengths.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Structural correlates of affinity and specificity for a PIK3CA public NeoAg-specific TCR."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Structural correlates of affinity and specificity for a <i>PIK3CA</i> public NeoAg-specific TCR.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="844"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, CDR3α and CDR3β AA lengths of <i>PIK3CA</i> public NeoAg-specific TCR3, TCR4 and a panel of <i>n</i> = 17,414 HLA-A*03:01-restricted and HLA-A*11:01-restricted TCR sequences. Results shown as median ± interquartile range. Representative SPR sensorgram (<b>b</b>) and steady-state binding equilibrium (<b>c</b>) measuring the dissociation constant (<i>K</i><sub>d</sub>) for TCR4 to the pMut/HLA-I complex. Results shown are the average ± s.d. of <i>n</i> = 4 independent experiments. <b>d</b>, Structural overview of the TCR4 pMut/HLA-A*03:01 ternary complex at 3.1-Å resolution. The color scheme is indicated and replicated throughout. <b>e</b>, Top view of the pMut/HLA-A*03:01 complex displaying the crossing angle and positions of the six CDR loops of TCR4. Spheres represent the centers of mass of the TCR’s Vα and Vβ domains. <b>f</b>, AAs of TCR4’s CDRα and CDRβ loops that interact with the pMut peptide. Hydrogen bonds (<i>n</i> = 6) are indicated by red dashed lines. AAs are identified by standard one-letter codes followed by position number. The side chains of contacting residues from the TCR are also identified by AA and hemi-chain position number. <b>g</b>, Identification of TCR4’s peptide recognition motif using alanine and glycine scanning. Intracellular FACS for TNFα production to Ala (upper) or Gly (lower) substituted peptides. TCR4-transduced T cells (identified by gating on mTCR<sup>+</sup> lymphocytes) were co-cultured with HLA-A*03:01<sup>+</sup> targets pulsed with 1 μM of indicated peptides. Results are shown as the mean ± s.e.m. percent maximum response relative to the native pMut peptide using <i>n</i> = 3 biologic replicates per condition. ‘x’ indicates positions not amenable to substitution. <b>h</b>, Measurement of the cross-reactivity potential of TCR4. TCR4-transduced T cells were co-cultured with HLA-A*03:01<sup>+</sup> targets pulsed with 1 μM of peptides containing the motif ‘x-x-x-x-G-W-T-T-K’. Results are shown as mean ± s.e.m. using <i>n</i> = 3 biologic replicates per condition. ****P &lt; 0.0001 using two-sided Student’s <i>t</i>-test with Bonferroni correction. NS, not significant.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We generated recombinant, soluble versions of both TCRs and measured their affinity toward the pMut/HLA-I complex using surface plasmon resonance (SPR). TCR4 has a <i>K</i><sub>d</sub> of 49 ± 11 μM (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig4">4b,c</a>), an affinity higher than many self-antigen-specific TCRs and within a range of pathogen<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Aleksic, M. et al. Different affinity windows for virus and cancer-specific T-cell receptors: implications for therapeutic strategies. Eur. J. Immunol. 42, 3174–3179 (2012)." href="/articles/s41591-022-01786-3#ref-CR34" id="ref-link-section-d73616779e2000">34</a></sup> and neoantigen-specific<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wu, D., Gallagher, D. T., Gowthaman, R., Pierce, B. G. &amp; Mariuzza, R. A. Structural basis for oligoclonal T cell recognition of a shared p53 cancer neoantigen. Nat. Commun. 11, 2908 (2020)." href="#ref-CR35" id="ref-link-section-d73616779e2004">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sim, M. J. W. et al. High-affinity oligoclonal TCRs define effective adoptive T cell therapy targeting mutant KRAS-G12D. Proc. Natl Acad. Sci. USA 117, 12826–12835 (2020)." href="#ref-CR36" id="ref-link-section-d73616779e2004_1">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Devlin, J. R. et al. Structural dissimilarity from self drives neoepitope escape from immune tolerance. Nat. Chem. Biol. 16, 1269–1276 (2020)." href="/articles/s41591-022-01786-3#ref-CR37" id="ref-link-section-d73616779e2007">37</a></sup> receptors. TCR3, by comparison, has a modestly lower affinity (<i>K</i><sub>d</sub> = 200 ± 22 μM) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig9">3</a>).</p><p>We next crystallized and determined the X-ray structures of each TCR/pMut/HLA-A*03:01 ternary complex at resolutions of 3.1 Å and 2.1 Å, respectively (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig4">4d</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig9">3a</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">2</a>). Both TCRs dock using a conventional, diagonal orientation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig4">4e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig9">3b</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Gowthaman, R. &amp; Pierce, B. G. TCR3d: the T cell receptor structural repertoire database. Bioinformatics 35, 5323–5325 (2019)." href="/articles/s41591-022-01786-3#ref-CR32" id="ref-link-section-d73616779e2038">32</a></sup>. In most TCR structures, the two CDR3 loops focus solely on the central regions of the peptide<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Singh, N. K. et al. Geometrical characterization of T cell receptor binding modes reveals class-specific binding to maximize access to antigen. Proteins 88, 503–513 (2020)." href="/articles/s41591-022-01786-3#ref-CR38" id="ref-link-section-d73616779e2042">38</a></sup>. This is not the case with TCR4. The CDR3α loop is positioned parallel to the core of the peptide backbone, extending from the P3 His to the P6 Trp (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig4">4f</a>). The unusually long CDR3β loop, on the other hand, reaches to the C-terminus of the peptide, forming hydrogen bonds with the side chain of the P8 Thr and the C-terminal carboxylate. By contrast, TCR3 engages only the central core of pMut (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig9">3c</a>). It is noteworthy that both TCRs induce a major repositioning of the P6 Trp side chain upon binding (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig10">4</a>). In this conformation, the residue becomes buried along the α<sub>2</sub> helix of HLA-A*03:01 where it cannot be contacted by either receptor. We generated interatomic contact matrices to identify and quantify interactions between each TCR and the pMut/HLA-A*03:01 complex (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig11">5</a>). Overall, TCR4 engages pMut with <i>n</i> = 30 intermolecular interactions, including <i>n</i> = 6 hydrogen bonds. TCR3, by contrast, engages pMut with far fewer contacts (<i>n</i> = 17) and only one hydrogen bond. Together, these data illustrate how the unique structural properties of TCR4, including its long CDR3β loop, permit the receptor to form an extended interface spanning nearly the entire length of pMut. By contrast, TCR3 engages pMut with more focused intermolecular interactions.</p><p>We next assessed the fine specificity of TCR4 and TCR3 using AA scanning to functionally define each receptor’s recognition motif (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig4">4g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig9">3d</a>). TCR-transduced T cells were co-cultured with HLA-A*03:01<sup>+</sup> target cells pulsed with Ala or Gly substituted peptides, and the effect of these changes on cytokine production was determined. Replacement with either AA at positions P5–P9 resulted in near complete loss of TCR4’s function. By contrast, TCR3 was either completely or partially tolerant to changes at P8, correlating with the receptor’s minimal contact to this region of pMut. Similarly, whereas alterations in P2 and P3 also led to loss of TCR4 function, TCR3 was permissive of substitutions at these sites.</p><p>To establish TCR4’s cross-reactivity potential, we selected residues that permitted reactivity when altered regardless of magnitude and identified TCR4’s peptide recognition motif as ‘x-x-x-x-G-W-T-T-K’ (‘x’ = any AA). Using the ScanProsite tool<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="de Castro, E. et al. ScanProsite: detection of PROSITE signature matches and ProRule-associated functional and structural residues in proteins. Nucleic Acids Res. 34, W362–W365 (2006)." href="/articles/s41591-022-01786-3#ref-CR39" id="ref-link-section-d73616779e2085">39</a></sup>, we surveyed the human proteome for sequences containing this motif. This search afforded two candidates: WT PI3Kα, as expected, and an unrelated peptide derived from transmembrane protein 87B (<i>TM87B</i>) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">3</a>). Notably, the TM87B-derived peptide failed to activate TCR4-transduced T cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig4">4h</a>). We conclude that TCR4 binds the pMut/HLA-I complex in a unique manner that maximizes CDR3 interactions across the length of pMut, thus contributing to its relatively high-affinity and low cross-reactivity profile.</p><h3 class="c-article__sub-heading" id="Sec7">Therapeutic targeting of a <i>PIK3CA</i> public NeoAg</h3><p>We next established TCR4’s capacity to therapeutically engage cancer cells. First, we measured the response of TCR4-transduced T cells to a patient-derived xenograft (PDX) generated from an HLA-A*03:01<sup>+</sup> patient with Mut <i>PIK3CA</i> uterine serous carcinoma (PDX USC_X10) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig12">6a</a>). Next-generation sequencing (NGS) and fluorescence-activated cell sorting (FACS) analysis confirmed that the mutational landscape and HLA-I haplotype of the PDX matched the patient’s primary tumor except for a subclonal <i>ARID5B</i> mutation (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig12">6b,c</a>). To quantify T cell recognition of tumor cells, we measured changes in 4-1BB expression. Compared to TCR4-transduced T cells alone, co-culture with the PDX resulted in a 15.3 ± 0.7-fold increase in 4-1BB (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig12">6d</a>). This change was significantly blocked by an anti-HLA-I antibody (<i>P</i> &lt; 0.001).</p><p>The prolonged passage time of PDX USC_X10 precluded its use for in vivo therapeutic experiments. Therefore, we developed an isogenic tumor model using HCC70, a breast adenocarcinoma cell line that expresses HLA-A*03:01 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig13">7a,b</a>). HCC70 harboring WT <i>PIK3CA</i> (HCC70-WT <i>PIK3CA</i>) and <i>PIK3CA</i> (H1047L) (HCC70-Mut <i>PIK3CA</i>) efficiently engrafted in immune-deficient NOD scid gamma (NSG) mice. However, the growth rate of HCC70-Mut <i>PIK3CA</i> was significantly greater than its WT counterpart (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig13">7c,d</a>), a finding consistent with Mut <i>PIK3CA</i>’s function as a driver oncogene. Explanted HCC70-Mut <i>PIK3CA</i>, but not HCC70-WT <i>PIK3CA</i>, triggered 4-1BB upregulation on TCR4-transduced T cells ex vivo (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig13">7e,f</a>).</p><p>Using this model, we tested whether adoptive cell transfer (ACT) of TCR4-transduced CD8<sup>+</sup> T cells could mediate regression of established HCC70 tumors without causing off-target effects. For these experiments, we used an HLA-A*03:01-restricted influenza nucleoprotein (Flu) TCR as a specificity control (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">9</a>). After HCC70-WT <i>PIK3CA</i> or HCC70-Mut <i>PIK3CA</i> tumors grew to ~75 mm<sup>3</sup>, mice were randomized to receive by intravenous injection one of the following treatments: (1) TCR4-transduced CD8<sup>+</sup> T cells, (2) CD8<sup>+</sup> T cells transduced with the Flu-specific TCR or (3) a PBS control (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig5">5a</a>). All mice received an extended half-life variant of IL-15 to mimic the physiologic effect of host lymphodepletion<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Gattinoni, L. et al. Removal of homeostatic cytokine sinks by lymphodepletion enhances the efficacy of adoptively transferred tumor-specific CD8+ T cells. J. Exp. Med. 202, 907–912 (2005)." href="/articles/s41591-022-01786-3#ref-CR40" id="ref-link-section-d73616779e2193">40</a></sup>. Mice with HCC70-Mut <i>PIK3CA</i> tumors and treated with TCR4 had a significant reduction in tumor volumes and enhanced survival compared to mice that received the Flu-specific TCR or PBS controls (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig5">5b,c</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">10a</a>). By contrast, no significant differences were observed in tumor growth or survival between treatment arms in mice with HCC70-WT <i>PIK3CA</i> tumors (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig5">5d,e</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">10b</a>). This latter finding suggests that TCR4 did not mediate an off-target effect on cells expressing WT <i>PIK3CA</i>. None of the TCR-treated mice experienced a significant loss of body weight compared to PBS controls (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">11</a>). We conclude that ACT of TCR4-transduced T cells safely mediates Mut-specific anti-tumor responses against established tumors in vivo.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="In vivo anti-tumor efficacy of adoptively transferred T cells genetically engineered with a PIK3CA public NeoAg TCR."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: In vivo anti-tumor efficacy of adoptively transferred T cells genetically engineered with a <i>PIK3CA</i> public NeoAg TCR.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="734"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Experimental overview for the evaluation of the in vivo anti-tumor efficacy and safety of targeting Mut <i>PIK3CA</i> using TCR-transduced CD8<sup>+</sup> T cells. Mice were randomized between indicated treatment groups once subcutaneously (s.c.) implanted HCC70-Mut <i>PIK3CA</i> or HCC70-WT <i>PIK3CA</i> tumors were established to ~75 mm<sup>3</sup>. All mice received a twice-weekly intraperitoneal injection of 1 μg of IL-15 pre-complexed with IL-15Rα-Fc (1:1 M) and intravenous injection of CD8<sup>+</sup> T cells transduced with TCR4, CD8<sup>+</sup> T cells transduced with an influenza (Flu)-specific TCR, or PBS. Mice received 7.5 × 10<sup>6</sup> and 2.5 × 10<sup>6</sup> TCR<sup>+</sup> T cells on D0 and D3 after randomization. Tumor volumes (<b>b</b>) and overall survival (<b>c</b>) of mice bearing HCC70-Mut <i>PIK3CA</i> tumors infused with the indicated treatment. Tumor volumes (<b>d</b>) and overall survival (<b>e</b>) of mice bearing established HCC70-WT <i>PIK3CA</i> tumors infused with the indicated treatment. Data in <b>b</b> and <b>d</b> are shown as mean ± s.e.m. and are representative of two independently performed experiments (TCR4 <i>n</i> = 10, Flu TCR <i>n</i> = 5, PBS <i>n</i> = 5). ***<i>P</i> &lt; 0.001 and NS using two-way ANOVA. Pooled survival data from two identically performed experiments are shown in <b>c</b> and <b>e</b> and are plotted as a Kaplan–Meier survival curve (TCR4 <i>n</i> = 20, Flu TCR <i>n</i> = 10, PBS <i>n</i> = 10). ***<i>P</i> &lt; 0.001 and NS using log-rank test. D, day; NS, not significant.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec8">Clonality and immunogenicity of a <i>PIK3CA</i> public NeoAg</h3><p>NeoAg clonal heterogeneity is a major resistance mechanism to T-cell-based immunotherapies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="McGranahan, N. et al. Clonal neoantigens elicit T cell immunoreactivity and sensitivity to immune checkpoint blockade. Science 351, 1463–1469 (2016)." href="#ref-CR4" id="ref-link-section-d73616779e2342">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Miao, D. et al. Genomic correlates of response to immune checkpoint blockade in microsatellite-stable solid tumors. Nat. Genet. 50, 1271–1281 (2018)." href="#ref-CR5" id="ref-link-section-d73616779e2342_1">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Wolf, Y. et al. UVB-induced tumor heterogeneity diminishes immune response in melanoma. Cell 179, 219–235 e221 (2019)." href="/articles/s41591-022-01786-3#ref-CR6" id="ref-link-section-d73616779e2345">6</a></sup>. Because <i>PIK3CA</i> gain-of-function mutations are critical to cancer cell fitness<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Samuels, Y. et al. Mutant PIK3CA promotes cell growth and invasion of human cancer cells. Cancer Cell 7, 561–573 (2005)." href="/articles/s41591-022-01786-3#ref-CR41" id="ref-link-section-d73616779e2352">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Kang, S., Bader, A. G. &amp; Vogt, P. K. Phosphatidylinositol 3-kinase mutations identified in human cancer are oncogenic. Proc. Natl Acad. Sci. USA 102, 802–807 (2005)." href="/articles/s41591-022-01786-3#ref-CR42" id="ref-link-section-d73616779e2355">42</a></sup>, we hypothesized that Mut <i>PIK3CA</i> would be clonally conserved across patients and disease sites. To test this hypothesis, we determined the cancer cell fraction (CCF), a measurement of mutation clonality, in a pan-cancer cohort of <i>n</i> = 131 patient samples harboring <i>PIK3CA</i> (H1047L). We discovered that Mut <i>PIK3CA</i> was clonally expressed in most tumors (<i>n</i> = 102/131; 77.9%) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig6">6a</a>)<sub>.</sub> Critically, Mut <i>PIK3CA</i> clonality was observed in both primary and metastatic sites (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig6">6b</a>). Phylogenetic analysis using matched biospecimens revealed clonal preservation of Mut <i>PIK3CA</i> in the face of branched evolution for other mutated genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig6">6c</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="Clonality, immunogenicity and immune resistance to a PIK3CA public NeoAg in patients with cancer."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6: Clonality, immunogenicity and immune resistance to a <i>PIK3CA</i> public NeoAg in patients with cancer.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig6_HTML.png?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="843"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>, Pan-cancer analysis measuring the clonality of <i>PIK3CA</i> (H1047L) in <i>n</i> = 131 unique MSK patients with cancer. Clonality is defined as a CCF of ≥80%, indicated by the shaded gray area. <b>b</b>, Comparison of <i>PIK3CA</i> (H1047L) CCF in primary versus metastatic tumor sites. <i>P</i> = 0.0245 using two-sided Student’s <i>t</i>-test. <b>c</b>, Phylogenetic analysis measuring the clonal conservation of <i>PIK3CA</i> (H1047L) in primary (P) and metastatic (M) tumor sites within the same patient. <b>d</b>, Representative FACS and summary plot for the detection of circulating CD8<sup>+</sup> T cells specific for the pMut/HLA-A*03:01 (A*03:01) epitope in <i>n</i> = 14 HLA-A*03:01<sup>+</sup> patients with a history of a <i>PIK3CA</i> (H1047L) cancer or <i>n</i> = 5 HLA-A*03:01<sup>+</sup> healthy donors. Percentages in FACS plots represent the frequency of gated live<sup>+</sup>CD8<sup>+</sup>dual pMut/HLA-A*03:01 dextramer<sup>+</sup> lymphocytes. 0 = no detection; 1 = detection. <b>e</b>, <b>f</b>, Variant allele frequency (VAF) at each of the positions in <i>HLA-A</i> that are mismatched between <i>HLA-A*02:01</i> and <i>HLA-A*03:01</i> are shown for the PMBC and tumor samples of patient MSK_01 who demonstrated a tumor-specific <i>HLA-A*03:01</i> LOH in the setting of <i>PIK3CA</i> (H1047L) clonal conservation. <i>P</i> &lt; 0.0001 by two-sided Student’s <i>t</i>-test. Results in <b>a</b>, <b>b</b> and <b>f</b> are displayed as median ± 95% confidence interval. Violin distributions in <b>e</b> are centered around the median (solid horizontal line) with quartile ranges displayed above and below (dashed horizontal lines). The maxima and minima are represented by the top and bottom of each plot. ATC, anaplastic thyroid cancer; BLCA, bladder cancer; BRCA, breast invasive carcinoma; COAD, colon adenocarcinoma; DEX, HLA dextramer; UCEC, uterine corpus endometrial carcinoma.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01786-3/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>To establish whether the <i>PIK3CA</i> public NeoAg is spontaneously immunogenic, we collected PBMC samples from <i>n</i> = 14 HLA-A*03:01<sup>+</sup> patients with a history of a Mut <i>PIK3CA</i> cancer identified using MSK-Integrated Mutation Profiling of Actionable Cancer Targets (IMPACT) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig14">8</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">12</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Cheng, D. T. et al. Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (MSK-IMPACT): a hybridization capture-based next-generation sequencing clinical assay for solid tumor molecular oncology. J. Mol. Diagn. 17, 251–264 (2015)." href="#ref-CR43" id="ref-link-section-d73616779e2532">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rosenbaum, E. et al. HLA genotyping in synovial sarcoma: identifying HLA-A*02 and its association with clinical outcome. Clin. Cancer Res. 26, 5448–5455 (2020)." href="#ref-CR44" id="ref-link-section-d73616779e2532_1">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Eubank, M. H. et al. Automated eligibility screening and monitoring for genotype-driven precision oncology trials. J. Am. Med. Inform. Assoc. 23, 777–781 (2016)." href="/articles/s41591-022-01786-3#ref-CR45" id="ref-link-section-d73616779e2535">45</a></sup>. We analyzed patient PBMCs directly ex vivo and after IVS using dual fluorochrome-labeled HLA-A*03:01 multimers loaded with pMut. As controls, we analyzed PBMCs from <i>n</i> = 5 HLA-A*03:01<sup>+</sup> HDs using identical stimulation conditions. We detected <i>PIK3CA</i> public NeoAg-specific T cells in four of 14 (29%) patients and zero of five (0%) HDs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig6">6d</a>). Application of the SIFT-seq platform to PBMCs from patient MSK_02 successfully retrieved a patient-derived TCR that confers reactivity to target cells that express Mut, but not WT, <i>PIK3CA</i> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig15">9</a>). In patient MSK_04, we detected <i>PIK3CA</i> public NeoAg-specific T cells in serially collected samples, indicating a sustained response to this public NeoAg. Together, these data indicate that the <i>PIK3CA</i> public NeoAg is immunogenic and capable of driving T cell clonal expansion in vivo in a subset of cancer patients.</p><p>Immunogenic NeoAgs apply evolutionary pressure to cancer cells, selecting for populations with resistance to immune recognition<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Rosenthal, R. et al. Neoantigen-directed immune escape in lung cancer evolution. Nature 567, 479–485 (2019)." href="/articles/s41591-022-01786-3#ref-CR46" id="ref-link-section-d73616779e2566">46</a></sup>. Resistance can occur through multiple mechanisms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Chandran, S. S. &amp; Klebanoff, C. A. T cell receptor-based cancer immunotherapy: emerging efficacy and pathways of resistance. Immunol. Rev. 290, 127–147 (2019)." href="/articles/s41591-022-01786-3#ref-CR47" id="ref-link-section-d73616779e2570">47</a></sup>, including allele-specific HLA loss of heterozygosity (LOH)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="McGranahan, N. et al. Allele-specific HLA loss and immune escape in lung cancer evolution. Cell 171, 1259–1271 (2017)." href="/articles/s41591-022-01786-3#ref-CR48" id="ref-link-section-d73616779e2574">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Montesion, M. et al. Somatic HLA class I loss is a widespread mechanism of immune evasion which refines the use of tumor mutational burden as a biomarker of checkpoint inhibitor response. Cancer Discov. 11, 282–292 (2021)." href="/articles/s41591-022-01786-3#ref-CR49" id="ref-link-section-d73616779e2577">49</a></sup>. We sought to determine whether <i>PIK3CA</i> public NeoAg-expressing tumors exhibited evidence of cancer cell-intrinsic resistance to immunologic recognition. Targeted exome sequencing did not reveal loss-of-function mutations in genes associated with resistance to T cell immunotherapies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Zaretsky, J. M. et al. Mutations associated with acquired resistance to PD-1 blockade in melanoma. N. Engl. J. Med. 375, 819–829 (2016)." href="/articles/s41591-022-01786-3#ref-CR50" id="ref-link-section-d73616779e2584">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Shin, D. S. et al. Primary resistance to PD-1 blockade mediated by JAK1/2 mutations. Cancer Discov. 7, 188–201 (2017)." href="/articles/s41591-022-01786-3#ref-CR51" id="ref-link-section-d73616779e2587">51</a></sup>, including <i>TAP1/2</i>, <i>CALR</i>, <i>B2M</i>, <i>HLA-A</i>, <i>JAK1/2</i> and <i>IFNGR1</i>. By contrast, NGS data from <i>n</i> = 31 Mut <i>PIK3CA</i><sup>+</sup>/HLA-A*03:01<sup>+</sup> patients revealed HLA-A LOH involving the <i>HLA-A*03:01</i> allele in two patients (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig6">6e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig16">10a–d</a>), whereas no patients exhibited LOH involving the non-<i>HLA-A*03:01</i> allele. Both patients maintained clonal expression of Mut <i>PIK3CA</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig6">6f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig16">10b</a>). As controls, we measured the proportion of <i>HLA-A</i> LOH events involving <i>HLA-A*03:01</i> versus the alternative <i>HLA-A</i> allele in cancers with <i>PIK3CA</i> (E542K) or <i>PIK3CA</i> (E545K) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig16">10e</a>). Because neither of these <i>PIK3CA</i> mutations generate an HLA-A*03:01-restricted neoepitope (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig8">2</a>), we hypothesized that <i>HLA-A</i> LOH events would occur at random. Among HLA-A*03:01<sup>+</sup> patients with <i>PIK3CA</i> (E542K) and an <i>HLA-A</i> LOH event (<i>n</i> = 32), 13 of 32 (40.6%) lost <i>HLA-A*03:01</i>, whereas 19 of 32 (59.4%) lost the alternative <i>HLA-A</i> allele. Similarly, of <i>n</i> = 44 HLA-A*03:01<sup>+</sup> patients with a <i>PIK3CA</i> (E545K) cancer and an <i>HLA-A</i> LOH event, 23 of 44 (52.2%) lost <i>HLA-A*03:01</i>, whereas 21 of 44 (47.7%) lost an alternative <i>HLA-A</i> allele. Finally, we measured the proportion of <i>HLA-A</i> LOH events in HLA-A*03:01<sup>+</sup>/WT <i>PIK3CA</i> breast cancers. Here also, of <i>n</i> = 210 <i>HLA-A</i> LOH events, 99 of 210 (47.1%) involved <i>HLA-A*03:01</i>, whereas 111 of 210 (52.9%) involved the alternative <i>HLA-A</i> allele. These findings suggest an unbiased loss of one <i>HLA-A</i> allele over another in cases where a cancer is incapable of generating an HLA-A*03:01-restricted <i>PIK3CA</i> neoepitope. We conclude that the <i>PIK3CA</i> public NeoAg is typically clonal, immunogenic in patients with cancer and, in select cases, correlated with a mechanism of neoepitope-specific immune resistance.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec9-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">Discussion</h2><div class="c-article-section__content" id="Sec9-content"><p>Here we demonstrate that Mut <i>PIK3CA</i>, among the most common genomically altered driver genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. Cell 173, 371–385 (2018)." href="/articles/s41591-022-01786-3#ref-CR18" id="ref-link-section-d73616779e2749">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Priestley, P. et al. Pan-cancer whole-genome analyses of metastatic solid tumours. Nature 575, 210–216 (2019)." href="/articles/s41591-022-01786-3#ref-CR19" id="ref-link-section-d73616779e2752">19</a></sup>, is immunogenic in the context of a prevalent HLA-I allele, thereby creating a public NeoAg. Mechanistically, we determined that the molecular basis for <i>PIK3CA</i> public NeoAg immunogenicity results from the creation of an optimal HLA-I anchor residue. Unlike private NeoAgs, we discovered that the <i>PIK3CA</i> public NeoAg is shared among patients and frequently clonally conserved. Using SIFT-seq, we generated with high efficiency a panel of TCR gene sequences that confer specificity to this shared neopeptide from both HDs and a patient with cancer (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">4</a>). Several recent reports have described alternative methods for retrieving NeoAg-specific TCRs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Wolfl, M. &amp; Greenberg, P. D. Antigen-specific activation and cytokine-facilitated expansion of naive, human CD8+ T cells. Nat. Protoc. 9, 950–966 (2014)." href="#ref-CR52" id="ref-link-section-d73616779e2766">52</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Gee, M. H. et al. Antigen identification for orphan T cell receptors expressed on tumor-infiltrating lymphocytes. Cell 172, 549–563 (2018)." href="#ref-CR53" id="ref-link-section-d73616779e2766_1">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ali, M. et al. Induction of neoantigen-reactive T cells from healthy donors. Nat. Protoc. 14, 1926–1943 (2019)." href="#ref-CR54" id="ref-link-section-d73616779e2766_2">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Peng, S. et al. Sensitive detection and analysis of neoantigen-specific T cell populations from tumors and blood. Cell Rep. 28, 2728–2738 (2019)." href="#ref-CR55" id="ref-link-section-d73616779e2766_3">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Arnaud, M. et al. Sensitive identification of neoantigens and cognate TCRs in human solid tumors. Nat. Biotechnol. 
                  https://doi.org/10.1038/s41587-021-01072-6
                  
                 (2021)." href="/articles/s41591-022-01786-3#ref-CR56" id="ref-link-section-d73616779e2769">56</a></sup>. SIFT-seq uniquely incorporates six features that are relevant for TCR candidate development (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">5</a>), including establishing that a putative neopeptide is endogenously processed and presented. The importance of this latter attribute is highlighted by the following: although HLA-binding algorithms correctly anticipate the ALHGGWTTK neopeptide identified in our studies, more than 88% of predicted HLA-A*03:01-restricted high-affinity peptides derived from full-length Mut or WT <i>PIK3CA</i> were not detected by HLA-IP LC–MS/MS (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01786-3#Fig8">2</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">6</a>). TCR clonotypes with desirable specificity and binding attributes are expected to be rare. Screening strategies that focus exclusively on bona fide neoepitopes increases the likelihood of retrieving clonotypes with diverse and potentially unique attributes.</p><p>Stratification of our TCR panel members revealed that TCR4 is co-receptor-independent, functionally avid and capable of sustained cytolysis of Mut <i>PIK3CA</i><sup>+</sup>/HLA-A*03:01<sup>+</sup> target cells. Comparison of TCR4’s gene sequence to more than 17,000 TCRs restricted by HLA-A*03:01 and HLA-A*11:01 revealed that this receptor possesses an unusually long CDR3β loop. The functional significance of this feature was uncovered by resolving the structure of this receptor in complex with pMut/HLA-I. The CDR3 loops of most TCRs, including TCR3, engage the central core (P4–P7) of an HLA-I-bound peptide<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Garcia, K. C. &amp; Adams, E. J. How the T cell receptor sees antigen—a structural view. Cell 122, 333–336 (2005)." href="/articles/s41591-022-01786-3#ref-CR31" id="ref-link-section-d73616779e2794">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Singh, N. K. et al. Geometrical characterization of T cell receptor binding modes reveals class-specific binding to maximize access to antigen. Proteins 88, 503–513 (2020)." href="/articles/s41591-022-01786-3#ref-CR38" id="ref-link-section-d73616779e2797">38</a></sup>. However, the <i>PIK3CA</i> pMut/HLA-I complex presents a challenge: the surface it displays to a TCR is relatively ‘bland’ because it is dominated by the peptide’s backbone rather than exposed side chains typically associated with peptide immunogenicity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Schmidt, J. et al. Prediction of neo-epitope immunogenicity reveals TCR recognition determinants and provides insight into immunoediting. Cell Rep. Med. 2, 100194 (2021)." href="/articles/s41591-022-01786-3#ref-CR57" id="ref-link-section-d73616779e2804">57</a></sup>. Indeed, upon both TCR3 and TCR4 binding, the side chain of the neopeptide’s most distinctive residue (P6 Trp) becomes fully buried in the HLA-A*03:01 peptide-binding groove, further increasing its largely featureless nature. Recent efforts to rationalize the importance of bulky, multifunctional AAs such as Trp in peptide immunogenicity have emphasized their importance in forming key TCR contacts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Devlin, J. R. et al. Structural dissimilarity from self drives neoepitope escape from immune tolerance. Nat. Chem. Biol. 16, 1269–1276 (2020)." href="/articles/s41591-022-01786-3#ref-CR37" id="ref-link-section-d73616779e2809">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Schmidt, J. et al. Prediction of neo-epitope immunogenicity reveals TCR recognition determinants and provides insight into immunoediting. Cell Rep. Med. 2, 100194 (2021)." href="/articles/s41591-022-01786-3#ref-CR57" id="ref-link-section-d73616779e2812">57</a></sup>. The recognition of the <i>PIK3CA</i> public NeoAg by TCR3 and TCR4 defies such expectations.</p><p>TCR4’s unusually long CDR3β loop enables the receptor to form an extended and highly complementary interaction with the peptide. This unusual configuration likely contributes not only to TCR4’s affinity but also to its specificity as the receptor was found to lack measurable cross-reactivity to alternative epitopes derived from the human proteome. The exquisite specificity of TCR4, combined with (1) the tumor-specific expression of Mut <i>PIK3CA</i> and (2) the absence of a stable pWT/HLA complex on normal tissues, establishes a favorable safety profile for this receptor’s potential as a clinical candidate. Indeed, in proof-of-concept experiments, ACT of T cells genetically engineered with TCR4 caused mutation-specific tumor regression in the absence of measurable off-target effects.</p><p>Circulating T cells specific for the <i>PIK3CA</i> public NeoAg were detected in ~30% of HLA-A*03:01<sup>+</sup>/Mut <i>PIK3CA</i><sup>+</sup> patients with cancer. LOH for the restricting HLA-I allele of a NeoAg is a precise means of immune-editing that permits cancer cells to retain expression of a Mut driver allele<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="McGranahan, N. et al. Allele-specific HLA loss and immune escape in lung cancer evolution. Cell 171, 1259–1271 (2017)." href="/articles/s41591-022-01786-3#ref-CR48" id="ref-link-section-d73616779e2837">48</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Montesion, M. et al. Somatic HLA class I loss is a widespread mechanism of immune evasion which refines the use of tumor mutational burden as a biomarker of checkpoint inhibitor response. Cancer Discov. 11, 282–292 (2021)." href="/articles/s41591-022-01786-3#ref-CR49" id="ref-link-section-d73616779e2840">49</a></sup>. We found selective LOH for <i>HLA-A*03:01</i>, but not alternative <i>HLA-A</i> alleles, in two of 31 (7%) patients. By contrast, in HLA-A*03:01<sup>+</sup> patients with cancers harboring either WT <i>PIK3CA</i> or a <i>PIK3CA</i> mutation that does not generate a public NeoAg, <i>HLA-A</i> LOH events were randomly divided between <i>HLA-A*03:01</i> and the alternative <i>HLA-A</i> allele. Similar findings were recently reported using an independent pan-cancer dataset of more than 83,000 HLA-I-typed patients<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Montesion, M. et al. Somatic HLA class I loss is a widespread mechanism of immune evasion which refines the use of tumor mutational burden as a biomarker of checkpoint inhibitor response. Cancer Discov. 11, 282–292 (2021)." href="/articles/s41591-022-01786-3#ref-CR49" id="ref-link-section-d73616779e2869">49</a></sup>. Although these observations will require additional experiments to establish causality, they suggest that the immunogenicity of <i>PIK3CA</i> public NeoAg might lead to acquired immune-escape while preserving oncogenic driver expression in a subset of patients.</p><p>Across our cohort of public NeoAg-expressing cancers, targeted exome sequencing did not reveal alternative genomic alterations that might compromise antigen processing and presentation. However, this finding does not preclude tumor cell-intrinsic transcriptional and/or epigenetic mechanisms of antigen presentation immune-escape, which have also recently been reported<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Rosenthal, R. et al. Neoantigen-directed immune escape in lung cancer evolution. Nature 567, 479–485 (2019)." href="/articles/s41591-022-01786-3#ref-CR46" id="ref-link-section-d73616779e2880">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Paulson, K. G. et al. Acquired cancer resistance to combination immunotherapy from transcriptional loss of class I HLA. Nat. Commun. 9, 3868 (2018)." href="/articles/s41591-022-01786-3#ref-CR58" id="ref-link-section-d73616779e2883">58</a></sup>. We have assembled and studied what is, to our knowledge, the largest collection of biospecimens from patients who express an identical NeoAg resulting from an NSSM. Nevertheless, a limitation of our study is that the sample size remains underpowered to conclusively establish correlates among patient demographics, tumor-specific parameters and immunogenicity. Finally, although we demonstrated that Mut <i>PIK3CA</i> is typically clonal, gain-of-function mutations in <i>PIK3CA</i> often occur as a late evolutionary event during tumorigenesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Samuels, Y. et al. High frequency of mutations of the PIK3CA gene in human cancers. Science 304, 554 (2004)." href="/articles/s41591-022-01786-3#ref-CR17" id="ref-link-section-d73616779e2893">17</a></sup>. Consequently, it is possible that a WT <i>PIK3CA</i> cancer may recur despite an effective T cell response targeting the <i>PIK3CA</i> public NeoAg.</p><p>Although we focused on Mut <i>PIK3CA</i>, the modular nature of the SIFT-seq platform might readily be applied to the discovery of public NeoAg-specific TCRs resulting from the more than 200 additional driver genes identified to date<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. Cell 173, 371–385 (2018)." href="/articles/s41591-022-01786-3#ref-CR18" id="ref-link-section-d73616779e2910">18</a></sup>. Recent regulatory approvals have been made for tissue-agnostic therapies targeting specific genomic alterations identified using clinical NGS, including NTRK fusion inhibitors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Drilon, A. et al. Safety and antitumor activity of the multitargeted pan-TRK, ROS1, and ALK inhibitor entrectinib: combined results from two phase I trials (ALKA-372-001 and STARTRK-1). Cancer Discov. 7, 400–409 (2017)." href="/articles/s41591-022-01786-3#ref-CR59" id="ref-link-section-d73616779e2914">59</a></sup> and immune checkpoint inhibitors in microsatellite-unstable cancers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Marabelle, A. et al. Efficacy of pembrolizumab in patients with noncolorectal high microsatellite instability/mismatch repair-deficient cancer: results from the phase II KEYNOTE-158 study. J. Clin. Oncol. 38, 1–10 (2020)." href="/articles/s41591-022-01786-3#ref-CR60" id="ref-link-section-d73616779e2918">60</a></sup>. An analogous approach might be applied for the future clinical development of target-specific but tissue-agnostic TCR-based therapies targeting public NeoAgs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Chandran, S. S. &amp; Klebanoff, C. A. T cell receptor-based cancer immunotherapy: emerging efficacy and pathways of resistance. Immunol. Rev. 290, 127–147 (2019)." href="/articles/s41591-022-01786-3#ref-CR47" id="ref-link-section-d73616779e2922">47</a></sup>, including the <i>PIK3CA</i> public NeoAg described here.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec10-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">Methods</h2><div class="c-article-section__content" id="Sec10-content"><h3 class="c-article__sub-heading" id="Sec11">Primary cells and cell lines</h3><p>Leukopaks from HDs were purchased from the New York Blood Center. HLA-typed leukoreduction system chambers from HDs were obtained from the Stanford Blood Center. PBMCs were isolated by density-gradient centrifugation using lymphocyte separation medium (Corning) and cryopreserved until ready for use. High-resolution genomic <i>HLA</i> typing for HD PBMCs was performed by HistoGenetics. The retroviral packaging line 293GP was purchased from Takara Bio (cat. no. 631458), and the HCC70 cell line was purchased from the American Type Culture Collection (cat. no. CRL-2315). Isogenic HCC70 cell lines stably expressing either WT <i>PIK3CA</i> or Mut <i>PIK3CA</i> were obtained by RV infection, followed by subcloning. COS-7 cells were obtained through a material transfer agreement (MTA) from S. A. Rosenberg (National Cancer Institute). Human primary cells were cultured in complete RPMI 1640 (Gibco) media supplemented with antibiotics and 10% human AB serum (GeminiBio). Cell lines were cultured and maintained in RPMI 1640 (Gibco) media supplemented with Pen–Strep (Gibco), gentamycin (MP Biomedicals) and 10% FBS (GeminiBio).</p><h3 class="c-article__sub-heading" id="Sec12">PBMC collection from <i>PIK3CA</i> public NeoAg-expressing cancer patients</h3><p>All patients provided written informed consent for tumor and white blood cell sequencing and review of patient medical records for demographic, pathological and treatment information under a Memorial Sloan Kettering Cancer Center (MSKCC) institutional review board-approved biospecimen umbrella protocol (protocol 12-245; ClinicalTrials.gov ID: <a href="https://clinicaltrials.gov/ct2/show/NCT01775072">NCT01775072</a>). PBMCs were collected in CPT tubes (BD Biosciences) and processed according to the manufacturer’s instructions. Candidate patients were identified based on results from the MSK-IMPACT clinical NGS platform<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Cheng, D. T. et al. Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (MSK-IMPACT): a hybridization capture-based next-generation sequencing clinical assay for solid tumor molecular oncology. J. Mol. Diagn. 17, 251–264 (2015)." href="/articles/s41591-022-01786-3#ref-CR43" id="ref-link-section-d73616779e2970">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Rosenbaum, E. et al. HLA genotyping in synovial sarcoma: identifying HLA-A*02 and its association with clinical outcome. Clin. Cancer Res. 26, 5448–5455 (2020)." href="/articles/s41591-022-01786-3#ref-CR44" id="ref-link-section-d73616779e2973">44</a></sup>. DARWIN, an automated genotype-driven enrollment tool that matches NGS results with upcoming patient clinic visits<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Eubank, M. H. et al. Automated eligibility screening and monitoring for genotype-driven precision oncology trials. J. Am. Med. Inform. Assoc. 23, 777–781 (2016)." href="/articles/s41591-022-01786-3#ref-CR45" id="ref-link-section-d73616779e2977">45</a></sup>, was used to identify <i>PIK3CA</i> (H1047L)<sup>+</sup>/HLA-A*03:01<sup>+</sup> patients for biospecimen collection.</p><h3 class="c-article__sub-heading" id="Sec13">Plasmids and peptides</h3><p>All <i>PIK3CA</i> and <i>HLA</i> gene sequences were synthesized and cloned into the pcDNA3.1<sup>+</sup> vector (GenScript). Constructs encoding full-length and overlapping minigenes of WT <i>PIK3CA</i>, individual <i>PIK3CA</i> hotspot mutations (E542K, E545K and H1047L/R<i>)</i> and HLA-A*03:01, HLA-A*03:02, HLA-A*11:01, HLA-A*69:01, HLA-B*35:03, HLA-B*49:01, HLA-C*04:01 and HLA-C*07:01 were generated. Plasmid vectors were linearized using XbaI (New England Biolabs) and purified using the QIAquick PCR Purification Kit (Qiagen) following manufacturer instructions. In vitro transcribed RNA was generated using the HiScribe T7 High Yield RNA Synthesis Kit (New England Biolabs) following the manufacturer’s instructions. TCR α/β was synthesized and cloned into the pMSGV-1 RV plasmid, as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Stevanovic, S. et al. Landscape of immunogenic tumor antigens in successful immunotherapy of virally induced epithelial cancer. Science 356, 200–205 (2017)." href="/articles/s41591-022-01786-3#ref-CR61" id="ref-link-section-d73616779e3015">61</a></sup>. The human variable regions of retrieved TCRs were fused to a modified murine constant chain (mTCR) to facilitate identification of transduced T cells and promote proper chain pairing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Cohen, C. J., Zhao, Y., Zheng, Z., Rosenberg, S. A. &amp; Morgan, R. A. Enhanced antitumor activity of murine-human hybrid T-cell receptor (TCR) in human lymphocytes is associated with improved pairing and TCR/CD3 stability. Cancer Res. 66, 8878–8886 (2006)." href="/articles/s41591-022-01786-3#ref-CR62" id="ref-link-section-d73616779e3019">62</a></sup>. HPLC grade 9-mer peptides (&gt;99%) for pWT, pMut, alanine and glycine substituted variants of pMut and a peptide derived from TM87B were manufactured by GenScript.</p><h3 class="c-article__sub-heading" id="Sec14">IVS of HD and patient PBMCs</h3><p>PBMCs were plated in tissue culture flasks at 1 × 10<sup>6</sup> cells per cm<sup>2</sup> in complete media in the absence of cytokine for 2 hours at 37 °C to separate the adherent (monocyte-containing) and non-adherent (T-cell-containing) fractions. T cells were enriched from the non-adherent fraction by untouched negative selection (STEMCELL Technologies). To generate moDCs, the adherent fraction was washed with PBS, and fresh complete media supplemented with recombinant human IL-4 and GM-CSF (400 IU ml<sup>−1</sup>) were provided every alternate day. moDCs were stimulated with LPS (Invitrogen) and IFN-γ (Miltenyi Biotec) for 16–24 hours before transfection. moDCs were electroporated with 100 μg ml<sup>−1</sup> of mRNA encoding an individual <i>PIK3CA</i> hotspot mutation using the Neon Transfection system (10-μl tip, setting: 1,325 V/10 ms/3 pulses). HD T cells were co-cultured with electroporated moDCs at an effector-to-target (E:T) ratio of 3:1 in the presence of IL-21 (30 ng ml<sup>−1</sup>) in 24-well non-TC plates (Falcon). Wells were supplemented with fresh media containing IL-7 and IL-15 (10 ng ml<sup>−1</sup> each) every 3 days during the duration of the in vitro culture. All cytokines were purchased from Miltenyi Biotec. For patient-derived samples, PBMCs were enriched for CD8<sup>+</sup> T cells and stimulated with autologous monocytes pulsed with pMut (1 μg ml<sup>−1</sup>) in the presence of 300 IU ml<sup>−1</sup> of IL-2.</p><h3 class="c-article__sub-heading" id="Sec15">Mutation-specific qPCR screen</h3><p>Aliquots containing 2 × 10<sup>4</sup> cells from individual parent IVS wells were harvested, split equally into two daughter wells and stimulated with moDCs electroporated with mRNA encoding WT or Mut <i>PIK3CA</i> for 3 hours at an E:T ratio of 1:2. After stimulation, total mRNA was purified (Qiagen), transcribed into cDNA (ABI), and levels of <i>IFNG</i> transcript were quantified by real-time qPCR (ABI QuantStudio 5) using the TaqMan Fast Advanced MasterMix (ABI). Delta cycle threshold (Ct) values for matched Mut versus WT stimulated wells were calculated. Wells with a positive delta Ct value of ≥2 s.d. from the mean were selected for downstream single-cell immune-profiling using SIFT-seq.</p><h3 class="c-article__sub-heading" id="Sec16">Single-cell immune-profiling</h3><p>Single-cell RNA and V(D)J sequencing was performed using the 10x Genomics platform. One × 10<sup>5</sup> cells from a qPCR-positive well were harvested and co-cultured with moDCs electroporated with mRNA encoding WT or Mut <i>PIK3CA</i> at an E:T ratio of 1:10 for 3 hours. Final cell concentration was determined by cell counting on a hemocytometer, and cell concentration was adjusted to approximately 700 to 1,200 cells per microliter to maximize the likelihood of achieving the desired cell recovery target, with an initial cell viability of more than 90%. The single-cell suspension was mixed with RT Master Mix and loaded together with barcoded single-cell 5′ gel beads and partitioning oil onto Single Cell A Chip to generate gel beads in emulsion (GEMs) using Chromium Controller. Cell lysis and barcoded reverse transcription of RNAs from single cells were finished inside each GEM. Barcoded cDNA product was recovered through post GEM-RT cleanup and PCR amplification. cDNA quality control and quantification were determined by High Sensitivity D5000 DNA ScreenTape analysis (Agilent Technologies) and Qubit dsDNA HS Assay Kit (Thermo Fisher Scientific). Fifty nanograms of cDNA was used for 5′ gene expression library construction, and each sample was indexed by a Chromium i7 Sample Index Kit, which was run on an Illumina HiSeq 4000 sequencer with 2 × 100 base pairs (bp) paired reads to achieve at least 30,000 read pairs per cell. Full-length V, D and J gene segments were amplified using a Chromium Single Cell V(D)J Enrichment Kit (human T cells) to generate enrichment products. Enriched product was measured by D5000 DNA ScreenTape analysis and Qubit dsDNA HS Assay Kit, and then 50 ng of enrichment TCR product was used for library construction. Single Cell V(D)J enriched libraries were sequenced on HiSeq 4000 to produce paired 2 × 150-bp reads at 5,000 read pairs per cell. The raw single-cell RNA sequencing and VDJ data were pre-processed (demultiplexing of cellular barcodes, read alignment and generation of feature-barcode matrix) using Cell Ranger (10x Genomics, version 2.1.1) with the annotation files ‘vdj_GRCh38_alts_ensembl-3.1.0-3.1.0’ and ‘GRCh38-3.0.0’ for the V(D)J sequencing and gene expression libraries, respectively. Detailed quality control metrics were generated and evaluated using single-cell analysis packages from Bioconductor, including DropletUtils, scater and scran. Genes detected in fewer than three cells and cells where fewer than 200 genes had non-zero counts were filtered out and excluded from subsequent analysis. Low-quality cells with more than 15% of the read counts derived from the mitochondrial genome were also discarded. For downstream analyses, only cells for which clonotype information was available were retained. To remove likely doublet or multiplet captures, cells with more than 7,000 detected genes were discarded as well as cells for which more than two <i>TRA</i> or <i>TRB</i> sequences were detected. For SIFT-seq-based identification of (1) T cell clonotypes associated with a Mut <i>PIK3CA</i>-specific response, (2) gene expression pattern analyses of candidate clonotypes and (3) visualizations and TCR retrieval, custom R scripts were developed. V(D)J sequences of candidate T cell clonotypes were analyzed on the Loupe V(D)J browser 4.0.</p><h3 class="c-article__sub-heading" id="Sec17">HLA-IP LC–MS/MS</h3><p>COS-7 cells were co-electroporated with 100 μg ml<sup>−1</sup> each of mRNA encoding an individual <i>HLA</i> allele with either full-length or truncated WT or Mut <i>PIK3CA</i> using the Neon Transfection system (100-μl tip, setting: 1,050 V/10 ms/2 pulses). Next, 15–20 × 10<sup>6</sup> cells were electroporated per condition and plated in six-well non-TC plates overnight. Cells were harvested by incubating with 1 mM EDTA (Millipore Sigma) for 10 minutes at 37 °C. Harvested cells were pelleted and washed three times in ice-cold PBS (Gibco). Immunoprecipitation, HLA ligand separation and LC–MS/MS were performed as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Klatt, M. G. et al. Solving an MHC allele-specific bias in the reported immunopeptidome. JCI Insight 5, e141264 (2020)." href="/articles/s41591-022-01786-3#ref-CR63" id="ref-link-section-d73616779e3110">63</a></sup>. In brief, cells were lysed in 7.5 ml of 1% CHAPS (Millipore Sigma) for 1 hour at 4 °C, lysates were spun down for 1 hour with 20,000<i>g</i> at 4 °C, and supernatant fluids were isolated. For immunopurification of HLA-I ligands, 0.5 mg of W6/32 antibody (Bio X Cell) was bound to 40 mg of CN Br-activated sepharose and incubated with the protein lysate overnight. HLA complexes and binding peptides were eluted five times using 1% TFA. Peptides and HLA-I complexes were separated using C18 columns (Sep-Pak C18 1 cc Vac Cartridge, 50 mg of sorbent per cartridge, 37–55-μm particle size, Waters). C18 columns were pre-conditioned with 80% ACN (Millipore Sigma) in 0.1% TFA and equilibrated with two washes of 0.1% TFA. Samples were loaded, washed again twice with 0.1% TFA and eluted in 300 μl of 30%, 40% and 50% acetonitrile in 0.1% TFA. All three fractions were pooled, dried down using vacuum centrifugation and stored at −80 °C until further processing. HLA-I ligands were isolated by solid-phase extractions using in-house C18 mini-columns. Samples were analyzed by high-resolution/high-accuracy LC–MS/MS (Lumos Fusion, Thermo Fisher Scientific). MS and MS/MS were operated at resolutions of 60,000 and 30,000, respectively. Only charge states 1, 2 and 3 were allowed. The isolation window was chosen as 1.6 Thomson, and collision energy was set at 30%. For MS/MS, maximum injection time was 100 ms with an automatic gain control of 50,000. MS data were processed using Byonic software (version 2.7.84, Protein Metrics) through a custom-built computer server equipped with four Intel Xeon E5-4620 8-core CPUs operating at 2.2 GHz and 512 GB physical memory (Exxact Corporation). Protein FDR was disabled to allow complete assessment of potential peptide identifications. Oxidization of methionine, phosphorylation of serine, threonine and tyrosine, as well as N-terminal acetylation were set as variable modifications for all samples. Samples were searched against a database comprising UniProt <i>Cercopithecus</i> <i>aethiops</i> reviewed proteins supplemented with human HLA allele sequences used in this study, WT and Mut sequences of PI3Kα, as well as common contaminants. For mirror plots, ion intensities were exported and re-plotted with GraphPad Prism. For precursor quantitation as well as retention time analyses, Skyline software (version 4.2, MacCoss Lab Software) was used. Precursor masses of peptide target sequences were searched in all relevant.raw files, and peak areas of all replicates were compared. Retention times for the best-scoring matches were selected with a total of four isotopes.</p><h3 class="c-article__sub-heading" id="Sec18">TCR transduction</h3><p>The 293GP packaging line was plated overnight onto poly-<span class="u-small-caps">d</span>-lysine-coated 60-mm<sup>2</sup> plates at 1.6 × 10<sup>6</sup> cells per plate in complete DMEM. 293GP cells were transfected with 6 μg of a candidate TCR pMSGV1-plasmid along with 3 μg of the RD114 envelope using the Lipofectamine 3000 (Invitrogen) reagent set. Both the pMSGV1 and RD114 plasmids were obtained through an MTA from S. A. Rosenberg. Viral supernatant was collected after 48 hours and loaded onto retronectin-coated (10 ug ml<sup>−1</sup>, Takara Bio) non-TC 24-well plates and centrifuged at 2,000<i>g</i> for 2 hours at 32 °C. CD8<sup>+</sup> and CD4<sup>+</sup> T cells enriched from HD polyclonal PBMCs were stimulated with plate-bound OKT3 (5 μg ml<sup>−1</sup>, Miltenyi Biotec), anti-CD28 (2 μg ml<sup>−1</sup>, Miltenyi Biotec) and rhIL-2 (300 IU ml<sup>−1</sup>) for 48 hours before transduction. Stimulated cells were plated at 1–5 × 10<sup>5</sup> cells per well and spinoculated at 1,500 r.p.m. for 15 minutes. Cells were assessed for transduction efficiency after 3–4 days by measuring surface expression of mTCR by FACS.</p><h3 class="c-article__sub-heading" id="Sec19">T cell immunoassays</h3><p><u>Mutation specificity</u>: Specificity of candidate TCRs was assessed by intracellular cytokine staining (ICS) using the BD Cytofix/CytoPerm Plus Kit, following the manufacturer’s instructions. Autologous moDCs from MSK 21LT2 and 0606T donors were electroporated with 100 μg ml<sup>−1</sup> of mRNA encoding WT or Mut <i>PIK3CA</i> and plated into 96-well round-bottom plates overnight. To determine the class of HLA restriction, a set of plated moDCS were treated with an anti-HLA-A/B/C antibody (40 μg ml<sup>−1</sup>, Clone W6/32, BioLegend) or anti-class II (10 μg ml<sup>−1</sup>, Clone IVA12, MSKCC in-house) antibody for 3 hours at 37 °C. Candidate TCR-transduced T cells were co-cultured at an E:T ratio of 1:1 for 6 hours in the presence of anti-CD107A- BV650 (Clone H4A3, BioLegend) and Golgi block. PMA-ionomycin stimulation was included in all experiments as a positive control. Cells were washed in 1× PBS and surface labeled with Live/Dead fixable dye (Invitrogen), anti-CD3-APC-H7 (Clone SK7, Invitrogen), anti-CD4-Alexa Fluor 700 (Clone RPA-T4, Invitrogen), anti-CD8-efluor450 (Clone SK1, Invitrogen) and anti-mouse TCR-PerCpCy5.5 (Clone H57-597, Invitrogen) for 30 minutes at 4 °C. Cells were washed with 1× PBS and then fixed and permeabilized for 15 minutes at 4 °C. Surface-labeled cells were then washed with 1× perm-wash buffer and labeled with anti-IL-2-PE-Cy7 (Clone MQ1-17H12, Invitrogen), anti-TNFα-PE (Clone Mab11, Invitrogen) and anti-IFN-γ-FITC (Clone B27, BD) for 30 minutes at 4 °C in perm-wash buffer. All antibodies were used at a final concentration of 5 µg ml<sup>−1</sup>. Finally, cells were washed with perm-wash buffer, suspended in 2% FBS in PBS and acquired on an X20 LSR Fortessa flow cytometer with the BD FACSDiva software. Data were analyzed using FlowJo software version 10.6.2. Representative gating strategies are shown in Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">12</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM1">15</a>. <u>HLA-restriction determination</u>: COS-7 cells were co-electroporated with 100 μg ml<sup>−1</sup> each of mRNA encoding an individual <i>HLA</i> allele and either WT or Mut <i>PIK3CA</i>. Transfected COS-7 cells were plated overnight in a 96-well round-bottom plate in complete media to allow surface p/HLA-I complex expression. TCR-transduced T cells were added in at an E:T ratio of 1:1, and cytokine production was measured by ICS as described above. <u>TCR coreceptor dependence and functional avidity</u>: Target cells expressing HLA-A*03:01 were electroporated with titrated quantities of mRNA encoding Mut or WT <i>PIK3CA</i> and plated in 96-well round-bottom plates overnight. Enriched CD4<sup>+</sup> or CD8<sup>+</sup> T cells expressing an individual Mut <i>PIK3CA</i>-specific TCR were added in at a 1:1 E:T ratio. EC<sub>50</sub> values for individual TCRs were calculated by determining the antigen concentration that elicits 50% of maximal response (EC<sub>50</sub>). <u>TCR killing assay</u>: TCR cytolytic capability was measured by a tumor impedance assay using the xCELLigence System (ACEA Biosciences). Target cells expressing HLA-A*03:01 and either Mut or WT <i>PIK3CA</i> were plated overnight into custom xCELLigence E-plate 96-well flat-bottom plates. Before adding T cells, a baseline impedance measurement was taken. TCR-transduced T cells were added at an E:T ratio of 1:4, and impedance measurements were recorded at 15-minute intervals for up to 96 hours. Targets plated in complete media or exposed to 1% Triton X-100 allowed minimum and maximum lysis measurements, respectively. Percent cytolysis was calculated using RTCA Software Pro. <u>TCR recognition motif</u>: Target cells expressing HLA-A*03:01 were pulsed with pWT, pMut or variants of pMut that contain an Ala or Gly substitution at each individual peptide position (1 μg ml<sup>−1</sup>) for 60 minutes at 37 °C. Unbound peptide was washed off, and either TCR3-transduced or TCR4-transduced T cells were added at an E:T ratio of 1:1. Each TCR’s recognition motif was defined by positions resulting in ≥50% loss of function with an AA substitution compared to the native peptide sequence as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Cameron, B. J. et al. Identification of a Titin-derived HLA-A1-presented peptide as a cross-reactive target for engineered MAGE A3-directed T cells. Sci. Transl. Med. 5, 197ra103 (2013)." href="/articles/s41591-022-01786-3#ref-CR64" id="ref-link-section-d73616779e3226">64</a></sup>. <u>TCR4 cross-reactivity screening</u>: ScanProsite was used to search all UniProtKB/Swiss-Prot database sequences, including splice variants, for proteins containing the motif ‘X-X-X-X-G-W-T-T-K’. No filters were used. <u>PDX reactivity</u>: An established PDX (USC_X10) derived from an HLA-A*03:01<sup>+</sup> patient with <i>PIK3CA</i> (H1047L)<sup>+</sup> uterine serous cancer was explanted from an immune-deficient NSG mouse. Tumor cells were digested by enzymatic disruption with DNase I and collagenase IV, followed by filtering through a 100-µm filter to obtain a single-cell digest. Tumor cells were labeled with Live/Dead Fixable Dye (Invitrogen), anti-HLA-A/B/C-PerCpCy5.5 (Clone W6/32, BioLegend) and anti-HLA-A*03 (Clone GAP.A3, Invitrogen) for 30 minutes at 4 °C. Cells were washed with 1× PBS and resuspended in 2% FBS in PBS before acquisition by FACS. Then, 1 × 10<sup>5</sup> tumor cells were co-cultured with TCR4-tranduced T cells at an E:T ratio of 1:1 overnight at 37 °C in the absence or presence of an anti-HLA-A/B/C blocking antibody (40 μg ml<sup>−1</sup>, Clone W6/32). TCR4-transduced T cells cultured alone were used as a negative control. Reactivity was measured by changes in 4-1BB expression on mTCR<sup>+</sup>CD8<sup>+</sup> T cells using anti-CD137-APC (Clone 4B4-1, BD Biosciences).</p><h3 class="c-article__sub-heading" id="Sec20">ACT xenograft model</h3><p>All animal procedures were performed in accordance with an MSKCC Institutional Animal Care and Use Committee-approved protocol (19-08-013). Four to six-week-old female NOD.Cg-<i>Prkdc</i><sup>scid</sup> <i>Il2rg</i><sup><i>tm1Wjl</i></sup>/SzJ (NSG) mice were purchased from Jackson Laboratory and housed in pathogen-free conditions at the MSKCC vivarium. The mouse room maintained a 12-hour light/dark cycle, temperature of 65–75 <sup>°</sup>F and humidity levels of 40–60%. Three × 10<sup>6</sup> cells of the indicated HCC70 isogenic cell lines (HCC70-WT <i>PIK3CA</i> or HCC70-Mut <i>PIK3CA</i>) were subcutaneously implanted into the right flank of mice. Mice were randomized to indicated treatment groups once tumors were established to ~75 mm<sup>3</sup>. Tumor width and diameter as well as animal weights were measured by an investigator blinded to treatment conditions at baseline and twice weekly thereafter. TCR4-tranduced CD8<sup>+</sup> T cells were intravenously transferred at day 3 (7.5 × 10<sup>6</sup>) and day 7 (2.5 × 10<sup>6</sup>). Control groups received an equal number of CD8<sup>+</sup> T cells transduced with a previously described HLA-A*03:01-restricted Flu-specific TCR<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Pizzolla, A. et al. Influenza-specific lung-resident memory T cells are proliferative and polyfunctional and maintain diverse TCR profiles. J. Clin. Invest. 128, 721–733 (2018)." href="/articles/s41591-022-01786-3#ref-CR65" id="ref-link-section-d73616779e3293">65</a></sup> or PBS. Mice simultaneously received IL-15/IL-15Rα complex intraperitoneally every 3 days for the duration of the experiment, starting at the time of the first T cell transfer. The complex was freshly prepared by incubating recombinant human IL-15 (Miltenyi Biotec) with IL-15R-α-Fc (R&amp;D Systems) at a 1:1 molar ratio for 30 minutes at 37 °C and administered at the final concentration of 1 μg of IL-15 per mouse. The maximum tumor size permitted by the approved protocol is 1,000 mm<sup>3</sup> and this value was not exceeded.</p><h3 class="c-article__sub-heading" id="Sec21">Dextramer labeling</h3><p>HLA-A*03:01 multimers bound to pMut and conjugated to PE or APC were purchased from Immudex. Cells were labeled with dual fluorophore-conjugated dextramers for 10 minutes at room temperature, followed by surface antibodies against CD3, CD4, CD8 and viability dye for an additional 20 minutes at 4 °C. Cells were washed and acquired on a BD Fortessa X20 flow cytometer.</p><h3 class="c-article__sub-heading" id="Sec22">Recombinant protein synthesis and isolation</h3><p>Proteins for biophysical assays and crystallography, including the HLA-A*03:01 heavy chain, β<sub>2</sub>-microglobulin (β<sub>2</sub>m), the S3-4 pan-HLA TCR binding variant and the TCR α/β chains, were expressed in <i>Escherichia coli</i> as inclusion bodies and dissolved in 8 M urea and 6 M guanidinium-HCl. Denatured proteins were refolded and purified in vitro as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Davis-Harrison, R. L., Armstrong, K. M. &amp; Baker, B. M. Two different T cell receptors use different thermodynamic strategies to recognize the same peptide/MHC ligand. J. Mol. Biol. 346, 533–550 (2005)." href="/articles/s41591-022-01786-3#ref-CR66" id="ref-link-section-d73616779e3323">66</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Singh, N. K. et al. An engineered T cell receptor variant realizes the limits of functional binding modes. Biochemistry 59, 4163–4175 (2020)." href="/articles/s41591-022-01786-3#ref-CR67" id="ref-link-section-d73616779e3326">67</a></sup>. In brief, for TCR folding, TCR α/β chains at a 1:1 ratio were diluted into TCR refolding buffer (50 mM Tris-HCl, 2.5 M urea, 2 mM NaEDTA, 6.5 mM cysteamine, 3.7 mM cystamine and 0.2 mM PMSF, pH 8.15) and incubated at 4 °C overnight. The refolding buffer was then dialyzed against ddH<sub>2</sub>O and 10 mM Tris-HCl (pH 8.3) at 4 °C for 36 hours. For p/HLA folding, human β<sub>2</sub>m and HLA-A*03:01 were injected into the refolding buffer (400 mM L-arginine, 100 mM Tris-HCl, 2 mM NaEDTA, 6.3 mM cysteamine, 3.7 mM cystamine and 0.2 mM PMSF, pH 8.30), respectively, at a 3:1 ratio in the presence of a ten-fold excess of peptides at 4 °C. After an overnight incubation, the solution was dialyzed against ddH<sub>2</sub>O and 10 mM Tris-HCl (pH 8.3) at room temperature for 48 hours. The refolded proteins were then purified via ion exchange, followed by size-exclusion chromatography. Protein concentrations were determined through UV absorbance using sequence-determined extinction coefficients. WT and Mut PI3Kα peptides were purchased from AAPPTec or GenScript at &gt;80% purity, diluted to 30 mM in DMSO and stored at −80 °C.</p><h3 class="c-article__sub-heading" id="Sec23">Differential scanning fluorimetry</h3><p>Melting temperatures of WT and Mut PIK3α peptide/HLA-A*03:01 complexes were measured via differential scanning fluorimetry on a StepOnePlus Real-Time PCR System (Applied Biosystems), as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Hellman, L. M. et al. Differential scanning fluorimetry based assessments of the thermal and kinetic stability of peptide-MHC complexes. J. Immunol. Methods 432, 95–101 (2016)." href="/articles/s41591-022-01786-3#ref-CR68" id="ref-link-section-d73616779e3345">68</a></sup>. In brief, a 96-well plate was loaded with mixtures of 2 μl of 100× SYPRO Orange Protein Gel Stain (Invitrogen) and 18 μl of target protein at 15 μM in HBS-EP buffer (10 mM HEPES, 150 mM NaCl, 3 mM NaEDTA, 0.005% surfactant P20, pH 7.4). The excitation and emission wavelengths for the measurement were 587 nm and 607 nm, respectively. Temperature was scanned from 20 °C to 95 °C at a rate of 1 °C min<sup>−1</sup>. Melting temperatures were determined by fitting the 1st derivative of each melting curve to a bi-Gaussian function in OriginPro.</p><h3 class="c-article__sub-heading" id="Sec24">Fluorescence anisotropy</h3><p>Dissociation of WT and Mut PI3Kα peptides from peptide/HLA-A*03:01 complexes was determined through fluorescence anisotropy, as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Baxter, T. K. et al. Strategic mutations in the class I major histocompatibility complex HLA-A2 independently affect both peptide binding and T cell receptor recognition. J. Biol. Chem. 279, 29175–29184 (2004)." href="/articles/s41591-022-01786-3#ref-CR69" id="ref-link-section-d73616779e3359">69</a></sup>. In brief, purified peptide/HLA-A:03:01 complexes were generated using derivatives of the pWT and pMut peptides where the P5 Gly was substituted with a 5-carboxyfluorescein-modified lysine. Experiments were performed on a Beacon 2000 Fluorescence Polarization instrument (Invitrogen). Next, 100 nM fluorescein-labeled peptide/MHC complexes were mixed with 100 μM unlabeled peptide in 20 mM NaH<sub>2</sub>PO<sub>4</sub> and 75 mM NaCl (pH 7.4). The excitation wavelength was 488 nm, and anisotropy was detected at 535 nm. Changes in anisotropy were recorded as a function of time. Dissociation kinetics of peptides were determined by fitting the anisotropy curve to a single or biphasic exponential decay function in MATLAB. Half-lives were computed from the slowest dissociation rate from the relationship t<sub>1/2</sub> = 0.693/k<sub>off</sub>.</p><h3 class="c-article__sub-heading" id="Sec25">Protein crystallization</h3><p>Purified proteins were exchanged into 10 mM HEPES and 20 mM NaCl (pH 7.4) before crystallization. Crystals of pWT/HLA-A*03:01 grew in 10% PEG 8000, 200 mM calcium acetate and 100 mM HEPES (pH 7.5) at a protein concentration of 15 mg ml<sup>−1</sup>. Crystals of pMut/HLA-A*03:01 grew in 20% PEG 3350 and 200 mM ammonium formate (pH 6.6) at a protein concentration of 15 mg ml<sup>−1</sup>. Crystals of the TCR3-pMut/HLA-A*03:01 and TCR4-pMut/HLA-A*03:01 complexes grew in 10% PEG 8000 and 200 mM magnesium acetate at a protein concentration of 5 mg ml<sup>−1</sup>. Crystals were obtained by hanging drop vapor diffusion at 4 °C and were cryoprotected with 15% glycerol and flash-frozen before data collection.</p><h3 class="c-article__sub-heading" id="Sec26">X-ray diffraction and structure determination</h3><p>X-ray diffraction was performed at the beamline 22-ID or 24-ID-E of the Advanced Photon Source at Argonne National Laboratory. Diffraction data were processed with HKL2000 and solved by molecular replacement via Phaser in Phenix. The search model for the HLA-A*03:01 complexes was PDB 2XPG<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="McMahon, R. M. et al. Structure of HLA-A*0301 in complex with a peptide of proteolipid protein: insights into the role of HLA-A alleles in susceptibility to multiple sclerosis. Acta Crystallogr. D. Biol. Crystallogr. 67, 447–454 (2011)." href="/articles/s41591-022-01786-3#ref-CR70" id="ref-link-section-d73616779e3393">70</a></sup>. The search models for the complexes with TCR3 and TCR4 were built by Sculptor with the α chain from Protein Data Bank (PDB) <a href="https://doi.org/10.2210/pdb3QH3/pdb">3QH3</a> and the β chain from PDB <a href="https://www.rcsb.org/structure/4prh">4PRH</a><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Liu, Y. C. et al. A molecular basis for the interplay between T cells, viral mutants, and human leukocyte antigen micropolymorphism. J. Biol. Chem. 289, 16688–16698 (2014)." href="/articles/s41591-022-01786-3#ref-CR71" id="ref-link-section-d73616779e3410">71</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Scott, D. R., Borbulevych, O. Y., Piepenbrink, K. H., Corcelli, S. A. &amp; Baker, B. M. Disparate degrees of hypervariable loop flexibility control T-cell receptor cross-reactivity, specificity, and binding mechanism. J. Mol. Biol. 414, 385–400 (2011)." href="/articles/s41591-022-01786-3#ref-CR72" id="ref-link-section-d73616779e3413">72</a></sup>. The peptide and CDR loops were removed from the models before molecular replacement and manually rebuilt in Coot after the model was obtained from Phenix AutoBuild. Models were further refined automatically in Phenix and manually in Coot. Structures were visualized using PyMOL 2.3.4 and Discovery Studio 2019.</p><h3 class="c-article__sub-heading" id="Sec27">SPR</h3><p>Binding measurements between either TCR3 or TCR4 and the pMut/HLA-A*03:01 and pWT/HLA-A*03:01 complexes were performed using SPR on a Biacore T200 instrument, as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Devlin, J. R. et al. Structural dissimilarity from self drives neoepitope escape from immune tolerance. Nat. Chem. Biol. 16, 1269–1276 (2020)." href="/articles/s41591-022-01786-3#ref-CR37" id="ref-link-section-d73616779e3426">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Singh, N. K. et al. An engineered T cell receptor variant realizes the limits of functional binding modes. Biochemistry 59, 4163–4175 (2020)." href="/articles/s41591-022-01786-3#ref-CR67" id="ref-link-section-d73616779e3429">67</a></sup>. In brief, proteins were exchanged into 10 mM HEPES, 150 mM NaCl, 3 mM EDTA and 0.005% surfactant P-20, pH 7.4. The TCRs were immobilized on a CM5 Series S sensor chip to 900−1,300 response units via amine coupling. Experiments were performed at 25 °C with blank activated and deactivated flow cells as a reference. pMHC complexes were injected at a flow rate of 5 μl ml<sup>−1</sup> as analytes in duplicate. A pan HLA-I TCR variant that binds the ‘side’ of HLA-I molecules was used to verify the stability of the p/HLA complex during SPR measurements<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Singh, N. K. et al. An engineered T cell receptor variant realizes the limits of functional binding modes. Biochemistry 59, 4163–4175 (2020)." href="/articles/s41591-022-01786-3#ref-CR67" id="ref-link-section-d73616779e3435">67</a></sup>. Binding affinities were determined by fitting the steady-state data from both sets of injections to a 1:1 binding model in OriginPro.</p><h3 class="c-article__sub-heading" id="Sec28">
                           <i>PIK3CA</i> clonality, tumor phylogeny and <i>HLA</i> LOH</h3><p>The proportion of cancer cells harboring <i>PIK3CA</i> (H1047L) (CCF) was estimated by integrating the number of copies of the mutant allele, the purity of the tumor and the variant allele frequency of the mutation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="McGranahan, N. et al. Clonal status of actionable driver events and the timing of mutational processes in cancer evolution. Sci. Transl. Med. 7, 283ra254 (2015)." href="/articles/s41591-022-01786-3#ref-CR73" id="ref-link-section-d73616779e3457">73</a></sup>. Allele-specific copy number inferences were made using FACETS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Shen, R. &amp; Seshan, V. E. FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing. Nucleic Acids Res. 44, e131 (2016)." href="/articles/s41591-022-01786-3#ref-CR74" id="ref-link-section-d73616779e3461">74</a></sup>, and the number of copies of the mutant allele was inferred as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Bielski, C. M. et al. Widespread selection for oncogenic mutant allele imbalance in cancer. Cancer Cell 34, 852–862 (2018)." href="/articles/s41591-022-01786-3#ref-CR75" id="ref-link-section-d73616779e3465">75</a></sup>. Mutations were deemed clonal if the CCF value is &gt;0.8 or if the CCF value is &gt;0.7 and the upper bound of the 95th percentile confidence interval is &gt;0.9. All mutations for which CCF was determinable but that did not meet the criteria for being clonal were designated as sub-clonal. HLA genotypes from MSK-IMPACT were inferred using POLYSOLVER<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Rosenbaum, E. et al. HLA genotyping in synovial sarcoma: identifying HLA-A*02 and its association with clinical outcome. Clin. Cancer Res. 26, 5448–5455 (2020)." href="/articles/s41591-022-01786-3#ref-CR44" id="ref-link-section-d73616779e3469">44</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Shukla, S. A. et al. Comprehensive analysis of cancer-associated somatic mutations in class I HLA genes. Nat. Biotechnol. 33, 1152–1158 (2015)." href="/articles/s41591-022-01786-3#ref-CR76" id="ref-link-section-d73616779e3472">76</a></sup>. LOH at HLA-I alleles was inferred using LOHHLA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="McGranahan, N. et al. Allele-specific HLA loss and immune escape in lung cancer evolution. Cell 171, 1259–1271 (2017)." href="/articles/s41591-022-01786-3#ref-CR48" id="ref-link-section-d73616779e3477">48</a></sup>. At each of the HLA-I loci, LOH was called if the median copy number calculated by LOHHLA was less than 0.5 and the <i>P</i> value reflecting the allelic imbalance was less than 0.001. To visualize LOH at the <i>HLA</i> gene locus, we identified mismatched sites between the two <i>HLA-A</i> alleles and then computed the total number of reads aligning to each mismatched site in each tissue site.</p><h3 class="c-article__sub-heading" id="Sec29">Statistics and reproducibility</h3><p>No statistical methods were used to predetermine sample size. Appropriate statistical tests were used to analyze data, as described in each figure legend. Statistical analyses were performed with GraphPad Prism version 8.4 software. Significance was preset at <i>P</i> &lt; 0.05. In vitro experimental data were generated from two or more independent experiments containing <i>n</i> = 3 biological replicates per condition per experiment. For in vivo mouse experiments, treatment groups had <i>n</i> = 10 mice per condition, and control groups had <i>n</i> = 5 mice. Mice were randomized into three groups after tumor implantation. An investigator blinded to treatment groups performed both data acquisition and data analysis. In vivo experiments were independently repeated two times, and all attempts at replication were successful. For both in vitro and in vivo experiments, no data were excluded from any analysis.</p><h3 class="c-article__sub-heading" id="Sec30">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01786-3#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>Single-cell RNA sequencing datasets are deposited in the Gene Expression Omnibus database (<a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE172403">GSE172403</a>). Structural data, including coordinates and structure factors of pWT/HLA-I, pMut/HLA-I and the ternary TCR4/pMut/HLA-I and TCR3/pMut/HLA-I complexes, are available at the Protein Data Bank (<a href="https://www.rcsb.org/">https://www.rcsb.org/</a>) under PDB accession codes <a href="https://www.rcsb.org/structure/7L1b">7L1B</a>, <a href="https://www.rcsb.org/structure/7L1C">7L1C</a>, <a href="https://www.rcsb.org/structure/7L1d">7L1D</a> and <a href="https://www.rcsb.org/structure/7rrg">7RRG</a>. Additional materials are available upon reasonable request to C.A.K. (klebanoc@mskcc.org).</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>Custom codes to analyze SIFT-seq data are available on GitHub at <a href="https://github.com/abcwcm/Chandran2021">https://github.com/abcwcm/Chandran2021</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Schumacher, T. N. &amp; Schreiber, R. D. Neoantigens in cancer immunotherapy. <i>Science</i> <b>348</b>, 69–74 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aaa4971" data-track-item_id="10.1126/science.aaa4971" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aaa4971" aria-label="Article reference 1" data-doi="10.1126/science.aaa4971">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXls1Wmurc%3D" aria-label="CAS reference 1">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25838375" aria-label="PubMed reference 1">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Neoantigens%20in%20cancer%20immunotherapy&amp;journal=Science&amp;doi=10.1126%2Fscience.aaa4971&amp;volume=348&amp;pages=69-74&amp;publication_year=2015&amp;author=Schumacher%2CTN&amp;author=Schreiber%2CRD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Parkhurst, M. R. et al. Unique neoantigens arise from somatic mutations in patients with gastrointestinal cancers. <i>Cancer Discov.</i> <b>9</b>, 1022–1035 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1158/2159-8290.CD-18-1494" data-track-item_id="10.1158/2159-8290.CD-18-1494" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1158%2F2159-8290.CD-18-1494" aria-label="Article reference 2" data-doi="10.1158/2159-8290.CD-18-1494">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFGjs77L" aria-label="CAS reference 2">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31164343" aria-label="PubMed reference 2">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7138461" aria-label="PubMed Central reference 2">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Unique%20neoantigens%20arise%20from%20somatic%20mutations%20in%20patients%20with%20gastrointestinal%20cancers&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-18-1494&amp;volume=9&amp;pages=1022-1035&amp;publication_year=2019&amp;author=Parkhurst%2CMR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Hu, Z., Ott, P. A. &amp; Wu, C. J. Towards personalized, tumour-specific, therapeutic vaccines for cancer. <i>Nat. Rev. Immunol.</i> <b>18</b>, 168–182 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nri.2017.131" data-track-item_id="10.1038/nri.2017.131" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnri.2017.131" aria-label="Article reference 3" data-doi="10.1038/nri.2017.131">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFGmtbbL" aria-label="CAS reference 3">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29226910" aria-label="PubMed reference 3">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20personalized%2C%20tumour-specific%2C%20therapeutic%20vaccines%20for%20cancer&amp;journal=Nat.%20Rev.%20Immunol.&amp;doi=10.1038%2Fnri.2017.131&amp;volume=18&amp;pages=168-182&amp;publication_year=2018&amp;author=Hu%2CZ&amp;author=Ott%2CPA&amp;author=Wu%2CCJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">McGranahan, N. et al. Clonal neoantigens elicit T cell immunoreactivity and sensitivity to immune checkpoint blockade. <i>Science</i> <b>351</b>, 1463–1469 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aaf1490" data-track-item_id="10.1126/science.aaf1490" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aaf1490" aria-label="Article reference 4" data-doi="10.1126/science.aaf1490">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XksFCht70%3D" aria-label="CAS reference 4">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26940869" aria-label="PubMed reference 4">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4984254" aria-label="PubMed Central reference 4">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20neoantigens%20elicit%20T%20cell%20immunoreactivity%20and%20sensitivity%20to%20immune%20checkpoint%20blockade&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf1490&amp;volume=351&amp;pages=1463-1469&amp;publication_year=2016&amp;author=McGranahan%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Miao, D. et al. Genomic correlates of response to immune checkpoint blockade in microsatellite-stable solid tumors. <i>Nat. Genet.</i> <b>50</b>, 1271–1281 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0200-2" data-track-item_id="10.1038/s41588-018-0200-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0200-2" aria-label="Article reference 5" data-doi="10.1038/s41588-018-0200-2">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2nsrvM" aria-label="CAS reference 5">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30150660" aria-label="PubMed reference 5">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6119118" aria-label="PubMed Central reference 5">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20correlates%20of%20response%20to%20immune%20checkpoint%20blockade%20in%20microsatellite-stable%20solid%20tumors&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0200-2&amp;volume=50&amp;pages=1271-1281&amp;publication_year=2018&amp;author=Miao%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Wolf, Y. et al. UVB-induced tumor heterogeneity diminishes immune response in melanoma. <i>Cell</i> <b>179</b>, 219–235 e221 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2019.08.032" data-track-item_id="10.1016/j.cell.2019.08.032" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.08.032" aria-label="Article reference 6" data-doi="10.1016/j.cell.2019.08.032">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslOgt7vK" aria-label="CAS reference 6">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31522890" aria-label="PubMed reference 6">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6863386" aria-label="PubMed Central reference 6">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=UVB-induced%20tumor%20heterogeneity%20diminishes%20immune%20response%20in%20melanoma&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.08.032&amp;volume=179&amp;pages=219-235%20e221&amp;publication_year=2019&amp;author=Wolf%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Reiter, J. G. et al. Minimal functional driver gene heterogeneity among untreated metastases. <i>Science</i> <b>361</b>, 1033–1037 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aat7171" data-track-item_id="10.1126/science.aat7171" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aat7171" aria-label="Article reference 7" data-doi="10.1126/science.aat7171">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1Chsb7K" aria-label="CAS reference 7">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30190408" aria-label="PubMed reference 7">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6329287" aria-label="PubMed Central reference 7">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Minimal%20functional%20driver%20gene%20heterogeneity%20among%20untreated%20metastases&amp;journal=Science&amp;doi=10.1126%2Fscience.aat7171&amp;volume=361&amp;pages=1033-1037&amp;publication_year=2018&amp;author=Reiter%2CJG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Klebanoff, C. A. &amp; Wolchok, J. D. Shared cancer neoantigens: making private matters public. <i>J. Exp. Med.</i> <b>215</b>, 5–7 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1084/jem.20172188" data-track-item_id="10.1084/jem.20172188" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1084%2Fjem.20172188" aria-label="Article reference 8" data-doi="10.1084/jem.20172188">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2rt7vJ" aria-label="CAS reference 8">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29269561" aria-label="PubMed reference 8">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5748868" aria-label="PubMed Central reference 8">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Shared%20cancer%20neoantigens%3A%20making%20private%20matters%20public&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20172188&amp;volume=215&amp;pages=5-7&amp;publication_year=2018&amp;author=Klebanoff%2CCA&amp;author=Wolchok%2CJD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Wolfel, T. et al. A p16<sup>INK4a</sup>-insensitive CDK4 mutant targeted by cytolytic T lymphocytes in a human melanoma. <i>Science</i> <b>269</b>, 1281–1284 (1995).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.7652577" data-track-item_id="10.1126/science.7652577" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.7652577" aria-label="Article reference 9" data-doi="10.1126/science.7652577">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaK2MzotFSltQ%3D%3D" aria-label="CAS reference 9">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7652577" aria-label="PubMed reference 9">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20p16INK4a-insensitive%20CDK4%20mutant%20targeted%20by%20cytolytic%20T%20lymphocytes%20in%20a%20human%20melanoma&amp;journal=Science&amp;doi=10.1126%2Fscience.7652577&amp;volume=269&amp;pages=1281-1284&amp;publication_year=1995&amp;author=Wolfel%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Linard, B. et al. A <i>ras</i>-mutated peptide targeted by CTL infiltrating a human melanoma lesion. <i>J. Immunol.</i> <b>168</b>, 4802–4808 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.4049/jimmunol.168.9.4802" data-track-item_id="10.4049/jimmunol.168.9.4802" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.4049%2Fjimmunol.168.9.4802" aria-label="Article reference 10" data-doi="10.4049/jimmunol.168.9.4802">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38Xjt12ju74%3D" aria-label="CAS reference 10">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11971032" aria-label="PubMed reference 10">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20ras-mutated%20peptide%20targeted%20by%20CTL%20infiltrating%20a%20human%20melanoma%20lesion&amp;journal=J.%20Immunol.&amp;doi=10.4049%2Fjimmunol.168.9.4802&amp;volume=168&amp;pages=4802-4808&amp;publication_year=2002&amp;author=Linard%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Schumacher, T. et al. A vaccine targeting mutant IDH1 induces antitumour immunity. <i>Nature</i> <b>512</b>, 324–327 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature13387" data-track-item_id="10.1038/nature13387" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature13387" aria-label="Article reference 11" data-doi="10.1038/nature13387">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVSitrjM" aria-label="CAS reference 11">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25043048" aria-label="PubMed reference 11">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20vaccine%20targeting%20mutant%20IDH1%20induces%20antitumour%20immunity&amp;journal=Nature&amp;doi=10.1038%2Fnature13387&amp;volume=512&amp;pages=324-327&amp;publication_year=2014&amp;author=Schumacher%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Tran, E. et al. Immunogenicity of somatic mutations in human gastrointestinal cancers. <i>Science</i> <b>350</b>, 1387–1390 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aad1253" data-track-item_id="10.1126/science.aad1253" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aad1253" aria-label="Article reference 12" data-doi="10.1126/science.aad1253">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFKru7%2FL" aria-label="CAS reference 12">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26516200" aria-label="PubMed reference 12">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7445892" aria-label="PubMed Central reference 12">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Immunogenicity%20of%20somatic%20mutations%20in%20human%20gastrointestinal%20cancers&amp;journal=Science&amp;doi=10.1126%2Fscience.aad1253&amp;volume=350&amp;pages=1387-1390&amp;publication_year=2015&amp;author=Tran%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Chheda, Z. S. et al. Novel and shared neoantigen derived from histone 3 variant H3.3K27M mutation for glioma T cell therapy. <i>J. Exp. Med.</i> <b>215</b>, 141–157 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1084/jem.20171046" data-track-item_id="10.1084/jem.20171046" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1084%2Fjem.20171046" aria-label="Article reference 13" data-doi="10.1084/jem.20171046">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2rt73N" aria-label="CAS reference 13">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29203539" aria-label="PubMed reference 13">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5748856" aria-label="PubMed Central reference 13">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Novel%20and%20shared%20neoantigen%20derived%20from%20histone%203%20variant%20H3.3K27M%20mutation%20for%20glioma%20T%20cell%20therapy&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20171046&amp;volume=215&amp;pages=141-157&amp;publication_year=2018&amp;author=Chheda%2CZS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Veatch, J. R. et al. Tumor-infiltrating BRAFV600E-specific CD4<sup>+</sup> T cells correlated with complete clinical response in melanoma. <i>J. Clin. Invest.</i> <b>128</b>, 1563–1568 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1172/JCI98689" data-track-item_id="10.1172/JCI98689" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1172%2FJCI98689" aria-label="Article reference 14" data-doi="10.1172/JCI98689">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29360643" aria-label="PubMed reference 14">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5873881" aria-label="PubMed Central reference 14">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Tumor-infiltrating%20BRAFV600E-specific%20CD4%2B%20T%20cells%20correlated%20with%20complete%20clinical%20response%20in%20melanoma&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI98689&amp;volume=128&amp;pages=1563-1568&amp;publication_year=2018&amp;author=Veatch%2CJR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Malekzadeh, P. et al. Neoantigen screening identifies broad <i>TP53</i> mutant immunogenicity in patients with epithelial cancers. <i>J. Clin. Invest.</i> <b>129</b>, 1109–1114 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1172/JCI123791" data-track-item_id="10.1172/JCI123791" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1172%2FJCI123791" aria-label="Article reference 15" data-doi="10.1172/JCI123791">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30714987" aria-label="PubMed reference 15">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Neoantigen%20screening%20identifies%20broad%20TP53%20mutant%20immunogenicity%20in%20patients%20with%20epithelial%20cancers&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI123791&amp;volume=129&amp;pages=1109-1114&amp;publication_year=2019&amp;author=Malekzadeh%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Hsiue, E. H. et al. Targeting a neoantigen derived from a common <i>TP53</i> mutation. <i>Science</i> <b>371</b>, eabc8697 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.abc8697" data-track-item_id="10.1126/science.abc8697" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.abc8697" aria-label="Article reference 16" data-doi="10.1126/science.abc8697">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXlvFylurs%3D" aria-label="CAS reference 16">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33649166" aria-label="PubMed reference 16">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8208645" aria-label="PubMed Central reference 16">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Targeting%20a%20neoantigen%20derived%20from%20a%20common%20TP53%20mutation&amp;journal=Science&amp;doi=10.1126%2Fscience.abc8697&amp;volume=371&amp;publication_year=2021&amp;author=Hsiue%2CEH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Samuels, Y. et al. High frequency of mutations of the <i>PIK3CA</i> gene in human cancers. <i>Science</i> <b>304</b>, 554 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.1096502" data-track-item_id="10.1126/science.1096502" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1096502" aria-label="Article reference 17" data-doi="10.1126/science.1096502">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXjsVGmsbk%3D" aria-label="CAS reference 17">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15016963" aria-label="PubMed reference 17">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=High%20frequency%20of%20mutations%20of%20the%20PIK3CA%20gene%20in%20human%20cancers&amp;journal=Science&amp;doi=10.1126%2Fscience.1096502&amp;volume=304&amp;publication_year=2004&amp;author=Samuels%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. <i>Cell</i> <b>173</b>, 371–385 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2018.02.060" data-track-item_id="10.1016/j.cell.2018.02.060" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2018.02.060" aria-label="Article reference 18" data-doi="10.1016/j.cell.2018.02.060">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXntFaiu7c%3D" aria-label="CAS reference 18">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29625053" aria-label="PubMed reference 18">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6029450" aria-label="PubMed Central reference 18">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20characterization%20of%20cancer%20driver%20genes%20and%20mutations&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.02.060&amp;volume=173&amp;pages=371-385&amp;publication_year=2018&amp;author=Bailey%2CMH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Priestley, P. et al. Pan-cancer whole-genome analyses of metastatic solid tumours. <i>Nature</i> <b>575</b>, 210–216 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-019-1689-y" data-track-item_id="10.1038/s41586-019-1689-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-019-1689-y" aria-label="Article reference 19" data-doi="10.1038/s41586-019-1689-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVagsLjK" aria-label="CAS reference 19">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31645765" aria-label="PubMed reference 19">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6872491" aria-label="PubMed Central reference 19">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Pan-cancer%20whole-genome%20analyses%20of%20metastatic%20solid%20tumours&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1689-y&amp;volume=575&amp;pages=210-216&amp;publication_year=2019&amp;author=Priestley%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Andre, F. et al. Alpelisib for <i>PIK3CA</i>-mutated, hormone receptor-positive advanced breast cancer. <i>N. Engl. J. Med.</i> <b>380</b>, 1929–1940 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa1813904" data-track-item_id="10.1056/NEJMoa1813904" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa1813904" aria-label="Article reference 20" data-doi="10.1056/NEJMoa1813904">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVWis7nP" aria-label="CAS reference 20">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31091374" aria-label="PubMed reference 20">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Alpelisib%20for%20PIK3CA-mutated%2C%20hormone%20receptor-positive%20advanced%20breast%20cancer&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1813904&amp;volume=380&amp;pages=1929-1940&amp;publication_year=2019&amp;author=Andre%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Tate, J. G. et al. COSMIC: the Catalogue of Somatic Mutations in Cancer. <i>Nucleic Acids Res.</i> <b>47</b>, D941–D947 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gky1015" data-track-item_id="10.1093/nar/gky1015" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgky1015" aria-label="Article reference 21" data-doi="10.1093/nar/gky1015">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Cgt7bM" aria-label="CAS reference 21">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30371878" aria-label="PubMed reference 21">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=COSMIC%3A%20the%20Catalogue%20of%20Somatic%20Mutations%20in%20Cancer&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky1015&amp;volume=47&amp;pages=D941-D947&amp;publication_year=2019&amp;author=Tate%2CJG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Gonzalez-Galarza, F. F. et al. Allele frequency net database (AFND) 2020 update: gold-standard data classification, open access genotype data and new query tools. <i>Nucleic Acids Res.</i> <b>48</b>, D783–D788 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Glt7%2FI" aria-label="CAS reference 22">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31722398" aria-label="PubMed reference 22">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Allele%20frequency%20net%20database%20%28AFND%29%202020%20update%3A%20gold-standard%20data%20classification%2C%20open%20access%20genotype%20data%20and%20new%20query%20tools&amp;journal=Nucleic%20Acids%20Res.&amp;volume=48&amp;pages=D783-D788&amp;publication_year=2020&amp;author=Gonzalez-Galarza%2CFF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Sidney, J., Peters, B., Frahm, N., Brander, C. &amp; Sette, A. HLA class I supertypes: a revised and updated classification. <i>BMC Immunol.</i> <b>9</b>, 1 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/1471-2172-9-1" data-track-item_id="10.1186/1471-2172-9-1" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/1471-2172-9-1" aria-label="Article reference 23" data-doi="10.1186/1471-2172-9-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18211710" aria-label="PubMed reference 23">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2245908" aria-label="PubMed Central reference 23">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtFWrsbw%3D" aria-label="CAS reference 23">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=HLA%20class%20I%20supertypes%3A%20a%20revised%20and%20updated%20classification&amp;journal=BMC%20Immunol.&amp;doi=10.1186%2F1471-2172-9-1&amp;volume=9&amp;publication_year=2008&amp;author=Sidney%2CJ&amp;author=Peters%2CB&amp;author=Frahm%2CN&amp;author=Brander%2CC&amp;author=Sette%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Laugel, B. et al. Different T cell receptor affinity thresholds and CD8 coreceptor dependence govern cytotoxic T lymphocyte activation and tetramer binding properties. <i>J. Biol. Chem.</i> <b>282</b>, 23799–23810 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1074/jbc.M700976200" data-track-item_id="10.1074/jbc.M700976200" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1074%2Fjbc.M700976200" aria-label="Article reference 24" data-doi="10.1074/jbc.M700976200">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXoslOntrw%3D" aria-label="CAS reference 24">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17540778" aria-label="PubMed reference 24">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Different%20T%20cell%20receptor%20affinity%20thresholds%20and%20CD8%20coreceptor%20dependence%20govern%20cytotoxic%20T%20lymphocyte%20activation%20and%20tetramer%20binding%20properties&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M700976200&amp;volume=282&amp;pages=23799-23810&amp;publication_year=2007&amp;author=Laugel%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Johnson, L. A. et al. Gene therapy with human and mouse T-cell receptors mediates cancer regression and targets normal tissues expressing cognate antigen. <i>Blood</i> <b>114</b>, 535–546 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1182/blood-2009-03-211714" data-track-item_id="10.1182/blood-2009-03-211714" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1182%2Fblood-2009-03-211714" aria-label="Article reference 25" data-doi="10.1182/blood-2009-03-211714">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXpt1Wmsb0%3D" aria-label="CAS reference 25">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19451549" aria-label="PubMed reference 25">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929689" aria-label="PubMed Central reference 25">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20therapy%20with%20human%20and%20mouse%20T-cell%20receptors%20mediates%20cancer%20regression%20and%20targets%20normal%20tissues%20expressing%20cognate%20antigen&amp;journal=Blood&amp;doi=10.1182%2Fblood-2009-03-211714&amp;volume=114&amp;pages=535-546&amp;publication_year=2009&amp;author=Johnson%2CLA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Lu, Y. C. et al. Treatment of patients with metastatic cancer using a major histocompatibility complex class II-restricted T-cell receptor targeting the cancer germline antigen MAGE-A3. <i>J. Clin. Oncol.</i> <b>35</b>, 3322–3329 (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">D’Angelo, S. P. et al. Antitumor activity associated with prolonged persistence of adoptively transferred NY-ESO-1 (c259)T cells in synovial sarcoma. <i>Cancer Discov.</i> <b>8</b>, 944–957 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1158/2159-8290.CD-17-1417" data-track-item_id="10.1158/2159-8290.CD-17-1417" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1158%2F2159-8290.CD-17-1417" aria-label="Article reference 27" data-doi="10.1158/2159-8290.CD-17-1417">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29891538" aria-label="PubMed reference 27">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8092079" aria-label="PubMed Central reference 27">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVarsbfK" aria-label="CAS reference 27">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Antitumor%20activity%20associated%20with%20prolonged%20persistence%20of%20adoptively%20transferred%20NY-ESO-1%20%28c259%29T%20cells%20in%20synovial%20sarcoma&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-17-1417&amp;volume=8&amp;pages=944-957&amp;publication_year=2018&amp;author=D%E2%80%99Angelo%2CSP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Nagarsheth, N. B. et al. TCR-engineered T cells targeting E7 for patients with metastatic HPV-associated epithelial cancers. <i>Nat. Med</i>. <b>27</b>, 419–425 (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Reynisson, B., Alvarez, B., Paul, S., Peters, B. &amp; Nielsen, M. NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. <i>Nucleic Acids Res.</i> <b>48</b>, W449–W454 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Sarkizova, S. et al. A large peptidome dataset improves HLA class I epitope prediction across most of the human population. <i>Nat. Biotechnol.</i> <b>38</b>, 199–209 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41587-019-0322-9" data-track-item_id="10.1038/s41587-019-0322-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-019-0322-9" aria-label="Article reference 30" data-doi="10.1038/s41587-019-0322-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSktbvP" aria-label="CAS reference 30">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31844290" aria-label="PubMed reference 30">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20large%20peptidome%20dataset%20improves%20HLA%20class%20I%20epitope%20prediction%20across%20most%20of%20the%20human%20population&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-019-0322-9&amp;volume=38&amp;pages=199-209&amp;publication_year=2020&amp;author=Sarkizova%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Garcia, K. C. &amp; Adams, E. J. How the T cell receptor sees antigen—a structural view. <i>Cell</i> <b>122</b>, 333–336 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2005.07.015" data-track-item_id="10.1016/j.cell.2005.07.015" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2005.07.015" aria-label="Article reference 31" data-doi="10.1016/j.cell.2005.07.015">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXovV2qsro%3D" aria-label="CAS reference 31">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16096054" aria-label="PubMed reference 31">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20the%20T%20cell%20receptor%20sees%20antigen%E2%80%94a%20structural%20view&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2005.07.015&amp;volume=122&amp;pages=333-336&amp;publication_year=2005&amp;author=Garcia%2CKC&amp;author=Adams%2CEJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Gowthaman, R. &amp; Pierce, B. G. TCR3d: the T cell receptor structural repertoire database. <i>Bioinformatics</i> <b>35</b>, 5323–5325 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btz517" data-track-item_id="10.1093/bioinformatics/btz517" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtz517" aria-label="Article reference 32" data-doi="10.1093/bioinformatics/btz517">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtl2ksr%2FP" aria-label="CAS reference 32">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31240309" aria-label="PubMed reference 32">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6954642" aria-label="PubMed Central reference 32">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=TCR3d%3A%20the%20T%20cell%20receptor%20structural%20repertoire%20database&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtz517&amp;volume=35&amp;pages=5323-5325&amp;publication_year=2019&amp;author=Gowthaman%2CR&amp;author=Pierce%2CBG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Bagaev, D. V. et al. VDJdb in 2019: database extension, new analysis infrastructure and a T-cell receptor motif compendium. <i>Nucleic Acids Res.</i> <b>48</b>, D1057–D1062 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkz874" data-track-item_id="10.1093/nar/gkz874" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkz874" aria-label="Article reference 33" data-doi="10.1093/nar/gkz874">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31588507" aria-label="PubMed reference 33">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=VDJdb%20in%202019%3A%20database%20extension%2C%20new%20analysis%20infrastructure%20and%20a%20T-cell%20receptor%20motif%20compendium&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkz874&amp;volume=48&amp;pages=D1057-D1062&amp;publication_year=2020&amp;author=Bagaev%2CDV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Aleksic, M. et al. Different affinity windows for virus and cancer-specific T-cell receptors: implications for therapeutic strategies. <i>Eur. J. Immunol.</i> <b>42</b>, 3174–3179 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/eji.201242606" data-track-item_id="10.1002/eji.201242606" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2Feji.201242606" aria-label="Article reference 34" data-doi="10.1002/eji.201242606">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFWiu73E" aria-label="CAS reference 34">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22949370" aria-label="PubMed reference 34">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3776049" aria-label="PubMed Central reference 34">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Different%20affinity%20windows%20for%20virus%20and%20cancer-specific%20T-cell%20receptors%3A%20implications%20for%20therapeutic%20strategies&amp;journal=Eur.%20J.%20Immunol.&amp;doi=10.1002%2Feji.201242606&amp;volume=42&amp;pages=3174-3179&amp;publication_year=2012&amp;author=Aleksic%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Wu, D., Gallagher, D. T., Gowthaman, R., Pierce, B. G. &amp; Mariuzza, R. A. Structural basis for oligoclonal T cell recognition of a shared p53 cancer neoantigen. <i>Nat. Commun.</i> <b>11</b>, 2908 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-16755-y" data-track-item_id="10.1038/s41467-020-16755-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-16755-y" aria-label="Article reference 35" data-doi="10.1038/s41467-020-16755-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFGhsrbM" aria-label="CAS reference 35">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32518267" aria-label="PubMed reference 35">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7283474" aria-label="PubMed Central reference 35">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20basis%20for%20oligoclonal%20T%20cell%20recognition%20of%20a%20shared%20p53%20cancer%20neoantigen&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-16755-y&amp;volume=11&amp;publication_year=2020&amp;author=Wu%2CD&amp;author=Gallagher%2CDT&amp;author=Gowthaman%2CR&amp;author=Pierce%2CBG&amp;author=Mariuzza%2CRA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Sim, M. J. W. et al. High-affinity oligoclonal TCRs define effective adoptive T cell therapy targeting mutant KRAS-G12D. <i>Proc. Natl Acad. Sci. USA</i> <b>117</b>, 12826–12835 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.1921964117" data-track-item_id="10.1073/pnas.1921964117" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.1921964117" aria-label="Article reference 36" data-doi="10.1073/pnas.1921964117">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFWksL%2FF" aria-label="CAS reference 36">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32461371" aria-label="PubMed reference 36">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7293613" aria-label="PubMed Central reference 36">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=High-affinity%20oligoclonal%20TCRs%20define%20effective%20adoptive%20T%20cell%20therapy%20targeting%20mutant%20KRAS-G12D&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1921964117&amp;volume=117&amp;pages=12826-12835&amp;publication_year=2020&amp;author=Sim%2CMJW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Devlin, J. R. et al. Structural dissimilarity from self drives neoepitope escape from immune tolerance. <i>Nat. Chem. Biol.</i> <b>16</b>, 1269–1276 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41589-020-0610-1" data-track-item_id="10.1038/s41589-020-0610-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41589-020-0610-1" aria-label="Article reference 37" data-doi="10.1038/s41589-020-0610-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Oit77N" aria-label="CAS reference 37">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32807968" aria-label="PubMed reference 37">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8210748" aria-label="PubMed Central reference 37">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20dissimilarity%20from%20self%20drives%20neoepitope%20escape%20from%20immune%20tolerance&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fs41589-020-0610-1&amp;volume=16&amp;pages=1269-1276&amp;publication_year=2020&amp;author=Devlin%2CJR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Singh, N. K. et al. Geometrical characterization of T cell receptor binding modes reveals class-specific binding to maximize access to antigen. <i>Proteins</i> <b>88</b>, 503–513 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/prot.25829" data-track-item_id="10.1002/prot.25829" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2Fprot.25829" aria-label="Article reference 38" data-doi="10.1002/prot.25829">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFymt7nN" aria-label="CAS reference 38">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31589793" aria-label="PubMed reference 38">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Geometrical%20characterization%20of%20T%20cell%20receptor%20binding%20modes%20reveals%20class-specific%20binding%20to%20maximize%20access%20to%20antigen&amp;journal=Proteins&amp;doi=10.1002%2Fprot.25829&amp;volume=88&amp;pages=503-513&amp;publication_year=2020&amp;author=Singh%2CNK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">de Castro, E. et al. ScanProsite: detection of PROSITE signature matches and ProRule-associated functional and structural residues in proteins. <i>Nucleic Acids Res.</i> <b>34</b>, W362–W365 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkl124" data-track-item_id="10.1093/nar/gkl124" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkl124" aria-label="Article reference 39" data-doi="10.1093/nar/gkl124">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16845026" aria-label="PubMed reference 39">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1538847" aria-label="PubMed Central reference 39">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28Xps1yit7Y%3D" aria-label="CAS reference 39">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=ScanProsite%3A%20detection%20of%20PROSITE%20signature%20matches%20and%20ProRule-associated%20functional%20and%20structural%20residues%20in%20proteins&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkl124&amp;volume=34&amp;pages=W362-W365&amp;publication_year=2006&amp;author=Castro%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Gattinoni, L. et al. Removal of homeostatic cytokine sinks by lymphodepletion enhances the efficacy of adoptively transferred tumor-specific CD8<sup>+</sup> T cells. <i>J. Exp. Med.</i> <b>202</b>, 907–912 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1084/jem.20050732" data-track-item_id="10.1084/jem.20050732" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1084%2Fjem.20050732" aria-label="Article reference 40" data-doi="10.1084/jem.20050732">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtVygt73L" aria-label="CAS reference 40">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16203864" aria-label="PubMed reference 40">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1397916" aria-label="PubMed Central reference 40">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Removal%20of%20homeostatic%20cytokine%20sinks%20by%20lymphodepletion%20enhances%20the%20efficacy%20of%20adoptively%20transferred%20tumor-specific%20CD8%2B%20T%20cells&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20050732&amp;volume=202&amp;pages=907-912&amp;publication_year=2005&amp;author=Gattinoni%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Samuels, Y. et al. Mutant PIK3CA promotes cell growth and invasion of human cancer cells. <i>Cancer Cell</i> <b>7</b>, 561–573 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ccr.2005.05.014" data-track-item_id="10.1016/j.ccr.2005.05.014" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ccr.2005.05.014" aria-label="Article reference 41" data-doi="10.1016/j.ccr.2005.05.014">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXlslCgsrY%3D" aria-label="CAS reference 41">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15950905" aria-label="PubMed reference 41">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Mutant%20PIK3CA%20promotes%20cell%20growth%20and%20invasion%20of%20human%20cancer%20cells&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccr.2005.05.014&amp;volume=7&amp;pages=561-573&amp;publication_year=2005&amp;author=Samuels%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Kang, S., Bader, A. G. &amp; Vogt, P. K. Phosphatidylinositol 3-kinase mutations identified in human cancer are oncogenic. <i>Proc. Natl Acad. Sci. USA</i> <b>102</b>, 802–807 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.0408864102" data-track-item_id="10.1073/pnas.0408864102" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0408864102" aria-label="Article reference 42" data-doi="10.1073/pnas.0408864102">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXis1Cqsb0%3D" aria-label="CAS reference 42">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15647370" aria-label="PubMed reference 42">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC545580" aria-label="PubMed Central reference 42">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Phosphatidylinositol%203-kinase%20mutations%20identified%20in%20human%20cancer%20are%20oncogenic&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0408864102&amp;volume=102&amp;pages=802-807&amp;publication_year=2005&amp;author=Kang%2CS&amp;author=Bader%2CAG&amp;author=Vogt%2CPK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Cheng, D. T. et al. Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (MSK-IMPACT): a hybridization capture-based next-generation sequencing clinical assay for solid tumor molecular oncology. <i>J. Mol. Diagn.</i> <b>17</b>, 251–264 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.jmoldx.2014.12.006" data-track-item_id="10.1016/j.jmoldx.2014.12.006" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jmoldx.2014.12.006" aria-label="Article reference 43" data-doi="10.1016/j.jmoldx.2014.12.006">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXlt1aitLc%3D" aria-label="CAS reference 43">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25801821" aria-label="PubMed reference 43">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5808190" aria-label="PubMed Central reference 43">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Memorial%20Sloan%20Kettering-Integrated%20Mutation%20Profiling%20of%20Actionable%20Cancer%20Targets%20%28MSK-IMPACT%29%3A%20a%20hybridization%20capture-based%20next-generation%20sequencing%20clinical%20assay%20for%20solid%20tumor%20molecular%20oncology&amp;journal=J.%20Mol.%20Diagn.&amp;doi=10.1016%2Fj.jmoldx.2014.12.006&amp;volume=17&amp;pages=251-264&amp;publication_year=2015&amp;author=Cheng%2CDT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Rosenbaum, E. et al. HLA genotyping in synovial sarcoma: identifying HLA-A*02 and its association with clinical outcome. <i>Clin. Cancer Res.</i> <b>26</b>, 5448–5455 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1158/1078-0432.CCR-20-0832" data-track-item_id="10.1158/1078-0432.CCR-20-0832" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1158%2F1078-0432.CCR-20-0832" aria-label="Article reference 44" data-doi="10.1158/1078-0432.CCR-20-0832">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFCku7vM" aria-label="CAS reference 44">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32816945" aria-label="PubMed reference 44">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7572879" aria-label="PubMed Central reference 44">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=HLA%20genotyping%20in%20synovial%20sarcoma%3A%20identifying%20HLA-A%2A02%20and%20its%20association%20with%20clinical%20outcome&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-20-0832&amp;volume=26&amp;pages=5448-5455&amp;publication_year=2020&amp;author=Rosenbaum%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Eubank, M. H. et al. Automated eligibility screening and monitoring for genotype-driven precision oncology trials. <i>J. Am. Med. Inform. Assoc.</i> <b>23</b>, 777–781 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/jamia/ocw020" data-track-item_id="10.1093/jamia/ocw020" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fjamia%2Focw020" aria-label="Article reference 45" data-doi="10.1093/jamia/ocw020">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27016727" aria-label="PubMed reference 45">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6370254" aria-label="PubMed Central reference 45">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20eligibility%20screening%20and%20monitoring%20for%20genotype-driven%20precision%20oncology%20trials&amp;journal=J.%20Am.%20Med.%20Inform.%20Assoc.&amp;doi=10.1093%2Fjamia%2Focw020&amp;volume=23&amp;pages=777-781&amp;publication_year=2016&amp;author=Eubank%2CMH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Rosenthal, R. et al. Neoantigen-directed immune escape in lung cancer evolution. <i>Nature</i> <b>567</b>, 479–485 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-019-1032-7" data-track-item_id="10.1038/s41586-019-1032-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-019-1032-7" aria-label="Article reference 46" data-doi="10.1038/s41586-019-1032-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXmslKntrY%3D" aria-label="CAS reference 46">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30894752" aria-label="PubMed reference 46">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6954100" aria-label="PubMed Central reference 46">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Neoantigen-directed%20immune%20escape%20in%20lung%20cancer%20evolution&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1032-7&amp;volume=567&amp;pages=479-485&amp;publication_year=2019&amp;author=Rosenthal%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Chandran, S. S. &amp; Klebanoff, C. A. T cell receptor-based cancer immunotherapy: emerging efficacy and pathways of resistance. <i>Immunol. Rev.</i> <b>290</b>, 127–147 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/imr.12772" data-track-item_id="10.1111/imr.12772" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fimr.12772" aria-label="Article reference 47" data-doi="10.1111/imr.12772">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVChsb%2FO" aria-label="CAS reference 47">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31355495" aria-label="PubMed reference 47">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7027847" aria-label="PubMed Central reference 47">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=T%20cell%20receptor-based%20cancer%20immunotherapy%3A%20emerging%20efficacy%20and%20pathways%20of%20resistance&amp;journal=Immunol.%20Rev.&amp;doi=10.1111%2Fimr.12772&amp;volume=290&amp;pages=127-147&amp;publication_year=2019&amp;author=Chandran%2CSS&amp;author=Klebanoff%2CCA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">McGranahan, N. et al. Allele-specific HLA loss and immune escape in lung cancer evolution. <i>Cell</i> <b>171</b>, 1259–1271 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2017.10.001" data-track-item_id="10.1016/j.cell.2017.10.001" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.10.001" aria-label="Article reference 48" data-doi="10.1016/j.cell.2017.10.001">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslehs7nL" aria-label="CAS reference 48">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29107330" aria-label="PubMed reference 48">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5720478" aria-label="PubMed Central reference 48">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Allele-specific%20HLA%20loss%20and%20immune%20escape%20in%20lung%20cancer%20evolution&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.10.001&amp;volume=171&amp;pages=1259-1271&amp;publication_year=2017&amp;author=McGranahan%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Montesion, M. et al. Somatic HLA class I loss is a widespread mechanism of immune evasion which refines the use of tumor mutational burden as a biomarker of checkpoint inhibitor response. <i>Cancer Discov.</i> <b>11</b>, 282–292 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1158/2159-8290.CD-20-0672" data-track-item_id="10.1158/2159-8290.CD-20-0672" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1158%2F2159-8290.CD-20-0672" aria-label="Article reference 49" data-doi="10.1158/2159-8290.CD-20-0672">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXosV2hsrY%3D" aria-label="CAS reference 49">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33127846" aria-label="PubMed reference 49">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Somatic%20HLA%20class%20I%20loss%20is%20a%20widespread%20mechanism%20of%20immune%20evasion%20which%20refines%20the%20use%20of%20tumor%20mutational%20burden%20as%20a%20biomarker%20of%20checkpoint%20inhibitor%20response&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-20-0672&amp;volume=11&amp;pages=282-292&amp;publication_year=2021&amp;author=Montesion%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Zaretsky, J. M. et al. Mutations associated with acquired resistance to PD-1 blockade in melanoma. <i>N. Engl. J. Med.</i> <b>375</b>, 819–829 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa1604958" data-track-item_id="10.1056/NEJMoa1604958" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa1604958" aria-label="Article reference 50" data-doi="10.1056/NEJMoa1604958">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhvF2ks7%2FP" aria-label="CAS reference 50">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27433843" aria-label="PubMed reference 50">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5007206" aria-label="PubMed Central reference 50">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Mutations%20associated%20with%20acquired%20resistance%20to%20PD-1%20blockade%20in%20melanoma&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1604958&amp;volume=375&amp;pages=819-829&amp;publication_year=2016&amp;author=Zaretsky%2CJM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Shin, D. S. et al. Primary resistance to PD-1 blockade mediated by <i>JAK1/2</i> mutations. <i>Cancer Discov.</i> <b>7</b>, 188–201 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1158/2159-8290.CD-16-1223" data-track-item_id="10.1158/2159-8290.CD-16-1223" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1158%2F2159-8290.CD-16-1223" aria-label="Article reference 51" data-doi="10.1158/2159-8290.CD-16-1223">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVCjsr0%3D" aria-label="CAS reference 51">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27903500" aria-label="PubMed reference 51">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Primary%20resistance%20to%20PD-1%20blockade%20mediated%20by%20JAK1%2F2%20mutations&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-16-1223&amp;volume=7&amp;pages=188-201&amp;publication_year=2017&amp;author=Shin%2CDS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Wolfl, M. &amp; Greenberg, P. D. Antigen-specific activation and cytokine-facilitated expansion of naive, human CD8<sup>+</sup> T cells. <i>Nat. Protoc.</i> <b>9</b>, 950–966 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nprot.2014.064" data-track-item_id="10.1038/nprot.2014.064" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnprot.2014.064" aria-label="Article reference 52" data-doi="10.1038/nprot.2014.064">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24675735" aria-label="PubMed reference 52">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4312138" aria-label="PubMed Central reference 52">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXos1Oisr4%3D" aria-label="CAS reference 52">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Antigen-specific%20activation%20and%20cytokine-facilitated%20expansion%20of%20naive%2C%20human%20CD8%2B%20T%20cells&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fnprot.2014.064&amp;volume=9&amp;pages=950-966&amp;publication_year=2014&amp;author=Wolfl%2CM&amp;author=Greenberg%2CPD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Gee, M. H. et al. Antigen identification for orphan T cell receptors expressed on tumor-infiltrating lymphocytes. <i>Cell</i> <b>172</b>, 549–563 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2017.11.043" data-track-item_id="10.1016/j.cell.2017.11.043" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.11.043" aria-label="Article reference 53" data-doi="10.1016/j.cell.2017.11.043">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVeht7bI" aria-label="CAS reference 53">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29275860" aria-label="PubMed reference 53">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Antigen%20identification%20for%20orphan%20T%20cell%20receptors%20expressed%20on%20tumor-infiltrating%20lymphocytes&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.11.043&amp;volume=172&amp;pages=549-563&amp;publication_year=2018&amp;author=Gee%2CMH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Ali, M. et al. Induction of neoantigen-reactive T cells from healthy donors. <i>Nat. Protoc.</i> <b>14</b>, 1926–1943 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41596-019-0170-6" data-track-item_id="10.1038/s41596-019-0170-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41596-019-0170-6" aria-label="Article reference 54" data-doi="10.1038/s41596-019-0170-6">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVWqurrP" aria-label="CAS reference 54">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31101906" aria-label="PubMed reference 54">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Induction%20of%20neoantigen-reactive%20T%20cells%20from%20healthy%20donors&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fs41596-019-0170-6&amp;volume=14&amp;pages=1926-1943&amp;publication_year=2019&amp;author=Ali%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Peng, S. et al. Sensitive detection and analysis of neoantigen-specific T cell populations from tumors and blood. <i>Cell Rep.</i> <b>28</b>, 2728–2738 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.celrep.2019.07.106" data-track-item_id="10.1016/j.celrep.2019.07.106" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.celrep.2019.07.106" aria-label="Article reference 55" data-doi="10.1016/j.celrep.2019.07.106">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslaqsLjK" aria-label="CAS reference 55">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31484081" aria-label="PubMed reference 55">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6774618" aria-label="PubMed Central reference 55">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Sensitive%20detection%20and%20analysis%20of%20neoantigen-specific%20T%20cell%20populations%20from%20tumors%20and%20blood&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2019.07.106&amp;volume=28&amp;pages=2728-2738&amp;publication_year=2019&amp;author=Peng%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Arnaud, M. et al. Sensitive identification of neoantigens and cognate TCRs in human solid tumors. <i>Nat. Biotechnol.</i> <a href="https://doi.org/10.1038/s41587-021-01072-6" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41587-021-01072-6">https://doi.org/10.1038/s41587-021-01072-6</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Schmidt, J. et al. Prediction of neo-epitope immunogenicity reveals TCR recognition determinants and provides insight into immunoediting. <i>Cell Rep. Med.</i> <b>2</b>, 100194 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.xcrm.2021.100194" data-track-item_id="10.1016/j.xcrm.2021.100194" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.xcrm.2021.100194" aria-label="Article reference 57" data-doi="10.1016/j.xcrm.2021.100194">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33665637" aria-label="PubMed reference 57">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7897774" aria-label="PubMed Central reference 57">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20neo-epitope%20immunogenicity%20reveals%20TCR%20recognition%20determinants%20and%20provides%20insight%20into%20immunoediting&amp;journal=Cell%20Rep.%20Med.&amp;doi=10.1016%2Fj.xcrm.2021.100194&amp;volume=2&amp;publication_year=2021&amp;author=Schmidt%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Paulson, K. G. et al. Acquired cancer resistance to combination immunotherapy from transcriptional loss of class I HLA. <i>Nat. Commun.</i> <b>9</b>, 3868 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-018-06300-3" data-track-item_id="10.1038/s41467-018-06300-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-018-06300-3" aria-label="Article reference 58" data-doi="10.1038/s41467-018-06300-3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BB3czit12nsQ%3D%3D" aria-label="CAS reference 58">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30250229" aria-label="PubMed reference 58">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6155241" aria-label="PubMed Central reference 58">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Acquired%20cancer%20resistance%20to%20combination%20immunotherapy%20from%20transcriptional%20loss%20of%20class%20I%20HLA&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-06300-3&amp;volume=9&amp;publication_year=2018&amp;author=Paulson%2CKG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Drilon, A. et al. Safety and antitumor activity of the multitargeted pan-TRK, ROS1, and ALK inhibitor entrectinib: combined results from two phase I trials (ALKA-372-001 and STARTRK-1). <i>Cancer Discov.</i> <b>7</b>, 400–409 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1158/2159-8290.CD-16-1237" data-track-item_id="10.1158/2159-8290.CD-16-1237" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1158%2F2159-8290.CD-16-1237" aria-label="Article reference 59" data-doi="10.1158/2159-8290.CD-16-1237">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXltlOqsL8%3D" aria-label="CAS reference 59">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28183697" aria-label="PubMed reference 59">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5380583" aria-label="PubMed Central reference 59">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Safety%20and%20antitumor%20activity%20of%20the%20multitargeted%20pan-TRK%2C%20ROS1%2C%20and%20ALK%20inhibitor%20entrectinib%3A%20combined%20results%20from%20two%20phase%20I%20trials%20%28ALKA-372-001%20and%20STARTRK-1%29&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-16-1237&amp;volume=7&amp;pages=400-409&amp;publication_year=2017&amp;author=Drilon%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Marabelle, A. et al. Efficacy of pembrolizumab in patients with noncolorectal high microsatellite instability/mismatch repair-deficient cancer: results from the phase II KEYNOTE-158 study. <i>J. Clin. Oncol.</i> <b>38</b>, 1–10 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1200/JCO.19.02105" data-track-item_id="10.1200/JCO.19.02105" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1200%2FJCO.19.02105" aria-label="Article reference 60" data-doi="10.1200/JCO.19.02105">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslSktb%2FL" aria-label="CAS reference 60">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31682550" aria-label="PubMed reference 60">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficacy%20of%20pembrolizumab%20in%20patients%20with%20noncolorectal%20high%20microsatellite%20instability%2Fmismatch%20repair-deficient%20cancer%3A%20results%20from%20the%20phase%20II%20KEYNOTE-158%20study&amp;journal=J.%20Clin.%20Oncol.&amp;doi=10.1200%2FJCO.19.02105&amp;volume=38&amp;pages=1-10&amp;publication_year=2020&amp;author=Marabelle%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Stevanovic, S. et al. Landscape of immunogenic tumor antigens in successful immunotherapy of virally induced epithelial cancer. <i>Science</i> <b>356</b>, 200–205 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aak9510" data-track-item_id="10.1126/science.aak9510" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aak9510" aria-label="Article reference 61" data-doi="10.1126/science.aak9510">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXlvVems7g%3D" aria-label="CAS reference 61">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28408606" aria-label="PubMed reference 61">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6295311" aria-label="PubMed Central reference 61">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Landscape%20of%20immunogenic%20tumor%20antigens%20in%20successful%20immunotherapy%20of%20virally%20induced%20epithelial%20cancer&amp;journal=Science&amp;doi=10.1126%2Fscience.aak9510&amp;volume=356&amp;pages=200-205&amp;publication_year=2017&amp;author=Stevanovic%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Cohen, C. J., Zhao, Y., Zheng, Z., Rosenberg, S. A. &amp; Morgan, R. A. Enhanced antitumor activity of murine-human hybrid T-cell receptor (TCR) in human lymphocytes is associated with improved pairing and TCR/CD3 stability. <i>Cancer Res.</i> <b>66</b>, 8878–8886 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1158/0008-5472.CAN-06-1450" data-track-item_id="10.1158/0008-5472.CAN-06-1450" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1158%2F0008-5472.CAN-06-1450" aria-label="Article reference 62" data-doi="10.1158/0008-5472.CAN-06-1450">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XovF2ht7c%3D" aria-label="CAS reference 62">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16951205" aria-label="PubMed reference 62">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2147082" aria-label="PubMed Central reference 62">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Enhanced%20antitumor%20activity%20of%20murine-human%20hybrid%20T-cell%20receptor%20%28TCR%29%20in%20human%20lymphocytes%20is%20associated%20with%20improved%20pairing%20and%20TCR%2FCD3%20stability&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-06-1450&amp;volume=66&amp;pages=8878-8886&amp;publication_year=2006&amp;author=Cohen%2CCJ&amp;author=Zhao%2CY&amp;author=Zheng%2CZ&amp;author=Rosenberg%2CSA&amp;author=Morgan%2CRA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Klatt, M. G. et al. Solving an MHC allele-specific bias in the reported immunopeptidome. <i>JCI Insight</i> <b>5</b>, e141264 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1172/jci.insight.141264" data-track-item_id="10.1172/jci.insight.141264" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1172%2Fjci.insight.141264" aria-label="Article reference 63" data-doi="10.1172/jci.insight.141264">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7566711" aria-label="PubMed Central reference 63">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Solving%20an%20MHC%20allele-specific%20bias%20in%20the%20reported%20immunopeptidome&amp;journal=JCI%20Insight&amp;doi=10.1172%2Fjci.insight.141264&amp;volume=5&amp;publication_year=2020&amp;author=Klatt%2CMG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Cameron, B. J. et al. Identification of a Titin-derived HLA-A1-presented peptide as a cross-reactive target for engineered MAGE A3-directed T cells. <i>Sci. Transl. Med.</i> <b>5</b>, 197ra103 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/scitranslmed.3006034" data-track-item_id="10.1126/scitranslmed.3006034" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscitranslmed.3006034" aria-label="Article reference 64" data-doi="10.1126/scitranslmed.3006034">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23926201" aria-label="PubMed reference 64">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6002776" aria-label="PubMed Central reference 64">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsV2ht7%2FK" aria-label="CAS reference 64">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20a%20Titin-derived%20HLA-A1-presented%20peptide%20as%20a%20cross-reactive%20target%20for%20engineered%20MAGE%20A3-directed%20T%20cells&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.3006034&amp;volume=5&amp;publication_year=2013&amp;author=Cameron%2CBJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Pizzolla, A. et al. Influenza-specific lung-resident memory T cells are proliferative and polyfunctional and maintain diverse TCR profiles. <i>J. Clin. Invest.</i> <b>128</b>, 721–733 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1172/JCI96957" data-track-item_id="10.1172/JCI96957" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1172%2FJCI96957" aria-label="Article reference 65" data-doi="10.1172/JCI96957">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29309047" aria-label="PubMed reference 65">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5785253" aria-label="PubMed Central reference 65">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Influenza-specific%20lung-resident%20memory%20T%20cells%20are%20proliferative%20and%20polyfunctional%20and%20maintain%20diverse%20TCR%20profiles&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI96957&amp;volume=128&amp;pages=721-733&amp;publication_year=2018&amp;author=Pizzolla%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Davis-Harrison, R. L., Armstrong, K. M. &amp; Baker, B. M. Two different T cell receptors use different thermodynamic strategies to recognize the same peptide/MHC ligand. <i>J. Mol. Biol.</i> <b>346</b>, 533–550 (2005).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.jmb.2004.11.063" data-track-item_id="10.1016/j.jmb.2004.11.063" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jmb.2004.11.063" aria-label="Article reference 66" data-doi="10.1016/j.jmb.2004.11.063">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2MXntVShtQ%3D%3D" aria-label="CAS reference 66">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15670602" aria-label="PubMed reference 66">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Two%20different%20T%20cell%20receptors%20use%20different%20thermodynamic%20strategies%20to%20recognize%20the%20same%20peptide%2FMHC%20ligand&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2Fj.jmb.2004.11.063&amp;volume=346&amp;pages=533-550&amp;publication_year=2005&amp;author=Davis-Harrison%2CRL&amp;author=Armstrong%2CKM&amp;author=Baker%2CBM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Singh, N. K. et al. An engineered T cell receptor variant realizes the limits of functional binding modes. <i>Biochemistry</i> <b>59</b>, 4163–4175 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1021/acs.biochem.0c00689" data-track-item_id="10.1021/acs.biochem.0c00689" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1021%2Facs.biochem.0c00689" aria-label="Article reference 67" data-doi="10.1021/acs.biochem.0c00689">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVykurbP" aria-label="CAS reference 67">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33074657" aria-label="PubMed reference 67">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20engineered%20T%20cell%20receptor%20variant%20realizes%20the%20limits%20of%20functional%20binding%20modes&amp;journal=Biochemistry&amp;doi=10.1021%2Facs.biochem.0c00689&amp;volume=59&amp;pages=4163-4175&amp;publication_year=2020&amp;author=Singh%2CNK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Hellman, L. M. et al. Differential scanning fluorimetry based assessments of the thermal and kinetic stability of peptide-MHC complexes. <i>J. Immunol. Methods</i> <b>432</b>, 95–101 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.jim.2016.02.016" data-track-item_id="10.1016/j.jim.2016.02.016" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jim.2016.02.016" aria-label="Article reference 68" data-doi="10.1016/j.jim.2016.02.016">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XjtFOls70%3D" aria-label="CAS reference 68">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26906089" aria-label="PubMed reference 68">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4837003" aria-label="PubMed Central reference 68">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20scanning%20fluorimetry%20based%20assessments%20of%20the%20thermal%20and%20kinetic%20stability%20of%20peptide-MHC%20complexes&amp;journal=J.%20Immunol.%20Methods&amp;doi=10.1016%2Fj.jim.2016.02.016&amp;volume=432&amp;pages=95-101&amp;publication_year=2016&amp;author=Hellman%2CLM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Baxter, T. K. et al. Strategic mutations in the class I major histocompatibility complex HLA-A2 independently affect both peptide binding and T cell receptor recognition. <i>J. Biol. Chem.</i> <b>279</b>, 29175–29184 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1074/jbc.M403372200" data-track-item_id="10.1074/jbc.M403372200" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1074%2Fjbc.M403372200" aria-label="Article reference 69" data-doi="10.1074/jbc.M403372200">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2cXlsVCmsrw%3D" aria-label="CAS reference 69">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15131131" aria-label="PubMed reference 69">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Strategic%20mutations%20in%20the%20class%20I%20major%20histocompatibility%20complex%20HLA-A2%20independently%20affect%20both%20peptide%20binding%20and%20T%20cell%20receptor%20recognition&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M403372200&amp;volume=279&amp;pages=29175-29184&amp;publication_year=2004&amp;author=Baxter%2CTK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">McMahon, R. M. et al. Structure of HLA-A*0301 in complex with a peptide of proteolipid protein: insights into the role of HLA-A alleles in susceptibility to multiple sclerosis. <i>Acta Crystallogr. D. Biol. Crystallogr.</i> <b>67</b>, 447–454 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1107/S0907444911007888" data-track-item_id="10.1107/S0907444911007888" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1107%2FS0907444911007888" aria-label="Article reference 70" data-doi="10.1107/S0907444911007888">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvFCrsL0%3D" aria-label="CAS reference 70">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21543847" aria-label="PubMed reference 70">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3087623" aria-label="PubMed Central reference 70">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20of%20HLA-A%2A0301%20in%20complex%20with%20a%20peptide%20of%20proteolipid%20protein%3A%20insights%20into%20the%20role%20of%20HLA-A%20alleles%20in%20susceptibility%20to%20multiple%20sclerosis&amp;journal=Acta%20Crystallogr.%20D.%20Biol.%20Crystallogr.&amp;doi=10.1107%2FS0907444911007888&amp;volume=67&amp;pages=447-454&amp;publication_year=2011&amp;author=McMahon%2CRM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Liu, Y. C. et al. A molecular basis for the interplay between T cells, viral mutants, and human leukocyte antigen micropolymorphism. <i>J. Biol. Chem.</i> <b>289</b>, 16688–16698 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1074/jbc.M114.563502" data-track-item_id="10.1074/jbc.M114.563502" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1074%2Fjbc.M114.563502" aria-label="Article reference 71" data-doi="10.1074/jbc.M114.563502">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXps1ymu7o%3D" aria-label="CAS reference 71">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24759101" aria-label="PubMed reference 71">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4059114" aria-label="PubMed Central reference 71">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20molecular%20basis%20for%20the%20interplay%20between%20T%20cells%2C%20viral%20mutants%2C%20and%20human%20leukocyte%20antigen%20micropolymorphism&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M114.563502&amp;volume=289&amp;pages=16688-16698&amp;publication_year=2014&amp;author=Liu%2CYC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Scott, D. R., Borbulevych, O. Y., Piepenbrink, K. H., Corcelli, S. A. &amp; Baker, B. M. Disparate degrees of hypervariable loop flexibility control T-cell receptor cross-reactivity, specificity, and binding mechanism. <i>J. Mol. Biol.</i> <b>414</b>, 385–400 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.jmb.2011.10.006" data-track-item_id="10.1016/j.jmb.2011.10.006" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jmb.2011.10.006" aria-label="Article reference 72" data-doi="10.1016/j.jmb.2011.10.006">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsFKltbfP" aria-label="CAS reference 72">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22019736" aria-label="PubMed reference 72">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3230710" aria-label="PubMed Central reference 72">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Disparate%20degrees%20of%20hypervariable%20loop%20flexibility%20control%20T-cell%20receptor%20cross-reactivity%2C%20specificity%2C%20and%20binding%20mechanism&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2Fj.jmb.2011.10.006&amp;volume=414&amp;pages=385-400&amp;publication_year=2011&amp;author=Scott%2CDR&amp;author=Borbulevych%2COY&amp;author=Piepenbrink%2CKH&amp;author=Corcelli%2CSA&amp;author=Baker%2CBM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">McGranahan, N. et al. Clonal status of actionable driver events and the timing of mutational processes in cancer evolution. <i>Sci. Transl. Med.</i> <b>7</b>, 283ra254 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/scitranslmed.aaa1408" data-track-item_id="10.1126/scitranslmed.aaa1408" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscitranslmed.aaa1408" aria-label="Article reference 73" data-doi="10.1126/scitranslmed.aaa1408">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20status%20of%20actionable%20driver%20events%20and%20the%20timing%20of%20mutational%20processes%20in%20cancer%20evolution&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.aaa1408&amp;volume=7&amp;publication_year=2015&amp;author=McGranahan%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Shen, R. &amp; Seshan, V. E. FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing. <i>Nucleic Acids Res.</i> <b>44</b>, e131 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkw520" data-track-item_id="10.1093/nar/gkw520" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkw520" aria-label="Article reference 74" data-doi="10.1093/nar/gkw520">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27270079" aria-label="PubMed reference 74">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5027494" aria-label="PubMed Central reference 74">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhvVKmsrvE" aria-label="CAS reference 74">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=FACETS%3A%20allele-specific%20copy%20number%20and%20clonal%20heterogeneity%20analysis%20tool%20for%20high-throughput%20DNA%20sequencing&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw520&amp;volume=44&amp;publication_year=2016&amp;author=Shen%2CR&amp;author=Seshan%2CVE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Bielski, C. M. et al. Widespread selection for oncogenic mutant allele imbalance in cancer. <i>Cancer Cell</i> <b>34</b>, 852–862 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ccell.2018.10.003" data-track-item_id="10.1016/j.ccell.2018.10.003" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ccell.2018.10.003" aria-label="Article reference 75" data-doi="10.1016/j.ccell.2018.10.003">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVOhsrbE" aria-label="CAS reference 75">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30393068" aria-label="PubMed reference 75">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6234065" aria-label="PubMed Central reference 75">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=Widespread%20selection%20for%20oncogenic%20mutant%20allele%20imbalance%20in%20cancer&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2018.10.003&amp;volume=34&amp;pages=852-862&amp;publication_year=2018&amp;author=Bielski%2CCM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Shukla, S. A. et al. Comprehensive analysis of cancer-associated somatic mutations in class I HLA genes. <i>Nat. Biotechnol.</i> <b>33</b>, 1152–1158 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nbt.3344" data-track-item_id="10.1038/nbt.3344" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.3344" aria-label="Article reference 76" data-doi="10.1038/nbt.3344">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFWht7nO" aria-label="CAS reference 76">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26372948" aria-label="PubMed reference 76">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4747795" aria-label="PubMed Central reference 76">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20analysis%20of%20cancer-associated%20somatic%20mutations%20in%20class%20I%20HLA%20genes&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3344&amp;volume=33&amp;pages=1152-1158&amp;publication_year=2015&amp;author=Shukla%2CSA">
                    Google Scholar</a>&nbsp;
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01786-3?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>This study was supported, in part, by National Institutes of Heath (NIH) grants R37CA259177 (C.A.K.), P30CA008748 (C.A.K. and D.A.S.) and R01AI129543 (B.M.B.); the Parker Institute for Cancer Immunotherapy (C.A.K.); the Damon Runyon Cancer Research Foundation CI-96-18 (C.A.K.); Cancer Research Institute CRI3176 (C.A.K); the Functional Genomics Initiative Rapid Response Pilot Grant (C.A.K.); the Manhasset Women’s Coalition Against Breast Cancer (C.A.K.); the Emerson Collective Cancer Fund (C.A.K.); the Breast Cancer Alliance Exceptional Project Grant (C.A.K.); the Sussman Family Fund (C.A.K.); and a sponsored research agreement with Intima Bioscience (C.A.K.). X-ray diffraction data were collected at the Advanced Photon Source, supported by Department of Energy contract DE-AC02-06CH11357, and the NE-CAT and SER-CAT beamlines were supported by member institutions and NIH grants P30GM124165, S10OD021527, S10RR25528 and S10RR028976. M.G.K. was supported by an individual research grant from the German Research Foundation (DFG, KL3118/1-1). L.B.B. was supported, in part, by the Conquer Cancer Foundation. E.V. was supported by Programa de Ayudas de Movilidad para la Obtención de Mención ‘Doctor Internacional’ of the University of Navarra, FUNDACIONCAJANAVARRA-Obra Social ‘La Caixa’ and the Company of Biologists Travelling Fellowships Grant. F.Y. was supported, in part, by a fellowship from the Hearst Foundation. I.X. was supported, in part, by a fellowship from the Fundación Alfonso Martín Escudero and the Parker Institute for Cancer Immunotherapy. We thank A. Kentsis for access to the Byonic software as well as the Proteomics Resource Center at Rockefeller University for the performance of all LC–MS/MS experiments. We thank the Last Wish Program (MSKCC) for tissue collection, the Antitumor Assessment Core (MSKCC) for PDX development and assistance with in vivo tumor models and the Epigenomics Core of Weill Cornell Medical College for enabling single-cell immune-profiling experiments. We thank M.R. Femia for technical assistance with pilot IVS screens and L. Albacker and M. Montesion for helpful discussions. The content of this manuscript is solely the responsibility of the authors and does not necessarily represent the official views of the NIH.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Human Oncology and Pathogenesis Program (HOPP), Immuno-Oncology Service, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Smita S. Chandran,&nbsp;Pedram Razavi,&nbsp;Si Ning Fu,&nbsp;Lauren B. Banks,&nbsp;Fei Yi,&nbsp;Enric Vercher,&nbsp;Inaki Etxeberria,&nbsp;Ilinca S. Aricescu&nbsp;&amp;&nbsp;Christopher A. Klebanoff</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Center for Cell Engineering, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Smita S. Chandran,&nbsp;Si Ning Fu,&nbsp;Lauren B. Banks,&nbsp;Fei Yi&nbsp;&amp;&nbsp;Christopher A. Klebanoff</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Parker Institute for Cancer Immunotherapy, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Smita S. Chandran&nbsp;&amp;&nbsp;Christopher A. Klebanoff</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Department of Chemistry and Biochemistry, University of Notre Dame, South Bend, IN, USA</p><p class="c-article-author-affiliation__authors-list">Jiaqi Ma&nbsp;&amp;&nbsp;Brian M. Baker</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Harper Cancer Research Institute, University of Notre Dame, South Bend, IN, USA</p><p class="c-article-author-affiliation__authors-list">Jiaqi Ma&nbsp;&amp;&nbsp;Brian M. Baker</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Molecular Pharmacology Program, Sloan Kettering Institute, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Martin G. Klatt&nbsp;&amp;&nbsp;David A. Scheinberg</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Friederike Dündar&nbsp;&amp;&nbsp;Paul Zumbo</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Applied Bioinformatics Core, Weill Cornell Medicine, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Friederike Dündar,&nbsp;Paul Zumbo&nbsp;&amp;&nbsp;Doron Betel</p></li><li id="Aff9"><p class="c-article-author-affiliation__address">Marie-Josée and Henry R. Kravis Center for Molecular Oncology, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Chaitanya Bandlamudi</p></li><li id="Aff10"><p class="c-article-author-affiliation__address">Department of Medicine, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Pedram Razavi,&nbsp;Lauren B. Banks,&nbsp;Watchain D. Bestman,&nbsp;Alexander Drilon,&nbsp;David A. Scheinberg&nbsp;&amp;&nbsp;Christopher A. Klebanoff</p></li><li id="Aff11"><p class="c-article-author-affiliation__address">Department of Medicine, Division of Hematology and Medical Oncology, Weill Cornell Medicine, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Pedram Razavi,&nbsp;Alexander Drilon,&nbsp;Doron Betel&nbsp;&amp;&nbsp;Christopher A. Klebanoff</p></li><li id="Aff12"><p class="c-article-author-affiliation__address">Department of Pathology, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Hannah Y. Wen&nbsp;&amp;&nbsp;Britta Weigelt</p></li><li id="Aff13"><p class="c-article-author-affiliation__address">Department of Surgery, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Watchain D. Bestman&nbsp;&amp;&nbsp;Arnaud Da Cruz Paula</p></li><li id="Aff14"><p class="c-article-author-affiliation__address">Early Drug Development Service, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Alexander Drilon&nbsp;&amp;&nbsp;Christopher A. Klebanoff</p></li><li id="Aff15"><p class="c-article-author-affiliation__address">Institute for Computational Biomedicine, Weill Cornell Medicine, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Doron Betel</p></li><li id="Aff16"><p class="c-article-author-affiliation__address">Cell Therapy Service, Memorial Sloan Kettering Cancer Center, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Christopher A. Klebanoff</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Smita_S_-Chandran-Aff1-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Smita S. Chandran</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Smita%20S.%20Chandran" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Smita%20S.%20Chandran" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Smita%20S.%20Chandran%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jiaqi-Ma-Aff4-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Jiaqi Ma</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jiaqi%20Ma" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jiaqi%20Ma" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jiaqi%20Ma%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Martin_G_-Klatt-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Martin G. Klatt</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Martin%20G.%20Klatt" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Martin%20G.%20Klatt" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Martin%20G.%20Klatt%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Friederike-D_ndar-Aff7-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Friederike Dündar</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Friederike%20D%C3%BCndar" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Friederike%20D%C3%BCndar" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Friederike%20D%C3%BCndar%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Chaitanya-Bandlamudi-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Chaitanya Bandlamudi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Chaitanya%20Bandlamudi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chaitanya%20Bandlamudi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chaitanya%20Bandlamudi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Pedram-Razavi-Aff1-Aff10-Aff11"><span class="c-article-authors-search__title u-h3 js-search-name">Pedram Razavi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Pedram%20Razavi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Pedram%20Razavi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Pedram%20Razavi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Hannah_Y_-Wen-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Hannah Y. Wen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Hannah%20Y.%20Wen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Hannah%20Y.%20Wen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Hannah%20Y.%20Wen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Britta-Weigelt-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Britta Weigelt</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Britta%20Weigelt" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Britta%20Weigelt" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Britta%20Weigelt%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Paul-Zumbo-Aff7-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Paul Zumbo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Paul%20Zumbo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Paul%20Zumbo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Paul%20Zumbo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Si_Ning-Fu-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Si Ning Fu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Si%20Ning%20Fu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Si%20Ning%20Fu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Si%20Ning%20Fu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Lauren_B_-Banks-Aff1-Aff2-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Lauren B. Banks</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Lauren%20B.%20Banks" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Lauren%20B.%20Banks" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Lauren%20B.%20Banks%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Fei-Yi-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Fei Yi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Fei%20Yi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Fei%20Yi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Fei%20Yi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Enric-Vercher-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Enric Vercher</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Enric%20Vercher" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Enric%20Vercher" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Enric%20Vercher%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Inaki-Etxeberria-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Inaki Etxeberria</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Inaki%20Etxeberria" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Inaki%20Etxeberria" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Inaki%20Etxeberria%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Watchain_D_-Bestman-Aff10-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Watchain D. Bestman</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Watchain%20D.%20Bestman" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Watchain%20D.%20Bestman" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Watchain%20D.%20Bestman%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Arnaud-Da_Cruz_Paula-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Arnaud Da Cruz Paula</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Arnaud%20Da%20Cruz%20Paula" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Arnaud%20Da%20Cruz%20Paula" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Arnaud%20Da%20Cruz%20Paula%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ilinca_S_-Aricescu-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Ilinca S. Aricescu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ilinca%20S.%20Aricescu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ilinca%20S.%20Aricescu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ilinca%20S.%20Aricescu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Alexander-Drilon-Aff10-Aff11-Aff14"><span class="c-article-authors-search__title u-h3 js-search-name">Alexander Drilon</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Alexander%20Drilon" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Alexander%20Drilon" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alexander%20Drilon%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Doron-Betel-Aff8-Aff11-Aff15"><span class="c-article-authors-search__title u-h3 js-search-name">Doron Betel</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Doron%20Betel" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Doron%20Betel" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Doron%20Betel%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-David_A_-Scheinberg-Aff6-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">David A. Scheinberg</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=David%20A.%20Scheinberg" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=David%20A.%20Scheinberg" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22David%20A.%20Scheinberg%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Brian_M_-Baker-Aff4-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Brian M. Baker</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Brian%20M.%20Baker" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Brian%20M.%20Baker" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Brian%20M.%20Baker%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Christopher_A_-Klebanoff-Aff1-Aff2-Aff3-Aff10-Aff11-Aff14-Aff16"><span class="c-article-authors-search__title u-h3 js-search-name">Christopher A. Klebanoff</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Christopher%20A.%20Klebanoff" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christopher%20A.%20Klebanoff" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christopher%20A.%20Klebanoff%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>S.S.C. performed all in vitro sensitization, TCR cloning, TCR transduction and TCR functional assays. Single-cell sequencing analysis was performed by F.D., P.Z. and D.B. IP LC–MS/MS experiments were performed and analyzed by M.G.K. and D.A.S. Crystallographic, SPR, p/HLA-I thermal and p/HLA-I kinetic stability experiments were performed by J.M. and B.M.B. Determination of CCF was performed by C.B. and P.R. HLA-I calls, antigen processing and presentation mutations and phylogenetic analyses were performed by C.B. HLA-LOH calls were analyzed by C.B. and I.S.A. Pathology review and tissue processing were performed by H.Y.M., B.W., P.R. and A.D.C.P. H.Y.M. performed TIL calls. E.V.H. and I.E. identified and reconstructed the influenza-specific T cell clonotype for in vivo tumor treatment experiments. S.S.C. and F.Y. performed the in vivo tumor treatment experiments. <i>PIK3CA</i> public NeoAg biospecimen identification, collection and archiving was performed by L.B.B., C.B., S.S.C., S.N.F., W.D.B. and A.D.C.P. The manuscript was written by S.S.C., B.M.B. and C.A.K., with editorial input from all co-authors. S.S.C. and C.A.K. supervised and coordinated all aspects of this work.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:chandrs1@mskcc.org">Smita S. Chandran</a> or <a id="corresp-c2" href="mailto:klebanoc@mskcc.org">Christopher A. Klebanoff</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar4">Competing interests</h3>
                <p>S.S.C. and C.A.K. are inventors of the TCR discovery platform and the <i>PIK3CA</i> public NeoAg TCRs described in this manuscript, which were licensed to Intima Bioscience in December 2021. Licensing revenue is shared with S.S.C. and C.A.K. according to MSKCC institutional policies. MSKCC has filed for patent protection for D.A.S. and M.G.K. for work related to mass spectrometry. C.A.K. has consulted for, or is on the scientific and/or clinical advisory boards for Achilles Therapeutics, Aleta BioTherapeutics, Bellicum Pharmaceuticals, Bristol Myers Squibb, Catamaran Bio, Cell Design Labs, Decheng Capital, G1 Therapeutics, Klus Pharma, Obsidian Therapeutics, PACT Pharma, Roche/Genentech and T-knife. B.M.B. has previously consulted for Eurkea Therapuetics and is on the scientific advisory board of T-Cure Bioscience. D.A.S. has an equity interest in, consults for or is on the board of Sellas Life Sciences, Pfizer, Oncopep, Actinium, Co-Immune, Eureka, Repertoire, Sapience, Iovance and Arvinas. B.W. reports ad hoc membership of the scientific advisory board of Repare Therapeutics. A.D.C.P. has received honoraria or has served on advisory boards for Ignyta/Genentech/Roche, Loxo/Bayer/Lilly, Takeda/Ariad/Millenium, TP Therapeutics, AstraZeneca, Pfizer, Blueprint Medicines, Helsinn, Beigene, BergenBio, Hengrui Therapeutics, Exelixis, Tyra Biosciences, Verastem, MORE Health, Abbvie, 14ner/Elevation Oncology, ArcherDX, Monopteros, Novartis, EMD Serono, Medendi, Repare RX, Nuvalent, Merus, Chugai Pharmaceutical, Remedica, mBrace, AXIS, EPG Health, Harborside Nexus, Liberum, RV More, Ology, Amgen, TouchIME and Janssen; Associated Research Paid to Institution from Pfizer, Exelixis, GlaxoSmithKlein, Teva, Taiho and PharmaMar; Research support from Foundation Medicine; Royalties from Wolters Kluwer; Other from Merck, Puma, Merus and Boehringer Ingelheim; and CME Honoraria from Medscape, OncLive, PeerVoice, Physicians Education Resources, Targeted Oncology, Research to Practice, Axis, Peerview Institute, Paradigm Medical Communications, WebMD, MJH Life Sciences, Med Learning, Imedex, Answers in CME, Clinical Care Options, EPG Health, JNCC/Harborside, Liberum and Remedica. All other authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
              
              
                <h3 class="c-article__sub-heading" id="FPar3">Peer review information</h3>
                <p><i>Nature Medicine</i> thanks Michael Birnbaum and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Editor recognition statement: Saheli Sadanand was the primary editor on this article and managed its editorial process and peer review in collaboration with the rest of the editorial team.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec32-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec32">Extended data</h2><div class="c-article-section__content" id="Sec32-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 variable chain gene sequences" href="/articles/s41591-022-01786-3/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig7_ESM.jpg">Extended Data Fig. 1 Variable chain gene sequences and HLA specificity of a <i>PIK3CA</i> public NeoAg-specific TCR panel.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Table listing <i>TRAV</i>, <i>TRAJ</i>, <i>TRBV</i> and <i>TRBJ</i> gene segment composition, CDR3 sequences, and CDR3 lengths for a panel of <i>PIK3CA</i> public NeoAg TCRs retrieved using the SIFT-seq discovery platform. (<b>b</b>) Representative FACS plots and (<b>c</b>) summary bar graphs demonstrating the specificity of <i>PIK3CA</i> public NeoAg TCRs 1-4 for HLA-A03 supertype members HLA-A*03:01, HLA-A*03:02, and HLA-A*11:01. The number of shared amino acids and % homology of each supertype member to HLA-A*03:01 are shown. T cells transduced with individual TCR panel members were co-cultured with mono-allelic cell lines co-expressing the indicated HLA-A03 supertype member and either Mut or WT <i>PIK3CA</i>. Numbers within each FACS plot and bar graph indicate the frequency of TNFα producing T cells after pre-gating on live<sup>+</sup>mTCR<sup>+</sup>CD8<sup>+</sup> cells. Bar graphs displayed as mean ± SEM using <i>n</i> = 3 biologic replicates per condition. Data shown is representative of 2 independent experiments.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 an immuno-peptidomic screen f" href="/articles/s41591-022-01786-3/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig8_ESM.jpg">Extended Data Fig. 2 An immuno-peptidomic screen for HLA-A3 supertype-restricted public neoantigens resulting from common <i>PIK3CA</i> hotspot mutations.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Schematic overview of an HLA immunoprecipitation/liquid chromatography tandem mass-spectrometry (HLA-IP/MS) screen to identify endogenously processed and presented peptides resulting from the <i>PIK3CA</i> hotspot mutations E542K, E545K, H1047R and H1047L in the context of various HLA-A3 supertype members. COS-7 was co-electroporated with IVT mRNA encoding either HLA-A*03:01, HLA-A*03:02, or HLA-A*11:01 and an individual <i>PIK3CA</i> hotspot mutation. (<b>b</b>) Summary table listing the NetMHCpan 4.1 predicted peptide sequence, eluted ligand % rank, and binding affinity (B.A.) of various peptides containing a PI3Kα hotspot mutations to HLA-A*03:01. Whether the peptide was detected by HLA-IP/MS is listed in the adjacent column. (<b>c</b>) Summary table listing the NetMHCpan 4.1 predicted peptide sequence, eluted ligand (E.L.) % rank, and binding affinity (B.A.) of the ALHGGWTTK peptide (pMut) to HLA-A*03:01, HLA-A*03:02, and HLA-A*11:01. Whether the peptide was detected by HLA-IP/MS is listed in the adjacent column. Heat map colors in (<b>b</b>) and (<b>c</b>) indicate the relative value for the NetMHCpan 4.1 prediction (red = low, blue = high) or whether the peptide sequence was detected by HLA-IP/MS (red = yes, blue = no).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 structural details of the tcr" href="/articles/s41591-022-01786-3/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig9_ESM.jpg">Extended Data Fig. 3 Structural details of the TCR3/pMut/HLA-A*03:01 complex, correlations with TCR3’s peptide recognition motif, and measurement of TCR3’s binding affinity.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Structural overview of the TCR3 pMut/HLA-A*03:01 ternary complex at 2.1 Å resolution. The color scheme is indicated and replicated throughout. (<b>b</b>) Top view of the pMut/HLA-A*03:01 complex displaying the crossing angle and positions of TCR3’s six CDR loops. Spheres show the centers of mass of the TCR Vα and Vβ domains. (<b>c</b>) Amino acids of TCR3’s CDRα and CDRβ loops that interact with the pMut peptide. The single hydrogen bond formed between TCR3 and pMut is indicated by a dashed red line. Amino acids are identified by standard one-letter codes followed by position number. The side chains of contacting residues from the TCR are also identified by AA and hemi-chain position number. (<b>d</b>) Identification of TCR3’s peptide recognition motif using alanine and glycine scanning. Intracellular FACS for TNFα production to Gly (upper) or Ala (lower) substituted peptides. TCR3-transduced T cells (identified by gating on mTCR<sup>+</sup> lymphocytes) were co-cultured with HLA-A*03:01<sup>+</sup> targets pulsed with 1μM of indicated peptides. Results shown as the mean ± SEM percent maximum response relative to the native pMut peptide using <i>n</i> = 3 biologic replicates per condition. ‘x’ indicates positions not amenable to substitution. (<b>e</b>) Steady-state binding equilibrium measuring the dissociation constant (<i>K</i><sub>d</sub>) for TCR3 to the pMut/HLA-I complex. Results shown are the average ± standard deviation of <i>n</i> = 3 independent experiments.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 mutant peptide conformational" href="/articles/s41591-022-01786-3/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig10_ESM.jpg">Extended Data Fig. 4 Mutant peptide conformational changes following TCR3 or TCR4 binding.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Visualization of the conformations of the pMut backbone in the unbound state and upon binding of TCR3 and TCR4, emphasizing changes that occur in the positions of the P6 Trp side chain and the P4 Gly backbone carbonyl. (<b>b</b>) Table summarizing changes (in degrees) in the backbone dihedral angles (ϕ) and (ψ) of the pMut peptide following binding of TCR4 (top) or TCR3 (bottom). For binding of TCR4, the peptide conformational change is primarily driven by a change in the P4 ψ and P5 ϕ, whereas for TCR3 the change is driven by smaller dihedral changes spanning P4 to P7.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 contact matrices defining int" href="/articles/s41591-022-01786-3/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig11_ESM.jpg">Extended Data Fig. 5 Contact matrices defining interatomic interactions between either TCR3 or TCR4 and the pMut/HLA-A*03:01 complex.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Contact matrices for the (<b>a</b>) TCR3/pMut/HLA-A*03:01 or the (<b>b</b>) TCR4/pMut/HLA-A*03:01 complexes. Contacting amino acids, defined as interatomic distances ≤4 Å, are shown. The numbers in each cell represent the number of interatomic contacts for each amino acid pair listed. Red boxes indicate the presence of a hydrogen bond; red circles indicate a salt-bridge (can be &gt;4 Å). Cells are colored according to the total number of contacts, from white (minimum) to purple (maximum). Residue numbers listed for each TCR hemichain and HLA-A*03:01 reflect subtraction of each protein’s leader sequence as indicated in the PDB data files.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 hla class i-dependent recogni" href="/articles/s41591-022-01786-3/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig12_ESM.jpg">Extended Data Fig. 6 HLA class I-dependent recognition of an HLA-A*03:01<sup>+</sup> / <i>PIK3CA</i> (H1047L)<sup>+</sup> patient-derived xenograft by TCR4-transduced CD8<sup>+</sup> T cells directly <i>ex vivo</i>.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Graphical overview of the generation and experimental testing of a patient-derived xenograft (PDX) from an HLA-A*03:01<sup>+</sup> patient with uterine serous carcinoma (USC) associated with the <i>PIK3CA</i> (H1047L) mutation. The PDX (henceforth PDX USC_X10) was established from a surgically resected lung metastasis. (<b>b</b>) Comparison of the mutational landscape and HLA-I haplotype of the primary cancer versus PDX USC_X10 using high-coverage next-gen sequencing (NGS) with the MSK-IMPACT platform. The clonal <i>PIK3CA</i> (H1047L) mutation and <i>TP53</i> mutation were shared; however, a subclonal <i>ARID5B</i> (p.L1124fs) mutation (variant allele frequency &lt; 7%) detected in the primary tumor was not found in the lung metastasis-derived PDX. (<b>c</b>) FACS analysis of HLA-I and HLA-A*03 expression on a single-cell digest of the explanted USC_X10 PDX after two serial passages in NSG mice. (<b>d</b>) Comparison of the fold-change (fc) in 4-1BB expression on TCR4 transduced CD8<sup>+</sup> T cells co-cultured overnight with a single cell-digest of explanted PDX USC_X10 cells in the absence or presence of an anti-HLA-I antibody. Results shown as mean ± SEM using <i>n</i> = 3 biologic replicates per condition. ****<i>P</i> &lt; 0.0001 and ***<i>P</i> = 0.001 using a two-sided Student’s t-test with Bonferroni correction.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 establishment of a transplant" href="/articles/s41591-022-01786-3/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig13_ESM.jpg">Extended Data Fig. 7 Establishment of a transplantable HLA-A*03:01<sup>+</sup> / <i>PIK3CA</i> (H1047L) breast cancer tumor model and associated WT <i>PIK3CA</i> control.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Mutational landscape and (<b>b</b>) HLA-I haplotype of isogenic HCC70 breast adenocarcinoma tumor cell lines that express either WT <i>PIK3CA</i> (HCC70-WT <i>PIK3CA</i>) or <i>PIK3CA</i> (H1047L) (HCC70-Mut <i>PIK3CA</i>). (<b>c</b>) In vivo growth kinetics of HCC70-WT <i>PIK3CA</i> or HCC70-Mut <i>PIK3CA</i> following subcutaneous (s.c.) implantation of 3 x 10<sup>6</sup> tumor cells into immune-deficient NSG mice. Data shown as mean ± SEM for <i>n</i> = 3 mice per tumor cell line. **<i>P</i> = 0.0056 using a two-way ANOVA. (<b>d</b>) Image comparing explanted tumor sizes and (<b>e</b>) FACS analysis of HLA-A*03 expression on HCC70-WT <i>PIK3CA</i> or HCC70-Mut <i>PIK3CA</i> cells 24 days after s.c. implantation. (<b>f</b>) Comparison of the fold-change (fc) in 4-1BB expression on TCR4 transduced (Td) CD8<sup>+</sup> T cells co-cultured overnight with a single cell-digest of explanted HCC70-WT <i>PIK3CA</i> or HCC70-Mut <i>PIK3CA</i> tumor cells. Results shown as mean ± SEM using <i>n</i> = 3 biologic replicates per condition. ****<i>P</i> &lt; 0.0001, ***<i>P</i> = 0.0002, and **<i>P</i> = 0.001 using a two-sided Student’s t-test.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8" href="/articles/s41591-022-01786-3/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig14_ESM.jpg">Extended Data Fig. 8</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Demographics of HLA-A*03:01<sup>+</sup> cancer patients expressing the <i>PIK3CA</i> (H1047L) public neoantigen.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig15"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 application of the sift-seq d" href="/articles/s41591-022-01786-3/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig15_ESM.jpg">Extended Data Fig. 9 Application of the SIFT-seq discovery platform for the identification and retrieval of a Mut <i>PIK3CA</i>-specific TCR from a public NeoAg-expressing breast cancer patient.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) PBMC was sampled from an HLA-A*03:01<sup>+</sup> breast cancer patient (subject MSK_02) whose tumor harbored the <i>PIK3CA</i> (H1047L) mutation (Mut). (<b>b</b>) Log<sub>2</sub> fold-change (FC) ratio of <i>IFNG</i> transcripts from <i>n</i> = 69 unique TCR clonotypes identified using single-cell RNA and V(D)J TCR sequencing from a screen positive ‘hit’ well. Matched aliquots of sensitized T cells from the patient were stimulated with Mut <i>PIK3CA</i> or wildtype <i>PIK3CA</i> (WT) prior to single-cell sequencing. The mean <i>IFNG</i> ratio for all evaluable TCR clonotypes is shown. Statistical analyses were performed for all clonotypes with a minimal ratio ≥2 (dashed line). <b>*</b>***<i>P</i> = 1.98e<sup>−17</sup> and ns = not significant using a two-sided Welch’s t-test. Volcano plots displaying global transcriptomic changes for (<b>c</b>) non-reactive MSK_02 clonotype 12 or (<b>d</b>) reactive MSK_02 clonotype 19 following stimulation with Mut versus WT <i>PIK3CA</i>. Vertical and horizontal dashed lines indicate thresholds for gene expression FC and statistical significance, respectively. Orange and blue dots represent significantly up and down-regulated genes following Mut <i>PIK3CA</i> stimulation, respectively. (<b>e</b>) Violin plots depicting transcript levels for the lineage markers <i>CD3E</i>, <i>CD4</i>, and <i>CD8A</i> from MSK_02 clonotype 19. Violin distributions are centered around the median (red horizontal line) with quartiles ranges displayed above and below (dashed horizontal lines). The maxima and minima are represented by the top and bottom of each plot. ****<i>P</i> &lt; 0.0001 using a two-sided Student’s t-test with Bonferroni correction. (<b>f</b>) Representative FACS plots and (<b>g</b>) summary bar graph of TNFα production by CD8<sup>+</sup> T cells transduced with the SIFT-seq retrieved MSK_02 clonotype 19 TCR. Transduced T cells were co-cultured with HLA-A*03:01<sup>+</sup> target cells that express either WT or Mut <i>PIK3CA</i>. Symbols and bar graphs displayed as mean ± SEM using <i>n</i> = 3 biologic replicates per condition. ***<i>P</i> = 0.0003 using a two-sided Student’s t-test.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig16"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 hla-a loss of heterozygosity" href="/articles/s41591-022-01786-3/figures/16" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig16_ESM.jpg">Extended Data Fig. 10 <i>HLA-A</i> loss of heterozygosity events in HLA-A*03:01<sup>+</sup> cancer patients with tumors harboring <i>PIK3CA</i> (H1047L), <i>PIK3CA</i> (E542K), <i>PIK3CA</i> (E545K), or WT <i>PIK3CA</i>.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a</b>) Variant allele frequency (VAF) at each of the positions in <i>HLA-A</i> that are mismatched between <i>HLA-A*32:01</i> and <i>HLA-A*03:01</i> are shown for the PMBC and tumor samples of subject MSK_02. Violin distributions are centered around the median (solid horizontal line) with quartiles ranges displayed above and below (dashed horizontal lines). The maxima and minima are represented by the top and bottom of each plot. ****<i>P</i> &lt; 0.0001 using a two-sided Student’s t-test. (<b>b</b>) Cancer cell fraction (CCF) for <i>PIK3CA</i> (H1047L) in subject MSK_02. Results displayed as median ± 95% confidence interval. <b>(c,d)</b> Display of <i>HLA-A</i> allele-specific copy numbers for subjects MSK_01 and MSK_02 as an alternative method for establishing <i>HLA</i> loss of heterozygosity. Results were generated using the LOHHLA algorithm. (<b>e</b>) Comparison of which <i>HLA-A</i> allele is lost in HLA-A*03:01<sup>+</sup> cancer patients with an <i>HLA-A</i> LOH event and a tumor (any histology) harboring <i>PIK3CA</i> (H1047L), <i>PIK3CA</i> (E542K), or <i>PIK3CA</i> (E545K) mutation. <i>HLA-A</i> LOH events in HLA-A*03:01<sup>+</sup> patients with a WT <i>PIK3CA</i> breast cancer are shown as an additional comparison. Hashed lines indicate ±50% threshold.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec33-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec33">Supplementary information</h2><div class="c-article-section__content" id="Sec33-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Figs. 1–15 and Supplementary Tables 1–5</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Immunogenicity%20and%20therapeutic%20targeting%20of%20a%20public%20neoantigen%20derived%20from%20mutated%20PIK3CA&amp;author=Smita%20S.%20Chandran%20et%20al&amp;contentID=10.1038%2Fs41591-022-01786-3&amp;copyright=The%20Author%28s%29&amp;publication=1078-8956&amp;publicationDate=2022-04-28&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41591-022-01786-3" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41591-022-01786-3" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Chandran, S.S., Ma, J., Klatt, M.G. <i>et al.</i> Immunogenicity and therapeutic targeting of a public neoantigen derived from mutated <i>PIK3CA</i>.
                    <i>Nat Med</i> <b>28</b>, 946–957 (2022). https://doi.org/10.1038/s41591-022-01786-3</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01786-3?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-04-02">02 April 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-03-16">16 March 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-04-28">28 April 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-05">May 2022</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41591-022-01786-3</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-block"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/cancer-immunotherapy" data-track="click" data-track-action="view subject" data-track-label="link">Cancer immunotherapy</a></li><li class="c-article-subject-list__subject"><a href="/subjects/sequencing" data-track="click" data-track-action="view subject" data-track-label="link">Sequencing</a></li><li class="c-article-subject-list__subject"><a href="/subjects/t-cells" data-track="click" data-track-action="view subject" data-track-label="link">T cells</a></li><li class="c-article-subject-list__subject"><a href="/subjects/tumour-immunology" data-track="click" data-track-action="view subject" data-track-label="link">Tumour immunology</a></li>
        </ul>
    </div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section" data-test="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Identification of novel KRASG12D neoantigen specific TCRs and a strategy to eliminate off-target recognition" href="https://doi.org/10.1186/s12967-025-06094-1">
                                        Identification of novel KRASG12D neoantigen specific TCRs and a strategy to eliminate off-target recognition
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Xiaojian Han</li><li>Xiaxia Han</li><li>Meiying Shen</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Journal of Translational Medicine</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:HLA class II-restricted T cell epitopes in public neoantigens of ESR1 and PIK3CA in breast cancer" href="https://doi.org/10.1186/s12885-025-13992-6">
                                        HLA class II-restricted T cell epitopes in public neoantigens of ESR1 and PIK3CA in breast cancer
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Yukari Ando</li><li>Hiroko Miyadera</li><li>Hisato Hara</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>BMC Cancer</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Dynamic allostery in the peptide/MHC complex enables TCR neoantigen selectivity" href="https://doi.org/10.1038/s41467-025-56004-8">
                                        Dynamic allostery in the peptide/MHC complex enables TCR neoantigen selectivity
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Jiaqi Ma</li><li>Cory M. Ayres</li><li>Brian M. Baker</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Tumour-wide RNA splicing aberrations generate actionable public neoantigens" href="https://doi.org/10.1038/s41586-024-08552-0">
                                        Tumour-wide RNA splicing aberrations generate actionable public neoantigens
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Darwin W. Kwok</li><li>Nicholas O. Stevers</li><li>Hideho Okada</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:The potential and promise for clinical application of adoptive T cell therapy in cancer" href="https://doi.org/10.1186/s12967-024-05206-7">
                                        The potential and promise for clinical application of adoptive T cell therapy in cancer
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Yinqi Li</li><li>Yeteng Zheng</li><li>Zhiyao He</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Journal of Translational Medicine</i> (2024)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01786-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                
            
        
    </div>

    
        
    

    
    
        <div class="c-article-associated-content__container">
            <section>
                <h2 class="c-article-associated-content__title u-mb-24">Associated content</h2>
                
                    
                    
                        <div class="u-full-height u-mb-24">
                            
    <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
            <div class="c-card__body">
                <h3 class="c-card__title">
                    <a href="https://www.nature.com/articles/s41591-022-01806-2" class="c-card__link u-link-inherit" data-track="click" data-track-action="view article" data-track-category="associated content" data-track-label="news_and_views"><i>PIK3CA</i> hotspot mutation generates a shared neoantigen targetable by TCR gene therapy</a>
                </h3>
                
                
    <div class="c-card__section c-meta">
        
            <span class="c-meta__item">Nature Medicine</span>
        
        <span class="c-meta__item" data-test="article.type"><span class="c-meta__type">Research Briefing</span></span>
        
        
            <time class="c-meta__item" datetime="2022-04-28">28 Apr 2022</time>
        
    </div>

            </div>
        </div>
    </article>


                        </div>
                    
                
            </section>
        </div>
        <script>
            window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "news_and_views";
            
        </script>
    

    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
            <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" role="tabpanel"><div class="c-reading-companion__scroll-pane" style="max-height: none;"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Main">Main</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec9" class="c-reading-companion__section-item"><a href="#Sec9" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec10" class="c-reading-companion__section-item"><a href="#Sec10" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-data-availability" class="c-reading-companion__section-item"><a href="#data-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Data availability">Data availability</a></li><li id="rc-sec-code-availability" class="c-reading-companion__section-item"><a href="#code-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Code availability">Code availability</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-peer-review" class="c-reading-companion__section-item"><a href="#peer-review" data-track="click" data-track-action="section anchor" data-track-label="link:Peer review">Peer review</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec32" class="c-reading-companion__section-item"><a href="#Sec32" data-track="click" data-track-action="section anchor" data-track-label="link:Extended data">Extended data</a></li><li id="rc-sec-Sec33" class="c-reading-companion__section-item"><a href="#Sec33" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by">This article is cited by</a></li></ul></div>
                <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="" style="display: none !important;">
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-test="right-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41591-022-01786-3;doi=10.1038/s41591-022-01786-3;subjmeta=1059,1619,1647,2325,250,514,554,580,631,67,692,699;kwrd=Cancer+immunotherapy,Sequencing,T+cells,Tumour+immunology">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=328183956&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01786-3%26doi%3D10.1038/s41591-022-01786-3%26subjmeta%3D1059,1619,1647,2325,250,514,554,580,631,67,692,699%26kwrd%3DCancer+immunotherapy,Sequencing,T+cells,Tumour+immunology">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=328183956&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01786-3%26doi%3D10.1038/s41591-022-01786-3%26subjmeta%3D1059,1619,1647,2325,250,514,554,580,631,67,692,699%26kwrd%3DCancer+immunotherapy,Sequencing,T+cells,Tumour+immunology"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Fig. 1: Development and validation of the SIFT-seq discovery platform for Mut <i>PIK3CA</i>-specific TCRs.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig1_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig1_HTML.png" alt="figure 1" aria-describedby="rc-Fig1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Fig. 2: HLA restriction and functional characterization of <i>PIK3CA</i> public NeoAg-specific TCRs.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig2_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig2_HTML.png" alt="figure 2" aria-describedby="rc-Fig2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Fig. 3: Mechanism of immunogenicity for a <i>PIK3CA</i> public NeoAg.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig3_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig3_HTML.png" alt="figure 3" aria-describedby="rc-Fig3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Fig. 4: Structural correlates of affinity and specificity for a <i>PIK3CA</i> public NeoAg-specific TCR.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig4_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig4_HTML.png" alt="figure 4" aria-describedby="rc-Fig4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Fig. 5: In vivo anti-tumor efficacy of adoptively transferred T cells genetically engineered with a <i>PIK3CA</i> public NeoAg TCR.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig5_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig5_HTML.png" alt="figure 5" aria-describedby="rc-Fig5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig6">Fig. 6: Clonality, immunogenicity and immune resistance to a <i>PIK3CA</i> public NeoAg in patients with cancer.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig6_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig6_HTML.png" alt="figure 6" aria-describedby="rc-Fig6"></picture><p class="c-reading-companion__figure-links"><a href="#Fig6" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/6" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig7">Extended Data Fig. 1 Variable chain gene sequences and HLA specificity of a <i>PIK3CA</i> public NeoAg-specific TCR panel.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig7_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig7_ESM.jpg" alt="extended data figure 7" aria-describedby="rc-Fig7"></picture><p class="c-reading-companion__figure-links"><a href="#Fig7" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/7" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig8">Extended Data Fig. 2 An immuno-peptidomic screen for HLA-A3 supertype-restricted public neoantigens resulting from common <i>PIK3CA</i> hotspot mutations.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig8_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig8_ESM.jpg" alt="extended data figure 8" aria-describedby="rc-Fig8"></picture><p class="c-reading-companion__figure-links"><a href="#Fig8" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/8" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig9">Extended Data Fig. 3 Structural details of the TCR3/pMut/HLA-A*03:01 complex, correlations with TCR3’s peptide recognition motif, and measurement of TCR3’s binding affinity.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig9_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig9_ESM.jpg" alt="extended data figure 9" aria-describedby="rc-Fig9"></picture><p class="c-reading-companion__figure-links"><a href="#Fig9" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/9" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig10">Extended Data Fig. 4 Mutant peptide conformational changes following TCR3 or TCR4 binding.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig10_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig10_ESM.jpg" alt="extended data figure 10" aria-describedby="rc-Fig10"></picture><p class="c-reading-companion__figure-links"><a href="#Fig10" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/10" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig11">Extended Data Fig. 5 Contact matrices defining interatomic interactions between either TCR3 or TCR4 and the pMut/HLA-A*03:01 complex.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig11_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig11_ESM.jpg" alt="extended data figure 11" aria-describedby="rc-Fig11"></picture><p class="c-reading-companion__figure-links"><a href="#Fig11" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/11" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig12">Extended Data Fig. 6 HLA class I-dependent recognition of an HLA-A*03:01<sup>+</sup> / <i>PIK3CA</i> (H1047L)<sup>+</sup> patient-derived xenograft by TCR4-transduced CD8<sup>+</sup> T cells directly <i>ex vivo</i>.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig12_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig12_ESM.jpg" alt="extended data figure 12" aria-describedby="rc-Fig12"></picture><p class="c-reading-companion__figure-links"><a href="#Fig12" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/12" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig13">Extended Data Fig. 7 Establishment of a transplantable HLA-A*03:01<sup>+</sup> / <i>PIK3CA</i> (H1047L) breast cancer tumor model and associated WT <i>PIK3CA</i> control.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig13_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig13_ESM.jpg" alt="extended data figure 13" aria-describedby="rc-Fig13"></picture><p class="c-reading-companion__figure-links"><a href="#Fig13" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/13" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig14">Extended Data Fig. 8</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig14_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig14_ESM.jpg" alt="extended data figure 14" aria-describedby="rc-Fig14"></picture><p class="c-reading-companion__figure-links"><a href="#Fig14" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/14" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig15">Extended Data Fig. 9 Application of the SIFT-seq discovery platform for the identification and retrieval of a Mut <i>PIK3CA</i>-specific TCR from a public NeoAg-expressing breast cancer patient.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig15_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig15_ESM.jpg" alt="extended data figure 15" aria-describedby="rc-Fig15"></picture><p class="c-reading-companion__figure-links"><a href="#Fig15" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/15" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig16">Extended Data Fig. 10 <i>HLA-A</i> loss of heterozygosity events in HLA-A*03:01<sup>+</sup> cancer patients with tumors harboring <i>PIK3CA</i> (H1047L), <i>PIK3CA</i> (E542K), <i>PIK3CA</i> (E545K), or WT <i>PIK3CA</i>.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig16_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01786-3/MediaObjects/41591_2022_1786_Fig16_ESM.jpg" alt="extended data figure 16" aria-describedby="rc-Fig16"></picture><p class="c-reading-companion__figure-links"><a href="#Fig16" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01786-3/figures/16" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric" data-track-context="article references sidebar"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">Schumacher, T. N. &amp; Schreiber, R. D. Neoantigens in cancer immunotherapy. <i>Science</i> <b>348</b>, 69–74 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aaa4971" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aaa4971" data-track-item_id="10.1126/science.aaa4971">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXls1Wmurc%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25838375" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Neoantigens%20in%20cancer%20immunotherapy&amp;journal=Science&amp;doi=10.1126%2Fscience.aaa4971&amp;volume=348&amp;pages=69-74&amp;publication_year=2015&amp;author=Schumacher%2CTN&amp;author=Schreiber%2CRD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">Parkhurst, M. R. et al. Unique neoantigens arise from somatic mutations in patients with gastrointestinal cancers. <i>Cancer Discov.</i> <b>9</b>, 1022–1035 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1158%2F2159-8290.CD-18-1494" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1158/2159-8290.CD-18-1494" data-track-item_id="10.1158/2159-8290.CD-18-1494">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFGjs77L" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31164343" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7138461" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Unique%20neoantigens%20arise%20from%20somatic%20mutations%20in%20patients%20with%20gastrointestinal%20cancers&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-18-1494&amp;volume=9&amp;pages=1022-1035&amp;publication_year=2019&amp;author=Parkhurst%2CMR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Hu, Z., Ott, P. A. &amp; Wu, C. J. Towards personalized, tumour-specific, therapeutic vaccines for cancer. <i>Nat. Rev. Immunol.</i> <b>18</b>, 168–182 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnri.2017.131" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nri.2017.131" data-track-item_id="10.1038/nri.2017.131">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFGmtbbL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29226910" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20personalized%2C%20tumour-specific%2C%20therapeutic%20vaccines%20for%20cancer&amp;journal=Nat.%20Rev.%20Immunol.&amp;doi=10.1038%2Fnri.2017.131&amp;volume=18&amp;pages=168-182&amp;publication_year=2018&amp;author=Hu%2CZ&amp;author=Ott%2CPA&amp;author=Wu%2CCJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">McGranahan, N. et al. Clonal neoantigens elicit T cell immunoreactivity and sensitivity to immune checkpoint blockade. <i>Science</i> <b>351</b>, 1463–1469 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aaf1490" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aaf1490" data-track-item_id="10.1126/science.aaf1490">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XksFCht70%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26940869" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4984254" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20neoantigens%20elicit%20T%20cell%20immunoreactivity%20and%20sensitivity%20to%20immune%20checkpoint%20blockade&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf1490&amp;volume=351&amp;pages=1463-1469&amp;publication_year=2016&amp;author=McGranahan%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">Miao, D. et al. Genomic correlates of response to immune checkpoint blockade in microsatellite-stable solid tumors. <i>Nat. Genet.</i> <b>50</b>, 1271–1281 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-018-0200-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-018-0200-2" data-track-item_id="10.1038/s41588-018-0200-2">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2nsrvM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30150660" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6119118" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20correlates%20of%20response%20to%20immune%20checkpoint%20blockade%20in%20microsatellite-stable%20solid%20tumors&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0200-2&amp;volume=50&amp;pages=1271-1281&amp;publication_year=2018&amp;author=Miao%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Wolf, Y. et al. UVB-induced tumor heterogeneity diminishes immune response in melanoma. <i>Cell</i> <b>179</b>, 219–235 e221 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2019.08.032" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2019.08.032" data-track-item_id="10.1016/j.cell.2019.08.032">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslOgt7vK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31522890" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6863386" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=UVB-induced%20tumor%20heterogeneity%20diminishes%20immune%20response%20in%20melanoma&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.08.032&amp;volume=179&amp;pages=219-235%20e221&amp;publication_year=2019&amp;author=Wolf%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Reiter, J. G. et al. Minimal functional driver gene heterogeneity among untreated metastases. <i>Science</i> <b>361</b>, 1033–1037 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aat7171" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aat7171" data-track-item_id="10.1126/science.aat7171">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1Chsb7K" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30190408" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6329287" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Minimal%20functional%20driver%20gene%20heterogeneity%20among%20untreated%20metastases&amp;journal=Science&amp;doi=10.1126%2Fscience.aat7171&amp;volume=361&amp;pages=1033-1037&amp;publication_year=2018&amp;author=Reiter%2CJG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Klebanoff, C. A. &amp; Wolchok, J. D. Shared cancer neoantigens: making private matters public. <i>J. Exp. Med.</i> <b>215</b>, 5–7 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1084%2Fjem.20172188" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1084/jem.20172188" data-track-item_id="10.1084/jem.20172188">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2rt7vJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29269561" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5748868" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Shared%20cancer%20neoantigens%3A%20making%20private%20matters%20public&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20172188&amp;volume=215&amp;pages=5-7&amp;publication_year=2018&amp;author=Klebanoff%2CCA&amp;author=Wolchok%2CJD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Wolfel, T. et al. A p16<sup>INK4a</sup>-insensitive CDK4 mutant targeted by cytolytic T lymphocytes in a human melanoma. <i>Science</i> <b>269</b>, 1281–1284 (1995).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.7652577" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.7652577" data-track-item_id="10.1126/science.7652577">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaK2MzotFSltQ%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=7652577" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20p16INK4a-insensitive%20CDK4%20mutant%20targeted%20by%20cytolytic%20T%20lymphocytes%20in%20a%20human%20melanoma&amp;journal=Science&amp;doi=10.1126%2Fscience.7652577&amp;volume=269&amp;pages=1281-1284&amp;publication_year=1995&amp;author=Wolfel%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Linard, B. et al. A <i>ras</i>-mutated peptide targeted by CTL infiltrating a human melanoma lesion. <i>J. Immunol.</i> <b>168</b>, 4802–4808 (2002).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.4049%2Fjimmunol.168.9.4802" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.4049/jimmunol.168.9.4802" data-track-item_id="10.4049/jimmunol.168.9.4802">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38Xjt12ju74%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11971032" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20ras-mutated%20peptide%20targeted%20by%20CTL%20infiltrating%20a%20human%20melanoma%20lesion&amp;journal=J.%20Immunol.&amp;doi=10.4049%2Fjimmunol.168.9.4802&amp;volume=168&amp;pages=4802-4808&amp;publication_year=2002&amp;author=Linard%2CB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Schumacher, T. et al. A vaccine targeting mutant IDH1 induces antitumour immunity. <i>Nature</i> <b>512</b>, 324–327 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnature13387" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nature13387" data-track-item_id="10.1038/nature13387">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsVSitrjM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25043048" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20vaccine%20targeting%20mutant%20IDH1%20induces%20antitumour%20immunity&amp;journal=Nature&amp;doi=10.1038%2Fnature13387&amp;volume=512&amp;pages=324-327&amp;publication_year=2014&amp;author=Schumacher%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Tran, E. et al. Immunogenicity of somatic mutations in human gastrointestinal cancers. <i>Science</i> <b>350</b>, 1387–1390 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aad1253" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aad1253" data-track-item_id="10.1126/science.aad1253">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFKru7%2FL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26516200" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7445892" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Immunogenicity%20of%20somatic%20mutations%20in%20human%20gastrointestinal%20cancers&amp;journal=Science&amp;doi=10.1126%2Fscience.aad1253&amp;volume=350&amp;pages=1387-1390&amp;publication_year=2015&amp;author=Tran%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Chheda, Z. S. et al. Novel and shared neoantigen derived from histone 3 variant H3.3K27M mutation for glioma T cell therapy. <i>J. Exp. Med.</i> <b>215</b>, 141–157 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1084%2Fjem.20171046" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1084/jem.20171046" data-track-item_id="10.1084/jem.20171046">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2rt73N" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29203539" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5748856" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Novel%20and%20shared%20neoantigen%20derived%20from%20histone%203%20variant%20H3.3K27M%20mutation%20for%20glioma%20T%20cell%20therapy&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20171046&amp;volume=215&amp;pages=141-157&amp;publication_year=2018&amp;author=Chheda%2CZS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Veatch, J. R. et al. Tumor-infiltrating BRAFV600E-specific CD4<sup>+</sup> T cells correlated with complete clinical response in melanoma. <i>J. Clin. Invest.</i> <b>128</b>, 1563–1568 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1172%2FJCI98689" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1172/JCI98689" data-track-item_id="10.1172/JCI98689">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29360643" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5873881" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Tumor-infiltrating%20BRAFV600E-specific%20CD4%2B%20T%20cells%20correlated%20with%20complete%20clinical%20response%20in%20melanoma&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI98689&amp;volume=128&amp;pages=1563-1568&amp;publication_year=2018&amp;author=Veatch%2CJR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">Malekzadeh, P. et al. Neoantigen screening identifies broad <i>TP53</i> mutant immunogenicity in patients with epithelial cancers. <i>J. Clin. Invest.</i> <b>129</b>, 1109–1114 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1172%2FJCI123791" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1172/JCI123791" data-track-item_id="10.1172/JCI123791">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30714987" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Neoantigen%20screening%20identifies%20broad%20TP53%20mutant%20immunogenicity%20in%20patients%20with%20epithelial%20cancers&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI123791&amp;volume=129&amp;pages=1109-1114&amp;publication_year=2019&amp;author=Malekzadeh%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">Hsiue, E. H. et al. Targeting a neoantigen derived from a common <i>TP53</i> mutation. <i>Science</i> <b>371</b>, eabc8697 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.abc8697" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.abc8697" data-track-item_id="10.1126/science.abc8697">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXlvFylurs%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33649166" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8208645" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Targeting%20a%20neoantigen%20derived%20from%20a%20common%20TP53%20mutation&amp;journal=Science&amp;doi=10.1126%2Fscience.abc8697&amp;volume=371&amp;publication_year=2021&amp;author=Hsiue%2CEH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Samuels, Y. et al. High frequency of mutations of the <i>PIK3CA</i> gene in human cancers. <i>Science</i> <b>304</b>, 554 (2004).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.1096502" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.1096502" data-track-item_id="10.1126/science.1096502">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXjsVGmsbk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15016963" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=High%20frequency%20of%20mutations%20of%20the%20PIK3CA%20gene%20in%20human%20cancers&amp;journal=Science&amp;doi=10.1126%2Fscience.1096502&amp;volume=304&amp;publication_year=2004&amp;author=Samuels%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">Bailey, M. H. et al. Comprehensive characterization of cancer driver genes and mutations. <i>Cell</i> <b>173</b>, 371–385 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2018.02.060" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2018.02.060" data-track-item_id="10.1016/j.cell.2018.02.060">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXntFaiu7c%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29625053" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6029450" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20characterization%20of%20cancer%20driver%20genes%20and%20mutations&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.02.060&amp;volume=173&amp;pages=371-385&amp;publication_year=2018&amp;author=Bailey%2CMH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">Priestley, P. et al. Pan-cancer whole-genome analyses of metastatic solid tumours. <i>Nature</i> <b>575</b>, 210–216 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-019-1689-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-019-1689-y" data-track-item_id="10.1038/s41586-019-1689-y">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVagsLjK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31645765" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6872491" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Pan-cancer%20whole-genome%20analyses%20of%20metastatic%20solid%20tumours&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1689-y&amp;volume=575&amp;pages=210-216&amp;publication_year=2019&amp;author=Priestley%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Andre, F. et al. Alpelisib for <i>PIK3CA</i>-mutated, hormone receptor-positive advanced breast cancer. <i>N. Engl. J. Med.</i> <b>380</b>, 1929–1940 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa1813904" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa1813904" data-track-item_id="10.1056/NEJMoa1813904">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVWis7nP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31091374" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Alpelisib%20for%20PIK3CA-mutated%2C%20hormone%20receptor-positive%20advanced%20breast%20cancer&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1813904&amp;volume=380&amp;pages=1929-1940&amp;publication_year=2019&amp;author=Andre%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Tate, J. G. et al. COSMIC: the Catalogue of Somatic Mutations in Cancer. <i>Nucleic Acids Res.</i> <b>47</b>, D941–D947 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgky1015" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gky1015" data-track-item_id="10.1093/nar/gky1015">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Cgt7bM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30371878" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=COSMIC%3A%20the%20Catalogue%20of%20Somatic%20Mutations%20in%20Cancer&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky1015&amp;volume=47&amp;pages=D941-D947&amp;publication_year=2019&amp;author=Tate%2CJG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Gonzalez-Galarza, F. F. et al. Allele frequency net database (AFND) 2020 update: gold-standard data classification, open access genotype data and new query tools. <i>Nucleic Acids Res.</i> <b>48</b>, D783–D788 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Glt7%2FI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31722398" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Allele%20frequency%20net%20database%20%28AFND%29%202020%20update%3A%20gold-standard%20data%20classification%2C%20open%20access%20genotype%20data%20and%20new%20query%20tools&amp;journal=Nucleic%20Acids%20Res.&amp;volume=48&amp;pages=D783-D788&amp;publication_year=2020&amp;author=Gonzalez-Galarza%2CFF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">Sidney, J., Peters, B., Frahm, N., Brander, C. &amp; Sette, A. HLA class I supertypes: a revised and updated classification. <i>BMC Immunol.</i> <b>9</b>, 1 (2008).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/1471-2172-9-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/1471-2172-9-1" data-track-item_id="10.1186/1471-2172-9-1">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18211710" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2245908" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtFWrsbw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=HLA%20class%20I%20supertypes%3A%20a%20revised%20and%20updated%20classification&amp;journal=BMC%20Immunol.&amp;doi=10.1186%2F1471-2172-9-1&amp;volume=9&amp;publication_year=2008&amp;author=Sidney%2CJ&amp;author=Peters%2CB&amp;author=Frahm%2CN&amp;author=Brander%2CC&amp;author=Sette%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Laugel, B. et al. Different T cell receptor affinity thresholds and CD8 coreceptor dependence govern cytotoxic T lymphocyte activation and tetramer binding properties. <i>J. Biol. Chem.</i> <b>282</b>, 23799–23810 (2007).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1074%2Fjbc.M700976200" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1074/jbc.M700976200" data-track-item_id="10.1074/jbc.M700976200">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXoslOntrw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17540778" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Different%20T%20cell%20receptor%20affinity%20thresholds%20and%20CD8%20coreceptor%20dependence%20govern%20cytotoxic%20T%20lymphocyte%20activation%20and%20tetramer%20binding%20properties&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M700976200&amp;volume=282&amp;pages=23799-23810&amp;publication_year=2007&amp;author=Laugel%2CB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">Johnson, L. A. et al. Gene therapy with human and mouse T-cell receptors mediates cancer regression and targets normal tissues expressing cognate antigen. <i>Blood</i> <b>114</b>, 535–546 (2009).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1182%2Fblood-2009-03-211714" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1182/blood-2009-03-211714" data-track-item_id="10.1182/blood-2009-03-211714">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXpt1Wmsb0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19451549" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929689" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20therapy%20with%20human%20and%20mouse%20T-cell%20receptors%20mediates%20cancer%20regression%20and%20targets%20normal%20tissues%20expressing%20cognate%20antigen&amp;journal=Blood&amp;doi=10.1182%2Fblood-2009-03-211714&amp;volume=114&amp;pages=535-546&amp;publication_year=2009&amp;author=Johnson%2CLA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Lu, Y. C. et al. Treatment of patients with metastatic cancer using a major histocompatibility complex class II-restricted T-cell receptor targeting the cancer germline antigen MAGE-A3. <i>J. Clin. Oncol.</i> <b>35</b>, 3322–3329 (2017).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">D’Angelo, S. P. et al. Antitumor activity associated with prolonged persistence of adoptively transferred NY-ESO-1 (c259)T cells in synovial sarcoma. <i>Cancer Discov.</i> <b>8</b>, 944–957 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1158%2F2159-8290.CD-17-1417" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1158/2159-8290.CD-17-1417" data-track-item_id="10.1158/2159-8290.CD-17-1417">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29891538" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8092079" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsVarsbfK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Antitumor%20activity%20associated%20with%20prolonged%20persistence%20of%20adoptively%20transferred%20NY-ESO-1%20%28c259%29T%20cells%20in%20synovial%20sarcoma&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-17-1417&amp;volume=8&amp;pages=944-957&amp;publication_year=2018&amp;author=D%E2%80%99Angelo%2CSP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Nagarsheth, N. B. et al. TCR-engineered T cells targeting E7 for patients with metastatic HPV-associated epithelial cancers. <i>Nat. Med</i>. <b>27</b>, 419–425 (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">Reynisson, B., Alvarez, B., Paul, S., Peters, B. &amp; Nielsen, M. NetMHCpan-4.1 and NetMHCIIpan-4.0: improved predictions of MHC antigen presentation by concurrent motif deconvolution and integration of MS MHC eluted ligand data. <i>Nucleic Acids Res.</i> <b>48</b>, W449–W454 (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">Sarkizova, S. et al. A large peptidome dataset improves HLA class I epitope prediction across most of the human population. <i>Nat. Biotechnol.</i> <b>38</b>, 199–209 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41587-019-0322-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41587-019-0322-9" data-track-item_id="10.1038/s41587-019-0322-9">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVSktbvP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31844290" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20large%20peptidome%20dataset%20improves%20HLA%20class%20I%20epitope%20prediction%20across%20most%20of%20the%20human%20population&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-019-0322-9&amp;volume=38&amp;pages=199-209&amp;publication_year=2020&amp;author=Sarkizova%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Garcia, K. C. &amp; Adams, E. J. How the T cell receptor sees antigen—a structural view. <i>Cell</i> <b>122</b>, 333–336 (2005).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2005.07.015" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2005.07.015" data-track-item_id="10.1016/j.cell.2005.07.015">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXovV2qsro%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16096054" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=How%20the%20T%20cell%20receptor%20sees%20antigen%E2%80%94a%20structural%20view&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2005.07.015&amp;volume=122&amp;pages=333-336&amp;publication_year=2005&amp;author=Garcia%2CKC&amp;author=Adams%2CEJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Gowthaman, R. &amp; Pierce, B. G. TCR3d: the T cell receptor structural repertoire database. <i>Bioinformatics</i> <b>35</b>, 5323–5325 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtz517" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btz517" data-track-item_id="10.1093/bioinformatics/btz517">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtl2ksr%2FP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31240309" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6954642" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=TCR3d%3A%20the%20T%20cell%20receptor%20structural%20repertoire%20database&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtz517&amp;volume=35&amp;pages=5323-5325&amp;publication_year=2019&amp;author=Gowthaman%2CR&amp;author=Pierce%2CBG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">Bagaev, D. V. et al. VDJdb in 2019: database extension, new analysis infrastructure and a T-cell receptor motif compendium. <i>Nucleic Acids Res.</i> <b>48</b>, D1057–D1062 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgkz874" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gkz874" data-track-item_id="10.1093/nar/gkz874">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31588507" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=VDJdb%20in%202019%3A%20database%20extension%2C%20new%20analysis%20infrastructure%20and%20a%20T-cell%20receptor%20motif%20compendium&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkz874&amp;volume=48&amp;pages=D1057-D1062&amp;publication_year=2020&amp;author=Bagaev%2CDV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">Aleksic, M. et al. Different affinity windows for virus and cancer-specific T-cell receptors: implications for therapeutic strategies. <i>Eur. J. Immunol.</i> <b>42</b>, 3174–3179 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1002%2Feji.201242606" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1002/eji.201242606" data-track-item_id="10.1002/eji.201242606">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsFWiu73E" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22949370" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3776049" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Different%20affinity%20windows%20for%20virus%20and%20cancer-specific%20T-cell%20receptors%3A%20implications%20for%20therapeutic%20strategies&amp;journal=Eur.%20J.%20Immunol.&amp;doi=10.1002%2Feji.201242606&amp;volume=42&amp;pages=3174-3179&amp;publication_year=2012&amp;author=Aleksic%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">Wu, D., Gallagher, D. T., Gowthaman, R., Pierce, B. G. &amp; Mariuzza, R. A. Structural basis for oligoclonal T cell recognition of a shared p53 cancer neoantigen. <i>Nat. Commun.</i> <b>11</b>, 2908 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-16755-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-16755-y" data-track-item_id="10.1038/s41467-020-16755-y">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFGhsrbM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32518267" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7283474" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20basis%20for%20oligoclonal%20T%20cell%20recognition%20of%20a%20shared%20p53%20cancer%20neoantigen&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-16755-y&amp;volume=11&amp;publication_year=2020&amp;author=Wu%2CD&amp;author=Gallagher%2CDT&amp;author=Gowthaman%2CR&amp;author=Pierce%2CBG&amp;author=Mariuzza%2CRA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Sim, M. J. W. et al. High-affinity oligoclonal TCRs define effective adoptive T cell therapy targeting mutant KRAS-G12D. <i>Proc. Natl Acad. Sci. USA</i> <b>117</b>, 12826–12835 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1073%2Fpnas.1921964117" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1073/pnas.1921964117" data-track-item_id="10.1073/pnas.1921964117">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtFWksL%2FF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32461371" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7293613" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=High-affinity%20oligoclonal%20TCRs%20define%20effective%20adoptive%20T%20cell%20therapy%20targeting%20mutant%20KRAS-G12D&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.1921964117&amp;volume=117&amp;pages=12826-12835&amp;publication_year=2020&amp;author=Sim%2CMJW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Devlin, J. R. et al. Structural dissimilarity from self drives neoepitope escape from immune tolerance. <i>Nat. Chem. Biol.</i> <b>16</b>, 1269–1276 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41589-020-0610-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41589-020-0610-1" data-track-item_id="10.1038/s41589-020-0610-1">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Oit77N" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32807968" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8210748" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Structural%20dissimilarity%20from%20self%20drives%20neoepitope%20escape%20from%20immune%20tolerance&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fs41589-020-0610-1&amp;volume=16&amp;pages=1269-1276&amp;publication_year=2020&amp;author=Devlin%2CJR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Singh, N. K. et al. Geometrical characterization of T cell receptor binding modes reveals class-specific binding to maximize access to antigen. <i>Proteins</i> <b>88</b>, 503–513 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1002%2Fprot.25829" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1002/prot.25829" data-track-item_id="10.1002/prot.25829">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhvFymt7nN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31589793" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Geometrical%20characterization%20of%20T%20cell%20receptor%20binding%20modes%20reveals%20class-specific%20binding%20to%20maximize%20access%20to%20antigen&amp;journal=Proteins&amp;doi=10.1002%2Fprot.25829&amp;volume=88&amp;pages=503-513&amp;publication_year=2020&amp;author=Singh%2CNK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">de Castro, E. et al. ScanProsite: detection of PROSITE signature matches and ProRule-associated functional and structural residues in proteins. <i>Nucleic Acids Res.</i> <b>34</b>, W362–W365 (2006).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgkl124" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gkl124" data-track-item_id="10.1093/nar/gkl124">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16845026" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1538847" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28Xps1yit7Y%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ScanProsite%3A%20detection%20of%20PROSITE%20signature%20matches%20and%20ProRule-associated%20functional%20and%20structural%20residues%20in%20proteins&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkl124&amp;volume=34&amp;pages=W362-W365&amp;publication_year=2006&amp;author=Castro%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">Gattinoni, L. et al. Removal of homeostatic cytokine sinks by lymphodepletion enhances the efficacy of adoptively transferred tumor-specific CD8<sup>+</sup> T cells. <i>J. Exp. Med.</i> <b>202</b>, 907–912 (2005).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1084%2Fjem.20050732" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1084/jem.20050732" data-track-item_id="10.1084/jem.20050732">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXhtVygt73L" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16203864" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1397916" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Removal%20of%20homeostatic%20cytokine%20sinks%20by%20lymphodepletion%20enhances%20the%20efficacy%20of%20adoptively%20transferred%20tumor-specific%20CD8%2B%20T%20cells&amp;journal=J.%20Exp.%20Med.&amp;doi=10.1084%2Fjem.20050732&amp;volume=202&amp;pages=907-912&amp;publication_year=2005&amp;author=Gattinoni%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">Samuels, Y. et al. Mutant PIK3CA promotes cell growth and invasion of human cancer cells. <i>Cancer Cell</i> <b>7</b>, 561–573 (2005).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ccr.2005.05.014" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ccr.2005.05.014" data-track-item_id="10.1016/j.ccr.2005.05.014">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXlslCgsrY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15950905" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Mutant%20PIK3CA%20promotes%20cell%20growth%20and%20invasion%20of%20human%20cancer%20cells&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccr.2005.05.014&amp;volume=7&amp;pages=561-573&amp;publication_year=2005&amp;author=Samuels%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">Kang, S., Bader, A. G. &amp; Vogt, P. K. Phosphatidylinositol 3-kinase mutations identified in human cancer are oncogenic. <i>Proc. Natl Acad. Sci. USA</i> <b>102</b>, 802–807 (2005).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1073%2Fpnas.0408864102" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1073/pnas.0408864102" data-track-item_id="10.1073/pnas.0408864102">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXis1Cqsb0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15647370" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC545580" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Phosphatidylinositol%203-kinase%20mutations%20identified%20in%20human%20cancer%20are%20oncogenic&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0408864102&amp;volume=102&amp;pages=802-807&amp;publication_year=2005&amp;author=Kang%2CS&amp;author=Bader%2CAG&amp;author=Vogt%2CPK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">Cheng, D. T. et al. Memorial Sloan Kettering-Integrated Mutation Profiling of Actionable Cancer Targets (MSK-IMPACT): a hybridization capture-based next-generation sequencing clinical assay for solid tumor molecular oncology. <i>J. Mol. Diagn.</i> <b>17</b>, 251–264 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.jmoldx.2014.12.006" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.jmoldx.2014.12.006" data-track-item_id="10.1016/j.jmoldx.2014.12.006">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlt1aitLc%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25801821" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5808190" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Memorial%20Sloan%20Kettering-Integrated%20Mutation%20Profiling%20of%20Actionable%20Cancer%20Targets%20%28MSK-IMPACT%29%3A%20a%20hybridization%20capture-based%20next-generation%20sequencing%20clinical%20assay%20for%20solid%20tumor%20molecular%20oncology&amp;journal=J.%20Mol.%20Diagn.&amp;doi=10.1016%2Fj.jmoldx.2014.12.006&amp;volume=17&amp;pages=251-264&amp;publication_year=2015&amp;author=Cheng%2CDT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">Rosenbaum, E. et al. HLA genotyping in synovial sarcoma: identifying HLA-A*02 and its association with clinical outcome. <i>Clin. Cancer Res.</i> <b>26</b>, 5448–5455 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1158%2F1078-0432.CCR-20-0832" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1158/1078-0432.CCR-20-0832" data-track-item_id="10.1158/1078-0432.CCR-20-0832">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFCku7vM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32816945" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7572879" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=HLA%20genotyping%20in%20synovial%20sarcoma%3A%20identifying%20HLA-A%2A02%20and%20its%20association%20with%20clinical%20outcome&amp;journal=Clin.%20Cancer%20Res.&amp;doi=10.1158%2F1078-0432.CCR-20-0832&amp;volume=26&amp;pages=5448-5455&amp;publication_year=2020&amp;author=Rosenbaum%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">Eubank, M. H. et al. Automated eligibility screening and monitoring for genotype-driven precision oncology trials. <i>J. Am. Med. Inform. Assoc.</i> <b>23</b>, 777–781 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fjamia%2Focw020" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/jamia/ocw020" data-track-item_id="10.1093/jamia/ocw020">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27016727" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6370254" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Automated%20eligibility%20screening%20and%20monitoring%20for%20genotype-driven%20precision%20oncology%20trials&amp;journal=J.%20Am.%20Med.%20Inform.%20Assoc.&amp;doi=10.1093%2Fjamia%2Focw020&amp;volume=23&amp;pages=777-781&amp;publication_year=2016&amp;author=Eubank%2CMH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR46">Rosenthal, R. et al. Neoantigen-directed immune escape in lung cancer evolution. <i>Nature</i> <b>567</b>, 479–485 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-019-1032-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-019-1032-7" data-track-item_id="10.1038/s41586-019-1032-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmslKntrY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30894752" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6954100" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Neoantigen-directed%20immune%20escape%20in%20lung%20cancer%20evolution&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1032-7&amp;volume=567&amp;pages=479-485&amp;publication_year=2019&amp;author=Rosenthal%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR47">Chandran, S. S. &amp; Klebanoff, C. A. T cell receptor-based cancer immunotherapy: emerging efficacy and pathways of resistance. <i>Immunol. Rev.</i> <b>290</b>, 127–147 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2Fimr.12772" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/imr.12772" data-track-item_id="10.1111/imr.12772">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVChsb%2FO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31355495" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7027847" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=T%20cell%20receptor-based%20cancer%20immunotherapy%3A%20emerging%20efficacy%20and%20pathways%20of%20resistance&amp;journal=Immunol.%20Rev.&amp;doi=10.1111%2Fimr.12772&amp;volume=290&amp;pages=127-147&amp;publication_year=2019&amp;author=Chandran%2CSS&amp;author=Klebanoff%2CCA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR48">McGranahan, N. et al. Allele-specific HLA loss and immune escape in lung cancer evolution. <i>Cell</i> <b>171</b>, 1259–1271 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2017.10.001" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2017.10.001" data-track-item_id="10.1016/j.cell.2017.10.001">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslehs7nL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29107330" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5720478" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Allele-specific%20HLA%20loss%20and%20immune%20escape%20in%20lung%20cancer%20evolution&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.10.001&amp;volume=171&amp;pages=1259-1271&amp;publication_year=2017&amp;author=McGranahan%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR49">Montesion, M. et al. Somatic HLA class I loss is a widespread mechanism of immune evasion which refines the use of tumor mutational burden as a biomarker of checkpoint inhibitor response. <i>Cancer Discov.</i> <b>11</b>, 282–292 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1158%2F2159-8290.CD-20-0672" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1158/2159-8290.CD-20-0672" data-track-item_id="10.1158/2159-8290.CD-20-0672">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXosV2hsrY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33127846" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Somatic%20HLA%20class%20I%20loss%20is%20a%20widespread%20mechanism%20of%20immune%20evasion%20which%20refines%20the%20use%20of%20tumor%20mutational%20burden%20as%20a%20biomarker%20of%20checkpoint%20inhibitor%20response&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-20-0672&amp;volume=11&amp;pages=282-292&amp;publication_year=2021&amp;author=Montesion%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR50">Zaretsky, J. M. et al. Mutations associated with acquired resistance to PD-1 blockade in melanoma. <i>N. Engl. J. Med.</i> <b>375</b>, 819–829 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa1604958" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa1604958" data-track-item_id="10.1056/NEJMoa1604958">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvF2ks7%2FP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27433843" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5007206" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Mutations%20associated%20with%20acquired%20resistance%20to%20PD-1%20blockade%20in%20melanoma&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1604958&amp;volume=375&amp;pages=819-829&amp;publication_year=2016&amp;author=Zaretsky%2CJM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR51">Shin, D. S. et al. Primary resistance to PD-1 blockade mediated by <i>JAK1/2</i> mutations. <i>Cancer Discov.</i> <b>7</b>, 188–201 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1158%2F2159-8290.CD-16-1223" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1158/2159-8290.CD-16-1223" data-track-item_id="10.1158/2159-8290.CD-16-1223">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVCjsr0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27903500" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Primary%20resistance%20to%20PD-1%20blockade%20mediated%20by%20JAK1%2F2%20mutations&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-16-1223&amp;volume=7&amp;pages=188-201&amp;publication_year=2017&amp;author=Shin%2CDS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR52">Wolfl, M. &amp; Greenberg, P. D. Antigen-specific activation and cytokine-facilitated expansion of naive, human CD8<sup>+</sup> T cells. <i>Nat. Protoc.</i> <b>9</b>, 950–966 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnprot.2014.064" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nprot.2014.064" data-track-item_id="10.1038/nprot.2014.064">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24675735" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4312138" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXos1Oisr4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Antigen-specific%20activation%20and%20cytokine-facilitated%20expansion%20of%20naive%2C%20human%20CD8%2B%20T%20cells&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fnprot.2014.064&amp;volume=9&amp;pages=950-966&amp;publication_year=2014&amp;author=Wolfl%2CM&amp;author=Greenberg%2CPD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR53">Gee, M. H. et al. Antigen identification for orphan T cell receptors expressed on tumor-infiltrating lymphocytes. <i>Cell</i> <b>172</b>, 549–563 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2017.11.043" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2017.11.043" data-track-item_id="10.1016/j.cell.2017.11.043">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVeht7bI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29275860" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Antigen%20identification%20for%20orphan%20T%20cell%20receptors%20expressed%20on%20tumor-infiltrating%20lymphocytes&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.11.043&amp;volume=172&amp;pages=549-563&amp;publication_year=2018&amp;author=Gee%2CMH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR54">Ali, M. et al. Induction of neoantigen-reactive T cells from healthy donors. <i>Nat. Protoc.</i> <b>14</b>, 1926–1943 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41596-019-0170-6" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41596-019-0170-6" data-track-item_id="10.1038/s41596-019-0170-6">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVWqurrP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31101906" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Induction%20of%20neoantigen-reactive%20T%20cells%20from%20healthy%20donors&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fs41596-019-0170-6&amp;volume=14&amp;pages=1926-1943&amp;publication_year=2019&amp;author=Ali%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR55">Peng, S. et al. Sensitive detection and analysis of neoantigen-specific T cell populations from tumors and blood. <i>Cell Rep.</i> <b>28</b>, 2728–2738 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.celrep.2019.07.106" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.celrep.2019.07.106" data-track-item_id="10.1016/j.celrep.2019.07.106">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhslaqsLjK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31484081" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6774618" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Sensitive%20detection%20and%20analysis%20of%20neoantigen-specific%20T%20cell%20populations%20from%20tumors%20and%20blood&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2019.07.106&amp;volume=28&amp;pages=2728-2738&amp;publication_year=2019&amp;author=Peng%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR56">Arnaud, M. et al. Sensitive identification of neoantigens and cognate TCRs in human solid tumors. <i>Nat. Biotechnol.</i> <a href="https://doi.org/10.1038/s41587-021-01072-6" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41587-021-01072-6">https://doi.org/10.1038/s41587-021-01072-6</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR57">Schmidt, J. et al. Prediction of neo-epitope immunogenicity reveals TCR recognition determinants and provides insight into immunoediting. <i>Cell Rep. Med.</i> <b>2</b>, 100194 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.xcrm.2021.100194" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.xcrm.2021.100194" data-track-item_id="10.1016/j.xcrm.2021.100194">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33665637" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7897774" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20neo-epitope%20immunogenicity%20reveals%20TCR%20recognition%20determinants%20and%20provides%20insight%20into%20immunoediting&amp;journal=Cell%20Rep.%20Med.&amp;doi=10.1016%2Fj.xcrm.2021.100194&amp;volume=2&amp;publication_year=2021&amp;author=Schmidt%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR58">Paulson, K. G. et al. Acquired cancer resistance to combination immunotherapy from transcriptional loss of class I HLA. <i>Nat. Commun.</i> <b>9</b>, 3868 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-018-06300-3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-018-06300-3" data-track-item_id="10.1038/s41467-018-06300-3">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BB3czit12nsQ%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30250229" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6155241" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Acquired%20cancer%20resistance%20to%20combination%20immunotherapy%20from%20transcriptional%20loss%20of%20class%20I%20HLA&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-06300-3&amp;volume=9&amp;publication_year=2018&amp;author=Paulson%2CKG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR59">Drilon, A. et al. Safety and antitumor activity of the multitargeted pan-TRK, ROS1, and ALK inhibitor entrectinib: combined results from two phase I trials (ALKA-372-001 and STARTRK-1). <i>Cancer Discov.</i> <b>7</b>, 400–409 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1158%2F2159-8290.CD-16-1237" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1158/2159-8290.CD-16-1237" data-track-item_id="10.1158/2159-8290.CD-16-1237">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXltlOqsL8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28183697" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5380583" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Safety%20and%20antitumor%20activity%20of%20the%20multitargeted%20pan-TRK%2C%20ROS1%2C%20and%20ALK%20inhibitor%20entrectinib%3A%20combined%20results%20from%20two%20phase%20I%20trials%20%28ALKA-372-001%20and%20STARTRK-1%29&amp;journal=Cancer%20Discov.&amp;doi=10.1158%2F2159-8290.CD-16-1237&amp;volume=7&amp;pages=400-409&amp;publication_year=2017&amp;author=Drilon%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR60">Marabelle, A. et al. Efficacy of pembrolizumab in patients with noncolorectal high microsatellite instability/mismatch repair-deficient cancer: results from the phase II KEYNOTE-158 study. <i>J. Clin. Oncol.</i> <b>38</b>, 1–10 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1200%2FJCO.19.02105" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1200/JCO.19.02105" data-track-item_id="10.1200/JCO.19.02105">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslSktb%2FL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31682550" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Efficacy%20of%20pembrolizumab%20in%20patients%20with%20noncolorectal%20high%20microsatellite%20instability%2Fmismatch%20repair-deficient%20cancer%3A%20results%20from%20the%20phase%20II%20KEYNOTE-158%20study&amp;journal=J.%20Clin.%20Oncol.&amp;doi=10.1200%2FJCO.19.02105&amp;volume=38&amp;pages=1-10&amp;publication_year=2020&amp;author=Marabelle%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR61">Stevanovic, S. et al. Landscape of immunogenic tumor antigens in successful immunotherapy of virally induced epithelial cancer. <i>Science</i> <b>356</b>, 200–205 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aak9510" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aak9510" data-track-item_id="10.1126/science.aak9510">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXlvVems7g%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28408606" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6295311" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Landscape%20of%20immunogenic%20tumor%20antigens%20in%20successful%20immunotherapy%20of%20virally%20induced%20epithelial%20cancer&amp;journal=Science&amp;doi=10.1126%2Fscience.aak9510&amp;volume=356&amp;pages=200-205&amp;publication_year=2017&amp;author=Stevanovic%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR62">Cohen, C. J., Zhao, Y., Zheng, Z., Rosenberg, S. A. &amp; Morgan, R. A. Enhanced antitumor activity of murine-human hybrid T-cell receptor (TCR) in human lymphocytes is associated with improved pairing and TCR/CD3 stability. <i>Cancer Res.</i> <b>66</b>, 8878–8886 (2006).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1158%2F0008-5472.CAN-06-1450" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1158/0008-5472.CAN-06-1450" data-track-item_id="10.1158/0008-5472.CAN-06-1450">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XovF2ht7c%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16951205" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2147082" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Enhanced%20antitumor%20activity%20of%20murine-human%20hybrid%20T-cell%20receptor%20%28TCR%29%20in%20human%20lymphocytes%20is%20associated%20with%20improved%20pairing%20and%20TCR%2FCD3%20stability&amp;journal=Cancer%20Res.&amp;doi=10.1158%2F0008-5472.CAN-06-1450&amp;volume=66&amp;pages=8878-8886&amp;publication_year=2006&amp;author=Cohen%2CCJ&amp;author=Zhao%2CY&amp;author=Zheng%2CZ&amp;author=Rosenberg%2CSA&amp;author=Morgan%2CRA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR63">Klatt, M. G. et al. Solving an MHC allele-specific bias in the reported immunopeptidome. <i>JCI Insight</i> <b>5</b>, e141264 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1172%2Fjci.insight.141264" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1172/jci.insight.141264" data-track-item_id="10.1172/jci.insight.141264">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7566711" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Solving%20an%20MHC%20allele-specific%20bias%20in%20the%20reported%20immunopeptidome&amp;journal=JCI%20Insight&amp;doi=10.1172%2Fjci.insight.141264&amp;volume=5&amp;publication_year=2020&amp;author=Klatt%2CMG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR64">Cameron, B. J. et al. Identification of a Titin-derived HLA-A1-presented peptide as a cross-reactive target for engineered MAGE A3-directed T cells. <i>Sci. Transl. Med.</i> <b>5</b>, 197ra103 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscitranslmed.3006034" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/scitranslmed.3006034" data-track-item_id="10.1126/scitranslmed.3006034">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23926201" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6002776" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhsV2ht7%2FK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Identification%20of%20a%20Titin-derived%20HLA-A1-presented%20peptide%20as%20a%20cross-reactive%20target%20for%20engineered%20MAGE%20A3-directed%20T%20cells&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.3006034&amp;volume=5&amp;publication_year=2013&amp;author=Cameron%2CBJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR65">Pizzolla, A. et al. Influenza-specific lung-resident memory T cells are proliferative and polyfunctional and maintain diverse TCR profiles. <i>J. Clin. Invest.</i> <b>128</b>, 721–733 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1172%2FJCI96957" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1172/JCI96957" data-track-item_id="10.1172/JCI96957">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29309047" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5785253" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Influenza-specific%20lung-resident%20memory%20T%20cells%20are%20proliferative%20and%20polyfunctional%20and%20maintain%20diverse%20TCR%20profiles&amp;journal=J.%20Clin.%20Invest.&amp;doi=10.1172%2FJCI96957&amp;volume=128&amp;pages=721-733&amp;publication_year=2018&amp;author=Pizzolla%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR66">Davis-Harrison, R. L., Armstrong, K. M. &amp; Baker, B. M. Two different T cell receptors use different thermodynamic strategies to recognize the same peptide/MHC ligand. <i>J. Mol. Biol.</i> <b>346</b>, 533–550 (2005).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.jmb.2004.11.063" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.jmb.2004.11.063" data-track-item_id="10.1016/j.jmb.2004.11.063">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2MXntVShtQ%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15670602" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Two%20different%20T%20cell%20receptors%20use%20different%20thermodynamic%20strategies%20to%20recognize%20the%20same%20peptide%2FMHC%20ligand&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2Fj.jmb.2004.11.063&amp;volume=346&amp;pages=533-550&amp;publication_year=2005&amp;author=Davis-Harrison%2CRL&amp;author=Armstrong%2CKM&amp;author=Baker%2CBM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR67">Singh, N. K. et al. An engineered T cell receptor variant realizes the limits of functional binding modes. <i>Biochemistry</i> <b>59</b>, 4163–4175 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1021%2Facs.biochem.0c00689" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1021/acs.biochem.0c00689" data-track-item_id="10.1021/acs.biochem.0c00689">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVykurbP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33074657" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20engineered%20T%20cell%20receptor%20variant%20realizes%20the%20limits%20of%20functional%20binding%20modes&amp;journal=Biochemistry&amp;doi=10.1021%2Facs.biochem.0c00689&amp;volume=59&amp;pages=4163-4175&amp;publication_year=2020&amp;author=Singh%2CNK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR68">Hellman, L. M. et al. Differential scanning fluorimetry based assessments of the thermal and kinetic stability of peptide-MHC complexes. <i>J. Immunol. Methods</i> <b>432</b>, 95–101 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.jim.2016.02.016" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.jim.2016.02.016" data-track-item_id="10.1016/j.jim.2016.02.016">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XjtFOls70%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26906089" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4837003" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Differential%20scanning%20fluorimetry%20based%20assessments%20of%20the%20thermal%20and%20kinetic%20stability%20of%20peptide-MHC%20complexes&amp;journal=J.%20Immunol.%20Methods&amp;doi=10.1016%2Fj.jim.2016.02.016&amp;volume=432&amp;pages=95-101&amp;publication_year=2016&amp;author=Hellman%2CLM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR69">Baxter, T. K. et al. Strategic mutations in the class I major histocompatibility complex HLA-A2 independently affect both peptide binding and T cell receptor recognition. <i>J. Biol. Chem.</i> <b>279</b>, 29175–29184 (2004).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1074%2Fjbc.M403372200" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1074/jbc.M403372200" data-track-item_id="10.1074/jbc.M403372200">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2cXlsVCmsrw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15131131" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Strategic%20mutations%20in%20the%20class%20I%20major%20histocompatibility%20complex%20HLA-A2%20independently%20affect%20both%20peptide%20binding%20and%20T%20cell%20receptor%20recognition&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M403372200&amp;volume=279&amp;pages=29175-29184&amp;publication_year=2004&amp;author=Baxter%2CTK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR70">McMahon, R. M. et al. Structure of HLA-A*0301 in complex with a peptide of proteolipid protein: insights into the role of HLA-A alleles in susceptibility to multiple sclerosis. <i>Acta Crystallogr. D. Biol. Crystallogr.</i> <b>67</b>, 447–454 (2011).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1107%2FS0907444911007888" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1107/S0907444911007888" data-track-item_id="10.1107/S0907444911007888">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXlvFCrsL0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21543847" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3087623" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Structure%20of%20HLA-A%2A0301%20in%20complex%20with%20a%20peptide%20of%20proteolipid%20protein%3A%20insights%20into%20the%20role%20of%20HLA-A%20alleles%20in%20susceptibility%20to%20multiple%20sclerosis&amp;journal=Acta%20Crystallogr.%20D.%20Biol.%20Crystallogr.&amp;doi=10.1107%2FS0907444911007888&amp;volume=67&amp;pages=447-454&amp;publication_year=2011&amp;author=McMahon%2CRM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR71">Liu, Y. C. et al. A molecular basis for the interplay between T cells, viral mutants, and human leukocyte antigen micropolymorphism. <i>J. Biol. Chem.</i> <b>289</b>, 16688–16698 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1074%2Fjbc.M114.563502" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1074/jbc.M114.563502" data-track-item_id="10.1074/jbc.M114.563502">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXps1ymu7o%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24759101" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4059114" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20molecular%20basis%20for%20the%20interplay%20between%20T%20cells%2C%20viral%20mutants%2C%20and%20human%20leukocyte%20antigen%20micropolymorphism&amp;journal=J.%20Biol.%20Chem.&amp;doi=10.1074%2Fjbc.M114.563502&amp;volume=289&amp;pages=16688-16698&amp;publication_year=2014&amp;author=Liu%2CYC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR72">Scott, D. R., Borbulevych, O. Y., Piepenbrink, K. H., Corcelli, S. A. &amp; Baker, B. M. Disparate degrees of hypervariable loop flexibility control T-cell receptor cross-reactivity, specificity, and binding mechanism. <i>J. Mol. Biol.</i> <b>414</b>, 385–400 (2011).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.jmb.2011.10.006" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.jmb.2011.10.006" data-track-item_id="10.1016/j.jmb.2011.10.006">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhsFKltbfP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22019736" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3230710" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Disparate%20degrees%20of%20hypervariable%20loop%20flexibility%20control%20T-cell%20receptor%20cross-reactivity%2C%20specificity%2C%20and%20binding%20mechanism&amp;journal=J.%20Mol.%20Biol.&amp;doi=10.1016%2Fj.jmb.2011.10.006&amp;volume=414&amp;pages=385-400&amp;publication_year=2011&amp;author=Scott%2CDR&amp;author=Borbulevych%2COY&amp;author=Piepenbrink%2CKH&amp;author=Corcelli%2CSA&amp;author=Baker%2CBM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR73">McGranahan, N. et al. Clonal status of actionable driver events and the timing of mutational processes in cancer evolution. <i>Sci. Transl. Med.</i> <b>7</b>, 283ra254 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscitranslmed.aaa1408" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/scitranslmed.aaa1408" data-track-item_id="10.1126/scitranslmed.aaa1408">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20status%20of%20actionable%20driver%20events%20and%20the%20timing%20of%20mutational%20processes%20in%20cancer%20evolution&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.aaa1408&amp;volume=7&amp;publication_year=2015&amp;author=McGranahan%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR74">Shen, R. &amp; Seshan, V. E. FACETS: allele-specific copy number and clonal heterogeneity analysis tool for high-throughput DNA sequencing. <i>Nucleic Acids Res.</i> <b>44</b>, e131 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgkw520" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gkw520" data-track-item_id="10.1093/nar/gkw520">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27270079" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5027494" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhvVKmsrvE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=FACETS%3A%20allele-specific%20copy%20number%20and%20clonal%20heterogeneity%20analysis%20tool%20for%20high-throughput%20DNA%20sequencing&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw520&amp;volume=44&amp;publication_year=2016&amp;author=Shen%2CR&amp;author=Seshan%2CVE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR75">Bielski, C. M. et al. Widespread selection for oncogenic mutant allele imbalance in cancer. <i>Cancer Cell</i> <b>34</b>, 852–862 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ccell.2018.10.003" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ccell.2018.10.003" data-track-item_id="10.1016/j.ccell.2018.10.003">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVOhsrbE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30393068" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6234065" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Widespread%20selection%20for%20oncogenic%20mutant%20allele%20imbalance%20in%20cancer&amp;journal=Cancer%20Cell&amp;doi=10.1016%2Fj.ccell.2018.10.003&amp;volume=34&amp;pages=852-862&amp;publication_year=2018&amp;author=Bielski%2CCM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR76">Shukla, S. A. et al. Comprehensive analysis of cancer-associated somatic mutations in class I HLA genes. <i>Nat. Biotechnol.</i> <b>33</b>, 1152–1158 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnbt.3344" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nbt.3344" data-track-item_id="10.1038/nbt.3344">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFWht7nO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26372948" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4747795" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20analysis%20of%20cancer-associated%20somatic%20mutations%20in%20class%20I%20HLA%20genes&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.3344&amp;volume=33&amp;pages=1152-1158&amp;publication_year=2015&amp;author=Shukla%2CSA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li></ol></div></div>
        </div>
    </div>
</aside>
</div>


    
        
    
    
        
            
        

        
            
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>




<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Medicine
                    </span>
                    (<i itemprop="alternateName">Nat Med</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-170X</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1078-8956</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/" data-track="click" data-track-action="protocols.io" data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/authors/editorial_policies" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nature-research/open-access" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/language-editing/" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/" data-track="click" data-track-action="research solutions" data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library" data-track="click" data-track-action="Recommend to library" data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica" data-track="click" data-track-action="nature africa" data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly" data-track="click" data-track-action="nature Italy" data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies" data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice" data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/accessibility-statement" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20">
    </a>
    <p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"></path></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"></path><clipPath id="b"><use xlink:href="#a" style="overflow:visible"></use></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"></path><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"></path><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"></path><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"></path></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"></path></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif c-site-messages--nature-briefing-cancer" data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Cancer">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Cancer">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing: Cancer" src="/static/images/logos/nature-briefing-logo-cancer-white-bc263e7d57.svg" width="373" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing: Cancer</em> newsletter — what matters in cancer research, free to your inbox weekly.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/cancer" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Cancer">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="CancerBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="CancerBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required="" data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:cancer" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required="">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Cancer">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get what matters in cancer research, free to your inbox weekly.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="/briefing/cancer/?brieferEntryPoint=CancerBriefingBanner">Sign up for Nature Briefing: Cancer
            </a>
        </div>

    </div>

</div>

    




<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41591-022-01786-3&amp;format=js&amp;last_modified=2022-04-28" async=""></script>

<ndl-frjajegbqdml></ndl-frjajegbqdml><div data-cc-ghost="" style="height: 352px;"></div><script type="text/javascript" id="fetch-contextual-ads-campaign-data" charset="">(function(){var g=function(a){a=new CustomEvent("campaignDataLoaded",{detail:a});document.dispatchEvent(a)},b=google_tag_manager["rm"]["50443292"](18);if(b){var k=1500;b=google_tag_manager["rm"]["50443292"](19);b="link"===b?google_tag_manager["rm"]["50443292"](20):google_tag_manager["rm"]["50443292"](21);var m=function(a,c){var h=!1,l=setTimeout(function(){h=!0;c(null)},k),d=new XMLHttpRequest;d.onload=function(){var e=null,f=[];if(200===d.status){try{e=JSON.parse(d.responseText)}catch(p){e={}}f=e.campaigns||[]}h||(clearTimeout(l),c(f.length?f.join(","):null))};d.open("GET",
"/platform/contextual?doi\x3d"+a);d.send()},n=function(a,c){a&&-1===a.indexOf("not set")?m(a,c):setTimeout(c,1)};n(b,function(a){window.campaignsForContextualAds=a;g(a);window.dataLayer.push({content:{article:{campaignID:a}}})})}else window.campaignsForContextualAds="",g(null)})();</script>
<script type="text/javascript" id="gtm-setup-accessdetailsloaded-handler" charset="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail||{};var b={event:"update-access-details"};a.institutional_business_partner_ids&&a.resolved_by||console.log("BPID data could not be retrieved from /exposed-details");a.logged_in_to_sn_profile||console.log("logged_in_to_sn_profile could not be retrieved from /exposed-details");var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?
a.resolved_by.join(";"):"",e=a.personal_business_partner_id_found,f=a.logged_in_to_sn_profile,g=a.snid;b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null,personal:e||null,logged_in_to_sn_profile:f,snid:g||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy={};window.dataLayer.push(b);window.idpUserDataLoaded=!0},!1);window.dataLayer.push({event:"accessdetailsloaded-handler-added"});
window.accessDetailsLoadedHandlerAdded=!0;</script><script type="text/javascript" id="gtm-mediatrust" charset="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="crossmark-script" text="" charset="" type="text/javascript" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="polyfill-matches" charset="">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="fetch-idp-user-data" charset="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},site=google_tag_manager["rm"]["50443292"](59),idpUrl;
switch(site){case "nature":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?"https://staging-idp.nature.com/exposed-details":"https://idp.nature.com/exposed-details";break;case "link":idpUrl=-1<window.location.hostname.indexOf("link-qa")?"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details";break;case "springer":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?
"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details"}if(void 0!==idpUrl){var transport=new XMLHttpRequest;transport.open("GET",idpUrl,!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script src="https://verify.nature.com/verify/nature.min.js"></script><script type="text/javascript" id="create-ad-slots" charset="">function createAndLoadAds(){function h(b){"nature"===a&&google_tag_manager["rm"]["50443292"](87)(b);"bmc"!==a&&"springeropen"!==a||google_tag_manager["rm"]["50443292"](99)(b);("link"===a&&"oscar"===google_tag_manager["rm"]["50443292"](100)||"springer"===a||"link"===a&&!0===google_tag_manager["rm"]["50443292"](101))&&google_tag_manager["rm"]["50443292"](126)(b);"link"===a&&"bunsen"===google_tag_manager["rm"]["50443292"](127)&&google_tag_manager["rm"]["50443292"](152)(b);"link"===a&&"Core"===google_tag_manager["rm"]["50443292"](153)&&google_tag_manager["rm"]["50443292"](165)(b)}window.googletag=window.googletag||{cmd:[]};var a=google_tag_manager["rm"]["50443292"](166),e=google_tag_manager["rm"]["50443292"](168);
if(-1===window.location.search.indexOf("hide_ads\x3dtrue"))if(window.adSlots&&0!==Object.keys(window.adSlots).length)console.log("Ads previously loaded. Will not update ad slots until next page load.");else{window.adSlots||(window.adSlots={});window.getAd=function(b,f){for(var c in window.adSlots)if(-1<c.indexOf(b)){if("object"===f)return adSlots[c];if("slot"===f)return adSlots[c].slot}};googletag.cmd.push(function(){googletag.pubads().setPrivacySettings({limitedAds:e});console.log("limitedAds is ",
e);googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["rm"]["50443292"](170));googletag.pubads().enableSingleRequest();googletag.pubads().disableInitialLoad()});for(var g=document.querySelectorAll("[data-gpt]"),d=0;g[d];++d)h(g[d]);googletag.cmd.push(function(){googletag.enableServices()})}}createAndLoadAds();</script>
</body></html>