<html lang="en" class="js"><head><link rel="preload" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" as="font" type="font/woff2" crossorigin=""><script async="" src="//cdn.jsdelivr.net/npm/mathjax@2.7.5/MathJax.js?config=TeX-AMS-MML_SVG.js"></script><script async="" src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script type="text/javascript" async="" src="https://pagead2.googlesyndication.com/tag/js/gpt.js"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-MRVXSHQ"></script><script type="text/javascript" src="https://www.nature.com/NPTHVeMYFLI104dWuKP_7Hmadu49zuczwaaLqfi7NmEFUcQXHhAL4zMbO8m1bP0ZgbjfX58tKy4UXDi--K8ahA=="></script><script src="https://cmp.nature.com/production_live/en/consent-bundle-8-85.js" onload="initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')"></script>
    <title>Genetic risk factors have a substantial impact on healthy life years | Nature Medicine</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nm.rss">


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin="">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480">

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"epidemiology;genetics;genetics-research;public-health;risk-factors","webtrendsContentCategory":null,"webtrendsContentCollection":"FinnGen","webtrendsContentGroup":"Nature Medicine","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41591-022-01957-2"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Sakari Jukarainen","Tuomo Kiiskinen","Sara Kuitunen","Aki S. Havulinna","Juha Karjalainen","Mattia Cordioli","Joel T. Rämö","Nina Mars","Kaitlin E. Samocha","Hanna M. Ollila","Matti Pirinen","Andrea Ganna"],"publishedAt":1662940800,"publishedAtString":"2022-09-12","title":"Genetic risk factors have a substantial impact on healthy life years","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Epidemiology,Genetics,Genetics research,Public health,Risk factors"},"journal":{"pcode":"nm","title":"nature medicine","volume":"28","issue":"9","id":41591,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":{"id":"ahigjjgihc"}},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"BR","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{font-weight:700;-webkit-font-smoothing:antialiased;font-size:1.5rem;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.app-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{margin:4px 4px 0;fill:#888;height:10px;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-c9121fecab.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png">
<link rel="icon" type="image/png" sizes="48x48" href="/static/images/favicons/nature/favicon-48x48-b52890008c.png">
<link rel="icon" type="image/png" sizes="32x32" href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="/static/images/favicons/nature/favicon-16x16-951651ab72.png">
<link rel="manifest" href="/static/manifest.json" crossorigin="use-credentials">
<link rel="mask-icon" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="/static/images/favicons/nature/favicon.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-e9d2061c95.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-1cbf3a8b56.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script><script id="js-position1" data-test="global-article-js" defer="" src="/static/js/global-article-es6-bundle-c8a573ca90.js"></script><script id="js-position2" data-test="shared-js" defer="" src="/static/js/shared-es6-bundle-e9d2061c95.js"></script><script id="js-position3" data-test="header-150-js" defer="" src="/static/js/header-150-es6-bundle-5bb959eaa1.js"></script><script id="js-position4" data-test="math-js" async="" src="/static/js/math-es6-bundle-23597ae350.js"></script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Genetic risk factors have a substantial impact on healthy life years","description":"The impact of genetic variation on overall disease burden has not been comprehensively evaluated. We introduce an approach to estimate the effect of genetic risk factors on disability-adjusted life years (DALYs; ‘lost healthy life years’). We use genetic information from 735,748 individuals and consider 80 diseases. Rare variants had the highest effect on DALYs at the individual level. Among common variants, \n                rs3798220\n                \n               (LPA) had the strongest individual-level effect, with 1.18 DALYs from carrying 1 versus 0 copies. Being in the top 10% versus the bottom 90% of a polygenic score for multisite chronic pain had an effect of 3.63 DALYs. Some common variants had a population-level effect comparable to modifiable risk factors such as high sodium intake and low physical activity. Attributable DALYs vary between males and females for some genetic exposures. Genetic risk factors can explain a sizable number of healthy life years lost both at the individual and population level. A new analysis combining data from large biobanks and the Global Burden of Disease study estimates that genetic risk factors significantly impact the number of healthy life years lost both at the individual and population level","datePublished":"2022-09-12T00:00:00Z","dateModified":"2022-09-12T00:00:00Z","pageStart":"1893","pageEnd":"1901","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41591-022-01957-2","keywords":["Epidemiology","Genetics","Genetics research","Public health","Risk factors","Biomedicine","general","Cancer Research","Metabolic Diseases","Infectious Diseases","Molecular Medicine","Neurosciences"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig6_HTML.png"],"isPartOf":{"name":"Nature Medicine","issn":["1546-170X","1078-8956"],"volumeNumber":"28","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Sakari Jukarainen","url":"http://orcid.org/0000-0001-8169-8954","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"}],"email":"sakari.jukarainen@helsinki.fi","@type":"Person"},{"name":"Tuomo Kiiskinen","url":"http://orcid.org/0000-0002-6306-8227","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"},{"name":"Finnish Institute for Health and Welfare","address":{"name":"Finnish Institute for Health and Welfare, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"},{"name":"Broad Institute of MIT and Harvard","address":{"name":"Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sara Kuitunen","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"},{"name":"Finnish Institute for Health and Welfare","address":{"name":"Finnish Institute for Health and Welfare, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Aki S. Havulinna","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"},{"name":"Finnish Institute for Health and Welfare","address":{"name":"Finnish Institute for Health and Welfare, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Juha Karjalainen","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"},{"name":"Broad Institute of MIT and Harvard","address":{"name":"Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Mattia Cordioli","url":"http://orcid.org/0000-0002-4872-0520","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Joel T. Rämö","url":"http://orcid.org/0000-0002-6429-5149","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Nina Mars","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Kaitlin E. Samocha","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Center for Genomic Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Hanna M. Ollila","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"},{"name":"Broad Institute of MIT and Harvard","address":{"name":"Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Center for Genomic Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital and Harvard Medical School","address":{"name":"Anesthesia, Critical Care and Pain Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Matti Pirinen","url":"http://orcid.org/0000-0002-1664-1350","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Helsinki","address":{"name":"Department of Public Health, University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Helsinki","address":{"name":"Helsinki Institute for Information Technology HIIT and Department of Mathematics and Statistics, University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Andrea Ganna","url":"http://orcid.org/0000-0002-8147-240X","affiliation":[{"name":"University of Helsinki","address":{"name":"Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"},{"name":"Broad Institute of MIT and Harvard","address":{"name":"Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"andrea.ganna@helsinki.fi","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41591-022-01957-2">
    
    
    <meta name="journal_id" content="41591">
    <meta name="dc.title" content="Genetic risk factors have a substantial impact on healthy life years">
    <meta name="dc.source" content="Nature Medicine 2022 28:9">
    <meta name="dc.format" content="text/html">
    <meta name="dc.publisher" content="Nature Publishing Group">
    <meta name="dc.date" content="2022-09-12">
    <meta name="dc.type" content="OriginalPaper">
    <meta name="dc.language" content="En">
    <meta name="dc.copyright" content="2022 The Author(s)">
    <meta name="dc.rights" content="2022 The Author(s)">
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="dc.description" content="The impact of genetic variation on overall disease burden has not been comprehensively evaluated. We introduce an approach to estimate the effect of genetic risk factors on disability-adjusted life years (DALYs; ‘lost healthy life years’). We use genetic information from 735,748 individuals and consider 80 diseases. Rare variants had the highest effect on DALYs at the individual level. Among common variants, rs3798220 (LPA) had the strongest individual-level effect, with 1.18 DALYs from carrying 1 versus 0 copies. Being in the top 10% versus the bottom 90% of a polygenic score for multisite chronic pain had an effect of 3.63 DALYs. Some common variants had a population-level effect comparable to modifiable risk factors such as high sodium intake and low physical activity. Attributable DALYs vary between males and females for some genetic exposures. Genetic risk factors can explain a sizable number of healthy life years lost both at the individual and population level. A new analysis combining data from large biobanks and the Global Burden of Disease study estimates that genetic risk factors significantly impact the number of healthy life years lost both at the individual and population level">
    <meta name="prism.issn" content="1546-170X">
    <meta name="prism.publicationName" content="Nature Medicine">
    <meta name="prism.publicationDate" content="2022-09-12">
    <meta name="prism.volume" content="28">
    <meta name="prism.number" content="9">
    <meta name="prism.section" content="OriginalPaper">
    <meta name="prism.startingPage" content="1893">
    <meta name="prism.endingPage" content="1901">
    <meta name="prism.copyright" content="2022 The Author(s)">
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="prism.url" content="https://www.nature.com/articles/s41591-022-01957-2">
    <meta name="prism.doi" content="doi:10.1038/s41591-022-01957-2">
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41591-022-01957-2.pdf">
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41591-022-01957-2">
    <meta name="citation_journal_title" content="Nature Medicine">
    <meta name="citation_journal_abbrev" content="Nat Med">
    <meta name="citation_publisher" content="Nature Publishing Group">
    <meta name="citation_issn" content="1546-170X">
    <meta name="citation_title" content="Genetic risk factors have a substantial impact on healthy life years">
    <meta name="citation_volume" content="28">
    <meta name="citation_issue" content="9">
    <meta name="citation_publication_date" content="2022/09">
    <meta name="citation_online_date" content="2022/09/12">
    <meta name="citation_firstpage" content="1893">
    <meta name="citation_lastpage" content="1901">
    <meta name="citation_article_type" content="Article">
    <meta name="citation_fulltext_world_readable" content="">
    <meta name="citation_language" content="en">
    <meta name="dc.identifier" content="doi:10.1038/s41591-022-01957-2">
    <meta name="DOI" content="10.1038/s41591-022-01957-2">
    <meta name="size" content="318801">
    <meta name="citation_doi" content="10.1038/s41591-022-01957-2">
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41591-022-01957-2&amp;api_key=">
    <meta name="description" content="The impact of genetic variation on overall disease burden has not been comprehensively evaluated. We introduce an approach to estimate the effect of genetic risk factors on disability-adjusted life years (DALYs; ‘lost healthy life years’). We use genetic information from 735,748 individuals and consider 80 diseases. Rare variants had the highest effect on DALYs at the individual level. Among common variants, rs3798220 (LPA) had the strongest individual-level effect, with 1.18 DALYs from carrying 1 versus 0 copies. Being in the top 10% versus the bottom 90% of a polygenic score for multisite chronic pain had an effect of 3.63 DALYs. Some common variants had a population-level effect comparable to modifiable risk factors such as high sodium intake and low physical activity. Attributable DALYs vary between males and females for some genetic exposures. Genetic risk factors can explain a sizable number of healthy life years lost both at the individual and population level. A new analysis combining data from large biobanks and the Global Burden of Disease study estimates that genetic risk factors significantly impact the number of healthy life years lost both at the individual and population level">
    <meta name="dc.creator" content="Jukarainen, Sakari">
    <meta name="dc.creator" content="Kiiskinen, Tuomo">
    <meta name="dc.creator" content="Kuitunen, Sara">
    <meta name="dc.creator" content="Havulinna, Aki S.">
    <meta name="dc.creator" content="Karjalainen, Juha">
    <meta name="dc.creator" content="Cordioli, Mattia">
    <meta name="dc.creator" content="Rämö, Joel T.">
    <meta name="dc.creator" content="Mars, Nina">
    <meta name="dc.creator" content="Samocha, Kaitlin E.">
    <meta name="dc.creator" content="Ollila, Hanna M.">
    <meta name="dc.creator" content="Pirinen, Matti">
    <meta name="dc.creator" content="Ganna, Andrea">
    <meta name="dc.subject" content="Epidemiology">
    <meta name="dc.subject" content="Genetics">
    <meta name="dc.subject" content="Genetics research">
    <meta name="dc.subject" content="Public health">
    <meta name="dc.subject" content="Risk factors">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet; citation_title=10 Years of GWAS discovery: biology, function, and translation; citation_author=PM Visscher; citation_volume=101; citation_publication_date=2017; citation_pages=5-22; citation_doi=10.1016/j.ajhg.2017.06.005; citation_id=CR1">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=A global overview of pleiotropy and genetic architecture in complex traits; citation_author=K Watanabe; citation_volume=51; citation_publication_date=2019; citation_pages=1339-1348; citation_doi=10.1038/s41588-019-0481-0; citation_id=CR2">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=The role of polygenic risk and susceptibility genes in breast cancer over the course of life; citation_author=N Mars; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-19966-5; citation_id=CR3">
    <meta name="citation_reference" content="citation_journal_title=CMAJ; citation_title=Absolute 10-year risk of dementia by age, sex and APOE genotype: a population-based cohort study; citation_author=KL Rasmussen, A Tybjærg-Hansen, BG Nordestgaard, R Frikke-Schmidt; citation_volume=190; citation_publication_date=2018; citation_pages=E1033-E1041; citation_doi=10.1503/cmaj.180066; citation_id=CR4">
    <meta name="citation_reference" content="citation_journal_title=Clin. Genet; citation_title=The risk of breast cancer in BRCA1 and BRCA2 mutation carriers without a first-degree relative with breast cancer; citation_author=KA Metcalfe; citation_volume=93; citation_publication_date=2018; citation_pages=1063-1068; citation_doi=10.1111/cge.13191; citation_id=CR5">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Total genetic contribution assessment across the human genome; citation_author=T Li; citation_volume=12; citation_publication_date=2021; citation_doi=10.1038/s41467-021-23124-w; citation_id=CR6">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=Polygenic and clinical risk scores and their impact on age at onset and prediction of cardiometabolic diseases and common cancers; citation_author=N Mars; citation_volume=26; citation_publication_date=2020; citation_pages=549-557; citation_doi=10.1038/s41591-020-0800-0; citation_id=CR7">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=Trans-biobank analysis with 676,000 individuals elucidates the association of polygenic risk scores of complex traits with human lifespan; citation_author=S Sakaue; citation_volume=26; citation_publication_date=2020; citation_pages=542-548; citation_doi=10.1038/s41591-020-0785-8; citation_id=CR8">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.; citation_title=Combined utility of 25 disease and risk factor polygenic risk scores for stratifying risk of all-cause mortality; citation_author=A Meisner; citation_volume=107; citation_publication_date=2020; citation_pages=418-431; citation_doi=10.1016/j.ajhg.2020.07.002; citation_id=CR9">
    <meta name="citation_reference" content="citation_journal_title=eLife; citation_title=Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances; citation_author=PR Timmers; citation_volume=8; citation_publication_date=2019; citation_pages=e39856; citation_doi=10.7554/eLife.39856; citation_id=CR10">
    <meta name="citation_reference" content="citation_journal_title=Lancet; citation_title=Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019; citation_author=CJL Murray; citation_volume=396; citation_publication_date=2020; citation_pages=1223-1249; citation_doi=10.1016/S0140-6736(20)30752-2; citation_id=CR11">
    <meta name="citation_reference" content="citation_journal_title=Genome Med.; citation_title=A guide for the diagnosis of rare and undiagnosed disease: beyond the exome; citation_author=S Marwaha, JW Knowles, EA Ashley; citation_volume=14; citation_publication_date=2022; citation_doi=10.1186/s13073-022-01026-w; citation_id=CR12">
    <meta name="citation_reference" content="citation_journal_title=JAAPA; citation_title=A review of inherited cancer susceptibility syndromes; citation_author=GR Brown; citation_volume=33; citation_publication_date=2020; citation_pages=10-16; citation_doi=10.1097/01.JAA.0000721648.46099.2c; citation_id=CR13">
    <meta name="citation_reference" content="citation_journal_title=Cancers; citation_title=Population-based testing for primary prevention: a systematic review; citation_author=R Manchanda, F Gaba; citation_volume=10; citation_publication_date=2018; citation_pages=E424; citation_doi=10.3390/cancers10110424; citation_id=CR14">
    <meta name="citation_reference" content="Kullo, I. J. et al. Polygenic scores in biomedical research. Nat. Rev. Genet. (2022) 
                  https://doi.org/10.1038/s41576-022-00470-z
                  
                 (2022).">
    <meta name="citation_reference" content="citation_journal_title=PLoS Med.; citation_title=Polygenic risk scores in cardiovascular risk prediction: A cohort study and modelling analyses; citation_author=L Sun; citation_volume=18; citation_publication_date=2021; citation_pages=e1003498; citation_doi=10.1371/journal.pmed.1003498; citation_id=CR16">
    <meta name="citation_reference" content="citation_journal_title=Genome Med.; citation_title=Polygenic risk scores: from research tools to clinical instruments; citation_author=CM Lewis, E Vassos; citation_volume=12; citation_publication_date=2020; citation_doi=10.1186/s13073-020-00742-5; citation_id=CR17">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Improving reporting standards for polygenic scores in risk prediction studies; citation_author=H Wand; citation_volume=591; citation_publication_date=2021; citation_pages=211-219; citation_doi=10.1038/s41586-021-03243-6; citation_id=CR18">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=In vivo CRISPR base editing of PCSK9 durably lowers cholesterol in primates; citation_author=K Musunuru; citation_volume=593; citation_publication_date=2021; citation_pages=429-434; citation_doi=10.1038/s41586-021-03534-y; citation_id=CR19">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=CRISPR-Cas9 gene editing for sickle cell disease and β-thalassemia; citation_author=H Frangoul; citation_volume=384; citation_publication_date=2021; citation_pages=252-260; citation_doi=10.1056/NEJMoa2031054; citation_id=CR20">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=CRISPR-Cas9 in vivo gene editing for transthyretin amyloidosis; citation_author=JH Rim, R Gopalappa, HY Gee; citation_volume=385; citation_publication_date=2021; citation_pages=1722; citation_id=CR21">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Problems with using polygenic scores to select embryos; citation_author=P Turley; citation_volume=385; citation_publication_date=2021; citation_pages=78-86; citation_doi=10.1056/NEJMsr2105065; citation_id=CR22">
    <meta name="citation_reference" content="citation_journal_title=eLife; citation_title=Utility of polygenic embryo screening for disease depends on the selection strategy; citation_author=T Lencz; citation_volume=10; citation_publication_date=2021; citation_pages=e64716; citation_doi=10.7554/eLife.64716; citation_id=CR23">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Screening human embryos for polygenic traits has limited utility; citation_author=E Karavani; citation_volume=179; citation_publication_date=2019; citation_pages=1424-1435; citation_doi=10.1016/j.cell.2019.10.033; citation_id=CR24">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=Whole-genome risk prediction of common diseases in human preimplantation embryos; citation_author=A Kumar; citation_volume=28; citation_publication_date=2022; citation_pages=513-516; citation_doi=10.1038/s41591-022-01735-0; citation_id=CR25">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=Polygenic embryo testing: understated ethics, unclear utility; citation_author=J Johnston, LJ Matthews; citation_volume=28; citation_publication_date=2022; citation_pages=446-448; citation_doi=10.1038/s41591-022-01743-0; citation_id=CR26">
    <meta name="citation_reference" content="citation_journal_title=Lancet; citation_title=Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019; citation_author=T Vos; citation_volume=396; citation_publication_date=2020; citation_pages=1204-1222; citation_doi=10.1016/S0140-6736(20)30925-9; citation_id=CR27">
    <meta name="citation_reference" content="Kurki, M. I. et al. FinnGen: unique genetic insights from combining isolated population and national health register data. Preprint at medRxiv 
                  https://doi.org/10.1101/2022.03.03.22271360
                  
                 (2022).">
    <meta name="citation_reference" content="citation_journal_title=PLoS Med.; citation_title=UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age; citation_author=C Sudlow; citation_volume=12; citation_publication_date=2015; citation_pages=e1001779; citation_doi=10.1371/journal.pmed.1001779; citation_id=CR29">
    <meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. Ser. B Stat. Methodol.; citation_title=A simple new approach to variable selection in regression, with application to genetic fine mapping; citation_author=G Wang, A Sarkar, P Carbonetto, M Stephens; citation_volume=82; citation_publication_date=2020; citation_pages=1273-1300; citation_doi=10.1111/rssb.12388; citation_id=CR30">
    <meta name="citation_reference" content="Kanai, M. et al. Insights from complex trait fine-mapping across diverse populations. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.09.03.21262975
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Germline variation at 8q24 and prostate cancer risk in men of European ancestry; citation_author=M Matejcic; citation_volume=9; citation_publication_date=2018; citation_doi=10.1038/s41467-018-06863-1; citation_id=CR32">
    <meta name="citation_reference" content="citation_journal_title=Biol. Psychiat.; citation_title=The CHRNA5/A3/B4 gene cluster variability as an important determinant of early alcohol and tobacco initiation in young adults; citation_author=IR Schlaepfer; citation_volume=63; citation_publication_date=2008; citation_pages=1039-1046; citation_doi=10.1016/j.biopsych.2007.10.024; citation_id=CR33">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Variants near CHRNA3/5 and APOE have age- and sex-related effects on human lifespan; citation_author=PK Joshi; citation_volume=7; citation_publication_date=2016; citation_doi=10.1038/ncomms11174; citation_id=CR34">
    <meta name="citation_reference" content="citation_journal_title=Genet. Med.; citation_title=ACMG recommendations for reporting of incidental findings in clinical exome and genome sequencing; citation_author=RC Green; citation_volume=15; citation_publication_date=2013; citation_pages=565-574; citation_doi=10.1038/gim.2013.73; citation_id=CR35">
    <meta name="citation_reference" content="citation_journal_title=Genet. Med.; citation_title=ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG); citation_author=DT Miller; citation_volume=23; citation_publication_date=2021; citation_pages=1381-1390; citation_doi=10.1038/s41436-021-01172-3; citation_id=CR36">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=ClinVar: public archive of relationships among sequence variation and human phenotype; citation_author=MJ Landrum; citation_volume=42; citation_publication_date=2014; citation_pages=D980-D985; citation_doi=10.1093/nar/gkt1113; citation_id=CR37">
    <meta name="citation_reference" content="citation_journal_title=Hum. Mutat.; citation_title=ENIGMA–evidence-based network for the interpretation of germline mutant alleles: an international initiative to evaluate risk and clinical significance associated with sequence variation in BRCA1 and BRCA2 genes; citation_author=AB Spurdle; citation_volume=33; citation_publication_date=2012; citation_pages=2-7; citation_doi=10.1002/humu.21628; citation_id=CR38">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Association analyses identify 38 susceptibility loci for inflammatory bowel disease and highlight shared genetic risk across populations; citation_author=JZ Liu; citation_volume=47; citation_publication_date=2015; citation_pages=979-986; citation_doi=10.1038/ng.3359; citation_id=CR39">
    <meta name="citation_reference" content="Johnston, K. J. A. et al. Genome-wide association study of multisite chronic pain in UK Biobank. PLoS Genet. 
                  https://doi.org/10.1371/journal.pgen.1008164
                  
                 (2019).">
    <meta name="citation_reference" content="citation_journal_title=Adipocyte; citation_title=PNPLA3-I148M: a problem of plenty in non-alcoholic fatty liver disease; citation_author=S BasuRay; citation_volume=8; citation_publication_date=2019; citation_pages=201-208; citation_doi=10.1080/21623945.2019.1607423; citation_id=CR41">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Association studies of up to 1.2 million individuals yield new insights into the genetic etiology of tobacco and alcohol use; citation_author=M Liu; citation_volume=51; citation_publication_date=2019; citation_pages=237-244; citation_doi=10.1038/s41588-018-0307-5; citation_id=CR42">
    <meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet.; citation_title=Whole-genome view of the consequences of a population bottleneck using 2926 genome sequences from Finland and United Kingdom; citation_author=H Chheda; citation_volume=25; citation_publication_date=2017; citation_pages=477-484; citation_doi=10.1038/ejhg.2016.205; citation_id=CR43">
    <meta name="citation_reference" content="citation_journal_title=PLoS Genet.; citation_title=Distribution and medical impact of loss-of-function variants in the Finnish founder population; citation_author=ET Lim; citation_volume=10; citation_publication_date=2014; citation_pages=e1004494; citation_doi=10.1371/journal.pgen.1004494; citation_id=CR44">
    <meta name="citation_reference" content="Ruotsalainen, S. E. et al. Loss-of-function of MFGE8 and protection against coronary atherosclerosis. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.06.23.21259381
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=A comprehensive 1,000 Genomes-based genome-wide association meta-analysis of coronary artery disease; citation_author=M Nikpay; citation_volume=47; citation_publication_date=2015; citation_pages=1121-1130; citation_doi=10.1038/ng.3396; citation_id=CR46">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps; citation_author=A Mahajan; citation_volume=50; citation_publication_date=2018; citation_pages=1505-1513; citation_doi=10.1038/s41588-018-0241-6; citation_id=CR47">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals; citation_author=JJ Lee; citation_volume=50; citation_publication_date=2018; citation_pages=1112-1121; citation_doi=10.1038/s41588-018-0147-3; citation_id=CR48">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression; citation_author=NR Wray; citation_volume=50; citation_publication_date=2018; citation_pages=668-681; citation_doi=10.1038/s41588-018-0090-3; citation_id=CR49">
    <meta name="citation_reference" content="citation_journal_title=Lancet; citation_title=Strengthening accountability of the global health metrics enterprise; citation_author=J Shiffman, YR Shawar; citation_volume=395; citation_publication_date=2020; citation_pages=1452-1456; citation_doi=10.1016/S0140-6736(20)30416-5; citation_id=CR50">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Protein-coding repeat polymorphisms strongly shape diverse human phenotypes; citation_author=RE Mukamel; citation_volume=373; citation_publication_date=2021; citation_pages=1499-1505; citation_doi=10.1126/science.abg8289; citation_id=CR51">
    <meta name="citation_reference" content="citation_journal_title=PLoS Genet.; citation_title=Bayesian test for colocalisation between pairs of genetic association studies using summary statistics; citation_author=C Giambartolomei; citation_volume=10; citation_publication_date=2014; citation_pages=e1004383; citation_doi=10.1371/journal.pgen.1004383; citation_id=CR52">
    <meta name="citation_reference" content="citation_journal_title=Sci. Rep.; citation_title=Quantifying genetic heterogeneity between continental populations for human height and body mass index; citation_author=J Guo; citation_volume=11; citation_publication_date=2021; citation_doi=10.1038/s41598-021-84739-z; citation_id=CR53">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=The transferability of lipid loci across African, Asian and European cohorts; citation_author=K Kuchenbaecker; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-12026-7; citation_id=CR54">
    <meta name="citation_reference" content="Huang, Q. Q. et al. Transferability of genetic loci and polygenic scores for cardiometabolic traits in British Pakistani and Bangladeshi individuals. Nat. Commun. 13, 4664 (2022). 
                  https://doi.org/10.1038/s41467-022-32095-5
                  
                ">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Population-specific causal disease effect sizes in functionally important regions impacted by selection; citation_author=H Shi; citation_volume=12; citation_publication_date=2021; citation_pages=1-15; citation_id=CR56">
    <meta name="citation_reference" content="Patel, R. A. et al. Effect sizes of causal variants for gene expression and complex traits differ between populations. Preprint at bioRxiv 
                  https://doi.org/10.1101/2021.12.06.471235
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Clinical use of current polygenic risk scores may exacerbate health disparities; citation_author=AR Martin; citation_volume=51; citation_publication_date=2019; citation_pages=584-591; citation_doi=10.1038/s41588-019-0379-x; citation_id=CR58">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The mutational constraint spectrum quantified from variation in 141,456 humans; citation_author=KJ Karczewski; citation_volume=581; citation_publication_date=2020; citation_pages=434-443; citation_doi=10.1038/s41586-020-2308-7; citation_id=CR59">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=The ensembl variant effect predictor; citation_author=W McLaren; citation_volume=17; citation_publication_date=2016; citation_doi=10.1186/s13059-016-0974-4; citation_id=CR60">
    <meta name="citation_reference" content="citation_journal_title=Clin. Epidemiol.; citation_title=Nordic health registry-based research: a review of health care systems and key registries; citation_author=K Laugesen; citation_volume=13; citation_publication_date=2021; citation_pages=533-554; citation_doi=10.2147/CLEP.S314959; citation_id=CR61">
    <meta name="citation_reference" content="Vuori, M. A. et al. The validity of heart failure diagnoses in the Finnish Hospital Discharge Register. Scand. J. Public Health 
                  https://doi.org/10.1177/1403494819847051
                  
                 (2019).">
    <meta name="citation_reference" content="citation_journal_title=Eur. J. Cancer; citation_title=Quality measures of the population-based finnish cancer registry indicate sound data quality for solid malignant tumours; citation_author=MK Leinonen, J Miettinen, S Heikkinen, J Pitkäniemi, N Malila; citation_volume=77; citation_publication_date=2017; citation_pages=31-39; citation_doi=10.1016/j.ejca.2017.02.017; citation_id=CR63">
    <meta name="citation_reference" content="citation_journal_title=Methods Inf. Med.; citation_title=Comparing properties of audit data and routinely collected register data in case of performance assessment of hip fracture treatment in Finland; citation_author=R Sund; citation_volume=46; citation_publication_date=2007; citation_pages=558-566; citation_doi=10.1160/ME0382; citation_id=CR64">
    <meta name="citation_reference" content="citation_journal_title=Scand. J. Public Health; citation_title=Quality of the Finnish Hospital Discharge Register: a systematic review; citation_author=R Sund; citation_volume=40; citation_publication_date=2012; citation_pages=505-515; citation_doi=10.1177/1403494812456637; citation_id=CR65">
    <meta name="citation_reference" content="Haukka, J. Finnish health and social welfare registers in epidemiological research. Norsk Epidemiologi 14, 113−120 (2004).">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The UK Biobank resource with deep phenotyping and genomic data; citation_author=C Bycroft; citation_volume=562; citation_publication_date=2018; citation_pages=203-209; citation_doi=10.1038/s41586-018-0579-z; citation_id=CR67">
    <meta name="citation_reference" content="Pärn, K., Nunez-Fontarnau, J., Isokallio M. A., Sipilä, T., Kilpeläinen, E., Palotie, A., Ripatti, S. &amp; Palta, P. Genotyping chip data lift-over to reference genome build GRCh38/hg38. protocols.io. 
                  https://doi.org/10.17504/protocols.io.xbhfij6
                  
                 (2019).">
    <meta name="citation_reference" content="Pärn, K., Isokallio, M. A., Nunez-Fontarnau, J., Palotie, A., Ripatti, S. &amp; Palta, P. Genotype imputation workflow v3.0. protocols.io. 
                  https://doi.org/10.17504/protocols.io.xbgfijw
                  
                 (2019).">
    <meta name="citation_reference" content="citation_journal_title=NAR Genomics Bioinforma.; citation_title=Increasing accuracy of HLA imputation by a population-specific reference panel in a FinnGen biobank cohort; citation_author=J Ritari; citation_volume=2; citation_publication_date=2020; citation_pages=lqaa030; citation_doi=10.1093/nargab/lqaa030; citation_id=CR70">
    <meta name="citation_reference" content="citation_journal_title=Pharmacogenomics J.; citation_title=HIBAG—HLA genotype imputation with attribute bagging; citation_author=X Zheng; citation_volume=14; citation_publication_date=2014; citation_pages=192-200; citation_doi=10.1038/tpj.2013.18; citation_id=CR71">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies; citation_author=W Zhou; citation_volume=50; citation_publication_date=2018; citation_pages=1335-1341; citation_doi=10.1038/s41588-018-0184-y; citation_id=CR72">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Efficient Bayesian mixed-model analysis increases association power in large cohorts; citation_author=P-R Loh; citation_volume=47; citation_publication_date=2015; citation_pages=284-290; citation_doi=10.1038/ng.3190; citation_id=CR73">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet; citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses; citation_author=S Purcell; citation_volume=81; citation_publication_date=2007; citation_pages=559-575; citation_doi=10.1086/519795; citation_id=CR74">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Polygenic prediction via Bayesian regression and continuous shrinkage priors; citation_author=T Ge, C-Y Chen, Y Ni, Y-CA Feng, JW Smoller; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-09718-5; citation_id=CR75">
    <meta name="citation_reference" content="citation_journal_title=Biometrics; citation_title=A solution to the problem of monotone likelihood in Cox regression; citation_author=G Heinze, M Schemper; citation_volume=57; citation_publication_date=2001; citation_pages=114-119; citation_doi=10.1111/j.0006-341X.2001.00114.x; citation_id=CR76">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Robust relationship inference in genome-wide association studies; citation_author=A Manichaikul; citation_volume=26; citation_publication_date=2010; citation_pages=2867-2873; citation_doi=10.1093/bioinformatics/btq559; citation_id=CR77">
    <meta name="citation_reference" content="citation_journal_title=JAMA; citation_title=Why test for proportional hazards?; citation_author=MJ Stensrud, MA Hernán; citation_volume=323; citation_publication_date=2020; citation_pages=1401-1402; citation_doi=10.1001/jama.2020.1267; citation_id=CR78">
    <meta name="citation_reference" content="citation_journal_title=Glob. Epidemiol.; citation_title=Population attributable fraction in textbooks: time to revise; citation_author=A Khosravi, M Nazemipour, T Shinozaki, MA Mansournia; citation_volume=3; citation_publication_date=2021; citation_pages=100062; citation_doi=10.1016/j.gloepi.2021.100062; citation_id=CR79">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Prev. Med.; citation_title=Why population attributable fractions can sum to more than one; citation_author=AK Rowe, KE Powell, WD Flanders; citation_volume=26; citation_publication_date=2004; citation_pages=243-249; citation_doi=10.1016/j.amepre.2003.12.007; citation_id=CR80">
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.; citation_title=The contribution of genetic variants to disease depends on the ruler; citation_author=JS Witte, PM Visscher, NR Wray; citation_volume=15; citation_publication_date=2014; citation_pages=765-776; citation_doi=10.1038/nrg3786; citation_id=CR81">
    <meta name="citation_author" content="Jukarainen, Sakari">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author" content="Kiiskinen, Tuomo">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author_institution" content="Finnish Institute for Health and Welfare, Helsinki, Finland">
    <meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Kuitunen, Sara">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author_institution" content="Finnish Institute for Health and Welfare, Helsinki, Finland">
    <meta name="citation_author" content="Havulinna, Aki S.">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author_institution" content="Finnish Institute for Health and Welfare, Helsinki, Finland">
    <meta name="citation_author" content="Karjalainen, Juha">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Cordioli, Mattia">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author" content="Rämö, Joel T.">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author" content="Mars, Nina">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author" content="Samocha, Kaitlin E.">
    <meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Center for Genomic Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Ollila, Hanna M.">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Center for Genomic Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author_institution" content="Anesthesia, Critical Care and Pain Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Pirinen, Matti">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author_institution" content="Department of Public Health, University of Helsinki, Helsinki, Finland">
    <meta name="citation_author_institution" content="Helsinki Institute for Information Technology HIIT and Department of Mathematics and Statistics, University of Helsinki, Helsinki, Finland">
    <meta name="citation_author" content="Ganna, Andrea">
    <meta name="citation_author_institution" content="Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland">
    <meta name="citation_author_institution" content="Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, USA">
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">
    <meta name="twitter:site" content="@naturemedicine">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image:alt" content="Content cover image">
    <meta name="twitter:title" content="Genetic risk factors have a substantial impact on healthy life years">
    <meta name="twitter:description" content="Nature Medicine - A new analysis combining data from large biobanks and the Global Burden of Disease study estimates that genetic risk factors significantly impact the number of healthy life years...">
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig1_HTML.png">
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41591-022-01957-2">
    <meta property="og:type" content="article">
    <meta property="og:site_name" content="Nature">
    <meta property="og:title" content="Genetic risk factors have a substantial impact on healthy life years - Nature Medicine">
    <meta property="og:description" content="A new analysis combining data from large biobanks and the Global Burden of Disease study estimates that genetic risk factors significantly impact the number of healthy life years lost both at the individual and population level">
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig1_HTML.png">
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
<style type="text/css">.cc-banner{background-color:#01324b;border:none!important;bottom:0;box-sizing:border-box;color:#fff!important;left:0;line-height:1.3;margin:auto 0 0;max-width:100%;outline:0;overflow:visible;padding:0;position:fixed;right:0;width:100%;z-index:99999}.cc-banner::backdrop{background-color:#0000004d}.cc-banner *{color:inherit!important}.cc-banner:focus{box-shadow:none;outline:0}.cc-banner a{color:#fff!important;text-decoration:underline}.cc-banner a:active,.cc-banner a:focus,.cc-banner a:hover{color:inherit;text-decoration:none}.cc-banner a:focus{outline:3px solid #08c!important}.cc-banner h2,.cc-banner h3,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-banner .cc-h2,.cc-banner h2{font-size:18px}.cc-banner .cc-h3,.cc-banner h3{font-size:16px}.cc-banner .cc-h4,.cc-banner .cc-h5,.cc-banner .cc-h6,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-size:14px}.cc-banner .cc-button{font-size:16px}.cc-banner__content{background-color:#01324b;display:flex;flex-direction:column;margin:0 auto;max-height:90vh;max-width:100%;padding:16px;position:relative}.cc-banner__content:focus{outline:0}@media (min-width:680px){.cc-banner__content{padding:12px}}@media (min-width:980px){.cc-banner__content{max-height:60vh;padding-bottom:20px;padding-top:20px}}@media (min-width:1320px){.cc-banner__content{padding-bottom:40px;padding-top:40px}}.cc-banner__container{display:flex;flex-direction:column;margin:auto;max-width:1320px;overflow:auto}.cc-banner__title{background:none!important;border:0;flex-shrink:0;font-size:18px!important;font-size:22px!important;font-weight:700!important;letter-spacing:normal;margin:0 0 12px!important;padding:0}@media (min-width:680px){.cc-banner__title{font-size:24px!important;margin:0 0 16px!important}}@media (min-width:1320px){.cc-banner__title{font-size:26px!important;margin:0 0 24px!important}}.cc-banner__body{display:flex;flex-direction:column;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif!important;overflow-x:hidden;overflow-y:auto;padding:3px 3px 16px}@media (min-width:980px){.cc-banner__body{flex-direction:row}}.cc-banner__policy p{font-size:16px!important;margin:0;max-width:none;padding:0}.cc-banner__policy p:not(:last-child){margin:0 0 16px}@media (min-width:980px){.cc-banner__policy p:not(:last-child){margin:0 0 24px}}.cc-banner__policy p a{font-size:16px!important;font-weight:700}.cc-banner__footer{box-shadow:none;flex-shrink:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif !important!important;margin:0;padding:12px 0 3px 3px;position:relative}@media (min-width:380px){.cc-banner__footer{align-items:stretch;display:flex;flex-wrap:wrap}}@media (min-width:680px){.cc-banner__footer{box-shadow:none;flex-wrap:nowrap;width:40%}}.cc-banner__footer .cc-banner__button{flex:1 1 auto;font-weight:700;overflow:hidden;padding:.5em 1em;width:100%}@media (min-width:680px){.cc-banner__footer .cc-banner__button{max-width:275px}}.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:12px}@media (min-width:680px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:0;margin-right:16px}}@media (min-width:980px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-right:24px}}@media (min-width:680px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:16px!important;margin-right:0!important}}@media (min-width:980px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:24px!important;margin-right:0!important}}.cc-banner__button-preferences{padding-left:0;padding-right:0}@media (min-width:380px){.cc-banner__button-preferences{flex:0 0 auto;margin:auto}}@media (min-width:680px){.cc-banner__button-preferences{margin:0}}@media (min-width:380px) and (max-width:680px){.cc-banner__button-break{display:block}}@media (min-width:680px){.cc-banner--is-tcf .cc-banner__footer{width:auto}}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__footer{padding-right:48px;width:66%}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px!important}.cc-banner--is-tcf .cc-banner__stacks{margin:16px 0 0}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__stacks{display:flex;flex:0 0 33%;flex-direction:column;margin:0 0 0 48px;overflow:auto}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px;margin:0 0 12px}.cc-banner--is-tcf .cc-banner__stacks-details{padding:0 3px}.cc-banner--is-tcf .cc-banner .cc-details{border-color:#fff3!important}.cc-box{border-radius:10px;padding:12px}.cc-box--info{background-color:#eff6fb}.cc-box--light{background-color:#faf9f6}.cc-button{border:2px solid #0000;border-radius:32px!important;color:inherit;cursor:pointer;font-size:14px;font-weight:700!important;left:auto;line-height:1.2;margin:0;padding:.5em 1em;right:auto;text-transform:none!important;transition:all .2s}@media (min-width:680px){.cc-button{font-size:14px;padding:.75em 1em}}.cc-button--sm{font-size:12px;padding:2px 8px!important}.cc-button--primary{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--primary:focus{outline:3px solid #08c}.cc-button--primary:focus,.cc-button--primary:hover,.cc-button--secondary{background-color:#fff!important;border-color:#025e8d!important;color:#025e8d!important}.cc-button--secondary:focus{background-color:#025e8d!important;border-color:#fff!important;color:#fff!important;outline:3px solid #08c!important}.cc-button--secondary:hover{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--secondary:active{border:2px solid #01324b!important}.cc-button--secondary.cc-button--contrast{border-color:#fff!important}.cc-button--secondary.cc-button--contrast:hover{background-color:initial!important;color:#fff!important}.cc-button--tertiary{background-color:#f0f7fc!important;background-image:none;border:2px solid #f0f7fc;color:#025e8d!important;text-decoration:underline!important;text-underline-offset:.25em}.cc-button--tertiary:focus{outline:3px solid #08c!important}.cc-button--tertiary:hover{text-decoration-thickness:.25em}.cc-button--tertiary:active{color:#013c5b}.cc-button--link{background-color:initial!important;border-color:#0000!important;border-radius:0!important;color:inherit!important;padding:0!important;text-decoration:underline!important;width:inherit!important}.cc-button--link:focus{outline:3px solid #08c}.cc-button--link:hover{background-color:initial!important;box-shadow:none;text-decoration:none}.cc-button--text{border-radius:0;padding:0}.cc-button--details{padding-right:24px!important;position:relative;width:auto}.cc-button--details:after,.cc-button--details:before{background-color:currentColor;content:"";position:absolute;transition:transform .25s ease-out}.cc-button--details:before{height:10px;margin-top:-5px;right:12px;top:50%;width:2px}.cc-button--details:after{height:2px;margin-top:-1px;right:8px;top:50%;width:10px}.cc-button--details.cc-active:before{transform:rotate(90deg);transform-origin:center}.cc-button--details.cc-active:after{display:none}.cc-details{border-bottom:1px solid #0000001a;padding:12px 0;position:relative;width:100%}.cc-details__summary{align-items:center;display:flex;font-size:14px;font-weight:700;list-style-type:none}.cc-details__summary:focus,.cc-details__summary:focus-visible{outline:3px solid #08c!important;will-change:auto}.cc-details__title{align-items:baseline;display:flex}.cc-details__title h2,.cc-details__title h3,.cc-details__title h4,.cc-details__title h5{line-height:1.4;margin:0!important;padding:0!important}.cc-details__title svg{flex-shrink:0;height:auto;margin-right:8px;position:relative;top:-1px;transition:all .2s;width:auto}.cc-details[open] .cc-details__title svg{top:2px;transform:rotate(90deg)}.cc-details__switch{margin-left:auto}.cc-details__section{padding:16px 0 0 18px}.cc-details__section p{margin:0}.cc-details__section p:not(:last-child){margin-bottom:12px}details summary::-webkit-details-marker{display:none}.cc-radio{align-items:center;display:flex;position:relative}.cc-radio *{cursor:pointer}.cc-radio__input{height:22px;left:0;position:absolute;top:0;width:22px}.cc-radio__input:focus{outline:none}.cc-radio__label{color:inherit;font-size:14px;font-weight:700;line-height:23px;margin:0;padding-left:28px}.cc-radio__label:after,.cc-radio__label:before{background-color:#fff;content:"";display:block;position:absolute;transition:transform .25s ease-out}.cc-radio__label:before{border:1px solid #777;border-radius:50%;height:22px;left:0;top:0;width:22px}.cc-radio__label:after{border:7px solid #025e8d;border-radius:50%;height:0;left:4px;opacity:0;top:4px;width:0}.cc-radio__input:focus+.cc-radio__label:before{box-shadow:0 0 0 2px #08c;outline:none}.cc-radio__label--hidden{display:none}.cc-radio__input:checked+.cc-radio__label:after{opacity:1}.cc-radio__input:disabled{cursor:default}.cc-radio__input:disabled+.cc-radio__label{cursor:default;opacity:.5}.cc-switch *{cursor:pointer}.cc-switch{align-items:center;display:flex;line-height:1}.cc-switch__label{cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;-webkit-user-select:none;user-select:none}.cc-switch__input[type=checkbox]{-webkit-appearance:none;appearance:none;background-color:#fff!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='rgba(0, 0, 0, 0.25)'/%3E%3C/svg%3E")!important;background-position:0!important;background-repeat:no-repeat!important;border:1px solid #777!important;border-radius:16px!important;height:18px!important;margin:0 8px 0 0!important;transition:background-position .2s ease-in-out!important;width:32px!important}.cc-switch__input[type=checkbox]:focus{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%2380b3cc'/%3E%3C/svg%3E")!important}.cc-switch__input[type=checkbox]:checked{background-color:#025e8d!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%23fff'/%3E%3C/svg%3E")!important;background-position:100%!important}.cc-switch__input[type=checkbox]:checked~.cc-switch__label>.cc-switch__text-off,.cc-switch__input[type=checkbox]:not(:checked)~.cc-switch__label>.cc-switch__text-on{display:none}.cc-switch__input[type=checkbox]:hover{cursor:pointer;outline:none}.cc-switch__input[type=checkbox]:focus-visible{cursor:pointer;outline:3px solid #fc0}.cc-switch__input[type=checkbox]:checked:before{background-color:none!important;color:#0000!important}html[dir=rtl] .cc-switch__input[type=checkbox]{margin-left:8px!important;margin-right:0!important}.cc-list>ul,ul.cc-list{list-style-type:disc;margin-left:0;padding-left:0}.cc-list>ul>li,ul.cc-list>li{margin-left:1em}.cc-list>ul>li:not(:last-child),ul.cc-list>li:not(:last-child){margin-bottom:4px}.cc-list__title{margin-bottom:8px!important}.cc-list--inline>ul,ul.cc-list--inline{display:flex;flex-wrap:wrap;list-style-type:none}.cc-list--inline>ul>li,ul.cc-list--inline>li{margin:0 .5em .5em 0}.cc-overlay{background-color:#0000004d;z-index:99998}.cc-overlay,.cc-preferences{bottom:0;left:0;position:fixed;right:0;top:0}.cc-preferences{background-color:#050a14f2!important;border:0;box-sizing:border-box;color:#111;font-family:sans-serif!important;line-height:1.4;margin:auto;max-height:100vh;overflow:auto;padding:0;z-index:100000}.cc-preferences:focus{outline:none}.cc-preferences *,.cc-preferences :after,.cc-preferences :before{box-sizing:inherit!important}.cc-preferences h2,.cc-preferences h3,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-preferences .cc-h2,.cc-preferences h2{font-size:18px}.cc-preferences .cc-h3,.cc-preferences h3{font-size:16px}.cc-preferences .cc-h4,.cc-preferences .cc-h5,.cc-preferences .cc-h6,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-size:14px}.cc-preferences a{color:#025e8d;text-decoration:underline}.cc-preferences a:hover{color:inherit;text-decoration:none}.cc-preferences h3{background:none;color:#111;padding:0;text-transform:none}dialog.cc-preferences{background-color:initial}dialog.cc-preferences::backdrop{background-color:#000000e6}.cc-preferences__dialog{display:flex;flex-direction:column;margin:auto;max-height:100vh;max-width:860px;padding:12px;position:relative}.cc-preferences__dialog>:last-child{border-bottom-left-radius:10px;border-bottom-right-radius:10px}@media (min-width:980px){.cc-preferences__dialog{padding:16px}}.cc-preferences__close{background:#0000!important;border:1px solid #ececec;border-radius:50%;color:#111!important;cursor:pointer;font-family:Times New Roman,serif;font-size:40px;height:40px;left:auto;line-height:1;margin-top:-20px;padding:0!important;position:absolute;right:20px;top:50%;width:40px}.cc-preferences__close:focus{outline:3px solid #08c}.cc-preferences__close-label{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__header{background:#fff!important;border-bottom:1px solid #0000001a;border-top-left-radius:10px;border-top-right-radius:10px;padding:16px;position:relative;text-align:center}.cc-preferences__title{background:none!important;color:#111!important;font-family:sans-serif!important;font-size:18px!important;font-weight:700!important;margin:0!important;padding:0 16px 0 0!important}@media (min-width:480px){.cc-preferences__title{padding-right:0!important}}@media (min-width:980px){.cc-preferences__title{font-size:22px!important}}.cc-preferences__body{background:#fff!important;flex:1 1 auto;min-height:200px;overflow-x:hidden;overflow-y:auto;padding:16px}.cc-preferences__footer{background:#fff!important;border-top:1px solid #d0d0d0;box-shadow:0 0 5px 0 #0003;margin-bottom:0;padding:12px;position:relative}@media (min-width:480px){.cc-preferences__footer{align-items:stretch;display:flex}}.cc-preferences__footer>.cc-button{display:block;width:100%}@media (min-width:480px){.cc-preferences__footer>.cc-button{flex:1 1 auto}}@media (min-width:980px){.cc-preferences__footer>.cc-button{flex-basis:auto}}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:first-child){margin-left:12px}}.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:8px}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:0}}.cc-preferences__categories{list-style:none;margin:0;padding:0}.cc-preferences__category:not(:last-child){border-bottom:1px solid #0000001a;margin-bottom:12px;padding-bottom:12px}.cc-preferences__category-description{font-size:14px;margin:0 0 8px;padding:0}.cc-preferences__category-footer{align-items:center;display:flex;justify-content:space-between}.cc-preferences__status{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__controls{display:flex;margin:0}.cc-preferences__controls>:not(:last-child){margin-right:12px}.cc-preferences__always-on{font-size:14px;font-weight:700;padding-left:26px;position:relative}.cc-preferences__always-on:before{background:#0000!important;border:solid;border-top-color:#0000;border-width:0 0 4px 4px;content:"";display:block;height:10px;left:0;position:absolute;top:2px;transform:rotate(-45deg);width:18px}.cc-preferences__details{background-color:#eff6fb!important;border-radius:10px;font-size:14px;margin:12px 0 0;padding:12px}.cc-preferences__cookie-list,.cc-preferences__provider-list{list-style:none;margin:0;padding:0}.cc-preferences__provider-list{columns:170px}.cc-preferences__cookie-title{font-size:1em;font-style:normal;font-weight:400;margin:0}.cc-preferences__cookie-description{font-size:1em;margin:0 0 8px}.cc-preferences__cookie-domain,.cc-preferences__cookie-lifespan{border-left:1px solid #999;color:#666;margin-left:8px;padding-left:4px}body.cc-has-preferences-open{overflow:hidden;position:relative}.cc-table{border-collapse:collapse;width:100%}.cc-table tbody tr{border-top:1px solid #0000001a}.cc-table td,.cc-table th{font-size:14px;padding:4px 8px;vertical-align:top}.cc-table+.cc-table{margin-top:8px}.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:70%}@media (min-width:680px){.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:80%}}.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{padding-left:8px;width:40%}@media (min-width:680px){.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{width:30%}}.cc-tabs,.cc-tabs>ul{background-color:#faf9f6;display:flex;list-style:none;margin:0;padding:0}.cc-tabs__panel{border-top:1px solid #ececec;margin-top:-1px}.cc-tabs__button{background-color:initial;border:0;border-left:1px solid #0000;border-right:1px solid #0000;color:#111;font-size:16px;font-weight:700;margin:0;padding:15px 16px 12px;position:relative}.cc-tabs__button:before{background-color:initial;content:"";display:block;height:3px;left:0;position:absolute;top:0;width:100%}.cc-tabs__button.cc-active{background-color:#fff;border-color:#025e8d #ececec #ececec;color:#111}.cc-tabs__button.cc-active:before{background-color:#025e8d}.cc-tabs__content{background-color:#fff;display:none;font-size:14px;padding:16px}.cc-tabs__section:not(:first-child){padding-top:24px}.cc-tcf{font-size:14px;margin-top:16px}.cc-tcf__list{list-style:none;margin:0;padding:0}.cc-vendor-count{color:#666;font-size:16px;margin:0 0 8px}.cc-hide{display:none!important}.cc-show{display:block!important}.cc-external-link{background-color:#ececec;background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTcgMGExIDEgMCAxIDEgMCAySDIuNmMtLjM3MSAwLS42LjIwOS0uNi41djE1YzAgLjI5MS4yMjkuNS42LjVoMTQuOGMuMzcxIDAgLjYtLjIwOS42LS41VjEzYTEgMSAwIDAgMSAyIDB2NC41YzAgMS40MzgtMS4xNjIgMi41LTIuNiAyLjVIMi42QzEuMTYyIDIwIDAgMTguOTM4IDAgMTcuNXYtMTVDMCAxLjA2MiAxLjE2MiAwIDIuNiAwem02IDBoNmwuMDc1LjAwMy4xMjYuMDE3LjExMS4wMy4xMTEuMDQ0LjA5OC4wNTIuMDk2LjA2Ny4wOS4wOGExIDEgMCAwIDEgLjA5Ny4xMTJsLjA3MS4xMS4wNTQuMTE0LjAzNS4xMDUuMDMuMTQ4TDIwIDF2NmExIDEgMCAwIDEtMiAwVjMuNDE0bC02LjY5MyA2LjY5M2ExIDEgMCAwIDEtMS40MTQtMS40MTRMMTYuNTg0IDJIMTNhMSAxIDAgMCAxLS45OTMtLjg4M0wxMiAxYTEgMSAwIDAgMSAxLTEiLz48L3N2Zz4=");background-position:right 8px center;background-repeat:no-repeat;background-size:10px auto;border-radius:3px;box-shadow:0 1px 0 #0003;color:#111!important;display:inline-block;font-size:12px;padding:4px 26px 4px 8px;text-decoration:none!important}.cc-external-link:hover{text-decoration:underline!important}.cc-m-0{margin:0!important}.cc-grey{color:#666}</style><script async="" type="text/javascript" src="https://injections.readcube.com/nature/inject.fa744f3f.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>
<body class="article-page"><div id="MathJax_Message" style="display: none;"></div><dialog class="cc-banner" data-cc-banner="" data-nosnippet="" aria-labelledby="cc-banner-label" open="">
		<div class="cc-banner__content" autofocus="" tabindex="-1">
			<div class="cc-banner__container">
				<div class="cc-banner__header">
					<h2 class="cc-banner__title" id="cc-banner-label">Your privacy, your choice</h2>
				</div>
				<div class="cc-banner__body">
					<div class="cc-banner__policy">
						<p>We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.</p><p>By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.</p><p>See our <a href="https://www.nature.com/info/privacy" data-cc-action="privacy">privacy policy</a> for more information on the use of your personal data.</p><p><button type="button" data-cc-action="preferences" class="cc-button cc-button--link cc-button--text">Manage preferences</button> for further information and to change your choices.</p>
					</div>
					
				</div>
				<div class="cc-banner__footer">
					
			<button data-cc-action="accept" class="cc-button cc-button--secondary cc-button--contrast cc-banner__button cc-banner__button-accept">Accept all cookies</button>
			
		
				</div>
			</div>
		</div>
	</dialog>

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90" style="display: none !important;">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-test="top-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41591-022-01957-2;doi=10.1038/s41591-022-01957-2;subjmeta=174,2056,208,308,478,499,631,692,700;kwrd=Epidemiology,Genetics,Genetics+research,Public+health,Risk+factors">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=207706257&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01957-2%26doi%3D10.1038/s41591-022-01957-2%26subjmeta%3D174,2056,208,308,478,499,631,692,700%26kwrd%3DEpidemiology,Genetics,Genetics+research,Public+health,Risk+factors">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=207706257&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01957-2%26doi%3D10.1038/s41591-022-01957-2%26subjmeta%3D174,2056,208,308,478,499,631,692,700%26kwrd%3DEpidemiology,Genetics,Genetics+research,Public+health,Risk+factors"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header="" data-track-component="nature-150-split-header" style="border-color:#e40428">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nm" data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" height="32" alt="Nature Medicine">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
                            </a><div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected="">All journals</option>
                                    <option value="nm">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41591-022-01957-2"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header" hidden="">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-articles" data-track="click" data-track-action="research articles" data-track-label="link" data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-and-analysis" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link" data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/podcast" data-track="click" data-track-action="podcasts" data-track-label="link" data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/current-issue" data-track="click" data-track-action="current issue" data-track-label="link" data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/collections" data-track="click" data-track-action="collections" data-track-label="link" data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://www.facebook.com/Nature-Medicine-193691346949/" data-track="click" data-track-action="facebook" data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://twitter.com/naturemedicine" data-track="click" data-track-action="twitter" data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=5" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/aims" data-track="click" data-track-action="aims &amp; scope" data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-information" data-track="click" data-track-action="journal information" data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-impact" data-track="click" data-track-action="journal metrics" data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editors" data-track="click" data-track-action="about the editors" data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-cross-journal-editorial-team" data-track="click" data-track-action="research cross-journal editorial team" data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-cross-journal-editorial-team" data-track="click" data-track-action="reviews cross-journal editorial team" data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/statistics-advisory-panel" data-track="click" data-track-action="statistical advisory panel" data-track-label="link">
                                    Statistical Advisory Panel
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/our-publishing-models" data-track="click" data-track-action="our publishing models" data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-values-statement" data-track="click" data-track-action="editorial values statement" data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-policies" data-track="click" data-track-action="editorial policies" data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/content" data-track="click" data-track-action="content types" data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="javascript:;" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                        </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/submission-guidelines" data-track="click" data-track-action="submission guidelines" data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-context="header publish with us dropdown menu" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/open-access-funding" data-test="funding-eligibility-link" data-track="click_explore_funding" data-track-context="header publish with us" data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link" href="https://mts-nmed.nature.com/cgi-bin/main.plex" data-track="click_submit_manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-action="submit manuscript" data-track-label="link (publish with us dropdown menu)" data-track-external="" data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link" href="https://journal-alerts.springernature.com/subscribe?journal_id=41591" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-label="link (desktop site header)" data-track-external="">
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature medicine"><span itemprop="name">nature medicine</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm/articles?type=article" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" id="js-enable-context-bar" data-test="context-bar" data-context-bar="" aria-hidden="true">
                    <div class="c-context-bar__container u-container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            Genetic risk factors have a substantial impact on healthy life years
                        </div>
                        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01957-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01957-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2022-09-12">12 September 2022</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Genetic risk factors have a substantial impact on healthy life years</h1>
                    <ul class="c-article-author-list c-article-author-list--short js-no-scroll" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sakari-Jukarainen-Aff1" data-author-popup="auth-Sakari-Jukarainen-Aff1" data-author-search="Jukarainen, Sakari" data-corresp-id="c1" data-track-context="researcher popup with no profile" data-track-index="1_13">Sakari Jukarainen<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-8169-8954"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-8169-8954</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tuomo-Kiiskinen-Aff1-Aff2-Aff3" data-author-popup="auth-Tuomo-Kiiskinen-Aff1-Aff2-Aff3" data-author-search="Kiiskinen, Tuomo" data-track-context="researcher popup with no profile" data-track-index="2_13">Tuomo Kiiskinen</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-6306-8227"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6306-8227</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sara-Kuitunen-Aff1-Aff2" data-author-popup="auth-Sara-Kuitunen-Aff1-Aff2" data-author-search="Kuitunen, Sara" data-track-context="researcher popup with no profile" data-track-index="3_13">Sara Kuitunen</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Aki_S_-Havulinna-Aff1-Aff2" data-author-popup="auth-Aki_S_-Havulinna-Aff1-Aff2" data-author-search="Havulinna, Aki S." data-track-context="researcher popup with no profile" data-track-index="4_13">Aki S. Havulinna</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Juha-Karjalainen-Aff1-Aff3-Aff4" data-author-popup="auth-Juha-Karjalainen-Aff1-Aff3-Aff4" data-author-search="Karjalainen, Juha" data-track-context="researcher popup with no profile" data-track-index="5_13">Juha Karjalainen</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Mattia-Cordioli-Aff1" data-author-popup="auth-Mattia-Cordioli-Aff1" data-author-search="Cordioli, Mattia" data-track-context="researcher popup with no profile" data-track-index="6_13">Mattia Cordioli</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-4872-0520"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-4872-0520</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Joel_T_-R_m_-Aff1" data-author-popup="auth-Joel_T_-R_m_-Aff1" data-author-search="Rämö, Joel T." data-track-context="researcher popup with no profile" data-track-index="7_13">Joel T. Rämö</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-6429-5149"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6429-5149</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Nina-Mars-Aff1" data-author-popup="auth-Nina-Mars-Aff1" data-author-search="Mars, Nina" data-track-context="researcher popup with no profile" data-track-index="8_13">Nina Mars</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-author-popup="group-1" href="#group-1" data-track-context="researcher popup with no profile" data-track-index="9_13">FinnGen</a>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kaitlin_E_-Samocha-Aff3-Aff5" data-author-popup="auth-Kaitlin_E_-Samocha-Aff3-Aff5" data-author-search="Samocha, Kaitlin E." data-track-context="researcher popup with no profile" data-track-index="10_13">Kaitlin E. Samocha</a><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff5" tabindex="-1">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Hanna_M_-Ollila-Aff1-Aff3-Aff5-Aff6" data-author-popup="auth-Hanna_M_-Ollila-Aff1-Aff3-Aff5-Aff6" data-author-search="Ollila, Hanna M." data-track-context="researcher popup with no profile" data-track-index="11_13">Hanna M. Ollila</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff5" tabindex="-1">5</a>,<a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Matti-Pirinen-Aff1-Aff7-Aff8" data-author-popup="auth-Matti-Pirinen-Aff1-Aff7-Aff8" data-author-search="Pirinen, Matti" data-track-context="researcher popup with no profile" data-track-index="12_13">Matti Pirinen</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-1664-1350"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-1664-1350</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff8" tabindex="-1">8</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 13 authors for this article" title="Show all 13 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Andrea-Ganna-Aff1-Aff3-Aff4" data-author-popup="auth-Andrea-Ganna-Aff1-Aff3-Aff4" data-author-search="Ganna, Andrea" data-corresp-id="c2" data-track-context="researcher popup with no profile" data-track-index="13_13">Andrea Ganna<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-8147-240X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8147-240X</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup>&nbsp;</li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nm" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Medicine</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;28</b>,&nbsp;<span class="u-visually-hidden">pages </span>1893–1901 (<span data-test="article-publication-year">2022</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item" data-test="access-count">
                        <p class="c-article-metrics-bar__count">27k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item" data-test="citation-count">
                        <p class="c-article-metrics-bar__count">32 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item" data-test="altmetric-score">
                            <p class="c-article-metrics-bar__count">268 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41591-022-01957-2/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>The impact of genetic variation on overall disease burden has not been comprehensively evaluated. We introduce an approach to estimate the effect of genetic risk factors on disability-adjusted life years (DALYs; ‘lost healthy life years’). We use genetic information from 735,748 individuals and consider 80 diseases. Rare variants had the highest effect on DALYs at the individual level. Among common variants, <a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs3798220">rs3798220</a> (<i>LPA</i>) had the strongest individual-level effect, with 1.18 DALYs from carrying 1 versus 0 copies. Being in the top 10% versus the bottom 90% of a polygenic score for multisite chronic pain had an effect of 3.63 DALYs. Some common variants had a population-level effect comparable to modifiable risk factors such as high sodium intake and low physical activity. Attributable DALYs vary between males and females for some genetic exposures. Genetic risk factors can explain a sizable number of healthy life years lost both at the individual and population level.</p></div></div></section>

            
                
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-022-22637-8/MediaObjects/41598_2022_22637_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41598-022-22637-8?fromPaywallRec=false" data-track="select_recommendations_1" data-track-context="inline recommendations" data-track-action="click recommendations inline - 1" data-track-label="10.1038/s41598-022-22637-8">Polygenic Health Index, General Health, and Pleiotropy: Sibling Analysis and Disease Risk Reduction
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">28 October 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41431-021-00813-0/MediaObjects/41431_2021_813_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41431-021-00813-0?fromPaywallRec=false" data-track="select_recommendations_2" data-track-context="inline recommendations" data-track-action="click recommendations inline - 2" data-track-label="10.1038/s41431-021-00813-0">Polygenic risk modeling with latent trait-related genetic components
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">08 February 2021</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs43587-021-00159-8/MediaObjects/43587_2021_159_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s43587-021-00159-8?fromPaywallRec=false" data-track="select_recommendations_3" data-track-context="inline recommendations" data-track-action="click recommendations inline - 3" data-track-label="10.1038/s43587-021-00159-8">Mendelian randomization of genetically independent aging phenotypes identifies LPA and VCAM1 as biological targets for human aging
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">20 January 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1749523450,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Genome-wide association studies (GWASs) have identified thousands of variants associated with biological traits and diseases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Visscher, P. M. et al. 10 Years of GWAS discovery: biology, function, and translation. Am. J. Hum. Genet. 101, 5–22 (2017)." href="/articles/s41591-022-01957-2#ref-CR1" id="ref-link-section-d72754869e715">1</a></sup>. Overall, these results demonstrate widespread pleiotropy (genetic variants associated with more than one trait)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Watanabe, K. et al. A global overview of pleiotropy and genetic architecture in complex traits. Nat. Genet. 51, 1339–1348 (2019)." href="/articles/s41591-022-01957-2#ref-CR2" id="ref-link-section-d72754869e719">2</a></sup>. Studies commonly quantify the impact of genetic variation on a single disease at a time<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Mars, N. et al. The role of polygenic risk and susceptibility genes in breast cancer over the course of life. Nat. Commun. 11, 6383 (2020)." href="#ref-CR3" id="ref-link-section-d72754869e723">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rasmussen, K. L., Tybjærg-Hansen, A., Nordestgaard, B. G. &amp; Frikke-Schmidt, R. Absolute 10-year risk of dementia by age, sex and APOE genotype: a population-based cohort study. CMAJ 190, E1033–E1041 (2018)." href="#ref-CR4" id="ref-link-section-d72754869e723_1">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Metcalfe, K. A. et al. The risk of breast cancer in BRCA1 and BRCA2 mutation carriers without a first-degree relative with breast cancer. Clin. Genet. 93, 1063–1068 (2018)." href="/articles/s41591-022-01957-2#ref-CR5" id="ref-link-section-d72754869e726">5</a></sup>, or when considering multiple diseases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Li, T. et al. Total genetic contribution assessment across the human genome. Nat. Commun. 12, 2845 (2021)." href="/articles/s41591-022-01957-2#ref-CR6" id="ref-link-section-d72754869e730">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Mars, N. et al. Polygenic and clinical risk scores and their impact on age at onset and prediction of cardiometabolic diseases and common cancers. Nat. Med. 26, 549–557 (2020)." href="/articles/s41591-022-01957-2#ref-CR7" id="ref-link-section-d72754869e733">7</a></sup>, do not use a single metric that can capture overall disease burden, apart from studies on lifespan<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sakaue, S. et al. Trans-biobank analysis with 676,000 individuals elucidates the association of polygenic risk scores of complex traits with human lifespan. Nat. Med. 26, 542–548 (2020)." href="#ref-CR8" id="ref-link-section-d72754869e737">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Meisner, A. et al. Combined utility of 25 disease and risk factor polygenic risk scores for stratifying risk of all-cause mortality. Am. J. Hum. Genet. 107, 418–431 (2020)." href="#ref-CR9" id="ref-link-section-d72754869e737_1">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Timmers, P. R. et al. Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances. eLife 8, e39856 (2019)." href="/articles/s41591-022-01957-2#ref-CR10" id="ref-link-section-d72754869e740">10</a></sup>. It is therefore challenging to assess the impact of genetic variation on overall health and to compare the total impact of different variants.</p><p>Past efforts in comparative risk assessment involve quantifying the effects of modifiable exposures (for example, sodium intake) on health outcomes to inform public health measures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e747">11</a></sup>. This type of assessment has not been systematically performed for genetic risk factors. The utility of genetic information for rare disease diagnosis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Marwaha, S., Knowles, J. W. &amp; Ashley, E. A. A guide for the diagnosis of rare and undiagnosed disease: beyond the exome. Genome Med. 14, 23 (2022)." href="/articles/s41591-022-01957-2#ref-CR12" id="ref-link-section-d72754869e751">12</a></sup> and inherited cancer syndromes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Brown, G. R. et al. A review of inherited cancer susceptibility syndromes. JAAPA 33, 10–16 (2020)." href="/articles/s41591-022-01957-2#ref-CR13" id="ref-link-section-d72754869e755">13</a></sup> is well established. In contrast, despite major advances and ongoing research investments, the viability, utility and cost-effectiveness of other applications such as genetic screening for common diseases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Manchanda, R. &amp; Gaba, F. Population-based testing for primary prevention: a systematic review. Cancers 10, E424 (2018)." href="/articles/s41591-022-01957-2#ref-CR14" id="ref-link-section-d72754869e759">14</a></sup>, risk stratification via polygenic scores (PGSs)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kullo, I. J. et al. Polygenic scores in biomedical research. Nat. Rev. Genet. (2022) 
                  https://doi.org/10.1038/s41576-022-00470-z
                  
                 (2022)." href="#ref-CR15" id="ref-link-section-d72754869e763">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sun, L. et al. Polygenic risk scores in cardiovascular risk prediction: A cohort study and modelling analyses. PLoS Med. 18, e1003498 (2021)." href="#ref-CR16" id="ref-link-section-d72754869e763_1">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lewis, C. M. &amp; Vassos, E. Polygenic risk scores: from research tools to clinical instruments. Genome Med. 12, 44 (2020)." href="#ref-CR17" id="ref-link-section-d72754869e763_2">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Wand, H. et al. Improving reporting standards for polygenic scores in risk prediction studies. Nature 591, 211–219 (2021)." href="/articles/s41591-022-01957-2#ref-CR18" id="ref-link-section-d72754869e766">18</a></sup>, in vivo gene editing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Musunuru, K. et al. In vivo CRISPR base editing of PCSK9 durably lowers cholesterol in primates. Nature 593, 429–434 (2021)." href="#ref-CR19" id="ref-link-section-d72754869e771">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Frangoul, H. et al. CRISPR-Cas9 gene editing for sickle cell disease and β-thalassemia. N. Engl. J. Med. 384, 252–260 (2021)." href="#ref-CR20" id="ref-link-section-d72754869e771_1">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Rim, J. H., Gopalappa, R. &amp; Gee, H. Y. CRISPR-Cas9 in vivo gene editing for transthyretin amyloidosis. N. Engl. J. Med. 385, 1722 (2021)." href="/articles/s41591-022-01957-2#ref-CR21" id="ref-link-section-d72754869e774">21</a></sup> and, most controversially, embryo selection<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Turley, P. et al. Problems with using polygenic scores to select embryos. N. Engl. J. Med. 385, 78–86 (2021)." href="#ref-CR22" id="ref-link-section-d72754869e778">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lencz, T. et al. Utility of polygenic embryo screening for disease depends on the selection strategy. eLife 10, e64716 (2021)." href="#ref-CR23" id="ref-link-section-d72754869e778_1">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Karavani, E. et al. Screening human embryos for polygenic traits has limited utility. Cell 179, 1424–1435 (2019)." href="#ref-CR24" id="ref-link-section-d72754869e778_2">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kumar, A. et al. Whole-genome risk prediction of common diseases in human preimplantation embryos. Nat. Med. 28, 513–516 (2022)." href="#ref-CR25" id="ref-link-section-d72754869e778_3">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Johnston, J. &amp; Matthews, L. J. Polygenic embryo testing: understated ethics, unclear utility. Nat. Med. 28, 446–448 (2022)." href="/articles/s41591-022-01957-2#ref-CR26" id="ref-link-section-d72754869e781">26</a></sup> remain uncertain. A comparative risk assessment framework for genetic risk factors can help to develop and evaluate these attempts to make genetic information actionable.</p><p>One prominent metric for disease burden is the DALY. DALYs represent the loss of healthy life years through worsened quality of life and premature death attributable to a disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e788">27</a></sup>. Combining both quality of life and mortality into a single metric, DALYs are used to monitor disease burden across hundreds of countries in the Global Burden of Disease (GBD) study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e792">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e795">27</a></sup>. GBD estimates the yearly amount of DALYs in each country that are attributable to a list of collectively exhaustive and non-overlapping diseases and injuries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e799">27</a></sup>. DALYs are the sum of years lived with disability (YLDs; ‘lowered quality of life’) and years of life lost (YLLs; ‘premature death’) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig7">1</a>).</p><p>We present an approach for combining genetic association results for 80 diseases from two biobank studies with DALY estimates from the 2019 GBD study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e809">27</a></sup> to provide an overview of the impact of genetic variation on lost healthy life years both at an individual and population level. We rank different genetic risk factors in terms of their health impact and compare genetic risk factors with traditional modifiable risk factors, presenting a template for comparative risk assessment of genetic risk factors.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Estimating attributable DALYs</h3><p>Our method is similar to the GBD approach, which estimates the disease burden attributable to modifiable risk factors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e825">11</a></sup>, except here we consider different classes of genetic risk factors: common variants, rare deleterious variants, human leukocyte antigen (HLA) alleles, <i>APOE</i> haplotypes and PGSs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig1">1</a>). To estimate genetic associations, we used individual-level data from two biobank studies: FinnGen<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Kurki, M. I. et al. FinnGen: unique genetic insights from combining isolated population and national health register data. Preprint at medRxiv 
                  https://doi.org/10.1101/2022.03.03.22271360
                  
                 (2022)." href="/articles/s41591-022-01957-2#ref-CR28" id="ref-link-section-d72754869e835">28</a></sup> (<i>n</i> = 309,136) and UK Biobank<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015)." href="/articles/s41591-022-01957-2#ref-CR29" id="ref-link-section-d72754869e843">29</a></sup> (UKB; <i>n</i> = 426,612) with registry-based follow-up of 48.7 and 22.4 years, respectively (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">1</a>). In total, we considered 80 non-communicable diseases that account for 83.1% of the total DALYs out of all non-communicable diseases in Finland 2019 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e853">27</a></sup>) (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">3</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Study overview."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Study overview.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="426"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>AFp, population-attributable fraction.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>For each genetic exposure–disease pair, we estimated the hazard ratio (HR) using a Cox proportional hazards model. Because a single genetic variant is expected to affect only a minority of the considered 80 diseases, we used a shrinkage approach with a spike-and-slab type prior distribution for the log HRs of each genetic exposure for the 80 diseases. We discarded any HRs where the posterior probability of the null model was above 10%.</p><p>Overall, we estimated the HRs through 92,560 survival analyses: associations of 1,044 common variants, 9 rare variant gene burdens, 74 HLA alleles and 30 PGSs with 80 diseases. After shrinkage, we retained 3,123 HRs for genetic exposure–disease pairs, most of which (67.1%) were genome-wide significant (<i>P</i> &lt; 5 × 10<sup>−8</sup>) and 99% had an association with <i>P</i> &lt; 7.3 × 10<sup>−4</sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig8">2</a>). Using the HR estimates and frequencies of the genetic exposures, we estimated the population-attributable fraction of disease cases for each genetic exposure (proportion of cases prevented if the exposure was removed). We combined attributable fractions with disease-specific population DALYs for Finland 2019 from GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e903">27</a></sup> (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">3</a>) to obtain attributable DALY estimates. Finally, for each genetic exposure, we summed attributable DALYs across the 80 diseases to estimate the total impact. The individual-attributable DALYs, our main measure of interest, can roughly be interpreted as the expected lost healthy life years for an individual attributable to the genetic exposure.</p><h3 class="c-article__sub-heading" id="Sec4">Attributable DALYs for common variants</h3><p>We considered 1,044 independent common variants with minor allele frequency (MAF) over 0.01 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">4</a>). We selected 564 of these based on having at least one <i>P</i> &lt; 5 × 10<sup>−8</sup> association with any of the 80 diseases and having the highest probability of being causal within a sum-of-single-effects (SuSiE) fine-mapped<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Wang, G., Sarkar, A., Carbonetto, P. &amp; Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. J. R. Stat. Soc. Ser. B Stat. Methodol. 82, 1273–1300 (2020)." href="/articles/s41591-022-01957-2#ref-CR30" id="ref-link-section-d72754869e926">30</a></sup> 95% credible set in FinnGen. Additionally, we selected 155 common variants with at least one <i>P</i> &lt; 5 × 10<sup>−12</sup> association with six traditional risk factor traits (body mass index (BMI), glycated hemoglobin (HbA1c), high-density lipoprotein (HDL) cholesterol, low-density lipoprotein (LDL) cholesterol, systolic blood pressure and cigarettes per day) that had the highest probability of being causal within a SuSiE fine-mapped 95% credible set in UKB<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Kanai, M. et al. Insights from complex trait fine-mapping across diverse populations. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.09.03.21262975
                  
                 (2021)." href="/articles/s41591-022-01957-2#ref-CR31" id="ref-link-section-d72754869e936">31</a></sup>. Last, we included 325 coding variants having a <i>P</i> &lt; 5 × 10<sup>−8</sup> association with one of the diseases in FinnGen. Among the 1,044 variants, 34.6% were annotated as missense (<i>n</i> = 335) or putative loss-of-function (pLOF; <i>n</i> = 26). The HRs for common variants were comparable between FinnGen and UKB (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig9">3</a>) and we consequently meta-analyzed the HRs. All estimates are for the comparison of one versus zero copies of the minor allele, so the individual-attributable DALYs correspond to the expected loss of healthy life years if an individual with no copies of the minor allele had instead one copy at birth.</p><p>Overall, carrying one versus zero copies of the common variants resulted in relatively small effects on lost healthy life years (DALYs), with only 56 out of 1,044 (5.4%) variants with over 0.25 attributable DALYs (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">5</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">6</a>). Many of the top variants were in chromosome 6, both inside and outside the HLA region (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig2">2a</a>). We provide attributable DALYs for HLA alleles in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig10">4</a>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Effect of common variants on DALYs."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Effect of common variants on DALYs.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/2" rel="nofollow"><picture><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="520"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Absolute effect on DALYs from carrying one versus zero copies of the minor allele for each common variant. We separately analyzed imputed alleles in the HLA region (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig10">4</a>). <b>b</b>–<b>e</b>, For three common variants and the <i>APOE</i> haplotypes, we reported attributable DALYs and HRs separately for each disease. Estimates are based on 735,748 individuals (<i>n</i> = 309,136 for <b>e</b>). Error bars denote 95% CIs. AMD, age-related macular degeneration.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>The variant with the highest number of attributable DALYs was <a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs3798220">rs3798220</a>, a missense variant in <i>LPA</i>, with 1.18 (95% confidence interval (CI) 1.03–1.32) attributable DALYs from carrying one versus zero copies of the C allele (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig2">2b</a>). The effect was almost exclusively through ischemic heart disease (1.11 DALYs) and to a lesser extent through non-rheumatic valvular heart disease (0.046 DALYs) and lower extremity peripheral artery disease (0.016 DALYs) despite similar relative risk increases. This is because of the larger number of population DALYs attributed to ischemic heart disease by the GBD (60-fold difference to lower extremity peripheral artery disease; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">3</a>), highlighting the relevance of considering absolute measures of disease burden.</p><p>One notable example is <a href="https://www.ncbi.nlm.nih.gov/snp/rs183373024">rs183373024</a>, a noncoding variant near the <i>POU5F1B</i> gene<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Matejcic, M. et al. Germline variation at 8q24 and prostate cancer risk in men of European ancestry. Nat. Commun. 9, 4616 (2018)." href="/articles/s41591-022-01957-2#ref-CR32" id="ref-link-section-d72754869e1043">32</a></sup>, with 0.54 (CI 0.48–0.60) attributable DALYs, mainly due to prostate cancer (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig2">2c</a>). Another example is <a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs8040868">rs8040868</a>, a synonymous variant in the well-known <i>CHRNA5/A3/B4</i> gene cluster associated with nicotine dependence<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Schlaepfer, I. R. et al. The CHRNA5/A3/B4 gene cluster variability as an important determinant of early alcohol and tobacco initiation in young adults. Biol. Psychiat. 63, 1039–1046 (2008)." href="/articles/s41591-022-01957-2#ref-CR33" id="ref-link-section-d72754869e1061">33</a></sup>, with 0.25 (CI 0.23–0.27) attributable DALYs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig2">2d</a>), with effects through lung cancer, chronic obstructive pulmonary disease (COPD), aortic aneurysm, vascular intestinal disorders and lower extremity peripheral artery disease (all consequences of smoking).</p><p>Given the strong associations between <i>APOE</i> alleles and longevity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Joshi, P. K. et al. Variants near CHRNA3/5 and APOE have age- and sex-related effects on human lifespan. Nat. Commun. 7, 11174 (2016)." href="/articles/s41591-022-01957-2#ref-CR34" id="ref-link-section-d72754869e1075">34</a></sup>, we defined the three main <i>APOE</i> alleles determined by <a href="https://www.ncbi.nlm.nih.gov/snp/rs429358">rs429358</a> and <a href="https://www.ncbi.nlm.nih.gov/snp/rs7412">rs7412</a>. Carrying the most deleterious Apo-ε4/ε4 haplotype instead of the most common Apo-ε3/ε3 resulted in 2.48 (CI 2.28–2.68) attributable DALYs, mainly through an increase in the risk of Alzheimer’s disease and other dementias (HR = 5.97, CI 5.57–6.40; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig2">2e</a>). Overall, out of the top 10% common variants with the highest number of attributable DALYs, 49.4% were significantly associated (<i>P</i> &lt; 0.05) with longevity in the largest GWAS on lifespan<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Timmers, P. R. et al. Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances. eLife 8, e39856 (2019)." href="/articles/s41591-022-01957-2#ref-CR10" id="ref-link-section-d72754869e1103">10</a></sup> versus 18% in the bottom 10%.</p><h3 class="c-article__sub-heading" id="Sec5">Attributable DALYs for rare deleterious variants</h3><p>We used whole-exome sequencing data from UKB (<i>n</i> = 174,379) to estimate attributable DALYs for rare deleterious coding variants (MAF &lt; 0.001). The American College of Medical Genetics and Genomics (ACMG) recommends reporting incidental findings in clinical exome and genome sequencing for 73 genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Green, R. C. et al. ACMG recommendations for reporting of incidental findings in clinical exome and genome sequencing. Genet. Med. 15, 565–574 (2013)." href="/articles/s41591-022-01957-2#ref-CR35" id="ref-link-section-d72754869e1118">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Miller, D. T. et al. ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG). Genet. Med. 23, 1381–1390 (2021)." href="/articles/s41591-022-01957-2#ref-CR36" id="ref-link-section-d72754869e1121">36</a></sup>. We estimated the attributable DALYs for two types of burdens for these ACMG genes: (1) putative loss-of-function (pLOF) variant burden and (2) ‘pathogenic’ or ‘likely pathogenic’ Clinvar<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res. 42, D980–D985 (2014)." href="/articles/s41591-022-01957-2#ref-CR37" id="ref-link-section-d72754869e1125">37</a></sup> variant burden (for <i>BRCA1</i>/<i>2</i> we used ‘pathogenic’ ENIGMA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Spurdle, A. B. et al. ENIGMA–evidence-based network for the interpretation of germline mutant alleles: an international initiative to evaluate risk and clinical significance associated with sequence variation in BRCA1 and BRCA2 genes. Hum. Mutat. 33, 2–7 (2012)." href="/articles/s41591-022-01957-2#ref-CR38" id="ref-link-section-d72754869e1136">38</a></sup> variants instead). We report results for nine genes with at least 35 individuals with a positive burden and at least one disease association.</p><p>The five most impactful genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig3">3</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">7</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">9</a>) were <i>LDLR</i> (ischemic heart disease), <i>BRCA2</i> (breast, ovarian, liver and prostate cancer, and COPD), <i>MYBPC3</i> (cardiomyopathy and myocarditis), <i>BRCA1</i> (breast and ovarian cancer) and <i>MLH1</i> (colon and rectum cancer). As an example, individuals carrying one pLOF in <i>BRCA1</i> lose on average 4.08 (CI 2.74–6.32, <i>P</i> = 1.4 × 10<sup>−5</sup>) healthy life years through breast cancer (2.11 DALYs, CI 1.39–3.14; HR = 7.01, CI 4.94–9.94) and ovarian cancer (1.97 DALYs, CI 0.95–3.93; HR = 16.2, CI 8.22–31.8).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="DALYs attributable to carrying a deleterious rare variant in ACMG genes."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: DALYs attributable to carrying a deleterious rare variant in ACMG genes.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/3" rel="nofollow"><picture><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="669"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>The ClinVar<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res. 42, D980–D985 (2014)." href="/articles/s41591-022-01957-2#ref-CR37" id="ref-link-section-d72754869e1189">37</a></sup> burden contains all variants annotated as ‘pathogenic’ or ‘likely pathogenic’. *For <i>BRCA1</i> and <i>BRCA2</i> we only considered variants from ENIGMA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Spurdle, A. B. et al. ENIGMA–evidence-based network for the interpretation of germline mutant alleles: an international initiative to evaluate risk and clinical significance associated with sequence variation in BRCA1 and BRCA2 genes. Hum. Mutat. 33, 2–7 (2012)." href="/articles/s41591-022-01957-2#ref-CR38" id="ref-link-section-d72754869e1199">38</a></sup> annotated as ‘pathogenic’. The loss-of-function burden contains all variants annotated as pLOF with high confidence<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="McLaren, W. et al. The ensembl variant effect predictor. Genome Biol. 17, 122 (2016)." href="/articles/s41591-022-01957-2#ref-CR60" id="ref-link-section-d72754869e1203">60</a></sup>. Estimates are based on 174,379 individuals from UKB. Error bars denote 95% CIs.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec6">Attributable DALYs for polygenic scores</h3><p>We considered 30 PGSs (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">10</a>) covering major diseases, risk factors and psychobehavioral traits in a FinnGen-only analysis. We estimated individual-attributable DALYs as the expected loss of healthy life years if an individual in the bottom 90% of a PGS was instead in the top 10% at birth. Overall, the attributable DALYs varied from 0.07 (inflammatory bowel disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Liu, J. Z. et al. Association analyses identify 38 susceptibility loci for inflammatory bowel disease and highlight shared genetic risk across populations. Nat. Genet. 47, 979–986 (2015)." href="/articles/s41591-022-01957-2#ref-CR39" id="ref-link-section-d72754869e1226">39</a></sup>) to 3.81 (shorter lifespan<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Timmers, P. R. et al. Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances. eLife 8, e39856 (2019)." href="/articles/s41591-022-01957-2#ref-CR10" id="ref-link-section-d72754869e1230">10</a></sup>) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig4">4a</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">11</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">12</a>). Many of the PGSs exhibited substantial pleiotropy, with a median of 16 (interquartile range (IQR) 9–28) PGS-disease associations remaining after shrinkage.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Polygenic score impact on DALYs."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Polygenic score impact on DALYs.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/4" rel="nofollow"><picture><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="629"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, DALYs attributable to belonging in the top 10% versus bottom 90% of each PGS. <b>b</b>,<b>c</b>, Top 25 diseases in terms of attributable DALYs and HRs for two PGSs (bold in <b>a</b>). Estimates are based on 309,136 individuals (FinnGen). Error bars denote 95% CIs. ADHD, attention deficit hyperactivity disorder.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>The shorter lifespan<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Timmers, P. R. et al. Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances. eLife 8, e39856 (2019)." href="/articles/s41591-022-01957-2#ref-CR10" id="ref-link-section-d72754869e1278">10</a></sup> PGS had the highest impact. Individuals in the top 10% of the PGS are expected to lose 3.81 (CI 3.52–4.13) healthy life years compared to an individual in the bottom 90% (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig4">4b</a>). This PGS acts mainly through ischemic heart disease (1.2 DALYs) and to a lesser extent through lung cancer, ischemic stroke, COPD, type 2 diabetes, substance use disorders and low back pain (0.21–0.26 DALYs each). Notably, the PGS for multisite chronic pain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Johnston, K. J. A. et al. Genome-wide association study of multisite chronic pain in UK Biobank. PLoS Genet. 
                  https://doi.org/10.1371/journal.pgen.1008164
                  
                 (2019)." href="/articles/s41591-022-01957-2#ref-CR40" id="ref-link-section-d72754869e1285">40</a></sup> had the second-highest impact at 3.63 (CI 3.33–3.93) DALYs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig4">4c</a>), mainly through low back pain (0.48 DALYs), ischemic heart disease (0.47), substance use disorder (0.37), COPD (0.23), depression (0.22) and neck pain (0.21).</p><p>Following the methodology by Meisner et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Meisner, A. et al. Combined utility of 25 disease and risk factor polygenic risk scores for stratifying risk of all-cause mortality. Am. J. Hum. Genet. 107, 418–431 (2020)." href="/articles/s41591-022-01957-2#ref-CR9" id="ref-link-section-d72754869e1295">9</a></sup> we used the 30 PGSs to predict mortality in a Cox model, then extracted the linear predictors for each individual from that model to form a composite PGS of mortality. Individuals in the top 10% versus bottom 90% of this composite PGS had a higher hazard of death at HR = 1.56 (CI 1.50–1.62) and the highest individual attributable DALYs at 6.52 (CI 6.15–6.88, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig11">5</a>) out of all the PGSs.</p><h3 class="c-article__sub-heading" id="Sec7">Sex-specific effects</h3><p>We repeated some of the analyses stratified by sex (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">5</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">6</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">11</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">14</a>). We observed significant sex differences in total DALYs at <i>P</i> &lt; 0.05 for 474 (45%) of the common variants (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig5">5a</a>). Sex differences in attributable DALYs can result from differences in the effect of the genetic exposure on the disease or differences in DALYs attributed to men and women by the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e1330">27</a></sup>. <a href="https://www.ncbi.nlm.nih.gov/snp/rs738409">rs738409</a> (PNPLA3-I148M), a missense variant in <i>PNPLA3</i> linked to liver fat accumulation and steatohepatitis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="BasuRay, S. PNPLA3-I148M: a problem of plenty in non-alcoholic fatty liver disease. Adipocyte 8, 201–208 (2019)." href="/articles/s41591-022-01957-2#ref-CR41" id="ref-link-section-d72754869e1344">41</a></sup>, provides a clarifying example: carrying one versus zero copies of the minor allele resulted in 0.27 (CI 0.24–0.30) attributable DALYs in males and 0.05 (CI 0.03–0.07) DALYs in females (sex difference <i>P</i> = 1.0 × 10<sup>−34</sup>). The sex difference is in part driven by differences in HRs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig5">5c</a>) for chronic liver disease (HR = 1.32, CI 1.28–1.37 in males versus HR = 1.21, CI 1.17–1.26 in females) and, in part, because DALYs for chronic liver disease are higher in men than women<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e1357">27</a></sup> (431 versus 158 yearly DALYs per 100,000; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">3</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Sex-specific impact of common variants and polygenic scores on DALYs."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Sex-specific impact of common variants and polygenic scores on DALYs.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="618"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Absolute attributable DALYs from carrying one versus zero copies of the minor allele separately for males (<i>n</i> = 331,211) and females (<i>n</i> = 404,537). <b>b</b>, PGSs (top 10% versus the rest) with a significant sex difference in attributable DALYs in FinnGen (males, <i>n</i> = 135,396; females, <i>n</i> = 173,740). <b>c</b>, For <a href="https://www.ncbi.nlm.nih.gov/snp/rs738409">rs738409</a> (<i>PNPLA3</i>), we report the attributable DALYs and HRs for each disease by sex (males, <i>n</i> = 331,211; females, <i>n</i> = 404,537). <b>d</b>, Attributable DALYs and HRs by disease and sex for the PGS predicting drinks per week in FinnGen (males, <i>n</i> = 135,396; females, <i>n</i> = 173,740). Error bars denote 95% CIs.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Eight out of 30 PGSs exhibited significant sex differences in attributable DALYs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig5">5b</a>). Most of the differences were explained by different DALYs attributed to men and women rather than differences in HRs. For example, the PGS for weekly alcohol consumption<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Liu, M. et al. Association studies of up to 1.2 million individuals yield new insights into the genetic etiology of tobacco and alcohol use. Nat. Genet. 51, 237–244 (2019)." href="/articles/s41591-022-01957-2#ref-CR42" id="ref-link-section-d72754869e1437">42</a></sup> had similar HRs between sexes across all diseases (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig5">5d</a>) but markedly different effect on DALYs for substance use disorders reflecting the threefold higher population DALYs for men as estimated by the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e1444">27</a></sup> (1,500 versus 497 yearly DALYs per 100,000; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">3</a>).</p><h3 class="c-article__sub-heading" id="Sec8">Population-attributable DALYs for common variants</h3><p>Next, for the Finnish population, we estimated the amount of population-attributable DALYs per year per 100,000 from all (heterozygous and homozygous) carriers of the minor allele: the expected amount of healthy life years per year per 100,000 individuals in the population that would be gained if the minor allele were completely removed. <a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs7859727">rs7859727</a> (<i>CDKN2B-CDKN2A</i>) had the highest population-attributable DALYs, with minor allele carriers accounting for 447 (CI 420–473) yearly population DALYs per 100,000 in Finland 2019 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig6">6a</a>). The large population effect of this variant is explained by its effect on ischemic heart disease (HR = 1.17, CI 1.16–1.18) and high frequency in the Finnish population (MAF = 0.41). Compared to population DALY estimates for modifiable risk factors from the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e1472">11</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig6">6a</a>), the population-attributable DALY estimates of several common variants are similar to the total impact of a diet high in sodium (300 yearly population DALYs per 100,000), low physical activity (415) and drug use (595), but less impactful than the most important modifiable risk factors such as high systolic blood pressure (3,666), smoking (2,992) and high BMI (2,506)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e1480">11</a></sup>. We also compared the HRs for ischemic heart disease between eight common variants and four modifiable risk factors (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig6">6b</a>). Clinically meaningful changes in modifiable risk factors (for example, 10 mm Hg higher systolic blood pressure) as estimated by the GBD, lead to comparable increases in ischemic heart disease risk (HR = 1.20 to 1.69)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e1487">11</a></sup> as having one versus zero copies of the risk variants (HR = 1.16 to 1.39).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="Effect of common variants on population-level DALYs in the Finnish population."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6: Effect of common variants on population-level DALYs in the Finnish population.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/6" rel="nofollow"><picture><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="303"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>, Bars represent yearly population-level DALYs attributable to top-ten-ranking common variants. The vertical lines represent yearly population-level DALYs attributable to three selected modifiable risk factors as estimated by the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e1505">11</a></sup> for Finland 2019. <b>b</b>, Top-ten HRs for ischemic heart disease of common variants and four HRs of modifiable risk factors as estimated by the GBD (50–54-year age group)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e1512">11</a></sup>. Estimates are based on 735,748 individuals. Error bars denote 95% CIs.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01957-2/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec9">Additional population DALYs attributable to Finnish enrichment</h3><p>Finland is a well-known example of an isolated population where multiple historical bottlenecks<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Chheda, H. et al. Whole-genome view of the consequences of a population bottleneck using 2926 genome sequences from Finland and United Kingdom. Eur. J. Hum. Genet. 25, 477–484 (2017)." href="/articles/s41591-022-01957-2#ref-CR43" id="ref-link-section-d72754869e1532">43</a></sup> have contributed to the enrichment of several functional genetic variants<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Chheda, H. et al. Whole-genome view of the consequences of a population bottleneck using 2926 genome sequences from Finland and United Kingdom. Eur. J. Hum. Genet. 25, 477–484 (2017)." href="/articles/s41591-022-01957-2#ref-CR43" id="ref-link-section-d72754869e1536">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Lim, E. T. et al. Distribution and medical impact of loss-of-function variants in the Finnish founder population. PLoS Genet. 10, e1004494 (2014)." href="/articles/s41591-022-01957-2#ref-CR44" id="ref-link-section-d72754869e1539">44</a></sup> otherwise rare in non-Finnish populations. We estimated the population-attributable DALYs that are due to the enrichment in the Finnish population compared to non-Finnish, non-Swedish, non-Estonian European (NFSEE) populations. The largest impact on population-attributable DALYs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig12">6</a>) was observed for <a href="https://www.ncbi.nlm.nih.gov/snp/rs143473297">rs143473297</a> (<i>TOMM40</i>) contributing to 56.1 (CI 50.1−62.0) yearly population DALYs per 100,000 individuals of Finnish ancestry through increased risk of dementia (0.57 individual-attributable DALYs, HR = 1.95; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">5</a>). One remarkable example of a protective Finnish-enriched variant is <a href="https://www.ncbi.nlm.nih.gov/snp/rs191156695">rs191156695</a>, an inframe insertion in <i>MFGE8</i> (ref<i>.</i> <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Ruotsalainen, S. E. et al. Loss-of-function of MFGE8 and protection against coronary atherosclerosis. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.06.23.21259381
                  
                 (2021)." href="/articles/s41591-022-01957-2#ref-CR45" id="ref-link-section-d72754869e1573">45</a></sup>). The presence of the Finnish-enriched allele in the population contributes to preventing 39.1 (CI 32.0–45.8) yearly population DALYs per 100,000 individuals of Finnish ancestry (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig12">6</a>) solely through decreasing ischemic heart disease risk (HR = 0.80, CI 0.77–0.84).</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec10-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">Discussion</h2><div class="c-article-section__content" id="Sec10-content"><p>As genetic risk factors are becoming increasingly relevant to various fields of medicine, the ability to evaluate their impact on disease burden is crucial. In this study, by combining genetic information with DALYs from the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e1589">27</a></sup>, we present a comparative risk assessment approach<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e1593">11</a></sup> for genetic risk factors, which makes it possible to uniformly compare the impact of genetic exposures through multiple diseases in terms of DALYs (‘lost healthy life years’). Overall, rare deleterious variants tended to have higher effects on DALYs than common variants at the individual level. Genetic exposures increasing the risk of ischemic heart disease tended to be most impactful in terms of DALYs (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">5</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">6</a>) as it accounts for the largest share of population DALYs in the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e1603">27</a></sup> for Finland 2019 (11.5%; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">3</a>).</p><p>The largest effects on individual DALYs were observed for deleterious rare variants in <i>BRCA1</i> (breast and ovarian cancer), <i>BRCA2</i> (breast, ovarian, liver and prostate cancer), <i>MYBPC3</i> (cardiomyopathy and myocarditis), <i>LDLR</i> (ischemic heart disease) and <i>MLH1</i> (colon and rectum cancer); however, due to the rarity of these variants, the population impact was, at most, 21 yearly population DALYs per 100,000 for <i>BRCA2</i> (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">8</a>), which is substantially lower than for the top common variant <a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs7859727">rs7859727</a> (<i>CDKN2B-CDKN2A</i>) where minor allele carriers account for 447 yearly population DALYs per 100,000 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">4</a>).</p><p>Overall, the top PGSs exert their effect through cardiometabolic traits (for example, through ischemic heart disease for shorter lifespan<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Timmers, P. R. et al. Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances. eLife 8, e39856 (2019)." href="/articles/s41591-022-01957-2#ref-CR10" id="ref-link-section-d72754869e1652">10</a></sup>, coronary artery disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Nikpay, M. et al. A comprehensive 1,000 Genomes-based genome-wide association meta-analysis of coronary artery disease. Nat. Genet. 47, 1121–1130 (2015)." href="/articles/s41591-022-01957-2#ref-CR46" id="ref-link-section-d72754869e1656">46</a></sup> and type 2 diabetes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Mahajan, A. et al. Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps. Nat. Genet. 50, 1505–1513 (2018)." href="/articles/s41591-022-01957-2#ref-CR47" id="ref-link-section-d72754869e1660">47</a></sup> PGSs) or pain/addiction-related traits (for example, through low back pain, substance use disorders, lung cancer and COPD for multisite chronic pain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Johnston, K. J. A. et al. Genome-wide association study of multisite chronic pain in UK Biobank. PLoS Genet. 
                  https://doi.org/10.1371/journal.pgen.1008164
                  
                 (2019)." href="/articles/s41591-022-01957-2#ref-CR40" id="ref-link-section-d72754869e1664">40</a></sup>, lower educational attainment<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Lee, J. J. et al. Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals. Nat. Genet. 50, 1112–1121 (2018)." href="/articles/s41591-022-01957-2#ref-CR48" id="ref-link-section-d72754869e1668">48</a></sup>, major depressive disorder<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Wray, N. R. et al. Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression. Nat. Genet. 50, 668–681 (2018)." href="/articles/s41591-022-01957-2#ref-CR49" id="ref-link-section-d72754869e1673">49</a></sup> and smoking initiation<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Liu, M. et al. Association studies of up to 1.2 million individuals yield new insights into the genetic etiology of tobacco and alcohol use. Nat. Genet. 51, 237–244 (2019)." href="/articles/s41591-022-01957-2#ref-CR42" id="ref-link-section-d72754869e1677">42</a></sup> PGSs; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">12</a>). Note that the effect estimates for PGSs depend on the cutoff used. For the shorter lifespan PGS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Timmers, P. R. et al. Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances. eLife 8, e39856 (2019)." href="/articles/s41591-022-01957-2#ref-CR10" id="ref-link-section-d72754869e1684">10</a></sup>, if we instead used top 1% versus the rest or top 50% versus the rest cutoffs, the individual DALYs would have been 5.60 and 2.76, respectively, instead of the reported 3.81 for top 10% versus the rest. Note also that the effects of PGSs depend on their predictive performance, so their relative importance can change and their effects on DALYs will increase as larger GWASs are used to construct them or methodological advances improve PGS performance.</p><p>Most common variants with the largest effects on DALYs affected ischemic heart disease risk (for example, <a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs3798220">rs3798220</a>, <i>LPA</i>; <a href="https://www.ncbi.nlm.nih.gov/snp/rs11591147">rs11591147</a>, <i>PCSK9</i>; and <a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs1537371">rs1537371</a>, <i>CDKN2B-CDKN2A</i>) and some affected risk of dementia (<a href="https://www.ncbi.nlm.nih.gov/snp/rs429358">rs429358</a>, <i>APOE</i>), prostate cancer (<a href="https://www.ncbi.nlm.nih.gov/snp/rs183373024">rs183373024</a>, <i>POU5F1B</i>) or type 2 diabetes (<a href="https://www.ncbi.nlm.nih.gov/snp/rs117361510">rs117361510</a>, <i>JPH2</i>). The number of DALYs for a disease in the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Johnston, J. &amp; Matthews, L. J. Polygenic embryo testing: understated ethics, unclear utility. Nat. Med. 28, 446–448 (2022)." href="/articles/s41591-022-01957-2#ref-CR26" id="ref-link-section-d72754869e1753">26</a></sup> is driven by how common it is, how much it contributes to premature mortality and how much it lowers quality of life. Common diseases that either lead to premature mortality (high YLLs, such as heart disease) and/or long periods of living with disability (high YLDs, such as low back pain) account for a large number of population DALYs (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">3</a>). Consequently, genetic exposures increasing the risk of high-DALY diseases predominate the results. Some variants might affect DALYs by modifying intermediate risk factors such as BMI and blood pressure and, in our analyses, we have included several variants that were associated with six major modifiable risk factors. Nonetheless, the high-ranking variants are associated directly with the diseases rather than with intermediate risk factors (<a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs8040868">rs8040868</a> in <i>CHRNA5/A3/B4</i> being a notable exception due to its impact on smoking behavior).</p><p>There are multiple strengths and limitations to our study. One key strength is that the genetic associations were estimated using individual-level data from two large biobank studies with long registry-based follow-ups. We apply a shrinkage procedure to the associations and thus obtain a conservative overview of the pleiotropic effect of genetic exposures on 80 major diseases accounting for 83.1% of the total DALYs in Finland through all non-communicable diseases in 2019 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e1774">27</a></sup>). In quantifying the disease burden, we combine the association results with population DALYs from the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e1778">27</a></sup>, which reports perhaps the most accurate and unbiased estimates of population-level disease burden. This is important for many diseases for which defining the absolute amount of disease burden relying on available data is biased because of under-ascertainment (for example, migraine) or non-representativeness (for example, schizophrenia; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig13">7</a>). One important aspect of the DALY estimates from the GBD is that the disease definitions are non-overlapping and the DALYs avoid double-counting by allowing a single cause of death when estimating the YLLs and through a comorbidity-correction procedure when estimating YLDs (Supplementary Appendix 1 Section 4.9 of GBD 2019 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e1785">11</a></sup>)). Finally, compared to the GBD risk factors approach<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e1789">11</a></sup>, which estimates attributable DALYs relying on effect estimates of the modifiable risk factors based mostly on (non-genetic) observational analyses, genetic exposures that we examine are less likely to be impacted by confounding and other biases, making a causal interpretation of our estimates more credible.</p><p>In Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">15</a> we provide an overview of the study limitations, some of which can be overcome in future iterations of this work. Here we discuss perhaps the most important ones. First, we take it as given that the DALYs estimated by the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e1799">27</a></sup> are accurate and that DALYs are a valid and meaningful measure of disease burden (which has been debated<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Shiffman, J. &amp; Shawar, Y. R. Strengthening accountability of the global health metrics enterprise. Lancet 395, 1452–1456 (2020)." href="/articles/s41591-022-01957-2#ref-CR50" id="ref-link-section-d72754869e1803">50</a></sup>). Second, the DALYs for individuals with a disease are assumed to be the same among those with and without the genetic exposures (for example, individuals with and without a damaging <i>BRCA1</i> mutation that develop breast cancer accumulate DALYs similarly). Third, we used DALYs estimated for Finland 2019 and estimated genetic associations using data between 1972 and 2020 in Finland and the United Kingdom, so the estimated effects on lifetime individual DALYs for someone born today can change as the disease incidence, medical care and mediating factors (for example, smoking) change. Fourth, although we suggest a causal interpretation for the attributable DALYs, there are important caveats. Despite rigorous statistical fine-mapping, the reported variants with the highest posterior probability might only tag underlying causal genetic variation. For example, the <a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs3798220">rs3798220</a> (<i>LPA</i>) variant is known to tag copy number variation of the Kringle IV-like domain 2 in the <i>LPA</i> locus, which is the probable mechanism behind the association of <a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs3798220">rs3798220</a> with ischemic heart disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Mukamel, R. E. et al. Protein-coding repeat polymorphisms strongly shape diverse human phenotypes. Science 373, 1499–1505 (2021)." href="/articles/s41591-022-01957-2#ref-CR51" id="ref-link-section-d72754869e1831">51</a></sup>, so <a href="https://www.ncbi.nlm.nih.gov/snp/?term=rs3798220">rs3798220</a> is not actually responsible for the effect on DALYs. Another caveat lies in the possibility that a reported variant does not tag one causal variant, but multiple causal variants in linkage disequilibrium (LD); however, using expression quantitative trait loci data, previous work on quantitative lipid traits has shown that a minority of pleiotropic effects at a given locus are explained by this configuration<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Giambartolomei, C. et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. PLoS Genet. 10, e1004383 (2014)." href="/articles/s41591-022-01957-2#ref-CR52" id="ref-link-section-d72754869e1842">52</a></sup>. Fifth, note that we mainly present attributable DALYs for both sexes in aggregate, which might be misleading for exposures with sex-specific effects. For example, <a href="https://www.ncbi.nlm.nih.gov/snp/rs183373024">rs183373024</a> (<i>POU5F1B</i>) affected DALYs only through prostate cancer and benign prostatic hyperplasia (0.54 DALYs in aggregate, 1.06 in males and 0 in females; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">5</a>). Finally, our genetic association results are estimated in participants of Finnish ancestry in FinnGen and European ancestry in UKB, which limits the generalizability to populations of non-European ancestry.</p><p>Although our results are presented using DALY estimates for Finland, limiting generalizability across countries, the presented framework can be applied to other countries and ancestries under certain assumptions. First, the effect sizes need to be similar in the target population. There is increasing evidence that many causal variants have similar effects across continental ancestry groups<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Guo, J. et al. Quantifying genetic heterogeneity between continental populations for human height and body mass index. Sci. Rep. 11, 5240 (2021)." href="#ref-CR53" id="ref-link-section-d72754869e1863">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kuchenbaecker, K. et al. The transferability of lipid loci across African, Asian and European cohorts. Nat. Commun. 10, 4330 (2019)." href="#ref-CR54" id="ref-link-section-d72754869e1863_1">54</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Huang, Q. Q. et al. Transferability of genetic loci and polygenic scores for cardiometabolic traits in British Pakistani and Bangladeshi individuals. Nat. Commun. 13, 4664 (2022). 
                  https://doi.org/10.1038/s41467-022-32095-5
                  
                " href="#ref-CR55" id="ref-link-section-d72754869e1863_2">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Shi, H. et al. Population-specific causal disease effect sizes in functionally important regions impacted by selection. Nat. Commun. 12, 1–15 (2021)." href="#ref-CR56" id="ref-link-section-d72754869e1863_3">56</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Patel, R. A. et al. Effect sizes of causal variants for gene expression and complex traits differ between populations. Preprint at bioRxiv 
                  https://doi.org/10.1101/2021.12.06.471235
                  
                 (2021)." href="/articles/s41591-022-01957-2#ref-CR57" id="ref-link-section-d72754869e1866">57</a></sup>, but this does not apply to PGSs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. Nat. Genet. 51, 584–591 (2019)." href="/articles/s41591-022-01957-2#ref-CR58" id="ref-link-section-d72754869e1870">58</a></sup>. Assuming effect sizes are consistent, a plausible assumption for populations of mostly European ancestry, one needs to know the frequency of the genetic exposures in the target population, which is particularly challenging in countries with a heterogeneous ancestry composition. In the absence of representative genetic surveys, it might be possible to use self-reported ancestry information combined with frequency datasets such as gnomAD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)." href="/articles/s41591-022-01957-2#ref-CR59" id="ref-link-section-d72754869e1874">59</a></sup>. With the effect estimates and frequencies, one can use the GBD estimates for the target country to derive localized estimates.</p><p>In summary, we present an approach to combine genetic association results with disease burden estimates from the GBD and provide an overview of the impact of genetic exposures on DALYs. We show that some common variants account for as many DALYs as some well-established modifiable risk factors and that PGSs are highly predictive of DALYs. For drug development, the attributable DALYs for a variant can provide an initial assessment of the potential impact of drugs targeting the relevant biological pathway, whereas extending the approach to the transcriptome-wide association study framework can provide information on potential drug effects in specific tissues. While genetic risk factors are not yet modifiable in practice, information on genetic risk can be used to target preventive measures and screening. Knowing the relative contribution of different genetic risk factors on disease burden can help prioritize and design interventions using genetic information. Approaches based on in vivo gene editing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Musunuru, K. et al. In vivo CRISPR base editing of PCSK9 durably lowers cholesterol in primates. Nature 593, 429–434 (2021)." href="#ref-CR19" id="ref-link-section-d72754869e1881">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Frangoul, H. et al. CRISPR-Cas9 gene editing for sickle cell disease and β-thalassemia. N. Engl. J. Med. 384, 252–260 (2021)." href="#ref-CR20" id="ref-link-section-d72754869e1881_1">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Rim, J. H., Gopalappa, R. &amp; Gee, H. Y. CRISPR-Cas9 in vivo gene editing for transthyretin amyloidosis. N. Engl. J. Med. 385, 1722 (2021)." href="/articles/s41591-022-01957-2#ref-CR21" id="ref-link-section-d72754869e1884">21</a></sup> could use the information on expected DALYs prevented to prioritize potential targets for clinical trials and evaluate the effects of the intervention across the lifespan. Finally, the DALYs perspective can be useful for studies evaluating the utility of monogenic and polygenic embryo screening<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Turley, P. et al. Problems with using polygenic scores to select embryos. N. Engl. J. Med. 385, 78–86 (2021)." href="#ref-CR22" id="ref-link-section-d72754869e1888">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lencz, T. et al. Utility of polygenic embryo screening for disease depends on the selection strategy. eLife 10, e64716 (2021)." href="#ref-CR23" id="ref-link-section-d72754869e1888_1">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Karavani, E. et al. Screening human embryos for polygenic traits has limited utility. Cell 179, 1424–1435 (2019)." href="#ref-CR24" id="ref-link-section-d72754869e1888_2">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kumar, A. et al. Whole-genome risk prediction of common diseases in human preimplantation embryos. Nat. Med. 28, 513–516 (2022)." href="#ref-CR25" id="ref-link-section-d72754869e1888_3">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Johnston, J. &amp; Matthews, L. J. Polygenic embryo testing: understated ethics, unclear utility. Nat. Med. 28, 446–448 (2022)." href="/articles/s41591-022-01957-2#ref-CR26" id="ref-link-section-d72754869e1891">26</a></sup> in assessing the potential impact and making tradeoffs between genetic risk for different diseases. To conclude, by translating information on genetic risk into expected healthy life years lost, genetic risk factors can be put in the larger context of traditional risk factors and compared in terms of their effect across multiple diseases, which enables the development and implementation of clinical applications utilizing genetic information in a more guided way.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec11-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec11">Methods</h2><div class="c-article-section__content" id="Sec11-content"><h3 class="c-article__sub-heading" id="Sec12">General statistical methods</h3><p>All <i>P</i> values are nominal. Error bars represent nominal 95% CIs. Confidence intervals for the disease-specific DALYs were generated using the delta method. Confidence intervals for total DALYs for each exposure were estimated by repeating 10,000 times the computation of total DALYs while resampling the log HR estimates from a multivariate normal distribution corresponding to the approximate sampling distribution of log HRs and by taking the 2.5th and 97.5th percentiles as the bounds for the 95% CIs (see below for detailed description). The sex-specific results were generated by estimating the HRs stratified by sex and using sex-specific population DALYs reported by the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e1910">27</a></sup>. For all error estimates we assumed that there was no uncertainty in frequencies of the exposures (for example, allele frequencies) or DALYs reported by the GBD, so we only considered variation from HR estimation in the error estimates.</p><h3 class="c-article__sub-heading" id="Sec13">Cohort description</h3><p>FinnGen is a public-private partnership research project combining genotype data generated from Finnish biobanks and digital health record data from Finnish health registries (<a href="https://www.finngen.fi/en">https://www.finngen.fi/en</a>). The <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01957-2#Sec33">Supplementary Information</a> contains the full list of contributors. For a comprehensive description of the cohort and methods, see the FinnGen flagship manuscript<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Kurki, M. I. et al. FinnGen: unique genetic insights from combining isolated population and national health register data. Preprint at medRxiv 
                  https://doi.org/10.1101/2022.03.03.22271360
                  
                 (2022)." href="/articles/s41591-022-01957-2#ref-CR28" id="ref-link-section-d72754869e1932">28</a></sup>. Participants for FinnGen include participants of legacy cohorts, both population-based epidemiological cohorts (initiated as far back as 1992) and disease-based cohorts and volunteers from biobanks. All samples donated to Finnish biobanks are eligible for FinnGen. Information about the opportunity to donate a sample and medical information for biobank research was distributed by leaflets, advertisement campaigns in press and on TV and by dedicated biobank nurses in hospitals.</p><p>Like other Nordic countries, Finland has nationwide electronic health registers originally established primarily for administrative purposes to monitor healthcare usage<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Laugesen, K. et al. Nordic health registry-based research: a review of health care systems and key registries. Clin. Epidemiol. 13, 533–554 (2021)." href="/articles/s41591-022-01957-2#ref-CR61" id="ref-link-section-d72754869e1939">61</a></sup>. These registers cover virtually all major health-related events, such as hospitalizations, prescription drug purchases, medical operations, cancers and deaths. All registers can be linked using the unique personal identity code which is given to every permanent resident of Finland. In this context, loss to follow-up can only result from emigration.</p><p>Registry data on all participants were collected from different national registers, including hospital and outpatient visits in HILMO, Care Register for Health Care (diagnoses, ICD-8–10; operations, NOMESCO Classification of Surgical Procedures), Causes of Death (immediate, contributing and underlying causes of death as ICD codes), reimbursed medication entitlements and all prescription medication purchases (ATC codes and special reimbursement codes) and the Finnish Cancer Registry (ICD-O-3 codes). The diagnostic accuracy and validity of these registers has been reviewed in many previous publications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vuori, M. A. et al. The validity of heart failure diagnoses in the Finnish Hospital Discharge Register. Scand. J. Public Health 
                  https://doi.org/10.1177/1403494819847051
                  
                 (2019)." href="#ref-CR62" id="ref-link-section-d72754869e1946">62</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Leinonen, M. K., Miettinen, J., Heikkinen, S., Pitkäniemi, J. &amp; Malila, N. Quality measures of the population-based finnish cancer registry indicate sound data quality for solid malignant tumours. Eur. J. Cancer 77, 31–39 (2017)." href="#ref-CR63" id="ref-link-section-d72754869e1946_1">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sund, R. et al. Comparing properties of audit data and routinely collected register data in case of performance assessment of hip fracture treatment in Finland. Methods Inf. Med. 46, 558–566 (2007)." href="#ref-CR64" id="ref-link-section-d72754869e1946_2">64</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Sund, R. Quality of the Finnish Hospital Discharge Register: a systematic review. Scand. J. Public Health 40, 505–515 (2012)." href="#ref-CR65" id="ref-link-section-d72754869e1946_3">65</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Haukka, J. Finnish health and social welfare registers in epidemiological research. Norsk Epidemiologi 14, 113−120 (2004)." href="/articles/s41591-022-01957-2#ref-CR66" id="ref-link-section-d72754869e1949">66</a></sup>.</p><p>We additionally included participants from the UKB, which is a population-based biobank cohort study that recruited around 500,000 people aged between 40 and 69 years from 2006 to 2010 across the United Kingdom. For details on participants and recruitment see Sudlow<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. PLoS Med. 12, e1001779 (2015)." href="/articles/s41591-022-01957-2#ref-CR29" id="ref-link-section-d72754869e1956">29</a></sup> and Bycroft<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)." href="/articles/s41591-022-01957-2#ref-CR67" id="ref-link-section-d72754869e1960">67</a></sup>. We used UKB data for common variants (meta-analysis with FinnGen) and rare variants (UKB only).</p><p>In FinnGen, we included participants from the Finnish ancestry passing genotypic quality control (data freeze 7) with <i>n</i> = 309,136 (56.2% female) with median (IQR) age at start of follow-up of 15.1 (0–26.3) and at end of follow-up 62.2 (47.1–72.9). The median (IQR) follow-up length was 47.3 (41.6–47.7) years (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">1</a>). Follow-up for FinnGen started on 1 January 1972 and ended on 31 August 2020. For UKB we used participants of European ancestry passing genotyping quality control: <i>n</i> = 426,612 (54.1% female) with median (IQR) age at start of follow-up of 47.2 (39.5–52.4) and at end of follow-up 69.1 (61.7–74.2). The median (IQR) follow-up length was 22.3 (22.3–22.3) years (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">1</a>). Follow-up for UKB started on 1 January 1998 and ended on 30 April 2020.</p><h3 class="c-article__sub-heading" id="Sec14">DALYs</h3><p>As the measure of disease burden, we used the DALYs from the 2019 GBD study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e1988">27</a></sup> with data publicly available at <a href="https://ghdx.healthdata.org/">https://ghdx.healthdata.org/</a>. DALYs are a metric for measuring population-level disease burden that combines a measure of premature mortality (YLL) and a measure of healthy life years lost due to lowered quality of life (YLD), so DALYs are the sum of YLDs and YLLs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig7">1</a>). The GBD is a longitudinal study that estimates the incidence, prevalence, mortality, YLLs, YLDs and DALYs due to various collectively exhaustive and mutually exclusive diseases and injuries (369 in 2019) for hundreds of countries (204 in 2019)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e2002">27</a></sup>. GBD strives to model unbiased estimates using various sources of information, including census data, household surveys, civil registration and vital statistics, disease registries, health service use data and more. The estimation process for DALYs is complex and varies from disease to disease (see the 2019 GBD publication<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e2006">27</a></sup> and its Supplementary Appendix 1 for a description of the methods).</p><p>Simplifying the GBD estimation process, the yearly YLDs in a population are estimated by multiplying the prevalence of a disease by its disability weight, which represents the magnitude of health loss due to living with the disease scaled between 0 (perfect health) and 1 (death). Yearly YLLs are estimated by multiplying the number of deaths attributable to a disease by the standard life expectancy at age of death<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e2013">27</a></sup>. The DALYs estimated by GBD avoid double-counting (for YLLs, there can only be one cause of death and for YLDs they implement a comorbidity correction<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e2017">27</a></sup>). Because all of the genetic exposures, except rare variants, were measured in FinnGen, we used the 2019 GBD metrics for Finland and present all estimates for Finland (see Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM1">3</a> for a list of DALYs, YLLs and YLDs for the included diseases).</p><h3 class="c-article__sub-heading" id="Sec15">Disease phenotypes</h3><p>To define similar disease phenotypes in FinnGen and UKB as those in the GBD study, we manually mapped 89 non-overlapping non-communicable diseases from the 2019 GBD study<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1204–1222 (2020)." href="/articles/s41591-022-01957-2#ref-CR27" id="ref-link-section-d72754869e2032">27</a></sup> into FinnGen clinical end points (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">2</a>). For more information on the FinnGen clinical end points, please see the FinnGen flagship manuscript<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Kurki, M. I. et al. FinnGen: unique genetic insights from combining isolated population and national health register data. Preprint at medRxiv 
                  https://doi.org/10.1101/2022.03.03.22271360
                  
                 (2022)." href="/articles/s41591-022-01957-2#ref-CR28" id="ref-link-section-d72754869e2039">28</a></sup>. The clinical end points can be explored in <a href="https://risteys.finngen.fi/">https://risteys.finngen.fi/</a>. The mapping was performed by matching the disease categories in the GBD as closely as possible to existing single FinnGen clinical end points or combinations of multiple FinnGen end points (separated with ‘|’ operator in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">2</a> FinnGen end points column).</p><p>We did not attempt to map infectious diseases, accidents, injuries, sufficiently rare conditions, conditions not ascertainable through registries or conditions for which the GBD 2019 does not report DALYs for Finland. Due to statistical power considerations, we removed 9 conditions for which there are under 500 cases, ending up at 80 medical conditions, which account for 83.1% of yearly total population-level DALYs from non-communicable diseases in Finland 2019 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">3</a>).</p><p>For example, the ischemic heart disease definition in GBD through causes of death encompasses ICD-10 codes I20–I25.9, but our definition of ischemic heart disease in FinnGen consists of combining five FinnGen end points: I9_MI (myocardial infarction, IC10-codes I21–I22, ICD-9 and ICD-8 code 410), I9_MI_COMPLICATIONS (complications following myocardial infarction, ICD-10 code I23), I9_POSTAMI (status post-acute myocardial infarction, ICD-10 code I25.3, ICD-9 and ICD-8 code 412), I9_CORATHER (coronary atherosclerosis, ICD-10 codes I24-I25, T82.2 and Z95.1) and I9_REVASC (coronary revascularization, defined through NOMESCO operation codes for coronary angioplasty and coronary artery bypass grafting). The first occurrence of any of these five end points was coded as the event of ischemic heart disease.</p><p>For FinnGen, we did not define disease phenotypes just using the ICD-10 codes provided in GBD for multiple reasons. First, ICD-10-based definitions for all disease conditions in GBD are not available (for example, low back pain). Second, even when GBD presents ICD-10 diagnoses, as follow-up for FinnGen for this study started in 1968, diseases need to also be defined through ICD-8 (in use until Dec 1987) and ICD-9 (in use from Jan 1987 to Dec 1995). Third, the FinnGen clinical end point phenotyping algorithms have been hand-crafted to utilize various sources of registry data in addition to diagnosis codes (for example, operation codes, prescription drug purchases and special drug reimbursement codes for certain diseases).</p><p>For example, migraine is a disease commonly managed in the primary care setting, so relying on diagnosis codes present in hospital and secondary care specialist visits data will lead to serious under-ascertainment of migraine. Also, there is misclassification of other types of headaches as migraine. The FinnGen end point that we used to define migraine (MIGRAINE_TRIPTAN) requires that a patient has at least one purchase of prescribed triptans, capturing, for example, patients who do not have a diagnosis code for migraine in the available registries that do not include primary care.</p><p>Phenotyping in FinnGen<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Kurki, M. I. et al. FinnGen: unique genetic insights from combining isolated population and national health register data. Preprint at medRxiv 
                  https://doi.org/10.1101/2022.03.03.22271360
                  
                 (2022)." href="/articles/s41591-022-01957-2#ref-CR28" id="ref-link-section-d72754869e2072">28</a></sup> relied on information on diagnoses starting from 1972 in the hospital discharge registry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Sund, R. Quality of the Finnish Hospital Discharge Register: a systematic review. Scand. J. Public Health 40, 505–515 (2012)." href="/articles/s41591-022-01957-2#ref-CR65" id="ref-link-section-d72754869e2076">65</a></sup>, the causes of death registry and the cancer registry. The drug purchases registry was additionally used starting from 1995 for selected diseases (for example, migraine). The cumulative incidence of the diseases varied from 14.93% (cataract) to 0.16% (acute glomerulonephritis) in FinnGen. Phenotyping in UKB was performed via groups of ICD-10 codes mapped from the FinnGen end points, relying on ICD-10 diagnosis codes from the hospital episode statistics, cancer registry and causes of death registry data starting from 1998 (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">2</a>). Due to statistical power considerations, we analyzed the following diseases only in FinnGen as in UKB there were fewer than 500 cases: hemoglobinopathies and hemolytic anemias (<i>n</i> = 480 in UKB), Hodgkin lymphoma (<i>n</i> = 464), autism spectrum disorders (<i>n</i> = 164), eating disorders (<i>n</i> = 163), acne vulgaris (<i>n</i> = 106), acute glomerulonephritis (<i>n</i> = 58), ADHD (<i>n</i> = 26) and conduct disorder (<i>n</i> = 13). As both FinnGen and UKB rely mainly on diagnosis codes recorded at hospitals, conditions that are usually managed in the primary or outpatient care setting are under-ascertained, but this does not bias the attributable DALY estimates if the HR estimates are unbiased.</p><h3 class="c-article__sub-heading" id="Sec16">Genotyping, imputation and quality control</h3><p>Samples for FinnGen were genotyped using Illumina and Affymetrix arrays (Illumina and Thermo Fisher Scientific). Genotype calls were made with GenCall or zCall for Illumina and the AxiomGT1 algorithm for Affymetrix data. Chip genotyping data produced with previous chip platforms and reference genome builds were lifted over to build v.38 (GRCh38/hg38) following the protocol described in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Pärn, K., Nunez-Fontarnau, J., Isokallio M. A., Sipilä, T., Kilpeläinen, E., Palotie, A., Ripatti, S. &amp; Palta, P. Genotyping chip data lift-over to reference genome build GRCh38/hg38. protocols.io. 
                  https://doi.org/10.17504/protocols.io.xbhfij6
                  
                 (2019)." href="/articles/s41591-022-01957-2#ref-CR68" id="ref-link-section-d72754869e2117">68</a></sup>. Participants with ambiguous sex, genotype missingness of over 5%, excess heterozygosity (±4 s.d.) and non-Finnish ancestry were excluded. Variants with over 2% missingness, low Hardy–Weinberg equilibrium <i>P</i> value (&lt;10<sup>−6</sup>) and minor allele count &lt;3 were excluded. Array data pre-phasing was performed with Eagle v.2.3.527 with default parameters, except the number of conditioning haplotypes was set at 20,000. Genotype imputation was performed with Beagle v.4.128 as described in ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Pärn, K., Isokallio, M. A., Nunez-Fontarnau, J., Palotie, A., Ripatti, S. &amp; Palta, P. Genotype imputation workflow v3.0. protocols.io. 
                  https://doi.org/10.17504/protocols.io.xbgfijw
                  
                 (2019)." href="/articles/s41591-022-01957-2#ref-CR69" id="ref-link-section-d72754869e2126">69</a></sup> by using the SISu v.3 reference panel developed from data on 3,775 high-coverage (25–30×) whole-genome sequenced Finns. For more details, please see the FinnGen flagship manuscript<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Kurki, M. I. et al. FinnGen: unique genetic insights from combining isolated population and national health register data. Preprint at medRxiv 
                  https://doi.org/10.1101/2022.03.03.22271360
                  
                 (2022)." href="/articles/s41591-022-01957-2#ref-CR28" id="ref-link-section-d72754869e2130">28</a></sup>.</p><p>For detailed information on genotyping, imputation and quality control for the UKB data, see Bycroft<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. Nature 562, 203–209 (2018)." href="/articles/s41591-022-01957-2#ref-CR67" id="ref-link-section-d72754869e2137">67</a></sup>. The genotyping was performed using the Applied Biosystems UK BiLEVE Axiom Array or the Applied Biosystems UKB Axiom Array. The genotype imputation was performed using a combination of the Haplotype Reference Consortium, UK10K and 1000 Genomes Project phase 3 reference panels by IMPUTE4 software. Variants with INFO score ≤ 0.8, MAF ≤ 0.01 and Hardy–Weinberg equilibrium <i>P</i> value ≤ 1 × 10<sup>−10</sup> were excluded.</p><p>For variant annotation, we utilized the Variant Effect Predictor<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="McLaren, W. et al. The ensembl variant effect predictor. Genome Biol. 17, 122 (2016)." href="/articles/s41591-022-01957-2#ref-CR60" id="ref-link-section-d72754869e2149">60</a></sup> (VEP) v.103. For coding variants, we chose a single most severe consequence and corresponding gene among canonical transcripts. We considered stop_gained, frameshift_variant, splice_donor, splice_acceptor, missense_variant, start_lost, stop_lost, inframe_insertion and inframe_deletion as protein-truncating variants.</p><h3 class="c-article__sub-heading" id="Sec17">Principal components and genetic ancestry assignment</h3><p>For FinnGen, the principal component analysis (PCA) for population structure was performed using the following approach. First, the following filters were applied: (1) exclusion of chromosome 23; (2) exclusion of variants with INFO score 0.95; (3) exclusion of variants with missingness &gt;0.01; (4) exclusion of variants with MAF &lt; 0.05; and (5) LD pruning with window 500 kb, step 50 kb and <i>r</i><sup>2</sup> filter of 0.1.</p><p>The imputed genotypes were merged with 1000 Genomes Project phase 3 data into a single dataset of 49,451 pruned single-nucleotide polymorphisms (SNPs), on which the principal components (PCs) were estimated. An unsupervised Bayesian algorithm (Aberrant) was used to spot outliers in the PCA space and remove them. While this method automatically detected the 1000 Genomes Project samples with non-European and southern European ancestries as outliers, it did not manage to exclude some samples with western European origins. As the signal from these samples would have been too small to allow a second round to be performed without detecting substructures of the Finnish population, another approach was used. The FinnGen samples that survived the first round were used to compute another PCA. The European and Finnish 1000 Genomes Project samples were projected onto the space generated by the first three PCs. For each sample, the probability of belonging to the EUR/FIN cluster was estimated through using a chi-squared distribution based on the Mahalanobis distance to the centroid of each cluster. Samples whose relative probability of being part of the Finnish cluster was &gt;95% were classified as belonging to the Finnish ancestry and retained in all following analyses for FinnGen (<i>n</i> = 309,136).</p><p>PCA and ancestry assignment in UKB followed the procedure of the Pan-UKB analysis (<a href="https://pan.ukbb.broadinstitute.org/">https://pan.ukbb.broadinstitute.org/</a>) and the procedure is described at <a href="https://pan.ukbb.broadinstitute.org/docs/qc#ancestry-definitions">https://pan.ukbb.broadinstitute.org/docs/qc#ancestry-definitions</a>.</p><h3 class="c-article__sub-heading" id="Sec18">HLA imputation in FinnGen</h3><p>The HLA imputation is described in detail elsewhere<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Ritari, J. et al. Increasing accuracy of HLA imputation by a population-specific reference panel in a FinnGen biobank cohort. NAR Genomics Bioinforma. 2, lqaa030 (2020)." href="/articles/s41591-022-01957-2#ref-CR70" id="ref-link-section-d72754869e2196">70</a></sup>. Briefly, HLA typing on 1,150 Finnish samples was performed by the HLA Laboratory of the Finnish Red Cross Blood Service using procedures accredited by the European Federation for Immunogenetics. Allele assignment of the seven HLA genes at two-field resolution level (unique protein sequence level) was performed by polymerase chain reaction (PCR)-based methods. HIBAG<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Zheng, X. et al. HIBAG—HLA genotype imputation with attribute bagging. Pharmacogenomics J. 14, 192–200 (2014)." href="/articles/s41591-022-01957-2#ref-CR71" id="ref-link-section-d72754869e2200">71</a></sup> v.1.14.0 with 100 classifiers for each of the seven HLA genes was fitted using the training data of 1,150 individuals to construct an imputation reference for the Finnish population, which was used to impute the HLA alleles in FinnGen<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Ritari, J. et al. Increasing accuracy of HLA imputation by a population-specific reference panel in a FinnGen biobank cohort. NAR Genomics Bioinforma. 2, lqaa030 (2020)." href="/articles/s41591-022-01957-2#ref-CR70" id="ref-link-section-d72754869e2204">70</a></sup>. We analyzed 74 alleles in seven HLA genes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">14</a>) for the HLA region (chr6:29691116 to chr6:3054976 in GRCh38).</p><h3 class="c-article__sub-heading" id="Sec19">Statistical fine-mapping</h3><p>Summary statistics for fine-mapping were obtained from standard FinnGen pipeline summary statistics, where mixed-model logistic regression using SAIGE<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Zhou, W. et al. Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies. Nat. Genet. 50, 1335–1341 (2018)." href="/articles/s41591-022-01957-2#ref-CR72" id="ref-link-section-d72754869e2219">72</a></sup> was used to obtain summary statistics for each FinnGen end point used to define the 80 diseases (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">2</a>). The models used sex and age as precision covariates. Genotyping batch and the ten first genetic PCs were used to control for confounding due to population stratification and batch effects. Using the summary statistics, we fine-mapped all regions with at least one variant having <i>P</i> &lt; 10<sup>−8</sup> and extended the regions 1.5 Mb upstream and downstream from each lead variant. Overlapping regions were merged and used in SuSiE<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Wang, G., Sarkar, A., Carbonetto, P. &amp; Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. J. R. Stat. Soc. Ser. B Stat. Methodol. 82, 1273–1300 (2020)." href="/articles/s41591-022-01957-2#ref-CR30" id="ref-link-section-d72754869e2231">30</a></sup> fine-mapping, allowing for up to ten causal variants per region and constructing 95% credible sets for each independent signal. In-sample dosage LD was computed using LDStore2. The FinnGen fine-mapping pipeline is available at <a href="https://github.com/FINNGEN/finemapping-pipeline">https://github.com/FINNGEN/finemapping-pipeline</a>.</p><p>The fine-mapping in UKB using SuSiE followed a similar procedure. Regions for fine-mapping were defined by greedily starting with the most significantly associated (highest chi-squared) variant, including all genome-wide significant (<i>P</i> &lt; 10<sup>−8</sup>) variants within a window of 3 Mb centered at the variant and merging overlapping regions. Summary statistics were obtained using BOLT-LMM<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Loh, P.-R. et al. Efficient Bayesian mixed-model analysis increases association power in large cohorts. Nat. Genet. 47, 284–290 (2015)." href="/articles/s41591-022-01957-2#ref-CR73" id="ref-link-section-d72754869e2251">73</a></sup> and SAIGE<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Zhou, W. et al. Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies. Nat. Genet. 50, 1335–1341 (2018)." href="/articles/s41591-022-01957-2#ref-CR72" id="ref-link-section-d72754869e2255">72</a></sup>. In-sample dosage LD was estimated using LDStore2. The maximum number of causal variants for each locus was ten. We only considered fine-mapping results for six quantitative risk factor traits (BMI, HbA1c, HDL cholesterol, LDL cholesterol, systolic blood pressure and cigarettes per day) and considered variants with <i>P</i> &lt; 10<sup>−12</sup> to restrict the number of variants to be selected.</p><h3 class="c-article__sub-heading" id="Sec20">Common variants</h3><p>We selected an initial list of 2,562 common variants (MAF &gt; 0.01) for inclusion in the analysis that either (1) had at least one <i>P</i> &lt; 5 × 10<sup>−8</sup> association with any of the 80 diseases and had the highest probability of being causal within a SuSiE fine-mapped<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Wang, G., Sarkar, A., Carbonetto, P. &amp; Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. J. R. Stat. Soc. Ser. B Stat. Methodol. 82, 1273–1300 (2020)." href="/articles/s41591-022-01957-2#ref-CR30" id="ref-link-section-d72754869e2278">30</a></sup> 95% credible set in FinnGen; (2) were coding variants with a <i>P</i> &lt; 5 × 10<sup>−8</sup> association with at least one of the diseases in FinnGen; or (3) had at least one <i>P</i> &lt; 5 × 10<sup>−12</sup> association with any of the selected six modifiable risk factor traits (BMI, HbA1c, HDL cholesterol, LDL cholesterol, systolic blood pressure and cigarettes per day) and had the highest probability of being causal within a SuSiE fine-mapped 95% credible set in UKB<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Kanai, M. et al. Insights from complex trait fine-mapping across diverse populations. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.09.03.21262975
                  
                 (2021)." href="/articles/s41591-022-01957-2#ref-CR31" id="ref-link-section-d72754869e2293">31</a></sup>. We labeled common variants as Finnish-enriched if they had at least fivefold MAF enrichment in FinnGen compared to NFSEE ancestry MAF in gnomAD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)." href="/articles/s41591-022-01957-2#ref-CR59" id="ref-link-section-d72754869e2297">59</a></sup> and the NFSEE MAF was &lt;0.01.</p><p>The 2,562 common variants in the initial list for inclusion can be in high LD with each other because we use fine-mapping results and coding variants for multiple phenotypes to select them. To select a set of common variants that are independent of each other for analysis, we performed LD clumping of the 2,562 variants as follows: We used PLINK<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet 81, 559–575 (2007)." href="/articles/s41591-022-01957-2#ref-CR74" id="ref-link-section-d72754869e2304">74</a></sup> v.1.90 to clump all common variants using an <i>r</i><sup>2</sup> threshold of 0.2, a 250-kb clumping window and FinnGen release 4 genotypes as the reference panel to remove SNPs in LD with variants having a smaller minimum <i>P</i> value among the 80 HRs for all examined diseases (meta-analysis estimates combining FinnGen and UKB); however, if there was a coding variant among the variants in LD, that was instead kept and others were discarded. Additionally, if there was a Finnish-enriched variant among the variants in LD (but no coding variant), the Finnish-enriched variant was kept. This resulted in 1,044 independent common variants to be included in the analyses (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">4</a>). For each variant, we defined the minor allele to be the allele less common in FinnGen. Due to symmetry in attributable DALY estimation, going from one to zero copies versus zero to one copies of an allele only changes the sign of the individual-attributable DALYs estimates. Additionally, we determined six haplotypes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">13</a>) for <i>APOE</i> based on <a href="https://www.ncbi.nlm.nih.gov/snp/rs429358">rs429358</a> and <a href="https://www.ncbi.nlm.nih.gov/snp/rs7412">rs7412</a> alleles<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Joshi, P. K. et al. Variants near CHRNA3/5 and APOE have age- and sex-related effects on human lifespan. Nat. Commun. 7, 11174 (2016)." href="/articles/s41591-022-01957-2#ref-CR34" id="ref-link-section-d72754869e2339">34</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec21">Rare deleterious variants</h3><p>To analyze the effects of rare deleterious variants on DALYs, we used whole-exome sequencing data from a subset of participants in UKB (<i>n</i> = 174,379) from the December 2020 release. Data was preprocessed using hail v.0.2. We used the quality-controlled PLINK<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. Am. J. Hum. Genet 81, 559–575 (2007)." href="/articles/s41591-022-01957-2#ref-CR74" id="ref-link-section-d72754869e2354">74</a></sup> files provided by UKB. Variants were annotated using the Ensembl Variant Effect Predictor (VEP)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="McLaren, W. et al. The ensembl variant effect predictor. Genome Biol. 17, 122 (2016)." href="/articles/s41591-022-01957-2#ref-CR60" id="ref-link-section-d72754869e2358">60</a></sup> following the approach in gnomAD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)." href="/articles/s41591-022-01957-2#ref-CR59" id="ref-link-section-d72754869e2362">59</a></sup> (gnomadvep.vep_or_lookup_vep). VEP annotations were processed using the function ‘gnomadvep.process_consequences’, consistently with the gnomAD definition of pLOF, missense and synonymous variants, using the canonical transcript. We also extracted ClinVar-annotated<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. Nucleic Acids Res. 42, D980–D985 (2014)." href="/articles/s41591-022-01957-2#ref-CR37" id="ref-link-section-d72754869e2366">37</a></sup> variants (accessed in November 2020) and germline variants in <i>BRCA1</i> and <i>BRCA2</i> (accessed in November 2020) annotated by the ENIGMA consortium<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Spurdle, A. B. et al. ENIGMA–evidence-based network for the interpretation of germline mutant alleles: an international initiative to evaluate risk and clinical significance associated with sequence variation in BRCA1 and BRCA2 genes. Hum. Mutat. 33, 2–7 (2012)." href="/articles/s41591-022-01957-2#ref-CR38" id="ref-link-section-d72754869e2377">38</a></sup>. From ClinVar we considered ‘pathogenic’ and ‘likely-pathogenic’ variants (no filtering on star status or number of submitters) and from ENIGMA we considered ‘pathogenic’ variants. Variants with a frequency &gt;0.001 were excluded because they are less likely to be deleterious. We considered all genes part of the ACMG recommendations for reporting incidental findings in clinical exome and genome sequencing studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Miller, D. T. et al. ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG). Genet. Med. 23, 1381–1390 (2021)." href="/articles/s41591-022-01957-2#ref-CR36" id="ref-link-section-d72754869e2381">36</a></sup>. We formed two types of rare variant burdens for individuals for each gene: (1) the pLOF burden was set as positive if there was at least one variant annotated as pLOF; and (2) the ClinVar/ENIGMA burden was set as positive if there was at least one variant annotated as ‘pathogenic’ in ENIGMA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Spurdle, A. B. et al. ENIGMA–evidence-based network for the interpretation of germline mutant alleles: an international initiative to evaluate risk and clinical significance associated with sequence variation in BRCA1 and BRCA2 genes. Hum. Mutat. 33, 2–7 (2012)." href="/articles/s41591-022-01957-2#ref-CR38" id="ref-link-section-d72754869e2385">38</a></sup> for <i>BRCA1</i> and <i>BRCA2</i> and for other genes ‘likely pathogenic’ or ‘pathogenic’ in ClinVar. Due to statistical power considerations, we restricted our analysis so that at least 35 individuals had a positive burden, resulting in nine genes for both burden types (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">7</a>).</p><h3 class="c-article__sub-heading" id="Sec22">Polygenic scores</h3><p>We included 30 genome-wide PGSs for traits of interest constructed from publicly available summary statistics (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">10</a>). We selected PGSs for psychobehavioral traits (for example cognitive ability and neuroticism), major chronic diseases (for example coronary artery disease and depression) and major risk factors (for example LDL cholesterol and blood pressure) to cover traits of interest with high-quality summary statistics available. PGSs were only analyzed in FinnGen, as many of the original summary statistics included UKB.</p><p>We used PRS-CS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Ge, T., Chen, C.-Y., Ni, Y., Feng, Y.-C. A. &amp; Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. Nat. Commun. 10, 1776 (2019)." href="/articles/s41591-022-01957-2#ref-CR75" id="ref-link-section-d72754869e2414">75</a></sup> for generating the PGSs using external LD reference panel (1000 Genomes Project Europeans). We used the PRS-CS-auto algorithm, which learns the global scaling parameter ϕ from the data and performs well with large datasets. Default PRS-CS parameters were used and only HapMap 3 variants were considered (<a href="https://github.com/FINNGEN/CS-PRS-pipeline">https://github.com/FINNGEN/CS-PRS-pipeline</a>). Scores for lifespan, educational attainment, cognitive performance and intelligence were reversed before analysis to make all scores on net deleterious in terms of DALYs. For defining the exposure for survival models, we coded individuals for each PGS as one if they were in the top 10% of the score distribution and zero if not. Consequently, the individual DALYs can be interpreted as the effect on lifetime DALYs if the average individual in top 10% of the PGS were to have their PGS be that of the average in the bottom 90% of the PGS.</p><p>We additionally formed a composite PGS of mortality using all the 30 PGSs (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">10</a>) emulating the approach by Meisner et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Meisner, A. et al. Combined utility of 25 disease and risk factor polygenic risk scores for stratifying risk of all-cause mortality. Am. J. Hum. Genet. 107, 418–431 (2020)." href="/articles/s41591-022-01957-2#ref-CR9" id="ref-link-section-d72754869e2431">9</a></sup> by modeling survival via a Cox proportional hazards model with sex, the 30 PGSs and ten PCs of population structure as predictors. We then extracted the linear predictor for all individuals, while setting sex to 0.5 and the PCs to 0 (the mean) and interpreted these values as a composite PGS of mortality.</p><h3 class="c-article__sub-heading" id="Sec23">Survival models</h3><p>To estimate the HRs between all genetic exposure–disease pairs we used Cox proportional hazards regression via the coxph function in the survival package v.3.2-11 in R. The model was additive for allele counts. Sex and the first ten PCs of population structure were included as covariates. We used calendar age as the timescale and age at first record of the disease in the registries as time-to-event. Individuals were censored at death, emigration or end of registry-based follow-up (31 August 2020 in FinnGen and 30 April 2020 in UKB). For the common variants, HRs were estimated both in FinnGen and UKB separately and combined using fixed effects inverse-variance-weighted meta-analysis. A comparison of effect sizes between FinnGen and UKB is provided in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig9">3</a>. We did not account for left censoring or relatedness of the individuals due to computational limitations. For the rare variant burden analysis in UKB, we used Cox regression with Firth’s Penalized Likelihood<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Heinze, G. &amp; Schemper, M. A solution to the problem of monotone likelihood in Cox regression. Biometrics 57, 114–119 (2001)." href="/articles/s41591-022-01957-2#ref-CR76" id="ref-link-section-d72754869e2446">76</a></sup> via the coxphf package v.1.13.1 and included sex and the four first genetic PCs as covariates.</p><p>As a sensitivity analysis in FinnGen, we examined whether accounting for relatedness would meaningfully change the standard errors of the log HRs. For 2,562 common variant–disease pairs we estimated the log HRs using a survival model clustered by family indicator to generate robust standard errors. We generated a family indicator from genotype data using KING<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Manichaikul, A. et al. Robust relationship inference in genome-wide association studies. Bioinformatics 26, 2867–2873 (2010)." href="/articles/s41591-022-01957-2#ref-CR77" id="ref-link-section-d72754869e2453">77</a></sup> v.2.2 only including HapMap 3 variants. Individuals up to a third degree of relatedness were included in the same family. The robust standard errors were estimated using a family indicator to compute robust standard errors by including cluster(family_id) as a covariate in coxph. Compared to the main analysis estimates, robust standard errors were a median 1.0128-times (IQR 1.0044–1.0205) larger. Thus, accounting for relatedness would not meaningfully affect the CIs and <i>P</i> values.</p><p>As a second sensitivity analysis, we explored whether the effect of the genetic exposures on the diseases was age-dependent by performing age-stratified survival analyses for a subset of genetic exposures. Perhaps unsurprisingly<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Stensrud, M. J. &amp; Hernán, M. A. Why test for proportional hazards? JAMA 323, 1401–1402 (2020)." href="/articles/s41591-022-01957-2#ref-CR78" id="ref-link-section-d72754869e2463">78</a></sup>, we observed age-varying HRs for genetic exposures (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig14">8</a>). For example, for the coronary artery disease PGS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Nikpay, M. et al. A comprehensive 1,000 Genomes-based genome-wide association meta-analysis of coronary artery disease. Nat. Genet. 47, 1121–1130 (2015)." href="/articles/s41591-022-01957-2#ref-CR46" id="ref-link-section-d72754869e2470">46</a></sup>, the HRs were 2.50 (CI 2.31–2.70) for the 50–54-year age group and 1.75 (CI 1.62–1.87) for the 70–74-year age group.</p><h3 class="c-article__sub-heading" id="Sec24">Shrinkage</h3><p>We use prior information to apply a shrinkage procedure to the HRs for exposure–disease pairs to reduce the effect of sampling variation at the cost of being more conservative (biasing total attributable DALYs toward zero). The possible benefits of shrinkage include: (1) the top-ranking variants suffer less from the Winner’s curse (overestimation due to sampling variation); (2) by reducing the number of diseases through which a variant contributes to DALYs, variants increasing risk of low-DALY diseases do not have their total DALY estimates overshadowed by noisy weak effects through high-DALY diseases reflecting sampling variation; and (3) the shrinkage helps remove possible weak effects due to residual confounding from population stratification.</p><p>We denoted by <i>b</i><sub><i>e</i>,<i>d</i></sub> the log HR of genetic exposure <i>e</i> on disease <i>d</i> = <i>1,..,80</i>. One genetic exposure at a time, we set the prior distribution of <i>b</i><sub><i>e</i>,<i>d</i></sub>, to be a mixture distribution between the point mass at 0 and a 50:50 mixture of two normal distributions with means at −0.3 and 0.3, respectively and with a s.d. of 0.1. We denote the mixture weight of the non-zero component by <i>p</i><sub><i>e</i></sub>, which can be interpreted as the exposure-specific proportion of non-zero effects across the 80 diseases. We set the prior distribution of <i>p</i><sub><i>e</i></sub> to a <span class="mathjax-tex">\({{{\mathrm{Beta}}}}\left( {\alpha = 1,\beta = 19} \right)\)</span> distribution that has an expected value of 0.05. The full probability model is:</p><div id="Equa" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\begin{array}{l}p_e\sim {{{\mathrm{Beta}}}}\left( {\alpha ,\beta } \right),\pi _{e,d}\sim {{{\mathrm{Bernoulli}}}}\left( {0.5} \right),\\ b_{e,d}\sim {{{\mathrm{Bernoulli}}}}\left( {p_e} \right)\left( {\left( {1 - \pi _{e,d}} \right){{{\mathcal{N}}}}\left( {\mu ,\sigma ^2} \right) + \pi _{e,d}{{{\mathcal{N}}}}\left( { - \mu ,\sigma ^2} \right)} \right),\end{array}$$</span></div></div><p>where <i>α</i> = 1, <i>β</i> = 19, <i>μ</i> = 0.3 and <i>σ</i> = 0.1.</p><p>This model implies that, before seeing the data, for each genetic exposure, we expect a non-zero effect for four (= 0.05 × 80) diseases and the direction of the non-zero effects are equally likely to be risk increasing (centered around HR = 1.35) as protective (centered around HR = 0.74). In practice, which effects are shrunk to zero and which are retained as non-zero, does not vary considerably when these prior parameters are varied (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig15">9</a>). For each genetic exposure <i>e</i> at a time, we used a Markov Chain Monte Carlo procedure with 10,000 iterations to estimate the posterior probabilities of the log HRs (<i>b</i><sub><i>e,d</i></sub>) for diseases <i>d</i> = 1,2,…,80 coming from the null model (point mass at zero). We discarded any log HRs where the null probability was over 10% and, for the remaining log HRs, we used the maximum partial likelihood estimates from the Cox proportional hazards model in the downstream analyses.</p><h3 class="c-article__sub-heading" id="Sec25">Examining shrinkage performance using simulated data</h3><p>As a sensitivity analysis to explore the performance of the shrinkage method, we used hail v.0.2 to simulate variant–phenotype associations for 80 diseases where the true underlying effects are known. The approach used genetic data from 361,194 individuals from UKB with European ancestry and has the advantage of using realistic variant frequencies and population structure as compared to simulated genetic data. Only 558,240 independent HapMap 3 SNPs were considered in the analysis. Using the ldscsim.simulate_phenotypes function we simulated 80 phenotypes based on a spike-and-slab model with a different probability of SNPs being causal (<i>π</i>). The heritability of the phenotypes was randomly sampled from a uniform distribution ranging from 10 to 60%. The phenotypes were consequently binarized based on the disease prevalence observed in FinGen using the function ldscsim.binarize. We considered four <i>π</i> values (0.001, 0.002, 0.005 and 0.01) meaning that 0.1%, 0.2%, 0.5% and 1% of the 558,240 independent variants had a true underlying effect different from 0.</p><p>We then ran a GWAS for each of the phenotypes across the four different <i>π</i> scenarios. This allows us to obtain an observed effect size from the GWAS and an expected true underlying effect. Out of the observed effects, consistently with the variant selection process used on the real data, we only included variants that had at least one genome-wide significant association (<i>P</i> &lt; 5 × 10<sup>−8</sup>). For this selected group of variants, we applied the same shrinkage procedure as in the main analysis.</p><p>Our procedure shrinks most of the variant–phenotype associations to 0, while maintaining others unshrunk. Because we know the true underlying effect sizes, that is, which variants have effect size of 0 (null model) and effect sizes different from 0 (alternative model), we can compare how well our procedure shrinks variants from the null model versus does not shrink those from the alternative model. Overall, our approach results in area under the curve values of 0.817 to 0.897 for different values of <i>π</i>. Thus, the shrinkage approach can identify true causal variants in simulated GWAS data (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01957-2#Fig16">10</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">16</a>) reasonably well.</p><h3 class="c-article__sub-heading" id="Sec26">Attributable DALYs</h3><p>Similarly to the GBD<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. Lancet 396, 1223–1249 (2020)." href="/articles/s41591-022-01957-2#ref-CR11" id="ref-link-section-d72754869e2889">11</a></sup>, we used the HRs and frequencies of the exposures to estimate attributable DALYs one disease at a time. We used the multilevel exposure formula<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Khosravi, A., Nazemipour, M., Shinozaki, T. &amp; Mansournia, M. A. Population attributable fraction in textbooks: time to revise. Glob. Epidemiol. 3, 100062 (2021)." href="/articles/s41591-022-01957-2#ref-CR79" id="ref-link-section-d72754869e2893">79</a></sup> for the population-attributable fraction (the fraction of cases of disease <i>d</i> caused by the exposure levels in the population deviating from counterfactual levels):</p><div id="Equb" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${{{\mathrm{Population}}}}{\mbox{-}}{{{\mathrm{attributable}}}}{\mbox{-}}{{{\mathrm{fraction}}}} = {{{\mathrm{AFp}}}}_d = \frac{{\mathop {\sum}\nolimits_i {{{{\mathrm{P}}}}_i{{{\mathrm{HR}}}}_{d,i}} - \mathop {\sum}\nolimits_i {{{{\mathrm{P}}}}_i^\prime {{{\mathrm{HR}}}}_{d,i}} }}{{\mathop {\sum}\nolimits_i {{{{\mathrm{P}}}}_i{{{\mathrm{HR}}}}_{d,i}} }}$$</span></div></div><p>where <span class="mathjax-tex">\({{{\mathrm{HR}}}}_{d,i}\)</span> is the HR for disease <i>d</i> at exposure level <i>i</i> (for example, one copy) relative to reference (for example, zero copies) and <sub><i>Pi</i></sub> is the fraction of the population at exposure level <i>i</i> and <span class="mathjax-tex">\({{{\mathrm{P}}}}_i^\prime\)</span> represents the fraction of the population at exposure level <i>i</i> in the counterfactual scenario (for example, if all with one copy had zero instead: <span class="mathjax-tex">\({{{\mathrm{P}}}}_1^\prime = 0\)</span>, <span class="mathjax-tex">\({{{\mathrm{P}}}}_0^\prime = {{{\mathrm{P}}}}_0 + {{{\mathrm{P}}}}_1\)</span> and <span class="mathjax-tex">\({{{\mathrm{P}}}}_2^\prime = {{{\mathrm{P}}}}_2\)</span>).</p><p>As an example, for individuals carrying zero, one and two alleles with respective population frequencies of <i>P</i><sub>0</sub> = 0.7, <i>P</i><sub>1</sub> = 0.2 and <i>P</i><sub>2</sub> = 0.1 and HRs for disease <i>d</i> of <span class="mathjax-tex">\({{{\mathrm{HR}}}}_{d,0} = 1.00\)</span>, <span class="mathjax-tex">\({{{\mathrm{HR}}}}_{d,1} = 1.35\)</span> and <span class="mathjax-tex">\({{{\mathrm{HR}}}}_{d,2} = 1.82\)</span>. For calculating attributable DALYs from individuals carrying one versus zero copies of the allele, define the counterfactual frequencies as <span class="mathjax-tex">\({{{\mathrm{P}}}}_0^\prime = 0.9\)</span>, <span class="mathjax-tex">\({{{\mathrm{P}}}}_1^\prime = 0.0\)</span> and <span class="mathjax-tex">\({{{\mathrm{P}}}}_2^\prime = 0.1\)</span> (making all with one copy have zero copies instead). Plugging these numbers into the AFp<sub><i>d</i></sub> formula produces the population-attributable fraction of disease cases from those carrying one versus zero copies of the allele (the fraction of cases that would be prevented if all with one allele had zero instead), which is 0.061 in this case, so approximately 6.1% of disease cases would be prevented if all with one copy had instead zero copies at birth.</p><p>We then assumed that the estimated population-attributable fraction of disease cases can be interpreted as the population-attributable fraction of DALYs, which is true if all disease cases contribute on average the same amount of DALYs independent of whether they have the genetic exposure or not. This assumption does not hold if, for example, deleterious <i>BRCA1</i> mutation carriers develop breast cancer earlier and consequently accumulate more DALYs per case. Then, multiplying the population-attributable fraction of DALYs (AFp<sub><i>d</i></sub>) by the population DALYs per year per 100,000 reported by the GBD (DALY<sub><i>d</i></sub> gives the population-attributable DALYs), interpreted in our example as the expected loss of healthy life years per year per 100,000 if the population frequencies of the exposure were <span class="mathjax-tex">\({{{\mathrm{P}}}}_i^\prime\)</span> instead of <sub><i>Pi</i></sub> (in our example, all with one copy had zero copies instead).</p><div id="Equc" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${{{\mathrm{Population}}}}\,{{{\mathrm{attributable}}}}\,{{{\mathrm{DALYs}}}}_{{{\mathrm{d}}}} = {{{\mathrm{AFp}}}}_{{{\mathrm{d}}}} \times {{{\mathrm{DALY}}}}_d$$</span></div></div><p>We further estimated individual-attributable DALYs for binary counterfactuals (for example, having one versus zero copies, being in the top 10% of a PGS versus the bottom 90%) by dividing the population-attributable DALYs per 100,000 by the number of individuals with the exposure out of 100,000 (100,000 × <sub><i>Pi</i></sub>) and multiplying by life expectancy at birth (L):</p><div id="Equd" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\begin{array}{*{20}{l}} {{{{\mathrm{Individual}}}}\,{{{\mathrm{attributable}}}}\,{{{\mathrm{DALYs}}}}_{{{\mathrm{d}}}}} \hfill &amp; = \hfill &amp; {\frac{{{{{\mathrm{AFp}}}}_{{{\mathrm{d}}}} \times {{{\mathrm{DALY}}}}_d}}{{100,000 \times {{{\mathrm{P}}}}_i}} \times }{{{\mathrm{L}}}} \hfill \\ \hfill &amp; {} \hfill {{{\mathrm{=}}}} &amp; {\frac{{{{{\mathrm{Population}}}}\,{{{\mathrm{attributable}}}}\,{{{\mathrm{DALYs}}}}_{{{\mathrm{d}}}}}}{{{{{\mathrm{No}}}}{{{\mathrm{.individuals}}}}\,{{{\mathrm{exposed}}}}\,{{{\mathrm{per}}}}\,{{{\mathrm{100,000}}}}}} \times {{{\mathrm{L}}}}} \hfill \end{array}$$</span></div></div><p>where DALY<sub><i>d</i></sub> represents the population DALYs per year per 100,000 through disease <i>d</i> from GBD, <sub><i>Pi</i></sub> is the fraction of population for which the exposure is changed (for example, fraction of those with one copy) and L is included to convert yearly DALYs into lifetime estimates. These individual DALYs are interpreted as the expected loss of healthy life years for an individual caused by having the genetic exposure at birth. Finally, both population-attributable DALYs and individual DALYs can be summed up across the 80 diseases to arrive at the total impact of the genetic exposure. Note that the attributable DALYs (or the population-attributable fractions) for multiple exposures cannot be added together to estimate the effect of jointly intervening on multiple exposures<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Rowe, A. K., Powell, K. E. &amp; Flanders, W. D. Why population attributable fractions can sum to more than one. Am. J. Prev. Med. 26, 243–249 (2004)." href="/articles/s41591-022-01957-2#ref-CR80" id="ref-link-section-d72754869e3970">80</a></sup>. Thus, summing the attributable DALYs for multiple genetic exposures does not prove a correct estimate for the counterfactual joint intervention on multiple exposures (for example, summing attributable DALYs of two variants). Also see Witte et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Witte, J. S., Visscher, P. M. &amp; Wray, N. R. The contribution of genetic variants to disease depends on the ruler. Nat. Rev. Genet. 15, 765–776 (2014)." href="/articles/s41591-022-01957-2#ref-CR81" id="ref-link-section-d72754869e3974">81</a></sup> for discussion on how population-attributable fractions relate to other measures of genetic contribution.</p><h3 class="c-article__sub-heading" id="Sec27">Uncertainty estimation of total attributable DALYs</h3><p>Assuming that there is no uncertainty in the DALY estimates from GBD and the estimated population prevalence of the exposures (for example, allele frequencies), for a single disease both attributable individual and population DALYs are a deterministic function of the HRs between the exposure and the diseases. Therefore, CIs for the effect of a genetic exposure on attributable DALYs through one disease was estimated using the delta method.</p><p>Estimating the total attributable DALYs through the 80 examined diseases is less straightforward, as the HRs for different diseases are not independent (for example, ischemic heart disease and lower extremity peripheral artery disease are comorbid, so risk variants tend to increase risk for both). Bootstrapping was not computationally feasible, so we estimated the uncertainty via resampling the multivariate normal distribution of the log HR estimates.</p><p>Considering a single genetic exposure <i>e</i>, let <span class="mathjax-tex">\({{{\boldsymbol{b}}}}_e = (b_{e,1}, \ldots ,b_{e,80})^{{{\mathrm{T}}}}\)</span> denote the random vector of the log HRs between exposure <i>e</i> and the <span class="mathjax-tex">\(d = 1,2, \ldots ,80\)</span> diseases, let <b><i>β</i></b><sub>e</sub> denote the estimated vector of Cox model coefficients (log HRs) for the 80 diseases, let Σ<sub><i>e</i></sub> denote the covariance matrix of those coefficients, where the diagonal represents the standard errors of the coefficients. The coefficients follow a multivariate normal distribution:</p><div id="Eque" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${{{\boldsymbol{b}}}}_e\sim {{{\mathcal{N}}}}\left( {{{{\mathbf{\upbeta }}}}_e,{{{\mathbf{{\Sigma}}}}}_e} \right)$$</span></div></div><p>We can express the covariance matrix Σ<sub><i>e</i></sub> in terms of the diagonal matrix <b><i>D</i></b><sub><i>e</i></sub> = diag (<b>σ</b><sub><i>e</i></sub>) that has the standard errors of the coefficients <span class="mathjax-tex">\(\sigma _e = (\sigma _{e,1}, \ldots ,\sigma _{e,80})^T\)</span> on the diagonal and the correlation matrix of the coefficients <b><i>C</i></b> as:</p><div id="Equf" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${\Sigma}_e = {{{\boldsymbol{D}}}}_e{{{\boldsymbol{CD}}}}_e$$</span></div></div><p>so that</p><div id="Equg" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${{{\boldsymbol{b}}}}_e\sim {{{\mathcal{N}}}}\left( {{{{\mathbf{\upbeta }}}}_e,{{{\boldsymbol{D}}}}_e{{{\boldsymbol{CD}}}}_e} \right)$$</span></div></div><p>We estimate <b><i>β</i></b><sub><i>e</i></sub> and <b><i>σ</i></b><sub><i>e</i></sub> from the 80 Cox models for each disease (for common variants we use the meta-analysis estimates from FinnGen and UKB). Let <span class="mathjax-tex">\(d = 1,2, \ldots ,80\)</span> index all the different diseases. We estimated <b><i>C</i></b><sub><i>d</i> × <i>d</i></sub> by taking all the shrunk log HRs (assuming that they represent null effects) between all common variant–disease pairs and calculating the Pearson’s correlation coefficient between the log HRs of two diseases:</p><div id="Equh" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$\widehat {{{\boldsymbol{C}}}}_{i,j} = {{{\boldsymbol{r}}}}(\widehat {{{\mathbf{\upbeta }}}}_i,\widehat {{{\mathbf{\upbeta }}}}_j)$$</span></div></div><p>where <span class="mathjax-tex">\(\widehat {{{\mathbf{\upbeta }}}}_i\)</span> and <span class="mathjax-tex">\(\widehat {{{\mathbf{\upbeta }}}}_j\)</span> are the Cox model coefficients for disease <span class="mathjax-tex">\(i,j = 1,2, \ldots ,80\)</span> for common variants not shrunk for diseases <sub><i>i</i></sub>, <sub><i>j</i></sub> (at most 1,044). We restricted the correlation estimation to unshrunk variants to to make the coefficients reflect sampling variability, not true effects.</p><p>For each genetic exposure <i>e</i> we then resample the vector of log HRs <span class="mathjax-tex">\(B = 1,2, \ldots ,10,000\)</span> times from the multivariate normal distribution</p><div id="Equi" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${{{\boldsymbol{b}}}}_{e,B}^ \ast \sim {{{\mathcal{N}}}}\left( {\widehat {{{\mathbf{\upbeta }}}}_{{{\boldsymbol{e}}}},\widehat {{{\boldsymbol{D}}}}_e\widehat {{{\boldsymbol{C}}}}\widehat {{{\boldsymbol{D}}}}_e} \right)$$</span></div></div><p>to emulate the sampling distribution of the vector of log HRs across all diseases that accounts for dependence in log HRs between diseases. We then repeat the estimation procedure for individual and population total attributable DALYs for each genetic exposure 10,000 times using the resampled <span class="mathjax-tex">\({{{\boldsymbol{b}}}}_{e,B}^ \ast\)</span> to calculate the HRs instead of the maximum partial likelihood estimates from the Cox model (<span class="mathjax-tex">\(\widehat \beta _i\)</span>). We then use the 2.5% and 97.5% percentiles of the resampled distribution as estimates of the 95% CIs and estimate the <i>P</i> values via a normal approximation.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec28">FinnGen ethics statement</h4><p>Patients and control participants in FinnGen provided informed consent for biobank research, based on the Finnish Biobank Act. Alternatively, separate research cohorts, collected before the Finnish Biobank Act came into effect (in September 2013) and start of FinnGen (August 2017), were collected based on study-specific consents and later transferred to the Finnish biobanks after approval by Fimea (Finnish Medicines Agency), the National Supervisory Authority for Welfare and Health. Recruitment protocols followed the biobank protocols approved by Fimea. The Coordinating Ethics Committee of the Hospital District of Helsinki and Uusimaa (HUS) statement number for the FinnGen study is HUS/990/2017.</p><p>The FinnGen study is approved by the Finnish Institute for Health and Welfare (permit nos. THL/2031/6.02.00/2017, THL/1101/5.05.00/2017, THL/341/6.02.00/2018, THL/2222/6.02.00/2018, THL/283/6.02.00/2019, THL/1721/5.05.00/2019 and THL/1524/5.05.00/2020), Digital and Population Data Service Agency (permit nos. VRK43431/2017-3, VRK/6909/2018-3 and VRK/4415/2019-3), the Social Insurance Institution (permit nos. KELA 58/522/2017, KELA 131/522/2018, KELA 70/522/2019, KELA 98/522/2019, KELA 134/522/2019, KELA 138/522/2019, KELA 2/522/2020 and KELA 16/522/2020), Findata permit nos. THL/2364/14.02/2020, THL/4055/14.06.00/2020, THL/3433/14.06.00/2020, THL/4432/14.06/2020, THL/5189/14.06/2020, THL/5894/14.06.00/2020, THL/6619/14.06.00/2020, THL/209/14.06.00/2021, THL/688/14.06.00/2021, THL/1284/14.06.00/2021, THL/1965/14.06.00/2021 and THL/5546/14.02.00/2020 and Statistics Finland (permit nos. TK-53-1041-17 and TK/143/07.03.00/2020 (earlier TK-53-90-20)).</p><p>The Biobank Access Decisions for FinnGen samples and data utilized in FinnGen Data Freeze 7 include: THL Biobank BB2017_55, BB2017_111, BB2018_19, BB_2018_34, BB_2018_67, BB2018_71, BB2019_7, BB2019_8, BB2019_26 and BB2020_1, Finnish Red Cross Blood Service Biobank 7.12.2017, Helsinki Biobank HUS/359/2017, Auria Biobank AB17-5154 and amendment no. 1 (17 August 2020), Biobank Borealis of Northern Finland_2017_1013, Biobank of Eastern Finland 1186/2018 and amendment 22 §/2020, Finnish Clinical Biobank Tampere MH0004 and amendments (21 February 2020 and 6 October 2020), Central Finland Biobank 1-2017 and Terveystalo Biobank STB 2018001.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec29">UK Biobank ethics statement</h4><p>UKB obtained ethics approval from the North West Multicentre Research Ethics Committee, which covers the United Kingdom (approval no. 11/NW/0382) and obtained informed consent from all participants. Our analyses were conducted under the UKB application no. 31063.</p><h3 class="c-article__sub-heading" id="Sec30">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>We present meta-analyzed HR estimates and all attributable DALY results in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">5</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">6</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">8</a>, <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">9</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01957-2#Sec33">11−14</a>. Results for common variants, HLA alleles and PGSs can additionally be explored through plots at <a href="https://dsge-lab.shinyapps.io/daly_genetics/">https://dsge-lab.shinyapps.io/daly_genetics/</a>. Individual-level genotypes and register data from FinnGen participants can be accessed by approved researchers via the Fingenious portal (<a href="https://site.fingenious.fi/en/">https://site.fingenious.fi/en/</a>) hosted by the Finnish Biobank Cooperative FinBB (<a href="https://finbb.fi/en/">https://finbb.fi/en/</a>). Data release to FinBB is timed to the biannual public release of FinnGen summary results, which occurs 12 months after FinnGen consortium members can start working with the data.</p>
              <p>UKB data are available to approved researchers upon application (<a href="https://www.ukbiobank.ac.uk/enable-your-research/apply-for-access">https://www.ukbiobank.ac.uk/enable-your-research/apply-for-access</a>).</p>
              <p>Some of the datasets used in this study can be accessed in ClinVar (<a href="https://www.ncbi.nlm.nih.gov/clinvar/">https://www.ncbi.nlm.nih.gov/clinvar/</a>) and gnomAD (<a href="https://gnomad.broadinstitute.org/">https://gnomad.broadinstitute.org/</a>).</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>Custom code for the central parts of analyses can be found at <a href="https://github.com/dsgelab/dalys_code">https://github.com/dsgelab/dalys_code.</a></p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Visscher, P. M. et al. 10 Years of GWAS discovery: biology, function, and translation. <i>Am. J. Hum. Genet</i>. <b>101</b>, 5–22 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2017.06.005" data-track-item_id="10.1016/j.ajhg.2017.06.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2017.06.005" aria-label="Article reference 1" data-doi="10.1016/j.ajhg.2017.06.005">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFChu7rI" aria-label="CAS reference 1">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28686856" aria-label="PubMed reference 1">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5501872" aria-label="PubMed Central reference 1">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=10%20Years%20of%20GWAS%20discovery%3A%20biology%2C%20function%2C%20and%20translation&amp;journal=Am.%20J.%20Hum.%20Genet&amp;doi=10.1016%2Fj.ajhg.2017.06.005&amp;volume=101&amp;pages=5-22&amp;publication_year=2017&amp;author=Visscher%2CPM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Watanabe, K. et al. A global overview of pleiotropy and genetic architecture in complex traits. <i>Nat. Genet.</i> <b>51</b>, 1339–1348 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-019-0481-0" data-track-item_id="10.1038/s41588-019-0481-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-019-0481-0" aria-label="Article reference 2" data-doi="10.1038/s41588-019-0481-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1WjurvF" aria-label="CAS reference 2">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31427789" aria-label="PubMed reference 2">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20global%20overview%20of%20pleiotropy%20and%20genetic%20architecture%20in%20complex%20traits&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0481-0&amp;volume=51&amp;pages=1339-1348&amp;publication_year=2019&amp;author=Watanabe%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Mars, N. et al. The role of polygenic risk and susceptibility genes in breast cancer over the course of life. <i>Nat. Commun.</i> <b>11</b>, 6383 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-19966-5" data-track-item_id="10.1038/s41467-020-19966-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-19966-5" aria-label="Article reference 3" data-doi="10.1038/s41467-020-19966-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1agtr%2FK" aria-label="CAS reference 3">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33318493" aria-label="PubMed reference 3">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7736877" aria-label="PubMed Central reference 3">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20polygenic%20risk%20and%20susceptibility%20genes%20in%20breast%20cancer%20over%20the%20course%20of%20life&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19966-5&amp;volume=11&amp;publication_year=2020&amp;author=Mars%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Rasmussen, K. L., Tybjærg-Hansen, A., Nordestgaard, B. G. &amp; Frikke-Schmidt, R. Absolute 10-year risk of dementia by age, sex and APOE genotype: a population-based cohort study. <i>CMAJ</i> <b>190</b>, E1033–E1041 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1503/cmaj.180066" data-track-item_id="10.1503/cmaj.180066" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1503%2Fcmaj.180066" aria-label="Article reference 4" data-doi="10.1503/cmaj.180066">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30181149" aria-label="PubMed reference 4">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148649" aria-label="PubMed Central reference 4">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Absolute%2010-year%20risk%20of%20dementia%20by%20age%2C%20sex%20and%20APOE%20genotype%3A%20a%20population-based%20cohort%20study&amp;journal=CMAJ&amp;doi=10.1503%2Fcmaj.180066&amp;volume=190&amp;pages=E1033-E1041&amp;publication_year=2018&amp;author=Rasmussen%2CKL&amp;author=Tybj%C3%A6rg-Hansen%2CA&amp;author=Nordestgaard%2CBG&amp;author=Frikke-Schmidt%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Metcalfe, K. A. et al. The risk of breast cancer in BRCA1 and BRCA2 mutation carriers without a first-degree relative with breast cancer. <i>Clin. Genet</i>. <b>93</b>, 1063–1068 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/cge.13191" data-track-item_id="10.1111/cge.13191" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fcge.13191" aria-label="Article reference 5" data-doi="10.1111/cge.13191">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXnsFSns7k%3D" aria-label="CAS reference 5">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29206279" aria-label="PubMed reference 5">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20risk%20of%20breast%20cancer%20in%20BRCA1%20and%20BRCA2%20mutation%20carriers%20without%20a%20first-degree%20relative%20with%20breast%20cancer&amp;journal=Clin.%20Genet&amp;doi=10.1111%2Fcge.13191&amp;volume=93&amp;pages=1063-1068&amp;publication_year=2018&amp;author=Metcalfe%2CKA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Li, T. et al. Total genetic contribution assessment across the human genome. <i>Nat. Commun.</i> <b>12</b>, 2845 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-021-23124-w" data-track-item_id="10.1038/s41467-021-23124-w" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-021-23124-w" aria-label="Article reference 6" data-doi="10.1038/s41467-021-23124-w">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFahtL%2FI" aria-label="CAS reference 6">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33990588" aria-label="PubMed reference 6">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8121943" aria-label="PubMed Central reference 6">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Total%20genetic%20contribution%20assessment%20across%20the%20human%20genome&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-23124-w&amp;volume=12&amp;publication_year=2021&amp;author=Li%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Mars, N. et al. Polygenic and clinical risk scores and their impact on age at onset and prediction of cardiometabolic diseases and common cancers. <i>Nat. Med.</i> <b>26</b>, 549–557 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-020-0800-0" data-track-item_id="10.1038/s41591-020-0800-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-020-0800-0" aria-label="Article reference 7" data-doi="10.1038/s41591-020-0800-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXmvVCiu7k%3D" aria-label="CAS reference 7">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32273609" aria-label="PubMed reference 7">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20and%20clinical%20risk%20scores%20and%20their%20impact%20on%20age%20at%20onset%20and%20prediction%20of%20cardiometabolic%20diseases%20and%20common%20cancers&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-020-0800-0&amp;volume=26&amp;pages=549-557&amp;publication_year=2020&amp;author=Mars%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Sakaue, S. et al. Trans-biobank analysis with 676,000 individuals elucidates the association of polygenic risk scores of complex traits with human lifespan. <i>Nat. Med.</i> <b>26</b>, 542–548 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-020-0785-8" data-track-item_id="10.1038/s41591-020-0785-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-020-0785-8" aria-label="Article reference 8" data-doi="10.1038/s41591-020-0785-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXlsVeisLY%3D" aria-label="CAS reference 8">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32251405" aria-label="PubMed reference 8">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Trans-biobank%20analysis%20with%20676%2C000%20individuals%20elucidates%20the%20association%20of%20polygenic%20risk%20scores%20of%20complex%20traits%20with%20human%20lifespan&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-020-0785-8&amp;volume=26&amp;pages=542-548&amp;publication_year=2020&amp;author=Sakaue%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Meisner, A. et al. Combined utility of 25 disease and risk factor polygenic risk scores for stratifying risk of all-cause mortality. <i>Am. J. Hum. Genet.</i> <b>107</b>, 418–431 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2020.07.002" data-track-item_id="10.1016/j.ajhg.2020.07.002" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2020.07.002" aria-label="Article reference 9" data-doi="10.1016/j.ajhg.2020.07.002">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFKgs7fI" aria-label="CAS reference 9">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32758451" aria-label="PubMed reference 9">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7477009" aria-label="PubMed Central reference 9">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Combined%20utility%20of%2025%20disease%20and%20risk%20factor%20polygenic%20risk%20scores%20for%20stratifying%20risk%20of%20all-cause%20mortality&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2020.07.002&amp;volume=107&amp;pages=418-431&amp;publication_year=2020&amp;author=Meisner%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Timmers, P. R. et al. Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances. <i>eLife</i> <b>8</b>, e39856 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7554/eLife.39856" data-track-item_id="10.7554/eLife.39856" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.39856" aria-label="Article reference 10" data-doi="10.7554/eLife.39856">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30642433" aria-label="PubMed reference 10">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6333444" aria-label="PubMed Central reference 10">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomics%20of%201%20million%20parent%20lifespans%20implicates%20novel%20pathways%20and%20common%20diseases%20and%20distinguishes%20survival%20chances&amp;journal=eLife&amp;doi=10.7554%2FeLife.39856&amp;volume=8&amp;publication_year=2019&amp;author=Timmers%2CPR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. <i>Lancet</i> <b>396</b>, 1223–1249 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(20)30752-2" data-track-item_id="10.1016/S0140-6736(20)30752-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2820%2930752-2" aria-label="Article reference 11" data-doi="10.1016/S0140-6736(20)30752-2">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20burden%20of%2087%20risk%20factors%20in%20204%20countries%20and%20territories%2C%201990%E2%80%932019%3A%20a%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202019&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2820%2930752-2&amp;volume=396&amp;pages=1223-1249&amp;publication_year=2020&amp;author=Murray%2CCJL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Marwaha, S., Knowles, J. W. &amp; Ashley, E. A. A guide for the diagnosis of rare and undiagnosed disease: beyond the exome. <i>Genome Med.</i> <b>14</b>, 23 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13073-022-01026-w" data-track-item_id="10.1186/s13073-022-01026-w" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13073-022-01026-w" aria-label="Article reference 12" data-doi="10.1186/s13073-022-01026-w">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35220969" aria-label="PubMed reference 12">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8883622" aria-label="PubMed Central reference 12">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20guide%20for%20the%20diagnosis%20of%20rare%20and%20undiagnosed%20disease%3A%20beyond%20the%20exome&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-022-01026-w&amp;volume=14&amp;publication_year=2022&amp;author=Marwaha%2CS&amp;author=Knowles%2CJW&amp;author=Ashley%2CEA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Brown, G. R. et al. A review of inherited cancer susceptibility syndromes. <i>JAAPA</i> <b>33</b>, 10–16 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1097/01.JAA.0000721648.46099.2c" data-track-item_id="10.1097/01.JAA.0000721648.46099.2c" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1097%2F01.JAA.0000721648.46099.2c" aria-label="Article reference 13" data-doi="10.1097/01.JAA.0000721648.46099.2c">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33234888" aria-label="PubMed reference 13">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20review%20of%20inherited%20cancer%20susceptibility%20syndromes&amp;journal=JAAPA&amp;doi=10.1097%2F01.JAA.0000721648.46099.2c&amp;volume=33&amp;pages=10-16&amp;publication_year=2020&amp;author=Brown%2CGR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Manchanda, R. &amp; Gaba, F. Population-based testing for primary prevention: a systematic review. <i>Cancers</i> <b>10</b>, E424 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3390/cancers10110424" data-track-item_id="10.3390/cancers10110424" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3390%2Fcancers10110424" aria-label="Article reference 14" data-doi="10.3390/cancers10110424">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30400647" aria-label="PubMed reference 14">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Population-based%20testing%20for%20primary%20prevention%3A%20a%20systematic%20review&amp;journal=Cancers&amp;doi=10.3390%2Fcancers10110424&amp;volume=10&amp;publication_year=2018&amp;author=Manchanda%2CR&amp;author=Gaba%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Kullo, I. J. et al. Polygenic scores in biomedical research. <i>Nat. Rev. Genet</i>. (2022) <a href="https://doi.org/10.1038/s41576-022-00470-z" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41576-022-00470-z">https://doi.org/10.1038/s41576-022-00470-z</a> (2022).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Sun, L. et al. Polygenic risk scores in cardiovascular risk prediction: A cohort study and modelling analyses. <i>PLoS Med.</i> <b>18</b>, e1003498 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pmed.1003498" data-track-item_id="10.1371/journal.pmed.1003498" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pmed.1003498" aria-label="Article reference 16" data-doi="10.1371/journal.pmed.1003498">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFWqu7k%3D" aria-label="CAS reference 16">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33444330" aria-label="PubMed reference 16">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7808664" aria-label="PubMed Central reference 16">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%20in%20cardiovascular%20risk%20prediction%3A%20A%20cohort%20study%20and%20modelling%20analyses&amp;journal=PLoS%20Med.&amp;doi=10.1371%2Fjournal.pmed.1003498&amp;volume=18&amp;publication_year=2021&amp;author=Sun%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Lewis, C. M. &amp; Vassos, E. Polygenic risk scores: from research tools to clinical instruments. <i>Genome Med.</i> <b>12</b>, 44 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13073-020-00742-5" data-track-item_id="10.1186/s13073-020-00742-5" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13073-020-00742-5" aria-label="Article reference 17" data-doi="10.1186/s13073-020-00742-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32423490" aria-label="PubMed reference 17">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7236300" aria-label="PubMed Central reference 17">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%3A%20from%20research%20tools%20to%20clinical%20instruments&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-020-00742-5&amp;volume=12&amp;publication_year=2020&amp;author=Lewis%2CCM&amp;author=Vassos%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Wand, H. et al. Improving reporting standards for polygenic scores in risk prediction studies. <i>Nature</i> <b>591</b>, 211–219 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-021-03243-6" data-track-item_id="10.1038/s41586-021-03243-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-03243-6" aria-label="Article reference 18" data-doi="10.1038/s41586-021-03243-6">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXmt1ems7Y%3D" aria-label="CAS reference 18">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33692554" aria-label="PubMed reference 18">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8609771" aria-label="PubMed Central reference 18">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Improving%20reporting%20standards%20for%20polygenic%20scores%20in%20risk%20prediction%20studies&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03243-6&amp;volume=591&amp;pages=211-219&amp;publication_year=2021&amp;author=Wand%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Musunuru, K. et al. In vivo CRISPR base editing of PCSK9 durably lowers cholesterol in primates. <i>Nature</i> <b>593</b>, 429–434 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-021-03534-y" data-track-item_id="10.1038/s41586-021-03534-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-03534-y" aria-label="Article reference 19" data-doi="10.1038/s41586-021-03534-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFanur3I" aria-label="CAS reference 19">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34012082" aria-label="PubMed reference 19">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vivo%20CRISPR%20base%20editing%20of%20PCSK9%20durably%20lowers%20cholesterol%20in%20primates&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03534-y&amp;volume=593&amp;pages=429-434&amp;publication_year=2021&amp;author=Musunuru%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Frangoul, H. et al. CRISPR-Cas9 gene editing for sickle cell disease and β-thalassemia. <i>N. Engl. J. Med.</i> <b>384</b>, 252–260 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa2031054" data-track-item_id="10.1056/NEJMoa2031054" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa2031054" aria-label="Article reference 20" data-doi="10.1056/NEJMoa2031054">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVKhs7c%3D" aria-label="CAS reference 20">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33283989" aria-label="PubMed reference 20">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-Cas9%20gene%20editing%20for%20sickle%20cell%20disease%20and%20%CE%B2-thalassemia&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa2031054&amp;volume=384&amp;pages=252-260&amp;publication_year=2021&amp;author=Frangoul%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Rim, J. H., Gopalappa, R. &amp; Gee, H. Y. CRISPR-Cas9 in vivo gene editing for transthyretin amyloidosis. <i>N. Engl. J. Med.</i> <b>385</b>, 1722 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34706180" aria-label="PubMed reference 21">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-Cas9%20in%20vivo%20gene%20editing%20for%20transthyretin%20amyloidosis&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=385&amp;publication_year=2021&amp;author=Rim%2CJH&amp;author=Gopalappa%2CR&amp;author=Gee%2CHY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Turley, P. et al. Problems with using polygenic scores to select embryos. <i>N. Engl. J. Med.</i> <b>385</b>, 78–86 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMsr2105065" data-track-item_id="10.1056/NEJMsr2105065" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMsr2105065" aria-label="Article reference 22" data-doi="10.1056/NEJMsr2105065">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34192436" aria-label="PubMed reference 22">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8387884" aria-label="PubMed Central reference 22">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Problems%20with%20using%20polygenic%20scores%20to%20select%20embryos&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMsr2105065&amp;volume=385&amp;pages=78-86&amp;publication_year=2021&amp;author=Turley%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Lencz, T. et al. Utility of polygenic embryo screening for disease depends on the selection strategy. <i>eLife</i> <b>10</b>, e64716 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7554/eLife.64716" data-track-item_id="10.7554/eLife.64716" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.64716" aria-label="Article reference 23" data-doi="10.7554/eLife.64716">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XotVSltrk%3D" aria-label="CAS reference 23">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34635206" aria-label="PubMed reference 23">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8510582" aria-label="PubMed Central reference 23">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Utility%20of%20polygenic%20embryo%20screening%20for%20disease%20depends%20on%20the%20selection%20strategy&amp;journal=eLife&amp;doi=10.7554%2FeLife.64716&amp;volume=10&amp;publication_year=2021&amp;author=Lencz%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Karavani, E. et al. Screening human embryos for polygenic traits has limited utility. <i>Cell</i> <b>179</b>, 1424–1435 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2019.10.033" data-track-item_id="10.1016/j.cell.2019.10.033" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.10.033" aria-label="Article reference 24" data-doi="10.1016/j.cell.2019.10.033">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1Crt7vL" aria-label="CAS reference 24">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31761530" aria-label="PubMed reference 24">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6957074" aria-label="PubMed Central reference 24">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Screening%20human%20embryos%20for%20polygenic%20traits%20has%20limited%20utility&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.10.033&amp;volume=179&amp;pages=1424-1435&amp;publication_year=2019&amp;author=Karavani%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Kumar, A. et al. Whole-genome risk prediction of common diseases in human preimplantation embryos. <i>Nat. Med.</i> <b>28</b>, 513–516 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-022-01735-0" data-track-item_id="10.1038/s41591-022-01735-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-022-01735-0" aria-label="Article reference 25" data-doi="10.1038/s41591-022-01735-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XnslSku74%3D" aria-label="CAS reference 25">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35314819" aria-label="PubMed reference 25">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8938270" aria-label="PubMed Central reference 25">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-genome%20risk%20prediction%20of%20common%20diseases%20in%20human%20preimplantation%20embryos&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-022-01735-0&amp;volume=28&amp;pages=513-516&amp;publication_year=2022&amp;author=Kumar%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Johnston, J. &amp; Matthews, L. J. Polygenic embryo testing: understated ethics, unclear utility. <i>Nat. Med.</i> <b>28</b>, 446–448 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-022-01743-0" data-track-item_id="10.1038/s41591-022-01743-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-022-01743-0" aria-label="Article reference 26" data-doi="10.1038/s41591-022-01743-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XnslSkurk%3D" aria-label="CAS reference 26">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35314818" aria-label="PubMed reference 26">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9208380" aria-label="PubMed Central reference 26">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20embryo%20testing%3A%20understated%20ethics%2C%20unclear%20utility&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-022-01743-0&amp;volume=28&amp;pages=446-448&amp;publication_year=2022&amp;author=Johnston%2CJ&amp;author=Matthews%2CLJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. <i>Lancet</i> <b>396</b>, 1204–1222 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(20)30925-9" data-track-item_id="10.1016/S0140-6736(20)30925-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2820%2930925-9" aria-label="Article reference 27" data-doi="10.1016/S0140-6736(20)30925-9">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20burden%20of%20369%20diseases%20and%20injuries%20in%20204%20countries%20and%20territories%2C%201990%E2%80%932019%3A%20a%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202019&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2820%2930925-9&amp;volume=396&amp;pages=1204-1222&amp;publication_year=2020&amp;author=Vos%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Kurki, M. I. et al. FinnGen: unique genetic insights from combining isolated population and national health register data. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2022.03.03.22271360" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2022.03.03.22271360">https://doi.org/10.1101/2022.03.03.22271360</a> (2022).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. <i>PLoS Med.</i> <b>12</b>, e1001779 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pmed.1001779" data-track-item_id="10.1371/journal.pmed.1001779" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pmed.1001779" aria-label="Article reference 29" data-doi="10.1371/journal.pmed.1001779">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25826379" aria-label="PubMed reference 29">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4380465" aria-label="PubMed Central reference 29">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=UK%20Biobank%3A%20an%20open%20access%20resource%20for%20identifying%20the%20causes%20of%20a%20wide%20range%20of%20complex%20diseases%20of%20middle%20and%20old%20age&amp;journal=PLoS%20Med.&amp;doi=10.1371%2Fjournal.pmed.1001779&amp;volume=12&amp;publication_year=2015&amp;author=Sudlow%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Wang, G., Sarkar, A., Carbonetto, P. &amp; Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. <i>J. R. Stat. Soc. Ser. B Stat. Methodol.</i> <b>82</b>, 1273–1300 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/rssb.12388" data-track-item_id="10.1111/rssb.12388" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Frssb.12388" aria-label="Article reference 30" data-doi="10.1111/rssb.12388">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20simple%20new%20approach%20to%20variable%20selection%20in%20regression%2C%20with%20application%20to%20genetic%20fine%20mapping&amp;journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B%20Stat.%20Methodol.&amp;doi=10.1111%2Frssb.12388&amp;volume=82&amp;pages=1273-1300&amp;publication_year=2020&amp;author=Wang%2CG&amp;author=Sarkar%2CA&amp;author=Carbonetto%2CP&amp;author=Stephens%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Kanai, M. et al. Insights from complex trait fine-mapping across diverse populations. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2021.09.03.21262975" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.09.03.21262975">https://doi.org/10.1101/2021.09.03.21262975</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Matejcic, M. et al. Germline variation at 8q24 and prostate cancer risk in men of European ancestry. <i>Nat. Commun.</i> <b>9</b>, 4616 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-018-06863-1" data-track-item_id="10.1038/s41467-018-06863-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-018-06863-1" aria-label="Article reference 32" data-doi="10.1038/s41467-018-06863-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30397198" aria-label="PubMed reference 32">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6218483" aria-label="PubMed Central reference 32">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Germline%20variation%20at%208q24%20and%20prostate%20cancer%20risk%20in%20men%20of%20European%20ancestry&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-06863-1&amp;volume=9&amp;publication_year=2018&amp;author=Matejcic%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Schlaepfer, I. R. et al. The CHRNA5/A3/B4 gene cluster variability as an important determinant of early alcohol and tobacco initiation in young adults. <i>Biol. Psychiat.</i> <b>63</b>, 1039–1046 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.biopsych.2007.10.024" data-track-item_id="10.1016/j.biopsych.2007.10.024" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.biopsych.2007.10.024" aria-label="Article reference 33" data-doi="10.1016/j.biopsych.2007.10.024">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtVagsro%3D" aria-label="CAS reference 33">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18163978" aria-label="PubMed reference 33">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20CHRNA5%2FA3%2FB4%20gene%20cluster%20variability%20as%20an%20important%20determinant%20of%20early%20alcohol%20and%20tobacco%20initiation%20in%20young%20adults&amp;journal=Biol.%20Psychiat.&amp;doi=10.1016%2Fj.biopsych.2007.10.024&amp;volume=63&amp;pages=1039-1046&amp;publication_year=2008&amp;author=Schlaepfer%2CIR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Joshi, P. K. et al. Variants near CHRNA3/5 and APOE have age- and sex-related effects on human lifespan. <i>Nat. Commun.</i> <b>7</b>, 11174 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ncomms11174" data-track-item_id="10.1038/ncomms11174" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms11174" aria-label="Article reference 34" data-doi="10.1038/ncomms11174">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xlt1Kms7Y%3D" aria-label="CAS reference 34">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27029810" aria-label="PubMed reference 34">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5438072" aria-label="PubMed Central reference 34">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Variants%20near%20CHRNA3%2F5%20and%20APOE%20have%20age-%20and%20sex-related%20effects%20on%20human%20lifespan&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms11174&amp;volume=7&amp;publication_year=2016&amp;author=Joshi%2CPK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Green, R. C. et al. ACMG recommendations for reporting of incidental findings in clinical exome and genome sequencing. <i>Genet. Med.</i> <b>15</b>, 565–574 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/gim.2013.73" data-track-item_id="10.1038/gim.2013.73" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fgim.2013.73" aria-label="Article reference 35" data-doi="10.1038/gim.2013.73">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVKku73K" aria-label="CAS reference 35">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23788249" aria-label="PubMed reference 35">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3727274" aria-label="PubMed Central reference 35">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=ACMG%20recommendations%20for%20reporting%20of%20incidental%20findings%20in%20clinical%20exome%20and%20genome%20sequencing&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fgim.2013.73&amp;volume=15&amp;pages=565-574&amp;publication_year=2013&amp;author=Green%2CRC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Miller, D. T. et al. ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG). <i>Genet. Med.</i> <b>23</b>, 1381–1390 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41436-021-01172-3" data-track-item_id="10.1038/s41436-021-01172-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41436-021-01172-3" aria-label="Article reference 36" data-doi="10.1038/s41436-021-01172-3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34012068" aria-label="PubMed reference 36">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=ACMG%20SF%20v3.0%20list%20for%20reporting%20of%20secondary%20findings%20in%20clinical%20exome%20and%20genome%20sequencing%3A%20a%20policy%20statement%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics%20%28ACMG%29&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fs41436-021-01172-3&amp;volume=23&amp;pages=1381-1390&amp;publication_year=2021&amp;author=Miller%2CDT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. <i>Nucleic Acids Res.</i> <b>42</b>, D980–D985 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkt1113" data-track-item_id="10.1093/nar/gkt1113" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkt1113" aria-label="Article reference 37" data-doi="10.1093/nar/gkt1113">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXoslGh" aria-label="CAS reference 37">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24234437" aria-label="PubMed reference 37">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=ClinVar%3A%20public%20archive%20of%20relationships%20among%20sequence%20variation%20and%20human%20phenotype&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkt1113&amp;volume=42&amp;pages=D980-D985&amp;publication_year=2014&amp;author=Landrum%2CMJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Spurdle, A. B. et al. ENIGMA–evidence-based network for the interpretation of germline mutant alleles: an international initiative to evaluate risk and clinical significance associated with sequence variation in BRCA1 and BRCA2 genes. <i>Hum. Mutat.</i> <b>33</b>, 2–7 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/humu.21628" data-track-item_id="10.1002/humu.21628" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2Fhumu.21628" aria-label="Article reference 38" data-doi="10.1002/humu.21628">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1aju7%2FI" aria-label="CAS reference 38">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21990146" aria-label="PubMed reference 38">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=ENIGMA%E2%80%93evidence-based%20network%20for%20the%20interpretation%20of%20germline%20mutant%20alleles%3A%20an%20international%20initiative%20to%20evaluate%20risk%20and%20clinical%20significance%20associated%20with%20sequence%20variation%20in%20BRCA1%20and%20BRCA2%20genes&amp;journal=Hum.%20Mutat.&amp;doi=10.1002%2Fhumu.21628&amp;volume=33&amp;pages=2-7&amp;publication_year=2012&amp;author=Spurdle%2CAB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Liu, J. Z. et al. Association analyses identify 38 susceptibility loci for inflammatory bowel disease and highlight shared genetic risk across populations. <i>Nat. Genet.</i> <b>47</b>, 979–986 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ng.3359" data-track-item_id="10.1038/ng.3359" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3359" aria-label="Article reference 39" data-doi="10.1038/ng.3359">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1WltLvI" aria-label="CAS reference 39">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26192919" aria-label="PubMed reference 39">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4881818" aria-label="PubMed Central reference 39">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Association%20analyses%20identify%2038%20susceptibility%20loci%20for%20inflammatory%20bowel%20disease%20and%20highlight%20shared%20genetic%20risk%20across%20populations&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3359&amp;volume=47&amp;pages=979-986&amp;publication_year=2015&amp;author=Liu%2CJZ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Johnston, K. J. A. et al. Genome-wide association study of multisite chronic pain in UK Biobank. <i>PLoS Genet</i>. <a href="https://doi.org/10.1371/journal.pgen.1008164" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1371/journal.pgen.1008164">https://doi.org/10.1371/journal.pgen.1008164</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">BasuRay, S. PNPLA3-I148M: a problem of plenty in non-alcoholic fatty liver disease. <i>Adipocyte</i> <b>8</b>, 201–208 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1080/21623945.2019.1607423" data-track-item_id="10.1080/21623945.2019.1607423" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1080%2F21623945.2019.1607423" aria-label="Article reference 41" data-doi="10.1080/21623945.2019.1607423">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6768214" aria-label="PubMed Central reference 41">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=PNPLA3-I148M%3A%20a%20problem%20of%20plenty%20in%20non-alcoholic%20fatty%20liver%20disease&amp;journal=Adipocyte&amp;doi=10.1080%2F21623945.2019.1607423&amp;volume=8&amp;pages=201-208&amp;publication_year=2019&amp;author=BasuRay%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Liu, M. et al. Association studies of up to 1.2 million individuals yield new insights into the genetic etiology of tobacco and alcohol use. <i>Nat. Genet.</i> <b>51</b>, 237–244 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0307-5" data-track-item_id="10.1038/s41588-018-0307-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0307-5" aria-label="Article reference 42" data-doi="10.1038/s41588-018-0307-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvFSns7g%3D" aria-label="CAS reference 42">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30643251" aria-label="PubMed reference 42">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6358542" aria-label="PubMed Central reference 42">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Association%20studies%20of%20up%20to%201.2%20million%20individuals%20yield%20new%20insights%20into%20the%20genetic%20etiology%20of%20tobacco%20and%20alcohol%20use&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0307-5&amp;volume=51&amp;pages=237-244&amp;publication_year=2019&amp;author=Liu%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Chheda, H. et al. Whole-genome view of the consequences of a population bottleneck using 2926 genome sequences from Finland and United Kingdom. <i>Eur. J. Hum. Genet.</i> <b>25</b>, 477–484 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ejhg.2016.205" data-track-item_id="10.1038/ejhg.2016.205" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fejhg.2016.205" aria-label="Article reference 43" data-doi="10.1038/ejhg.2016.205">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28145424" aria-label="PubMed reference 43">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5346294" aria-label="PubMed Central reference 43">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-genome%20view%20of%20the%20consequences%20of%20a%20population%20bottleneck%20using%202926%20genome%20sequences%20from%20Finland%20and%20United%20Kingdom&amp;journal=Eur.%20J.%20Hum.%20Genet.&amp;doi=10.1038%2Fejhg.2016.205&amp;volume=25&amp;pages=477-484&amp;publication_year=2017&amp;author=Chheda%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Lim, E. T. et al. Distribution and medical impact of loss-of-function variants in the Finnish founder population. <i>PLoS Genet.</i> <b>10</b>, e1004494 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pgen.1004494" data-track-item_id="10.1371/journal.pgen.1004494" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1004494" aria-label="Article reference 44" data-doi="10.1371/journal.pgen.1004494">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25078778" aria-label="PubMed reference 44">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4117444" aria-label="PubMed Central reference 44">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Distribution%20and%20medical%20impact%20of%20loss-of-function%20variants%20in%20the%20Finnish%20founder%20population&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1004494&amp;volume=10&amp;publication_year=2014&amp;author=Lim%2CET">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Ruotsalainen, S. E. et al. Loss-of-function of MFGE8 and protection against coronary atherosclerosis. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2021.06.23.21259381" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.06.23.21259381">https://doi.org/10.1101/2021.06.23.21259381</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Nikpay, M. et al. A comprehensive 1,000 Genomes-based genome-wide association meta-analysis of coronary artery disease. <i>Nat. Genet.</i> <b>47</b>, 1121–1130 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ng.3396" data-track-item_id="10.1038/ng.3396" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3396" aria-label="Article reference 46" data-doi="10.1038/ng.3396">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsV2lt7zO" aria-label="CAS reference 46">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26343387" aria-label="PubMed reference 46">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4589895" aria-label="PubMed Central reference 46">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20comprehensive%201%2C000%20Genomes-based%20genome-wide%20association%20meta-analysis%20of%20coronary%20artery%20disease&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3396&amp;volume=47&amp;pages=1121-1130&amp;publication_year=2015&amp;author=Nikpay%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Mahajan, A. et al. Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps. <i>Nat. Genet.</i> <b>50</b>, 1505–1513 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0241-6" data-track-item_id="10.1038/s41588-018-0241-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0241-6" aria-label="Article reference 47" data-doi="10.1038/s41588-018-0241-6">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVCitLrM" aria-label="CAS reference 47">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30297969" aria-label="PubMed reference 47">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6287706" aria-label="PubMed Central reference 47">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Fine-mapping%20type%202%20diabetes%20loci%20to%20single-variant%20resolution%20using%20high-density%20imputation%20and%20islet-specific%20epigenome%20maps&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0241-6&amp;volume=50&amp;pages=1505-1513&amp;publication_year=2018&amp;author=Mahajan%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Lee, J. J. et al. Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals. <i>Nat. Genet.</i> <b>50</b>, 1112–1121 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0147-3" data-track-item_id="10.1038/s41588-018-0147-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0147-3" aria-label="Article reference 48" data-doi="10.1038/s41588-018-0147-3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlOis7zI" aria-label="CAS reference 48">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30038396" aria-label="PubMed reference 48">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6393768" aria-label="PubMed Central reference 48">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20discovery%20and%20polygenic%20prediction%20from%20a%20genome-wide%20association%20study%20of%20educational%20attainment%20in%201.1%20million%20individuals&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0147-3&amp;volume=50&amp;pages=1112-1121&amp;publication_year=2018&amp;author=Lee%2CJJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Wray, N. R. et al. Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression. <i>Nat. Genet.</i> <b>50</b>, 668–681 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0090-3" data-track-item_id="10.1038/s41588-018-0090-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0090-3" aria-label="Article reference 49" data-doi="10.1038/s41588-018-0090-3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXosVCks74%3D" aria-label="CAS reference 49">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29700475" aria-label="PubMed reference 49">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5934326" aria-label="PubMed Central reference 49">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20association%20analyses%20identify%2044%20risk%20variants%20and%20refine%20the%20genetic%20architecture%20of%20major%20depression&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0090-3&amp;volume=50&amp;pages=668-681&amp;publication_year=2018&amp;author=Wray%2CNR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Shiffman, J. &amp; Shawar, Y. R. Strengthening accountability of the global health metrics enterprise. <i>Lancet</i> <b>395</b>, 1452–1456 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(20)30416-5" data-track-item_id="10.1016/S0140-6736(20)30416-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2820%2930416-5" aria-label="Article reference 50" data-doi="10.1016/S0140-6736(20)30416-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXnvFertL8%3D" aria-label="CAS reference 50">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32305072" aria-label="PubMed reference 50">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7162633" aria-label="PubMed Central reference 50">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Strengthening%20accountability%20of%20the%20global%20health%20metrics%20enterprise&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2820%2930416-5&amp;volume=395&amp;pages=1452-1456&amp;publication_year=2020&amp;author=Shiffman%2CJ&amp;author=Shawar%2CYR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Mukamel, R. E. et al. Protein-coding repeat polymorphisms strongly shape diverse human phenotypes. <i>Science</i> <b>373</b>, 1499–1505 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.abg8289" data-track-item_id="10.1126/science.abg8289" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.abg8289" aria-label="Article reference 51" data-doi="10.1126/science.abg8289">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFOisbjF" aria-label="CAS reference 51">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34554798" aria-label="PubMed reference 51">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8549062" aria-label="PubMed Central reference 51">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Protein-coding%20repeat%20polymorphisms%20strongly%20shape%20diverse%20human%20phenotypes&amp;journal=Science&amp;doi=10.1126%2Fscience.abg8289&amp;volume=373&amp;pages=1499-1505&amp;publication_year=2021&amp;author=Mukamel%2CRE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Giambartolomei, C. et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. <i>PLoS Genet.</i> <b>10</b>, e1004383 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pgen.1004383" data-track-item_id="10.1371/journal.pgen.1004383" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1004383" aria-label="Article reference 52" data-doi="10.1371/journal.pgen.1004383">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24830394" aria-label="PubMed reference 52">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4022491" aria-label="PubMed Central reference 52">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Bayesian%20test%20for%20colocalisation%20between%20pairs%20of%20genetic%20association%20studies%20using%20summary%20statistics&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1004383&amp;volume=10&amp;publication_year=2014&amp;author=Giambartolomei%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Guo, J. et al. Quantifying genetic heterogeneity between continental populations for human height and body mass index. <i>Sci. Rep.</i> <b>11</b>, 5240 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41598-021-84739-z" data-track-item_id="10.1038/s41598-021-84739-z" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-021-84739-z" aria-label="Article reference 53" data-doi="10.1038/s41598-021-84739-z">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXlvF2gur4%3D" aria-label="CAS reference 53">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33664403" aria-label="PubMed reference 53">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7933291" aria-label="PubMed Central reference 53">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Quantifying%20genetic%20heterogeneity%20between%20continental%20populations%20for%20human%20height%20and%20body%20mass%20index&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-021-84739-z&amp;volume=11&amp;publication_year=2021&amp;author=Guo%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Kuchenbaecker, K. et al. The transferability of lipid loci across African, Asian and European cohorts. <i>Nat. Commun.</i> <b>10</b>, 4330 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-019-12026-7" data-track-item_id="10.1038/s41467-019-12026-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-12026-7" aria-label="Article reference 54" data-doi="10.1038/s41467-019-12026-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31551420" aria-label="PubMed reference 54">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6760173" aria-label="PubMed Central reference 54">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20transferability%20of%20lipid%20loci%20across%20African%2C%20Asian%20and%20European%20cohorts&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-12026-7&amp;volume=10&amp;publication_year=2019&amp;author=Kuchenbaecker%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Huang, Q. Q. et al. Transferability of genetic loci and polygenic scores for cardiometabolic traits in British Pakistani and Bangladeshi individuals. <i>Nat. Commun.</i> <b>13</b>, 4664 (2022). <a href="https://doi.org/10.1038/s41467-022-32095-5" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41467-022-32095-5">https://doi.org/10.1038/s41467-022-32095-5</a></p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Shi, H. et al. Population-specific causal disease effect sizes in functionally important regions impacted by selection. <i>Nat. Commun.</i> <b>12</b>, 1–15 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Population-specific%20causal%20disease%20effect%20sizes%20in%20functionally%20important%20regions%20impacted%20by%20selection&amp;journal=Nat.%20Commun.&amp;volume=12&amp;pages=1-15&amp;publication_year=2021&amp;author=Shi%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Patel, R. A. et al. Effect sizes of causal variants for gene expression and complex traits differ between populations. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/2021.12.06.471235" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.12.06.471235">https://doi.org/10.1101/2021.12.06.471235</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. <i>Nat. Genet.</i> <b>51</b>, 584–591 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-019-0379-x" data-track-item_id="10.1038/s41588-019-0379-x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-019-0379-x" aria-label="Article reference 58" data-doi="10.1038/s41588-019-0379-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslGltLw%3D" aria-label="CAS reference 58">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30926966" aria-label="PubMed reference 58">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6563838" aria-label="PubMed Central reference 58">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20use%20of%20current%20polygenic%20risk%20scores%20may%20exacerbate%20health%20disparities&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0379-x&amp;volume=51&amp;pages=584-591&amp;publication_year=2019&amp;author=Martin%2CAR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. <i>Nature</i> <b>581</b>, 434–443 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-020-2308-7" data-track-item_id="10.1038/s41586-020-2308-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-2308-7" aria-label="Article reference 59" data-doi="10.1038/s41586-020-2308-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVanu7jF" aria-label="CAS reference 59">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32461654" aria-label="PubMed reference 59">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334197" aria-label="PubMed Central reference 59">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141%2C456%20humans&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2308-7&amp;volume=581&amp;pages=434-443&amp;publication_year=2020&amp;author=Karczewski%2CKJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">McLaren, W. et al. The ensembl variant effect predictor. <i>Genome Biol.</i> <b>17</b>, 122 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-016-0974-4" data-track-item_id="10.1186/s13059-016-0974-4" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-016-0974-4" aria-label="Article reference 60" data-doi="10.1186/s13059-016-0974-4">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27268795" aria-label="PubMed reference 60">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4893825" aria-label="PubMed Central reference 60">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20ensembl%20variant%20effect%20predictor&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-016-0974-4&amp;volume=17&amp;publication_year=2016&amp;author=McLaren%2CW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Laugesen, K. et al. Nordic health registry-based research: a review of health care systems and key registries. <i>Clin. Epidemiol.</i> <b>13</b>, 533–554 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.2147/CLEP.S314959" data-track-item_id="10.2147/CLEP.S314959" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.2147%2FCLEP.S314959" aria-label="Article reference 61" data-doi="10.2147/CLEP.S314959">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34321928" aria-label="PubMed reference 61">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8302231" aria-label="PubMed Central reference 61">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Nordic%20health%20registry-based%20research%3A%20a%20review%20of%20health%20care%20systems%20and%20key%20registries&amp;journal=Clin.%20Epidemiol.&amp;doi=10.2147%2FCLEP.S314959&amp;volume=13&amp;pages=533-554&amp;publication_year=2021&amp;author=Laugesen%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Vuori, M. A. et al. The validity of heart failure diagnoses in the Finnish Hospital Discharge Register. <i>Scand. J. Public Health</i> <a href="https://doi.org/10.1177/1403494819847051" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1177/1403494819847051">https://doi.org/10.1177/1403494819847051</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Leinonen, M. K., Miettinen, J., Heikkinen, S., Pitkäniemi, J. &amp; Malila, N. Quality measures of the population-based finnish cancer registry indicate sound data quality for solid malignant tumours. <i>Eur. J. Cancer</i> <b>77</b>, 31–39 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ejca.2017.02.017" data-track-item_id="10.1016/j.ejca.2017.02.017" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ejca.2017.02.017" aria-label="Article reference 63" data-doi="10.1016/j.ejca.2017.02.017">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28350996" aria-label="PubMed reference 63">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Quality%20measures%20of%20the%20population-based%20finnish%20cancer%20registry%20indicate%20sound%20data%20quality%20for%20solid%20malignant%20tumours&amp;journal=Eur.%20J.%20Cancer&amp;doi=10.1016%2Fj.ejca.2017.02.017&amp;volume=77&amp;pages=31-39&amp;publication_year=2017&amp;author=Leinonen%2CMK&amp;author=Miettinen%2CJ&amp;author=Heikkinen%2CS&amp;author=Pitk%C3%A4niemi%2CJ&amp;author=Malila%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Sund, R. et al. Comparing properties of audit data and routinely collected register data in case of performance assessment of hip fracture treatment in Finland. <i>Methods Inf. Med.</i> <b>46</b>, 558–566 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1160/ME0382" data-track-item_id="10.1160/ME0382" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1160%2FME0382" aria-label="Article reference 64" data-doi="10.1160/ME0382">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD2snhsVGlsg%3D%3D" aria-label="CAS reference 64">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17938779" aria-label="PubMed reference 64">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Comparing%20properties%20of%20audit%20data%20and%20routinely%20collected%20register%20data%20in%20case%20of%20performance%20assessment%20of%20hip%20fracture%20treatment%20in%20Finland&amp;journal=Methods%20Inf.%20Med.&amp;doi=10.1160%2FME0382&amp;volume=46&amp;pages=558-566&amp;publication_year=2007&amp;author=Sund%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Sund, R. Quality of the Finnish Hospital Discharge Register: a systematic review. <i>Scand. J. Public Health</i> <b>40</b>, 505–515 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1177/1403494812456637" data-track-item_id="10.1177/1403494812456637" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1177%2F1403494812456637" aria-label="Article reference 65" data-doi="10.1177/1403494812456637">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22899561" aria-label="PubMed reference 65">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Quality%20of%20the%20Finnish%20Hospital%20Discharge%20Register%3A%20a%20systematic%20review&amp;journal=Scand.%20J.%20Public%20Health&amp;doi=10.1177%2F1403494812456637&amp;volume=40&amp;pages=505-515&amp;publication_year=2012&amp;author=Sund%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Haukka, J. Finnish health and social welfare registers in epidemiological research. <i>Norsk Epidemiologi</i> <b>14</b>, 113−120 (2004).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. <i>Nature</i> <b>562</b>, 203–209 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-018-0579-z" data-track-item_id="10.1038/s41586-018-0579-z" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-018-0579-z" aria-label="Article reference 67" data-doi="10.1038/s41586-018-0579-z">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvV2qsbvI" aria-label="CAS reference 67">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30305743" aria-label="PubMed reference 67">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786975" aria-label="PubMed Central reference 67">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20UK%20Biobank%20resource%20with%20deep%20phenotyping%20and%20genomic%20data&amp;journal=Nature&amp;doi=10.1038%2Fs41586-018-0579-z&amp;volume=562&amp;pages=203-209&amp;publication_year=2018&amp;author=Bycroft%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Pärn, K., Nunez-Fontarnau, J., Isokallio M. A., Sipilä, T., Kilpeläinen, E., Palotie, A., Ripatti, S. &amp; Palta, P. Genotyping chip data lift-over to reference genome build GRCh38/hg38. protocols.io. <a href="https://doi.org/10.17504/protocols.io.xbhfij6" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.17504/protocols.io.xbhfij6">https://doi.org/10.17504/protocols.io.xbhfij6</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Pärn, K., Isokallio, M. A., Nunez-Fontarnau, J., Palotie, A., Ripatti, S. &amp; Palta, P. Genotype imputation workflow v3.0. protocols.io. <a href="https://doi.org/10.17504/protocols.io.xbgfijw" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.17504/protocols.io.xbgfijw">https://doi.org/10.17504/protocols.io.xbgfijw</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Ritari, J. et al. Increasing accuracy of HLA imputation by a population-specific reference panel in a FinnGen biobank cohort. <i>NAR Genomics Bioinforma.</i> <b>2</b>, lqaa030 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nargab/lqaa030" data-track-item_id="10.1093/nargab/lqaa030" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnargab%2Flqaa030" aria-label="Article reference 70" data-doi="10.1093/nargab/lqaa030">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=Increasing%20accuracy%20of%20HLA%20imputation%20by%20a%20population-specific%20reference%20panel%20in%20a%20FinnGen%20biobank%20cohort&amp;journal=NAR%20Genomics%20Bioinforma.&amp;doi=10.1093%2Fnargab%2Flqaa030&amp;volume=2&amp;publication_year=2020&amp;author=Ritari%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Zheng, X. et al. HIBAG—HLA genotype imputation with attribute bagging. <i>Pharmacogenomics J.</i> <b>14</b>, 192–200 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/tpj.2013.18" data-track-item_id="10.1038/tpj.2013.18" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Ftpj.2013.18" aria-label="Article reference 71" data-doi="10.1038/tpj.2013.18">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXot1Cmurw%3D" aria-label="CAS reference 71">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23712092" aria-label="PubMed reference 71">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=HIBAG%E2%80%94HLA%20genotype%20imputation%20with%20attribute%20bagging&amp;journal=Pharmacogenomics%20J.&amp;doi=10.1038%2Ftpj.2013.18&amp;volume=14&amp;pages=192-200&amp;publication_year=2014&amp;author=Zheng%2CX">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Zhou, W. et al. Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies. <i>Nat. Genet.</i> <b>50</b>, 1335–1341 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0184-y" data-track-item_id="10.1038/s41588-018-0184-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0184-y" aria-label="Article reference 72" data-doi="10.1038/s41588-018-0184-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFSqtbfN" aria-label="CAS reference 72">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30104761" aria-label="PubMed reference 72">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6119127" aria-label="PubMed Central reference 72">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficiently%20controlling%20for%20case-control%20imbalance%20and%20sample%20relatedness%20in%20large-scale%20genetic%20association%20studies&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0184-y&amp;volume=50&amp;pages=1335-1341&amp;publication_year=2018&amp;author=Zhou%2CW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Loh, P.-R. et al. Efficient Bayesian mixed-model analysis increases association power in large cohorts. <i>Nat. Genet.</i> <b>47</b>, 284–290 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ng.3190" data-track-item_id="10.1038/ng.3190" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3190" aria-label="Article reference 73" data-doi="10.1038/ng.3190">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFCntb4%3D" aria-label="CAS reference 73">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25642633" aria-label="PubMed reference 73">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342297" aria-label="PubMed Central reference 73">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20Bayesian%20mixed-model%20analysis%20increases%20association%20power%20in%20large%20cohorts&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3190&amp;volume=47&amp;pages=284-290&amp;publication_year=2015&amp;author=Loh%2CP-R">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. <i>Am. J. Hum. Genet</i> <b>81</b>, 559–575 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1086/519795" data-track-item_id="10.1086/519795" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1086%2F519795" aria-label="Article reference 74" data-doi="10.1086/519795">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL" aria-label="CAS reference 74">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17701901" aria-label="PubMed reference 74">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838" aria-label="PubMed Central reference 74">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;journal=Am.%20J.%20Hum.%20Genet&amp;doi=10.1086%2F519795&amp;volume=81&amp;pages=559-575&amp;publication_year=2007&amp;author=Purcell%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Ge, T., Chen, C.-Y., Ni, Y., Feng, Y.-C. A. &amp; Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. <i>Nat. Commun.</i> <b>10</b>, 1776 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-019-09718-5" data-track-item_id="10.1038/s41467-019-09718-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-09718-5" aria-label="Article reference 75" data-doi="10.1038/s41467-019-09718-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30992449" aria-label="PubMed reference 75">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467998" aria-label="PubMed Central reference 75">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20prediction%20via%20Bayesian%20regression%20and%20continuous%20shrinkage%20priors&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-09718-5&amp;volume=10&amp;publication_year=2019&amp;author=Ge%2CT&amp;author=Chen%2CC-Y&amp;author=Ni%2CY&amp;author=Feng%2CY-CA&amp;author=Smoller%2CJW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Heinze, G. &amp; Schemper, M. A solution to the problem of monotone likelihood in Cox regression. <i>Biometrics</i> <b>57</b>, 114–119 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/j.0006-341X.2001.00114.x" data-track-item_id="10.1111/j.0006-341X.2001.00114.x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.0006-341X.2001.00114.x" aria-label="Article reference 76" data-doi="10.1111/j.0006-341X.2001.00114.x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BD3MzmvVaqsQ%3D%3D" aria-label="CAS reference 76">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11252585" aria-label="PubMed reference 76">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20solution%20to%20the%20problem%20of%20monotone%20likelihood%20in%20Cox%20regression&amp;journal=Biometrics&amp;doi=10.1111%2Fj.0006-341X.2001.00114.x&amp;volume=57&amp;pages=114-119&amp;publication_year=2001&amp;author=Heinze%2CG&amp;author=Schemper%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Manichaikul, A. et al. Robust relationship inference in genome-wide association studies. <i>Bioinformatics</i> <b>26</b>, 2867–2873 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btq559" data-track-item_id="10.1093/bioinformatics/btq559" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq559" aria-label="Article reference 77" data-doi="10.1093/bioinformatics/btq559">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsVSlt7bK" aria-label="CAS reference 77">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20926424" aria-label="PubMed reference 77">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3025716" aria-label="PubMed Central reference 77">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20relationship%20inference%20in%20genome-wide%20association%20studies&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtq559&amp;volume=26&amp;pages=2867-2873&amp;publication_year=2010&amp;author=Manichaikul%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Stensrud, M. J. &amp; Hernán, M. A. Why test for proportional hazards? <i>JAMA</i> <b>323</b>, 1401–1402 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1001/jama.2020.1267" data-track-item_id="10.1001/jama.2020.1267" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1001%2Fjama.2020.1267" aria-label="Article reference 78" data-doi="10.1001/jama.2020.1267">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32167523" aria-label="PubMed reference 78">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=Why%20test%20for%20proportional%20hazards%3F&amp;journal=JAMA&amp;doi=10.1001%2Fjama.2020.1267&amp;volume=323&amp;pages=1401-1402&amp;publication_year=2020&amp;author=Stensrud%2CMJ&amp;author=Hern%C3%A1n%2CMA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Khosravi, A., Nazemipour, M., Shinozaki, T. &amp; Mansournia, M. A. Population attributable fraction in textbooks: time to revise. <i>Glob. Epidemiol.</i> <b>3</b>, 100062 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.gloepi.2021.100062" data-track-item_id="10.1016/j.gloepi.2021.100062" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.gloepi.2021.100062" aria-label="Article reference 79" data-doi="10.1016/j.gloepi.2021.100062">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=Population%20attributable%20fraction%20in%20textbooks%3A%20time%20to%20revise&amp;journal=Glob.%20Epidemiol.&amp;doi=10.1016%2Fj.gloepi.2021.100062&amp;volume=3&amp;publication_year=2021&amp;author=Khosravi%2CA&amp;author=Nazemipour%2CM&amp;author=Shinozaki%2CT&amp;author=Mansournia%2CMA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Rowe, A. K., Powell, K. E. &amp; Flanders, W. D. Why population attributable fractions can sum to more than one. <i>Am. J. Prev. Med.</i> <b>26</b>, 243–249 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.amepre.2003.12.007" data-track-item_id="10.1016/j.amepre.2003.12.007" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.amepre.2003.12.007" aria-label="Article reference 80" data-doi="10.1016/j.amepre.2003.12.007">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15026106" aria-label="PubMed reference 80">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=Why%20population%20attributable%20fractions%20can%20sum%20to%20more%20than%20one&amp;journal=Am.%20J.%20Prev.%20Med.&amp;doi=10.1016%2Fj.amepre.2003.12.007&amp;volume=26&amp;pages=243-249&amp;publication_year=2004&amp;author=Rowe%2CAK&amp;author=Powell%2CKE&amp;author=Flanders%2CWD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Witte, J. S., Visscher, P. M. &amp; Wray, N. R. The contribution of genetic variants to disease depends on the ruler. <i>Nat. Rev. Genet.</i> <b>15</b>, 765–776 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nrg3786" data-track-item_id="10.1038/nrg3786" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrg3786" aria-label="Article reference 81" data-doi="10.1038/nrg3786">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFOltbfE" aria-label="CAS reference 81">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25223781" aria-label="PubMed reference 81">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4412738" aria-label="PubMed Central reference 81">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20contribution%20of%20genetic%20variants%20to%20disease%20depends%20on%20the%20ruler&amp;journal=Nat.%20Rev.%20Genet.&amp;doi=10.1038%2Fnrg3786&amp;volume=15&amp;pages=765-776&amp;publication_year=2014&amp;author=Witte%2CJS&amp;author=Visscher%2CPM&amp;author=Wray%2CNR">
                    Google Scholar</a>&nbsp;
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01957-2?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank the FinnGen team for their contribution. In particular, we thank M.K. Niemi for methodological support and comments on the manuscript, S. Ripatti and M. Kanai for insightful comments on the manuscript, P. della Briotta Parolo for constructing the PGSs and kinship information in FinnGen. We also thank J.J. McGrath (Aarhus University) for very helpful comments on the manuscript. S.J. was supported by the Academy of Finland (grant no. 341747). A.G. was supported by the Academy of Finland (grant no. 323116) and by the European Research Council under the European Union’s Horizon 2020 Research and Innovation Programme (grant no. 945733). This project has also received funding from the European Union’s Horizon 2020 Research and Innovation Programme under grant agreement no. 101016775. The FinnGen project is funded by two grants from Business Finland (HUS 4685/31/2016 and UH 4386/31/2016) and nine industry partners (AbbVie, AstraZeneca, Biogen, Celgene, Genentech, GSK, MSD, Pfizer and Sanofi). The following biobanks are acknowledged for collecting the FinnGen project samples: Auria Biobank, THL Biobank, Helsinki Biobank, Northern Finland Biobank Borealis, Finnish Clinical Biobank Tampere, Biobank of Eastern Finland, Central Finland Biobank and the Finnish Red Cross Blood Service Biobank. UKB analyses were conducted under application no. 31063.</p></div></div></section><section data-title="Funding"><div class="c-article-section" id="Fun-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Fun">Funding</h2><div class="c-article-section__content" id="Fun-content"><p>Open Access funding provided by University of Helsinki including Helsinki University Central Hospital</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="na2"><p>A full list of members and their affiliations appears in the Supplementary Information.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Institute for Molecular Medicine Finland (FIMM), University of Helsinki, Helsinki, Finland</p><p class="c-article-author-affiliation__authors-list">Sakari Jukarainen,&nbsp;Tuomo Kiiskinen,&nbsp;Sara Kuitunen,&nbsp;Aki S. Havulinna,&nbsp;Juha Karjalainen,&nbsp;Mattia Cordioli,&nbsp;Joel T. Rämö,&nbsp;Nina Mars,&nbsp;Hanna M. Ollila,&nbsp;Matti Pirinen&nbsp;&amp;&nbsp;Andrea Ganna</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Finnish Institute for Health and Welfare, Helsinki, Finland</p><p class="c-article-author-affiliation__authors-list">Tuomo Kiiskinen,&nbsp;Sara Kuitunen&nbsp;&amp;&nbsp;Aki S. Havulinna</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Program in Medical and Population Genetics, Broad Institute of MIT and Harvard, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Tuomo Kiiskinen,&nbsp;Juha Karjalainen,&nbsp;Kaitlin E. Samocha,&nbsp;Hanna M. Ollila&nbsp;&amp;&nbsp;Andrea Ganna</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Analytic and Translational Genetics Unit, Massachusetts General Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Juha Karjalainen&nbsp;&amp;&nbsp;Andrea Ganna</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Center for Genomic Medicine, Massachusetts General Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Kaitlin E. Samocha&nbsp;&amp;&nbsp;Hanna M. Ollila</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Anesthesia, Critical Care and Pain Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Hanna M. Ollila</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Department of Public Health, University of Helsinki, Helsinki, Finland</p><p class="c-article-author-affiliation__authors-list">Matti Pirinen</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Helsinki Institute for Information Technology HIIT and Department of Mathematics and Statistics, University of Helsinki, Helsinki, Finland</p><p class="c-article-author-affiliation__authors-list">Matti Pirinen</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Sakari-Jukarainen-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Sakari Jukarainen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sakari%20Jukarainen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sakari%20Jukarainen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sakari%20Jukarainen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Tuomo-Kiiskinen-Aff1-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Tuomo Kiiskinen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Tuomo%20Kiiskinen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tuomo%20Kiiskinen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tuomo%20Kiiskinen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sara-Kuitunen-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Sara Kuitunen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sara%20Kuitunen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sara%20Kuitunen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sara%20Kuitunen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Aki_S_-Havulinna-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Aki S. Havulinna</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Aki%20S.%20Havulinna" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Aki%20S.%20Havulinna" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Aki%20S.%20Havulinna%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Juha-Karjalainen-Aff1-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Juha Karjalainen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Juha%20Karjalainen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Juha%20Karjalainen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Juha%20Karjalainen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Mattia-Cordioli-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Mattia Cordioli</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Mattia%20Cordioli" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Mattia%20Cordioli" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Mattia%20Cordioli%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Joel_T_-R_m_-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Joel T. Rämö</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Joel%20T.%20R%C3%A4m%C3%B6" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Joel%20T.%20R%C3%A4m%C3%B6" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Joel%20T.%20R%C3%A4m%C3%B6%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Nina-Mars-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Nina Mars</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Nina%20Mars" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nina%20Mars" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nina%20Mars%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Kaitlin_E_-Samocha-Aff3-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Kaitlin E. Samocha</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Kaitlin%20E.%20Samocha" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kaitlin%20E.%20Samocha" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kaitlin%20E.%20Samocha%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Hanna_M_-Ollila-Aff1-Aff3-Aff5-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Hanna M. Ollila</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Hanna%20M.%20Ollila" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Hanna%20M.%20Ollila" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Hanna%20M.%20Ollila%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Matti-Pirinen-Aff1-Aff7-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Matti Pirinen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Matti%20Pirinen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Matti%20Pirinen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Matti%20Pirinen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Andrea-Ganna-Aff1-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Andrea Ganna</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Andrea%20Ganna" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Andrea%20Ganna" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Andrea%20Ganna%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="groups">Consortia</h3><div class="c-article-author-institutional-author" id="group-1"><h3 class="c-article-author-institutional-author__name u-h3">FinnGen</h3><ul class="c-article-author-institutional-author__author-list"><li class="c-article-author-institutional-author__author-name">Tuomo Kiiskinen</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Aki S. Havulinna</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Juha Karjalainen</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Nina Mars</li><li class="c-article-author-institutional-author__author-name">&nbsp;&amp;&nbsp;Andrea Ganna</li></ul></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>A.G. conceptualized the study. A.G. and S.J. designed the analysis plan. S.J. performed most of the analyses that were not part of the FinnGen GWAS pipeline. The manuscript was mainly written by S.J. and A.G. The phenotyping approach for FinnGen was mainly developed by T.K. and A.H. and they also provided methodological support. S.K. was responsible for implementing the shinyapp. J.K. participated in genetic fine-mapping and provided methodological support. M.C., J.T.R., N.M. and K.E.S. provided methodological support. H.M.O. provided advice on the HLA analyses. M.P. provided statistical support by helping develop the approach for quantifying uncertainty via resampling and helping develop the shrinkage approach. All authors participated in writing and reviewing the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:sakari.jukarainen@helsinki.fi">Sakari Jukarainen</a> or <a id="corresp-c2" href="mailto:andrea.ganna@helsinki.fi">Andrea Ganna</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar4">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
              
              
                <h3 class="c-article__sub-heading" id="FPar3">Peer review information</h3>
                <p><i>Nature Medicine</i> thanks Todd Edwards and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary Handling editor: Anna Maria Ranzoni, in collaboration with the <i>Nature Medicine</i> team.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec32-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec32">Extended data</h2><div class="c-article-section__content" id="Sec32-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 schematic representation of h" href="/articles/s41591-022-01957-2/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig7_ESM.jpg">Extended Data Fig. 1 Schematic representation of how disability-adjusted life years (DALYs) are constructed from years lived with disability (YLD), and years of life lost (YLL).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>DALYs are a metric for measuring population-level disease burden that combines a measure of premature mortality called years of life lost (YLL) and a measure of healthy life years lost due to lowered quality of life called years lived with disability (YLD), so DALYs are the sum of YLDs and YLLs.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 cumulative distribution plot " href="/articles/s41591-022-01957-2/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig8_ESM.jpg">Extended Data Fig. 2 Cumulative distribution plot of log<sub>10</sub> <i>P</i>-values for all 3,123 HRs between genetic exposures and diseases.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>67.1% of the associations were genome-wide significant (<i>P</i> &lt; 5×10<sup>−8</sup>) and 99% had an association with <i>P</i> &lt; 7.3×10<sup>−4</sup>. Estimates are based on 735,748 individuals. <i>P</i>-values are for two-sided Wald tests.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 comparison of effect sizes (l" href="/articles/s41591-022-01957-2/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig9_ESM.jpg">Extended Data Fig. 3 Comparison of effect sizes (log-HR <i>Z</i>-score) for common variant-disease associations (n = 68,616) estimated both in FinnGen (n = 309,136) and UK Biobank (n = 426,612).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Axes are truncated at ±20, leaving out 36 points.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 effect of hla alleles on daly" href="/articles/s41591-022-01957-2/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig10_ESM.jpg">Extended Data Fig. 4 Effect of HLA alleles on DALYs.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Note that each gene is multiallelic, so the comparison of ‘1 vs 0 copies’ is more precisely the effect for having ‘1 copy of the allele in question and 1 average copy of other allele’ vs ‘having 0 copies of the allele in question and 2 average copies other alleles’. The interpretation of attributable DALYs in this context is not straightforward since these represent multiallelic loci. Estimates are based on 309,136 individuals (FinnGen). Error bars denote 95% confidence intervals.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 attributable dalys and hrs th" href="/articles/s41591-022-01957-2/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig11_ESM.jpg">Extended Data Fig. 5 Attributable DALYs and HRs the composite PGS of mortality formed in FinnGen for top 25 diseases.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>The composite PGS for mortality was formed using all the 30 PGSes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">10</a>) to predict survival by emulating the approach by Meisner et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Meisner, A. et al. Combined utility of 25 disease and risk factor polygenic risk scores for stratifying risk of all-cause mortality. Am. J. Hum. Genet. 107, 418–431 (2020)." href="/articles/s41591-022-01957-2#ref-CR9" id="ref-link-section-d72754869e5384">9</a></sup> Estimates are based on 309,136 individuals (FinnGen). Error bars denote 95% confidence intervals.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 impact of finnish-enriched va" href="/articles/s41591-022-01957-2/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig12_ESM.jpg">Extended Data Fig. 6 Impact of Finnish-enriched variants on population-level DALYs.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>The proportion of population DALYs accounted for by the most impactful 9 Finnish-enriched variants. The white part of the bars denotes the number of population DALYs attributable to the variant if the Finnish MAF were equal to the NFSEE MAF, the grey part represents additional population DALYs resulting from the enrichment in the Finnish population. Estimates are based on 309,136 individuals (FinnGen). Error bars denote 95% confidence intervals. NFSEE, non-Finnish-non-Swedish-non-Estonian Europeans.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 5-year incidence estimates of" href="/articles/s41591-022-01957-2/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig13_ESM.jpg">Extended Data Fig. 7 5-year incidence estimates of COPD, migraine, prostate cancer, and schizophrenia in FinnGen and UK Biobank (UKBB) compared to the population incidence estimates from the Global Burden of Disease Study 2019 for Finland 2019 (GBD).</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Number of new cases per 100,000 was calculated as number of new cases per 100,000 during each age interval divided by person-years contributed by individuals that were not cases in the earlier age intervals. The incidence of some diseases like prostate cancer is well captured in UKB and FinnGen, while other disease, such as migraine, as severely under-estimated. Schizophrenia incidence is elevated in FinnGen as one of the legacy cohorts recruited individuals with a psychotic illness.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 hazard ratios by age group in" href="/articles/s41591-022-01957-2/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig14_ESM.jpg">Extended Data Fig. 8 Hazard ratios by age group in FinnGen.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>For each age group, we estimated the HRs via a Cox proportional hazards model including individuals whose follow-up started before the beginning of the age interval that did not have a previous record of the disease. Dashed horizontal lines indicate the overall HR in FinnGen. The figure demonstrates age-varying HRs, especially for ischemic heart disease, Alzheimer’s disease and other dementias, and low back pain (<b>a,b,c,d</b>). For breast cancer and prostate cancer, the HRs were approximately constant across age groups (<b>e,f</b>). Estimates are based on 309,136 individuals (FinnGen). Error bars denote 95% confidence intervals.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig15"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 histograms of the number of r" href="/articles/s41591-022-01957-2/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig15_ESM.jpg">Extended Data Fig. 9 Histograms of the number of retained disease associations for common variants.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Panel <b>a</b>: after shrinkage using different prior parameters, ‘mean’ and ‘sd’ correspond to the <i>μ</i> and <i>σ</i> prior parameters, ‘a’ and ‘b’ correspond to the <i>α</i> and <i>β</i> prior parameters. Panels <b>b</b> and <b>c</b>: using <i>P</i> &lt; 5×10<sup>−8</sup> or <i>P</i> &lt; 0.05 cutoff instead of shrinkage. Overall, the distribution of retained association is not sensitive to the choice of prior parameters and is rather similar compared to using a <i>P</i> &lt; 5×10<sup>−8</sup> cutoff instead of shrinkage.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig16"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 receiver operating character" href="/articles/s41591-022-01957-2/figures/16" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig16_ESM.jpg">Extended Data Fig. 10 Receiver operating characteristic curves using our shrinkage method as a classifier to identify true causal variants in simulated data.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Using <i>hail</i>, we simulated GWAS summary statistics for 80 binary phenotypes with heritability sampled uniformly between 0.1 and 0.6. We repeated this for four scenarios where the probability of a variant being causal (pi) is 0.001, 0.002, 0.005, and 0.01. We then applied our shrinkage procedure and classified those variants with a posterior probability of being from the null model of &lt;10% as being causal (colored dots on lines represent this threshold). See Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01957-2#MOESM3">16</a> for details on classification performance.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec33-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec33">Supplementary information</h2><div class="c-article-section__content" id="Sec33-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>FinnGen contributors</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary tables" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_MOESM3_ESM.xlsx" data-supp-info-image="">Supplementary Tables</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Tables 1−16</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Genetic%20risk%20factors%20have%20a%20substantial%20impact%20on%20healthy%20life%20years&amp;author=Sakari%20Jukarainen%20et%20al&amp;contentID=10.1038%2Fs41591-022-01957-2&amp;copyright=The%20Author%28s%29&amp;publication=1078-8956&amp;publicationDate=2022-09-12&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41591-022-01957-2" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41591-022-01957-2" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Jukarainen, S., Kiiskinen, T., Kuitunen, S. <i>et al.</i> Genetic risk factors have a substantial impact on healthy life years.
                    <i>Nat Med</i> <b>28</b>, 1893–1901 (2022). https://doi.org/10.1038/s41591-022-01957-2</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01957-2?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-01-13">13 January 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-07-19">19 July 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-09-12">12 September 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-09">September 2022</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41591-022-01957-2</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-block"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/epidemiology" data-track="click" data-track-action="view subject" data-track-label="link">Epidemiology</a></li><li class="c-article-subject-list__subject"><a href="/subjects/genetics" data-track="click" data-track-action="view subject" data-track-label="link">Genetics</a></li><li class="c-article-subject-list__subject"><a href="/subjects/genetics-research" data-track="click" data-track-action="view subject" data-track-label="link">Genetics research</a></li><li class="c-article-subject-list__subject"><a href="/subjects/public-health" data-track="click" data-track-action="view subject" data-track-label="link">Public health</a></li><li class="c-article-subject-list__subject"><a href="/subjects/risk-factors" data-track="click" data-track-action="view subject" data-track-label="link">Risk factors</a></li>
        </ul>
    </div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section" data-test="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Impact of SGLT-2 inhibitors on modifiable cardiovascular risk factors in Romanian patients with type 2 diabetes mellitus" href="https://doi.org/10.1186/s13098-024-01326-8">
                                        Impact of SGLT-2 inhibitors on modifiable cardiovascular risk factors in Romanian patients with type 2 diabetes mellitus
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Adriana Gherbon</li><li>Mirela Frandes</li><li>Bogdan Timar</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Diabetology &amp; Metabolic Syndrome</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Joint impact of polygenic risk score and lifestyles on early- and late-onset cardiovascular diseases" href="https://doi.org/10.1038/s41562-024-01923-7">
                                        Joint impact of polygenic risk score and lifestyles on early- and late-onset cardiovascular diseases
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Dong Sun</li><li>Yinqi Ding</li><li>Jun Lv</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Human Behaviour</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Genetic modifiers of rare variants in monogenic developmental disorder loci" href="https://doi.org/10.1038/s41588-024-01710-0">
                                        Genetic modifiers of rare variants in monogenic developmental disorder loci
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Rebecca Kingdom</li><li>Robin N. Beaumont</li><li>Caroline F. Wright</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Genetics</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Polygenic prediction of human longevity on the supposition of pervasive pleiotropy" href="https://doi.org/10.1038/s41598-024-69069-0">
                                        Polygenic prediction of human longevity on the supposition of pervasive pleiotropy
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>M. Reza Jabalameli</li><li>Jhih-Rong Lin</li><li>Zhengdong D. Zhang</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Scientific Reports</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:The relationship between 11 different polygenic longevity scores, parental lifespan, and disease diagnosis in the UK Biobank" href="https://doi.org/10.1007/s11357-024-01107-1">
                                        The relationship between 11 different polygenic longevity scores, parental lifespan, and disease diagnosis in the UK Biobank
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Janith Don</li><li>Andrew J. Schork</li><li>Nicholas J. Schork</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>GeroScience</i> (2024)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01957-2.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                
            
        
    </div>

    
        
    

    
    
        <div class="c-article-associated-content__container">
            <section>
                <h2 class="c-article-associated-content__title u-mb-24">Associated content</h2>
                
                    
                        <div class="c-article-associated-content__collection special u-mb-24">
                            <section>
                                <p class="c-article-associated-content__collection-label u-sans-serif u-text-bold u-mb-8">Special</p>
                                <h3 class="c-article-associated-content__collection-title u-h3 u-mb-8">
                                    <a href="https://www.nature.com/collections/ahigjjgihc" class="u-link-inherit" data-track="click" data-track-action="view special" data-track-category="associated content" data-track-label="special" data-test="collection-link">FinnGen</a>
                                </h3>
                            </section>
                        </div>
                    
                    
                        <div class="u-full-height u-mb-24">
                            
    <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
            <div class="c-card__body">
                <h3 class="c-card__title">
                    <a href="https://www.nature.com/articles/s41591-022-01958-1" class="c-card__link u-link-inherit" data-track="click" data-track-action="view article" data-track-category="associated content" data-track-label="news_and_views">A framework to assess the effects of genetic risk factors on disability-adjusted life years</a>
                </h3>
                
                
    <div class="c-card__section c-meta">
        
            <span class="c-meta__item">Nature Medicine</span>
        
        <span class="c-meta__item" data-test="article.type"><span class="c-meta__type">Research Briefing</span></span>
        
        
            <time class="c-meta__item" datetime="2022-09-12">12 Sept 2022</time>
        
    </div>

            </div>
        </div>
    </article>


                        </div>
                    
                
            </section>
        </div>
        <script>
            window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "special;news_and_views;";
            window.dataLayer[0].content.collections = "ahigjjgihc";
        </script>
    

    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
            <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" role="tabpanel"><div class="c-reading-companion__scroll-pane" style="max-height: none;"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Main">Main</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec10" class="c-reading-companion__section-item"><a href="#Sec10" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec11" class="c-reading-companion__section-item"><a href="#Sec11" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-data-availability" class="c-reading-companion__section-item"><a href="#data-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Data availability">Data availability</a></li><li id="rc-sec-code-availability" class="c-reading-companion__section-item"><a href="#code-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Code availability">Code availability</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-Fun" class="c-reading-companion__section-item"><a href="#Fun" data-track="click" data-track-action="section anchor" data-track-label="link:Funding">Funding</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-peer-review" class="c-reading-companion__section-item"><a href="#peer-review" data-track="click" data-track-action="section anchor" data-track-label="link:Peer review">Peer review</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec32" class="c-reading-companion__section-item"><a href="#Sec32" data-track="click" data-track-action="section anchor" data-track-label="link:Extended data">Extended data</a></li><li id="rc-sec-Sec33" class="c-reading-companion__section-item"><a href="#Sec33" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by">This article is cited by</a></li></ul></div>
                <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="" style="display: none !important;">
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-test="right-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41591-022-01957-2;doi=10.1038/s41591-022-01957-2;subjmeta=174,2056,208,308,478,499,631,692,700;kwrd=Epidemiology,Genetics,Genetics+research,Public+health,Risk+factors">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=653296140&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01957-2%26doi%3D10.1038/s41591-022-01957-2%26subjmeta%3D174,2056,208,308,478,499,631,692,700%26kwrd%3DEpidemiology,Genetics,Genetics+research,Public+health,Risk+factors">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=653296140&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01957-2%26doi%3D10.1038/s41591-022-01957-2%26subjmeta%3D174,2056,208,308,478,499,631,692,700%26kwrd%3DEpidemiology,Genetics,Genetics+research,Public+health,Risk+factors"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Fig. 1: Study overview.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig1_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig1_HTML.png" alt="figure 1" aria-describedby="rc-Fig1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Fig. 2: Effect of common variants on DALYs.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig2_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig2_HTML.png" alt="figure 2" aria-describedby="rc-Fig2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Fig. 3: DALYs attributable to carrying a deleterious rare variant in ACMG genes.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig3_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig3_HTML.png" alt="figure 3" aria-describedby="rc-Fig3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Fig. 4: Polygenic score impact on DALYs.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig4_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig4_HTML.png" alt="figure 4" aria-describedby="rc-Fig4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Fig. 5: Sex-specific impact of common variants and polygenic scores on DALYs.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig5_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig5_HTML.png" alt="figure 5" aria-describedby="rc-Fig5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig6">Fig. 6: Effect of common variants on population-level DALYs in the Finnish population.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig6_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig6_HTML.png" alt="figure 6" aria-describedby="rc-Fig6"></picture><p class="c-reading-companion__figure-links"><a href="#Fig6" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/6" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig7">Extended Data Fig. 1 Schematic representation of how disability-adjusted life years (DALYs) are constructed from years lived with disability (YLD), and years of life lost (YLL).</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig7_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig7_ESM.jpg" alt="extended data figure 7" aria-describedby="rc-Fig7"></picture><p class="c-reading-companion__figure-links"><a href="#Fig7" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/7" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig8">Extended Data Fig. 2 Cumulative distribution plot of log<sub>10</sub> <i>P</i>-values for all 3,123 HRs between genetic exposures and diseases.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig8_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig8_ESM.jpg" alt="extended data figure 8" aria-describedby="rc-Fig8"></picture><p class="c-reading-companion__figure-links"><a href="#Fig8" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/8" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig9">Extended Data Fig. 3 Comparison of effect sizes (log-HR <i>Z</i>-score) for common variant-disease associations (n = 68,616) estimated both in FinnGen (n = 309,136) and UK Biobank (n = 426,612).</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig9_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig9_ESM.jpg" alt="extended data figure 9" aria-describedby="rc-Fig9"></picture><p class="c-reading-companion__figure-links"><a href="#Fig9" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/9" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig10">Extended Data Fig. 4 Effect of HLA alleles on DALYs.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig10_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig10_ESM.jpg" alt="extended data figure 10" aria-describedby="rc-Fig10"></picture><p class="c-reading-companion__figure-links"><a href="#Fig10" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/10" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig11">Extended Data Fig. 5 Attributable DALYs and HRs the composite PGS of mortality formed in FinnGen for top 25 diseases.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig11_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig11_ESM.jpg" alt="extended data figure 11" aria-describedby="rc-Fig11"></picture><p class="c-reading-companion__figure-links"><a href="#Fig11" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/11" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig12">Extended Data Fig. 6 Impact of Finnish-enriched variants on population-level DALYs.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig12_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig12_ESM.jpg" alt="extended data figure 12" aria-describedby="rc-Fig12"></picture><p class="c-reading-companion__figure-links"><a href="#Fig12" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/12" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig13">Extended Data Fig. 7 5-year incidence estimates of COPD, migraine, prostate cancer, and schizophrenia in FinnGen and UK Biobank (UKBB) compared to the population incidence estimates from the Global Burden of Disease Study 2019 for Finland 2019 (GBD).</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig13_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig13_ESM.jpg" alt="extended data figure 13" aria-describedby="rc-Fig13"></picture><p class="c-reading-companion__figure-links"><a href="#Fig13" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/13" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig14">Extended Data Fig. 8 Hazard ratios by age group in FinnGen.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig14_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig14_ESM.jpg" alt="extended data figure 14" aria-describedby="rc-Fig14"></picture><p class="c-reading-companion__figure-links"><a href="#Fig14" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/14" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig15">Extended Data Fig. 9 Histograms of the number of retained disease associations for common variants.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig15_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig15_ESM.jpg" alt="extended data figure 15" aria-describedby="rc-Fig15"></picture><p class="c-reading-companion__figure-links"><a href="#Fig15" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/15" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig16">Extended Data Fig. 10 Receiver operating characteristic curves using our shrinkage method as a classifier to identify true causal variants in simulated data.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig16_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01957-2/MediaObjects/41591_2022_1957_Fig16_ESM.jpg" alt="extended data figure 16" aria-describedby="rc-Fig16"></picture><p class="c-reading-companion__figure-links"><a href="#Fig16" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01957-2/figures/16" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric" data-track-context="article references sidebar"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">Visscher, P. M. et al. 10 Years of GWAS discovery: biology, function, and translation. <i>Am. J. Hum. Genet</i>. <b>101</b>, 5–22 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ajhg.2017.06.005" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ajhg.2017.06.005" data-track-item_id="10.1016/j.ajhg.2017.06.005">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFChu7rI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28686856" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5501872" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=10%20Years%20of%20GWAS%20discovery%3A%20biology%2C%20function%2C%20and%20translation&amp;journal=Am.%20J.%20Hum.%20Genet&amp;doi=10.1016%2Fj.ajhg.2017.06.005&amp;volume=101&amp;pages=5-22&amp;publication_year=2017&amp;author=Visscher%2CPM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">Watanabe, K. et al. A global overview of pleiotropy and genetic architecture in complex traits. <i>Nat. Genet.</i> <b>51</b>, 1339–1348 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-019-0481-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-019-0481-0" data-track-item_id="10.1038/s41588-019-0481-0">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1WjurvF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31427789" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20global%20overview%20of%20pleiotropy%20and%20genetic%20architecture%20in%20complex%20traits&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0481-0&amp;volume=51&amp;pages=1339-1348&amp;publication_year=2019&amp;author=Watanabe%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Mars, N. et al. The role of polygenic risk and susceptibility genes in breast cancer over the course of life. <i>Nat. Commun.</i> <b>11</b>, 6383 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-19966-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-19966-5" data-track-item_id="10.1038/s41467-020-19966-5">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXis1agtr%2FK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33318493" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7736877" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20polygenic%20risk%20and%20susceptibility%20genes%20in%20breast%20cancer%20over%20the%20course%20of%20life&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19966-5&amp;volume=11&amp;publication_year=2020&amp;author=Mars%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">Rasmussen, K. L., Tybjærg-Hansen, A., Nordestgaard, B. G. &amp; Frikke-Schmidt, R. Absolute 10-year risk of dementia by age, sex and APOE genotype: a population-based cohort study. <i>CMAJ</i> <b>190</b>, E1033–E1041 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1503%2Fcmaj.180066" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1503/cmaj.180066" data-track-item_id="10.1503/cmaj.180066">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30181149" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6148649" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Absolute%2010-year%20risk%20of%20dementia%20by%20age%2C%20sex%20and%20APOE%20genotype%3A%20a%20population-based%20cohort%20study&amp;journal=CMAJ&amp;doi=10.1503%2Fcmaj.180066&amp;volume=190&amp;pages=E1033-E1041&amp;publication_year=2018&amp;author=Rasmussen%2CKL&amp;author=Tybj%C3%A6rg-Hansen%2CA&amp;author=Nordestgaard%2CBG&amp;author=Frikke-Schmidt%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">Metcalfe, K. A. et al. The risk of breast cancer in BRCA1 and BRCA2 mutation carriers without a first-degree relative with breast cancer. <i>Clin. Genet</i>. <b>93</b>, 1063–1068 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2Fcge.13191" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/cge.13191" data-track-item_id="10.1111/cge.13191">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXnsFSns7k%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29206279" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20risk%20of%20breast%20cancer%20in%20BRCA1%20and%20BRCA2%20mutation%20carriers%20without%20a%20first-degree%20relative%20with%20breast%20cancer&amp;journal=Clin.%20Genet&amp;doi=10.1111%2Fcge.13191&amp;volume=93&amp;pages=1063-1068&amp;publication_year=2018&amp;author=Metcalfe%2CKA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Li, T. et al. Total genetic contribution assessment across the human genome. <i>Nat. Commun.</i> <b>12</b>, 2845 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-021-23124-w" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-021-23124-w" data-track-item_id="10.1038/s41467-021-23124-w">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFahtL%2FI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33990588" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8121943" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Total%20genetic%20contribution%20assessment%20across%20the%20human%20genome&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-23124-w&amp;volume=12&amp;publication_year=2021&amp;author=Li%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Mars, N. et al. Polygenic and clinical risk scores and their impact on age at onset and prediction of cardiometabolic diseases and common cancers. <i>Nat. Med.</i> <b>26</b>, 549–557 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41591-020-0800-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41591-020-0800-0" data-track-item_id="10.1038/s41591-020-0800-0">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXmvVCiu7k%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32273609" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20and%20clinical%20risk%20scores%20and%20their%20impact%20on%20age%20at%20onset%20and%20prediction%20of%20cardiometabolic%20diseases%20and%20common%20cancers&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-020-0800-0&amp;volume=26&amp;pages=549-557&amp;publication_year=2020&amp;author=Mars%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Sakaue, S. et al. Trans-biobank analysis with 676,000 individuals elucidates the association of polygenic risk scores of complex traits with human lifespan. <i>Nat. Med.</i> <b>26</b>, 542–548 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41591-020-0785-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41591-020-0785-8" data-track-item_id="10.1038/s41591-020-0785-8">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlsVeisLY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32251405" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Trans-biobank%20analysis%20with%20676%2C000%20individuals%20elucidates%20the%20association%20of%20polygenic%20risk%20scores%20of%20complex%20traits%20with%20human%20lifespan&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-020-0785-8&amp;volume=26&amp;pages=542-548&amp;publication_year=2020&amp;author=Sakaue%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Meisner, A. et al. Combined utility of 25 disease and risk factor polygenic risk scores for stratifying risk of all-cause mortality. <i>Am. J. Hum. Genet.</i> <b>107</b>, 418–431 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ajhg.2020.07.002" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ajhg.2020.07.002" data-track-item_id="10.1016/j.ajhg.2020.07.002">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFKgs7fI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32758451" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7477009" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Combined%20utility%20of%2025%20disease%20and%20risk%20factor%20polygenic%20risk%20scores%20for%20stratifying%20risk%20of%20all-cause%20mortality&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2020.07.002&amp;volume=107&amp;pages=418-431&amp;publication_year=2020&amp;author=Meisner%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Timmers, P. R. et al. Genomics of 1 million parent lifespans implicates novel pathways and common diseases and distinguishes survival chances. <i>eLife</i> <b>8</b>, e39856 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.7554%2FeLife.39856" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.7554/eLife.39856" data-track-item_id="10.7554/eLife.39856">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30642433" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6333444" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genomics%20of%201%20million%20parent%20lifespans%20implicates%20novel%20pathways%20and%20common%20diseases%20and%20distinguishes%20survival%20chances&amp;journal=eLife&amp;doi=10.7554%2FeLife.39856&amp;volume=8&amp;publication_year=2019&amp;author=Timmers%2CPR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Murray, C. J. L. et al. Global burden of 87 risk factors in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. <i>Lancet</i> <b>396</b>, 1223–1249 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0140-6736%2820%2930752-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0140-6736(20)30752-2" data-track-item_id="10.1016/S0140-6736(20)30752-2">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20burden%20of%2087%20risk%20factors%20in%20204%20countries%20and%20territories%2C%201990%E2%80%932019%3A%20a%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202019&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2820%2930752-2&amp;volume=396&amp;pages=1223-1249&amp;publication_year=2020&amp;author=Murray%2CCJL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Marwaha, S., Knowles, J. W. &amp; Ashley, E. A. A guide for the diagnosis of rare and undiagnosed disease: beyond the exome. <i>Genome Med.</i> <b>14</b>, 23 (2022).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13073-022-01026-w" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13073-022-01026-w" data-track-item_id="10.1186/s13073-022-01026-w">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35220969" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8883622" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20guide%20for%20the%20diagnosis%20of%20rare%20and%20undiagnosed%20disease%3A%20beyond%20the%20exome&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-022-01026-w&amp;volume=14&amp;publication_year=2022&amp;author=Marwaha%2CS&amp;author=Knowles%2CJW&amp;author=Ashley%2CEA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Brown, G. R. et al. A review of inherited cancer susceptibility syndromes. <i>JAAPA</i> <b>33</b>, 10–16 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1097%2F01.JAA.0000721648.46099.2c" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1097/01.JAA.0000721648.46099.2c" data-track-item_id="10.1097/01.JAA.0000721648.46099.2c">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33234888" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20review%20of%20inherited%20cancer%20susceptibility%20syndromes&amp;journal=JAAPA&amp;doi=10.1097%2F01.JAA.0000721648.46099.2c&amp;volume=33&amp;pages=10-16&amp;publication_year=2020&amp;author=Brown%2CGR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Manchanda, R. &amp; Gaba, F. Population-based testing for primary prevention: a systematic review. <i>Cancers</i> <b>10</b>, E424 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3390%2Fcancers10110424" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3390/cancers10110424" data-track-item_id="10.3390/cancers10110424">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30400647" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Population-based%20testing%20for%20primary%20prevention%3A%20a%20systematic%20review&amp;journal=Cancers&amp;doi=10.3390%2Fcancers10110424&amp;volume=10&amp;publication_year=2018&amp;author=Manchanda%2CR&amp;author=Gaba%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">Kullo, I. J. et al. Polygenic scores in biomedical research. <i>Nat. Rev. Genet</i>. (2022) <a href="https://doi.org/10.1038/s41576-022-00470-z" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41576-022-00470-z">https://doi.org/10.1038/s41576-022-00470-z</a> (2022).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">Sun, L. et al. Polygenic risk scores in cardiovascular risk prediction: A cohort study and modelling analyses. <i>PLoS Med.</i> <b>18</b>, e1003498 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pmed.1003498" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pmed.1003498" data-track-item_id="10.1371/journal.pmed.1003498">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXisFWqu7k%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33444330" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7808664" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%20in%20cardiovascular%20risk%20prediction%3A%20A%20cohort%20study%20and%20modelling%20analyses&amp;journal=PLoS%20Med.&amp;doi=10.1371%2Fjournal.pmed.1003498&amp;volume=18&amp;publication_year=2021&amp;author=Sun%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Lewis, C. M. &amp; Vassos, E. Polygenic risk scores: from research tools to clinical instruments. <i>Genome Med.</i> <b>12</b>, 44 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13073-020-00742-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13073-020-00742-5" data-track-item_id="10.1186/s13073-020-00742-5">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32423490" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7236300" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%3A%20from%20research%20tools%20to%20clinical%20instruments&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-020-00742-5&amp;volume=12&amp;publication_year=2020&amp;author=Lewis%2CCM&amp;author=Vassos%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">Wand, H. et al. Improving reporting standards for polygenic scores in risk prediction studies. <i>Nature</i> <b>591</b>, 211–219 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-021-03243-6" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-021-03243-6" data-track-item_id="10.1038/s41586-021-03243-6">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmt1ems7Y%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33692554" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8609771" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Improving%20reporting%20standards%20for%20polygenic%20scores%20in%20risk%20prediction%20studies&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03243-6&amp;volume=591&amp;pages=211-219&amp;publication_year=2021&amp;author=Wand%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">Musunuru, K. et al. In vivo CRISPR base editing of PCSK9 durably lowers cholesterol in primates. <i>Nature</i> <b>593</b>, 429–434 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-021-03534-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-021-03534-y" data-track-item_id="10.1038/s41586-021-03534-y">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFanur3I" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34012082" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vivo%20CRISPR%20base%20editing%20of%20PCSK9%20durably%20lowers%20cholesterol%20in%20primates&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03534-y&amp;volume=593&amp;pages=429-434&amp;publication_year=2021&amp;author=Musunuru%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Frangoul, H. et al. CRISPR-Cas9 gene editing for sickle cell disease and β-thalassemia. <i>N. Engl. J. Med.</i> <b>384</b>, 252–260 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa2031054" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa2031054" data-track-item_id="10.1056/NEJMoa2031054">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVKhs7c%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33283989" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-Cas9%20gene%20editing%20for%20sickle%20cell%20disease%20and%20%CE%B2-thalassemia&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa2031054&amp;volume=384&amp;pages=252-260&amp;publication_year=2021&amp;author=Frangoul%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Rim, J. H., Gopalappa, R. &amp; Gee, H. Y. CRISPR-Cas9 in vivo gene editing for transthyretin amyloidosis. <i>N. Engl. J. Med.</i> <b>385</b>, 1722 (2021).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34706180" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-Cas9%20in%20vivo%20gene%20editing%20for%20transthyretin%20amyloidosis&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=385&amp;publication_year=2021&amp;author=Rim%2CJH&amp;author=Gopalappa%2CR&amp;author=Gee%2CHY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Turley, P. et al. Problems with using polygenic scores to select embryos. <i>N. Engl. J. Med.</i> <b>385</b>, 78–86 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMsr2105065" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMsr2105065" data-track-item_id="10.1056/NEJMsr2105065">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34192436" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8387884" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Problems%20with%20using%20polygenic%20scores%20to%20select%20embryos&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMsr2105065&amp;volume=385&amp;pages=78-86&amp;publication_year=2021&amp;author=Turley%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">Lencz, T. et al. Utility of polygenic embryo screening for disease depends on the selection strategy. <i>eLife</i> <b>10</b>, e64716 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.7554%2FeLife.64716" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.7554/eLife.64716" data-track-item_id="10.7554/eLife.64716">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XotVSltrk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34635206" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8510582" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Utility%20of%20polygenic%20embryo%20screening%20for%20disease%20depends%20on%20the%20selection%20strategy&amp;journal=eLife&amp;doi=10.7554%2FeLife.64716&amp;volume=10&amp;publication_year=2021&amp;author=Lencz%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Karavani, E. et al. Screening human embryos for polygenic traits has limited utility. <i>Cell</i> <b>179</b>, 1424–1435 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2019.10.033" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2019.10.033" data-track-item_id="10.1016/j.cell.2019.10.033">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1Crt7vL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31761530" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6957074" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Screening%20human%20embryos%20for%20polygenic%20traits%20has%20limited%20utility&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.10.033&amp;volume=179&amp;pages=1424-1435&amp;publication_year=2019&amp;author=Karavani%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">Kumar, A. et al. Whole-genome risk prediction of common diseases in human preimplantation embryos. <i>Nat. Med.</i> <b>28</b>, 513–516 (2022).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41591-022-01735-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41591-022-01735-0" data-track-item_id="10.1038/s41591-022-01735-0">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XnslSku74%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35314819" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8938270" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-genome%20risk%20prediction%20of%20common%20diseases%20in%20human%20preimplantation%20embryos&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-022-01735-0&amp;volume=28&amp;pages=513-516&amp;publication_year=2022&amp;author=Kumar%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Johnston, J. &amp; Matthews, L. J. Polygenic embryo testing: understated ethics, unclear utility. <i>Nat. Med.</i> <b>28</b>, 446–448 (2022).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41591-022-01743-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41591-022-01743-0" data-track-item_id="10.1038/s41591-022-01743-0">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XnslSkurk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35314818" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9208380" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20embryo%20testing%3A%20understated%20ethics%2C%20unclear%20utility&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-022-01743-0&amp;volume=28&amp;pages=446-448&amp;publication_year=2022&amp;author=Johnston%2CJ&amp;author=Matthews%2CLJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">Vos, T. et al. Global burden of 369 diseases and injuries in 204 countries and territories, 1990–2019: a systematic analysis for the Global Burden of Disease Study 2019. <i>Lancet</i> <b>396</b>, 1204–1222 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0140-6736%2820%2930925-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0140-6736(20)30925-9" data-track-item_id="10.1016/S0140-6736(20)30925-9">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20burden%20of%20369%20diseases%20and%20injuries%20in%20204%20countries%20and%20territories%2C%201990%E2%80%932019%3A%20a%20systematic%20analysis%20for%20the%20Global%20Burden%20of%20Disease%20Study%202019&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2820%2930925-9&amp;volume=396&amp;pages=1204-1222&amp;publication_year=2020&amp;author=Vos%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Kurki, M. I. et al. FinnGen: unique genetic insights from combining isolated population and national health register data. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2022.03.03.22271360" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2022.03.03.22271360">https://doi.org/10.1101/2022.03.03.22271360</a> (2022).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">Sudlow, C. et al. UK Biobank: an open access resource for identifying the causes of a wide range of complex diseases of middle and old age. <i>PLoS Med.</i> <b>12</b>, e1001779 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pmed.1001779" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pmed.1001779" data-track-item_id="10.1371/journal.pmed.1001779">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25826379" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4380465" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=UK%20Biobank%3A%20an%20open%20access%20resource%20for%20identifying%20the%20causes%20of%20a%20wide%20range%20of%20complex%20diseases%20of%20middle%20and%20old%20age&amp;journal=PLoS%20Med.&amp;doi=10.1371%2Fjournal.pmed.1001779&amp;volume=12&amp;publication_year=2015&amp;author=Sudlow%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">Wang, G., Sarkar, A., Carbonetto, P. &amp; Stephens, M. A simple new approach to variable selection in regression, with application to genetic fine mapping. <i>J. R. Stat. Soc. Ser. B Stat. Methodol.</i> <b>82</b>, 1273–1300 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2Frssb.12388" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/rssb.12388" data-track-item_id="10.1111/rssb.12388">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20simple%20new%20approach%20to%20variable%20selection%20in%20regression%2C%20with%20application%20to%20genetic%20fine%20mapping&amp;journal=J.%20R.%20Stat.%20Soc.%20Ser.%20B%20Stat.%20Methodol.&amp;doi=10.1111%2Frssb.12388&amp;volume=82&amp;pages=1273-1300&amp;publication_year=2020&amp;author=Wang%2CG&amp;author=Sarkar%2CA&amp;author=Carbonetto%2CP&amp;author=Stephens%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Kanai, M. et al. Insights from complex trait fine-mapping across diverse populations. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2021.09.03.21262975" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.09.03.21262975">https://doi.org/10.1101/2021.09.03.21262975</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Matejcic, M. et al. Germline variation at 8q24 and prostate cancer risk in men of European ancestry. <i>Nat. Commun.</i> <b>9</b>, 4616 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-018-06863-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-018-06863-1" data-track-item_id="10.1038/s41467-018-06863-1">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30397198" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6218483" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Germline%20variation%20at%208q24%20and%20prostate%20cancer%20risk%20in%20men%20of%20European%20ancestry&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-06863-1&amp;volume=9&amp;publication_year=2018&amp;author=Matejcic%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">Schlaepfer, I. R. et al. The CHRNA5/A3/B4 gene cluster variability as an important determinant of early alcohol and tobacco initiation in young adults. <i>Biol. Psychiat.</i> <b>63</b>, 1039–1046 (2008).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.biopsych.2007.10.024" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.biopsych.2007.10.024" data-track-item_id="10.1016/j.biopsych.2007.10.024">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXmtVagsro%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18163978" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20CHRNA5%2FA3%2FB4%20gene%20cluster%20variability%20as%20an%20important%20determinant%20of%20early%20alcohol%20and%20tobacco%20initiation%20in%20young%20adults&amp;journal=Biol.%20Psychiat.&amp;doi=10.1016%2Fj.biopsych.2007.10.024&amp;volume=63&amp;pages=1039-1046&amp;publication_year=2008&amp;author=Schlaepfer%2CIR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">Joshi, P. K. et al. Variants near CHRNA3/5 and APOE have age- and sex-related effects on human lifespan. <i>Nat. Commun.</i> <b>7</b>, 11174 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fncomms11174" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ncomms11174" data-track-item_id="10.1038/ncomms11174">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xlt1Kms7Y%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27029810" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5438072" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Variants%20near%20CHRNA3%2F5%20and%20APOE%20have%20age-%20and%20sex-related%20effects%20on%20human%20lifespan&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms11174&amp;volume=7&amp;publication_year=2016&amp;author=Joshi%2CPK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">Green, R. C. et al. ACMG recommendations for reporting of incidental findings in clinical exome and genome sequencing. <i>Genet. Med.</i> <b>15</b>, 565–574 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fgim.2013.73" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/gim.2013.73" data-track-item_id="10.1038/gim.2013.73">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtVKku73K" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23788249" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3727274" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ACMG%20recommendations%20for%20reporting%20of%20incidental%20findings%20in%20clinical%20exome%20and%20genome%20sequencing&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fgim.2013.73&amp;volume=15&amp;pages=565-574&amp;publication_year=2013&amp;author=Green%2CRC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Miller, D. T. et al. ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG). <i>Genet. Med.</i> <b>23</b>, 1381–1390 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41436-021-01172-3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41436-021-01172-3" data-track-item_id="10.1038/s41436-021-01172-3">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34012068" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ACMG%20SF%20v3.0%20list%20for%20reporting%20of%20secondary%20findings%20in%20clinical%20exome%20and%20genome%20sequencing%3A%20a%20policy%20statement%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics%20%28ACMG%29&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fs41436-021-01172-3&amp;volume=23&amp;pages=1381-1390&amp;publication_year=2021&amp;author=Miller%2CDT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Landrum, M. J. et al. ClinVar: public archive of relationships among sequence variation and human phenotype. <i>Nucleic Acids Res.</i> <b>42</b>, D980–D985 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgkt1113" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gkt1113" data-track-item_id="10.1093/nar/gkt1113">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXoslGh" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24234437" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ClinVar%3A%20public%20archive%20of%20relationships%20among%20sequence%20variation%20and%20human%20phenotype&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkt1113&amp;volume=42&amp;pages=D980-D985&amp;publication_year=2014&amp;author=Landrum%2CMJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Spurdle, A. B. et al. ENIGMA–evidence-based network for the interpretation of germline mutant alleles: an international initiative to evaluate risk and clinical significance associated with sequence variation in BRCA1 and BRCA2 genes. <i>Hum. Mutat.</i> <b>33</b>, 2–7 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1002%2Fhumu.21628" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1002/humu.21628" data-track-item_id="10.1002/humu.21628">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXhs1aju7%2FI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21990146" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ENIGMA%E2%80%93evidence-based%20network%20for%20the%20interpretation%20of%20germline%20mutant%20alleles%3A%20an%20international%20initiative%20to%20evaluate%20risk%20and%20clinical%20significance%20associated%20with%20sequence%20variation%20in%20BRCA1%20and%20BRCA2%20genes&amp;journal=Hum.%20Mutat.&amp;doi=10.1002%2Fhumu.21628&amp;volume=33&amp;pages=2-7&amp;publication_year=2012&amp;author=Spurdle%2CAB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">Liu, J. Z. et al. Association analyses identify 38 susceptibility loci for inflammatory bowel disease and highlight shared genetic risk across populations. <i>Nat. Genet.</i> <b>47</b>, 979–986 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fng.3359" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ng.3359" data-track-item_id="10.1038/ng.3359">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1WltLvI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26192919" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4881818" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Association%20analyses%20identify%2038%20susceptibility%20loci%20for%20inflammatory%20bowel%20disease%20and%20highlight%20shared%20genetic%20risk%20across%20populations&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3359&amp;volume=47&amp;pages=979-986&amp;publication_year=2015&amp;author=Liu%2CJZ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">Johnston, K. J. A. et al. Genome-wide association study of multisite chronic pain in UK Biobank. <i>PLoS Genet</i>. <a href="https://doi.org/10.1371/journal.pgen.1008164" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1371/journal.pgen.1008164">https://doi.org/10.1371/journal.pgen.1008164</a> (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">BasuRay, S. PNPLA3-I148M: a problem of plenty in non-alcoholic fatty liver disease. <i>Adipocyte</i> <b>8</b>, 201–208 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1080%2F21623945.2019.1607423" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1080/21623945.2019.1607423" data-track-item_id="10.1080/21623945.2019.1607423">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6768214" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=PNPLA3-I148M%3A%20a%20problem%20of%20plenty%20in%20non-alcoholic%20fatty%20liver%20disease&amp;journal=Adipocyte&amp;doi=10.1080%2F21623945.2019.1607423&amp;volume=8&amp;pages=201-208&amp;publication_year=2019&amp;author=BasuRay%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">Liu, M. et al. Association studies of up to 1.2 million individuals yield new insights into the genetic etiology of tobacco and alcohol use. <i>Nat. Genet.</i> <b>51</b>, 237–244 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-018-0307-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-018-0307-5" data-track-item_id="10.1038/s41588-018-0307-5">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvFSns7g%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30643251" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6358542" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Association%20studies%20of%20up%20to%201.2%20million%20individuals%20yield%20new%20insights%20into%20the%20genetic%20etiology%20of%20tobacco%20and%20alcohol%20use&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0307-5&amp;volume=51&amp;pages=237-244&amp;publication_year=2019&amp;author=Liu%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">Chheda, H. et al. Whole-genome view of the consequences of a population bottleneck using 2926 genome sequences from Finland and United Kingdom. <i>Eur. J. Hum. Genet.</i> <b>25</b>, 477–484 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fejhg.2016.205" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ejhg.2016.205" data-track-item_id="10.1038/ejhg.2016.205">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28145424" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5346294" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-genome%20view%20of%20the%20consequences%20of%20a%20population%20bottleneck%20using%202926%20genome%20sequences%20from%20Finland%20and%20United%20Kingdom&amp;journal=Eur.%20J.%20Hum.%20Genet.&amp;doi=10.1038%2Fejhg.2016.205&amp;volume=25&amp;pages=477-484&amp;publication_year=2017&amp;author=Chheda%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">Lim, E. T. et al. Distribution and medical impact of loss-of-function variants in the Finnish founder population. <i>PLoS Genet.</i> <b>10</b>, e1004494 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pgen.1004494" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pgen.1004494" data-track-item_id="10.1371/journal.pgen.1004494">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25078778" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4117444" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Distribution%20and%20medical%20impact%20of%20loss-of-function%20variants%20in%20the%20Finnish%20founder%20population&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1004494&amp;volume=10&amp;publication_year=2014&amp;author=Lim%2CET" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">Ruotsalainen, S. E. et al. Loss-of-function of MFGE8 and protection against coronary atherosclerosis. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2021.06.23.21259381" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.06.23.21259381">https://doi.org/10.1101/2021.06.23.21259381</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR46">Nikpay, M. et al. A comprehensive 1,000 Genomes-based genome-wide association meta-analysis of coronary artery disease. <i>Nat. Genet.</i> <b>47</b>, 1121–1130 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fng.3396" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ng.3396" data-track-item_id="10.1038/ng.3396">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsV2lt7zO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26343387" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4589895" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20comprehensive%201%2C000%20Genomes-based%20genome-wide%20association%20meta-analysis%20of%20coronary%20artery%20disease&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3396&amp;volume=47&amp;pages=1121-1130&amp;publication_year=2015&amp;author=Nikpay%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR47">Mahajan, A. et al. Fine-mapping type 2 diabetes loci to single-variant resolution using high-density imputation and islet-specific epigenome maps. <i>Nat. Genet.</i> <b>50</b>, 1505–1513 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-018-0241-6" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-018-0241-6" data-track-item_id="10.1038/s41588-018-0241-6">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVCitLrM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30297969" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6287706" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fine-mapping%20type%202%20diabetes%20loci%20to%20single-variant%20resolution%20using%20high-density%20imputation%20and%20islet-specific%20epigenome%20maps&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0241-6&amp;volume=50&amp;pages=1505-1513&amp;publication_year=2018&amp;author=Mahajan%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR48">Lee, J. J. et al. Gene discovery and polygenic prediction from a genome-wide association study of educational attainment in 1.1 million individuals. <i>Nat. Genet.</i> <b>50</b>, 1112–1121 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-018-0147-3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-018-0147-3" data-track-item_id="10.1038/s41588-018-0147-3">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlOis7zI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30038396" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6393768" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Gene%20discovery%20and%20polygenic%20prediction%20from%20a%20genome-wide%20association%20study%20of%20educational%20attainment%20in%201.1%20million%20individuals&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0147-3&amp;volume=50&amp;pages=1112-1121&amp;publication_year=2018&amp;author=Lee%2CJJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR49">Wray, N. R. et al. Genome-wide association analyses identify 44 risk variants and refine the genetic architecture of major depression. <i>Nat. Genet.</i> <b>50</b>, 668–681 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-018-0090-3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-018-0090-3" data-track-item_id="10.1038/s41588-018-0090-3">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXosVCks74%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29700475" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5934326" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20association%20analyses%20identify%2044%20risk%20variants%20and%20refine%20the%20genetic%20architecture%20of%20major%20depression&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0090-3&amp;volume=50&amp;pages=668-681&amp;publication_year=2018&amp;author=Wray%2CNR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR50">Shiffman, J. &amp; Shawar, Y. R. Strengthening accountability of the global health metrics enterprise. <i>Lancet</i> <b>395</b>, 1452–1456 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0140-6736%2820%2930416-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0140-6736(20)30416-5" data-track-item_id="10.1016/S0140-6736(20)30416-5">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXnvFertL8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32305072" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7162633" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Strengthening%20accountability%20of%20the%20global%20health%20metrics%20enterprise&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2820%2930416-5&amp;volume=395&amp;pages=1452-1456&amp;publication_year=2020&amp;author=Shiffman%2CJ&amp;author=Shawar%2CYR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR51">Mukamel, R. E. et al. Protein-coding repeat polymorphisms strongly shape diverse human phenotypes. <i>Science</i> <b>373</b>, 1499–1505 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.abg8289" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.abg8289" data-track-item_id="10.1126/science.abg8289">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFOisbjF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34554798" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8549062" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Protein-coding%20repeat%20polymorphisms%20strongly%20shape%20diverse%20human%20phenotypes&amp;journal=Science&amp;doi=10.1126%2Fscience.abg8289&amp;volume=373&amp;pages=1499-1505&amp;publication_year=2021&amp;author=Mukamel%2CRE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR52">Giambartolomei, C. et al. Bayesian test for colocalisation between pairs of genetic association studies using summary statistics. <i>PLoS Genet.</i> <b>10</b>, e1004383 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pgen.1004383" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pgen.1004383" data-track-item_id="10.1371/journal.pgen.1004383">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24830394" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4022491" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Bayesian%20test%20for%20colocalisation%20between%20pairs%20of%20genetic%20association%20studies%20using%20summary%20statistics&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1004383&amp;volume=10&amp;publication_year=2014&amp;author=Giambartolomei%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR53">Guo, J. et al. Quantifying genetic heterogeneity between continental populations for human height and body mass index. <i>Sci. Rep.</i> <b>11</b>, 5240 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41598-021-84739-z" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41598-021-84739-z" data-track-item_id="10.1038/s41598-021-84739-z">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXlvF2gur4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33664403" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7933291" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Quantifying%20genetic%20heterogeneity%20between%20continental%20populations%20for%20human%20height%20and%20body%20mass%20index&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-021-84739-z&amp;volume=11&amp;publication_year=2021&amp;author=Guo%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR54">Kuchenbaecker, K. et al. The transferability of lipid loci across African, Asian and European cohorts. <i>Nat. Commun.</i> <b>10</b>, 4330 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-019-12026-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-019-12026-7" data-track-item_id="10.1038/s41467-019-12026-7">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31551420" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6760173" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20transferability%20of%20lipid%20loci%20across%20African%2C%20Asian%20and%20European%20cohorts&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-12026-7&amp;volume=10&amp;publication_year=2019&amp;author=Kuchenbaecker%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR55">Huang, Q. Q. et al. Transferability of genetic loci and polygenic scores for cardiometabolic traits in British Pakistani and Bangladeshi individuals. <i>Nat. Commun.</i> <b>13</b>, 4664 (2022). <a href="https://doi.org/10.1038/s41467-022-32095-5" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41467-022-32095-5">https://doi.org/10.1038/s41467-022-32095-5</a></p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR56">Shi, H. et al. Population-specific causal disease effect sizes in functionally important regions impacted by selection. <i>Nat. Commun.</i> <b>12</b>, 1–15 (2021).</p><p class="c-reading-companion__reference-links"><a href="http://scholar.google.com/scholar_lookup?&amp;title=Population-specific%20causal%20disease%20effect%20sizes%20in%20functionally%20important%20regions%20impacted%20by%20selection&amp;journal=Nat.%20Commun.&amp;volume=12&amp;pages=1-15&amp;publication_year=2021&amp;author=Shi%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR57">Patel, R. A. et al. Effect sizes of causal variants for gene expression and complex traits differ between populations. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/2021.12.06.471235" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.12.06.471235">https://doi.org/10.1101/2021.12.06.471235</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR58">Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. <i>Nat. Genet.</i> <b>51</b>, 584–591 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-019-0379-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-019-0379-x" data-track-item_id="10.1038/s41588-019-0379-x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslGltLw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30926966" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6563838" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20use%20of%20current%20polygenic%20risk%20scores%20may%20exacerbate%20health%20disparities&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0379-x&amp;volume=51&amp;pages=584-591&amp;publication_year=2019&amp;author=Martin%2CAR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR59">Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. <i>Nature</i> <b>581</b>, 434–443 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-020-2308-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-020-2308-7" data-track-item_id="10.1038/s41586-020-2308-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVanu7jF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32461654" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334197" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141%2C456%20humans&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2308-7&amp;volume=581&amp;pages=434-443&amp;publication_year=2020&amp;author=Karczewski%2CKJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR60">McLaren, W. et al. The ensembl variant effect predictor. <i>Genome Biol.</i> <b>17</b>, 122 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-016-0974-4" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-016-0974-4" data-track-item_id="10.1186/s13059-016-0974-4">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27268795" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4893825" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20ensembl%20variant%20effect%20predictor&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-016-0974-4&amp;volume=17&amp;publication_year=2016&amp;author=McLaren%2CW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR61">Laugesen, K. et al. Nordic health registry-based research: a review of health care systems and key registries. <i>Clin. Epidemiol.</i> <b>13</b>, 533–554 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.2147%2FCLEP.S314959" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.2147/CLEP.S314959" data-track-item_id="10.2147/CLEP.S314959">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34321928" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8302231" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Nordic%20health%20registry-based%20research%3A%20a%20review%20of%20health%20care%20systems%20and%20key%20registries&amp;journal=Clin.%20Epidemiol.&amp;doi=10.2147%2FCLEP.S314959&amp;volume=13&amp;pages=533-554&amp;publication_year=2021&amp;author=Laugesen%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR62">Vuori, M. A. et al. The validity of heart failure diagnoses in the Finnish Hospital Discharge Register. <i>Scand. J. Public Health</i> <a href="https://doi.org/10.1177/1403494819847051" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1177/1403494819847051">https://doi.org/10.1177/1403494819847051</a> (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR63">Leinonen, M. K., Miettinen, J., Heikkinen, S., Pitkäniemi, J. &amp; Malila, N. Quality measures of the population-based finnish cancer registry indicate sound data quality for solid malignant tumours. <i>Eur. J. Cancer</i> <b>77</b>, 31–39 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ejca.2017.02.017" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ejca.2017.02.017" data-track-item_id="10.1016/j.ejca.2017.02.017">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28350996" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Quality%20measures%20of%20the%20population-based%20finnish%20cancer%20registry%20indicate%20sound%20data%20quality%20for%20solid%20malignant%20tumours&amp;journal=Eur.%20J.%20Cancer&amp;doi=10.1016%2Fj.ejca.2017.02.017&amp;volume=77&amp;pages=31-39&amp;publication_year=2017&amp;author=Leinonen%2CMK&amp;author=Miettinen%2CJ&amp;author=Heikkinen%2CS&amp;author=Pitk%C3%A4niemi%2CJ&amp;author=Malila%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR64">Sund, R. et al. Comparing properties of audit data and routinely collected register data in case of performance assessment of hip fracture treatment in Finland. <i>Methods Inf. Med.</i> <b>46</b>, 558–566 (2007).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1160%2FME0382" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1160/ME0382" data-track-item_id="10.1160/ME0382">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BD2snhsVGlsg%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17938779" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Comparing%20properties%20of%20audit%20data%20and%20routinely%20collected%20register%20data%20in%20case%20of%20performance%20assessment%20of%20hip%20fracture%20treatment%20in%20Finland&amp;journal=Methods%20Inf.%20Med.&amp;doi=10.1160%2FME0382&amp;volume=46&amp;pages=558-566&amp;publication_year=2007&amp;author=Sund%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR65">Sund, R. Quality of the Finnish Hospital Discharge Register: a systematic review. <i>Scand. J. Public Health</i> <b>40</b>, 505–515 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1177%2F1403494812456637" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1177/1403494812456637" data-track-item_id="10.1177/1403494812456637">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22899561" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Quality%20of%20the%20Finnish%20Hospital%20Discharge%20Register%3A%20a%20systematic%20review&amp;journal=Scand.%20J.%20Public%20Health&amp;doi=10.1177%2F1403494812456637&amp;volume=40&amp;pages=505-515&amp;publication_year=2012&amp;author=Sund%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR66">Haukka, J. Finnish health and social welfare registers in epidemiological research. <i>Norsk Epidemiologi</i> <b>14</b>, 113−120 (2004).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR67">Bycroft, C. et al. The UK Biobank resource with deep phenotyping and genomic data. <i>Nature</i> <b>562</b>, 203–209 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-018-0579-z" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-018-0579-z" data-track-item_id="10.1038/s41586-018-0579-z">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvV2qsbvI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30305743" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786975" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20UK%20Biobank%20resource%20with%20deep%20phenotyping%20and%20genomic%20data&amp;journal=Nature&amp;doi=10.1038%2Fs41586-018-0579-z&amp;volume=562&amp;pages=203-209&amp;publication_year=2018&amp;author=Bycroft%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR68">Pärn, K., Nunez-Fontarnau, J., Isokallio M. A., Sipilä, T., Kilpeläinen, E., Palotie, A., Ripatti, S. &amp; Palta, P. Genotyping chip data lift-over to reference genome build GRCh38/hg38. protocols.io. <a href="https://doi.org/10.17504/protocols.io.xbhfij6" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.17504/protocols.io.xbhfij6">https://doi.org/10.17504/protocols.io.xbhfij6</a> (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR69">Pärn, K., Isokallio, M. A., Nunez-Fontarnau, J., Palotie, A., Ripatti, S. &amp; Palta, P. Genotype imputation workflow v3.0. protocols.io. <a href="https://doi.org/10.17504/protocols.io.xbgfijw" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.17504/protocols.io.xbgfijw">https://doi.org/10.17504/protocols.io.xbgfijw</a> (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR70">Ritari, J. et al. Increasing accuracy of HLA imputation by a population-specific reference panel in a FinnGen biobank cohort. <i>NAR Genomics Bioinforma.</i> <b>2</b>, lqaa030 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnargab%2Flqaa030" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nargab/lqaa030" data-track-item_id="10.1093/nargab/lqaa030">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Increasing%20accuracy%20of%20HLA%20imputation%20by%20a%20population-specific%20reference%20panel%20in%20a%20FinnGen%20biobank%20cohort&amp;journal=NAR%20Genomics%20Bioinforma.&amp;doi=10.1093%2Fnargab%2Flqaa030&amp;volume=2&amp;publication_year=2020&amp;author=Ritari%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR71">Zheng, X. et al. HIBAG—HLA genotype imputation with attribute bagging. <i>Pharmacogenomics J.</i> <b>14</b>, 192–200 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Ftpj.2013.18" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/tpj.2013.18" data-track-item_id="10.1038/tpj.2013.18">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXot1Cmurw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23712092" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=HIBAG%E2%80%94HLA%20genotype%20imputation%20with%20attribute%20bagging&amp;journal=Pharmacogenomics%20J.&amp;doi=10.1038%2Ftpj.2013.18&amp;volume=14&amp;pages=192-200&amp;publication_year=2014&amp;author=Zheng%2CX" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR72">Zhou, W. et al. Efficiently controlling for case-control imbalance and sample relatedness in large-scale genetic association studies. <i>Nat. Genet.</i> <b>50</b>, 1335–1341 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-018-0184-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-018-0184-y" data-track-item_id="10.1038/s41588-018-0184-y">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFSqtbfN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30104761" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6119127" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Efficiently%20controlling%20for%20case-control%20imbalance%20and%20sample%20relatedness%20in%20large-scale%20genetic%20association%20studies&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0184-y&amp;volume=50&amp;pages=1335-1341&amp;publication_year=2018&amp;author=Zhou%2CW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR73">Loh, P.-R. et al. Efficient Bayesian mixed-model analysis increases association power in large cohorts. <i>Nat. Genet.</i> <b>47</b>, 284–290 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fng.3190" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ng.3190" data-track-item_id="10.1038/ng.3190">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFCntb4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25642633" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342297" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Efficient%20Bayesian%20mixed-model%20analysis%20increases%20association%20power%20in%20large%20cohorts&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3190&amp;volume=47&amp;pages=284-290&amp;publication_year=2015&amp;author=Loh%2CP-R" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR74">Purcell, S. et al. PLINK: a tool set for whole-genome association and population-based linkage analyses. <i>Am. J. Hum. Genet</i> <b>81</b>, 559–575 (2007).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1086%2F519795" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1086/519795" data-track-item_id="10.1086/519795">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtVSqurrL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17701901" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1950838" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;journal=Am.%20J.%20Hum.%20Genet&amp;doi=10.1086%2F519795&amp;volume=81&amp;pages=559-575&amp;publication_year=2007&amp;author=Purcell%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR75">Ge, T., Chen, C.-Y., Ni, Y., Feng, Y.-C. A. &amp; Smoller, J. W. Polygenic prediction via Bayesian regression and continuous shrinkage priors. <i>Nat. Commun.</i> <b>10</b>, 1776 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-019-09718-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-019-09718-5" data-track-item_id="10.1038/s41467-019-09718-5">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30992449" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6467998" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20prediction%20via%20Bayesian%20regression%20and%20continuous%20shrinkage%20priors&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-09718-5&amp;volume=10&amp;publication_year=2019&amp;author=Ge%2CT&amp;author=Chen%2CC-Y&amp;author=Ni%2CY&amp;author=Feng%2CY-CA&amp;author=Smoller%2CJW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR76">Heinze, G. &amp; Schemper, M. A solution to the problem of monotone likelihood in Cox regression. <i>Biometrics</i> <b>57</b>, 114–119 (2001).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2Fj.0006-341X.2001.00114.x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/j.0006-341X.2001.00114.x" data-track-item_id="10.1111/j.0006-341X.2001.00114.x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BD3MzmvVaqsQ%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11252585" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20solution%20to%20the%20problem%20of%20monotone%20likelihood%20in%20Cox%20regression&amp;journal=Biometrics&amp;doi=10.1111%2Fj.0006-341X.2001.00114.x&amp;volume=57&amp;pages=114-119&amp;publication_year=2001&amp;author=Heinze%2CG&amp;author=Schemper%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR77">Manichaikul, A. et al. Robust relationship inference in genome-wide association studies. <i>Bioinformatics</i> <b>26</b>, 2867–2873 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq559" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btq559" data-track-item_id="10.1093/bioinformatics/btq559">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsVSlt7bK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20926424" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3025716" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20relationship%20inference%20in%20genome-wide%20association%20studies&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtq559&amp;volume=26&amp;pages=2867-2873&amp;publication_year=2010&amp;author=Manichaikul%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR78">Stensrud, M. J. &amp; Hernán, M. A. Why test for proportional hazards? <i>JAMA</i> <b>323</b>, 1401–1402 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1001%2Fjama.2020.1267" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1001/jama.2020.1267" data-track-item_id="10.1001/jama.2020.1267">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32167523" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Why%20test%20for%20proportional%20hazards%3F&amp;journal=JAMA&amp;doi=10.1001%2Fjama.2020.1267&amp;volume=323&amp;pages=1401-1402&amp;publication_year=2020&amp;author=Stensrud%2CMJ&amp;author=Hern%C3%A1n%2CMA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR79">Khosravi, A., Nazemipour, M., Shinozaki, T. &amp; Mansournia, M. A. Population attributable fraction in textbooks: time to revise. <i>Glob. Epidemiol.</i> <b>3</b>, 100062 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.gloepi.2021.100062" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.gloepi.2021.100062" data-track-item_id="10.1016/j.gloepi.2021.100062">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Population%20attributable%20fraction%20in%20textbooks%3A%20time%20to%20revise&amp;journal=Glob.%20Epidemiol.&amp;doi=10.1016%2Fj.gloepi.2021.100062&amp;volume=3&amp;publication_year=2021&amp;author=Khosravi%2CA&amp;author=Nazemipour%2CM&amp;author=Shinozaki%2CT&amp;author=Mansournia%2CMA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR80">Rowe, A. K., Powell, K. E. &amp; Flanders, W. D. Why population attributable fractions can sum to more than one. <i>Am. J. Prev. Med.</i> <b>26</b>, 243–249 (2004).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.amepre.2003.12.007" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.amepre.2003.12.007" data-track-item_id="10.1016/j.amepre.2003.12.007">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15026106" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Why%20population%20attributable%20fractions%20can%20sum%20to%20more%20than%20one&amp;journal=Am.%20J.%20Prev.%20Med.&amp;doi=10.1016%2Fj.amepre.2003.12.007&amp;volume=26&amp;pages=243-249&amp;publication_year=2004&amp;author=Rowe%2CAK&amp;author=Powell%2CKE&amp;author=Flanders%2CWD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR81">Witte, J. S., Visscher, P. M. &amp; Wray, N. R. The contribution of genetic variants to disease depends on the ruler. <i>Nat. Rev. Genet.</i> <b>15</b>, 765–776 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnrg3786" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nrg3786" data-track-item_id="10.1038/nrg3786">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFOltbfE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25223781" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4412738" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20contribution%20of%20genetic%20variants%20to%20disease%20depends%20on%20the%20ruler&amp;journal=Nat.%20Rev.%20Genet.&amp;doi=10.1038%2Fnrg3786&amp;volume=15&amp;pages=765-776&amp;publication_year=2014&amp;author=Witte%2CJS&amp;author=Visscher%2CPM&amp;author=Wray%2CNR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li></ol></div></div>
        </div>
    </div>
</aside>
</div>


    
        
    
    
        
            
        

        
            
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>




<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Medicine
                    </span>
                    (<i itemprop="alternateName">Nat Med</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-170X</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1078-8956</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/" data-track="click" data-track-action="protocols.io" data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/authors/editorial_policies" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nature-research/open-access" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/language-editing/" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/" data-track="click" data-track-action="research solutions" data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library" data-track="click" data-track-action="Recommend to library" data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica" data-track="click" data-track-action="nature africa" data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly" data-track="click" data-track-action="nature Italy" data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies" data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice" data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/accessibility-statement" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20">
    </a>
    <p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"></path></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"></path><clipPath id="b"><use xlink:href="#a" style="overflow:visible"></use></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"></path><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"></path><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"></path><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"></path></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"></path></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif " data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required="" data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required="">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41591-022-01957-2&amp;format=js&amp;last_modified=2022-09-12" async=""></script>

<mxx-slpkxvoqclcf></mxx-slpkxvoqclcf><div data-cc-ghost="" style="height: 352px;"></div><script type="text/javascript" id="fetch-contextual-ads-campaign-data" charset="">(function(){var g=function(a){a=new CustomEvent("campaignDataLoaded",{detail:a});document.dispatchEvent(a)},b=google_tag_manager["rm"]["50443292"](18);if(b){var k=1500;b=google_tag_manager["rm"]["50443292"](19);b="link"===b?google_tag_manager["rm"]["50443292"](20):google_tag_manager["rm"]["50443292"](21);var m=function(a,c){var h=!1,l=setTimeout(function(){h=!0;c(null)},k),d=new XMLHttpRequest;d.onload=function(){var e=null,f=[];if(200===d.status){try{e=JSON.parse(d.responseText)}catch(p){e={}}f=e.campaigns||[]}h||(clearTimeout(l),c(f.length?f.join(","):null))};d.open("GET",
"/platform/contextual?doi\x3d"+a);d.send()},n=function(a,c){a&&-1===a.indexOf("not set")?m(a,c):setTimeout(c,1)};n(b,function(a){window.campaignsForContextualAds=a;g(a);window.dataLayer.push({content:{article:{campaignID:a}}})})}else window.campaignsForContextualAds="",g(null)})();</script>
<script type="text/javascript" id="gtm-setup-accessdetailsloaded-handler" charset="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail||{};var b={event:"update-access-details"};a.institutional_business_partner_ids&&a.resolved_by||console.log("BPID data could not be retrieved from /exposed-details");a.logged_in_to_sn_profile||console.log("logged_in_to_sn_profile could not be retrieved from /exposed-details");var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?
a.resolved_by.join(";"):"",e=a.personal_business_partner_id_found,f=a.logged_in_to_sn_profile,g=a.snid;b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null,personal:e||null,logged_in_to_sn_profile:f,snid:g||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy={};window.dataLayer.push(b);window.idpUserDataLoaded=!0},!1);window.dataLayer.push({event:"accessdetailsloaded-handler-added"});
window.accessDetailsLoadedHandlerAdded=!0;</script><script type="text/javascript" id="gtm-mediatrust" charset="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="crossmark-script" text="" charset="" type="text/javascript" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="polyfill-matches" charset="">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="fetch-idp-user-data" charset="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},site=google_tag_manager["rm"]["50443292"](59),idpUrl;
switch(site){case "nature":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?"https://staging-idp.nature.com/exposed-details":"https://idp.nature.com/exposed-details";break;case "link":idpUrl=-1<window.location.hostname.indexOf("link-qa")?"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details";break;case "springer":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?
"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details"}if(void 0!==idpUrl){var transport=new XMLHttpRequest;transport.open("GET",idpUrl,!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script src="https://verify.nature.com/verify/nature.min.js"></script><script type="text/javascript" id="create-ad-slots" charset="">function createAndLoadAds(){function h(b){"nature"===a&&google_tag_manager["rm"]["50443292"](87)(b);"bmc"!==a&&"springeropen"!==a||google_tag_manager["rm"]["50443292"](99)(b);("link"===a&&"oscar"===google_tag_manager["rm"]["50443292"](100)||"springer"===a||"link"===a&&!0===google_tag_manager["rm"]["50443292"](101))&&google_tag_manager["rm"]["50443292"](126)(b);"link"===a&&"bunsen"===google_tag_manager["rm"]["50443292"](127)&&google_tag_manager["rm"]["50443292"](152)(b);"link"===a&&"Core"===google_tag_manager["rm"]["50443292"](153)&&google_tag_manager["rm"]["50443292"](165)(b)}window.googletag=window.googletag||{cmd:[]};var a=google_tag_manager["rm"]["50443292"](166),e=google_tag_manager["rm"]["50443292"](168);
if(-1===window.location.search.indexOf("hide_ads\x3dtrue"))if(window.adSlots&&0!==Object.keys(window.adSlots).length)console.log("Ads previously loaded. Will not update ad slots until next page load.");else{window.adSlots||(window.adSlots={});window.getAd=function(b,f){for(var c in window.adSlots)if(-1<c.indexOf(b)){if("object"===f)return adSlots[c];if("slot"===f)return adSlots[c].slot}};googletag.cmd.push(function(){googletag.pubads().setPrivacySettings({limitedAds:e});console.log("limitedAds is ",
e);googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["rm"]["50443292"](170));googletag.pubads().enableSingleRequest();googletag.pubads().disableInitialLoad()});for(var g=document.querySelectorAll("[data-gpt]"),d=0;g[d];++d)h(g[d]);googletag.cmd.push(function(){googletag.enableServices()})}}createAndLoadAds();</script>
</body></html>