<html lang="en" class="js"><head><link rel="preload" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" as="font" type="font/woff2" crossorigin=""><script async="" src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script type="text/javascript" async="" src="https://pagead2.googlesyndication.com/tag/js/gpt.js"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-MRVXSHQ"></script><script type="text/javascript" src="https://www.nature.com/rFxLXpM81qiOODayMwnzyJfBS1bMIN1Xv2p5Z4rLKbNKK0BzLus2POJgmPqLtqDbdNWlJT8d2opx5J6jdNoyrw=="></script><script src="https://cmp.nature.com/production_live/en/consent-bundle-8-85.js" onload="initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')"></script>
    <title>A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors | Nature Medicine</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nm.rss">


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin="">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480">

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"resource","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"biological-techniques;cancer;computational-biology-and-bioinformatics;gene-expression-analysis;genomic-analysis","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Medicine","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Resource","status":null}},"article":{"doi":"10.1038/s41591-020-0844-1"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Michal Slyper","Caroline B. M. Porter","Orr Ashenberg","Julia Waldman","Eugene Drokhlyansky","Isaac Wakiro","Christopher Smillie","Gabriela Smith-Rosario","Jingyi Wu","Danielle Dionne","Sébastien Vigneau","Judit Jané-Valbuena","Timothy L. Tickle","Sara Napolitano","Mei-Ju Su","Anand G. Patel","Asa Karlstrom","Simon Gritsch","Masashi Nomura","Avinash Waghray","Satyen H. Gohil","Alexander M. Tsankov","Livnat Jerby-Arnon","Ofir Cohen","Johanna Klughammer","Yanay Rosen","Joshua Gould","Lan Nguyen","Matan Hofree","Peter J. Tramontozzi","Bo Li","Catherine J. Wu","Benjamin Izar","Rizwan Haq","F. Stephen Hodi","Charles H. Yoon","Aaron N. Hata","Suzanne J. Baker","Mario L. Suvà","Raphael Bueno","Elizabeth H. Stover","Michael R. Clay","Michael A. Dyer","Natalie B. Collins","Ursula A. Matulonis","Nikhil Wagle","Bruce E. Johnson","Asaf Rotem","Orit Rozenblatt-Rosen","Aviv Regev"],"publishedAt":1589155200,"publishedAtString":"2020-05-11","title":"A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Biological techniques,Cancer,Computational biology and bioinformatics,Gene expression analysis,Genomic analysis"},"journal":{"pcode":"nm","title":"nature medicine","volume":"26","issue":"5","id":41591,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"BR","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{font-weight:700;-webkit-font-smoothing:antialiased;font-size:1.5rem;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.app-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{margin:4px 4px 0;fill:#888;height:10px;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-c9121fecab.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png">
<link rel="icon" type="image/png" sizes="48x48" href="/static/images/favicons/nature/favicon-48x48-b52890008c.png">
<link rel="icon" type="image/png" sizes="32x32" href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="/static/images/favicons/nature/favicon-16x16-951651ab72.png">
<link rel="manifest" href="/static/manifest.json" crossorigin="use-credentials">
<link rel="mask-icon" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="/static/images/favicons/nature/favicon.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-e9d2061c95.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-1cbf3a8b56.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script><script id="js-position1" data-test="global-article-js" defer="" src="/static/js/global-article-es6-bundle-c8a573ca90.js"></script><script id="js-position2" data-test="shared-js" defer="" src="/static/js/shared-es6-bundle-e9d2061c95.js"></script><script id="js-position3" data-test="header-150-js" defer="" src="/static/js/header-150-es6-bundle-5bb959eaa1.js"></script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors","description":"Single-cell genomics is essential to chart tumor ecosystems. Although single-cell RNA-Seq (scRNA-Seq) profiles RNA from cells dissociated from fresh tumors, single-nucleus RNA-Seq (snRNA-Seq) is needed to profile frozen or hard-to-dissociate tumors. Each requires customization to different tissue and tumor types, posing a barrier to adoption. Here, we have developed a systematic toolbox for profiling fresh and frozen clinical tumor samples using scRNA-Seq and snRNA-Seq, respectively. We analyzed 216,490 cells and nuclei from 40 samples across 23 specimens spanning eight tumor types of varying tissue and sample characteristics. We evaluated protocols by cell and nucleus quality, recovery rate and cellular composition. scRNA-Seq and snRNA-Seq from matched samples recovered the same cell types, but at different proportions. Our work provides guidance for studies in a broad range of tumors, including criteria for testing and selecting methods from the toolbox for other tumors, thus paving the way for charting tumor atlases. A set of ready-to-use tools for profiling fresh and frozen clinical tumor samples using scRNA-Seq and snRNA-Seq facilitates the implementation of single-cell technologies in clinical settings and the construction of single-cell tumor atlases.","datePublished":"2020-05-11T00:00:00Z","dateModified":"2020-06-25T00:00:00Z","pageStart":"792","pageEnd":"802","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41591-020-0844-1","keywords":["Biological techniques","Cancer","Computational biology and bioinformatics","Gene expression analysis","Genomic analysis","Biomedicine","general","Cancer Research","Metabolic Diseases","Infectious Diseases","Molecular Medicine","Neurosciences"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig6_HTML.png"],"isPartOf":{"name":"Nature Medicine","issn":["1546-170X","1078-8956"],"volumeNumber":"26","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Michal Slyper","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Caroline B. M. Porter","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Orr Ashenberg","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Julia Waldman","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Eugene Drokhlyansky","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Isaac Wakiro","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute","address":{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Christopher Smillie","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Gabriela Smith-Rosario","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jingyi Wu","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute","address":{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Danielle Dionne","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sébastien Vigneau","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute","address":{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Judit Jané-Valbuena","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Timothy L. Tickle","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sara Napolitano","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute","address":{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Mei-Ju Su","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute","address":{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Anand G. Patel","url":"http://orcid.org/0000-0002-1438-8275","affiliation":[{"name":"Department of Developmental Neurobiology, St Jude Children’s Research Hospital","address":{"name":"Department of Developmental Neurobiology, St Jude Children’s Research Hospital, Memphis, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Oncology, St Jude Children’s Research Hospital","address":{"name":"Department of Oncology, St Jude Children’s Research Hospital, Memphis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Asa Karlstrom","url":"http://orcid.org/0000-0003-3847-7309","affiliation":[{"name":"Department of Developmental Neurobiology, St Jude Children’s Research Hospital","address":{"name":"Department of Developmental Neurobiology, St Jude Children’s Research Hospital, Memphis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Simon Gritsch","affiliation":[{"name":"Department of Pathology, Massachusetts General Hospital and Harvard Medical School","address":{"name":"Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School","address":{"name":"Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Masashi Nomura","url":"http://orcid.org/0000-0001-6095-7849","affiliation":[{"name":"Department of Pathology, Massachusetts General Hospital and Harvard Medical School","address":{"name":"Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School","address":{"name":"Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Avinash Waghray","affiliation":[{"name":"Center for Regenerative Medicine, Massachusetts General Hospital","address":{"name":"Center for Regenerative Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Satyen H. Gohil","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Alexander M. Tsankov","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Icahn School of Medicine at Mount Sinai","address":{"name":"Icahn School of Medicine at Mount Sinai, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Livnat Jerby-Arnon","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ofir Cohen","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute","address":{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Johanna Klughammer","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Yanay Rosen","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Joshua Gould","url":"http://orcid.org/0000-0001-9581-5461","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Lan Nguyen","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Matan Hofree","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Peter J. Tramontozzi","url":"http://orcid.org/0000-0002-5540-0111","affiliation":[{"name":"Division of Thoracic Surgery, Brigham and Women’s Hospital","address":{"name":"Division of Thoracic Surgery, Brigham and Women’s Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Bo Li","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Immunology and Inflammatory Diseases, Division of Rheumatology, Allergy, and Immunology, Massachusetts General Hospital and Harvard Medical School","address":{"name":"Center for Immunology and Inflammatory Diseases, Division of Rheumatology, Allergy, and Immunology, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Catherine J. Wu","url":"http://orcid.org/0000-0002-3348-5054","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Internal Medicine, Brigham and Women’s Hospital","address":{"name":"Department of Internal Medicine, Brigham and Women’s Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School","address":{"name":"Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Benjamin Izar","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute","address":{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Laboratory for Systems Pharmacology, Harvard Medical School","address":{"name":"Laboratory for Systems Pharmacology, Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Immunology and Virology, Dana-Farber Cancer Institute","address":{"name":"Center for Immunology and Virology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Ludwig Center for Cancer Research at Harvard","address":{"name":"Ludwig Center for Cancer Research at Harvard, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Melanoma Disease Center, Dana-Farber Cancer Institute","address":{"name":"Melanoma Disease Center, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Columbia Center for Translational Immunology and Division of Hematology and Oncology, Columbia University Medical Center","address":{"name":"Columbia Center for Translational Immunology and Division of Hematology and Oncology, Columbia University Medical Center, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Rizwan Haq","url":"http://orcid.org/0000-0002-1290-9276","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"F. Stephen Hodi","affiliation":[{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Melanoma Disease Center, Dana-Farber Cancer Institute","address":{"name":"Melanoma Disease Center, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Charles H. Yoon","affiliation":[{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Surgical Oncology, Brigham and Women’s Hospital","address":{"name":"Department of Surgical Oncology, Brigham and Women’s Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Aaron N. Hata","affiliation":[{"name":"Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School","address":{"name":"Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medicine, Massachusetts General Hospital and Harvard Medical School","address":{"name":"Department of Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Suzanne J. Baker","affiliation":[{"name":"Department of Developmental Neurobiology, St Jude Children’s Research Hospital","address":{"name":"Department of Developmental Neurobiology, St Jude Children’s Research Hospital, Memphis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Mario L. Suvà","url":"http://orcid.org/0000-0001-9898-5351","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Pathology, Massachusetts General Hospital and Harvard Medical School","address":{"name":"Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School","address":{"name":"Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Raphael Bueno","affiliation":[{"name":"Division of Thoracic Surgery, Brigham and Women’s Hospital","address":{"name":"Division of Thoracic Surgery, Brigham and Women’s Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Elizabeth H. Stover","url":"http://orcid.org/0000-0002-6578-3216","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michael R. Clay","url":"http://orcid.org/0000-0003-1659-8927","affiliation":[{"name":"Department of Pathology, St Jude Children’s Research Hospital","address":{"name":"Department of Pathology, St Jude Children’s Research Hospital, Memphis, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michael A. Dyer","affiliation":[{"name":"Department of Developmental Neurobiology, St Jude Children’s Research Hospital","address":{"name":"Department of Developmental Neurobiology, St Jude Children’s Research Hospital, Memphis, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Howard Hughes Medical Institute","address":{"name":"Howard Hughes Medical Institute, Chevy Chase, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Natalie B. Collins","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Pediatric Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Pediatric Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Division of Pediatric Hematology and Oncology, Boston Children’s Hospital","address":{"name":"Division of Pediatric Hematology and Oncology, Boston Children’s Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ursula A. Matulonis","affiliation":[{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Nikhil Wagle","url":"http://orcid.org/0000-0003-3332-9438","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute","address":{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Internal Medicine, Brigham and Women’s Hospital","address":{"name":"Department of Internal Medicine, Brigham and Women’s Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School","address":{"name":"Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Bruce E. Johnson","affiliation":[{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute","address":{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Asaf Rotem","affiliation":[{"name":"Broad Institute of Harvard and MIT","address":{"name":"Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute","address":{"name":"Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute","address":{"name":"Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Orit Rozenblatt-Rosen","url":"http://orcid.org/0000-0001-6313-3570","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"orit@broadinstitute.org","@type":"Person"},{"name":"Aviv Regev","url":"http://orcid.org/0000-0003-3293-3158","affiliation":[{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT","address":{"name":"Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Howard Hughes Medical Institute","address":{"name":"Howard Hughes Medical Institute, Chevy Chase, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Koch Institute for Integrative Cancer Research, Department of Biology, Massachusetts Institute of Technology","address":{"name":"Koch Institute for Integrative Cancer Research, Department of Biology, Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"aregev@broadinstitute.org","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41591-020-0844-1">
    
    
    <meta name="journal_id" content="41591">
    <meta name="dc.title" content="A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors">
    <meta name="dc.source" content="Nature Medicine 2020 26:5">
    <meta name="dc.format" content="text/html">
    <meta name="dc.publisher" content="Nature Publishing Group">
    <meta name="dc.date" content="2020-05-11">
    <meta name="dc.type" content="OriginalPaper">
    <meta name="dc.language" content="En">
    <meta name="dc.copyright" content="2020 The Author(s)">
    <meta name="dc.rights" content="2020 The Author(s)">
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="dc.description" content="Single-cell genomics is essential to chart tumor ecosystems. Although single-cell RNA-Seq (scRNA-Seq) profiles RNA from cells dissociated from fresh tumors, single-nucleus RNA-Seq (snRNA-Seq) is needed to profile frozen or hard-to-dissociate tumors. Each requires customization to different tissue and tumor types, posing a barrier to adoption. Here, we have developed a systematic toolbox for profiling fresh and frozen clinical tumor samples using scRNA-Seq and snRNA-Seq, respectively. We analyzed 216,490 cells and nuclei from 40 samples across 23 specimens spanning eight tumor types of varying tissue and sample characteristics. We evaluated protocols by cell and nucleus quality, recovery rate and cellular composition. scRNA-Seq and snRNA-Seq from matched samples recovered the same cell types, but at different proportions. Our work provides guidance for studies in a broad range of tumors, including criteria for testing and selecting methods from the toolbox for other tumors, thus paving the way for charting tumor atlases. A set of ready-to-use tools for profiling fresh and frozen clinical tumor samples using scRNA-Seq and snRNA-Seq facilitates the implementation of single-cell technologies in clinical settings and the construction of single-cell tumor atlases.">
    <meta name="prism.issn" content="1546-170X">
    <meta name="prism.publicationName" content="Nature Medicine">
    <meta name="prism.publicationDate" content="2020-05-11">
    <meta name="prism.volume" content="26">
    <meta name="prism.number" content="5">
    <meta name="prism.section" content="OriginalPaper">
    <meta name="prism.startingPage" content="792">
    <meta name="prism.endingPage" content="802">
    <meta name="prism.copyright" content="2020 The Author(s)">
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="prism.url" content="https://www.nature.com/articles/s41591-020-0844-1">
    <meta name="prism.doi" content="doi:10.1038/s41591-020-0844-1">
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41591-020-0844-1.pdf">
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41591-020-0844-1">
    <meta name="citation_journal_title" content="Nature Medicine">
    <meta name="citation_journal_abbrev" content="Nat Med">
    <meta name="citation_publisher" content="Nature Publishing Group">
    <meta name="citation_issn" content="1546-170X">
    <meta name="citation_title" content="A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors">
    <meta name="citation_volume" content="26">
    <meta name="citation_issue" content="5">
    <meta name="citation_publication_date" content="2020/05">
    <meta name="citation_online_date" content="2020/05/11">
    <meta name="citation_firstpage" content="792">
    <meta name="citation_lastpage" content="802">
    <meta name="citation_article_type" content="Resource">
    <meta name="citation_fulltext_world_readable" content="">
    <meta name="citation_language" content="en">
    <meta name="dc.identifier" content="doi:10.1038/s41591-020-0844-1">
    <meta name="DOI" content="10.1038/s41591-020-0844-1">
    <meta name="size" content="238296">
    <meta name="citation_doi" content="10.1038/s41591-020-0844-1">
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41591-020-0844-1&amp;api_key=">
    <meta name="description" content="Single-cell genomics is essential to chart tumor ecosystems. Although single-cell RNA-Seq (scRNA-Seq) profiles RNA from cells dissociated from fresh tumors, single-nucleus RNA-Seq (snRNA-Seq) is needed to profile frozen or hard-to-dissociate tumors. Each requires customization to different tissue and tumor types, posing a barrier to adoption. Here, we have developed a systematic toolbox for profiling fresh and frozen clinical tumor samples using scRNA-Seq and snRNA-Seq, respectively. We analyzed 216,490 cells and nuclei from 40 samples across 23 specimens spanning eight tumor types of varying tissue and sample characteristics. We evaluated protocols by cell and nucleus quality, recovery rate and cellular composition. scRNA-Seq and snRNA-Seq from matched samples recovered the same cell types, but at different proportions. Our work provides guidance for studies in a broad range of tumors, including criteria for testing and selecting methods from the toolbox for other tumors, thus paving the way for charting tumor atlases. A set of ready-to-use tools for profiling fresh and frozen clinical tumor samples using scRNA-Seq and snRNA-Seq facilitates the implementation of single-cell technologies in clinical settings and the construction of single-cell tumor atlases.">
    <meta name="dc.creator" content="Slyper, Michal">
    <meta name="dc.creator" content="Porter, Caroline B. M.">
    <meta name="dc.creator" content="Ashenberg, Orr">
    <meta name="dc.creator" content="Waldman, Julia">
    <meta name="dc.creator" content="Drokhlyansky, Eugene">
    <meta name="dc.creator" content="Wakiro, Isaac">
    <meta name="dc.creator" content="Smillie, Christopher">
    <meta name="dc.creator" content="Smith-Rosario, Gabriela">
    <meta name="dc.creator" content="Wu, Jingyi">
    <meta name="dc.creator" content="Dionne, Danielle">
    <meta name="dc.creator" content="Vigneau, Sébastien">
    <meta name="dc.creator" content="Jané-Valbuena, Judit">
    <meta name="dc.creator" content="Tickle, Timothy L.">
    <meta name="dc.creator" content="Napolitano, Sara">
    <meta name="dc.creator" content="Su, Mei-Ju">
    <meta name="dc.creator" content="Patel, Anand G.">
    <meta name="dc.creator" content="Karlstrom, Asa">
    <meta name="dc.creator" content="Gritsch, Simon">
    <meta name="dc.creator" content="Nomura, Masashi">
    <meta name="dc.creator" content="Waghray, Avinash">
    <meta name="dc.creator" content="Gohil, Satyen H.">
    <meta name="dc.creator" content="Tsankov, Alexander M.">
    <meta name="dc.creator" content="Jerby-Arnon, Livnat">
    <meta name="dc.creator" content="Cohen, Ofir">
    <meta name="dc.creator" content="Klughammer, Johanna">
    <meta name="dc.creator" content="Rosen, Yanay">
    <meta name="dc.creator" content="Gould, Joshua">
    <meta name="dc.creator" content="Nguyen, Lan">
    <meta name="dc.creator" content="Hofree, Matan">
    <meta name="dc.creator" content="Tramontozzi, Peter J.">
    <meta name="dc.creator" content="Li, Bo">
    <meta name="dc.creator" content="Wu, Catherine J.">
    <meta name="dc.creator" content="Izar, Benjamin">
    <meta name="dc.creator" content="Haq, Rizwan">
    <meta name="dc.creator" content="Hodi, F. Stephen">
    <meta name="dc.creator" content="Yoon, Charles H.">
    <meta name="dc.creator" content="Hata, Aaron N.">
    <meta name="dc.creator" content="Baker, Suzanne J.">
    <meta name="dc.creator" content="Suvà, Mario L.">
    <meta name="dc.creator" content="Bueno, Raphael">
    <meta name="dc.creator" content="Stover, Elizabeth H.">
    <meta name="dc.creator" content="Clay, Michael R.">
    <meta name="dc.creator" content="Dyer, Michael A.">
    <meta name="dc.creator" content="Collins, Natalie B.">
    <meta name="dc.creator" content="Matulonis, Ursula A.">
    <meta name="dc.creator" content="Wagle, Nikhil">
    <meta name="dc.creator" content="Johnson, Bruce E.">
    <meta name="dc.creator" content="Rotem, Asaf">
    <meta name="dc.creator" content="Rozenblatt-Rosen, Orit">
    <meta name="dc.creator" content="Regev, Aviv">
    <meta name="dc.subject" content="Biological techniques">
    <meta name="dc.subject" content="Cancer">
    <meta name="dc.subject" content="Computational biology and bioinformatics">
    <meta name="dc.subject" content="Gene expression analysis">
    <meta name="dc.subject" content="Genomic analysis">
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Genet.; citation_title=Cancer transcriptome profiling at the juncture of clinical translation; citation_author=M Cieslik, AM Chinnaiyan; citation_volume=19; citation_publication_date=2018; citation_pages=93-109; citation_doi=10.1038/nrg.2017.96; citation_id=CR1">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Developmental and oncogenic programs in H3K27M gliomas dissected by single-cell RNA-seq; citation_author=MG Filbin; citation_volume=360; citation_publication_date=2018; citation_pages=331-335; citation_doi=10.1126/science.aao4750; citation_id=CR2">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=A cancer cell program promotes T cell exclusion and resistance to checkpoint blockade; citation_author=L Jerby-Arnon; citation_volume=175; citation_publication_date=2018; citation_pages=984-997.e924; citation_doi=10.1016/j.cell.2018.09.006; citation_id=CR3">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer; citation_author=SV Puram; citation_volume=171; citation_publication_date=2017; citation_pages=1611-1624.e1624; citation_doi=10.1016/j.cell.2017.10.044; citation_id=CR4">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma; citation_author=I Tirosh; citation_volume=539; citation_publication_date=2016; citation_pages=309-313; citation_doi=10.1038/nature20123; citation_id=CR5">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Decoupling genetics, lineages and microenvironment in IDH-mutant gliomas by single-cell RNA-seq; citation_author=AS Venteicher; citation_volume=355; citation_publication_date=2017; citation_pages=eaai8478; citation_doi=10.1126/science.aai8478; citation_id=CR6">
    <meta name="citation_reference" content="Moonshot Cancer Initiative, Generation of Human Tumor Atlases National Cancer Institute (accessed 20 April 2020); 
                  https://ccr.cancer.gov/research/cancer-moonshot
                  
                ">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Massively parallel single-nucleus RNA-seq with DroNc-seq; citation_author=N Habib; citation_volume=14; citation_publication_date=2017; citation_pages=955-958; citation_doi=10.1038/nmeth.4407; citation_id=CR8">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Div-Seq: single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons; citation_author=N Habib; citation_volume=353; citation_publication_date=2016; citation_pages=925-928; citation_doi=10.1126/science.aad7038; citation_id=CR9">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Nuclei multiplexing with barcoded antibodies for single-nucleus genomics; citation_author=JT Gaublomme; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-10756-2; citation_id=CR10">
    <meta name="citation_reference" content="Drokhlyansky, E. et al. The enteric nervous system of the human and mouse colon at a single-cell resolution. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/746743v4
                  
                 (2019).">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data; citation_author=ATL Lun; citation_volume=20; citation_publication_date=2019; citation_doi=10.1186/s13059-019-1662-y; citation_id=CR12">
    <meta name="citation_reference" content="citation_journal_title=Cell Syst; citation_title=Scrublet: computational identification of cell doublets in single-cell transcriptomic data; citation_author=SL Wolock, R Lopez, AM Klein; citation_volume=8; citation_publication_date=2019; citation_pages=281-291.e289; citation_doi=10.1016/j.cels.2018.11.005; citation_id=CR13">
    <meta name="citation_reference" content="Li, B. et al. Cumulus: a cloud-based data analysis framework for large-scale single-cell and single-nucleus RNA-seq. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/823682v1
                  
                 (2019).">
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=Prospective identification of tumorigenic breast cancer cells; citation_author=M Al-Hajj, MS Wicha, A Benito-Hernandez, SJ Morrison, MF Clarke; citation_volume=100; citation_publication_date=2003; citation_pages=3983-3988; citation_doi=10.1073/pnas.0530291100; citation_id=CR15">
    <meta name="citation_reference" content="citation_journal_title=Cancer Res.; citation_title=A method for dissociation of viable human breast cancer cells that produces flow cytometric kinetic information similar to that obtained by thymidine labeling; citation_author=RW McDivitt, KR Stone, JS Meyer; citation_volume=44; citation_publication_date=1984; citation_pages=2628-2633; citation_id=CR16">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=An integrative model of cellular states, plasticity and genetics for glioblastoma; citation_author=C Neftel; citation_volume=178; citation_publication_date=2019; citation_pages=835-849; citation_doi=10.1016/j.cell.2019.06.024; citation_id=CR17">
    <meta name="citation_reference" content="Young, M. D. &amp; Behjati, S. SoupX removes ambient RNA contamination from droplet based single cell RNA sequencing data. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/303727v1
                  
                 (2018).">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Orthotopic patient-derived xenografts of paediatric solid tumours; citation_author=E Stewart; citation_volume=549; citation_publication_date=2017; citation_pages=96-100; citation_doi=10.1038/nature23647; citation_id=CR19">
    <meta name="citation_reference" content="citation_journal_title=Dev. Biol.; citation_title=Development and characterization of a human orthotopic neuroblastoma xenograft; citation_author=E Stewart; citation_volume=407; citation_publication_date=2015; citation_pages=344-355; citation_doi=10.1016/j.ydbio.2015.02.002; citation_id=CR20">
    <meta name="citation_reference" content="citation_journal_title=Sci. Rep.; citation_title=Cryopreservation of primary B cells minimally influences their signaling responses; citation_author=JU Hermansen, GE Tjonnfjord, LA Munthe, K Tasken, SS Skanland; citation_volume=8; citation_publication_date=2018; citation_doi=10.1038/s41598-018-36121-9; citation_id=CR21">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Single-cell transcriptome conservation in cryopreserved cells and tissues; citation_author=A Guillaumet-Adkins; citation_volume=18; citation_publication_date=2017; citation_doi=10.1186/s13059-017-1171-9; citation_id=CR22">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Nanogrid single-nucleus RNA sequencing reveals phenotypic diversity in breast cancer; citation_author=R Gao; citation_volume=8; citation_publication_date=2017; citation_doi=10.1038/s41467-017-00244-w; citation_id=CR23">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Integrating single-cell transcriptomic data across different conditions, technologies and species; citation_author=A Butler, P Hoffman, P Smibert, E Papalexi, R Satija; citation_volume=36; citation_publication_date=2018; citation_pages=411-420; citation_doi=10.1038/nbt.4096; citation_id=CR24">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Single-cell sequencing reveals dissociation-induced gene expression in tissue subpopulations; citation_author=SC Brink; citation_volume=14; citation_publication_date=2017; citation_pages=935-936; citation_doi=10.1038/nmeth.4437; citation_id=CR25">
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Cancer; citation_title=Unravelling biology and shifting paradigms in cancer with single-cell sequencing; citation_author=T Baslan, J Hicks; citation_volume=17; citation_publication_date=2017; citation_pages=557-569; citation_doi=10.1038/nrc.2017.58; citation_id=CR26">
    <meta name="citation_reference" content="citation_journal_title=Mol. Cell; citation_title=Single-cell RNA sequencing in cancer: lessons learned and emerging challenges; citation_author=ML Suva, I Tirosh; citation_volume=75; citation_publication_date=2019; citation_pages=7-12; citation_doi=10.1016/j.molcel.2019.05.003; citation_id=CR27">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Toward minimal residual disease-directed therapy in melanoma; citation_author=F Rambow; citation_volume=174; citation_publication_date=2018; citation_pages=843-855.e819; citation_doi=10.1016/j.cell.2018.06.025; citation_id=CR28">
    <meta name="citation_reference" content="citation_journal_title=Oral Oncol; citation_title=Single-cell sequencing and its applications in head and neck cancer; citation_author=Z Qi, T Barrett, AS Parikh, I Tirosh, SV Puram; citation_volume=99; citation_publication_date=2019; citation_pages=104441; citation_doi=10.1016/j.oraloncology.2019.104441; citation_id=CR29">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Single-cell RNA-Seq reveals AML hierarchies relevant to disease progression and immunity; citation_author=P Galen; citation_volume=176; citation_publication_date=2019; citation_pages=1265-1281.e1224; citation_doi=10.1016/j.cell.2019.01.031; citation_id=CR30">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=An immune atlas of clear cell renal cell carcinoma; citation_author=S Chevrier; citation_volume=169; citation_publication_date=2017; citation_pages=736-749.e718; citation_doi=10.1016/j.cell.2017.04.016; citation_id=CR31">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=Single-cell profiling of breast cancer T cells reveals a tissue-resident memory subset associated with improved prognosis; citation_author=P Savas; citation_volume=24; citation_publication_date=2018; citation_pages=986-993; citation_doi=10.1038/s41591-018-0078-7; citation_id=CR32">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Unravelling subclonal heterogeneity and aggressive disease states in TNBC through single-cell RNA-seq; citation_author=M Karaayvaz; citation_volume=9; citation_publication_date=2018; citation_doi=10.1038/s41467-018-06052-0; citation_id=CR33">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=Phenotype molding of stromal cells in the lung tumor microenvironment; citation_author=D Lambrechts; citation_volume=24; citation_publication_date=2018; citation_pages=1277-1289; citation_doi=10.1038/s41591-018-0096-5; citation_id=CR34">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Simultaneous epitope and transcriptome measurement in single cells; citation_author=M Stoeckius; citation_volume=14; citation_publication_date=2017; citation_pages=865-868; citation_doi=10.1038/nmeth.4380; citation_id=CR35">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Multiplexed droplet single-cell RNA-sequencing using natural genetic variation; citation_author=HM Kang; citation_volume=36; citation_publication_date=2018; citation_pages=89-94; citation_doi=10.1038/nbt.4042; citation_id=CR36">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol; citation_title=Cell hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics; citation_author=M Stoeckius; citation_volume=19; citation_publication_date=2018; citation_doi=10.1186/s13059-018-1603-1; citation_id=CR37">
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Single-nucleus and single-cell transcriptomes compared in matched cortical cell types; citation_author=TE Bakken; citation_volume=13; citation_publication_date=2018; citation_pages=e0209648; citation_doi=10.1371/journal.pone.0209648; citation_id=CR38">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Comprehensive classification of retinal bipolar neurons by single-cell transcriptomics; citation_author=K Shekhar; citation_volume=166; citation_publication_date=2016; citation_pages=1308-1323.e1330; citation_doi=10.1016/j.cell.2016.07.054; citation_id=CR39">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=SCANPY: large-scale single-cell gene expression data analysis; citation_author=FA Wolf, P Angerer, FJ Theis; citation_volume=19; citation_publication_date=2018; citation_doi=10.1186/s13059-017-1382-0; citation_id=CR40">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets; citation_author=EZ Macosko; citation_volume=161; citation_publication_date=2015; citation_pages=1202-1214; citation_id=CR41">
    <meta name="citation_reference" content="citation_journal_title=J. R. Stat. Soc. B; citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing; citation_author=Y Benjamini, Hochberg Yosef; citation_volume=57.1; citation_publication_date=1995; citation_pages=289-300; citation_id=CR42">
    <meta name="citation_reference" content="McInnes, L., Healy, J. &amp; Melville, J. UMAP: Uniform Manifold Approximation and Projection for dimension reduction. Preprint at 
                  https://arxiv.org/abs/1802.03426
                  
                 (2018).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Immunol.; citation_title=Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage; citation_author=D Aran; citation_volume=20; citation_publication_date=2019; citation_pages=163-172; citation_doi=10.1038/s41590-018-0276-y; citation_id=CR44">
    <meta name="citation_reference" content="Tickle, T. I., Georgescu, C., Brown, M. &amp; Haas, B. inferCNV of the Trinity CTAT Project (2019); 
                  https://github.com/broadinstitute/inferCNV
                  
                ">
    <meta name="citation_author" content="Slyper, Michal">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Porter, Caroline B. M.">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Ashenberg, Orr">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Waldman, Julia">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Drokhlyansky, Eugene">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Wakiro, Isaac">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Smillie, Christopher">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Smith-Rosario, Gabriela">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Wu, Jingyi">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Dionne, Danielle">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Vigneau, Sébastien">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Jané-Valbuena, Judit">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Tickle, Timothy L.">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Napolitano, Sara">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Su, Mei-Ju">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Patel, Anand G.">
    <meta name="citation_author_institution" content="Department of Developmental Neurobiology, St Jude Children’s Research Hospital, Memphis, USA">
    <meta name="citation_author_institution" content="Department of Oncology, St Jude Children’s Research Hospital, Memphis, USA">
    <meta name="citation_author" content="Karlstrom, Asa">
    <meta name="citation_author_institution" content="Department of Developmental Neurobiology, St Jude Children’s Research Hospital, Memphis, USA">
    <meta name="citation_author" content="Gritsch, Simon">
    <meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Nomura, Masashi">
    <meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Waghray, Avinash">
    <meta name="citation_author_institution" content="Center for Regenerative Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Gohil, Satyen H.">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Tsankov, Alexander M.">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Icahn School of Medicine at Mount Sinai, New York, USA">
    <meta name="citation_author" content="Jerby-Arnon, Livnat">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Cohen, Ofir">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Klughammer, Johanna">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Rosen, Yanay">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Gould, Joshua">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Nguyen, Lan">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Hofree, Matan">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Tramontozzi, Peter J.">
    <meta name="citation_author_institution" content="Division of Thoracic Surgery, Brigham and Women’s Hospital, Boston, USA">
    <meta name="citation_author" content="Li, Bo">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Center for Immunology and Inflammatory Diseases, Division of Rheumatology, Allergy, and Immunology, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Wu, Catherine J.">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Department of Internal Medicine, Brigham and Women’s Hospital, Boston, USA">
    <meta name="citation_author_institution" content="Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Izar, Benjamin">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Laboratory for Systems Pharmacology, Harvard Medical School, Boston, USA">
    <meta name="citation_author_institution" content="Center for Immunology and Virology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Ludwig Center for Cancer Research at Harvard, Boston, USA">
    <meta name="citation_author_institution" content="Melanoma Disease Center, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Columbia Center for Translational Immunology and Division of Hematology and Oncology, Columbia University Medical Center, New York, USA">
    <meta name="citation_author" content="Haq, Rizwan">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Hodi, F. Stephen">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Melanoma Disease Center, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Yoon, Charles H.">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Department of Surgical Oncology, Brigham and Women’s Hospital, Boston, USA">
    <meta name="citation_author" content="Hata, Aaron N.">
    <meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Baker, Suzanne J.">
    <meta name="citation_author_institution" content="Department of Developmental Neurobiology, St Jude Children’s Research Hospital, Memphis, USA">
    <meta name="citation_author" content="Suvà, Mario L.">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Bueno, Raphael">
    <meta name="citation_author_institution" content="Division of Thoracic Surgery, Brigham and Women’s Hospital, Boston, USA">
    <meta name="citation_author" content="Stover, Elizabeth H.">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Clay, Michael R.">
    <meta name="citation_author_institution" content="Department of Pathology, St Jude Children’s Research Hospital, Memphis, USA">
    <meta name="citation_author" content="Dyer, Michael A.">
    <meta name="citation_author_institution" content="Department of Developmental Neurobiology, St Jude Children’s Research Hospital, Memphis, USA">
    <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, USA">
    <meta name="citation_author" content="Collins, Natalie B.">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Pediatric Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Division of Pediatric Hematology and Oncology, Boston Children’s Hospital, Boston, USA">
    <meta name="citation_author" content="Matulonis, Ursula A.">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Wagle, Nikhil">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Department of Internal Medicine, Brigham and Women’s Hospital, Boston, USA">
    <meta name="citation_author_institution" content="Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Johnson, Bruce E.">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Rotem, Asaf">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author_institution" content="Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, USA">
    <meta name="citation_author" content="Rozenblatt-Rosen, Orit">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author" content="Regev, Aviv">
    <meta name="citation_author_institution" content="Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, USA">
    <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, USA">
    <meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research, Department of Biology, Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">
    <meta name="twitter:site" content="@naturemedicine">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image:alt" content="Content cover image">
    <meta name="twitter:title" content="A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors">
    <meta name="twitter:description" content="Nature Medicine - A set of ready-to-use tools for profiling fresh and frozen clinical tumor samples using scRNA-Seq and snRNA-Seq facilitates the implementation of single-cell technologies in...">
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig1_HTML.png">
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41591-020-0844-1">
    <meta property="og:type" content="article">
    <meta property="og:site_name" content="Nature">
    <meta property="og:title" content="A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors - Nature Medicine">
    <meta property="og:description" content="A set of ready-to-use tools for profiling fresh and frozen clinical tumor samples using scRNA-Seq and snRNA-Seq facilitates the implementation of single-cell technologies in clinical settings and the construction of single-cell tumor atlases.">
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig1_HTML.png">
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
<style type="text/css">.cc-banner{background-color:#01324b;border:none!important;bottom:0;box-sizing:border-box;color:#fff!important;left:0;line-height:1.3;margin:auto 0 0;max-width:100%;outline:0;overflow:visible;padding:0;position:fixed;right:0;width:100%;z-index:99999}.cc-banner::backdrop{background-color:#0000004d}.cc-banner *{color:inherit!important}.cc-banner:focus{box-shadow:none;outline:0}.cc-banner a{color:#fff!important;text-decoration:underline}.cc-banner a:active,.cc-banner a:focus,.cc-banner a:hover{color:inherit;text-decoration:none}.cc-banner a:focus{outline:3px solid #08c!important}.cc-banner h2,.cc-banner h3,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-banner .cc-h2,.cc-banner h2{font-size:18px}.cc-banner .cc-h3,.cc-banner h3{font-size:16px}.cc-banner .cc-h4,.cc-banner .cc-h5,.cc-banner .cc-h6,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-size:14px}.cc-banner .cc-button{font-size:16px}.cc-banner__content{background-color:#01324b;display:flex;flex-direction:column;margin:0 auto;max-height:90vh;max-width:100%;padding:16px;position:relative}.cc-banner__content:focus{outline:0}@media (min-width:680px){.cc-banner__content{padding:12px}}@media (min-width:980px){.cc-banner__content{max-height:60vh;padding-bottom:20px;padding-top:20px}}@media (min-width:1320px){.cc-banner__content{padding-bottom:40px;padding-top:40px}}.cc-banner__container{display:flex;flex-direction:column;margin:auto;max-width:1320px;overflow:auto}.cc-banner__title{background:none!important;border:0;flex-shrink:0;font-size:18px!important;font-size:22px!important;font-weight:700!important;letter-spacing:normal;margin:0 0 12px!important;padding:0}@media (min-width:680px){.cc-banner__title{font-size:24px!important;margin:0 0 16px!important}}@media (min-width:1320px){.cc-banner__title{font-size:26px!important;margin:0 0 24px!important}}.cc-banner__body{display:flex;flex-direction:column;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif!important;overflow-x:hidden;overflow-y:auto;padding:3px 3px 16px}@media (min-width:980px){.cc-banner__body{flex-direction:row}}.cc-banner__policy p{font-size:16px!important;margin:0;max-width:none;padding:0}.cc-banner__policy p:not(:last-child){margin:0 0 16px}@media (min-width:980px){.cc-banner__policy p:not(:last-child){margin:0 0 24px}}.cc-banner__policy p a{font-size:16px!important;font-weight:700}.cc-banner__footer{box-shadow:none;flex-shrink:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif !important!important;margin:0;padding:12px 0 3px 3px;position:relative}@media (min-width:380px){.cc-banner__footer{align-items:stretch;display:flex;flex-wrap:wrap}}@media (min-width:680px){.cc-banner__footer{box-shadow:none;flex-wrap:nowrap;width:40%}}.cc-banner__footer .cc-banner__button{flex:1 1 auto;font-weight:700;overflow:hidden;padding:.5em 1em;width:100%}@media (min-width:680px){.cc-banner__footer .cc-banner__button{max-width:275px}}.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:12px}@media (min-width:680px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:0;margin-right:16px}}@media (min-width:980px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-right:24px}}@media (min-width:680px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:16px!important;margin-right:0!important}}@media (min-width:980px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:24px!important;margin-right:0!important}}.cc-banner__button-preferences{padding-left:0;padding-right:0}@media (min-width:380px){.cc-banner__button-preferences{flex:0 0 auto;margin:auto}}@media (min-width:680px){.cc-banner__button-preferences{margin:0}}@media (min-width:380px) and (max-width:680px){.cc-banner__button-break{display:block}}@media (min-width:680px){.cc-banner--is-tcf .cc-banner__footer{width:auto}}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__footer{padding-right:48px;width:66%}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px!important}.cc-banner--is-tcf .cc-banner__stacks{margin:16px 0 0}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__stacks{display:flex;flex:0 0 33%;flex-direction:column;margin:0 0 0 48px;overflow:auto}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px;margin:0 0 12px}.cc-banner--is-tcf .cc-banner__stacks-details{padding:0 3px}.cc-banner--is-tcf .cc-banner .cc-details{border-color:#fff3!important}.cc-box{border-radius:10px;padding:12px}.cc-box--info{background-color:#eff6fb}.cc-box--light{background-color:#faf9f6}.cc-button{border:2px solid #0000;border-radius:32px!important;color:inherit;cursor:pointer;font-size:14px;font-weight:700!important;left:auto;line-height:1.2;margin:0;padding:.5em 1em;right:auto;text-transform:none!important;transition:all .2s}@media (min-width:680px){.cc-button{font-size:14px;padding:.75em 1em}}.cc-button--sm{font-size:12px;padding:2px 8px!important}.cc-button--primary{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--primary:focus{outline:3px solid #08c}.cc-button--primary:focus,.cc-button--primary:hover,.cc-button--secondary{background-color:#fff!important;border-color:#025e8d!important;color:#025e8d!important}.cc-button--secondary:focus{background-color:#025e8d!important;border-color:#fff!important;color:#fff!important;outline:3px solid #08c!important}.cc-button--secondary:hover{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--secondary:active{border:2px solid #01324b!important}.cc-button--secondary.cc-button--contrast{border-color:#fff!important}.cc-button--secondary.cc-button--contrast:hover{background-color:initial!important;color:#fff!important}.cc-button--tertiary{background-color:#f0f7fc!important;background-image:none;border:2px solid #f0f7fc;color:#025e8d!important;text-decoration:underline!important;text-underline-offset:.25em}.cc-button--tertiary:focus{outline:3px solid #08c!important}.cc-button--tertiary:hover{text-decoration-thickness:.25em}.cc-button--tertiary:active{color:#013c5b}.cc-button--link{background-color:initial!important;border-color:#0000!important;border-radius:0!important;color:inherit!important;padding:0!important;text-decoration:underline!important;width:inherit!important}.cc-button--link:focus{outline:3px solid #08c}.cc-button--link:hover{background-color:initial!important;box-shadow:none;text-decoration:none}.cc-button--text{border-radius:0;padding:0}.cc-button--details{padding-right:24px!important;position:relative;width:auto}.cc-button--details:after,.cc-button--details:before{background-color:currentColor;content:"";position:absolute;transition:transform .25s ease-out}.cc-button--details:before{height:10px;margin-top:-5px;right:12px;top:50%;width:2px}.cc-button--details:after{height:2px;margin-top:-1px;right:8px;top:50%;width:10px}.cc-button--details.cc-active:before{transform:rotate(90deg);transform-origin:center}.cc-button--details.cc-active:after{display:none}.cc-details{border-bottom:1px solid #0000001a;padding:12px 0;position:relative;width:100%}.cc-details__summary{align-items:center;display:flex;font-size:14px;font-weight:700;list-style-type:none}.cc-details__summary:focus,.cc-details__summary:focus-visible{outline:3px solid #08c!important;will-change:auto}.cc-details__title{align-items:baseline;display:flex}.cc-details__title h2,.cc-details__title h3,.cc-details__title h4,.cc-details__title h5{line-height:1.4;margin:0!important;padding:0!important}.cc-details__title svg{flex-shrink:0;height:auto;margin-right:8px;position:relative;top:-1px;transition:all .2s;width:auto}.cc-details[open] .cc-details__title svg{top:2px;transform:rotate(90deg)}.cc-details__switch{margin-left:auto}.cc-details__section{padding:16px 0 0 18px}.cc-details__section p{margin:0}.cc-details__section p:not(:last-child){margin-bottom:12px}details summary::-webkit-details-marker{display:none}.cc-radio{align-items:center;display:flex;position:relative}.cc-radio *{cursor:pointer}.cc-radio__input{height:22px;left:0;position:absolute;top:0;width:22px}.cc-radio__input:focus{outline:none}.cc-radio__label{color:inherit;font-size:14px;font-weight:700;line-height:23px;margin:0;padding-left:28px}.cc-radio__label:after,.cc-radio__label:before{background-color:#fff;content:"";display:block;position:absolute;transition:transform .25s ease-out}.cc-radio__label:before{border:1px solid #777;border-radius:50%;height:22px;left:0;top:0;width:22px}.cc-radio__label:after{border:7px solid #025e8d;border-radius:50%;height:0;left:4px;opacity:0;top:4px;width:0}.cc-radio__input:focus+.cc-radio__label:before{box-shadow:0 0 0 2px #08c;outline:none}.cc-radio__label--hidden{display:none}.cc-radio__input:checked+.cc-radio__label:after{opacity:1}.cc-radio__input:disabled{cursor:default}.cc-radio__input:disabled+.cc-radio__label{cursor:default;opacity:.5}.cc-switch *{cursor:pointer}.cc-switch{align-items:center;display:flex;line-height:1}.cc-switch__label{cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;-webkit-user-select:none;user-select:none}.cc-switch__input[type=checkbox]{-webkit-appearance:none;appearance:none;background-color:#fff!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='rgba(0, 0, 0, 0.25)'/%3E%3C/svg%3E")!important;background-position:0!important;background-repeat:no-repeat!important;border:1px solid #777!important;border-radius:16px!important;height:18px!important;margin:0 8px 0 0!important;transition:background-position .2s ease-in-out!important;width:32px!important}.cc-switch__input[type=checkbox]:focus{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%2380b3cc'/%3E%3C/svg%3E")!important}.cc-switch__input[type=checkbox]:checked{background-color:#025e8d!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%23fff'/%3E%3C/svg%3E")!important;background-position:100%!important}.cc-switch__input[type=checkbox]:checked~.cc-switch__label>.cc-switch__text-off,.cc-switch__input[type=checkbox]:not(:checked)~.cc-switch__label>.cc-switch__text-on{display:none}.cc-switch__input[type=checkbox]:hover{cursor:pointer;outline:none}.cc-switch__input[type=checkbox]:focus-visible{cursor:pointer;outline:3px solid #fc0}.cc-switch__input[type=checkbox]:checked:before{background-color:none!important;color:#0000!important}html[dir=rtl] .cc-switch__input[type=checkbox]{margin-left:8px!important;margin-right:0!important}.cc-list>ul,ul.cc-list{list-style-type:disc;margin-left:0;padding-left:0}.cc-list>ul>li,ul.cc-list>li{margin-left:1em}.cc-list>ul>li:not(:last-child),ul.cc-list>li:not(:last-child){margin-bottom:4px}.cc-list__title{margin-bottom:8px!important}.cc-list--inline>ul,ul.cc-list--inline{display:flex;flex-wrap:wrap;list-style-type:none}.cc-list--inline>ul>li,ul.cc-list--inline>li{margin:0 .5em .5em 0}.cc-overlay{background-color:#0000004d;z-index:99998}.cc-overlay,.cc-preferences{bottom:0;left:0;position:fixed;right:0;top:0}.cc-preferences{background-color:#050a14f2!important;border:0;box-sizing:border-box;color:#111;font-family:sans-serif!important;line-height:1.4;margin:auto;max-height:100vh;overflow:auto;padding:0;z-index:100000}.cc-preferences:focus{outline:none}.cc-preferences *,.cc-preferences :after,.cc-preferences :before{box-sizing:inherit!important}.cc-preferences h2,.cc-preferences h3,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-preferences .cc-h2,.cc-preferences h2{font-size:18px}.cc-preferences .cc-h3,.cc-preferences h3{font-size:16px}.cc-preferences .cc-h4,.cc-preferences .cc-h5,.cc-preferences .cc-h6,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-size:14px}.cc-preferences a{color:#025e8d;text-decoration:underline}.cc-preferences a:hover{color:inherit;text-decoration:none}.cc-preferences h3{background:none;color:#111;padding:0;text-transform:none}dialog.cc-preferences{background-color:initial}dialog.cc-preferences::backdrop{background-color:#000000e6}.cc-preferences__dialog{display:flex;flex-direction:column;margin:auto;max-height:100vh;max-width:860px;padding:12px;position:relative}.cc-preferences__dialog>:last-child{border-bottom-left-radius:10px;border-bottom-right-radius:10px}@media (min-width:980px){.cc-preferences__dialog{padding:16px}}.cc-preferences__close{background:#0000!important;border:1px solid #ececec;border-radius:50%;color:#111!important;cursor:pointer;font-family:Times New Roman,serif;font-size:40px;height:40px;left:auto;line-height:1;margin-top:-20px;padding:0!important;position:absolute;right:20px;top:50%;width:40px}.cc-preferences__close:focus{outline:3px solid #08c}.cc-preferences__close-label{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__header{background:#fff!important;border-bottom:1px solid #0000001a;border-top-left-radius:10px;border-top-right-radius:10px;padding:16px;position:relative;text-align:center}.cc-preferences__title{background:none!important;color:#111!important;font-family:sans-serif!important;font-size:18px!important;font-weight:700!important;margin:0!important;padding:0 16px 0 0!important}@media (min-width:480px){.cc-preferences__title{padding-right:0!important}}@media (min-width:980px){.cc-preferences__title{font-size:22px!important}}.cc-preferences__body{background:#fff!important;flex:1 1 auto;min-height:200px;overflow-x:hidden;overflow-y:auto;padding:16px}.cc-preferences__footer{background:#fff!important;border-top:1px solid #d0d0d0;box-shadow:0 0 5px 0 #0003;margin-bottom:0;padding:12px;position:relative}@media (min-width:480px){.cc-preferences__footer{align-items:stretch;display:flex}}.cc-preferences__footer>.cc-button{display:block;width:100%}@media (min-width:480px){.cc-preferences__footer>.cc-button{flex:1 1 auto}}@media (min-width:980px){.cc-preferences__footer>.cc-button{flex-basis:auto}}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:first-child){margin-left:12px}}.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:8px}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:0}}.cc-preferences__categories{list-style:none;margin:0;padding:0}.cc-preferences__category:not(:last-child){border-bottom:1px solid #0000001a;margin-bottom:12px;padding-bottom:12px}.cc-preferences__category-description{font-size:14px;margin:0 0 8px;padding:0}.cc-preferences__category-footer{align-items:center;display:flex;justify-content:space-between}.cc-preferences__status{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__controls{display:flex;margin:0}.cc-preferences__controls>:not(:last-child){margin-right:12px}.cc-preferences__always-on{font-size:14px;font-weight:700;padding-left:26px;position:relative}.cc-preferences__always-on:before{background:#0000!important;border:solid;border-top-color:#0000;border-width:0 0 4px 4px;content:"";display:block;height:10px;left:0;position:absolute;top:2px;transform:rotate(-45deg);width:18px}.cc-preferences__details{background-color:#eff6fb!important;border-radius:10px;font-size:14px;margin:12px 0 0;padding:12px}.cc-preferences__cookie-list,.cc-preferences__provider-list{list-style:none;margin:0;padding:0}.cc-preferences__provider-list{columns:170px}.cc-preferences__cookie-title{font-size:1em;font-style:normal;font-weight:400;margin:0}.cc-preferences__cookie-description{font-size:1em;margin:0 0 8px}.cc-preferences__cookie-domain,.cc-preferences__cookie-lifespan{border-left:1px solid #999;color:#666;margin-left:8px;padding-left:4px}body.cc-has-preferences-open{overflow:hidden;position:relative}.cc-table{border-collapse:collapse;width:100%}.cc-table tbody tr{border-top:1px solid #0000001a}.cc-table td,.cc-table th{font-size:14px;padding:4px 8px;vertical-align:top}.cc-table+.cc-table{margin-top:8px}.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:70%}@media (min-width:680px){.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:80%}}.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{padding-left:8px;width:40%}@media (min-width:680px){.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{width:30%}}.cc-tabs,.cc-tabs>ul{background-color:#faf9f6;display:flex;list-style:none;margin:0;padding:0}.cc-tabs__panel{border-top:1px solid #ececec;margin-top:-1px}.cc-tabs__button{background-color:initial;border:0;border-left:1px solid #0000;border-right:1px solid #0000;color:#111;font-size:16px;font-weight:700;margin:0;padding:15px 16px 12px;position:relative}.cc-tabs__button:before{background-color:initial;content:"";display:block;height:3px;left:0;position:absolute;top:0;width:100%}.cc-tabs__button.cc-active{background-color:#fff;border-color:#025e8d #ececec #ececec;color:#111}.cc-tabs__button.cc-active:before{background-color:#025e8d}.cc-tabs__content{background-color:#fff;display:none;font-size:14px;padding:16px}.cc-tabs__section:not(:first-child){padding-top:24px}.cc-tcf{font-size:14px;margin-top:16px}.cc-tcf__list{list-style:none;margin:0;padding:0}.cc-vendor-count{color:#666;font-size:16px;margin:0 0 8px}.cc-hide{display:none!important}.cc-show{display:block!important}.cc-external-link{background-color:#ececec;background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTcgMGExIDEgMCAxIDEgMCAySDIuNmMtLjM3MSAwLS42LjIwOS0uNi41djE1YzAgLjI5MS4yMjkuNS42LjVoMTQuOGMuMzcxIDAgLjYtLjIwOS42LS41VjEzYTEgMSAwIDAgMSAyIDB2NC41YzAgMS40MzgtMS4xNjIgMi41LTIuNiAyLjVIMi42QzEuMTYyIDIwIDAgMTguOTM4IDAgMTcuNXYtMTVDMCAxLjA2MiAxLjE2MiAwIDIuNiAwem02IDBoNmwuMDc1LjAwMy4xMjYuMDE3LjExMS4wMy4xMTEuMDQ0LjA5OC4wNTIuMDk2LjA2Ny4wOS4wOGExIDEgMCAwIDEgLjA5Ny4xMTJsLjA3MS4xMS4wNTQuMTE0LjAzNS4xMDUuMDMuMTQ4TDIwIDF2NmExIDEgMCAwIDEtMiAwVjMuNDE0bC02LjY5MyA2LjY5M2ExIDEgMCAwIDEtMS40MTQtMS40MTRMMTYuNTg0IDJIMTNhMSAxIDAgMCAxLS45OTMtLjg4M0wxMiAxYTEgMSAwIDAgMSAxLTEiLz48L3N2Zz4=");background-position:right 8px center;background-repeat:no-repeat;background-size:10px auto;border-radius:3px;box-shadow:0 1px 0 #0003;color:#111!important;display:inline-block;font-size:12px;padding:4px 26px 4px 8px;text-decoration:none!important}.cc-external-link:hover{text-decoration:underline!important}.cc-m-0{margin:0!important}.cc-grey{color:#666}</style><script async="" type="text/javascript" src="https://injections.readcube.com/nature/inject.fa744f3f.js"></script></head>
<body class="article-page"><dialog class="cc-banner" data-cc-banner="" data-nosnippet="" aria-labelledby="cc-banner-label" open="">
		<div class="cc-banner__content" autofocus="" tabindex="-1">
			<div class="cc-banner__container">
				<div class="cc-banner__header">
					<h2 class="cc-banner__title" id="cc-banner-label">Your privacy, your choice</h2>
				</div>
				<div class="cc-banner__body">
					<div class="cc-banner__policy">
						<p>We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.</p><p>By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.</p><p>See our <a href="https://www.nature.com/info/privacy" data-cc-action="privacy">privacy policy</a> for more information on the use of your personal data.</p><p><button type="button" data-cc-action="preferences" class="cc-button cc-button--link cc-button--text">Manage preferences</button> for further information and to change your choices.</p>
					</div>
					
				</div>
				<div class="cc-banner__footer">
					
			<button data-cc-action="accept" class="cc-button cc-button--secondary cc-button--contrast cc-banner__button cc-banner__button-accept">Accept all cookies</button>
			
		
				</div>
			</div>
		</div>
	</dialog>

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90" style="display: none !important;">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-test="top-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41591-020-0844-1;doi=10.1038/s41591-020-0844-1;subjmeta=114,1647,2017,2217,631,67;kwrd=Biological+techniques,Cancer,Computational+biology+and+bioinformatics,Gene+expression+analysis,Genomic+analysis">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=303790749&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-020-0844-1%26doi%3D10.1038/s41591-020-0844-1%26subjmeta%3D114,1647,2017,2217,631,67%26kwrd%3DBiological+techniques,Cancer,Computational+biology+and+bioinformatics,Gene+expression+analysis,Genomic+analysis">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=303790749&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-020-0844-1%26doi%3D10.1038/s41591-020-0844-1%26subjmeta%3D114,1647,2017,2217,631,67%26kwrd%3DBiological+techniques,Cancer,Computational+biology+and+bioinformatics,Gene+expression+analysis,Genomic+analysis"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header="" data-track-component="nature-150-split-header" style="border-color:#e40428">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nm" data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" height="32" alt="Nature Medicine">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
                            </a><div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected="">All journals</option>
                                    <option value="nm">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41591-020-0844-1"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header" hidden="">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-articles" data-track="click" data-track-action="research articles" data-track-label="link" data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-and-analysis" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link" data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/podcast" data-track="click" data-track-action="podcasts" data-track-label="link" data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/current-issue" data-track="click" data-track-action="current issue" data-track-label="link" data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/collections" data-track="click" data-track-action="collections" data-track-label="link" data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://www.facebook.com/Nature-Medicine-193691346949/" data-track="click" data-track-action="facebook" data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://twitter.com/naturemedicine" data-track="click" data-track-action="twitter" data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=5" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/aims" data-track="click" data-track-action="aims &amp; scope" data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-information" data-track="click" data-track-action="journal information" data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-impact" data-track="click" data-track-action="journal metrics" data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editors" data-track="click" data-track-action="about the editors" data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-cross-journal-editorial-team" data-track="click" data-track-action="research cross-journal editorial team" data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-cross-journal-editorial-team" data-track="click" data-track-action="reviews cross-journal editorial team" data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/statistics-advisory-panel" data-track="click" data-track-action="statistical advisory panel" data-track-label="link">
                                    Statistical Advisory Panel
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/our-publishing-models" data-track="click" data-track-action="our publishing models" data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-values-statement" data-track="click" data-track-action="editorial values statement" data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-policies" data-track="click" data-track-action="editorial policies" data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/content" data-track="click" data-track-action="content types" data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="javascript:;" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                        </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/submission-guidelines" data-track="click" data-track-action="submission guidelines" data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-context="header publish with us dropdown menu" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/open-access-funding" data-test="funding-eligibility-link" data-track="click_explore_funding" data-track-context="header publish with us" data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link" href="https://mts-nmed.nature.com/cgi-bin/main.plex" data-track="click_submit_manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-action="submit manuscript" data-track-label="link (publish with us dropdown menu)" data-track-external="" data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link" href="https://journal-alerts.springernature.com/subscribe?journal_id=41591" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-label="link (desktop site header)" data-track-external="">
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature medicine"><span itemprop="name">nature medicine</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm/articles?type=resource" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:resources"><span itemprop="name">resources</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" id="js-enable-context-bar" data-test="context-bar" data-context-bar="" aria-hidden="true">
                    <div class="c-context-bar__container u-container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors
                        </div>
                        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-020-0844-1.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-020-0844-1.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Resource</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2020-05-11">11 May 2020</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors</h1>
                    <ul class="c-article-author-list c-article-author-list--long js-no-scroll" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michal-Slyper-Aff1" data-author-popup="auth-Michal-Slyper-Aff1" data-author-search="Slyper, Michal" data-track-context="researcher popup with no profile" data-track-index="1_50">Michal Slyper</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Caroline_B__M_-Porter-Aff1" data-author-popup="auth-Caroline_B__M_-Porter-Aff1" data-author-search="Porter, Caroline B. M." data-track-context="researcher popup with no profile" data-track-index="2_50">Caroline B. M. Porter</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Orr-Ashenberg-Aff1" data-author-popup="auth-Orr-Ashenberg-Aff1" data-author-search="Ashenberg, Orr" data-track-context="researcher popup with no profile" data-track-index="3_50">Orr Ashenberg</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Julia-Waldman-Aff1" data-author-popup="auth-Julia-Waldman-Aff1" data-author-search="Waldman, Julia" data-track-context="researcher popup with no profile" data-track-index="4_50">Julia Waldman</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Eugene-Drokhlyansky-Aff1" data-author-popup="auth-Eugene-Drokhlyansky-Aff1" data-author-search="Drokhlyansky, Eugene" data-track-context="researcher popup with no profile" data-track-index="5_50">Eugene Drokhlyansky</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Isaac-Wakiro-Aff2-Aff3-Aff4" data-author-popup="auth-Isaac-Wakiro-Aff2-Aff3-Aff4" data-author-search="Wakiro, Isaac" data-track-context="researcher popup with no profile" data-track-index="6_50">Isaac Wakiro</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Christopher-Smillie-Aff1" data-author-popup="auth-Christopher-Smillie-Aff1" data-author-search="Smillie, Christopher" data-track-context="researcher popup with no profile" data-track-index="7_50">Christopher Smillie</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Gabriela-Smith_Rosario-Aff1" data-author-popup="auth-Gabriela-Smith_Rosario-Aff1" data-author-search="Smith-Rosario, Gabriela" data-track-context="researcher popup with no profile" data-track-index="8_50">Gabriela Smith-Rosario</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jingyi-Wu-Aff2-Aff3-Aff4" data-author-popup="auth-Jingyi-Wu-Aff2-Aff3-Aff4" data-author-search="Wu, Jingyi" data-track-context="researcher popup with no profile" data-track-index="9_50">Jingyi Wu</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Danielle-Dionne-Aff1" data-author-popup="auth-Danielle-Dionne-Aff1" data-author-search="Dionne, Danielle" data-track-context="researcher popup with no profile" data-track-index="10_50">Danielle Dionne</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-S_bastien-Vigneau-Aff2-Aff3-Aff4" data-author-popup="auth-S_bastien-Vigneau-Aff2-Aff3-Aff4" data-author-search="Vigneau, Sébastien" data-track-context="researcher popup with no profile" data-track-index="11_50">Sébastien Vigneau</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Judit-Jan__Valbuena-Aff1" data-author-popup="auth-Judit-Jan__Valbuena-Aff1" data-author-search="Jané-Valbuena, Judit" data-track-context="researcher popup with no profile" data-track-index="12_50">Judit Jané-Valbuena</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Timothy_L_-Tickle-Aff1" data-author-popup="auth-Timothy_L_-Tickle-Aff1" data-author-search="Tickle, Timothy L." data-track-context="researcher popup with no profile" data-track-index="13_50">Timothy L. Tickle</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sara-Napolitano-Aff2-Aff3-Aff4" data-author-popup="auth-Sara-Napolitano-Aff2-Aff3-Aff4" data-author-search="Napolitano, Sara" data-track-context="researcher popup with no profile" data-track-index="14_50">Sara Napolitano</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Mei_Ju-Su-Aff2-Aff3-Aff4" data-author-popup="auth-Mei_Ju-Su-Aff2-Aff3-Aff4" data-author-search="Su, Mei-Ju" data-track-context="researcher popup with no profile" data-track-index="15_50">Mei-Ju Su</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Anand_G_-Patel-Aff5-Aff6" data-author-popup="auth-Anand_G_-Patel-Aff5-Aff6" data-author-search="Patel, Anand G." data-track-context="researcher popup with no profile" data-track-index="16_50">Anand G. Patel</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-1438-8275"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-1438-8275</a></span><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a>,<a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Asa-Karlstrom-Aff5" data-author-popup="auth-Asa-Karlstrom-Aff5" data-author-search="Karlstrom, Asa" data-track-context="researcher popup with no profile" data-track-index="17_50">Asa Karlstrom</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-3847-7309"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-3847-7309</a></span><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Simon-Gritsch-Aff7-Aff8" data-author-popup="auth-Simon-Gritsch-Aff7-Aff8" data-author-search="Gritsch, Simon" data-track-context="researcher popup with no profile" data-track-index="18_50">Simon Gritsch</a><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Masashi-Nomura-Aff7-Aff8" data-author-popup="auth-Masashi-Nomura-Aff7-Aff8" data-author-search="Nomura, Masashi" data-track-context="researcher popup with no profile" data-track-index="19_50">Masashi Nomura</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6095-7849"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6095-7849</a></span><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Avinash-Waghray-Aff9" data-author-popup="auth-Avinash-Waghray-Aff9" data-author-search="Waghray, Avinash" data-track-context="researcher popup with no profile" data-track-index="20_50">Avinash Waghray</a><sup class="u-js-hide"><a href="#Aff9" tabindex="-1">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Satyen_H_-Gohil-Aff2-Aff3" data-author-popup="auth-Satyen_H_-Gohil-Aff2-Aff3" data-author-search="Gohil, Satyen H." data-track-context="researcher popup with no profile" data-track-index="21_50">Satyen H. Gohil</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Alexander_M_-Tsankov-Aff1-Aff25" data-author-popup="auth-Alexander_M_-Tsankov-Aff1-Aff25" data-author-search="Tsankov, Alexander M." data-track-context="researcher popup with no profile" data-track-index="22_50">Alexander M. Tsankov</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#nAff25" tabindex="-1">nAff25</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Livnat-Jerby_Arnon-Aff1" data-author-popup="auth-Livnat-Jerby_Arnon-Aff1" data-author-search="Jerby-Arnon, Livnat" data-track-context="researcher popup with no profile" data-track-index="23_50">Livnat Jerby-Arnon</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ofir-Cohen-Aff2-Aff3-Aff4" data-author-popup="auth-Ofir-Cohen-Aff2-Aff3-Aff4" data-author-search="Cohen, Ofir" data-track-context="researcher popup with no profile" data-track-index="24_50">Ofir Cohen</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Johanna-Klughammer-Aff1" data-author-popup="auth-Johanna-Klughammer-Aff1" data-author-search="Klughammer, Johanna" data-track-context="researcher popup with no profile" data-track-index="25_50">Johanna Klughammer</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Yanay-Rosen-Aff1" data-author-popup="auth-Yanay-Rosen-Aff1" data-author-search="Rosen, Yanay" data-track-context="researcher popup with no profile" data-track-index="26_50">Yanay Rosen</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Joshua-Gould-Aff1" data-author-popup="auth-Joshua-Gould-Aff1" data-author-search="Gould, Joshua" data-track-context="researcher popup with no profile" data-track-index="27_50">Joshua Gould</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-9581-5461"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9581-5461</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Lan-Nguyen-Aff1" data-author-popup="auth-Lan-Nguyen-Aff1" data-author-search="Nguyen, Lan" data-track-context="researcher popup with no profile" data-track-index="28_50">Lan Nguyen</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Matan-Hofree-Aff1" data-author-popup="auth-Matan-Hofree-Aff1" data-author-search="Hofree, Matan" data-track-context="researcher popup with no profile" data-track-index="29_50">Matan Hofree</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Peter_J_-Tramontozzi-Aff10" data-author-popup="auth-Peter_J_-Tramontozzi-Aff10" data-author-search="Tramontozzi, Peter J." data-track-context="researcher popup with no profile" data-track-index="30_50">Peter J. Tramontozzi</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-5540-0111"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-5540-0111</a></span><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Bo-Li-Aff2-Aff11" data-author-popup="auth-Bo-Li-Aff2-Aff11" data-author-search="Li, Bo" data-track-context="researcher popup with no profile" data-track-index="31_50">Bo Li</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff11" tabindex="-1">11</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Catherine_J_-Wu-Aff2-Aff3-Aff12-Aff13" data-author-popup="auth-Catherine_J_-Wu-Aff2-Aff3-Aff12-Aff13" data-author-search="Wu, Catherine J." data-track-context="researcher popup with no profile" data-track-index="32_50">Catherine J. Wu</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3348-5054"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3348-5054</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff12" tabindex="-1">12</a>,<a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Benjamin-Izar-Aff2-Aff3-Aff4-Aff14-Aff15-Aff16-Aff17-Aff26" data-author-popup="auth-Benjamin-Izar-Aff2-Aff3-Aff4-Aff14-Aff15-Aff16-Aff17-Aff26" data-author-search="Izar, Benjamin" data-track-context="researcher popup with no profile" data-track-index="33_50">Benjamin Izar</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff14" tabindex="-1">14</a>,<a href="#Aff15" tabindex="-1">15</a>,<a href="#Aff16" tabindex="-1">16</a>,<a href="#Aff17" tabindex="-1">17</a></sup><sup class="u-js-hide">&nbsp;<a href="#nAff26" tabindex="-1">nAff26</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Rizwan-Haq-Aff2-Aff3" data-author-popup="auth-Rizwan-Haq-Aff2-Aff3" data-author-search="Haq, Rizwan" data-track-context="researcher popup with no profile" data-track-index="34_50">Rizwan Haq</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-1290-9276"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-1290-9276</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-F__Stephen-Hodi-Aff3-Aff17" data-author-popup="auth-F__Stephen-Hodi-Aff3-Aff17" data-author-search="Hodi, F. Stephen" data-track-context="researcher popup with no profile" data-track-index="35_50">F. Stephen Hodi</a><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff17" tabindex="-1">17</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Charles_H_-Yoon-Aff3-Aff18" data-author-popup="auth-Charles_H_-Yoon-Aff3-Aff18" data-author-search="Yoon, Charles H." data-track-context="researcher popup with no profile" data-track-index="36_50">Charles H. Yoon</a><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff18" tabindex="-1">18</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Aaron_N_-Hata-Aff8-Aff19" data-author-popup="auth-Aaron_N_-Hata-Aff8-Aff19" data-author-search="Hata, Aaron N." data-track-context="researcher popup with no profile" data-track-index="37_50">Aaron N. Hata</a><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff19" tabindex="-1">19</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Suzanne_J_-Baker-Aff5" data-author-popup="auth-Suzanne_J_-Baker-Aff5" data-author-search="Baker, Suzanne J." data-track-context="researcher popup with no profile" data-track-index="38_50">Suzanne J. Baker</a><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Mario_L_-Suv_-Aff2-Aff7-Aff8" data-author-popup="auth-Mario_L_-Suv_-Aff2-Aff7-Aff8" data-author-search="Suvà, Mario L." data-track-context="researcher popup with no profile" data-track-index="39_50">Mario L. Suvà</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-9898-5351"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-9898-5351</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Raphael-Bueno-Aff10" data-author-popup="auth-Raphael-Bueno-Aff10" data-author-search="Bueno, Raphael" data-track-context="researcher popup with no profile" data-track-index="40_50">Raphael Bueno</a><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Elizabeth_H_-Stover-Aff2-Aff3" data-author-popup="auth-Elizabeth_H_-Stover-Aff2-Aff3" data-author-search="Stover, Elizabeth H." data-track-context="researcher popup with no profile" data-track-index="41_50">Elizabeth H. Stover</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-6578-3216"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-6578-3216</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael_R_-Clay-Aff20" data-author-popup="auth-Michael_R_-Clay-Aff20" data-author-search="Clay, Michael R." data-track-context="researcher popup with no profile" data-track-index="42_50">Michael R. Clay</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-1659-8927"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1659-8927</a></span><sup class="u-js-hide"><a href="#Aff20" tabindex="-1">20</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael_A_-Dyer-Aff5-Aff21" data-author-popup="auth-Michael_A_-Dyer-Aff5-Aff21" data-author-search="Dyer, Michael A." data-track-context="researcher popup with no profile" data-track-index="43_50">Michael A. Dyer</a><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a>,<a href="#Aff21" tabindex="-1">21</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Natalie_B_-Collins-Aff2-Aff22-Aff23" data-author-popup="auth-Natalie_B_-Collins-Aff2-Aff22-Aff23" data-author-search="Collins, Natalie B." data-track-context="researcher popup with no profile" data-track-index="44_50">Natalie B. Collins</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff22" tabindex="-1">22</a>,<a href="#Aff23" tabindex="-1">23</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ursula_A_-Matulonis-Aff3" data-author-popup="auth-Ursula_A_-Matulonis-Aff3" data-author-search="Matulonis, Ursula A." data-track-context="researcher popup with no profile" data-track-index="45_50">Ursula A. Matulonis</a><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Nikhil-Wagle-Aff2-Aff3-Aff4-Aff12-Aff13" data-author-popup="auth-Nikhil-Wagle-Aff2-Aff3-Aff4-Aff12-Aff13" data-author-search="Wagle, Nikhil" data-track-context="researcher popup with no profile" data-track-index="46_50">Nikhil Wagle</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-3332-9438"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-3332-9438</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff12" tabindex="-1">12</a>,<a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Bruce_E_-Johnson-Aff3-Aff4" data-author-popup="auth-Bruce_E_-Johnson-Aff3-Aff4" data-author-search="Johnson, Bruce E." data-track-context="researcher popup with no profile" data-track-index="47_50">Bruce E. Johnson</a><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Asaf-Rotem-Aff2-Aff3-Aff4" data-author-popup="auth-Asaf-Rotem-Aff2-Aff3-Aff4" data-author-search="Rotem, Asaf" data-track-context="researcher popup with no profile" data-track-index="48_50">Asaf Rotem</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Orit-Rozenblatt_Rosen-Aff1" data-author-popup="auth-Orit-Rozenblatt_Rosen-Aff1" data-author-search="Rozenblatt-Rosen, Orit" data-corresp-id="c1" data-track-context="researcher popup with no profile" data-track-index="49_50">Orit Rozenblatt-Rosen<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6313-3570"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6313-3570</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 50 authors for this article" title="Show all 50 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Aviv-Regev-Aff1-Aff21-Aff24" data-author-popup="auth-Aviv-Regev-Aff1-Aff21-Aff24" data-author-search="Regev, Aviv" data-corresp-id="c2" data-track-context="researcher popup with no profile" data-track-index="50_50">Aviv Regev<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-3293-3158"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-3293-3158</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff21" tabindex="-1">21</a>,<a href="#Aff24" tabindex="-1">24</a></sup>&nbsp;</li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nm" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Medicine</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;26</b>,&nbsp;<span class="u-visually-hidden">pages </span>792–802 (<span data-test="article-publication-year">2020</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item" data-test="access-count">
                        <p class="c-article-metrics-bar__count">162k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item" data-test="citation-count">
                        <p class="c-article-metrics-bar__count">412 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item" data-test="altmetric-score">
                            <p class="c-article-metrics-bar__count">120 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41591-020-0844-1/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    

                
    
    
        <div class="u-mb-8 c-status-message c-status-message--boxed c-status-message--info">
            
                <span class="c-status-message__icon">
                    <svg class="u-icon" width="18" height="18" aria-hidden="true" focusable="false">
                        <use xlink:href="#icon-eds-i-info-filled-medium"></use>
                    </svg>
                </span>
            
            
                
                    <p class="u-mt-0">An <a href="https://doi.org/10.1038/s41591-020-0976-3" class="relation-link" data-track="click" data-track-action="view linked article" data-track-label="link">Author Correction</a> to this article was published on 25 June 2020</p>
                
            
        </div>
    

    
        <div class="u-mb-8 c-status-message c-status-message--boxed c-status-message--info">
            
                <span class="c-status-message__icon">
                    <svg class="u-icon" width="18" height="18" aria-hidden="true" focusable="false">
                        <use xlink:href="#icon-eds-i-info-filled-medium"></use>
                    </svg>
                </span>
            
            <p class="u-mt-0">This article has been <a href="#change-history">updated</a></p>
        </div>
    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Single-cell genomics is essential to chart tumor ecosystems. Although single-cell RNA-Seq (scRNA-Seq) profiles RNA from cells dissociated from fresh tumors, single-nucleus RNA-Seq (snRNA-Seq) is needed to profile frozen or hard-to-dissociate tumors. Each requires customization to different tissue and tumor types, posing a barrier to adoption. Here, we have developed a systematic toolbox for profiling fresh and frozen clinical tumor samples using scRNA-Seq and snRNA-Seq, respectively. We analyzed 216,490 cells and nuclei from 40 samples across 23 specimens spanning eight tumor types of varying tissue and sample characteristics. We evaluated protocols by cell and nucleus quality, recovery rate and cellular composition. scRNA-Seq and snRNA-Seq from matched samples recovered the same cell types, but at different proportions. Our work provides guidance for studies in a broad range of tumors, including criteria for testing and selecting methods from the toolbox for other tumors, thus paving the way for charting tumor atlases.</p></div></div></section>

            
                
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs42003-024-07043-2/MediaObjects/42003_2024_7043_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s42003-024-07043-2?fromPaywallRec=false" data-track="select_recommendations_1" data-track-context="inline recommendations" data-track-action="click recommendations inline - 1" data-track-label="10.1038/s42003-024-07043-2">snPATHO-seq, a versatile FFPE single-nucleus RNA sequencing method to unlock pathology archives
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">16 October 2024</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41587-020-00748-9/MediaObjects/41587_2020_748_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41587-020-00748-9?fromPaywallRec=false" data-track="select_recommendations_2" data-track-context="inline recommendations" data-track-action="click recommendations inline - 2" data-track-label="10.1038/s41587-020-00748-9">A multicenter study benchmarking single-cell RNA sequencing technologies using reference samples
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">21 December 2020</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41596-024-00991-3/MediaObjects/41596_2024_991_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41596-024-00991-3?fromPaywallRec=false" data-track="select_recommendations_3" data-track-context="inline recommendations" data-track-action="click recommendations inline - 3" data-track-label="10.1038/s41596-024-00991-3">Scanorama: integrating large and diverse single-cell transcriptomic datasets
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">06 June 2024</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1749521187,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Tumors encompass complex cellular ecosystems of malignant and non-malignant cells, whose diversity and interactions affect cancer progression and drug response and resistance. Recent advances in single-cell genomics, especially single-cell RNA-Seq (scRNA-Seq), have transformed our ability to analyze tumors, revealing cell types, states, genetic diversity and interactions in the complex tumor ecosystem<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Cieslik, M. &amp; Chinnaiyan, A. M. Cancer transcriptome profiling at the juncture of clinical translation. Nat. Rev. Genet. 19, 93–109 (2018)." href="#ref-CR1" id="ref-link-section-d74065049e1462">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Filbin, M. G. et al. Developmental and oncogenic programs in H3K27M gliomas dissected by single-cell RNA-seq. Science 360, 331–335 (2018)." href="#ref-CR2" id="ref-link-section-d74065049e1462_1">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Jerby-Arnon, L. et al. A cancer cell program promotes T cell exclusion and resistance to checkpoint blockade. Cell 175, 984–997.e924 (2018)." href="#ref-CR3" id="ref-link-section-d74065049e1462_2">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Puram, S. V. et al. Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. Cell 171, 1611–1624.e1624 (2017)." href="#ref-CR4" id="ref-link-section-d74065049e1462_3">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Tirosh, I. et al. Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. Nature 539, 309–313 (2016)." href="#ref-CR5" id="ref-link-section-d74065049e1462_4">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Venteicher, A. S. et al. Decoupling genetics, lineages and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. Science 355, eaai8478 (2017)." href="/articles/s41591-020-0844-1#ref-CR6" id="ref-link-section-d74065049e1465">6</a></sup>. Single-cell analysis of tumors is rapidly expanding, including the launch of a Human Tumor Atlas Network (HTAPP) as part of the Cancer Moonshot<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Moonshot Cancer Initiative, Generation of Human Tumor Atlases National Cancer Institute (accessed 20 April 2020); 
                  https://ccr.cancer.gov/research/cancer-moonshot
                  
                " href="/articles/s41591-020-0844-1#ref-CR7" id="ref-link-section-d74065049e1469">7</a></sup>.</p><p>Successful scRNA-Seq of clinical tumor specimens poses several challenges. First, it requires quick dissociation tailored to the tumor type, and involves enzymatic digestion, which can lead to loss of sensitive cells or changes in gene expression. Moreover, obtaining fresh tissue is time-sensitive and requires tight coordination between tissue acquisition and processing teams, posing a challenge in clinical settings. Conversely, single-nucleus RNA-Seq (snRNA-Seq) allows profiling of single nuclei isolated from frozen tissues, decoupling tissue acquisition from immediate sample processing. snRNA-Seq can also handle samples that cannot be successfully dissociated even when fresh, due to size or cell fragility<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Habib, N. et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. Nat. Methods 14, 955–958 (2017)." href="/articles/s41591-020-0844-1#ref-CR8" id="ref-link-section-d74065049e1476">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Habib, N. et al. Div-Seq: single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons. Science 353, 925–928 (2016)." href="/articles/s41591-020-0844-1#ref-CR9" id="ref-link-section-d74065049e1479">9</a></sup>, as well as multiplexed analysis of longitudinal samples from the same individual<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Gaublomme, J. T. et al. Nuclei multiplexing with barcoded antibodies for single-nucleus genomics. Nat. Commun. 10, 2907 (2019)." href="/articles/s41591-020-0844-1#ref-CR10" id="ref-link-section-d74065049e1483">10</a></sup>. However, nuclei have lower amounts of mRNA compared to cells and are more challenging to enrich or deplete for specific cell types of interest. Both scRNA-Seq and snRNA-Seq pose experimental challenges when applied to different tumor types, due to the distinct cellular composition and extracellular matrix (ECM) in different tumors, and thus each assay requires dedicated customizations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Drokhlyansky, E. et al. The enteric nervous system of the human and mouse colon at a single-cell resolution. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/746743v4
                  
                 (2019)." href="/articles/s41591-020-0844-1#ref-CR11" id="ref-link-section-d74065049e1487">11</a></sup>.</p><p>To address these challenges, we developed a systematic toolbox for fresh and frozen tumor processing using scRNA-Seq and snRNA-Seq, respectively (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1a</a>). The toolbox contains our experimental workflow and methods, computational pipelines and evaluation metrics. To generalize across tumor and sample types, we tested eight tumor types with different tissue characteristics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1b</a>), including comparisons of matched fresh and frozen preparations from the same tumor specimen. Our work provides direct recommended protocols for multiple tumor types, decision trees that allow researchers to choose the most suitable protocol for their research goals, and guidelines on how to customize protocols for new tumor and specimen types.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Study and toolbox overview."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Study and toolbox overview.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="849"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, sc/snRNA-Seq workflow, experimental and computational pipelines, and protocol selection criteria. <b>b</b>, Tumor types and samples processed in the study. Tested and selected protocols for fresh (white circles, cells), frozen (blue circles, nuclei) and cryopreserved (underlined circles, cells (white) and nuclei (blue)) are indicated. O-PDX, orthotopic patient-derived xenograft; EZ, EZPrep; ST, salts and Tris; QC, quality control; DE, differentially expressed; InferCNV, Infer Copy Number Variation, a method for detecting copy number abberations; C4, collagenase 4 and DNase I; PDEC, pronase, dispase, elastase, collagenases A and 4 and DNase I; LE, Liberase TM, elastase and DNase I; EZ, EZPrep; NST, Nonidet P40 with salts and Tris; CST, CHAPS with salts and Tris; TST, Tween with salts and Tris; LD, Liberase TM and DNase I; BTD, brain tumor dissociation; MHTD, Miltenyi Biotec human tumor dissociation.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">A systematic study to develop an sc/snRNA-Seq toolbox</h3><p>To develop a toolbox of customized protocols for sc/snRNA-Seq of tumors, we studied eight tumor types with different tissue characteristics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1</a>). The tumor types span the following characteristics: different cells of origin (for example, epithelial, neuronal), solid and non-solid, patient ages and transitions (for example, primary, metastatic). We tested varying tissue and sample characteristics including resection, biopsy, ascites and orthotopic patient-derived xenograft (O-PDX). We included samples from non-small cell lung carcinoma (NSCLC), metastatic breast cancer (MBC), ovarian cancer, neuroblastoma, glioblastoma (GBM), pediatric high-grade glioma, chronic lymphocytic leukemia (CLL), pediatric sarcoma and melanoma (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1b</a>). In total, we analyzed 216,490 cells and nuclei across 23 tumors, from 22 patients spanning 40 sample preparations. We provide a comprehensive analysis summary for each sample tested in a dedicated website (<a href="https://tumor-toolbox.broadinstitute.org">https://tumor-toolbox.broadinstitute.org</a>).</p><h3 class="c-article__sub-heading" id="Sec4">Experimental and computational QCs assess quality and composition</h3><p>We evaluated and compared protocols based on (1) cell/nucleus quality; (2) number of recovered versus expected cells/nuclei; (3) cellular composition (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1a</a>). For ‘cell/nucleus quality’, we considered both experimental and computational metrics. Experimentally, we measured cell viability (for scRNA-Seq), the extent of doublets or aggregates in the cell/nucleus suspension and cDNA quality recovered after whole transcriptome amplification. Computationally, we evaluated the percent of reads mapping to the transcriptome, genome and intergenic regions, the number of cells/nuclei exceeding a minimal number of genes and unique transcripts (reflected by unique molecular identifiers, UMIs), the number of reads, transcripts (UMIs) and genes detected per cell/nucleus and the percent of UMIs from mitochondrial genes (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>). To compare protocols, when there was a notable difference in sequencing saturation or total reads across samples, we also downsampled reads to equal numbers across samples and then re-estimated and compared their QC metrics. For ‘number of recovered versus expected cells/nuclei’, we evaluated the proportion of droplets scored as likely empty (that is, containing only ambient RNA rather than the RNA from an encapsulated cell<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Lun, A. T. L. et al. EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data. Genome Biol. 20, 63 (2019)." href="/articles/s41591-020-0844-1#ref-CR12" id="ref-link-section-d74065049e1564">12</a></sup>), and the proportion of doublets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Wolock, S. L., Lopez, R. &amp; Klein, A. M. Scrublet: computational identification of cell doublets in single-cell transcriptomic data. Cell Syst. 8, 281–291.e289 (2019)." href="/articles/s41591-020-0844-1#ref-CR13" id="ref-link-section-d74065049e1568">13</a></sup> (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>). Because the algorithm for scoring empty droplets was developed for cells, we did not use it to evaluate snRNA-Seq. Finally, for ‘cellular composition’, we considered the diversity of cell types captured, the proportion of cells/nuclei recovered from each subset and the copy number aberration (CNA) pattern classes that are recovered in malignant cells (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>). We considered it a virtue if a protocol recovers a larger diversity of cell types, because this facilitates comprehensive studies. However, capture of diverse cell types may not always be a researcher’s desired goal, nor would it always be the most accurate representation of a tumor’s composition (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec12">Discussion</a>). We annotated the malignant cells based on the presence of CNAs (when detectable) and the cell type signature they most closely resembled (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>). We conducted initial data analysis using Cumulus, a cloud-based data analysis framework<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Li, B. et al. Cumulus: a cloud-based data analysis framework for large-scale single-cell and single-nucleus RNA-seq. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/823682v1
                  
                 (2019)." href="/articles/s41591-020-0844-1#ref-CR14" id="ref-link-section-d74065049e1585">14</a></sup> (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a> and Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1a</a>), and developed a dedicated pipeline for additional quality control, tumor sample characterization and protocol comparison.</p><h3 class="c-article__sub-heading" id="Sec5">Customization of workflows and dissociation protocols for scRNA-Seq of fresh tumors</h3><p>We customized successful protocols for specimen acquisition and dissociation for scRNA-Seq across five types of fresh tumor (NSCLC, ovarian cancer, MBC, neuroblastoma, GBM and cryopreserved CLL (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1b</a>)). We constructed workflows that minimize the time interval between removal of the sample from the patient in a clinical setting and its dissociation into cells, to maximize cell viability and preservation of RNA profiles. We determined dissociation conditions for each tumor type and constructed specific steps as a decision tree to adjust for differences between clinical samples (for example, size, presence of red blood cells (RBCs); Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2a</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>). To choose the best performing dissociation method, when possible, we apportioned large tumor specimens into smaller pieces (~0.5–2 cm), dissociating each piece following a different protocol. When specimen size was limiting (for example, biopsies), optimization spanned multiple samples. We subjected the samples that yielded highly viable single cell suspensions to droplet-based scRNA-Seq (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>), to allow sampling of larger number of cells for calculation of QC metrics. We tested protocols several times to confirm similar performance trends.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Fresh tumor processing and protocol selection for scRNA-Seq."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Fresh tumor processing and protocol selection for scRNA-Seq.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="992"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Flow chart recommended for collection and processing of fresh tumor samples. <b>b</b>–<b>f</b>, Comparison of three dissociation protocols applied to one NSCLC sample. <b>b</b>, Protocol performance varies across cell types. Top and middle: distribution (median and first and third quartiles) of the number of reads per cell, the number of UMIs per cell, the number of genes per cell and fraction of UMIs per cell mapping to mitochondrial genes (Fr. mito. genes) (<i>y</i> axes) in each protocol (<i>x</i> axis) across the entire dataset. Bottom: distribution (median and first and third quartiles) of the number of genes per cell (<i>y</i> axis) only in epithelial cells (left) or in B cells (right). <b>c</b>, The protocols detect similar numbers of doublets. Uniform manifold approximation and projection (UMAP) embedding of single cell profiles (dots) for each protocol, colored by assignment as single cell (gray) or doublet (red). Horizontal bars (bottom): fraction of single (gray) and doublet (red) cells. <b>d</b>, The protocols vary in the number of empty drops. UMAP embedding of single cell profiles (dots) for each protocol, colored by assignment as cell (gray) or empty drop (red). Horizontal bars (bottom): fraction of assigned cells (gray) and empty drops (red). <b>e</b>, The protocols vary in the diversity of cell types captured. UMAP embedding of single cell profiles (dots) from all three protocols, colored by assigned cell subset signature (left) or by protocol (right). Bottom: proportion of cells in each subset in each of the three protocols; <i>k</i>, number of cells passing QC. <b>f</b>, Inferred CNA profiles. Chromosomal amplification (red) and deletion (blue) are inferred in each chromosomal position (columns) across the single cells (rows) using the PDEC protocol. Top: reference cells not expected to contain CNAs in this tumor. Bottom: cells tested for CNAs relative to the reference cells. Color bar: assigned cell type signature for each cell. <b>g</b>, Successful depletion of CD45<sup>+</sup> cells. The proportion of cells in each subset without and with CD45<sup>+</sup> depletion in NSCLCs (top) and ovarian ascites (bottom) samples is shown; <i>k</i>, number of cells passing QC. <i>n</i> = 1 sample per protocol. The numbers of cells (<i>k</i>) are indicated in <b>e</b> and <b>g</b>. Numbers of epithelial cells from NSCLC-C4, PDEC and LE are <i>k</i> = 1,284, 641 and 260, respectively, and the number of B cells is <i>k</i> = 100, 121 and 78, respectively. ACK, ammonium-chloride-potassium.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>For dissociation, we selected enzymatic mixtures for processing fresh tissues based on the specific characteristics of each tumor type, such as cell type composition and ECM components, literature review and experience from processing similar human or mouse tissues. For example, to break down collagen fibers in breast cancer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Al-Hajj, M., Wicha, M. S., Benito-Hernandez, A., Morrison, S. J. &amp; Clarke, M. F. Prospective identification of tumorigenic breast cancer cells. Proc. Natl Acad. Sci. USA 100, 3983–3988 (2003)." href="/articles/s41591-020-0844-1#ref-CR15" id="ref-link-section-d74065049e1704">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="McDivitt, R. W., Stone, K. R. &amp; Meyer, J. S. A method for dissociation of viable human breast cancer cells that produces flow cytometric kinetic information similar to that obtained by thymidine labeling. Cancer Res. 44, 2628–2633 (1984)." href="/articles/s41591-020-0844-1#ref-CR16" id="ref-link-section-d74065049e1707">16</a></sup> we used Liberase TM (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>), whereas to break down ECM in GBM<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Neftel, C. et al. An integrative model of cellular states, plasticity and genetics for glioblastoma. Cell 178, 835–849 (2019)." href="/articles/s41591-020-0844-1#ref-CR17" id="ref-link-section-d74065049e1714">17</a></sup> we used papain (cysteine protease). We also included DNase I to digest DNA released from dead cells to decrease viscosity in all dissociation mixtures. In the following, we recommend those methods that broke down the ECM and cell-to-cell adhesions sufficiently, while minimizing processing time, maintaining high viability and supporting cell type diversity in the sample.</p><h3 class="c-article__sub-heading" id="Sec6">Cell-type-specific and cell-composition QCs are important in protocol evaluation</h3><p>As an example of the optimization process, consider the in-depth analysis of an NSCLC resection sample (sample NSCLC14, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2b–f</a> and Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig7">1</a>–<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig9">3</a>). We used three processing protocols: (1) collagenase 4 (NSCLC-C4), (2) a mixture of pronase, dispase, elastase and collagenases A and 4 (PDEC) and (3) Liberase TM and elastase (LE), each in combination with DNase I (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>). For other tumor types, we show the results of selected protocols out of those tested (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1b</a>, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2g</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig3">3a–e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig10">4a,b</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="scRNA-Seq protocol comparison across tumor types."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: scRNA-Seq protocol comparison across tumor types.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="739"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>–<b>d</b>, QC metrics. The number of UMIs per cell (<b>a</b>), number of genes per cell (<b>b</b>), fraction of UMIs per cell mapping to mitochondrial genes (<b>c</b>) and fraction of empty drops (<b>d</b>) (<i>x</i> axes) for each sample (<i>y</i> axis). Median and first and third quartiles are shown in <b>a</b>–<b>c</b>. <b>e</b>, Cell type composition. Proportion of cells assigned to each cell type signature (color) for each sample. O-PDX, orthotopic patient-derived xenograft. Tested protocols for processing each tumor type are indicated. <b>f</b>, Inferred CNA profiles for matched pre- and post-treatment neuroblastoma samples. Chromosomal amplification (red) and deletion (blue) inferred in each chromosomal position (columns) across the single cells (rows) from pre-treatment biopsy HTAPP-312-SMP-901 (left) and post-treatment resection HTAPP-312-SMP-902 (right). Top: reference cells not expected to contain CNAs in this tumor. Bottom: cells tested for CNAs relative to the reference cells. Color bars: assigned cell type signature for each cell. <i>n</i> = 1 sample per protocol. The numbers of cells (<i>k</i>) are indicated in <b>e</b>. Prot., protocol; NB, neuroblastoma; OPC, oligodendrocyte progenitor like cell; NPC, neural progenitor like cell; NK, natural killer; k., kit.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Protocols often performed similarly on standard QC measures (for example, number of cells recovered), but differed markedly in recovered cellular diversity or in the fraction of droplets predicted to contain only ambient RNA (‘empty drops’)—two evaluation criteria that we prioritized. For example, in the NSCLC14 resection sample, all methods yielded a similar number of cells with high-quality expression profiles (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2b,e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig7">1a–c</a>), doublets (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2c</a>) and CNA patterns in malignant cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2f</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig7">1d</a>). However, only the PDEC and LE protocols recovered fibroblasts and endothelial cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig7">1c</a>), and NSCLC-C4 had a 100-fold higher fraction of droplets called as ‘empty’ (7% versus 0.08% and 0.04% in PDEC and LE, respectively; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig7">1a</a>). The drops designated ‘empty’ in NSCLC-C4 clustered within macrophages (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig7">1c</a>), the most prevalent cell type, suggesting that these cell barcodes either had lower sequencing saturation or that the sample itself had higher ambient RNA content. Although we estimated similarly low levels of ambient RNA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Young, M. D. &amp; Behjati, S. SoupX removes ambient RNA contamination from droplet based single cell RNA sequencing data. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/303727v1
                  
                 (2018)." href="/articles/s41591-020-0844-1#ref-CR18" id="ref-link-section-d74065049e1855">18</a></sup> across the three protocols (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig7">1e</a>), NSCLC-C4 indeed had lower overall sequencing saturation (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig7">1a</a>).</p><p>Comparing QC metrics across protocols can be challenging due to differences in cell type recovery and in sequencing depth between preparations, which we controlled for in the NSCLC14 sample by also evaluating QC metrics within each cell type and downsampling by total reads across protocols (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig8">2</a>). For example, when we consider all cells in the NSCLC14 samples, NSCLC-C4 had a significantly higher number of detected genes (<i>P</i> = 1.3 × 10<sup>−90</sup> versus PDEC; 1.4 × 10<sup>−62</sup> versus LE, two-sided Mann–Whitney U test), but within B cells, PDEC had a significantly higher number of detected genes (<i>P</i> = 2 × 10<sup>−15</sup> versus NSCLC-C4; 2 × 10<sup>−10</sup> versus LE), whereas within epithelial cells, LE had the highest number (<i>P</i> = 5 × 10<sup>−6</sup> versus NSCLC-C4; 2 × 10<sup>−4</sup> versus PDEC) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2b</a>). Because the number of detected genes (and other metrics) varies between cell types, and cell type composition varies between the protocols (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2e</a>), it is important to assess cell-type-specific QCs when selecting a protocol. Moreover, the lower sequencing saturation does not directly reflect the performance of the NSCLC-C4 protocol, and downsampling by total reads did not qualitatively change any of our protocol evaluation metrics (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig9">3</a>). Considering all of these features, we selected the PDEC protocol for processing NSCLC tumor samples, as it balances cell type diversity and QCs per cell type.</p><h3 class="c-article__sub-heading" id="Sec7">Fast depletion of immune cells for enrichment of malignant and stromal cells</h3><p>Because in some tumor specimens the proportion of malignant cells is relatively low and that of immune cells is particularly high, we considered strategies to deplete CD45<sup>+</sup> immune cells as a way to both enrich for epithelial cells without specific markers and to maintain any stromal cells. We chose to use MACS MicroBeads with anti-CD45 antibodies rather than sorting by flow cytometry (FACS), because samples are not always available at the designated time for which sorters are booked, and FACS requires longer sample processing, which may introduce additional cell stress, as we have found for epithelial cells (data not shown).</p><p>We optimized a CD45<sup>+</sup> cell depletion strategy by testing different commercial kits and assessing the impact of one versus two rounds of depletion (data not shown). For example, we profiled an NSCLC tumor sample (NSCLC17) by scRNA-Seq before and after depletion, finding an increase from 26% to 82% epithelial cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig10">4a</a>) and virtually no immune cells profiled post depletion. Similarly, in an ovarian ascites sample (HTAPP-727), we recovered 32% epithelial cells by scRNA-Seq post depletion (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2g</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig10">4b</a>), compared to &lt;1% CD45<sup>−</sup>EpCAM<sup>+</sup> cells typically found in ovarian ascites by FACS (data not shown). Consistently, FACS shows that CD45<sup>+</sup> cell depletion of another ascites sample increased the overall proportion of non-immune (CD45<sup>−</sup>) cells from 0.75% to 29.4% and increased the proportion of EpCAM<sup>+</sup> cells from 0.17% to 4.9% (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig10">4c</a>).</p><h3 class="c-article__sub-heading" id="Sec8">Successful scRNA-Seq of biopsies and post-treatment samples from diverse tumors</h3><p>We successfully applied the scRNA-Seq toolbox to much smaller core biopsy clinical samples from different anatomical sites. For example, in MBC, we applied the LD (Liberase TM and DNase I) protocol to a resection (HTAPP-254) and a biopsy (HTAPP-735) from lymph node metastases from two patients, yielding similarly successful QCs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig3">3a–d</a>). The resection and biopsy of the two patients had different cellular compositions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig3">3e</a>): the biopsy had a higher proportion of epithelial, endothelial and fibroblast cells and a lower proportion of T cells compared to the resection. We similarly successfully profiled biopsies of MBC liver metastases (HTAPP-285 and HTAPP-963) with the same protocol (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig3">3a–d</a>), recovering some hepatocytes in addition to a similar range of cell types as was recovered in the lymph node biopsy (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig3">3e</a>). Thus, this protocol can be used across breast cancer metastases from different anatomical metastatic sites.</p><p>The scRNA-Seq toolbox also performs well on samples obtained post-treatment, which can pose challenges as a result of cell death and changes in cell type composition with treatment. For example, both a pre-treatment diagnostic biopsy (HTAPP-312-pre) and a post-treatment resection (HTAPP-312-post) from the same neuroblastoma patient profiled with the NB-C4 protocol yielded high QCs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig3">3a–d</a>) and similar CNA patterns in malignant cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig3">3f</a>). More cells, but of fewer cell types, were recovered in the pre-treatment biopsy (4,369 cells: neuroendocrine, T cells and macrophages) than the post-treatment resection (786 cells: neuroendocrine, T cells, macrophages, as well as endothelial cells and fibroblasts) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig3">3e</a>), consistent with observed post-treatment fibrosis. We tested an additional dissociation protocol (papain) in a neuroblastoma O-PDX sample (O-PDX1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Stewart, E. et al. Orthotopic patient-derived xenografts of paediatric solid tumours. Nature 549, 96–100 (2017)." href="/articles/s41591-020-0844-1#ref-CR19" id="ref-link-section-d74065049e1980">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Stewart, E. et al. Development and characterization of a human orthotopic neuroblastoma xenograft. Dev. Biol. 407, 344–355 (2015)." href="/articles/s41591-020-0844-1#ref-CR20" id="ref-link-section-d74065049e1983">20</a></sup>, which is not expected to include non-malignant human cells and indeed resulted in high-quality malignant cell profiles (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig3">3a–e</a>). The experimental QCs for the papain protocol were superior to those we had observed in other samples with NB-C4, a trend we corroborated in additional neuroblastoma samples (data not shown). Thus, we ultimately selected the papain protocol for neuroblastoma tumors.</p><p>In addition to such NSCLC, ovarian cancer ascite, MBC and neuroblastoma samples, we established effective scRNA-Seq protocols for GBM (GBM125), CLL (CLL1) and ovarian cancer tumors (HTAPP-624) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig3">3a–e</a>). In particular, in CLL, we successfully recovered the expected cell types from a cryopreserved sample, containing viable cells. This reflects the increased resilience of immune cells to freezing compared to other cell types, also observed in other settings<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Hermansen, J. U., Tjonnfjord, G. E., Munthe, L. A., Tasken, K. &amp; Skanland, S. S. Cryopreservation of primary B cells minimally influences their signaling responses. Sci. Rep. 8, 17651 (2018)." href="/articles/s41591-020-0844-1#ref-CR21" id="ref-link-section-d74065049e1996">21</a></sup>, and the lack of a dissociation step in CLL scRNA-Seq (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>). Cryopreservation, however, can increase the proportion of damaged cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Guillaumet-Adkins, A. et al. Single-cell transcriptome conservation in cryopreserved cells and tissues. Genome Biol. 18, 45 (2017)." href="/articles/s41591-020-0844-1#ref-CR22" id="ref-link-section-d74065049e2003">22</a></sup> and may not successfully recover all the malignant and other non-malignant cells in the tumor.</p><h3 class="c-article__sub-heading" id="Sec9">Four nucleus isolation protocols assessed for snRNA-Seq of frozen tumors</h3><p>For frozen specimens from solid tumors, we optimized snRNA-Seq, assessing different methods for nucleus isolation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4a</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>) across seven tumor types: neuroblastoma, MBC, ovarian cancer, pediatric sarcoma, melanoma, pediatric high-grade glioma and CLL (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1b</a>). We initially apportioned larger samples or used multiple biopsies to compare four isolation methods: EZPrep<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Habib, N. et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. Nat. Methods 14, 955–958 (2017)." href="/articles/s41591-020-0844-1#ref-CR8" id="ref-link-section-d74065049e2024">8</a></sup>, Nonidet P40 with salts and Tris (NST) (modified from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Gao, R. et al. Nanogrid single-nucleus RNA sequencing reveals phenotypic diversity in breast cancer. Nat. Commun. 8, 228 (2017)." href="/articles/s41591-020-0844-1#ref-CR23" id="ref-link-section-d74065049e2028">23</a></sup>), CHAPS, with salts and Tris (CST)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Drokhlyansky, E. et al. The enteric nervous system of the human and mouse colon at a single-cell resolution. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/746743v4
                  
                 (2019)." href="/articles/s41591-020-0844-1#ref-CR11" id="ref-link-section-d74065049e2033">11</a></sup>, and Tween with salts and Tris (TST)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Drokhlyansky, E. et al. The enteric nervous system of the human and mouse colon at a single-cell resolution. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/746743v4
                  
                 (2019)." href="/articles/s41591-020-0844-1#ref-CR11" id="ref-link-section-d74065049e2037">11</a></sup>. The methods differ primarily in the mechanical force (for example, chopping or douncing), buffer (EZ versus ST) and/or detergent composition (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>). Because in early tests EZPrep routinely underperformed CST, NST and TST (data not shown), we only included EZPrep in initial comparisons (below). We used single nucleus suspensions from all tested protocols as input for droplet-based snRNA-Seq (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>), thus sampling sufficiently large numbers of cells to evaluate cell diversity and QCs.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Frozen tumor processing and protocol selection for snRNA-Seq."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Frozen tumor processing and protocol selection for snRNA-Seq.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="752"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Flow chart for collection and processing of frozen tumor samples. <b>b</b>–<b>e</b>, Comparison of four nucleus isolation protocols in one neuroblastoma sample. <b>b</b>, Variation in protocol performance: distributions (median and first and third quartiles) of the number of UMIs per nucleus, the number of genes per nucleus and fraction of UMIs per nucleus mapping to mitochondrial genes (<i>y</i> axes) in each protocol (<i>x</i> axis) across all nuclei in the dataset. <b>c</b>, The protocols detect similar numbers of doublets. UMAP embedding of single nucleus profiles (dots) for each protocol is colored by assignment as nucleus (gray) or doublet (red). Horizontal bars (bottom): fraction of single (gray) and doublet (red) nuclei. <b>d</b>, The protocols vary in the diversity of cell types captured. UMAP embedding of single nucleus profiles (dots) from all four protocols is colored by assigned cell subset signature (left) or protocol (right). Bottom: proportion of cells from each subset in each of the protocols. <i>k</i>, number of nuclei passing QC. <b>e</b>, Inferred CNA profiles. Chromosomal amplification (red) and deletion (blue) inferred in each chromosomal position (columns) across the single nuclei (rows) from the TST protocol are shown. Top: reference nuclei not expected to contain CNAs in this tumor. Bottom: nuclei tested for CNAs relative to the reference nuclei. Color bar: assigned cell type signature for each nucleus. <i>n</i> = 1 sample per protocol. The numbers of nuclei (<i>k</i>) are indicated in <b>d</b>. OCT, optimal cutting temperature compound.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>To evaluate protocols, we used the post hoc computational criteria above (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1a</a>), except we excluded the estimation of empty drops because it was only developed and tested on single-cell RNA-Seq data. We further customized Cumulus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Li, B. et al. Cumulus: a cloud-based data analysis framework for large-scale single-cell and single-nucleus RNA-seq. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/823682v1
                  
                 (2019)." href="/articles/s41591-020-0844-1#ref-CR14" id="ref-link-section-d74065049e2113">14</a></sup> for snRNA-Seq data, mapping reads to both exons and introns, and adapted the QC thresholds for transcript (UMI) and gene counts to reflect the lower expected mRNA content in nuclei (see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>). Experimentally, we added in-process light microscopy QCs to ensure complete nucleus isolation and to estimate doublets, aggregates and debris (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4a</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>). As with scRNA-Seq, we tested protocols several times to confirm similar performance trends.</p><h3 class="c-article__sub-heading" id="Sec10">TST protocol typically recovers the highest diversity of cell types</h3><p>Overall, three nucleus isolation methods—TST, CST and NST—had comparable performances based on the assessed nucleus quality (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4b–d</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5</a> and Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig11">5</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig12">6</a>), with TST typically yielding the greatest cell type diversity and number of nuclei per cell type, together with the highest expression of mitochondrial genes, and NST typically having the fewest genes per nucleus and lowest diversity of cell types. For example, in neuroblastoma, testing each of the four protocols on a single resection sample (HTAPP-244) yielded a similar number of high-quality nuclei (7,896, 6,157, 7,531 and 7,415 for EZ, CST, NST and TST, respectively) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4b–d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig11">5a–c</a>), nucleus doublets (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4c</a>), cell types—with malignant neuroendocrine cells being the most prevalent (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig11">5c</a>) and malignant cells with similarly detectable CNAs (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig11">5d</a>; CNAs are less prominent, as expected for this pediatric, low-risk sample). Nuclei prepared with the EZ protocol had lower numbers of UMIs and genes detected (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4b</a>) compared to the three ST protocols. TST recovered more endothelial cells, fibroblasts, neural crest cells and T cells than the other protocols (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4d</a>). TST also yielded a higher expression of mitochondrial genes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4b</a>), in this and all other tumors tested (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5c</a>), because the nuclear membrane, endoplasmic reticulum and ribosomes remain attached to the nucleus when using this method<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Drokhlyansky, E. et al. The enteric nervous system of the human and mouse colon at a single-cell resolution. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/746743v4
                  
                 (2019)." href="/articles/s41591-020-0844-1#ref-CR11" id="ref-link-section-d74065049e2181">11</a></sup>. The same trends were preserved in cell-type-specific QCs (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig12">6</a>) and after downsampling by the total number of sequencing reads (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig13">7</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="snRNA-Seq protocol comparison across tumor types."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: snRNA-Seq protocol comparison across tumor types.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="524"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>–<b>c</b>, QC metrics: distributions (median and first and third quartiles) of the number of UMIs per nucleus (<b>a</b>), the number of genes per nucleus (<b>b</b>) and the fraction of UMIs per nucleus mapping to mitochondrial genes (<b>c</b>) (<i>x</i> axes) for each sample (<i>y</i> axis). <b>d</b>, Cell type composition, showing the proportion of nuclei assigned to each cell type signature (color) for each sample. <i>n</i> = 1 sample per protocol. The numbers of nuclei (<i>k</i>) are indicated in <b>d</b>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>The CST, NST and TST nucleus isolation methods had similar performance characteristics when tested with MBC, ovarian cancer and pediatric sarcoma samples, with TST providing the most diversity in cell types, especially in non-malignant cells. In MBC, we compared CST and NST in one metastatic brain resection (HTAPP-394) and CST and TST in another metastatic brain resection (HTAPP-589) and in a metastatic liver biopsy (HTAPP-963) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5</a>). In all cases, QC statistics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5a–c</a>) and CNA patterns (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig14">8a–f</a>) were similar between protocols, and nuclei from epithelial cells were the most prevalent (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5d</a>). CST and NST captured a very similar distribution of cell types, while TST captured more non-malignant cells, including T cells (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5d</a>), and a higher fraction of mitochondrial reads (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5c</a>). In ovarian cancer, CST, NST and TST recovered similar CNA patterns from the same sample (HTAPP-316; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig14">8g–i</a>), but TST captured the greatest cell type diversity (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5d</a>), whereas NST recovered fewer nuclei, genes per nucleus and UMIs per nucleus (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5a–c</a>), and had a lower cell type diversity (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5d</a>), despite having greater overall sequencing depth (73% sequencing saturation versus 57% in CST and 50% in TST). In a rhabdomyosarcoma sample (HTAPP-951), CST and TST captured the same cell types at similar proportions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5d</a>) and showed similar CNA patterns (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig14">8j,k</a>).</p><p>Overall, we chose the TST protocol for most tumor types and CST for tumors from neuronal tissues, such as pediatric high-grade glioma. With the protocols we selected (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1b</a>, right column), we profiled additional neuroblastoma tumors (HTAPP-656, O-PDX1) as well as Ewing sarcoma (HTAPP-975), melanoma (MEL112, MEL128), pediatric high-grade glioma (HTAPP-443) and CLL (CLL1) tumor samples—spanning biopsies, resections and treated samples (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1b</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig5">5</a>). We also tested a pediatric rhabdomyosarcoma sample (HTAPP-951) by two different chemistries for droplet-based snRNA-Seq (V2 versus V3 from 10x Genomics; see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>), obtaining, overall, similar results in terms of cell types detected, but an improved number of recovered versus expected nuclei and higher complexity per nucleus in V3 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig15">9</a>).</p><h3 class="c-article__sub-heading" id="Sec11">Different cell composition recovered by scRNA-Seq and snRNA-Seq</h3><p>We compared scRNA-Seq and snRNA-Seq by testing matching samples from the same specimen each in neuroblastoma (HTAPP-656, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig6">6a–g</a>), MBC (HTAPP-963, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig6">6h–n</a>), CLL (CLL1, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig16">10a–g</a>) and O-PDX (O-PDX1, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig16">10h–n</a>). The two approaches typically recovered similar cell types, but sometimes at varying proportions. In both neuroblastoma and MBC, immune cells were much more prevalent in scRNA-Seq, and parenchymal (especially malignant) cells were much more prevalent in snRNA-Seq (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig6">6a–d,h–k</a>). In all tested tumor types, cells and nuclei readily aligned following batch correction by canonical correlation analysis (CCA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies and species. Nat. Biotechnol. 36, 411–420 (2018)." href="/articles/s41591-020-0844-1#ref-CR24" id="ref-link-section-d74065049e2328">24</a></sup>, see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>), grouping by cell type (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig6">6e–g,l–n</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig16">10e–g,l–n</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="scRNA-Seq and snRNA-Seq recover comparable cells in different proportions."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6: scRNA-Seq and snRNA-Seq recover comparable cells in different proportions.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig6_HTML.png?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="1518"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>–<b>g</b>, Neuroblastoma. <b>a</b>–<b>c</b>, UMAP embedding of scRNA-Seq and snRNA-Seq profiles of the same neuroblastoma sample combined by CCA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies and species. Nat. Biotechnol. 36, 411–420 (2018)." href="/articles/s41591-020-0844-1#ref-CR24" id="ref-link-section-d74065049e2364">24</a></sup> (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0844-1#Sec13">Methods</a>) showing profiles (dots) from both (<b>a</b>), scRNA-Seq (<b>b</b>) and snRNA-Seq (<b>c</b>), colored by the assigned cell type signatures. <b>d</b>, Proportion of cells from each subset in the two protocols. <i>k</i>, number of cells or nuclei passing QC. <b>e</b>,<b>f</b>, Same UMAP embedding as in <b>a</b>, colored by cells or nuclei (<b>e</b>) or by unsupervised clustering (<b>f</b>). <b>g</b>, Fraction of cells and nuclei in each cluster from <b>f</b>. <i>n</i> = 1 sample per protocol. The numbers of cells and nuclei (<i>k</i>) are indicated in <b>d</b>. <b>h</b>–<b>n</b>, MBC. As in <b>a</b>–<b>g</b> for an MBC sample. <i>n</i> = 1 sample per protocol. The numbers of cells and nuclei are indicated in <b>k</b>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0844-1/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Finally, we leveraged the matched data to examine the extent to which expression patterns in cells from dissociated fresh tissue indicate specific stress and compared these to those in nuclei from snap-frozen tissue. To this end, we scored each cell or nucleus in the matched data from recently published ‘dissociation signatures’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="van den Brink, S. C. et al. Single-cell sequencing reveals dissociation-induced gene expression in tissue subpopulations. Nat. Methods 14, 935–936 (2017)." href="/articles/s41591-020-0844-1#ref-CR25" id="ref-link-section-d74065049e2449">25</a></sup> (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig16">10o–r</a>). In general, dissociation signatures were detectable in a larger proportion of cells than of nuclei, especially in solid tumors (neuroblastoma and MBC), and scored significantly higher in cells (<i>P</i> &lt; 1 × 10<sup>−100</sup>, two-sided Mann–Whitney U test). When present in nuclei, however, these nuclei are embedded in the same regions of the phenotypic space as high scoring cells. Notably, in both cells and nuclei, the signature is more prominent in immune cells (for example, T, NK and macrophages) and stroma cells (for example, fibroblasts and endothelial). Although this may be a signature of damage from dissociation in some parenchymal cells, it is also probably a signature of immune activation and the immediate early response more generally. As a result, the interpretation of a ‘dissociation signature’ derived in a distinct setting must be done with extreme care, accounting for its cell (rather than nucleus) data source and its relation to immediate early gene expression (a native response in situ as well).</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec12-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec12">Discussion</h2><div class="c-article-section__content" id="Sec12-content"><p>Single-cell genomics of clinical tumor specimens obtained as part of routine disease management or through research biopsies should help guide new discoveries and better deployment of therapies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Baslan, T. &amp; Hicks, J. Unravelling biology and shifting paradigms in cancer with single-cell sequencing. Nat. Rev. Cancer 17, 557–569 (2017)." href="/articles/s41591-020-0844-1#ref-CR26" id="ref-link-section-d74065049e2470">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Suva, M. L. &amp; Tirosh, I. Single-cell RNA sequencing in cancer: lessons learned and emerging challenges. Mol. Cell 75, 7–12 (2019)." href="/articles/s41591-020-0844-1#ref-CR27" id="ref-link-section-d74065049e2473">27</a></sup>, as initial studies have shown in the context of gliomas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Venteicher, A. S. et al. Decoupling genetics, lineages and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. Science 355, eaai8478 (2017)." href="/articles/s41591-020-0844-1#ref-CR6" id="ref-link-section-d74065049e2477">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Neftel, C. et al. An integrative model of cellular states, plasticity and genetics for glioblastoma. Cell 178, 835–849 (2019)." href="/articles/s41591-020-0844-1#ref-CR17" id="ref-link-section-d74065049e2480">17</a></sup>, melanoma<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Jerby-Arnon, L. et al. A cancer cell program promotes T cell exclusion and resistance to checkpoint blockade. Cell 175, 984–997.e924 (2018)." href="/articles/s41591-020-0844-1#ref-CR3" id="ref-link-section-d74065049e2484">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Rambow, F. et al. Toward minimal residual disease-directed therapy in melanoma. Cell 174, 843–855.e819 (2018)." href="/articles/s41591-020-0844-1#ref-CR28" id="ref-link-section-d74065049e2487">28</a></sup>, head and neck squamous cell carcinoma<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Puram, S. V. et al. Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. Cell 171, 1611–1624.e1624 (2017)." href="/articles/s41591-020-0844-1#ref-CR4" id="ref-link-section-d74065049e2491">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Qi, Z., Barrett, T., Parikh, A. S., Tirosh, I. &amp; Puram, S. V. Single-cell sequencing and its applications in head and neck cancer. Oral Oncol. 99, 104441 (2019)." href="/articles/s41591-020-0844-1#ref-CR29" id="ref-link-section-d74065049e2494">29</a></sup> and other malignancies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="van Galen, P. et al. Single-cell RNA-Seq reveals AML hierarchies relevant to disease progression and immunity. Cell 176, 1265–1281.e1224 (2019)." href="#ref-CR30" id="ref-link-section-d74065049e2498">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Chevrier, S. et al. An immune atlas of clear cell renal cell carcinoma. Cell 169, 736–749.e718 (2017)." href="#ref-CR31" id="ref-link-section-d74065049e2498_1">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Savas, P. et al. Single-cell profiling of breast cancer T cells reveals a tissue-resident memory subset associated with improved prognosis. Nat. Med. 24, 986–993 (2018)." href="#ref-CR32" id="ref-link-section-d74065049e2498_2">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Karaayvaz, M. et al. Unravelling subclonal heterogeneity and aggressive disease states in TNBC through single-cell RNA-seq. Nat. Commun. 9, 3588 (2018)." href="#ref-CR33" id="ref-link-section-d74065049e2498_3">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Lambrechts, D. et al. Phenotype molding of stromal cells in the lung tumor microenvironment. Nat. Med. 24, 1277–1289 (2018)." href="/articles/s41591-020-0844-1#ref-CR34" id="ref-link-section-d74065049e2501">34</a></sup>. However, this requires adaptation of laboratory protocols, initially developed in research settings and requiring very rapid handling of fresh tissue, into the context of clinical sample acquisition and processing. Indeed, these initial studies were applied to small numbers of samples, often from resections, or focused on readily isolated immune cells. Analyzing the larger numbers of samples that would be required in the context of a clinical trial or longitudinal research, involving multiple sites, calls for streamlined and robust protocols. Moreover, systematically characterizing the diverse cells in solid tumors requires robust recovery of cells, many of which, including malignant epithelial cells, are highly sensitive. These challenges are further compounded by the diversity of tissues in which tumors and metastases reside.</p><p>Here, we take on these challenges by developing a systematic toolbox for protocols for single-cell and single-nucleus RNA-Seq, with detailed workflows and protocols from sample acquisition to library preparation for processing fresh and frozen clinical tumor samples across eight tumor types, as well as guidelines for testing and selecting protocols for processing future clinical tumor samples. We provide computational pipelines for extensive QCs at <a href="https://github.com/klarman-cell-observatory/HTAPP-Pipelines">https://github.com/klarman-cell-observatory/HTAPP-Pipelines</a>, and all laboratory protocols are provided in detailed form in the open access platform protocols.io.</p><p>When selecting a protocol for fresh tissue dissociation, we suggest testing two to three dissociation methods, chosen based on tumor type and tissue composition, and processing according to the fresh sample decision tree (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2a</a>). We selected the best performing protocol by assessing both experimental and computational QC metrics, and, if desired, added a depletion step. When selecting a protocol for frozen tissues, we suggest testing the NST, TST and CST protocols, and processing according to our snRNA-Seq workflow (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig4">4a</a>). Although TST is often favorable due to its ability to capture the most diverse set of cells, in some tumors we recommend CST or NST (for example, CST for pediatric high-grade glioma; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig1">1b</a>). CST also yields fewer mitochondrial reads, reducing sequencing cost. For both sc- and snRNA-Seq, it is important to evaluate the selected protocol on multiple samples to ensure consistent performance, given the inherent variation in tumors.</p><p>Although we indicate the protocol we ultimately selected for the eight tumor types tested, the optimal protocol could be different for other studies due to sample characteristics and research questions. First, each patient and each sample are different, and researchers must strike a balance between a uniform protocol and realistic expectations of success. Second, the ‘ideal’ protocol depends on the research goals. As we show, most protocols vary in cell recovery, and it is not clear which, if any, provides the full ground truth of cellular composition. Moreover, even when one protocol does provide a faithful cellular composition, a researcher may opt for another approach. For example, some researchers may want to detect as many cell types as possible (and may favor one that enriches rare cells), others may be interested in a specific category of cells and opt for the one that is most successful in their recovery and yet others would want to compare cell proportions across tumors and would want their most faithful representation. Our decision trees will help researchers in making informed choices best suited to their samples and questions.</p><p>When researchers set out to test new protocols, several principles can help in experimental design. First, because clinical samples are often inherently limiting, benchmarking and technical development often cannot be performed on a single matched sample. This is especially the case for fresh sample dissociation, because it requires both larger input specimens and very rapid processing. In this case, we suggest testing different protocols across several samples. Researchers may also choose one of the protocols we presented as a starting point for further optimization. For fresh tissue, researchers should first evaluate tissue dissociation by in-process QCs, especially cell viability and extent of dissociation to single-cell suspension, and only samples that pass those should proceed to scRNA-Seq (for example, in MBC we have previously ruled out the use of Accumax for dissociation (data not shown)). FACS could also be used prior to profiling as an additional QC. It is easier to perform a side-by-side evaluation of nucleus isolation protocols, because they require smaller portions of tissue and can be started at a convenient time. However, in-process experimental QCs for nuclei are less informative and snRNA-Seq is typically required to assess performance.</p><p>Because scRNA-Seq and snRNA-Seq vary in their recovered cellular compositions, it is advantageous, when possible, to analyze both fresh and frozen tumor samples. The choice between scRNA-Seq and snRNA-Seq is typically driven by sample availability, logistics and biological questions. scRNA-Seq measures the expression in the whole cell, and the intact cell membrane allows for selection of specific cellular populations and protein profiling by CITE-Seq<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Stoeckius, M. et al. Simultaneous epitope and transcriptome measurement in single cells. Nat. Methods 14, 865–868 (2017)." href="/articles/s41591-020-0844-1#ref-CR35" id="ref-link-section-d74065049e2537">35</a></sup>. snRNA-Seq decouples sample procurement from processing, recovers nuclei from hard-to-dissociate samples (for example, bone, adipose and liver), and allows multiplexing of samples accrued over time<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Kang, H. M. et al. Multiplexed droplet single-cell RNA-sequencing using natural genetic variation. Nat. Biotechnol. 36, 89–94 (2018)." href="/articles/s41591-020-0844-1#ref-CR36" id="ref-link-section-d74065049e2541">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Stoeckius, M. et al. Cell hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics. Genome Biol. 19, 224 (2018)." href="/articles/s41591-020-0844-1#ref-CR37" id="ref-link-section-d74065049e2544">37</a></sup>, including from banks. This can aid in sample selection and experimental design, reduce batch effects and open the study of rare or unusual samples that may be collected from many sites.</p><p>Our toolbox will help researchers systematically profile additional human tumors, leading to a deeper understanding of tumor biology. The toolbox will support charting of high-resolution tumor cell atlases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Moonshot Cancer Initiative, Generation of Human Tumor Atlases National Cancer Institute (accessed 20 April 2020); 
                  https://ccr.cancer.gov/research/cancer-moonshot
                  
                " href="/articles/s41591-020-0844-1#ref-CR7" id="ref-link-section-d74065049e2551">7</a></sup>, which will yield insights that inform clinical work and should help improve precision in diagnostics and therapeutics.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec13-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec13">Methods</h2><div class="c-article-section__content" id="Sec13-content"><h3 class="c-article__sub-heading" id="Sec14">Experimental methods</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec15">Human patient samples</h4><p>External sample cohorts were added to the Broad Institute’s Molecular Classification of Cancer protocol (15–370B) and reviewed and approved by the Dana-Farber Cancer Institute’s (DFCI) Institutional Review Board (IRB). No subject recruitment or ascertainment was performed as part of the Broad protocol. Samples added to this protocol also underwent IRB review and approval at the institutions where the samples were originally collected. Specifically, DFCI IRB approved the following protocols: NSCLC (IRB protocol 98-063), MBC (IRB protocol 05-246), neuroblastoma (IRB protocols 11-104 and 17-104), ovarian cancer (IRB protocol 02-051), melanoma (IRB protocol 11-104), sarcoma (IRB protocol 17-104), GBM (IRB protocol 10-417) and CLL (IRB protocol 99-224), and the St Jude Children’s Research Hospital IRB approved the following protocol: pediatric high-grade glioma (IRB protocol 97BANK).</p><p>The XPD 09-234 MAST (Molecular Analysis of Solid Tumor) protocol for creating the neuroblastoma O-PDX sample was reviewed and approved by the St Jude Children’s Research Hospital IRB.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec16">Laboratory animals</h4><p>For the neuroblastoma O-PDX sample, animal use was restricted to one female nude athymic mouse for para-adrenal injection of O-PDX cells. This study was carried out in strict accordance with the recommendations in the Guide to Care and Use of Laboratory Animals of the National Institute of Health. The protocol was approved by the Institutional Animal Care and Use Committee at St Jude Children’s Research Hospital. All efforts were made to minimize suffering. All mice were housed in accordance with approved IACUC protocols. Animals were housed on a 12–12 h light cycle (light on 6:00 and off 18:00) and provided food and water ad libitum. Athymic nude female mice were purchased from Charles River Laboratories (strain code 553).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec17">Collection of fresh tissue for scRNA-Seq</h4><p>Collection of fresh solid tumor tissue for NSCLC, ovarian cancer and MBC at Brigham and Women’s Hospital (BWH)/DFCI was performed following protocols established to reduce the time elapsed between removal of the tumor tissue from the body, placement of the specimen in media and processing for scRNA-Seq. To this end, we established procedures between the hospital team (surgeon/clinical research coordinator/clinical pathologist), the coordinating team (project managers/pathology technician) and the processing team (staff scientists/research technicians) before procedure day. This included providing the hospital team with collection containers with appropriate media and predefining allocation priorities to ensure quick handling by the pathology technician of the sample received. On the day of the procedure, timely communication between the teams ensured quick specimen transfer from the hospital team to the research team, timely transport to the Broad Institute for processing, and immediate loading of the single cell suspension into the 10x Genomics Single-Cell Chromium Controller (as described in the section ‘Dissociation workflow from fresh solid tumor samples’ below).</p><p>In all cases, the tissue received from the hospital team was examined by the research pathology technician and, following procurement of a specimen for anatomic pathology review, the highest-quality portion (or core) was allocated for scRNA-Seq, placed in medium and transported to the Broad Institute for dissociation following the appropriate protocol (protocols are detailed for each tumor type below). Tissue quality was assessed based on visual examination and rapid pathology interpretation at the time of collection, and determined based on tumor content, necrosis, calcification, fat and hemorrhage.</p><p>For ovarian cancer ascites, ~300 ml was usually received from the hospital team within 1 h after being taken out of the body, and contained a vast majority of non-malignant (mainly immune) cells. Hence, all ascites samples were subjected to CD45<sup>+</sup> cell depletion (below) to enrich for malignant cells.</p><p>For CLL, samples were generated from peripheral blood mononuclear cells isolated using density centrifugation (Ficoll-Paque) and stored in freezing medium (FBS + 10% DMSO) in liquid nitrogen until processing.</p><p>For O-PDX of neuroblastoma samples <i>Foxn1</i><sup>−<i>/</i>−</sup> nude mice (Charles River Laboratories) were orthotopically injected via ultrasound-guided para-adrenal injection with cells derived from a patient MYCN-amplified neuroblastoma (available as sample SJNBL046_X1 through the Childhood Solid Tumor Network)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Stewart, E. et al. Orthotopic patient-derived xenografts of paediatric solid tumours. Nature 549, 96–100 (2017)." href="/articles/s41591-020-0844-1#ref-CR19" id="ref-link-section-d74065049e2613">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Stewart, E. et al. Development and characterization of a human orthotopic neuroblastoma xenograft. Dev. Biol. 407, 344–355 (2015)." href="/articles/s41591-020-0844-1#ref-CR20" id="ref-link-section-d74065049e2616">20</a></sup>. A portion of O-PDX tumor was flash-frozen for future snRNA-Seq, while the remainder underwent dissociation as described below.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec18">Preservation of tissue for snRNA-Seq</h4><p>For those samples that we prospectively collected for snRNA-Seq (neuroblastoma HTAPP-244-SMP-451 and HTAPP-656-SMP-3481), freezing of tumor samples was performed as quickly as possible after sample collection using a standard biobanking technique and the dates when samples were frozen were recorded. (Other samples were obtained from tissue banks with a limited record on how they were frozen, which is a typical scenario.) Samples were placed in cryo-tubes without any liquid. Complete removal of liquid from the sample was accomplished by gently wiping it (not patting, as this would damage the tissue) on the side of the container, before placing in the cryotube. The tubes were then covered in dry ice and transferred to −80 °C for long-term storage.</p><p>The other frozen samples from snRNA-Seq were obtained from tissue banks as follows: ovarian optimal cutting temperature compound (OCT)-frozen archival samples were obtained from the DFCI Gynecology Oncology Tissue Bank; sarcoma snap-frozen samples were obtained from the Boston Children’s Hospital Tissue Bank; pediatric snap-frozen glioma samples were obtained from the St Jude Children’s Research Hospital Biorepository; neuroblastoma snap-frozen samples were obtained from the St Jude Children’s Research Hospital Biorepository and the Boston Children’s Hospital Precision Link Biobank for Health Discovery; MBC OCT-frozen samples were obtained from the Center for Cancer Precision Medicine Bank; snap-frozen melanoma samples were obtained through the laboratory of Dr C. Yoon at BWH.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec19">Dissociation workflow from fresh solid tumor samples to a single-cell suspension for scRNA-Seq</h4>
                    <h3 class="c-article__sub-heading" id="FPar1">MBC, NSCLC (protocols PDEC and LE), ovarian cancer solid tumor and neuroblastoma workflows</h3>
                    <p>Fresh tissue dissociation of MBC, NSCLC (protocols PDEC and LE), ovarian cancer solid tumor and neuroblastoma were performed using a similar workflow (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2a</a>), with different components of the dissociation mixture for each tumor type, as described in the next section.</p>
                    <p>Samples were transferred from interventional radiology (biopsies) or the operating room (resections) in DMEM (MBC), RPMI (NSCLC) or RPMI with HEPES (ovarian cancer and neuroblastoma) medium. On arrival at the laboratory, the sample was washed in cold PBS and transferred into either a 2 ml Eppendorf tube containing dissociation mixture (for biopsies) or a 5 ml Eppendorf tube containing 3 ml dissociation mixture (for resections). Next, the sample was minced in the Eppendorf tube using spring scissors (Fine Science Tools, cat. no. 15514-12) into fragments under ~0.4 mm, and incubated at 37 °C, while rotating horizontally at ~14 r.p.m., for 10 min. After 10 min, the sample was pipetted 20 times with a 1 ml pipette tip at room temperature and placed back into incubation with rotation for an additional 10 min. The sample was pipetted again 20 times using a 1 ml pipette tip, transferred to a 1.7 ml Eppendorf tube and centrifuged at 300–580<i>g</i> for 4–7 min at 4 °C. The supernatant was removed and the pellet was resuspended in 200–500 µl of ACK (ammonium-chloride-potassium) RBC lysis buffer (Thermo Fisher Scientific, A1049201). The ACK volume added depended on the size of the pellet; while pellet size is hard to quantify, we suggest adding ~100 µl ACK lysis buffer per 100,000 cells, with a minimum volume of 200 µl. The sample was incubated in ACK RBC lysis buffer for 1 min on ice, followed by the addition of cold PBS at twice the volume of the ACK. The cells were pelleted by a short centrifugation for 8 s at 4 °C using the short spin setting with centrifugal force ramping up to, but not exceeding, 11,000<i>g</i>. The supernatant was removed. The pellet color was assessed; if RBCs remained (pellet color pink or red), the ACK step was repeated up to two additional times. To remove cell clumps in the MBC protocol (or sample), the pellet was resuspended in 100 µl of TrypLE (Life Technologies, cat. no. 12604013) and incubated while constantly pipetting at room temperature for 1 min with a 200 µl pipette tip. TrypLE was inactivated by adding 200 µl of cold RPMI 1640 with 10% FBS. The cells were pelleted using short centrifugation as described above. The pellet was resuspended in 50 µl of 0.4% BSA (Ambion, cat. no. AM2616) in PBS. To assess the single-cell suspension, viability and cell count, 5 µl of Trypan blue (Thermo Fisher Scientific, cat. no. T10282) was mixed with 5 µl of the sample and loaded onto an INCYTO C-Chip Disposable Hemocytometer, Neubauer Improved (VWR, cat. no. 82030-468). The cell concentration was adjusted if necessary to a range of 200–2,000 cells per µl. A total of 8,000 cells were loaded into each channel of the 10x Genomics Single-Cell Chromium Controller. Due to differences between clinical samples, some steps may need to be repeated or adjusted; for a general overview of guidelines see Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig2">2a</a>.</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar2">NSCLC-C4 protocol workflow</h3>
                    <p>A similar workflow was used for protocol NSCLC-C4 with the following modifications. Following mechanical chopping as above, sample was dissociated for 15 min in a 15 ml falcon tube, with a gentle vortex every 5 min, followed by filtration through a 70 µm filter, and washed with 20 ml of ice-cold PBS and centrifuged at 580<i>g</i> for 5 min. RBC lysis was performed similarly to the above workflow by resuspending the pellet in 1 ml ACK lysis buffer with incubation on ice for 1 min. A 20 ml volume of ice-cold PBS was added to quench the ACK lysis buffer, followed by filtration through a 70 µm filter, and centrifugation at 580<i>g</i> for 5 min. Sample NSCLC14 was further cleaned using a Viahance dead-cell removal kit (BioPAL, cat. no. CP-50VQ02) according to the manufacturer’s instructions. Cells were then resuspended in M199 and loaded on the 10x Genomics Single-Cell Chromium Controller as described above.</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar3">GBM workflow</h3>
                    <p>All steps were completed on ice. Each sample was minced thoroughly in a Petri dish, 4 ml HBSS was added (Life Technologies, cat. no. 14175095), then the sample was transferred to 15 ml tubes and centrifuged at 1,000 r.p.m. for 2 min. After centrifugation, supernatant was removed, pre-heated dissociation mixture was added, and the sample was incubated while shaking at 37 °C for 15 min. Sample was pipetted up–down 20 times, incubated at 37 °C for an additional 15 min, and pipetted again. After dissociation, the sample was filtered through a 100 μm cell strainer (Fisher Scientific, cat. no. 22–363–549) into a 50 ml tube. We recommend keeping any tissue fragments left in the cell strainer, as they can be reprocessed with the same protocol if initial cell recovery is low. The filtrate was centrifuged at 1,000 r.p.m. for 3 min, and the supernatant was removed. If the pellet was bloody, RBC removal was performed when needed using Lympholyte H (Cedarlane, cat. no. CL5015) or RBC Lysis Solution (10×) (Miltenyi Biotec, cat. no. 130-094-183). The pellet was washed with 10 ml of cold PBS/1% BSA, transferred to a 15 ml tube and centrifuged at 1,200 r.p.m. for 3 min. Supernatant was removed and the pellet was resuspended in 0.4% BSA in PBS. The single-cell suspension was visualized, counted and loaded on the 10x Genomics Single-Cell Chromium Controller as described above.</p>
                  <h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec20">Dissociation mixtures for different tumor types</h4><p>Dissociation mixtures were prepared ~5–10 min before sample processing from frozen aliquoted stocks, as follows.</p>
                    <h3 class="c-article__sub-heading" id="FPar4">MBC LD protocol</h3>
                    <p>A 950 µl volume of RPMI 1640 (Thermo Fisher Scientific, cat. no. 11875093) was used with 10 µl of 10 mg ml<sup>−1</sup> DNAse I (Sigma Aldrich, cat. no. 11284932001) to a final concentration of 100 µg ml<sup>−1</sup>, and 40 µl of 2.5 mg ml<sup>−1</sup> Liberase TM (Sigma Aldrich, cat. no. 5401127001).</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar5">Ovarian cancer resection MHTD kit</h3>
                    <p>The dissociation mixture was based on the Miltenyi Human Tumor Dissociation Kit (Miltenyi Biotec, cat. no. 130-095-929). Before starting, enzymes H, R and A were resuspended according to the manufacturer’s instructions. Dissociation mix containing 2.2 ml RPMI, 100 µl enzyme H, 50 µl enzyme R and 12.5 µl enzyme A was prepared immediately before use.</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar6">Neuroblastoma NB-C4 protocol</h3>
                    <p>Medium 199 with Hanks balanced salts buffer (Thermo Fisher Scientific) was used with 100 µg ml<sup>−1</sup> of DNAse I (Millipore Sigma, cat. no. 11284932001) and 100 µg ml<sup>−1</sup> collagenase IV (Worthington, cat. no. LS004186).</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar7">O-PDX neuroblastoma</h3>
                    <p>A papain kit, the Worthington Papain Dissociation System (cat. no. LK003150), was used. Dissociation was performed according to the manufacturer’s instructions, with deviation of the dissociation duration, which was shortened to 15 min.</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar8">NSCLC PDEC protocol</h3>
                    <p>We used 2692 µl HBSS (Thermo Fisher Scientific, cat. no. 14170112), 187.5 µl of 20 mg ml<sup>−1</sup> pronase (Sigma Aldrich, cat. no. 10165921001) to a final concentration of 1,250 µg ml<sup>−1</sup>, 27.6 µl of 1 mg ml<sup>−1</sup> elastase (Thermo Fisher Scientific, cat. no. NC9301601) to a final concentration of 9.2 µg ml<sup>−1</sup>, 30 µl of 10 mg ml<sup>−1</sup> DNase I (Sigma Aldrich, cat. no. 11284932001) to a final concentration of 100 µg ml<sup>−1</sup>, 30 µl of 10 mg ml<sup>−1</sup> Dispase (Sigma Aldrich, cat. no. 4942078001) to a final concentration of 100 µg ml<sup>−1</sup>, 30 µl of 150 mg ml<sup>−1</sup> collagenase A (Sigma Aldrich, cat. no. 10103578001) to a final concentration of 1,500 µg ml<sup>−1</sup> and 3 µl of 100 mg ml<sup>−1</sup> collagenase IV (Thermo Fisher Scientific, cat. no. NC9836075) to a final concentration of 100 µg ml<sup>−1</sup>.</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar9">NSCLC LE protocol</h3>
                    <p>We used 4.7 ml RPMI 1640 (Thermo Fisher Scientific, cat. no. 11875093), 200 µl of 2.5 mg ml<sup>−1</sup> Liberase TM (Millipore Sigma, cat. no. 5401119001) to a final concentration of 100 µg ml<sup>−1</sup>, 50 µl of 10 mg ml<sup>−1</sup> DNase I (Sigma Aldrich, cat. no. 11284932001) to a final concentration of 100 µg ml<sup>−1</sup> and 46 µl of 1 mg ml<sup>−1</sup> elastase (Thermo Fisher Scientific, cat. no. NC9301601) to a final concentration of 9.2 µg ml<sup>−1</sup>.</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar10">NSCLC-C4 protocol</h3>
                    <p>M199 (5 ml) was used with DNase 1 (final concentration of 10 µg ml) and collagenase IV (final concentration of 100 µg ml<sup>−1</sup>).</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar11">GBM BTD kit</h3>
                    <p>A Brain Tumor Dissociation Kit (P) (Miltenyi Biotech, cat. no. 130-095-942) was used with 4 ml buffer X, 40 µl buffer Y, 50 µl enzyme N and 20 µl enzyme A.</p>
                  <h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">Processing of non-solid tumor samples for scRNA-Seq</h4>
                    <h3 class="c-article__sub-heading" id="FPar12">CLL</h3>
                    <p>Frozen (cryopreserved) cells were thawed in 10 ml RPMI, pelleted and washed with an additional 10 ml RPMI. Live cells were sorted using the MoFlo Astrios EQ Cell Sorter and 8,000 cells were loaded on one channel of the 10x Genomics Single-Cell Chromium Controller. Remaining cells were pelleted by short centrifugation, the supernatant was discarded and the pellet was frozen on dry ice and stored at −80 °C.</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar13">Ovarian cancer ascites</h3>
                    <p>Ascites samples without spheres were selected and delivered in four 50 ml conical tubes, for a total of 200 ml of fluid. Tubes were spun down at 580<i>g</i> for 5 min in a 4 °C pre-cooled centrifuge and supernatants were aspirated.</p>
                    <p>Pellets were resuspended in 5 ml cold ACK lysing buffer and combined from all tubes at this step. ACK lysis was done on ice for 3 min, and quenched by adding 10 ml of cold PBS, followed by centrifugation at 580<i>g</i> for 5 min at 4 °C. Pellet color was assessed; if it was pink or red, revealing a significant portion of erythrocytes, ACK treatment steps were repeated as needed for two additional times, at most. Post ACK treatment, the pellet was resuspended in 20 ml cold PBS, filtered through a 70 µm cell strainer into a 50 ml conical tube, and the filter was washed with additional 20 ml cold PBS to recover as many cells as possible. The sample was then centrifuged at 580<i>g</i> for 5 min at 4 °C. To reduce the fraction of immune cells in the sample, CD45<sup>+</sup> cell depletion was performed using the MACS CD45 depletion protocol described below.</p>
                  <h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec22">Depletion of CD45<sup>+</sup> cells for scRNA-Seq</h4><p>Depletion of CD45<sup>+</sup> cells in ovarian cancer ascites samples and NSCLC samples was performed using CD45 MicroBeads (Miltenyi Biotec, cat. no. 130-045-801) according to the manufacturer’s protocol. Briefly, following filtration of the ovarian cells from ascites or dissociation of NSCLC tissue samples, cells were counted. The single-cell suspension was centrifuged at 500<i>g</i> for 4 min at 4 °C. The supernatant was removed and the pellet was resuspended in 80 µl of MACS buffer (PBS supplemented with 0.5% BSA, and 2 mM EDTA) per 10<sup>6</sup> cells. MACS CD45 microbeads were added to the cell suspension (20 µl per 10 million cells). The cells were incubated on ice for 15 min. During incubation, the column (MS for NSCLC and LS for ovarian ascites) was prepared by attaching the column to a MidiMACS separator and rinsing the column with 3 ml MACS buffer. Following incubation, the cells and bead conjugate were washed with 900 µl MACS buffer per 10 million cells. The cells were centrifuged at 500<i>g</i> for 4 min at 4 °C. The supernatant was removed and the pellet was resuspended in 500 µl MACS buffer. The cell suspension was transferred to the column and the effluent was collected (CD45<sup>−</sup> fraction). The column was washed three times with 3 ml MACS buffer. The CD45<sup>−</sup> fraction was centrifuged at 500<i>g</i> for 4 min at 4 °C. In the ascites sample, bead attachment and column separation can be repeated to increase the number of tumor and stromal cells relative to immune cells. The pellet was resuspended in 50 µl of 0.4% BSA (Ambion, cat. no. AM2616) in PBS. Cells were counted by mixing 5 µl of Trypan blue (Thermo Fisher Scientific, cat. no. T10282) with 5 µl of the sample and loaded on INCYTO C-Chip Disposable Hemocytometer, Neubauer Improved (VWR, cat. no. 82030-468). The cell concentration was adjusted if necessary to a range of 200–2,000 cells per µl. A total of 8,000 cells were loaded into each channel of the 10x Genomics Single-Cell Chromium Controller.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec23">Flow cytometry analysis</h4><p>For flow cytometry analysis of CD45<sup>+</sup> depletion in the ovarian cancer ascites sample, cells were resuspended in PBS complemented with 2% FBS and stained with FITC anti-human CD45 antibody (BioLegend, cat. no. 304006CD45; 1:200 dilution), PE anti-human EpCAM antibody (Miltenyi Biotech, cat. no. 130-113-264; 1:50 dilution), APC anti-human CD14 (BioLegend, cat. no. 367118, clone 63D3; 1:20 dilution) and PE-cy7 anti-human CD24 (BioLegend, cat. no. 311120, clone ML5; 1:20 dilution) for 20 min, and with 7-AAD (Invitrogen, cat. no. A1310; 1:200 dilution) for 5 min. The same cells were also used for single-stain and unstained controls to perform compensation and adjust gating. Analysis was performed on a BD LSRFortessa cell analyzer with BD FACSDiva Software Version 8.0.1 and plots were generated with FlowJo Version 10.5.3. Gating for CD45 and EpCAM was performed as described in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig10">4c</a>. CD24 and CD14 antibodies were included in the antibody panel for FACS analysis to provide additional information and better inform scRNA-Seq. Specifically, expression of CD24 on tumor cells has been shown to relate to ovarian cancer invasiveness and expression of CD14 identifies monocytes/macrophages.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec24">ST-based buffers for snRNA-Seq</h4><p>A 2× stock of salt-Tris solution (ST buffer) containing 292 mM NaCl (Thermo Fisher Scientific, cat. no. AM9759), 20 mM Tris-HCl pH 7.5 (Thermo Fisher Scientific, cat. no. 15567027), 2 mM CaCl<sub>2</sub> (VWR International Ltd, cat. no. 97062-820) and 42 mM MgCl<sub>2</sub> (Sigma Aldrich, cat. no. M1028) in ultrapure water was made and used to prepare three buffers: for CST, 1 ml of 2× ST buffer, 980 µl of 1% CHAPS (Millipore, cat. no. 220201), 10 µl of 2% BSA (New England BioLabs, cat. no. B9000S) and 10 µl of nuclease-free water; for TST, 1 ml of 2× ST buffer, 60 µl of 1% Tween-20 (Sigma Aldrich, cat. no. P-7949), 10 µl of 2% BSA (New England Biolabs, cat. no. B9000S) and 930 µl of nuclease-free water; for NST, 1 ml of 2× ST buffer, 40 µl of 10% Nonidet P40 Substitute (Fisher Scientific, cat. no. AAJ19628AP), 10 µl of 2% BSA (NEB) and 950 µl of nuclease-free water. 1× ST buffer was prepared by dilution 2× ST with ultrapure water (Thermo Fisher Scientific cat. no. 10977023) in a ratio of 1:1.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec25">Nucleus isolation from frozen samples for snRNA-Seq</h4><p>On dry ice, tissue was split and subjected to one of three ST-based nucleus isolation protocols<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Drokhlyansky, E. et al. The enteric nervous system of the human and mouse colon at a single-cell resolution. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/746743v4
                  
                 (2019)." href="/articles/s41591-020-0844-1#ref-CR11" id="ref-link-section-d74065049e2904">11</a></sup> and the EZ nucleus isolation buffer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Habib, N. et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. Nat. Methods 14, 955–958 (2017)." href="/articles/s41591-020-0844-1#ref-CR8" id="ref-link-section-d74065049e2908">8</a></sup>, as detailed in the following.</p>
                    <h3 class="c-article__sub-heading" id="FPar14">Nucleus isolation workflow for ST-based buffers</h3>
                    <p>On ice, a piece of frozen tumor tissue was placed into a well of a 6-well plate (Stem Cell Technologies, cat. no. 38015) with 1 ml of CST, TST or NST buffer. For samples frozen in OCT, an additional step of removing the surrounding OCT and washing any residual OCT from the sample with PBS was performed in a 10 cm Petri dish. Tissue was then chopped using Noyes Spring Scissors (Fine Science Tools, cat. no. 15514-12) for 10 min on ice. For cell pellets, such as for CLL frozen cells, sample was pipetted in the buffer on ice, instead of chopping. The homogenized solution was then filtered through a 40 µm Falcon cell strainer (Thermo Fisher Scientific, cat. no. 08-771-1). An additional 1 ml of the detergent buffer solution was used to wash the well and filter. The volume was brought up to 5 ml with 3 ml of 1× ST buffer. The sample was then transferred to a 15 ml conical tube and centrifuged at 4 °C for 5 min at 500<i>g</i> in a swinging bucket centrifuge. The pellet was resuspended in 1× ST buffer. Resuspension volume was dependent on the size of the pellet, usually within the range of 100–200 µl. The nucleus solution was then filtered through a 35 µm Falcon cell strainer (Corning, cat. no. 352235). Nuclei were counted using a C-chip disposable hemocytometer (VWR, cat. no. 82030-468). Either 10,000 or 8,000 nuclei (V2 or V3 10x Genomics, respectively) of the single-nucleus suspension were loaded onto the Chromium Chips for the Chromium Single Cell 3′ Library (V2, PN-120233; V3, PN-1000075) according to the manufacturer’s recommendations (10x Genomics).</p>
                  
                    <h3 class="c-article__sub-heading" id="FPar15">Nucleus isolation workflow using EZ lysis buffer</h3>
                    <p>Nucleus isolation was done as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Habib, N. et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. Nat. Methods 14, 955–958 (2017)." href="/articles/s41591-020-0844-1#ref-CR8" id="ref-link-section-d74065049e2930">8</a></sup>. Briefly, tissue samples were cut into pieces &lt;0.5 cm and homogenized using a glass Dounce tissue grinder (Sigma, cat. no. D8938). The tissue was homogenized 25 times with pestle A and 25 times with pestle B in 2 ml of ice-cold nuclei EZ lysis buffer. The sample was then incubated on ice for 5 min, with an additional 3 ml of cold EZ lysis buffer. Nuclei were centrifuged at 500<i>g</i> for 5 min at 4 °C, washed with 5 ml ice-cold EZ lysis buffer and incubated on ice for 5 min. After centrifugation, the nucleus pellet was washed with 5 ml nuclei suspension buffer (NSB; consisting of 1× PBS, 0.01% BSA and 0.1% RNase inhibitor (Clontech, cat. no. 2313A)). Isolated nuclei were resuspended in 2 ml NSB, filtered through a 35 μm cell strainer (Corning-Falcon, cat. no. 352235) and counted. A final concentration of 1,000 nuclei per µl was used for loading on a 10x channel.</p>
                  <h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec26">Droplet-based sc/snRNA-Seq</h4><p>For V2 10x technology, either 8,000 single cells or 10,000 single nuclei were loaded into each channel of a Chromium single-cell 3′ Chip. For V3 10x technology, 8,000 single cells and 8,000 single nuclei were loaded. Single cells/nuclei were partitioned into droplets with gel beads in the Chromium Controller. After emulsions were formed, barcoded reverse transcription of RNA took place. This was followed by cDNA amplification, fragmentation and adapter and sample index attachment, all according to the manufacturer’s recommendations. Libraries from four 10x channels were pooled together and sequenced on one lane of an Illumina HiSeq X, or on one flow cell of a NextSeq, with paired end reads as follows: read 1, 26 nt; read 2, 55 nt; index 1, 8 nt; index 2, 0 nt.</p><h3 class="c-article__sub-heading" id="Sec27">Computational methods</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec28">scRNA-Seq data processing</h4><p>We used Cell Ranger mkfastq (v2.0 and v3.0) (10x Genomics) to generate demultiplexed FASTQ files from the raw sequencing reads. We aligned these reads to the human GRCh38 genome and quantified gene counts as UMIs using Cell Ranger count (v2.0 and v3.0) (10x Genomics). For snRNA-Seq reads, we counted reads mapping to introns as well as exons, as this results in a greater number of genes detected per nucleus, more nuclei passing quality control and better cell type identification, as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Bakken, T. E. et al. Single-nucleus and single-cell transcriptomes compared in matched cortical cell types. PLoS ONE 13, e0209648 (2018)." href="/articles/s41591-020-0844-1#ref-CR38" id="ref-link-section-d74065049e2959">38</a></sup>. To count introns during read mapping, we followed the approach described at <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/advanced/references">https://support.10xgenomics.com/single-cell-gene-expression/software/pipelines/latest/advanced/references</a>. Briefly, we built a ‘pre-mRNA’ human GRCh38 reference using Cell Ranger mkref (v3.0) (10x Genomics) and a modified gene transfer format (GTF) file, where, for each transcript, the feature type had been changed from transcript to exon. The starting GTF files came from refdata-cellranger-GRCh38-1.2.0.tar.gz or refdata-cellranger-GRCh38-3.0.0.tar.gz, and are available for download at <a href="https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/3.0">https://support.10xgenomics.com/single-cell-gene-expression/software/downloads/3.0</a>.</p><p>To downsample sequencing reads or gene counts (UMIs) when comparing protocols, we used downsampleReads and downsampleMatrix, respectively, from the R package DropletUtils (v1.0.3 or higher)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Lun, A. T. L. et al. EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data. Genome Biol. 20, 63 (2019)." href="/articles/s41591-020-0844-1#ref-CR12" id="ref-link-section-d74065049e2980">12</a></sup>. Reads were downsampled to match the protocol with the lowest number of total reads. After downsampling by total reads, we used write10xCounts from DropletUtils and a custom Python script to generate an HDF5 file for input into our analysis pipelines, as described in the sections that follow and in the ‘Code availability’ section.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec29">QC of scRNA-Seq data</h4><p>To maintain explicit control over all gene and cell quality control filters, in all our downstream analyses we used the raw feature-barcode matrix, rather than the filtered feature-barcode matrix generated by Cell Ranger. We removed low-quality cells by requiring each cell to have a minimal number of UMIs and genes detected. We used different thresholds depending on the experimental modality (single cell or single nucleus) and on the 10x kit (V2 or V3 chemistry). For single nucleus data, we retained nuclei with at least 200 genes and 400 UMIs detected by V2 chemistry and with at least 500 genes and 1,000 UMIs detected by V3 chemistry. For single-cell data, we retained cells with at least 500 genes and 1,000 UMIs detected by either V2 or V3 chemistry. For the V2–V3 comparison in HTAPP-951-SMP-4652 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig15">9</a>), we used the same thresholds for both chemistries: at least 200 genes and 400 UMIs detected. For both data types, we filtered out those cells or nuclei where &gt;20% of UMIs came from mitochondrial genes. Finally, we normalized the total UMIs per cell or nucleus to 100,000 (CP100K) and log-transformed these values to report gene expression as <i>E</i> = log(CP100K + 1).</p><p>We reported the following QC metrics: number of total reads per library sample, sequencing saturation (fraction of reads originating from an already-observed UMI as reported by Cell Ranger count), total recovered cells or nuclei, number of reads per cell or nucleus, number of UMIs per cell or nucleus, number of genes detected per cell or nucleus, fraction of UMIs in a cell or nucleus aligned to mitochondrial genes, fraction of droplets estimated to contain only ambient RNA (‘empty drops’), fraction of cell or nucleus doublets, the number of detected cell types and the pattern of CNAs for malignant cells. For a subset of samples, we also calculated the number of cells or nuclei per detected cell type and the estimated level of ambient RNA in droplets containing cells.</p><p>We predicted droplets containing only ambient RNA and no cells using EmptyDrops (part of DropletUtils, v1.0.3 or higher), with the retain parameter set by the knee of the curve in the barcode rank plot (cell barcodes ranked by their total UMIs)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Lun, A. T. L. et al. EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data. Genome Biol. 20, 63 (2019)." href="/articles/s41591-020-0844-1#ref-CR12" id="ref-link-section-d74065049e3004">12</a></sup>. We predicted potential doublets using Scrublet (v0.2) with expected_doublet_rate = 0.06 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Wolock, S. L., Lopez, R. &amp; Klein, A. M. Scrublet: computational identification of cell doublets in single-cell transcriptomic data. Cell Syst. 8, 281–291.e289 (2019)." href="/articles/s41591-020-0844-1#ref-CR13" id="ref-link-section-d74065049e3008">13</a></sup>). We estimated the levels of ambient RNA using SoupX (v0.3.1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Young, M. D. &amp; Behjati, S. SoupX removes ambient RNA contamination from droplet based single cell RNA sequencing data. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/303727v1
                  
                 (2018)." href="/articles/s41591-020-0844-1#ref-CR18" id="ref-link-section-d74065049e3012">18</a></sup> and a set of cell-type-specific marker genes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0844-1#MOESM2">1</a>). Importantly, we flagged the doublets and empty drops and retained them in our analysis, instead of immediately filtering them out. Droplets that appear to contain doublets or empty drops can arise from many different effects, such as cellular differentiation or insufficient sequencing, and by carrying them through the analysis, potential doublets or empty drops can be more clearly interpreted in the context of the full dataset.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec30">Dimensionality reduction, clustering and visualization</h4><p>For each tumor sample, we analyzed the filtered expression matrix to identify cell subsets, as previously described<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Shekhar, K. et al. Comprehensive classification of retinal bipolar neurons by single-cell transcriptomics. Cell 166, 1308–1323.e1330 (2016)." href="/articles/s41591-020-0844-1#ref-CR39" id="ref-link-section-d74065049e3027">39</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Wolf, F. A., Angerer, P. &amp; Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. Genome Biol. 19, 15 (2018)." href="/articles/s41591-020-0844-1#ref-CR40" id="ref-link-section-d74065049e3030">40</a></sup>. We chose highly variable genes with a <i>z</i>-score cutoff of 0.5 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Macosko, E. Z. et al. Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets. Cell 161, 1202–1214 (2015)." href="/articles/s41591-020-0844-1#ref-CR41" id="ref-link-section-d74065049e3037">41</a></sup>), centered and scaled the expression of each gene to have a mean of zero and standard deviation of one, and performed dimensionality reduction on the variable genes using principal component analysis. We used the top 50 principal components (PCs) as input to Louvain graph-based clustering, with the resolution parameter set to 1.3. For each cluster of cells, we identified cluster-specific differentially expressed genes using the following tests: an AUC classifier, Welch’s <i>t</i>-test and Fisher’s exact test. For tests that returned a <i>P</i> value, we controlled the false discovery rate at 5% with the Benjamini–Hochberg procedure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Benjamini, Y. &amp; Yosef, Hochberg Controlling the false discovery rate: a practical and powerful approach to multiple testing. J. R. Stat. Soc. B 57.1, 289–300 (1995)." href="/articles/s41591-020-0844-1#ref-CR42" id="ref-link-section-d74065049e3048">42</a></sup>. We visualized gene expression and clustering results by embedding cells or nuclei profiles in a Uniform Manifold Approximation and Projection (UMAP)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="McInnes, L., Healy, J. &amp; Melville, J. UMAP: Uniform Manifold Approximation and Projection for dimension reduction. Preprint at 
                  https://arxiv.org/abs/1802.03426
                  
                 (2018)." href="/articles/s41591-020-0844-1#ref-CR43" id="ref-link-section-d74065049e3052">43</a></sup> of the top 50 PCs, with min_dist = 0.5, spread = 1.0, number of neighbors = 15 and the Euclidean distance metric.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec31">Annotating cell subsets</h4><p>For each cell subset identified by clustering, we assigned a cell type from the malignant, parenchymal, stromal and immune compartments of the tumor microenvironment using a combination of differentially expressed genes, known gene signatures (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0844-1#MOESM2">1</a>) and SingleR (v0.2.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Aran, D. et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. Nat. Immunol. 20, 163–172 (2019)." href="/articles/s41591-020-0844-1#ref-CR44" id="ref-link-section-d74065049e3067">44</a></sup>, an automated annotation package. When running SingleR, only cell types assigned to 30 or more cells were considered. When scoring cells for the expression of known gene signatures, we used the AddModuleScore function in Seurat (v2.3.4)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies and species. Nat. Biotechnol. 36, 411–420 (2018)." href="/articles/s41591-020-0844-1#ref-CR24" id="ref-link-section-d74065049e3071">24</a></sup>. We note that overlapping expression programs between T cells and NK cells make these cell types sometimes more difficult to identify accurately. We did not distinguish macrophages, monocytes and dendritic cells, and annotated all of these as scoring for ‘macrophages’ signatures.</p><p>We identified the malignant cells by inferring chromosomal CNAs from the gene-expression data using inferCNV (v1.1.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Tickle, T. I., Georgescu, C., Brown, M. &amp; Haas, B. inferCNV of the Trinity CTAT Project (2019); 
                  https://github.com/broadinstitute/inferCNV
                  
                " href="/articles/s41591-020-0844-1#ref-CR45" id="ref-link-section-d74065049e3078">45</a></sup>. On a sample-by-sample basis, we used the immune and endothelial cells as a healthy reference to estimate CNAs in the malignant cells. We created the count matrix file and annotation file for inferCNV by randomly subsetting the counts data to sample at most 2,000 cells or nuclei. We created a gene ordering file from the human GRCh38 assembly, which contains the chromosomal start and end positions for each gene. To run inferCNV, we used a cutoff of 0.1 for the minimum average read counts per gene among reference cells or nuclei, clustered according to the annotated cell types, denoised our output, ran a hidden Markov model (HMM) to predict the CNA level, implemented inferCNV’s i6 HMM model, and requested eight threads for parallel steps.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec32">Comparing sc- and snRNA-Seq data</h4><p>To compare profiles between sc- and snRNA-Seq data collected from the same sample, we used a batch correction approach.</p><p>We performed batch correction using CCA as implemented in Seurat (v2.3.4)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies and species. Nat. Biotechnol. 36, 411–420 (2018)." href="/articles/s41591-020-0844-1#ref-CR24" id="ref-link-section-d74065049e3094">24</a></sup>. We selected 1,500 genes that were variable across both the cell and nucleus data, used those genes as input to RunCCA to compute the first 20 canonical components, and aligned the first 12 canonical components with AlignSubspace. The aligned canonical components represent a co-embedding of the cell and nucleus data, and we carried out clustering in this dimensionality-reduced space using FindClusters.</p><p>Following batch correction by CCA, we scored the dissociation signature from ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="van den Brink, S. C. et al. Single-cell sequencing reveals dissociation-induced gene expression in tissue subpopulations. Nat. Methods 14, 935–936 (2017)." href="/articles/s41591-020-0844-1#ref-CR25" id="ref-link-section-d74065049e3101">25</a></sup> (from their Supplementary Table 5) on our matched cell/nuclei samples using the AddModuleScore function in Seurat (v2.3.4)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies and species. Nat. Biotechnol. 36, 411–420 (2018)." href="/articles/s41591-020-0844-1#ref-CR24" id="ref-link-section-d74065049e3105">24</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec33">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0844-1#MOESM1">Nature Research Reporting Summary</a> linked to this Article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>All main and extended data figures have associated raw data. Raw data will be available in the controlled access repository dbGaP (<a href="https://www.ncbi.nlm.nih.gov/gap/">https://www.ncbi.nlm.nih.gov/gap/</a>), under dbGaP Study Accession phs001983.v1.p1. Raw data will also be available in the controlled access repository DUOS (<a href="https://duos.broadinstitute.org/">https://duos.broadinstitute.org/</a>), under DUOS Dataset IDs DUOS-000111, DUOS-000112, DUOS-000113 and DUOS-000114. The counts matrices and metadata for each sample will be publicly available in the Gene Expression Omnibus (<a href="https://www.ncbi.nlm.nih.gov/geo/">https://www.ncbi.nlm.nih.gov/geo/</a>) under data repository accession no. <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE140819">GSE140819</a>. Finally, we provide a website that displays a comprehensive analysis summary for each sample tested (<a href="https://tumor-toolbox.broadinstitute.org">https://tumor-toolbox.broadinstitute.org</a>).</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>We implemented all initial analysis steps, from FASTQ files to clustering cell subsets, in Cumulus<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Li, B. et al. Cumulus: a cloud-based data analysis framework for large-scale single-cell and single-nucleus RNA-seq. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/823682v1
                  
                 (2019)." href="/articles/s41591-020-0844-1#ref-CR14" id="ref-link-section-d74065049e3257">14</a></sup>, which may be executed in both a cloud-based environment and locally. Pipelines were written in the Workflow Description Language (WDL) and run on Cromwell in the Terra Cloud platform (<a href="https://app.terra.bio/">https://app.terra.bio/</a>), and data were stored in Google Cloud Platform storage buckets. Cumulus workflows are publicly available at <a href="https://github.com/klarman-cell-observatory/Cumulus">https://github.com/klarman-cell-observatory/Cumulus</a>.</p>
              <p>We built and ran an additional pipeline for subsequent QC steps, including detecting empty drops and doublets, annotating cell subsets, evaluating cell type specific QCs and inferring CNAs. This pipeline was implemented in R (v3.5 or higher) by converting the single-cell AnnData objects from Cumulus into Seurat objects, and was used to compare and evaluate processing protocols. Our analysis pipeline, including complete example analysis for one scRNA-Seq sample and one snRNA-Seq sample, will be made publicly available at <a href="https://github.com/klarman-cell-observatory/HTAPP-Pipelines">https://github.com/klarman-cell-observatory/HTAPP-Pipelines</a>.</p>
            </div></div></section><section data-title="Change history"><div class="c-article-section" id="change-history-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="change-history">Change history</h2><div class="c-article-section__content" id="change-history-content"><ul class="c-article-change-list"><li class="c-article-change-list__item u-mb-24" id="chg1"><ins datetime="2020-06-25"><h3 class="c-article-change-list__heading u-h3 u-pr-8 u-display-inline">25 June 2020</h3><div class="c-article-change-list__details"><p>A Correction to this paper has been published: <a href="https://doi.org/10.1038/s41591-020-0976-3">https://doi.org/10.1038/s41591-020-0976-3</a></p></div></ins></li></ul></div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Cieslik, M. &amp; Chinnaiyan, A. M. Cancer transcriptome profiling at the juncture of clinical translation. <i>Nat. Rev. Genet.</i> <b>19</b>, 93–109 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nrg.2017.96" data-track-item_id="10.1038/nrg.2017.96" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrg.2017.96" aria-label="Article reference 1" data-doi="10.1038/nrg.2017.96">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVCjsr7K" aria-label="CAS reference 1">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29279605" aria-label="PubMed reference 1">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Cancer%20transcriptome%20profiling%20at%20the%20juncture%20of%20clinical%20translation&amp;journal=Nat.%20Rev.%20Genet.&amp;doi=10.1038%2Fnrg.2017.96&amp;volume=19&amp;pages=93-109&amp;publication_year=2018&amp;author=Cieslik%2CM&amp;author=Chinnaiyan%2CAM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Filbin, M. G. et al. Developmental and oncogenic programs in H3K27M gliomas dissected by single-cell RNA-seq. <i>Science</i> <b>360</b>, 331–335 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aao4750" data-track-item_id="10.1126/science.aao4750" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aao4750" aria-label="Article reference 2" data-doi="10.1126/science.aao4750">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXns12gtLg%3D" aria-label="CAS reference 2">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29674595" aria-label="PubMed reference 2">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5949869" aria-label="PubMed Central reference 2">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Developmental%20and%20oncogenic%20programs%20in%20H3K27M%20gliomas%20dissected%20by%20single-cell%20RNA-seq&amp;journal=Science&amp;doi=10.1126%2Fscience.aao4750&amp;volume=360&amp;pages=331-335&amp;publication_year=2018&amp;author=Filbin%2CMG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Jerby-Arnon, L. et al. A cancer cell program promotes T cell exclusion and resistance to checkpoint blockade. <i>Cell</i> <b>175</b>, 984–997.e924 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2018.09.006" data-track-item_id="10.1016/j.cell.2018.09.006" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2018.09.006" aria-label="Article reference 3" data-doi="10.1016/j.cell.2018.09.006">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVyhurvK" aria-label="CAS reference 3">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30388455" aria-label="PubMed reference 3">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6410377" aria-label="PubMed Central reference 3">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20cancer%20cell%20program%20promotes%20T%20cell%20exclusion%20and%20resistance%20to%20checkpoint%20blockade&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.09.006&amp;volume=175&amp;pages=984-997.e924&amp;publication_year=2018&amp;author=Jerby-Arnon%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Puram, S. V. et al. Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. <i>Cell</i> <b>171</b>, 1611–1624.e1624 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2017.10.044" data-track-item_id="10.1016/j.cell.2017.10.044" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.10.044" aria-label="Article reference 4" data-doi="10.1016/j.cell.2017.10.044">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFWms7nE" aria-label="CAS reference 4">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29198524" aria-label="PubMed reference 4">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5878932" aria-label="PubMed Central reference 4">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20transcriptomic%20analysis%20of%20primary%20and%20metastatic%20tumor%20ecosystems%20in%20head%20and%20neck%20cancer&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.10.044&amp;volume=171&amp;pages=1611-1624.e1624&amp;publication_year=2017&amp;author=Puram%2CSV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Tirosh, I. et al. Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. <i>Nature</i> <b>539</b>, 309–313 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature20123" data-track-item_id="10.1038/nature20123" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature20123" aria-label="Article reference 5" data-doi="10.1038/nature20123">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27806376" aria-label="PubMed reference 5">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5465819" aria-label="PubMed Central reference 5">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20RNA-seq%20supports%20a%20developmental%20hierarchy%20in%20human%20oligodendroglioma&amp;journal=Nature&amp;doi=10.1038%2Fnature20123&amp;volume=539&amp;pages=309-313&amp;publication_year=2016&amp;author=Tirosh%2CI">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Venteicher, A. S. et al. Decoupling genetics, lineages and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. <i>Science</i> <b>355</b>, eaai8478 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aai8478" data-track-item_id="10.1126/science.aai8478" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aai8478" aria-label="Article reference 6" data-doi="10.1126/science.aai8478">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28360267" aria-label="PubMed reference 6">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5519096" aria-label="PubMed Central reference 6">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Decoupling%20genetics%2C%20lineages%20and%20microenvironment%20in%20IDH-mutant%20gliomas%20by%20single-cell%20RNA-seq&amp;journal=Science&amp;doi=10.1126%2Fscience.aai8478&amp;volume=355&amp;publication_year=2017&amp;author=Venteicher%2CAS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Moonshot Cancer Initiative, Generation of Human Tumor Atlases <i>National Cancer Institute</i> (accessed 20 April 2020); <a href="https://ccr.cancer.gov/research/cancer-moonshot" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://ccr.cancer.gov/research/cancer-moonshot">https://ccr.cancer.gov/research/cancer-moonshot</a></p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Habib, N. et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. <i>Nat. Methods</i> <b>14</b>, 955–958 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmeth.4407" data-track-item_id="10.1038/nmeth.4407" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.4407" aria-label="Article reference 8" data-doi="10.1038/nmeth.4407">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtl2gtL7L" aria-label="CAS reference 8">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28846088" aria-label="PubMed reference 8">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5623139" aria-label="PubMed Central reference 8">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Massively%20parallel%20single-nucleus%20RNA-seq%20with%20DroNc-seq&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4407&amp;volume=14&amp;pages=955-958&amp;publication_year=2017&amp;author=Habib%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Habib, N. et al. Div-Seq: single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons. <i>Science</i> <b>353</b>, 925–928 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aad7038" data-track-item_id="10.1126/science.aad7038" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aad7038" aria-label="Article reference 9" data-doi="10.1126/science.aad7038">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhtlyls7rK" aria-label="CAS reference 9">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27471252" aria-label="PubMed reference 9">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5480621" aria-label="PubMed Central reference 9">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Div-Seq%3A%20single-nucleus%20RNA-Seq%20reveals%20dynamics%20of%20rare%20adult%20newborn%20neurons&amp;journal=Science&amp;doi=10.1126%2Fscience.aad7038&amp;volume=353&amp;pages=925-928&amp;publication_year=2016&amp;author=Habib%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Gaublomme, J. T. et al. Nuclei multiplexing with barcoded antibodies for single-nucleus genomics. <i>Nat. Commun.</i> <b>10</b>, 2907 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-019-10756-2" data-track-item_id="10.1038/s41467-019-10756-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-10756-2" aria-label="Article reference 10" data-doi="10.1038/s41467-019-10756-2">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31266958" aria-label="PubMed reference 10">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6606589" aria-label="PubMed Central reference 10">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Nuclei%20multiplexing%20with%20barcoded%20antibodies%20for%20single-nucleus%20genomics&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-10756-2&amp;volume=10&amp;publication_year=2019&amp;author=Gaublomme%2CJT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Drokhlyansky, E. et al. The enteric nervous system of the human and mouse colon at a single-cell resolution. Preprint at <i>bioRxiv</i> <a href="https://www.biorxiv.org/content/10.1101/746743v4" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.biorxiv.org/content/10.1101/746743v4">https://www.biorxiv.org/content/10.1101/746743v4</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Lun, A. T. L. et al. EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data. <i>Genome Biol.</i> <b>20</b>, 63 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-019-1662-y" data-track-item_id="10.1186/s13059-019-1662-y" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-019-1662-y" aria-label="Article reference 12" data-doi="10.1186/s13059-019-1662-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30902100" aria-label="PubMed reference 12">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6431044" aria-label="PubMed Central reference 12">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=EmptyDrops%3A%20distinguishing%20cells%20from%20empty%20droplets%20in%20droplet-based%20single-cell%20RNA%20sequencing%20data&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-019-1662-y&amp;volume=20&amp;publication_year=2019&amp;author=Lun%2CATL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Wolock, S. L., Lopez, R. &amp; Klein, A. M. Scrublet: computational identification of cell doublets in single-cell transcriptomic data. <i>Cell Syst</i>. <b>8</b>, 281–291.e289 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cels.2018.11.005" data-track-item_id="10.1016/j.cels.2018.11.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cels.2018.11.005" aria-label="Article reference 13" data-doi="10.1016/j.cels.2018.11.005">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyhtbw%3D" aria-label="CAS reference 13">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30954476" aria-label="PubMed reference 13">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6625319" aria-label="PubMed Central reference 13">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Scrublet%3A%20computational%20identification%20of%20cell%20doublets%20in%20single-cell%20transcriptomic%20data&amp;journal=Cell%20Syst&amp;doi=10.1016%2Fj.cels.2018.11.005&amp;volume=8&amp;pages=281-291.e289&amp;publication_year=2019&amp;author=Wolock%2CSL&amp;author=Lopez%2CR&amp;author=Klein%2CAM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Li, B. et al. Cumulus: a cloud-based data analysis framework for large-scale single-cell and single-nucleus RNA-seq. Preprint at <i>bioRxiv</i> <a href="https://www.biorxiv.org/content/10.1101/823682v1" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.biorxiv.org/content/10.1101/823682v1">https://www.biorxiv.org/content/10.1101/823682v1</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Al-Hajj, M., Wicha, M. S., Benito-Hernandez, A., Morrison, S. J. &amp; Clarke, M. F. Prospective identification of tumorigenic breast cancer cells. <i>Proc. Natl Acad. Sci. USA</i> <b>100</b>, 3983–3988 (2003).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.0530291100" data-track-item_id="10.1073/pnas.0530291100" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.0530291100" aria-label="Article reference 15" data-doi="10.1073/pnas.0530291100">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3sXivFSqtbs%3D" aria-label="CAS reference 15">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12629218" aria-label="PubMed reference 15">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC153034" aria-label="PubMed Central reference 15">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Prospective%20identification%20of%20tumorigenic%20breast%20cancer%20cells&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0530291100&amp;volume=100&amp;pages=3983-3988&amp;publication_year=2003&amp;author=Al-Hajj%2CM&amp;author=Wicha%2CMS&amp;author=Benito-Hernandez%2CA&amp;author=Morrison%2CSJ&amp;author=Clarke%2CMF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">McDivitt, R. W., Stone, K. R. &amp; Meyer, J. S. A method for dissociation of viable human breast cancer cells that produces flow cytometric kinetic information similar to that obtained by thymidine labeling. <i>Cancer Res.</i> <b>44</b>, 2628–2633 (1984).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaL2c3gsFCitA%3D%3D" aria-label="CAS reference 16">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6327021" aria-label="PubMed reference 16">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20method%20for%20dissociation%20of%20viable%20human%20breast%20cancer%20cells%20that%20produces%20flow%20cytometric%20kinetic%20information%20similar%20to%20that%20obtained%20by%20thymidine%20labeling&amp;journal=Cancer%20Res.&amp;volume=44&amp;pages=2628-2633&amp;publication_year=1984&amp;author=McDivitt%2CRW&amp;author=Stone%2CKR&amp;author=Meyer%2CJS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Neftel, C. et al. An integrative model of cellular states, plasticity and genetics for glioblastoma. <i>Cell</i> <b>178</b>, 835–849 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2019.06.024" data-track-item_id="10.1016/j.cell.2019.06.024" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.06.024" aria-label="Article reference 17" data-doi="10.1016/j.cell.2019.06.024">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVWlur7I" aria-label="CAS reference 17">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31327527" aria-label="PubMed reference 17">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6703186" aria-label="PubMed Central reference 17">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20integrative%20model%20of%20cellular%20states%2C%20plasticity%20and%20genetics%20for%20glioblastoma&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.06.024&amp;volume=178&amp;pages=835-849&amp;publication_year=2019&amp;author=Neftel%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Young, M. D. &amp; Behjati, S. SoupX removes ambient RNA contamination from droplet based single cell RNA sequencing data. Preprint at <i>bioRxiv</i> <a href="https://www.biorxiv.org/content/10.1101/303727v1" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.biorxiv.org/content/10.1101/303727v1">https://www.biorxiv.org/content/10.1101/303727v1</a> (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Stewart, E. et al. Orthotopic patient-derived xenografts of paediatric solid tumours. <i>Nature</i> <b>549</b>, 96–100 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature23647" data-track-item_id="10.1038/nature23647" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature23647" aria-label="Article reference 19" data-doi="10.1038/nature23647">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVWhu7jJ" aria-label="CAS reference 19">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28854174" aria-label="PubMed reference 19">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659286" aria-label="PubMed Central reference 19">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Orthotopic%20patient-derived%20xenografts%20of%20paediatric%20solid%20tumours&amp;journal=Nature&amp;doi=10.1038%2Fnature23647&amp;volume=549&amp;pages=96-100&amp;publication_year=2017&amp;author=Stewart%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Stewart, E. et al. Development and characterization of a human orthotopic neuroblastoma xenograft. <i>Dev. Biol.</i> <b>407</b>, 344–355 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ydbio.2015.02.002" data-track-item_id="10.1016/j.ydbio.2015.02.002" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ydbio.2015.02.002" aria-label="Article reference 20" data-doi="10.1016/j.ydbio.2015.02.002">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXmsFSjtL8%3D" aria-label="CAS reference 20">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25863122" aria-label="PubMed reference 20">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4995597" aria-label="PubMed Central reference 20">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20and%20characterization%20of%20a%20human%20orthotopic%20neuroblastoma%20xenograft&amp;journal=Dev.%20Biol.&amp;doi=10.1016%2Fj.ydbio.2015.02.002&amp;volume=407&amp;pages=344-355&amp;publication_year=2015&amp;author=Stewart%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Hermansen, J. U., Tjonnfjord, G. E., Munthe, L. A., Tasken, K. &amp; Skanland, S. S. Cryopreservation of primary B cells minimally influences their signaling responses. <i>Sci. Rep.</i> <b>8</b>, 17651 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41598-018-36121-9" data-track-item_id="10.1038/s41598-018-36121-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41598-018-36121-9" aria-label="Article reference 21" data-doi="10.1038/s41598-018-36121-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30518828" aria-label="PubMed reference 21">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6281576" aria-label="PubMed Central reference 21">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Cryopreservation%20of%20primary%20B%20cells%20minimally%20influences%20their%20signaling%20responses&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-018-36121-9&amp;volume=8&amp;publication_year=2018&amp;author=Hermansen%2CJU&amp;author=Tjonnfjord%2CGE&amp;author=Munthe%2CLA&amp;author=Tasken%2CK&amp;author=Skanland%2CSS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Guillaumet-Adkins, A. et al. Single-cell transcriptome conservation in cryopreserved cells and tissues. <i>Genome Biol</i>. <b>18</b>, 45 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-017-1171-9" data-track-item_id="10.1186/s13059-017-1171-9" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-017-1171-9" aria-label="Article reference 22" data-doi="10.1186/s13059-017-1171-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28249587" aria-label="PubMed reference 22">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5333448" aria-label="PubMed Central reference 22">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20transcriptome%20conservation%20in%20cryopreserved%20cells%20and%20tissues&amp;journal=Genome%20Biol&amp;doi=10.1186%2Fs13059-017-1171-9&amp;volume=18&amp;publication_year=2017&amp;author=Guillaumet-Adkins%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Gao, R. et al. Nanogrid single-nucleus RNA sequencing reveals phenotypic diversity in breast cancer. <i>Nat. Commun.</i> <b>8</b>, 228 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-017-00244-w" data-track-item_id="10.1038/s41467-017-00244-w" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-017-00244-w" aria-label="Article reference 23" data-doi="10.1038/s41467-017-00244-w">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28794488" aria-label="PubMed reference 23">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5550415" aria-label="PubMed Central reference 23">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Nanogrid%20single-nucleus%20RNA%20sequencing%20reveals%20phenotypic%20diversity%20in%20breast%20cancer&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-00244-w&amp;volume=8&amp;publication_year=2017&amp;author=Gao%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies and species. <i>Nat. Biotechnol.</i> <b>36</b>, 411–420 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nbt.4096" data-track-item_id="10.1038/nbt.4096" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.4096" aria-label="Article reference 24" data-doi="10.1038/nbt.4096">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXmslKrtL0%3D" aria-label="CAS reference 24">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29608179" aria-label="PubMed reference 24">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6700744" aria-label="PubMed Central reference 24">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Integrating%20single-cell%20transcriptomic%20data%20across%20different%20conditions%2C%20technologies%20and%20species&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.4096&amp;volume=36&amp;pages=411-420&amp;publication_year=2018&amp;author=Butler%2CA&amp;author=Hoffman%2CP&amp;author=Smibert%2CP&amp;author=Papalexi%2CE&amp;author=Satija%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">van den Brink, S. C. et al. Single-cell sequencing reveals dissociation-induced gene expression in tissue subpopulations. <i>Nat. Methods</i> <b>14</b>, 935–936 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmeth.4437" data-track-item_id="10.1038/nmeth.4437" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.4437" aria-label="Article reference 25" data-doi="10.1038/nmeth.4437">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28960196" aria-label="PubMed reference 25">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20sequencing%20reveals%20dissociation-induced%20gene%20expression%20in%20tissue%20subpopulations&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4437&amp;volume=14&amp;pages=935-936&amp;publication_year=2017&amp;author=Brink%2CSC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Baslan, T. &amp; Hicks, J. Unravelling biology and shifting paradigms in cancer with single-cell sequencing. <i>Nat. Rev. Cancer</i> <b>17</b>, 557–569 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nrc.2017.58" data-track-item_id="10.1038/nrc.2017.58" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrc.2017.58" aria-label="Article reference 26" data-doi="10.1038/nrc.2017.58">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlyisb3M" aria-label="CAS reference 26">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28835719" aria-label="PubMed reference 26">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Unravelling%20biology%20and%20shifting%20paradigms%20in%20cancer%20with%20single-cell%20sequencing&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc.2017.58&amp;volume=17&amp;pages=557-569&amp;publication_year=2017&amp;author=Baslan%2CT&amp;author=Hicks%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Suva, M. L. &amp; Tirosh, I. Single-cell RNA sequencing in cancer: lessons learned and emerging challenges. <i>Mol. Cell</i> <b>75</b>, 7–12 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.molcel.2019.05.003" data-track-item_id="10.1016/j.molcel.2019.05.003" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.molcel.2019.05.003" aria-label="Article reference 27" data-doi="10.1016/j.molcel.2019.05.003">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVelu7bJ" aria-label="CAS reference 27">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31299208" aria-label="PubMed reference 27">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20RNA%20sequencing%20in%20cancer%3A%20lessons%20learned%20and%20emerging%20challenges&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2019.05.003&amp;volume=75&amp;pages=7-12&amp;publication_year=2019&amp;author=Suva%2CML&amp;author=Tirosh%2CI">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Rambow, F. et al. Toward minimal residual disease-directed therapy in melanoma. <i>Cell</i> <b>174</b>, 843–855.e819 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2018.06.025" data-track-item_id="10.1016/j.cell.2018.06.025" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2018.06.025" aria-label="Article reference 28" data-doi="10.1016/j.cell.2018.06.025">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlartrfI" aria-label="CAS reference 28">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30017245" aria-label="PubMed reference 28">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Toward%20minimal%20residual%20disease-directed%20therapy%20in%20melanoma&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.06.025&amp;volume=174&amp;pages=843-855.e819&amp;publication_year=2018&amp;author=Rambow%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Qi, Z., Barrett, T., Parikh, A. S., Tirosh, I. &amp; Puram, S. V. Single-cell sequencing and its applications in head and neck cancer. <i>Oral Oncol</i>. <b>99</b>, 104441 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.oraloncology.2019.104441" data-track-item_id="10.1016/j.oraloncology.2019.104441" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.oraloncology.2019.104441" aria-label="Article reference 29" data-doi="10.1016/j.oraloncology.2019.104441">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitV2ms7nJ" aria-label="CAS reference 29">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31689639" aria-label="PubMed reference 29">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6981283" aria-label="PubMed Central reference 29">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20sequencing%20and%20its%20applications%20in%20head%20and%20neck%20cancer&amp;journal=Oral%20Oncol&amp;doi=10.1016%2Fj.oraloncology.2019.104441&amp;volume=99&amp;publication_year=2019&amp;author=Qi%2CZ&amp;author=Barrett%2CT&amp;author=Parikh%2CAS&amp;author=Tirosh%2CI&amp;author=Puram%2CSV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">van Galen, P. et al. Single-cell RNA-Seq reveals AML hierarchies relevant to disease progression and immunity. <i>Cell</i> <b>176</b>, 1265–1281.e1224 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2019.01.031" data-track-item_id="10.1016/j.cell.2019.01.031" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.01.031" aria-label="Article reference 30" data-doi="10.1016/j.cell.2019.01.031">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30827681" aria-label="PubMed reference 30">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6515904" aria-label="PubMed Central reference 30">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20RNA-Seq%20reveals%20AML%20hierarchies%20relevant%20to%20disease%20progression%20and%20immunity&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.01.031&amp;volume=176&amp;pages=1265-1281.e1224&amp;publication_year=2019&amp;author=Galen%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Chevrier, S. et al. An immune atlas of clear cell renal cell carcinoma. <i>Cell</i> <b>169</b>, 736–749.e718 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2017.04.016" data-track-item_id="10.1016/j.cell.2017.04.016" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.04.016" aria-label="Article reference 31" data-doi="10.1016/j.cell.2017.04.016">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXnt1Cms7s%3D" aria-label="CAS reference 31">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28475899" aria-label="PubMed reference 31">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5422211" aria-label="PubMed Central reference 31">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20immune%20atlas%20of%20clear%20cell%20renal%20cell%20carcinoma&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.04.016&amp;volume=169&amp;pages=736-749.e718&amp;publication_year=2017&amp;author=Chevrier%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Savas, P. et al. Single-cell profiling of breast cancer T cells reveals a tissue-resident memory subset associated with improved prognosis. <i>Nat. Med.</i> <b>24</b>, 986–993 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-018-0078-7" data-track-item_id="10.1038/s41591-018-0078-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-018-0078-7" aria-label="Article reference 32" data-doi="10.1038/s41591-018-0078-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1WqsLjL" aria-label="CAS reference 32">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29942092" aria-label="PubMed reference 32">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20profiling%20of%20breast%20cancer%20T%20cells%20reveals%20a%20tissue-resident%20memory%20subset%20associated%20with%20improved%20prognosis&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-018-0078-7&amp;volume=24&amp;pages=986-993&amp;publication_year=2018&amp;author=Savas%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Karaayvaz, M. et al. Unravelling subclonal heterogeneity and aggressive disease states in TNBC through single-cell RNA-seq. <i>Nat. Commun.</i> <b>9</b>, 3588 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-018-06052-0" data-track-item_id="10.1038/s41467-018-06052-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-018-06052-0" aria-label="Article reference 33" data-doi="10.1038/s41467-018-06052-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30181541" aria-label="PubMed reference 33">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6123496" aria-label="PubMed Central reference 33">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Unravelling%20subclonal%20heterogeneity%20and%20aggressive%20disease%20states%20in%20TNBC%20through%20single-cell%20RNA-seq&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-06052-0&amp;volume=9&amp;publication_year=2018&amp;author=Karaayvaz%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Lambrechts, D. et al. Phenotype molding of stromal cells in the lung tumor microenvironment. <i>Nat. Med.</i> <b>24</b>, 1277–1289 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-018-0096-5" data-track-item_id="10.1038/s41591-018-0096-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-018-0096-5" aria-label="Article reference 34" data-doi="10.1038/s41591-018-0096-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjs7vM" aria-label="CAS reference 34">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29988129" aria-label="PubMed reference 34">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Phenotype%20molding%20of%20stromal%20cells%20in%20the%20lung%20tumor%20microenvironment&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-018-0096-5&amp;volume=24&amp;pages=1277-1289&amp;publication_year=2018&amp;author=Lambrechts%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Stoeckius, M. et al. Simultaneous epitope and transcriptome measurement in single cells. <i>Nat. Methods</i> <b>14</b>, 865–868 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmeth.4380" data-track-item_id="10.1038/nmeth.4380" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.4380" aria-label="Article reference 35" data-doi="10.1038/nmeth.4380">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1CkurzK" aria-label="CAS reference 35">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28759029" aria-label="PubMed reference 35">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5669064" aria-label="PubMed Central reference 35">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Simultaneous%20epitope%20and%20transcriptome%20measurement%20in%20single%20cells&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4380&amp;volume=14&amp;pages=865-868&amp;publication_year=2017&amp;author=Stoeckius%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Kang, H. M. et al. Multiplexed droplet single-cell RNA-sequencing using natural genetic variation. <i>Nat. Biotechnol.</i> <b>36</b>, 89–94 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nbt.4042" data-track-item_id="10.1038/nbt.4042" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.4042" aria-label="Article reference 36" data-doi="10.1038/nbt.4042">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFGmur3F" aria-label="CAS reference 36">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29227470" aria-label="PubMed reference 36">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%20droplet%20single-cell%20RNA-sequencing%20using%20natural%20genetic%20variation&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.4042&amp;volume=36&amp;pages=89-94&amp;publication_year=2018&amp;author=Kang%2CHM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Stoeckius, M. et al. Cell hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics. <i>Genome Biol</i>. <b>19</b>, 224 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-018-1603-1" data-track-item_id="10.1186/s13059-018-1603-1" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-018-1603-1" aria-label="Article reference 37" data-doi="10.1186/s13059-018-1603-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVKksbfJ" aria-label="CAS reference 37">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30567574" aria-label="PubMed reference 37">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6300015" aria-label="PubMed Central reference 37">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Cell%20hashing%20with%20barcoded%20antibodies%20enables%20multiplexing%20and%20doublet%20detection%20for%20single%20cell%20genomics&amp;journal=Genome%20Biol&amp;doi=10.1186%2Fs13059-018-1603-1&amp;volume=19&amp;publication_year=2018&amp;author=Stoeckius%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Bakken, T. E. et al. Single-nucleus and single-cell transcriptomes compared in matched cortical cell types. <i>PLoS ONE</i> <b>13</b>, e0209648 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0209648" data-track-item_id="10.1371/journal.pone.0209648" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0209648" aria-label="Article reference 38" data-doi="10.1371/journal.pone.0209648">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30586455" aria-label="PubMed reference 38">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6306246" aria-label="PubMed Central reference 38">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-nucleus%20and%20single-cell%20transcriptomes%20compared%20in%20matched%20cortical%20cell%20types&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0209648&amp;volume=13&amp;publication_year=2018&amp;author=Bakken%2CTE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Shekhar, K. et al. Comprehensive classification of retinal bipolar neurons by single-cell transcriptomics. <i>Cell</i> <b>166</b>, 1308–1323.e1330 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2016.07.054" data-track-item_id="10.1016/j.cell.2016.07.054" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2016.07.054" aria-label="Article reference 39" data-doi="10.1016/j.cell.2016.07.054">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVehu73M" aria-label="CAS reference 39">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27565351" aria-label="PubMed reference 39">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5003425" aria-label="PubMed Central reference 39">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20classification%20of%20retinal%20bipolar%20neurons%20by%20single-cell%20transcriptomics&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2016.07.054&amp;volume=166&amp;pages=1308-1323.e1330&amp;publication_year=2016&amp;author=Shekhar%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Wolf, F. A., Angerer, P. &amp; Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. <i>Genome Biol.</i> <b>19</b>, 15 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-017-1382-0" data-track-item_id="10.1186/s13059-017-1382-0" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-017-1382-0" aria-label="Article reference 40" data-doi="10.1186/s13059-017-1382-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29409532" aria-label="PubMed reference 40">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5802054" aria-label="PubMed Central reference 40">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=SCANPY%3A%20large-scale%20single-cell%20gene%20expression%20data%20analysis&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-017-1382-0&amp;volume=19&amp;publication_year=2018&amp;author=Wolf%2CFA&amp;author=Angerer%2CP&amp;author=Theis%2CFJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Macosko, E. Z. et al. Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets. <i>Cell</i> <b>161</b>, 1202–1214 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXpt1Sgt7o%3D" aria-label="CAS reference 41">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26000488" aria-label="PubMed reference 41">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4481139" aria-label="PubMed Central reference 41">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Highly%20parallel%20genome-wide%20expression%20profiling%20of%20individual%20cells%20using%20nanoliter%20droplets&amp;journal=Cell&amp;volume=161&amp;pages=1202-1214&amp;publication_year=2015&amp;author=Macosko%2CEZ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Benjamini, Y. &amp; Yosef, Hochberg Controlling the false discovery rate: a practical and powerful approach to multiple testing. <i>J. R. Stat. Soc. B</i> <b>57.1</b>, 289–300 (1995).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Controlling%20the%20false%20discovery%20rate%3A%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&amp;journal=J.%20R.%20Stat.%20Soc.%20B&amp;volume=57.1&amp;pages=289-300&amp;publication_year=1995&amp;author=Benjamini%2CY&amp;author=Yosef%2CHochberg">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">McInnes, L., Healy, J. &amp; Melville, J. UMAP: Uniform Manifold Approximation and Projection for dimension reduction. Preprint at <a href="https://arxiv.org/abs/1802.03426" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://arxiv.org/abs/1802.03426">https://arxiv.org/abs/1802.03426</a> (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Aran, D. et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. <i>Nat. Immunol.</i> <b>20</b>, 163–172 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41590-018-0276-y" data-track-item_id="10.1038/s41590-018-0276-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41590-018-0276-y" aria-label="Article reference 44" data-doi="10.1038/s41590-018-0276-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvFSgu7k%3D" aria-label="CAS reference 44">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30643263" aria-label="PubMed reference 44">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6340744" aria-label="PubMed Central reference 44">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Reference-based%20analysis%20of%20lung%20single-cell%20sequencing%20reveals%20a%20transitional%20profibrotic%20macrophage&amp;journal=Nat.%20Immunol.&amp;doi=10.1038%2Fs41590-018-0276-y&amp;volume=20&amp;pages=163-172&amp;publication_year=2019&amp;author=Aran%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Tickle, T. I., Georgescu, C., Brown, M. &amp; Haas, B. inferCNV of the Trinity CTAT Project (2019); <a href="https://github.com/broadinstitute/inferCNV" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://github.com/broadinstitute/inferCNV">https://github.com/broadinstitute/inferCNV</a></p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-020-0844-1?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank all patients and their families. We thank J. Rood for help with editing, A. Hupalowska for help with figure preparation, the clinical research teams who supplied the samples, E. Todres Gelfand, N. Straub, L. DelloStritto, K. Helvie and S. Periyasamy for project management, R. Levy for NSCLC patient identification, patient consent and protocol optimization, M. Patel, A. Lako and S. Rodig for pathology review, the scientific team at Leidos Biomedical Research, Frederick National Laboratory for Cancer Research, especially R. Agarwal and Y. Mori, the team at National Cancer Institute (NCI), especially S. Hughes, P. Oberdoerffer and D. Singer, and the Human Tumor Atlas Network for helpful discussions. This project has been funded in part with federal funds from the NCI, National Institutes of Health, task order no. HHSN261100039 under contract no. HHSN261201500003I. The content of this publication does not necessarily reflect the views or policies of the Department of Health and Human Services, nor does mention of trade names, commercial products or organizations imply endorsement by the US Government. This project was also funded in part by the ‘START: Standardization of Single-Cell and Single-Nucleus RNA-Seq Protocols for Tumors’ project from the Chan Zuckerberg Initiative and the Klarman Cell Observatory. R. Agarwal and Y. Mori from Leidos Biomedical Research and S. Hughes and P. Oberdoerffer from NCI were periodically updated about the progress of the project and the manuscript preparation; our funders did not play a role in the study design or manuscript preparation. A. Regev is an Investigator of the Howard Hughes Medical Institute. C.J.W. is a Scholar of the Leukemia and Lymphoma Society. J.K. was supported by an EMBO Long-Term Fellowship (ALTF 738-2017). S.G. was supported by the Kay Kendall Leukaemia Fund. B.I. was supported by NIH/NCI K08CA222663, NIH/NCI CA225088 and a Burroughs Wellcome Fund Career Award for Medical Scientists.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="nAff25"><p class="c-article-author-information__authors-list">Alexander M. Tsankov</p><p class="js-present-address">Present address: Icahn School of Medicine at Mount Sinai, New York, NY, USA</p></li><li class="c-article-author-information__item" id="nAff26"><p class="c-article-author-information__authors-list">Benjamin Izar</p><p class="js-present-address">Present address: Columbia Center for Translational Immunology and Division of Hematology and Oncology, Columbia University Medical Center, New York, NY, USA</p></li><li class="c-article-author-information__item" id="na1"><p>These authors contributed equally: Michal Slyper, Caroline B. M. Porter, Orr Ashenberg.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Klarman Cell Observatory, Broad Institute of Harvard and MIT, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Michal Slyper,&nbsp;Caroline B. M. Porter,&nbsp;Orr Ashenberg,&nbsp;Julia Waldman,&nbsp;Eugene Drokhlyansky,&nbsp;Christopher Smillie,&nbsp;Gabriela Smith-Rosario,&nbsp;Danielle Dionne,&nbsp;Judit Jané-Valbuena,&nbsp;Timothy L. Tickle,&nbsp;Alexander M. Tsankov,&nbsp;Livnat Jerby-Arnon,&nbsp;Johanna Klughammer,&nbsp;Yanay Rosen,&nbsp;Joshua Gould,&nbsp;Lan Nguyen,&nbsp;Matan Hofree,&nbsp;Orit Rozenblatt-Rosen&nbsp;&amp;&nbsp;Aviv Regev</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Broad Institute of Harvard and MIT, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Isaac Wakiro,&nbsp;Jingyi Wu,&nbsp;Sébastien Vigneau,&nbsp;Sara Napolitano,&nbsp;Mei-Ju Su,&nbsp;Satyen H. Gohil,&nbsp;Ofir Cohen,&nbsp;Bo Li,&nbsp;Catherine J. Wu,&nbsp;Benjamin Izar,&nbsp;Rizwan Haq,&nbsp;Mario L. Suvà,&nbsp;Elizabeth H. Stover,&nbsp;Natalie B. Collins,&nbsp;Nikhil Wagle&nbsp;&amp;&nbsp;Asaf Rotem</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Department of Medical Oncology, Dana-Farber Cancer Institute, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Isaac Wakiro,&nbsp;Jingyi Wu,&nbsp;Sébastien Vigneau,&nbsp;Sara Napolitano,&nbsp;Mei-Ju Su,&nbsp;Satyen H. Gohil,&nbsp;Ofir Cohen,&nbsp;Catherine J. Wu,&nbsp;Benjamin Izar,&nbsp;Rizwan Haq,&nbsp;F. Stephen Hodi,&nbsp;Charles H. Yoon,&nbsp;Elizabeth H. Stover,&nbsp;Ursula A. Matulonis,&nbsp;Nikhil Wagle,&nbsp;Bruce E. Johnson&nbsp;&amp;&nbsp;Asaf Rotem</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Center for Cancer Precision Medicine of Dana-Farber Cancer Institute, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Isaac Wakiro,&nbsp;Jingyi Wu,&nbsp;Sébastien Vigneau,&nbsp;Sara Napolitano,&nbsp;Mei-Ju Su,&nbsp;Ofir Cohen,&nbsp;Benjamin Izar,&nbsp;Nikhil Wagle,&nbsp;Bruce E. Johnson&nbsp;&amp;&nbsp;Asaf Rotem</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Department of Developmental Neurobiology, St Jude Children’s Research Hospital, Memphis, TN, USA</p><p class="c-article-author-affiliation__authors-list">Anand G. Patel,&nbsp;Asa Karlstrom,&nbsp;Suzanne J. Baker&nbsp;&amp;&nbsp;Michael A. Dyer</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Department of Oncology, St Jude Children’s Research Hospital, Memphis, TN, USA</p><p class="c-article-author-affiliation__authors-list">Anand G. Patel</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Simon Gritsch,&nbsp;Masashi Nomura&nbsp;&amp;&nbsp;Mario L. Suvà</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Center for Cancer Research, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Simon Gritsch,&nbsp;Masashi Nomura,&nbsp;Aaron N. Hata&nbsp;&amp;&nbsp;Mario L. Suvà</p></li><li id="Aff9"><p class="c-article-author-affiliation__address">Center for Regenerative Medicine, Massachusetts General Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Avinash Waghray</p></li><li id="Aff10"><p class="c-article-author-affiliation__address">Division of Thoracic Surgery, Brigham and Women’s Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Peter J. Tramontozzi&nbsp;&amp;&nbsp;Raphael Bueno</p></li><li id="Aff11"><p class="c-article-author-affiliation__address">Center for Immunology and Inflammatory Diseases, Division of Rheumatology, Allergy, and Immunology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Bo Li</p></li><li id="Aff12"><p class="c-article-author-affiliation__address">Department of Internal Medicine, Brigham and Women’s Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Catherine J. Wu&nbsp;&amp;&nbsp;Nikhil Wagle</p></li><li id="Aff13"><p class="c-article-author-affiliation__address">Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Catherine J. Wu&nbsp;&amp;&nbsp;Nikhil Wagle</p></li><li id="Aff14"><p class="c-article-author-affiliation__address">Laboratory for Systems Pharmacology, Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Benjamin Izar</p></li><li id="Aff15"><p class="c-article-author-affiliation__address">Center for Immunology and Virology, Dana-Farber Cancer Institute, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Benjamin Izar</p></li><li id="Aff16"><p class="c-article-author-affiliation__address">Ludwig Center for Cancer Research at Harvard, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Benjamin Izar</p></li><li id="Aff17"><p class="c-article-author-affiliation__address">Melanoma Disease Center, Dana-Farber Cancer Institute, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Benjamin Izar&nbsp;&amp;&nbsp;F. Stephen Hodi</p></li><li id="Aff18"><p class="c-article-author-affiliation__address">Department of Surgical Oncology, Brigham and Women’s Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Charles H. Yoon</p></li><li id="Aff19"><p class="c-article-author-affiliation__address">Department of Medicine, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Aaron N. Hata</p></li><li id="Aff20"><p class="c-article-author-affiliation__address">Department of Pathology, St Jude Children’s Research Hospital, Memphis, TN, USA</p><p class="c-article-author-affiliation__authors-list">Michael R. Clay</p></li><li id="Aff21"><p class="c-article-author-affiliation__address">Howard Hughes Medical Institute, Chevy Chase, MD, USA</p><p class="c-article-author-affiliation__authors-list">Michael A. Dyer&nbsp;&amp;&nbsp;Aviv Regev</p></li><li id="Aff22"><p class="c-article-author-affiliation__address">Department of Pediatric Oncology, Dana-Farber Cancer Institute, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Natalie B. Collins</p></li><li id="Aff23"><p class="c-article-author-affiliation__address">Division of Pediatric Hematology and Oncology, Boston Children’s Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Natalie B. Collins</p></li><li id="Aff24"><p class="c-article-author-affiliation__address">Koch Institute for Integrative Cancer Research, Department of Biology, Massachusetts Institute of Technology, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Aviv Regev</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Michal-Slyper-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Michal Slyper</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michal%20Slyper" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michal%20Slyper" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michal%20Slyper%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Caroline_B__M_-Porter-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Caroline B. M. Porter</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Caroline%20B.%20M.%20Porter" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Caroline%20B.%20M.%20Porter" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Caroline%20B.%20M.%20Porter%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Orr-Ashenberg-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Orr Ashenberg</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Orr%20Ashenberg" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Orr%20Ashenberg" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Orr%20Ashenberg%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Julia-Waldman-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Julia Waldman</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Julia%20Waldman" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Julia%20Waldman" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Julia%20Waldman%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Eugene-Drokhlyansky-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Eugene Drokhlyansky</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Eugene%20Drokhlyansky" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Eugene%20Drokhlyansky" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Eugene%20Drokhlyansky%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Isaac-Wakiro-Aff2-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Isaac Wakiro</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Isaac%20Wakiro" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Isaac%20Wakiro" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Isaac%20Wakiro%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Christopher-Smillie-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Christopher Smillie</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Christopher%20Smillie" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christopher%20Smillie" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christopher%20Smillie%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Gabriela-Smith_Rosario-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Gabriela Smith-Rosario</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Gabriela%20Smith-Rosario" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Gabriela%20Smith-Rosario" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Gabriela%20Smith-Rosario%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jingyi-Wu-Aff2-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Jingyi Wu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jingyi%20Wu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jingyi%20Wu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jingyi%20Wu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Danielle-Dionne-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Danielle Dionne</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Danielle%20Dionne" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Danielle%20Dionne" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Danielle%20Dionne%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-S_bastien-Vigneau-Aff2-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Sébastien Vigneau</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=S%C3%A9bastien%20Vigneau" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=S%C3%A9bastien%20Vigneau" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22S%C3%A9bastien%20Vigneau%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Judit-Jan__Valbuena-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Judit Jané-Valbuena</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Judit%20Jan%C3%A9-Valbuena" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Judit%20Jan%C3%A9-Valbuena" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Judit%20Jan%C3%A9-Valbuena%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Timothy_L_-Tickle-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Timothy L. Tickle</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Timothy%20L.%20Tickle" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Timothy%20L.%20Tickle" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Timothy%20L.%20Tickle%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sara-Napolitano-Aff2-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Sara Napolitano</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sara%20Napolitano" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sara%20Napolitano" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sara%20Napolitano%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Mei_Ju-Su-Aff2-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Mei-Ju Su</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Mei-Ju%20Su" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Mei-Ju%20Su" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Mei-Ju%20Su%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Anand_G_-Patel-Aff5-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Anand G. Patel</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Anand%20G.%20Patel" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anand%20G.%20Patel" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anand%20G.%20Patel%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Asa-Karlstrom-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Asa Karlstrom</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Asa%20Karlstrom" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Asa%20Karlstrom" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Asa%20Karlstrom%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Simon-Gritsch-Aff7-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Simon Gritsch</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Simon%20Gritsch" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Simon%20Gritsch" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Simon%20Gritsch%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Masashi-Nomura-Aff7-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Masashi Nomura</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Masashi%20Nomura" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Masashi%20Nomura" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Masashi%20Nomura%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Avinash-Waghray-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Avinash Waghray</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Avinash%20Waghray" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Avinash%20Waghray" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Avinash%20Waghray%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Satyen_H_-Gohil-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Satyen H. Gohil</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Satyen%20H.%20Gohil" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Satyen%20H.%20Gohil" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Satyen%20H.%20Gohil%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Alexander_M_-Tsankov-Aff1-Aff25"><span class="c-article-authors-search__title u-h3 js-search-name">Alexander M. Tsankov</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Alexander%20M.%20Tsankov" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Alexander%20M.%20Tsankov" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alexander%20M.%20Tsankov%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Livnat-Jerby_Arnon-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Livnat Jerby-Arnon</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Livnat%20Jerby-Arnon" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Livnat%20Jerby-Arnon" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Livnat%20Jerby-Arnon%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ofir-Cohen-Aff2-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Ofir Cohen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ofir%20Cohen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ofir%20Cohen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ofir%20Cohen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Johanna-Klughammer-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Johanna Klughammer</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Johanna%20Klughammer" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Johanna%20Klughammer" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Johanna%20Klughammer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Yanay-Rosen-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Yanay Rosen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Yanay%20Rosen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Yanay%20Rosen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Yanay%20Rosen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Joshua-Gould-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Joshua Gould</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Joshua%20Gould" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Joshua%20Gould" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Joshua%20Gould%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Lan-Nguyen-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Lan Nguyen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Lan%20Nguyen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Lan%20Nguyen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Lan%20Nguyen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Matan-Hofree-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Matan Hofree</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Matan%20Hofree" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Matan%20Hofree" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Matan%20Hofree%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Peter_J_-Tramontozzi-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Peter J. Tramontozzi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Peter%20J.%20Tramontozzi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Peter%20J.%20Tramontozzi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Peter%20J.%20Tramontozzi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Bo-Li-Aff2-Aff11"><span class="c-article-authors-search__title u-h3 js-search-name">Bo Li</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Bo%20Li" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Bo%20Li" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Bo%20Li%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Catherine_J_-Wu-Aff2-Aff3-Aff12-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Catherine J. Wu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Catherine%20J.%20Wu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Catherine%20J.%20Wu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Catherine%20J.%20Wu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Benjamin-Izar-Aff2-Aff3-Aff4-Aff14-Aff15-Aff16-Aff17-Aff26"><span class="c-article-authors-search__title u-h3 js-search-name">Benjamin Izar</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Benjamin%20Izar" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Benjamin%20Izar" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Benjamin%20Izar%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Rizwan-Haq-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Rizwan Haq</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Rizwan%20Haq" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Rizwan%20Haq" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Rizwan%20Haq%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-F__Stephen-Hodi-Aff3-Aff17"><span class="c-article-authors-search__title u-h3 js-search-name">F. Stephen Hodi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=F.%20Stephen%20Hodi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=F.%20Stephen%20Hodi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22F.%20Stephen%20Hodi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Charles_H_-Yoon-Aff3-Aff18"><span class="c-article-authors-search__title u-h3 js-search-name">Charles H. Yoon</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Charles%20H.%20Yoon" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Charles%20H.%20Yoon" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Charles%20H.%20Yoon%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Aaron_N_-Hata-Aff8-Aff19"><span class="c-article-authors-search__title u-h3 js-search-name">Aaron N. Hata</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Aaron%20N.%20Hata" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Aaron%20N.%20Hata" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Aaron%20N.%20Hata%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Suzanne_J_-Baker-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Suzanne J. Baker</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Suzanne%20J.%20Baker" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Suzanne%20J.%20Baker" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Suzanne%20J.%20Baker%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Mario_L_-Suv_-Aff2-Aff7-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Mario L. Suvà</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Mario%20L.%20Suv%C3%A0" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Mario%20L.%20Suv%C3%A0" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Mario%20L.%20Suv%C3%A0%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Raphael-Bueno-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Raphael Bueno</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Raphael%20Bueno" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Raphael%20Bueno" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Raphael%20Bueno%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Elizabeth_H_-Stover-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Elizabeth H. Stover</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Elizabeth%20H.%20Stover" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Elizabeth%20H.%20Stover" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Elizabeth%20H.%20Stover%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Michael_R_-Clay-Aff20"><span class="c-article-authors-search__title u-h3 js-search-name">Michael R. Clay</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michael%20R.%20Clay" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michael%20R.%20Clay" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20R.%20Clay%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Michael_A_-Dyer-Aff5-Aff21"><span class="c-article-authors-search__title u-h3 js-search-name">Michael A. Dyer</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michael%20A.%20Dyer" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michael%20A.%20Dyer" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20A.%20Dyer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Natalie_B_-Collins-Aff2-Aff22-Aff23"><span class="c-article-authors-search__title u-h3 js-search-name">Natalie B. Collins</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Natalie%20B.%20Collins" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Natalie%20B.%20Collins" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Natalie%20B.%20Collins%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ursula_A_-Matulonis-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Ursula A. Matulonis</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ursula%20A.%20Matulonis" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ursula%20A.%20Matulonis" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ursula%20A.%20Matulonis%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Nikhil-Wagle-Aff2-Aff3-Aff4-Aff12-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Nikhil Wagle</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Nikhil%20Wagle" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nikhil%20Wagle" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nikhil%20Wagle%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Bruce_E_-Johnson-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Bruce E. Johnson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Bruce%20E.%20Johnson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Bruce%20E.%20Johnson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Bruce%20E.%20Johnson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Asaf-Rotem-Aff2-Aff3-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Asaf Rotem</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Asaf%20Rotem" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Asaf%20Rotem" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Asaf%20Rotem%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Orit-Rozenblatt_Rosen-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Orit Rozenblatt-Rosen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Orit%20Rozenblatt-Rosen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Orit%20Rozenblatt-Rosen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Orit%20Rozenblatt-Rosen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Aviv-Regev-Aff1-Aff21-Aff24"><span class="c-article-authors-search__title u-h3 js-search-name">Aviv Regev</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Aviv%20Regev" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Aviv%20Regev" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Aviv%20Regev%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>M.S., C.B.M.P., O.A., O.R.-R. and A. Regev conceived and led the study. M.S. designed protocols and carried out experiments together with J. Waldman, I.W., S.V., J. Wu, M.-J.S., S.N., D.D., M.N., A.G.P., S.G. and L.N. C.B.M.P. and O.A. designed and performed computational analysis, with input and assistance from A.M.T., L.J.-A., O.C., J.K., M.H., C.S., G.S.-R. and T.L.T. G.S.-R. contributed CNA inference analyses. I.W. and S.V. performed FACS analysis and scRNA-Seq of ovarian ascites and NSCLC CD45 depletion. P.J.T. carried out NSCLC specimen collection and protocol optimization. E.D. and C.S. provided protocols for snRNA-Seq. A.W. provided input for the scRNA-Seq PDEC protocol. A.G.P. provided input and performed the neuroblastoma papain protocol. S.H.G. provided input for CLL. S.G. provided and carried out the GBM protocol. J.J.-V. managed the study and tissue acquisition for all samples except neuroblastoma O-PDX, which A.K. managed. B.L., Y.R. and J.G. contributed Cumulus expertise. A. Rotem, C.J.W., B.I., R.H., F.S.H., C.H.Y., A.N.H., S.J.B., M.L.S., R.B., E.H.S., M.R.C., M.A.D., N.B.C., U.A.M., N.W. and B.E.J. supervised research, provided tissues and shared clinical insights. M.S., C.B.M.P., O.A., O.R.-R. and A. Regev wrote the manuscript, with input from all authors.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:orit@broadinstitute.org">Orit Rozenblatt-Rosen</a> or <a id="corresp-c2" href="mailto:aregev@broadinstitute.org">Aviv Regev</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar18">Competing interests</h3>
                <p>A. Regev is a founder of and equity holder in Celsius Therapeutics, an equity holder in Imunitas, and a SAB member of Syros Pharmaceuticals, Thermo Fisher Scientific, Asimov and NeoGene Therapeutics. M.S., O.A., E.D., O.R.-R. and A. Regev are co-inventors on patent applications filed by the Broad Institute for inventions relating to work in this manuscript, such as in PCT/US2018/060860 and US provisional application no. 62/745,259. C.J.W. is a founder and member of the scientific advisory board of Neon Therapeutics and receives research funding from Pharmacyclics. A. Rotem is a consultant and equity holder in Celsius Therapeutics. F.S.H. reports grants, personal fees from Bristol-Myers Squibb and Novartis and personal fees from Merck, EMD Serono, Takeda, Surface, Genentech/Roche, Compass Therapeutics, Apricity, Bayer, Aduro, Sanofi, Pfizer, Pionyr, Verastem, Torque and Rheos. In addition, F.S.H. has a patent ‘Methods for Treating MICA-Related Disorders’ (#20100111973) with royalties paid, a patent ‘Tumor Antigens and Uses Thereof’ (#7250291) issued, a patent ‘Angiopoiten-2 Biomarkers Predictive of Anti-immune Checkpoint Response’ (#20170248603) pending, a patent ‘Compositions and Methods for Identification, Assessment, Prevention and Treatment of Melanoma using PD-L1 Isoforms’ (#20160340407) pending, a patent ‘Therapeutic Peptides’ (#20160046716) pending, a patent ‘Therapeutic Peptides’ (#20140004112) pending, a patent ‘Therapeutic Peptides’ (#20170022275) pending, a patent ‘Therapeutic Peptides’ (#20170008962) pending, a patent ‘Therapeutic Peptides’ (patent no. 9402905) issued and a patent Methods of using ‘Pembrolizumab and Trebananib’ pending. R.H. has received research support from Novartis and Bristol-Myers and is a consultant for Tango Therapeutics. R.B. has received research support from Roche, Genentech, Merck, Siemens, Verastem, Gritstone, Epizyme, Medgenome and HTG and has equity in Navigation Sciences. A.N.H. has received research support from Novartis, Amgen, Pfizer, Roche/Genentech and Relay Therapeutics. B.I. is a paid consultant for Merck. U.A.M. received consulting fees from Novartis and Merck.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Peer review information</b> Joao Monteiro was the primary editor on this article and managed its editorial process and peer review in collaboration with the rest of the editorial team.</p><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec35-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec35">Extended data</h2><div class="c-article-section__content" id="Sec35-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 scrna-seq protocol comparison" href="/articles/s41591-020-0844-1/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig7_ESM.jpg">Extended Data Fig. 1 scRNA-Seq protocol comparison for a single NSCLC sample.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a)</b> Sample processing and QC overview. For each protocol, shown are the number of cells passing QC, and the number of sequencing reads and sequencing saturation across all cells. The remaining metrics are reported for cells passing QC: the median number of reads per cell, median number of UMIs per cell, median number of genes per cell, median fraction of UMIs mapping to mitochondrial genes, fraction of cell barcodes called as empty droplets and fraction of cell barcodes called as doublets. <b>(b)</b> Read mapping QCs. The percent of bases in the sequencing reads (<i>y</i> axis) mapping to the genome, transcriptome and intergenic regions (<i>x</i> axis) across the three protocols (colored bars). <b>(c)</b> Cell type assignment. UMAP embedding of single cell profiles from each protocol colored by assigned cell type signature. <b>(d)</b> Inferred CNA profiles. Chromosomal amplification (red) and deletion (blue) inferred in each chromosomal position (columns) across the single cells (rows) from the NSCLC-C4 (left) and LE (right) protocols. Top: reference cells not expected to contain CNA in this cancer type. Bottom: cells tested for CNA relative to the reference cells. Color bar: assigned cell type signature for each cell. <b>(e)</b> Ambient RNA estimates. Estimates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Young, M. D. &amp; Behjati, S. SoupX removes ambient RNA contamination from droplet based single cell RNA sequencing data. Preprint at bioRxiv 
                  https://www.biorxiv.org/content/10.1101/303727v1
                  
                 (2018)." href="/articles/s41591-020-0844-1#ref-CR18" id="ref-link-section-d74065049e3338">18</a></sup> of the fraction of RNA in each cell type derived from ambient RNA contamination (y axis), with cell types ordered by their mean number of UMIs/cell (x axis). Red line: global average of contamination fraction; Green line: LOWESS (locally weighted scatterplot smoothing) smoothed estimate of the contamination fraction within each cell type, along with the associated binomial 95% confidence interval (Clopper–Pearson interval). <i>n</i> = 1 sample per protocol and number of cells (<i>k</i>) is indicated in (a).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 cell type specific qc metrics" href="/articles/s41591-020-0844-1/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig8_ESM.jpg">Extended Data Fig. 2 Cell type specific QC metrics for scRNA-Seq protocol comparison in a single NSCLC sample.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Cell type specific QCs for NSCLC14. Distribution (median and first and third quartiles) of the number of reads per cell, number of UMIs per cell, number of genes per cell and fraction of UMIs mapping to mitochondrial genes in each cell (<i>y</i> axes) in each of the three protocols (<i>x</i> axis), for cells passing QC from each cell type (rows). <i>n</i> = 1 sample per protocol. Number of B cells (<i>k</i>) from NSCLC-C4, PDEC and LE, respectively, is: 100, 121, 78; endothelial cells: 0, 920, 1,078; epithelial cells: 1,284, 641, 260; fibroblasts: 0, 1,476, 1,403; macrophages: 3,306, 911, 727; mast cells: 0, 119, 77; T cells: 449, 723, 722.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 scrna-seq protocol comparison" href="/articles/s41591-020-0844-1/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig9_ESM.jpg">Extended Data Fig. 3 scRNA-Seq protocol comparison for NSCLC following read down-sampling.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Shown are analyses for NSCLC14 (as in Extended Data Figs. 1 and 2), but after the total number of sequencing reads within each sample was down-sampled to match the protocol with the fewest total sequencing reads. <b>(a)</b> Sample processing and QC overview. For each protocol, shown are the number of cells passing QC. The remaining metrics are reported for those cells passing QC: median number of UMIs per cell, median number of genes per cell, median fraction of UMIs mapping to mitochondrial genes in each cell, fraction of cell barcodes called as empty droplets and fraction of cell barcodes called as doublets. <b>(b, c)</b> Overall and cell types specific QCs. Distribution (median and first and third quartiles) of the number of UMIs per cell, number of genes per cell and fraction of gene expression per cell from mitochondrial genes (<i>y</i> axes) in each of the three protocols (<i>x</i> axis), for all cells passing QC (b) and for cells from each cell type (c, rows). <b>(d,e)</b> Relation of empty droplets and doublets to cell types. UMAP embedding and fraction (horizontal bar) of single cell (gray), empty droplet (red, d) and doublet (red, e) profiles for each protocol <b>(f)</b> Cell type assignment. UMAP embedding of single cell profiles from each protocol colored by assigned cell type signature. <i>n</i> = 1 sample per protocol and number of cells (<i>k</i>) is indicated in (a). Number of B cells (<i>k</i>) from NSCLC-C4, PDEC and LE, respectively, is: 114, 157, 78; endothelial cells: 0, 879, 1,078; epithelial cells: 1,283, 644, 260; fibroblasts: 0, 1,439, 1,403; macrophages: 3,278, 853, 727; mast cells: 0, 106, 77; T cells: 432, 663, 722.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 cd45+ depletion protocol enri" href="/articles/s41591-020-0844-1/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig10_ESM.jpg">Extended Data Fig. 4 CD45<sup>+</sup> depletion protocol enriches for non-immune cells in freshly processed NSCLC and ovarian ascites.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a, b)</b> QCs. Distribution (median and first and third quartiles) of the number of reads per cell, number of UMIs per cell, number of genes per cell and fraction of gene expression per cell from mitochondrial genes (y axes) for all cells passing QC from NSCLC (a) before and after CD45<sup>+</sup> cell depletion, and for ovarian ascites (<i>k</i> = 2,998 and 10,716 cells, respectively) or (b) after CD45<sup>+</sup> cell depletion (2,359 cells) (x axis). <i>n</i> = 1 sample per protocol. <b>(c)</b> CD45<sup>+</sup> cell depletion estimates in ovarian cancer ascites by FACS. Flow-cytometry comparison of single cells isolated without (top) or with (bottom) depletion of CD45<sup>+</sup> cells. Cells were gated by FSC and SSC (first column), doublets removed using FSC-A and FSC-H (second column), 7-AAD gating of dead cells to identify live cells (third column), the distribution of immune and non-immune cells quantified using a CD45 antibody (fourth column) and the distribution of EPCAM<sup>+</sup> cells quantified using an EPCAM antibody (fifth column). (Efficient removal of CD45<sup>+</sup> cells from ovarian cancer ascites was also demonstrated with an independent sample from a different patient (data not shown).) Number of cells without and with depletion, respectively are: 10,000, 10,000 (1<sup>st</sup> column), 3,468, 2,256 (2<sup>nd</sup> column), 3,467, 2,251 (3<sup>rd</sup> column), 2,936, 2,174 (4<sup>th</sup> and 5<sup>th</sup> columns).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 snrna-seq protocol comparison" href="/articles/s41591-020-0844-1/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig11_ESM.jpg">Extended Data Fig. 5 snRNA-Seq protocol comparison in a single neuroblastoma sample.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a)</b> Sample processing and QC overview. For each protocol, shown are the number of nuclei passing QC, number of sequencing reads and sequencing saturation across all nuclei. The remaining metrics are reported for those nuclei passing QC: median number of reads per nucleus, median number of UMIs per nucleus, median number of genes per nucleus, median fraction of UMIs mapping to mitochondrial genes in each nucleus and fraction of nucleus barcodes called as doublets. <b>(b)</b> Read mapping QCs. The percent of bases in the sequencing reads (<i>y</i> axis) mapping to the genome, transcriptome and intergenic regions (<i>x</i> axis) across the four protocols (colored bars). <b>(c)</b> Cell type assignment. UMAP embedding of single nucleus profiles from each protocol colored by assigned cell type signature. <b>(d)</b> Inferred CNA profiles. Chromosomal amplification (red) and deletion (blue) inferred in each chromosomal position (columns) across the single nuclei (rows). Top: reference nuclei not expected to contain CNA in this cancer type. Bottom: nuclei tested for CNA relative to the reference nuclei. Color bar: assigned cell type signature for each nucleus. <i>n</i> = 1 sample per protocol and number of nuclei (<i>k</i>) is indicated in (a).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 cell type specific qc metrics" href="/articles/s41591-020-0844-1/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig12_ESM.jpg">Extended Data Fig. 6 Cell type specific QC metrics for snRNA-Seq protocol comparison in a single neuroblastoma sample.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Cell type specific QCs for HTAPP-244-SMP-451. Distribution (median and first and third quartiles) of the number of reads per nucleus, number of UMIs per nucleus, number of genes per nucleus and fraction of UMIs mapping to mitochondrial genes in each nucleus (<i>y</i> axes) in each of the four protocols (<i>x</i> axis), for nuclei passing QC from each cell type (rows). <i>n</i> = 1 sample per protocol. Number of endothelial nuclei (<i>k</i>) from EZ, CST, NST and TST, respectively, is: 69, 32, 91, 95; erythrocyte: 0, 18, 0, 15; T cell: 157, 171, 229, 337; neuroendocrine: 7,379, 5,728, 6,790, 6,477; neural crest: 18, 27, 50, 67; macrophage: 119, 107, 189, 230; fibroblast: 138, 74, 182, 194; zona glomerulosa: 16, 0, 0, 0.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 snrna-seq protocol comparison" href="/articles/s41591-020-0844-1/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig13_ESM.jpg">Extended Data Fig. 7 snRNA-Seq protocol comparison in a single neuroblastoma sample following read down-sampling.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Shown are analyses for NB HTAPP-244-SMP-451 (as in Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig11">5</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0844-1#Fig12">6</a>), but after the total number of sequencing reads within each sample was down-sampled to match the protocol with the fewest total sequencing reads. <b>(a)</b> Sample processing and QC overview. For each protocol, shown are the number of nuclei passing QC. The remaining metrics are reported for those nuclei passing QC: median number of UMIs per nucleus, median number of genes per nucleus, median fraction of UMIs mapping to mitochondrial genes in each nucleus and fraction of nucleus barcodes called as doublets. <b>(b,c)</b> Overall and cell types specific QCs. Distribution (median and first and third quartiles) of the number of UMIs per nucleus, number of genes per nucleus and fraction of UMIs mapping to mitochondrial genes in each nucleus (<i>y</i> axes) in each of the four protocols (<i>x</i> axis), for all nuclei passing QC (b) and for nuclei from each cell type (<b>c</b>, rows). <b>(d)</b> Relation of doublets to cell types. UMAP embedding and fraction (horizontal bar) of single nucleus (gray) and doublet (red) profiles for each protocol. <b>(e)</b> Cell type assignment. UMAP embedding of single nucleus profiles from each protocol colored by assigned cell type signature. <i>n</i> = 1 sample per protocol and number of nuclei (<i>k</i>) is indicated in (a). Number of endothelial nuclei (<i>k</i>) from EZ, CST, NST, TST is: 53, 31, 95, 98; erythrocyte: 0, 0, 0, 23; T cell: 146, 177, 230, 345; neuroendocrine: 7,407, 5,726, 6,776, 6,454; neural crest: 14, 27, 50, 69; macrophage: 123, 104, 196, 240; fibroblast: 111, 81, 177, 186; zona glomerulosa: 21, 0, 0, 0.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 inferred cna profiles from sn" href="/articles/s41591-020-0844-1/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig14_ESM.jpg">Extended Data Fig. 8 Inferred CNA profiles from snRNA-Seq in diverse tumors.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Chromosomal amplification (red) and deletion (blue) inferred in each chromosomal position (columns) across the single nuclei (rows) from three MBC samples (<b>a–f</b>), one ovarian cancer sample (<b>g-i</b>) and one sarcoma sample (<b>j, k</b>). Top: reference nuclei not expected to contain CNA in this cancer type. Bottom: nuclei tested for CNA relative to the reference nuclei. Color bar: assigned cell type signature for each nucleus. <i>n</i> = 1 sample per protocol and number of nuclei (<i>k</i>) per sample: MBC HTAPP-963-SMP-4741—9,857 (CST), 7,260 (TST); MBC HTAPP-394-SMP-1561—6,948 (CST), 8,058 (NST); MBC HTAPP-589-SMP-2851—7,858 (CST), 8,373 (TST); ovarian HTAPP-316-SMP-991—9,026 (CST), 5,970 (NST), 10,493 (TST); sarcoma HTAPP-951-SMP-4652—7,858 (CST), 4,458 (TST).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig15"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 snrna-seq protocol comparison" href="/articles/s41591-020-0844-1/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig15_ESM.jpg">Extended Data Fig. 9 snRNA-Seq protocol comparison of V2 and V3 chemistry from 10x Genomics on a resection of sarcoma.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>(a)</b> Sample processing and QC overview. For each protocol, shown are the number of nuclei passing QC, after the total number of sequencing reads from the V3 protocol data was down-sampled to match the number of reads in the V2 data. The remaining metrics are reported for those nuclei passing QC: median number of UMIs per nucleus, median number of genes per nucleus, median fraction of UMIs mapping to mitochondrial genes in each nucleus and fraction of nucleus barcodes called as doublets. <b>(b)</b> Overall QCs. Distribution (median and first and third quartiles) of number of UMIs per nucleus, number of genes per nucleus and fraction of UMIs mapping to mitochondrial genes in each nucleus (y axes) for all nuclei passing QC. <i>n</i> = 1 sample per chemistry type and number of nuclei (<i>k</i>) is indicated in (a).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig16"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 comparison of scrna-seq and " href="/articles/s41591-020-0844-1/figures/16" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig16_ESM.jpg">Extended Data Fig. 10 Comparison of scRNA-Seq and snRNA-Seq from the same tumor sample.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>(<b>a–g</b>) CLL. UMAP embedding of scRNA-Seq and snRNA-Seq profiles of the same CLL sample combined by CCA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies and species. Nat. Biotechnol. 36, 411–420 (2018)." href="/articles/s41591-020-0844-1#ref-CR24" id="ref-link-section-d74065049e3729">24</a></sup> (<b>Methods</b>) showing profiles (dots) from both (a), scRNA-Seq (b) and snRNA-Seq (c), colored by assigned cell type signatures. (d) Proportion of cells from each subset in the two protocols. <i>k</i>: number of cells or nuclei passing QC. (e-f) Same UMAP embedding as in (a), colored by cells or nuclei (e) or unsupervised clustering (f). (g) Fraction of cells and nuclei in each cluster. <i>n</i> = 1 sample per protocol and number of cells and nuclei is indicated in (d). (<b>h–n</b>) O-PDX neuroblastoma. As in (a-g) for an O-PDX neuroblastoma sample. <i>n</i> = 1 sample per protocol and number of cells and nuclei is indicated in (k). (<b>o-r</b>) Dissociation signatures are more prominent in cells than in nuclei from the same tumors. Left and middle: UMAP embedding of scRNA-Seq (left) and snRNA-Seq (middle) profiles (dots) of the same tumor combined by CCA<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies and species. Nat. Biotechnol. 36, 411–420 (2018)." href="/articles/s41591-020-0844-1#ref-CR24" id="ref-link-section-d74065049e3752">24</a></sup> and colored by the score of a dissociation signature (color bar). Right: Distribution of dissociation signature score (y axis; median and first and third quartiles) in cells (green) and nuclei (orange). (o) neuroblastoma (3,449 cells, 7,810 nuclei), (p) MBC (5,163 cells, 7,260 nuclei), (q) CLL (2,562 cells, 2,339 nuclei), (r) O-PDX neuroblastoma (3,495 cells, 4,946 nuclei).</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec36-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec36">Supplementary information</h2><div class="c-article-section__content" id="Sec36-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_MOESM1_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary table 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_MOESM2_ESM.xlsx" data-supp-info-image="">Supplementary Table 1</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Table 1</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=A%20single-cell%20and%20single-nucleus%20RNA-Seq%20toolbox%20for%20fresh%20and%20frozen%20human%20tumors&amp;author=Michal%20Slyper%20et%20al&amp;contentID=10.1038%2Fs41591-020-0844-1&amp;copyright=The%20Author%28s%29&amp;publication=1078-8956&amp;publicationDate=2020-05-11&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41591-020-0844-1" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41591-020-0844-1" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Slyper, M., Porter, C.B.M., Ashenberg, O. <i>et al.</i> A single-cell and single-nucleus RNA-Seq toolbox for fresh and frozen human tumors.
                    <i>Nat Med</i> <b>26</b>, 792–802 (2020). https://doi.org/10.1038/s41591-020-0844-1</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-020-0844-1?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2019-08-13">13 August 2019</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-03-20">20 March 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-05-11">11 May 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-05-01">01 May 2020</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41591-020-0844-1</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-block"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/biological-techniques" data-track="click" data-track-action="view subject" data-track-label="link">Biological techniques</a></li><li class="c-article-subject-list__subject"><a href="/subjects/cancer" data-track="click" data-track-action="view subject" data-track-label="link">Cancer</a></li><li class="c-article-subject-list__subject"><a href="/subjects/computational-biology-and-bioinformatics" data-track="click" data-track-action="view subject" data-track-label="link">Computational biology and bioinformatics</a></li><li class="c-article-subject-list__subject"><a href="/subjects/gene-expression-analysis" data-track="click" data-track-action="view subject" data-track-label="link">Gene expression analysis</a></li><li class="c-article-subject-list__subject"><a href="/subjects/genomic-analysis" data-track="click" data-track-action="view subject" data-track-label="link">Genomic analysis</a></li>
        </ul>
    </div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section" data-test="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Dissecting endothelial cell heterogeneity with new tools" href="https://doi.org/10.1186/s13619-025-00223-3">
                                        Dissecting endothelial cell heterogeneity with new tools
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Jing Zhong</li><li>Rong-rong Gao</li><li>Qi Chen</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Cell Regeneration</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Cell-type-specific subtyping of epigenomes improves prognostic stratification of cancer" href="https://doi.org/10.1186/s13073-025-01453-5">
                                        Cell-type-specific subtyping of epigenomes improves prognostic stratification of cancer
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Qi Luo</li><li>Andrew E. Teschendorff</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Genome Medicine</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Transcriptomic characterization of transitioning cell types in the skin of Atlantic salmon" href="https://doi.org/10.1186/s12915-025-02196-w">
                                        Transcriptomic characterization of transitioning cell types in the skin of Atlantic salmon
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>R. Ruiz Daniels</li><li>S. J. Salisbury</li><li>D Robledo</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>BMC Biology</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Exploratory analysis of a Novel RACK1 mutation and its potential role in epileptic seizures via Microglia activation" href="https://doi.org/10.1186/s12974-025-03350-5">
                                        Exploratory analysis of a Novel RACK1 mutation and its potential role in epileptic seizures via Microglia activation
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Sai Zhang</li><li>Zhaofei Dong</li><li>Tao Zeng</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Journal of Neuroinflammation</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:SlideCNA: spatial copy number alteration detection from Slide-seq-like spatial transcriptomics data" href="https://doi.org/10.1186/s13059-025-03573-y">
                                        SlideCNA: spatial copy number alteration detection from Slide-seq-like spatial transcriptomics data
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Diane Zhang</li><li>Åsa Segerstolpe</li><li>Johanna Klughammer</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Genome Biology</i> (2025)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-020-0844-1.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                
            
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
            <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" role="tabpanel"><div class="c-reading-companion__scroll-pane" style="max-height: none;"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Main">Main</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec12" class="c-reading-companion__section-item"><a href="#Sec12" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec13" class="c-reading-companion__section-item"><a href="#Sec13" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-data-availability" class="c-reading-companion__section-item"><a href="#data-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Data availability">Data availability</a></li><li id="rc-sec-code-availability" class="c-reading-companion__section-item"><a href="#code-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Code availability">Code availability</a></li><li id="rc-sec-change-history" class="c-reading-companion__section-item"><a href="#change-history" data-track="click" data-track-action="section anchor" data-track-label="link:Change history">Change history</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec35" class="c-reading-companion__section-item"><a href="#Sec35" data-track="click" data-track-action="section anchor" data-track-label="link:Extended data">Extended data</a></li><li id="rc-sec-Sec36" class="c-reading-companion__section-item"><a href="#Sec36" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by">This article is cited by</a></li></ul></div>
                <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="" style="display: none !important;">
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-test="right-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41591-020-0844-1;doi=10.1038/s41591-020-0844-1;subjmeta=114,1647,2017,2217,631,67;kwrd=Biological+techniques,Cancer,Computational+biology+and+bioinformatics,Gene+expression+analysis,Genomic+analysis">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=-1029166426&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-020-0844-1%26doi%3D10.1038/s41591-020-0844-1%26subjmeta%3D114,1647,2017,2217,631,67%26kwrd%3DBiological+techniques,Cancer,Computational+biology+and+bioinformatics,Gene+expression+analysis,Genomic+analysis">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=-1029166426&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-020-0844-1%26doi%3D10.1038/s41591-020-0844-1%26subjmeta%3D114,1647,2017,2217,631,67%26kwrd%3DBiological+techniques,Cancer,Computational+biology+and+bioinformatics,Gene+expression+analysis,Genomic+analysis"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Fig. 1: Study and toolbox overview.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig1_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig1_HTML.png" alt="figure 1" aria-describedby="rc-Fig1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Fig. 2: Fresh tumor processing and protocol selection for scRNA-Seq.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig2_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig2_HTML.png" alt="figure 2" aria-describedby="rc-Fig2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Fig. 3: scRNA-Seq protocol comparison across tumor types.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig3_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig3_HTML.png" alt="figure 3" aria-describedby="rc-Fig3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Fig. 4: Frozen tumor processing and protocol selection for snRNA-Seq.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig4_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig4_HTML.png" alt="figure 4" aria-describedby="rc-Fig4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Fig. 5: snRNA-Seq protocol comparison across tumor types.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig5_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig5_HTML.png" alt="figure 5" aria-describedby="rc-Fig5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig6">Fig. 6: scRNA-Seq and snRNA-Seq recover comparable cells in different proportions.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig6_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig6_HTML.png" alt="figure 6" aria-describedby="rc-Fig6"></picture><p class="c-reading-companion__figure-links"><a href="#Fig6" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/6" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig7">Extended Data Fig. 1 scRNA-Seq protocol comparison for a single NSCLC sample.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig7_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig7_ESM.jpg" alt="extended data figure 7" aria-describedby="rc-Fig7"></picture><p class="c-reading-companion__figure-links"><a href="#Fig7" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/7" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig8">Extended Data Fig. 2 Cell type specific QC metrics for scRNA-Seq protocol comparison in a single NSCLC sample.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig8_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig8_ESM.jpg" alt="extended data figure 8" aria-describedby="rc-Fig8"></picture><p class="c-reading-companion__figure-links"><a href="#Fig8" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/8" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig9">Extended Data Fig. 3 scRNA-Seq protocol comparison for NSCLC following read down-sampling.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig9_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig9_ESM.jpg" alt="extended data figure 9" aria-describedby="rc-Fig9"></picture><p class="c-reading-companion__figure-links"><a href="#Fig9" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/9" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig10">Extended Data Fig. 4 CD45<sup>+</sup> depletion protocol enriches for non-immune cells in freshly processed NSCLC and ovarian ascites.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig10_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig10_ESM.jpg" alt="extended data figure 10" aria-describedby="rc-Fig10"></picture><p class="c-reading-companion__figure-links"><a href="#Fig10" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/10" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig11">Extended Data Fig. 5 snRNA-Seq protocol comparison in a single neuroblastoma sample.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig11_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig11_ESM.jpg" alt="extended data figure 11" aria-describedby="rc-Fig11"></picture><p class="c-reading-companion__figure-links"><a href="#Fig11" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/11" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig12">Extended Data Fig. 6 Cell type specific QC metrics for snRNA-Seq protocol comparison in a single neuroblastoma sample.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig12_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig12_ESM.jpg" alt="extended data figure 12" aria-describedby="rc-Fig12"></picture><p class="c-reading-companion__figure-links"><a href="#Fig12" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/12" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig13">Extended Data Fig. 7 snRNA-Seq protocol comparison in a single neuroblastoma sample following read down-sampling.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig13_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig13_ESM.jpg" alt="extended data figure 13" aria-describedby="rc-Fig13"></picture><p class="c-reading-companion__figure-links"><a href="#Fig13" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/13" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig14">Extended Data Fig. 8 Inferred CNA profiles from snRNA-Seq in diverse tumors.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig14_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig14_ESM.jpg" alt="extended data figure 14" aria-describedby="rc-Fig14"></picture><p class="c-reading-companion__figure-links"><a href="#Fig14" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/14" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig15">Extended Data Fig. 9 snRNA-Seq protocol comparison of V2 and V3 chemistry from 10x Genomics on a resection of sarcoma.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig15_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig15_ESM.jpg" alt="extended data figure 15" aria-describedby="rc-Fig15"></picture><p class="c-reading-companion__figure-links"><a href="#Fig15" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/15" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig16">Extended Data Fig. 10 Comparison of scRNA-Seq and snRNA-Seq from the same tumor sample.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig16_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-020-0844-1/MediaObjects/41591_2020_844_Fig16_ESM.jpg" alt="extended data figure 16" aria-describedby="rc-Fig16"></picture><p class="c-reading-companion__figure-links"><a href="#Fig16" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0844-1/figures/16" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric" data-track-context="article references sidebar"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">Cieslik, M. &amp; Chinnaiyan, A. M. Cancer transcriptome profiling at the juncture of clinical translation. <i>Nat. Rev. Genet.</i> <b>19</b>, 93–109 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnrg.2017.96" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nrg.2017.96" data-track-item_id="10.1038/nrg.2017.96">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVCjsr7K" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29279605" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Cancer%20transcriptome%20profiling%20at%20the%20juncture%20of%20clinical%20translation&amp;journal=Nat.%20Rev.%20Genet.&amp;doi=10.1038%2Fnrg.2017.96&amp;volume=19&amp;pages=93-109&amp;publication_year=2018&amp;author=Cieslik%2CM&amp;author=Chinnaiyan%2CAM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">Filbin, M. G. et al. Developmental and oncogenic programs in H3K27M gliomas dissected by single-cell RNA-seq. <i>Science</i> <b>360</b>, 331–335 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aao4750" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aao4750" data-track-item_id="10.1126/science.aao4750">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXns12gtLg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29674595" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5949869" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Developmental%20and%20oncogenic%20programs%20in%20H3K27M%20gliomas%20dissected%20by%20single-cell%20RNA-seq&amp;journal=Science&amp;doi=10.1126%2Fscience.aao4750&amp;volume=360&amp;pages=331-335&amp;publication_year=2018&amp;author=Filbin%2CMG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Jerby-Arnon, L. et al. A cancer cell program promotes T cell exclusion and resistance to checkpoint blockade. <i>Cell</i> <b>175</b>, 984–997.e924 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2018.09.006" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2018.09.006" data-track-item_id="10.1016/j.cell.2018.09.006">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVyhurvK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30388455" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6410377" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20cancer%20cell%20program%20promotes%20T%20cell%20exclusion%20and%20resistance%20to%20checkpoint%20blockade&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.09.006&amp;volume=175&amp;pages=984-997.e924&amp;publication_year=2018&amp;author=Jerby-Arnon%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">Puram, S. V. et al. Single-cell transcriptomic analysis of primary and metastatic tumor ecosystems in head and neck cancer. <i>Cell</i> <b>171</b>, 1611–1624.e1624 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2017.10.044" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2017.10.044" data-track-item_id="10.1016/j.cell.2017.10.044">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFWms7nE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29198524" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5878932" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20transcriptomic%20analysis%20of%20primary%20and%20metastatic%20tumor%20ecosystems%20in%20head%20and%20neck%20cancer&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.10.044&amp;volume=171&amp;pages=1611-1624.e1624&amp;publication_year=2017&amp;author=Puram%2CSV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">Tirosh, I. et al. Single-cell RNA-seq supports a developmental hierarchy in human oligodendroglioma. <i>Nature</i> <b>539</b>, 309–313 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnature20123" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nature20123" data-track-item_id="10.1038/nature20123">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27806376" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5465819" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20RNA-seq%20supports%20a%20developmental%20hierarchy%20in%20human%20oligodendroglioma&amp;journal=Nature&amp;doi=10.1038%2Fnature20123&amp;volume=539&amp;pages=309-313&amp;publication_year=2016&amp;author=Tirosh%2CI" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Venteicher, A. S. et al. Decoupling genetics, lineages and microenvironment in IDH-mutant gliomas by single-cell RNA-seq. <i>Science</i> <b>355</b>, eaai8478 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aai8478" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aai8478" data-track-item_id="10.1126/science.aai8478">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28360267" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5519096" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Decoupling%20genetics%2C%20lineages%20and%20microenvironment%20in%20IDH-mutant%20gliomas%20by%20single-cell%20RNA-seq&amp;journal=Science&amp;doi=10.1126%2Fscience.aai8478&amp;volume=355&amp;publication_year=2017&amp;author=Venteicher%2CAS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Moonshot Cancer Initiative, Generation of Human Tumor Atlases <i>National Cancer Institute</i> (accessed 20 April 2020); <a href="https://ccr.cancer.gov/research/cancer-moonshot" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://ccr.cancer.gov/research/cancer-moonshot">https://ccr.cancer.gov/research/cancer-moonshot</a></p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Habib, N. et al. Massively parallel single-nucleus RNA-seq with DroNc-seq. <i>Nat. Methods</i> <b>14</b>, 955–958 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnmeth.4407" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nmeth.4407" data-track-item_id="10.1038/nmeth.4407">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtl2gtL7L" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28846088" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5623139" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Massively%20parallel%20single-nucleus%20RNA-seq%20with%20DroNc-seq&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4407&amp;volume=14&amp;pages=955-958&amp;publication_year=2017&amp;author=Habib%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Habib, N. et al. Div-Seq: single-nucleus RNA-Seq reveals dynamics of rare adult newborn neurons. <i>Science</i> <b>353</b>, 925–928 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aad7038" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aad7038" data-track-item_id="10.1126/science.aad7038">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhtlyls7rK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27471252" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5480621" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Div-Seq%3A%20single-nucleus%20RNA-Seq%20reveals%20dynamics%20of%20rare%20adult%20newborn%20neurons&amp;journal=Science&amp;doi=10.1126%2Fscience.aad7038&amp;volume=353&amp;pages=925-928&amp;publication_year=2016&amp;author=Habib%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Gaublomme, J. T. et al. Nuclei multiplexing with barcoded antibodies for single-nucleus genomics. <i>Nat. Commun.</i> <b>10</b>, 2907 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-019-10756-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-019-10756-2" data-track-item_id="10.1038/s41467-019-10756-2">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31266958" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6606589" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Nuclei%20multiplexing%20with%20barcoded%20antibodies%20for%20single-nucleus%20genomics&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-10756-2&amp;volume=10&amp;publication_year=2019&amp;author=Gaublomme%2CJT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Drokhlyansky, E. et al. The enteric nervous system of the human and mouse colon at a single-cell resolution. Preprint at <i>bioRxiv</i> <a href="https://www.biorxiv.org/content/10.1101/746743v4" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.biorxiv.org/content/10.1101/746743v4">https://www.biorxiv.org/content/10.1101/746743v4</a> (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Lun, A. T. L. et al. EmptyDrops: distinguishing cells from empty droplets in droplet-based single-cell RNA sequencing data. <i>Genome Biol.</i> <b>20</b>, 63 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-019-1662-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-019-1662-y" data-track-item_id="10.1186/s13059-019-1662-y">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30902100" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6431044" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=EmptyDrops%3A%20distinguishing%20cells%20from%20empty%20droplets%20in%20droplet-based%20single-cell%20RNA%20sequencing%20data&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-019-1662-y&amp;volume=20&amp;publication_year=2019&amp;author=Lun%2CATL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Wolock, S. L., Lopez, R. &amp; Klein, A. M. Scrublet: computational identification of cell doublets in single-cell transcriptomic data. <i>Cell Syst</i>. <b>8</b>, 281–291.e289 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cels.2018.11.005" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cels.2018.11.005" data-track-item_id="10.1016/j.cels.2018.11.005">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosVyhtbw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30954476" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6625319" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Scrublet%3A%20computational%20identification%20of%20cell%20doublets%20in%20single-cell%20transcriptomic%20data&amp;journal=Cell%20Syst&amp;doi=10.1016%2Fj.cels.2018.11.005&amp;volume=8&amp;pages=281-291.e289&amp;publication_year=2019&amp;author=Wolock%2CSL&amp;author=Lopez%2CR&amp;author=Klein%2CAM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Li, B. et al. Cumulus: a cloud-based data analysis framework for large-scale single-cell and single-nucleus RNA-seq. Preprint at <i>bioRxiv</i> <a href="https://www.biorxiv.org/content/10.1101/823682v1" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.biorxiv.org/content/10.1101/823682v1">https://www.biorxiv.org/content/10.1101/823682v1</a> (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">Al-Hajj, M., Wicha, M. S., Benito-Hernandez, A., Morrison, S. J. &amp; Clarke, M. F. Prospective identification of tumorigenic breast cancer cells. <i>Proc. Natl Acad. Sci. USA</i> <b>100</b>, 3983–3988 (2003).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1073%2Fpnas.0530291100" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1073/pnas.0530291100" data-track-item_id="10.1073/pnas.0530291100">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3sXivFSqtbs%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12629218" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC153034" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Prospective%20identification%20of%20tumorigenic%20breast%20cancer%20cells&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.0530291100&amp;volume=100&amp;pages=3983-3988&amp;publication_year=2003&amp;author=Al-Hajj%2CM&amp;author=Wicha%2CMS&amp;author=Benito-Hernandez%2CA&amp;author=Morrison%2CSJ&amp;author=Clarke%2CMF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">McDivitt, R. W., Stone, K. R. &amp; Meyer, J. S. A method for dissociation of viable human breast cancer cells that produces flow cytometric kinetic information similar to that obtained by thymidine labeling. <i>Cancer Res.</i> <b>44</b>, 2628–2633 (1984).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaL2c3gsFCitA%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=6327021" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20method%20for%20dissociation%20of%20viable%20human%20breast%20cancer%20cells%20that%20produces%20flow%20cytometric%20kinetic%20information%20similar%20to%20that%20obtained%20by%20thymidine%20labeling&amp;journal=Cancer%20Res.&amp;volume=44&amp;pages=2628-2633&amp;publication_year=1984&amp;author=McDivitt%2CRW&amp;author=Stone%2CKR&amp;author=Meyer%2CJS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Neftel, C. et al. An integrative model of cellular states, plasticity and genetics for glioblastoma. <i>Cell</i> <b>178</b>, 835–849 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2019.06.024" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2019.06.024" data-track-item_id="10.1016/j.cell.2019.06.024">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVWlur7I" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31327527" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6703186" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20integrative%20model%20of%20cellular%20states%2C%20plasticity%20and%20genetics%20for%20glioblastoma&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.06.024&amp;volume=178&amp;pages=835-849&amp;publication_year=2019&amp;author=Neftel%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">Young, M. D. &amp; Behjati, S. SoupX removes ambient RNA contamination from droplet based single cell RNA sequencing data. Preprint at <i>bioRxiv</i> <a href="https://www.biorxiv.org/content/10.1101/303727v1" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.biorxiv.org/content/10.1101/303727v1">https://www.biorxiv.org/content/10.1101/303727v1</a> (2018).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">Stewart, E. et al. Orthotopic patient-derived xenografts of paediatric solid tumours. <i>Nature</i> <b>549</b>, 96–100 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnature23647" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nature23647" data-track-item_id="10.1038/nature23647">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhsVWhu7jJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28854174" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5659286" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Orthotopic%20patient-derived%20xenografts%20of%20paediatric%20solid%20tumours&amp;journal=Nature&amp;doi=10.1038%2Fnature23647&amp;volume=549&amp;pages=96-100&amp;publication_year=2017&amp;author=Stewart%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Stewart, E. et al. Development and characterization of a human orthotopic neuroblastoma xenograft. <i>Dev. Biol.</i> <b>407</b>, 344–355 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ydbio.2015.02.002" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ydbio.2015.02.002" data-track-item_id="10.1016/j.ydbio.2015.02.002">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXmsFSjtL8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25863122" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4995597" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20and%20characterization%20of%20a%20human%20orthotopic%20neuroblastoma%20xenograft&amp;journal=Dev.%20Biol.&amp;doi=10.1016%2Fj.ydbio.2015.02.002&amp;volume=407&amp;pages=344-355&amp;publication_year=2015&amp;author=Stewart%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Hermansen, J. U., Tjonnfjord, G. E., Munthe, L. A., Tasken, K. &amp; Skanland, S. S. Cryopreservation of primary B cells minimally influences their signaling responses. <i>Sci. Rep.</i> <b>8</b>, 17651 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41598-018-36121-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41598-018-36121-9" data-track-item_id="10.1038/s41598-018-36121-9">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30518828" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6281576" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Cryopreservation%20of%20primary%20B%20cells%20minimally%20influences%20their%20signaling%20responses&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fs41598-018-36121-9&amp;volume=8&amp;publication_year=2018&amp;author=Hermansen%2CJU&amp;author=Tjonnfjord%2CGE&amp;author=Munthe%2CLA&amp;author=Tasken%2CK&amp;author=Skanland%2CSS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Guillaumet-Adkins, A. et al. Single-cell transcriptome conservation in cryopreserved cells and tissues. <i>Genome Biol</i>. <b>18</b>, 45 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-017-1171-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-017-1171-9" data-track-item_id="10.1186/s13059-017-1171-9">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28249587" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5333448" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20transcriptome%20conservation%20in%20cryopreserved%20cells%20and%20tissues&amp;journal=Genome%20Biol&amp;doi=10.1186%2Fs13059-017-1171-9&amp;volume=18&amp;publication_year=2017&amp;author=Guillaumet-Adkins%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">Gao, R. et al. Nanogrid single-nucleus RNA sequencing reveals phenotypic diversity in breast cancer. <i>Nat. Commun.</i> <b>8</b>, 228 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-017-00244-w" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-017-00244-w" data-track-item_id="10.1038/s41467-017-00244-w">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28794488" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5550415" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Nanogrid%20single-nucleus%20RNA%20sequencing%20reveals%20phenotypic%20diversity%20in%20breast%20cancer&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-00244-w&amp;volume=8&amp;publication_year=2017&amp;author=Gao%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Butler, A., Hoffman, P., Smibert, P., Papalexi, E. &amp; Satija, R. Integrating single-cell transcriptomic data across different conditions, technologies and species. <i>Nat. Biotechnol.</i> <b>36</b>, 411–420 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnbt.4096" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nbt.4096" data-track-item_id="10.1038/nbt.4096">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXmslKrtL0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29608179" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6700744" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Integrating%20single-cell%20transcriptomic%20data%20across%20different%20conditions%2C%20technologies%20and%20species&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.4096&amp;volume=36&amp;pages=411-420&amp;publication_year=2018&amp;author=Butler%2CA&amp;author=Hoffman%2CP&amp;author=Smibert%2CP&amp;author=Papalexi%2CE&amp;author=Satija%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">van den Brink, S. C. et al. Single-cell sequencing reveals dissociation-induced gene expression in tissue subpopulations. <i>Nat. Methods</i> <b>14</b>, 935–936 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnmeth.4437" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nmeth.4437" data-track-item_id="10.1038/nmeth.4437">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28960196" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20sequencing%20reveals%20dissociation-induced%20gene%20expression%20in%20tissue%20subpopulations&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4437&amp;volume=14&amp;pages=935-936&amp;publication_year=2017&amp;author=Brink%2CSC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Baslan, T. &amp; Hicks, J. Unravelling biology and shifting paradigms in cancer with single-cell sequencing. <i>Nat. Rev. Cancer</i> <b>17</b>, 557–569 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnrc.2017.58" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nrc.2017.58" data-track-item_id="10.1038/nrc.2017.58">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlyisb3M" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28835719" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Unravelling%20biology%20and%20shifting%20paradigms%20in%20cancer%20with%20single-cell%20sequencing&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc.2017.58&amp;volume=17&amp;pages=557-569&amp;publication_year=2017&amp;author=Baslan%2CT&amp;author=Hicks%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">Suva, M. L. &amp; Tirosh, I. Single-cell RNA sequencing in cancer: lessons learned and emerging challenges. <i>Mol. Cell</i> <b>75</b>, 7–12 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.molcel.2019.05.003" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.molcel.2019.05.003" data-track-item_id="10.1016/j.molcel.2019.05.003">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVelu7bJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31299208" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20RNA%20sequencing%20in%20cancer%3A%20lessons%20learned%20and%20emerging%20challenges&amp;journal=Mol.%20Cell&amp;doi=10.1016%2Fj.molcel.2019.05.003&amp;volume=75&amp;pages=7-12&amp;publication_year=2019&amp;author=Suva%2CML&amp;author=Tirosh%2CI" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Rambow, F. et al. Toward minimal residual disease-directed therapy in melanoma. <i>Cell</i> <b>174</b>, 843–855.e819 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2018.06.025" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2018.06.025" data-track-item_id="10.1016/j.cell.2018.06.025">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlartrfI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30017245" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Toward%20minimal%20residual%20disease-directed%20therapy%20in%20melanoma&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.06.025&amp;volume=174&amp;pages=843-855.e819&amp;publication_year=2018&amp;author=Rambow%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">Qi, Z., Barrett, T., Parikh, A. S., Tirosh, I. &amp; Puram, S. V. Single-cell sequencing and its applications in head and neck cancer. <i>Oral Oncol</i>. <b>99</b>, 104441 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.oraloncology.2019.104441" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.oraloncology.2019.104441" data-track-item_id="10.1016/j.oraloncology.2019.104441">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitV2ms7nJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31689639" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6981283" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20sequencing%20and%20its%20applications%20in%20head%20and%20neck%20cancer&amp;journal=Oral%20Oncol&amp;doi=10.1016%2Fj.oraloncology.2019.104441&amp;volume=99&amp;publication_year=2019&amp;author=Qi%2CZ&amp;author=Barrett%2CT&amp;author=Parikh%2CAS&amp;author=Tirosh%2CI&amp;author=Puram%2CSV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">van Galen, P. et al. Single-cell RNA-Seq reveals AML hierarchies relevant to disease progression and immunity. <i>Cell</i> <b>176</b>, 1265–1281.e1224 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2019.01.031" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2019.01.031" data-track-item_id="10.1016/j.cell.2019.01.031">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30827681" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6515904" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20RNA-Seq%20reveals%20AML%20hierarchies%20relevant%20to%20disease%20progression%20and%20immunity&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.01.031&amp;volume=176&amp;pages=1265-1281.e1224&amp;publication_year=2019&amp;author=Galen%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Chevrier, S. et al. An immune atlas of clear cell renal cell carcinoma. <i>Cell</i> <b>169</b>, 736–749.e718 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2017.04.016" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2017.04.016" data-track-item_id="10.1016/j.cell.2017.04.016">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXnt1Cms7s%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28475899" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5422211" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20immune%20atlas%20of%20clear%20cell%20renal%20cell%20carcinoma&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.04.016&amp;volume=169&amp;pages=736-749.e718&amp;publication_year=2017&amp;author=Chevrier%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Savas, P. et al. Single-cell profiling of breast cancer T cells reveals a tissue-resident memory subset associated with improved prognosis. <i>Nat. Med.</i> <b>24</b>, 986–993 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41591-018-0078-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41591-018-0078-7" data-track-item_id="10.1038/s41591-018-0078-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXht1WqsLjL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29942092" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-cell%20profiling%20of%20breast%20cancer%20T%20cells%20reveals%20a%20tissue-resident%20memory%20subset%20associated%20with%20improved%20prognosis&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-018-0078-7&amp;volume=24&amp;pages=986-993&amp;publication_year=2018&amp;author=Savas%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">Karaayvaz, M. et al. Unravelling subclonal heterogeneity and aggressive disease states in TNBC through single-cell RNA-seq. <i>Nat. Commun.</i> <b>9</b>, 3588 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-018-06052-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-018-06052-0" data-track-item_id="10.1038/s41467-018-06052-0">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30181541" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6123496" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Unravelling%20subclonal%20heterogeneity%20and%20aggressive%20disease%20states%20in%20TNBC%20through%20single-cell%20RNA-seq&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-06052-0&amp;volume=9&amp;publication_year=2018&amp;author=Karaayvaz%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">Lambrechts, D. et al. Phenotype molding of stromal cells in the lung tumor microenvironment. <i>Nat. Med.</i> <b>24</b>, 1277–1289 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41591-018-0096-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41591-018-0096-5" data-track-item_id="10.1038/s41591-018-0096-5">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjs7vM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29988129" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Phenotype%20molding%20of%20stromal%20cells%20in%20the%20lung%20tumor%20microenvironment&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-018-0096-5&amp;volume=24&amp;pages=1277-1289&amp;publication_year=2018&amp;author=Lambrechts%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">Stoeckius, M. et al. Simultaneous epitope and transcriptome measurement in single cells. <i>Nat. Methods</i> <b>14</b>, 865–868 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnmeth.4380" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nmeth.4380" data-track-item_id="10.1038/nmeth.4380">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXht1CkurzK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28759029" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5669064" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Simultaneous%20epitope%20and%20transcriptome%20measurement%20in%20single%20cells&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.4380&amp;volume=14&amp;pages=865-868&amp;publication_year=2017&amp;author=Stoeckius%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Kang, H. M. et al. Multiplexed droplet single-cell RNA-sequencing using natural genetic variation. <i>Nat. Biotechnol.</i> <b>36</b>, 89–94 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnbt.4042" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nbt.4042" data-track-item_id="10.1038/nbt.4042">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvFGmur3F" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29227470" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%20droplet%20single-cell%20RNA-sequencing%20using%20natural%20genetic%20variation&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.4042&amp;volume=36&amp;pages=89-94&amp;publication_year=2018&amp;author=Kang%2CHM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Stoeckius, M. et al. Cell hashing with barcoded antibodies enables multiplexing and doublet detection for single cell genomics. <i>Genome Biol</i>. <b>19</b>, 224 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-018-1603-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-018-1603-1" data-track-item_id="10.1186/s13059-018-1603-1">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtVKksbfJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30567574" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6300015" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Cell%20hashing%20with%20barcoded%20antibodies%20enables%20multiplexing%20and%20doublet%20detection%20for%20single%20cell%20genomics&amp;journal=Genome%20Biol&amp;doi=10.1186%2Fs13059-018-1603-1&amp;volume=19&amp;publication_year=2018&amp;author=Stoeckius%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Bakken, T. E. et al. Single-nucleus and single-cell transcriptomes compared in matched cortical cell types. <i>PLoS ONE</i> <b>13</b>, e0209648 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pone.0209648" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pone.0209648" data-track-item_id="10.1371/journal.pone.0209648">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30586455" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6306246" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-nucleus%20and%20single-cell%20transcriptomes%20compared%20in%20matched%20cortical%20cell%20types&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0209648&amp;volume=13&amp;publication_year=2018&amp;author=Bakken%2CTE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">Shekhar, K. et al. Comprehensive classification of retinal bipolar neurons by single-cell transcriptomics. <i>Cell</i> <b>166</b>, 1308–1323.e1330 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2016.07.054" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2016.07.054" data-track-item_id="10.1016/j.cell.2016.07.054">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVehu73M" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27565351" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5003425" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20classification%20of%20retinal%20bipolar%20neurons%20by%20single-cell%20transcriptomics&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2016.07.054&amp;volume=166&amp;pages=1308-1323.e1330&amp;publication_year=2016&amp;author=Shekhar%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">Wolf, F. A., Angerer, P. &amp; Theis, F. J. SCANPY: large-scale single-cell gene expression data analysis. <i>Genome Biol.</i> <b>19</b>, 15 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-017-1382-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-017-1382-0" data-track-item_id="10.1186/s13059-017-1382-0">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29409532" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5802054" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SCANPY%3A%20large-scale%20single-cell%20gene%20expression%20data%20analysis&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-017-1382-0&amp;volume=19&amp;publication_year=2018&amp;author=Wolf%2CFA&amp;author=Angerer%2CP&amp;author=Theis%2CFJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">Macosko, E. Z. et al. Highly parallel genome-wide expression profiling of individual cells using nanoliter droplets. <i>Cell</i> <b>161</b>, 1202–1214 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXpt1Sgt7o%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26000488" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4481139" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Highly%20parallel%20genome-wide%20expression%20profiling%20of%20individual%20cells%20using%20nanoliter%20droplets&amp;journal=Cell&amp;volume=161&amp;pages=1202-1214&amp;publication_year=2015&amp;author=Macosko%2CEZ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">Benjamini, Y. &amp; Yosef, Hochberg Controlling the false discovery rate: a practical and powerful approach to multiple testing. <i>J. R. Stat. Soc. B</i> <b>57.1</b>, 289–300 (1995).</p><p class="c-reading-companion__reference-links"><a href="http://scholar.google.com/scholar_lookup?&amp;title=Controlling%20the%20false%20discovery%20rate%3A%20a%20practical%20and%20powerful%20approach%20to%20multiple%20testing&amp;journal=J.%20R.%20Stat.%20Soc.%20B&amp;volume=57.1&amp;pages=289-300&amp;publication_year=1995&amp;author=Benjamini%2CY&amp;author=Yosef%2CHochberg" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">McInnes, L., Healy, J. &amp; Melville, J. UMAP: Uniform Manifold Approximation and Projection for dimension reduction. Preprint at <a href="https://arxiv.org/abs/1802.03426" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://arxiv.org/abs/1802.03426">https://arxiv.org/abs/1802.03426</a> (2018).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">Aran, D. et al. Reference-based analysis of lung single-cell sequencing reveals a transitional profibrotic macrophage. <i>Nat. Immunol.</i> <b>20</b>, 163–172 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41590-018-0276-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41590-018-0276-y" data-track-item_id="10.1038/s41590-018-0276-y">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvFSgu7k%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30643263" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6340744" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Reference-based%20analysis%20of%20lung%20single-cell%20sequencing%20reveals%20a%20transitional%20profibrotic%20macrophage&amp;journal=Nat.%20Immunol.&amp;doi=10.1038%2Fs41590-018-0276-y&amp;volume=20&amp;pages=163-172&amp;publication_year=2019&amp;author=Aran%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">Tickle, T. I., Georgescu, C., Brown, M. &amp; Haas, B. inferCNV of the Trinity CTAT Project (2019); <a href="https://github.com/broadinstitute/inferCNV" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://github.com/broadinstitute/inferCNV">https://github.com/broadinstitute/inferCNV</a></p></li></ol></div></div>
        </div>
    </div>
</aside>
</div>


    
        
    
    
        
            
        

        
            
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>




<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Medicine
                    </span>
                    (<i itemprop="alternateName">Nat Med</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-170X</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1078-8956</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/" data-track="click" data-track-action="protocols.io" data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/authors/editorial_policies" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nature-research/open-access" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/language-editing/" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/" data-track="click" data-track-action="research solutions" data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library" data-track="click" data-track-action="Recommend to library" data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica" data-track="click" data-track-action="nature africa" data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly" data-track="click" data-track-action="nature Italy" data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies" data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice" data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/accessibility-statement" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20">
    </a>
    <p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"></path></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"></path><clipPath id="b"><use xlink:href="#a" style="overflow:visible"></use></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"></path><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"></path><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"></path><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"></path></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"></path></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif c-site-messages--nature-briefing-cancer" data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Cancer">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Cancer">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing: Cancer" src="/static/images/logos/nature-briefing-logo-cancer-white-bc263e7d57.svg" width="373" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing: Cancer</em> newsletter — what matters in cancer research, free to your inbox weekly.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/cancer" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Cancer">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="CancerBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="CancerBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required="" data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:cancer" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required="">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Cancer">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get what matters in cancer research, free to your inbox weekly.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="/briefing/cancer/?brieferEntryPoint=CancerBriefingBanner">Sign up for Nature Briefing: Cancer
            </a>
        </div>

    </div>

</div>

    




<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41591-020-0844-1&amp;format=js&amp;last_modified=2020-06-25" async=""></script>

<vhi-yodrnjawtfbz></vhi-yodrnjawtfbz><div data-cc-ghost="" style="height: 352px;"></div><script type="text/javascript" id="fetch-contextual-ads-campaign-data" charset="">(function(){var g=function(a){a=new CustomEvent("campaignDataLoaded",{detail:a});document.dispatchEvent(a)},b=google_tag_manager["rm"]["50443292"](18);if(b){var k=1500;b=google_tag_manager["rm"]["50443292"](19);b="link"===b?google_tag_manager["rm"]["50443292"](20):google_tag_manager["rm"]["50443292"](21);var m=function(a,c){var h=!1,l=setTimeout(function(){h=!0;c(null)},k),d=new XMLHttpRequest;d.onload=function(){var e=null,f=[];if(200===d.status){try{e=JSON.parse(d.responseText)}catch(p){e={}}f=e.campaigns||[]}h||(clearTimeout(l),c(f.length?f.join(","):null))};d.open("GET",
"/platform/contextual?doi\x3d"+a);d.send()},n=function(a,c){a&&-1===a.indexOf("not set")?m(a,c):setTimeout(c,1)};n(b,function(a){window.campaignsForContextualAds=a;g(a);window.dataLayer.push({content:{article:{campaignID:a}}})})}else window.campaignsForContextualAds="",g(null)})();</script>
<script type="text/javascript" id="gtm-setup-accessdetailsloaded-handler" charset="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail||{};var b={event:"update-access-details"};a.institutional_business_partner_ids&&a.resolved_by||console.log("BPID data could not be retrieved from /exposed-details");a.logged_in_to_sn_profile||console.log("logged_in_to_sn_profile could not be retrieved from /exposed-details");var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?
a.resolved_by.join(";"):"",e=a.personal_business_partner_id_found,f=a.logged_in_to_sn_profile,g=a.snid;b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null,personal:e||null,logged_in_to_sn_profile:f,snid:g||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy={};window.dataLayer.push(b);window.idpUserDataLoaded=!0},!1);window.dataLayer.push({event:"accessdetailsloaded-handler-added"});
window.accessDetailsLoadedHandlerAdded=!0;</script><script type="text/javascript" id="gtm-mediatrust" charset="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="crossmark-script" text="" charset="" type="text/javascript" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="polyfill-matches" charset="">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="fetch-idp-user-data" charset="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},site=google_tag_manager["rm"]["50443292"](59),idpUrl;
switch(site){case "nature":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?"https://staging-idp.nature.com/exposed-details":"https://idp.nature.com/exposed-details";break;case "link":idpUrl=-1<window.location.hostname.indexOf("link-qa")?"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details";break;case "springer":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?
"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details"}if(void 0!==idpUrl){var transport=new XMLHttpRequest;transport.open("GET",idpUrl,!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script src="https://verify.nature.com/verify/nature.min.js"></script><script type="text/javascript" id="create-ad-slots" charset="">function createAndLoadAds(){function h(b){"nature"===a&&google_tag_manager["rm"]["50443292"](87)(b);"bmc"!==a&&"springeropen"!==a||google_tag_manager["rm"]["50443292"](99)(b);("link"===a&&"oscar"===google_tag_manager["rm"]["50443292"](100)||"springer"===a||"link"===a&&!0===google_tag_manager["rm"]["50443292"](101))&&google_tag_manager["rm"]["50443292"](126)(b);"link"===a&&"bunsen"===google_tag_manager["rm"]["50443292"](127)&&google_tag_manager["rm"]["50443292"](152)(b);"link"===a&&"Core"===google_tag_manager["rm"]["50443292"](153)&&google_tag_manager["rm"]["50443292"](165)(b)}window.googletag=window.googletag||{cmd:[]};var a=google_tag_manager["rm"]["50443292"](166),e=google_tag_manager["rm"]["50443292"](168);
if(-1===window.location.search.indexOf("hide_ads\x3dtrue"))if(window.adSlots&&0!==Object.keys(window.adSlots).length)console.log("Ads previously loaded. Will not update ad slots until next page load.");else{window.adSlots||(window.adSlots={});window.getAd=function(b,f){for(var c in window.adSlots)if(-1<c.indexOf(b)){if("object"===f)return adSlots[c];if("slot"===f)return adSlots[c].slot}};googletag.cmd.push(function(){googletag.pubads().setPrivacySettings({limitedAds:e});console.log("limitedAds is ",
e);googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["rm"]["50443292"](170));googletag.pubads().enableSingleRequest();googletag.pubads().disableInitialLoad()});for(var g=document.querySelectorAll("[data-gpt]"),d=0;g[d];++d)h(g[d]);googletag.cmd.push(function(){googletag.enableServices()})}}createAndLoadAds();</script>
</body></html>