<html lang="en" class="js"><head><link rel="preload" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" as="font" type="font/woff2" crossorigin=""><script async="" src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script type="text/javascript" async="" src="https://pagead2.googlesyndication.com/tag/js/gpt.js"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-MRVXSHQ"></script><script type="text/javascript" src="https://www.nature.com/EcIxe_RDmwsgbOGxJDy7rtRym-W8VMKcfZ_Vu7KE0TNAwL7Ed3X3cUYKQ8mh3gYNFaf6E_eSzPC_NV_NvoSbXg=="></script><script src="https://cmp.nature.com/production_live/en/consent-bundle-8-85.js" onload="initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')"></script>
    <title>Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment | Nature Medicine</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nm.rss">


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin="">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480">

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"disease-prevention;microbial-genetics","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Medicine","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41591-020-0894-4"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Kern Rei Chng","Chenhao Li","Denis Bertrand","Amanda Hui Qi Ng","Junmei Samantha Kwah","Hwee Meng Low","Chengxuan Tong","Maanasa Natrajan","Michael Hongjie Zhang","Licheng Xu","Karrie Kwan Ki Ko","Eliza Xin Pei Ho","Tamar V. Av-Shalom","Jeanette Woon Pei Teo","Chiea Chuen Khor","David Danko","Daniela Bezdan","Ebrahim Afshinnekoo","Sofia Ahsanuddin","Chandrima Bhattacharya","Daniel J. Butler","Kern Rei Chng","Francesca De Filippis","Jochen Hecht","Andre Kahles","Mikhail Karasikov","Nikos C. Kyrpides","Marcus H. Y. Leung","Dmitry Meleshko","Harun Mustafa","Beth Mutai","Russell Y. Neches","Amanda Ng","Marina Nieto-Caballero","Olga Nikolayeva","Tatyana Nikolayeva","Eileen Png","Jorge L. Sanchez","Heba Shaaban","Maria A. Sierra","Xinzhao Tong","Ben Young","Josue Alicea","Malay Bhattacharyya","Ran Blekhman","Eduardo Castro-Nallar","Ana M. Cañas","Aspassia D. Chatziefthimiou","Robert W. Crawford","Youping Deng","Christelle Desnues","Emmanuel Dias-Neto","Daisy Donnellan","Marius Dybwad","Eran Elhaik","Danilo Ercolini","Alina Frolova","Alexandra B. Graf","David C. Green","Iman Hajirasouliha","Mark Hernandez","Gregorio Iraola","Soojin Jang","Angela Jones","Frank J. Kelly","Kaymisha Knights","Paweł P. Łabaj","Patrick K. H. Lee","Levy Shawn","Per Ljungdahl","Abigail Lyons","Gabriella Mason-Buck","Ken McGrath","Emmanuel F. Mongodin","Milton Ozorio Moraes","Niranjan Nagarajan","Houtan Noushmehr","Manuela Oliveira","Stephan Ossowski","Olayinka O. Osuolale","Orhan Özcan","David Paez-Espino","Nicolas Rascovan","Hugues Richard","Gunnar Rätsch","Lynn M. Schriml","Torsten Semmler","Osman U. Sezerman","Leming Shi","Le Huu Song","Haruo Suzuki","Denise Syndercombe Court","Dominique Thomas","Scott W. Tighe","Klas I. Udekwu","Juan A. Ugalde","Brandon Valentine","Dimitar I. Vassilev","Elena Vayndorf","Thirumalaisamy P. Velavan","María M. Zambrano","Jifeng Zhu","Sibo Zhu","Christopher E. Mason","Swaine L. Chen","Christopher E. Mason","Oon Tek Ng","Kalisvar Marimuthu","Brenda Ang","Niranjan Nagarajan"],"publishedAt":1591574400,"publishedAtString":"2020-06-08","title":"Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Disease prevention,Microbial genetics"},"journal":{"pcode":"nm","title":"nature medicine","volume":"26","issue":"6","id":41591,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"BR","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{font-weight:700;-webkit-font-smoothing:antialiased;font-size:1.5rem;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.app-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{margin:4px 4px 0;fill:#888;height:10px;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-c9121fecab.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png">
<link rel="icon" type="image/png" sizes="48x48" href="/static/images/favicons/nature/favicon-48x48-b52890008c.png">
<link rel="icon" type="image/png" sizes="32x32" href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="/static/images/favicons/nature/favicon-16x16-951651ab72.png">
<link rel="manifest" href="/static/manifest.json" crossorigin="use-credentials">
<link rel="mask-icon" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="/static/images/favicons/nature/favicon.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-e9d2061c95.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-1cbf3a8b56.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script><script id="js-position1" data-test="global-article-js" defer="" src="/static/js/global-article-es6-bundle-c8a573ca90.js"></script><script id="js-position2" data-test="shared-js" defer="" src="/static/js/shared-es6-bundle-e9d2061c95.js"></script><script id="js-position3" data-test="header-150-js" defer="" src="/static/js/header-150-es6-bundle-5bb959eaa1.js"></script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment","description":"Although disinfection is key to infection control, the colonization patterns and resistomes of hospital-environment microbes remain underexplored. We report the first extensive genomic characterization of microbiomes, pathogens and antibiotic resistance cassettes in a tertiary-care hospital, from repeated sampling (up to 1.5 years apart) of 179 sites associated with 45 beds. Deep shotgun metagenomics unveiled distinct ecological niches of microbes and antibiotic resistance genes characterized by biofilm-forming and human-microbiome-influenced environments with corresponding patterns of spatiotemporal divergence. Quasi-metagenomics with nanopore sequencing provided thousands of high-contiguity genomes, phage and plasmid sequences (>60% novel), enabling characterization of resistome and mobilome diversity and dynamic architectures in hospital environments. Phylogenetics identified multidrug-resistant strains as being widely distributed and stably colonizing across sites. Comparisons with clinical isolates indicated that such microbes can persist in hospitals for extended periods (>8 years), to opportunistically infect patients. These findings highlight the importance of characterizing antibiotic resistance reservoirs in hospitals and establish the feasibility of systematic surveys to target resources for preventing infections. Spatiotemporal characterization of microbial diversity and antibiotic resistance in a tertiary-care hospital reveals broad distribution and persistence of antibiotic-resistant organisms that could cause opportunistic infections in a healthcare setting.","datePublished":"2020-06-08T00:00:00Z","dateModified":"2020-06-08T00:00:00Z","pageStart":"941","pageEnd":"951","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41591-020-0894-4","keywords":["Disease prevention","Microbial genetics","Biomedicine","general","Cancer Research","Metabolic Diseases","Infectious Diseases","Molecular Medicine","Neurosciences"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig5_HTML.png"],"isPartOf":{"name":"Nature Medicine","issn":["1546-170X","1078-8956"],"volumeNumber":"26","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Kern Rei Chng","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Chenhao Li","url":"http://orcid.org/0000-0003-1182-6804","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Denis Bertrand","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Amanda Hui Qi Ng","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Junmei Samantha Kwah","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Hwee Meng Low","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Chengxuan Tong","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Maanasa Natrajan","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michael Hongjie Zhang","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Licheng Xu","affiliation":[{"name":"Singapore University of Technology and Design","address":{"name":"Information Systems Technology and Design, Singapore University of Technology and Design, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Karrie Kwan Ki Ko","affiliation":[{"name":"Singapore General Hospital","address":{"name":"Department of Microbiology, Singapore General Hospital, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"},{"name":"Singapore General Hospital","address":{"name":"Department of Molecular Pathology, Singapore General Hospital, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"},{"name":"Duke-NUS Graduate Medical School","address":{"name":"Duke-NUS Graduate Medical School, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Eliza Xin Pei Ho","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Tamar V. Av-Shalom","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jeanette Woon Pei Teo","affiliation":[{"name":"National University Hospital","address":{"name":"Department of Laboratory Medicine, National University Hospital, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Chiea Chuen Khor","url":"http://orcid.org/0000-0002-1128-4729","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Swaine L. Chen","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Christopher E. Mason","url":"http://orcid.org/0000-0002-1850-1642","affiliation":[{"name":"Weill Cornell Medicine","address":{"name":"Department of Physiology and Biophysics, Weill Cornell Medicine, New York, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Oon Tek Ng","affiliation":[{"name":"National Centre for Infectious Diseases","address":{"name":"National Centre for Infectious Diseases, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"},{"name":"Tan Tock Seng Hospital","address":{"name":"Department of Infectious Diseases, Tan Tock Seng Hospital, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"},{"name":"Nanyang Technological University","address":{"name":"Lee Kong Chian School of Medicine, Nanyang Technological University, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Kalisvar Marimuthu","url":"http://orcid.org/0000-0002-0395-5633","affiliation":[{"name":"National Centre for Infectious Diseases","address":{"name":"National Centre for Infectious Diseases, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"},{"name":"Tan Tock Seng Hospital","address":{"name":"Department of Infectious Diseases, Tan Tock Seng Hospital, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"},{"name":"National University of Singapore","address":{"name":"Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Brenda Ang","affiliation":[{"name":"National Centre for Infectious Diseases","address":{"name":"National Centre for Infectious Diseases, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"},{"name":"Tan Tock Seng Hospital","address":{"name":"Department of Infectious Diseases, Tan Tock Seng Hospital, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Niranjan Nagarajan","url":"http://orcid.org/0000-0003-0850-5604","affiliation":[{"name":"Genome Institute of Singapore","address":{"name":"Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"},{"name":"National University of Singapore","address":{"name":"Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore","@type":"PostalAddress"},"@type":"Organization"}],"email":"nagarajann@gis.a-star.edu.sg","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41591-020-0894-4">
    
    
    <meta name="journal_id" content="41591">
    <meta name="dc.title" content="Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment">
    <meta name="dc.source" content="Nature Medicine 2020 26:6">
    <meta name="dc.format" content="text/html">
    <meta name="dc.publisher" content="Nature Publishing Group">
    <meta name="dc.date" content="2020-06-08">
    <meta name="dc.type" content="OriginalPaper">
    <meta name="dc.language" content="En">
    <meta name="dc.copyright" content="2020 The Author(s)">
    <meta name="dc.rights" content="2020 The Author(s)">
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="dc.description" content="Although disinfection is key to infection control, the colonization patterns and resistomes of hospital-environment microbes remain underexplored. We report the first extensive genomic characterization of microbiomes, pathogens and antibiotic resistance cassettes in a tertiary-care hospital, from repeated sampling (up to 1.5 years apart) of 179 sites associated with 45 beds. Deep shotgun metagenomics unveiled distinct ecological niches of microbes and antibiotic resistance genes characterized by biofilm-forming and human-microbiome-influenced environments with corresponding patterns of spatiotemporal divergence. Quasi-metagenomics with nanopore sequencing provided thousands of high-contiguity genomes, phage and plasmid sequences (&amp;gt;60% novel), enabling characterization of resistome and mobilome diversity and dynamic architectures in hospital environments. Phylogenetics identified multidrug-resistant strains as being widely distributed and stably colonizing across sites. Comparisons with clinical isolates indicated that such microbes can persist in hospitals for extended periods (&amp;gt;8 years), to opportunistically infect patients. These findings highlight the importance of characterizing antibiotic resistance reservoirs in hospitals and establish the feasibility of systematic surveys to target resources for preventing infections. Spatiotemporal characterization of microbial diversity and antibiotic resistance in a tertiary-care hospital reveals broad distribution and persistence of antibiotic-resistant organisms that could cause opportunistic infections in a healthcare setting.">
    <meta name="prism.issn" content="1546-170X">
    <meta name="prism.publicationName" content="Nature Medicine">
    <meta name="prism.publicationDate" content="2020-06-08">
    <meta name="prism.volume" content="26">
    <meta name="prism.number" content="6">
    <meta name="prism.section" content="OriginalPaper">
    <meta name="prism.startingPage" content="941">
    <meta name="prism.endingPage" content="951">
    <meta name="prism.copyright" content="2020 The Author(s)">
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="prism.url" content="https://www.nature.com/articles/s41591-020-0894-4">
    <meta name="prism.doi" content="doi:10.1038/s41591-020-0894-4">
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41591-020-0894-4.pdf">
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41591-020-0894-4">
    <meta name="citation_journal_title" content="Nature Medicine">
    <meta name="citation_journal_abbrev" content="Nat Med">
    <meta name="citation_publisher" content="Nature Publishing Group">
    <meta name="citation_issn" content="1546-170X">
    <meta name="citation_title" content="Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment">
    <meta name="citation_volume" content="26">
    <meta name="citation_issue" content="6">
    <meta name="citation_publication_date" content="2020/06">
    <meta name="citation_online_date" content="2020/06/08">
    <meta name="citation_firstpage" content="941">
    <meta name="citation_lastpage" content="951">
    <meta name="citation_article_type" content="Article">
    <meta name="citation_fulltext_world_readable" content="">
    <meta name="citation_language" content="en">
    <meta name="dc.identifier" content="doi:10.1038/s41591-020-0894-4">
    <meta name="DOI" content="10.1038/s41591-020-0894-4">
    <meta name="size" content="304855">
    <meta name="citation_doi" content="10.1038/s41591-020-0894-4">
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41591-020-0894-4&amp;api_key=">
    <meta name="description" content="Although disinfection is key to infection control, the colonization patterns and resistomes of hospital-environment microbes remain underexplored. We report the first extensive genomic characterization of microbiomes, pathogens and antibiotic resistance cassettes in a tertiary-care hospital, from repeated sampling (up to 1.5 years apart) of 179 sites associated with 45 beds. Deep shotgun metagenomics unveiled distinct ecological niches of microbes and antibiotic resistance genes characterized by biofilm-forming and human-microbiome-influenced environments with corresponding patterns of spatiotemporal divergence. Quasi-metagenomics with nanopore sequencing provided thousands of high-contiguity genomes, phage and plasmid sequences (&amp;gt;60% novel), enabling characterization of resistome and mobilome diversity and dynamic architectures in hospital environments. Phylogenetics identified multidrug-resistant strains as being widely distributed and stably colonizing across sites. Comparisons with clinical isolates indicated that such microbes can persist in hospitals for extended periods (&amp;gt;8 years), to opportunistically infect patients. These findings highlight the importance of characterizing antibiotic resistance reservoirs in hospitals and establish the feasibility of systematic surveys to target resources for preventing infections. Spatiotemporal characterization of microbial diversity and antibiotic resistance in a tertiary-care hospital reveals broad distribution and persistence of antibiotic-resistant organisms that could cause opportunistic infections in a healthcare setting.">
    <meta name="dc.creator" content="Chng, Kern Rei">
    <meta name="dc.creator" content="Li, Chenhao">
    <meta name="dc.creator" content="Bertrand, Denis">
    <meta name="dc.creator" content="Ng, Amanda Hui Qi">
    <meta name="dc.creator" content="Kwah, Junmei Samantha">
    <meta name="dc.creator" content="Low, Hwee Meng">
    <meta name="dc.creator" content="Tong, Chengxuan">
    <meta name="dc.creator" content="Natrajan, Maanasa">
    <meta name="dc.creator" content="Zhang, Michael Hongjie">
    <meta name="dc.creator" content="Xu, Licheng">
    <meta name="dc.creator" content="Ko, Karrie Kwan Ki">
    <meta name="dc.creator" content="Ho, Eliza Xin Pei">
    <meta name="dc.creator" content="Av-Shalom, Tamar V.">
    <meta name="dc.creator" content="Teo, Jeanette Woon Pei">
    <meta name="dc.creator" content="Khor, Chiea Chuen">
    <meta name="dc.creator" content="Chen, Swaine L.">
    <meta name="dc.creator" content="Mason, Christopher E.">
    <meta name="dc.creator" content="Ng, Oon Tek">
    <meta name="dc.creator" content="Marimuthu, Kalisvar">
    <meta name="dc.creator" content="Ang, Brenda">
    <meta name="dc.creator" content="Nagarajan, Niranjan">
    <meta name="dc.subject" content="Disease prevention">
    <meta name="dc.subject" content="Microbial genetics">
    <meta name="citation_reference" content="Fischer, M.M. &amp; Bild, M. Hospital use of antibiotics as the main driver of infections with antibiotic-resistant bacteria—a reanalysis of recent data from the European Union. Preprint at bioRxiv 
                  https://doi.org/10.1101/553537
                  
                 (2019).">
    <meta name="citation_reference" content="O’Neill, J. Tackling drug-resistant infections globally: final report and recommendations. In Review on Antimicrobial Resistance (2016).">
    <meta name="citation_reference" content="citation_journal_title=Lancet; citation_title=Burden of endemic health-care-associated infection in developing countries: systematic review and meta-analysis; citation_author=B Allegranzi; citation_volume=377; citation_publication_date=2011; citation_pages=228-241; citation_doi=10.1016/S0140-6736(10)61458-4; citation_id=CR3">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Multistate point-prevalence survey of health care-associated infections; citation_author=SS Magill; citation_volume=370; citation_publication_date=2014; citation_pages=1198-1208; citation_doi=10.1056/NEJMoa1306801; citation_id=CR4">
    <meta name="citation_reference" content="citation_journal_title=Lancet Infect. Dis.; citation_title=Antibiotic resistance—the need for global solutions; citation_author=R Laxminarayan; citation_volume=13; citation_publication_date=2013; citation_pages=1057-1098; citation_doi=10.1016/S1473-3099(13)70318-9; citation_id=CR5">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Videos in clinical medicine. Hand hygiene; citation_author=Y Longtin, H Sax, B Allegranzi, F Schneider, D Pittet; citation_volume=364; citation_publication_date=2011; citation_pages=e24; citation_doi=10.1056/NEJMvcm0903599; citation_id=CR6">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Infect. Control; citation_title=Evidence that contaminated surfaces contribute to the transmission of hospital pathogens and an overview of strategies to address contaminated surfaces in hospital settings; citation_author=JA Otter, S Yezli, JA Salkeld, GL French; citation_volume=41; citation_publication_date=2013; citation_pages=S6-S11; citation_doi=10.1016/j.ajic.2012.12.004; citation_id=CR7">
    <meta name="citation_reference" content="citation_journal_title=Lancet Infect. Dis.; citation_title=An environmental cleaning bundle and health-care-associated infections in hospitals (REACH): a multicentre, randomised trial; citation_author=BG Mitchell; citation_volume=19; citation_publication_date=2019; citation_pages=410-418; citation_doi=10.1016/S1473-3099(18)30714-X; citation_id=CR8">
    <meta name="citation_reference" content="citation_journal_title=Lancet Infect. Dis.; citation_title=Transmission of Staphylococcus aureus between health-care workers, the environment, and patients in an intensive care unit: a longitudinal cohort study based on whole-genome sequencing; citation_author=JR Price; citation_volume=17; citation_publication_date=2017; citation_pages=207-214; citation_doi=10.1016/S1473-3099(16)30413-3; citation_id=CR9">
    <meta name="citation_reference" content="citation_journal_title=MBio; citation_title=Genomic analysis of hospital plumbing reveals diverse reservoir of bacterial plasmids conferring carbapenem resistance; citation_author=RA Weingarten; citation_volume=9; citation_publication_date=2018; citation_pages=e02011-17; citation_doi=10.1128/mBio.02011-17; citation_id=CR10">
    <meta name="citation_reference" content="citation_journal_title=Sci. Transl. Med.; citation_title=Single-molecule sequencing to track plasmid diversity of hospital-associated carbapenemase-producing Enterobacteriaceae; citation_author=S Conlan; citation_volume=6; citation_publication_date=2014; citation_pages=254ra126; citation_doi=10.1126/scitranslmed.3009845; citation_id=CR11">
    <meta name="citation_reference" content="citation_journal_title=Trends Mol. Med.; citation_title=Hospital-associated microbiota and implications for nosocomial infections; citation_author=S Lax, JA Gilbert; citation_volume=21; citation_publication_date=2015; citation_pages=427-432; citation_doi=10.1016/j.molmed.2015.03.005; citation_id=CR12">
    <meta name="citation_reference" content="citation_journal_title=Genome Med.; citation_title=Metagenomics for pathogen detection in public health; citation_author=RR Miller, V Montoya, JL Gardy, DM Patrick, P Tang; citation_volume=5; citation_publication_date=2013; citation_doi=10.1186/gm485; citation_id=CR13">
    <meta name="citation_reference" content="citation_journal_title=Sci. Rep.; citation_title=The ignored diversity: complex bacterial communities in intensive care units revealed by 16S pyrosequencing; citation_author=L Oberauner; citation_volume=3; citation_publication_date=2013; citation_doi=10.1038/srep01413; citation_id=CR14">
    <meta name="citation_reference" content="citation_journal_title=Sci. Transl. Med.; citation_title=Bacterial colonization and succession in a newly opened hospital; citation_author=S Lax; citation_volume=9; citation_publication_date=2017; citation_pages=eaah6500; citation_doi=10.1126/scitranslmed.aah6500; citation_id=CR15">
    <meta name="citation_reference" content="citation_journal_title=J. Clin. Microbiol.; citation_title=16S rRNA gene sequencing for bacterial identification in the diagnostic laboratory: pluses, perils, and pitfalls; citation_author=JM Janda, SL Abbott; citation_volume=45; citation_publication_date=2007; citation_pages=2761-2764; citation_doi=10.1128/JCM.01228-07; citation_id=CR16">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome; citation_author=B Brooks; citation_volume=8; citation_publication_date=2017; citation_doi=10.1038/s41467-017-02018-w; citation_id=CR17">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=Comprehensive benchmarking and ensemble approaches for metagenomic classifiers; citation_author=ABR McIntyre; citation_volume=18; citation_publication_date=2017; citation_doi=10.1186/s13059-017-1299-7; citation_id=CR18">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=Hybrid metagenomic assembly enables high-resolution analysis of resistance determinants and mobile elements in human microbiomes; citation_author=D Bertrand; citation_volume=37; citation_publication_date=2019; citation_pages=937-944; citation_doi=10.1038/s41587-019-0191-2; citation_id=CR19">
    <meta name="citation_reference" content="citation_journal_title=Microbiome; citation_title=Mobile antibiotic resistome in wastewater treatment plants revealed by nanopore metagenomic sequencing; citation_author=Y Che; citation_volume=7; citation_publication_date=2019; citation_doi=10.1186/s40168-019-0663-0; citation_id=CR20">
    <meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.; citation_title=Healthcare outbreaks associated with a water reservoir and infection prevention strategies; citation_author=H Kanamori, DJ Weber, WA Rutala; citation_volume=62; citation_publication_date=2016; citation_pages=1423-1435; citation_doi=10.1093/cid/ciw122; citation_id=CR21">
    <meta name="citation_reference" content="citation_journal_title=Lancet Infect. Dis.; citation_title=Importance of the environment in methicillin-resistant Staphylococcus aureus acquisition: the case for hospital cleaning; citation_author=SJ Dancer; citation_volume=8; citation_publication_date=2008; citation_pages=101-113; citation_doi=10.1016/S1473-3099(07)70241-4; citation_id=CR22">
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Recognizing the reagent microbiome; citation_author=MC Goffau; citation_volume=3; citation_publication_date=2018; citation_pages=851-853; citation_doi=10.1038/s41564-018-0202-y; citation_id=CR23">
    <meta name="citation_reference" content="citation_journal_title=MMWR Recomm. Rep.; citation_title=Guidelines for environmental infection control in health-care facilities. Recommendations of CDC and the Healthcare Infection Control Practices Advisory Committee (HICPAC); citation_author=L Sehulster, RY Chinn; citation_volume=52; citation_publication_date=2003; citation_pages=1-42; citation_id=CR24">
    <meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.; citation_title=The hospital water environment as a reservoir for carbapenem-resistant organisms causing hospital-acquired infections—a systematic review of the literature; citation_author=AE Kizny Gordon; citation_volume=64; citation_publication_date=2017; citation_pages=1435-1444; citation_doi=10.1093/cid/cix132; citation_id=CR25">
    <meta name="citation_reference" content="Danko, D.C. et al. Global genetic cartography of urban metagenomes and anti-microbial resistance. Preprint at bioRxiv 
                  https://doi.org/10.1101/724526
                  
                 (2019).">
    <meta name="citation_reference" content="citation_journal_title=Indoor Air; citation_title=Environmental contamination and hospital-acquired infection: factors that are easily overlooked; citation_author=C Beggs, LD Knibbs, GR Johnson, L Morawska; citation_volume=25; citation_publication_date=2015; citation_pages=462-474; citation_doi=10.1111/ina.12170; citation_id=CR27">
    <meta name="citation_reference" content="Ministry of Health Singapore. Guidelines for Control and Prevention of Multi-drug Resistant Organisms (MDROs) in Healthcare Facilities (2013).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Global spread of three multidrug-resistant lineages of Staphylococcus epidermidis; citation_author=JYH Lee; citation_volume=3; citation_publication_date=2018; citation_pages=1175-1185; citation_doi=10.1038/s41564-018-0230-7; citation_id=CR29">
    <meta name="citation_reference" content="citation_journal_title=Curr. Opin. Microbiol.; citation_title=Environmental and clinical antibiotic resistomes, same only different; citation_author=GD Wright; citation_volume=51; citation_publication_date=2019; citation_pages=57-63; citation_doi=10.1016/j.mib.2019.06.005; citation_id=CR30">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Extensive unexplored human microbiome diversity revealed by over 150,000 genomes from metagenomes spanning age, geography, and lifestyle; citation_author=E Pasolli; citation_volume=176; citation_publication_date=2019; citation_pages=649-662; citation_doi=10.1016/j.cell.2019.01.001; citation_id=CR31">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=A new genomic blueprint of the human gut microbiota; citation_author=A Almeida; citation_volume=568; citation_publication_date=2019; citation_pages=499-504; citation_doi=10.1038/s41586-019-0965-1; citation_id=CR32">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life; citation_author=DH Parks; citation_volume=36; citation_publication_date=2018; citation_pages=996-1004; citation_doi=10.1038/nbt.4229; citation_id=CR33">
    <meta name="citation_reference" content="citation_journal_title=Trends Microbiol.; citation_title=Evolution of plasmid-mediated antibiotic resistance in the clinical context; citation_author=A San Millan; citation_volume=26; citation_publication_date=2018; citation_pages=978-985; citation_doi=10.1016/j.tim.2018.06.007; citation_id=CR34">
    <meta name="citation_reference" content="citation_journal_title=Trends Microbiol.; citation_title=Bacteriophages as environmental reservoirs of antibiotic resistance; citation_author=W Calero-Caceres, M Ye, JL Balcazar; citation_volume=27; citation_publication_date=2019; citation_pages=570-577; citation_doi=10.1016/j.tim.2019.02.008; citation_id=CR35">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=PLSDB: a resource of complete bacterial plasmids; citation_author=V Galata, T Fehlmann, C Backes, A Keller; citation_volume=47; citation_publication_date=2019; citation_pages=D195-D202; citation_doi=10.1093/nar/gky1050; citation_id=CR36">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res; citation_title=MVP: a microbe–phage interaction database; citation_author=NL Gao; citation_volume=46; citation_publication_date=2018; citation_pages=D700-D707; citation_doi=10.1093/nar/gkx1124; citation_id=CR37">
    <meta name="citation_reference" content="citation_journal_title=Med. Sci. Monit.; citation_title=Relationship between the resistance genes to quaternary ammonium compounds and antibiotic resistance in staphylococci isolated from surgical site infections; citation_author=N Duran, M Temiz, GG Duran, N Eryilmaz, K Jenedi; citation_volume=20; citation_publication_date=2014; citation_pages=544-550; citation_doi=10.12659/MSM.890177; citation_id=CR38">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Investigation of a cluster of Sphingomonas koreensis infections; citation_author=RC Johnson; citation_volume=379; citation_publication_date=2018; citation_pages=2529-2539; citation_doi=10.1056/NEJMoa1803238; citation_id=CR39">
    <meta name="citation_reference" content="Fahimipour, A.K. et al. Antimicrobial chemicals associate with microbial function and antibiotic resistance indoors. mSystems 3, e00200-18 (2018).">
    <meta name="citation_reference" content="citation_journal_title=Environ. Sci. Technol.; citation_title=Antimicrobial chemicals are associated with elevated antibiotic resistance genes in the indoor dust microbiome; citation_author=EM Hartmann; citation_volume=50; citation_publication_date=2016; citation_pages=9807-9815; citation_doi=10.1021/acs.est.6b00262; citation_id=CR41">
    <meta name="citation_reference" content="citation_journal_title=Expert Opin. Investig. Drugs; citation_title=Fosfomycin for the treatment of infections caused by Gram-positive cocci with advanced antimicrobial drug resistance: a review of microbiological, animal and clinical studies; citation_author=ME Falagas, N Roussos, ID Gkegkes, PI Rafailidis, DE Karageorgopoulos; citation_volume=18; citation_publication_date=2009; citation_pages=921-944; citation_doi=10.1517/13543780902967624; citation_id=CR42">
    <meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.; citation_title=In vitro double and triple bactericidal activities of doripenem, polymyxin B, and rifampin against multidrug-resistant Acinetobacter baumannii, Pseudomonas aeruginosa, Klebsiella pneumoniae, and Escherichia coli; citation_author=C Urban, N Mariano, JJ Rahal; citation_volume=54; citation_publication_date=2010; citation_pages=2732-2734; citation_doi=10.1128/AAC.01768-09; citation_id=CR43">
    <meta name="citation_reference" content="citation_journal_title=J. Res. Pharm. Pract.; citation_title=Antibiotic regimens for treatment of infections due to multidrug-resistant Gram-negative pathogens: an evidence-based literature review; citation_author=M Izadpanah, H Khalili; citation_volume=4; citation_publication_date=2015; citation_pages=105-114; citation_doi=10.4103/2279-042X.162360; citation_id=CR44">
    <meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.; citation_title=Aminoglycoside-streptothricin resistance gene cluster aadE–sat4–aphA-3 disseminated among multiresistant isolates of Enterococcus faecium; citation_author=G Werner, B Hildebrandt, W Witte; citation_volume=45; citation_publication_date=2001; citation_pages=3267-3269; citation_doi=10.1128/AAC.45.11.3267-3269.2001; citation_id=CR45">
    <meta name="citation_reference" content="citation_journal_title=Ann. NY Acad. Sci.; citation_title=The concurrent use of sulfonamides and antibiotics in the treatment of infections: in vivo and in vitro studies of the effect of sulfonamide-antibiotic combinations on the emergence of drug resistance; citation_author=L Weinstein, TW Chang, JB Hudson, W Hartl; citation_volume=69; citation_publication_date=1957; citation_pages=408-416; citation_doi=10.1111/j.1749-6632.1957.tb49681.x; citation_id=CR46">
    <meta name="citation_reference" content="citation_journal_title=Mol. Ecol.; citation_title=Dissemination of the strA–strB streptomycin-resistance genes among commensal and pathogenic bacteria from humans, animals, and plants; citation_author=GW Sundin, CL Bender; citation_volume=5; citation_publication_date=1996; citation_pages=133-143; citation_doi=10.1111/j.1365-294X.1996.tb00299.x; citation_id=CR47">
    <meta name="citation_reference" content="citation_journal_title=Clin. Microbiol. Infect.; citation_title=Microarray-based characterisation of a Panton–Valentine leukocidin-positive community-acquired strain of methicillin-resistant Staphylococcus aureus; citation_author=S Monecke; citation_volume=12; citation_publication_date=2006; citation_pages=718-728; citation_doi=10.1111/j.1469-0691.2006.01420.x; citation_id=CR48">
    <meta name="citation_reference" content="citation_journal_title=Lancet; citation_title=First case of E. anophelis outbreak in an intensive-care unit; citation_author=J Teo; citation_volume=382; citation_publication_date=2013; citation_pages=855-856; citation_doi=10.1016/S0140-6736(13)61858-9; citation_id=CR49">
    <meta name="citation_reference" content="citation_journal_title=Epidemiol. Infect.; citation_title=Methicillin-resistant Staphylococcus aureus colonisation: epidemiological and molecular characteristics in an acute-care tertiary hospital in Singapore; citation_author=HL Htun; citation_volume=146; citation_publication_date=2018; citation_pages=1785-1792; citation_doi=10.1017/S0950268818001966; citation_id=CR50">
    <meta name="citation_reference" content="citation_journal_title=Genome Announc.; citation_title=Draft genome sequences of six multidrug-resistant clinical strains of Acinetobacter baumannii, isolated at two major hospitals in Kuwait; citation_author=K Nasser; citation_volume=6; citation_publication_date=2018; citation_pages=e00264-18; citation_doi=10.1128/genomeA.00264-18; citation_id=CR51">
    <meta name="citation_reference" content="citation_journal_title=J. Hosp. Infect.; citation_title=Hospital and community acquired infection and the built environment—design and testing of infection control rooms; citation_author=JT Walker; citation_volume=65; citation_publication_date=2007; citation_pages=43-49; citation_doi=10.1016/S0195-6701(07)60014-0; citation_id=CR52">
    <meta name="citation_reference" content="citation_journal_title=Nat. Med.; citation_title=Precision epidemiology for infectious disease control; citation_author=JT Ladner, ND Grubaugh, OG Pybus, KG Andersen; citation_volume=25; citation_publication_date=2019; citation_pages=206-211; citation_doi=10.1038/s41591-019-0345-2; citation_id=CR53">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Man-made microbial resistances in built environments; citation_author=A Mahnert; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-08864-0; citation_id=CR54">
    <meta name="citation_reference" content="citation_journal_title=Infect. Control Hosp. Epidemiol.; citation_title=Impact of an environmental cleaning intervention on the presence of methicillin-resistant Staphylococcus aureus and vancomycin-resistant enterococci on surfaces in intensive care unit rooms; citation_author=ER Goodman; citation_volume=29; citation_publication_date=2008; citation_pages=593-599; citation_doi=10.1086/588566; citation_id=CR55">
    <meta name="citation_reference" content="citation_journal_title=Can. J. Microbiol.; citation_title=Horizontal transfer of antibiotic resistance genes in clinical environments; citation_author=NA Lerminiaux, ADS Cameron; citation_volume=65; citation_publication_date=2019; citation_pages=34-44; citation_doi=10.1139/cjm-2018-0275; citation_id=CR56">
    <meta name="citation_reference" content="citation_journal_title=FEMS Microbiol. Rev.; citation_title=Strategies to combat antimicrobial resistance: anti-plasmid and plasmid curing; citation_author=MMC Buckner, ML Ciusa, LJV Piddock; citation_volume=42; citation_publication_date=2018; citation_pages=781-804; citation_doi=10.1093/femsre/fuy031; citation_id=CR57">
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Epidemic of carbapenem-resistant Klebsiella pneumoniae in Europe is driven by nosocomial spread; citation_author=S David; citation_volume=4; citation_publication_date=2019; citation_pages=1919-1929; citation_doi=10.1038/s41564-019-0492-8; citation_id=CR58">
    <meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.; citation_title=High rate of qacA- and qacB-positive methicillin-resistant Staphylococcus aureus isolates from chlorhexidine-impregnated catheter-related bloodstream infections; citation_author=CM Ho; citation_volume=56; citation_publication_date=2012; citation_pages=5693-5697; citation_doi=10.1128/AAC.00761-12; citation_id=CR59">
    <meta name="citation_reference" content="citation_journal_title=Infect. Control Hosp. Epidemiol.; citation_title=Antimicrobial-resistant pathogens associated with healthcare-associated infections: summary of data reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2011–2014; citation_author=LM Weiner; citation_volume=37; citation_publication_date=2016; citation_pages=1288-1301; citation_doi=10.1017/ice.2016.174; citation_id=CR60">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=MetaPhlAn2 for enhanced metagenomic taxonomic profiling; citation_author=DT Truong; citation_volume=12; citation_publication_date=2015; citation_pages=902-903; citation_doi=10.1038/nmeth.3589; citation_id=CR61">
    <meta name="citation_reference" content="citation_journal_title=Genome Med.; citation_title=SRST2: rapid genomic surveillance for public health and hospital microbiology labs; citation_author=M Inouye; citation_volume=6; citation_publication_date=2014; citation_doi=10.1186/s13073-014-0090-6; citation_id=CR62">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=Kraken: ultrafast metagenomic sequence classification using exact alignments; citation_author=DE Wood, SL Salzberg; citation_volume=15; citation_publication_date=2014; citation_doi=10.1186/gb-2014-15-3-r46; citation_id=CR63">
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation; citation_author=S Koren; citation_volume=27; citation_publication_date=2017; citation_pages=722-736; citation_doi=10.1101/gr.215087.116; citation_id=CR64">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=Versatile and open software for comparing large genomes; citation_author=S Kurtz; citation_volume=5; citation_publication_date=2004; citation_doi=10.1186/gb-2004-5-2-r12; citation_id=CR65">
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement; citation_author=BJ Walker; citation_volume=9; citation_publication_date=2014; citation_pages=e112963; citation_doi=10.1371/journal.pone.0112963; citation_id=CR66">
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes; citation_author=DH Parks, M Imelfort, CT Skennerton, P Hugenholtz, GW Tyson; citation_volume=25; citation_publication_date=2015; citation_pages=1043-1055; citation_doi=10.1101/gr.186072.114; citation_id=CR67">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=Mash: fast genome and metagenome distance estimation using MinHash; citation_author=BD Ondov; citation_volume=17; citation_publication_date=2016; citation_doi=10.1186/s13059-016-0997-x; citation_id=CR68">
    <meta name="citation_reference" content="citation_journal_title=F1000Res; citation_title=Large-scale sequence comparisons with sourmash; citation_author=NT Pierce, L Irber, T Reiter, P Brooks, CT Brown; citation_volume=8; citation_publication_date=2019; citation_pages=1006; citation_doi=10.12688/f1000research.19675.1; citation_id=CR69">
    <meta name="citation_reference" content="citation_journal_title=Antimicrob. Agents Chemother.; citation_title=ARG-ANNOT, a new bioinformatic tool to discover antibiotic resistance genes in bacterial genomes; citation_author=SK Gupta; citation_volume=58; citation_publication_date=2014; citation_pages=212-220; citation_doi=10.1128/AAC.01310-13; citation_id=CR70">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=PATRIC, the bacterial bioinformatics database and analysis resource; citation_author=AR Wattam; citation_volume=42; citation_publication_date=2014; citation_pages=D581-D591; citation_doi=10.1093/nar/gkt1099; citation_id=CR71">
    <meta name="citation_reference" content="citation_journal_title=PeerJ; citation_title=VirSorter: mining viral signal from microbial genomic data; citation_author=S Roux, F Enault, BL Hurwitz, MB Sullivan; citation_volume=3; citation_publication_date=2015; citation_pages=e985; citation_doi=10.7717/peerj.985; citation_id=CR72">
    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=The RAST Server: rapid annotations using subsystems technology; citation_author=RK Aziz; citation_volume=9; citation_publication_date=2008; citation_doi=10.1186/1471-2164-9-75; citation_id=CR73">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=OPERA-LG: efficient and exact scaffolding of large, repeat-rich eukaryotic genomes with performance guarantees; citation_author=S Gao, D Bertrand, BK Chia, N Nagarajan; citation_volume=17; citation_publication_date=2016; citation_doi=10.1186/s13059-016-0951-y; citation_id=CR74">
    <meta name="citation_reference" content="Gao, S., Bertrand, D., &amp; Nagarajan, N. FinIS: improved in silico finishing using an exact quadratic programming formulation. In Algorithms in Bioinformatics (Eds. Raphael, B. &amp; Tang, J.) 314–325 (Springer, 2012).">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=The Harvest suite for rapid core-genome alignment and visualization of thousands of intraspecific microbial genomes; citation_author=TJ Treangen, BD Ondov, S Koren, AM Phillippy; citation_volume=15; citation_publication_date=2014; citation_doi=10.1186/s13059-014-0524-x; citation_id=CR76">
    <meta name="citation_reference" content="citation_journal_title=Genetics; citation_title=A simple and robust statistical test for detecting the presence of recombination; citation_author=TC Bruen, H Philippe, D Bryant; citation_volume=172; citation_publication_date=2006; citation_pages=2665-2681; citation_doi=10.1534/genetics.105.048975; citation_id=CR77">
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=FastTree 2—approximately maximum-likelihood trees for large alignments; citation_author=MN Price, PS Dehal, AP Arkin; citation_volume=5; citation_publication_date=2010; citation_pages=e9490; citation_doi=10.1371/journal.pone.0009490; citation_id=CR78">
    <meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.; citation_title=Two methods for mapping and visualizing associated data on phylogeny using ggtree; citation_author=G Yu, TT Lam, H Zhu, Y Guan; citation_volume=35; citation_publication_date=2018; citation_pages=3041-3043; citation_doi=10.1093/molbev/msy194; citation_id=CR79">
    <meta name="citation_reference" content="citation_journal_title=Methods Ecol. Evol.; citation_title=iNEXT: an R package for rarefaction and extrapolation of species diversity (Hill numbers); citation_author=TC Hsieh, KH Ma, A Chao; citation_volume=7; citation_publication_date=2016; citation_pages=1451-1456; citation_doi=10.1111/2041-210X.12613; citation_id=CR80">
    <meta name="citation_author" content="Chng, Kern Rei">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Li, Chenhao">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Bertrand, Denis">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Ng, Amanda Hui Qi">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Kwah, Junmei Samantha">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Low, Hwee Meng">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Tong, Chengxuan">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Natrajan, Maanasa">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Zhang, Michael Hongjie">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Xu, Licheng">
    <meta name="citation_author_institution" content="Information Systems Technology and Design, Singapore University of Technology and Design, Singapore, Singapore">
    <meta name="citation_author" content="Ko, Karrie Kwan Ki">
    <meta name="citation_author_institution" content="Department of Microbiology, Singapore General Hospital, Singapore, Singapore">
    <meta name="citation_author_institution" content="Department of Molecular Pathology, Singapore General Hospital, Singapore, Singapore">
    <meta name="citation_author_institution" content="Duke-NUS Graduate Medical School, Singapore, Singapore">
    <meta name="citation_author" content="Ho, Eliza Xin Pei">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Av-Shalom, Tamar V.">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Teo, Jeanette Woon Pei">
    <meta name="citation_author_institution" content="Department of Laboratory Medicine, National University Hospital, Singapore, Singapore">
    <meta name="citation_author" content="Khor, Chiea Chuen">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Chen, Swaine L.">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Mason, Christopher E.">
    <meta name="citation_author_institution" content="Department of Physiology and Biophysics, Weill Cornell Medicine, New York, USA">
    <meta name="citation_author" content="Ng, Oon Tek">
    <meta name="citation_author_institution" content="National Centre for Infectious Diseases, Singapore, Singapore">
    <meta name="citation_author_institution" content="Department of Infectious Diseases, Tan Tock Seng Hospital, Singapore, Singapore">
    <meta name="citation_author_institution" content="Lee Kong Chian School of Medicine, Nanyang Technological University, Singapore, Singapore">
    <meta name="citation_author" content="Marimuthu, Kalisvar">
    <meta name="citation_author_institution" content="National Centre for Infectious Diseases, Singapore, Singapore">
    <meta name="citation_author_institution" content="Department of Infectious Diseases, Tan Tock Seng Hospital, Singapore, Singapore">
    <meta name="citation_author_institution" content="Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore">
    <meta name="citation_author" content="Ang, Brenda">
    <meta name="citation_author_institution" content="National Centre for Infectious Diseases, Singapore, Singapore">
    <meta name="citation_author_institution" content="Department of Infectious Diseases, Tan Tock Seng Hospital, Singapore, Singapore">
    <meta name="citation_author" content="Nagarajan, Niranjan">
    <meta name="citation_author_institution" content="Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore">
    <meta name="citation_author_institution" content="Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore">
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">
    <meta name="twitter:site" content="@naturemedicine">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image:alt" content="Content cover image">
    <meta name="twitter:title" content="Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment">
    <meta name="twitter:description" content="Nature Medicine - Spatiotemporal characterization of microbial diversity and antibiotic resistance in a tertiary-care hospital reveals broad distribution and persistence of antibiotic-resistant...">
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig1_HTML.png">
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41591-020-0894-4">
    <meta property="og:type" content="article">
    <meta property="og:site_name" content="Nature">
    <meta property="og:title" content="Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment - Nature Medicine">
    <meta property="og:description" content="Spatiotemporal characterization of microbial diversity and antibiotic resistance in a tertiary-care hospital reveals broad distribution and persistence of antibiotic-resistant organisms that could cause opportunistic infections in a healthcare setting.">
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig1_HTML.png">
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
<style type="text/css">.cc-banner{background-color:#01324b;border:none!important;bottom:0;box-sizing:border-box;color:#fff!important;left:0;line-height:1.3;margin:auto 0 0;max-width:100%;outline:0;overflow:visible;padding:0;position:fixed;right:0;width:100%;z-index:99999}.cc-banner::backdrop{background-color:#0000004d}.cc-banner *{color:inherit!important}.cc-banner:focus{box-shadow:none;outline:0}.cc-banner a{color:#fff!important;text-decoration:underline}.cc-banner a:active,.cc-banner a:focus,.cc-banner a:hover{color:inherit;text-decoration:none}.cc-banner a:focus{outline:3px solid #08c!important}.cc-banner h2,.cc-banner h3,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-banner .cc-h2,.cc-banner h2{font-size:18px}.cc-banner .cc-h3,.cc-banner h3{font-size:16px}.cc-banner .cc-h4,.cc-banner .cc-h5,.cc-banner .cc-h6,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-size:14px}.cc-banner .cc-button{font-size:16px}.cc-banner__content{background-color:#01324b;display:flex;flex-direction:column;margin:0 auto;max-height:90vh;max-width:100%;padding:16px;position:relative}.cc-banner__content:focus{outline:0}@media (min-width:680px){.cc-banner__content{padding:12px}}@media (min-width:980px){.cc-banner__content{max-height:60vh;padding-bottom:20px;padding-top:20px}}@media (min-width:1320px){.cc-banner__content{padding-bottom:40px;padding-top:40px}}.cc-banner__container{display:flex;flex-direction:column;margin:auto;max-width:1320px;overflow:auto}.cc-banner__title{background:none!important;border:0;flex-shrink:0;font-size:18px!important;font-size:22px!important;font-weight:700!important;letter-spacing:normal;margin:0 0 12px!important;padding:0}@media (min-width:680px){.cc-banner__title{font-size:24px!important;margin:0 0 16px!important}}@media (min-width:1320px){.cc-banner__title{font-size:26px!important;margin:0 0 24px!important}}.cc-banner__body{display:flex;flex-direction:column;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif!important;overflow-x:hidden;overflow-y:auto;padding:3px 3px 16px}@media (min-width:980px){.cc-banner__body{flex-direction:row}}.cc-banner__policy p{font-size:16px!important;margin:0;max-width:none;padding:0}.cc-banner__policy p:not(:last-child){margin:0 0 16px}@media (min-width:980px){.cc-banner__policy p:not(:last-child){margin:0 0 24px}}.cc-banner__policy p a{font-size:16px!important;font-weight:700}.cc-banner__footer{box-shadow:none;flex-shrink:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif !important!important;margin:0;padding:12px 0 3px 3px;position:relative}@media (min-width:380px){.cc-banner__footer{align-items:stretch;display:flex;flex-wrap:wrap}}@media (min-width:680px){.cc-banner__footer{box-shadow:none;flex-wrap:nowrap;width:40%}}.cc-banner__footer .cc-banner__button{flex:1 1 auto;font-weight:700;overflow:hidden;padding:.5em 1em;width:100%}@media (min-width:680px){.cc-banner__footer .cc-banner__button{max-width:275px}}.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:12px}@media (min-width:680px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:0;margin-right:16px}}@media (min-width:980px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-right:24px}}@media (min-width:680px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:16px!important;margin-right:0!important}}@media (min-width:980px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:24px!important;margin-right:0!important}}.cc-banner__button-preferences{padding-left:0;padding-right:0}@media (min-width:380px){.cc-banner__button-preferences{flex:0 0 auto;margin:auto}}@media (min-width:680px){.cc-banner__button-preferences{margin:0}}@media (min-width:380px) and (max-width:680px){.cc-banner__button-break{display:block}}@media (min-width:680px){.cc-banner--is-tcf .cc-banner__footer{width:auto}}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__footer{padding-right:48px;width:66%}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px!important}.cc-banner--is-tcf .cc-banner__stacks{margin:16px 0 0}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__stacks{display:flex;flex:0 0 33%;flex-direction:column;margin:0 0 0 48px;overflow:auto}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px;margin:0 0 12px}.cc-banner--is-tcf .cc-banner__stacks-details{padding:0 3px}.cc-banner--is-tcf .cc-banner .cc-details{border-color:#fff3!important}.cc-box{border-radius:10px;padding:12px}.cc-box--info{background-color:#eff6fb}.cc-box--light{background-color:#faf9f6}.cc-button{border:2px solid #0000;border-radius:32px!important;color:inherit;cursor:pointer;font-size:14px;font-weight:700!important;left:auto;line-height:1.2;margin:0;padding:.5em 1em;right:auto;text-transform:none!important;transition:all .2s}@media (min-width:680px){.cc-button{font-size:14px;padding:.75em 1em}}.cc-button--sm{font-size:12px;padding:2px 8px!important}.cc-button--primary{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--primary:focus{outline:3px solid #08c}.cc-button--primary:focus,.cc-button--primary:hover,.cc-button--secondary{background-color:#fff!important;border-color:#025e8d!important;color:#025e8d!important}.cc-button--secondary:focus{background-color:#025e8d!important;border-color:#fff!important;color:#fff!important;outline:3px solid #08c!important}.cc-button--secondary:hover{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--secondary:active{border:2px solid #01324b!important}.cc-button--secondary.cc-button--contrast{border-color:#fff!important}.cc-button--secondary.cc-button--contrast:hover{background-color:initial!important;color:#fff!important}.cc-button--tertiary{background-color:#f0f7fc!important;background-image:none;border:2px solid #f0f7fc;color:#025e8d!important;text-decoration:underline!important;text-underline-offset:.25em}.cc-button--tertiary:focus{outline:3px solid #08c!important}.cc-button--tertiary:hover{text-decoration-thickness:.25em}.cc-button--tertiary:active{color:#013c5b}.cc-button--link{background-color:initial!important;border-color:#0000!important;border-radius:0!important;color:inherit!important;padding:0!important;text-decoration:underline!important;width:inherit!important}.cc-button--link:focus{outline:3px solid #08c}.cc-button--link:hover{background-color:initial!important;box-shadow:none;text-decoration:none}.cc-button--text{border-radius:0;padding:0}.cc-button--details{padding-right:24px!important;position:relative;width:auto}.cc-button--details:after,.cc-button--details:before{background-color:currentColor;content:"";position:absolute;transition:transform .25s ease-out}.cc-button--details:before{height:10px;margin-top:-5px;right:12px;top:50%;width:2px}.cc-button--details:after{height:2px;margin-top:-1px;right:8px;top:50%;width:10px}.cc-button--details.cc-active:before{transform:rotate(90deg);transform-origin:center}.cc-button--details.cc-active:after{display:none}.cc-details{border-bottom:1px solid #0000001a;padding:12px 0;position:relative;width:100%}.cc-details__summary{align-items:center;display:flex;font-size:14px;font-weight:700;list-style-type:none}.cc-details__summary:focus,.cc-details__summary:focus-visible{outline:3px solid #08c!important;will-change:auto}.cc-details__title{align-items:baseline;display:flex}.cc-details__title h2,.cc-details__title h3,.cc-details__title h4,.cc-details__title h5{line-height:1.4;margin:0!important;padding:0!important}.cc-details__title svg{flex-shrink:0;height:auto;margin-right:8px;position:relative;top:-1px;transition:all .2s;width:auto}.cc-details[open] .cc-details__title svg{top:2px;transform:rotate(90deg)}.cc-details__switch{margin-left:auto}.cc-details__section{padding:16px 0 0 18px}.cc-details__section p{margin:0}.cc-details__section p:not(:last-child){margin-bottom:12px}details summary::-webkit-details-marker{display:none}.cc-radio{align-items:center;display:flex;position:relative}.cc-radio *{cursor:pointer}.cc-radio__input{height:22px;left:0;position:absolute;top:0;width:22px}.cc-radio__input:focus{outline:none}.cc-radio__label{color:inherit;font-size:14px;font-weight:700;line-height:23px;margin:0;padding-left:28px}.cc-radio__label:after,.cc-radio__label:before{background-color:#fff;content:"";display:block;position:absolute;transition:transform .25s ease-out}.cc-radio__label:before{border:1px solid #777;border-radius:50%;height:22px;left:0;top:0;width:22px}.cc-radio__label:after{border:7px solid #025e8d;border-radius:50%;height:0;left:4px;opacity:0;top:4px;width:0}.cc-radio__input:focus+.cc-radio__label:before{box-shadow:0 0 0 2px #08c;outline:none}.cc-radio__label--hidden{display:none}.cc-radio__input:checked+.cc-radio__label:after{opacity:1}.cc-radio__input:disabled{cursor:default}.cc-radio__input:disabled+.cc-radio__label{cursor:default;opacity:.5}.cc-switch *{cursor:pointer}.cc-switch{align-items:center;display:flex;line-height:1}.cc-switch__label{cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;-webkit-user-select:none;user-select:none}.cc-switch__input[type=checkbox]{-webkit-appearance:none;appearance:none;background-color:#fff!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='rgba(0, 0, 0, 0.25)'/%3E%3C/svg%3E")!important;background-position:0!important;background-repeat:no-repeat!important;border:1px solid #777!important;border-radius:16px!important;height:18px!important;margin:0 8px 0 0!important;transition:background-position .2s ease-in-out!important;width:32px!important}.cc-switch__input[type=checkbox]:focus{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%2380b3cc'/%3E%3C/svg%3E")!important}.cc-switch__input[type=checkbox]:checked{background-color:#025e8d!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%23fff'/%3E%3C/svg%3E")!important;background-position:100%!important}.cc-switch__input[type=checkbox]:checked~.cc-switch__label>.cc-switch__text-off,.cc-switch__input[type=checkbox]:not(:checked)~.cc-switch__label>.cc-switch__text-on{display:none}.cc-switch__input[type=checkbox]:hover{cursor:pointer;outline:none}.cc-switch__input[type=checkbox]:focus-visible{cursor:pointer;outline:3px solid #fc0}.cc-switch__input[type=checkbox]:checked:before{background-color:none!important;color:#0000!important}html[dir=rtl] .cc-switch__input[type=checkbox]{margin-left:8px!important;margin-right:0!important}.cc-list>ul,ul.cc-list{list-style-type:disc;margin-left:0;padding-left:0}.cc-list>ul>li,ul.cc-list>li{margin-left:1em}.cc-list>ul>li:not(:last-child),ul.cc-list>li:not(:last-child){margin-bottom:4px}.cc-list__title{margin-bottom:8px!important}.cc-list--inline>ul,ul.cc-list--inline{display:flex;flex-wrap:wrap;list-style-type:none}.cc-list--inline>ul>li,ul.cc-list--inline>li{margin:0 .5em .5em 0}.cc-overlay{background-color:#0000004d;z-index:99998}.cc-overlay,.cc-preferences{bottom:0;left:0;position:fixed;right:0;top:0}.cc-preferences{background-color:#050a14f2!important;border:0;box-sizing:border-box;color:#111;font-family:sans-serif!important;line-height:1.4;margin:auto;max-height:100vh;overflow:auto;padding:0;z-index:100000}.cc-preferences:focus{outline:none}.cc-preferences *,.cc-preferences :after,.cc-preferences :before{box-sizing:inherit!important}.cc-preferences h2,.cc-preferences h3,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-preferences .cc-h2,.cc-preferences h2{font-size:18px}.cc-preferences .cc-h3,.cc-preferences h3{font-size:16px}.cc-preferences .cc-h4,.cc-preferences .cc-h5,.cc-preferences .cc-h6,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-size:14px}.cc-preferences a{color:#025e8d;text-decoration:underline}.cc-preferences a:hover{color:inherit;text-decoration:none}.cc-preferences h3{background:none;color:#111;padding:0;text-transform:none}dialog.cc-preferences{background-color:initial}dialog.cc-preferences::backdrop{background-color:#000000e6}.cc-preferences__dialog{display:flex;flex-direction:column;margin:auto;max-height:100vh;max-width:860px;padding:12px;position:relative}.cc-preferences__dialog>:last-child{border-bottom-left-radius:10px;border-bottom-right-radius:10px}@media (min-width:980px){.cc-preferences__dialog{padding:16px}}.cc-preferences__close{background:#0000!important;border:1px solid #ececec;border-radius:50%;color:#111!important;cursor:pointer;font-family:Times New Roman,serif;font-size:40px;height:40px;left:auto;line-height:1;margin-top:-20px;padding:0!important;position:absolute;right:20px;top:50%;width:40px}.cc-preferences__close:focus{outline:3px solid #08c}.cc-preferences__close-label{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__header{background:#fff!important;border-bottom:1px solid #0000001a;border-top-left-radius:10px;border-top-right-radius:10px;padding:16px;position:relative;text-align:center}.cc-preferences__title{background:none!important;color:#111!important;font-family:sans-serif!important;font-size:18px!important;font-weight:700!important;margin:0!important;padding:0 16px 0 0!important}@media (min-width:480px){.cc-preferences__title{padding-right:0!important}}@media (min-width:980px){.cc-preferences__title{font-size:22px!important}}.cc-preferences__body{background:#fff!important;flex:1 1 auto;min-height:200px;overflow-x:hidden;overflow-y:auto;padding:16px}.cc-preferences__footer{background:#fff!important;border-top:1px solid #d0d0d0;box-shadow:0 0 5px 0 #0003;margin-bottom:0;padding:12px;position:relative}@media (min-width:480px){.cc-preferences__footer{align-items:stretch;display:flex}}.cc-preferences__footer>.cc-button{display:block;width:100%}@media (min-width:480px){.cc-preferences__footer>.cc-button{flex:1 1 auto}}@media (min-width:980px){.cc-preferences__footer>.cc-button{flex-basis:auto}}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:first-child){margin-left:12px}}.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:8px}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:0}}.cc-preferences__categories{list-style:none;margin:0;padding:0}.cc-preferences__category:not(:last-child){border-bottom:1px solid #0000001a;margin-bottom:12px;padding-bottom:12px}.cc-preferences__category-description{font-size:14px;margin:0 0 8px;padding:0}.cc-preferences__category-footer{align-items:center;display:flex;justify-content:space-between}.cc-preferences__status{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__controls{display:flex;margin:0}.cc-preferences__controls>:not(:last-child){margin-right:12px}.cc-preferences__always-on{font-size:14px;font-weight:700;padding-left:26px;position:relative}.cc-preferences__always-on:before{background:#0000!important;border:solid;border-top-color:#0000;border-width:0 0 4px 4px;content:"";display:block;height:10px;left:0;position:absolute;top:2px;transform:rotate(-45deg);width:18px}.cc-preferences__details{background-color:#eff6fb!important;border-radius:10px;font-size:14px;margin:12px 0 0;padding:12px}.cc-preferences__cookie-list,.cc-preferences__provider-list{list-style:none;margin:0;padding:0}.cc-preferences__provider-list{columns:170px}.cc-preferences__cookie-title{font-size:1em;font-style:normal;font-weight:400;margin:0}.cc-preferences__cookie-description{font-size:1em;margin:0 0 8px}.cc-preferences__cookie-domain,.cc-preferences__cookie-lifespan{border-left:1px solid #999;color:#666;margin-left:8px;padding-left:4px}body.cc-has-preferences-open{overflow:hidden;position:relative}.cc-table{border-collapse:collapse;width:100%}.cc-table tbody tr{border-top:1px solid #0000001a}.cc-table td,.cc-table th{font-size:14px;padding:4px 8px;vertical-align:top}.cc-table+.cc-table{margin-top:8px}.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:70%}@media (min-width:680px){.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:80%}}.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{padding-left:8px;width:40%}@media (min-width:680px){.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{width:30%}}.cc-tabs,.cc-tabs>ul{background-color:#faf9f6;display:flex;list-style:none;margin:0;padding:0}.cc-tabs__panel{border-top:1px solid #ececec;margin-top:-1px}.cc-tabs__button{background-color:initial;border:0;border-left:1px solid #0000;border-right:1px solid #0000;color:#111;font-size:16px;font-weight:700;margin:0;padding:15px 16px 12px;position:relative}.cc-tabs__button:before{background-color:initial;content:"";display:block;height:3px;left:0;position:absolute;top:0;width:100%}.cc-tabs__button.cc-active{background-color:#fff;border-color:#025e8d #ececec #ececec;color:#111}.cc-tabs__button.cc-active:before{background-color:#025e8d}.cc-tabs__content{background-color:#fff;display:none;font-size:14px;padding:16px}.cc-tabs__section:not(:first-child){padding-top:24px}.cc-tcf{font-size:14px;margin-top:16px}.cc-tcf__list{list-style:none;margin:0;padding:0}.cc-vendor-count{color:#666;font-size:16px;margin:0 0 8px}.cc-hide{display:none!important}.cc-show{display:block!important}.cc-external-link{background-color:#ececec;background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTcgMGExIDEgMCAxIDEgMCAySDIuNmMtLjM3MSAwLS42LjIwOS0uNi41djE1YzAgLjI5MS4yMjkuNS42LjVoMTQuOGMuMzcxIDAgLjYtLjIwOS42LS41VjEzYTEgMSAwIDAgMSAyIDB2NC41YzAgMS40MzgtMS4xNjIgMi41LTIuNiAyLjVIMi42QzEuMTYyIDIwIDAgMTguOTM4IDAgMTcuNXYtMTVDMCAxLjA2MiAxLjE2MiAwIDIuNiAwem02IDBoNmwuMDc1LjAwMy4xMjYuMDE3LjExMS4wMy4xMTEuMDQ0LjA5OC4wNTIuMDk2LjA2Ny4wOS4wOGExIDEgMCAwIDEgLjA5Ny4xMTJsLjA3MS4xMS4wNTQuMTE0LjAzNS4xMDUuMDMuMTQ4TDIwIDF2NmExIDEgMCAwIDEtMiAwVjMuNDE0bC02LjY5MyA2LjY5M2ExIDEgMCAwIDEtMS40MTQtMS40MTRMMTYuNTg0IDJIMTNhMSAxIDAgMCAxLS45OTMtLjg4M0wxMiAxYTEgMSAwIDAgMSAxLTEiLz48L3N2Zz4=");background-position:right 8px center;background-repeat:no-repeat;background-size:10px auto;border-radius:3px;box-shadow:0 1px 0 #0003;color:#111!important;display:inline-block;font-size:12px;padding:4px 26px 4px 8px;text-decoration:none!important}.cc-external-link:hover{text-decoration:underline!important}.cc-m-0{margin:0!important}.cc-grey{color:#666}</style><script async="" type="text/javascript" src="https://injections.readcube.com/nature/inject.fa744f3f.js"></script></head>
<body class="article-page"><dialog class="cc-banner" data-cc-banner="" data-nosnippet="" aria-labelledby="cc-banner-label" open="">
		<div class="cc-banner__content" autofocus="" tabindex="-1">
			<div class="cc-banner__container">
				<div class="cc-banner__header">
					<h2 class="cc-banner__title" id="cc-banner-label">Your privacy, your choice</h2>
				</div>
				<div class="cc-banner__body">
					<div class="cc-banner__policy">
						<p>We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.</p><p>By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.</p><p>See our <a href="https://www.nature.com/info/privacy" data-cc-action="privacy">privacy policy</a> for more information on the use of your personal data.</p><p><button type="button" data-cc-action="preferences" class="cc-button cc-button--link cc-button--text">Manage preferences</button> for further information and to change your choices.</p>
					</div>
					
				</div>
				<div class="cc-banner__footer">
					
			<button data-cc-action="accept" class="cc-button cc-button--secondary cc-button--contrast cc-banner__button cc-banner__button-accept">Accept all cookies</button>
			
		
				</div>
			</div>
		</div>
	</dialog>

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90" style="display: none !important;">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-test="top-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41591-020-0894-4;doi=10.1038/s41591-020-0894-4;subjmeta=208,325,459,631,692,700;kwrd=Disease+prevention,Microbial+genetics">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=704620909&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-020-0894-4%26doi%3D10.1038/s41591-020-0894-4%26subjmeta%3D208,325,459,631,692,700%26kwrd%3DDisease+prevention,Microbial+genetics">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=704620909&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-020-0894-4%26doi%3D10.1038/s41591-020-0894-4%26subjmeta%3D208,325,459,631,692,700%26kwrd%3DDisease+prevention,Microbial+genetics"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header="" data-track-component="nature-150-split-header" style="border-color:#e40428">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nm" data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" height="32" alt="Nature Medicine">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
                            </a><div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected="">All journals</option>
                                    <option value="nm">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41591-020-0894-4"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header" hidden="">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-articles" data-track="click" data-track-action="research articles" data-track-label="link" data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-and-analysis" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link" data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/podcast" data-track="click" data-track-action="podcasts" data-track-label="link" data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/current-issue" data-track="click" data-track-action="current issue" data-track-label="link" data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/collections" data-track="click" data-track-action="collections" data-track-label="link" data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://www.facebook.com/Nature-Medicine-193691346949/" data-track="click" data-track-action="facebook" data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://twitter.com/naturemedicine" data-track="click" data-track-action="twitter" data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=5" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/aims" data-track="click" data-track-action="aims &amp; scope" data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-information" data-track="click" data-track-action="journal information" data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-impact" data-track="click" data-track-action="journal metrics" data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editors" data-track="click" data-track-action="about the editors" data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-cross-journal-editorial-team" data-track="click" data-track-action="research cross-journal editorial team" data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-cross-journal-editorial-team" data-track="click" data-track-action="reviews cross-journal editorial team" data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/statistics-advisory-panel" data-track="click" data-track-action="statistical advisory panel" data-track-label="link">
                                    Statistical Advisory Panel
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/our-publishing-models" data-track="click" data-track-action="our publishing models" data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-values-statement" data-track="click" data-track-action="editorial values statement" data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-policies" data-track="click" data-track-action="editorial policies" data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/content" data-track="click" data-track-action="content types" data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="javascript:;" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                        </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/submission-guidelines" data-track="click" data-track-action="submission guidelines" data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-context="header publish with us dropdown menu" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/open-access-funding" data-test="funding-eligibility-link" data-track="click_explore_funding" data-track-context="header publish with us" data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link" href="https://mts-nmed.nature.com/cgi-bin/main.plex" data-track="click_submit_manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-action="submit manuscript" data-track-label="link (publish with us dropdown menu)" data-track-external="" data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link" href="https://journal-alerts.springernature.com/subscribe?journal_id=41591" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-label="link (desktop site header)" data-track-external="">
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature medicine"><span itemprop="name">nature medicine</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm/articles?type=article" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" id="js-enable-context-bar" data-test="context-bar" data-context-bar="" aria-hidden="true">
                    <div class="c-context-bar__container u-container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment
                        </div>
                        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-020-0894-4.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-020-0894-4.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2020-06-08">08 June 2020</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment</h1>
                    <ul class="c-article-author-list c-article-author-list--short js-no-scroll" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kern_Rei-Chng-Aff1" data-author-popup="auth-Kern_Rei-Chng-Aff1" data-author-search="Chng, Kern Rei" data-track-context="researcher popup with no profile" data-track-index="1_22">Kern Rei Chng</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Chenhao-Li-Aff1" data-author-popup="auth-Chenhao-Li-Aff1" data-author-search="Li, Chenhao" data-track-context="researcher popup with no profile" data-track-index="2_22">Chenhao Li</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-1182-6804"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1182-6804</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Denis-Bertrand-Aff1" data-author-popup="auth-Denis-Bertrand-Aff1" data-author-search="Bertrand, Denis" data-track-context="researcher popup with no profile" data-track-index="3_22">Denis Bertrand</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Amanda_Hui_Qi-Ng-Aff1" data-author-popup="auth-Amanda_Hui_Qi-Ng-Aff1" data-author-search="Ng, Amanda Hui Qi" data-track-context="researcher popup with no profile" data-track-index="4_22">Amanda Hui Qi Ng</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Junmei_Samantha-Kwah-Aff1" data-author-popup="auth-Junmei_Samantha-Kwah-Aff1" data-author-search="Kwah, Junmei Samantha" data-track-context="researcher popup with no profile" data-track-index="5_22">Junmei Samantha Kwah</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Hwee_Meng-Low-Aff1" data-author-popup="auth-Hwee_Meng-Low-Aff1" data-author-search="Low, Hwee Meng" data-track-context="researcher popup with no profile" data-track-index="6_22">Hwee Meng Low</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Chengxuan-Tong-Aff1" data-author-popup="auth-Chengxuan-Tong-Aff1" data-author-search="Tong, Chengxuan" data-track-context="researcher popup with no profile" data-track-index="7_22">Chengxuan Tong</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Maanasa-Natrajan-Aff1" data-author-popup="auth-Maanasa-Natrajan-Aff1" data-author-search="Natrajan, Maanasa" data-track-context="researcher popup with no profile" data-track-index="8_22">Maanasa Natrajan</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael_Hongjie-Zhang-Aff1" data-author-popup="auth-Michael_Hongjie-Zhang-Aff1" data-author-search="Zhang, Michael Hongjie" data-track-context="researcher popup with no profile" data-track-index="9_22">Michael Hongjie Zhang</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Licheng-Xu-Aff2" data-author-popup="auth-Licheng-Xu-Aff2" data-author-search="Xu, Licheng" data-track-context="researcher popup with no profile" data-track-index="10_22">Licheng Xu</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Karrie_Kwan_Ki-Ko-Aff3-Aff4-Aff5" data-author-popup="auth-Karrie_Kwan_Ki-Ko-Aff3-Aff4-Aff5" data-author-search="Ko, Karrie Kwan Ki" data-track-context="researcher popup with no profile" data-track-index="11_22">Karrie Kwan Ki Ko</a><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff5" tabindex="-1">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Eliza_Xin_Pei-Ho-Aff1" data-author-popup="auth-Eliza_Xin_Pei-Ho-Aff1" data-author-search="Ho, Eliza Xin Pei" data-track-context="researcher popup with no profile" data-track-index="12_22">Eliza Xin Pei Ho</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tamar_V_-Av_Shalom-Aff1" data-author-popup="auth-Tamar_V_-Av_Shalom-Aff1" data-author-search="Av-Shalom, Tamar V." data-track-context="researcher popup with no profile" data-track-index="13_22">Tamar V. Av-Shalom</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jeanette_Woon_Pei-Teo-Aff6" data-author-popup="auth-Jeanette_Woon_Pei-Teo-Aff6" data-author-search="Teo, Jeanette Woon Pei" data-track-context="researcher popup with no profile" data-track-index="14_22">Jeanette Woon Pei Teo</a><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Chiea_Chuen-Khor-Aff1" data-author-popup="auth-Chiea_Chuen-Khor-Aff1" data-author-search="Khor, Chiea Chuen" data-track-context="researcher popup with no profile" data-track-index="15_22">Chiea Chuen Khor</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-1128-4729"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-1128-4729</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-author-popup="group-1" href="#group-1" data-track-context="researcher popup with no profile" data-track-index="16_22">MetaSUB Consortium</a>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Swaine_L_-Chen-Aff1" data-author-popup="auth-Swaine_L_-Chen-Aff1" data-author-search="Chen, Swaine L." data-track-context="researcher popup with no profile" data-track-index="17_22">Swaine L. Chen</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Christopher_E_-Mason-Aff7" data-author-popup="auth-Christopher_E_-Mason-Aff7" data-author-search="Mason, Christopher E." data-track-context="researcher popup with no profile" data-track-index="18_22">Christopher E. Mason</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-1850-1642"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-1850-1642</a></span><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Oon_Tek-Ng-Aff8-Aff9-Aff10" data-author-popup="auth-Oon_Tek-Ng-Aff8-Aff9-Aff10" data-author-search="Ng, Oon Tek" data-track-context="researcher popup with no profile" data-track-index="19_22">Oon Tek Ng</a><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff9" tabindex="-1">9</a>,<a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kalisvar-Marimuthu-Aff8-Aff9-Aff11" data-author-popup="auth-Kalisvar-Marimuthu-Aff8-Aff9-Aff11" data-author-search="Marimuthu, Kalisvar" data-track-context="researcher popup with no profile" data-track-index="20_22">Kalisvar Marimuthu</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-0395-5633"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0395-5633</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff9" tabindex="-1">9</a>,<a href="#Aff11" tabindex="-1">11</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Brenda-Ang-Aff8-Aff9" data-author-popup="auth-Brenda-Ang-Aff8-Aff9" data-author-search="Ang, Brenda" data-track-context="researcher popup with no profile" data-track-index="21_22">Brenda Ang</a><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff9" tabindex="-1">9</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 22 authors for this article" title="Show all 22 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Niranjan-Nagarajan-Aff1-Aff11" data-author-popup="auth-Niranjan-Nagarajan-Aff1-Aff11" data-author-search="Nagarajan, Niranjan" data-corresp-id="c1" data-track-context="researcher popup with no profile" data-track-index="22_22">Niranjan Nagarajan<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-0850-5604"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0850-5604</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff11" tabindex="-1">11</a></sup>&nbsp;</li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nm" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Medicine</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;26</b>,&nbsp;<span class="u-visually-hidden">pages </span>941–951 (<span data-test="article-publication-year">2020</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item" data-test="access-count">
                        <p class="c-article-metrics-bar__count">35k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item" data-test="citation-count">
                        <p class="c-article-metrics-bar__count">158 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item" data-test="altmetric-score">
                            <p class="c-article-metrics-bar__count">267 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41591-020-0894-4/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Although disinfection is key to infection control, the colonization patterns and resistomes of hospital-environment microbes remain underexplored. We report the first extensive genomic characterization of microbiomes, pathogens and antibiotic resistance cassettes in a tertiary-care hospital, from repeated sampling (up to 1.5 years apart) of 179 sites associated with 45 beds. Deep shotgun metagenomics unveiled distinct ecological niches of microbes and antibiotic resistance genes characterized by biofilm-forming and human-microbiome-influenced environments with corresponding patterns of spatiotemporal divergence. Quasi-metagenomics with nanopore sequencing provided thousands of high-contiguity genomes, phage and plasmid sequences (&gt;60% novel), enabling characterization of resistome and mobilome diversity and dynamic architectures in hospital environments. Phylogenetics identified multidrug-resistant strains as being widely distributed and stably colonizing across sites. Comparisons with clinical isolates indicated that such microbes can persist in hospitals for extended periods (&gt;8 years), to opportunistically infect patients. These findings highlight the importance of characterizing antibiotic resistance reservoirs in hospitals and establish the feasibility of systematic surveys to target resources for preventing infections.</p></div></div></section>

            
                
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41564-022-01221-w/MediaObjects/41564_2022_1221_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41564-022-01221-w?fromPaywallRec=false" data-track="select_recommendations_1" data-track-context="inline recommendations" data-track-action="click recommendations inline - 1" data-track-label="10.1038/s41564-022-01221-w">Long-term ecological and evolutionary dynamics in the gut microbiomes of carbapenemase-producing Enterobacteriaceae colonized subjects
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">15 September 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs42003-022-04187-x/MediaObjects/42003_2022_4187_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s42003-022-04187-x?fromPaywallRec=false" data-track="select_recommendations_2" data-track-context="inline recommendations" data-track-action="click recommendations inline - 2" data-track-label="10.1038/s42003-022-04187-x">Screening of global microbiomes implies ecological boundaries impacting the distribution and dissemination of clinically relevant antimicrobial resistance genes
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">18 November 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs43856-022-00124-5/MediaObjects/43856_2022_124_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s43856-022-00124-5?fromPaywallRec=false" data-track="select_recommendations_3" data-track-context="inline recommendations" data-track-action="click recommendations inline - 3" data-track-label="10.1038/s43856-022-00124-5">Antibiotic-resistant organisms establish reservoirs in new hospital built environments and are related to patient blood infection isolates
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">01 June 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1749514602,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>The global epidemic of antibiotic resistance has refocused attention on infection prevention and control in hospitals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Fischer, M.M. &amp; Bild, M. Hospital use of antibiotics as the main driver of infections with antibiotic-resistant bacteria—a reanalysis of recent data from the European Union. Preprint at bioRxiv 
                  https://doi.org/10.1101/553537
                  
                 (2019)." href="/articles/s41591-020-0894-4#ref-CR1" id="ref-link-section-d56841402e2641">1</a></sup>. It is estimated that if the spread of antibiotic resistance grows unchecked, it will cause millions of deaths worldwide, with an economic impact of more than US$100 trillion by 2050 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="O’Neill, J. Tackling drug-resistant infections globally: final report and recommendations. In Review on Antimicrobial Resistance (2016)." href="/articles/s41591-020-0894-4#ref-CR2" id="ref-link-section-d56841402e2645">2</a></sup>). Hospital-acquired infections (HAIs) pose a high healthcare burden in both developed and developing countries<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Allegranzi, B. et al. Burden of endemic health-care-associated infection in developing countries: systematic review and meta-analysis. Lancet 377, 228–241 (2011)." href="/articles/s41591-020-0894-4#ref-CR3" id="ref-link-section-d56841402e2649">3</a></sup>. US estimates highlight that 1 in 25 acute-care patients have active HAIs daily (721,800 HAIs each year), with 11.5% of patients dying during hospitalization<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Magill, S. S. et al. Multistate point-prevalence survey of health care-associated infections. N. Engl. J. Med. 370, 1198–1208 (2014)." href="/articles/s41591-020-0894-4#ref-CR4" id="ref-link-section-d56841402e2653">4</a></sup>. The problem of HAIs is further compounded by the global spread of multidrug-resistant organisms (MDROs), complicating infection management, limiting therapy options and resulting in poorer outcomes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Laxminarayan, R. et al. Antibiotic resistance—the need for global solutions. Lancet Infect. Dis. 13, 1057–1098 (2013)." href="/articles/s41591-020-0894-4#ref-CR5" id="ref-link-section-d56841402e2657">5</a></sup>. The risk of HAIs can be mitigated through good infection prevention practice, with hand hygiene advocated as an important strategy to limit spread between patients and medical staff<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Longtin, Y., Sax, H., Allegranzi, B., Schneider, F. &amp; Pittet, D. Videos in clinical medicine. Hand hygiene. N. Engl. J. Med. 364, e24 (2011)." href="/articles/s41591-020-0894-4#ref-CR6" id="ref-link-section-d56841402e2662">6</a></sup>.</p><p>In addition to human-to-human transfer, the hospital environment is another key transmission network node, with mounting evidence that it harbors opportunistic antibiotic-resistant pathogens contributing to HAIs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Otter, J. A., Yezli, S., Salkeld, J. A. &amp; French, G. L. Evidence that contaminated surfaces contribute to the transmission of hospital pathogens and an overview of strategies to address contaminated surfaces in hospital settings. Am. J. Infect. Control 41, S6–S11 (2013)." href="/articles/s41591-020-0894-4#ref-CR7" id="ref-link-section-d56841402e2669">7</a></sup>. Reinforced environmental cleaning measures have shown effectiveness in reducing HAIs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Mitchell, B. G. et al. An environmental cleaning bundle and health-care-associated infections in hospitals (REACH): a multicentre, randomised trial. Lancet Infect. Dis. 19, 410–418 (2019)." href="/articles/s41591-020-0894-4#ref-CR8" id="ref-link-section-d56841402e2673">8</a></sup>. The microbial ecology and uncharacterized genetic reservoirs of hospital environments are thus of interest for both infection epidemiology and microbiology. For example, transmission and recombination profiles of antibiotic resistance genes (ARGs) in hospitals remain largely unknown and could help gauge risk for emergence of novel resistance combinations. Similarly, comparative genomics of hospital-adapted and epidemic strains could identify the source of outbreaks and inform infection control. While large-scale surveillance holds promise to reveal clinical and biological insights pertaining to the hospital microbiome as a reservoir of pathogens and ARGs, significant technological challenges remain. Traditionally, efforts to survey the hospital environment have focused on culture-based isolation of specific pathogens, with each isolate individually characterized via functional profiling, genotyping and/or whole-genome sequencing<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Price, J. R. et al. Transmission of Staphylococcus aureus between health-care workers, the environment, and patients in an intensive care unit: a longitudinal cohort study based on whole-genome sequencing. Lancet Infect. Dis. 17, 207–214 (2017)." href="#ref-CR9" id="ref-link-section-d56841402e2677">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Weingarten, R. A. et al. Genomic analysis of hospital plumbing reveals diverse reservoir of bacterial plasmids conferring carbapenem resistance. MBio 9, e02011-17 (2018)." href="#ref-CR10" id="ref-link-section-d56841402e2677_1">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Conlan, S. et al. Single-molecule sequencing to track plasmid diversity of hospital-associated carbapenemase-producing Enterobacteriaceae. Sci. Transl. Med. 6, 254ra126 (2014)." href="/articles/s41591-020-0894-4#ref-CR11" id="ref-link-section-d56841402e2680">11</a></sup>. This is laborious, is prone to isolation bias and precludes insights into overall community structure and how that interacts with the built environment to impact HAIs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Lax, S. &amp; Gilbert, J. A. Hospital-associated microbiota and implications for nosocomial infections. Trends Mol. Med. 21, 427–432 (2015)." href="/articles/s41591-020-0894-4#ref-CR12" id="ref-link-section-d56841402e2684">12</a></sup>.</p><p>The development of metagenomics enables profiling of overall community structure, characterizing individual microbes without isolation, and represents a scalable, high-throughput method for surveying hospital environments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Miller, R. R., Montoya, V., Gardy, J. L., Patrick, D. M. &amp; Tang, P. Metagenomics for pathogen detection in public health. Genome Med. 5, 81 (2013)." href="/articles/s41591-020-0894-4#ref-CR13" id="ref-link-section-d56841402e2691">13</a></sup>. This has been leveraged through 16S rRNA sequencing in early studies of bacterial diversity, particularly in intensive care units (ICUs)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Oberauner, L. et al. The ignored diversity: complex bacterial communities in intensive care units revealed by 16S pyrosequencing. Sci. Rep. 3, 1413 (2013)." href="/articles/s41591-020-0894-4#ref-CR14" id="ref-link-section-d56841402e2695">14</a></sup>. Lax et al. used this approach to extensively characterize microbial ecology, colonization and succession in a newly built hospital<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Lax, S. et al. Bacterial colonization and succession in a newly opened hospital. Sci. Transl. Med. 9, eaah6500 (2017)." href="/articles/s41591-020-0894-4#ref-CR15" id="ref-link-section-d56841402e2699">15</a></sup>. Using bioinformatics approaches, the authors identified ecological signatures of bacterial exchange between the environment, patients and healthcare workers. However, 16S rRNA sequencing precludes detailed analysis of nosocomial strains, resistomes, metabolic pathways and transmission of pathogenomes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Janda, J. M. &amp; Abbott, S. L. 16S rRNA gene sequencing for bacterial identification in the diagnostic laboratory: pluses, perils, and pitfalls. J. Clin. Microbiol. 45, 2761–2764 (2007)." href="/articles/s41591-020-0894-4#ref-CR16" id="ref-link-section-d56841402e2703">16</a></sup>. Brooks et al. used Illumina shotgun metagenomics to characterize strain polymorphisms and relatedness of pathogens in low-diversity neonatal ICU environments<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Brooks, B. et al. Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome. Nat. Commun. 8, 1814 (2017)." href="/articles/s41591-020-0894-4#ref-CR17" id="ref-link-section-d56841402e2707">17</a></sup>. Several limitations remain for the use of shotgun metagenomics in general, including low biomass, the presence of multiple strains and pathogens at low abundances, inaccuracies in strain-level analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="McIntyre, A. B. R. et al. Comprehensive benchmarking and ensemble approaches for metagenomic classifiers. Genome Biol. 18, 182 (2017)." href="/articles/s41591-020-0894-4#ref-CR18" id="ref-link-section-d56841402e2712">18</a></sup>, and shortcomings of short reads for assembling high-contiguity, strain-resolved genomes for detailed genetic analyses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Bertrand, D. et al. Hybrid metagenomic assembly enables high-resolution analysis of resistance determinants and mobile elements in human microbiomes. Nat. Biotechnol. 37, 937–944 (2019)." href="/articles/s41591-020-0894-4#ref-CR19" id="ref-link-section-d56841402e2716">19</a></sup>.</p><p>The availability of long-read sequencing presents new opportunities and challenges for pathogenome and resistome monitoring<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Che, Y. et al. Mobile antibiotic resistome in wastewater treatment plants revealed by nanopore metagenomic sequencing. Microbiome 7, 44 (2019)." href="/articles/s41591-020-0894-4#ref-CR20" id="ref-link-section-d56841402e2723">20</a></sup>. Here, we combined extensive short-read shotgun metagenomics of multiple sites, wards and time points (<i>n</i> = 428) with enrichment and nanopore sequencing of antibiotic-resistant mixed cultures (<i>n</i> = 1,661) to provide the most extensive genetic characterization of hospital environments to date. The combination of metagenomic surveys (short-read based) with detailed genomic analysis of nosocomial strains (long-read based) is ideal for studying distribution, abundance and turnover patterns of pathogens and ARGs. Nanopore metagenomics enabled the generation of thousands of high-contiguity genomes (<i>n</i> = 2,347), phage sequences (<i>n</i> = 1,693) and closed plasmid sequences (<i>n</i> = 5,910), revealing substantial uncharacterized genetic diversity (&gt;60% novel). These were harbored in distinct ecological niches characterized by biofilm-forming and human-microbiome-associated bacteria, with divergent patterns of spatiotemporal variation. Phylogenetic analysis highlighted that MDROs are more likely to be widely distributed and stably colonizing across hospital sites. Analysis of ARG combinations and phage and plasmid architectures revealed the dynamic nature of hospital-environment resistomes. Genomic comparisons with patient isolates across multiple species indicated that MDROs persist in the hospital environment for extended periods (&gt;8 years) to opportunistically infect patients. These findings underscore the importance of characterizing hospital microbiomes to understand niches and genetic reservoirs, the need for improved disinfection methods and the feasibility of large-scale genomic surveys to inform infection control.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Hospital-environment microbiomes offer distinct ecological niches for opportunistic pathogens and ARGs</h3><p>A diverse set of sites (<i>n</i> = 7) of concern for infection control<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Kanamori, H., Weber, D. J. &amp; Rutala, W. A. Healthcare outbreaks associated with a water reservoir and infection prevention strategies. Clin. Infect. Dis. 62, 1423–1435 (2016)." href="/articles/s41591-020-0894-4#ref-CR21" id="ref-link-section-d56841402e2758">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Dancer, S. J. Importance of the environment in methicillin-resistant Staphylococcus aureus acquisition: the case for hospital cleaning. Lancet Infect. Dis. 8, 101–113 (2008)." href="/articles/s41591-020-0894-4#ref-CR22" id="ref-link-section-d56841402e2761">22</a></sup> and different room types distributed around the building (5 single-bed isolation rooms together with 4 MDRO and 4 standard five-bed wards) were selected for initial sampling at two time points (1 week apart) of a tertiary-care hospital in Singapore (45 beds (4% of total), 179 sites, 358 samples; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig1">1a</a> and Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM3">1</a>). Illumina shotgun metagenomics (2 × 101 bp) was used to deeply characterize each sample (average = 30 × 10<sup>6</sup> reads; 3 of 358 libraries were excluded due to low biomass) to obtain taxonomic profiles and resistomes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig1">1b</a>, Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM4">2</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). Controls, spike-ins and validation experiments were used to assess and account for the impact of kit contaminants on low-biomass samples<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="de Goffau, M. C. et al. Recognizing the reagent microbiome. Nat. Microbiol. 3, 851–853 (2018)." href="/articles/s41591-020-0894-4#ref-CR23" id="ref-link-section-d56841402e2783">23</a></sup>, with likely contaminants identified using batch and correlation analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="de Goffau, M. C. et al. Recognizing the reagent microbiome. Nat. Microbiol. 3, 851–853 (2018)." href="/articles/s41591-020-0894-4#ref-CR23" id="ref-link-section-d56841402e2787">23</a></sup> and filtered from profiles (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">1</a>, Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM4">2</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). Taxonomic profiles were visualized using a principal-coordinates analysis (PCoA) plot to identify two distinct microbial community configurations in the hospital environment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig2">2a</a>). While community type A (CTA) sites were more taxonomically diverse (Wilcoxon <i>P</i> value &lt; 10<sup>−3</sup>; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">1</a>) and largely high-touch surfaces with frequent contact from patients and healthcare workers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Sehulster, L. &amp; Chinn, R. Y. Guidelines for environmental infection control in health-care facilities. Recommendations of CDC and the Healthcare Infection Control Practices Advisory Committee (HICPAC). MMWR Recomm. Rep. 52, 1–42 (2003)." href="/articles/s41591-020-0894-4#ref-CR24" id="ref-link-section-d56841402e2813">24</a></sup>, community type B (CTB) represents sites of increasing concern for infection control for their propensity to harbor MDROs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Weingarten, R. A. et al. Genomic analysis of hospital plumbing reveals diverse reservoir of bacterial plasmids conferring carbapenem resistance. MBio 9, e02011-17 (2018)." href="/articles/s41591-020-0894-4#ref-CR10" id="ref-link-section-d56841402e2817">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Kanamori, H., Weber, D. J. &amp; Rutala, W. A. Healthcare outbreaks associated with a water reservoir and infection prevention strategies. Clin. Infect. Dis. 62, 1423–1435 (2016)." href="/articles/s41591-020-0894-4#ref-CR21" id="ref-link-section-d56841402e2820">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Kizny Gordon, A. E. et al. The hospital water environment as a reservoir for carbapenem-resistant organisms causing hospital-acquired infections—a systematic review of the literature. Clin. Infect. Dis. 64, 1435–1444 (2017)." href="/articles/s41591-020-0894-4#ref-CR25" id="ref-link-section-d56841402e2823">25</a></sup>. Joint analysis of these community types helped to identify key taxonomic features that differentiate them, including several human-microbiome-associated genera (for example, <i>Cutibacterium</i>) and aquatic and terrestrial environment-associated genera (for example, <i>Achromobacter</i>) in CTA and CTB, respectively, although not all genera could be defined in these terms (for example, <i>Pseudomonas</i>, <i>Acinetobacter</i> and <i>Ralstonia</i>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig2">2b</a>). At the species level, key differences included enrichment of common skin bacteria (for example, <i>Cutibacterium acnes</i> and <i>Staphylococcus epidermidis</i>) and biofilm-associated organisms in hospitals (for example, <i>Elizabethkingia anophelis</i> and <i>Serratia marcescens</i>) in CTA and CTB sites, respectively, although their occurrences were not mutually exclusive, indicating shared influences (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig2">2c</a>). The comparison of hospital microbiome CTA and CTB sites to similar sites in an indoor office environment (<i>n</i> = 30, office; Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM3">1</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>) and other high-touch environmental microbiomes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Danko, D.C. et al. Global genetic cartography of urban metagenomes and anti-microbial resistance. Preprint at bioRxiv 
                  https://doi.org/10.1101/724526
                  
                 (2019)." href="/articles/s41591-020-0894-4#ref-CR26" id="ref-link-section-d56841402e2871">26</a></sup> (<i>n</i> = 99, MetaSUB Singapore; Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM3">1</a>) further highlighted the distinctness of hospital environments and community types (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">2</a>) and the corresponding utility as an organizing principle for studying clinical impact<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Oberauner, L. et al. The ignored diversity: complex bacterial communities in intensive care units revealed by 16S pyrosequencing. Sci. Rep. 3, 1413 (2013)." href="/articles/s41591-020-0894-4#ref-CR14" id="ref-link-section-d56841402e2885">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Beggs, C., Knibbs, L. D., Johnson, G. R. &amp; Morawska, L. Environmental contamination and hospital-acquired infection: factors that are easily overlooked. Indoor Air 25, 462–474 (2015)." href="/articles/s41591-020-0894-4#ref-CR27" id="ref-link-section-d56841402e2888">27</a></sup>.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Overview of sampling sites and analysis workflow."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Overview of sampling sites and analysis workflow.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0894-4/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="432"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Diagram showing the various sites that were sampled, including cardiac tables (A), bed rails (B), pulse oximeters (C), bedside lockers (D), sink aerators (E), sink traps (F) and door handles (G). Each ward (MDRO and standard) had five beds (sites A–D individually sampled), one sink (E,F) and no doors, while isolation rooms had one bed (A–D), one sink (E,F) and a door (G). <b>b</b>, Diagram showing the analysis workflow for the four swabs that were collected from each site in terms of culturing, DNA extraction and sequencing. Samples from each site were analyzed with shotgun metagenomic sequencing on the Illumina platform and multiple (<i>n</i> = 6) culture-enriched quasi-metagenomics on a GridION system.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0894-4/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Distinct ecological niches in the hospital environment for microbes and ARGs."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Distinct ecological niches in the hospital environment for microbes and ARGs.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0894-4/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="432"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, PCoA plot based on genus-level Bray–Curtis dissimilarity of taxonomic profiles (<i>n</i> = 176 independent samples, time point 1) indicating two distinct community types (denoted as CTA and CTB) for microbiomes from the hospital environment. <b>b</b>, Heat map showing relative abundances (log-scale, log<sub>2</sub> (RA)) for differentially abundant genera between CTA and CTB (false discovery rate (FDR)-adjusted two-sided Wilcoxon <i>P</i> value &lt; 0.01; <i>n</i> = 151 and 25 independent samples for CTA and CTB sites, respectively). <b>c</b>, Box plots showing relative abundances for differentially abundant species between CTA and CTB (FDR-adjusted two-sided Wilcoxon <i>P</i> value &lt; 0.01; <i>n</i> = 151 and 25 independent samples for CTA and CTB sites, respectively). In the box plots, the center line is the median; box limits are the upper and lower quartiles; and whiskers are 1.5 times the interquartile range (outlier points are not included in the visualization). <b>d</b>, PCoA plots (genus-level Bray–Curtis dissimilarity) showing variation in environmental microbiomes over time (lines connect two time points, 1 week apart) for different sites (<i>n</i> = 24, 26, 90, 90, 90, 10 and 22 independent samples for sink traps, aerators, bed rails, bedside lockers, cardiac tables, door handles and pulse oximeters, respectively). <b>e</b>, Radar plot showing the microbiome turnover index (fraction of taxa that are gained or lost across time points), human influence index (fraction of human reads) and site specificity index (uniqueness of site-specific taxonomic composition in relation to physically proximal sites). A positive site-specificity index indicates a stronger site-specific microbiome composition signature. <b>f</b>, Box plots showing relative abundances of common nosocomial pathogens that were differentially abundant across ward types in sites with high human contact (FDR-adjusted Kruskal–Wallis test, <i>P</i> value &lt; 0.01; <i>n</i> = 48, 128 and 128 independent samples for isolation rooms, MDRO wards and standard wards, respectively). In the box plots, the center line is the median; box limits are the upper and lower quartiles; and whiskers are 1.5 times the interquartile range (outlier points are not included in the visualization). <b>g</b>, Heat map depicting the frequency of detection for beta-lactamases at different sites in hospital wards. Multiple carbapenemases and the <i>mecA</i> gene were detected as part of the resistomes that were primarily defined by the community types (CTA and CTB).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0894-4/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Microbiomes associated with the community types exhibited varying stability across the sampled time points, with CTA sites demonstrating larger fluctuations (except door handles; Wilcoxon <i>P</i> value &lt; 10<sup>−3</sup>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig2">2d</a>). Microbial profiles diverged with distance (within a bed, within wards and across wards) and time (1 week apart), with temporal variability within a week being lower than spatial variability within a ward (Wilcoxon <i>P</i> value &lt; 10<sup>−3</sup>; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">3a</a>). Analysis of a subset of sites (<i>n</i> = 80) resampled at a third time point &gt;1 year later confirmed long-term stability of community types across sites (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">3b</a>). Microbial composition of sites is expected to be influenced by several factors, including abiotic conditions (humidity, temperature and surface type), seeding from microbial reservoirs (human or environmental) and exchange across sites. Based on sequencing data, we computed scores to quantify these factors, including a microbiome turnover index (fraction of taxa gained or lost across time points), a human influence index (fraction of human reads) and a site specificity index (uniqueness of site-specific taxonomic composition relative to proximal sites), each of which exhibited significantly correlated trends across time points (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">4a</a>). The computed indices reinforce the notion that CTB sites (primarily sink traps and aerators) have stable compositions (low turnover) based on site-specific biofilm configurations with limited human microbiome seeding (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig2">2e</a>). CTA sites showed higher human influence (Wilcoxon <i>P</i> value &lt; 10<sup>−15</sup>) and microbiome turnover (Wilcoxon <i>P</i> value &lt; 10<sup>−4</sup>) indices, although they were not directly correlated, and showed weaker site specificity (Wilcoxon <i>P</i> value &lt; 10<sup>−12</sup>), concordant with a model where human activities (patient discharge and admittance events) have a systemic role in shaping site compositions (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig2">2e</a>). Species that were enriched in CTA sites were also observed in CTB sites (and vice versa) but had higher turnover in these cases (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">4b</a>), with some exceptions such as <i>Siphoviridae</i>, which had high turnover in both CTA and CTB sites.</p><p>Overall, patterns of microbiome variability were consistent across ward types, although isolation rooms exhibited lower variability across time points (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">5</a>). In line with Singapore’s MDRO management guidelines<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Ministry of Health Singapore. Guidelines for Control and Prevention of Multi-drug Resistant Organisms (MDROs) in Healthcare Facilities (2013)." href="/articles/s41591-020-0894-4#ref-CR28" id="ref-link-section-d56841402e3055">28</a></sup>, patients colonized with carbapenem-resistant Enterobacteriaceae (CRE; for example, <i>Klebsiella</i> <i>pneumoniae</i>) were typically in single-bed isolation rooms, while patients with methicillin-resistant <i>Staphylococcus aureus</i> (MRSA) were in MDRO wards. An analysis of differentially abundant nosocomial pathogens (curated from <a href="https://www.cdc.gov/hai/organisms/organisms.html">https://www.cdc.gov/hai/organisms/organisms.html</a> and publications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Magill, S. S. et al. Multistate point-prevalence survey of health care-associated infections. N. Engl. J. Med. 370, 1198–1208 (2014)." href="/articles/s41591-020-0894-4#ref-CR4" id="ref-link-section-d56841402e3076">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lee, J. Y. H. et al. Global spread of three multidrug-resistant lineages of Staphylococcus epidermidis. Nat. Microbiol. 3, 1175–1185 (2018)." href="/articles/s41591-020-0894-4#ref-CR29" id="ref-link-section-d56841402e3079">29</a></sup>) detected across ward types identified <i>K. pneumoniae</i> and <i>S. aureus</i> as being enriched in CTA sites for isolation rooms and MDRO wards, respectively, providing further evidence for the influence of patient microbiomes on CTA sites (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig2">2f</a>). Consistent with observed taxonomic differences, CTA and CTB sites harbored distinct complements of ARGs in their resistomes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig2">2g</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">6</a>). While some ARGs were frequently detected in CTB sites (for example, <i>ges</i> and <i>oxa-7</i>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig2">2g</a>), CTA sites carried a wider diversity of ARGs at lower frequencies. Despite recent focus on CTB sites as ARG reservoirs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Weingarten, R. A. et al. Genomic analysis of hospital plumbing reveals diverse reservoir of bacterial plasmids conferring carbapenem resistance. MBio 9, e02011-17 (2018)." href="/articles/s41591-020-0894-4#ref-CR10" id="ref-link-section-d56841402e3108">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Kizny Gordon, A. E. et al. The hospital water environment as a reservoir for carbapenem-resistant organisms causing hospital-acquired infections—a systematic review of the literature. Clin. Infect. Dis. 64, 1435–1444 (2017)." href="/articles/s41591-020-0894-4#ref-CR25" id="ref-link-section-d56841402e3111">25</a></sup>, some clinically important ARGs such as <i>oxa-23</i> (encoding a carbapenemase) and <i>mecA</i> (methicillin resistance) were more frequently found in CTA sites, while genes such as <i>imp-1</i> (carbapenemase) and <i>cme-1</i> (extended-spectrum beta-lactamase) were more common in CTB sites. Different sites also exhibited distinct resistome patterns—for example, specific tetracycline (<i>tetC</i>) and macrolide (<i>mphE</i>) resistance genes were highly enriched only in aerators, while vancomycin resistance genes were only observed in bedside lockers and on bed rails—highlighting the importance of considering site- and ward-specific patterns for infection control and drug resistance mitigation strategies (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">6</a>). While a higher proportion of ARGs was consistently detected across time points in CTB sites compared to CTA sites (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">7a</a>), overall, hospital microbiomes exhibited significantly higher abundance (&gt;3-fold versus MetaSUB Singapore and &gt;12-fold versus office sites, Wilcoxon <i>P</i> value &lt; 10<sup>−15</sup> for both comparisons; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">7b</a>) and higher diversity (Wilcoxon <i>P</i> value &lt; 10<sup>−15</sup>; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">7c</a>) of ARGs compared to other high-touch urban environmental microbiomes. Even though the presence of ARGs does not always translate to resistance phenotypes, these results further underscore the distinctness of hospital microbiomes as ARG reservoirs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Wright, G. D. Environmental and clinical antibiotic resistomes, same only different. Curr. Opin. Microbiol. 51, 57–63 (2019)." href="/articles/s41591-020-0894-4#ref-CR30" id="ref-link-section-d56841402e3158">30</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec4">Quasi-metagenomics with nanopore sequencing reveals distribution of multidrug-resistant opportunistic pathogens in the hospital environment</h3><p>Based on Illumina metagenomic profiles, we noted that nosocomial pathogens were generally present at low relative abundances (median relative abundance &lt; 0.5%; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig3">3a</a>) in hospital environments (even though this was higher than in other urban sites; Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">7d</a>), precluding detailed genomic characterization of transmission patterns, ARG combinations and plasmids. The distribution of common pathogens exhibited site-specific patterns (PERMANOVA <i>P</i> value &lt; 0.001; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig3">3a</a>), in agreement with the distinct niches observed in hospital environments (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig2">2a–c</a>), and indicated that enrichment cultures could capture a diverse set of species. We exploited this observation to use a culturing, antibiotic selection (five antibiotics) and metagenomic nanopore sequencing approach (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig1">1b</a>) to obtain a large database of high-contiguity assemblies (<i>n</i> = 2,347) from the hospital environment (median N50 &gt; 1 Mb; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig3">3b</a>, Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM5">3</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>), expanding substantially on genomic resources established by previous studies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Weingarten, R. A. et al. Genomic analysis of hospital plumbing reveals diverse reservoir of bacterial plasmids conferring carbapenem resistance. MBio 9, e02011-17 (2018)." href="/articles/s41591-020-0894-4#ref-CR10" id="ref-link-section-d56841402e3201">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Conlan, S. et al. Single-molecule sequencing to track plasmid diversity of hospital-associated carbapenemase-producing Enterobacteriaceae. Sci. Transl. Med. 6, 254ra126 (2014)." href="/articles/s41591-020-0894-4#ref-CR11" id="ref-link-section-d56841402e3204">11</a></sup>. Overall, a large percentage of sites led to viable cultures (&gt;95%), with antibiotic selection resulting in growth in &gt;80% of plates (1,495 of 1,790) and &gt;42% of sites resulting in cultures for all five antibiotics. Control swabs led to no cultures (0 of 10), confirming that cultures were not likely due to contamination (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>), and further testing of isolates confirmed that the vast majority of strains in the cultures were likely to be antibiotic resistant (99%; Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">2</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Genome-resolved characterization of nosocomial multidrug-resistant strains that spread and persist at low relative abundances in the hospital environment."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Genome-resolved characterization of nosocomial multidrug-resistant strains that spread and persist at low relative abundances in the hospital environment.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0894-4/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="391"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Heat map displaying the distinct median relative abundance (RA) of common nosocomial pathogens at different sites (<i>n</i> = 26, 90, 90, 90, 10, 22 and 24 for aerators, bed rails, bedside lockers, cardiac tables, door handles, pulse oximeters and sink traps, respectively) in hospital environments (PERMANOVA <i>P</i> value &lt; 0.001). <b>b</b>, Distribution of assembly contiguity statistics (N50 = fragment size such that more than 50% of the genome is in longer sequences) for common nosocomial pathogens, highlighting the high genomic contiguity that was obtained (median N50 &gt; 1 Mb). <b>c</b>, Dot plots highlighting that genomes can be rapidly obtained for several nosocomial pathogens despite their low relative abundances in corresponding environmental microbiomes (<i>y</i> axis) through an enrichment and long-read metagenomic sequencing-based protocol. Represented species are associated with more than 20 genome drafts in the overall database of 2,347 genomes. <b>d</b>, Left, phylogenetic relationships of <i>S. aureus</i> derivative clusters (&gt;99.99% ANI; each node represents the consensus genome for the cluster) detected in the hospital environment together with their antibiotic resistance profiles. The scale bar represents the number of substitutions per site in the core alignment. Right, hive map representation showing localization of <i>S. aureus</i> clusters that spread (detected at two or more locations) and/or persist (detected in time points 1 and 2) in the hospital environment. The colored lines represent occurrence at time point 1 (orange) and time point 2 (blue); line thickness represents the number of instances of such occurrences. Note that multidrug-resistant strains such as s3, s2 and s1 tend to be more widely distributed and persistent in the hospital environment.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0894-4/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>DNA was extracted from 1,661 plates and sequenced on a GridION to provide 535 Mb of data on average per sample (median read length &gt; 2.5 kb). Long-read metagenomic assembly enabled the reconstruction of megabase-pair sized contigs (versus average N50 &lt; 5 kb for Illumina assemblies) as the communities were largely simple (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">8</a>, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig3">3b</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). Evaluation of these draft genomes based on conserved single-copy genes confirmed that they were of high quality (completeness &gt; 99%, contamination &lt; 0.5%; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). In total, we obtained genomes for 69 species from the hospital environment, 40% of which belonged to common pathogens (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). Our results confirm the viability of these species in different hospital environments and the ability to enrich them for sequencing and genome reconstruction despite their low abundances in hospital microbiomes (median relative abundance = 0.68%, averaged across species; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig3">3c</a>). Large-scale homology analysis with public databases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pasolli, E. et al. Extensive unexplored human microbiome diversity revealed by over 150,000 genomes from metagenomes spanning age, geography, and lifestyle. Cell 176, 649–662 (2019)." href="#ref-CR31" id="ref-link-section-d56841402e3284">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Almeida, A. et al. A new genomic blueprint of the human gut microbiota. Nature 568, 499–504 (2019)." href="#ref-CR32" id="ref-link-section-d56841402e3284_1">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Parks, D. H. et al. A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life. Nat. Biotechnol. 36, 996–1004 (2018)." href="/articles/s41591-020-0894-4#ref-CR33" id="ref-link-section-d56841402e3287">33</a></sup> also helped to identify 13 (out of &gt;80) species-level clusters (11 different genera including <i>Bacillus</i>, <i>Pseudomonas</i> and <i>Staphylococcus</i>) with no representatives from known species, highlighting recovery of high-quality genomes for novel species using this approach (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). Rarefaction analysis showed that &gt;90% of the species and resistance gene diversity (&gt;50% richness) that could be sampled from sites in this study was captured by our sample size (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">9</a>), while substantial additional diversity remains to be captured for plasmids and HAI-associated strains (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">3</a>). This confirms the viability of future surveys of ARGs in hospitals with much fewer samples (<i>n</i> ≈ 50), making regular surveys feasible, affordable and potentially actionable.</p><p>As plasmids and phages serve as an important medium for the evolution and spread of ARGs and emergence of multidrug resistance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="San Millan, A. Evolution of plasmid-mediated antibiotic resistance in the clinical context. Trends Microbiol. 26, 978–985 (2018)." href="/articles/s41591-020-0894-4#ref-CR34" id="ref-link-section-d56841402e3316">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Calero-Caceres, W., Ye, M. &amp; Balcazar, J. L. Bacteriophages as environmental reservoirs of antibiotic resistance. Trends Microbiol. 27, 570–577 (2019)." href="/articles/s41591-020-0894-4#ref-CR35" id="ref-link-section-d56841402e3319">35</a></sup>, we characterized corresponding sequences in our genomic database (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). In total, we recovered 696 Mb of plasmid sequences (<i>n</i> = 5,910 closed and 493 Mb of linear fragments) and 63 Mb of phage sequences (<i>n</i> = 1,693, of which 277 are circular), most of which are not present in existing databases for plasmids<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Galata, V., Fehlmann, T., Backes, C. &amp; Keller, A. PLSDB: a resource of complete bacterial plasmids. Nucleic Acids Res. 47, D195–D202 (2019)." href="/articles/s41591-020-0894-4#ref-CR36" id="ref-link-section-d56841402e3332">36</a></sup> or phages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Gao, N. L. et al. MVP: a microbe–phage interaction database. Nucleic Acids Res 46, D700–D707 (2018)." href="/articles/s41591-020-0894-4#ref-CR37" id="ref-link-section-d56841402e3337">37</a></sup> (&gt;90%; 1,505 of 1,588 plasmid clusters and 501 of 557 phage clusters; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>) despite being commonly distributed in the hospital (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">10</a>), highlighting its underexplored genetic diversity. Many closed plasmids were &gt;100 kb long (&gt;9%, <i>n</i> = 536), rich in repeats and present at low abundance, impeding characterization using Illumina metagenomics. We noted the presence of several large <i>mecA</i>-carrying plasmids that contained antiseptic or disinfectant resistance genes (<i>qacA</i> or <i>qacC</i>), a combination that is not represented in existing databases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Galata, V., Fehlmann, T., Backes, C. &amp; Keller, A. PLSDB: a resource of complete bacterial plasmids. Nucleic Acids Res. 47, D195–D202 (2019)." href="/articles/s41591-020-0894-4#ref-CR36" id="ref-link-section-d56841402e3360">36</a></sup> but is in agreement with high biocide resistance for MRSA in clinical settings<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Duran, N., Temiz, M., Duran, G. G., Eryilmaz, N. &amp; Jenedi, K. Relationship between the resistance genes to quaternary ammonium compounds and antibiotic resistance in staphylococci isolated from surgical site infections. Med. Sci. Monit. 20, 544–550 (2014)." href="/articles/s41591-020-0894-4#ref-CR38" id="ref-link-section-d56841402e3364">38</a></sup>. One of the plasmids had genes from several additional ARG classes that have not been seen in combination (for example, <i>dfrC</i>, <i>lnuA</i> and <i>aac6-Aph2</i>), highlighting the value of closed sequences for characterization of novel ARG combinations.</p><p>The availability of a large collection of highly contiguous plasmid (closed) and chromosomal (megabase-pair contigs) assemblies allowed us to perform genomic relatedness (with environmental and patient strains) and structural (common gene cassettes and exchange across cassettes) analysis. We first analyzed evolutionary relationships between genomes from the hospital, with previously used thresholds of average nucleotide identity (ANI) to define strain-level<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Johnson, R. C. et al. Investigation of a cluster of Sphingomonas koreensis infections. N. Engl. J. Med. 379, 2529–2539 (2018)." href="/articles/s41591-020-0894-4#ref-CR39" id="ref-link-section-d56841402e3380">39</a></sup> (&gt;99.9% ANI), derivative<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Weingarten, R. A. et al. Genomic analysis of hospital plumbing reveals diverse reservoir of bacterial plasmids conferring carbapenem resistance. MBio 9, e02011-17 (2018)." href="/articles/s41591-020-0894-4#ref-CR10" id="ref-link-section-d56841402e3384">10</a></sup> (&gt;99.99% ANI) and direct-transfer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Brooks, B. et al. Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome. Nat. Commun. 8, 1814 (2017)." href="/articles/s41591-020-0894-4#ref-CR17" id="ref-link-section-d56841402e3388">17</a></sup> (&gt;99.999% ANI) genome clusters, and understand their spatiotemporal distribution. For many species, a diverse set of clusters was observed across the hospital (<i>n</i> = 6, <i>Pseudomonas aeruginosa</i> to <i>n</i> = 46, <i>S. epidermidis</i>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig3">3d</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">11</a>). Some genome clusters were frequently detected at multiple sites and ward types in the hospital, and these were also significantly enriched for clusters detected in the first and second time points (Fisher’s exact test <i>P</i> value &lt; 1.5 × 10<sup>−9</sup>). Even at the most stringent threshold (direct transfer), a substantial fraction of genomes observed in the third time point (1.5 years later) clustered with genomes from earlier time points for various species (<i>E. anophelis</i>: 92%, as few as 5 single nucleotide polymorphisms (SNPs); <i>S. marcescens</i>: 20%, 16 SNPs; <i>Staphylococcus haemolyticus</i>: 21%, 8 SNPs), emphasizing the stability of environmental pathogenomes.</p><p>Overlaying antibiotic resistance information with these patterns, we noted an enrichment of multiantibiotic resistance among strains that were widely distributed through space and time (&gt;2 antibiotics; Fisher’s exact test <i>P</i> value &lt; 3 × 10<sup>−8</sup>; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig3">3d</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">12</a>). This was also consistently observed across several common pathogens in the hospital (Fisher’s exact test <i>P</i> value: 1.6 × 10<sup>−2</sup>, <i>S. aureus</i>; 2.3 × 10<sup>−3</sup>, <i>S. epidermidis</i>; 3.7 × 10<sup>−3</sup>, <i>Enterococcus faecalis</i>; 5.0 × 10<sup>−2</sup>, <i>Acinetobacter baumannii</i>). For a subset of species (<i>S. aureus</i>, <i>S. epidermidis</i> and <i>A. baumannii</i>), we used Illumina sequencing to generate hybrid assemblies and reliably detect derivative and direct-transfer relationships (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). Genomes that were related across early time points based on these stringent criteria continued to be significantly enriched for multidrug resistance (binomial test <i>P</i> value &lt; 10<sup>−5</sup>, all species and both thresholds) and were also enriched in the third time point (derivative clusters, binomial test <i>P</i> value: 0.028, <i>S</i>. <i>epidermidis</i>; 5.0 × 10<sup>−5</sup>, <i>S. aureus</i>), highlighting the presence of stable, viable environmental reservoirs for pathogens and the need to understand the mechanisms contributing to enrichment of multidrug-resistant strains<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Fahimipour, A.K. et al. Antimicrobial chemicals associate with microbial function and antibiotic resistance indoors. mSystems 3, e00200-18 (2018)." href="/articles/s41591-020-0894-4#ref-CR40" id="ref-link-section-d56841402e3499">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Hartmann, E. M. et al. Antimicrobial chemicals are associated with elevated antibiotic resistance genes in the indoor dust microbiome. Environ. Sci. Technol. 50, 9807–9815 (2016)." href="/articles/s41591-020-0894-4#ref-CR41" id="ref-link-section-d56841402e3502">41</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec5">Diversity and dynamics of ARG cassettes in the hospital environment</h3><p>With increasing multidrug resistance, the specific combination of ARGs that is harbored is important to know from a clinical perspective. In hospital environments, little is known about the diversity of ARG combinations and genetic exchange across genomic cassettes and plasmids. Comparing our database of 2,347 high-contiguity genomes and 5,910 closed plasmids against existing databases, we found that 34% of the ARG combinations observed were novel (255 of 752; Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM6">4</a>). Certain ARG combinations have obvious clinical importance, for example, the co-occurrence of <i>mecA</i> with <i>fosB</i> (fosfomycin resistance) in several environmental <i>S. aureus</i> strains, an observation that is concerning given the potential utility of fosfomycin for treating MRSA infections<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Falagas, M. E., Roussos, N., Gkegkes, I. D., Rafailidis, P. I. &amp; Karageorgopoulos, D. E. Fosfomycin for the treatment of infections caused by Gram-positive cocci with advanced antimicrobial drug resistance: a review of microbiological, animal and clinical studies. Expert Opin. Investig. Drugs 18, 921–944 (2009)." href="/articles/s41591-020-0894-4#ref-CR42" id="ref-link-section-d56841402e3526">42</a></sup>. Notably, we detected Enterobacteriaceae<i>-</i>associated genes that can confer resistance to gentamicin (<i>aac3-IIa</i>), fosfomycin (<i>fosA</i>, <i>fosA2</i>) and colistin (<i>mcr1</i>), all last-resort antibiotics for CRE infections. Additionally, two Enterobacteriaceae<i>-</i>associated plasmids, one carrying <i>fosA</i> and the other carrying <i>mcr1</i>, were obtained from the same bedside locker, highlighting the potential reservoir for emergence of co-resistance to colistin and fosfomycin. Another Enterobacteriaceae<i>-</i>associated plasmid carried a rifampicin resistance gene (<i>arr</i>), a telling observation given the growing interest in using rifampicin in combination treatments for a variety of Gram-negative infections, for example, <i>A. baumannii</i><sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Urban, C., Mariano, N. &amp; Rahal, J. J. In vitro double and triple bactericidal activities of doripenem, polymyxin B, and rifampin against multidrug-resistant Acinetobacter baumannii, Pseudomonas aeruginosa, Klebsiella pneumoniae, and Escherichia coli. Antimicrob. Agents Chemother. 54, 2732–2734 (2010)." href="/articles/s41591-020-0894-4#ref-CR43" id="ref-link-section-d56841402e3564">43</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Izadpanah, M. &amp; Khalili, H. Antibiotic regimens for treatment of infections due to multidrug-resistant Gram-negative pathogens: an evidence-based literature review. J. Res. Pharm. Pract. 4, 105–114 (2015)." href="/articles/s41591-020-0894-4#ref-CR44" id="ref-link-section-d56841402e3567">44</a></sup>.</p><p>We next identified common ARG pairs that were in close proximity (&lt;10 kb apart) to determine chromosomal cassettes that may serve as the unit of evolution, co-regulation and ARG exchange (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). Chromosomal cassettes were generally small (2–6 genes, average = 3) and specific to a species, although two large cassettes carrying extended-spectrum beta-lactamases were found to overlap for <i>K. pneumonia</i> and <i>Enterobacter cloacae</i> (KpnC1, KpnC2 and EclC1, EclC2; KpnC3 and EclC3; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig4">4a</a> and Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM7">5</a>). Selective pressure from rampant use of beta-lactams and plasmid-mediated transmission could have contributed to the sharing of these large cassettes across species. Cassettes for Gram-negative species were larger and more stable (solid lines to genes), while those for Gram-positive species were smaller with many variably present members (dashed lines to genes). The largest shared cassette among Gram-positive species (aminoglycoside-streptothricin resistance; <i>ant6-Ia</i>, <i>sat4A</i> and <i>aph3-III</i>) was in <i>Enterococcus</i> and <i>Staphylococcus</i> but with no discernible signals of mobile elements<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Werner, G., Hildebrandt, B. &amp; Witte, W. Aminoglycoside-streptothricin resistance gene cluster aadE–sat4–aphA-3 disseminated among multiresistant isolates of Enterococcus faecium. Antimicrob. Agents Chemother. 45, 3267–3269 (2001)." href="/articles/s41591-020-0894-4#ref-CR45" id="ref-link-section-d56841402e3605">45</a></sup>. While most genes were stably present in cassettes, with some exceptions (for example, <i>tetK</i>, <i>far1</i> and <i>catA</i>), the exchange of genes across cassettes was rarely observed (for example, <i>blaZ</i>), indicating that chromosomal cassettes tend to be relatively fixed.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Species distribution and genomic proximity of drug resistance genes in the hospital-environment microbiome."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Species distribution and genomic proximity of drug resistance genes in the hospital-environment microbiome.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0894-4/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="337"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>Genomic proximity network and clustering of ARGs based on 2,347 microbial genomes and 5,910 closed plasmid sequences obtained from the hospital environment. <b>a</b>, Multigraph of genomic proximity between ARGs. Colored edges indicate gene pairs found &lt;10 kb apart in the genomes for a species (excluding plasmids). Line widths indicate the frequency of occurrence of gene pairs (normalized by count for the rarer gene), and frequencies &gt;80% are marked with solid lines. Solid-line cliques in each species were used to define cassettes and assign names (Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM7">5</a>), and the number after the colon indicates clique size. Genes are colored according to their respective antibiotic classes. <b>b</b>, Circles represent different plasmid clusters (95% identity), and their corresponding ARGs are connected by edges and indicated by diamonds. Plasmid nodes were labeled based on a three-letter short form for the host species and assigned a number (for example, Kpn1 for a <i>K. pneumoniae</i> plasmid); the number after the colon indicates how many representatives of the plasmid family were observed in the database. Edges are weighted by the frequency at which a gene is present in a plasmid, and frequencies &gt;80% are indicated with red solid lines. Genes and backbones are color coded according to their respective ARG classes and inferred host species for ease of reference.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0894-4/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Performing a similar analysis for closed plasmids, we first clustered them into shared backbones and annotated them for known hosts (identity ≥ 95%; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). By analyzing ARGs in this context, we found that many ARGs were variably present in backbones (93 of 143). For ARGs stably found in one backbone, many were variably present in another backbone (19 of 31), highlighting the dynamic nature of ARG combinations from plasmids in the hospital environment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig4">4b</a>). Despite this, some ARG combinations were stably present in multiple plasmid backbones, indicating strong selection for coexistence. For example, the genes <i>strA</i>, <i>strB</i> (streptomycin resistance) and <i>sulII</i> (sulfonamide resistance) co-occurred in two distinct backbones (Sen1 and Kpn2, sequence overlap &lt; 54%), likely as a signature from past co-administration of streptomycin and sulfonamides<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Weinstein, L., Chang, T. W., Hudson, J. B. &amp; Hartl, W. The concurrent use of sulfonamides and antibiotics in the treatment of infections: in vivo and in vitro studies of the effect of sulfonamide-antibiotic combinations on the emergence of drug resistance. Ann. NY Acad. Sci. 69, 408–416 (1957)." href="/articles/s41591-020-0894-4#ref-CR46" id="ref-link-section-d56841402e3673">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Sundin, G. W. &amp; Bender, C. L. Dissemination of the strA–strB streptomycin-resistance genes among commensal and pathogenic bacteria from humans, animals, and plants. Mol. Ecol. 5, 133–143 (1996)." href="/articles/s41591-020-0894-4#ref-CR47" id="ref-link-section-d56841402e3676">47</a></sup>. Similarly, while aminoglycoside resistance genes such as <i>aadD</i> and <i>aac6-Aph2</i> were widely distributed across plasmid backbones, <i>ant6-Ia</i> and <i>aph3-III</i> were stably shared by two distinct backbones (Efa4 and Efs1, sequence overlap &lt;11%) indicating that they may provide synergistic resistance to aminoglycosides by catalyzing different modifications. Notably, genes that are widely distributed across plasmids (for example, <i>tetK</i>, <i>far1</i> and <i>blaZ</i>) can come together in a novel, clinically relevant backbone (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig4">4b</a>; Slu3, with 38 sequences in our database), as described for a cytotoxin-producing MRSA strain<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Monecke, S. et al. Microarray-based characterisation of a Panton–Valentine leukocidin-positive community-acquired strain of methicillin-resistant Staphylococcus aureus. Clin. Microbiol. Infect. 12, 718–728 (2006)." href="/articles/s41591-020-0894-4#ref-CR48" id="ref-link-section-d56841402e3705">48</a></sup>. While the previously isolated strain was resistant to fusidic acid and tetracycline, but susceptible to erythromycin and clindamycin, we noted the presence of a common plasmid backbone in our database (Sha2 with 88 sequences) that carried a new combination of resistance genes for all four antibiotics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig4">4b</a> and Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM6">4</a>). Similarly, we observed that ARGs found in phages, such as <i>aac6-Aph2</i> and <i>far1</i>, tended to be more widely present (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig4">4</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">13a</a>), with evidence for recent phage-mediated dissemination of <i>far1</i> across <i>Staphylococcus</i> species (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">13b</a>). In general, ARGs found in plasmids tended to have more ARGs in close proximity (&lt;10 kb apart) in chromosomes than chromosome-exclusive ARGs (Wilcoxon test <i>P</i> value = 6 × 10<sup>−7</sup>), characteristic of higher gene mobility and shuffling for plasmid-associated genes. Thus, plasmid backbones seen in the hospital environment likely represent a more plastic framework to generate diverse ARG combinations, many of which are not seen in genomic cassettes (25%) despite strong overlap in the complement of ARGs that they harbor (84% of plasmid genes).</p><h3 class="c-article__sub-heading" id="Sec6">Hospital-environment strains overlapping with patient isolates are globally disseminated and enriched for multidrug resistance</h3><p>The availability of a large database of genomes from many species in the hospital environment, an obvious hub for patient colonization, prompted us to ask how environmental strains are related to patient-colonizing strains. To examine this, we constructed phylogenetic trees for environmental strains and patient isolates across species (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig5">5</a>). We started with Singaporean <i>E. anophelis</i> isolates from a 2012 outbreak<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Teo, J. et al. First case of E. anophelis outbreak in an intensive-care unit. Lancet 382, 855–856 (2013)." href="/articles/s41591-020-0894-4#ref-CR49" id="ref-link-section-d56841402e3757">49</a></sup> (<i>n</i> = 10) and an additional set of patient isolates from 2009–2012 (<i>n</i> = 52; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig5">5a</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-020-0894-4#Sec8">Methods</a>). Despite sampling from different Singaporean hospitals after a span of 5–8 years, patient-associated genomes matched environmental genomes with just 16 SNPs (s1; 99.9996% ANI). The environmental <i>E. anophelis</i> genomes in our studies primarily originated from sinks, which, as noted earlier, tend to have stable communities, indicating that these strains may have originated from a common reservoir upstream of water-piping systems<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Johnson, R. C. et al. Investigation of a cluster of Sphingomonas koreensis infections. N. Engl. J. Med. 379, 2529–2539 (2018)." href="/articles/s41591-020-0894-4#ref-CR39" id="ref-link-section-d56841402e3777">39</a></sup>. The <i>E. anophelis</i> clusters shared between patients and the environment were also detected at the third time point 1.5 years later (&gt;99.999% ANI, direct transfer) and exhibited resistance to more antibiotics than the clusters that were not shared (1.25-foldchange; one-sided Wilcoxon <i>P</i> value = 0.059).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Multispecies analysis of phylogenetic relationships between environmental and patient genomes."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Multispecies analysis of phylogenetic relationships between environmental and patient genomes.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-020-0894-4/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="394"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p>Phylogenies depict the evolutionary relationships between derivative clusters (&gt;99.99% ANI), with each node representing the consensus genome for a cluster. <b>a</b>, <i>E. anophelis</i> from a nosocomial outbreak in Singapore in 2012 and other patient isolates from 2009–2012, <b>b</b>, patient-colonizing <i>S. aureus</i> from a 2009–2011 surveillance study in Singaporean hospitals, <b>c</b>, infectious <i>A</i>. <i>baumannii</i> isolates from patients in two major Kuwaiti hospitals and Singaporean patient isolates, and <b>d</b>, recent globally disseminated multidrug-resistant <i>S</i>. <i>epidermidis</i> lineages, together with environmental genomes for corresponding species from this study. While <b>a</b> and <b>b</b> highlight the close relationships between the strains circulating in Singaporean hospitals up to 8 years apart, <b>c</b> and <b>d</b> reveal the global dissemination of several lineages. The matrices next to the trees indicate the antibiotic resistance profiles for corresponding derivative clusters. Scale bars depict the number of substitutions for each site in the core alignment. For all species tested, clusters shared between environmental and patient genomes were enriched for multidrug resistance.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-020-0894-4/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We next analyzed <i>S. aureus</i> genomes (<i>n</i> = 221) from a surveillance study of patients in the same hospital almost a decade ago<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Htun, H. L. et al. Methicillin-resistant Staphylococcus aureus colonisation: epidemiological and molecular characteristics in an acute-care tertiary hospital in Singapore. Epidemiol. Infect. 146, 1785–1792 (2018)." href="/articles/s41591-020-0894-4#ref-CR50" id="ref-link-section-d56841402e3861">50</a></sup>. These strains matched 5 of 17 strains obtained in the current study, with environmental and patient genomes having just 39 SNPs and 99.9985% ANI (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig5">5b</a>). The co-occurrence of patient and environmental genomes was significantly enriched in multidrug-resistant clusters at the derivative genomes threshold (for example, s1, s2 and s3; binomial test <i>P</i> value &lt; 10<sup>−15</sup>). These clusters were also enriched for genomes detected in the third time point (binomial test <i>P</i> value &lt; 2 × 10<sup>−7</sup>) with &lt;60 SNPs (99.998% ANI) from genomes in early time points, highlighting the stability of antibiotic-resistant derivative clusters in the hospital environment.</p><p>To extend these observations, <i>A. baumannii</i> patient isolates (<i>n</i> = 108) from a hospital surveillance cohort in Singapore established &gt;8 years ago were sequenced. Many isolate genomes from this cohort had high identity to our environmental genomes (s6, 99.995% ANI) while being temporally separated by almost a decade. In addition, patient isolates that overlapped with environmental genomes were enriched for multidrug resistance (derivative clusters; binomial test <i>P</i> value &lt; 4 × 10<sup>−3</sup>). Extending to a regional context, analysis of <i>A. baumannii</i> patient isolates (<i>n</i> = 36) from two major Kuwaiti hospitals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Nasser, K. et al. Draft genome sequences of six multidrug-resistant clinical strains of Acinetobacter baumannii, isolated at two major hospitals in Kuwait. Genome Announc. 6, e00264-18 (2018)." href="/articles/s41591-020-0894-4#ref-CR51" id="ref-link-section-d56841402e3900">51</a></sup> with our environmental genomes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig5">5c</a>) identified a shared derivative cluster resistant to all five antibiotics, including Singaporean and Kuwaiti patient isolate genomes at high identity (s6, &gt;99.99% ANI). This highlights the presence of multidrug-resistant <i>A. baumannii</i> derivative clusters in hospital environments that are persistent, enriched in overlap with patient isolates and globally disseminated.</p><p>Similar patterns were observed recently for <i>S. epidermidis</i> lineages (ST2/ST2 mixed), which seem to have disseminated globally within a short period of time (<i>n</i> = 229; ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lee, J. Y. H. et al. Global spread of three multidrug-resistant lineages of Staphylococcus epidermidis. Nat. Microbiol. 3, 1175–1185 (2018)." href="/articles/s41591-020-0894-4#ref-CR29" id="ref-link-section-d56841402e3919">29</a></sup>). We confirmed detection of these rifampicin-resistant<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lee, J. Y. H. et al. Global spread of three multidrug-resistant lineages of Staphylococcus epidermidis. Nat. Microbiol. 3, 1175–1185 (2018)." href="/articles/s41591-020-0894-4#ref-CR29" id="ref-link-section-d56841402e3923">29</a></sup> lineages in our data (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig5">5d</a>), with 80 SNPs (99.997% ANI) from our hospital-environment genomes. One other lineage (ST16) not known to be globally disseminated (isolated from a patient sample in the United States<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lee, J. Y. H. et al. Global spread of three multidrug-resistant lineages of Staphylococcus epidermidis. Nat. Microbiol. 3, 1175–1185 (2018)." href="/articles/s41591-020-0894-4#ref-CR29" id="ref-link-section-d56841402e3931">29</a></sup>) was represented by a genome in our database with similarity at the derivative threshold (99.991% ANI). Finally, we found that the overlap between <i>S. epidermidis</i> patient isolates (surveillance samples from Austin Health in Australia<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lee, J. Y. H. et al. Global spread of three multidrug-resistant lineages of Staphylococcus epidermidis. Nat. Microbiol. 3, 1175–1185 (2018)." href="/articles/s41591-020-0894-4#ref-CR29" id="ref-link-section-d56841402e3938">29</a></sup>) and environmental genomes from this study was enriched for multidrug resistance in derivative clusters (binomial test <i>P</i> value &lt; 4 × 10<sup>−12</sup>). Together with the observation that multiantibiotic-resistant strains are persistent and widely distributed across the hospital environment (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig3">3d</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">11</a>), these data point to selective advantages for MDROs to persist and spread in hospital environments and patients.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec7-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec7">Discussion</h2><div class="c-article-section__content" id="Sec7-content"><p>While the importance of hospital design for preventing infections is known<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Walker, J. T. et al. Hospital and community acquired infection and the built environment—design and testing of infection control rooms. J. Hosp. Infect. 65, 43–49 (2007)." href="/articles/s41591-020-0894-4#ref-CR52" id="ref-link-section-d56841402e3963">52</a></sup>, the utility of metagenomic surveys in medical facilities remains underexplored<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Lax, S. &amp; Gilbert, J. A. Hospital-associated microbiota and implications for nosocomial infections. Trends Mol. Med. 21, 427–432 (2015)." href="/articles/s41591-020-0894-4#ref-CR12" id="ref-link-section-d56841402e3967">12</a></sup>. A detailed survey helps provide a reference map (with three-dimensional (3D) visualization; <a href="https://github.com/csb5/hospital_microbiome_explorer">https://github.com/csb5/hospital_microbiome_explorer</a>) that can be updated based on periodic scans whose frequency and locations can be informed by the initial survey. For example, the turnover score and specificity of a site can determine whether and how frequently it should be sampled. Variations in human influence scores could fine-tune cleaning practices, and distribution of specific pathogens could inform infection control in outbreak settings. As genomics-guided infection control advances, this knowledge could feed back into better hospital designs. With further improvements in the cost and ease of short-read sequencing, hospital-wide surveys will be increasingly feasible, provide valuable information for infection control and eventually be part of routine practice.</p><p>The microbial community types observed here highlight distinct niches found in hospitals compared to other urban environments, providing an organizing principle for further study. For example, while many pathogens were substantially enriched in hospitals, this was also prominent in CTA sites that had a greater diversity of ARGs (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">7c,d</a>). Recent clinical studies have focused on wash-area sites (such as sinks and showers; CTB sites), as outbreak-associated pathogens are often isolated there<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Johnson, R. C. et al. Investigation of a cluster of Sphingomonas koreensis infections. N. Engl. J. Med. 379, 2529–2539 (2018)." href="/articles/s41591-020-0894-4#ref-CR39" id="ref-link-section-d56841402e3984">39</a></sup>. This focus on CTB sites is concordant with the presence of biofilm-forming bacteria and their harboring viable reservoirs for extended periods (for example, in the plumbing). Our data show that many pathogens (for example, <i>K. pneumoniae</i>), ARGs (for example, carbapenemases such as <i>oxa-23</i>) and ARG-containing plasmids (in &gt;85% of sites) are more common in CTA sites. While CTA sites have higher turnover, the detection of highly similar strains over extended periods indicates that they have distinct reservoirs (for example, in ventilation or air-conditioning ducts) and that culture-based screening may bias against sites with lower biomass or variable colonization. Combining the strengths of metagenomics and culturing may therefore be needed to systematically explore the source of outbreaks.</p><p>Large-scale genomics of nosocomial pathogens through isolation can be laborious and time consuming, while metagenomics may not provide genomes for low abundance species. The intermediate approach proposed here addresses both issues. Culture-based enrichment allows us to shift the distribution away from abundant species (for example, <i>C. acnes</i>) and toward pathogens at low abundances (for example, <i>K. pneumoniae</i>, <i>S. aureus</i> and <i>A. baumannii</i>) while allowing functional selection such as for antibiotic resistance. Culture-based enrichment in combination with long-read metagenomics is powerful, enabling direct recovery of genomes (chromosomal, plasmid and phage) without isolation. With further automation (for example, library preparation), this workflow can enable high-throughput analysis and wider surveillance, to achieve the vision of precision epidemiology for infectious diseases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Ladner, J. T., Grubaugh, N. D., Pybus, O. G. &amp; Andersen, K. G. Precision epidemiology for infectious disease control. Nat. Med. 25, 206–211 (2019)." href="/articles/s41591-020-0894-4#ref-CR53" id="ref-link-section-d56841402e4009">53</a></sup>. Future improvements in nanopore sequencing throughput and lower DNA-input requirements could accelerate time-to-answer via point-of-care usage and reduce or eliminate the culturing period.</p><p>The availability of many high-contiguity assemblies (&gt;8 Gb; 2,347 genomes and 1,693 phage and 5,910 plasmid sequences) provides a unique resource for studying the distribution of strains and diversity of ARG cassettes in the hospital environment. Leveraging this, we observed that multidrug-resistant strains are preferentially distributed and persistent in hospitals across a range of species. This represents a worrisome pattern, with several explanations that warrant investigation. One scenario is that hospitals are repeatedly seeded by resistant strains that preferentially persist in the community (humans or environment). This explanation seems less plausible as some species where this pattern is observed are rarely found in humans (for example, <i>E. anophelis</i> and <i>A. baumannii</i>), and it is based on observations that other urban microbiomes are distinct from hospitals in taxonomic composition, the frequency at which they harbor pathogens and diversity of ARGs. Nevertheless, this does not rule out the possibility that urban environments (1) harbor pathogens and resistant strains at lower abundances compared to hospitals and (2) resistant strains are also widespread and persistent in these environments. Another hypothesis is that hospital cleaning measures select for more antibiotic-resistant organisms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Mahnert, A. et al. Man-made microbial resistances in built environments. Nat. Commun. 10, 968 (2019)." href="/articles/s41591-020-0894-4#ref-CR54" id="ref-link-section-d56841402e4022">54</a></sup>, a model that is supported by the presence of multiple copies of disinfectant resistance genes in widely distributed multidrug-resistant <i>S. aureus</i> strains in our study. Comparisons with surveys from built environments that are intensively cleaned but do not house patients (for example, operating rooms) or are not intensively cleaned but have high patient traffic (for example, clinic waiting areas) can help explore this hypothesis. Studies across wards and in hospitals with different protocols could also reveal how ARG reservoirs are shaped by cleaning practices<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Goodman, E. R. et al. Impact of an environmental cleaning intervention on the presence of methicillin-resistant Staphylococcus aureus and vancomycin-resistant enterococci on surfaces in intensive care unit rooms. Infect. Control Hosp. Epidemiol. 29, 593–599 (2008)." href="/articles/s41591-020-0894-4#ref-CR55" id="ref-link-section-d56841402e4029">55</a></sup>.</p><p>Despite their importance as an epicenter for the battle against growing antibiotic resistance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Fischer, M.M. &amp; Bild, M. Hospital use of antibiotics as the main driver of infections with antibiotic-resistant bacteria—a reanalysis of recent data from the European Union. Preprint at bioRxiv 
                  https://doi.org/10.1101/553537
                  
                 (2019)." href="/articles/s41591-020-0894-4#ref-CR1" id="ref-link-section-d56841402e4037">1</a></sup>, hospital environments have received little attention compared to agricultural and animal farms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Lax, S. &amp; Gilbert, J. A. Hospital-associated microbiota and implications for nosocomial infections. Trends Mol. Med. 21, 427–432 (2015)." href="/articles/s41591-020-0894-4#ref-CR12" id="ref-link-section-d56841402e4041">12</a></sup>. Our analysis highlights that hospitals harbor a significant uncharacterized diversity of microbes (<i>n</i> = 13 novel species) and ARG combinations (<i>n</i> = 255). This reservoir can be the origin of new opportunistic infections or fertile ground for the evolution of clinically relevant ARG combinations (for example, colistin and fosfomycin resistance). In particular, the prevalence of plasmids containing ARGs (<i>n</i> = 1,400) could enable gene transfer across species<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Lerminiaux, N. A. &amp; Cameron, A. D. S. Horizontal transfer of antibiotic resistance genes in clinical environments. Can. J. Microbiol. 65, 34–44 (2019)." href="/articles/s41591-020-0894-4#ref-CR56" id="ref-link-section-d56841402e4055">56</a></sup>. The development and use of anti-plasmid agents<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Buckner, M. M. C., Ciusa, M. L. &amp; Piddock, L. J. V. Strategies to combat antimicrobial resistance: anti-plasmid and plasmid curing. FEMS Microbiol. Rev. 42, 781–804 (2018)." href="/articles/s41591-020-0894-4#ref-CR57" id="ref-link-section-d56841402e4059">57</a></sup> could thus be a complementary strategy to curb the spread of ARGs through hospital environments.</p><p>While most studies have focused on patient isolates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="David, S. et al. Epidemic of carbapenem-resistant Klebsiella pneumoniae in Europe is driven by nosocomial spread. Nat. Microbiol. 4, 1919–1929 (2019)." href="/articles/s41591-020-0894-4#ref-CR58" id="ref-link-section-d56841402e4066">58</a></sup>, relatedness between environmental and patient-colonizing strains is important for understanding the risk that environmental strains pose<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Lax, S. et al. Bacterial colonization and succession in a newly opened hospital. Sci. Transl. Med. 9, eaah6500 (2017)." href="/articles/s41591-020-0894-4#ref-CR15" id="ref-link-section-d56841402e4070">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Brooks, B. et al. Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome. Nat. Commun. 8, 1814 (2017)." href="/articles/s41591-020-0894-4#ref-CR17" id="ref-link-section-d56841402e4073">17</a></sup>. For contemporary and co-located strains, high relatedness between a subset is expected. Despite samples being separated by &gt;8 years, obtaining highly similar genomes suggests that large reservoirs of multidrug-resistant strains are maintained with limited diversification. The identification and elimination of these reservoirs may reduce the incidence of corresponding infections and the risk from maintenance of ARGs. Another interesting observation is the high genomic similarity between MDROs in Singaporean hospitals and those from patients globally. The consistency of these patterns across species emphasizes the global dissemination of newly emerging MDRO lineages; thus, the role of hospital environments deserves investigation, leveraging multinational metagenomic datasets<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Danko, D.C. et al. Global genetic cartography of urban metagenomes and anti-microbial resistance. Preprint at bioRxiv 
                  https://doi.org/10.1101/724526
                  
                 (2019)." href="/articles/s41591-020-0894-4#ref-CR26" id="ref-link-section-d56841402e4077">26</a></sup>.</p><p>Overall, our data indicate selective advantages for MDROs to persist and spread in hospital environments (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig3">3d</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">11</a>) and be shared with patients (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig5">5</a>). The <i>S. aureus</i> derivative clusters that persisted in the hospital are enriched in virulence factors (1.5-fold; one-sided Wilcoxon <i>P</i> value = 0.015) and have three copies of disinfectant resistance genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Duran, N., Temiz, M., Duran, G. G., Eryilmaz, N. &amp; Jenedi, K. Relationship between the resistance genes to quaternary ammonium compounds and antibiotic resistance in staphylococci isolated from surgical site infections. Med. Sci. Monit. 20, 544–550 (2014)." href="/articles/s41591-020-0894-4#ref-CR38" id="ref-link-section-d56841402e4100">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Ho, C. M. et al. High rate of qacA- and qacB-positive methicillin-resistant Staphylococcus aureus isolates from chlorhexidine-impregnated catheter-related bloodstream infections. Antimicrob. Agents Chemother. 56, 5693–5697 (2012)." href="/articles/s41591-020-0894-4#ref-CR59" id="ref-link-section-d56841402e4103">59</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig3">3d</a>), potentially enabling colonization of hospital environments and patients and facilitating transfer between them. This points to a vicious cycle where disinfectant resistance, antibiotic resistance and virulence may in turn be selected for, enriching for strains adept at colonizing both niches with depleted microbial competition and offering an explanation for the high incidence of multidrug-resistant HAIs worldwide despite increased surveillance and aggressive cleaning measures in hospitals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Weiner, L. M. et al. Antimicrobial-resistant pathogens associated with healthcare-associated infections: summary of data reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2011–2014. Infect. Control Hosp. Epidemiol. 37, 1288–1301 (2016)." href="/articles/s41591-020-0894-4#ref-CR60" id="ref-link-section-d56841402e4110">60</a></sup>.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec8-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec8">Methods</h2><div class="c-article-section__content" id="Sec8-content"><h3 class="c-article__sub-heading" id="Sec9">Sample collection and storage</h3><p>Environmental swabs were collected from Tan Tock Seng Hospital (TTSH), a major tertiary-care hospital with &gt;2,000 patient visits daily, serving as the national referral center for communicable diseases in Singapore. Sampling was conducted in November 2017 and in May 2019. Samples were collected in 2 days for the first time point and in 3 days for the second time point, with 1 week separating the time points. The third time point was 1.5 years later, with samples collected in 4 days across 2 weeks. Samples were collected from isolation rooms (1 bed, typically for patients colonized with CRE), MDRO wards (5 beds, typically for patients colonized with MRSA) and standard wards (5 beds) at seven different sites, including the aerator, sink trap, bed rail, bedside locker, cardiac table, pulse oximeter and door handle (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-020-0894-4#Fig1">1</a> and Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM3">1</a>). Standard cleaning protocols at TTSH require that high-touch areas and sinks be cleaned daily with chlorine (5,000 ppm) and cleaning detergent, respectively, excluding beds that are cleaned upon patient discharge. Isohelix DNA Buccal Swabs (SK-4S) were used for sampling according to MetaSUB protocols<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Danko, D.C. et al. Global genetic cartography of urban metagenomes and anti-microbial resistance. Preprint at bioRxiv 
                  https://doi.org/10.1101/724526
                  
                 (2019)." href="/articles/s41591-020-0894-4#ref-CR26" id="ref-link-section-d56841402e4132">26</a></sup>. Briefly, a total of four swabs were collected; one swab (for culturing) was moistened with 1× PBS (pH 7.2), and three swabs (two swabs for metagenomic DNA isolation and one swab for storage) were moistened with DNA/RNA shield (Zymo Research, ZYR.R1100-250). Swabbing was performed for 2 min in each site, and swabs were stored in respective storage liquids (that is, 1× PBS, pH 7.2, or Zymo DNA/RNA shield). Swabs in PBS were placed on ice and sent for culturing while the other swabs were transported at room temperature to the laboratory and stored at −80 °C. In total, 1,752 swabs were collected from 179 sites in the hospital at three time points, representing 438 unique samples. Swabs were also collected from an office environment (Genome Institute of Singapore) with sites selected to approximately match those from which samples were collected in the hospital (aerator, sink trap, chair handle, office desk, keyboard and door handle; <i>n</i> = 30; Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM3">1</a>). MetaSUB Singapore samples were collected from high-touch surfaces in different parts of the city and analyzed based on MetaSUB protocols as described in Danko et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Danko, D.C. et al. Global genetic cartography of urban metagenomes and anti-microbial resistance. Preprint at bioRxiv 
                  https://doi.org/10.1101/724526
                  
                 (2019)." href="/articles/s41591-020-0894-4#ref-CR26" id="ref-link-section-d56841402e4143">26</a></sup> (<i>n</i> = 99; Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM3">1</a>).</p><h3 class="c-article__sub-heading" id="Sec10">DNA extraction from swabs</h3><p>DNA was extracted from swabs using a bead-beating and automated DNA purification system. Briefly, 300 µl of lysis buffer was added to Lysing Matrix E tubes (MP Biomedicals, 116914500). Samples were homogenized using the FastPrep-24 instrument at 6 m s<sup>–1</sup> for 40 s before centrifugation at maximum speed for 5 min. The supernatant was treated with proteinase K (Qiagen Singapore, 19133) for 20 min at 56 °C before DNA was purified with the Maxwell RSC Blood DNA Kit (Promega, AS1400). DNA concentration was quantified using a Qubit 2.0 fluorometer, prepared with the Qubit dsDNA HS Assay Kit (Life Technologies, Q32854). DNA extraction from backup swabs was carried out for samples with insufficient amounts of DNA. Samples that still had less than 0.5 ng of DNA were excluded from library preparation (10 of 438).</p><h3 class="c-article__sub-heading" id="Sec11">Illumina library preparation</h3><p>Extracted DNA was sheared using Adaptive Focused Acoustics (Covaris) with the following parameters: 240 s, duty factor of 30, PIP of 450 and 200 cycles per burst. Metagenomic libraries for the first two time points were prepared with the NEBNext Ultra DNA Kit (New England Biolabs, E7370) according to the manufacturer’s instructions. Paired-end sequencing (2 × 101-bp reads) was performed on the Illumina HiSeq2500 platform. For the third time point, metagenomic libraries were prepared using the NEBNext Ultra II DNA Library Prep Kit (New England Biolabs, E7645) according to the manufacturer’s instructions. Paired-end sequencing (2 × 151-bp reads) was performed on the Illumina HiSeq 4000 platform.</p><h3 class="c-article__sub-heading" id="Sec12">Culture enrichment</h3><p>Following MetaSUB protocols, swabs were directly incubated with 7 ml of Brain Heart Infusion (BHI) broth (Thermo Scientific Microbiology, CM1135B) at 37 °C until turbidity was observed (14–16 h for &gt;95% of samples), up to a maximum of 48 h. Culture tubes were centrifuged at 3,200<i>g</i> for 12 min. For the first two time points, cell pellets were resuspended with 550 µl of 1× PBS, while the cell pellets for the third time point were resuspended with 1 ml of 1× PBS. Fifty microliters of resuspended cultures was then plated on each of six agar plates (without antibiotics, BHI; ampicillin: 100 µg ml<sup>–1</sup>, AMP; streptomycin sulfate: 100 µg ml<sup>–1</sup>, STREP; tetracycline: 10 µg ml<sup>–1</sup>, TET; kanamycin: 50 µg ml<sup>–1</sup>, KAN; and chloramphenicol: 35 µg ml<sup>–1</sup>, CHLOR), and plates were incubated overnight at 37 °C. Cells were harvested by a plate sweep and were pelleted by centrifugation at 8,000<i>g</i> for 15 min at 4 °C for the first two time points. For the third time point, a loopful of harvested cells was streaked out on an antibiotic-free BHI plate to obtain single colonies for whole-genome sequencing. Plates were only excluded if no cells were growing on the plates or when the growth was insufficient to generate enough DNA for sequencing.</p><h3 class="c-article__sub-heading" id="Sec13">DNA extraction from enrichment cultures</h3><p>Frozen cells were thawed on ice and manually mixed with a wide-bore pipette tip. A volume of 30–50 µl of cells was resuspended in 100 µl of 1× PBS (pH 7.4). Twenty microliters of suspended cells was added to 20 µl of metapolyzyme (6.7 µg µl<sup>–1</sup>; Sigma Aldrich, MAC4L). The mixture was incubated at 35 °C for 4 h. RNase treatment was carried out by adding 350 µl of 1× TE buffer and 10 µl of RNase A (4 mg µl<sup>–1</sup>) and incubating on a rotator for 10 min at room temperature. DNA was extracted with the Maxwell RSC Cultured Cells Kit (Promega, AS1620). DNA was cleaned up and concentrated with 0.4× Agencourt AMPure XP beads (Beckman Coulter, A63882). DNA purity and concentration were measured with a NanoDrop and Qubit fluorometer. DNA integrity was assessed on a 0.5% agarose gel. DNA samples with the following quality measurements were selected for nanopore sequencing: DNA amount: &gt;400 ng; A260/280: 1.8–2.0; A260/230: 1.7–3.0; Qubit:NanoDrop: 0.7–1.3; DNA integrity on 0.5% agarose gel: &gt;1 kb. The Qubit:NanoDrop ratio was used to estimate and control the amount of single-stranded DNA in the sample and ensure successful nanopore sequencing.</p><h3 class="c-article__sub-heading" id="Sec14">Collection and testing of bacterial isolates from patients</h3><p><i>E. anophelis</i> isolates (<i>n</i> = 52) were obtained from consecutive positive blood cultures and respiratory samples collected in a 3-year period (2009–2012) at the National University Hospital in Singapore (DSRB reference 2017/00879). <i>A. baumannii</i> complex isolates (<i>n</i> = 108) were consecutively obtained from all clinical specimens (including blood, tissue, respiratory and urine samples) sent for routine bacterial culture between February 2009 and May 2009 at the Singapore General Hospital Diagnostic Bacteriology Laboratory (de-identified and archived, hence institutional review board approval was not required). Antibiotic susceptibility testing for <i>E. anophelis</i> isolates was performed with 13 antimicrobial agents (cefotaxime, ceftazadime, cefepime, imipenem, meropenem, ampicillin-sulbactam, piperacillin/tazobactam, tigecycline, gentamicin, nalidixic acid, ciprofloxacin, levofloxacin and trimethoprim/sulfamethoxazole) using Etest strips (bioMérieux). Minimum inhibitory concentrations (MICs) were interpreted according to the Clinical and Laboratory Standards Institute (CLSI) guidelines for non-Enterobacteriaceae Gram-negative bacilli (performance standards for antimicrobial susceptibility testing, M100-S22 and CLSI 2012; Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM8">6</a>). Antibiotics to which all strains were resistant were excluded from statistical analysis. Antibiotic susceptibility testing for <i>A. baumannii</i> complex isolates was conducted with 11 antimicrobial agents (ampicillin/sulbactam, piperacillin/tazobactam, cefepime, imipenem, gentamicin, amikacin, ciprofloxacin, levofloxacin, trimethoprim/sulfamethoxazole, minocycline and polymixin B). Polymixin B susceptibility testing was performed using Etest strips (bioMérieux), and disk diffusion was performed for all other antimicrobial agents. Polymixin B MICs and zone diameters for all other tested agents were interpreted in accordance with CLSI breakpoints for <i>Acinetobacter</i> spp. (performance standards for antimicrobial susceptibility testing, M100-S19 and CLSI 2009; Supplementary Data <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM8">6</a>). Multidrug-resistant status for patient isolates were defined according to US Centers for Disease Control and Prevention (CDC) guidelines (<a href="https://www.cdc.gov/nhsn/pdfs/ps-analysis-resources/phenotype_definitions.pdf">https://www.cdc.gov/nhsn/pdfs/ps-analysis-resources/phenotype_definitions.pdf</a>).</p><h3 class="c-article__sub-heading" id="Sec15">DNA extraction for bacterial isolates</h3><p>Cell pellets were allowed to thaw slowly on ice and resuspended in 400 µl of ATL buffer (Qiagen Singapore, 19076). Cells were lysed in Lysing Matrix E tubes (MP Biomedicals, 116914500) on a vortex adapter at maximum speed for 10 min. Cell lysates were centrifuged at 16,000<i>g</i> for 5 min, and supernatant was treated with 4 µl of RNase A (100 mg ml<sup>–1</sup>; Qiagen Singapore, 19101), gently mixed by flicking of the tube and incubated at room temperature for 2 min. The cell lysate was further treated with 25 µl of proteinase K (20 mg ml<sup>–1</sup>; Qiagen Singapore, 19133), gently mixed by flicking of the tube and incubated at 56 °C for 20 min. DNA was purified twice using 1 volume of AMPure XP beads (Beckman Coulter, A63882) with slight modifications to the manufacturer’s protocol. All mixing steps were replaced with gentle flicking of the tube and incubation on the hula rotor for gentle mixing. Fresh 70% ethanol was prepared for washing, and magnetic beads were incubated on a 37 °C heat block for 3–5 min to dry off residual ethanol. The quality and quantity of DNA were assessed using a NanoDrop, Qubit fluorometer and 0.5% agarose gel. Samples that were unable to pass the following criteria were omitted from sequencing: DNA amount measured by Qubit: &gt;510 ng; DNA concentration measured by Qubit: &gt;11 ng µl<sup>–1</sup>. A260/280 ratio: between 1.7–2.0; A260/230 ratio: between 1.5–3.3; and DNA length: &gt;1 kb. Purified DNA was stored at 4 °C.</p><h3 class="c-article__sub-heading" id="Sec16">Nanopore library preparation</h3><p>DNA was prepared with either the 1D<sup>2</sup> sequencing kit (SQK-LSK308) or the 1D sequencing kit (SQK-LSK108 or SQK-LSK109) together with the native barcoding kit (EXP-NBD103 or EXP-NBD104 and EXP-NBD114) according to the native barcoding genomic DNA protocol. DNA was not sheared and was used directly for DNA repair and end preparation. Both native barcode ligation and adaptor ligation steps were extended to 30 min instead of 10 min. In addition, to maximize library yields, more than 700 ng of pooled sample (where possible) was used for adaptor ligation. Samples were multiplexed (9–12 samples for each pool for culture-enriched samples and 24 samples for each pool for isolates) and sequenced with MIN106, MIN106D or MIN107 flowcells on a GridION machine.</p><h3 class="c-article__sub-heading" id="Sec17">Taxonomic and resistome profiling with Illumina shotgun metagenomic data</h3><p>Illumina shotgun metagenomic sequencing reads were processed using a Snakemake pipeline (<a href="https://github.com/gis-rpd/pipelines/tree/master/metagenomics/shotgun-metagenomics">https://github.com/gis-rpd/pipelines/tree/master/metagenomics/shotgun-metagenomics</a>). Briefly, raw reads were filtered to remove low-quality bases using skewer (v0.2.2; -q 3 -l 30 -n) and human reads were removed by mapping to the hg19 reference using BWA-MEM (v0.7.10-r789). The remaining microbial reads were profiled with MetaPhlAn2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Truong, D. T. et al. MetaPhlAn2 for enhanced metagenomic taxonomic profiling. Nat. Methods 12, 902–903 (2015)." href="/articles/s41591-020-0894-4#ref-CR61" id="ref-link-section-d56841402e4293">61</a></sup>; v2.6.0) and SRST2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Inouye, M. et al. SRST2: rapid genomic surveillance for public health and hospital microbiology labs. Genome Med. 6, 90 (2014)." href="/articles/s41591-020-0894-4#ref-CR62" id="ref-link-section-d56841402e4297">62</a></sup>; v0.1.4; --min_coverage 100, hits with identity &lt;99% were filtered out) for taxonomic and ARG abundances, respectively. Microbial reads were also assembled using MEGAHIT (v1.0.4-beta; default parameters) for comparison to nanopore assemblies. The site specificity score was computed as the <i>z</i>-score for the closest taxonomic profile for a sample (Bray–Curtis dissimilarity) among physically proximal sites (in the same room or cubicle and at the same time point), compared to the distribution of Bray–Curtis dissimilarities across all samples of a site (for example, all bed rails). Results based on analysis of taxonomic and resistome profiles were obtained for each time point independently and compared across time points to check for consistency and filter out potential sequencing artefacts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="de Goffau, M. C. et al. Recognizing the reagent microbiome. Nat. Microbiol. 3, 851–853 (2018)." href="/articles/s41591-020-0894-4#ref-CR23" id="ref-link-section-d56841402e4304">23</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec18">Removal of likely contaminant species</h3><p>Likely contaminant species were identified based on batch and correlation analysis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="de Goffau, M. C. et al. Recognizing the reagent microbiome. Nat. Microbiol. 3, 851–853 (2018)." href="/articles/s41591-020-0894-4#ref-CR23" id="ref-link-section-d56841402e4316">23</a></sup> (Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM1">2</a>) and were removed from species-level abundance profiles. For genus-level profiles, relative abundances of the filtered species were subtracted from the abundance of the corresponding genera for each sample. Filtered profiles were then renormalized to sum to 100% and used for all downstream analyses.</p><h3 class="c-article__sub-heading" id="Sec19">Preprocessing of nanopore sequencing data</h3><p>Raw nanopore reads were base-called with the latest version of the basecaller available at the point of sequencing (Guppy v0.5.1 to v3.0.6 or Albacore v2.3.1 to v2.3.3, for libraries that failed live base-calling). Base-called nanopore reads were demultiplexed and filtered for adaptors with Porechop (v0.2.3; <a href="https://github.com/rrwick/Porechop">https://github.com/rrwick/Porechop</a>) or qcat (v.1.1.0; <a href="https://github.com/nanoporetech/qcat">https://github.com/nanoporetech/qcat</a>). Sequencing statistics were summarized using SeqKit (v0.10.1). Reads were taxonomically classified with Kraken<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Wood, D. E. &amp; Salzberg, S. L. Kraken: ultrafast metagenomic sequence classification using exact alignments. Genome Biol. 15, r46 (2014)." href="/articles/s41591-020-0894-4#ref-CR63" id="ref-link-section-d56841402e4346">63</a></sup> (v0.10.5-beta) against the miniKraken database to assess the diversity of cultures on the plates (minikraken_201711_01_8GB_dustmasked).</p><h3 class="c-article__sub-heading" id="Sec20">Genome assembly and species assignment</h3><p>Nanopore reads for each plate were assembled using Canu<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Koren, S. et al. Canu: scalable and accurate long-read assembly via adaptive k-mer weighting and repeat separation. Genome Res. 27, 722–736 (2017)." href="/articles/s41591-020-0894-4#ref-CR64" id="ref-link-section-d56841402e4358">64</a></sup> (v1.3 and v1.7; genomeSize = 8 m). For samples where both Illumina and nanopore reads were available, a higher-quality hybrid assembly was obtained using OPERA-MS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Bertrand, D. et al. Hybrid metagenomic assembly enables high-resolution analysis of resistance determinants and mobile elements in human microbiomes. Nat. Biotechnol. 37, 937–944 (2019)." href="/articles/s41591-020-0894-4#ref-CR19" id="ref-link-section-d56841402e4362">19</a></sup> (v0.8.3; --polish --no-gap-filling --short-read-assembler spades). Assembled contigs were mapped to the NCBI nt database with BLAST (v2.2.28), to identify microbial species or plasmid assignments according to the best BLAST hit (highest total reference coverage). Circular sequences were identified using MUMmer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Kurtz, S. et al. Versatile and open software for comparing large genomes. Genome Biol. 5, r12 (2004)." href="/articles/s41591-020-0894-4#ref-CR65" id="ref-link-section-d56841402e4366">65</a></sup> (v3.23; --maxmatch --nosimplify, alignments &lt;1 kb long or with identity &lt;95% were filtered out) as recommended in the documentation for Canu (<a href="https://canu.readthedocs.io/en/latest/faq.html#my-circular-element-is-duplicated-has-overlap">https://canu.readthedocs.io/en/latest/faq.html#my-circular-element-is-duplicated-has-overlap</a>). Contigs assigned to the same species were binned into genomic bins. Metagenomic Illumina reads were used to polish Canu assemblies where feasible using Pilon<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Walker, B. J. et al. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. PLoS ONE 9, e112963 (2014)." href="/articles/s41591-020-0894-4#ref-CR66" id="ref-link-section-d56841402e4377">66</a></sup> (v1.22; --fix indel). We noted that annotation errors were substantially reduced after polishing and that genomic bins whose length was within 10% of the expected length met the criteria for high-quality genomes (completeness &gt; 90% and contamination &lt; 5% using CheckM<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. &amp; Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Genome Res. 25, 1043–1055 (2015)." href="/articles/s41591-020-0894-4#ref-CR67" id="ref-link-section-d56841402e4382">67</a></sup>; v1.0.7; --reduced_tree). Genomic bins that met these criteria were therefore designated as high quality, and incomplete bins (&lt;50% of the expected length) were removed from further analysis. Genomes corresponding to novel species were identified as those with identity &lt;95% or coverage &lt;80% when compared with known genomes (BLAST with nt) and three recent catalogs that include environmental and human microbiome assembled genomes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Pasolli, E. et al. Extensive unexplored human microbiome diversity revealed by over 150,000 genomes from metagenomes spanning age, geography, and lifestyle. Cell 176, 649–662 (2019)." href="#ref-CR31" id="ref-link-section-d56841402e4386">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Almeida, A. et al. A new genomic blueprint of the human gut microbiota. Nature 568, 499–504 (2019)." href="#ref-CR32" id="ref-link-section-d56841402e4386_1">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Parks, D. H. et al. A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life. Nat. Biotechnol. 36, 996–1004 (2018)." href="/articles/s41591-020-0894-4#ref-CR33" id="ref-link-section-d56841402e4389">33</a></sup> (with Mash<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. Genome Biol. 17, 132 (2016)." href="/articles/s41591-020-0894-4#ref-CR68" id="ref-link-section-d56841402e4393">68</a></sup>). The genomes were hierarchically clustered (single linkage with Mash distance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. Genome Biol. 17, 132 (2016)." href="/articles/s41591-020-0894-4#ref-CR68" id="ref-link-section-d56841402e4397">68</a></sup>) to identify species-level clusters at 95% identity, and genus-level taxonomic classification was obtained using sourmash<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Pierce, N. T., Irber, L., Reiter, T., Brooks, P. &amp; Brown, C. T. Large-scale sequence comparisons with sourmash. F1000Res 8, 1006 (2019)." href="/articles/s41591-020-0894-4#ref-CR69" id="ref-link-section-d56841402e4401">69</a></sup>. Similarly, novel circular plasmids were identified by comparing to the PLSDB<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Galata, V., Fehlmann, T., Backes, C. &amp; Keller, A. PLSDB: a resource of complete bacterial plasmids. Nucleic Acids Res. 47, D195–D202 (2019)." href="/articles/s41591-020-0894-4#ref-CR36" id="ref-link-section-d56841402e4405">36</a></sup> database with Mash distance and identifying clusters at 99% identity (single linkage) with no known sequence.</p><h3 class="c-article__sub-heading" id="Sec21">Analysis of ARG combinations</h3><p>ARGs were annotated to contigs by mapping them to the ARG-ANNOT<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Gupta, S. K. et al. ARG-ANNOT, a new bioinformatic tool to discover antibiotic resistance genes in bacterial genomes. Antimicrob. Agents Chemother. 58, 212–220 (2014)." href="/articles/s41591-020-0894-4#ref-CR70" id="ref-link-section-d56841402e4417">70</a></sup> database provided in SRST2 (v3) with BLAST (best hit with &gt;90% identity and &gt;90% reference coverage). ARG combinations present in chromosomes and plasmid sequences were considered novel when they were not found in the reference databases (nt or PLSDB<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Galata, V., Fehlmann, T., Backes, C. &amp; Keller, A. PLSDB: a resource of complete bacterial plasmids. Nucleic Acids Res. 47, D195–D202 (2019)." href="/articles/s41591-020-0894-4#ref-CR36" id="ref-link-section-d56841402e4421">36</a></sup>). Assembled circular plasmids were clustered and annotated based on their best BLAST hit with identity &gt;95% and &gt;60% query coverage. A bipartite graph was constructed by connecting each plasmid cluster to ARGs found in it, with edge weights representing the frequency of occurrence (clusters with &lt;5 representatives were excluded). For each species, an ARG co-occurrence graph was created for ARGs found in the assembled genomes by connecting the ARG pairs that were found within 10 kb on the same contig (discarding ARG pairs occurring fewer than five times). Each edge was weighted by the frequency of ARG pairs divided by the minimal frequency of the two ARGs. All ARG co-occurrence graphs were merged into a final co-occurrence multigraph. The graphs were visualized using Cytoscape (v3.7.1).</p><h3 class="c-article__sub-heading" id="Sec22">Analysis of virulence factor and biocide resistance genes</h3><p>Nanopore assemblies were aligned to virulence factors in the PATRIC database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Wattam, A. R. et al. PATRIC, the bacterial bioinformatics database and analysis resource. Nucleic Acids Res. 42, D581–D591 (2014)." href="/articles/s41591-020-0894-4#ref-CR71" id="ref-link-section-d56841402e4433">71</a></sup> (20 December 2018) with DIAMOND (v0.9.24.125; blastx --long-reads), and alignments with <i>E</i> value &gt; 0.001 were filtered out. To identify biocide resistance genes, the assemblies were aligned to nucleotide sequences for the genes <i>qacA</i> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_014369.1">NC_014369.1</a>) and <i>qacC</i> (<a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_013339.1">NC_013339.1</a>) with BLAST (&gt;90% identity and &gt;90% reference coverage).</p><h3 class="c-article__sub-heading" id="Sec23">Analysis of phages and prophages</h3><p>Phage-like elements (phages and prophages) were identified using VirSorter<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Roux, S., Enault, F., Hurwitz, B. L. &amp; Sullivan, M. B. VirSorter: mining viral signal from microbial genomic data. PeerJ 3, e985 (2015)." href="/articles/s41591-020-0894-4#ref-CR72" id="ref-link-section-d56841402e4469">72</a></sup> (v1.0.5; phages and prophages in category 3 or with length &lt;10 kb were filtered out). The assembled phages and reference phages from the MVP database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Gao, N. L. et al. MVP: a microbe–phage interaction database. Nucleic Acids Res 46, D700–D707 (2018)." href="/articles/s41591-020-0894-4#ref-CR37" id="ref-link-section-d56841402e4473">37</a></sup> were hierarchically clustered (single linkage with Mash distance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. Genome Biol. 17, 132 (2016)." href="/articles/s41591-020-0894-4#ref-CR68" id="ref-link-section-d56841402e4477">68</a></sup>) to identify phage clusters at 95% identity. Clusters without any phages from the reference database were considered novel. For each cluster, subclusters were defined at 99.9% ANI by single-linkage clustering with nucleotide identities from nucmer (--maxmatch --nosimplify, followed by dnadiff and minimum sequence overlap of 80%). Phage-like elements were annotated using RAST<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Aziz, R. K. et al. The RAST Server: rapid annotations using subsystems technology. BMC Genomics 9, 75 (2008)." href="/articles/s41591-020-0894-4#ref-CR73" id="ref-link-section-d56841402e4481">73</a></sup> (virus domain, fix frame-shifts parameters).</p><h3 class="c-article__sub-heading" id="Sec24">Analysis of patient isolates and strain relationships</h3><p>Raw reads corresponding to genomes for outbreak isolates<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lee, J. Y. H. et al. Global spread of three multidrug-resistant lineages of Staphylococcus epidermidis. Nat. Microbiol. 3, 1175–1185 (2018)." href="/articles/s41591-020-0894-4#ref-CR29" id="ref-link-section-d56841402e4493">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Teo, J. et al. First case of E. anophelis outbreak in an intensive-care unit. Lancet 382, 855–856 (2013)." href="#ref-CR49" id="ref-link-section-d56841402e4496">49</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Htun, H. L. et al. Methicillin-resistant Staphylococcus aureus colonisation: epidemiological and molecular characteristics in an acute-care tertiary hospital in Singapore. Epidemiol. Infect. 146, 1785–1792 (2018)." href="#ref-CR50" id="ref-link-section-d56841402e4496_1">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Nasser, K. et al. Draft genome sequences of six multidrug-resistant clinical strains of Acinetobacter baumannii, isolated at two major hospitals in Kuwait. Genome Announc. 6, e00264-18 (2018)." href="/articles/s41591-020-0894-4#ref-CR51" id="ref-link-section-d56841402e4499">51</a></sup> were downloaded and assembled using the Velvet assembler (v1.2.10) with parameters optimized by Velvet Optimiser (<i>k</i>-mer length: ranging from 81–127), scaffolded with OPERA-LG<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Gao, S., Bertrand, D., Chia, B. K. &amp; Nagarajan, N. OPERA-LG: efficient and exact scaffolding of large, repeat-rich eukaryotic genomes with performance guarantees. Genome Biol. 17, 102 (2016)." href="/articles/s41591-020-0894-4#ref-CR74" id="ref-link-section-d56841402e4506">74</a></sup> (v1.4.1) and gap-filled with FinIS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Gao, S., Bertrand, D., &amp; Nagarajan, N. FinIS: improved in silico finishing using an exact quadratic programming formulation. In Algorithms in Bioinformatics (Eds. Raphael, B. &amp; Tang, J.) 314–325 (Springer, 2012)." href="/articles/s41591-020-0894-4#ref-CR75" id="ref-link-section-d56841402e4510">75</a></sup> (v0.3). Outbreak genomes from the same species were jointly analyzed with high-quality genomes from the hospital environment. To identify high-confidence SNPs, we adapted the method from Brooks et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Brooks, B. et al. Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome. Nat. Commun. 8, 1814 (2017)." href="/articles/s41591-020-0894-4#ref-CR17" id="ref-link-section-d56841402e4514">17</a></sup>. Specifically, we performed pairwise alignments between genomes using nucmer and considered genome pairs with alignment coverage &gt; 80% for ANI computation. SNPs between genome pairs were called using MUMmer’s ‘show-snps’ function, and regions containing more than one SNP within 20 bp were filtered out to mask potential artefacts from horizontal gene transfer, recombination or repeats. Finally, the genomic distance matrix (number of SNPs/alignment size) was clustered hierarchically (single linkage) and clusters were obtained at 99.99% identity for species with hybrid assemblies (nanopore and Illumina) or at 99.9% identity for species with nanopore-only assemblies. Single-linkage clustering was used to avoid having highly similar genomes assigned to separate clusters, and we confirmed that despite this, most members (99%) had an average distance to other members of the cluster below the clustering thresholds used. Antibiotic resistance profiles and multidrug resistance status (&gt;2 antibiotic types) for each cluster were derived from the union of resistance profiles for each genome obtained in various selection plates.</p><p>For phylogenetic analysis, a consensus genome was derived for each cluster based on reference-guided alignment with nucmer (<i>S. aureus</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_020529">NC_020529</a>; <i>S. epidermidis</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_004461">NC_004461</a>; <i>E. anophelis</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NZ_CP007547">NZ_CP007547</a>; <i>E. faecalis</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_017312">NC_017312</a>; <i>E. faecium</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_017960">NC_017960</a>; <i>P. aeruginosa</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_018080">NC_018080</a>; <i>K. pneumoniae</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_018522">NC_018522</a>; and <i>A. baumannii</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NC_009085">NC_009085</a>) and the cons utility in the EMBOSS suite. Maximum-likelihood phylogenetic trees were constructed for each species with Parsnp<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Treangen, T. J., Ondov, B. D., Koren, S. &amp; Phillippy, A. M. The Harvest suite for rapid core-genome alignment and visualization of thousands of intraspecific microbial genomes. Genome Biol. 15, 524 (2014)." href="/articles/s41591-020-0894-4#ref-CR76" id="ref-link-section-d56841402e4603">76</a></sup> (v1.2; -c -x; accounting for recombination events using PhiPack<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Bruen, T. C., Philippe, H. &amp; Bryant, D. A simple and robust statistical test for detecting the presence of recombination. Genetics 172, 2665–2681 (2006)." href="/articles/s41591-020-0894-4#ref-CR77" id="ref-link-section-d56841402e4608">77</a></sup>) based on consensus genomes for each cluster, where multiple-sequence alignments for each species varied in length from 0.6 Mb (<i>S. epidermidis</i>) to 5.1 Mb (<i>P. aeruginosa</i>). For the species-level tree, full-length 16S rRNA sequences (<i>S. epidermidis</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/L37605">L37605.1</a>; <i>S. aureus</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_118997.2">NR_118997.2</a>; <i>E. anophelis</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_116021.1">NR_116021.1</a>; and <i>A. baumannii</i>: <a href="https://www.ncbi.nlm.nih.gov/nuccore/NR_026206.1">NR_026206.1</a>) were aligned with MAFFT (v7.154b; default parameters) and the phylogeny was determined using FastTree2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Price, M. N., Dehal, P. S. &amp; Arkin, A. P. FastTree 2—approximately maximum-likelihood trees for large alignments. PLoS ONE 5, e9490 (2010)." href="/articles/s41591-020-0894-4#ref-CR78" id="ref-link-section-d56841402e4659">78</a></sup>; v2.1.8; default parameters). The trees were visualized using the ‘ggtree’ R package<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Yu, G., Lam, T. T., Zhu, H. &amp; Guan, Y. Two methods for mapping and visualizing associated data on phylogeny using ggtree. Mol. Biol. Evol. 35, 3041–3043 (2018)." href="/articles/s41591-020-0894-4#ref-CR79" id="ref-link-section-d56841402e4664">79</a></sup>. Strain distributions across sites were visualized with the ‘HiveR’ R package (<a href="https://github.com/bryanhanson/HiveR">https://github.com/bryanhanson/HiveR</a>). Rarefaction analysis for species, plasmids, strains and resistance genes was performed using the iNEXT R package<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Hsieh, T. C., Ma, K. H. &amp; Chao, A. iNEXT: an R package for rarefaction and extrapolation of species diversity (Hill numbers). Methods Ecol. Evol. 7, 1451–1456 (2016)." href="/articles/s41591-020-0894-4#ref-CR80" id="ref-link-section-d56841402e4675">80</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec25">Statistical analysis</h3><p>Statistical tests were performed using R and were two sided unless otherwise specified. For enrichment analysis at the cluster level (overlap across time, cohorts or resistance status), Fisher’s exact test was used. The binomial test was used for analysis at the genome level (fraction of genomes with a specific property).</p><h3 class="c-article__sub-heading" id="Sec26">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-020-0894-4#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>Sequencing reads and assemblies are available from the European Nucleotide Archive under project <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB31632">PRJEB31632</a>. Source code and data for reproducing figures are available under MIT license at <a href="https://github.com/csb5/hospital_microbiome">https://github.com/csb5/hospital_microbiome</a>. Assemblies and annotations for genomes, plasmids and phages are available at <a href="https://t.co/bdZxADGM7z">https://t.co/bdZxADGM7z</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Fischer, M.M. &amp; Bild, M. Hospital use of antibiotics as the main driver of infections with antibiotic-resistant bacteria—a reanalysis of recent data from the European Union. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/553537" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/553537">https://doi.org/10.1101/553537</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">O’Neill, J. Tackling drug-resistant infections globally: final report and recommendations. In <i>Review on Antimicrobial Resistance</i> (2016).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Allegranzi, B. et al. Burden of endemic health-care-associated infection in developing countries: systematic review and meta-analysis. <i>Lancet</i> <b>377</b>, 228–241 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(10)61458-4" data-track-item_id="10.1016/S0140-6736(10)61458-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2810%2961458-4" aria-label="Article reference 3" data-doi="10.1016/S0140-6736(10)61458-4">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21146207" aria-label="PubMed reference 3">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Burden%20of%20endemic%20health-care-associated%20infection%20in%20developing%20countries%3A%20systematic%20review%20and%20meta-analysis&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2810%2961458-4&amp;volume=377&amp;pages=228-241&amp;publication_year=2011&amp;author=Allegranzi%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Magill, S. S. et al. Multistate point-prevalence survey of health care-associated infections. <i>N. Engl. J. Med.</i> <b>370</b>, 1198–1208 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa1306801" data-track-item_id="10.1056/NEJMoa1306801" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa1306801" aria-label="Article reference 4" data-doi="10.1056/NEJMoa1306801">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXlvVCitLo%3D" aria-label="CAS reference 4">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24670166" aria-label="PubMed reference 4">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4648343" aria-label="PubMed Central reference 4">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Multistate%20point-prevalence%20survey%20of%20health%20care-associated%20infections&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1306801&amp;volume=370&amp;pages=1198-1208&amp;publication_year=2014&amp;author=Magill%2CSS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Laxminarayan, R. et al. Antibiotic resistance—the need for global solutions. <i>Lancet Infect. Dis.</i> <b>13</b>, 1057–1098 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S1473-3099(13)70318-9" data-track-item_id="10.1016/S1473-3099(13)70318-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS1473-3099%2813%2970318-9" aria-label="Article reference 5" data-doi="10.1016/S1473-3099(13)70318-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24252483" aria-label="PubMed reference 5">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Antibiotic%20resistance%E2%80%94the%20need%20for%20global%20solutions&amp;journal=Lancet%20Infect.%20Dis.&amp;doi=10.1016%2FS1473-3099%2813%2970318-9&amp;volume=13&amp;pages=1057-1098&amp;publication_year=2013&amp;author=Laxminarayan%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Longtin, Y., Sax, H., Allegranzi, B., Schneider, F. &amp; Pittet, D. Videos in clinical medicine. Hand hygiene. <i>N. Engl. J. Med.</i> <b>364</b>, e24 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMvcm0903599" data-track-item_id="10.1056/NEJMvcm0903599" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMvcm0903599" aria-label="Article reference 6" data-doi="10.1056/NEJMvcm0903599">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3MXkt1Cgsro%3D" aria-label="CAS reference 6">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21449775" aria-label="PubMed reference 6">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Videos%20in%20clinical%20medicine.%20Hand%20hygiene&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMvcm0903599&amp;volume=364&amp;publication_year=2011&amp;author=Longtin%2CY&amp;author=Sax%2CH&amp;author=Allegranzi%2CB&amp;author=Schneider%2CF&amp;author=Pittet%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Otter, J. A., Yezli, S., Salkeld, J. A. &amp; French, G. L. Evidence that contaminated surfaces contribute to the transmission of hospital pathogens and an overview of strategies to address contaminated surfaces in hospital settings. <i>Am. J. Infect. Control</i> <b>41</b>, S6–S11 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ajic.2012.12.004" data-track-item_id="10.1016/j.ajic.2012.12.004" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajic.2012.12.004" aria-label="Article reference 7" data-doi="10.1016/j.ajic.2012.12.004">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23622751" aria-label="PubMed reference 7">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Evidence%20that%20contaminated%20surfaces%20contribute%20to%20the%20transmission%20of%20hospital%20pathogens%20and%20an%20overview%20of%20strategies%20to%20address%20contaminated%20surfaces%20in%20hospital%20settings&amp;journal=Am.%20J.%20Infect.%20Control&amp;doi=10.1016%2Fj.ajic.2012.12.004&amp;volume=41&amp;pages=S6-S11&amp;publication_year=2013&amp;author=Otter%2CJA&amp;author=Yezli%2CS&amp;author=Salkeld%2CJA&amp;author=French%2CGL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Mitchell, B. G. et al. An environmental cleaning bundle and health-care-associated infections in hospitals (REACH): a multicentre, randomised trial. <i>Lancet Infect. Dis.</i> <b>19</b>, 410–418 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S1473-3099(18)30714-X" data-track-item_id="10.1016/S1473-3099(18)30714-X" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS1473-3099%2818%2930714-X" aria-label="Article reference 8" data-doi="10.1016/S1473-3099(18)30714-X">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30858014" aria-label="PubMed reference 8">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20environmental%20cleaning%20bundle%20and%20health-care-associated%20infections%20in%20hospitals%20%28REACH%29%3A%20a%20multicentre%2C%20randomised%20trial&amp;journal=Lancet%20Infect.%20Dis.&amp;doi=10.1016%2FS1473-3099%2818%2930714-X&amp;volume=19&amp;pages=410-418&amp;publication_year=2019&amp;author=Mitchell%2CBG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Price, J. R. et al. Transmission of <i>Staphylococcus aureus</i> between health-care workers, the environment, and patients in an intensive care unit: a longitudinal cohort study based on whole-genome sequencing. <i>Lancet Infect. Dis.</i> <b>17</b>, 207–214 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S1473-3099(16)30413-3" data-track-item_id="10.1016/S1473-3099(16)30413-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS1473-3099%2816%2930413-3" aria-label="Article reference 9" data-doi="10.1016/S1473-3099(16)30413-3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27863959" aria-label="PubMed reference 9">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5266793" aria-label="PubMed Central reference 9">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Transmission%20of%20Staphylococcus%20aureus%20between%20health-care%20workers%2C%20the%20environment%2C%20and%20patients%20in%20an%20intensive%20care%20unit%3A%20a%20longitudinal%20cohort%20study%20based%20on%20whole-genome%20sequencing&amp;journal=Lancet%20Infect.%20Dis.&amp;doi=10.1016%2FS1473-3099%2816%2930413-3&amp;volume=17&amp;pages=207-214&amp;publication_year=2017&amp;author=Price%2CJR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Weingarten, R. A. et al. Genomic analysis of hospital plumbing reveals diverse reservoir of bacterial plasmids conferring carbapenem resistance. <i>MBio</i> <b>9</b>, e02011-17 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/mBio.02011-17" data-track-item_id="10.1128/mBio.02011-17" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FmBio.02011-17" aria-label="Article reference 10" data-doi="10.1128/mBio.02011-17">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29437920" aria-label="PubMed reference 10">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5801463" aria-label="PubMed Central reference 10">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20analysis%20of%20hospital%20plumbing%20reveals%20diverse%20reservoir%20of%20bacterial%20plasmids%20conferring%20carbapenem%20resistance&amp;journal=MBio&amp;doi=10.1128%2FmBio.02011-17&amp;volume=9&amp;publication_year=2018&amp;author=Weingarten%2CRA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Conlan, S. et al. Single-molecule sequencing to track plasmid diversity of hospital-associated carbapenemase-producing Enterobacteriaceae. <i>Sci. Transl. Med.</i> <b>6</b>, 254ra126 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/scitranslmed.3009845" data-track-item_id="10.1126/scitranslmed.3009845" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscitranslmed.3009845" aria-label="Article reference 11" data-doi="10.1126/scitranslmed.3009845">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25232178" aria-label="PubMed reference 11">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4203314" aria-label="PubMed Central reference 11">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXitVWqsLzE" aria-label="CAS reference 11">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Single-molecule%20sequencing%20to%20track%20plasmid%20diversity%20of%20hospital-associated%20carbapenemase-producing%20Enterobacteriaceae&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.3009845&amp;volume=6&amp;publication_year=2014&amp;author=Conlan%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Lax, S. &amp; Gilbert, J. A. Hospital-associated microbiota and implications for nosocomial infections. <i>Trends Mol. Med.</i> <b>21</b>, 427–432 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.molmed.2015.03.005" data-track-item_id="10.1016/j.molmed.2015.03.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.molmed.2015.03.005" aria-label="Article reference 12" data-doi="10.1016/j.molmed.2015.03.005">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25907678" aria-label="PubMed reference 12">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Hospital-associated%20microbiota%20and%20implications%20for%20nosocomial%20infections&amp;journal=Trends%20Mol.%20Med.&amp;doi=10.1016%2Fj.molmed.2015.03.005&amp;volume=21&amp;pages=427-432&amp;publication_year=2015&amp;author=Lax%2CS&amp;author=Gilbert%2CJA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Miller, R. R., Montoya, V., Gardy, J. L., Patrick, D. M. &amp; Tang, P. Metagenomics for pathogen detection in public health. <i>Genome Med.</i> <b>5</b>, 81 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/gm485" data-track-item_id="10.1186/gm485" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/gm485" aria-label="Article reference 13" data-doi="10.1186/gm485">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24050114" aria-label="PubMed reference 13">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3978900" aria-label="PubMed Central reference 13">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXitVGjtro%3D" aria-label="CAS reference 13">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Metagenomics%20for%20pathogen%20detection%20in%20public%20health&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fgm485&amp;volume=5&amp;publication_year=2013&amp;author=Miller%2CRR&amp;author=Montoya%2CV&amp;author=Gardy%2CJL&amp;author=Patrick%2CDM&amp;author=Tang%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Oberauner, L. et al. The ignored diversity: complex bacterial communities in intensive care units revealed by 16S pyrosequencing. <i>Sci. Rep.</i> <b>3</b>, 1413 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/srep01413" data-track-item_id="10.1038/srep01413" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fsrep01413" aria-label="Article reference 14" data-doi="10.1038/srep01413">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23475210" aria-label="PubMed reference 14">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3593336" aria-label="PubMed Central reference 14">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXpvFKjsrg%3D" aria-label="CAS reference 14">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20ignored%20diversity%3A%20complex%20bacterial%20communities%20in%20intensive%20care%20units%20revealed%20by%2016S%20pyrosequencing&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fsrep01413&amp;volume=3&amp;publication_year=2013&amp;author=Oberauner%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Lax, S. et al. Bacterial colonization and succession in a newly opened hospital. <i>Sci. Transl. Med.</i> <b>9</b>, eaah6500 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/scitranslmed.aah6500" data-track-item_id="10.1126/scitranslmed.aah6500" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscitranslmed.aah6500" aria-label="Article reference 15" data-doi="10.1126/scitranslmed.aah6500">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28539477" aria-label="PubMed reference 15">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5706123" aria-label="PubMed Central reference 15">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Bacterial%20colonization%20and%20succession%20in%20a%20newly%20opened%20hospital&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.aah6500&amp;volume=9&amp;publication_year=2017&amp;author=Lax%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Janda, J. M. &amp; Abbott, S. L. 16S rRNA gene sequencing for bacterial identification in the diagnostic laboratory: pluses, perils, and pitfalls. <i>J. Clin. Microbiol.</i> <b>45</b>, 2761–2764 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/JCM.01228-07" data-track-item_id="10.1128/JCM.01228-07" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FJCM.01228-07" aria-label="Article reference 16" data-doi="10.1128/JCM.01228-07">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFemt7bI" aria-label="CAS reference 16">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17626177" aria-label="PubMed reference 16">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2045242" aria-label="PubMed Central reference 16">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=16S%20rRNA%20gene%20sequencing%20for%20bacterial%20identification%20in%20the%20diagnostic%20laboratory%3A%20pluses%2C%20perils%2C%20and%20pitfalls&amp;journal=J.%20Clin.%20Microbiol.&amp;doi=10.1128%2FJCM.01228-07&amp;volume=45&amp;pages=2761-2764&amp;publication_year=2007&amp;author=Janda%2CJM&amp;author=Abbott%2CSL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Brooks, B. et al. Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome. <i>Nat. Commun.</i> <b>8</b>, 1814 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-017-02018-w" data-track-item_id="10.1038/s41467-017-02018-w" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-017-02018-w" aria-label="Article reference 17" data-doi="10.1038/s41467-017-02018-w">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29180750" aria-label="PubMed reference 17">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5703836" aria-label="PubMed Central reference 17">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXpsVWitr0%3D" aria-label="CAS reference 17">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Strain-resolved%20analysis%20of%20hospital%20rooms%20and%20infants%20reveals%20overlap%20between%20the%20human%20and%20room%20microbiome&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-02018-w&amp;volume=8&amp;publication_year=2017&amp;author=Brooks%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">McIntyre, A. B. R. et al. Comprehensive benchmarking and ensemble approaches for metagenomic classifiers. <i>Genome Biol.</i> <b>18</b>, 182 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-017-1299-7" data-track-item_id="10.1186/s13059-017-1299-7" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-017-1299-7" aria-label="Article reference 18" data-doi="10.1186/s13059-017-1299-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28934964" aria-label="PubMed reference 18">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5609029" aria-label="PubMed Central reference 18">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVOrur%2FF" aria-label="CAS reference 18">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20benchmarking%20and%20ensemble%20approaches%20for%20metagenomic%20classifiers&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-017-1299-7&amp;volume=18&amp;publication_year=2017&amp;author=McIntyre%2CABR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Bertrand, D. et al. Hybrid metagenomic assembly enables high-resolution analysis of resistance determinants and mobile elements in human microbiomes. <i>Nat. Biotechnol.</i> <b>37</b>, 937–944 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41587-019-0191-2" data-track-item_id="10.1038/s41587-019-0191-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-019-0191-2" aria-label="Article reference 19" data-doi="10.1038/s41587-019-0191-2">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVKnsb%2FE" aria-label="CAS reference 19">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31359005" aria-label="PubMed reference 19">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Hybrid%20metagenomic%20assembly%20enables%20high-resolution%20analysis%20of%20resistance%20determinants%20and%20mobile%20elements%20in%20human%20microbiomes&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-019-0191-2&amp;volume=37&amp;pages=937-944&amp;publication_year=2019&amp;author=Bertrand%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Che, Y. et al. Mobile antibiotic resistome in wastewater treatment plants revealed by nanopore metagenomic sequencing. <i>Microbiome</i> <b>7</b>, 44 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s40168-019-0663-0" data-track-item_id="10.1186/s40168-019-0663-0" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s40168-019-0663-0" aria-label="Article reference 20" data-doi="10.1186/s40168-019-0663-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30898140" aria-label="PubMed reference 20">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6429696" aria-label="PubMed Central reference 20">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Mobile%20antibiotic%20resistome%20in%20wastewater%20treatment%20plants%20revealed%20by%20nanopore%20metagenomic%20sequencing&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-019-0663-0&amp;volume=7&amp;publication_year=2019&amp;author=Che%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Kanamori, H., Weber, D. J. &amp; Rutala, W. A. Healthcare outbreaks associated with a water reservoir and infection prevention strategies. <i>Clin. Infect. Dis.</i> <b>62</b>, 1423–1435 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/cid/ciw122" data-track-item_id="10.1093/cid/ciw122" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fcid%2Fciw122" aria-label="Article reference 21" data-doi="10.1093/cid/ciw122">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26936670" aria-label="PubMed reference 21">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Healthcare%20outbreaks%20associated%20with%20a%20water%20reservoir%20and%20infection%20prevention%20strategies&amp;journal=Clin.%20Infect.%20Dis.&amp;doi=10.1093%2Fcid%2Fciw122&amp;volume=62&amp;pages=1423-1435&amp;publication_year=2016&amp;author=Kanamori%2CH&amp;author=Weber%2CDJ&amp;author=Rutala%2CWA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Dancer, S. J. Importance of the environment in methicillin-resistant <i>Staphylococcus aureus</i> acquisition: the case for hospital cleaning. <i>Lancet Infect. Dis.</i> <b>8</b>, 101–113 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S1473-3099(07)70241-4" data-track-item_id="10.1016/S1473-3099(07)70241-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS1473-3099%2807%2970241-4" aria-label="Article reference 22" data-doi="10.1016/S1473-3099(07)70241-4">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17974481" aria-label="PubMed reference 22">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Importance%20of%20the%20environment%20in%20methicillin-resistant%20Staphylococcus%20aureus%20acquisition%3A%20the%20case%20for%20hospital%20cleaning&amp;journal=Lancet%20Infect.%20Dis.&amp;doi=10.1016%2FS1473-3099%2807%2970241-4&amp;volume=8&amp;pages=101-113&amp;publication_year=2008&amp;author=Dancer%2CSJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">de Goffau, M. C. et al. Recognizing the reagent microbiome. <i>Nat. Microbiol.</i> <b>3</b>, 851–853 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41564-018-0202-y" data-track-item_id="10.1038/s41564-018-0202-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-018-0202-y" aria-label="Article reference 23" data-doi="10.1038/s41564-018-0202-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30046175" aria-label="PubMed reference 23">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtl2jt7%2FJ" aria-label="CAS reference 23">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Recognizing%20the%20reagent%20microbiome&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-018-0202-y&amp;volume=3&amp;pages=851-853&amp;publication_year=2018&amp;author=Goffau%2CMC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Sehulster, L. &amp; Chinn, R. Y. Guidelines for environmental infection control in health-care facilities. Recommendations of CDC and the Healthcare Infection Control Practices Advisory Committee (HICPAC). <i>MMWR Recomm. Rep.</i> <b>52</b>, 1–42 (2003).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12836624" aria-label="PubMed reference 24">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Guidelines%20for%20environmental%20infection%20control%20in%20health-care%20facilities.%20Recommendations%20of%20CDC%20and%20the%20Healthcare%20Infection%20Control%20Practices%20Advisory%20Committee%20%28HICPAC%29&amp;journal=MMWR%20Recomm.%20Rep.&amp;volume=52&amp;pages=1-42&amp;publication_year=2003&amp;author=Sehulster%2CL&amp;author=Chinn%2CRY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Kizny Gordon, A. E. et al. The hospital water environment as a reservoir for carbapenem-resistant organisms causing hospital-acquired infections—a systematic review of the literature. <i>Clin. Infect. Dis.</i> <b>64</b>, 1435–1444 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/cid/cix132" data-track-item_id="10.1093/cid/cix132" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fcid%2Fcix132" aria-label="Article reference 25" data-doi="10.1093/cid/cix132">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28200000" aria-label="PubMed reference 25">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20hospital%20water%20environment%20as%20a%20reservoir%20for%20carbapenem-resistant%20organisms%20causing%20hospital-acquired%20infections%E2%80%94a%20systematic%20review%20of%20the%20literature&amp;journal=Clin.%20Infect.%20Dis.&amp;doi=10.1093%2Fcid%2Fcix132&amp;volume=64&amp;pages=1435-1444&amp;publication_year=2017&amp;author=Kizny%20Gordon%2CAE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Danko, D.C. et al. Global genetic cartography of urban metagenomes and anti-microbial resistance. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/724526" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/724526">https://doi.org/10.1101/724526</a> (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Beggs, C., Knibbs, L. D., Johnson, G. R. &amp; Morawska, L. Environmental contamination and hospital-acquired infection: factors that are easily overlooked. <i>Indoor Air</i> <b>25</b>, 462–474 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/ina.12170" data-track-item_id="10.1111/ina.12170" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fina.12170" aria-label="Article reference 27" data-doi="10.1111/ina.12170">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFWkur3L" aria-label="CAS reference 27">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25346039" aria-label="PubMed reference 27">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Environmental%20contamination%20and%20hospital-acquired%20infection%3A%20factors%20that%20are%20easily%20overlooked&amp;journal=Indoor%20Air&amp;doi=10.1111%2Fina.12170&amp;volume=25&amp;pages=462-474&amp;publication_year=2015&amp;author=Beggs%2CC&amp;author=Knibbs%2CLD&amp;author=Johnson%2CGR&amp;author=Morawska%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Ministry of Health Singapore. <i>Guidelines for Control and Prevention of Multi-drug Resistant Organisms (MDROs) in Healthcare Facilities</i> (2013).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Lee, J. Y. H. et al. Global spread of three multidrug-resistant lineages of <i>Staphylococcus epidermidis</i>. <i>Nat. Microbiol.</i> <b>3</b>, 1175–1185 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41564-018-0230-7" data-track-item_id="10.1038/s41564-018-0230-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-018-0230-7" aria-label="Article reference 29" data-doi="10.1038/s41564-018-0230-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1CgsLbO" aria-label="CAS reference 29">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30177740" aria-label="PubMed reference 29">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6660648" aria-label="PubMed Central reference 29">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20spread%20of%20three%20multidrug-resistant%20lineages%20of%20Staphylococcus%20epidermidis&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-018-0230-7&amp;volume=3&amp;pages=1175-1185&amp;publication_year=2018&amp;author=Lee%2CJYH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Wright, G. D. Environmental and clinical antibiotic resistomes, same only different. <i>Curr. Opin. Microbiol.</i> <b>51</b>, 57–63 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.mib.2019.06.005" data-track-item_id="10.1016/j.mib.2019.06.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.mib.2019.06.005" aria-label="Article reference 30" data-doi="10.1016/j.mib.2019.06.005">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlKjsr3E" aria-label="CAS reference 30">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31330416" aria-label="PubMed reference 30">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Environmental%20and%20clinical%20antibiotic%20resistomes%2C%20same%20only%20different&amp;journal=Curr.%20Opin.%20Microbiol.&amp;doi=10.1016%2Fj.mib.2019.06.005&amp;volume=51&amp;pages=57-63&amp;publication_year=2019&amp;author=Wright%2CGD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Pasolli, E. et al. Extensive unexplored human microbiome diversity revealed by over 150,000 genomes from metagenomes spanning age, geography, and lifestyle. <i>Cell</i> <b>176</b>, 649–662 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2019.01.001" data-track-item_id="10.1016/j.cell.2019.01.001" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.01.001" aria-label="Article reference 31" data-doi="10.1016/j.cell.2019.01.001">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVars7w%3D" aria-label="CAS reference 31">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30661755" aria-label="PubMed reference 31">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6349461" aria-label="PubMed Central reference 31">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Extensive%20unexplored%20human%20microbiome%20diversity%20revealed%20by%20over%20150%2C000%20genomes%20from%20metagenomes%20spanning%20age%2C%20geography%2C%20and%20lifestyle&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.01.001&amp;volume=176&amp;pages=649-662&amp;publication_year=2019&amp;author=Pasolli%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Almeida, A. et al. A new genomic blueprint of the human gut microbiota. <i>Nature</i> <b>568</b>, 499–504 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-019-0965-1" data-track-item_id="10.1038/s41586-019-0965-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-019-0965-1" aria-label="Article reference 32" data-doi="10.1038/s41586-019-0965-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXmslKhu7s%3D" aria-label="CAS reference 32">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30745586" aria-label="PubMed reference 32">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6784870" aria-label="PubMed Central reference 32">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20new%20genomic%20blueprint%20of%20the%20human%20gut%20microbiota&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-0965-1&amp;volume=568&amp;pages=499-504&amp;publication_year=2019&amp;author=Almeida%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Parks, D. H. et al. A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life. <i>Nat. Biotechnol.</i> <b>36</b>, 996–1004 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nbt.4229" data-track-item_id="10.1038/nbt.4229" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.4229" aria-label="Article reference 33" data-doi="10.1038/nbt.4229">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2js7jN" aria-label="CAS reference 33">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30148503" aria-label="PubMed reference 33">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20standardized%20bacterial%20taxonomy%20based%20on%20genome%20phylogeny%20substantially%20revises%20the%20tree%20of%20life&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.4229&amp;volume=36&amp;pages=996-1004&amp;publication_year=2018&amp;author=Parks%2CDH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">San Millan, A. Evolution of plasmid-mediated antibiotic resistance in the clinical context. <i>Trends Microbiol.</i> <b>26</b>, 978–985 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.tim.2018.06.007" data-track-item_id="10.1016/j.tim.2018.06.007" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.tim.2018.06.007" aria-label="Article reference 34" data-doi="10.1016/j.tim.2018.06.007">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlSksb7F" aria-label="CAS reference 34">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30049587" aria-label="PubMed reference 34">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Evolution%20of%20plasmid-mediated%20antibiotic%20resistance%20in%20the%20clinical%20context&amp;journal=Trends%20Microbiol.&amp;doi=10.1016%2Fj.tim.2018.06.007&amp;volume=26&amp;pages=978-985&amp;publication_year=2018&amp;author=San%20Millan%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Calero-Caceres, W., Ye, M. &amp; Balcazar, J. L. Bacteriophages as environmental reservoirs of antibiotic resistance. <i>Trends Microbiol.</i> <b>27</b>, 570–577 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.tim.2019.02.008" data-track-item_id="10.1016/j.tim.2019.02.008" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.tim.2019.02.008" aria-label="Article reference 35" data-doi="10.1016/j.tim.2019.02.008">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXks1elt74%3D" aria-label="CAS reference 35">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30905524" aria-label="PubMed reference 35">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Bacteriophages%20as%20environmental%20reservoirs%20of%20antibiotic%20resistance&amp;journal=Trends%20Microbiol.&amp;doi=10.1016%2Fj.tim.2019.02.008&amp;volume=27&amp;pages=570-577&amp;publication_year=2019&amp;author=Calero-Caceres%2CW&amp;author=Ye%2CM&amp;author=Balcazar%2CJL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Galata, V., Fehlmann, T., Backes, C. &amp; Keller, A. PLSDB: a resource of complete bacterial plasmids. <i>Nucleic Acids Res.</i> <b>47</b>, D195–D202 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gky1050" data-track-item_id="10.1093/nar/gky1050" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgky1050" aria-label="Article reference 36" data-doi="10.1093/nar/gky1050">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1GqsbrF" aria-label="CAS reference 36">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30380090" aria-label="PubMed reference 36">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=PLSDB%3A%20a%20resource%20of%20complete%20bacterial%20plasmids&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky1050&amp;volume=47&amp;pages=D195-D202&amp;publication_year=2019&amp;author=Galata%2CV&amp;author=Fehlmann%2CT&amp;author=Backes%2CC&amp;author=Keller%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Gao, N. L. et al. MVP: a microbe–phage interaction database. <i>Nucleic Acids Res</i> <b>46</b>, D700–D707 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkx1124" data-track-item_id="10.1093/nar/gkx1124" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkx1124" aria-label="Article reference 37" data-doi="10.1093/nar/gkx1124">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlGisLvL" aria-label="CAS reference 37">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29177508" aria-label="PubMed reference 37">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=MVP%3A%20a%20microbe%E2%80%93phage%20interaction%20database&amp;journal=Nucleic%20Acids%20Res&amp;doi=10.1093%2Fnar%2Fgkx1124&amp;volume=46&amp;pages=D700-D707&amp;publication_year=2018&amp;author=Gao%2CNL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Duran, N., Temiz, M., Duran, G. G., Eryilmaz, N. &amp; Jenedi, K. Relationship between the resistance genes to quaternary ammonium compounds and antibiotic resistance in staphylococci isolated from surgical site infections. <i>Med. Sci. Monit.</i> <b>20</b>, 544–550 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.12659/MSM.890177" data-track-item_id="10.12659/MSM.890177" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.12659%2FMSM.890177" aria-label="Article reference 38" data-doi="10.12659/MSM.890177">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24691183" aria-label="PubMed reference 38">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3983099" aria-label="PubMed Central reference 38">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhs12rtrnO" aria-label="CAS reference 38">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Relationship%20between%20the%20resistance%20genes%20to%20quaternary%20ammonium%20compounds%20and%20antibiotic%20resistance%20in%20staphylococci%20isolated%20from%20surgical%20site%20infections&amp;journal=Med.%20Sci.%20Monit.&amp;doi=10.12659%2FMSM.890177&amp;volume=20&amp;pages=544-550&amp;publication_year=2014&amp;author=Duran%2CN&amp;author=Temiz%2CM&amp;author=Duran%2CGG&amp;author=Eryilmaz%2CN&amp;author=Jenedi%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Johnson, R. C. et al. Investigation of a cluster of <i>Sphingomonas koreensis</i> infections. <i>N. Engl. J. Med.</i> <b>379</b>, 2529–2539 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa1803238" data-track-item_id="10.1056/NEJMoa1803238" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa1803238" aria-label="Article reference 39" data-doi="10.1056/NEJMoa1803238">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXltF2qtw%3D%3D" aria-label="CAS reference 39">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30586509" aria-label="PubMed reference 39">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6322212" aria-label="PubMed Central reference 39">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=Investigation%20of%20a%20cluster%20of%20Sphingomonas%20koreensis%20infections&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1803238&amp;volume=379&amp;pages=2529-2539&amp;publication_year=2018&amp;author=Johnson%2CRC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Fahimipour, A.K. et al. Antimicrobial chemicals associate with microbial function and antibiotic resistance indoors. <i>mSystems</i> <b>3</b>, e00200-18 (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Hartmann, E. M. et al. Antimicrobial chemicals are associated with elevated antibiotic resistance genes in the indoor dust microbiome. <i>Environ. Sci. Technol.</i> <b>50</b>, 9807–9815 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1021/acs.est.6b00262" data-track-item_id="10.1021/acs.est.6b00262" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1021%2Facs.est.6b00262" aria-label="Article reference 41" data-doi="10.1021/acs.est.6b00262">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVGnsLnN" aria-label="CAS reference 41">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27599587" aria-label="PubMed reference 41">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5032049" aria-label="PubMed Central reference 41">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Antimicrobial%20chemicals%20are%20associated%20with%20elevated%20antibiotic%20resistance%20genes%20in%20the%20indoor%20dust%20microbiome&amp;journal=Environ.%20Sci.%20Technol.&amp;doi=10.1021%2Facs.est.6b00262&amp;volume=50&amp;pages=9807-9815&amp;publication_year=2016&amp;author=Hartmann%2CEM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Falagas, M. E., Roussos, N., Gkegkes, I. D., Rafailidis, P. I. &amp; Karageorgopoulos, D. E. Fosfomycin for the treatment of infections caused by Gram-positive cocci with advanced antimicrobial drug resistance: a review of microbiological, animal and clinical studies. <i>Expert Opin. Investig. Drugs</i> <b>18</b>, 921–944 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1517/13543780902967624" data-track-item_id="10.1517/13543780902967624" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1517%2F13543780902967624" aria-label="Article reference 42" data-doi="10.1517/13543780902967624">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXns1erurY%3D" aria-label="CAS reference 42">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19548851" aria-label="PubMed reference 42">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Fosfomycin%20for%20the%20treatment%20of%20infections%20caused%20by%20Gram-positive%20cocci%20with%20advanced%20antimicrobial%20drug%20resistance%3A%20a%20review%20of%20microbiological%2C%20animal%20and%20clinical%20studies&amp;journal=Expert%20Opin.%20Investig.%20Drugs&amp;doi=10.1517%2F13543780902967624&amp;volume=18&amp;pages=921-944&amp;publication_year=2009&amp;author=Falagas%2CME&amp;author=Roussos%2CN&amp;author=Gkegkes%2CID&amp;author=Rafailidis%2CPI&amp;author=Karageorgopoulos%2CDE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Urban, C., Mariano, N. &amp; Rahal, J. J. In vitro double and triple bactericidal activities of doripenem, polymyxin B, and rifampin against multidrug-resistant <i>Acinetobacter baumannii</i>, <i>Pseudomonas aeruginosa</i>, <i>Klebsiella pneumoniae</i>, and <i>Escherichia coli</i>. <i>Antimicrob. Agents Chemother.</i> <b>54</b>, 2732–2734 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/AAC.01768-09" data-track-item_id="10.1128/AAC.01768-09" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FAAC.01768-09" aria-label="Article reference 43" data-doi="10.1128/AAC.01768-09">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXos1Ogsbk%3D" aria-label="CAS reference 43">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20368401" aria-label="PubMed reference 43">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2876416" aria-label="PubMed Central reference 43">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vitro%20double%20and%20triple%20bactericidal%20activities%20of%20doripenem%2C%20polymyxin%20B%2C%20and%20rifampin%20against%20multidrug-resistant%20Acinetobacter%20baumannii%2C%20Pseudomonas%20aeruginosa%2C%20Klebsiella%20pneumoniae%2C%20and%20Escherichia%20coli&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;doi=10.1128%2FAAC.01768-09&amp;volume=54&amp;pages=2732-2734&amp;publication_year=2010&amp;author=Urban%2CC&amp;author=Mariano%2CN&amp;author=Rahal%2CJJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Izadpanah, M. &amp; Khalili, H. Antibiotic regimens for treatment of infections due to multidrug-resistant Gram-negative pathogens: an evidence-based literature review. <i>J. Res. Pharm. Pract.</i> <b>4</b>, 105–114 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.4103/2279-042X.162360" data-track-item_id="10.4103/2279-042X.162360" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.4103%2F2279-042X.162360" aria-label="Article reference 44" data-doi="10.4103/2279-042X.162360">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVaitLfI" aria-label="CAS reference 44">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26312249" aria-label="PubMed reference 44">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4548428" aria-label="PubMed Central reference 44">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Antibiotic%20regimens%20for%20treatment%20of%20infections%20due%20to%20multidrug-resistant%20Gram-negative%20pathogens%3A%20an%20evidence-based%20literature%20review&amp;journal=J.%20Res.%20Pharm.%20Pract.&amp;doi=10.4103%2F2279-042X.162360&amp;volume=4&amp;pages=105-114&amp;publication_year=2015&amp;author=Izadpanah%2CM&amp;author=Khalili%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Werner, G., Hildebrandt, B. &amp; Witte, W. Aminoglycoside-streptothricin resistance gene cluster <i>aadE</i>–<i>sat4</i>–<i>aphA</i>-<i>3</i> disseminated among multiresistant isolates of <i>Enterococcus faecium</i>. <i>Antimicrob. Agents Chemother.</i> <b>45</b>, 3267–3269 (2001).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/AAC.45.11.3267-3269.2001" data-track-item_id="10.1128/AAC.45.11.3267-3269.2001" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FAAC.45.11.3267-3269.2001" aria-label="Article reference 45" data-doi="10.1128/AAC.45.11.3267-3269.2001">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD3MXnvF2jtb4%3D" aria-label="CAS reference 45">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11600397" aria-label="PubMed reference 45">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC90823" aria-label="PubMed Central reference 45">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Aminoglycoside-streptothricin%20resistance%20gene%20cluster%20aadE%E2%80%93sat4%E2%80%93aphA-3%20disseminated%20among%20multiresistant%20isolates%20of%20Enterococcus%20faecium&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;doi=10.1128%2FAAC.45.11.3267-3269.2001&amp;volume=45&amp;pages=3267-3269&amp;publication_year=2001&amp;author=Werner%2CG&amp;author=Hildebrandt%2CB&amp;author=Witte%2CW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Weinstein, L., Chang, T. W., Hudson, J. B. &amp; Hartl, W. The concurrent use of sulfonamides and antibiotics in the treatment of infections: in vivo and in vitro studies of the effect of sulfonamide-antibiotic combinations on the emergence of drug resistance. <i>Ann. NY Acad. Sci.</i> <b>69</b>, 408–416 (1957).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/j.1749-6632.1957.tb49681.x" data-track-item_id="10.1111/j.1749-6632.1957.tb49681.x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1749-6632.1957.tb49681.x" aria-label="Article reference 46" data-doi="10.1111/j.1749-6632.1957.tb49681.x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaG1c%2FjtV2rsg%3D%3D" aria-label="CAS reference 46">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=13488271" aria-label="PubMed reference 46">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20concurrent%20use%20of%20sulfonamides%20and%20antibiotics%20in%20the%20treatment%20of%20infections%3A%20in%20vivo%20and%20in%20vitro%20studies%20of%20the%20effect%20of%20sulfonamide-antibiotic%20combinations%20on%20the%20emergence%20of%20drug%20resistance&amp;journal=Ann.%20NY%20Acad.%20Sci.&amp;doi=10.1111%2Fj.1749-6632.1957.tb49681.x&amp;volume=69&amp;pages=408-416&amp;publication_year=1957&amp;author=Weinstein%2CL&amp;author=Chang%2CTW&amp;author=Hudson%2CJB&amp;author=Hartl%2CW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Sundin, G. W. &amp; Bender, C. L. Dissemination of the <i>strA</i>–<i>strB</i> streptomycin-resistance genes among commensal and pathogenic bacteria from humans, animals, and plants. <i>Mol. Ecol.</i> <b>5</b>, 133–143 (1996).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/j.1365-294X.1996.tb00299.x" data-track-item_id="10.1111/j.1365-294X.1996.tb00299.x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1365-294X.1996.tb00299.x" aria-label="Article reference 47" data-doi="10.1111/j.1365-294X.1996.tb00299.x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DyaK28Xit12hu7w%3D" aria-label="CAS reference 47">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9147689" aria-label="PubMed reference 47">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=Dissemination%20of%20the%20strA%E2%80%93strB%20streptomycin-resistance%20genes%20among%20commensal%20and%20pathogenic%20bacteria%20from%20humans%2C%20animals%2C%20and%20plants&amp;journal=Mol.%20Ecol.&amp;doi=10.1111%2Fj.1365-294X.1996.tb00299.x&amp;volume=5&amp;pages=133-143&amp;publication_year=1996&amp;author=Sundin%2CGW&amp;author=Bender%2CCL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Monecke, S. et al. Microarray-based characterisation of a Panton–Valentine leukocidin-positive community-acquired strain of methicillin-resistant <i>Staphylococcus aureus</i>. <i>Clin. Microbiol. Infect.</i> <b>12</b>, 718–728 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/j.1469-0691.2006.01420.x" data-track-item_id="10.1111/j.1469-0691.2006.01420.x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2Fj.1469-0691.2006.01420.x" aria-label="Article reference 48" data-doi="10.1111/j.1469-0691.2006.01420.x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XosFamu70%3D" aria-label="CAS reference 48">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16842566" aria-label="PubMed reference 48">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Microarray-based%20characterisation%20of%20a%20Panton%E2%80%93Valentine%20leukocidin-positive%20community-acquired%20strain%20of%20methicillin-resistant%20Staphylococcus%20aureus&amp;journal=Clin.%20Microbiol.%20Infect.&amp;doi=10.1111%2Fj.1469-0691.2006.01420.x&amp;volume=12&amp;pages=718-728&amp;publication_year=2006&amp;author=Monecke%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Teo, J. et al. First case of <i>E. anophelis</i> outbreak in an intensive-care unit. <i>Lancet</i> <b>382</b>, 855–856 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(13)61858-9" data-track-item_id="10.1016/S0140-6736(13)61858-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2813%2961858-9" aria-label="Article reference 49" data-doi="10.1016/S0140-6736(13)61858-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24012265" aria-label="PubMed reference 49">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=First%20case%20of%20E.%20anophelis%20outbreak%20in%20an%20intensive-care%20unit&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2813%2961858-9&amp;volume=382&amp;pages=855-856&amp;publication_year=2013&amp;author=Teo%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Htun, H. L. et al. Methicillin-resistant <i>Staphylococcus aureus</i> colonisation: epidemiological and molecular characteristics in an acute-care tertiary hospital in Singapore. <i>Epidemiol. Infect.</i> <b>146</b>, 1785–1792 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1017/S0950268818001966" data-track-item_id="10.1017/S0950268818001966" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1017%2FS0950268818001966" aria-label="Article reference 50" data-doi="10.1017/S0950268818001966">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVyhsbjP" aria-label="CAS reference 50">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30019657" aria-label="PubMed reference 50">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Methicillin-resistant%20Staphylococcus%20aureus%20colonisation%3A%20epidemiological%20and%20molecular%20characteristics%20in%20an%20acute-care%20tertiary%20hospital%20in%20Singapore&amp;journal=Epidemiol.%20Infect.&amp;doi=10.1017%2FS0950268818001966&amp;volume=146&amp;pages=1785-1792&amp;publication_year=2018&amp;author=Htun%2CHL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Nasser, K. et al. Draft genome sequences of six multidrug-resistant clinical strains of <i>Acinetobacter baumannii</i>, isolated at two major hospitals in Kuwait. <i>Genome Announc.</i> <b>6</b>, e00264-18 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/genomeA.00264-18" data-track-item_id="10.1128/genomeA.00264-18" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FgenomeA.00264-18" aria-label="Article reference 51" data-doi="10.1128/genomeA.00264-18">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29674542" aria-label="PubMed reference 51">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5908928" aria-label="PubMed Central reference 51">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Draft%20genome%20sequences%20of%20six%20multidrug-resistant%20clinical%20strains%20of%20Acinetobacter%20baumannii%2C%20isolated%20at%20two%20major%20hospitals%20in%20Kuwait&amp;journal=Genome%20Announc.&amp;doi=10.1128%2FgenomeA.00264-18&amp;volume=6&amp;publication_year=2018&amp;author=Nasser%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Walker, J. T. et al. Hospital and community acquired infection and the built environment—design and testing of infection control rooms. <i>J. Hosp. Infect.</i> <b>65</b>, 43–49 (2007).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0195-6701(07)60014-0" data-track-item_id="10.1016/S0195-6701(07)60014-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0195-6701%2807%2960014-0" aria-label="Article reference 52" data-doi="10.1016/S0195-6701(07)60014-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17540241" aria-label="PubMed reference 52">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7134456" aria-label="PubMed Central reference 52">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 52" href="http://scholar.google.com/scholar_lookup?&amp;title=Hospital%20and%20community%20acquired%20infection%20and%20the%20built%20environment%E2%80%94design%20and%20testing%20of%20infection%20control%20rooms&amp;journal=J.%20Hosp.%20Infect.&amp;doi=10.1016%2FS0195-6701%2807%2960014-0&amp;volume=65&amp;pages=43-49&amp;publication_year=2007&amp;author=Walker%2CJT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Ladner, J. T., Grubaugh, N. D., Pybus, O. G. &amp; Andersen, K. G. Precision epidemiology for infectious disease control. <i>Nat. Med.</i> <b>25</b>, 206–211 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41591-019-0345-2" data-track-item_id="10.1038/s41591-019-0345-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41591-019-0345-2" aria-label="Article reference 53" data-doi="10.1038/s41591-019-0345-2">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXntlKrtbk%3D" aria-label="CAS reference 53">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30728537" aria-label="PubMed reference 53">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095960" aria-label="PubMed Central reference 53">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Precision%20epidemiology%20for%20infectious%20disease%20control&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-019-0345-2&amp;volume=25&amp;pages=206-211&amp;publication_year=2019&amp;author=Ladner%2CJT&amp;author=Grubaugh%2CND&amp;author=Pybus%2COG&amp;author=Andersen%2CKG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Mahnert, A. et al. Man-made microbial resistances in built environments. <i>Nat. Commun.</i> <b>10</b>, 968 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-019-08864-0" data-track-item_id="10.1038/s41467-019-08864-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-08864-0" aria-label="Article reference 54" data-doi="10.1038/s41467-019-08864-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30814504" aria-label="PubMed reference 54">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6393488" aria-label="PubMed Central reference 54">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXnslCns70%3D" aria-label="CAS reference 54">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Man-made%20microbial%20resistances%20in%20built%20environments&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-08864-0&amp;volume=10&amp;publication_year=2019&amp;author=Mahnert%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Goodman, E. R. et al. Impact of an environmental cleaning intervention on the presence of methicillin-resistant <i>Staphylococcus aureus</i> and vancomycin-resistant enterococci on surfaces in intensive care unit rooms. <i>Infect. Control Hosp. Epidemiol.</i> <b>29</b>, 593–599 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1086/588566" data-track-item_id="10.1086/588566" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1086%2F588566" aria-label="Article reference 55" data-doi="10.1086/588566">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18624666" aria-label="PubMed reference 55">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2670228" aria-label="PubMed Central reference 55">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Impact%20of%20an%20environmental%20cleaning%20intervention%20on%20the%20presence%20of%20methicillin-resistant%20Staphylococcus%20aureus%20and%20vancomycin-resistant%20enterococci%20on%20surfaces%20in%20intensive%20care%20unit%20rooms&amp;journal=Infect.%20Control%20Hosp.%20Epidemiol.&amp;doi=10.1086%2F588566&amp;volume=29&amp;pages=593-599&amp;publication_year=2008&amp;author=Goodman%2CER">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Lerminiaux, N. A. &amp; Cameron, A. D. S. Horizontal transfer of antibiotic resistance genes in clinical environments. <i>Can. J. Microbiol.</i> <b>65</b>, 34–44 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1139/cjm-2018-0275" data-track-item_id="10.1139/cjm-2018-0275" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1139%2Fcjm-2018-0275" aria-label="Article reference 56" data-doi="10.1139/cjm-2018-0275">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1CrtL%2FM" aria-label="CAS reference 56">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30248271" aria-label="PubMed reference 56">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Horizontal%20transfer%20of%20antibiotic%20resistance%20genes%20in%20clinical%20environments&amp;journal=Can.%20J.%20Microbiol.&amp;doi=10.1139%2Fcjm-2018-0275&amp;volume=65&amp;pages=34-44&amp;publication_year=2019&amp;author=Lerminiaux%2CNA&amp;author=Cameron%2CADS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Buckner, M. M. C., Ciusa, M. L. &amp; Piddock, L. J. V. Strategies to combat antimicrobial resistance: anti-plasmid and plasmid curing. <i>FEMS Microbiol. Rev.</i> <b>42</b>, 781–804 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/femsre/fuy031" data-track-item_id="10.1093/femsre/fuy031" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Ffemsre%2Ffuy031" aria-label="Article reference 57" data-doi="10.1093/femsre/fuy031">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1WitLfM" aria-label="CAS reference 57">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30085063" aria-label="PubMed reference 57">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6199537" aria-label="PubMed Central reference 57">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Strategies%20to%20combat%20antimicrobial%20resistance%3A%20anti-plasmid%20and%20plasmid%20curing&amp;journal=FEMS%20Microbiol.%20Rev.&amp;doi=10.1093%2Ffemsre%2Ffuy031&amp;volume=42&amp;pages=781-804&amp;publication_year=2018&amp;author=Buckner%2CMMC&amp;author=Ciusa%2CML&amp;author=Piddock%2CLJV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">David, S. et al. Epidemic of carbapenem-resistant <i>Klebsiella pneumoniae</i> in Europe is driven by nosocomial spread. <i>Nat. Microbiol.</i> <b>4</b>, 1919–1929 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41564-019-0492-8" data-track-item_id="10.1038/s41564-019-0492-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-019-0492-8" aria-label="Article reference 58" data-doi="10.1038/s41564-019-0492-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVKnsbrF" aria-label="CAS reference 58">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31358985" aria-label="PubMed reference 58">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7244338" aria-label="PubMed Central reference 58">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Epidemic%20of%20carbapenem-resistant%20Klebsiella%20pneumoniae%20in%20Europe%20is%20driven%20by%20nosocomial%20spread&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-019-0492-8&amp;volume=4&amp;pages=1919-1929&amp;publication_year=2019&amp;author=David%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Ho, C. M. et al. High rate of <i>qacA</i>- and <i>qacB</i>-positive methicillin-resistant <i>Staphylococcus aureus</i> isolates from chlorhexidine-impregnated catheter-related bloodstream infections. <i>Antimicrob. Agents Chemother.</i> <b>56</b>, 5693–5697 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/AAC.00761-12" data-track-item_id="10.1128/AAC.00761-12" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FAAC.00761-12" aria-label="Article reference 59" data-doi="10.1128/AAC.00761-12">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhsF2iurrJ" aria-label="CAS reference 59">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22908163" aria-label="PubMed reference 59">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486537" aria-label="PubMed Central reference 59">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=High%20rate%20of%20qacA-%20and%20qacB-positive%20methicillin-resistant%20Staphylococcus%20aureus%20isolates%20from%20chlorhexidine-impregnated%20catheter-related%20bloodstream%20infections&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;doi=10.1128%2FAAC.00761-12&amp;volume=56&amp;pages=5693-5697&amp;publication_year=2012&amp;author=Ho%2CCM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Weiner, L. M. et al. Antimicrobial-resistant pathogens associated with healthcare-associated infections: summary of data reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2011–2014. <i>Infect. Control Hosp. Epidemiol.</i> <b>37</b>, 1288–1301 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1017/ice.2016.174" data-track-item_id="10.1017/ice.2016.174" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1017%2Fice.2016.174" aria-label="Article reference 60" data-doi="10.1017/ice.2016.174">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27573805" aria-label="PubMed reference 60">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6857725" aria-label="PubMed Central reference 60">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Antimicrobial-resistant%20pathogens%20associated%20with%20healthcare-associated%20infections%3A%20summary%20of%20data%20reported%20to%20the%20National%20Healthcare%20Safety%20Network%20at%20the%20Centers%20for%20Disease%20Control%20and%20Prevention%2C%202011%E2%80%932014&amp;journal=Infect.%20Control%20Hosp.%20Epidemiol.&amp;doi=10.1017%2Fice.2016.174&amp;volume=37&amp;pages=1288-1301&amp;publication_year=2016&amp;author=Weiner%2CLM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Truong, D. T. et al. MetaPhlAn2 for enhanced metagenomic taxonomic profiling. <i>Nat. Methods</i> <b>12</b>, 902–903 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmeth.3589" data-track-item_id="10.1038/nmeth.3589" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.3589" aria-label="Article reference 61" data-doi="10.1038/nmeth.3589">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFyqsL7I" aria-label="CAS reference 61">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26418763" aria-label="PubMed reference 61">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=MetaPhlAn2%20for%20enhanced%20metagenomic%20taxonomic%20profiling&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.3589&amp;volume=12&amp;pages=902-903&amp;publication_year=2015&amp;author=Truong%2CDT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Inouye, M. et al. SRST2: rapid genomic surveillance for public health and hospital microbiology labs. <i>Genome Med.</i> <b>6</b>, 90 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13073-014-0090-6" data-track-item_id="10.1186/s13073-014-0090-6" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13073-014-0090-6" aria-label="Article reference 62" data-doi="10.1186/s13073-014-0090-6">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25422674" aria-label="PubMed reference 62">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4237778" aria-label="PubMed Central reference 62">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=SRST2%3A%20rapid%20genomic%20surveillance%20for%20public%20health%20and%20hospital%20microbiology%20labs&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-014-0090-6&amp;volume=6&amp;publication_year=2014&amp;author=Inouye%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Wood, D. E. &amp; Salzberg, S. L. Kraken: ultrafast metagenomic sequence classification using exact alignments. <i>Genome Biol.</i> <b>15</b>, r46 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/gb-2014-15-3-r46" data-track-item_id="10.1186/gb-2014-15-3-r46" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/gb-2014-15-3-r46" aria-label="Article reference 63" data-doi="10.1186/gb-2014-15-3-r46">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24580807" aria-label="PubMed reference 63">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053813" aria-label="PubMed Central reference 63">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Kraken%3A%20ultrafast%20metagenomic%20sequence%20classification%20using%20exact%20alignments&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fgb-2014-15-3-r46&amp;volume=15&amp;publication_year=2014&amp;author=Wood%2CDE&amp;author=Salzberg%2CSL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Koren, S. et al. Canu: scalable and accurate long-read assembly via adaptive <i>k</i>-mer weighting and repeat separation. <i>Genome Res.</i> <b>27</b>, 722–736 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/gr.215087.116" data-track-item_id="10.1101/gr.215087.116" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.215087.116" aria-label="Article reference 64" data-doi="10.1101/gr.215087.116">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFyjsrvI" aria-label="CAS reference 64">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28298431" aria-label="PubMed reference 64">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5411767" aria-label="PubMed Central reference 64">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Canu%3A%20scalable%20and%20accurate%20long-read%20assembly%20via%20adaptive%20k-mer%20weighting%20and%20repeat%20separation&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.215087.116&amp;volume=27&amp;pages=722-736&amp;publication_year=2017&amp;author=Koren%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Kurtz, S. et al. Versatile and open software for comparing large genomes. <i>Genome Biol.</i> <b>5</b>, r12 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/gb-2004-5-2-r12" data-track-item_id="10.1186/gb-2004-5-2-r12" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/gb-2004-5-2-r12" aria-label="Article reference 65" data-doi="10.1186/gb-2004-5-2-r12">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14759262" aria-label="PubMed reference 65">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC395750" aria-label="PubMed Central reference 65">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=Versatile%20and%20open%20software%20for%20comparing%20large%20genomes&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fgb-2004-5-2-r12&amp;volume=5&amp;publication_year=2004&amp;author=Kurtz%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Walker, B. J. et al. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. <i>PLoS ONE</i> <b>9</b>, e112963 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0112963" data-track-item_id="10.1371/journal.pone.0112963" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0112963" aria-label="Article reference 66" data-doi="10.1371/journal.pone.0112963">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25409509" aria-label="PubMed reference 66">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4237348" aria-label="PubMed Central reference 66">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXisFCmurk%3D" aria-label="CAS reference 66">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0112963&amp;volume=9&amp;publication_year=2014&amp;author=Walker%2CBJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. &amp; Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. <i>Genome Res.</i> <b>25</b>, 1043–1055 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/gr.186072.114" data-track-item_id="10.1101/gr.186072.114" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.186072.114" aria-label="Article reference 67" data-doi="10.1101/gr.186072.114">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1SltbfE" aria-label="CAS reference 67">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25977477" aria-label="PubMed reference 67">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4484387" aria-label="PubMed Central reference 67">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=CheckM%3A%20assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates%2C%20single%20cells%2C%20and%20metagenomes&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.186072.114&amp;volume=25&amp;pages=1043-1055&amp;publication_year=2015&amp;author=Parks%2CDH&amp;author=Imelfort%2CM&amp;author=Skennerton%2CCT&amp;author=Hugenholtz%2CP&amp;author=Tyson%2CGW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. <i>Genome Biol.</i> <b>17</b>, 132 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-016-0997-x" data-track-item_id="10.1186/s13059-016-0997-x" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-016-0997-x" aria-label="Article reference 68" data-doi="10.1186/s13059-016-0997-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27323842" aria-label="PubMed reference 68">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4915045" aria-label="PubMed Central reference 68">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXns1entr0%3D" aria-label="CAS reference 68">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Mash%3A%20fast%20genome%20and%20metagenome%20distance%20estimation%20using%20MinHash&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-016-0997-x&amp;volume=17&amp;publication_year=2016&amp;author=Ondov%2CBD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Pierce, N. T., Irber, L., Reiter, T., Brooks, P. &amp; Brown, C. T. Large-scale sequence comparisons with sourmash. <i>F1000Res</i> <b>8</b>, 1006 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.12688/f1000research.19675.1" data-track-item_id="10.12688/f1000research.19675.1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.12688%2Ff1000research.19675.1" aria-label="Article reference 69" data-doi="10.12688/f1000research.19675.1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31508216" aria-label="PubMed reference 69">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6720031" aria-label="PubMed Central reference 69">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20sequence%20comparisons%20with%20sourmash&amp;journal=F1000Res&amp;doi=10.12688%2Ff1000research.19675.1&amp;volume=8&amp;publication_year=2019&amp;author=Pierce%2CNT&amp;author=Irber%2CL&amp;author=Reiter%2CT&amp;author=Brooks%2CP&amp;author=Brown%2CCT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Gupta, S. K. et al. ARG-ANNOT, a new bioinformatic tool to discover antibiotic resistance genes in bacterial genomes. <i>Antimicrob. Agents Chemother.</i> <b>58</b>, 212–220 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/AAC.01310-13" data-track-item_id="10.1128/AAC.01310-13" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FAAC.01310-13" aria-label="Article reference 70" data-doi="10.1128/AAC.01310-13">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24145532" aria-label="PubMed reference 70">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3910750" aria-label="PubMed Central reference 70">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFCiu7Y%3D" aria-label="CAS reference 70">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=ARG-ANNOT%2C%20a%20new%20bioinformatic%20tool%20to%20discover%20antibiotic%20resistance%20genes%20in%20bacterial%20genomes&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;doi=10.1128%2FAAC.01310-13&amp;volume=58&amp;pages=212-220&amp;publication_year=2014&amp;author=Gupta%2CSK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Wattam, A. R. et al. PATRIC, the bacterial bioinformatics database and analysis resource. <i>Nucleic Acids Res.</i> <b>42</b>, D581–D591 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkt1099" data-track-item_id="10.1093/nar/gkt1099" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkt1099" aria-label="Article reference 71" data-doi="10.1093/nar/gkt1099">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXos1Wh" aria-label="CAS reference 71">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24225323" aria-label="PubMed reference 71">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=PATRIC%2C%20the%20bacterial%20bioinformatics%20database%20and%20analysis%20resource&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkt1099&amp;volume=42&amp;pages=D581-D591&amp;publication_year=2014&amp;author=Wattam%2CAR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Roux, S., Enault, F., Hurwitz, B. L. &amp; Sullivan, M. B. VirSorter: mining viral signal from microbial genomic data. <i>PeerJ</i> <b>3</b>, e985 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7717/peerj.985" data-track-item_id="10.7717/peerj.985" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7717%2Fpeerj.985" aria-label="Article reference 72" data-doi="10.7717/peerj.985">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26038737" aria-label="PubMed reference 72">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4451026" aria-label="PubMed Central reference 72">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XlsVCntLk%3D" aria-label="CAS reference 72">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=VirSorter%3A%20mining%20viral%20signal%20from%20microbial%20genomic%20data&amp;journal=PeerJ&amp;doi=10.7717%2Fpeerj.985&amp;volume=3&amp;publication_year=2015&amp;author=Roux%2CS&amp;author=Enault%2CF&amp;author=Hurwitz%2CBL&amp;author=Sullivan%2CMB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Aziz, R. K. et al. The RAST Server: rapid annotations using subsystems technology. <i>BMC Genomics</i> <b>9</b>, 75 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/1471-2164-9-75" data-track-item_id="10.1186/1471-2164-9-75" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/1471-2164-9-75" aria-label="Article reference 73" data-doi="10.1186/1471-2164-9-75">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18261238" aria-label="PubMed reference 73">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2265698" aria-label="PubMed Central reference 73">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXlslSnsLo%3D" aria-label="CAS reference 73">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20RAST%20Server%3A%20rapid%20annotations%20using%20subsystems%20technology&amp;journal=BMC%20Genomics&amp;doi=10.1186%2F1471-2164-9-75&amp;volume=9&amp;publication_year=2008&amp;author=Aziz%2CRK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Gao, S., Bertrand, D., Chia, B. K. &amp; Nagarajan, N. OPERA-LG: efficient and exact scaffolding of large, repeat-rich eukaryotic genomes with performance guarantees. <i>Genome Biol.</i> <b>17</b>, 102 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-016-0951-y" data-track-item_id="10.1186/s13059-016-0951-y" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-016-0951-y" aria-label="Article reference 74" data-doi="10.1186/s13059-016-0951-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27169502" aria-label="PubMed reference 74">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4864936" aria-label="PubMed Central reference 74">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVektb%2FN" aria-label="CAS reference 74">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=OPERA-LG%3A%20efficient%20and%20exact%20scaffolding%20of%20large%2C%20repeat-rich%20eukaryotic%20genomes%20with%20performance%20guarantees&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-016-0951-y&amp;volume=17&amp;publication_year=2016&amp;author=Gao%2CS&amp;author=Bertrand%2CD&amp;author=Chia%2CBK&amp;author=Nagarajan%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Gao, S., Bertrand, D., &amp; Nagarajan, N. FinIS: improved in silico finishing using an exact quadratic programming formulation. In <i>Algorithms in Bioinformatics</i> (Eds. Raphael, B. &amp; Tang, J.) 314–325 (Springer, 2012).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Treangen, T. J., Ondov, B. D., Koren, S. &amp; Phillippy, A. M. The Harvest suite for rapid core-genome alignment and visualization of thousands of intraspecific microbial genomes. <i>Genome Biol.</i> <b>15</b>, 524 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-014-0524-x" data-track-item_id="10.1186/s13059-014-0524-x" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-014-0524-x" aria-label="Article reference 76" data-doi="10.1186/s13059-014-0524-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25410596" aria-label="PubMed reference 76">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4262987" aria-label="PubMed Central reference 76">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXivVKnsr0%3D" aria-label="CAS reference 76">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Harvest%20suite%20for%20rapid%20core-genome%20alignment%20and%20visualization%20of%20thousands%20of%20intraspecific%20microbial%20genomes&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-014-0524-x&amp;volume=15&amp;publication_year=2014&amp;author=Treangen%2CTJ&amp;author=Ondov%2CBD&amp;author=Koren%2CS&amp;author=Phillippy%2CAM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Bruen, T. C., Philippe, H. &amp; Bryant, D. A simple and robust statistical test for detecting the presence of recombination. <i>Genetics</i> <b>172</b>, 2665–2681 (2006).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1534/genetics.105.048975" data-track-item_id="10.1534/genetics.105.048975" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1534%2Fgenetics.105.048975" aria-label="Article reference 77" data-doi="10.1534/genetics.105.048975">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD28XkvFOgtLc%3D" aria-label="CAS reference 77">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16489234" aria-label="PubMed reference 77">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1456386" aria-label="PubMed Central reference 77">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20simple%20and%20robust%20statistical%20test%20for%20detecting%20the%20presence%20of%20recombination&amp;journal=Genetics&amp;doi=10.1534%2Fgenetics.105.048975&amp;volume=172&amp;pages=2665-2681&amp;publication_year=2006&amp;author=Bruen%2CTC&amp;author=Philippe%2CH&amp;author=Bryant%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Price, M. N., Dehal, P. S. &amp; Arkin, A. P. FastTree 2—approximately maximum-likelihood trees for large alignments. <i>PLoS ONE</i> <b>5</b>, e9490 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0009490" data-track-item_id="10.1371/journal.pone.0009490" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0009490" aria-label="Article reference 78" data-doi="10.1371/journal.pone.0009490">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20224823" aria-label="PubMed reference 78">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2835736" aria-label="PubMed Central reference 78">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXjsFCmsLo%3D" aria-label="CAS reference 78">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=FastTree%202%E2%80%94approximately%20maximum-likelihood%20trees%20for%20large%20alignments&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0009490&amp;volume=5&amp;publication_year=2010&amp;author=Price%2CMN&amp;author=Dehal%2CPS&amp;author=Arkin%2CAP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Yu, G., Lam, T. T., Zhu, H. &amp; Guan, Y. Two methods for mapping and visualizing associated data on phylogeny using ggtree. <i>Mol. Biol. Evol.</i> <b>35</b>, 3041–3043 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/molbev/msy194" data-track-item_id="10.1093/molbev/msy194" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fmolbev%2Fmsy194" aria-label="Article reference 79" data-doi="10.1093/molbev/msy194">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFKntrbF" aria-label="CAS reference 79">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30351396" aria-label="PubMed reference 79">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6278858" aria-label="PubMed Central reference 79">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 79" href="http://scholar.google.com/scholar_lookup?&amp;title=Two%20methods%20for%20mapping%20and%20visualizing%20associated%20data%20on%20phylogeny%20using%20ggtree&amp;journal=Mol.%20Biol.%20Evol.&amp;doi=10.1093%2Fmolbev%2Fmsy194&amp;volume=35&amp;pages=3041-3043&amp;publication_year=2018&amp;author=Yu%2CG&amp;author=Lam%2CTT&amp;author=Zhu%2CH&amp;author=Guan%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Hsieh, T. C., Ma, K. H. &amp; Chao, A. iNEXT: an R package for rarefaction and extrapolation of species diversity (Hill numbers). <i>Methods Ecol. Evol.</i> <b>7</b>, 1451–1456 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1111/2041-210X.12613" data-track-item_id="10.1111/2041-210X.12613" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1111%2F2041-210X.12613" aria-label="Article reference 80" data-doi="10.1111/2041-210X.12613">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=iNEXT%3A%20an%20R%20package%20for%20rarefaction%20and%20extrapolation%20of%20species%20diversity%20%28Hill%20numbers%29&amp;journal=Methods%20Ecol.%20Evol.&amp;doi=10.1111%2F2041-210X.12613&amp;volume=7&amp;pages=1451-1456&amp;publication_year=2016&amp;author=Hsieh%2CTC&amp;author=Ma%2CKH&amp;author=Chao%2CA">
                    Google Scholar</a>&nbsp;
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-020-0894-4?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>Funding for this work was provided by A*STAR (N.N.), and we are grateful for support from NMRC (NMRC CGAug16C005: O.T.N. and K.M.). C.E.M. acknowledges support from the WorldQuant Foundation, the Bill and Melinda Gates Foundation (OPP1151054) and the Alfred P. Sloan Foundation (G-2015-13964). The funders had no role in the study design, data collection and analysis, decision to publish or preparation of the manuscript. We would like to thank J. Gilbert for insightful comments and feedback on this work.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="na1"><p>These authors contributed equally: Kern Rei Chng, Chenhao Li, Denis Bertrand.</p></li><li class="c-article-author-information__item" id="na2"><p>A full list of authors and their affiliations appears at the end of the paper.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Computational and Systems Biology, Genome Institute of Singapore, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Kern Rei Chng,&nbsp;Chenhao Li,&nbsp;Denis Bertrand,&nbsp;Amanda Hui Qi Ng,&nbsp;Junmei Samantha Kwah,&nbsp;Hwee Meng Low,&nbsp;Chengxuan Tong,&nbsp;Maanasa Natrajan,&nbsp;Michael Hongjie Zhang,&nbsp;Eliza Xin Pei Ho,&nbsp;Tamar V. Av-Shalom,&nbsp;Chiea Chuen Khor,&nbsp;Swaine L. Chen&nbsp;&amp;&nbsp;Niranjan Nagarajan</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Information Systems Technology and Design, Singapore University of Technology and Design, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Licheng Xu</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Department of Microbiology, Singapore General Hospital, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Karrie Kwan Ki Ko</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Department of Molecular Pathology, Singapore General Hospital, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Karrie Kwan Ki Ko</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Duke-NUS Graduate Medical School, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Karrie Kwan Ki Ko</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Department of Laboratory Medicine, National University Hospital, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Jeanette Woon Pei Teo</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Department of Physiology and Biophysics, Weill Cornell Medicine, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Christopher E. Mason</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">National Centre for Infectious Diseases, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Oon Tek Ng,&nbsp;Kalisvar Marimuthu&nbsp;&amp;&nbsp;Brenda Ang</p></li><li id="Aff9"><p class="c-article-author-affiliation__address">Department of Infectious Diseases, Tan Tock Seng Hospital, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Oon Tek Ng,&nbsp;Kalisvar Marimuthu&nbsp;&amp;&nbsp;Brenda Ang</p></li><li id="Aff10"><p class="c-article-author-affiliation__address">Lee Kong Chian School of Medicine, Nanyang Technological University, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Oon Tek Ng</p></li><li id="Aff11"><p class="c-article-author-affiliation__address">Yong Loo Lin School of Medicine, National University of Singapore, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Kalisvar Marimuthu&nbsp;&amp;&nbsp;Niranjan Nagarajan</p></li><li id="Aff12"><p class="c-article-author-affiliation__address">Weill Cornell Medicine, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">David Danko,&nbsp;Daniela Bezdan,&nbsp;Ebrahim Afshinnekoo,&nbsp;Chandrima Bhattacharya,&nbsp;Daniel J. Butler,&nbsp;Dmitry Meleshko,&nbsp;Jorge L. Sanchez,&nbsp;Heba Shaaban,&nbsp;Maria A. Sierra,&nbsp;Ben Young,&nbsp;Josue Alicea,&nbsp;Ana M. Cañas,&nbsp;Daisy Donnellan,&nbsp;Iman Hajirasouliha,&nbsp;Kaymisha Knights,&nbsp;Abigail Lyons,&nbsp;Dominique Thomas,&nbsp;Brandon Valentine,&nbsp;Jifeng Zhu&nbsp;&amp;&nbsp;Christopher E. Mason</p></li><li id="Aff13"><p class="c-article-author-affiliation__address">The Bin Talal Bin Abdulaziz Alsaud Institute for Computational Biomedicine, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">David Danko,&nbsp;Daniela Bezdan,&nbsp;Ebrahim Afshinnekoo,&nbsp;Chandrima Bhattacharya,&nbsp;Daniel J. Butler,&nbsp;Dmitry Meleshko,&nbsp;Heba Shaaban,&nbsp;Maria A. Sierra,&nbsp;Ben Young,&nbsp;Josue Alicea,&nbsp;Iman Hajirasouliha&nbsp;&amp;&nbsp;Christopher E. Mason</p></li><li id="Aff14"><p class="c-article-author-affiliation__address">Icahn School of Medicine at Mount Sinai, New York, NY, USA</p><p class="c-article-author-affiliation__authors-list">Sofia Ahsanuddin</p></li><li id="Aff15"><p class="c-article-author-affiliation__address">Genome Institute of Singapore, Singapore, Singapore</p><p class="c-article-author-affiliation__authors-list">Kern Rei Chng,&nbsp;Amanda Ng,&nbsp;Eileen Png&nbsp;&amp;&nbsp;Niranjan Nagarajan</p></li><li id="Aff16"><p class="c-article-author-affiliation__address">University of Naples Federico II, Naples, Italy</p><p class="c-article-author-affiliation__authors-list">Francesca De Filippis&nbsp;&amp;&nbsp;Danilo Ercolini</p></li><li id="Aff17"><p class="c-article-author-affiliation__address">The Barcelona Institute of Science and Technology, Barcelona, Spain</p><p class="c-article-author-affiliation__authors-list">Jochen Hecht&nbsp;&amp;&nbsp;Beth Mutai</p></li><li id="Aff18"><p class="c-article-author-affiliation__address">ETH Zurich, Zurich, Switzerland</p><p class="c-article-author-affiliation__authors-list">Andre Kahles,&nbsp;Mikhail Karasikov,&nbsp;Harun Mustafa,&nbsp;Olga Nikolayeva,&nbsp;Tatyana Nikolayeva&nbsp;&amp;&nbsp;Gunnar Rätsch</p></li><li id="Aff19"><p class="c-article-author-affiliation__address">Joint Genome Institute, Walnut Creek, CA, USA</p><p class="c-article-author-affiliation__authors-list">Nikos C. Kyrpides,&nbsp;Russell Y. Neches&nbsp;&amp;&nbsp;David Paez-Espino</p></li><li id="Aff20"><p class="c-article-author-affiliation__address">City University of Hong Kong, Hong Kong SAR, China</p><p class="c-article-author-affiliation__authors-list">Marcus H. Y. Leung,&nbsp;Xinzhao Tong&nbsp;&amp;&nbsp;Patrick K. H. Lee</p></li><li id="Aff21"><p class="c-article-author-affiliation__address">Kenya Medical Research Institute/Medical Research Directorate-Africa, Kisumu, Kenya</p><p class="c-article-author-affiliation__authors-list">Beth Mutai</p></li><li id="Aff22"><p class="c-article-author-affiliation__address">University of Colorado Boulder, Boulder, CO, USA</p><p class="c-article-author-affiliation__authors-list">Marina Nieto-Caballero&nbsp;&amp;&nbsp;Mark Hernandez</p></li><li id="Aff23"><p class="c-article-author-affiliation__address">Indian Statistical Institute, Kolkata, India</p><p class="c-article-author-affiliation__authors-list">Malay Bhattacharyya</p></li><li id="Aff24"><p class="c-article-author-affiliation__address">University of Minnesota, Minneapolis, MN, USA</p><p class="c-article-author-affiliation__authors-list">Ran Blekhman</p></li><li id="Aff25"><p class="c-article-author-affiliation__address">Universidad Andrés Bello, Santiago, Chile</p><p class="c-article-author-affiliation__authors-list">Eduardo Castro-Nallar</p></li><li id="Aff26"><p class="c-article-author-affiliation__address">Weill Cornell Medicine—Qatar, Doha, Qatar</p><p class="c-article-author-affiliation__authors-list">Aspassia D. Chatziefthimiou</p></li><li id="Aff27"><p class="c-article-author-affiliation__address">California State University Sacramento, Sacramento, CA, USA</p><p class="c-article-author-affiliation__authors-list">Robert W. Crawford</p></li><li id="Aff28"><p class="c-article-author-affiliation__address">University of Hawaii, Honolulu, HI, USA</p><p class="c-article-author-affiliation__authors-list">Youping Deng</p></li><li id="Aff29"><p class="c-article-author-affiliation__address">Aix-Marseille Université, Marseille, France</p><p class="c-article-author-affiliation__authors-list">Christelle Desnues&nbsp;&amp;&nbsp;Nicolas Rascovan</p></li><li id="Aff30"><p class="c-article-author-affiliation__address">A.C.Camargo Cancer Center, São Paulo, Brazil</p><p class="c-article-author-affiliation__authors-list">Emmanuel Dias-Neto</p></li><li id="Aff31"><p class="c-article-author-affiliation__address">Norwegian Defence Research Establishment, Kjeller, Norway</p><p class="c-article-author-affiliation__authors-list">Marius Dybwad</p></li><li id="Aff32"><p class="c-article-author-affiliation__address">University of Sheffield, Sheffield, UK</p><p class="c-article-author-affiliation__authors-list">Eran Elhaik</p></li><li id="Aff33"><p class="c-article-author-affiliation__address">Institute of Molecular Biology and Genetics of National Academy of Sciences of Ukraine, Kyiv, Ukraine</p><p class="c-article-author-affiliation__authors-list">Alina Frolova</p></li><li id="Aff34"><p class="c-article-author-affiliation__address">University of Applied Sciences FH-Campus Wien, Vienna, Austria</p><p class="c-article-author-affiliation__authors-list">Alexandra B. Graf</p></li><li id="Aff35"><p class="c-article-author-affiliation__address">King’s College London, London, UK</p><p class="c-article-author-affiliation__authors-list">David C. Green,&nbsp;Frank J. Kelly,&nbsp;Gabriella Mason-Buck&nbsp;&amp;&nbsp;Denise Syndercombe Court</p></li><li id="Aff36"><p class="c-article-author-affiliation__address">Institut Pasteur de Montevideo, Montevideo, Uruguay</p><p class="c-article-author-affiliation__authors-list">Gregorio Iraola</p></li><li id="Aff37"><p class="c-article-author-affiliation__address">Institut Pasteur Korea, Seongnam, South Korea</p><p class="c-article-author-affiliation__authors-list">Soojin Jang</p></li><li id="Aff38"><p class="c-article-author-affiliation__address">Corporación Corpogen, Bogotá, Colombia</p><p class="c-article-author-affiliation__authors-list">Angela Jones&nbsp;&amp;&nbsp;María M. Zambrano</p></li><li id="Aff39"><p class="c-article-author-affiliation__address">Jagiellonian University, Kraków, Poland</p><p class="c-article-author-affiliation__authors-list">Paweł P. Łabaj</p></li><li id="Aff40"><p class="c-article-author-affiliation__address">HudsonAlpha Institute for Biotechnology, Huntsville, AL, USA</p><p class="c-article-author-affiliation__authors-list">Levy Shawn</p></li><li id="Aff41"><p class="c-article-author-affiliation__address">Stockholm University, Stockholm, Sweden</p><p class="c-article-author-affiliation__authors-list">Per Ljungdahl&nbsp;&amp;&nbsp;Klas I. Udekwu</p></li><li id="Aff42"><p class="c-article-author-affiliation__address">Microba, Brisbane, Australia</p><p class="c-article-author-affiliation__authors-list">Ken McGrath</p></li><li id="Aff43"><p class="c-article-author-affiliation__address">University of Maryland School of Medicine, Baltimore, MD, USA</p><p class="c-article-author-affiliation__authors-list">Emmanuel F. Mongodin&nbsp;&amp;&nbsp;Lynn M. Schriml</p></li><li id="Aff44"><p class="c-article-author-affiliation__address">Fundação Oswaldo Cruz Laboratório de Hanseníase, Rio de Janeiro, Brazil</p><p class="c-article-author-affiliation__authors-list">Milton Ozorio Moraes</p></li><li id="Aff45"><p class="c-article-author-affiliation__address">Ribeirão Preto Medical School University of São Paulo, São Paulo, Brazil</p><p class="c-article-author-affiliation__authors-list">Houtan Noushmehr</p></li><li id="Aff46"><p class="c-article-author-affiliation__address">Instituto de Patologia e Imunologia Molecular da Universidade do Porto, Porto, Portugal</p><p class="c-article-author-affiliation__authors-list">Manuela Oliveira</p></li><li id="Aff47"><p class="c-article-author-affiliation__address">University of Tübingen, Tübingen, Germany</p><p class="c-article-author-affiliation__authors-list">Stephan Ossowski</p></li><li id="Aff48"><p class="c-article-author-affiliation__address">Elizade University, Ondo State, Nigeria</p><p class="c-article-author-affiliation__authors-list">Olayinka O. Osuolale</p></li><li id="Aff49"><p class="c-article-author-affiliation__address">Acibadem Mehmet Ali Aydinlar University, Istanbul, Turkey</p><p class="c-article-author-affiliation__authors-list">Orhan Özcan&nbsp;&amp;&nbsp;Osman U. Sezerman</p></li><li id="Aff50"><p class="c-article-author-affiliation__address">Sorbonne University, Paris, France</p><p class="c-article-author-affiliation__authors-list">Hugues Richard</p></li><li id="Aff51"><p class="c-article-author-affiliation__address">Robert Koch Institute, Berlin, Germany</p><p class="c-article-author-affiliation__authors-list">Torsten Semmler</p></li><li id="Aff52"><p class="c-article-author-affiliation__address">Fudan University, Shanghai, China</p><p class="c-article-author-affiliation__authors-list">Leming Shi&nbsp;&amp;&nbsp;Sibo Zhu</p></li><li id="Aff53"><p class="c-article-author-affiliation__address">Vietnamese-German Center of Excellence, Hanoi, Vietnam</p><p class="c-article-author-affiliation__authors-list">Le Huu Song</p></li><li id="Aff54"><p class="c-article-author-affiliation__address">Keio University, Fujisawa, Japan</p><p class="c-article-author-affiliation__authors-list">Haruo Suzuki</p></li><li id="Aff55"><p class="c-article-author-affiliation__address">University of Vermont, Burlington, VT, USA</p><p class="c-article-author-affiliation__authors-list">Scott W. Tighe</p></li><li id="Aff56"><p class="c-article-author-affiliation__address">Millennium Initiative for Collaborative Research on Bacterial Resistance, Santiago, Chile</p><p class="c-article-author-affiliation__authors-list">Juan A. Ugalde</p></li><li id="Aff57"><p class="c-article-author-affiliation__address">Sofia University, Sofia, Bulgaria</p><p class="c-article-author-affiliation__authors-list">Dimitar I. Vassilev</p></li><li id="Aff58"><p class="c-article-author-affiliation__address">University of Alaska Fairbanks, Fairbanks, AK, USA</p><p class="c-article-author-affiliation__authors-list">Elena Vayndorf</p></li><li id="Aff59"><p class="c-article-author-affiliation__address">Univeristätsklinikum Tübingen, Tübingen, Germany</p><p class="c-article-author-affiliation__authors-list">Thirumalaisamy P. Velavan</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Kern_Rei-Chng-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Kern Rei Chng</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Kern%20Rei%20Chng" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kern%20Rei%20Chng" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kern%20Rei%20Chng%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Chenhao-Li-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Chenhao Li</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Chenhao%20Li" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chenhao%20Li" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chenhao%20Li%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Denis-Bertrand-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Denis Bertrand</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Denis%20Bertrand" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Denis%20Bertrand" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Denis%20Bertrand%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Amanda_Hui_Qi-Ng-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Amanda Hui Qi Ng</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Amanda%20Hui%20Qi%20Ng" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Amanda%20Hui%20Qi%20Ng" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Amanda%20Hui%20Qi%20Ng%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Junmei_Samantha-Kwah-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Junmei Samantha Kwah</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Junmei%20Samantha%20Kwah" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Junmei%20Samantha%20Kwah" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Junmei%20Samantha%20Kwah%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Hwee_Meng-Low-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Hwee Meng Low</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Hwee%20Meng%20Low" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Hwee%20Meng%20Low" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Hwee%20Meng%20Low%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Chengxuan-Tong-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Chengxuan Tong</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Chengxuan%20Tong" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chengxuan%20Tong" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chengxuan%20Tong%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Maanasa-Natrajan-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Maanasa Natrajan</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Maanasa%20Natrajan" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Maanasa%20Natrajan" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Maanasa%20Natrajan%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Michael_Hongjie-Zhang-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Michael Hongjie Zhang</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michael%20Hongjie%20Zhang" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michael%20Hongjie%20Zhang" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20Hongjie%20Zhang%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Licheng-Xu-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Licheng Xu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Licheng%20Xu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Licheng%20Xu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Licheng%20Xu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Karrie_Kwan_Ki-Ko-Aff3-Aff4-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Karrie Kwan Ki Ko</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Karrie%20Kwan%20Ki%20Ko" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Karrie%20Kwan%20Ki%20Ko" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Karrie%20Kwan%20Ki%20Ko%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Eliza_Xin_Pei-Ho-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Eliza Xin Pei Ho</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Eliza%20Xin%20Pei%20Ho" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Eliza%20Xin%20Pei%20Ho" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Eliza%20Xin%20Pei%20Ho%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Tamar_V_-Av_Shalom-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Tamar V. Av-Shalom</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Tamar%20V.%20Av-Shalom" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tamar%20V.%20Av-Shalom" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tamar%20V.%20Av-Shalom%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jeanette_Woon_Pei-Teo-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Jeanette Woon Pei Teo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jeanette%20Woon%20Pei%20Teo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jeanette%20Woon%20Pei%20Teo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jeanette%20Woon%20Pei%20Teo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Chiea_Chuen-Khor-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Chiea Chuen Khor</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Chiea%20Chuen%20Khor" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Chiea%20Chuen%20Khor" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Chiea%20Chuen%20Khor%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Swaine_L_-Chen-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Swaine L. Chen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Swaine%20L.%20Chen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Swaine%20L.%20Chen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Swaine%20L.%20Chen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Christopher_E_-Mason-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Christopher E. Mason</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Christopher%20E.%20Mason" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christopher%20E.%20Mason" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christopher%20E.%20Mason%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Oon_Tek-Ng-Aff8-Aff9-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Oon Tek Ng</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Oon%20Tek%20Ng" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Oon%20Tek%20Ng" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Oon%20Tek%20Ng%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Kalisvar-Marimuthu-Aff8-Aff9-Aff11"><span class="c-article-authors-search__title u-h3 js-search-name">Kalisvar Marimuthu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Kalisvar%20Marimuthu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kalisvar%20Marimuthu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kalisvar%20Marimuthu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Brenda-Ang-Aff8-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Brenda Ang</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Brenda%20Ang" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Brenda%20Ang" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Brenda%20Ang%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Niranjan-Nagarajan-Aff1-Aff11"><span class="c-article-authors-search__title u-h3 js-search-name">Niranjan Nagarajan</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Niranjan%20Nagarajan" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Niranjan%20Nagarajan" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Niranjan%20Nagarajan%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="groups">Consortia</h3><div class="c-article-author-institutional-author" id="group-1"><h3 class="c-article-author-institutional-author__name u-h3">MetaSUB Consortium</h3><ul class="c-article-author-institutional-author__author-list"><li class="c-article-author-institutional-author__author-name">David Danko</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Daniela Bezdan</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ebrahim Afshinnekoo</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Sofia Ahsanuddin</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Chandrima Bhattacharya</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Daniel J. Butler</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Kern Rei Chng</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Francesca De Filippis</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jochen Hecht</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Andre Kahles</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Mikhail Karasikov</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Nikos C. Kyrpides</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Marcus H. Y. Leung</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Dmitry Meleshko</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Harun Mustafa</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Beth Mutai</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Russell Y. Neches</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Amanda Ng</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Marina Nieto-Caballero</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Olga Nikolayeva</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Tatyana Nikolayeva</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Eileen Png</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jorge L. Sanchez</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Heba Shaaban</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Maria A. Sierra</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Xinzhao Tong</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ben Young</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Josue Alicea</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Malay Bhattacharyya</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ran Blekhman</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Eduardo Castro-Nallar</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ana M. Cañas</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Aspassia D. Chatziefthimiou</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Robert W. Crawford</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Youping Deng</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Christelle Desnues</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Emmanuel Dias-Neto</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Daisy Donnellan</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Marius Dybwad</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Eran Elhaik</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Danilo Ercolini</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Alina Frolova</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Alexandra B. Graf</li><li class="c-article-author-institutional-author__author-name">,&nbsp;David C. Green</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Iman Hajirasouliha</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Mark Hernandez</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Gregorio Iraola</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Soojin Jang</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Angela Jones</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Frank J. Kelly</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Kaymisha Knights</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Paweł P. Łabaj</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Patrick K. H. Lee</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Levy Shawn</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Per Ljungdahl</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Abigail Lyons</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Gabriella Mason-Buck</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Ken McGrath</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Emmanuel F. Mongodin</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Milton Ozorio Moraes</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Niranjan Nagarajan</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Houtan Noushmehr</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Manuela Oliveira</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Stephan Ossowski</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Olayinka O. Osuolale</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Orhan Özcan</li><li class="c-article-author-institutional-author__author-name">,&nbsp;David Paez-Espino</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Nicolas Rascovan</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Hugues Richard</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Gunnar Rätsch</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Lynn M. Schriml</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Torsten Semmler</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Osman U. Sezerman</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Leming Shi</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Le Huu Song</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Haruo Suzuki</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Denise Syndercombe Court</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Dominique Thomas</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Scott W. Tighe</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Klas I. Udekwu</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Juan A. Ugalde</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Brandon Valentine</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Dimitar I. Vassilev</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Elena Vayndorf</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Thirumalaisamy P. Velavan</li><li class="c-article-author-institutional-author__author-name">,&nbsp;María M. Zambrano</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Jifeng Zhu</li><li class="c-article-author-institutional-author__author-name">,&nbsp;Sibo Zhu</li><li class="c-article-author-institutional-author__author-name">&nbsp;&amp;&nbsp;Christopher E. Mason</li></ul></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>N.N., S.L.C., B.A., K.M. and O.T.N. planned and designed the project. A.H.Q.N., J.S.K. and E.X.P.H. conducted wet-lab experiments with K.R.C. and N.N.’s supervision. H.M.L. performed nanopore sequencing under C.C.K.’s supervision. K.R.C., C.L. and D.B. coordinated all computational analysis with help from C.T., M.N., M.H.Z. and T.V.A.-S., and N.N.’s guidance. The 3D visualization tool was developed by L.X. with C.L.’s guidance. K.K.K.K. and J.W.P.T. collected clinical isolates and antibiotic resistance information. M.C. and C.E.M. provided guidance on environmental sampling protocols. K.R.C., D.B., C.L. and N.N. wrote the manuscript with input from all authors.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:nagarajann@gis.a-star.edu.sg">Niranjan Nagarajan</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar1">Competing interests</h3>
                <p>The authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Peer review information</b> Alison Farrell was the primary editor on this article and managed its editorial process and peer review in collaboration with the rest of the editorial team.</p><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec28-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec28">Supplementary information</h2><div class="c-article-section__content" id="Sec28-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Figs. 1–13 and Supplementary Notes 1–3</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary data 1" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_MOESM3_ESM.xlsx" data-supp-info-image="">Supplementary Data 1</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Metadata for sampled sites.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary data 2" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_MOESM4_ESM.xlsx" data-supp-info-image="">Supplementary Data 2</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Taxonomic and resistome profiles.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary data 3" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_MOESM5_ESM.xlsx" data-supp-info-image="">Supplementary Data 3</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Metadata for culture plates and isolates.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary data 4" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_MOESM6_ESM.xlsx" data-supp-info-image="">Supplementary Data 4</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Novel antibiotic resistance gene combinations.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary data 5" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_MOESM7_ESM.xlsx" data-supp-info-image="">Supplementary Data 5</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Common resistance gene cassettes.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary data 6" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_MOESM8_ESM.xlsx" data-supp-info-image="">Supplementary Data 6</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Antibiograms for Singaporean patient isolates.</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Cartography%20of%20opportunistic%20pathogens%20and%20antibiotic%20resistance%20genes%20in%20a%20tertiary%20hospital%20environment&amp;author=Kern%20Rei%20Chng%20David%20Danko%20et%20al&amp;contentID=10.1038%2Fs41591-020-0894-4&amp;copyright=The%20Author%28s%29&amp;publication=1078-8956&amp;publicationDate=2020-06-08&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41591-020-0894-4" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41591-020-0894-4" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Chng, K.R., Li, C., Bertrand, D. <i>et al.</i> Cartography of opportunistic pathogens and antibiotic resistance genes in a tertiary hospital environment.
                    <i>Nat Med</i> <b>26</b>, 941–951 (2020). https://doi.org/10.1038/s41591-020-0894-4</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-020-0894-4?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2019-05-20">20 May 2019</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-04-20">20 April 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-06-08">08 June 2020</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2020-06">June 2020</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41591-020-0894-4</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-block"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/disease-prevention" data-track="click" data-track-action="view subject" data-track-label="link">Disease prevention</a></li><li class="c-article-subject-list__subject"><a href="/subjects/microbial-genetics" data-track="click" data-track-action="view subject" data-track-label="link">Microbial genetics</a></li>
        </ul>
    </div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section" data-test="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Elevated antibiotic resistance gene abundance of ICU healthcare workers, a multicentre, cross-sectional study" href="https://doi.org/10.1186/s13054-025-05408-5">
                                        Elevated antibiotic resistance gene abundance of ICU healthcare workers, a multicentre, cross-sectional study
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Lingtong Huang</li><li>Kangchen Li</li><li>Chao Jiang</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Critical Care</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:An early microbial landscape: inspiring endeavor from the China Space Station Habitation Area Microbiome Program (CHAMP)" href="https://doi.org/10.1007/s11427-024-2894-2">
                                        An early microbial landscape: inspiring endeavor from the China Space Station Habitation Area Microbiome Program (CHAMP)
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Junxia Yuan</li><li>Jinlu Yang</li><li>Wenyu Shi</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Science China Life Sciences</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Analysis of metagenomic data" href="https://doi.org/10.1038/s43586-024-00376-6">
                                        Analysis of metagenomic data
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Shaopeng Liu</li><li>Judith S. Rodriguez</li><li>Serghei Mangul</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Reviews Methods Primers</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Outbreak investigation of Serratia marcescens bloodstream infection in an obstetric ward for high-risk pregnant women" href="https://doi.org/10.1186/s12879-024-09134-1">
                                        Outbreak investigation of Serratia marcescens bloodstream infection in an obstetric ward for high-risk pregnant women
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Seulki Kim</li><li>Sunah Jung</li><li>Seungjin Lim</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>BMC Infectious Diseases</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:The indoors microbiome and human health" href="https://doi.org/10.1038/s41579-024-01077-3">
                                        The indoors microbiome and human health
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Jack A. Gilbert</li><li>Erica M. Hartmann</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Reviews Microbiology</i> (2024)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-020-0894-4.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                
            
        
    </div>

    
        
    

    
    
        <div class="c-article-associated-content__container">
            <section>
                <h2 class="c-article-associated-content__title u-mb-24">Associated content</h2>
                
                    
                    
                        <div class="u-full-height u-mb-24">
                            
    <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
            <div class="c-card__body">
                <h3 class="c-card__title">
                    <a href="https://www.nature.com/articles/s41591-020-0919-z" class="c-card__link u-link-inherit" data-track="click" data-track-action="view article" data-track-category="associated content" data-track-label="news_and_views">Uncovering hidden antimicrobial resistance patterns within the hospital microbiome</a>
                </h3>
                
<ul data-test="author-list" class="app-author-list app-author-list--compact">
    <li>Bastiaan W. Haak</li><li>W. Joost Wiersinga</li>
</ul>

                
    <div class="c-card__section c-meta">
        
            <span class="c-meta__item">Nature Medicine</span>
        
        <span class="c-meta__item" data-test="article.type"><span class="c-meta__type">News &amp; Views</span></span>
        
        
            <time class="c-meta__item" datetime="2020-06-08">08 Jun 2020</time>
        
    </div>

            </div>
        </div>
    </article>


                        </div>
                    
                        <div class="u-full-height u-mb-24">
                            
    <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
            <div class="c-card__body">
                <h3 class="c-card__title">
                    <a href="https://www.nature.com/articles/d41586-020-01717-7" class="c-card__link u-link-inherit" data-track="click" data-track-action="view article" data-track-category="associated content" data-track-label="research_highlights">Feared microbes’ hospital hangouts are revealed </a>
                </h3>
                
                
    <div class="c-card__section c-meta">
        
            <span class="c-meta__item">Nature</span>
        
        <span class="c-meta__item" data-test="article.type"><span class="c-meta__type">Research Highlight</span></span>
        
        
            <time class="c-meta__item" datetime="2020-06-09">09 Jun 2020</time>
        
    </div>

            </div>
        </div>
    </article>


                        </div>
                    
                
            </section>
        </div>
        <script>
            window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "news_and_views;research_highlights";
            
        </script>
    

    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
            <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" role="tabpanel"><div class="c-reading-companion__scroll-pane" style="max-height: none;"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Main">Main</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec7" class="c-reading-companion__section-item"><a href="#Sec7" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec8" class="c-reading-companion__section-item"><a href="#Sec8" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-data-availability" class="c-reading-companion__section-item"><a href="#data-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Data availability">Data availability</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec28" class="c-reading-companion__section-item"><a href="#Sec28" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by">This article is cited by</a></li></ul></div>
                <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="" style="display: none !important;">
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-test="right-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41591-020-0894-4;doi=10.1038/s41591-020-0894-4;subjmeta=208,325,459,631,692,700;kwrd=Disease+prevention,Microbial+genetics">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=6273678&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-020-0894-4%26doi%3D10.1038/s41591-020-0894-4%26subjmeta%3D208,325,459,631,692,700%26kwrd%3DDisease+prevention,Microbial+genetics">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=6273678&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-020-0894-4%26doi%3D10.1038/s41591-020-0894-4%26subjmeta%3D208,325,459,631,692,700%26kwrd%3DDisease+prevention,Microbial+genetics"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Fig. 1: Overview of sampling sites and analysis workflow.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig1_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig1_HTML.png" alt="figure 1" aria-describedby="rc-Fig1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0894-4/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Fig. 2: Distinct ecological niches in the hospital environment for microbes and ARGs.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig2_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig2_HTML.png" alt="figure 2" aria-describedby="rc-Fig2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0894-4/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Fig. 3: Genome-resolved characterization of nosocomial multidrug-resistant strains that spread and persist at low relative abundances in the hospital environment.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig3_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig3_HTML.png" alt="figure 3" aria-describedby="rc-Fig3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0894-4/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Fig. 4: Species distribution and genomic proximity of drug resistance genes in the hospital-environment microbiome.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig4_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig4_HTML.png" alt="figure 4" aria-describedby="rc-Fig4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0894-4/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Fig. 5: Multispecies analysis of phylogenetic relationships between environmental and patient genomes.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig5_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-020-0894-4/MediaObjects/41591_2020_894_Fig5_HTML.png" alt="figure 5" aria-describedby="rc-Fig5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-020-0894-4/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric" data-track-context="article references sidebar"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">Fischer, M.M. &amp; Bild, M. Hospital use of antibiotics as the main driver of infections with antibiotic-resistant bacteria—a reanalysis of recent data from the European Union. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/553537" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/553537">https://doi.org/10.1101/553537</a> (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">O’Neill, J. Tackling drug-resistant infections globally: final report and recommendations. In <i>Review on Antimicrobial Resistance</i> (2016).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Allegranzi, B. et al. Burden of endemic health-care-associated infection in developing countries: systematic review and meta-analysis. <i>Lancet</i> <b>377</b>, 228–241 (2011).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0140-6736%2810%2961458-4" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0140-6736(10)61458-4" data-track-item_id="10.1016/S0140-6736(10)61458-4">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21146207" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Burden%20of%20endemic%20health-care-associated%20infection%20in%20developing%20countries%3A%20systematic%20review%20and%20meta-analysis&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2810%2961458-4&amp;volume=377&amp;pages=228-241&amp;publication_year=2011&amp;author=Allegranzi%2CB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">Magill, S. S. et al. Multistate point-prevalence survey of health care-associated infections. <i>N. Engl. J. Med.</i> <b>370</b>, 1198–1208 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa1306801" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa1306801" data-track-item_id="10.1056/NEJMoa1306801">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXlvVCitLo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24670166" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4648343" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Multistate%20point-prevalence%20survey%20of%20health%20care-associated%20infections&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1306801&amp;volume=370&amp;pages=1198-1208&amp;publication_year=2014&amp;author=Magill%2CSS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">Laxminarayan, R. et al. Antibiotic resistance—the need for global solutions. <i>Lancet Infect. Dis.</i> <b>13</b>, 1057–1098 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS1473-3099%2813%2970318-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S1473-3099(13)70318-9" data-track-item_id="10.1016/S1473-3099(13)70318-9">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24252483" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Antibiotic%20resistance%E2%80%94the%20need%20for%20global%20solutions&amp;journal=Lancet%20Infect.%20Dis.&amp;doi=10.1016%2FS1473-3099%2813%2970318-9&amp;volume=13&amp;pages=1057-1098&amp;publication_year=2013&amp;author=Laxminarayan%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Longtin, Y., Sax, H., Allegranzi, B., Schneider, F. &amp; Pittet, D. Videos in clinical medicine. Hand hygiene. <i>N. Engl. J. Med.</i> <b>364</b>, e24 (2011).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMvcm0903599" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMvcm0903599" data-track-item_id="10.1056/NEJMvcm0903599">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3MXkt1Cgsro%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21449775" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Videos%20in%20clinical%20medicine.%20Hand%20hygiene&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMvcm0903599&amp;volume=364&amp;publication_year=2011&amp;author=Longtin%2CY&amp;author=Sax%2CH&amp;author=Allegranzi%2CB&amp;author=Schneider%2CF&amp;author=Pittet%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Otter, J. A., Yezli, S., Salkeld, J. A. &amp; French, G. L. Evidence that contaminated surfaces contribute to the transmission of hospital pathogens and an overview of strategies to address contaminated surfaces in hospital settings. <i>Am. J. Infect. Control</i> <b>41</b>, S6–S11 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ajic.2012.12.004" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ajic.2012.12.004" data-track-item_id="10.1016/j.ajic.2012.12.004">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23622751" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Evidence%20that%20contaminated%20surfaces%20contribute%20to%20the%20transmission%20of%20hospital%20pathogens%20and%20an%20overview%20of%20strategies%20to%20address%20contaminated%20surfaces%20in%20hospital%20settings&amp;journal=Am.%20J.%20Infect.%20Control&amp;doi=10.1016%2Fj.ajic.2012.12.004&amp;volume=41&amp;pages=S6-S11&amp;publication_year=2013&amp;author=Otter%2CJA&amp;author=Yezli%2CS&amp;author=Salkeld%2CJA&amp;author=French%2CGL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Mitchell, B. G. et al. An environmental cleaning bundle and health-care-associated infections in hospitals (REACH): a multicentre, randomised trial. <i>Lancet Infect. Dis.</i> <b>19</b>, 410–418 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS1473-3099%2818%2930714-X" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S1473-3099(18)30714-X" data-track-item_id="10.1016/S1473-3099(18)30714-X">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30858014" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20environmental%20cleaning%20bundle%20and%20health-care-associated%20infections%20in%20hospitals%20%28REACH%29%3A%20a%20multicentre%2C%20randomised%20trial&amp;journal=Lancet%20Infect.%20Dis.&amp;doi=10.1016%2FS1473-3099%2818%2930714-X&amp;volume=19&amp;pages=410-418&amp;publication_year=2019&amp;author=Mitchell%2CBG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Price, J. R. et al. Transmission of <i>Staphylococcus aureus</i> between health-care workers, the environment, and patients in an intensive care unit: a longitudinal cohort study based on whole-genome sequencing. <i>Lancet Infect. Dis.</i> <b>17</b>, 207–214 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS1473-3099%2816%2930413-3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S1473-3099(16)30413-3" data-track-item_id="10.1016/S1473-3099(16)30413-3">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27863959" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5266793" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Transmission%20of%20Staphylococcus%20aureus%20between%20health-care%20workers%2C%20the%20environment%2C%20and%20patients%20in%20an%20intensive%20care%20unit%3A%20a%20longitudinal%20cohort%20study%20based%20on%20whole-genome%20sequencing&amp;journal=Lancet%20Infect.%20Dis.&amp;doi=10.1016%2FS1473-3099%2816%2930413-3&amp;volume=17&amp;pages=207-214&amp;publication_year=2017&amp;author=Price%2CJR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Weingarten, R. A. et al. Genomic analysis of hospital plumbing reveals diverse reservoir of bacterial plasmids conferring carbapenem resistance. <i>MBio</i> <b>9</b>, e02011-17 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1128%2FmBio.02011-17" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1128/mBio.02011-17" data-track-item_id="10.1128/mBio.02011-17">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29437920" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5801463" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20analysis%20of%20hospital%20plumbing%20reveals%20diverse%20reservoir%20of%20bacterial%20plasmids%20conferring%20carbapenem%20resistance&amp;journal=MBio&amp;doi=10.1128%2FmBio.02011-17&amp;volume=9&amp;publication_year=2018&amp;author=Weingarten%2CRA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Conlan, S. et al. Single-molecule sequencing to track plasmid diversity of hospital-associated carbapenemase-producing Enterobacteriaceae. <i>Sci. Transl. Med.</i> <b>6</b>, 254ra126 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscitranslmed.3009845" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/scitranslmed.3009845" data-track-item_id="10.1126/scitranslmed.3009845">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25232178" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4203314" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitVWqsLzE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Single-molecule%20sequencing%20to%20track%20plasmid%20diversity%20of%20hospital-associated%20carbapenemase-producing%20Enterobacteriaceae&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.3009845&amp;volume=6&amp;publication_year=2014&amp;author=Conlan%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Lax, S. &amp; Gilbert, J. A. Hospital-associated microbiota and implications for nosocomial infections. <i>Trends Mol. Med.</i> <b>21</b>, 427–432 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.molmed.2015.03.005" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.molmed.2015.03.005" data-track-item_id="10.1016/j.molmed.2015.03.005">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25907678" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Hospital-associated%20microbiota%20and%20implications%20for%20nosocomial%20infections&amp;journal=Trends%20Mol.%20Med.&amp;doi=10.1016%2Fj.molmed.2015.03.005&amp;volume=21&amp;pages=427-432&amp;publication_year=2015&amp;author=Lax%2CS&amp;author=Gilbert%2CJA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Miller, R. R., Montoya, V., Gardy, J. L., Patrick, D. M. &amp; Tang, P. Metagenomics for pathogen detection in public health. <i>Genome Med.</i> <b>5</b>, 81 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/gm485" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/gm485" data-track-item_id="10.1186/gm485">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24050114" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3978900" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitVGjtro%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Metagenomics%20for%20pathogen%20detection%20in%20public%20health&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fgm485&amp;volume=5&amp;publication_year=2013&amp;author=Miller%2CRR&amp;author=Montoya%2CV&amp;author=Gardy%2CJL&amp;author=Patrick%2CDM&amp;author=Tang%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Oberauner, L. et al. The ignored diversity: complex bacterial communities in intensive care units revealed by 16S pyrosequencing. <i>Sci. Rep.</i> <b>3</b>, 1413 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fsrep01413" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/srep01413" data-track-item_id="10.1038/srep01413">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23475210" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3593336" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXpvFKjsrg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20ignored%20diversity%3A%20complex%20bacterial%20communities%20in%20intensive%20care%20units%20revealed%20by%2016S%20pyrosequencing&amp;journal=Sci.%20Rep.&amp;doi=10.1038%2Fsrep01413&amp;volume=3&amp;publication_year=2013&amp;author=Oberauner%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">Lax, S. et al. Bacterial colonization and succession in a newly opened hospital. <i>Sci. Transl. Med.</i> <b>9</b>, eaah6500 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscitranslmed.aah6500" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/scitranslmed.aah6500" data-track-item_id="10.1126/scitranslmed.aah6500">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28539477" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5706123" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Bacterial%20colonization%20and%20succession%20in%20a%20newly%20opened%20hospital&amp;journal=Sci.%20Transl.%20Med.&amp;doi=10.1126%2Fscitranslmed.aah6500&amp;volume=9&amp;publication_year=2017&amp;author=Lax%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">Janda, J. M. &amp; Abbott, S. L. 16S rRNA gene sequencing for bacterial identification in the diagnostic laboratory: pluses, perils, and pitfalls. <i>J. Clin. Microbiol.</i> <b>45</b>, 2761–2764 (2007).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1128%2FJCM.01228-07" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1128/JCM.01228-07" data-track-item_id="10.1128/JCM.01228-07">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD2sXhtFemt7bI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17626177" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2045242" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=16S%20rRNA%20gene%20sequencing%20for%20bacterial%20identification%20in%20the%20diagnostic%20laboratory%3A%20pluses%2C%20perils%2C%20and%20pitfalls&amp;journal=J.%20Clin.%20Microbiol.&amp;doi=10.1128%2FJCM.01228-07&amp;volume=45&amp;pages=2761-2764&amp;publication_year=2007&amp;author=Janda%2CJM&amp;author=Abbott%2CSL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Brooks, B. et al. Strain-resolved analysis of hospital rooms and infants reveals overlap between the human and room microbiome. <i>Nat. Commun.</i> <b>8</b>, 1814 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-017-02018-w" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-017-02018-w" data-track-item_id="10.1038/s41467-017-02018-w">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29180750" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5703836" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXpsVWitr0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Strain-resolved%20analysis%20of%20hospital%20rooms%20and%20infants%20reveals%20overlap%20between%20the%20human%20and%20room%20microbiome&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-017-02018-w&amp;volume=8&amp;publication_year=2017&amp;author=Brooks%2CB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">McIntyre, A. B. R. et al. Comprehensive benchmarking and ensemble approaches for metagenomic classifiers. <i>Genome Biol.</i> <b>18</b>, 182 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-017-1299-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-017-1299-7" data-track-item_id="10.1186/s13059-017-1299-7">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28934964" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5609029" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitVOrur%2FF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Comprehensive%20benchmarking%20and%20ensemble%20approaches%20for%20metagenomic%20classifiers&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-017-1299-7&amp;volume=18&amp;publication_year=2017&amp;author=McIntyre%2CABR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">Bertrand, D. et al. Hybrid metagenomic assembly enables high-resolution analysis of resistance determinants and mobile elements in human microbiomes. <i>Nat. Biotechnol.</i> <b>37</b>, 937–944 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41587-019-0191-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41587-019-0191-2" data-track-item_id="10.1038/s41587-019-0191-2">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVKnsb%2FE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31359005" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Hybrid%20metagenomic%20assembly%20enables%20high-resolution%20analysis%20of%20resistance%20determinants%20and%20mobile%20elements%20in%20human%20microbiomes&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-019-0191-2&amp;volume=37&amp;pages=937-944&amp;publication_year=2019&amp;author=Bertrand%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Che, Y. et al. Mobile antibiotic resistome in wastewater treatment plants revealed by nanopore metagenomic sequencing. <i>Microbiome</i> <b>7</b>, 44 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s40168-019-0663-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s40168-019-0663-0" data-track-item_id="10.1186/s40168-019-0663-0">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30898140" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6429696" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Mobile%20antibiotic%20resistome%20in%20wastewater%20treatment%20plants%20revealed%20by%20nanopore%20metagenomic%20sequencing&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-019-0663-0&amp;volume=7&amp;publication_year=2019&amp;author=Che%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Kanamori, H., Weber, D. J. &amp; Rutala, W. A. Healthcare outbreaks associated with a water reservoir and infection prevention strategies. <i>Clin. Infect. Dis.</i> <b>62</b>, 1423–1435 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fcid%2Fciw122" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/cid/ciw122" data-track-item_id="10.1093/cid/ciw122">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26936670" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Healthcare%20outbreaks%20associated%20with%20a%20water%20reservoir%20and%20infection%20prevention%20strategies&amp;journal=Clin.%20Infect.%20Dis.&amp;doi=10.1093%2Fcid%2Fciw122&amp;volume=62&amp;pages=1423-1435&amp;publication_year=2016&amp;author=Kanamori%2CH&amp;author=Weber%2CDJ&amp;author=Rutala%2CWA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Dancer, S. J. Importance of the environment in methicillin-resistant <i>Staphylococcus aureus</i> acquisition: the case for hospital cleaning. <i>Lancet Infect. Dis.</i> <b>8</b>, 101–113 (2008).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS1473-3099%2807%2970241-4" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S1473-3099(07)70241-4" data-track-item_id="10.1016/S1473-3099(07)70241-4">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17974481" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Importance%20of%20the%20environment%20in%20methicillin-resistant%20Staphylococcus%20aureus%20acquisition%3A%20the%20case%20for%20hospital%20cleaning&amp;journal=Lancet%20Infect.%20Dis.&amp;doi=10.1016%2FS1473-3099%2807%2970241-4&amp;volume=8&amp;pages=101-113&amp;publication_year=2008&amp;author=Dancer%2CSJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">de Goffau, M. C. et al. Recognizing the reagent microbiome. <i>Nat. Microbiol.</i> <b>3</b>, 851–853 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41564-018-0202-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41564-018-0202-y" data-track-item_id="10.1038/s41564-018-0202-y">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30046175" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtl2jt7%2FJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Recognizing%20the%20reagent%20microbiome&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-018-0202-y&amp;volume=3&amp;pages=851-853&amp;publication_year=2018&amp;author=Goffau%2CMC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Sehulster, L. &amp; Chinn, R. Y. Guidelines for environmental infection control in health-care facilities. Recommendations of CDC and the Healthcare Infection Control Practices Advisory Committee (HICPAC). <i>MMWR Recomm. Rep.</i> <b>52</b>, 1–42 (2003).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=12836624" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Guidelines%20for%20environmental%20infection%20control%20in%20health-care%20facilities.%20Recommendations%20of%20CDC%20and%20the%20Healthcare%20Infection%20Control%20Practices%20Advisory%20Committee%20%28HICPAC%29&amp;journal=MMWR%20Recomm.%20Rep.&amp;volume=52&amp;pages=1-42&amp;publication_year=2003&amp;author=Sehulster%2CL&amp;author=Chinn%2CRY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">Kizny Gordon, A. E. et al. The hospital water environment as a reservoir for carbapenem-resistant organisms causing hospital-acquired infections—a systematic review of the literature. <i>Clin. Infect. Dis.</i> <b>64</b>, 1435–1444 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fcid%2Fcix132" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/cid/cix132" data-track-item_id="10.1093/cid/cix132">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28200000" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20hospital%20water%20environment%20as%20a%20reservoir%20for%20carbapenem-resistant%20organisms%20causing%20hospital-acquired%20infections%E2%80%94a%20systematic%20review%20of%20the%20literature&amp;journal=Clin.%20Infect.%20Dis.&amp;doi=10.1093%2Fcid%2Fcix132&amp;volume=64&amp;pages=1435-1444&amp;publication_year=2017&amp;author=Kizny%20Gordon%2CAE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Danko, D.C. et al. Global genetic cartography of urban metagenomes and anti-microbial resistance. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/724526" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/724526">https://doi.org/10.1101/724526</a> (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">Beggs, C., Knibbs, L. D., Johnson, G. R. &amp; Morawska, L. Environmental contamination and hospital-acquired infection: factors that are easily overlooked. <i>Indoor Air</i> <b>25</b>, 462–474 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2Fina.12170" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/ina.12170" data-track-item_id="10.1111/ina.12170">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFWkur3L" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25346039" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Environmental%20contamination%20and%20hospital-acquired%20infection%3A%20factors%20that%20are%20easily%20overlooked&amp;journal=Indoor%20Air&amp;doi=10.1111%2Fina.12170&amp;volume=25&amp;pages=462-474&amp;publication_year=2015&amp;author=Beggs%2CC&amp;author=Knibbs%2CLD&amp;author=Johnson%2CGR&amp;author=Morawska%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Ministry of Health Singapore. <i>Guidelines for Control and Prevention of Multi-drug Resistant Organisms (MDROs) in Healthcare Facilities</i> (2013).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">Lee, J. Y. H. et al. Global spread of three multidrug-resistant lineages of <i>Staphylococcus epidermidis</i>. <i>Nat. Microbiol.</i> <b>3</b>, 1175–1185 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41564-018-0230-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41564-018-0230-7" data-track-item_id="10.1038/s41564-018-0230-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1CgsLbO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30177740" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6660648" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Global%20spread%20of%20three%20multidrug-resistant%20lineages%20of%20Staphylococcus%20epidermidis&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-018-0230-7&amp;volume=3&amp;pages=1175-1185&amp;publication_year=2018&amp;author=Lee%2CJYH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">Wright, G. D. Environmental and clinical antibiotic resistomes, same only different. <i>Curr. Opin. Microbiol.</i> <b>51</b>, 57–63 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.mib.2019.06.005" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.mib.2019.06.005" data-track-item_id="10.1016/j.mib.2019.06.005">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtlKjsr3E" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31330416" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Environmental%20and%20clinical%20antibiotic%20resistomes%2C%20same%20only%20different&amp;journal=Curr.%20Opin.%20Microbiol.&amp;doi=10.1016%2Fj.mib.2019.06.005&amp;volume=51&amp;pages=57-63&amp;publication_year=2019&amp;author=Wright%2CGD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Pasolli, E. et al. Extensive unexplored human microbiome diversity revealed by over 150,000 genomes from metagenomes spanning age, geography, and lifestyle. <i>Cell</i> <b>176</b>, 649–662 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2019.01.001" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2019.01.001" data-track-item_id="10.1016/j.cell.2019.01.001">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVars7w%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30661755" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6349461" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Extensive%20unexplored%20human%20microbiome%20diversity%20revealed%20by%20over%20150%2C000%20genomes%20from%20metagenomes%20spanning%20age%2C%20geography%2C%20and%20lifestyle&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.01.001&amp;volume=176&amp;pages=649-662&amp;publication_year=2019&amp;author=Pasolli%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Almeida, A. et al. A new genomic blueprint of the human gut microbiota. <i>Nature</i> <b>568</b>, 499–504 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-019-0965-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-019-0965-1" data-track-item_id="10.1038/s41586-019-0965-1">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXmslKhu7s%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30745586" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6784870" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20new%20genomic%20blueprint%20of%20the%20human%20gut%20microbiota&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-0965-1&amp;volume=568&amp;pages=499-504&amp;publication_year=2019&amp;author=Almeida%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">Parks, D. H. et al. A standardized bacterial taxonomy based on genome phylogeny substantially revises the tree of life. <i>Nat. Biotechnol.</i> <b>36</b>, 996–1004 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnbt.4229" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nbt.4229" data-track-item_id="10.1038/nbt.4229">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsF2js7jN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30148503" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20standardized%20bacterial%20taxonomy%20based%20on%20genome%20phylogeny%20substantially%20revises%20the%20tree%20of%20life&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.4229&amp;volume=36&amp;pages=996-1004&amp;publication_year=2018&amp;author=Parks%2CDH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">San Millan, A. Evolution of plasmid-mediated antibiotic resistance in the clinical context. <i>Trends Microbiol.</i> <b>26</b>, 978–985 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.tim.2018.06.007" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.tim.2018.06.007" data-track-item_id="10.1016/j.tim.2018.06.007">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlSksb7F" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30049587" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Evolution%20of%20plasmid-mediated%20antibiotic%20resistance%20in%20the%20clinical%20context&amp;journal=Trends%20Microbiol.&amp;doi=10.1016%2Fj.tim.2018.06.007&amp;volume=26&amp;pages=978-985&amp;publication_year=2018&amp;author=San%20Millan%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">Calero-Caceres, W., Ye, M. &amp; Balcazar, J. L. Bacteriophages as environmental reservoirs of antibiotic resistance. <i>Trends Microbiol.</i> <b>27</b>, 570–577 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.tim.2019.02.008" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.tim.2019.02.008" data-track-item_id="10.1016/j.tim.2019.02.008">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXks1elt74%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30905524" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Bacteriophages%20as%20environmental%20reservoirs%20of%20antibiotic%20resistance&amp;journal=Trends%20Microbiol.&amp;doi=10.1016%2Fj.tim.2019.02.008&amp;volume=27&amp;pages=570-577&amp;publication_year=2019&amp;author=Calero-Caceres%2CW&amp;author=Ye%2CM&amp;author=Balcazar%2CJL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Galata, V., Fehlmann, T., Backes, C. &amp; Keller, A. PLSDB: a resource of complete bacterial plasmids. <i>Nucleic Acids Res.</i> <b>47</b>, D195–D202 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgky1050" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gky1050" data-track-item_id="10.1093/nar/gky1050">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1GqsbrF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30380090" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=PLSDB%3A%20a%20resource%20of%20complete%20bacterial%20plasmids&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky1050&amp;volume=47&amp;pages=D195-D202&amp;publication_year=2019&amp;author=Galata%2CV&amp;author=Fehlmann%2CT&amp;author=Backes%2CC&amp;author=Keller%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Gao, N. L. et al. MVP: a microbe–phage interaction database. <i>Nucleic Acids Res</i> <b>46</b>, D700–D707 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgkx1124" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gkx1124" data-track-item_id="10.1093/nar/gkx1124">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitlGisLvL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29177508" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MVP%3A%20a%20microbe%E2%80%93phage%20interaction%20database&amp;journal=Nucleic%20Acids%20Res&amp;doi=10.1093%2Fnar%2Fgkx1124&amp;volume=46&amp;pages=D700-D707&amp;publication_year=2018&amp;author=Gao%2CNL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Duran, N., Temiz, M., Duran, G. G., Eryilmaz, N. &amp; Jenedi, K. Relationship between the resistance genes to quaternary ammonium compounds and antibiotic resistance in staphylococci isolated from surgical site infections. <i>Med. Sci. Monit.</i> <b>20</b>, 544–550 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.12659%2FMSM.890177" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.12659/MSM.890177" data-track-item_id="10.12659/MSM.890177">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24691183" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3983099" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhs12rtrnO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Relationship%20between%20the%20resistance%20genes%20to%20quaternary%20ammonium%20compounds%20and%20antibiotic%20resistance%20in%20staphylococci%20isolated%20from%20surgical%20site%20infections&amp;journal=Med.%20Sci.%20Monit.&amp;doi=10.12659%2FMSM.890177&amp;volume=20&amp;pages=544-550&amp;publication_year=2014&amp;author=Duran%2CN&amp;author=Temiz%2CM&amp;author=Duran%2CGG&amp;author=Eryilmaz%2CN&amp;author=Jenedi%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">Johnson, R. C. et al. Investigation of a cluster of <i>Sphingomonas koreensis</i> infections. <i>N. Engl. J. Med.</i> <b>379</b>, 2529–2539 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa1803238" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa1803238" data-track-item_id="10.1056/NEJMoa1803238">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXltF2qtw%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30586509" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6322212" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Investigation%20of%20a%20cluster%20of%20Sphingomonas%20koreensis%20infections&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1803238&amp;volume=379&amp;pages=2529-2539&amp;publication_year=2018&amp;author=Johnson%2CRC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">Fahimipour, A.K. et al. Antimicrobial chemicals associate with microbial function and antibiotic resistance indoors. <i>mSystems</i> <b>3</b>, e00200-18 (2018).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">Hartmann, E. M. et al. Antimicrobial chemicals are associated with elevated antibiotic resistance genes in the indoor dust microbiome. <i>Environ. Sci. Technol.</i> <b>50</b>, 9807–9815 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1021%2Facs.est.6b00262" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1021/acs.est.6b00262" data-track-item_id="10.1021/acs.est.6b00262">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVGnsLnN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27599587" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5032049" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Antimicrobial%20chemicals%20are%20associated%20with%20elevated%20antibiotic%20resistance%20genes%20in%20the%20indoor%20dust%20microbiome&amp;journal=Environ.%20Sci.%20Technol.&amp;doi=10.1021%2Facs.est.6b00262&amp;volume=50&amp;pages=9807-9815&amp;publication_year=2016&amp;author=Hartmann%2CEM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">Falagas, M. E., Roussos, N., Gkegkes, I. D., Rafailidis, P. I. &amp; Karageorgopoulos, D. E. Fosfomycin for the treatment of infections caused by Gram-positive cocci with advanced antimicrobial drug resistance: a review of microbiological, animal and clinical studies. <i>Expert Opin. Investig. Drugs</i> <b>18</b>, 921–944 (2009).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1517%2F13543780902967624" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1517/13543780902967624" data-track-item_id="10.1517/13543780902967624">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXns1erurY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19548851" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fosfomycin%20for%20the%20treatment%20of%20infections%20caused%20by%20Gram-positive%20cocci%20with%20advanced%20antimicrobial%20drug%20resistance%3A%20a%20review%20of%20microbiological%2C%20animal%20and%20clinical%20studies&amp;journal=Expert%20Opin.%20Investig.%20Drugs&amp;doi=10.1517%2F13543780902967624&amp;volume=18&amp;pages=921-944&amp;publication_year=2009&amp;author=Falagas%2CME&amp;author=Roussos%2CN&amp;author=Gkegkes%2CID&amp;author=Rafailidis%2CPI&amp;author=Karageorgopoulos%2CDE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">Urban, C., Mariano, N. &amp; Rahal, J. J. In vitro double and triple bactericidal activities of doripenem, polymyxin B, and rifampin against multidrug-resistant <i>Acinetobacter baumannii</i>, <i>Pseudomonas aeruginosa</i>, <i>Klebsiella pneumoniae</i>, and <i>Escherichia coli</i>. <i>Antimicrob. Agents Chemother.</i> <b>54</b>, 2732–2734 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1128%2FAAC.01768-09" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1128/AAC.01768-09" data-track-item_id="10.1128/AAC.01768-09">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXos1Ogsbk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20368401" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2876416" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=In%20vitro%20double%20and%20triple%20bactericidal%20activities%20of%20doripenem%2C%20polymyxin%20B%2C%20and%20rifampin%20against%20multidrug-resistant%20Acinetobacter%20baumannii%2C%20Pseudomonas%20aeruginosa%2C%20Klebsiella%20pneumoniae%2C%20and%20Escherichia%20coli&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;doi=10.1128%2FAAC.01768-09&amp;volume=54&amp;pages=2732-2734&amp;publication_year=2010&amp;author=Urban%2CC&amp;author=Mariano%2CN&amp;author=Rahal%2CJJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">Izadpanah, M. &amp; Khalili, H. Antibiotic regimens for treatment of infections due to multidrug-resistant Gram-negative pathogens: an evidence-based literature review. <i>J. Res. Pharm. Pract.</i> <b>4</b>, 105–114 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.4103%2F2279-042X.162360" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.4103/2279-042X.162360" data-track-item_id="10.4103/2279-042X.162360">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVaitLfI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26312249" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4548428" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Antibiotic%20regimens%20for%20treatment%20of%20infections%20due%20to%20multidrug-resistant%20Gram-negative%20pathogens%3A%20an%20evidence-based%20literature%20review&amp;journal=J.%20Res.%20Pharm.%20Pract.&amp;doi=10.4103%2F2279-042X.162360&amp;volume=4&amp;pages=105-114&amp;publication_year=2015&amp;author=Izadpanah%2CM&amp;author=Khalili%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">Werner, G., Hildebrandt, B. &amp; Witte, W. Aminoglycoside-streptothricin resistance gene cluster <i>aadE</i>–<i>sat4</i>–<i>aphA</i>-<i>3</i> disseminated among multiresistant isolates of <i>Enterococcus faecium</i>. <i>Antimicrob. Agents Chemother.</i> <b>45</b>, 3267–3269 (2001).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1128%2FAAC.45.11.3267-3269.2001" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1128/AAC.45.11.3267-3269.2001" data-track-item_id="10.1128/AAC.45.11.3267-3269.2001">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD3MXnvF2jtb4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11600397" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC90823" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Aminoglycoside-streptothricin%20resistance%20gene%20cluster%20aadE%E2%80%93sat4%E2%80%93aphA-3%20disseminated%20among%20multiresistant%20isolates%20of%20Enterococcus%20faecium&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;doi=10.1128%2FAAC.45.11.3267-3269.2001&amp;volume=45&amp;pages=3267-3269&amp;publication_year=2001&amp;author=Werner%2CG&amp;author=Hildebrandt%2CB&amp;author=Witte%2CW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR46">Weinstein, L., Chang, T. W., Hudson, J. B. &amp; Hartl, W. The concurrent use of sulfonamides and antibiotics in the treatment of infections: in vivo and in vitro studies of the effect of sulfonamide-antibiotic combinations on the emergence of drug resistance. <i>Ann. NY Acad. Sci.</i> <b>69</b>, 408–416 (1957).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2Fj.1749-6632.1957.tb49681.x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/j.1749-6632.1957.tb49681.x" data-track-item_id="10.1111/j.1749-6632.1957.tb49681.x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaG1c%2FjtV2rsg%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=13488271" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20concurrent%20use%20of%20sulfonamides%20and%20antibiotics%20in%20the%20treatment%20of%20infections%3A%20in%20vivo%20and%20in%20vitro%20studies%20of%20the%20effect%20of%20sulfonamide-antibiotic%20combinations%20on%20the%20emergence%20of%20drug%20resistance&amp;journal=Ann.%20NY%20Acad.%20Sci.&amp;doi=10.1111%2Fj.1749-6632.1957.tb49681.x&amp;volume=69&amp;pages=408-416&amp;publication_year=1957&amp;author=Weinstein%2CL&amp;author=Chang%2CTW&amp;author=Hudson%2CJB&amp;author=Hartl%2CW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR47">Sundin, G. W. &amp; Bender, C. L. Dissemination of the <i>strA</i>–<i>strB</i> streptomycin-resistance genes among commensal and pathogenic bacteria from humans, animals, and plants. <i>Mol. Ecol.</i> <b>5</b>, 133–143 (1996).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2Fj.1365-294X.1996.tb00299.x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/j.1365-294X.1996.tb00299.x" data-track-item_id="10.1111/j.1365-294X.1996.tb00299.x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DyaK28Xit12hu7w%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=9147689" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Dissemination%20of%20the%20strA%E2%80%93strB%20streptomycin-resistance%20genes%20among%20commensal%20and%20pathogenic%20bacteria%20from%20humans%2C%20animals%2C%20and%20plants&amp;journal=Mol.%20Ecol.&amp;doi=10.1111%2Fj.1365-294X.1996.tb00299.x&amp;volume=5&amp;pages=133-143&amp;publication_year=1996&amp;author=Sundin%2CGW&amp;author=Bender%2CCL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR48">Monecke, S. et al. Microarray-based characterisation of a Panton–Valentine leukocidin-positive community-acquired strain of methicillin-resistant <i>Staphylococcus aureus</i>. <i>Clin. Microbiol. Infect.</i> <b>12</b>, 718–728 (2006).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2Fj.1469-0691.2006.01420.x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/j.1469-0691.2006.01420.x" data-track-item_id="10.1111/j.1469-0691.2006.01420.x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XosFamu70%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16842566" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Microarray-based%20characterisation%20of%20a%20Panton%E2%80%93Valentine%20leukocidin-positive%20community-acquired%20strain%20of%20methicillin-resistant%20Staphylococcus%20aureus&amp;journal=Clin.%20Microbiol.%20Infect.&amp;doi=10.1111%2Fj.1469-0691.2006.01420.x&amp;volume=12&amp;pages=718-728&amp;publication_year=2006&amp;author=Monecke%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR49">Teo, J. et al. First case of <i>E. anophelis</i> outbreak in an intensive-care unit. <i>Lancet</i> <b>382</b>, 855–856 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0140-6736%2813%2961858-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0140-6736(13)61858-9" data-track-item_id="10.1016/S0140-6736(13)61858-9">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24012265" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=First%20case%20of%20E.%20anophelis%20outbreak%20in%20an%20intensive-care%20unit&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2813%2961858-9&amp;volume=382&amp;pages=855-856&amp;publication_year=2013&amp;author=Teo%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR50">Htun, H. L. et al. Methicillin-resistant <i>Staphylococcus aureus</i> colonisation: epidemiological and molecular characteristics in an acute-care tertiary hospital in Singapore. <i>Epidemiol. Infect.</i> <b>146</b>, 1785–1792 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1017%2FS0950268818001966" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1017/S0950268818001966" data-track-item_id="10.1017/S0950268818001966">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisVyhsbjP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30019657" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Methicillin-resistant%20Staphylococcus%20aureus%20colonisation%3A%20epidemiological%20and%20molecular%20characteristics%20in%20an%20acute-care%20tertiary%20hospital%20in%20Singapore&amp;journal=Epidemiol.%20Infect.&amp;doi=10.1017%2FS0950268818001966&amp;volume=146&amp;pages=1785-1792&amp;publication_year=2018&amp;author=Htun%2CHL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR51">Nasser, K. et al. Draft genome sequences of six multidrug-resistant clinical strains of <i>Acinetobacter baumannii</i>, isolated at two major hospitals in Kuwait. <i>Genome Announc.</i> <b>6</b>, e00264-18 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1128%2FgenomeA.00264-18" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1128/genomeA.00264-18" data-track-item_id="10.1128/genomeA.00264-18">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29674542" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5908928" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Draft%20genome%20sequences%20of%20six%20multidrug-resistant%20clinical%20strains%20of%20Acinetobacter%20baumannii%2C%20isolated%20at%20two%20major%20hospitals%20in%20Kuwait&amp;journal=Genome%20Announc.&amp;doi=10.1128%2FgenomeA.00264-18&amp;volume=6&amp;publication_year=2018&amp;author=Nasser%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR52">Walker, J. T. et al. Hospital and community acquired infection and the built environment—design and testing of infection control rooms. <i>J. Hosp. Infect.</i> <b>65</b>, 43–49 (2007).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0195-6701%2807%2960014-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0195-6701(07)60014-0" data-track-item_id="10.1016/S0195-6701(07)60014-0">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=17540241" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7134456" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Hospital%20and%20community%20acquired%20infection%20and%20the%20built%20environment%E2%80%94design%20and%20testing%20of%20infection%20control%20rooms&amp;journal=J.%20Hosp.%20Infect.&amp;doi=10.1016%2FS0195-6701%2807%2960014-0&amp;volume=65&amp;pages=43-49&amp;publication_year=2007&amp;author=Walker%2CJT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR53">Ladner, J. T., Grubaugh, N. D., Pybus, O. G. &amp; Andersen, K. G. Precision epidemiology for infectious disease control. <i>Nat. Med.</i> <b>25</b>, 206–211 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41591-019-0345-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41591-019-0345-2" data-track-item_id="10.1038/s41591-019-0345-2">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXntlKrtbk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30728537" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7095960" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Precision%20epidemiology%20for%20infectious%20disease%20control&amp;journal=Nat.%20Med.&amp;doi=10.1038%2Fs41591-019-0345-2&amp;volume=25&amp;pages=206-211&amp;publication_year=2019&amp;author=Ladner%2CJT&amp;author=Grubaugh%2CND&amp;author=Pybus%2COG&amp;author=Andersen%2CKG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR54">Mahnert, A. et al. Man-made microbial resistances in built environments. <i>Nat. Commun.</i> <b>10</b>, 968 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-019-08864-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-019-08864-0" data-track-item_id="10.1038/s41467-019-08864-0">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30814504" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6393488" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXnslCns70%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Man-made%20microbial%20resistances%20in%20built%20environments&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-08864-0&amp;volume=10&amp;publication_year=2019&amp;author=Mahnert%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR55">Goodman, E. R. et al. Impact of an environmental cleaning intervention on the presence of methicillin-resistant <i>Staphylococcus aureus</i> and vancomycin-resistant enterococci on surfaces in intensive care unit rooms. <i>Infect. Control Hosp. Epidemiol.</i> <b>29</b>, 593–599 (2008).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1086%2F588566" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1086/588566" data-track-item_id="10.1086/588566">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18624666" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2670228" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Impact%20of%20an%20environmental%20cleaning%20intervention%20on%20the%20presence%20of%20methicillin-resistant%20Staphylococcus%20aureus%20and%20vancomycin-resistant%20enterococci%20on%20surfaces%20in%20intensive%20care%20unit%20rooms&amp;journal=Infect.%20Control%20Hosp.%20Epidemiol.&amp;doi=10.1086%2F588566&amp;volume=29&amp;pages=593-599&amp;publication_year=2008&amp;author=Goodman%2CER" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR56">Lerminiaux, N. A. &amp; Cameron, A. D. S. Horizontal transfer of antibiotic resistance genes in clinical environments. <i>Can. J. Microbiol.</i> <b>65</b>, 34–44 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1139%2Fcjm-2018-0275" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1139/cjm-2018-0275" data-track-item_id="10.1139/cjm-2018-0275">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXit1CrtL%2FM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30248271" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Horizontal%20transfer%20of%20antibiotic%20resistance%20genes%20in%20clinical%20environments&amp;journal=Can.%20J.%20Microbiol.&amp;doi=10.1139%2Fcjm-2018-0275&amp;volume=65&amp;pages=34-44&amp;publication_year=2019&amp;author=Lerminiaux%2CNA&amp;author=Cameron%2CADS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR57">Buckner, M. M. C., Ciusa, M. L. &amp; Piddock, L. J. V. Strategies to combat antimicrobial resistance: anti-plasmid and plasmid curing. <i>FEMS Microbiol. Rev.</i> <b>42</b>, 781–804 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Ffemsre%2Ffuy031" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/femsre/fuy031" data-track-item_id="10.1093/femsre/fuy031">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXht1WitLfM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30085063" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6199537" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Strategies%20to%20combat%20antimicrobial%20resistance%3A%20anti-plasmid%20and%20plasmid%20curing&amp;journal=FEMS%20Microbiol.%20Rev.&amp;doi=10.1093%2Ffemsre%2Ffuy031&amp;volume=42&amp;pages=781-804&amp;publication_year=2018&amp;author=Buckner%2CMMC&amp;author=Ciusa%2CML&amp;author=Piddock%2CLJV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR58">David, S. et al. Epidemic of carbapenem-resistant <i>Klebsiella pneumoniae</i> in Europe is driven by nosocomial spread. <i>Nat. Microbiol.</i> <b>4</b>, 1919–1929 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41564-019-0492-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41564-019-0492-8" data-track-item_id="10.1038/s41564-019-0492-8">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsVKnsbrF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31358985" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7244338" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Epidemic%20of%20carbapenem-resistant%20Klebsiella%20pneumoniae%20in%20Europe%20is%20driven%20by%20nosocomial%20spread&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-019-0492-8&amp;volume=4&amp;pages=1919-1929&amp;publication_year=2019&amp;author=David%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR59">Ho, C. M. et al. High rate of <i>qacA</i>- and <i>qacB</i>-positive methicillin-resistant <i>Staphylococcus aureus</i> isolates from chlorhexidine-impregnated catheter-related bloodstream infections. <i>Antimicrob. Agents Chemother.</i> <b>56</b>, 5693–5697 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1128%2FAAC.00761-12" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1128/AAC.00761-12" data-track-item_id="10.1128/AAC.00761-12">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhsF2iurrJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22908163" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3486537" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=High%20rate%20of%20qacA-%20and%20qacB-positive%20methicillin-resistant%20Staphylococcus%20aureus%20isolates%20from%20chlorhexidine-impregnated%20catheter-related%20bloodstream%20infections&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;doi=10.1128%2FAAC.00761-12&amp;volume=56&amp;pages=5693-5697&amp;publication_year=2012&amp;author=Ho%2CCM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR60">Weiner, L. M. et al. Antimicrobial-resistant pathogens associated with healthcare-associated infections: summary of data reported to the National Healthcare Safety Network at the Centers for Disease Control and Prevention, 2011–2014. <i>Infect. Control Hosp. Epidemiol.</i> <b>37</b>, 1288–1301 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1017%2Fice.2016.174" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1017/ice.2016.174" data-track-item_id="10.1017/ice.2016.174">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27573805" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6857725" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Antimicrobial-resistant%20pathogens%20associated%20with%20healthcare-associated%20infections%3A%20summary%20of%20data%20reported%20to%20the%20National%20Healthcare%20Safety%20Network%20at%20the%20Centers%20for%20Disease%20Control%20and%20Prevention%2C%202011%E2%80%932014&amp;journal=Infect.%20Control%20Hosp.%20Epidemiol.&amp;doi=10.1017%2Fice.2016.174&amp;volume=37&amp;pages=1288-1301&amp;publication_year=2016&amp;author=Weiner%2CLM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR61">Truong, D. T. et al. MetaPhlAn2 for enhanced metagenomic taxonomic profiling. <i>Nat. Methods</i> <b>12</b>, 902–903 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnmeth.3589" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nmeth.3589" data-track-item_id="10.1038/nmeth.3589">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFyqsL7I" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26418763" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MetaPhlAn2%20for%20enhanced%20metagenomic%20taxonomic%20profiling&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.3589&amp;volume=12&amp;pages=902-903&amp;publication_year=2015&amp;author=Truong%2CDT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR62">Inouye, M. et al. SRST2: rapid genomic surveillance for public health and hospital microbiology labs. <i>Genome Med.</i> <b>6</b>, 90 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13073-014-0090-6" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13073-014-0090-6" data-track-item_id="10.1186/s13073-014-0090-6">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25422674" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4237778" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SRST2%3A%20rapid%20genomic%20surveillance%20for%20public%20health%20and%20hospital%20microbiology%20labs&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-014-0090-6&amp;volume=6&amp;publication_year=2014&amp;author=Inouye%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR63">Wood, D. E. &amp; Salzberg, S. L. Kraken: ultrafast metagenomic sequence classification using exact alignments. <i>Genome Biol.</i> <b>15</b>, r46 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/gb-2014-15-3-r46" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/gb-2014-15-3-r46" data-track-item_id="10.1186/gb-2014-15-3-r46">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24580807" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4053813" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Kraken%3A%20ultrafast%20metagenomic%20sequence%20classification%20using%20exact%20alignments&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fgb-2014-15-3-r46&amp;volume=15&amp;publication_year=2014&amp;author=Wood%2CDE&amp;author=Salzberg%2CSL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR64">Koren, S. et al. Canu: scalable and accurate long-read assembly via adaptive <i>k</i>-mer weighting and repeat separation. <i>Genome Res.</i> <b>27</b>, 722–736 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1101%2Fgr.215087.116" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1101/gr.215087.116" data-track-item_id="10.1101/gr.215087.116">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFyjsrvI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28298431" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5411767" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Canu%3A%20scalable%20and%20accurate%20long-read%20assembly%20via%20adaptive%20k-mer%20weighting%20and%20repeat%20separation&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.215087.116&amp;volume=27&amp;pages=722-736&amp;publication_year=2017&amp;author=Koren%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR65">Kurtz, S. et al. Versatile and open software for comparing large genomes. <i>Genome Biol.</i> <b>5</b>, r12 (2004).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/gb-2004-5-2-r12" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/gb-2004-5-2-r12" data-track-item_id="10.1186/gb-2004-5-2-r12">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14759262" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC395750" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Versatile%20and%20open%20software%20for%20comparing%20large%20genomes&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fgb-2004-5-2-r12&amp;volume=5&amp;publication_year=2004&amp;author=Kurtz%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR66">Walker, B. J. et al. Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement. <i>PLoS ONE</i> <b>9</b>, e112963 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pone.0112963" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pone.0112963" data-track-item_id="10.1371/journal.pone.0112963">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25409509" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4237348" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXisFCmurk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0112963&amp;volume=9&amp;publication_year=2014&amp;author=Walker%2CBJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR67">Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. &amp; Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. <i>Genome Res.</i> <b>25</b>, 1043–1055 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1101%2Fgr.186072.114" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1101/gr.186072.114" data-track-item_id="10.1101/gr.186072.114">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1SltbfE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25977477" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4484387" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CheckM%3A%20assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates%2C%20single%20cells%2C%20and%20metagenomes&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.186072.114&amp;volume=25&amp;pages=1043-1055&amp;publication_year=2015&amp;author=Parks%2CDH&amp;author=Imelfort%2CM&amp;author=Skennerton%2CCT&amp;author=Hugenholtz%2CP&amp;author=Tyson%2CGW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR68">Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. <i>Genome Biol.</i> <b>17</b>, 132 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-016-0997-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-016-0997-x" data-track-item_id="10.1186/s13059-016-0997-x">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27323842" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4915045" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXns1entr0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Mash%3A%20fast%20genome%20and%20metagenome%20distance%20estimation%20using%20MinHash&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-016-0997-x&amp;volume=17&amp;publication_year=2016&amp;author=Ondov%2CBD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR69">Pierce, N. T., Irber, L., Reiter, T., Brooks, P. &amp; Brown, C. T. Large-scale sequence comparisons with sourmash. <i>F1000Res</i> <b>8</b>, 1006 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.12688%2Ff1000research.19675.1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.12688/f1000research.19675.1" data-track-item_id="10.12688/f1000research.19675.1">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31508216" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6720031" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Large-scale%20sequence%20comparisons%20with%20sourmash&amp;journal=F1000Res&amp;doi=10.12688%2Ff1000research.19675.1&amp;volume=8&amp;publication_year=2019&amp;author=Pierce%2CNT&amp;author=Irber%2CL&amp;author=Reiter%2CT&amp;author=Brooks%2CP&amp;author=Brown%2CCT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR70">Gupta, S. K. et al. ARG-ANNOT, a new bioinformatic tool to discover antibiotic resistance genes in bacterial genomes. <i>Antimicrob. Agents Chemother.</i> <b>58</b>, 212–220 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1128%2FAAC.01310-13" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1128/AAC.01310-13" data-track-item_id="10.1128/AAC.01310-13">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24145532" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3910750" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhvFCiu7Y%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ARG-ANNOT%2C%20a%20new%20bioinformatic%20tool%20to%20discover%20antibiotic%20resistance%20genes%20in%20bacterial%20genomes&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;doi=10.1128%2FAAC.01310-13&amp;volume=58&amp;pages=212-220&amp;publication_year=2014&amp;author=Gupta%2CSK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR71">Wattam, A. R. et al. PATRIC, the bacterial bioinformatics database and analysis resource. <i>Nucleic Acids Res.</i> <b>42</b>, D581–D591 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgkt1099" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gkt1099" data-track-item_id="10.1093/nar/gkt1099">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXos1Wh" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24225323" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=PATRIC%2C%20the%20bacterial%20bioinformatics%20database%20and%20analysis%20resource&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkt1099&amp;volume=42&amp;pages=D581-D591&amp;publication_year=2014&amp;author=Wattam%2CAR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR72">Roux, S., Enault, F., Hurwitz, B. L. &amp; Sullivan, M. B. VirSorter: mining viral signal from microbial genomic data. <i>PeerJ</i> <b>3</b>, e985 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.7717%2Fpeerj.985" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.7717/peerj.985" data-track-item_id="10.7717/peerj.985">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26038737" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4451026" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XlsVCntLk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=VirSorter%3A%20mining%20viral%20signal%20from%20microbial%20genomic%20data&amp;journal=PeerJ&amp;doi=10.7717%2Fpeerj.985&amp;volume=3&amp;publication_year=2015&amp;author=Roux%2CS&amp;author=Enault%2CF&amp;author=Hurwitz%2CBL&amp;author=Sullivan%2CMB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR73">Aziz, R. K. et al. The RAST Server: rapid annotations using subsystems technology. <i>BMC Genomics</i> <b>9</b>, 75 (2008).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/1471-2164-9-75" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/1471-2164-9-75" data-track-item_id="10.1186/1471-2164-9-75">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18261238" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2265698" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXlslSnsLo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20RAST%20Server%3A%20rapid%20annotations%20using%20subsystems%20technology&amp;journal=BMC%20Genomics&amp;doi=10.1186%2F1471-2164-9-75&amp;volume=9&amp;publication_year=2008&amp;author=Aziz%2CRK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR74">Gao, S., Bertrand, D., Chia, B. K. &amp; Nagarajan, N. OPERA-LG: efficient and exact scaffolding of large, repeat-rich eukaryotic genomes with performance guarantees. <i>Genome Biol.</i> <b>17</b>, 102 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-016-0951-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-016-0951-y" data-track-item_id="10.1186/s13059-016-0951-y">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27169502" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4864936" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVektb%2FN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=OPERA-LG%3A%20efficient%20and%20exact%20scaffolding%20of%20large%2C%20repeat-rich%20eukaryotic%20genomes%20with%20performance%20guarantees&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-016-0951-y&amp;volume=17&amp;publication_year=2016&amp;author=Gao%2CS&amp;author=Bertrand%2CD&amp;author=Chia%2CBK&amp;author=Nagarajan%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR75">Gao, S., Bertrand, D., &amp; Nagarajan, N. FinIS: improved in silico finishing using an exact quadratic programming formulation. In <i>Algorithms in Bioinformatics</i> (Eds. Raphael, B. &amp; Tang, J.) 314–325 (Springer, 2012).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR76">Treangen, T. J., Ondov, B. D., Koren, S. &amp; Phillippy, A. M. The Harvest suite for rapid core-genome alignment and visualization of thousands of intraspecific microbial genomes. <i>Genome Biol.</i> <b>15</b>, 524 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-014-0524-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-014-0524-x" data-track-item_id="10.1186/s13059-014-0524-x">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25410596" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4262987" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXivVKnsr0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Harvest%20suite%20for%20rapid%20core-genome%20alignment%20and%20visualization%20of%20thousands%20of%20intraspecific%20microbial%20genomes&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-014-0524-x&amp;volume=15&amp;publication_year=2014&amp;author=Treangen%2CTJ&amp;author=Ondov%2CBD&amp;author=Koren%2CS&amp;author=Phillippy%2CAM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR77">Bruen, T. C., Philippe, H. &amp; Bryant, D. A simple and robust statistical test for detecting the presence of recombination. <i>Genetics</i> <b>172</b>, 2665–2681 (2006).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1534%2Fgenetics.105.048975" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1534/genetics.105.048975" data-track-item_id="10.1534/genetics.105.048975">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD28XkvFOgtLc%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=16489234" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC1456386" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20simple%20and%20robust%20statistical%20test%20for%20detecting%20the%20presence%20of%20recombination&amp;journal=Genetics&amp;doi=10.1534%2Fgenetics.105.048975&amp;volume=172&amp;pages=2665-2681&amp;publication_year=2006&amp;author=Bruen%2CTC&amp;author=Philippe%2CH&amp;author=Bryant%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR78">Price, M. N., Dehal, P. S. &amp; Arkin, A. P. FastTree 2—approximately maximum-likelihood trees for large alignments. <i>PLoS ONE</i> <b>5</b>, e9490 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pone.0009490" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pone.0009490" data-track-item_id="10.1371/journal.pone.0009490">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20224823" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2835736" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXjsFCmsLo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=FastTree%202%E2%80%94approximately%20maximum-likelihood%20trees%20for%20large%20alignments&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0009490&amp;volume=5&amp;publication_year=2010&amp;author=Price%2CMN&amp;author=Dehal%2CPS&amp;author=Arkin%2CAP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR79">Yu, G., Lam, T. T., Zhu, H. &amp; Guan, Y. Two methods for mapping and visualizing associated data on phylogeny using ggtree. <i>Mol. Biol. Evol.</i> <b>35</b>, 3041–3043 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fmolbev%2Fmsy194" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/molbev/msy194" data-track-item_id="10.1093/molbev/msy194">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhtFKntrbF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30351396" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6278858" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Two%20methods%20for%20mapping%20and%20visualizing%20associated%20data%20on%20phylogeny%20using%20ggtree&amp;journal=Mol.%20Biol.%20Evol.&amp;doi=10.1093%2Fmolbev%2Fmsy194&amp;volume=35&amp;pages=3041-3043&amp;publication_year=2018&amp;author=Yu%2CG&amp;author=Lam%2CTT&amp;author=Zhu%2CH&amp;author=Guan%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR80">Hsieh, T. C., Ma, K. H. &amp; Chao, A. iNEXT: an R package for rarefaction and extrapolation of species diversity (Hill numbers). <i>Methods Ecol. Evol.</i> <b>7</b>, 1451–1456 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1111%2F2041-210X.12613" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1111/2041-210X.12613" data-track-item_id="10.1111/2041-210X.12613">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=iNEXT%3A%20an%20R%20package%20for%20rarefaction%20and%20extrapolation%20of%20species%20diversity%20%28Hill%20numbers%29&amp;journal=Methods%20Ecol.%20Evol.&amp;doi=10.1111%2F2041-210X.12613&amp;volume=7&amp;pages=1451-1456&amp;publication_year=2016&amp;author=Hsieh%2CTC&amp;author=Ma%2CKH&amp;author=Chao%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li></ol></div></div>
        </div>
    </div>
</aside>
</div>


    
        
    
    
        
            
        

        
            
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>




<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Medicine
                    </span>
                    (<i itemprop="alternateName">Nat Med</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-170X</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1078-8956</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/" data-track="click" data-track-action="protocols.io" data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/authors/editorial_policies" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nature-research/open-access" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/language-editing/" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/" data-track="click" data-track-action="research solutions" data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library" data-track="click" data-track-action="Recommend to library" data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica" data-track="click" data-track-action="nature africa" data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly" data-track="click" data-track-action="nature Italy" data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies" data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice" data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/accessibility-statement" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20">
    </a>
    <p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"></path></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"></path><clipPath id="b"><use xlink:href="#a" style="overflow:visible"></use></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"></path><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"></path><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"></path><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"></path></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"></path></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif " data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing Microbiology" src="/static/images/logos/nature-briefing-microbiology-logo-3fa570e5fb.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing: Microbiology</em> newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/microbiology" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MicrobiologyBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="MicrobiologyBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required="" data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:microbiology" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required="">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="/briefing/microbiology/?brieferEntryPoint=MicrobiologyBriefingBanner">Sign up for Nature Briefing: Microbiology
            </a>
        </div>

    </div>

</div>

    




<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41591-020-0894-4&amp;format=js&amp;last_modified=2020-06-08" async=""></script>

<qjq-tbhgptzoxtep></qjq-tbhgptzoxtep><div data-cc-ghost="" style="height: 352px;"></div><script type="text/javascript" id="fetch-contextual-ads-campaign-data" charset="">(function(){var g=function(a){a=new CustomEvent("campaignDataLoaded",{detail:a});document.dispatchEvent(a)},b=google_tag_manager["rm"]["50443292"](18);if(b){var k=1500;b=google_tag_manager["rm"]["50443292"](19);b="link"===b?google_tag_manager["rm"]["50443292"](20):google_tag_manager["rm"]["50443292"](21);var m=function(a,c){var h=!1,l=setTimeout(function(){h=!0;c(null)},k),d=new XMLHttpRequest;d.onload=function(){var e=null,f=[];if(200===d.status){try{e=JSON.parse(d.responseText)}catch(p){e={}}f=e.campaigns||[]}h||(clearTimeout(l),c(f.length?f.join(","):null))};d.open("GET",
"/platform/contextual?doi\x3d"+a);d.send()},n=function(a,c){a&&-1===a.indexOf("not set")?m(a,c):setTimeout(c,1)};n(b,function(a){window.campaignsForContextualAds=a;g(a);window.dataLayer.push({content:{article:{campaignID:a}}})})}else window.campaignsForContextualAds="",g(null)})();</script>
<script type="text/javascript" id="gtm-setup-accessdetailsloaded-handler" charset="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail||{};var b={event:"update-access-details"};a.institutional_business_partner_ids&&a.resolved_by||console.log("BPID data could not be retrieved from /exposed-details");a.logged_in_to_sn_profile||console.log("logged_in_to_sn_profile could not be retrieved from /exposed-details");var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?
a.resolved_by.join(";"):"",e=a.personal_business_partner_id_found,f=a.logged_in_to_sn_profile,g=a.snid;b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null,personal:e||null,logged_in_to_sn_profile:f,snid:g||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy={};window.dataLayer.push(b);window.idpUserDataLoaded=!0},!1);window.dataLayer.push({event:"accessdetailsloaded-handler-added"});
window.accessDetailsLoadedHandlerAdded=!0;</script><script type="text/javascript" id="gtm-mediatrust" charset="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="crossmark-script" text="" charset="" type="text/javascript" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="polyfill-matches" charset="">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="fetch-idp-user-data" charset="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},site=google_tag_manager["rm"]["50443292"](59),idpUrl;
switch(site){case "nature":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?"https://staging-idp.nature.com/exposed-details":"https://idp.nature.com/exposed-details";break;case "link":idpUrl=-1<window.location.hostname.indexOf("link-qa")?"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details";break;case "springer":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?
"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details"}if(void 0!==idpUrl){var transport=new XMLHttpRequest;transport.open("GET",idpUrl,!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script src="https://verify.nature.com/verify/nature.min.js"></script><script type="text/javascript" id="create-ad-slots" charset="">function createAndLoadAds(){function h(b){"nature"===a&&google_tag_manager["rm"]["50443292"](87)(b);"bmc"!==a&&"springeropen"!==a||google_tag_manager["rm"]["50443292"](99)(b);("link"===a&&"oscar"===google_tag_manager["rm"]["50443292"](100)||"springer"===a||"link"===a&&!0===google_tag_manager["rm"]["50443292"](101))&&google_tag_manager["rm"]["50443292"](126)(b);"link"===a&&"bunsen"===google_tag_manager["rm"]["50443292"](127)&&google_tag_manager["rm"]["50443292"](152)(b);"link"===a&&"Core"===google_tag_manager["rm"]["50443292"](153)&&google_tag_manager["rm"]["50443292"](165)(b)}window.googletag=window.googletag||{cmd:[]};var a=google_tag_manager["rm"]["50443292"](166),e=google_tag_manager["rm"]["50443292"](168);
if(-1===window.location.search.indexOf("hide_ads\x3dtrue"))if(window.adSlots&&0!==Object.keys(window.adSlots).length)console.log("Ads previously loaded. Will not update ad slots until next page load.");else{window.adSlots||(window.adSlots={});window.getAd=function(b,f){for(var c in window.adSlots)if(-1<c.indexOf(b)){if("object"===f)return adSlots[c];if("slot"===f)return adSlots[c].slot}};googletag.cmd.push(function(){googletag.pubads().setPrivacySettings({limitedAds:e});console.log("limitedAds is ",
e);googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["rm"]["50443292"](170));googletag.pubads().enableSingleRequest();googletag.pubads().disableInitialLoad()});for(var g=document.querySelectorAll("[data-gpt]"),d=0;g[d];++d)h(g[d]);googletag.cmd.push(function(){googletag.enableServices()})}}createAndLoadAds();</script>
</body></html>