<html lang="en" class="js"><head><link rel="preload" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" as="font" type="font/woff2" crossorigin=""><script async="" src="//cdn.jsdelivr.net/npm/mathjax@2.7.5/MathJax.js?config=TeX-AMS-MML_SVG.js"></script><script async="" src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script type="text/javascript" async="" src="https://pagead2.googlesyndication.com/tag/js/gpt.js"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-MRVXSHQ"></script><script type="text/javascript" src="https://www.nature.com/vIeRYRA7NIJ4E2or9fin7U_BiziDTZrItPbCszQ4GTYxzMss40CDXUcc1JEmny1VxNqvR3Hl61Srj5GU2jF6tg=="></script><script src="https://cmp.nature.com/production_live/en/consent-bundle-8-85.js" onload="initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')"></script>
    <title>Drivers and determinants of strain dynamics following fecal microbiota transplantation | Nature Medicine</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nm.rss">


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin="">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480">

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"clinical-microbiology;metagenomics;microbial-ecology;microbiome","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Medicine","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41591-022-01913-0"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Thomas S. B. Schmidt","Simone S. Li","Oleksandr M. Maistrenko","Wasiu Akanni","Luis Pedro Coelho","Sibasish Dolai","Anthony Fullam","Anna M. Glazek","Rajna Hercog","Hilde Herrema","Ferris Jung","Stefanie Kandels","Askarbek Orakov","Roman Thielemann","Moritz von Stetten","Thea Van Rossum","Vladimir Benes","Thomas J. Borody","Willem M. de Vos","Cyriel Y. Ponsioen","Max Nieuwdorp","Peer Bork"],"publishedAt":1663200000,"publishedAtString":"2022-09-15","title":"Drivers and determinants of strain dynamics following fecal microbiota transplantation","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Clinical microbiology,Metagenomics,Microbial ecology,Microbiome"},"journal":{"pcode":"nm","title":"nature medicine","volume":"28","issue":"9","id":41591,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"BR","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{font-weight:700;-webkit-font-smoothing:antialiased;font-size:1.5rem;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.app-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{margin:4px 4px 0;fill:#888;height:10px;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-c9121fecab.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png">
<link rel="icon" type="image/png" sizes="48x48" href="/static/images/favicons/nature/favicon-48x48-b52890008c.png">
<link rel="icon" type="image/png" sizes="32x32" href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="/static/images/favicons/nature/favicon-16x16-951651ab72.png">
<link rel="manifest" href="/static/manifest.json" crossorigin="use-credentials">
<link rel="mask-icon" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="/static/images/favicons/nature/favicon.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-e9d2061c95.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-1cbf3a8b56.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script><script id="js-position1" data-test="global-article-js" defer="" src="/static/js/global-article-es6-bundle-c8a573ca90.js"></script><script id="js-position2" data-test="shared-js" defer="" src="/static/js/shared-es6-bundle-e9d2061c95.js"></script><script id="js-position3" data-test="header-150-js" defer="" src="/static/js/header-150-es6-bundle-5bb959eaa1.js"></script><script id="js-position4" data-test="math-js" async="" src="/static/js/math-es6-bundle-23597ae350.js"></script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Drivers and determinants of strain dynamics following fecal microbiota transplantation","description":"Fecal microbiota transplantation (FMT) is a therapeutic intervention for inflammatory diseases of the gastrointestinal tract, but its clinical mode of action and subsequent microbiome dynamics remain poorly understood. Here we analyzed metagenomes from 316 FMTs, sampled pre and post intervention, for the treatment of ten different disease indications. We quantified strain-level dynamics of 1,089 microbial species, complemented by 47,548 newly constructed metagenome-assembled genomes. Donor strain colonization and recipient strain resilience were mostly independent of clinical outcomes, but accurately predictable using LASSO-regularized regression models that accounted for host, microbiome and procedural variables. Recipient factors and donor–recipient complementarity, encompassing entire microbial communities to individual strains, were the main determinants of strain population dynamics, providing insights into the underlying processes that shape the post-FMT gut microbiome. Applying an ecology-based framework to our findings indicated parameters that may inform the development of more effective, targeted microbiome therapies in the future, and suggested how patient stratification can be used to enhance donor microbiota colonization or the displacement of recipient microbes in clinical practice. Understanding the factors underlying colonization of donor microbes in recipients of fecal microbiota transplantation is a necessary first step to aid development of directed approaches that aim to couple colonization to clinical outcomes.","datePublished":"2022-09-15T00:00:00Z","dateModified":"2022-09-15T00:00:00Z","pageStart":"1902","pageEnd":"1912","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41591-022-01913-0","keywords":["Clinical microbiology","Metagenomics","Microbial ecology","Microbiome","Biomedicine","general","Cancer Research","Metabolic Diseases","Infectious Diseases","Molecular Medicine","Neurosciences"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig6_HTML.png"],"isPartOf":{"name":"Nature Medicine","issn":["1546-170X","1078-8956"],"volumeNumber":"28","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Thomas S. B. Schmidt","url":"http://orcid.org/0000-0001-8587-4177","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Simone S. Li","url":"http://orcid.org/0000-0002-0073-3656","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"},{"name":"Technical University of Denmark","address":{"name":"The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, Kongens Lyngby, Denmark","@type":"PostalAddress"},"@type":"Organization"},{"name":"The University of Queensland, School of Chemistry and Molecular Biosciences","address":{"name":"The University of Queensland, School of Chemistry and Molecular Biosciences, St Lucia, Australia","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Oleksandr M. Maistrenko","url":"http://orcid.org/0000-0003-1961-7548","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"},{"name":"Royal Netherlands Institute for Sea Research","address":{"name":"Department of Marine Microbiology & Biogeochemistry, Royal Netherlands Institute for Sea Research, ‘t Horntje, the Netherlands","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Wasiu Akanni","url":"http://orcid.org/0000-0002-2075-2387","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Luis Pedro Coelho","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"},{"name":"Key Laboratory of Computational Neuroscience and Brain-Inspired Intelligence (Fudan University), Ministry of Education","address":{"name":"Key Laboratory of Computational Neuroscience and Brain-Inspired Intelligence (Fudan University), Ministry of Education, Shanghai, China","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sibasish Dolai","affiliation":[{"name":"Centre for Digestive Diseases","address":{"name":"Centre for Digestive Diseases, Sydney, Australia","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Anthony Fullam","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Anna M. Glazek","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Rajna Hercog","url":"http://orcid.org/0000-0003-2587-9739","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Genomic Core Facility, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Hilde Herrema","affiliation":[{"name":"Amsterdam University Medical Centers","address":{"name":"Department of Experimental Vascular Medicine, Amsterdam University Medical Centers, Amsterdam, the Netherlands","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ferris Jung","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Genomic Core Facility, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Stefanie Kandels","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Askarbek Orakov","url":"http://orcid.org/0000-0001-6823-5269","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Roman Thielemann","url":"http://orcid.org/0000-0003-2582-6178","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Moritz von Stetten","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Thea Van Rossum","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Vladimir Benes","url":"http://orcid.org/0000-0002-0352-2547","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Genomic Core Facility, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Thomas J. Borody","url":"http://orcid.org/0000-0002-0519-4698","affiliation":[{"name":"Centre for Digestive Diseases","address":{"name":"Centre for Digestive Diseases, Sydney, Australia","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Willem M. de Vos","url":"http://orcid.org/0000-0002-0273-3166","affiliation":[{"name":"Wageningen University","address":{"name":"Laboratory of Microbiology, Wageningen University, Wageningen, the Netherlands","@type":"PostalAddress"},"@type":"Organization"},{"name":"University of Helsinki","address":{"name":"Human Microbiome Research Program, Faculty of Medicine, University of Helsinki, Helsinki, Finland","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Cyriel Y. Ponsioen","affiliation":[{"name":"Amsterdam University Medical Centers","address":{"name":"Department of Gastroenterology & Hepatology, Amsterdam University Medical Centers, Amsterdam, the Netherlands","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Max Nieuwdorp","affiliation":[{"name":"Amsterdam University Medical Centers","address":{"name":"Department of Experimental Vascular Medicine, Amsterdam University Medical Centers, Amsterdam, the Netherlands","@type":"PostalAddress"},"@type":"Organization"},{"name":"Amsterdam University Medical Centers","address":{"name":"Department of Vascular Medicine, Amsterdam University Medical Centers, Amsterdam, the Netherlands","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Peer Bork","url":"http://orcid.org/0000-0002-2627-833X","affiliation":[{"name":"European Molecular Biology Laboratory","address":{"name":"Structural & Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany","@type":"PostalAddress"},"@type":"Organization"},{"name":"Max Delbrück Center for Molecular Medicine","address":{"name":"Max Delbrück Center for Molecular Medicine, Berlin, Germany","@type":"PostalAddress"},"@type":"Organization"},{"name":"Yonsei University","address":{"name":"Yonsei Frontier Lab, Yonsei University, Seoul, South Korea","@type":"PostalAddress"},"@type":"Organization"},{"name":"Biocenter, University of Würzburg","address":{"name":"Department of Bioinformatics, Biocenter, University of Würzburg, Würzburg, Germany","@type":"PostalAddress"},"@type":"Organization"}],"email":"peer.bork@embl.org","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41591-022-01913-0">
    
    
    <meta name="journal_id" content="41591">
    <meta name="dc.title" content="Drivers and determinants of strain dynamics following fecal microbiota transplantation">
    <meta name="dc.source" content="Nature Medicine 2022 28:9">
    <meta name="dc.format" content="text/html">
    <meta name="dc.publisher" content="Nature Publishing Group">
    <meta name="dc.date" content="2022-09-15">
    <meta name="dc.type" content="OriginalPaper">
    <meta name="dc.language" content="En">
    <meta name="dc.copyright" content="2022 The Author(s)">
    <meta name="dc.rights" content="2022 The Author(s)">
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="dc.description" content="Fecal microbiota transplantation (FMT) is a therapeutic intervention for inflammatory diseases of the gastrointestinal tract, but its clinical mode of action and subsequent microbiome dynamics remain poorly understood. Here we analyzed metagenomes from 316 FMTs, sampled pre and post intervention, for the treatment of ten different disease indications. We quantified strain-level dynamics of 1,089 microbial species, complemented by 47,548 newly constructed metagenome-assembled genomes. Donor strain colonization and recipient strain resilience were mostly independent of clinical outcomes, but accurately predictable using LASSO-regularized regression models that accounted for host, microbiome and procedural variables. Recipient factors and donor–recipient complementarity, encompassing entire microbial communities to individual strains, were the main determinants of strain population dynamics, providing insights into the underlying processes that shape the post-FMT gut microbiome. Applying an ecology-based framework to our findings indicated parameters that may inform the development of more effective, targeted microbiome therapies in the future, and suggested how patient stratification can be used to enhance donor microbiota colonization or the displacement of recipient microbes in clinical practice. Understanding the factors underlying colonization of donor microbes in recipients of fecal microbiota transplantation is a necessary first step to aid development of directed approaches that aim to couple colonization to clinical outcomes.">
    <meta name="prism.issn" content="1546-170X">
    <meta name="prism.publicationName" content="Nature Medicine">
    <meta name="prism.publicationDate" content="2022-09-15">
    <meta name="prism.volume" content="28">
    <meta name="prism.number" content="9">
    <meta name="prism.section" content="OriginalPaper">
    <meta name="prism.startingPage" content="1902">
    <meta name="prism.endingPage" content="1912">
    <meta name="prism.copyright" content="2022 The Author(s)">
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="prism.url" content="https://www.nature.com/articles/s41591-022-01913-0">
    <meta name="prism.doi" content="doi:10.1038/s41591-022-01913-0">
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41591-022-01913-0.pdf">
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41591-022-01913-0">
    <meta name="citation_journal_title" content="Nature Medicine">
    <meta name="citation_journal_abbrev" content="Nat Med">
    <meta name="citation_publisher" content="Nature Publishing Group">
    <meta name="citation_issn" content="1546-170X">
    <meta name="citation_title" content="Drivers and determinants of strain dynamics following fecal microbiota transplantation">
    <meta name="citation_volume" content="28">
    <meta name="citation_issue" content="9">
    <meta name="citation_publication_date" content="2022/09">
    <meta name="citation_online_date" content="2022/09/15">
    <meta name="citation_firstpage" content="1902">
    <meta name="citation_lastpage" content="1912">
    <meta name="citation_article_type" content="Article">
    <meta name="citation_fulltext_world_readable" content="">
    <meta name="citation_language" content="en">
    <meta name="dc.identifier" content="doi:10.1038/s41591-022-01913-0">
    <meta name="DOI" content="10.1038/s41591-022-01913-0">
    <meta name="size" content="272615">
    <meta name="citation_doi" content="10.1038/s41591-022-01913-0">
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41591-022-01913-0&amp;api_key=">
    <meta name="description" content="Fecal microbiota transplantation (FMT) is a therapeutic intervention for inflammatory diseases of the gastrointestinal tract, but its clinical mode of action and subsequent microbiome dynamics remain poorly understood. Here we analyzed metagenomes from 316 FMTs, sampled pre and post intervention, for the treatment of ten different disease indications. We quantified strain-level dynamics of 1,089 microbial species, complemented by 47,548 newly constructed metagenome-assembled genomes. Donor strain colonization and recipient strain resilience were mostly independent of clinical outcomes, but accurately predictable using LASSO-regularized regression models that accounted for host, microbiome and procedural variables. Recipient factors and donor–recipient complementarity, encompassing entire microbial communities to individual strains, were the main determinants of strain population dynamics, providing insights into the underlying processes that shape the post-FMT gut microbiome. Applying an ecology-based framework to our findings indicated parameters that may inform the development of more effective, targeted microbiome therapies in the future, and suggested how patient stratification can be used to enhance donor microbiota colonization or the displacement of recipient microbes in clinical practice. Understanding the factors underlying colonization of donor microbes in recipients of fecal microbiota transplantation is a necessary first step to aid development of directed approaches that aim to couple colonization to clinical outcomes.">
    <meta name="dc.creator" content="Schmidt, Thomas S. B.">
    <meta name="dc.creator" content="Li, Simone S.">
    <meta name="dc.creator" content="Maistrenko, Oleksandr M.">
    <meta name="dc.creator" content="Akanni, Wasiu">
    <meta name="dc.creator" content="Coelho, Luis Pedro">
    <meta name="dc.creator" content="Dolai, Sibasish">
    <meta name="dc.creator" content="Fullam, Anthony">
    <meta name="dc.creator" content="Glazek, Anna M.">
    <meta name="dc.creator" content="Hercog, Rajna">
    <meta name="dc.creator" content="Herrema, Hilde">
    <meta name="dc.creator" content="Jung, Ferris">
    <meta name="dc.creator" content="Kandels, Stefanie">
    <meta name="dc.creator" content="Orakov, Askarbek">
    <meta name="dc.creator" content="Thielemann, Roman">
    <meta name="dc.creator" content="von Stetten, Moritz">
    <meta name="dc.creator" content="Van Rossum, Thea">
    <meta name="dc.creator" content="Benes, Vladimir">
    <meta name="dc.creator" content="Borody, Thomas J.">
    <meta name="dc.creator" content="de Vos, Willem M.">
    <meta name="dc.creator" content="Ponsioen, Cyriel Y.">
    <meta name="dc.creator" content="Nieuwdorp, Max">
    <meta name="dc.creator" content="Bork, Peer">
    <meta name="dc.subject" content="Clinical microbiology">
    <meta name="dc.subject" content="Metagenomics">
    <meta name="dc.subject" content="Microbial ecology">
    <meta name="dc.subject" content="Microbiome">
    <meta name="citation_reference" content="citation_journal_title=J. Clin. Gastroenterol.; citation_title=Bacteriotherapy using fecal flora: toying with human motions; citation_author=TJ Borody; citation_volume=38; citation_publication_date=2004; citation_pages=475-483; citation_doi=10.1097/01.mcg.0000128988.13808.dc; citation_id=CR1">
    <meta name="citation_reference" content="citation_journal_title=World J. Gastroenterol.; citation_title=Fecal microbiota transplantation as novel therapy in gastroenterology: a systematic review; citation_author=NG Rossen; citation_volume=21; citation_publication_date=2015; citation_pages=5359-5371; citation_doi=10.3748/wjg.v21.i17.5359; citation_id=CR2">
    <meta name="citation_reference" content="citation_journal_title=Cell Metab.; citation_title=Fecal microbiota transplantation in human metabolic diseases: from a murky past to a bright future?; citation_author=NMJ Hanssen, WM Vos, M Nieuwdorp; citation_volume=33; citation_publication_date=2021; citation_pages=1098-1110; citation_doi=10.1016/j.cmet.2021.05.005; citation_id=CR3">
    <meta name="citation_reference" content="citation_journal_title=Clin. Infect. Dis.; citation_title=Systematic review of intestinal microbiota transplantation (fecal bacteriotherapy) for recurrent Clostridium difficile infection; citation_author=E Gough, H Shaikh, AR Manges; citation_volume=53; citation_publication_date=2011; citation_pages=994-1002; citation_doi=10.1093/cid/cir632; citation_id=CR4">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Duodenal infusion of donor feces for recurrent Clostridium difficile; citation_author=E Nood; citation_volume=368; citation_publication_date=2013; citation_pages=407-415; citation_doi=10.1056/NEJMoa1205037; citation_id=CR5">
    <meta name="citation_reference" content="citation_journal_title=Inflamm. Bowel Dis.; citation_title=Systematic review and meta-analysis fecal microbiota transplantation for treatment of active ulcerative colitis; citation_author=N Narula; citation_volume=23; citation_publication_date=2017; citation_pages=1702-1709; citation_doi=10.1097/MIB.0000000000001228; citation_id=CR6">
    <meta name="citation_reference" content="citation_journal_title=Curr. Opin. Pharmacol.; citation_title=The role of fecal microbiota transplantation in the treatment of inflammatory bowel disease; citation_author=C Haifer, RW Leong, S Paramsothy; citation_volume=55; citation_publication_date=2020; citation_pages=8-16; citation_doi=10.1016/j.coph.2020.08.009; citation_id=CR7">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Post-antibiotic gut mucosal microbiome reconstitution is impaired by probiotics and improved by autologous FMT; citation_author=J Suez; citation_volume=174; citation_publication_date=2018; citation_pages=1406-1423; citation_doi=10.1016/j.cell.2018.08.047; citation_id=CR8">
    <meta name="citation_reference" content="Baruch, E. N. et al. Fecal microbiota transplant promotes response in immunotherapy-refractory melanoma patients. Science 371, 602–609 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Fecal microbiota transplant overcomes resistance to anti-PD-1 therapy in melanoma patients; citation_author=D Davar; citation_volume=371; citation_publication_date=2021; citation_pages=595-602; citation_doi=10.1126/science.abf3363; citation_id=CR10">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Therapeutic fecal microbiota transplantation controls intestinal inflammation through IL10 secretion by immune cells; citation_author=C Burrello; citation_volume=9; citation_publication_date=2018; citation_doi=10.1038/s41467-018-07359-8; citation_id=CR11">
    <meta name="citation_reference" content="citation_journal_title=Anaerobe; citation_title=Restoration of short chain fatty acid and bile acid metabolism following fecal microbiota transplantation in patients with recurrent Clostridium difficile infection; citation_author=AM Seekatz; citation_volume=53; citation_publication_date=2018; citation_pages=64-73; citation_doi=10.1016/j.anaerobe.2018.04.001; citation_id=CR12">
    <meta name="citation_reference" content="citation_journal_title=Gastroenterology; citation_title=Bacteriophage transfer during fecal microbiota transplantation is associated with treatment response in Clostridium difficile infection; citation_author=T Zuo; citation_volume=152; citation_publication_date=2017; citation_pages=S140-S141; citation_doi=10.1016/S0016-5085(17)30798-9; citation_id=CR13">
    <meta name="citation_reference" content="citation_journal_title=Gut Microbes; citation_title=Gut bacteriophage dynamics during fecal microbial transplantation in subjects with metabolic syndrome; citation_author=P Manrique; citation_volume=13; citation_publication_date=2021; citation_pages=1-15; citation_doi=10.1080/19490976.2021.1897217; citation_id=CR14">
    <meta name="citation_reference" content="citation_journal_title=Front. Cell. Infect. Microbiol.; citation_title=The super-donor phenomenon in fecal microbiota transplantation; citation_author=BC Wilson, T Vatanen, WS Cutfield, JM O’Sullivan; citation_volume=9; citation_publication_date=2019; citation_pages=2; citation_doi=10.3389/fcimb.2019.00002; citation_id=CR15">
    <meta name="citation_reference" content="Duvallet, C. et al. Framework for rational donor selection in fecal microbiota transplant clinical trials. PloS ONE 14, e0222881 (2019).">
    <meta name="citation_reference" content="citation_journal_title=J. Crohns Colitis; citation_title=Re-evaluating the evidence for fecal microbiota transplantation “super-donors” in inflammatory bowel disease; citation_author=SW Olesen, Y Gerardin; citation_volume=15; citation_publication_date=2021; citation_pages=453-461; citation_doi=10.1093/ecco-jcc/jjaa170; citation_id=CR17">
    <meta name="citation_reference" content="citation_journal_title=Cell Metab.; citation_title=Improvement of insulin sensitivity after lean donor feces in metabolic syndrome is driven by baseline intestinal imcrobiota composition; citation_author=RS Kootte; citation_volume=26; citation_publication_date=2017; citation_pages=611-619; citation_doi=10.1016/j.cmet.2017.09.008; citation_id=CR18">
    <meta name="citation_reference" content="Danne, C., Rolhion, N. &amp; Sokol, H. Recipient factors in fecal microbiota transplantation: one stool does not fit all. Nat. Rev. Gastroenterol. Hepatol. 18, 503–513 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Gastroenterology; citation_title=Functional restoration of bacteriomes and viromes by fecal microbiota transplantation; citation_author=K Fujimoto; citation_volume=160; citation_publication_date=2021; citation_pages=2089-2102; citation_doi=10.1053/j.gastro.2021.02.013; citation_id=CR20">
    <meta name="citation_reference" content="citation_journal_title=United European Gastroenterol. J.; citation_title=The impact of technical and clinical factors on fecal microbiota transfer outcomes for the treatment of recurrent Clostridioides difficile infections in Germany; citation_author=R Peri; citation_volume=7; citation_publication_date=2019; citation_pages=716-722; citation_doi=10.1177/2050640619839918; citation_id=CR21">
    <meta name="citation_reference" content="citation_journal_title=Microbiome; citation_title=Long-term colonisation with donor bacteriophages following successful fecal microbial transplantation; citation_author=LA Draper; citation_volume=6; citation_publication_date=2018; citation_doi=10.1186/s40168-018-0598-x; citation_id=CR22">
    <meta name="citation_reference" content="Leonardi, I. et al. Fungal trans-kingdom dynamics linked to responsiveness to fecal microbiota transplantation (FMT) therapy in ulcerative colitis. Cell Host Microbe 27, 823–829 (2020).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Gut fungal dysbiosis correlates with reduced efficacy of fecal microbiota transplantation in Clostridium difficile infection; citation_author=T Zuo; citation_volume=9; citation_publication_date=2018; citation_doi=10.1038/s41467-018-06103-6; citation_id=CR24">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Durable coexistence of donor and recipient strains after fecal microbiota transplantation; citation_author=SS Li; citation_volume=352; citation_publication_date=2016; citation_pages=586-589; citation_doi=10.1126/science.aad8852; citation_id=CR25">
    <meta name="citation_reference" content="citation_journal_title=Cell Host Microbe; citation_title=Strain tracking reveals the determinants of bacterial engraftment in the human gut following fecal microbiota transplantation; citation_author=CS Smillie; citation_volume=23; citation_publication_date=2018; citation_pages=229-240; citation_doi=10.1016/j.chom.2018.01.003; citation_id=CR26">
    <meta name="citation_reference" content="citation_journal_title=Microbiome; citation_title=Metagenomic strain detection with SameStr: identification of a persisting core gut microbiota transferable by fecal transplantation; citation_author=D Podlesny; citation_volume=10; citation_publication_date=2022; citation_doi=10.1186/s40168-022-01251-w; citation_id=CR27">
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes; citation_author=V Aggarwala; citation_volume=6; citation_publication_date=2021; citation_pages=1309-1318; citation_doi=10.1038/s41564-021-00966-0; citation_id=CR28">
    <meta name="citation_reference" content="citation_journal_title=Microbiome; citation_title=Tracking microbial colonization in fecal microbiota transplantation experiments via genome-resolved metagenomics; citation_author=STM Lee; citation_volume=5; citation_publication_date=2017; citation_doi=10.1186/s40168-017-0270-x; citation_id=CR29">
    <meta name="citation_reference" content="citation_journal_title=Microbiome; citation_title=Strain engraftment competition and functional augmentation in a multi-donor fecal microbiota transplantation trial for obesity; citation_author=BC Wilson; citation_volume=9; citation_publication_date=2021; citation_doi=10.1186/s40168-021-01060-7; citation_id=CR30">
    <meta name="citation_reference" content="citation_journal_title=Gut; citation_title=Microbiota engraftment after fecal microbiota transplantation in obese subjects with type 2 diabetes: a 24-week, double-blind, randomised controlled trial; citation_author=SC Ng; citation_volume=71; citation_publication_date=2022; citation_pages=716-723; citation_doi=10.1136/gutjnl-2020-323617; citation_id=CR31">
    <meta name="citation_reference" content="Watson, A. R., Fuessel, J., Veseli, I. &amp; DeLongchamp, J. Z. Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut. Preprint at 
                  https://doi.org/10.1101/2021.03.02.433653
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Curr. Opin. Biotechnol.; citation_title=To engraft or not to engraft: an ecological framework for gut microbiome modulation with live microbes; citation_author=J Walter, MX Maldonado-Gómez, I Martínez; citation_volume=49; citation_publication_date=2018; citation_pages=129-139; citation_doi=10.1016/j.copbio.2017.08.008; citation_id=CR33">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=The human gut microbiome: from association to modulation; citation_author=TSB Schmidt, J Raes, P Bork; citation_volume=172; citation_publication_date=2018; citation_pages=1198-1215; citation_doi=10.1016/j.cell.2018.02.044; citation_id=CR34">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=An ecological framework to understand the efficacy of fecal microbiota transplantation; citation_author=Y Xiao, MT Angulo, S Lao, ST Weiss, Y-Y Liu; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-17180-x; citation_id=CR35">
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Long-term taxonomic and functional divergence from donor bacterial strains following fecal microbiota transplantation in immunocompromised patients; citation_author=EL Moss; citation_volume=12; citation_publication_date=2017; citation_pages=e0182585; citation_doi=10.1371/journal.pone.0182585; citation_id=CR36">
    <meta name="citation_reference" content="citation_journal_title=BMC Res. Notes; citation_title=Fecal microbiota transplantation against intestinal colonization by extended spectrum beta-lactamase producing Enterobacteriaceae: a proof of principle study; citation_author=R Singh; citation_volume=11; citation_publication_date=2018; citation_doi=10.1186/s13104-018-3293-x; citation_id=CR37">
    <meta name="citation_reference" content="Bar-Yoseph, H. et al. Oral capsulized fecal microbiota transplantation for eradication of carbapenemase-producing Enterobacteriaceae colonization with a metagenomic perspective. Clin. Infect. Dis. 73, e166–e175 (2020).">
    <meta name="citation_reference" content="Leo, S. et al. Metagenomic characterization of gut microbiota of carriers of extended-spectrum beta-lactamase or carbapenemase-producing Enterobacteriaceae following treatment with oral antibiotics and fecal microbiota transplantation: results from a multicenter randomized trial. Microorganisms 8, 941 (2020).">
    <meta name="citation_reference" content="citation_journal_title=Front. Microbiol.; citation_title=Effect of fecal microbiota transplantation combined with Mediterranean diet on insulin sensitivity in subjects with metabolic syndrome; citation_author=AM Koopen; citation_volume=12; citation_publication_date=2021; citation_pages=662159; citation_doi=10.3389/fmicb.2021.662159; citation_id=CR40">
    <meta name="citation_reference" content="citation_journal_title=Gastroenterology; citation_title=Findings from a randomized controlled trial of fecal transplantation for patients with ulcerative colitis; citation_author=NG Rossen; citation_volume=149; citation_publication_date=2015; citation_pages=110-118; citation_doi=10.1053/j.gastro.2015.03.045; citation_id=CR41">
    <meta name="citation_reference" content="Nusbaum, D. J. et al. Gut microbial and metabolomic profiles after fecal microbiota transplantation in pediatric ulcerative colitis patients. FEMS Microbiol. Ecol. 94, fiy133 (2018).">
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=Low level engraftment and improvement following a single colonoscopic administration of fecal microbiota to patients with ulcerative colitis; citation_author=CJ Damman; citation_volume=10; citation_publication_date=2015; citation_pages=e0133925; citation_doi=10.1371/journal.pone.0133925; citation_id=CR43">
    <meta name="citation_reference" content="citation_journal_title=Gut Microbes; citation_title=Effects of fecal microbiota transplantation in subjects with irritable bowel syndrome are mirrored by changes in gut microbiome; citation_author=R Goll; citation_volume=12; citation_publication_date=2020; citation_pages=1794263; citation_doi=10.1080/19490976.2020.1794263; citation_id=CR44">
    <meta name="citation_reference" content="citation_journal_title=Inflamm. Bowel Dis.; citation_title=Increased intestinal microbial diversity following fecal microbiota transplant for active Crohn’s disease; citation_author=BP Vaughn; citation_volume=22; citation_publication_date=2016; citation_pages=2182-2190; citation_doi=10.1097/MIB.0000000000000893; citation_id=CR45">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Fecal microbiota transplantation for the treatment of diarrhoea induced by tyrosine-kinase inhibitors in patients with metastatic renal cell carcinoma; citation_author=G Ianiro; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-18127-y; citation_id=CR46">
    <meta name="citation_reference" content="citation_journal_title=Front. Psychiatry; citation_title=The efficacy of fecal microbiota transplantation for children with Tourette syndrome: a preliminary study; citation_author=H-J Zhao; citation_volume=11; citation_publication_date=2020; citation_pages=554441; citation_doi=10.3389/fpsyt.2020.554441; citation_id=CR47">
    <meta name="citation_reference" content="citation_journal_title=BMC Microbiol.; citation_title=Long-term impact of fecal transplantation in healthy volunteers; citation_author=OV Goloshchapov; citation_volume=19; citation_publication_date=2019; citation_pages=312; citation_doi=10.1186/s12866-019-1689-y; citation_id=CR48">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Deciphering functional redundancy in the human microbiome; citation_author=L Tian; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-19940-1; citation_id=CR49">
    <meta name="citation_reference" content="citation_journal_title=Cell Host Microbe; citation_title=Dispersal strategies shape persistence and evolution of human gut bacteria; citation_author=F Hildebrand; citation_volume=29; citation_publication_date=2021; citation_pages=1167-1176; citation_doi=10.1016/j.chom.2021.05.008; citation_id=CR50">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Tipping elements in the human intestinal ecosystem; citation_author=L Lahti, J Salojärvi, A Salonen, M Scheffer, WM Vos; citation_volume=5; citation_publication_date=2014; citation_doi=10.1038/ncomms5344; citation_id=CR51">
    <meta name="citation_reference" content="Scheffer, M., Carpenter, S. R., Dakos, V. &amp; van Nes, E. H. Generic indicators of ecological resilience: inferring the chance of a critical transition. Annu. Rev. Ecol. Evol. Syst. 46, 145–167 (2015).">
    <meta name="citation_reference" content="citation_journal_title=ISME J.; citation_title=Multi-stability and the origin of microbial community types; citation_author=D Gonze, L Lahti, J Raes, K Faust; citation_volume=11; citation_publication_date=2017; citation_pages=2159–2166; citation_doi=10.1038/ismej.2017.60; citation_id=CR53">
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Enterotypes in the landscape of gut microbial community composition; citation_author=PI Costea; citation_volume=3; citation_publication_date=2018; citation_pages=8-16; citation_doi=10.1038/s41564-017-0072-8; citation_id=CR54">
    <meta name="citation_reference" content="Debray, R. et al. Priority effects in microbiome assembly. Nat. Rev. Microbiol. 20, 109–121 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Stress and stability: applying the Anna Karenina principle to animal microbiomes; citation_author=JR Zaneveld, R McMinds, RV Thurber; citation_volume=2; citation_publication_date=2017; citation_pages=17121; citation_doi=10.1038/nmicrobiol.2017.121; citation_id=CR56">
    <meta name="citation_reference" content="citation_journal_title=Transl. Res.; citation_title=Autologous fecal microbiota transplantation for the treatment of inflammatory bowel disease; citation_author=AR Basson, Y Zhou, B Seo, A Rodriguez-Palacios, F Cominelli; citation_volume=226; citation_publication_date=2020; citation_pages=1-11; citation_doi=10.1016/j.trsl.2020.05.008; citation_id=CR57">
    <meta name="citation_reference" content="citation_journal_title=Gut; citation_title=Fecal microbiota transplantation halts progression of human new-onset type 1 diabetes in a randomised controlled trial; citation_author=P Groot; citation_volume=70; citation_publication_date=2021; citation_pages=92-105; citation_doi=10.1136/gutjnl-2020-322630; citation_id=CR58">
    <meta name="citation_reference" content="citation_journal_title=Gastroenterology; citation_title=Efficacy of sterile fecal filtrate transfer for treating patients with Clostridium difficile infection; citation_author=SJ Ott; citation_volume=152; citation_publication_date=2017; citation_pages=799-811; citation_doi=10.1053/j.gastro.2016.11.010; citation_id=CR59">
    <meta name="citation_reference" content="citation_journal_title=PLoS Biol.; citation_title=Fecal transplants: what is being transferred?; citation_author=DP Bojanova, SR Bordenstein; citation_volume=14; citation_publication_date=2016; citation_pages=e1002503; citation_doi=10.1371/journal.pbio.1002503; citation_id=CR60">
    <meta name="citation_reference" content="citation_journal_title=ISME J.; citation_title=Microbial shifts and signatures of long-term remission in ulcerative colitis after fecal microbiota transplantation; citation_author=S Fuentes; citation_volume=11; citation_publication_date=2017; citation_pages=1877-1889; citation_doi=10.1038/ismej.2017.44; citation_id=CR61">
    <meta name="citation_reference" content="citation_journal_title=Microbiome; citation_title=NG-meta-profiler: fast processing of metagenomes using NGLess, a domain-specific language; citation_author=LP Coelho; citation_volume=7; citation_publication_date=2019; citation_doi=10.1186/s40168-019-0684-8; citation_id=CR62">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Microbial abundance, activity and population genomic profiling with mOTUs2; citation_author=A Milanese; citation_volume=10; citation_publication_date=2019; citation_doi=10.1038/s41467-019-08844-4; citation_id=CR63">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Towards the biogeography of prokaryotic genes; citation_author=LP Coelho; citation_volume=601; citation_publication_date=2022; citation_pages=252-256; citation_doi=10.1038/s41586-021-04233-4; citation_id=CR64">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=EggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses; citation_author=J Huerta-Cepas; citation_volume=47; citation_publication_date=2019; citation_pages=D309-D314; citation_doi=10.1093/nar/gky1085; citation_id=CR65">
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Species–function relationships shape ecological properties of the human gut microbiome; citation_author=S Vieira-Silva; citation_volume=1; citation_publication_date=2016; citation_pages=16088; citation_doi=10.1038/nmicrobiol.2016.88; citation_id=CR66">
    <meta name="citation_reference" content="citation_journal_title=ISME J.; citation_title=Towards biome-specific analysis of meta-omics data; citation_author=Y Darzi, G Falony, S Vieira-Silva, J Raes; citation_volume=10; citation_publication_date=2016; citation_pages=1025-1028; citation_doi=10.1038/ismej.2015.188; citation_id=CR67">
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=MetaSPAdes: a new versatile metagenomic assembler; citation_author=S Nurk, D Meleshko, A Korobeynikov, PA Pevzner; citation_volume=27; citation_publication_date=2017; citation_pages=824-834; citation_doi=10.1101/gr.213959.116; citation_id=CR68">
    <meta name="citation_reference" content="Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Preprint at 
                  https://doi.org/10.48550/arXiv.1303.3997
                  
                 (2013).">
    <meta name="citation_reference" content="citation_journal_title=PeerJ.; citation_title=MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies; citation_author=DD Kang; citation_volume=7; citation_publication_date=2019; citation_pages=e7359; citation_doi=10.7717/peerj.7359; citation_id=CR70">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph; citation_author=D Li, C-M Liu, R Luo, K Sadakane, T-W Lam; citation_volume=31; citation_publication_date=2015; citation_pages=1674-1676; citation_doi=10.1093/bioinformatics/btv033; citation_id=CR71">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Binning metagenomic contigs by coverage and composition; citation_author=J Alneberg; citation_volume=11; citation_publication_date=2014; citation_pages=1144-1146; citation_doi=10.1038/nmeth.3103; citation_id=CR72">
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=Recovery of genomes from metagenomes via a dereplication, aggregation and scoring strategy; citation_author=CMK Sieber; citation_volume=3; citation_publication_date=2018; citation_pages=836-843; citation_doi=10.1038/s41564-018-0171-1; citation_id=CR73">
    <meta name="citation_reference" content="citation_journal_title=Microbiome; citation_title=MetaWRAP—a flexible pipeline for genome-resolved metagenomic data analysis; citation_author=GV Uritskiy, J DiRuggiero, J Taylor; citation_volume=6; citation_publication_date=2018; citation_pages=158; citation_doi=10.1186/s40168-018-0541-1; citation_id=CR74">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=ProGenomes: a resource for consistent functional and taxonomic annotations of prokaryotic genomes; citation_author=DR Mende; citation_volume=45; citation_publication_date=2017; citation_pages=D529-D534; citation_doi=10.1093/nar/gkw989; citation_id=CR75">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=ProGenomes2: an improved database for accurate and consistent habitat, taxonomic and functional annotations of prokaryotic genomes; citation_author=DR Mende; citation_volume=48; citation_publication_date=2020; citation_pages=D621-D625; citation_id=CR76">
    <meta name="citation_reference" content="citation_journal_title=Genome Res.; citation_title=CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes; citation_author=DH Parks, M Imelfort, CT Skennerton, P Hugenholtz, GW Tyson; citation_volume=25; citation_publication_date=2015; citation_pages=1043-1055; citation_doi=10.1101/gr.186072.114; citation_id=CR77">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=GUNC: detection of chimerism and contamination in prokaryotic genomes; citation_author=A Orakov; citation_volume=22; citation_publication_date=2021; citation_doi=10.1186/s13059-021-02393-0; citation_id=CR78">
    <meta name="citation_reference" content="Chaumeil, P.-A., Mussig, A. J., Hugenholtz, P. &amp; Parks, D. H. GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database. Bioinformatics 36, 1925–1927 (2019).">
    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=Prodigal: prokaryotic gene recognition and translation initiation site identification; citation_author=D Hyatt; citation_volume=11; citation_publication_date=2010; citation_pages=119; citation_doi=10.1186/1471-2105-11-119; citation_id=CR80">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Prokka: rapid prokaryotic genome annotation; citation_author=T Seemann; citation_volume=30; citation_publication_date=2014; citation_pages=2068-2069; citation_doi=10.1093/bioinformatics/btu153; citation_id=CR81">
    <meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.; citation_title=Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper; citation_author=J Huerta-Cepas; citation_volume=34; citation_publication_date=2017; citation_pages=2115-2122; citation_doi=10.1093/molbev/msx148; citation_id=CR82">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=CARD 2020: antibiotic resistome surveillance with the comprehensive antibiotic resistance database; citation_author=BP Alcock; citation_volume=48; citation_publication_date=2020; citation_pages=D517-D525; citation_id=CR83">
    <meta name="citation_reference" content="citation_journal_title=ISME J.; citation_title=Improved annotation of antibiotic resistance determinants reveals microbial resistomes cluster by ecology; citation_author=MK Gibson, KJ Forsberg, G Dantas; citation_volume=9; citation_publication_date=2015; citation_pages=207-216; citation_doi=10.1038/ismej.2014.106; citation_id=CR84">
    <meta name="citation_reference" content="citation_journal_title=Nat. Publ. Group; citation_title=Accurate and universal delineation of prokaryotic species; citation_author=DR Mende, S Sunagawa, G Zeller, P Bork; citation_volume=10; citation_publication_date=2013; citation_pages=881-884; citation_id=CR85">
    <meta name="citation_reference" content="Schmidt, T. S. B. et al. Drivers and determinants of strain dynamics following fecal microbiota transplantation. Zenodo 
                  https://doi.org/10.5281/ZENODO.5534163
                  
                 (2021).">
    <meta name="citation_reference" content="Rodrigues, J. F. M., Schmidt, S. B. T., Tackmann, J. &amp; von Mering, C. MAPseq: highly efficient k-mer search with confidence estimates, for rRNA sequence analysis. Bioinformatics 33, 3808–3810 (2017).">
    <meta name="citation_reference" content="citation_journal_title=ISME J.; citation_title=DRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication; citation_author=MR Olm, CT Brown, B Brooks, JF Banfield; citation_volume=11; citation_publication_date=2017; citation_pages=2864-2868; citation_doi=10.1038/ismej.2017.126; citation_id=CR88">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=Mash: fast genome and metagenome distance estimation using MinHash; citation_author=BD Ondov; citation_volume=17; citation_publication_date=2016; citation_doi=10.1186/s13059-016-0997-x; citation_id=CR89">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=High throughput ANI analysis of 90 K prokaryotic genomes reveals clear species boundaries; citation_author=C Jain, LM Rodriguez-R, AM Phillippy, KT Konstantinidis, S Aluru; citation_volume=9; citation_publication_date=2018; citation_doi=10.1038/s41467-018-07641-9; citation_id=CR90">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Roary: rapid large-scale prokaryote pan genome analysis; citation_author=AJ Page; citation_volume=31; citation_publication_date=2015; citation_pages=3691-3693; citation_doi=10.1093/bioinformatics/btv421; citation_id=CR91">
    <meta name="citation_reference" content="citation_journal_title=BMC Bioinformatics; citation_title=MUSCLE: a multiple sequence alignment method with reduced time and space complexity; citation_author=RC Edgar; citation_volume=5; citation_publication_date=2004; citation_pages=113; citation_doi=10.1186/1471-2105-5-113; citation_id=CR92">
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=FastTree 2–approximately maximum-likelihood trees for large alignments; citation_author=MN Price, PS Dehal, AP Arkin; citation_volume=5; citation_publication_date=2010; citation_pages=e9490; citation_doi=10.1371/journal.pone.0009490; citation_id=CR93">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=The Sequence Alignment/Map format and SAMtools; citation_author=H Li; citation_volume=25; citation_publication_date=2009; citation_pages=2078-2079; citation_doi=10.1093/bioinformatics/btp352; citation_id=CR94">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=BEDTools: a flexible suite of utilities for comparing genomic features; citation_author=AR Quinlan, IM Hall; citation_volume=26; citation_publication_date=2010; citation_pages=841-842; citation_doi=10.1093/bioinformatics/btq033; citation_id=CR95">
    <meta name="citation_reference" content="citation_journal_title=PLoS ONE; citation_title=MetaSNV: a tool for metagenomic strain level analysis; citation_author=PI Costea; citation_volume=12; citation_publication_date=2017; citation_pages=e0182392; citation_doi=10.1371/journal.pone.0182392; citation_id=CR96">
    <meta name="citation_reference" content="Schmidt, T. S. B. et al. Extensive transmission of microbes along the gastrointestinal tract. eLife 8, e42693 (2019).">
    <meta name="citation_reference" content="citation_journal_title=J. Stat. Softw.; citation_title=Regularization paths for generalized linear models via coordinate descent; citation_author=J Friedman, T Hastie, R Tibshirani; citation_volume=33; citation_publication_date=2010; citation_pages=1-22; citation_doi=10.18637/jss.v033.i01; citation_id=CR98">
    <meta name="citation_reference" content="Schmidt, T. S. B. et al. Analysis data, “Drivers and Determinants of Strain Dynamics Following Fecal Microbiota Transplantation”. Zenodo 
                  https://doi.org/10.5281/ZENODO.5534163
                  
                 (2021).">
    <meta name="citation_author" content="Schmidt, Thomas S. B.">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Li, Simone S.">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author_institution" content="The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, Kongens Lyngby, Denmark">
    <meta name="citation_author_institution" content="The University of Queensland, School of Chemistry and Molecular Biosciences, St Lucia, Australia">
    <meta name="citation_author" content="Maistrenko, Oleksandr M.">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author_institution" content="Department of Marine Microbiology &amp; Biogeochemistry, Royal Netherlands Institute for Sea Research, ‘t Horntje, the Netherlands">
    <meta name="citation_author" content="Akanni, Wasiu">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Coelho, Luis Pedro">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author_institution" content="Key Laboratory of Computational Neuroscience and Brain-Inspired Intelligence (Fudan University), Ministry of Education, Shanghai, China">
    <meta name="citation_author" content="Dolai, Sibasish">
    <meta name="citation_author_institution" content="Centre for Digestive Diseases, Sydney, Australia">
    <meta name="citation_author" content="Fullam, Anthony">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Glazek, Anna M.">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Hercog, Rajna">
    <meta name="citation_author_institution" content="Genomic Core Facility, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Herrema, Hilde">
    <meta name="citation_author_institution" content="Department of Experimental Vascular Medicine, Amsterdam University Medical Centers, Amsterdam, the Netherlands">
    <meta name="citation_author" content="Jung, Ferris">
    <meta name="citation_author_institution" content="Genomic Core Facility, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Kandels, Stefanie">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Orakov, Askarbek">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Thielemann, Roman">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="von Stetten, Moritz">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Van Rossum, Thea">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Benes, Vladimir">
    <meta name="citation_author_institution" content="Genomic Core Facility, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author" content="Borody, Thomas J.">
    <meta name="citation_author_institution" content="Centre for Digestive Diseases, Sydney, Australia">
    <meta name="citation_author" content="de Vos, Willem M.">
    <meta name="citation_author_institution" content="Laboratory of Microbiology, Wageningen University, Wageningen, the Netherlands">
    <meta name="citation_author_institution" content="Human Microbiome Research Program, Faculty of Medicine, University of Helsinki, Helsinki, Finland">
    <meta name="citation_author" content="Ponsioen, Cyriel Y.">
    <meta name="citation_author_institution" content="Department of Gastroenterology &amp; Hepatology, Amsterdam University Medical Centers, Amsterdam, the Netherlands">
    <meta name="citation_author" content="Nieuwdorp, Max">
    <meta name="citation_author_institution" content="Department of Experimental Vascular Medicine, Amsterdam University Medical Centers, Amsterdam, the Netherlands">
    <meta name="citation_author_institution" content="Department of Vascular Medicine, Amsterdam University Medical Centers, Amsterdam, the Netherlands">
    <meta name="citation_author" content="Bork, Peer">
    <meta name="citation_author_institution" content="Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany">
    <meta name="citation_author_institution" content="Max Delbrück Center for Molecular Medicine, Berlin, Germany">
    <meta name="citation_author_institution" content="Yonsei Frontier Lab, Yonsei University, Seoul, South Korea">
    <meta name="citation_author_institution" content="Department of Bioinformatics, Biocenter, University of Würzburg, Würzburg, Germany">
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">
    <meta name="twitter:site" content="@naturemedicine">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image:alt" content="Content cover image">
    <meta name="twitter:title" content="Drivers and determinants of strain dynamics following fecal microbiota transplantation">
    <meta name="twitter:description" content="Nature Medicine - Understanding the factors underlying colonization of donor microbes in recipients of fecal microbiota transplantation is a necessary first step to aid development of directed...">
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig1_HTML.png">
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41591-022-01913-0">
    <meta property="og:type" content="article">
    <meta property="og:site_name" content="Nature">
    <meta property="og:title" content="Drivers and determinants of strain dynamics following fecal microbiota transplantation - Nature Medicine">
    <meta property="og:description" content="Understanding the factors underlying colonization of donor microbes in recipients of fecal microbiota transplantation is a necessary first step to aid development of directed approaches that aim to couple colonization to clinical outcomes.">
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig1_HTML.png">
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
<style type="text/css">.cc-banner{background-color:#01324b;border:none!important;bottom:0;box-sizing:border-box;color:#fff!important;left:0;line-height:1.3;margin:auto 0 0;max-width:100%;outline:0;overflow:visible;padding:0;position:fixed;right:0;width:100%;z-index:99999}.cc-banner::backdrop{background-color:#0000004d}.cc-banner *{color:inherit!important}.cc-banner:focus{box-shadow:none;outline:0}.cc-banner a{color:#fff!important;text-decoration:underline}.cc-banner a:active,.cc-banner a:focus,.cc-banner a:hover{color:inherit;text-decoration:none}.cc-banner a:focus{outline:3px solid #08c!important}.cc-banner h2,.cc-banner h3,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-banner .cc-h2,.cc-banner h2{font-size:18px}.cc-banner .cc-h3,.cc-banner h3{font-size:16px}.cc-banner .cc-h4,.cc-banner .cc-h5,.cc-banner .cc-h6,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-size:14px}.cc-banner .cc-button{font-size:16px}.cc-banner__content{background-color:#01324b;display:flex;flex-direction:column;margin:0 auto;max-height:90vh;max-width:100%;padding:16px;position:relative}.cc-banner__content:focus{outline:0}@media (min-width:680px){.cc-banner__content{padding:12px}}@media (min-width:980px){.cc-banner__content{max-height:60vh;padding-bottom:20px;padding-top:20px}}@media (min-width:1320px){.cc-banner__content{padding-bottom:40px;padding-top:40px}}.cc-banner__container{display:flex;flex-direction:column;margin:auto;max-width:1320px;overflow:auto}.cc-banner__title{background:none!important;border:0;flex-shrink:0;font-size:18px!important;font-size:22px!important;font-weight:700!important;letter-spacing:normal;margin:0 0 12px!important;padding:0}@media (min-width:680px){.cc-banner__title{font-size:24px!important;margin:0 0 16px!important}}@media (min-width:1320px){.cc-banner__title{font-size:26px!important;margin:0 0 24px!important}}.cc-banner__body{display:flex;flex-direction:column;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif!important;overflow-x:hidden;overflow-y:auto;padding:3px 3px 16px}@media (min-width:980px){.cc-banner__body{flex-direction:row}}.cc-banner__policy p{font-size:16px!important;margin:0;max-width:none;padding:0}.cc-banner__policy p:not(:last-child){margin:0 0 16px}@media (min-width:980px){.cc-banner__policy p:not(:last-child){margin:0 0 24px}}.cc-banner__policy p a{font-size:16px!important;font-weight:700}.cc-banner__footer{box-shadow:none;flex-shrink:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif !important!important;margin:0;padding:12px 0 3px 3px;position:relative}@media (min-width:380px){.cc-banner__footer{align-items:stretch;display:flex;flex-wrap:wrap}}@media (min-width:680px){.cc-banner__footer{box-shadow:none;flex-wrap:nowrap;width:40%}}.cc-banner__footer .cc-banner__button{flex:1 1 auto;font-weight:700;overflow:hidden;padding:.5em 1em;width:100%}@media (min-width:680px){.cc-banner__footer .cc-banner__button{max-width:275px}}.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:12px}@media (min-width:680px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:0;margin-right:16px}}@media (min-width:980px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-right:24px}}@media (min-width:680px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:16px!important;margin-right:0!important}}@media (min-width:980px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:24px!important;margin-right:0!important}}.cc-banner__button-preferences{padding-left:0;padding-right:0}@media (min-width:380px){.cc-banner__button-preferences{flex:0 0 auto;margin:auto}}@media (min-width:680px){.cc-banner__button-preferences{margin:0}}@media (min-width:380px) and (max-width:680px){.cc-banner__button-break{display:block}}@media (min-width:680px){.cc-banner--is-tcf .cc-banner__footer{width:auto}}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__footer{padding-right:48px;width:66%}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px!important}.cc-banner--is-tcf .cc-banner__stacks{margin:16px 0 0}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__stacks{display:flex;flex:0 0 33%;flex-direction:column;margin:0 0 0 48px;overflow:auto}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px;margin:0 0 12px}.cc-banner--is-tcf .cc-banner__stacks-details{padding:0 3px}.cc-banner--is-tcf .cc-banner .cc-details{border-color:#fff3!important}.cc-box{border-radius:10px;padding:12px}.cc-box--info{background-color:#eff6fb}.cc-box--light{background-color:#faf9f6}.cc-button{border:2px solid #0000;border-radius:32px!important;color:inherit;cursor:pointer;font-size:14px;font-weight:700!important;left:auto;line-height:1.2;margin:0;padding:.5em 1em;right:auto;text-transform:none!important;transition:all .2s}@media (min-width:680px){.cc-button{font-size:14px;padding:.75em 1em}}.cc-button--sm{font-size:12px;padding:2px 8px!important}.cc-button--primary{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--primary:focus{outline:3px solid #08c}.cc-button--primary:focus,.cc-button--primary:hover,.cc-button--secondary{background-color:#fff!important;border-color:#025e8d!important;color:#025e8d!important}.cc-button--secondary:focus{background-color:#025e8d!important;border-color:#fff!important;color:#fff!important;outline:3px solid #08c!important}.cc-button--secondary:hover{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--secondary:active{border:2px solid #01324b!important}.cc-button--secondary.cc-button--contrast{border-color:#fff!important}.cc-button--secondary.cc-button--contrast:hover{background-color:initial!important;color:#fff!important}.cc-button--tertiary{background-color:#f0f7fc!important;background-image:none;border:2px solid #f0f7fc;color:#025e8d!important;text-decoration:underline!important;text-underline-offset:.25em}.cc-button--tertiary:focus{outline:3px solid #08c!important}.cc-button--tertiary:hover{text-decoration-thickness:.25em}.cc-button--tertiary:active{color:#013c5b}.cc-button--link{background-color:initial!important;border-color:#0000!important;border-radius:0!important;color:inherit!important;padding:0!important;text-decoration:underline!important;width:inherit!important}.cc-button--link:focus{outline:3px solid #08c}.cc-button--link:hover{background-color:initial!important;box-shadow:none;text-decoration:none}.cc-button--text{border-radius:0;padding:0}.cc-button--details{padding-right:24px!important;position:relative;width:auto}.cc-button--details:after,.cc-button--details:before{background-color:currentColor;content:"";position:absolute;transition:transform .25s ease-out}.cc-button--details:before{height:10px;margin-top:-5px;right:12px;top:50%;width:2px}.cc-button--details:after{height:2px;margin-top:-1px;right:8px;top:50%;width:10px}.cc-button--details.cc-active:before{transform:rotate(90deg);transform-origin:center}.cc-button--details.cc-active:after{display:none}.cc-details{border-bottom:1px solid #0000001a;padding:12px 0;position:relative;width:100%}.cc-details__summary{align-items:center;display:flex;font-size:14px;font-weight:700;list-style-type:none}.cc-details__summary:focus,.cc-details__summary:focus-visible{outline:3px solid #08c!important;will-change:auto}.cc-details__title{align-items:baseline;display:flex}.cc-details__title h2,.cc-details__title h3,.cc-details__title h4,.cc-details__title h5{line-height:1.4;margin:0!important;padding:0!important}.cc-details__title svg{flex-shrink:0;height:auto;margin-right:8px;position:relative;top:-1px;transition:all .2s;width:auto}.cc-details[open] .cc-details__title svg{top:2px;transform:rotate(90deg)}.cc-details__switch{margin-left:auto}.cc-details__section{padding:16px 0 0 18px}.cc-details__section p{margin:0}.cc-details__section p:not(:last-child){margin-bottom:12px}details summary::-webkit-details-marker{display:none}.cc-radio{align-items:center;display:flex;position:relative}.cc-radio *{cursor:pointer}.cc-radio__input{height:22px;left:0;position:absolute;top:0;width:22px}.cc-radio__input:focus{outline:none}.cc-radio__label{color:inherit;font-size:14px;font-weight:700;line-height:23px;margin:0;padding-left:28px}.cc-radio__label:after,.cc-radio__label:before{background-color:#fff;content:"";display:block;position:absolute;transition:transform .25s ease-out}.cc-radio__label:before{border:1px solid #777;border-radius:50%;height:22px;left:0;top:0;width:22px}.cc-radio__label:after{border:7px solid #025e8d;border-radius:50%;height:0;left:4px;opacity:0;top:4px;width:0}.cc-radio__input:focus+.cc-radio__label:before{box-shadow:0 0 0 2px #08c;outline:none}.cc-radio__label--hidden{display:none}.cc-radio__input:checked+.cc-radio__label:after{opacity:1}.cc-radio__input:disabled{cursor:default}.cc-radio__input:disabled+.cc-radio__label{cursor:default;opacity:.5}.cc-switch *{cursor:pointer}.cc-switch{align-items:center;display:flex;line-height:1}.cc-switch__label{cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;-webkit-user-select:none;user-select:none}.cc-switch__input[type=checkbox]{-webkit-appearance:none;appearance:none;background-color:#fff!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='rgba(0, 0, 0, 0.25)'/%3E%3C/svg%3E")!important;background-position:0!important;background-repeat:no-repeat!important;border:1px solid #777!important;border-radius:16px!important;height:18px!important;margin:0 8px 0 0!important;transition:background-position .2s ease-in-out!important;width:32px!important}.cc-switch__input[type=checkbox]:focus{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%2380b3cc'/%3E%3C/svg%3E")!important}.cc-switch__input[type=checkbox]:checked{background-color:#025e8d!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%23fff'/%3E%3C/svg%3E")!important;background-position:100%!important}.cc-switch__input[type=checkbox]:checked~.cc-switch__label>.cc-switch__text-off,.cc-switch__input[type=checkbox]:not(:checked)~.cc-switch__label>.cc-switch__text-on{display:none}.cc-switch__input[type=checkbox]:hover{cursor:pointer;outline:none}.cc-switch__input[type=checkbox]:focus-visible{cursor:pointer;outline:3px solid #fc0}.cc-switch__input[type=checkbox]:checked:before{background-color:none!important;color:#0000!important}html[dir=rtl] .cc-switch__input[type=checkbox]{margin-left:8px!important;margin-right:0!important}.cc-list>ul,ul.cc-list{list-style-type:disc;margin-left:0;padding-left:0}.cc-list>ul>li,ul.cc-list>li{margin-left:1em}.cc-list>ul>li:not(:last-child),ul.cc-list>li:not(:last-child){margin-bottom:4px}.cc-list__title{margin-bottom:8px!important}.cc-list--inline>ul,ul.cc-list--inline{display:flex;flex-wrap:wrap;list-style-type:none}.cc-list--inline>ul>li,ul.cc-list--inline>li{margin:0 .5em .5em 0}.cc-overlay{background-color:#0000004d;z-index:99998}.cc-overlay,.cc-preferences{bottom:0;left:0;position:fixed;right:0;top:0}.cc-preferences{background-color:#050a14f2!important;border:0;box-sizing:border-box;color:#111;font-family:sans-serif!important;line-height:1.4;margin:auto;max-height:100vh;overflow:auto;padding:0;z-index:100000}.cc-preferences:focus{outline:none}.cc-preferences *,.cc-preferences :after,.cc-preferences :before{box-sizing:inherit!important}.cc-preferences h2,.cc-preferences h3,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-preferences .cc-h2,.cc-preferences h2{font-size:18px}.cc-preferences .cc-h3,.cc-preferences h3{font-size:16px}.cc-preferences .cc-h4,.cc-preferences .cc-h5,.cc-preferences .cc-h6,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-size:14px}.cc-preferences a{color:#025e8d;text-decoration:underline}.cc-preferences a:hover{color:inherit;text-decoration:none}.cc-preferences h3{background:none;color:#111;padding:0;text-transform:none}dialog.cc-preferences{background-color:initial}dialog.cc-preferences::backdrop{background-color:#000000e6}.cc-preferences__dialog{display:flex;flex-direction:column;margin:auto;max-height:100vh;max-width:860px;padding:12px;position:relative}.cc-preferences__dialog>:last-child{border-bottom-left-radius:10px;border-bottom-right-radius:10px}@media (min-width:980px){.cc-preferences__dialog{padding:16px}}.cc-preferences__close{background:#0000!important;border:1px solid #ececec;border-radius:50%;color:#111!important;cursor:pointer;font-family:Times New Roman,serif;font-size:40px;height:40px;left:auto;line-height:1;margin-top:-20px;padding:0!important;position:absolute;right:20px;top:50%;width:40px}.cc-preferences__close:focus{outline:3px solid #08c}.cc-preferences__close-label{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__header{background:#fff!important;border-bottom:1px solid #0000001a;border-top-left-radius:10px;border-top-right-radius:10px;padding:16px;position:relative;text-align:center}.cc-preferences__title{background:none!important;color:#111!important;font-family:sans-serif!important;font-size:18px!important;font-weight:700!important;margin:0!important;padding:0 16px 0 0!important}@media (min-width:480px){.cc-preferences__title{padding-right:0!important}}@media (min-width:980px){.cc-preferences__title{font-size:22px!important}}.cc-preferences__body{background:#fff!important;flex:1 1 auto;min-height:200px;overflow-x:hidden;overflow-y:auto;padding:16px}.cc-preferences__footer{background:#fff!important;border-top:1px solid #d0d0d0;box-shadow:0 0 5px 0 #0003;margin-bottom:0;padding:12px;position:relative}@media (min-width:480px){.cc-preferences__footer{align-items:stretch;display:flex}}.cc-preferences__footer>.cc-button{display:block;width:100%}@media (min-width:480px){.cc-preferences__footer>.cc-button{flex:1 1 auto}}@media (min-width:980px){.cc-preferences__footer>.cc-button{flex-basis:auto}}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:first-child){margin-left:12px}}.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:8px}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:0}}.cc-preferences__categories{list-style:none;margin:0;padding:0}.cc-preferences__category:not(:last-child){border-bottom:1px solid #0000001a;margin-bottom:12px;padding-bottom:12px}.cc-preferences__category-description{font-size:14px;margin:0 0 8px;padding:0}.cc-preferences__category-footer{align-items:center;display:flex;justify-content:space-between}.cc-preferences__status{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__controls{display:flex;margin:0}.cc-preferences__controls>:not(:last-child){margin-right:12px}.cc-preferences__always-on{font-size:14px;font-weight:700;padding-left:26px;position:relative}.cc-preferences__always-on:before{background:#0000!important;border:solid;border-top-color:#0000;border-width:0 0 4px 4px;content:"";display:block;height:10px;left:0;position:absolute;top:2px;transform:rotate(-45deg);width:18px}.cc-preferences__details{background-color:#eff6fb!important;border-radius:10px;font-size:14px;margin:12px 0 0;padding:12px}.cc-preferences__cookie-list,.cc-preferences__provider-list{list-style:none;margin:0;padding:0}.cc-preferences__provider-list{columns:170px}.cc-preferences__cookie-title{font-size:1em;font-style:normal;font-weight:400;margin:0}.cc-preferences__cookie-description{font-size:1em;margin:0 0 8px}.cc-preferences__cookie-domain,.cc-preferences__cookie-lifespan{border-left:1px solid #999;color:#666;margin-left:8px;padding-left:4px}body.cc-has-preferences-open{overflow:hidden;position:relative}.cc-table{border-collapse:collapse;width:100%}.cc-table tbody tr{border-top:1px solid #0000001a}.cc-table td,.cc-table th{font-size:14px;padding:4px 8px;vertical-align:top}.cc-table+.cc-table{margin-top:8px}.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:70%}@media (min-width:680px){.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:80%}}.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{padding-left:8px;width:40%}@media (min-width:680px){.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{width:30%}}.cc-tabs,.cc-tabs>ul{background-color:#faf9f6;display:flex;list-style:none;margin:0;padding:0}.cc-tabs__panel{border-top:1px solid #ececec;margin-top:-1px}.cc-tabs__button{background-color:initial;border:0;border-left:1px solid #0000;border-right:1px solid #0000;color:#111;font-size:16px;font-weight:700;margin:0;padding:15px 16px 12px;position:relative}.cc-tabs__button:before{background-color:initial;content:"";display:block;height:3px;left:0;position:absolute;top:0;width:100%}.cc-tabs__button.cc-active{background-color:#fff;border-color:#025e8d #ececec #ececec;color:#111}.cc-tabs__button.cc-active:before{background-color:#025e8d}.cc-tabs__content{background-color:#fff;display:none;font-size:14px;padding:16px}.cc-tabs__section:not(:first-child){padding-top:24px}.cc-tcf{font-size:14px;margin-top:16px}.cc-tcf__list{list-style:none;margin:0;padding:0}.cc-vendor-count{color:#666;font-size:16px;margin:0 0 8px}.cc-hide{display:none!important}.cc-show{display:block!important}.cc-external-link{background-color:#ececec;background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTcgMGExIDEgMCAxIDEgMCAySDIuNmMtLjM3MSAwLS42LjIwOS0uNi41djE1YzAgLjI5MS4yMjkuNS42LjVoMTQuOGMuMzcxIDAgLjYtLjIwOS42LS41VjEzYTEgMSAwIDAgMSAyIDB2NC41YzAgMS40MzgtMS4xNjIgMi41LTIuNiAyLjVIMi42QzEuMTYyIDIwIDAgMTguOTM4IDAgMTcuNXYtMTVDMCAxLjA2MiAxLjE2MiAwIDIuNiAwem02IDBoNmwuMDc1LjAwMy4xMjYuMDE3LjExMS4wMy4xMTEuMDQ0LjA5OC4wNTIuMDk2LjA2Ny4wOS4wOGExIDEgMCAwIDEgLjA5Ny4xMTJsLjA3MS4xMS4wNTQuMTE0LjAzNS4xMDUuMDMuMTQ4TDIwIDF2NmExIDEgMCAwIDEtMiAwVjMuNDE0bC02LjY5MyA2LjY5M2ExIDEgMCAwIDEtMS40MTQtMS40MTRMMTYuNTg0IDJIMTNhMSAxIDAgMCAxLS45OTMtLjg4M0wxMiAxYTEgMSAwIDAgMSAxLTEiLz48L3N2Zz4=");background-position:right 8px center;background-repeat:no-repeat;background-size:10px auto;border-radius:3px;box-shadow:0 1px 0 #0003;color:#111!important;display:inline-block;font-size:12px;padding:4px 26px 4px 8px;text-decoration:none!important}.cc-external-link:hover{text-decoration:underline!important}.cc-m-0{margin:0!important}.cc-grey{color:#666}</style><script async="" type="text/javascript" src="https://injections.readcube.com/nature/inject.fa744f3f.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>
<body class="article-page"><div id="MathJax_Message" style="display: none;"></div><dialog class="cc-banner" data-cc-banner="" data-nosnippet="" aria-labelledby="cc-banner-label" open="">
		<div class="cc-banner__content" autofocus="" tabindex="-1">
			<div class="cc-banner__container">
				<div class="cc-banner__header">
					<h2 class="cc-banner__title" id="cc-banner-label">Your privacy, your choice</h2>
				</div>
				<div class="cc-banner__body">
					<div class="cc-banner__policy">
						<p>We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.</p><p>By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.</p><p>See our <a href="https://www.nature.com/info/privacy" data-cc-action="privacy">privacy policy</a> for more information on the use of your personal data.</p><p><button type="button" data-cc-action="preferences" class="cc-button cc-button--link cc-button--text">Manage preferences</button> for further information and to change your choices.</p>
					</div>
					
				</div>
				<div class="cc-banner__footer">
					
			<button data-cc-action="accept" class="cc-button cc-button--secondary cc-button--contrast cc-banner__button cc-banner__button-accept">Accept all cookies</button>
			
		
				</div>
			</div>
		</div>
	</dialog>

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90" style="display: none !important;">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-test="top-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41591-022-01913-0;doi=10.1038/s41591-022-01913-0;subjmeta=107,2134,2142,2565,326,631,855;kwrd=Clinical+microbiology,Metagenomics,Microbial+ecology,Microbiome">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=1645669305&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01913-0%26doi%3D10.1038/s41591-022-01913-0%26subjmeta%3D107,2134,2142,2565,326,631,855%26kwrd%3DClinical+microbiology,Metagenomics,Microbial+ecology,Microbiome">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=1645669305&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01913-0%26doi%3D10.1038/s41591-022-01913-0%26subjmeta%3D107,2134,2142,2565,326,631,855%26kwrd%3DClinical+microbiology,Metagenomics,Microbial+ecology,Microbiome"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header="" data-track-component="nature-150-split-header" style="border-color:#e40428">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nm" data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" height="32" alt="Nature Medicine">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
                            </a><div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected="">All journals</option>
                                    <option value="nm">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41591-022-01913-0"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header" hidden="">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-articles" data-track="click" data-track-action="research articles" data-track-label="link" data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-and-analysis" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link" data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/podcast" data-track="click" data-track-action="podcasts" data-track-label="link" data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/current-issue" data-track="click" data-track-action="current issue" data-track-label="link" data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/collections" data-track="click" data-track-action="collections" data-track-label="link" data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://www.facebook.com/Nature-Medicine-193691346949/" data-track="click" data-track-action="facebook" data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://twitter.com/naturemedicine" data-track="click" data-track-action="twitter" data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=5" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/aims" data-track="click" data-track-action="aims &amp; scope" data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-information" data-track="click" data-track-action="journal information" data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-impact" data-track="click" data-track-action="journal metrics" data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editors" data-track="click" data-track-action="about the editors" data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-cross-journal-editorial-team" data-track="click" data-track-action="research cross-journal editorial team" data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-cross-journal-editorial-team" data-track="click" data-track-action="reviews cross-journal editorial team" data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/statistics-advisory-panel" data-track="click" data-track-action="statistical advisory panel" data-track-label="link">
                                    Statistical Advisory Panel
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/our-publishing-models" data-track="click" data-track-action="our publishing models" data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-values-statement" data-track="click" data-track-action="editorial values statement" data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-policies" data-track="click" data-track-action="editorial policies" data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/content" data-track="click" data-track-action="content types" data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="javascript:;" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                        </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/submission-guidelines" data-track="click" data-track-action="submission guidelines" data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-context="header publish with us dropdown menu" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/open-access-funding" data-test="funding-eligibility-link" data-track="click_explore_funding" data-track-context="header publish with us" data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link" href="https://mts-nmed.nature.com/cgi-bin/main.plex" data-track="click_submit_manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-action="submit manuscript" data-track-label="link (publish with us dropdown menu)" data-track-external="" data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link" href="https://journal-alerts.springernature.com/subscribe?journal_id=41591" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-label="link (desktop site header)" data-track-external="">
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature medicine"><span itemprop="name">nature medicine</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm/articles?type=article" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" id="js-enable-context-bar" data-test="context-bar" data-context-bar="" aria-hidden="true">
                    <div class="c-context-bar__container u-container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            Drivers and determinants of strain dynamics following fecal microbiota transplantation
                        </div>
                        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01913-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01913-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2022-09-15">15 September 2022</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Drivers and determinants of strain dynamics following fecal microbiota transplantation</h1>
                    <ul class="c-article-author-list c-article-author-list--short js-no-scroll" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Thomas_S__B_-Schmidt-Aff1" data-author-popup="auth-Thomas_S__B_-Schmidt-Aff1" data-author-search="Schmidt, Thomas S. B." data-track-context="researcher popup with no profile" data-track-index="1_22">Thomas S. B. Schmidt</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-8587-4177"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-8587-4177</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Simone_S_-Li-Aff1-Aff12-Aff13" data-author-popup="auth-Simone_S_-Li-Aff1-Aff12-Aff13" data-author-search="Li, Simone S." data-track-context="researcher popup with no profile" data-track-index="2_22">Simone S. Li</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-0073-3656"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0073-3656</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup><sup class="u-js-hide">&nbsp;<a href="#nAff12" tabindex="-1">nAff12</a></sup><sup class="u-js-hide">&nbsp;<a href="#nAff13" tabindex="-1">nAff13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Oleksandr_M_-Maistrenko-Aff1-Aff14" data-author-popup="auth-Oleksandr_M_-Maistrenko-Aff1-Aff14" data-author-search="Maistrenko, Oleksandr M." data-track-context="researcher popup with no profile" data-track-index="3_22">Oleksandr M. Maistrenko</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-1961-7548"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1961-7548</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#nAff14" tabindex="-1">nAff14</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Wasiu-Akanni-Aff1" data-author-popup="auth-Wasiu-Akanni-Aff1" data-author-search="Akanni, Wasiu" data-track-context="researcher popup with no profile" data-track-index="4_22">Wasiu Akanni</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-2075-2387"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2075-2387</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Luis_Pedro-Coelho-Aff1-Aff15" data-author-popup="auth-Luis_Pedro-Coelho-Aff1-Aff15" data-author-search="Coelho, Luis Pedro" data-track-context="researcher popup with no profile" data-track-index="5_22">Luis Pedro Coelho</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#nAff15" tabindex="-1">nAff15</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sibasish-Dolai-Aff2" data-author-popup="auth-Sibasish-Dolai-Aff2" data-author-search="Dolai, Sibasish" data-track-context="researcher popup with no profile" data-track-index="6_22">Sibasish Dolai</a><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Anthony-Fullam-Aff1" data-author-popup="auth-Anthony-Fullam-Aff1" data-author-search="Fullam, Anthony" data-track-context="researcher popup with no profile" data-track-index="7_22">Anthony Fullam</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Anna_M_-Glazek-Aff1" data-author-popup="auth-Anna_M_-Glazek-Aff1" data-author-search="Glazek, Anna M." data-track-context="researcher popup with no profile" data-track-index="8_22">Anna M. Glazek</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Rajna-Hercog-Aff3" data-author-popup="auth-Rajna-Hercog-Aff3" data-author-search="Hercog, Rajna" data-track-context="researcher popup with no profile" data-track-index="9_22">Rajna Hercog</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-2587-9739"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2587-9739</a></span><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Hilde-Herrema-Aff4" data-author-popup="auth-Hilde-Herrema-Aff4" data-author-search="Herrema, Hilde" data-track-context="researcher popup with no profile" data-track-index="10_22">Hilde Herrema</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ferris-Jung-Aff3" data-author-popup="auth-Ferris-Jung-Aff3" data-author-search="Jung, Ferris" data-track-context="researcher popup with no profile" data-track-index="11_22">Ferris Jung</a><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stefanie-Kandels-Aff1" data-author-popup="auth-Stefanie-Kandels-Aff1" data-author-search="Kandels, Stefanie" data-track-context="researcher popup with no profile" data-track-index="12_22">Stefanie Kandels</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Askarbek-Orakov-Aff1" data-author-popup="auth-Askarbek-Orakov-Aff1" data-author-search="Orakov, Askarbek" data-track-context="researcher popup with no profile" data-track-index="13_22">Askarbek Orakov</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6823-5269"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6823-5269</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Roman-Thielemann-Aff1" data-author-popup="auth-Roman-Thielemann-Aff1" data-author-search="Thielemann, Roman" data-track-context="researcher popup with no profile" data-track-index="14_22">Roman Thielemann</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-2582-6178"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2582-6178</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Moritz-Stetten-Aff1" data-author-popup="auth-Moritz-Stetten-Aff1" data-author-search="von Stetten, Moritz" data-track-context="researcher popup with no profile" data-track-index="15_22">Moritz von Stetten</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Thea-Van_Rossum-Aff1" data-author-popup="auth-Thea-Van_Rossum-Aff1" data-author-search="Van Rossum, Thea" data-track-context="researcher popup with no profile" data-track-index="16_22">Thea Van Rossum</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Vladimir-Benes-Aff3" data-author-popup="auth-Vladimir-Benes-Aff3" data-author-search="Benes, Vladimir" data-track-context="researcher popup with no profile" data-track-index="17_22">Vladimir Benes</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-0352-2547"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0352-2547</a></span><sup class="u-js-hide"><a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Thomas_J_-Borody-Aff2" data-author-popup="auth-Thomas_J_-Borody-Aff2" data-author-search="Borody, Thomas J." data-track-context="researcher popup with no profile" data-track-index="18_22">Thomas J. Borody</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-0519-4698"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0519-4698</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Willem_M_-Vos-Aff5-Aff6" data-author-popup="auth-Willem_M_-Vos-Aff5-Aff6" data-author-search="de Vos, Willem M." data-track-context="researcher popup with no profile" data-track-index="19_22">Willem M. de Vos</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-0273-3166"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-0273-3166</a></span><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a>,<a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Cyriel_Y_-Ponsioen-Aff7" data-author-popup="auth-Cyriel_Y_-Ponsioen-Aff7" data-author-search="Ponsioen, Cyriel Y." data-track-context="researcher popup with no profile" data-track-index="20_22">Cyriel Y. Ponsioen</a><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Max-Nieuwdorp-Aff4-Aff8" data-author-popup="auth-Max-Nieuwdorp-Aff4-Aff8" data-author-search="Nieuwdorp, Max" data-track-context="researcher popup with no profile" data-track-index="21_22">Max Nieuwdorp</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff8" tabindex="-1">8</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 22 authors for this article" title="Show all 22 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Peer-Bork-Aff1-Aff9-Aff10-Aff11" data-author-popup="auth-Peer-Bork-Aff1-Aff9-Aff10-Aff11" data-author-search="Bork, Peer" data-corresp-id="c1" data-track-context="researcher popup with no profile" data-track-index="22_22">Peer Bork<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-2627-833X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2627-833X</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff9" tabindex="-1">9</a>,<a href="#Aff10" tabindex="-1">10</a>,<a href="#Aff11" tabindex="-1">11</a></sup>&nbsp;</li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nm" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Medicine</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;28</b>,&nbsp;<span class="u-visually-hidden">pages </span>1902–1912 (<span data-test="article-publication-year">2022</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item" data-test="access-count">
                        <p class="c-article-metrics-bar__count">27k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item" data-test="citation-count">
                        <p class="c-article-metrics-bar__count">109 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item" data-test="altmetric-score">
                            <p class="c-article-metrics-bar__count">146 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41591-022-01913-0/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Fecal microbiota transplantation (FMT) is a therapeutic intervention for inflammatory diseases of the gastrointestinal tract, but its clinical mode of action and subsequent microbiome dynamics remain poorly understood. Here we analyzed metagenomes from 316 FMTs, sampled pre and post intervention, for the treatment of ten different disease indications. We quantified strain-level dynamics of 1,089 microbial species, complemented by 47,548 newly constructed metagenome-assembled genomes. Donor strain colonization and recipient strain resilience were mostly independent of clinical outcomes, but accurately predictable using LASSO-regularized regression models that accounted for host, microbiome and procedural variables. Recipient factors and donor–recipient complementarity, encompassing entire microbial communities to individual strains, were the main determinants of strain population dynamics, providing insights into the underlying processes that shape the post-FMT gut microbiome. Applying an ecology-based framework to our findings indicated parameters that may inform the development of more effective, targeted microbiome therapies in the future, and suggested how patient stratification can be used to enhance donor microbiota colonization or the displacement of recipient microbes in clinical practice.</p></div></div></section>

            
                
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41591-022-01964-3/MediaObjects/41591_2022_1964_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41591-022-01964-3?fromPaywallRec=false" data-track="select_recommendations_1" data-track-context="inline recommendations" data-track-action="click recommendations inline - 1" data-track-label="10.1038/s41591-022-01964-3">Variability of strain engraftment and predictability of microbiome composition after fecal microbiota transplantation across different diseases
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">15 September 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41564-021-00966-0/MediaObjects/41564_2021_966_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41564-021-00966-0?fromPaywallRec=false" data-track="select_recommendations_2" data-track-context="inline recommendations" data-track-action="click recommendations inline - 2" data-track-label="10.1038/s41564-021-00966-0">Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">27 September 2021</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-022-09307-5/MediaObjects/41598_2022_9307_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41598-022-09307-5?fromPaywallRec=false" data-track="select_recommendations_3" data-track-context="inline recommendations" data-track-action="click recommendations inline - 3" data-track-label="10.1038/s41598-022-09307-5">Strain-resolved analysis in a randomized trial of antibiotic pretreatment and maintenance dose delivery mode with fecal microbiota transplant for ulcerative colitis
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">01 April 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1749522427,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Fecal microbiota transplantation involves the transfer of gut microbes, viruses and luminal content to modulate a recipient’s microbiome, for therapeutic purposes. While the efficacy of FMT has been demonstrated for various diseases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Borody, T. J. et al. Bacteriotherapy using fecal flora: toying with human motions. J. Clin. Gastroenterol. 38, 475–483 (2004)." href="#ref-CR1" id="ref-link-section-d32986547e983">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rossen, N. G. et al. Fecal microbiota transplantation as novel therapy in gastroenterology: a systematic review. World J. Gastroenterol. 21, 5359–5371 (2015)." href="#ref-CR2" id="ref-link-section-d32986547e983_1">2</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Hanssen, N. M. J., de Vos, W. M. &amp; Nieuwdorp, M. Fecal microbiota transplantation in human metabolic diseases: from a murky past to a bright future? Cell Metab. 33, 1098–1110 (2021)." href="/articles/s41591-022-01913-0#ref-CR3" id="ref-link-section-d32986547e986">3</a></sup>, such as recurrent <i>Clostridioides difficile</i> infection (rCDI)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Gough, E., Shaikh, H. &amp; Manges, A. R. Systematic review of intestinal microbiota transplantation (fecal bacteriotherapy) for recurrent Clostridium difficile infection. Clin. Infect. Dis. 53, 994–1002 (2011)." href="/articles/s41591-022-01913-0#ref-CR4" id="ref-link-section-d32986547e993">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="van Nood, E. et al. Duodenal infusion of donor feces for recurrent Clostridium difficile. N. Engl. J. Med. 368, 407–415 (2013)." href="/articles/s41591-022-01913-0#ref-CR5" id="ref-link-section-d32986547e996">5</a></sup> or ulcerative colitis (UC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Narula, N. et al. Systematic review and meta-analysis fecal microbiota transplantation for treatment of active ulcerative colitis. Inflamm. Bowel Dis. 23, 1702–1709 (2017)." href="/articles/s41591-022-01913-0#ref-CR6" id="ref-link-section-d32986547e1000">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Haifer, C., Leong, R. W. &amp; Paramsothy, S. The role of fecal microbiota transplantation in the treatment of inflammatory bowel disease. Curr. Opin. Pharmacol. 55, 8–16 (2020)." href="/articles/s41591-022-01913-0#ref-CR7" id="ref-link-section-d32986547e1003">7</a></sup>), it may also facilitate microbiome recovery following disturbance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Suez, J. et al. Post-antibiotic gut mucosal microbiome reconstitution is impaired by probiotics and improved by autologous FMT. Cell 174, 1406–1423 (2018)." href="/articles/s41591-022-01913-0#ref-CR8" id="ref-link-section-d32986547e1007">8</a></sup> and can enhance microbiome-mediated responses to other therapies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Baruch, E. N. et al. Fecal microbiota transplant promotes response in immunotherapy-refractory melanoma patients. Science 371, 602–609 (2021)." href="/articles/s41591-022-01913-0#ref-CR9" id="ref-link-section-d32986547e1012">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Davar, D. et al. Fecal microbiota transplant overcomes resistance to anti-PD-1 therapy in melanoma patients. Science 371, 595–602 (2021)." href="/articles/s41591-022-01913-0#ref-CR10" id="ref-link-section-d32986547e1015">10</a></sup>. Nevertheless, despite demonstrable efficacy in a growing range of clinical applications, the mode of action of FMT remains poorly understood<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Hanssen, N. M. J., de Vos, W. M. &amp; Nieuwdorp, M. Fecal microbiota transplantation in human metabolic diseases: from a murky past to a bright future? Cell Metab. 33, 1098–1110 (2021)." href="/articles/s41591-022-01913-0#ref-CR3" id="ref-link-section-d32986547e1019">3</a></sup> and neither clinical success nor adverse outcomes are currently predictable with accuracy.</p><p>Because FMT primarily targets the microbiome, the engraftment of ‘beneficial’ and/or displacement of ‘detrimental’ microbes are expected to cause clinical effects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Hanssen, N. M. J., de Vos, W. M. &amp; Nieuwdorp, M. Fecal microbiota transplantation in human metabolic diseases: from a murky past to a bright future? Cell Metab. 33, 1098–1110 (2021)." href="/articles/s41591-022-01913-0#ref-CR3" id="ref-link-section-d32986547e1026">3</a></sup>, in conjunction with more specific processes of host–microbiome interplay, such as the modulation of immune responses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 11" title="Burrello, C. et al. Therapeutic fecal microbiota transplantation controls intestinal inflammation through IL10 secretion by immune cells. Nat. Commun. 9, 5184 (2018)." href="/articles/s41591-022-01913-0#ref-CR11" id="ref-link-section-d32986547e1030">11</a></sup>, restored short-chain fatty acid (SCFA) metabolism<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Seekatz, A. M. et al. Restoration of short chain fatty acid and bile acid metabolism following fecal microbiota transplantation in patients with recurrent Clostridium difficile infection. Anaerobe 53, 64–73 (2018)." href="/articles/s41591-022-01913-0#ref-CR12" id="ref-link-section-d32986547e1034">12</a></sup> or reinstated phage pressure<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Zuo, T. et al. Bacteriophage transfer during fecal microbiota transplantation is associated with treatment response in Clostridium difficile infection. Gastroenterology 152, S140–S141 (2017)." href="/articles/s41591-022-01913-0#ref-CR13" id="ref-link-section-d32986547e1038">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Manrique, P. et al. Gut bacteriophage dynamics during fecal microbial transplantation in subjects with metabolic syndrome. Gut Microbes 13, 1–15 (2021)." href="/articles/s41591-022-01913-0#ref-CR14" id="ref-link-section-d32986547e1041">14</a></sup>. It has been argued that both microbiome engraftment and clinical success are mainly determined by donor factors, and that rationally selected ‘super-donors’ may improve therapeutic efficacy<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Wilson, B. C., Vatanen, T., Cutfield, W. S. &amp; O’Sullivan, J. M. The super-donor phenomenon in fecal microbiota transplantation. Front. Cell. Infect. Microbiol. 9, 2 (2019)." href="/articles/s41591-022-01913-0#ref-CR15" id="ref-link-section-d32986547e1045">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Duvallet, C. et al. Framework for rational donor selection in fecal microbiota transplant clinical trials. PloS ONE 14, e0222881 (2019)." href="/articles/s41591-022-01913-0#ref-CR16" id="ref-link-section-d32986547e1048">16</a></sup>. This donor-centric view has since been questioned, at least for some indications<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Olesen, S. W. &amp; Gerardin, Y. Re-evaluating the evidence for fecal microbiota transplantation “super-donors” in inflammatory bowel disease. J. Crohns Colitis 15, 453–461 (2021)." href="/articles/s41591-022-01913-0#ref-CR17" id="ref-link-section-d32986547e1053">17</a></sup>, highlighting the importance of recipient<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kootte, R. S. et al. Improvement of insulin sensitivity after lean donor feces in metabolic syndrome is driven by baseline intestinal imcrobiota composition. Cell Metab. 26, 611–619 (2017)." href="#ref-CR18" id="ref-link-section-d32986547e1057">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Danne, C., Rolhion, N. &amp; Sokol, H. Recipient factors in fecal microbiota transplantation: one stool does not fit all. Nat. Rev. Gastroenterol. Hepatol. 18, 503–513 (2021)." href="#ref-CR19" id="ref-link-section-d32986547e1057_1">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Fujimoto, K. et al. Functional restoration of bacteriomes and viromes by fecal microbiota transplantation. Gastroenterology 160, 2089–2102 (2021)." href="/articles/s41591-022-01913-0#ref-CR20" id="ref-link-section-d32986547e1060">20</a></sup> or procedural<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Peri, R. et al. The impact of technical and clinical factors on fecal microbiota transfer outcomes for the treatment of recurrent Clostridioides difficile infections in Germany. United European Gastroenterol. J. 7, 716–722 (2019)." href="/articles/s41591-022-01913-0#ref-CR21" id="ref-link-section-d32986547e1064">21</a></sup> factors instead.</p><p>Changes in microbial compositions following FMT have been studied with regard to phages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Draper, L. A. et al. Long-term colonisation with donor bacteriophages following successful fecal microbial transplantation. Microbiome 6, 220 (2018)." href="/articles/s41591-022-01913-0#ref-CR22" id="ref-link-section-d32986547e1071">22</a></sup> or fungi<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Leonardi, I. et al. Fungal trans-kingdom dynamics linked to responsiveness to fecal microbiota transplantation (FMT) therapy in ulcerative colitis. Cell Host Microbe 27, 823–829 (2020)." href="/articles/s41591-022-01913-0#ref-CR23" id="ref-link-section-d32986547e1075">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Zuo, T. et al. Gut fungal dysbiosis correlates with reduced efficacy of fecal microbiota transplantation in Clostridium difficile infection. Nat. Commun. 9, 3663 (2018)." href="/articles/s41591-022-01913-0#ref-CR24" id="ref-link-section-d32986547e1078">24</a></sup>, yet the bulk of current knowledge is focused on bacteria and archaea where colonization by donor microbes and the persistence of indigenous recipient microbes emerge at the strain level of microbial populations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Li, S. S. et al. Durable coexistence of donor and recipient strains after fecal microbiota transplantation. Science 352, 586–589 (2016)." href="/articles/s41591-022-01913-0#ref-CR25" id="ref-link-section-d32986547e1082">25</a></sup>. Strain-level studies suggest that colonization levels following FMT vary across indications: whereas donor and recipient strains coexist long term in metabolic syndrome (MetS) patients<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Li, S. S. et al. Durable coexistence of donor and recipient strains after fecal microbiota transplantation. Science 352, 586–589 (2016)." href="/articles/s41591-022-01913-0#ref-CR25" id="ref-link-section-d32986547e1086">25</a></sup>, donor takeover is the most common outcome in rCDI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Smillie, C. S. et al. Strain tracking reveals the determinants of bacterial engraftment in the human gut following fecal microbiota transplantation. Cell Host Microbe 23, 229–240 (2018)." href="#ref-CR26" id="ref-link-section-d32986547e1090">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Podlesny, D. et al. Metagenomic strain detection with SameStr: identification of a persisting core gut microbiota transferable by fecal transplantation. Microbiome 10, 53 (2022)." href="#ref-CR27" id="ref-link-section-d32986547e1090_1">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Aggarwala, V. et al. Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes. Nat. Microbiol. 6, 1309–1318 (2021)." href="/articles/s41591-022-01913-0#ref-CR28" id="ref-link-section-d32986547e1093">28</a></sup>, with intermediate outcomes in UC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lee, S. T. M. et al. Tracking microbial colonization in fecal microbiota transplantation experiments via genome-resolved metagenomics. Microbiome 5, 50 (2017)." href="/articles/s41591-022-01913-0#ref-CR29" id="ref-link-section-d32986547e1098">29</a></sup> or obesity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Wilson, B. C. et al. Strain engraftment competition and functional augmentation in a multi-donor fecal microbiota transplantation trial for obesity. Microbiome 9, 107 (2021)." href="/articles/s41591-022-01913-0#ref-CR30" id="ref-link-section-d32986547e1102">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Ng, S. C. et al. Microbiota engraftment after fecal microbiota transplantation in obese subjects with type 2 diabetes: a 24-week, double-blind, randomised controlled trial. Gut 71, 716–723 (2022)." href="/articles/s41591-022-01913-0#ref-CR31" id="ref-link-section-d32986547e1105">31</a></sup>. However, the factors shaping these differential strain-level outcomes remain poorly understood. In small pilot study cohorts, colonization success of donor strains leading to short-term persistence was associated with species phylogeny, broad microbial phenotypes and relative fecal abundances in rCDI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Smillie, C. S. et al. Strain tracking reveals the determinants of bacterial engraftment in the human gut following fecal microbiota transplantation. Cell Host Microbe 23, 229–240 (2018)." href="/articles/s41591-022-01913-0#ref-CR26" id="ref-link-section-d32986547e1109">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Podlesny, D. et al. Metagenomic strain detection with SameStr: identification of a persisting core gut microbiota transferable by fecal transplantation. Microbiome 10, 53 (2022)." href="/articles/s41591-022-01913-0#ref-CR27" id="ref-link-section-d32986547e1112">27</a></sup>, but with more adaptive metabolic phenotypes in UC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Watson, A. R., Fuessel, J., Veseli, I. &amp; DeLongchamp, J. Z. Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut. Preprint at 
                  https://doi.org/10.1101/2021.03.02.433653
                  
                 (2021)." href="/articles/s41591-022-01913-0#ref-CR32" id="ref-link-section-d32986547e1116">32</a></sup>.</p><p>Here we conducted a meta-analysis of novel and published metagenomes from fecal samples collected before and after FMT to compare the fate of donor and recipient strain populations across multiple disease indications. We hypothesized that drivers of FMT response are best studied from an ecological perspective:<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Walter, J., Maldonado-Gómez, M. X. &amp; Martínez, I. To engraft or not to engraft: an ecological framework for gut microbiome modulation with live microbes. Curr. Opin. Biotechnol. 49, 129–139 (2018)." href="#ref-CR33" id="ref-link-section-d32986547e1123">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Schmidt, T. S. B., Raes, J. &amp; Bork, P. The human gut microbiome: from association to modulation. Cell 172, 1198–1215 (2018)." href="#ref-CR34" id="ref-link-section-d32986547e1123_1">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Xiao, Y., Angulo, M. T., Lao, S., Weiss, S. T. &amp; Liu, Y.-Y. An ecological framework to understand the efficacy of fecal microbiota transplantation. Nat. Commun. 11, 3329 (2020)." href="/articles/s41591-022-01913-0#ref-CR35" id="ref-link-section-d32986547e1126">35</a></sup> FMTs can be thought of as untargeted perturbation experiments on the gut microbiome in natura, pitting donor communities against those of the recipient, with outcomes that emerge from underlying ecological processes. We therefore quantified strain-level patterns of donor strain colonization, recipient strain resilience and turnover following FMT, both at the broad level of entire communities and specifically for individual species. We built cross-validated models to predict FMT outcome—defined here as colonization of donor strains and resilience of resident strains of the recipient—based on either ex ante variables (that is, knowable before the intervention) or post hoc readouts (measured after the intervention), further categorized by scope (procedural, donor related or recipient related) and resolution (host, community and strain level), yielding testable hypotheses. Linking informative variables and their predictive performance to putative underlying ecological processes, we provide a comprehensive view of host- and microbiome-level determinants of strain dynamics following FMT with relevance to gut microbial ecology in the clinical context and beyond.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">A meta-analysis of strain dynamics after FMT, across diseases</h3><p>We analyzed a total of 1,492 fecal metagenomes collected in 316 time series of FMTs conducted for rCDI infection (<i>n</i> = 62 FMTs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Smillie, C. S. et al. Strain tracking reveals the determinants of bacterial engraftment in the human gut following fecal microbiota transplantation. Cell Host Microbe 23, 229–240 (2018)." href="#ref-CR26" id="ref-link-section-d32986547e1145">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Podlesny, D. et al. Metagenomic strain detection with SameStr: identification of a persisting core gut microbiota transferable by fecal transplantation. Microbiome 10, 53 (2022)." href="#ref-CR27" id="ref-link-section-d32986547e1145_1">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Aggarwala, V. et al. Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes. Nat. Microbiol. 6, 1309–1318 (2021)." href="/articles/s41591-022-01913-0#ref-CR28" id="ref-link-section-d32986547e1148">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Watson, A. R., Fuessel, J., Veseli, I. &amp; DeLongchamp, J. Z. Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut. Preprint at 
                  https://doi.org/10.1101/2021.03.02.433653
                  
                 (2021)." href="/articles/s41591-022-01913-0#ref-CR32" id="ref-link-section-d32986547e1151">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Moss, E. L. et al. Long-term taxonomic and functional divergence from donor bacterial strains following fecal microbiota transplantation in immunocompromised patients. PLoS ONE 12, e0182585 (2017)." href="/articles/s41591-022-01913-0#ref-CR36" id="ref-link-section-d32986547e1154">36</a></sup>), infection with extended-spectrum beta-lactamase-producing bacteria (ESBL, <i>n</i> = 59 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Singh, R. et al. Fecal microbiota transplantation against intestinal colonization by extended spectrum beta-lactamase producing Enterobacteriaceae: a proof of principle study. BMC Res. Notes 11, 190 (2018)." href="#ref-CR37" id="ref-link-section-d32986547e1161">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bar-Yoseph, H. et al. Oral capsulized fecal microbiota transplantation for eradication of carbapenemase-producing Enterobacteriaceae colonization with a metagenomic perspective. Clin. Infect. Dis. 73, e166–e175 (2020)." href="#ref-CR38" id="ref-link-section-d32986547e1161_1">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Leo, S. et al. Metagenomic characterization of gut microbiota of carriers of extended-spectrum beta-lactamase or carbapenemase-producing Enterobacteriaceae following treatment with oral antibiotics and fecal microbiota transplantation: results from a multicenter randomized trial. Microorganisms 8, 941 (2020)." href="/articles/s41591-022-01913-0#ref-CR39" id="ref-link-section-d32986547e1164">39</a></sup>)), MetS (<i>n</i> = 50 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Kootte, R. S. et al. Improvement of insulin sensitivity after lean donor feces in metabolic syndrome is driven by baseline intestinal imcrobiota composition. Cell Metab. 26, 611–619 (2017)." href="/articles/s41591-022-01913-0#ref-CR18" id="ref-link-section-d32986547e1172">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Li, S. S. et al. Durable coexistence of donor and recipient strains after fecal microbiota transplantation. Science 352, 586–589 (2016)." href="/articles/s41591-022-01913-0#ref-CR25" id="ref-link-section-d32986547e1175">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Koopen, A. M. et al. Effect of fecal microbiota transplantation combined with Mediterranean diet on insulin sensitivity in subjects with metabolic syndrome. Front. Microbiol. 12, 662159 (2021)." href="/articles/s41591-022-01913-0#ref-CR40" id="ref-link-section-d32986547e1178">40</a></sup>)), UC (<i>n</i> = 42 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lee, S. T. M. et al. Tracking microbial colonization in fecal microbiota transplantation experiments via genome-resolved metagenomics. Microbiome 5, 50 (2017)." href="/articles/s41591-022-01913-0#ref-CR29" id="ref-link-section-d32986547e1185">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rossen, N. G. et al. Findings from a randomized controlled trial of fecal transplantation for patients with ulcerative colitis. Gastroenterology 149, 110–118 (2015)." href="#ref-CR41" id="ref-link-section-d32986547e1188">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nusbaum, D. J. et al. Gut microbial and metabolomic profiles after fecal microbiota transplantation in pediatric ulcerative colitis patients. FEMS Microbiol. Ecol. 94, fiy133 (2018)." href="#ref-CR42" id="ref-link-section-d32986547e1188_1">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Damman, C. J. et al. Low level engraftment and improvement following a single colonoscopic administration of fecal microbiota to patients with ulcerative colitis. PLoS ONE 10, e0133925 (2015)." href="/articles/s41591-022-01913-0#ref-CR43" id="ref-link-section-d32986547e1191">43</a></sup>)), anti-PD1 therapy resistance in patients with melanoma (<i>n</i> = 37 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Baruch, E. N. et al. Fecal microbiota transplant promotes response in immunotherapy-refractory melanoma patients. Science 371, 602–609 (2021)." href="/articles/s41591-022-01913-0#ref-CR9" id="ref-link-section-d32986547e1198">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Davar, D. et al. Fecal microbiota transplant overcomes resistance to anti-PD-1 therapy in melanoma patients. Science 371, 595–602 (2021)." href="/articles/s41591-022-01913-0#ref-CR10" id="ref-link-section-d32986547e1201">10</a></sup>)), irritable bowel syndrome (IBS, <i>n</i> = 30 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Goll, R. et al. Effects of fecal microbiota transplantation in subjects with irritable bowel syndrome are mirrored by changes in gut microbiome. Gut Microbes 12, 1794263 (2020)." href="/articles/s41591-022-01913-0#ref-CR44" id="ref-link-section-d32986547e1209">44</a></sup>)), Crohn’s disease (<i>n</i> = 18 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Vaughn, B. P. et al. Increased intestinal microbial diversity following fecal microbiota transplant for active Crohn’s disease. Inflamm. Bowel Dis. 22, 2182–2190 (2016)." href="/articles/s41591-022-01913-0#ref-CR45" id="ref-link-section-d32986547e1216">45</a></sup>)), chemotherapy-induced diarrhea in patients with renal carcinoma (<i>n</i> = 10 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Ianiro, G. et al. Fecal microbiota transplantation for the treatment of diarrhoea induced by tyrosine-kinase inhibitors in patients with metastatic renal cell carcinoma. Nat. Commun. 11, 4333 (2020)." href="/articles/s41591-022-01913-0#ref-CR46" id="ref-link-section-d32986547e1223">46</a></sup>)), Tourette’s syndrome (<i>n</i> = 5 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Zhao, H.-J. et al. The efficacy of fecal microbiota transplantation for children with Tourette syndrome: a preliminary study. Front. Psychiatry 11, 554441 (2020)." href="/articles/s41591-022-01913-0#ref-CR47" id="ref-link-section-d32986547e1231">47</a></sup>)) and in healthy volunteers (<i>n</i> = 3 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Goloshchapov, O. V. et al. Long-term impact of fecal transplantation in healthy volunteers. BMC Microbiol. 19, 312 (2019)." href="/articles/s41591-022-01913-0#ref-CR48" id="ref-link-section-d32986547e1238">48</a></sup>)). Of these, 269 samples (from four independent cohorts) were metagenomically sequenced for this study (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">1</a>).</p><p>Full sample triads (donor, recipient pre FMT as baseline and at least one post-FMT sample) were available for 228 of the 283 allogenic FMT cases in our study; the remaining 33 FMTs in the dataset were autologous transfers, of the recipient’s own stool; 3 ± 3 post-FMT samples were available per time series, with a final sampling time point on average 159.4 days after the intervention (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">1</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">3</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01913-0#Sec11">Methods</a>).</p><p>We profiled 1,089 microbial species, including 144 previously undescribed, via pangenomes (the total set of identified genes for a microbial species) constructed from 47,548 newly built metagenome-assembled genomes (MAGs) and 25,037 high-quality reference genomes (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig1">1a</a> and <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01913-0#Sec11">Methods</a>). We compared the pre-FMT microbiome of recipients with their respective donors to identify single-nucleotide variants (determinant SNVs, as defined previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Li, S. S. et al. Durable coexistence of donor and recipient strains after fecal microbiota transplantation. Science 352, 586–589 (2016)." href="/articles/s41591-022-01913-0#ref-CR25" id="ref-link-section-d32986547e1266">25</a></sup>) and differences in gene content, and used these (meta)genomic markers to evaluate the fate of donor and recipient strains in post-FMT samples (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig1">1b,c</a>). For each species we classified outcomes as: donor colonization (that is, the post-FMT strain population was dominated by donor strains); recipient persistence (dominated by recipient strains); coexistence of conspecific donor and recipient strains; influx of ‘novel’ strains not detected in baseline samples (representing the expansion of low-abundance strains, or introduction of new strains post FMT); donor rejection (failure to engraft at detectable concentrations); and loss of all recipient strains (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig1">1c,d</a>, <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01913-0#Sec11">Methods</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">5</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Study design and workflow overview."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Study design and workflow overview.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="549"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, We analyzed a dataset of 316 FMT time series across ten disease indications and 22 cohorts, totaling 1,492 fecal metagenomes. Species pangenomes were built from reference genomes and newly generated MAGs and profiled across samples for taxonomic, functional and strain population composition, based on microbial SNVs and differential gene content. <b>b</b>, Each allogenic FMT was represented as a triad of donor pre-FMT (blue hues), recipient pre-FMT (yellow) and post-FMT (purple) samples; each sample’s strain population is indicated as an overlapping circle. <b>c</b>, FMT strain-level outcomes for each species were scored using patterns of determinant SNVs and gene content (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">5</a>). <b>d</b>, Ternary diagram of the strain population space for conspecific recipient strain persistence, donor strain colonization, donor–recipient coexistence and influx of novel strains.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec4">Donor strain colonization is independent of clinical outcome</h3><p>Summarized across all tracked species, the colonization and persistence of donor and recipient strains, respectively, varied greatly among allogenic FMT patients (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig2">2a,b</a>). We observed neither complete recipient strain turnover (loss of all strains) nor complete donor rejection (failure to colonize) in any analyzed FMT instance, although persistence of recipient strains or colonization by donor strains was very low in some patients. Outcomes varied depending on the presence of the species before FMT: takeover by donor strains (accounting for 18.0 ± 16.0% species post FMT) and persistence of recipient strains (11.3 ± 9.1%) occurred more frequently among species present in either donor or recipient, but not in both. In contrast, in cases where species were present in both donor and recipient before FMT, coexistence of donor and recipient strains (19.0 ± 11.8%) was the most frequent outcome compared with donor colonization (4.5 ± 4.0%) and recipient persistence (5.6 ± 5.2%). Among post-FMT strain populations, 41.5 ± 21.0% were attributable to novel strains or entirely novel species not present in either donor or recipient pre FMT (or previously below detection limits). Such major turnover towards novel strains was probably associated with the intervention itself, because novel or previously undetected strains accounted for 50 ± 10.1% in autologous FMTs.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Community-wide FMT outcomes vary across patients and indications."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Community-wide FMT outcomes vary across patients and indications.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="336"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Microbiome-level outcomes of 228 scorable allogenic FMT time series, summarized across all strain populations observed in donor and recipient (rec.). Fractions are normalized to the number of species observed in the recipient post FMT. <b>b</b>, Contextual data on indication, procedure and clinical outcome for each FMT time series in <b>a</b>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Takeover by donor and novel strains was characteristic of patients with rCDI or UC whereas MetS FMTs mostly resulted in conspecific strain coexistence, with varied outcomes in the other tested indications. Clinical response was not associated with strain-level dynamics for any indication; in other words, patient remission was not significantly linked to donor strain colonization or recipient strain displacement—for individual species and across all tracked species (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">1</a>). In particular, our data did not support earlier hypotheses that reinstatement of SCFA production is a hallmark of remission in UC and rCDI, because an increased carriage of gut metabolic modules (GMMs; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01913-0#Sec11">Methods</a>) for acetogenesis, propionigenesis and butyrogenesis following FMT did not correlate with clinical outcome.</p><h3 class="c-article__sub-heading" id="Sec5">Recipient, not donor, factors drive post-FMT strain dynamics</h3><p>To identify factors associated with colonization outcome, we trained a series of predictive machine learning models using cross-validated LASSO-regularized linear regression (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01913-0#Sec11">Methods</a>). Among possible predictors we distinguished ex ante variables (that is, knowable before the FMT intervention; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig3">3a</a>) from post hoc variables (measurable after FMT; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig3">3b</a>). Moreover, we categorized predictors based on variable scope (procedural, donor related and recipient related) and resolution (host, community and species level), totaling &gt;400 variables as regularization inputs (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">6</a>). We then built cross-validated models for individual predictor categories (for example, using procedural variables only), as well as combined models to assess the overall predictability of outcomes.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Drivers and determinants of FMT community-level outcomes."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Drivers and determinants of FMT community-level outcomes.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="425"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Ex ante predictability of microbial community-wide outcomes for individual FMTs (summarized across all trackable strain populations in a triad of donor, recipient pre-FMT and recipient post-FMT samples; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig2">2</a>) using cross-validated LASSO linear models with regularized subsets of different variable categories or a combination of all variables (‘full’ model) knowable before the intervention (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01913-0#Sec11">Methods</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">6</a>). Within each category, only the most relevant predictors are included. Predictive performance for each outcome index is shown as <i>R</i><sup>2</sup> on the left, and variable importance and directionality for the most predictive factors as cross-validated LASSO coefficients on the right. <b>b</b>, Association of FMT outcomes with LASSO-regularized sets of post hoc variables (measured after the intervention).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Using regularized combinations of ex ante variables, the fractions of species exhibiting post-FMT coexistence of donor and recipient strains and post-FMT recipient strain persistence were predictable with moderate accuracy (LASSO <i>R</i><sup>2</sup> = 0.58 and 0.49, respectively), with lower variation explained for colonization by donor (<i>R</i><sup>2</sup> = 0.34) and pre-FMT recipient strain resilience (<i>R</i><sup>2</sup> = 0.35; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig3">3a</a>). Interestingly, the fraction of donor strains that successfully took over was not well predicted (<i>R</i><sup>2</sup> = 0.1309).</p><p>To identify the major determinants of strain outcomes, we compared the accuracy of models that used restricted subsets of variables with those of full models (which chose from all variables). Models that were restricted to community diversity indices (including species richness) or species abundances in the recipient before FMT achieved similar accuracies, reflecting the importance of these two factors in predicting the fate of donor and recipient strains after FMT. Moreover, across all models, variables capturing recipient factors or donor–recipient microbiome complementarity (for example, community dissimilarity) were more predictive than donor factors. The most important predictors of strain-level outcome included recipient species richness and abundances of selected species in the recipient before FMT, in particular <i>Bacteroides uniformis</i>, <i>Bacteroides vulgatus</i> and one <i>Oscillibacter</i> species, which were positively associated with overall recipient strain persistence and coexistence). In contrast, models based on procedural, metabolic or donor species variables were less accurate (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig3">3a</a>, left). Notably, donor carriage of GMMs related to SCFA synthesis was not associated with increased strain colonization, contrary to previous findings<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Seekatz, A. M. et al. Restoration of short chain fatty acid and bile acid metabolism following fecal microbiota transplantation in patients with recurrent Clostridium difficile infection. Anaerobe 53, 64–73 (2018)." href="/articles/s41591-022-01913-0#ref-CR12" id="ref-link-section-d32986547e1462">12</a></sup>. However, high carriage of butyrogenesis genes in the recipient before FMT was moderately associated with overall strain persistence—that is, recipient communities with higher butyrogenesis potential were generally more resilient, further highlighting the role of the recipient microbiome in post-FMT strain dynamics.</p><p>In the study population used here, rCDI state was associated with a higher fraction of successfully colonizing donor strains in the post-FMT microbiome. However, we note that while &gt;90% of patients with rCDI in our dataset received antibiotics before intervention, most patients for other indications did not (or underwent extended washout periods), hence rCDI and the effect of antibiotics cannot be disentangled. Moreover, in full models choosing from all variables, higher species richness in the recipient and individual species abundances were more robust predictors for the persistence of recipient strains than rCDI state. This suggests that the high levels of donor strain colonization observed in patients with rCDI may be due in part to a more precarious microbial community (possibly instigated or exacerbated by antibiotic use), rather than being a disease-specific effect.</p><p>Models trained on post hoc variables were found to be highly accurate, in particular when describing donor colonization (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig3">3b</a>). As expected, the strength of community-wide compositional shifts in the recipient (Bray–Curtis dissimilarity and metabolic dissimilarity pre to post FMT) were associated with lower persistence of recipient strains. Interestingly, no individual species’ abundance post FMT was strongly associated with colonization outcome. However, successful colonization of particular species (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig3">3b</a>, right) was highly predictive of overall colonization of donor strains, in particular <i>B. uniformis</i>, <i>B. vulgatus</i>, several <i>Oscillospiraceae</i> sp. and <i>Lachnospiraceae</i> sp., including <i>Anaerostipes hadrus</i>. These might be considered indicator species, the successful engraftment of which is associated with an overall higher influx of donor strains.</p><h3 class="c-article__sub-heading" id="Sec6">Post-FMT strain outcomes are species specific and predictable</h3><p>Whereas the above analyses describe summarized outcomes across all tracked species, we next investigated the strain population dynamics within each species post FMT. For sufficient statistical power, we focused on the 307 species detected in &gt;50 allogenic FMTs across our study dataset (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig4">4</a> and Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">2</a>). Recipient persistence, donor colonization, coexistence and influx of novel strains were observed for all species, with no notable phylogenetic signal. We did not observe any species with consistent patterns of colonization (‘super-colonizers’) or persistence (‘super-persisters’) across all FMTs. However, we observed two broadly distinct types of post-FMT strain dynamics in conspecific FMT triads (that is, for species present in both donor and recipient before the intervention; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig4">4a</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">2</a>). Most species showed a strong propensity towards donor–recipient strain coexistence that was independent of initial strain abundances. Notably, these included prevalent commensals like <i>Bacteroides</i> sp., <i>Blautia</i> sp., <i>Dorea</i> sp.<i>, Ruminoccocus</i> sp. and <i>Faecalibacterium</i> sp. In contrast, for <i>Veillonella parvula</i>, several <i>Streptococcus</i> spp., <i>Eggerthella lenta</i>, <i>Akkermansia muciniphila</i> and <i>Prevotella copri</i>, strain populations strongly tended towards dominance of either donor, recipient or novel strains, with infrequent coexistence, indicating that these species may be inherently less prone to conspecific strain carriage within the same host.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Strain-level FMT outcomes vary between species but are predictable ex ante."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Strain-level FMT outcomes vary between species but are predictable ex ante.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="614"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Strain-level outcomes for selected species are shown for conspecific FMT triads—that is, time series where the focal species was present in both donor and recipient pre FMT. Outcomes are scored as recipient strain persistence (dominance by recipient strains, yellow), donor takeover (blue), donor–recipient coexistence (orange) or influx of novel or previously undetected strains (purple), as indicated in the schematic on the left. Each dot corresponds to one scored FMT. <b>b</b>, Stacked bars representing outcomes for each species across scorable FMTs, scaled to the number of FMTs where the species was observed in the recipient following the intervention. Dashed lines indicate averages for recipient strain persistence within taxonomic groups (<i>x</i> axis). Outcome frequencies across all species are summarized on the left. <b>c</b>, Frequency of colonization by donor or novel (previously undetected) strains per species, as subsets of the data in <b>b</b>. Averages per taxonomic group are represented by dotted lines. <b>d</b>, Prediction accuracies of LASSO models for different binarized FMT outcomes (indicated on the left; <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01913-0#Sec11">Methods</a>) as AUROC, averaged across cross-validation folds per species.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Strain-level FMT outcomes varied within each major taxonomic group, with no relevant differences between clades (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig4">4b,c</a>). Strains of facultatively aerobic species colonized less successfully (analysis of variance (ANOVA), <i>R</i><sup>2</sup> = 0.02, <i>P</i> = 0.002), whereas carriage of butyrogenesis (<i>R</i><sup>2</sup> = 0.026, <i>P</i> = 2 × 10<sup>−4</sup>) or propionigenesis (<i>R</i><sup>2</sup> = 0.008, <i>P</i> = 0.05) pathway genes or a generally saccharolytic (<i>R</i><sup>2</sup> = 0.046, <i>P</i> = 1.1 × 10<sup>−6</sup>) or proteolytic (<i>R</i><sup>2</sup> = 0.047, <i>P</i> = 8.5 × 10<sup>−7</sup>) metabolic setup was associated with higher colonization success.</p><p>To disentangle the factors contributing to post-FMT strain outcomes for each species, we built species-specific cross-validated logistic LASSO regression models using ex ante and post hoc sets of predictor variables, analogous to those discussed above (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig4">4d</a>). For each species we categorized strain-level outcomes, defining recipient resilience as events where recipient strains persisted (as dominant populations or coexisting with donor strains; yellow), donor colonization (donor strains successfully colonized as dominant or coexisting populations; light blue), donor takeover (donor strains become dominant; dark blue) and recipient turnover (dominance by donor strains and/or new or previously undetectable strains; purple). When training models using all available ex ante variables, recipient resilience (LASSO area under the curve (AUC) = 0.62 ± 0.13), donor colonization (0.58 ± 0.10) and donor takeover (0.65 ± 0.14) were predictable with moderate accuracy, with some variation within and between taxonomic clades (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig4">4d</a>). In contrast, recipient strain turnover (AUC = 0.94 ± 0.05) was predictable with high accuracy across almost all species, indicating that the displacement of resident strain populations in the recipient (not only by donor strain takeover, but by any means) may in general be a more deterministic process.</p><h3 class="c-article__sub-heading" id="Sec7">Recipient microbiome drives species-specific strain dynamics</h3><p>We built LASSO models that were restricted to different subcategories of predictor variables and compared their performance with full models trained on the entire complements of ex ante or post hoc variables (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig5">5a</a>). Models trained exclusively on recipient pre-FMT species abundances, on abundance and strain population characteristics of the focal species and, to a lesser degree, on microbiome community diversity variables achieved highest accuracies, comparable to those of full models. Notably, predictive power of individual recipient species was due almost entirely to exclusion effects, meaning that the enrichment of certain species in the recipient was associated with less donor takeover or recipient strain turnover of others, while facilitation effects did not have a contributing role. Models restricted to procedural factors (including disease indication), pre-FMT metabolic state or donor species abundances achieved much lower accuracies than full models, indicating that these variable groups were less predictive of strain-level outcomes. Overall, we observed similar trends for models trained on post hoc variables (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig5">5a</a>, right).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="Drivers and determinants of FMT strain-level outcomes for individual species."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: Drivers and determinants of FMT strain-level outcomes for individual species.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="650"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Logistic LASSO models were trained to predict FMT binarized outcomes (recipient resilience, yellow; recipient turnover, purple; donor takeover, blue) for <i>n</i> = 307 species across FMT time series, using different subsets of ex ante variables (knowable before the intervention). Each dot represents data for one species. Data are shown for full models (choosing from all available variables) and models trained on variable subsets categorized by type (procedural, community-level diversity and so on). Predictive performance of species models is shown as average AUROC across LASSO cross-validation folds in marginal box plots, ranging from 0.5 to 1.0; center line, median; box limits, upper and lower quartiles; whiskers, maxima/minima within 1.5× interquartile range from upper/lower quartiles. <b>b</b>, Variable importance across full models to predict takeover by donor strains. Each edge indicates the importance of a predictor variable (top row) when predicting donor takeover for a given species (bottom row). Dot size for predictors indicates summed variable importance across all species; dot size for species (bottom) indicates total number of relevant predictors. Edge color and width indicate direction and strength of the association, respectively. <b>c</b>, Variable importance for individual predictor categories, as subsets of the data in <b>b</b>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>For most species, we found that strain turnover could be accurately predicted using only two community-level microbiome diversity measures—species richness in the pre-FMT recipient and donor–recipient community dissimilarity, the main factors selected in models restricted to community diversity variables (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig5">5b</a>). Low richness and a strong compositional shift in the recipient microbiome relative to healthy donors are hallmarks of disease-associated microbiome states, and our data indicate that the strength of this diffuse imbalance, correlated to disease (such as rCDI or UC in our dataset) or other disturbances (for example, antibiotics pretreatment or bowel cleansing), is directly linked with FMT outcome in most species. In contrast, donor richness or functional redundancy, previously proposed to be relevant<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Tian, L. et al. Deciphering functional redundancy in the human microbiome. Nat. Commun. 11, 6217 (2020)." href="/articles/s41591-022-01913-0#ref-CR49" id="ref-link-section-d32986547e1704">49</a></sup>, were only subordinately predictive, if at all. Metabolic variables were likewise unreliable predictors. Community-wide butyrogenesis potential was negatively associated with turnover in the recipient (that is, strain populations were more resilient in recipients carrying high loads of butyrate production genes), but higher butyrogenesis levels in the donor did not correspondingly promote colonization. However, in full models for recipient strain turnover, these variables were superseded by indicator species in the recipient microbiome (see below) and focal species characteristics (in particular, recipient strain population diversity; Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig5">5b</a>).</p><p>The strongest predictor of takeover by donor strains was a high donor/recipient abundance ratio of a species (as suggested previously for rCDI<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Podlesny, D. et al. Metagenomic strain detection with SameStr: identification of a persisting core gut microbiota transferable by fecal transplantation. Microbiome 10, 53 (2022)." href="/articles/s41591-022-01913-0#ref-CR27" id="ref-link-section-d32986547e1714">27</a></sup>), indicating that the amount of incoming viable donor microbes (also referred to as propagule pressure) may provide a neutral baseline estimate for donor strain colonization success, in particular for species not present in the recipient pre FMT (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig5">5b,c</a>). In general, while the donor/recipient ratio was most predictive, the underlying signal was driven by species abundance (or absence) in the recipient microbiota, much less so in the donor microbiota. Intraspecific strain population properties—donor/recipient strain population dissimilarity and recipient (and, to a much lesser extent, donor) strain population diversity—were also highly predictive but effects were more nuanced: donor strain takeover was more likely in species with complementary strain populations between donor and recipient, while diverse recipient populations (not dominated by individual strains) were more resilient than uneven ones. Moreover, incoming species that were phylogenetically complementary to the recipient community (that is, adding novelty—for example, by filling an unoccupied niche) were more likely to colonize or turn over the resident population.</p><h3 class="c-article__sub-heading" id="Sec8">Resident ‘gatekeeper’ species inhibit donor strain engraftment</h3><p>Given that FMTs involve the pitting of the recipient’s residual microbial community against incoming microbiota from the donor, we specifically explored the impact of individual species on the engraftment of others by training models restricted to donor or recipient pre-FMT species abundances (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig5">5a</a>) and exploration of individual species’ relevance as predictors in full models (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig5">5b,c</a>). We extracted networks of engraftment inhibition and facilitation, associating the abundance of putative effector species in the donor and recipient with donor takeover events in focal species. The vast majority of interactions was inhibitive (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig5">5a–c</a>): for most species, higher abundance in both donor and recipient correlated negatively with engraftment of other species. These exclusion effects were stronger for the resident community of the recipient (AUC = 0.63 ± 0.14) than the donor (AUC = 0.53 ± 0.06).</p><p>Colonization inhibition was phylogenetically concentrated—that is, inhibitive interactions were more common between related species within the same clade than between clades (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig5">5B</a>). Bacteroidales in the recipient microbiota, in particular <i>B. uniformis</i>, <i>B. vulgatus, Alistipes shahii</i> and <i>Parabacteroides distasonis</i>, were among the strongest colonization inhibitors, but also included two of the most strongly inhibited species, <i>Bacteroides xylanisolvens</i> and <i>Bacteroides ovatus</i>. In other words, the enrichment of gatekeeper species such as Bacteroidales in the recipient microbiota inhibited colonization for a broad panel of species, and vice versa, in line with previous findings that subgroups of Bacteroidales are generally highly persistent also in healthy individuals<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Hildebrand, F. et al. Dispersal strategies shape persistence and evolution of human gut bacteria. Cell Host Microbe 29, 1167–1176 (2021)." href="/articles/s41591-022-01913-0#ref-CR50" id="ref-link-section-d32986547e1760">50</a></sup>. <i>Lactococcus lactis, Streptococcus salivarius</i> and <i>Dialister invisus</i> in the recipient were the foremost colonization facilitators. In contrast to colonization inhibition, facilitation typically affected phylogenetically distant species—for example, the facilitation of <i>Paraprevotella clara</i> and <i>Erysipelatoclostridium ramosum</i> colonization by recipient <i>Pauljensenia</i> sp. (an Actinobacterium) were among the strongest interactions observed across all species.</p><p>We observed few prominent predictive species in the donor microbiota, most notably <i>B. vulgatus</i> and <i>Evtepia gabavorous</i>. Facilitation and inhibition effects of donor species were generally limited and overall less predictive of colonization success, indicating that the donor microbiota has limited impact on colonization outcome beyond intraspecific strain dynamics.</p><h3 class="c-article__sub-heading" id="Sec9">Adaptive and neutral processes shape the post-FMT microbiome</h3><p>The accurate prediction of strain-level outcomes after FMT is informative beyond mere descriptive associations when construed through the lens of gut ecology: FMTs are community-level perturbation experiments, interpretable in a framework of invasion ecology and community assembly to identify processes and mechanisms that shape the microbiome<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Walter, J., Maldonado-Gómez, M. X. &amp; Martínez, I. To engraft or not to engraft: an ecological framework for gut microbiome modulation with live microbes. Curr. Opin. Biotechnol. 49, 129–139 (2018)." href="#ref-CR33" id="ref-link-section-d32986547e1797">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Schmidt, T. S. B., Raes, J. &amp; Bork, P. The human gut microbiome: from association to modulation. Cell 172, 1198–1215 (2018)." href="#ref-CR34" id="ref-link-section-d32986547e1797_1">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Xiao, Y., Angulo, M. T., Lao, S., Weiss, S. T. &amp; Liu, Y.-Y. An ecological framework to understand the efficacy of fecal microbiota transplantation. Nat. Commun. 11, 3329 (2020)." href="/articles/s41591-022-01913-0#ref-CR35" id="ref-link-section-d32986547e1800">35</a></sup>. We therefore linked the various tested variables in our models to putative underlying mechanisms (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig6">6</a>), categorized along a gradient from neutral/stochastic factors (for example, donor propagule pressure: the amount of incoming viable donor microbes) to adaptive/selective ones (for example, niche effects). We further distinguished recipient-specific, donor-specific and donor–recipient complementarity effects and organized variables by granularity, from host-level factors (for example, clinical or procedural) to the level of microbiome communities (overall composition and possible species interactions) and intraspecific (strain-level) effects.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="FMT strain-level outcomes are shaped by both neutral and adaptive processes."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6: FMT strain-level outcomes are shaped by both neutral and adaptive processes.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig6_HTML.png?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="619"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p>Each of the tested variables used to predict FMT outcome can be linked to putative underlying ecological processes, as suggested previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Walter, J., Maldonado-Gómez, M. X. &amp; Martínez, I. To engraft or not to engraft: an ecological framework for gut microbiome modulation with live microbes. Curr. Opin. Biotechnol. 49, 129–139 (2018)." href="/articles/s41591-022-01913-0#ref-CR33" id="ref-link-section-d32986547e1819">33</a></sup>. Factors are organized by scope (pertaining to the donor, recipient or donor–recipient complementarity, top) and resolution (host, community, species and strain level; left to right). Underlying ecological processes can be roughly ranked along the gradient, from neutral/stochastic to adaptive/selective; each process is illustrated with a toy example on the right. Circle size corresponds to average variable importance, calculated across all tested species from LASSO coefficients and overall model performance (less predictive models penalize variable importance). Recipient factors and, in particular, donor–recipient complementarity measures across all resolutions, were generally far more relevant to species-level outcome than donor factors. neg, negative; pos., positive; abd, abundance.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01913-0/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Factors pertaining to the recipient or to donor–recipient complementarity were far more relevant to FMT colonization outcome than donor readouts across all tested variables, and consistently across different species. In other words, as outlined above, the donor microbiome did not specifically influence colonization or turnover in its own right, but instead mattered only to the extent of its complementarity with the recipient microbiota. Donor/recipient abundance ratios were highly determinant of FMT outcome, interpretable as the balance between propagule pressure of incoming donor cells and native abundance of the residual recipient population, providing a baseline of how neutral mechanisms shape post-FMT communities. In this, exclusion effects by resident strains of the recipient were dominant—that is, depletion of the recipient’s microbiota is more relevant for successful colonization than a higher dosage of donor strains. In practice, this interplay may be modulated procedurally to some extent, for example, by the use of fresh versus frozen stool (impacting the viability of donor cells), FMT route (rectal or duodenal) or the purging of recipient communities via bowel preparations or antibiotic pretreatment, although these procedural variables were not in themselves robust predictors in our analysis, possibly because they were confounded with the individual studies included in the dataset.</p><p>Microbiome composition of recipients (but not their respective donors) was likewise relevant to FMT strain-level outcome: broad community depletion (low richness) and pronounced compositional differences in regard to healthy donors may indicate generally disturbed and precarious microbiomes that are less resistant to takeover by donor strains. Conversely, the residual enrichment of gatekeeper species, such as <i>B. uniformis</i> or <i>B. vulgatus</i>, was also negatively associated with colonization by donor strains, possibly indicating competitive exclusion processes and interspecific priority effects. While by design, causality cannot be inferred from our data, these results tie in with existing ecological theories on microbiome stability and resilience—for example, on tipping elements and critical transitions<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Lahti, L., Salojärvi, J., Salonen, A., Scheffer, M. &amp; de Vos, W. M. Tipping elements in the human intestinal ecosystem. Nat. Commun. 5, 4344 (2014)." href="/articles/s41591-022-01913-0#ref-CR51" id="ref-link-section-d32986547e1843">51</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Scheffer, M., Carpenter, S. R., Dakos, V. &amp; van Nes, E. H. Generic indicators of ecological resilience: inferring the chance of a critical transition. Annu. Rev. Ecol. Evol. Syst. 46, 145–167 (2015)." href="/articles/s41591-022-01913-0#ref-CR52" id="ref-link-section-d32986547e1846">52</a></sup>, community multistability leading to enterotypes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Gonze, D., Lahti, L., Raes, J. &amp; Faust, K. Multi-stability and the origin of microbial community types. ISME J. 11, 2159–2166 (2017)." href="/articles/s41591-022-01913-0#ref-CR53" id="ref-link-section-d32986547e1850">53</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Costea, P. I. et al. Enterotypes in the landscape of gut microbial community composition. Nat. Microbiol. 3, 8–16 (2018)." href="/articles/s41591-022-01913-0#ref-CR54" id="ref-link-section-d32986547e1853">54</a></sup>, priority<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Debray, R. et al. Priority effects in microbiome assembly. Nat. Rev. Microbiol. 20, 109–121 (2021)." href="/articles/s41591-022-01913-0#ref-CR55" id="ref-link-section-d32986547e1857">55</a></sup> or ‘Anna Karenina’ effects<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Zaneveld, J. R., McMinds, R. &amp; Thurber, R. V. Stress and stability: applying the Anna Karenina principle to animal microbiomes. Nat. Microbiol. 2, 17121 (2017)." href="/articles/s41591-022-01913-0#ref-CR56" id="ref-link-section-d32986547e1862">56</a></sup>. We found limited evidence for colonization facilitation across species boundaries, both in donor and recipient. Likewise, our data did not support a strong role for community-wide metabolic states: neither general metabolic setup nor specific metabolic modules such as SCFA production in donor or recipient greatly impacted FMT outcomes.</p><p>The strongest effects toward donor strain colonization emerged at species and strain level. Incoming species were more likely to colonize if they were phylogenetically or metabolically complementary to the residual community, implying that they were able to take over unoccupied niches. Colonization success was associated with complementarity specifically to the local community. High conspecific diversity in the donor and low diversity in the recipient were also linked with engraftment success: recipient populations dominated by single strains were less resilient, and donor strains from more diverse panels were more likely to colonize, probably due to strain-level-limiting similarity effects. Indeed, conspecific donor strain populations colonized more successfully if they were dissimilar to recipient strains, indicating strong inhibitive intraspecific priority effects.</p><p>However, we note once more that the colonization of individual species was predictable with only moderate accuracy, irrespective of the variable sets used—unlike residual strain population turnover, which was highly predictable. This implies that colonization success may be stochastic to a large extent.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec10-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">Discussion</h2><div class="c-article-section__content" id="Sec10-content"><p>Fecal microbiota transplantations are clinical procedures that can also be thought of as complex in natura perturbation experiments, pitting gut microbial communities of the donor against those of the recipient. An FMT is considered to be clinically successful if it triggers patient remission or recovery, whereas success from an ecological perspective is the extent to which the donor’s microbiota can colonize in the recipient. Given that FMT targets the gut microbiome, engraftment and clinical success are expected to correlate, implying that successful microbiome modulation mediates clinical effects. However, this hypothesis had not previously been systematically tested and is indeed not supported by our data. In our meta-study of 316 FMTs, clinical success was associated neither with colonization by donor strains, displacement of recipient species nor the reinstatement of specific functions (such as SCFA synthesis) for any of the studied disease indications. To some extent, this is in line with previous observations that autologous FMTs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Basson, A. R., Zhou, Y., Seo, B., Rodriguez-Palacios, A. &amp; Cominelli, F. Autologous fecal microbiota transplantation for the treatment of inflammatory bowel disease. Transl. Res. 226, 1–11 (2020)." href="/articles/s41591-022-01913-0#ref-CR57" id="ref-link-section-d32986547e1882">57</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="de Groot, P. et al. Fecal microbiota transplantation halts progression of human new-onset type 1 diabetes in a randomised controlled trial. Gut 70, 92–105 (2021)." href="/articles/s41591-022-01913-0#ref-CR58" id="ref-link-section-d32986547e1885">58</a></sup> or even transfers of sterile-filtered fecal water<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Ott, S. J. et al. Efficacy of sterile fecal filtrate transfer for treating patients with Clostridium difficile infection. Gastroenterology 152, 799–811 (2017)." href="/articles/s41591-022-01913-0#ref-CR59" id="ref-link-section-d32986547e1889">59</a></sup> can be efficacious. Our data do not rule out more subtle links, in particular given our limited sample size per indication and differences between FMT protocols across studies, but a clear role of donor microbiota colonization in shaping clinical responses did not emerge. We did observe overall higher levels of donor strain colonization in patients suffering from rCDI or UC, coinciding with higher clinical response rates in these diseases compared with others in our dataset. However, this was arguably due to overall more perturbed microbiome states associated with these diseases (possibly instigated by antibiotic treatment regimes) that outweighed disease-specific effects: we found no significant differences in strain-level outcomes between clinical responders and nonresponders to FMT.</p><p>Understanding microbiome-level FMT outcomes is both clinically relevant (for example, for informed donor selection or to avoid possible adverse effects) and more generally informative of ecological processes shaping the gut microbiome. All studied species exhibited all FMT outcomes, depending on context; we did not find strong evidence that any species was inherently more invasive or resilient than others. Rather, fine-scale intraspecific strain population structure and diversity, as well as donor–recipient strain population complementarity, determined resilience, coexistence and colonization, although we noted that while the majority of species tended towards conspecific donor–recipient strain coexistence, a smaller subset of species generally gravitated towards dominance by either recipient or donor strains or those undetectable at baseline. Interactions between species were less relevant, but clearly structured: several gatekeeper species in the recipient, in particular of the genus <i>Bacteroides</i>, inhibited colonization by other, phylogenetically unrelated species whereas colonization facilitation across species boundaries was scarce.</p><p>We found that the turnover of recipient strains was very accurately predictable for almost all studied species, using a consistent and surprisingly small selection of ex ante microbiome variables. In contrast, our models achieved only moderate predictive accuracies when predicting takeover by donor strains, indicating that colonization is, to a large extent, stochastic or influenced by other factors outside the scope of our study, such as viral or eukaryotic microbiome members, recipient immune state, medication or reduced viability of anaerobic donor fecal cells following the intervention.</p><p>Recipient factors consistently outweighed donor factors in driving FMT strain-level outcomes. Thus, our data did not support the super-donor hypothesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Wilson, B. C., Vatanen, T., Cutfield, W. S. &amp; O’Sullivan, J. M. The super-donor phenomenon in fecal microbiota transplantation. Front. Cell. Infect. Microbiol. 9, 2 (2019)." href="/articles/s41591-022-01913-0#ref-CR15" id="ref-link-section-d32986547e1905">15</a></sup> which states that certain donor microbiome properties are crucial to colonization and, by proxy, clinical success. Rather, we found that complementarity of donor and recipient microbiomes promoted donor colonization and recipient turnover. This phenomenon was observed across microbial resolutions, from community-level effects to conspecific strain population dissimilarity. Indeed, strain-level diversity and complementarity were the strongest determinants of FMT outcome, with relevance to rational donor selection in clinical practice<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Duvallet, C. et al. Framework for rational donor selection in fecal microbiota transplant clinical trials. PloS ONE 14, e0222881 (2019)." href="/articles/s41591-022-01913-0#ref-CR16" id="ref-link-section-d32986547e1909">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Xiao, Y., Angulo, M. T., Lao, S., Weiss, S. T. &amp; Liu, Y.-Y. An ecological framework to understand the efficacy of fecal microbiota transplantation. Nat. Commun. 11, 3329 (2020)." href="/articles/s41591-022-01913-0#ref-CR35" id="ref-link-section-d32986547e1912">35</a></sup>. Beyond screening for donor health, matching of donors to recipients based on microbiome complementarity at community, species and, in particular, strain levels may increase colonization success, make clinical outcomes more predictable and reduce adverse effects.</p><p>Our data suggest that the gut microbiome is shaped by both neutral and adaptive processes post FMT, reconciling previous reports<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Podlesny, D. et al. Metagenomic strain detection with SameStr: identification of a persisting core gut microbiota transferable by fecal transplantation. Microbiome 10, 53 (2022)." href="/articles/s41591-022-01913-0#ref-CR27" id="ref-link-section-d32986547e1920">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Watson, A. R., Fuessel, J., Veseli, I. &amp; DeLongchamp, J. Z. Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut. Preprint at 
                  https://doi.org/10.1101/2021.03.02.433653
                  
                 (2021)." href="/articles/s41591-022-01913-0#ref-CR32" id="ref-link-section-d32986547e1923">32</a></sup>. We found that limits to gut microbiome resilience at community, species and strain level can be defined by a relatively small set of measurable variables that point to distinct underlying processes. The (complementary) interplay between propagule pressure and residual species abundance provided a neutral baseline for colonization although, again, recipient effects outweighed donor effects. At the same time, our data also suggested niche effects, in particular at the level of complementary intraspecific strain populations, although no consistently adaptive traits emerged in the analysis. Previous hypotheses pertaining to the importance of metabolic capabilities such as SCFA synthesis were not supported, although we note that the inference of SCFA biosynthesis pathways from metagenomic data remains challenging and does not capture putatively differential expression of SCFA synthesis genes.</p><p>By design, our study is predominantly descriptive and only probes potential mechanisms underlying our observations to the extent of interpreting them in an ecological context. Moreover, our study focuses on the bacterial and archaeal microbiota (not accounting for viruses and eukaryotes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Bojanova, D. P. &amp; Bordenstein, S. R. Fecal transplants: what is being transferred? PLoS Biol. 14, e1002503 (2016)." href="/articles/s41591-022-01913-0#ref-CR60" id="ref-link-section-d32986547e1930">60</a></sup>) and is limited by the relatively small available sample size for some disease indications in our dataset, by the technical, procedural and physiological heterogeneity between cohorts and by the inherent detection limits of metagenomic strain calls. Nevertheless, our core findings were robust in spite of these sources of variation and may thus inform the clinical use of FMT in several ways, in particular if microbiome modulation is a desired endpoint beyond alleviation or remission of symptoms. Patients may be stratified before the intervention based on surprisingly crude, robust and easily obtainable microbiome readouts, such as community richness and high-level composition, or with regard to the presence of gatekeeper species associated with overall microbiome resilience. The relevance of donor selection, in contrast, appears mostly limited to the extent of the donor’s (strain-level) complementarity to the recipient. Tuning of procedural parameters (antibiotic pretreatment, stool preparation, dosage, FMT route, dietary intake of donors and so on) may mainly impact recipient microbiome resilience, and an overall more resilient response (excluding, of course, target pathogens to be displaced) is often desirable. Both inhibition and facilitation of colonization across species boundaries were surprisingly sparse and mild, with few exceptions, indicating that the targeted colonization or turnover of individual species may be achievable mostly independent of residual and cotransferred communities, minimizing collateral effects on the recipient’s microbiota.</p><p>Our results indicate that microbiome dynamics following FMT are impacted by defined parameters that are tunable in clinical practice, thus supporting the notion that predictable and efficacious microbiome modulation using personalized probiotic mixtures, rather than entire complex fecal samples, is possible and may profit from an ecological perspective. In particular, our findings suggest that the targeted depletion of selected microbes in the recipient, with concurrent introduction of diverse strain populations of the same species rather than a single strain, presents a promising approach to enhancing colonization and turnover in the recipient, although links to clinical outcomes remain to be established. Thus, levering of both neutral and relevant adaptive ecological processes may pave the way towards targeted modulatory interventions on the gut microbiome, personalized to patients, with predictable microbiome-level outcomes.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec11-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec11">Methods</h2><div class="c-article-section__content" id="Sec11-content"><h3 class="c-article__sub-heading" id="Sec12">Data overview</h3><p>The study dataset comprised 22 independent cohorts recruited in centers in the United States, the Netherlands and Australia, with a total of 316 FMTs conducted in 311 patients suffering from rCDI (<i>n</i> = 62 FMTs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Smillie, C. S. et al. Strain tracking reveals the determinants of bacterial engraftment in the human gut following fecal microbiota transplantation. Cell Host Microbe 23, 229–240 (2018)." href="#ref-CR26" id="ref-link-section-d32986547e1952">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Podlesny, D. et al. Metagenomic strain detection with SameStr: identification of a persisting core gut microbiota transferable by fecal transplantation. Microbiome 10, 53 (2022)." href="#ref-CR27" id="ref-link-section-d32986547e1952_1">27</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Aggarwala, V. et al. Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes. Nat. Microbiol. 6, 1309–1318 (2021)." href="/articles/s41591-022-01913-0#ref-CR28" id="ref-link-section-d32986547e1955">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Watson, A. R., Fuessel, J., Veseli, I. &amp; DeLongchamp, J. Z. Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut. Preprint at 
                  https://doi.org/10.1101/2021.03.02.433653
                  
                 (2021)." href="/articles/s41591-022-01913-0#ref-CR32" id="ref-link-section-d32986547e1958">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Moss, E. L. et al. Long-term taxonomic and functional divergence from donor bacterial strains following fecal microbiota transplantation in immunocompromised patients. PLoS ONE 12, e0182585 (2017)." href="/articles/s41591-022-01913-0#ref-CR36" id="ref-link-section-d32986547e1961">36</a></sup>), infection with ESBL (<i>n</i> = 59 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Singh, R. et al. Fecal microbiota transplantation against intestinal colonization by extended spectrum beta-lactamase producing Enterobacteriaceae: a proof of principle study. BMC Res. Notes 11, 190 (2018)." href="#ref-CR37" id="ref-link-section-d32986547e1968">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bar-Yoseph, H. et al. Oral capsulized fecal microbiota transplantation for eradication of carbapenemase-producing Enterobacteriaceae colonization with a metagenomic perspective. Clin. Infect. Dis. 73, e166–e175 (2020)." href="#ref-CR38" id="ref-link-section-d32986547e1968_1">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Leo, S. et al. Metagenomic characterization of gut microbiota of carriers of extended-spectrum beta-lactamase or carbapenemase-producing Enterobacteriaceae following treatment with oral antibiotics and fecal microbiota transplantation: results from a multicenter randomized trial. Microorganisms 8, 941 (2020)." href="/articles/s41591-022-01913-0#ref-CR39" id="ref-link-section-d32986547e1971">39</a></sup>)), MetS (<i>n</i> = 50 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Kootte, R. S. et al. Improvement of insulin sensitivity after lean donor feces in metabolic syndrome is driven by baseline intestinal imcrobiota composition. Cell Metab. 26, 611–619 (2017)." href="/articles/s41591-022-01913-0#ref-CR18" id="ref-link-section-d32986547e1979">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Li, S. S. et al. Durable coexistence of donor and recipient strains after fecal microbiota transplantation. Science 352, 586–589 (2016)." href="/articles/s41591-022-01913-0#ref-CR25" id="ref-link-section-d32986547e1982">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Koopen, A. M. et al. Effect of fecal microbiota transplantation combined with Mediterranean diet on insulin sensitivity in subjects with metabolic syndrome. Front. Microbiol. 12, 662159 (2021)." href="/articles/s41591-022-01913-0#ref-CR40" id="ref-link-section-d32986547e1985">40</a></sup>)), UC (<i>n</i> = 42 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lee, S. T. M. et al. Tracking microbial colonization in fecal microbiota transplantation experiments via genome-resolved metagenomics. Microbiome 5, 50 (2017)." href="/articles/s41591-022-01913-0#ref-CR29" id="ref-link-section-d32986547e1992">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Rossen, N. G. et al. Findings from a randomized controlled trial of fecal transplantation for patients with ulcerative colitis. Gastroenterology 149, 110–118 (2015)." href="#ref-CR41" id="ref-link-section-d32986547e1995">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Nusbaum, D. J. et al. Gut microbial and metabolomic profiles after fecal microbiota transplantation in pediatric ulcerative colitis patients. FEMS Microbiol. Ecol. 94, fiy133 (2018)." href="#ref-CR42" id="ref-link-section-d32986547e1995_1">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Damman, C. J. et al. Low level engraftment and improvement following a single colonoscopic administration of fecal microbiota to patients with ulcerative colitis. PLoS ONE 10, e0133925 (2015)." href="/articles/s41591-022-01913-0#ref-CR43" id="ref-link-section-d32986547e1998">43</a></sup>)), anti-PD1 therapy resistance in patients with melanoma (<i>n</i> = 37 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Baruch, E. N. et al. Fecal microbiota transplant promotes response in immunotherapy-refractory melanoma patients. Science 371, 602–609 (2021)." href="/articles/s41591-022-01913-0#ref-CR9" id="ref-link-section-d32986547e2005">9</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Davar, D. et al. Fecal microbiota transplant overcomes resistance to anti-PD-1 therapy in melanoma patients. Science 371, 595–602 (2021)." href="/articles/s41591-022-01913-0#ref-CR10" id="ref-link-section-d32986547e2008">10</a></sup>)), IBS (<i>n</i> = 30 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Goll, R. et al. Effects of fecal microbiota transplantation in subjects with irritable bowel syndrome are mirrored by changes in gut microbiome. Gut Microbes 12, 1794263 (2020)." href="/articles/s41591-022-01913-0#ref-CR44" id="ref-link-section-d32986547e2016">44</a></sup>)), Crohn’s disease (<i>n</i> = 18 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Vaughn, B. P. et al. Increased intestinal microbial diversity following fecal microbiota transplant for active Crohn’s disease. Inflamm. Bowel Dis. 22, 2182–2190 (2016)." href="/articles/s41591-022-01913-0#ref-CR45" id="ref-link-section-d32986547e2023">45</a></sup>)), chemotherapy-induced diarrhea in patients with renal carcinoma (<i>n</i> = 10 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Ianiro, G. et al. Fecal microbiota transplantation for the treatment of diarrhoea induced by tyrosine-kinase inhibitors in patients with metastatic renal cell carcinoma. Nat. Commun. 11, 4333 (2020)." href="/articles/s41591-022-01913-0#ref-CR46" id="ref-link-section-d32986547e2030">46</a></sup>)), Tourette’s syndrome (<i>n</i> = 5 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Zhao, H.-J. et al. The efficacy of fecal microbiota transplantation for children with Tourette syndrome: a preliminary study. Front. Psychiatry 11, 554441 (2020)." href="/articles/s41591-022-01913-0#ref-CR47" id="ref-link-section-d32986547e2038">47</a></sup>) and in healthy volunteers (<i>n</i> = 3 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Goloshchapov, O. V. et al. Long-term impact of fecal transplantation in healthy volunteers. BMC Microbiol. 19, 312 (2019)." href="/articles/s41591-022-01913-0#ref-CR48" id="ref-link-section-d32986547e2045">48</a></sup>)). On average, 4.11 recipient stool samples were available per FMT time series, including baseline samples taken before the intervention (pre-FMT). Overall, 7.9 Terabases (Tb) of sequencing data were analyzed across 1,492 fecal metagenomes, of which 269 (for 76 time series) were generated as part of the present study (for cohorts UC_NL, ESBL_NL, MetS_NL_1 and div_AU).</p><p>Three cohorts (UC_NL, MetS_NL_1 and MetS_NL_Koopen) were randomized controlled trials during which a subset of patients received autologous FMTs (transplantation of the recipient’s own stool, <i>n</i> = 33 FMTs). All other FMTs (<i>n</i> = 283) were allogenic, using stool donors. For 228 FMT time series, a full complement of donor baseline, recipient baseline and at least one recipient post-FMT sample were available after filtering.</p><p>A full description of all cohorts is provided in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">1</a>, detailed information per FMT time series in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">2</a> and per-sample information in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">3</a>.</p><h3 class="c-article__sub-heading" id="Sec13">Sample collection, processing and metagenomic sequencing</h3><p>Study design and fecal sample collection for cohorts MetS_NL_1 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Kootte, R. S. et al. Improvement of insulin sensitivity after lean donor feces in metabolic syndrome is driven by baseline intestinal imcrobiota composition. Cell Metab. 26, 611–619 (2017)." href="/articles/s41591-022-01913-0#ref-CR18" id="ref-link-section-d32986547e2078">18</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Li, S. S. et al. Durable coexistence of donor and recipient strains after fecal microbiota transplantation. Science 352, 586–589 (2016)." href="/articles/s41591-022-01913-0#ref-CR25" id="ref-link-section-d32986547e2081">25</a></sup>), UC_NL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Rossen, N. G. et al. Findings from a randomized controlled trial of fecal transplantation for patients with ulcerative colitis. Gastroenterology 149, 110–118 (2015)." href="/articles/s41591-022-01913-0#ref-CR41" id="ref-link-section-d32986547e2085">41</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Fuentes, S. et al. Microbial shifts and signatures of long-term remission in ulcerative colitis after fecal microbiota transplantation. ISME J. 11, 1877–1889 (2017)." href="/articles/s41591-022-01913-0#ref-CR61" id="ref-link-section-d32986547e2088">61</a></sup> and ESBL_NL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Singh, R. et al. Fecal microbiota transplantation against intestinal colonization by extended spectrum beta-lactamase producing Enterobacteriaceae: a proof of principle study. BMC Res. Notes 11, 190 (2018)." href="/articles/s41591-022-01913-0#ref-CR37" id="ref-link-section-d32986547e2092">37</a></sup> were described previously. rCDI_AU and UC_AU samples were obtained from a single-center, proof-of-concept, parallel and controlled study in collaboration with the Centre for Digestive Diseases (Sydney, Australia), which aimed to assess donor microbiota implantation in two patients with CDI and three with UC up to 28 days following a 2-day fecal microbiota transplantation infusion via transcolonoscopy and rectal enema. The study is registered with the Australian New Zealand Clinical Trials Registry under ACTRN12614000503628 (Universal Trial no, U1111-1156-5909). Written, informed participant consent and ethical approval were obtained via the Centre for Digestive Diseases Human Research Ethics Committee. Deidentified participant data relevant to the study are provided in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">3</a>.</p><p>For cohorts MetS_NL_1 and UC_NL, fecal DNA extraction was described in the original studies. DNA from ESBL_NL samples was extracted using the GNOME DNA Isolation Kit (MP Biomedicals) with the following minor modifications: cell lysis/denaturation was performed (30 min, 55 °C) before protease digestion was carried out overnight (55 °C), and RNAse digestion (50 μl, 30 min, 55 °C) was performed after mechanical lysis. After final precipitation, DNA was resuspended in TE buffer and stored at −20 °C for further analysis.</p><p>Metagenomic sequencing libraries for MetS_NL_1, UC_NL, ESBL_NL and div_AU samples were prepared to a target insert size of 350–400 base pairs (bp) on a Biomek FXp Dual Hybrid with high-density layout adapters, orbital shaker, static peltier and shaking peltier (Beckman Coulter) and a robotic PCR cycler (Biometra), using SPRIworks HT kits (Beckman Coulter) according to the supplier’s recommendation, with the following modifications: 500 ng of DNA initially, adapter dilution 1:25, kit chemical dilution 1:1 in process. For samples with low-input DNA concentrations, libraries were instead prepared manually using NEBNext Ultra II DNA Library Prep kits with NEBNext Singleplex primers. Libraries were sequenced on an Illumina HiSeq 4000 platform with 2 × 150-bp paired-end reads.</p><h3 class="c-article__sub-heading" id="Sec14">Public datasets</h3><p>Based on a literature search, 18 datasets on FMT cohorts that met the following criteria were included in the study: (1) public availability of metagenomic sequencing data in January 2022; (2) sufficient available description to unambiguously match donors and recipients per FMT time series; and (3) no restrictions on data reuse. They were included in this study as RCDI_US_Smillie (<i>n</i> = 22 FMT time series<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Smillie, C. S. et al. Strain tracking reveals the determinants of bacterial engraftment in the human gut following fecal microbiota transplantation. Cell Host Microbe 23, 229–240 (2018)." href="/articles/s41591-022-01913-0#ref-CR26" id="ref-link-section-d32986547e2119">26</a></sup>), RCDI_US_Aggarwala (<i>n</i> = 14 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Aggarwala, V. et al. Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes. Nat. Microbiol. 6, 1309–1318 (2021)." href="/articles/s41591-022-01913-0#ref-CR28" id="ref-link-section-d32986547e2126">28</a></sup>)), RCDI_US_Watson (<i>n</i> = 10 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Watson, A. R., Fuessel, J., Veseli, I. &amp; DeLongchamp, J. Z. Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut. Preprint at 
                  https://doi.org/10.1101/2021.03.02.433653
                  
                 (2021)." href="/articles/s41591-022-01913-0#ref-CR32" id="ref-link-section-d32986547e2134">32</a></sup>)), RCDI_US_Podlesny (<i>n</i> = 8 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Podlesny, D. et al. Metagenomic strain detection with SameStr: identification of a persisting core gut microbiota transferable by fecal transplantation. Microbiome 10, 53 (2022)." href="/articles/s41591-022-01913-0#ref-CR27" id="ref-link-section-d32986547e2141">27</a></sup>)), RCDI_US_Moss (<i>n</i> = 6 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Moss, E. L. et al. Long-term taxonomic and functional divergence from donor bacterial strains following fecal microbiota transplantation in immunocompromised patients. PLoS ONE 12, e0182585 (2017)." href="/articles/s41591-022-01913-0#ref-CR36" id="ref-link-section-d32986547e2148">36</a></sup>)), MetS_NL_Koopen (<i>n</i> = 24 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Koopen, A. M. et al. Effect of fecal microbiota transplantation combined with Mediterranean diet on insulin sensitivity in subjects with metabolic syndrome. Front. Microbiol. 12, 662159 (2021)." href="/articles/s41591-022-01913-0#ref-CR40" id="ref-link-section-d32986547e2156">40</a></sup>)), UC_US_Damman (<i>n</i> = 6 (ref.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Damman, C. J. et al. Low level engraftment and improvement following a single colonoscopic administration of fecal microbiota to patients with ulcerative colitis. PLoS ONE 10, e0133925 (2015)." href="/articles/s41591-022-01913-0#ref-CR43" id="ref-link-section-d32986547e2163">43</a></sup>)), UC_US_Nusbaum (<i>n</i> = 4 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Nusbaum, D. J. et al. Gut microbial and metabolomic profiles after fecal microbiota transplantation in pediatric ulcerative colitis patients. FEMS Microbiol. Ecol. 94, fiy133 (2018)." href="/articles/s41591-022-01913-0#ref-CR42" id="ref-link-section-d32986547e2170">42</a></sup>)), UC_US_Lee (<i>n</i> = 2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Lee, S. T. M. et al. Tracking microbial colonization in fecal microbiota transplantation experiments via genome-resolved metagenomics. Microbiome 5, 50 (2017)." href="/articles/s41591-022-01913-0#ref-CR29" id="ref-link-section-d32986547e2178">29</a></sup>)), CD_US_Vaughn (<i>n</i> = 18 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Vaughn, B. P. et al. Increased intestinal microbial diversity following fecal microbiota transplant for active Crohn’s disease. Inflamm. Bowel Dis. 22, 2182–2190 (2016)." href="/articles/s41591-022-01913-0#ref-CR45" id="ref-link-section-d32986547e2185">45</a></sup>)), ABXR_div_Leo (<i>n</i> = 26 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Leo, S. et al. Metagenomic characterization of gut microbiota of carriers of extended-spectrum beta-lactamase or carbapenemase-producing Enterobacteriaceae following treatment with oral antibiotics and fecal microbiota transplantation: results from a multicenter randomized trial. Microorganisms 8, 941 (2020)." href="/articles/s41591-022-01913-0#ref-CR39" id="ref-link-section-d32986547e2192">39</a></sup>)), ABXR_IS_BarYoseph (<i>n</i> = 14 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Bar-Yoseph, H. et al. Oral capsulized fecal microbiota transplantation for eradication of carbapenemase-producing Enterobacteriaceae colonization with a metagenomic perspective. Clin. Infect. Dis. 73, e166–e175 (2020)." href="/articles/s41591-022-01913-0#ref-CR38" id="ref-link-section-d32986547e2200">38</a></sup>)), IBS_NO_Goll (<i>n</i> = 30 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Goll, R. et al. Effects of fecal microbiota transplantation in subjects with irritable bowel syndrome are mirrored by changes in gut microbiome. Gut Microbes 12, 1794263 (2020)." href="/articles/s41591-022-01913-0#ref-CR44" id="ref-link-section-d32986547e2207">44</a></sup>)), MEL_US_Davar (<i>n</i> = 27 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Davar, D. et al. Fecal microbiota transplant overcomes resistance to anti-PD-1 therapy in melanoma patients. Science 371, 595–602 (2021)." href="/articles/s41591-022-01913-0#ref-CR10" id="ref-link-section-d32986547e2214">10</a></sup>)), MEL_US_Baruch (<i>n</i> = 10<sup>9</sup>), REN_IT_Ianiro (<i>n</i> = 10 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Ianiro, G. et al. Fecal microbiota transplantation for the treatment of diarrhoea induced by tyrosine-kinase inhibitors in patients with metastatic renal cell carcinoma. Nat. Commun. 11, 4333 (2020)." href="/articles/s41591-022-01913-0#ref-CR46" id="ref-link-section-d32986547e2227">46</a></sup>)), TOU_CN_Zhao (<i>n</i> = 5 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="Zhao, H.-J. et al. The efficacy of fecal microbiota transplantation for children with Tourette syndrome: a preliminary study. Front. Psychiatry 11, 554441 (2020)." href="/articles/s41591-022-01913-0#ref-CR47" id="ref-link-section-d32986547e2234">47</a></sup>)) and CTR_RU_Goloshchapov (<i>n</i> = 3 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Goloshchapov, O. V. et al. Long-term impact of fecal transplantation in healthy volunteers. BMC Microbiol. 19, 312 (2019)." href="/articles/s41591-022-01913-0#ref-CR48" id="ref-link-section-d32986547e2242">48</a></sup>)). Contextual data, including donor–recipient matchings and information about clinical response, were curated from the study publications and, in some cases, kindly amended by the studies’ original authors on request (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">1</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">3</a>).</p><h3 class="c-article__sub-heading" id="Sec15">Metagenomic data processing and taxonomic and functional profiling</h3><p>Metagenomic reads were quality trimmed to remove base calls with a Phred score of &lt;25. Reads were then discarded if they were &lt;45 nucleotides or if they mapped to the human genome (GRCh38.p10) with at least 90% identity over 45 nucleotides. This processing was performed using NGLess<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Coelho, L. P. et al. NG-meta-profiler: fast processing of metagenomes using NGLess, a domain-specific language. Microbiome 7, 84 (2019)." href="/articles/s41591-022-01913-0#ref-CR62" id="ref-link-section-d32986547e2260">62</a></sup>. Taxonomic profiles per sample were obtained using mOTUs v.2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Milanese, A. et al. Microbial abundance, activity and population genomic profiling with mOTUs2. Nat. Commun. 10, 1014 (2019)." href="/articles/s41591-022-01913-0#ref-CR63" id="ref-link-section-d32986547e2264">63</a></sup>). For functional profiling, reads were mapped against the Global Microbial Gene Catalog v.1 gut subcatalogue (gmgc.embl.de<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 64" title="Coelho, L. P. et al. Towards the biogeography of prokaryotic genes. Nature 601, 252–256 (2022)." href="/articles/s41591-022-01913-0#ref-CR64" id="ref-link-section-d32986547e2268">64</a></sup>) with a minimum match length of 45 nucleotides with at least 97% identity, and summarized based on antimicrobial resistance gene (ARG) annotations and Kyoto Encyclopedia of Genes and Genomes orthologs (KOs) via eggNOG annotations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Huerta-Cepas, J. et al. EggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. Nucleic Acids Res. 47, D309–D314 (2019)." href="/articles/s41591-022-01913-0#ref-CR65" id="ref-link-section-d32986547e2272">65</a></sup>. Based on the resulting KO profiles, GMMs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Vieira-Silva, S. et al. Species–function relationships shape ecological properties of the human gut microbiome. Nat. Microbiol. 1, 16088 (2016)." href="/articles/s41591-022-01913-0#ref-CR66" id="ref-link-section-d32986547e2276">66</a></sup> were quantified in each sample using omixer-rpmR (v.0.3.2)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Darzi, Y., Falony, G., Vieira-Silva, S. &amp; Raes, J. Towards biome-specific analysis of meta-omics data. ISME J. 10, 1025–1028 (2016)." href="/articles/s41591-022-01913-0#ref-CR67" id="ref-link-section-d32986547e2281">67</a></sup>. Taxonomic and GMM profiles per sample, normalized by read depth, are available in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">7</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">8</a>.</p><h3 class="c-article__sub-heading" id="Sec16">MAGs</h3><p>We demarcated MAGs from samples of studies MetS_NL_1, UC_NL, ABXR_NL, div_AU, RCDI_US_Smillie, RCDI_US_Moss, UC_US_Damman, UC_US_Nusbaum, UC_US_Lee and CD_US_Vaughn using several complementary strategies to obtain both high resolution from sample-specific assemblies and deep coverage of lowly abundant species from coassemblies of multiple samples. Unless otherwise indicated, all tools in the following were run with default parameters.</p><p>To generate single-sample MAGs, fecal metagenomes were assembled individually using metaSPAdes v.3.12.0 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Nurk, S., Meleshko, D., Korobeynikov, A. &amp; Pevzner, P. A. MetaSPAdes: a new versatile metagenomic assembler. Genome Res. 27, 824–834 (2017)." href="/articles/s41591-022-01913-0#ref-CR68" id="ref-link-section-d32986547e2303">68</a></sup>), reads were mapped back to contigs using bwa-mem v.0.7.17 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Preprint at 
                  https://doi.org/10.48550/arXiv.1303.3997
                  
                 (2013)." href="/articles/s41591-022-01913-0#ref-CR69" id="ref-link-section-d32986547e2307">69</a></sup>) and contigs were binned using metaBAT v.2.12.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Kang, D. D. et al. MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies. PeerJ. 7, e7359 (2019)." href="/articles/s41591-022-01913-0#ref-CR70" id="ref-link-section-d32986547e2311">70</a></sup>). Multisample MAGs were built for each cohort separately. Reads were first coassembled using megahit v.1.1.3 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Li, D., Liu, C.-M., Luo, R., Sadakane, K. &amp; Lam, T.-W. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. Bioinformatics 31, 1674–1676 (2015)." href="/articles/s41591-022-01913-0#ref-CR71" id="ref-link-section-d32986547e2315">71</a></sup>) and mapped back to contigs using bwa-mem v.0.7.17. Coassembled contigs were then binned using both CONCOCT v.0.5.0 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Alneberg, J. et al. Binning metagenomic contigs by coverage and composition. Nat. Methods 11, 1144–1146 (2014)." href="/articles/s41591-022-01913-0#ref-CR72" id="ref-link-section-d32986547e2319">72</a></sup>) and metaBAT v.2.12.1. The resulting coassembled MAG sets were further refined using DAS TOOL<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Sieber, C. M. K. et al. Recovery of genomes from metagenomes via a dereplication, aggregation and scoring strategy. Nat. Microbiol. 3, 836–843 (2018)." href="/articles/s41591-022-01913-0#ref-CR73" id="ref-link-section-d32986547e2324">73</a></sup> and metaWRAP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Uritskiy, G. V., DiRuggiero, J. &amp; Taylor, J. MetaWRAP—a flexible pipeline for genome-resolved metagenomic data analysis. Microbiome 6, 158 (2018)." href="/articles/s41591-022-01913-0#ref-CR74" id="ref-link-section-d32986547e2328">74</a></sup>. In total, 47,548 MAGs were demarcated using these five approaches (single-sample MAGs, multisample coassembled CONCOCT, metaBAT2, DAS TOOL and metaWRAP MAGs). In addition, we included 25,037 high-quality reference genomes from the proGenomes database<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Mende, D. R. et al. ProGenomes: a resource for consistent functional and taxonomic annotations of prokaryotic genomes. Nucleic Acids Res. 45, D529–D534 (2017)." href="/articles/s41591-022-01913-0#ref-CR75" id="ref-link-section-d32986547e2332">75</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Mende, D. R. et al. ProGenomes2: an improved database for accurate and consistent habitat, taxonomic and functional annotations of prokaryotic genomes. Nucleic Acids Res. 48, D621–D625 (2020)." href="/articles/s41591-022-01913-0#ref-CR76" id="ref-link-section-d32986547e2335">76</a></sup> in downstream analyses.</p><p>Genome quality was estimated using CheckM<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. &amp; Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. Genome Res. 25, 1043–1055 (2015)." href="/articles/s41591-022-01913-0#ref-CR77" id="ref-link-section-d32986547e2342">77</a></sup> and GUNC v.0.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Orakov, A. et al. GUNC: detection of chimerism and contamination in prokaryotic genomes. Genome Biol. 22, 178 (2021)." href="/articles/s41591-022-01913-0#ref-CR78" id="ref-link-section-d32986547e2346">78</a></sup>), and all genomes were taxonomically classified using GTDB-tk<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 79" title="Chaumeil, P.-A., Mussig, A. J., Hugenholtz, P. &amp; Parks, D. H. GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database. Bioinformatics 36, 1925–1927 (2019)." href="/articles/s41591-022-01913-0#ref-CR79" id="ref-link-section-d32986547e2350">79</a></sup>. Open reading frames (ORFs) were predicted using prodigal<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 80" title="Hyatt, D. et al. Prodigal: prokaryotic gene recognition and translation initiation site identification. BMC Bioinformatics 11, 119 (2010)." href="/articles/s41591-022-01913-0#ref-CR80" id="ref-link-section-d32986547e2354">80</a></sup> and annotated via prokka workflow v.1.14.6 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 81" title="Seemann, T. Prokka: rapid prokaryotic genome annotation. Bioinformatics 30, 2068–2069 (2014)." href="/articles/s41591-022-01913-0#ref-CR81" id="ref-link-section-d32986547e2358">81</a></sup>). Orthologs to known gene families were detected using eggNOG-mapper v.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 82" title="Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. Mol. Biol. Evol. 34, 2115–2122 (2017)." href="/articles/s41591-022-01913-0#ref-CR82" id="ref-link-section-d32986547e2363">82</a></sup>). ARGs were annotated using a workflow combining information from databases CARD v.3.0.0 (via rgi v.4.2.4 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 83" title="Alcock, B. P. et al. CARD 2020: antibiotic resistome surveillance with the comprehensive antibiotic resistance database. Nucleic Acids Res. 48, D517–D525 (2020)." href="/articles/s41591-022-01913-0#ref-CR83" id="ref-link-section-d32986547e2367">83</a></sup>) and ResFams v.1.2.2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 84" title="Gibson, M. K., Forsberg, K. J. &amp; Dantas, G. Improved annotation of antibiotic resistance determinants reveals microbial resistomes cluster by ecology. ISME J. 9, 207–216 (2015)." href="/articles/s41591-022-01913-0#ref-CR84" id="ref-link-section-d32986547e2371">84</a></sup>), as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Mende, D. R. et al. ProGenomes2: an improved database for accurate and consistent habitat, taxonomic and functional annotations of prokaryotic genomes. Nucleic Acids Res. 48, D621–D625 (2020)." href="/articles/s41591-022-01913-0#ref-CR76" id="ref-link-section-d32986547e2375">76</a></sup>. The ‘specI’ set of 40 near-universal single-copy marker genes were detected in each genome using fetchMG<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 85" title="Mende, D. R., Sunagawa, S., Zeller, G. &amp; Bork, P. Accurate and universal delineation of prokaryotic species. Nat. Publ. Group 10, 881–884 (2013)." href="/articles/s41591-022-01913-0#ref-CR85" id="ref-link-section-d32986547e2379">85</a></sup>.</p><p>The full set of generated MAGs and contextual data are available via Zenodo (DOI <a href="https://zenodo.org/record/5534163#.YuWR3y8RrzA">10.5281/zenodo.5534163</a> (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Schmidt, T. S. B. et al. Drivers and determinants of strain dynamics following fecal microbiota transplantation. Zenodo 
                  https://doi.org/10.5281/ZENODO.5534163
                  
                 (2021)." href="/articles/s41591-022-01913-0#ref-CR86" id="ref-link-section-d32986547e2393">86</a></sup>)).</p><h3 class="c-article__sub-heading" id="Sec17">Genome clustering, species metapangenomes and phylogeny</h3><p>Genomes were clustered into species-level groups using an ‘open-reference’ approach in multiple steps. Initial prefiltering using lenient quality criteria (CheckM-estimated completeness ≥70%, contamination ≤25%; additional criteria were applied downstream) removed 57.7% of MAGs. The remaining 20,093 MAGs were mapped to the clustered proGenomes v.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Mende, D. R. et al. ProGenomes: a resource for consistent functional and taxonomic annotations of prokaryotic genomes. Nucleic Acids Res. 45, D529–D534 (2017)." href="/articles/s41591-022-01913-0#ref-CR75" id="ref-link-section-d32986547e2405">75</a></sup>) and mOTUs v.2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Milanese, A. et al. Microbial abundance, activity and population genomic profiling with mOTUs2. Nat. Commun. 10, 1014 (2019)." href="/articles/s41591-022-01913-0#ref-CR63" id="ref-link-section-d32986547e2409">63</a></sup>) taxonomic marker gene databases using MAPseq v.1.2.3 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 87" title="Rodrigues, J. F. M., Schmidt, S. B. T., Tackmann, J. &amp; von Mering, C. MAPseq: highly efficient k-mer search with confidence estimates, for rRNA sequence analysis. Bioinformatics 33, 3808–3810 (2017)." href="/articles/s41591-022-01913-0#ref-CR87" id="ref-link-section-d32986547e2413">87</a></sup>). A total of 17,720 MAGs were confidently assigned to a ref-mOTU (specI cluster) or meta-mOTU based on the following criteria: (1) detection of at least 20% of the screened taxonomic marker genes and (2) a majority of markers assigning to the same mOTU at a conservative MAPseq confidence threshold of ≥0.9.</p><p>In an independent approach, quality-filtered MAGs and reference genomes were also clustered by average nucleotide identity (ANI) using a modified and scalable reimplementation of the dRep workflow<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 88" title="Olm, M. R., Brown, C. T., Brooks, B. &amp; Banfield, J. F. DRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication. ISME J. 11, 2864–2868 (2017)." href="/articles/s41591-022-01913-0#ref-CR88" id="ref-link-section-d32986547e2420">88</a></sup>. Using pairwise distances computed with mash v.2.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 89" title="Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. Genome Biol. 17, 132 (2016)." href="/articles/s41591-022-01913-0#ref-CR89" id="ref-link-section-d32986547e2424">89</a></sup>), sequences were first preclustered to 90% mash-ANI using the single-linkage algorithm, asserting that all genome pairs sharing ≥90% mash-ANI were grouped together. Each mash precluster was then resolved to 95 and 99% average linkage ANI clusters using fastANI v.1.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 90" title="Jain, C., Rodriguez-R, L. M., Phillippy, A. M., Konstantinidis, K. T. &amp; Aluru, S. High throughput ANI analysis of 90 K prokaryotic genomes reveals clear species boundaries. Nat. Commun. 9, 5114 (2018)." href="/articles/s41591-022-01913-0#ref-CR90" id="ref-link-section-d32986547e2428">90</a></sup>). For each cluster, a representative genome was picked as either the corresponding reference specI cluster representative in the proGenomes database or the MAG with the highest dRep score (calculated based on estimated completeness and contamination). Genome partitions based on 95% average linkage ANI clustering and specI marker gene mappings matched almost perfectly, at an adjusted Rand index of &gt;0.99. We therefore defined a total of 1,089 species-level clusters (‘species’) from our dataset (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">4</a>), primarily based on marker gene mappings to precomputed ref-mOTUs (or specI clusters, <i>n</i> = 295) and meta-mOTUs (<i>n</i> = 528), and as 95% average linkage ANI clusters for genomes that did not map to either of these databases (<i>n</i> = 233).</p><p>Species pangenomes were generated by clustering all genes within each species-level cluster at 95% amino acid identity, using Roary 3.12.0 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 91" title="Page, A. J. et al. Roary: rapid large-scale prokaryote pan genome analysis. Bioinformatics 31, 3691–3693 (2015)." href="/articles/s41591-022-01913-0#ref-CR91" id="ref-link-section-d32986547e2448">91</a></sup>). Spurious and putatively contaminant gene clusters (as introduced by misbinned contigs in MAGs) were removed by asserting that the underlying gene sequences originated (1) from a reference genome in the proGenomes database or (2) from at least two independent MAGs, assembled from distinct samples or studies. To account for incomplete genomes, ‘extended core genes’ were defined as gene clusters present in &gt;80% of genomes in a species-level cluster. If too few gene clusters satisfied this criterion, as was the case for some pangenomes containing many incomplete MAGs, the 50 most prevalent gene clusters were used instead. Representative sequences for each gene cluster were picked as ORFs originating from specI representative genomes (that is, high-quality reference genomes), or otherwise as the longest ORF in the cluster.</p><p>A phylogenetic tree of species-level cluster representatives was inferred based on the ‘mOTU’ set of ten near-universal marker genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 63" title="Milanese, A. et al. Microbial abundance, activity and population genomic profiling with mOTUs2. Nat. Commun. 10, 1014 (2019)." href="/articles/s41591-022-01913-0#ref-CR63" id="ref-link-section-d32986547e2455">63</a></sup>. Marker genes were aligned in amino acid sequence space across all species using Muscle v.3.8.31 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 92" title="Edgar, R. C. MUSCLE: a multiple sequence alignment method with reduced time and space complexity. BMC Bioinformatics 5, 113 (2004)." href="/articles/s41591-022-01913-0#ref-CR92" id="ref-link-section-d32986547e2459">92</a></sup>), concatenated and then used to construct a species tree with FastTree2 (v.2.1.11)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 93" title="Price, M. N., Dehal, P. S. &amp; Arkin, A. P. FastTree 2–approximately maximum-likelihood trees for large alignments. PLoS ONE 5, e9490 (2010)." href="/articles/s41591-022-01913-0#ref-CR93" id="ref-link-section-d32986547e2463">93</a></sup> with default parameters.</p><h3 class="c-article__sub-heading" id="Sec18">Inference of microbial strain populations</h3><p>Metagenomic reads for each sample were mapped against gene cluster representative sequences for all species pangenomes using bwa-mem v.0.7.17 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Preprint at 
                  https://doi.org/10.48550/arXiv.1303.3997
                  
                 (2013)." href="/articles/s41591-022-01913-0#ref-CR69" id="ref-link-section-d32986547e2475">69</a></sup>). Mapped reads were filtered for matches of ≥45 bp and ≥97% sequence identity, sorted and filtered against multiple mappings using samtools v.1.7 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 94" title="Li, H. et al. The Sequence Alignment/Map format and SAMtools. Bioinformatics 25, 2078–2079 (2009)." href="/articles/s41591-022-01913-0#ref-CR94" id="ref-link-section-d32986547e2479">94</a></sup>). Horizontal (‘breadth’) and vertical (‘depth’) coverage of each gene cluster in each sample were calculated using bedtools v.2.27.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 95" title="Quinlan, A. R. &amp; Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. Bioinformatics 26, 841–842 (2010)." href="/articles/s41591-022-01913-0#ref-CR95" id="ref-link-section-d32986547e2483">95</a></sup>).</p><p>A species was considered present in a sample if at least three mOTU taxonomic marker genes were confidently detected either via the mOTU v.2 profiler (for specI clusters and meta-mOTUs) or based on pangenome-wide read mappings (for non-mOTU species-level clusters). Gene clusters within each pangenome were considered present in a sample if (1) the species was detectable (see above), (2) horizontal coverage exceeded 100 bp and 20% of the representative gene’s length and (3) average vertical coverage exceeded 0.5. Gene clusters were considered confidently absent if they did not attract any mappings in samples where the species’ set of extended core genes (see above) was covered at &gt;1 median vertical coverage (that is, present with high confidence). Using these criteria, strain population-specific gene content profiles were computed for each species in each sample.</p><p>Raw microbial SNVs were called from uniquely mapping reads using metaSNV v.1.0.3 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 96" title="Costea, P. I. et al. MetaSNV: a tool for metagenomic strain level analysis. PLoS ONE 12, e0182392 (2017)." href="/articles/s41591-022-01913-0#ref-CR96" id="ref-link-section-d32986547e2493">96</a></sup>) with permissive parameters (-c 10 -t 2 -p 0.001 -d 1000000). Candidate SNVs were retained if they were supported by two or more reads each in two or more samples in which the focal gene cluster was confidently detected (see above), before differential downstream filtering. At multiallelic positions the frequency of each observed allele (A, C, G, T) was normalized by the total read depth for all alleles.</p><p>Based on these data, strain populations were represented based on both their specific gene content profile and SNV profile in each sample.</p><p>Each species’ local strain population diversity (SPD) and allele distances (AD) between strain populations across samples were estimated as follows. SPD was calculated based on the inverse Simpson index of allele frequencies <i>p</i><sub>(ACGT)</sub> at each variant position <i>i</i> in the extended core genome (n<sub>var</sub>), normalized by total horizontal coverage (number of covered positions) cov<sub>hor</sub>:</p><div id="Equa" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$${\mathrm{SPD}} = \frac{{\mathop {\sum}\nolimits_{i = 1}^{n_{{\mathrm{var}}}} {\left( {p_{\mathrm{A}}^2 + p_{\mathrm{C}}^2 + p_{\mathrm{G}}^2 + p_{\mathrm{T}}^2} \right)^{ - 1} - 1} }}{{{\mathrm{cov}}_{{\mathrm{hor}}}}}$$</span></div></div><p>Thus defined, SPD can be interpreted as the average effective number of nondominant alleles in a strain population. SPD ranges between 0 (only one dominant strain detected—that is, no multiallelic positions) and 3 (all four possible alleles present at equal proportions at each variant position). Normalization by total horizontal coverage, cov<sub>hor</sub> of the extended core genome ensures that values are comparable between samples even if a species’ coverage in a sample is incomplete.</p><p>Intraspecific ADs between strain populations across samples were calculated as the average Euclidean distance between observed allele frequencies at variant positions in the species’ extended core genome, requiring at least 20 variant positions with shared coverage between samples. If a species was not observed in a sample, ADs to that sample were set to 1.</p><h3 class="c-article__sub-heading" id="Sec19">Quantification of strain-level outcomes</h3><p>Colonization by donor strains, persistence of recipient strains and influx of novel strains (environmental or previously below detection limit) in the recipient microbiome following FMT were quantified for every species based on determinant microbial SNVs and gene content profiles using an approach extending previous work<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Li, S. S. et al. Durable coexistence of donor and recipient strains after fecal microbiota transplantation. Science 352, 586–589 (2016)." href="/articles/s41591-022-01913-0#ref-CR25" id="ref-link-section-d32986547e2717">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 97" title="Schmidt, T. S. B. et al. Extensive transmission of microbes along the gastrointestinal tract. eLife 8, e42693 (2019)." href="/articles/s41591-022-01913-0#ref-CR97" id="ref-link-section-d32986547e2720">97</a></sup>. In total, 261 FMT time series (228 allogenic and 33 autologous transfers) for which a donor baseline (in allogenic FMTs; ‘D’), a recipient pre-FMT baseline (‘R’) and at least one recipient post-FMT (‘P’) sample were available were taken into account, and each FMT was represented as a D-R-P sample triad. If available, multiple time points post FMT were scored independently. By definition, because no donor samples were available for autologous FMTs, recipient pre-FMT samples were used instead. An overview of potential strain-level FMT outcomes is provided in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig1">1c,d</a>.</p><p>For each D-R-P sample triad, conspecific strain dynamics were calculated if a species was observed in all three samples (see above) with at least 100 informative (determinant) variant positions either covered with two or more reads or confidently absent (see below). Donor determinant alleles were defined as variants unique to the donor (D) relative to the recipient pre-FMT (R) sample, and vice versa. Post-FMT determinant alleles were defined as variants unique in P relative to both D and R. Given that intraspecific fecal strain populations are often heterogeneous—that is, consist of more than one strain per species—multiple observed alleles at the same variant position were taken into account. In addition, if a gene containing a putative variant position was absent from a sample although the species’ extended core genome was detected, the variant was considered ‘confidently absent’ and treated as informative (and potentially determinant) as well, thereby taking into account differential gene content between strains.</p><p>The fractions of donor and recipient strains post FMT were quantified based on the detection of donor- and recipient-determinant variants across all informative positions in the P sample. The fraction of novel strains (environmental or previously below detection limit in donor and recipient) was quantified as the fraction of post-FMT determinant variants. Based on these three readouts (fraction of donor, recipient and novel strains) and cutoffs previously established by Li et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Li, S. S. et al. Durable coexistence of donor and recipient strains after fecal microbiota transplantation. Science 352, 586–589 (2016)." href="/articles/s41591-022-01913-0#ref-CR25" id="ref-link-section-d32986547e2733">25</a></sup>, FMT outcomes were scored categorically as ‘donor colonization’, ‘recipient persistence’, ‘donor–recipient coexistence’ or ‘influx of novel (previously undetected) strains’ for every species (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">5</a>).</p><p>In addition to conspecific strain dynamics (that is, where a species was present in D, R and P), we also quantified FMT outcomes that involved the acquisition or loss of entire strain populations. For example, if a species was present in the recipient at baseline but not post FMT, this was considered a ‘species loss’ event. See Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01913-0#Fig1">1c</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">5</a> for a full overview of how different FMT outcome scenarios were scored.</p><p>To assert the accuracy of our approach, we simulated FMT time series by shuffling (1) the donor sample, (2) the recipient pre-FMT sample or (3) both. Randomizations were stratified by subject (accounting for the fact that some donors were used in multiple FMTs and that some recipients received repeated treatments) and geography. For each observed D-R-P sample triad, we simulated ten triads per each of the above setups.</p><p>Outcomes were further summarized across species by calculating a series of strain population-level metrics for each FMT, defined as follows.</p><p>Persistence index: average fraction of persistent recipient strains among all species observed post FMT (that is, fraction of post-FMT strain populations attributable to recipient baseline strains).</p><p>Colonization index: average fraction of donor strains among all species post FMT.</p><h3 class="c-article__sub-heading" id="Sec20">Modeling and prediction of FMT outcomes</h3><p>We explored a large set of covariates as putative predictor variables for FMT outcomes, grouped into the following categories: (1) host clinical and procedural variables (for example, FMT indication, pre-FMT bowel preparation, FMT route and so on); (2) community-level taxonomic diversity (species richness, community composition and so on); (3) community-level metabolic profiles (abundance of specific pathways); (4) abundance profiles of individual species; (5) strain-level outcomes for other species in the system; and (6) focal species characteristics, including strain-level diversity; see Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">6</a> for a full list of covariates and their definitions. We further classified covariates as either predictive ex ante variables (that is, knowable before the FMT is conducted) or post hoc variables (that is, pertaining to the post-FMT state, or the relation between pre- and post-FMT states).</p><p>We built two types of model to predict FMT strain-level outcomes based on these covariates: (1) FMT-wide models, using summary outcome metrics across all species in a time series (persistence index, colonization index; see above) as response variables; and (2) per-species models for 307 species observed in ≥50 FMTs, using each species’ strain-level outcome in every scored time series as response variable. Unless otherwise indicated, the last available time point for each FMT time series was used. Models were built for each covariate category separately, as well as for combinations of all ex ante and all post hoc variables, respectively.</p><p>Given that the number of covariates greatly exceeded the number of available FMT time series, and that several covariates were correlated with each other (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">3</a>), FMT outcomes were modeled using ten times fivefold cross-validated LASSO-regularized regression, as implemented in the R package glmnet (v.4.1.3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 98" title="Friedman, J., Hastie, T. &amp; Tibshirani, R. Regularization paths for generalized linear models via coordinate descent. J. Stat. Softw. 33, 1–22 (2010)." href="/articles/s41591-022-01913-0#ref-CR98" id="ref-link-section-d32986547e2782">98</a></sup>. Regression coefficients were chosen at one standard error from the cross-validated minimum lambda value and averaged across validation folds.</p><p>Linear LASSO regression was used to model outcomes with continuous response variables, both for FMT-wide outcomes (persistence index and soon) and for the fraction of colonizing, persisting and coexisting strains per species across FMTs. For linear models, <i>R</i><sup>2</sup> of predictions on test sets was averaged across validation folds. Moreover, logistic LASSO regression was used to additionally model binarized FMT outcomes per species, defined as recipient strain resilience, recipient strain turnover and donor strain takeover, based on further summarizing outcome categories in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">5</a>. For logistic models, accuracy was assessed as area under the receiver operating characteristic curve (AUROC) averaged across validation folds.</p><h3 class="c-article__sub-heading" id="Sec21">Statistical analyses</h3><p>Association of clinical outcomes (excluding a subset of cohorts for which clinical success was not reported; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">3</a>) with FMT strain-level outcomes was tested using Wilcoxon tests (responders versus nonresponders), and also by sequential ANOVA on linear regression models (accounting for additional variables), in each case followed by Benjamini–Hochberg correction for multiple hypothesis tests. Differences in strain-level outcomes between species across taxonomic clades and inferred species phenotypes were tested using ANOVA on linear regression models.</p><h3 class="c-article__sub-heading" id="Sec22">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>Raw metagenomic sequencing data have been uploaded to the European Nucleotide Archive under accession nos. <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB46777">PRJEB46777</a>, <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB46778">PRJEB46778</a>, <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB46779">PRJEB46779</a> and <a href="https://www.ebi.ac.uk/ena/data/view/PRJEB46780">PRJEB46780</a>. The full list of included publicly available datasets, including accession numbers and associated PMIDs, is available in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM3">1</a>. Contextual data are available in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">2</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01913-0#MOESM1">3</a>. MAGs (<a href="https://doi.org/10.5281/zenodo.5534163">https://doi.org/10.5281/zenodo.5534163</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 86" title="Schmidt, T. S. B. et al. Drivers and determinants of strain dynamics following fecal microbiota transplantation. Zenodo 
                  https://doi.org/10.5281/ZENODO.5534163
                  
                 (2021)." href="/articles/s41591-022-01913-0#ref-CR86" id="ref-link-section-d32986547e2949">86</a></sup> and source data (<a href="https://doi.org/10.5281/zenodo.6611040">https://doi.org/10.5281/zenodo.6611040</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 99" title="Schmidt, T. S. B. et al. Analysis data, “Drivers and Determinants of Strain Dynamics Following Fecal Microbiota Transplantation”. Zenodo 
                  https://doi.org/10.5281/ZENODO.5534163
                  
                 (2021)." href="/articles/s41591-022-01913-0#ref-CR99" id="ref-link-section-d32986547e2960">99</a></sup> are available for download via Zenodo.</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>Analysis code is available via github (<a href="https://github.com/grp-bork/fmt_metastudy">https://github.com/grp-bork/fmt_metastudy</a>).</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Borody, T. J. et al. Bacteriotherapy using fecal flora: toying with human motions. <i>J. Clin. Gastroenterol.</i> <b>38</b>, 475–483 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1097/01.mcg.0000128988.13808.dc" data-track-item_id="10.1097/01.mcg.0000128988.13808.dc" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1097%2F01.mcg.0000128988.13808.dc" aria-label="Article reference 1" data-doi="10.1097/01.mcg.0000128988.13808.dc">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15220681" aria-label="PubMed reference 1">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Bacteriotherapy%20using%20fecal%20flora%3A%20toying%20with%20human%20motions&amp;journal=J.%20Clin.%20Gastroenterol.&amp;doi=10.1097%2F01.mcg.0000128988.13808.dc&amp;volume=38&amp;pages=475-483&amp;publication_year=2004&amp;author=Borody%2CTJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Rossen, N. G. et al. Fecal microbiota transplantation as novel therapy in gastroenterology: a systematic review. <i>World J. Gastroenterol.</i> <b>21</b>, 5359–5371 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3748/wjg.v21.i17.5359" data-track-item_id="10.3748/wjg.v21.i17.5359" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3748%2Fwjg.v21.i17.5359" aria-label="Article reference 2" data-doi="10.3748/wjg.v21.i17.5359">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25954111" aria-label="PubMed reference 2">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4419078" aria-label="PubMed Central reference 2">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplantation%20as%20novel%20therapy%20in%20gastroenterology%3A%20a%20systematic%20review&amp;journal=World%20J.%20Gastroenterol.&amp;doi=10.3748%2Fwjg.v21.i17.5359&amp;volume=21&amp;pages=5359-5371&amp;publication_year=2015&amp;author=Rossen%2CNG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Hanssen, N. M. J., de Vos, W. M. &amp; Nieuwdorp, M. Fecal microbiota transplantation in human metabolic diseases: from a murky past to a bright future? <i>Cell Metab.</i> <b>33</b>, 1098–1110 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cmet.2021.05.005" data-track-item_id="10.1016/j.cmet.2021.05.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cmet.2021.05.005" aria-label="Article reference 3" data-doi="10.1016/j.cmet.2021.05.005">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1KhtLnP" aria-label="CAS reference 3">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34077717" aria-label="PubMed reference 3">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplantation%20in%20human%20metabolic%20diseases%3A%20from%20a%20murky%20past%20to%20a%20bright%20future%3F&amp;journal=Cell%20Metab.&amp;doi=10.1016%2Fj.cmet.2021.05.005&amp;volume=33&amp;pages=1098-1110&amp;publication_year=2021&amp;author=Hanssen%2CNMJ&amp;author=Vos%2CWM&amp;author=Nieuwdorp%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Gough, E., Shaikh, H. &amp; Manges, A. R. Systematic review of intestinal microbiota transplantation (fecal bacteriotherapy) for recurrent <i>Clostridium difficile</i> infection. <i>Clin. Infect. Dis.</i> <b>53</b>, 994–1002 (2011).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/cid/cir632" data-track-item_id="10.1093/cid/cir632" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fcid%2Fcir632" aria-label="Article reference 4" data-doi="10.1093/cid/cir632">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22002980" aria-label="PubMed reference 4">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20review%20of%20intestinal%20microbiota%20transplantation%20%28fecal%20bacteriotherapy%29%20for%20recurrent%20Clostridium%20difficile%20infection&amp;journal=Clin.%20Infect.%20Dis.&amp;doi=10.1093%2Fcid%2Fcir632&amp;volume=53&amp;pages=994-1002&amp;publication_year=2011&amp;author=Gough%2CE&amp;author=Shaikh%2CH&amp;author=Manges%2CAR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">van Nood, E. et al. Duodenal infusion of donor feces for recurrent <i>Clostridium difficile</i>. <i>N. Engl. J. Med.</i> <b>368</b>, 407–415 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa1205037" data-track-item_id="10.1056/NEJMoa1205037" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa1205037" aria-label="Article reference 5" data-doi="10.1056/NEJMoa1205037">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23323867" aria-label="PubMed reference 5">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Duodenal%20infusion%20of%20donor%20feces%20for%20recurrent%20Clostridium%20difficile&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1205037&amp;volume=368&amp;pages=407-415&amp;publication_year=2013&amp;author=Nood%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Narula, N. et al. Systematic review and meta-analysis fecal microbiota transplantation for treatment of active ulcerative colitis. <i>Inflamm. Bowel Dis.</i> <b>23</b>, 1702–1709 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1097/MIB.0000000000001228" data-track-item_id="10.1097/MIB.0000000000001228" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1097%2FMIB.0000000000001228" aria-label="Article reference 6" data-doi="10.1097/MIB.0000000000001228">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28906291" aria-label="PubMed reference 6">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20review%20and%20meta-analysis%20fecal%20microbiota%20transplantation%20for%20treatment%20of%20active%20ulcerative%20colitis&amp;journal=Inflamm.%20Bowel%20Dis.&amp;doi=10.1097%2FMIB.0000000000001228&amp;volume=23&amp;pages=1702-1709&amp;publication_year=2017&amp;author=Narula%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Haifer, C., Leong, R. W. &amp; Paramsothy, S. The role of fecal microbiota transplantation in the treatment of inflammatory bowel disease. <i>Curr. Opin. Pharmacol.</i> <b>55</b>, 8–16 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.coph.2020.08.009" data-track-item_id="10.1016/j.coph.2020.08.009" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.coph.2020.08.009" aria-label="Article reference 7" data-doi="10.1016/j.coph.2020.08.009">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvV2msr3F" aria-label="CAS reference 7">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33035780" aria-label="PubMed reference 7">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7538387" aria-label="PubMed Central reference 7">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20fecal%20microbiota%20transplantation%20in%20the%20treatment%20of%20inflammatory%20bowel%20disease&amp;journal=Curr.%20Opin.%20Pharmacol.&amp;doi=10.1016%2Fj.coph.2020.08.009&amp;volume=55&amp;pages=8-16&amp;publication_year=2020&amp;author=Haifer%2CC&amp;author=Leong%2CRW&amp;author=Paramsothy%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Suez, J. et al. Post-antibiotic gut mucosal microbiome reconstitution is impaired by probiotics and improved by autologous FMT. <i>Cell</i> <b>174</b>, 1406–1423 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2018.08.047" data-track-item_id="10.1016/j.cell.2018.08.047" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2018.08.047" aria-label="Article reference 8" data-doi="10.1016/j.cell.2018.08.047">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1Omu7bP" aria-label="CAS reference 8">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30193113" aria-label="PubMed reference 8">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Post-antibiotic%20gut%20mucosal%20microbiome%20reconstitution%20is%20impaired%20by%20probiotics%20and%20improved%20by%20autologous%20FMT&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.08.047&amp;volume=174&amp;pages=1406-1423&amp;publication_year=2018&amp;author=Suez%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Baruch, E. N. et al. Fecal microbiota transplant promotes response in immunotherapy-refractory melanoma patients. <i>Science</i> <b>371</b>, 602–609 (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Davar, D. et al. Fecal microbiota transplant overcomes resistance to anti-PD-1 therapy in melanoma patients. <i>Science</i> <b>371</b>, 595–602 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.abf3363" data-track-item_id="10.1126/science.abf3363" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.abf3363" aria-label="Article reference 10" data-doi="10.1126/science.abf3363">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsFCrs7s%3D" aria-label="CAS reference 10">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33542131" aria-label="PubMed reference 10">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8097968" aria-label="PubMed Central reference 10">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplant%20overcomes%20resistance%20to%20anti-PD-1%20therapy%20in%20melanoma%20patients&amp;journal=Science&amp;doi=10.1126%2Fscience.abf3363&amp;volume=371&amp;pages=595-602&amp;publication_year=2021&amp;author=Davar%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Burrello, C. et al. Therapeutic fecal microbiota transplantation controls intestinal inflammation through IL10 secretion by immune cells. <i>Nat. Commun.</i> <b>9</b>, 5184 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-018-07359-8" data-track-item_id="10.1038/s41467-018-07359-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-018-07359-8" aria-label="Article reference 11" data-doi="10.1038/s41467-018-07359-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30518790" aria-label="PubMed reference 11">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6281577" aria-label="PubMed Central reference 11">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Therapeutic%20fecal%20microbiota%20transplantation%20controls%20intestinal%20inflammation%20through%20IL10%20secretion%20by%20immune%20cells&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-07359-8&amp;volume=9&amp;publication_year=2018&amp;author=Burrello%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Seekatz, A. M. et al. Restoration of short chain fatty acid and bile acid metabolism following fecal microbiota transplantation in patients with recurrent <i>Clostridium difficile</i> infection. <i>Anaerobe</i> <b>53</b>, 64–73 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.anaerobe.2018.04.001" data-track-item_id="10.1016/j.anaerobe.2018.04.001" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.anaerobe.2018.04.001" aria-label="Article reference 12" data-doi="10.1016/j.anaerobe.2018.04.001">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXos1Kgs70%3D" aria-label="CAS reference 12">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29654837" aria-label="PubMed reference 12">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6185828" aria-label="PubMed Central reference 12">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Restoration%20of%20short%20chain%20fatty%20acid%20and%20bile%20acid%20metabolism%20following%20fecal%20microbiota%20transplantation%20in%20patients%20with%20recurrent%20Clostridium%20difficile%20infection&amp;journal=Anaerobe&amp;doi=10.1016%2Fj.anaerobe.2018.04.001&amp;volume=53&amp;pages=64-73&amp;publication_year=2018&amp;author=Seekatz%2CAM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Zuo, T. et al. Bacteriophage transfer during fecal microbiota transplantation is associated with treatment response in <i>Clostridium difficile</i> infection. <i>Gastroenterology</i> <b>152</b>, S140–S141 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0016-5085(17)30798-9" data-track-item_id="10.1016/S0016-5085(17)30798-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0016-5085%2817%2930798-9" aria-label="Article reference 13" data-doi="10.1016/S0016-5085(17)30798-9">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Bacteriophage%20transfer%20during%20fecal%20microbiota%20transplantation%20is%20associated%20with%20treatment%20response%20in%20Clostridium%20difficile%20infection&amp;journal=Gastroenterology&amp;doi=10.1016%2FS0016-5085%2817%2930798-9&amp;volume=152&amp;pages=S140-S141&amp;publication_year=2017&amp;author=Zuo%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Manrique, P. et al. Gut bacteriophage dynamics during fecal microbial transplantation in subjects with metabolic syndrome. <i>Gut Microbes</i> <b>13</b>, 1–15 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1080/19490976.2021.1897217" data-track-item_id="10.1080/19490976.2021.1897217" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1080%2F19490976.2021.1897217" aria-label="Article reference 14" data-doi="10.1080/19490976.2021.1897217">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33794724" aria-label="PubMed reference 14">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Gut%20bacteriophage%20dynamics%20during%20fecal%20microbial%20transplantation%20in%20subjects%20with%20metabolic%20syndrome&amp;journal=Gut%20Microbes&amp;doi=10.1080%2F19490976.2021.1897217&amp;volume=13&amp;pages=1-15&amp;publication_year=2021&amp;author=Manrique%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Wilson, B. C., Vatanen, T., Cutfield, W. S. &amp; O’Sullivan, J. M. The super-donor phenomenon in fecal microbiota transplantation. <i>Front. Cell. Infect. Microbiol.</i> <b>9</b>, 2 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3389/fcimb.2019.00002" data-track-item_id="10.3389/fcimb.2019.00002" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3389%2Ffcimb.2019.00002" aria-label="Article reference 15" data-doi="10.3389/fcimb.2019.00002">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1KiurrE" aria-label="CAS reference 15">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30719428" aria-label="PubMed reference 15">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6348388" aria-label="PubMed Central reference 15">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20super-donor%20phenomenon%20in%20fecal%20microbiota%20transplantation&amp;journal=Front.%20Cell.%20Infect.%20Microbiol.&amp;doi=10.3389%2Ffcimb.2019.00002&amp;volume=9&amp;publication_year=2019&amp;author=Wilson%2CBC&amp;author=Vatanen%2CT&amp;author=Cutfield%2CWS&amp;author=O%E2%80%99Sullivan%2CJM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Duvallet, C. et al. Framework for rational donor selection in fecal microbiota transplant clinical trials. <i>PloS ONE</i> <b>14</b>, e0222881 (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Olesen, S. W. &amp; Gerardin, Y. Re-evaluating the evidence for fecal microbiota transplantation “super-donors” in inflammatory bowel disease. <i>J. Crohns Colitis</i> <b>15</b>, 453–461 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/ecco-jcc/jjaa170" data-track-item_id="10.1093/ecco-jcc/jjaa170" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fecco-jcc%2Fjjaa170" aria-label="Article reference 17" data-doi="10.1093/ecco-jcc/jjaa170">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32808030" aria-label="PubMed reference 17">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Re-evaluating%20the%20evidence%20for%20fecal%20microbiota%20transplantation%20%E2%80%9Csuper-donors%E2%80%9D%20in%20inflammatory%20bowel%20disease&amp;journal=J.%20Crohns%20Colitis&amp;doi=10.1093%2Fecco-jcc%2Fjjaa170&amp;volume=15&amp;pages=453-461&amp;publication_year=2021&amp;author=Olesen%2CSW&amp;author=Gerardin%2CY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Kootte, R. S. et al. Improvement of insulin sensitivity after lean donor feces in metabolic syndrome is driven by baseline intestinal imcrobiota composition. <i>Cell Metab.</i> <b>26</b>, 611–619 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cmet.2017.09.008" data-track-item_id="10.1016/j.cmet.2017.09.008" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cmet.2017.09.008" aria-label="Article reference 18" data-doi="10.1016/j.cmet.2017.09.008">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1Siur7J" aria-label="CAS reference 18">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28978426" aria-label="PubMed reference 18">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Improvement%20of%20insulin%20sensitivity%20after%20lean%20donor%20feces%20in%20metabolic%20syndrome%20is%20driven%20by%20baseline%20intestinal%20imcrobiota%20composition&amp;journal=Cell%20Metab.&amp;doi=10.1016%2Fj.cmet.2017.09.008&amp;volume=26&amp;pages=611-619&amp;publication_year=2017&amp;author=Kootte%2CRS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Danne, C., Rolhion, N. &amp; Sokol, H. Recipient factors in fecal microbiota transplantation: one stool does not fit all. <i>Nat. Rev. Gastroenterol. Hepatol</i>. <b>18</b>, 503–513 (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Fujimoto, K. et al. Functional restoration of bacteriomes and viromes by fecal microbiota transplantation. <i>Gastroenterology</i> <b>160</b>, 2089–2102 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1053/j.gastro.2021.02.013" data-track-item_id="10.1053/j.gastro.2021.02.013" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1053%2Fj.gastro.2021.02.013" aria-label="Article reference 20" data-doi="10.1053/j.gastro.2021.02.013">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33577875" aria-label="PubMed reference 20">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20restoration%20of%20bacteriomes%20and%20viromes%20by%20fecal%20microbiota%20transplantation&amp;journal=Gastroenterology&amp;doi=10.1053%2Fj.gastro.2021.02.013&amp;volume=160&amp;pages=2089-2102&amp;publication_year=2021&amp;author=Fujimoto%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Peri, R. et al. The impact of technical and clinical factors on fecal microbiota transfer outcomes for the treatment of recurrent <i>Clostridioides difficile</i> infections in Germany. <i>United European Gastroenterol. J.</i> <b>7</b>, 716–722 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1177/2050640619839918" data-track-item_id="10.1177/2050640619839918" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1177%2F2050640619839918" aria-label="Article reference 21" data-doi="10.1177/2050640619839918">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31210950" aria-label="PubMed reference 21">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6545715" aria-label="PubMed Central reference 21">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20impact%20of%20technical%20and%20clinical%20factors%20on%20fecal%20microbiota%20transfer%20outcomes%20for%20the%20treatment%20of%20recurrent%20Clostridioides%20difficile%20infections%20in%20Germany&amp;journal=United%20European%20Gastroenterol.%20J.&amp;doi=10.1177%2F2050640619839918&amp;volume=7&amp;pages=716-722&amp;publication_year=2019&amp;author=Peri%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Draper, L. A. et al. Long-term colonisation with donor bacteriophages following successful fecal microbial transplantation. <i>Microbiome</i> <b>6</b>, 220 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s40168-018-0598-x" data-track-item_id="10.1186/s40168-018-0598-x" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s40168-018-0598-x" aria-label="Article reference 22" data-doi="10.1186/s40168-018-0598-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DC%2BB3crpslWqsw%3D%3D" aria-label="CAS reference 22">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30526683" aria-label="PubMed reference 22">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6288847" aria-label="PubMed Central reference 22">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Long-term%20colonisation%20with%20donor%20bacteriophages%20following%20successful%20fecal%20microbial%20transplantation&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-018-0598-x&amp;volume=6&amp;publication_year=2018&amp;author=Draper%2CLA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Leonardi, I. et al. Fungal trans-kingdom dynamics linked to responsiveness to fecal microbiota transplantation (FMT) therapy in ulcerative colitis. <i>Cell Host Microbe</i> <b>27</b>, 823–829 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Zuo, T. et al. Gut fungal dysbiosis correlates with reduced efficacy of fecal microbiota transplantation in <i>Clostridium difficile</i> infection. <i>Nat. Commun.</i> <b>9</b>, 3663 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-018-06103-6" data-track-item_id="10.1038/s41467-018-06103-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-018-06103-6" aria-label="Article reference 24" data-doi="10.1038/s41467-018-06103-6">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30202057" aria-label="PubMed reference 24">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6131390" aria-label="PubMed Central reference 24">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Gut%20fungal%20dysbiosis%20correlates%20with%20reduced%20efficacy%20of%20fecal%20microbiota%20transplantation%20in%20Clostridium%20difficile%20infection&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-06103-6&amp;volume=9&amp;publication_year=2018&amp;author=Zuo%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Li, S. S. et al. Durable coexistence of donor and recipient strains after fecal microbiota transplantation. <i>Science</i> <b>352</b>, 586–589 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aad8852" data-track-item_id="10.1126/science.aad8852" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aad8852" aria-label="Article reference 25" data-doi="10.1126/science.aad8852">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xms1KisLg%3D" aria-label="CAS reference 25">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27126044" aria-label="PubMed reference 25">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Durable%20coexistence%20of%20donor%20and%20recipient%20strains%20after%20fecal%20microbiota%20transplantation&amp;journal=Science&amp;doi=10.1126%2Fscience.aad8852&amp;volume=352&amp;pages=586-589&amp;publication_year=2016&amp;author=Li%2CSS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Smillie, C. S. et al. Strain tracking reveals the determinants of bacterial engraftment in the human gut following fecal microbiota transplantation. <i>Cell Host Microbe</i> <b>23</b>, 229–240 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.chom.2018.01.003" data-track-item_id="10.1016/j.chom.2018.01.003" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.chom.2018.01.003" aria-label="Article reference 26" data-doi="10.1016/j.chom.2018.01.003">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXivVSlsr0%3D" aria-label="CAS reference 26">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29447696" aria-label="PubMed reference 26">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8318347" aria-label="PubMed Central reference 26">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Strain%20tracking%20reveals%20the%20determinants%20of%20bacterial%20engraftment%20in%20the%20human%20gut%20following%20fecal%20microbiota%20transplantation&amp;journal=Cell%20Host%20Microbe&amp;doi=10.1016%2Fj.chom.2018.01.003&amp;volume=23&amp;pages=229-240&amp;publication_year=2018&amp;author=Smillie%2CCS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Podlesny, D. et al. Metagenomic strain detection with SameStr: identification of a persisting core gut microbiota transferable by fecal transplantation. <i>Microbiome</i> <b>10</b>, 53 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s40168-022-01251-w" data-track-item_id="10.1186/s40168-022-01251-w" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s40168-022-01251-w" aria-label="Article reference 27" data-doi="10.1186/s40168-022-01251-w">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1GqsLvE" aria-label="CAS reference 27">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35337386" aria-label="PubMed reference 27">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8951724" aria-label="PubMed Central reference 27">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Metagenomic%20strain%20detection%20with%20SameStr%3A%20identification%20of%20a%20persisting%20core%20gut%20microbiota%20transferable%20by%20fecal%20transplantation&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-022-01251-w&amp;volume=10&amp;publication_year=2022&amp;author=Podlesny%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Aggarwala, V. et al. Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes. <i>Nat. Microbiol.</i> <b>6</b>, 1309–1318 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41564-021-00966-0" data-track-item_id="10.1038/s41564-021-00966-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-021-00966-0" aria-label="Article reference 28" data-doi="10.1038/s41564-021-00966-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFamtrrL" aria-label="CAS reference 28">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34580445" aria-label="PubMed reference 28">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8993687" aria-label="PubMed Central reference 28">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Precise%20quantification%20of%20bacterial%20strains%20after%20fecal%20microbiota%20transplantation%20delineates%20long-term%20engraftment%20and%20explains%20outcomes&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-021-00966-0&amp;volume=6&amp;pages=1309-1318&amp;publication_year=2021&amp;author=Aggarwala%2CV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Lee, S. T. M. et al. Tracking microbial colonization in fecal microbiota transplantation experiments via genome-resolved metagenomics. <i>Microbiome</i> <b>5</b>, 50 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s40168-017-0270-x" data-track-item_id="10.1186/s40168-017-0270-x" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s40168-017-0270-x" aria-label="Article reference 29" data-doi="10.1186/s40168-017-0270-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28473000" aria-label="PubMed reference 29">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5418705" aria-label="PubMed Central reference 29">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Tracking%20microbial%20colonization%20in%20fecal%20microbiota%20transplantation%20experiments%20via%20genome-resolved%20metagenomics&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-017-0270-x&amp;volume=5&amp;publication_year=2017&amp;author=Lee%2CSTM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Wilson, B. C. et al. Strain engraftment competition and functional augmentation in a multi-donor fecal microbiota transplantation trial for obesity. <i>Microbiome</i> <b>9</b>, 107 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s40168-021-01060-7" data-track-item_id="10.1186/s40168-021-01060-7" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s40168-021-01060-7" aria-label="Article reference 30" data-doi="10.1186/s40168-021-01060-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XjsVems7c%3D" aria-label="CAS reference 30">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33985595" aria-label="PubMed reference 30">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8120839" aria-label="PubMed Central reference 30">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Strain%20engraftment%20competition%20and%20functional%20augmentation%20in%20a%20multi-donor%20fecal%20microbiota%20transplantation%20trial%20for%20obesity&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-021-01060-7&amp;volume=9&amp;publication_year=2021&amp;author=Wilson%2CBC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Ng, S. C. et al. Microbiota engraftment after fecal microbiota transplantation in obese subjects with type 2 diabetes: a 24-week, double-blind, randomised controlled trial. <i>Gut</i> <b>71</b>, 716–723 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1136/gutjnl-2020-323617" data-track-item_id="10.1136/gutjnl-2020-323617" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1136%2Fgutjnl-2020-323617" aria-label="Article reference 31" data-doi="10.1136/gutjnl-2020-323617">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFyqu7vJ" aria-label="CAS reference 31">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33785557" aria-label="PubMed reference 31">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Microbiota%20engraftment%20after%20fecal%20microbiota%20transplantation%20in%20obese%20subjects%20with%20type%202%20diabetes%3A%20a%2024-week%2C%20double-blind%2C%20randomised%20controlled%20trial&amp;journal=Gut&amp;doi=10.1136%2Fgutjnl-2020-323617&amp;volume=71&amp;pages=716-723&amp;publication_year=2022&amp;author=Ng%2CSC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Watson, A. R., Fuessel, J., Veseli, I. &amp; DeLongchamp, J. Z. Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut. Preprint at <a href="https://doi.org/10.1101/2021.03.02.433653" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.03.02.433653">https://doi.org/10.1101/2021.03.02.433653</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Walter, J., Maldonado-Gómez, M. X. &amp; Martínez, I. To engraft or not to engraft: an ecological framework for gut microbiome modulation with live microbes. <i>Curr. Opin. Biotechnol.</i> <b>49</b>, 129–139 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.copbio.2017.08.008" data-track-item_id="10.1016/j.copbio.2017.08.008" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.copbio.2017.08.008" aria-label="Article reference 33" data-doi="10.1016/j.copbio.2017.08.008">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlylsLnE" aria-label="CAS reference 33">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28866242" aria-label="PubMed reference 33">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=To%20engraft%20or%20not%20to%20engraft%3A%20an%20ecological%20framework%20for%20gut%20microbiome%20modulation%20with%20live%20microbes&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;doi=10.1016%2Fj.copbio.2017.08.008&amp;volume=49&amp;pages=129-139&amp;publication_year=2018&amp;author=Walter%2CJ&amp;author=Maldonado-G%C3%B3mez%2CMX&amp;author=Mart%C3%ADnez%2CI">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Schmidt, T. S. B., Raes, J. &amp; Bork, P. The human gut microbiome: from association to modulation. <i>Cell</i> <b>172</b>, 1198–1215 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2018.02.044" data-track-item_id="10.1016/j.cell.2018.02.044" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2018.02.044" aria-label="Article reference 34" data-doi="10.1016/j.cell.2018.02.044">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXktlSksbo%3D" aria-label="CAS reference 34">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29522742" aria-label="PubMed reference 34">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20human%20gut%20microbiome%3A%20from%20association%20to%20modulation&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.02.044&amp;volume=172&amp;pages=1198-1215&amp;publication_year=2018&amp;author=Schmidt%2CTSB&amp;author=Raes%2CJ&amp;author=Bork%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Xiao, Y., Angulo, M. T., Lao, S., Weiss, S. T. &amp; Liu, Y.-Y. An ecological framework to understand the efficacy of fecal microbiota transplantation. <i>Nat. Commun.</i> <b>11</b>, 3329 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-17180-x" data-track-item_id="10.1038/s41467-020-17180-x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-17180-x" aria-label="Article reference 35" data-doi="10.1038/s41467-020-17180-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlGmsrfJ" aria-label="CAS reference 35">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32620839" aria-label="PubMed reference 35">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334230" aria-label="PubMed Central reference 35">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20ecological%20framework%20to%20understand%20the%20efficacy%20of%20fecal%20microbiota%20transplantation&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-17180-x&amp;volume=11&amp;publication_year=2020&amp;author=Xiao%2CY&amp;author=Angulo%2CMT&amp;author=Lao%2CS&amp;author=Weiss%2CST&amp;author=Liu%2CY-Y">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Moss, E. L. et al. Long-term taxonomic and functional divergence from donor bacterial strains following fecal microbiota transplantation in immunocompromised patients. <i>PLoS ONE</i> <b>12</b>, e0182585 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0182585" data-track-item_id="10.1371/journal.pone.0182585" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0182585" aria-label="Article reference 36" data-doi="10.1371/journal.pone.0182585">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28827811" aria-label="PubMed reference 36">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5565110" aria-label="PubMed Central reference 36">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Long-term%20taxonomic%20and%20functional%20divergence%20from%20donor%20bacterial%20strains%20following%20fecal%20microbiota%20transplantation%20in%20immunocompromised%20patients&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0182585&amp;volume=12&amp;publication_year=2017&amp;author=Moss%2CEL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Singh, R. et al. Fecal microbiota transplantation against intestinal colonization by extended spectrum beta-lactamase producing Enterobacteriaceae: a proof of principle study. <i>BMC Res. Notes</i> <b>11</b>, 190 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13104-018-3293-x" data-track-item_id="10.1186/s13104-018-3293-x" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13104-018-3293-x" aria-label="Article reference 37" data-doi="10.1186/s13104-018-3293-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29566738" aria-label="PubMed reference 37">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5863815" aria-label="PubMed Central reference 37">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplantation%20against%20intestinal%20colonization%20by%20extended%20spectrum%20beta-lactamase%20producing%20Enterobacteriaceae%3A%20a%20proof%20of%20principle%20study&amp;journal=BMC%20Res.%20Notes&amp;doi=10.1186%2Fs13104-018-3293-x&amp;volume=11&amp;publication_year=2018&amp;author=Singh%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Bar-Yoseph, H. et al. Oral capsulized fecal microbiota transplantation for eradication of carbapenemase-producing Enterobacteriaceae colonization with a metagenomic perspective. <i>Clin. Infect. Dis</i>. <b>73</b>, e166–e175 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Leo, S. et al. Metagenomic characterization of gut microbiota of carriers of extended-spectrum beta-lactamase or carbapenemase-producing Enterobacteriaceae following treatment with oral antibiotics and fecal microbiota transplantation: results from a multicenter randomized trial. <i>Microorganisms</i> <b>8</b>, 941 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Koopen, A. M. et al. Effect of fecal microbiota transplantation combined with Mediterranean diet on insulin sensitivity in subjects with metabolic syndrome. <i>Front. Microbiol.</i> <b>12</b>, 662159 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3389/fmicb.2021.662159" data-track-item_id="10.3389/fmicb.2021.662159" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3389%2Ffmicb.2021.662159" aria-label="Article reference 40" data-doi="10.3389/fmicb.2021.662159">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34177842" aria-label="PubMed reference 40">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8222733" aria-label="PubMed Central reference 40">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Effect%20of%20fecal%20microbiota%20transplantation%20combined%20with%20Mediterranean%20diet%20on%20insulin%20sensitivity%20in%20subjects%20with%20metabolic%20syndrome&amp;journal=Front.%20Microbiol.&amp;doi=10.3389%2Ffmicb.2021.662159&amp;volume=12&amp;publication_year=2021&amp;author=Koopen%2CAM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Rossen, N. G. et al. Findings from a randomized controlled trial of fecal transplantation for patients with ulcerative colitis. <i>Gastroenterology</i> <b>149</b>, 110–118 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1053/j.gastro.2015.03.045" data-track-item_id="10.1053/j.gastro.2015.03.045" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1053%2Fj.gastro.2015.03.045" aria-label="Article reference 41" data-doi="10.1053/j.gastro.2015.03.045">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25836986" aria-label="PubMed reference 41">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Findings%20from%20a%20randomized%20controlled%20trial%20of%20fecal%20transplantation%20for%20patients%20with%20ulcerative%20colitis&amp;journal=Gastroenterology&amp;doi=10.1053%2Fj.gastro.2015.03.045&amp;volume=149&amp;pages=110-118&amp;publication_year=2015&amp;author=Rossen%2CNG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Nusbaum, D. J. et al. Gut microbial and metabolomic profiles after fecal microbiota transplantation in pediatric ulcerative colitis patients. <i>FEMS Microbiol. Ecol</i>. <b>94</b>, fiy133 (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Damman, C. J. et al. Low level engraftment and improvement following a single colonoscopic administration of fecal microbiota to patients with ulcerative colitis. <i>PLoS ONE</i> <b>10</b>, e0133925 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0133925" data-track-item_id="10.1371/journal.pone.0133925" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0133925" aria-label="Article reference 43" data-doi="10.1371/journal.pone.0133925">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26288277" aria-label="PubMed reference 43">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4544847" aria-label="PubMed Central reference 43">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Low%20level%20engraftment%20and%20improvement%20following%20a%20single%20colonoscopic%20administration%20of%20fecal%20microbiota%20to%20patients%20with%20ulcerative%20colitis&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0133925&amp;volume=10&amp;publication_year=2015&amp;author=Damman%2CCJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Goll, R. et al. Effects of fecal microbiota transplantation in subjects with irritable bowel syndrome are mirrored by changes in gut microbiome. <i>Gut Microbes</i> <b>12</b>, 1794263 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1080/19490976.2020.1794263" data-track-item_id="10.1080/19490976.2020.1794263" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1080%2F19490976.2020.1794263" aria-label="Article reference 44" data-doi="10.1080/19490976.2020.1794263">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32991818" aria-label="PubMed reference 44">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583512" aria-label="PubMed Central reference 44">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20fecal%20microbiota%20transplantation%20in%20subjects%20with%20irritable%20bowel%20syndrome%20are%20mirrored%20by%20changes%20in%20gut%20microbiome&amp;journal=Gut%20Microbes&amp;doi=10.1080%2F19490976.2020.1794263&amp;volume=12&amp;publication_year=2020&amp;author=Goll%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Vaughn, B. P. et al. Increased intestinal microbial diversity following fecal microbiota transplant for active Crohn’s disease. <i>Inflamm. Bowel Dis.</i> <b>22</b>, 2182–2190 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1097/MIB.0000000000000893" data-track-item_id="10.1097/MIB.0000000000000893" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1097%2FMIB.0000000000000893" aria-label="Article reference 45" data-doi="10.1097/MIB.0000000000000893">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27542133" aria-label="PubMed reference 45">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Increased%20intestinal%20microbial%20diversity%20following%20fecal%20microbiota%20transplant%20for%20active%20Crohn%E2%80%99s%20disease&amp;journal=Inflamm.%20Bowel%20Dis.&amp;doi=10.1097%2FMIB.0000000000000893&amp;volume=22&amp;pages=2182-2190&amp;publication_year=2016&amp;author=Vaughn%2CBP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Ianiro, G. et al. Fecal microbiota transplantation for the treatment of diarrhoea induced by tyrosine-kinase inhibitors in patients with metastatic renal cell carcinoma. <i>Nat. Commun.</i> <b>11</b>, 4333 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-18127-y" data-track-item_id="10.1038/s41467-020-18127-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-18127-y" aria-label="Article reference 46" data-doi="10.1038/s41467-020-18127-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32859933" aria-label="PubMed reference 46">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7455693" aria-label="PubMed Central reference 46">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplantation%20for%20the%20treatment%20of%20diarrhoea%20induced%20by%20tyrosine-kinase%20inhibitors%20in%20patients%20with%20metastatic%20renal%20cell%20carcinoma&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-18127-y&amp;volume=11&amp;publication_year=2020&amp;author=Ianiro%2CG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">Zhao, H.-J. et al. The efficacy of fecal microbiota transplantation for children with Tourette syndrome: a preliminary study. <i>Front. Psychiatry</i> <b>11</b>, 554441 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3389/fpsyt.2020.554441" data-track-item_id="10.3389/fpsyt.2020.554441" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3389%2Ffpsyt.2020.554441" aria-label="Article reference 47" data-doi="10.3389/fpsyt.2020.554441">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33424650" aria-label="PubMed reference 47">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7793740" aria-label="PubMed Central reference 47">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20efficacy%20of%20fecal%20microbiota%20transplantation%20for%20children%20with%20Tourette%20syndrome%3A%20a%20preliminary%20study&amp;journal=Front.%20Psychiatry&amp;doi=10.3389%2Ffpsyt.2020.554441&amp;volume=11&amp;publication_year=2020&amp;author=Zhao%2CH-J">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Goloshchapov, O. V. et al. Long-term impact of fecal transplantation in healthy volunteers. <i>BMC Microbiol.</i> <b>19</b>, 312 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s12866-019-1689-y" data-track-item_id="10.1186/s12866-019-1689-y" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s12866-019-1689-y" aria-label="Article reference 48" data-doi="10.1186/s12866-019-1689-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVSrsQ%3D%3D" aria-label="CAS reference 48">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31888470" aria-label="PubMed reference 48">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6938016" aria-label="PubMed Central reference 48">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Long-term%20impact%20of%20fecal%20transplantation%20in%20healthy%20volunteers&amp;journal=BMC%20Microbiol.&amp;doi=10.1186%2Fs12866-019-1689-y&amp;volume=19&amp;publication_year=2019&amp;author=Goloshchapov%2COV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Tian, L. et al. Deciphering functional redundancy in the human microbiome. <i>Nat. Commun.</i> <b>11</b>, 6217 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-19940-1" data-track-item_id="10.1038/s41467-020-19940-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-19940-1" aria-label="Article reference 49" data-doi="10.1038/s41467-020-19940-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFems77N" aria-label="CAS reference 49">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33277504" aria-label="PubMed reference 49">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7719190" aria-label="PubMed Central reference 49">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=Deciphering%20functional%20redundancy%20in%20the%20human%20microbiome&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19940-1&amp;volume=11&amp;publication_year=2020&amp;author=Tian%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Hildebrand, F. et al. Dispersal strategies shape persistence and evolution of human gut bacteria. <i>Cell Host Microbe</i> <b>29</b>, 1167–1176 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.chom.2021.05.008" data-track-item_id="10.1016/j.chom.2021.05.008" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.chom.2021.05.008" aria-label="Article reference 50" data-doi="10.1016/j.chom.2021.05.008">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlalsL7O" aria-label="CAS reference 50">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34111423" aria-label="PubMed reference 50">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8288446" aria-label="PubMed Central reference 50">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Dispersal%20strategies%20shape%20persistence%20and%20evolution%20of%20human%20gut%20bacteria&amp;journal=Cell%20Host%20Microbe&amp;doi=10.1016%2Fj.chom.2021.05.008&amp;volume=29&amp;pages=1167-1176&amp;publication_year=2021&amp;author=Hildebrand%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Lahti, L., Salojärvi, J., Salonen, A., Scheffer, M. &amp; de Vos, W. M. Tipping elements in the human intestinal ecosystem. <i>Nat. Commun.</i> <b>5</b>, 4344 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ncomms5344" data-track-item_id="10.1038/ncomms5344" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fncomms5344" aria-label="Article reference 51" data-doi="10.1038/ncomms5344">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXitVWgsLjJ" aria-label="CAS reference 51">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25003530" aria-label="PubMed reference 51">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Tipping%20elements%20in%20the%20human%20intestinal%20ecosystem&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms5344&amp;volume=5&amp;publication_year=2014&amp;author=Lahti%2CL&amp;author=Saloj%C3%A4rvi%2CJ&amp;author=Salonen%2CA&amp;author=Scheffer%2CM&amp;author=Vos%2CWM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Scheffer, M., Carpenter, S. R., Dakos, V. &amp; van Nes, E. H. Generic indicators of ecological resilience: inferring the chance of a critical transition. <i>Annu. Rev. Ecol. Evol. Syst</i>. <b>46</b>, 145–167 (2015).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Gonze, D., Lahti, L., Raes, J. &amp; Faust, K. Multi-stability and the origin of microbial community types. <i>ISME J.</i> <b>11</b>, 2159–2166 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ismej.2017.60" data-track-item_id="10.1038/ismej.2017.60" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fismej.2017.60" aria-label="Article reference 53" data-doi="10.1038/ismej.2017.60">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28475180" aria-label="PubMed reference 53">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5607358" aria-label="PubMed Central reference 53">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-stability%20and%20the%20origin%20of%20microbial%20community%20types&amp;journal=ISME%20J.&amp;doi=10.1038%2Fismej.2017.60&amp;volume=11&amp;publication_year=2017&amp;author=Gonze%2CD&amp;author=Lahti%2CL&amp;author=Raes%2CJ&amp;author=Faust%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Costea, P. I. et al. Enterotypes in the landscape of gut microbial community composition. <i>Nat. Microbiol.</i> <b>3</b>, 8–16 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41564-017-0072-8" data-track-item_id="10.1038/s41564-017-0072-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-017-0072-8" aria-label="Article reference 54" data-doi="10.1038/s41564-017-0072-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVSmtLvK" aria-label="CAS reference 54">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29255284" aria-label="PubMed reference 54">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=Enterotypes%20in%20the%20landscape%20of%20gut%20microbial%20community%20composition&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-017-0072-8&amp;volume=3&amp;pages=8-16&amp;publication_year=2018&amp;author=Costea%2CPI">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Debray, R. et al. Priority effects in microbiome assembly. <i>Nat. Rev. Microbiol</i>. <b>20</b>, 109–121 (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Zaneveld, J. R., McMinds, R. &amp; Thurber, R. V. Stress and stability: applying the Anna Karenina principle to animal microbiomes. <i>Nat. Microbiol.</i> <b>2</b>, 17121 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmicrobiol.2017.121" data-track-item_id="10.1038/nmicrobiol.2017.121" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmicrobiol.2017.121" aria-label="Article reference 56" data-doi="10.1038/nmicrobiol.2017.121">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlyit7vK" aria-label="CAS reference 56">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28836573" aria-label="PubMed reference 56">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 56" href="http://scholar.google.com/scholar_lookup?&amp;title=Stress%20and%20stability%3A%20applying%20the%20Anna%20Karenina%20principle%20to%20animal%20microbiomes&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fnmicrobiol.2017.121&amp;volume=2&amp;publication_year=2017&amp;author=Zaneveld%2CJR&amp;author=McMinds%2CR&amp;author=Thurber%2CRV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Basson, A. R., Zhou, Y., Seo, B., Rodriguez-Palacios, A. &amp; Cominelli, F. Autologous fecal microbiota transplantation for the treatment of inflammatory bowel disease. <i>Transl. Res.</i> <b>226</b>, 1–11 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.trsl.2020.05.008" data-track-item_id="10.1016/j.trsl.2020.05.008" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.trsl.2020.05.008" aria-label="Article reference 57" data-doi="10.1016/j.trsl.2020.05.008">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32585148" aria-label="PubMed reference 57">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7308243" aria-label="PubMed Central reference 57">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Autologous%20fecal%20microbiota%20transplantation%20for%20the%20treatment%20of%20inflammatory%20bowel%20disease&amp;journal=Transl.%20Res.&amp;doi=10.1016%2Fj.trsl.2020.05.008&amp;volume=226&amp;pages=1-11&amp;publication_year=2020&amp;author=Basson%2CAR&amp;author=Zhou%2CY&amp;author=Seo%2CB&amp;author=Rodriguez-Palacios%2CA&amp;author=Cominelli%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">de Groot, P. et al. Fecal microbiota transplantation halts progression of human new-onset type 1 diabetes in a randomised controlled trial. <i>Gut</i> <b>70</b>, 92–105 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1136/gutjnl-2020-322630" data-track-item_id="10.1136/gutjnl-2020-322630" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1136%2Fgutjnl-2020-322630" aria-label="Article reference 58" data-doi="10.1136/gutjnl-2020-322630">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33106354" aria-label="PubMed reference 58">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplantation%20halts%20progression%20of%20human%20new-onset%20type%201%20diabetes%20in%20a%20randomised%20controlled%20trial&amp;journal=Gut&amp;doi=10.1136%2Fgutjnl-2020-322630&amp;volume=70&amp;pages=92-105&amp;publication_year=2021&amp;author=Groot%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Ott, S. J. et al. Efficacy of sterile fecal filtrate transfer for treating patients with <i>Clostridium difficile</i> infection. <i>Gastroenterology</i> <b>152</b>, 799–811 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1053/j.gastro.2016.11.010" data-track-item_id="10.1053/j.gastro.2016.11.010" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1053%2Fj.gastro.2016.11.010" aria-label="Article reference 59" data-doi="10.1053/j.gastro.2016.11.010">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27866880" aria-label="PubMed reference 59">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Efficacy%20of%20sterile%20fecal%20filtrate%20transfer%20for%20treating%20patients%20with%20Clostridium%20difficile%20infection&amp;journal=Gastroenterology&amp;doi=10.1053%2Fj.gastro.2016.11.010&amp;volume=152&amp;pages=799-811&amp;publication_year=2017&amp;author=Ott%2CSJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Bojanova, D. P. &amp; Bordenstein, S. R. Fecal transplants: what is being transferred? <i>PLoS Biol.</i> <b>14</b>, e1002503 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pbio.1002503" data-track-item_id="10.1371/journal.pbio.1002503" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.1002503" aria-label="Article reference 60" data-doi="10.1371/journal.pbio.1002503">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27404502" aria-label="PubMed reference 60">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4942072" aria-label="PubMed Central reference 60">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20transplants%3A%20what%20is%20being%20transferred%3F&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.1002503&amp;volume=14&amp;publication_year=2016&amp;author=Bojanova%2CDP&amp;author=Bordenstein%2CSR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Fuentes, S. et al. Microbial shifts and signatures of long-term remission in ulcerative colitis after fecal microbiota transplantation. <i>ISME J.</i> <b>11</b>, 1877–1889 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ismej.2017.44" data-track-item_id="10.1038/ismej.2017.44" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fismej.2017.44" aria-label="Article reference 61" data-doi="10.1038/ismej.2017.44">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28398347" aria-label="PubMed reference 61">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5520032" aria-label="PubMed Central reference 61">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Microbial%20shifts%20and%20signatures%20of%20long-term%20remission%20in%20ulcerative%20colitis%20after%20fecal%20microbiota%20transplantation&amp;journal=ISME%20J.&amp;doi=10.1038%2Fismej.2017.44&amp;volume=11&amp;pages=1877-1889&amp;publication_year=2017&amp;author=Fuentes%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Coelho, L. P. et al. NG-meta-profiler: fast processing of metagenomes using NGLess, a domain-specific language. <i>Microbiome</i> <b>7</b>, 84 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s40168-019-0684-8" data-track-item_id="10.1186/s40168-019-0684-8" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s40168-019-0684-8" aria-label="Article reference 62" data-doi="10.1186/s40168-019-0684-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31159881" aria-label="PubMed reference 62">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6547473" aria-label="PubMed Central reference 62">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=NG-meta-profiler%3A%20fast%20processing%20of%20metagenomes%20using%20NGLess%2C%20a%20domain-specific%20language&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-019-0684-8&amp;volume=7&amp;publication_year=2019&amp;author=Coelho%2CLP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Milanese, A. et al. Microbial abundance, activity and population genomic profiling with mOTUs2. <i>Nat. Commun.</i> <b>10</b>, 1014 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-019-08844-4" data-track-item_id="10.1038/s41467-019-08844-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-019-08844-4" aria-label="Article reference 63" data-doi="10.1038/s41467-019-08844-4">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30833550" aria-label="PubMed reference 63">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6399450" aria-label="PubMed Central reference 63">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Microbial%20abundance%2C%20activity%20and%20population%20genomic%20profiling%20with%20mOTUs2&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-08844-4&amp;volume=10&amp;publication_year=2019&amp;author=Milanese%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Coelho, L. P. et al. Towards the biogeography of prokaryotic genes. <i>Nature</i> <b>601</b>, 252–256 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-021-04233-4" data-track-item_id="10.1038/s41586-021-04233-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-04233-4" aria-label="Article reference 64" data-doi="10.1038/s41586-021-04233-4">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXislGntbrE" aria-label="CAS reference 64">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34912116" aria-label="PubMed reference 64">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20the%20biogeography%20of%20prokaryotic%20genes&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-04233-4&amp;volume=601&amp;pages=252-256&amp;publication_year=2022&amp;author=Coelho%2CLP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Huerta-Cepas, J. et al. EggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. <i>Nucleic Acids Res.</i> <b>47</b>, D309–D314 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gky1085" data-track-item_id="10.1093/nar/gky1085" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgky1085" aria-label="Article reference 65" data-doi="10.1093/nar/gky1085">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FL" aria-label="CAS reference 65">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30418610" aria-label="PubMed reference 65">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=EggNOG%205.0%3A%20a%20hierarchical%2C%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky1085&amp;volume=47&amp;pages=D309-D314&amp;publication_year=2019&amp;author=Huerta-Cepas%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Vieira-Silva, S. et al. Species–function relationships shape ecological properties of the human gut microbiome. <i>Nat. Microbiol.</i> <b>1</b>, 16088 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmicrobiol.2016.88" data-track-item_id="10.1038/nmicrobiol.2016.88" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmicrobiol.2016.88" aria-label="Article reference 66" data-doi="10.1038/nmicrobiol.2016.88">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXkvF2ju74%3D" aria-label="CAS reference 66">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27573110" aria-label="PubMed reference 66">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=Species%E2%80%93function%20relationships%20shape%20ecological%20properties%20of%20the%20human%20gut%20microbiome&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fnmicrobiol.2016.88&amp;volume=1&amp;publication_year=2016&amp;author=Vieira-Silva%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Darzi, Y., Falony, G., Vieira-Silva, S. &amp; Raes, J. Towards biome-specific analysis of meta-omics data. <i>ISME J.</i> <b>10</b>, 1025–1028 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ismej.2015.188" data-track-item_id="10.1038/ismej.2015.188" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fismej.2015.188" aria-label="Article reference 67" data-doi="10.1038/ismej.2015.188">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xmsl2qu7w%3D" aria-label="CAS reference 67">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26623543" aria-label="PubMed reference 67">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20biome-specific%20analysis%20of%20meta-omics%20data&amp;journal=ISME%20J.&amp;doi=10.1038%2Fismej.2015.188&amp;volume=10&amp;pages=1025-1028&amp;publication_year=2016&amp;author=Darzi%2CY&amp;author=Falony%2CG&amp;author=Vieira-Silva%2CS&amp;author=Raes%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Nurk, S., Meleshko, D., Korobeynikov, A. &amp; Pevzner, P. A. MetaSPAdes: a new versatile metagenomic assembler. <i>Genome Res.</i> <b>27</b>, 824–834 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/gr.213959.116" data-track-item_id="10.1101/gr.213959.116" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.213959.116" aria-label="Article reference 68" data-doi="10.1101/gr.213959.116">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFyjsrrJ" aria-label="CAS reference 68">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28298430" aria-label="PubMed reference 68">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5411777" aria-label="PubMed Central reference 68">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=MetaSPAdes%3A%20a%20new%20versatile%20metagenomic%20assembler&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.213959.116&amp;volume=27&amp;pages=824-834&amp;publication_year=2017&amp;author=Nurk%2CS&amp;author=Meleshko%2CD&amp;author=Korobeynikov%2CA&amp;author=Pevzner%2CPA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Preprint at <a href="https://doi.org/10.48550/arXiv.1303.3997" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.48550/arXiv.1303.3997">https://doi.org/10.48550/arXiv.1303.3997</a> (2013).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Kang, D. D. et al. MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies. <i>PeerJ.</i> <b>7</b>, e7359 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7717/peerj.7359" data-track-item_id="10.7717/peerj.7359" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7717%2Fpeerj.7359" aria-label="Article reference 70" data-doi="10.7717/peerj.7359">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31388474" aria-label="PubMed reference 70">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6662567" aria-label="PubMed Central reference 70">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=MetaBAT%202%3A%20an%20adaptive%20binning%20algorithm%20for%20robust%20and%20efficient%20genome%20reconstruction%20from%20metagenome%20assemblies&amp;journal=PeerJ.&amp;doi=10.7717%2Fpeerj.7359&amp;volume=7&amp;publication_year=2019&amp;author=Kang%2CDD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Li, D., Liu, C.-M., Luo, R., Sadakane, K. &amp; Lam, T.-W. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. <i>Bioinformatics</i> <b>31</b>, 1674–1676 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btv033" data-track-item_id="10.1093/bioinformatics/btv033" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv033" aria-label="Article reference 71" data-doi="10.1093/bioinformatics/btv033">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFyltL3N" aria-label="CAS reference 71">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25609793" aria-label="PubMed reference 71">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=MEGAHIT%3A%20an%20ultra-fast%20single-node%20solution%20for%20large%20and%20complex%20metagenomics%20assembly%20via%20succinct%20de%20Bruijn%20graph&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtv033&amp;volume=31&amp;pages=1674-1676&amp;publication_year=2015&amp;author=Li%2CD&amp;author=Liu%2CC-M&amp;author=Luo%2CR&amp;author=Sadakane%2CK&amp;author=Lam%2CT-W">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Alneberg, J. et al. Binning metagenomic contigs by coverage and composition. <i>Nat. Methods</i> <b>11</b>, 1144–1146 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmeth.3103" data-track-item_id="10.1038/nmeth.3103" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.3103" aria-label="Article reference 72" data-doi="10.1038/nmeth.3103">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFOksrbF" aria-label="CAS reference 72">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25218180" aria-label="PubMed reference 72">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Binning%20metagenomic%20contigs%20by%20coverage%20and%20composition&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.3103&amp;volume=11&amp;pages=1144-1146&amp;publication_year=2014&amp;author=Alneberg%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Sieber, C. M. K. et al. Recovery of genomes from metagenomes via a dereplication, aggregation and scoring strategy. <i>Nat. Microbiol.</i> <b>3</b>, 836–843 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41564-018-0171-1" data-track-item_id="10.1038/s41564-018-0171-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-018-0171-1" aria-label="Article reference 73" data-doi="10.1038/s41564-018-0171-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVelsbvM" aria-label="CAS reference 73">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29807988" aria-label="PubMed reference 73">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786971" aria-label="PubMed Central reference 73">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Recovery%20of%20genomes%20from%20metagenomes%20via%20a%20dereplication%2C%20aggregation%20and%20scoring%20strategy&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-018-0171-1&amp;volume=3&amp;pages=836-843&amp;publication_year=2018&amp;author=Sieber%2CCMK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Uritskiy, G. V., DiRuggiero, J. &amp; Taylor, J. MetaWRAP—a flexible pipeline for genome-resolved metagenomic data analysis. <i>Microbiome</i> <b>6</b>, 158 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s40168-018-0541-1" data-track-item_id="10.1186/s40168-018-0541-1" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s40168-018-0541-1" aria-label="Article reference 74" data-doi="10.1186/s40168-018-0541-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30219103" aria-label="PubMed reference 74">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6138922" aria-label="PubMed Central reference 74">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 74" href="http://scholar.google.com/scholar_lookup?&amp;title=MetaWRAP%E2%80%94a%20flexible%20pipeline%20for%20genome-resolved%20metagenomic%20data%20analysis&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-018-0541-1&amp;volume=6&amp;publication_year=2018&amp;author=Uritskiy%2CGV&amp;author=DiRuggiero%2CJ&amp;author=Taylor%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Mende, D. R. et al. ProGenomes: a resource for consistent functional and taxonomic annotations of prokaryotic genomes. <i>Nucleic Acids Res.</i> <b>45</b>, D529–D534 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkw989" data-track-item_id="10.1093/nar/gkw989" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkw989" aria-label="Article reference 75" data-doi="10.1093/nar/gkw989">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslWht7o%3D" aria-label="CAS reference 75">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28053165" aria-label="PubMed reference 75">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 75" href="http://scholar.google.com/scholar_lookup?&amp;title=ProGenomes%3A%20a%20resource%20for%20consistent%20functional%20and%20taxonomic%20annotations%20of%20prokaryotic%20genomes&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw989&amp;volume=45&amp;pages=D529-D534&amp;publication_year=2017&amp;author=Mende%2CDR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Mende, D. R. et al. ProGenomes2: an improved database for accurate and consistent habitat, taxonomic and functional annotations of prokaryotic genomes. <i>Nucleic Acids Res.</i> <b>48</b>, D621–D625 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Glt7vK" aria-label="CAS reference 76">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31647096" aria-label="PubMed reference 76">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=ProGenomes2%3A%20an%20improved%20database%20for%20accurate%20and%20consistent%20habitat%2C%20taxonomic%20and%20functional%20annotations%20of%20prokaryotic%20genomes&amp;journal=Nucleic%20Acids%20Res.&amp;volume=48&amp;pages=D621-D625&amp;publication_year=2020&amp;author=Mende%2CDR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. &amp; Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. <i>Genome Res.</i> <b>25</b>, 1043–1055 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1101/gr.186072.114" data-track-item_id="10.1101/gr.186072.114" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1101%2Fgr.186072.114" aria-label="Article reference 77" data-doi="10.1101/gr.186072.114">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1SltbfE" aria-label="CAS reference 77">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25977477" aria-label="PubMed reference 77">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4484387" aria-label="PubMed Central reference 77">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=CheckM%3A%20assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates%2C%20single%20cells%2C%20and%20metagenomes&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.186072.114&amp;volume=25&amp;pages=1043-1055&amp;publication_year=2015&amp;author=Parks%2CDH&amp;author=Imelfort%2CM&amp;author=Skennerton%2CCT&amp;author=Hugenholtz%2CP&amp;author=Tyson%2CGW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Orakov, A. et al. GUNC: detection of chimerism and contamination in prokaryotic genomes. <i>Genome Biol.</i> <b>22</b>, 178 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-021-02393-0" data-track-item_id="10.1186/s13059-021-02393-0" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-021-02393-0" aria-label="Article reference 78" data-doi="10.1186/s13059-021-02393-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFKmtLrN" aria-label="CAS reference 78">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34120611" aria-label="PubMed reference 78">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8201837" aria-label="PubMed Central reference 78">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=GUNC%3A%20detection%20of%20chimerism%20and%20contamination%20in%20prokaryotic%20genomes&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-021-02393-0&amp;volume=22&amp;publication_year=2021&amp;author=Orakov%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="79."><p class="c-article-references__text" id="ref-CR79">Chaumeil, P.-A., Mussig, A. J., Hugenholtz, P. &amp; Parks, D. H. GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database. <i>Bioinformatics</i> <b>36</b>, 1925–1927 (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="80."><p class="c-article-references__text" id="ref-CR80">Hyatt, D. et al. Prodigal: prokaryotic gene recognition and translation initiation site identification. <i>BMC Bioinformatics</i> <b>11</b>, 119 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/1471-2105-11-119" data-track-item_id="10.1186/1471-2105-11-119" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/1471-2105-11-119" aria-label="Article reference 80" data-doi="10.1186/1471-2105-11-119">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20211023" aria-label="PubMed reference 80">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2848648" aria-label="PubMed Central reference 80">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 80" href="http://scholar.google.com/scholar_lookup?&amp;title=Prodigal%3A%20prokaryotic%20gene%20recognition%20and%20translation%20initiation%20site%20identification&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2F1471-2105-11-119&amp;volume=11&amp;publication_year=2010&amp;author=Hyatt%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="81."><p class="c-article-references__text" id="ref-CR81">Seemann, T. Prokka: rapid prokaryotic genome annotation. <i>Bioinformatics</i> <b>30</b>, 2068–2069 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btu153" data-track-item_id="10.1093/bioinformatics/btu153" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu153" aria-label="Article reference 81" data-doi="10.1093/bioinformatics/btu153">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFCrtLjI" aria-label="CAS reference 81">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24642063" aria-label="PubMed reference 81">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 81" href="http://scholar.google.com/scholar_lookup?&amp;title=Prokka%3A%20rapid%20prokaryotic%20genome%20annotation&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu153&amp;volume=30&amp;pages=2068-2069&amp;publication_year=2014&amp;author=Seemann%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="82."><p class="c-article-references__text" id="ref-CR82">Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. <i>Mol. Biol. Evol.</i> <b>34</b>, 2115–2122 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/molbev/msx148" data-track-item_id="10.1093/molbev/msx148" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fmolbev%2Fmsx148" aria-label="Article reference 82" data-doi="10.1093/molbev/msx148">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFaru7vF" aria-label="CAS reference 82">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28460117" aria-label="PubMed reference 82">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850834" aria-label="PubMed Central reference 82">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 82" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20genome-wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG-Mapper&amp;journal=Mol.%20Biol.%20Evol.&amp;doi=10.1093%2Fmolbev%2Fmsx148&amp;volume=34&amp;pages=2115-2122&amp;publication_year=2017&amp;author=Huerta-Cepas%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="83."><p class="c-article-references__text" id="ref-CR83">Alcock, B. P. et al. CARD 2020: antibiotic resistome surveillance with the comprehensive antibiotic resistance database. <i>Nucleic Acids Res.</i> <b>48</b>, D517–D525 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslWltrnK" aria-label="CAS reference 83">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31665441" aria-label="PubMed reference 83">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 83" href="http://scholar.google.com/scholar_lookup?&amp;title=CARD%202020%3A%20antibiotic%20resistome%20surveillance%20with%20the%20comprehensive%20antibiotic%20resistance%20database&amp;journal=Nucleic%20Acids%20Res.&amp;volume=48&amp;pages=D517-D525&amp;publication_year=2020&amp;author=Alcock%2CBP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="84."><p class="c-article-references__text" id="ref-CR84">Gibson, M. K., Forsberg, K. J. &amp; Dantas, G. Improved annotation of antibiotic resistance determinants reveals microbial resistomes cluster by ecology. <i>ISME J.</i> <b>9</b>, 207–216 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ismej.2014.106" data-track-item_id="10.1038/ismej.2014.106" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fismej.2014.106" aria-label="Article reference 84" data-doi="10.1038/ismej.2014.106">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFKlsLzN" aria-label="CAS reference 84">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25003965" aria-label="PubMed reference 84">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 84" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20annotation%20of%20antibiotic%20resistance%20determinants%20reveals%20microbial%20resistomes%20cluster%20by%20ecology&amp;journal=ISME%20J.&amp;doi=10.1038%2Fismej.2014.106&amp;volume=9&amp;pages=207-216&amp;publication_year=2015&amp;author=Gibson%2CMK&amp;author=Forsberg%2CKJ&amp;author=Dantas%2CG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="85."><p class="c-article-references__text" id="ref-CR85">Mende, D. R., Sunagawa, S., Zeller, G. &amp; Bork, P. Accurate and universal delineation of prokaryotic species. <i>Nat. Publ. Group</i> <b>10</b>, 881–884 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOht7vJ" aria-label="CAS reference 85">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 85" href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%20and%20universal%20delineation%20of%20prokaryotic%20species&amp;journal=Nat.%20Publ.%20Group&amp;volume=10&amp;pages=881-884&amp;publication_year=2013&amp;author=Mende%2CDR&amp;author=Sunagawa%2CS&amp;author=Zeller%2CG&amp;author=Bork%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="86."><p class="c-article-references__text" id="ref-CR86">Schmidt, T. S. B. et al. Drivers and determinants of strain dynamics following fecal microbiota transplantation. <i>Zenodo</i> <a href="https://doi.org/10.5281/ZENODO.5534163" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.5281/ZENODO.5534163">https://doi.org/10.5281/ZENODO.5534163</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="87."><p class="c-article-references__text" id="ref-CR87">Rodrigues, J. F. M., Schmidt, S. B. T., Tackmann, J. &amp; von Mering, C. MAPseq: highly efficient k-mer search with confidence estimates, for rRNA sequence analysis. <i>Bioinformatics</i> <b>33</b>, 3808–3810 (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="88."><p class="c-article-references__text" id="ref-CR88">Olm, M. R., Brown, C. T., Brooks, B. &amp; Banfield, J. F. DRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication. <i>ISME J.</i> <b>11</b>, 2864–2868 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ismej.2017.126" data-track-item_id="10.1038/ismej.2017.126" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fismej.2017.126" aria-label="Article reference 88" data-doi="10.1038/ismej.2017.126">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVeiur7K" aria-label="CAS reference 88">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28742071" aria-label="PubMed reference 88">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5702732" aria-label="PubMed Central reference 88">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 88" href="http://scholar.google.com/scholar_lookup?&amp;title=DRep%3A%20a%20tool%20for%20fast%20and%20accurate%20genomic%20comparisons%20that%20enables%20improved%20genome%20recovery%20from%20metagenomes%20through%20de-replication&amp;journal=ISME%20J.&amp;doi=10.1038%2Fismej.2017.126&amp;volume=11&amp;pages=2864-2868&amp;publication_year=2017&amp;author=Olm%2CMR&amp;author=Brown%2CCT&amp;author=Brooks%2CB&amp;author=Banfield%2CJF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="89."><p class="c-article-references__text" id="ref-CR89">Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. <i>Genome Biol.</i> <b>17</b>, 132 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-016-0997-x" data-track-item_id="10.1186/s13059-016-0997-x" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-016-0997-x" aria-label="Article reference 89" data-doi="10.1186/s13059-016-0997-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27323842" aria-label="PubMed reference 89">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4915045" aria-label="PubMed Central reference 89">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 89" href="http://scholar.google.com/scholar_lookup?&amp;title=Mash%3A%20fast%20genome%20and%20metagenome%20distance%20estimation%20using%20MinHash&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-016-0997-x&amp;volume=17&amp;publication_year=2016&amp;author=Ondov%2CBD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="90."><p class="c-article-references__text" id="ref-CR90">Jain, C., Rodriguez-R, L. M., Phillippy, A. M., Konstantinidis, K. T. &amp; Aluru, S. High throughput ANI analysis of 90 K prokaryotic genomes reveals clear species boundaries. <i>Nat. Commun.</i> <b>9</b>, 5114 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-018-07641-9" data-track-item_id="10.1038/s41467-018-07641-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-018-07641-9" aria-label="Article reference 90" data-doi="10.1038/s41467-018-07641-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30504855" aria-label="PubMed reference 90">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6269478" aria-label="PubMed Central reference 90">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 90" href="http://scholar.google.com/scholar_lookup?&amp;title=High%20throughput%20ANI%20analysis%20of%2090%E2%80%89K%20prokaryotic%20genomes%20reveals%20clear%20species%20boundaries&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-07641-9&amp;volume=9&amp;publication_year=2018&amp;author=Jain%2CC&amp;author=Rodriguez-R%2CLM&amp;author=Phillippy%2CAM&amp;author=Konstantinidis%2CKT&amp;author=Aluru%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="91."><p class="c-article-references__text" id="ref-CR91">Page, A. J. et al. Roary: rapid large-scale prokaryote pan genome analysis. <i>Bioinformatics</i> <b>31</b>, 3691–3693 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btv421" data-track-item_id="10.1093/bioinformatics/btv421" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv421" aria-label="Article reference 91" data-doi="10.1093/bioinformatics/btv421">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlamsrfO" aria-label="CAS reference 91">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26198102" aria-label="PubMed reference 91">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4817141" aria-label="PubMed Central reference 91">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 91" href="http://scholar.google.com/scholar_lookup?&amp;title=Roary%3A%20rapid%20large-scale%20prokaryote%20pan%20genome%20analysis&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtv421&amp;volume=31&amp;pages=3691-3693&amp;publication_year=2015&amp;author=Page%2CAJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="92."><p class="c-article-references__text" id="ref-CR92">Edgar, R. C. MUSCLE: a multiple sequence alignment method with reduced time and space complexity. <i>BMC Bioinformatics</i> <b>5</b>, 113 (2004).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/1471-2105-5-113" data-track-item_id="10.1186/1471-2105-5-113" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/1471-2105-5-113" aria-label="Article reference 92" data-doi="10.1186/1471-2105-5-113">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15318951" aria-label="PubMed reference 92">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC517706" aria-label="PubMed Central reference 92">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 92" href="http://scholar.google.com/scholar_lookup?&amp;title=MUSCLE%3A%20a%20multiple%20sequence%20alignment%20method%20with%20reduced%20time%20and%20space%20complexity&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2F1471-2105-5-113&amp;volume=5&amp;publication_year=2004&amp;author=Edgar%2CRC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="93."><p class="c-article-references__text" id="ref-CR93">Price, M. N., Dehal, P. S. &amp; Arkin, A. P. FastTree 2–approximately maximum-likelihood trees for large alignments. <i>PLoS ONE</i> <b>5</b>, e9490 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0009490" data-track-item_id="10.1371/journal.pone.0009490" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0009490" aria-label="Article reference 93" data-doi="10.1371/journal.pone.0009490">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20224823" aria-label="PubMed reference 93">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2835736" aria-label="PubMed Central reference 93">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 93" href="http://scholar.google.com/scholar_lookup?&amp;title=FastTree%202%E2%80%93approximately%20maximum-likelihood%20trees%20for%20large%20alignments&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0009490&amp;volume=5&amp;publication_year=2010&amp;author=Price%2CMN&amp;author=Dehal%2CPS&amp;author=Arkin%2CAP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="94."><p class="c-article-references__text" id="ref-CR94">Li, H. et al. The Sequence Alignment/Map format and SAMtools. <i>Bioinformatics</i> <b>25</b>, 2078–2079 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btp352" data-track-item_id="10.1093/bioinformatics/btp352" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp352" aria-label="Article reference 94" data-doi="10.1093/bioinformatics/btp352">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19505943" aria-label="PubMed reference 94">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002" aria-label="PubMed Central reference 94">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 94" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp352&amp;volume=25&amp;pages=2078-2079&amp;publication_year=2009&amp;author=Li%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="95."><p class="c-article-references__text" id="ref-CR95">Quinlan, A. R. &amp; Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. <i>Bioinformatics</i> <b>26</b>, 841–842 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btq033" data-track-item_id="10.1093/bioinformatics/btq033" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq033" aria-label="Article reference 95" data-doi="10.1093/bioinformatics/btq033">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXivFGkurc%3D" aria-label="CAS reference 95">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20110278" aria-label="PubMed reference 95">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832824" aria-label="PubMed Central reference 95">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 95" href="http://scholar.google.com/scholar_lookup?&amp;title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtq033&amp;volume=26&amp;pages=841-842&amp;publication_year=2010&amp;author=Quinlan%2CAR&amp;author=Hall%2CIM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="96."><p class="c-article-references__text" id="ref-CR96">Costea, P. I. et al. MetaSNV: a tool for metagenomic strain level analysis. <i>PLoS ONE</i> <b>12</b>, e0182392 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pone.0182392" data-track-item_id="10.1371/journal.pone.0182392" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pone.0182392" aria-label="Article reference 96" data-doi="10.1371/journal.pone.0182392">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28753663" aria-label="PubMed reference 96">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5533426" aria-label="PubMed Central reference 96">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 96" href="http://scholar.google.com/scholar_lookup?&amp;title=MetaSNV%3A%20a%20tool%20for%20metagenomic%20strain%20level%20analysis&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0182392&amp;volume=12&amp;publication_year=2017&amp;author=Costea%2CPI">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="97."><p class="c-article-references__text" id="ref-CR97">Schmidt, T. S. B. et al. Extensive transmission of microbes along the gastrointestinal tract. <i>eLife</i> <b>8</b>, e42693 (2019).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="98."><p class="c-article-references__text" id="ref-CR98">Friedman, J., Hastie, T. &amp; Tibshirani, R. Regularization paths for generalized linear models via coordinate descent. <i>J. Stat. Softw.</i> <b>33</b>, 1–22 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.18637/jss.v033.i01" data-track-item_id="10.18637/jss.v033.i01" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.18637%2Fjss.v033.i01" aria-label="Article reference 98" data-doi="10.18637/jss.v033.i01">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20808728" aria-label="PubMed reference 98">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929880" aria-label="PubMed Central reference 98">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 98" href="http://scholar.google.com/scholar_lookup?&amp;title=Regularization%20paths%20for%20generalized%20linear%20models%20via%20coordinate%20descent&amp;journal=J.%20Stat.%20Softw.&amp;doi=10.18637%2Fjss.v033.i01&amp;volume=33&amp;pages=1-22&amp;publication_year=2010&amp;author=Friedman%2CJ&amp;author=Hastie%2CT&amp;author=Tibshirani%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="99."><p class="c-article-references__text" id="ref-CR99">Schmidt, T. S. B. et al. Analysis data, “Drivers and Determinants of Strain Dynamics Following Fecal Microbiota Transplantation”. <i>Zenodo</i> <a href="https://doi.org/10.5281/ZENODO.5534163" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.5281/ZENODO.5534163">https://doi.org/10.5281/ZENODO.5534163</a> (2021).</p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01913-0?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank A. Moss (Harvard University, USA), C. Morrow (University of Alabama at Birmingham, USA), S. Leo (University of Geneva, Switzerland), R. Goll (University of Tromsø, Norway) and D. Podlesny (University of Hohenheim, Germany) for providing additional information on the cohorts used in this study. We further thank R. J. Alves, A. Schwartz, M. Kuhn, P. Ferretti, S. K. Forslund and members of the Bork laboratory at EMBL for support and constructive discussions. This work was supported by the European Research Council under the European Union’s Horizon2020 research and innovation program (nos. ERC-AdG-669830 to T.S.B.S., S.S.L., O.M.M., R.H., F.J. and P.B.; and ERC-AdG-686070 to L.P.C. and T.V.R.), by the German Federal Ministry of Education and Research (LAMarCK, no. 031L0181A to A.O.) and by the European Molecular Biology Laboratory (to W.A., A.F., A.M.G., S.K., R.T., M.v.S., V.B. and P.B.). S.S.L. acknowledges support from the European Molecular Biology Organisation (no. ALTF 137-2018) and the National Health and Medical Research Council of Australia (no. APP1166180). H.H. is supported by a Senior Fellowship of the Dutch Diabetes Research Foundation (no. 2019.82.004). W.M.d.V. is supported by the Spinoza 2008 award and a SIAM Gravitation grant from the Netherlands Organization for Scientific Research (no. 024.002.002). M.N. is supported by a ZONMW VICI grant 2020 (no. 09150182010020). The funders had no role in study design, data collection and analysis, decision to publish or preparation of the manuscript.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="nAff12"><p class="c-article-author-information__authors-list">Simone S. Li</p><p class="js-present-address">Present address: The Novo Nordisk Foundation Center for Biosustainability, Technical University of Denmark, Kongens Lyngby, Denmark</p></li><li class="c-article-author-information__item" id="nAff13"><p class="c-article-author-information__authors-list">Simone S. Li</p><p class="js-present-address">Present address: The University of Queensland, School of Chemistry and Molecular Biosciences, St Lucia, Queensland, Australia</p></li><li class="c-article-author-information__item" id="nAff14"><p class="c-article-author-information__authors-list">Oleksandr M. Maistrenko</p><p class="js-present-address">Present address: Department of Marine Microbiology &amp; Biogeochemistry, Royal Netherlands Institute for Sea Research, ‘t Horntje, the Netherlands</p></li><li class="c-article-author-information__item" id="nAff15"><p class="c-article-author-information__authors-list">Luis Pedro Coelho</p><p class="js-present-address">Present address: Key Laboratory of Computational Neuroscience and Brain-Inspired Intelligence (Fudan University), Ministry of Education, Shanghai, China</p></li><li class="c-article-author-information__item" id="na1"><p>These authors contributed equally: Thomas S. B. Schmidt, Simone S. Li.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Structural &amp; Computational Biology Unit, European Molecular Biology Laboratory, Heidelberg, Germany</p><p class="c-article-author-affiliation__authors-list">Thomas S. B. Schmidt,&nbsp;Simone S. Li,&nbsp;Oleksandr M. Maistrenko,&nbsp;Wasiu Akanni,&nbsp;Luis Pedro Coelho,&nbsp;Anthony Fullam,&nbsp;Anna M. Glazek,&nbsp;Stefanie Kandels,&nbsp;Askarbek Orakov,&nbsp;Roman Thielemann,&nbsp;Moritz von Stetten,&nbsp;Thea Van Rossum&nbsp;&amp;&nbsp;Peer Bork</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Centre for Digestive Diseases, Sydney, New South Wales, Australia</p><p class="c-article-author-affiliation__authors-list">Sibasish Dolai&nbsp;&amp;&nbsp;Thomas J. Borody</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Genomic Core Facility, European Molecular Biology Laboratory, Heidelberg, Germany</p><p class="c-article-author-affiliation__authors-list">Rajna Hercog,&nbsp;Ferris Jung&nbsp;&amp;&nbsp;Vladimir Benes</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Department of Experimental Vascular Medicine, Amsterdam University Medical Centers, Amsterdam, the Netherlands</p><p class="c-article-author-affiliation__authors-list">Hilde Herrema&nbsp;&amp;&nbsp;Max Nieuwdorp</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Laboratory of Microbiology, Wageningen University, Wageningen, the Netherlands</p><p class="c-article-author-affiliation__authors-list">Willem M. de Vos</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Human Microbiome Research Program, Faculty of Medicine, University of Helsinki, Helsinki, Finland</p><p class="c-article-author-affiliation__authors-list">Willem M. de Vos</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Department of Gastroenterology &amp; Hepatology, Amsterdam University Medical Centers, Amsterdam, the Netherlands</p><p class="c-article-author-affiliation__authors-list">Cyriel Y. Ponsioen</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Department of Vascular Medicine, Amsterdam University Medical Centers, Amsterdam, the Netherlands</p><p class="c-article-author-affiliation__authors-list">Max Nieuwdorp</p></li><li id="Aff9"><p class="c-article-author-affiliation__address">Max Delbrück Center for Molecular Medicine, Berlin, Germany</p><p class="c-article-author-affiliation__authors-list">Peer Bork</p></li><li id="Aff10"><p class="c-article-author-affiliation__address">Yonsei Frontier Lab, Yonsei University, Seoul, South Korea</p><p class="c-article-author-affiliation__authors-list">Peer Bork</p></li><li id="Aff11"><p class="c-article-author-affiliation__address">Department of Bioinformatics, Biocenter, University of Würzburg, Würzburg, Germany</p><p class="c-article-author-affiliation__authors-list">Peer Bork</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Thomas_S__B_-Schmidt-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Thomas S. B. Schmidt</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Thomas%20S.%20B.%20Schmidt" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Thomas%20S.%20B.%20Schmidt" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Thomas%20S.%20B.%20Schmidt%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Simone_S_-Li-Aff1-Aff12-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Simone S. Li</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Simone%20S.%20Li" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Simone%20S.%20Li" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Simone%20S.%20Li%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Oleksandr_M_-Maistrenko-Aff1-Aff14"><span class="c-article-authors-search__title u-h3 js-search-name">Oleksandr M. Maistrenko</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Oleksandr%20M.%20Maistrenko" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Oleksandr%20M.%20Maistrenko" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Oleksandr%20M.%20Maistrenko%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Wasiu-Akanni-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Wasiu Akanni</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Wasiu%20Akanni" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Wasiu%20Akanni" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Wasiu%20Akanni%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Luis_Pedro-Coelho-Aff1-Aff15"><span class="c-article-authors-search__title u-h3 js-search-name">Luis Pedro Coelho</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Luis%20Pedro%20Coelho" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Luis%20Pedro%20Coelho" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Luis%20Pedro%20Coelho%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sibasish-Dolai-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Sibasish Dolai</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sibasish%20Dolai" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sibasish%20Dolai" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sibasish%20Dolai%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Anthony-Fullam-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Anthony Fullam</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Anthony%20Fullam" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anthony%20Fullam" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anthony%20Fullam%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Anna_M_-Glazek-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Anna M. Glazek</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Anna%20M.%20Glazek" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anna%20M.%20Glazek" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anna%20M.%20Glazek%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Rajna-Hercog-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Rajna Hercog</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Rajna%20Hercog" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Rajna%20Hercog" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Rajna%20Hercog%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Hilde-Herrema-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Hilde Herrema</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Hilde%20Herrema" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Hilde%20Herrema" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Hilde%20Herrema%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ferris-Jung-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Ferris Jung</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ferris%20Jung" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ferris%20Jung" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ferris%20Jung%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Stefanie-Kandels-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Stefanie Kandels</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Stefanie%20Kandels" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Stefanie%20Kandels" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stefanie%20Kandels%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Askarbek-Orakov-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Askarbek Orakov</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Askarbek%20Orakov" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Askarbek%20Orakov" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Askarbek%20Orakov%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Roman-Thielemann-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Roman Thielemann</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Roman%20Thielemann" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Roman%20Thielemann" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Roman%20Thielemann%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Moritz-Stetten-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Moritz von Stetten</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Moritz%20von%20Stetten" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Moritz%20von%20Stetten" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Moritz%20von%20Stetten%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Thea-Van_Rossum-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Thea Van Rossum</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Thea%20Van%20Rossum" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Thea%20Van%20Rossum" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Thea%20Van%20Rossum%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Vladimir-Benes-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Vladimir Benes</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Vladimir%20Benes" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Vladimir%20Benes" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Vladimir%20Benes%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Thomas_J_-Borody-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Thomas J. Borody</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Thomas%20J.%20Borody" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Thomas%20J.%20Borody" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Thomas%20J.%20Borody%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Willem_M_-Vos-Aff5-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Willem M. de Vos</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Willem%20M.%20de%20Vos" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Willem%20M.%20de%20Vos" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Willem%20M.%20de%20Vos%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Cyriel_Y_-Ponsioen-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Cyriel Y. Ponsioen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Cyriel%20Y.%20Ponsioen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Cyriel%20Y.%20Ponsioen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Cyriel%20Y.%20Ponsioen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Max-Nieuwdorp-Aff4-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Max Nieuwdorp</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Max%20Nieuwdorp" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Max%20Nieuwdorp" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Max%20Nieuwdorp%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Peer-Bork-Aff1-Aff9-Aff10-Aff11"><span class="c-article-authors-search__title u-h3 js-search-name">Peer Bork</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Peer%20Bork" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Peer%20Bork" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Peer%20Bork%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>T.S.B.S., S.S.L., W.M.d.V., C.Y.P., M.N. and P.B. conceived and designed the study. S.D., H.H., T.J.B., C.Y.P. and M.N. managed the clinical studies and collection of stool samples and clinical data. W.M.d.V., M.N., S.K., R.H., F.J. and V.B. conducted and oversaw sample processing and metagenomic sequencing. W.A., A.F., A.M.G., A.O., L.P.C., R.T., M.V.S. and T.V.R. provided data processing and bioinformatic support. T.S.B.S., S.S.L., O.M.M. and P.B. processed and analyzed the data. T.S.B.S., S.S.L. and P.B. wrote the manuscript, with input from all authors.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:peer.bork@embl.org">Peer Bork</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar2">Competing interests</h3>
                <p>T.J.B. has a pecuniary interest in the Centre for Digestive Diseases in Australia and holds patents in the use of FMT for gastrointestinal diseases. C.Y.P. received funding grants from Gilead and Perspectum, speaker’s fees from Tillotts and consultancy fees from Shire and Pliant. M.N. and W.M.d.V. are founders and members of the Scientific Advisory Board of Caelus Health (the Netherlands). M.N. is a Scientific Advisory Board member of Kaleido Biosciences (USA). W.M.d.V. is founder and Scientific Advisory Board member of A-mansia Biotech (Belgium). These conflicts bear no relevance to the content of this manuscript.</p>
              
            </div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
              
              
                <h3 class="c-article__sub-heading" id="FPar1">Peer review information</h3>
                <p><i>Nature Medicine</i> thanks the anonymous reviewers for their contribution to the peer review of this work. Primary Handling editor: Alison Farrell, in collaboration with the <i>Nature Medicine</i> team.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec24-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec24">Supplementary information</h2><div class="c-article-section__content" id="Sec24-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Figs. 1–3.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary tables" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_MOESM3_ESM.xlsx" data-supp-info-image="">Supplementary Tables</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Tables 1–8.</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Drivers%20and%20determinants%20of%20strain%20dynamics%20following%20fecal%20microbiota%20transplantation&amp;author=Thomas%20S.%20B.%20Schmidt%20et%20al&amp;contentID=10.1038%2Fs41591-022-01913-0&amp;copyright=The%20Author%28s%29&amp;publication=1078-8956&amp;publicationDate=2022-09-15&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41591-022-01913-0" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41591-022-01913-0" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Schmidt, T.S.B., Li, S.S., Maistrenko, O.M. <i>et al.</i> Drivers and determinants of strain dynamics following fecal microbiota transplantation.
                    <i>Nat Med</i> <b>28</b>, 1902–1912 (2022). https://doi.org/10.1038/s41591-022-01913-0</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01913-0?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-10-05">05 October 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-06-23">23 June 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-09-15">15 September 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-09">September 2022</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41591-022-01913-0</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-block"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/clinical-microbiology" data-track="click" data-track-action="view subject" data-track-label="link">Clinical microbiology</a></li><li class="c-article-subject-list__subject"><a href="/subjects/metagenomics" data-track="click" data-track-action="view subject" data-track-label="link">Metagenomics</a></li><li class="c-article-subject-list__subject"><a href="/subjects/microbial-ecology" data-track="click" data-track-action="view subject" data-track-label="link">Microbial ecology</a></li><li class="c-article-subject-list__subject"><a href="/subjects/microbiome" data-track="click" data-track-action="view subject" data-track-label="link">Microbiome</a></li>
        </ul>
    </div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section" data-test="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Cannabinoid-2 receptor depletion promotes non-alcoholic fatty liver disease in mice via disturbing gut microbiota and tryptophan metabolism" href="https://doi.org/10.1038/s41401-025-01495-w">
                                        Cannabinoid-2 receptor depletion promotes non-alcoholic fatty liver disease in mice via disturbing gut microbiota and tryptophan metabolism
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Wei-ting Cheng</li><li>Si-ya Pei</li><li>Ze-bing Huang</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Acta Pharmacologica Sinica</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Human microbiome acquisition and transmission" href="https://doi.org/10.1038/s41579-025-01166-x">
                                        Human microbiome acquisition and transmission
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Vitor Heidrich</li><li>Mireia Valles-Colomer</li><li>Nicola Segata</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Reviews Microbiology</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Detecting microbial engraftment after FMT using placebo sequencing and culture enriched metagenomics to sort signals from noise" href="https://doi.org/10.1038/s41467-025-58673-x">
                                        Detecting microbial engraftment after FMT using placebo sequencing and culture enriched metagenomics to sort signals from noise
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Shahrokh Shekarriz</li><li>Jake C. Szamosi</li><li>Michael G. Surette</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Gut microbiota strain richness is species specific and affects engraftment" href="https://doi.org/10.1038/s41586-024-08242-x">
                                        Gut microbiota strain richness is species specific and affects engraftment
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Alice Chen-Liaw</li><li>Varun Aggarwala</li><li>Jeremiah J. Faith</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Fecal microbiota transplantation alters gut phage communities in a clinical trial for obesity" href="https://doi.org/10.1186/s40168-024-01833-w">
                                        Fecal microbiota transplantation alters gut phage communities in a clinical trial for obesity
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Michele Zuppi</li><li>Tommi Vatanen</li><li>Justin M. O’Sullivan</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Microbiome</i> (2024)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01913-0.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                
            
        
    </div>

    
        
    

    
    
        <div class="c-article-associated-content__container">
            <section>
                <h2 class="c-article-associated-content__title u-mb-24">Associated content</h2>
                
                    
                    
                        <div class="u-full-height u-mb-24">
                            
    <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
            <div class="c-card__body">
                <h3 class="c-card__title">
                    <a href="https://www.nature.com/articles/s41591-022-01964-3" class="c-card__link u-link-inherit" data-track="click" data-track-action="view article" data-track-category="associated content" data-track-label="research">Variability of strain engraftment and predictability of microbiome composition after fecal microbiota transplantation across different diseases</a>
                </h3>
                
<ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated">
    <li>Gianluca Ianiro</li><li>Michal Punčochář</li><li>Nicola Segata</li>
</ul>

                
    <div class="c-card__section c-meta">
        
            <span class="c-meta__item">Nature Medicine</span>
        
        <span class="c-meta__item" data-test="article.type"><span class="c-meta__type">Article</span></span>
        
            <span class="c-meta__item" data-test="open-access"><span class="u-color-open-access">Open Access</span></span>
        
        
            <time class="c-meta__item" datetime="2022-09-15">15 Sept 2022</time>
        
    </div>

            </div>
        </div>
    </article>


                        </div>
                    
                        <div class="u-full-height u-mb-24">
                            
    <article class="u-full-height c-card c-card--flush">
        <div class="c-card__layout u-full-height">
            <div class="c-card__body">
                <h3 class="c-card__title">
                    <a href="https://www.nature.com/articles/s41591-022-01991-0" class="c-card__link u-link-inherit" data-track="click" data-track-action="view article" data-track-category="associated content" data-track-label="news_and_views">Understanding and predicting the efficacy of FMT</a>
                </h3>
                
<ul data-test="author-list" class="app-author-list app-author-list--compact">
    <li>Aonghus Lavelle</li><li>Harry Sokol</li>
</ul>

                
    <div class="c-card__section c-meta">
        
            <span class="c-meta__item">Nature Medicine</span>
        
        <span class="c-meta__item" data-test="article.type"><span class="c-meta__type">News &amp; Views</span></span>
        
        
            <time class="c-meta__item" datetime="2022-09-15">15 Sept 2022</time>
        
    </div>

            </div>
        </div>
    </article>


                        </div>
                    
                
            </section>
        </div>
        <script>
            window.dataLayer = window.dataLayer || [];
            window.dataLayer[0] = window.dataLayer[0] || {};
            window.dataLayer[0].content = window.dataLayer[0].content || {};
            window.dataLayer[0].content.associatedContentTypes = "research;news_and_views";
            
        </script>
    

    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
            <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" role="tabpanel"><div class="c-reading-companion__scroll-pane" style="max-height: none;"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Main">Main</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec10" class="c-reading-companion__section-item"><a href="#Sec10" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec11" class="c-reading-companion__section-item"><a href="#Sec11" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-data-availability" class="c-reading-companion__section-item"><a href="#data-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Data availability">Data availability</a></li><li id="rc-sec-code-availability" class="c-reading-companion__section-item"><a href="#code-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Code availability">Code availability</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-peer-review" class="c-reading-companion__section-item"><a href="#peer-review" data-track="click" data-track-action="section anchor" data-track-label="link:Peer review">Peer review</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec24" class="c-reading-companion__section-item"><a href="#Sec24" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by">This article is cited by</a></li></ul></div>
                <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="" style="display: none !important;">
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-test="right-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41591-022-01913-0;doi=10.1038/s41591-022-01913-0;subjmeta=107,2134,2142,2565,326,631,855;kwrd=Clinical+microbiology,Metagenomics,Microbial+ecology,Microbiome">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=1067902395&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01913-0%26doi%3D10.1038/s41591-022-01913-0%26subjmeta%3D107,2134,2142,2565,326,631,855%26kwrd%3DClinical+microbiology,Metagenomics,Microbial+ecology,Microbiome">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=1067902395&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01913-0%26doi%3D10.1038/s41591-022-01913-0%26subjmeta%3D107,2134,2142,2565,326,631,855%26kwrd%3DClinical+microbiology,Metagenomics,Microbial+ecology,Microbiome"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Fig. 1: Study design and workflow overview.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig1_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig1_HTML.png" alt="figure 1" aria-describedby="rc-Fig1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01913-0/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Fig. 2: Community-wide FMT outcomes vary across patients and indications.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig2_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig2_HTML.png" alt="figure 2" aria-describedby="rc-Fig2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01913-0/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Fig. 3: Drivers and determinants of FMT community-level outcomes.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig3_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig3_HTML.png" alt="figure 3" aria-describedby="rc-Fig3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01913-0/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Fig. 4: Strain-level FMT outcomes vary between species but are predictable ex ante.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig4_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig4_HTML.png" alt="figure 4" aria-describedby="rc-Fig4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01913-0/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Fig. 5: Drivers and determinants of FMT strain-level outcomes for individual species.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig5_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig5_HTML.png" alt="figure 5" aria-describedby="rc-Fig5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01913-0/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig6">Fig. 6: FMT strain-level outcomes are shaped by both neutral and adaptive processes.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig6_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01913-0/MediaObjects/41591_2022_1913_Fig6_HTML.png" alt="figure 6" aria-describedby="rc-Fig6"></picture><p class="c-reading-companion__figure-links"><a href="#Fig6" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01913-0/figures/6" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric" data-track-context="article references sidebar"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">Borody, T. J. et al. Bacteriotherapy using fecal flora: toying with human motions. <i>J. Clin. Gastroenterol.</i> <b>38</b>, 475–483 (2004).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1097%2F01.mcg.0000128988.13808.dc" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1097/01.mcg.0000128988.13808.dc" data-track-item_id="10.1097/01.mcg.0000128988.13808.dc">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15220681" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Bacteriotherapy%20using%20fecal%20flora%3A%20toying%20with%20human%20motions&amp;journal=J.%20Clin.%20Gastroenterol.&amp;doi=10.1097%2F01.mcg.0000128988.13808.dc&amp;volume=38&amp;pages=475-483&amp;publication_year=2004&amp;author=Borody%2CTJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">Rossen, N. G. et al. Fecal microbiota transplantation as novel therapy in gastroenterology: a systematic review. <i>World J. Gastroenterol.</i> <b>21</b>, 5359–5371 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3748%2Fwjg.v21.i17.5359" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3748/wjg.v21.i17.5359" data-track-item_id="10.3748/wjg.v21.i17.5359">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25954111" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4419078" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplantation%20as%20novel%20therapy%20in%20gastroenterology%3A%20a%20systematic%20review&amp;journal=World%20J.%20Gastroenterol.&amp;doi=10.3748%2Fwjg.v21.i17.5359&amp;volume=21&amp;pages=5359-5371&amp;publication_year=2015&amp;author=Rossen%2CNG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Hanssen, N. M. J., de Vos, W. M. &amp; Nieuwdorp, M. Fecal microbiota transplantation in human metabolic diseases: from a murky past to a bright future? <i>Cell Metab.</i> <b>33</b>, 1098–1110 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cmet.2021.05.005" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cmet.2021.05.005" data-track-item_id="10.1016/j.cmet.2021.05.005">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1KhtLnP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34077717" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplantation%20in%20human%20metabolic%20diseases%3A%20from%20a%20murky%20past%20to%20a%20bright%20future%3F&amp;journal=Cell%20Metab.&amp;doi=10.1016%2Fj.cmet.2021.05.005&amp;volume=33&amp;pages=1098-1110&amp;publication_year=2021&amp;author=Hanssen%2CNMJ&amp;author=Vos%2CWM&amp;author=Nieuwdorp%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">Gough, E., Shaikh, H. &amp; Manges, A. R. Systematic review of intestinal microbiota transplantation (fecal bacteriotherapy) for recurrent <i>Clostridium difficile</i> infection. <i>Clin. Infect. Dis.</i> <b>53</b>, 994–1002 (2011).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fcid%2Fcir632" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/cid/cir632" data-track-item_id="10.1093/cid/cir632">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22002980" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20review%20of%20intestinal%20microbiota%20transplantation%20%28fecal%20bacteriotherapy%29%20for%20recurrent%20Clostridium%20difficile%20infection&amp;journal=Clin.%20Infect.%20Dis.&amp;doi=10.1093%2Fcid%2Fcir632&amp;volume=53&amp;pages=994-1002&amp;publication_year=2011&amp;author=Gough%2CE&amp;author=Shaikh%2CH&amp;author=Manges%2CAR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">van Nood, E. et al. Duodenal infusion of donor feces for recurrent <i>Clostridium difficile</i>. <i>N. Engl. J. Med.</i> <b>368</b>, 407–415 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa1205037" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa1205037" data-track-item_id="10.1056/NEJMoa1205037">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23323867" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Duodenal%20infusion%20of%20donor%20feces%20for%20recurrent%20Clostridium%20difficile&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1205037&amp;volume=368&amp;pages=407-415&amp;publication_year=2013&amp;author=Nood%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Narula, N. et al. Systematic review and meta-analysis fecal microbiota transplantation for treatment of active ulcerative colitis. <i>Inflamm. Bowel Dis.</i> <b>23</b>, 1702–1709 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1097%2FMIB.0000000000001228" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1097/MIB.0000000000001228" data-track-item_id="10.1097/MIB.0000000000001228">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28906291" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Systematic%20review%20and%20meta-analysis%20fecal%20microbiota%20transplantation%20for%20treatment%20of%20active%20ulcerative%20colitis&amp;journal=Inflamm.%20Bowel%20Dis.&amp;doi=10.1097%2FMIB.0000000000001228&amp;volume=23&amp;pages=1702-1709&amp;publication_year=2017&amp;author=Narula%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Haifer, C., Leong, R. W. &amp; Paramsothy, S. The role of fecal microbiota transplantation in the treatment of inflammatory bowel disease. <i>Curr. Opin. Pharmacol.</i> <b>55</b>, 8–16 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.coph.2020.08.009" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.coph.2020.08.009" data-track-item_id="10.1016/j.coph.2020.08.009">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvV2msr3F" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33035780" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7538387" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20fecal%20microbiota%20transplantation%20in%20the%20treatment%20of%20inflammatory%20bowel%20disease&amp;journal=Curr.%20Opin.%20Pharmacol.&amp;doi=10.1016%2Fj.coph.2020.08.009&amp;volume=55&amp;pages=8-16&amp;publication_year=2020&amp;author=Haifer%2CC&amp;author=Leong%2CRW&amp;author=Paramsothy%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Suez, J. et al. Post-antibiotic gut mucosal microbiome reconstitution is impaired by probiotics and improved by autologous FMT. <i>Cell</i> <b>174</b>, 1406–1423 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2018.08.047" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2018.08.047" data-track-item_id="10.1016/j.cell.2018.08.047">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1Omu7bP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30193113" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Post-antibiotic%20gut%20mucosal%20microbiome%20reconstitution%20is%20impaired%20by%20probiotics%20and%20improved%20by%20autologous%20FMT&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.08.047&amp;volume=174&amp;pages=1406-1423&amp;publication_year=2018&amp;author=Suez%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Baruch, E. N. et al. Fecal microbiota transplant promotes response in immunotherapy-refractory melanoma patients. <i>Science</i> <b>371</b>, 602–609 (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Davar, D. et al. Fecal microbiota transplant overcomes resistance to anti-PD-1 therapy in melanoma patients. <i>Science</i> <b>371</b>, 595–602 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.abf3363" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.abf3363" data-track-item_id="10.1126/science.abf3363">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjsFCrs7s%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33542131" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8097968" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplant%20overcomes%20resistance%20to%20anti-PD-1%20therapy%20in%20melanoma%20patients&amp;journal=Science&amp;doi=10.1126%2Fscience.abf3363&amp;volume=371&amp;pages=595-602&amp;publication_year=2021&amp;author=Davar%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Burrello, C. et al. Therapeutic fecal microbiota transplantation controls intestinal inflammation through IL10 secretion by immune cells. <i>Nat. Commun.</i> <b>9</b>, 5184 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-018-07359-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-018-07359-8" data-track-item_id="10.1038/s41467-018-07359-8">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30518790" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6281577" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Therapeutic%20fecal%20microbiota%20transplantation%20controls%20intestinal%20inflammation%20through%20IL10%20secretion%20by%20immune%20cells&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-07359-8&amp;volume=9&amp;publication_year=2018&amp;author=Burrello%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Seekatz, A. M. et al. Restoration of short chain fatty acid and bile acid metabolism following fecal microbiota transplantation in patients with recurrent <i>Clostridium difficile</i> infection. <i>Anaerobe</i> <b>53</b>, 64–73 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.anaerobe.2018.04.001" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.anaerobe.2018.04.001" data-track-item_id="10.1016/j.anaerobe.2018.04.001">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXos1Kgs70%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29654837" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6185828" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Restoration%20of%20short%20chain%20fatty%20acid%20and%20bile%20acid%20metabolism%20following%20fecal%20microbiota%20transplantation%20in%20patients%20with%20recurrent%20Clostridium%20difficile%20infection&amp;journal=Anaerobe&amp;doi=10.1016%2Fj.anaerobe.2018.04.001&amp;volume=53&amp;pages=64-73&amp;publication_year=2018&amp;author=Seekatz%2CAM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Zuo, T. et al. Bacteriophage transfer during fecal microbiota transplantation is associated with treatment response in <i>Clostridium difficile</i> infection. <i>Gastroenterology</i> <b>152</b>, S140–S141 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0016-5085%2817%2930798-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0016-5085(17)30798-9" data-track-item_id="10.1016/S0016-5085(17)30798-9">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Bacteriophage%20transfer%20during%20fecal%20microbiota%20transplantation%20is%20associated%20with%20treatment%20response%20in%20Clostridium%20difficile%20infection&amp;journal=Gastroenterology&amp;doi=10.1016%2FS0016-5085%2817%2930798-9&amp;volume=152&amp;pages=S140-S141&amp;publication_year=2017&amp;author=Zuo%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Manrique, P. et al. Gut bacteriophage dynamics during fecal microbial transplantation in subjects with metabolic syndrome. <i>Gut Microbes</i> <b>13</b>, 1–15 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1080%2F19490976.2021.1897217" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1080/19490976.2021.1897217" data-track-item_id="10.1080/19490976.2021.1897217">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33794724" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Gut%20bacteriophage%20dynamics%20during%20fecal%20microbial%20transplantation%20in%20subjects%20with%20metabolic%20syndrome&amp;journal=Gut%20Microbes&amp;doi=10.1080%2F19490976.2021.1897217&amp;volume=13&amp;pages=1-15&amp;publication_year=2021&amp;author=Manrique%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">Wilson, B. C., Vatanen, T., Cutfield, W. S. &amp; O’Sullivan, J. M. The super-donor phenomenon in fecal microbiota transplantation. <i>Front. Cell. Infect. Microbiol.</i> <b>9</b>, 2 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3389%2Ffcimb.2019.00002" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3389/fcimb.2019.00002" data-track-item_id="10.3389/fcimb.2019.00002">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXit1KiurrE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30719428" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6348388" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20super-donor%20phenomenon%20in%20fecal%20microbiota%20transplantation&amp;journal=Front.%20Cell.%20Infect.%20Microbiol.&amp;doi=10.3389%2Ffcimb.2019.00002&amp;volume=9&amp;publication_year=2019&amp;author=Wilson%2CBC&amp;author=Vatanen%2CT&amp;author=Cutfield%2CWS&amp;author=O%E2%80%99Sullivan%2CJM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">Duvallet, C. et al. Framework for rational donor selection in fecal microbiota transplant clinical trials. <i>PloS ONE</i> <b>14</b>, e0222881 (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Olesen, S. W. &amp; Gerardin, Y. Re-evaluating the evidence for fecal microbiota transplantation “super-donors” in inflammatory bowel disease. <i>J. Crohns Colitis</i> <b>15</b>, 453–461 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fecco-jcc%2Fjjaa170" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/ecco-jcc/jjaa170" data-track-item_id="10.1093/ecco-jcc/jjaa170">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32808030" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Re-evaluating%20the%20evidence%20for%20fecal%20microbiota%20transplantation%20%E2%80%9Csuper-donors%E2%80%9D%20in%20inflammatory%20bowel%20disease&amp;journal=J.%20Crohns%20Colitis&amp;doi=10.1093%2Fecco-jcc%2Fjjaa170&amp;volume=15&amp;pages=453-461&amp;publication_year=2021&amp;author=Olesen%2CSW&amp;author=Gerardin%2CY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">Kootte, R. S. et al. Improvement of insulin sensitivity after lean donor feces in metabolic syndrome is driven by baseline intestinal imcrobiota composition. <i>Cell Metab.</i> <b>26</b>, 611–619 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cmet.2017.09.008" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cmet.2017.09.008" data-track-item_id="10.1016/j.cmet.2017.09.008">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhs1Siur7J" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28978426" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Improvement%20of%20insulin%20sensitivity%20after%20lean%20donor%20feces%20in%20metabolic%20syndrome%20is%20driven%20by%20baseline%20intestinal%20imcrobiota%20composition&amp;journal=Cell%20Metab.&amp;doi=10.1016%2Fj.cmet.2017.09.008&amp;volume=26&amp;pages=611-619&amp;publication_year=2017&amp;author=Kootte%2CRS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">Danne, C., Rolhion, N. &amp; Sokol, H. Recipient factors in fecal microbiota transplantation: one stool does not fit all. <i>Nat. Rev. Gastroenterol. Hepatol</i>. <b>18</b>, 503–513 (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Fujimoto, K. et al. Functional restoration of bacteriomes and viromes by fecal microbiota transplantation. <i>Gastroenterology</i> <b>160</b>, 2089–2102 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1053%2Fj.gastro.2021.02.013" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1053/j.gastro.2021.02.013" data-track-item_id="10.1053/j.gastro.2021.02.013">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33577875" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Functional%20restoration%20of%20bacteriomes%20and%20viromes%20by%20fecal%20microbiota%20transplantation&amp;journal=Gastroenterology&amp;doi=10.1053%2Fj.gastro.2021.02.013&amp;volume=160&amp;pages=2089-2102&amp;publication_year=2021&amp;author=Fujimoto%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Peri, R. et al. The impact of technical and clinical factors on fecal microbiota transfer outcomes for the treatment of recurrent <i>Clostridioides difficile</i> infections in Germany. <i>United European Gastroenterol. J.</i> <b>7</b>, 716–722 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1177%2F2050640619839918" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1177/2050640619839918" data-track-item_id="10.1177/2050640619839918">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31210950" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6545715" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20impact%20of%20technical%20and%20clinical%20factors%20on%20fecal%20microbiota%20transfer%20outcomes%20for%20the%20treatment%20of%20recurrent%20Clostridioides%20difficile%20infections%20in%20Germany&amp;journal=United%20European%20Gastroenterol.%20J.&amp;doi=10.1177%2F2050640619839918&amp;volume=7&amp;pages=716-722&amp;publication_year=2019&amp;author=Peri%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Draper, L. A. et al. Long-term colonisation with donor bacteriophages following successful fecal microbial transplantation. <i>Microbiome</i> <b>6</b>, 220 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s40168-018-0598-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s40168-018-0598-x" data-track-item_id="10.1186/s40168-018-0598-x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:STN:280:DC%2BB3crpslWqsw%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30526683" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6288847" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Long-term%20colonisation%20with%20donor%20bacteriophages%20following%20successful%20fecal%20microbial%20transplantation&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-018-0598-x&amp;volume=6&amp;publication_year=2018&amp;author=Draper%2CLA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">Leonardi, I. et al. Fungal trans-kingdom dynamics linked to responsiveness to fecal microbiota transplantation (FMT) therapy in ulcerative colitis. <i>Cell Host Microbe</i> <b>27</b>, 823–829 (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Zuo, T. et al. Gut fungal dysbiosis correlates with reduced efficacy of fecal microbiota transplantation in <i>Clostridium difficile</i> infection. <i>Nat. Commun.</i> <b>9</b>, 3663 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-018-06103-6" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-018-06103-6" data-track-item_id="10.1038/s41467-018-06103-6">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30202057" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6131390" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Gut%20fungal%20dysbiosis%20correlates%20with%20reduced%20efficacy%20of%20fecal%20microbiota%20transplantation%20in%20Clostridium%20difficile%20infection&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-06103-6&amp;volume=9&amp;publication_year=2018&amp;author=Zuo%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">Li, S. S. et al. Durable coexistence of donor and recipient strains after fecal microbiota transplantation. <i>Science</i> <b>352</b>, 586–589 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aad8852" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aad8852" data-track-item_id="10.1126/science.aad8852">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xms1KisLg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27126044" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Durable%20coexistence%20of%20donor%20and%20recipient%20strains%20after%20fecal%20microbiota%20transplantation&amp;journal=Science&amp;doi=10.1126%2Fscience.aad8852&amp;volume=352&amp;pages=586-589&amp;publication_year=2016&amp;author=Li%2CSS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Smillie, C. S. et al. Strain tracking reveals the determinants of bacterial engraftment in the human gut following fecal microbiota transplantation. <i>Cell Host Microbe</i> <b>23</b>, 229–240 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.chom.2018.01.003" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.chom.2018.01.003" data-track-item_id="10.1016/j.chom.2018.01.003">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXivVSlsr0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29447696" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8318347" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Strain%20tracking%20reveals%20the%20determinants%20of%20bacterial%20engraftment%20in%20the%20human%20gut%20following%20fecal%20microbiota%20transplantation&amp;journal=Cell%20Host%20Microbe&amp;doi=10.1016%2Fj.chom.2018.01.003&amp;volume=23&amp;pages=229-240&amp;publication_year=2018&amp;author=Smillie%2CCS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">Podlesny, D. et al. Metagenomic strain detection with SameStr: identification of a persisting core gut microbiota transferable by fecal transplantation. <i>Microbiome</i> <b>10</b>, 53 (2022).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s40168-022-01251-w" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s40168-022-01251-w" data-track-item_id="10.1186/s40168-022-01251-w">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xht1GqsLvE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35337386" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8951724" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Metagenomic%20strain%20detection%20with%20SameStr%3A%20identification%20of%20a%20persisting%20core%20gut%20microbiota%20transferable%20by%20fecal%20transplantation&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-022-01251-w&amp;volume=10&amp;publication_year=2022&amp;author=Podlesny%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Aggarwala, V. et al. Precise quantification of bacterial strains after fecal microbiota transplantation delineates long-term engraftment and explains outcomes. <i>Nat. Microbiol.</i> <b>6</b>, 1309–1318 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41564-021-00966-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41564-021-00966-0" data-track-item_id="10.1038/s41564-021-00966-0">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFamtrrL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34580445" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8993687" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Precise%20quantification%20of%20bacterial%20strains%20after%20fecal%20microbiota%20transplantation%20delineates%20long-term%20engraftment%20and%20explains%20outcomes&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-021-00966-0&amp;volume=6&amp;pages=1309-1318&amp;publication_year=2021&amp;author=Aggarwala%2CV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">Lee, S. T. M. et al. Tracking microbial colonization in fecal microbiota transplantation experiments via genome-resolved metagenomics. <i>Microbiome</i> <b>5</b>, 50 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s40168-017-0270-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s40168-017-0270-x" data-track-item_id="10.1186/s40168-017-0270-x">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28473000" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5418705" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Tracking%20microbial%20colonization%20in%20fecal%20microbiota%20transplantation%20experiments%20via%20genome-resolved%20metagenomics&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-017-0270-x&amp;volume=5&amp;publication_year=2017&amp;author=Lee%2CSTM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">Wilson, B. C. et al. Strain engraftment competition and functional augmentation in a multi-donor fecal microbiota transplantation trial for obesity. <i>Microbiome</i> <b>9</b>, 107 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s40168-021-01060-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s40168-021-01060-7" data-track-item_id="10.1186/s40168-021-01060-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XjsVems7c%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33985595" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8120839" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Strain%20engraftment%20competition%20and%20functional%20augmentation%20in%20a%20multi-donor%20fecal%20microbiota%20transplantation%20trial%20for%20obesity&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-021-01060-7&amp;volume=9&amp;publication_year=2021&amp;author=Wilson%2CBC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Ng, S. C. et al. Microbiota engraftment after fecal microbiota transplantation in obese subjects with type 2 diabetes: a 24-week, double-blind, randomised controlled trial. <i>Gut</i> <b>71</b>, 716–723 (2022).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1136%2Fgutjnl-2020-323617" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1136/gutjnl-2020-323617" data-track-item_id="10.1136/gutjnl-2020-323617">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XhsFyqu7vJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33785557" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Microbiota%20engraftment%20after%20fecal%20microbiota%20transplantation%20in%20obese%20subjects%20with%20type%202%20diabetes%3A%20a%2024-week%2C%20double-blind%2C%20randomised%20controlled%20trial&amp;journal=Gut&amp;doi=10.1136%2Fgutjnl-2020-323617&amp;volume=71&amp;pages=716-723&amp;publication_year=2022&amp;author=Ng%2CSC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Watson, A. R., Fuessel, J., Veseli, I. &amp; DeLongchamp, J. Z. Adaptive ecological processes and metabolic independence drive microbial colonization and resilience in the human gut. Preprint at <a href="https://doi.org/10.1101/2021.03.02.433653" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.03.02.433653">https://doi.org/10.1101/2021.03.02.433653</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">Walter, J., Maldonado-Gómez, M. X. &amp; Martínez, I. To engraft or not to engraft: an ecological framework for gut microbiome modulation with live microbes. <i>Curr. Opin. Biotechnol.</i> <b>49</b>, 129–139 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.copbio.2017.08.008" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.copbio.2017.08.008" data-track-item_id="10.1016/j.copbio.2017.08.008">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlylsLnE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28866242" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=To%20engraft%20or%20not%20to%20engraft%3A%20an%20ecological%20framework%20for%20gut%20microbiome%20modulation%20with%20live%20microbes&amp;journal=Curr.%20Opin.%20Biotechnol.&amp;doi=10.1016%2Fj.copbio.2017.08.008&amp;volume=49&amp;pages=129-139&amp;publication_year=2018&amp;author=Walter%2CJ&amp;author=Maldonado-G%C3%B3mez%2CMX&amp;author=Mart%C3%ADnez%2CI" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">Schmidt, T. S. B., Raes, J. &amp; Bork, P. The human gut microbiome: from association to modulation. <i>Cell</i> <b>172</b>, 1198–1215 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2018.02.044" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2018.02.044" data-track-item_id="10.1016/j.cell.2018.02.044">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXktlSksbo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29522742" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20human%20gut%20microbiome%3A%20from%20association%20to%20modulation&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2018.02.044&amp;volume=172&amp;pages=1198-1215&amp;publication_year=2018&amp;author=Schmidt%2CTSB&amp;author=Raes%2CJ&amp;author=Bork%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">Xiao, Y., Angulo, M. T., Lao, S., Weiss, S. T. &amp; Liu, Y.-Y. An ecological framework to understand the efficacy of fecal microbiota transplantation. <i>Nat. Commun.</i> <b>11</b>, 3329 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-17180-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-17180-x" data-track-item_id="10.1038/s41467-020-17180-x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtlGmsrfJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32620839" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334230" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20ecological%20framework%20to%20understand%20the%20efficacy%20of%20fecal%20microbiota%20transplantation&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-17180-x&amp;volume=11&amp;publication_year=2020&amp;author=Xiao%2CY&amp;author=Angulo%2CMT&amp;author=Lao%2CS&amp;author=Weiss%2CST&amp;author=Liu%2CY-Y" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Moss, E. L. et al. Long-term taxonomic and functional divergence from donor bacterial strains following fecal microbiota transplantation in immunocompromised patients. <i>PLoS ONE</i> <b>12</b>, e0182585 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pone.0182585" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pone.0182585" data-track-item_id="10.1371/journal.pone.0182585">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28827811" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5565110" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Long-term%20taxonomic%20and%20functional%20divergence%20from%20donor%20bacterial%20strains%20following%20fecal%20microbiota%20transplantation%20in%20immunocompromised%20patients&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0182585&amp;volume=12&amp;publication_year=2017&amp;author=Moss%2CEL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Singh, R. et al. Fecal microbiota transplantation against intestinal colonization by extended spectrum beta-lactamase producing Enterobacteriaceae: a proof of principle study. <i>BMC Res. Notes</i> <b>11</b>, 190 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13104-018-3293-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13104-018-3293-x" data-track-item_id="10.1186/s13104-018-3293-x">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29566738" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5863815" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplantation%20against%20intestinal%20colonization%20by%20extended%20spectrum%20beta-lactamase%20producing%20Enterobacteriaceae%3A%20a%20proof%20of%20principle%20study&amp;journal=BMC%20Res.%20Notes&amp;doi=10.1186%2Fs13104-018-3293-x&amp;volume=11&amp;publication_year=2018&amp;author=Singh%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Bar-Yoseph, H. et al. Oral capsulized fecal microbiota transplantation for eradication of carbapenemase-producing Enterobacteriaceae colonization with a metagenomic perspective. <i>Clin. Infect. Dis</i>. <b>73</b>, e166–e175 (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">Leo, S. et al. Metagenomic characterization of gut microbiota of carriers of extended-spectrum beta-lactamase or carbapenemase-producing Enterobacteriaceae following treatment with oral antibiotics and fecal microbiota transplantation: results from a multicenter randomized trial. <i>Microorganisms</i> <b>8</b>, 941 (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">Koopen, A. M. et al. Effect of fecal microbiota transplantation combined with Mediterranean diet on insulin sensitivity in subjects with metabolic syndrome. <i>Front. Microbiol.</i> <b>12</b>, 662159 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3389%2Ffmicb.2021.662159" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3389/fmicb.2021.662159" data-track-item_id="10.3389/fmicb.2021.662159">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34177842" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8222733" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Effect%20of%20fecal%20microbiota%20transplantation%20combined%20with%20Mediterranean%20diet%20on%20insulin%20sensitivity%20in%20subjects%20with%20metabolic%20syndrome&amp;journal=Front.%20Microbiol.&amp;doi=10.3389%2Ffmicb.2021.662159&amp;volume=12&amp;publication_year=2021&amp;author=Koopen%2CAM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">Rossen, N. G. et al. Findings from a randomized controlled trial of fecal transplantation for patients with ulcerative colitis. <i>Gastroenterology</i> <b>149</b>, 110–118 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1053%2Fj.gastro.2015.03.045" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1053/j.gastro.2015.03.045" data-track-item_id="10.1053/j.gastro.2015.03.045">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25836986" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Findings%20from%20a%20randomized%20controlled%20trial%20of%20fecal%20transplantation%20for%20patients%20with%20ulcerative%20colitis&amp;journal=Gastroenterology&amp;doi=10.1053%2Fj.gastro.2015.03.045&amp;volume=149&amp;pages=110-118&amp;publication_year=2015&amp;author=Rossen%2CNG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">Nusbaum, D. J. et al. Gut microbial and metabolomic profiles after fecal microbiota transplantation in pediatric ulcerative colitis patients. <i>FEMS Microbiol. Ecol</i>. <b>94</b>, fiy133 (2018).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">Damman, C. J. et al. Low level engraftment and improvement following a single colonoscopic administration of fecal microbiota to patients with ulcerative colitis. <i>PLoS ONE</i> <b>10</b>, e0133925 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pone.0133925" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pone.0133925" data-track-item_id="10.1371/journal.pone.0133925">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26288277" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4544847" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Low%20level%20engraftment%20and%20improvement%20following%20a%20single%20colonoscopic%20administration%20of%20fecal%20microbiota%20to%20patients%20with%20ulcerative%20colitis&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0133925&amp;volume=10&amp;publication_year=2015&amp;author=Damman%2CCJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">Goll, R. et al. Effects of fecal microbiota transplantation in subjects with irritable bowel syndrome are mirrored by changes in gut microbiome. <i>Gut Microbes</i> <b>12</b>, 1794263 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1080%2F19490976.2020.1794263" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1080/19490976.2020.1794263" data-track-item_id="10.1080/19490976.2020.1794263">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32991818" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7583512" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Effects%20of%20fecal%20microbiota%20transplantation%20in%20subjects%20with%20irritable%20bowel%20syndrome%20are%20mirrored%20by%20changes%20in%20gut%20microbiome&amp;journal=Gut%20Microbes&amp;doi=10.1080%2F19490976.2020.1794263&amp;volume=12&amp;publication_year=2020&amp;author=Goll%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">Vaughn, B. P. et al. Increased intestinal microbial diversity following fecal microbiota transplant for active Crohn’s disease. <i>Inflamm. Bowel Dis.</i> <b>22</b>, 2182–2190 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1097%2FMIB.0000000000000893" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1097/MIB.0000000000000893" data-track-item_id="10.1097/MIB.0000000000000893">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27542133" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Increased%20intestinal%20microbial%20diversity%20following%20fecal%20microbiota%20transplant%20for%20active%20Crohn%E2%80%99s%20disease&amp;journal=Inflamm.%20Bowel%20Dis.&amp;doi=10.1097%2FMIB.0000000000000893&amp;volume=22&amp;pages=2182-2190&amp;publication_year=2016&amp;author=Vaughn%2CBP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR46">Ianiro, G. et al. Fecal microbiota transplantation for the treatment of diarrhoea induced by tyrosine-kinase inhibitors in patients with metastatic renal cell carcinoma. <i>Nat. Commun.</i> <b>11</b>, 4333 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-18127-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-18127-y" data-track-item_id="10.1038/s41467-020-18127-y">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32859933" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7455693" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplantation%20for%20the%20treatment%20of%20diarrhoea%20induced%20by%20tyrosine-kinase%20inhibitors%20in%20patients%20with%20metastatic%20renal%20cell%20carcinoma&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-18127-y&amp;volume=11&amp;publication_year=2020&amp;author=Ianiro%2CG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR47">Zhao, H.-J. et al. The efficacy of fecal microbiota transplantation for children with Tourette syndrome: a preliminary study. <i>Front. Psychiatry</i> <b>11</b>, 554441 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3389%2Ffpsyt.2020.554441" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3389/fpsyt.2020.554441" data-track-item_id="10.3389/fpsyt.2020.554441">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33424650" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7793740" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20efficacy%20of%20fecal%20microbiota%20transplantation%20for%20children%20with%20Tourette%20syndrome%3A%20a%20preliminary%20study&amp;journal=Front.%20Psychiatry&amp;doi=10.3389%2Ffpsyt.2020.554441&amp;volume=11&amp;publication_year=2020&amp;author=Zhao%2CH-J" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR48">Goloshchapov, O. V. et al. Long-term impact of fecal transplantation in healthy volunteers. <i>BMC Microbiol.</i> <b>19</b>, 312 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s12866-019-1689-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s12866-019-1689-y" data-track-item_id="10.1186/s12866-019-1689-y">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVSrsQ%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31888470" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6938016" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Long-term%20impact%20of%20fecal%20transplantation%20in%20healthy%20volunteers&amp;journal=BMC%20Microbiol.&amp;doi=10.1186%2Fs12866-019-1689-y&amp;volume=19&amp;publication_year=2019&amp;author=Goloshchapov%2COV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR49">Tian, L. et al. Deciphering functional redundancy in the human microbiome. <i>Nat. Commun.</i> <b>11</b>, 6217 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-19940-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-19940-1" data-track-item_id="10.1038/s41467-020-19940-1">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisFems77N" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33277504" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7719190" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Deciphering%20functional%20redundancy%20in%20the%20human%20microbiome&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19940-1&amp;volume=11&amp;publication_year=2020&amp;author=Tian%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR50">Hildebrand, F. et al. Dispersal strategies shape persistence and evolution of human gut bacteria. <i>Cell Host Microbe</i> <b>29</b>, 1167–1176 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.chom.2021.05.008" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.chom.2021.05.008" data-track-item_id="10.1016/j.chom.2021.05.008">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtlalsL7O" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34111423" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8288446" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Dispersal%20strategies%20shape%20persistence%20and%20evolution%20of%20human%20gut%20bacteria&amp;journal=Cell%20Host%20Microbe&amp;doi=10.1016%2Fj.chom.2021.05.008&amp;volume=29&amp;pages=1167-1176&amp;publication_year=2021&amp;author=Hildebrand%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR51">Lahti, L., Salojärvi, J., Salonen, A., Scheffer, M. &amp; de Vos, W. M. Tipping elements in the human intestinal ecosystem. <i>Nat. Commun.</i> <b>5</b>, 4344 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fncomms5344" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ncomms5344" data-track-item_id="10.1038/ncomms5344">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitVWgsLjJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25003530" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Tipping%20elements%20in%20the%20human%20intestinal%20ecosystem&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fncomms5344&amp;volume=5&amp;publication_year=2014&amp;author=Lahti%2CL&amp;author=Saloj%C3%A4rvi%2CJ&amp;author=Salonen%2CA&amp;author=Scheffer%2CM&amp;author=Vos%2CWM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR52">Scheffer, M., Carpenter, S. R., Dakos, V. &amp; van Nes, E. H. Generic indicators of ecological resilience: inferring the chance of a critical transition. <i>Annu. Rev. Ecol. Evol. Syst</i>. <b>46</b>, 145–167 (2015).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR53">Gonze, D., Lahti, L., Raes, J. &amp; Faust, K. Multi-stability and the origin of microbial community types. <i>ISME J.</i> <b>11</b>, 2159–2166 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fismej.2017.60" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ismej.2017.60" data-track-item_id="10.1038/ismej.2017.60">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28475180" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5607358" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Multi-stability%20and%20the%20origin%20of%20microbial%20community%20types&amp;journal=ISME%20J.&amp;doi=10.1038%2Fismej.2017.60&amp;volume=11&amp;publication_year=2017&amp;author=Gonze%2CD&amp;author=Lahti%2CL&amp;author=Raes%2CJ&amp;author=Faust%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR54">Costea, P. I. et al. Enterotypes in the landscape of gut microbial community composition. <i>Nat. Microbiol.</i> <b>3</b>, 8–16 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41564-017-0072-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41564-017-0072-8" data-track-item_id="10.1038/s41564-017-0072-8">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVSmtLvK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29255284" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Enterotypes%20in%20the%20landscape%20of%20gut%20microbial%20community%20composition&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-017-0072-8&amp;volume=3&amp;pages=8-16&amp;publication_year=2018&amp;author=Costea%2CPI" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR55">Debray, R. et al. Priority effects in microbiome assembly. <i>Nat. Rev. Microbiol</i>. <b>20</b>, 109–121 (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR56">Zaneveld, J. R., McMinds, R. &amp; Thurber, R. V. Stress and stability: applying the Anna Karenina principle to animal microbiomes. <i>Nat. Microbiol.</i> <b>2</b>, 17121 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnmicrobiol.2017.121" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nmicrobiol.2017.121" data-track-item_id="10.1038/nmicrobiol.2017.121">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtlyit7vK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28836573" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Stress%20and%20stability%3A%20applying%20the%20Anna%20Karenina%20principle%20to%20animal%20microbiomes&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fnmicrobiol.2017.121&amp;volume=2&amp;publication_year=2017&amp;author=Zaneveld%2CJR&amp;author=McMinds%2CR&amp;author=Thurber%2CRV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR57">Basson, A. R., Zhou, Y., Seo, B., Rodriguez-Palacios, A. &amp; Cominelli, F. Autologous fecal microbiota transplantation for the treatment of inflammatory bowel disease. <i>Transl. Res.</i> <b>226</b>, 1–11 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.trsl.2020.05.008" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.trsl.2020.05.008" data-track-item_id="10.1016/j.trsl.2020.05.008">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32585148" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7308243" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Autologous%20fecal%20microbiota%20transplantation%20for%20the%20treatment%20of%20inflammatory%20bowel%20disease&amp;journal=Transl.%20Res.&amp;doi=10.1016%2Fj.trsl.2020.05.008&amp;volume=226&amp;pages=1-11&amp;publication_year=2020&amp;author=Basson%2CAR&amp;author=Zhou%2CY&amp;author=Seo%2CB&amp;author=Rodriguez-Palacios%2CA&amp;author=Cominelli%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR58">de Groot, P. et al. Fecal microbiota transplantation halts progression of human new-onset type 1 diabetes in a randomised controlled trial. <i>Gut</i> <b>70</b>, 92–105 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1136%2Fgutjnl-2020-322630" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1136/gutjnl-2020-322630" data-track-item_id="10.1136/gutjnl-2020-322630">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33106354" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20microbiota%20transplantation%20halts%20progression%20of%20human%20new-onset%20type%201%20diabetes%20in%20a%20randomised%20controlled%20trial&amp;journal=Gut&amp;doi=10.1136%2Fgutjnl-2020-322630&amp;volume=70&amp;pages=92-105&amp;publication_year=2021&amp;author=Groot%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR59">Ott, S. J. et al. Efficacy of sterile fecal filtrate transfer for treating patients with <i>Clostridium difficile</i> infection. <i>Gastroenterology</i> <b>152</b>, 799–811 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1053%2Fj.gastro.2016.11.010" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1053/j.gastro.2016.11.010" data-track-item_id="10.1053/j.gastro.2016.11.010">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27866880" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Efficacy%20of%20sterile%20fecal%20filtrate%20transfer%20for%20treating%20patients%20with%20Clostridium%20difficile%20infection&amp;journal=Gastroenterology&amp;doi=10.1053%2Fj.gastro.2016.11.010&amp;volume=152&amp;pages=799-811&amp;publication_year=2017&amp;author=Ott%2CSJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR60">Bojanova, D. P. &amp; Bordenstein, S. R. Fecal transplants: what is being transferred? <i>PLoS Biol.</i> <b>14</b>, e1002503 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pbio.1002503" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pbio.1002503" data-track-item_id="10.1371/journal.pbio.1002503">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27404502" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4942072" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fecal%20transplants%3A%20what%20is%20being%20transferred%3F&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.1002503&amp;volume=14&amp;publication_year=2016&amp;author=Bojanova%2CDP&amp;author=Bordenstein%2CSR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR61">Fuentes, S. et al. Microbial shifts and signatures of long-term remission in ulcerative colitis after fecal microbiota transplantation. <i>ISME J.</i> <b>11</b>, 1877–1889 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fismej.2017.44" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ismej.2017.44" data-track-item_id="10.1038/ismej.2017.44">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28398347" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5520032" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Microbial%20shifts%20and%20signatures%20of%20long-term%20remission%20in%20ulcerative%20colitis%20after%20fecal%20microbiota%20transplantation&amp;journal=ISME%20J.&amp;doi=10.1038%2Fismej.2017.44&amp;volume=11&amp;pages=1877-1889&amp;publication_year=2017&amp;author=Fuentes%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR62">Coelho, L. P. et al. NG-meta-profiler: fast processing of metagenomes using NGLess, a domain-specific language. <i>Microbiome</i> <b>7</b>, 84 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s40168-019-0684-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s40168-019-0684-8" data-track-item_id="10.1186/s40168-019-0684-8">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31159881" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6547473" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=NG-meta-profiler%3A%20fast%20processing%20of%20metagenomes%20using%20NGLess%2C%20a%20domain-specific%20language&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-019-0684-8&amp;volume=7&amp;publication_year=2019&amp;author=Coelho%2CLP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR63">Milanese, A. et al. Microbial abundance, activity and population genomic profiling with mOTUs2. <i>Nat. Commun.</i> <b>10</b>, 1014 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-019-08844-4" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-019-08844-4" data-track-item_id="10.1038/s41467-019-08844-4">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30833550" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6399450" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Microbial%20abundance%2C%20activity%20and%20population%20genomic%20profiling%20with%20mOTUs2&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-019-08844-4&amp;volume=10&amp;publication_year=2019&amp;author=Milanese%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR64">Coelho, L. P. et al. Towards the biogeography of prokaryotic genes. <i>Nature</i> <b>601</b>, 252–256 (2022).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-021-04233-4" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-021-04233-4" data-track-item_id="10.1038/s41586-021-04233-4">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXislGntbrE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34912116" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20the%20biogeography%20of%20prokaryotic%20genes&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-04233-4&amp;volume=601&amp;pages=252-256&amp;publication_year=2022&amp;author=Coelho%2CLP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR65">Huerta-Cepas, J. et al. EggNOG 5.0: a hierarchical, functionally and phylogenetically annotated orthology resource based on 5090 organisms and 2502 viruses. <i>Nucleic Acids Res.</i> <b>47</b>, D309–D314 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgky1085" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gky1085" data-track-item_id="10.1093/nar/gky1085">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Gqtr%2FL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30418610" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=EggNOG%205.0%3A%20a%20hierarchical%2C%20functionally%20and%20phylogenetically%20annotated%20orthology%20resource%20based%20on%205090%20organisms%20and%202502%20viruses&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky1085&amp;volume=47&amp;pages=D309-D314&amp;publication_year=2019&amp;author=Huerta-Cepas%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR66">Vieira-Silva, S. et al. Species–function relationships shape ecological properties of the human gut microbiome. <i>Nat. Microbiol.</i> <b>1</b>, 16088 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnmicrobiol.2016.88" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nmicrobiol.2016.88" data-track-item_id="10.1038/nmicrobiol.2016.88">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXkvF2ju74%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27573110" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Species%E2%80%93function%20relationships%20shape%20ecological%20properties%20of%20the%20human%20gut%20microbiome&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fnmicrobiol.2016.88&amp;volume=1&amp;publication_year=2016&amp;author=Vieira-Silva%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR67">Darzi, Y., Falony, G., Vieira-Silva, S. &amp; Raes, J. Towards biome-specific analysis of meta-omics data. <i>ISME J.</i> <b>10</b>, 1025–1028 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fismej.2015.188" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ismej.2015.188" data-track-item_id="10.1038/ismej.2015.188">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xmsl2qu7w%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26623543" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Towards%20biome-specific%20analysis%20of%20meta-omics%20data&amp;journal=ISME%20J.&amp;doi=10.1038%2Fismej.2015.188&amp;volume=10&amp;pages=1025-1028&amp;publication_year=2016&amp;author=Darzi%2CY&amp;author=Falony%2CG&amp;author=Vieira-Silva%2CS&amp;author=Raes%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR68">Nurk, S., Meleshko, D., Korobeynikov, A. &amp; Pevzner, P. A. MetaSPAdes: a new versatile metagenomic assembler. <i>Genome Res.</i> <b>27</b>, 824–834 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1101%2Fgr.213959.116" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1101/gr.213959.116" data-track-item_id="10.1101/gr.213959.116">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtFyjsrrJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28298430" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5411777" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MetaSPAdes%3A%20a%20new%20versatile%20metagenomic%20assembler&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.213959.116&amp;volume=27&amp;pages=824-834&amp;publication_year=2017&amp;author=Nurk%2CS&amp;author=Meleshko%2CD&amp;author=Korobeynikov%2CA&amp;author=Pevzner%2CPA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR69">Li, H. Aligning sequence reads, clone sequences and assembly contigs with BWA-MEM. Preprint at <a href="https://doi.org/10.48550/arXiv.1303.3997" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.48550/arXiv.1303.3997">https://doi.org/10.48550/arXiv.1303.3997</a> (2013).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR70">Kang, D. D. et al. MetaBAT 2: an adaptive binning algorithm for robust and efficient genome reconstruction from metagenome assemblies. <i>PeerJ.</i> <b>7</b>, e7359 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.7717%2Fpeerj.7359" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.7717/peerj.7359" data-track-item_id="10.7717/peerj.7359">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31388474" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6662567" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MetaBAT%202%3A%20an%20adaptive%20binning%20algorithm%20for%20robust%20and%20efficient%20genome%20reconstruction%20from%20metagenome%20assemblies&amp;journal=PeerJ.&amp;doi=10.7717%2Fpeerj.7359&amp;volume=7&amp;publication_year=2019&amp;author=Kang%2CDD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR71">Li, D., Liu, C.-M., Luo, R., Sadakane, K. &amp; Lam, T.-W. MEGAHIT: an ultra-fast single-node solution for large and complex metagenomics assembly via succinct de Bruijn graph. <i>Bioinformatics</i> <b>31</b>, 1674–1676 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv033" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btv033" data-track-item_id="10.1093/bioinformatics/btv033">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtFyltL3N" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25609793" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MEGAHIT%3A%20an%20ultra-fast%20single-node%20solution%20for%20large%20and%20complex%20metagenomics%20assembly%20via%20succinct%20de%20Bruijn%20graph&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtv033&amp;volume=31&amp;pages=1674-1676&amp;publication_year=2015&amp;author=Li%2CD&amp;author=Liu%2CC-M&amp;author=Luo%2CR&amp;author=Sadakane%2CK&amp;author=Lam%2CT-W" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR72">Alneberg, J. et al. Binning metagenomic contigs by coverage and composition. <i>Nat. Methods</i> <b>11</b>, 1144–1146 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnmeth.3103" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nmeth.3103" data-track-item_id="10.1038/nmeth.3103">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsFOksrbF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25218180" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Binning%20metagenomic%20contigs%20by%20coverage%20and%20composition&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.3103&amp;volume=11&amp;pages=1144-1146&amp;publication_year=2014&amp;author=Alneberg%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR73">Sieber, C. M. K. et al. Recovery of genomes from metagenomes via a dereplication, aggregation and scoring strategy. <i>Nat. Microbiol.</i> <b>3</b>, 836–843 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41564-018-0171-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41564-018-0171-1" data-track-item_id="10.1038/s41564-018-0171-1">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtVelsbvM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29807988" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6786971" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Recovery%20of%20genomes%20from%20metagenomes%20via%20a%20dereplication%2C%20aggregation%20and%20scoring%20strategy&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-018-0171-1&amp;volume=3&amp;pages=836-843&amp;publication_year=2018&amp;author=Sieber%2CCMK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR74">Uritskiy, G. V., DiRuggiero, J. &amp; Taylor, J. MetaWRAP—a flexible pipeline for genome-resolved metagenomic data analysis. <i>Microbiome</i> <b>6</b>, 158 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s40168-018-0541-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s40168-018-0541-1" data-track-item_id="10.1186/s40168-018-0541-1">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30219103" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6138922" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MetaWRAP%E2%80%94a%20flexible%20pipeline%20for%20genome-resolved%20metagenomic%20data%20analysis&amp;journal=Microbiome&amp;doi=10.1186%2Fs40168-018-0541-1&amp;volume=6&amp;publication_year=2018&amp;author=Uritskiy%2CGV&amp;author=DiRuggiero%2CJ&amp;author=Taylor%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR75">Mende, D. R. et al. ProGenomes: a resource for consistent functional and taxonomic annotations of prokaryotic genomes. <i>Nucleic Acids Res.</i> <b>45</b>, D529–D534 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgkw989" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gkw989" data-track-item_id="10.1093/nar/gkw989">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslWht7o%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28053165" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ProGenomes%3A%20a%20resource%20for%20consistent%20functional%20and%20taxonomic%20annotations%20of%20prokaryotic%20genomes&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkw989&amp;volume=45&amp;pages=D529-D534&amp;publication_year=2017&amp;author=Mende%2CDR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR76">Mende, D. R. et al. ProGenomes2: an improved database for accurate and consistent habitat, taxonomic and functional annotations of prokaryotic genomes. <i>Nucleic Acids Res.</i> <b>48</b>, D621–D625 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Glt7vK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31647096" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ProGenomes2%3A%20an%20improved%20database%20for%20accurate%20and%20consistent%20habitat%2C%20taxonomic%20and%20functional%20annotations%20of%20prokaryotic%20genomes&amp;journal=Nucleic%20Acids%20Res.&amp;volume=48&amp;pages=D621-D625&amp;publication_year=2020&amp;author=Mende%2CDR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR77">Parks, D. H., Imelfort, M., Skennerton, C. T., Hugenholtz, P. &amp; Tyson, G. W. CheckM: assessing the quality of microbial genomes recovered from isolates, single cells, and metagenomes. <i>Genome Res.</i> <b>25</b>, 1043–1055 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1101%2Fgr.186072.114" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1101/gr.186072.114" data-track-item_id="10.1101/gr.186072.114">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXht1SltbfE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25977477" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4484387" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CheckM%3A%20assessing%20the%20quality%20of%20microbial%20genomes%20recovered%20from%20isolates%2C%20single%20cells%2C%20and%20metagenomes&amp;journal=Genome%20Res.&amp;doi=10.1101%2Fgr.186072.114&amp;volume=25&amp;pages=1043-1055&amp;publication_year=2015&amp;author=Parks%2CDH&amp;author=Imelfort%2CM&amp;author=Skennerton%2CCT&amp;author=Hugenholtz%2CP&amp;author=Tyson%2CGW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR78">Orakov, A. et al. GUNC: detection of chimerism and contamination in prokaryotic genomes. <i>Genome Biol.</i> <b>22</b>, 178 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-021-02393-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-021-02393-0" data-track-item_id="10.1186/s13059-021-02393-0">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvFKmtLrN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34120611" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8201837" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=GUNC%3A%20detection%20of%20chimerism%20and%20contamination%20in%20prokaryotic%20genomes&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-021-02393-0&amp;volume=22&amp;publication_year=2021&amp;author=Orakov%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR79">Chaumeil, P.-A., Mussig, A. J., Hugenholtz, P. &amp; Parks, D. H. GTDB-Tk: a toolkit to classify genomes with the Genome Taxonomy Database. <i>Bioinformatics</i> <b>36</b>, 1925–1927 (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR80">Hyatt, D. et al. Prodigal: prokaryotic gene recognition and translation initiation site identification. <i>BMC Bioinformatics</i> <b>11</b>, 119 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/1471-2105-11-119" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/1471-2105-11-119" data-track-item_id="10.1186/1471-2105-11-119">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20211023" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2848648" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Prodigal%3A%20prokaryotic%20gene%20recognition%20and%20translation%20initiation%20site%20identification&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2F1471-2105-11-119&amp;volume=11&amp;publication_year=2010&amp;author=Hyatt%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR81">Seemann, T. Prokka: rapid prokaryotic genome annotation. <i>Bioinformatics</i> <b>30</b>, 2068–2069 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu153" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btu153" data-track-item_id="10.1093/bioinformatics/btu153">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhtFCrtLjI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24642063" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Prokka%3A%20rapid%20prokaryotic%20genome%20annotation&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu153&amp;volume=30&amp;pages=2068-2069&amp;publication_year=2014&amp;author=Seemann%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR82">Huerta-Cepas, J. et al. Fast genome-wide functional annotation through orthology assignment by eggNOG-Mapper. <i>Mol. Biol. Evol.</i> <b>34</b>, 2115–2122 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fmolbev%2Fmsx148" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/molbev/msx148" data-track-item_id="10.1093/molbev/msx148">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhvFaru7vF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28460117" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5850834" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20genome-wide%20functional%20annotation%20through%20orthology%20assignment%20by%20eggNOG-Mapper&amp;journal=Mol.%20Biol.%20Evol.&amp;doi=10.1093%2Fmolbev%2Fmsx148&amp;volume=34&amp;pages=2115-2122&amp;publication_year=2017&amp;author=Huerta-Cepas%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR83">Alcock, B. P. et al. CARD 2020: antibiotic resistome surveillance with the comprehensive antibiotic resistance database. <i>Nucleic Acids Res.</i> <b>48</b>, D517–D525 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslWltrnK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31665441" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CARD%202020%3A%20antibiotic%20resistome%20surveillance%20with%20the%20comprehensive%20antibiotic%20resistance%20database&amp;journal=Nucleic%20Acids%20Res.&amp;volume=48&amp;pages=D517-D525&amp;publication_year=2020&amp;author=Alcock%2CBP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR84">Gibson, M. K., Forsberg, K. J. &amp; Dantas, G. Improved annotation of antibiotic resistance determinants reveals microbial resistomes cluster by ecology. <i>ISME J.</i> <b>9</b>, 207–216 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fismej.2014.106" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ismej.2014.106" data-track-item_id="10.1038/ismej.2014.106">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXitFKlsLzN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25003965" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20annotation%20of%20antibiotic%20resistance%20determinants%20reveals%20microbial%20resistomes%20cluster%20by%20ecology&amp;journal=ISME%20J.&amp;doi=10.1038%2Fismej.2014.106&amp;volume=9&amp;pages=207-216&amp;publication_year=2015&amp;author=Gibson%2CMK&amp;author=Forsberg%2CKJ&amp;author=Dantas%2CG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR85">Mende, D. R., Sunagawa, S., Zeller, G. &amp; Bork, P. Accurate and universal delineation of prokaryotic species. <i>Nat. Publ. Group</i> <b>10</b>, 881–884 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXhtFOht7vJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Accurate%20and%20universal%20delineation%20of%20prokaryotic%20species&amp;journal=Nat.%20Publ.%20Group&amp;volume=10&amp;pages=881-884&amp;publication_year=2013&amp;author=Mende%2CDR&amp;author=Sunagawa%2CS&amp;author=Zeller%2CG&amp;author=Bork%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR86">Schmidt, T. S. B. et al. Drivers and determinants of strain dynamics following fecal microbiota transplantation. <i>Zenodo</i> <a href="https://doi.org/10.5281/ZENODO.5534163" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.5281/ZENODO.5534163">https://doi.org/10.5281/ZENODO.5534163</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR87">Rodrigues, J. F. M., Schmidt, S. B. T., Tackmann, J. &amp; von Mering, C. MAPseq: highly efficient k-mer search with confidence estimates, for rRNA sequence analysis. <i>Bioinformatics</i> <b>33</b>, 3808–3810 (2017).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR88">Olm, M. R., Brown, C. T., Brooks, B. &amp; Banfield, J. F. DRep: a tool for fast and accurate genomic comparisons that enables improved genome recovery from metagenomes through de-replication. <i>ISME J.</i> <b>11</b>, 2864–2868 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fismej.2017.126" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ismej.2017.126" data-track-item_id="10.1038/ismej.2017.126">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhvVeiur7K" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28742071" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5702732" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=DRep%3A%20a%20tool%20for%20fast%20and%20accurate%20genomic%20comparisons%20that%20enables%20improved%20genome%20recovery%20from%20metagenomes%20through%20de-replication&amp;journal=ISME%20J.&amp;doi=10.1038%2Fismej.2017.126&amp;volume=11&amp;pages=2864-2868&amp;publication_year=2017&amp;author=Olm%2CMR&amp;author=Brown%2CCT&amp;author=Brooks%2CB&amp;author=Banfield%2CJF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR89">Ondov, B. D. et al. Mash: fast genome and metagenome distance estimation using MinHash. <i>Genome Biol.</i> <b>17</b>, 132 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-016-0997-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-016-0997-x" data-track-item_id="10.1186/s13059-016-0997-x">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27323842" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4915045" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Mash%3A%20fast%20genome%20and%20metagenome%20distance%20estimation%20using%20MinHash&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-016-0997-x&amp;volume=17&amp;publication_year=2016&amp;author=Ondov%2CBD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR90">Jain, C., Rodriguez-R, L. M., Phillippy, A. M., Konstantinidis, K. T. &amp; Aluru, S. High throughput ANI analysis of 90 K prokaryotic genomes reveals clear species boundaries. <i>Nat. Commun.</i> <b>9</b>, 5114 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-018-07641-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-018-07641-9" data-track-item_id="10.1038/s41467-018-07641-9">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30504855" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6269478" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=High%20throughput%20ANI%20analysis%20of%2090%E2%80%89K%20prokaryotic%20genomes%20reveals%20clear%20species%20boundaries&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-018-07641-9&amp;volume=9&amp;publication_year=2018&amp;author=Jain%2CC&amp;author=Rodriguez-R%2CLM&amp;author=Phillippy%2CAM&amp;author=Konstantinidis%2CKT&amp;author=Aluru%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR91">Page, A. J. et al. Roary: rapid large-scale prokaryote pan genome analysis. <i>Bioinformatics</i> <b>31</b>, 3691–3693 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtv421" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btv421" data-track-item_id="10.1093/bioinformatics/btv421">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhtlamsrfO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26198102" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4817141" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Roary%3A%20rapid%20large-scale%20prokaryote%20pan%20genome%20analysis&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtv421&amp;volume=31&amp;pages=3691-3693&amp;publication_year=2015&amp;author=Page%2CAJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR92">Edgar, R. C. MUSCLE: a multiple sequence alignment method with reduced time and space complexity. <i>BMC Bioinformatics</i> <b>5</b>, 113 (2004).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/1471-2105-5-113" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/1471-2105-5-113" data-track-item_id="10.1186/1471-2105-5-113">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=15318951" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC517706" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MUSCLE%3A%20a%20multiple%20sequence%20alignment%20method%20with%20reduced%20time%20and%20space%20complexity&amp;journal=BMC%20Bioinformatics&amp;doi=10.1186%2F1471-2105-5-113&amp;volume=5&amp;publication_year=2004&amp;author=Edgar%2CRC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR93">Price, M. N., Dehal, P. S. &amp; Arkin, A. P. FastTree 2–approximately maximum-likelihood trees for large alignments. <i>PLoS ONE</i> <b>5</b>, e9490 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pone.0009490" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pone.0009490" data-track-item_id="10.1371/journal.pone.0009490">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20224823" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2835736" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=FastTree%202%E2%80%93approximately%20maximum-likelihood%20trees%20for%20large%20alignments&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0009490&amp;volume=5&amp;publication_year=2010&amp;author=Price%2CMN&amp;author=Dehal%2CPS&amp;author=Arkin%2CAP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR94">Li, H. et al. The Sequence Alignment/Map format and SAMtools. <i>Bioinformatics</i> <b>25</b>, 2078–2079 (2009).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp352" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btp352" data-track-item_id="10.1093/bioinformatics/btp352">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19505943" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2723002" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp352&amp;volume=25&amp;pages=2078-2079&amp;publication_year=2009&amp;author=Li%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR95">Quinlan, A. R. &amp; Hall, I. M. BEDTools: a flexible suite of utilities for comparing genomic features. <i>Bioinformatics</i> <b>26</b>, 841–842 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtq033" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btq033" data-track-item_id="10.1093/bioinformatics/btq033">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXivFGkurc%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20110278" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2832824" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtq033&amp;volume=26&amp;pages=841-842&amp;publication_year=2010&amp;author=Quinlan%2CAR&amp;author=Hall%2CIM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR96">Costea, P. I. et al. MetaSNV: a tool for metagenomic strain level analysis. <i>PLoS ONE</i> <b>12</b>, e0182392 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pone.0182392" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pone.0182392" data-track-item_id="10.1371/journal.pone.0182392">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28753663" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5533426" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MetaSNV%3A%20a%20tool%20for%20metagenomic%20strain%20level%20analysis&amp;journal=PLoS%20ONE&amp;doi=10.1371%2Fjournal.pone.0182392&amp;volume=12&amp;publication_year=2017&amp;author=Costea%2CPI" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR97">Schmidt, T. S. B. et al. Extensive transmission of microbes along the gastrointestinal tract. <i>eLife</i> <b>8</b>, e42693 (2019).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR98">Friedman, J., Hastie, T. &amp; Tibshirani, R. Regularization paths for generalized linear models via coordinate descent. <i>J. Stat. Softw.</i> <b>33</b>, 1–22 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.18637%2Fjss.v033.i01" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.18637/jss.v033.i01" data-track-item_id="10.18637/jss.v033.i01">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20808728" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2929880" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Regularization%20paths%20for%20generalized%20linear%20models%20via%20coordinate%20descent&amp;journal=J.%20Stat.%20Softw.&amp;doi=10.18637%2Fjss.v033.i01&amp;volume=33&amp;pages=1-22&amp;publication_year=2010&amp;author=Friedman%2CJ&amp;author=Hastie%2CT&amp;author=Tibshirani%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR99">Schmidt, T. S. B. et al. Analysis data, “Drivers and Determinants of Strain Dynamics Following Fecal Microbiota Transplantation”. <i>Zenodo</i> <a href="https://doi.org/10.5281/ZENODO.5534163" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.5281/ZENODO.5534163">https://doi.org/10.5281/ZENODO.5534163</a> (2021).</p></li></ol></div></div>
        </div>
    </div>
</aside>
</div>


    
        
    
    
        
            
        

        
            
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>




<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Medicine
                    </span>
                    (<i itemprop="alternateName">Nat Med</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-170X</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1078-8956</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/" data-track="click" data-track-action="protocols.io" data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/authors/editorial_policies" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nature-research/open-access" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/language-editing/" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/" data-track="click" data-track-action="research solutions" data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library" data-track="click" data-track-action="Recommend to library" data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica" data-track="click" data-track-action="nature africa" data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly" data-track="click" data-track-action="nature Italy" data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies" data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice" data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/accessibility-statement" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20">
    </a>
    <p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"></path></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"></path><clipPath id="b"><use xlink:href="#a" style="overflow:visible"></use></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"></path><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"></path><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"></path><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"></path></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"></path></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif " data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing Microbiology" src="/static/images/logos/nature-briefing-microbiology-logo-3fa570e5fb.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing: Microbiology</em> newsletter — what matters in microbiology research, free to your inbox weekly.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/microbiology" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MicrobiologyBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="MicrobiologyBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required="" data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:microbiology" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required="">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="/briefing/microbiology/?brieferEntryPoint=MicrobiologyBriefingBanner">Sign up for Nature Briefing: Microbiology
            </a>
        </div>

    </div>

</div>

    




<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41591-022-01913-0&amp;format=js&amp;last_modified=2022-09-15" async=""></script>

<lol-yzhjmkfiaqmm></lol-yzhjmkfiaqmm><div data-cc-ghost="" style="height: 352px;"></div><script type="text/javascript" id="fetch-contextual-ads-campaign-data" charset="">(function(){var g=function(a){a=new CustomEvent("campaignDataLoaded",{detail:a});document.dispatchEvent(a)},b=google_tag_manager["rm"]["50443292"](18);if(b){var k=1500;b=google_tag_manager["rm"]["50443292"](19);b="link"===b?google_tag_manager["rm"]["50443292"](20):google_tag_manager["rm"]["50443292"](21);var m=function(a,c){var h=!1,l=setTimeout(function(){h=!0;c(null)},k),d=new XMLHttpRequest;d.onload=function(){var e=null,f=[];if(200===d.status){try{e=JSON.parse(d.responseText)}catch(p){e={}}f=e.campaigns||[]}h||(clearTimeout(l),c(f.length?f.join(","):null))};d.open("GET",
"/platform/contextual?doi\x3d"+a);d.send()},n=function(a,c){a&&-1===a.indexOf("not set")?m(a,c):setTimeout(c,1)};n(b,function(a){window.campaignsForContextualAds=a;g(a);window.dataLayer.push({content:{article:{campaignID:a}}})})}else window.campaignsForContextualAds="",g(null)})();</script>
<script type="text/javascript" id="gtm-setup-accessdetailsloaded-handler" charset="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail||{};var b={event:"update-access-details"};a.institutional_business_partner_ids&&a.resolved_by||console.log("BPID data could not be retrieved from /exposed-details");a.logged_in_to_sn_profile||console.log("logged_in_to_sn_profile could not be retrieved from /exposed-details");var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?
a.resolved_by.join(";"):"",e=a.personal_business_partner_id_found,f=a.logged_in_to_sn_profile,g=a.snid;b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null,personal:e||null,logged_in_to_sn_profile:f,snid:g||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy={};window.dataLayer.push(b);window.idpUserDataLoaded=!0},!1);window.dataLayer.push({event:"accessdetailsloaded-handler-added"});
window.accessDetailsLoadedHandlerAdded=!0;</script><script type="text/javascript" id="gtm-mediatrust" charset="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="crossmark-script" text="" charset="" type="text/javascript" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="polyfill-matches" charset="">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="fetch-idp-user-data" charset="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},site=google_tag_manager["rm"]["50443292"](59),idpUrl;
switch(site){case "nature":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?"https://staging-idp.nature.com/exposed-details":"https://idp.nature.com/exposed-details";break;case "link":idpUrl=-1<window.location.hostname.indexOf("link-qa")?"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details";break;case "springer":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?
"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details"}if(void 0!==idpUrl){var transport=new XMLHttpRequest;transport.open("GET",idpUrl,!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script src="https://verify.nature.com/verify/nature.min.js"></script><script type="text/javascript" id="create-ad-slots" charset="">function createAndLoadAds(){function h(b){"nature"===a&&google_tag_manager["rm"]["50443292"](87)(b);"bmc"!==a&&"springeropen"!==a||google_tag_manager["rm"]["50443292"](99)(b);("link"===a&&"oscar"===google_tag_manager["rm"]["50443292"](100)||"springer"===a||"link"===a&&!0===google_tag_manager["rm"]["50443292"](101))&&google_tag_manager["rm"]["50443292"](126)(b);"link"===a&&"bunsen"===google_tag_manager["rm"]["50443292"](127)&&google_tag_manager["rm"]["50443292"](152)(b);"link"===a&&"Core"===google_tag_manager["rm"]["50443292"](153)&&google_tag_manager["rm"]["50443292"](165)(b)}window.googletag=window.googletag||{cmd:[]};var a=google_tag_manager["rm"]["50443292"](166),e=google_tag_manager["rm"]["50443292"](168);
if(-1===window.location.search.indexOf("hide_ads\x3dtrue"))if(window.adSlots&&0!==Object.keys(window.adSlots).length)console.log("Ads previously loaded. Will not update ad slots until next page load.");else{window.adSlots||(window.adSlots={});window.getAd=function(b,f){for(var c in window.adSlots)if(-1<c.indexOf(b)){if("object"===f)return adSlots[c];if("slot"===f)return adSlots[c].slot}};googletag.cmd.push(function(){googletag.pubads().setPrivacySettings({limitedAds:e});console.log("limitedAds is ",
e);googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["rm"]["50443292"](170));googletag.pubads().enableSingleRequest();googletag.pubads().disableInitialLoad()});for(var g=document.querySelectorAll("[data-gpt]"),d=0;g[d];++d)h(g[d]);googletag.cmd.push(function(){googletag.enableServices()})}}createAndLoadAds();</script>
</body></html>