<html lang="en" class="js"><head><link rel="preload" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" as="font" type="font/woff2" crossorigin=""><script async="" src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script type="text/javascript" async="" src="https://pagead2.googlesyndication.com/tag/js/gpt.js"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-MRVXSHQ"></script><script type="text/javascript" src="https://www.nature.com/Mb59802-brf95Y5Q2DmIa81JpZqmMcsXej8NwxLNU_UC0JQX7ghDgGyYewNj1UAlY7L19ZWR4Zgo5SosePPk4g=="></script><script src="https://cmp.nature.com/production_live/en/consent-bundle-8-85.js" onload="initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')"></script>
    <title>Development of a clinical polygenic risk score assay and reporting workflow | Nature Medicine</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nm.rss">


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin="">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480">

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"disease-prevention;population-genetics;risk-factors","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Medicine","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41591-022-01767-6"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Limin Hao","Peter Kraft","Gabriel F. Berriz","Elizabeth D. Hynes","Christopher Koch","Prathik Korategere V Kumar","Shruti S. Parpattedar","Marcie Steeves","Wanfeng Yu","Ashley A. Antwi","Charles A. Brunette","Morgan Danowski","Manish K. Gala","Robert C. Green","Natalie E. Jones","Anna C. F. Lewis","Steven A. Lubitz","Pradeep Natarajan","Jason L. Vassy","Matthew S. Lebo"],"publishedAt":1650240000,"publishedAtString":"2022-04-18","title":"Development of a clinical polygenic risk score assay and reporting workflow","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Disease prevention,Population genetics,Risk factors"},"journal":{"pcode":"nm","title":"nature medicine","volume":"28","issue":"5","id":41591,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"BR","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{font-weight:700;-webkit-font-smoothing:antialiased;font-size:1.5rem;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.app-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{margin:4px 4px 0;fill:#888;height:10px;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-c9121fecab.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png">
<link rel="icon" type="image/png" sizes="48x48" href="/static/images/favicons/nature/favicon-48x48-b52890008c.png">
<link rel="icon" type="image/png" sizes="32x32" href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="/static/images/favicons/nature/favicon-16x16-951651ab72.png">
<link rel="manifest" href="/static/manifest.json" crossorigin="use-credentials">
<link rel="mask-icon" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="/static/images/favicons/nature/favicon.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-e9d2061c95.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-1cbf3a8b56.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script><script id="js-position1" data-test="global-article-js" defer="" src="/static/js/global-article-es6-bundle-c8a573ca90.js"></script><script id="js-position2" data-test="shared-js" defer="" src="/static/js/shared-es6-bundle-e9d2061c95.js"></script><script id="js-position3" data-test="header-150-js" defer="" src="/static/js/header-150-es6-bundle-5bb959eaa1.js"></script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Development of a clinical polygenic risk score assay and reporting workflow","description":"Implementation of polygenic risk scores (PRS) may improve disease prevention and management but poses several challenges: the construction of clinically valid assays, interpretation for individual patients, and the development of clinical workflows and resources to support their use in patient care. For the ongoing Veterans Affairs Genomic Medicine at Veterans Affairs (GenoVA) Study we developed a clinical genotype array-based assay for six published PRS. We used data from 36,423 Mass General Brigham Biobank participants and adjustment for population structure to replicate known PRS–disease associations and published PRS thresholds for a disease odds ratio (OR) of 2 (ranging from 1.75 (95% CI: 1.57–1.95) for type 2 diabetes to 2.38 (95% CI: 2.07–2.73) for breast cancer). After confirming the high performance and robustness of the pipeline for use as a clinical assay for individual patients, we analyzed the first 227 prospective samples from the GenoVA Study and found that the frequency of PRS corresponding to published OR > 2 ranged from 13/227 (5.7%) for colorectal cancer to 23/150 (15.3%) for prostate cancer. In addition to the PRS laboratory report, we developed physician- and patient-oriented informational materials to support decision-making about PRS results. Our work illustrates the generalizable development of a clinical PRS assay for multiple conditions and the technical, reporting and clinical workflow challenges for implementing PRS information in the clinic. The first report from the prospective GenoVA Study provides preliminary insights into the development of a polygenic risk score assay in a clinical setting and discusses the challenges of generating, interpreting and reporting results.","datePublished":"2022-04-18T00:00:00Z","dateModified":"2022-04-18T00:00:00Z","pageStart":"1006","pageEnd":"1013","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41591-022-01767-6","keywords":["Disease prevention","Population genetics","Risk factors","Biomedicine","general","Cancer Research","Metabolic Diseases","Infectious Diseases","Molecular Medicine","Neurosciences"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig4_HTML.png"],"isPartOf":{"name":"Nature Medicine","issn":["1546-170X","1078-8956"],"volumeNumber":"28","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Limin Hao","affiliation":[{"name":"Mass General Brigham Personalized Medicine","address":{"name":"Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Peter Kraft","url":"http://orcid.org/0000-0002-4472-8103","affiliation":[{"name":"Harvard T.H. Chan School of Public Health","address":{"name":"Department of Epidemiology, Harvard T.H. Chan School of Public Health, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Gabriel F. Berriz","affiliation":[{"name":"Mass General Brigham Personalized Medicine","address":{"name":"Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Elizabeth D. Hynes","url":"http://orcid.org/0000-0002-3817-2145","affiliation":[{"name":"Mass General Brigham Personalized Medicine","address":{"name":"Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Christopher Koch","affiliation":[{"name":"Mass General Brigham Personalized Medicine","address":{"name":"Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Prathik Korategere V Kumar","affiliation":[{"name":"Mass General Brigham Personalized Medicine","address":{"name":"Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Shruti S. Parpattedar","affiliation":[{"name":"Mass General Brigham Personalized Medicine","address":{"name":"Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marcie Steeves","affiliation":[{"name":"Mass General Brigham Personalized Medicine","address":{"name":"Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Medical Genetics, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Wanfeng Yu","affiliation":[{"name":"Mass General Brigham Personalized Medicine","address":{"name":"Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ashley A. Antwi","url":"http://orcid.org/0000-0002-2591-4410","affiliation":[{"name":"Veterans Affairs Boston Healthcare System","address":{"name":"Veterans Affairs Boston Healthcare System, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Charles A. Brunette","url":"http://orcid.org/0000-0003-1620-3526","affiliation":[{"name":"Veterans Affairs Boston Healthcare System","address":{"name":"Veterans Affairs Boston Healthcare System, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Morgan Danowski","affiliation":[{"name":"Veterans Affairs Boston Healthcare System","address":{"name":"Veterans Affairs Boston Healthcare System, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Manish K. Gala","affiliation":[{"name":"Massachusetts General Hospital","address":{"name":"Division of Gastroenterology, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School","address":{"name":"Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Robert C. Green","url":"http://orcid.org/0000-0001-8472-0424","affiliation":[{"name":"Harvard Medical School","address":{"name":"Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Broad Institute of Harvard and the Massachusetts Institute of Technology","address":{"name":"Broad Institute of Harvard and the Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Brigham and Women’s Hospital","address":{"name":"Department of Medicine, Brigham and Women’s Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Ariadne Labs","address":{"name":"Precision Population Health, Ariadne Labs, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Natalie E. Jones","affiliation":[{"name":"Veterans Affairs Boston Healthcare System","address":{"name":"Veterans Affairs Boston Healthcare System, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Broad Institute of Harvard and the Massachusetts Institute of Technology","address":{"name":"Broad Institute of Harvard and the Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Anna C. F. Lewis","url":"http://orcid.org/0000-0002-1274-386X","affiliation":[{"name":"Harvard University","address":{"name":"E J Safra Center for Ethics, Harvard University, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Steven A. Lubitz","url":"http://orcid.org/0000-0002-9599-4866","affiliation":[{"name":"Broad Institute of Harvard and the Massachusetts Institute of Technology","address":{"name":"Cardiovascular Disease Initiative, Broad Institute of Harvard and the Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Cardiovascular Research Center, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Demoulas Center for Cardiac Arrhythmias, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Pradeep Natarajan","affiliation":[{"name":"Harvard Medical School","address":{"name":"Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Broad Institute of Harvard and the Massachusetts Institute of Technology","address":{"name":"Cardiovascular Disease Initiative, Broad Institute of Harvard and the Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Cardiovascular Research Center, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jason L. Vassy","url":"http://orcid.org/0000-0001-6113-5564","affiliation":[{"name":"Veterans Affairs Boston Healthcare System","address":{"name":"Veterans Affairs Boston Healthcare System, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School","address":{"name":"Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Brigham and Women’s Hospital","address":{"name":"Department of Medicine, Brigham and Women’s Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Ariadne Labs","address":{"name":"Precision Population Health, Ariadne Labs, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"jvassy@bwh.harvard.edu","@type":"Person"},{"name":"Matthew S. Lebo","url":"http://orcid.org/0000-0002-9733-5207","affiliation":[{"name":"Mass General Brigham Personalized Medicine","address":{"name":"Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School","address":{"name":"Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Brigham and Women’s Hospital","address":{"name":"Department of Pathology, Brigham and Women’s Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41591-022-01767-6">
    
    
    <meta name="journal_id" content="41591">
    <meta name="dc.title" content="Development of a clinical polygenic risk score assay and reporting workflow">
    <meta name="dc.source" content="Nature Medicine 2022 28:5">
    <meta name="dc.format" content="text/html">
    <meta name="dc.publisher" content="Nature Publishing Group">
    <meta name="dc.date" content="2022-04-18">
    <meta name="dc.type" content="OriginalPaper">
    <meta name="dc.language" content="En">
    <meta name="dc.copyright" content="2022 This is a U.S. government work and not under copyright protection in the U.S.; foreign copyright protection may apply">
    <meta name="dc.rights" content="2022 This is a U.S. government work and not under copyright protection in the U.S.; foreign copyright protection may apply">
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="dc.description" content="Implementation of polygenic risk scores (PRS) may improve disease prevention and management but poses several challenges: the construction of clinically valid assays, interpretation for individual patients, and the development of clinical workflows and resources to support their use in patient care. For the ongoing Veterans Affairs Genomic Medicine at Veterans Affairs (GenoVA) Study we developed a clinical genotype array-based assay for six published PRS. We used data from 36,423 Mass General Brigham Biobank participants and adjustment for population structure to replicate known PRS–disease associations and published PRS thresholds for a disease odds ratio (OR) of 2 (ranging from 1.75 (95% CI: 1.57–1.95) for type 2 diabetes to 2.38 (95% CI: 2.07–2.73) for breast cancer). After confirming the high performance and robustness of the pipeline for use as a clinical assay for individual patients, we analyzed the first 227 prospective samples from the GenoVA Study and found that the frequency of PRS corresponding to published OR &amp;gt; 2 ranged from 13/227 (5.7%) for colorectal cancer to 23/150 (15.3%) for prostate cancer. In addition to the PRS laboratory report, we developed physician- and patient-oriented informational materials to support decision-making about PRS results. Our work illustrates the generalizable development of a clinical PRS assay for multiple conditions and the technical, reporting and clinical workflow challenges for implementing PRS information in the clinic. The first report from the prospective GenoVA Study provides preliminary insights into the development of a polygenic risk score assay in a clinical setting and discusses the challenges of generating, interpreting and reporting results.">
    <meta name="prism.issn" content="1546-170X">
    <meta name="prism.publicationName" content="Nature Medicine">
    <meta name="prism.publicationDate" content="2022-04-18">
    <meta name="prism.volume" content="28">
    <meta name="prism.number" content="5">
    <meta name="prism.section" content="OriginalPaper">
    <meta name="prism.startingPage" content="1006">
    <meta name="prism.endingPage" content="1013">
    <meta name="prism.copyright" content="2022 This is a U.S. government work and not under copyright protection in the U.S.; foreign copyright protection may apply">
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="prism.url" content="https://www.nature.com/articles/s41591-022-01767-6">
    <meta name="prism.doi" content="doi:10.1038/s41591-022-01767-6">
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41591-022-01767-6.pdf">
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41591-022-01767-6">
    <meta name="citation_journal_title" content="Nature Medicine">
    <meta name="citation_journal_abbrev" content="Nat Med">
    <meta name="citation_publisher" content="Nature Publishing Group">
    <meta name="citation_issn" content="1546-170X">
    <meta name="citation_title" content="Development of a clinical polygenic risk score assay and reporting workflow">
    <meta name="citation_volume" content="28">
    <meta name="citation_issue" content="5">
    <meta name="citation_publication_date" content="2022/05">
    <meta name="citation_online_date" content="2022/04/18">
    <meta name="citation_firstpage" content="1006">
    <meta name="citation_lastpage" content="1013">
    <meta name="citation_article_type" content="Article">
    <meta name="citation_fulltext_world_readable" content="">
    <meta name="citation_language" content="en">
    <meta name="dc.identifier" content="doi:10.1038/s41591-022-01767-6">
    <meta name="DOI" content="10.1038/s41591-022-01767-6">
    <meta name="size" content="242004">
    <meta name="citation_doi" content="10.1038/s41591-022-01767-6">
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41591-022-01767-6&amp;api_key=">
    <meta name="description" content="Implementation of polygenic risk scores (PRS) may improve disease prevention and management but poses several challenges: the construction of clinically valid assays, interpretation for individual patients, and the development of clinical workflows and resources to support their use in patient care. For the ongoing Veterans Affairs Genomic Medicine at Veterans Affairs (GenoVA) Study we developed a clinical genotype array-based assay for six published PRS. We used data from 36,423 Mass General Brigham Biobank participants and adjustment for population structure to replicate known PRS–disease associations and published PRS thresholds for a disease odds ratio (OR) of 2 (ranging from 1.75 (95% CI: 1.57–1.95) for type 2 diabetes to 2.38 (95% CI: 2.07–2.73) for breast cancer). After confirming the high performance and robustness of the pipeline for use as a clinical assay for individual patients, we analyzed the first 227 prospective samples from the GenoVA Study and found that the frequency of PRS corresponding to published OR &amp;gt; 2 ranged from 13/227 (5.7%) for colorectal cancer to 23/150 (15.3%) for prostate cancer. In addition to the PRS laboratory report, we developed physician- and patient-oriented informational materials to support decision-making about PRS results. Our work illustrates the generalizable development of a clinical PRS assay for multiple conditions and the technical, reporting and clinical workflow challenges for implementing PRS information in the clinic. The first report from the prospective GenoVA Study provides preliminary insights into the development of a polygenic risk score assay in a clinical setting and discusses the challenges of generating, interpreting and reporting results.">
    <meta name="dc.creator" content="Hao, Limin">
    <meta name="dc.creator" content="Kraft, Peter">
    <meta name="dc.creator" content="Berriz, Gabriel F.">
    <meta name="dc.creator" content="Hynes, Elizabeth D.">
    <meta name="dc.creator" content="Koch, Christopher">
    <meta name="dc.creator" content="Korategere V Kumar, Prathik">
    <meta name="dc.creator" content="Parpattedar, Shruti S.">
    <meta name="dc.creator" content="Steeves, Marcie">
    <meta name="dc.creator" content="Yu, Wanfeng">
    <meta name="dc.creator" content="Antwi, Ashley A.">
    <meta name="dc.creator" content="Brunette, Charles A.">
    <meta name="dc.creator" content="Danowski, Morgan">
    <meta name="dc.creator" content="Gala, Manish K.">
    <meta name="dc.creator" content="Green, Robert C.">
    <meta name="dc.creator" content="Jones, Natalie E.">
    <meta name="dc.creator" content="Lewis, Anna C. F.">
    <meta name="dc.creator" content="Lubitz, Steven A.">
    <meta name="dc.creator" content="Natarajan, Pradeep">
    <meta name="dc.creator" content="Vassy, Jason L.">
    <meta name="dc.creator" content="Lebo, Matthew S.">
    <meta name="dc.subject" content="Disease prevention">
    <meta name="dc.subject" content="Population genetics">
    <meta name="dc.subject" content="Risk factors">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Genomic medicine: progress, pitfalls, and promise; citation_author=J Shendure, GM Findlay, MW Snyder; citation_volume=177; citation_publication_date=2019; citation_pages=45-57; citation_doi=10.1016/j.cell.2019.02.003; citation_id=CR1">
    <meta name="citation_reference" content="GWAS Catalog (National Human Genome Research Institute); 
                  https://www.ebi.ac.uk/gwas/
                  
                ">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Genotype score in addition to common risk factors for prediction of type 2 diabetes; citation_author=JB Meigs; citation_volume=359; citation_publication_date=2008; citation_pages=2208-2219; citation_doi=10.1056/NEJMoa0804742; citation_id=CR3">
    <meta name="citation_reference" content="citation_journal_title=Lancet; citation_title=A multilocus genetic risk score for coronary heart disease: case-control and prospective cohort analyses; citation_author=S Ripatti; citation_volume=376; citation_publication_date=2010; citation_pages=1393-1400; citation_doi=10.1016/S0140-6736(10)61267-6; citation_id=CR4">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Cumulative association of five genetic variants with prostate cancer; citation_author=SL Zheng; citation_volume=358; citation_publication_date=2008; citation_pages=910-919; citation_doi=10.1056/NEJMoa075819; citation_id=CR5">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=An expanded view of complex traits: from polygenic to omnigenic; citation_author=EA Boyle, YI Li, JK Pritchard; citation_volume=169; citation_publication_date=2017; citation_pages=1177-1186; citation_doi=10.1016/j.cell.2017.05.038; citation_id=CR6">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations; citation_author=AV Khera; citation_volume=50; citation_publication_date=2018; citation_pages=1219-1224; citation_doi=10.1038/s41588-018-0183-z; citation_id=CR7">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.; citation_title=Modeling linkage disequilibrium increases accuracy of polygenic risk scores; citation_author=BJ Vilhjálmsson; citation_volume=97; citation_publication_date=2015; citation_pages=576-592; citation_doi=10.1016/j.ajhg.2015.09.001; citation_id=CR8">
    <meta name="citation_reference" content="citation_journal_title=J. Am. Coll. Cardiol.; citation_title=Genomic risk prediction of coronary artery disease in 480,000 adults: implications for primary prevention; citation_author=M Inouye; citation_volume=72; citation_publication_date=2018; citation_pages=1883-1893; citation_doi=10.1016/j.jacc.2018.07.079; citation_id=CR9">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Trans-ancestry genome-wide association meta-analysis of prostate cancer identifies new susceptibility loci and informs genetic risk prediction; citation_author=DV Conti; citation_volume=53; citation_publication_date=2021; citation_pages=65-75; citation_doi=10.1038/s41588-020-00748-0; citation_id=CR10">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Genome-wide association analysis of venous thromboembolism identifies new risk loci and genetic overlap with arterial vascular disease; citation_author=D Klarin; citation_volume=51; citation_publication_date=2019; citation_pages=1574-1579; citation_doi=10.1038/s41588-019-0519-3; citation_id=CR11">
    <meta name="citation_reference" content="citation_journal_title=JAMA; citation_title=Predictive accuracy of a polygenic risk score compared with a clinical risk score for incident coronary heart disease; citation_author=JD Mosley; citation_volume=323; citation_publication_date=2020; citation_pages=627-635; citation_doi=10.1001/jama.2019.21782; citation_id=CR12">
    <meta name="citation_reference" content="citation_journal_title=Diabetes; citation_title=Polygenic type 2 diabetes prediction at the limit of common variant detection; citation_author=JL Vassy; citation_volume=63; citation_publication_date=2014; citation_pages=2172-2182; citation_doi=10.2337/db13-1663; citation_id=CR13">
    <meta name="citation_reference" content="citation_journal_title=BMJ; citation_title=Polygenic hazard score to guide screening for aggressive prostate cancer: development and validation in large scale cohorts; citation_author=TM Seibert; citation_volume=360; citation_publication_date=2018; citation_pages=j5757; citation_doi=10.1136/bmj.j5757; citation_id=CR14">
    <meta name="citation_reference" content="National Human Genome Research Institute (NHGRI). Electronic Medical Records and Genomics (eMERGE) Network 
                  https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE
                  
                 (2020).">
    <meta name="citation_reference" content="Shieh, Y. et al. Breast cancer screening in the precision medicine era: risk-based screening in a population-based trial. J. Natl Cancer Inst. 109, 
                  https://doi.org/10.1093/jnci/djw290
                  
                 (2017).">
    <meta name="citation_reference" content="citation_journal_title=BMC Med. Genomics; citation_title=Design and user experience testing of a polygenic score report: a qualitative study of prospective users; citation_author=DG Brockman; citation_volume=14; citation_publication_date=2021; citation_doi=10.1186/s12920-021-01056-0; citation_id=CR17">
    <meta name="citation_reference" content="citation_journal_title=Nat. Protoc.; citation_title=Tutorial: a guide to performing polygenic risk score analyses; citation_author=SW Choi, TS-H Mak, PF O’Reilly; citation_volume=15; citation_publication_date=2020; citation_pages=2759-2772; citation_doi=10.1038/s41596-020-0353-1; citation_id=CR18">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Clinical use of current polygenic risk scores may exacerbate health disparities; citation_author=AR Martin; citation_volume=51; citation_publication_date=2019; citation_pages=584-591; citation_doi=10.1038/s41588-019-0379-x; citation_id=CR19">
    <meta name="citation_reference" content="citation_journal_title=Genome Med.; citation_title=Polygenic risk scores: from research tools to clinical instruments; citation_author=CM Lewis, E Vassos; citation_volume=12; citation_publication_date=2020; citation_doi=10.1186/s13073-020-00742-5; citation_id=CR20">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation; citation_author=SA Lambert; citation_volume=53; citation_publication_date=2021; citation_pages=420-425; citation_doi=10.1038/s41588-021-00783-5; citation_id=CR21">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.; citation_title=Cancer PRSweb: an online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks; citation_author=LG Fritsche; citation_volume=107; citation_publication_date=2020; citation_pages=815-836; citation_doi=10.1016/j.ajhg.2020.08.025; citation_id=CR22">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Discovery of common and rare genetic risk variants for colorectal cancer; citation_author=JR Huyghe; citation_volume=51; citation_publication_date=2019; citation_pages=76-87; citation_doi=10.1038/s41588-018-0286-6; citation_id=CR23">
    <meta name="citation_reference" content="citation_journal_title=Genet. Med.; citation_title=Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics; citation_author=SS Kalia; citation_volume=19; citation_publication_date=2017; citation_pages=249-255; citation_doi=10.1038/gim.2016.190; citation_id=CR24">
    <meta name="citation_reference" content="citation_journal_title=Sci. Data; citation_title=Extensive sequencing of seven human genomes to characterize benchmark reference materials; citation_author=JM Zook; citation_volume=3; citation_publication_date=2016; citation_doi=10.1038/sdata.2016.25; citation_id=CR25">
    <meta name="citation_reference" content="citation_journal_title=Genet. Med.; citation_title=Identifying rare, medically relevant variation via population-based genomic screening in Alabama: opportunities and pitfalls; citation_author=KM Bowling; citation_volume=23; citation_publication_date=2021; citation_pages=280-288; citation_doi=10.1038/s41436-020-00976-z; citation_id=CR26">
    <meta name="citation_reference" content="citation_journal_title=BMJ; citation_title=Use of SNP chips to detect rare pathogenic variants: retrospective, population based diagnostic evaluation; citation_author=MN Weedon, CF Wright; citation_volume=372; citation_publication_date=2021; citation_pages=n214; citation_id=CR27">
    <meta name="citation_reference" content="citation_journal_title=Genet. Med.; citation_title=Effective communication of molecular genetic test results to primary care providers; citation_author=MT Scheuner, MO Edelen, LH Hilborne, IM Lubin; citation_volume=15; citation_publication_date=2013; citation_pages=444-449; citation_doi=10.1038/gim.2012.151; citation_id=CR28">
    <meta name="citation_reference" content="citation_journal_title=BMC Med. Genet.; citation_title=A systematic approach to the reporting of medically relevant findings from whole genome sequencing; citation_author=HM McLaughlin; citation_volume=15; citation_publication_date=2014; citation_doi=10.1186/s12881-014-0134-1; citation_id=CR29">
    <meta name="citation_reference" content="citation_journal_title=Eur. J. Hum. Genet.; citation_title=Recommendations for designing genetic test reports to be understood by patients and non-specialists; citation_author=GD Farmer, H Gray, G Chandratillake, FL Raymond, ALJ Freeman; citation_volume=28; citation_publication_date=2020; citation_pages=885-895; citation_doi=10.1038/s41431-020-0579-y; citation_id=CR30">
    <meta name="citation_reference" content="citation_journal_title=Genet. Med.; citation_title=Considerations for clinical curation, classification, and reporting of low-penetrance and low effect size variants associated with disease risk; citation_author=O Senol-Cosar; citation_volume=21; citation_publication_date=2019; citation_pages=2765-2773; citation_doi=10.1038/s41436-019-0560-8; citation_id=CR31">
    <meta name="citation_reference" content="citation_journal_title=Circulation; citation_title=2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American College of Cardiology/American Heart Association Task Force on practice guidelines; citation_author=DC Goff; citation_volume=129; citation_issue=Suppl. 2; citation_publication_date=2014; citation_pages=S49-S73; citation_id=CR32">
    <meta name="citation_reference" content="citation_journal_title=Diabetes Care; citation_title=2. Classification and diagnosis of diabetes: standards of medical care in diabetes – 2021; citation_author=; citation_volume=44; citation_issue=Suppl. 1; citation_publication_date=2021; citation_pages=S15-S33; citation_doi=10.2337/dc21-S002; citation_id=CR33">
    <meta name="citation_reference" content="citation_journal_title=JAMA; citation_title=Screening for prostate cancer: US Preventive Services Task Force recommendation statement; citation_author=DC Grossman; citation_volume=319; citation_publication_date=2018; citation_pages=1901-1913; citation_doi=10.1001/jama.2018.0161; citation_id=CR34">
    <meta name="citation_reference" content="citation_journal_title=Ann. Intern. Med.; citation_title=Screening for breast cancer: US Preventive Services Task Force recommendation statement; citation_author=AL Siu; citation_volume=164; citation_publication_date=2016; citation_pages=279-296; citation_doi=10.7326/M15-2886; citation_id=CR35">
    <meta name="citation_reference" content="citation_journal_title=JAMA; citation_title=Screening for colorectal cancer: US Preventive Services Task Force recommendation statement; citation_author=KW Davidson; citation_volume=325; citation_publication_date=2021; citation_pages=1965-1977; citation_doi=10.1001/jama.2021.6238; citation_id=CR36">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Improving reporting standards for polygenic scores in risk prediction studies; citation_author=H Wand; citation_volume=591; citation_publication_date=2021; citation_pages=211-219; citation_doi=10.1038/s41586-021-03243-6; citation_id=CR37">
    <meta name="citation_reference" content="citation_journal_title=Genome Med.; citation_title=Low coverage whole genome sequencing enables accurate assessment of common variants and calculation of genome-wide polygenic scores; citation_author=JR Homburger; citation_volume=11; citation_publication_date=2019; citation_doi=10.1186/s13073-019-0682-2; citation_id=CR38">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=The ‘All of Us’ Research Program; citation_author=JC Denny; citation_volume=381; citation_publication_date=2019; citation_pages=668-676; citation_doi=10.1056/NEJMsr1809937; citation_id=CR39">
    <meta name="citation_reference" content="citation_journal_title=Circulation; citation_title=Whole-genome sequencing to characterize monogenic and polygenic contributions in patients hospitalized with early-onset myocardial infarction; citation_author=AV Khera; citation_volume=139; citation_publication_date=2019; citation_pages=1593-1602; citation_doi=10.1161/CIRCULATIONAHA.118.035658; citation_id=CR40">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.; citation_title=Predictive utility of polygenic risk scores for coronary heart disease in three major racial and ethnic groups; citation_author=O Dikilitas; citation_volume=106; citation_publication_date=2020; citation_pages=707-716; citation_doi=10.1016/j.ajhg.2020.04.002; citation_id=CR41">
    <meta name="citation_reference" content="citation_journal_title=PLoS Genet.; citation_title=Use of >100,000 NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium whole genome sequences improves imputation quality and detection of rare variant associations in admixed African and Hispanic/Latino populations; citation_author=MH Kowalski; citation_volume=15; citation_publication_date=2019; citation_pages=e1008500; citation_doi=10.1371/journal.pgen.1008500; citation_id=CR42">
    <meta name="citation_reference" content="citation_journal_title=HGG Adv.; citation_title=Polygenic risk scores in the clinic: translating risk into action; citation_author=ACF Lewis, RC Green, JL Vassy; citation_volume=2; citation_publication_date=2021; citation_pages=100047; citation_id=CR43">
    <meta name="citation_reference" content="citation_journal_title=Genet. Med.; citation_title=BOADICEA: a comprehensive breast cancer risk prediction model incorporating genetic and nongenetic risk factors; citation_author=A Lee; citation_volume=21; citation_publication_date=2019; citation_pages=1708-1718; citation_doi=10.1038/s41436-018-0406-9; citation_id=CR44">
    <meta name="citation_reference" content="citation_journal_title=Circulation; citation_title=2013 ACC/AHA guideline on the treatment of blood cholesterol to reduce atherosclerotic cardiovascular risk in adults: a report of the American College of Cardiology/American Heart Association Task Force on Practice Guidelines; citation_author=NJ Stone; citation_volume=129; citation_issue=Suppl. 2; citation_publication_date=2014; citation_pages=S1-S45; citation_id=CR45">
    <meta name="citation_reference" content="citation_journal_title=JCO Precis. Oncol.; citation_title=Development and validation of a clinical polygenic risk score to predict breast cancer risk; citation_author=E Hughes; citation_volume=4; citation_publication_date=2020; citation_pages=585-592; citation_doi=10.1200/PO.19.00360; citation_id=CR46">
    <meta name="citation_reference" content="Our Health + Ancestry DNA Service - 23andMe (23andMe); 
                  https://www.23andme.com/dna-health-ancestry/
                  
                ">
    <meta name="citation_reference" content="citation_journal_title=Genome Med.; citation_title=Genotype imputation and variability in polygenic risk score estimation; citation_author=S-F Chen; citation_volume=12; citation_publication_date=2020; citation_doi=10.1186/s13073-020-00801-x; citation_id=CR48">
    <meta name="citation_reference" content="National Human Genome Research Institute. Polygenic RIsk MEthods in Diverse populations (PRIMED) Consortium 
                  https://www.genome.gov/Funded-Programs-Projects/PRIMED-Consortium
                  
                ">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.; citation_title=Using the data we have: improving diversity in genomic research; citation_author=TA Manolio; citation_volume=105; citation_publication_date=2019; citation_pages=233-236; citation_doi=10.1016/j.ajhg.2019.07.008; citation_id=CR50">
    <meta name="citation_reference" content="citation_journal_title=Genome Med.; citation_title=Polygenic risk scores in the clinic: new perspectives needed on familiar ethical issues; citation_author=ACF Lewis, RC Green; citation_volume=13; citation_publication_date=2021; citation_doi=10.1186/s13073-021-00829-7; citation_id=CR51">
    <meta name="citation_reference" content="Ray, T. Myriad Genetics recalibrates breast cancer PRS for all ancestries in anticipation of broader launch. Genomeweb 
                  https://www.genomeweb.com/molecular-diagnostics/myriad-genetics-recalibrates-breast-cancer-prs-all-ancestries-anticipation
                  
                 (2021).">
    <meta name="citation_reference" content="Ambry Product Team. Important discontinuation notice: AmbryScore: polygenic risk scores (PRS) 
                  https://info.ambrygenetics.com/take-a-brief-survey-for-entry-into-amazon-gift-card-drawing
                  
                ">
    <meta name="citation_reference" content="Ge, T. et al. Validation of a trans-ancestry polygenic risk score for type 2 diabetes in diverse populations. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.09.11.21263413
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Ancestry deconvolution and partial polygenic score can improve susceptibility predictions in recently admixed individuals; citation_author=D Marnetto; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-15464-w; citation_id=CR55">
    <meta name="citation_reference" content="Ruan, Y. et al. Improving polygenic prediction in ancestrally diverse populations. Preprint at medRxiv 
                  https://doi.org/10.1101/2020.12.27.20248738
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Ann. Intern. Med.; citation_title=Annals clinical decision making: translating population evidence to individual patients; citation_author=KA Armstrong, JP Metlay; citation_volume=172; citation_publication_date=2020; citation_pages=610-616; citation_doi=10.7326/M19-3496; citation_id=CR57">
    <meta name="citation_reference" content="citation_journal_title=Mayo Clin. Proc.; citation_title=The necessity for clinical reasoning in the era of evidence-based medicine; citation_author=AD Sniderman, KJ LaChapelle, NA Rachon, CD Furberg; citation_volume=88; citation_publication_date=2013; citation_pages=1108-1114; citation_doi=10.1016/j.mayocp.2013.07.012; citation_id=CR58">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Association analysis identifies 65 new breast cancer risk loci; citation_author=K Michailidou; citation_volume=551; citation_publication_date=2017; citation_pages=92-94; citation_doi=10.1038/nature24284; citation_id=CR59">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet.; citation_title=Polygenic risk scores for prediction of breast cancer and breast cancer subtypes; citation_author=N Mavaddat; citation_volume=104; citation_publication_date=2019; citation_pages=21-34; citation_doi=10.1016/j.ajhg.2018.11.002; citation_id=CR60">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Association analyses of more than 140,000 men identify 63 new prostate cancer susceptibility loci; citation_author=FR Schumacher; citation_volume=50; citation_publication_date=2018; citation_pages=928-936; citation_doi=10.1038/s41588-018-0142-8; citation_id=CR61">
    <meta name="citation_reference" content="citation_journal_title=J. Pers. Med.; citation_title=Building the Partners Healthcare Biobank at Partners Personalized Medicine: informed consent, return of research results, recruitment lessons and operational considerations; citation_author=EW Karlson, NT Boutin, AG Hoffnagle, NL Allen; citation_volume=6; citation_publication_date=2016; citation_pages=2; citation_doi=10.3390/jpm6010002; citation_id=CR62">
    <meta name="citation_reference" content="citation_journal_title=J. Am. Med. Inform. Assoc.; citation_title=Toward high-throughput phenotyping: unbiased automated feature extraction and selection from knowledge sources; citation_author=S Yu; citation_volume=22; citation_publication_date=2015; citation_pages=993-1000; citation_doi=10.1093/jamia/ocv034; citation_id=CR63">
    <meta name="citation_reference" content="citation_journal_title=J. Am. Med. Inform. Assoc.; citation_title=Enabling phenotypic big data with PheNorm; citation_author=S Yu; citation_volume=25; citation_publication_date=2018; citation_pages=54-60; citation_doi=10.1093/jamia/ocx111; citation_id=CR64">
    <meta name="citation_reference" content="citation_journal_title=J. Pers. Med.; citation_title=The Biobank portal for Partners Personalized Medicine: a query tool for working with consented Biobank samples, genotypes, and phenotypes using i2b2; citation_author=VS Gainer; citation_volume=6; citation_publication_date=2016; citation_pages=11; citation_doi=10.3390/jpm6010011; citation_id=CR65">
    <meta name="citation_reference" content="citation_journal_title=J. Pers. Med.; citation_title=The translational genomics core at Partners Personalized Medicine: facilitating the transition of research towards personalized medicine; citation_author=A Blau, A Brown, L Mahanta, SS Amr; citation_volume=6; citation_publication_date=2016; citation_pages=10; citation_doi=10.3390/jpm6010010; citation_id=CR66">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Next-generation genotype imputation service and methods; citation_author=S Das; citation_volume=48; citation_publication_date=2016; citation_pages=1284-1287; citation_doi=10.1038/ng.3656; citation_id=CR67">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Improved whole-chromosome phasing for disease and population genetic studies; citation_author=O Delaneau, J-F Zagury, J Marchini; citation_volume=10; citation_publication_date=2013; citation_pages=5-6; citation_doi=10.1038/nmeth.2307; citation_id=CR68">
    <meta name="citation_reference" content="citation_journal_title=Front. Genet.; citation_title=A method for inferring an individual’s genetic ancestry and degree of admixture associated with six major continental populations; citation_author=O Libiger, NJ Schork; citation_volume=3; citation_publication_date=2013; citation_pages=322; citation_doi=10.3389/fgene.2012.00322; citation_id=CR69">
    <meta name="citation_reference" content="citation_journal_title=Genet. Med.; citation_title=ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG); citation_author=DT Miller; citation_volume=23; citation_publication_date=2021; citation_pages=1381-1390; citation_doi=10.1038/s41436-021-01172-3; citation_id=CR70">
    <meta name="citation_reference" content="citation_journal_title=Nat. Genet.; citation_title=Reference-based phasing using the Haplotype Reference Consortium panel; citation_author=P-R Loh; citation_volume=48; citation_publication_date=2016; citation_pages=1443-1448; citation_doi=10.1038/ng.3679; citation_id=CR71">
    <meta name="citation_reference" content="citation_journal_title=GigaScience; citation_title=Second-generation PLINK: rising to the challenge of larger and richer datasets; citation_author=CC Chang; citation_volume=4; citation_publication_date=2015; citation_doi=10.1186/s13742-015-0047-8; citation_id=CR72">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform; citation_author=H Li, R Durbin; citation_volume=25; citation_publication_date=2009; citation_pages=1754-1760; citation_doi=10.1093/bioinformatics/btp324; citation_id=CR73">
    <meta name="citation_reference" content="Poplin, R. et al. Scaling accurate genetic variant discovery to tens of thousands of samples. Preprint at bioRxiv 
                  https://doi.org/10.1101/201178
                  
                 (2018).">
    <meta name="citation_reference" content="Van der Auwera, G. A. &amp; O’Connor, B. D. Genomics in the Cloud: Using Docker, GATK, and WDL in Terra 1st edn (O’Reilly Media, 2020).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=The MicroArray Quality Control (MAQC)-II study of common practices for the development and validation of microarray-based predictive models; citation_author=L Shi; citation_volume=28; citation_publication_date=2010; citation_pages=827-838; citation_doi=10.1038/nbt.1665; citation_id=CR76">
    <meta name="citation_reference" content="citation_journal_title=Genet. Med.; citation_title=Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology; citation_author=S Richards; citation_volume=17; citation_publication_date=2015; citation_pages=405-424; citation_doi=10.1038/gim.2015.30; citation_id=CR77">
    <meta name="citation_reference" content="citation_journal_title=Hum. Mutat.; citation_title=ClinGen Variant Curation Expert Panel experiences and standardized processes for disease and gene‐level specification of the ACMG/AMP guidelines for sequence variant interpretation; citation_author=EA Rivera‐Muñoz; citation_volume=39; citation_publication_date=2018; citation_pages=1614-1622; citation_doi=10.1002/humu.23645; citation_id=CR78">
    <meta name="citation_author" content="Hao, Limin">
    <meta name="citation_author_institution" content="Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA">
    <meta name="citation_author" content="Kraft, Peter">
    <meta name="citation_author_institution" content="Department of Epidemiology, Harvard T.H. Chan School of Public Health, Boston, USA">
    <meta name="citation_author" content="Berriz, Gabriel F.">
    <meta name="citation_author_institution" content="Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA">
    <meta name="citation_author" content="Hynes, Elizabeth D.">
    <meta name="citation_author_institution" content="Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA">
    <meta name="citation_author" content="Koch, Christopher">
    <meta name="citation_author_institution" content="Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA">
    <meta name="citation_author" content="Korategere V Kumar, Prathik">
    <meta name="citation_author_institution" content="Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA">
    <meta name="citation_author" content="Parpattedar, Shruti S.">
    <meta name="citation_author_institution" content="Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA">
    <meta name="citation_author" content="Steeves, Marcie">
    <meta name="citation_author_institution" content="Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA">
    <meta name="citation_author_institution" content="Medical Genetics, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Yu, Wanfeng">
    <meta name="citation_author_institution" content="Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA">
    <meta name="citation_author" content="Antwi, Ashley A.">
    <meta name="citation_author_institution" content="Veterans Affairs Boston Healthcare System, Boston, USA">
    <meta name="citation_author" content="Brunette, Charles A.">
    <meta name="citation_author_institution" content="Veterans Affairs Boston Healthcare System, Boston, USA">
    <meta name="citation_author" content="Danowski, Morgan">
    <meta name="citation_author_institution" content="Veterans Affairs Boston Healthcare System, Boston, USA">
    <meta name="citation_author" content="Gala, Manish K.">
    <meta name="citation_author_institution" content="Division of Gastroenterology, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author_institution" content="Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Green, Robert C.">
    <meta name="citation_author_institution" content="Harvard Medical School, Boston, USA">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and the Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, USA">
    <meta name="citation_author_institution" content="Precision Population Health, Ariadne Labs, Boston, USA">
    <meta name="citation_author" content="Jones, Natalie E.">
    <meta name="citation_author_institution" content="Veterans Affairs Boston Healthcare System, Boston, USA">
    <meta name="citation_author_institution" content="Broad Institute of Harvard and the Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author" content="Lewis, Anna C. F.">
    <meta name="citation_author_institution" content="E J Safra Center for Ethics, Harvard University, Cambridge, USA">
    <meta name="citation_author" content="Lubitz, Steven A.">
    <meta name="citation_author_institution" content="Cardiovascular Disease Initiative, Broad Institute of Harvard and the Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author_institution" content="Cardiovascular Research Center, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author_institution" content="Demoulas Center for Cardiac Arrhythmias, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Natarajan, Pradeep">
    <meta name="citation_author_institution" content="Harvard Medical School, Boston, USA">
    <meta name="citation_author_institution" content="Cardiovascular Disease Initiative, Broad Institute of Harvard and the Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author_institution" content="Cardiovascular Research Center, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Vassy, Jason L.">
    <meta name="citation_author_institution" content="Veterans Affairs Boston Healthcare System, Boston, USA">
    <meta name="citation_author_institution" content="Harvard Medical School, Boston, USA">
    <meta name="citation_author_institution" content="Department of Medicine, Brigham and Women’s Hospital, Boston, USA">
    <meta name="citation_author_institution" content="Precision Population Health, Ariadne Labs, Boston, USA">
    <meta name="citation_author" content="Lebo, Matthew S.">
    <meta name="citation_author_institution" content="Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, USA">
    <meta name="citation_author_institution" content="Harvard Medical School, Boston, USA">
    <meta name="citation_author_institution" content="Department of Pathology, Brigham and Women’s Hospital, Boston, USA">
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">
    <meta name="twitter:site" content="@naturemedicine">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image:alt" content="Content cover image">
    <meta name="twitter:title" content="Development of a clinical polygenic risk score assay and reporting workflow">
    <meta name="twitter:description" content="Nature Medicine - The first report from the prospective GenoVA Study provides preliminary insights into the development of a polygenic risk score assay in a clinical setting and discusses the...">
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig1_HTML.png">
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41591-022-01767-6">
    <meta property="og:type" content="article">
    <meta property="og:site_name" content="Nature">
    <meta property="og:title" content="Development of a clinical polygenic risk score assay and reporting workflow - Nature Medicine">
    <meta property="og:description" content="The first report from the prospective GenoVA Study provides preliminary insights into the development of a polygenic risk score assay in a clinical setting and discusses the challenges of generating, interpreting and reporting results.">
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig1_HTML.png">
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
<style type="text/css">.cc-banner{background-color:#01324b;border:none!important;bottom:0;box-sizing:border-box;color:#fff!important;left:0;line-height:1.3;margin:auto 0 0;max-width:100%;outline:0;overflow:visible;padding:0;position:fixed;right:0;width:100%;z-index:99999}.cc-banner::backdrop{background-color:#0000004d}.cc-banner *{color:inherit!important}.cc-banner:focus{box-shadow:none;outline:0}.cc-banner a{color:#fff!important;text-decoration:underline}.cc-banner a:active,.cc-banner a:focus,.cc-banner a:hover{color:inherit;text-decoration:none}.cc-banner a:focus{outline:3px solid #08c!important}.cc-banner h2,.cc-banner h3,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-banner .cc-h2,.cc-banner h2{font-size:18px}.cc-banner .cc-h3,.cc-banner h3{font-size:16px}.cc-banner .cc-h4,.cc-banner .cc-h5,.cc-banner .cc-h6,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-size:14px}.cc-banner .cc-button{font-size:16px}.cc-banner__content{background-color:#01324b;display:flex;flex-direction:column;margin:0 auto;max-height:90vh;max-width:100%;padding:16px;position:relative}.cc-banner__content:focus{outline:0}@media (min-width:680px){.cc-banner__content{padding:12px}}@media (min-width:980px){.cc-banner__content{max-height:60vh;padding-bottom:20px;padding-top:20px}}@media (min-width:1320px){.cc-banner__content{padding-bottom:40px;padding-top:40px}}.cc-banner__container{display:flex;flex-direction:column;margin:auto;max-width:1320px;overflow:auto}.cc-banner__title{background:none!important;border:0;flex-shrink:0;font-size:18px!important;font-size:22px!important;font-weight:700!important;letter-spacing:normal;margin:0 0 12px!important;padding:0}@media (min-width:680px){.cc-banner__title{font-size:24px!important;margin:0 0 16px!important}}@media (min-width:1320px){.cc-banner__title{font-size:26px!important;margin:0 0 24px!important}}.cc-banner__body{display:flex;flex-direction:column;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif!important;overflow-x:hidden;overflow-y:auto;padding:3px 3px 16px}@media (min-width:980px){.cc-banner__body{flex-direction:row}}.cc-banner__policy p{font-size:16px!important;margin:0;max-width:none;padding:0}.cc-banner__policy p:not(:last-child){margin:0 0 16px}@media (min-width:980px){.cc-banner__policy p:not(:last-child){margin:0 0 24px}}.cc-banner__policy p a{font-size:16px!important;font-weight:700}.cc-banner__footer{box-shadow:none;flex-shrink:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif !important!important;margin:0;padding:12px 0 3px 3px;position:relative}@media (min-width:380px){.cc-banner__footer{align-items:stretch;display:flex;flex-wrap:wrap}}@media (min-width:680px){.cc-banner__footer{box-shadow:none;flex-wrap:nowrap;width:40%}}.cc-banner__footer .cc-banner__button{flex:1 1 auto;font-weight:700;overflow:hidden;padding:.5em 1em;width:100%}@media (min-width:680px){.cc-banner__footer .cc-banner__button{max-width:275px}}.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:12px}@media (min-width:680px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:0;margin-right:16px}}@media (min-width:980px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-right:24px}}@media (min-width:680px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:16px!important;margin-right:0!important}}@media (min-width:980px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:24px!important;margin-right:0!important}}.cc-banner__button-preferences{padding-left:0;padding-right:0}@media (min-width:380px){.cc-banner__button-preferences{flex:0 0 auto;margin:auto}}@media (min-width:680px){.cc-banner__button-preferences{margin:0}}@media (min-width:380px) and (max-width:680px){.cc-banner__button-break{display:block}}@media (min-width:680px){.cc-banner--is-tcf .cc-banner__footer{width:auto}}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__footer{padding-right:48px;width:66%}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px!important}.cc-banner--is-tcf .cc-banner__stacks{margin:16px 0 0}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__stacks{display:flex;flex:0 0 33%;flex-direction:column;margin:0 0 0 48px;overflow:auto}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px;margin:0 0 12px}.cc-banner--is-tcf .cc-banner__stacks-details{padding:0 3px}.cc-banner--is-tcf .cc-banner .cc-details{border-color:#fff3!important}.cc-box{border-radius:10px;padding:12px}.cc-box--info{background-color:#eff6fb}.cc-box--light{background-color:#faf9f6}.cc-button{border:2px solid #0000;border-radius:32px!important;color:inherit;cursor:pointer;font-size:14px;font-weight:700!important;left:auto;line-height:1.2;margin:0;padding:.5em 1em;right:auto;text-transform:none!important;transition:all .2s}@media (min-width:680px){.cc-button{font-size:14px;padding:.75em 1em}}.cc-button--sm{font-size:12px;padding:2px 8px!important}.cc-button--primary{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--primary:focus{outline:3px solid #08c}.cc-button--primary:focus,.cc-button--primary:hover,.cc-button--secondary{background-color:#fff!important;border-color:#025e8d!important;color:#025e8d!important}.cc-button--secondary:focus{background-color:#025e8d!important;border-color:#fff!important;color:#fff!important;outline:3px solid #08c!important}.cc-button--secondary:hover{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--secondary:active{border:2px solid #01324b!important}.cc-button--secondary.cc-button--contrast{border-color:#fff!important}.cc-button--secondary.cc-button--contrast:hover{background-color:initial!important;color:#fff!important}.cc-button--tertiary{background-color:#f0f7fc!important;background-image:none;border:2px solid #f0f7fc;color:#025e8d!important;text-decoration:underline!important;text-underline-offset:.25em}.cc-button--tertiary:focus{outline:3px solid #08c!important}.cc-button--tertiary:hover{text-decoration-thickness:.25em}.cc-button--tertiary:active{color:#013c5b}.cc-button--link{background-color:initial!important;border-color:#0000!important;border-radius:0!important;color:inherit!important;padding:0!important;text-decoration:underline!important;width:inherit!important}.cc-button--link:focus{outline:3px solid #08c}.cc-button--link:hover{background-color:initial!important;box-shadow:none;text-decoration:none}.cc-button--text{border-radius:0;padding:0}.cc-button--details{padding-right:24px!important;position:relative;width:auto}.cc-button--details:after,.cc-button--details:before{background-color:currentColor;content:"";position:absolute;transition:transform .25s ease-out}.cc-button--details:before{height:10px;margin-top:-5px;right:12px;top:50%;width:2px}.cc-button--details:after{height:2px;margin-top:-1px;right:8px;top:50%;width:10px}.cc-button--details.cc-active:before{transform:rotate(90deg);transform-origin:center}.cc-button--details.cc-active:after{display:none}.cc-details{border-bottom:1px solid #0000001a;padding:12px 0;position:relative;width:100%}.cc-details__summary{align-items:center;display:flex;font-size:14px;font-weight:700;list-style-type:none}.cc-details__summary:focus,.cc-details__summary:focus-visible{outline:3px solid #08c!important;will-change:auto}.cc-details__title{align-items:baseline;display:flex}.cc-details__title h2,.cc-details__title h3,.cc-details__title h4,.cc-details__title h5{line-height:1.4;margin:0!important;padding:0!important}.cc-details__title svg{flex-shrink:0;height:auto;margin-right:8px;position:relative;top:-1px;transition:all .2s;width:auto}.cc-details[open] .cc-details__title svg{top:2px;transform:rotate(90deg)}.cc-details__switch{margin-left:auto}.cc-details__section{padding:16px 0 0 18px}.cc-details__section p{margin:0}.cc-details__section p:not(:last-child){margin-bottom:12px}details summary::-webkit-details-marker{display:none}.cc-radio{align-items:center;display:flex;position:relative}.cc-radio *{cursor:pointer}.cc-radio__input{height:22px;left:0;position:absolute;top:0;width:22px}.cc-radio__input:focus{outline:none}.cc-radio__label{color:inherit;font-size:14px;font-weight:700;line-height:23px;margin:0;padding-left:28px}.cc-radio__label:after,.cc-radio__label:before{background-color:#fff;content:"";display:block;position:absolute;transition:transform .25s ease-out}.cc-radio__label:before{border:1px solid #777;border-radius:50%;height:22px;left:0;top:0;width:22px}.cc-radio__label:after{border:7px solid #025e8d;border-radius:50%;height:0;left:4px;opacity:0;top:4px;width:0}.cc-radio__input:focus+.cc-radio__label:before{box-shadow:0 0 0 2px #08c;outline:none}.cc-radio__label--hidden{display:none}.cc-radio__input:checked+.cc-radio__label:after{opacity:1}.cc-radio__input:disabled{cursor:default}.cc-radio__input:disabled+.cc-radio__label{cursor:default;opacity:.5}.cc-switch *{cursor:pointer}.cc-switch{align-items:center;display:flex;line-height:1}.cc-switch__label{cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;-webkit-user-select:none;user-select:none}.cc-switch__input[type=checkbox]{-webkit-appearance:none;appearance:none;background-color:#fff!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='rgba(0, 0, 0, 0.25)'/%3E%3C/svg%3E")!important;background-position:0!important;background-repeat:no-repeat!important;border:1px solid #777!important;border-radius:16px!important;height:18px!important;margin:0 8px 0 0!important;transition:background-position .2s ease-in-out!important;width:32px!important}.cc-switch__input[type=checkbox]:focus{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%2380b3cc'/%3E%3C/svg%3E")!important}.cc-switch__input[type=checkbox]:checked{background-color:#025e8d!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%23fff'/%3E%3C/svg%3E")!important;background-position:100%!important}.cc-switch__input[type=checkbox]:checked~.cc-switch__label>.cc-switch__text-off,.cc-switch__input[type=checkbox]:not(:checked)~.cc-switch__label>.cc-switch__text-on{display:none}.cc-switch__input[type=checkbox]:hover{cursor:pointer;outline:none}.cc-switch__input[type=checkbox]:focus-visible{cursor:pointer;outline:3px solid #fc0}.cc-switch__input[type=checkbox]:checked:before{background-color:none!important;color:#0000!important}html[dir=rtl] .cc-switch__input[type=checkbox]{margin-left:8px!important;margin-right:0!important}.cc-list>ul,ul.cc-list{list-style-type:disc;margin-left:0;padding-left:0}.cc-list>ul>li,ul.cc-list>li{margin-left:1em}.cc-list>ul>li:not(:last-child),ul.cc-list>li:not(:last-child){margin-bottom:4px}.cc-list__title{margin-bottom:8px!important}.cc-list--inline>ul,ul.cc-list--inline{display:flex;flex-wrap:wrap;list-style-type:none}.cc-list--inline>ul>li,ul.cc-list--inline>li{margin:0 .5em .5em 0}.cc-overlay{background-color:#0000004d;z-index:99998}.cc-overlay,.cc-preferences{bottom:0;left:0;position:fixed;right:0;top:0}.cc-preferences{background-color:#050a14f2!important;border:0;box-sizing:border-box;color:#111;font-family:sans-serif!important;line-height:1.4;margin:auto;max-height:100vh;overflow:auto;padding:0;z-index:100000}.cc-preferences:focus{outline:none}.cc-preferences *,.cc-preferences :after,.cc-preferences :before{box-sizing:inherit!important}.cc-preferences h2,.cc-preferences h3,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-preferences .cc-h2,.cc-preferences h2{font-size:18px}.cc-preferences .cc-h3,.cc-preferences h3{font-size:16px}.cc-preferences .cc-h4,.cc-preferences .cc-h5,.cc-preferences .cc-h6,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-size:14px}.cc-preferences a{color:#025e8d;text-decoration:underline}.cc-preferences a:hover{color:inherit;text-decoration:none}.cc-preferences h3{background:none;color:#111;padding:0;text-transform:none}dialog.cc-preferences{background-color:initial}dialog.cc-preferences::backdrop{background-color:#000000e6}.cc-preferences__dialog{display:flex;flex-direction:column;margin:auto;max-height:100vh;max-width:860px;padding:12px;position:relative}.cc-preferences__dialog>:last-child{border-bottom-left-radius:10px;border-bottom-right-radius:10px}@media (min-width:980px){.cc-preferences__dialog{padding:16px}}.cc-preferences__close{background:#0000!important;border:1px solid #ececec;border-radius:50%;color:#111!important;cursor:pointer;font-family:Times New Roman,serif;font-size:40px;height:40px;left:auto;line-height:1;margin-top:-20px;padding:0!important;position:absolute;right:20px;top:50%;width:40px}.cc-preferences__close:focus{outline:3px solid #08c}.cc-preferences__close-label{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__header{background:#fff!important;border-bottom:1px solid #0000001a;border-top-left-radius:10px;border-top-right-radius:10px;padding:16px;position:relative;text-align:center}.cc-preferences__title{background:none!important;color:#111!important;font-family:sans-serif!important;font-size:18px!important;font-weight:700!important;margin:0!important;padding:0 16px 0 0!important}@media (min-width:480px){.cc-preferences__title{padding-right:0!important}}@media (min-width:980px){.cc-preferences__title{font-size:22px!important}}.cc-preferences__body{background:#fff!important;flex:1 1 auto;min-height:200px;overflow-x:hidden;overflow-y:auto;padding:16px}.cc-preferences__footer{background:#fff!important;border-top:1px solid #d0d0d0;box-shadow:0 0 5px 0 #0003;margin-bottom:0;padding:12px;position:relative}@media (min-width:480px){.cc-preferences__footer{align-items:stretch;display:flex}}.cc-preferences__footer>.cc-button{display:block;width:100%}@media (min-width:480px){.cc-preferences__footer>.cc-button{flex:1 1 auto}}@media (min-width:980px){.cc-preferences__footer>.cc-button{flex-basis:auto}}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:first-child){margin-left:12px}}.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:8px}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:0}}.cc-preferences__categories{list-style:none;margin:0;padding:0}.cc-preferences__category:not(:last-child){border-bottom:1px solid #0000001a;margin-bottom:12px;padding-bottom:12px}.cc-preferences__category-description{font-size:14px;margin:0 0 8px;padding:0}.cc-preferences__category-footer{align-items:center;display:flex;justify-content:space-between}.cc-preferences__status{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__controls{display:flex;margin:0}.cc-preferences__controls>:not(:last-child){margin-right:12px}.cc-preferences__always-on{font-size:14px;font-weight:700;padding-left:26px;position:relative}.cc-preferences__always-on:before{background:#0000!important;border:solid;border-top-color:#0000;border-width:0 0 4px 4px;content:"";display:block;height:10px;left:0;position:absolute;top:2px;transform:rotate(-45deg);width:18px}.cc-preferences__details{background-color:#eff6fb!important;border-radius:10px;font-size:14px;margin:12px 0 0;padding:12px}.cc-preferences__cookie-list,.cc-preferences__provider-list{list-style:none;margin:0;padding:0}.cc-preferences__provider-list{columns:170px}.cc-preferences__cookie-title{font-size:1em;font-style:normal;font-weight:400;margin:0}.cc-preferences__cookie-description{font-size:1em;margin:0 0 8px}.cc-preferences__cookie-domain,.cc-preferences__cookie-lifespan{border-left:1px solid #999;color:#666;margin-left:8px;padding-left:4px}body.cc-has-preferences-open{overflow:hidden;position:relative}.cc-table{border-collapse:collapse;width:100%}.cc-table tbody tr{border-top:1px solid #0000001a}.cc-table td,.cc-table th{font-size:14px;padding:4px 8px;vertical-align:top}.cc-table+.cc-table{margin-top:8px}.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:70%}@media (min-width:680px){.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:80%}}.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{padding-left:8px;width:40%}@media (min-width:680px){.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{width:30%}}.cc-tabs,.cc-tabs>ul{background-color:#faf9f6;display:flex;list-style:none;margin:0;padding:0}.cc-tabs__panel{border-top:1px solid #ececec;margin-top:-1px}.cc-tabs__button{background-color:initial;border:0;border-left:1px solid #0000;border-right:1px solid #0000;color:#111;font-size:16px;font-weight:700;margin:0;padding:15px 16px 12px;position:relative}.cc-tabs__button:before{background-color:initial;content:"";display:block;height:3px;left:0;position:absolute;top:0;width:100%}.cc-tabs__button.cc-active{background-color:#fff;border-color:#025e8d #ececec #ececec;color:#111}.cc-tabs__button.cc-active:before{background-color:#025e8d}.cc-tabs__content{background-color:#fff;display:none;font-size:14px;padding:16px}.cc-tabs__section:not(:first-child){padding-top:24px}.cc-tcf{font-size:14px;margin-top:16px}.cc-tcf__list{list-style:none;margin:0;padding:0}.cc-vendor-count{color:#666;font-size:16px;margin:0 0 8px}.cc-hide{display:none!important}.cc-show{display:block!important}.cc-external-link{background-color:#ececec;background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTcgMGExIDEgMCAxIDEgMCAySDIuNmMtLjM3MSAwLS42LjIwOS0uNi41djE1YzAgLjI5MS4yMjkuNS42LjVoMTQuOGMuMzcxIDAgLjYtLjIwOS42LS41VjEzYTEgMSAwIDAgMSAyIDB2NC41YzAgMS40MzgtMS4xNjIgMi41LTIuNiAyLjVIMi42QzEuMTYyIDIwIDAgMTguOTM4IDAgMTcuNXYtMTVDMCAxLjA2MiAxLjE2MiAwIDIuNiAwem02IDBoNmwuMDc1LjAwMy4xMjYuMDE3LjExMS4wMy4xMTEuMDQ0LjA5OC4wNTIuMDk2LjA2Ny4wOS4wOGExIDEgMCAwIDEgLjA5Ny4xMTJsLjA3MS4xMS4wNTQuMTE0LjAzNS4xMDUuMDMuMTQ4TDIwIDF2NmExIDEgMCAwIDEtMiAwVjMuNDE0bC02LjY5MyA2LjY5M2ExIDEgMCAwIDEtMS40MTQtMS40MTRMMTYuNTg0IDJIMTNhMSAxIDAgMCAxLS45OTMtLjg4M0wxMiAxYTEgMSAwIDAgMSAxLTEiLz48L3N2Zz4=");background-position:right 8px center;background-repeat:no-repeat;background-size:10px auto;border-radius:3px;box-shadow:0 1px 0 #0003;color:#111!important;display:inline-block;font-size:12px;padding:4px 26px 4px 8px;text-decoration:none!important}.cc-external-link:hover{text-decoration:underline!important}.cc-m-0{margin:0!important}.cc-grey{color:#666}</style><script async="" type="text/javascript" src="https://injections.readcube.com/nature/inject.fa744f3f.js"></script></head>
<body class="article-page"><dialog class="cc-banner" data-cc-banner="" data-nosnippet="" aria-labelledby="cc-banner-label" open="">
		<div class="cc-banner__content" autofocus="" tabindex="-1">
			<div class="cc-banner__container">
				<div class="cc-banner__header">
					<h2 class="cc-banner__title" id="cc-banner-label">Your privacy, your choice</h2>
				</div>
				<div class="cc-banner__body">
					<div class="cc-banner__policy">
						<p>We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.</p><p>By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.</p><p>See our <a href="https://www.nature.com/info/privacy" data-cc-action="privacy">privacy policy</a> for more information on the use of your personal data.</p><p><button type="button" data-cc-action="preferences" class="cc-button cc-button--link cc-button--text">Manage preferences</button> for further information and to change your choices.</p>
					</div>
					
				</div>
				<div class="cc-banner__footer">
					
			<button data-cc-action="accept" class="cc-button cc-button--secondary cc-button--contrast cc-banner__button cc-banner__button-accept">Accept all cookies</button>
			
		
				</div>
			</div>
		</div>
	</dialog>

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90" style="display: none !important;">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-test="top-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41591-022-01767-6;doi=10.1038/s41591-022-01767-6;subjmeta=208,457,459,499,631,692,700;kwrd=Disease+prevention,Population+genetics,Risk+factors">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=-1932855548&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01767-6%26doi%3D10.1038/s41591-022-01767-6%26subjmeta%3D208,457,459,499,631,692,700%26kwrd%3DDisease+prevention,Population+genetics,Risk+factors">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=-1932855548&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01767-6%26doi%3D10.1038/s41591-022-01767-6%26subjmeta%3D208,457,459,499,631,692,700%26kwrd%3DDisease+prevention,Population+genetics,Risk+factors"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header="" data-track-component="nature-150-split-header" style="border-color:#e40428">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nm" data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" height="32" alt="Nature Medicine">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
                            </a><div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected="">All journals</option>
                                    <option value="nm">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41591-022-01767-6"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header" hidden="">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-articles" data-track="click" data-track-action="research articles" data-track-label="link" data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-and-analysis" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link" data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/podcast" data-track="click" data-track-action="podcasts" data-track-label="link" data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/current-issue" data-track="click" data-track-action="current issue" data-track-label="link" data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/collections" data-track="click" data-track-action="collections" data-track-label="link" data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://www.facebook.com/Nature-Medicine-193691346949/" data-track="click" data-track-action="facebook" data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://twitter.com/naturemedicine" data-track="click" data-track-action="twitter" data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=5" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/aims" data-track="click" data-track-action="aims &amp; scope" data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-information" data-track="click" data-track-action="journal information" data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-impact" data-track="click" data-track-action="journal metrics" data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editors" data-track="click" data-track-action="about the editors" data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-cross-journal-editorial-team" data-track="click" data-track-action="research cross-journal editorial team" data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-cross-journal-editorial-team" data-track="click" data-track-action="reviews cross-journal editorial team" data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/statistics-advisory-panel" data-track="click" data-track-action="statistical advisory panel" data-track-label="link">
                                    Statistical Advisory Panel
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/our-publishing-models" data-track="click" data-track-action="our publishing models" data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-values-statement" data-track="click" data-track-action="editorial values statement" data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-policies" data-track="click" data-track-action="editorial policies" data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/content" data-track="click" data-track-action="content types" data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="javascript:;" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                        </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/submission-guidelines" data-track="click" data-track-action="submission guidelines" data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-context="header publish with us dropdown menu" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/open-access-funding" data-test="funding-eligibility-link" data-track="click_explore_funding" data-track-context="header publish with us" data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link" href="https://mts-nmed.nature.com/cgi-bin/main.plex" data-track="click_submit_manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-action="submit manuscript" data-track-label="link (publish with us dropdown menu)" data-track-external="" data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link" href="https://journal-alerts.springernature.com/subscribe?journal_id=41591" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-label="link (desktop site header)" data-track-external="">
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature medicine"><span itemprop="name">nature medicine</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm/articles?type=article" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" id="js-enable-context-bar" data-test="context-bar" data-context-bar="" aria-hidden="true">
                    <div class="c-context-bar__container u-container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            Development of a clinical polygenic risk score assay and reporting workflow
                        </div>
                        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01767-6.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01767-6.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2022-04-18">18 April 2022</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Development of a clinical polygenic risk score assay and reporting workflow</h1>
                    <ul class="c-article-author-list c-article-author-list--short js-no-scroll" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Limin-Hao-Aff1" data-author-popup="auth-Limin-Hao-Aff1" data-author-search="Hao, Limin" data-track-context="researcher popup with no profile" data-track-index="1_20">Limin Hao</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Peter-Kraft-Aff2" data-author-popup="auth-Peter-Kraft-Aff2" data-author-search="Kraft, Peter" data-track-context="researcher popup with no profile" data-track-index="2_20">Peter Kraft</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-4472-8103"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-4472-8103</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Gabriel_F_-Berriz-Aff1" data-author-popup="auth-Gabriel_F_-Berriz-Aff1" data-author-search="Berriz, Gabriel F." data-track-context="researcher popup with no profile" data-track-index="3_20">Gabriel F. Berriz</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Elizabeth_D_-Hynes-Aff1" data-author-popup="auth-Elizabeth_D_-Hynes-Aff1" data-author-search="Hynes, Elizabeth D." data-track-context="researcher popup with no profile" data-track-index="4_20">Elizabeth D. Hynes</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3817-2145"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3817-2145</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Christopher-Koch-Aff1" data-author-popup="auth-Christopher-Koch-Aff1" data-author-search="Koch, Christopher" data-track-context="researcher popup with no profile" data-track-index="5_20">Christopher Koch</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Prathik-Korategere_V_Kumar-Aff1" data-author-popup="auth-Prathik-Korategere_V_Kumar-Aff1" data-author-search="Korategere V Kumar, Prathik" data-track-context="researcher popup with no profile" data-track-index="6_20">Prathik Korategere V Kumar</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Shruti_S_-Parpattedar-Aff1" data-author-popup="auth-Shruti_S_-Parpattedar-Aff1" data-author-search="Parpattedar, Shruti S." data-track-context="researcher popup with no profile" data-track-index="7_20">Shruti S. Parpattedar</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Marcie-Steeves-Aff1-Aff3" data-author-popup="auth-Marcie-Steeves-Aff1-Aff3" data-author-search="Steeves, Marcie" data-track-context="researcher popup with no profile" data-track-index="8_20">Marcie Steeves</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Wanfeng-Yu-Aff1" data-author-popup="auth-Wanfeng-Yu-Aff1" data-author-search="Yu, Wanfeng" data-track-context="researcher popup with no profile" data-track-index="9_20">Wanfeng Yu</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ashley_A_-Antwi-Aff4" data-author-popup="auth-Ashley_A_-Antwi-Aff4" data-author-search="Antwi, Ashley A." data-track-context="researcher popup with no profile" data-track-index="10_20">Ashley A. Antwi</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-2591-4410"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2591-4410</a></span><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Charles_A_-Brunette-Aff4" data-author-popup="auth-Charles_A_-Brunette-Aff4" data-author-search="Brunette, Charles A." data-track-context="researcher popup with no profile" data-track-index="11_20">Charles A. Brunette</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-1620-3526"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1620-3526</a></span><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Morgan-Danowski-Aff4" data-author-popup="auth-Morgan-Danowski-Aff4" data-author-search="Danowski, Morgan" data-track-context="researcher popup with no profile" data-track-index="12_20">Morgan Danowski</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Manish_K_-Gala-Aff5-Aff6" data-author-popup="auth-Manish_K_-Gala-Aff5-Aff6" data-author-search="Gala, Manish K." data-track-context="researcher popup with no profile" data-track-index="13_20">Manish K. Gala</a><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a>,<a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Robert_C_-Green-Aff6-Aff7-Aff8-Aff9" data-author-popup="auth-Robert_C_-Green-Aff6-Aff7-Aff8-Aff9" data-author-search="Green, Robert C." data-track-context="researcher popup with no profile" data-track-index="14_20">Robert C. Green</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-8472-0424"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-8472-0424</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a>,<a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff9" tabindex="-1">9</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Natalie_E_-Jones-Aff4-Aff7" data-author-popup="auth-Natalie_E_-Jones-Aff4-Aff7" data-author-search="Jones, Natalie E." data-track-context="researcher popup with no profile" data-track-index="15_20">Natalie E. Jones</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Anna_C__F_-Lewis-Aff10" data-author-popup="auth-Anna_C__F_-Lewis-Aff10" data-author-search="Lewis, Anna C. F." data-track-context="researcher popup with no profile" data-track-index="16_20">Anna C. F. Lewis</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-1274-386X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-1274-386X</a></span><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Steven_A_-Lubitz-Aff11-Aff12-Aff13" data-author-popup="auth-Steven_A_-Lubitz-Aff11-Aff12-Aff13" data-author-search="Lubitz, Steven A." data-track-context="researcher popup with no profile" data-track-index="17_20">Steven A. Lubitz</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-9599-4866"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-9599-4866</a></span><sup class="u-js-hide"><a href="#Aff11" tabindex="-1">11</a>,<a href="#Aff12" tabindex="-1">12</a>,<a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Pradeep-Natarajan-Aff6-Aff11-Aff12" data-author-popup="auth-Pradeep-Natarajan-Aff6-Aff11-Aff12" data-author-search="Natarajan, Pradeep" data-track-context="researcher popup with no profile" data-track-index="18_20">Pradeep Natarajan</a><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a>,<a href="#Aff11" tabindex="-1">11</a>,<a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jason_L_-Vassy-Aff4-Aff6-Aff8-Aff9" data-author-popup="auth-Jason_L_-Vassy-Aff4-Aff6-Aff8-Aff9" data-author-search="Vassy, Jason L." data-corresp-id="c1" data-track-context="researcher popup with no profile" data-track-index="19_20">Jason L. Vassy<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6113-5564"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6113-5564</a></span><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff6" tabindex="-1">6</a>,<a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff9" tabindex="-1">9</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 20 authors for this article" title="Show all 20 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Matthew_S_-Lebo-Aff1-Aff6-Aff14" data-author-popup="auth-Matthew_S_-Lebo-Aff1-Aff6-Aff14" data-author-search="Lebo, Matthew S." data-track-context="researcher popup with no profile" data-track-index="20_20">Matthew S. Lebo</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-9733-5207"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-9733-5207</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff6" tabindex="-1">6</a>,<a href="#Aff14" tabindex="-1">14</a></sup><sup class="u-js-hide">&nbsp;<a href="#na2" tabindex="-1">na2</a></sup>&nbsp;</li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nm" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Medicine</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;28</b>,&nbsp;<span class="u-visually-hidden">pages </span>1006–1013 (<span data-test="article-publication-year">2022</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item" data-test="access-count">
                        <p class="c-article-metrics-bar__count">36k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item" data-test="citation-count">
                        <p class="c-article-metrics-bar__count">89 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item" data-test="altmetric-score">
                            <p class="c-article-metrics-bar__count">229 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41591-022-01767-6/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Implementation of polygenic risk scores (PRS) may improve disease prevention and management but poses several challenges: the construction of clinically valid assays, interpretation for individual patients, and the development of clinical workflows and resources to support their use in patient care. For the ongoing Veterans Affairs Genomic Medicine at Veterans Affairs (GenoVA) Study we developed a clinical genotype array-based assay for six published PRS. We used data from 36,423 Mass General Brigham Biobank participants and adjustment for population structure to replicate known PRS–disease associations and published PRS thresholds for a disease odds ratio (OR) of 2 (ranging from 1.75 (95% CI: 1.57–1.95) for type 2 diabetes to 2.38 (95% CI: 2.07–2.73) for breast cancer). After confirming the high performance and robustness of the pipeline for use as a clinical assay for individual patients, we analyzed the first 227 prospective samples from the GenoVA Study and found that the frequency of PRS corresponding to published OR &gt; 2 ranged from 13/227 (5.7%) for colorectal cancer to 23/150 (15.3%) for prostate cancer. In addition to the PRS laboratory report, we developed physician- and patient-oriented informational materials to support decision-making about PRS results. Our work illustrates the generalizable development of a clinical PRS assay for multiple conditions and the technical, reporting and clinical workflow challenges for implementing PRS information in the clinic.</p></div></div></section>

            
                
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-021-03243-6/MediaObjects/41586_2021_3243_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41586-021-03243-6?fromPaywallRec=false" data-track="select_recommendations_1" data-track-context="inline recommendations" data-track-action="click recommendations inline - 1" data-track-label="10.1038/s41586-021-03243-6">Improving reporting standards for polygenic scores in risk prediction studies
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">10 March 2021</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41576-024-00794-y/MediaObjects/41576_2024_794_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41576-024-00794-y?fromPaywallRec=false" data-track="select_recommendations_2" data-track-context="inline recommendations" data-track-action="click recommendations inline - 2" data-track-label="10.1038/s41576-024-00794-y">Biobanking with genetics shapes precision medicine and global health
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">20 November 2024</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41591-024-02796-z/MediaObjects/41591_2024_2796_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41591-024-02796-z?fromPaywallRec=false" data-track="select_recommendations_3" data-track-context="inline recommendations" data-track-action="click recommendations inline - 3" data-track-label="10.1038/s41591-024-02796-z">Selection, optimization and validation of ten chronic disease polygenic risk scores for clinical implementation in diverse US populations
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">19 February 2024</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1749514239,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Genome-wide association studies (GWAS) have identified thousands of genomic variants significantly associated with a range of common complex human diseases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Shendure, J., Findlay, G. M. &amp; Snyder, M. W. Genomic medicine: progress, pitfalls, and promise. Cell 177, 45–57 (2019)." href="/articles/s41591-022-01767-6#ref-CR1" id="ref-link-section-d73985140e894">1</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="GWAS Catalog (National Human Genome Research Institute); 
                  https://www.ebi.ac.uk/gwas/
                  
                " href="/articles/s41591-022-01767-6#ref-CR2" id="ref-link-section-d73985140e897">2</a></sup>. Given that the risk conferred by an individual common variant is usually insignificantly small, investigators have aggregated risk alleles across the genome into genetic risk scores to provide a single measure of genetic association for a given trait due to known common variant effects. Although the earliest genetic scores consisted only of variants meeting genome-wide significance<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Meigs, J. B. et al. Genotype score in addition to common risk factors for prediction of type 2 diabetes. N. Engl. J. Med. 359, 2208–2219 (2008)." href="#ref-CR3" id="ref-link-section-d73985140e901">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ripatti, S. et al. A multilocus genetic risk score for coronary heart disease: case-control and prospective cohort analyses. Lancet 376, 1393–1400 (2010)." href="#ref-CR4" id="ref-link-section-d73985140e901_1">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Zheng, S. L. et al. Cumulative association of five genetic variants with prostate cancer. N. Engl. J. Med. 358, 910–919 (2008)." href="/articles/s41591-022-01767-6#ref-CR5" id="ref-link-section-d73985140e904">5</a></sup>, recent computational and methodological advances have leveraged the summary statistics of all available variants from increasingly larger GWAS to calculate polygenic risk scores (PRS)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Boyle, E. A., Li, Y. I. &amp; Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. Cell 169, 1177–1186 (2017)." href="#ref-CR6" id="ref-link-section-d73985140e908">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Khera, A. V. et al. Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations. Nat. Genet. 50, 1219–1224 (2018)." href="#ref-CR7" id="ref-link-section-d73985140e908_1">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vilhjálmsson, B. J. et al. Modeling linkage disequilibrium increases accuracy of polygenic risk scores. Am. J. Hum. Genet. 97, 576–592 (2015)." href="#ref-CR8" id="ref-link-section-d73985140e908_2">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Inouye, M. et al. Genomic risk prediction of coronary artery disease in 480,000 adults: implications for primary prevention. J. Am. Coll. Cardiol. 72, 1883–1893 (2018)." href="/articles/s41591-022-01767-6#ref-CR9" id="ref-link-section-d73985140e911">9</a></sup>. For some diseases, a PRS in the upper tail of the distribution approximates risks equivalent to those conferred by established clinical risk factors and by genetic variants associated with monogenic disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Khera, A. V. et al. Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations. Nat. Genet. 50, 1219–1224 (2018)." href="/articles/s41591-022-01767-6#ref-CR7" id="ref-link-section-d73985140e915">7</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Conti, D. V. et al. Trans-ancestry genome-wide association meta-analysis of prostate cancer identifies new susceptibility loci and informs genetic risk prediction. Nat. Genet. 53, 65–75 (2021)." href="/articles/s41591-022-01767-6#ref-CR10" id="ref-link-section-d73985140e918">10</a></sup>. Although PRS are typically derived from weights from cross-sectional GWAS of prevalent disease cases and controls, further work has demonstrated their potential to estimate the risk of incident disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Klarin, D. et al. Genome-wide association analysis of venous thromboembolism identifies new risk loci and genetic overlap with arterial vascular disease. Nat. Genet. 51, 1574–1579 (2019)." href="#ref-CR11" id="ref-link-section-d73985140e922">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Mosley, J. D. et al. Predictive accuracy of a polygenic risk score compared with a clinical risk score for incident coronary heart disease. JAMA 323, 627–635 (2020)." href="#ref-CR12" id="ref-link-section-d73985140e922_1">12</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vassy, J. L. et al. Polygenic type 2 diabetes prediction at the limit of common variant detection. Diabetes 63, 2172–2182 (2014)." href="#ref-CR13" id="ref-link-section-d73985140e922_2">13</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Seibert, T. M. et al. Polygenic hazard score to guide screening for aggressive prostate cancer: development and validation in large scale cohorts. BMJ 360, j5757 (2018)." href="/articles/s41591-022-01767-6#ref-CR14" id="ref-link-section-d73985140e925">14</a></sup>.</p><p>Suitable clinical implementation of PRS is now an area of active research across many disease areas<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="National Human Genome Research Institute (NHGRI). Electronic Medical Records and Genomics (eMERGE) Network 
                  https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE
                  
                 (2020)." href="#ref-CR15" id="ref-link-section-d73985140e932">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Shieh, Y. et al. Breast cancer screening in the precision medicine era: risk-based screening in a population-based trial. J. Natl Cancer Inst. 109, 
                  https://doi.org/10.1093/jnci/djw290
                  
                 (2017)." href="#ref-CR16" id="ref-link-section-d73985140e932_1">16</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Brockman, D. G. et al. Design and user experience testing of a polygenic score report: a qualitative study of prospective users. BMC Med. Genomics 14, 238 (2021)." href="/articles/s41591-022-01767-6#ref-CR17" id="ref-link-section-d73985140e935">17</a></sup>. The translation of PRS from discovery to the clinic can be conceptualized as having at least three necessary phases (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig1">1</a>): the first phase relates to epidemiology and statistical genetics, in which PRS are developed and validated in large cohorts and improved with advances in statistical methods; the second phase involves the laboratory, in which laboratory geneticists must develop an analytically and clinically valid pipeline for calculating, interpreting and reporting PRS results for an individual patient; and the third phase involves patient care, in which a treating physician makes medical decisions after putting a patient’s PRS results into the larger clinical context, which involves non-genetic risk factors, comorbidities and patient preferences. The first phase has seen significant methodological advances<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Choi, S. W., Mak, T. S.-H. &amp; O’Reilly, P. F. Tutorial: a guide to performing polygenic risk score analyses. Nat. Protoc. 15, 2759–2772 (2020)." href="/articles/s41591-022-01767-6#ref-CR18" id="ref-link-section-d73985140e942">18</a></sup> but challenges for the second and third phases remain.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Translation of PRS from discovery to the clinic, including a clinical PRS laboratory pipeline for prospectively collected samples."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Translation of PRS from discovery to the clinic, including a clinical PRS laboratory pipeline for prospectively collected samples.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01767-6/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="455"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p>In phase 1, PRS are developed, validated and compared to optimize performance in large populations. In phase 2, a clinical laboratory chooses publicly available PRS to implement and develop an analytically and clinically valid assay. For the GenoVA Study, genotype array data are imputed against 1000 Genomes Project data and used to calculate published PRS (PRS<sub>raw</sub>). PRS<sub>raw</sub> is adjusted for population structure and standardized as described in the text (PRS<sub>std-adj</sub>). High-risk status for each disease is defined as PRS values above published thresholds for OR &gt; 2. A parallel pipeline annotates and filters variants for potentially actionable pathogenic (P) and likely pathogenic (LP) variants in the ACMG SF v2.0 secondary finding gene list. Variants are manually classified according to American College of Medical Genetics and Genomics–Association for Molecular Pathology (ACMG-AMP) criteria by qualified laboratorians and confirmed using Sanger sequencing. Results from both components of the pipeline are included on the laboratory report. In phase 3 the treating physician uses the whole patient context to interpret the significance of the PRS for the patient’s health and healthcare management. Both the physician and patient will probably need educational and consultative support to make medical decisions based on PRS results.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01767-6/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>A key assumption underlying the laboratory phase is that a laboratory can develop and implement a valid clinical assay and interpretation pipeline to report PRS results for an individual patient. The development of a clinical assay from a published PRS is not trivial, and significant barriers to the process persist. First, uncertainty exists about whether commonly used, cost-effective genotyping arrays and clinical imputation pipelines can calculate a PRS for an individual with the analytic validity expected of a clinical assay, as opposed to one that is adequate for research. Second, laboratories must implement methods to account for the reduced validity of most PRS in patients of non-European and admixed ancestry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. Nat. Genet. 51, 584–591 (2019)." href="/articles/s41591-022-01767-6#ref-CR19" id="ref-link-section-d73985140e975">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Lewis, C. M. &amp; Vassos, E. Polygenic risk scores: from research tools to clinical instruments. Genome Med. 12, 44 (2020)." href="/articles/s41591-022-01767-6#ref-CR20" id="ref-link-section-d73985140e978">20</a></sup>. This limitation applies both to the calculation of the PRS itself for an individual patient and to its clinical interpretation, given that published effect sizes are from populations of primarily European ancestry<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. Nat. Genet. 51, 584–591 (2019)." href="/articles/s41591-022-01767-6#ref-CR19" id="ref-link-section-d73985140e982">19</a></sup>. Third, laboratories must make several decisions about the content and format of a clinical PRS report, including decisions about where the laboratory’s role as an interpretative service ends and where the role of the treating physician in patient care begins. In the patient care phase, there remain unanswered questions about the information and support that physicians need when contextualizing the PRS results of an individual patient to make clinical decisions, and how those decisions affect patient outcomes.</p><p>In the Genomic Medicine at Veterans Affairs (GenoVA) Study (ClinicalTrials.gov identifier: <a href="https://clinicaltrials.gov/ct2/show/NCT04331535">NCT04331535</a>) we have developed processes to advance the laboratory and patient care phases of the clinical translation of PRS. The GenoVA Study is a clinical trial in which patients and their primary care physicians receive a clinical PRS laboratory report on five diseases commonly screened for and initially managed in primary care: coronary artery disease (CAD), type 2 diabetes mellitus (T2D), atrial fibrillation (AFib), colorectal cancer (CRCa), and either prostate cancer (PrCa) in male patients or breast cancer (BrCa) in female patients. Because the objectives of the GenoVA Study are to observe how PRS impact existing disease screening and diagnosis paradigms and enable increased detection of undiagnosed prevalent or newly incident disease, eligible patients have no known diagnoses of the target diseases and are aged 50–70 years, an age range during which much guideline-recommended screening and diagnosis of new disease occurs. Here, we describe the processes created in the GenoVA Study to develop and validate a genotype array-based clinical assay and report for six PRS and to support their effective translation into clinical care by the treating physicians.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Replication of published PRS</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec4">Sample characteristics</h4><p>To demonstrate the accuracy of a prospective PRS pipeline, we first wanted to ensure that we could implement published PRS effectively. We used data from 36,423 Mass General Brigham Biobank (MGBB) participants to replicate the performance of PRS for the six target diseases (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">1</a>). The mean (s.d.) age of MGBB participants was 58.8 (17.1) years (range, 9–106 years), 19,719 (54.1%) were female, and 5,706 (15.7%) were of reported race other than white (white, <i>n</i> = 30,716 (84.3%); Black, <i>n</i> = 1,807 (5.0%); Asian, <i>n</i> = 786 (2.2%); and other or unknown race, <i>n</i> = 3,113 (8.5%) as determined from electronic health record data). Case counts ranged from 392 for CRCa to 3,554 for CAD. Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig2">2a</a> shows the counts of participants with one or multiple target diseases. The most common disease co-occurrences were the combinations of CAD and T2D (<i>n</i> = 641) and CAD and AFib (<i>n</i> = 495).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Frequency of disease and high-risk PRS results by race in the MGBB."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Frequency of disease and high-risk PRS results by race in the MGBB.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01767-6/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="295"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, UpSet plot of total cases of each of six phenotypes in 36,423 biobank participants and the counts of participants with one or more diseases, by reported race. <b>b</b>, UpSet plot of total counts of high-risk PRS results (population structure-adjusted PRS corresponding to OR &gt; 2) for each of six diseases and the counts of participants with one or more high-risk PRS<sub>std-adj</sub> results, by reported race.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01767-6/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec5">Unadjusted and adjusted PRS distributions</h4><p>We identified PRS from large GWAS for the six target diseases, for which the summary statistics (base files with alleles and weights) were publicly available from the Polygenic Score Catalog<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Lambert, S. A. et al. The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation. Nat. Genet. 53, 420–425 (2021)." href="/articles/s41591-022-01767-6#ref-CR21" id="ref-link-section-d73985140e1072">21</a></sup> (AFib, CAD, T2D, BrCa) or the Cancer PRSWeb (CRCa, PrCa)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Fritsche, L. G. et al. Cancer PRSweb: an online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks. Am. J. Hum. Genet. 107, 815–836 (2020)." href="/articles/s41591-022-01767-6#ref-CR22" id="ref-link-section-d73985140e1076">22</a></sup> as of 26 December 2019. Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">2</a> lists the number of single-nucleotide polymorphisms (SNPs) in the base file for each of the six published PRS, ranging from 81 SNPs for CRCa<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Huyghe, J. R. et al. Discovery of common and rare genetic risk variants for colorectal cancer. Nat. Genet. 51, 76–87 (2019)." href="/articles/s41591-022-01767-6#ref-CR23" id="ref-link-section-d73985140e1083">23</a></sup> to 6,917,436 for T2D<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Khera, A. V. et al. Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations. Nat. Genet. 50, 1219–1224 (2018)." href="/articles/s41591-022-01767-6#ref-CR7" id="ref-link-section-d73985140e1087">7</a></sup>, and the subsets of these available as directly genotyped or imputed data from each of three arrays used for MGBB participants, demonstrating minimal loss of information compared with the original published PRS. As shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig3">3</a>, when using the published weights to calculate standardized PRS (PRS<sub>std-raw</sub>, see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01767-6#Sec14">Methods</a>) we observed marked variation in the distribution of each PRS by reported race in the MGBB, most notably in AFib, CAD, and T2D. For example, only 1.7% of white MGBB participants (516/30,716) but almost all of the Black MGBB participants (88.9%, 1,606/1,807) had PRS<sub>std-raw</sub> above the published threshold associated with an odds ratio (OR) = 2 for T2D in the 2018 study by Khera et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Khera, A. V. et al. Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations. Nat. Genet. 50, 1219–1224 (2018)." href="/articles/s41591-022-01767-6#ref-CR7" id="ref-link-section-d73985140e1102">7</a></sup> (Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">3</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">4</a>). The use of residualized, population structure-adjusted, standardized PRS (PRS<sub>std-adj</sub>, see <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01767-6#Sec14">Methods</a>) minimized this variation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig3">3</a>), such that, for example, 8.6% of white MGBB participants (2,651/30,716) and 4.2% of Black MGBB participants (75/1,807) had a T2D PRS<sub>std-adj</sub> above the published OR &gt; 2 threshold. The distributions of PRS<sub>std-adj</sub> were well aligned when examined by genotyping batch, decile of age, and sex (Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig5">1</a>–<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig7">3</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="PRS distributions by reported race before and after adjustment for population structure."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: PRS distributions by reported race before and after adjustment for population structure.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01767-6/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="508"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p>Plots to the left of each arrow show the distributions of unadjusted published PRS (PRS<sub>std-raw</sub>) by race for each of six diseases in up to 36,423 MGBB participants. Plots to the right of each arrow show these distributions after adjustment for population structure (PRS<sub>std-adj</sub>), as described in the text. The red vertical line indicates the standardized PRS threshold corresponding to OR &gt; 2 for each disease, based on the OR per standard deviation from the original publication.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01767-6/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec6">Replication of PRS–disease association</h4><p>As shown in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig4">4</a>, quantile of PRS<sub>std-adj</sub> was highly correlated with log(odds) of disease across the six phenotypes in the MGBB, with correlation coefficients ranging from 0.68 for CRCa to 0.95 for T2D. Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig8">4</a>–<a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig11">7</a> show the correlation of PRS<sub>std-adj</sub> quantile and log(odds) of disease in the reported racial groups separately. Our analyses also replicated the published PRS thresholds corresponding to OR &gt; 2. As shown in Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/s41591-022-01767-6#Tab1">1</a>, at the published PRS<sub>std-adj</sub> thresholds we observed OR ranging from 1.75 (95% CI: 1.57–1.95) for T2D to 2.38 (95% CI: 2.07–2.73) for BrCa in MGBB participants overall. Except for T2D, the 95% confidence interval of the replicated OR for all diseases either included or, in the case of BrCa and AFib, exceeded a point estimate of 2. Results were consistent in analyses restricted to white participants but were variable in other groups, largely because of the small number of disease cases in certain racial subgroups. In 22 of 24 analyses stratified by reported race, subjects with PRS<sub>std-adj</sub> above the published OR &gt; 2 thresholds had higher odds of disease than those below these thresholds. In the MGBB overall, the prevalence of a high-risk PRS<sub>std-adj</sub> ranged from 5.4% for CRCa to 13.2% for PrCa (in men). Figure <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig2">2b</a> illustrates the number of participants with PRS<sub>std-adj</sub> above the published OR &gt; 2 threshold for one or more of the target diseases. Of note, similar to the disease co-occurrences observed in MGBB participants, the most common co-occurrences of high-risk PRS<sub>std-adj</sub> were the combinations of CAD and T2D (<i>n</i> = 333) and CAD and AFib (<i>n</i> = 211).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Correlation between adjusted PRS and odds of disease."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Correlation between adjusted PRS and odds of disease.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01767-6/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="482"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p>The plots show log(odds) of each of six diseases versus quantile (<i>n</i> = 50) of standardized population structure-adjusted PRS (PRS<sub>std-adj</sub>) in up to 36,423 MGBB participants.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01767-6/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-1"><figure><figcaption class="c-article-table__figcaption"><b id="Tab1" data-test="table-caption">Table 1 Prevalence and disease associations of high-risk PRS for six diseases in MGBB overall and by reported race</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/s41591-022-01767-6/tables/1" aria-label="Full size table 1"><span>Full size table</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec7">Prospective PRS assay</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec8">Sensitivity and specificity of array and imputation</h4><p>The replication results above supported the development of a genotype array-based clinical assay for PRS and secondary findings from the American College of Medical Genetics and Genomics v2.0 list (ACMG SF v2.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Kalia, S. S. et al. Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics. Genet. Med. 19, 249–255 (2017)." href="/articles/s41591-022-01767-6#ref-CR24" id="ref-link-section-d73985140e1825">24</a></sup>. To determine the performance of the arrays used in the prospective assay and of the imputation pipeline, we used three reference Genome In A Bottle (GIAB) samples (NA12878, NA24385 and NA24631, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">5</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Zook, J. M. et al. Extensive sequencing of seven human genomes to characterize benchmark reference materials. Sci. Data 3, 160025 (2016)." href="/articles/s41591-022-01767-6#ref-CR25" id="ref-link-section-d73985140e1832">25</a></sup>. Sensitivity and positive predictive value (PPV) for single-nucleotide variants (SNV) were &gt; 99.7% on average, with lower performance for indels (sensitivity, 96.3%; PPV, 97.8%). Of note, although sensitivity in the ACMG SF v2.0 regions was high (96.2%), PPV was low (63.6%) due to these regions having an excess of poorly performing rare variants<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Bowling, K. M. et al. Identifying rare, medically relevant variation via population-based genomic screening in Alabama: opportunities and pitfalls. Genet. Med. 23, 280–288 (2021)." href="/articles/s41591-022-01767-6#ref-CR26" id="ref-link-section-d73985140e1836">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Weedon, M. N. &amp; Wright, C. F. et al. Use of SNP chips to detect rare pathogenic variants: retrospective, population based diagnostic evaluation. BMJ 372, n214 (2021)." href="/articles/s41591-022-01767-6#ref-CR27" id="ref-link-section-d73985140e1839">27</a></sup>.</p><p>As expected, sensitivity and PPV decreased for imputed data, especially for indels (SNV sensitivity, 98.0%; SNV PPV, 97.5%; indel sensitivity, 92.8%; indel PPV, 90.7%) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">5</a>). NA12878 was not evaluated for imputation accuracy because it is present in the imputation reference dataset from the 1000 Genomes Project and has artificially high imputation accuracy. To further evaluate imputation accuracy, we compared genome sequencing data to array data for 22 diverse samples. Analytical performance was lower in this dataset than in the GIAB high-confidence data (~3% reduction in performance for sensitivity and PPV, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">6</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec9">Performance of prospective PRS assay</h4><p>For the GIAB samples, PRS<sub>std-adj</sub> was robust across different array versions and consistent with results from whole genome sequence (WGS) data; all three GIAB samples were below the high-risk threshold (OR &gt; 2) for all diseases in all methods (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">7</a>). In evaluating the 22 samples with WGS and prospective array data, PRS<sub>std-adj</sub> scores were similarly concordant, particularly for AFib, CAD and T2D (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig12">8</a>). Additionally, 108/110 high-risk status classifications were concordant in this dataset (98.2% agreement; Matthews correlation coefficient, 0.84; <i>P</i> &lt; 0.001), with the two discordant values (one in CAD and one in CRCa) being very close to the high-risk threshold (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">8</a>). Finally, we compared nine individuals with high-risk PRS for 10 diseases identified in the MGBB genotyping data to their PRS risk status using the prospective assay (one individual at high risk for AFib, one individual at high risk for BrCa, three individuals at high risk for CAD, three individuals at high risk for CRCa, one individual at high risk for PrCa and one individual at high risk for T2D). All PRS categories were consistent across the two different arrays used for MGBB genotyping and for the clinical assay (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">9</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec10">Clinical PRS report</h4><p>We then developed a PRS laboratory report consistent in format and content with other clinical genetic test reports (Supplementary File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">1</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Scheuner, M. T., Edelen, M. O., Hilborne, L. H. &amp; Lubin, I. M. Effective communication of molecular genetic test results to primary care providers. Genet. Med. 15, 444–449 (2013)." href="#ref-CR28" id="ref-link-section-d73985140e1891">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="McLaughlin, H. M. et al. A systematic approach to the reporting of medically relevant findings from whole genome sequencing. BMC Med. Genet. 15, 134 (2014)." href="#ref-CR29" id="ref-link-section-d73985140e1891_1">29</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Farmer, G. D., Gray, H., Chandratillake, G., Raymond, F. L. &amp; Freeman, A. L. J. Recommendations for designing genetic test reports to be understood by patients and non-specialists. Eur. J. Hum. Genet. 28, 885–895 (2020)." href="/articles/s41591-022-01767-6#ref-CR30" id="ref-link-section-d73985140e1894">30</a></sup>. That is, it includes a description of the test performed and a prominently displayed summary of important findings and their interpretations. Subsequent sections of the report give more detail about the results, including, for each disease, general population prevalence and a brief summary of the GWAS from which the PRS was derived. Sections on methodology and literature references are at the end of the report. The report also reflects several choices made during its development. A graphic highlights in red the disease(s) for which the patient has increased polygenic disease risk, as defined by a PRS corresponding to a published OR &gt; 2 for disease, mirroring both a common threshold from Mendelian genetics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Senol-Cosar, O. et al. Considerations for clinical curation, classification, and reporting of low-penetrance and low effect size variants associated with disease risk. Genet. Med. 21, 2765–2773 (2019)." href="/articles/s41591-022-01767-6#ref-CR31" id="ref-link-section-d73985140e1898">31</a></sup> and the effect sizes for disease risk factors already considered in current clinical care<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Goff, D. C. et al. 2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American College of Cardiology/American Heart Association Task Force on practice guidelines. Circulation 129(Suppl. 2), S49–S73 (2014)." href="#ref-CR32" id="ref-link-section-d73985140e1902">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="American Diabetes Association. 2. Classification and diagnosis of diabetes: standards of medical care in diabetes – 2021. Diabetes Care 44(Suppl. 1), S15–S33 (2021)." href="#ref-CR33" id="ref-link-section-d73985140e1902_1">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Grossman, D. C. et al. Screening for prostate cancer: US Preventive Services Task Force recommendation statement. JAMA 319, 1901–1913 (2018)." href="#ref-CR34" id="ref-link-section-d73985140e1902_2">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Siu, A. L., US Preventive Services Task Force Screening for breast cancer: US Preventive Services Task Force recommendation statement. Ann. Intern. Med. 164, 279–296 (2016)." href="#ref-CR35" id="ref-link-section-d73985140e1902_3">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Davidson, K. W. et al., US Preventive Services Task Force Screening for colorectal cancer: US Preventive Services Task Force recommendation statement. JAMA 325, 1965–1977 (2021)." href="/articles/s41591-022-01767-6#ref-CR36" id="ref-link-section-d73985140e1905">36</a></sup>. Any PRS not categorized as high risk is described as conferring average risk. Monogenic disease variants and PRS results are reported separately, without comment on any possible interaction between a monogenic result and a relevant PRS (for example, an average-risk BrCa PRS and a pathogenic variant in <i>BRCA1</i> associated with hereditary breast and ovarian cancer). Illustrating the boundary where the role of the clinical laboratory ends (phase 2 of Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig1">1</a>) and the role of the treating physician begins (phase 3), the laboratory report does not include information about absolute disease risk or the role of other, non-genetic factors in disease risk, and it is not directive in its recommendations for clinical management of high-risk results.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec11">Clinical processes and supportive materials</h4><p>In recognition that physicians and patients require additional guidance in contextualizing high-risk PRS results, the GenoVA Study has developed processes and materials to support the clinical use of PRS. A genetic counselor contacts each patient with a high-risk PRS result or monogenic disease variant to discuss the result’s health significance and offer guidance for a conversation to have with their physician. All patients and their primary care physicians receive a copy of the laboratory report, and each patient with at least one high-risk PRS result is additionally given patient-oriented educational materials about the relevant disease(s) (Supplementary File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">2</a>). The patient’s primary care physician also receives a copy of physician-oriented educational materials to support their decision-making about PRS (Supplementary File <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">3</a>). Given the current state of the evidence, the physician materials note that professional guidelines do not recommend specific changes to general screening or prevention recommendations based on PRS results, but these materials are updated over the course of the study as evidence accrues to support distinct recommendations.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec12">Results from the first 227 prospective samples</h4><p>As of 21 October 2021, 227 GenoVA trial participants have been assayed using the prospective PRS pipeline from two primary sample types (130 blood, 97 saliva). Of these, 108 participants (48%) self-report as white race and non-Hispanic/Latinx ethnicity, and 78 (34%) identify as women. In this preliminary sample of trial enrollees, the proportions of participants whose PRS are above the study threshold for high risk are consistent with those observed in the MGBB, ranging from 5.7% for CRCa to 15.3% for PrCa (Table <a data-track="click" data-track-label="link" data-track-action="table anchor" href="/articles/s41591-022-01767-6#Tab2">2</a>). Two actionable ACMG SF v2.0 variants have been identified and confirmed in the first 227 enrollees (<i>BRCA1</i>:<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_007294">NM_007294</a> c.2748delT (p.Asn916LysfsX84), likely pathogenic; <i>BRCA2</i>:<a href="https://www.ncbi.nlm.nih.gov/nuccore/NM_000059">NM_000059</a> c.3545_3546delTT (p.Phe1182X), pathogenic). The reporting of these results to trial participants and their physicians is underway. The study will determine whether PRS implementation affects clinical management and enables the detection of undiagnosed prevalent cases and incident cases during the observation period.</p><div class="c-article-table" data-test="inline-table" data-container-section="table" id="table-2"><figure><figcaption class="c-article-table__figcaption"><b id="Tab2" data-test="table-caption">Table 2 Summary of PRS results from the first six batches of clinical samples in the GenoVA Study</b></figcaption><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="table-link" data-track="click" data-track-action="view table" data-track-label="button" rel="nofollow" href="/articles/s41591-022-01767-6/tables/2" aria-label="Full size table 2"><span>Full size table</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec13-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec13">Discussion</h2><div class="c-article-section__content" id="Sec13-content"><p>Bridging two significant gaps between PRS development and clinical implementation, we developed a clinical genotyping array-based assay for six PRS and a process to report the results to patients and primary care physicians. The PRS were robust across multiple genotyping arrays and imputation pipelines. The distributions of unadjusted PRS varied by reported race in a large biobank, impeding clinical validation, but adjustment for population structure enabled the replication of published PRS–disease associations. These results supported the development of a population structure-adjusted pipeline for PRS calculation and reporting for individual patients, now implemented in a clinical trial of PRS testing along with patient and physician educational materials and genetic counseling support.</p><p>The development and implementation of our PRS assay and report illustrate key choices that laboratories must make in what we term phase 2 of the PRS implementation pathway. First, for each target disease, we had to choose the specific PRS to implement among multiple publicly available options (that is, PRS developed and validated by others in phase 1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Lambert, S. A. et al. The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation. Nat. Genet. 53, 420–425 (2021)." href="/articles/s41591-022-01767-6#ref-CR21" id="ref-link-section-d73985140e2190">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Fritsche, L. G. et al. Cancer PRSweb: an online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks. Am. J. Hum. Genet. 107, 815–836 (2020)." href="/articles/s41591-022-01767-6#ref-CR22" id="ref-link-section-d73985140e2193">22</a></sup>. Considerations include the performance of the PRS in both the published discovery and replication cohorts in addition to the population that the laboratory is interested in targeting. Guidelines are emerging on what defines high-quality PRS reporting<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Wand, H. et al. Improving reporting standards for polygenic scores in risk prediction studies. Nature 591, 211–219 (2021)." href="/articles/s41591-022-01767-6#ref-CR37" id="ref-link-section-d73985140e2197">37</a></sup>, and this improved transparency should help laboratories to select appropriate PRS from the many available. Second, we chose to use a genotype array-based approach instead of genome sequencing. Like genotyping, low-coverage genome sequencing technology is also relatively low cost<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Homburger, J. R. et al. Low coverage whole genome sequencing enables accurate assessment of common variants and calculation of genome-wide polygenic scores. Genome Med. 11, 74 (2019)." href="/articles/s41591-022-01767-6#ref-CR38" id="ref-link-section-d73985140e2201">38</a></sup>. We chose the Illumina GDA because its widespread use in the All of Us Research Program<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Denny, J. C. et al. The ‘All of Us’ Research Program. N. Engl. J. Med. 381, 668–676 (2019)." href="/articles/s41591-022-01767-6#ref-CR39" id="ref-link-section-d73985140e2205">39</a></sup>, eMERGE Consortium<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="National Human Genome Research Institute (NHGRI). Electronic Medical Records and Genomics (eMERGE) Network 
                  https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE
                  
                 (2020)." href="/articles/s41591-022-01767-6#ref-CR15" id="ref-link-section-d73985140e2209">15</a></sup> and other projects optimizes the likelihood that it will be a well-supported genotyping platform for future improvements, and enhances the generalizability of our methods to other institutions looking to implement clinical PRS testing. Third, although published methods can adjust for population structure in large cohorts of people<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Khera, A. V. et al. Whole-genome sequencing to characterize monogenic and polygenic contributions in patients hospitalized with early-onset myocardial infarction. Circulation 139, 1593–1602 (2019)." href="/articles/s41591-022-01767-6#ref-CR40" id="ref-link-section-d73985140e2214">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Dikilitas, O. et al. Predictive utility of polygenic risk scores for coronary heart disease in three major racial and ethnic groups. Am. J. Hum. Genet. 106, 707–716 (2020)." href="/articles/s41591-022-01767-6#ref-CR41" id="ref-link-section-d73985140e2217">41</a></sup>, these methods are not immediately applicable for correcting a PRS for a prospectively genotyped individual patient, whose sample is at best part of a small clinically analyzed batch with insufficient data for robust population structure adjustment. Correction thus requires additional decisions about how to adjust for population structure and which reference to use. We chose to impute data against 1000 Genomes Project phase 3 data and to project each new individual patient sample onto the principal components from the MGBB. Other laboratories may choose to impute against the larger TOPMed (Trans-Omics for Precision Medicine) population<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Kowalski, M. H. et al. Use of >100,000 NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium whole genome sequences improves imputation quality and detection of rare variant associations in admixed African and Hispanic/Latino populations. PLoS Genet. 15, e1008500 (2019)." href="/articles/s41591-022-01767-6#ref-CR42" id="ref-link-section-d73985140e2221">42</a></sup>, although issues of genome build discrepancy and regulatory prohibition against sending patient data to external research servers are limitations. Fourth, once a platform is selected, a clinical laboratory must determine the benchmarks that define an analytically valid PRS assay. We chose to verify the PRS performance in our laboratory to determine the appropriate parameters for our assay; calculate the analytical performance of the genotyping array and imputation pipeline using both well-characterized reference samples and individual level genome data; and calculate the robustness and performance of the PRS using genome data and multiple array platforms from both reference and individual samples. This multi-step approach helped ensure the accuracy of the data going into the PRS as well as the final performance of the PRS itself.</p><p>We also made numerous choices in how to report PRS results and interpretations to patients and physicians. We decided to report a dichotomous PRS interpretation (that is, high risk versus average risk) instead of a continuous result (for example, percentile rank, relative risk or absolute risk). We have previously described the trade-offs of these approaches, including the need for actionability thresholds; transparency about the limitations of PRS, particularly in underrepresented populations; and the absence of validated predictions models that incorporate both PRS and other clinical risk factors<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Lewis, A. C. F., Green, R. C. &amp; Vassy, J. L. Polygenic risk scores in the clinic: translating risk into action. HGG Adv. 2, 100047 (2021)." href="/articles/s41591-022-01767-6#ref-CR43" id="ref-link-section-d73985140e2228">43</a></sup>. For the GenoVA Study we favored a dichotomous result to indicate a possible clinical action threshold to the treating physician. We chose OR &gt; 2 to define high polygenic risk, consistent with effect sizes of traditional risk factors considered for the target diseases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Goff, D. C. et al. 2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American College of Cardiology/American Heart Association Task Force on practice guidelines. Circulation 129(Suppl. 2), S49–S73 (2014)." href="#ref-CR32" id="ref-link-section-d73985140e2232">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="American Diabetes Association. 2. Classification and diagnosis of diabetes: standards of medical care in diabetes – 2021. Diabetes Care 44(Suppl. 1), S15–S33 (2021)." href="#ref-CR33" id="ref-link-section-d73985140e2232_1">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Grossman, D. C. et al. Screening for prostate cancer: US Preventive Services Task Force recommendation statement. JAMA 319, 1901–1913 (2018)." href="#ref-CR34" id="ref-link-section-d73985140e2232_2">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Siu, A. L., US Preventive Services Task Force Screening for breast cancer: US Preventive Services Task Force recommendation statement. Ann. Intern. Med. 164, 279–296 (2016)." href="#ref-CR35" id="ref-link-section-d73985140e2232_3">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Davidson, K. W. et al., US Preventive Services Task Force Screening for colorectal cancer: US Preventive Services Task Force recommendation statement. JAMA 325, 1965–1977 (2021)." href="/articles/s41591-022-01767-6#ref-CR36" id="ref-link-section-d73985140e2235">36</a></sup>. Another laboratory may use the methods we describe to produce measures of continuous risk or of categorical risk at different thresholds thought to be clinically meaningful, which will probably vary among the diseases for which they choose to implement PRS. Estimating absolute disease risk (for example, with the BOADICEA model for breast cancer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Lee, A. et al. BOADICEA: a comprehensive breast cancer risk prediction model incorporating genetic and nongenetic risk factors. Genet. Med. 21, 1708–1718 (2019)." href="/articles/s41591-022-01767-6#ref-CR44" id="ref-link-section-d73985140e2239">44</a></sup> or the Pooled Cohort Equations for atherosclerotic cardiovascular disease<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Stone, N. J. et al. 2013 ACC/AHA guideline on the treatment of blood cholesterol to reduce atherosclerotic cardiovascular risk in adults: a report of the American College of Cardiology/American Heart Association Task Force on Practice Guidelines. Circulation 129(Suppl. 2), S1–S45 (2014)." href="/articles/s41591-022-01767-6#ref-CR45" id="ref-link-section-d73985140e2243">45</a></sup>) may be considered the gold standard for risk stratification, but validated absolute risk models are not available for most diseases and require patient information (for example, mammographic breast density and blood pressure) that is often unavailable to the interpreting laboratory. Drawing on other examples from primary care, we chose not to include directive clinical recommendations on the PRS laboratory report itself, instead assigning such activities to phase 3 of the PRS implementation pathway, supported by informational materials and genetic counseling support. We note that, for example, although a laboratory reports the results of a patient’s low-density lipoprotein cholesterol and reference range for the assay, it is the treating physician who contextualizes that result with the patient’s other characteristics to decide whether to offer cholesterol-lowering therapy.</p><p>The question of how to support physician management of PRS results without under- or overselling the potential benefits of PRS is controversial, given the lack of prospective data showing that the clinical use of PRS improves patient outcomes. In this early era of PRS implementation, the most prudent course of action is probably to develop educational and consultant resources, such as those used in the GenoVA Study, to present transparently the evidence for and limitations of PRS interpretations without being overly prescriptive in their recommendations. Given the participant age range and choice of diseases in the GenoVA Study, we anticipate that most physician actions will fall within already clinically acceptable practices (for example, more frequent hemoglobin A1c screening for T2D or favoring colonoscopy screening over fecal immunochemical testing for CRCa screening). Stronger evidence of benefit will be needed to justify actions that deviate more significantly from accepted practice, such as screening starting at much younger ages or requiring more invasive or expensive procedures. As they do in all areas of medicine, physicians will need to use available evidence and clinical judgment to make the best decisions with their patients. The GenoVA Study is collecting data on what physicians do with PRS results and their preferences for how they can be supported in this decision-making.</p><p>Although other laboratories are developing PRS assays in both clinical and research settings and have reported the aggregate performance of these PRS in a population, including biobanks or customers of direct-to-consumer companies<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="National Human Genome Research Institute (NHGRI). Electronic Medical Records and Genomics (eMERGE) Network 
                  https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE
                  
                 (2020)." href="/articles/s41591-022-01767-6#ref-CR15" id="ref-link-section-d73985140e2254">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Homburger, J. R. et al. Low coverage whole genome sequencing enables accurate assessment of common variants and calculation of genome-wide polygenic scores. Genome Med. 11, 74 (2019)." href="/articles/s41591-022-01767-6#ref-CR38" id="ref-link-section-d73985140e2257">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hughes, E. et al. Development and validation of a clinical polygenic risk score to predict breast cancer risk. JCO Precis. Oncol. 4, 585–592 (2020)." href="#ref-CR46" id="ref-link-section-d73985140e2260">46</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Our Health + Ancestry DNA Service - 23andMe (23andMe); 
                  https://www.23andme.com/dna-health-ancestry/
                  
                " href="#ref-CR47" id="ref-link-section-d73985140e2260_1">47</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Chen, S.-F. et al. Genotype imputation and variability in polygenic risk score estimation. Genome Med. 12, 100 (2020)." href="/articles/s41591-022-01767-6#ref-CR48" id="ref-link-section-d73985140e2263">48</a></sup>, none has described the development and validation of a clinical, population structure-adjusted assay for prospectively tested individuals. Although the eMERGE consortium and other studies are actively developing trans-ancestry PRS for a number of common diseases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="National Human Genome Research Institute (NHGRI). Electronic Medical Records and Genomics (eMERGE) Network 
                  https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE
                  
                 (2020)." href="/articles/s41591-022-01767-6#ref-CR15" id="ref-link-section-d73985140e2267">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="National Human Genome Research Institute. Polygenic RIsk MEthods in Diverse populations (PRIMED) Consortium 
                  https://www.genome.gov/Funded-Programs-Projects/PRIMED-Consortium
                  
                " href="/articles/s41591-022-01767-6#ref-CR49" id="ref-link-section-d73985140e2270">49</a></sup>, we report, here, a single clinical assay for population structure-adjusted PRS for multiple diseases. And while other laboratories may make different decisions about the number of disease PRS they choose to implement, whether and how to compare the performance of multiple available PRS for each disease, and the format of the clinical PRS report, our work provides a framework for how a laboratory can clinically validate and implement a prospective PRS suitable for an individual patient.</p><p>Much has been written about the reduced validity of most PRS in populations of non-European ancestry, due to their use of non-causal loci and effect sizes from GWAS in predominantly European discovery cohorts<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. Nat. Genet. 51, 584–591 (2019)." href="/articles/s41591-022-01767-6#ref-CR19" id="ref-link-section-d73985140e2277">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Lewis, C. M. &amp; Vassos, E. Polygenic risk scores: from research tools to clinical instruments. Genome Med. 12, 44 (2020)." href="/articles/s41591-022-01767-6#ref-CR20" id="ref-link-section-d73985140e2280">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Manolio, T. A. Using the data we have: improving diversity in genomic research. Am. J. Hum. Genet. 105, 233–236 (2019)." href="/articles/s41591-022-01767-6#ref-CR50" id="ref-link-section-d73985140e2283">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Lewis, A. C. F. &amp; Green, R. C. Polygenic risk scores in the clinic: new perspectives needed on familiar ethical issues. Genome Med. 13, 14 (2021)." href="/articles/s41591-022-01767-6#ref-CR51" id="ref-link-section-d73985140e2286">51</a></sup>. As we await larger datasets from more diverse populations and the methodological advances that will improve the performance of trans-ancestry PRS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Conti, D. V. et al. Trans-ancestry genome-wide association meta-analysis of prostate cancer identifies new susceptibility loci and informs genetic risk prediction. Nat. Genet. 53, 65–75 (2021)." href="/articles/s41591-022-01767-6#ref-CR10" id="ref-link-section-d73985140e2290">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="National Human Genome Research Institute (NHGRI). Electronic Medical Records and Genomics (eMERGE) Network 
                  https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE
                  
                 (2020)." href="/articles/s41591-022-01767-6#ref-CR15" id="ref-link-section-d73985140e2293">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="National Human Genome Research Institute. Polygenic RIsk MEthods in Diverse populations (PRIMED) Consortium 
                  https://www.genome.gov/Funded-Programs-Projects/PRIMED-Consortium
                  
                " href="/articles/s41591-022-01767-6#ref-CR49" id="ref-link-section-d73985140e2296">49</a></sup>, a clinical laboratory looking to develop a PRS assay for a given disease has the following options: (1) postpone implementation, as done by some commercial laboratories;<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Ray, T. Myriad Genetics recalibrates breast cancer PRS for all ancestries in anticipation of broader launch. Genomeweb 
                  https://www.genomeweb.com/molecular-diagnostics/myriad-genetics-recalibrates-breast-cancer-prs-all-ancestries-anticipation
                  
                 (2021)." href="/articles/s41591-022-01767-6#ref-CR52" id="ref-link-section-d73985140e2300">52</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Ambry Product Team. Important discontinuation notice: AmbryScore: polygenic risk scores (PRS) 
                  https://info.ambrygenetics.com/take-a-brief-survey-for-entry-into-amazon-gift-card-drawing
                  
                " href="/articles/s41591-022-01767-6#ref-CR53" id="ref-link-section-d73985140e2303">53</a></sup> (2) implement separate ancestry-specific published PRS only in those ancestral groups from which they were derived and validated; or (3) implement a single PRS that aims for applicability across ancestry groups and report transparently any applicable limitations in the underlying evidence and its interpretation for specific individuals or ancestral groups. Because the second option requires the assignment of an individual patient to a specific ancestry group, either before or during PRS analysis, and, problematically, risks the inequitable provision of PRS to some populations but not to others, we chose the third option for the GenoVA Study and implemented a single method of adjustment for population structure. After doing so, we observed that the chosen PRS threshold corresponding to OR &gt; 2 generally identified subjects at higher risk of disease across reported race in the MGBB replication cohort. The magnitude and precision of this effect did vary by reported race, probably due to two factors: small numbers of MGBB cases for certain diseases in certain racial groups; and real differences in the ability of these PRS to correlate with disease risk in non-European ancestry groups, as has been observed even in well-developed trans-ancestry PRS<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 10" title="Conti, D. V. et al. Trans-ancestry genome-wide association meta-analysis of prostate cancer identifies new susceptibility loci and informs genetic risk prediction. Nat. Genet. 53, 65–75 (2021)." href="/articles/s41591-022-01767-6#ref-CR10" id="ref-link-section-d73985140e2307">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Ge, T. et al. Validation of a trans-ancestry polygenic risk score for type 2 diabetes in diverse populations. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.09.11.21263413
                  
                 (2021)." href="/articles/s41591-022-01767-6#ref-CR54" id="ref-link-section-d73985140e2310">54</a></sup>. Methodological advances that leverage local ancestry or GWAS summary statistics from multiple diverse populations will improve the performance of PRS across ancestry groups<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Marnetto, D. et al. Ancestry deconvolution and partial polygenic score can improve susceptibility predictions in recently admixed individuals. Nat. Commun. 11, 1628 (2020)." href="/articles/s41591-022-01767-6#ref-CR55" id="ref-link-section-d73985140e2314">55</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 56" title="Ruan, Y. et al. Improving polygenic prediction in ancestrally diverse populations. Preprint at medRxiv 
                  https://doi.org/10.1101/2020.12.27.20248738
                  
                 (2021)." href="/articles/s41591-022-01767-6#ref-CR56" id="ref-link-section-d73985140e2317">56</a></sup>. In the meantime, we have developed a clinically validated PRS assay, the application of which in diverse ancestry groups is defensible but the results of which, nonetheless, have limitations. These limitations are clearly presented in a clinical laboratory report (phase 2), which can then be contextualized by the physician for each individual (phase 3). Applying population-level data to individual patient care represents both the science and art of medical practice, particularly when the individual patient is not well represented in the available data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 57" title="Armstrong, K. A. &amp; Metlay, J. P. Annals clinical decision making: translating population evidence to individual patients. Ann. Intern. Med. 172, 610–616 (2020)." href="/articles/s41591-022-01767-6#ref-CR57" id="ref-link-section-d73985140e2322">57</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 58" title="Sniderman, A. D., LaChapelle, K. J., Rachon, N. A. &amp; Furberg, C. D. The necessity for clinical reasoning in the era of evidence-based medicine. Mayo Clin. Proc. 88, 1108–1114 (2013)." href="/articles/s41591-022-01767-6#ref-CR58" id="ref-link-section-d73985140e2325">58</a></sup>.</p><p>In conclusion, data from increasingly larger and more diverse populations, coupled with computational advances, are propelling PRS into consideration for clinical implementation. We have shown that laboratory assay development and PRS reporting to patients and physicians are feasible (but non-trivial) next phases in PRS implementation. As the performance of PRS continues to improve, particularly for individuals of underrepresented ancestry groups, the implementation processes we describe can serve as generalizable models for laboratories and health systems looking to realize the potential of PRS for improved patient health.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec14-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec14">Methods</h2><div class="c-article-section__content" id="Sec14-content"><h3 class="c-article__sub-heading" id="Sec15">Selection of PRS for implementation</h3><p>We identified large GWAS for the six target diseases for which the summary statistics (base files with alleles and weights) were freely available from the Polygenic Score (PGS) Catalog<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 21" title="Lambert, S. A. et al. The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation. Nat. Genet. 53, 420–425 (2021)." href="/articles/s41591-022-01767-6#ref-CR21" id="ref-link-section-d73985140e2344">21</a></sup> (AFib, CAD, T2D, BrCa) or the Cancer PRSWeb (CRCa, PrCa)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Fritsche, L. G. et al. Cancer PRSweb: an online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks. Am. J. Hum. Genet. 107, 815–836 (2020)." href="/articles/s41591-022-01767-6#ref-CR22" id="ref-link-section-d73985140e2348">22</a></sup> as of 26 December 2019. For the three cardiometabolic diseases (AFib, CAD and T2D) we chose the PRS derived from the UK Biobank in Khera et al. 2018 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Khera, A. V. et al. Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations. Nat. Genet. 50, 1219–1224 (2018)." href="/articles/s41591-022-01767-6#ref-CR7" id="ref-link-section-d73985140e2352">7</a></sup>): for AFib, the PGS Catalog Publication (PGP) ID is PGP000006 and the PGS ID is PGS000016; for CAD, the PGP ID is PGP000006 and the PGS ID is PGS000013; and for T2D the PGP ID is PGP000006 and the PGS ID is PGS000014. For the three cancers we chose PRS derived from the largest published GWAS at the time: for BrCa we used Michailidou et al. 2017 and Mavaddat et al. 2019 (PGS ID = PGS000007, PGP ID = PGP000002) (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 59" title="Michailidou, K. et al. Association analysis identifies 65 new breast cancer risk loci. Nature 551, 92–94 (2017)." href="/articles/s41591-022-01767-6#ref-CR59" id="ref-link-section-d73985140e2356">59</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 60" title="Mavaddat, N. et al. Polygenic risk scores for prediction of breast cancer and breast cancer subtypes. Am. J. Hum. Genet. 104, 21–34 (2019)." href="/articles/s41591-022-01767-6#ref-CR60" id="ref-link-section-d73985140e2359">60</a></sup>); for CRCa we used Huyghe et al. 2019 (PRSWEB_PHECODE153_CRC-Huyghe_PT_MGI_20191112, PRS tuning parameter: 3.98107170553497e-07) (ref.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 23" title="Huyghe, J. R. et al. Discovery of common and rare genetic risk variants for colorectal cancer. Nat. Genet. 51, 76–87 (2019)." href="/articles/s41591-022-01767-6#ref-CR23" id="ref-link-section-d73985140e2363">23</a></sup>); and for PrCa we used Schumacher et al. 2018 (PRSWEB_PHECODE185_Pca- PRACTICAL_LASSOSUM_MGI_20191112, PRS tuning parameter: s0.5_Lambda0.00695192796177561) (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 61" title="Schumacher, F. R. et al. Association analyses of more than 140,000 men identify 63 new prostate cancer susceptibility loci. Nat. Genet. 50, 928–936 (2018)." href="/articles/s41591-022-01767-6#ref-CR61" id="ref-link-section-d73985140e2368">61</a></sup>).</p><h3 class="c-article__sub-heading" id="Sec16">Replication of published PRS</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec17">Population and sample</h4><p>Given that the GenoVA Study is enrolling participants from eastern Massachusetts, USA, we used data from the Mass General Brigham (formerly Partners Healthcare) Biobank (MGBB)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 62" title="Karlson, E. W., Boutin, N. T., Hoffnagle, A. G. &amp; Allen, N. L. Building the Partners Healthcare Biobank at Partners Personalized Medicine: informed consent, return of research results, recruitment lessons and operational considerations. J. Pers. Med. 6, 2 (2016)." href="/articles/s41591-022-01767-6#ref-CR62" id="ref-link-section-d73985140e2384">62</a></sup> to evaluate the performance of the selected PRS in a similar population and workflow for our study and assay. MGBB participants were not included in the published derivation and validation studies for the PRS used. In brief, MGBB was launched in 2010 with the initial goal of collecting DNA, plasma, and serum samples from 75,000 patients from Brigham and Women’s Hospital, Massachusetts General Hospital, and other MGBB-affiliated healthcare facilities, and obtaining patient consent for the linkage between biospecimen data, medical record data and survey data. We use the terms ‘race’ and ‘ethnicity’ to refer to social constructs often used in healthcare operations and biomedical research to evaluate and address disparities between populations. Racial categories of participants in the MGBB (for example, white or Asian) are derived from electronic health record (EHR) data. For the present analysis we collapsed reported race in MGBB into four categories: Asian, Black, white, and other/unknown. Race and ethnicity of GenoVA Study participants were collected through EHR data and self-report and categorized using the five racial categories (American Indian or Alaska Native, Asian, Black or African American, Native Hawaiian or Other Pacific Islander, and white) and two ethnic categories (Hispanic/Latinx and Not Hispanic/Latinx) required by US federal data collection standards. We use the term ‘ancestry’ to describe the genetic construct describing inheritance of variants from global ancestral populations.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec18">Disease phenotyping</h4><p>We used validated computed phenotypes from MGBB to define case and control status for each of the six diseases (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">1</a>). Validated MGB phenotypes are available for CAD (PPV = 95%), AFib (PPV = 94%), T2D (PPV = 95%), and colorectal (PPV = 100%), breast (PPV = 95%) and prostate cancer (PPV = 100%)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yu, S. et al. Toward high-throughput phenotyping: unbiased automated feature extraction and selection from knowledge sources. J. Am. Med. Inform. Assoc. 22, 993–1000 (2015)." href="#ref-CR63" id="ref-link-section-d73985140e2399">63</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Yu, S. et al. Enabling phenotypic big data with PheNorm. J. Am. Med. Inform. Assoc. 25, 54–60 (2018)." href="#ref-CR64" id="ref-link-section-d73985140e2399_1">64</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 65" title="Gainer, V. S. et al. The Biobank portal for Partners Personalized Medicine: a query tool for working with consented Biobank samples, genotypes, and phenotypes using i2b2. J. Pers. Med. 6, 11 (2016)." href="/articles/s41591-022-01767-6#ref-CR65" id="ref-link-section-d73985140e2402">65</a></sup>. For each disease, ‘caseness’ was defined as prevalent disease on 16 December 2019. For subgroup analyses, participant age was determined on 16 December 2019 or at death, if earlier. Only women and men were assigned case or control status for breast and prostate cancer, respectively.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec19">Genotyping and imputation</h4><p>We used genotype data from the 36,423 MGBB participants with available genotyping data as of 16 December 2019. Genotyping was performed using standard processing described previously on one of three Illumina Infinium genotyping arrays: (1) a pre-release version developed by the Multi-Ethnic Genotyping Array Consortium (Multi-Ethnic Genotyping Array (MEGA), <i>n</i> = 4,924); (2) an expanded version of this pre-commercial array (Expanded Multi-Ethnic Genotyping Array (MEGAEX), <i>n</i> = 5,345); and (3) the final commercial version (Multi-Ethnic Global (MEG), <i>n</i> = 26,157). The MEGA, MEGAEX and MEG arrays consisted of 1.39, 1.74 and 1.78 million probes, respectively<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 66" title="Blau, A., Brown, A., Mahanta, L. &amp; Amr, S. S. The translational genomics core at Partners Personalized Medicine: facilitating the transition of research towards personalized medicine. J. Pers. Med. 6, 10 (2016)." href="/articles/s41591-022-01767-6#ref-CR66" id="ref-link-section-d73985140e2423">66</a></sup>. For MEGA and MEGAEX data, only probes found in the commercial version of the array (MEG) were used in the present analysis. Quality control for the genotyping requires samples to have at least a 99% call rate and concordant sex between the EHR and what is computed from the array data. We used existing MGBB imputed data generated by batching sets of ~5,000 participants and imputing against the 1000 Genomes Project phase 3 data using the Michigan Imputation Server<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Das, S. et al. Next-generation genotype imputation service and methods. Nat. Genet. 48, 1284–1287 (2016)." href="/articles/s41591-022-01767-6#ref-CR67" id="ref-link-section-d73985140e2427">67</a></sup> (<a href="https://imputationserver.sph.umich.edu/index.html#!">https://imputationserver.sph.umich.edu/index.html#!</a>), with ShapeIT (v2.r790) (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 68" title="Delaneau, O., Zagury, J.-F. &amp; Marchini, J. Improved whole-chromosome phasing for disease and population genetic studies. Nat. Methods 10, 5–6 (2013)." href="/articles/s41591-022-01767-6#ref-CR68" id="ref-link-section-d73985140e2439">68</a></sup>) used for phasing and Minimac3 used for imputation with default settings. Sets of imputed variants were compared with the base files for each PRS to ensure sufficient representation of probes (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">2</a>) (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Das, S. et al. Next-generation genotype imputation service and methods. Nat. Genet. 48, 1284–1287 (2016)." href="/articles/s41591-022-01767-6#ref-CR67" id="ref-link-section-d73985140e2446">67</a></sup>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec20">Calculation of PRS and adjustment for population structure</h4><p>Unadjusted raw PRS (PRS<sub>raw</sub>) for each disease were calculated using PLINK (v.2.0a) by taking the product of the count of risk alleles and the risk allele weight at each locus in the PRS and then summing across available risk loci. The loci included in each PRS, the risk alleles and the corresponding weights were downloaded from the PGS Catalog or Cancer PRSWeb. A population structure-adjusted PRS was calculated for each disease, using a previously described approach<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Khera, A. V. et al. Whole-genome sequencing to characterize monogenic and polygenic contributions in patients hospitalized with early-onset myocardial infarction. Circulation 139, 1593–1602 (2019)." href="/articles/s41591-022-01767-6#ref-CR40" id="ref-link-section-d73985140e2460">40</a></sup> implementing principal components analysis to compute adjusted residualized PRS for each disease. Principal components were calculated using all genotyped MGBB participants and a set of 16,385 of 16,443 previously reported ancestry-informative SNPs<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 69" title="Libiger, O. &amp; Schork, N. J. A method for inferring an individual’s genetic ancestry and degree of admixture associated with six major continental populations. Front. Genet. 3, 322 (2013)." href="/articles/s41591-022-01767-6#ref-CR69" id="ref-link-section-d73985140e2464">69</a></sup>. For each disease we then fit a linear model for PRS<sub>raw</sub> as a function of the first four principal components in controls for that disease (PRS<sub>raw</sub> ~ PC1 + PC2 + PC3 + PC4) in R (v.4.0.3). We then applied this model to calculate a predicted PRS (PRS<sub>pred</sub>) for each disease in all cases and controls. Residualized, population structure-adjusted PRS (PRS<sub>adj</sub>) were then computed for each individual for each disease as the difference between the raw and the predicted PRS (PRS<sub>raw</sub> − PRS<sub>pred</sub>). For PRS<sub>raw</sub>, values were standardized (PRS<sub>std-raw</sub>) using the mean and standard deviation in the MGBB of the PRS<sub>raw</sub> values (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">3</a>). Similarly, PRS<sub>std-adj</sub> was computed using the mean and standard deviation in the MGBB of the PRS<sub>adj</sub> values (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">3</a>). The distributions of PRS<sub>std-raw</sub> and PRS<sub>std-adj</sub> by genotype array, sex, age deciles and reported race were compared among all subjects using the density function in R (v.4.0.3).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec21">PRS–disease association</h4><p>The association of PRS<sub>std-adj</sub> with the odds of disease was replicated in MGBB participants using the six disease phenotypes described above. For each PRS and disease, odds of disease (<i>n</i><sub>cases</sub>/n<sub>controls</sub>) were calculated for each of 50 PRS quantiles. For race-stratified analyses, PRS deciles were used if too few cases were available for analysis across 50 quantiles. To visualize the PRS–disease associations, we plotted the log(odds) of disease against the mean PRS<sub>std-adj</sub> in each quantile. Correlation was measured with Pearson correlation coefficients using RStudio (v.1.1.383) with R (v.4.0.3).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec22">PRS threshold for high risk</h4><p>We set a predicted polygenic OR &gt; 2 to identify individuals at high polygenic risk for each disease, mirroring both a common threshold from Mendelian genetics<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Senol-Cosar, O. et al. Considerations for clinical curation, classification, and reporting of low-penetrance and low effect size variants associated with disease risk. Genet. Med. 21, 2765–2773 (2019)." href="/articles/s41591-022-01767-6#ref-CR31" id="ref-link-section-d73985140e2530">31</a></sup> and the effect sizes for disease risk factors already considered in current clinical care<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Goff, D. C. et al. 2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American College of Cardiology/American Heart Association Task Force on practice guidelines. Circulation 129(Suppl. 2), S49–S73 (2014)." href="#ref-CR32" id="ref-link-section-d73985140e2534">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="American Diabetes Association. 2. Classification and diagnosis of diabetes: standards of medical care in diabetes – 2021. Diabetes Care 44(Suppl. 1), S15–S33 (2021)." href="#ref-CR33" id="ref-link-section-d73985140e2534_1">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Grossman, D. C. et al. Screening for prostate cancer: US Preventive Services Task Force recommendation statement. JAMA 319, 1901–1913 (2018)." href="#ref-CR34" id="ref-link-section-d73985140e2534_2">34</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Siu, A. L., US Preventive Services Task Force Screening for breast cancer: US Preventive Services Task Force recommendation statement. Ann. Intern. Med. 164, 279–296 (2016)." href="#ref-CR35" id="ref-link-section-d73985140e2534_3">35</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Davidson, K. W. et al., US Preventive Services Task Force Screening for colorectal cancer: US Preventive Services Task Force recommendation statement. JAMA 325, 1965–1977 (2021)." href="/articles/s41591-022-01767-6#ref-CR36" id="ref-link-section-d73985140e2537">36</a></sup>. To operationalize this OR &gt; 2 threshold, we compared standardized PRS <i>Z</i> scores for each individual to a disease-specific cut off <span class="stix">𝛕</span>, based on previously published estimates of the change in odds of disease per standard deviation change in the PRS (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">3</a>). Specifically, <span class="stix">𝛕</span> = ln(2)/ln(OR<sub>s.d.</sub>), where 2 is the target OR threshold defining high risk and OR<sub>s.d.</sub> is the estimated multiplicative change in odds per standard deviation change in the PRS. Assuming that the published OR<sub>s.d.</sub> accurately captures the relationship between PRS and disease, the odds of disease for individuals with standardized PRS <i>Z</i> score = <span class="stix">𝛕</span> are twofold that of individuals with a median PRS <i>Z</i> score. These standardized PRS thresholds were used to assign individual patients to risk categories as described below (PRS calculation for clinical assay for individual samples).</p><h3 class="c-article__sub-heading" id="Sec23">Clinical PRS assay for individual samples</h3><p>Based on the results of the above methods, we developed and validated a genotype array-based clinical assay for PRS, in addition to secondary findings from the ACMG v2.0 list (ACMG SF v2.0, Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01767-6#Fig1">1</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Kalia, S. S. et al. Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics. Genet. Med. 19, 249–255 (2017)." href="/articles/s41591-022-01767-6#ref-CR24" id="ref-link-section-d73985140e2572">24</a></sup>. We include additional variants identified by the ACMG or other organizations as important secondary findings as updated recommendations accrue<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 70" title="Miller, D. T. et al. ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG). Genet. Med. 23, 1381–1390 (2021)." href="/articles/s41591-022-01767-6#ref-CR70" id="ref-link-section-d73985140e2576">70</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec24">Validation samples</h4><p>Replicates of each of three reference samples from GIAB<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Zook, J. M. et al. Extensive sequencing of seven human genomes to characterize benchmark reference materials. Sci. Data 3, 160025 (2016)." href="/articles/s41591-022-01767-6#ref-CR25" id="ref-link-section-d73985140e2587">25</a></sup> maintained by the National Institute of Standards and Technology were included in the validation assay: NA12878 × 9, NA24631 × 6 and NA24385 × 6. Analytical performance (sensitivity and PPV for presence or absence of variant sites) was determined in the benchmarking regions (v3.3.2). In addition, we included 22 samples with polymerase chain reaction-free genome sequencing data (described below) and 9 samples with high-risk PRS for one of the six diseases as determined by the MGBB data, including one individual with high-risk PRS for two diseases. To test the sensitivity of the secondary finding analysis, we genotyped 20 samples with previously identified pathogenic or likely pathogenic variants in the ACMG SF v2.0 list.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec25">Genotyping and imputation</h4><p>Validation samples were genotyped according to manufacturer-standard workflows on either a pre-commercial release of the Illumina Global Diversity Array (GDA-PC) or the final commercial release of the Global Diversity Array (GDA). The Illumina-specific files containing called genotypes in AA/AB/BB format (GTC files) generated by genotype array were converted to variant call format (VCF) using a modified version of the gtc2vcf script from Illumina. All samples required an overall call rate of greater than 98.5%. Imputation was performed using updated software, with EAGLE v2.4.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 71" title="Loh, P.-R. et al. Reference-based phasing using the Haplotype Reference Consortium panel. Nat. Genet. 48, 1443–1448 (2016)." href="/articles/s41591-022-01767-6#ref-CR71" id="ref-link-section-d73985140e2599">71</a></sup>) for phasing and Minimac4 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 67" title="Das, S. et al. Next-generation genotype imputation service and methods. Nat. Genet. 48, 1284–1287 (2016)." href="/articles/s41591-022-01767-6#ref-CR67" id="ref-link-section-d73985140e2603">67</a></sup>) for imputation using the 1000 Genomes Project phase 3 dataset. Importantly, monomorphic sites were not removed during the imputation process due to the small batch sizes used in the prospective assay.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec26">PRS calculation for clinical assay for individual samples</h4><p>PRS<sub>raw</sub> was calculated for each sample as described above. To determine PRS<sub>adj</sub>, unadjusted PRS (PRS<sub>raw</sub>) were first calculated for each individual sample as described for the overall MGBB cohort. For each individual, the eigenvariable, eigenvalue and frequency output from the MGBB principal components analysis were used to project each new individual sample onto the MGBB principal components, using the following command in PLINK v.2.0a:<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 72" title="Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. GigaScience 4, 7 (2015)." href="/articles/s41591-022-01767-6#ref-CR72" id="ref-link-section-d73985140e2621">72</a></sup></p><p class="c-code-block">
                    
                      <span class="c-code-block__line">plink2 —pfile individual_data —read-freq ref_pcs.acount —score ref_pcs.eigenvec.allele 2 5 header-read no-mean-imputation variance-standardize —score-col-nums 6-15 —out new_projection</span>
                    
                  </p><p>The resulting projected principal components were then scaled to match the MGBB principal components by taking the square root of the eigenvalue and then multiplying by 2. The scaled principal components (PCs) were fitted into the linear model for each disease developed in the MGBB data to obtain PRS<sub>pred</sub>:</p><p><span class="u-monospace">BrCa: PRSpred</span> = <span class="u-monospace">17.609341*PC1 − 4.146935*PC2</span> + <span class="u-monospace">5.335144*PC3</span> + <span class="u-monospace">3.833931*PC4 − 0.421679</span></p><p><span class="u-monospace">CRCa</span>: <span class="u-monospace">PRSpred</span> = −<span class="u-monospace">13.659121*PC1</span> + <span class="u-monospace">6.411109*PC2 − 2.483703*PC3 − 6.869127*PC4</span> + <span class="u-monospace">6.131384</span></p><p><span class="u-monospace">PrCa</span>: <span class="u-monospace">PRSpred</span> = <span class="u-monospace">23.441147*PC1 </span>+ <span class="u-monospace">13.724771*PC2 − 9.528270*PC3 </span>+ <span class="u-monospace">4.118756*PC4</span> + <span class="u-monospace">11.506243</span></p><p><span class="u-monospace">AFib: PRSpred</span> = <span class="u-monospace">9.6269881*PC1 − 3.2878238*PC2 − 6.6519006*PC3 − 3.0149108*PC4</span> + <span class="u-monospace">32.4067610</span></p><p><span class="u-monospace">CAD</span>: <span class="u-monospace">PRSpred</span> = −<span class="u-monospace">6.1974327*PC1 − 3.6757094*PC2 − 1.3488677*PC3 − 1.3490566*PC4</span> + <span class="u-monospace">18.0582457</span></p><p><span class="u-monospace">T2D</span>: <span class="u-monospace">PRSpred</span> = <span class="u-monospace">26.4700782*PC1</span> − <span class="u-monospace">7.4283370*PC2</span> + <span class="u-monospace">9.3782116*PC3</span> + <span class="u-monospace">1.6994457*PC4</span> + <span class="u-monospace">55.6998719</span></p><p>PRS<sub>adj</sub> was then calculated as the difference between PRS<sub>raw</sub> and PRS<sub>pred</sub>. Standardized, adjusted PRS values (PRS<sub>std-adj</sub>) were calculated using the mean and standard deviation of PRS<sub>adj</sub> in MGBB and compared against the PRS threshold corresponding to OR &gt; 2 as determined from the original publications (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM1">3</a>). Any PRS<sub>std-adj</sub> result above the PRS threshold corresponding to OR &gt; 2 was categorized as high polygenic risk.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec27">Genome sequencing</h4><p>We selected 22 diverse samples that had previously undergone clinical whole genome sequencing to determine the robustness of PRS across different platforms. Genome sequencing was performed at the Clinical Research Sequencing Platform of the Broad Institute using polymerase chain reaction-free library construction and sequencing on an Illumina NovaSeq with two 150 bp paired-end reads with ≥95% of bases covered at ≥20-fold. Reads were aligned to GRCh37 using the Burrows–Wheeler Aligner (BWA v.0.7.15)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 73" title="Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. Bioinformatics 25, 1754–1760 (2009)." href="/articles/s41591-022-01767-6#ref-CR73" id="ref-link-section-d73985140e2761">73</a></sup> and variant calls were made using HaplotypeCaller from the Genomic Analysis Tool Kit (GATK v.4.0.3.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 74" title="Poplin, R. et al. Scaling accurate genetic variant discovery to tens of thousands of samples. Preprint at bioRxiv 
                  https://doi.org/10.1101/201178
                  
                 (2018)." href="/articles/s41591-022-01767-6#ref-CR74" id="ref-link-section-d73985140e2765">74</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 75" title="Van der Auwera, G. A. &amp; O’Connor, B. D. Genomics in the Cloud: Using Docker, GATK, and WDL in Terra 1st edn (O’Reilly Media, 2020)." href="/articles/s41591-022-01767-6#ref-CR75" id="ref-link-section-d73985140e2768">75</a></sup>. PRS<sub>raw</sub>, PRS<sub>std-raw</sub>, PRS<sub>adj</sub> and PRS<sub>std-adj</sub> were calculated as above for the other prospective samples. As stated above, these 22 samples were also analyzed on the GDA-PC array to compare PRS between genome sequencing and array. The difference between the sequence-based and array-based PRS were visualized, and dichotomous risk classifications were formally compared using the Matthews correlation coefficient<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 76" title="Shi, L. et al. The MicroArray Quality Control (MAQC)-II study of common practices for the development and validation of microarray-based predictive models. Nat. Biotechnol. 28, 827–838 (2010)." href="/articles/s41591-022-01767-6#ref-CR76" id="ref-link-section-d73985140e2781">76</a></sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec28">Identification of actionable variants associated with monogenic disease</h4><p>Variants from the original genotyping VCF were annotated and filtered to the 59 genes suggested for screening of secondary findings as recommended by the ACMG (ACMG SF v2.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 24" title="Kalia, S. S. et al. Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics. Genet. Med. 19, 249–255 (2017)." href="/articles/s41591-022-01767-6#ref-CR24" id="ref-link-section-d73985140e2793">24</a></sup> to find: (1) variants previously identified as disease causing by the MGB Laboratory for Molecular Medicine; (2) variants classified as pathogenic or likely pathogenic in ClinVar with a minor allele frequency (MAF) &lt; 0.1%; (3) variants classified as a disease-causing mutation in the Human Gene Mutation Database with a MAF &lt; 0.03%; and (4) loss-of-function variants (nonsense, frameshift, canonical splice-site, and initiating methionine variants) with a MAF &lt; 0.1% in genes in which that is a disease mechanism. Clinical variant classification was carried out in accordance with the criteria set by the guidelines by the ACMG and the Association of Molecular Pathology<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 77" title="Richards, S. et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. Genet. Med. 17, 405–424 (2015)." href="/articles/s41591-022-01767-6#ref-CR77" id="ref-link-section-d73985140e2797">77</a></sup>, with disease-specific modifications as recommended by the Clinical Genome Resource Expert Panels<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 78" title="Rivera‐Muñoz, E. A. et al. ClinGen Variant Curation Expert Panel experiences and standardized processes for disease and gene‐level specification of the ACMG/AMP guidelines for sequence variant interpretation. Hum. Mutat. 39, 1614–1622 (2018)." href="/articles/s41591-022-01767-6#ref-CR78" id="ref-link-section-d73985140e2801">78</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec29">Prospectively enrolled trial participants</h3><p>The assay described above is now in use in the ongoing GenoVA Study randomized trial of clinical PRS (ClinicalTrials.gov identifier: <a href="https://clinicaltrials.gov/ct2/show/NCT04331535">NCT04331535</a>), in which eligible participants are patients of the VA Boston Healthcare System, aged 50–70 years, without known diagnoses of the six target diseases. Enrollees provide a clinical blood or saliva sample for analysis at the Laboratory for Molecular Medicine.</p><h3 class="c-article__sub-heading" id="Sec30">Ethics declaration</h3><p>Analyses of the genomic and MGBB samples and data have been reviewed and approved by the Mass General Brigham institutional review board (2019P001933). Analyses for the prospective pipeline, including the use of prior clinical samples, were conducted under the Mass General Brigham institutional review board (2004P001056); all individuals with clinical testing, including those with genome sequencing data, gave consent for clinical testing, and all individual data were de-identified. The GenoVA Study is approved by the VA Boston Healthcare System (no. 3241) and Harvard Medical School institutional review board (IRB19-0594), and all enrollees provided written informed consent.</p><h3 class="c-article__sub-heading" id="Sec31">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01767-6#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>The majority of the MGBB genotyped samples are deposited in dbGAP as part of the eMERGE consortium, phase 3 (<a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001584.v2.p2">https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs001584.v2.p2</a>). Additional MGBB data were accessed under institutional review board protocol for this current study and are not publicly available due to restrictions on the data. Data from the GenoVA Study trial will be made publicly available after study completion. The 1000 Genomes Project phase 3 dataset used in this study was v5a and was downloaded from <a href="ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/release/20130502/</a>.</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>The code used to adjust the PRS for population structure is available for download here: <a href="https://github.com/MGB-Personalized-Medicine/PRS-adjustment">https://github.com/MGB-Personalized-Medicine/PRS-adjustment</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Shendure, J., Findlay, G. M. &amp; Snyder, M. W. Genomic medicine: progress, pitfalls, and promise. <i>Cell</i> <b>177</b>, 45–57 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2019.02.003" data-track-item_id="10.1016/j.cell.2019.02.003" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2019.02.003" aria-label="Article reference 1" data-doi="10.1016/j.cell.2019.02.003">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvVWgt70%3D" aria-label="CAS reference 1">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30901547" aria-label="PubMed reference 1">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6531313" aria-label="PubMed Central reference 1">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20medicine%3A%20progress%2C%20pitfalls%2C%20and%20promise&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.02.003&amp;volume=177&amp;pages=45-57&amp;publication_year=2019&amp;author=Shendure%2CJ&amp;author=Findlay%2CGM&amp;author=Snyder%2CMW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2"><i>GWAS Catalog</i> (National Human Genome Research Institute); <a href="https://www.ebi.ac.uk/gwas/" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.ebi.ac.uk/gwas/">https://www.ebi.ac.uk/gwas/</a></p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Meigs, J. B. et al. Genotype score in addition to common risk factors for prediction of type 2 diabetes. <i>N. Engl. J. Med.</i> <b>359</b>, 2208–2219 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa0804742" data-track-item_id="10.1056/NEJMoa0804742" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa0804742" aria-label="Article reference 3" data-doi="10.1056/NEJMoa0804742">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtl2gurnE" aria-label="CAS reference 3">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19020323" aria-label="PubMed reference 3">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2746946" aria-label="PubMed Central reference 3">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Genotype%20score%20in%20addition%20to%20common%20risk%20factors%20for%20prediction%20of%20type%202%20diabetes&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa0804742&amp;volume=359&amp;pages=2208-2219&amp;publication_year=2008&amp;author=Meigs%2CJB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Ripatti, S. et al. A multilocus genetic risk score for coronary heart disease: case-control and prospective cohort analyses. <i>Lancet</i> <b>376</b>, 1393–1400 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(10)61267-6" data-track-item_id="10.1016/S0140-6736(10)61267-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2810%2961267-6" aria-label="Article reference 4" data-doi="10.1016/S0140-6736(10)61267-6">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20971364" aria-label="PubMed reference 4">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2965351" aria-label="PubMed Central reference 4">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multilocus%20genetic%20risk%20score%20for%20coronary%20heart%20disease%3A%20case-control%20and%20prospective%20cohort%20analyses&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2810%2961267-6&amp;volume=376&amp;pages=1393-1400&amp;publication_year=2010&amp;author=Ripatti%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Zheng, S. L. et al. Cumulative association of five genetic variants with prostate cancer. <i>N. Engl. J. Med.</i> <b>358</b>, 910–919 (2008).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa075819" data-track-item_id="10.1056/NEJMoa075819" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa075819" aria-label="Article reference 5" data-doi="10.1056/NEJMoa075819">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1cXisF2mtrg%3D" aria-label="CAS reference 5">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18199855" aria-label="PubMed reference 5">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Cumulative%20association%20of%20five%20genetic%20variants%20with%20prostate%20cancer&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa075819&amp;volume=358&amp;pages=910-919&amp;publication_year=2008&amp;author=Zheng%2CSL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Boyle, E. A., Li, Y. I. &amp; Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. <i>Cell</i> <b>169</b>, 1177–1186 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2017.05.038" data-track-item_id="10.1016/j.cell.2017.05.038" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2017.05.038" aria-label="Article reference 6" data-doi="10.1016/j.cell.2017.05.038">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVWrtrjM" aria-label="CAS reference 6">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28622505" aria-label="PubMed reference 6">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5536862" aria-label="PubMed Central reference 6">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=An%20expanded%20view%20of%20complex%20traits%3A%20from%20polygenic%20to%20omnigenic&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.05.038&amp;volume=169&amp;pages=1177-1186&amp;publication_year=2017&amp;author=Boyle%2CEA&amp;author=Li%2CYI&amp;author=Pritchard%2CJK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Khera, A. V. et al. Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations. <i>Nat. Genet.</i> <b>50</b>, 1219–1224 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0183-z" data-track-item_id="10.1038/s41588-018-0183-z" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0183-z" aria-label="Article reference 7" data-doi="10.1038/s41588-018-0183-z">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFSqtbbI" aria-label="CAS reference 7">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30104762" aria-label="PubMed reference 7">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6128408" aria-label="PubMed Central reference 7">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20polygenic%20scores%20for%20common%20diseases%20identify%20individuals%20with%20risk%20equivalent%20to%20monogenic%20mutations&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0183-z&amp;volume=50&amp;pages=1219-1224&amp;publication_year=2018&amp;author=Khera%2CAV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Vilhjálmsson, B. J. et al. Modeling linkage disequilibrium increases accuracy of polygenic risk scores. <i>Am. J. Hum. Genet.</i> <b>97</b>, 576–592 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2015.09.001" data-track-item_id="10.1016/j.ajhg.2015.09.001" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2015.09.001" aria-label="Article reference 8" data-doi="10.1016/j.ajhg.2015.09.001">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26430803" aria-label="PubMed reference 8">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4596916" aria-label="PubMed Central reference 8">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFKhsbzK" aria-label="CAS reference 8">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Modeling%20linkage%20disequilibrium%20increases%20accuracy%20of%20polygenic%20risk%20scores&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2015.09.001&amp;volume=97&amp;pages=576-592&amp;publication_year=2015&amp;author=Vilhj%C3%A1lmsson%2CBJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Inouye, M. et al. Genomic risk prediction of coronary artery disease in 480,000 adults: implications for primary prevention. <i>J. Am. Coll. Cardiol.</i> <b>72</b>, 1883–1893 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.jacc.2018.07.079" data-track-item_id="10.1016/j.jacc.2018.07.079" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.jacc.2018.07.079" aria-label="Article reference 9" data-doi="10.1016/j.jacc.2018.07.079">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30309464" aria-label="PubMed reference 9">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6176870" aria-label="PubMed Central reference 9">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20risk%20prediction%20of%20coronary%20artery%20disease%20in%20480%2C000%20adults%3A%20implications%20for%20primary%20prevention&amp;journal=J.%20Am.%20Coll.%20Cardiol.&amp;doi=10.1016%2Fj.jacc.2018.07.079&amp;volume=72&amp;pages=1883-1893&amp;publication_year=2018&amp;author=Inouye%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Conti, D. V. et al. Trans-ancestry genome-wide association meta-analysis of prostate cancer identifies new susceptibility loci and informs genetic risk prediction. <i>Nat. Genet.</i> <b>53</b>, 65–75 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-020-00748-0" data-track-item_id="10.1038/s41588-020-00748-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-020-00748-0" aria-label="Article reference 10" data-doi="10.1038/s41588-020-00748-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXkslSqsA%3D%3D" aria-label="CAS reference 10">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33398198" aria-label="PubMed reference 10">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8148035" aria-label="PubMed Central reference 10">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Trans-ancestry%20genome-wide%20association%20meta-analysis%20of%20prostate%20cancer%20identifies%20new%20susceptibility%20loci%20and%20informs%20genetic%20risk%20prediction&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-020-00748-0&amp;volume=53&amp;pages=65-75&amp;publication_year=2021&amp;author=Conti%2CDV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Klarin, D. et al. Genome-wide association analysis of venous thromboembolism identifies new risk loci and genetic overlap with arterial vascular disease. <i>Nat. Genet.</i> <b>51</b>, 1574–1579 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-019-0519-3" data-track-item_id="10.1038/s41588-019-0519-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-019-0519-3" aria-label="Article reference 11" data-doi="10.1038/s41588-019-0519-3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFSjsrnO" aria-label="CAS reference 11">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31676865" aria-label="PubMed reference 11">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858581" aria-label="PubMed Central reference 11">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20association%20analysis%20of%20venous%20thromboembolism%20identifies%20new%20risk%20loci%20and%20genetic%20overlap%20with%20arterial%20vascular%20disease&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0519-3&amp;volume=51&amp;pages=1574-1579&amp;publication_year=2019&amp;author=Klarin%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Mosley, J. D. et al. Predictive accuracy of a polygenic risk score compared with a clinical risk score for incident coronary heart disease. <i>JAMA</i> <b>323</b>, 627–635 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1001/jama.2019.21782" data-track-item_id="10.1001/jama.2019.21782" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1001%2Fjama.2019.21782" aria-label="Article reference 12" data-doi="10.1001/jama.2019.21782">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXjslGrsLs%3D" aria-label="CAS reference 12">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32068817" aria-label="PubMed reference 12">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7042849" aria-label="PubMed Central reference 12">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Predictive%20accuracy%20of%20a%20polygenic%20risk%20score%20compared%20with%20a%20clinical%20risk%20score%20for%20incident%20coronary%20heart%20disease&amp;journal=JAMA&amp;doi=10.1001%2Fjama.2019.21782&amp;volume=323&amp;pages=627-635&amp;publication_year=2020&amp;author=Mosley%2CJD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Vassy, J. L. et al. Polygenic type 2 diabetes prediction at the limit of common variant detection. <i>Diabetes</i> <b>63</b>, 2172–2182 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.2337/db13-1663" data-track-item_id="10.2337/db13-1663" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.2337%2Fdb13-1663" aria-label="Article reference 13" data-doi="10.2337/db13-1663">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsV2ls7rN" aria-label="CAS reference 13">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24520119" aria-label="PubMed reference 13">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4030114" aria-label="PubMed Central reference 13">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20type%202%20diabetes%20prediction%20at%20the%20limit%20of%20common%20variant%20detection&amp;journal=Diabetes&amp;doi=10.2337%2Fdb13-1663&amp;volume=63&amp;pages=2172-2182&amp;publication_year=2014&amp;author=Vassy%2CJL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Seibert, T. M. et al. Polygenic hazard score to guide screening for aggressive prostate cancer: development and validation in large scale cohorts. <i>BMJ</i> <b>360</b>, j5757 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1136/bmj.j5757" data-track-item_id="10.1136/bmj.j5757" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1136%2Fbmj.j5757" aria-label="Article reference 14" data-doi="10.1136/bmj.j5757">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29321194" aria-label="PubMed reference 14">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5759091" aria-label="PubMed Central reference 14">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20hazard%20score%20to%20guide%20screening%20for%20aggressive%20prostate%20cancer%3A%20development%20and%20validation%20in%20large%20scale%20cohorts&amp;journal=BMJ&amp;doi=10.1136%2Fbmj.j5757&amp;volume=360&amp;publication_year=2018&amp;author=Seibert%2CTM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">National Human Genome Research Institute (NHGRI). <i>Electronic Medical Records and Genomics (eMERGE) Network</i> <a href="https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE">https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE</a> (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Shieh, Y. et al. Breast cancer screening in the precision medicine era: risk-based screening in a population-based trial. <i>J. Natl Cancer Inst</i>. <b>109</b>, <a href="https://doi.org/10.1093/jnci/djw290" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1093/jnci/djw290">https://doi.org/10.1093/jnci/djw290</a> (2017).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Brockman, D. G. et al. Design and user experience testing of a polygenic score report: a qualitative study of prospective users. <i>BMC Med. Genomics</i> <b>14</b>, 238 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s12920-021-01056-0" data-track-item_id="10.1186/s12920-021-01056-0" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s12920-021-01056-0" aria-label="Article reference 17" data-doi="10.1186/s12920-021-01056-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34598685" aria-label="PubMed reference 17">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8485114" aria-label="PubMed Central reference 17">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Design%20and%20user%20experience%20testing%20of%20a%20polygenic%20score%20report%3A%20a%20qualitative%20study%20of%20prospective%20users&amp;journal=BMC%20Med.%20Genomics&amp;doi=10.1186%2Fs12920-021-01056-0&amp;volume=14&amp;publication_year=2021&amp;author=Brockman%2CDG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Choi, S. W., Mak, T. S.-H. &amp; O’Reilly, P. F. Tutorial: a guide to performing polygenic risk score analyses. <i>Nat. Protoc.</i> <b>15</b>, 2759–2772 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41596-020-0353-1" data-track-item_id="10.1038/s41596-020-0353-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41596-020-0353-1" aria-label="Article reference 18" data-doi="10.1038/s41596-020-0353-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVGltLbI" aria-label="CAS reference 18">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32709988" aria-label="PubMed reference 18">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7612115" aria-label="PubMed Central reference 18">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Tutorial%3A%20a%20guide%20to%20performing%20polygenic%20risk%20score%20analyses&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fs41596-020-0353-1&amp;volume=15&amp;pages=2759-2772&amp;publication_year=2020&amp;author=Choi%2CSW&amp;author=Mak%2CTS-H&amp;author=O%E2%80%99Reilly%2CPF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. <i>Nat. Genet.</i> <b>51</b>, 584–591 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-019-0379-x" data-track-item_id="10.1038/s41588-019-0379-x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-019-0379-x" aria-label="Article reference 19" data-doi="10.1038/s41588-019-0379-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslGltLw%3D" aria-label="CAS reference 19">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30926966" aria-label="PubMed reference 19">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6563838" aria-label="PubMed Central reference 19">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20use%20of%20current%20polygenic%20risk%20scores%20may%20exacerbate%20health%20disparities&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0379-x&amp;volume=51&amp;pages=584-591&amp;publication_year=2019&amp;author=Martin%2CAR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Lewis, C. M. &amp; Vassos, E. Polygenic risk scores: from research tools to clinical instruments. <i>Genome Med.</i> <b>12</b>, 44 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13073-020-00742-5" data-track-item_id="10.1186/s13073-020-00742-5" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13073-020-00742-5" aria-label="Article reference 20" data-doi="10.1186/s13073-020-00742-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32423490" aria-label="PubMed reference 20">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7236300" aria-label="PubMed Central reference 20">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%3A%20from%20research%20tools%20to%20clinical%20instruments&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-020-00742-5&amp;volume=12&amp;publication_year=2020&amp;author=Lewis%2CCM&amp;author=Vassos%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Lambert, S. A. et al. The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation. <i>Nat. Genet.</i> <b>53</b>, 420–425 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-021-00783-5" data-track-item_id="10.1038/s41588-021-00783-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-021-00783-5" aria-label="Article reference 21" data-doi="10.1038/s41588-021-00783-5">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXmt1entLg%3D" aria-label="CAS reference 21">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33692568" aria-label="PubMed reference 21">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Polygenic%20Score%20Catalog%20as%20an%20open%20database%20for%20reproducibility%20and%20systematic%20evaluation&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-021-00783-5&amp;volume=53&amp;pages=420-425&amp;publication_year=2021&amp;author=Lambert%2CSA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Fritsche, L. G. et al. Cancer PRSweb: an online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks. <i>Am. J. Hum. Genet.</i> <b>107</b>, 815–836 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2020.08.025" data-track-item_id="10.1016/j.ajhg.2020.08.025" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2020.08.025" aria-label="Article reference 22" data-doi="10.1016/j.ajhg.2020.08.025">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFKjs7nK" aria-label="CAS reference 22">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32991828" aria-label="PubMed reference 22">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7675001" aria-label="PubMed Central reference 22">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Cancer%20PRSweb%3A%20an%20online%20repository%20with%20polygenic%20risk%20scores%20for%20major%20cancer%20traits%20and%20their%20evaluation%20in%20two%20independent%20biobanks&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2020.08.025&amp;volume=107&amp;pages=815-836&amp;publication_year=2020&amp;author=Fritsche%2CLG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Huyghe, J. R. et al. Discovery of common and rare genetic risk variants for colorectal cancer. <i>Nat. Genet.</i> <b>51</b>, 76–87 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0286-6" data-track-item_id="10.1038/s41588-018-0286-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0286-6" aria-label="Article reference 23" data-doi="10.1038/s41588-018-0286-6">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXitl2iu7vF" aria-label="CAS reference 23">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30510241" aria-label="PubMed reference 23">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=Discovery%20of%20common%20and%20rare%20genetic%20risk%20variants%20for%20colorectal%20cancer&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0286-6&amp;volume=51&amp;pages=76-87&amp;publication_year=2019&amp;author=Huyghe%2CJR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Kalia, S. S. et al. Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics. <i>Genet. Med.</i> <b>19</b>, 249–255 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/gim.2016.190" data-track-item_id="10.1038/gim.2016.190" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fgim.2016.190" aria-label="Article reference 24" data-doi="10.1038/gim.2016.190">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27854360" aria-label="PubMed reference 24">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Recommendations%20for%20reporting%20of%20secondary%20findings%20in%20clinical%20exome%20and%20genome%20sequencing%2C%202016%20update%20%28ACMG%20SF%20v2.0%29%3A%20a%20policy%20statement%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fgim.2016.190&amp;volume=19&amp;pages=249-255&amp;publication_year=2017&amp;author=Kalia%2CSS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Zook, J. M. et al. Extensive sequencing of seven human genomes to characterize benchmark reference materials. <i>Sci. Data</i> <b>3</b>, 160025 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/sdata.2016.25" data-track-item_id="10.1038/sdata.2016.25" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fsdata.2016.25" aria-label="Article reference 25" data-doi="10.1038/sdata.2016.25">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XpvVWrtLk%3D" aria-label="CAS reference 25">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27271295" aria-label="PubMed reference 25">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4896128" aria-label="PubMed Central reference 25">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Extensive%20sequencing%20of%20seven%20human%20genomes%20to%20characterize%20benchmark%20reference%20materials&amp;journal=Sci.%20Data&amp;doi=10.1038%2Fsdata.2016.25&amp;volume=3&amp;publication_year=2016&amp;author=Zook%2CJM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Bowling, K. M. et al. Identifying rare, medically relevant variation via population-based genomic screening in Alabama: opportunities and pitfalls. <i>Genet. Med.</i> <b>23</b>, 280–288 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41436-020-00976-z" data-track-item_id="10.1038/s41436-020-00976-z" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41436-020-00976-z" aria-label="Article reference 26" data-doi="10.1038/s41436-020-00976-z">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVSju73P" aria-label="CAS reference 26">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32989269" aria-label="PubMed reference 26">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Identifying%20rare%2C%20medically%20relevant%20variation%20via%20population-based%20genomic%20screening%20in%20Alabama%3A%20opportunities%20and%20pitfalls&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fs41436-020-00976-z&amp;volume=23&amp;pages=280-288&amp;publication_year=2021&amp;author=Bowling%2CKM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Weedon, M. N. &amp; Wright, C. F. et al. Use of SNP chips to detect rare pathogenic variants: retrospective, population based diagnostic evaluation. <i>BMJ</i> <b>372</b>, n214 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=Use%20of%20SNP%20chips%20to%20detect%20rare%20pathogenic%20variants%3A%20retrospective%2C%20population%20based%20diagnostic%20evaluation&amp;journal=BMJ&amp;volume=372&amp;publication_year=2021&amp;author=Weedon%2CMN&amp;author=Wright%2CCF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Scheuner, M. T., Edelen, M. O., Hilborne, L. H. &amp; Lubin, I. M. Effective communication of molecular genetic test results to primary care providers. <i>Genet. Med.</i> <b>15</b>, 444–449 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/gim.2012.151" data-track-item_id="10.1038/gim.2012.151" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fgim.2012.151" aria-label="Article reference 28" data-doi="10.1038/gim.2012.151">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23222660" aria-label="PubMed reference 28">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=Effective%20communication%20of%20molecular%20genetic%20test%20results%20to%20primary%20care%20providers&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fgim.2012.151&amp;volume=15&amp;pages=444-449&amp;publication_year=2013&amp;author=Scheuner%2CMT&amp;author=Edelen%2CMO&amp;author=Hilborne%2CLH&amp;author=Lubin%2CIM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">McLaughlin, H. M. et al. A systematic approach to the reporting of medically relevant findings from whole genome sequencing. <i>BMC Med. Genet.</i> <b>15</b>, 134 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s12881-014-0134-1" data-track-item_id="10.1186/s12881-014-0134-1" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s12881-014-0134-1" aria-label="Article reference 29" data-doi="10.1186/s12881-014-0134-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25714468" aria-label="PubMed reference 29">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342199" aria-label="PubMed Central reference 29">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXitlCntr0%3D" aria-label="CAS reference 29">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20systematic%20approach%20to%20the%20reporting%20of%20medically%20relevant%20findings%20from%20whole%20genome%20sequencing&amp;journal=BMC%20Med.%20Genet.&amp;doi=10.1186%2Fs12881-014-0134-1&amp;volume=15&amp;publication_year=2014&amp;author=McLaughlin%2CHM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Farmer, G. D., Gray, H., Chandratillake, G., Raymond, F. L. &amp; Freeman, A. L. J. Recommendations for designing genetic test reports to be understood by patients and non-specialists. <i>Eur. J. Hum. Genet.</i> <b>28</b>, 885–895 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41431-020-0579-y" data-track-item_id="10.1038/s41431-020-0579-y" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41431-020-0579-y" aria-label="Article reference 30" data-doi="10.1038/s41431-020-0579-y">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32024982" aria-label="PubMed reference 30">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7316722" aria-label="PubMed Central reference 30">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Recommendations%20for%20designing%20genetic%20test%20reports%20to%20be%20understood%20by%20patients%20and%20non-specialists&amp;journal=Eur.%20J.%20Hum.%20Genet.&amp;doi=10.1038%2Fs41431-020-0579-y&amp;volume=28&amp;pages=885-895&amp;publication_year=2020&amp;author=Farmer%2CGD&amp;author=Gray%2CH&amp;author=Chandratillake%2CG&amp;author=Raymond%2CFL&amp;author=Freeman%2CALJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Senol-Cosar, O. et al. Considerations for clinical curation, classification, and reporting of low-penetrance and low effect size variants associated with disease risk. <i>Genet. Med.</i> <b>21</b>, 2765–2773 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41436-019-0560-8" data-track-item_id="10.1038/s41436-019-0560-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41436-019-0560-8" aria-label="Article reference 31" data-doi="10.1038/s41436-019-0560-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31147632" aria-label="PubMed reference 31">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Considerations%20for%20clinical%20curation%2C%20classification%2C%20and%20reporting%20of%20low-penetrance%20and%20low%20effect%20size%20variants%20associated%20with%20disease%20risk&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fs41436-019-0560-8&amp;volume=21&amp;pages=2765-2773&amp;publication_year=2019&amp;author=Senol-Cosar%2CO">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Goff, D. C. et al. 2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American College of Cardiology/American Heart Association Task Force on practice guidelines. <i>Circulation</i> <b>129</b>(Suppl. 2), S49–S73 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24222018" aria-label="PubMed reference 32">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=2013%20ACC%2FAHA%20guideline%20on%20the%20assessment%20of%20cardiovascular%20risk%3A%20a%20report%20of%20the%20American%20College%20of%20Cardiology%2FAmerican%20Heart%20Association%20Task%20Force%20on%20practice%20guidelines&amp;journal=Circulation&amp;volume=129&amp;issue=Suppl.%202&amp;pages=S49-S73&amp;publication_year=2014&amp;author=Goff%2CDC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">American Diabetes Association. 2. Classification and diagnosis of diabetes: standards of medical care in diabetes – 2021. <i>Diabetes Care</i> <b>44</b>(Suppl. 1), S15–S33 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.2337/dc21-S002" data-track-item_id="10.2337/dc21-S002" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.2337%2Fdc21-S002" aria-label="Article reference 33" data-doi="10.2337/dc21-S002">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=2.%20Classification%20and%20diagnosis%20of%20diabetes%3A%20standards%20of%20medical%20care%20in%20diabetes%20%E2%80%93%202021&amp;journal=Diabetes%20Care&amp;doi=10.2337%2Fdc21-S002&amp;volume=44&amp;issue=Suppl.%201&amp;pages=S15-S33&amp;publication_year=2021">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Grossman, D. C. et al. Screening for prostate cancer: US Preventive Services Task Force recommendation statement. <i>JAMA</i> <b>319</b>, 1901–1913 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1001/jama.2018.0161" data-track-item_id="10.1001/jama.2018.0161" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1001%2Fjama.2018.0161" aria-label="Article reference 34" data-doi="10.1001/jama.2018.0161">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29801017" aria-label="PubMed reference 34">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Screening%20for%20prostate%20cancer%3A%20US%20Preventive%20Services%20Task%20Force%20recommendation%20statement&amp;journal=JAMA&amp;doi=10.1001%2Fjama.2018.0161&amp;volume=319&amp;pages=1901-1913&amp;publication_year=2018&amp;author=Grossman%2CDC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Siu, A. L., US Preventive Services Task Force Screening for breast cancer: US Preventive Services Task Force recommendation statement. <i>Ann. Intern. Med.</i> <b>164</b>, 279–296 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7326/M15-2886" data-track-item_id="10.7326/M15-2886" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7326%2FM15-2886" aria-label="Article reference 35" data-doi="10.7326/M15-2886">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26757170" aria-label="PubMed reference 35">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Screening%20for%20breast%20cancer%3A%20US%20Preventive%20Services%20Task%20Force%20recommendation%20statement&amp;journal=Ann.%20Intern.%20Med.&amp;doi=10.7326%2FM15-2886&amp;volume=164&amp;pages=279-296&amp;publication_year=2016&amp;author=Siu%2CAL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Davidson, K. W. et al., US Preventive Services Task Force Screening for colorectal cancer: US Preventive Services Task Force recommendation statement. <i>JAMA</i> <b>325</b>, 1965–1977 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1001/jama.2021.6238" data-track-item_id="10.1001/jama.2021.6238" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1001%2Fjama.2021.6238" aria-label="Article reference 36" data-doi="10.1001/jama.2021.6238">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34003218" aria-label="PubMed reference 36">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Screening%20for%20colorectal%20cancer%3A%20US%20Preventive%20Services%20Task%20Force%20recommendation%20statement&amp;journal=JAMA&amp;doi=10.1001%2Fjama.2021.6238&amp;volume=325&amp;pages=1965-1977&amp;publication_year=2021&amp;author=Davidson%2CKW">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Wand, H. et al. Improving reporting standards for polygenic scores in risk prediction studies. <i>Nature</i> <b>591</b>, 211–219 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-021-03243-6" data-track-item_id="10.1038/s41586-021-03243-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-03243-6" aria-label="Article reference 37" data-doi="10.1038/s41586-021-03243-6">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXmt1ems7Y%3D" aria-label="CAS reference 37">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33692554" aria-label="PubMed reference 37">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8609771" aria-label="PubMed Central reference 37">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=Improving%20reporting%20standards%20for%20polygenic%20scores%20in%20risk%20prediction%20studies&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03243-6&amp;volume=591&amp;pages=211-219&amp;publication_year=2021&amp;author=Wand%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Homburger, J. R. et al. Low coverage whole genome sequencing enables accurate assessment of common variants and calculation of genome-wide polygenic scores. <i>Genome Med.</i> <b>11</b>, 74 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13073-019-0682-2" data-track-item_id="10.1186/s13073-019-0682-2" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13073-019-0682-2" aria-label="Article reference 38" data-doi="10.1186/s13073-019-0682-2">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVyksbvK" aria-label="CAS reference 38">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31771638" aria-label="PubMed reference 38">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6880438" aria-label="PubMed Central reference 38">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Low%20coverage%20whole%20genome%20sequencing%20enables%20accurate%20assessment%20of%20common%20variants%20and%20calculation%20of%20genome-wide%20polygenic%20scores&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-019-0682-2&amp;volume=11&amp;publication_year=2019&amp;author=Homburger%2CJR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Denny, J. C. et al. The ‘All of Us’ Research Program. <i>N. Engl. J. Med.</i> <b>381</b>, 668–676 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMsr1809937" data-track-item_id="10.1056/NEJMsr1809937" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMsr1809937" aria-label="Article reference 39" data-doi="10.1056/NEJMsr1809937">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31412182" aria-label="PubMed reference 39">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20%E2%80%98All%20of%20Us%E2%80%99%20Research%20Program&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMsr1809937&amp;volume=381&amp;pages=668-676&amp;publication_year=2019&amp;author=Denny%2CJC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Khera, A. V. et al. Whole-genome sequencing to characterize monogenic and polygenic contributions in patients hospitalized with early-onset myocardial infarction. <i>Circulation</i> <b>139</b>, 1593–1602 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1161/CIRCULATIONAHA.118.035658" data-track-item_id="10.1161/CIRCULATIONAHA.118.035658" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1161%2FCIRCULATIONAHA.118.035658" aria-label="Article reference 40" data-doi="10.1161/CIRCULATIONAHA.118.035658">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXlslaqsb0%3D" aria-label="CAS reference 40">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30586733" aria-label="PubMed reference 40">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6433484" aria-label="PubMed Central reference 40">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-genome%20sequencing%20to%20characterize%20monogenic%20and%20polygenic%20contributions%20in%20patients%20hospitalized%20with%20early-onset%20myocardial%20infarction&amp;journal=Circulation&amp;doi=10.1161%2FCIRCULATIONAHA.118.035658&amp;volume=139&amp;pages=1593-1602&amp;publication_year=2019&amp;author=Khera%2CAV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Dikilitas, O. et al. Predictive utility of polygenic risk scores for coronary heart disease in three major racial and ethnic groups. <i>Am. J. Hum. Genet.</i> <b>106</b>, 707–716 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2020.04.002" data-track-item_id="10.1016/j.ajhg.2020.04.002" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2020.04.002" aria-label="Article reference 41" data-doi="10.1016/j.ajhg.2020.04.002">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXptFGgurY%3D" aria-label="CAS reference 41">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32386537" aria-label="PubMed reference 41">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7212267" aria-label="PubMed Central reference 41">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Predictive%20utility%20of%20polygenic%20risk%20scores%20for%20coronary%20heart%20disease%20in%20three%20major%20racial%20and%20ethnic%20groups&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2020.04.002&amp;volume=106&amp;pages=707-716&amp;publication_year=2020&amp;author=Dikilitas%2CO">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Kowalski, M. H. et al. Use of &gt;100,000 NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium whole genome sequences improves imputation quality and detection of rare variant associations in admixed African and Hispanic/Latino populations. <i>PLoS Genet.</i> <b>15</b>, e1008500 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pgen.1008500" data-track-item_id="10.1371/journal.pgen.1008500" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pgen.1008500" aria-label="Article reference 42" data-doi="10.1371/journal.pgen.1008500">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31869403" aria-label="PubMed reference 42">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6953885" aria-label="PubMed Central reference 42">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlKnsb8%3D" aria-label="CAS reference 42">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=Use%20of%20%3E100%2C000%20NHLBI%20Trans-Omics%20for%20Precision%20Medicine%20%28TOPMed%29%20Consortium%20whole%20genome%20sequences%20improves%20imputation%20quality%20and%20detection%20of%20rare%20variant%20associations%20in%20admixed%20African%20and%20Hispanic%2FLatino%20populations&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1008500&amp;volume=15&amp;publication_year=2019&amp;author=Kowalski%2CMH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Lewis, A. C. F., Green, R. C. &amp; Vassy, J. L. Polygenic risk scores in the clinic: translating risk into action. <i>HGG Adv.</i> <b>2</b>, 100047 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35047839" aria-label="PubMed reference 43">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8756548" aria-label="PubMed Central reference 43">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%20in%20the%20clinic%3A%20translating%20risk%20into%20action&amp;journal=HGG%20Adv.&amp;volume=2&amp;publication_year=2021&amp;author=Lewis%2CACF&amp;author=Green%2CRC&amp;author=Vassy%2CJL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Lee, A. et al. BOADICEA: a comprehensive breast cancer risk prediction model incorporating genetic and nongenetic risk factors. <i>Genet. Med.</i> <b>21</b>, 1708–1718 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41436-018-0406-9" data-track-item_id="10.1038/s41436-018-0406-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41436-018-0406-9" aria-label="Article reference 44" data-doi="10.1038/s41436-018-0406-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30643217" aria-label="PubMed reference 44">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6687499" aria-label="PubMed Central reference 44">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=BOADICEA%3A%20a%20comprehensive%20breast%20cancer%20risk%20prediction%20model%20incorporating%20genetic%20and%20nongenetic%20risk%20factors&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fs41436-018-0406-9&amp;volume=21&amp;pages=1708-1718&amp;publication_year=2019&amp;author=Lee%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Stone, N. J. et al. 2013 ACC/AHA guideline on the treatment of blood cholesterol to reduce atherosclerotic cardiovascular risk in adults: a report of the American College of Cardiology/American Heart Association Task Force on Practice Guidelines. <i>Circulation</i> <b>129</b>(Suppl. 2), S1–S45 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24222016" aria-label="PubMed reference 45">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=2013%20ACC%2FAHA%20guideline%20on%20the%20treatment%20of%20blood%20cholesterol%20to%20reduce%20atherosclerotic%20cardiovascular%20risk%20in%20adults%3A%20a%20report%20of%20the%20American%20College%20of%20Cardiology%2FAmerican%20Heart%20Association%20Task%20Force%20on%20Practice%20Guidelines&amp;journal=Circulation&amp;volume=129&amp;issue=Suppl.%202&amp;pages=S1-S45&amp;publication_year=2014&amp;author=Stone%2CNJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Hughes, E. et al. Development and validation of a clinical polygenic risk score to predict breast cancer risk. <i>JCO Precis. Oncol.</i> <b>4</b>, 585–592 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1200/PO.19.00360" data-track-item_id="10.1200/PO.19.00360" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1200%2FPO.19.00360" aria-label="Article reference 46" data-doi="10.1200/PO.19.00360">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20and%20validation%20of%20a%20clinical%20polygenic%20risk%20score%20to%20predict%20breast%20cancer%20risk&amp;journal=JCO%20Precis.%20Oncol.&amp;doi=10.1200%2FPO.19.00360&amp;volume=4&amp;pages=585-592&amp;publication_year=2020&amp;author=Hughes%2CE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47"><i>Our Health + Ancestry DNA Service - 23andMe</i> (23andMe); <a href="https://www.23andme.com/dna-health-ancestry/" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.23andme.com/dna-health-ancestry/">https://www.23andme.com/dna-health-ancestry/</a></p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Chen, S.-F. et al. Genotype imputation and variability in polygenic risk score estimation. <i>Genome Med.</i> <b>12</b>, 100 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13073-020-00801-x" data-track-item_id="10.1186/s13073-020-00801-x" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13073-020-00801-x" aria-label="Article reference 48" data-doi="10.1186/s13073-020-00801-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33225976" aria-label="PubMed reference 48">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7682022" aria-label="PubMed Central reference 48">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 48" href="http://scholar.google.com/scholar_lookup?&amp;title=Genotype%20imputation%20and%20variability%20in%20polygenic%20risk%20score%20estimation&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-020-00801-x&amp;volume=12&amp;publication_year=2020&amp;author=Chen%2CS-F">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">National Human Genome Research Institute. <i>Polygenic RIsk MEthods in Diverse populations (PRIMED) Consortium</i> <a href="https://www.genome.gov/Funded-Programs-Projects/PRIMED-Consortium" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.genome.gov/Funded-Programs-Projects/PRIMED-Consortium">https://www.genome.gov/Funded-Programs-Projects/PRIMED-Consortium</a></p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Manolio, T. A. Using the data we have: improving diversity in genomic research. <i>Am. J. Hum. Genet.</i> <b>105</b>, 233–236 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2019.07.008" data-track-item_id="10.1016/j.ajhg.2019.07.008" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2019.07.008" aria-label="Article reference 50" data-doi="10.1016/j.ajhg.2019.07.008">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFemtbzM" aria-label="CAS reference 50">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31374201" aria-label="PubMed reference 50">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6698933" aria-label="PubMed Central reference 50">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20the%20data%20we%20have%3A%20improving%20diversity%20in%20genomic%20research&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2019.07.008&amp;volume=105&amp;pages=233-236&amp;publication_year=2019&amp;author=Manolio%2CTA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Lewis, A. C. F. &amp; Green, R. C. Polygenic risk scores in the clinic: new perspectives needed on familiar ethical issues. <i>Genome Med.</i> <b>13</b>, 14 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13073-021-00829-7" data-track-item_id="10.1186/s13073-021-00829-7" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13073-021-00829-7" aria-label="Article reference 51" data-doi="10.1186/s13073-021-00829-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33509269" aria-label="PubMed reference 51">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7844961" aria-label="PubMed Central reference 51">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%20in%20the%20clinic%3A%20new%20perspectives%20needed%20on%20familiar%20ethical%20issues&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-021-00829-7&amp;volume=13&amp;publication_year=2021&amp;author=Lewis%2CACF&amp;author=Green%2CRC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Ray, T. Myriad Genetics recalibrates breast cancer PRS for all ancestries in anticipation of broader launch. <i>Genomeweb</i> <a href="https://www.genomeweb.com/molecular-diagnostics/myriad-genetics-recalibrates-breast-cancer-prs-all-ancestries-anticipation" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.genomeweb.com/molecular-diagnostics/myriad-genetics-recalibrates-breast-cancer-prs-all-ancestries-anticipation">https://www.genomeweb.com/molecular-diagnostics/myriad-genetics-recalibrates-breast-cancer-prs-all-ancestries-anticipation</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Ambry Product Team. <i>Important discontinuation notice: AmbryScore: polygenic risk scores (PRS</i>) <a href="https://info.ambrygenetics.com/take-a-brief-survey-for-entry-into-amazon-gift-card-drawing" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://info.ambrygenetics.com/take-a-brief-survey-for-entry-into-amazon-gift-card-drawing">https://info.ambrygenetics.com/take-a-brief-survey-for-entry-into-amazon-gift-card-drawing</a></p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Ge, T. et al. Validation of a trans-ancestry polygenic risk score for type 2 diabetes in diverse populations. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2021.09.11.21263413" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.09.11.21263413">https://doi.org/10.1101/2021.09.11.21263413</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Marnetto, D. et al. Ancestry deconvolution and partial polygenic score can improve susceptibility predictions in recently admixed individuals. <i>Nat. Commun.</i> <b>11</b>, 1628 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-15464-w" data-track-item_id="10.1038/s41467-020-15464-w" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-15464-w" aria-label="Article reference 55" data-doi="10.1038/s41467-020-15464-w">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32242022" aria-label="PubMed reference 55">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118071" aria-label="PubMed Central reference 55">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXmsVSksr4%3D" aria-label="CAS reference 55">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Ancestry%20deconvolution%20and%20partial%20polygenic%20score%20can%20improve%20susceptibility%20predictions%20in%20recently%20admixed%20individuals&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-15464-w&amp;volume=11&amp;publication_year=2020&amp;author=Marnetto%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="56."><p class="c-article-references__text" id="ref-CR56">Ruan, Y. et al. Improving polygenic prediction in ancestrally diverse populations. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2020.12.27.20248738" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2020.12.27.20248738">https://doi.org/10.1101/2020.12.27.20248738</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="57."><p class="c-article-references__text" id="ref-CR57">Armstrong, K. A. &amp; Metlay, J. P. Annals clinical decision making: translating population evidence to individual patients. <i>Ann. Intern. Med.</i> <b>172</b>, 610–616 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7326/M19-3496" data-track-item_id="10.7326/M19-3496" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7326%2FM19-3496" aria-label="Article reference 57" data-doi="10.7326/M19-3496">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32311741" aria-label="PubMed reference 57">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 57" href="http://scholar.google.com/scholar_lookup?&amp;title=Annals%20clinical%20decision%20making%3A%20translating%20population%20evidence%20to%20individual%20patients&amp;journal=Ann.%20Intern.%20Med.&amp;doi=10.7326%2FM19-3496&amp;volume=172&amp;pages=610-616&amp;publication_year=2020&amp;author=Armstrong%2CKA&amp;author=Metlay%2CJP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="58."><p class="c-article-references__text" id="ref-CR58">Sniderman, A. D., LaChapelle, K. J., Rachon, N. A. &amp; Furberg, C. D. The necessity for clinical reasoning in the era of evidence-based medicine. <i>Mayo Clin. Proc.</i> <b>88</b>, 1108–1114 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.mayocp.2013.07.012" data-track-item_id="10.1016/j.mayocp.2013.07.012" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.mayocp.2013.07.012" aria-label="Article reference 58" data-doi="10.1016/j.mayocp.2013.07.012">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24079680" aria-label="PubMed reference 58">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 58" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20necessity%20for%20clinical%20reasoning%20in%20the%20era%20of%20evidence-based%20medicine&amp;journal=Mayo%20Clin.%20Proc.&amp;doi=10.1016%2Fj.mayocp.2013.07.012&amp;volume=88&amp;pages=1108-1114&amp;publication_year=2013&amp;author=Sniderman%2CAD&amp;author=LaChapelle%2CKJ&amp;author=Rachon%2CNA&amp;author=Furberg%2CCD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="59."><p class="c-article-references__text" id="ref-CR59">Michailidou, K. et al. Association analysis identifies 65 new breast cancer risk loci. <i>Nature</i> <b>551</b>, 92–94 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nature24284" data-track-item_id="10.1038/nature24284" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnature24284" aria-label="Article reference 59" data-doi="10.1038/nature24284">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29059683" aria-label="PubMed reference 59">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5798588" aria-label="PubMed Central reference 59">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslajtr3L" aria-label="CAS reference 59">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 59" href="http://scholar.google.com/scholar_lookup?&amp;title=Association%20analysis%20identifies%2065%20new%20breast%20cancer%20risk%20loci&amp;journal=Nature&amp;doi=10.1038%2Fnature24284&amp;volume=551&amp;pages=92-94&amp;publication_year=2017&amp;author=Michailidou%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="60."><p class="c-article-references__text" id="ref-CR60">Mavaddat, N. et al. Polygenic risk scores for prediction of breast cancer and breast cancer subtypes. <i>Am. J. Hum. Genet.</i> <b>104</b>, 21–34 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2018.11.002" data-track-item_id="10.1016/j.ajhg.2018.11.002" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2018.11.002" aria-label="Article reference 60" data-doi="10.1016/j.ajhg.2018.11.002">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXisFWqsb%2FO" aria-label="CAS reference 60">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30554720" aria-label="PubMed reference 60">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 60" href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%20for%20prediction%20of%20breast%20cancer%20and%20breast%20cancer%20subtypes&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2018.11.002&amp;volume=104&amp;pages=21-34&amp;publication_year=2019&amp;author=Mavaddat%2CN">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="61."><p class="c-article-references__text" id="ref-CR61">Schumacher, F. R. et al. Association analyses of more than 140,000 men identify 63 new prostate cancer susceptibility loci. <i>Nat. Genet.</i> <b>50</b>, 928–936 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41588-018-0142-8" data-track-item_id="10.1038/s41588-018-0142-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41588-018-0142-8" aria-label="Article reference 61" data-doi="10.1038/s41588-018-0142-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFWru7jL" aria-label="CAS reference 61">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29892016" aria-label="PubMed reference 61">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6568012" aria-label="PubMed Central reference 61">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 61" href="http://scholar.google.com/scholar_lookup?&amp;title=Association%20analyses%20of%20more%20than%20140%2C000%20men%20identify%2063%20new%20prostate%20cancer%20susceptibility%20loci&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0142-8&amp;volume=50&amp;pages=928-936&amp;publication_year=2018&amp;author=Schumacher%2CFR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="62."><p class="c-article-references__text" id="ref-CR62">Karlson, E. W., Boutin, N. T., Hoffnagle, A. G. &amp; Allen, N. L. Building the Partners Healthcare Biobank at Partners Personalized Medicine: informed consent, return of research results, recruitment lessons and operational considerations. <i>J. Pers. Med.</i> <b>6</b>, 2 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3390/jpm6010002" data-track-item_id="10.3390/jpm6010002" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3390%2Fjpm6010002" aria-label="Article reference 62" data-doi="10.3390/jpm6010002">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4810381" aria-label="PubMed Central reference 62">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 62" href="http://scholar.google.com/scholar_lookup?&amp;title=Building%20the%20Partners%20Healthcare%20Biobank%20at%20Partners%20Personalized%20Medicine%3A%20informed%20consent%2C%20return%20of%20research%20results%2C%20recruitment%20lessons%20and%20operational%20considerations&amp;journal=J.%20Pers.%20Med.&amp;doi=10.3390%2Fjpm6010002&amp;volume=6&amp;publication_year=2016&amp;author=Karlson%2CEW&amp;author=Boutin%2CNT&amp;author=Hoffnagle%2CAG&amp;author=Allen%2CNL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="63."><p class="c-article-references__text" id="ref-CR63">Yu, S. et al. Toward high-throughput phenotyping: unbiased automated feature extraction and selection from knowledge sources. <i>J. Am. Med. Inform. Assoc.</i> <b>22</b>, 993–1000 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/jamia/ocv034" data-track-item_id="10.1093/jamia/ocv034" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fjamia%2Focv034" aria-label="Article reference 63" data-doi="10.1093/jamia/ocv034">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25929596" aria-label="PubMed reference 63">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4986664" aria-label="PubMed Central reference 63">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 63" href="http://scholar.google.com/scholar_lookup?&amp;title=Toward%20high-throughput%20phenotyping%3A%20unbiased%20automated%20feature%20extraction%20and%20selection%20from%20knowledge%20sources&amp;journal=J.%20Am.%20Med.%20Inform.%20Assoc.&amp;doi=10.1093%2Fjamia%2Focv034&amp;volume=22&amp;pages=993-1000&amp;publication_year=2015&amp;author=Yu%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="64."><p class="c-article-references__text" id="ref-CR64">Yu, S. et al. Enabling phenotypic big data with PheNorm. <i>J. Am. Med. Inform. Assoc.</i> <b>25</b>, 54–60 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/jamia/ocx111" data-track-item_id="10.1093/jamia/ocx111" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fjamia%2Focx111" aria-label="Article reference 64" data-doi="10.1093/jamia/ocx111">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29126253" aria-label="PubMed reference 64">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 64" href="http://scholar.google.com/scholar_lookup?&amp;title=Enabling%20phenotypic%20big%20data%20with%20PheNorm&amp;journal=J.%20Am.%20Med.%20Inform.%20Assoc.&amp;doi=10.1093%2Fjamia%2Focx111&amp;volume=25&amp;pages=54-60&amp;publication_year=2018&amp;author=Yu%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="65."><p class="c-article-references__text" id="ref-CR65">Gainer, V. S. et al. The Biobank portal for Partners Personalized Medicine: a query tool for working with consented Biobank samples, genotypes, and phenotypes using i2b2. <i>J. Pers. Med.</i> <b>6</b>, 11 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3390/jpm6010011" data-track-item_id="10.3390/jpm6010011" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3390%2Fjpm6010011" aria-label="Article reference 65" data-doi="10.3390/jpm6010011">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4810390" aria-label="PubMed Central reference 65">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 65" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Biobank%20portal%20for%20Partners%20Personalized%20Medicine%3A%20a%20query%20tool%20for%20working%20with%20consented%20Biobank%20samples%2C%20genotypes%2C%20and%20phenotypes%20using%20i2b2&amp;journal=J.%20Pers.%20Med.&amp;doi=10.3390%2Fjpm6010011&amp;volume=6&amp;publication_year=2016&amp;author=Gainer%2CVS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="66."><p class="c-article-references__text" id="ref-CR66">Blau, A., Brown, A., Mahanta, L. &amp; Amr, S. S. The translational genomics core at Partners Personalized Medicine: facilitating the transition of research towards personalized medicine. <i>J. Pers. Med.</i> <b>6</b>, 10 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3390/jpm6010010" data-track-item_id="10.3390/jpm6010010" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3390%2Fjpm6010010" aria-label="Article reference 66" data-doi="10.3390/jpm6010010">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4810389" aria-label="PubMed Central reference 66">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 66" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20translational%20genomics%20core%20at%20Partners%20Personalized%20Medicine%3A%20facilitating%20the%20transition%20of%20research%20towards%20personalized%20medicine&amp;journal=J.%20Pers.%20Med.&amp;doi=10.3390%2Fjpm6010010&amp;volume=6&amp;publication_year=2016&amp;author=Blau%2CA&amp;author=Brown%2CA&amp;author=Mahanta%2CL&amp;author=Amr%2CSS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="67."><p class="c-article-references__text" id="ref-CR67">Das, S. et al. Next-generation genotype imputation service and methods. <i>Nat. Genet.</i> <b>48</b>, 1284–1287 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ng.3656" data-track-item_id="10.1038/ng.3656" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3656" aria-label="Article reference 67" data-doi="10.1038/ng.3656">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVWksL%2FK" aria-label="CAS reference 67">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27571263" aria-label="PubMed reference 67">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5157836" aria-label="PubMed Central reference 67">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 67" href="http://scholar.google.com/scholar_lookup?&amp;title=Next-generation%20genotype%20imputation%20service%20and%20methods&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3656&amp;volume=48&amp;pages=1284-1287&amp;publication_year=2016&amp;author=Das%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="68."><p class="c-article-references__text" id="ref-CR68">Delaneau, O., Zagury, J.-F. &amp; Marchini, J. Improved whole-chromosome phasing for disease and population genetic studies. <i>Nat. Methods</i> <b>10</b>, 5–6 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nmeth.2307" data-track-item_id="10.1038/nmeth.2307" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnmeth.2307" aria-label="Article reference 68" data-doi="10.1038/nmeth.2307">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhvFShtbnN" aria-label="CAS reference 68">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23269371" aria-label="PubMed reference 68">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 68" href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20whole-chromosome%20phasing%20for%20disease%20and%20population%20genetic%20studies&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.2307&amp;volume=10&amp;pages=5-6&amp;publication_year=2013&amp;author=Delaneau%2CO&amp;author=Zagury%2CJ-F&amp;author=Marchini%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="69."><p class="c-article-references__text" id="ref-CR69">Libiger, O. &amp; Schork, N. J. A method for inferring an individual’s genetic ancestry and degree of admixture associated with six major continental populations. <i>Front. Genet.</i> <b>3</b>, 322 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.3389/fgene.2012.00322" data-track-item_id="10.3389/fgene.2012.00322" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.3389%2Ffgene.2012.00322" aria-label="Article reference 69" data-doi="10.3389/fgene.2012.00322">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23335941" aria-label="PubMed reference 69">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3543981" aria-label="PubMed Central reference 69">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 69" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20method%20for%20inferring%20an%20individual%E2%80%99s%20genetic%20ancestry%20and%20degree%20of%20admixture%20associated%20with%20six%20major%20continental%20populations&amp;journal=Front.%20Genet.&amp;doi=10.3389%2Ffgene.2012.00322&amp;volume=3&amp;publication_year=2013&amp;author=Libiger%2CO&amp;author=Schork%2CNJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="70."><p class="c-article-references__text" id="ref-CR70">Miller, D. T. et al. ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG). <i>Genet. Med.</i> <b>23</b>, 1381–1390 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41436-021-01172-3" data-track-item_id="10.1038/s41436-021-01172-3" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41436-021-01172-3" aria-label="Article reference 70" data-doi="10.1038/s41436-021-01172-3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34012068" aria-label="PubMed reference 70">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 70" href="http://scholar.google.com/scholar_lookup?&amp;title=ACMG%20SF%20v3.0%20list%20for%20reporting%20of%20secondary%20findings%20in%20clinical%20exome%20and%20genome%20sequencing%3A%20a%20policy%20statement%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics%20%28ACMG%29&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fs41436-021-01172-3&amp;volume=23&amp;pages=1381-1390&amp;publication_year=2021&amp;author=Miller%2CDT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="71."><p class="c-article-references__text" id="ref-CR71">Loh, P.-R. et al. Reference-based phasing using the Haplotype Reference Consortium panel. <i>Nat. Genet.</i> <b>48</b>, 1443–1448 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/ng.3679" data-track-item_id="10.1038/ng.3679" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fng.3679" aria-label="Article reference 71" data-doi="10.1038/ng.3679">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1SltLzF" aria-label="CAS reference 71">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27694958" aria-label="PubMed reference 71">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5096458" aria-label="PubMed Central reference 71">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 71" href="http://scholar.google.com/scholar_lookup?&amp;title=Reference-based%20phasing%20using%20the%20Haplotype%20Reference%20Consortium%20panel&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3679&amp;volume=48&amp;pages=1443-1448&amp;publication_year=2016&amp;author=Loh%2CP-R">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="72."><p class="c-article-references__text" id="ref-CR72">Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. <i>GigaScience</i> <b>4</b>, 7 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13742-015-0047-8" data-track-item_id="10.1186/s13742-015-0047-8" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13742-015-0047-8" aria-label="Article reference 72" data-doi="10.1186/s13742-015-0047-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25722852" aria-label="PubMed reference 72">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342193" aria-label="PubMed Central reference 72">PubMed Central</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsFShsLY%3D" aria-label="CAS reference 72">CAS</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 72" href="http://scholar.google.com/scholar_lookup?&amp;title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&amp;journal=GigaScience&amp;doi=10.1186%2Fs13742-015-0047-8&amp;volume=4&amp;publication_year=2015&amp;author=Chang%2CCC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="73."><p class="c-article-references__text" id="ref-CR73">Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. <i>Bioinformatics</i> <b>25</b>, 1754–1760 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btp324" data-track-item_id="10.1093/bioinformatics/btp324" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp324" aria-label="Article reference 73" data-doi="10.1093/bioinformatics/btp324">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D" aria-label="CAS reference 73">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19451168" aria-label="PubMed reference 73">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234" aria-label="PubMed Central reference 73">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 73" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp324&amp;volume=25&amp;pages=1754-1760&amp;publication_year=2009&amp;author=Li%2CH&amp;author=Durbin%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="74."><p class="c-article-references__text" id="ref-CR74">Poplin, R. et al. Scaling accurate genetic variant discovery to tens of thousands of samples. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/201178" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/201178">https://doi.org/10.1101/201178</a> (2018).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="75."><p class="c-article-references__text" id="ref-CR75">Van der Auwera, G. A. &amp; O’Connor, B. D. <i>Genomics in the Cloud: Using Docker, GATK, and WDL in Terra</i> 1st edn (O’Reilly Media, 2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="76."><p class="c-article-references__text" id="ref-CR76">Shi, L. et al. The MicroArray Quality Control (MAQC)-II study of common practices for the development and validation of microarray-based predictive models. <i>Nat. Biotechnol.</i> <b>28</b>, 827–838 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nbt.1665" data-track-item_id="10.1038/nbt.1665" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnbt.1665" aria-label="Article reference 76" data-doi="10.1038/nbt.1665">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXpsFWku7w%3D" aria-label="CAS reference 76">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20676074" aria-label="PubMed reference 76">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 76" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20MicroArray%20Quality%20Control%20%28MAQC%29-II%20study%20of%20common%20practices%20for%20the%20development%20and%20validation%20of%20microarray-based%20predictive%20models&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.1665&amp;volume=28&amp;pages=827-838&amp;publication_year=2010&amp;author=Shi%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="77."><p class="c-article-references__text" id="ref-CR77">Richards, S. et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. <i>Genet. Med.</i> <b>17</b>, 405–424 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/gim.2015.30" data-track-item_id="10.1038/gim.2015.30" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fgim.2015.30" aria-label="Article reference 77" data-doi="10.1038/gim.2015.30">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25741868" aria-label="PubMed reference 77">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4544753" aria-label="PubMed Central reference 77">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 77" href="http://scholar.google.com/scholar_lookup?&amp;title=Standards%20and%20guidelines%20for%20the%20interpretation%20of%20sequence%20variants%3A%20a%20joint%20consensus%20recommendation%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics%20and%20the%20Association%20for%20Molecular%20Pathology&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fgim.2015.30&amp;volume=17&amp;pages=405-424&amp;publication_year=2015&amp;author=Richards%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="78."><p class="c-article-references__text" id="ref-CR78">Rivera‐Muñoz, E. A. et al. ClinGen Variant Curation Expert Panel experiences and standardized processes for disease and gene‐level specification of the ACMG/AMP guidelines for sequence variant interpretation. <i>Hum. Mutat.</i> <b>39</b>, 1614–1622 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1002/humu.23645" data-track-item_id="10.1002/humu.23645" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1002%2Fhumu.23645" aria-label="Article reference 78" data-doi="10.1002/humu.23645">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30311389" aria-label="PubMed reference 78">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6225902" aria-label="PubMed Central reference 78">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 78" href="http://scholar.google.com/scholar_lookup?&amp;title=ClinGen%20Variant%20Curation%20Expert%20Panel%20experiences%20and%20standardized%20processes%20for%20disease%20and%20gene%E2%80%90level%20specification%20of%20the%20ACMG%2FAMP%20guidelines%20for%20sequence%20variant%20interpretation&amp;journal=Hum.%20Mutat.&amp;doi=10.1002%2Fhumu.23645&amp;volume=39&amp;pages=1614-1622&amp;publication_year=2018&amp;author=Rivera%E2%80%90Mu%C3%B1oz%2CEA">
                    Google Scholar</a>&nbsp;
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01767-6?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>This work was supported by NIH National Human Genome Research Institute grant R35HG010706 (J.L.V.). S.A.L. is supported by NIH grants R01HL139731, R01HL157635 and American Heart Association 18SFRN34250007. P.N. is supported by grants from the National Heart, Lung and Blood Institute (R01HL142711, R01HL148050, R01HL151283, R01HL127564, R01HL148565, R01HL135242, R01HL151152), National Institute of Diabetes and Digestive and Kidney Diseases (R01DK125782), Fondation Leducq (TNE-18CVD04) and Massachusetts General Hospital (Fireman Chair). M.S.L. is supported by NIH grants U01HG008685, R01HG010372, R01HL143295, OT2OD02750, U41HG006834 and U01TR003201, all unrelated to the present work.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="na1"><p>Posthumous authorship: Wanfeng Yu</p></li><li class="c-article-author-information__item" id="na2"><p>These authors contributed equally: Jason L. Vassy, Matthew S. Lebo.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Laboratory for Molecular Medicine, Mass General Brigham Personalized Medicine, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Limin Hao,&nbsp;Gabriel F. Berriz,&nbsp;Elizabeth D. Hynes,&nbsp;Christopher Koch,&nbsp;Prathik Korategere V Kumar,&nbsp;Shruti S. Parpattedar,&nbsp;Marcie Steeves,&nbsp;Wanfeng Yu&nbsp;&amp;&nbsp;Matthew S. Lebo</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Department of Epidemiology, Harvard T.H. Chan School of Public Health, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Peter Kraft</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Medical Genetics, Massachusetts General Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Marcie Steeves</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Veterans Affairs Boston Healthcare System, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Ashley A. Antwi,&nbsp;Charles A. Brunette,&nbsp;Morgan Danowski,&nbsp;Natalie E. Jones&nbsp;&amp;&nbsp;Jason L. Vassy</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Division of Gastroenterology, Massachusetts General Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Manish K. Gala</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Manish K. Gala,&nbsp;Robert C. Green,&nbsp;Pradeep Natarajan,&nbsp;Jason L. Vassy&nbsp;&amp;&nbsp;Matthew S. Lebo</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Broad Institute of Harvard and the Massachusetts Institute of Technology, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Robert C. Green&nbsp;&amp;&nbsp;Natalie E. Jones</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Department of Medicine, Brigham and Women’s Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Robert C. Green&nbsp;&amp;&nbsp;Jason L. Vassy</p></li><li id="Aff9"><p class="c-article-author-affiliation__address">Precision Population Health, Ariadne Labs, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Robert C. Green&nbsp;&amp;&nbsp;Jason L. Vassy</p></li><li id="Aff10"><p class="c-article-author-affiliation__address">E J Safra Center for Ethics, Harvard University, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Anna C. F. Lewis</p></li><li id="Aff11"><p class="c-article-author-affiliation__address">Cardiovascular Disease Initiative, Broad Institute of Harvard and the Massachusetts Institute of Technology, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Steven A. Lubitz&nbsp;&amp;&nbsp;Pradeep Natarajan</p></li><li id="Aff12"><p class="c-article-author-affiliation__address">Cardiovascular Research Center, Massachusetts General Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Steven A. Lubitz&nbsp;&amp;&nbsp;Pradeep Natarajan</p></li><li id="Aff13"><p class="c-article-author-affiliation__address">Demoulas Center for Cardiac Arrhythmias, Massachusetts General Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Steven A. Lubitz</p></li><li id="Aff14"><p class="c-article-author-affiliation__address">Department of Pathology, Brigham and Women’s Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Matthew S. Lebo</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Limin-Hao-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Limin Hao</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Limin%20Hao" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Limin%20Hao" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Limin%20Hao%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Peter-Kraft-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Peter Kraft</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Peter%20Kraft" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Peter%20Kraft" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Peter%20Kraft%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Gabriel_F_-Berriz-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Gabriel F. Berriz</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Gabriel%20F.%20Berriz" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Gabriel%20F.%20Berriz" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Gabriel%20F.%20Berriz%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Elizabeth_D_-Hynes-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Elizabeth D. Hynes</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Elizabeth%20D.%20Hynes" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Elizabeth%20D.%20Hynes" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Elizabeth%20D.%20Hynes%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Christopher-Koch-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Christopher Koch</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Christopher%20Koch" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christopher%20Koch" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christopher%20Koch%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Prathik-Korategere_V_Kumar-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Prathik Korategere V Kumar</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Prathik%20Korategere%20V%20Kumar" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Prathik%20Korategere%20V%20Kumar" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Prathik%20Korategere%20V%20Kumar%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Shruti_S_-Parpattedar-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Shruti S. Parpattedar</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Shruti%20S.%20Parpattedar" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Shruti%20S.%20Parpattedar" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Shruti%20S.%20Parpattedar%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marcie-Steeves-Aff1-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Marcie Steeves</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Marcie%20Steeves" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marcie%20Steeves" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marcie%20Steeves%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Wanfeng-Yu-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Wanfeng Yu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Wanfeng%20Yu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Wanfeng%20Yu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Wanfeng%20Yu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ashley_A_-Antwi-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Ashley A. Antwi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ashley%20A.%20Antwi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ashley%20A.%20Antwi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ashley%20A.%20Antwi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Charles_A_-Brunette-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Charles A. Brunette</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Charles%20A.%20Brunette" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Charles%20A.%20Brunette" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Charles%20A.%20Brunette%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Morgan-Danowski-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Morgan Danowski</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Morgan%20Danowski" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Morgan%20Danowski" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Morgan%20Danowski%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Manish_K_-Gala-Aff5-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Manish K. Gala</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Manish%20K.%20Gala" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Manish%20K.%20Gala" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Manish%20K.%20Gala%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Robert_C_-Green-Aff6-Aff7-Aff8-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Robert C. Green</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Robert%20C.%20Green" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Robert%20C.%20Green" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Robert%20C.%20Green%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Natalie_E_-Jones-Aff4-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Natalie E. Jones</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Natalie%20E.%20Jones" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Natalie%20E.%20Jones" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Natalie%20E.%20Jones%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Anna_C__F_-Lewis-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Anna C. F. Lewis</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Anna%20C.%20F.%20Lewis" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Anna%20C.%20F.%20Lewis" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Anna%20C.%20F.%20Lewis%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Steven_A_-Lubitz-Aff11-Aff12-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Steven A. Lubitz</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Steven%20A.%20Lubitz" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Steven%20A.%20Lubitz" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Steven%20A.%20Lubitz%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Pradeep-Natarajan-Aff6-Aff11-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Pradeep Natarajan</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Pradeep%20Natarajan" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Pradeep%20Natarajan" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Pradeep%20Natarajan%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jason_L_-Vassy-Aff4-Aff6-Aff8-Aff9"><span class="c-article-authors-search__title u-h3 js-search-name">Jason L. Vassy</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jason%20L.%20Vassy" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jason%20L.%20Vassy" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jason%20L.%20Vassy%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Matthew_S_-Lebo-Aff1-Aff6-Aff14"><span class="c-article-authors-search__title u-h3 js-search-name">Matthew S. Lebo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Matthew%20S.%20Lebo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Matthew%20S.%20Lebo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Matthew%20S.%20Lebo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>L.H., P.K., C.A.B., M.K.G., R.C.G., A.C.F.L., S.A.L., P.N., J.L.V. and M.S.L. conceived the study and contributed to its design. L.H., G.F.B., E.D.H., C.K., P.K.V.K., S.S.P., M.S., W.Y. and C.A.B. analyzed the data from the MGBB and GenoVA samples. A.A.A., C.A.B., M.D., N.E.J. and J.L.V. collected GenoVA data. L.H., J.L.V. and M.S.L. drafted the manuscript, and all authors reviewed the scientific content of the manuscript prior to submission.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding author</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:jvassy@bwh.harvard.edu">Jason L. Vassy</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar4">Competing interests</h3>
                <p>A.C.F.L. owns stock in Fabric Genomics. S.A.L. receives sponsored research support from Bristol Myers Squibb, Pfizer, Boehringer Ingelheim, Fitbit, Medtronic, Premier and IBM, and has consulted for Bristol Myers Squibb, Pfizer, Blackstone Life Sciences and Invitae. P.N. reports investigator-initiated grants from Amgen, Apple, Boston Scientific, AstraZeneca and Novartis, personal fees from Apple, Genentech, AstraZeneca, Blackstone Life Science, Foresite Labs and Novartis, spousal employment at Vertex, and being co-founder of TenSixteen Bio, all unrelated to the present work. R.C.G. has received compensation for advising the following companies: AIA, Allelica, Embryome, GenomeWeb, Genomic Life, Grail, Humanity, Meenta, OptumLabs, Plumcare, Verily, VinBigData; and is co-founder of Genome Medical, Inc. C.K. now works at Novartis Institutes for BioMedical Research. A.A.A., C.A.B., M.D. and J.L.V. are employees of the US Department of Veterans Affairs (DVA); the views expressed in this paper do not represent those of the DVA or US government. All other authors have no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
              
              
                <h3 class="c-article__sub-heading" id="FPar3">Peer review information</h3>
                <p><i>Nature Medicine</i> thanks Krista Fischer and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Anna Maria Ranzoni was the primary editor on this article and managed its editorial process and peer review in collaboration with the rest of the editorial team.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec33-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec33">Extended data</h2><div class="c-article-section__content" id="Sec33-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 distribution of standardized," href="/articles/s41591-022-01767-6/figures/5" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig5_ESM.jpg">Extended Data Fig. 1 Distribution of standardized, adjusted PRS by release batch for six diseases in MGBB.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Standardized, adjusted PRS (PRS<sub>std-adj</sub>) plotted by eight batches of three versions of Illumina genotyping arrays (MEG, MEGA, MEGAEX) used to analyze data from up to 36,423 MGBB participants. Abbreviations: AFib, atrial fibrillation; BrCa, breast cancer; CAD, coronary artery disease; CRCa, colorectal cancer; MEG, Multi-Ethnic Global; MEGA, Multi-Ethnic Genotyping Array; MEGAEX, Expanded Multi-Ethnic Genotyping Array; MGBB, Mass General Brigham Biobank; PrCa, prostate cancer; PRS, polygenic risk score; T2D, type 2 diabetes.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 distribution of standardized," href="/articles/s41591-022-01767-6/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig6_ESM.jpg">Extended Data Fig. 2 Distribution of standardized, adjusted PRS by age for six diseases in MGBB.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Standardized, adjusted PRS (PRS<sub>std-adj</sub>) plotted by decade of age among up to 36,423 MGBB participants. Abbreviations: AFib, atrial fibrillation; BrCa, breast cancer; CAD, coronary artery disease; CRCa, colorectal cancer; MGBB, Mass General Brigham Biobank; PrCa, prostate cancer; PRS, polygenic risk score; T2D, type 2 diabetes.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 distribution of adjusted prs " href="/articles/s41591-022-01767-6/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig7_ESM.jpg">Extended Data Fig. 3 Distribution of adjusted PRS by sex for four diseases in MGBB.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Standardized, adjusted PRS plotted by sex among 16,704 male and 19,719 female MGBB participants. Abbreviations: AFib, atrial fibrillation; BrCa, breast cancer; CAD, coronary artery disease; CRCa, colorectal cancer; MGBB, Mass General Brigham Biobank; PrCa, prostate cancer; PRS, polygenic risk score; T2D, type 2 diabetes.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 correlation between standardi" href="/articles/s41591-022-01767-6/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig8_ESM.jpg">Extended Data Fig. 4 Correlation between standardized, adjusted PRS and odds of disease in reported white MGBB participants.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Plots show log(odds) of each of six diseases versus quantile (n = 50) of standardized population structure-adjusted PRS (PRS<sub>std-adj</sub>) among up to 30,716 MGBB participants of reported white race. The correlation coefficient, r, is shown in each panel. Abbreviations: AFib, atrial fibrillation; BrCa, breast cancer; CAD, coronary artery disease; CRCa, colorectal cancer; MGBB, Mass General Brigham Biobank; OR, odds ratio; PrCa, prostate cancer; PRS, polygenic risk score; T2D, type 2 diabetes.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 correlation between standardi" href="/articles/s41591-022-01767-6/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig9_ESM.jpg">Extended Data Fig. 5 Correlation between standardized, adjusted PRS and odds of disease in reported Black MGBB participants.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Plots show log(odds) of each of six diseases versus quantile (n = 10) of standardized population structure-adjusted PRS (PRS<sub>std-adj</sub>) among up to 1,807 MGBB participants of reported Black race. Results not reported for CRCa due to 0 CRCa cases in at least one quantile. The correlation coefficient, r, is shown in each panel. Abbreviations: AFib, atrial fibrillation; BrCa, breast cancer; CAD, coronary artery disease; CRCa, colorectal cancer; MGBB, Mass General Brigham Biobank; OR, odds ratio; PrCa, prostate cancer; PRS, polygenic risk score; T2D, type 2 diabetes.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 correlation between standardi" href="/articles/s41591-022-01767-6/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig10_ESM.jpg">Extended Data Fig. 6 Correlation between standardized, adjusted PRS and odds of disease in reported Asian MGBB participants.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Plots show log(odds) of each of six diseases versus quantile (n = 10) of standardized population structure-adjusted PRS (PRS<sub>std-adj</sub>) among up to 786 MGBB participants of reported Asian race. Results not reported for CRCa, PrCa, or AFib due to 0 cases in at least one quantile. The correlation coefficient, r, is shown in each panel. Abbreviations: AFib, atrial fibrillation; BrCa, breast cancer; CAD, coronary artery disease; CRCa, colorectal cancer; MGBB, Mass General Brigham Biobank; OR, odds ratio; PrCa, prostate cancer; PRS, polygenic risk score; T2D, type 2 diabetes.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 correlation between standardi" href="/articles/s41591-022-01767-6/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig11_ESM.jpg">Extended Data Fig. 7 Correlation between standardized, adjusted PRS and odds of disease in MGBB participants of unknown or other reported race.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Plots show log(odds) of each of six diseases versus quantile (n = 50 for T2D, n = 10 for all other disease) of standardized population structure-adjusted PRS (PRS<sub>std-adj</sub>) among up to 3,113 MGBB participants of unknown or other reported race. Results not reported for CRCa due to 0 cases in at least one quantile (n = 10). The correlation coefficient, r, is shown in each panel. Abbreviations: AFib, atrial fibrillation; BrCa, breast cancer; CAD, coronary artery disease; CRCa, colorectal cancer; MGBB, Mass General Brigham Biobank; OR, odds ratio; PrCa, prostate cancer; PRS, polygenic risk score; T2D, type 2 diabetes.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 difference in standardized, a" href="/articles/s41591-022-01767-6/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig12_ESM.jpg">Extended Data Fig. 8 Difference in standardized, adjusted PRS between WGS and imputed genotyping arrays for 22 individual samples.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>The PRS<sub>std-adj</sub> of 22 samples obtained from WGS and from imputed genotyping arrays are subtracted, and the distribution of the difference of the scores is plotted for each disease. Abbreviations: AFib, atrial fibrillation; BrCa, breast cancer; CAD, coronary artery disease; CRCa, colorectal cancer; IMPU, imputed genotype data; MGBB, Mass General Brigham Biobank; PrCa, prostate cancer; PRS, polygenic risk score; T2D, type 2 diabetes; WGS, whole genome sequencing.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec34-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec34">Supplementary information</h2><div class="c-article-section__content" id="Sec34-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Tables 1–9 and Files 1–3.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Development%20of%20a%20clinical%20polygenic%20risk%20score%20assay%20and%20reporting%20workflow&amp;author=Limin%20Hao%20et%20al&amp;contentID=10.1038%2Fs41591-022-01767-6&amp;copyright=This%20is%20a%20U.S.%20government%20work%20and%20not%20under%20copyright%20protection%20in%20the%20U.S.%3B%20foreign%20copyright%20protection%20may%20apply&amp;publication=1078-8956&amp;publicationDate=2022-04-18&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41591-022-01767-6" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41591-022-01767-6" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Hao, L., Kraft, P., Berriz, G.F. <i>et al.</i> Development of a clinical polygenic risk score assay and reporting workflow.
                    <i>Nat Med</i> <b>28</b>, 1006–1013 (2022). https://doi.org/10.1038/s41591-022-01767-6</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01767-6?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-07-22">22 July 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-03-02">02 March 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-04-18">18 April 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-05">May 2022</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41591-022-01767-6</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-block"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/disease-prevention" data-track="click" data-track-action="view subject" data-track-label="link">Disease prevention</a></li><li class="c-article-subject-list__subject"><a href="/subjects/population-genetics" data-track="click" data-track-action="view subject" data-track-label="link">Population genetics</a></li><li class="c-article-subject-list__subject"><a href="/subjects/risk-factors" data-track="click" data-track-action="view subject" data-track-label="link">Risk factors</a></li>
        </ul>
    </div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section" data-test="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Efficient blockLASSO for polygenic scores with applications to All of Us and UK Biobank" href="https://doi.org/10.1186/s12864-025-11505-0">
                                        Efficient blockLASSO for polygenic scores with applications to All of Us and UK Biobank
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Timothy G. Raben</li><li>Louis Lello</li><li>Stephen D. H. Hsu</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>BMC Genomics</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Polygenic overlap between subjective well-being and psychiatric disorders and cross-ancestry validation" href="https://doi.org/10.1038/s41562-025-02155-z">
                                        Polygenic overlap between subjective well-being and psychiatric disorders and cross-ancestry validation
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Jin Young Jung</li><li>Yeeun Ahn</li><li>Hong-Hee Won</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Human Behaviour</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Instability of high polygenic risk classification and mitigation by integrative scoring" href="https://doi.org/10.1038/s41467-025-56945-0">
                                        Instability of high polygenic risk classification and mitigation by integrative scoring
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Anika Misra</li><li>Buu Truong</li><li>Pradeep Natarajan</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Genome-wide association testing beyond SNPs" href="https://doi.org/10.1038/s41576-024-00778-y">
                                        Genome-wide association testing beyond SNPs
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Laura Harris</li><li>Ellen M. McDonagh</li><li>Tomas Fitzgerald</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Reviews Genetics</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Integrating Polygenic Risk Scores (PRS) for Personalized Diabetes Care: Advancing Clinical Practice with Tailored Pharmacological Approaches" href="https://doi.org/10.1007/s13300-024-01676-6">
                                        Integrating Polygenic Risk Scores (PRS) for Personalized Diabetes Care: Advancing Clinical Practice with Tailored Pharmacological Approaches
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Omna Singh</li><li>Madhur Verma</li><li>Sanjay Kalra</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Diabetes Therapy</i> (2025)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01767-6.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                
            
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
            <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" role="tabpanel"><div class="c-reading-companion__scroll-pane" style="max-height: none;"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Main">Main</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec13" class="c-reading-companion__section-item"><a href="#Sec13" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec14" class="c-reading-companion__section-item"><a href="#Sec14" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-data-availability" class="c-reading-companion__section-item"><a href="#data-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Data availability">Data availability</a></li><li id="rc-sec-code-availability" class="c-reading-companion__section-item"><a href="#code-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Code availability">Code availability</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-peer-review" class="c-reading-companion__section-item"><a href="#peer-review" data-track="click" data-track-action="section anchor" data-track-label="link:Peer review">Peer review</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec33" class="c-reading-companion__section-item"><a href="#Sec33" data-track="click" data-track-action="section anchor" data-track-label="link:Extended data">Extended data</a></li><li id="rc-sec-Sec34" class="c-reading-companion__section-item"><a href="#Sec34" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by">This article is cited by</a></li></ul></div>
                <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="" style="display: none !important;">
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-test="right-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41591-022-01767-6;doi=10.1038/s41591-022-01767-6;subjmeta=208,457,459,499,631,692,700;kwrd=Disease+prevention,Population+genetics,Risk+factors">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=-504124221&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01767-6%26doi%3D10.1038/s41591-022-01767-6%26subjmeta%3D208,457,459,499,631,692,700%26kwrd%3DDisease+prevention,Population+genetics,Risk+factors">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=-504124221&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01767-6%26doi%3D10.1038/s41591-022-01767-6%26subjmeta%3D208,457,459,499,631,692,700%26kwrd%3DDisease+prevention,Population+genetics,Risk+factors"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Fig. 1: Translation of PRS from discovery to the clinic, including a clinical PRS laboratory pipeline for prospectively collected samples.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig1_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig1_HTML.png" alt="figure 1" aria-describedby="rc-Fig1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Fig. 2: Frequency of disease and high-risk PRS results by race in the MGBB.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig2_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig2_HTML.png" alt="figure 2" aria-describedby="rc-Fig2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Fig. 3: PRS distributions by reported race before and after adjustment for population structure.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig3_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig3_HTML.png" alt="figure 3" aria-describedby="rc-Fig3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Fig. 4: Correlation between adjusted PRS and odds of disease.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig4_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig4_HTML.png" alt="figure 4" aria-describedby="rc-Fig4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Extended Data Fig. 1 Distribution of standardized, adjusted PRS by release batch for six diseases in MGBB.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig5_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig5_ESM.jpg" alt="extended data figure 5" aria-describedby="rc-Fig5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig6">Extended Data Fig. 2 Distribution of standardized, adjusted PRS by age for six diseases in MGBB.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig6_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig6_ESM.jpg" alt="extended data figure 6" aria-describedby="rc-Fig6"></picture><p class="c-reading-companion__figure-links"><a href="#Fig6" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/6" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig7">Extended Data Fig. 3 Distribution of adjusted PRS by sex for four diseases in MGBB.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig7_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig7_ESM.jpg" alt="extended data figure 7" aria-describedby="rc-Fig7"></picture><p class="c-reading-companion__figure-links"><a href="#Fig7" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/7" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig8">Extended Data Fig. 4 Correlation between standardized, adjusted PRS and odds of disease in reported white MGBB participants.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig8_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig8_ESM.jpg" alt="extended data figure 8" aria-describedby="rc-Fig8"></picture><p class="c-reading-companion__figure-links"><a href="#Fig8" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/8" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig9">Extended Data Fig. 5 Correlation between standardized, adjusted PRS and odds of disease in reported Black MGBB participants.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig9_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig9_ESM.jpg" alt="extended data figure 9" aria-describedby="rc-Fig9"></picture><p class="c-reading-companion__figure-links"><a href="#Fig9" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/9" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig10">Extended Data Fig. 6 Correlation between standardized, adjusted PRS and odds of disease in reported Asian MGBB participants.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig10_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig10_ESM.jpg" alt="extended data figure 10" aria-describedby="rc-Fig10"></picture><p class="c-reading-companion__figure-links"><a href="#Fig10" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/10" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig11">Extended Data Fig. 7 Correlation between standardized, adjusted PRS and odds of disease in MGBB participants of unknown or other reported race.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig11_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig11_ESM.jpg" alt="extended data figure 11" aria-describedby="rc-Fig11"></picture><p class="c-reading-companion__figure-links"><a href="#Fig11" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/11" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig12">Extended Data Fig. 8 Difference in standardized, adjusted PRS between WGS and imputed genotyping arrays for 22 individual samples.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig12_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01767-6/MediaObjects/41591_2022_1767_Fig12_ESM.jpg" alt="extended data figure 12" aria-describedby="rc-Fig12"></picture><p class="c-reading-companion__figure-links"><a href="#Fig12" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01767-6/figures/12" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric" data-track-context="article references sidebar"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">Shendure, J., Findlay, G. M. &amp; Snyder, M. W. Genomic medicine: progress, pitfalls, and promise. <i>Cell</i> <b>177</b>, 45–57 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2019.02.003" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2019.02.003" data-track-item_id="10.1016/j.cell.2019.02.003">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlvVWgt70%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30901547" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6531313" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20medicine%3A%20progress%2C%20pitfalls%2C%20and%20promise&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2019.02.003&amp;volume=177&amp;pages=45-57&amp;publication_year=2019&amp;author=Shendure%2CJ&amp;author=Findlay%2CGM&amp;author=Snyder%2CMW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2"><i>GWAS Catalog</i> (National Human Genome Research Institute); <a href="https://www.ebi.ac.uk/gwas/" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.ebi.ac.uk/gwas/">https://www.ebi.ac.uk/gwas/</a></p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Meigs, J. B. et al. Genotype score in addition to common risk factors for prediction of type 2 diabetes. <i>N. Engl. J. Med.</i> <b>359</b>, 2208–2219 (2008).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa0804742" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa0804742" data-track-item_id="10.1056/NEJMoa0804742">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXhtl2gurnE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19020323" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2746946" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genotype%20score%20in%20addition%20to%20common%20risk%20factors%20for%20prediction%20of%20type%202%20diabetes&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa0804742&amp;volume=359&amp;pages=2208-2219&amp;publication_year=2008&amp;author=Meigs%2CJB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">Ripatti, S. et al. A multilocus genetic risk score for coronary heart disease: case-control and prospective cohort analyses. <i>Lancet</i> <b>376</b>, 1393–1400 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0140-6736%2810%2961267-6" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0140-6736(10)61267-6" data-track-item_id="10.1016/S0140-6736(10)61267-6">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20971364" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2965351" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20multilocus%20genetic%20risk%20score%20for%20coronary%20heart%20disease%3A%20case-control%20and%20prospective%20cohort%20analyses&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2810%2961267-6&amp;volume=376&amp;pages=1393-1400&amp;publication_year=2010&amp;author=Ripatti%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">Zheng, S. L. et al. Cumulative association of five genetic variants with prostate cancer. <i>N. Engl. J. Med.</i> <b>358</b>, 910–919 (2008).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa075819" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa075819" data-track-item_id="10.1056/NEJMoa075819">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1cXisF2mtrg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=18199855" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Cumulative%20association%20of%20five%20genetic%20variants%20with%20prostate%20cancer&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa075819&amp;volume=358&amp;pages=910-919&amp;publication_year=2008&amp;author=Zheng%2CSL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Boyle, E. A., Li, Y. I. &amp; Pritchard, J. K. An expanded view of complex traits: from polygenic to omnigenic. <i>Cell</i> <b>169</b>, 1177–1186 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2017.05.038" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2017.05.038" data-track-item_id="10.1016/j.cell.2017.05.038">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhtVWrtrjM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28622505" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5536862" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=An%20expanded%20view%20of%20complex%20traits%3A%20from%20polygenic%20to%20omnigenic&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2017.05.038&amp;volume=169&amp;pages=1177-1186&amp;publication_year=2017&amp;author=Boyle%2CEA&amp;author=Li%2CYI&amp;author=Pritchard%2CJK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Khera, A. V. et al. Genome-wide polygenic scores for common diseases identify individuals with risk equivalent to monogenic mutations. <i>Nat. Genet.</i> <b>50</b>, 1219–1224 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-018-0183-z" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-018-0183-z" data-track-item_id="10.1038/s41588-018-0183-z">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhsFSqtbbI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30104762" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6128408" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20polygenic%20scores%20for%20common%20diseases%20identify%20individuals%20with%20risk%20equivalent%20to%20monogenic%20mutations&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0183-z&amp;volume=50&amp;pages=1219-1224&amp;publication_year=2018&amp;author=Khera%2CAV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Vilhjálmsson, B. J. et al. Modeling linkage disequilibrium increases accuracy of polygenic risk scores. <i>Am. J. Hum. Genet.</i> <b>97</b>, 576–592 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ajhg.2015.09.001" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ajhg.2015.09.001" data-track-item_id="10.1016/j.ajhg.2015.09.001">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26430803" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4596916" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhsFKhsbzK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Modeling%20linkage%20disequilibrium%20increases%20accuracy%20of%20polygenic%20risk%20scores&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2015.09.001&amp;volume=97&amp;pages=576-592&amp;publication_year=2015&amp;author=Vilhj%C3%A1lmsson%2CBJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Inouye, M. et al. Genomic risk prediction of coronary artery disease in 480,000 adults: implications for primary prevention. <i>J. Am. Coll. Cardiol.</i> <b>72</b>, 1883–1893 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.jacc.2018.07.079" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.jacc.2018.07.079" data-track-item_id="10.1016/j.jacc.2018.07.079">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30309464" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6176870" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genomic%20risk%20prediction%20of%20coronary%20artery%20disease%20in%20480%2C000%20adults%3A%20implications%20for%20primary%20prevention&amp;journal=J.%20Am.%20Coll.%20Cardiol.&amp;doi=10.1016%2Fj.jacc.2018.07.079&amp;volume=72&amp;pages=1883-1893&amp;publication_year=2018&amp;author=Inouye%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Conti, D. V. et al. Trans-ancestry genome-wide association meta-analysis of prostate cancer identifies new susceptibility loci and informs genetic risk prediction. <i>Nat. Genet.</i> <b>53</b>, 65–75 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-020-00748-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-020-00748-0" data-track-item_id="10.1038/s41588-020-00748-0">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXkslSqsA%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33398198" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8148035" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Trans-ancestry%20genome-wide%20association%20meta-analysis%20of%20prostate%20cancer%20identifies%20new%20susceptibility%20loci%20and%20informs%20genetic%20risk%20prediction&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-020-00748-0&amp;volume=53&amp;pages=65-75&amp;publication_year=2021&amp;author=Conti%2CDV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Klarin, D. et al. Genome-wide association analysis of venous thromboembolism identifies new risk loci and genetic overlap with arterial vascular disease. <i>Nat. Genet.</i> <b>51</b>, 1574–1579 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-019-0519-3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-019-0519-3" data-track-item_id="10.1038/s41588-019-0519-3">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitFSjsrnO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31676865" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6858581" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genome-wide%20association%20analysis%20of%20venous%20thromboembolism%20identifies%20new%20risk%20loci%20and%20genetic%20overlap%20with%20arterial%20vascular%20disease&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0519-3&amp;volume=51&amp;pages=1574-1579&amp;publication_year=2019&amp;author=Klarin%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Mosley, J. D. et al. Predictive accuracy of a polygenic risk score compared with a clinical risk score for incident coronary heart disease. <i>JAMA</i> <b>323</b>, 627–635 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1001%2Fjama.2019.21782" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1001/jama.2019.21782" data-track-item_id="10.1001/jama.2019.21782">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjslGrsLs%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32068817" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7042849" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Predictive%20accuracy%20of%20a%20polygenic%20risk%20score%20compared%20with%20a%20clinical%20risk%20score%20for%20incident%20coronary%20heart%20disease&amp;journal=JAMA&amp;doi=10.1001%2Fjama.2019.21782&amp;volume=323&amp;pages=627-635&amp;publication_year=2020&amp;author=Mosley%2CJD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Vassy, J. L. et al. Polygenic type 2 diabetes prediction at the limit of common variant detection. <i>Diabetes</i> <b>63</b>, 2172–2182 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.2337%2Fdb13-1663" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.2337/db13-1663" data-track-item_id="10.2337/db13-1663">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXhsV2ls7rN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24520119" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4030114" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20type%202%20diabetes%20prediction%20at%20the%20limit%20of%20common%20variant%20detection&amp;journal=Diabetes&amp;doi=10.2337%2Fdb13-1663&amp;volume=63&amp;pages=2172-2182&amp;publication_year=2014&amp;author=Vassy%2CJL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Seibert, T. M. et al. Polygenic hazard score to guide screening for aggressive prostate cancer: development and validation in large scale cohorts. <i>BMJ</i> <b>360</b>, j5757 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1136%2Fbmj.j5757" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1136/bmj.j5757" data-track-item_id="10.1136/bmj.j5757">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29321194" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5759091" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20hazard%20score%20to%20guide%20screening%20for%20aggressive%20prostate%20cancer%3A%20development%20and%20validation%20in%20large%20scale%20cohorts&amp;journal=BMJ&amp;doi=10.1136%2Fbmj.j5757&amp;volume=360&amp;publication_year=2018&amp;author=Seibert%2CTM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">National Human Genome Research Institute (NHGRI). <i>Electronic Medical Records and Genomics (eMERGE) Network</i> <a href="https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE">https://www.genome.gov/Funded-Programs-Projects/Electronic-Medical-Records-and-Genomics-Network-eMERGE</a> (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">Shieh, Y. et al. Breast cancer screening in the precision medicine era: risk-based screening in a population-based trial. <i>J. Natl Cancer Inst</i>. <b>109</b>, <a href="https://doi.org/10.1093/jnci/djw290" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1093/jnci/djw290">https://doi.org/10.1093/jnci/djw290</a> (2017).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Brockman, D. G. et al. Design and user experience testing of a polygenic score report: a qualitative study of prospective users. <i>BMC Med. Genomics</i> <b>14</b>, 238 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s12920-021-01056-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s12920-021-01056-0" data-track-item_id="10.1186/s12920-021-01056-0">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34598685" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8485114" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Design%20and%20user%20experience%20testing%20of%20a%20polygenic%20score%20report%3A%20a%20qualitative%20study%20of%20prospective%20users&amp;journal=BMC%20Med.%20Genomics&amp;doi=10.1186%2Fs12920-021-01056-0&amp;volume=14&amp;publication_year=2021&amp;author=Brockman%2CDG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">Choi, S. W., Mak, T. S.-H. &amp; O’Reilly, P. F. Tutorial: a guide to performing polygenic risk score analyses. <i>Nat. Protoc.</i> <b>15</b>, 2759–2772 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41596-020-0353-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41596-020-0353-1" data-track-item_id="10.1038/s41596-020-0353-1">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsVGltLbI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32709988" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7612115" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Tutorial%3A%20a%20guide%20to%20performing%20polygenic%20risk%20score%20analyses&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fs41596-020-0353-1&amp;volume=15&amp;pages=2759-2772&amp;publication_year=2020&amp;author=Choi%2CSW&amp;author=Mak%2CTS-H&amp;author=O%E2%80%99Reilly%2CPF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">Martin, A. R. et al. Clinical use of current polygenic risk scores may exacerbate health disparities. <i>Nat. Genet.</i> <b>51</b>, 584–591 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-019-0379-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-019-0379-x" data-track-item_id="10.1038/s41588-019-0379-x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXoslGltLw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30926966" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6563838" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20use%20of%20current%20polygenic%20risk%20scores%20may%20exacerbate%20health%20disparities&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-019-0379-x&amp;volume=51&amp;pages=584-591&amp;publication_year=2019&amp;author=Martin%2CAR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Lewis, C. M. &amp; Vassos, E. Polygenic risk scores: from research tools to clinical instruments. <i>Genome Med.</i> <b>12</b>, 44 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13073-020-00742-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13073-020-00742-5" data-track-item_id="10.1186/s13073-020-00742-5">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32423490" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7236300" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%3A%20from%20research%20tools%20to%20clinical%20instruments&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-020-00742-5&amp;volume=12&amp;publication_year=2020&amp;author=Lewis%2CCM&amp;author=Vassos%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Lambert, S. A. et al. The Polygenic Score Catalog as an open database for reproducibility and systematic evaluation. <i>Nat. Genet.</i> <b>53</b>, 420–425 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-021-00783-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-021-00783-5" data-track-item_id="10.1038/s41588-021-00783-5">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmt1entLg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33692568" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Polygenic%20Score%20Catalog%20as%20an%20open%20database%20for%20reproducibility%20and%20systematic%20evaluation&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-021-00783-5&amp;volume=53&amp;pages=420-425&amp;publication_year=2021&amp;author=Lambert%2CSA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Fritsche, L. G. et al. Cancer PRSweb: an online repository with polygenic risk scores for major cancer traits and their evaluation in two independent biobanks. <i>Am. J. Hum. Genet.</i> <b>107</b>, 815–836 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ajhg.2020.08.025" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ajhg.2020.08.025" data-track-item_id="10.1016/j.ajhg.2020.08.025">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhvFKjs7nK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32991828" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7675001" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Cancer%20PRSweb%3A%20an%20online%20repository%20with%20polygenic%20risk%20scores%20for%20major%20cancer%20traits%20and%20their%20evaluation%20in%20two%20independent%20biobanks&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2020.08.025&amp;volume=107&amp;pages=815-836&amp;publication_year=2020&amp;author=Fritsche%2CLG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">Huyghe, J. R. et al. Discovery of common and rare genetic risk variants for colorectal cancer. <i>Nat. Genet.</i> <b>51</b>, 76–87 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-018-0286-6" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-018-0286-6" data-track-item_id="10.1038/s41588-018-0286-6">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXitl2iu7vF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30510241" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Discovery%20of%20common%20and%20rare%20genetic%20risk%20variants%20for%20colorectal%20cancer&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0286-6&amp;volume=51&amp;pages=76-87&amp;publication_year=2019&amp;author=Huyghe%2CJR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Kalia, S. S. et al. Recommendations for reporting of secondary findings in clinical exome and genome sequencing, 2016 update (ACMG SF v2.0): a policy statement of the American College of Medical Genetics and Genomics. <i>Genet. Med.</i> <b>19</b>, 249–255 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fgim.2016.190" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/gim.2016.190" data-track-item_id="10.1038/gim.2016.190">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27854360" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Recommendations%20for%20reporting%20of%20secondary%20findings%20in%20clinical%20exome%20and%20genome%20sequencing%2C%202016%20update%20%28ACMG%20SF%20v2.0%29%3A%20a%20policy%20statement%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fgim.2016.190&amp;volume=19&amp;pages=249-255&amp;publication_year=2017&amp;author=Kalia%2CSS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">Zook, J. M. et al. Extensive sequencing of seven human genomes to characterize benchmark reference materials. <i>Sci. Data</i> <b>3</b>, 160025 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fsdata.2016.25" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/sdata.2016.25" data-track-item_id="10.1038/sdata.2016.25">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XpvVWrtLk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27271295" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4896128" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Extensive%20sequencing%20of%20seven%20human%20genomes%20to%20characterize%20benchmark%20reference%20materials&amp;journal=Sci.%20Data&amp;doi=10.1038%2Fsdata.2016.25&amp;volume=3&amp;publication_year=2016&amp;author=Zook%2CJM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Bowling, K. M. et al. Identifying rare, medically relevant variation via population-based genomic screening in Alabama: opportunities and pitfalls. <i>Genet. Med.</i> <b>23</b>, 280–288 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41436-020-00976-z" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41436-020-00976-z" data-track-item_id="10.1038/s41436-020-00976-z">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVSju73P" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32989269" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Identifying%20rare%2C%20medically%20relevant%20variation%20via%20population-based%20genomic%20screening%20in%20Alabama%3A%20opportunities%20and%20pitfalls&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fs41436-020-00976-z&amp;volume=23&amp;pages=280-288&amp;publication_year=2021&amp;author=Bowling%2CKM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">Weedon, M. N. &amp; Wright, C. F. et al. Use of SNP chips to detect rare pathogenic variants: retrospective, population based diagnostic evaluation. <i>BMJ</i> <b>372</b>, n214 (2021).</p><p class="c-reading-companion__reference-links"><a href="http://scholar.google.com/scholar_lookup?&amp;title=Use%20of%20SNP%20chips%20to%20detect%20rare%20pathogenic%20variants%3A%20retrospective%2C%20population%20based%20diagnostic%20evaluation&amp;journal=BMJ&amp;volume=372&amp;publication_year=2021&amp;author=Weedon%2CMN&amp;author=Wright%2CCF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Scheuner, M. T., Edelen, M. O., Hilborne, L. H. &amp; Lubin, I. M. Effective communication of molecular genetic test results to primary care providers. <i>Genet. Med.</i> <b>15</b>, 444–449 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fgim.2012.151" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/gim.2012.151" data-track-item_id="10.1038/gim.2012.151">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23222660" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Effective%20communication%20of%20molecular%20genetic%20test%20results%20to%20primary%20care%20providers&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fgim.2012.151&amp;volume=15&amp;pages=444-449&amp;publication_year=2013&amp;author=Scheuner%2CMT&amp;author=Edelen%2CMO&amp;author=Hilborne%2CLH&amp;author=Lubin%2CIM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">McLaughlin, H. M. et al. A systematic approach to the reporting of medically relevant findings from whole genome sequencing. <i>BMC Med. Genet.</i> <b>15</b>, 134 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s12881-014-0134-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s12881-014-0134-1" data-track-item_id="10.1186/s12881-014-0134-1">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25714468" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342199" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXitlCntr0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20systematic%20approach%20to%20the%20reporting%20of%20medically%20relevant%20findings%20from%20whole%20genome%20sequencing&amp;journal=BMC%20Med.%20Genet.&amp;doi=10.1186%2Fs12881-014-0134-1&amp;volume=15&amp;publication_year=2014&amp;author=McLaughlin%2CHM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">Farmer, G. D., Gray, H., Chandratillake, G., Raymond, F. L. &amp; Freeman, A. L. J. Recommendations for designing genetic test reports to be understood by patients and non-specialists. <i>Eur. J. Hum. Genet.</i> <b>28</b>, 885–895 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41431-020-0579-y" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41431-020-0579-y" data-track-item_id="10.1038/s41431-020-0579-y">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32024982" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7316722" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Recommendations%20for%20designing%20genetic%20test%20reports%20to%20be%20understood%20by%20patients%20and%20non-specialists&amp;journal=Eur.%20J.%20Hum.%20Genet.&amp;doi=10.1038%2Fs41431-020-0579-y&amp;volume=28&amp;pages=885-895&amp;publication_year=2020&amp;author=Farmer%2CGD&amp;author=Gray%2CH&amp;author=Chandratillake%2CG&amp;author=Raymond%2CFL&amp;author=Freeman%2CALJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Senol-Cosar, O. et al. Considerations for clinical curation, classification, and reporting of low-penetrance and low effect size variants associated with disease risk. <i>Genet. Med.</i> <b>21</b>, 2765–2773 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41436-019-0560-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41436-019-0560-8" data-track-item_id="10.1038/s41436-019-0560-8">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31147632" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Considerations%20for%20clinical%20curation%2C%20classification%2C%20and%20reporting%20of%20low-penetrance%20and%20low%20effect%20size%20variants%20associated%20with%20disease%20risk&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fs41436-019-0560-8&amp;volume=21&amp;pages=2765-2773&amp;publication_year=2019&amp;author=Senol-Cosar%2CO" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Goff, D. C. et al. 2013 ACC/AHA guideline on the assessment of cardiovascular risk: a report of the American College of Cardiology/American Heart Association Task Force on practice guidelines. <i>Circulation</i> <b>129</b>(Suppl. 2), S49–S73 (2014).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24222018" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=2013%20ACC%2FAHA%20guideline%20on%20the%20assessment%20of%20cardiovascular%20risk%3A%20a%20report%20of%20the%20American%20College%20of%20Cardiology%2FAmerican%20Heart%20Association%20Task%20Force%20on%20practice%20guidelines&amp;journal=Circulation&amp;volume=129&amp;issue=Suppl.%202&amp;pages=S49-S73&amp;publication_year=2014&amp;author=Goff%2CDC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">American Diabetes Association. 2. Classification and diagnosis of diabetes: standards of medical care in diabetes – 2021. <i>Diabetes Care</i> <b>44</b>(Suppl. 1), S15–S33 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.2337%2Fdc21-S002" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.2337/dc21-S002" data-track-item_id="10.2337/dc21-S002">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=2.%20Classification%20and%20diagnosis%20of%20diabetes%3A%20standards%20of%20medical%20care%20in%20diabetes%20%E2%80%93%202021&amp;journal=Diabetes%20Care&amp;doi=10.2337%2Fdc21-S002&amp;volume=44&amp;issue=Suppl.%201&amp;pages=S15-S33&amp;publication_year=2021" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">Grossman, D. C. et al. Screening for prostate cancer: US Preventive Services Task Force recommendation statement. <i>JAMA</i> <b>319</b>, 1901–1913 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1001%2Fjama.2018.0161" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1001/jama.2018.0161" data-track-item_id="10.1001/jama.2018.0161">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29801017" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Screening%20for%20prostate%20cancer%3A%20US%20Preventive%20Services%20Task%20Force%20recommendation%20statement&amp;journal=JAMA&amp;doi=10.1001%2Fjama.2018.0161&amp;volume=319&amp;pages=1901-1913&amp;publication_year=2018&amp;author=Grossman%2CDC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">Siu, A. L., US Preventive Services Task Force Screening for breast cancer: US Preventive Services Task Force recommendation statement. <i>Ann. Intern. Med.</i> <b>164</b>, 279–296 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.7326%2FM15-2886" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.7326/M15-2886" data-track-item_id="10.7326/M15-2886">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26757170" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Screening%20for%20breast%20cancer%3A%20US%20Preventive%20Services%20Task%20Force%20recommendation%20statement&amp;journal=Ann.%20Intern.%20Med.&amp;doi=10.7326%2FM15-2886&amp;volume=164&amp;pages=279-296&amp;publication_year=2016&amp;author=Siu%2CAL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Davidson, K. W. et al., US Preventive Services Task Force Screening for colorectal cancer: US Preventive Services Task Force recommendation statement. <i>JAMA</i> <b>325</b>, 1965–1977 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1001%2Fjama.2021.6238" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1001/jama.2021.6238" data-track-item_id="10.1001/jama.2021.6238">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34003218" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Screening%20for%20colorectal%20cancer%3A%20US%20Preventive%20Services%20Task%20Force%20recommendation%20statement&amp;journal=JAMA&amp;doi=10.1001%2Fjama.2021.6238&amp;volume=325&amp;pages=1965-1977&amp;publication_year=2021&amp;author=Davidson%2CKW" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Wand, H. et al. Improving reporting standards for polygenic scores in risk prediction studies. <i>Nature</i> <b>591</b>, 211–219 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-021-03243-6" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-021-03243-6" data-track-item_id="10.1038/s41586-021-03243-6">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmt1ems7Y%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33692554" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8609771" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Improving%20reporting%20standards%20for%20polygenic%20scores%20in%20risk%20prediction%20studies&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03243-6&amp;volume=591&amp;pages=211-219&amp;publication_year=2021&amp;author=Wand%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Homburger, J. R. et al. Low coverage whole genome sequencing enables accurate assessment of common variants and calculation of genome-wide polygenic scores. <i>Genome Med.</i> <b>11</b>, 74 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13073-019-0682-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13073-019-0682-2" data-track-item_id="10.1186/s13073-019-0682-2">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXisVyksbvK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31771638" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6880438" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Low%20coverage%20whole%20genome%20sequencing%20enables%20accurate%20assessment%20of%20common%20variants%20and%20calculation%20of%20genome-wide%20polygenic%20scores&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-019-0682-2&amp;volume=11&amp;publication_year=2019&amp;author=Homburger%2CJR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">Denny, J. C. et al. The ‘All of Us’ Research Program. <i>N. Engl. J. Med.</i> <b>381</b>, 668–676 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMsr1809937" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMsr1809937" data-track-item_id="10.1056/NEJMsr1809937">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31412182" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20%E2%80%98All%20of%20Us%E2%80%99%20Research%20Program&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMsr1809937&amp;volume=381&amp;pages=668-676&amp;publication_year=2019&amp;author=Denny%2CJC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">Khera, A. V. et al. Whole-genome sequencing to characterize monogenic and polygenic contributions in patients hospitalized with early-onset myocardial infarction. <i>Circulation</i> <b>139</b>, 1593–1602 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1161%2FCIRCULATIONAHA.118.035658" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1161/CIRCULATIONAHA.118.035658" data-track-item_id="10.1161/CIRCULATIONAHA.118.035658">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXlslaqsb0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30586733" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6433484" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Whole-genome%20sequencing%20to%20characterize%20monogenic%20and%20polygenic%20contributions%20in%20patients%20hospitalized%20with%20early-onset%20myocardial%20infarction&amp;journal=Circulation&amp;doi=10.1161%2FCIRCULATIONAHA.118.035658&amp;volume=139&amp;pages=1593-1602&amp;publication_year=2019&amp;author=Khera%2CAV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">Dikilitas, O. et al. Predictive utility of polygenic risk scores for coronary heart disease in three major racial and ethnic groups. <i>Am. J. Hum. Genet.</i> <b>106</b>, 707–716 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ajhg.2020.04.002" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ajhg.2020.04.002" data-track-item_id="10.1016/j.ajhg.2020.04.002">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXptFGgurY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32386537" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7212267" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Predictive%20utility%20of%20polygenic%20risk%20scores%20for%20coronary%20heart%20disease%20in%20three%20major%20racial%20and%20ethnic%20groups&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2020.04.002&amp;volume=106&amp;pages=707-716&amp;publication_year=2020&amp;author=Dikilitas%2CO" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">Kowalski, M. H. et al. Use of &gt;100,000 NHLBI Trans-Omics for Precision Medicine (TOPMed) Consortium whole genome sequences improves imputation quality and detection of rare variant associations in admixed African and Hispanic/Latino populations. <i>PLoS Genet.</i> <b>15</b>, e1008500 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pgen.1008500" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pgen.1008500" data-track-item_id="10.1371/journal.pgen.1008500">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31869403" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6953885" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXktlKnsb8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Use%20of%20%3E100%2C000%20NHLBI%20Trans-Omics%20for%20Precision%20Medicine%20%28TOPMed%29%20Consortium%20whole%20genome%20sequences%20improves%20imputation%20quality%20and%20detection%20of%20rare%20variant%20associations%20in%20admixed%20African%20and%20Hispanic%2FLatino%20populations&amp;journal=PLoS%20Genet.&amp;doi=10.1371%2Fjournal.pgen.1008500&amp;volume=15&amp;publication_year=2019&amp;author=Kowalski%2CMH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">Lewis, A. C. F., Green, R. C. &amp; Vassy, J. L. Polygenic risk scores in the clinic: translating risk into action. <i>HGG Adv.</i> <b>2</b>, 100047 (2021).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=35047839" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8756548" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%20in%20the%20clinic%3A%20translating%20risk%20into%20action&amp;journal=HGG%20Adv.&amp;volume=2&amp;publication_year=2021&amp;author=Lewis%2CACF&amp;author=Green%2CRC&amp;author=Vassy%2CJL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">Lee, A. et al. BOADICEA: a comprehensive breast cancer risk prediction model incorporating genetic and nongenetic risk factors. <i>Genet. Med.</i> <b>21</b>, 1708–1718 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41436-018-0406-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41436-018-0406-9" data-track-item_id="10.1038/s41436-018-0406-9">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30643217" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6687499" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=BOADICEA%3A%20a%20comprehensive%20breast%20cancer%20risk%20prediction%20model%20incorporating%20genetic%20and%20nongenetic%20risk%20factors&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fs41436-018-0406-9&amp;volume=21&amp;pages=1708-1718&amp;publication_year=2019&amp;author=Lee%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">Stone, N. J. et al. 2013 ACC/AHA guideline on the treatment of blood cholesterol to reduce atherosclerotic cardiovascular risk in adults: a report of the American College of Cardiology/American Heart Association Task Force on Practice Guidelines. <i>Circulation</i> <b>129</b>(Suppl. 2), S1–S45 (2014).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24222016" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=2013%20ACC%2FAHA%20guideline%20on%20the%20treatment%20of%20blood%20cholesterol%20to%20reduce%20atherosclerotic%20cardiovascular%20risk%20in%20adults%3A%20a%20report%20of%20the%20American%20College%20of%20Cardiology%2FAmerican%20Heart%20Association%20Task%20Force%20on%20Practice%20Guidelines&amp;journal=Circulation&amp;volume=129&amp;issue=Suppl.%202&amp;pages=S1-S45&amp;publication_year=2014&amp;author=Stone%2CNJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR46">Hughes, E. et al. Development and validation of a clinical polygenic risk score to predict breast cancer risk. <i>JCO Precis. Oncol.</i> <b>4</b>, 585–592 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1200%2FPO.19.00360" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1200/PO.19.00360" data-track-item_id="10.1200/PO.19.00360">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Development%20and%20validation%20of%20a%20clinical%20polygenic%20risk%20score%20to%20predict%20breast%20cancer%20risk&amp;journal=JCO%20Precis.%20Oncol.&amp;doi=10.1200%2FPO.19.00360&amp;volume=4&amp;pages=585-592&amp;publication_year=2020&amp;author=Hughes%2CE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR47"><i>Our Health + Ancestry DNA Service - 23andMe</i> (23andMe); <a href="https://www.23andme.com/dna-health-ancestry/" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.23andme.com/dna-health-ancestry/">https://www.23andme.com/dna-health-ancestry/</a></p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR48">Chen, S.-F. et al. Genotype imputation and variability in polygenic risk score estimation. <i>Genome Med.</i> <b>12</b>, 100 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13073-020-00801-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13073-020-00801-x" data-track-item_id="10.1186/s13073-020-00801-x">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33225976" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7682022" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Genotype%20imputation%20and%20variability%20in%20polygenic%20risk%20score%20estimation&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-020-00801-x&amp;volume=12&amp;publication_year=2020&amp;author=Chen%2CS-F" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR49">National Human Genome Research Institute. <i>Polygenic RIsk MEthods in Diverse populations (PRIMED) Consortium</i> <a href="https://www.genome.gov/Funded-Programs-Projects/PRIMED-Consortium" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.genome.gov/Funded-Programs-Projects/PRIMED-Consortium">https://www.genome.gov/Funded-Programs-Projects/PRIMED-Consortium</a></p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR50">Manolio, T. A. Using the data we have: improving diversity in genomic research. <i>Am. J. Hum. Genet.</i> <b>105</b>, 233–236 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ajhg.2019.07.008" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ajhg.2019.07.008" data-track-item_id="10.1016/j.ajhg.2019.07.008">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhsFemtbzM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31374201" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6698933" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20the%20data%20we%20have%3A%20improving%20diversity%20in%20genomic%20research&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2019.07.008&amp;volume=105&amp;pages=233-236&amp;publication_year=2019&amp;author=Manolio%2CTA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR51">Lewis, A. C. F. &amp; Green, R. C. Polygenic risk scores in the clinic: new perspectives needed on familiar ethical issues. <i>Genome Med.</i> <b>13</b>, 14 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13073-021-00829-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13073-021-00829-7" data-track-item_id="10.1186/s13073-021-00829-7">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33509269" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7844961" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%20in%20the%20clinic%3A%20new%20perspectives%20needed%20on%20familiar%20ethical%20issues&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-021-00829-7&amp;volume=13&amp;publication_year=2021&amp;author=Lewis%2CACF&amp;author=Green%2CRC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR52">Ray, T. Myriad Genetics recalibrates breast cancer PRS for all ancestries in anticipation of broader launch. <i>Genomeweb</i> <a href="https://www.genomeweb.com/molecular-diagnostics/myriad-genetics-recalibrates-breast-cancer-prs-all-ancestries-anticipation" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.genomeweb.com/molecular-diagnostics/myriad-genetics-recalibrates-breast-cancer-prs-all-ancestries-anticipation">https://www.genomeweb.com/molecular-diagnostics/myriad-genetics-recalibrates-breast-cancer-prs-all-ancestries-anticipation</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR53">Ambry Product Team. <i>Important discontinuation notice: AmbryScore: polygenic risk scores (PRS</i>) <a href="https://info.ambrygenetics.com/take-a-brief-survey-for-entry-into-amazon-gift-card-drawing" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://info.ambrygenetics.com/take-a-brief-survey-for-entry-into-amazon-gift-card-drawing">https://info.ambrygenetics.com/take-a-brief-survey-for-entry-into-amazon-gift-card-drawing</a></p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR54">Ge, T. et al. Validation of a trans-ancestry polygenic risk score for type 2 diabetes in diverse populations. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2021.09.11.21263413" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.09.11.21263413">https://doi.org/10.1101/2021.09.11.21263413</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR55">Marnetto, D. et al. Ancestry deconvolution and partial polygenic score can improve susceptibility predictions in recently admixed individuals. <i>Nat. Commun.</i> <b>11</b>, 1628 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-15464-w" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-15464-w" data-track-item_id="10.1038/s41467-020-15464-w">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32242022" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7118071" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXmsVSksr4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Ancestry%20deconvolution%20and%20partial%20polygenic%20score%20can%20improve%20susceptibility%20predictions%20in%20recently%20admixed%20individuals&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-15464-w&amp;volume=11&amp;publication_year=2020&amp;author=Marnetto%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR56">Ruan, Y. et al. Improving polygenic prediction in ancestrally diverse populations. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2020.12.27.20248738" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2020.12.27.20248738">https://doi.org/10.1101/2020.12.27.20248738</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR57">Armstrong, K. A. &amp; Metlay, J. P. Annals clinical decision making: translating population evidence to individual patients. <i>Ann. Intern. Med.</i> <b>172</b>, 610–616 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.7326%2FM19-3496" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.7326/M19-3496" data-track-item_id="10.7326/M19-3496">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32311741" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Annals%20clinical%20decision%20making%3A%20translating%20population%20evidence%20to%20individual%20patients&amp;journal=Ann.%20Intern.%20Med.&amp;doi=10.7326%2FM19-3496&amp;volume=172&amp;pages=610-616&amp;publication_year=2020&amp;author=Armstrong%2CKA&amp;author=Metlay%2CJP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR58">Sniderman, A. D., LaChapelle, K. J., Rachon, N. A. &amp; Furberg, C. D. The necessity for clinical reasoning in the era of evidence-based medicine. <i>Mayo Clin. Proc.</i> <b>88</b>, 1108–1114 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.mayocp.2013.07.012" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.mayocp.2013.07.012" data-track-item_id="10.1016/j.mayocp.2013.07.012">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24079680" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20necessity%20for%20clinical%20reasoning%20in%20the%20era%20of%20evidence-based%20medicine&amp;journal=Mayo%20Clin.%20Proc.&amp;doi=10.1016%2Fj.mayocp.2013.07.012&amp;volume=88&amp;pages=1108-1114&amp;publication_year=2013&amp;author=Sniderman%2CAD&amp;author=LaChapelle%2CKJ&amp;author=Rachon%2CNA&amp;author=Furberg%2CCD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR59">Michailidou, K. et al. Association analysis identifies 65 new breast cancer risk loci. <i>Nature</i> <b>551</b>, 92–94 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnature24284" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nature24284" data-track-item_id="10.1038/nature24284">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29059683" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5798588" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXhslajtr3L" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Association%20analysis%20identifies%2065%20new%20breast%20cancer%20risk%20loci&amp;journal=Nature&amp;doi=10.1038%2Fnature24284&amp;volume=551&amp;pages=92-94&amp;publication_year=2017&amp;author=Michailidou%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR60">Mavaddat, N. et al. Polygenic risk scores for prediction of breast cancer and breast cancer subtypes. <i>Am. J. Hum. Genet.</i> <b>104</b>, 21–34 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ajhg.2018.11.002" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ajhg.2018.11.002" data-track-item_id="10.1016/j.ajhg.2018.11.002">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXisFWqsb%2FO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30554720" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Polygenic%20risk%20scores%20for%20prediction%20of%20breast%20cancer%20and%20breast%20cancer%20subtypes&amp;journal=Am.%20J.%20Hum.%20Genet.&amp;doi=10.1016%2Fj.ajhg.2018.11.002&amp;volume=104&amp;pages=21-34&amp;publication_year=2019&amp;author=Mavaddat%2CN" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR61">Schumacher, F. R. et al. Association analyses of more than 140,000 men identify 63 new prostate cancer susceptibility loci. <i>Nat. Genet.</i> <b>50</b>, 928–936 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41588-018-0142-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41588-018-0142-8" data-track-item_id="10.1038/s41588-018-0142-8">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtFWru7jL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29892016" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6568012" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Association%20analyses%20of%20more%20than%20140%2C000%20men%20identify%2063%20new%20prostate%20cancer%20susceptibility%20loci&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fs41588-018-0142-8&amp;volume=50&amp;pages=928-936&amp;publication_year=2018&amp;author=Schumacher%2CFR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR62">Karlson, E. W., Boutin, N. T., Hoffnagle, A. G. &amp; Allen, N. L. Building the Partners Healthcare Biobank at Partners Personalized Medicine: informed consent, return of research results, recruitment lessons and operational considerations. <i>J. Pers. Med.</i> <b>6</b>, 2 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3390%2Fjpm6010002" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3390/jpm6010002" data-track-item_id="10.3390/jpm6010002">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4810381" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Building%20the%20Partners%20Healthcare%20Biobank%20at%20Partners%20Personalized%20Medicine%3A%20informed%20consent%2C%20return%20of%20research%20results%2C%20recruitment%20lessons%20and%20operational%20considerations&amp;journal=J.%20Pers.%20Med.&amp;doi=10.3390%2Fjpm6010002&amp;volume=6&amp;publication_year=2016&amp;author=Karlson%2CEW&amp;author=Boutin%2CNT&amp;author=Hoffnagle%2CAG&amp;author=Allen%2CNL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR63">Yu, S. et al. Toward high-throughput phenotyping: unbiased automated feature extraction and selection from knowledge sources. <i>J. Am. Med. Inform. Assoc.</i> <b>22</b>, 993–1000 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fjamia%2Focv034" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/jamia/ocv034" data-track-item_id="10.1093/jamia/ocv034">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25929596" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4986664" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Toward%20high-throughput%20phenotyping%3A%20unbiased%20automated%20feature%20extraction%20and%20selection%20from%20knowledge%20sources&amp;journal=J.%20Am.%20Med.%20Inform.%20Assoc.&amp;doi=10.1093%2Fjamia%2Focv034&amp;volume=22&amp;pages=993-1000&amp;publication_year=2015&amp;author=Yu%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR64">Yu, S. et al. Enabling phenotypic big data with PheNorm. <i>J. Am. Med. Inform. Assoc.</i> <b>25</b>, 54–60 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fjamia%2Focx111" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/jamia/ocx111" data-track-item_id="10.1093/jamia/ocx111">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29126253" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Enabling%20phenotypic%20big%20data%20with%20PheNorm&amp;journal=J.%20Am.%20Med.%20Inform.%20Assoc.&amp;doi=10.1093%2Fjamia%2Focx111&amp;volume=25&amp;pages=54-60&amp;publication_year=2018&amp;author=Yu%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR65">Gainer, V. S. et al. The Biobank portal for Partners Personalized Medicine: a query tool for working with consented Biobank samples, genotypes, and phenotypes using i2b2. <i>J. Pers. Med.</i> <b>6</b>, 11 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3390%2Fjpm6010011" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3390/jpm6010011" data-track-item_id="10.3390/jpm6010011">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4810390" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20Biobank%20portal%20for%20Partners%20Personalized%20Medicine%3A%20a%20query%20tool%20for%20working%20with%20consented%20Biobank%20samples%2C%20genotypes%2C%20and%20phenotypes%20using%20i2b2&amp;journal=J.%20Pers.%20Med.&amp;doi=10.3390%2Fjpm6010011&amp;volume=6&amp;publication_year=2016&amp;author=Gainer%2CVS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR66">Blau, A., Brown, A., Mahanta, L. &amp; Amr, S. S. The translational genomics core at Partners Personalized Medicine: facilitating the transition of research towards personalized medicine. <i>J. Pers. Med.</i> <b>6</b>, 10 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3390%2Fjpm6010010" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3390/jpm6010010" data-track-item_id="10.3390/jpm6010010">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4810389" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20translational%20genomics%20core%20at%20Partners%20Personalized%20Medicine%3A%20facilitating%20the%20transition%20of%20research%20towards%20personalized%20medicine&amp;journal=J.%20Pers.%20Med.&amp;doi=10.3390%2Fjpm6010010&amp;volume=6&amp;publication_year=2016&amp;author=Blau%2CA&amp;author=Brown%2CA&amp;author=Mahanta%2CL&amp;author=Amr%2CSS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR67">Das, S. et al. Next-generation genotype imputation service and methods. <i>Nat. Genet.</i> <b>48</b>, 1284–1287 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fng.3656" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ng.3656" data-track-item_id="10.1038/ng.3656">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsVWksL%2FK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27571263" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5157836" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Next-generation%20genotype%20imputation%20service%20and%20methods&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3656&amp;volume=48&amp;pages=1284-1287&amp;publication_year=2016&amp;author=Das%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR68">Delaneau, O., Zagury, J.-F. &amp; Marchini, J. Improved whole-chromosome phasing for disease and population genetic studies. <i>Nat. Methods</i> <b>10</b>, 5–6 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnmeth.2307" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nmeth.2307" data-track-item_id="10.1038/nmeth.2307">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvFShtbnN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23269371" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Improved%20whole-chromosome%20phasing%20for%20disease%20and%20population%20genetic%20studies&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fnmeth.2307&amp;volume=10&amp;pages=5-6&amp;publication_year=2013&amp;author=Delaneau%2CO&amp;author=Zagury%2CJ-F&amp;author=Marchini%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR69">Libiger, O. &amp; Schork, N. J. A method for inferring an individual’s genetic ancestry and degree of admixture associated with six major continental populations. <i>Front. Genet.</i> <b>3</b>, 322 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.3389%2Ffgene.2012.00322" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.3389/fgene.2012.00322" data-track-item_id="10.3389/fgene.2012.00322">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23335941" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3543981" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20method%20for%20inferring%20an%20individual%E2%80%99s%20genetic%20ancestry%20and%20degree%20of%20admixture%20associated%20with%20six%20major%20continental%20populations&amp;journal=Front.%20Genet.&amp;doi=10.3389%2Ffgene.2012.00322&amp;volume=3&amp;publication_year=2013&amp;author=Libiger%2CO&amp;author=Schork%2CNJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR70">Miller, D. T. et al. ACMG SF v3.0 list for reporting of secondary findings in clinical exome and genome sequencing: a policy statement of the American College of Medical Genetics and Genomics (ACMG). <i>Genet. Med.</i> <b>23</b>, 1381–1390 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41436-021-01172-3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41436-021-01172-3" data-track-item_id="10.1038/s41436-021-01172-3">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34012068" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ACMG%20SF%20v3.0%20list%20for%20reporting%20of%20secondary%20findings%20in%20clinical%20exome%20and%20genome%20sequencing%3A%20a%20policy%20statement%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics%20%28ACMG%29&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fs41436-021-01172-3&amp;volume=23&amp;pages=1381-1390&amp;publication_year=2021&amp;author=Miller%2CDT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR71">Loh, P.-R. et al. Reference-based phasing using the Haplotype Reference Consortium panel. <i>Nat. Genet.</i> <b>48</b>, 1443–1448 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fng.3679" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/ng.3679" data-track-item_id="10.1038/ng.3679">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28Xhs1SltLzF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27694958" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5096458" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Reference-based%20phasing%20using%20the%20Haplotype%20Reference%20Consortium%20panel&amp;journal=Nat.%20Genet.&amp;doi=10.1038%2Fng.3679&amp;volume=48&amp;pages=1443-1448&amp;publication_year=2016&amp;author=Loh%2CP-R" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR72">Chang, C. C. et al. Second-generation PLINK: rising to the challenge of larger and richer datasets. <i>GigaScience</i> <b>4</b>, 7 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13742-015-0047-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13742-015-0047-8" data-track-item_id="10.1186/s13742-015-0047-8">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25722852" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4342193" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXlsFShsLY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Second-generation%20PLINK%3A%20rising%20to%20the%20challenge%20of%20larger%20and%20richer%20datasets&amp;journal=GigaScience&amp;doi=10.1186%2Fs13742-015-0047-8&amp;volume=4&amp;publication_year=2015&amp;author=Chang%2CCC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR73">Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows-Wheeler transform. <i>Bioinformatics</i> <b>25</b>, 1754–1760 (2009).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp324" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btp324" data-track-item_id="10.1093/bioinformatics/btp324">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19451168" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp324&amp;volume=25&amp;pages=1754-1760&amp;publication_year=2009&amp;author=Li%2CH&amp;author=Durbin%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR74">Poplin, R. et al. Scaling accurate genetic variant discovery to tens of thousands of samples. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/201178" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/201178">https://doi.org/10.1101/201178</a> (2018).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR75">Van der Auwera, G. A. &amp; O’Connor, B. D. <i>Genomics in the Cloud: Using Docker, GATK, and WDL in Terra</i> 1st edn (O’Reilly Media, 2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR76">Shi, L. et al. The MicroArray Quality Control (MAQC)-II study of common practices for the development and validation of microarray-based predictive models. <i>Nat. Biotechnol.</i> <b>28</b>, 827–838 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnbt.1665" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nbt.1665" data-track-item_id="10.1038/nbt.1665">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXpsFWku7w%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=20676074" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20MicroArray%20Quality%20Control%20%28MAQC%29-II%20study%20of%20common%20practices%20for%20the%20development%20and%20validation%20of%20microarray-based%20predictive%20models&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fnbt.1665&amp;volume=28&amp;pages=827-838&amp;publication_year=2010&amp;author=Shi%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR77">Richards, S. et al. Standards and guidelines for the interpretation of sequence variants: a joint consensus recommendation of the American College of Medical Genetics and Genomics and the Association for Molecular Pathology. <i>Genet. Med.</i> <b>17</b>, 405–424 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fgim.2015.30" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/gim.2015.30" data-track-item_id="10.1038/gim.2015.30">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25741868" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4544753" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Standards%20and%20guidelines%20for%20the%20interpretation%20of%20sequence%20variants%3A%20a%20joint%20consensus%20recommendation%20of%20the%20American%20College%20of%20Medical%20Genetics%20and%20Genomics%20and%20the%20Association%20for%20Molecular%20Pathology&amp;journal=Genet.%20Med.&amp;doi=10.1038%2Fgim.2015.30&amp;volume=17&amp;pages=405-424&amp;publication_year=2015&amp;author=Richards%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR78">Rivera‐Muñoz, E. A. et al. ClinGen Variant Curation Expert Panel experiences and standardized processes for disease and gene‐level specification of the ACMG/AMP guidelines for sequence variant interpretation. <i>Hum. Mutat.</i> <b>39</b>, 1614–1622 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1002%2Fhumu.23645" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1002/humu.23645" data-track-item_id="10.1002/humu.23645">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30311389" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6225902" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=ClinGen%20Variant%20Curation%20Expert%20Panel%20experiences%20and%20standardized%20processes%20for%20disease%20and%20gene%E2%80%90level%20specification%20of%20the%20ACMG%2FAMP%20guidelines%20for%20sequence%20variant%20interpretation&amp;journal=Hum.%20Mutat.&amp;doi=10.1002%2Fhumu.23645&amp;volume=39&amp;pages=1614-1622&amp;publication_year=2018&amp;author=Rivera%E2%80%90Mu%C3%B1oz%2CEA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li></ol></div></div>
        </div>
    </div>
</aside>
</div>


    
        
    
    
        
            
        

        
            
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>




<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Medicine
                    </span>
                    (<i itemprop="alternateName">Nat Med</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-170X</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1078-8956</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/" data-track="click" data-track-action="protocols.io" data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/authors/editorial_policies" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nature-research/open-access" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/language-editing/" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/" data-track="click" data-track-action="research solutions" data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library" data-track="click" data-track-action="Recommend to library" data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica" data-track="click" data-track-action="nature africa" data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly" data-track="click" data-track-action="nature Italy" data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies" data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice" data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/accessibility-statement" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20">
    </a>
    <p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"></path></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"></path><clipPath id="b"><use xlink:href="#a" style="overflow:visible"></use></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"></path><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"></path><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"></path><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"></path></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"></path></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif " data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Flagship">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing" src="/static/images/logos/nature-briefing-logo-n150-white-d81c9da3ec.svg" width="250" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing</em> newsletter — what matters in science, free to your inbox daily.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/briefing" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Flagship">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="MainBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="MainBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required="" data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:nature_briefing_daily" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required="">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Flagship">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get the most important science stories of the day, free in your inbox.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="https://www.nature.com/briefing/signup/?brieferEntryPoint=MainBriefingBanner">Sign up for Nature Briefing
            </a>
        </div>

    </div>

</div>






<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41591-022-01767-6&amp;format=js&amp;last_modified=2022-04-18" async=""></script>

<qcw-lghvuzrboxbm></qcw-lghvuzrboxbm><div data-cc-ghost="" style="height: 352px;"></div><script type="text/javascript" id="fetch-contextual-ads-campaign-data" charset="">(function(){var g=function(a){a=new CustomEvent("campaignDataLoaded",{detail:a});document.dispatchEvent(a)},b=google_tag_manager["rm"]["50443292"](18);if(b){var k=1500;b=google_tag_manager["rm"]["50443292"](19);b="link"===b?google_tag_manager["rm"]["50443292"](20):google_tag_manager["rm"]["50443292"](21);var m=function(a,c){var h=!1,l=setTimeout(function(){h=!0;c(null)},k),d=new XMLHttpRequest;d.onload=function(){var e=null,f=[];if(200===d.status){try{e=JSON.parse(d.responseText)}catch(p){e={}}f=e.campaigns||[]}h||(clearTimeout(l),c(f.length?f.join(","):null))};d.open("GET",
"/platform/contextual?doi\x3d"+a);d.send()},n=function(a,c){a&&-1===a.indexOf("not set")?m(a,c):setTimeout(c,1)};n(b,function(a){window.campaignsForContextualAds=a;g(a);window.dataLayer.push({content:{article:{campaignID:a}}})})}else window.campaignsForContextualAds="",g(null)})();</script>
<script type="text/javascript" id="gtm-setup-accessdetailsloaded-handler" charset="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail||{};var b={event:"update-access-details"};a.institutional_business_partner_ids&&a.resolved_by||console.log("BPID data could not be retrieved from /exposed-details");a.logged_in_to_sn_profile||console.log("logged_in_to_sn_profile could not be retrieved from /exposed-details");var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?
a.resolved_by.join(";"):"",e=a.personal_business_partner_id_found,f=a.logged_in_to_sn_profile,g=a.snid;b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null,personal:e||null,logged_in_to_sn_profile:f,snid:g||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy={};window.dataLayer.push(b);window.idpUserDataLoaded=!0},!1);window.dataLayer.push({event:"accessdetailsloaded-handler-added"});
window.accessDetailsLoadedHandlerAdded=!0;</script><script type="text/javascript" id="gtm-mediatrust" charset="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="crossmark-script" text="" charset="" type="text/javascript" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="polyfill-matches" charset="">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="fetch-idp-user-data" charset="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},site=google_tag_manager["rm"]["50443292"](59),idpUrl;
switch(site){case "nature":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?"https://staging-idp.nature.com/exposed-details":"https://idp.nature.com/exposed-details";break;case "link":idpUrl=-1<window.location.hostname.indexOf("link-qa")?"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details";break;case "springer":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?
"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details"}if(void 0!==idpUrl){var transport=new XMLHttpRequest;transport.open("GET",idpUrl,!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script src="https://verify.nature.com/verify/nature.min.js"></script><script type="text/javascript" id="create-ad-slots" charset="">function createAndLoadAds(){function h(b){"nature"===a&&google_tag_manager["rm"]["50443292"](87)(b);"bmc"!==a&&"springeropen"!==a||google_tag_manager["rm"]["50443292"](99)(b);("link"===a&&"oscar"===google_tag_manager["rm"]["50443292"](100)||"springer"===a||"link"===a&&!0===google_tag_manager["rm"]["50443292"](101))&&google_tag_manager["rm"]["50443292"](126)(b);"link"===a&&"bunsen"===google_tag_manager["rm"]["50443292"](127)&&google_tag_manager["rm"]["50443292"](152)(b);"link"===a&&"Core"===google_tag_manager["rm"]["50443292"](153)&&google_tag_manager["rm"]["50443292"](165)(b)}window.googletag=window.googletag||{cmd:[]};var a=google_tag_manager["rm"]["50443292"](166),e=google_tag_manager["rm"]["50443292"](168);
if(-1===window.location.search.indexOf("hide_ads\x3dtrue"))if(window.adSlots&&0!==Object.keys(window.adSlots).length)console.log("Ads previously loaded. Will not update ad slots until next page load.");else{window.adSlots||(window.adSlots={});window.getAd=function(b,f){for(var c in window.adSlots)if(-1<c.indexOf(b)){if("object"===f)return adSlots[c];if("slot"===f)return adSlots[c].slot}};googletag.cmd.push(function(){googletag.pubads().setPrivacySettings({limitedAds:e});console.log("limitedAds is ",
e);googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["rm"]["50443292"](170));googletag.pubads().enableSingleRequest();googletag.pubads().disableInitialLoad()});for(var g=document.querySelectorAll("[data-gpt]"),d=0;g[d];++d)h(g[d]);googletag.cmd.push(function(){googletag.enableServices()})}}createAndLoadAds();</script>
</body></html>