<html lang="en" class="js"><head><link rel="preload" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" as="font" type="font/woff2" crossorigin=""><script async="" src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script type="text/javascript" async="" src="https://pagead2.googlesyndication.com/tag/js/gpt.js"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-MRVXSHQ"></script><script type="text/javascript" src="https://www.nature.com/d8Cxh3AinHhGSP4jO4JY36mAFGhiO1Myb_RzSPX1oLMovp9k5sFPQLp17DkrVTV4lcKA7E-IPN_1hQY4H9cxQg=="></script><script src="https://cmp.nature.com/production_live/en/consent-bundle-8-85.js" onload="initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')"></script>
    <title>Multiplexed CRISPR-based microfluidic platform for clinical testing of respiratory viruses and identification of SARS-CoV-2 variants | Nature Medicine</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nm.rss">


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin="">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480">

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"biotechnology;infectious-disease-diagnostics","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Medicine","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41591-022-01734-1"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Nicole L. Welch","Meilin Zhu","Catherine Hua","Juliane Weller","Marzieh Ezzaty Mirhashemi","Tien G. Nguyen","Sreekar Mantena","Matthew R. Bauer","Bennett M. Shaw","Cheri M. Ackerman","Sri Gowtham Thakku","Megan W. Tse","Jared Kehe","Marie-Martine Uwera","Jacqueline S. Eversley","Derek A. Bielwaski","Graham McGrath","Joseph Braidt","Jeremy Johnson","Felecia Cerrato","Gage K. Moreno","Lydia A. Krasilnikova","Brittany A. Petros","Gabrielle L. Gionet","Ewa King","Richard C. Huard","Samantha K. Jalbert","Michael L. Cleary","Nicholas A. Fitzgerald","Stacey B. Gabriel","Glen R. Gallagher","Sandra C. Smole","Lawrence C. Madoff","Catherine M. Brown","Matthew W. Keller","Malania M. Wilson","Marie K. Kirby","John R. Barnes","Daniel J. Park","Katherine J. Siddle","Christian T. Happi","Deborah T. Hung","Michael Springer","Bronwyn L. MacInnis","Jacob E. Lemieux","Eric Rosenberg","John A. Branda","Paul C. Blainey","Pardis C. Sabeti","Cameron Myhrvold"],"publishedAt":1644192000,"publishedAtString":"2022-02-07","title":"Multiplexed CRISPR-based microfluidic platform for clinical testing of respiratory viruses and identification of SARS-CoV-2 variants","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Biotechnology,Infectious-disease diagnostics"},"journal":{"pcode":"nm","title":"nature medicine","volume":"28","issue":"5","id":41591,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"BR","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{font-weight:700;-webkit-font-smoothing:antialiased;font-size:1.5rem;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.app-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{margin:4px 4px 0;fill:#888;height:10px;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-c9121fecab.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png">
<link rel="icon" type="image/png" sizes="48x48" href="/static/images/favicons/nature/favicon-48x48-b52890008c.png">
<link rel="icon" type="image/png" sizes="32x32" href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="/static/images/favicons/nature/favicon-16x16-951651ab72.png">
<link rel="manifest" href="/static/manifest.json" crossorigin="use-credentials">
<link rel="mask-icon" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="/static/images/favicons/nature/favicon.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-e9d2061c95.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-1cbf3a8b56.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script><script id="js-position1" data-test="global-article-js" defer="" src="/static/js/global-article-es6-bundle-c8a573ca90.js"></script><script id="js-position2" data-test="shared-js" defer="" src="/static/js/shared-es6-bundle-e9d2061c95.js"></script><script id="js-position3" data-test="header-150-js" defer="" src="/static/js/header-150-es6-bundle-5bb959eaa1.js"></script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Multiplexed CRISPR-based microfluidic platform for clinical testing of respiratory viruses and identification of SARS-CoV-2 variants","description":"The coronavirus disease 2019 (COVID-19) pandemic has demonstrated a clear need for high-throughput, multiplexed and sensitive assays for detecting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and other respiratory viruses and their emerging variants. Here, we present a cost-effective virus and variant detection platform, called microfluidic Combinatorial Arrayed Reactions for Multiplexed Evaluation of Nucleic acids (mCARMEN), which combines CRISPR-based diagnostics and microfluidics with a streamlined workflow for clinical use. We developed the mCARMEN respiratory virus panel to test for up to 21 viruses, including SARS-CoV-2, other coronaviruses and both influenza strains, and demonstrated its diagnostic-grade performance on 525 patient specimens in an academic setting and 166 specimens in a clinical setting. We further developed an mCARMEN panel to enable the identification of 6 SARS-CoV-2 variant lineages, including Delta and Omicron, and evaluated it on 2,088 patient specimens with near-perfect concordance to sequencing-based variant classification. Lastly, we implemented a combined Cas13 and Cas12 approach that enables quantitative measurement of SARS-CoV-2 and influenza A viral copies in samples. The mCARMEN platform enables high-throughput surveillance of multiple viruses and variants simultaneously, enabling rapid detection of SARS-CoV-2 variants. A microfluidic CRISPR-based platform enables multiplexed and high-throughput identification of a wide range of respiratory viruses, including the ability to identify specific SARS-CoV-2 variants such as Delta and Omicron.","datePublished":"2022-02-07T00:00:00Z","dateModified":"2023-11-09T00:00:00Z","pageStart":"1083","pageEnd":"1094","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41591-022-01734-1","keywords":["Biotechnology","Infectious-disease diagnostics","Biomedicine","general","Cancer Research","Metabolic Diseases","Infectious Diseases","Molecular Medicine","Neurosciences"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig4_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig5_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig6_HTML.png"],"isPartOf":{"name":"Nature Medicine","issn":["1546-170X","1078-8956"],"volumeNumber":"28","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Nicole L. Welch","url":"http://orcid.org/0000-0002-1163-7002","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School","address":{"name":"Harvard Program in Virology, Division of Medical Sciences, Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"nicole_welch@g.harvard.edu","@type":"Person"},{"name":"Meilin Zhu","url":"http://orcid.org/0000-0003-1302-4020","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts Institute of Technology","address":{"name":"Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Catherine Hua","url":"http://orcid.org/0000-0003-0455-5364","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Juliane Weller","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Wellcome Sanger Institute, Wellcome Genome Campus","address":{"name":"Wellcome Sanger Institute, Wellcome Genome Campus, Hinxton, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marzieh Ezzaty Mirhashemi","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Tien G. Nguyen","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sreekar Mantena","url":"http://orcid.org/0000-0002-8834-1461","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Matthew R. Bauer","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School","address":{"name":"Harvard Program in Biological and Biomedical Sciences, Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Bennett M. Shaw","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Cheri M. Ackerman","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts Institute of Technology","address":{"name":"Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sri Gowtham Thakku","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School and Massachusetts Institute of Technology","address":{"name":"Division of Health Sciences and Technology, Harvard Medical School and Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Megan W. Tse","url":"http://orcid.org/0000-0002-4968-7072","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts Institute of Technology","address":{"name":"Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jared Kehe","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts Institute of Technology","address":{"name":"Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marie-Martine Uwera","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jacqueline S. Eversley","affiliation":[{"name":"Massachusetts General Hospital","address":{"name":"Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Derek A. Bielwaski","affiliation":[{"name":"Massachusetts General Hospital","address":{"name":"Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Graham McGrath","affiliation":[{"name":"Massachusetts General Hospital","address":{"name":"Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Joseph Braidt","affiliation":[{"name":"Massachusetts General Hospital","address":{"name":"Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jeremy Johnson","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Felecia Cerrato","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Gage K. Moreno","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Lydia A. Krasilnikova","url":"http://orcid.org/0000-0003-4163-706X","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard University","address":{"name":"Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Brittany A. Petros","url":"http://orcid.org/0000-0003-2110-7475","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School and Massachusetts Institute of Technology","address":{"name":"Division of Health Sciences and Technology, Harvard Medical School and Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School","address":{"name":"Harvard/Massachusetts Institute of Technology MD-PhD Program, Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School","address":{"name":"Department of Systems Biology, Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Gabrielle L. Gionet","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ewa King","affiliation":[{"name":"Rhode Island Department of Health","address":{"name":"State Health Laboratories, Rhode Island Department of Health, Providence, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Richard C. Huard","affiliation":[{"name":"Rhode Island Department of Health","address":{"name":"State Health Laboratories, Rhode Island Department of Health, Providence, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Samantha K. Jalbert","affiliation":[{"name":"Harvard Medical School","address":{"name":"Department of Systems Biology, Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michael L. Cleary","affiliation":[{"name":"Harvard Medical School","address":{"name":"Department of Systems Biology, Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Nicholas A. Fitzgerald","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Stacey B. Gabriel","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Glen R. Gallagher","url":"http://orcid.org/0000-0001-6837-9473","affiliation":[{"name":"Massachusetts Department of Public Health","address":{"name":"Massachusetts Department of Public Health, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sandra C. Smole","affiliation":[{"name":"Massachusetts Department of Public Health","address":{"name":"Massachusetts Department of Public Health, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Lawrence C. Madoff","url":"http://orcid.org/0000-0003-2589-7777","affiliation":[{"name":"Massachusetts Department of Public Health","address":{"name":"Massachusetts Department of Public Health, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Catherine M. Brown","affiliation":[{"name":"Massachusetts Department of Public Health","address":{"name":"Massachusetts Department of Public Health, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Matthew W. Keller","url":"http://orcid.org/0000-0002-5850-1698","affiliation":[{"name":"Centers for Disease Control and Prevention","address":{"name":"Influenza Division, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention, Atlanta, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Malania M. Wilson","affiliation":[{"name":"Centers for Disease Control and Prevention","address":{"name":"Influenza Division, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention, Atlanta, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Marie K. Kirby","url":"http://orcid.org/0000-0003-0859-9604","affiliation":[{"name":"Centers for Disease Control and Prevention","address":{"name":"Influenza Division, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention, Atlanta, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"John R. Barnes","affiliation":[{"name":"Centers for Disease Control and Prevention","address":{"name":"Influenza Division, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention, Atlanta, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Daniel J. Park","url":"http://orcid.org/0000-0001-7226-7781","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Katherine J. Siddle","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard University","address":{"name":"Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Christian T. Happi","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Redeemer’s University","address":{"name":"African Centre of Excellence for Genomics of Infectious Diseases, Redeemer’s University, Ede, Nigeria","@type":"PostalAddress"},"@type":"Organization"},{"name":"Redeemer’s University","address":{"name":"Department of Biological Sciences, College of Natural Sciences, Redeemer’s University, Ede, Nigeria","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Deborah T. Hung","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Molecular Biology Department and Center for Computational and Integrative Biology, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Michael Springer","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard Medical School","address":{"name":"Department of Systems Biology, Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Bronwyn L. MacInnis","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard University","address":{"name":"Department of Immunology and Infectious Disease, Harvard T.H. Chan School of Public Health, Harvard University, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jacob E. Lemieux","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital","address":{"name":"Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Eric Rosenberg","affiliation":[{"name":"Massachusetts General Hospital","address":{"name":"Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts General Hospital and Harvard Medical School","address":{"name":"Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"John A. Branda","affiliation":[{"name":"Massachusetts General Hospital and Harvard Medical School","address":{"name":"Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Paul C. Blainey","url":"http://orcid.org/0000-0002-4889-8783","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Massachusetts Institute of Technology","address":{"name":"Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Koch Institute for Integrative Cancer Research at Massachusetts Institute of Technology","address":{"name":"Koch Institute for Integrative Cancer Research at Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Pardis C. Sabeti","url":"http://orcid.org/0000-0002-9843-1890","affiliation":[{"name":"Broad Institute of MIT and Harvard","address":{"name":"Broad Institute of MIT and Harvard, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard University","address":{"name":"Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Harvard University","address":{"name":"Department of Immunology and Infectious Disease, Harvard T.H. Chan School of Public Health, Harvard University, Boston, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Koch Institute for Integrative Cancer Research at Massachusetts Institute of Technology","address":{"name":"Koch Institute for Integrative Cancer Research at Massachusetts Institute of Technology, Cambridge, USA","@type":"PostalAddress"},"@type":"Organization"},{"name":"Howard Hughes Medical Institute","address":{"name":"Howard Hughes Medical Institute, Chevy Chase, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"pardis@broadinstitute.org","@type":"Person"},{"name":"Cameron Myhrvold","url":"http://orcid.org/0000-0002-8971-184X","affiliation":[{"name":"Princeton University","address":{"name":"Department of Molecular Biology, Princeton University, Princeton, USA","@type":"PostalAddress"},"@type":"Organization"}],"email":"cmyhrvol@princeton.edu","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41591-022-01734-1">
    
    
    <meta name="journal_id" content="41591">
    <meta name="dc.title" content="Multiplexed CRISPR-based microfluidic platform for clinical testing of respiratory viruses and identification of SARS-CoV-2 variants">
    <meta name="dc.source" content="Nature Medicine 2022 28:5">
    <meta name="dc.format" content="text/html">
    <meta name="dc.publisher" content="Nature Publishing Group">
    <meta name="dc.date" content="2022-02-07">
    <meta name="dc.type" content="OriginalPaper">
    <meta name="dc.language" content="En">
    <meta name="dc.copyright" content="2022 The Author(s)">
    <meta name="dc.rights" content="2022 The Author(s)">
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="dc.description" content="The coronavirus disease 2019 (COVID-19) pandemic has demonstrated a clear need for high-throughput, multiplexed and sensitive assays for detecting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and other respiratory viruses and their emerging variants. Here, we present a cost-effective virus and variant detection platform, called microfluidic Combinatorial Arrayed Reactions for Multiplexed Evaluation of Nucleic acids (mCARMEN), which combines CRISPR-based diagnostics and microfluidics with a streamlined workflow for clinical use. We developed the mCARMEN respiratory virus panel to test for up to 21 viruses, including SARS-CoV-2, other coronaviruses and both influenza strains, and demonstrated its diagnostic-grade performance on 525 patient specimens in an academic setting and 166 specimens in a clinical setting. We further developed an mCARMEN panel to enable the identification of 6 SARS-CoV-2 variant lineages, including Delta and Omicron, and evaluated it on 2,088 patient specimens with near-perfect concordance to sequencing-based variant classification. Lastly, we implemented a combined Cas13 and Cas12 approach that enables quantitative measurement of SARS-CoV-2 and influenza A viral copies in samples. The mCARMEN platform enables high-throughput surveillance of multiple viruses and variants simultaneously, enabling rapid detection of SARS-CoV-2 variants. A microfluidic CRISPR-based platform enables multiplexed and high-throughput identification of a wide range of respiratory viruses, including the ability to identify specific SARS-CoV-2 variants such as Delta and Omicron.">
    <meta name="prism.issn" content="1546-170X">
    <meta name="prism.publicationName" content="Nature Medicine">
    <meta name="prism.publicationDate" content="2022-02-07">
    <meta name="prism.volume" content="28">
    <meta name="prism.number" content="5">
    <meta name="prism.section" content="OriginalPaper">
    <meta name="prism.startingPage" content="1083">
    <meta name="prism.endingPage" content="1094">
    <meta name="prism.copyright" content="2022 The Author(s)">
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="prism.url" content="https://www.nature.com/articles/s41591-022-01734-1">
    <meta name="prism.doi" content="doi:10.1038/s41591-022-01734-1">
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41591-022-01734-1.pdf">
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41591-022-01734-1">
    <meta name="citation_journal_title" content="Nature Medicine">
    <meta name="citation_journal_abbrev" content="Nat Med">
    <meta name="citation_publisher" content="Nature Publishing Group">
    <meta name="citation_issn" content="1546-170X">
    <meta name="citation_title" content="Multiplexed CRISPR-based microfluidic platform for clinical testing of respiratory viruses and identification of SARS-CoV-2 variants">
    <meta name="citation_volume" content="28">
    <meta name="citation_issue" content="5">
    <meta name="citation_publication_date" content="2022/05">
    <meta name="citation_online_date" content="2022/02/07">
    <meta name="citation_firstpage" content="1083">
    <meta name="citation_lastpage" content="1094">
    <meta name="citation_article_type" content="Article">
    <meta name="citation_fulltext_world_readable" content="">
    <meta name="citation_language" content="en">
    <meta name="dc.identifier" content="doi:10.1038/s41591-022-01734-1">
    <meta name="DOI" content="10.1038/s41591-022-01734-1">
    <meta name="size" content="273249">
    <meta name="citation_doi" content="10.1038/s41591-022-01734-1">
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41591-022-01734-1&amp;api_key=">
    <meta name="description" content="The coronavirus disease 2019 (COVID-19) pandemic has demonstrated a clear need for high-throughput, multiplexed and sensitive assays for detecting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and other respiratory viruses and their emerging variants. Here, we present a cost-effective virus and variant detection platform, called microfluidic Combinatorial Arrayed Reactions for Multiplexed Evaluation of Nucleic acids (mCARMEN), which combines CRISPR-based diagnostics and microfluidics with a streamlined workflow for clinical use. We developed the mCARMEN respiratory virus panel to test for up to 21 viruses, including SARS-CoV-2, other coronaviruses and both influenza strains, and demonstrated its diagnostic-grade performance on 525 patient specimens in an academic setting and 166 specimens in a clinical setting. We further developed an mCARMEN panel to enable the identification of 6 SARS-CoV-2 variant lineages, including Delta and Omicron, and evaluated it on 2,088 patient specimens with near-perfect concordance to sequencing-based variant classification. Lastly, we implemented a combined Cas13 and Cas12 approach that enables quantitative measurement of SARS-CoV-2 and influenza A viral copies in samples. The mCARMEN platform enables high-throughput surveillance of multiple viruses and variants simultaneously, enabling rapid detection of SARS-CoV-2 variants. A microfluidic CRISPR-based platform enables multiplexed and high-throughput identification of a wide range of respiratory viruses, including the ability to identify specific SARS-CoV-2 variants such as Delta and Omicron.">
    <meta name="dc.creator" content="Welch, Nicole L.">
    <meta name="dc.creator" content="Zhu, Meilin">
    <meta name="dc.creator" content="Hua, Catherine">
    <meta name="dc.creator" content="Weller, Juliane">
    <meta name="dc.creator" content="Mirhashemi, Marzieh Ezzaty">
    <meta name="dc.creator" content="Nguyen, Tien G.">
    <meta name="dc.creator" content="Mantena, Sreekar">
    <meta name="dc.creator" content="Bauer, Matthew R.">
    <meta name="dc.creator" content="Shaw, Bennett M.">
    <meta name="dc.creator" content="Ackerman, Cheri M.">
    <meta name="dc.creator" content="Thakku, Sri Gowtham">
    <meta name="dc.creator" content="Tse, Megan W.">
    <meta name="dc.creator" content="Kehe, Jared">
    <meta name="dc.creator" content="Uwera, Marie-Martine">
    <meta name="dc.creator" content="Eversley, Jacqueline S.">
    <meta name="dc.creator" content="Bielwaski, Derek A.">
    <meta name="dc.creator" content="McGrath, Graham">
    <meta name="dc.creator" content="Braidt, Joseph">
    <meta name="dc.creator" content="Johnson, Jeremy">
    <meta name="dc.creator" content="Cerrato, Felecia">
    <meta name="dc.creator" content="Moreno, Gage K.">
    <meta name="dc.creator" content="Krasilnikova, Lydia A.">
    <meta name="dc.creator" content="Petros, Brittany A.">
    <meta name="dc.creator" content="Gionet, Gabrielle L.">
    <meta name="dc.creator" content="King, Ewa">
    <meta name="dc.creator" content="Huard, Richard C.">
    <meta name="dc.creator" content="Jalbert, Samantha K.">
    <meta name="dc.creator" content="Cleary, Michael L.">
    <meta name="dc.creator" content="Fitzgerald, Nicholas A.">
    <meta name="dc.creator" content="Gabriel, Stacey B.">
    <meta name="dc.creator" content="Gallagher, Glen R.">
    <meta name="dc.creator" content="Smole, Sandra C.">
    <meta name="dc.creator" content="Madoff, Lawrence C.">
    <meta name="dc.creator" content="Brown, Catherine M.">
    <meta name="dc.creator" content="Keller, Matthew W.">
    <meta name="dc.creator" content="Wilson, Malania M.">
    <meta name="dc.creator" content="Kirby, Marie K.">
    <meta name="dc.creator" content="Barnes, John R.">
    <meta name="dc.creator" content="Park, Daniel J.">
    <meta name="dc.creator" content="Siddle, Katherine J.">
    <meta name="dc.creator" content="Happi, Christian T.">
    <meta name="dc.creator" content="Hung, Deborah T.">
    <meta name="dc.creator" content="Springer, Michael">
    <meta name="dc.creator" content="MacInnis, Bronwyn L.">
    <meta name="dc.creator" content="Lemieux, Jacob E.">
    <meta name="dc.creator" content="Rosenberg, Eric">
    <meta name="dc.creator" content="Branda, John A.">
    <meta name="dc.creator" content="Blainey, Paul C.">
    <meta name="dc.creator" content="Sabeti, Pardis C.">
    <meta name="dc.creator" content="Myhrvold, Cameron">
    <meta name="dc.subject" content="Biotechnology">
    <meta name="dc.subject" content="Infectious-disease diagnostics">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=COVID-19 testing: one size does not fit all; citation_author=MJ Mina, KG Andersen; citation_volume=371; citation_publication_date=2021; citation_pages=126-127; citation_doi=10.1126/science.abe9187; citation_id=CR1">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=SARS-CoV-2 transmission without symptoms; citation_author=AL Rasmussen, SV Popescu; citation_volume=371; citation_publication_date=2021; citation_pages=1206-1207; citation_doi=10.1126/science.abf9569; citation_id=CR2">
    <meta name="citation_reference" content="citation_journal_title=J. Clin. Microbiol.; citation_title=Negative nasopharyngeal and oropharyngeal swabs do not rule out COVID-19; citation_author=P Winichakoon; citation_volume=58; citation_publication_date=2020; citation_pages=e00297-20; citation_doi=10.1128/JCM.00297-20; citation_id=CR3">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=False negative tests for SARS-CoV-2 infection—challenges and implications; citation_author=S Woloshin, N Patel, AS Kesselheim; citation_volume=383; citation_publication_date=2020; citation_pages=e38; citation_doi=10.1056/NEJMp2015897; citation_id=CR4">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Phylogenetic analysis of SARS-CoV-2 in Boston highlights the impact of superspreading events; citation_author=JE Lemieux; citation_volume=371; citation_publication_date=2021; citation_pages=eabe326; citation_doi=10.1126/science.abe3261; citation_id=CR5">
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Microbiol.; citation_title=SARS-CoV-2 variants, spike mutations and immune escape; citation_author=WT Harvey; citation_volume=19; citation_publication_date=2021; citation_pages=409-424; citation_doi=10.1038/s41579-021-00573-0; citation_id=CR6">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Possible future waves of SARS-CoV-2 infection generated by variants of concern with a range of characteristics; citation_author=L Dyson; citation_volume=12; citation_publication_date=2021; citation_pages=5730; citation_doi=10.1038/s41467-021-25915-7; citation_id=CR7">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=SARS-CoV-2 viral load in upper respiratory specimens of infected patients; citation_author=L Zou; citation_volume=382; citation_publication_date=2020; citation_pages=1177-1179; citation_doi=10.1056/NEJMc2001737; citation_id=CR8">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Virological assessment of hospitalized patients with COVID-2019; citation_author=R Wölfel; citation_volume=581; citation_publication_date=2020; citation_pages=465-469; citation_doi=10.1038/s41586-020-2196-x; citation_id=CR9">
    <meta name="citation_reference" content="citation_journal_title=PLoS Biol.; citation_title=Multiplex qPCR discriminates variants of concern to enhance global surveillance of SARS-CoV-2; citation_author=CBF Vogels; citation_volume=19; citation_publication_date=2021; citation_pages=e3001236; citation_doi=10.1371/journal.pbio.3001236; citation_id=CR10">
    <meta name="citation_reference" content="citation_journal_title=Sci. Adv.; citation_title=Multiplexed, quantitative serological profiling of COVID-19 from blood by a point-of-care test; citation_author=JT Heggestad; citation_volume=7; citation_publication_date=2021; citation_pages=eabg4901; citation_doi=10.1126/sciadv.abg4901; citation_id=CR11">
    <meta name="citation_reference" content="citation_journal_title=J. Clin. Microbiol.; citation_title=Performance characteristics of a high-throughput automated transcription-mediated amplification test for SARS-CoV-2 detection; citation_author=J Pham; citation_volume=58; citation_publication_date=2020; citation_pages=e01669-20; citation_doi=10.1128/JCM.01669-20; citation_id=CR12">
    <meta name="citation_reference" content="citation_journal_title=Diagn. Microbiol. Infect. Dis.; citation_title=Evaluation of the BioFire® COVID-19 test and Respiratory Panel 2.1 for rapid identification of SARS-CoV-2 in nasopharyngeal swab samples; citation_author=EJ Eckbo; citation_volume=99; citation_publication_date=2021; citation_pages=115260; citation_doi=10.1016/j.diagmicrobio.2020.115260; citation_id=CR13">
    <meta name="citation_reference" content="citation_journal_title=Nat. Microbiol.; citation_title=SARS-CoV-2 variants of interest and concern naming scheme conducive for global discourse; citation_author=F Konings; citation_volume=6; citation_publication_date=2021; citation_pages=821-823; citation_doi=10.1038/s41564-021-00932-w; citation_id=CR14">
    <meta name="citation_reference" content="citation_journal_title=Clin. Microbiol. Infect.; citation_title=Multiplex PCR system for the rapid diagnosis of respiratory virus infection: systematic review and meta-analysis; citation_author=H-S Huang; citation_volume=24; citation_publication_date=2018; citation_pages=1055-1063; citation_doi=10.1016/j.cmi.2017.11.018; citation_id=CR15">
    <meta name="citation_reference" content="citation_journal_title=Anal. Chem.; citation_title=Robust multichannel encoding for highly multiplexed quantitative PCR; citation_author=L Jacky; citation_volume=93; citation_publication_date=2021; citation_pages=4208-4216; citation_doi=10.1021/acs.analchem.0c04626; citation_id=CR16">
    <meta name="citation_reference" content="citation_journal_title=Lancet; citation_title=Understanding variants of SARS-CoV-2; citation_author=T Burki; citation_volume=397; citation_publication_date=2021; citation_pages=462; citation_doi=10.1016/S0140-6736(21)00298-1; citation_id=CR17">
    <meta name="citation_reference" content="citation_journal_title=Euro Surveill.; citation_title=Tracking SARS-CoV-2 lineage B.1.1.7 dissemination: insights from nationwide spike gene target failure (SGTF) and spike gene late detection (SGTL) data, Portugal, week 49 2020 to week 3 2021; citation_author=V Borges; citation_volume=26; citation_publication_date=2021; citation_pages=2100131; citation_doi=10.2807/1560-7917.ES.2021.26.10.2100130; citation_id=CR18">
    <meta name="citation_reference" content="citation_journal_title=Cancer Lett.; citation_title=Next-generation sequencing in the clinic: promises and challenges; citation_author=J Xuan, Y Yu, T Qing, L Guo, L Shi; citation_volume=340; citation_publication_date=2013; citation_pages=284-295; citation_doi=10.1016/j.canlet.2012.11.025; citation_id=CR19">
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Microbiol.; citation_title=Clinical and biological insights from viral genome sequencing; citation_author=CJ Houldcroft, MA Beale, J Breuer; citation_volume=15; citation_publication_date=2017; citation_pages=183-192; citation_doi=10.1038/nrmicro.2016.182; citation_id=CR20">
    <meta name="citation_reference" content="citation_journal_title=Clin. Chem.; citation_title=Metagenomic analysis reveals clinical SARS-CoV-2 infection and bacterial or viral superinfection and colonization; citation_author=V Peddu; citation_volume=66; citation_publication_date=2020; citation_pages=966-972; citation_doi=10.1093/clinchem/hvaa106; citation_id=CR21">
    <meta name="citation_reference" content="Brito, A. F. et al. Global disparities in SARS-CoV-2 genomic surveillance. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.08.21.21262393
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biomed. Eng.; citation_title=CRISPR-based diagnostics; citation_author=MM Kaminski, OO Abudayyeh, JS Gootenberg, F Zhang, JJ Collins; citation_volume=5; citation_publication_date=2021; citation_pages=643-656; citation_doi=10.1038/s41551-021-00760-7; citation_id=CR23">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Noncanonical crRNAs derived from host transcripts enable multiplexable RNA detection by Cas9; citation_author=C Jiao; citation_volume=372; citation_publication_date=2021; citation_pages=941-948; citation_doi=10.1126/science.abe7106; citation_id=CR24">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6; citation_author=JS Gootenberg; citation_volume=360; citation_publication_date=2018; citation_pages=439-444; citation_doi=10.1126/science.aaq0179; citation_id=CR25">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity; citation_author=JS Chen; citation_volume=360; citation_publication_date=2018; citation_pages=436-439; citation_doi=10.1126/science.aar6245; citation_id=CR26">
    <meta name="citation_reference" content="citation_journal_title=Cell Discov.; citation_title=CRISPR-Cas12a-assisted nucleic acid detection; citation_author=S-Y Li; citation_volume=4; citation_publication_date=2018; citation_pages=20; citation_doi=10.1038/s41421-018-0028-z; citation_id=CR27">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector; citation_author=OO Abudayyeh; citation_volume=353; citation_publication_date=2016; citation_pages=aaf5573; citation_doi=10.1126/science.aaf5573; citation_id=CR28">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Nucleic acid detection with CRISPR-Cas13a/C2c2; citation_author=JS Gootenberg; citation_volume=356; citation_publication_date=2017; citation_pages=438-442; citation_doi=10.1126/science.aam9321; citation_id=CR29">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Field-deployable viral diagnostics using CRISPR-Cas13; citation_author=C Myhrvold; citation_volume=360; citation_publication_date=2018; citation_pages=444-448; citation_doi=10.1126/science.aas8836; citation_id=CR30">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Streamlined inactivation, amplification, and Cas13-based detection of SARS-CoV-2; citation_author=J Arizti-Sanz; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-19097-x; citation_id=CR31">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotechnol.; citation_title=CRISPR–Cas12-based detection of SARS-CoV-2; citation_author=JP Broughton; citation_volume=38; citation_publication_date=2020; citation_pages=870-874; citation_doi=10.1038/s41587-020-0513-4; citation_id=CR32">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biomed. Eng.; citation_title=Clinical validation of a Cas13-based assay for the detection of SARS-CoV-2 RNA; citation_author=M Patchsung; citation_volume=4; citation_publication_date=2020; citation_pages=1140-1149; citation_doi=10.1038/s41551-020-00603-x; citation_id=CR33">
    <meta name="citation_reference" content="citation_journal_title=Nat. Chem. Biol.; citation_title=Accelerated RNA detection using tandem CRISPR nucleases; citation_author=TY Liu; citation_volume=17; citation_publication_date=2021; citation_pages=982-988; citation_doi=10.1038/s41589-021-00842-2; citation_id=CR34">
    <meta name="citation_reference" content="citation_journal_title=Nat. Biotech.; citation_title=Designing sensitive viral diagnostics with machine learning; citation_author=HC Metsky; citation_publication_date=2022; citation_doi=10.1038/s41587-022-01213-5; citation_id=CR35">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Detection of SARS-CoV-2 with SHERLOCK one-pot testing; citation_author=J Joung; citation_volume=383; citation_publication_date=2020; citation_pages=1492-1494; citation_doi=10.1056/NEJMc2026172; citation_id=CR36">
    <meta name="citation_reference" content="citation_journal_title=Trends Biotechnol.; citation_title=CRISPR/Cas powered multiplexed biosensing; citation_author=R Bruch, GA Urban, C Dincer; citation_volume=37; citation_publication_date=2019; citation_pages=791-792; citation_doi=10.1016/j.tibtech.2019.04.005; citation_id=CR37">
    <meta name="citation_reference" content="citation_journal_title=Biosens. Bioelectron.; citation_title=Exploiting the orthogonal CRISPR-Cas12a/Cas13a trans-cleavage for dual-gene virus detection using a handheld device; citation_author=T Tian, Z Qiu, Y Jiang, D Zhu, X Zhou; citation_volume=196; citation_publication_date=2022; citation_pages=113701; citation_doi=10.1016/j.bios.2021.113701; citation_id=CR38">
    <meta name="citation_reference" content="citation_journal_title=Biosens. Bioelectron.; citation_title=CRISPR-powered electrochemical microfluidic multiplexed biosensor for target amplification-free miRNA diagnostics; citation_author=R Bruch; citation_volume=177; citation_publication_date=2021; citation_pages=112887; citation_doi=10.1016/j.bios.2020.112887; citation_id=CR39">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Massively multiplexed nucleic acid detection with Cas13; citation_author=CM Ackerman; citation_volume=582; citation_publication_date=2020; citation_pages=277-282; citation_doi=10.1038/s41586-020-2279-8; citation_id=CR40">
    <meta name="citation_reference" content="Crowe, J. E. Jr. Human respiratory viruses. Reference Module in Biomedical Sciences 
                  https://doi.org/10.1016/B978-0-12-801238-3.02600-3
                  
                 (2014).">
    <meta name="citation_reference" content="García Fernández, X., Álvarez-Argüelles, M. E., Rojo, S. &amp; de-Oña, M. Stability of viral RNA in clinical specimens for viral diagnosis. Enferm. Infecc. Microbiol. Clin. (Engl. Ed.) 38, 297–298 (2020).">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Sequencing and analyses of all known human rhinovirus genomes reveal structure and evolution; citation_author=AC Palmenberg; citation_volume=324; citation_publication_date=2009; citation_pages=55-59; citation_doi=10.1126/science.1165557; citation_id=CR43">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=Real-time PCR in virology; citation_author=IM Mackay, KE Arden, A Nitsche; citation_volume=30; citation_publication_date=2002; citation_pages=1292-1305; citation_doi=10.1093/nar/30.6.1292; citation_id=CR44">
    <meta name="citation_reference" content="Pulliam, J. R. C. et al. Increased risk of SARS-CoV-2 reinfection associated with emergence of the Omicron variant in South Africa. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.11.11.21266068
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Euro. Surveill.; citation_title=Outbreak caused by the SARS-CoV-2 Omicron variant in Norway, November to December 2021; citation_author=LT Brandal; citation_volume=26; citation_publication_date=2021; citation_pages=2101147; citation_doi=10.2807/1560-7917.ES.2021.26.50.2101147; citation_id=CR46">
    <meta name="citation_reference" content="citation_journal_title=MMWR Morb. Mortal. Wkly. Rep.; citation_title=SARS-CoV-2 B.1.1.529 (Omicron) Variant—United States, December 1–8, 2021; citation_author=; citation_volume=70; citation_publication_date=2021; citation_pages=1731-1734; citation_doi=10.15585/mmwr.mm7050e1; citation_id=CR47">
    <meta name="citation_reference" content="Li, A., Maier, A., Carter, M. &amp; Guan, T. H. Omicron and S-gene target failure cases in the highest COVID-19 case rate region in Canada—December 2021. J. Med. Virol. 
                  https://doi.org/10.1002/jmv.27562
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Pathology; citation_title=The impact of viral transport media on PCR assay results for the detection of nucleic acid from SARS-CoV-2; citation_author=PD Kirkland, MJ Frost; citation_volume=52; citation_publication_date=2020; citation_pages=811-814; citation_doi=10.1016/j.pathol.2020.09.013; citation_id=CR49">
    <meta name="citation_reference" content="citation_journal_title=Cell; citation_title=Amplification-free detection of SARS-CoV-2 with CRISPR–Cas13a and mobile phone microscopy; citation_author=P Fozouni; citation_volume=184; citation_publication_date=2021; citation_pages=323-333.e9; citation_doi=10.1016/j.cell.2020.12.001; citation_id=CR50">
    <meta name="citation_reference" content="citation_journal_title=iScience; citation_title=Kinetic analysis of Cas12a and Cas13a RNA-guided nucleases for development of improved CRISPR-based diagnostics; citation_author=EA Nalefski; citation_volume=24; citation_publication_date=2021; citation_pages=102996; citation_doi=10.1016/j.isci.2021.102996; citation_id=CR51">
    <meta name="citation_reference" content="Thakku, S. G. et al. Multiplexed detection of bacterial nucleic acids using Cas13 in droplet microarrays. Preprint at bioRxiv 
                  https://doi.org/10.1101/2021.11.12.468388
                  
                 (2021).">
    <meta name="citation_reference" content="citation_journal_title=Nat. Commun.; citation_title=Deployable CRISPR-Cas13a diagnostic tools to detect and report Ebola and Lassa virus cases in real-time; citation_author=KG Barnes; citation_volume=11; citation_publication_date=2020; citation_doi=10.1038/s41467-020-17994-9; citation_id=CR53">
    <meta name="citation_reference" content="citation_journal_title=Mol. Biol. Evol.; citation_title=MAFFT multiple sequence alignment software version 7: improvements in performance and usability; citation_author=K Katoh, DM Standley; citation_volume=30; citation_publication_date=2013; citation_pages=772-780; citation_doi=10.1093/molbev/mst010; citation_id=CR54">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=Enhanced methods for unbiased deep sequencing of Lassa and Ebola RNA viruses from clinical and biological samples; citation_author=CB Matranga; citation_volume=15; citation_publication_date=2014; citation_doi=10.1186/s13059-014-0519-7; citation_id=CR55">
    <meta name="citation_author" content="Welch, Nicole L.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Harvard Program in Virology, Division of Medical Sciences, Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Zhu, Meilin">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author" content="Hua, Catherine">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Weller, Juliane">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Wellcome Sanger Institute, Wellcome Genome Campus, Hinxton, UK">
    <meta name="citation_author" content="Mirhashemi, Marzieh Ezzaty">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Nguyen, Tien G.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Mantena, Sreekar">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Bauer, Matthew R.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Harvard Program in Biological and Biomedical Sciences, Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Shaw, Bennett M.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Ackerman, Cheri M.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author" content="Thakku, Sri Gowtham">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Division of Health Sciences and Technology, Harvard Medical School and Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author" content="Tse, Megan W.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author" content="Kehe, Jared">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author" content="Uwera, Marie-Martine">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Eversley, Jacqueline S.">
    <meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Bielwaski, Derek A.">
    <meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="McGrath, Graham">
    <meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Braidt, Joseph">
    <meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Johnson, Jeremy">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Cerrato, Felecia">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Moreno, Gage K.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Krasilnikova, Lydia A.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, USA">
    <meta name="citation_author" content="Petros, Brittany A.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Division of Health Sciences and Technology, Harvard Medical School and Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author_institution" content="Harvard/Massachusetts Institute of Technology MD-PhD Program, Harvard Medical School, Boston, USA">
    <meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Gionet, Gabrielle L.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="King, Ewa">
    <meta name="citation_author_institution" content="State Health Laboratories, Rhode Island Department of Health, Providence, USA">
    <meta name="citation_author" content="Huard, Richard C.">
    <meta name="citation_author_institution" content="State Health Laboratories, Rhode Island Department of Health, Providence, USA">
    <meta name="citation_author" content="Jalbert, Samantha K.">
    <meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Cleary, Michael L.">
    <meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Fitzgerald, Nicholas A.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Gabriel, Stacey B.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Gallagher, Glen R.">
    <meta name="citation_author_institution" content="Massachusetts Department of Public Health, Boston, USA">
    <meta name="citation_author" content="Smole, Sandra C.">
    <meta name="citation_author_institution" content="Massachusetts Department of Public Health, Boston, USA">
    <meta name="citation_author" content="Madoff, Lawrence C.">
    <meta name="citation_author_institution" content="Massachusetts Department of Public Health, Boston, USA">
    <meta name="citation_author" content="Brown, Catherine M.">
    <meta name="citation_author_institution" content="Massachusetts Department of Public Health, Boston, USA">
    <meta name="citation_author" content="Keller, Matthew W.">
    <meta name="citation_author_institution" content="Influenza Division, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention, Atlanta, USA">
    <meta name="citation_author" content="Wilson, Malania M.">
    <meta name="citation_author_institution" content="Influenza Division, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention, Atlanta, USA">
    <meta name="citation_author" content="Kirby, Marie K.">
    <meta name="citation_author_institution" content="Influenza Division, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention, Atlanta, USA">
    <meta name="citation_author" content="Barnes, John R.">
    <meta name="citation_author_institution" content="Influenza Division, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention, Atlanta, USA">
    <meta name="citation_author" content="Park, Daniel J.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author" content="Siddle, Katherine J.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, USA">
    <meta name="citation_author" content="Happi, Christian T.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="African Centre of Excellence for Genomics of Infectious Diseases, Redeemer’s University, Ede, Nigeria">
    <meta name="citation_author_institution" content="Department of Biological Sciences, College of Natural Sciences, Redeemer’s University, Ede, Nigeria">
    <meta name="citation_author" content="Hung, Deborah T.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Molecular Biology Department and Center for Computational and Integrative Biology, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Springer, Michael">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Systems Biology, Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="MacInnis, Bronwyn L.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Immunology and Infectious Disease, Harvard T.H. Chan School of Public Health, Harvard University, Boston, USA">
    <meta name="citation_author" content="Lemieux, Jacob E.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author" content="Rosenberg, Eric">
    <meta name="citation_author_institution" content="Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, USA">
    <meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Branda, John A.">
    <meta name="citation_author_institution" content="Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, USA">
    <meta name="citation_author" content="Blainey, Paul C.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research at Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author" content="Sabeti, Pardis C.">
    <meta name="citation_author_institution" content="Broad Institute of MIT and Harvard, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, USA">
    <meta name="citation_author_institution" content="Department of Immunology and Infectious Disease, Harvard T.H. Chan School of Public Health, Harvard University, Boston, USA">
    <meta name="citation_author_institution" content="Koch Institute for Integrative Cancer Research at Massachusetts Institute of Technology, Cambridge, USA">
    <meta name="citation_author_institution" content="Howard Hughes Medical Institute, Chevy Chase, USA">
    <meta name="citation_author" content="Myhrvold, Cameron">
    <meta name="citation_author_institution" content="Department of Molecular Biology, Princeton University, Princeton, USA">
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">
    <meta name="twitter:site" content="@naturemedicine">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image:alt" content="Content cover image">
    <meta name="twitter:title" content="Multiplexed CRISPR-based microfluidic platform for clinical testing of respiratory viruses and identification of SARS-CoV-2 variants">
    <meta name="twitter:description" content="Nature Medicine - A microfluidic CRISPR-based platform enables multiplexed and high-throughput identification of a wide range of respiratory viruses, including the ability to identify specific...">
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig1_HTML.png">
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41591-022-01734-1">
    <meta property="og:type" content="article">
    <meta property="og:site_name" content="Nature">
    <meta property="og:title" content="Multiplexed CRISPR-based microfluidic platform for clinical testing of respiratory viruses and identification of SARS-CoV-2 variants - Nature Medicine">
    <meta property="og:description" content="A microfluidic CRISPR-based platform enables multiplexed and high-throughput identification of a wide range of respiratory viruses, including the ability to identify specific SARS-CoV-2 variants such as Delta and Omicron.">
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig1_HTML.png">
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
<style type="text/css">.cc-banner{background-color:#01324b;border:none!important;bottom:0;box-sizing:border-box;color:#fff!important;left:0;line-height:1.3;margin:auto 0 0;max-width:100%;outline:0;overflow:visible;padding:0;position:fixed;right:0;width:100%;z-index:99999}.cc-banner::backdrop{background-color:#0000004d}.cc-banner *{color:inherit!important}.cc-banner:focus{box-shadow:none;outline:0}.cc-banner a{color:#fff!important;text-decoration:underline}.cc-banner a:active,.cc-banner a:focus,.cc-banner a:hover{color:inherit;text-decoration:none}.cc-banner a:focus{outline:3px solid #08c!important}.cc-banner h2,.cc-banner h3,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-banner .cc-h2,.cc-banner h2{font-size:18px}.cc-banner .cc-h3,.cc-banner h3{font-size:16px}.cc-banner .cc-h4,.cc-banner .cc-h5,.cc-banner .cc-h6,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-size:14px}.cc-banner .cc-button{font-size:16px}.cc-banner__content{background-color:#01324b;display:flex;flex-direction:column;margin:0 auto;max-height:90vh;max-width:100%;padding:16px;position:relative}.cc-banner__content:focus{outline:0}@media (min-width:680px){.cc-banner__content{padding:12px}}@media (min-width:980px){.cc-banner__content{max-height:60vh;padding-bottom:20px;padding-top:20px}}@media (min-width:1320px){.cc-banner__content{padding-bottom:40px;padding-top:40px}}.cc-banner__container{display:flex;flex-direction:column;margin:auto;max-width:1320px;overflow:auto}.cc-banner__title{background:none!important;border:0;flex-shrink:0;font-size:18px!important;font-size:22px!important;font-weight:700!important;letter-spacing:normal;margin:0 0 12px!important;padding:0}@media (min-width:680px){.cc-banner__title{font-size:24px!important;margin:0 0 16px!important}}@media (min-width:1320px){.cc-banner__title{font-size:26px!important;margin:0 0 24px!important}}.cc-banner__body{display:flex;flex-direction:column;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif!important;overflow-x:hidden;overflow-y:auto;padding:3px 3px 16px}@media (min-width:980px){.cc-banner__body{flex-direction:row}}.cc-banner__policy p{font-size:16px!important;margin:0;max-width:none;padding:0}.cc-banner__policy p:not(:last-child){margin:0 0 16px}@media (min-width:980px){.cc-banner__policy p:not(:last-child){margin:0 0 24px}}.cc-banner__policy p a{font-size:16px!important;font-weight:700}.cc-banner__footer{box-shadow:none;flex-shrink:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif !important!important;margin:0;padding:12px 0 3px 3px;position:relative}@media (min-width:380px){.cc-banner__footer{align-items:stretch;display:flex;flex-wrap:wrap}}@media (min-width:680px){.cc-banner__footer{box-shadow:none;flex-wrap:nowrap;width:40%}}.cc-banner__footer .cc-banner__button{flex:1 1 auto;font-weight:700;overflow:hidden;padding:.5em 1em;width:100%}@media (min-width:680px){.cc-banner__footer .cc-banner__button{max-width:275px}}.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:12px}@media (min-width:680px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:0;margin-right:16px}}@media (min-width:980px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-right:24px}}@media (min-width:680px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:16px!important;margin-right:0!important}}@media (min-width:980px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:24px!important;margin-right:0!important}}.cc-banner__button-preferences{padding-left:0;padding-right:0}@media (min-width:380px){.cc-banner__button-preferences{flex:0 0 auto;margin:auto}}@media (min-width:680px){.cc-banner__button-preferences{margin:0}}@media (min-width:380px) and (max-width:680px){.cc-banner__button-break{display:block}}@media (min-width:680px){.cc-banner--is-tcf .cc-banner__footer{width:auto}}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__footer{padding-right:48px;width:66%}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px!important}.cc-banner--is-tcf .cc-banner__stacks{margin:16px 0 0}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__stacks{display:flex;flex:0 0 33%;flex-direction:column;margin:0 0 0 48px;overflow:auto}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px;margin:0 0 12px}.cc-banner--is-tcf .cc-banner__stacks-details{padding:0 3px}.cc-banner--is-tcf .cc-banner .cc-details{border-color:#fff3!important}.cc-box{border-radius:10px;padding:12px}.cc-box--info{background-color:#eff6fb}.cc-box--light{background-color:#faf9f6}.cc-button{border:2px solid #0000;border-radius:32px!important;color:inherit;cursor:pointer;font-size:14px;font-weight:700!important;left:auto;line-height:1.2;margin:0;padding:.5em 1em;right:auto;text-transform:none!important;transition:all .2s}@media (min-width:680px){.cc-button{font-size:14px;padding:.75em 1em}}.cc-button--sm{font-size:12px;padding:2px 8px!important}.cc-button--primary{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--primary:focus{outline:3px solid #08c}.cc-button--primary:focus,.cc-button--primary:hover,.cc-button--secondary{background-color:#fff!important;border-color:#025e8d!important;color:#025e8d!important}.cc-button--secondary:focus{background-color:#025e8d!important;border-color:#fff!important;color:#fff!important;outline:3px solid #08c!important}.cc-button--secondary:hover{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--secondary:active{border:2px solid #01324b!important}.cc-button--secondary.cc-button--contrast{border-color:#fff!important}.cc-button--secondary.cc-button--contrast:hover{background-color:initial!important;color:#fff!important}.cc-button--tertiary{background-color:#f0f7fc!important;background-image:none;border:2px solid #f0f7fc;color:#025e8d!important;text-decoration:underline!important;text-underline-offset:.25em}.cc-button--tertiary:focus{outline:3px solid #08c!important}.cc-button--tertiary:hover{text-decoration-thickness:.25em}.cc-button--tertiary:active{color:#013c5b}.cc-button--link{background-color:initial!important;border-color:#0000!important;border-radius:0!important;color:inherit!important;padding:0!important;text-decoration:underline!important;width:inherit!important}.cc-button--link:focus{outline:3px solid #08c}.cc-button--link:hover{background-color:initial!important;box-shadow:none;text-decoration:none}.cc-button--text{border-radius:0;padding:0}.cc-button--details{padding-right:24px!important;position:relative;width:auto}.cc-button--details:after,.cc-button--details:before{background-color:currentColor;content:"";position:absolute;transition:transform .25s ease-out}.cc-button--details:before{height:10px;margin-top:-5px;right:12px;top:50%;width:2px}.cc-button--details:after{height:2px;margin-top:-1px;right:8px;top:50%;width:10px}.cc-button--details.cc-active:before{transform:rotate(90deg);transform-origin:center}.cc-button--details.cc-active:after{display:none}.cc-details{border-bottom:1px solid #0000001a;padding:12px 0;position:relative;width:100%}.cc-details__summary{align-items:center;display:flex;font-size:14px;font-weight:700;list-style-type:none}.cc-details__summary:focus,.cc-details__summary:focus-visible{outline:3px solid #08c!important;will-change:auto}.cc-details__title{align-items:baseline;display:flex}.cc-details__title h2,.cc-details__title h3,.cc-details__title h4,.cc-details__title h5{line-height:1.4;margin:0!important;padding:0!important}.cc-details__title svg{flex-shrink:0;height:auto;margin-right:8px;position:relative;top:-1px;transition:all .2s;width:auto}.cc-details[open] .cc-details__title svg{top:2px;transform:rotate(90deg)}.cc-details__switch{margin-left:auto}.cc-details__section{padding:16px 0 0 18px}.cc-details__section p{margin:0}.cc-details__section p:not(:last-child){margin-bottom:12px}details summary::-webkit-details-marker{display:none}.cc-radio{align-items:center;display:flex;position:relative}.cc-radio *{cursor:pointer}.cc-radio__input{height:22px;left:0;position:absolute;top:0;width:22px}.cc-radio__input:focus{outline:none}.cc-radio__label{color:inherit;font-size:14px;font-weight:700;line-height:23px;margin:0;padding-left:28px}.cc-radio__label:after,.cc-radio__label:before{background-color:#fff;content:"";display:block;position:absolute;transition:transform .25s ease-out}.cc-radio__label:before{border:1px solid #777;border-radius:50%;height:22px;left:0;top:0;width:22px}.cc-radio__label:after{border:7px solid #025e8d;border-radius:50%;height:0;left:4px;opacity:0;top:4px;width:0}.cc-radio__input:focus+.cc-radio__label:before{box-shadow:0 0 0 2px #08c;outline:none}.cc-radio__label--hidden{display:none}.cc-radio__input:checked+.cc-radio__label:after{opacity:1}.cc-radio__input:disabled{cursor:default}.cc-radio__input:disabled+.cc-radio__label{cursor:default;opacity:.5}.cc-switch *{cursor:pointer}.cc-switch{align-items:center;display:flex;line-height:1}.cc-switch__label{cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;-webkit-user-select:none;user-select:none}.cc-switch__input[type=checkbox]{-webkit-appearance:none;appearance:none;background-color:#fff!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='rgba(0, 0, 0, 0.25)'/%3E%3C/svg%3E")!important;background-position:0!important;background-repeat:no-repeat!important;border:1px solid #777!important;border-radius:16px!important;height:18px!important;margin:0 8px 0 0!important;transition:background-position .2s ease-in-out!important;width:32px!important}.cc-switch__input[type=checkbox]:focus{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%2380b3cc'/%3E%3C/svg%3E")!important}.cc-switch__input[type=checkbox]:checked{background-color:#025e8d!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%23fff'/%3E%3C/svg%3E")!important;background-position:100%!important}.cc-switch__input[type=checkbox]:checked~.cc-switch__label>.cc-switch__text-off,.cc-switch__input[type=checkbox]:not(:checked)~.cc-switch__label>.cc-switch__text-on{display:none}.cc-switch__input[type=checkbox]:hover{cursor:pointer;outline:none}.cc-switch__input[type=checkbox]:focus-visible{cursor:pointer;outline:3px solid #fc0}.cc-switch__input[type=checkbox]:checked:before{background-color:none!important;color:#0000!important}html[dir=rtl] .cc-switch__input[type=checkbox]{margin-left:8px!important;margin-right:0!important}.cc-list>ul,ul.cc-list{list-style-type:disc;margin-left:0;padding-left:0}.cc-list>ul>li,ul.cc-list>li{margin-left:1em}.cc-list>ul>li:not(:last-child),ul.cc-list>li:not(:last-child){margin-bottom:4px}.cc-list__title{margin-bottom:8px!important}.cc-list--inline>ul,ul.cc-list--inline{display:flex;flex-wrap:wrap;list-style-type:none}.cc-list--inline>ul>li,ul.cc-list--inline>li{margin:0 .5em .5em 0}.cc-overlay{background-color:#0000004d;z-index:99998}.cc-overlay,.cc-preferences{bottom:0;left:0;position:fixed;right:0;top:0}.cc-preferences{background-color:#050a14f2!important;border:0;box-sizing:border-box;color:#111;font-family:sans-serif!important;line-height:1.4;margin:auto;max-height:100vh;overflow:auto;padding:0;z-index:100000}.cc-preferences:focus{outline:none}.cc-preferences *,.cc-preferences :after,.cc-preferences :before{box-sizing:inherit!important}.cc-preferences h2,.cc-preferences h3,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-preferences .cc-h2,.cc-preferences h2{font-size:18px}.cc-preferences .cc-h3,.cc-preferences h3{font-size:16px}.cc-preferences .cc-h4,.cc-preferences .cc-h5,.cc-preferences .cc-h6,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-size:14px}.cc-preferences a{color:#025e8d;text-decoration:underline}.cc-preferences a:hover{color:inherit;text-decoration:none}.cc-preferences h3{background:none;color:#111;padding:0;text-transform:none}dialog.cc-preferences{background-color:initial}dialog.cc-preferences::backdrop{background-color:#000000e6}.cc-preferences__dialog{display:flex;flex-direction:column;margin:auto;max-height:100vh;max-width:860px;padding:12px;position:relative}.cc-preferences__dialog>:last-child{border-bottom-left-radius:10px;border-bottom-right-radius:10px}@media (min-width:980px){.cc-preferences__dialog{padding:16px}}.cc-preferences__close{background:#0000!important;border:1px solid #ececec;border-radius:50%;color:#111!important;cursor:pointer;font-family:Times New Roman,serif;font-size:40px;height:40px;left:auto;line-height:1;margin-top:-20px;padding:0!important;position:absolute;right:20px;top:50%;width:40px}.cc-preferences__close:focus{outline:3px solid #08c}.cc-preferences__close-label{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__header{background:#fff!important;border-bottom:1px solid #0000001a;border-top-left-radius:10px;border-top-right-radius:10px;padding:16px;position:relative;text-align:center}.cc-preferences__title{background:none!important;color:#111!important;font-family:sans-serif!important;font-size:18px!important;font-weight:700!important;margin:0!important;padding:0 16px 0 0!important}@media (min-width:480px){.cc-preferences__title{padding-right:0!important}}@media (min-width:980px){.cc-preferences__title{font-size:22px!important}}.cc-preferences__body{background:#fff!important;flex:1 1 auto;min-height:200px;overflow-x:hidden;overflow-y:auto;padding:16px}.cc-preferences__footer{background:#fff!important;border-top:1px solid #d0d0d0;box-shadow:0 0 5px 0 #0003;margin-bottom:0;padding:12px;position:relative}@media (min-width:480px){.cc-preferences__footer{align-items:stretch;display:flex}}.cc-preferences__footer>.cc-button{display:block;width:100%}@media (min-width:480px){.cc-preferences__footer>.cc-button{flex:1 1 auto}}@media (min-width:980px){.cc-preferences__footer>.cc-button{flex-basis:auto}}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:first-child){margin-left:12px}}.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:8px}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:0}}.cc-preferences__categories{list-style:none;margin:0;padding:0}.cc-preferences__category:not(:last-child){border-bottom:1px solid #0000001a;margin-bottom:12px;padding-bottom:12px}.cc-preferences__category-description{font-size:14px;margin:0 0 8px;padding:0}.cc-preferences__category-footer{align-items:center;display:flex;justify-content:space-between}.cc-preferences__status{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__controls{display:flex;margin:0}.cc-preferences__controls>:not(:last-child){margin-right:12px}.cc-preferences__always-on{font-size:14px;font-weight:700;padding-left:26px;position:relative}.cc-preferences__always-on:before{background:#0000!important;border:solid;border-top-color:#0000;border-width:0 0 4px 4px;content:"";display:block;height:10px;left:0;position:absolute;top:2px;transform:rotate(-45deg);width:18px}.cc-preferences__details{background-color:#eff6fb!important;border-radius:10px;font-size:14px;margin:12px 0 0;padding:12px}.cc-preferences__cookie-list,.cc-preferences__provider-list{list-style:none;margin:0;padding:0}.cc-preferences__provider-list{columns:170px}.cc-preferences__cookie-title{font-size:1em;font-style:normal;font-weight:400;margin:0}.cc-preferences__cookie-description{font-size:1em;margin:0 0 8px}.cc-preferences__cookie-domain,.cc-preferences__cookie-lifespan{border-left:1px solid #999;color:#666;margin-left:8px;padding-left:4px}body.cc-has-preferences-open{overflow:hidden;position:relative}.cc-table{border-collapse:collapse;width:100%}.cc-table tbody tr{border-top:1px solid #0000001a}.cc-table td,.cc-table th{font-size:14px;padding:4px 8px;vertical-align:top}.cc-table+.cc-table{margin-top:8px}.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:70%}@media (min-width:680px){.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:80%}}.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{padding-left:8px;width:40%}@media (min-width:680px){.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{width:30%}}.cc-tabs,.cc-tabs>ul{background-color:#faf9f6;display:flex;list-style:none;margin:0;padding:0}.cc-tabs__panel{border-top:1px solid #ececec;margin-top:-1px}.cc-tabs__button{background-color:initial;border:0;border-left:1px solid #0000;border-right:1px solid #0000;color:#111;font-size:16px;font-weight:700;margin:0;padding:15px 16px 12px;position:relative}.cc-tabs__button:before{background-color:initial;content:"";display:block;height:3px;left:0;position:absolute;top:0;width:100%}.cc-tabs__button.cc-active{background-color:#fff;border-color:#025e8d #ececec #ececec;color:#111}.cc-tabs__button.cc-active:before{background-color:#025e8d}.cc-tabs__content{background-color:#fff;display:none;font-size:14px;padding:16px}.cc-tabs__section:not(:first-child){padding-top:24px}.cc-tcf{font-size:14px;margin-top:16px}.cc-tcf__list{list-style:none;margin:0;padding:0}.cc-vendor-count{color:#666;font-size:16px;margin:0 0 8px}.cc-hide{display:none!important}.cc-show{display:block!important}.cc-external-link{background-color:#ececec;background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTcgMGExIDEgMCAxIDEgMCAySDIuNmMtLjM3MSAwLS42LjIwOS0uNi41djE1YzAgLjI5MS4yMjkuNS42LjVoMTQuOGMuMzcxIDAgLjYtLjIwOS42LS41VjEzYTEgMSAwIDAgMSAyIDB2NC41YzAgMS40MzgtMS4xNjIgMi41LTIuNiAyLjVIMi42QzEuMTYyIDIwIDAgMTguOTM4IDAgMTcuNXYtMTVDMCAxLjA2MiAxLjE2MiAwIDIuNiAwem02IDBoNmwuMDc1LjAwMy4xMjYuMDE3LjExMS4wMy4xMTEuMDQ0LjA5OC4wNTIuMDk2LjA2Ny4wOS4wOGExIDEgMCAwIDEgLjA5Ny4xMTJsLjA3MS4xMS4wNTQuMTE0LjAzNS4xMDUuMDMuMTQ4TDIwIDF2NmExIDEgMCAwIDEtMiAwVjMuNDE0bC02LjY5MyA2LjY5M2ExIDEgMCAwIDEtMS40MTQtMS40MTRMMTYuNTg0IDJIMTNhMSAxIDAgMCAxLS45OTMtLjg4M0wxMiAxYTEgMSAwIDAgMSAxLTEiLz48L3N2Zz4=");background-position:right 8px center;background-repeat:no-repeat;background-size:10px auto;border-radius:3px;box-shadow:0 1px 0 #0003;color:#111!important;display:inline-block;font-size:12px;padding:4px 26px 4px 8px;text-decoration:none!important}.cc-external-link:hover{text-decoration:underline!important}.cc-m-0{margin:0!important}.cc-grey{color:#666}</style><script async="" type="text/javascript" src="https://injections.readcube.com/nature/inject.fa744f3f.js"></script></head>
<body class="article-page"><dialog class="cc-banner" data-cc-banner="" data-nosnippet="" aria-labelledby="cc-banner-label" open="">
		<div class="cc-banner__content" autofocus="" tabindex="-1">
			<div class="cc-banner__container">
				<div class="cc-banner__header">
					<h2 class="cc-banner__title" id="cc-banner-label">Your privacy, your choice</h2>
				</div>
				<div class="cc-banner__body">
					<div class="cc-banner__policy">
						<p>We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.</p><p>By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.</p><p>See our <a href="https://www.nature.com/info/privacy" data-cc-action="privacy">privacy policy</a> for more information on the use of your personal data.</p><p><button type="button" data-cc-action="preferences" class="cc-button cc-button--link cc-button--text">Manage preferences</button> for further information and to change your choices.</p>
					</div>
					
				</div>
				<div class="cc-banner__footer">
					
			<button data-cc-action="accept" class="cc-button cc-button--secondary cc-button--contrast cc-banner__button cc-banner__button-accept">Accept all cookies</button>
			
		
				</div>
			</div>
		</div>
	</dialog>

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90" style="display: none !important;">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-test="top-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41591-022-01734-1;doi=10.1038/s41591-022-01734-1;subjmeta=2521,326,61,631;kwrd=Biotechnology,Infectious-disease+diagnostics">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=-1274591664&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01734-1%26doi%3D10.1038/s41591-022-01734-1%26subjmeta%3D2521,326,61,631%26kwrd%3DBiotechnology,Infectious-disease+diagnostics">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=-1274591664&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01734-1%26doi%3D10.1038/s41591-022-01734-1%26subjmeta%3D2521,326,61,631%26kwrd%3DBiotechnology,Infectious-disease+diagnostics"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header="" data-track-component="nature-150-split-header" style="border-color:#e40428">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nm" data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" height="32" alt="Nature Medicine">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
                            </a><div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected="">All journals</option>
                                    <option value="nm">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41591-022-01734-1"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header" hidden="">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-articles" data-track="click" data-track-action="research articles" data-track-label="link" data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-and-analysis" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link" data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/podcast" data-track="click" data-track-action="podcasts" data-track-label="link" data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/current-issue" data-track="click" data-track-action="current issue" data-track-label="link" data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/collections" data-track="click" data-track-action="collections" data-track-label="link" data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://www.facebook.com/Nature-Medicine-193691346949/" data-track="click" data-track-action="facebook" data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://twitter.com/naturemedicine" data-track="click" data-track-action="twitter" data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=5" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/aims" data-track="click" data-track-action="aims &amp; scope" data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-information" data-track="click" data-track-action="journal information" data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-impact" data-track="click" data-track-action="journal metrics" data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editors" data-track="click" data-track-action="about the editors" data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-cross-journal-editorial-team" data-track="click" data-track-action="research cross-journal editorial team" data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-cross-journal-editorial-team" data-track="click" data-track-action="reviews cross-journal editorial team" data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/statistics-advisory-panel" data-track="click" data-track-action="statistical advisory panel" data-track-label="link">
                                    Statistical Advisory Panel
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/our-publishing-models" data-track="click" data-track-action="our publishing models" data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-values-statement" data-track="click" data-track-action="editorial values statement" data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-policies" data-track="click" data-track-action="editorial policies" data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/content" data-track="click" data-track-action="content types" data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="javascript:;" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                        </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/submission-guidelines" data-track="click" data-track-action="submission guidelines" data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-context="header publish with us dropdown menu" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/open-access-funding" data-test="funding-eligibility-link" data-track="click_explore_funding" data-track-context="header publish with us" data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link" href="https://mts-nmed.nature.com/cgi-bin/main.plex" data-track="click_submit_manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-action="submit manuscript" data-track-label="link (publish with us dropdown menu)" data-track-external="" data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link" href="https://journal-alerts.springernature.com/subscribe?journal_id=41591" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-label="link (desktop site header)" data-track-external="">
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature medicine"><span itemprop="name">nature medicine</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm/articles?type=article" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" id="js-enable-context-bar" data-test="context-bar" data-context-bar="" aria-hidden="true">
                    <div class="c-context-bar__container u-container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            Multiplexed CRISPR-based microfluidic platform for clinical testing of respiratory viruses and identification of SARS-CoV-2 variants
                        </div>
                        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01734-1.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01734-1.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2022-02-07">07 February 2022</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Multiplexed CRISPR-based microfluidic platform for clinical testing of respiratory viruses and identification of SARS-CoV-2 variants</h1>
                    <ul class="c-article-author-list c-article-author-list--long js-no-scroll" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Nicole_L_-Welch-Aff1-Aff2" data-author-popup="auth-Nicole_L_-Welch-Aff1-Aff2" data-author-search="Welch, Nicole L." data-corresp-id="c1" data-track-context="researcher popup with no profile" data-track-index="1_50">Nicole L. Welch<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-1163-7002"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-1163-7002</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff2" tabindex="-1">2</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Meilin-Zhu-Aff1-Aff3" data-author-popup="auth-Meilin-Zhu-Aff1-Aff3" data-author-search="Zhu, Meilin" data-track-context="researcher popup with no profile" data-track-index="2_50">Meilin Zhu</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-1302-4020"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-1302-4020</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Catherine-Hua-Aff1-Aff4" data-author-popup="auth-Catherine-Hua-Aff1-Aff4" data-author-search="Hua, Catherine" data-track-context="researcher popup with no profile" data-track-index="3_50">Catherine Hua</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-0455-5364"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0455-5364</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Juliane-Weller-Aff1-Aff5" data-author-popup="auth-Juliane-Weller-Aff1-Aff5" data-author-search="Weller, Juliane" data-track-context="researcher popup with no profile" data-track-index="4_50">Juliane Weller</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff5" tabindex="-1">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Marzieh_Ezzaty-Mirhashemi-Aff1" data-author-popup="auth-Marzieh_Ezzaty-Mirhashemi-Aff1" data-author-search="Mirhashemi, Marzieh Ezzaty" data-track-context="researcher popup with no profile" data-track-index="5_50">Marzieh Ezzaty Mirhashemi</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tien_G_-Nguyen-Aff1" data-author-popup="auth-Tien_G_-Nguyen-Aff1" data-author-search="Nguyen, Tien G." data-track-context="researcher popup with no profile" data-track-index="6_50">Tien G. Nguyen</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sreekar-Mantena-Aff1" data-author-popup="auth-Sreekar-Mantena-Aff1" data-author-search="Mantena, Sreekar" data-track-context="researcher popup with no profile" data-track-index="7_50">Sreekar Mantena</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-8834-1461"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8834-1461</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Matthew_R_-Bauer-Aff1-Aff6" data-author-popup="auth-Matthew_R_-Bauer-Aff1-Aff6" data-author-search="Bauer, Matthew R." data-track-context="researcher popup with no profile" data-track-index="8_50">Matthew R. Bauer</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Bennett_M_-Shaw-Aff1-Aff4" data-author-popup="auth-Bennett_M_-Shaw-Aff1-Aff4" data-author-search="Shaw, Bennett M." data-track-context="researcher popup with no profile" data-track-index="9_50">Bennett M. Shaw</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Cheri_M_-Ackerman-Aff1-Aff3" data-author-popup="auth-Cheri_M_-Ackerman-Aff1-Aff3" data-author-search="Ackerman, Cheri M." data-track-context="researcher popup with no profile" data-track-index="10_50">Cheri M. Ackerman</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sri_Gowtham-Thakku-Aff1-Aff7" data-author-popup="auth-Sri_Gowtham-Thakku-Aff1-Aff7" data-author-search="Thakku, Sri Gowtham" data-track-context="researcher popup with no profile" data-track-index="11_50">Sri Gowtham Thakku</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Megan_W_-Tse-Aff1-Aff3" data-author-popup="auth-Megan_W_-Tse-Aff1-Aff3" data-author-search="Tse, Megan W." data-track-context="researcher popup with no profile" data-track-index="12_50">Megan W. Tse</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-4968-7072"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-4968-7072</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jared-Kehe-Aff1-Aff3" data-author-popup="auth-Jared-Kehe-Aff1-Aff3" data-author-search="Kehe, Jared" data-track-context="researcher popup with no profile" data-track-index="13_50">Jared Kehe</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Marie_Martine-Uwera-Aff1" data-author-popup="auth-Marie_Martine-Uwera-Aff1" data-author-search="Uwera, Marie-Martine" data-track-context="researcher popup with no profile" data-track-index="14_50">Marie-Martine Uwera</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jacqueline_S_-Eversley-Aff4" data-author-popup="auth-Jacqueline_S_-Eversley-Aff4" data-author-search="Eversley, Jacqueline S." data-track-context="researcher popup with no profile" data-track-index="15_50">Jacqueline S. Eversley</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Derek_A_-Bielwaski-Aff4" data-author-popup="auth-Derek_A_-Bielwaski-Aff4" data-author-search="Bielwaski, Derek A." data-track-context="researcher popup with no profile" data-track-index="16_50">Derek A. Bielwaski</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Graham-McGrath-Aff4" data-author-popup="auth-Graham-McGrath-Aff4" data-author-search="McGrath, Graham" data-track-context="researcher popup with no profile" data-track-index="17_50">Graham McGrath</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Joseph-Braidt-Aff4" data-author-popup="auth-Joseph-Braidt-Aff4" data-author-search="Braidt, Joseph" data-track-context="researcher popup with no profile" data-track-index="18_50">Joseph Braidt</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jeremy-Johnson-Aff1" data-author-popup="auth-Jeremy-Johnson-Aff1" data-author-search="Johnson, Jeremy" data-track-context="researcher popup with no profile" data-track-index="19_50">Jeremy Johnson</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Felecia-Cerrato-Aff1" data-author-popup="auth-Felecia-Cerrato-Aff1" data-author-search="Cerrato, Felecia" data-track-context="researcher popup with no profile" data-track-index="20_50">Felecia Cerrato</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Gage_K_-Moreno-Aff1" data-author-popup="auth-Gage_K_-Moreno-Aff1" data-author-search="Moreno, Gage K." data-track-context="researcher popup with no profile" data-track-index="21_50">Gage K. Moreno</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Lydia_A_-Krasilnikova-Aff1-Aff8" data-author-popup="auth-Lydia_A_-Krasilnikova-Aff1-Aff8" data-author-search="Krasilnikova, Lydia A." data-track-context="researcher popup with no profile" data-track-index="22_50">Lydia A. Krasilnikova</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-4163-706X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-4163-706X</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Brittany_A_-Petros-Aff1-Aff7-Aff9-Aff10" data-author-popup="auth-Brittany_A_-Petros-Aff1-Aff7-Aff9-Aff10" data-author-search="Petros, Brittany A." data-track-context="researcher popup with no profile" data-track-index="23_50">Brittany A. Petros</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-2110-7475"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2110-7475</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff7" tabindex="-1">7</a>,<a href="#Aff9" tabindex="-1">9</a>,<a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Gabrielle_L_-Gionet-Aff1" data-author-popup="auth-Gabrielle_L_-Gionet-Aff1" data-author-search="Gionet, Gabrielle L." data-track-context="researcher popup with no profile" data-track-index="24_50">Gabrielle L. Gionet</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ewa-King-Aff11" data-author-popup="auth-Ewa-King-Aff11" data-author-search="King, Ewa" data-track-context="researcher popup with no profile" data-track-index="25_50">Ewa King</a><sup class="u-js-hide"><a href="#Aff11" tabindex="-1">11</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Richard_C_-Huard-Aff11" data-author-popup="auth-Richard_C_-Huard-Aff11" data-author-search="Huard, Richard C." data-track-context="researcher popup with no profile" data-track-index="26_50">Richard C. Huard</a><sup class="u-js-hide"><a href="#Aff11" tabindex="-1">11</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Samantha_K_-Jalbert-Aff10" data-author-popup="auth-Samantha_K_-Jalbert-Aff10" data-author-search="Jalbert, Samantha K." data-track-context="researcher popup with no profile" data-track-index="27_50">Samantha K. Jalbert</a><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael_L_-Cleary-Aff10" data-author-popup="auth-Michael_L_-Cleary-Aff10" data-author-search="Cleary, Michael L." data-track-context="researcher popup with no profile" data-track-index="28_50">Michael L. Cleary</a><sup class="u-js-hide"><a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Nicholas_A_-Fitzgerald-Aff1" data-author-popup="auth-Nicholas_A_-Fitzgerald-Aff1" data-author-search="Fitzgerald, Nicholas A." data-track-context="researcher popup with no profile" data-track-index="29_50">Nicholas A. Fitzgerald</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Stacey_B_-Gabriel-Aff1" data-author-popup="auth-Stacey_B_-Gabriel-Aff1" data-author-search="Gabriel, Stacey B." data-track-context="researcher popup with no profile" data-track-index="30_50">Stacey B. Gabriel</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Glen_R_-Gallagher-Aff12" data-author-popup="auth-Glen_R_-Gallagher-Aff12" data-author-search="Gallagher, Glen R." data-track-context="researcher popup with no profile" data-track-index="31_50">Glen R. Gallagher</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6837-9473"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6837-9473</a></span><sup class="u-js-hide"><a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sandra_C_-Smole-Aff12" data-author-popup="auth-Sandra_C_-Smole-Aff12" data-author-search="Smole, Sandra C." data-track-context="researcher popup with no profile" data-track-index="32_50">Sandra C. Smole</a><sup class="u-js-hide"><a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Lawrence_C_-Madoff-Aff12" data-author-popup="auth-Lawrence_C_-Madoff-Aff12" data-author-search="Madoff, Lawrence C." data-track-context="researcher popup with no profile" data-track-index="33_50">Lawrence C. Madoff</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-2589-7777"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-2589-7777</a></span><sup class="u-js-hide"><a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Catherine_M_-Brown-Aff12" data-author-popup="auth-Catherine_M_-Brown-Aff12" data-author-search="Brown, Catherine M." data-track-context="researcher popup with no profile" data-track-index="34_50">Catherine M. Brown</a><sup class="u-js-hide"><a href="#Aff12" tabindex="-1">12</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Matthew_W_-Keller-Aff13" data-author-popup="auth-Matthew_W_-Keller-Aff13" data-author-search="Keller, Matthew W." data-track-context="researcher popup with no profile" data-track-index="35_50">Matthew W. Keller</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-5850-1698"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-5850-1698</a></span><sup class="u-js-hide"><a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Malania_M_-Wilson-Aff13" data-author-popup="auth-Malania_M_-Wilson-Aff13" data-author-search="Wilson, Malania M." data-track-context="researcher popup with no profile" data-track-index="36_50">Malania M. Wilson</a><sup class="u-js-hide"><a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Marie_K_-Kirby-Aff13" data-author-popup="auth-Marie_K_-Kirby-Aff13" data-author-search="Kirby, Marie K." data-track-context="researcher popup with no profile" data-track-index="37_50">Marie K. Kirby</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-0859-9604"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0859-9604</a></span><sup class="u-js-hide"><a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-John_R_-Barnes-Aff13" data-author-popup="auth-John_R_-Barnes-Aff13" data-author-search="Barnes, John R." data-track-context="researcher popup with no profile" data-track-index="38_50">John R. Barnes</a><sup class="u-js-hide"><a href="#Aff13" tabindex="-1">13</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Daniel_J_-Park-Aff1" data-author-popup="auth-Daniel_J_-Park-Aff1" data-author-search="Park, Daniel J." data-track-context="researcher popup with no profile" data-track-index="39_50">Daniel J. Park</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-7226-7781"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7226-7781</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Katherine_J_-Siddle-Aff1-Aff8" data-author-popup="auth-Katherine_J_-Siddle-Aff1-Aff8" data-author-search="Siddle, Katherine J." data-track-context="researcher popup with no profile" data-track-index="40_50">Katherine J. Siddle</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Christian_T_-Happi-Aff1-Aff14-Aff15" data-author-popup="auth-Christian_T_-Happi-Aff1-Aff14-Aff15" data-author-search="Happi, Christian T." data-track-context="researcher popup with no profile" data-track-index="41_50">Christian T. Happi</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff14" tabindex="-1">14</a>,<a href="#Aff15" tabindex="-1">15</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Deborah_T_-Hung-Aff1-Aff16" data-author-popup="auth-Deborah_T_-Hung-Aff1-Aff16" data-author-search="Hung, Deborah T." data-track-context="researcher popup with no profile" data-track-index="42_50">Deborah T. Hung</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff16" tabindex="-1">16</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Michael-Springer-Aff1-Aff10" data-author-popup="auth-Michael-Springer-Aff1-Aff10" data-author-search="Springer, Michael" data-track-context="researcher popup with no profile" data-track-index="43_50">Michael Springer</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff10" tabindex="-1">10</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Bronwyn_L_-MacInnis-Aff1-Aff17" data-author-popup="auth-Bronwyn_L_-MacInnis-Aff1-Aff17" data-author-search="MacInnis, Bronwyn L." data-track-context="researcher popup with no profile" data-track-index="44_50">Bronwyn L. MacInnis</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff17" tabindex="-1">17</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jacob_E_-Lemieux-Aff1-Aff4" data-author-popup="auth-Jacob_E_-Lemieux-Aff1-Aff4" data-author-search="Lemieux, Jacob E." data-track-context="researcher popup with no profile" data-track-index="45_50">Jacob E. Lemieux</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Eric-Rosenberg-Aff4-Aff18" data-author-popup="auth-Eric-Rosenberg-Aff4-Aff18" data-author-search="Rosenberg, Eric" data-track-context="researcher popup with no profile" data-track-index="46_50">Eric Rosenberg</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a>,<a href="#Aff18" tabindex="-1">18</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-John_A_-Branda-Aff18" data-author-popup="auth-John_A_-Branda-Aff18" data-author-search="Branda, John A." data-track-context="researcher popup with no profile" data-track-index="47_50">John A. Branda</a><sup class="u-js-hide"><a href="#Aff18" tabindex="-1">18</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Paul_C_-Blainey-Aff1-Aff3-Aff19" data-author-popup="auth-Paul_C_-Blainey-Aff1-Aff3-Aff19" data-author-search="Blainey, Paul C." data-track-context="researcher popup with no profile" data-track-index="48_50">Paul C. Blainey</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-4889-8783"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-4889-8783</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff3" tabindex="-1">3</a>,<a href="#Aff19" tabindex="-1">19</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Pardis_C_-Sabeti-Aff1-Aff8-Aff17-Aff19-Aff20" data-author-popup="auth-Pardis_C_-Sabeti-Aff1-Aff8-Aff17-Aff19-Aff20" data-author-search="Sabeti, Pardis C." data-corresp-id="c2" data-track-context="researcher popup with no profile" data-track-index="49_50">Pardis C. Sabeti<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-9843-1890"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-9843-1890</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a>,<a href="#Aff8" tabindex="-1">8</a>,<a href="#Aff17" tabindex="-1">17</a>,<a href="#Aff19" tabindex="-1">19</a>,<a href="#Aff20" tabindex="-1">20</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 50 authors for this article" title="Show all 50 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Cameron-Myhrvold-Aff21" data-author-popup="auth-Cameron-Myhrvold-Aff21" data-author-search="Myhrvold, Cameron" data-corresp-id="c3" data-track-context="researcher popup with no profile" data-track-index="50_50">Cameron Myhrvold<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-8971-184X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-8971-184X</a></span><sup class="u-js-hide"><a href="#Aff21" tabindex="-1">21</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>&nbsp;</li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nm" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Medicine</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;28</b>,&nbsp;<span class="u-visually-hidden">pages </span>1083–1094 (<span data-test="article-publication-year">2022</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item" data-test="access-count">
                        <p class="c-article-metrics-bar__count">33k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item" data-test="citation-count">
                        <p class="c-article-metrics-bar__count">175 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item" data-test="altmetric-score">
                            <p class="c-article-metrics-bar__count">85 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41591-022-01734-1/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    

                
    
    
        <div class="u-mb-8 c-status-message c-status-message--boxed c-status-message--info">
            
                <span class="c-status-message__icon">
                    <svg class="u-icon" width="18" height="18" aria-hidden="true" focusable="false">
                        <use xlink:href="#icon-eds-i-info-filled-medium"></use>
                    </svg>
                </span>
            
            
                
                    <p class="u-mt-0">An <a href="https://doi.org/10.1038/s41591-023-02684-y" class="relation-link" data-track="click" data-track-action="view linked article" data-track-label="link">Author Correction</a> to this article was published on 09 November 2023</p>
                
            
        </div>
    

    
        <div class="u-mb-8 c-status-message c-status-message--boxed c-status-message--info">
            
                <span class="c-status-message__icon">
                    <svg class="u-icon" width="18" height="18" aria-hidden="true" focusable="false">
                        <use xlink:href="#icon-eds-i-info-filled-medium"></use>
                    </svg>
                </span>
            
            <p class="u-mt-0">This article has been <a href="#change-history">updated</a></p>
        </div>
    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>The coronavirus disease 2019 (COVID-19) pandemic has demonstrated a clear need for high-throughput, multiplexed and sensitive assays for detecting severe acute respiratory syndrome coronavirus 2 (SARS-CoV-2) and other respiratory viruses and their emerging variants. Here, we present a cost-effective virus and variant detection platform, called microfluidic Combinatorial Arrayed Reactions for Multiplexed Evaluation of Nucleic acids (mCARMEN), which combines CRISPR-based diagnostics and microfluidics with a streamlined workflow for clinical use. We developed the mCARMEN respiratory virus panel to test for up to 21 viruses, including SARS-CoV-2, other coronaviruses and both influenza strains, and demonstrated its diagnostic-grade performance on 525 patient specimens in an academic setting and 166 specimens in a clinical setting. We further developed an mCARMEN panel to enable the identification of 6 SARS-CoV-2 variant lineages, including Delta and Omicron, and evaluated it on 2,088 patient specimens with near-perfect concordance to sequencing-based variant classification. Lastly, we implemented a combined Cas13 and Cas12 approach that enables quantitative measurement of SARS-CoV-2 and influenza A viral copies in samples. The mCARMEN platform enables high-throughput surveillance of multiple viruses and variants simultaneously, enabling rapid detection of SARS-CoV-2 variants.</p></div></div></section>

            
                
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41467-022-34086-y/MediaObjects/41467_2022_34086_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41467-022-34086-y?fromPaywallRec=false" data-track="select_recommendations_1" data-track-context="inline recommendations" data-track-action="click recommendations inline - 1" data-track-label="10.1038/s41467-022-34086-y">Microfluidic space coding for multiplexed nucleic acid detection via CRISPR-Cas12a and recombinase polymerase amplification
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">29 October 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41598-024-79407-x/MediaObjects/41598_2024_79407_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41598-024-79407-x?fromPaywallRec=false" data-track="select_recommendations_2" data-track-context="inline recommendations" data-track-action="click recommendations inline - 2" data-track-label="10.1038/s41598-024-79407-x">Microfluidic qPCR for detection of 21 common respiratory viruses in children with influenza-like illness
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">16 November 2024</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41551-022-00889-z/MediaObjects/41551_2022_889_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41551-022-00889-z?fromPaywallRec=false" data-track="select_recommendations_3" data-track-context="inline recommendations" data-track-action="click recommendations inline - 3" data-track-label="10.1038/s41551-022-00889-z">Simplified Cas13-based assays for the fast identification of SARS-CoV-2 and its variants
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">30 May 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1749523295,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>COVID-19 has exposed critical gaps in our global infectious disease diagnostic and surveillance capacity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="Mina, M. J. &amp; Andersen, K. G. COVID-19 testing: one size does not fit all. Science 371, 126–127 (2021)." href="/articles/s41591-022-01734-1#ref-CR1" id="ref-link-section-d38468781e1471">1</a></sup>. The pandemic rapidly necessitated high-throughput diagnostics to test large populations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Rasmussen, A. L. &amp; Popescu, S. V. SARS-CoV-2 transmission without symptoms. Science 371, 1206–1207 (2021)." href="/articles/s41591-022-01734-1#ref-CR2" id="ref-link-section-d38468781e1475">2</a></sup>, yet early diagnostic efforts met technical challenges that cost the United States precious time in its early response<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 3" title="Winichakoon, P. et al. Negative nasopharyngeal and oropharyngeal swabs do not rule out COVID-19. J. Clin. Microbiol. 58, e00297-20 (2020)." href="/articles/s41591-022-01734-1#ref-CR3" id="ref-link-section-d38468781e1479">3</a></sup>. Other challenges developed as the pandemic progressed that point toward an additional need for highly multiplexed surveillance technologies. These challenges include the cocirculating human respiratory viruses that cause symptoms similar to COVID-19 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Woloshin, S., Patel, N. &amp; Kesselheim, A. S. False negative tests for SARS-CoV-2 infection—challenges and implications. N. Engl. J. Med. 383, e38 (2020)." href="/articles/s41591-022-01734-1#ref-CR4" id="ref-link-section-d38468781e1483">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Lemieux, J. E. et al. Phylogenetic analysis of SARS-CoV-2 in Boston highlights the impact of superspreading events. Science 371, eabe326 (2021)." href="/articles/s41591-022-01734-1#ref-CR5" id="ref-link-section-d38468781e1486">5</a></sup>) and emerging SARS-CoV-2 variants of concern (VOCs) with mutations that impact viral fitness and clinical disease prognosis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Harvey, W. T. et al. SARS-CoV-2 variants, spike mutations and immune escape. Nat. Rev. Microbiol. 19, 409–424 (2021)." href="/articles/s41591-022-01734-1#ref-CR6" id="ref-link-section-d38468781e1490">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Dyson, L. et al. Possible future waves of SARS-CoV-2 infection generated by variants of concern with a range of characteristics. Nat. Commun. 12, 5730 (2021)." href="/articles/s41591-022-01734-1#ref-CR7" id="ref-link-section-d38468781e1493">7</a></sup>.</p><p>An ideal diagnostic method would have surveillance capabilities to process hundreds of patient samples simultaneously, detect multiple viruses, differentiate between viral variants and quantify viral load<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Zou, L. et al. SARS-CoV-2 viral load in upper respiratory specimens of infected patients. N. Engl. J. Med. 382, 1177–1179 (2020)." href="/articles/s41591-022-01734-1#ref-CR8" id="ref-link-section-d38468781e1500">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Wölfel, R. et al. Virological assessment of hospitalized patients with COVID-2019. Nature 581, 465–469 (2020)." href="/articles/s41591-022-01734-1#ref-CR9" id="ref-link-section-d38468781e1503">9</a></sup>; yet no such test currently exists. As it stands, there is a trade-off between clinically approved high-throughput diagnostics and multiplexed methods in the number of patient samples and/or pathogens tested simultaneously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Vogels, C. B. F. et al. Multiplex qPCR discriminates variants of concern to enhance global surveillance of SARS-CoV-2. PLoS Biol. 19, e3001236 (2021)." href="#ref-CR10" id="ref-link-section-d38468781e1507">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Heggestad, J. T. et al. Multiplexed, quantitative serological profiling of COVID-19 from blood by a point-of-care test. Sci. Adv. 7, eabg4901 (2021)." href="#ref-CR11" id="ref-link-section-d38468781e1507_1">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Pham, J. et al. Performance characteristics of a high-throughput automated transcription-mediated amplification test for SARS-CoV-2 detection. J. Clin. Microbiol. 58, e01669-20 (2020)." href="/articles/s41591-022-01734-1#ref-CR12" id="ref-link-section-d38468781e1510">12</a></sup>. For example, quantitative PCR with reverse transcription (RT–qPCR) is high-throughput by testing at least 88 samples but for 1–3 analytes at a time; multiplexed techniques such as Cepheid Xpert Xpress can detect 4 respiratory viruses in up to 16 samples per run and BioFire can detect 22 respiratory pathogens in 1 sample simultaneously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Eckbo, E. J. et al. Evaluation of the BioFire® COVID-19 test and Respiratory Panel 2.1 for rapid identification of SARS-CoV-2 in nasopharyngeal swab samples. Diagn. Microbiol. Infect. Dis. 99, 115260 (2021)." href="/articles/s41591-022-01734-1#ref-CR13" id="ref-link-section-d38468781e1514">13</a></sup>. Only a few clinical diagnostic methods comprehensively detect SARS-CoV-2 variant mutations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Konings, F. et al. SARS-CoV-2 variants of interest and concern naming scheme conducive for global discourse. Nat. Microbiol. 6, 821–823 (2021)." href="#ref-CR14" id="ref-link-section-d38468781e1518">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Huang, H.-S. et al. Multiplex PCR system for the rapid diagnosis of respiratory virus infection: systematic review and meta-analysis. Clin. Microbiol. Infect. 24, 1055–1063 (2018)." href="#ref-CR15" id="ref-link-section-d38468781e1518_1">15</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Jacky, L. et al. Robust multichannel encoding for highly multiplexed quantitative PCR. Anal. Chem. 93, 4208–4216 (2021)." href="/articles/s41591-022-01734-1#ref-CR16" id="ref-link-section-d38468781e1521">16</a></sup>, which is why this has largely been achieved through next-generation sequencing (NGS)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Burki, T. Understanding variants of SARS-CoV-2. Lancet 397, 462 (2021)." href="/articles/s41591-022-01734-1#ref-CR17" id="ref-link-section-d38468781e1525">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Borges, V. &amp; et al. Tracking SARS-CoV-2 lineage B.1.1.7 dissemination: insights from nationwide spike gene target failure (SGTF) and spike gene late detection (SGTL) data, Portugal, week 49 2020 to week 3 2021. Euro Surveill. 26, 2100131 (2021)." href="/articles/s41591-022-01734-1#ref-CR18" id="ref-link-section-d38468781e1528">18</a></sup>, though it is time-consuming, expensive and requires bioinformatic expertise to interpret<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Harvey, W. T. et al. SARS-CoV-2 variants, spike mutations and immune escape. Nat. Rev. Microbiol. 19, 409–424 (2021)." href="/articles/s41591-022-01734-1#ref-CR6" id="ref-link-section-d38468781e1533">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Xuan, J., Yu, Y., Qing, T., Guo, L. &amp; Shi, L. Next-generation sequencing in the clinic: promises and challenges. Cancer Lett. 340, 284–295 (2013)." href="#ref-CR19" id="ref-link-section-d38468781e1536">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Houldcroft, C. J., Beale, M. A. &amp; Breuer, J. Clinical and biological insights from viral genome sequencing. Nat. Rev. Microbiol. 15, 183–192 (2017)." href="#ref-CR20" id="ref-link-section-d38468781e1536_1">20</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Peddu, V. et al. Metagenomic analysis reveals clinical SARS-CoV-2 infection and bacterial or viral superinfection and colonization. Clin. Chem. 66, 966–972 (2020)." href="#ref-CR21" id="ref-link-section-d38468781e1536_2">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 22" title="Brito, A. F. et al. Global disparities in SARS-CoV-2 genomic surveillance. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.08.21.21262393
                  
                 (2021)." href="/articles/s41591-022-01734-1#ref-CR22" id="ref-link-section-d38468781e1539">22</a></sup>.</p><p>CRISPR-based diagnostics offer an alternative approach to detecting multiple viruses and variants<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Kaminski, M. M., Abudayyeh, O. O., Gootenberg, J. S., Zhang, F. &amp; Collins, J. J. CRISPR-based diagnostics. Nat. Biomed. Eng. 5, 643–656 (2021)." href="#ref-CR23" id="ref-link-section-d38468781e1546">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Jiao, C. et al. Noncanonical crRNAs derived from host transcripts enable multiplexable RNA detection by Cas9. Science 372, 941–948 (2021)." href="#ref-CR24" id="ref-link-section-d38468781e1546_1">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 25" title="Gootenberg, J. S. et al. Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6. Science 360, 439–444 (2018)." href="/articles/s41591-022-01734-1#ref-CR25" id="ref-link-section-d38468781e1549">25</a></sup>. The CRISPR effector proteins Cas12 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Chen, J. S. et al. CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity. Science 360, 436–439 (2018)." href="/articles/s41591-022-01734-1#ref-CR26" id="ref-link-section-d38468781e1553">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Li, S.-Y. et al. CRISPR-Cas12a-assisted nucleic acid detection. Cell Discov. 4, 20 (2018)." href="/articles/s41591-022-01734-1#ref-CR27" id="ref-link-section-d38468781e1556">27</a></sup>) or Cas13 (refs. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Abudayyeh, O. O. et al. C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. Science 353, aaf5573 (2016)." href="/articles/s41591-022-01734-1#ref-CR28" id="ref-link-section-d38468781e1560">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Gootenberg, J. S. et al. Nucleic acid detection with CRISPR-Cas13a/C2c2. Science 356, 438–442 (2017)." href="/articles/s41591-022-01734-1#ref-CR29" id="ref-link-section-d38468781e1563">29</a></sup>) activate upon CRISPR RNA (crRNA) target binding, which unleashes their collateral cleavage activity on a fluorescent reporter for detection of viral nucleic acids<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Myhrvold, C. et al. Field-deployable viral diagnostics using CRISPR-Cas13. Science 360, 444–448 (2018)." href="#ref-CR30" id="ref-link-section-d38468781e1567">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Arizti-Sanz, J. et al. Streamlined inactivation, amplification, and Cas13-based detection of SARS-CoV-2. Nat. Commun. 11, 5921 (2020)." href="#ref-CR31" id="ref-link-section-d38468781e1567_1">31</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Broughton, J. P. et al. CRISPR–Cas12-based detection of SARS-CoV-2. Nat. Biotechnol. 38, 870–874 (2020)." href="#ref-CR32" id="ref-link-section-d38468781e1567_2">32</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Patchsung, M. et al. Clinical validation of a Cas13-based assay for the detection of SARS-CoV-2 RNA. Nat. Biomed. Eng. 4, 1140–1149 (2020)." href="#ref-CR33" id="ref-link-section-d38468781e1567_3">33</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Liu, T. Y. et al. Accelerated RNA detection using tandem CRISPR nucleases. Nat. Chem. Biol. 17, 982–988 (2021)." href="/articles/s41591-022-01734-1#ref-CR34" id="ref-link-section-d38468781e1570">34</a></sup>. The crRNA target binding events are highly specific and altered by the presence of sequence variation. Maximally active crRNA design has been accelerated by machine learning and other computational methods<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Metsky, H. C. et al. Designing sensitive viral diagnostics with machine learning. Nat. Biotech. 
                  https://doi.org/10.1038/s41587-022-01213-5
                  
                 (2022)." href="/articles/s41591-022-01734-1#ref-CR35" id="ref-link-section-d38468781e1574">35</a></sup>. Nonetheless, most CRISPR diagnostics detect one to three targets per sample<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Myhrvold, C. et al. Field-deployable viral diagnostics using CRISPR-Cas13. Science 360, 444–448 (2018)." href="/articles/s41591-022-01734-1#ref-CR30" id="ref-link-section-d38468781e1579">30</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Joung, J. et al. Detection of SARS-CoV-2 with SHERLOCK one-pot testing. N. Engl. J. Med. 383, 1492–1494 (2020)." href="#ref-CR36" id="ref-link-section-d38468781e1582">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Bruch, R., Urban, G. A. &amp; Dincer, C. CRISPR/Cas powered multiplexed biosensing. Trends Biotechnol. 37, 791–792 (2019)." href="#ref-CR37" id="ref-link-section-d38468781e1582_1">37</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Tian, T., Qiu, Z., Jiang, Y., Zhu, D. &amp; Zhou, X. Exploiting the orthogonal CRISPR-Cas12a/Cas13a trans-cleavage for dual-gene virus detection using a handheld device. Biosens. Bioelectron. 196, 113701 (2022)." href="#ref-CR38" id="ref-link-section-d38468781e1582_2">38</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bruch, R. et al. CRISPR-powered electrochemical microfluidic multiplexed biosensor for target amplification-free miRNA diagnostics. Biosens. Bioelectron. 177, 112887 (2021)." href="/articles/s41591-022-01734-1#ref-CR39" id="ref-link-section-d38468781e1585">39</a></sup>, which is not sufficient for differential diagnosis via comprehensive microbe or variant identification.</p><p>To scale up the capabilities of CRISPR-based diagnostics, we developed CARMEN<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Ackerman, C. M. et al. Massively multiplexed nucleic acid detection with Cas13. Nature 582, 277–282 (2020)." href="/articles/s41591-022-01734-1#ref-CR40" id="ref-link-section-d38468781e1592">40</a></sup>, which parallelizes nucleic acid detection. The first generation of CARMEN, referred to in this article as CARMEN v.1, could detect 169 human-associated viruses in 8 samples simultaneously. In CARMEN v.1, samples and Cas13–crRNA complexes are separately confined for barcoding and emulsification before pairwise droplet combination for detection by fluorescence microscopy. This allows each sample to be tested against every crRNA. CARMEN v.1 is a powerful proof of concept for multiplexed CRISPR-based detection but it is difficult to use in a clinical setting given its use of custom-made imaging chips and readout hardware, manually intensive 8–10 h workflow and low-throughput sample evaluation.</p><p>To fulfill the public health need for a clinically relevant surveillance technology that detects multiple viruses and variants quickly, we developed mCARMEN. mCARMEN builds on CARMEN v.1 and uses commercially available Fluidigm microfluidics and instrumentation. To our knowledge, mCARMEN is the only diagnostic method that combines surveillance capabilities into a single technology platform with the ability to test hundreds of samples in a day for multiple respiratory viruses and variants, while also being able to quantify viral genomic copies.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">CARMEN implementation on Fluidigm for respiratory virus detection</h3><p>CARMEN v.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Ackerman, C. M. et al. Massively multiplexed nucleic acid detection with Cas13. Nature 582, 277–282 (2020)." href="/articles/s41591-022-01734-1#ref-CR40" id="ref-link-section-d38468781e1612">40</a></sup>) is limited by custom instrumentation requirements and labor-intensive protocols, which is why we sought to develop a scalable technology that could be broadly implemented. mCARMEN meets these requirements and eliminates the color-coding and dropletization needs of CARMEN v.1 by using commercially available integrated fluidic circuits (IFCs) on the Fluidigm Biomark for &lt;US$13 per sample (Fluidigm) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig1">1a</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">1</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">2</a>). By leveraging Fluidigm microfluidics, we overcame the need for a custom microscope and chips as well as data analysis expertise, which were required for CARMEN v.1. The Fluidigm IFCs use a specific number of assay combinations: 192 samples by 24 detection assays or 96 samples by 96 detection assays, which are all spatially separated (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">3</a>). After manual IFC loading, the Fluidigm controller moves the samples and detection assays through individual channels on the IFC until they reach the chip reaction chamber, where they are thoroughly mixed. We measured fluorescence on the Fluidigm Biomark with our custom automated protocols that take images of the IFC chip every 5 min for 1–3 h at 37 °C (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig7">1a</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Implementation of CARMEN using a microfluidic system improves sensitivity and speed."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Implementation of CARMEN using a microfluidic system improves sensitivity and speed.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="891"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Schematic of CARMEN v.1 (top) and mCARMEN (bottom) workflows. <b>b</b>, Heatmap showing mCARMEN fluorescent data across 21 human respiratory viruses (synthetic DNA fragments and corresponding viral Cas13 crRNAs) at 1 h post-reaction initiation, which were serially diluted from 10<sup>3</sup> to 10<sup>1</sup> copies μl<sup>−1</sup> and amplified using 2 separate primer pools. All samples were background-subtracted from the no target control (NTC)-noMg negative control. <b>c</b>, Concordance between CARMEN v.1 and mCARMEN from <b>b</b>. Blue: targets at 10<sup>3</sup> copies μl<sup>−1</sup>; green: targets at 10<sup>2</sup> copies μl<sup>−1</sup>; red: targets at 10<sup>1</sup> copies μl<sup>−1</sup>. <b>d</b>, Fluorescence kinetics of amplified SARS-CoV-2 DNA gene fragments from 10<sup>4</sup>–10<sup>1</sup> copies μl<sup>−1</sup> at 0, 1 and 3 h post-reaction initiation. Blue: mCARMEN; red: CARMEN v.1. <b>e</b>, A 21-human respiratory virus panel was tested on clinical specimens from 6 SARS-CoV-2-positive, 4 SARS-CoV-2-negative nasopharyngeal (NP) swabs and 8 FLUAV-positive specimens, collected before December 2019, and 5 NTCs. The heatmap shows fluorescent signals from SARS-CoV-2 crRNA, FLUAV crRNA and no crRNA control. Blue: mCARMEN at 1 h post-reaction initiation; red: CARMEN v.1 at 3 h post-reaction initiation. <b>f</b>, Concordance of mCARMEN and NGS on 58 suspected respiratory virus-infected patient specimens collected before December 2019 shown as a bar graph; overall concordance is shown as a confusion matrix. Black: detected by both mCARMEN and NGS; blue: detected by mCARMEN only; green: detected by NGS only. mCARMEN values are shown as the normalized fluorescence signal (FAM/ROX) (FAM signal divided by the signal for the passive reference dye, ROX, 1 h). CARMEN v.1 values are shown as the raw fluorescence signal (FAM). NTC-extract: no target control taken through extraction, cDNA synthesis, amplification, and detection; NTC-cDNA: no target control taken through cDNA synthesis, amplification and detection; NTC-amp: no target control taken through amplification and detection; NTC-det: no target control taken through detection; NTC-noMg: no target control expected to have no fluorescent signal due to lack on Mg2<sup>+</sup> needed to activate Cas13.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>In our first implementation of the mCARMEN platform, we designed a panel to detect 21 clinically relevant human respiratory viruses (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">4</a>). This included all viruses covered by BioFire RP2.1-SARS-CoV-2, four other human-associated coronaviruses and both influenza strains- as well as a few additional illness-inducing viruses<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Crowe, J. E. Jr. Human respiratory viruses. Reference Module in Biomedical Sciences 
                  https://doi.org/10.1016/B978-0-12-801238-3.02600-3
                  
                 (2014)." href="/articles/s41591-022-01734-1#ref-CR41" id="ref-link-section-d38468781e1707">41</a></sup>. To generate maximally active virus-specific crRNAs and PCR primers to detect the 21 viruses, we applied the assay design method ADAPT (Activity-informed Design with All-inclusive Patrolling of Targets; described in the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01734-1#Sec10">Methods</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Metsky, H. C. et al. Designing sensitive viral diagnostics with machine learning. Nat. Biotech. 
                  https://doi.org/10.1038/s41587-022-01213-5
                  
                 (2022)." href="/articles/s41591-022-01734-1#ref-CR35" id="ref-link-section-d38468781e1714">35</a></sup>. We were able to encompass the full genomic diversity of these viral families by including multiple primers, if needed.</p><p>We compared the performance of mCARMEN to CARMEN v.1 for detecting synthetic DNA fragments recapitulating the 21 viral targets and found that mCARMEN had the same (13 viruses) or better (8 viruses) analytical sensitivity compared with CARMEN v.1 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig1">1b,c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig7">1b</a>). Both mCARMEN and CARMEN v.1 had 100% analytical specificity but mCARMEN was 100% sensitive to 10<sup>2</sup> copies μl<sup>−1</sup> and 98.4% sensitive to 10<sup>1</sup> copies μl<sup>−1</sup> while CARMEN v.1 was only 86% and 77.8% sensitive, respectively. Moreover, the mCARMEN reaction rate was accelerated compared with CARMEN v.1, resulting in faster initial detection and signal saturation of targets (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig1">1d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig7">1c</a>). This is likely due to the higher temperature at reaction initiation for mCARMEN (37 °C) than for CARMEN v.1 (25 °C) and the extensive sample detection assay mixing that occurs in the mCARMEN IFC, rather than merged droplets mixing by diffusion in CARMEN v.1.</p><p>We then benchmarked the performance of both CARMEN diagnostics against RT–qPCR (CDC 2019-nCoV Kit) and/or unbiased metagenomic NGS on patient specimens. We obtained a set of 6 SARS-CoV-2-positive, 4 SARS-CoV-2-negative and 8 influenza A virus (FLUAV)-positive patient specimens for initial testing. mCARMEN and CARMEN v.1 had 100% concordance with RT–qPCR, NGS and each other (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig1">1e</a>). We also compared performance using 2 different fluorescent reporters, RNase Alert (Integrated DNA Technologies) and a custom 6-Uracil-FAM (polyU) reporter<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Arizti-Sanz, J. et al. Streamlined inactivation, amplification, and Cas13-based detection of SARS-CoV-2. Nat. Commun. 11, 5921 (2020)." href="/articles/s41591-022-01734-1#ref-CR31" id="ref-link-section-d38468781e1748">31</a></sup>. We found enhanced sensitivity when using a polyU fluorescent reporter due to the preference of LwaCas13 to cleave at uracils<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Abudayyeh, O. O. et al. C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. Science 353, aaf5573 (2016)." href="/articles/s41591-022-01734-1#ref-CR28" id="ref-link-section-d38468781e1752">28</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Gootenberg, J. S. et al. Nucleic acid detection with CRISPR-Cas13a/C2c2. Science 356, 438–442 (2017)." href="/articles/s41591-022-01734-1#ref-CR29" id="ref-link-section-d38468781e1755">29</a></sup> (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">1a,b</a>).</p><p>Aside from SARS-CoV-2 and influenza viruses, the remaining 19 viruses detectable by mCARMEN lacked a recognized criterion standard clinical diagnostic. Thus, we compared mCARMEN to unbiased metagenomic NGS results for the characterization of 58 prepandemic unknown samples collected from patients with a presumed upper respiratory infection (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig1">1f</a>, Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">5</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">1c,d</a>). Both mCARMEN and NGS detected the same respiratory viruses in 13 specimens (7 FLUAV, 2 HCoV-229E, 1 HCoV-NL63, 1 HCoV-OC43, 1 human metapneumovirus (HMPV) and 1 human rhinovirus (HRV)), neither detected respiratory viruses in 42 specimens and they had differing results for 3 specimens, with 93% overall concordance based on an average of approximately 3 million reads per specimen. Nine of the 13 specimens positive by both methods assembled complete genomes while the remaining 4 assembled partial or no genomes but had &gt;10 reads (2 FLUAV, 1 HMPV, 1 HRV). mCARMEN missed 1 virus-positive specimen detected by NGS, a partial FLUAV genome. We found no sequencing reads spanning the mCARMEN amplicon, suggesting that degradation was responsible for the result. mCARMEN detected virus in 2 specimens (1 FLUAV, 1 HRV) where NGS did not detect any viral reads. While we cannot rule out false positive results, metagenomic sequencing has been shown to have poor sensitivity for low viral copy samples<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Lemieux, J. E. et al. Phylogenetic analysis of SARS-CoV-2 in Boston highlights the impact of superspreading events. Science 371, eabe326 (2021)." href="/articles/s41591-022-01734-1#ref-CR5" id="ref-link-section-d38468781e1775">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Xuan, J., Yu, Y., Qing, T., Guo, L. &amp; Shi, L. Next-generation sequencing in the clinic: promises and challenges. Cancer Lett. 340, 284–295 (2013)." href="/articles/s41591-022-01734-1#ref-CR19" id="ref-link-section-d38468781e1778">19</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="García Fernández, X., Álvarez-Argüelles, M. E., Rojo, S. &amp; de-Oña, M. Stability of viral RNA in clinical specimens for viral diagnosis. Enferm. Infecc. Microbiol. Clin. (Engl. Ed.) 38, 297–298 (2020)." href="/articles/s41591-022-01734-1#ref-CR42" id="ref-link-section-d38468781e1781">42</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec4">Streamlining mCARMEN for future clinical use</h3><p>With a drive toward clinical applications, we aimed to optimize the mCARMEN workflow. To do so, we decreased the manual labor and processing time from &gt;8 h to &lt;5 h by implementing automated RNA extraction, using a single-step RNA-to-DNA amplification with 1 primer pool and reducing the duration of detection readout (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig2">2a</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig8">2a</a>). We then preliminarily evaluated the optimized workflow on 21 SARS-CoV-2-positive and 8 SARS-CoV-2-negative patient specimens and found greater sensitivity over the original two-step amplification method (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig8">2b</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Evaluation of an automated and condensed mCARMEN workflow."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Evaluation of an automated and condensed mCARMEN workflow.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="413"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Schematic of the streamlined mCARMEN workflow for testing of 188 patient specimens using a panel of 9 human respiratory viruses, RVP (SARS-CoV-2, HCoV-HKU1, HCoV-OC43, HCoV-NL63, FLUAV, FLUBV, HPIV-3, HRSV, HMPV) and a human internal control (RNase P). <b>b</b>, Concordance of RVP and RT–qPCR results. Top, RT–qPCR results were obtained from concurrent testing with mCARMEN. Bottom, RT–qPCR results were obtained from the original testing. <b>c</b>, Scaled normalized fluorescence at 1 h post-reaction initiation for 525 NP swabs ranked by increasing SARS-CoV-2 signal (blue); the respective RNase P signal (gray) is also shown. Normalized fluorescence signal (FAM/ROX) scaled from 0 to 1. The NTC-noMg signal was set as 0 and the maximum normalized fluorescence value at 1 h was set as 1. Dashed horizontal line: threshold for RVP positivity, calculated by multiplying the NTC-extract fluorescence value by 1.8; NTC-extract: no template control taken through the entire workflow. Gray X represents a failed sample excluded from concordance calculations and other analyses. <b>d</b>, Scatter plot of the scaled normalized fluorescence values from <b>b</b> compared to viral Ct values obtained from concurrent testing with the CDC 2019-nCoV Kit. Green: positive SARS-CoV-2 signal detected by both RVP and RT–qPCR; gray: inconclusive RT–qPCR result indicating that one or two of the three technical replicates were undetermined; black: undetermined RT–qPCR result indicating that all three technical replicates were negative for SARS-CoV-2. Dashed horizontal lines: threshold for RVP positivity. Solid vertical line: Ct value of 40 (CDC positivity cutoff).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>For an end-to-end mCARMEN workflow, we developed software to be used alongside clinical testing to provide patient diagnoses (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">2</a>). The software uses the final image at 1 h post-reaction initiation as input, then automatically validates the controls to make 1 of 3 calls—‘detected’, ‘not detected’ or ‘invalid’—for each combination of sample and crRNA.</p><p>Lastly, we wanted to condense mCARMEN for focused clinical use and did so by developing a respiratory virus panel (RVP) to detect nine of the most clinically relevant viruses (SARS-CoV-2, HCoV-HKU1, HCoV-OC43, HCoV-NL63, FLUAV, influenza B virus (FLUBV), human parainfluenza virus serotype 3 (HPIV-3), human respiratory syncytial virus (HRSV) and HMPV) and a human internal control, (RNase P). These nine viruses were included in the RVP based on if they heavily circulate in the population and have capacity to cause respiratory virus symptoms, while others were excluded if genomic diversity was difficult to account for concisely, such as HRV<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Palmenberg, A. C. et al. Sequencing and analyses of all known human rhinovirus genomes reveal structure and evolution. Science 324, 55–59 (2009)." href="/articles/s41591-022-01734-1#ref-CR43" id="ref-link-section-d38468781e1845">43</a></sup>. We first conducted range-determining limit of detection (LOD) studies for the nine viruses on the mCARMEN RVP in a research laboratory. The preliminary LOD was within the range of 100–1,000 copies μl<sup>−1</sup> for SARS-CoV-2, FLUAV, FLUBV, HCoV-HKU1, HCoV-NL63 and HCoV-OC42 and 1,000–20,000 copies μl<sup>−1</sup> for HPIV-3, HMPV and HRSV (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig9">3a</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">6</a>), with robust performance from the SARS-CoV-2 crRNA and all RVP crRNAs in combination (median areas under the curve (AUCs) of 1 and 0.989, respectively) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig9">3b–g</a>).</p><p>To benchmark mCARMEN RVP performance to comparator assay results, we analyzed 385 SARS-CoV-2-positive and 140 SARS-CoV-2-negative patient specimens and compared these results to both prior and concurrent RT–qPCR evaluation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig2">2b</a>). By the time of comparative evaluation, the number of RT–qPCR-positive specimens dropped from 385 to 316, suggesting substantial viral degradation either from extended sample storage or multiple freeze–thaw cycles. Nonetheless, mCARMEN identified all 316 (100% sensitivity) of the concurrent RT–qPCR-positive specimens. We noted that mCARMEN further detected SARS-CoV-2 in 42 specimens that tested positive by prior RT–qPCR but were missed by concurrent RT–qPCR testing, suggesting that mCARMEN is more robust to low viral quantity.</p><p>To confirm RT–qPCR sensitivity relative to mCARMEN, we tested the impact of multiple freeze–thaw cycles at several concentrations of SARS-CoV-2 seed stock on assay reproducibility. We found that the freeze–thaw cycles had no impact on mCARMEN sensitivity across all concentrations while RT–qPCR was negatively impacted by freeze–thaw cycles at the lowest concentration, suggesting that the 42 discrepant specimens had low initial viral quantities (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">3a–d</a>).</p><p>Indeed, if we categorize putative true positives as all specimens that tested positive by prior RT–qPCR as well as present-day RT–qPCR and/or mCARMEN, mCARMEN would have 100% sensitivity compared to 88% for RT–qPCR. mCARMEN also detected SARS-CoV-2 in three specimens that tested negative by both prior and concurrent RT–qPCR (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">3e</a>). While we cannot rule out the possibility of false positives, several pieces of evidence suggest they are more likely to be true positives: mCARMEN demonstrated higher sensitivity over concurrent RT–qPCR testing, these specimens were from suspected SARS-CoV-2 cases based on clinical features and mCARMEN did not detect SARS-CoV-2 in any clinical specimens before the pandemic (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig1">1</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">1</a>).</p><p>We further evaluated the analytical sensitivity of RVP by correlating RVP fluorescence signals to Ct values obtained from concurrent RT–qPCR testing (CDC 2019-nCoV Kit). Of the 316 specimens positive for SARS-CoV-2 by mCARMEN RVP and both RT–qPCR results, 217 had Ct values &lt;30, suggesting moderate-to-high viral genome copies. By RVP, all 217 specimens (100%) reached signal saturation by 1 h post-reaction initiation (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig2">2c,d</a> and Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">3f</a>). The remaining 100 specimens had Ct values between 30 and 36 and all but 6 samples (94%) reached signal saturation by 1 h. In total, 98% (311 out of 316) of the specimens reached saturation by 1 h indicating that mCARMEN can rapidly deem viral positivity status for a range of Ct values. Even 17 of the 42 (approximately 40%) RVP-positive specimens, but not concurrently RT–qPCR-positive, reached saturation by 1 h; the slower saturation of the remaining 25 specimens further suggested detection issues caused by low viral genome copy number (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig10">4a–c</a>). We also evaluated RVP fluorescence for detecting an internal control and human housekeeping gene, RNase P. We found that 520 of the 525 (99%) patient specimens reached saturation for RNase P by 1 h (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig10">4d</a>, described in the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01734-1#Sec10">Methods</a>).</p><p>Additionally, we used unbiased metagenomic NGS as a metric to evaluate RVP performance. As controls for NGS, we sequenced a set of true SARS-CoV-2-negative specimens (that is, negative by all three results, RVP and 2× RT–qPCR) (<i>n</i> = 16)) and true SARS-CoV-2-positives (<i>n</i> = 15) with a range of Ct values (15–34) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig10">4e</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">5</a>). Fifteen out of the 16 true negatives had no more than 2 reads mapped to the SARS-CoV-2 genome, in line with &lt;10 reads expected for negative specimens, while 1 specimen had 11 reads by NGS (an average of approximately 8.8 million reads per specimen). All true positive specimens had &gt;10 aligned viral reads, ranging from 16 to 802,306 reads, by NGS (100% sensitivity). Only specimens with Ct values &lt;25 (<i>n</i> = 8) were able to assemble complete genomes, while specimens with Ct values &gt;25 (<i>n</i> = 7) had &lt;200 reads mapped to SARS-CoV-2.</p><p>Using NGS, we then evaluated 22 specimens discordant between RVP and RT–qPCR results and 8 specimens for which RVP detected other respiratory viruses. The 22 discordant specimens included 13 positive by RVP and previous testing but concurrently negative by RT–qPCR, 6 positive by previous testing but negative by concurrent RVP and RT–qPCR and 3 positive by RVP but negative by both RT–qPCR results. All but 1 of the 22 (95%) discordant specimens had &lt;10 viral reads by NGS. The single specimen with &gt;10 reads was positive by RVP and prior RT–qPCR but not concurrent testing, yet just 22 reads mapped to SARS-CoV-2. NGS additionally failed to detect other respiratory viruses in the eight RVP-positive specimens. RVP identified 4 SARS-CoV-2 coinfections (2 HCoV-HKU1, 1 HPIV-3 and 1 HRSV) and 4 viruses in SARS-CoV-2-negative specimens (3 FLUAV and 1 HCoV-NL63). Given these specimens also had &lt;10 viral reads aligned by NGS, we can neither validate our results as positive nor rule out the possibility of false negatives by NGS; these samples are likely of low viral quantity suggesting that mCARMEN and RT–qPCR are more sensitive.</p><h3 class="c-article__sub-heading" id="Sec5">Evaluation of RVP performance in a clinical setting</h3><p>We implemented mCARMEN RVP in the Clinical Laboratory Improvement Amendments (CLIA)-certified Clinical Microbiology Laboratory at Massachusetts General Hospital (MGH) to establish assay sensitivity and specificity for clinical validation according to U.S. Food and Drug Administration (FDA) guidelines. We first evaluated the LOD, defined as the lowest concentration yielding positive results for at least 19 of 20 replicates. After recapitulating the 9 viral targets on RVP, we found that the LOD for HCoV-HKU1, HCoV-NL63, HCoV-OC43, FLUAV and FLUBV were 500 copies μl<sup>−1</sup> while HMPV and SARS-CoV-2 were 1,000 copies μl<sup>−1</sup> and HPIV-3 and HRSV were 10,000 copies μl<sup>−1</sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig3">3a,b</a>, Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig11">5a</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">7</a>). The LOD likely varies between viral targets for a few reasons: the crRNAs have varying activity levels on their intended target and differing input materials were used based on sample availability.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="Clinical evaluation of RVP in a CLIA-certified laboratory."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: Clinical evaluation of RVP in a CLIA-certified laboratory.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="509"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Workflow for LOD studies according to the FDA guidelines for establishing assay sensitivity. <b>b</b>, Fluorescence values for SARS-CoV-2 target LOD at the indicated SARS-CoV-2 concentrations; 20 replicates were performed. <b>c</b>, Normalized fluorescence signal at 1 h post-reaction initiation for each virus on the RVP using the on-target sequence (should detect), closely related sequences (should not detect) and an NTC (see Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">2</a> for sequence information). Should and should not detect activities were based on ADAPT design predictions (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01734-1#Sec10">Methods</a>). Closest to further relatives are based on percentage nucleotide homology to the corresponding on-target sequence. <b>d</b>, Positive and negative percentage agreement (PPA, NPA, respectively) for each virus on the RVP, calculated based on clinical data in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">5</a>. Dashed line: FDA agreement cutoff for assay performance. <b>e</b>, Concordance of the performance of RVP to concurrent comparator assays for 166 retrospective patient specimens tested (left) and 150 contrived samples (right).</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>After establishing the single-analyte LODs, we asked whether coinfections impacted the sensitivity for each virus detected by RVP. To do so, we added SARS-CoV-2 at a constant, 2× LOD concentration to the remaining 8 viruses on the RVP at varying concentrations at and above their respective LOD (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig11">5b</a>). We observed no loss in our ability to detect SARS-CoV-2. However, we noticed a decrease in signal intensity for the other viruses at lower concentrations, yet only one virus, HPIV-3, had a tenfold higher LOD.</p><p>Although we observed no cross-reactivity between RVP panel members in the research setting (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig1">1</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig2">2</a>), we followed FDA guidelines to conduct more stringent assay inclusivity and specificity analyses against common respiratory flora and other viral pathogens. In silico analysis revealed that the primers on RVP were &gt;92% inclusive of the known genetic diversity of each viral species, with additional inclusivity coming from crRNA target recognition, for an overall &gt;95% inclusivity (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">8</a>). When examining off-target activity in silico, the FDA defines cross-reactivity as &gt;80% homology between one of the primers or probes to any microorganism. We found no more than 75% homology between the RVP primer and crRNA sequences to other closely related human pathogens (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">9</a>). This implies that off-target detection will rarely, if ever, occur.</p><p>After in silico analysis, we evaluated RVP specificity experimentally. We computationally designed position-matched synthetic gene fragments from closely related viral species, including both human- and nonhuman-infecting species. When evaluating these gene fragments, only SARS-CoV-2 and bat coronavirus RaTG13 showed cross-reactivity (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig3">3c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig12">6</a>). However, this cross-reactivity was expected because the RaTG13 amplicon evaluated shares 100% nucleotide identity with the SARS-CoV-2 amplicon in our assay. We did not observe any cross-reactivity when using viral seed stocks, genomic RNA or synthetic RNA from ATCC or BEI (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">10</a>). Therefore, we found the RVP to have 100% analytical specificity.</p><p>Finally, the FDA recommends testing a minimum of 30 known-positive clinical specimens for each pathogen in an assay, as well as 30 negative specimens. Where positive specimens are not available, the FDA allows the creation of contrived samples by spiking viral genomic material at clinically relevant concentrations into a negative specimen. Each virus evaluated must have a minimum of 95% agreement performance, both positive percentage agreement (PPA) and negative percentage agreement (NPA), to clinically approved comparator assays.</p><p>At MGH, archived clinical specimens had been evaluated at the time of collection using one of two comparator assays: Cepheid Xpert Xpress SARS-CoV-2/Flu/RSV multiplexed assay or BioFire RP2.0 multiplexed assay (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig13">7</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">5</a>). These included 166 specimens with 137 total viral clinical results: 31 FLUAV, 30 SARS-CoV-2, 30 HRSV, 29 FLUBV, 8 HMPV, 5 HCoV-NL63, 1 FLUBV and HCoV-NL63 coinfection, 1 HCoV-HKU1, 1 HCoV-OC43 and 30 clinically negative. Given that these specimens can be degraded by multiple freeze–thaw cycles, we concurrently tested all specimens by BioFire RP2.0 or TaqPath COVID-19 Combo Kit. We also supplemented this evaluation with 30 contrived samples for each of the following viruses for which we did not have enough positive specimens: HCoV-HKU1, HCoV-OC43, HCoV-NL63, HPIV-3 and HMPV (described in the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01734-1#Sec10">Methods</a>), for a total of 150 contrived samples.</p><p>All of the RVP viral targets individually had 100% NPA, and all, except HMPV, had &gt;95% PPA to their respective previous comparator assay result, exceeding the minimum clinical performance set by the FDA (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig3">3d</a>). Of the 137 previously positive clinical results, mCARMEN correctly detected viral nucleic acids 95% (130 out of 137) of the time. For specimens that were evaluated concurrently, mCARMEN and the comparator assay had 9 discordant results (128 out of 137) with equivalent sensitivity to prior results for all but the HMPV specimens; BioFire did not detect virus in 3 specimens (1 FLUAV, 1 FLUBV and 1 HRSV) and mCARMEN did not detect virus in 6 specimens (1 FLUAV, 1 FLUBV, 1 HRSV and 3 HMPV). Both mCARMEN and BioFire identified 5 specimens with coinfections (HCoV-NL63 in a FLUAV specimen, HPIV-3 in a FLUBV specimen, HCoV-HKU-1 in 2 HRSV specimens and HCoV-NL63 in an HRSV specimen). Together with the original clinically detected coinfection, there were 6 (1.1%) coinfections in our specimen set (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig11">5c</a>). Overall, mCARMEN and BioFire were 99.4% (1,485 out of 1,494 individual tests) concordant (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig3">3e</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig13">7</a>). For the contrived samples, mCARMEN correctly identified 99% (148 out of 150) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig3">3e</a>).</p><p>We used unbiased metagenomic NGS to further evaluate the 9 discordant specimens (2 FLUAV, 2 FLUBV, 2 HRSV and 3 HMPV), generating an average of 13 million reads per specimen. Either no viral reads were present by NGS or partial genomes were assembled but the RVP amplicon was missing, making it unlikely for our assay to return a positive result (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig13">7</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">5</a>). Based on these results and our previous NGS testing, which indicated that NGS was not as sensitive as RVP or the comparator assays, we cannot determine the viral positivity status of these specimens.</p><h3 class="c-article__sub-heading" id="Sec6">Quantification of viral copies using Cas12 and Cas13 kinetics</h3><p>Similar to widely used multiplexed approaches, such as BioFire<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Eckbo, E. J. et al. Evaluation of the BioFire® COVID-19 test and Respiratory Panel 2.1 for rapid identification of SARS-CoV-2 in nasopharyngeal swab samples. Diagn. Microbiol. Infect. Dis. 99, 115260 (2021)." href="/articles/s41591-022-01734-1#ref-CR13" id="ref-link-section-d38468781e2083">13</a></sup>, the original design of CARMEN<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Ackerman, C. M. et al. Massively multiplexed nucleic acid detection with Cas13. Nature 582, 277–282 (2020)." href="/articles/s41591-022-01734-1#ref-CR40" id="ref-link-section-d38468781e2087">40</a></sup> did not provide a quantitative assessment of viral genome copies present in a sample. Establishing the total viral quantity in a patient is important for assessing the stage of infection, transmission risk and most effective treatment plan<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Zou, L. et al. SARS-CoV-2 viral load in upper respiratory specimens of infected patients. N. Engl. J. Med. 382, 1177–1179 (2020)." href="/articles/s41591-022-01734-1#ref-CR8" id="ref-link-section-d38468781e2091">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Wölfel, R. et al. Virological assessment of hospitalized patients with COVID-2019. Nature 581, 465–469 (2020)." href="/articles/s41591-022-01734-1#ref-CR9" id="ref-link-section-d38468781e2094">9</a></sup>. The criterion standard assay for sample quantification, RT–qPCR, leverages the standard curve—serial dilutions of a given target at a known concentration—as a means of using Ct values to approximate viral quantity<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="Mackay, I. M., Arden, K. E. &amp; Nitsche, A. Real-time PCR in virology. Nucleic Acids Res. 30, 1292–1305 (2002)." href="/articles/s41591-022-01734-1#ref-CR44" id="ref-link-section-d38468781e2098">44</a></sup>. We wanted to determine if a similar approach could be applied to mCARMEN.</p><p>To make mCARMEN quantitative, we took advantage of the existence of multiple CRISPR–Cas proteins with differing reaction kinetics and enzymatic activities as well as the three fluorescent channels detected by the Fluidigm Biomark (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig4">4a</a>). We incorporated DNA-targeting CRISPR–Cas12 into the Cas13 reaction and used protein-specific reporters in different fluorescent channels, HEX and FAM, respectively to maximize our multiplexing capabilities. To capture reaction kinetics, images of the IFC chip were taken every 5 min for 3 h to generate sigmoidal curves from the fluorescent signals over time. When considering enzymatic activities, Cas13 has enhanced sensitivity compared to Cas12 since the process of in vitro transcribing the double-stranded DNA (dsDNA) sample input for Cas13 detection results in an increased starting concentration. Thus, we used Cas12 to capture the kinetic curves of higher copy material on the standard curve and Cas13 to capture lower copy material.</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Viral quantification using Cas12 and Cas13 in combination."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Viral quantification using Cas12 and Cas13 in combination.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="588"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Schematic of the procedure for the use of both Cas12 and Cas13 for quantification of viral copy number in samples. Fluorescence was plotted over time to determine the time at which fluorescence signal reaches 50% calculated as the IC<sub>50</sub> value at each concentration using a sigmoidal 4PL fit. The IC<sub>50</sub> values were then plotted by concentration to generate a semilog line with an <i>R</i><sup>2</sup> &gt; 0.8 for Cas12 and Cas13 individually. After line generation, the IC<sub>50</sub> value of each patient sample was plotted onto these lines to determine viral copies μl<sup>−1</sup>. <b>b</b>, Normalized fluorescence ratio of Cas13 crRNA (top) and Cas12 crRNA (bottom) signal over time at varying concentrations of synthetic SARS-CoV-2 Orf1ab RNA. <b>c</b>, Plots showing the semilog lines generated by IC<sub>50</sub> values from the Cas12 and Cas13 crRNA signals and the Ct values from RT–qPCR for more than four target concentrations of SARS-CoV-2 (left) and FLUAV (right) synthetic RNA. Blue: Cas13; orange: Cas12; gray: Ct from RT–qPCR. <b>d</b>, Comparison of mCARMEN IC<sub>50</sub> values to RT–qPCR Ct values using linear regression, with the best line fit shown as a dashed line. Black: SARS-CoV-2; gray: FLUAV.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We integrated our quantification efforts into the RVP because this assay was evaluated extensively in both research and clinical settings. We manually designed Cas12 crRNAs in the same region of the viral genome that the RVP Cas13 crRNAs target for a two-step standard curve generation on the same target amplicon for Cas12 and Cas13 proteins individually. The first step requires plotting the fluorescence for a range of concentrations (Cas12: 10<sup>7</sup>–10<sup>3</sup> copies μl<sup>−1</sup>; Cas13: 10<sup>3</sup>–10<sup>0</sup> copies μl<sup>−1</sup>) at each time point to calculate the time at which the fluorescence intensity concentration reaches 50% (IC50) through a sigmoidal, four-parameter logistic (4PL) curve, <i>R</i><sup>2</sup> &gt; 0.9 (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig4">4b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig14">8</a>). In some cases, we could not determine the IC<sub>50</sub> value because signal saturation occurred too quickly or not at all; therefore, that concentration was excluded from the analysis. In the second step, we plotted the IC<sub>50</sub> values onto a semilog line, where concentration is logarithmic and time is linear, to generate the standard curves (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig4">4c</a>). We compared these results to a standard curve generated from RT–qPCR using the same serial dilutions and found a linear relationship between SARS-CoV-2 and FLUAV IC<sub>50</sub> values to Ct values (<i>R</i><sup>2</sup> = 0.901 and 0.881, respectively) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig4">4d</a>). Taken together, these results suggest that by using Cas12 and Cas13 in combination, we could extrapolate viral quantification—spanning a 10<sup>0</sup>–10<sup>6</sup> range of target concentrations—from patient specimens with performance similar to RT–qPCR.</p><h3 class="c-article__sub-heading" id="Sec7">Allelic discrimination distinguishes between SARS-CoV-2 variant lineages</h3><p>Since current clinical diagnostics are not well positioned to identify mutations—single-nucleotide polymorphisms (SNPs), insertions or deletions—carried in SARS-CoV-2 variant lineages<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 6" title="Harvey, W. T. et al. SARS-CoV-2 variants, spike mutations and immune escape. Nat. Rev. Microbiol. 19, 409–424 (2021)." href="/articles/s41591-022-01734-1#ref-CR6" id="ref-link-section-d38468781e2213">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Burki, T. Understanding variants of SARS-CoV-2. Lancet 397, 462 (2021)." href="/articles/s41591-022-01734-1#ref-CR17" id="ref-link-section-d38468781e2216">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Borges, V. &amp; et al. Tracking SARS-CoV-2 lineage B.1.1.7 dissemination: insights from nationwide spike gene target failure (SGTF) and spike gene late detection (SGTL) data, Portugal, week 49 2020 to week 3 2021. Euro Surveill. 26, 2100131 (2021)." href="/articles/s41591-022-01734-1#ref-CR18" id="ref-link-section-d38468781e2219">18</a></sup>, we wanted to develop a single platform with both diagnostic and surveillance capabilities for comprehensive detection of 26 SARS-CoV-2 Spike gene mutations. We selected these 26 mutations to distinguish between or detect mutations shared among the Alpha, Beta, Gamma, Delta and Epsilon variant lineages (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">11</a>; B.1.1.7, B.1.351, P.1, B.1.617.2 and B.1.427/9 using the Pango nomenclature system, respectively; World Health Organization (WHO) Tracking SARS-CoV-2 variants) and then used a generative sequence design algorithm (Mantena, S. et al., manuscript in preparation) to produce crRNAs for allelic discrimination.</p><p>With the continuous emergence of mutations that can lead to increased transmissibility or enhanced virulence, we also wanted to greatly streamline assay generation for each new SARS-CoV-2 mutation or variant. Thus, we developed an easily adaptable method to track these changes that we called the mCARMEN variant identification panel (VIP). VIP has two nonoverlapping primer pair sets within conserved regions of the Spike gene to amplify the full-length sequence for use with any crRNA pair. These 26 crRNA pairs, individually or in combination, allowed us to track existing variants and identify emerging variants (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig5">5a</a>). Initially, we tested over 60 combinations of crRNAs on unamplified synthetic material to identify the crRNA pairs with the largest fluorescence ratio of expected divided by unexpected signal for each mutation (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">4</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-5" data-title="SARS-CoV-2 variant identification using SNP-determining Cas13 crRNA combinations."><figure><figcaption><b id="Fig5" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 5: SARS-CoV-2 variant identification using SNP-determining Cas13 crRNA combinations.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/5" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig5_HTML.png?as=webp"><img aria-describedby="Fig5" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig5_HTML.png" alt="figure 5" loading="lazy" width="685" height="525"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-5-desc"><p><b>a</b>, Schematic of the procedure for the mCARMEN VIP. The entire SARS-CoV-2 Spike gene was amplified to detect the presence of a mutation by differentiating between the ancestral, no mutation, sequence or the derived, mutation-containing sequence with highly specific Cas13 crRNAs. <b>b</b>, SARS-CoV-2 viral seed stocks from ancestral (Washington isolate), Alpha, Beta, Gamma, Delta or Epsilon variant lineages (10<sup>6</sup> copies μl<sup>−1</sup>) were amplified by 2 primer pairs and tested for the presence or absence of Spike gene mutations. Data are shown as the log<sub>2</sub> of the maximum crRNA fluorescence ratio at any time point up to 3 h post-reaction initiation. log<sub>2</sub> fluorescence ratios were calculated by (−1)*ancestral/mutation or mutation/ancestral representing either the presence of the ancestral (blue) or derived sequence (purple), respectively. The single asterisk indicates that the particular mutation was confirmed by NGS. <b>c</b>, Comparison of the performance of and NGS on 101 SARS-CoV-2-positive variant patient specimens, based on the final variant call as assessed by unique combinations of mutations (see the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01734-1#Sec10">Methods</a> for details). Black: variant correctly identified by both VIP and NGS; yellow: NGS only; green: VIP only. <b>d</b>, Plot showing the log<sub>2</sub> maximum crRNA fluorescence ratio of mutation/ancestral (positive, derived) or ancestral/mutation (negative, ancestral) at any time point up to 3 h post-reaction initiation for 101 variant patient specimens tested for various SNPs by VIP. Patient specimens are classified as Alpha (purple), Beta (blue), Gamma (teal), Delta (green) or Epsilon (yellow) based on a combination of mutations expected for that variant lineage. <b>e</b>, Analysis of how VIP compared to NGS for the 101 variant patient specimens. Black: mutation correctly identified by both VIP and NGS; yellow: NGS only; green: VIP only; gray: ancestral for VIP and NGS.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/5" data-track-dest="link:Figure5 Full size image" aria-label="Full size image figure 5" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We validated the flexible VIP method by testing RNA extracted from SARS-CoV-2 viral seed stocks, for the ancestral (Washington isolate: USA-WA1; ATCC) lineage and the Alpha, Beta, Gamma, Delta and Epsilon lineages (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig5">5b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig15">9</a>). As expected, the Washington SARS-CoV-2 viral seed stock isolate showed ancestral signals for all mutations tested. Alpha, Beta, Gamma, Delta and Epsilon had expected signals for every mutation confirmed by NGS (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">11</a>; description in the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01734-1#Sec10">Methods</a>). Although each crRNA had different kinetics owing to varying hit-calling thresholds, we almost always observed a higher expected signal above the unexpected signal, which is important in the prevention of false positive results (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">5</a>).</p><p>For clinical relevance, we developed an automated variant calling procedure that evaluates the mutation-specific signal in SARS-CoV-2-positive patient specimens and returns a variant lineage result (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">6a</a>; described in the <a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01734-1#Sec10">Methods</a>). For some mutations at the same or similar genomic position, we observed cross-reactive signals that we overcame by comparing the maximum fluorescence ratios between those mutations and assigning the positive call to the higher of the two (Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">6b</a>).</p><p>We applied VIP and the analysis pipeline to identify the variant lineage in 101 known SARS-CoV-2-positive patient specimens: 24 Alpha, 23 Beta, 24 Gamma, 6 Delta and 24 Epsilon. Of the 101 specimens with NGS results, all but 3 (97%) specimens (1 Beta and 2 Gammas) were given the correct variant lineage identification (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig5">5c,d</a>, Supplementary Fig. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">7a</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">5</a>). The Beta specimen had signal for a Beta-specific SNP, K417N, but also had signal for Δ156/57, a Delta-specific SNP. The Gamma specimens had no unique signals and shared signals for mutations overlapping with the Beta lineage resulting in a ‘variant not identified’ call.</p><p>Focusing on the results for the individual mutations themselves, we found only one mutation, E484K, had more than five specimens that differed in their results between NGS and VIP (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig5">5e</a>). The 7 E484K discrepancies are attributed to the cross-reactive signals between E484K and T478K; thus, new crRNA designs are likely needed to optimally differentiate these signals (Supplementary Figs. <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">6b</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM1">7b,c</a>). Altogether, we found that VIP had 97.7% concordance to NGS at allelic discrimination.</p><h3 class="c-article__sub-heading" id="Sec8">VIP identifies Omicron at local and statewide levels</h3><p>In November 2021, the SARS-CoV-2 variant lineage Omicron (BA.1) was first identified by NGS in South Africa and was quickly associated with a rapid increase in case counts (WHO Coronavirus (COVID-19) Dashboard)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Pulliam, J. R. C. et al. Increased risk of SARS-CoV-2 reinfection associated with emergence of the Omicron variant in South Africa. Preprint at medRxiv 
                  https://doi.org/10.1101/2021.11.11.21266068
                  
                 (2021)." href="/articles/s41591-022-01734-1#ref-CR45" id="ref-link-section-d38468781e2346">45</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Brandal, L. T. et al. Outbreak caused by the SARS-CoV-2 Omicron variant in Norway, November to December 2021. Euro. Surveill. 26, 2101147 (2021)." href="/articles/s41591-022-01734-1#ref-CR46" id="ref-link-section-d38468781e2349">46</a></sup>. By December, Omicron was detected in the USA and has since driven the recent global COVID-19 wave<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="CDC COVID-19 Response Team SARS-CoV-2 B.1.1.529 (Omicron) Variant—United States, December 1–8, 2021. MMWR Morb. Mortal. Wkly. Rep. 70, 1731–1734 (2021)." href="/articles/s41591-022-01734-1#ref-CR47" id="ref-link-section-d38468781e2353">47</a></sup>. However, detecting and tracking Omicron has been challenging, with NGS results lagging behind by 7–14 d from collection date. Although S gene target failure (SGTF) by the TaqPath COVID-19 Combo Kit (Thermo Fisher Scientific) can be associated with Omicron, the failure is not specific to Omicron<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 47" title="CDC COVID-19 Response Team SARS-CoV-2 B.1.1.529 (Omicron) Variant—United States, December 1–8, 2021. MMWR Morb. Mortal. Wkly. Rep. 70, 1731–1734 (2021)." href="/articles/s41591-022-01734-1#ref-CR47" id="ref-link-section-d38468781e2357">47</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 48" title="Li, A., Maier, A., Carter, M. &amp; Guan, T. H. Omicron and S-gene target failure cases in the highest COVID-19 case rate region in Canada—December 2021. J. Med. Virol. 
                  https://doi.org/10.1002/jmv.27562
                  
                 (2021)." href="/articles/s41591-022-01734-1#ref-CR48" id="ref-link-section-d38468781e2360">48</a></sup>. The swift emergence of Omicron has revealed a need for a nucleic acid-based diagnostic with turnaround times similar to RT–qPCR but with mutation-specific information that only NGS currently provides. mCARMEN is uniquely poised to fulfill this need by providing results the same day approximately one week before NGS.</p><p>At the time of Omicron emergence, mCARMEN VIP could already uniquely differentiate it from the other SARS-CoV-2 variants by specifically detecting nine Omicron-tagging mutations among our variant panel (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig6">6a</a>). The unique combination of these Spike gene mutations allows for the specificity required to make the proper variant lineage call. In particular, just the combination of S477N and N501Y covers 98.6% of Omicron sequences in GISAID and is 99.9% specific to Omicron (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig16">10a</a>). We rapidly applied VIP to 430 specimens collected at Harvard University CLIA Laboratory (HUCL) from 6 to 16 December and found that the rate of Omicron increased from 15% to 80% in 10 d, overtaking the previously predominant variant, Delta (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig6">6b</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig16">10b</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-6" data-title="Rapid and specific identification of the Omicron variant using the mCARMEN VIP."><figure><figcaption><b id="Fig6" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 6: Rapid and specific identification of the Omicron variant using the mCARMEN VIP.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/6" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig6_HTML.png?as=webp"><img aria-describedby="Fig6" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig6_HTML.png" alt="figure 6" loading="lazy" width="685" height="536"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-6-desc"><p><b>a</b>, Expected mutations across six SARS-CoV-2 variant lineages (Alpha, Beta, Gamma, Delta, Epsilon, Omicron) detectable by VIP. The blue boxes represent the presence of a mutation; the white boxes represent the absence of a mutation. <b>b</b>, Proportion of Omicron and Delta variant lineages as assessed by VIP in specimens (<i>n</i> = 430) collected on 6–16 December at HUCL. Green: Omicron; black: Delta. The error bars represent binomial sampling 95% confidence intervals (CIs). <b>c</b>, Proportion of Omicron and Delta variant lineages as assessed by VIP and NGS in specimens (<i>n</i> = 1,557) collected on 13–22 December from throughout the state of Massachusetts. Blue: Omicron; black: Delta; closed circles: mCARMEN; open circles: NGS. The error bars represent binomial sampling 95% CIs. <b>d</b>, Scatter plot of the proportion of Omicron based on the VIP and NGS variant lineage results. The linear regression line fit is shown as a blue line; <i>R</i><sup>2</sup> = 0.998. <b>e</b>, Comparison of the time delay from specimen extraction to determination of the variant lineage for VIP and NGS. The data represent the proportion of Omicron from specimens collected at HUCL and within the state of Massachusetts. Blue closed circles: VIP Massachusetts specimens from <b>c</b>; blue open circles: NGS Massachusetts specimens from <b>c</b>; green: VIP HUCL specimens from <b>b</b>.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01734-1/figures/6" data-track-dest="link:Figure6 Full size image" aria-label="Full size image figure 6" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Based on the public health importance of this data, the Massachusetts Department of Public Health (MADPH) requested our support for Omicron surveillance across the state. Using the mCARMEN VIP, we tested 1,557 specimens collected across the state of Massachusetts for the presence of Delta or Omicron from 13 to 22 December 2021. The rate of Omicron increased from approximately 20% to 77% across Massachusetts in 10 d (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig6">6c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig16">10c</a>). In partnership with MADPH and the Broad Genomics Platform, we were able to confirm the mCARMEN variant lineage results with lineage results determined by NGS and found 99.5% (1,549 out of 1,557) concordance between mCARMEN and NGS (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig6">6d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig16">10d</a>). Of the eight discordant samples, seven had low signal for all mutations evaluated by VIP, suggesting low viral quantity. The remaining discordant specimen had clear signal for several Omicron-specific mutations; yet, by NGS, it had Delta signatures, which would suggest likely contamination or sample swap in one of the two tests. In all, the mCARMEN VIP was applied in real-time to a local Omicron outbreak and a statewide Omicron wave with near-perfect concordance to NGS, by providing results the same or following day while NGS lagged behind by approximately 4–7 d (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig6">6e</a>).</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec9-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec9">Discussion</h2><div class="c-article-section__content" id="Sec9-content"><p>In this study, we report mCARMEN, a high-throughput, multiplexed and microfluidic diagnostic and surveillance platform with panels for respiratory viruses and SARS-CoV-2 variants that can be parallelized to test 300–550 patient specimens in an 8-h working day. To make mCARMEN a clinically relevant technology, we built on CARMEN v.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Ackerman, C. M. et al. Massively multiplexed nucleic acid detection with Cas13. Nature 582, 277–282 (2020)." href="/articles/s41591-022-01734-1#ref-CR40" id="ref-link-section-d38468781e2460">40</a></sup>) by streamlining the workflow and incorporating commercially available Fluidigm instrumentation. We validated mCARMEN on 2,881 patient specimens to detect 9–21 human respiratory viruses (RVP) or SARS-CoV-2 variant mutations (VIP) with high concordance to comparator assays that passed the FDA’s performance criteria for all but 1 virus. Notably, when testing previously positive clinical specimens, we found that a substantial proportion were not positive by concurrent testing but were positive by mCARMEN. This suggests sample degradation issues—a known problem when detecting RNA viruses in clinical specimens<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="García Fernández, X., Álvarez-Argüelles, M. E., Rojo, S. &amp; de-Oña, M. Stability of viral RNA in clinical specimens for viral diagnosis. Enferm. Infecc. Microbiol. Clin. (Engl. Ed.) 38, 297–298 (2020)." href="/articles/s41591-022-01734-1#ref-CR42" id="ref-link-section-d38468781e2464">42</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 49" title="Kirkland, P. D. &amp; Frost, M. J. The impact of viral transport media on PCR assay results for the detection of nucleic acid from SARS-CoV-2. Pathology 52, 811–814 (2020)." href="/articles/s41591-022-01734-1#ref-CR49" id="ref-link-section-d38468781e2467">49</a></sup>—that mCARMEN is more robust at handling than RT–qPCR or NGS. Although we cannot rule out false positives, we did not detect SARS-CoV-2 in specimens before the pandemic and we had 100% concordance with true virus-negative specimens.</p><p>To enhance mCARMEN’s clinical diagnostic relevance and meld it with surveillance technology requirements, we further maximized its multiplexing capabilities by discriminating between mutations for variant lineage classification in patient specimens. Currently, variant lineage classification is only evaluated by NGS, which is costly and relies on specialized expertise found outside the clinic<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Burki, T. Understanding variants of SARS-CoV-2. Lancet 397, 462 (2021)." href="/articles/s41591-022-01734-1#ref-CR17" id="ref-link-section-d38468781e2474">17</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="Xuan, J., Yu, Y., Qing, T., Guo, L. &amp; Shi, L. Next-generation sequencing in the clinic: promises and challenges. Cancer Lett. 340, 284–295 (2013)." href="/articles/s41591-022-01734-1#ref-CR19" id="ref-link-section-d38468781e2477">19</a></sup>. VIP gives similarly rich information about key SARS-CoV-2 mutations at 5–10 times cheaper in cost, per sample, than NGS and is far more comprehensive than current nucleic acid-based diagnostics. Importantly, since we routinely design guides to preemptively identify mutations of interest in the Spike gene in preparation for emerging variants, VIP was poised to differentiate Omicron immediately. VIP allowed us to identify the rapid emergence of Omicron in Massachusetts approximately 8 d before NGS and provided us with specificity, unlike the widely used SGTF of RT–qPCR. Given the number of mutations detected by VIP, we expect to observe distinct mutation signatures between variant lineages that will allow us to differentiate these and future VOCs from each other without assay redesign.</p><p>We also adapted mCARMEN for dual Cas12 and Cas13 detection by capitalizing on the differing protein kinetics. A few groups have studied Cas12 and Cas13 reaction kinetics to inform assay quantification<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 50" title="Fozouni, P. et al. Amplification-free detection of SARS-CoV-2 with CRISPR–Cas13a and mobile phone microscopy. Cell 184, 323–333.e9 (2021)." href="/articles/s41591-022-01734-1#ref-CR50" id="ref-link-section-d38468781e2484">50</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 51" title="Nalefski, E. A. et al. Kinetic analysis of Cas12a and Cas13a RNA-guided nucleases for development of improved CRISPR-based diagnostics. iScience 24, 102996 (2021)." href="/articles/s41591-022-01734-1#ref-CR51" id="ref-link-section-d38468781e2487">51</a></sup> but the range of quantifiable concentrations has been limited due to reaction saturation. We expanded the quantifiable concentration range to 5–6 orders of magnitude, which is similar to RT–qPCR. These mCARMEN applications have the potential to provide a more holistic diagnosis to the patient but validation on patient samples is needed.</p><p>We rapidly developed mCARMEN for use in the COVID-19 pandemic but faced challenges during the clinical validation and approval process needed for a large-scale rollout. Specifically, it was difficult to obtain at least 30 previously confirmed clinical specimens for each virus on RVP with enough material available for extensive concurrent testing, while also facing specimen degradation issues that inevitably occur over time. Although our findings indicate that mCARMEN’s performance exceeds the FDA’s requirements for emergency use authorization (EUA), such authorization has not yet been granted.</p><p>Further work will be required to bring mCARMEN fully to the clinic, such as obtaining FDA approval, integrating RVP and VIP into a single panel, decreasing the amount of manual labor and easing Fluidigm equipment constraints. Nonetheless, we have taken substantial steps to streamline the assay workflow while enhancing sensitivity without sacrificing specificity. By combining high-throughput, multiplexed pathogen testing with variant tracking, the mCARMEN platform is highly scalable and amenable to clinical laboratory settings for the detection of respiratory pathogens and variants. This technology also has the potential to test for other types of infectious diseases<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 52" title="Thakku, S. G. et al. Multiplexed detection of bacterial nucleic acids using Cas13 in droplet microarrays. Preprint at bioRxiv 
                  https://doi.org/10.1101/2021.11.12.468388
                  
                 (2021)." href="/articles/s41591-022-01734-1#ref-CR52" id="ref-link-section-d38468781e2498">52</a></sup> and can be used on other sample types<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Ackerman, C. M. et al. Massively multiplexed nucleic acid detection with Cas13. Nature 582, 277–282 (2020)." href="/articles/s41591-022-01734-1#ref-CR40" id="ref-link-section-d38468781e2502">40</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 53" title="Barnes, K. G. et al. Deployable CRISPR-Cas13a diagnostic tools to detect and report Ebola and Lassa virus cases in real-time. Nat. Commun. 11, 4131 (2020)." href="/articles/s41591-022-01734-1#ref-CR53" id="ref-link-section-d38468781e2505">53</a></sup> to achieve even more comprehensive diagnostic and surveillance capabilities.</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec10-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec10">Methods</h2><div class="c-article-section__content" id="Sec10-content"><h3 class="c-article__sub-heading" id="Sec11">Patient samples and ethics statement</h3><p>Use of clinical excess of human specimens from patients with SARS-CoV-2 from the Broad Institute’s Genomics Platform CLIA Laboratory was approved by the MIT institutional review board (IRB) protocol no. 1612793224. Additional SARS-CoV-2 samples were collected from consented individuals under Harvard Longwood Campus IRB no. 20-1877 and covered by an exempt determination (EX-7295) at the Broad Institute. Other human-derived samples from patients with SARS-CoV-2 were collected by the CDC and determined to be nonhuman participant research; the Broad Office of Research Subject Protections determined these samples to be exempt (EX-7209). Human specimens from patients with SARS-CoV-2, HCoV-HKU1, HCoV-NL63, FLUAV, FLUBV, HRSV and HMPV were obtained under a waiver of consent from the Mass General Brigham IRB protocol no. 2019P003305. Researchers at Princeton were determined to be conducting not-engaged human participant research by the Princeton University IRB.</p><p>We gratefully acknowledge the personnel at Rhode Island Department of Public Health for the samples they provided, in particular: E. King, Associate Director of Health, and R. C. Huard, Chief Clinical Laboratory Scientist, both at the Division of State Laboratories and Medical Examiner at Rhode Island Department of Health.</p><h3 class="c-article__sub-heading" id="Sec12">General mCARMEN procedures</h3><p>A detailed description of running the mCARMEN RVP as a standard operating procedure can be found in Supplementary Note <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM4">1</a>.</p><h3 class="c-article__sub-heading" id="Sec13">Preparation and handling of synthetic materials</h3><p>crRNAs were synthesized by Integrated DNA Technologies, resuspended in nuclease-free water to 100 μM and further diluted for input into the detection reaction. Primer sequences were ordered from Eton or Integrated DNA Technologies, resuspended in nuclease-free water to 100 μM and further combined at varying concentrations for pooled amplification.</p><h3 class="c-article__sub-heading" id="Sec14">Preparation of IVT material</h3><p>DNA targets were ordered from Integrated DNA Technologies and in vitro-transcribed (IVT) using the HiScribe T7 High Yield RNA Synthesis Kit (New England Biolabs). Transcriptions were performed according to the manufacturer’s recommendations with a reaction volume of 20 μl that was incubated overnight at 37 °C. The transcribed RNA products were purified using RNAClean XP beads (Beckman Coulter) and quantified using NanoDrop One (Thermo Fisher Scientific). Depending on the experiment, the RNA was serially diluted from 10<sup>11</sup> down to 10<sup>−</sup><sup>3</sup> copies μl<sup>−1</sup> and used as input into the amplification reaction.</p><h3 class="c-article__sub-heading" id="Sec15">Manual or automated extraction</h3><p>RNA was manually extracted from input material using the QIAamp Viral RNA Mini Kit (QIAGEN) according to the manufacturer’s instructions. RNA was extracted from 140 μl of input material with carrier RNA and samples were eluted in 60 μl of nuclease-free water and stored at −80 °C until use. RNA was automatically extracted using the MagMAX DNA Multi-Sample Ultra 2.0 Kit on a KingFisher Flex Magnetic Particle Processor with 96 Deep Well Head (Thermo Fisher Scientific). RNA was extracted from 200 µl of input material and was run according to the ‘Extract RNA-automated method (200-μl sample input volume)’ protocol in the TaqPath COVID-19 Combo Kit Protocol, on pages 21–24. The MVP_2Wash_200_Flex protocol was used. Samples were eluted in 50 μl of elution solution and either directly added to the amplification reaction or stored at −80 °C until use.</p><h3 class="c-article__sub-heading" id="Sec16">QIAGEN or SuperScript IV amplification</h3><p>We followed the CARMEN v.1 platform for two-step reverse transcription amplification then transitioned to a single-step amplification reaction after the experiments depicted in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig1">1</a>. We used the QIAGEN OneStep RT–PCR Mix for Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig2">2</a>, <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig3">3</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig5">5</a> and the Invitrogen SuperScript IV One-Step RT–PCR System for Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig4">4</a>. For the QIAGEN OneStep RT–PCR, a total reaction volume of 50 μl was used with some modifications to the manufacturer’s recommended reagent volumes, specifically a 1.25× final concentration of OneStep RT–PCR buffer, 2× more QIAGEN enzyme mix and 20% RNA input. Final concentrations for all viral primers were 300 and 100 nM for the RNase P primers. The following thermal cycling conditions were used: (1) reverse transcription at 50 °C for 30 min; (2) initial PCR activation at 95 °C for 15 min; and (3) 40 cycles at 94 °C for 30 s, 58 °C for 30 s and 72 °C for 30 s. For the Invitrogen SuperScript IV One-Step RT–PCR, a total reaction volume of 25 μl with 20% RNA input and final primer concentrations at 1 μM were used. The following thermal cycling conditions were used: (1) reverse transcription at 50 °C for 10 min; (2) initial PCR activation at 98 °C for 2 min; (3) 35 cycles at 98 °C for 10 s, 60 °C for 10 s and 72 °C for 1 min 30 s; and (4) final extension at 72 °C for 5 min. See Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">4</a> for information on the primer sequences used in each mCARMEN panel.</p><h3 class="c-article__sub-heading" id="Sec17">Fluidigm detection</h3><p>The Cas13 detection reactions were made into two separate mixes—assay mix and sample mix—for loading onto a microfluidic IFC (depending on the experiment, either gene expression or genotyping IFCs were used in either a 96.96 or 192.24 format) (Fluidigm).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec18">Assay mix</h4><p>The assay mix contained LwaCas13a (GenScript) and on occasion LbaCas12a (New England Biolabs). Concentration varied with experiment: 1× Assay Loading Reagent (Fluidigm), 69U T7 RNA Polymerase mix (Lucigen) and crRNA concentration varied with experiment for a total volume of 16 μl per reaction. See below for details pertaining to each mCARMEN panel.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec19">Sample mix</h4><p>The sample mix contained 25.2 U RNase Inhibitor (New England Biolabs), 1× ROX reference dye (Invitrogen), 1× gene expression sample loading reagent (Fluidigm), 1 mM ATP, 1 mM GTP, 1 mM UTP, 1 mM CTP, 9 mM MgCl<sub>2</sub> in a nuclease assay buffer (40 mM Tris-HCl, pH 7.5, 1 mM dithiothreitol). Either a 500-nM quenched synthetic fluorescent RNA reporter (FAM/rUrUrUrUrUrUrU/3IABkFQ/ or VIC/rTrTrArTrTrArTrT/3IABkFQ/; Integrated DNA Technologies) or RNaseAlert v2 (Invitrogen) was used for a total volume of 12.6 μl. See below for details on each mCARMEN panel.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec20">IFC loading and run</h4><p>Syringe, actuation fluid, pressure fluid (Fluidigm) and 4 μl of assay or sample mixture were loaded into their respective locations on a microfluidic IFC (depending on the experiment, either gene expression or genotyping IFCs were used in either a 96.96 or 192.24 format) and were run according to the manufacturer’s instructions. The IFC was loaded onto the IFC Controller RX or Juno (Fluidigm) where the ‘Load Mix’ script was run. After proper IFC loading, images over either a 1- or 3-h period were collected using a custom protocol on Fluidigm’s EP1 or Biomark HD.</p><h3 class="c-article__sub-heading" id="Sec21">Fluidigm data analysis</h3><p>We plotted reference-normalized, background-subtracted fluorescence for guide-target pairs. For a guide-target pair (at a given time point, t, and target concentration), we first computed the reference-normalized value as (median(P<sub>t</sub> − P<sub>0</sub>) / (R<sub>t</sub> − R<sub>0</sub>)) where P<sub>t</sub> is the guide signal (FAM) at the time point, P<sub>0</sub> is its background measurement before the reaction, R<sub>t</sub> is the reference signal (ROX) at the time point, R<sub>0</sub> is its background measurement and the median is taken across replicates. We performed the same calculation for the NTC of the guide, providing a background fluorescence value for the guide at t. (When there were multiple technical replicates of such controls, we took the mean value across them.) The reference-normalized, background-subtracted fluorescence for a guide-target pair is the difference between these two values.</p><h3 class="c-article__sub-heading" id="Sec22">21 respiratory viruses</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec23">Design</h4><p>The oligonucleotide primers and crRNA guides were designed to detect the conserved regions of the following respiratory viruses: SARS-CoV-2, HCoV-229E, HCoV-HKU1, HCoV-NL63, HCoV-OC43, FLUAV, FLUBV, HMPV, HRSV, HPIV-1, 2, 3, 4, adenovirus, HEV-A, B, C, D, SARS-CoV, Middle East respiratory syndrome (MERS)-CoV and HRV. More specifically, complete genomes for all viruses on the panel were downloaded from the National Center for Biotechnology Information (NCBI) and aligned using MAFFT<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 54" title="Katoh, K. &amp; Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. Mol. Biol. Evol. 30, 772–780 (2013)." href="/articles/s41591-022-01734-1#ref-CR54" id="ref-link-section-d38468781e2665">54</a></sup>. For viral species with fewer than 1,000 sequences, the MAFFT ‘FFT-NS-ix1000’ algorithm was used. For viral species with &gt;1,000 sequences, the MAFFT ‘FFT-NS-1’ algorithm was used. These aligned sequences were then fed into ADAPT for crRNA design with high coverage using the ‘minimize guides’ objective (&gt;90% of sequences detected). Once highly conserved regions of the viral genome were selected with ADAPT for optimal guide design, primers were manually designed to amplify a 100–250 base pair (bp) target region with the crRNA predicted to bind in the middle of the fragment. ADAPT’s constraints on primer specificity were relaxed; in some cases, multiple primers were needed to encompass the full genomic diversity of a particular virus species. For optimal amplification, the primers were split into two pools. These primer pools and crRNA sequences are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">4</a>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec24">Target control: PIC1 and PIC2</h4><p>The consensus sequences generated directly above after multiple genome alignment with MAFFT were used to order a 500-bp dsDNA fragment encompassing the primer and crRNA binding sites. RNA was generated according to the method described in ‘General mCARMEN procedures-Preparation of IVT material’ and diluted to 10<sup>6</sup> copies μl<sup>−1</sup> in pools based on the primer pools mentioned above (PIC1, PIC2). The PICs were used as input into the CARMEN v.1 or mCARMEN detection reaction to function as a detection-positive control.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec25">Manual or automated sample extraction</h4><p>Automated and manual extraction was performed according to methods described under ‘General mCARMEN procedures-Extraction’.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec26">Two-step amplification</h4><p>We followed the CARMEN v.1 platform for two-step reverse transcription amplification, which was performed first by complementary DNA synthesis and then by PCR.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec27">cDNA synthesis using SuperScript IV</h4><p>A total of 10 μl of extracted RNA was converted into single-stranded cDNA in a 40-μl reaction. First, Random Hexamer Primers (Thermo Fisher Scientific) were annealed to sample RNA at 70 °C for 7 min, followed by reverse transcription using SuperScript IV for 20 min at 55 °C. cDNA was stored at −20 °C until use. DNase treatment was not performed at any point during sample preparation.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec28">Q5 DNA amplification</h4><p>Nucleic acid amplification was performed via PCR using Q5 Hot Start High-Fidelity DNA Polymerase (New England Biolabs) using primer pools (with 150 nM of each primer) in 20-μl reactions. Amplified samples were added directly into the detection reaction or stored at −20 °C until use. The following thermal cycling conditions were used: (1) initial denaturation at 98 °C for 2 min; (2) 45 cycles at 98 °C for 15 s, 50 °C for 30 s and 72 °C for 30 s; and (3) final extension at 72 °C for 2 min. Each target was amplified with its corresponding primer pool as listed under ‘oligonucleotides used in this study’.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec29">CARMEN v.1 detection</h4><p>For color coding, unless specified otherwise, amplified samples were diluted 1:10 into nuclease-free water supplemented with 13.2 mM MgCl<sub>2</sub> before color coding to achieve a final concentration of 6 mM after droplet merging. Detection mixes were not diluted. Color-coded stocks (2 µl) were arrayed in 96-well plates. (For detailed information on the construction of color codes, see ‘Color code design, construction and characterization’.) Each amplified sample or detection mix (18 µl) was added to a distinct color code and mixed by pipetting.</p><p>For emulsification, the color-coded reagents (20 µl) and 2% 008-fluorosurfactant (RAN Biotechnologies) in fluorous oil (3M 7500, 70 µl) were added to a droplet generator cartridge (Bio-Rad Laboratories) and reagents were emulsified into droplets using a QX200 droplet generator (Bio-Rad Laboratories) or a custom aluminum pressure manifold.</p><p>For droplet pooling, a total droplet pool volume of 150 µl of droplets was used to load each standard chip; a total of 800 µl of droplets was used to load each mChip. To maximize the probability of forming productive droplet pairings (amplified sample droplet + detection reagent droplet), half the total droplet pool volume was devoted to target droplets and half to detection reagent droplets. For pooling, individual droplet mixes were arrayed in 96-well plates. A multichannel pipette was used to transfer the requisite volumes of each droplet type into a single row of eight droplet pools, which were further combined to make a single droplet pool. The final droplet pool was pipetted up and down gently to fully randomize the arrangement of the droplets in the pool. The pooling step was rapid (&lt;10 min) and the small-molecule exchange between droplets during this period did not substantially alter the color codes.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec30">mCARMEN detection</h4><p>We followed the methods under ‘General CARMEN Procedures-Detection-Fluidigm detection’ with the following modifications: 42.5 nM LwaCas13 and 212.5 nM crRNA in each assay mix reaction and 500 nM RNaseAlert v2 in each sample mix reaction.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec31">CARMEN v.1 analysis</h4><p>We followed the data analysis pipeline from CARMEN v.1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Ackerman, C. M. et al. Massively multiplexed nucleic acid detection with Cas13. Nature 582, 277–282 (2020)." href="/articles/s41591-022-01734-1#ref-CR40" id="ref-link-section-d38468781e2749">40</a></sup>) to demultiplex and read out the fluorescence intensity of the reporter channel for each droplet reaction performed (MATLAB 2013). Briefly, premerge imaging data were processed using custom Python 3 scripts to detect fluorescently encoded droplets in microwells and identify their inputs based on their fluorescence intensity in 3 encoding channels, 647 nm, 594 nm and 555 nm. Subsequently, post-merge imaging data were analyzed to extract the reporter signal of the assay in the 488 nm channel and those reporter fluorescence intensities were physically mapped to the contents of each microwell. Quality control filtering was performed based on the appropriate size of a merged droplet from two input droplets and the closeness of a droplet’s color code to its assigned color code cluster centroid. The median and standard error were extracted from the replicates of all assay combinations generated on the array.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec32">mCARMEN analysis</h4><p>We followed the methods under ‘General CARMEN Procedures-Fluidigm Data Analysis’ and further visualized the data using Python 3, R v.4 and Prism 9 (GraphPad Software).</p><h3 class="c-article__sub-heading" id="Sec33">Single-step amplification troubleshooting</h3><p>The following RT–PCR kits were tested to determine the best performing assay: (1) OneStep RT–PCR Kit (QIAGEN); (2) TaqPath 1-Step Multiplex Master Mix (Thermo Fisher Scientific); (3) One-Step PrimeScript RT–PCR Kit (Takara Bio); (4) GoTaq Probe qPCR Kit (Promega Corporation); (5) UltraPlex 1-Step ToughMix (4X) (Quanta BioSciences); and (6) iTaq Universal One-Step Kit for RT–PCR (Bio-Rad Laboratories). Of the kits tested, the OneStep RT–PCR Kit (QIAGEN) was chosen for the final mCARMEN protocol.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec34">Qiagen OneStep RT–PCR Kit</h4><p>All or a combination of the following thermal cycling conditions were tested to shorten assay run time: reverse transcription at 50 °C for 15–30 min, PCR activation at 95 °C for 5–15 min, denaturation step at 94 °C for 10–30 s and extension step at 72 °C for 10 s to 1 min. The final extension at 72 °C for 10 min was omitted in all runs. The following primer pool conditions were also tested to optimize the assay: 150, 300, 500 and 600 nM of virus-specific primer and 100 and 150 nM of RNase P primers, with 5 μM of each virus-specific primer and 1.7 μM of RNase P primers. The reaction volumes tested include: 20 μl with 10% RNA template input, 30 μl with 20% RNA template input and 50 μl with 20% RNA template input. The final amplification conditions used for the RVP panel are described under ‘General mCARMEN Procedures-Amplification’.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec35">TaqPath 1-Step Multiplex Master Mix Kit</h4><p>The TaqPath 1-Step Multiplex Master Mix Kit was used to amplify nucleic acid according to the manufacturer’s instructions, using custom primer pools in 20-μl reactions. Primer pools of 150, 300 and 500 nM and annealing temperatures of 58 and 60 °C were all tested and compared to determine optimal conditions. The following thermal cycling conditions were used: (1) uracil-DNA glycosylase passive reference incubation at 25 °C for 2 min; (2) reverse transcription incubation at 50 °C for 15 min; (3) enzyme activation at 95 °C for 2 min; and (4) 40 cycles at 95 °C for 3 s and 60 °C for 30 s. Amplified samples were directly added into the detection reaction or stored at −20 °C until use.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec36">GoTaq Probe qPCR Kit</h4><p>The GoTaq Probe qPCR Kit was used to amplify nucleic acid via RT–PCR according to the manufacturer’s instructions, using custom primer pools in 20-μl reactions. Primer pools of 200, 300 and 500 nM were tested and compared to determine optimal conditions. Each target in the panel was amplified with its corresponding primer pool. The following thermal cycling conditions were used: (1) reverse transcription at 45 °C for 15 min and 95 °C for 2 min; and (2) 40 cycles at 95 °C for 15 s, 60 °C for 1 min. Amplified samples were directly added into the detection reaction or stored at −20 °C until use.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec37">UltraPlex 1-Step ToughMix (4X)</h4><p>The UltraPlex 1-Step ToughMix (4X) was used to amplify nucleic acid via RT–PCR according to the manufacturer’s instructions, using custom primer pools in 20-μl reactions. Primer pools of 200, 300 and 500 nM were tested and compared to determine optimal conditions. Each target in the panel was amplified with its corresponding primer pool. The following thermal cycling conditions were used: (1) reverse transcription at 50 °C for 10 min and 95 °C for 3 min; and (2) 45 cycles at 95 °C for 10 s, 60 °C for 1 min. Amplified samples were directly added into the detection reaction or stored at −20 °C until use.</p><h3 class="c-article__sub-heading" id="Sec38">RVP testing at the Broad Institute research laboratory</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec39">Design of nine virus respiratory panel and RNase P</h4><p>We designed this panel according to the methods described above under ‘Respiratory Panel-Design’ for these nine viruses: SARS-CoV-2, HCoV-HKU1, HCoV-OC43, HCoV-NL63, FLUAV/FLUAV-g4, FLUBV, HPIV-3, HRSV and HMPV, with the addition of an RNase P primer pair and crRNA. RNase P primers and crRNAs were designed within the same region of the gene as the CDC RT–qPCR assay (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">4</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec40">Patient specimen validation</h4><p>All patient specimens evaluated on the RVP were additionally evaluated concurrently with the CDC 2019-nCoV Real-Time RT–PCR Diagnostic Panel for N1 and RNase P. A specimen subset was selected for further study using NGS.</p><p>The CDC 2019-nCoV EUA recommends a Ct cutoff of &lt;40 for RNase P and/or SARS-CoV-2. Eight specimens failed quality control metrics and were therefore removed from further analysis. Five specimens were previously positive and three were negative by prior RT–PCR testing done by the Broad Genomics Platform.</p><p>Of the 525 patient specimens evaluated by mCARMEN, only 2 specimens had no detectable levels of RNase P above threshold, 1 of which was positive for SARS-CoV-2 while the other was virus-negative. The RNase P-negative, but virus-positive, specimen likely had a high concentration of viral RNA, which sequestered amplification materials during the reaction, limiting RNase P amplification. The double-negative specimen suggested possible extraction failure or sample integrity issues and was thus excluded from further analysis.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec41">RVP detection</h4><p>Specimen preparation was performed according to the method outlined in ‘General mCARMEN Procedures-Sample extraction’ with 200 µl of input material. Amplification was performed according to the methods outlined in ‘General mCARMEN Procedures-Amplification’. Detection reactions were prepared as described in ‘General mCARMEN Procedures-Detection’ with the following modifications: 42.5 nM LwaCas13 and 212.5 nM crRNA in each assay mix reaction and 500 nM quenched synthetic fluorescent RNA reporter (FAM/rUrUrUrUrUrUrU/3IABkFQ/) in each sample mix reaction. Results were analyzed according to the methods outlined under ‘General mCARMEN procedures-Data analysis’.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec42">CDC 2019-nCoV Real-Time RT–PCR Diagnostic Panel (research use only)</h4><p>The CDC 2019-nCoV Real-Time RT–PCR Diagnostic Panel was performed using the TaqPath 1-Step RT–qPCR Master Mix (Thermo Fisher Scientific) with a 1-μl template RNA of either SARS-CoV-2 or RNase P in 10-μl reactions, run in triplicate. Primers from the 2019-nCoV RUO Kit (Integrated DNA Technologies) were used. For SARS-CoV-2, a primer pool at 800 μM and probe at 200 μM were used. For RNase P, a primer pool at 500 μM and a probe at 125 μM were used. The following thermal cycling conditions were used: (1) enzyme activation at 25 °C for 2 min; (2) reverse transcription at 50 °C for 15 min; (3) PCR activation at 95 °C for 2 min; and (4) 45 cycles at 95 °C for 3 s and 55 °C for 30 s. Standard curves were made with spike-in of the RNA template (SARS-CoV-2 and RNase P) to make a tenfold serial dilution from 10<sup>0</sup> to 10<sup>6</sup> copies μl<sup>−1</sup>. This was run on the QuantStudio 6 Flex Real-Time PCR System (Thermo Fisher Scientific).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec43">NGS</h4><p>Metagenomic sequencing libraries were generated as described previously<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Lemieux, J. E. et al. Phylogenetic analysis of SARS-CoV-2 in Boston highlights the impact of superspreading events. Science 371, eabe326 (2021)." href="/articles/s41591-022-01734-1#ref-CR5" id="ref-link-section-d38468781e2864">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 55" title="Matranga, C. B. et al. Enhanced methods for unbiased deep sequencing of Lassa and Ebola RNA viruses from clinical and biological samples. Genome Biol. 15, 519 (2014)." href="/articles/s41591-022-01734-1#ref-CR55" id="ref-link-section-d38468781e2867">55</a></sup>. Briefly, extracted RNA was DNase-treated to remove residual DNA then human ribosomal RNA was depleted. cDNA was synthesized using random hexamer primers. Sequencing libraries were prepared with the Illumina Nextera XT DNA Library Preparation Kit and sequenced with 100- or 150-nucleotide paired-end reads. Data analysis was conducted on the Terra platform (<a href="https://app.terra.bio/">app.terra.bio</a>); all workflows are publicly available on the Dockstore tool registry service. Samples were demultiplexed using demux_plus to filter out known sequencing contaminants. Viral genomes were assembled using assemble_refbased, discordant specimens with viral genomes were assembled using assemble_denovo and additionally visualized using classify_kraken, blastn, blastx, geneious and R. A virus was determined to be present if more than 10 reads mapped to a particular viral genome. Full genomes were deposited with GenBank (BioProject accession no. <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA802370">PRJNA802370</a>).</p><h3 class="c-article__sub-heading" id="Sec44">Clinical evaluation of RVP in CLIA-certified laboratory at MGH</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec45">Design</h4><p>We designed this panel according to the methods described above under RVP Testing at the Broad Institute Research Laboratory.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec46">Extraction control</h4><p>Extraction negative control is an RNA extraction control and is prepared by adding 200 μl pooled human sample (negative for all viruses on the panel) to a well with 280 μl binding bead mix. The extraction negative control should yield a positive result for the RNase P crRNA and primer pair and a negative result for all other targets.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec47">Amplification control</h4><p>NTC is a negative control for nucleic acid amplification and is prepared by adding 10 μl nuclease-free water (instead of RNA) into 40 μl of OneStep RT–PCR Kit mastermix. This should yield a negative result for all targets on the panel. Combined positive control is a positive control for nucleic acid amplification and is prepared by pooling IVT synthetic RNA of all the targets on the panel to 10<sup>3</sup> copy µl<sup>−1</sup>; 11-µl aliquots of this mix were stored at −80 °C until use, when 10 μl were added to 40 μl of OneStep RT–PCR Kit mastermix. This should yield a positive result for all targets on the panel.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec48">Detection controls</h4><p>Negative detection control is a negative control for the Fluidigm detection step and is prepared by adding nuclease-free water (instead of amplified RNA) to the sample mix without MgCl<sub>2</sub>. This should yield a negative result for all targets on the panel. No crRNA control is a negative control for the Fluidigm detection step and is prepared by adding nuclease-free water (instead of 1 µM crRNA) to the assay mix. This should yield a negative result for all targets on the panel.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec49">Batch preparation of sample and assay mixtures</h4><p>Sample and assay mixtures can be prepared in advance for multiple 96-sample batches according to similar methods, with the following changes: the batch sample mix contained all reagents described above excluding 9 mM MgCl<sub>2</sub>, and the batch assay mix contained all reagents described above excluding the 2x Assay Loading Reagent. Both mixtures were calculated with 10% coverage. Both mixtures were stored at −80 °C until use; 9 mM MgCl<sub>2</sub> was added to the sample mix and 2x Assay Loading Reagent was added to each assay mix before use.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec50">SYBR RT–qPCR of viral seed stock and genomic RNA</h4><p>Quantification of all viral seed stock and genomic RNA received from ATCC and BEI Resources was performed using the Power SYBR Green RNA-to-C<sub>T</sub> <i>1-Step</i> Kit (Thermo Fisher Scientific). Reactions were run in triplicate with 1 μl RNA input in 10-μl reactions. A primer mix at 500 nM was used and all primer sequences used are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">4</a>. The following thermal cycling conditions were used: (1) reverse transcription at 48 °C for 30 min; (2) enzyme activation at 95 °C for 10 min; (3) 40 cycles at 95 °C for 15 s and 60 °C for 1 min; and (4) melt curve of 95 °C for 15 s, 60 °C for 15 s and 95 °C for 15 s. Standard curves were made with spike-in of RNA template to make a tenfold serial dilution from 10<sup>0</sup> to 10<sup>6</sup> copies μl<sup>−1</sup>. This was run on the QuantStudio 6 Flex Real-Time PCR System.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec51">LOD</h4><p>Samples were prepared for the LOD experiments using either quantified viral isolates, genomic RNA or IVT partial gene fragments. For the SARS-CoV-2, HCoV-OC43, HRSV and HPIV-3 assays, quantified viral isolates of known titer (RNA copies µl<sup>−1</sup>) spiked into a pooled negative human sample (negative for all viruses on the panel) in universal transport medium (UTM), to mimic a clinical specimen. The pooled human negative samples were incubated in the binding bead mix solution according to the methods described in ‘General mCARMEN Procedures-Automated extraction’. Since no quantified virus isolates for HCoV-NL63, HCoV-HKU-1, FLUAV, FLUAV-g4, FLUBV and HMPV were available for use at the time the study was conducted, assays designed for RNA detection of these viruses were tested with either genomic RNA from ATCC (FLUAV: catalog no. NR-43756; FLUBV: catalog no. VR-1804) or IVT RNA of known titer and spiked into pooled negative human samples in UTM.</p><p>RNA was extracted from 200 µl of input material using the MagMAX DNA Multi-Sample Ultra 2.0 Kit on a KingFisher Flex Magnetic Particle Processor with 96 Deep Well Head. This was run according to the protocol listed in the TaqPath COVID-19 Combo Kit Protocol under ‘KingFisher, Extract RNA-Automated method (200-μl input volume)’ with the following differences. To prepare the binding bead mix, the following was added: 265 μl binding solution, 10 μl total nucleic acid magnetic beads, 5 μl Proteinase K with 10% coverage for multiple samples. Then, 280 μl of the binding bead mix was added to each sample well. The 200 μl of input material included negative human sample and RNA: 160 μl of pooled human samples (negative for all viruses on the panel) was added to each sample well and incubated for 20 min before 40 μl of RNA was spiked in. Samples were eluted in 50 μl of elution solution and either directly added to the amplification reaction or stored at −80 °C until use.</p><p>A preliminary LOD for each assay was determined by testing triplicates of RNA purified using the extraction method described in the ‘RVP panel’. The approximate LOD was identified by extraction, amplification and detection of tenfold serial dilutions of IVT RNA of known titer (copies µl<sup>−1</sup>) for five replicates. These concentrations ranged from 10<sup>4</sup>–10<sup>−3</sup> copies μl<sup>−1</sup>. The lower bound of the LOD range was determined as the lowest concentration where five out of five replicates were positive; the upper bound was determined as the concentration tenfold above the lower bound.</p><p>A confirmation of the LOD for each assay was determined by testing 20 replicates of RNA purified using the extraction method described in the ‘RVP panel’. The approximate LOD was identified by extraction, amplification and detection of twofold serial dilutions of the input sample, quantified viral isolates and genomic RNA or IVT RNA. These concentrations ranged from 20 to 0.5 copies μl<sup>−1</sup>, depending on the virus. The LOD was determined as the lowest concentration where ≥95% (19 out of 20) of the replicates were positive.</p><h3 class="c-article__sub-heading" id="Sec52">Specificity</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec53">In silico analysis: inclusivity</h4><p>Inclusivity was tested by performing an in silico analysis using all publicly available sequences of all targets on the RVP panel. Complete genomes for all viruses were downloaded from NCBI on 2 April 2021 and aligned using MAFFT v.7. For viral species with fewer than 1,000 sequences, the FFT-NS-ix1000 algorithm was used to create the MAFFT alignment. For viral species with &gt;1,000 sequences, the FFT-NS-1 algorithm was used to create the MAFFT alignment. The primer and crRNA sequences were then mapped to the aligned viral sequences using a consensus alignment to determine the percentage identity (homology) and the number of mismatches. The average homology and mismatches were taken across the total number of sequences evaluated. Please note that mismatches for crRNA sequences do not take wobble base pairing (G-U pairing) into account. Results are summarized in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">8</a>.</p><p>Additionally, the SARS-CoV-2 crRNA and primer sequences were tested by NCBI BLAST+ against the nonredundant/nucleotide databases (updated 31 March 2021, <i>n</i> = 68,965,867 sequences analyzed) and the Betacoronavirus database (updated 1 April 2021, <i>n</i> = 140,760). The search parameters were adjusted to blastn-short for short input sequences. The match and mismatch scores were 1 and −3, respectively. The penalty to create and extend a gap in an alignment was 5 and 2, respectively. The BLAST results confirmed only perfect matches to SARS-CoV-2.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec54">In silico analysis: specificity</h4><p>Complete genomes for all viruses were downloaded from the NCBI on 2 April 2021 and aligned using MAFFT. For viral species with fewer than 1,000 sequences, FFT-NS-ix1000 was used. For viral species with &gt;1,000 sequences, FFT-NS-1 was used for the MAFFT alignment. The primer and crRNA sequences were then mapped to the aligned viral sequences using a consensus alignment to determine percentage identity (homology). The average homology was taken across the panel sequences and the total number of sequences were evaluated. The text in bold represents on-target primers/crRNA to the intended viral sequences. Not all sequence combinations were evaluated because whole-genome homology between many viruses is less than 80%. All primer and crRNA sequences do not have &gt;80% homology to other unintended viral or bacterial sequences, making the panel highly specific to particular viruses of interest. More specifically, no in silico cross-reactivity &gt;80% homology between any primers and crRNA sequences on the RVP was observed for the following common respiratory flora and other viral pathogens: SARS-CoV-1, HCoV-MERS, adenovirus, enterovirus, rhinovirus, <i>Chlamydia pneumoniae</i>, <i>Haemophilus influenzae</i>, <i>Legionella pneumophila</i>, <i>Mycobacterium tuberculosis</i>, <i>Streptococcus pneumoniae</i>, <i>Streptococcus pyogenes</i>, <i>Bordetella pertussis</i>, <i>Mycoplasma pneumoniae</i>, <i>Pneumocystis jirovecii</i>, <i>Candida albicans</i>, <i>Pseudomonas aeruginosa</i>, <i>Staphylococcus epidermis</i> and <i>Streptococcus salivarius</i>. The results of the in silico analysis are summarized in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">9</a>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec55">In vitro analysis</h4><p>Targets were selected for in vitro specificity testing based on closely related viral species with high nucleotide identity. The synthetic DNA targets contained the consensus sequence of a particular virus that was position-matched to the location of the RVP virus of interest targets in the viral genome. Samples were prepared for the specificity experiments according to the methods described above in ‘General mCARMEN procedures-Preparation of IVT material’; samples were serially diluted down to a concentration of 10<sup>6</sup> and 10<sup>5</sup> copies µl<sup>−1</sup>. For all samples prepared for the specificity experiments, RNA was extracted from 200 µl of input material using the MagMAX DNA Multi-Sample Ultra 2.0 Kit on a KingFisher Flex Magnetic Particle Processor. This was run according to the extraction, amplification and detection methods described above under ‘RVP testing at the Broad Institute research laboratory’.</p><h3 class="c-article__sub-heading" id="Sec56">Patient specimen validation</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec57">Specimen preparation before extraction</h4><p>All patient specimens from the MGH Clinical Microbiology Laboratory were initially reported to be positive for HCoV-HKU1, HCoV-NL63 and HMPV via BioFire FilmArray Respiratory Panel (RP2) (Biofire Diagnostics) or positive for SARS-CoV-2, FLUAV (H3), FLUBV and HRSV via Xpert Xpress SARS-CoV-2/Flu/RSV (Cepheid). SARS-CoV-2 was aliquoted as follows: 220 μl for testing using the RVP panel and 220 μl for testing using the TaqPath COVID-19 Combo Kit; remaining specimens were stored at −80 °C. All negative specimens were aliquoted as follows: 220 μl for RVP panel testing, 220 μl for TaqPath COVID-19 Combo Kit testing and 400 μl for BioFire FilmArray Respiratory Panel (RP2) testing; remaining specimen was stored at −80 °C. All other specimens were aliquoted as follows: 220 μl for RVP panel testing and 400 μl for BioFire FilmArray Respiratory Panel (RP2) testing; remaining specimen was stored at −80 °C.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec58">Preparation of contrived samples before extraction</h4><p>Contrived patient samples of the HCoV-HKU1, HCoV-OC43, HCoV-NL63, FLUAV-g4, HPIV-3 and HMPV viruses were prepared by diluting either viral seed stock (HCoV-OC43 and HPIV-3) or template RNA (HCoV-HKU1 and HCoV-NL63). See Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">10</a> for viral seed stock vendor details. Viral seed stock or template RNA was added to non-pooled human specimens (negative for all targets, except RNase P, on the RVP panel) at a concentration 2 times the LOD; the concentrations for these samples ranged from 10<sup>2</sup> to 10<sup>6</sup> copies µl<sup>−1</sup>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec59">RVP</h4><p>All materials were extracted, amplified, detected and analyzed using the methods described under ‘General mCARMEN procedures’ and ‘RVP testing at the Broad Institute research laboratory-Patient specimen validation’.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec60">TaqPath COVID-19 Combo Kit</h4><p>A subset of patient specimens, all SARS-CoV-2 and negative patient specimens, from the MGH Clinical Microbiology Laboratory were verified using the TaqPath COVID‑19 Combo Kit. These samples were initially reported to be positive for SARS-CoV-2 via Xpert Xpress SARS-CoV-2/Flu/RSV or reported to be negative for all targets (excluding RNase P) on the RVP panel via BioFire FilmArray Respiratory Panel (RP2). The TaqPath COVID‑19 Combo Kit was performed according to the manufacturer’s instructions. The assay was performed using the Applied Biosystems 7500 Real-Time PCR System.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec61">BioFire FilmArray Respiratory Panel (RP2)</h4><p>A subset of patient specimens from the MGH Clinical Microbiology Laboratory, all HCoV-HKU1, HCoV-NL63, FLUAV (H3), FLUBV, HRSV, HMPV and negative patient specimens, were verified using the BioFire FilmArray Respiratory Panel (RP2). These specimens were either initially reported to be positive for HCoV-HKU1, HCoV-NL63 and HMPV via BioFire FilmArray Respiratory Panel (RP2) or positive for FLUAV (H3), FLUBV and HRSV via Xpert Xpress Flu/RSV (Cepheid). For each run, one patient specimens in UTM at 300 µl was verified using the BioFire FilmArray Respiratory Panel (RP2) according to the manufacturer’s instructions. Any remaining specimen was stored at −80 °C.</p><p>Controls for this assay were received with the kit and ready for use. Control 1 was expected to be positive for adenovirus, HMPV, human rhino/enterovirus, FLUAV (H1-2009), FLUAV (H3), HPIV-1 and HPIV-2. Control 2 was expected to be positive for HCoV-229E, HCoV-HKU1, HCoV-NL63, HCoV-OC43, FLUAV (H1), FLUBV, HPIV-2, HPIV-3 and HRSV.</p><p>The results were automatically displayed on the FilmArray System with each target in a run reported as ‘detected’ or ‘not detected’. If either control failed, the software marked this run as ‘invalid’. When sufficient human sample volume was available, samples with invalid results were rerun.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec62">Analysis software</h4><p>The analysis software comprises Python scripts executing the data analysis described under ‘General CARMEN Procedures-Fluidigm Data Analysis.’ They were packaged into an executable with graphical user interface using the Python module Gooey v.1.0.7.</p><p>Briefly, the reference-normalized, background-subtracted fluorescence was calculated for the guide-target pairs for measurement after 60 min. Then, the dynamic range and the separation band were assessed: separation band = (mean of positive controls − 3 s.d. of positive controls) − (mean of negative controls − 3 s.d. of negative controls); dynamic range = mean of positive controls − mean of negative controls.</p><p>If the ratio of separation band to dynamic range was ≤0.2, the whole assay was invalid. Next, for the positive and negative controls, outliers based on three s.d. were identified. If a positive control had a too low value, or a negative control had a too high value, the respective assay was invalid. For the remaining samples, hit calling was performed based on comparing the signal to the water control. If the signal was 1.8× higher than the water control, the guide-target pair was called a hit. Based on this hit calling, the extraction control, negative and positive detection controls and internal controls were verified. If their result did not correspond to their expected hit status, either the respective assay or specimen was deemed invalid. To be valid, all specimens needed to be either positive for RNase P or at least one other assay. Finally, the software annotated the results as CSV files and visualized them as an annotated heatmap.</p><h3 class="c-article__sub-heading" id="Sec63">Cas13- and Cas12-based detection with mCARMEN</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec64">Design for Cas12-based detection</h4><p>Cas13 crRNAs from the RVP were utilized. Cas12 crRNAs were manually designed in the same region of the viral genome as the Cas13 crRNAs to reduce the need for additional primer design while maintaining the protospacer adjacent motif requirements of Cas12. Only one additional primer was designed to properly amplify all targets on the RVP. All crRNAs and primers are listed in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">4</a>.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec65">Detection</h4><p>We followed the methods outlined under ‘General mCARMEN procedures-Detection’ with the following modifications: 10–60 nM LwaCas13, 10–60 nM LbaCas12a, 125 nM Cas13a crRNA and 125 nM Cas12 crRNA in each assay mix reaction and 500 nM quenched synthetic fluorescent RNA reporter (FAM/rUrUrUrUrUrUrU/3IABkFQ/ and VIC/rTrTrArTrTrArTrT/3IABkFQ) in each sample mix reaction.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec66">Data analysis</h4><p>We generally followed the methods outlined under ‘General mCARMEN procedures-Analysis’, this time taking into account the VIC signal separate from the FAM signal. We used a custom Python script to determine whether the FAM signal of a reaction was significantly above background by comparing it to the NTC. If the background-subtracted and normalized fluorescence intensity was 1.8 higher than the normalized and background-subtracted NTC, the assay was considered positive.</p><h3 class="c-article__sub-heading" id="Sec67">Variant testing</h3><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec68">Design</h4><p>The crRNAs for SNP discrimination were designed using a generative sequence design algorithm (Mantena, S. et al., manuscript in preparation). This approach uses ADAPT’s predictive model to predict the activity of candidate crRNA sequences against on-target and off-target sequences<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Metsky, H. C. et al. Designing sensitive viral diagnostics with machine learning. Nat. Biotech. 
                  https://doi.org/10.1038/s41587-022-01213-5
                  
                 (2022)." href="/articles/s41591-022-01734-1#ref-CR35" id="ref-link-section-d38468781e3204">35</a></sup>. These predictions of candidate crRNA activity steer the generative algorithm’s optimization process, where it seeks to design crRNA probes that have maximal predicted on-target activity and minimal predicted off-target activity. Using this design algorithm, we selected 26 mutations to detect and discriminate between the variants (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">4</a>).</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec69">Amplification: SuperScript IV One-Step RT–PCR System</h4><p>The SuperScript IV One-Step RT–PCR System was used to amplify nucleic acids according to the manufacturer’s instructions, using custom primer pools in 25-μl reactions. Primer pools were made at 10 μM. The following thermal cycling conditions were used: (1) reverse transcription incubation at 50 °C for 15 min; (2) enzyme activation at 98 °C for 2 min; (3) 35–40 cycles at 98 °C for 10 s, 60 °C for 10 s and 72 °C for 1 min 30 s; and (4) final extension at 72 °C for 5 min. Amplified samples were directly added into the detection reaction or stored at −20 °C until use.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec70">Detection</h4><p>We followed the methods outlined under ‘General mCARMEN procedures-Detection’ with the following modifications: 42.5 nM LwaCas13 and 2–212.5 nM crRNA in each assay mix reaction.</p><h4 class="c-article__sub-heading c-article__sub-heading--small" id="Sec71">Data analysis: threshold calculation</h4><p>To determine if an ancestral or derived sequence was present, the signals between respective ancestral and mutation crRNA pairs had to be evaluated and compared (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM3">4</a>). First, background-subtracted reporter fluorescence was normalized to the background-subtracted passive reference dye (ROX) fluorescence for each assay in the IFC. Next, the ancestral:mutation and mutation:ancestral ratios were calculated for each 5-min interval time point across 180 min. For each crRNA pair, the ratio reaching a crRNA pair-specific threshold at the earliest time point was selected. If the ancestral:mutation ratio was selected, then the sequence present was determined to be ancestral. If the mutation:ancestral ratio was selected, then the sequence present was determined to contain the mutation targeted by the mutation crRNA within the SARS-CoV-2 spike gene. crRNA pair-specific thresholds were determined based on ancestral and variant control samples, also referred to as seed stock samples, tested in parallel with the unknown samples. For a given crRNA pair, the threshold was set to the lowest value with the maximum combined sensitivity and specificity when applied to the seed stock samples. For crRNA to detect an SNP at the same position, the second lowest threshold with the maximum combined sensitivity and specificity was chosen if possible, without compromising the maximum combined sensitivity and specificity. For crRNA pairs targeting mutations not represented in the variant control samples, a default crRNA pair threshold of 1.5 was set.</p><p>The variant identified hit-calling parameters were as follows: (1) if no mutations were detected, a result of ‘ancestral’ was returned; (2) at least one unique crRNA specific to a single SARS-CoV-2 variant must be above the fluorescence ratio threshold. If there was no unique crRNA signal above threshold, a result of ‘variant not identified’ was returned; (3) if two or more mutations for a given variant fell below the threshold, a result of ‘variant not identified’ was returned. All other mutations must surpass the threshold; (4) if three or more unexpected mutations for a given variant were above threshold, a result of ‘variant not identified’ was returned. At most, two unexpected signals could occur as long as parameters 1 and 2 were met.If all three parameters were met, a result of ‘variant identified’ was returned. If the parameters were not met, a result of ‘variant not identified’ was returned. Samples that contained additional mutation signals falling outside the typical variant lineage mutation list follow these parameters: (1) if 1–2 unexpected signals were observed to be slightly above the threshold yet all other signals were correct for a specific variant lineage, then the unexpected signal was disregarded and the variant call was made on the remaining signals. (2) if more than two unexpected signals were observed above the threshold and either all other signals were correct for a specific variant lineage or were not perfectly matching, a result of ‘variant uncertain’ was returned.</p><p>Please note that the variant identification pipeline will need to be updated as new SARS-CoV-2 mutations and variant lineages arise for proper identification.</p><p>A few exceptions are worth mentioning: we observed crRNAs for SNPs E484Q, P681R, N501T and L452Q, which had undesirable cross-reactive signals with a position-matched or adjacent mutation; thus, they were excluded from further evaluation.</p><h3 class="c-article__sub-heading" id="Sec72">Reporting Summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01734-1#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>All requests for raw and analyzed data and materials will be reviewed by the Broad Institute of Harvard and MIT to verify if the request is subject to any intellectual property or confidentiality obligations. Data and materials that can be shared will be released via a material transfer agreement. RNA sequencing data have been deposited with the Sequence Read Archive under the BioProject accession no. <a href="https://www.ncbi.nlm.nih.gov/bioproject/?term=PRJNA802370">PRJNA802370</a> and will be made available on request for academic use and within the limitations of the provided informed consent by the corresponding author upon acceptance.</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>The code used for data analysis in this study is available on Github: <a href="https://github.com/broadinstitute/mcarmen">https://github.com/broadinstitute/mcarmen</a>.</p>
            </div></div></section><section data-title="Change history"><div class="c-article-section" id="change-history-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="change-history">Change history</h2><div class="c-article-section__content" id="change-history-content"><ul class="c-article-change-list"><li class="c-article-change-list__item u-mb-24" id="chg1"><ins datetime="2023-11-09"><h3 class="c-article-change-list__heading u-h3 u-pr-8 u-display-inline">09 November 2023</h3><div class="c-article-change-list__details"><p>A Correction to this paper has been published: <a href="https://doi.org/10.1038/s41591-023-02684-y">https://doi.org/10.1038/s41591-023-02684-y</a></p></div></ins></li></ul></div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">Mina, M. J. &amp; Andersen, K. G. COVID-19 testing: one size does not fit all. <i>Science</i> <b>371</b>, 126–127 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.abe9187" data-track-item_id="10.1126/science.abe9187" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.abe9187" aria-label="Article reference 1" data-doi="10.1126/science.abe9187">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVCgsLk%3D" aria-label="CAS reference 1">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33414210" aria-label="PubMed reference 1">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=COVID-19%20testing%3A%20one%20size%20does%20not%20fit%20all&amp;journal=Science&amp;doi=10.1126%2Fscience.abe9187&amp;volume=371&amp;pages=126-127&amp;publication_year=2021&amp;author=Mina%2CMJ&amp;author=Andersen%2CKG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Rasmussen, A. L. &amp; Popescu, S. V. SARS-CoV-2 transmission without symptoms. <i>Science</i> <b>371</b>, 1206–1207 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.abf9569" data-track-item_id="10.1126/science.abf9569" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.abf9569" aria-label="Article reference 2" data-doi="10.1126/science.abf9569">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXmvF2jsr0%3D" aria-label="CAS reference 2">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33737476" aria-label="PubMed reference 2">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20transmission%20without%20symptoms&amp;journal=Science&amp;doi=10.1126%2Fscience.abf9569&amp;volume=371&amp;pages=1206-1207&amp;publication_year=2021&amp;author=Rasmussen%2CAL&amp;author=Popescu%2CSV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Winichakoon, P. et al. Negative nasopharyngeal and oropharyngeal swabs do not rule out COVID-19. <i>J. Clin. Microbiol.</i> <b>58</b>, e00297-20 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/JCM.00297-20" data-track-item_id="10.1128/JCM.00297-20" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FJCM.00297-20" aria-label="Article reference 3" data-doi="10.1128/JCM.00297-20">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32102856" aria-label="PubMed reference 3">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7180262" aria-label="PubMed Central reference 3">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=Negative%20nasopharyngeal%20and%20oropharyngeal%20swabs%20do%20not%20rule%20out%20COVID-19&amp;journal=J.%20Clin.%20Microbiol.&amp;doi=10.1128%2FJCM.00297-20&amp;volume=58&amp;publication_year=2020&amp;author=Winichakoon%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Woloshin, S., Patel, N. &amp; Kesselheim, A. S. False negative tests for SARS-CoV-2 infection—challenges and implications. <i>N. Engl. J. Med.</i> <b>383</b>, e38 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMp2015897" data-track-item_id="10.1056/NEJMp2015897" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMp2015897" aria-label="Article reference 4" data-doi="10.1056/NEJMp2015897">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFKjt7zK" aria-label="CAS reference 4">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32502334" aria-label="PubMed reference 4">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=False%20negative%20tests%20for%20SARS-CoV-2%20infection%E2%80%94challenges%20and%20implications&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMp2015897&amp;volume=383&amp;publication_year=2020&amp;author=Woloshin%2CS&amp;author=Patel%2CN&amp;author=Kesselheim%2CAS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Lemieux, J. E. et al. Phylogenetic analysis of SARS-CoV-2 in Boston highlights the impact of superspreading events. <i>Science</i> <b>371</b>, eabe326 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.abe3261" data-track-item_id="10.1126/science.abe3261" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.abe3261" aria-label="Article reference 5" data-doi="10.1126/science.abe3261">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenetic%20analysis%20of%20SARS-CoV-2%20in%20Boston%20highlights%20the%20impact%20of%20superspreading%20events&amp;journal=Science&amp;doi=10.1126%2Fscience.abe3261&amp;volume=371&amp;publication_year=2021&amp;author=Lemieux%2CJE">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Harvey, W. T. et al. SARS-CoV-2 variants, spike mutations and immune escape. <i>Nat. Rev. Microbiol.</i> <b>19</b>, 409–424 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41579-021-00573-0" data-track-item_id="10.1038/s41579-021-00573-0" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41579-021-00573-0" aria-label="Article reference 6" data-doi="10.1038/s41579-021-00573-0">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1WqtbrP" aria-label="CAS reference 6">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34075212" aria-label="PubMed reference 6">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8167834" aria-label="PubMed Central reference 6">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20variants%2C%20spike%20mutations%20and%20immune%20escape&amp;journal=Nat.%20Rev.%20Microbiol.&amp;doi=10.1038%2Fs41579-021-00573-0&amp;volume=19&amp;pages=409-424&amp;publication_year=2021&amp;author=Harvey%2CWT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Dyson, L. et al. Possible future waves of SARS-CoV-2 infection generated by variants of concern with a range of characteristics. <i>Nat. Commun.</i> <b>12</b>, 5730 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-021-25915-7" data-track-item_id="10.1038/s41467-021-25915-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-021-25915-7" aria-label="Article reference 7" data-doi="10.1038/s41467-021-25915-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFGhsbnN" aria-label="CAS reference 7">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34593807" aria-label="PubMed reference 7">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8484271" aria-label="PubMed Central reference 7">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=Possible%20future%20waves%20of%20SARS-CoV-2%20infection%20generated%20by%20variants%20of%20concern%20with%20a%20range%20of%20characteristics&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-25915-7&amp;volume=12&amp;publication_year=2021&amp;author=Dyson%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Zou, L. et al. SARS-CoV-2 viral load in upper respiratory specimens of infected patients. <i>N. Engl. J. Med.</i> <b>382</b>, 1177–1179 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMc2001737" data-track-item_id="10.1056/NEJMc2001737" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMc2001737" aria-label="Article reference 8" data-doi="10.1056/NEJMc2001737">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32074444" aria-label="PubMed reference 8">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121626" aria-label="PubMed Central reference 8">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20viral%20load%20in%20upper%20respiratory%20specimens%20of%20infected%20patients&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMc2001737&amp;volume=382&amp;pages=1177-1179&amp;publication_year=2020&amp;author=Zou%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Wölfel, R. et al. Virological assessment of hospitalized patients with COVID-2019. <i>Nature</i> <b>581</b>, 465–469 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-020-2196-x" data-track-item_id="10.1038/s41586-020-2196-x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-2196-x" aria-label="Article reference 9" data-doi="10.1038/s41586-020-2196-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32235945" aria-label="PubMed reference 9">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Virological%20assessment%20of%20hospitalized%20patients%20with%20COVID-2019&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2196-x&amp;volume=581&amp;pages=465-469&amp;publication_year=2020&amp;author=W%C3%B6lfel%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Vogels, C. B. F. et al. Multiplex qPCR discriminates variants of concern to enhance global surveillance of SARS-CoV-2. <i>PLoS Biol.</i> <b>19</b>, e3001236 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1371/journal.pbio.3001236" data-track-item_id="10.1371/journal.pbio.3001236" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1371%2Fjournal.pbio.3001236" aria-label="Article reference 10" data-doi="10.1371/journal.pbio.3001236">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFGju7jM" aria-label="CAS reference 10">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33961632" aria-label="PubMed reference 10">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8133773" aria-label="PubMed Central reference 10">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplex%20qPCR%20discriminates%20variants%20of%20concern%20to%20enhance%20global%20surveillance%20of%20SARS-CoV-2&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.3001236&amp;volume=19&amp;publication_year=2021&amp;author=Vogels%2CCBF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Heggestad, J. T. et al. Multiplexed, quantitative serological profiling of COVID-19 from blood by a point-of-care test. <i>Sci. Adv.</i> <b>7</b>, eabg4901 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/sciadv.abg4901" data-track-item_id="10.1126/sciadv.abg4901" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fsciadv.abg4901" aria-label="Article reference 11" data-doi="10.1126/sciadv.abg4901">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFeltL7O" aria-label="CAS reference 11">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34172447" aria-label="PubMed reference 11">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8232907" aria-label="PubMed Central reference 11">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%2C%20quantitative%20serological%20profiling%20of%20COVID-19%20from%20blood%20by%20a%20point-of-care%20test&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.abg4901&amp;volume=7&amp;publication_year=2021&amp;author=Heggestad%2CJT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Pham, J. et al. Performance characteristics of a high-throughput automated transcription-mediated amplification test for SARS-CoV-2 detection. <i>J. Clin. Microbiol.</i> <b>58</b>, e01669-20 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1128/JCM.01669-20" data-track-item_id="10.1128/JCM.01669-20" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1128%2FJCM.01669-20" aria-label="Article reference 12" data-doi="10.1128/JCM.01669-20">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32727828" aria-label="PubMed reference 12">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7512162" aria-label="PubMed Central reference 12">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=Performance%20characteristics%20of%20a%20high-throughput%20automated%20transcription-mediated%20amplification%20test%20for%20SARS-CoV-2%20detection&amp;journal=J.%20Clin.%20Microbiol.&amp;doi=10.1128%2FJCM.01669-20&amp;volume=58&amp;publication_year=2020&amp;author=Pham%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Eckbo, E. J. et al. Evaluation of the BioFire® COVID-19 test and Respiratory Panel 2.1 for rapid identification of SARS-CoV-2 in nasopharyngeal swab samples. <i>Diagn. Microbiol. Infect. Dis.</i> <b>99</b>, 115260 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.diagmicrobio.2020.115260" data-track-item_id="10.1016/j.diagmicrobio.2020.115260" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.diagmicrobio.2020.115260" aria-label="Article reference 13" data-doi="10.1016/j.diagmicrobio.2020.115260">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXksFyltg%3D%3D" aria-label="CAS reference 13">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33340934" aria-label="PubMed reference 13">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=Evaluation%20of%20the%20BioFire%C2%AE%20COVID-19%20test%20and%20Respiratory%20Panel%202.1%20for%20rapid%20identification%20of%20SARS-CoV-2%20in%20nasopharyngeal%20swab%20samples&amp;journal=Diagn.%20Microbiol.%20Infect.%20Dis.&amp;doi=10.1016%2Fj.diagmicrobio.2020.115260&amp;volume=99&amp;publication_year=2021&amp;author=Eckbo%2CEJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Konings, F. et al. SARS-CoV-2 variants of interest and concern naming scheme conducive for global discourse. <i>Nat. Microbiol.</i> <b>6</b>, 821–823 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41564-021-00932-w" data-track-item_id="10.1038/s41564-021-00932-w" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41564-021-00932-w" aria-label="Article reference 14" data-doi="10.1038/s41564-021-00932-w">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1yhsLbL" aria-label="CAS reference 14">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34108654" aria-label="PubMed reference 14">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20variants%20of%20interest%20and%20concern%20naming%20scheme%20conducive%20for%20global%20discourse&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-021-00932-w&amp;volume=6&amp;pages=821-823&amp;publication_year=2021&amp;author=Konings%2CF">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Huang, H.-S. et al. Multiplex PCR system for the rapid diagnosis of respiratory virus infection: systematic review and meta-analysis. <i>Clin. Microbiol. Infect.</i> <b>24</b>, 1055–1063 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cmi.2017.11.018" data-track-item_id="10.1016/j.cmi.2017.11.018" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cmi.2017.11.018" aria-label="Article reference 15" data-doi="10.1016/j.cmi.2017.11.018">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVCkt7fL" aria-label="CAS reference 15">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29208560" aria-label="PubMed reference 15">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplex%20PCR%20system%20for%20the%20rapid%20diagnosis%20of%20respiratory%20virus%20infection%3A%20systematic%20review%20and%20meta-analysis&amp;journal=Clin.%20Microbiol.%20Infect.&amp;doi=10.1016%2Fj.cmi.2017.11.018&amp;volume=24&amp;pages=1055-1063&amp;publication_year=2018&amp;author=Huang%2CH-S">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Jacky, L. et al. Robust multichannel encoding for highly multiplexed quantitative PCR. <i>Anal. Chem.</i> <b>93</b>, 4208–4216 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1021/acs.analchem.0c04626" data-track-item_id="10.1021/acs.analchem.0c04626" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1021%2Facs.analchem.0c04626" aria-label="Article reference 16" data-doi="10.1021/acs.analchem.0c04626">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXkvFyis7o%3D" aria-label="CAS reference 16">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33631072" aria-label="PubMed reference 16">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20multichannel%20encoding%20for%20highly%20multiplexed%20quantitative%20PCR&amp;journal=Anal.%20Chem.&amp;doi=10.1021%2Facs.analchem.0c04626&amp;volume=93&amp;pages=4208-4216&amp;publication_year=2021&amp;author=Jacky%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Burki, T. Understanding variants of SARS-CoV-2. <i>Lancet</i> <b>397</b>, 462 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/S0140-6736(21)00298-1" data-track-item_id="10.1016/S0140-6736(21)00298-1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2FS0140-6736%2821%2900298-1" aria-label="Article reference 17" data-doi="10.1016/S0140-6736(21)00298-1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXjslWltrw%3D" aria-label="CAS reference 17">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33549181" aria-label="PubMed reference 17">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7906644" aria-label="PubMed Central reference 17">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Understanding%20variants%20of%20SARS-CoV-2&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2821%2900298-1&amp;volume=397&amp;publication_year=2021&amp;author=Burki%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Borges, V. &amp; et al. Tracking SARS-CoV-2 lineage B.1.1.7 dissemination: insights from nationwide spike gene target failure (SGTF) and spike gene late detection (SGTL) data, Portugal, week 49 2020 to week 3 2021. <i>Euro Surveill.</i> <b>26</b>, 2100131 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.2807/1560-7917.ES.2021.26.10.2100130" data-track-item_id="10.2807/1560-7917.ES.2021.26.10.2100130" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.2807%2F1560-7917.ES.2021.26.10.2100130" aria-label="Article reference 18" data-doi="10.2807/1560-7917.ES.2021.26.10.2100130">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXnt1WmsLg%3D" aria-label="CAS reference 18">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33706862" aria-label="PubMed reference 18">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7953529" aria-label="PubMed Central reference 18">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Tracking%20SARS-CoV-2%20lineage%20B.1.1.7%20dissemination%3A%20insights%20from%20nationwide%20spike%20gene%20target%20failure%20%28SGTF%29%20and%20spike%20gene%20late%20detection%20%28SGTL%29%20data%2C%20Portugal%2C%20week%2049%202020%20to%20week%203%202021&amp;journal=Euro%20Surveill.&amp;doi=10.2807%2F1560-7917.ES.2021.26.10.2100130&amp;volume=26&amp;publication_year=2021&amp;author=Borges%2CV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">Xuan, J., Yu, Y., Qing, T., Guo, L. &amp; Shi, L. Next-generation sequencing in the clinic: promises and challenges. <i>Cancer Lett.</i> <b>340</b>, 284–295 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.canlet.2012.11.025" data-track-item_id="10.1016/j.canlet.2012.11.025" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.canlet.2012.11.025" aria-label="Article reference 19" data-doi="10.1016/j.canlet.2012.11.025">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXntFWksw%3D%3D" aria-label="CAS reference 19">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23174106" aria-label="PubMed reference 19">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Next-generation%20sequencing%20in%20the%20clinic%3A%20promises%20and%20challenges&amp;journal=Cancer%20Lett.&amp;doi=10.1016%2Fj.canlet.2012.11.025&amp;volume=340&amp;pages=284-295&amp;publication_year=2013&amp;author=Xuan%2CJ&amp;author=Yu%2CY&amp;author=Qing%2CT&amp;author=Guo%2CL&amp;author=Shi%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Houldcroft, C. J., Beale, M. A. &amp; Breuer, J. Clinical and biological insights from viral genome sequencing. <i>Nat. Rev. Microbiol.</i> <b>15</b>, 183–192 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nrmicro.2016.182" data-track-item_id="10.1038/nrmicro.2016.182" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrmicro.2016.182" aria-label="Article reference 20" data-doi="10.1038/nrmicro.2016.182">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXpvVWgtA%3D%3D" aria-label="CAS reference 20">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28090077" aria-label="PubMed reference 20">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097211" aria-label="PubMed Central reference 20">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20and%20biological%20insights%20from%20viral%20genome%20sequencing&amp;journal=Nat.%20Rev.%20Microbiol.&amp;doi=10.1038%2Fnrmicro.2016.182&amp;volume=15&amp;pages=183-192&amp;publication_year=2017&amp;author=Houldcroft%2CCJ&amp;author=Beale%2CMA&amp;author=Breuer%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Peddu, V. et al. Metagenomic analysis reveals clinical SARS-CoV-2 infection and bacterial or viral superinfection and colonization. <i>Clin. Chem.</i> <b>66</b>, 966–972 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/clinchem/hvaa106" data-track-item_id="10.1093/clinchem/hvaa106" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fclinchem%2Fhvaa106" aria-label="Article reference 21" data-doi="10.1093/clinchem/hvaa106">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32379863" aria-label="PubMed reference 21">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Metagenomic%20analysis%20reveals%20clinical%20SARS-CoV-2%20infection%20and%20bacterial%20or%20viral%20superinfection%20and%20colonization&amp;journal=Clin.%20Chem.&amp;doi=10.1093%2Fclinchem%2Fhvaa106&amp;volume=66&amp;pages=966-972&amp;publication_year=2020&amp;author=Peddu%2CV">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Brito, A. F. et al. Global disparities in SARS-CoV-2 genomic surveillance. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2021.08.21.21262393" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.08.21.21262393">https://doi.org/10.1101/2021.08.21.21262393</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Kaminski, M. M., Abudayyeh, O. O., Gootenberg, J. S., Zhang, F. &amp; Collins, J. J. CRISPR-based diagnostics. <i>Nat. Biomed. Eng.</i> <b>5</b>, 643–656 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41551-021-00760-7" data-track-item_id="10.1038/s41551-021-00760-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41551-021-00760-7" aria-label="Article reference 23" data-doi="10.1038/s41551-021-00760-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1ClsLrP" aria-label="CAS reference 23">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34272525" aria-label="PubMed reference 23">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-based%20diagnostics&amp;journal=Nat.%20Biomed.%20Eng.&amp;doi=10.1038%2Fs41551-021-00760-7&amp;volume=5&amp;pages=643-656&amp;publication_year=2021&amp;author=Kaminski%2CMM&amp;author=Abudayyeh%2COO&amp;author=Gootenberg%2CJS&amp;author=Zhang%2CF&amp;author=Collins%2CJJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Jiao, C. et al. Noncanonical crRNAs derived from host transcripts enable multiplexable RNA detection by Cas9. <i>Science</i> <b>372</b>, 941–948 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.abe7106" data-track-item_id="10.1126/science.abe7106" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.abe7106" aria-label="Article reference 24" data-doi="10.1126/science.abe7106">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1WrsL%2FO" aria-label="CAS reference 24">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33906967" aria-label="PubMed reference 24">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8224270" aria-label="PubMed Central reference 24">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Noncanonical%20crRNAs%20derived%20from%20host%20transcripts%20enable%20multiplexable%20RNA%20detection%20by%20Cas9&amp;journal=Science&amp;doi=10.1126%2Fscience.abe7106&amp;volume=372&amp;pages=941-948&amp;publication_year=2021&amp;author=Jiao%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Gootenberg, J. S. et al. Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6. <i>Science</i> <b>360</b>, 439–444 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aaq0179" data-track-item_id="10.1126/science.aaq0179" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aaq0179" aria-label="Article reference 25" data-doi="10.1126/science.aaq0179">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXotFyrsro%3D" aria-label="CAS reference 25">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29449508" aria-label="PubMed reference 25">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5961727" aria-label="PubMed Central reference 25">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%20and%20portable%20nucleic%20acid%20detection%20platform%20with%20Cas13%2C%20Cas12a%2C%20and%20Csm6&amp;journal=Science&amp;doi=10.1126%2Fscience.aaq0179&amp;volume=360&amp;pages=439-444&amp;publication_year=2018&amp;author=Gootenberg%2CJS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Chen, J. S. et al. CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity. <i>Science</i> <b>360</b>, 436–439 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aar6245" data-track-item_id="10.1126/science.aar6245" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aar6245" aria-label="Article reference 26" data-doi="10.1126/science.aar6245">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXotFyrsr8%3D" aria-label="CAS reference 26">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29449511" aria-label="PubMed reference 26">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6628903" aria-label="PubMed Central reference 26">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-Cas12a%20target%20binding%20unleashes%20indiscriminate%20single-stranded%20DNase%20activity&amp;journal=Science&amp;doi=10.1126%2Fscience.aar6245&amp;volume=360&amp;pages=436-439&amp;publication_year=2018&amp;author=Chen%2CJS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Li, S.-Y. et al. CRISPR-Cas12a-assisted nucleic acid detection. <i>Cell Discov.</i> <b>4</b>, 20 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41421-018-0028-z" data-track-item_id="10.1038/s41421-018-0028-z" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41421-018-0028-z" aria-label="Article reference 27" data-doi="10.1038/s41421-018-0028-z">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29707234" aria-label="PubMed reference 27">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5913299" aria-label="PubMed Central reference 27">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-Cas12a-assisted%20nucleic%20acid%20detection&amp;journal=Cell%20Discov.&amp;doi=10.1038%2Fs41421-018-0028-z&amp;volume=4&amp;publication_year=2018&amp;author=Li%2CS-Y">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Abudayyeh, O. O. et al. C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. <i>Science</i> <b>353</b>, aaf5573 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aaf5573" data-track-item_id="10.1126/science.aaf5573" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aaf5573" aria-label="Article reference 28" data-doi="10.1126/science.aaf5573">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27256883" aria-label="PubMed reference 28">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5127784" aria-label="PubMed Central reference 28">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=C2c2%20is%20a%20single-component%20programmable%20RNA-guided%20RNA-targeting%20CRISPR%20effector&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf5573&amp;volume=353&amp;publication_year=2016&amp;author=Abudayyeh%2COO">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Gootenberg, J. S. et al. Nucleic acid detection with CRISPR-Cas13a/C2c2. <i>Science</i> <b>356</b>, 438–442 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aam9321" data-track-item_id="10.1126/science.aam9321" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aam9321" aria-label="Article reference 29" data-doi="10.1126/science.aam9321">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2sXmvFejtb8%3D" aria-label="CAS reference 29">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28408723" aria-label="PubMed reference 29">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5526198" aria-label="PubMed Central reference 29">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Nucleic%20acid%20detection%20with%20CRISPR-Cas13a%2FC2c2&amp;journal=Science&amp;doi=10.1126%2Fscience.aam9321&amp;volume=356&amp;pages=438-442&amp;publication_year=2017&amp;author=Gootenberg%2CJS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Myhrvold, C. et al. Field-deployable viral diagnostics using CRISPR-Cas13. <i>Science</i> <b>360</b>, 444–448 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aas8836" data-track-item_id="10.1126/science.aas8836" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aas8836" aria-label="Article reference 30" data-doi="10.1126/science.aas8836">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXotFyrsr4%3D" aria-label="CAS reference 30">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29700266" aria-label="PubMed reference 30">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6197056" aria-label="PubMed Central reference 30">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Field-deployable%20viral%20diagnostics%20using%20CRISPR-Cas13&amp;journal=Science&amp;doi=10.1126%2Fscience.aas8836&amp;volume=360&amp;pages=444-448&amp;publication_year=2018&amp;author=Myhrvold%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Arizti-Sanz, J. et al. Streamlined inactivation, amplification, and Cas13-based detection of SARS-CoV-2. <i>Nat. Commun.</i> <b>11</b>, 5921 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-19097-x" data-track-item_id="10.1038/s41467-020-19097-x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-19097-x" aria-label="Article reference 31" data-doi="10.1038/s41467-020-19097-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVOgur%2FJ" aria-label="CAS reference 31">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33219225" aria-label="PubMed reference 31">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7680145" aria-label="PubMed Central reference 31">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Streamlined%20inactivation%2C%20amplification%2C%20and%20Cas13-based%20detection%20of%20SARS-CoV-2&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19097-x&amp;volume=11&amp;publication_year=2020&amp;author=Arizti-Sanz%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Broughton, J. P. et al. CRISPR–Cas12-based detection of SARS-CoV-2. <i>Nat. Biotechnol.</i> <b>38</b>, 870–874 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41587-020-0513-4" data-track-item_id="10.1038/s41587-020-0513-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-020-0513-4" aria-label="Article reference 32" data-doi="10.1038/s41587-020-0513-4">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXntlejt7w%3D" aria-label="CAS reference 32">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32300245" aria-label="PubMed reference 32">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9107629" aria-label="PubMed Central reference 32">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%E2%80%93Cas12-based%20detection%20of%20SARS-CoV-2&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-020-0513-4&amp;volume=38&amp;pages=870-874&amp;publication_year=2020&amp;author=Broughton%2CJP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Patchsung, M. et al. Clinical validation of a Cas13-based assay for the detection of SARS-CoV-2 RNA. <i>Nat. Biomed. Eng.</i> <b>4</b>, 1140–1149 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41551-020-00603-x" data-track-item_id="10.1038/s41551-020-00603-x" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41551-020-00603-x" aria-label="Article reference 33" data-doi="10.1038/s41551-020-00603-x">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs12jsbzO" aria-label="CAS reference 33">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32848209" aria-label="PubMed reference 33">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20validation%20of%20a%20Cas13-based%20assay%20for%20the%20detection%20of%20SARS-CoV-2%20RNA&amp;journal=Nat.%20Biomed.%20Eng.&amp;doi=10.1038%2Fs41551-020-00603-x&amp;volume=4&amp;pages=1140-1149&amp;publication_year=2020&amp;author=Patchsung%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Liu, T. Y. et al. Accelerated RNA detection using tandem CRISPR nucleases. <i>Nat. Chem. Biol.</i> <b>17</b>, 982–988 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41589-021-00842-2" data-track-item_id="10.1038/s41589-021-00842-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41589-021-00842-2" aria-label="Article reference 34" data-doi="10.1038/s41589-021-00842-2">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhslart7fK" aria-label="CAS reference 34">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34354262" aria-label="PubMed reference 34">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10184463" aria-label="PubMed Central reference 34">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 34" href="http://scholar.google.com/scholar_lookup?&amp;title=Accelerated%20RNA%20detection%20using%20tandem%20CRISPR%20nucleases&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fs41589-021-00842-2&amp;volume=17&amp;pages=982-988&amp;publication_year=2021&amp;author=Liu%2CTY">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Metsky, H. C. et al. Designing sensitive viral diagnostics with machine learning. <i>Nat. Biotech.</i> <a href="https://doi.org/10.1038/s41587-022-01213-5" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41587-022-01213-5">https://doi.org/10.1038/s41587-022-01213-5</a> (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41587-022-01213-5" data-track-item_id="10.1038/s41587-022-01213-5" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41587-022-01213-5" aria-label="Article reference 35" data-doi="10.1038/s41587-022-01213-5">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=Designing%20sensitive%20viral%20diagnostics%20with%20machine%20learning&amp;journal=Nat.%20Biotech.&amp;doi=10.1038%2Fs41587-022-01213-5&amp;publication_year=2022&amp;author=Metsky%2CHC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Joung, J. et al. Detection of SARS-CoV-2 with SHERLOCK one-pot testing. <i>N. Engl. J. Med.</i> <b>383</b>, 1492–1494 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMc2026172" data-track-item_id="10.1056/NEJMc2026172" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMc2026172" aria-label="Article reference 36" data-doi="10.1056/NEJMc2026172">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVCgtrnF" aria-label="CAS reference 36">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32937062" aria-label="PubMed reference 36">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Detection%20of%20SARS-CoV-2%20with%20SHERLOCK%20one-pot%20testing&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMc2026172&amp;volume=383&amp;pages=1492-1494&amp;publication_year=2020&amp;author=Joung%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Bruch, R., Urban, G. A. &amp; Dincer, C. CRISPR/Cas powered multiplexed biosensing. <i>Trends Biotechnol.</i> <b>37</b>, 791–792 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.tibtech.2019.04.005" data-track-item_id="10.1016/j.tibtech.2019.04.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.tibtech.2019.04.005" aria-label="Article reference 37" data-doi="10.1016/j.tibtech.2019.04.005">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXosFWjt7s%3D" aria-label="CAS reference 37">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31078316" aria-label="PubMed reference 37">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FCas%20powered%20multiplexed%20biosensing&amp;journal=Trends%20Biotechnol.&amp;doi=10.1016%2Fj.tibtech.2019.04.005&amp;volume=37&amp;pages=791-792&amp;publication_year=2019&amp;author=Bruch%2CR&amp;author=Urban%2CGA&amp;author=Dincer%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Tian, T., Qiu, Z., Jiang, Y., Zhu, D. &amp; Zhou, X. Exploiting the orthogonal CRISPR-Cas12a/Cas13a <i>trans</i>-cleavage for dual-gene virus detection using a handheld device. <i>Biosens. Bioelectron.</i> <b>196</b>, 113701 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.bios.2021.113701" data-track-item_id="10.1016/j.bios.2021.113701" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.bios.2021.113701" aria-label="Article reference 38" data-doi="10.1016/j.bios.2021.113701">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1aqtLbL" aria-label="CAS reference 38">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34653714" aria-label="PubMed reference 38">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=Exploiting%20the%20orthogonal%20CRISPR-Cas12a%2FCas13a%20trans-cleavage%20for%20dual-gene%20virus%20detection%20using%20a%20handheld%20device&amp;journal=Biosens.%20Bioelectron.&amp;doi=10.1016%2Fj.bios.2021.113701&amp;volume=196&amp;publication_year=2022&amp;author=Tian%2CT&amp;author=Qiu%2CZ&amp;author=Jiang%2CY&amp;author=Zhu%2CD&amp;author=Zhou%2CX">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Bruch, R. et al. CRISPR-powered electrochemical microfluidic multiplexed biosensor for target amplification-free miRNA diagnostics. <i>Biosens. Bioelectron.</i> <b>177</b>, 112887 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.bios.2020.112887" data-track-item_id="10.1016/j.bios.2020.112887" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.bios.2020.112887" aria-label="Article reference 39" data-doi="10.1016/j.bios.2020.112887">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFWmuro%3D" aria-label="CAS reference 39">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33493854" aria-label="PubMed reference 39">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 39" href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-powered%20electrochemical%20microfluidic%20multiplexed%20biosensor%20for%20target%20amplification-free%20miRNA%20diagnostics&amp;journal=Biosens.%20Bioelectron.&amp;doi=10.1016%2Fj.bios.2020.112887&amp;volume=177&amp;publication_year=2021&amp;author=Bruch%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Ackerman, C. M. et al. Massively multiplexed nucleic acid detection with Cas13. <i>Nature</i> <b>582</b>, 277–282 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-020-2279-8" data-track-item_id="10.1038/s41586-020-2279-8" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-2279-8" aria-label="Article reference 40" data-doi="10.1038/s41586-020-2279-8">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVantLfE" aria-label="CAS reference 40">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32349121" aria-label="PubMed reference 40">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7332423" aria-label="PubMed Central reference 40">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 40" href="http://scholar.google.com/scholar_lookup?&amp;title=Massively%20multiplexed%20nucleic%20acid%20detection%20with%20Cas13&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2279-8&amp;volume=582&amp;pages=277-282&amp;publication_year=2020&amp;author=Ackerman%2CCM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Crowe, J. E. Jr. Human respiratory viruses. Reference Module in Biomedical Sciences <a href="https://doi.org/10.1016/B978-0-12-801238-3.02600-3" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/B978-0-12-801238-3.02600-3">https://doi.org/10.1016/B978-0-12-801238-3.02600-3</a> (2014).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">García Fernández, X., Álvarez-Argüelles, M. E., Rojo, S. &amp; de-Oña, M. Stability of viral RNA in clinical specimens for viral diagnosis. <i>Enferm. Infecc. Microbiol. Clin. (Engl. Ed.)</i> <b>38</b>, 297–298 (2020).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Palmenberg, A. C. et al. Sequencing and analyses of all known human rhinovirus genomes reveal structure and evolution. <i>Science</i> <b>324</b>, 55–59 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.1165557" data-track-item_id="10.1126/science.1165557" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.1165557" aria-label="Article reference 43" data-doi="10.1126/science.1165557">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXjvVajtr4%3D" aria-label="CAS reference 43">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19213880" aria-label="PubMed reference 43">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3923423" aria-label="PubMed Central reference 43">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=Sequencing%20and%20analyses%20of%20all%20known%20human%20rhinovirus%20genomes%20reveal%20structure%20and%20evolution&amp;journal=Science&amp;doi=10.1126%2Fscience.1165557&amp;volume=324&amp;pages=55-59&amp;publication_year=2009&amp;author=Palmenberg%2CAC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">Mackay, I. M., Arden, K. E. &amp; Nitsche, A. Real-time PCR in virology. <i>Nucleic Acids Res.</i> <b>30</b>, 1292–1305 (2002).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/30.6.1292" data-track-item_id="10.1093/nar/30.6.1292" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2F30.6.1292" aria-label="Article reference 44" data-doi="10.1093/nar/30.6.1292">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD38XisVaksbY%3D" aria-label="CAS reference 44">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11884626" aria-label="PubMed reference 44">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC101343" aria-label="PubMed Central reference 44">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 44" href="http://scholar.google.com/scholar_lookup?&amp;title=Real-time%20PCR%20in%20virology&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2F30.6.1292&amp;volume=30&amp;pages=1292-1305&amp;publication_year=2002&amp;author=Mackay%2CIM&amp;author=Arden%2CKE&amp;author=Nitsche%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Pulliam, J. R. C. et al. Increased risk of SARS-CoV-2 reinfection associated with emergence of the Omicron variant in South Africa. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2021.11.11.21266068" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.11.11.21266068">https://doi.org/10.1101/2021.11.11.21266068</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Brandal, L. T. et al. Outbreak caused by the SARS-CoV-2 Omicron variant in Norway, November to December 2021. <i>Euro. Surveill.</i> <b>26</b>, 2101147 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.2807/1560-7917.ES.2021.26.50.2101147" data-track-item_id="10.2807/1560-7917.ES.2021.26.50.2101147" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.2807%2F1560-7917.ES.2021.26.50.2101147" aria-label="Article reference 46" data-doi="10.2807/1560-7917.ES.2021.26.50.2101147">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XkvVOnuw%3D%3D" aria-label="CAS reference 46">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34915975" aria-label="PubMed reference 46">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8728491" aria-label="PubMed Central reference 46">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 46" href="http://scholar.google.com/scholar_lookup?&amp;title=Outbreak%20caused%20by%20the%20SARS-CoV-2%20Omicron%20variant%20in%20Norway%2C%20November%20to%20December%202021&amp;journal=Euro.%20Surveill.&amp;doi=10.2807%2F1560-7917.ES.2021.26.50.2101147&amp;volume=26&amp;publication_year=2021&amp;author=Brandal%2CLT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="47."><p class="c-article-references__text" id="ref-CR47">CDC COVID-19 Response Team SARS-CoV-2 B.1.1.529 (Omicron) Variant—United States, December 1–8, 2021. <i>MMWR Morb. Mortal. Wkly. Rep.</i> <b>70</b>, 1731–1734 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.15585/mmwr.mm7050e1" data-track-item_id="10.15585/mmwr.mm7050e1" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.15585%2Fmmwr.mm7050e1" aria-label="Article reference 47" data-doi="10.15585/mmwr.mm7050e1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8675659" aria-label="PubMed Central reference 47">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 47" href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20B.1.1.529%20%28Omicron%29%20Variant%E2%80%94United%20States%2C%20December%201%E2%80%938%2C%202021&amp;journal=MMWR%20Morb.%20Mortal.%20Wkly.%20Rep.&amp;doi=10.15585%2Fmmwr.mm7050e1&amp;volume=70&amp;pages=1731-1734&amp;publication_year=2021">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="48."><p class="c-article-references__text" id="ref-CR48">Li, A., Maier, A., Carter, M. &amp; Guan, T. H. Omicron and S-gene target failure cases in the highest COVID-19 case rate region in Canada—December 2021. <i>J. Med. Virol</i>. <a href="https://doi.org/10.1002/jmv.27562" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1002/jmv.27562">https://doi.org/10.1002/jmv.27562</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="49."><p class="c-article-references__text" id="ref-CR49">Kirkland, P. D. &amp; Frost, M. J. The impact of viral transport media on PCR assay results for the detection of nucleic acid from SARS-CoV-2. <i>Pathology</i> <b>52</b>, 811–814 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.pathol.2020.09.013" data-track-item_id="10.1016/j.pathol.2020.09.013" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.pathol.2020.09.013" aria-label="Article reference 49" data-doi="10.1016/j.pathol.2020.09.013">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFCrsbfI" aria-label="CAS reference 49">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33250079" aria-label="PubMed reference 49">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 49" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20impact%20of%20viral%20transport%20media%20on%20PCR%20assay%20results%20for%20the%20detection%20of%20nucleic%20acid%20from%20SARS-CoV-2&amp;journal=Pathology&amp;doi=10.1016%2Fj.pathol.2020.09.013&amp;volume=52&amp;pages=811-814&amp;publication_year=2020&amp;author=Kirkland%2CPD&amp;author=Frost%2CMJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="50."><p class="c-article-references__text" id="ref-CR50">Fozouni, P. et al. Amplification-free detection of SARS-CoV-2 with CRISPR–Cas13a and mobile phone microscopy. <i>Cell</i> <b>184</b>, 323–333.e9 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cell.2020.12.001" data-track-item_id="10.1016/j.cell.2020.12.001" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cell.2020.12.001" aria-label="Article reference 50" data-doi="10.1016/j.cell.2020.12.001">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXisF2gsbnE" aria-label="CAS reference 50">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33306959" aria-label="PubMed reference 50">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 50" href="http://scholar.google.com/scholar_lookup?&amp;title=Amplification-free%20detection%20of%20SARS-CoV-2%20with%20CRISPR%E2%80%93Cas13a%20and%20mobile%20phone%20microscopy&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2020.12.001&amp;volume=184&amp;pages=323-333.e9&amp;publication_year=2021&amp;author=Fozouni%2CP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="51."><p class="c-article-references__text" id="ref-CR51">Nalefski, E. A. et al. Kinetic analysis of Cas12a and Cas13a RNA-guided nucleases for development of improved CRISPR-based diagnostics. <i>iScience</i> <b>24</b>, 102996 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.isci.2021.102996" data-track-item_id="10.1016/j.isci.2021.102996" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.isci.2021.102996" aria-label="Article reference 51" data-doi="10.1016/j.isci.2021.102996">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38XitFajtLk%3D" aria-label="CAS reference 51">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34505008" aria-label="PubMed reference 51">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8411246" aria-label="PubMed Central reference 51">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 51" href="http://scholar.google.com/scholar_lookup?&amp;title=Kinetic%20analysis%20of%20Cas12a%20and%20Cas13a%20RNA-guided%20nucleases%20for%20development%20of%20improved%20CRISPR-based%20diagnostics&amp;journal=iScience&amp;doi=10.1016%2Fj.isci.2021.102996&amp;volume=24&amp;publication_year=2021&amp;author=Nalefski%2CEA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="52."><p class="c-article-references__text" id="ref-CR52">Thakku, S. G. et al. Multiplexed detection of bacterial nucleic acids using Cas13 in droplet microarrays. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/2021.11.12.468388" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.11.12.468388">https://doi.org/10.1101/2021.11.12.468388</a> (2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="53."><p class="c-article-references__text" id="ref-CR53">Barnes, K. G. et al. Deployable CRISPR-Cas13a diagnostic tools to detect and report Ebola and Lassa virus cases in real-time. <i>Nat. Commun.</i> <b>11</b>, 4131 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41467-020-17994-9" data-track-item_id="10.1038/s41467-020-17994-9" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41467-020-17994-9" aria-label="Article reference 53" data-doi="10.1038/s41467-020-17994-9">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Crt7%2FP" aria-label="CAS reference 53">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32807807" aria-label="PubMed reference 53">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7431545" aria-label="PubMed Central reference 53">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 53" href="http://scholar.google.com/scholar_lookup?&amp;title=Deployable%20CRISPR-Cas13a%20diagnostic%20tools%20to%20detect%20and%20report%20Ebola%20and%20Lassa%20virus%20cases%20in%20real-time&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-17994-9&amp;volume=11&amp;publication_year=2020&amp;author=Barnes%2CKG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="54."><p class="c-article-references__text" id="ref-CR54">Katoh, K. &amp; Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. <i>Mol. Biol. Evol.</i> <b>30</b>, 772–780 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/molbev/mst010" data-track-item_id="10.1093/molbev/mst010" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fmolbev%2Fmst010" aria-label="Article reference 54" data-doi="10.1093/molbev/mst010">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3sXksFWisLc%3D" aria-label="CAS reference 54">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23329690" aria-label="PubMed reference 54">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603318" aria-label="PubMed Central reference 54">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 54" href="http://scholar.google.com/scholar_lookup?&amp;title=MAFFT%20multiple%20sequence%20alignment%20software%20version%207%3A%20improvements%20in%20performance%20and%20usability&amp;journal=Mol.%20Biol.%20Evol.&amp;doi=10.1093%2Fmolbev%2Fmst010&amp;volume=30&amp;pages=772-780&amp;publication_year=2013&amp;author=Katoh%2CK&amp;author=Standley%2CDM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="55."><p class="c-article-references__text" id="ref-CR55">Matranga, C. B. et al. Enhanced methods for unbiased deep sequencing of Lassa and Ebola RNA viruses from clinical and biological samples. <i>Genome Biol.</i> <b>15</b>, 519 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13059-014-0519-7" data-track-item_id="10.1186/s13059-014-0519-7" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13059-014-0519-7" aria-label="Article reference 55" data-doi="10.1186/s13059-014-0519-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25403361" aria-label="PubMed reference 55">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4262991" aria-label="PubMed Central reference 55">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 55" href="http://scholar.google.com/scholar_lookup?&amp;title=Enhanced%20methods%20for%20unbiased%20deep%20sequencing%20of%20Lassa%20and%20Ebola%20RNA%20viruses%20from%20clinical%20and%20biological%20samples&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-014-0519-7&amp;volume=15&amp;publication_year=2014&amp;author=Matranga%2CCB">
                    Google Scholar</a>&nbsp;
                </p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01734-1?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We thank the Blainey and Hung laboratories at the Broad Institute for providing additional laboratory space to perform the work; J. Arizti Sanz, Y. Zhang and A. Bradley for helping with guide design or sharing reagents; G. Adams, S. Dobbins, S. Slack, K. DeRuff and other members of the Sabeti laboratory COVID-19 sequencing team for providing patient samples; K. Carpenter-Azevedo from RIDOH for organizing and sending the Rhode Island patient samples; C. Tomkins-Tinch, C. Loreth and other Sabeti laboratory members for providing assistance with the NGS analysis on Terra; B. Zhou for helping provide the SARS-CoV-2 VOC seed stocks and patient samples; and H. Metsky, C. Freije, J. Arizti Sanz and S. Siddiqui for their thoughtful discussions and reading of the manuscript. We also acknowledge all members of the Massachusetts Department of Public Health, Broad Institute SARS-CoV-2 Genomics and Data Science Platforms and the Rhode Island Department of Public Health. Funding was provided by the Defense Advanced Research Projects Agency (no. D18AC00006). This work was made possible by support from the Flu laboratory and a cohort of generous donors through TED’s Audacious Project, including the ELMA Foundation, MacKenzie Scott, Skoll Foundation and Open Philanthropy. Funding for NGS was provided by the Centers for Disease Control and Prevention (CDC) COVID-19 baseline genomic surveillance contract sequencing (no. 75D30121C10501 to the Clinical Research Sequencing Platform), a CDC Broad Agency Announcement (no. 75D30120C09605 to B.L.M.) and the National Institute of Allergy and Infectious Diseases (nos. U19AI110818 and U01AI151812 to P.C.S.). M.Z. and M.W.T. were supported by the National Science Foundation Graduate Research Fellowship under grant no. 1745302. B.A.P. was supported by the National Institute of General Medical Sciences grant no. T32GM007753. P.C.S. was supported by the Howard Hughes Medical Institute and Merck KGaA Future Insight Prize. C.M. was supported by start-up funds from Princeton University. The views, opinions, conclusions and/or findings expressed should not be interpreted as representing the official views or policies, either expressed or implied, of the Department of Defense, US Government, National Institute of General Medical Sciences, Department of Homeland Security (DHS) or the National Institutes of Health. The DHS does not endorse any products or commercial services mentioned in this presentation. In no event shall the DHS, Battelle National Biodefense Institute or National Biodefense Analysis and Countermeasures Center have any responsibility or liability for any use, misuse, inability to use or reliance on the information contained herein. In addition, no warranty of fitness for a particular purpose, merchantability, accuracy or adequacy is provided regarding the contents of this document. The US Government retains and the publisher, by accepting the article for publication, acknowledges that the US Government retains a nonexclusive, paid-up, irrevocable, worldwide license to publish or reproduce the published form of this manuscript, or allow others to do so, for US Government purposes. The findings and conclusions in this report are those of the authors and do not necessarily represent the official position of the CDC. The use of trade names and commercial sources is for identification purposes only and does not imply endorsement.</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="na1"><p>These authors jointly supervised this work: John A. Branda, Paul C. Blainey, Pardis C. Sabeti, Cameron Myhrvold.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">Broad Institute of MIT and Harvard, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Nicole L. Welch,&nbsp;Meilin Zhu,&nbsp;Catherine Hua,&nbsp;Juliane Weller,&nbsp;Marzieh Ezzaty Mirhashemi,&nbsp;Tien G. Nguyen,&nbsp;Sreekar Mantena,&nbsp;Matthew R. Bauer,&nbsp;Bennett M. Shaw,&nbsp;Cheri M. Ackerman,&nbsp;Sri Gowtham Thakku,&nbsp;Megan W. Tse,&nbsp;Jared Kehe,&nbsp;Marie-Martine Uwera,&nbsp;Jeremy Johnson,&nbsp;Felecia Cerrato,&nbsp;Gage K. Moreno,&nbsp;Lydia A. Krasilnikova,&nbsp;Brittany A. Petros,&nbsp;Gabrielle L. Gionet,&nbsp;Nicholas A. Fitzgerald,&nbsp;Stacey B. Gabriel,&nbsp;Daniel J. Park,&nbsp;Katherine J. Siddle,&nbsp;Christian T. Happi,&nbsp;Deborah T. Hung,&nbsp;Michael Springer,&nbsp;Bronwyn L. MacInnis,&nbsp;Jacob E. Lemieux,&nbsp;Paul C. Blainey&nbsp;&amp;&nbsp;Pardis C. Sabeti</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Harvard Program in Virology, Division of Medical Sciences, Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Nicole L. Welch</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Department of Biological Engineering, Massachusetts Institute of Technology, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Meilin Zhu,&nbsp;Cheri M. Ackerman,&nbsp;Megan W. Tse,&nbsp;Jared Kehe&nbsp;&amp;&nbsp;Paul C. Blainey</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Division of Infectious Diseases, Department of Medicine, Massachusetts General Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Catherine Hua,&nbsp;Bennett M. Shaw,&nbsp;Jacqueline S. Eversley,&nbsp;Derek A. Bielwaski,&nbsp;Graham McGrath,&nbsp;Joseph Braidt,&nbsp;Jacob E. Lemieux&nbsp;&amp;&nbsp;Eric Rosenberg</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Wellcome Sanger Institute, Wellcome Genome Campus, Hinxton, UK</p><p class="c-article-author-affiliation__authors-list">Juliane Weller</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Harvard Program in Biological and Biomedical Sciences, Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Matthew R. Bauer</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Division of Health Sciences and Technology, Harvard Medical School and Massachusetts Institute of Technology, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Sri Gowtham Thakku&nbsp;&amp;&nbsp;Brittany A. Petros</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Department of Organismic and Evolutionary Biology, Harvard University, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Lydia A. Krasilnikova,&nbsp;Katherine J. Siddle&nbsp;&amp;&nbsp;Pardis C. Sabeti</p></li><li id="Aff9"><p class="c-article-author-affiliation__address">Harvard/Massachusetts Institute of Technology MD-PhD Program, Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Brittany A. Petros</p></li><li id="Aff10"><p class="c-article-author-affiliation__address">Department of Systems Biology, Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Brittany A. Petros,&nbsp;Samantha K. Jalbert,&nbsp;Michael L. Cleary&nbsp;&amp;&nbsp;Michael Springer</p></li><li id="Aff11"><p class="c-article-author-affiliation__address">State Health Laboratories, Rhode Island Department of Health, Providence, RI, USA</p><p class="c-article-author-affiliation__authors-list">Ewa King&nbsp;&amp;&nbsp;Richard C. Huard</p></li><li id="Aff12"><p class="c-article-author-affiliation__address">Massachusetts Department of Public Health, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Glen R. Gallagher,&nbsp;Sandra C. Smole,&nbsp;Lawrence C. Madoff&nbsp;&amp;&nbsp;Catherine M. Brown</p></li><li id="Aff13"><p class="c-article-author-affiliation__address">Influenza Division, National Center for Immunization and Respiratory Diseases, Centers for Disease Control and Prevention, Atlanta, GA, USA</p><p class="c-article-author-affiliation__authors-list">Matthew W. Keller,&nbsp;Malania M. Wilson,&nbsp;Marie K. Kirby&nbsp;&amp;&nbsp;John R. Barnes</p></li><li id="Aff14"><p class="c-article-author-affiliation__address">African Centre of Excellence for Genomics of Infectious Diseases, Redeemer’s University, Ede, Nigeria</p><p class="c-article-author-affiliation__authors-list">Christian T. Happi</p></li><li id="Aff15"><p class="c-article-author-affiliation__address">Department of Biological Sciences, College of Natural Sciences, Redeemer’s University, Ede, Nigeria</p><p class="c-article-author-affiliation__authors-list">Christian T. Happi</p></li><li id="Aff16"><p class="c-article-author-affiliation__address">Molecular Biology Department and Center for Computational and Integrative Biology, Massachusetts General Hospital, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Deborah T. Hung</p></li><li id="Aff17"><p class="c-article-author-affiliation__address">Department of Immunology and Infectious Disease, Harvard T.H. Chan School of Public Health, Harvard University, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Bronwyn L. MacInnis&nbsp;&amp;&nbsp;Pardis C. Sabeti</p></li><li id="Aff18"><p class="c-article-author-affiliation__address">Department of Pathology, Massachusetts General Hospital and Harvard Medical School, Boston, MA, USA</p><p class="c-article-author-affiliation__authors-list">Eric Rosenberg&nbsp;&amp;&nbsp;John A. Branda</p></li><li id="Aff19"><p class="c-article-author-affiliation__address">Koch Institute for Integrative Cancer Research at Massachusetts Institute of Technology, Cambridge, MA, USA</p><p class="c-article-author-affiliation__authors-list">Paul C. Blainey&nbsp;&amp;&nbsp;Pardis C. Sabeti</p></li><li id="Aff20"><p class="c-article-author-affiliation__address">Howard Hughes Medical Institute, Chevy Chase, MD, USA</p><p class="c-article-author-affiliation__authors-list">Pardis C. Sabeti</p></li><li id="Aff21"><p class="c-article-author-affiliation__address">Department of Molecular Biology, Princeton University, Princeton, NJ, USA</p><p class="c-article-author-affiliation__authors-list">Cameron Myhrvold</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Nicole_L_-Welch-Aff1-Aff2"><span class="c-article-authors-search__title u-h3 js-search-name">Nicole L. Welch</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Nicole%20L.%20Welch" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nicole%20L.%20Welch" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nicole%20L.%20Welch%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Meilin-Zhu-Aff1-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Meilin Zhu</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Meilin%20Zhu" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Meilin%20Zhu" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Meilin%20Zhu%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Catherine-Hua-Aff1-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Catherine Hua</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Catherine%20Hua" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Catherine%20Hua" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Catherine%20Hua%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Juliane-Weller-Aff1-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Juliane Weller</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Juliane%20Weller" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Juliane%20Weller" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Juliane%20Weller%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marzieh_Ezzaty-Mirhashemi-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Marzieh Ezzaty Mirhashemi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Marzieh%20Ezzaty%20Mirhashemi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marzieh%20Ezzaty%20Mirhashemi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marzieh%20Ezzaty%20Mirhashemi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Tien_G_-Nguyen-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Tien G. Nguyen</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Tien%20G.%20Nguyen" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tien%20G.%20Nguyen" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tien%20G.%20Nguyen%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sreekar-Mantena-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Sreekar Mantena</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sreekar%20Mantena" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sreekar%20Mantena" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sreekar%20Mantena%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Matthew_R_-Bauer-Aff1-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Matthew R. Bauer</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Matthew%20R.%20Bauer" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Matthew%20R.%20Bauer" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Matthew%20R.%20Bauer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Bennett_M_-Shaw-Aff1-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Bennett M. Shaw</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Bennett%20M.%20Shaw" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Bennett%20M.%20Shaw" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Bennett%20M.%20Shaw%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Cheri_M_-Ackerman-Aff1-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Cheri M. Ackerman</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Cheri%20M.%20Ackerman" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Cheri%20M.%20Ackerman" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Cheri%20M.%20Ackerman%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sri_Gowtham-Thakku-Aff1-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Sri Gowtham Thakku</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sri%20Gowtham%20Thakku" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sri%20Gowtham%20Thakku" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sri%20Gowtham%20Thakku%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Megan_W_-Tse-Aff1-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Megan W. Tse</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Megan%20W.%20Tse" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Megan%20W.%20Tse" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Megan%20W.%20Tse%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jared-Kehe-Aff1-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Jared Kehe</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jared%20Kehe" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jared%20Kehe" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jared%20Kehe%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marie_Martine-Uwera-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Marie-Martine Uwera</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Marie-Martine%20Uwera" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marie-Martine%20Uwera" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marie-Martine%20Uwera%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jacqueline_S_-Eversley-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Jacqueline S. Eversley</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jacqueline%20S.%20Eversley" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jacqueline%20S.%20Eversley" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jacqueline%20S.%20Eversley%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Derek_A_-Bielwaski-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Derek A. Bielwaski</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Derek%20A.%20Bielwaski" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Derek%20A.%20Bielwaski" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Derek%20A.%20Bielwaski%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Graham-McGrath-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Graham McGrath</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Graham%20McGrath" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Graham%20McGrath" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Graham%20McGrath%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Joseph-Braidt-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Joseph Braidt</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Joseph%20Braidt" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Joseph%20Braidt" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Joseph%20Braidt%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jeremy-Johnson-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Jeremy Johnson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jeremy%20Johnson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jeremy%20Johnson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jeremy%20Johnson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Felecia-Cerrato-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Felecia Cerrato</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Felecia%20Cerrato" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Felecia%20Cerrato" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Felecia%20Cerrato%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Gage_K_-Moreno-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Gage K. Moreno</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Gage%20K.%20Moreno" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Gage%20K.%20Moreno" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Gage%20K.%20Moreno%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Lydia_A_-Krasilnikova-Aff1-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Lydia A. Krasilnikova</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Lydia%20A.%20Krasilnikova" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Lydia%20A.%20Krasilnikova" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Lydia%20A.%20Krasilnikova%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Brittany_A_-Petros-Aff1-Aff7-Aff9-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Brittany A. Petros</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Brittany%20A.%20Petros" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Brittany%20A.%20Petros" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Brittany%20A.%20Petros%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Gabrielle_L_-Gionet-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Gabrielle L. Gionet</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Gabrielle%20L.%20Gionet" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Gabrielle%20L.%20Gionet" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Gabrielle%20L.%20Gionet%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ewa-King-Aff11"><span class="c-article-authors-search__title u-h3 js-search-name">Ewa King</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ewa%20King" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ewa%20King" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ewa%20King%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Richard_C_-Huard-Aff11"><span class="c-article-authors-search__title u-h3 js-search-name">Richard C. Huard</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Richard%20C.%20Huard" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Richard%20C.%20Huard" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Richard%20C.%20Huard%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Samantha_K_-Jalbert-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Samantha K. Jalbert</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Samantha%20K.%20Jalbert" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Samantha%20K.%20Jalbert" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Samantha%20K.%20Jalbert%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Michael_L_-Cleary-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Michael L. Cleary</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michael%20L.%20Cleary" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michael%20L.%20Cleary" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20L.%20Cleary%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Nicholas_A_-Fitzgerald-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Nicholas A. Fitzgerald</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Nicholas%20A.%20Fitzgerald" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Nicholas%20A.%20Fitzgerald" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Nicholas%20A.%20Fitzgerald%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Stacey_B_-Gabriel-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Stacey B. Gabriel</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Stacey%20B.%20Gabriel" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Stacey%20B.%20Gabriel" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Stacey%20B.%20Gabriel%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Glen_R_-Gallagher-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Glen R. Gallagher</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Glen%20R.%20Gallagher" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Glen%20R.%20Gallagher" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Glen%20R.%20Gallagher%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sandra_C_-Smole-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Sandra C. Smole</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sandra%20C.%20Smole" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sandra%20C.%20Smole" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sandra%20C.%20Smole%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Lawrence_C_-Madoff-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Lawrence C. Madoff</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Lawrence%20C.%20Madoff" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Lawrence%20C.%20Madoff" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Lawrence%20C.%20Madoff%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Catherine_M_-Brown-Aff12"><span class="c-article-authors-search__title u-h3 js-search-name">Catherine M. Brown</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Catherine%20M.%20Brown" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Catherine%20M.%20Brown" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Catherine%20M.%20Brown%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Matthew_W_-Keller-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Matthew W. Keller</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Matthew%20W.%20Keller" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Matthew%20W.%20Keller" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Matthew%20W.%20Keller%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Malania_M_-Wilson-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Malania M. Wilson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Malania%20M.%20Wilson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Malania%20M.%20Wilson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Malania%20M.%20Wilson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Marie_K_-Kirby-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">Marie K. Kirby</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Marie%20K.%20Kirby" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Marie%20K.%20Kirby" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Marie%20K.%20Kirby%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-John_R_-Barnes-Aff13"><span class="c-article-authors-search__title u-h3 js-search-name">John R. Barnes</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=John%20R.%20Barnes" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=John%20R.%20Barnes" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22John%20R.%20Barnes%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Daniel_J_-Park-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Daniel J. Park</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Daniel%20J.%20Park" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Daniel%20J.%20Park" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Daniel%20J.%20Park%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Katherine_J_-Siddle-Aff1-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Katherine J. Siddle</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Katherine%20J.%20Siddle" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Katherine%20J.%20Siddle" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Katherine%20J.%20Siddle%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Christian_T_-Happi-Aff1-Aff14-Aff15"><span class="c-article-authors-search__title u-h3 js-search-name">Christian T. Happi</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Christian%20T.%20Happi" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Christian%20T.%20Happi" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Christian%20T.%20Happi%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Deborah_T_-Hung-Aff1-Aff16"><span class="c-article-authors-search__title u-h3 js-search-name">Deborah T. Hung</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Deborah%20T.%20Hung" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Deborah%20T.%20Hung" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Deborah%20T.%20Hung%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Michael-Springer-Aff1-Aff10"><span class="c-article-authors-search__title u-h3 js-search-name">Michael Springer</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Michael%20Springer" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Michael%20Springer" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Michael%20Springer%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Bronwyn_L_-MacInnis-Aff1-Aff17"><span class="c-article-authors-search__title u-h3 js-search-name">Bronwyn L. MacInnis</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Bronwyn%20L.%20MacInnis" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Bronwyn%20L.%20MacInnis" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Bronwyn%20L.%20MacInnis%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jacob_E_-Lemieux-Aff1-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Jacob E. Lemieux</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jacob%20E.%20Lemieux" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jacob%20E.%20Lemieux" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jacob%20E.%20Lemieux%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Eric-Rosenberg-Aff4-Aff18"><span class="c-article-authors-search__title u-h3 js-search-name">Eric Rosenberg</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Eric%20Rosenberg" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Eric%20Rosenberg" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Eric%20Rosenberg%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-John_A_-Branda-Aff18"><span class="c-article-authors-search__title u-h3 js-search-name">John A. Branda</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=John%20A.%20Branda" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=John%20A.%20Branda" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22John%20A.%20Branda%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Paul_C_-Blainey-Aff1-Aff3-Aff19"><span class="c-article-authors-search__title u-h3 js-search-name">Paul C. Blainey</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Paul%20C.%20Blainey" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Paul%20C.%20Blainey" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Paul%20C.%20Blainey%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Pardis_C_-Sabeti-Aff1-Aff8-Aff17-Aff19-Aff20"><span class="c-article-authors-search__title u-h3 js-search-name">Pardis C. Sabeti</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Pardis%20C.%20Sabeti" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Pardis%20C.%20Sabeti" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Pardis%20C.%20Sabeti%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Cameron-Myhrvold-Aff21"><span class="c-article-authors-search__title u-h3 js-search-name">Cameron Myhrvold</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Cameron%20Myhrvold" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Cameron%20Myhrvold" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Cameron%20Myhrvold%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>N.L.W., C.M., P.C.B. and P.C.S. initially conceived this study and then involved C.M.A., S.G.T. and J.W. in the preliminary implementation. N.L.W., M.Z., J.W., C.M.A., S.G.T., M.W.T. and J.K. set up the 21 respiratory virus testing on CARMEN v.1. N.L.W. and S.M. designed the primers and crRNAs for the 21 respiratory viruses tested on CARMEN v.1 and mCARMEN. N.L.W., J.W., C.M.A. and S.G.T. performed the initial experiments on the Fluidigm instrumentation. M.Z., J.W. and C.M.A. wrote the Python scripts for the mCARMEN data analysis. N.L.W. performed the experiments to streamline the mCARMEN workflow with help from C.H., J.W. and S.G.T. N.L.W., C.H. and M.E.M. conducted the SARS-CoV-2 patient sample testing in an academic setting. B.L.M. and F.C. helped obtain these samples. C.H., E.M.M., B.M.S., J.S.E., D.A.B. and G.M. performed the clinical evaluation of the mCARMEN RVP at MGH under guidance from N.L.W., J.J., J.E.L., E.R., J.A.B. and C.M. J.W. wrote and generated the software used for the RVP and N.L.W., J.W. and M.U. conducted the thresholding analysis. N.L.W., with help from M.R.B., conducted NGS on the patient samples. N.L.W. designed and tested the primers for VIP and N.L.W. and S.M. designed the crRNAs for VIP. N.L.W. conducted the experimental validation of VIP. M.K.K., M.M.W., M.W.K. and J.R.B. provided the FLUAV samples, SARS-CoV-2 VOC seed stocks and SARS-CoV-2 VOC patient samples and the corresponding NGS data. K.J.S., N.A.F., B.A.P., G.L.G., E.K., R.C.H., S.K.J. and M.L.C. provided assistance with patient sample collection and associated IRBs. N.L.W. tested all SARS-CoV-2 Alpha-Delta samples with mCARMEN. M.Z. wrote and generated the variant calling analysis pipeline for VIP testing under guidance from N.L.W. N.L.W., T.G.N. and M.R.B. tested all Omicron samples by mCARMEN with analysis help from G.K.M., L.A.K. and D.J.P. N.L.W. conducted experiments to make mCARMEN quantitative with assistance from T.G.N. K.J.S., M.S., S.B.G., G.R.G., S.C.S., L.C.M., C.M.B., D.J.P., B.L.M., C.T.H., D.T.H., J.E.L., J.R.B., E.R., J.A.B., P.C.B., P.C.S. and C.M. provided insights into the work overall. N.L.W. generated the figures with help from M.Z. and J.W. N.L.W. wrote the paper with help from C.H. and guidance from P.C.S. and C.M. All authors reviewed the manuscript.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:nicole_welch@g.harvard.edu">Nicole L. Welch</a>, <a id="corresp-c2" href="mailto:pardis@broadinstitute.org">Pardis C. Sabeti</a> or <a id="corresp-c3" href="mailto:cmyhrvol@princeton.edu">Cameron Myhrvold</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar4">Competing interests</h3>
                <p>N.L.W., S.G.T., C.M.A., D.T.H., P.C.B., P.C.S. and C.M. are coinventors on a patent related to this work. P.C.B is a coinventor on patent applications concerning droplet array technologies and serves as a consultant and equity holder of companies in the microfluidics and life sciences industries, including 10x Genomics, GALT, Celsius Therapeutics, Next Gen Diagnostics, Cache DNA and Concerto Biosciences. P.C.B.’s laboratory receives funding from industry for unrelated work. J.E.L. has received consulting fees from Sherlock Biosciences. J.A.B. has received research support for other studies from Pfizer, Zeus, bioMérieux, Immunetics, Abbott Rapid Diagnostics (formerly Alere), Diasorin and the Bay Area Lyme Foundation. P.C.S. is a cofounder of and consultant to Sherlock Biosciences and a board member of the Danaher Corporation, and holds equity in the companies. The other authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
              
              
                <h3 class="c-article__sub-heading" id="FPar3">Peer review information</h3>
                <p><i>Nature Medicine</i> thanks Can Dincer, Francisco J. Sánchez-Rivera and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Michael Basson was the primary editor on this article and managed its editorial process and peer review in collaboration with the rest of the editorial team.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec74-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec74">Extended data</h2><div class="c-article-section__content" id="Sec74-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 detection of 21 respiratory v" href="/articles/s41591-022-01734-1/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig7_ESM.jpg">Extended Data Fig. 1 Detection of 21 respiratory viruses by mCARMEN and CARMEN v.1.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Detailed schematic of loading and running 192.24 IFC in mCARMEN workflow. (Step 1) Up to 196 pre-amplified samples are added in the red microwells of the Fluidigm 192.24 Dynamic Array IFC. Up to 24 detection assay reactions containing Cas13-crRNA complexes are added on the sides of the IFC shown in blue. Control line, actuation, and pressure fluids are added according to manufacturer’s guidelines. (Step 2) The IFC is loaded in a controller where the load and mix protocol is selected. The controller automatically moves nanoliter volumes of reagent, sample, and assay mixes through individual chambers on the IFC until reaching a predetermined chamber in the chip. In the last minute of the protocol, the individual channels open in the chip to allow sample and detection assay mixing to occur. (Step 3) Sequence specific Cas13-crRNA complexes recognize and bind to RNA which unleashes the collateral cleavage activity of Cas13. Cas13 cleaves the quenched fluorescent reporter in solution. (Step 4) The IFC is loaded into the Fluidigm Biomark HD after the controller protocol finishes. The Biomark, incubating at 37 °C, takes images in the ROX, FAM, and HEX channels every 5 mins for up to 3 h post-reaction initiation to monitor fluorescence from the detection reactions. <b>b</b>, Synthetic DNA fragments from each of the 21 human respiratory viruses were serially diluted from 10<sup>3</sup>-10<sup>1</sup> copies µl<sup>−1</sup> and added to Q5 amplification master mix containing primer pool 1 or 2. After amplification, separate sample and detection assay reactions were prepared for fluorescence-based readout using CARMEN v.1. CARMEN v.1 values are shown as raw fluorescence (red), FAM signal at 3 h post-reaction initiation. All samples were background subtracted from NTC-det negative control. <b>c</b>, Kinetic curves of mCARMEN (blue) from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig1">1b</a> and CARMEN v.1 (red) from <b>b</b> at 0, 60, and 180 mins post-reaction initiation.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 single-step amplification tro" href="/articles/s41591-022-01734-1/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig8_ESM.jpg">Extended Data Fig. 2 Single-step amplification troubleshooting of synthetic material and patient specimens.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Kinetic curves of the fluorescence for SARS-CoV-2, FLUBV, and HMPV comparing four different single-step amplification kits Qiagen One-Step RT-PCR, TaqPath, QuantaBio, and Promega. Error bars represent the 95% confidence intervals around the median fluorescence values (n = 3). <b>b</b>, Single-step amplification by Qiagen One-Step RT-PCR kit (top, blue) compared to two-step amplification by SSIV then Q5 (bottom, green) on 21 SARS-CoV-2 positive and 8 SARS-CoV-2 negative patient specimens. Prior RT-qPCR results shown as expected infection positive (+) or negative (-). All specimens were background subtracted using NTC-amp as a negative control. Asterisk (*) represent differences between one-step and two-step amplification results.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 crrna performance during mcar" href="/articles/s41591-022-01734-1/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig9_ESM.jpg">Extended Data Fig. 3 crRNA performance during mCARMEN RVP pre-clinical LOD evaluation.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Fluorescence for 12 replicate testing of SARS-CoV-2 viral seed stock at 5,000, 2,500, 1,000, and 500 copies µl<sup>−1</sup> spiked into negative patient specimens. All samples were background subtracted using NTC-amp as a negative control. Plus sign (+) represents negative sample calls by RVP results. <b>b</b> and <b>e</b>, Area under the receiver operating characteristic (AUROC) curve for crRNA detection. Blue bar represents the maximal AUROC time point at 1 hr post-reaction initiation <b>c</b> and <b>f</b>, Evaluation of sensitivity and specificity of crRNA detection to establish hit calling threshold. Black line: sensitivity/true positive rate; gray line: specificity/true negative rate; red line: hit calling threshold. <b>d</b> and <b>g</b>, Confusion matrix of expected vs predicted detection. <b>b-d</b>, SARS-CoV-2 crRNA detection only. <b>e-g</b>, all nine respiratory viruses on RVP.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 evaluation of discrepant sars" href="/articles/s41591-022-01734-1/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig10_ESM.jpg">Extended Data Fig. 4 Evaluation of discrepant SARS-CoV-2 clinical specimens tested in an academic setting.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, SARS-CoV-2 N1 RT-qPCR inconclusive specimens (n = 45) from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig2">2b</a>. Average N1 Ct of 1 or 2 replicates compared to average N1 Ct across all concordant specimens. <b>b</b>, RNase P RT-qPCR inconclusive specimens from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig2">2b</a>. compared to RNase P Ct values from all concordant specimens with RNase P Ct &lt;40. <b>a</b> and <b>b</b>, Box plot with center lines at the medians; box limits indicate the 25<sup>th</sup> and 75<sup>th</sup> percentiles; whiskers extend 1.5 times the interquartile range from the 25<sup>th</sup> to 75<sup>th</sup> percentiles, all points shown as dots. One-sided t-test was used to measure significance between discrepant and concordant specimens (****, p &lt; 0.0001; ns, not significant). Gray: discrepant specimens; Green: concordant specimens. <b>c</b>, Scatter plot of inconclusive specimens from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig2">2b</a> to compare RNase P Ct to SARS-CoV-2 N1 Ct, if available. Blue: inconclusive N1 Ct &lt;33; Gray: inconclusive N1 Ct &gt;33; X: no N1 Ct. <b>d</b>, Scatter plot of RNase P scaled normalized fluorescence from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig1">1b</a> compared to RNase P Ct values obtained from concurrent RT-qPCR testing of 525 patient specimens. Purple: positive RNase P signal detected by both RVP and RT-qPCR; gray: inconclusive RT-qPCR result indicating one or two of the three technical replicates were undetermined; black: undetermined RT-qPCR result indicating all three technical replicates were negative for RNAse P. <b>e</b>, Concordance of RVP scaled normalized fluorescence values and SARS-CoV-2 read counts by unbiased NGS. Dashed horizontal line: threshold for RVP positivity; dashed vertical line: 10 reads mapped to the SARS-CoV-2 genome by NGS, threshold for NGS positivity. Blue: SARS-CoV-2 N1 RT-qPCR Ct &lt; 25; green: Ct &gt; 25; black: negative by RT-qPCR.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig11"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 mcarmen rvp sensitivity evalu" href="/articles/s41591-022-01734-1/figures/11" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig11_ESM.jpg">Extended Data Fig. 5 mCARMEN RVP sensitivity evaluation, LOD and co-infections, at MGH.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Fluorescent values at the LOD of all 9 viruses on RVP as established by spiking viral seed stock, isolated RNA extracted from clinical specimens, or synthetic RNA into confirmed negative patient specimen for 20 replicates. NTC, no template control; Viral scaled normalized fluorescence values, circles; RNase P values, squares; NTC-noMg values, ‘X’; Line at median fluorescence. <b>b</b>, Scaled normalized fluorescence values at 1 hr post-reaction initiation are shown for SARS-CoV-2 and RNase P, as well as another RVP virus. SARS-CoV-2 viral seed stock was spiked into pooled negative specimen at 2,000 copies µl<sup>−1</sup> (2x the LOD established in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig3">3b</a>). Additionally, viral seed stock, isolated RNA extracted from clinical specimens, or synthetic RNA was spiked at 10<sup>7</sup>-10<sup>3</sup> copies µl<sup>−1</sup>. Error bars represent the 95% confidence intervals around the mean fluorescence values (n = 3). Blue: RVP virus at varying concentrations; light purple: SARS-CoV-2; black: RNase P. <b>c</b>, Concordance between RVP and Biofire RP 2.01/2.1 for 6 patient specimens with co-infections visualized as a split heatmap. Purple: RVP and BioFire 2.0/2.1 positive; blue: RVP virus-positive; green: Biofire RP2.0/2.1 virus-positive; white: virus-negative.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig12"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 mcarmen rvp specificity testi" href="/articles/s41591-022-01734-1/figures/12" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig12_ESM.jpg">Extended Data Fig. 6 mCARMEN RVP specificity testing at MGH on <i>in silico</i> designed synthetic targets and patient specimens.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Fluorescence at 1 hr post-reaction initiation on <i>in silico</i> designed synthetic targets spiked into pooled negative patient specimen at 10<sup>6</sup> copies µl<sup>−1</sup> for specificity evaluation. Plus sign (+) represents expected cross-reactive signal. Dashed vertical line for visual aid. <b>b</b>, Fluorescence at 1 hr post-reaction initiation for 166 patient specimens with known infection status from Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig3">3d–f</a>. Dashed vertical line for visual aid. <b>c</b>, Patient specimen summary from <b>b</b> comparing RVP and comparator assay results (Cepheid and BioFire 2.0/2.1). Asterisk (*) represents co-infections that were confirmed by BioFire 2.0/2.1.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig13"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 7 agreement of mcarmen rvp pati" href="/articles/s41591-022-01734-1/figures/13" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig13_ESM.jpg">Extended Data Fig. 7 Agreement of mCARMEN RVP patient and contrived sample testing in CLIA-certified laboratory.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Workflow of patient specimen evaluation. Patient specimens were originally tested by either Cepheid and BioFire RP2.0/2.1. We tested these specimens by both RVP and BioFire RP2.0/2.1 or TaqPath COVID-19 Combo Kit then a subset of these specimens were subjected to unbiased NGS. <b>b</b>, Concordance between RVP and either Cepheid or BioFire RP2.0/2.1 for 166 patient specimens. Positive and negative percent agreements are shown in parenthesis. <b>c</b>, Concordance between RVP and NGS for 30 out of the 166 patient specimens. <b>d</b>, Concordance of contrived samples to RVP. Contrived samples were prepared by spiking viral seed stock, genomic RNA, or synthetic RNA into negative patient specimens.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig14"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 8 sample quantification with co" href="/articles/s41591-022-01734-1/figures/14" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig14_ESM.jpg">Extended Data Fig. 8 Sample quantification with combined Cas12 and Cas13 detection.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Kinetic curves of SARS-CoV-2 detection by both Cas13 (blue) and Cas12 (orange) for 5 or 7 concentrations, respectively, up to 2 h post-reaction initiation. Dashed lines represent the time at which signal reaches 50% (IC<sub>50</sub> infection point) for each concentration of target. Error bars represent the 95% confidence intervals around the median fluorescence values (n = 2). <b>b</b>, Standard curves for SARS-CoV-2 and FLUAV based on RT-qPCR Ct values. Data related to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig4">4c</a>, now showing a standard 10-fold dilution series. Error bars represent the 95% confidence intervals around the mean RT-qPCR Ct (n = 6).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig15"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 9 mutation detection for alpha," href="/articles/s41591-022-01734-1/figures/15" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig15_ESM.jpg">Extended Data Fig. 9 Mutation detection for Alpha, Beta, Gamma, and Delta SARS-CoV-2 variant lineages.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Kinetics curves of 4-7 SNPs that make up four different variant lineage seed stocks being evaluated. Data shown as normalized fluorescence every 5 mins for 3-4 concentrations of viral seed stock. WA, Alpha, Beta, Gamma: 10<sup>4</sup>-10<sup>2</sup> copies µl<sup>−1</sup>, and Delta: 10<sup>3</sup>-10<sup>1</sup> copies µl<sup>−1</sup>. Purple: variant lineage; Blue: ancestral seed stock; colored from high-to-low concentration. Shaded regions represent 95% confidence intervals around the median fluorescence values and lines are at the median fluorescence.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig16"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 10 specimens evaluated for the " href="/articles/s41591-022-01734-1/figures/16" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig16_ESM.jpg">Extended Data Fig. 10 Specimens evaluated for the presence of Omicron by VIP.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>, Heatmap colored by the percent of available variant genomes on GISAID with a particular variant mutation broken down by variant lineage. <b>b</b>, Delta and Omicron case counts and proportion of Omicron by mCARMEN VIP from specimens collected at the Harvard University CLIA Laboratory (HUCL) from December 6-16 2021, related to Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig6">6b</a> (n = 430). Green: Omicron; Black: Delta. <b>c</b>, Delta and Omicron case counts between VIP and NGS results with the proportion of Omicron by VIP below (n = 1,557). Specimens collected at the Broad Institute from the state of Massachusetts. Blue: Omicron; Black: Delta. <b>d</b>, Scatter plot of the proportion of Delta based on the VIP and NGS variant lineage results from c and Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01734-1#Fig6">6c</a>. Linear regression line fit shown as black line; R<sup>2</sup> = 0.998. <b>e</b>, Time delay from specimen collection to variant lineage result between VIP and NGS. Data represents the proportion of Omicron from specimens collected at HUCL and within the state of Massachusetts. Blue closed circles: VIP MA specimens from c; Blue open circles: NGS MA specimens from c; Green: VIP HUCL specimens from b.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec75-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec75">Supplementary information</h2><div class="c-article-section__content" id="Sec75-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Figs. 1–7 and Tables 1–11 (titles and captions only).</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary tables" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_MOESM3_ESM.xlsx" data-supp-info-image="">Supplementary Tables</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>See Supplementary Information File for all 11 table titles.</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM4"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary data" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_MOESM4_ESM.pdf" data-supp-info-image="">Supplementary Data</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>mCARMEN standard operating procedure.</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Multiplexed%20CRISPR-based%20microfluidic%20platform%20for%20clinical%20testing%20of%20respiratory%20viruses%20and%20identification%20of%20SARS-CoV-2%20variants&amp;author=Nicole%20L.%20Welch%20et%20al&amp;contentID=10.1038%2Fs41591-022-01734-1&amp;copyright=The%20Author%28s%29&amp;publication=1078-8956&amp;publicationDate=2022-02-07&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41591-022-01734-1" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41591-022-01734-1" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Welch, N.L., Zhu, M., Hua, C. <i>et al.</i> Multiplexed CRISPR-based microfluidic platform for clinical testing of respiratory viruses and identification of SARS-CoV-2 variants.
                    <i>Nat Med</i> <b>28</b>, 1083–1094 (2022). https://doi.org/10.1038/s41591-022-01734-1</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01734-1?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-11-22">22 November 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-02-03">03 February 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-02-07">07 February 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-05">May 2022</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41591-022-01734-1</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-block"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/biotechnology" data-track="click" data-track-action="view subject" data-track-label="link">Biotechnology</a></li><li class="c-article-subject-list__subject"><a href="/subjects/infectious-disease-diagnostics" data-track="click" data-track-action="view subject" data-track-label="link">Infectious-disease diagnostics</a></li>
        </ul>
    </div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section" data-test="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:PathoGD: an integrative genomics approach to primer and guide RNA design for CRISPR-based diagnostics" href="https://doi.org/10.1038/s42003-025-07591-1">
                                        PathoGD: an integrative genomics approach to primer and guide RNA design for CRISPR-based diagnostics
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Soo Jen Low</li><li>Matthew O’Neill</li><li>Deborah A. Williamson</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Communications Biology</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Ultrasensitive detection of clinical pathogens through a target-amplification-free collateral-cleavage-enhancing CRISPR-CasΦ tool" href="https://doi.org/10.1038/s41467-025-59219-x">
                                        Ultrasensitive detection of clinical pathogens through a target-amplification-free collateral-cleavage-enhancing CRISPR-CasΦ tool
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Huiyou Chen</li><li>Fengge Song</li><li>Chuanbin Mao</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Porous GNPs assisted LAMP-CRISPR/Cas12a amperometric biosensor as a potential point of care testing system for SARS-CoV-2" href="https://doi.org/10.1007/s00604-025-07094-0">
                                        Porous GNPs assisted LAMP-CRISPR/Cas12a amperometric biosensor as a potential point of care testing system for SARS-CoV-2
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Mohammad Behnam Rad</li><li>Seyed Reza Mohebbi</li><li>Hedayatollah Ghourchian</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Microchimica Acta</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:The next viral pandemic—where do we stand?" href="https://doi.org/10.1007/s12223-025-01256-6">
                                        The next viral pandemic—where do we stand?
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Rui Fan</li><li>Qun Liang</li><li>Xingxing Yuan</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Folia Microbiologica</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Leveraging CRISPR-Cas-Enhanced Isothermal Amplification Tools for Quick Identification of Pathogens Causing Livestock Diseases" href="https://doi.org/10.1007/s00284-025-04226-w">
                                        Leveraging CRISPR-Cas-Enhanced Isothermal Amplification Tools for Quick Identification of Pathogens Causing Livestock Diseases
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Ayan Mukherjee</li><li>Sukhen Samanta</li><li>Premanshu Dandapat</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Current Microbiology</i> (2025)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01734-1.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                
            
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
            <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" role="tabpanel"><div class="c-reading-companion__scroll-pane" style="max-height: none;"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Main">Main</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec9" class="c-reading-companion__section-item"><a href="#Sec9" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec10" class="c-reading-companion__section-item"><a href="#Sec10" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-data-availability" class="c-reading-companion__section-item"><a href="#data-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Data availability">Data availability</a></li><li id="rc-sec-code-availability" class="c-reading-companion__section-item"><a href="#code-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Code availability">Code availability</a></li><li id="rc-sec-change-history" class="c-reading-companion__section-item"><a href="#change-history" data-track="click" data-track-action="section anchor" data-track-label="link:Change history">Change history</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-peer-review" class="c-reading-companion__section-item"><a href="#peer-review" data-track="click" data-track-action="section anchor" data-track-label="link:Peer review">Peer review</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec74" class="c-reading-companion__section-item"><a href="#Sec74" data-track="click" data-track-action="section anchor" data-track-label="link:Extended data">Extended data</a></li><li id="rc-sec-Sec75" class="c-reading-companion__section-item"><a href="#Sec75" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by">This article is cited by</a></li></ul></div>
                <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="" style="display: none !important;">
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-test="right-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41591-022-01734-1;doi=10.1038/s41591-022-01734-1;subjmeta=2521,326,61,631;kwrd=Biotechnology,Infectious-disease+diagnostics">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=-723085697&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01734-1%26doi%3D10.1038/s41591-022-01734-1%26subjmeta%3D2521,326,61,631%26kwrd%3DBiotechnology,Infectious-disease+diagnostics">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=-723085697&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01734-1%26doi%3D10.1038/s41591-022-01734-1%26subjmeta%3D2521,326,61,631%26kwrd%3DBiotechnology,Infectious-disease+diagnostics"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Fig. 1: Implementation of CARMEN using a microfluidic system improves sensitivity and speed.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig1_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig1_HTML.png" alt="figure 1" aria-describedby="rc-Fig1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Fig. 2: Evaluation of an automated and condensed mCARMEN workflow.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig2_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig2_HTML.png" alt="figure 2" aria-describedby="rc-Fig2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Fig. 3: Clinical evaluation of RVP in a CLIA-certified laboratory.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig3_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig3_HTML.png" alt="figure 3" aria-describedby="rc-Fig3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Fig. 4: Viral quantification using Cas12 and Cas13 in combination.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig4_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig4_HTML.png" alt="figure 4" aria-describedby="rc-Fig4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Fig. 5: SARS-CoV-2 variant identification using SNP-determining Cas13 crRNA combinations.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig5_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig5_HTML.png" alt="figure 5" aria-describedby="rc-Fig5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig6">Fig. 6: Rapid and specific identification of the Omicron variant using the mCARMEN VIP.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig6_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig6_HTML.png" alt="figure 6" aria-describedby="rc-Fig6"></picture><p class="c-reading-companion__figure-links"><a href="#Fig6" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/6" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig7">Extended Data Fig. 1 Detection of 21 respiratory viruses by mCARMEN and CARMEN v.1.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig7_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig7_ESM.jpg" alt="extended data figure 7" aria-describedby="rc-Fig7"></picture><p class="c-reading-companion__figure-links"><a href="#Fig7" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/7" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig8">Extended Data Fig. 2 Single-step amplification troubleshooting of synthetic material and patient specimens.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig8_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig8_ESM.jpg" alt="extended data figure 8" aria-describedby="rc-Fig8"></picture><p class="c-reading-companion__figure-links"><a href="#Fig8" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/8" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig9">Extended Data Fig. 3 crRNA performance during mCARMEN RVP pre-clinical LOD evaluation.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig9_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig9_ESM.jpg" alt="extended data figure 9" aria-describedby="rc-Fig9"></picture><p class="c-reading-companion__figure-links"><a href="#Fig9" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/9" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig10">Extended Data Fig. 4 Evaluation of discrepant SARS-CoV-2 clinical specimens tested in an academic setting.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig10_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig10_ESM.jpg" alt="extended data figure 10" aria-describedby="rc-Fig10"></picture><p class="c-reading-companion__figure-links"><a href="#Fig10" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/10" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig11">Extended Data Fig. 5 mCARMEN RVP sensitivity evaluation, LOD and co-infections, at MGH.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig11_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig11_ESM.jpg" alt="extended data figure 11" aria-describedby="rc-Fig11"></picture><p class="c-reading-companion__figure-links"><a href="#Fig11" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/11" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig12">Extended Data Fig. 6 mCARMEN RVP specificity testing at MGH on <i>in silico</i> designed synthetic targets and patient specimens.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig12_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig12_ESM.jpg" alt="extended data figure 12" aria-describedby="rc-Fig12"></picture><p class="c-reading-companion__figure-links"><a href="#Fig12" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/12" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig13">Extended Data Fig. 7 Agreement of mCARMEN RVP patient and contrived sample testing in CLIA-certified laboratory.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig13_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig13_ESM.jpg" alt="extended data figure 13" aria-describedby="rc-Fig13"></picture><p class="c-reading-companion__figure-links"><a href="#Fig13" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/13" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig14">Extended Data Fig. 8 Sample quantification with combined Cas12 and Cas13 detection.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig14_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig14_ESM.jpg" alt="extended data figure 14" aria-describedby="rc-Fig14"></picture><p class="c-reading-companion__figure-links"><a href="#Fig14" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/14" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig15">Extended Data Fig. 9 Mutation detection for Alpha, Beta, Gamma, and Delta SARS-CoV-2 variant lineages.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig15_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig15_ESM.jpg" alt="extended data figure 15" aria-describedby="rc-Fig15"></picture><p class="c-reading-companion__figure-links"><a href="#Fig15" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/15" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig16">Extended Data Fig. 10 Specimens evaluated for the presence of Omicron by VIP.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig16_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01734-1/MediaObjects/41591_2022_1734_Fig16_ESM.jpg" alt="extended data figure 16" aria-describedby="rc-Fig16"></picture><p class="c-reading-companion__figure-links"><a href="#Fig16" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01734-1/figures/16" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric" data-track-context="article references sidebar"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">Mina, M. J. &amp; Andersen, K. G. COVID-19 testing: one size does not fit all. <i>Science</i> <b>371</b>, 126–127 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.abe9187" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.abe9187" data-track-item_id="10.1126/science.abe9187">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtVCgsLk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33414210" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=COVID-19%20testing%3A%20one%20size%20does%20not%20fit%20all&amp;journal=Science&amp;doi=10.1126%2Fscience.abe9187&amp;volume=371&amp;pages=126-127&amp;publication_year=2021&amp;author=Mina%2CMJ&amp;author=Andersen%2CKG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">Rasmussen, A. L. &amp; Popescu, S. V. SARS-CoV-2 transmission without symptoms. <i>Science</i> <b>371</b>, 1206–1207 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.abf9569" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.abf9569" data-track-item_id="10.1126/science.abf9569">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXmvF2jsr0%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33737476" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20transmission%20without%20symptoms&amp;journal=Science&amp;doi=10.1126%2Fscience.abf9569&amp;volume=371&amp;pages=1206-1207&amp;publication_year=2021&amp;author=Rasmussen%2CAL&amp;author=Popescu%2CSV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Winichakoon, P. et al. Negative nasopharyngeal and oropharyngeal swabs do not rule out COVID-19. <i>J. Clin. Microbiol.</i> <b>58</b>, e00297-20 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1128%2FJCM.00297-20" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1128/JCM.00297-20" data-track-item_id="10.1128/JCM.00297-20">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32102856" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7180262" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Negative%20nasopharyngeal%20and%20oropharyngeal%20swabs%20do%20not%20rule%20out%20COVID-19&amp;journal=J.%20Clin.%20Microbiol.&amp;doi=10.1128%2FJCM.00297-20&amp;volume=58&amp;publication_year=2020&amp;author=Winichakoon%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">Woloshin, S., Patel, N. &amp; Kesselheim, A. S. False negative tests for SARS-CoV-2 infection—challenges and implications. <i>N. Engl. J. Med.</i> <b>383</b>, e38 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMp2015897" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMp2015897" data-track-item_id="10.1056/NEJMp2015897">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFKjt7zK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32502334" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=False%20negative%20tests%20for%20SARS-CoV-2%20infection%E2%80%94challenges%20and%20implications&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMp2015897&amp;volume=383&amp;publication_year=2020&amp;author=Woloshin%2CS&amp;author=Patel%2CN&amp;author=Kesselheim%2CAS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">Lemieux, J. E. et al. Phylogenetic analysis of SARS-CoV-2 in Boston highlights the impact of superspreading events. <i>Science</i> <b>371</b>, eabe326 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.abe3261" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.abe3261" data-track-item_id="10.1126/science.abe3261">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Phylogenetic%20analysis%20of%20SARS-CoV-2%20in%20Boston%20highlights%20the%20impact%20of%20superspreading%20events&amp;journal=Science&amp;doi=10.1126%2Fscience.abe3261&amp;volume=371&amp;publication_year=2021&amp;author=Lemieux%2CJE" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Harvey, W. T. et al. SARS-CoV-2 variants, spike mutations and immune escape. <i>Nat. Rev. Microbiol.</i> <b>19</b>, 409–424 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41579-021-00573-0" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41579-021-00573-0" data-track-item_id="10.1038/s41579-021-00573-0">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1WqtbrP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34075212" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8167834" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20variants%2C%20spike%20mutations%20and%20immune%20escape&amp;journal=Nat.%20Rev.%20Microbiol.&amp;doi=10.1038%2Fs41579-021-00573-0&amp;volume=19&amp;pages=409-424&amp;publication_year=2021&amp;author=Harvey%2CWT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Dyson, L. et al. Possible future waves of SARS-CoV-2 infection generated by variants of concern with a range of characteristics. <i>Nat. Commun.</i> <b>12</b>, 5730 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-021-25915-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-021-25915-7" data-track-item_id="10.1038/s41467-021-25915-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFGhsbnN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34593807" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8484271" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Possible%20future%20waves%20of%20SARS-CoV-2%20infection%20generated%20by%20variants%20of%20concern%20with%20a%20range%20of%20characteristics&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-021-25915-7&amp;volume=12&amp;publication_year=2021&amp;author=Dyson%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Zou, L. et al. SARS-CoV-2 viral load in upper respiratory specimens of infected patients. <i>N. Engl. J. Med.</i> <b>382</b>, 1177–1179 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMc2001737" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMc2001737" data-track-item_id="10.1056/NEJMc2001737">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32074444" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7121626" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20viral%20load%20in%20upper%20respiratory%20specimens%20of%20infected%20patients&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMc2001737&amp;volume=382&amp;pages=1177-1179&amp;publication_year=2020&amp;author=Zou%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Wölfel, R. et al. Virological assessment of hospitalized patients with COVID-2019. <i>Nature</i> <b>581</b>, 465–469 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-020-2196-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-020-2196-x" data-track-item_id="10.1038/s41586-020-2196-x">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32235945" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Virological%20assessment%20of%20hospitalized%20patients%20with%20COVID-2019&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2196-x&amp;volume=581&amp;pages=465-469&amp;publication_year=2020&amp;author=W%C3%B6lfel%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Vogels, C. B. F. et al. Multiplex qPCR discriminates variants of concern to enhance global surveillance of SARS-CoV-2. <i>PLoS Biol.</i> <b>19</b>, e3001236 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1371%2Fjournal.pbio.3001236" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1371/journal.pbio.3001236" data-track-item_id="10.1371/journal.pbio.3001236">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhtFGju7jM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33961632" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8133773" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplex%20qPCR%20discriminates%20variants%20of%20concern%20to%20enhance%20global%20surveillance%20of%20SARS-CoV-2&amp;journal=PLoS%20Biol.&amp;doi=10.1371%2Fjournal.pbio.3001236&amp;volume=19&amp;publication_year=2021&amp;author=Vogels%2CCBF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Heggestad, J. T. et al. Multiplexed, quantitative serological profiling of COVID-19 from blood by a point-of-care test. <i>Sci. Adv.</i> <b>7</b>, eabg4901 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fsciadv.abg4901" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/sciadv.abg4901" data-track-item_id="10.1126/sciadv.abg4901">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhsFeltL7O" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34172447" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8232907" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%2C%20quantitative%20serological%20profiling%20of%20COVID-19%20from%20blood%20by%20a%20point-of-care%20test&amp;journal=Sci.%20Adv.&amp;doi=10.1126%2Fsciadv.abg4901&amp;volume=7&amp;publication_year=2021&amp;author=Heggestad%2CJT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Pham, J. et al. Performance characteristics of a high-throughput automated transcription-mediated amplification test for SARS-CoV-2 detection. <i>J. Clin. Microbiol.</i> <b>58</b>, e01669-20 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1128%2FJCM.01669-20" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1128/JCM.01669-20" data-track-item_id="10.1128/JCM.01669-20">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32727828" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7512162" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Performance%20characteristics%20of%20a%20high-throughput%20automated%20transcription-mediated%20amplification%20test%20for%20SARS-CoV-2%20detection&amp;journal=J.%20Clin.%20Microbiol.&amp;doi=10.1128%2FJCM.01669-20&amp;volume=58&amp;publication_year=2020&amp;author=Pham%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Eckbo, E. J. et al. Evaluation of the BioFire® COVID-19 test and Respiratory Panel 2.1 for rapid identification of SARS-CoV-2 in nasopharyngeal swab samples. <i>Diagn. Microbiol. Infect. Dis.</i> <b>99</b>, 115260 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.diagmicrobio.2020.115260" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.diagmicrobio.2020.115260" data-track-item_id="10.1016/j.diagmicrobio.2020.115260">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXksFyltg%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33340934" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Evaluation%20of%20the%20BioFire%C2%AE%20COVID-19%20test%20and%20Respiratory%20Panel%202.1%20for%20rapid%20identification%20of%20SARS-CoV-2%20in%20nasopharyngeal%20swab%20samples&amp;journal=Diagn.%20Microbiol.%20Infect.%20Dis.&amp;doi=10.1016%2Fj.diagmicrobio.2020.115260&amp;volume=99&amp;publication_year=2021&amp;author=Eckbo%2CEJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Konings, F. et al. SARS-CoV-2 variants of interest and concern naming scheme conducive for global discourse. <i>Nat. Microbiol.</i> <b>6</b>, 821–823 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41564-021-00932-w" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41564-021-00932-w" data-track-item_id="10.1038/s41564-021-00932-w">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1yhsLbL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34108654" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20variants%20of%20interest%20and%20concern%20naming%20scheme%20conducive%20for%20global%20discourse&amp;journal=Nat.%20Microbiol.&amp;doi=10.1038%2Fs41564-021-00932-w&amp;volume=6&amp;pages=821-823&amp;publication_year=2021&amp;author=Konings%2CF" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">Huang, H.-S. et al. Multiplex PCR system for the rapid diagnosis of respiratory virus infection: systematic review and meta-analysis. <i>Clin. Microbiol. Infect.</i> <b>24</b>, 1055–1063 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cmi.2017.11.018" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cmi.2017.11.018" data-track-item_id="10.1016/j.cmi.2017.11.018">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXitVCkt7fL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29208560" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplex%20PCR%20system%20for%20the%20rapid%20diagnosis%20of%20respiratory%20virus%20infection%3A%20systematic%20review%20and%20meta-analysis&amp;journal=Clin.%20Microbiol.%20Infect.&amp;doi=10.1016%2Fj.cmi.2017.11.018&amp;volume=24&amp;pages=1055-1063&amp;publication_year=2018&amp;author=Huang%2CH-S" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">Jacky, L. et al. Robust multichannel encoding for highly multiplexed quantitative PCR. <i>Anal. Chem.</i> <b>93</b>, 4208–4216 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1021%2Facs.analchem.0c04626" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1021/acs.analchem.0c04626" data-track-item_id="10.1021/acs.analchem.0c04626">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXkvFyis7o%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33631072" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Robust%20multichannel%20encoding%20for%20highly%20multiplexed%20quantitative%20PCR&amp;journal=Anal.%20Chem.&amp;doi=10.1021%2Facs.analchem.0c04626&amp;volume=93&amp;pages=4208-4216&amp;publication_year=2021&amp;author=Jacky%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Burki, T. Understanding variants of SARS-CoV-2. <i>Lancet</i> <b>397</b>, 462 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2FS0140-6736%2821%2900298-1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/S0140-6736(21)00298-1" data-track-item_id="10.1016/S0140-6736(21)00298-1">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXjslWltrw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33549181" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7906644" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Understanding%20variants%20of%20SARS-CoV-2&amp;journal=Lancet&amp;doi=10.1016%2FS0140-6736%2821%2900298-1&amp;volume=397&amp;publication_year=2021&amp;author=Burki%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">Borges, V. &amp; et al. Tracking SARS-CoV-2 lineage B.1.1.7 dissemination: insights from nationwide spike gene target failure (SGTF) and spike gene late detection (SGTL) data, Portugal, week 49 2020 to week 3 2021. <i>Euro Surveill.</i> <b>26</b>, 2100131 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.2807%2F1560-7917.ES.2021.26.10.2100130" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.2807/1560-7917.ES.2021.26.10.2100130" data-track-item_id="10.2807/1560-7917.ES.2021.26.10.2100130">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXnt1WmsLg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33706862" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7953529" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Tracking%20SARS-CoV-2%20lineage%20B.1.1.7%20dissemination%3A%20insights%20from%20nationwide%20spike%20gene%20target%20failure%20%28SGTF%29%20and%20spike%20gene%20late%20detection%20%28SGTL%29%20data%2C%20Portugal%2C%20week%2049%202020%20to%20week%203%202021&amp;journal=Euro%20Surveill.&amp;doi=10.2807%2F1560-7917.ES.2021.26.10.2100130&amp;volume=26&amp;publication_year=2021&amp;author=Borges%2CV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">Xuan, J., Yu, Y., Qing, T., Guo, L. &amp; Shi, L. Next-generation sequencing in the clinic: promises and challenges. <i>Cancer Lett.</i> <b>340</b>, 284–295 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.canlet.2012.11.025" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.canlet.2012.11.025" data-track-item_id="10.1016/j.canlet.2012.11.025">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXntFWksw%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23174106" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Next-generation%20sequencing%20in%20the%20clinic%3A%20promises%20and%20challenges&amp;journal=Cancer%20Lett.&amp;doi=10.1016%2Fj.canlet.2012.11.025&amp;volume=340&amp;pages=284-295&amp;publication_year=2013&amp;author=Xuan%2CJ&amp;author=Yu%2CY&amp;author=Qing%2CT&amp;author=Guo%2CL&amp;author=Shi%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Houldcroft, C. J., Beale, M. A. &amp; Breuer, J. Clinical and biological insights from viral genome sequencing. <i>Nat. Rev. Microbiol.</i> <b>15</b>, 183–192 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnrmicro.2016.182" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nrmicro.2016.182" data-track-item_id="10.1038/nrmicro.2016.182">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXpvVWgtA%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28090077" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7097211" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20and%20biological%20insights%20from%20viral%20genome%20sequencing&amp;journal=Nat.%20Rev.%20Microbiol.&amp;doi=10.1038%2Fnrmicro.2016.182&amp;volume=15&amp;pages=183-192&amp;publication_year=2017&amp;author=Houldcroft%2CCJ&amp;author=Beale%2CMA&amp;author=Breuer%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Peddu, V. et al. Metagenomic analysis reveals clinical SARS-CoV-2 infection and bacterial or viral superinfection and colonization. <i>Clin. Chem.</i> <b>66</b>, 966–972 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fclinchem%2Fhvaa106" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/clinchem/hvaa106" data-track-item_id="10.1093/clinchem/hvaa106">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32379863" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Metagenomic%20analysis%20reveals%20clinical%20SARS-CoV-2%20infection%20and%20bacterial%20or%20viral%20superinfection%20and%20colonization&amp;journal=Clin.%20Chem.&amp;doi=10.1093%2Fclinchem%2Fhvaa106&amp;volume=66&amp;pages=966-972&amp;publication_year=2020&amp;author=Peddu%2CV" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Brito, A. F. et al. Global disparities in SARS-CoV-2 genomic surveillance. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2021.08.21.21262393" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.08.21.21262393">https://doi.org/10.1101/2021.08.21.21262393</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">Kaminski, M. M., Abudayyeh, O. O., Gootenberg, J. S., Zhang, F. &amp; Collins, J. J. CRISPR-based diagnostics. <i>Nat. Biomed. Eng.</i> <b>5</b>, 643–656 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41551-021-00760-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41551-021-00760-7" data-track-item_id="10.1038/s41551-021-00760-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhs1ClsLrP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34272525" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-based%20diagnostics&amp;journal=Nat.%20Biomed.%20Eng.&amp;doi=10.1038%2Fs41551-021-00760-7&amp;volume=5&amp;pages=643-656&amp;publication_year=2021&amp;author=Kaminski%2CMM&amp;author=Abudayyeh%2COO&amp;author=Gootenberg%2CJS&amp;author=Zhang%2CF&amp;author=Collins%2CJJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Jiao, C. et al. Noncanonical crRNAs derived from host transcripts enable multiplexable RNA detection by Cas9. <i>Science</i> <b>372</b>, 941–948 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.abe7106" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.abe7106" data-track-item_id="10.1126/science.abe7106">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXht1WrsL%2FO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33906967" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8224270" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Noncanonical%20crRNAs%20derived%20from%20host%20transcripts%20enable%20multiplexable%20RNA%20detection%20by%20Cas9&amp;journal=Science&amp;doi=10.1126%2Fscience.abe7106&amp;volume=372&amp;pages=941-948&amp;publication_year=2021&amp;author=Jiao%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">Gootenberg, J. S. et al. Multiplexed and portable nucleic acid detection platform with Cas13, Cas12a, and Csm6. <i>Science</i> <b>360</b>, 439–444 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aaq0179" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aaq0179" data-track-item_id="10.1126/science.aaq0179">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXotFyrsro%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29449508" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5961727" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Multiplexed%20and%20portable%20nucleic%20acid%20detection%20platform%20with%20Cas13%2C%20Cas12a%2C%20and%20Csm6&amp;journal=Science&amp;doi=10.1126%2Fscience.aaq0179&amp;volume=360&amp;pages=439-444&amp;publication_year=2018&amp;author=Gootenberg%2CJS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Chen, J. S. et al. CRISPR-Cas12a target binding unleashes indiscriminate single-stranded DNase activity. <i>Science</i> <b>360</b>, 436–439 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aar6245" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aar6245" data-track-item_id="10.1126/science.aar6245">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXotFyrsr8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29449511" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6628903" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-Cas12a%20target%20binding%20unleashes%20indiscriminate%20single-stranded%20DNase%20activity&amp;journal=Science&amp;doi=10.1126%2Fscience.aar6245&amp;volume=360&amp;pages=436-439&amp;publication_year=2018&amp;author=Chen%2CJS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">Li, S.-Y. et al. CRISPR-Cas12a-assisted nucleic acid detection. <i>Cell Discov.</i> <b>4</b>, 20 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41421-018-0028-z" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41421-018-0028-z" data-track-item_id="10.1038/s41421-018-0028-z">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29707234" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5913299" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-Cas12a-assisted%20nucleic%20acid%20detection&amp;journal=Cell%20Discov.&amp;doi=10.1038%2Fs41421-018-0028-z&amp;volume=4&amp;publication_year=2018&amp;author=Li%2CS-Y" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Abudayyeh, O. O. et al. C2c2 is a single-component programmable RNA-guided RNA-targeting CRISPR effector. <i>Science</i> <b>353</b>, aaf5573 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aaf5573" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aaf5573" data-track-item_id="10.1126/science.aaf5573">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27256883" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5127784" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=C2c2%20is%20a%20single-component%20programmable%20RNA-guided%20RNA-targeting%20CRISPR%20effector&amp;journal=Science&amp;doi=10.1126%2Fscience.aaf5573&amp;volume=353&amp;publication_year=2016&amp;author=Abudayyeh%2COO" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">Gootenberg, J. S. et al. Nucleic acid detection with CRISPR-Cas13a/C2c2. <i>Science</i> <b>356</b>, 438–442 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aam9321" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aam9321" data-track-item_id="10.1126/science.aam9321">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2sXmvFejtb8%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28408723" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5526198" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Nucleic%20acid%20detection%20with%20CRISPR-Cas13a%2FC2c2&amp;journal=Science&amp;doi=10.1126%2Fscience.aam9321&amp;volume=356&amp;pages=438-442&amp;publication_year=2017&amp;author=Gootenberg%2CJS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">Myhrvold, C. et al. Field-deployable viral diagnostics using CRISPR-Cas13. <i>Science</i> <b>360</b>, 444–448 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aas8836" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aas8836" data-track-item_id="10.1126/science.aas8836">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXotFyrsr4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29700266" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6197056" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Field-deployable%20viral%20diagnostics%20using%20CRISPR-Cas13&amp;journal=Science&amp;doi=10.1126%2Fscience.aas8836&amp;volume=360&amp;pages=444-448&amp;publication_year=2018&amp;author=Myhrvold%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Arizti-Sanz, J. et al. Streamlined inactivation, amplification, and Cas13-based detection of SARS-CoV-2. <i>Nat. Commun.</i> <b>11</b>, 5921 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-19097-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-19097-x" data-track-item_id="10.1038/s41467-020-19097-x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisVOgur%2FJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33219225" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7680145" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Streamlined%20inactivation%2C%20amplification%2C%20and%20Cas13-based%20detection%20of%20SARS-CoV-2&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-19097-x&amp;volume=11&amp;publication_year=2020&amp;author=Arizti-Sanz%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Broughton, J. P. et al. CRISPR–Cas12-based detection of SARS-CoV-2. <i>Nat. Biotechnol.</i> <b>38</b>, 870–874 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41587-020-0513-4" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41587-020-0513-4" data-track-item_id="10.1038/s41587-020-0513-4">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXntlejt7w%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32300245" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC9107629" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%E2%80%93Cas12-based%20detection%20of%20SARS-CoV-2&amp;journal=Nat.%20Biotechnol.&amp;doi=10.1038%2Fs41587-020-0513-4&amp;volume=38&amp;pages=870-874&amp;publication_year=2020&amp;author=Broughton%2CJP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">Patchsung, M. et al. Clinical validation of a Cas13-based assay for the detection of SARS-CoV-2 RNA. <i>Nat. Biomed. Eng.</i> <b>4</b>, 1140–1149 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41551-020-00603-x" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41551-020-00603-x" data-track-item_id="10.1038/s41551-020-00603-x">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs12jsbzO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32848209" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clinical%20validation%20of%20a%20Cas13-based%20assay%20for%20the%20detection%20of%20SARS-CoV-2%20RNA&amp;journal=Nat.%20Biomed.%20Eng.&amp;doi=10.1038%2Fs41551-020-00603-x&amp;volume=4&amp;pages=1140-1149&amp;publication_year=2020&amp;author=Patchsung%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">Liu, T. Y. et al. Accelerated RNA detection using tandem CRISPR nucleases. <i>Nat. Chem. Biol.</i> <b>17</b>, 982–988 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41589-021-00842-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41589-021-00842-2" data-track-item_id="10.1038/s41589-021-00842-2">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhslart7fK" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34354262" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC10184463" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Accelerated%20RNA%20detection%20using%20tandem%20CRISPR%20nucleases&amp;journal=Nat.%20Chem.%20Biol.&amp;doi=10.1038%2Fs41589-021-00842-2&amp;volume=17&amp;pages=982-988&amp;publication_year=2021&amp;author=Liu%2CTY" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">Metsky, H. C. et al. Designing sensitive viral diagnostics with machine learning. <i>Nat. Biotech.</i> <a href="https://doi.org/10.1038/s41587-022-01213-5" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1038/s41587-022-01213-5">https://doi.org/10.1038/s41587-022-01213-5</a> (2022).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41587-022-01213-5" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41587-022-01213-5" data-track-item_id="10.1038/s41587-022-01213-5">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Designing%20sensitive%20viral%20diagnostics%20with%20machine%20learning&amp;journal=Nat.%20Biotech.&amp;doi=10.1038%2Fs41587-022-01213-5&amp;publication_year=2022&amp;author=Metsky%2CHC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Joung, J. et al. Detection of SARS-CoV-2 with SHERLOCK one-pot testing. <i>N. Engl. J. Med.</i> <b>383</b>, 1492–1494 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMc2026172" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMc2026172" data-track-item_id="10.1056/NEJMc2026172">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitVCgtrnF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32937062" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Detection%20of%20SARS-CoV-2%20with%20SHERLOCK%20one-pot%20testing&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMc2026172&amp;volume=383&amp;pages=1492-1494&amp;publication_year=2020&amp;author=Joung%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Bruch, R., Urban, G. A. &amp; Dincer, C. CRISPR/Cas powered multiplexed biosensing. <i>Trends Biotechnol.</i> <b>37</b>, 791–792 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.tibtech.2019.04.005" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.tibtech.2019.04.005" data-track-item_id="10.1016/j.tibtech.2019.04.005">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXosFWjt7s%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31078316" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR%2FCas%20powered%20multiplexed%20biosensing&amp;journal=Trends%20Biotechnol.&amp;doi=10.1016%2Fj.tibtech.2019.04.005&amp;volume=37&amp;pages=791-792&amp;publication_year=2019&amp;author=Bruch%2CR&amp;author=Urban%2CGA&amp;author=Dincer%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Tian, T., Qiu, Z., Jiang, Y., Zhu, D. &amp; Zhou, X. Exploiting the orthogonal CRISPR-Cas12a/Cas13a <i>trans</i>-cleavage for dual-gene virus detection using a handheld device. <i>Biosens. Bioelectron.</i> <b>196</b>, 113701 (2022).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.bios.2021.113701" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.bios.2021.113701" data-track-item_id="10.1016/j.bios.2021.113701">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXit1aqtLbL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34653714" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Exploiting%20the%20orthogonal%20CRISPR-Cas12a%2FCas13a%20trans-cleavage%20for%20dual-gene%20virus%20detection%20using%20a%20handheld%20device&amp;journal=Biosens.%20Bioelectron.&amp;doi=10.1016%2Fj.bios.2021.113701&amp;volume=196&amp;publication_year=2022&amp;author=Tian%2CT&amp;author=Qiu%2CZ&amp;author=Jiang%2CY&amp;author=Zhu%2CD&amp;author=Zhou%2CX" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">Bruch, R. et al. CRISPR-powered electrochemical microfluidic multiplexed biosensor for target amplification-free miRNA diagnostics. <i>Biosens. Bioelectron.</i> <b>177</b>, 112887 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.bios.2020.112887" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.bios.2020.112887" data-track-item_id="10.1016/j.bios.2020.112887">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXitFWmuro%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33493854" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=CRISPR-powered%20electrochemical%20microfluidic%20multiplexed%20biosensor%20for%20target%20amplification-free%20miRNA%20diagnostics&amp;journal=Biosens.%20Bioelectron.&amp;doi=10.1016%2Fj.bios.2020.112887&amp;volume=177&amp;publication_year=2021&amp;author=Bruch%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">Ackerman, C. M. et al. Massively multiplexed nucleic acid detection with Cas13. <i>Nature</i> <b>582</b>, 277–282 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-020-2279-8" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-020-2279-8" data-track-item_id="10.1038/s41586-020-2279-8">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVantLfE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32349121" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7332423" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Massively%20multiplexed%20nucleic%20acid%20detection%20with%20Cas13&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2279-8&amp;volume=582&amp;pages=277-282&amp;publication_year=2020&amp;author=Ackerman%2CCM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">Crowe, J. E. Jr. Human respiratory viruses. Reference Module in Biomedical Sciences <a href="https://doi.org/10.1016/B978-0-12-801238-3.02600-3" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1016/B978-0-12-801238-3.02600-3">https://doi.org/10.1016/B978-0-12-801238-3.02600-3</a> (2014).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">García Fernández, X., Álvarez-Argüelles, M. E., Rojo, S. &amp; de-Oña, M. Stability of viral RNA in clinical specimens for viral diagnosis. <i>Enferm. Infecc. Microbiol. Clin. (Engl. Ed.)</i> <b>38</b>, 297–298 (2020).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">Palmenberg, A. C. et al. Sequencing and analyses of all known human rhinovirus genomes reveal structure and evolution. <i>Science</i> <b>324</b>, 55–59 (2009).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.1165557" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.1165557" data-track-item_id="10.1126/science.1165557">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXjvVajtr4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19213880" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3923423" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Sequencing%20and%20analyses%20of%20all%20known%20human%20rhinovirus%20genomes%20reveal%20structure%20and%20evolution&amp;journal=Science&amp;doi=10.1126%2Fscience.1165557&amp;volume=324&amp;pages=55-59&amp;publication_year=2009&amp;author=Palmenberg%2CAC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">Mackay, I. M., Arden, K. E. &amp; Nitsche, A. Real-time PCR in virology. <i>Nucleic Acids Res.</i> <b>30</b>, 1292–1305 (2002).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2F30.6.1292" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/30.6.1292" data-track-item_id="10.1093/nar/30.6.1292">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD38XisVaksbY%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=11884626" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC101343" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Real-time%20PCR%20in%20virology&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2F30.6.1292&amp;volume=30&amp;pages=1292-1305&amp;publication_year=2002&amp;author=Mackay%2CIM&amp;author=Arden%2CKE&amp;author=Nitsche%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">Pulliam, J. R. C. et al. Increased risk of SARS-CoV-2 reinfection associated with emergence of the Omicron variant in South Africa. Preprint at <i>medRxiv</i> <a href="https://doi.org/10.1101/2021.11.11.21266068" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.11.11.21266068">https://doi.org/10.1101/2021.11.11.21266068</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR46">Brandal, L. T. et al. Outbreak caused by the SARS-CoV-2 Omicron variant in Norway, November to December 2021. <i>Euro. Surveill.</i> <b>26</b>, 2101147 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.2807%2F1560-7917.ES.2021.26.50.2101147" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.2807/1560-7917.ES.2021.26.50.2101147" data-track-item_id="10.2807/1560-7917.ES.2021.26.50.2101147">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XkvVOnuw%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34915975" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8728491" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Outbreak%20caused%20by%20the%20SARS-CoV-2%20Omicron%20variant%20in%20Norway%2C%20November%20to%20December%202021&amp;journal=Euro.%20Surveill.&amp;doi=10.2807%2F1560-7917.ES.2021.26.50.2101147&amp;volume=26&amp;publication_year=2021&amp;author=Brandal%2CLT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR47">CDC COVID-19 Response Team SARS-CoV-2 B.1.1.529 (Omicron) Variant—United States, December 1–8, 2021. <i>MMWR Morb. Mortal. Wkly. Rep.</i> <b>70</b>, 1731–1734 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.15585%2Fmmwr.mm7050e1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.15585/mmwr.mm7050e1" data-track-item_id="10.15585/mmwr.mm7050e1">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8675659" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SARS-CoV-2%20B.1.1.529%20%28Omicron%29%20Variant%E2%80%94United%20States%2C%20December%201%E2%80%938%2C%202021&amp;journal=MMWR%20Morb.%20Mortal.%20Wkly.%20Rep.&amp;doi=10.15585%2Fmmwr.mm7050e1&amp;volume=70&amp;pages=1731-1734&amp;publication_year=2021" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR48">Li, A., Maier, A., Carter, M. &amp; Guan, T. H. Omicron and S-gene target failure cases in the highest COVID-19 case rate region in Canada—December 2021. <i>J. Med. Virol</i>. <a href="https://doi.org/10.1002/jmv.27562" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1002/jmv.27562">https://doi.org/10.1002/jmv.27562</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR49">Kirkland, P. D. &amp; Frost, M. J. The impact of viral transport media on PCR assay results for the detection of nucleic acid from SARS-CoV-2. <i>Pathology</i> <b>52</b>, 811–814 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.pathol.2020.09.013" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.pathol.2020.09.013" data-track-item_id="10.1016/j.pathol.2020.09.013">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitFCrsbfI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33250079" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20impact%20of%20viral%20transport%20media%20on%20PCR%20assay%20results%20for%20the%20detection%20of%20nucleic%20acid%20from%20SARS-CoV-2&amp;journal=Pathology&amp;doi=10.1016%2Fj.pathol.2020.09.013&amp;volume=52&amp;pages=811-814&amp;publication_year=2020&amp;author=Kirkland%2CPD&amp;author=Frost%2CMJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR50">Fozouni, P. et al. Amplification-free detection of SARS-CoV-2 with CRISPR–Cas13a and mobile phone microscopy. <i>Cell</i> <b>184</b>, 323–333.e9 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cell.2020.12.001" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cell.2020.12.001" data-track-item_id="10.1016/j.cell.2020.12.001">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXisF2gsbnE" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=33306959" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Amplification-free%20detection%20of%20SARS-CoV-2%20with%20CRISPR%E2%80%93Cas13a%20and%20mobile%20phone%20microscopy&amp;journal=Cell&amp;doi=10.1016%2Fj.cell.2020.12.001&amp;volume=184&amp;pages=323-333.e9&amp;publication_year=2021&amp;author=Fozouni%2CP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR51">Nalefski, E. A. et al. Kinetic analysis of Cas12a and Cas13a RNA-guided nucleases for development of improved CRISPR-based diagnostics. <i>iScience</i> <b>24</b>, 102996 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.isci.2021.102996" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.isci.2021.102996" data-track-item_id="10.1016/j.isci.2021.102996">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38XitFajtLk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34505008" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8411246" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Kinetic%20analysis%20of%20Cas12a%20and%20Cas13a%20RNA-guided%20nucleases%20for%20development%20of%20improved%20CRISPR-based%20diagnostics&amp;journal=iScience&amp;doi=10.1016%2Fj.isci.2021.102996&amp;volume=24&amp;publication_year=2021&amp;author=Nalefski%2CEA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR52">Thakku, S. G. et al. Multiplexed detection of bacterial nucleic acids using Cas13 in droplet microarrays. Preprint at <i>bioRxiv</i> <a href="https://doi.org/10.1101/2021.11.12.468388" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1101/2021.11.12.468388">https://doi.org/10.1101/2021.11.12.468388</a> (2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR53">Barnes, K. G. et al. Deployable CRISPR-Cas13a diagnostic tools to detect and report Ebola and Lassa virus cases in real-time. <i>Nat. Commun.</i> <b>11</b>, 4131 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41467-020-17994-9" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41467-020-17994-9" data-track-item_id="10.1038/s41467-020-17994-9">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhs1Crt7%2FP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32807807" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7431545" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Deployable%20CRISPR-Cas13a%20diagnostic%20tools%20to%20detect%20and%20report%20Ebola%20and%20Lassa%20virus%20cases%20in%20real-time&amp;journal=Nat.%20Commun.&amp;doi=10.1038%2Fs41467-020-17994-9&amp;volume=11&amp;publication_year=2020&amp;author=Barnes%2CKG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR54">Katoh, K. &amp; Standley, D. M. MAFFT multiple sequence alignment software version 7: improvements in performance and usability. <i>Mol. Biol. Evol.</i> <b>30</b>, 772–780 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fmolbev%2Fmst010" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/molbev/mst010" data-track-item_id="10.1093/molbev/mst010">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3sXksFWisLc%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23329690" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3603318" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=MAFFT%20multiple%20sequence%20alignment%20software%20version%207%3A%20improvements%20in%20performance%20and%20usability&amp;journal=Mol.%20Biol.%20Evol.&amp;doi=10.1093%2Fmolbev%2Fmst010&amp;volume=30&amp;pages=772-780&amp;publication_year=2013&amp;author=Katoh%2CK&amp;author=Standley%2CDM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR55">Matranga, C. B. et al. Enhanced methods for unbiased deep sequencing of Lassa and Ebola RNA viruses from clinical and biological samples. <i>Genome Biol.</i> <b>15</b>, 519 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13059-014-0519-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13059-014-0519-7" data-track-item_id="10.1186/s13059-014-0519-7">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25403361" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4262991" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Enhanced%20methods%20for%20unbiased%20deep%20sequencing%20of%20Lassa%20and%20Ebola%20RNA%20viruses%20from%20clinical%20and%20biological%20samples&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fs13059-014-0519-7&amp;volume=15&amp;publication_year=2014&amp;author=Matranga%2CCB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li></ol></div></div>
        </div>
    </div>
</aside>
</div>


    
        
    
    
        
            
        

        
            
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>




<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Medicine
                    </span>
                    (<i itemprop="alternateName">Nat Med</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-170X</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1078-8956</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/" data-track="click" data-track-action="protocols.io" data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/authors/editorial_policies" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nature-research/open-access" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/language-editing/" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/" data-track="click" data-track-action="research solutions" data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library" data-track="click" data-track-action="Recommend to library" data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica" data-track="click" data-track-action="nature africa" data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly" data-track="click" data-track-action="nature Italy" data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies" data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice" data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/accessibility-statement" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20">
    </a>
    <p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"></path></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"></path><clipPath id="b"><use xlink:href="#a" style="overflow:visible"></use></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"></path><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"></path><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"></path><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"></path></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"></path></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>




    


    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif c-site-messages--nature-briefing-transres" data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: TranslationalResearch">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: TranslationalResearch">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing: Translational Research" src="/static/images/logos/nature-briefing-logo-transres-white-1245a3c374.svg" width="213" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing: Translational Research</em> newsletter — top stories in biotechnology, drug discovery and pharma.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/translational_research" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: TranslationalResearch">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="TransResBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="TransResBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required="" data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:translational_research" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required="">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: TranslationalResearch">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get what matters in translational research, free to your inbox weekly.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="/briefing/translational-research/?brieferEntryPoint=TransResBriefingBanner">Sign up for Nature Briefing: Translational Research
            </a>
        </div>

    </div>

</div>

    




<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41591-022-01734-1&amp;format=js&amp;last_modified=2023-11-09" async=""></script>

<vgc-xfcmzyjyvrdy></vgc-xfcmzyjyvrdy><div data-cc-ghost="" style="height: 352px;"></div><script type="text/javascript" id="fetch-contextual-ads-campaign-data" charset="">(function(){var g=function(a){a=new CustomEvent("campaignDataLoaded",{detail:a});document.dispatchEvent(a)},b=google_tag_manager["rm"]["50443292"](18);if(b){var k=1500;b=google_tag_manager["rm"]["50443292"](19);b="link"===b?google_tag_manager["rm"]["50443292"](20):google_tag_manager["rm"]["50443292"](21);var m=function(a,c){var h=!1,l=setTimeout(function(){h=!0;c(null)},k),d=new XMLHttpRequest;d.onload=function(){var e=null,f=[];if(200===d.status){try{e=JSON.parse(d.responseText)}catch(p){e={}}f=e.campaigns||[]}h||(clearTimeout(l),c(f.length?f.join(","):null))};d.open("GET",
"/platform/contextual?doi\x3d"+a);d.send()},n=function(a,c){a&&-1===a.indexOf("not set")?m(a,c):setTimeout(c,1)};n(b,function(a){window.campaignsForContextualAds=a;g(a);window.dataLayer.push({content:{article:{campaignID:a}}})})}else window.campaignsForContextualAds="",g(null)})();</script>
<script type="text/javascript" id="gtm-setup-accessdetailsloaded-handler" charset="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail||{};var b={event:"update-access-details"};a.institutional_business_partner_ids&&a.resolved_by||console.log("BPID data could not be retrieved from /exposed-details");a.logged_in_to_sn_profile||console.log("logged_in_to_sn_profile could not be retrieved from /exposed-details");var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?
a.resolved_by.join(";"):"",e=a.personal_business_partner_id_found,f=a.logged_in_to_sn_profile,g=a.snid;b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null,personal:e||null,logged_in_to_sn_profile:f,snid:g||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy={};window.dataLayer.push(b);window.idpUserDataLoaded=!0},!1);window.dataLayer.push({event:"accessdetailsloaded-handler-added"});
window.accessDetailsLoadedHandlerAdded=!0;</script><script type="text/javascript" id="gtm-mediatrust" charset="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="crossmark-script" text="" charset="" type="text/javascript" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="polyfill-matches" charset="">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="fetch-idp-user-data" charset="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},site=google_tag_manager["rm"]["50443292"](59),idpUrl;
switch(site){case "nature":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?"https://staging-idp.nature.com/exposed-details":"https://idp.nature.com/exposed-details";break;case "link":idpUrl=-1<window.location.hostname.indexOf("link-qa")?"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details";break;case "springer":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?
"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details"}if(void 0!==idpUrl){var transport=new XMLHttpRequest;transport.open("GET",idpUrl,!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script src="https://verify.nature.com/verify/nature.min.js"></script><script type="text/javascript" id="create-ad-slots" charset="">function createAndLoadAds(){function h(b){"nature"===a&&google_tag_manager["rm"]["50443292"](87)(b);"bmc"!==a&&"springeropen"!==a||google_tag_manager["rm"]["50443292"](99)(b);("link"===a&&"oscar"===google_tag_manager["rm"]["50443292"](100)||"springer"===a||"link"===a&&!0===google_tag_manager["rm"]["50443292"](101))&&google_tag_manager["rm"]["50443292"](126)(b);"link"===a&&"bunsen"===google_tag_manager["rm"]["50443292"](127)&&google_tag_manager["rm"]["50443292"](152)(b);"link"===a&&"Core"===google_tag_manager["rm"]["50443292"](153)&&google_tag_manager["rm"]["50443292"](165)(b)}window.googletag=window.googletag||{cmd:[]};var a=google_tag_manager["rm"]["50443292"](166),e=google_tag_manager["rm"]["50443292"](168);
if(-1===window.location.search.indexOf("hide_ads\x3dtrue"))if(window.adSlots&&0!==Object.keys(window.adSlots).length)console.log("Ads previously loaded. Will not update ad slots until next page load.");else{window.adSlots||(window.adSlots={});window.getAd=function(b,f){for(var c in window.adSlots)if(-1<c.indexOf(b)){if("object"===f)return adSlots[c];if("slot"===f)return adSlots[c].slot}};googletag.cmd.push(function(){googletag.pubads().setPrivacySettings({limitedAds:e});console.log("limitedAds is ",
e);googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["rm"]["50443292"](170));googletag.pubads().enableSingleRequest();googletag.pubads().disableInitialLoad()});for(var g=document.querySelectorAll("[data-gpt]"),d=0;g[d];++d)h(g[d]);googletag.cmd.push(function(){googletag.enableServices()})}}createAndLoadAds();</script>
</body></html>