<html lang="en" class="js"><head><link rel="preload" href="/static/fonts/HardingText-Regular-Web-cecd90984f.woff2" as="font" type="font/woff2" crossorigin=""><script async="" src="//cdn.jsdelivr.net/npm/mathjax@2.7.5/MathJax.js?config=TeX-AMS-MML_SVG.js"></script><script async="" src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2"></script><script type="text/javascript" async="" src="https://pagead2.googlesyndication.com/tag/js/gpt.js"></script><script async="" src="https://www.googletagmanager.com/gtm.js?id=GTM-MRVXSHQ"></script><script type="text/javascript" src="https://www.nature.com/driXi8DSS19Hxd_tj6F96VgjLDhlsWIlr7RUT5z7GHg4eduei35QY2r_84vWPPOF3XtZRxe_FYeaX52uULUIww=="></script><script src="https://cmp.nature.com/production_live/en/consent-bundle-8-85.js" onload="initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')"></script>
    <title>Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects | Nature Medicine</title>
    
        
<link rel="alternate" type="application/rss+xml" href="https://www.nature.com/nm.rss">


    
        

        <script id="save-data-connection-testing">
            function hasConnection() {
                return navigator.connection || navigator.mozConnection || navigator.webkitConnection || navigator.msConnection;
            }

            function createLink(src) {
                var preloadLink = document.createElement("link");
                preloadLink.rel = "preload";
                preloadLink.href = src;
                preloadLink.as = "font";
                preloadLink.type = "font/woff2";
                preloadLink.crossOrigin = "";
                document.head.insertBefore(preloadLink, document.head.firstChild);
            }

            var connectionDetail = {
                saveDataEnabled: false,
                slowConnection: false
            };

            var connection = hasConnection();
            if (connection) {
                connectionDetail.saveDataEnabled = connection.saveData;
                if (/\slow-2g|2g/.test(connection.effectiveType)) {
                    connectionDetail.slowConnection = true;
                }
            }

            if (!(connectionDetail.saveDataEnabled || connectionDetail.slowConnection)) {
                createLink("/static/fonts/HardingText-Regular-Web-cecd90984f.woff2");
            } else {
                document.documentElement.classList.add('save-data');
            }
        </script>
    

<link rel="preconnect" href="https://cmp.nature.com" crossorigin="">

<meta http-equiv="X-UA-Compatible" content="IE=edge">
<meta name="applicable-device" content="pc,mobile">
<meta name="viewport" content="width=device-width,initial-scale=1.0,maximum-scale=5,user-scalable=yes">
<meta name="360-site-verification" content="5a2dc4ab3fcb9b0393241ffbbb490480">

<script data-test="dataLayer">
    window.dataLayer = [{"content":{"category":{"contentType":"article","legacy":{"webtrendsPrimaryArticleType":"research","webtrendsSubjectTerms":"cancer-genomics;cancer-stem-cells;haematological-cancer;haematopoietic-stem-cells;risk-factors","webtrendsContentCategory":null,"webtrendsContentCollection":null,"webtrendsContentGroup":"Nature Medicine","webtrendsContentGroupType":null,"webtrendsContentSubGroup":"Article","status":null}},"article":{"doi":"10.1038/s41591-022-01883-3"},"attributes":{"cms":null,"deliveryPlatform":"oscar","copyright":{"open":true,"legacy":{"webtrendsLicenceType":"http://creativecommons.org/licenses/by/4.0/"}}},"contentInfo":{"authors":["Neil A. Robertson","Eric Latorre-Crespo","Maria Terradas-Terradas","Jorge Lemos-Portela","Alison C. Purcell","Benjamin J. Livesey","Robert F. Hillary","Lee Murphy","Angie Fawkes","Louise MacGillivray","Mhairi Copland","Riccardo E. Marioni","Joseph A. Marsh","Sarah E. Harris","Simon R. Cox","Ian J. Deary","Linus J. Schumacher","Kristina Kirschner","Tamir Chandra"],"publishedAt":1656892800,"publishedAtString":"2022-07-04","title":"Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects","legacy":null,"publishedAtTime":null,"documentType":"aplusplus","subjects":"Cancer genomics,Cancer stem cells,Haematological cancer,Haematopoietic stem cells,Risk factors"},"journal":{"pcode":"nm","title":"nature medicine","volume":"28","issue":"7","id":41591,"publishingModel":"Hybrid Access"},"authorization":{"status":true},"features":[{"name":"furtherReadingSection","present":true}],"collection":null},"page":{"category":{"pageType":"article"},"attributes":{"template":"mosaic","featureFlags":[{"name":"nature-onwards-journey","active":false}],"testGroup":null},"search":null},"privacy":{},"version":"1.0.0","product":null,"session":null,"user":null,"backHalfContent":true,"country":"BR","hasBody":true,"uneditedManuscript":false,"twitterId":["o3xnx","o43y9","o3ef7"],"baiduId":"d38bce82bcb44717ccc29a90c4b781ea","japan":false}];
    window.dataLayer.push({
        ga4MeasurementId: 'G-ERRNTNZ807',
        ga360TrackingId: 'UA-71668177-1',
        twitterId: ['3xnx', 'o43y9', 'o3ef7'],
        baiduId: 'd38bce82bcb44717ccc29a90c4b781ea',
        ga4ServerUrl: 'https://collect.nature.com',
        imprint: 'nature'
    });
</script>

<script>
    (function(w, d) {
        w.config = w.config || {};
        w.config.mustardcut = false;

        
        if (w.matchMedia && w.matchMedia('only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)').matches) {
            w.config.mustardcut = true;
            d.classList.add('js');
            d.classList.remove('grade-c');
            d.classList.remove('no-js');
        }
    })(window, document.documentElement);
</script>
 



     
    
    
        
    
    <style>@media only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark) {  .c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card--major .c-card__title,.c-card__title,.u-h2,.u-h3,h2,h3{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,.u-h3,h3{font-size:1.25rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}html{line-height:1.15;text-size-adjust:100%;box-sizing:border-box;font-size:100%;height:100%;overflow-y:scroll}body{background:#eee;color:#222;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;line-height:1.76;margin:0;min-height:100%}details,main{display:block}h1{font-size:2em;margin:.67em 0}a,sup{vertical-align:baseline}a{background-color:transparent;color:#069;overflow-wrap:break-word;text-decoration:underline;text-decoration-skip-ink:auto;word-break:break-word}b{font-weight:bolder}sup{font-size:75%;line-height:0;position:relative;top:-.5em}img{border:0;height:auto;max-width:100%;vertical-align:middle}button,input,select{font-family:inherit;font-size:100%;line-height:1.15;margin:0}button,input{overflow:visible}button,select{text-transform:none}[type=submit],button{-webkit-appearance:button}[type=checkbox]{box-sizing:border-box;padding:0}summary{display:list-item}[hidden]{display:none}button{border-radius:0;cursor:pointer;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}h1{font-weight:700;-webkit-font-smoothing:antialiased;font-family:Harding,Palatino,serif;font-size:2rem;letter-spacing:-.0390625rem;line-height:2.25rem}.c-card--major .c-card__title,.u-h2,.u-h3,h2{font-family:Harding,Palatino,serif;letter-spacing:-.0117156rem}.c-card--major .c-card__title,.u-h2,h2{font-weight:700;-webkit-font-smoothing:antialiased;font-size:1.5rem;line-height:1.6rem}.u-h3{font-size:1.25rem}.c-card__title,.c-reading-companion__figure-title,.u-h3,.u-h4,h4,h5,h6{font-weight:700;-webkit-font-smoothing:antialiased;line-height:1.4rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,.c-card__title,h3{font-family:Harding,Palatino,serif;font-size:1.25rem}.c-article-editorial-summary__container .c-article-editorial-summary__article-title,h3{font-weight:700;-webkit-font-smoothing:antialiased;letter-spacing:-.0117156rem;line-height:1.4rem}.c-reading-companion__figure-title,.u-h4,h4{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.125rem;letter-spacing:-.0117156rem}input+label{padding-left:.5em}nav ol,nav ul{list-style:none none}p:empty{display:none}.sans-serif{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.article-page{background:#fff}.c-article-header{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;margin-bottom:40px}.c-article-identifiers{color:#6f6f6f;display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3;list-style:none;margin:0 0 8px;padding:0}.c-article-identifiers__item{border-right:1px solid #6f6f6f;list-style:none;margin-right:8px;padding-right:8px}.c-article-identifiers__item:last-child{border-right:0;margin-right:0;padding-right:0}.c-article-title{font-size:1.5rem;line-height:1.25;margin:0 0 16px}@media only screen and (min-width:768px){.c-article-title{font-size:1.875rem;line-height:1.2}}.c-article-author-list{display:inline;font-size:1rem;list-style:none;margin:0 8px 0 0;padding:0;width:100%}.c-article-author-list__item{display:inline;padding-right:0}.c-article-author-list svg{margin-left:4px}.c-article-author-list__show-more{display:none;margin-right:4px}.c-article-author-list__button,.js .c-article-author-list__item--hide,.js .c-article-author-list__show-more{display:none}.js .c-article-author-list--long .c-article-author-list__show-more,.js .c-article-author-list--long+.c-article-author-list__button{display:inline}@media only screen and (max-width:539px){.js .c-article-author-list__item--hide-small-screen{display:none}.js .c-article-author-list--short .c-article-author-list__show-more,.js .c-article-author-list--short+.c-article-author-list__button{display:inline}}#uptodate-client,.js .c-article-author-list--expanded .c-article-author-list__show-more{display:none!important}.js .c-article-author-list--expanded .c-article-author-list__item--hide-small-screen{display:inline!important}.c-article-author-list__button,.c-button-author-list{background:#ebf1f5;border:4px solid #ebf1f5;border-radius:20px;color:#666;font-size:.875rem;line-height:1.4;padding:2px 11px 2px 8px;text-decoration:none}.c-article-author-list__button svg,.c-button-author-list svg{margin:1px 4px 0 0}.c-article-author-list__button:hover,.c-button-author-list:hover{background:#069;border-color:transparent;color:#fff}.c-article-info-details{font-size:1rem;margin-bottom:8px;margin-top:16px}.c-article-info-details__cite-as{border-left:1px solid #6f6f6f;margin-left:8px;padding-left:8px}.c-article-metrics-bar{display:flex;flex-wrap:wrap;font-size:1rem;line-height:1.3}.c-article-metrics-bar__wrapper{margin:16px 0}.c-article-metrics-bar__item{align-items:baseline;border-right:1px solid #6f6f6f;margin-right:8px}.c-article-metrics-bar__item:last-child{border-right:0}.c-article-metrics-bar__count{font-weight:700;margin:0}.c-article-metrics-bar__label{color:#626262;font-style:normal;font-weight:400;margin:0 10px 0 5px}.c-article-metrics-bar__details{margin:0}.c-article-main-column{font-family:Harding,Palatino,serif;margin-right:8.6%;width:60.2%}@media only screen and (max-width:1023px){.c-article-main-column{margin-right:0;width:100%}}.c-article-extras{float:left;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;width:31.2%}@media only screen and (max-width:1023px){.c-article-extras{display:none}}.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{border-bottom:2px solid #d5d5d5;font-size:1.25rem;margin:0;padding-bottom:8px}@media only screen and (min-width:768px){.c-article-associated-content__container .c-article-associated-content__title,.c-article-section__title{font-size:1.5rem;line-height:1.24}}.c-article-associated-content__container .c-article-associated-content__title{margin-bottom:8px}.c-article-body p{margin-bottom:24px;margin-top:0}.c-article-section{clear:both}.c-article-section__content{margin-bottom:40px;padding-top:8px}@media only screen and (max-width:1023px){.c-article-section__content{padding-left:0}}.c-article-authors-search{margin-bottom:24px;margin-top:0}.c-article-authors-search__item,.c-article-authors-search__title{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif}.c-article-authors-search__title{color:#626262;font-size:1.05rem;font-weight:700;margin:0;padding:0}.c-article-authors-search__item{font-size:1rem}.c-article-authors-search__text{margin:0}.c-code-block{border:1px solid #eee;font-family:monospace;margin:0 0 24px;padding:20px}.c-code-block__heading{font-weight:400;margin-bottom:16px}.c-code-block__line{display:block;overflow-wrap:break-word;white-space:pre-wrap}.c-article-share-box__no-sharelink-info{font-size:.813rem;font-weight:700;margin-bottom:24px;padding-top:4px}.c-article-share-box__only-read-input{border:1px solid #d5d5d5;box-sizing:content-box;display:inline-block;font-size:.875rem;font-weight:700;height:24px;margin-bottom:8px;padding:8px 10px}.c-article-share-box__button--link-like{background-color:transparent;border:0;color:#069;cursor:pointer;font-size:.875rem;margin-bottom:8px;margin-left:10px}.c-article-editorial-summary__container{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem}.c-article-editorial-summary__container .c-article-editorial-summary__content p:last-child{margin-bottom:0}.c-article-editorial-summary__container .c-article-editorial-summary__content--less{max-height:9.5rem;overflow:hidden}.c-article-editorial-summary__container .c-article-editorial-summary__button{background-color:#fff;border:0;color:#069;font-size:.875rem;margin-bottom:16px}.c-article-editorial-summary__container .c-article-editorial-summary__button.active,.c-article-editorial-summary__container .c-article-editorial-summary__button.hover,.c-article-editorial-summary__container .c-article-editorial-summary__button:active,.c-article-editorial-summary__container .c-article-editorial-summary__button:hover{text-decoration:underline;text-decoration-skip-ink:auto}.c-article-associated-content__container .c-article-associated-content__collection-label{font-size:.875rem;line-height:1.4}.c-article-associated-content__container .c-article-associated-content__collection-title{line-height:1.3}.c-context-bar{box-shadow:0 0 10px 0 rgba(51,51,51,.2);position:relative;width:100%}.c-context-bar__title{display:none}.c-reading-companion{clear:both;min-height:389px}.c-reading-companion__sticky{max-width:389px}.c-reading-companion__scroll-pane{margin:0;min-height:200px;overflow:hidden auto}.c-reading-companion__tabs{display:flex;flex-flow:row nowrap;font-size:1rem;list-style:none;margin:0 0 8px;padding:0}.c-reading-companion__tabs>li{flex-grow:1}.c-reading-companion__tab{background-color:#eee;border:1px solid #d5d5d5;border-image:initial;border-left-width:0;color:#069;font-size:1rem;padding:8px 8px 8px 15px;text-align:left;width:100%}.c-reading-companion__tabs li:first-child .c-reading-companion__tab{border-left-width:1px}.c-reading-companion__tab--active{background-color:#fff;border-bottom:1px solid #fff;color:#222;font-weight:700}.c-reading-companion__sections-list{list-style:none;padding:0}.c-reading-companion__figures-list,.c-reading-companion__references-list{list-style:none;min-height:389px;padding:0}.c-reading-companion__references-list--numeric{list-style:decimal inside}.c-reading-companion__sections-list{margin:0 0 8px;min-height:50px}.c-reading-companion__section-item{font-size:1rem;padding:0}.c-reading-companion__section-item a{display:block;line-height:1.5;overflow:hidden;padding:8px 0 8px 16px;text-overflow:ellipsis;white-space:nowrap}.c-reading-companion__figure-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:16px 8px 16px 0}.c-reading-companion__figure-item:first-child{border-top:none;padding-top:8px}.c-reading-companion__reference-item{border-top:1px solid #d5d5d5;font-size:1rem;padding:8px 8px 8px 16px}.c-reading-companion__reference-item:first-child{border-top:none}.c-reading-companion__reference-item a{word-break:break-word}.c-reading-companion__reference-citation{display:inline}.c-reading-companion__reference-links{font-size:.813rem;font-weight:700;list-style:none;margin:8px 0 0;padding:0;text-align:right}.c-reading-companion__reference-links>a{display:inline-block;padding-left:8px}.c-reading-companion__reference-links>a:first-child{display:inline-block;padding-left:0}.c-reading-companion__figure-title{display:block;margin:0 0 8px}.c-reading-companion__figure-links{display:flex;justify-content:space-between;margin:8px 0 0}.c-reading-companion__figure-links>a{align-items:center;display:flex}.c-reading-companion__figure-full-link svg{height:.8em;margin-left:2px}.c-reading-companion__panel{border-top:none;display:none;margin-top:0;padding-top:0}.c-cod,.c-reading-companion__panel--active{display:block}.c-cod{font-size:1rem;width:100%}.c-cod__form{background:#ebf0f3}.c-cod__prompt{font-size:1.125rem;line-height:1.3;margin:0 0 24px}.c-cod__label{display:block;margin:0 0 4px}.c-cod__row{display:flex;margin:0 0 16px}.c-cod__row:last-child{margin:0}.c-cod__input{border:1px solid #d5d5d5;border-radius:2px;flex-basis:75%;flex-shrink:0;margin:0;padding:13px}.c-cod__input--submit{background-color:#069;border:1px solid #069;color:#fff;flex-shrink:1;margin-left:8px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-cod__input--submit-single{flex-basis:100%;flex-shrink:0;margin:0}.c-cod__input--submit:focus,.c-cod__input--submit:hover{background-color:#fff;color:#069}.c-pdf-download__link .u-icon{padding-top:2px}.c-pdf-download{display:flex;margin-bottom:16px;max-height:48px}@media only screen and (min-width:540px){.c-pdf-download{max-height:none}}@media only screen and (min-width:1024px){.c-pdf-download{max-height:48px}}.c-pdf-download__link{display:flex;flex:1 1 0%}.c-pdf-download__link:hover{text-decoration:none}.c-pdf-download__text{padding-right:4px}@media only screen and (max-width:539px){.c-pdf-download__text{text-transform:capitalize}}@media only screen and (min-width:540px){.c-pdf-download__text{padding-right:8px}}.c-context-bar--sticky .c-pdf-download{display:block;margin-bottom:0;white-space:nowrap}@media only screen and (max-width:539px){.c-pdf-download .u-sticky-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}}.c-pdf-container{display:flex;justify-content:flex-end}@media only screen and (max-width:539px){.c-pdf-container .c-pdf-download{display:flex;flex-basis:100%}}.c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:16px}.c-article-extras .c-pdf-container .c-pdf-download{width:100%}.c-article-extras .c-pdf-container .c-pdf-download+.c-pdf-download{margin-left:0}@media only screen and (min-width:540px){.c-context-bar--sticky .c-pdf-download__link{align-items:center;flex:1 1 183px}}@media only screen and (max-width:320px){.c-context-bar--sticky .c-pdf-download__link{padding:16px}}.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:none}@media only screen and (max-width:1023px){.article-page--commercial .c-article-main-column .c-pdf-button__container .c-pdf-download{display:block}}.c-recommendations-column-switch .c-meta{margin-top:auto}.c-article-recommendations-card__meta-type,.c-meta .c-meta__item:first-child{font-weight:700}.c-article-metrics__heading a,.c-article-metrics__posts .c-card__title a{color:inherit}.c-article-metrics__posts .c-card__title{font-size:1.05rem}.c-article-metrics__posts .c-card__title+span{color:#6f6f6f;font-size:1rem}p{overflow-wrap:break-word;word-break:break-word}.c-ad{text-align:center}@media only screen and (min-width:320px){.c-ad{padding:8px}}.c-ad--728x90{background-color:#ccc;display:none}.c-ad--728x90 .c-ad__inner{min-height:calc(1.5em + 94px)}@media only screen and (min-width:768px){.js .c-ad--728x90{display:none}.js .u-show-following-ad+.c-ad--728x90{display:block}}.c-ad__label{color:#333;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;font-weight:400;line-height:1.5;margin-bottom:4px}.app-author-list{color:#6f6f6f;font-family:inherit;font-size:1rem;line-height:inherit;list-style:none;margin:0;padding:0}.app-author-list>li,.c-breadcrumbs>li,.c-footer__links>li,.js .app-author-list,.u-list-comma-separated>li,.u-list-inline>li{display:inline}.app-author-list>li:not(:first-child):not(:last-child):before{content:", "}.app-author-list>li:not(:only-child):last-child:before{content:" & "}.app-author-list--compact{font-size:.875rem;line-height:1.4}.app-author-list--truncated>li:not(:only-child):last-child:before{content:" ... "}.js .app-author-list__hide{display:none;visibility:hidden}.js .app-author-list__hide:first-child+*{margin-block-start:0}.c-meta{color:inherit;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;list-style:none;margin:0;padding:0}.c-meta--large{font-size:1rem}.c-meta--large .c-meta__item{margin-bottom:8px}.c-meta__item{display:inline-block;margin-bottom:4px}.c-meta__item:not(:last-child){border-right:1px solid #d5d5d5;margin-right:4px;padding-right:4px}@media only screen and (max-width:539px){.c-meta__item--block-sm-max{display:block}.c-meta__item--block-sm-max:not(:last-child){border-right:none;margin-right:0;padding-right:0}}@media only screen and (min-width:1024px){.c-meta__item--block-at-lg{display:block}.c-meta__item--block-at-lg:not(:last-child){border-right:none;margin-right:0;padding-right:0}}.c-meta__type{font-weight:700;text-transform:none}.c-skip-link{background:#069;bottom:auto;color:#fff;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;padding:8px;position:absolute;text-align:center;transform:translateY(-100%);z-index:9999}@media (prefers-reduced-motion:reduce){.c-skip-link{transition:top .3s ease-in-out 0s}}@media print{.c-skip-link{display:none}}.c-skip-link:link{color:#fff}.c-status-message{align-items:center;box-sizing:border-box;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;position:relative;width:100%}.c-card__summary>p:last-child,.c-status-message :last-child{margin-bottom:0}.c-status-message--boxed{background-color:#fff;border:1px solid #eee;border-radius:2px;line-height:1.4;padding:16px}.c-status-message__heading{font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;font-weight:700}.c-status-message__icon{fill:currentcolor;display:inline-block;flex:0 0 auto;height:1.5em;margin-right:8px;transform:translate(0);vertical-align:text-top;width:1.5em}.c-status-message__icon--top{align-self:flex-start}.c-status-message--info .c-status-message__icon{color:#003f8d}.c-status-message--boxed.c-status-message--info{border-bottom:4px solid #003f8d}.c-status-message--error .c-status-message__icon{color:#c40606}.c-status-message--boxed.c-status-message--error{border-bottom:4px solid #c40606}.c-status-message--success .c-status-message__icon{color:#00b8b0}.c-status-message--boxed.c-status-message--success{border-bottom:4px solid #00b8b0}.c-status-message--warning .c-status-message__icon{color:#edbc53}.c-status-message--boxed.c-status-message--warning{border-bottom:4px solid #edbc53}.c-breadcrumbs{color:#000;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1rem;list-style:none;margin:0;padding:0}.c-breadcrumbs__link{color:#666}svg.c-breadcrumbs__chevron{margin:4px 4px 0;fill:#888;height:10px;width:10px}@media only screen and (max-width:539px){.c-breadcrumbs .c-breadcrumbs__item{display:none}.c-breadcrumbs .c-breadcrumbs__item:last-child,.c-breadcrumbs .c-breadcrumbs__item:nth-last-child(2){display:inline}}.c-card{background-color:transparent;border:0;box-shadow:none;display:flex;flex-direction:column;font-size:14px;min-width:0;overflow:hidden;padding:0;position:relative}.c-card--no-shape{background:0 0;border:0;box-shadow:none}.c-card__image{display:flex;justify-content:center;overflow:hidden;padding-bottom:56.25%;position:relative}@supports (aspect-ratio:1/1){.c-card__image{padding-bottom:0}}.c-card__image img{left:0;min-height:100%;min-width:100%;position:absolute}@supports ((-o-object-fit:cover) or (object-fit:cover)){.c-card__image img{height:100%;object-fit:cover;width:100%}}.c-card__body{flex:1 1 auto;padding:16px}.c-card--no-shape .c-card__body{padding:0}.c-card--no-shape .c-card__body:not(:first-child){padding-top:16px}.c-card__title{letter-spacing:-.01875rem;margin-bottom:8px;margin-top:0}[lang=de] .c-card__title{hyphens:auto}.c-card__summary{line-height:1.4}.c-card__summary>p{margin-bottom:5px}.c-card__summary a{text-decoration:underline}.c-card__link:not(.c-card__link--no-block-link):before{bottom:0;content:"";left:0;position:absolute;right:0;top:0}.c-card--flush .c-card__body{padding:0}.c-card--major{font-size:1rem}.c-card--dark{background-color:#29303c;border-width:0;color:#e3e4e5}.c-card--dark .c-card__title{color:#fff}.c-card--dark .c-card__link,.c-card--dark .c-card__summary a{color:inherit}.c-header{background-color:#fff;border-bottom:5px solid #000;font-size:1rem;line-height:1.4;margin-bottom:16px}.c-header__row{padding:0;position:relative}.c-header__row:not(:last-child){border-bottom:1px solid #eee}.c-header__split{align-items:center;display:flex;justify-content:space-between}.c-header__logo-container{flex:1 1 0px;line-height:0;margin:8px 24px 8px 0}.c-header__logo{transform:translateZ(0)}.c-header__logo img{max-height:32px}.c-header__container{margin:0 auto;max-width:1280px}.c-header__menu{align-items:center;display:flex;flex:0 1 auto;flex-wrap:wrap;font-weight:700;gap:8px 8px;line-height:1.4;list-style:none;margin:0 -8px;padding:0}@media print{.c-header__menu{display:none}}@media only screen and (max-width:1023px){.c-header__menu--hide-lg-max{display:none;visibility:hidden}}.c-header__menu--global{font-weight:400;justify-content:flex-end}.c-header__menu--global svg{display:none;visibility:hidden}.c-header__menu--global svg:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__menu--global svg{display:block;visibility:visible}}.c-header__menu--journal{font-size:.875rem;margin:8px 0 8px -8px}@media only screen and (min-width:540px){.c-header__menu--journal{flex-wrap:nowrap;font-size:1rem}}.c-header__item{padding-bottom:0;padding-top:0;position:static}.c-header__item--pipe{border-left:2px solid #eee;padding-left:8px}.c-header__item--padding{padding-bottom:8px;padding-top:8px}@media only screen and (min-width:540px){.c-header__item--dropdown-menu{position:relative}}@media only screen and (min-width:1024px){.c-header__item--hide-lg{display:none;visibility:hidden}}@media only screen and (max-width:767px){.c-header__item--hide-md-max{display:none;visibility:hidden}.c-header__item--hide-md-max:first-child+*{margin-block-start:0}}.c-header__link{align-items:center;color:inherit;display:inline-flex;gap:4px 4px;padding:8px;white-space:nowrap}.c-header__link svg{transition-duration:.2s}.c-header__show-text{display:none;visibility:hidden}.has-tethered .c-header__heading--js-hide:first-child+*{margin-block-start:0}@media only screen and (min-width:540px){.c-header__show-text{display:inline;visibility:visible}}.c-header__dropdown{background-color:#000;border-bottom:1px solid #2f2f2f;color:#eee;font-size:.875rem;line-height:1.2;padding:16px 0}@media print{.c-header__dropdown{display:none}}.c-header__heading{display:inline-block;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:1.25rem;font-weight:400;line-height:1.4;margin-bottom:8px}.c-header__heading--keyline{border-top:1px solid;border-color:#2f2f2f;margin-top:16px;padding-top:16px;width:100%}.c-header__list{display:flex;flex-wrap:wrap;gap:0 16px;list-style:none;margin:0 -8px}.c-header__flush{margin:0 -8px}.c-header__visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.c-header__search-form{margin-bottom:8px}.c-header__search-layout{display:flex;flex-wrap:wrap;gap:16px 16px}.c-header__search-layout>:first-child{flex:999 1 auto}.c-header__search-layout>*{flex:1 1 auto}.c-header__search-layout--max-width{max-width:720px}.c-header__search-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #fff;border-radius:2px;color:#fff;cursor:pointer;display:flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.15;margin:0;padding:8px 16px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:100%}.u-button svg,.u-button--primary svg{fill:currentcolor}.c-header__input,.c-header__select{border:1px solid;border-radius:3px;box-sizing:border-box;font-size:1rem;padding:8px 16px;width:100%}.c-header__select{-webkit-appearance:none;background-image:url("data:image/svg+xml,%3Csvg height='16' viewBox='0 0 16 16' width='16' xmlns='http://www.w3.org/2000/svg'%3E%3Cpath d='m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z' fill='%23333' fill-rule='evenodd' transform='matrix(0 1 -1 0 11 3)'/%3E%3C/svg%3E");background-position:right .7em top 50%;background-repeat:no-repeat;background-size:1em;box-shadow:0 1px 0 1px rgba(0,0,0,.04);display:block;margin:0;max-width:100%;min-width:150px}@media only screen and (min-width:540px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:auto;right:0}}@media only screen and (min-width:768px){.c-header__menu--journal .c-header__item--dropdown-menu:last-child .c-header__dropdown.has-tethered{left:0;right:auto}}.c-header__dropdown.has-tethered{border-bottom:0;border-radius:0 0 2px 2px;left:0;position:absolute;top:100%;transform:translateY(5px);width:100%;z-index:1}@media only screen and (min-width:540px){.c-header__dropdown.has-tethered{transform:translateY(8px);width:auto}}@media only screen and (min-width:768px){.c-header__dropdown.has-tethered{min-width:225px}}.c-header__dropdown--full-width.has-tethered{padding:32px 0 24px;transform:none;width:100%}.has-tethered .c-header__heading--js-hide{display:none;visibility:hidden}.has-tethered .c-header__list--js-stack{flex-direction:column}.has-tethered .c-header__item--keyline,.has-tethered .c-header__list~.c-header__list .c-header__item:first-child{border-top:1px solid #d5d5d5;margin-top:8px;padding-top:8px}.c-header__item--snid-account-widget{display:flex}.c-header__container{padding:0 4px}.c-header__list{padding:0 12px}.c-header__menu .c-header__link{font-size:14px}.c-header__item--snid-account-widget .c-header__link{padding:8px}.c-header__menu--journal{margin-left:0}@media only screen and (min-width:540px){.c-header__container{padding:0 16px}.c-header__menu--journal{margin-left:-8px}.c-header__menu .c-header__link{font-size:16px}.c-header__link--search{gap:13px 13px}}.u-button{align-items:center;background-color:transparent;background-image:none;border:1px solid #069;border-radius:2px;color:#069;cursor:pointer;display:inline-flex;font-family:sans-serif;font-size:1rem;justify-content:center;line-height:1.3;margin:0;padding:8px;position:relative;text-decoration:none;transition:all .25s ease 0s,color .25s ease 0s,border-color .25s ease 0s;width:auto}.u-button--primary{background-color:#069;background-image:none;border:1px solid #069;color:#fff}.u-button--full-width{display:flex;width:100%}.u-display-none{display:none}.js .u-js-hide,.u-hide{display:none;visibility:hidden}.u-hide:first-child+*{margin-block-start:0}.u-visually-hidden{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}@media print{.u-hide-print{display:none}}@media only screen and (min-width:1024px){.u-hide-at-lg{display:none;visibility:hidden}.u-hide-at-lg:first-child+*{margin-block-start:0}}.u-clearfix:after,.u-clearfix:before{content:"";display:table}.u-clearfix:after{clear:both}.u-color-open-access{color:#b74616}.u-float-left{float:left}.u-icon{fill:currentcolor;display:inline-block;height:1em;transform:translate(0);vertical-align:text-top;width:1em}.u-full-height{height:100%}.u-list-reset{list-style:none;margin:0;padding:0}.u-container{margin:0 auto;max-width:1280px;padding:0 16px}.u-justify-content-space-between{justify-content:space-between}.u-mt-32{margin-top:32px}.u-mb-8{margin-bottom:8px}.u-mb-16{margin-bottom:16px}.u-mb-24{margin-bottom:24px}.u-mb-32{margin-bottom:32px}.c-nature-box svg+.c-article__button-text,.u-ml-8{margin-left:8px}.u-pa-16{padding:16px}html *,html :after,html :before{box-sizing:inherit}.c-article-section__title,.c-article-title{font-weight:700}.c-card__title{line-height:1.4em}.c-article__button{background-color:#069;border:1px solid #069;border-radius:2px;color:#fff;display:flex;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif;font-size:.875rem;line-height:1.4;margin-bottom:16px;padding:13px;transition:background-color .2s ease-out 0s,color .2s ease-out 0s}.c-article__button,.c-article__button:hover{text-decoration:none}.c-article__button--inverted,.c-article__button:hover{background-color:#fff;color:#069}.c-article__button--inverted:hover{background-color:#069;color:#fff}.c-header__link{text-decoration:inherit}.grade-c-hide{display:block}.c-nature-box{background-color:#fff;border:1px solid #d5d5d5;border-radius:2px;box-shadow:0 0 5px 0 rgba(51,51,51,.1);line-height:1.3;margin-bottom:24px;padding:16px 16px 3px}.c-nature-box__text{font-size:1rem;margin-bottom:16px}.c-nature-box--access-to-pdf{display:none}@media only screen and (min-width:1024px){.c-nature-box--mobile{display:none}}.c-nature-box .c-pdf-download{margin-bottom:16px!important}.c-nature-box--version{background-color:#eee}.c-nature-box__wrapper{transform:translateZ(0)}.c-nature-box__wrapper--placeholder{min-height:165px}.c-pdf-download__link{padding:13px 24px} } </style>




    
        <link data-test="critical-css-handler" data-inline-css-source="critical-css" rel="stylesheet" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)" onload="this.media='only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)';this.onload=null">
    
    <noscript>
        <link rel="stylesheet" type="text/css" href="/static/css/enhanced-article-nature-branded-1dd982e2cc.css" media="only print, only all and (prefers-color-scheme: no-preference), only all and (prefers-color-scheme: light), only all and (prefers-color-scheme: dark)">
    </noscript>

<link rel="stylesheet" type="text/css" href="/static/css/article-print-c9121fecab.css" media="print">
    



<link rel="apple-touch-icon" sizes="180x180" href="/static/images/favicons/nature/apple-touch-icon-f39cb19454.png">
<link rel="icon" type="image/png" sizes="48x48" href="/static/images/favicons/nature/favicon-48x48-b52890008c.png">
<link rel="icon" type="image/png" sizes="32x32" href="/static/images/favicons/nature/favicon-32x32-3fe59ece92.png">
<link rel="icon" type="image/png" sizes="16x16" href="/static/images/favicons/nature/favicon-16x16-951651ab72.png">
<link rel="manifest" href="/static/manifest.json" crossorigin="use-credentials">
<link rel="mask-icon" href="/static/images/favicons/nature/safari-pinned-tab-69bff48fe6.svg" color="#000000">
<link rel="shortcut icon" href="/static/images/favicons/nature/favicon.ico">
<meta name="msapplication-TileColor" content="#000000">
<meta name="msapplication-config" content="/static/browserconfig.xml">
<meta name="theme-color" content="#000000">
<meta name="application-name" content="Nature">


<script>
    (function () {
        if ( typeof window.CustomEvent === "function" ) return false;
        function CustomEvent ( event, params ) {
            params = params || { bubbles: false, cancelable: false, detail: null };
            var evt = document.createEvent( 'CustomEvent' );
            evt.initCustomEvent( event, params.bubbles, params.cancelable, params.detail );
            return evt;
        }

        CustomEvent.prototype = window.Event.prototype;

        window.CustomEvent = CustomEvent;
    })();
</script>



<!-- Google Tag Manager -->
<script data-test="gtm-head">
    window.initGTM = function() {
        if (window.config.mustardcut) {
            (function (w, d, s, l, i) {
                w[l] = w[l] || [];
                w[l].push({'gtm.start': new Date().getTime(), event: 'gtm.js'});
                var f = d.getElementsByTagName(s)[0],
                        j = d.createElement(s),
                        dl = l != 'dataLayer' ? '&l=' + l : '';
                j.async = true;
                j.src = 'https://www.googletagmanager.com/gtm.js?id=' + i + dl;
                f.parentNode.insertBefore(j, f);
            })(window, document, 'script', 'dataLayer', 'GTM-MRVXSHQ');
        }
    }
</script>
<!-- End Google Tag Manager -->

    <script>
    (function(w,d,t) {
        function cc() {
            var h = w.location.hostname;
            if (h.indexOf('preview-www.nature.com') > -1) return;

            var e = d.createElement(t),
                    s = d.getElementsByTagName(t)[0];

            if (h.indexOf('nature.com') > -1) {
                if (h.indexOf('test-www.nature.com') > -1) {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                } else {
                    e.src = 'https://cmp.nature.com/production_live/en/consent-bundle-8-85.js';
                    e.setAttribute('onload', "initGTM(window,document,'script','dataLayer','GTM-MRVXSHQ')");
                }
            } else {
                e.src = '/static/js/cookie-consent-es5-bundle-cb57c2c98a.js';
                e.setAttribute('data-consent', h);
            }
            s.insertAdjacentElement('afterend', e);
        }

        cc();
    })(window,document,'script');
</script>


<script id="js-position0">
    (function(w, d) {
        w.idpVerifyPrefix = 'https://verify.nature.com';
        w.ra21Host = 'https://wayf.springernature.com';
        var moduleSupport = (function() {
            return 'noModule' in d.createElement('script');
        })();

        if (w.config.mustardcut === true) {
            w.loader = {
                index: 0,
                registered: [],
                scripts: [
                    
                        {src: '/static/js/global-article-es6-bundle-c8a573ca90.js', test: 'global-article-js', module: true},
                        {src: '/static/js/global-article-es5-bundle-d17603b9e9.js', test: 'global-article-js', nomodule: true},
                        {src: '/static/js/shared-es6-bundle-e9d2061c95.js', test: 'shared-js', module: true},
                        {src: '/static/js/shared-es5-bundle-1cbf3a8b56.js', test: 'shared-js', nomodule: true},
                        {src: '/static/js/header-150-es6-bundle-5bb959eaa1.js', test: 'header-150-js', module: true},
                        {src: '/static/js/header-150-es5-bundle-994fde5b1d.js', test: 'header-150-js', nomodule: true}
                    
                ].filter(function (s) {
                    if (s.src === null) return false;
                    if (moduleSupport && s.nomodule) return false;
                    return !(!moduleSupport && s.module);
                }),

                register: function (value) {
                    this.registered.push(value);
                },

                ready: function () {
                    if (this.registered.length === this.scripts.length) {
                        this.registered.forEach(function (fn) {
                            if (typeof fn === 'function') {
                                setTimeout(fn, 0); 
                            }
                        });
                        this.ready = function () {};
                    }
                },

                insert: function (s) {
                    var t = d.getElementById('js-position' + this.index);
                    if (t && t.insertAdjacentElement) {
                        t.insertAdjacentElement('afterend', s);
                    } else {
                        d.head.appendChild(s);
                    }
                    ++this.index;
                },

                createScript: function (script, beforeLoad) {
                    var s = d.createElement('script');
                    s.id = 'js-position' + (this.index + 1);
                    s.setAttribute('data-test', script.test);
                    if (beforeLoad) {
                        s.defer = 'defer';
                        s.onload = function () {
                            if (script.noinit) {
                                loader.register(true);
                            }
                            if (d.readyState === 'interactive' || d.readyState === 'complete') {
                                loader.ready();
                            }
                        };
                    } else {
                        s.async = 'async';
                    }
                    s.src = script.src;
                    return s;
                },

                init: function () {
                    this.scripts.forEach(function (s) {
                        loader.insert(loader.createScript(s, true));
                    });

                    d.addEventListener('DOMContentLoaded', function () {
                        loader.ready();
                        var conditionalScripts;
                        
                            conditionalScripts = [
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es6-bundle-464a2af269.js', test: 'pan-zoom-js',  module: true },
                                {match: 'div[data-pan-container]', src: '/static/js/pan-zoom-es5-bundle-98fb9b653b.js', test: 'pan-zoom-js',  nomodule: true },
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es6-bundle-23597ae350.js', test: 'math-js', module: true},
                                {match: 'math,span.mathjax-tex', src: '/static/js/math-es5-bundle-6532c6f78b.js', test: 'math-js', nomodule: true}
                            ];
                        

                        if (conditionalScripts) {
                            conditionalScripts.filter(function (script) {
                                return !!document.querySelector(script.match) && !((moduleSupport && script.nomodule) || (!moduleSupport && script.module));
                            }).forEach(function (script) {
                                loader.insert(loader.createScript(script));
                            });
                        }
                    }, false);
                }
            };
            loader.init();
        }
    })(window, document);
</script><script id="js-position1" data-test="global-article-js" defer="" src="/static/js/global-article-es6-bundle-c8a573ca90.js"></script><script id="js-position2" data-test="shared-js" defer="" src="/static/js/shared-es6-bundle-e9d2061c95.js"></script><script id="js-position3" data-test="header-150-js" defer="" src="/static/js/header-150-es6-bundle-5bb959eaa1.js"></script><script id="js-position4" data-test="math-js" async="" src="/static/js/math-es6-bundle-23597ae350.js"></script>










<meta name="robots" content="noarchive">
<meta name="access" content="Yes">


<link rel="search" href="https://www.nature.com/search">
<link rel="search" href="https://www.nature.com/opensearch/opensearch.xml" type="application/opensearchdescription+xml" title="nature.com">
<link rel="search" href="https://www.nature.com/opensearch/request" type="application/sru+xml" title="nature.com">





    
    <script type="application/ld+json">{"mainEntity":{"headline":"Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects","description":"Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells carrying no or only neutral mutations. If mutations in different genes lead to distinct fitness advantages, this could enable patient stratification. We quantified the fitness effects of mutations over 12 years in older age using longitudinal sequencing and developed a filtering method that considers individual mutational context alongside mutation co-occurrence to quantify the growth potential of variants within individuals. We found that gene-specific fitness differences can outweigh inter-individual variation and, therefore, could form the basis for personalized clinical management. An analysis of blood samples from longitudinal cohorts reveals insights on the dynamics of clonal hematopoiesis of indeterminate potential and proposes a model that could be used for individualized clone monitoring over time.","datePublished":"2022-07-04T00:00:00Z","dateModified":"2022-07-04T00:00:00Z","pageStart":"1439","pageEnd":"1446","license":"http://creativecommons.org/licenses/by/4.0/","sameAs":"https://doi.org/10.1038/s41591-022-01883-3","keywords":["Cancer genomics","Cancer stem cells","Haematological cancer","Haematopoietic stem cells","Risk factors","Biomedicine","general","Cancer Research","Metabolic Diseases","Infectious Diseases","Molecular Medicine","Neurosciences"],"image":["https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig1_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig2_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig3_HTML.png","https://media.springernature.com/lw1200/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig4_HTML.png"],"isPartOf":{"name":"Nature Medicine","issn":["1546-170X","1078-8956"],"volumeNumber":"28","@type":["Periodical","PublicationVolume"]},"publisher":{"name":"Nature Publishing Group US","logo":{"url":"https://www.springernature.com/app-sn/public/images/logo-springernature.png","@type":"ImageObject"},"@type":"Organization"},"author":[{"name":"Neil A. Robertson","url":"http://orcid.org/0000-0002-9509-1157","affiliation":[{"name":"University of Edinburgh","address":{"name":"MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Eric Latorre-Crespo","affiliation":[{"name":"University of Edinburgh","address":{"name":"MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Maria Terradas-Terradas","url":"http://orcid.org/0000-0002-3823-8968","affiliation":[{"name":"University of Glasgow","address":{"name":"Institute of Cancer Sciences, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Cancer Research UK Beatson Institute","address":{"name":"Cancer Research UK Beatson Institute, Glasgow, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Jorge Lemos-Portela","affiliation":[{"name":"University of Edinburgh","address":{"name":"Centre for Regenerative Medicine, Institute for Regeneration and Repair, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Alison C. Purcell","url":"http://orcid.org/0000-0002-2911-9035","affiliation":[{"name":"University of Glasgow","address":{"name":"Institute of Cancer Sciences, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Cancer Research UK Beatson Institute","address":{"name":"Cancer Research UK Beatson Institute, Glasgow, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Benjamin J. Livesey","url":"http://orcid.org/0000-0001-6866-1452","affiliation":[{"name":"University of Edinburgh","address":{"name":"MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Robert F. Hillary","url":"http://orcid.org/0000-0002-2595-552X","affiliation":[{"name":"University of Edinburgh","address":{"name":"Centre for Genomic and Experimental Medicine, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Lee Murphy","url":"http://orcid.org/0000-0001-6467-7449","affiliation":[{"name":"University of Edinburgh","address":{"name":"Edinburgh Clinical Research Facility, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Angie Fawkes","affiliation":[{"name":"University of Edinburgh","address":{"name":"Edinburgh Clinical Research Facility, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Louise MacGillivray","affiliation":[{"name":"University of Edinburgh","address":{"name":"Edinburgh Clinical Research Facility, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Mhairi Copland","url":"http://orcid.org/0000-0002-7655-016X","affiliation":[{"name":"University of Glasgow","address":{"name":"Paul O’Gorman Leukaemia Research Centre, Institute of Cancer Sciences, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Riccardo E. Marioni","affiliation":[{"name":"University of Edinburgh","address":{"name":"Centre for Genomic and Experimental Medicine, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Joseph A. Marsh","url":"http://orcid.org/0000-0003-4132-0628","affiliation":[{"name":"University of Edinburgh","address":{"name":"MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Sarah E. Harris","url":"http://orcid.org/0000-0002-4941-5106","affiliation":[{"name":"University of Edinburgh","address":{"name":"Lothian Birth Cohorts, Department of Psychology, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Simon R. Cox","affiliation":[{"name":"University of Edinburgh","address":{"name":"Lothian Birth Cohorts, Department of Psychology, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Ian J. Deary","affiliation":[{"name":"University of Edinburgh","address":{"name":"Lothian Birth Cohorts, Department of Psychology, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"@type":"Person"},{"name":"Linus J. Schumacher","url":"http://orcid.org/0000-0003-0797-3406","affiliation":[{"name":"University of Edinburgh","address":{"name":"Centre for Regenerative Medicine, Institute for Regeneration and Repair, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"linus.schumacher@ed.ac.uk","@type":"Person"},{"name":"Kristina Kirschner","url":"http://orcid.org/0000-0001-7607-8670","affiliation":[{"name":"University of Glasgow","address":{"name":"Institute of Cancer Sciences, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, UK","@type":"PostalAddress"},"@type":"Organization"},{"name":"Cancer Research UK Beatson Institute","address":{"name":"Cancer Research UK Beatson Institute, Glasgow, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"kristina.kirschner@glasgow.ac.uk","@type":"Person"},{"name":"Tamir Chandra","url":"http://orcid.org/0000-0002-7935-317X","affiliation":[{"name":"University of Edinburgh","address":{"name":"MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK","@type":"PostalAddress"},"@type":"Organization"}],"email":"tamir.chandra@igmm.ed.ac.uk","@type":"Person"}],"isAccessibleForFree":true,"@type":"ScholarlyArticle"},"@context":"https://schema.org","@type":"WebPage"}</script>




    
    
    


    
    <link rel="canonical" href="https://www.nature.com/articles/s41591-022-01883-3">
    
    
    <meta name="journal_id" content="41591">
    <meta name="dc.title" content="Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects">
    <meta name="dc.source" content="Nature Medicine 2022 28:7">
    <meta name="dc.format" content="text/html">
    <meta name="dc.publisher" content="Nature Publishing Group">
    <meta name="dc.date" content="2022-07-04">
    <meta name="dc.type" content="OriginalPaper">
    <meta name="dc.language" content="En">
    <meta name="dc.copyright" content="2022 The Author(s)">
    <meta name="dc.rights" content="2022 The Author(s)">
    <meta name="dc.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="dc.description" content="Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells carrying no or only neutral mutations. If mutations in different genes lead to distinct fitness advantages, this could enable patient stratification. We quantified the fitness effects of mutations over 12 years in older age using longitudinal sequencing and developed a filtering method that considers individual mutational context alongside mutation co-occurrence to quantify the growth potential of variants within individuals. We found that gene-specific fitness differences can outweigh inter-individual variation and, therefore, could form the basis for personalized clinical management. An analysis of blood samples from longitudinal cohorts reveals insights on the dynamics of clonal hematopoiesis of indeterminate potential and proposes a model that could be used for individualized clone monitoring over time.">
    <meta name="prism.issn" content="1546-170X">
    <meta name="prism.publicationName" content="Nature Medicine">
    <meta name="prism.publicationDate" content="2022-07-04">
    <meta name="prism.volume" content="28">
    <meta name="prism.number" content="7">
    <meta name="prism.section" content="OriginalPaper">
    <meta name="prism.startingPage" content="1439">
    <meta name="prism.endingPage" content="1446">
    <meta name="prism.copyright" content="2022 The Author(s)">
    <meta name="prism.rightsAgent" content="journalpermissions@springernature.com">
    <meta name="prism.url" content="https://www.nature.com/articles/s41591-022-01883-3">
    <meta name="prism.doi" content="doi:10.1038/s41591-022-01883-3">
    <meta name="citation_pdf_url" content="https://www.nature.com/articles/s41591-022-01883-3.pdf">
    <meta name="citation_fulltext_html_url" content="https://www.nature.com/articles/s41591-022-01883-3">
    <meta name="citation_journal_title" content="Nature Medicine">
    <meta name="citation_journal_abbrev" content="Nat Med">
    <meta name="citation_publisher" content="Nature Publishing Group">
    <meta name="citation_issn" content="1546-170X">
    <meta name="citation_title" content="Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects">
    <meta name="citation_volume" content="28">
    <meta name="citation_issue" content="7">
    <meta name="citation_publication_date" content="2022/07">
    <meta name="citation_online_date" content="2022/07/04">
    <meta name="citation_firstpage" content="1439">
    <meta name="citation_lastpage" content="1446">
    <meta name="citation_article_type" content="Article">
    <meta name="citation_fulltext_world_readable" content="">
    <meta name="citation_language" content="en">
    <meta name="dc.identifier" content="doi:10.1038/s41591-022-01883-3">
    <meta name="DOI" content="10.1038/s41591-022-01883-3">
    <meta name="size" content="187413">
    <meta name="citation_doi" content="10.1038/s41591-022-01883-3">
    <meta name="citation_springer_api_url" content="http://api.springer.com/xmldata/jats?q=doi:10.1038/s41591-022-01883-3&amp;api_key=">
    <meta name="description" content="Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells carrying no or only neutral mutations. If mutations in different genes lead to distinct fitness advantages, this could enable patient stratification. We quantified the fitness effects of mutations over 12 years in older age using longitudinal sequencing and developed a filtering method that considers individual mutational context alongside mutation co-occurrence to quantify the growth potential of variants within individuals. We found that gene-specific fitness differences can outweigh inter-individual variation and, therefore, could form the basis for personalized clinical management. An analysis of blood samples from longitudinal cohorts reveals insights on the dynamics of clonal hematopoiesis of indeterminate potential and proposes a model that could be used for individualized clone monitoring over time.">
    <meta name="dc.creator" content="Robertson, Neil A.">
    <meta name="dc.creator" content="Latorre-Crespo, Eric">
    <meta name="dc.creator" content="Terradas-Terradas, Maria">
    <meta name="dc.creator" content="Lemos-Portela, Jorge">
    <meta name="dc.creator" content="Purcell, Alison C.">
    <meta name="dc.creator" content="Livesey, Benjamin J.">
    <meta name="dc.creator" content="Hillary, Robert F.">
    <meta name="dc.creator" content="Murphy, Lee">
    <meta name="dc.creator" content="Fawkes, Angie">
    <meta name="dc.creator" content="MacGillivray, Louise">
    <meta name="dc.creator" content="Copland, Mhairi">
    <meta name="dc.creator" content="Marioni, Riccardo E.">
    <meta name="dc.creator" content="Marsh, Joseph A.">
    <meta name="dc.creator" content="Harris, Sarah E.">
    <meta name="dc.creator" content="Cox, Simon R.">
    <meta name="dc.creator" content="Deary, Ian J.">
    <meta name="dc.creator" content="Schumacher, Linus J.">
    <meta name="dc.creator" content="Kirschner, Kristina">
    <meta name="dc.creator" content="Chandra, Tamir">
    <meta name="dc.subject" content="Cancer genomics">
    <meta name="dc.subject" content="Cancer stem cells">
    <meta name="dc.subject" content="Haematological cancer">
    <meta name="dc.subject" content="Haematopoietic stem cells">
    <meta name="dc.subject" content="Risk factors">
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Cancer; citation_title=How ageing processes influence cancer; citation_author=JP Magalhaes; citation_volume=13; citation_publication_date=2013; citation_pages=357-365; citation_doi=10.1038/nrc3497; citation_id=CR1">
    <meta name="citation_reference" content="citation_journal_title=Genome Med.; citation_title=Somatic mutation and clonal expansions in human tissues; citation_author=I Martincorena; citation_volume=11; citation_publication_date=2019; citation_doi=10.1186/s13073-019-0648-4; citation_id=CR2">
    <meta name="citation_reference" content="citation_journal_title=Exp. Hematol.; citation_title=60 years of clonal hematopoiesis research: from X-chromosome inactivation studies to the identification of driver mutations; citation_author=S Ayachi, M Buscarlet, L Busque; citation_volume=83; citation_publication_date=2020; citation_pages=2-11; citation_doi=10.1016/j.exphem.2020.01.008; citation_id=CR3">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence; citation_author=G Genovese; citation_volume=371; citation_publication_date=2014; citation_pages=2477-2487; citation_doi=10.1056/NEJMoa1409405; citation_id=CR4">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=Age-related clonal hematopoiesis associated with adverse outcomes; citation_author=S Jaiswal; citation_volume=371; citation_publication_date=2014; citation_pages=2488-2498; citation_doi=10.1056/NEJMoa1408617; citation_id=CR5">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The landscape of somatic mutation in normal colorectal epithelial cells; citation_author=H Lee-Six; citation_volume=574; citation_publication_date=2019; citation_pages=532-537; citation_doi=10.1038/s41586-019-1672-7; citation_id=CR6">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=High burden and pervasive positive selection of somatic mutations in normal human skin; citation_author=I Martincorena; citation_volume=348; citation_publication_date=2015; citation_pages=880-886; citation_doi=10.1126/science.aaa6806; citation_id=CR7">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=Clonal hematopoiesis in human aging and disease; citation_author=S Jaiswal, BL Ebert; citation_volume=366; citation_publication_date=2019; citation_pages=eaan4673; citation_doi=10.1126/science.aan4673; citation_id=CR8">
    <meta name="citation_reference" content="citation_journal_title=Exp. Hematol.; citation_title=Clonal hematopoiesis in cancer; citation_author=SJ Park, R Bejar; citation_volume=83; citation_publication_date=2020; citation_pages=105-112; citation_doi=10.1016/j.exphem.2020.02.001; citation_id=CR9">
    <meta name="citation_reference" content="citation_journal_title=Mech. Ageing Dev.; citation_title=Clonality in haematopoietic stem cell ageing; citation_author=M Terradas-Terradas, NA Robertson, T Chandra, K Kirschner; citation_volume=189; citation_publication_date=2020; citation_pages=111279; citation_doi=10.1016/j.mad.2020.111279; citation_id=CR10">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=Clonal hematopoiesis: mechanisms driving dominance of stem cell clones; citation_author=GA Challen, MA Goodell; citation_volume=136; citation_publication_date=2020; citation_pages=1590-1598; citation_id=CR11">
    <meta name="citation_reference" content="citation_journal_title=Nat. Rev. Cancer; citation_title=The role of mutations in epigenetic regulators in myeloid malignancies; citation_author=AH Shih, O Abdel-Wahab, JP Patel, RL Levine; citation_volume=12; citation_publication_date=2012; citation_pages=599-612; citation_doi=10.1038/nrc3343; citation_id=CR12">
    <meta name="citation_reference" content="citation_journal_title=Blood; citation_title=What to tell your patient with clonal hematopoiesis and why: insights from two specialized clinics; citation_author=DP Steensma, KL Bolton; citation_volume=136; citation_publication_date=2020; citation_pages=1623-1631; citation_id=CR13">
    <meta name="citation_reference" content="citation_journal_title=Science; citation_title=The evolutionary dynamics and fitness landscape of clonal hematopoiesis; citation_author=CJ Watson; citation_volume=367; citation_publication_date=2020; citation_pages=1449-1454; citation_doi=10.1126/science.aay9333; citation_id=CR14">
    <meta name="citation_reference" content="citation_journal_title=eLife; citation_title=Measuring the distribution of fitness effects in somatic evolution by combining clonal dynamics with dN/dS ratios; citation_author=MJ Williams; citation_volume=9; citation_publication_date=2020; citation_pages=e48714; citation_doi=10.7554/eLife.48714; citation_id=CR15">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Prediction of acute myeloid leukaemia risk in healthy individuals; citation_author=S Abelson; citation_volume=559; citation_publication_date=2018; citation_pages=400-404; citation_doi=10.1038/s41586-018-0317-6; citation_id=CR16">
    <meta name="citation_reference" content="citation_journal_title=Int. J. Epidemiol.; citation_title=Cohort profile update: the Lothian Birth Cohorts of 1921 and 1936; citation_author=AM Taylor, A Pattie, IJ Deary; citation_volume=47; citation_publication_date=2018; citation_pages=1042-1042r; citation_doi=10.1093/ije/dyy022; citation_id=CR17">
    <meta name="citation_reference" content="citation_journal_title=Curr. Biol.; citation_title=Age-related clonal haemopoiesis is associated with increased epigenetic age; citation_author=NA Robertson; citation_volume=29; citation_publication_date=2019; citation_pages=R786-R787; citation_doi=10.1016/j.cub.2019.07.011; citation_id=CR18">
    <meta name="citation_reference" content="citation_journal_title=Cell Rep.; citation_title=Leukemia-associated somatic mutations drive distinct patterns of age-related clonal hemopoiesis; citation_author=T McKerrell; citation_volume=10; citation_publication_date=2015; citation_pages=1239-1245; citation_doi=10.1016/j.celrep.2015.02.005; citation_id=CR19">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=COSMIC: the Catalogue Of Somatic Mutations In Cancer; citation_author=JG Tate; citation_volume=47; citation_publication_date=2019; citation_pages=D941-D947; citation_doi=10.1093/nar/gky1015; citation_id=CR20">
    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Identifying Mendelian disease genes with the Variant Effect Scoring Tool; citation_author=H Carter, C Douville, PD Stenson, DN Cooper, R Karchin; citation_volume=14; citation_publication_date=2013; citation_doi=10.1186/1471-2164-14-S3-S3; citation_id=CR21">
    <meta name="citation_reference" content="citation_journal_title=BMC Genomics; citation_title=Better prediction of functional effects for sequence variants; citation_author=M Hecht, Y Bromberg, B Rost; citation_volume=16; citation_publication_date=2015; citation_doi=10.1186/1471-2164-16-S8-S1; citation_id=CR22">
    <meta name="citation_reference" content="citation_journal_title=Am. J. Hum. Genet; citation_title=REVEL: an ensemble method for predicting the pathogenicity of rare missense variants; citation_author=NM Ioannidis; citation_volume=99; citation_publication_date=2016; citation_pages=877-885; citation_doi=10.1016/j.ajhg.2016.08.016; citation_id=CR23">
    <meta name="citation_reference" content="citation_journal_title=Mol. Syst. Biol.; citation_title=Using deep mutational scanning to benchmark variant effect predictors and identify disease mutations; citation_author=BJ Livesey, JA Marsh; citation_volume=16; citation_publication_date=2020; citation_pages=e9380; citation_doi=10.15252/msb.20199380; citation_id=CR24">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=DEOGEN2: prediction and interactive visualization of single amino acid variant deleteriousness in human proteins; citation_author=D Raimondi; citation_volume=45; citation_publication_date=2017; citation_pages=W201-W206; citation_doi=10.1093/nar/gkx390; citation_id=CR25">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=Deep generative models of genetic variation capture the effects of mutations; citation_author=AJ Riesselman, JB Ingraham, DS Marks; citation_volume=15; citation_publication_date=2018; citation_pages=816-822; citation_doi=10.1038/s41592-018-0138-4; citation_id=CR26">
    <meta name="citation_reference" content="citation_journal_title=Nat. Protoc.; citation_title=SIFT missense predictions for genomes; citation_author=R Vaser, S Adusumalli, SN Leng, M Sikic, PC Ng; citation_volume=11; citation_publication_date=2016; citation_pages=1-9; citation_doi=10.1038/nprot.2015.123; citation_id=CR27">
    <meta name="citation_reference" content="citation_journal_title=N. Engl. J. Med.; citation_title=DNMT3A mutations in acute myeloid leukemia; citation_author=TJ Ley; citation_volume=363; citation_publication_date=2010; citation_pages=2424-2433; citation_doi=10.1056/NEJMoa1005143; citation_id=CR28">
    <meta name="citation_reference" content="citation_journal_title=Sci. Data; citation_title=Extensive sequencing of seven human genomes to characterize benchmark reference materials; citation_author=JM Zook; citation_volume=3; citation_publication_date=2016; citation_doi=10.1038/sdata.2016.25; citation_id=CR29">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Trimmomatic: a flexible trimmer for Illumina sequence data; citation_author=AM Bolger, M Lohse, B Usadel; citation_volume=30; citation_publication_date=2014; citation_pages=2114-2120; citation_doi=10.1093/bioinformatics/btu170; citation_id=CR30">
    <meta name="citation_reference" content="citation_journal_title=Bioinformatics; citation_title=Fast and accurate short read alignment with Burrows–Wheeler transform; citation_author=H Li, R Durbin; citation_volume=25; citation_publication_date=2009; citation_pages=1754-1760; citation_doi=10.1093/bioinformatics/btp324; citation_id=CR31">
    <meta name="citation_reference" content="citation_journal_title=Genome Biol.; citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome; citation_author=B Langmead, C Trapnell, M Pop, SL Salzberg; citation_volume=10; citation_publication_date=2009; citation_doi=10.1186/gb-2009-10-3-r25; citation_id=CR32">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets; citation_author=A Wilm; citation_volume=40; citation_publication_date=2012; citation_pages=11189-11201; citation_doi=10.1093/nar/gks918; citation_id=CR33">
    <meta name="citation_reference" content="Garrison, E. &amp; Marth, G. Haplotype-based variant detection from short-read sequencing. Preprint at 
                  https://arxiv.org/abs/1207.3907
                  
                 (2012).">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=The mutational constraint spectrum quantified from variation in 141,456 humans; citation_author=KJ Karczewski; citation_volume=581; citation_publication_date=2020; citation_pages=434-443; citation_doi=10.1038/s41586-020-2308-7; citation_id=CR35">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Highly accurate protein structure prediction with AlphaFold; citation_author=J Jumper; citation_volume=596; citation_publication_date=2021; citation_pages=583-589; citation_doi=10.1038/s41586-021-03819-2; citation_id=CR36">
    <meta name="citation_reference" content="citation_journal_title=Nucleic Acids Res.; citation_title=AlphaFold Protein Structure Database: massively expanding the structural coverage of protein-sequence space with high-accuracy models; citation_author=M Varadi; citation_volume=50; citation_publication_date=2022; citation_pages=D439-D444; citation_doi=10.1093/nar/gkab1061; citation_id=CR37">
    <meta name="citation_reference" content="citation_journal_title=Proc. Natl Acad. Sci. USA; citation_title=A stochastic model of stem cell proliferation, based on the growth of spleen colony-forming cells; citation_author=JE Till, EA Mcculloch, L Siminovitch; citation_volume=51; citation_publication_date=1964; citation_pages=29-36; citation_doi=10.1073/pnas.51.1.29; citation_id=CR38">
    <meta name="citation_reference" content="Bayes, T. An essay towards solving a problem in the doctrine of chances. By the late Rev. Mr. Bayes, F. R. S. communicated by Mr. Price, in a letter to John Canton, A. M. F. R. S. 
                  https://doi.org/10.1098/rstl.1763.0053
                  
                 (1763).">
    <meta name="citation_reference" content="Van Rossum, G. &amp; Drake, F. L. Python 3 Reference Manual. 
                  https://dl.acm.org/doi/book/10.5555/1593511
                  
                 (CreateSpace, 2009).">
    <meta name="citation_reference" content="citation_journal_title=Nature; citation_title=Array programming with NumPy; citation_author=CR Harris; citation_volume=585; citation_publication_date=2020; citation_pages=357-362; citation_doi=10.1038/s41586-020-2649-2; citation_id=CR41">
    <meta name="citation_reference" content="citation_journal_title=Nat. Methods; citation_title=SciPy 1.0: fundamental algorithms for scientific computing in Python; citation_author=null Virtanen, P.; citation_volume=17; citation_publication_date=2020; citation_pages=261-272; citation_doi=10.1038/s41592-019-0686-2; citation_id=CR42">
    <meta name="citation_reference" content="citation_journal_title=J. Open Source Softw.; citation_title=lifelines: survival analysis in Python; citation_author=C Davidson-Pilon; citation_volume=4; citation_publication_date=2019; citation_pages=1317; citation_doi=10.21105/joss.01317; citation_id=CR43">
    <meta name="citation_reference" content="R Core Team. R: a language and environment for statistical computing. 
                  https://www.R-project.org/
                  
                 (R Foundation for Statistical Computing, 2021).">
    <meta name="citation_reference" content="citation_journal_title=J. Open Source Softw.; citation_title=Welcome to the tidyverse; citation_author=H Wickham; citation_volume=4; citation_publication_date=2019; citation_pages=1686; citation_doi=10.21105/joss.01686; citation_id=CR45">
    <meta name="citation_reference" content="Wickham, H. ggplot2: Elegant Graphics for Data Analysis. 
                  https://ggplot2.tidyverse.org
                  
                 (Springer-Verlag, 2016).">
    <meta name="citation_author" content="Robertson, Neil A.">
    <meta name="citation_author_institution" content="MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Latorre-Crespo, Eric">
    <meta name="citation_author_institution" content="MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Terradas-Terradas, Maria">
    <meta name="citation_author_institution" content="Institute of Cancer Sciences, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, UK">
    <meta name="citation_author_institution" content="Cancer Research UK Beatson Institute, Glasgow, UK">
    <meta name="citation_author" content="Lemos-Portela, Jorge">
    <meta name="citation_author_institution" content="Centre for Regenerative Medicine, Institute for Regeneration and Repair, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Purcell, Alison C.">
    <meta name="citation_author_institution" content="Institute of Cancer Sciences, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, UK">
    <meta name="citation_author_institution" content="Cancer Research UK Beatson Institute, Glasgow, UK">
    <meta name="citation_author" content="Livesey, Benjamin J.">
    <meta name="citation_author_institution" content="MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Hillary, Robert F.">
    <meta name="citation_author_institution" content="Centre for Genomic and Experimental Medicine, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Murphy, Lee">
    <meta name="citation_author_institution" content="Edinburgh Clinical Research Facility, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Fawkes, Angie">
    <meta name="citation_author_institution" content="Edinburgh Clinical Research Facility, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="MacGillivray, Louise">
    <meta name="citation_author_institution" content="Edinburgh Clinical Research Facility, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Copland, Mhairi">
    <meta name="citation_author_institution" content="Paul O’Gorman Leukaemia Research Centre, Institute of Cancer Sciences, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, UK">
    <meta name="citation_author" content="Marioni, Riccardo E.">
    <meta name="citation_author_institution" content="Centre for Genomic and Experimental Medicine, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Marsh, Joseph A.">
    <meta name="citation_author_institution" content="MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Harris, Sarah E.">
    <meta name="citation_author_institution" content="Lothian Birth Cohorts, Department of Psychology, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Cox, Simon R.">
    <meta name="citation_author_institution" content="Lothian Birth Cohorts, Department of Psychology, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Deary, Ian J.">
    <meta name="citation_author_institution" content="Lothian Birth Cohorts, Department of Psychology, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Schumacher, Linus J.">
    <meta name="citation_author_institution" content="Centre for Regenerative Medicine, Institute for Regeneration and Repair, University of Edinburgh, Edinburgh, UK">
    <meta name="citation_author" content="Kirschner, Kristina">
    <meta name="citation_author_institution" content="Institute of Cancer Sciences, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, UK">
    <meta name="citation_author_institution" content="Cancer Research UK Beatson Institute, Glasgow, UK">
    <meta name="citation_author" content="Chandra, Tamir">
    <meta name="citation_author_institution" content="MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK">
    <meta name="access_endpoint" content="https://www.nature.com/platform/readcube-access">
    <meta name="twitter:site" content="@naturemedicine">
    <meta name="twitter:card" content="summary_large_image">
    <meta name="twitter:image:alt" content="Content cover image">
    <meta name="twitter:title" content="Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects">
    <meta name="twitter:description" content="Nature Medicine - An analysis of blood samples from longitudinal cohorts reveals insights on the dynamics of clonal hematopoiesis of indeterminate potential and proposes a model that could be used...">
    <meta name="twitter:image" content="https://media.springernature.com/full/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig1_HTML.png">
    

    
    
    <meta property="og:url" content="https://www.nature.com/articles/s41591-022-01883-3">
    <meta property="og:type" content="article">
    <meta property="og:site_name" content="Nature">
    <meta property="og:title" content="Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects - Nature Medicine">
    <meta property="og:description" content="An analysis of blood samples from longitudinal cohorts reveals insights on the dynamics of clonal hematopoiesis of indeterminate potential and proposes a model that could be used for individualized clone monitoring over time.">
    <meta property="og:image" content="https://media.springernature.com/m685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig1_HTML.png">
    

    <script>
        window.eligibleForRa21 = 'false'; 
    </script>
<style type="text/css">.cc-banner{background-color:#01324b;border:none!important;bottom:0;box-sizing:border-box;color:#fff!important;left:0;line-height:1.3;margin:auto 0 0;max-width:100%;outline:0;overflow:visible;padding:0;position:fixed;right:0;width:100%;z-index:99999}.cc-banner::backdrop{background-color:#0000004d}.cc-banner *{color:inherit!important}.cc-banner:focus{box-shadow:none;outline:0}.cc-banner a{color:#fff!important;text-decoration:underline}.cc-banner a:active,.cc-banner a:focus,.cc-banner a:hover{color:inherit;text-decoration:none}.cc-banner a:focus{outline:3px solid #08c!important}.cc-banner h2,.cc-banner h3,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-banner .cc-h2,.cc-banner h2{font-size:18px}.cc-banner .cc-h3,.cc-banner h3{font-size:16px}.cc-banner .cc-h4,.cc-banner .cc-h5,.cc-banner .cc-h6,.cc-banner h4,.cc-banner h5,.cc-banner h6{font-size:14px}.cc-banner .cc-button{font-size:16px}.cc-banner__content{background-color:#01324b;display:flex;flex-direction:column;margin:0 auto;max-height:90vh;max-width:100%;padding:16px;position:relative}.cc-banner__content:focus{outline:0}@media (min-width:680px){.cc-banner__content{padding:12px}}@media (min-width:980px){.cc-banner__content{max-height:60vh;padding-bottom:20px;padding-top:20px}}@media (min-width:1320px){.cc-banner__content{padding-bottom:40px;padding-top:40px}}.cc-banner__container{display:flex;flex-direction:column;margin:auto;max-width:1320px;overflow:auto}.cc-banner__title{background:none!important;border:0;flex-shrink:0;font-size:18px!important;font-size:22px!important;font-weight:700!important;letter-spacing:normal;margin:0 0 12px!important;padding:0}@media (min-width:680px){.cc-banner__title{font-size:24px!important;margin:0 0 16px!important}}@media (min-width:1320px){.cc-banner__title{font-size:26px!important;margin:0 0 24px!important}}.cc-banner__body{display:flex;flex-direction:column;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif!important;overflow-x:hidden;overflow-y:auto;padding:3px 3px 16px}@media (min-width:980px){.cc-banner__body{flex-direction:row}}.cc-banner__policy p{font-size:16px!important;margin:0;max-width:none;padding:0}.cc-banner__policy p:not(:last-child){margin:0 0 16px}@media (min-width:980px){.cc-banner__policy p:not(:last-child){margin:0 0 24px}}.cc-banner__policy p a{font-size:16px!important;font-weight:700}.cc-banner__footer{box-shadow:none;flex-shrink:0;font-family:-apple-system,BlinkMacSystemFont,Segoe UI,Roboto,Oxygen-Sans,Ubuntu,Cantarell,Helvetica Neue,sans-serif !important!important;margin:0;padding:12px 0 3px 3px;position:relative}@media (min-width:380px){.cc-banner__footer{align-items:stretch;display:flex;flex-wrap:wrap}}@media (min-width:680px){.cc-banner__footer{box-shadow:none;flex-wrap:nowrap;width:40%}}.cc-banner__footer .cc-banner__button{flex:1 1 auto;font-weight:700;overflow:hidden;padding:.5em 1em;width:100%}@media (min-width:680px){.cc-banner__footer .cc-banner__button{max-width:275px}}.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:12px}@media (min-width:680px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-bottom:0;margin-right:16px}}@media (min-width:980px){.cc-banner__footer .cc-banner__button:not(:last-child){margin-right:24px}}@media (min-width:680px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:16px!important;margin-right:0!important}}@media (min-width:980px){html[dir=rtl] .cc-banner__footer .cc-banner__button:not(:last-child){margin-left:24px!important;margin-right:0!important}}.cc-banner__button-preferences{padding-left:0;padding-right:0}@media (min-width:380px){.cc-banner__button-preferences{flex:0 0 auto;margin:auto}}@media (min-width:680px){.cc-banner__button-preferences{margin:0}}@media (min-width:380px) and (max-width:680px){.cc-banner__button-break{display:block}}@media (min-width:680px){.cc-banner--is-tcf .cc-banner__footer{width:auto}}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__footer{padding-right:48px;width:66%}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px!important}.cc-banner--is-tcf .cc-banner__stacks{margin:16px 0 0}@media (min-width:980px){.cc-banner--is-tcf .cc-banner__stacks{display:flex;flex:0 0 33%;flex-direction:column;margin:0 0 0 48px;overflow:auto}}.cc-banner--is-tcf .cc-banner__stacks-intro{font-size:16px;margin:0 0 12px}.cc-banner--is-tcf .cc-banner__stacks-details{padding:0 3px}.cc-banner--is-tcf .cc-banner .cc-details{border-color:#fff3!important}.cc-box{border-radius:10px;padding:12px}.cc-box--info{background-color:#eff6fb}.cc-box--light{background-color:#faf9f6}.cc-button{border:2px solid #0000;border-radius:32px!important;color:inherit;cursor:pointer;font-size:14px;font-weight:700!important;left:auto;line-height:1.2;margin:0;padding:.5em 1em;right:auto;text-transform:none!important;transition:all .2s}@media (min-width:680px){.cc-button{font-size:14px;padding:.75em 1em}}.cc-button--sm{font-size:12px;padding:2px 8px!important}.cc-button--primary{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--primary:focus{outline:3px solid #08c}.cc-button--primary:focus,.cc-button--primary:hover,.cc-button--secondary{background-color:#fff!important;border-color:#025e8d!important;color:#025e8d!important}.cc-button--secondary:focus{background-color:#025e8d!important;border-color:#fff!important;color:#fff!important;outline:3px solid #08c!important}.cc-button--secondary:hover{background-color:#025e8d!important;border-color:#025e8d!important;color:#fff!important}.cc-button--secondary:active{border:2px solid #01324b!important}.cc-button--secondary.cc-button--contrast{border-color:#fff!important}.cc-button--secondary.cc-button--contrast:hover{background-color:initial!important;color:#fff!important}.cc-button--tertiary{background-color:#f0f7fc!important;background-image:none;border:2px solid #f0f7fc;color:#025e8d!important;text-decoration:underline!important;text-underline-offset:.25em}.cc-button--tertiary:focus{outline:3px solid #08c!important}.cc-button--tertiary:hover{text-decoration-thickness:.25em}.cc-button--tertiary:active{color:#013c5b}.cc-button--link{background-color:initial!important;border-color:#0000!important;border-radius:0!important;color:inherit!important;padding:0!important;text-decoration:underline!important;width:inherit!important}.cc-button--link:focus{outline:3px solid #08c}.cc-button--link:hover{background-color:initial!important;box-shadow:none;text-decoration:none}.cc-button--text{border-radius:0;padding:0}.cc-button--details{padding-right:24px!important;position:relative;width:auto}.cc-button--details:after,.cc-button--details:before{background-color:currentColor;content:"";position:absolute;transition:transform .25s ease-out}.cc-button--details:before{height:10px;margin-top:-5px;right:12px;top:50%;width:2px}.cc-button--details:after{height:2px;margin-top:-1px;right:8px;top:50%;width:10px}.cc-button--details.cc-active:before{transform:rotate(90deg);transform-origin:center}.cc-button--details.cc-active:after{display:none}.cc-details{border-bottom:1px solid #0000001a;padding:12px 0;position:relative;width:100%}.cc-details__summary{align-items:center;display:flex;font-size:14px;font-weight:700;list-style-type:none}.cc-details__summary:focus,.cc-details__summary:focus-visible{outline:3px solid #08c!important;will-change:auto}.cc-details__title{align-items:baseline;display:flex}.cc-details__title h2,.cc-details__title h3,.cc-details__title h4,.cc-details__title h5{line-height:1.4;margin:0!important;padding:0!important}.cc-details__title svg{flex-shrink:0;height:auto;margin-right:8px;position:relative;top:-1px;transition:all .2s;width:auto}.cc-details[open] .cc-details__title svg{top:2px;transform:rotate(90deg)}.cc-details__switch{margin-left:auto}.cc-details__section{padding:16px 0 0 18px}.cc-details__section p{margin:0}.cc-details__section p:not(:last-child){margin-bottom:12px}details summary::-webkit-details-marker{display:none}.cc-radio{align-items:center;display:flex;position:relative}.cc-radio *{cursor:pointer}.cc-radio__input{height:22px;left:0;position:absolute;top:0;width:22px}.cc-radio__input:focus{outline:none}.cc-radio__label{color:inherit;font-size:14px;font-weight:700;line-height:23px;margin:0;padding-left:28px}.cc-radio__label:after,.cc-radio__label:before{background-color:#fff;content:"";display:block;position:absolute;transition:transform .25s ease-out}.cc-radio__label:before{border:1px solid #777;border-radius:50%;height:22px;left:0;top:0;width:22px}.cc-radio__label:after{border:7px solid #025e8d;border-radius:50%;height:0;left:4px;opacity:0;top:4px;width:0}.cc-radio__input:focus+.cc-radio__label:before{box-shadow:0 0 0 2px #08c;outline:none}.cc-radio__label--hidden{display:none}.cc-radio__input:checked+.cc-radio__label:after{opacity:1}.cc-radio__input:disabled{cursor:default}.cc-radio__input:disabled+.cc-radio__label{cursor:default;opacity:.5}.cc-switch *{cursor:pointer}.cc-switch{align-items:center;display:flex;line-height:1}.cc-switch__label{cursor:pointer;display:inline-block;font-size:14px;font-weight:700;margin:0;-webkit-user-select:none;user-select:none}.cc-switch__input[type=checkbox]{-webkit-appearance:none;appearance:none;background-color:#fff!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='rgba(0, 0, 0, 0.25)'/%3E%3C/svg%3E")!important;background-position:0!important;background-repeat:no-repeat!important;border:1px solid #777!important;border-radius:16px!important;height:18px!important;margin:0 8px 0 0!important;transition:background-position .2s ease-in-out!important;width:32px!important}.cc-switch__input[type=checkbox]:focus{background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%2380b3cc'/%3E%3C/svg%3E")!important}.cc-switch__input[type=checkbox]:checked{background-color:#025e8d!important;background-image:url("data:image/svg+xml;charset=utf-8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='-4 -4 8 8'%3E%3Ccircle r='3' fill='%23fff'/%3E%3C/svg%3E")!important;background-position:100%!important}.cc-switch__input[type=checkbox]:checked~.cc-switch__label>.cc-switch__text-off,.cc-switch__input[type=checkbox]:not(:checked)~.cc-switch__label>.cc-switch__text-on{display:none}.cc-switch__input[type=checkbox]:hover{cursor:pointer;outline:none}.cc-switch__input[type=checkbox]:focus-visible{cursor:pointer;outline:3px solid #fc0}.cc-switch__input[type=checkbox]:checked:before{background-color:none!important;color:#0000!important}html[dir=rtl] .cc-switch__input[type=checkbox]{margin-left:8px!important;margin-right:0!important}.cc-list>ul,ul.cc-list{list-style-type:disc;margin-left:0;padding-left:0}.cc-list>ul>li,ul.cc-list>li{margin-left:1em}.cc-list>ul>li:not(:last-child),ul.cc-list>li:not(:last-child){margin-bottom:4px}.cc-list__title{margin-bottom:8px!important}.cc-list--inline>ul,ul.cc-list--inline{display:flex;flex-wrap:wrap;list-style-type:none}.cc-list--inline>ul>li,ul.cc-list--inline>li{margin:0 .5em .5em 0}.cc-overlay{background-color:#0000004d;z-index:99998}.cc-overlay,.cc-preferences{bottom:0;left:0;position:fixed;right:0;top:0}.cc-preferences{background-color:#050a14f2!important;border:0;box-sizing:border-box;color:#111;font-family:sans-serif!important;line-height:1.4;margin:auto;max-height:100vh;overflow:auto;padding:0;z-index:100000}.cc-preferences:focus{outline:none}.cc-preferences *,.cc-preferences :after,.cc-preferences :before{box-sizing:inherit!important}.cc-preferences h2,.cc-preferences h3,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-family:sans-serif;font-style:normal;font-weight:700;margin:0 0 .5em}.cc-preferences .cc-h2,.cc-preferences h2{font-size:18px}.cc-preferences .cc-h3,.cc-preferences h3{font-size:16px}.cc-preferences .cc-h4,.cc-preferences .cc-h5,.cc-preferences .cc-h6,.cc-preferences h4,.cc-preferences h5,.cc-preferences h6{font-size:14px}.cc-preferences a{color:#025e8d;text-decoration:underline}.cc-preferences a:hover{color:inherit;text-decoration:none}.cc-preferences h3{background:none;color:#111;padding:0;text-transform:none}dialog.cc-preferences{background-color:initial}dialog.cc-preferences::backdrop{background-color:#000000e6}.cc-preferences__dialog{display:flex;flex-direction:column;margin:auto;max-height:100vh;max-width:860px;padding:12px;position:relative}.cc-preferences__dialog>:last-child{border-bottom-left-radius:10px;border-bottom-right-radius:10px}@media (min-width:980px){.cc-preferences__dialog{padding:16px}}.cc-preferences__close{background:#0000!important;border:1px solid #ececec;border-radius:50%;color:#111!important;cursor:pointer;font-family:Times New Roman,serif;font-size:40px;height:40px;left:auto;line-height:1;margin-top:-20px;padding:0!important;position:absolute;right:20px;top:50%;width:40px}.cc-preferences__close:focus{outline:3px solid #08c}.cc-preferences__close-label{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__header{background:#fff!important;border-bottom:1px solid #0000001a;border-top-left-radius:10px;border-top-right-radius:10px;padding:16px;position:relative;text-align:center}.cc-preferences__title{background:none!important;color:#111!important;font-family:sans-serif!important;font-size:18px!important;font-weight:700!important;margin:0!important;padding:0 16px 0 0!important}@media (min-width:480px){.cc-preferences__title{padding-right:0!important}}@media (min-width:980px){.cc-preferences__title{font-size:22px!important}}.cc-preferences__body{background:#fff!important;flex:1 1 auto;min-height:200px;overflow-x:hidden;overflow-y:auto;padding:16px}.cc-preferences__footer{background:#fff!important;border-top:1px solid #d0d0d0;box-shadow:0 0 5px 0 #0003;margin-bottom:0;padding:12px;position:relative}@media (min-width:480px){.cc-preferences__footer{align-items:stretch;display:flex}}.cc-preferences__footer>.cc-button{display:block;width:100%}@media (min-width:480px){.cc-preferences__footer>.cc-button{flex:1 1 auto}}@media (min-width:980px){.cc-preferences__footer>.cc-button{flex-basis:auto}}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:first-child){margin-left:12px}}.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:8px}@media (min-width:480px){.cc-preferences__footer>.cc-button:not(:last-child){margin-bottom:0}}.cc-preferences__categories{list-style:none;margin:0;padding:0}.cc-preferences__category:not(:last-child){border-bottom:1px solid #0000001a;margin-bottom:12px;padding-bottom:12px}.cc-preferences__category-description{font-size:14px;margin:0 0 8px;padding:0}.cc-preferences__category-footer{align-items:center;display:flex;justify-content:space-between}.cc-preferences__status{border:0;clip:rect(0,0,0,0);height:1px;margin:-100%;overflow:hidden;padding:0;position:absolute!important;width:1px}.cc-preferences__controls{display:flex;margin:0}.cc-preferences__controls>:not(:last-child){margin-right:12px}.cc-preferences__always-on{font-size:14px;font-weight:700;padding-left:26px;position:relative}.cc-preferences__always-on:before{background:#0000!important;border:solid;border-top-color:#0000;border-width:0 0 4px 4px;content:"";display:block;height:10px;left:0;position:absolute;top:2px;transform:rotate(-45deg);width:18px}.cc-preferences__details{background-color:#eff6fb!important;border-radius:10px;font-size:14px;margin:12px 0 0;padding:12px}.cc-preferences__cookie-list,.cc-preferences__provider-list{list-style:none;margin:0;padding:0}.cc-preferences__provider-list{columns:170px}.cc-preferences__cookie-title{font-size:1em;font-style:normal;font-weight:400;margin:0}.cc-preferences__cookie-description{font-size:1em;margin:0 0 8px}.cc-preferences__cookie-domain,.cc-preferences__cookie-lifespan{border-left:1px solid #999;color:#666;margin-left:8px;padding-left:4px}body.cc-has-preferences-open{overflow:hidden;position:relative}.cc-table{border-collapse:collapse;width:100%}.cc-table tbody tr{border-top:1px solid #0000001a}.cc-table td,.cc-table th{font-size:14px;padding:4px 8px;vertical-align:top}.cc-table+.cc-table{margin-top:8px}.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:70%}@media (min-width:680px){.cc-table--purposes td:first-child,.cc-table--purposes th:first-child{width:80%}}.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{padding-left:8px;width:40%}@media (min-width:680px){.cc-table--purposes td:last-child,.cc-table--purposes th:last-child{width:30%}}.cc-tabs,.cc-tabs>ul{background-color:#faf9f6;display:flex;list-style:none;margin:0;padding:0}.cc-tabs__panel{border-top:1px solid #ececec;margin-top:-1px}.cc-tabs__button{background-color:initial;border:0;border-left:1px solid #0000;border-right:1px solid #0000;color:#111;font-size:16px;font-weight:700;margin:0;padding:15px 16px 12px;position:relative}.cc-tabs__button:before{background-color:initial;content:"";display:block;height:3px;left:0;position:absolute;top:0;width:100%}.cc-tabs__button.cc-active{background-color:#fff;border-color:#025e8d #ececec #ececec;color:#111}.cc-tabs__button.cc-active:before{background-color:#025e8d}.cc-tabs__content{background-color:#fff;display:none;font-size:14px;padding:16px}.cc-tabs__section:not(:first-child){padding-top:24px}.cc-tcf{font-size:14px;margin-top:16px}.cc-tcf__list{list-style:none;margin:0;padding:0}.cc-vendor-count{color:#666;font-size:16px;margin:0 0 8px}.cc-hide{display:none!important}.cc-show{display:block!important}.cc-external-link{background-color:#ececec;background-image:url("data:image/svg+xml;base64,PHN2ZyB4bWxucz0iaHR0cDovL3d3dy53My5vcmcvMjAwMC9zdmciIHdpZHRoPSIyMCIgaGVpZ2h0PSIyMCI+PHBhdGggZD0iTTcgMGExIDEgMCAxIDEgMCAySDIuNmMtLjM3MSAwLS42LjIwOS0uNi41djE1YzAgLjI5MS4yMjkuNS42LjVoMTQuOGMuMzcxIDAgLjYtLjIwOS42LS41VjEzYTEgMSAwIDAgMSAyIDB2NC41YzAgMS40MzgtMS4xNjIgMi41LTIuNiAyLjVIMi42QzEuMTYyIDIwIDAgMTguOTM4IDAgMTcuNXYtMTVDMCAxLjA2MiAxLjE2MiAwIDIuNiAwem02IDBoNmwuMDc1LjAwMy4xMjYuMDE3LjExMS4wMy4xMTEuMDQ0LjA5OC4wNTIuMDk2LjA2Ny4wOS4wOGExIDEgMCAwIDEgLjA5Ny4xMTJsLjA3MS4xMS4wNTQuMTE0LjAzNS4xMDUuMDMuMTQ4TDIwIDF2NmExIDEgMCAwIDEtMiAwVjMuNDE0bC02LjY5MyA2LjY5M2ExIDEgMCAwIDEtMS40MTQtMS40MTRMMTYuNTg0IDJIMTNhMSAxIDAgMCAxLS45OTMtLjg4M0wxMiAxYTEgMSAwIDAgMSAxLTEiLz48L3N2Zz4=");background-position:right 8px center;background-repeat:no-repeat;background-size:10px auto;border-radius:3px;box-shadow:0 1px 0 #0003;color:#111!important;display:inline-block;font-size:12px;padding:4px 26px 4px 8px;text-decoration:none!important}.cc-external-link:hover{text-decoration:underline!important}.cc-m-0{margin:0!important}.cc-grey{color:#666}</style><script async="" type="text/javascript" src="https://injections.readcube.com/nature/inject.fa744f3f.js"></script><style type="text/css">.MathJax_Hover_Frame {border-radius: .25em; -webkit-border-radius: .25em; -moz-border-radius: .25em; -khtml-border-radius: .25em; box-shadow: 0px 0px 15px #83A; -webkit-box-shadow: 0px 0px 15px #83A; -moz-box-shadow: 0px 0px 15px #83A; -khtml-box-shadow: 0px 0px 15px #83A; border: 1px solid #A6D ! important; display: inline-block; position: absolute}
.MathJax_Menu_Button .MathJax_Hover_Arrow {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 4px; -webkit-border-radius: 4px; -moz-border-radius: 4px; -khtml-border-radius: 4px; font-family: 'Courier New',Courier; font-size: 9px; color: #F0F0F0}
.MathJax_Menu_Button .MathJax_Hover_Arrow span {display: block; background-color: #AAA; border: 1px solid; border-radius: 3px; line-height: 0; padding: 4px}
.MathJax_Hover_Arrow:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_Hover_Arrow:hover span {background-color: #CCC!important}
</style><style type="text/css">#MathJax_About {position: fixed; left: 50%; width: auto; text-align: center; border: 3px outset; padding: 1em 2em; background-color: #DDDDDD; color: black; cursor: default; font-family: message-box; font-size: 120%; font-style: normal; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; border-radius: 15px; -webkit-border-radius: 15px; -moz-border-radius: 15px; -khtml-border-radius: 15px; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_About.MathJax_MousePost {outline: none}
.MathJax_Menu {position: absolute; background-color: white; color: black; width: auto; padding: 2px; border: 1px solid #CCCCCC; margin: 0; cursor: default; font: menu; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; z-index: 201; box-shadow: 0px 10px 20px #808080; -webkit-box-shadow: 0px 10px 20px #808080; -moz-box-shadow: 0px 10px 20px #808080; -khtml-box-shadow: 0px 10px 20px #808080; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
.MathJax_MenuItem {padding: 2px 2em; background: transparent}
.MathJax_MenuArrow {position: absolute; right: .5em; padding-top: .25em; color: #666666; font-size: .75em}
.MathJax_MenuActive .MathJax_MenuArrow {color: white}
.MathJax_MenuArrow.RTL {left: .5em; right: auto}
.MathJax_MenuCheck {position: absolute; left: .7em}
.MathJax_MenuCheck.RTL {right: .7em; left: auto}
.MathJax_MenuRadioCheck {position: absolute; left: 1em}
.MathJax_MenuRadioCheck.RTL {right: 1em; left: auto}
.MathJax_MenuLabel {padding: 2px 2em 4px 1.33em; font-style: italic}
.MathJax_MenuRule {border-top: 1px solid #CCCCCC; margin: 4px 1px 0px}
.MathJax_MenuDisabled {color: GrayText}
.MathJax_MenuActive {background-color: Highlight; color: HighlightText}
.MathJax_MenuDisabled:focus, .MathJax_MenuLabel:focus {background-color: #E8E8E8}
.MathJax_ContextMenu:focus {outline: none}
.MathJax_ContextMenu .MathJax_MenuItem:focus {outline: none}
#MathJax_AboutClose {top: .2em; right: .2em}
.MathJax_Menu .MathJax_MenuClose {top: -10px; left: -10px}
.MathJax_MenuClose {position: absolute; cursor: pointer; display: inline-block; border: 2px solid #AAA; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; font-family: 'Courier New',Courier; font-size: 24px; color: #F0F0F0}
.MathJax_MenuClose span {display: block; background-color: #AAA; border: 1.5px solid; border-radius: 18px; -webkit-border-radius: 18px; -moz-border-radius: 18px; -khtml-border-radius: 18px; line-height: 0; padding: 8px 0 6px}
.MathJax_MenuClose:hover {color: white!important; border: 2px solid #CCC!important}
.MathJax_MenuClose:hover span {background-color: #CCC!important}
.MathJax_MenuClose:hover:focus {outline: none}
</style><style type="text/css">.MJX_Assistive_MathML {position: absolute!important; top: 0; left: 0; clip: rect(1px, 1px, 1px, 1px); padding: 1px 0 0 0!important; border: 0!important; height: 1px!important; width: 1px!important; overflow: hidden!important; display: block!important; -webkit-touch-callout: none; -webkit-user-select: none; -khtml-user-select: none; -moz-user-select: none; -ms-user-select: none; user-select: none}
.MJX_Assistive_MathML.MJX_Assistive_MathML_Block {width: 100%!important}
</style><style type="text/css">#MathJax_Zoom {position: absolute; background-color: #F0F0F0; overflow: auto; display: block; z-index: 301; padding: .5em; border: 1px solid black; margin: 0; font-weight: normal; font-style: normal; text-align: left; text-indent: 0; text-transform: none; line-height: normal; letter-spacing: normal; word-spacing: normal; word-wrap: normal; white-space: nowrap; float: none; -webkit-box-sizing: content-box; -moz-box-sizing: content-box; box-sizing: content-box; box-shadow: 5px 5px 15px #AAAAAA; -webkit-box-shadow: 5px 5px 15px #AAAAAA; -moz-box-shadow: 5px 5px 15px #AAAAAA; -khtml-box-shadow: 5px 5px 15px #AAAAAA; filter: progid:DXImageTransform.Microsoft.dropshadow(OffX=2, OffY=2, Color='gray', Positive='true')}
#MathJax_ZoomOverlay {position: absolute; left: 0; top: 0; z-index: 300; display: inline-block; width: 100%; height: 100%; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
#MathJax_ZoomFrame {position: relative; display: inline-block; height: 0; width: 0}
#MathJax_ZoomEventTrap {position: absolute; left: 0; top: 0; z-index: 302; display: inline-block; border: 0; padding: 0; margin: 0; background-color: white; opacity: 0; filter: alpha(opacity=0)}
</style><style type="text/css">.MathJax_Preview {color: #888}
#MathJax_Message {position: fixed; left: 1em; bottom: 1.5em; background-color: #E6E6E6; border: 1px solid #959595; margin: 0px; padding: 2px 8px; z-index: 102; color: black; font-size: 80%; width: auto; white-space: nowrap}
#MathJax_MSIE_Frame {position: absolute; top: 0; left: 0; width: 0px; z-index: 101; border: 0px; margin: 0px; padding: 0px}
.MathJax_Error {color: #CC0000; font-style: italic}
</style><style type="text/css">.MJXp-script {font-size: .8em}
.MJXp-right {-webkit-transform-origin: right; -moz-transform-origin: right; -ms-transform-origin: right; -o-transform-origin: right; transform-origin: right}
.MJXp-bold {font-weight: bold}
.MJXp-italic {font-style: italic}
.MJXp-scr {font-family: MathJax_Script,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-frak {font-family: MathJax_Fraktur,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-sf {font-family: MathJax_SansSerif,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-cal {font-family: MathJax_Caligraphic,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-mono {font-family: MathJax_Typewriter,'Times New Roman',Times,STIXGeneral,serif}
.MJXp-largeop {font-size: 150%}
.MJXp-largeop.MJXp-int {vertical-align: -.2em}
.MJXp-math {display: inline-block; line-height: 1.2; text-indent: 0; font-family: 'Times New Roman',Times,STIXGeneral,serif; white-space: nowrap; border-collapse: collapse}
.MJXp-display {display: block; text-align: center; margin: 1em 0}
.MJXp-math span {display: inline-block}
.MJXp-box {display: block!important; text-align: center}
.MJXp-box:after {content: " "}
.MJXp-rule {display: block!important; margin-top: .1em}
.MJXp-char {display: block!important}
.MJXp-mo {margin: 0 .15em}
.MJXp-mfrac {margin: 0 .125em; vertical-align: .25em}
.MJXp-denom {display: inline-table!important; width: 100%}
.MJXp-denom > * {display: table-row!important}
.MJXp-surd {vertical-align: top}
.MJXp-surd > * {display: block!important}
.MJXp-script-box > *  {display: table!important; height: 50%}
.MJXp-script-box > * > * {display: table-cell!important; vertical-align: top}
.MJXp-script-box > *:last-child > * {vertical-align: bottom}
.MJXp-script-box > * > * > * {display: block!important}
.MJXp-mphantom {visibility: hidden}
.MJXp-munderover, .MJXp-munder {display: inline-table!important}
.MJXp-over {display: inline-block!important; text-align: center}
.MJXp-over > * {display: block!important}
.MJXp-munderover > *, .MJXp-munder > * {display: table-row!important}
.MJXp-mtable {vertical-align: .25em; margin: 0 .125em}
.MJXp-mtable > * {display: inline-table!important; vertical-align: middle}
.MJXp-mtr {display: table-row!important}
.MJXp-mtd {display: table-cell!important; text-align: center; padding: .5em 0 0 .5em}
.MJXp-mtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-mlabeledtr {display: table-row!important}
.MJXp-mlabeledtr > .MJXp-mtd:first-child {padding-left: 0}
.MJXp-mlabeledtr:first-child > .MJXp-mtd {padding-top: 0}
.MJXp-merror {background-color: #FFFF88; color: #CC0000; border: 1px solid #CC0000; padding: 1px 3px; font-style: normal; font-size: 90%}
.MJXp-scale0 {-webkit-transform: scaleX(.0); -moz-transform: scaleX(.0); -ms-transform: scaleX(.0); -o-transform: scaleX(.0); transform: scaleX(.0)}
.MJXp-scale1 {-webkit-transform: scaleX(.1); -moz-transform: scaleX(.1); -ms-transform: scaleX(.1); -o-transform: scaleX(.1); transform: scaleX(.1)}
.MJXp-scale2 {-webkit-transform: scaleX(.2); -moz-transform: scaleX(.2); -ms-transform: scaleX(.2); -o-transform: scaleX(.2); transform: scaleX(.2)}
.MJXp-scale3 {-webkit-transform: scaleX(.3); -moz-transform: scaleX(.3); -ms-transform: scaleX(.3); -o-transform: scaleX(.3); transform: scaleX(.3)}
.MJXp-scale4 {-webkit-transform: scaleX(.4); -moz-transform: scaleX(.4); -ms-transform: scaleX(.4); -o-transform: scaleX(.4); transform: scaleX(.4)}
.MJXp-scale5 {-webkit-transform: scaleX(.5); -moz-transform: scaleX(.5); -ms-transform: scaleX(.5); -o-transform: scaleX(.5); transform: scaleX(.5)}
.MJXp-scale6 {-webkit-transform: scaleX(.6); -moz-transform: scaleX(.6); -ms-transform: scaleX(.6); -o-transform: scaleX(.6); transform: scaleX(.6)}
.MJXp-scale7 {-webkit-transform: scaleX(.7); -moz-transform: scaleX(.7); -ms-transform: scaleX(.7); -o-transform: scaleX(.7); transform: scaleX(.7)}
.MJXp-scale8 {-webkit-transform: scaleX(.8); -moz-transform: scaleX(.8); -ms-transform: scaleX(.8); -o-transform: scaleX(.8); transform: scaleX(.8)}
.MJXp-scale9 {-webkit-transform: scaleX(.9); -moz-transform: scaleX(.9); -ms-transform: scaleX(.9); -o-transform: scaleX(.9); transform: scaleX(.9)}
.MathJax_PHTML .noError {vertical-align: ; font-size: 90%; text-align: left; color: black; padding: 1px 3px; border: 1px solid}
</style></head>
<body class="article-page"><div id="MathJax_Message" style="display: none;"></div><dialog class="cc-banner" data-cc-banner="" data-nosnippet="" aria-labelledby="cc-banner-label" open="">
		<div class="cc-banner__content" autofocus="" tabindex="-1">
			<div class="cc-banner__container">
				<div class="cc-banner__header">
					<h2 class="cc-banner__title" id="cc-banner-label">Your privacy, your choice</h2>
				</div>
				<div class="cc-banner__body">
					<div class="cc-banner__policy">
						<p>We use essential cookies to make sure the site can function. We also use optional cookies for advertising, personalisation of content, usage analysis, and social media.</p><p>By accepting optional cookies, you consent to the processing of your personal data - including transfers to third parties. Some third parties are outside of the European Economic Area, with varying standards of data protection.</p><p>See our <a href="https://www.nature.com/info/privacy" data-cc-action="privacy">privacy policy</a> for more information on the use of your personal data.</p><p><button type="button" data-cc-action="preferences" class="cc-button cc-button--link cc-button--text">Manage preferences</button> for further information and to change your choices.</p>
					</div>
					
				</div>
				<div class="cc-banner__footer">
					
			<button data-cc-action="accept" class="cc-button cc-button--secondary cc-button--contrast cc-banner__button cc-banner__button-accept">Accept all cookies</button>
			
		
				</div>
			</div>
		</div>
	</dialog>

<noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-MRVXSHQ"
                  height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>



<div class="position-relative cleared z-index-50 background-white" data-test="top-containers">
    <a class="c-skip-link" href="#content">Skip to main content</a>



<div class="c-grade-c-banner u-hide">
    <div class="c-grade-c-banner__container">
        
        <p>Thank you for visiting nature.com. You are using a browser version with limited support for CSS. To obtain
            the best experience, we recommend you use a more up to date browser (or turn off compatibility mode in
            Internet Explorer). In the meantime, to ensure continued support, we are displaying the site without styles
            and JavaScript.</p>

    </div>
</div>

    

    

    
    
        <div class="u-hide u-show-following-ad"></div>

    <aside class="c-ad c-ad--728x90" style="display: none !important;">
        <div class="c-ad__inner" data-container-type="banner-advert">
            <p class="c-ad__label">Advertisement</p>
            
        
            
    <div id="div-gpt-ad-top-1" class="div-gpt-ad advert leaderboard js-ad text-center hide-print grade-c-hide" data-ad-type="top" data-test="top-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="728x90" data-gpt-targeting="type=article;pos=top;artid=s41591-022-01883-3;doi=10.1038/s41591-022-01883-3;subjmeta=1542,1990,499,532,631,67,69,692,71;kwrd=Cancer+genomics,Cancer+stem+cells,Haematological+cancer,Haematopoietic+stem+cells,Risk+factors">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=1350846533&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01883-3%26doi%3D10.1038/s41591-022-01883-3%26subjmeta%3D1542,1990,499,532,631,67,69,692,71%26kwrd%3DCancer+genomics,Cancer+stem+cells,Haematological+cancer,Haematopoietic+stem+cells,Risk+factors">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=728x90&amp;c=1350846533&amp;t=pos%3Dtop%26type%3Darticle%26artid%3Ds41591-022-01883-3%26doi%3D10.1038/s41591-022-01883-3%26subjmeta%3D1542,1990,499,532,631,67,69,692,71%26kwrd%3DCancer+genomics,Cancer+stem+cells,Haematological+cancer,Haematopoietic+stem+cells,Risk+factors"
                     alt="Advertisement"
                     width="728"
                     height="90"></a>
        </noscript>
    </div>

        
    
        </div>
    </aside>


    <header class="c-header" id="header" data-header="" data-track-component="nature-150-split-header" style="border-color:#e40428">
        <div class="c-header__row">
            <div class="c-header__container">
                <div class="c-header__split">
                    
                    
                    <div class="c-header__logo-container">
                        
                        <a href="/nm" data-track="click" data-track-action="home" data-track-label="image">
                            <picture class="c-header__logo">
                                <source srcset="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" media="(min-width: 875px)">
                                <img src="https://media.springernature.com/full/nature-cms/uploads/product/nm/header-95e59e63930e5d6009bad2c23a42ab2d.svg" height="32" alt="Nature Medicine">
                            </picture>
                        </a>
                    
                    </div>
                    
                    <ul class="c-header__menu c-header__menu--global">
                        <li class="c-header__item c-header__item--padding c-header__item--hide-md-max">
                            <a class="c-header__link" href="https://www.nature.com/siteindex" data-test="siteindex-link" data-track="click" data-track-action="open nature research index" data-track-label="link">
                                <span>View all journals</span>
                            </a>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--pipe">
                            <a class="c-header__link c-header__link--search" href="javascript:;" data-header-expander="" data-test="search-link" data-track="click" data-track-action="open search tray" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                <svg role="img" aria-hidden="true" focusable="false" height="22" width="22" viewBox="0 0 18 18" xmlns="http://www.w3.org/2000/svg"><path d="M16.48 15.455c.283.282.29.749.007 1.032a.738.738 0 01-1.032-.007l-3.045-3.044a7 7 0 111.026-1.026zM8 14A6 6 0 108 2a6 6 0 000 12z"></path></svg><span>Search</span>
                            </a><div id="search-menu" class="c-header__dropdown c-header__dropdown--full-width has-tethered u-js-hide" data-track-component="nature-150-split-header" hidden="">
    <div class="c-header__container">
        <h2 class="c-header__visually-hidden">Search</h2>
        <form class="c-header__search-form" action="/search" method="get" role="search" autocomplete="off" data-test="inline-search">
            <label class="c-header__heading" for="keywords">Search articles by subject, keyword or author</label>
            <div class="c-header__search-layout c-header__search-layout--max-width">
                <div>
                    <input type="text" required="" class="c-header__input" id="keywords" name="q" value="">
                </div>
                <div class="c-header__search-layout">
                    <div>
                        <label for="results-from" class="c-header__visually-hidden">Show results from</label>
                        <select id="results-from" name="journal" class="c-header__select">
                            
                                
                                    <option value="" selected="">All journals</option>
                                    <option value="nm">This journal</option>
                                
                            
                        </select>
                    </div>
                    <div>
                        <button type="submit" class="c-header__search-button">Search</button>
                    </div>
                </div>

            </div>
        </form>

        <div class="c-header__flush">
            <a class="c-header__link" href="/search/advanced" data-track="click" data-track-action="advanced search" data-track-label="link">
                Advanced search
            </a>
        </div>

        <h3 class="c-header__heading c-header__heading--keyline">Quick links</h3>
        <ul class="c-header__list">
            <li><a class="c-header__link" href="/subjects" data-track="click" data-track-action="explore articles by subject" data-track-label="link">Explore articles by subject</a></li>
            <li><a class="c-header__link" href="/naturecareers" data-track="click" data-track-action="find a job" data-track-label="link">Find a job</a></li>
            <li><a class="c-header__link" href="/authors/index.html" data-track="click" data-track-action="guide to authors" data-track-label="link">Guide to authors</a></li>
            <li><a class="c-header__link" href="/authors/editorial_policies/" data-track="click" data-track-action="editorial policies" data-track-label="link">Editorial policies</a></li>
        </ul>
    </div>
</div>
                        </li>
                        <li class="c-header__item c-header__item--padding c-header__item--snid-account-widget c-header__item--pipe">
                            
                                <a class="c-header__link eds-c-header__link" id="identity-account-widget" data-track="click_login" data-track-context="header" href="https://idp.nature.com/auth/personal/springernature?redirect_uri=https://www.nature.com/articles/s41591-022-01883-3"><span class="eds-c-header__widget-fragment-title">Log in</span></a>
                            
                        </li>
                    </ul>
                </div>
            </div>
        </div>
        
            <div class="c-header__row">
                <div class="c-header__container" data-test="navigation-row">
                    <div class="c-header__split">
                        <ul class="c-header__menu c-header__menu--journal">
                            
                                <li class="c-header__item c-header__item--dropdown-menu" data-test="explore-content-button">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--explore" data-track="click" data-track-action="open explore expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span><span class="c-header__show-text">Explore</span> content</span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Explore-content" data-test="Explore-content" id="explore" data-track-component="nature-150-split-header" hidden="">
            <div class="c-header__container">
                <h2 id="Explore-content" class="c-header__heading c-header__heading--js-hide">Explore content</h2>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-articles" data-track="click" data-track-action="research articles" data-track-label="link" data-test="explore-nav-item">
                                    Research articles
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-and-analysis" data-track="click" data-track-action="reviews &amp; analysis" data-track-label="link" data-test="explore-nav-item">
                                    Reviews &amp; Analysis
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/news-and-comment" data-track="click" data-track-action="news &amp; comment" data-track-label="link" data-test="explore-nav-item">
                                    News &amp; Comment
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/podcast" data-track="click" data-track-action="podcasts" data-track-label="link" data-test="explore-nav-item">
                                    Podcasts
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/current-issue" data-track="click" data-track-action="current issue" data-track-label="link" data-test="explore-nav-item">
                                    Current issue
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/collections" data-track="click" data-track-action="collections" data-track-label="link" data-test="explore-nav-item">
                                    Collections
                                </a>
                            </li>
                        
                    
                </ul>
                <ul class="c-header__list c-header__list--js-stack">
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://www.facebook.com/Nature-Medicine-193691346949/" data-track="click" data-track-action="facebook" data-track-label="link">Follow us on Facebook
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item">
                            <a class="c-header__link" href="https://twitter.com/naturemedicine" data-track="click" data-track-action="twitter" data-track-label="link">Follow us on Twitter
                            </a>
                        </li>
                    
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/my-account/alerts/subscribe-journal?list-id=5" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-external="" data-track-label="link (mobile dropdown)">Sign up for alerts<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#fff"></path></svg>
                            </a>
                        </li>
                    
                    
                        <li class="c-header__item c-header__item--hide-lg">
                            <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                <span>RSS feed</span>
                            </a>
                        </li>
                    
                </ul>
            </div>
        </nav>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--dropdown-menu">
                                    <a href="javascript:;" class="c-header__link" data-header-expander="" data-test="menu-button--about-the-journal" data-track="click" data-track-action="open about the journal expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                        <span>About <span class="c-header__show-text">the journal</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                    </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="About-the-journal" id="about-the-journal" data-test="about-the-journal" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="About-the-journal" class="c-header__heading c-header__heading--js-hide">About the journal</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/aims" data-track="click" data-track-action="aims &amp; scope" data-track-label="link">
                                    Aims &amp; Scope
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-information" data-track="click" data-track-action="journal information" data-track-label="link">
                                    Journal Information
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/journal-impact" data-track="click" data-track-action="journal metrics" data-track-label="link">
                                    Journal Metrics
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editors" data-track="click" data-track-action="about the editors" data-track-label="link">
                                    About the Editors
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/research-cross-journal-editorial-team" data-track="click" data-track-action="research cross-journal editorial team" data-track-label="link">
                                    Research Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/reviews-cross-journal-editorial-team" data-track="click" data-track-action="reviews cross-journal editorial team" data-track-label="link">
                                    Reviews Cross-Journal Editorial Team
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/statistics-advisory-panel" data-track="click" data-track-action="statistical advisory panel" data-track-label="link">
                                    Statistical Advisory Panel
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/our-publishing-models" data-track="click" data-track-action="our publishing models" data-track-label="link">
                                    Our publishing models
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-values-statement" data-track="click" data-track-action="editorial values statement" data-track-label="link">
                                    Editorial Values Statement
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/editorial-policies" data-track="click" data-track-action="editorial policies" data-track-label="link">
                                    Editorial Policies
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/content" data-track="click" data-track-action="content types" data-track-label="link">
                                    Content Types
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/web-feeds" data-track="click" data-track-action="web feeds" data-track-label="link">
                                    Web Feeds
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/contact" data-track="click" data-track-action="contact" data-track-label="link">
                                    Contact
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                </li>
                                
                                    <li class="c-header__item c-header__item--dropdown-menu" data-test="publish-with-us-button">
                                        <a href="javascript:;" class="c-header__link c-header__link--dropdown-menu" data-header-expander="" data-test="menu-button--publish" data-track="click" data-track-action="open publish with us expander" data-track-label="button" role="button" aria-haspopup="true" aria-expanded="false">
                                            <span>Publish <span class="c-header__show-text">with us</span></span><svg role="img" aria-hidden="true" focusable="false" height="16" viewBox="0 0 16 16" width="16" xmlns="http://www.w3.org/2000/svg"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" transform="matrix(0 1 -1 0 11 3)"></path></svg>
                                        </a><nav class="c-header__dropdown has-tethered u-js-hide" aria-labelledby="Publish-with-us-label" id="publish-with-us" data-test="publish-with-us" data-track-component="nature-150-split-header" hidden="">
                <div class="c-header__container">
                    <h2 id="Publish-with-us-label" class="c-header__heading c-header__heading--js-hide">Publish with us</h2>
                    <ul class="c-header__list c-header__list--js-stack">
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/submission-guidelines" data-track="click" data-track-action="submission guidelines" data-track-label="link">
                                    Submission Guidelines
                                </a>
                            </li>
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/for-reviewers" data-track="click" data-track-action="for reviewers" data-track-label="link">
                                    For Reviewers
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" data-test="nature-author-services" data-track="nav_language_services" data-track-context="header publish with us dropdown menu" data-track-action="manuscript author services" data-track-label="link manuscript author services" href="https://authorservices.springernature.com/go/sn/?utm_source=For+Authors&amp;utm_medium=Website_Nature&amp;utm_campaign=Platform+Experimentation+2022&amp;utm_id=PE2022">
                                    Language editing services
                                </a>
                            </li>
                        
                        
                            <li class="c-header__item">
                                <a class="c-header__link" href="/nm/open-access-funding" data-test="funding-eligibility-link" data-track="click_explore_funding" data-track-context="header publish with us" data-track-action="funding eligibility">Open access funding</a>
                            </li>
                        
                        
                            <li class="c-header__item c-header__item--keyline">
                                <a class="c-header__link" href="https://mts-nmed.nature.com/cgi-bin/main.plex" data-track="click_submit_manuscript" data-track-context="submit link in Nature header dropdown menu" data-track-action="submit manuscript" data-track-label="link (publish with us dropdown menu)" data-track-external="" data-gtm-criteo="submit-manuscript">Submit manuscript<svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m15 0c1.1045695 0 2 .8954305 2 2v5.5c0 .27614237-.2238576.5-.5.5s-.5-.22385763-.5-.5v-5.5c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-9v3c0 1.1045695-.8954305 2-2 2h-3v10c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h7.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-7.5c-1.1045695 0-2-.8954305-2-2v-10.17157288c0-.53043297.21071368-1.0391408.58578644-1.41421356l3.82842712-3.82842712c.37507276-.37507276.88378059-.58578644 1.41421356-.58578644zm-.5442863 8.18867991 3.3545404 3.35454039c.2508994.2508994.2538696.6596433.0035959.909917-.2429543.2429542-.6561449.2462671-.9065387-.0089489l-2.2609825-2.3045251.0010427 7.2231989c0 .3569916-.2898381.6371378-.6473715.6371378-.3470771 0-.6473715-.2852563-.6473715-.6371378l-.0010428-7.2231995-2.2611222 2.3046654c-.2531661.2580415-.6562868.2592444-.9065605.0089707-.24295423-.2429542-.24865597-.6576651.0036132-.9099343l3.3546673-3.35466731c.2509089-.25090888.6612706-.25227691.9135302-.00001728zm-.9557137-3.18867991c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm-8.5-3.587-3.587 3.587h2.587c.55228475 0 1-.44771525 1-1zm8.5 1.587c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill="#fff"></path></svg>
                                </a>
                            </li>
                        
                    </ul>
                </div>
            </nav>
                                    </li>
                                
                            
                            
                        </ul>
                        <ul class="c-header__menu c-header__menu--hide-lg-max">
                            
                                <li class="c-header__item" data-test="alert-link">
                                    <a class="c-header__link" href="https://journal-alerts.springernature.com/subscribe?journal_id=41591" rel="nofollow" data-track="nav_sign_up_for_alerts" data-track-action="Sign up for alerts" data-track-label="link (desktop site header)" data-track-external="">
                                        <span>Sign up for alerts</span><svg role="img" aria-hidden="true" focusable="false" height="18" viewBox="0 0 18 18" width="18" xmlns="http://www.w3.org/2000/svg"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill="#222"></path></svg>
                                    </a>
                                </li>
                            
                            
                                <li class="c-header__item c-header__item--pipe">
                                    <a class="c-header__link" href="https://www.nature.com/nm.rss" data-track="click" data-track-action="rss feed" data-track-label="link">
                                            <span>RSS feed</span>
                                    </a>
                                </li>
                            
                        </ul>
                    </div>
                </div>
            </div>
        
    </header>


    
    
        <nav class="u-mb-16" aria-label="breadcrumbs">
            <div class="u-container">
                <ol class="c-breadcrumbs" itemscope="" itemtype="https://schema.org/BreadcrumbList">
                    <li class="c-breadcrumbs__item" id="breadcrumb0" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature"><span itemprop="name">nature</span></a><meta itemprop="position" content="1">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb1" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:nature medicine"><span itemprop="name">nature medicine</span></a><meta itemprop="position" content="2">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb2" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem"><a class="c-breadcrumbs__link" href="/nm/articles?type=article" itemprop="item" data-track="click" data-track-action="breadcrumb" data-track-category="header" data-track-label="link:articles"><span itemprop="name">articles</span></a><meta itemprop="position" content="3">
                                    <svg class="c-breadcrumbs__chevron" role="img" aria-hidden="true" focusable="false" height="10" viewBox="0 0 10 10" width="10" xmlns="http://www.w3.org/2000/svg">
                                        <path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill="#666" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path>
                                    </svg>
                                </li><li class="c-breadcrumbs__item" id="breadcrumb3" itemprop="itemListElement" itemscope="" itemtype="https://schema.org/ListItem">
                                    <span itemprop="name">article</span><meta itemprop="position" content="4"></li>
                </ol>
            </div>
        </nav>
    



    

</div>


<div class="u-container u-mt-32 u-mb-32 u-clearfix" id="content" data-component="article-container" data-container-type="article">
    <main class="c-article-main-column u-float-left js-main-column" data-track-component="article body">
        
            
                <div class="c-context-bar u-hide" id="js-enable-context-bar" data-test="context-bar" data-context-bar="" aria-hidden="true">
                    <div class="c-context-bar__container u-container" data-track-context="sticky banner">
                        <div class="c-context-bar__title">
                            Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects
                        </div>
                        
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01883-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                    </div>
                </div>
            
        
        <article lang="en">
            
                <div class="c-pdf-button__container u-mb-16 u-hide-at-lg js-context-bar-sticky-point-mobile">
                    <div class="c-pdf-container" data-track-context="article body">
                        
                            
                                
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01883-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                            
                        
                    </div>
                </div>
            
            <div class="c-article-header">
                <header>
                    <ul class="c-article-identifiers" data-test="article-identifier">
                        
        <li class="c-article-identifiers__item" data-test="article-category">Article</li>
    
        <li class="c-article-identifiers__item">
            <a href="https://www.springernature.com/gp/open-research/about/the-fundamentals-of-open-access-and-open-research" data-track="click" data-track-action="open access" data-track-label="link" class="u-color-open-access" data-test="open-access">Open access</a>
        </li>
    
    

                        <li class="c-article-identifiers__item">Published: <time datetime="2022-07-04">04 July 2022</time></li>
                    </ul>

                    <h1 class="c-article-title" data-test="article-title" data-article-title="">Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects</h1>
                    <ul class="c-article-author-list c-article-author-list--short js-no-scroll" data-test="authors-list" data-component-authors-activator="authors-list"><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Neil_A_-Robertson-Aff1" data-author-popup="auth-Neil_A_-Robertson-Aff1" data-author-search="Robertson, Neil A." data-track-context="researcher popup with no profile" data-track-index="1_19">Neil A. Robertson</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-9509-1157"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-9509-1157</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Eric-Latorre_Crespo-Aff1" data-author-popup="auth-Eric-Latorre_Crespo-Aff1" data-author-search="Latorre-Crespo, Eric" data-track-context="researcher popup with no profile" data-track-index="2_19">Eric Latorre-Crespo</a><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Maria-Terradas_Terradas-Aff2-Aff3" data-author-popup="auth-Maria-Terradas_Terradas-Aff2-Aff3" data-author-search="Terradas-Terradas, Maria" data-track-context="researcher popup with no profile" data-track-index="3_19">Maria Terradas-Terradas</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-3823-8968"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-3823-8968</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Jorge-Lemos_Portela-Aff4" data-author-popup="auth-Jorge-Lemos_Portela-Aff4" data-author-search="Lemos-Portela, Jorge" data-track-context="researcher popup with no profile" data-track-index="4_19">Jorge Lemos-Portela</a><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Alison_C_-Purcell-Aff2-Aff3" data-author-popup="auth-Alison_C_-Purcell-Aff2-Aff3" data-author-search="Purcell, Alison C." data-track-context="researcher popup with no profile" data-track-index="5_19">Alison C. Purcell</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-2911-9035"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2911-9035</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Benjamin_J_-Livesey-Aff1" data-author-popup="auth-Benjamin_J_-Livesey-Aff1" data-author-search="Livesey, Benjamin J." data-track-context="researcher popup with no profile" data-track-index="6_19">Benjamin J. Livesey</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6866-1452"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6866-1452</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Robert_F_-Hillary-Aff5" data-author-popup="auth-Robert_F_-Hillary-Aff5" data-author-search="Hillary, Robert F." data-track-context="researcher popup with no profile" data-track-index="7_19">Robert F. Hillary</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-2595-552X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-2595-552X</a></span><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Lee-Murphy-Aff6" data-author-popup="auth-Lee-Murphy-Aff6" data-author-search="Murphy, Lee" data-track-context="researcher popup with no profile" data-track-index="8_19">Lee Murphy</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-6467-7449"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-6467-7449</a></span><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Angie-Fawkes-Aff6" data-author-popup="auth-Angie-Fawkes-Aff6" data-author-search="Fawkes, Angie" data-track-context="researcher popup with no profile" data-track-index="9_19">Angie Fawkes</a><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Louise-MacGillivray-Aff6" data-author-popup="auth-Louise-MacGillivray-Aff6" data-author-search="MacGillivray, Louise" data-track-context="researcher popup with no profile" data-track-index="10_19">Louise MacGillivray</a><sup class="u-js-hide"><a href="#Aff6" tabindex="-1">6</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Mhairi-Copland-Aff7" data-author-popup="auth-Mhairi-Copland-Aff7" data-author-search="Copland, Mhairi" data-track-context="researcher popup with no profile" data-track-index="11_19">Mhairi Copland</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-7655-016X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-7655-016X</a></span><sup class="u-js-hide"><a href="#Aff7" tabindex="-1">7</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Riccardo_E_-Marioni-Aff5" data-author-popup="auth-Riccardo_E_-Marioni-Aff5" data-author-search="Marioni, Riccardo E." data-track-context="researcher popup with no profile" data-track-index="12_19">Riccardo E. Marioni</a><sup class="u-js-hide"><a href="#Aff5" tabindex="-1">5</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Joseph_A_-Marsh-Aff1" data-author-popup="auth-Joseph_A_-Marsh-Aff1" data-author-search="Marsh, Joseph A." data-track-context="researcher popup with no profile" data-track-index="13_19">Joseph A. Marsh</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-4132-0628"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-4132-0628</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Sarah_E_-Harris-Aff8" data-author-popup="auth-Sarah_E_-Harris-Aff8" data-author-search="Harris, Sarah E." data-track-context="researcher popup with no profile" data-track-index="14_19">Sarah E. Harris</a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-4941-5106"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-4941-5106</a></span><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Simon_R_-Cox-Aff8" data-author-popup="auth-Simon_R_-Cox-Aff8" data-author-search="Cox, Simon R." data-track-context="researcher popup with no profile" data-track-index="15_19">Simon R. Cox</a><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Ian_J_-Deary-Aff8" data-author-popup="auth-Ian_J_-Deary-Aff8" data-author-search="Deary, Ian J." data-track-context="researcher popup with no profile" data-track-index="16_19">Ian J. Deary</a><sup class="u-js-hide"><a href="#Aff8" tabindex="-1">8</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Linus_J_-Schumacher-Aff4" data-author-popup="auth-Linus_J_-Schumacher-Aff4" data-author-search="Schumacher, Linus J." data-corresp-id="c1" data-track-context="researcher popup with no profile" data-track-index="17_19">Linus J. Schumacher<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0003-0797-3406"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0003-0797-3406</a></span><sup class="u-js-hide"><a href="#Aff4" tabindex="-1">4</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>, </li><li class="c-article-author-list__item c-article-author-list__item--hide-small-screen"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Kristina-Kirschner-Aff2-Aff3" data-author-popup="auth-Kristina-Kirschner-Aff2-Aff3" data-author-search="Kirschner, Kristina" data-corresp-id="c2" data-track-context="researcher popup with no profile" data-track-index="18_19">Kristina Kirschner<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0001-7607-8670"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0001-7607-8670</a></span><sup class="u-js-hide"><a href="#Aff2" tabindex="-1">2</a>,<a href="#Aff3" tabindex="-1">3</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup> &amp; </li><li class="c-article-author-list__show-more" aria-label="Show all 19 authors for this article" title="Show all 19 authors for this article">…</li><li class="c-article-author-list__item"><a data-test="author-name" data-track="click" data-track-action="open author" data-track-label="link" href="#auth-Tamir-Chandra-Aff1" data-author-popup="auth-Tamir-Chandra-Aff1" data-author-search="Chandra, Tamir" data-corresp-id="c3" data-track-context="researcher popup with no profile" data-track-index="19_19">Tamir Chandra<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-mail-medium"></use></svg></a><span class="u-js-hide">&nbsp;
            <a class="js-orcid" href="http://orcid.org/0000-0002-7935-317X"><span class="u-visually-hidden">ORCID: </span>orcid.org/0000-0002-7935-317X</a></span><sup class="u-js-hide"><a href="#Aff1" tabindex="-1">1</a></sup><sup class="u-js-hide">&nbsp;<a href="#na1" tabindex="-1">na1</a></sup>&nbsp;</li></ul><button aria-expanded="false" class="c-article-author-list__button"><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-down-medium"></use></svg><span>Show authors</span></button>

                    

                    <p class="c-article-info-details" data-container-section="info">
                        
    <a data-test="journal-link" href="/nm" data-track="click" data-track-action="journal homepage" data-track-category="article body" data-track-label="link"><i data-test="journal-title">Nature Medicine</i></a>

                        <b data-test="journal-volume"><span class="u-visually-hidden">volume</span>&nbsp;28</b>,&nbsp;<span class="u-visually-hidden">pages </span>1439–1446 (<span data-test="article-publication-year">2022</span>)<a href="#citeas" class="c-article-info-details__cite-as u-hide-print" data-track="click" data-track-action="cite this article" data-track-label="link">Cite this article</a>
                    </p>
                    
        <div class="c-article-metrics-bar__wrapper u-clear-both">
            <ul class="c-article-metrics-bar u-list-reset">
                
                    <li class=" c-article-metrics-bar__item" data-test="access-count">
                        <p class="c-article-metrics-bar__count">22k <span class="c-article-metrics-bar__label">Accesses</span></p>
                    </li>
                
                
                    <li class="c-article-metrics-bar__item" data-test="citation-count">
                        <p class="c-article-metrics-bar__count">65 <span class="c-article-metrics-bar__label">Citations</span></p>
                    </li>
                
                
                    
                        <li class="c-article-metrics-bar__item" data-test="altmetric-score">
                            <p class="c-article-metrics-bar__count">155 <span class="c-article-metrics-bar__label">Altmetric</span></p>
                        </li>
                    
                
                <li class="c-article-metrics-bar__item">
                    <p class="c-article-metrics-bar__details"><a href="/articles/s41591-022-01883-3/metrics" data-track="click" data-track-action="view metrics" data-track-label="link" rel="nofollow">Metrics <span class="u-visually-hidden">details</span></a></p>
                </li>
            </ul>
        </div>
    
                    
                </header>

                
    

                
    
    

    
    

                
            </div>

        <div class="c-article-body">
            <section aria-labelledby="Abs1" data-title="Abstract" lang="en"><div class="c-article-section" id="Abs1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Abs1">Abstract</h2><div class="c-article-section__content" id="Abs1-content"><p>Clonal hematopoiesis of indeterminate potential (CHIP) increases rapidly in prevalence beyond age 60 and has been associated with increased risk for malignancy, heart disease and ischemic stroke. CHIP is driven by somatic mutations in hematopoietic stem and progenitor cells (HSPCs). Because mutations in HSPCs often drive leukemia, we hypothesized that HSPC fitness substantially contributes to transformation from CHIP to leukemia. HSPC fitness is defined as the proliferative advantage over cells carrying no or only neutral mutations. If mutations in different genes lead to distinct fitness advantages, this could enable patient stratification. We quantified the fitness effects of mutations over 12 years in older age using longitudinal sequencing and developed a filtering method that considers individual mutational context alongside mutation co-occurrence to quantify the growth potential of variants within individuals. We found that gene-specific fitness differences can outweigh inter-individual variation and, therefore, could form the basis for personalized clinical management.</p></div></div></section>

            
                
            

            
                
                    
        
            <section aria-labelledby="inline-recommendations" data-title="Inline Recommendations" class="c-article-recommendations" data-track-component="inline-recommendations">
                <h3 class="c-article-recommendations-title" id="inline-recommendations">Similar content being viewed by others</h3>
                <div class="c-article-recommendations-list">
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-023-05806-1/MediaObjects/41586_2023_5806_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41586-023-05806-1?fromPaywallRec=false" data-track="select_recommendations_1" data-track-context="inline recommendations" data-track-action="click recommendations inline - 1" data-track-label="10.1038/s41586-023-05806-1">Aberrant activation of TCL1A promotes stem cell expansion in clonal haematopoiesis
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                        
                                         <span class="c-article-meta-recommendations__date">12 April 2023</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41588-024-01755-1/MediaObjects/41588_2024_1755_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41588-024-01755-1?fromPaywallRec=false" data-track="select_recommendations_2" data-track-context="inline recommendations" data-track-action="click recommendations inline - 2" data-track-label="10.1038/s41588-024-01755-1">Analysis of somatic mutations in whole blood from 200,618 individuals identifies pervasive positive selection and novel drivers of clonal hematopoiesis
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">14 May 2024</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                        <div class="c-article-recommendations-list__item">
                            <article class="c-article-recommendations-card" itemscope="" itemtype="http://schema.org/ScholarlyArticle">
                                
                                    <div class="c-article-recommendations-card__img"><img src="https://media.springernature.com/w215h120/springer-static/image/art%3A10.1038%2Fs41586-022-04786-y/MediaObjects/41586_2022_4786_Fig1_HTML.png" loading="lazy" alt=""></div>
                                
                                <div class="c-article-recommendations-card__main">
                                    <h3 class="c-article-recommendations-card__heading" itemprop="name headline">
                                        <a class="c-article-recommendations-card__link" itemprop="url" href="https://www.nature.com/articles/s41586-022-04786-y?fromPaywallRec=false" data-track="select_recommendations_3" data-track-context="inline recommendations" data-track-action="click recommendations inline - 3" data-track-label="10.1038/s41586-022-04786-y">Clonal dynamics of haematopoiesis across the human lifespan
                                        </a>
                                    </h3>
                                    <div class="c-article-meta-recommendations" data-test="recommendation-info">
                                        <span class="c-article-meta-recommendations__item-type">Article</span>
                                         <span class="c-article-meta-recommendations__access-type">Open access</span>
                                         <span class="c-article-meta-recommendations__date">01 June 2022</span>
                                    </div>
                                </div>
                            </article>
                        </div>
                    
                </div>
            </section>
        
            <script>
                window.dataLayer = window.dataLayer || [];
                window.dataLayer.push({
                    recommendations: {
                        recommender: 'semantic',
                        model: 'specter',
                        policy_id: 'NA',
                        timestamp: 1749525146,
                        embedded_user: 'null'
                    }
                });
            </script>
        
    
                
                
                <div class="main-content">
                    
                        <section data-title="Main"><div class="c-article-section" id="Sec1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec1">Main</h2><div class="c-article-section__content" id="Sec1-content"><p>Age is the single largest factor underlying the onset of many cancers<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 1" title="de Magalhaes, J. P. How ageing processes influence cancer. Nat. Rev. Cancer 13, 357–365 (2013)." href="/articles/s41591-022-01883-3#ref-CR1" id="ref-link-section-d73177660e750">1</a></sup>. Age-related accumulation and clonal expansion of cancer-associated somatic mutations in healthy tissues has been posited recently as a pre-malignant status consistent with the multi-stage model of carcinogenesis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 2" title="Martincorena, I. Somatic mutation and clonal expansions in human tissues. Genome Med. 11, 35 (2019)." href="/articles/s41591-022-01883-3#ref-CR2" id="ref-link-section-d73177660e754">2</a></sup>. However, the widespread presence of cancer-associated mutations in healthy tissues highlights the complexity of early detection and diagnosis of cancer<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ayachi, S., Buscarlet, M. &amp; Busque, L. 60 years of clonal hematopoiesis research: from X-chromosome inactivation studies to the identification of driver mutations. Exp. Hematol. 83, 2–11 (2020)." href="#ref-CR3" id="ref-link-section-d73177660e758">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)." href="#ref-CR4" id="ref-link-section-d73177660e758_1">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)." href="#ref-CR5" id="ref-link-section-d73177660e758_2">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. Nature 574, 532–537 (2019)." href="#ref-CR6" id="ref-link-section-d73177660e758_3">6</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 7" title="Martincorena, I. et al. High burden and pervasive positive selection of somatic mutations in normal human skin. Science 348, 880–886 (2015)." href="/articles/s41591-022-01883-3#ref-CR7" id="ref-link-section-d73177660e761">7</a></sup>.</p><p>CHIP is defined as the clonal expansion of HSPCs in healthy aged individuals. CHIP affects more than 10% of individuals over the age of 60 years and is associated with an estimated ten-fold increased risk for the later onset of hematological neoplasms<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ayachi, S., Buscarlet, M. &amp; Busque, L. 60 years of clonal hematopoiesis research: from X-chromosome inactivation studies to the identification of driver mutations. Exp. Hematol. 83, 2–11 (2020)." href="#ref-CR3" id="ref-link-section-d73177660e768">3</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)." href="#ref-CR4" id="ref-link-section-d73177660e768_1">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)." href="/articles/s41591-022-01883-3#ref-CR5" id="ref-link-section-d73177660e771">5</a></sup>. There is a clear benefit of detecting CHIP early for close clinical monitoring and early detection, as the association between clone size and malignancy progression is well-established<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)." href="/articles/s41591-022-01883-3#ref-CR5" id="ref-link-section-d73177660e775">5</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Jaiswal, S. &amp; Ebert, B. L. Clonal hematopoiesis in human aging and disease. Science 366, eaan4673 (2019)." href="/articles/s41591-022-01883-3#ref-CR8" id="ref-link-section-d73177660e778">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 9" title="Park, S. J. &amp; Bejar, R. Clonal hematopoiesis in cancer. Exp. Hematol. 83, 105–112 (2020)." href="/articles/s41591-022-01883-3#ref-CR9" id="ref-link-section-d73177660e781">9</a></sup>.</p><p>The particular mechanisms by which common mutations of CHIP—for example, <i>DNMT3A</i> and <i>TET2</i>—contribute to the progression of leukemia are still not understood, which hinders early diagnosis of CHIP on a gene or variant basis<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Jaiswal, S. &amp; Ebert, B. L. Clonal hematopoiesis in human aging and disease. Science 366, eaan4673 (2019)." href="/articles/s41591-022-01883-3#ref-CR8" id="ref-link-section-d73177660e794">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Terradas-Terradas, M., Robertson, N. A., Chandra, T. &amp; Kirschner, K. Clonality in haematopoietic stem cell ageing. Mech. Ageing Dev. 189, 111279 (2020)." href="#ref-CR10" id="ref-link-section-d73177660e797">10</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Challen, G. A. &amp; Goodell, M. A. Clonal hematopoiesis: mechanisms driving dominance of stem cell clones. Blood 136, 1590–1598 (2020)." href="#ref-CR11" id="ref-link-section-d73177660e797_1">11</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 12" title="Shih, A. H., Abdel-Wahab, O., Patel, J. P. &amp; Levine, R. L. The role of mutations in epigenetic regulators in myeloid malignancies. Nat. Rev. Cancer 12, 599–612 (2012)." href="/articles/s41591-022-01883-3#ref-CR12" id="ref-link-section-d73177660e800">12</a></sup>. In clinical practice, CHIP is diagnosed by the presence of somatic mutations at variant allele frequencies (VAFs) of at least 2% in cancer-associated genes, that is in more than 4% of all blood cells<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Jaiswal, S. &amp; Ebert, B. L. Clonal hematopoiesis in human aging and disease. Science 366, eaan4673 (2019)." href="/articles/s41591-022-01883-3#ref-CR8" id="ref-link-section-d73177660e804">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Steensma, D. P. &amp; Bolton, K. L. What to tell your patient with clonal hematopoiesis and why: insights from two specialized clinics. Blood 136, 1623–1631 (2020)." href="/articles/s41591-022-01883-3#ref-CR13" id="ref-link-section-d73177660e807">13</a></sup>. Clonal fitness, defined as the proliferative advantage of stem cells carrying a mutation over cells carrying no or only neutral mutations, has emerged as an alternative clone-specific quantitative marker of CHIP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020)." href="/articles/s41591-022-01883-3#ref-CR14" id="ref-link-section-d73177660e811">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 15" title="Williams, M. J. et al. Measuring the distribution of fitness effects in somatic evolution by combining clonal dynamics with dN/dS ratios. eLife 9, e48714 (2020)." href="/articles/s41591-022-01883-3#ref-CR15" id="ref-link-section-d73177660e814">15</a></sup>. As mutations in stem cells often drive leukemia<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)." href="/articles/s41591-022-01883-3#ref-CR5" id="ref-link-section-d73177660e819">5</a></sup>, we hypothesized that stem cell fitness contributes substantially to transformation from CHIP to leukemia.</p><p>Stratification of individuals to inform close clinical monitoring for early detection or prevention of leukemia in the future will depend on the ability to accurately associate genes and their variants with progression to disease. However, it remains unresolved whether variant-specific or gene-specific fitness effects outweigh other factors contributing to variable progression among individuals, such as environment or genetics.</p><p>Hitherto, fitness effects have been predicted from large cross-sectional cohort data<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020)." href="/articles/s41591-022-01883-3#ref-CR14" id="ref-link-section-d73177660e830">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Abelson, S. et al. Prediction of acute myeloid leukaemia risk in healthy individuals. Nature 559, 400–404 (2018)." href="/articles/s41591-022-01883-3#ref-CR16" id="ref-link-section-d73177660e833">16</a></sup>. In this approach, single-timepoint data from many individuals are pooled to generate allele frequency distributions. Although this method allows the study of a large collection of variants, pooling prevents estimation of an individual’s mutational fitness effects from cross-sectional data. Inferring fitness from a single timepoint creates additional uncertainty about whether a mutation has arisen recently and has grown rapidly (high fitness advantage) or arose a long time ago and has grown slowly (low fitness advantage). With longitudinal samples, fitness effects of individual mutations can be estimated directly from the change in VAF over multiple timepoints.</p><p>In this study, we worked with longitudinal data from the Lothian Birth Cohort of 1921 (LBC1921) and the Lothian Birth Cohort of 1936 (LBC1936)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Taylor, A. M., Pattie, A. &amp; Deary, I. J. Cohort profile update: the Lothian Birth Cohorts of 1921 and 1936. Int. J. Epidemiol. 47, 1042–1042r (2018)." href="/articles/s41591-022-01883-3#ref-CR17" id="ref-link-section-d73177660e840">17</a></sup>. Such longitudinal data are rare worldwide owing to their participants’ older age (70–90 years) and their three-yearly follow-ups over 12 years in each cohort and over 21 years of total timespan. We developed a new framework for extracting fitness effects from longitudinal data using Bayesian inference. First, a likelihood-based filter for time series data (LiFT) allowed us to segregate between sequencing artifacts or naturally drifting populations of cells and fast-growing clones. Second, we inferred the growth potential or fitness effects simultaneously for all growing mutations within each individual and also allowed for clones with multiple mutations if these are favored by Bayesian model comparison. We detected gene-specific fitness effects within our cohorts, highlighting the potential for personalized clinical management.</p></div></div></section><section data-title="Results"><div class="c-article-section" id="Sec2-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec2">Results</h2><div class="c-article-section__content" id="Sec2-content"><h3 class="c-article__sub-heading" id="Sec3">Longitudinal profiling of CHIP variants in advanced age</h3><p>The Lothian Birth Cohorts (LBCs) of 1921 (<i>n</i> = 550) and 1936 (<i>n</i> = 1091) are two independent, longitudinal studies of aging with approximately three-yearly follow-up for five waves, from the age of 70 years (LBC1936) and 79 years (LBC1921)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Taylor, A. M., Pattie, A. &amp; Deary, I. J. Cohort profile update: the Lothian Birth Cohorts of 1921 and 1936. Int. J. Epidemiol. 47, 1042–1042r (2018)." href="/articles/s41591-022-01883-3#ref-CR17" id="ref-link-section-d73177660e862">17</a></sup>. We previously identified 73 participants with CHIP at wave 1 through whole-genome sequencing (WGS)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Robertson, N. A. et al. Age-related clonal haemopoiesis is associated with increased epigenetic age. Curr. Biol. 29, R786–R787 (2019)." href="/articles/s41591-022-01883-3#ref-CR18" id="ref-link-section-d73177660e866">18</a></sup>. Here, we used a targeted error-corrected sequencing approach using a 75-gene panel (ArcherDX/Invitae) to assess longitudinal changes in VAFs and clonal evolution over 21 years across both LBC cohorts (6 years in LBC1921 and 12 years in LBC1936; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">1</a>). Error-corrected sequencing allowed accurate quantification, providing more sensitive clonal outgrowth estimates than our previous WGS data. We sequenced 248 LBC samples (85 individuals across 2–5 timepoints) and achieved a sequencing depth of 2,238× mean coverage (2,153× median) over all targeted sites with an average of 1.6 unique somatic variants (pan-cohort VAF 0.03–87%, median VAF 4.4%) detected per participant. We examined all participant-matched events across the time course: sequence quality control metrics revealed that only seven of 275 data points failed to meet our quality criteria, likely due to low initial VAF. Most of our variant loci generally displayed a high number of supporting reads, with a mean of 258 (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig5">1a</a>).</p><p>For our initial analysis, we retained variants with at least one timepoint at 2% VAF (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">2</a>). <i>DNMT3A</i> was the most commonly mutated CHIP gene (<i>n</i> = 39 events in 33 participants), followed by <i>TET2</i> (<i>n</i> = 18 events in 15 participants), <i>JAK2</i> (<i>n</i> = 8 events in eight participants) and <i>ASXL1</i> (<i>n</i> = 3 events in three participants) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1a–c</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig5">1e</a>). Our mutation spectrum is consistent with previous studies in finding <i>DNMT3A</i> and <i>TET2</i> as the most frequently mutated genes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 4" title="Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. N. Engl. J. Med. 371, 2477–2487 (2014)." href="/articles/s41591-022-01883-3#ref-CR4" id="ref-link-section-d73177660e921">4</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)." href="/articles/s41591-022-01883-3#ref-CR5" id="ref-link-section-d73177660e924">5</a></sup>. We detected some variants more frequently at certain hotspots within a gene, such as R882H in <i>DNMT3A</i>, with previously unreported variants being present as well (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1d–i</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">2</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)." href="/articles/s41591-022-01883-3#ref-CR5" id="ref-link-section-d73177660e938">5</a></sup>. We most frequently detected missense mutations with several other key protein-altering event types ranking highly, including frameshift insertions and deletions and nonsense mutations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1a–c</a>). Participants broadly cluster together across their time course, driven by the expanding or stable VAF of their harbored mutations, underscoring the high prevalence and large clone size of common clonal hematopoietic drivers, namely <i>DNMT3A</i>, <i>TET2</i> and <i>JAK2</i> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1a–c</a>). In the case of <i>JAK2V617F</i>, we identified two individuals who developed leukemia at wave 2 and received treatment between waves 2 and 3, likely driving a clear reduction in clone size (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1h</a>). Those individuals were excluded from further analysis. In our data, we identified a lower frequency of mutations in splicing genes, such as <i>SF3B1</i>, despite the older age of the cohorts (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1a</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig5">1e</a>). This is in contrast to previously published cohort data, where splicing mutations became more prominent with increased age<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 19" title="McKerrell, T. et al. Leukemia-associated somatic mutations drive distinct patterns of age-related clonal hemopoiesis. Cell Rep. 10, 1239–1245 (2015)." href="/articles/s41591-022-01883-3#ref-CR19" id="ref-link-section-d73177660e973">19</a></sup>. Most mutations were missense, frameshift and nonsense mutations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1b</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-1" data-title="Clonal hematopoiesis in the LBCs."><figure><figcaption><b id="Fig1" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 1: Clonal hematopoiesis in the LBCs.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01883-3/figures/1" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig1_HTML.png?as=webp"><img aria-describedby="Fig1" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig1_HTML.png" alt="figure 1" loading="lazy" width="685" height="1071"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-1-desc"><p><b>a</b>, Counts of unique events that exceeded 2% VAF across the range of the longitudinal cohorts in our panel of 75 hematopoietic genes. <b>b</b>, Counts of the functional consequences of the unique events listed in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1a</a>. Missense mutations, frameshift insertions and deletions and nonsense mutations are indicated. Exact counts, <i>n</i>, are for each category. <b>c</b>, Schematic of the top seven most affected genes in the cohort with the largest clone size of an event in any given gene shown. All affected participants were clustered across all timepoints, with the point size scaled by VAF and colored by the functional consequence of the variant (as per Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1b</a> and legend). <b>d</b>, Clone size trajectories of all <i>DNMT3A</i> mutations across the time series in both LBC1921 and LBC1936 colored by the functional consequence of the variant (as per Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1b,c</a>). <b>e</b>, Locations of somatic mutations discovered in <i>DNMT3A</i>. Protein-affecting events are marked and labeled across the structure of the gene (missense in red, truncating in purple, stacked for multiple events) with the structure of the gene labeled along the amino acid length of its protein. <b>f</b>. Clone size trajectories of all <i>TET2</i> mutations across the time series in both LBC1921 and LBC1936 colored by the functional consequence of the variant (as per Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1b,c</a>). <b>g</b>, The locations of somatic mutations in <i>TET2</i>. Protein-affecting events are marked and labeled across the structure of the protein (missense in red, truncating in purple, stacked for multiple events). <b>h</b>, Clone size trajectories of all <i>JAK2</i> mutations across the time series in both LBC1921 and LBC1936 colored by the functional consequence of the variant (as per Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1b,c</a>). Points marked in black denote timepoints after which the affected participant received treatment for leukemia. <b>i</b>, The locations of somatic mutations in <i>JAK2</i>. Protein-affecting events are marked and labeled across the structure of the protein (missense in red, truncating in purple, stacked for multiple events). All eight <i>JAK2</i> mutations are p.Val617Phe (<i>JAK2 V617F</i>) missense variants. del, deltion; FS, frameshift; ins, insertion.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01883-3/figures/1" data-track-dest="link:Figure1 Full size image" aria-label="Full size image figure 1" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Overall, our sequencing approach allowed for high-resolution, longitudinal mapping of CHIP variants over 6-year and 12-year time spans in LBC1921 and LBC1936, respectively, and 21-year time span across both cohorts from the same geographical region and born 9 years apart.</p><h3 class="c-article__sub-heading" id="Sec4">Cataloguing of fitness effects for CHIP variants at &gt;2% VAF</h3><p>Stem cell fitness is defined as the proliferative advantage over cells carrying no or only neutral mutations. It remains incompletely understood to what extent fitness is gene-specific or variant-specific or determined by the bone marrow microenvironment and clonal composition. Earlier estimates suggested a wide spread of fitness effects even for variants of the same gene<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020)." href="/articles/s41591-022-01883-3#ref-CR14" id="ref-link-section-d73177660e1083">14</a></sup>, which would make it difficult to clinically stratify individuals with CHIP. To determine the fitness effects of the variants identified in our cohorts (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1a</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig5">1e</a>), we initially selected all CHIP variants in our data using the commonly used criterion of defining any variants with VAF &gt; 2% as CHIP<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 8" title="Jaiswal, S. &amp; Ebert, B. L. Clonal hematopoiesis in human aging and disease. Science 366, eaan4673 (2019)." href="/articles/s41591-022-01883-3#ref-CR8" id="ref-link-section-d73177660e1093">8</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 13" title="Steensma, D. P. &amp; Bolton, K. L. What to tell your patient with clonal hematopoiesis and why: insights from two specialized clinics. Blood 136, 1623–1631 (2020)." href="/articles/s41591-022-01883-3#ref-CR13" id="ref-link-section-d73177660e1096">13</a></sup> and retaining only those variants with at least two timepoints (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig2">2b</a>). This approach identified 76 CHIP mutations overall (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig2">2c</a>). To estimate the fitness effect that each variant confers, we used Bayesian inference and birth–death models of clonal dynamics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig2">2a</a>), including all trajectories with at least two timepoints (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">3</a>). The resulting fitness values show an overall dependence of fitness on the gene level (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig2">2d</a>), with a wide distribution of fitness for some genes, such as <i>TET2</i> and <i>DNMT3A</i>, but not others, such as <i>JAK2</i> (which are all the same variant).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-2" data-title="Fitness effects of variants at 2% VAF threshold in longitudinal data."><figure><figcaption><b id="Fig2" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 2: Fitness effects of variants at 2% VAF threshold in longitudinal data.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01883-3/figures/2" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig2_HTML.png?as=webp"><img aria-describedby="Fig2" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig2_HTML.png" alt="figure 2" loading="lazy" width="685" height="765"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-2-desc"><p><b>a</b>, Schematic of the mathematical model (top) and workflow (bottom) used to infer the fitness of mutations reaching VAF &gt; 2% during the observed time span. Clonal structure and fitness inference are based on a mathematical model of clonal dynamics (<a data-track="click" data-track-label="link" data-track-action="section anchor" href="/articles/s41591-022-01883-3#Sec8">Methods</a>). HSPCs (top, yellow cells) naturally acquire mutations over time that can be neutral (<i>s</i> = 0, green cell) or increase self-renewal bias (<i>s</i> &gt; 0, brown cell), leading to the formation of genetic clones. Artwork includes images by Servier Medical Art licensed under CC BY 3.0. <b>b</b>, VAF measurement <i>v</i>(<i>t</i><sub>0</sub>) at initial timepoint <i>t</i><sub>0</sub> versus gradient in VAF, <span class="mathjax-tex">\((v(t_{end}) - v(t_0))/(t_{end} - t_0)\)</span>, between initial and last timepoints <i>t</i><sub>0</sub> and <i>t</i><sub><i>end</i></sub> of all variants detected in the LBCs with at least two timepoints. Each data point corresponds to a trajectory in the LBCs and has been colored according to its CHIP status based on the 2% VAF threshold (red box). Blue and orange, respectively, denote whether trajectories achieved a VAF &gt; 2% during the observed time span or not. Note: VAF is displayed on a logarithmic scale, as most mutations are concentrated at low VAF. <b>c</b>, Number of trajectories passing the currently used 2% VAF threshold, broken down into whether VAF is increasing or decreasing from the first to last timepoint. <b>d</b>, Fitness effects of mutations grouped by gene and ranked by median fitness. The posterior probability distribution of the fitness as inferred from our model of clonal dynamics is displayed for each mutation (only the 90% interval is shown). The sample size, <i>n</i>, of observed variants in each gene is denoted in brackets. When more than one mutation is observed in a gene, we further display a box plot showing the median and exclusive interquartile range of the MAP fitness estimates associated with the gene.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01883-3/figures/2" data-track-dest="link:Figure2 Full size image" aria-label="Full size image figure 2" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><h3 class="c-article__sub-heading" id="Sec5">Longitudinal trajectories accurately stratify CHIP variants</h3><p>Because longitudinal data allow direct quantification of the growth in VAF over time, we can inspect the gradients (fluctuations) in VAF for variants that were classified as CHIP based on thresholding. We found that a VAF &gt; 2% threshold not only misses fast-growing and potentially harmful variants (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig2">2b</a>) but can also include variants whose frequencies are shrinking (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig2">2b,c</a>) and, thus, either do not confer a fitness advantage or are being outcompeted by other clones. Overall, 70% of CHIP mutations detected by thresholding at 2% VAF were growing during the observed time span (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig2">2b,c</a>). Longitudinal data, thus, reveal limitations in defining CHIP mutations based on a widely used VAF threshold.</p><p>To overcome the limitations of a threshold-based selection of fit variants, we sought to filter variants based on longitudinal information, by comparing a stochastic model of clonal dynamics with a model of sequencing artifacts (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3a</a>). This novel approach, which we named LiFT, allows classification of fit variants even for VAF &lt; 2%. LiFT classification of fit variants broadly agreed with noise profile statistics from the ArcherDX pipeline (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig6">2f,g</a>) but identified additional variants by leveraging the longitudinal nature of the data. LiFT classification resulted in 114 variant trajectories (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3b–d</a> and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig6">2a–g</a>), 86% of which grew over the observed time span. We note that the VAF of fit mutations may still shrink over time due to the presence of an even fitter clone in the same individual. This is in contrast to thresholding at 2% VAF, with only 70% of variants identified to be growing and, thus, likely to confer a fitness advantage. Of the 114 variants we detected, 50 would not have been detected using the previous VAF threshold filter. We, therefore, recomputed fitness estimates for this new set of fit trajectories (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3e,f</a>). Growing variants that were missed by the traditional filtering method include highly fit variants such as <i>U2AF1</i> Q157R (fitness 33.5%) and <i>DNMT3A</i> R882H (fitness 16%) (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3c,g</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">4</a>). VAF thresholding did not identify any <i>TP53</i> variants. However, LiFT identified four <i>TP53</i> mutations, all of which were growing over the observed time course (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3c,g</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">4</a>). In addition, all of those were either termination/frameshift mutations or previously reported as cancer-associated in the Catalogue of Somatic Mutations in Cancer (COSMIC)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Tate, J. G. et al. COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids Res. 47, D941–D947 (2019)." href="/articles/s41591-022-01883-3#ref-CR20" id="ref-link-section-d73177660e1379">20</a></sup> and classified as likely damaging (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">5</a>). Moreover, all <i>TP53</i> variants led to high fitness effects; thus, our filtering method allows us to identify potentially harmful variants at very low VAFs. Overall, the variants detected by LiFT were of higher fitness than those detected by VAF thresholding (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3f</a>; Kruskal–Wallis H = 14, <i>P</i> = 1 × 10<sup>−4</sup>), with an even larger effect size when comparing variants that are exclusive to each filtering algorithm (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3f</a>; Kruskal–Wallis H = 18, <i>P</i> = 1 × 10<sup>−5</sup>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-3" data-title="LiFT allows classification of fit variants <2% VAF."><figure><figcaption><b id="Fig3" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 3: LiFT allows classification of fit variants &lt;2% VAF.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01883-3/figures/3" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig3_HTML.png?as=webp"><img aria-describedby="Fig3" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig3_HTML.png" alt="figure 3" loading="lazy" width="685" height="831"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-3-desc"><p><b>a</b>, Schematic of LiFT algorithm. LiFT compares a model of clonal dynamics (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1a</a>) with an artifact model and performs Bayesian model selection. The subsequent steps to infer clonal structure and fitness distributions are as in Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1a</a>. <b>b</b>, Gradient in VAF versus VAF for variants detected in the LBCs with at least two timepoints and at least one VAF &gt; 1% per trajectory, with filtered (orange), fit (blue) and synonymous (light green dots) mutations, classified by LiFT on a logarithmic scale. <b>c</b>, Longitudinal trajectories of fit (blue) and filtered (orange) mutations linked to age in years. <b>d</b>, Number of trajectories classified as fit by LiFT, broken down into increasing or decreasing VAF from first to last timepoint. <b>e</b>, Left, deterministic fit of all mutations selected by LiFT in an individual of the LBC cohorts using the inferred optimal clonal structure (Supplementary Information Methods, Appendix B). 90% CIs associated with binomial sampling noise are shown for each data point. VAF is displayed on a logarithmic scale. Right, posterior distribution of fitness associated to each clonal structure. <b>f</b>, Fitness effects of variants broken down by filtering method. The sample size, <i>n</i>, and statistical analyses comparing the distribution of fitness, computed using the non-parametric Kruskal–Wallis test, are highlighted (*H = 14, <i>P</i> = 1 × 10<sup>−4</sup>; **H = 18, <i>P</i> = 1 × 10<sup>−5</sup>). <b>g</b>, Fitness of variants selected as fit by LiFT broken down by their maximum VAF, &gt;2% and &lt;2%, and damage prediction. The top row displays a bar plot of variant counts for each category. The bottom row displays box plots showing the median and interquartile range of the distribution of MAP fitnesses by damaging prediction displayed on a logarithmic scale to emphasize relative differences in fitness between variants. Consequently, of a total of 89 variants with a damage prediction, 17 variants with fitness below 2% are not shown but are reported in Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">4</a>–<a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">6</a>. A marginal plot shows the Gaussian kernel density estimation of the MAP fitness values. fs, frameshifts; ter, terminations.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01883-3/figures/3" data-track-dest="link:Figure3 Full size image" aria-label="Full size image figure 3" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>We further stratified variants using seven computational predictors recently identified as being most useful for identifying pathogenic mutations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Carter, H., Douville, C., Stenson, P. D., Cooper, D. N. &amp; Karchin, R. Identifying Mendelian disease genes with the Variant Effect Scoring Tool. BMC Genomics 14, S3 (2013)." href="#ref-CR21" id="ref-link-section-d73177660e1476">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hecht, M., Bromberg, Y. &amp; Rost, B. Better prediction of functional effects for sequence variants. BMC Genomics 16, S1 (2015)." href="#ref-CR22" id="ref-link-section-d73177660e1476_1">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ioannidis, N. M. et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. Am. J. Hum. Genet 99, 877–885 (2016)." href="#ref-CR23" id="ref-link-section-d73177660e1476_2">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Livesey, B. J. &amp; Marsh, J. A. Using deep mutational scanning to benchmark variant effect predictors and identify disease mutations. Mol. Syst. Biol. 16, e9380 (2020)." href="#ref-CR24" id="ref-link-section-d73177660e1476_3">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Raimondi, D. et al. DEOGEN2: prediction and interactive visualization of single amino acid variant deleteriousness in human proteins. Nucleic Acids Res. 45, W201–W206 (2017)." href="#ref-CR25" id="ref-link-section-d73177660e1476_4">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Riesselman, A. J., Ingraham, J. B. &amp; Marks, D. S. Deep generative models of genetic variation capture the effects of mutations. Nat. Methods 15, 816–822 (2018)." href="#ref-CR26" id="ref-link-section-d73177660e1476_5">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vaser, R., Adusumalli, S., Leng, S. N., Sikic, M. &amp; Ng, P. C. SIFT missense predictions for genomes. Nat. Protoc. 11, 1–9 (2016)." href="/articles/s41591-022-01883-3#ref-CR27" id="ref-link-section-d73177660e1479">27</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3g</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">5</a>), categorizing the most prevalent CHIP variants into likely damaging (21 variants), possibly damaging (20 variants) and likely benign (11 variants) as well as frameshifts and terminations (37 variants, which are also most likely damaging to protein structure and, thus, protein function; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">6</a>). Our novel LiFT algorithm, therefore, produces a low false discovery rate of pathogenic variants, with 88% of the detected fit variants being predicted to be possibly damaging, frameshift or termination.</p><p>Taken together, applying a probabilistic model of clonal dynamics to longitudinal sequencing data results in a novel method—the LiFT algorithm—that improves on the threshold-based definition of CHIP mutations (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3a</a>). The LiFT algorithm replaces an arbitrary cutoff on VAF by a choice of false discovery rate (through a Bayes factor threshold) and, as a result, selects fewer trajectories with shrinking VAF (Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig2">2b,c</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3b–d</a>).</p><h3 class="c-article__sub-heading" id="Sec6">Clinical relevance of LiFT</h3><p>We further analyzed differences in the distributions of fitness between genes using a non-parametric test. Despite having small sample sizes for many genes, we still detected statistically significant differences among the distributions of fitness effects (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig4">4a,b</a>). In particular, we found that mutations in <i>TP53</i>, <i>SF3B1</i> and <i>SRSF2</i> conferred a higher fitness advantage over mutations in commonly mutated CHIP genes, such as <i>JAK2</i> and <i>DNMT3A</i>. We also tested differences in fitness by genes when summarized into functional categories and found trajectories of genes involved in DNA methylation to have lower fitness than genes involved in splicing and genes for transcription factors that are relevant in development (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig7">3a,b</a>).</p><div class="c-article-section__figure js-c-reading-companion-figures-item" data-test="figure" data-container-section="figure" id="figure-4" data-title="Clinical relevance of LiFT."><figure><figcaption><b id="Fig4" class="c-article-section__figure-caption" data-test="figure-caption-text">Fig. 4: Clinical relevance of LiFT.</b></figcaption><div class="c-article-section__figure-content"><div class="c-article-section__figure-item"><a class="c-article-section__figure-link" data-test="img-link" data-track="click" data-track-label="image" data-track-action="view figure" href="/articles/s41591-022-01883-3/figures/4" rel="nofollow"><picture><source type="image/webp" srcset="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig4_HTML.png?as=webp"><img aria-describedby="Fig4" src="//media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig4_HTML.png" alt="figure 4" loading="lazy" width="685" height="640"></picture></a></div><div class="c-article-section__figure-description" data-test="bottom-caption" id="figure-4-desc"><p><b>a</b>, Fitness effects of mutations selected as fit by the LiFT algorithm, grouped by gene and ranked by median fitness. The posterior probability distribution of the fitness as inferred from our model of clonal dynamics is displayed for each mutation (only the 90% interval is shown). The sample size, <i>n</i>, of observed variants in each gene is denoted in brackets. When more than one mutation is observed in a gene, we further display a box plot showing the median and exclusive interquartile range of the MAP estimates of fitness associated with the gene. <b>b</b>, Analysis of variance of the distribution of fitness across genes. Heat map of all statistically significant (<i>P</i> &lt; 0.05) Kruskal–Wallis H statistics, labeled by effect size, computed for all combinations of pairs of genes. The effect size is only shown for statistically significant relations. Variants with a fitness below 2% were left out of this study, as our prediction classifies them as conferring no or a negligible fitness advantage. <b>c</b>, Minimum referral time in years based on 2 standard deviations below the expected growth of a clone given an initial VAF and fitness. Each line shows the initial size of mutation versus referral time for a given fitness. <b>d</b>, Minimum detectable fitness at referral observation based on 2 standard deviations below the expected growth of a clone given an initial VAF and fitness. Each line shows minimum detectable fitness versus referral time for an initial clone size.</p></div></div><div class="u-text-right u-hide-print"><a class="c-article__pill-button" data-test="article-link" data-track="click" data-track-label="button" data-track-action="view figure" href="/articles/s41591-022-01883-3/figures/4" data-track-dest="link:Figure4 Full size image" aria-label="Full size image figure 4" rel="nofollow"><span>Full size image</span><svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-chevron-right-small"></use></svg></a></div></figure></div><p>Differences in the distribution of fitness allow us to predict the future growth of mutations from initial timepoints. For example, if a patient presents with a variant in a gene with 10% fitness at 1% VAF, its growth could be confidently measured after 7 months (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig4">4c</a>), warranting a clinical follow-up over that timeframe to confirm or revise the fitness estimate. Conversely, the time between observations places a lower bound on the fitness that can be measured for mutations of a given VAF (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig4">4d</a>). These data can then inform on the timeframe for close clinical monitoring and early detection of disease.</p><p>Ableson et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 16" title="Abelson, S. et al. Prediction of acute myeloid leukaemia risk in healthy individuals. Nature 559, 400–404 (2018)." href="/articles/s41591-022-01883-3#ref-CR16" id="ref-link-section-d73177660e1583">16</a></sup> compared CHIP carriers who never developed acute myeloid leukemia (AML) with CHIP where individuals subsequently developed AML, and they found that the number of mutations, the mutational burden and the size of the larger driver clone were associated with the risk of progression to AML. In the present study, we carried out a survival analysis to correlate the maximum observed VAF of mutations and survival. This correlation was stronger in the older cohort (LBC1921) although not statistically significant (hazard ratio (HR) = 1.35; 95% confidence interval (CI) 0.83, 2.19; <i>P</i> = 0.23) due to the small sample size (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig7">3d</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">7</a>). In the younger cohort (LBC1936), we found that survival better correlated with the speed of growth of a mutation, although this was, again, not statistically significant (HR = 1.35; 95% CI 0.76, 2.4; <i>P</i> = 0.3) (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig7">3d</a> and Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">7</a>).</p><p>Notably, only two timepoints are necessary to apply LiFT, making this a widely applicable method for existing cohorts and future studies (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig7">3c</a>). We propose the use of LiFT over thresholding for clinical practice.</p></div></div></section><section data-title="Discussion"><div class="c-article-section" id="Sec7-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec7">Discussion</h2><div class="c-article-section__content" id="Sec7-content"><p>The clinical potential for stratifying progression of CHIP depends on whether genes confer distinct fitness advantages. Indeed, most studies so far have not shown a clear distinction of fitness effects on a gene basis and have shown considerable overlap in fitness coefficients among variants of different genes. We show that fitness can substantially differ by gene and gene category. Combining longitudinal data with a new method to identify CHIP variants allows for more accurate fitness estimates of CHIP than cross-sectional cohort data and motivates further studies with increased sample sizes.</p><p>Our fitness estimates are independent of the time when the mutation was acquired. In cross-sectional studies, fitness estimates are generally (inversely) correlated with the mutation rate, introducing additional uncertainty<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020)." href="/articles/s41591-022-01883-3#ref-CR14" id="ref-link-section-d73177660e1624">14</a></sup>. In contrast, our fitness estimates are based on the observed growth among longitudinal samples and, thus, also take into account other mutations in an individual. The resulting fitness estimates are largely independent of hematopoietic stem cell absolute numbers (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig8">4b,c</a>).</p><p>The strength of our approach, combining longitudinal data with our LiFT algorithm, is exemplified by <i>U2AF1</i> and <i>TP53</i>, for which no variants were identified by a 2% VAF threshold (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig2">2b,c</a>). In contrast, our LiFT method identified one <i>U2AF1</i> and four <i>TP53</i> variants, all of which are conferring a fitness advantage, scored as possibly damaging in our missense variant effect analysis and have been previously reported in COSMIC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Tate, J. G. et al. COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids Res. 47, D941–D947 (2019)." href="/articles/s41591-022-01883-3#ref-CR20" id="ref-link-section-d73177660e1650">20</a></sup> (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig3">3g</a> and Supplementary Tables <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">4</a> and <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">5</a>). Moreover, we pick up the <i>DNMT3A R88H</i> variant with LiFT but not with 2% VAF thresholding—a mutation that is well-reported in the context of leukemia<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 28" title="Ley, T. J. et al. DNMT3A mutations in acute myeloid leukemia. N. Engl. J. Med. 363, 2424–2433 (2010)." href="/articles/s41591-022-01883-3#ref-CR28" id="ref-link-section-d73177660e1666">28</a></sup>. Therefore, for patients with those variants, close clinical monitoring for early detection of disease such as leukemia is merited.</p><p>Combining longitudinal data with LiFT enables a personalized approach managing CHIP (Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig9">5</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig10">6</a>). Longitudinal data allow quantifying fitness effects even for mutations not seen in large cohorts, as cross-sectional fitness estimation requires a mutation to be observed in multiple individuals. Our method offers clinicians a way forward for patient stratification even for unique variants occurring in single individuals, because two timepoints for one individual suffice to estimate fitness, including uncertainty quantification (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig4">4e</a>). We have provided a prediction of the time required between first and second observations to be able to accurately infer fitness, depending on the initial VAF of a mutation in an individual (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig4">4c</a>). For high fitness mutations (&gt;10%), a follow-up clinical observation could be performed after only a few months, even for small clones (1% VAF or less). Conversely, the time between observations places a lower bound on the fitness that can be measured for mutations of a given VAF (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig4">4d</a>). In the future, these data can be used to inform time to the next appointment for close clinical monitoring of patients with clones containing highly fit variants, which will likely outcompete other clones. Using longitudinal data to better quantify and predict clonal progression in our study, however, comes with a tradeoff in the lower number of participants in our cohort and limits the power of cross-sectional analysis to find associations.</p><p>In addition, our inference method aims to resolve the clonal composition of multiple mutations in an individual. Specifically, we can now infer the likely co-occurrence of mutations from longitudinal data. Current cross-sectional studies do not take into account the clonal composition of individuals and, therefore, make predictions of the isolated effect of a mutation. In contrast, we are able to link fitness to clones carrying a specific combination of mutations that is unique to each individual, without relying on any prior knowledge of variant-specific fitness effects (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">4</a>).</p></div></div></section><section data-title="Methods"><div class="c-article-section" id="Sec8-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec8">Methods</h2><div class="c-article-section__content" id="Sec8-content"><h3 class="c-article__sub-heading" id="Sec9">Participant samples and ethics</h3><p>This study complies with all relevant ethical regulations. The study protocol was approved by NHS Lothian (formerly Lothian Health). Informed consent was given by all participants. Ethics permission for LBC1936 was obtained from the Multi-Centre Research Ethics Committee for Scotland (wave 1: MREC/01/0/56), the Lothian Research Ethics Committee (wave 1: LREC/2003/2/29) and the Scotland A Research Ethics Committee (waves 2, 3, 4 and 5: 07/MRE00/58). Ethics permission for LBC1921 was obtained from the Lothian Research Ethics Committee (wave 1: LREC/1998/4/183; wave 2: LREC/2003/7/23; wave 3: 1702/98/4/183) and the Scotland A Research Ethics Committee (waves 4 and 5: 10/MRE00/87).</p><p>LBC1921 contains a total of 550 participants at wave 1 of their testing (performed between 1999 and 2001) with a gender ratio of 234:316 (male:female) and a mean age at wave 1 of 79.1 years (s.d. = 0.6) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">1</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Taylor, A. M., Pattie, A. &amp; Deary, I. J. Cohort profile update: the Lothian Birth Cohorts of 1921 and 1936. Int. J. Epidemiol. 47, 1042–1042r (2018)." href="/articles/s41591-022-01883-3#ref-CR17" id="ref-link-section-d73177660e1713">17</a></sup>. LBC1936 contains a total of 1,091 participants at wave 1 of their testing (performed between 2004 and 2007) with a gender ratio of 548:543 (male:female) and a mean age at wave 1 of 69.5 years (s.d. = 0.8) (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">1</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 17" title="Taylor, A. M., Pattie, A. &amp; Deary, I. J. Cohort profile update: the Lothian Birth Cohorts of 1921 and 1936. Int. J. Epidemiol. 47, 1042–1042r (2018)." href="/articles/s41591-022-01883-3#ref-CR17" id="ref-link-section-d73177660e1720">17</a></sup>. We previously identified 73 participants with CHIP at wave 1 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 18" title="Robertson, N. A. et al. Age-related clonal haemopoiesis is associated with increased epigenetic age. Curr. Biol. 29, R786–R787 (2019)." href="/articles/s41591-022-01883-3#ref-CR18" id="ref-link-section-d73177660e1724">18</a></sup>). We sequenced DNA from those 73 LBC participants using a targeted gene panel (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">8</a>) and added 16 LBC participants with previously unidentified CHIP and 4–5 timepoints. We have accepted 85 of 89 participants for inclusion in our study, removing four participants for failing to meet quality criteria (low library complexity), with a total of 248 samples together with 14 ‘Genome in a Bottle’ (GIAB) controls, two per sequencing batch (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">9</a>)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Zook, J. M. et al. Extensive sequencing of seven human genomes to characterize benchmark reference materials. Sci. Data 3, 160025 (2016)." href="/articles/s41591-022-01883-3#ref-CR29" id="ref-link-section-d73177660e1735">29</a></sup>. In addition, two individuals carrying the <i>JAK2V617F</i> mutation received treatment for leukemia after the first respective timepoint available, potentially driving the observed reductions in clone size. Those patients were omitted from further analysis after sequencing (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig1">1h</a>).</p><h3 class="c-article__sub-heading" id="Sec10">Targeted, error-corrected sequencing and data filtering</h3><p>DNA was extracted from Ethylenediaminetetraacetic acid (EDTA) whole blood using the Nucleon BACC3 kit (Sigma-Aldrich, GERPN8512), following the manufacturer’s instructions. Libraries were prepared from 200 ng of each DNA sample using the Archer VariantPlex® 75 Myeloid gene panel and VariantPlex® Somatic Protocol for Illumina sequencing (Invitae, AB0108, and VariantPlex®-HGC Myeloid Kit for Illumina; Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">9</a>), including modifications for detecting low allele frequencies. Sequencing of each pool was performed using the NextSeq 500/550 High-Output version 2.5 (300 cycle) kit on the NextSeq 550 platform (Illumina). To inform reproducibility, background model for error and batch correction, we sequenced two GIAB DNA samples in each batch of samples (DNA NA12878, Coriell Institute)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 29" title="Zook, J. M. et al. Extensive sequencing of seven human genomes to characterize benchmark reference materials. Sci. Data 3, 160025 (2016)." href="/articles/s41591-022-01883-3#ref-CR29" id="ref-link-section-d73177660e1756">29</a></sup>.</p><p>Reads were filtered for phred ≥30 and adapters removed using Trimmomatic (version 0.27)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 30" title="Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. Bioinformatics 30, 2114–2120 (2014)." href="/articles/s41591-022-01883-3#ref-CR30" id="ref-link-section-d73177660e1763">30</a></sup> before undergoing guided alignment to human genome assembly hg19 using bwa-mem (version 0.7.17)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 31" title="Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. Bioinformatics 25, 1754–1760 (2009)." href="/articles/s41591-022-01883-3#ref-CR31" id="ref-link-section-d73177660e1767">31</a></sup> and bowtie2 (version 2.2.1)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 32" title="Langmead, B., Trapnell, C., Pop, M. &amp; Salzberg, S. L. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. Genome Biol. 10, R25 (2009)." href="/articles/s41591-022-01883-3#ref-CR32" id="ref-link-section-d73177660e1771">32</a></sup>. Unique molecular barcodes (ligated before PCR amplification) were used for read de-duplication to support quantitative multiplexed analysis and confident mutation detection. Within targeted regions, variants were called using three tools (Lofreq (version 2.1.0)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 33" title="Wilm, A. et al. LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets. Nucleic Acids Res. 40, 11189–11201 (2012)." href="/articles/s41591-022-01883-3#ref-CR33" id="ref-link-section-d73177660e1775">33</a></sup>, Freebayes<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 34" title="Garrison, E. &amp; Marth, G. Haplotype-based variant detection from short-read sequencing. Preprint at 
                  https://arxiv.org/abs/1207.3907
                  
                 (2012)." href="/articles/s41591-022-01883-3#ref-CR34" id="ref-link-section-d73177660e1779">34</a></sup> and Vision (ArcherDX version 6.2.7, unpublished)), building a consensus from the output of all callers (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">2</a>).</p><p>All filtered variants at 2% VAF met the following criteria: (1) the number of reads supporting the alternative allele surpasses the coverage criteria while exhibiting no directional biases (AO ≥ 5, UAO ≥ 3); (2) variants are significantly underrepresented in the Genome Aggregation Database (gnomAD; <i>P</i> ≤ 0.05)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 35" title="Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. Nature 581, 434–443 (2020)." href="/articles/s41591-022-01883-3#ref-CR35" id="ref-link-section-d73177660e1793">35</a></sup>; (3) variants are not obviously germline variants (stable VAF across all waves ~0.5 or ~1) that may have been underrepresented in the gnomAD due to the narrow geographical origin of the LBC participants; and (4) contain events that are overrepresented across the dataset—generally frameshift duplications and deletions—whose reads share some sequence homology to target regions yet are likely misaligned artifact from the capture method (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">2</a>). In addition, we manually curated this list, checking for variants that were previously reported, as per Jaiswal et al.<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 5" title="Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. N. Engl. J. Med. 371, 2488–2498 (2014)." href="/articles/s41591-022-01883-3#ref-CR5" id="ref-link-section-d73177660e1800">5</a></sup>, in COSMIC<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 20" title="Tate, J. G. et al. COSMIC: the Catalogue Of Somatic Mutations In Cancer. Nucleic Acids Res. 47, D941–D947 (2019)." href="/articles/s41591-022-01883-3#ref-CR20" id="ref-link-section-d73177660e1804">20</a></sup> or in the published literature (Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">10</a>). Finally, for any variant that surpassed the above criteria at VAF ≥ 2% across the measured time period, we included other participant-matched data points regardless of VAF level (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig5">1a,b</a>).</p><p>To further mitigate against the diverse sources of noise that can occur in any sequencing experiment, which can become especially problematic when attempting to detect variants at low VAFs, the ArcherDX variant-calling platform leverages the pan-dataset coverage levels of each sample and the GIAB controls to establish a position-specific noise profile and, thus, ascertain the limit of detection (LOD) for each variant discovered in our panel. Here, we report two parameters for each variant: (1) the minimal detectable allele fraction (95% MDAF; Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig5">1c</a>), which describes the minimum VAF that a variant can be detected in our data, in essence describing the LOD for each event; and (2) the VAF outlier <i>P</i> value, which denotes the probability that any variant call could have been generated by sequencing noise given the position-specific noise distribution across our GIAB controls and the pan-dataset coverage levels of our samples, thus allowing us to discern overrepresented sequencing artifacts from real events (Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig5">1d</a>).</p><h3 class="c-article__sub-heading" id="Sec11">Computational prediction of missense variant effects</h3><p>To predict which missense variants are most likely to be damaging, we used seven computational variant effect predictors recently identified as being most useful for identifying pathogenic mutations<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Carter, H., Douville, C., Stenson, P. D., Cooper, D. N. &amp; Karchin, R. Identifying Mendelian disease genes with the Variant Effect Scoring Tool. BMC Genomics 14, S3 (2013)." href="#ref-CR21" id="ref-link-section-d73177660e1835">21</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Hecht, M., Bromberg, Y. &amp; Rost, B. Better prediction of functional effects for sequence variants. BMC Genomics 16, S1 (2015)." href="#ref-CR22" id="ref-link-section-d73177660e1835_1">22</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Ioannidis, N. M. et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. Am. J. Hum. Genet 99, 877–885 (2016)." href="#ref-CR23" id="ref-link-section-d73177660e1835_2">23</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Livesey, B. J. &amp; Marsh, J. A. Using deep mutational scanning to benchmark variant effect predictors and identify disease mutations. Mol. Syst. Biol. 16, e9380 (2020)." href="#ref-CR24" id="ref-link-section-d73177660e1835_3">24</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Raimondi, D. et al. DEOGEN2: prediction and interactive visualization of single amino acid variant deleteriousness in human proteins. Nucleic Acids Res. 45, W201–W206 (2017)." href="#ref-CR25" id="ref-link-section-d73177660e1835_4">25</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" title="Riesselman, A. J., Ingraham, J. B. &amp; Marks, D. S. Deep generative models of genetic variation capture the effects of mutations. Nat. Methods 15, 816–822 (2018)." href="#ref-CR26" id="ref-link-section-d73177660e1835_5">26</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 27" title="Vaser, R., Adusumalli, S., Leng, S. N., Sikic, M. &amp; Ng, P. C. SIFT missense predictions for genomes. Nat. Protoc. 11, 1–9 (2016)." href="/articles/s41591-022-01883-3#ref-CR27" id="ref-link-section-d73177660e1838">27</a></sup>. Specifically, for each variant identified in this study, we determined what fraction of previously identified pathogenic and likely pathogenic missense variants from ClinVar and what fraction of variants observed in the human population from gnomAD version 2.1 for each computational predictor. We then averaged these fractions across all predictors. Note that DeepSequence<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 26" title="Riesselman, A. J., Ingraham, J. B. &amp; Marks, D. S. Deep generative models of genetic variation capture the effects of mutations. Nat. Methods 15, 816–822 (2018)." href="/articles/s41591-022-01883-3#ref-CR26" id="ref-link-section-d73177660e1842">26</a></sup> was not run for all proteins due to its computational intensiveness and difficulty of running on long protein sequences. We also performed predictions of missense variant (de)stabilization using FoldX 5.0, using the experimentally determined protein structure, if available, and the AlphaFold model<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 36" title="Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. Nature 596, 583–589 (2021)." href="/articles/s41591-022-01883-3#ref-CR36" id="ref-link-section-d73177660e1846">36</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 37" title="Varadi, M. et al. AlphaFold Protein Structure Database: massively expanding the structural coverage of protein-sequence space with high-accuracy models. Nucleic Acids Res. 50, D439–D444 (2022)." href="/articles/s41591-022-01883-3#ref-CR37" id="ref-link-section-d73177660e1849">37</a></sup>.</p><h3 class="c-article__sub-heading" id="Sec12">Mathematical model of clonal dynamics to infer fitness</h3><p>Given the longitudinal nature of this study, we can use the probabilistic solution of an established minimal model of cell division<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 14" title="Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. Science 367, 1449–1454 (2020)." href="/articles/s41591-022-01883-3#ref-CR14" id="ref-link-section-d73177660e1861">14</a>,<a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 38" title="Till, J. E., Mcculloch, E. A. &amp; Siminovitch, L. A stochastic model of stem cell proliferation, based on the growth of spleen colony-forming cells. Proc. Natl Acad. Sci. USA 51, 29–36 (1964)." href="/articles/s41591-022-01883-3#ref-CR38" id="ref-link-section-d73177660e1864">38</a></sup> to infer the parameter distribution resulting in the observed time evolution of VAF trajectories in a participant’s genetic profile (Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig2">2a</a>). For each individual, we simultaneously estimated the fitness of variants as well as the size of the stem cell pool, without needing to estimate the time of mutation acquisition.</p><p>In this model, cells exist in two states: stem cells (SCs) or differentiated cells (DCs). Under the assumption that DCs cannot revert to a SC state, differentiation inevitably leads to cell death and is treated as such. Furthermore, assuming that each SC produces the same amount of fully differentiated blood cells allows a direct comparison between the VAF of a variant as observed in blood samples and the number of SCs forming the genetic clone (clone size). For an individual with a collection of clones <span class="mathjax-tex">\(\{ c_i\} _{i \in I}\)</span>, the VAF evolution in time <i>v</i><sub><i>i</i></sub>(<i>t</i>) of a clone <i>c</i><sub><i>i</i></sub> corresponds to <span class="mathjax-tex">\(v_i(t) = \frac{{n_i(t)}}{{2N(t)}}\)</span>, where <i>v</i><sub><i>i</i></sub>(<i>t</i>) is the VAF of the variant at time <i>t</i>; <i>n</i><sub><i>i</i></sub>(<i>t</i>) is the number of SCs carrying the variant; and <i>N</i>(<i>t</i>) corresponds to the total number of diploid HSPCs present in the individual. Finally, we assume that <span class="mathjax-tex">\(N(t) = N_w + \mathop {\sum}\nolimits_{i \in I} {n_i(t)}\)</span>, where <i>N</i><sub><i>w</i></sub> is the average number of wild-type HSPCs in the individual. The bias toward self-renewal of symmetric divisions is parameterized by parameter <i>s</i> and determines the fitness advantage of a clone. In normal hematopoiesis, <i>s</i> = 0, in which case clones undergo neutral drift. For clones with non-neutral (fitness-increasing) mutations, <i>s</i> &gt; 0, and this average clone size grows exponentially in time as <span class="mathjax-tex">\(e^{s(t - t_0)}\)</span> from an initial population of one SC at the time of mutation acquisition <i>t</i><sub>0</sub>. The full distribution of clone sizes is well-approximated by a negative binomial distribution matching the mean (exponential growth) and variance of the full stochastic solution (Supplementary Information Methods, section 1, and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig8">4a</a>). Because the model dynamics are Markovian (without memory), once we condition on a previously observed timepoint in a trajectory, the prediction for all future times is independent of <i>t</i><sub>0</sub>. From the predicted clone size distributions, we can infer the marginal posterior distribution of parameter <i>s</i> using Bayes’ theorem (Supplementary Information Methods, section 3)<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 39" title="Bayes, T. An essay towards solving a problem in the doctrine of chances. By the late Rev. Mr. Bayes, F. R. S. communicated by Mr. Price, in a letter to John Canton, A. M. F. R. S. 
                  https://doi.org/10.1098/rstl.1763.0053
                  
                 (1763)." href="/articles/s41591-022-01883-3#ref-CR39" id="ref-link-section-d73177660e2244">39</a></sup>. We further take into account the sampling error during sequencing to estimate the distribution of clone sizes at the start and end of each time interval in the longitudinal sequencing data. Here, we approximate this sampling error as binomial.</p><p>When multiple fit clones are present in an individual, we constrain the inference to share the SC pool size <i>N</i>(<i>t</i>) for all variant trajectories in this individual. This increases the data:parameter ratio and produces richer dynamics, where the evolution of exponentially growing clones can be suppressed by the growth of a fitter clone. This implies that even non-competitive models, where trajectories grow independently of each other, will result in competitive dynamics in the observed VAF trajectories as variants strive for dominance of the total production of blood cells.</p><p>We take into account possible clonal substructures for all fit variants in an individual, selecting models with co-occurring mutations on the same clone if they are more likely after biasing against models with multiple mutations per clone, as these are presumed to be rarer (Supplementary Information Methods, section 2.4.7). The evidence supporting the optimal clonal structure, determined by Bayesian model comparison, relative to the model assuming no mutations co-occur on the same clone is shown in Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig8">4d</a>. We then infer the posterior fitness distributions per clone for the most likely clonal model in every participant.</p><p>Once we have inferred the posterior distributions of the parameters, we use the mode of the distribution (maximum a posteriori (MAP) estimate) for each mutation to visualize the deterministic—that is, average—growth curves. These result in the logistic time evolution of its corresponding VAF,</p><div id="Equa" class="c-article-equation"><div class="c-article-equation__content"><span class="mathjax-tex">$$v(t) = \frac{1}{{2 + 2N_we^{ - s(t - t_0)}}},$$</span></div></div><p>where we determine the time of mutation acquisition <i>t</i><sub>0</sub>, which is used only for plotting, using maximum likelihood (Supplementary Information Methods, Appendix B). Although deterministic fits are not a direct reflection of the inference results of our stochastic model, these can be used to visually assess the ‘goodness of fit’ of the fitness MAP estimates and have been included for each participant in LBC1921 and LBC1936, respectively, in Extended Data Figs. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig9">5</a> and <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig10">6</a>.</p><p>Note that this model cannot account for loss-of-heterozygosity events.</p><h3 class="c-article__sub-heading" id="Sec13">LiFT</h3><p>To select fit variants, we compare the likelihood of the clonal model, including binomial sampling error, to a model of sequencing artifacts. The artifact model assumes that all variability arises from sampling error with a proportion that remains constant over time. For variants that occur more than once in our dataset, we use a beta-binomial model to account for overdispersion, and, for unique variants, we use a binomial model. We select variants as fit only if the model evidence for the clonal model is at least four times that of the artifact model (Supplementary Information Methods, section 2.4, and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig6">2c,d</a>). Fit variants thus selected are taken through to clonal structure model selection and fitness inference as described above.</p><h3 class="c-article__sub-heading" id="Sec14">Workflow overview</h3><p>A workflow chart describing the full pipeline and implementation guidance is included in the GitHub repository (see ‘Code availability’). Our pipeline can be applied to other datasets with a few adjustments. Our LiFT algorithm has been tailored to the LBC dataset by extracting parameters from the distribution of synonymous mutation reads, which inform the priors used for our Bayesian inference method (Supplementary Information Methods, section <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM1">2.3.3</a>, and Extended Data Fig. <a data-track="click" data-track-label="link" data-track-action="figure anchor" href="/articles/s41591-022-01883-3#Fig6">2a–c</a>). Guidance on how to adapt our LiFT algorithm to other datasets is included in the code repository. All other parts of the pipeline, including the extraction of variants using ArcherDx software and the inference of clonal structures and fitness, are directly applicable to other datasets.</p><h3 class="c-article__sub-heading" id="Sec15">Framework implementation</h3><p>Both LiFT and Bayesian inference of the posterior distribution of model parameters were implemented in Python version 3.7 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Van Rossum, G. &amp; Drake, F. L. Python 3 Reference Manual. 
                  https://dl.acm.org/doi/book/10.5555/1593511
                  
                 (CreateSpace, 2009)." href="/articles/s41591-022-01883-3#ref-CR40" id="ref-link-section-d73177660e2422">40</a></sup>) with dependencies on Numpy version 1.21.5 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 41" title="Harris, C. R. et al. Array programming with NumPy. Nature 585, 357–362 (2020)." href="/articles/s41591-022-01883-3#ref-CR41" id="ref-link-section-d73177660e2426">41</a></sup>), Scipy version 1.7.3 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 42" title="Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. Nat. Methods 17, 261–272 (2020)." href="/articles/s41591-022-01883-3#ref-CR42" id="ref-link-section-d73177660e2430">42</a></sup>) and Pandas version 1.3.4. Survival analysis was implemented using Python version 3.7 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Van Rossum, G. &amp; Drake, F. L. Python 3 Reference Manual. 
                  https://dl.acm.org/doi/book/10.5555/1593511
                  
                 (CreateSpace, 2009)." href="/articles/s41591-022-01883-3#ref-CR40" id="ref-link-section-d73177660e2434">40</a></sup>) with dependencies on lifelines version 0.26.4 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 43" title="Davidson-Pilon, C. lifelines: survival analysis in Python. J. Open Source Softw. 4, 1317 (2019)." href="/articles/s41591-022-01883-3#ref-CR43" id="ref-link-section-d73177660e2438">43</a></sup>). Data curation was undertaken in Python version 3.7 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 40" title="Van Rossum, G. &amp; Drake, F. L. Python 3 Reference Manual. 
                  https://dl.acm.org/doi/book/10.5555/1593511
                  
                 (CreateSpace, 2009)." href="/articles/s41591-022-01883-3#ref-CR40" id="ref-link-section-d73177660e2443">40</a></sup>) and R base<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 44" title="R Core Team. R: a language and environment for statistical computing. 
                  https://www.R-project.org/
                  
                 (R Foundation for Statistical Computing, 2021)." href="/articles/s41591-022-01883-3#ref-CR44" id="ref-link-section-d73177660e2447">44</a></sup>, with use of the ‘tidyverse’<sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 45" title="Wickham, H. et al. Welcome to the tidyverse. J. Open Source Softw. 4, 1686 (2019)." href="/articles/s41591-022-01883-3#ref-CR45" id="ref-link-section-d73177660e2451">45</a></sup> suite of packages and plotted with ggplot2 (ref. <sup><a data-track="click" data-track-action="reference anchor" data-track-label="link" data-test="citation-ref" aria-label="Reference 46" title="Wickham, H. ggplot2: Elegant Graphics for Data Analysis. 
                  https://ggplot2.tidyverse.org
                  
                 (Springer-Verlag, 2016)." href="/articles/s41591-022-01883-3#ref-CR46" id="ref-link-section-d73177660e2455">46</a></sup>).</p><h3 class="c-article__sub-heading" id="Sec16">Reporting summary</h3><p>Further information on research design is available in the <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM2">Nature Research Reporting Summary</a> linked to this article.</p></div></div></section>
                    
                </div>
            

            <div class="u-mt-32">
                <section data-title="Data availability"><div class="c-article-section" id="data-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="data-availability">Data availability</h2><div class="c-article-section__content" id="data-availability-content">
              
              <p>We have deposited all data pertinent to this analysis, including the de-identified raw FASTQ read data and processed variant calls for our longitudinal cohort, onto the National Center of Biotechnology Information Gene Expression Omnibus under accession ID <a href="https://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE178936">GSE178936</a>. LBC phenotypic data are available in the database of Genotypes and Phenotypes (dbGAP) under accession number <a href="https://www.ncbi.nlm.nih.gov/projects/gap/cgi-bin/study.cgi?study_id=phs000821.v1.p1">phs000821.v1.p1</a>. All other Lothian Birth Cohort data are deposited in dbGAP or are provided via the LBC Data Access Collaboration (<a href="https://www.ed.ac.uk/lothian-birth-cohorts/data-access-collaboration">https://www.ed.ac.uk/lothian-birth-cohorts/data-access-collaboration</a>). Information concerning the cohort is contained here, including its history, data summary tables for both LBC1921 and LBC1936 and data access request forms and contact information to obtain all data points (contact: <a href="https://www.ed.ac.uk/profile/simon-cox">https://www.ed.ac.uk/profile/simon-cox</a>, simon.cox@ed.ac.uk; timeframe: 1 month to respond).</p>
            </div></div></section><section data-title="Code availability"><div class="c-article-section" id="code-availability-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="code-availability">Code availability</h2><div class="c-article-section__content" id="code-availability-content">
              
              <p>All code used in this manuscript is available at <a href="https://github.com/neilrobertson/LBC_ARCHER">https://github.com/neilrobertson/LBC_ARCHER</a>.</p>
            </div></div></section><div id="MagazineFulltextArticleBodySuffix"><section aria-labelledby="Bib1" data-title="References"><div class="c-article-section" id="Bib1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Bib1">References</h2><div class="c-article-section__content" id="Bib1-content"><div data-container-section="references"><ol class="c-article-references" data-track-component="outbound reference" data-track-context="references section"><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="1."><p class="c-article-references__text" id="ref-CR1">de Magalhaes, J. P. How ageing processes influence cancer. <i>Nat. Rev. Cancer</i> <b>13</b>, 357–365 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nrc3497" data-track-item_id="10.1038/nrc3497" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrc3497" aria-label="Article reference 1" data-doi="10.1038/nrc3497">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23612461" aria-label="PubMed reference 1">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 1" href="http://scholar.google.com/scholar_lookup?&amp;title=How%20ageing%20processes%20influence%20cancer&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3497&amp;volume=13&amp;pages=357-365&amp;publication_year=2013&amp;author=Magalhaes%2CJP">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="2."><p class="c-article-references__text" id="ref-CR2">Martincorena, I. Somatic mutation and clonal expansions in human tissues. <i>Genome Med.</i> <b>11</b>, 35 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/s13073-019-0648-4" data-track-item_id="10.1186/s13073-019-0648-4" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/s13073-019-0648-4" aria-label="Article reference 2" data-doi="10.1186/s13073-019-0648-4">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31138277" aria-label="PubMed reference 2">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6537184" aria-label="PubMed Central reference 2">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 2" href="http://scholar.google.com/scholar_lookup?&amp;title=Somatic%20mutation%20and%20clonal%20expansions%20in%20human%20tissues&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-019-0648-4&amp;volume=11&amp;publication_year=2019&amp;author=Martincorena%2CI">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="3."><p class="c-article-references__text" id="ref-CR3">Ayachi, S., Buscarlet, M. &amp; Busque, L. 60 years of clonal hematopoiesis research: from X-chromosome inactivation studies to the identification of driver mutations. <i>Exp. Hematol.</i> <b>83</b>, 2–11 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.exphem.2020.01.008" data-track-item_id="10.1016/j.exphem.2020.01.008" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.exphem.2020.01.008" aria-label="Article reference 3" data-doi="10.1016/j.exphem.2020.01.008">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsValsbo%3D" aria-label="CAS reference 3">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32001340" aria-label="PubMed reference 3">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 3" href="http://scholar.google.com/scholar_lookup?&amp;title=60%20years%20of%20clonal%20hematopoiesis%20research%3A%20from%20X-chromosome%20inactivation%20studies%20to%20the%20identification%20of%20driver%20mutations&amp;journal=Exp.%20Hematol.&amp;doi=10.1016%2Fj.exphem.2020.01.008&amp;volume=83&amp;pages=2-11&amp;publication_year=2020&amp;author=Ayachi%2CS&amp;author=Buscarlet%2CM&amp;author=Busque%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="4."><p class="c-article-references__text" id="ref-CR4">Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. <i>N. Engl. J. Med.</i> <b>371</b>, 2477–2487 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa1409405" data-track-item_id="10.1056/NEJMoa1409405" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa1409405" aria-label="Article reference 4" data-doi="10.1056/NEJMoa1409405">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25426838" aria-label="PubMed reference 4">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4290021" aria-label="PubMed Central reference 4">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 4" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20hematopoiesis%20and%20blood-cancer%20risk%20inferred%20from%20blood%20DNA%20sequence&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1409405&amp;volume=371&amp;pages=2477-2487&amp;publication_year=2014&amp;author=Genovese%2CG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="5."><p class="c-article-references__text" id="ref-CR5">Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. <i>N. Engl. J. Med.</i> <b>371</b>, 2488–2498 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa1408617" data-track-item_id="10.1056/NEJMoa1408617" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa1408617" aria-label="Article reference 5" data-doi="10.1056/NEJMoa1408617">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25426837" aria-label="PubMed reference 5">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4306669" aria-label="PubMed Central reference 5">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 5" href="http://scholar.google.com/scholar_lookup?&amp;title=Age-related%20clonal%20hematopoiesis%20associated%20with%20adverse%20outcomes&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1408617&amp;volume=371&amp;pages=2488-2498&amp;publication_year=2014&amp;author=Jaiswal%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="6."><p class="c-article-references__text" id="ref-CR6">Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. <i>Nature</i> <b>574</b>, 532–537 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-019-1672-7" data-track-item_id="10.1038/s41586-019-1672-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-019-1672-7" aria-label="Article reference 6" data-doi="10.1038/s41586-019-1672-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVagsb3J" aria-label="CAS reference 6">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31645730" aria-label="PubMed reference 6">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 6" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20landscape%20of%20somatic%20mutation%20in%20normal%20colorectal%20epithelial%20cells&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1672-7&amp;volume=574&amp;pages=532-537&amp;publication_year=2019&amp;author=Lee-Six%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="7."><p class="c-article-references__text" id="ref-CR7">Martincorena, I. et al. High burden and pervasive positive selection of somatic mutations in normal human skin. <i>Science</i> <b>348</b>, 880–886 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aaa6806" data-track-item_id="10.1126/science.aaa6806" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aaa6806" aria-label="Article reference 7" data-doi="10.1126/science.aaa6806">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXosFaktbg%3D" aria-label="CAS reference 7">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25999502" aria-label="PubMed reference 7">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4471149" aria-label="PubMed Central reference 7">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 7" href="http://scholar.google.com/scholar_lookup?&amp;title=High%20burden%20and%20pervasive%20positive%20selection%20of%20somatic%20mutations%20in%20normal%20human%20skin&amp;journal=Science&amp;doi=10.1126%2Fscience.aaa6806&amp;volume=348&amp;pages=880-886&amp;publication_year=2015&amp;author=Martincorena%2CI">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="8."><p class="c-article-references__text" id="ref-CR8">Jaiswal, S. &amp; Ebert, B. L. Clonal hematopoiesis in human aging and disease. <i>Science</i> <b>366</b>, eaan4673 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aan4673" data-track-item_id="10.1126/science.aan4673" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aan4673" aria-label="Article reference 8" data-doi="10.1126/science.aan4673">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXitV2rt7nL" aria-label="CAS reference 8">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31672865" aria-label="PubMed reference 8">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8050831" aria-label="PubMed Central reference 8">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 8" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20hematopoiesis%20in%20human%20aging%20and%20disease&amp;journal=Science&amp;doi=10.1126%2Fscience.aan4673&amp;volume=366&amp;publication_year=2019&amp;author=Jaiswal%2CS&amp;author=Ebert%2CBL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="9."><p class="c-article-references__text" id="ref-CR9">Park, S. J. &amp; Bejar, R. Clonal hematopoiesis in cancer. <i>Exp. Hematol.</i> <b>83</b>, 105–112 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.exphem.2020.02.001" data-track-item_id="10.1016/j.exphem.2020.02.001" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.exphem.2020.02.001" aria-label="Article reference 9" data-doi="10.1016/j.exphem.2020.02.001">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXjs12qtbc%3D" aria-label="CAS reference 9">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32044376" aria-label="PubMed reference 9">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103485" aria-label="PubMed Central reference 9">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 9" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20hematopoiesis%20in%20cancer&amp;journal=Exp.%20Hematol.&amp;doi=10.1016%2Fj.exphem.2020.02.001&amp;volume=83&amp;pages=105-112&amp;publication_year=2020&amp;author=Park%2CSJ&amp;author=Bejar%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="10."><p class="c-article-references__text" id="ref-CR10">Terradas-Terradas, M., Robertson, N. A., Chandra, T. &amp; Kirschner, K. Clonality in haematopoietic stem cell ageing. <i>Mech. Ageing Dev.</i> <b>189</b>, 111279 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.mad.2020.111279" data-track-item_id="10.1016/j.mad.2020.111279" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.mad.2020.111279" aria-label="Article reference 10" data-doi="10.1016/j.mad.2020.111279">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Wrs7jM" aria-label="CAS reference 10">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32526214" aria-label="PubMed reference 10">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7347006" aria-label="PubMed Central reference 10">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 10" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonality%20in%20haematopoietic%20stem%20cell%20ageing&amp;journal=Mech.%20Ageing%20Dev.&amp;doi=10.1016%2Fj.mad.2020.111279&amp;volume=189&amp;publication_year=2020&amp;author=Terradas-Terradas%2CM&amp;author=Robertson%2CNA&amp;author=Chandra%2CT&amp;author=Kirschner%2CK">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="11."><p class="c-article-references__text" id="ref-CR11">Challen, G. A. &amp; Goodell, M. A. Clonal hematopoiesis: mechanisms driving dominance of stem cell clones. <i>Blood</i> <b>136</b>, 1590–1598 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitF2nt77L" aria-label="CAS reference 11">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32746453" aria-label="PubMed reference 11">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7530644" aria-label="PubMed Central reference 11">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 11" href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20hematopoiesis%3A%20mechanisms%20driving%20dominance%20of%20stem%20cell%20clones&amp;journal=Blood&amp;volume=136&amp;pages=1590-1598&amp;publication_year=2020&amp;author=Challen%2CGA&amp;author=Goodell%2CMA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="12."><p class="c-article-references__text" id="ref-CR12">Shih, A. H., Abdel-Wahab, O., Patel, J. P. &amp; Levine, R. L. The role of mutations in epigenetic regulators in myeloid malignancies. <i>Nat. Rev. Cancer</i> <b>12</b>, 599–612 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nrc3343" data-track-item_id="10.1038/nrc3343" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnrc3343" aria-label="Article reference 12" data-doi="10.1038/nrc3343">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1Wkt7zF" aria-label="CAS reference 12">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22898539" aria-label="PubMed reference 12">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 12" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20mutations%20in%20epigenetic%20regulators%20in%20myeloid%20malignancies&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3343&amp;volume=12&amp;pages=599-612&amp;publication_year=2012&amp;author=Shih%2CAH&amp;author=Abdel-Wahab%2CO&amp;author=Patel%2CJP&amp;author=Levine%2CRL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="13."><p class="c-article-references__text" id="ref-CR13">Steensma, D. P. &amp; Bolton, K. L. What to tell your patient with clonal hematopoiesis and why: insights from two specialized clinics. <i>Blood</i> <b>136</b>, 1623–1631 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32736381" aria-label="PubMed reference 13">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7530645" aria-label="PubMed Central reference 13">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 13" href="http://scholar.google.com/scholar_lookup?&amp;title=What%20to%20tell%20your%20patient%20with%20clonal%20hematopoiesis%20and%20why%3A%20insights%20from%20two%20specialized%20clinics&amp;journal=Blood&amp;volume=136&amp;pages=1623-1631&amp;publication_year=2020&amp;author=Steensma%2CDP&amp;author=Bolton%2CKL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="14."><p class="c-article-references__text" id="ref-CR14">Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. <i>Science</i> <b>367</b>, 1449–1454 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1126/science.aay9333" data-track-item_id="10.1126/science.aay9333" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1126%2Fscience.aay9333" aria-label="Article reference 14" data-doi="10.1126/science.aay9333">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslymtb4%3D" aria-label="CAS reference 14">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32217721" aria-label="PubMed reference 14">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 14" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20evolutionary%20dynamics%20and%20fitness%20landscape%20of%20clonal%20hematopoiesis&amp;journal=Science&amp;doi=10.1126%2Fscience.aay9333&amp;volume=367&amp;pages=1449-1454&amp;publication_year=2020&amp;author=Watson%2CCJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="15."><p class="c-article-references__text" id="ref-CR15">Williams, M. J. et al. Measuring the distribution of fitness effects in somatic evolution by combining clonal dynamics with dN/dS ratios. <i>eLife</i> <b>9</b>, e48714 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.7554/eLife.48714" data-track-item_id="10.7554/eLife.48714" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.7554%2FeLife.48714" aria-label="Article reference 15" data-doi="10.7554/eLife.48714">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslWiu77K" aria-label="CAS reference 15">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32223898" aria-label="PubMed reference 15">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7105384" aria-label="PubMed Central reference 15">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 15" href="http://scholar.google.com/scholar_lookup?&amp;title=Measuring%20the%20distribution%20of%20fitness%20effects%20in%20somatic%20evolution%20by%20combining%20clonal%20dynamics%20with%20dN%2FdS%20ratios&amp;journal=eLife&amp;doi=10.7554%2FeLife.48714&amp;volume=9&amp;publication_year=2020&amp;author=Williams%2CMJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="16."><p class="c-article-references__text" id="ref-CR16">Abelson, S. et al. Prediction of acute myeloid leukaemia risk in healthy individuals. <i>Nature</i> <b>559</b>, 400–404 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-018-0317-6" data-track-item_id="10.1038/s41586-018-0317-6" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-018-0317-6" aria-label="Article reference 16" data-doi="10.1038/s41586-018-0317-6">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjs7rI" aria-label="CAS reference 16">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29988082" aria-label="PubMed reference 16">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6485381" aria-label="PubMed Central reference 16">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 16" href="http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20acute%20myeloid%20leukaemia%20risk%20in%20healthy%20individuals&amp;journal=Nature&amp;doi=10.1038%2Fs41586-018-0317-6&amp;volume=559&amp;pages=400-404&amp;publication_year=2018&amp;author=Abelson%2CS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="17."><p class="c-article-references__text" id="ref-CR17">Taylor, A. M., Pattie, A. &amp; Deary, I. J. Cohort profile update: the Lothian Birth Cohorts of 1921 and 1936. <i>Int. J. Epidemiol.</i> <b>47</b>, 1042–1042r (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/ije/dyy022" data-track-item_id="10.1093/ije/dyy022" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fije%2Fdyy022" aria-label="Article reference 17" data-doi="10.1093/ije/dyy022">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29546429" aria-label="PubMed reference 17">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6124629" aria-label="PubMed Central reference 17">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 17" href="http://scholar.google.com/scholar_lookup?&amp;title=Cohort%20profile%20update%3A%20the%20Lothian%20Birth%20Cohorts%20of%201921%20and%201936&amp;journal=Int.%20J.%20Epidemiol.&amp;doi=10.1093%2Fije%2Fdyy022&amp;volume=47&amp;pages=1042-1042r&amp;publication_year=2018&amp;author=Taylor%2CAM&amp;author=Pattie%2CA&amp;author=Deary%2CIJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="18."><p class="c-article-references__text" id="ref-CR18">Robertson, N. A. et al. Age-related clonal haemopoiesis is associated with increased epigenetic age. <i>Curr. Biol.</i> <b>29</b>, R786–R787 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.cub.2019.07.011" data-track-item_id="10.1016/j.cub.2019.07.011" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.cub.2019.07.011" aria-label="Article reference 18" data-doi="10.1016/j.cub.2019.07.011">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1alt7nO" aria-label="CAS reference 18">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31430471" aria-label="PubMed reference 18">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 18" href="http://scholar.google.com/scholar_lookup?&amp;title=Age-related%20clonal%20haemopoiesis%20is%20associated%20with%20increased%20epigenetic%20age&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2019.07.011&amp;volume=29&amp;pages=R786-R787&amp;publication_year=2019&amp;author=Robertson%2CNA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="19."><p class="c-article-references__text" id="ref-CR19">McKerrell, T. et al. Leukemia-associated somatic mutations drive distinct patterns of age-related clonal hemopoiesis. <i>Cell Rep.</i> <b>10</b>, 1239–1245 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.celrep.2015.02.005" data-track-item_id="10.1016/j.celrep.2015.02.005" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.celrep.2015.02.005" aria-label="Article reference 19" data-doi="10.1016/j.celrep.2015.02.005">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXjs1Omt7o%3D" aria-label="CAS reference 19">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25732814" aria-label="PubMed reference 19">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4542313" aria-label="PubMed Central reference 19">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 19" href="http://scholar.google.com/scholar_lookup?&amp;title=Leukemia-associated%20somatic%20mutations%20drive%20distinct%20patterns%20of%20age-related%20clonal%20hemopoiesis&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2015.02.005&amp;volume=10&amp;pages=1239-1245&amp;publication_year=2015&amp;author=McKerrell%2CT">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="20."><p class="c-article-references__text" id="ref-CR20">Tate, J. G. et al. COSMIC: the Catalogue Of Somatic Mutations In Cancer. <i>Nucleic Acids Res.</i> <b>47</b>, D941–D947 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gky1015" data-track-item_id="10.1093/nar/gky1015" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgky1015" aria-label="Article reference 20" data-doi="10.1093/nar/gky1015">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Cgt7bM" aria-label="CAS reference 20">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30371878" aria-label="PubMed reference 20">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 20" href="http://scholar.google.com/scholar_lookup?&amp;title=COSMIC%3A%20the%20Catalogue%20Of%20Somatic%20Mutations%20In%20Cancer&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky1015&amp;volume=47&amp;pages=D941-D947&amp;publication_year=2019&amp;author=Tate%2CJG">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="21."><p class="c-article-references__text" id="ref-CR21">Carter, H., Douville, C., Stenson, P. D., Cooper, D. N. &amp; Karchin, R. Identifying Mendelian disease genes with the Variant Effect Scoring Tool. <i>BMC Genomics</i> <b>14</b>, S3 (2013).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/1471-2164-14-S3-S3" data-track-item_id="10.1186/1471-2164-14-S3-S3" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/1471-2164-14-S3-S3" aria-label="Article reference 21" data-doi="10.1186/1471-2164-14-S3-S3">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23819870" aria-label="PubMed reference 21">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3665549" aria-label="PubMed Central reference 21">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 21" href="http://scholar.google.com/scholar_lookup?&amp;title=Identifying%20Mendelian%20disease%20genes%20with%20the%20Variant%20Effect%20Scoring%20Tool&amp;journal=BMC%20Genomics&amp;doi=10.1186%2F1471-2164-14-S3-S3&amp;volume=14&amp;publication_year=2013&amp;author=Carter%2CH&amp;author=Douville%2CC&amp;author=Stenson%2CPD&amp;author=Cooper%2CDN&amp;author=Karchin%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="22."><p class="c-article-references__text" id="ref-CR22">Hecht, M., Bromberg, Y. &amp; Rost, B. Better prediction of functional effects for sequence variants. <i>BMC Genomics</i> <b>16</b>, S1 (2015).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/1471-2164-16-S8-S1" data-track-item_id="10.1186/1471-2164-16-S8-S1" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/1471-2164-16-S8-S1" aria-label="Article reference 22" data-doi="10.1186/1471-2164-16-S8-S1">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26110438" aria-label="PubMed reference 22">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4480835" aria-label="PubMed Central reference 22">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 22" href="http://scholar.google.com/scholar_lookup?&amp;title=Better%20prediction%20of%20functional%20effects%20for%20sequence%20variants&amp;journal=BMC%20Genomics&amp;doi=10.1186%2F1471-2164-16-S8-S1&amp;volume=16&amp;publication_year=2015&amp;author=Hecht%2CM&amp;author=Bromberg%2CY&amp;author=Rost%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="23."><p class="c-article-references__text" id="ref-CR23">Ioannidis, N. M. et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. <i>Am. J. Hum. Genet</i> <b>99</b>, 877–885 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1016/j.ajhg.2016.08.016" data-track-item_id="10.1016/j.ajhg.2016.08.016" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1016%2Fj.ajhg.2016.08.016" aria-label="Article reference 23" data-doi="10.1016/j.ajhg.2016.08.016">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFGrurnM" aria-label="CAS reference 23">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27666373" aria-label="PubMed reference 23">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5065685" aria-label="PubMed Central reference 23">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 23" href="http://scholar.google.com/scholar_lookup?&amp;title=REVEL%3A%20an%20ensemble%20method%20for%20predicting%20the%20pathogenicity%20of%20rare%20missense%20variants&amp;journal=Am.%20J.%20Hum.%20Genet&amp;doi=10.1016%2Fj.ajhg.2016.08.016&amp;volume=99&amp;pages=877-885&amp;publication_year=2016&amp;author=Ioannidis%2CNM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="24."><p class="c-article-references__text" id="ref-CR24">Livesey, B. J. &amp; Marsh, J. A. Using deep mutational scanning to benchmark variant effect predictors and identify disease mutations. <i>Mol. Syst. Biol.</i> <b>16</b>, e9380 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.15252/msb.20199380" data-track-item_id="10.15252/msb.20199380" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.15252%2Fmsb.20199380" aria-label="Article reference 24" data-doi="10.15252/msb.20199380">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFWqurvN" aria-label="CAS reference 24">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32627955" aria-label="PubMed reference 24">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7336272" aria-label="PubMed Central reference 24">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 24" href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20deep%20mutational%20scanning%20to%20benchmark%20variant%20effect%20predictors%20and%20identify%20disease%20mutations&amp;journal=Mol.%20Syst.%20Biol.&amp;doi=10.15252%2Fmsb.20199380&amp;volume=16&amp;publication_year=2020&amp;author=Livesey%2CBJ&amp;author=Marsh%2CJA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="25."><p class="c-article-references__text" id="ref-CR25">Raimondi, D. et al. DEOGEN2: prediction and interactive visualization of single amino acid variant deleteriousness in human proteins. <i>Nucleic Acids Res.</i> <b>45</b>, W201–W206 (2017).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkx390" data-track-item_id="10.1093/nar/gkx390" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkx390" aria-label="Article reference 25" data-doi="10.1093/nar/gkx390">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1amtrg%3D" aria-label="CAS reference 25">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28498993" aria-label="PubMed reference 25">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5570203" aria-label="PubMed Central reference 25">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 25" href="http://scholar.google.com/scholar_lookup?&amp;title=DEOGEN2%3A%20prediction%20and%20interactive%20visualization%20of%20single%20amino%20acid%20variant%20deleteriousness%20in%20human%20proteins&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkx390&amp;volume=45&amp;pages=W201-W206&amp;publication_year=2017&amp;author=Raimondi%2CD">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="26."><p class="c-article-references__text" id="ref-CR26">Riesselman, A. J., Ingraham, J. B. &amp; Marks, D. S. Deep generative models of genetic variation capture the effects of mutations. <i>Nat. Methods</i> <b>15</b>, 816–822 (2018).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41592-018-0138-4" data-track-item_id="10.1038/s41592-018-0138-4" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-018-0138-4" aria-label="Article reference 26" data-doi="10.1038/s41592-018-0138-4">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslymt77M" aria-label="CAS reference 26">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30250057" aria-label="PubMed reference 26">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6693876" aria-label="PubMed Central reference 26">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 26" href="http://scholar.google.com/scholar_lookup?&amp;title=Deep%20generative%20models%20of%20genetic%20variation%20capture%20the%20effects%20of%20mutations&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-018-0138-4&amp;volume=15&amp;pages=816-822&amp;publication_year=2018&amp;author=Riesselman%2CAJ&amp;author=Ingraham%2CJB&amp;author=Marks%2CDS">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="27."><p class="c-article-references__text" id="ref-CR27">Vaser, R., Adusumalli, S., Leng, S. N., Sikic, M. &amp; Ng, P. C. SIFT missense predictions for genomes. <i>Nat. Protoc.</i> <b>11</b>, 1–9 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/nprot.2015.123" data-track-item_id="10.1038/nprot.2015.123" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fnprot.2015.123" aria-label="Article reference 27" data-doi="10.1038/nprot.2015.123">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFKht7fF" aria-label="CAS reference 27">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26633127" aria-label="PubMed reference 27">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 27" href="http://scholar.google.com/scholar_lookup?&amp;title=SIFT%20missense%20predictions%20for%20genomes&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fnprot.2015.123&amp;volume=11&amp;pages=1-9&amp;publication_year=2016&amp;author=Vaser%2CR&amp;author=Adusumalli%2CS&amp;author=Leng%2CSN&amp;author=Sikic%2CM&amp;author=Ng%2CPC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="28."><p class="c-article-references__text" id="ref-CR28">Ley, T. J. et al. <i>DNMT3A</i> mutations in acute myeloid leukemia. <i>N. Engl. J. Med.</i> <b>363</b>, 2424–2433 (2010).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1056/NEJMoa1005143" data-track-item_id="10.1056/NEJMoa1005143" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1056%2FNEJMoa1005143" aria-label="Article reference 28" data-doi="10.1056/NEJMoa1005143">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsF2ltb7K" aria-label="CAS reference 28">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21067377" aria-label="PubMed reference 28">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3201818" aria-label="PubMed Central reference 28">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 28" href="http://scholar.google.com/scholar_lookup?&amp;title=DNMT3A%20mutations%20in%20acute%20myeloid%20leukemia&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1005143&amp;volume=363&amp;pages=2424-2433&amp;publication_year=2010&amp;author=Ley%2CTJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="29."><p class="c-article-references__text" id="ref-CR29">Zook, J. M. et al. Extensive sequencing of seven human genomes to characterize benchmark reference materials. <i>Sci. Data</i> <b>3</b>, 160025 (2016).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/sdata.2016.25" data-track-item_id="10.1038/sdata.2016.25" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fsdata.2016.25" aria-label="Article reference 29" data-doi="10.1038/sdata.2016.25">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC28XpvVWrtLk%3D" aria-label="CAS reference 29">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27271295" aria-label="PubMed reference 29">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4896128" aria-label="PubMed Central reference 29">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 29" href="http://scholar.google.com/scholar_lookup?&amp;title=Extensive%20sequencing%20of%20seven%20human%20genomes%20to%20characterize%20benchmark%20reference%20materials&amp;journal=Sci.%20Data&amp;doi=10.1038%2Fsdata.2016.25&amp;volume=3&amp;publication_year=2016&amp;author=Zook%2CJM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="30."><p class="c-article-references__text" id="ref-CR30">Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. <i>Bioinformatics</i> <b>30</b>, 2114–2120 (2014).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btu170" data-track-item_id="10.1093/bioinformatics/btu170" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu170" aria-label="Article reference 30" data-doi="10.1093/bioinformatics/btu170">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Sqt7nP" aria-label="CAS reference 30">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24695404" aria-label="PubMed reference 30">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590" aria-label="PubMed Central reference 30">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 30" href="http://scholar.google.com/scholar_lookup?&amp;title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu170&amp;volume=30&amp;pages=2114-2120&amp;publication_year=2014&amp;author=Bolger%2CAM&amp;author=Lohse%2CM&amp;author=Usadel%2CB">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="31."><p class="c-article-references__text" id="ref-CR31">Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. <i>Bioinformatics</i> <b>25</b>, 1754–1760 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/bioinformatics/btp324" data-track-item_id="10.1093/bioinformatics/btp324" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp324" aria-label="Article reference 31" data-doi="10.1093/bioinformatics/btp324">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D" aria-label="CAS reference 31">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19451168" aria-label="PubMed reference 31">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234" aria-label="PubMed Central reference 31">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 31" href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp324&amp;volume=25&amp;pages=1754-1760&amp;publication_year=2009&amp;author=Li%2CH&amp;author=Durbin%2CR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="32."><p class="c-article-references__text" id="ref-CR32">Langmead, B., Trapnell, C., Pop, M. &amp; Salzberg, S. L. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. <i>Genome Biol.</i> <b>10</b>, R25 (2009).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="noopener" data-track-label="10.1186/gb-2009-10-3-r25" data-track-item_id="10.1186/gb-2009-10-3-r25" data-track-value="article reference" data-track-action="article reference" href="https://link.springer.com/doi/10.1186/gb-2009-10-3-r25" aria-label="Article reference 32" data-doi="10.1186/gb-2009-10-3-r25">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19261174" aria-label="PubMed reference 32">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2690996" aria-label="PubMed Central reference 32">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 32" href="http://scholar.google.com/scholar_lookup?&amp;title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fgb-2009-10-3-r25&amp;volume=10&amp;publication_year=2009&amp;author=Langmead%2CB&amp;author=Trapnell%2CC&amp;author=Pop%2CM&amp;author=Salzberg%2CSL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="33."><p class="c-article-references__text" id="ref-CR33">Wilm, A. et al. LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets. <i>Nucleic Acids Res.</i> <b>40</b>, 11189–11201 (2012).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gks918" data-track-item_id="10.1093/nar/gks918" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgks918" aria-label="Article reference 33" data-doi="10.1093/nar/gks918">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVCitLjJ" aria-label="CAS reference 33">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23066108" aria-label="PubMed reference 33">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3526318" aria-label="PubMed Central reference 33">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 33" href="http://scholar.google.com/scholar_lookup?&amp;title=LoFreq%3A%20a%20sequence-quality%20aware%2C%20ultra-sensitive%20variant%20caller%20for%20uncovering%20cell-population%20heterogeneity%20from%20high-throughput%20sequencing%20datasets&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgks918&amp;volume=40&amp;pages=11189-11201&amp;publication_year=2012&amp;author=Wilm%2CA">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="34."><p class="c-article-references__text" id="ref-CR34">Garrison, E. &amp; Marth, G. Haplotype-based variant detection from short-read sequencing. Preprint at <a href="https://arxiv.org/abs/1207.3907" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://arxiv.org/abs/1207.3907">https://arxiv.org/abs/1207.3907</a> (2012).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="35."><p class="c-article-references__text" id="ref-CR35">Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. <i>Nature</i> <b>581</b>, 434–443 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-020-2308-7" data-track-item_id="10.1038/s41586-020-2308-7" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-2308-7" aria-label="Article reference 35" data-doi="10.1038/s41586-020-2308-7">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVanu7jF" aria-label="CAS reference 35">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32461654" aria-label="PubMed reference 35">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334197" aria-label="PubMed Central reference 35">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 35" href="http://scholar.google.com/scholar_lookup?&amp;title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141%2C456%20humans&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2308-7&amp;volume=581&amp;pages=434-443&amp;publication_year=2020&amp;author=Karczewski%2CKJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="36."><p class="c-article-references__text" id="ref-CR36">Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <i>Nature</i> <b>596</b>, 583–589 (2021).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-021-03819-2" data-track-item_id="10.1038/s41586-021-03819-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-021-03819-2" aria-label="Article reference 36" data-doi="10.1038/s41586-021-03819-2">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVaktrrL" aria-label="CAS reference 36">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34265844" aria-label="PubMed reference 36">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8371605" aria-label="PubMed Central reference 36">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 36" href="http://scholar.google.com/scholar_lookup?&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03819-2&amp;volume=596&amp;pages=583-589&amp;publication_year=2021&amp;author=Jumper%2CJ">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="37."><p class="c-article-references__text" id="ref-CR37">Varadi, M. et al. AlphaFold Protein Structure Database: massively expanding the structural coverage of protein-sequence space with high-accuracy models. <i>Nucleic Acids Res.</i> <b>50</b>, D439–D444 (2022).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1093/nar/gkab1061" data-track-item_id="10.1093/nar/gkab1061" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1093%2Fnar%2Fgkab1061" aria-label="Article reference 37" data-doi="10.1093/nar/gkab1061">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1Churw%3D" aria-label="CAS reference 37">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34791371" aria-label="PubMed reference 37">PubMed</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 37" href="http://scholar.google.com/scholar_lookup?&amp;title=AlphaFold%20Protein%20Structure%20Database%3A%20massively%20expanding%20the%20structural%20coverage%20of%20protein-sequence%20space%20with%20high-accuracy%20models&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkab1061&amp;volume=50&amp;pages=D439-D444&amp;publication_year=2022&amp;author=Varadi%2CM">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="38."><p class="c-article-references__text" id="ref-CR38">Till, J. E., Mcculloch, E. A. &amp; Siminovitch, L. A stochastic model of stem cell proliferation, based on the growth of spleen colony-forming cells. <i>Proc. Natl Acad. Sci. USA</i> <b>51</b>, 29–36 (1964).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1073/pnas.51.1.29" data-track-item_id="10.1073/pnas.51.1.29" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1073%2Fpnas.51.1.29" aria-label="Article reference 38" data-doi="10.1073/pnas.51.1.29">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:STN:280:DyaF2c%2Fmt1artw%3D%3D" aria-label="CAS reference 38">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14104600" aria-label="PubMed reference 38">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC300599" aria-label="PubMed Central reference 38">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 38" href="http://scholar.google.com/scholar_lookup?&amp;title=A%20stochastic%20model%20of%20stem%20cell%20proliferation%2C%20based%20on%20the%20growth%20of%20spleen%20colony-forming%20cells&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.51.1.29&amp;volume=51&amp;pages=29-36&amp;publication_year=1964&amp;author=Till%2CJE&amp;author=Mcculloch%2CEA&amp;author=Siminovitch%2CL">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="39."><p class="c-article-references__text" id="ref-CR39">Bayes, T. An essay towards solving a problem in the doctrine of chances. By the late Rev. Mr. Bayes, F. R. S. communicated by Mr. Price, in a letter to John Canton, A. M. F. R. S. <a href="https://doi.org/10.1098/rstl.1763.0053" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1098/rstl.1763.0053">https://doi.org/10.1098/rstl.1763.0053</a> (1763).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="40."><p class="c-article-references__text" id="ref-CR40">Van Rossum, G. &amp; Drake, F. L. <i>Python 3 Reference Manual</i>. <a href="https://dl.acm.org/doi/book/10.5555/1593511" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://dl.acm.org/doi/book/10.5555/1593511">https://dl.acm.org/doi/book/10.5555/1593511</a> (CreateSpace, 2009).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="41."><p class="c-article-references__text" id="ref-CR41">Harris, C. R. et al. Array programming with NumPy. <i>Nature</i> <b>585</b>, 357–362 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41586-020-2649-2" data-track-item_id="10.1038/s41586-020-2649-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41586-020-2649-2" aria-label="Article reference 41" data-doi="10.1038/s41586-020-2649-2">Article</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="cas reference" data-track-action="cas reference" href="/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlWmsbbN" aria-label="CAS reference 41">CAS</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed reference" data-track-action="pubmed reference" href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32939066" aria-label="PubMed reference 41">PubMed</a>&nbsp;
    <a data-track="click_references" rel="nofollow noopener" data-track-label="link" data-track-item_id="link" data-track-value="pubmed central reference" data-track-action="pubmed central reference" href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7759461" aria-label="PubMed Central reference 41">PubMed Central</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 41" href="http://scholar.google.com/scholar_lookup?&amp;title=Array%20programming%20with%20NumPy&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2649-2&amp;volume=585&amp;pages=357-362&amp;publication_year=2020&amp;author=Harris%2CCR">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="42."><p class="c-article-references__text" id="ref-CR42">Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. <i>Nat. Methods</i> <b>17</b>, 261–272 (2020).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.1038/s41592-019-0686-2" data-track-item_id="10.1038/s41592-019-0686-2" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.1038%2Fs41592-019-0686-2" aria-label="Article reference 42" data-doi="10.1038/s41592-019-0686-2">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 42" href="http://scholar.google.com/scholar_lookup?&amp;title=SciPy%201.0%3A%20fundamental%20algorithms%20for%20scientific%20computing%20in%20Python&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0686-2&amp;volume=17&amp;pages=261-272&amp;publication_year=2020&amp;author=Virtanen%2C%20P.%2C">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="43."><p class="c-article-references__text" id="ref-CR43">Davidson-Pilon, C. lifelines: survival analysis in Python. <i>J. Open Source Softw.</i> <b>4</b>, 1317 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.21105/joss.01317" data-track-item_id="10.21105/joss.01317" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.21105%2Fjoss.01317" aria-label="Article reference 43" data-doi="10.21105/joss.01317">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 43" href="http://scholar.google.com/scholar_lookup?&amp;title=lifelines%3A%20survival%20analysis%20in%20Python&amp;journal=J.%20Open%20Source%20Softw.&amp;doi=10.21105%2Fjoss.01317&amp;volume=4&amp;publication_year=2019&amp;author=Davidson-Pilon%2CC">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="44."><p class="c-article-references__text" id="ref-CR44">R Core Team. R: a language and environment for statistical computing. <a href="https://www.R-project.org/" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.R-project.org/">https://www.R-project.org/</a> (R Foundation for Statistical Computing, 2021).</p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="45."><p class="c-article-references__text" id="ref-CR45">Wickham, H. et al. Welcome to the tidyverse. <i>J. Open Source Softw.</i> <b>4</b>, 1686 (2019).</p><p class="c-article-references__links u-hide-print"><a data-track="click_references" rel="nofollow noopener" data-track-label="10.21105/joss.01686" data-track-item_id="10.21105/joss.01686" data-track-value="article reference" data-track-action="article reference" href="https://doi.org/10.21105%2Fjoss.01686" aria-label="Article reference 45" data-doi="10.21105/joss.01686">Article</a>&nbsp;
    <a data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link" rel="nofollow noopener" aria-label="Google Scholar reference 45" href="http://scholar.google.com/scholar_lookup?&amp;title=Welcome%20to%20the%20tidyverse&amp;journal=J.%20Open%20Source%20Softw.&amp;doi=10.21105%2Fjoss.01686&amp;volume=4&amp;publication_year=2019&amp;author=Wickham%2CH">
                    Google Scholar</a>&nbsp;
                </p></li><li class="c-article-references__item js-c-reading-companion-references-item" data-counter="46."><p class="c-article-references__text" id="ref-CR46">Wickham, H. <i>ggplot2: Elegant Graphics for Data Analysis</i>. <a href="https://ggplot2.tidyverse.org" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://ggplot2.tidyverse.org">https://ggplot2.tidyverse.org</a> (Springer-Verlag, 2016).</p></li></ol><p class="c-article-references__download u-hide-print"><a data-track="click" data-track-action="download citation references" data-track-label="link" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01883-3?format=refman&amp;flavour=references">Download references<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p></div></div></div></section></div><section data-title="Acknowledgements"><div class="c-article-section" id="Ack1-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Ack1">Acknowledgements</h2><div class="c-article-section__content" id="Ack1-content"><p>We gratefully acknowledge the contributions of the LBC participants and members of the LBC research team who collect and manage the LBC data. We thank C. P. Ponting for critical reading of the manuscript. We would also like to thank B. Tait for his help, advice and patience throughout. LBC1921 was supported by the UK’s Biotechnology and Biological Sciences Research Council (BBSRC) (SR176) to I.J.D; by a Royal Society–Wolfson Research Merit Award to I.J.D.; and by the Chief Scientist Office of the Scottish Government’s Health Directorates (CZB/4/505; ETM/55) to I.J.D. LBC1936 is supported by the BBSRC and the Economic and Social Research Council (BB/W008793/1) to S.R.C; Age UK (Disconnected Mind project, which supports S.E.H) to I.J.D and S.R.C.; the Medical Research Council (MR/M01311/1 to I.J.D. and MR/K026992/1 to S.R.C.); and the University of Edinburgh. K.K. is funded by a John Goldman Fellowship, sponsored by Leukaemia U.K. (2019/JGF/003 to K.K.) and received CRUK Glasgow Centre funding (C7932/A25142 to K.K.) and CRUK Scotland Centre funding (CTRQQR-2021\100006 to K.K.). M.T.T. and N.A.R. are supported by Medical Research Council-funded Ph.D. studentships (MR/N013166/1 to M.T.T. and N.A.R.). T.C. and L.S. are supported by Chancellor’s Fellowships held at the University of Edinburgh. J.A.M. is a Lister Institute Research Prize Fellow. E.L.C. is a cross-disciplinary postdoctoral fellow supported by funding from the University of Edinburgh and the Medical Research Council (MC_UU_00009/2). S.R.C. is supported by a Sir Henry Dale Fellowship, jointly funded by the Wellcome Trust and the Royal Society (221890/Z/20/Z). We are also grateful for funding from the Howat Foundation (grant holder, M.C.).</p></div></div></section><section aria-labelledby="author-information" data-title="Author information"><div class="c-article-section" id="author-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="author-information">Author information</h2><div class="c-article-section__content" id="author-information-content"><span class="c-article-author-information__subtitle u-visually-hidden" id="author-notes">Author notes</span><ol class="c-article-author-information__list"><li class="c-article-author-information__item" id="na1"><p>These authors contributed equally: Neil A. Robertson, Eric Latorre-Crespo, Linus J. Schumacher, Kristina Kirschner, Tamir Chandra.</p></li></ol><h3 class="c-article__sub-heading" id="affiliations">Authors and Affiliations</h3><ol class="c-article-author-affiliation__list"><li id="Aff1"><p class="c-article-author-affiliation__address">MRC Human Genetics Unit, University of Edinburgh, Edinburgh, UK</p><p class="c-article-author-affiliation__authors-list">Neil A. Robertson,&nbsp;Eric Latorre-Crespo,&nbsp;Benjamin J. Livesey,&nbsp;Joseph A. Marsh&nbsp;&amp;&nbsp;Tamir Chandra</p></li><li id="Aff2"><p class="c-article-author-affiliation__address">Institute of Cancer Sciences, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, UK</p><p class="c-article-author-affiliation__authors-list">Maria Terradas-Terradas,&nbsp;Alison C. Purcell&nbsp;&amp;&nbsp;Kristina Kirschner</p></li><li id="Aff3"><p class="c-article-author-affiliation__address">Cancer Research UK Beatson Institute, Glasgow, UK</p><p class="c-article-author-affiliation__authors-list">Maria Terradas-Terradas,&nbsp;Alison C. Purcell&nbsp;&amp;&nbsp;Kristina Kirschner</p></li><li id="Aff4"><p class="c-article-author-affiliation__address">Centre for Regenerative Medicine, Institute for Regeneration and Repair, University of Edinburgh, Edinburgh, UK</p><p class="c-article-author-affiliation__authors-list">Jorge Lemos-Portela&nbsp;&amp;&nbsp;Linus J. Schumacher</p></li><li id="Aff5"><p class="c-article-author-affiliation__address">Centre for Genomic and Experimental Medicine, Institute of Genetics and Cancer, University of Edinburgh, Edinburgh, UK</p><p class="c-article-author-affiliation__authors-list">Robert F. Hillary&nbsp;&amp;&nbsp;Riccardo E. Marioni</p></li><li id="Aff6"><p class="c-article-author-affiliation__address">Edinburgh Clinical Research Facility, University of Edinburgh, Edinburgh, UK</p><p class="c-article-author-affiliation__authors-list">Lee Murphy,&nbsp;Angie Fawkes&nbsp;&amp;&nbsp;Louise MacGillivray</p></li><li id="Aff7"><p class="c-article-author-affiliation__address">Paul O’Gorman Leukaemia Research Centre, Institute of Cancer Sciences, College of Medical, Veterinary and Life Sciences, University of Glasgow, Glasgow, UK</p><p class="c-article-author-affiliation__authors-list">Mhairi Copland</p></li><li id="Aff8"><p class="c-article-author-affiliation__address">Lothian Birth Cohorts, Department of Psychology, University of Edinburgh, Edinburgh, UK</p><p class="c-article-author-affiliation__authors-list">Sarah E. Harris,&nbsp;Simon R. Cox&nbsp;&amp;&nbsp;Ian J. Deary</p></li></ol><div class="u-js-hide u-hide-print" data-test="author-info"><span class="c-article__sub-heading">Authors</span><ol class="c-article-authors-search u-list-reset"><li id="auth-Neil_A_-Robertson-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Neil A. Robertson</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Neil%20A.%20Robertson" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Neil%20A.%20Robertson" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Neil%20A.%20Robertson%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Eric-Latorre_Crespo-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Eric Latorre-Crespo</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Eric%20Latorre-Crespo" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Eric%20Latorre-Crespo" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Eric%20Latorre-Crespo%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Maria-Terradas_Terradas-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Maria Terradas-Terradas</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Maria%20Terradas-Terradas" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Maria%20Terradas-Terradas" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Maria%20Terradas-Terradas%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Jorge-Lemos_Portela-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Jorge Lemos-Portela</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Jorge%20Lemos-Portela" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Jorge%20Lemos-Portela" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Jorge%20Lemos-Portela%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Alison_C_-Purcell-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Alison C. Purcell</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Alison%20C.%20Purcell" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Alison%20C.%20Purcell" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Alison%20C.%20Purcell%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Benjamin_J_-Livesey-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Benjamin J. Livesey</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Benjamin%20J.%20Livesey" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Benjamin%20J.%20Livesey" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Benjamin%20J.%20Livesey%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Robert_F_-Hillary-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Robert F. Hillary</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Robert%20F.%20Hillary" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Robert%20F.%20Hillary" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Robert%20F.%20Hillary%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Lee-Murphy-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Lee Murphy</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Lee%20Murphy" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Lee%20Murphy" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Lee%20Murphy%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Angie-Fawkes-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Angie Fawkes</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Angie%20Fawkes" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Angie%20Fawkes" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Angie%20Fawkes%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Louise-MacGillivray-Aff6"><span class="c-article-authors-search__title u-h3 js-search-name">Louise MacGillivray</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Louise%20MacGillivray" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Louise%20MacGillivray" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Louise%20MacGillivray%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Mhairi-Copland-Aff7"><span class="c-article-authors-search__title u-h3 js-search-name">Mhairi Copland</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Mhairi%20Copland" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Mhairi%20Copland" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Mhairi%20Copland%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Riccardo_E_-Marioni-Aff5"><span class="c-article-authors-search__title u-h3 js-search-name">Riccardo E. Marioni</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Riccardo%20E.%20Marioni" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Riccardo%20E.%20Marioni" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Riccardo%20E.%20Marioni%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Joseph_A_-Marsh-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Joseph A. Marsh</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Joseph%20A.%20Marsh" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Joseph%20A.%20Marsh" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Joseph%20A.%20Marsh%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Sarah_E_-Harris-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Sarah E. Harris</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Sarah%20E.%20Harris" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Sarah%20E.%20Harris" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Sarah%20E.%20Harris%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Simon_R_-Cox-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Simon R. Cox</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Simon%20R.%20Cox" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Simon%20R.%20Cox" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Simon%20R.%20Cox%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Ian_J_-Deary-Aff8"><span class="c-article-authors-search__title u-h3 js-search-name">Ian J. Deary</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Ian%20J.%20Deary" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Ian%20J.%20Deary" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Ian%20J.%20Deary%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Linus_J_-Schumacher-Aff4"><span class="c-article-authors-search__title u-h3 js-search-name">Linus J. Schumacher</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Linus%20J.%20Schumacher" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Linus%20J.%20Schumacher" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Linus%20J.%20Schumacher%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Kristina-Kirschner-Aff2-Aff3"><span class="c-article-authors-search__title u-h3 js-search-name">Kristina Kirschner</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Kristina%20Kirschner" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Kristina%20Kirschner" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Kristina%20Kirschner%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li><li id="auth-Tamir-Chandra-Aff1"><span class="c-article-authors-search__title u-h3 js-search-name">Tamir Chandra</span><div class="c-article-authors-search__list"><div class="c-article-authors-search__item c-article-authors-search__list-item--left"><a href="/search?author=Tamir%20Chandra" class="c-article-button" data-track="click" data-track-action="author link - publication" data-track-label="link" rel="nofollow">View author publications</a></div><div class="c-article-authors-search__item c-article-authors-search__list-item--right"><p class="search-in-title-js c-article-authors-search__text"><span class="c-article-authors-search__links-text">You can also search for this author in</span><span class="c-article-identifiers"><a class="c-article-identifiers__item" href="https://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=search&amp;term=Tamir%20Chandra" data-track="click" data-track-action="author link - pubmed" data-track-label="link" rel="nofollow">PubMed</a><span class="u-hide">&nbsp;</span><a class="c-article-identifiers__item" href="https://scholar.google.co.uk/scholar?as_q=&amp;num=10&amp;btnG=Search+Scholar&amp;as_epq=&amp;as_oq=&amp;as_eq=&amp;as_occt=any&amp;as_sauthors=%22Tamir%20Chandra%22&amp;as_publication=&amp;as_ylo=&amp;as_yhi=&amp;as_allsubj=all&amp;hl=en" data-track="click" data-track-action="author link - scholar" data-track-label="link" rel="nofollow">Google Scholar</a></span></p></div></div></li></ol></div><h3 class="c-article__sub-heading" id="contributions">Contributions</h3><p>L.J.S., K.K. and T.C. conceived and supervised the study. N.A.R., E.L.C., L.J.S., K.K. and T.C. wrote the manuscript. L.M., A.F. and L.M.G. generated data. N.A.R. and E.L.C. developed the methodology for data analysis. N.A.R., E.L.C., M.T.T., A.C.P., J.A.M., B.J.L., J.L.P., R.F.H., R.E.M. and J.L.P. conducted data analysis. S.E.H., S.R.C. and I.J.D. curated the LBCs and gave access to samples. M.C. advised on aspects of the study.</p><h3 class="c-article__sub-heading" id="corresponding-author">Corresponding authors</h3><p id="corresponding-author-list">Correspondence to
                <a id="corresp-c1" href="mailto:linus.schumacher@ed.ac.uk">Linus J. Schumacher</a>, <a id="corresp-c2" href="mailto:kristina.kirschner@glasgow.ac.uk">Kristina Kirschner</a> or <a id="corresp-c3" href="mailto:tamir.chandra@igmm.ed.ac.uk">Tamir Chandra</a>.</p></div></div></section><section data-title="Ethics declarations"><div class="c-article-section" id="ethics-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="ethics">Ethics declarations</h2><div class="c-article-section__content" id="ethics-content">
              
                <h3 class="c-article__sub-heading" id="FPar4">Competing interests</h3>
                <p>K.K. received a reagent grant from ArcherDX/Invitae. L.M. consults for Illumina. M.C. has received research funding from Cyclacel and Incyte, is/has been an advisory board member for Novartis, Incyte, Pfizer and Jazz Pharmaceuticals and has received honoraria from Astellas, Novartis, Incyte, Pfizer and Jazz Pharmaceuticals. The remaining authors declare no competing interests.</p>
              
            </div></div></section><section data-title="Peer review"><div class="c-article-section" id="peer-review-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="peer-review">Peer review</h2><div class="c-article-section__content" id="peer-review-content">
              
              
                <h3 class="c-article__sub-heading" id="FPar3">Peer review information</h3>
                <p><i>Nature Medicine</i> thanks Jamie Blundell, Alejo Rodriguez-Fraticelli, Hubert Serve and the other, anonymous, reviewer(s) for their contribution to the peer review of this work. Primary handling editor: Anna Maria Ranzoni, in collaboration with the <i>Nature Medicine</i> team.</p>
              
            </div></div></section><section data-title="Additional information"><div class="c-article-section" id="additional-information-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="additional-information">Additional information</h2><div class="c-article-section__content" id="additional-information-content"><p><b>Publisher’s note</b> Springer Nature remains neutral with regard to jurisdictional claims in published maps and institutional affiliations.</p></div></div></section><section data-title="Extended data"><div class="c-article-section" id="Sec18-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec18">Extended data</h2><div class="c-article-section__content" id="Sec18-content"><div data-test="supplementary-info"><div id="figshareContainer" class="c-article-figshare-container" data-test="figshare-container"></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig5"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 1 quality control metrics." href="/articles/s41591-022-01883-3/figures/5" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig5_ESM.jpg">Extended Data Fig. 1 Quality Control Metrics.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>. Sequence quality metrics for mutation calls across participants and time-points filtered for 2% VAF. Plotted are the AO (the number of sequenced reads supporting the alternative allele (mutation)) against the UAO (the number of sequenced reads with unique start sites that support the alternative allele - a measure of molecular complexity). Red dotted lines denote filter thresholds in both measurements (AO ≥ 5, UAO ≥ 3) and points are scaled by the VAF of the somatic mutation. Only 7 (of 275) data points failed to meet our filter criteria which were not excluded as they were supported with matching events across any participants’ time series. <b>b</b>. Box and jitter plot of the variant allele frequency of all observed events in the 1st Wave at 2% VAF coloured by variant classification and ordered by largest mean VAF showing the median and interquartile range. <b>c</b>. The 95% MDAF (Minimal Detectable Allele Fraction with 95% Confidence) versus the VAF for each event. All variants used in our analysis above 2% VAF are scaled by their clone size and coloured by their functional consequence. Points in red are events that failed to pass our quality criteria and are removed from subsequent work. <b>d</b>. The VAF Outlier P-Value (describing the pan-cohort position-specific background noise) versus VAF for each event. All variants used in the analysis above 2% VAF are scaled by their clone size and coloured by their functional consequence. Points in red are events that failed to pass our quality criteria and are removed from subsequent work. All accepted events that exceed VAF Outlier P-Value &gt; 0.1 are generally low VAF and are supported by matching events across the time-series that adhere to our acceptance criteria of VAF Outlier P-Value ≤ 0.1. <b>e</b>. Schematic of all affected genes in the cohort with the largest clone size of an event in any given gene shown above 2% VAF. All affected participants have been clustered across all time-points, with the point size scaled by VAF and coloured by the functional consequence of the variant (as per legend).</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig6"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 2 lift method details." href="/articles/s41591-022-01883-3/figures/6" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig6_ESM.jpg">Extended Data Fig. 2 LiFT Method Details.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>. Prior distributions for the beta-binomial model for sequencing artefacts. Priors are constructed separately for mutations with a single occurrence and mutations with multiple observations in the LBCs (see SI methods Section 2.3). <b>b</b>. Prior distribution of the proportion for the binomial model for sequencing artefacts. This prior is constructed only for mutations with a single occurrence in the LBC. <b>c</b>. Effect of the Bayes Factor (BF) threshold on the number of non-synonymous variants selected as fit using LiFT. In red, we show the results assuming that sequencing artefacts always follow a beta-binomial model, regardless of the mutation occurrence in the LBC. In green, we show the results where the sequencing artefact model assumes a binomial model for single occurring mutations and a beta-binomial model for mutations with multiple occurrences in the LBCs. <b>d</b>. Effect of the BF on the number of synonymous variants selected as fit using LiFT. Colour coding as in Fig. S2C. <b>e</b>. Longitudinal trajectories of non-synonymous variants coloured by their LiFT status; fit (blue) and filtered (orange). <b>f</b>. Comparison between LiFT status and the VAF Outlier P-value. Each data point corresponds to a trajectory in the LBC and has been coloured according to its LiFT status; fit (blue) and filtered (orange). The coordinates of each data point are given by the average VAF Outlier p-Value and their average VAF. <b>g</b>. Comparison between LiFT status and the Minimal Detectable Allele Fraction (MDAF). Each data point corresponds to a trajectory in the LBC and has been coloured according to its LiFT status; fit (blue) and filtered (orange). The coordinates of each data point are given by the average MDAF and their average VAF. Note that the MDAF is shown on a logarithmic scale.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig7"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 3 clinical relevance of lift - " href="/articles/s41591-022-01883-3/figures/7" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig7_ESM.jpg">Extended Data Fig. 3 Clinical Relevance of LiFT - Supporting Material.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p><b>a</b>. Distribution of fitness by gene category. Genes are grouped according to their biological function into DNA methylation (<i>TET2, DNMT3A</i>), Splicing (<i>SF3B1, U2AF1, SRSF2, U2AF2, ZRSR2, LUC7L2, DDX41</i>), mitogenic function (<i>KRAS, NF1, JAK2, JAK3, SH2B3, PTEN, PTPN11, NRAS</i>), cohesin (<i>RAD21, STAG2</i>), DNA damage (<i>TP53, CDKN2A, PPM1D, ATRX</i>) and Transcription factors (TF) important during development (<i>GATA2, RUNX1, NOTCH1, CUX1, ETV6</i>). The sample size, n, of each gene category is denoted in brackets. For each gene category we display a boxplot showing the maximum a posteriori (MAP) estimates of fitness for variants in the category, as well as the median and exclusive interquartile range. <b>b</b>. Analysis of variance of the maximum posterior fitness estimates across gene categories. Heatmap of all statistically significant (p &lt; 0.05) Kruskal-Wallis H statistics, labelled by effect size, computed for all combinations of pairs of genes. The effect size is only shown for statistically significant relations. Variants with a fitness below 2% were left out of this study as our prediction classifies them as conferring no or a negligible fitness advantage. <b>c</b>. Influence of the number of time-points in a trajectory on the inferred fitness distributions. We show the maximum posterior estimates for genes <i>DNMT3A</i> and <i>TET2</i> and for all LiFT variants split according to the number of time-points. <b>d</b>. Survival analysis (Cox proportional hazards regression model) broken down by cohort and covariates. LBC1921 and LBC1936 are analysed separately given their difference in age during the observed time-span. (left) Error bar showing the inferred hazard ratio coefficient and 95% CI for each regression study, as well as the sample size, n, and the number of observed events in each analysis. Note that none of the survival analyses shown are statistically significant. The complete summary for each analysis is found in Supplementary Table <a data-track="click" data-track-label="link" data-track-action="supplementary material anchor" href="/articles/s41591-022-01883-3#MOESM3">7</a>. (right) Kaplan-Meyer survival plots for the LBC cohort stratified using 2 standard deviations of the analysed covariate.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig8"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 4 clonal dynamics and inference" href="/articles/s41591-022-01883-3/figures/8" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig8_ESM.jpg">Extended Data Fig. 4 Clonal Dynamics and Inference - Supporting Material.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>. <b>a</b>. Approximation of a neutral birth-death model using the negative binomial distribution. The exact model assumes symmetric divisions occur every 40 weeks, or 1.3 divisions per year, and has no bias towards self-renewal (see SI methods Section 1). <b>b</b>. Deterministic trajectory (see SI methods Appendix B) with maximum a posteriori (MAP) fitness and fitted time of mutation (left) and joint posterior distribution of fitness and number of wild-type HSPCs population (right) inferred from an individual with a single mutation selected by LiFT. 90% confidence intervals associated with binomial sampling noise are shown for each data point. Note that VAF is displayed on a logarithmic scale to highlight relative differences and the initial exponential growth of clones. Also note that a small random horizontal jitter has been added to data points to avoid overlapping of confidence intervals. <b>c</b>. Deterministic trajectory (see SI methods Appendix B) with maximum posterior fitness and fitted time of mutation (left) and joint posterior distribution of fitnesses and number of wild-type HSPCs inferred from an individual with three mutations, selected by LiFT, occurring in two clones. 90% confidence intervals associated with binomial sampling noise are shown for each data point. Note that VAF is displayed on a logarithmic scale to highlight relative differences and the initial exponential growth of clones. Also note that a small random horizontal jitter has been added to data points to avoid overlapping of confidence intervals. <b>d</b>. Evidence supporting the clonal structure selected by our Bayesian model comparison relative to the model assuming no mutations co-occur on the same clone. The evidence is only shown for non-trivial cases where more than one mutation was selected by LiFT in an individual.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig9"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 5 deterministic visualisation o" href="/articles/s41591-022-01883-3/figures/9" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig9_ESM.jpg">Extended Data Fig. 5 Deterministic Visualisation of Mutational Trajectories in the LBC21.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>a. Deterministic trajectories (see SI methods Appendix B) with maximum a posteriori (MAP) fitness and wild-type stem cells and fitted time of mutation (left) and posterior distribution of fitness associated to each clonal structure (right) inferred for all mutations selected by LiFT in each participant of the LBC1921 cohort. 90% confidence intervals associated with binomial sampling noise are shown for each data point. Note that VAF is displayed on a logarithmic scale to highlight relative differences and the initial exponential growth of clones. To use a logarithmic axis, data points with zero observations have been replaced by VAF = 0.001, or a factor of 10 below our observation threshold. Also note that a small random horizontal jitter has been added to data points to avoid overlapping of confidence intervals.</p></div></div><div class="c-article-supplementary__item js-c-reading-companion-figures-item" data-test="supp-item" id="Fig10"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="extended data fig. 6 deterministic visualisation o" href="/articles/s41591-022-01883-3/figures/10" data-supp-info-image="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig10_ESM.jpg">Extended Data Fig. 6 Deterministic Visualisation of Mutational Trajectories in the LBC36.</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>a. Deterministic trajectories (see SI methods Appendix B) with maximum a posteriori (MAP) fitness and wild-type stem cells and fitted time of mutation (left) and posterior distribution of fitness associated to each clonal structure (right) inferred for all mutations selected by LiFT in each participant of the LBC1936 cohort. 90% confidence intervals associated with binomial sampling noise are shown for each data point. Note that VAF is displayed on a logarithmic scale to highlight relative differences and the initial exponential growth of clones. To use a logarithmic axis, data points with zero observations have been replaced by VAF = 0.001, or a factor of 10 below our observation threshold. Also note that a small random horizontal jitter has been added to data points to avoid overlapping of confidence intervals.</p></div></div></div></div></div></section><section data-title="Supplementary information"><div class="c-article-section" id="Sec19-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="Sec19">Supplementary information</h2><div class="c-article-section__content" id="Sec19-content"><div data-test="supplementary-info"><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM1"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="supplementary information" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_MOESM1_ESM.pdf" data-supp-info-image="">Supplementary Information</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Methods—Detailing mathematical framework of methodology in three sections with two appendices</p></div></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM2"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="reporting summary" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_MOESM2_ESM.pdf" data-supp-info-image="">Reporting Summary</a></h3></div><div class="c-article-supplementary__item" data-test="supp-item" id="MOESM3"><h3 class="c-article-supplementary__title u-h3"><a class="print-link" data-track="click" data-track-action="view supplementary info" data-test="supp-info-link" data-track-label="tables 1–10" href="https://static-content.springer.com/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_MOESM3_ESM.xlsx" data-supp-info-image="">Tables 1–10</a></h3><div class="c-article-supplementary__description" data-component="thumbnail-container"><p>Supplementary Table 1: Cohort information. Supplementary Table 2: Complete list of unique variants detected at 2% VAF, including the maximum clone size (VAF) for events in all participants. Supplementary Table 3: Complete list of unique fit CHIP variants at 2% VAF. Supplementary Table 4: LiFT-Filter variants fitness estimates. Supplementary Table 5: Damage predictions for all single-nucleotide variants. Supplementary Table 6: Variants with a frameshift or termination mutation below 2% VAF. Supplementary Table 7: Survival analysis on the effects of maximum VAF and clone growth speed. Supplementary Table 8: Target genes of the ArcherDX VariantPlex Myeloid Panel. Supplementary Table 9: Sequencing quality control metrics. Supplementary Table 10: Known CHIP mutations.</p></div></div></div></div></div></section><section data-title="Rights and permissions"><div class="c-article-section" id="rightslink-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="rightslink">Rights and permissions</h2><div class="c-article-section__content" id="rightslink-content">
                <p><b>Open Access</b>  This article is licensed under a Creative Commons Attribution 4.0 International License, which permits use, sharing, adaptation, distribution and reproduction in any medium or format, as long as you give appropriate credit to the original author(s) and the source, provide a link to the Creative Commons license, and indicate if changes were made. The images or other third party material in this article are included in the article’s Creative Commons license, unless indicated otherwise in a credit line to the material. If material is not included in the article’s Creative Commons license and your intended use is not permitted by statutory regulation or exceeds the permitted use, you will need to obtain permission directly from the copyright holder. To view a copy of this license, visit <a href="http://creativecommons.org/licenses/by/4.0/" rel="license">http://creativecommons.org/licenses/by/4.0/</a>.</p>
              <p class="c-article-rights"><a data-track="click" data-track-action="view rights and permissions" data-track-label="link" href="https://s100.copyright.com/AppDispatchServlet?title=Longitudinal%20dynamics%20of%20clonal%20hematopoiesis%20identifies%20gene-specific%20fitness%20effects&amp;author=Neil%20A.%20Robertson%20et%20al&amp;contentID=10.1038%2Fs41591-022-01883-3&amp;copyright=The%20Author%28s%29&amp;publication=1078-8956&amp;publicationDate=2022-07-04&amp;publisherName=SpringerNature&amp;orderBeanReset=true&amp;oa=CC%20BY">Reprints and permissions</a></p></div></div></section><section aria-labelledby="article-info" data-title="About this article"><div class="c-article-section" id="article-info-section"><h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="article-info">About this article</h2><div class="c-article-section__content" id="article-info-content"><div class="c-bibliographic-information"><div class="u-hide-print c-bibliographic-information__column c-bibliographic-information__column--border"><a data-crossmark="10.1038/s41591-022-01883-3" target="_blank" rel="noopener" href="https://crossmark.crossref.org/dialog/?doi=10.1038/s41591-022-01883-3" data-track="click" data-track-action="Click Crossmark" data-track-label="link" data-test="crossmark"><img loading="lazy" width="57" height="81" alt="Check for updates. Verify currency and authenticity via CrossMark" src="data:image/svg+xml;base64,<svg height="81" width="57" xmlns="http://www.w3.org/2000/svg"><g fill="none" fill-rule="evenodd"><path d="m17.35 35.45 21.3-14.2v-17.03h-21.3" fill="#989898"/><path d="m38.65 35.45-21.3-14.2v-17.03h21.3" fill="#747474"/><path d="m28 .5c-12.98 0-23.5 10.52-23.5 23.5s10.52 23.5 23.5 23.5 23.5-10.52 23.5-23.5c0-6.23-2.48-12.21-6.88-16.62-4.41-4.4-10.39-6.88-16.62-6.88zm0 41.25c-9.8 0-17.75-7.95-17.75-17.75s7.95-17.75 17.75-17.75 17.75 7.95 17.75 17.75c0 4.71-1.87 9.22-5.2 12.55s-7.84 5.2-12.55 5.2z" fill="#535353"/><path d="m41 36c-5.81 6.23-15.23 7.45-22.43 2.9-7.21-4.55-10.16-13.57-7.03-21.5l-4.92-3.11c-4.95 10.7-1.19 23.42 8.78 29.71 9.97 6.3 23.07 4.22 30.6-4.86z" fill="#9c9c9c"/><path d="m.2 58.45c0-.75.11-1.42.33-2.01s.52-1.09.91-1.5c.38-.41.83-.73 1.34-.94.51-.22 1.06-.32 1.65-.32.56 0 1.06.11 1.51.35.44.23.81.5 1.1.81l-.91 1.01c-.24-.24-.49-.42-.75-.56-.27-.13-.58-.2-.93-.2-.39 0-.73.08-1.05.23-.31.16-.58.37-.81.66-.23.28-.41.63-.53 1.04-.13.41-.19.88-.19 1.39 0 1.04.23 1.86.68 2.46.45.59 1.06.88 1.84.88.41 0 .77-.07 1.07-.23s.59-.39.85-.68l.91 1c-.38.43-.8.76-1.28.99-.47.22-1 .34-1.58.34-.59 0-1.13-.1-1.64-.31-.5-.2-.94-.51-1.31-.91-.38-.4-.67-.9-.88-1.48-.22-.59-.33-1.26-.33-2.02zm8.4-5.33h1.61v2.54l-.05 1.33c.29-.27.61-.51.96-.72s.76-.31 1.24-.31c.73 0 1.27.23 1.61.71.33.47.5 1.14.5 2.02v4.31h-1.61v-4.1c0-.57-.08-.97-.25-1.21-.17-.23-.45-.35-.83-.35-.3 0-.56.08-.79.22-.23.15-.49.36-.78.64v4.8h-1.61zm7.37 6.45c0-.56.09-1.06.26-1.51.18-.45.42-.83.71-1.14.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.36c.07.62.29 1.1.65 1.44.36.33.82.5 1.38.5.29 0 .57-.04.83-.13s.51-.21.76-.37l.55 1.01c-.33.21-.69.39-1.09.53-.41.14-.83.21-1.26.21-.48 0-.92-.08-1.34-.25-.41-.16-.76-.4-1.07-.7-.31-.31-.55-.69-.72-1.13-.18-.44-.26-.95-.26-1.52zm4.6-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.07.45-.31.29-.5.73-.58 1.3zm2.5.62c0-.57.09-1.08.28-1.53.18-.44.43-.82.75-1.13s.69-.54 1.1-.71c.42-.16.85-.24 1.31-.24.45 0 .84.08 1.17.23s.61.34.85.57l-.77 1.02c-.19-.16-.38-.28-.56-.37-.19-.09-.39-.14-.61-.14-.56 0-1.01.21-1.35.63-.35.41-.52.97-.52 1.67 0 .69.17 1.24.51 1.66.34.41.78.62 1.32.62.28 0 .54-.06.78-.17.24-.12.45-.26.64-.42l.67 1.03c-.33.29-.69.51-1.08.65-.39.15-.78.23-1.18.23-.46 0-.9-.08-1.31-.24-.4-.16-.75-.39-1.05-.7s-.53-.69-.7-1.13c-.17-.45-.25-.96-.25-1.53zm6.91-6.45h1.58v6.17h.05l2.54-3.16h1.77l-2.35 2.8 2.59 4.07h-1.75l-1.77-2.98-1.08 1.23v1.75h-1.58zm13.69 1.27c-.25-.11-.5-.17-.75-.17-.58 0-.87.39-.87 1.16v.75h1.34v1.27h-1.34v5.6h-1.61v-5.6h-.92v-1.2l.92-.07v-.72c0-.35.04-.68.13-.98.08-.31.21-.57.4-.79s.42-.39.71-.51c.28-.12.63-.18 1.04-.18.24 0 .48.02.69.07.22.05.41.1.57.17zm.48 5.18c0-.57.09-1.08.27-1.53.17-.44.41-.82.72-1.13.3-.31.65-.54 1.04-.71.39-.16.8-.24 1.23-.24s.84.08 1.24.24c.4.17.74.4 1.04.71s.54.69.72 1.13c.19.45.28.96.28 1.53s-.09 1.08-.28 1.53c-.18.44-.42.82-.72 1.13s-.64.54-1.04.7-.81.24-1.24.24-.84-.08-1.23-.24-.74-.39-1.04-.7c-.31-.31-.55-.69-.72-1.13-.18-.45-.27-.96-.27-1.53zm1.65 0c0 .69.14 1.24.43 1.66.28.41.68.62 1.18.62.51 0 .9-.21 1.19-.62.29-.42.44-.97.44-1.66 0-.7-.15-1.26-.44-1.67-.29-.42-.68-.63-1.19-.63-.5 0-.9.21-1.18.63-.29.41-.43.97-.43 1.67zm6.48-3.44h1.33l.12 1.21h.05c.24-.44.54-.79.88-1.02.35-.24.7-.36 1.07-.36.32 0 .59.05.78.14l-.28 1.4-.33-.09c-.11-.01-.23-.02-.38-.02-.27 0-.56.1-.86.31s-.55.58-.77 1.1v4.2h-1.61zm-47.87 15h1.61v4.1c0 .57.08.97.25 1.2.17.24.44.35.81.35.3 0 .57-.07.8-.22.22-.15.47-.39.73-.73v-4.7h1.61v6.87h-1.32l-.12-1.01h-.04c-.3.36-.63.64-.98.86-.35.21-.76.32-1.24.32-.73 0-1.27-.24-1.61-.71-.33-.47-.5-1.14-.5-2.02zm9.46 7.43v2.16h-1.61v-9.59h1.33l.12.72h.05c.29-.24.61-.45.97-.63.35-.17.72-.26 1.1-.26.43 0 .81.08 1.15.24.33.17.61.4.84.71.24.31.41.68.53 1.11.13.42.19.91.19 1.44 0 .59-.09 1.11-.25 1.57-.16.47-.38.85-.65 1.16-.27.32-.58.56-.94.73-.35.16-.72.25-1.1.25-.3 0-.6-.07-.9-.2s-.59-.31-.87-.56zm0-2.3c.26.22.5.37.73.45.24.09.46.13.66.13.46 0 .84-.2 1.15-.6.31-.39.46-.98.46-1.77 0-.69-.12-1.22-.35-1.61-.23-.38-.61-.57-1.13-.57-.49 0-.99.26-1.52.77zm5.87-1.69c0-.56.08-1.06.25-1.51.16-.45.37-.83.65-1.14.27-.3.58-.54.93-.71s.71-.25 1.08-.25c.39 0 .73.07 1 .2.27.14.54.32.81.55l-.06-1.1v-2.49h1.61v9.88h-1.33l-.11-.74h-.06c-.25.25-.54.46-.88.64-.33.18-.69.27-1.06.27-.87 0-1.56-.32-2.07-.95s-.76-1.51-.76-2.65zm1.67-.01c0 .74.13 1.31.4 1.7.26.38.65.58 1.15.58.51 0 .99-.26 1.44-.77v-3.21c-.24-.21-.48-.36-.7-.45-.23-.08-.46-.12-.7-.12-.45 0-.82.19-1.13.59-.31.39-.46.95-.46 1.68zm6.35 1.59c0-.73.32-1.3.97-1.71.64-.4 1.67-.68 3.08-.84 0-.17-.02-.34-.07-.51-.05-.16-.12-.3-.22-.43s-.22-.22-.38-.3c-.15-.06-.34-.1-.58-.1-.34 0-.68.07-1 .2s-.63.29-.93.47l-.59-1.08c.39-.24.81-.45 1.28-.63.47-.17.99-.26 1.54-.26.86 0 1.51.25 1.93.76s.63 1.25.63 2.21v4.07h-1.32l-.12-.76h-.05c-.3.27-.63.48-.98.66s-.73.27-1.14.27c-.61 0-1.1-.19-1.48-.56-.38-.36-.57-.85-.57-1.46zm1.57-.12c0 .3.09.53.27.67.19.14.42.21.71.21.28 0 .54-.07.77-.2s.48-.31.73-.56v-1.54c-.47.06-.86.13-1.18.23-.31.09-.57.19-.76.31s-.33.25-.41.4c-.09.15-.13.31-.13.48zm6.29-3.63h-.98v-1.2l1.06-.07.2-1.88h1.34v1.88h1.75v1.27h-1.75v3.28c0 .8.32 1.2.97 1.2.12 0 .24-.01.37-.04.12-.03.24-.07.34-.11l.28 1.19c-.19.06-.4.12-.64.17-.23.05-.49.08-.76.08-.4 0-.74-.06-1.02-.18-.27-.13-.49-.3-.67-.52-.17-.21-.3-.48-.37-.78-.08-.3-.12-.64-.12-1.01zm4.36 2.17c0-.56.09-1.06.27-1.51s.41-.83.71-1.14c.29-.3.63-.54 1.01-.71.39-.17.78-.25 1.18-.25.47 0 .88.08 1.23.24.36.16.65.38.89.67s.42.63.54 1.03c.12.41.18.84.18 1.32 0 .32-.02.57-.07.76h-4.37c.08.62.29 1.1.65 1.44.36.33.82.5 1.38.5.3 0 .58-.04.84-.13.25-.09.51-.21.76-.37l.54 1.01c-.32.21-.69.39-1.09.53s-.82.21-1.26.21c-.47 0-.92-.08-1.33-.25-.41-.16-.77-.4-1.08-.7-.3-.31-.54-.69-.72-1.13-.17-.44-.26-.95-.26-1.52zm4.61-.62c0-.55-.11-.98-.34-1.28-.23-.31-.58-.47-1.06-.47-.41 0-.77.15-1.08.45-.31.29-.5.73-.57 1.3zm3.01 2.23c.31.24.61.43.92.57.3.13.63.2.98.2.38 0 .65-.08.83-.23s.27-.35.27-.6c0-.14-.05-.26-.13-.37-.08-.1-.2-.2-.34-.28-.14-.09-.29-.16-.47-.23l-.53-.22c-.23-.09-.46-.18-.69-.3-.23-.11-.44-.24-.62-.4s-.33-.35-.45-.55c-.12-.21-.18-.46-.18-.75 0-.61.23-1.1.68-1.49.44-.38 1.06-.57 1.83-.57.48 0 .91.08 1.29.25s.71.36.99.57l-.74.98c-.24-.17-.49-.32-.73-.42-.25-.11-.51-.16-.78-.16-.35 0-.6.07-.76.21-.17.15-.25.33-.25.54 0 .14.04.26.12.36s.18.18.31.26c.14.07.29.14.46.21l.54.19c.23.09.47.18.7.29s.44.24.64.4c.19.16.34.35.46.58.11.23.17.5.17.82 0 .3-.06.58-.17.83-.12.26-.29.48-.51.68-.23.19-.51.34-.84.45-.34.11-.72.17-1.15.17-.48 0-.95-.09-1.41-.27-.46-.19-.86-.41-1.2-.68z" fill="#535353"/></g></svg>"></a></div><div class="c-bibliographic-information__column"><h3 class="c-article__sub-heading" id="citeas">Cite this article</h3><p class="c-bibliographic-information__citation">Robertson, N.A., Latorre-Crespo, E., Terradas-Terradas, M. <i>et al.</i> Longitudinal dynamics of clonal hematopoiesis identifies gene-specific fitness effects.
                    <i>Nat Med</i> <b>28</b>, 1439–1446 (2022). https://doi.org/10.1038/s41591-022-01883-3</p><p class="c-bibliographic-information__download-citation u-hide-print"><a data-test="citation-link" data-track="click" data-track-action="download article citation" data-track-label="link" data-track-external="" rel="nofollow" href="https://citation-needed.springer.com/v2/references/10.1038/s41591-022-01883-3?format=refman&amp;flavour=citation">Download citation<svg width="16" height="16" focusable="false" role="img" aria-hidden="true" class="u-icon"><use xmlns:xlink="http://www.w3.org/1999/xlink" xlink:href="#icon-eds-i-download-medium"></use></svg></a></p><ul class="c-bibliographic-information__list" data-test="publication-history"><li class="c-bibliographic-information__list-item"><p>Received<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2021-09-03">03 September 2021</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Accepted<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-05-27">27 May 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Published<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-07-04">04 July 2022</time></span></p></li><li class="c-bibliographic-information__list-item"><p>Issue Date<span class="u-hide">: </span><span class="c-bibliographic-information__value"><time datetime="2022-07">July 2022</time></span></p></li><li class="c-bibliographic-information__list-item c-bibliographic-information__list-item--full-width"><p><abbr title="Digital Object Identifier">DOI</abbr><span class="u-hide">: </span><span class="c-bibliographic-information__value">https://doi.org/10.1038/s41591-022-01883-3</span></p></li></ul><div data-component="share-box"><div class="c-article-share-box u-display-block"><h3 class="c-article__sub-heading">Share this article</h3><p class="c-article-share-box__description">Anyone you share the following link with will be able to read this content:</p><button class="js-get-share-url c-article-share-box__button" type="button" id="get-share-url" data-track="click" data-track-label="button" data-track-external="" data-track-action="get shareable link">Get shareable link</button><div class="js-no-share-url-container u-display-none" hidden=""><p class="js-c-article-share-box__no-sharelink-info c-article-share-box__no-sharelink-info">Sorry, a shareable link is not currently available for this article.</p></div><div class="js-share-url-container u-display-none" hidden=""><p class="js-share-url c-article-share-box__only-read-input" id="share-url" data-track="click" data-track-label="button" data-track-action="select share url"></p><button class="js-copy-share-url c-article-share-box__button--link-like" type="button" id="copy-share-url" data-track="click" data-track-label="button" data-track-action="copy share url" data-track-external="">Copy to clipboard</button></div><p class="js-c-article-share-box__additional-info c-article-share-box__additional-info">
                            Provided by the Springer Nature SharedIt content-sharing initiative
                        </p></div></div><div data-component="article-info-list">
        <h3 class="c-article__sub-heading">Subjects</h3>
        <ul class="c-article-subject-list">
            <li class="c-article-subject-list__subject"><a href="/subjects/cancer-genomics" data-track="click" data-track-action="view subject" data-track-label="link">Cancer genomics</a></li><li class="c-article-subject-list__subject"><a href="/subjects/cancer-stem-cells" data-track="click" data-track-action="view subject" data-track-label="link">Cancer stem cells</a></li><li class="c-article-subject-list__subject"><a href="/subjects/haematological-cancer" data-track="click" data-track-action="view subject" data-track-label="link">Haematological cancer</a></li><li class="c-article-subject-list__subject"><a href="/subjects/haematopoietic-stem-cells" data-track="click" data-track-action="view subject" data-track-label="link">Haematopoietic stem cells</a></li><li class="c-article-subject-list__subject"><a href="/subjects/risk-factors" data-track="click" data-track-action="view subject" data-track-label="link">Risk factors</a></li>
        </ul>
    </div></div></div></div></div></section>
            </div>

            
        <section>
            <div class="c-article-section js-article-section" id="further-reading-section" data-test="further-reading-section">
                <h2 class="c-article-section__title js-section-title js-c-reading-companion-sections-item" id="further-reading">This article is cited by</h2>
                <div class="c-article-section__content js-collapsible-section" id="further-reading-content">
                    <ul class="c-article-further-reading__list" id="further-reading-list">
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Microbial metabolite drives ageing-related clonal haematopoiesis via ALPK1" href="https://doi.org/10.1038/s41586-025-08938-8">
                                        Microbial metabolite drives ageing-related clonal haematopoiesis via ALPK1
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Puneet Agarwal</li><li>Avery Sampson</li><li>Daniel T. Starczynowski</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:How fast does leukemia progress?" href="https://doi.org/10.1038/s41375-025-02536-3">
                                        How fast does leukemia progress?
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Valentina Giudice</li><li>Carmine Selleri</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Leukemia</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:A genotype-first approach identifies high incidence of NF1 pathogenic variants with distinct disease associations" href="https://doi.org/10.1038/s41467-025-57077-1">
                                        A genotype-first approach identifies high incidence of NF1 pathogenic variants with distinct disease associations
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Anton Safonov</li><li>Tomoki T. Nomakuchi</li><li>Theodore G. Drivas</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2025)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life" href="https://doi.org/10.1038/s41467-024-54711-2">
                                        Developmental hematopoietic stem cell variation explains clonal hematopoiesis later in life
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Jesse Kreger</li><li>Jazlyn A. Mooney</li><li>Adam L. MacLean</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Nature Communications</i> (2024)</p>
                            </li>
                        
                            <li class="c-article-further-reading__item js-ref-item">
                            
                                <h3 class="c-article-further-reading__title" data-test="article-title">
                                    <a class="print-link" data-track="click" data-track-action="view further reading article" data-track-label="link:Dynamics of clonal hematopoiesis under DNA-damaging treatment in patients with ovarian cancer" href="https://doi.org/10.1038/s41375-024-02253-3">
                                        Dynamics of clonal hematopoiesis under DNA-damaging treatment in patients with ovarian cancer
                                    </a>
                                </h3>
                            
                                
                                    <ul data-test="author-list" class="app-author-list app-author-list--compact app-author-list--truncated u-sans-serif u-mb-4 u-mt-auto">
                                        <li>Christopher Maximilian Arends</li><li>Klara Kopp</li><li>Frederik Damm</li>
                                    </ul>
                                
                                <p class="c-article-further-reading__journal-title"><i>Leukemia</i> (2024)</p>
                            </li>
                        
                    </ul>
                </div>
            </div>
        </section>
    

        </div>
</article>
</main>

<aside class="c-article-extras u-hide-print" aria-label="Article navigation" data-component-reading-companion="" data-container-type="reading-companion" data-track-component="reading companion">
    <div class="js-context-bar-sticky-point-desktop" data-track-context="reading companion">
        

        
            
                
                    
    
        <div class="c-pdf-download u-clear-both js-pdf-download">
            <a href="/articles/s41591-022-01883-3.pdf" class="u-button u-button--full-width u-button--primary u-justify-content-space-between c-pdf-download__link" data-article-pdf="true" data-readcube-pdf-url="true" data-test="download-pdf" data-draft-ignore="true" data-track="content_download" data-track-type="article pdf download" data-track-action="download pdf" data-track-label="link" data-track-external="" download="">
                <span class="c-pdf-download__text">Download PDF</span>
                <svg aria-hidden="true" focusable="false" width="16" height="16" class="u-icon"><use xlink:href="#icon-download"></use></svg>
            </a>
        </div>
    

                
            
        
    </div>

    
        
    

    
    

    <div class="c-reading-companion">
        <div class="c-reading-companion__sticky" data-component="reading-companion-sticky" data-test="reading-companion-sticky" style="top: 40px;">
            <ul class="c-reading-companion__tabs" role="tablist"><li role="presentation"><button data-tab-target="sections" role="tab" id="tab-sections" aria-controls="tabpanel-sections" aria-selected="true" class="c-reading-companion__tab c-reading-companion__tab--active" data-track="click" data-track-action="sections tab" data-track-label="tab">Sections</button></li><li role="presentation"><button data-tab-target="figures" role="tab" id="tab-figures" aria-controls="tabpanel-figures" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="figures tab" data-track-label="tab">Figures</button></li><li role="presentation"><button data-tab-target="references" role="tab" id="tab-references" aria-controls="tabpanel-references" aria-selected="false" tabindex="-1" class="c-reading-companion__tab" data-track="click" data-track-action="references tab" data-track-label="tab">References</button></li></ul><div class="c-reading-companion__panel c-reading-companion__sections c-reading-companion__panel--active" id="tabpanel-sections" aria-labelledby="tab-sections" role="tabpanel"><div class="c-reading-companion__scroll-pane" style="max-height: 409.45px;"><ul class="c-reading-companion__sections-list"><li id="rc-sec-Abs1" class="c-reading-companion__section-item"><a href="#Abs1" data-track="click" data-track-action="section anchor" data-track-label="link:Abstract">Abstract</a></li><li id="rc-sec-Sec1" class="c-reading-companion__section-item"><a href="#Sec1" data-track="click" data-track-action="section anchor" data-track-label="link:Main">Main</a></li><li id="rc-sec-Sec2" class="c-reading-companion__section-item"><a href="#Sec2" data-track="click" data-track-action="section anchor" data-track-label="link:Results">Results</a></li><li id="rc-sec-Sec7" class="c-reading-companion__section-item"><a href="#Sec7" data-track="click" data-track-action="section anchor" data-track-label="link:Discussion">Discussion</a></li><li id="rc-sec-Sec8" class="c-reading-companion__section-item"><a href="#Sec8" data-track="click" data-track-action="section anchor" data-track-label="link:Methods">Methods</a></li><li id="rc-sec-data-availability" class="c-reading-companion__section-item"><a href="#data-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Data availability">Data availability</a></li><li id="rc-sec-code-availability" class="c-reading-companion__section-item"><a href="#code-availability" data-track="click" data-track-action="section anchor" data-track-label="link:Code availability">Code availability</a></li><li id="rc-sec-Bib1" class="c-reading-companion__section-item"><a href="#Bib1" data-track="click" data-track-action="section anchor" data-track-label="link:References">References</a></li><li id="rc-sec-Ack1" class="c-reading-companion__section-item"><a href="#Ack1" data-track="click" data-track-action="section anchor" data-track-label="link:Acknowledgements">Acknowledgements</a></li><li id="rc-sec-author-information" class="c-reading-companion__section-item"><a href="#author-information" data-track="click" data-track-action="section anchor" data-track-label="link:Author information">Author information</a></li><li id="rc-sec-ethics" class="c-reading-companion__section-item"><a href="#ethics" data-track="click" data-track-action="section anchor" data-track-label="link:Ethics declarations">Ethics declarations</a></li><li id="rc-sec-peer-review" class="c-reading-companion__section-item"><a href="#peer-review" data-track="click" data-track-action="section anchor" data-track-label="link:Peer review">Peer review</a></li><li id="rc-sec-additional-information" class="c-reading-companion__section-item"><a href="#additional-information" data-track="click" data-track-action="section anchor" data-track-label="link:Additional information">Additional information</a></li><li id="rc-sec-Sec18" class="c-reading-companion__section-item"><a href="#Sec18" data-track="click" data-track-action="section anchor" data-track-label="link:Extended data">Extended data</a></li><li id="rc-sec-Sec19" class="c-reading-companion__section-item"><a href="#Sec19" data-track="click" data-track-action="section anchor" data-track-label="link:Supplementary information">Supplementary information</a></li><li id="rc-sec-rightslink" class="c-reading-companion__section-item"><a href="#rightslink" data-track="click" data-track-action="section anchor" data-track-label="link:Rights and permissions">Rights and permissions</a></li><li id="rc-sec-article-info" class="c-reading-companion__section-item"><a href="#article-info" data-track="click" data-track-action="section anchor" data-track-label="link:About this article">About this article</a></li><li id="rc-sec-further-reading" class="c-reading-companion__section-item"><a href="#further-reading" data-track="click" data-track-action="section anchor" data-track-label="link:This article is cited by">This article is cited by</a></li></ul></div>
                <div class="u-lazy-ad-wrapper u-mt-16 u-show" data-component-mpu="" style="display: none !important;">
                    <div class="c-ad c-ad--300x250">
                        <div class="c-ad__inner">
                            <p class="c-ad__label">Advertisement</p>
                            
    <div id="div-gpt-ad-right-2" class="div-gpt-ad advert medium-rectangle js-ad text-center hide-print grade-c-hide" data-ad-type="right" data-test="right-ad" data-pa11y-ignore="" data-gpt="" data-gpt-unitpath="/285/medicine.nature.com/article" data-gpt-sizes="300x250" data-gpt-targeting="type=article;pos=right;artid=s41591-022-01883-3;doi=10.1038/s41591-022-01883-3;subjmeta=1542,1990,499,532,631,67,69,692,71;kwrd=Cancer+genomics,Cancer+stem+cells,Haematological+cancer,Haematopoietic+stem+cells,Risk+factors">
        <noscript>
            <a href="//pubads.g.doubleclick.net/gampad/jump?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=1362097886&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01883-3%26doi%3D10.1038/s41591-022-01883-3%26subjmeta%3D1542,1990,499,532,631,67,69,692,71%26kwrd%3DCancer+genomics,Cancer+stem+cells,Haematological+cancer,Haematopoietic+stem+cells,Risk+factors">
                <img data-test="gpt-advert-fallback-img"
                     src="//pubads.g.doubleclick.net/gampad/ad?iu=/285/medicine.nature.com/article&amp;sz=300x250&amp;c=1362097886&amp;t=pos%3Dright%26type%3Darticle%26artid%3Ds41591-022-01883-3%26doi%3D10.1038/s41591-022-01883-3%26subjmeta%3D1542,1990,499,532,631,67,69,692,71%26kwrd%3DCancer+genomics,Cancer+stem+cells,Haematological+cancer,Haematopoietic+stem+cells,Risk+factors"
                     alt="Advertisement"
                     width="300"
                     height="250"></a>
        </noscript>
    </div>

                        </div>
                    </div>
                </div>
            </div>
            <div class="c-reading-companion__panel c-reading-companion__figures c-reading-companion__panel--full-width" id="tabpanel-figures" aria-labelledby="tab-figures" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ul class="c-reading-companion__figures-list"><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig1">Fig. 1: Clonal hematopoiesis in the LBCs.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig1_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig1_HTML.png" alt="figure 1" aria-describedby="rc-Fig1"></picture><p class="c-reading-companion__figure-links"><a href="#Fig1" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01883-3/figures/1" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig2">Fig. 2: Fitness effects of variants at 2% VAF threshold in longitudinal data.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig2_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig2_HTML.png" alt="figure 2" aria-describedby="rc-Fig2"></picture><p class="c-reading-companion__figure-links"><a href="#Fig2" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01883-3/figures/2" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig3">Fig. 3: LiFT allows classification of fit variants &lt;2% VAF.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig3_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig3_HTML.png" alt="figure 3" aria-describedby="rc-Fig3"></picture><p class="c-reading-companion__figure-links"><a href="#Fig3" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01883-3/figures/3" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig4">Fig. 4: Clinical relevance of LiFT.</b></figcaption><picture><source data-srcset="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig4_HTML.png?"><img data-src="https://media.springernature.com/lw685/springer-static/image/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig4_HTML.png" alt="figure 4" aria-describedby="rc-Fig4"></picture><p class="c-reading-companion__figure-links"><a href="#Fig4" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01883-3/figures/4" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig5">Extended Data Fig. 1 Quality Control Metrics.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig5_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig5_ESM.jpg" alt="extended data figure 5" aria-describedby="rc-Fig5"></picture><p class="c-reading-companion__figure-links"><a href="#Fig5" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01883-3/figures/5" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig6">Extended Data Fig. 2 LiFT Method Details.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig6_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig6_ESM.jpg" alt="extended data figure 6" aria-describedby="rc-Fig6"></picture><p class="c-reading-companion__figure-links"><a href="#Fig6" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01883-3/figures/6" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig7">Extended Data Fig. 3 Clinical Relevance of LiFT - Supporting Material.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig7_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig7_ESM.jpg" alt="extended data figure 7" aria-describedby="rc-Fig7"></picture><p class="c-reading-companion__figure-links"><a href="#Fig7" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01883-3/figures/7" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig8">Extended Data Fig. 4 Clonal Dynamics and Inference - Supporting Material.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig8_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig8_ESM.jpg" alt="extended data figure 8" aria-describedby="rc-Fig8"></picture><p class="c-reading-companion__figure-links"><a href="#Fig8" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01883-3/figures/8" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig9">Extended Data Fig. 5 Deterministic Visualisation of Mutational Trajectories in the LBC21.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig9_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig9_ESM.jpg" alt="extended data figure 9" aria-describedby="rc-Fig9"></picture><p class="c-reading-companion__figure-links"><a href="#Fig9" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01883-3/figures/9" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li><li class="c-reading-companion__figure-item"><figure><figcaption><b class="c-reading-companion__figure-title u-h4" id="rc-Fig10">Extended Data Fig. 6 Deterministic Visualisation of Mutational Trajectories in the LBC36.</b></figcaption><picture><source data-srcset="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig10_ESM.jpg?"><img data-src="//media.springernature.com/lw685/springer-static/esm/art%3A10.1038%2Fs41591-022-01883-3/MediaObjects/41591_2022_1883_Fig10_ESM.jpg" alt="extended data figure 10" aria-describedby="rc-Fig10"></picture><p class="c-reading-companion__figure-links"><a href="#Fig10" data-track="click" data-track-action="figure anchor" data-track-label="link">View in article</a><a href="https://www.nature.com/articles/s41591-022-01883-3/figures/10" class="c-reading-companion__figure-full-link" data-track="click" data-track-action="view figure" data-track-label="link" rel="nofollow">Full size image<svg width="16" height="16" class="u-icon"><use href="#icon-chevron-right"></use></svg></a></p></figure></li></ul></div></div>
            <div class="c-reading-companion__panel c-reading-companion__references c-reading-companion__panel--full-width" id="tabpanel-references" aria-labelledby="tab-references" role="tabpanel"><div class="c-reading-companion__scroll-pane"><ol class="c-reading-companion__references-list c-reading-companion__references-list--numeric" data-track-context="article references sidebar"><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR1">de Magalhaes, J. P. How ageing processes influence cancer. <i>Nat. Rev. Cancer</i> <b>13</b>, 357–365 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnrc3497" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nrc3497" data-track-item_id="10.1038/nrc3497">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23612461" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=How%20ageing%20processes%20influence%20cancer&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3497&amp;volume=13&amp;pages=357-365&amp;publication_year=2013&amp;author=Magalhaes%2CJP" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR2">Martincorena, I. Somatic mutation and clonal expansions in human tissues. <i>Genome Med.</i> <b>11</b>, 35 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/s13073-019-0648-4" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/s13073-019-0648-4" data-track-item_id="10.1186/s13073-019-0648-4">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31138277" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6537184" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Somatic%20mutation%20and%20clonal%20expansions%20in%20human%20tissues&amp;journal=Genome%20Med.&amp;doi=10.1186%2Fs13073-019-0648-4&amp;volume=11&amp;publication_year=2019&amp;author=Martincorena%2CI" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR3">Ayachi, S., Buscarlet, M. &amp; Busque, L. 60 years of clonal hematopoiesis research: from X-chromosome inactivation studies to the identification of driver mutations. <i>Exp. Hematol.</i> <b>83</b>, 2–11 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.exphem.2020.01.008" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.exphem.2020.01.008" data-track-item_id="10.1016/j.exphem.2020.01.008">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjsValsbo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32001340" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=60%20years%20of%20clonal%20hematopoiesis%20research%3A%20from%20X-chromosome%20inactivation%20studies%20to%20the%20identification%20of%20driver%20mutations&amp;journal=Exp.%20Hematol.&amp;doi=10.1016%2Fj.exphem.2020.01.008&amp;volume=83&amp;pages=2-11&amp;publication_year=2020&amp;author=Ayachi%2CS&amp;author=Buscarlet%2CM&amp;author=Busque%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR4">Genovese, G. et al. Clonal hematopoiesis and blood-cancer risk inferred from blood DNA sequence. <i>N. Engl. J. Med.</i> <b>371</b>, 2477–2487 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa1409405" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa1409405" data-track-item_id="10.1056/NEJMoa1409405">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25426838" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4290021" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20hematopoiesis%20and%20blood-cancer%20risk%20inferred%20from%20blood%20DNA%20sequence&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1409405&amp;volume=371&amp;pages=2477-2487&amp;publication_year=2014&amp;author=Genovese%2CG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR5">Jaiswal, S. et al. Age-related clonal hematopoiesis associated with adverse outcomes. <i>N. Engl. J. Med.</i> <b>371</b>, 2488–2498 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa1408617" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa1408617" data-track-item_id="10.1056/NEJMoa1408617">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25426837" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4306669" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Age-related%20clonal%20hematopoiesis%20associated%20with%20adverse%20outcomes&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1408617&amp;volume=371&amp;pages=2488-2498&amp;publication_year=2014&amp;author=Jaiswal%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR6">Lee-Six, H. et al. The landscape of somatic mutation in normal colorectal epithelial cells. <i>Nature</i> <b>574</b>, 532–537 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-019-1672-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-019-1672-7" data-track-item_id="10.1038/s41586-019-1672-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitVagsb3J" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31645730" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20landscape%20of%20somatic%20mutation%20in%20normal%20colorectal%20epithelial%20cells&amp;journal=Nature&amp;doi=10.1038%2Fs41586-019-1672-7&amp;volume=574&amp;pages=532-537&amp;publication_year=2019&amp;author=Lee-Six%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR7">Martincorena, I. et al. High burden and pervasive positive selection of somatic mutations in normal human skin. <i>Science</i> <b>348</b>, 880–886 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aaa6806" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aaa6806" data-track-item_id="10.1126/science.aaa6806">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXosFaktbg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25999502" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4471149" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=High%20burden%20and%20pervasive%20positive%20selection%20of%20somatic%20mutations%20in%20normal%20human%20skin&amp;journal=Science&amp;doi=10.1126%2Fscience.aaa6806&amp;volume=348&amp;pages=880-886&amp;publication_year=2015&amp;author=Martincorena%2CI" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR8">Jaiswal, S. &amp; Ebert, B. L. Clonal hematopoiesis in human aging and disease. <i>Science</i> <b>366</b>, eaan4673 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aan4673" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aan4673" data-track-item_id="10.1126/science.aan4673">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXitV2rt7nL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31672865" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8050831" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20hematopoiesis%20in%20human%20aging%20and%20disease&amp;journal=Science&amp;doi=10.1126%2Fscience.aan4673&amp;volume=366&amp;publication_year=2019&amp;author=Jaiswal%2CS&amp;author=Ebert%2CBL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR9">Park, S. J. &amp; Bejar, R. Clonal hematopoiesis in cancer. <i>Exp. Hematol.</i> <b>83</b>, 105–112 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.exphem.2020.02.001" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.exphem.2020.02.001" data-track-item_id="10.1016/j.exphem.2020.02.001">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXjs12qtbc%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32044376" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7103485" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20hematopoiesis%20in%20cancer&amp;journal=Exp.%20Hematol.&amp;doi=10.1016%2Fj.exphem.2020.02.001&amp;volume=83&amp;pages=105-112&amp;publication_year=2020&amp;author=Park%2CSJ&amp;author=Bejar%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR10">Terradas-Terradas, M., Robertson, N. A., Chandra, T. &amp; Kirschner, K. Clonality in haematopoietic stem cell ageing. <i>Mech. Ageing Dev.</i> <b>189</b>, 111279 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.mad.2020.111279" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.mad.2020.111279" data-track-item_id="10.1016/j.mad.2020.111279">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXht1Wrs7jM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32526214" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7347006" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonality%20in%20haematopoietic%20stem%20cell%20ageing&amp;journal=Mech.%20Ageing%20Dev.&amp;doi=10.1016%2Fj.mad.2020.111279&amp;volume=189&amp;publication_year=2020&amp;author=Terradas-Terradas%2CM&amp;author=Robertson%2CNA&amp;author=Chandra%2CT&amp;author=Kirschner%2CK" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR11">Challen, G. A. &amp; Goodell, M. A. Clonal hematopoiesis: mechanisms driving dominance of stem cell clones. <i>Blood</i> <b>136</b>, 1590–1598 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitF2nt77L" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32746453" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7530644" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Clonal%20hematopoiesis%3A%20mechanisms%20driving%20dominance%20of%20stem%20cell%20clones&amp;journal=Blood&amp;volume=136&amp;pages=1590-1598&amp;publication_year=2020&amp;author=Challen%2CGA&amp;author=Goodell%2CMA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR12">Shih, A. H., Abdel-Wahab, O., Patel, J. P. &amp; Levine, R. L. The role of mutations in epigenetic regulators in myeloid malignancies. <i>Nat. Rev. Cancer</i> <b>12</b>, 599–612 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnrc3343" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nrc3343" data-track-item_id="10.1038/nrc3343">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38Xht1Wkt7zF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=22898539" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20role%20of%20mutations%20in%20epigenetic%20regulators%20in%20myeloid%20malignancies&amp;journal=Nat.%20Rev.%20Cancer&amp;doi=10.1038%2Fnrc3343&amp;volume=12&amp;pages=599-612&amp;publication_year=2012&amp;author=Shih%2CAH&amp;author=Abdel-Wahab%2CO&amp;author=Patel%2CJP&amp;author=Levine%2CRL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR13">Steensma, D. P. &amp; Bolton, K. L. What to tell your patient with clonal hematopoiesis and why: insights from two specialized clinics. <i>Blood</i> <b>136</b>, 1623–1631 (2020).</p><p class="c-reading-companion__reference-links"><a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32736381" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7530645" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=What%20to%20tell%20your%20patient%20with%20clonal%20hematopoiesis%20and%20why%3A%20insights%20from%20two%20specialized%20clinics&amp;journal=Blood&amp;volume=136&amp;pages=1623-1631&amp;publication_year=2020&amp;author=Steensma%2CDP&amp;author=Bolton%2CKL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR14">Watson, C. J. et al. The evolutionary dynamics and fitness landscape of clonal hematopoiesis. <i>Science</i> <b>367</b>, 1449–1454 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1126%2Fscience.aay9333" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1126/science.aay9333" data-track-item_id="10.1126/science.aay9333">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXlslymtb4%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32217721" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20evolutionary%20dynamics%20and%20fitness%20landscape%20of%20clonal%20hematopoiesis&amp;journal=Science&amp;doi=10.1126%2Fscience.aay9333&amp;volume=367&amp;pages=1449-1454&amp;publication_year=2020&amp;author=Watson%2CCJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR15">Williams, M. J. et al. Measuring the distribution of fitness effects in somatic evolution by combining clonal dynamics with dN/dS ratios. <i>eLife</i> <b>9</b>, e48714 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.7554%2FeLife.48714" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.7554/eLife.48714" data-track-item_id="10.7554/eLife.48714">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhslWiu77K" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32223898" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7105384" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Measuring%20the%20distribution%20of%20fitness%20effects%20in%20somatic%20evolution%20by%20combining%20clonal%20dynamics%20with%20dN%2FdS%20ratios&amp;journal=eLife&amp;doi=10.7554%2FeLife.48714&amp;volume=9&amp;publication_year=2020&amp;author=Williams%2CMJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR16">Abelson, S. et al. Prediction of acute myeloid leukaemia risk in healthy individuals. <i>Nature</i> <b>559</b>, 400–404 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-018-0317-6" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-018-0317-6" data-track-item_id="10.1038/s41586-018-0317-6">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhtlWjs7rI" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29988082" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6485381" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Prediction%20of%20acute%20myeloid%20leukaemia%20risk%20in%20healthy%20individuals&amp;journal=Nature&amp;doi=10.1038%2Fs41586-018-0317-6&amp;volume=559&amp;pages=400-404&amp;publication_year=2018&amp;author=Abelson%2CS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR17">Taylor, A. M., Pattie, A. &amp; Deary, I. J. Cohort profile update: the Lothian Birth Cohorts of 1921 and 1936. <i>Int. J. Epidemiol.</i> <b>47</b>, 1042–1042r (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fije%2Fdyy022" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/ije/dyy022" data-track-item_id="10.1093/ije/dyy022">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=29546429" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6124629" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Cohort%20profile%20update%3A%20the%20Lothian%20Birth%20Cohorts%20of%201921%20and%201936&amp;journal=Int.%20J.%20Epidemiol.&amp;doi=10.1093%2Fije%2Fdyy022&amp;volume=47&amp;pages=1042-1042r&amp;publication_year=2018&amp;author=Taylor%2CAM&amp;author=Pattie%2CA&amp;author=Deary%2CIJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR18">Robertson, N. A. et al. Age-related clonal haemopoiesis is associated with increased epigenetic age. <i>Curr. Biol.</i> <b>29</b>, R786–R787 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.cub.2019.07.011" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.cub.2019.07.011" data-track-item_id="10.1016/j.cub.2019.07.011">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1alt7nO" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=31430471" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Age-related%20clonal%20haemopoiesis%20is%20associated%20with%20increased%20epigenetic%20age&amp;journal=Curr.%20Biol.&amp;doi=10.1016%2Fj.cub.2019.07.011&amp;volume=29&amp;pages=R786-R787&amp;publication_year=2019&amp;author=Robertson%2CNA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR19">McKerrell, T. et al. Leukemia-associated somatic mutations drive distinct patterns of age-related clonal hemopoiesis. <i>Cell Rep.</i> <b>10</b>, 1239–1245 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.celrep.2015.02.005" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.celrep.2015.02.005" data-track-item_id="10.1016/j.celrep.2015.02.005">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXjs1Omt7o%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=25732814" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4542313" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Leukemia-associated%20somatic%20mutations%20drive%20distinct%20patterns%20of%20age-related%20clonal%20hemopoiesis&amp;journal=Cell%20Rep.&amp;doi=10.1016%2Fj.celrep.2015.02.005&amp;volume=10&amp;pages=1239-1245&amp;publication_year=2015&amp;author=McKerrell%2CT" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR20">Tate, J. G. et al. COSMIC: the Catalogue Of Somatic Mutations In Cancer. <i>Nucleic Acids Res.</i> <b>47</b>, D941–D947 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgky1015" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gky1015" data-track-item_id="10.1093/nar/gky1015">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1MXhs1Cgt7bM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30371878" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=COSMIC%3A%20the%20Catalogue%20Of%20Somatic%20Mutations%20In%20Cancer&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgky1015&amp;volume=47&amp;pages=D941-D947&amp;publication_year=2019&amp;author=Tate%2CJG" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR21">Carter, H., Douville, C., Stenson, P. D., Cooper, D. N. &amp; Karchin, R. Identifying Mendelian disease genes with the Variant Effect Scoring Tool. <i>BMC Genomics</i> <b>14</b>, S3 (2013).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/1471-2164-14-S3-S3" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/1471-2164-14-S3-S3" data-track-item_id="10.1186/1471-2164-14-S3-S3">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23819870" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3665549" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Identifying%20Mendelian%20disease%20genes%20with%20the%20Variant%20Effect%20Scoring%20Tool&amp;journal=BMC%20Genomics&amp;doi=10.1186%2F1471-2164-14-S3-S3&amp;volume=14&amp;publication_year=2013&amp;author=Carter%2CH&amp;author=Douville%2CC&amp;author=Stenson%2CPD&amp;author=Cooper%2CDN&amp;author=Karchin%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR22">Hecht, M., Bromberg, Y. &amp; Rost, B. Better prediction of functional effects for sequence variants. <i>BMC Genomics</i> <b>16</b>, S1 (2015).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/1471-2164-16-S8-S1" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/1471-2164-16-S8-S1" data-track-item_id="10.1186/1471-2164-16-S8-S1">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26110438" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4480835" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Better%20prediction%20of%20functional%20effects%20for%20sequence%20variants&amp;journal=BMC%20Genomics&amp;doi=10.1186%2F1471-2164-16-S8-S1&amp;volume=16&amp;publication_year=2015&amp;author=Hecht%2CM&amp;author=Bromberg%2CY&amp;author=Rost%2CB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR23">Ioannidis, N. M. et al. REVEL: an ensemble method for predicting the pathogenicity of rare missense variants. <i>Am. J. Hum. Genet</i> <b>99</b>, 877–885 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1016%2Fj.ajhg.2016.08.016" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1016/j.ajhg.2016.08.016" data-track-item_id="10.1016/j.ajhg.2016.08.016">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XhsFGrurnM" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27666373" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5065685" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=REVEL%3A%20an%20ensemble%20method%20for%20predicting%20the%20pathogenicity%20of%20rare%20missense%20variants&amp;journal=Am.%20J.%20Hum.%20Genet&amp;doi=10.1016%2Fj.ajhg.2016.08.016&amp;volume=99&amp;pages=877-885&amp;publication_year=2016&amp;author=Ioannidis%2CNM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR24">Livesey, B. J. &amp; Marsh, J. A. Using deep mutational scanning to benchmark variant effect predictors and identify disease mutations. <i>Mol. Syst. Biol.</i> <b>16</b>, e9380 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.15252%2Fmsb.20199380" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.15252/msb.20199380" data-track-item_id="10.15252/msb.20199380">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhsFWqurvN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32627955" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7336272" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Using%20deep%20mutational%20scanning%20to%20benchmark%20variant%20effect%20predictors%20and%20identify%20disease%20mutations&amp;journal=Mol.%20Syst.%20Biol.&amp;doi=10.15252%2Fmsb.20199380&amp;volume=16&amp;publication_year=2020&amp;author=Livesey%2CBJ&amp;author=Marsh%2CJA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR25">Raimondi, D. et al. DEOGEN2: prediction and interactive visualization of single amino acid variant deleteriousness in human proteins. <i>Nucleic Acids Res.</i> <b>45</b>, W201–W206 (2017).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgkx390" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gkx390" data-track-item_id="10.1093/nar/gkx390">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhs1amtrg%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=28498993" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC5570203" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=DEOGEN2%3A%20prediction%20and%20interactive%20visualization%20of%20single%20amino%20acid%20variant%20deleteriousness%20in%20human%20proteins&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkx390&amp;volume=45&amp;pages=W201-W206&amp;publication_year=2017&amp;author=Raimondi%2CD" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR26">Riesselman, A. J., Ingraham, J. B. &amp; Marks, D. S. Deep generative models of genetic variation capture the effects of mutations. <i>Nat. Methods</i> <b>15</b>, 816–822 (2018).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41592-018-0138-4" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41592-018-0138-4" data-track-item_id="10.1038/s41592-018-0138-4">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC1cXhslymt77M" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=30250057" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC6693876" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Deep%20generative%20models%20of%20genetic%20variation%20capture%20the%20effects%20of%20mutations&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-018-0138-4&amp;volume=15&amp;pages=816-822&amp;publication_year=2018&amp;author=Riesselman%2CAJ&amp;author=Ingraham%2CJB&amp;author=Marks%2CDS" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR27">Vaser, R., Adusumalli, S., Leng, S. N., Sikic, M. &amp; Ng, P. C. SIFT missense predictions for genomes. <i>Nat. Protoc.</i> <b>11</b>, 1–9 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fnprot.2015.123" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/nprot.2015.123" data-track-item_id="10.1038/nprot.2015.123">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2MXhvFKht7fF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=26633127" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SIFT%20missense%20predictions%20for%20genomes&amp;journal=Nat.%20Protoc.&amp;doi=10.1038%2Fnprot.2015.123&amp;volume=11&amp;pages=1-9&amp;publication_year=2016&amp;author=Vaser%2CR&amp;author=Adusumalli%2CS&amp;author=Leng%2CSN&amp;author=Sikic%2CM&amp;author=Ng%2CPC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR28">Ley, T. J. et al. <i>DNMT3A</i> mutations in acute myeloid leukemia. <i>N. Engl. J. Med.</i> <b>363</b>, 2424–2433 (2010).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1056%2FNEJMoa1005143" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1056/NEJMoa1005143" data-track-item_id="10.1056/NEJMoa1005143">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC3cXhsF2ltb7K" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=21067377" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3201818" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=DNMT3A%20mutations%20in%20acute%20myeloid%20leukemia&amp;journal=N.%20Engl.%20J.%20Med.&amp;doi=10.1056%2FNEJMoa1005143&amp;volume=363&amp;pages=2424-2433&amp;publication_year=2010&amp;author=Ley%2CTJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR29">Zook, J. M. et al. Extensive sequencing of seven human genomes to characterize benchmark reference materials. <i>Sci. Data</i> <b>3</b>, 160025 (2016).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fsdata.2016.25" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/sdata.2016.25" data-track-item_id="10.1038/sdata.2016.25">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC28XpvVWrtLk%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=27271295" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4896128" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Extensive%20sequencing%20of%20seven%20human%20genomes%20to%20characterize%20benchmark%20reference%20materials&amp;journal=Sci.%20Data&amp;doi=10.1038%2Fsdata.2016.25&amp;volume=3&amp;publication_year=2016&amp;author=Zook%2CJM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR30">Bolger, A. M., Lohse, M. &amp; Usadel, B. Trimmomatic: a flexible trimmer for Illumina sequence data. <i>Bioinformatics</i> <b>30</b>, 2114–2120 (2014).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtu170" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btu170" data-track-item_id="10.1093/bioinformatics/btu170">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC2cXht1Sqt7nP" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=24695404" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC4103590" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Trimmomatic%3A%20a%20flexible%20trimmer%20for%20Illumina%20sequence%20data&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtu170&amp;volume=30&amp;pages=2114-2120&amp;publication_year=2014&amp;author=Bolger%2CAM&amp;author=Lohse%2CM&amp;author=Usadel%2CB" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR31">Li, H. &amp; Durbin, R. Fast and accurate short read alignment with Burrows–Wheeler transform. <i>Bioinformatics</i> <b>25</b>, 1754–1760 (2009).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fbioinformatics%2Fbtp324" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/bioinformatics/btp324" data-track-item_id="10.1093/bioinformatics/btp324">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BD1MXot1Cjtbo%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19451168" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2705234" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;journal=Bioinformatics&amp;doi=10.1093%2Fbioinformatics%2Fbtp324&amp;volume=25&amp;pages=1754-1760&amp;publication_year=2009&amp;author=Li%2CH&amp;author=Durbin%2CR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR32">Langmead, B., Trapnell, C., Pop, M. &amp; Salzberg, S. L. Ultrafast and memory-efficient alignment of short DNA sequences to the human genome. <i>Genome Biol.</i> <b>10</b>, R25 (2009).</p><p class="c-reading-companion__reference-links"><a href="https://link.springer.com/doi/10.1186/gb-2009-10-3-r25" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1186/gb-2009-10-3-r25" data-track-item_id="10.1186/gb-2009-10-3-r25">Article</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=19261174" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC2690996" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;journal=Genome%20Biol.&amp;doi=10.1186%2Fgb-2009-10-3-r25&amp;volume=10&amp;publication_year=2009&amp;author=Langmead%2CB&amp;author=Trapnell%2CC&amp;author=Pop%2CM&amp;author=Salzberg%2CSL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR33">Wilm, A. et al. LoFreq: a sequence-quality aware, ultra-sensitive variant caller for uncovering cell-population heterogeneity from high-throughput sequencing datasets. <i>Nucleic Acids Res.</i> <b>40</b>, 11189–11201 (2012).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgks918" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gks918" data-track-item_id="10.1093/nar/gks918">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BC38XhvVCitLjJ" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=23066108" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC3526318" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=LoFreq%3A%20a%20sequence-quality%20aware%2C%20ultra-sensitive%20variant%20caller%20for%20uncovering%20cell-population%20heterogeneity%20from%20high-throughput%20sequencing%20datasets&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgks918&amp;volume=40&amp;pages=11189-11201&amp;publication_year=2012&amp;author=Wilm%2CA" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR34">Garrison, E. &amp; Marth, G. Haplotype-based variant detection from short-read sequencing. Preprint at <a href="https://arxiv.org/abs/1207.3907" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://arxiv.org/abs/1207.3907">https://arxiv.org/abs/1207.3907</a> (2012).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR35">Karczewski, K. J. et al. The mutational constraint spectrum quantified from variation in 141,456 humans. <i>Nature</i> <b>581</b>, 434–443 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-020-2308-7" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-020-2308-7" data-track-item_id="10.1038/s41586-020-2308-7">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXhtVanu7jF" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32461654" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7334197" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=The%20mutational%20constraint%20spectrum%20quantified%20from%20variation%20in%20141%2C456%20humans&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2308-7&amp;volume=581&amp;pages=434-443&amp;publication_year=2020&amp;author=Karczewski%2CKJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR36">Jumper, J. et al. Highly accurate protein structure prediction with AlphaFold. <i>Nature</i> <b>596</b>, 583–589 (2021).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-021-03819-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-021-03819-2" data-track-item_id="10.1038/s41586-021-03819-2">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3MXhvVaktrrL" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34265844" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC8371605" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Highly%20accurate%20protein%20structure%20prediction%20with%20AlphaFold&amp;journal=Nature&amp;doi=10.1038%2Fs41586-021-03819-2&amp;volume=596&amp;pages=583-589&amp;publication_year=2021&amp;author=Jumper%2CJ" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR37">Varadi, M. et al. AlphaFold Protein Structure Database: massively expanding the structural coverage of protein-sequence space with high-accuracy models. <i>Nucleic Acids Res.</i> <b>50</b>, D439–D444 (2022).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1093%2Fnar%2Fgkab1061" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1093/nar/gkab1061" data-track-item_id="10.1093/nar/gkab1061">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB38Xis1Churw%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=34791371" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=AlphaFold%20Protein%20Structure%20Database%3A%20massively%20expanding%20the%20structural%20coverage%20of%20protein-sequence%20space%20with%20high-accuracy%20models&amp;journal=Nucleic%20Acids%20Res.&amp;doi=10.1093%2Fnar%2Fgkab1061&amp;volume=50&amp;pages=D439-D444&amp;publication_year=2022&amp;author=Varadi%2CM" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR38">Till, J. E., Mcculloch, E. A. &amp; Siminovitch, L. A stochastic model of stem cell proliferation, based on the growth of spleen colony-forming cells. <i>Proc. Natl Acad. Sci. USA</i> <b>51</b>, 29–36 (1964).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1073%2Fpnas.51.1.29" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1073/pnas.51.1.29" data-track-item_id="10.1073/pnas.51.1.29">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:STN:280:DyaF2c%2Fmt1artw%3D%3D" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=14104600" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC300599" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=A%20stochastic%20model%20of%20stem%20cell%20proliferation%2C%20based%20on%20the%20growth%20of%20spleen%20colony-forming%20cells&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;doi=10.1073%2Fpnas.51.1.29&amp;volume=51&amp;pages=29-36&amp;publication_year=1964&amp;author=Till%2CJE&amp;author=Mcculloch%2CEA&amp;author=Siminovitch%2CL" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR39">Bayes, T. An essay towards solving a problem in the doctrine of chances. By the late Rev. Mr. Bayes, F. R. S. communicated by Mr. Price, in a letter to John Canton, A. M. F. R. S. <a href="https://doi.org/10.1098/rstl.1763.0053" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="10.1098/rstl.1763.0053">https://doi.org/10.1098/rstl.1763.0053</a> (1763).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR40">Van Rossum, G. &amp; Drake, F. L. <i>Python 3 Reference Manual</i>. <a href="https://dl.acm.org/doi/book/10.5555/1593511" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://dl.acm.org/doi/book/10.5555/1593511">https://dl.acm.org/doi/book/10.5555/1593511</a> (CreateSpace, 2009).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR41">Harris, C. R. et al. Array programming with NumPy. <i>Nature</i> <b>585</b>, 357–362 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41586-020-2649-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41586-020-2649-2" data-track-item_id="10.1038/s41586-020-2649-2">Article</a>&nbsp;<a href="https://www.nature.com/articles/cas-redirect/1:CAS:528:DC%2BB3cXitlWmsbbN" data-track="click_references" data-track-action="cas reference" data-track-value="cas reference" data-track-label="link" data-track-item_id="link">CAS</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/entrez/query.fcgi?cmd=Retrieve&amp;db=PubMed&amp;dopt=Abstract&amp;list_uids=32939066" data-track="click_references" data-track-action="pubmed reference" data-track-value="pubmed reference" data-track-label="link" data-track-item_id="link">PubMed</a>&nbsp;<a href="http://www.ncbi.nlm.nih.gov/pmc/articles/PMC7759461" data-track="click_references" data-track-action="pubmed central reference" data-track-value="pubmed central reference" data-track-label="link" data-track-item_id="link">PubMed Central</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Array%20programming%20with%20NumPy&amp;journal=Nature&amp;doi=10.1038%2Fs41586-020-2649-2&amp;volume=585&amp;pages=357-362&amp;publication_year=2020&amp;author=Harris%2CCR" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR42">Virtanen, P. et al. SciPy 1.0: fundamental algorithms for scientific computing in Python. <i>Nat. Methods</i> <b>17</b>, 261–272 (2020).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.1038%2Fs41592-019-0686-2" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.1038/s41592-019-0686-2" data-track-item_id="10.1038/s41592-019-0686-2">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=SciPy%201.0%3A%20fundamental%20algorithms%20for%20scientific%20computing%20in%20Python&amp;journal=Nat.%20Methods&amp;doi=10.1038%2Fs41592-019-0686-2&amp;volume=17&amp;pages=261-272&amp;publication_year=2020&amp;author=Virtanen%2C%20P.%2C" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR43">Davidson-Pilon, C. lifelines: survival analysis in Python. <i>J. Open Source Softw.</i> <b>4</b>, 1317 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.21105%2Fjoss.01317" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.21105/joss.01317" data-track-item_id="10.21105/joss.01317">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=lifelines%3A%20survival%20analysis%20in%20Python&amp;journal=J.%20Open%20Source%20Softw.&amp;doi=10.21105%2Fjoss.01317&amp;volume=4&amp;publication_year=2019&amp;author=Davidson-Pilon%2CC" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR44">R Core Team. R: a language and environment for statistical computing. <a href="https://www.R-project.org/" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://www.R-project.org/">https://www.R-project.org/</a> (R Foundation for Statistical Computing, 2021).</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR45">Wickham, H. et al. Welcome to the tidyverse. <i>J. Open Source Softw.</i> <b>4</b>, 1686 (2019).</p><p class="c-reading-companion__reference-links"><a href="https://doi.org/10.21105%2Fjoss.01686" data-track="click_references" data-track-action="article reference" data-track-value="article reference" data-track-label="10.21105/joss.01686" data-track-item_id="10.21105/joss.01686">Article</a>&nbsp;<a href="http://scholar.google.com/scholar_lookup?&amp;title=Welcome%20to%20the%20tidyverse&amp;journal=J.%20Open%20Source%20Softw.&amp;doi=10.21105%2Fjoss.01686&amp;volume=4&amp;publication_year=2019&amp;author=Wickham%2CH" data-track="click_references" data-track-action="google scholar reference" data-track-value="google scholar reference" data-track-label="link" data-track-item_id="link">
                    Google Scholar</a>&nbsp;</p></li><li class="c-reading-companion__reference-item"><p class="c-reading-companion__reference-citation u-font-family-serif" id="rc-ref-CR46">Wickham, H. <i>ggplot2: Elegant Graphics for Data Analysis</i>. <a href="https://ggplot2.tidyverse.org" data-track="click_references" data-track-action="external reference" data-track-value="external reference" data-track-label="https://ggplot2.tidyverse.org">https://ggplot2.tidyverse.org</a> (Springer-Verlag, 2016).</p></li></ol></div></div>
        </div>
    </div>
</aside>
</div>


    
        
    
    
        
            
        

        
            
        
    
    <script>
        window.dataLayer = window.dataLayer || [];
        window.dataLayer.push({
            page: {
                content: {
                    fundingWidget: "true",
                        }
                    }
                });
    </script>




<footer class="composite-layer" itemscope="" itemtype="http://schema.org/Periodical">
        <meta itemprop="publisher" content="Springer Nature">
        

        <div class="u-mt-16 u-mb-16">
    <div class="u-container">
        <div class="u-display-flex u-flex-wrap u-justify-content-space-between">
            

            <div class="c-meta u-ma-0 u-flex-shrink">
                <p class="c-meta__item c-meta__type u-mt-0">
                    <span itemprop="name">
                        Nature Medicine
                    </span>
                    (<i itemprop="alternateName">Nat Med</i>)
                </p>
                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1546-170X</span> (online)
        </p>
    

                
    
        <p class="c-meta__item u-mt-0">
            <abbr title="International Standard Serial Number">ISSN</abbr> <span itemprop="issn">1078-8956</span> (print)
        </p>
    

            </div>
        </div>
    </div>
</div>

    <div class="c-footer">
        <div class="u-hide-print" data-track-component="footer">
    <h2 class="u-visually-hidden">nature.com sitemap</h2>
    <div class="c-footer__container">
        <div class="c-footer__grid c-footer__group--separator">
            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">About Nature Portfolio</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/company_info/index.html" data-track="click" data-track-action="about us" data-track-label="link">About us</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/npg_/press_room/press_releases.html" data-track="click" data-track-action="press releases" data-track-label="link">Press releases</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://press.nature.com/" data-track="click" data-track-action="press office" data-track-label="link">Press office</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://support.nature.com/support/home" data-track="click" data-track-action="contact us" data-track-label="link">Contact us</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Discover content</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/siteindex" data-track="click" data-track-action="journals a-z" data-track-label="link">Journals A-Z</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/subjects" data-track="click" data-track-action="article by subject" data-track-label="link">Articles by subject</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.protocols.io/" data-track="click" data-track-action="protocols.io" data-track-label="link">protocols.io</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureindex.com/" data-track="click" data-track-action="nature index" data-track-label="link">Nature Index</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Publishing policies</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/authors/editorial_policies" data-track="click" data-track-action="Nature portfolio policies" data-track-label="link">Nature portfolio policies</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nature-research/open-access" data-track="click" data-track-action="open access" data-track-label="link">Open access</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Author &amp; Researcher services</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/reprints" data-track="click" data-track-action="reprints and permissions" data-track-label="link">Reprints &amp; permissions</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/authors/research-data" data-track="click" data-track-action="data research service" data-track-label="link">Research data</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/language-editing/" data-track="click" data-track-action="language editing" data-track-label="link">Language editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://authorservices.springernature.com/scientific-editing/" data-track="click" data-track-action="scientific editing" data-track-label="link">Scientific editing</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://masterclasses.nature.com/" data-track="click" data-track-action="nature masterclasses" data-track-label="link">Nature Masterclasses</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://solutions.springernature.com/" data-track="click" data-track-action="research solutions" data-track-label="link">Research Solutions</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Libraries &amp; institutions</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/tools-services" data-track="click" data-track-action="librarian service and tools" data-track-label="link">Librarian service &amp; tools</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/manage-your-account/librarianportal" data-track="click" data-track-action="librarian portal" data-track-label="link">Librarian portal</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/openresearch/about-open-access/information-for-institutions" data-track="click" data-track-action="open research" data-track-label="link">Open research</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/gp/librarians/recommend-to-your-library" data-track="click" data-track-action="Recommend to library" data-track-label="link">Recommend to library</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Advertising &amp; partnerships</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/digital-advertising/" data-track="click" data-track-action="advertising" data-track-label="link">Advertising</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/" data-track="click" data-track-action="partnerships and services" data-track-label="link">Partnerships &amp; Services</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/media-kits/" data-track="click" data-track-action="media kits" data-track-label="link">Media kits</a>
                    </li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://partnerships.nature.com/product/branded-content-native-advertising/" data-track-action="branded content" data-track-label="link">Branded
                        content</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Professional development</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/naturecareers/" data-track="click" data-track-action="nature careers" data-track-label="link">Nature Careers</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://conferences.nature.com" data-track="click" data-track-action="nature conferences" data-track-label="link">Nature<span class="u-visually-hidden"> </span>
                        Conferences</a></li>
                </ul>
            </div>

            <div class="c-footer__group">
                <h3 class="c-footer__heading u-mt-0">Regional websites</h3>
                <ul class="c-footer__list">
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natafrica" data-track="click" data-track-action="nature africa" data-track-label="link">Nature Africa</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="http://www.naturechina.com" data-track="click" data-track-action="nature china" data-track-label="link">Nature China</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nindia" data-track="click" data-track-action="nature india" data-track-label="link">Nature India</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/natitaly" data-track="click" data-track-action="nature Italy" data-track-label="link">Nature Italy</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.natureasia.com/ja-jp" data-track="click" data-track-action="nature japan" data-track-label="link">Nature Japan</a></li>
                    <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/nmiddleeast" data-track="click" data-track-action="nature middle east" data-track-label="link">Nature Middle East</a></li>
                </ul>
            </div>

        </div>
    </div>
    <div class="c-footer__container">
        <ul class="c-footer__links">
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/privacy" data-track="click" data-track-action="privacy policy" data-track-label="link">Privacy
                Policy</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/cookies" data-track="click" data-track-action="use of cookies" data-track-label="link">Use
                of cookies</a></li>
            <li class="c-footer__item">
                <button class="optanon-toggle-display c-footer__link" onclick="javascript:;" data-cc-action="preferences" data-track="click" data-track-action="manage cookies" data-track-label="link">Your privacy choices/Manage cookies
                </button>
            </li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/legal-notice" data-track="click" data-track-action="legal notice" data-track-label="link">Legal
                notice</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/accessibility-statement" data-track="click" data-track-action="accessibility statement" data-track-label="link">Accessibility
                statement</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.nature.com/info/terms-and-conditions" data-track="click" data-track-action="terms and conditions" data-track-label="link">Terms &amp; Conditions</a></li>
            <li class="c-footer__item"><a class="c-footer__link" href="https://www.springernature.com/ccpa" data-track="click" data-track-action="california privacy statement" data-track-label="link">Your US state privacy rights</a></li>
            
        </ul>
    </div>
</div>


        <div class="c-footer__container">
    <a href="https://www.springernature.com/" class="c-footer__link">
        <img src="/static/images/logos/sn-logo-white-ea63208b81.svg" alt="Springer Nature" loading="lazy" width="200" height="20">
    </a>
    <p class="c-footer__legal" data-test="copyright">© 2025 Springer Nature Limited</p>
</div>

    </div>
    <div class="u-visually-hidden" aria-hidden="true">
    
    <!--?xml version="1.0" encoding="UTF-8"?--><svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink"><defs><path id="a" d="M0 .74h56.72v55.24H0z"></path></defs><symbol id="icon-access" viewBox="0 0 18 18"><path d="m14 8c.5522847 0 1 .44771525 1 1v7h2.5c.2761424 0 .5.2238576.5.5v1.5h-18v-1.5c0-.2761424.22385763-.5.5-.5h2.5v-7c0-.55228475.44771525-1 1-1s1 .44771525 1 1v6.9996556h8v-6.9996556c0-.55228475.4477153-1 1-1zm-8 0 2 1v5l-2 1zm6 0v7l-2-1v-5zm-2.42653766-7.59857636 7.03554716 4.92488299c.4162533.29137735.5174853.86502537.226108 1.28127873-.1721584.24594054-.4534847.39241464-.7536934.39241464h-14.16284822c-.50810197 0-.92-.41189803-.92-.92 0-.30020869.1464741-.58153499.39241464-.75369337l7.03554714-4.92488299c.34432015-.2410241.80260453-.2410241 1.14692468 0zm-.57346234 2.03988748-3.65526982 2.55868888h7.31053962z" fill-rule="evenodd"></path></symbol><symbol id="icon-account" viewBox="0 0 18 18"><path d="m10.2379028 16.9048051c1.3083556-.2032362 2.5118471-.7235183 3.5294683-1.4798399-.8731327-2.5141501-2.0638925-3.935978-3.7673711-4.3188248v-1.27684611c1.1651924-.41183641 2-1.52307546 2-2.82929429 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.30621883.83480763 2.41745788 2 2.82929429v1.27684611c-1.70347856.3828468-2.89423845 1.8046747-3.76737114 4.3188248 1.01762123.7563216 2.22111275 1.2766037 3.52946833 1.4798399.40563808.0629726.81921174.0951949 1.23790281.0951949s.83226473-.0322223 1.2379028-.0951949zm4.3421782-2.1721994c1.4927655-1.4532925 2.419919-3.484675 2.419919-5.7326057 0-4.418278-3.581722-8-8-8s-8 3.581722-8 8c0 2.2479307.92715352 4.2793132 2.41991895 5.7326057.75688473-2.0164459 1.83949951-3.6071894 3.48926591-4.3218837-1.14534283-.70360829-1.90918486-1.96796271-1.90918486-3.410722 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.44275929-.763842 2.70711371-1.9091849 3.410722 1.6497664.7146943 2.7323812 2.3054378 3.4892659 4.3218837zm-5.580081 3.2673943c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-alert" viewBox="0 0 18 18"><path d="m4 10h2.5c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-3.08578644l-1.12132034 1.1213203c-.18753638.1875364-.29289322.4418903-.29289322.7071068v.1715729h14v-.1715729c0-.2652165-.1053568-.5195704-.2928932-.7071068l-1.7071068-1.7071067v-3.4142136c0-2.76142375-2.2385763-5-5-5-2.76142375 0-5 2.23857625-5 5zm3 4c0 1.1045695.8954305 2 2 2s2-.8954305 2-2zm-5 0c-.55228475 0-1-.4477153-1-1v-.1715729c0-.530433.21071368-1.0391408.58578644-1.4142135l1.41421356-1.4142136v-3c0-3.3137085 2.6862915-6 6-6s6 2.6862915 6 6v3l1.4142136 1.4142136c.3750727.3750727.5857864.8837805.5857864 1.4142135v.1715729c0 .5522847-.4477153 1-1 1h-4c0 1.6568542-1.3431458 3-3 3-1.65685425 0-3-1.3431458-3-3z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-broad" viewBox="0 0 16 16"><path d="m6.10307866 2.97190702v7.69043288l2.44965196-2.44676915c.38776071-.38730439 1.0088052-.39493524 1.38498697-.01919617.38609051.38563612.38643641 1.01053024-.00013864 1.39665039l-4.12239817 4.11754683c-.38616704.3857126-1.01187344.3861062-1.39846576-.0000311l-4.12258206-4.11773056c-.38618426-.38572979-.39254614-1.00476697-.01636437-1.38050605.38609047-.38563611 1.01018509-.38751562 1.4012233.00306241l2.44985644 2.4469734v-8.67638639c0-.54139983.43698413-.98042709.98493125-.98159081l7.89910522-.0043627c.5451687 0 .9871152.44142642.9871152.98595351s-.4419465.98595351-.9871152.98595351z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 14 15)"></path></symbol><symbol id="icon-arrow-down" viewBox="0 0 16 16"><path d="m3.28337502 11.5302405 4.03074001 4.176208c.37758093.3912076.98937525.3916069 1.367372-.0000316l4.03091977-4.1763942c.3775978-.3912252.3838182-1.0190815.0160006-1.4001736-.3775061-.39113013-.9877245-.39303641-1.3700683.003106l-2.39538585 2.4818345v-11.6147896l-.00649339-.11662112c-.055753-.49733869-.46370161-.88337888-.95867408-.88337888-.49497246 0-.90292107.38604019-.95867408.88337888l-.00649338.11662112v11.6147896l-2.39518594-2.4816273c-.37913917-.39282218-.98637524-.40056175-1.35419292-.0194697-.37750607.3911302-.37784433 1.0249269.00013556 1.4165479z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-left" viewBox="0 0 16 16"><path d="m4.46975946 3.28337502-4.17620792 4.03074001c-.39120768.37758093-.39160691.98937525.0000316 1.367372l4.1763942 4.03091977c.39122514.3775978 1.01908149.3838182 1.40017357.0160006.39113012-.3775061.3930364-.9877245-.00310603-1.3700683l-2.48183446-2.39538585h11.61478958l.1166211-.00649339c.4973387-.055753.8833789-.46370161.8833789-.95867408 0-.49497246-.3860402-.90292107-.8833789-.95867408l-.1166211-.00649338h-11.61478958l2.4816273-2.39518594c.39282216-.37913917.40056173-.98637524.01946965-1.35419292-.39113012-.37750607-1.02492687-.37784433-1.41654791.00013556z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-right" viewBox="0 0 16 16"><path d="m11.5302405 12.716625 4.176208-4.03074003c.3912076-.37758093.3916069-.98937525-.0000316-1.367372l-4.1763942-4.03091981c-.3912252-.37759778-1.0190815-.38381821-1.4001736-.01600053-.39113013.37750607-.39303641.98772445.003106 1.37006824l2.4818345 2.39538588h-11.6147896l-.11662112.00649339c-.49733869.055753-.88337888.46370161-.88337888.95867408 0 .49497246.38604019.90292107.88337888.95867408l.11662112.00649338h11.6147896l-2.4816273 2.39518592c-.39282218.3791392-.40056175.9863753-.0194697 1.3541929.3911302.3775061 1.0249269.3778444 1.4165479-.0001355z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-sub" viewBox="0 0 16 16"><path d="m7.89692134 4.97190702v7.69043288l-2.44965196-2.4467692c-.38776071-.38730434-1.0088052-.39493519-1.38498697-.0191961-.38609047.3856361-.38643643 1.0105302.00013864 1.3966504l4.12239817 4.1175468c.38616704.3857126 1.01187344.3861062 1.39846576-.0000311l4.12258202-4.1177306c.3861843-.3857298.3925462-1.0047669.0163644-1.380506-.3860905-.38563612-1.0101851-.38751563-1.4012233.0030624l-2.44985643 2.4469734v-8.67638639c0-.54139983-.43698413-.98042709-.98493125-.98159081l-7.89910525-.0043627c-.54516866 0-.98711517.44142642-.98711517.98595351s.44194651.98595351.98711517.98595351z" fill-rule="evenodd"></path></symbol><symbol id="icon-arrow-up" viewBox="0 0 16 16"><path d="m12.716625 4.46975946-4.03074003-4.17620792c-.37758093-.39120768-.98937525-.39160691-1.367372.0000316l-4.03091981 4.1763942c-.37759778.39122514-.38381821 1.01908149-.01600053 1.40017357.37750607.39113012.98772445.3930364 1.37006824-.00310603l2.39538588-2.48183446v11.61478958l.00649339.1166211c.055753.4973387.46370161.8833789.95867408.8833789.49497246 0 .90292107-.3860402.95867408-.8833789l.00649338-.1166211v-11.61478958l2.39518592 2.4816273c.3791392.39282216.9863753.40056173 1.3541929.01946965.3775061-.39113012.3778444-1.02492687-.0001355-1.41654791z" fill-rule="evenodd"></path></symbol><symbol id="icon-article" viewBox="0 0 18 18"><path d="m13 15v-12.9906311c0-.0073595-.0019884-.0093689.0014977-.0093689l-11.00158888.00087166v13.00506804c0 .5482678.44615281.9940603.99415146.9940603h10.27350412c-.1701701-.2941734-.2675644-.6357129-.2675644-1zm-12 .0059397v-13.00506804c0-.5562408.44704472-1.00087166.99850233-1.00087166h11.00299537c.5510129 0 .9985023.45190985.9985023 1.0093689v2.9906311h3v9.9914698c0 1.1065798-.8927712 2.0085302-1.9940603 2.0085302h-12.01187942c-1.09954652 0-1.99406028-.8927712-1.99406028-1.9940603zm13-9.0059397v9c0 .5522847.4477153 1 1 1s1-.4477153 1-1v-9zm-10-2h7v4h-7zm1 1v2h5v-2zm-1 4h7v1h-7zm0 2h7v1h-7zm0 2h7v1h-7z" fill-rule="evenodd"></path></symbol><symbol id="icon-audio" viewBox="0 0 18 18"><path d="m13.0957477 13.5588459c-.195279.1937043-.5119137.193729-.7072234.0000551-.1953098-.193674-.1953346-.5077061-.0000556-.7014104 1.0251004-1.0168342 1.6108711-2.3905226 1.6108711-3.85745208 0-1.46604976-.5850634-2.83898246-1.6090736-3.85566829-.1951894-.19379323-.1950192-.50782531.0003802-.70141028.1953993-.19358497.512034-.19341614.7072234.00037709 1.2094886 1.20083761 1.901635 2.8250555 1.901635 4.55670148 0 1.73268608-.6929822 3.35779608-1.9037571 4.55880738zm2.1233994 2.1025159c-.195234.193749-.5118687.1938462-.7072235.0002171-.1953548-.1936292-.1954528-.5076613-.0002189-.7014104 1.5832215-1.5711805 2.4881302-3.6939808 2.4881302-5.96012998 0-2.26581266-.9046382-4.3883241-2.487443-5.95944795-.1952117-.19377107-.1950777-.50780316.0002993-.70141031s.5120117-.19347426.7072234.00029682c1.7683321 1.75528196 2.7800854 4.12911258 2.7800854 6.66056144 0 2.53182498-1.0120556 4.90597838-2.7808529 6.66132328zm-14.21898205-3.6854911c-.5523759 0-1.00016505-.4441085-1.00016505-.991944v-3.96777631c0-.54783558.44778915-.99194407 1.00016505-.99194407h2.0003301l5.41965617-3.8393633c.44948677-.31842296 1.07413994-.21516983 1.39520191.23062232.12116339.16823446.18629727.36981184.18629727.57655577v12.01603479c0 .5478356-.44778914.9919441-1.00016505.9919441-.20845738 0-.41170538-.0645985-.58133413-.184766l-5.41965617-3.8393633zm0-.991944h2.32084805l5.68047235 4.0241292v-12.01603479l-5.68047235 4.02412928h-2.32084805z" fill-rule="evenodd"></path></symbol><symbol id="icon-block" viewBox="0 0 24 24"><path d="m0 0h24v24h-24z" fill-rule="evenodd"></path></symbol><symbol id="icon-book" viewBox="0 0 18 18"><path d="m4 13v-11h1v11h11v-11h-13c-.55228475 0-1 .44771525-1 1v10.2675644c.29417337-.1701701.63571286-.2675644 1-.2675644zm12 1h-13c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1h13zm0 3h-13c-1.1045695 0-2-.8954305-2-2v-12c0-1.1045695.8954305-2 2-2h13c.5522847 0 1 .44771525 1 1v14c0 .5522847-.4477153 1-1 1zm-8.5-13h6c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-6c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1 2h4c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-4c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-broad" viewBox="0 0 24 24"><path d="m9.18274226 7.81v7.7999954l2.48162734-2.4816273c.3928221-.3928221 1.0219731-.4005617 1.4030652-.0194696.3911301.3911301.3914806 1.0249268-.0001404 1.4165479l-4.17620796 4.1762079c-.39120769.3912077-1.02508144.3916069-1.41671995-.0000316l-4.1763942-4.1763942c-.39122514-.3912251-.39767006-1.0190815-.01657798-1.4001736.39113012-.3911301 1.02337106-.3930364 1.41951349.0031061l2.48183446 2.4818344v-8.7999954c0-.54911294.4426881-.99439484.99778758-.99557515l8.00221246-.00442485c.5522847 0 1 .44771525 1 1s-.4477153 1-1 1z" fill-rule="evenodd" transform="matrix(-1 0 0 -1 20.182742 24.805206)"></path></symbol><symbol id="icon-calendar" viewBox="0 0 18 18"><path d="m12.5 0c.2761424 0 .5.21505737.5.49047852v.50952148h2c1.1072288 0 2 .89451376 2 2v12c0 1.1072288-.8945138 2-2 2h-12c-1.1072288 0-2-.8945138-2-2v-12c0-1.1072288.89451376-2 2-2h1v1h-1c-.55393837 0-1 .44579254-1 1v3h14v-3c0-.55393837-.4457925-1-1-1h-2v1.50952148c0 .27088381-.2319336.49047852-.5.49047852-.2761424 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.2319336-.49047852.5-.49047852zm3.5 7h-14v8c0 .5539384.44579254 1 1 1h12c.5539384 0 1-.4457925 1-1zm-11 6v1h-1v-1zm3 0v1h-1v-1zm3 0v1h-1v-1zm-6-2v1h-1v-1zm3 0v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-3-2v1h-1v-1zm6 0v1h-1v-1zm-3 0v1h-1v-1zm-5.5-9c.27614237 0 .5.21505737.5.49047852v.50952148h5v1h-5v1.50952148c0 .27088381-.23193359.49047852-.5.49047852-.27614237 0-.5-.21505737-.5-.49047852v-3.01904296c0-.27088381.23193359-.49047852.5-.49047852z" fill-rule="evenodd"></path></symbol><symbol id="icon-cart" viewBox="0 0 18 18"><path d="m5 14c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm10 0c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm-10 1c-.55228475 0-1 .4477153-1 1s.44771525 1 1 1 1-.4477153 1-1-.44771525-1-1-1zm10 0c-.5522847 0-1 .4477153-1 1s.4477153 1 1 1 1-.4477153 1-1-.4477153-1-1-1zm-12.82032249-15c.47691417 0 .88746157.33678127.98070211.80449199l.23823144 1.19501025 13.36277974.00045554c.5522847.00001882.9999659.44774934.9999659 1.00004222 0 .07084994-.0075361.14150708-.022474.2107727l-1.2908094 5.98534344c-.1007861.46742419-.5432548.80388386-1.0571651.80388386h-10.24805106c-.59173366 0-1.07142857.4477153-1.07142857 1 0 .5128358.41361449.9355072.94647737.9932723l.1249512.0067277h10.35933776c.2749512 0 .4979349.2228539.4979349.4978051 0 .2749417-.2227336.4978951-.4976753.4980063l-10.35959736.0041886c-1.18346732 0-2.14285714-.8954305-2.14285714-2 0-.6625717.34520317-1.24989198.87690425-1.61383592l-1.63768102-8.19004794c-.01312273-.06561364-.01950005-.131011-.0196107-.19547395l-1.71961253-.00064219c-.27614237 0-.5-.22385762-.5-.5 0-.27614237.22385763-.5.5-.5zm14.53193359 2.99950224h-13.11300004l1.20580469 6.02530174c.11024034-.0163252.22327998-.02480398.33844139-.02480398h10.27064786z"></path></symbol><symbol id="icon-chevron-less" viewBox="0 0 10 10"><path d="m5.58578644 4-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 -1 -1 0 9 9)"></path></symbol><symbol id="icon-chevron-more" viewBox="0 0 10 10"><path d="m5.58578644 6-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4.00000002c-.39052429.3905243-1.02368927.3905243-1.41421356 0s-.39052429-1.02368929 0-1.41421358z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-chevron-right" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-circle-fill" viewBox="0 0 16 16"><path d="m8 14c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-circle" viewBox="0 0 16 16"><path d="m8 12c2.209139 0 4-1.790861 4-4s-1.790861-4-4-4-4 1.790861-4 4 1.790861 4 4 4zm0 2c-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6 6 2.6862915 6 6-2.6862915 6-6 6z" fill-rule="evenodd"></path></symbol><symbol id="icon-citation" viewBox="0 0 18 18"><path d="m8.63593473 5.99995183c2.20913897 0 3.99999997 1.79084375 3.99999997 3.99996146 0 1.40730761-.7267788 2.64486871-1.8254829 3.35783281 1.6240224.6764218 2.8754442 2.0093871 3.4610603 3.6412466l-1.0763845.000006c-.5310008-1.2078237-1.5108121-2.1940153-2.7691712-2.7181346l-.79002167-.329052v-1.023992l.63016577-.4089232c.8482885-.5504661 1.3698342-1.4895187 1.3698342-2.51898361 0-1.65683828-1.3431457-2.99996146-2.99999997-2.99996146-1.65685425 0-3 1.34312318-3 2.99996146 0 1.02946491.52154569 1.96851751 1.36983419 2.51898361l.63016581.4089232v1.023992l-.79002171.329052c-1.25835905.5241193-2.23817037 1.5103109-2.76917113 2.7181346l-1.07638453-.000006c.58561612-1.6318595 1.8370379-2.9648248 3.46106024-3.6412466-1.09870405-.7129641-1.82548287-1.9505252-1.82548287-3.35783281 0-2.20911771 1.790861-3.99996146 4-3.99996146zm7.36897597-4.99995183c1.1018574 0 1.9950893.89353404 1.9950893 2.00274083v5.994422c0 1.10608317-.8926228 2.00274087-1.9950893 2.00274087l-3.0049107-.0009037v-1l3.0049107.00091329c.5490631 0 .9950893-.44783123.9950893-1.00275046v-5.994422c0-.55646537-.4450595-1.00275046-.9950893-1.00275046h-14.00982141c-.54906309 0-.99508929.44783123-.99508929 1.00275046v5.9971821c0 .66666024.33333333.99999036 1 .99999036l2-.00091329v1l-2 .0009037c-1 0-2-.99999041-2-1.99998077v-5.9971821c0-1.10608322.8926228-2.00274083 1.99508929-2.00274083zm-8.5049107 2.9999711c.27614237 0 .5.22385547.5.5 0 .2761349-.22385763.5-.5.5h-4c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm3 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-1c-.27614237 0-.5-.2238651-.5-.5 0-.27614453.22385763-.5.5-.5zm4 0c.2761424 0 .5.22385547.5.5 0 .2761349-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238651-.5-.5 0-.27614453.2238576-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-close" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-collections" viewBox="0 0 18 18"><path d="m15 4c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2h1c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227l-.1166211-.00672773h-1v-1zm-4-3c1.1045695 0 2 .8954305 2 2v9c0 1.1045695-.8954305 2-2 2h-8c-1.1045695 0-2-.8954305-2-2v-9c0-1.1045695.8954305-2 2-2zm0 1h-8c-.51283584 0-.93550716.38604019-.99327227.88337887l-.00672773.11662113v9c0 .5128358.38604019.9355072.88337887.9932723l.11662113.0067277h8c.5128358 0 .9355072-.3860402.9932723-.8833789l.0067277-.1166211v-9c0-.51283584-.3860402-.93550716-.8833789-.99327227zm-1.5 7c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm0-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-5c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-compare" viewBox="0 0 18 18"><path d="m12 3c3.3137085 0 6 2.6862915 6 6s-2.6862915 6-6 6c-1.0928452 0-2.11744941-.2921742-2.99996061-.8026704-.88181407.5102749-1.90678042.8026704-3.00003939.8026704-3.3137085 0-6-2.6862915-6-6s2.6862915-6 6-6c1.09325897 0 2.11822532.29239547 3.00096303.80325037.88158756-.51107621 1.90619177-.80325037 2.99903697-.80325037zm-6 1c-2.76142375 0-5 2.23857625-5 5 0 2.7614237 2.23857625 5 5 5 .74397391 0 1.44999672-.162488 2.08451611-.4539116-1.27652344-1.1000812-2.08451611-2.7287264-2.08451611-4.5460884s.80799267-3.44600721 2.08434391-4.5463015c-.63434719-.29121054-1.34037-.4536985-2.08434391-.4536985zm6 0c-.7439739 0-1.4499967.16248796-2.08451611.45391156 1.27652341 1.10008123 2.08451611 2.72872644 2.08451611 4.54608844s-.8079927 3.4460072-2.08434391 4.5463015c.63434721.2912105 1.34037001.4536985 2.08434391.4536985 2.7614237 0 5-2.2385763 5-5 0-2.76142375-2.2385763-5-5-5zm-1.4162763 7.0005324h-3.16744736c.15614659.3572676.35283837.6927622.58425872 1.0006671h1.99892988c.23142036-.3079049.42811216-.6433995.58425876-1.0006671zm.4162763-2.0005324h-4c0 .34288501.0345146.67770871.10025909 1.0011864h3.79948181c.0657445-.32347769.1002591-.65830139.1002591-1.0011864zm-.4158423-1.99953894h-3.16831543c-.13859957.31730812-.24521946.651783-.31578599.99935097h3.79988742c-.0705665-.34756797-.1771864-.68204285-.315786-.99935097zm-1.58295822-1.999926-.08316107.06199199c-.34550042.27081213-.65446126.58611297-.91825862.93727862h2.00044041c-.28418626-.37830727-.6207872-.71499149-.99902072-.99927061z" fill-rule="evenodd"></path></symbol><symbol id="icon-download-file" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.5046024 4c.27614237 0 .5.21637201.5.49209595v6.14827645l1.7462789-1.77990922c.1933927-.1971171.5125222-.19455839.7001689-.0069117.1932998.19329992.1910058.50899492-.0027774.70277812l-2.59089271 2.5908927c-.19483374.1948337-.51177825.1937771-.70556873-.0000133l-2.59099079-2.5909908c-.19484111-.1948411-.19043735-.5151448-.00279066-.70279146.19329987-.19329987.50465175-.19237083.70018565.00692852l1.74638684 1.78001764v-6.14827695c0-.27177709.23193359-.49209595.5-.49209595z" fill-rule="evenodd"></path></symbol><symbol id="icon-download" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-editors" viewBox="0 0 18 18"><path d="m8.72592184 2.54588137c-.48811714-.34391207-1.08343326-.54588137-1.72592184-.54588137-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400182l-.79002171.32905522c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274v.9009805h-1v-.9009805c0-2.5479714 1.54557359-4.79153984 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4 1.09079823 0 2.07961816.43662103 2.80122451 1.1446278-.37707584.09278571-.7373238.22835063-1.07530267.40125357zm-2.72592184 14.45411863h-1v-.9009805c0-2.5479714 1.54557359-4.7915398 3.82548288-5.7411543-1.09870406-.71297106-1.82548288-1.95054399-1.82548288-3.3578652 0-2.209139 1.790861-4 4-4s4 1.790861 4 4c0 1.40732121-.7267788 2.64489414-1.8254829 3.3578652 2.2799093.9496145 3.8254829 3.1931829 3.8254829 5.7411543v.9009805h-1v-.9009805c0-2.1155483-1.2760206-4.0125067-3.2099783-4.8180274l-.7900217-.3290552v-1.02400184l.6301658-.40892721c.8482885-.55047139 1.3698342-1.489533 1.3698342-2.51900785 0-1.65685425-1.3431458-3-3-3-1.65685425 0-3 1.34314575-3 3 0 1.02947485.5215457 1.96853646 1.3698342 2.51900785l.6301658.40892721v1.02400184l-.79002171.3290552c-1.93395773.8055207-3.20997829 2.7024791-3.20997829 4.8180274z" fill-rule="evenodd"></path></symbol><symbol id="icon-email" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-.0049107 2.55749512v1.44250488l-7 4-7-4v-1.44250488l7 4z" fill-rule="evenodd"></path></symbol><symbol id="icon-error" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm2.8630343 4.71100931-2.8630343 2.86303426-2.86303426-2.86303426c-.39658757-.39658757-1.03281091-.39438847-1.4265779-.00062147-.39651227.39651226-.39348876 1.03246767.00062147 1.4265779l2.86303426 2.86303426-2.86303426 2.8630343c-.39658757.3965875-.39438847 1.0328109-.00062147 1.4265779.39651226.3965122 1.03246767.3934887 1.4265779-.0006215l2.86303426-2.8630343 2.8630343 2.8630343c.3965875.3965876 1.0328109.3943885 1.4265779.0006215.3965122-.3965123.3934887-1.0324677-.0006215-1.4265779l-2.8630343-2.8630343 2.8630343-2.86303426c.3965876-.39658757.3943885-1.03281091.0006215-1.4265779-.3965123-.39651227-1.0324677-.39348876-1.4265779.00062147z" fill-rule="evenodd"></path></symbol><symbol id="icon-ethics" viewBox="0 0 18 18"><path d="m6.76384967 1.41421356.83301651-.8330165c.77492941-.77492941 2.03133823-.77492941 2.80626762 0l.8330165.8330165c.3750728.37507276.8837806.58578644 1.4142136.58578644h1.3496361c1.1045695 0 2 .8954305 2 2v1.34963611c0 .53043298.2107137 1.03914081.5857864 1.41421356l.8330165.83301651c.7749295.77492941.7749295 2.03133823 0 2.80626762l-.8330165.8330165c-.3750727.3750728-.5857864.8837806-.5857864 1.4142136v1.3496361c0 1.1045695-.8954305 2-2 2h-1.3496361c-.530433 0-1.0391408.2107137-1.4142136.5857864l-.8330165.8330165c-.77492939.7749295-2.03133821.7749295-2.80626762 0l-.83301651-.8330165c-.37507275-.3750727-.88378058-.5857864-1.41421356-.5857864h-1.34963611c-1.1045695 0-2-.8954305-2-2v-1.3496361c0-.530433-.21071368-1.0391408-.58578644-1.4142136l-.8330165-.8330165c-.77492941-.77492939-.77492941-2.03133821 0-2.80626762l.8330165-.83301651c.37507276-.37507275.58578644-.88378058.58578644-1.41421356v-1.34963611c0-1.1045695.8954305-2 2-2h1.34963611c.53043298 0 1.03914081-.21071368 1.41421356-.58578644zm-1.41421356 1.58578644h-1.34963611c-.55228475 0-1 .44771525-1 1v1.34963611c0 .79564947-.31607052 1.55871121-.87867966 2.12132034l-.8330165.83301651c-.38440512.38440512-.38440512 1.00764896 0 1.39205408l.8330165.83301646c.56260914.5626092.87867966 1.3256709.87867966 2.1213204v1.3496361c0 .5522847.44771525 1 1 1h1.34963611c.79564947 0 1.55871121.3160705 2.12132034.8786797l.83301651.8330165c.38440512.3844051 1.00764896.3844051 1.39205408 0l.83301646-.8330165c.5626092-.5626092 1.3256709-.8786797 2.1213204-.8786797h1.3496361c.5522847 0 1-.4477153 1-1v-1.3496361c0-.7956495.3160705-1.5587112.8786797-2.1213204l.8330165-.83301646c.3844051-.38440512.3844051-1.00764896 0-1.39205408l-.8330165-.83301651c-.5626092-.56260913-.8786797-1.32567087-.8786797-2.12132034v-1.34963611c0-.55228475-.4477153-1-1-1h-1.3496361c-.7956495 0-1.5587112-.31607052-2.1213204-.87867966l-.83301646-.8330165c-.38440512-.38440512-1.00764896-.38440512-1.39205408 0l-.83301651.8330165c-.56260913.56260914-1.32567087.87867966-2.12132034.87867966zm3.58698944 11.4960218c-.02081224.002155-.04199226.0030286-.06345763.002542-.98766446-.0223875-1.93408568-.3063547-2.75885125-.8155622-.23496767-.1450683-.30784554-.4531483-.16277726-.688116.14506827-.2349677.45314827-.3078455.68811595-.1627773.67447084.4164161 1.44758575.6483839 2.25617384.6667123.01759529.0003988.03495764.0017019.05204365.0038639.01713363-.0017748.03452416-.0026845.05212715-.0026845 2.4852814 0 4.5-2.0147186 4.5-4.5 0-1.04888973-.3593547-2.04134635-1.0074477-2.83787157-.1742817-.21419731-.1419238-.5291218.0722736-.70340353.2141973-.17428173.5291218-.14192375.7034035.07227357.7919032.97327203 1.2317706 2.18808682 1.2317706 3.46900153 0 3.0375661-2.4624339 5.5-5.5 5.5-.02146768 0-.04261937-.0013529-.06337445-.0039782zm1.57975095-10.78419583c.2654788.07599731.419084.35281842.3430867.61829728-.0759973.26547885-.3528185.419084-.6182973.3430867-.37560116-.10752146-.76586237-.16587951-1.15568824-.17249193-2.5587807-.00064534-4.58547766 2.00216524-4.58547766 4.49928198 0 .62691557.12797645 1.23496.37274865 1.7964426.11035133.2531347-.0053975.5477984-.25853224.6581497-.25313473.1103514-.54779841-.0053975-.65814974-.2585322-.29947131-.6869568-.45606667-1.43097603-.45606667-2.1960601 0-3.05211432 2.47714695-5.50006595 5.59399617-5.49921198.48576182.00815502.96289603.0795037 1.42238033.21103795zm-1.9766658 6.41091303 2.69835-2.94655317c.1788432-.21040373.4943901-.23598862.7047939-.05714545.2104037.17884318.2359886.49439014.0571454.70479387l-3.01637681 3.34277395c-.18039088.1999106-.48669547.2210637-.69285412.0478478l-1.93095347-1.62240047c-.21213845-.17678204-.24080048-.49206439-.06401844-.70420284.17678204-.21213844.49206439-.24080048.70420284-.06401844z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand"><path d="M7.498 11.918a.997.997 0 0 0-.003-1.411.995.995 0 0 0-1.412-.003l-4.102 4.102v-3.51A1 1 0 0 0 .98 10.09.992.992 0 0 0 0 11.092V17c0 .554.448 1.002 1.002 1.002h5.907c.554 0 1.002-.45 1.002-1.003 0-.539-.45-.978-1.006-.978h-3.51zm3.005-5.835a.997.997 0 0 0 .003 1.412.995.995 0 0 0 1.411.003l4.103-4.103v3.51a1 1 0 0 0 1.001 1.006A.992.992 0 0 0 18 6.91V1.002A1 1 0 0 0 17 0h-5.907a1.003 1.003 0 0 0-1.002 1.003c0 .539.45.978 1.006.978h3.51z" fill-rule="evenodd"></path></symbol><symbol id="icon-explore" viewBox="0 0 18 18"><path d="m9 17c4.418278 0 8-3.581722 8-8s-3.581722-8-8-8-8 3.581722-8 8 3.581722 8 8 8zm0 1c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9zm0-2.5c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5c2.969509 0 5.400504-2.3575119 5.497023-5.31714844.0090007-.27599565.2400359-.49243782.5160315-.48343711.2759957.0090007.4924378.2400359.4834371.51603155-.114093 3.4985237-2.9869632 6.284554-6.4964916 6.284554zm-.29090657-12.99359748c.27587424-.01216621.50937715.20161139.52154336.47748563.01216621.27587423-.20161139.50937715-.47748563.52154336-2.93195733.12930094-5.25315116 2.54886451-5.25315116 5.49456849 0 .27614237-.22385763.5-.5.5s-.5-.22385763-.5-.5c0-3.48142406 2.74307146-6.34074398 6.20909343-6.49359748zm1.13784138 8.04763908-1.2004882-1.20048821c-.19526215-.19526215-.19526215-.51184463 0-.70710678s.51184463-.19526215.70710678 0l1.20048821 1.2004882 1.6006509-4.00162734-4.50670359 1.80268144-1.80268144 4.50670359zm4.10281269-6.50378907-2.6692597 6.67314927c-.1016411.2541026-.3029834.4554449-.557086.557086l-6.67314927 2.6692597 2.66925969-6.67314926c.10164107-.25410266.30298336-.45544495.55708602-.55708602z" fill-rule="evenodd"></path></symbol><symbol id="icon-filter" viewBox="0 0 16 16"><path d="m14.9738641 0c.5667192 0 1.0261359.4477136 1.0261359 1 0 .24221858-.0902161.47620768-.2538899.65849851l-5.6938314 6.34147206v5.49997973c0 .3147562-.1520673.6111434-.4104543.7999971l-2.05227171 1.4999945c-.45337535.3313696-1.09655869.2418269-1.4365902-.1999993-.13321514-.1730955-.20522717-.3836284-.20522717-.5999978v-6.99997423l-5.69383133-6.34147206c-.3731872-.41563511-.32996891-1.0473954.09653074-1.41107611.18705584-.15950448.42716133-.2474224.67571519-.2474224zm-5.9218641 8.5h-2.105v6.491l.01238459.0070843.02053271.0015705.01955278-.0070558 2.0532976-1.4990996zm-8.02585008-7.5-.01564945.00240169 5.83249953 6.49759831h2.313l5.836-6.499z"></path></symbol><symbol id="icon-home" viewBox="0 0 18 18"><path d="m9 5-6 6v5h4v-4h4v4h4v-5zm7 6.5857864v4.4142136c0 .5522847-.4477153 1-1 1h-5v-4h-2v4h-5c-.55228475 0-1-.4477153-1-1v-4.4142136c-.25592232 0-.51184464-.097631-.70710678-.2928932l-.58578644-.5857864c-.39052429-.3905243-.39052429-1.02368929 0-1.41421358l8.29289322-8.29289322 8.2928932 8.29289322c.3905243.39052429.3905243 1.02368928 0 1.41421358l-.5857864.5857864c-.1952622.1952622-.4511845.2928932-.7071068.2928932zm-7-9.17157284-7.58578644 7.58578644.58578644.5857864 7-6.99999996 7 6.99999996.5857864-.5857864z" fill-rule="evenodd"></path></symbol><symbol id="icon-image" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm-3.49645283 10.1752453-3.89407257 6.7495552c.11705545.048464.24538859.0751995.37998328.0751995h10.60290092l-2.4329715-4.2154691-1.57494129 2.7288098zm8.49779013 6.8247547c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v13.98991071l4.50814957-7.81026689 3.08089884 5.33809539 1.57494129-2.7288097 3.5875735 6.2159812zm-3.0059397-11c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2-2-.8954305-2-2 .8954305-2 2-2zm0 1c-.5522847 0-1 .44771525-1 1s.4477153 1 1 1 1-.44771525 1-1-.4477153-1-1-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-info" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-institution" viewBox="0 0 18 18"><path d="m7 16.9998189v-2.0003623h4v2.0003623h2v-3.0005434h-8v3.0005434zm-3-10.00181122h-1.52632364c-.27614237 0-.5-.22389817-.5-.50009056 0-.13995446.05863589-.27350497.16166338-.36820841l1.23156713-1.13206327h-2.36690687v12.00217346h3v-2.0003623h-3v-1.0001811h3v-1.0001811h1v-4.00072448h-1zm10 0v2.00036224h-1v4.00072448h1v1.0001811h3v1.0001811h-3v2.0003623h3v-12.00217346h-2.3695309l1.2315671 1.13206327c.2033191.186892.2166633.50325042.0298051.70660631-.0946863.10304615-.2282126.16169266-.3681417.16169266zm3-3.00054336c.5522847 0 1 .44779634 1 1.00018112v13.00235456h-18v-13.00235456c0-.55238478.44771525-1.00018112 1-1.00018112h3.45499992l4.20535144-3.86558216c.19129876-.17584288.48537447-.17584288.67667324 0l4.2053514 3.86558216zm-4 3.00054336h-8v1.00018112h8zm-2 6.00108672h1v-4.00072448h-1zm-1 0v-4.00072448h-2v4.00072448zm-3 0v-4.00072448h-1v4.00072448zm8-4.00072448c.5522847 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.4477153-1.00018112 1-1.00018112zm-12 0c.55228475 0 1 .44779634 1 1.00018112v2.00036226h-2v-2.00036226c0-.55238478.44771525-1.00018112 1-1.00018112zm5.99868798-7.81907007-5.24205601 4.81852671h10.48411203zm.00131202 3.81834559c-.55228475 0-1-.44779634-1-1.00018112s.44771525-1.00018112 1-1.00018112 1 .44779634 1 1.00018112-.44771525 1.00018112-1 1.00018112zm-1 11.00199236v1.0001811h2v-1.0001811z" fill-rule="evenodd"></path></symbol><symbol id="icon-location" viewBox="0 0 18 18"><path d="m9.39521328 16.2688008c.79596342-.7770119 1.59208152-1.6299956 2.33285652-2.5295081 1.4020032-1.7024324 2.4323601-3.3624519 2.9354918-4.871847.2228715-.66861448.3364384-1.29323246.3364384-1.8674457 0-3.3137085-2.6862915-6-6-6-3.36356866 0-6 2.60156856-6 6 0 .57421324.11356691 1.19883122.3364384 1.8674457.50313169 1.5093951 1.53348863 3.1694146 2.93549184 4.871847.74077492.8995125 1.53689309 1.7524962 2.33285648 2.5295081.13694479.1336842.26895677.2602648.39521328.3793207.12625651-.1190559.25826849-.2456365.39521328-.3793207zm-.39521328 1.7311992s-7-6-7-11c0-4 3.13400675-7 7-7 3.8659932 0 7 3.13400675 7 7 0 5-7 11-7 11zm0-8c-1.65685425 0-3-1.34314575-3-3s1.34314575-3 3-3c1.6568542 0 3 1.34314575 3 3s-1.3431458 3-3 3zm0-1c1.1045695 0 2-.8954305 2-2s-.8954305-2-2-2-2 .8954305-2 2 .8954305 2 2 2z" fill-rule="evenodd"></path></symbol><symbol id="icon-minus" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-newsletter" viewBox="0 0 18 18"><path d="m9 11.8482489 2-1.1428571v-1.7053918h-4v1.7053918zm-3-1.7142857v-2.1339632h6v2.1339632l3-1.71428574v-6.41967746h-12v6.41967746zm10-5.3839632 1.5299989.95624934c.2923814.18273835.4700011.50320827.4700011.8479983v8.44575236c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-8.44575236c0-.34479003.1776197-.66525995.47000106-.8479983l1.52999894-.95624934v-2.75c0-.55228475.44771525-1 1-1h12c.5522847 0 1 .44771525 1 1zm0 1.17924764v3.07075236l-7 4-7-4v-3.07075236l-1 .625v8.44575236c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-8.44575236zm-10-1.92924764h6v1h-6zm-1 2h8v1h-8z" fill-rule="evenodd"></path></symbol><symbol id="icon-orcid" viewBox="0 0 18 18"><path d="m9 1c4.418278 0 8 3.581722 8 8s-3.581722 8-8 8-8-3.581722-8-8 3.581722-8 8-8zm-2.90107518 5.2732337h-1.41865256v7.1712107h1.41865256zm4.55867178.02508949h-2.99247027v7.14612121h2.91062487c.7673039 0 1.4476365-.1483432 2.0410182-.445034s1.0511995-.7152915 1.3734671-1.2558144c.3222677-.540523.4833991-1.1603247.4833991-1.85942385 0-.68545815-.1602789-1.30270225-.4808414-1.85175082-.3205625-.54904856-.7707074-.97532211-1.3504481-1.27883343-.5797408-.30351132-1.2413173-.45526471-1.9847495-.45526471zm-.1892674 1.07933542c.7877654 0 1.4143875.22336734 1.8798852.67010873.4654977.44674138.698243 1.05546001.698243 1.82617415 0 .74343221-.2310402 1.34447791-.6931277 1.80315511-.4620874.4586773-1.0750688.6880124-1.8389625.6880124h-1.46810075v-4.98745039zm-5.08652545-3.71099194c-.21825533 0-.410525.08444276-.57681478.25333081-.16628977.16888806-.24943341.36245684-.24943341.58071218 0 .22345188.08314364.41961891.24943341.58850696.16628978.16888806.35855945.25333082.57681478.25333082.233845 0 .43390938-.08314364.60019916-.24943342.16628978-.16628977.24943342-.36375592.24943342-.59240436 0-.233845-.08314364-.43131115-.24943342-.59240437s-.36635416-.24163862-.60019916-.24163862z" fill-rule="evenodd"></path></symbol><symbol id="icon-plus" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-print" viewBox="0 0 18 18"><path d="m16.0049107 5h-14.00982141c-.54941618 0-.99508929.4467783-.99508929.99961498v6.00077002c0 .5570958.44271433.999615.99508929.999615h1.00491071v-3h12v3h1.0049107c.5494162 0 .9950893-.4467783.9950893-.999615v-6.00077002c0-.55709576-.4427143-.99961498-.9950893-.99961498zm-2.0049107-1v-2.00208688c0-.54777062-.4519464-.99791312-1.0085302-.99791312h-7.9829396c-.55661731 0-1.0085302.44910695-1.0085302.99791312v2.00208688zm1 10v2.0018986c0 1.103521-.9019504 1.9981014-2.0085302 1.9981014h-7.9829396c-1.1092806 0-2.0085302-.8867064-2.0085302-1.9981014v-2.0018986h-1.00491071c-1.10185739 0-1.99508929-.8874333-1.99508929-1.999615v-6.00077002c0-1.10435686.8926228-1.99961498 1.99508929-1.99961498h1.00491071v-2.00208688c0-1.10341695.90195036-1.99791312 2.0085302-1.99791312h7.9829396c1.1092806 0 2.0085302.89826062 2.0085302 1.99791312v2.00208688h1.0049107c1.1018574 0 1.9950893.88743329 1.9950893 1.99961498v6.00077002c0 1.1043569-.8926228 1.999615-1.9950893 1.999615zm-1-3h-10v5.0018986c0 .5546075.44702548.9981014 1.0085302.9981014h7.9829396c.5565964 0 1.0085302-.4491701 1.0085302-.9981014zm-9 1h8v1h-8zm0 2h5v1h-5zm9-5c-.5522847 0-1-.44771525-1-1s.4477153-1 1-1 1 .44771525 1 1-.4477153 1-1 1z" fill-rule="evenodd"></path></symbol><symbol id="icon-search" viewBox="0 0 22 22"><path d="M21.697 20.261a1.028 1.028 0 01.01 1.448 1.034 1.034 0 01-1.448-.01l-4.267-4.267A9.812 9.811 0 010 9.812a9.812 9.811 0 1117.43 6.182zM9.812 18.222A8.41 8.41 0 109.81 1.403a8.41 8.41 0 000 16.82z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-facebook" viewBox="0 0 24 24"><path d="m6.00368507 20c-1.10660471 0-2.00368507-.8945138-2.00368507-1.9940603v-12.01187942c0-1.10128908.89451376-1.99406028 1.99406028-1.99406028h12.01187942c1.1012891 0 1.9940603.89451376 1.9940603 1.99406028v12.01187942c0 1.1012891-.88679 1.9940603-2.0032184 1.9940603h-2.9570132v-6.1960818h2.0797387l.3114113-2.414723h-2.39115v-1.54164807c0-.69911803.1941355-1.1755439 1.1966615-1.1755439l1.2786739-.00055875v-2.15974763l-.2339477-.02492088c-.3441234-.03134957-.9500153-.07025255-1.6293054-.07025255-1.8435726 0-3.1057323 1.12531866-3.1057323 3.19187953v1.78079225h-2.0850778v2.414723h2.0850778v6.1960818z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-twitter" viewBox="0 0 24 24"><path d="m18.8767135 6.87445248c.7638174-.46908424 1.351611-1.21167363 1.6250764-2.09636345-.7135248.43394112-1.50406.74870123-2.3464594.91677702-.6695189-.73342162-1.6297913-1.19486605-2.6922204-1.19486605-2.0399895 0-3.6933555 1.69603749-3.6933555 3.78628909 0 .29642457.0314329.58673729.0942985.8617704-3.06469922-.15890802-5.78835241-1.66547825-7.60988389-3.9574208-.3174714.56076194-.49978171 1.21167363-.49978171 1.90536824 0 1.31404706.65223085 2.47224203 1.64236444 3.15218497-.60350999-.0198635-1.17401554-.1925232-1.67222562-.47366811v.04583885c0 1.83355406 1.27302891 3.36609966 2.96411421 3.71294696-.31118484.0886217-.63651445.1329326-.97441718.1329326-.2357461 0-.47149219-.0229194-.69466516-.0672303.47149219 1.5065703 1.83253297 2.6036468 3.44975116 2.632678-1.2651707 1.0160946-2.85724264 1.6196394-4.5891906 1.6196394-.29861172 0-.59093688-.0152796-.88011875-.0504227 1.63450624 1.0726291 3.57548241 1.6990934 5.66104951 1.6990934 6.79263079 0 10.50641749-5.7711113 10.50641749-10.7751859l-.0094298-.48894775c.7229547-.53478659 1.3516109-1.20250585 1.8419628-1.96190282-.6632323.30100846-1.3751855.50422736-2.1217148.59590507z" fill-rule="evenodd"></path></symbol><symbol id="icon-social-youtube" viewBox="0 0 24 24"><path d="m10.1415 14.3973208-.0005625-5.19318431 4.863375 2.60554491zm9.963-7.92753362c-.6845625-.73643756-1.4518125-.73990314-1.803375-.7826454-2.518875-.18714178-6.2971875-.18714178-6.2971875-.18714178-.007875 0-3.7861875 0-6.3050625.18714178-.352125.04274226-1.1188125.04620784-1.8039375.7826454-.5394375.56084773-.7149375 1.8344515-.7149375 1.8344515s-.18 1.49597903-.18 2.99138042v1.4024082c0 1.495979.18 2.9913804.18 2.9913804s.1755 1.2736038.7149375 1.8344515c.685125.7364376 1.5845625.7133337 1.9850625.7901542 1.44.1420891 6.12.1859866 6.12.1859866s3.78225-.005776 6.301125-.1929178c.3515625-.0433198 1.1188125-.0467854 1.803375-.783223.5394375-.5608477.7155-1.8344515.7155-1.8344515s.18-1.4954014.18-2.9913804v-1.4024082c0-1.49540139-.18-2.99138042-.18-2.99138042s-.1760625-1.27360377-.7155-1.8344515z" fill-rule="evenodd"></path></symbol><symbol id="icon-subject-medicine" viewBox="0 0 18 18"><path d="m12.5 8h-6.5c-1.65685425 0-3 1.34314575-3 3v1c0 1.6568542 1.34314575 3 3 3h1v-2h-.5c-.82842712 0-1.5-.6715729-1.5-1.5s.67157288-1.5 1.5-1.5h1.5 2 1 2c1.6568542 0 3-1.34314575 3-3v-1c0-1.65685425-1.3431458-3-3-3h-2v2h1.5c.8284271 0 1.5.67157288 1.5 1.5s-.6715729 1.5-1.5 1.5zm-5.5-1v-1h-3.5c-1.38071187 0-2.5-1.11928813-2.5-2.5s1.11928813-2.5 2.5-2.5h1.02786405c.46573528 0 .92507448.10843528 1.34164078.31671843l1.13382424.56691212c.06026365-1.05041141.93116291-1.88363055 1.99667093-1.88363055 1.1045695 0 2 .8954305 2 2h2c2.209139 0 4 1.790861 4 4v1c0 2.209139-1.790861 4-4 4h-2v1h2c1.1045695 0 2 .8954305 2 2s-.8954305 2-2 2h-2c0 1.1045695-.8954305 2-2 2s-2-.8954305-2-2h-1c-2.209139 0-4-1.790861-4-4v-1c0-2.209139 1.790861-4 4-4zm0-2v-2.05652691c-.14564246-.03538148-.28733393-.08714006-.42229124-.15461871l-1.15541752-.57770876c-.27771087-.13885544-.583937-.21114562-.89442719-.21114562h-1.02786405c-.82842712 0-1.5.67157288-1.5 1.5s.67157288 1.5 1.5 1.5zm4 1v1h1.5c.2761424 0 .5-.22385763.5-.5s-.2238576-.5-.5-.5zm-1 1v-5c0-.55228475-.44771525-1-1-1s-1 .44771525-1 1v5zm-2 4v5c0 .5522847.44771525 1 1 1s1-.4477153 1-1v-5zm3 2v2h2c.5522847 0 1-.4477153 1-1s-.4477153-1-1-1zm-4-1v-1h-.5c-.27614237 0-.5.2238576-.5.5s.22385763.5.5.5zm-3.5-9h1c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5z" fill-rule="evenodd"></path></symbol><symbol id="icon-success" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm3.4860198 4.98163161-4.71802968 5.50657859-2.62834168-2.02300024c-.42862421-.36730544-1.06564993-.30775346-1.42283677.13301307-.35718685.44076653-.29927542 1.0958383.12934879 1.46314377l3.40735508 2.7323063c.42215801.3385221 1.03700951.2798252 1.38749189-.1324571l5.38450527-6.33394549c.3613513-.43716226.3096573-1.09278382-.115462-1.46437175-.4251192-.37158792-1.0626796-.31842941-1.4240309.11873285z" fill-rule="evenodd"></path></symbol><symbol id="icon-table" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587l-4.0059107-.001.001.001h-1l-.001-.001h-5l.001.001h-1l-.001-.001-3.00391071.001c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm-11.0059107 5h-3.999v6.9941413c0 .5572961.44630695 1.0058587.99508929 1.0058587h3.00391071zm6 0h-5v8h5zm5.0059107-4h-4.0059107v3h5.001v1h-5.001v7.999l4.0059107.001c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-12.5049107 9c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.2238576.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.2238576-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.2238576.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.2238576-.5-.5s.22385763-.5.5-.5zm-6-2c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-1c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm12 0c.2761424 0 .5.22385763.5.5s-.2238576.5-.5.5h-2c-.2761424 0-.5-.22385763-.5-.5s.2238576-.5.5-.5zm-6 0c.27614237 0 .5.22385763.5.5s-.22385763.5-.5.5h-2c-.27614237 0-.5-.22385763-.5-.5s.22385763-.5.5-.5zm1.499-5h-5v3h5zm-6 0h-3.00391071c-.54871518 0-.99508929.44887827-.99508929 1.00585866v1.99414134h3.999z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick-circle" viewBox="0 0 24 24"><path d="m12 2c5.5228475 0 10 4.4771525 10 10s-4.4771525 10-10 10-10-4.4771525-10-10 4.4771525-10 10-10zm0 1c-4.97056275 0-9 4.02943725-9 9 0 4.9705627 4.02943725 9 9 9 4.9705627 0 9-4.0294373 9-9 0-4.97056275-4.0294373-9-9-9zm4.2199868 5.36606669c.3613514-.43716226.9989118-.49032077 1.424031-.11873285s.4768133 1.02720949.115462 1.46437175l-6.093335 6.94397871c-.3622945.4128716-.9897871.4562317-1.4054264.0971157l-3.89719065-3.3672071c-.42862421-.3673054-.48653564-1.0223772-.1293488-1.4631437s.99421256-.5003185 1.42283677-.1330131l3.11097438 2.6987741z" fill-rule="evenodd"></path></symbol><symbol id="icon-tick" viewBox="0 0 16 16"><path d="m6.76799012 9.21106946-3.1109744-2.58349728c-.42862421-.35161617-1.06564993-.29460792-1.42283677.12733148s-.29927541 1.04903009.1293488 1.40064626l3.91576307 3.23873978c.41034319.3393961 1.01467563.2976897 1.37450571-.0948578l6.10568327-6.660841c.3613513-.41848908.3096572-1.04610608-.115462-1.4018218-.4251192-.35571573-1.0626796-.30482786-1.424031.11366122z" fill-rule="evenodd"></path></symbol><symbol id="icon-update" viewBox="0 0 18 18"><path d="m1 13v1c0 .5522847.44771525 1 1 1h14c.5522847 0 1-.4477153 1-1v-1h-1v-10h-14v10zm16-1h1v2c0 1.1045695-.8954305 2-2 2h-14c-1.1045695 0-2-.8954305-2-2v-2h1v-9c0-.55228475.44771525-1 1-1h14c.5522847 0 1 .44771525 1 1zm-1 0v1h-4.5857864l-1 1h-2.82842716l-1-1h-4.58578644v-1h5l1 1h2l1-1zm-13-8h12v7h-12zm1 1v5h10v-5zm1 1h4v1h-4zm0 2h4v1h-4z" fill-rule="evenodd"></path></symbol><symbol id="icon-upload" viewBox="0 0 18 18"><path d="m10.0046024 0c.5497429 0 1.3179837.32258606 1.707238.71184039l4.5763192 4.57631922c.3931386.39313859.7118404 1.16760135.7118404 1.71431368v8.98899651c0 1.1092806-.8945138 2.0085302-1.9940603 2.0085302h-12.01187942c-1.10128908 0-1.99406028-.8926228-1.99406028-1.9950893v-14.00982141c0-1.10185739.88743329-1.99508929 1.99961498-1.99508929zm0 1h-7.00498742c-.55709576 0-.99961498.44271433-.99961498.99508929v14.00982141c0 .5500396.44491393.9950893.99406028.9950893h12.01187942c.5463747 0 .9940603-.4506622.9940603-1.0085302v-8.98899651c0-.28393444-.2150684-.80332809-.4189472-1.0072069l-4.5763192-4.57631922c-.2038461-.20384606-.718603-.41894717-1.0001312-.41894717zm-1.85576936 4.14572769c.19483374-.19483375.51177826-.19377714.70556874.00001334l2.59099082 2.59099079c.1948411.19484112.1904373.51514474.0027906.70279143-.1932998.19329987-.5046517.19237083-.7001856-.00692852l-1.74638687-1.7800176v6.14827687c0 .2717771-.23193359.492096-.5.492096-.27614237 0-.5-.216372-.5-.492096v-6.14827641l-1.74627892 1.77990922c-.1933927.1971171-.51252214.19455839-.70016883.0069117-.19329987-.19329988-.19100584-.50899493.00277731-.70277808z" fill-rule="evenodd"></path></symbol><symbol id="icon-video" viewBox="0 0 18 18"><path d="m16.0049107 2c1.1018574 0 1.9950893.89706013 1.9950893 2.00585866v9.98828264c0 1.1078052-.8926228 2.0058587-1.9950893 2.0058587h-14.00982141c-1.10185739 0-1.99508929-.8970601-1.99508929-2.0058587v-9.98828264c0-1.10780515.8926228-2.00585866 1.99508929-2.00585866zm0 1h-14.00982141c-.54871518 0-.99508929.44887827-.99508929 1.00585866v9.98828264c0 .5572961.44630695 1.0058587.99508929 1.0058587h14.00982141c.5487152 0 .9950893-.4488783.9950893-1.0058587v-9.98828264c0-.55729607-.446307-1.00585866-.9950893-1.00585866zm-8.30912922 2.24944486 4.60460462 2.73982242c.9365543.55726659.9290753 1.46522435 0 2.01804082l-4.60460462 2.7398224c-.93655425.5572666-1.69578148.1645632-1.69578148-.8937585v-5.71016863c0-1.05087579.76670616-1.446575 1.69578148-.89375851zm-.67492769.96085624v5.5750128c0 .2995102-.10753745.2442517.16578928.0847713l4.58452283-2.67497259c.3050619-.17799716.3051624-.21655446 0-.39461026l-4.58452283-2.67497264c-.26630747-.15538481-.16578928-.20699944-.16578928.08477139z" fill-rule="evenodd"></path></symbol><symbol id="icon-warning" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-checklist-banner" viewBox="0 0 56.69 56.69"><path style="fill:none" d="M0 0h56.69v56.69H0z"></path><clipPath id="b"><use xlink:href="#a" style="overflow:visible"></use></clipPath><path d="M21.14 34.46c0-6.77 5.48-12.26 12.24-12.26s12.24 5.49 12.24 12.26-5.48 12.26-12.24 12.26c-6.76-.01-12.24-5.49-12.24-12.26zm19.33 10.66 10.23 9.22s1.21 1.09 2.3-.12l2.09-2.32s1.09-1.21-.12-2.3l-10.23-9.22m-19.29-5.92c0-4.38 3.55-7.94 7.93-7.94s7.93 3.55 7.93 7.94c0 4.38-3.55 7.94-7.93 7.94-4.38-.01-7.93-3.56-7.93-7.94zm17.58 12.99 4.14-4.81" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round"></path><path d="M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5m14.42-5.2V4.86s0-2.93-2.93-2.93H4.13s-2.93 0-2.93 2.93v37.57s0 2.93 2.93 2.93h15.01M8.26 9.75H28.6M8.26 15.98H28.6m-20.34 6.2h12.5" style="clip-path:url(#b);fill:none;stroke:#01324b;stroke-width:2;stroke-linecap:round;stroke-linejoin:round"></path></symbol><symbol id="icon-chevron-down" viewBox="0 0 16 16"><path d="m5.58578644 3-3.29289322-3.29289322c-.39052429-.39052429-.39052429-1.02368927 0-1.41421356s1.02368927-.39052429 1.41421356 0l4 4c.39052429.39052429.39052429 1.02368927 0 1.41421356l-4 4c-.39052429.39052429-1.02368927.39052429-1.41421356 0s-.39052429-1.02368927 0-1.41421356z" fill-rule="evenodd" transform="matrix(0 1 -1 0 11 1)"></path></symbol><symbol id="icon-eds-i-arrow-right-medium" viewBox="0 0 24 24"><path d="m12.728 3.293 7.98 7.99a.996.996 0 0 1 .281.561l.011.157c0 .32-.15.605-.384.788l-7.908 7.918a1 1 0 0 1-1.416-1.414L17.576 13H4a1 1 0 0 1 0-2h13.598l-6.285-6.293a1 1 0 0 1-.082-1.32l.083-.095a1 1 0 0 1 1.414.001Z"></path></symbol><symbol id="icon-eds-i-check-circle-medium" viewBox="0 0 24 24"><path d="M12 1c6.075 0 11 4.925 11 11s-4.925 11-11 11S1 18.075 1 12 5.925 1 12 1Zm0 2a9 9 0 1 0 0 18 9 9 0 0 0 0-18Zm5.125 4.72a1 1 0 0 1 .156 1.405l-6 7.5a1 1 0 0 1-1.421.143l-3-2.5a1 1 0 0 1 1.28-1.536l2.217 1.846 5.362-6.703a1 1 0 0 1 1.406-.156Z"></path></symbol><symbol id="icon-eds-i-chevron-down-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h4.99912834v-4.99912834c0-.55276616.44386482-1.00087166 1-1.00087166.55228475 0 1 .44463086 1 1.00087166v4.99912834h4.9991283c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-4.9991283v4.9991283c0 .5527662-.44386482 1.0008717-1 1.0008717-.55228475 0-1-.4446309-1-1.0008717v-4.9991283h-4.99912834c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-chevron-down-small" viewBox="0 0 16 16"><path d="M13.692 5.278a1 1 0 0 1 .03 1.414L9.103 11.51a1.491 1.491 0 0 1-2.188.019L2.278 6.692a1 1 0 0 1 1.444-1.384L8 9.771l4.278-4.463a1 1 0 0 1 1.318-.111l.096.081Z"></path></symbol><symbol id="icon-eds-i-chevron-right-medium" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-right-small" viewBox="0 0 10 10"><path d="m5.96738168 4.70639573 2.39518594-2.41447274c.37913917-.38219212.98637524-.38972225 1.35419292-.01894278.37750606.38054586.37784436.99719163-.00013556 1.37821513l-4.03074001 4.06319683c-.37758093.38062133-.98937525.38100976-1.367372-.00003075l-4.03091981-4.06337806c-.37759778-.38063832-.38381821-.99150444-.01600053-1.3622839.37750607-.38054587.98772445-.38240057 1.37006824.00302197l2.39538588 2.4146743.96295325.98624457z" fill-rule="evenodd" transform="matrix(0 -1 1 0 0 10)"></path></symbol><symbol id="icon-eds-i-chevron-up-medium" viewBox="0 0 16 16"><path d="m2.00087166 7h11.99825664c.5527662 0 1.0008717.44386482 1.0008717 1 0 .55228475-.4446309 1-1.0008717 1h-11.99825664c-.55276616 0-1.00087166-.44386482-1.00087166-1 0-.55228475.44463086-1 1.00087166-1z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-close-medium" viewBox="0 0 16 16"><path d="m2.29679575 12.2772478c-.39658757.3965876-.39438847 1.0328109-.00062148 1.4265779.39651227.3965123 1.03246768.3934888 1.42657791-.0006214l4.27724782-4.27724787 4.2772478 4.27724787c.3965876.3965875 1.0328109.3943884 1.4265779.0006214.3965123-.3965122.3934888-1.0324677-.0006214-1.4265779l-4.27724787-4.2772478 4.27724787-4.27724782c.3965875-.39658757.3943884-1.03281091.0006214-1.42657791-.3965122-.39651226-1.0324677-.39348875-1.4265779.00062148l-4.2772478 4.27724782-4.27724782-4.27724782c-.39658757-.39658757-1.03281091-.39438847-1.42657791-.00062148-.39651226.39651227-.39348875 1.03246768.00062148 1.42657791l4.27724782 4.27724782z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-copy-link" viewBox="0 0 24 24"><path fill-rule="evenodd" clip-rule="evenodd" d="M19.4594 8.57015C19.0689 8.17963 19.0689 7.54646 19.4594 7.15594L20.2927 6.32261C20.2927 6.32261 20.2927 6.32261 20.2927 6.32261C21.0528 5.56252 21.0528 4.33019 20.2928 3.57014C19.5327 2.81007 18.3004 2.81007 17.5404 3.57014L16.7071 4.40347C16.3165 4.794 15.6834 4.794 15.2928 4.40348C14.9023 4.01296 14.9023 3.3798 15.2928 2.98927L16.1262 2.15594C17.6673 0.614803 20.1659 0.614803 21.707 2.15593C23.2481 3.69705 23.248 6.19569 21.707 7.7368L20.8737 8.57014C20.4831 8.96067 19.85 8.96067 19.4594 8.57015Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M18.0944 5.90592C18.4849 6.29643 18.4849 6.9296 18.0944 7.32013L16.4278 8.9868C16.0373 9.37733 15.4041 9.37734 15.0136 8.98682C14.6231 8.59631 14.6231 7.96314 15.0136 7.57261L16.6802 5.90594C17.0707 5.51541 17.7039 5.5154 18.0944 5.90592Z"></path><path fill-rule="evenodd" clip-rule="evenodd" d="M13.5113 6.32243C13.9018 6.71295 13.9018 7.34611 13.5113 7.73664L12.678 8.56997C12.678 8.56997 12.678 8.56997 12.678 8.56997C11.9179 9.33006 11.9179 10.5624 12.6779 11.3224C13.438 12.0825 14.6703 12.0825 15.4303 11.3224L16.2636 10.4891C16.6542 10.0986 17.2873 10.0986 17.6779 10.4891C18.0684 10.8796 18.0684 11.5128 17.6779 11.9033L16.8445 12.7366C15.3034 14.2778 12.8048 14.2778 11.2637 12.7366C9.72262 11.1955 9.72266 8.69689 11.2637 7.15578L12.097 6.32244C12.4876 5.93191 13.1207 5.93191 13.5113 6.32243Z"></path><path d="M8 20V22H19.4619C20.136 22 20.7822 21.7311 21.2582 21.2529C21.7333 20.7757 22 20.1289 22 19.4549V15C22 14.4477 21.5523 14 21 14C20.4477 14 20 14.4477 20 15V19.4549C20 19.6004 19.9426 19.7397 19.8408 19.842C19.7399 19.9433 19.6037 20 19.4619 20H8Z"></path><path d="M4 13H2V19.4619C2 20.136 2.26889 20.7822 2.74705 21.2582C3.22434 21.7333 3.87105 22 4.5451 22H9C9.55228 22 10 21.5523 10 21C10 20.4477 9.55228 20 9 20H4.5451C4.39957 20 4.26028 19.9426 4.15804 19.8408C4.05668 19.7399 4 19.6037 4 19.4619V13Z"></path><path d="M4 13H2V4.53808C2 3.86398 2.26889 3.21777 2.74705 2.74178C3.22434 2.26666 3.87105 2 4.5451 2H9C9.55228 2 10 2.44772 10 3C10 3.55228 9.55228 4 9 4H4.5451C4.39957 4 4.26028 4.05743 4.15804 4.15921C4.05668 4.26011 4 4.39633 4 4.53808V13Z"></path></symbol><symbol id="icon-eds-i-download-medium" viewBox="0 0 16 16"><path d="m12.9975267 12.999368c.5467123 0 1.0024733.4478567 1.0024733 1.000316 0 .5563109-.4488226 1.000316-1.0024733 1.000316h-9.99505341c-.54671233 0-1.00247329-.4478567-1.00247329-1.000316 0-.5563109.44882258-1.000316 1.00247329-1.000316zm-4.9975267-11.999368c.55228475 0 1 .44497754 1 .99589209v6.80214418l2.4816273-2.48241149c.3928222-.39294628 1.0219732-.4006883 1.4030652-.01947579.3911302.39125371.3914806 1.02525073-.0001404 1.41699553l-4.17620792 4.17752758c-.39120769.3913313-1.02508144.3917306-1.41671995-.0000316l-4.17639421-4.17771394c-.39122513-.39134876-.39767006-1.01940351-.01657797-1.40061601.39113012-.39125372 1.02337105-.3931606 1.41951349.00310701l2.48183446 2.48261871v-6.80214418c0-.55001601.44386482-.99589209 1-.99589209z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-info-filled-medium" viewBox="0 0 18 18"><path d="m9 0c4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9zm0 7h-1.5l-.11662113.00672773c-.49733868.05776511-.88337887.48043643-.88337887.99327227 0 .47338693.32893365.86994729.77070917.97358929l.1126697.01968298.11662113.00672773h.5v3h-.5l-.11662113.0067277c-.42082504.0488782-.76196299.3590206-.85696816.7639815l-.01968298.1126697-.00672773.1166211.00672773.1166211c.04887817.4208251.35902055.761963.76398144.8569682l.1126697.019683.11662113.0067277h3l.1166211-.0067277c.4973387-.0577651.8833789-.4804365.8833789-.9932723 0-.4733869-.3289337-.8699473-.7707092-.9735893l-.1126697-.019683-.1166211-.0067277h-.5v-4l-.00672773-.11662113c-.04887817-.42082504-.35902055-.76196299-.76398144-.85696816l-.1126697-.01968298zm0-3.25c-.69035594 0-1.25.55964406-1.25 1.25s.55964406 1.25 1.25 1.25 1.25-.55964406 1.25-1.25-.55964406-1.25-1.25-1.25z" fill-rule="evenodd"></path></symbol><symbol id="icon-eds-i-institution-medium" viewBox="0 0 24 24"><g><path fill-rule="evenodd" clip-rule="evenodd" d="M11.9967 1C11.6364 1 11.279 1.0898 10.961 1.2646C10.9318 1.28061 10.9035 1.29806 10.8761 1.31689L2.79765 6.87C2.46776 7.08001 2.20618 7.38466 2.07836 7.76668C1.94823 8.15561 1.98027 8.55648 2.12665 8.90067C2.42086 9.59246 3.12798 10 3.90107 10H4.99994V16H4.49994C3.11923 16 1.99994 17.1193 1.99994 18.5V19.5C1.99994 20.8807 3.11923 22 4.49994 22H19.4999C20.8807 22 21.9999 20.8807 21.9999 19.5V18.5C21.9999 17.1193 20.8807 16 19.4999 16H18.9999V10H20.0922C20.8653 10 21.5725 9.59252 21.8667 8.90065C22.0131 8.55642 22.0451 8.15553 21.9149 7.7666C21.7871 7.38459 21.5255 7.07997 21.1956 6.86998L13.1172 1.31689C13.0898 1.29806 13.0615 1.28061 13.0324 1.2646C12.7143 1.0898 12.357 1 11.9967 1ZM4.6844 8L11.9472 3.00755C11.9616 3.00295 11.9783 3 11.9967 3C12.015 3 12.0318 3.00295 12.0461 3.00755L19.3089 8H4.6844ZM16.9999 16V10H14.9999V16H16.9999ZM12.9999 16V10H10.9999V16H12.9999ZM8.99994 16V10H6.99994V16H8.99994ZM3.99994 18.5C3.99994 18.2239 4.2238 18 4.49994 18H19.4999C19.7761 18 19.9999 18.2239 19.9999 18.5V19.5C19.9999 19.7761 19.7761 20 19.4999 20H4.49994C4.2238 20 3.99994 19.7761 3.99994 19.5V18.5Z"></path></g></symbol><symbol id="icon-eds-i-mail-medium" viewBox="0 0 24 24"><path d="m19.462 0c1.413 0 2.538 1.184 2.538 2.619v12.762c0 1.435-1.125 2.619-2.538 2.619h-16.924c-1.413 0-2.538-1.184-2.538-2.619v-12.762c0-1.435 1.125-2.619 2.538-2.619zm.538 5.158-7.378 6.258a2.549 2.549 0 0 1 -3.253-.008l-7.369-6.248v10.222c0 .353.253.619.538.619h16.924c.285 0 .538-.266.538-.619zm-.538-3.158h-16.924c-.264 0-.5.228-.534.542l8.65 7.334c.2.165.492.165.684.007l8.656-7.342-.001-.025c-.044-.3-.274-.516-.531-.516z"></path></symbol><symbol id="icon-eds-i-menu-medium" viewBox="0 0 24 24"><path d="M21 4a1 1 0 0 1 0 2H3a1 1 0 1 1 0-2h18Zm-4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h14Zm4 7a1 1 0 0 1 0 2H3a1 1 0 0 1 0-2h18Z"></path></symbol><symbol id="icon-eds-i-search-medium" viewBox="0 0 24 24"><path d="M11 1c5.523 0 10 4.477 10 10 0 2.4-.846 4.604-2.256 6.328l3.963 3.965a1 1 0 0 1-1.414 1.414l-3.965-3.963A9.959 9.959 0 0 1 11 21C5.477 21 1 16.523 1 11S5.477 1 11 1Zm0 2a8 8 0 1 0 0 16 8 8 0 0 0 0-16Z"></path></symbol><symbol id="icon-eds-i-user-single-medium" viewBox="0 0 24 24"><path d="M12 1a5 5 0 1 1 0 10 5 5 0 0 1 0-10Zm0 2a3 3 0 1 0 0 6 3 3 0 0 0 0-6Zm-.406 9.008a8.965 8.965 0 0 1 6.596 2.494A9.161 9.161 0 0 1 21 21.025V22a1 1 0 0 1-1 1H4a1 1 0 0 1-1-1v-.985c.05-4.825 3.815-8.777 8.594-9.007Zm.39 1.992-.299.006c-3.63.175-6.518 3.127-6.678 6.775L5 21h13.998l-.009-.268a7.157 7.157 0 0 0-1.97-4.573l-.214-.213A6.967 6.967 0 0 0 11.984 14Z"></path></symbol><symbol id="icon-eds-i-warning-filled-medium" viewBox="0 0 18 18"><path d="m9 11.75c.69035594 0 1.25.5596441 1.25 1.25s-.55964406 1.25-1.25 1.25-1.25-.5596441-1.25-1.25.55964406-1.25 1.25-1.25zm.41320045-7.75c.55228475 0 1.00000005.44771525 1.00000005 1l-.0034543.08304548-.3333333 4c-.043191.51829212-.47645714.91695452-.99654578.91695452h-.15973424c-.52008864 0-.95335475-.3986624-.99654576-.91695452l-.33333333-4c-.04586475-.55037702.36312325-1.03372649.91350028-1.07959124l.04148683-.00259031zm-.41320045 14c-4.97056275 0-9-4.0294373-9-9 0-4.97056275 4.02943725-9 9-9 4.9705627 0 9 4.02943725 9 9 0 4.9705627-4.0294373 9-9 9z" fill-rule="evenodd"></path></symbol><symbol id="icon-expand-image" viewBox="0 0 18 18"><path d="m7.49754099 11.9178212c.38955542-.3895554.38761957-1.0207846-.00290473-1.4113089-.39324695-.3932469-1.02238878-.3918247-1.41130883-.0029047l-4.10273549 4.1027355.00055454-3.5103985c.00008852-.5603185-.44832171-1.006032-1.00155062-1.0059446-.53903074.0000852-.97857527.4487442-.97866268 1.0021075l-.00093318 5.9072465c-.00008751.553948.44841131 1.001882 1.00174994 1.0017946l5.906983-.0009331c.5539233-.0000875 1.00197907-.4486389 1.00206646-1.0018679.00008515-.5390307-.45026621-.9784332-1.00588841-.9783454l-3.51010549.0005545zm3.00571741-5.83449376c-.3895554.38955541-.3876196 1.02078454.0029047 1.41130883.393247.39324696 1.0223888.39182478 1.4113089.00290473l4.1027355-4.10273549-.0005546 3.5103985c-.0000885.56031852.4483217 1.006032 1.0015506 1.00594461.5390308-.00008516.9785753-.44874418.9786627-1.00210749l.0009332-5.9072465c.0000875-.553948-.4484113-1.00188204-1.0017499-1.00179463l-5.906983.00093313c-.5539233.00008751-1.0019791.44863892-1.0020665 1.00186784-.0000852.53903074.4502662.97843325 1.0058884.97834547l3.5101055-.00055449z" fill-rule="evenodd"></path></symbol><symbol id="icon-github" viewBox="0 0 100 100"><path fill-rule="evenodd" clip-rule="evenodd" d="M48.854 0C21.839 0 0 22 0 49.217c0 21.756 13.993 40.172 33.405 46.69 2.427.49 3.316-1.059 3.316-2.362 0-1.141-.08-5.052-.08-9.127-13.59 2.934-16.42-5.867-16.42-5.867-2.184-5.704-5.42-7.17-5.42-7.17-4.448-3.015.324-3.015.324-3.015 4.934.326 7.523 5.052 7.523 5.052 4.367 7.496 11.404 5.378 14.235 4.074.404-3.178 1.699-5.378 3.074-6.6-10.839-1.141-22.243-5.378-22.243-24.283 0-5.378 1.94-9.778 5.014-13.2-.485-1.222-2.184-6.275.486-13.038 0 0 4.125-1.304 13.426 5.052a46.97 46.97 0 0 1 12.214-1.63c4.125 0 8.33.571 12.213 1.63 9.302-6.356 13.427-5.052 13.427-5.052 2.67 6.763.97 11.816.485 13.038 3.155 3.422 5.015 7.822 5.015 13.2 0 18.905-11.404 23.06-22.324 24.283 1.78 1.548 3.316 4.481 3.316 9.126 0 6.6-.08 11.897-.08 13.526 0 1.304.89 2.853 3.316 2.364 19.412-6.52 33.405-24.935 33.405-46.691C97.707 22 75.788 0 48.854 0z"></path></symbol><symbol id="icon-springer-arrow-left"><path d="M15 7a1 1 0 000-2H3.385l2.482-2.482a.994.994 0 00.02-1.403 1.001 1.001 0 00-1.417 0L.294 5.292a1.001 1.001 0 000 1.416l4.176 4.177a.991.991 0 001.4.016 1 1 0 00-.003-1.42L3.385 7H15z"></path></symbol><symbol id="icon-springer-arrow-right"><path d="M1 7a1 1 0 010-2h11.615l-2.482-2.482a.994.994 0 01-.02-1.403 1.001 1.001 0 011.417 0l4.176 4.177a1.001 1.001 0 010 1.416l-4.176 4.177a.991.991 0 01-1.4.016 1 1 0 01.003-1.42L12.615 7H1z"></path></symbol><symbol id="icon-submit-open" viewBox="0 0 16 17"><path d="M12 0c1.10457 0 2 .895431 2 2v5c0 .276142-.223858.5-.5.5S13 7.276142 13 7V2c0-.512836-.38604-.935507-.883379-.993272L12 1H6v3c0 1.10457-.89543 2-2 2H1v8c0 .512836.38604.935507.883379.993272L2 15h6.5c.276142 0 .5.223858.5.5s-.223858.5-.5.5H2c-1.104569 0-2-.89543-2-2V5.828427c0-.530433.210714-1.039141.585786-1.414213L4.414214.585786C4.789286.210714 5.297994 0 5.828427 0H12Zm3.41 11.14c.250899.250899.250274.659726 0 .91-.242954.242954-.649606.245216-.9-.01l-1.863671-1.900337.001043 5.869492c0 .356992-.289839.637138-.647372.637138-.347077 0-.647371-.285256-.647371-.637138l-.001043-5.869492L9.5 12.04c-.253166.258042-.649726.260274-.9.01-.242954-.242954-.252269-.657731 0-.91l2.942184-2.951303c.250908-.250909.66127-.252277.91353-.000017L15.41 11.14ZM5 1.413 1.413 5H4c.552285 0 1-.447715 1-1V1.413ZM11 3c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Zm0 2c.276142 0 .5.223858.5.5s-.223858.5-.5.5H7.5c-.276142 0-.5-.223858-.5-.5s.223858-.5.5-.5H11Z" fill-rule="nonzero"></path></symbol></svg>
</div>
</footer>




    

    

<div class="c-site-messages message u-hide u-hide-print c-site-messages--nature-briefing c-site-messages--nature-briefing-email-variant c-site-messages--nature-briefing-redesign-2020 sans-serif c-site-messages--nature-briefing-cancer" data-component-id="nature-briefing-banner" data-component-expirydays="30" data-component-trigger-scroll-percentage="15" data-track="in-view" data-track-action="in-view" data-track-category="nature briefing" data-track-label="Briefing banner visible: Cancer">

    
    <div class="c-site-messages__banner-large">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Cancer">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__form-container">

            <div class="grid grid-12 last">
                <div class="grid grid-4">
                    <img alt="Nature Briefing: Cancer" src="/static/images/logos/nature-briefing-logo-cancer-white-bc263e7d57.svg" width="373" height="40">
                    <p class="c-site-messages--nature-briefing__strapline extra-tight-line-height">Sign up for the <em>Nature Briefing: Cancer</em> newsletter — what matters in cancer research, free to your inbox weekly.</p>
                </div>
                <div class="grid grid-8 last">
                    <form action="https://www.nature.com/briefing/cancer" method="post" data-location="banner" data-track="signup_nature_briefing_banner" data-track-action="transmit-form" data-track-category="nature briefing" data-track-label="Briefing banner submit: Cancer">
                        <input id="briefing-banner-signup-form-input-track-originReferralPoint" type="hidden" name="track_originReferralPoint" value="CancerBriefingBanner">
                        <input id="briefing-banner-signup-form-input-track-formType" type="hidden" name="track_formType" value="DirectEmailBanner">

                        <input type="hidden" value="false" name="gdpr_tick" id="gdpr_tick_banner">
                        <input type="hidden" value="false" name="marketing" id="marketing_input_banner">
                        <input type="hidden" value="false" name="marketing_tick" id="marketing_tick_banner">
                        <input type="hidden" value="CancerBriefingBanner" name="brieferEntryPoint" id="brieferEntryPoint_banner">

                        <label class="nature-briefing-banner__email-label" for="emailAddress">Email address</label>

                        <div class="nature-briefing-banner__email-wrapper">
                            <input class="nature-briefing-banner__email-input box-sizing text14" type="email" id="emailAddress" name="emailAddress" value="" placeholder="e.g. jo.smith@university.ac.uk" required="" data-test-element="briefing-emailbanner-email-input">
                            
                            <input type="hidden" value="true" name="N:cancer" id="defaultNewsletter_banner">
                            <button type="submit" class="nature-briefing-banner__submit-button box-sizing text14" data-test-element="briefing-emailbanner-signup-button">Sign up</button>
                        </div>

                        <div class="nature-briefing-banner__checkbox-wrapper grid grid-12 last">
                            <input class="nature-briefing-banner__checkbox-checkbox" id="gdpr-briefing-banner-checkbox" type="checkbox" name="gdpr" value="true" data-test-element="briefing-emailbanner-gdpr-checkbox" required="">
                            <label class="nature-briefing-banner__checkbox-label box-sizing text13 sans-serif block tighten-line-height" for="gdpr-briefing-banner-checkbox">I agree my information will be processed in accordance with the <em>Nature</em> and Springer Nature Limited <a href="https://www.nature.com/info/privacy">Privacy Policy</a>.</label>
                        </div>
                    </form>
                </div>
            </div>

        </div>

    </div>

    
    <div class="c-site-messages__banner-small">

        
<div class="c-site-messages__close-container">
    <button class="c-site-messages__close" data-track="click" data-track-category="nature briefing" data-track-label="Briefing banner dismiss: Cancer">
        <svg width="25px" height="25px" focusable="false" aria-hidden="true" viewBox="0 0 25 25" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
            <title>Close banner</title>
            <defs></defs>
            <g stroke="none" stroke-width="1" fill="none" fill-rule="evenodd">
                <rect opacity="0" x="0" y="0" width="25" height="25"></rect>
                <path d="M6.29679575,16.2772478 C5.90020818,16.6738354 5.90240728,17.3100587 6.29617427,17.7038257 C6.69268654,18.100338 7.32864195,18.0973145 7.72275218,17.7032043 L12,13.4259564 L16.2772478,17.7032043 C16.6738354,18.0997918 17.3100587,18.0975927 17.7038257,17.7038257 C18.100338,17.3073135 18.0973145,16.671358 17.7032043,16.2772478 L13.4259564,12 L17.7032043,7.72275218 C18.0997918,7.32616461 18.0975927,6.68994127 17.7038257,6.29617427 C17.3073135,5.89966201 16.671358,5.90268552 16.2772478,6.29679575 L12,10.5740436 L7.72275218,6.29679575 C7.32616461,5.90020818 6.68994127,5.90240728 6.29617427,6.29617427 C5.89966201,6.69268654 5.90268552,7.32864195 6.29679575,7.72275218 L10.5740436,12 L6.29679575,16.2772478 Z" fill="#ffffff"></path>
            </g>
        </svg>
        <span class="visually-hidden">Close</span>
    </button>
</div>


        <div class="c-site-messages__content text14">
            <span class="c-site-messages--nature-briefing__strapline strong">Get what matters in cancer research, free to your inbox weekly.</span>
            <a class="nature-briefing__link text14 sans-serif" data-track="click" data-track-category="nature briefing" data-track-label="Small-screen banner CTA to site" data-test-element="briefing-banner-link" target="_blank" rel="noreferrer noopener" href="/briefing/cancer/?brieferEntryPoint=CancerBriefingBanner">Sign up for Nature Briefing: Cancer
            </a>
        </div>

    </div>

</div>

    




<noscript>
    <img hidden src="https://verify.nature.com/verify/nature.png" width="0" height="0" style="display: none" alt="">
</noscript>




<script src="//content.readcube.com/ping?doi=10.1038/s41591-022-01883-3&amp;format=js&amp;last_modified=2022-07-04" async=""></script>

<egx-oexhwjmsfjgu></egx-oexhwjmsfjgu><div data-cc-ghost="" style="height: 352px;"></div><script type="text/javascript" id="fetch-contextual-ads-campaign-data" charset="">(function(){var g=function(a){a=new CustomEvent("campaignDataLoaded",{detail:a});document.dispatchEvent(a)},b=google_tag_manager["rm"]["50443292"](18);if(b){var k=1500;b=google_tag_manager["rm"]["50443292"](19);b="link"===b?google_tag_manager["rm"]["50443292"](20):google_tag_manager["rm"]["50443292"](21);var m=function(a,c){var h=!1,l=setTimeout(function(){h=!0;c(null)},k),d=new XMLHttpRequest;d.onload=function(){var e=null,f=[];if(200===d.status){try{e=JSON.parse(d.responseText)}catch(p){e={}}f=e.campaigns||[]}h||(clearTimeout(l),c(f.length?f.join(","):null))};d.open("GET",
"/platform/contextual?doi\x3d"+a);d.send()},n=function(a,c){a&&-1===a.indexOf("not set")?m(a,c):setTimeout(c,1)};n(b,function(a){window.campaignsForContextualAds=a;g(a);window.dataLayer.push({content:{article:{campaignID:a}}})})}else window.campaignsForContextualAds="",g(null)})();</script>
<script type="text/javascript" id="gtm-setup-accessdetailsloaded-handler" charset="">document.addEventListener("accessdetailsloaded",function(a){a=a.detail||{};var b={event:"update-access-details"};a.institutional_business_partner_ids&&a.resolved_by||console.log("BPID data could not be retrieved from /exposed-details");a.logged_in_to_sn_profile||console.log("logged_in_to_sn_profile could not be retrieved from /exposed-details");var c=a.institutional_business_partner_ids&&a.institutional_business_partner_ids.join?a.institutional_business_partner_ids.join(";"):"",d=a.resolved_by&&a.resolved_by.join?
a.resolved_by.join(";"):"",e=a.personal_business_partner_id_found,f=a.logged_in_to_sn_profile,g=a.snid;b.user={};b.user.profile={};b.user.profile.profileInfo={resolvedBy:d||null,bpid:c||null,personal:e||null,logged_in_to_sn_profile:f,snid:g||null};b.session={};b.session.authentication={};b.session.authentication.token=a.token||null;b.session.authentication.legacy={};window.dataLayer.push(b);window.idpUserDataLoaded=!0},!1);window.dataLayer.push({event:"accessdetailsloaded-handler-added"});
window.accessDetailsLoadedHandlerAdded=!0;</script><script type="text/javascript" id="gtm-mediatrust" charset="">var s=document.getElementsByTagName("script")[0],p=document.createElement("script");p.async="async";p.src="https://scripts.webcontentassessor.com/scripts/93dabb8d80079a87fec7bb6f67b807fce90e1688f8957ad7ad152bfd58ea01c2";s.parentNode.insertBefore(p,s);</script><script id="crossmark-script" text="" charset="" type="text/javascript" src="https://crossmark-cdn.crossref.org/widget/v2.0/widget.js"></script><script type="text/javascript" id="polyfill-matches" charset="">Element.prototype.matches||(Element.prototype.matches=Element.prototype.matchesSelector||Element.prototype.mozMatchesSelector||Element.prototype.msMatchesSelector||Element.prototype.oMatchesSelector||Element.prototype.webkitMatchesSelector||function(a){a=(this.document||this.ownerDocument).querySelectorAll(a);for(var b=a.length;0<=--b&&a.item(b)!==this;);return-1<b});</script><script type="text/javascript" id="fetch-idp-user-data" charset="">(function(a){if("function"===typeof window.CustomEvent)return!1;var c=function(d,b){b=b||{};var e=document.createEvent("CustomEvent");e.initCustomEvent(d,b.bubbles||!1,b.cancelable||!1,b.detail||a);return e};c.prototype=window.Event.prototype;window.CustomEvent=c})();var parse=function(a,c){try{return 200===a?JSON.parse(c):null}catch(d){return null}},dispatch=function(a){a=new CustomEvent("accessdetailsloaded",{detail:a});document.dispatchEvent(a)},site=google_tag_manager["rm"]["50443292"](59),idpUrl;
switch(site){case "nature":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?"https://staging-idp.nature.com/exposed-details":"https://idp.nature.com/exposed-details";break;case "link":idpUrl=-1<window.location.hostname.indexOf("link-qa")?"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details";break;case "springer":idpUrl=-1<window.location.hostname.indexOf("local-www")||-1<window.location.hostname.indexOf("test-www")?
"https://staging-idp.springer.com/exposed-details":"https://idp.springer.com/exposed-details"}if(void 0!==idpUrl){var transport=new XMLHttpRequest;transport.open("GET",idpUrl,!0);transport.withCredentials=!0;transport.onreadystatechange=function(){4===transport.readyState&&dispatch(parse(transport.status,transport.responseText))};transport.send()}else dispatch(null);</script><script src="https://verify.nature.com/verify/nature.min.js"></script><script type="text/javascript" id="create-ad-slots" charset="">function createAndLoadAds(){function h(b){"nature"===a&&google_tag_manager["rm"]["50443292"](87)(b);"bmc"!==a&&"springeropen"!==a||google_tag_manager["rm"]["50443292"](99)(b);("link"===a&&"oscar"===google_tag_manager["rm"]["50443292"](100)||"springer"===a||"link"===a&&!0===google_tag_manager["rm"]["50443292"](101))&&google_tag_manager["rm"]["50443292"](126)(b);"link"===a&&"bunsen"===google_tag_manager["rm"]["50443292"](127)&&google_tag_manager["rm"]["50443292"](152)(b);"link"===a&&"Core"===google_tag_manager["rm"]["50443292"](153)&&google_tag_manager["rm"]["50443292"](165)(b)}window.googletag=window.googletag||{cmd:[]};var a=google_tag_manager["rm"]["50443292"](166),e=google_tag_manager["rm"]["50443292"](168);
if(-1===window.location.search.indexOf("hide_ads\x3dtrue"))if(window.adSlots&&0!==Object.keys(window.adSlots).length)console.log("Ads previously loaded. Will not update ad slots until next page load.");else{window.adSlots||(window.adSlots={});window.getAd=function(b,f){for(var c in window.adSlots)if(-1<c.indexOf(b)){if("object"===f)return adSlots[c];if("slot"===f)return adSlots[c].slot}};googletag.cmd.push(function(){googletag.pubads().setPrivacySettings({limitedAds:e});console.log("limitedAds is ",
e);googletag.pubads().setRequestNonPersonalizedAds(google_tag_manager["rm"]["50443292"](170));googletag.pubads().enableSingleRequest();googletag.pubads().disableInitialLoad()});for(var g=document.querySelectorAll("[data-gpt]"),d=0;g[d];++d)h(g[d]);googletag.cmd.push(function(){googletag.enableServices()})}}createAndLoadAds();</script>
</body></html>